

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Conservation, Variability and the Modeling of Active Protein Kinases</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000982"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000982" />

    <meta name="citation_title" content="Conservation, Variability and the Modeling of Active Protein Kinases"/>
    <meta itemprop="name" content="Conservation, Variability and the Modeling of Active Protein Kinases"/>

      <meta name="citation_author" content="James D. R. Knight"/>
            <meta name="citation_author_institution" content="Molecular Medicine Program, Ottawa Health Research Institute, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="The University of Ottawa Centre for Neuromuscular Disease, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Department of Cellular and Molecular Medicine, University of Ottawa, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Bin Qian"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Washington, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="David Baker"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Washington, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Rashmi Kothary"/>
            <meta name="citation_author_institution" content="Molecular Medicine Program, Ottawa Health Research Institute, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="The University of Ottawa Centre for Neuromuscular Disease, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Department of Cellular and Molecular Medicine, University of Ottawa, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Department of Medicine, University of Ottawa, Ottawa, Ontario, Canada"/>

    <meta name="citation_date" content="2007/10/3"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000982.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e982"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The protein kinase complement of the human genome.; citation_author=G Manning; citation_author=DB Whyte; citation_author=R Martinez; citation_author=T Hunter; citation_author=S Sudarsanam; citation_journal_title=Science; citation_volume=298; citation_number=1; citation_pages=1912-1934; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the atypical protein kinase domain of a TRP channel with phosphotransferase activity.; citation_author=H Yamaguchi; citation_author=M Matsushita; citation_author=AC Nairn; citation_author=J Kuriyan; citation_journal_title=Mol Cell; citation_volume=7; citation_number=2; citation_pages=1047-1057; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A family portrait of the RIO kinases.; citation_author=N LaRonde-LeBlanc; citation_author=A Wlodawer; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=3; citation_pages=37297-37300; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Integrin-linked kinase (ILK): a regulator of integrin and growth-factor signaling.; citation_author=S Dedhar; citation_author=B Williams; citation_author=G Hannigan; citation_journal_title=Trends Cell Biol; citation_volume=9; citation_number=4; citation_pages=319-323; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Targets of the cyclin-dependent kinase Cdk1.; citation_author=JA Ubersax; citation_author=EL Woodbury; citation_author=PN Quang; citation_author=M Paraz; citation_author=JD Blethrow; citation_journal_title=Nature; citation_volume=425; citation_number=5; citation_pages=859-864; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Global analysis of protein phosphorylation in yeast.; citation_author=J Ptacek; citation_author=G Devgan; citation_author=G Michaud; citation_author=H Zhu; citation_author=X Zhu; citation_journal_title=Nature; citation_volume=438; citation_number=6; citation_pages=679-684; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene.; citation_author=DJ Slamon; citation_author=GM Clark; citation_author=SG Wong; citation_author=WJ Levin; citation_author=A Ullrich; citation_journal_title=Science; citation_volume=235; citation_number=7; citation_pages=177-182; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Chronic myeloid leukemia.; citation_author=CL Sawyers; citation_journal_title=N Engl J Med; citation_volume=340; citation_number=8; citation_pages=1330-1340; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Inhibition of stress-activated MAP kinases induces clinical improvement in moderate to severe Crohn's disease.; citation_author=D Hommes; citation_author=B van den Blink; citation_author=T Plasse; citation_author=J Bartelsman; citation_author=C Xu; citation_journal_title=Gastroenterology; citation_volume=122; citation_number=9; citation_pages=7-14; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=H-series protein kinase inhibitors and potential clinical applications.; citation_author=N Ono-Saito; citation_author=I Niki; citation_author=H Hidaka; citation_journal_title=Pharmacol Ther; citation_volume=82; citation_number=10; citation_pages=123-131; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of drug inhibition of signaling molecules.; citation_author=JS Sebolt-Leopold; citation_author=JM English; citation_journal_title=Nature; citation_volume=441; citation_number=11; citation_pages=457-462; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A small molecule-kinase interaction map for clinical kinase inhibitors.; citation_author=MA Fabian; citation_author=WH r d Biggs; citation_author=DK Treiber; citation_author=CE Atteridge; citation_author=MD Azimioara; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=12; citation_pages=329-336; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein structure prediction using Rosetta.; citation_author=CA Rohl; citation_author=CE Strauss; citation_author=KM Misura; citation_author=D Baker; citation_journal_title=Methods Enzymol; citation_volume=383; citation_number=13; citation_pages=66-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of A. fulgidus Rio2 defines a new family of serine protein kinases.; citation_author=N LaRonde-LeBlanc; citation_author=A Wlodawer; citation_journal_title=Structure; citation_volume=12; citation_number=14; citation_pages=1585-1594; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fold change in evolution of protein structures.; citation_author=NV Grishin; citation_journal_title=J Struct Biol; citation_volume=134; citation_number=15; citation_pages=167-185; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Plasticity of enzyme active sites.; citation_author=AE Todd; citation_author=CA Orengo; citation_author=JM Thornton; citation_journal_title=Trends Biochem Sci; citation_volume=27; citation_number=16; citation_pages=419-426; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The conserved lysine of the catalytic domain of protein kinases is actively involved in the phosphotransfer reaction and not required for anchoring ATP.; citation_author=AC Carrera; citation_author=K Alexandrov; citation_author=TM Roberts; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=17; citation_pages=442-446; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Mutation of position 52 in ERK2 creates a nonproductive binding mode for adenosine 5′-triphosphate.; citation_author=MJ Robinson; citation_author=PC Harkins; citation_author=J Zhang; citation_author=R Baer; citation_author=JW Haycock; citation_journal_title=Biochemistry; citation_volume=35; citation_number=18; citation_pages=5641-5646; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Catalytic independent functions of a protein kinase as revealed by a kinase-dead mutant: study of the Lys72His mutant of cAMP-dependent kinase.; citation_author=GH Iyer; citation_author=S Garrod; citation_author=J r Woods VL; citation_author=SS Taylor; citation_journal_title=J Mol Biol; citation_volume=351; citation_number=19; citation_pages=1110-1122; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Alpha-kinases: analysis of the family and comparison with conventional protein kinases.; citation_author=D Drennan; citation_author=AG Ryazanov; citation_journal_title=Prog Biophys Mol Biol; citation_volume=85; citation_number=20; citation_pages=1-32; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=WNK1, a novel mammalian serine/threonine protein kinase lacking the catalytic lysine in subdomain II.; citation_author=B Xu; citation_author=JM English; citation_author=JL Wilsbacher; citation_author=S Stippec; citation_author=EJ Goldsmith; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=21; citation_pages=16795-16801; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the kinase domain of WNK1, a kinase that causes a hereditary form of hypertension.; citation_author=X Min; citation_author=BH Lee; citation_author=MH Cobb; citation_author=EJ Goldsmith; citation_journal_title=Structure; citation_volume=12; citation_number=22; citation_pages=1303-1311; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=2.2 Å refined crystal structure of the catalytic subunit of cAMP-dependent protein kinase complexed with MnATP and a peptide inhibitor.; citation_author=J Zheng; citation_author=EA Trafny; citation_author=DR Knighton; citation_author=NH Xuong; citation_author=SS Taylor; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=49; citation_number=23; citation_pages=362-365; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Structure of PAK1 in an autoinhibited conformation reveals a multistage activation switch.; citation_author=M Lei; citation_author=W Lu; citation_author=W Meng; citation_author=MC Parrini; citation_author=MJ Eck; citation_journal_title=Cell; citation_volume=102; citation_number=24; citation_pages=387-397; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of an activated Akt/protein kinase B ternary complex with GSK3-peptide and AMP-PNP.; citation_author=J Yang; citation_author=P Cron; citation_author=VM Good; citation_author=V Thompson; citation_author=BA Hemmings; citation_journal_title=Nat Struct Biol; citation_volume=9; citation_number=25; citation_pages=940-944; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mutations in protein kinase subdomain X differentially affect MEKK2 and MEKK1 activity.; citation_author=J Huang; citation_author=Z Tu; citation_author=FS Lee; citation_journal_title=Biochem Biophys Res Commun; citation_volume=303; citation_number=26; citation_pages=532-540; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A subdomain of MEKK1 that is critical for binding to MKK4.; citation_author=Z Tu; citation_author=SM Mooney; citation_author=FS Lee; citation_journal_title=Cell Signal; citation_volume=15; citation_number=27; citation_pages=65-77; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Subdomain X of the kinase domain of Lck binds CD45 and facilitates dephosphorylation.; citation_author=J Felberg; citation_author=DC Lefebvre; citation_author=M Lam; citation_author=Y Wang; citation_author=DH Ng; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=28; citation_pages=3455-3462; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of a complex between the catalytic and regulatory (RIalpha) subunits of PKA.; citation_author=C Kim; citation_author=NH Xuong; citation_author=SS Taylor; citation_journal_title=Science; citation_volume=307; citation_number=29; citation_pages=690-696; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prediction of the structures of CASP7 targets using extensive all-atom refinement with Rosetta@home.; citation_author=R Das; citation_author=B Qian; citation_author=S Raman; citation_author=R Vernon; citation_author=J Thompson; citation_number=30; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=31; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=High resolution crystal structure of the human PDK1 catalytic domain defines the regulatory phosphopeptide docking site.; citation_author=RM Biondi; citation_author=D Komander; citation_author=CC Thomas; citation_author=JM Lizcano; citation_author=M Deak; citation_journal_title=EMBO J; citation_volume=21; citation_number=32; citation_pages=4219-4228; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=p38 map kinase substrate specificity differs greatly for protein and peptide substrates.; citation_author=J Hawkins; citation_author=S Zheng; citation_author=B Frantz; citation_author=P LoGrasso; citation_journal_title=Arch Biochem Biophys; citation_volume=382; citation_number=33; citation_pages=310-313; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Signaling specificity of Ser/Thr protein kinases through docking-site-mediated interactions.; citation_author=RM Biondi; citation_author=AR Nebreda; citation_journal_title=Biochem J; citation_volume=372; citation_number=34; citation_pages=1-13; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural basis for selective inhibition of Src family kinases by PP1.; citation_author=Y Liu; citation_author=A Bishop; citation_author=L Witucki; citation_author=B Kraybill; citation_author=E Shimizu; citation_journal_title=Chem Biol; citation_volume=6; citation_number=35; citation_pages=671-678; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Characterization of a conserved structural determinant controlling protein kinase sensitivity to selective inhibitors.; citation_author=S Blencke; citation_author=B Zech; citation_author=O Engkvist; citation_author=Z Greff; citation_author=L Orfi; citation_journal_title=Chem Biol; citation_volume=11; citation_number=36; citation_pages=691-701; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Peptide inhibitors of protein kinases-discovery, characterization and use.; citation_author=MA Bogoyevitch; citation_author=RK Barr; citation_author=AJ Ketterman; citation_journal_title=Biochim Biophys Acta; citation_volume=1754; citation_number=37; citation_pages=79-99; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Antitumor effect of a novel proapoptotic peptide that impairs the phosphorylation by the protein kinase 2 (casein kinase 2).; citation_author=SE Perea; citation_author=O Reyes; citation_author=Y Puchades; citation_author=O Mendoza; citation_author=NS Vispo; citation_journal_title=Cancer Res; citation_volume=64; citation_number=38; citation_pages=7127-7129; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A chemical genetic approach for the identification of direct substrates of protein kinases.; citation_author=K Shah; citation_author=KM Shokat; citation_journal_title=Methods Mol Biol; citation_volume=233; citation_number=39; citation_pages=253-271; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Procrustes methods in the statistical analysis of shape.; citation_author=C Goodall; citation_journal_title=J R Statist Soc B; citation_volume=53; citation_number=40; citation_pages=285-339; citation_date=1991; " />
      <meta name="citation_reference" content="citation_author=IL Dryden; citation_author=KV Mardia; citation_number=41; citation_date=1998; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Ecological correlates and evolutionary divergence in the skull of turtles: a geometric morphometric assessment.; citation_author=J Claude; citation_author=P Pritchard; citation_author=H Tong; citation_author=E Paradis; citation_author=JC Auffray; citation_journal_title=Syst Biol; citation_volume=53; citation_number=42; citation_pages=933-948; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=New directions in shape analysis.; citation_author=JT Kent; citation_number=43; citation_pages=115-127; citation_date=1992; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Understanding nature's catalytic toolkit.; citation_author=A Gutteridge; citation_author=JM Thornton; citation_journal_title=Trends Biochem Sci; citation_volume=30; citation_number=44; citation_pages=622-629; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase.; citation_author=DR Knighton; citation_author=JH Zheng; citation_author=LFT Eyck; citation_author=VA Ashford; citation_author=NH Xuong; citation_journal_title=Science; citation_volume=253; citation_number=45; citation_pages=407-414; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of the phosphorylated and unphosphorylated kinase domains of the Cdc42-associated tyrosine kinase ACK1.; citation_author=JC Lougheed; citation_author=RH Chen; citation_author=P Mak; citation_author=TJ Stout; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=46; citation_pages=44039-44045; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structural basis of cyclin-dependent kinase activation by phosphorylation.; citation_author=AA Russo; citation_author=PD Jeffrey; citation_author=NP Pavletich; citation_journal_title=Nat Struct Biol; citation_volume=3; citation_number=47; citation_pages=696-700; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of casein kinase-1, a phosphate-directed protein kinase.; citation_author=RM Xu; citation_author=G Carmel; citation_author=RM Sweet; citation_author=J Kuret; citation_author=X Cheng; citation_journal_title=EMBO J; citation_volume=14; citation_number=48; citation_pages=1015-1023; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Inclining the purine base binding plane in protein kinase CK2 by exchanging the flanking side-chains generates a preference for ATP as a cosubstrate.; citation_author=CW Yde; citation_author=I Ermakova; citation_author=OG Issinger; citation_author=K Niefind; citation_journal_title=J Mol Biol; citation_volume=347; citation_number=49; citation_pages=399-414; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of the catalytic domain of human protein kinase associated with apoptosis and tumor suppression.; citation_author=V Tereshko; citation_author=M Teplova; citation_author=J Brunzelle; citation_author=DM Watterson; citation_author=M Egli; citation_journal_title=Nat Struct Biol; citation_volume=8; citation_number=50; citation_pages=899-907; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the activated insulin receptor tyrosine kinase in complex with peptide substrate and ATP analog.; citation_author=SR Hubbard; citation_journal_title=EMBO J; citation_volume=16; citation_number=51; citation_pages=5572-5581; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The structure of phosphorylated p38gamma is monomeric and reveals a conserved activation-loop conformation.; citation_author=S Bellon; citation_author=MJ Fitzgibbon; citation_author=T Fox; citation_author=HM Hsiao; citation_author=KP Wilson; citation_journal_title=Structure; citation_volume=7; citation_number=52; citation_pages=1057-1065; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Two structures of the catalytic domain of phosphorylase kinase: an active protein kinase complexed with substrate analogue and product.; citation_author=DJ Owen; citation_author=ME Noble; citation_author=EF Garman; citation_author=AC Papageorgiou; citation_author=LN Johnson; citation_journal_title=Structure; citation_volume=3; citation_number=53; citation_pages=467-482; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Structural basis of constitutive activity and a unique nucleotide binding mode of human Pim-1 kinase.; citation_author=KC Qian; citation_author=L Wang; citation_author=ER Hickey; citation_author=J Studts; citation_author=K Barringer; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=54; citation_pages=6130-6137; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structure of Mycobacterium tuberculosis PknB supports a universal activation mechanism for Ser/Thr protein kinases.; citation_author=TA Young; citation_author=B Delagoutte; citation_author=JA Endrizzi; citation_author=AM Falick; citation_author=T Alber; citation_journal_title=Nat Struct Biol; citation_volume=10; citation_number=55; citation_pages=168-174; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Autophosphorylation of Archaeoglobus fulgidus Rio2 and crystal structures of its nucleotide-metal ion complexes.; citation_author=N LaRonde-LeBlanc; citation_author=T Guszczynski; citation_author=T Copeland; citation_author=A Wlodawer; citation_journal_title=FEBS J; citation_volume=272; citation_number=56; citation_pages=2800-2810; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nucleotide-induced conformational changes in the Saccharomyces cerevisiae SR protein kinase, Sky1p, revealed by X-ray crystallography.; citation_author=B Nolen; citation_author=J Ngo; citation_author=S Chakrabarti; citation_author=D Vu; citation_author=JA Adams; citation_journal_title=Biochemistry; citation_volume=42; citation_number=57; citation_pages=9575-9585; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the TAO2 kinase domain: activation and specificity of a Ste20p MAP3K.; citation_author=T Zhou; citation_author=M Raman; citation_author=Y Gao; citation_author=S Earnest; citation_author=Z Chen; citation_journal_title=Structure; citation_volume=12; citation_number=58; citation_pages=1891-1900; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The structure of Sky1p reveals a novel mechanism for constitutive activity.; citation_author=B Nolen; citation_author=CY Yun; citation_author=CF Wong; citation_author=JA McCammon; citation_author=XD Fu; citation_journal_title=Nat Struct Biol; citation_volume=8; citation_number=59; citation_pages=176-183; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000982" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Conservation, Variability and the Modeling of Active Protein Kinases"/>
    <meta name="twitter:description" content="The human proteome is rich with protein kinases, and this richness has made the kinase of crucial importance in initiating and maintaining cell behavior. Elucidating cell signaling networks and manipulating their components to understand and alter behavior require well designed inhibitors. These inhibitors are needed in culture to cause and study network perturbations, and the same compounds can be used as drugs to treat disease. Understanding the structural biology of protein kinases in detail, including their commonalities, differences and modes of substrate interaction, is necessary for designing high quality inhibitors that will be of true use for cell biology and disease therapy. To this end, we here report on a structural analysis of all available active-conformation protein kinases, discussing residue conservation, the novel features of such conservation, unique properties of atypical kinases and variability in the context of substrate binding. We also demonstrate how this information can be used for structure prediction. Our findings will be of use not only in understanding protein kinase function and evolution, but they highlight the flaws inherent in kinase drug design as commonly practiced and dictate an appropriate strategy for the sophisticated design of specific inhibitors for use in the laboratory and disease therapy."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000982.g009"/>

  <meta property="og:title" content="Conservation, Variability and the Modeling of Active Protein Kinases" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000982" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=968'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8893'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5923&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000982">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000982" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000982&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Conservation, Variability and the Modeling of Active Protein Kinases
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James D. R. Knight, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Molecular Medicine Program, Ottawa Health Research Institute, Ottawa, Ontario, Canada, 
                  The University of Ottawa Centre for Neuromuscular Disease, Ottawa, Ontario, Canada, 
                  Department of Cellular and Molecular Medicine, University of Ottawa, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bin Qian, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Washington, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Baker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Washington, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rashmi Kothary
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rkothary@ohri.ca">rkothary@ohri.ca</a></p>

                <p>Affiliations:
                  Molecular Medicine Program, Ottawa Health Research Institute, Ottawa, Ontario, Canada, 
                  The University of Ottawa Centre for Neuromuscular Disease, Ottawa, Ontario, Canada, 
                  Department of Cellular and Molecular Medicine, University of Ottawa, Ottawa, Ontario, Canada, 
                  Department of Medicine, University of Ottawa, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 03, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000982</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000982" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000982" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000982" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000982" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000982-g001" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-t001" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g002" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g003" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-t002" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g004" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g005" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g006" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g007" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g008" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000982-g009" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000982">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000982" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The human proteome is rich with protein kinases, and this richness has made the kinase of crucial importance in initiating and maintaining cell behavior. Elucidating cell signaling networks and manipulating their components to understand and alter behavior require well designed inhibitors. These inhibitors are needed in culture to cause and study network perturbations, and the same compounds can be used as drugs to treat disease. Understanding the structural biology of protein kinases in detail, including their commonalities, differences and modes of substrate interaction, is necessary for designing high quality inhibitors that will be of true use for cell biology and disease therapy. To this end, we here report on a structural analysis of all available active-conformation protein kinases, discussing residue conservation, the novel features of such conservation, unique properties of atypical kinases and variability in the context of substrate binding. We also demonstrate how this information can be used for structure prediction. Our findings will be of use not only in understanding protein kinase function and evolution, but they highlight the flaws inherent in kinase drug design as commonly practiced and dictate an appropriate strategy for the sophisticated design of specific inhibitors for use in the laboratory and disease therapy.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Knight JDR, Qian B, Baker D, Kothary R (2007) Conservation, Variability and the Modeling of Active Protein Kinases. PLoS ONE 2(10):
          e982.
            doi:10.1371/journal.pone.0000982</p><p><strong>Academic Editor: </strong>Sebastian Fugmann, National Institute on Aging, United States of America</p><p><strong>Received:</strong> July 20, 2007; <strong>Accepted:</strong> August 23, 2007; <strong>Published:</strong> October 3, 2007</p><p><strong>Copyright:</strong> © 2007 Knight et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a Canadian Institutes of Health Research Canadian Graduate Scholarship and a Multiple Sclerosis Society of Canada (MSSC) Studentship (JDRK), by an American Leukemia and Lymphoma Society Fellowship (BQ), the Howard Hughes Medical Institute (DB) and an MSSC grant (RK).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Protein kinases are the most ubiquitous single family of signaling molecules in the cell, accounting for approximately 2% of the proteins encoded by the human genome <a href="#pone.0000982-Manning1">[1]</a>. The simple mechanism of attaching an ATP-derived phosphate to a protein involves kinases in every aspect of cell behavior, from apoptosis to survival, proliferation to differentiation, maturation etc. Protein kinases provide a unique opportunity for understanding proteins in general by presenting us with a seeming paradox: wide scale similarity of sequence and structure combined with a diversity of behavioral consequences to their activity. The vast majority of protein kinases have readily detectable sequence similarity, which translates into structure. But even those known protein kinases that show no significant algorithm-detectable similarity at the level of sequence are believed to have very typical structures, as is evidenced by specific examples <a href="#pone.0000982-Yamaguchi1">[2]</a>, <a href="#pone.0000982-LaRondeLeBlanc1">[3]</a>. As they all have a shared function in transferring the terminal phosphate of ATP to another protein, similarity is understandable. Evidence to date also suggests a common catalytic mechanism (the possible exception may be the integrin-linked kinase <a href="#pone.0000982-Dedhar1">[4]</a>), whereby ATP and an active site divalent cation are bound in identical fashions and phosphotransfer is achieved by a shared set of amino acids. Studies in yeast <a href="#pone.0000982-Ubersax1">[5]</a>, <a href="#pone.0000982-Ptacek1">[6]</a> have shown that kinases can be promiscuous, phosphorylating hundreds of proteins, but they also have clear specificities. How is this specificity attained by one family of highly similar proteins? This paradox suggests the perfection of the kinase as an enzyme: a region ideally suited for the common function of catalysis, with another region(s) uniquely modifiable to attain substrate specificity without altering fold, compromising ligand binding or the subsequent reaction mechanism. A thorough understanding of this family of proteins would generate a tremendous knowledge base for discovering and predicting protein interactions, for designing highly specific and potent inhibitors, and, as a consequence of these facts, for understanding the cell and disease.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>As protein kinases are the key players in cell signaling, aberrations in their activity have been directly correlated with numerous disease states (for example, breast cancer <a href="#pone.0000982-Slamon1">[7]</a> and chronic myeloid leukemia <a href="#pone.0000982-Sawyers1">[8]</a>) and made them potential targets for drug design in many other diseases (for example, Crohn's <a href="#pone.0000982-Hommes1">[9]</a> and cerebral vasospasm <a href="#pone.0000982-OnoSaito1">[10]</a>). This has made the kinase the drug target of choice <a href="#pone.0000982-SeboltLeopold1">[11]</a>. However, there is an inherent flaw in traditional kinase inhibitor design. Almost all inhibitors target the ATP binding pocket based on a simple principle: if ATP cannot be bound, phosphorylation cannot occur. Building a molecule that can occupy this pocket is relatively simple, but since the ATP binding pocket and the regions in its immediate vicinity are the areas of greatest conservation, building a <em>specific</em> inhibitor is impossible. The inherent multi-target nature of inhibitors has been demonstrated by Fabian <em>et al</em>. <a href="#pone.0000982-Fabian1">[12]</a>, where the twenty compounds tested had multi-target coverage with only 23% of the kinome screened. Other ATP binding proteins could very likely display affinities for these compounds as well, making these inhibitors not just multi-kinase but multi-enzyme. In the laboratory, how can the effect of treating cells with such inhibitors be dissected? And when used for disease, what non-intended effects may arise in the targeted cell type or others over the long term? In the hopes of producing specific inhibitors, what is needed is a new approach to kinase drug design, one which logically targets the region of greatest dissimilarity.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>True dissimilarity can be known if similarity or conservation is understood in detail. For this, structure-based comparative approaches are needed to fully extract the information hidden in the three-dimensional protein-structure space. Traditional structure-driven alignment studies concentrate on maximizing fold overlap, and for the highly-similar protein kinase family which has a largely conserved fold, this can be a useful approach. But it is not necessarily the correct one, particularly where inhibitor design is concerned. Due to the information available in a three-dimensional space, structures can be aligned in other ways, for example by using geometry independent of connectivity. Fold can be ignored and focus directed upon residues free from their covalent associations. The positioning of side-chains and those functional groups involved in enzymatic catalysis and protein interactions can be directly overlain for studying similarity and variability. This type of alignment, and not that of fold, is of greater relevance for understanding protein interactions and therefore in designing small molecules or peptides to act as inhibitors.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Understanding the similar/conserved and dissimilar/non-conserved aspects of protein kinases allows for effective drug design. In addition, conservational studies will aid especially in structure prediction. There are at least 518 known human protein kinases <a href="#pone.0000982-Manning1">[1]</a> and deriving crystal structures for them all would involve a great deal of time and effort. As all known protein kinases have similar structures, homology-driven approaches to structure prediction that incorporate knowledge of conservation should prove fertile. Having a reliable predicted structural kinome would be of great practical use.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this paper we report on a structural analysis of and a modeling approach to active-conformation protein kinases. We describe the variability found between these kinases in terms of fold and amino-acid side-chain positioning. These results were produced using a novel structural alignment algorithm that will also be described. This algorithm superimposes structures independent of fold to maximize side-chain similarity. The result is not only an alignment but a consensus structure that depicts residue conservation as a distribution of amino acids and amino-acid categories. This consensus can be used to guide structure prediction, and we report here on its successful use with Rosetta <a href="#pone.0000982-Rohl1">[13]</a> in predicting the structure of 3-phosphoinositide dependent protein kinase-1 (PDK1) and the atypical protein kinase Rio2.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Alignment</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To examine conservation and variability between protein kinases we focused on a group of active-conformation structures. Obviously, it is important to examine like conformations so that any observed variability is in fact real. We defined an active kinase structure as one with ATP or a non-hydrolysable ATP analog, at least one divalent cation (always Mg<sup>2+</sup> or Mn<sup>2+</sup>), and any necessary phosphorylations. Kinases can be constitutively active or be regulated positively or negatively by phosphorylation, which is ultimately kinase specific. Information regarding the kinase structures used in our analysis can be found in <a href="#pone-0000982-t001"><strong>Table 1</strong></a>, and an example of an active-conformation protein kinase is shown in <a href="#pone-0000982-g001"><strong>Figure 1</strong></a>.</p>
<div class="figure" id="pone-0000982-g001"><div class="img"><a name="pone-0000982-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The structure of protein kinase A (PKA).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>PKA is shown in its active conformation with ATP in green sticks and Mn<sup>2+</sup> as black spheres. β-strands, helices and loops are labeled as in Knighton <em>et al</em>. <a href="#pone.0000982-Knighton1">[45]</a>. The active site is situated between the small and large lobes, located above and below ATP respectively. CL: catalytic loop; MPL: magnesium-positioning loop.</p>
<span>doi:10.1371/journal.pone.0000982.g001</span></div><div class="figure" id="pone-0000982-t001"><div class="img"><a name="pone-0000982-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Active-conformation kinase structures.<span class="xref"><sup>*</sup></span></span></strong></p><span>doi:10.1371/journal.pone.0000982.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Fifteen kinase structures were aligned using the sequence-order independent algorithm outlined in the Methods section to yield a consensus set of forty-four fully and partially conserved residues. The complete set is listed in <a href="#pone-0000982-t002"><strong>Table 2</strong></a>. The structural alignment produced is shown in <a href="#pone-0000982-g002"><strong>Figure 2</strong></a>. From such an image it can be seen that the overall kinase shape is a highly conserved feature. The active site occurs between two lobes: the small lobe above ATP and the large lobe below. Of particular importance for later discussion is the conservation of the substrate-binding groove, located between the catalytic loop, the P+1 loop, helix D, helix F, helix G and helix H. Conserved residues are shown in <a href="#pone-0000982-g003"><strong>Figure 3</strong></a>, in what we term a consensus structure. This is a distribution of amino acids and amino-acid categories conserved between the protein kinases we have examined. The consensus structure has three principle parts: 1) a region of hydrophobic residues clustered around the adenosine of ATP; 2) an area around the γ-phosphate of ATP – the active site – enclosed primarily by charged residues; and 3) a region in the large lobe, situated below ATP, of both hydrophobic and polar residues. The hydrophobic region around the adenosine creates a binding pocket for ATP. The charged residues in the active site bind and position the γ-phosphate, as well as the divalent cation, and participate in the catalytic mechanism. The conserved residues located in the large lobe serve to stabilize that region, and may play a role in mediating substrate interactions. Only five specific amino acids are fully conserved in all the kinases. These residues play critical parts in positioning ATP, stabilizing the active-conformation and in the catalytic mechanism. These are lysine 8, which interacts with the α- and β- phosphates of ATP, thereby stabilizing it. Glutamic acid 9, which forms a salt bridge with lysine 8 further stabilizing ATP. Aspartic acid 24 is the catalytic base that initiates phosphotransfer by deprotonating the acceptor serine, threonine or tyrosine. Asparagine 27 interacts with a secondary divalent cation, thereby positioning the γ-phosphate of ATP. And the final fully conserved residue is aspartic acid 34, which chelates the primary divalent cation, indirectly positioning ATP at the same time. Although these are the only residues fully conserved in terms of function, location and amino-acid type, there is one other residue with functional conservation but not locational or type, and in other kinases there is variability in the origin of lysine 8 and the type of amino acid fulfilling its role.</p>
<div class="figure" id="pone-0000982-g002"><div class="img"><a name="pone-0000982-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Multiple kinase alignment.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The fifteen active-conformation kinase structures listed in <a href="#pone-0000982-t001">Table 1</a> were aligned using our modified Procrustes approach. Shown in green sticks is the ATP or ATP analog molecule of each structure. Each kinase is colored uniquely.</p>
<span>doi:10.1371/journal.pone.0000982.g002</span></div><div class="figure" id="pone-0000982-g003"><div class="img"><a name="pone-0000982-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>A kinase consensus structure.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Each sphere represents a conserved residue. Red indicates full conservation of a particular amino acid in all fifteen kinase structures; orange, conservation in thirteen or fourteen structures; and yellow, conservation in eleven or twelve structures. Blue spheres indicate full conservation of an amino-acid category. The ATP molecule of protein kinase A is shown in green sticks. (A) The consensus structure consisting of the forty-four points listed in <a href="#pone-0000982-t002">Table 2</a>. (B) The consensus structure overlaid on the multiple alignment.</p>
<span>doi:10.1371/journal.pone.0000982.g003</span></div><div class="figure" id="pone-0000982-t002"><div class="img"><a name="pone-0000982-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Conserved residues found in the active-conformation kinases listed in <a href="#pone-0000982-t001">Table 1</a>.</span></strong></p><span>doi:10.1371/journal.pone.0000982.t002</span></div>

<h4>Rio2, ChaK and conserved residue variability</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Rio2 is the only atypical protein kinase amongst those included in our multiple alignment. No significant sequence similarity can be detected between Rio2 and conventional protein kinases. As such, it is classified with a distinct domain name (see <a href="#pone-0000982-t001"><strong>Table 1</strong></a>). The structure of the Rio family, as initially determined by <a href="#pone.0000982-LaRondeLeBlanc2">[14]</a>, shares significant similarity with serine/threonine and tyrosine kinases in the small lobe and in the regions of the large lobe directly adjacent to the active site (<a href="#pone-0000982-g004"><strong>Figure 4A</strong></a>). Dissimilarity in structure occurs predominantly in the large lobe, in regions involved in substrate specificity, suggesting Rio2 has evolved a novel mechanism of substrate recognition <a href="#pone.0000982-LaRondeLeBlanc2">[14]</a>. Our structural alignment algorithm concurs with these findings, showing high similarity in and around the active site (consensus residues 1–36) but not in the large lobe (consensus residues 37–44).</p>
<div class="figure" id="pone-0000982-g004"><div class="img"><a name="pone-0000982-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The atypical kinases (A) Rio2 and (B) transient receptor potential channel kinase (ChaK).</span></strong></p><span>doi:10.1371/journal.pone.0000982.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Slightly different results are produced for another atypical protein kinase: channel kinase (ChaK). The structure of this kinase was not included in our initial data set because it lacked a divalent cation in the active site. To examine structural similarity between ChaK and conventional protein kinases, we did align the partially active structure of ChaK (<a href="#pone-0000982-g004"><strong>Figure 4B</strong></a>) with the consensus structure generated from fully active-conformation kinases (see <a href="#pone-0000982-t002"><strong>Table 2</strong></a>). Most similarity is found in the region directly around the γ-phosphate of ATP, although there is some elsewhere. The large lobe, like Rio2, is quite distinct. One of the few differences between this kinase and the others is that the fully conserved asparagine 27 is replaced by the very similar carboxamide containing glutamine. The fully conserved aspartic acid 34 is present but does not align, likely due to the absence of a divalent cation.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We highlight these two kinases for the additional reason that they demonstrate structural variation within the bounds of certain functional constraints. An interesting case of this is lysine/arginine 26. In almost all serine/threonine kinases there is a lysine residue located two positions downstream of the catalytic aspartic acid (consensus residue 24). This lysine aids in orientating the γ-phosphate of ATP and it is thought may also act to neutralize the negative charge on this phosphate during catalysis. The position and orientation of these two residues with respect to ATP is shown in protein kinase A (PKA) in <a href="#pone-0000982-g005"><strong>Figure 5A</strong></a>. Tyrosine kinases, like activated CDC42 kinase (ACK1), do not have this lysine but instead have an arginine four positions downstream from the catalytic residue (<a href="#pone-0000982-g005"><strong>Figure 5B</strong></a>). This residue is orientated perpendicular to the lysine but occupies the same location, and since both are positively charged basic residues, both can fulfill the same function. ChaK, a serine/threonine kinase like PKA, utilizes a lysine for this function, which again occupies the same geometric position (<a href="#pone-0000982-g005"><strong>Figure 5C</strong></a>). However, this residue is not located two or four positions downstream, but thirty-eight positions upstream on a β-strand running adjacent to the catalytic loop. This is a good demonstration of the value behind a sequence-order independent alignment algorithm that ignores fold and residue connectivity. This lysine is located on part of a novel fold not found in any of the other kinases examined. Rio2 presents a fourth variation. Our alignment did not reveal Rio2 as having a basic residue at consensus point 26, as it did for all of the other kinases. Further examination led us to the conclusion that the function of this residue is conserved in Rio2 but the location of the residue accomplishing it is not (see also <a href="#pone.0000982-LaRondeLeBlanc2">[14]</a>). This is known as functional residue hopping <a href="#pone.0000982-Grishin1">[15]</a>, <a href="#pone.0000982-Todd1">[16]</a>. The function of consensus residue 26 is performed by a histidine located in the small lobe (<a href="#pone-0000982-g005"><strong>Figure 5D</strong></a>). This histidine is largely unique to the <em>Archaeoglobus fulgidus</em> Rio2 ortholog, from which the structure was derived – in most other species it is substituted by an arginine. As <em>A. fulgidus</em> is a hyperthermophile, the preference for histidine may be due to the extreme temperature environments in which it is found.</p>
<div class="figure" id="pone-0000982-g005"><div class="img"><a name="pone-0000982-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Residue variability in positioning the γ-phosphate of ATP.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>In each kinase the catalytic aspartic acid is shown in red and the positively-charged residue that interacts with the γ-phosphate of ATP is shown in yellow with nitrogen atoms colored blue. ATP is shown in green sticks. (A) Protein kinase A, (B) activated CDC42 kinase 1, (C) channel kinase, and (D) Rio2 kinase.</p>
<span>doi:10.1371/journal.pone.0000982.g005</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The only other variability we have found in conserved residues is for the so-called catalytic lysine (lysine 8). The function of this residue is somewhat debated <a href="#pone.0000982-Carrera1">[17]</a>, <a href="#pone.0000982-Robinson1">[18]</a>, <a href="#pone.0000982-Iyer1">[19]</a>, but at the very least it appears to position the phosphates of ATP and is absolutely crucial for catalysis. This residue is normally found on β-strand 3 and interacts with the α- and β-phosphates of ATP (<a href="#pone-0000982-g006"><strong>Figure 6A</strong></a>). In ChaK there is a homologous lysine present but it interacts with the α-phosphate and the adenine ring of ATP (<a href="#pone-0000982-g006"><strong>Figure 6B</strong></a>). Depending upon the alignment parameters used, this lysine can align structurally with that found in other kinases, although for the parameters we have chosen it does not. Instead there is an arginine residue that aligns and this very similar residue interacts with both the α- and β-phosphates of ATP (<a href="#pone-0000982-g006"><strong>Figure 6B</strong></a>), just as the catalytic lysine normally does. Although ChaK lacks a divalent cation, this is unlikely to affect the positions of these residues. The arginine in question, R1622, is fully conserved in the alpha kinase family <a href="#pone.0000982-Drennan1">[20]</a>. It is not known which residue in ChaK is functionally homologous to the catalytic lysine in typical protein kinases. Likely both share the function, and this represents a distinct feature of the alpha-kinase family. One other kinase is known to have a novelty in this area. This is the protein kinase with no lysine (WNK) kinase, named for the apparent absence of the catalytic lysine on β-strand 3. It was predicted by <a href="#pone.0000982-Xu1">[21]</a> that a lysine on β-strand 2 could be structurally equivalent (<a href="#pone-0000982-g006"><strong>Figure 6C</strong></a>), and the absolute requirement for a lysine at this position was confirmed by this group. A subsequent crystal structure appears to confirm these predictions <a href="#pone.0000982-Min1">[22]</a>. It is interesting that this lysine originates from the same position as the arginine in ChaK, showing that variation in conserved residues is possible and highlighting how proteins can incorporate novel features within certain functional constraints.</p>
<div class="figure" id="pone-0000982-g006"><div class="img"><a name="pone-0000982-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Variations in the catalytic lysine.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>(A) In almost all protein kinases (such as protein kinase A shown) a lysine residue originating from β-strand 3 interacts with the α- and β-phosphates of ATP. This lysine is required for catalytic activity and has been termed the catalytic lysine. (B) In channel kinase (ChaK), the homologous lysine interacts with the α-phosphate and the adenine ring of ATP. An unique arginine residue located on β-strand 2 instead interacts with the α- and β-phosphates. (C) In protein kinase with no lysine 1 (WNK1, PDB code: 1T4H <a href="#pone.0000982-Min1">[22]</a>), the catalytic lysine is present but originates from β-strand 2, much like the arginine from ChaK. ATP is shown in green sticks. Potential hydrogen bonds between the positively charged residues and ATP are shown as dashed lines, and nitrogen atoms are colored blue. The WNK structure has no ATP or ATP analog.</p>
<span>doi:10.1371/journal.pone.0000982.g006</span></div>

<h4>Substrate specific variation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The conservation we are detecting is found first and foremost in and around the active site, but is present elsewhere in the protein kinase domain. The single exception is in the cleft formed by the catalytic loop, the P+1 loop, helix D and the residues from the end of helix F through to the beginning of helix H. This is known as the substrate-binding cleft, and residues in this region have been shown crucial to substrate binding and regulatory protein-protein interactions <a href="#pone.0000982-Zheng1">[23]</a>, <a href="#pone.0000982-Lei1">[24]</a>, <a href="#pone.0000982-Yang1">[25]</a>, <a href="#pone.0000982-Huang1">[26]</a>, <a href="#pone.0000982-Tu1">[27]</a>, <a href="#pone.0000982-Felberg1">[28]</a>, <a href="#pone.0000982-Kim1">[29]</a>. The absence of conservation in this cleft is entirely expected. Protein kinases are substrate specific and that specificity is at least in part conferred through residue variability within this groove.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The crystal structures of PKA and Akt we have used in our analysis contain substrate peptides bound in this region. In <a href="#pone-0000982-g007"><strong>Figure 7</strong></a> we show PKA and Akt with bound substrate peptides and display variability in the context of proximity to the substrate and distance from conserved residues. In both PKA and Akt, there are a number of atoms in the substrate-binding cleft adjacent to or near the substrate peptide that are distant from conserved residues. We highlight this for the purpose of discussing inhibitor design. Successful inhibitor design requires a region where small molecules or peptides can be bound with high specificity. The absence of specificity comes from drug targets hitting regions of residue conservation. The substrate-binding cleft obviously has a binding capacity and, as shown, this region is highly variable. Designing inhibitors that target the atoms in this region, mimicking those residues present in substrates or regulatory proteins, would be a fruitful approach.</p>
<div class="figure" id="pone-0000982-g007"><div class="img"><a name="pone-0000982-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Substrate-specific variability.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Cross-eyed stereo views of (A) protein kinase A and (B) Akt2. Bound substrate peptides are shown in red sticks. Atoms belonging to non-conserved residues within 10 Å of the substrate peptide are shown as colored spheres. Red: atoms within 2 Å of a conserved residue; orange: atoms between 2 and 4 Å of a conserved residue; yellow: atoms between 4 and 6 Å of a conserved residue; green: atoms between 6 and 8 Å of a conserved residue; cyan: atoms between 8 and 10 Å of a conserved residue; and blue: atoms more than 10 Å from a conserved residue.</p>
<span>doi:10.1371/journal.pone.0000982.g007</span></div>

<h4>Structure prediction</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The conservation we are finding can be used for purposes other than understanding protein function, evolution and guiding drug design. It can also be used to predict the structure of protein kinases (and to other proteins if applied). The consensus structure shown in <a href="#pone-0000982-g003"><strong>Figure 3</strong></a> represents the typical position of specific and conserved amino acids or amino-acid categories. If this is a distribution that active protein kinases tend to adopt, then predicted structures of active-conformation protein kinases should be made to meet these criteria. At the most basic level, models can be generated through any method and then screened against a consensus structure to discriminate between good and bad models, or, alternatively, residues known to be equivalent from a sequence alignment can be forced to meet the constraints seen in the consensus structure and the rest of the protein can be modeled within this framework. We have explored these possibilities with the structure prediction method Rosetta <a href="#pone.0000982-Das1">[30]</a> and attempted to model two kinases, the typical 3-phosphoinositide dependent protein kinase-1 and the atypical Rio2 kinase.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We used fourteen active-conformation kinases (omitting Rio2) and generated a 52 residue consensus structure (consensus residues are listed in <a href="#pone.0000982.s001"><strong>Supplementary Table S1</strong></a>). A sequence alignment of these fourteen kinases and PDK1 was then generated using ClustalW <a href="#pone.0000982-Thompson1">[31]</a>. Residues from PDK1 apparently equivalent with those of the consensus were selected based on this sequence alignment (see <a href="#pone.0000982.s001"><strong>Supplementary Table S1</strong></a>). These residues were then constrained geometrically in accordance with the consensus while PDK1 was modeled with Rosetta as described in the Methods section. When compared against the partially active-conformation structure of PDK1 (PDB code: 1H1W, <a href="#pone.0000982-Biondi1">[32]</a>), the top ranked prediction had 198 of 285 side chains positioned within 2 Å of their actual location, a C<sub>α</sub> RMSD of 1.3 Å and an all-atom RMSD of 1.6 Å (<a href="#pone-0000982-g008"><strong>Figure 8A</strong></a>). The floor of the active site, where most conservation occurs, is highly accurate, with 24 of 25 side chains positioned with 2 Å, C<sub>α</sub> RMSD of 0.5 Å and an all-atom RMSD of 0.7 Å (<a href="#pone-0000982-g008"><strong>Figure 8B</strong></a>). In non-conserved regions, such as the substrate-binding groove, good modeling is dependent upon the ability of the prediction method applied. A well-proven method like Rosetta is, therefore, a good complement. 24 of 38 residues in the substrate-binding groove are within 2 Å of their actual position, a C<sub>α</sub> RMSD of 0.9 Å and an all-atom RMSD of 1.4 Å (<a href="#pone-0000982-g008"><strong>Figure 8C</strong></a>). Accuracy in this region may be due in part to the constraints applied elsewhere, which would reduce the potential conformational space to search.</p>
<div class="figure" id="pone-0000982-g008"><div class="img"><a name="pone-0000982-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Modeling a typical kinase.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>PDK1 was modeled using the consensus guided approach outlined in the Results and Methods sections. (A) The best model is in blue and the actual structure in red (PDB code: 1H1W). Structures are shown aligned based on C<sub>α</sub> RMSD. (B) Residues 201–225, comprising the floor of the active site, are highlighted as aligned on side chains. (C) Residues from the non-conserved regions of the substrate binding groove (residues 166–175 and 278–305) are shown as aligned on side chains. ATP from the true structure of PDK1 is shown in green sticks.</p>
<span>doi:10.1371/journal.pone.0000982.g008</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Rio2 kinase was modeled initially without constraints as these cannot be derived from a sequence alignment. 80,000 models were generated and the lowest 5% (in full atom energy) were screened against the consensus structure. The top ten were then scrutinized for potential constraints. All ten of the top models unambiguously agreed on the likely equivalent residues for the fully conserved lysine 7 (K120 in Rio2), aspartic acid 24 (D218), asparagine 27 (N223) and aspartic acid 34 (D235). In none of the models could a residue equivalent to the fully conserved glutamic acid 8 be found (it should be E154). We then proceeded with a second modeling phase using constraints from the consensus for K120, D218, N223 and D235, and allowing two candidates for the conserved glutamic acid: E134 or E154. Although models generated with E134 as the conserved glutamic acid scored equally well when compared against the consensus structure, they would likely be deemed implausible by visual inspection as helix C was distorted upwards away from the active site instead of lining the back of the ATP binding pocket as it does in other kinases. The best model for E154 as determined against the consensus structure is shown in <a href="#pone-0000982-g009"><strong>Figure 9A</strong></a> alongside the actual active-conformation structure (PDB code: 1ZAO). Fifty four of 180 side-chains are positioned within 2 Å of their true location. C<sub>α</sub> and all-atom RMSD are 6.1 Å and 7.1 Å respectively. However, these values do not accurately reflect the quality of the model. The N-terminus of protein kinases is a non-conserved region and the C-terminal lobe of Rio2 differs greatly from that of other kinases. It should not be expected that a structural-based consensus approach would be able to distinguish a correct model from an incorrect model in these regions. Looking solely at the conserved kinase regions, comprising residues 93–242 of Rio2, the C<sub>α</sub> and all-atom RMSD drop to 3.0 Å and 3.7 Å respectively. And all of the correctly positioned residues are found in this region. The floor of the active site has 16 of 24 side chains positioned with 2 Å, C<sub>α</sub> RMSD of 1.0 Å and an all-atom RMSD of 1.7 Å (<a href="#pone-0000982-g009"><strong>Figure 9B</strong></a>). A final note: although residues located in the non-conserved large lobe helices are not correctly positioned, this region is topologically correct.</p>
<div class="figure" id="pone-0000982-g009"><div class="img"><a name="pone-0000982-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000982" data-uri="info:doi/10.1371/journal.pone.0000982.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000982.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000982.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000982.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Modeling an atypical kinase, Rio2.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>(A) The best model is in blue and the actual structure in red (PDB code: 1ZAO). Structures are shown aligned based on C<sub>α</sub> RMSD. Residues 128–142 from the model are hidden because they are disordered in the actual structure. (B) Residues 214–237 are highlighted as aligned on side-chains. ATP is shown in green sticks.</p>
<span>doi:10.1371/journal.pone.0000982.g009</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The kinase domain can be conceptualized as two functional modules. A highly conserved ATP-binding and catalytic module, located between the small and large lobes, found in all typical and atypical protein kinase structures. Little variation is found here, with only a few kinases, such as Rio2, ChaK and WNK, containing structural (but not functional) novelties. The second module is involved in substrate binding and evidence suggests is localized primarily to the large lobe, in the region named the substrate/peptide binding groove (named from kinase-peptide co-crystal structures). Very little is known about this region of kinases as detailed knowledge would require many kinase structures co-crystallized with full-length substrates, and as yet no single such structure exists. Conservation of the groove fold suggests a common substrate-binding mechanism, while the absence of residue conservation in this region across the kinase family suggests the means by which substrate specificity is determined. The part of this groove located near the active site (between the catalytic and P+1 loops) does contain conserved residues and is entirely consistent with the broad peptide-substrate specificity of kinases. But peptides are not the same as full-length proteins, and studies on the mitogen-activated protein kinase p38 have shown that proteins can be phosphorylated at a hundred to thousand fold greater rate than peptides <a href="#pone.0000982-Hawkins1">[33]</a>. More than just the residues immediately attached to and surrounding the target serine, threonine or tyrosine are involved in the kinase-substrate interaction, and the general mechanism of interaction needs to be determined.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Although the substrate/peptide binding groove may be the key feature for understanding substrate specificity in most kinases, it is and need not be in all. The substrate-binding module can vary in three ways: typical kinases that preserve fold and merely vary the residues in the substrate-binding groove, typical kinases than rely on other regions/regulatory proteins in addition to the substrate-binding groove, and atypical kinases that vary the binding mechanism all together. It should be added that even in very typical kinases substrate interactions are not localized purely to the substrate-binding groove (see <a href="#pone.0000982-Biondi2">[34]</a>). Much work needs to be done on this area of protein kinases, and current knowledge of substrate-binding mechanisms is rudimentary.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>A thorough understanding of substrate interactions will help in discovering new proteins targeted by kinases. Insights into binding can guide computational docking approaches that test the fit of different structures onto kinases as a new means of substrate finding. More experimentally determined structures or advances in structure prediction would be a necessary requirement for this to have broad applicability. This may be a ways off yet, but a closer goal may be in applying structural binding information to the design of specific kinase inhibitors. Currently, inhibitors are primarily designed to target the ATP binding pocket, the region of greatest conservation amongst kinases. The ATP binding pocket does contain unique features between certain subsets of the kinome (the gatekeeper residue is an example <a href="#pone.0000982-Liu1">[35]</a>, <a href="#pone.0000982-Blencke1">[36]</a>), and these novelties can be exploited by drugs. However, this type of design approach will always be plagued by problems of specificity on the simple foundation that this region has evolved to bind a single thing: ATP. On the other hand, the substrate-binding groove has evolved to bind kinase-specific substrates and our analysis demonstrates that this is accompanied by large variability in the substrate-binding groove. Mimicking these substrates with peptides or small molecules that compete with substrates has proven effective in several cases (reviewed in <a href="#pone.0000982-Bogoyevitch1">[37]</a>), and a peptide inhibitor that mimics a substrate interaction domain has been shown useful <em>in vivo</em> at reducing tumor mass <a href="#pone.0000982-Perea1">[38]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Of great benefit in this regard would be data on kinase substrates. If wide-ranging data on kinase substrate pools were known, inhibitors could be designed to mimic exclusive targets. Or if there was a desire to inhibit multiple kinases at one time, a small molecule that mimics a common substrate could be made. Ultimately the viability of such a strategy would again require a greater output of crystal and NMR structures, advances in structure prediction and protein docking, but these are popular areas of research and should not prove a hindrance. A dendrogram classification of kinases based upon substrates would be of tremendous use for inhibitor design, cell biology and evolutionary studies. Such an aim would require systematic methods of substrate finding, which are beginning to become available <a href="#pone.0000982-Shah1">[39]</a>, <a href="#pone.0000982-Ptacek1">[6]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Protein structures are the key to what we have presented and are discussing. Unless there are significant advances in the rate and ease with which proteins can be cloned, expressed, purified and structures subsequently determined, reliable computational approaches at structure determination will be needed. We have found that structural conservation gives insight not only into protein function but also can be used with success for structure prediction. We have shown how knowledge of residue conservation can help in generating or selecting appropriate protein kinase models. The atypical Rio2 kinase, which possesses no significant sequence similarity to other protein kinases, would be a difficult target for standard approaches at homology modeling. As few as five geometric residue constraints derived from a consensus structure and further screening can select a highly accurate model. For a typical kinase, such as PDK1, where many constraints can be used (50 in our case), it was a simple matter to generate a highly accurate model. Residue conservation and other types of structural similarity quantified over protein families can provide the basis for model selection and act as guides that reduce the target search space for the computationally demanding task of structure prediction.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Understanding the individual role of every protein kinase in the genome is a daunting task, but necessary because of the important signaling role played by this family of enzymes and the potential for disregulation and subsequent disease. Ultimately, cellular studies that dissect the role of kinases <em>in vivo</em> and the treatment of disease will require a library of specific and multi-target inhibitors. Such an aim will require a strong union between computational, molecular and structural approaches, but this goal may be close at hand. Although empirically determined structures of all kinases are a long way off, quality models can be built. Kinome wide data on substrate pools will require a great deal of work, but the techniques are now becoming available. The mechanism(s) of substrate binding will need to be elucidated in detail. This may be the greatest challenge, and towards which maximum focus should be directed, as not a single kinase-substrate structure exists. But, if such can be achieved in a representative sample of cases, kinase-substrate specific docking algorithms may be able to do the rest. The attainment of these aims would allow for the quick and reliable design of specific inhibitors, thereby allowing the precise function(s) of kinases to be determined and creating the means by which signaling networks can be precisely manipulated.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Structural alignment overview</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The results presented in this paper were generated through the use of a multiple structure alignment algorithm we have developed. The motivation was to have a method for superimposing protein structures independent of residue connectivity, thereby facilitating comparisons between proteins that have distinct folds and maximize side-chain as opposed to main-chain overlap. The field of shape analysis, specifically the Procrustes methods, provided the foundation for accomplishing this task. What follows is an overview of Procrustes and a description of its modification and application to protein structure comparison as we have implemented it.</p>


<h4>Procrustes</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Procrustes is a method for comparing matrices and thereby any group of objects that can be represented in such a way <a href="#pone.0000982-Goodall1">[40]</a>, <a href="#pone.0000982-Dryden1">[41]</a>. It is part of the broader field of shape analysis which has its motivation in describing and understanding variation between objects through multivariate analysis. Procrustes is often applied in biological and anthropological studies of morphology, for example in studying the relationship between turtle skull shape and life style across species <a href="#pone.0000982-Claude1">[42]</a>. It has been used in agriculture, genetics, geography, geology and psychology to name a few fields. Although multivariate analysis is not of interest, the Procrustes method is valuable for its ability to efficiently superimpose objects. Procrustes superimposition initiates from a set of points, each of which has a representative in every object under consideration. These points, known as landmarks, must have some level of correspondence and be located within a geometric space. By manipulating scale, location and orientation, Procrustes aims to produce an optimal superimposition as defined by some criteria that acts to minimize distance between corresponding points. The term partial Procrustes specifically refers to the subset of methods that maintain scale, i.e. those approaches that only perform rigid body transformations <a href="#pone.0000982-Kent1">[43]</a>.</p>


<h4>Application and algorithm</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>An application for Procrustes in protein structure alignment is evident, where landmarks could be viewed as equivalent residues in a space of three dimensions. The term equivalent accounts for functional equivalence and/or homology, although emphasis should be placed on the former. Specifically it is the partial Procrustes method that is applicable since initial aspects of scale should be maintained in the resulting alignment.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Standard implementations of Procrustes require a set of user-specified landmarks with representatives in every input object. These two conditions are inadequate for multiple structure alignments. The primary motivation behind a structural alignment should be to find conserved residues, and a good algorithm must give leeway for less than total conservation. The algorithm we have developed allows for both. It proceeds in two phases. The first is a series of pair-wise comparisons to find residues conserved across all samples. Residue conservation means in terms of both location and amino-acid type (identity/similarity). The second phase begins with an initial multiple alignment using the landmarks found in the first phase. From the multiple alignment the set of landmarks is then extended to include any additional fully conserved residues not found in the first phase, and residues with less than total conservation. The multiple alignment and extension steps are repeated until a final non-extendable set is obtained or upon completion of a predefined number of iterations.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>In this approach, each of <em>n</em> structures is represented by a <em>l<sub>i</sub></em>×3 coordinate matrix, where <em>l<sub>i</sub></em> equals the number of residues in structure <em>i</em>. The mean of the non-hydrogen side chain atoms is used as the coordinate for each residue, with the exception of glycine residues for which the alpha carbon (C<sub>α</sub>) is used. Most structure alignment algorithms use the C<sub>α</sub> for all residues. This is inadequate. Residue function should be viewed largely in terms of side-chain characteristics: what atoms make up the side chain and where they are positioned. It is the side chain that is principally used to interact with ligands, substrates and other residues <a href="#pone.0000982-Gutteridge1">[44]</a>, making knowledge of its position the crucial feature. Superimposed C<sub>α</sub>'s of similar residues may have side chains positioned in different locations and are therefore unlikely to have the same function. For these reasons focus is placed on side-chain positioning.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>One object (structure) can be superimposed onto a second in three dimensions if at least three equivalent residues are known. It is possible these residues may or could be known beforehand, although this is not ideal. An alternative involves a search and score approach: choose three residues from each sample that have corresponding amino-acid labels and share some geometric feature, then score the alignment produced from superimposing these residue triplets. The highest scoring alignment produced from a series of triplet superimpositions can then be used to produce a set of residues conserved across a pair of samples (we term this a pair-wise consensus set). This approach has been employed in a hash table. For all samples every triplet of residues is stored based on the amino-acid labels of its members and the inter-residue distances are indexed. Each entry of the table can be thought of as a triangle with vertices labeled according to amino acid. Between pairs of samples highly congruent triangles with matching vertices are used as candidate triplets likely to produce high scoring superimpositions. Such criteria reduce the time spent on triplets least likely to be part of the final pair-wise consensus set. We have used congruency criteria of 1–10% as indicated below, i.e. corresponding vertices must have magnitudes within 1–10% of each other.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5"></a><p>The method of superimposition is taken from Procrustes. Candidate triplets from each sample <em>i</em> are stored in an 3×3 coordinate matrix <em>X<sub>i</sub></em>. These matrices are first Helmertized (centered), removing translational differences:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e002&amp;representation=PNG" class="inline-graphic"></span> is the centered matrix and <em>H</em> the (<em>m</em>−1)×<em>m</em> Helmert sub-matrix whose <em>j</em><sup>th</sup> row is equal to (<em>h<sub>j</sub></em>,…, <em>h<sub>j</sub></em>, −<em>jh<sub>j</sub></em>,0,…,0) for <em>h<sub>j</sub></em> = −[<em>j</em>(<em>j</em>+1)]<sup>−1/2</sup>. Other approaches for removing translation are available. The optimal rotation of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e003&amp;representation=PNG" class="inline-graphic"></span> onto <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e004&amp;representation=PNG" class="inline-graphic"></span> is found by minimizing<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where Γ is a 3×3 rotation matrix. It can be shown that if <em>VΔU<sup>T</sup></em> is the singular value decomposition of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e006&amp;representation=PNG" class="inline-graphic"></span>, the optimal rotation Γ = <em>UV<sup>T</sup></em> (provisions must be made to ensure Γ does not also encompass a reflection).</p>
<a id="article1.body1.sec4.sec3.p6" name="article1.body1.sec4.sec3.p6"></a><p>The translation resulting from centering triplets is then applied to each structure and the appropriate sample is rotated by Γ. This superimposes the equivalent triplet residues and pulls the rest of each structure along. The result is that the initial pair-wise consensus set a triplet comprises may be extended from the structure-wide superimposition. Residues from the superimposed structures are considered to be equivalent (i.e. in the pair-wise consensus set) if they are identical or similar amino acids within two angstroms of one another. Similar means sharing a category, with those allowed including acidic (D, E, Z), aliphatic (A, G, I, L, V), aromatic (F, H, W, Y), basic (H, K, R), charged (D, E, H, K, R, Z), hydrophobic (A, C, F, G, I, L, M, P, V, W, Y), polar (C, D, E, H, K, N, Q, R, S, T, Y, Z), small (A, C, D, G, N, P, S, T, V) and very small (A, G, S), where Z refers to a phosphorylated serine, threonine or tyrosine.</p>
<a id="article1.body1.sec4.sec3.p7" name="article1.body1.sec4.sec3.p7"></a><p>This procedure is performed on all triplets that match the congruency criteria and all alignments are scored based on the frequency with which the amino acids of any equivalent residues occur. The frequency of occurrence for each amino acid was obtained from the protein knowledge-base release statistics of Swiss-Prot (Release 49.5, 18/04/06). Tryptophan is the lowest occurring amino acid and superimposed tryptophans are therefore given a score of one. All other scores are relative to that of tryptophan: score(amino acid x) = occurrence(W/x). The occurrence of each category is equal to the sum of the occurrences of its members. For <em>m</em> equivalent residues, the alignment score is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e007&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <em>x<sub>i</sub></em> is the amino acid or amino-acid category of the <em>i</em><sup>th</sup> equivalent residue.</p>
<a id="article1.body1.sec4.sec3.p8" name="article1.body1.sec4.sec3.p8"></a><p>The top 100 scoring triplets are subjected to a second phase of testing to find maximum similarity between two structures. For these triplets the initial pair-wise consensus set is used for a second superimposition. In this case the <em>X<sub>i</sub></em> coordinate matrices take on dimensions <em>m</em>×3, for <em>m</em> equivalent residues in the pair-wise consensus set. The superimposition and extension steps are repeated until a final non-extendable set is obtained.</p>
<a id="article1.body1.sec4.sec3.p9" name="article1.body1.sec4.sec3.p9"></a><p>Computational time and memory are minimized in three ways. First, as has been mentioned, congruency between triangles is used to reduce the number of triplets tested. Second, the vertices of these triplets are only indexed by amino acid and not by category as well. If vertices could be indexed by amino-acid category, such as hydrophobic, the number of candidate triplets to search scales tremendously, as does demands upon memory if a hash table is employed. Third, we restrict triplets to a core of residues. For a structure of 300 residues there are nearly 27 million candidate triplets. Storing all in a hash table is too demanding, and is not necessary either. The best candidate triplets will be clustered in a core, which may be comprised of residues around an active site, ligand binding domain or phosphorylation site. Only searching this core will provide good candidate triplets and minimize time. By default a geometric core of fifty residues is used.</p>
<a id="article1.body1.sec4.sec3.p10" name="article1.body1.sec4.sec3.p10"></a><p>From the input structures a consensus set <em>S<sub>n</sub></em> is sought for of residues fully conserved across all <em>n</em> samples (a full consensus set). This can be done quickly through a series of pair-wise comparisons. First a consensus set <em>S<sub>2</sub></em> of conserved residues is sought for between samples 1 and 2 using the superimposition procedure outlined above. A second set <em>S<sub>3</sub></em> of conserved residues found in <em>S<sub>2</sub></em> and sample 3 is then sought for using the same procedure of selecting triplets from the consensus set and the sample. This process is repeated until a final full consensus set <em>S<sub>n</sub></em> is produced. From experience the procedure for deriving <em>S<sub>n</sub></em> is generally sample order independent, except in some cases where an extreme outlier is present. This is a structure with few of the conserved residues found across the other <em>n</em>−1 samples. In this case the outlier may align well with a set of residues conserved across the first <em>x</em> samples but not across the final <em>n</em>−<em>x</em>−1. By aligning well with a subset of <em>S<sub>x</sub></em> not in <em>S<sub>n</sub></em>, the outlier skews the derivation process away from the true set <em>S<sub>n</sub></em>. This can be accounted for by leaving such outliers until the end, by which time the consensus set has been reduced to a true group of highly conserved residues.</p>
<a id="article1.body1.sec4.sec3.p11" name="article1.body1.sec4.sec3.p11"></a><p>The multiple alignment procedure is a modification of the pair-wise superimposition method outlined above. The full consensus set <em>S<sub>n</sub></em> found through the pair-wise superimpositions is comprised of a set of <em>m</em> landmarks that can be used to simultaneously align the group of <em>n</em> structures. First, the conserved residues from each sample <em>i</em> are stored in an <em>m</em>×3 matrix <em>X<sub>i</sub></em> and Helmertized. The optimal rotation matrix for each structure Γ<em><sub>i</sub></em> is calculated by an iterative process that seeks to minimize the following value:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>The rotation Γ<em><sub>i</sub></em> is calculated from the singular value decomposition of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e009&amp;representation=PNG" class="inline-graphic"></span> with the mean of the rest, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e010&amp;representation=PNG" class="inline-graphic"></span>, for<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>However, with each successive <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e012&amp;representation=PNG" class="inline-graphic"></span> rotated, the previous <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e013&amp;representation=PNG" class="inline-graphic"></span> for <em>j</em>&lt;<em>i</em> will change and the previously calculated rotation of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000982.e014&amp;representation=PNG" class="inline-graphic"></span> will be incorrect. To account for this, after every sample has been rotated <em>G</em> is calculated and the process of rotation is repeated until the change in <em>G</em> falls below some arbitrary small threshold (i.e. convergence occurs). For <em>p</em> steps until convergence, the optimal rotation of each structure is equal to the product of the successive Γ<em><sub>i</sub></em> rotations.</p>
<a id="article1.body1.sec4.sec3.p12" name="article1.body1.sec4.sec3.p12"></a><p>This procedure superimposes the conserved residues found in the set <em>S<sub>n</sub></em>. The next goal is to extend this set. Multiple alignment is more accurate than a series of pair-wise alignments and this allows for <em>S<sub>n</sub></em> to be extended to include additional fully conserved residues. At the same time residues with less than full conservation can be incorporated. We have defined conserved residues as identical or similar amino acids from different structures within two angstroms of some point, termed a consensus point. Consensus points are considered redundant if they contain more than 50% of the same residues. The point closer to the relevant landmarks is retained. Partially conserved residues must only be present in a threshold of samples and the less than full conservation of an amino acid takes precedence over the conservation of a category. Leeway in the conservation of residues allows for evolutionary freedom, but is also required for practical reasons such as x-ray resolution, model construction, side-chain movement and non-significant sample-dependent positionings. As the superimposition procedure requires all landmarks in all samples, any samples with missing landmarks are temporally given the consensus point as a place filler. If new residues can be added following an alignment then the expanded set <em>S<sub>n</sub></em> is used to re-align the structures. The extension and alignment process is repeated until a final consensus set is converged upon or until the completion of a predetermined number of iterations.</p>


<h4>Aligning active-conformation kinases</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Active-conformation kinases were aligned using the above procedure with a congruency criteria of 1% and a distance constraint for conservation of two angstroms. The partially active-conformation structure of the transient receptor potential channel kinase was aligned directly onto the consensus using congruency criteria of 10% and a distance constraint of 2.25 angstroms.</p>


<h4>Availability and Performance</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The program is written in C and Fortran. Full source code is freely available under the GNU General Public License by contacting the authors. PDB formatted files are required for input. For multiple alignments output consists of PDB files modified according to the alignment, a consensus structure in PDB format, a csv file indicating the conserved and aligned residues, and a script for viewing the alignment and consensus in the open-source program PyMOL (<a href="http://www.pymol.org">http://www.pymol.org</a>). For a pair-wise alignment against a consensus structure, output is as above except no consensus structure is produced. The alignment and consensus structure of the fifteen kinases presented in the Results section were generated in ~twenty seconds using a single processing core of an AMD Athlon 64 X2 4200 processor with 1GB of RAM. A single pair-wise alignment takes ~2–8 seconds depending upon the degree of similarity between input samples. Comparisons between a sample and a consensus structure are very rapid (&lt;1 second due to the reduced set of residues in a consensus set).</p>


<h4>Structure prediction</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Constraints from fourteen active-conformation kinases were generated for both the side-chain and the α-carbon. Constraints were derived from the final structural alignment. Side-chain constraints were applied to the atom closest to the median of all side-chain atoms. Structure models were created following the Rosetta homology modeling protocol described in Das <em>et al</em>. <a href="#pone.0000982-Das1">[30]</a>. During full-atom refinement of a model, a penalty score is applied when the atom-atom distances in the model exceed the upper or lower limit of the corresponding distance constraints. If a distance exceeds the upper or lower constraint limit by <em>d</em> Angstrom, then the penalty score <em>Ec</em> is <em>d*d</em> when <em>d</em>&lt;0.5 and <em>d</em>−0.25 when <em>d</em>≥0.5. For PDK1, the lowest scoring model (in full atom energy) as output by Rosetta was selected as the best model. For Rio2, the lowest 5% of models were used for screening against the consensus structure as described in the Results section. Models were ranked based on their score from Equation.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000982"></div><a name="pone.0000982.s001" id="pone.0000982.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000982.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000982.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.23 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Miguel A. Andrade for critical reading and discussions during manuscript preparation.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JK RK. Performed the experiments: JK BQ. Analyzed the data: JK BQ. Wrote the paper: JK. Other: Developed and implemented the structure prediction method described: DB BQ JK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000982-Manning1" id="pone.0000982-Manning1"></a>Manning G, Whyte DB, Martinez R, Hunter T, Sudarsanam S (2002) The protein kinase complement of the human genome. Science  298: 1912–1934.  <ul class="find" data-citedArticleID="1000158" data-doi="10.1126/science.1075762"><li><a href="http://dx.doi.org/10.1126/science.1075762" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+kinase+complement+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+kinase+complement+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000982-Yamaguchi1" id="pone.0000982-Yamaguchi1"></a>Yamaguchi H, Matsushita M, Nairn AC, Kuriyan J (2001) Crystal structure of the atypical protein kinase domain of a TRP channel with phosphotransferase activity. Mol Cell  7: 1047–1057.  <ul class="find" data-citedArticleID="1000204" data-doi="10.1016/s1097-2765(01)00256-8"><li><a href="http://dx.doi.org/10.1016/s1097-2765(01)00256-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+atypical+protein+kinase+domain+of+a+TRP+channel+with+phosphotransferase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+atypical+protein+kinase+domain+of+a+TRP+channel+with+phosphotransferase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000982-LaRondeLeBlanc1" id="pone.0000982-LaRondeLeBlanc1"></a>LaRonde-LeBlanc N, Wlodawer A (2005) A family portrait of the RIO kinases. J Biol Chem  280: 37297–37300.  <ul class="find" data-citedArticleID="1000146" data-doi="10.1074/jbc.r500013200"><li><a href="http://dx.doi.org/10.1074/jbc.r500013200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+family+portrait+of+the+RIO+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+family+portrait+of+the+RIO+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000982-Dedhar1" id="pone.0000982-Dedhar1"></a>Dedhar S, Williams B, Hannigan G (1999) Integrin-linked kinase (ILK): a regulator of integrin and growth-factor signaling. Trends Cell Biol  9: 319–323.  <ul class="find" data-citedArticleID="1000114" data-doi="10.1016/s0962-8924(99)01612-8"><li><a href="http://dx.doi.org/10.1016/s0962-8924(99)01612-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrin-linked+kinase+%28ILK%29%3A+a+regulator+of+integrin+and+growth-factor+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrin-linked+kinase+%28ILK%29%3A+a+regulator+of+integrin+and+growth-factor+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000982-Ubersax1" id="pone.0000982-Ubersax1"></a>Ubersax JA, Woodbury EL, Quang PN, Paraz M, Blethrow JD, et al.  (2003) Targets of the cyclin-dependent kinase Cdk1. Nature  425: 859–864.  <ul class="find" data-citedArticleID="1000198" data-doi="10.1038/nature02062"><li><a href="http://dx.doi.org/10.1038/nature02062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targets+of+the+cyclin-dependent+kinase+Cdk1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targets+of+the+cyclin-dependent+kinase+Cdk1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000982-Ptacek1" id="pone.0000982-Ptacek1"></a>Ptacek J, Devgan G, Michaud G, Zhu H, Zhu X, et al.  (2005) Global analysis of protein phosphorylation in yeast. Nature  438: 679–684.  <ul class="find" data-citedArticleID="1000172" data-doi="10.1038/nature04187"><li><a href="http://dx.doi.org/10.1038/nature04187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+protein+phosphorylation+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+protein+phosphorylation+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000982-Slamon1" id="pone.0000982-Slamon1"></a>Slamon DJ, Clark GM, Wong SG, Levin WJ, Ullrich A, et al.  (1987) Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene. Science  235: 177–182.  <ul class="find" data-citedArticleID="1000188" data-doi="10.1126/science.3798106"><li><a href="http://dx.doi.org/10.1126/science.3798106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+breast+cancer%3A+correlation+of+relapse+and+survival+with+amplification+of+the+HER-2%2Fneu+oncogene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+breast+cancer%3A+correlation+of+relapse+and+survival+with+amplification+of+the+HER-2%2Fneu+oncogene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000982-Sawyers1" id="pone.0000982-Sawyers1"></a>Sawyers CL (1999) Chronic myeloid leukemia. N Engl J Med  340: 1330–1340.  <ul class="find" data-citedArticleID="1000182" data-doi="10.1056/nejm199904293401706"><li><a href="http://dx.doi.org/10.1056/nejm199904293401706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chronic+myeloid+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chronic+myeloid+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000982-Hommes1" id="pone.0000982-Hommes1"></a>Hommes D, van den Blink B, Plasse T, Bartelsman J, Xu C, et al.  (2002) Inhibition of stress-activated MAP kinases induces clinical improvement in moderate to severe Crohn's disease. Gastroenterology  122: 7–14.  <ul class="find" data-citedArticleID="1000132" data-doi="10.1053/gast.2002.30770"><li><a href="http://dx.doi.org/10.1053/gast.2002.30770" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+stress-activated+MAP+kinases+induces+clinical+improvement+in+moderate+to+severe+Crohn%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+stress-activated+MAP+kinases+induces+clinical+improvement+in+moderate+to+severe+Crohn%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000982-OnoSaito1" id="pone.0000982-OnoSaito1"></a>Ono-Saito N, Niki I, Hidaka H (1999) H-series protein kinase inhibitors and potential clinical applications. Pharmacol Ther  82: 123–131.  <ul class="find" data-citedArticleID="1000166" data-doi="10.1016/s0163-7258(98)00070-9"><li><a href="http://dx.doi.org/10.1016/s0163-7258(98)00070-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=H-series+protein+kinase+inhibitors+and+potential+clinical+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22H-series+protein+kinase+inhibitors+and+potential+clinical+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000982-SeboltLeopold1" id="pone.0000982-SeboltLeopold1"></a>Sebolt-Leopold JS, English JM (2006) Mechanisms of drug inhibition of signaling molecules. Nature  441: 457–462.  <ul class="find" data-citedArticleID="1000184" data-doi="10.1038/nature04874"><li><a href="http://dx.doi.org/10.1038/nature04874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+drug+inhibition+of+signaling+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+drug+inhibition+of+signaling+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000982-Fabian1" id="pone.0000982-Fabian1"></a>Fabian MA, r d Biggs WH, Treiber DK, Atteridge CE, Azimioara MD, et al.  (2005) A small molecule-kinase interaction map for clinical kinase inhibitors. Nat Biotechnol  23: 329–336.  <ul class="find" data-citedArticleID="1000120" data-doi="10.1038/nbt1068"><li><a href="http://dx.doi.org/10.1038/nbt1068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000982-Rohl1" id="pone.0000982-Rohl1"></a>Rohl CA, Strauss CE, Misura KM, Baker D (2004) Protein structure prediction using Rosetta. Methods Enzymol  383: 66–93.  <ul class="find" data-citedArticleID="1000178" data-doi="10.1016/s0076-6879(04)83004-0"><li><a href="http://dx.doi.org/10.1016/s0076-6879(04)83004-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+structure+prediction+using+Rosetta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+structure+prediction+using+Rosetta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000982-LaRondeLeBlanc2" id="pone.0000982-LaRondeLeBlanc2"></a>LaRonde-LeBlanc N, Wlodawer A (2004) Crystal structure of <em>A. fulgidus</em> Rio2 defines a new family of serine protein kinases. Structure  12: 1585–1594.  <ul class="find" data-citedArticleID="1000148" data-doi="10.1016/j.str.2004.06.016"><li><a href="http://dx.doi.org/10.1016/j.str.2004.06.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+A.+fulgidus+Rio2+defines+a+new+family+of+serine+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+A.+fulgidus+Rio2+defines+a+new+family+of+serine+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000982-Grishin1" id="pone.0000982-Grishin1"></a>Grishin NV (2001) Fold change in evolution of protein structures. J Struct Biol  134: 167–185.  <ul class="find" data-citedArticleID="1000126" data-doi="10.1006/jsbi.2001.4335"><li><a href="http://dx.doi.org/10.1006/jsbi.2001.4335" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fold+change+in+evolution+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fold+change+in+evolution+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000982-Todd1" id="pone.0000982-Todd1"></a>Todd AE, Orengo CA, Thornton JM (2002) Plasticity of enzyme active sites. Trends Biochem Sci  27: 419–426.  <ul class="find" data-citedArticleID="1000194" data-doi="10.1016/s0968-0004(02)02158-8"><li><a href="http://dx.doi.org/10.1016/s0968-0004(02)02158-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasticity+of+enzyme+active+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasticity+of+enzyme+active+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000982-Carrera1" id="pone.0000982-Carrera1"></a>Carrera AC, Alexandrov K, Roberts TM (1993) The conserved lysine of the catalytic domain of protein kinases is actively involved in the phosphotransfer reaction and not required for anchoring ATP. Proc Natl Acad Sci U S A  90: 442–446.  <ul class="find" data-citedArticleID="1000108" data-doi="10.1073/pnas.90.2.442"><li><a href="http://dx.doi.org/10.1073/pnas.90.2.442" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+conserved+lysine+of+the+catalytic+domain+of+protein+kinases+is+actively+involved+in+the+phosphotransfer+reaction+and+not+required+for+anchoring+ATP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+conserved+lysine+of+the+catalytic+domain+of+protein+kinases+is+actively+involved+in+the+phosphotransfer+reaction+and+not+required+for+anchoring+ATP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000982-Robinson1" id="pone.0000982-Robinson1"></a>Robinson MJ, Harkins PC, Zhang J, Baer R, Haycock JW, et al.  (1996) Mutation of position 52 in ERK2 creates a nonproductive binding mode for adenosine 5′-triphosphate. Biochemistry  35: 5641–5646.  <ul class="find" data-citedArticleID="1000176" data-doi="10.1021/bi952723e"><li><a href="http://dx.doi.org/10.1021/bi952723e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+of+position+52+in+ERK2+creates+a+nonproductive+binding+mode+for+adenosine+5%E2%80%B2-triphosphate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+of+position+52+in+ERK2+creates+a+nonproductive+binding+mode+for+adenosine+5%E2%80%B2-triphosphate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000982-Iyer1" id="pone.0000982-Iyer1"></a>Iyer GH, Garrod S, r Woods VL J, Taylor SS (2005) Catalytic independent functions of a protein kinase as revealed by a kinase-dead mutant: study of the Lys72His mutant of cAMP-dependent kinase. J Mol Biol  351: 1110–1122.  <ul class="find" data-citedArticleID="1000138" data-doi="10.1016/j.jmb.2005.06.011"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.06.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catalytic+independent+functions+of+a+protein+kinase+as+revealed+by+a+kinase-dead+mutant%3A+study+of+the+Lys72His+mutant+of+cAMP-dependent+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catalytic+independent+functions+of+a+protein+kinase+as+revealed+by+a+kinase-dead+mutant%3A+study+of+the+Lys72His+mutant+of+cAMP-dependent+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000982-Drennan1" id="pone.0000982-Drennan1"></a>Drennan D, Ryazanov AG (2004) Alpha-kinases: analysis of the family and comparison with conventional protein kinases. Prog Biophys Mol Biol  85: 1–32.  <ul class="find" data-citedArticleID="1000116" data-doi="10.1016/s0079-6107(03)00060-9"><li><a href="http://dx.doi.org/10.1016/s0079-6107(03)00060-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alpha-kinases%3A+analysis+of+the+family+and+comparison+with+conventional+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alpha-kinases%3A+analysis+of+the+family+and+comparison+with+conventional+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000982-Xu1" id="pone.0000982-Xu1"></a>Xu B, English JM, Wilsbacher JL, Stippec S, Goldsmith EJ, et al.  (2000) WNK1, a novel mammalian serine/threonine protein kinase lacking the catalytic lysine in subdomain II. J Biol Chem  275: 16795–16801.  <ul class="find" data-citedArticleID="1000200" data-doi="10.1074/jbc.275.22.16795"><li><a href="http://dx.doi.org/10.1074/jbc.275.22.16795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WNK1%2C+a+novel+mammalian+serine%2Fthreonine+protein+kinase+lacking+the+catalytic+lysine+in+subdomain+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WNK1%2C+a+novel+mammalian+serine%2Fthreonine+protein+kinase+lacking+the+catalytic+lysine+in+subdomain+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000982-Min1" id="pone.0000982-Min1"></a>Min X, Lee BH, Cobb MH, Goldsmith EJ (2004) Crystal structure of the kinase domain of WNK1, a kinase that causes a hereditary form of hypertension. Structure  12: 1303–1311.  <ul class="find" data-citedArticleID="1000160" data-doi="10.1016/j.str.2004.04.014"><li><a href="http://dx.doi.org/10.1016/j.str.2004.04.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+kinase+domain+of+WNK1%2C+a+kinase+that+causes+a+hereditary+form+of+hypertension." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+kinase+domain+of+WNK1%2C+a+kinase+that+causes+a+hereditary+form+of+hypertension.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000982-Zheng1" id="pone.0000982-Zheng1"></a>Zheng J, Trafny EA, Knighton DR, Xuong NH, Taylor SS, et al.  (1993) 2.2 Å refined crystal structure of the catalytic subunit of cAMP-dependent protein kinase complexed with MnATP and a peptide inhibitor. Acta Crystallogr D Biol Crystallogr  49: 362–365.  <ul class="find" data-citedArticleID="1000212" data-doi="10.1107/s0907444993000423"><li><a href="http://dx.doi.org/10.1107/s0907444993000423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=2.2+%C3%85+refined+crystal+structure+of+the+catalytic+subunit+of+cAMP-dependent+protein+kinase+complexed+with+MnATP+and+a+peptide+inhibitor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%222.2+%C3%85+refined+crystal+structure+of+the+catalytic+subunit+of+cAMP-dependent+protein+kinase+complexed+with+MnATP+and+a+peptide+inhibitor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000982-Lei1" id="pone.0000982-Lei1"></a>Lei M, Lu W, Meng W, Parrini MC, Eck MJ, et al.  (2000) Structure of PAK1 in an autoinhibited conformation reveals a multistage activation switch. Cell  102: 387–397.  <ul class="find" data-citedArticleID="1000152" data-doi="10.1016/s0092-8674(00)00043-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00043-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+PAK1+in+an+autoinhibited+conformation+reveals+a+multistage+activation+switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+PAK1+in+an+autoinhibited+conformation+reveals+a+multistage+activation+switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000982-Yang1" id="pone.0000982-Yang1"></a>Yang J, Cron P, Good VM, Thompson V, Hemmings BA, et al.  (2002) Crystal structure of an activated Akt/protein kinase B ternary complex with GSK3-peptide and AMP-PNP. Nat Struct Biol  9: 940–944.  <ul class="find" data-citedArticleID="1000206" data-doi="10.1038/nsb870"><li><a href="http://dx.doi.org/10.1038/nsb870" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+an+activated+Akt%2Fprotein+kinase+B+ternary+complex+with+GSK3-peptide+and+AMP-PNP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+an+activated+Akt%2Fprotein+kinase+B+ternary+complex+with+GSK3-peptide+and+AMP-PNP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000982-Huang1" id="pone.0000982-Huang1"></a>Huang J, Tu Z, Lee FS (2003) Mutations in protein kinase subdomain X differentially affect MEKK2 and MEKK1 activity. Biochem Biophys Res Commun  303: 532–540.  <ul class="find" data-citedArticleID="1000134" data-doi="10.1016/s0006-291x(03)00387-5"><li><a href="http://dx.doi.org/10.1016/s0006-291x(03)00387-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+protein+kinase+subdomain+X+differentially+affect+MEKK2+and+MEKK1+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+protein+kinase+subdomain+X+differentially+affect+MEKK2+and+MEKK1+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000982-Tu1" id="pone.0000982-Tu1"></a>Tu Z, Mooney SM, Lee FS (2003) A subdomain of MEKK1 that is critical for binding to MKK4. Cell Signal  15: 65–77.  <ul class="find" data-citedArticleID="1000196" data-doi="10.1016/s0898-6568(02)00056-6"><li><a href="http://dx.doi.org/10.1016/s0898-6568(02)00056-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+subdomain+of+MEKK1+that+is+critical+for+binding+to+MKK4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+subdomain+of+MEKK1+that+is+critical+for+binding+to+MKK4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000982-Felberg1" id="pone.0000982-Felberg1"></a>Felberg J, Lefebvre DC, Lam M, Wang Y, Ng DH, et al.  (2004) Subdomain X of the kinase domain of Lck binds CD45 and facilitates dephosphorylation. J Biol Chem  279: 3455–3462.  <ul class="find" data-citedArticleID="1000122" data-doi="10.1074/jbc.m309537200"><li><a href="http://dx.doi.org/10.1074/jbc.m309537200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subdomain+X+of+the+kinase+domain+of+Lck+binds+CD45+and+facilitates+dephosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subdomain+X+of+the+kinase+domain+of+Lck+binds+CD45+and+facilitates+dephosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000982-Kim1" id="pone.0000982-Kim1"></a>Kim C, Xuong NH, Taylor SS (2005) Crystal structure of a complex between the catalytic and regulatory (RIalpha) subunits of PKA. Science  307: 690–696.  <ul class="find" data-citedArticleID="1000142" data-doi="10.1126/science.1104607"><li><a href="http://dx.doi.org/10.1126/science.1104607" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+a+complex+between+the+catalytic+and+regulatory+%28RIalpha%29+subunits+of+PKA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+a+complex+between+the+catalytic+and+regulatory+%28RIalpha%29+subunits+of+PKA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000982-Das1" id="pone.0000982-Das1"></a>Das R, Qian B, Raman S, Vernon R, Thompson J, et al. Prediction of the structures of CASP7 targets using extensive all-atom refinement with Rosetta@home.  Proteins; In press.  <ul class="find" data-citedArticleID="1000112" data-doi="10.1002/prot.21636"><li><a href="http://dx.doi.org/10.1002/prot.21636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+the+structures+of+CASP7+targets+using+extensive+all-atom+refinement+with+Rosetta%40home." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+the+structures+of+CASP7+targets+using+extensive+all-atom+refinement+with+Rosetta%40home.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000982-Thompson1" id="pone.0000982-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1000192" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000982-Biondi1" id="pone.0000982-Biondi1"></a>Biondi RM, Komander D, Thomas CC, Lizcano JM, Deak M, et al.  (2002) High resolution crystal structure of the human PDK1 catalytic domain defines the regulatory phosphopeptide docking site. EMBO J  21: 4219–4228.  <ul class="find" data-citedArticleID="1000100" data-doi="10.1093/emboj/cdf437"><li><a href="http://dx.doi.org/10.1093/emboj/cdf437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+resolution+crystal+structure+of+the+human+PDK1+catalytic+domain+defines+the+regulatory+phosphopeptide+docking+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+resolution+crystal+structure+of+the+human+PDK1+catalytic+domain+defines+the+regulatory+phosphopeptide+docking+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000982-Hawkins1" id="pone.0000982-Hawkins1"></a>Hawkins J, Zheng S, Frantz B, LoGrasso P (2000) p38 map kinase substrate specificity differs greatly for protein and peptide substrates. Arch Biochem Biophys  382: 310–313.  <ul class="find" data-citedArticleID="1000130" data-doi="10.1006/abbi.2000.2005"><li><a href="http://dx.doi.org/10.1006/abbi.2000.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p38+map+kinase+substrate+specificity+differs+greatly+for+protein+and+peptide+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p38+map+kinase+substrate+specificity+differs+greatly+for+protein+and+peptide+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000982-Biondi2" id="pone.0000982-Biondi2"></a>Biondi RM, Nebreda AR (2003) Signaling specificity of Ser/Thr protein kinases through docking-site-mediated interactions. Biochem J  372: 1–13.  <ul class="find" data-citedArticleID="1000102" data-doi="10.1042/bj20021641"><li><a href="http://dx.doi.org/10.1042/bj20021641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+specificity+of+Ser%2FThr+protein+kinases+through+docking-site-mediated+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+specificity+of+Ser%2FThr+protein+kinases+through+docking-site-mediated+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000982-Liu1" id="pone.0000982-Liu1"></a>Liu Y, Bishop A, Witucki L, Kraybill B, Shimizu E, et al.  (1999) Structural basis for selective inhibition of Src family kinases by PP1. Chem Biol  6: 671–678.  <ul class="find" data-citedArticleID="1000154" data-doi="10.1016/s1074-5521(99)80118-5"><li><a href="http://dx.doi.org/10.1016/s1074-5521(99)80118-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+selective+inhibition+of+Src+family+kinases+by+PP1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+selective+inhibition+of+Src+family+kinases+by+PP1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000982-Blencke1" id="pone.0000982-Blencke1"></a>Blencke S, Zech B, Engkvist O, Greff Z, Orfi L, et al.  (2004) Characterization of a conserved structural determinant controlling protein kinase sensitivity to selective inhibitors. Chem Biol  11: 691–701.  <ul class="find" data-citedArticleID="1000104" data-doi="10.1016/j.chembiol.2004.02.029"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2004.02.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+conserved+structural+determinant+controlling+protein+kinase+sensitivity+to+selective+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+conserved+structural+determinant+controlling+protein+kinase+sensitivity+to+selective+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000982-Bogoyevitch1" id="pone.0000982-Bogoyevitch1"></a>Bogoyevitch MA, Barr RK, Ketterman AJ (2005) Peptide inhibitors of protein kinases-discovery, characterization and use. Biochim Biophys Acta  1754: 79–99.  <ul class="find" data-citedArticleID="1000106" data-doi="10.1016/j.bbapap.2005.07.025"><li><a href="http://dx.doi.org/10.1016/j.bbapap.2005.07.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peptide+inhibitors+of+protein+kinases-discovery%2C+characterization+and+use." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peptide+inhibitors+of+protein+kinases-discovery%2C+characterization+and+use.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000982-Perea1" id="pone.0000982-Perea1"></a>Perea SE, Reyes O, Puchades Y, Mendoza O, Vispo NS, et al.  (2004) Antitumor effect of a novel proapoptotic peptide that impairs the phosphorylation by the protein kinase 2 (casein kinase 2). Cancer Res  64: 7127–7129.  <ul class="find" data-citedArticleID="1000170" data-doi="10.1158/0008-5472.can-04-2086"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-04-2086" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antitumor+effect+of+a+novel+proapoptotic+peptide+that+impairs+the+phosphorylation+by+the+protein+kinase+2+%28casein+kinase+2%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antitumor+effect+of+a+novel+proapoptotic+peptide+that+impairs+the+phosphorylation+by+the+protein+kinase+2+%28casein+kinase+2%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000982-Shah1" id="pone.0000982-Shah1"></a>Shah K, Shokat KM (2003) A chemical genetic approach for the identification of direct substrates of protein kinases. Methods Mol Biol  233: 253–271.  <ul class="find" data-citedArticleID="1000186" data-doi="10.1385/1-59259-397-6:253"><li><a href="http://dx.doi.org/10.1385/1-59259-397-6:253" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+chemical+genetic+approach+for+the+identification+of+direct+substrates+of+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+chemical+genetic+approach+for+the+identification+of+direct+substrates+of+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000982-Goodall1" id="pone.0000982-Goodall1"></a>Goodall C (1991) Procrustes methods in the statistical analysis of shape. J R Statist Soc B  53: 285–339.  <ul class="find" data-citedArticleID="1000124"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Procrustes+methods+in+the+statistical+analysis+of+shape.&amp;auth=&amp;atitle=Procrustes+methods+in+the+statistical+analysis+of+shape." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Procrustes+methods+in+the+statistical+analysis+of+shape." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Procrustes+methods+in+the+statistical+analysis+of+shape.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000982-Dryden1" id="pone.0000982-Dryden1"></a>Dryden IL, Mardia KV (1998) Statistical shape analysis. Chichester: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0000982-Claude1" id="pone.0000982-Claude1"></a>Claude J, Pritchard P, Tong H, Paradis E, Auffray JC (2004) Ecological correlates and evolutionary divergence in the skull of turtles: a geometric morphometric assessment. Syst Biol  53: 933–948.  <ul class="find" data-citedArticleID="1000110"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Ecological+correlates+and+evolutionary+divergence+in+the+skull+of+turtles%3A+a+geometric+morphometric+assessment.&amp;auth=&amp;atitle=Ecological+correlates+and+evolutionary+divergence+in+the+skull+of+turtles%3A+a+geometric+morphometric+assessment." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+correlates+and+evolutionary+divergence+in+the+skull+of+turtles%3A+a+geometric+morphometric+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+correlates+and+evolutionary+divergence+in+the+skull+of+turtles%3A+a+geometric+morphometric+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000982-Kent1" id="pone.0000982-Kent1"></a>Kent JT (1992)  New directions in shape analysis. In: Mardia KV, editor. The art of statistical science. Chichester: Wiley. pp. 115–127.  <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0000982-Gutteridge1" id="pone.0000982-Gutteridge1"></a>Gutteridge A, Thornton JM (2005) Understanding nature's catalytic toolkit. Trends Biochem Sci  30: 622–629.  <ul class="find" data-citedArticleID="1000128" data-doi="10.1016/j.tibs.2005.09.006"><li><a href="http://dx.doi.org/10.1016/j.tibs.2005.09.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+nature%27s+catalytic+toolkit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+nature%27s+catalytic+toolkit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000982-Knighton1" id="pone.0000982-Knighton1"></a>Knighton DR, Zheng JH, Eyck LFT, Ashford VA, Xuong NH, et al.  (1991) Crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase. Science  253: 407–414.  <ul class="find" data-citedArticleID="1000144" data-doi="10.1126/science.1862342"><li><a href="http://dx.doi.org/10.1126/science.1862342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+catalytic+subunit+of+cyclic+adenosine+monophosphate-dependent+protein+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+catalytic+subunit+of+cyclic+adenosine+monophosphate-dependent+protein+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000982-Lougheed1" id="pone.0000982-Lougheed1"></a>Lougheed JC, Chen RH, Mak P, Stout TJ (2004) Crystal structures of the phosphorylated and unphosphorylated kinase domains of the Cdc42-associated tyrosine kinase ACK1. J Biol Chem  279: 44039–44045.  <ul class="find" data-citedArticleID="1000156" data-doi="10.1074/jbc.m406703200"><li><a href="http://dx.doi.org/10.1074/jbc.m406703200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+the+phosphorylated+and+unphosphorylated+kinase+domains+of+the+Cdc42-associated+tyrosine+kinase+ACK1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+the+phosphorylated+and+unphosphorylated+kinase+domains+of+the+Cdc42-associated+tyrosine+kinase+ACK1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000982-Russo1" id="pone.0000982-Russo1"></a>Russo AA, Jeffrey PD, Pavletich NP (1996) Structural basis of cyclin-dependent kinase activation by phosphorylation. Nat Struct Biol  3: 696–700.  <ul class="find" data-citedArticleID="1000180" data-doi="10.1038/nsb0896-696"><li><a href="http://dx.doi.org/10.1038/nsb0896-696" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+cyclin-dependent+kinase+activation+by+phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+cyclin-dependent+kinase+activation+by+phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000982-Xu2" id="pone.0000982-Xu2"></a>Xu RM, Carmel G, Sweet RM, Kuret J, Cheng X (1995) Crystal structure of casein kinase-1, a phosphate-directed protein kinase. EMBO J  14: 1015–1023.  <ul class="find" data-citedArticleID="1000202"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Crystal+structure+of+casein+kinase-1%2C+a+phosphate-directed+protein+kinase.&amp;auth=&amp;atitle=Crystal+structure+of+casein+kinase-1%2C+a+phosphate-directed+protein+kinase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+casein+kinase-1%2C+a+phosphate-directed+protein+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+casein+kinase-1%2C+a+phosphate-directed+protein+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000982-Yde1" id="pone.0000982-Yde1"></a>Yde CW, Ermakova I, Issinger OG, Niefind K (2005) Inclining the purine base binding plane in protein kinase CK2 by exchanging the flanking side-chains generates a preference for ATP as a cosubstrate. J Mol Biol  347: 399–414.  <ul class="find" data-citedArticleID="1000208" data-doi="10.1016/j.jmb.2005.01.003"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.01.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inclining+the+purine+base+binding+plane+in+protein+kinase+CK2+by+exchanging+the+flanking+side-chains+generates+a+preference+for+ATP+as+a+cosubstrate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inclining+the+purine+base+binding+plane+in+protein+kinase+CK2+by+exchanging+the+flanking+side-chains+generates+a+preference+for+ATP+as+a+cosubstrate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000982-Tereshko1" id="pone.0000982-Tereshko1"></a>Tereshko V, Teplova M, Brunzelle J, Watterson DM, Egli M (2001) Crystal structures of the catalytic domain of human protein kinase associated with apoptosis and tumor suppression. Nat Struct Biol  8: 899–907.  <ul class="find" data-citedArticleID="1000190" data-doi="10.1038/nsb1001-899"><li><a href="http://dx.doi.org/10.1038/nsb1001-899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+the+catalytic+domain+of+human+protein+kinase+associated+with+apoptosis+and+tumor+suppression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+the+catalytic+domain+of+human+protein+kinase+associated+with+apoptosis+and+tumor+suppression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000982-Hubbard1" id="pone.0000982-Hubbard1"></a>Hubbard SR (1997) Crystal structure of the activated insulin receptor tyrosine kinase in complex with peptide substrate and ATP analog. EMBO J  16: 5572–5581.  <ul class="find" data-citedArticleID="1000136" data-doi="10.1093/emboj/16.18.5572"><li><a href="http://dx.doi.org/10.1093/emboj/16.18.5572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+activated+insulin+receptor+tyrosine+kinase+in+complex+with+peptide+substrate+and+ATP+analog." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+activated+insulin+receptor+tyrosine+kinase+in+complex+with+peptide+substrate+and+ATP+analog.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000982-Bellon1" id="pone.0000982-Bellon1"></a>Bellon S, Fitzgibbon MJ, Fox T, Hsiao HM, Wilson KP (1999) The structure of phosphorylated p38gamma is monomeric and reveals a conserved activation-loop conformation. Structure  7: 1057–1065.  <ul class="find" data-citedArticleID="1000098" data-doi="10.1016/s0969-2126(99)80173-7"><li><a href="http://dx.doi.org/10.1016/s0969-2126(99)80173-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+phosphorylated+p38gamma+is+monomeric+and+reveals+a+conserved+activation-loop+conformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+phosphorylated+p38gamma+is+monomeric+and+reveals+a+conserved+activation-loop+conformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000982-Owen1" id="pone.0000982-Owen1"></a>Owen DJ, Noble ME, Garman EF, Papageorgiou AC, Johnson LN (1995) Two structures of the catalytic domain of phosphorylase kinase: an active protein kinase complexed with substrate analogue and product. Structure  3: 467–482.  <ul class="find" data-citedArticleID="1000168" data-doi="10.1016/s0969-2126(01)00180-0"><li><a href="http://dx.doi.org/10.1016/s0969-2126(01)00180-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+structures+of+the+catalytic+domain+of+phosphorylase+kinase%3A+an+active+protein+kinase+complexed+with+substrate+analogue+and+product." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+structures+of+the+catalytic+domain+of+phosphorylase+kinase%3A+an+active+protein+kinase+complexed+with+substrate+analogue+and+product.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000982-Qian1" id="pone.0000982-Qian1"></a>Qian KC, Wang L, Hickey ER, Studts J, Barringer K, et al.  (2005) Structural basis of constitutive activity and a unique nucleotide binding mode of human Pim-1 kinase. J Biol Chem  280: 6130–6137.  <ul class="find" data-citedArticleID="1000174" data-doi="10.1074/jbc.m409123200"><li><a href="http://dx.doi.org/10.1074/jbc.m409123200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+constitutive+activity+and+a+unique+nucleotide+binding+mode+of+human+Pim-1+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+constitutive+activity+and+a+unique+nucleotide+binding+mode+of+human+Pim-1+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000982-Young1" id="pone.0000982-Young1"></a>Young TA, Delagoutte B, Endrizzi JA, Falick AM, Alber T (2003) Structure of Mycobacterium tuberculosis PknB supports a universal activation mechanism for Ser/Thr protein kinases. Nat Struct Biol  10: 168–174.  <ul class="find" data-citedArticleID="1000210" data-doi="10.1038/nsb897"><li><a href="http://dx.doi.org/10.1038/nsb897" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+Mycobacterium+tuberculosis+PknB+supports+a+universal+activation+mechanism+for+Ser%2FThr+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+Mycobacterium+tuberculosis+PknB+supports+a+universal+activation+mechanism+for+Ser%2FThr+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000982-LaRondeLeBlanc3" id="pone.0000982-LaRondeLeBlanc3"></a>LaRonde-LeBlanc N, Guszczynski T, Copeland T, Wlodawer A (2005) Autophosphorylation of <em>Archaeoglobus fulgidus</em> Rio2 and crystal structures of its nucleotide-metal ion complexes. FEBS J  272: 2800–2810.  <ul class="find" data-citedArticleID="1000150" data-doi="10.1111/j.1742-4658.2005.04702.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2005.04702.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autophosphorylation+of+Archaeoglobus+fulgidus+Rio2+and+crystal+structures+of+its+nucleotide-metal+ion+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autophosphorylation+of+Archaeoglobus+fulgidus+Rio2+and+crystal+structures+of+its+nucleotide-metal+ion+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000982-Nolen1" id="pone.0000982-Nolen1"></a>Nolen B, Ngo J, Chakrabarti S, Vu D, Adams JA, et al.  (2003) Nucleotide-induced conformational changes in the <em>Saccharomyces cerevisiae</em> SR protein kinase, Sky1p, revealed by X-ray crystallography. Biochemistry  42: 9575–9585.  <ul class="find" data-citedArticleID="1000162" data-doi="10.1021/bi0344331"><li><a href="http://dx.doi.org/10.1021/bi0344331" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide-induced+conformational+changes+in+the+Saccharomyces+cerevisiae+SR+protein+kinase%2C+Sky1p%2C+revealed+by+X-ray+crystallography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide-induced+conformational+changes+in+the+Saccharomyces+cerevisiae+SR+protein+kinase%2C+Sky1p%2C+revealed+by+X-ray+crystallography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000982-Zhou1" id="pone.0000982-Zhou1"></a>Zhou T, Raman M, Gao Y, Earnest S, Chen Z, et al.  (2004) Crystal structure of the TAO2 kinase domain: activation and specificity of a Ste20p MAP3K. Structure  12: 1891–1900.  <ul class="find" data-citedArticleID="1000214" data-doi="10.1016/j.str.2004.07.021"><li><a href="http://dx.doi.org/10.1016/j.str.2004.07.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+TAO2+kinase+domain%3A+activation+and+specificity+of+a+Ste20p+MAP3K." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+TAO2+kinase+domain%3A+activation+and+specificity+of+a+Ste20p+MAP3K.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000982-Nolen2" id="pone.0000982-Nolen2"></a>Nolen B, Yun CY, Wong CF, McCammon JA, Fu XD, et al.  (2001) The structure of Sky1p reveals a novel mechanism for constitutive activity. Nat Struct Biol  8: 176–183.  <ul class="find" data-citedArticleID="1000164" data-doi="10.1038/84178"><li><a href="http://dx.doi.org/10.1038/84178" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+Sky1p+reveals+a+novel+mechanism+for+constitutive+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+Sky1p+reveals+a+novel+mechanism+for+constitutive+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.XML" value="170610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.PDF" value="2770714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g001.PNG_L" value="664342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g001.PNG_M" value="229464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g001.PNG_S" value="20255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g001.TIF" value="904886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g001.PNG_I" value="132359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t001.PNG_L" value="249541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t001.PNG_M" value="139858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t001.PNG_S" value="13312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t001.TIF" value="987862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t001.PNG_I" value="42486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g002.PNG_L" value="1121063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g002.PNG_M" value="410423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g002.PNG_S" value="20339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g002.TIF" value="1503148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g002.PNG_I" value="174922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g003.PNG_L" value="1133628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g003.PNG_M" value="168389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g003.PNG_S" value="8832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g003.TIF" value="1664816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g003.PNG_I" value="54447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t002.PNG_L" value="535005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t002.PNG_M" value="236017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t002.PNG_S" value="25019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t002.TIF" value="2287092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.t002.PNG_I" value="114115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g004.PNG_L" value="993157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g004.PNG_M" value="242074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g004.PNG_S" value="15462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g004.TIF" value="1355692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g004.PNG_I" value="80830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g005.PNG_L" value="1267153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g005.PNG_M" value="308348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g005.PNG_S" value="21530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g005.TIF" value="1834830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g005.PNG_I" value="110196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g006.PNG_L" value="893549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g006.PNG_M" value="226490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g006.PNG_S" value="15806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g006.TIF" value="1342942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g006.PNG_I" value="81689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g007.PNG_L" value="2019710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g007.PNG_M" value="356412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g007.PNG_S" value="22294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g007.TIF" value="2885576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g007.PNG_I" value="188616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g008.PNG_L" value="1024411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g008.PNG_M" value="238833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g008.PNG_S" value="14419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g008.TIF" value="1350474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g008.PNG_I" value="77812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g009.PNG_L" value="981486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g009.PNG_M" value="146053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g009.PNG_S" value="27708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g009.TIF" value="1274862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.g009.PNG_I" value="268677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e001.PNG" value="6949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e001.TIF" value="22900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e002.PNG" value="5101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e002.TIF" value="21048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e003.PNG" value="5131"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e003.TIF" value="21392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e004.PNG" value="5159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e004.TIF" value="21108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e005.PNG" value="6809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e005.TIF" value="22940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e006.PNG" value="5951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e006.TIF" value="21872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e007.PNG" value="9373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e007.TIF" value="25232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e008.PNG" value="10699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e008.TIF" value="26692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e009.PNG" value="5100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e009.TIF" value="21052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e010.PNG" value="5915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e010.TIF" value="21852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e011.PNG" value="8243"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e011.TIF" value="24108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e012.PNG" value="5137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e012.TIF" value="21396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e013.PNG" value="5212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e013.TIF" value="21136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e014.PNG" value="5197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.e014.TIF" value="21132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000982.s001.DOC" value="228352"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000982&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000982" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000982&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000982&volume=&issue=&title=Conservation, Variability and the Modeling of Active Protein Kinases&author_name=James%20D.%20R.%20Knight%2C%20Bin%20Qian%2C%20David%20Baker%2C%20Rashmi%20Kothary&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000982" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000982" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000982" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000982&amp;t=Conservation%2C%20Variability%20and%20the%20Modeling%20of%20Active%20Protein%20Kinases" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000982&title=Conservation%2C%20Variability%20and%20the%20Modeling%20of%20Active%20Protein%20Kinases&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000982&amp;title=Conservation%2C%20Variability%20and%20the%20Modeling%20of%20Active%20Protein%20Kinases" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000982" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000982&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Conservation, Variability and the Modeling of Active Protein Kinases';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000982';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Conservation%2C%20Variability%20and%20the%20Modeling%20of%20Active%20Protein%20Kinases http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000982" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000982" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="24798"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Kinase+inhibitors%22" title="Search for articles in the subject area:'Kinase inhibitors'"><div class="flagText">Kinase inhibitors</div></a>
              <div data-categoryid="21149" data-articleid="24798"
                   data-categoryname="Kinase inhibitors"
                   class="flagImage" title="Flag 'Kinase inhibitors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lysine%22" title="Search for articles in the subject area:'Lysine'"><div class="flagText">Lysine</div></a>
              <div data-categoryid="40333" data-articleid="24798"
                   data-categoryname="Lysine"
                   class="flagImage" title="Flag 'Lysine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+alignment+calculation%22" title="Search for articles in the subject area:'Multiple alignment calculation'"><div class="flagText">Multiple alignment calculation</div></a>
              <div data-categoryid="20427" data-articleid="24798"
                   data-categoryname="Multiple alignment calculation"
                   class="flagImage" title="Flag 'Multiple alignment calculation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="24798"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+prediction%22" title="Search for articles in the subject area:'Protein structure prediction'"><div class="flagText">Protein structure prediction</div></a>
              <div data-categoryid="33203" data-articleid="24798"
                   data-categoryname="Protein structure prediction"
                   class="flagImage" title="Flag 'Protein structure prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="24798"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Serine%22" title="Search for articles in the subject area:'Serine'"><div class="flagText">Serine</div></a>
              <div data-categoryid="40197" data-articleid="24798"
                   data-categoryname="Serine"
                   class="flagImage" title="Flag 'Serine' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2063'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6391'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5232&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
