

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Deinococcus geothermalis: The Pool of Extreme Radiation Resistance Genes Shrinks</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000955"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000955" />

    <meta name="citation_title" content="Deinococcus geothermalis: The Pool of Extreme Radiation Resistance Genes Shrinks"/>
    <meta itemprop="name" content="Deinococcus geothermalis: The Pool of Extreme Radiation Resistance Genes Shrinks"/>

      <meta name="citation_author" content="Kira S. Makarova"/>
            <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Marina V. Omelchenko"/>
            <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Elena K. Gaidamakova"/>
            <meta name="citation_author_institution" content="Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Vera Y. Matrosova"/>
            <meta name="citation_author_institution" content="Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Alexander Vasilenko"/>
            <meta name="citation_author_institution" content="Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Min Zhai"/>
            <meta name="citation_author_institution" content="Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Alla Lapidus"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="Alex Copeland"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="Edwin Kim"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="Miriam Land"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="Konstantinos Mavromatis"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="Samuel Pitluck"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="Paul M. Richardson"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America"/>
      <meta name="citation_author" content="Chris Detter"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
      <meta name="citation_author" content="Thomas Brettin"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
      <meta name="citation_author" content="Elizabeth Saunders"/>
            <meta name="citation_author_institution" content="US Department of Energy, Joint Genome Institute, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
      <meta name="citation_author" content="Barry Lai"/>
            <meta name="citation_author_institution" content="Environmental Research Division and Advanced Photon Source, Argonne National Laboratory, Argonne, Illinois, United States of America"/>
      <meta name="citation_author" content="Bruce Ravel"/>
            <meta name="citation_author_institution" content="Environmental Research Division and Advanced Photon Source, Argonne National Laboratory, Argonne, Illinois, United States of America"/>
      <meta name="citation_author" content="Kenneth M. Kemner"/>
            <meta name="citation_author_institution" content="Environmental Research Division and Advanced Photon Source, Argonne National Laboratory, Argonne, Illinois, United States of America"/>
      <meta name="citation_author" content="Yuri I. Wolf"/>
            <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Alexander Sorokin"/>
            <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Anna V. Gerasimova"/>
            <meta name="citation_author_institution" content="Research Institute of Genetics and Selection of Industrial Microorganisms, Moscow, Russia"/>
      <meta name="citation_author" content="Mikhail S. Gelfand"/>
            <meta name="citation_author_institution" content="Institute for Information Transmission Problems of RAS, Moscow, Russia"/>
            <meta name="citation_author_institution" content="Faculty of Bioengineering and Bioinformatics, M. V. Lomonosov Moscow State University, Moscow, Russia"/>
      <meta name="citation_author" content="James K. Fredrickson"/>
            <meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Eugene V. Koonin"/>
            <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Michael J. Daly"/>
            <meta name="citation_author_institution" content="Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America"/>

    <meta name="citation_date" content="2007/9/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000955.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e955"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Protein phylogenies and signature sequences: A reappraisal of evolutionary relationships among archaebacteria, eubacteria, and eukaryotes.; citation_author=RS Gupta; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=62; citation_number=1; citation_pages=1435-1491; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genome trees constructed using five different approaches suggest new major bacterial clades.; citation_author=YI Wolf; citation_author=IB Rogozin; citation_author=NV Grishin; citation_author=RL Tatusov; citation_author=EV Koonin; citation_journal_title=BMC Evol Biol; citation_volume=1; citation_number=2; citation_pages=8; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Deinococcus ficus sp. nov., isolated from the rhizosphere of Ficus religiosa L.; citation_author=WA Lai; citation_author=P Kampfer; citation_author=AB Arun; citation_author=FT Shen; citation_author=B Huber; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=56; citation_number=3; citation_pages=787-791; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Restoration of the DNA damage resistance of Deinococcus radiodurans DNA polymerase mutants by Escherichia coli DNA polymerase I and Klenow fragment.; citation_author=PD Gutman; citation_author=P Fuchs; citation_author=KW Minton; citation_journal_title=Mutat Res; citation_volume=314; citation_number=4; citation_pages=87-97; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Accumulation of Mn(II) in Deinococcus radiodurans facilitates gamma-radiation resistance.; citation_author=MJ Daly; citation_author=EK Gaidamakova; citation_author=VY Matrosova; citation_author=A Vasilenko; citation_author=M Zhai; citation_journal_title=Science; citation_volume=306; citation_number=5; citation_pages=1025-1028; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Engineering radiation-resistant bacteria for environmental biotechnology.; citation_author=MJ Daly; citation_journal_title=Curr Opin Biotechnol; citation_volume=11; citation_number=6; citation_pages=280-285; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Deinococcus geothermalis sp. nov. and Deinococcus murrayi sp. nov., two extremely radiation-resistant and slightly thermophilic species from hot springs.; citation_author=AC Ferreira; citation_author=MF Nobre; citation_author=FA Rainey; citation_author=MT Silva; citation_author=R Wait; citation_journal_title=Int J Syst Bacteriol; citation_volume=47; citation_number=7; citation_pages=939-947; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Characterization of adhesion threads of Deinococcus geothermalis as type IV pili.; citation_author=C Saarimaa; citation_author=M Peltola; citation_author=M Raulio; citation_author=TR Neu; citation_author=MS Salkinoja-Salonen; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=8; citation_pages=7016-7021; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Distribution of microorganisms in the subsurface of the manus basin hydrothermal vent field in Papua New Guinea.; citation_author=H Kimura; citation_author=R Asada; citation_author=A Masta; citation_author=T Naganuma; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=9; citation_pages=644-648; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic diversity analysis of subterranean hot springs in Iceland.; citation_author=VT Marteinsson; citation_author=S Hauksdottir; citation_author=CF Hobel; citation_author=H Kristmannsdottir; citation_author=GO Hreggvidsson; citation_journal_title=Appl Environ Microbiol; citation_volume=67; citation_number=10; citation_pages=4242-4248; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Engineering Deinococcus geothermalis for bioremediation of high-temperature radioactive waste environments.; citation_author=H Brim; citation_author=A Venkateswaran; citation_author=HM Kostandarithes; citation_author=JK Fredrickson; citation_author=MJ Daly; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=11; citation_pages=4575-4582; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=How radiation kills cells: survival of Deinococcus radiodurans and Shewanella oneidensis under oxidative stress.; citation_author=D Ghosal; citation_author=MV Omelchenko; citation_author=EK Gaidamakova; citation_author=VY Matrosova; citation_author=A Vasilenko; citation_journal_title=FEMS Microbiol Rev; citation_volume=29; citation_number=12; citation_pages=361-375; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reduction of Fe(III), Cr(VI), U(VI), and Tc(VII) by Deinococcus radiodurans R1.; citation_author=JK Fredrickson; citation_author=HM Kostandarithes; citation_author=SW Li; citation_author=AE Plymale; citation_author=MJ Daly; citation_journal_title=Appl Environ Microbiol; citation_volume=66; citation_number=13; citation_pages=2006-2011; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Engineering Deinococcus radiodurans for metal remediation in radioactive mixed waste environments.; citation_author=H Brim; citation_author=SC McFarlan; citation_author=JK Fredrickson; citation_author=KW Minton; citation_author=M Zhai; citation_journal_title=Nat Biotechnol; citation_volume=18; citation_number=14; citation_pages=85-90; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Deinococcus radiodurans engineered for complete toluene degradation facilitates Cr(VI) reduction.; citation_author=H Brim; citation_author=JP Osborne; citation_author=HM Kostandarithes; citation_author=JK Fredrickson; citation_author=LP Wackett; citation_journal_title=Microbiology; citation_volume=152; citation_number=15; citation_pages=2469-2477; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome Sequence of the Radioresistant Bacterium Deinococcus radiodurans R1.; citation_author=O White; citation_author=JA Eisen; citation_author=JF Heidelberg; citation_author=EK Hickey; citation_author=JD Peterson; citation_journal_title=Science; citation_volume=286; citation_number=16; citation_pages=1571-1577; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Protein Oxidation Implicated as the Primary Determinant of Bacterial Radioresistance.; citation_author=MJ Daly; citation_author=EK Gaidamakova; citation_author=VY Matrosova; citation_author=A Vasilenko; citation_author=M Zhai; citation_journal_title=PLoS Biol; citation_volume=5; citation_number=17; citation_pages=769-779; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=In vivo damage and recA-dependent repair of plasmid and chromosomal DNA in the radiation-resistant bacterium Deinococcus radiodurans.; citation_author=MJ Daly; citation_author=L Ouyang; citation_author=P Fuchs; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=18; citation_pages=3508-3517; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=DNA protection mechanisms are not involved in the radioresistance of the hyperthermophilic archaea Pyrococcus abyssi and P. furiosus.; citation_author=E Gerard; citation_author=E Jolivet; citation_author=D Prieur; citation_author=P Forterre; citation_journal_title=Mol Genet Genomics; citation_volume=266; citation_number=19; citation_pages=72-78; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Deinococcus radiodurans-the consummate survivor.; citation_author=MM Cox; citation_author=JR Battista; citation_journal_title=Nat Rev Microbiol; citation_volume=3; citation_number=20; citation_pages=882-892; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome of the extremely radiation-resistant bacterium Deinococcus radiodurans viewed from the perspective of comparative genomics.; citation_author=KS Makarova; citation_author=L Aravind; citation_author=YI Wolf; citation_author=RL Tatusov; citation_author=KW Minton; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=65; citation_number=21; citation_pages=44-79; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Reassembly of shattered chromosomes in Deinococcus radiodurans.; citation_author=K Zahradka; citation_author=D Slade; citation_author=A Bailone; citation_author=S Sommer; citation_author=D Averbeck; citation_journal_title=Nature; citation_volume=443; citation_number=22; citation_pages=569-573; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modulating radiation resistance: Insights based on defenses against reactive oxygen species in the radioresistant bacterium Deinococcus radiodurans.; citation_author=MJ Daly; citation_journal_title=Clin Lab Med; citation_volume=26; citation_number=23; citation_pages=491-504, x; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A ring-like nucleoid is not necessary for radioresistance in the Deinococcaceae.; citation_author=JM Zimmerman; citation_author=JR Battista; citation_journal_title=BMC Microbiol; citation_volume=5; citation_number=24; citation_pages=17; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Was Earth ever infected by martian biota? Clues from radioresistant bacteria.; citation_author=AK Pavlov; citation_author=VL Kalinin; citation_author=AN Konstantinov; citation_author=VN Shelegedin; citation_author=AA Pavlov; citation_journal_title=Astrobiology; citation_volume=6; citation_number=25; citation_pages=911-918; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Whole-genome shotgun optical mapping of Deinococcus radiodurans.; citation_author=J Lin; citation_author=R Qi; citation_author=C Aston; citation_author=J Jing; citation_author=TS Anantharaman; citation_journal_title=Science; citation_volume=285; citation_number=26; citation_pages=1558-1562; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics of Thermus thermophilus and Deinococcus radiodurans: divergent routes of adaptation to thermophily and radiation resistance.; citation_author=MV Omelchenko; citation_author=YI Wolf; citation_author=EK Gaidamakova; citation_author=VY Matrosova; citation_author=A Vasilenko; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=27; citation_pages=57; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for symmetric chromosomal inversions around the replication origin in bacteria.; citation_author=JA Eisen; citation_author=JF Heidelberg; citation_author=O White; citation_author=SL Salzberg; citation_journal_title=Genome Biol; citation_volume=1; citation_number=28; citation_pages=RESEARCH0011; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Short repeats and IS elements in the extremely radiation-resistant bacterium Deinococcus radiodurans and comparison to other bacterial species.; citation_author=KS Makarova; citation_author=YI Wolf; citation_author=O White; citation_author=K Minton; citation_author=MJ Daly; citation_journal_title=Res Microbiol; citation_volume=150; citation_number=29; citation_pages=711-724; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics of the Archaea (Euryarchaeota): evolution of conserved protein families, the stable core, and the variable shell.; citation_author=KS Makarova; citation_author=L Aravind; citation_author=MY Galperin; citation_author=NV Grishin; citation_author=RL Tatusov; citation_journal_title=Genome Res; citation_volume=9; citation_number=30; citation_pages=608-628; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis.; citation_author=D Sen; citation_author=W Gilbert; citation_journal_title=Nature; citation_volume=334; citation_number=31; citation_pages=364-366; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Biological significance of a family of regularly spaced repeats in the genomes of Archaea, Bacteria and mitochondria.; citation_author=FJ Mojica; citation_author=C Diez-Villasenor; citation_author=E Soria; citation_author=G Juez; citation_journal_title=Mol Microbiol; citation_volume=36; citation_number=32; citation_pages=244-246; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=CRISPR provides acquired resistance against viruses in prokaryotes.; citation_author=R Barrangou; citation_author=C Fremaux; citation_author=H Deveau; citation_author=M Richards; citation_author=P Boyaval; citation_journal_title=Science; citation_volume=315; citation_number=33; citation_pages=1709-1712; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A putative RNA-interference-based immune system in prokaryotes: computational analysis of the predicted enzymatic machinery, functional analogies with eukaryotic RNAi, and hypothetical mechanisms of action.; citation_author=KS Makarova; citation_author=NV Grishin; citation_author=SA Shabalina; citation_author=YI Wolf; citation_author=EV Koonin; citation_journal_title=Biol Direct; citation_volume=1; citation_number=34; citation_pages=7; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes.; citation_author=RL Tatusov; citation_author=ND Fedorova; citation_author=JD Jackson; citation_author=AR Jacobs; citation_author=B Kiryutin; citation_journal_title=BMC Bioinformatics; citation_volume=4; citation_number=35; citation_pages=41; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A genomic perspective on protein families.; citation_author=RL Tatusov; citation_author=EV Koonin; citation_author=DJ Lipman; citation_journal_title=Science; citation_volume=278; citation_number=36; citation_pages=631-637; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Feast/famine regulatory proteins (FFRPs): Escherichia coli Lrp, AsnC and related archaeal transcription factors.; citation_author=K Yokoyama; citation_author=SA Ishijima; citation_author=L Clowney; citation_author=H Koike; citation_author=H Aramaki; citation_journal_title=FEMS Microbiol Rev; citation_volume=30; citation_number=37; citation_pages=89-108; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Specific and global regulation of genes associated with the degradation of aromatic compounds in bacteria.; citation_author=U Gerischer; citation_journal_title=J Mol Microbiol Biotechnol; citation_volume=4; citation_number=38; citation_pages=111-121; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Members of the IclR family of bacterial transcriptional regulators function as activators and/or repressors.; citation_author=AJ Molina-Henares; citation_author=T Krell; citation_author=M Eugenia Guazzaroni; citation_author=A Segura; citation_author=JL Ramos; citation_journal_title=FEMS Microbiol Rev; citation_volume=30; citation_number=39; citation_pages=157-186; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The TetR family of transcriptional repressors.; citation_author=JL Ramos; citation_author=M Martinez-Bueno; citation_author=AJ Molina-Henares; citation_author=W Teran; citation_author=K Watanabe; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=69; citation_number=40; citation_pages=326-356; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A design for life: prokaryotic metal-binding MerR family regulators.; citation_author=JL Hobman; citation_author=J Wilkie; citation_author=NL Brown; citation_journal_title=Biometals; citation_volume=18; citation_number=41; citation_pages=429-436; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Oxygen and nitrate-dependent regulation of dmsABC operon expression in Escherichia coli: sites for Fnr and NarL protein interactions.; citation_author=SM Bearson; citation_author=JA Albrecht; citation_author=RP Gunsalus; citation_journal_title=BMC Microbiol; citation_volume=2; citation_number=42; citation_pages=13; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Thymidine kinase diversity in bacteria.; citation_author=MP Sandrini; citation_author=AR Clausen; citation_author=B Munch-Petersen; citation_author=J Piskur; citation_journal_title=Nucleosides Nucleotides Nucleic Acids; citation_volume=25; citation_number=43; citation_pages=1153-1158; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Purine catabolism in Escherichia coli and function of xanthine dehydrogenase in purine salvage.; citation_author=H Xi; citation_author=BL Schneider; citation_author=L Reitzer; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=44; citation_pages=5332-5341; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=X-ray structure of the Escherichia coli periplasmic 5′-nucleotidase containing a dimetal catalytic site.; citation_author=T Knofel; citation_author=N Strater; citation_journal_title=Nat Struct Biol; citation_volume=6; citation_number=45; citation_pages=448-453; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Transcriptome dynamics of Deinococcus radiodurans recovering from ionizing radiation.; citation_author=Y Liu; citation_author=J Zhou; citation_author=MV Omelchenko; citation_author=AS Beliaev; citation_author=A Venkateswaran; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=46; citation_pages=4191-4196; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=House cleaning, a part of good housekeeping.; citation_author=MY Galperin; citation_author=OV Moroz; citation_author=KS Wilson; citation_author=AG Murzin; citation_journal_title=Mol Microbiol; citation_volume=59; citation_number=47; citation_pages=5-19; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microbial hemicellulases.; citation_author=D Shallom; citation_author=Y Shoham; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=48; citation_pages=219-228; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Colored moderately thermophilic bacteria in paper-machine biofilms.; citation_author=M Kolari; citation_author=J Nuutinen; citation_author=FA Rainey; citation_author=MS Salkinoja-Salonen; citation_journal_title=J Ind Microbiol Biotechnol; citation_volume=30; citation_number=49; citation_pages=225-238; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Microbial communities of printing paper machines.; citation_author=OM Vaisanen; citation_author=A Weber; citation_author=A Bennasar; citation_author=FA Rainey; citation_author=HJ Busse; citation_journal_title=J Appl Microbiol; citation_volume=84; citation_number=50; citation_pages=1069-1084; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Physiologic determinants of radiation resistance in Deinococcus radiodurans.; citation_author=A Venkateswaran; citation_author=SC McFarlan; citation_author=D Ghostal; citation_author=KW Minton; citation_author=A Vasilenko; citation_journal_title=ApplEnvironMicrobiol; citation_volume=66; citation_number=51; citation_pages=2620-2626; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Development of a defined medium supporting rapid growth for Deinococcus radiodurans and analysis of metabolic capacities.; citation_author=AD Holland; citation_author=HM Rothfuss; citation_author=ME Lidstrom; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=72; citation_number=52; citation_pages=1074-1082; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bacterial heme biosynthesis and its biotechnological application.; citation_author=N Frankenberg; citation_author=J Moser; citation_author=D Jahn; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=63; citation_number=53; citation_pages=115-127; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A novel scenario for the evolution of haem-copper oxygen reductases.; citation_author=MM Pereira; citation_author=M Santana; citation_author=M Teixeira; citation_journal_title=Biochim Biophys Acta; citation_volume=1505; citation_number=54; citation_pages=185-208; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cytochrome bd terminal oxidase.; citation_author=S Junemann; citation_journal_title=Biochim Biophys Acta; citation_volume=1321; citation_number=55; citation_pages=107-127; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Transcriptional control, translation and function of the products of the five open reading frames of the Escherichia coli nir operon.; citation_author=NR Harborne; citation_author=L Griffiths; citation_author=SJ Busby; citation_author=JA Cole; citation_journal_title=Mol Microbiol; citation_volume=6; citation_number=56; citation_pages=2805-2813; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Hydroxylamine assimilation by Rhodobacter capsulatus E1F1. requirement of the hcp gene (hybrid cluster protein) located in the nitrate assimilation nas gene region for hydroxylamine reduction.; citation_author=P Cabello; citation_author=C Pino; citation_author=MF Olmo-Mira; citation_author=F Castillo; citation_author=MD Roldan; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=57; citation_pages=45485-45494; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cell biology and molecular basis of denitrification.; citation_author=WG Zumft; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=61; citation_number=58; citation_pages=533-616; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Novel types of two-domain multi-copper oxidases: possible missing links in the evolution.; citation_author=K Nakamura; citation_author=T Kawabata; citation_author=K Yura; citation_author=N Go; citation_journal_title=FEBS Lett; citation_volume=553; citation_number=59; citation_pages=239-244; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molybdenum and tungsten in biology.; citation_author=R Hille; citation_journal_title=Trends Biochem Sci; citation_volume=27; citation_number=60; citation_pages=360-367; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Specific expansion of protein families in the radioresistant bacterium Deinococcus radiodurans.; citation_author=KS Makarova; citation_author=L Aravind; citation_author=MJ Daly; citation_author=EV Koonin; citation_journal_title=Genetica; citation_volume=108; citation_number=61; citation_pages=25-34; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Characterization of the Mn(2+)-stimulated (di)adenosine polyphosphate hydrolase encoded by the Deinococcus radiodurans DR2356 nudix gene.; citation_author=DI Fisher; citation_author=JL Cartwright; citation_author=AG McLennan; citation_journal_title=Arch Microbiol; citation_volume=186; citation_number=62; citation_pages=415-424; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protective effects of inosine on mice subjected to lethal total-body ionizing irradiation.; citation_author=B Hou; citation_author=ZW Xu; citation_author=CW Yang; citation_author=Y Gao; citation_author=SF Zhao; citation_journal_title=J Radiat Res (Tokyo); citation_volume=48; citation_number=63; citation_pages=57-62; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cloning and sequencing of a chromosomal fragment from Clostridium acetobutylicum strain ABKn8 conferring chemical-damaging agents and UV resistance to E. coli recA strains.; citation_author=H Azeddoug; citation_author=G Reysset; citation_journal_title=Curr Microbiol; citation_volume=29; citation_number=64; citation_pages=229-235; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The nucleotide sequence of a plasmid determinant for resistance to tellurium anions.; citation_author=MG Jobling; citation_author=DA Ritchie; citation_journal_title=Gene; citation_volume=66; citation_number=65; citation_pages=245-258; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Plant pathogenesis-related proteins: molecular mechanisms of gene expression and protein function.; citation_author=S Kitajima; citation_author=F Sato; citation_journal_title=J Biochem (Tokyo); citation_volume=125; citation_number=66; citation_pages=1-8; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Screening of genes involved in isooctane tolerance in Saccharomyces cerevisiae by using mRNA differential display.; citation_author=S Miura; citation_author=W Zou; citation_author=M Ueda; citation_author=A Tanaka; citation_journal_title=Appl Environ Microbiol; citation_volume=66; citation_number=67; citation_pages=4883-4889; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulation of catabolic pathways for aromatic compounds in Corynebacterium glutamicum.; citation_author=K Brinkrolf; citation_author=I Brune; citation_author=A Tauch; citation_journal_title=Genet Mol Res; citation_volume=5; citation_number=68; citation_pages=773-789; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cloning, deletion, and characterization of PadR, the transcriptional repressor of the phenolic acid decarboxylase-encoding padA gene of Lactobacillus plantarum.; citation_author=J Gury; citation_author=L Barthelmebs; citation_author=NP Tran; citation_author=C Divies; citation_author=JF Cavin; citation_journal_title=Appl Environ Microbiol; citation_volume=70; citation_number=69; citation_pages=2146-2153; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=LadR, a new PadR-related transcriptional regulator from Listeria monocytogenes, negatively regulates the expression of the multidrug efflux pump MdrL.; citation_author=E Huillet; citation_author=P Velge; citation_author=T Vallaeys; citation_author=P Pardon; citation_journal_title=FEMS Microbiol Lett; citation_volume=254; citation_number=70; citation_pages=87-94; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization of monofunctional catalase KatA from radioresistant bacterium Deinococcus radiodurans.; citation_author=I Kobayashi; citation_author=T Tamura; citation_author=H Sghaier; citation_author=I Narumi; citation_author=S Yamaguchi; citation_journal_title=J Biosci Bioeng; citation_volume=101; citation_number=71; citation_pages=315-321; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Production of two different catalase-peroxidases by Deinococcus radiophilus.; citation_author=EJ Yun; citation_author=YN Lee; citation_journal_title=FEMS Microbiol Lett; citation_volume=184; citation_number=72; citation_pages=155-159; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Pathways of oxidative damage.; citation_author=JA Imlay; citation_journal_title=Annu Rev Microbiol; citation_volume=57; citation_number=73; citation_pages=395-418; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The crystal structure of aminoglycoside-3′-phosphotransferase-IIa, an enzyme responsible for antibiotic resistance.; citation_author=D Nurizzo; citation_author=SC Shewry; citation_author=MH Perlin; citation_author=SA Brown; citation_author=JN Dholakia; citation_journal_title=J Mol Biol; citation_volume=327; citation_number=74; citation_pages=491-506; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Targeted mutagenesis by duplication insertion in the radioresistant bacterium Deinococcus radiodurans: radiation sensitivities of catalase (katA) and superoxide dismutase (sodA) mutants.; citation_author=LM Markillie; citation_author=SM Varnum; citation_author=P Hradecky; citation_author=KK Wong; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=75; citation_pages=666-669; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Interplasmidic recombination following irradiation of the radioresistant bacterium Deinococcus radiodurans.; citation_author=MJ Daly; citation_author=O Ling; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=76; citation_pages=7506-7515; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Interchromosomal recombination in the extremely radioresistant bacterium Deinococcus radiodurans.; citation_author=MJ Daly; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=77; citation_pages=5495-5505; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=An alternative pathway of recombination of chromosomal fragments precedes recA-dependent recombination in the radioresistant bacterium Deinococcus radiodurans.; citation_author=MJ Daly; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=78; citation_pages=4461-4471; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Recombination between a resident plasmid and the chromosome following irradiation of the radioresistant bacterium Deinococcus radiodurans.; citation_author=MJ Daly; citation_author=KW Minton; citation_journal_title=Gene; citation_volume=187; citation_number=79; citation_pages=225-229; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Arac/XylS family of transcriptional regulators.; citation_author=MT Gallegos; citation_author=R Schleif; citation_author=A Bairoch; citation_author=K Hofmann; citation_author=JL Ramos; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=61; citation_number=80; citation_pages=393-410; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Ribose catabolism of Escherichia coli: characterization of the rpiB gene encoding ribose phosphate isomerase B and of the rpiR gene, which is involved in regulation of rpiB expression.; citation_author=KI Sorensen; citation_author=B Hove-Jensen; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=81; citation_pages=1003-1011; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Isolation and mapping of Escherichia coli mutations conferring resistance to division inhibition protein DicB.; citation_author=C Labie; citation_author=F Bouche; citation_author=JP Bouche; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=82; citation_pages=4315-4319; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Structural determinants of metal selectivity in prokaryotic metal-responsive transcriptional regulators.; citation_author=MA Pennella; citation_author=DP Giedroc; citation_journal_title=Biometals; citation_volume=18; citation_number=83; citation_pages=413-428; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Toxin-antitoxin modules as bacterial metabolic stress managers.; citation_author=L Buts; citation_author=J Lah; citation_author=MH Dao-Thi; citation_author=L Wyns; citation_author=R Loris; citation_journal_title=Trends Biochem Sci; citation_volume=30; citation_number=84; citation_pages=672-679; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prokaryotic toxin-antitoxin stress response loci.; citation_author=K Gerdes; citation_author=SK Christensen; citation_author=A Lobner-Olesen; citation_journal_title=Nat Rev Microbiol; citation_volume=3; citation_number=85; citation_pages=371-382; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacteriophytochromes are photochromic histidine kinases using a biliverdin chromophore.; citation_author=SH Bhoo; citation_author=SJ Davis; citation_author=J Walker; citation_author=B Karniol; citation_author=RD Vierstra; citation_journal_title=Nature; citation_volume=414; citation_number=86; citation_pages=776-779; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Bacteriophytochromes: phytochrome-like photoreceptors from nonphotosynthetic eubacteria.; citation_author=SJ Davis; citation_author=AV Vener; citation_author=RD Vierstra; citation_journal_title=Science; citation_volume=286; citation_number=87; citation_pages=2517-2520; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Pseudomonas aeruginosa biofilms: role of the alginate exopolysaccharide.; citation_author=A Boyd; citation_author=AM Chakrabarty; citation_journal_title=J Ind Microbiol; citation_volume=15; citation_number=88; citation_pages=162-168; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Bacterial alginate biosynthesis–recent progress and future prospects.; citation_author=P Gacesa; citation_journal_title=Microbiology; citation_volume=144 ( Pt 5); citation_number=89; citation_pages=1133-1143; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of uvrA, a DNA repair gene of Deinococcus radiodurans.; citation_author=HJ Agostini; citation_author=JD Carroll; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=90; citation_pages=6759-6765; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Global analysis of the Deinococcus radiodurans proteome by using accurate mass tags.; citation_author=MS Lipton; citation_author=L Pasa-Tolic; citation_author=GA Anderson; citation_author=DJ Anderson; citation_author=DL Auberry; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=91; citation_pages=11049-11054; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of Deinococcus radiodurans's transcriptional response to ionizing radiation and desiccation reveals novel proteins that contribute to extreme radioresistance.; citation_author=M Tanaka; citation_author=AM Earl; citation_author=HA Howell; citation_author=MJ Park; citation_author=JA Eisen; citation_journal_title=Genetics; citation_volume=168; citation_number=92; citation_pages=21-33; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sequencing, targeted mutagenesis and expression of a recA gene required for the extreme radioresistance of Deinococcus radiodurans.; citation_author=PD Gutman; citation_author=JD Carroll; citation_author=CI Masters; citation_author=KW Minton; citation_journal_title=Gene; citation_volume=141; citation_number=93; citation_pages=31-37; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Iterated profile searches with PSI-BLAST–a tool for discovery in protein databases.; citation_author=SF Altschul; citation_author=EV Koonin; citation_journal_title=Trends Biochem Sci; citation_volume=23; citation_number=94; citation_pages=444-447; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=SMART: a web-based tool for the study of genetically mobile domains.; citation_author=J Schultz; citation_author=RR Copley; citation_author=T Doerks; citation_author=CP Ponting; citation_author=P Bork; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=95; citation_pages=231-234; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=CDD: a database of conserved domain alignments with links to domain three-dimensional structure.; citation_author=A Marchler-Bauer; citation_author=AR Panchenko; citation_author=BA Shoemaker; citation_author=PA Thiessen; citation_author=LY Geer; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=96; citation_pages=281-283; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Classification and evolutionary history of the single-strand annealing proteins, RecT, Redbeta, ERF and RAD52.; citation_author=LM Iyer; citation_author=EV Koonin; citation_author=L Aravind; citation_journal_title=BMC Genomics; citation_volume=3; citation_number=97; citation_pages=8; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Preserving genome integrity: the DdrA protein of Deinococcus radiodurans R1.; citation_author=DR Harris; citation_author=M Tanaka; citation_author=SV Saveliev; citation_author=E Jolivet; citation_author=AM Earl; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=98; citation_pages=e304; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Computer analysis of transcription regulatory patterns in completely sequenced bacterial genomes.; citation_author=AA Mironov; citation_author=EV Koonin; citation_author=MA Roytberg; citation_author=MS Gelfand; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=99; citation_pages=2981-2989; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=[Software for analyzing bacterial genomes].; citation_author=AA Mironov; citation_author=NP Vinokurova; citation_author=MS Gel'fand; citation_journal_title=Mol Biol (Mosk); citation_volume=34; citation_number=100; citation_pages=253-262; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=DNA mismatch repair.; citation_author=TA Kunkel; citation_author=DA Erie; citation_journal_title=Annu Rev Biochem; citation_volume=74; citation_number=101; citation_pages=681-710; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Recombinational repair of DNA damage in Escherichia coli and bacteriophage lambda.; citation_author=A Kuzminov; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=63; citation_number=102; citation_pages=751-813, table of contents; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Global response of Escherichia coli cells to a treatment with 7-methoxy-2-nitronaphtho[2,1-b]furan (R7000), an extremely potent mutagen.; citation_author=E Touati; citation_author=C Laurent-Winter; citation_author=P Quillardet; citation_author=M Hofnung; citation_journal_title=Mutat Res; citation_volume=349; citation_number=103; citation_pages=193-200; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The DNA excision repair system of the highly radioresistant bacterium Deinococcus radiodurans is facilitated by the pentose phosphate pathway.; citation_author=YM Zhang; citation_author=JK Liu; citation_author=TY Wong; citation_journal_title=Mol Microbiol; citation_volume=48; citation_number=104; citation_pages=1317-1323; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Induction and repression of the histidine-degrading enzymes of Bacillus subtilis.; citation_author=LA Chasin; citation_author=B Magasanik; citation_journal_title=J Biol Chem; citation_volume=243; citation_number=105; citation_pages=5165-5178; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Histidine utilisation operon (hut) is upregulated at low temperature in the antarctic psychrotrophic bacterium Pseudomonas syringae.; citation_author=K Kannan; citation_author=KL Janiyani; citation_author=S Shivaji; citation_author=MK Ray; citation_journal_title=FEMS Microbiol Lett; citation_volume=161; citation_number=106; citation_pages=7-14; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Respiratory activity as a determinant of radiation survival response.; citation_author=AK Bruce; citation_author=JD Berner; citation_journal_title=Can J Microbiol; citation_volume=22; citation_number=107; citation_pages=1336-1344; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=The IrrE protein of Deinococcus radiodurans R1 is a novel regulator of recA expression.; citation_author=AM Earl; citation_author=MM Mohundro; citation_author=IS Mian; citation_author=JR Battista; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=108; citation_pages=6216-6224; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PprI: a general switch responsible for extreme radioresistance of Deinococcus radiodurans.; citation_author=Y Hua; citation_author=I Narumi; citation_author=G Gao; citation_author=B Tian; citation_author=K Satoh; citation_journal_title=Biochem Biophys Res Commun; citation_volume=306; citation_number=109; citation_pages=354-360; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Internal promoter characterization and expression of the Deinococcus radiodurans pprI-folP gene cluster.; citation_author=G Gao; citation_author=D Le; citation_author=L Huang; citation_author=H Lu; citation_author=I Narumi; citation_journal_title=FEMS Microbiol Lett; citation_volume=257; citation_number=110; citation_pages=195-201; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Expression of recA in Deinococcus radiodurans.; citation_author=JD Carroll; citation_author=MJ Daly; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=111; citation_pages=130-135; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cloning and characterization of DNA damage-inducible promoter regions from Bacillus subtilis.; citation_author=DL Cheo; citation_author=KW Bayles; citation_author=RE Yasbin; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=112; citation_pages=1696-1703; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The SOB system of Bacillus subtilis: a global regulon involved in DNA repair and differentiation.; citation_author=RE Yasbin; citation_author=D Cheo; citation_author=KW Bayles; citation_journal_title=Res Microbiol; citation_volume=142; citation_number=113; citation_pages=885-892; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Comparative gene expression profiles following UV exposure in wild-type and SOS-deficient Escherichia coli.; citation_author=J Courcelle; citation_author=A Khodursky; citation_author=B Peter; citation_author=PO Brown; citation_author=PC Hanawalt; citation_journal_title=Genetics; citation_volume=158; citation_number=114; citation_pages=41-64; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of additional genes belonging to the LexA regulon in Escherichia coli.; citation_author=AR Fernandez De Henestrosa; citation_author=T Ogi; citation_author=S Aoyagi; citation_author=D Chafin; citation_author=JJ Hayes; citation_journal_title=Mol Microbiol; citation_volume=35; citation_number=115; citation_pages=1560-1572; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The LexA protein from Deinococcus radiodurans is not involved in RecA induction following gamma irradiation.; citation_author=I Narumi; citation_author=K Satoh; citation_author=M Kikuchi; citation_author=T Funayama; citation_author=T Yanagisawa; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=116; citation_pages=6951-6956; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Purified LexA protein is a repressor of the recA and lexA genes.; citation_author=JW Little; citation_author=DW Mount; citation_author=CR Yanisch-Perron; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=78; citation_number=117; citation_pages=4199-4203; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Repair of irradiation transforming deoxyribonucleic acid in wild type and a radiation-sensitive mutant of Micrococcus radiodurans.; citation_author=BE Moseley; citation_author=A Mattingly; citation_journal_title=J Bacteriol; citation_volume=105; citation_number=118; citation_pages=976-983; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=Partial complementation of the UV sensitivity of Deinococcus radiodurans excision repair mutants by the cloned denV gene of bacteriophage T4.; citation_author=PD Gutman; citation_author=HL Yao; citation_author=KW Minton; citation_journal_title=Mutat Res; citation_volume=254; citation_number=119; citation_pages=207-215; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Repair of ionizing-radiation damage in the radiation resistant bacterium Deinococcus radiodurans.; citation_author=KW Minton; citation_journal_title=Mutat Res; citation_volume=363; citation_number=120; citation_pages=1-7; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Role of RecA in DNA damage repair in Deinococcus radiodurans.; citation_author=DJ Schlesinger; citation_journal_title=FEMS Microbiol Lett; citation_volume=274; citation_number=121; citation_pages=342-347; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Resistance to radiation.; citation_author=MJ Daly; citation_author=KW Minton; citation_journal_title=Science; citation_volume=270; citation_number=122; citation_pages=1318; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Fine structure of the Deinococcus radiodurans nucleoid revealed by cryoelectron microscopy of vitreous sections.; citation_author=M Eltsov; citation_author=J Dubochet; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=123; citation_pages=8047-8054; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Study of the Deinococcus radiodurans nucleoid by cryoelectron microscopy of vitreous sections: Supplementary comments.; citation_author=M Eltsov; citation_author=J Dubochet; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=124; citation_pages=6053-6058; discussion 6059; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ring-like nucleoid does not play a key role in radioresistance of Deinococcus radiodurans.; citation_author=G Gao; citation_author=H Lu; citation_author=L Yin; citation_author=Y Hua; citation_journal_title=Sci China C Life Sci; citation_volume=50; citation_number=125; citation_pages=525-529; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The resistance of Micrococcus radiodurans to ultraviolet radiation: Ultraviolet-induced lesions in the cell's DNA.; citation_author=DM Setlow; citation_author=DE Duggan; citation_journal_title=Biochim Biophys Acta; citation_volume=87; citation_number=126; citation_pages=664-668; citation_date=1964; " />
      <meta name="citation_reference" content="citation_title=Three tandem HRDC domains have synergistic effect on the RecQ functions in Deinococcus radiodurans.; citation_author=L Huang; citation_author=X Hua; citation_author=H Lu; citation_author=G Gao; citation_author=B Tian; citation_journal_title=DNA Repair (Amst); citation_volume=6; citation_number=127; citation_pages=167-176; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=RecA Protein from the extremely radioresistant bacterium Deinococcus radiodurans: expression, purification, and characterization.; citation_author=JI Kim; citation_author=AK Sharma; citation_author=SN Abbott; citation_author=EA Wood; citation_author=DW Dwyer; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=128; citation_pages=1649-1660; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of the Deinococcus radiodurans recA locus and identification of a mutation site in a DNA repair-deficient mutant, rec30.; citation_author=I Narumi; citation_author=K Satoh; citation_author=M Kikuchi; citation_author=T Funayama; citation_author=S Kitayama; citation_journal_title=Mutat Res; citation_volume=435; citation_number=129; citation_pages=233-243; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the Deinococcus radiodurans single-stranded DNA-binding protein suggests a mechanism for coping with DNA damage.; citation_author=DA Bernstein; citation_author=JM Eggington; citation_author=MP Killoran; citation_author=AM Misic; citation_author=MM Cox; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=130; citation_pages=8575-8580; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Making ends meet: repairing breaks in bacterial DNA by non-homologous end-joining.; citation_author=R Bowater; citation_author=AJ Doherty; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=131; citation_pages=e8; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA toroids: framework for DNA repair in Deinococcus radiodurans and in germinating bacterial spores.; citation_author=J Englander; citation_author=E Klein; citation_author=V Brumfeld; citation_author=AK Sharma; citation_author=AJ Doherty; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=132; citation_pages=5973-5977; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Radiation response mechanisms of the extremely radioresistant bacterium Deinococcus radiodurans.; citation_author=Y Kobayashi; citation_author=I Narumi; citation_author=K Satoh; citation_author=T Funayama; citation_author=M Kikuchi; citation_journal_title=Biol Sci Space; citation_volume=18; citation_number=133; citation_pages=134-135; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Involvement of an X family DNA polymerase in double-stranded break repair in the radioresistant organism Deinococcus radiodurans.; citation_author=F Lecointe; citation_author=IV Shevelev; citation_author=A Bailone; citation_author=S Sommer; citation_author=U Hubscher; citation_journal_title=Mol Microbiol; citation_volume=53; citation_number=134; citation_pages=1721-1730; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ringlike structure of the Deinococcus radiodurans genome: a key to radioresistance?; citation_author=S Levin-Zaidman; citation_author=J Englander; citation_author=E Shimoni; citation_author=AK Sharma; citation_author=KW Minton; citation_journal_title=Science; citation_volume=299; citation_number=135; citation_pages=254-256; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PprA: a novel protein from Deinococcus radiodurans that stimulates DNA ligation.; citation_author=I Narumi; citation_author=K Satoh; citation_author=S Cui; citation_author=T Funayama; citation_author=S Kitayama; citation_journal_title=Mol Microbiol; citation_volume=54; citation_number=136; citation_pages=278-285; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Enzymes involved in DNA ligation and end-healing in the radioresistant bacterium Deinococcus radiodurans.; citation_author=M Blasius; citation_author=R Buob; citation_author=IV Shevelev; citation_author=U Hubscher; citation_journal_title=BMC Mol Biol; citation_volume=8; citation_number=137; citation_pages=69; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Untargeted effects of ionizing radiation: implications for radiation pathology.; citation_author=EG Wright; citation_author=PJ Coates; citation_journal_title=Mutat Res; citation_volume=597; citation_number=138; citation_pages=119-132; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biological effects in unirradiated human tissue induced by radiation damage up to 1 mm away.; citation_author=OV Belyakov; citation_author=SA Mitchell; citation_author=D Parikh; citation_author=G Randers-Pehrson; citation_author=SA Marino; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=139; citation_pages=14203-14208; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The chemical basis of radiation biology.; citation_author=C von Sonntag; citation_number=140; citation_date=1987; citation_publisher=Taylor & Francis; " />
      <meta name="citation_reference" content="citation_title=Proteins are major initial cell targets of hydroxyl free radicals.; citation_author=J Du; citation_author=JM Gebicki; citation_journal_title=Int J Biochem Cell Biol; citation_volume=36; citation_number=141; citation_pages=2334-2343; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Elemental and redox analysis of single bacterial cells by x-ray microbeam analysis.; citation_author=KM Kemner; citation_author=SD Kelly; citation_author=B Lai; citation_author=J Maser; citation_author=J O'Loughlin E; citation_journal_title=Science; citation_volume=306; citation_number=142; citation_pages=686-687; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lesion bypass activity of DNA polymerase A from the extremely radioresistant organism Deinococcus radiodurans.; citation_author=K Heinz; citation_author=A Marx; citation_journal_title=J Biol Chem; citation_volume=282; citation_number=143; citation_pages=10908-10914; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Transcriptome analysis applied to survival of Shewanella oneidensis MR-1 exposed to ionizing radiation.; citation_author=X Qiu; citation_author=MJ Daly; citation_author=A Vasilenko; citation_author=MV Omelchenko; citation_author=EK Gaidamakova; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=144; citation_pages=1199-1204; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Parallel adaptive evolution cultures of Escherichia coli lead to convergent growth phenotypes with different gene expression states.; citation_author=SS Fong; citation_author=AR Joyce; citation_author=BO Palsson; citation_journal_title=Genome Res; citation_volume=15; citation_number=145; citation_pages=1365-1372; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chance and necessity in cellular response to challenge.; citation_author=EV Koonin; citation_number=146; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-wide transcriptional plasticity underlies cellular adaptation to novel changes.; citation_author=S Stern; citation_author=T Dror; citation_author=E Stolovicki; citation_author=N Brenner; citation_author=E Braun; citation_number=147; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Truepera radiovictrix gen. nov., sp. nov., a new radiation resistant species and the proposal of Trueperaceae fam. nov.; citation_author=L Albuquerque; citation_author=C Simoes; citation_author=MF Nobre; citation_author=NM Pino; citation_author=JR Battista; citation_journal_title=FEMS Microbiol Lett; citation_volume=247; citation_number=148; citation_pages=161-169; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Isothermal strand-displacement amplification applications for high-throughput genomics.; citation_author=JC Detter; citation_author=JM Jett; citation_author=SM Lucas; citation_author=E Dalin; citation_author=AR Arellano; citation_journal_title=Genomics; citation_volume=80; citation_number=149; citation_pages=691-698; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=150; citation_pages=186-194; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment.; citation_author=B Ewing; citation_author=L Hillier; citation_author=MC Wendl; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=151; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing.; citation_author=D Gordon; citation_author=C Abajian; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=152; citation_pages=195-202; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The COG database: new developments in phylogenetic classification of proteins from complete genomes.; citation_author=RL Tatusov; citation_author=DA Natale; citation_author=IV Garkavtsev; citation_author=TA Tatusova; citation_author=UT Shankavaram; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=153; citation_pages=22-28; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=RC Edgar; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=154; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics of the lactic acid bacteria.; citation_author=K Makarova; citation_author=A Slesarev; citation_author=Y Wolf; citation_author=A Sorokin; citation_author=B Mirkin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=155; citation_pages=15611-15616; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ancestral paralogs and pseudoparalogs and their role in the emergence of the eukaryotic cell.; citation_author=KS Makarova; citation_author=YI Wolf; citation_author=SL Mekhedov; citation_author=BG Mirkin; citation_author=EV Koonin; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=156; citation_pages=4626-4638; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conserved patterns in the Cu,Zn superoxide dismutase family.; citation_author=D Bordo; citation_author=K Djinovic; citation_author=M Bolognesi; citation_journal_title=J Mol Biol; citation_volume=238; citation_number=157; citation_pages=366-386; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Structure of the single-strand annealing domain of human RAD52 protein.; citation_author=MR Singleton; citation_author=LM Wentzell; citation_author=Y Liu; citation_author=SC West; citation_author=DB Wigley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=158; citation_pages=13492-13497; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Lactococcal phage genes involved in sensitivity to AbiK and their relation to single-strand annealing proteins.; citation_author=JD Bouchard; citation_author=S Moineau; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=159; citation_pages=3649-3652; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences.; citation_author=TD Schneider; citation_author=RM Stephens; citation_journal_title=Nucleic Acids Res; citation_volume=18; citation_number=160; citation_pages=6097-6100; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Mutation of D. radiodurans in a gene homologous to ruvB of E. coli.; citation_author=S Kitayama; citation_author=M Kohoroku; citation_author=A Takagi; citation_author=H Itoh; citation_journal_title=Mutat Res; citation_volume=385; citation_number=161; citation_pages=151-157; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Novel ionizing radiation-sensitive mutants of Deinococcus radiodurans.; citation_author=KS Udupa; citation_author=PA O'Cain; citation_author=V Mattimore; citation_author=JR Battista; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=162; citation_pages=7439-7446; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Inactivation of Two Homologues of Proteins Presumed to be Involved in the Desiccation Tolerance of Plants Sensitizes Deinococcus radiodurans R1 to Desiccation.; citation_author=JR Battista; citation_author=MJ Park; citation_author=AE McLemore; citation_journal_title=Cryobiology; citation_volume=43; citation_number=163; citation_pages=133-139; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mutation in recR gene of Deinococcus radiodurans and possible involvement of its product in the repair of DNA interstrand cross-links.; citation_author=S Kitayama; citation_author=I Narumi; citation_author=M Kikuchi; citation_author=H Watanabe; citation_journal_title=Mutat Res; citation_volume=461; citation_number=164; citation_pages=179-187; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification and disruption analysis of the recN gene in the extremely radioresistant bacterium Deinococcus radiodurans.; citation_author=T Funayama; citation_author=I Narumi; citation_author=M Kikuchi; citation_author=S Kitayama; citation_author=H Watanabe; citation_journal_title=Mutat Res; citation_volume=435; citation_number=165; citation_pages=151-161; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Identification, sequencing, and targeted mutagenesis of a DNA polymerase gene required for the extreme radioresistance of Deinococcus radiodurans.; citation_author=PD Gutman; citation_author=P Fuchs; citation_author=L Ouyang; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=166; citation_pages=3581-3590; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Radioresistance of Deinococcus radiodurans: functions necessary to survive ionizing radiation are also necessary to survive prolonged desiccation.; citation_author=V Mattimore; citation_author=JR Battista; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=167; citation_pages=633-637; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Down-regulation of radioresistance by LexA2 in Deinococcus radiodurans.; citation_author=K Satoh; citation_author=H Ohba; citation_author=H Sghaier; citation_author=I Narumi; citation_journal_title=Microbiology; citation_volume=152; citation_number=168; citation_pages=3217-3226; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=LexA analog (dra0074) is a regulatory protein that is irrelevant to recA induction.; citation_author=D Sheng; citation_author=Z Zheng; citation_author=B Tian; citation_author=B Shen; citation_author=Y Hua; citation_journal_title=J Biochem (Tokyo); citation_volume=136; citation_number=169; citation_pages=787-793; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=[Construction of the recQ double mutants and analysis of adversity in Deinococcus radiodurans].; citation_author=LF Huang; citation_author=SW Zhang; citation_author=XT Hua; citation_author=GJ Gao; citation_author=YJ Hua; citation_journal_title=Wei Sheng Wu Xue Bao; citation_volume=46; citation_number=170; citation_pages=205-209; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000955" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Deinococcus geothermalis: The Pool of Extreme Radiation Resistance Genes Shrinks"/>
    <meta name="twitter:description" content="Bacteria of the genus Deinococcus are extremely resistant to ionizing radiation (IR), ultraviolet light (UV) and desiccation. The mesophile Deinococcus radiodurans was the first member of this group whose genome was completely sequenced. Analysis of the genome sequence of D. radiodurans, however, failed to identify unique DNA repair systems. To further delineate the genes underlying the resistance phenotypes, we report the whole-genome sequence of a second Deinococcus species, the thermophile Deinococcus geothermalis, which at its optimal growth temperature is as resistant to IR, UV and desiccation as D. radiodurans, and a comparative analysis of the two Deinococcus genomes. Many D. radiodurans genes previously implicated in resistance, but for which no sensitive phenotype was observed upon disruption, are absent in D. geothermalis. In contrast, most D. radiodurans genes whose mutants displayed a radiation-sensitive phenotype in D. radiodurans are conserved in D. geothermalis. Supporting the existence of a Deinococcus radiation response regulon, a common palindromic DNA motif was identified in a conserved set of genes associated with resistance, and a dedicated transcriptional regulator was predicted. We present the case that these two species evolved essentially the same diverse set of gene families, and that the extreme stress-resistance phenotypes of the Deinococcus lineage emerged progressively by amassing cell-cleaning systems from different sources, but not by acquisition of novel DNA repair systems. Our reconstruction of the genomic evolution of the Deinococcus-Thermus phylum indicates that the corresponding set of enzymes proliferated mainly in the common ancestor of Deinococcus. Results of the comparative analysis weaken the arguments for a role of higher-order chromosome alignment structures in resistance; more clearly define and substantially revise downward the number of uncharacterized genes that might participate in DNA repair and contribute to resistance; and strengthen the case for a role in survival of systems involved in manganese and iron homeostasis."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000955.g007"/>

  <meta property="og:title" content="Deinococcus geothermalis: The Pool of Extreme Radiation Resistance Genes Shrinks" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000955" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3401'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1632'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5898&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000955">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000955" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000955&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    <i>Deinococcus geothermalis</i>: The Pool of Extreme Radiation Resistance Genes Shrinks
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kira S. Makarova
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:makarova@ncbi.nlm.nih.gov">makarova@ncbi.nlm.nih.gov</a> (KM); <a href="mailto:mdaly@usuhs.mil">mdaly@usuhs.mil</a> (MD)</p>

                <p>Affiliation:
                  National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marina V. Omelchenko, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elena K. Gaidamakova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vera Y. Matrosova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexander Vasilenko, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Min Zhai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alla Lapidus, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alex Copeland, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edwin Kim, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Miriam Land, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Konstantinos Mavromatis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Samuel Pitluck, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul M. Richardson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Walnut Creek, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chris Detter, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Brettin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elizabeth Saunders, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  US Department of Energy, Joint Genome Institute, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Barry Lai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Environmental Research Division and Advanced Photon Source, Argonne National Laboratory, Argonne, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bruce Ravel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Environmental Research Division and Advanced Photon Source, Argonne National Laboratory, Argonne, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth M. Kemner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Environmental Research Division and Advanced Photon Source, Argonne National Laboratory, Argonne, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuri I. Wolf, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexander Sorokin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anna V. Gerasimova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Research Institute of Genetics and Selection of Industrial Microorganisms, Moscow, Russia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mikhail S. Gelfand, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute for Information Transmission Problems of RAS, Moscow, Russia, 
                  Faculty of Bioengineering and Bioinformatics, M. V. Lomonosov Moscow State University, Moscow, Russia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James K. Fredrickson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eugene V. Koonin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael J. Daly
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:makarova@ncbi.nlm.nih.gov">makarova@ncbi.nlm.nih.gov</a> (KM); <a href="mailto:mdaly@usuhs.mil">mdaly@usuhs.mil</a> (MD)</p>

                <p>Affiliation:
                  Department of Pathology, Uniformed Services University of the Health Sciences (USUHS), Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000955</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000955" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000955" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000955" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000955" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000955-g001" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-t001" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-g002" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-t002" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-g003" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-g004" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-t003" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-g005" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-g006" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-t004" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000955-g007" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000955">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000955" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Bacteria of the genus <em>Deinococcus</em> are extremely resistant to ionizing radiation (IR), ultraviolet light (UV) and desiccation. The mesophile <em>Deinococcus radiodurans</em> was the first member of this group whose genome was completely sequenced. Analysis of the genome sequence of <em>D. radiodurans</em>, however, failed to identify unique DNA repair systems. To further delineate the genes underlying the resistance phenotypes, we report the whole-genome sequence of a second <em>Deinococcus</em> species, the thermophile <em>Deinococcus geothermalis</em>, which at its optimal growth temperature is as resistant to IR, UV and desiccation as <em>D. radiodurans</em>, and a comparative analysis of the two <em>Deinococcus</em> genomes. Many <em>D. radiodurans</em> genes previously implicated in resistance, but for which no sensitive phenotype was observed upon disruption, are absent in <em>D. geothermalis</em>. In contrast, most <em>D. radiodurans</em> genes whose mutants displayed a radiation-sensitive phenotype in <em>D. radiodurans</em> are conserved in <em>D. geothermalis</em>. Supporting the existence of a <em>Deinococcus</em> radiation response regulon, a common palindromic DNA motif was identified in a conserved set of genes associated with resistance, and a dedicated transcriptional regulator was predicted. We present the case that these two species evolved essentially the same diverse set of gene families, and that the extreme stress-resistance phenotypes of the <em>Deinococcus</em> lineage emerged progressively by amassing cell-cleaning systems from different sources, but not by acquisition of novel DNA repair systems. Our reconstruction of the genomic evolution of the <em>Deinococcus-Thermus</em> phylum indicates that the corresponding set of enzymes proliferated mainly in the common ancestor of <em>Deinococcus</em>. Results of the comparative analysis weaken the arguments for a role of higher-order chromosome alignment structures in resistance; more clearly define and substantially revise downward the number of uncharacterized genes that might participate in DNA repair and contribute to resistance; and strengthen the case for a role in survival of systems involved in manganese and iron homeostasis.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Makarova KS, Omelchenko MV, Gaidamakova EK, Matrosova VY, Vasilenko A, et al.  (2007) <em>Deinococcus geothermalis</em>: The Pool of Extreme Radiation Resistance Genes Shrinks. PLoS ONE 2(9):
          e955.
            doi:10.1371/journal.pone.0000955</p><p><strong>Academic Editor: </strong>Michael Lichten, National Cancer Institute, United States of America</p><p><strong>Received:</strong> July 24, 2007; <strong>Accepted:</strong> September 4, 2007; <strong>Published:</strong> September 26, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>The work of KSM, MVO, YIW, AS, and EVK was supported by the Intramural Research Program of the National Institutes of Health, National Library of Medicine. The work at USUHS was supported by grant DE-FG02-04ER63918 to MJD from the U. S. Department of Energy (DOE), Office of Science, Office of Biological and Environmental Research (BER), Environmental Remediation Sciences Program (ERSP); and by grant FA9550-07-1-0218 to MJD from the Air Force Office of Scientific Research. The work at the DOE-Joint Genome Institute was supported by the DOE Office of Science. Work at the Advanced Photon Source was supported by the DOE Office of Science, Office of Basic Energy Sciences, under Contract No. DE-AC02-06CH11357. The work of MSG and AVG was supported by grants from the Howard Hughes Medical Institute (55005610), INTAS (05-8028), and the Molecular and Cellular Virology program of the Russian Academy of Sciences. D. geothermalis was selected for genome sequencing by BER (<a href="http://www.science.doe.gov/ober/RFS-2.pdf">http://www.science.doe.gov/ober/RFS-2.pd​f</a>) with MJD as the Principal Investigator.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Deinococcus geothermalis</em> belongs to the <em>Deinococcus</em>-<em>Thermus</em> group, which is deeply branched in bacterial phylogenetic trees and has putative relationships with cyanobacteria <a href="#pone.0000955-Gupta1">[1]</a>, <a href="#pone.0000955-Wolf1">[2]</a>. The extremely radiation-resistant family <em>Deinococcaceae</em> is comprised of greater than twenty distinct species <a href="#pone.0000955-Lai1">[3]</a> that can survive acute exposures to ionizing radiation (IR) (10 kGy), ultraviolet light (UV) (1 kJ/m<sup>2</sup>), and desiccation (years) <a href="#pone.0000955-Gutman1">[4]</a>, <a href="#pone.0000955-Daly1">[5]</a>; and can grow under chronic IR (60 Gy/hour) <a href="#pone.0000955-Daly2">[6]</a>. <em>D. geothermalis</em> was originally isolated from a hot pool at the Termi di Agnano, Naples, Italy <a href="#pone.0000955-Ferreira1">[7]</a>, and subsequently identified at other locations poor in organic nutrients including industrial paper machine water <a href="#pone.0000955-Saarimaa1">[8]</a>, deep ocean subsurface environments <a href="#pone.0000955-Kimura1">[9]</a>, and subterranean hot springs in Iceland <a href="#pone.0000955-Marteinsson1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p><em>D. geothermalis</em> is distinct from most members of the genus <em>Deinococcus</em> in that it is a moderate thermophile, with an optimal growth temperature (T<sub>opt</sub>) of 50°C <a href="#pone.0000955-Ferreira1">[7]</a>, is not dependent on an exogenous source of amino acids or nicotinamide for growth <a href="#pone.0000955-Brim1">[11]</a>, <a href="#pone.0000955-Ghosal1">[12]</a>, is capable of forming biofilms <a href="#pone.0000955-Saarimaa1">[8]</a>, and possesses membranes with very low levels of unsaturated fatty acids compared to the other species <a href="#pone.0000955-Ferreira1">[7]</a>. Based on the ability of wild-type and engineered <em>D. geothermalis</em> and <em>D. radiodurans</em> to reduce a variety of metals including U(VI), Cr(VI), Hg(II), Tc(VII), Fe(III) and Mn(III,IV) <a href="#pone.0000955-Brim1">[11]</a>, <a href="#pone.0000955-Fredrickson1">[13]</a>, these two species have been proposed for bioremediation of radioactive waste sites maintained by the US Department of Energy (DOE) <a href="#pone.0000955-Brim1">[11]</a>, <a href="#pone.0000955-Brim2">[14]</a>, <a href="#pone.0000955-Brim3">[15]</a>. These characteristics were the impetus for whole-genome sequencing of <em>D. geothermalis</em> at DOE's Joint Genome Institute, and comparison with the mesophilic <em>D. radiodurans</em> (T<sub>opt</sub>, 32°C), to date the only other extremely IR-resistant bacterium for which a whole-genome sequence has been acquired <a href="#pone.0000955-White1">[16]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Chromosomal and plasmid DNAs in extremely resistant bacteria are as susceptible to IR-induced DNA double strand breaks (DSBs) as in sensitive bacteria <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Daly3">[17]</a>–<a href="#pone.0000955-Gerard1">[19]</a> and broad-based experimental and bioinformatic studies have converged on the conclusion that <em>D. radiodurans</em> uses a conventional set of DNA repair and protection functions, but with a far greater efficiency than IR-sensitive bacteria <a href="#pone.0000955-Daly3">[17]</a>, <a href="#pone.0000955-Cox1">[20]</a>, <a href="#pone.0000955-Makarova1">[21]</a>. This apparent contradiction is exemplified by work which showed that the repair protein DNA polymerase I (PolA) of <em>D. radiodurans</em> supports exceptionally efficient DNA replication at the earliest stages of recovery from IR, and could account for the high fidelity of RecA-mediated DNA fragment assembly <a href="#pone.0000955-Zahradka1">[22]</a>. Paradoxically, however, IR-, UV-, and mitomycin-C (MMC)-sensitive <em>D. radiodurans pol</em>A mutants are fully complemented by expression of the <em>pol</em>A gene from the IR-sensitive <em>Escherichia coli</em> <a href="#pone.0000955-Gutman1">[4]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The reason why repair proteins, either native or cloned, in <em>D. radiodurans</em> function so much better after irradiation than in sensitive bacteria is unknown. The prevailing hypotheses of extreme IR resistance in <em>D. radiodurans</em> fall into three categories: (i) chromosome alignment, morphology and/or repeated sequences facilitate genome reassembly <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Makarova1">[21]</a>, <a href="#pone.0000955-Daly5">[23]</a>, <a href="#pone.0000955-Zimmerman1">[24]</a>; (ii) a subset of uncharacterized genes encode functions that enhance the efficiency of DNA repair <a href="#pone.0000955-Cox1">[20]</a>; and (iii) non-enzymic Mn(II) complexes present in resistant bacteria protect proteins, but not DNA, from oxidation during irradiation, with the result that conventional enzyme systems involved in recovery survive and function with far greater efficiency than in sensitive bacteria <a href="#pone.0000955-Daly3">[17]</a>, <a href="#pone.0000955-Daly5">[23]</a>. The extraordinary survival of <em>Deinococcus</em> bacteria following irradiation has also given rise to some rather whimsical descriptions of their derivation, including that they evolved on Mars <a href="#pone.0000955-Pavlov1">[25]</a>. On the basis of whole-genome comparisons between two <em>Deinococcus</em> genomes and two <em>Thermus</em> genomes, we present a reconstruction of evolutionary events that are inferred to have occurred both before and after the divergence of the <em>D. radiodurans</em> and <em>D. geothermalis</em> lineages. We revise down substantially the number of potential genetic determinants of radiation resistance, predict a <em>Deinococcus</em> radiation response regulon, and consider the implications of these comparative-genomic findings for different models of recovery.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Resistance to IR and UV</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>One approach to delineating a minimal set of genes involved in extreme resistance is to compare the whole-genome sequences of two phylogenetically related but distinct species that are equally resistant, whereby genes that are unique to both organisms are ruled out, whereas shared genes are pooled as candidates for involvement in resistance. We show that D. geothermalis (DSM 11300) and D. radiodurans (ATCC BAA-816) are equally resistant to IR (<sup>60</sup>Co) (<a href="#pone-0000955-g001">Figure 1A</a>) and UV (254 nm) (<a href="#pone-0000955-g001">Figure 1B</a>) when pre-grown and recovered at their optimal growth temperatures, 50°C and 32°C, respectively. When recovered at 50°C, the survival of D. geothermalis exposed to 12 kGy was 1,000 times greater than at 32°C (<a href="#pone-0000955-g001">Figure 1A</a>) <a href="#pone.0000955-Ferreira1">[7]</a>. The extreme resistance to desiccation of D. geothermalis recovered at 50°C was demonstrated previously <a href="#pone.0000955-Daly1">[5]</a>. Thus, D. geothermalis and D. radiodurans are well-suited to defining a conserved set of genes responsible for extreme resistance.</p>
<div class="figure" id="pone-0000955-g001"><div class="img"><a name="pone-0000955-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Radiation resistance and genome structure of <em>D. geothermalis</em> and <em>D. radiodurans</em>. A, IR (<sup>60</sup>Co, 5.5 kGy/h).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>B, UV (254 nm) (3 J/m<sup>2</sup> s<sup>−1</sup>). Open circle, <em>D. radiodurans</em> (32°C); open triangle, <em>D. geothermalis</em> (50°C); and open square, <em>D. geothermalis</em> (32°C). Values are from three independent trials with standard deviations shown. At near-optimal growth temperatures, the 10% survival values (D<sub>10</sub>) following IR for <em>D. radiodurans</em> (32°C) and <em>D. geothermalis</em> (50°C) are 15 kGy; for <em>E. coli</em>, 0.7 kGy (37°C) <a href="#pone.0000955-Daly1">[5]</a>; and for <em>T. thermophilus</em> (HB27) 0.8 kGy (65°C) <a href="#pone.0000955-Omelchenko1">[27]</a>. C, PFGE of genomic DNA prepared from irradiated (0.2 kGy) <em>D. radiodurans</em> (DR+IR) and <em>D. geothermalis</em> (DG+IR); and genomic DNA from non-irradiated <em>D. geothermalis</em> digested with <em>Spe</em>I (DG+<em>Spe</em>I). (M) PFGE DNA size markers. PFGE was as described previously <a href="#pone.0000955-Daly7">[77]</a>.</p>
<span>doi:10.1371/journal.pone.0000955.g001</span></div>

<h4>Genome Sequence and Structure: General Features</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The random shotgun method <a href="#pone.0000955-White1">[16]</a> was used to acquire the complete sequence of the <em>D. geothermalis</em> (DSM 11300) genome, that is comprised of a main chromosome (2,467,205 base pairs (bp)), and two megaplasmids (574,127 bp and 205,686 bp). The general structure of the predicted <em>D. geothermalis</em> genome was tested by pulsed field gel electrophoresis (PFGE) of genomic DNA linearized <em>in vivo</em> by exposure to IR (0.2 kGy), and by restriction endonuclease (<em>Spe</em>I) cleavage (<a href="#pone-0000955-g001">Figure 1C</a>). The IR-treatment revealed the existence of a ~570 kb megaplasmid in <em>D. geothermalis</em>, and the <em>Spe</em>I-treatment yielded the expected number of chromosomal bands: 3 singlets (632 kb, 376 kb and 282 kb) and one doublet (574/579 kb); the plasmids do not contain a <em>Spe</em>I site. In comparison, IR-treated <em>D. radiodurans</em> (ATCC BAA-816) subjected to PFGE displayed the presence of the DR412 (412 kb) and DR177 (177 kb) megaplasmids, previously observed <a href="#pone.0000955-Lin1">[26]</a>. The approximately 206 kb <em>D. geothermalis</em> megaplasmid was not visualized by PFGE although its size lies between the two <em>D. radiodurans</em> megaplasmids, which were readily observed (<a href="#pone-0000955-g001">Figure 1C</a>). Consistently, the abundance of DNA clones for the 206 kb megaplasmid was significantly lower than the 574 kb megaplasmid during construction of the <em>D. geothermalis</em> genome-library used for sequencing (data not shown). Thus, the 574 kb megaplasmid of <em>D. geothermalis</em> exists at higher copy-number than the 206 kb megaplasmid.</p>


<h4>Genome Comparison: General Features</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Comparison of the general genome features of <em>D. geothermalis</em> and <em>D. radiodurans</em> revealed major differences in genome partitioning, and in the number of noncoding repeats (SNRs) (<a href="#pone-0000955-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000955-t001"><div class="img"><a name="pone-0000955-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>General Characteristics</span></strong></p><span>doi:10.1371/journal.pone.0000955.t001</span></div><h5>Genome Partitioning.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>We previously demonstrated homologous relationships between the DR412 megaplasmid of <em>D. radiodurans</em> and the sole 233 kb megaplasmid (pTT27) of <em>T. thermophilus</em> <a href="#pone.0000955-Omelchenko1">[27]</a>. Based on the gene contents of DR412 and pTT27, we concluded that these megaplasmids evolved from a common ancestor (<a href="#pone.0000955.s001">Figure S1</a>), are essential to the survival of both species, and appear to serve as a sink for horizontally transferred genes <a href="#pone.0000955-Omelchenko1">[27]</a>. In contrast, the 574 kb megaplasmid (DG574) of <em>D. geothermalis</em> is distinct from pTT27, and appears to have been derived from a fusion of DR412 and DR177 (<a href="#pone.0000955.s011">Table S1</a>), followed by numerous rearrangements. Levels of gene order conservation for the <em>D. geothermalis</em> and <em>D. radiodurans</em> chromosomes and megaplasmids were determined by genomic dot plots <a href="#pone.0000955-Eisen1">[28]</a> (<a href="#pone.0000955.s002">Figure S2</a>). The dot plots of the chromosomes showed a clear pattern characteristic of chromosomes of relatively closely related bacteria that retain significant colinearity of the gene order. The X-shape pattern is thought to arise from inversions of a chromosomal segment around the origin of replication <a href="#pone.0000955-Eisen1">[28]</a>. By contrast, DR412 and DR177 did not display any discernable colinearity (<a href="#pone.0000955.s002">Figure S2B</a>), indicating substantially greater levels of rearrangement in the megaplasmids.</p>

<h5>Repeats and Prophages.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>Dozens of small noncoding repeats (SNRs) of an unusual, mosaic structure have been identified in the <em>D. radiodurans</em> genome, suggesting a possible role in resistance <a href="#pone.0000955-Makarova2">[29]</a>. In stark contrast, no mosaic-type SNRs were found in the <em>D. geothermalis</em> genome (<a href="#pone-0000955-t001">Table 1</a>), suggesting that SNRs are not involved in recovery from radiation or desiccation <a href="#pone.0000955-Lin1">[26]</a>, <a href="#pone.0000955-Makarova2">[29]</a>, <a href="#pone.0000955-Makarova3">[30]</a>. Further, there are about 20 DNA repeats in <em>D. radiodurans</em> that contain oligoG stretches (<a href="#pone.0000955.s003">Figure S3</a>). Such DNA sequences might adopt an ordered helical structure (G-quadruplex), predicted to form parallel four-stranded complexes capable of promoting chromosomal alignment <a href="#pone.0000955-Sen1">[31]</a>. However, the absence of such oligoG stretches in the G-rich sequence of <em>D. geothermalis</em> (G+C content, 66%) indicates that G-quartets are not essential for resistance. In contrast, the <em>D. geothermalis</em> genome contains CRISPR repeats <a href="#pone.0000955-Mojica1">[32]</a>, whereas <em>D. radiodurans</em> does not (<a href="#pone-0000955-t001">Table 1</a>). CRISPR repeats are part of a predicted RNA-interference-based system implicated in immunity to phages and integrative plasmids <a href="#pone.0000955-Barrangou1">[33]</a>, <a href="#pone.0000955-Makarova4">[34]</a>. Since no homologous prophages were identified in the two <em>deinococci</em>, and no CRISPR repeats are present in <em>D. radiodurans</em>, these sequences apparently have no role in determining levels of resistance either.</p>
<a id="article1.body1.sec2.sec3.sec2.p2" name="article1.body1.sec2.sec3.sec2.p2"></a><p>The 206 kb <em>D. geothermalis</em> megaplasmid (DG206), predicted by genome sequencing, is in lower copy-number than DG574 (<a href="#pone-0000955-g001">Figure 1C</a>). The presence of DG206 in genomic DNA preparations was confirmed in <em>D. geothermalis</em> (DSM 11300) DNA samples used for sequencing and from independent preparations by polymerase chain reaction (PCR) using DG206-specific primers that yielded DNA products of the predicted sizes (<a href="#pone.0000955.s004">Figure S4</a>). DG206 contains 205 predicted open reading frames (ORFs), of which 103 have significant similarity to genes in current databases; approximately 40 are identical to genes in either the <em>D. geothermalis</em> chromosome or DG574; and 28 have homologs in <em>D. radiodurans</em>, including 3 ORFs encoding highly diverged single-stranded DNA-binding proteins. Among other sequences of interest in DG206 are 22 transposon-related ORFs; 11 ORFs related to phage proteins; and 5 ORFs related to conjugal plasmid replication systems. In summary, DG206 is enriched for phage-, integrative plasmid- or transposon-related ORFs, but encodes no known metabolic enzymes and very few replication or repair proteins. Thus, DR206 seems to mimic the trend seen for ORFs in the smallest plasmid (46 kb) of <em>D. radiodurans</em> <a href="#pone.0000955-White1">[16]</a>, <a href="#pone.0000955-Makarova1">[21]</a>, with no predicted genes implicated in resistance.</p>



<h4>The <em>Deinococcus</em>-<em>Thermus</em> Group: Gene-Gain and Gene-Loss</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Our previous analysis of the major events in the evolution of the <em>Deinococcus</em>-<em>Thermus</em> group was based on <em>D. radiodurans</em> (ATCC BAA-816) and <em>T. thermophilus</em> strain HB27 <a href="#pone.0000955-Omelchenko1">[27]</a>. The current study includes additional comparisons with <em>D. geothermalis</em> (DSM 11300) and a second strain of <em>T. thermophilus</em> (HB8). Based on the standard approach of COGs (clusters of orthologous groups of proteins) <a href="#pone.0000955-Tatusov1">[35]</a>, <a href="#pone.0000955-Tatusov2">[36]</a>, COGs for <em>Deinococcus</em> and <em>Thermus</em> (tdCOGs) were constructed (<a href="#pone.0000955.s012">Table S2</a>). The tdCOGs were used as a framework for the whole-genome comparisons and evolutionary reconstructions (<a href="#pone-0000955-g002">Figure 2</a>). Using a weighted parsimony method and distantly related bacteria as outgroups, the evolutionary reconstructions revealed significant and independent expansion of the repertoire of genes in the <em>Deinococcus</em> and <em>Thermus</em> lineages following their divergence from a common ancestor. The expansion appears to have occurred through both lineage-specific duplications and gene acquisition via horizontal gene transfer (HGT). The high level of protein family expansion (paralogy), and the larger complement of species-specific genes acquired principally by HGT, could account for the existence of 600–900 more genes in <em>Deinococcus</em> than <em>Thermus</em>.</p>
<div class="figure" id="pone-0000955-g002"><div class="img"><a name="pone-0000955-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Whole genome evolutionary reconstructions for <em>D. radiodurans, D. geothermalis, T. thermophilus</em> (HB8) and <em>T. thermophilus</em> (HB27).</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>For each internal node of tree (open boxes), the inferred number of tdCOGs is shown. For each tree branch the inferred number of tdCOGs lost (minus sign) and gained (plus sign) is shown. For the deep ancestor of the <em>Cyanobacteria, Actinobacteria</em> and <em>Deinococcus-Thermus</em> group (shaded box), the inferred number of COGs is shown. For the extant species, the number of tdCOGs, the number of proteins in tdCOGs (in parentheses), and the number of “free” (not assigned to tdCOGs) proteins (plus sign) are shown.</p>
<span>doi:10.1371/journal.pone.0000955.g002</span></div>

<h4>The Common Ancestor of the <em>Deinococcus</em> Lineage: Trends of Gene-Gain and Gene-Loss</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Our previous comparative analysis of <em>T. thermophilus</em> and <em>D. radiodurans</em> identified several evolutionary trends that correlate with the distinct phenotypes of these bacterial lineages <a href="#pone.0000955-Omelchenko1">[27]</a>. These trends were further refined through the analysis of the <em>D. geothermalis</em> sequence, and the unique features of the <em>Deinococcus</em> lineage were used to better define the pathways implicated in extreme radiation resistance (<a href="#pone.0000955.s012">Table S2</a>). One such trend in <em>Deinococcus</em>, in comparison to the inferred common ancestor of the <em>Deinococcus-Thermus</em> group, is the acquisition of a set of genes involved in transcriptional regulation and signal transduction. Examples of acquired transcriptional regulators include two proteins of the AsnC family, two proteins of the GntR family, and one protein of the IclR family. These families likely are involved in amino acid degradation and metabolism <a href="#pone.0000955-Yokoyama1">[37]</a>–<a href="#pone.0000955-MolinaHenares1">[39]</a>. Further, the <em>Deinococcus</em> lineage acquired at least six TetR and MerR family regulators dedicated to diverse stress response pathways <a href="#pone.0000955-Ramos1">[40]</a>, <a href="#pone.0000955-Hobman1">[41]</a>. Among the acquired signal transduction genes, the most notable examples are two-component regulators of the NarL family (four distinct tdCOGs) involved in the regulation of a variety of oxygen and nitrate-dependent pathways of <em>Escherichia coli</em> <a href="#pone.0000955-Bearson1">[42]</a>; and the presence of several diguanylate cyclase (GGDEF) domain-containing proteins supports an increased role of cyclic diGMP in <em>Deinococci</em>. A second evolutionary trend in <em>Deinococcus</em> is the acquisition of genes encoding proteins involved in nucleotide metabolism, in particular, degradation and salvage <a href="#pone.0000955-Sandrini1">[43]</a>–<a href="#pone.0000955-Knofel1">[45]</a>. For example, this group includes genes for xanthine dehydrogenase, urate oxidase, deoxynucleoside kinases, thymidine kinase, FlaR-like kinase, and two UshA family 5′-nucleotidases.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Other gene-gains in <em>Deinococcus</em> relative to <em>Thermus</em> include genes for enzymes of amino acid catabolism and the tricarboxylic acid (TCA) cycle (<a href="#pone.0000955.s012">Table S2</a>). Beyond the differences reported previously <a href="#pone.0000955-Brim1">[11]</a>, <a href="#pone.0000955-Ghosal1">[12]</a>, the new reconstructions indicate that several catabolic genes of <em>Deinococcus</em> were already present in the <em>Deinococcus-Thermus</em> common ancestor. Following their divergence, however, the <em>Thermus</em> lineage appears to have lost many of these systems, including all enzymes involved in histidine degradation. By contrast, the <em>Deinococcus</em> lineage not only retained a majority of the predicted ancestral catabolic functions, but acquired new pathways including ones involved in the degradation of tryptophan and lysine, and several peptidases (<a href="#pone.0000955.s012">Table S2</a>). A hallmark of the <em>Deinococcus</em> lineage is the presence of two predicted genes for malate synthase, an enzyme of the glyoxylate bypass which converts isocitrate into succinate and glyoxylate, allowing carbon that enters the TCA cycle to bypass the formation of α-ketoglutarate and succinyl-CoA <a href="#pone.0000955-Ghosal1">[12]</a>. It has been proposed that the strong upregulation of the glyoxylate bypass observed in <em>D. radiodurans</em> following irradiation facilitates recovery by limiting the production of metabolism-induced reactive oxygen species (ROS) <a href="#pone.0000955-Liu1">[46]</a>. Dgeo_2616/DRA0277 is the malate synthase ortholog present in the <em>Thermus</em> lineage, but the second predicted deinococcal malate synthetase (Dgeo_2611/DR1155) is unique and only distantly related to homologs in other bacteria. Although the two predicted deinococcal malate synthetases could have similar functions, the genomic context of Dgeo_2611/DR1155 indicates otherwise; Dgeo_2611/DR1155 are both located in a predicted operon with two cyclic amidases of unknown biochemical function.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>In a broader context, the present reconstruction indicates that many expanded families of paralogous genes in <em>D. radiodurans</em> proliferated before the emergence of the common ancestor of the <em>Deinococci</em>, but the expansions were not present in the ancestor of the <em>Deinococcus-Thermus</em> group (<a href="#pone-0000955-t002">Table 2</a>). Such <em>Deinococcus</em>-specific expanded families include the Yfit/DinB family of proteins, acetyltransferases of the GNAT family, Nudix hydrolases, α/β superfamily hydrolases, calcineurin family phosphoesterases, and others. Many of these expansions are for predicted hydrolases, phosphatases in particular, but their substrate specificities are either unknown or the affinity of known substrates is extremely low <a href="#pone.0000955-Galperin1">[47]</a>. It has been proposed, therefore, that the majority of these predicted enzymes perform cell-cleaning functions including degradation of damaged nucleic acids, proteins and lipids, and/or other stress-induced cytotoxins <a href="#pone.0000955-Galperin1">[47]</a>. The global proliferation of these enzymes in the <em>Deinococcus</em> lineage (<a href="#pone.0000955.s013">Table S3</a>) supports the acquisition of chemical stress-resistance determinants early in its evolution; and the independent proliferation of determinants within these deinococcal species (<em>e.g</em>., calcinurin phosphatses, <a href="#pone.0000955.s005">Figure S5</a>) might represent secondary adaptations to specific stress environments. In summary, these findings indicate that the <em>Deinococcus</em> stress-resistance phenotypes evolved continuously, both by lineage-specific gene duplications and by HGT from various sources (<a href="#pone.0000955.s013">Table S3</a>, <a href="#pone.0000955.s014">S4</a> and <a href="#pone.0000955.s015">S5</a>) <a href="#pone.0000955-Makarova1">[21]</a>.</p>
<div class="figure" id="pone-0000955-t002"><div class="img"><a name="pone-0000955-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Ancestral expansions: paralogous gene families expanded in the <em>Deinococcus</em> lineage (DD) versus the <em>Thermus</em> lineage (TT) ancestors</span></strong></p><span>doi:10.1371/journal.pone.0000955.t002</span></div>

<h4>Individual <em>Deinococcus</em> Species: Gene-Gain and Gene-loss</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The comparison of gene-gain and gene-loss events in the <em>D. radiodurans</em> and <em>D. geothermalis</em> lineages reveals numerous differences, many of which correlate with their distinct metabolic phenotypes (<a href="#pone-0000955-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0000955-g003"><div class="img"><a name="pone-0000955-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Gene-gain and gene-loss for different functional groups for <em>D. radiodurans</em> and <em>D. geothermalis.</em></span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Designations of functional groups (from the COG database): J–Translation, ribosomal structure and biogenesis; K–Transcription; L–DNA replication, recombination and repair; D–Cell division and chromosome partitioning; O–Posttranslational modification, protein turnover, chaperones; M–Cell envelope and outer membrane biogenesis; N–Cell motility and secretion; P–Inorganic ion transport and metabolism; T–Signal transduction mechanisms; C–Energy production and conversion; G–Carbohydrate transport and metabolism; E–Amino acid transport and metabolism; F–Nucleotide transport and metabolism; H–Coenzyme metabolism; I–Lipid metabolism; Q–Secondary metabolites biosynthesis, transport and catabolism; V–genes involved in stress response and microbial defense.</p>
<span>doi:10.1371/journal.pone.0000955.g003</span></div><h5>D. geothermalis.</h5><a id="article1.body1.sec2.sec6.sec1.p1" name="article1.body1.sec2.sec6.sec1.p1"></a><p>The most notable, distinctive feature of <em>D. geothermalis</em> is a greater abundance of genes for sugar metabolism enzymes, which could have been acquired after the divergence of the two <em>Deinococci</em>. The largest group within this set of genes is predicted to be involved in xylose utilization, needed for growth on plant material. D-xylose, which forms xylan polymers, is a major structural component of plant cell walls <a href="#pone.0000955-Shallom1">[48]</a>, and the presence of genes for aldopentose (xylose)-degradation explains why <em>D. geothermalis</em> is a persistent contaminant in paper mills <a href="#pone.0000955-Saarimaa1">[8]</a>. Specifically, <em>D. geothermalis</em> contains genes encoding xylanases (Dgeo_2723; Dgeo_2722), an ABC-type xylose transport system (Dgeo_2699-Dgeo_2703), xylose isomerase (Dgeo_2375, Dgeo_2692, Dgeo_2693, Dgeo_2826), and xylose kinase (Dgeo_2691). Several of the genes that encode enzymes of xylose metabolism form paralogous families (Table S4), most of which form a cluster on the megaplasmid DG574 (Dgeo_2703-Dgeo_2687), which also contains two gene clusters predicted to be involved in carbohydrate utilization (Dgeo_2669-Dgeo_2693, Dgeo_2832-Dgeo_2812). By comparison, there are no large clusters of functionally related genes on the <em>D. geothermalis</em> chromosome; approximately 80 and 120 encoding proteins involved in sugar-metabolism were identified on DG574 and the chromosome, respectively. The putative xylose metabolism functions of <em>D. geothermalis</em> appear to represent an expansion of a pre-existing, broad and diverse set of functions underlying the saccharolytic phenotypes of all <em>Deinococci</em> <a href="#pone.0000955-Ferreira1">[7]</a>, <a href="#pone.0000955-Brim1">[11]</a>, <a href="#pone.0000955-Kolari1">[49]</a>, <a href="#pone.0000955-Vaisanen1">[50]</a>. In contrast, <em>D. radiodurans</em> has a proteolytic lifestyle, where a loss of various amino acid biosynthetic pathways (<a href="#pone-0000955-g003">Figure 3</a>) <a href="#pone.0000955-Venkateswaran1">[51]</a> was accompanied by a gain of several predicted peptidases (DR0964, DR1070, DR2310, DR2503) and a urease system (DRA0311-DRA0319) <a href="#pone.0000955-Omelchenko1">[27]</a>. Thus, the evolutionary processes underlying the emergence of extreme resistance in <em>Deinococci</em> appear not to be dependent on a particular set of genes for sugar- or nitrogen-metabolism. In summary, these findings support that DG574 is essential to the natural growth modes of <em>D. geothermalis</em>, which is a proficient saccharolytic organism <a href="#pone.0000955-Ferreira1">[7]</a>, <a href="#pone.0000955-Kolari1">[49]</a>, <a href="#pone.0000955-Vaisanen1">[50]</a> and strengthen the case that the megaplasmids in the <em>Deinococcus-Thermus</em> group are major receptacles of horizontally acquired genes, as proposed previously <a href="#pone.0000955-Omelchenko1">[27]</a>.</p>
<a id="article1.body1.sec2.sec6.sec1.p2" name="article1.body1.sec2.sec6.sec1.p2"></a><p>Further supporting the notion that a distinct set of metabolic genes is not a prerequisite for high levels of radioresistance, there are patent differences between sulfate and energy metabolism in <em>D. geothermalis</em> and <em>D. radiodurans</em>. In agreement with previously published results <a href="#pone.0000955-Ferreira1">[7]</a>, <a href="#pone.0000955-Brim1">[11]</a>, <a href="#pone.0000955-Venkateswaran1">[51]</a>, the prototrophic <em>D. geothermalis</em> has orthologs of the <em>nadABCD</em> genes that are required for nicotinamide adenine dinucleotide (NAD) biosynthesis, whereas the auxotrophic <em>D. radiodurans</em> lacks these genes and is dependent on an exogenous source of this coenzyme <a href="#pone.0000955-Makarova1">[21]</a>, <a href="#pone.0000955-Venkateswaran1">[51]</a>. Another example illustrating the relationship in <em>D. radiodurans</em> between gene-loss and its growth requirements is that of cobalamine (vitamin B12). Whereas <em>D. geothermalis</em> and <em>T. thermophilus</em> are not dependent on B12 in minimal medium, <em>D. radiodurans</em> can utilize inorganic sulfate as the sole source of sulfur only when vitamin B12 is present <a href="#pone.0000955-Holland1">[52]</a>. Conversely, <em>D. geothermalis</em> has lost several genes for enzymes of protoheme biosynthesis (HemEZY) <a href="#pone.0000955-Frankenberg1">[53]</a>, which in <em>D. geothermalis</em> likely yields siroheme under the microaerophilic conditions which predominate at the T<sub>opt</sub> of <em>D. geothermalis</em>; the solubility of dioxygen in water at 50°C is significantly lower than at 32°C, the T<sub>opt</sub> of <em>D. radiodurans</em>.</p>
<a id="article1.body1.sec2.sec6.sec1.p3" name="article1.body1.sec2.sec6.sec1.p3"></a><p>There are also important differences between the systems for enzymes implicated in energy transformation in <em>D. geothermalis</em> and <em>D. radiodurans</em>. The <em>D. geothermalis</em> chromosome encodes two heme-copper cytochrome oxidases of types ba3 and caa3 <a href="#pone.0000955-Pereira1">[54]</a>; and a cytochrome bd ubiquinol oxidase system (Dgeo_2707-Dgeo_2704), known to be expressed under oxygen-limiting conditions <a href="#pone.0000955-Junemann1">[55]</a>, is encoded by DG574. In contrast, <em>D. radiodurans</em> encodes only the caa3 oxidase system (DR2616-DR2620), which apparently was present in the <em>Deinococcus</em>-<em>Thermus</em> common ancestor. Furthermore, <em>D. geothermalis</em> encodes genes for proteins that comprise an assimilatory nitrite NAD(P)H reductase and a molybdopterin-cofactor-dependent nitrate reductase system (Dgeo2392-Dgeo_2389), which also is known to be expressed under anaerobic conditions <a href="#pone.0000955-Harborne1">[56]</a>, <a href="#pone.0000955-Cabello1">[57]</a>; and <em>D. geothermalis</em> encodes several predicted multi-copper oxidases (Dgeo_2590, Dgeo_2559, Dgeo_2558) that are not present in <em>D. radiodurans</em> and are most similar to homologs from nitrogen-fixing bacteria. Since nitrogen fixation in <em>D. geothermalis</em> has not yet been studied, the possibility remains that these enzymes are involved in dissimilatory anaerobic reduction of nitrate or nitrite <a href="#pone.0000955-Zumft1">[58]</a>, <a href="#pone.0000955-Nakamura1">[59]</a>. <em>D. geothermalis</em>, but not <em>D. radiodurans</em>, also encodes a formate dehydrogenase, which is related to nitrate reductase and has a possible role in energy transfer under anaerobic conditions <a href="#pone.0000955-Hille1">[60]</a>.</p>

<h5>D. radiodurans.</h5><a id="article1.body1.sec2.sec6.sec2.p1" name="article1.body1.sec2.sec6.sec2.p1"></a><p>In general, the evolutionary trends in <em>D. radiodurans</em> lineage appear to mimic closely those of the <em>Deinococcus</em> lineage, which is evident from the analysis of expanded families of paralogous genes (<a href="#pone.0000955.s015">Table S5</a>). In particular, proliferation of genes for the Yfit/DinB family, Nudix enzymes, acetyltransferases of the GNAT superfamily, and the α/β hydrolase superfamily was observed (<a href="#pone-0000955-t002">Table 2</a>). Plausible resistance-related functions readily can be proposed for these and other expanded families of deinococci. For example, hydrolases might degrade oxidized lipids; Yfit/DinB proteins might be involved in cell damage-related pathways <a href="#pone.0000955-Makarova1">[21]</a>; subtilisin-like proteases might degrade proteins oxidized during irradiation <a href="#pone.0000955-Daly3">[17]</a>, <a href="#pone.0000955-Makarova5">[61]</a>; and the Nudix-related hydrolase, diadenosine polyphosphatase (ApnA), yields adenosine, a molecule that has been implicated in cytoprotection from oxidative stress and radiation <a href="#pone.0000955-Fisher1">[62]</a>, <a href="#pone.0000955-Hou1">[63]</a>.</p>
<a id="article1.body1.sec2.sec6.sec2.p2" name="article1.body1.sec2.sec6.sec2.p2"></a><p>Several families expanded in <em>D. radiodurans</em> are predicted to possess functions potentially relevant to stress response, but are not conserved in <em>D. geothermalis</em>; most likely, non-conserved families can be disqualified as major contributors to the extreme IR and desiccation resistance phenotypes. Families that are specifically expanded in <em>D. radiodurans</em> include the TerZ family of proteins, which are predicted to confer resistance to various DNA damaging agents <a href="#pone.0000955-Azeddoug1">[64]</a>, <a href="#pone.0000955-Jobling1">[65]</a>; secreted proteins of the PR1 family, whose homologs are involved in the response to pathogens in plants, and resistance to hydrophilic organic solvents in yeast <a href="#pone.0000955-Kitajima1">[66]</a>, <a href="#pone.0000955-Miura1">[67]</a>; PadR-like regulators, which are implicated in the regulation of amino acid catabolism and cellular response to chemical stress agents and drugs <a href="#pone.0000955-Brinkrolf1">[68]</a>–<a href="#pone.0000955-Huillet1">[70]</a>; TetR/AcrR transcriptional regulators, which are involved in antibiotic resistance regulation <a href="#pone.0000955-Ramos1">[40]</a>; and KatE-like catalases, which would decompose hydrogen peroxide <a href="#pone.0000955-Kobayashi1">[71]</a>–<a href="#pone.0000955-Imlay1">[73]</a>. In contrast, there are family expansions which are shared by <em>D. radiodurans</em> and <em>D. geothermalis</em>, but have no obvious role in radiation or desiccation resistance. These include SAM-dependent metyltransferases (COG0500) and an uncharacterized family of predicted P-loop kinases (COG0645). In some bacteria, homologs of these kinases are fused to phosphotransferases that mediate resistance to aminoglycosides <a href="#pone.0000955-Nurizzo1">[74]</a>.</p>
<a id="article1.body1.sec2.sec6.sec2.p3" name="article1.body1.sec2.sec6.sec2.p3"></a><p>Since the IR-, UV- and desiccation-resistance profiles of <em>D. radiodurans</em> and <em>D. geothermalis</em> are identical (<a href="#pone-0000955-g001">Figure 1</a>) <a href="#pone.0000955-Daly1">[5]</a>, the subset of stress response genes in <em>D. radiodurans</em> that are not unique, but exist in excess compared to <em>D. geothermalis</em> are unlikely to be required for extreme resistance either (<a href="#pone-0000955-g003">Figure 3</a>). This subset includes two Cu-Zn superoxide dismutases (SOD), a peroxidase, two HslJ-like heat shock proteins, and many genes implicated in antibiotic resistance (<a href="#pone.0000955.s015">Table S5</a>). Consistently, SodA and KatA of <em>D. radiodurans</em> can be disrupted with almost no loss in radiation resistance <a href="#pone.0000955-Markillie1">[75]</a>, and antibiotics have little effect on survival following irradiation provided corresponding antibiotic resistance genes are present <a href="#pone.0000955-Daly4">[18]</a>, <a href="#pone.0000955-Daly6">[76]</a>–<a href="#pone.0000955-Daly9">[79]</a>.</p>

<h5>The Deinococcus lineage.</h5><a id="article1.body1.sec2.sec6.sec3.p1" name="article1.body1.sec2.sec6.sec3.p1"></a><p>Considerable independent gene-gain was detected in both <em>D. geothermalis</em> and <em>D. radiodurans</em> lineages in several other functional categories including transcriptional regulation, signal transduction, membrane biogenesis, inorganic ions metabolism, and to a lesser extent DNA replication and repair (<a href="#pone-0000955-g003">Figure 3</a>). In general, regulatory functions mirror the metabolic and stress-response-related differentiation of these two species outlined above. For instance, among the 12 genes for predicted transcriptional regulators that apparently were acquired in the <em>D. geothermalis</em> lineage, five are similar to ones known to be involved in the regulation of sugar metabolism in other bacteria, two of the RpiR family and three of the AraC family <a href="#pone.0000955-Gallegos1">[80]</a>, <a href="#pone.0000955-Sorensen1">[81]</a>. By contrast, <em>D. radiodurans</em> has at least 25 unique genes for transcriptional regulators: three of the ArcR family; 16 of the Xre family; one of the CopG/Arc/MetJ family; and five of a species-specific expanded family reported previously <a href="#pone.0000955-Makarova5">[61]</a> that likely is responsible for stress-response control <a href="#pone.0000955-Labie1">[82]</a>-<a href="#pone.0000955-Gerdes1">[85]</a>. Other potentially independent gains involve genes predicted to be involved in signal transduction systems. <em>D. radiodurans</em>, for example, encodes photochromic histidine kinase, a protein that has been extensively studied in <em>D. radiodurans</em> and plays a role in the regulation of pigment biosynthesis <a href="#pone.0000955-Bhoo1">[86]</a>, <a href="#pone.0000955-Davis1">[87]</a>, but is missing in <em>D. geothermalis</em>. Alternatively, <em>D. geothermalis</em> encodes a putative negative regulator of sigma E, a periplasmic protein of the RseE/MucE family (Dgeo_2271). So far, RseE/MucE-members have been detected only in proteobacteria, where it regulates the synthesis of alginate, an extracellular polysaccharide which plays a key role in the formation of biofilms <a href="#pone.0000955-Boyd1">[88]</a>. <em>D. geothermalis</em>, however, likely does not produce alginate itself since it has no orthologs of the genes of the alignate pathway <a href="#pone.0000955-Gacesa1">[89]</a>. On the other hand, <em>D. geothermalis</em> has clusters of genes implicated in exopolysaccharide biosynthesis, with the most notable cluster located on DG574 (Dgeo_2671-Dgeo_2646). It seems likely that this cluster is involved in the biosynthesis of exopolysaccharides, which might facilitate biofilm formation in <em>D. geothermalis</em>, and the Dgeo_2271 protein could be a regulator of this process. Overall, <em>D. radiodurans</em> encodes approximately 470 unique, uncharacterized proteins, for which no function could be predicted, compared to approximately 286 such proteins in <em>D. geothermalis</em>. Thus, an additional 756 unique, uncharacterized genes of the <em>Deinococcus</em> lineage can be excluded from the pool of putative determinants of the extreme IR, UV and desiccation resistance phenotype.</p>



<h4>Reassessment of the Genetic Determinants of Radiation Resistance</h4>
<h5>Evolutionary Provenance of the Genomic Features Previously Implicated in the Radiation Resistance of D. radiodurans.</h5><a id="article1.body1.sec2.sec7.sec1.p1" name="article1.body1.sec2.sec7.sec1.p1"></a><p>Over the last two decades, extensive experimental and comparative-genomic analyses have been dedicated to the identification and evolutionary origin of the genetic determinants of radiation resistance in <em>D. radiodurans</em>. Early on, it became evident that the survival mechanisms underlying extreme radiation resistance in <em>D. radiodurans</em> probably were not unique. In 1994, for example, IR-sensitive <em>D. radiodurans pol</em>A mutants were fully complemented by expression of the <em>polA</em> gene from the IR-sensitive <em>E. coli</em> <a href="#pone.0000955-Gutman1">[4]</a>; and in 1996, UV-sensitve <em>D. radiodurans uvrA</em> mutants were complemented by <em>uvrA</em> from <em>E. coli</em> <a href="#pone.0000955-Agostini1">[90]</a>, suggesting that these recombination and excision repair genes are necessary but not sufficient to produce extreme DNA damage resistance. Following the whole-genome sequencing of <em>D. radiodurans</em> in 1999 <a href="#pone.0000955-White1">[16]</a>, comparative-genomic analysis revealed many distinctive genomic features that subsequently became the focus of high throughput experiments, including the analysis of transcriptome and proteome dynamics of <em>D. radiodurans</em> recovering from IR <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Lipton1">[91]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>. Surprisingly, the cellular transcriptional response to IR in <em>D. radiodurans</em> appeared largely stochastic, and mutant analyses confirmed that many of the highly induced uncharacterized genes were unrelated to survival. So far, those correlative studies have failed to produce a coherent, comprehensive picture of the complex interactions between different genes and systems that have been thought to be important for the resistance phenotype.</p>
<a id="article1.body1.sec2.sec7.sec1.p2" name="article1.body1.sec2.sec7.sec1.p2"></a><p>The complete sets of orthologous genes in <em>D. radiodurans</em> and <em>D. geothermalis</em> are listed in <a href="#pone.0000955.s012">Table S2</a>. Within the subgroup of genes in <em>D. radiodurans</em> previously implicated in resistance by transcriptional induction following exposure to IR <a href="#pone.0000955-Liu1">[46]</a> (3 hours after irradiation and displaying more than a 2-fold induction), 45% have no othologs in <em>D. geothermalis</em>. This raises the possibility that many genes induced in irradiated <em>D. radiodurans</em> do not functionally participate in recovery, or that <em>D. geothermalis</em> carries a distinct set of resistance determinants. From the subgroup of putative resistance genes lacking counterparts in <em>D. geothermalis</em>, we constructed <em>D. radiodurans</em> knockouts of four representative genes: i) a ligase predicted to be involved in DNA repair (DRB0100) <a href="#pone.0000955-Liu1">[46]</a>; ii) a LEA76 desiccation resistance protein homolog (DR0105) <a href="#pone.0000955-Liu1">[46]</a>; iii) a predicted protein implicated in stress response (DR2221) <a href="#pone.0000955-Liu1">[46]</a>; and iv) a protein of unknown function (DR0140) <a href="#pone.0000955-Liu1">[46]</a>. Homozygous disruptions of each of these genes in <em>D. radiodurans</em> (<a href="#pone.0000955.s006">Figure S6</a>) had no significant effect on IR resistance (<a href="#pone-0000955-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0000955-g004"><div class="img"><a name="pone-0000955-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>IR resistance of wild-type (ATCC BAA-816) and <em>D. radiodurans</em> mutants lacking orthologs in <em>D. geothermalis</em> (DSM 11300).</span></strong></p><a id="article1.body1.sec2.sec7.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec7.sec1.fig1.caption1.p1"></a><p>Survival values following 9 kGy (<sup>60</sup>Co) are from three independent trials with standard deviations shown. The structure of the homozygous mutants DRB0100, DR2221, DR105 and DR0140 are presented in <a href="#pone.0000955.s006">Figure S6</a>.</p>
<span>doi:10.1371/journal.pone.0000955.g004</span></div><a id="article1.body1.sec2.sec7.sec1.p3" name="article1.body1.sec2.sec7.sec1.p3"></a><p>By contrast, most of the genes whose mutants display radiation-sensitive phenotypes in <em>D. radiodurans</em> <a href="#pone.0000955-Gutman1">[4]</a>, <a href="#pone.0000955-Cox1">[20]</a>, <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>, <a href="#pone.0000955-Gutman2">[93]</a> are conserved in <em>D. geothermalis</em>. To date, 15 single-gene mutants of <em>D. radiodurans</em> have been reported to be moderately to highly radiation-sensitive; of these, 13 genes have orthologs in <em>D. geothermalis</em> (<a href="#pone-0000955-t003">Table 3</a>). The exceptions are DR0171 and DR1289, which encode the DNA helicase RecQ and a transcriptional regulator, respectively (<a href="#pone-0000955-t003">Table 3</a>). Remarkably, 10 of the 15 genes are conserved in other bacteria and are well-characterized components of DNA repair pathways. However, 5 of the 15 genes (DR0003, DR0070, DR0326, DR0423, DRA0346) are unique to the <em>Deinococcus</em> lineage, supporting the existence of at least a few novel resistance mechanisms.</p>
<div class="figure" id="pone-0000955-t003"><div class="img"><a name="pone-0000955-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span><em>D. radiodurans</em> genes implicated in radiation resistance</span></strong></p><span>doi:10.1371/journal.pone.0000955.t003</span></div><a id="article1.body1.sec2.sec7.sec1.p4" name="article1.body1.sec2.sec7.sec1.p4"></a><p>Given that the two <em>Deinococcus</em> species are equally resistant to IR (<a href="#pone-0000955-g001">Figure 1A</a>), genes dedicated specifically to the extreme radiation/desiccation response are expected to belong to the set of tdCOGs. <em>D. radiodurans</em> and <em>D. geothermalis</em> share 231 tdCOGs that are relatively uncommon in other prokaryotes, and 63 of these are unique to the <em>Deinococcus</em> lineage. Using the most sensitive methods available to predict function, we reanalyzed these tdCOGs by using a remote sequence similarity search, and genomic context analysis <a href="#pone.0000955-Altschul1">[94]</a>–<a href="#pone.0000955-MarchlerBauer1">[96]</a>. Interpretation of such analyses, however, is constrained by the complexity and ambiguities inherent in the approach, and by the knowledge base. In contrast, many cytosolic proteins (<em>e.g</em>., RecA, PolA, SodA and KatA) are known to be intimately involved in resistance, so we present functional predictions for 50 genes (<a href="#pone.0000955.s016">Table S6</a>). Among the predictions for cytosolic proteins, several are new and potentially relevant to resistance. For example, DR0644 (<a href="#pone-0000955-g005">Figure 5A</a>) is predicted to be a distinct Cu/Zn superoxide dismutase that could defend against metabolism-induced oxidative stress during recovery (<a href="#pone.0000955.s017">Table S7</a>); and DR0449 (<a href="#pone-0000955-g005">Figure 5B</a>) is a divergent member of the RNAse H family that is fused to a novel domain, a combination that is currently unique to <em>Deinococcus</em>. Other functional insights were for DR0041/Dgeo_0188, that is a paralog of DR0432 (DdrA) (<a href="#pone-0000955-g005">Figure 5C</a>); and a member of the RAD22/Rad52 family (<a href="#pone-0000955-g005">Figure 5C</a>) of single-stranded annealing proteins <a href="#pone.0000955-Iyer1">[97]</a>, that yields a moderately sensitive phenotype in <em>D. radiodurans</em> upon disruption <a href="#pone.0000955-Harris1">[98]</a>. Interestingly, the radiation-sensitive <em>T. thermophilus</em> encodes a homolog of DdrA (TTC1923), indicating that this protein had an ancestral role that was not directly related to radiation resistance. Notably, we continue to find proteins in <em>Deinococcus</em> species which are only remotely similar to well-characterized enzymes in other organisms, and it is difficult to predict their role in the cell or radiation resistance. For example, we have identified a protein that is conserved in both <em>D. geothermalis</em> and <em>D. radiodurans</em> and is distantly related to enzymes of the QueF/FolE family, which are involved in queuosine/folate biosynthesis (<a href="#pone-0000955-g005">Figure 5D</a>), but their role in the <em>Deinococci</em> remains undefined. Collectively, these results support the conclusion that many genes that are significantly induced in irradiated <em>D. radiodurans</em> are not involved in recovery (<a href="#pone-0000955-t003">Table 3</a>). Thus, the genome of <em>D. geothermalis</em> is a resource of major importance in delineating a reliable minimal set of resistance determinants, by corroborating those that are conserved and ruling out those which are unique.</p>
<div class="figure" id="pone-0000955-g005"><div class="img"><a name="pone-0000955-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Multiple alignments of selected families conserved in two <em>Deinococcus</em> species.</span></strong></p><a id="article1.body1.sec2.sec7.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec7.sec1.fig2.caption1.p1"></a><p>The multiple alignments were constructed for selected representative sets of sequences by the MUSCLE program <a href="#pone.0000955-Edgar1">[154]</a>. Where necessary, alignments were modified manually on the basis of PSI-BLAST outputs <a href="#pone.0000955-Altschul1">[94]</a>. The positions of the first and the last residue of the aligned region in the corresponding protein are indicated for each sequence. The numbers within the alignment refer to the length of inserts that are poorly conserved between all the families. Secondary structure elements are denoted as follows: E-β-strand; and H-α-helix. The coloring scheme is as follows: predominantly hydrophobic residues are high-lighted in yellow; positions with small residues are in green; positions with turn-promoting residues are in cyan; positions with polar residues are in red; hydroxyl-group containing residues are in blue; aromatic residues are in magenta; and invariant, highly conserved groups are in boldface. A, DR0644-Dgeo_0284 conserved pair of orthologs belong to the copper/Zinc superoxide dismutase family; shaded letters refer to amino acids that play an important role in the Cu<sup>2+</sup>/Zn<sup>2+</sup> coordination environment and in the active site region. The bottom line shows the correspondence between the most conserved regions corresponding to the β-stand structural core and conserved in most family members as denoted in Bordo et al <a href="#pone.0000955-Bordo1">[157]</a>. B, Dgeo_0137-DR0449 are highly specific for the <em>Deinococcus</em> lineage proteins that have an RNAse H-related domain. Catalytic residues conserved in the RNAse H family are shaded. Secondary structure elements are shown for <em>E. coli</em> RNase HI (PDB:2rn2). C, DR0041-Dgeo_0188 is another conserved pair (DdrA-related) of proteins belonging to the Rad52 family of DNA single-strand annealing proteins <a href="#pone.0000955-Iyer1">[97]</a>. Secondary structure elements are shown for human RAD52 (PDB:1KN0) <a href="#pone.0000955-Singleton1">[158]</a>. <em>sak</em> is a phage gene described previously <a href="#pone.0000955-Bouchard1">[159]</a>; D, DR0381-Dgeo_0373 are diverged homologs of NADPH-dependent nitrile reductase (GTP cyclohydrolase I family) that might be involved in nucleotide metabolism. The conserved Cys and Glu found in the substrate binding pocket of both protein families and specific zinc-binding and catalytic residues in the FolE family are shaded. The QueF family motif is boxed. Other catalytic residues in FolE not found in QueF are in yellow. Genbank Identifier (gi) numbers are listed on the right.</p>
<span>doi:10.1371/journal.pone.0000955.g005</span></div>
<h5>Delineation of the Deinococcus Radiation Response Regulon.</h5><a id="article1.body1.sec2.sec7.sec2.p1" name="article1.body1.sec2.sec7.sec2.p1"></a><p>A potential radiation-desiccation response regulon and the corresponding regulator common to <em>D. radiodurans</em> and <em>D. geothermalis</em> were identified using the approach developed by Mironov et al <a href="#pone.0000955-Mironov1">[99]</a>, <a href="#pone.0000955-Mironov2">[100]</a>. In the search for such a regulator, we used a training-set comprised of sequences flanking <em>D. radiodurans</em> genes that were strongly upregulated by IR, and for which the corresponding mutants were radiosensitive (<a href="#pone-0000955-t003">Table 3</a>) <a href="#pone.0000955-Tanaka1">[92]</a>. The upstream regions of several genes from the training set (DR0326, <em>ddrD</em>; DR0423, <em>ddrA</em>; DRA0346, <em>pprA</em>; DR0070, <em>ddrB</em>) revealed a strong palindromic motif, designated the <span class="underline">r</span>adiation/<span class="underline">d</span>esiccation <span class="underline">r</span>esponse <span class="underline">m</span>otif (RDRM). Using a positional weight matrix, the RDRM was used to generate the initial profile and to scan the entire <em>D. radiodurans</em> genome. This genome survey picked up a similar motif in the upstream regions of other genes upregulated after irradiation <a href="#pone.0000955-Tanaka1">[92]</a>. The upstream regions with the highest scores (DR0219, DR0906, DR1913 and DR0659) were then used to better define the RDRM, and the complete genomes of <em>D. radiodurans</em> and <em>D. geothermalis</em> were scanned with the updated motif. Using moderately relaxed parameters (Materials and Methods), approximately 120 genes in each of the <em>Deinococcus</em> genomes were selected by the screen. The final, most conservative prediction of the <span class="underline">r</span>adiation/<span class="underline">d</span>esiccation <span class="underline">r</span>esponse (RDR) regulon consisted of two groups: (i) a set of orthologous genes present in both <em>Deinococcus</em> species that contain the RDRM; and (ii) a set of unique genes of <em>D. radiodurans</em> that contain the RDRM and are upregulated during the recovery from irradiation <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>. Since microarray data for <em>D. geothermalis</em> are not available, it was not possible to predict a set of unique RDRM-dependent genes for this species. <a href="#pone-0000955-t004">Table 4</a> lists the set of genes predicted to comprise the regulon together with the corresponding RDRM sites (<a href="#pone-0000955-g006">Figure 6</a>). Collectively, the RDR regulon is predicted to consist of a minimum of 29 genes in <em>D. radiodurans</em> and 25 genes in <em>D. geothermalis</em>, contained within 20 operons in each species.</p>
<div class="figure" id="pone-0000955-g006"><div class="img"><a name="pone-0000955-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Sequence signature of a predicted site of a radiation response regulator.</span></strong></p><a id="article1.body1.sec2.sec7.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec7.sec2.fig1.caption1.p1"></a><p>Four different nucleotides are shown by four letters (A, G, C, T) in different colors. The height of the letter is proportional to its contribution to the information content in the corresponding position of the multiple alignment used for “sequence logo” construction. The figure was constructed by the “sequence logo” program described previously <a href="#pone.0000955-Schneider1">[160]</a>.</p>
<span>doi:10.1371/journal.pone.0000955.g006</span></div><div class="figure" id="pone-0000955-t004"><div class="img"><a name="pone-0000955-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The predicted radiation and desiccation resistance regulon of <em>Deinococci</em></span></strong></p><span>doi:10.1371/journal.pone.0000955.t004</span></div><a id="article1.body1.sec2.sec7.sec2.p2" name="article1.body1.sec2.sec7.sec2.p2"></a><p>The RDR regulon is dominated by DNA repair genes, including the recombinational repair proteins RecA and RecQ <a href="#pone.0000955-Kunkel1">[101]</a>, <a href="#pone.0000955-Kuzminov1">[102]</a>; the mismatch repair proteins MutS and MutL, that are located in one operon in <em>D. geothermalis</em>; and the UvrB and UvrC proteins, which are involved in nucleotide excision repair (<a href="#pone-0000955-t004">Table 4</a>). In addition, the predicted RDR regulon includes the transketolase gene. In bacteria, transketolase is a key enzyme of the pentose-phosphate pathway for carbohydrate metabolism and is known to be induced by a variety of stress conditions including cold shock, and mutagens that trigger the SOS response <a href="#pone.0000955-Touati1">[103]</a>. Moreover, the pentose-phosphate pathway in <em>D. radiodurans</em> is reported to facilitate DNA excision repair induced by UV irradiation and hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) <a href="#pone.0000955-Zhang1">[104]</a>. The RDRM also precedes a conserved histidine catabolism operon <a href="#pone.0000955-Chasin1">[105]</a>. Several bacterial biodegradative and related operons are known to be differentially induced in response to a decline in biosynthetic and energy-generating activities under oxidative stress <a href="#pone.0000955-Kannan1">[106]</a>. For example, the TCA cycle in <em>D. radiodurans</em> is strongly down-regulated following irradiation <a href="#pone.0000955-Liu1">[46]</a>, whereas the glyoxylate bypass of the TCA cycle, and the His operon are induced <a href="#pone.0000955-Liu1">[46]</a>. Several studies have provided direct evidence that survival of <em>D. radiodurans</em> following exposure to IR depends on a coordinated metabolic response and a high level of respiratory control <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Bruce1">[107]</a>.</p>
<a id="article1.body1.sec2.sec7.sec2.p3" name="article1.body1.sec2.sec7.sec2.p3"></a><p>The regulation of gene expression in <em>D. radiodurans</em> during recovery from IR has been the subject of considerable investigation. Recently, it has been shown that the induction of <em>recA</em> in irradiated <em>D. radiodurans</em> is regulated by the IrrE/PprI protein <a href="#pone.0000955-Earl1">[108]</a>, <a href="#pone.0000955-Hua1">[109]</a>, which consists of two domains, a Xre-like HTH domain and a Zn-dependent protease. In both <em>D. radiodurans</em> and <em>D. geothermalis</em>, the <em>irrE</em> gene is located upstream of the folate biosynthesis operon, but appears to be regulated independently <a href="#pone.0000955-Gao1">[110]</a>. Since <em>recA</em> in <em>D. radiodurans</em> is strongly induced following irradiation <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Carroll1">[111]</a>, it was surprising that the <em>irrE</em> gene of <em>D. radiodurans</em> was constitutively expressed, showing no post-irradiation induction <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>, <a href="#pone.0000955-Gao1">[110]</a>. Furthermore, the IrrE/PprI protein has an unusual domain structure and does not appear to bind the promoter region of <em>recA</em> or other induced genes <a href="#pone.0000955-Gao1">[110]</a>.</p>
<a id="article1.body1.sec2.sec7.sec2.p4" name="article1.body1.sec2.sec7.sec2.p4"></a><p>Compared to radiosensitive bacteria, the regulatory mechanisms underlying the response to radiation in <em>D. radiodurans</em> are still poorly characterized. For example, the LexA-regulated SOS-dependent radiation response regulon of <em>E. coli</em> is well-defined <a href="#pone.0000955-Touati1">[103]</a>, <a href="#pone.0000955-Cheo1">[112]</a>–<a href="#pone.0000955-FernandezDeHenestrosa1">[115]</a>, but an equivalent system in <em>D. radiodurans</em> has not been identified. <em>D. geothermalis</em> has one <em>lexA</em> gene (DG1366) and <em>D. radiodurans</em> has two <em>lexA</em> paralogs (DRA0344, DRA0074). However, the <em>lexA</em> genes in <em>D. radiodurans</em> are not induced after irradiation, are not involved in RecA induction <a href="#pone.0000955-Narumi1">[116]</a>, and are not preceded by RDRM sites <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>. Therefore, LexA is not a candidate for the role of the regulator of the <em>Deinococcus</em> RDR regulon. In the microarray experiments of Liu et al, several putative regulators were upregulated in <em>D. radiodurans</em> following exposure to 15,000 Gy <a href="#pone.0000955-Liu1">[46]</a>. In contrast, at lower doses (3,000 Gy), the <em>D. radiodurans</em> microarray experiments of Tanaka et al detected only one upregulated putative regulator (DdrO) (DR2574) <a href="#pone.0000955-Tanaka1">[92]</a>. An orthologous gene for DdrO is present in <em>D. geothermalis</em> (Dgeo_0336). DdrO is a Xre family protein and is the only <em>Deinococcus</em> gene for a predicted regulator that is preceded by a RDRM site (<a href="#pone-0000955-t004">Table 4</a>). This arrangement is common to many stress response regulators, e.g., the <em>lexA</em> genes of many species <a href="#pone.0000955-Little1">[117]</a>. Thus, we propose that DdrO is the global regulator of the RDR regulon in the <em>Deinococcus</em> lineage.</p>



<h4>Impact of the comparative-genomic analysis of the two <em>Deinococcus</em> genomes on Resistance Models</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>In 1971, Moseley and Mattingly reported the first mutant analyses for <em>D. radiodurans</em> that showed that its recovery from radiation is dependent on DNA repair <a href="#pone.0000955-Moseley1">[118]</a>. Subsequent research confirmed that DNA repair enzymes, which are central to recovery of irradiated bacteria in general, were key to <em>D. radiodurans</em> survival. Remarkably, several highly radiation-sensitive <em>D. radiodurans</em> DNA repair mutants were fully complemented by expression of orthologous genes from radiosensitive bacteria <a href="#pone.0000955-Gutman1">[4]</a>, <a href="#pone.0000955-Agostini1">[90]</a>, <a href="#pone.0000955-Gutman3">[119]</a>–<a href="#pone.0000955-Schlesinger1">[121]</a>. Thus, the extreme resistance phenotype appeared to be dependent, at least in part, on a conventional set of DNA repair functions <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Daly3">[17]</a>, <a href="#pone.0000955-Makarova1">[21]</a>. Generally, this view has been supported by the analysis of the complete genome sequence of <em>D. radiodurans</em> <a href="#pone.0000955-White1">[16]</a>, and subsequently, by whole-transcriptome and whole-proteome analyses for <em>D. radiodurans</em> recovering from IR <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Lipton1">[91]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>. Central to current models of extreme resistance are hypotheses that aim to reconcile the seemingly paradoxical findings that DNA repair proteins in <em>D. radiodurans</em> function extremely efficiently, yet appear structurally unremarkable, and often can be complemented by orthologs from radiosensitive bacteria. Within this conceptual framework, we examined the impact of the inferences on gene-gain and gene-loss derived from the comparative-genomic analysis of the two <em>Deinococcus</em> species on prevailing models of extreme radiation and desiccation resistance.</p>
<h5>Hypothesis I: Chromosome Alignment, Morphology and Repeated Sequences Facilitate Genome Reassembly.</h5><a id="article1.body1.sec2.sec8.sec1.p1" name="article1.body1.sec2.sec8.sec1.p1"></a><p><em>recA</em>-dependent homologous recombination occurs at hundreds of IR-induced DSB sites in <em>D. radiodurans</em> during recovery from 17.5 kGy IR <a href="#pone.0000955-Daly4">[18]</a>, <a href="#pone.0000955-Daly6">[76]</a>–<a href="#pone.0000955-Daly9">[79]</a>. In <em>D. radiodurans</em>, the alignment of its multiple identical chromosomes is often tacitly assumed as the starting point for a given repair model, yet little is known about how, or even if, such chromosomal alignment occurs. The first model that considered this possibility in the recovery of <em>D. radiodurans</em> was published by Minton and Daly in 1995 <a href="#pone.0000955-Daly10">[122]</a>. The model built on the idea that alignment of identical chromosomes is a natural and early consequence of semi-conservative replication, where persistent chromosomal pairing was predicted to facilitate the ‘search for homology’ that precedes homologous recombination. The model made two major predictions: first, transmission electron microscopy (TEM) of chromosomal DNA from <em>D. radiodurans</em> should reveal evidence of structures linking chromosomes; and second, <em>recA</em>-dependent recombination between homologous DNA fragments inserted at widely separated genomic locations should show strong positional effects upon irradiation. Both predictions have been tested and refuted: no linking structures have been observed by TEM-based optical mapping <a href="#pone.0000955-Lin1">[26]</a>, and molecular studies have shown high levels of recombination between homologous DSB fragments irrespective of their genomic origin <a href="#pone.0000955-Daly6">[76]</a>–<a href="#pone.0000955-Daly9">[79]</a>, <a href="#pone.0000955-Daly10">[122]</a>. Thus, it has been concluded that IR-induced DSB fragments in <em>D. radiodurans</em> are mobile and that the structural form of its nucleoids does not play a key role in radioresistance. These conclusions were subsequently strengthened by cryoelectron microscopy of vitreous sections of <em>D. radiodurans</em> <a href="#pone.0000955-Eltsov1">[123]</a>, <a href="#pone.0000955-Eltsov2">[124]</a>, and by nucleoid morphology studies <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Ghosal1">[12]</a>, <a href="#pone.0000955-Zimmerman1">[24]</a>, <a href="#pone.0000955-Gao2">[125]</a>.</p>
<a id="article1.body1.sec2.sec8.sec1.p2" name="article1.body1.sec2.sec8.sec1.p2"></a><p>The genome of <em>D. radiodurans</em> contains numerous, unusual, mosaic-type SNRs <a href="#pone.0000955-White1">[16]</a>, <a href="#pone.0000955-Makarova1">[21]</a>, <a href="#pone.0000955-Makarova2">[29]</a> which potentially could contribute to genome assembly by holding together homologous DSB pairs <a href="#pone.0000955-Lin1">[26]</a>. TEM optical mapping of <em>D. radiodurans</em> recovering from IR, however, showed that IR-induced DSB fragments in <em>D. radiodurans</em> were not linked <a href="#pone.0000955-Lin1">[26]</a>. Consistently, the present whole-genome comparison detected none of these repeats in <em>D. geothermalis</em>, nor any other expanded repeat families, including G-quadruplex sequences (<a href="#pone-0000955-t001">Table 1</a>) (<a href="#pone.0000955.s003">Figure S3</a>). We did not identify any unusual features in chromosome-binding proteins that are conserved in the two <em>Deinococcus</em> genomes compared to the orthologous proteins from other bacteria <a href="#pone.0000955-Makarova1">[21]</a> (<a href="#pone.0000955.s017">Table S7</a> and <a href="#pone.0000955.s018">S8</a>). Thus, our comparative analysis does not seem to support Hypothesis I. More broadly, there is currently no convincing experimental evidence supporting the idea that structural alignment, aggregation or organization of the <em>D. radiodurans</em> chromosomes has a significant effect on radiation/desiccation resistance. However, we cannot rule out the possibility that the genomes of sensitive bacteria have structural characteristics that predispose them to inefficient genome reassembly.</p>

<h5>Hypothesis II: A Subset of Uncharacterized Genes Encode Functions that Enhance the Efficiency of DNA repair.</h5><a id="article1.body1.sec2.sec8.sec2.p1" name="article1.body1.sec2.sec8.sec2.p1"></a><p>In general, bioinformatic and experimental studies suggest that genome configuration and copy-number or the protection and repair functions of sensitive bacteria do not have unique properties that predispose them to DNA damage or inefficient DNA repair <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Cox1">[20]</a>, <a href="#pone.0000955-Makarova1">[21]</a>. More specifically, chromosomes in sensitive and resistant bacteria are equally susceptible to IR-induced DSB damage <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Gerard1">[19]</a> and UV-induced base damage <a href="#pone.0000955-Setlow1">[126]</a>; and DNA repair and protection genes of <em>T. thermophilus</em>, a radio-sensitive representative of the <em>Deinococcus</em>-<em>Thermus</em> group, and <em>E. coli</em> do not show obvious differences from their counterparts in <em>D. radiodurans</em> or <em>D. geothermalis</em> <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Makarova1">[21]</a>, <a href="#pone.0000955-Omelchenko1">[27]</a> (<a href="#pone.0000955.s018">Table S8</a>). Furthermore, several <em>E. coli</em> DNA repair genes, including <em>polA</em> and <em>uvrA</em>, have been shown to restore the corresponding radiation-sensitive <em>D. radiodurans</em> mutants to wild-type levels of <em>D. radiodurans</em> resistance <a href="#pone.0000955-Gutman1">[4]</a>, <a href="#pone.0000955-Agostini1">[90]</a>, <a href="#pone.0000955-Minton1">[120]</a>; and the products of interchromosomal recombination in <em>D. radiodurans</em> following irradiation are consistent with the canonical version of the DSB repair model <a href="#pone.0000955-Daly6">[76]</a>–<a href="#pone.0000955-Daly9">[79]</a>. It has been proposed that <em>D. radiodurans</em> uses a core set of conventional DNA repair enzymes in novel ways, where conventional repair activities are enhanced by as yet uncharacterized proteins. For example, Zahradka et al have recently proposed a model called <span class="underline">e</span>xtended <span class="underline">s</span>ynthesis <span class="underline">d</span>ependent <span class="underline">s</span>trand <span class="underline">a</span>nnealing (ESDSA) that utilizes PolA in an unprecedented way <a href="#pone.0000955-Zahradka1">[22]</a>.</p>
<a id="article1.body1.sec2.sec8.sec2.p2" name="article1.body1.sec2.sec8.sec2.p2"></a><p>Under the ESDSA, DSB fragments formed in irradiated <em>D. radiodurans</em> are first subject to a 5′→3′ exonuclease resection mechanism that generates overhanging 3′ tails. A 3′ tail then invades a homologous DSB fragment derived from a different chromosomal copy, displacing the corresponding 5′ strand as a loop. Synthetic extension of the priming 3′ terminus might then proceed to the end of the invaded fragment, followed by annealing of the newly synthesized long 3′ extension with a complementary strand of another fragment engaged in ESDSA (<a href="#pone.0000955.s007">Figure S7</a>). For example, if the sequences of two priming fragments were <em>ABCD</em> and <em>GHIJ</em>, then a bridging and templating fragment could be <em>DEFG</em>, and the sequence of the assembled contig would be <em>ABCDEFGHIJ</em> <a href="#pone.0000955-Zahradka1">[22]</a>. The ESDSA model accounts for the formation of large, interspersed blocks of old and new DNA observed in repaired <em>D. radiodurans</em> chromosomes. Some aspects of the ESDSA model, however, are difficult to reconcile with earlier experimental findings for <em>recA</em>-independent single-stranded annealing (SSA) mechanisms in irradiated <em>D. radiodurans</em> <a href="#pone.0000955-Daly8">[78]</a> (<a href="#pone.0000955.s007">Figure S7</a>). Zharadka et al conceded that the SSA model is a potential alternative to ESDSA and could perhaps generate small blocks of old and new DNA <a href="#pone.0000955-Zahradka1">[22]</a>, but pointed out that the <em>E. coli</em> PolA Klenow fragment, that lacks the 5′→3′ exonuclease, fully complements <em>D. radiodurans polA</em> mutants for resistance to γ-radiation. The present analysis shows that, although <em>D. radiodurans</em> and <em>D. geothermalis</em> do not encode <em>recBCE</em>, they both encode <em>recJ</em>, a putative 5′→3′exonuclease that could potentially provide nuclease activity missing in the Klenow fragment (<a href="#pone.0000955.s018">Table S8</a>).</p>
<a id="article1.body1.sec2.sec8.sec2.p3" name="article1.body1.sec2.sec8.sec2.p3"></a><p>The possibility that extreme resistance in <em>D. radiodurans</em> is determined by novel genes that enhance conventional repair functions has also been examined <a href="#pone.0000955-Cox1">[20]</a>, <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Harris1">[98]</a>. At least 12 genes of <em>D. radiodurans</em>, which were implicated in resistance by transcriptional profiling following IR, have been knocked out and the resulting mutants were characterized for IR resistance (<a href="#pone-0000955-t003">Table 3</a>). Remarkably, for most of the novel mutants, the IR resistances remained high <a href="#pone.0000955-Cox1">[20]</a>, <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Harris1">[98]</a>, indicating that few of the uncharacterized genes, at least individually, makes a substantial contribution to the recovery of irradiated <em>D. radiodurans</em>. For example, the DR0423 protein has been reported to bind 3′ ends of single-stranded DNA molecules, perhaps, protecting 3′ termini generated by SSA or ESDSA from nuclease degradation. A DR0423 knockout mutant, however, retained approximately half of the wild-type level of IR resistance <a href="#pone.0000955-Tanaka1">[92]</a>, <a href="#pone.0000955-Harris1">[98]</a>. To date, only a few of the uncharacterized genes selected for disruption analysis have contained the RDRM (<a href="#pone-0000955-t003">Table 3</a> and <a href="#pone-0000955-t004">4</a>).</p>
<a id="article1.body1.sec2.sec8.sec2.p4" name="article1.body1.sec2.sec8.sec2.p4"></a><p>At least three <em>Deinococcus</em> proteins involved in repair show features that stand out against the overall, “garden-variety” of bacterial repair systems. First, <em>D. radiodurans</em> encodes a protein (DR1289) of the RecQ helicase family, which contains three Helicase and RNase D C-terminal (HRDC) domains, whereas most of the other bacterial RecQ proteins have a single HRDC domain. A <em>D. radiodurans recQ</em> knockout mutant is sensitive to IR, UV, H<sub>2</sub>O<sub>2</sub>, and MMC, and it has been reported that all three HRDC domains contribute to resistance <a href="#pone.0000955-Huang1">[127]</a>. However, <em>D. geothermalis</em> has no ortholog of the <em>D. radiodurans</em> RecQ, but does encode the Dgeo_1226 protein that contains a helicase superfamily II C-terminal domain and a second HDRC domain that has high similarity to the corresponding domains of DR1289. Both DR1289 and Dgeo_1226 belong to the predicted resistance regulon (<a href="#pone-0000955-t004">Table 4</a>). A second exceptional case is RecA, the key repair protein that is required for homologous DNA recombinational repair following irradiation <a href="#pone.0000955-Cox1">[20]</a>. The DNA strand-exchange reactions promoted by the RecA proteins from all other bacteria studied to date are ordered such that the single-stranded DNA is bound first, followed by the double-stranded DNA. In contrast, the <em>D. radiodurans</em> RecA binds the DNA duplex first and the homologous single-stranded DNA substrate second <a href="#pone.0000955-Kim1">[128]</a>. It seems likely, however, that these unusual properties of RecA are ancestral to the <em>Deinococcus-Thermus</em> group. Indeed, most of the amino acid residues that are distinct in <em>Deinococcus</em> and could be responsible for the structural and functional differences between the RecA proteins of <em>Deinococcus</em> and other bacteria are also present in the RecA sequence of <em>Thermus</em> (<a href="#pone.0000955.s008">Figure S8</a>). In this context, early work by Carroll <em>et al</em> <a href="#pone.0000955-Carroll1">[111]</a> reported that <em>E. coli</em> RecA did not complement an IR-sensitive <em>D. radiodurans recA</em> point-mutant (rec30) and that expression of <em>D. radiodurans</em> RecA in <em>E. coli</em> was lethal. More recently, however, it has been reported that <em>E. coli recA</em> can provide partial complementation to a <em>D. radiodurans recA</em> null mutant <a href="#pone.0000955-Schlesinger1">[121]</a>, and that <em>D. radiodurans recA</em> fully complements <em>E. coli recA</em> mutants <a href="#pone.0000955-Narumi2">[129]</a>. This suggests that the <em>D. radiodurans</em> RecA protein is not as unusual as initially believed, but rather is more analogous to <em>polA</em> and <em>uvrA</em> of <em>D. radiodurans</em>, which can be functionally replaced by <em>E. coli</em> orthologs <a href="#pone.0000955-Gutman1">[4]</a>, <a href="#pone.0000955-Agostini1">[90]</a>, <a href="#pone.0000955-Gutman2">[93]</a>, <a href="#pone.0000955-Minton1">[120]</a>. A third example, the <em>Deinococcus</em> single-stranded binding protein (Ssb) has a distinct structure, with two OB-fold domains in a monomer, but this feature was apparently already present in the common ancestor of <em>Deinococcus/Thermus</em> group and therefore cannot be linked to radiation resistance directly <a href="#pone.0000955-Bernstein1">[130]</a>.</p>
<a id="article1.body1.sec2.sec8.sec2.p5" name="article1.body1.sec2.sec8.sec2.p5"></a><p>It has been repeatedly proposed that nonhomologous end-joining (NHEJ) occurs in <em>D. radiodurans</em> <a href="#pone.0000955-Cox1">[20]</a>, <a href="#pone.0000955-Bowater1">[131]</a>–<a href="#pone.0000955-Narumi3">[136]</a>. However, experiments specifically designed to test for the occurrence of NHEJ in <em>D. radiodurans</em> have shown that NHEJ of irradiation-induced DSB fragments is extremely rare, if not absent <a href="#pone.0000955-Daly8">[78]</a>. More recent work also supports this conclusion <a href="#pone.0000955-Zahradka1">[22]</a>. In the present and a previous study, we did not identify any orthologs of genes from other organisms that might encode NHEJ in <em>D. geothermalis</em> or <em>D. radiodurans</em> <a href="#pone.0000955-Makarova1">[21]</a>. However, it cannot be ruled out that <em>Deinococcus</em> encodes a unique NHEJ system. For example, DRB0100 encodes an ATP-dependent ligase that contains domains that could potentially contribute to NHEJ, namely, a predicted phosphatase of the H2Macro superfamily and an HD family phosphatase and polynucleotide kinase <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>. Furthermore, DRB0100 belongs to a set of three genes comprising a putative operon (DRB0098-0100) that is strongly induced by IR. A homozygous disruption of the DRB0100 gene, however, is fully IR-resistant (<a href="#pone-0000955-t003">Table 3</a>) (<a href="#pone-0000955-g004">Figure 4</a>), and genome comparison showed that <em>D. geothermalis</em> has no orthologs of DRB0100 or any functionally related operons. Despite the strong induction of DRB0100 following irradiation and the apparent relevance of the predicted function of this protein to <em>D. radiodurans</em> repair, DRB0100 appears not to contribute to resistance (<a href="#pone-0000955-g004">Figure 4</a>), and when purified, does not display DNA or RNA ligase activity <em>in vitro</em> <a href="#pone.0000955-Blasius1">[137]</a>. These findings, therefore, reflect a broader paradox of <em>Deinococcus</em>: whereas computational analyses have revealed an increasing number of new proteins potentially involved in the extreme resistance phenotype, very few of the corresponding <em>D. radiodurans</em> mutants tested so far have had a significant effect on its IR resistance. The present work leads to further shrinking of the set of genes implicated as major contributors to the resistance phenotype by showing that many of the original candidates are not conserved between <em>D. geothermalis</em> and <em>D. radiodurans</em>. Thus, our comparative analysis appears to be inconsistent with Hypothesis II, and reinforces inferences from a growing body of experimental work on <em>Deinococcus</em> species, which support that these organisms rely on a relatively conventional set of DNA repair functions.</p>

<h5>Hypothesis III: The level of Oxidative Protein Damage during Irradiation Determines Survival.</h5><a id="article1.body1.sec2.sec8.sec3.p1" name="article1.body1.sec2.sec8.sec3.p1"></a><p>Over the past decade, several observations have challenged the DNA-centered view of IR toxicity in eukaryotes and prokaryotes <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Daly3">[17]</a>, <a href="#pone.0000955-Daly5">[23]</a>, <a href="#pone.0000955-Wright1">[138]</a>, including (i) IR-induced bystander-effects in mammalian cells, defined as cytotoxic effects elicited in non-irradiated cells by irradiated cells, or following microbeam irradiation of cells where the cytoplasm but not the nucleus is directly traversed by radiation <a href="#pone.0000955-Belyakov1">[139]</a>; (ii) the genomes of radiation-sensitive bacteria revealed nothing obviously lacking in their repertoire of DNA repair and protection systems compared to resistant bacteria <a href="#pone.0000955-Ghosal1">[12]</a>, <a href="#pone.0000955-Makarova1">[21]</a>; and (iii) for a group of phylogenetically diverse bacteria at the opposite ends of IR resistance, the amount of protein damage, but not DNA DSB damage, was quantifiably related to radioresistance <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Daly3">[17]</a>. Thus, while the etiological radicals underlying different oxidative toxicities appear closely related <a href="#pone.0000955-vonSonntag1">[140]</a>, the pathway connecting the formation of IR-induced ROS with endpoint biological damage is still not definitively established <a href="#pone.0000955-Daly5">[23]</a>. It has been proposed recently that proteins in IR-sensitive cells are major initial targets, where cytosolic proteins oxidized by IR might actively promote mutation by transmitting damage to DNA <a href="#pone.0000955-Du1">[141]</a>, and IR-damaged DNA repair enzymes might passively promote mutations by repair malfunction <a href="#pone.0000955-Daly3">[17]</a>. In comparison, Mn-dependent radioprotective complexes in IR-resistant bacteria <a href="#pone.0000955-Daly3">[17]</a> appear to protect proteins from oxidation during irradiation, with the result that enzymatic systems involved in recovery survive and function with great efficiency <a href="#pone.0000955-Daly3">[17]</a>. The proposed mechanism of extreme IR resistance requires a high intracellular Mn/Fe concentration ratio, where redox-cycling of Mn(II) complexes in resistant bacteria <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Daly3">[17]</a> scavenge a subset of IR-induced ROS that target proteins. Because the formation of ROS during irradiation is extremely rapid <a href="#pone.0000955-vonSonntag1">[140]</a>, an intracellular protection system that is ubiquitous, but not highly dependent on the induction of enzymes, stage of growth, or temperature over a range at which cells are metabolically active, could provide a selective advantage to the host in diverse settings.</p>
<a id="article1.body1.sec2.sec8.sec3.p2" name="article1.body1.sec2.sec8.sec3.p2"></a><p>Since high intracellular Mn/Fe ratios have been implicated in radiation and desiccation resistance <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Ghosal1">[12]</a>, <a href="#pone.0000955-Daly3">[17]</a>, <a href="#pone.0000955-Daly5">[23]</a>, we examined the intracellular concentrations and distributions of Mn, Fe and seven other elements in <em>D. geothermalis</em> compared to <em>D. radiodurans</em>, determined by x-ray fluorescence (XRF) microscopy (<a href="#pone-0000955-g007">Figure 7</a>) <a href="#pone.0000955-Kemner1">[142]</a>. The XRF analyses showed that the intracellular levels of Mn and Fe and their locations in <em>D. geothermalis</em> are essentially the same as <em>D. radiodurans</em> <a href="#pone.0000955-Daly3">[17]</a>, but very different from the concentrations and distributions in IR-sensitive bacteria <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Kemner1">[142]</a>. In this context, both <em>D. radiodurans</em> and <em>D. geothermalis</em> encode the Mn(II) transporter Nramp (DR1709) and a putative Mn-dependent transcriptional regulator TroR (DR2539) <a href="#pone.0000955-Daly1">[5]</a>, but lack many genes for Fe homeostasis common in other bacteria, including for siderophore biosynthesis (COG3486, COG4264, COG4771) and Fe transport (COG1629, COG0810) (<a href="#pone.0000955.s019">Table S9</a>) <a href="#pone.0000955-Ghosal1">[12]</a>. Consistently, <em>D. radiodurans</em> and <em>D. geothermalis</em> do not secrete siderophores (<a href="#pone.0000955.s009">Figure S9</a>), the <em>nramp</em> gene of <em>D. radiodurans</em> is essential and could not be disrupted, and the Fe uptake regulator (Fur) in <em>D. radiodurans</em> was dispensable (<a href="#pone.0000955.s010">Figure S10</a>); a system for gene disruption in <em>D. geothermalis</em> has not been developed. Other recent work that has strengthened the argument for a critical role of Mn(II) in the extreme resistance phenotypes of <em>D. radiodurans</em> includes <em>in vitro</em> studies of Heinz and Marx <a href="#pone.0000955-Heinz1">[143]</a>. They have shown that purified <em>D. radiodurans</em> PolA and <em>E. coli</em> PolA can bypass certain forms of IR-induced DNA damage during replication in the presence but not in the absence of 1 mM Mn(II), and suggested that Mn(II) ions might serve as important modulators of enzyme function <a href="#pone.0000955-Heinz1">[143]</a>. In summary, we conclude that our genome comparison (<a href="#pone.0000955.s019">Table S9</a>), gene knockout (<a href="#pone.0000955.s010">Figure S10</a>) and element analyses (<a href="#pone-0000955-g007">Figure 7</a>) appear to be consistent with Hypothesis III, whereby survival is facilitated by systems which regulate the concentration and distribution of intracellular Mn and Fe. Based on recent work, it appears that the presence of globally-distributed intracellular nonenzymic Mn(II) complexes in resistant bacteria facilitates recovery by preventing a form of IR-induced Fe-catalyzed protein oxidation known as carbonylation <a href="#pone.0000955-Daly3">[17]</a>.</p>
<div class="figure" id="pone-0000955-g007"><div class="img"><a name="pone-0000955-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000955" data-uri="info:doi/10.1371/journal.pone.0000955.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000955.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000955.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000955.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000955.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>X-ray fluorescence (XRF) microprobe element distribution maps <a href="#pone.0000955-Kemner1">[142]</a>.</span></strong></p><a id="article1.body1.sec2.sec8.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec8.sec3.fig1.caption1.p1"></a><p>A, <em>D. geothermalis</em> (diplococcus). B, <em>D. radiodurans</em> (tetracocus). Cells were harvested from mid-logarithmic cultures in undefined rich medium, imaged, and quantified as described previously <a href="#pone.0000955-Daly3">[17]</a>. The element distribution images are plotted to different scales designated by a single color-box, where red represents the highest concentration and black the lowest. ppm values in parentheses next to the element symbol correspond to red. XRF microprobe analysis measurements were made at beamline 2ID-D at the Advanced Photon Source, Argonne National Laboratory as described recently <a href="#pone.0000955-Daly3">[17]</a>.</p>
<span>doi:10.1371/journal.pone.0000955.g007</span></div>


<h4>Conclusions</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Based on their identical radiation resistance characteristics and close phylogenetic relationship, <em>D. geothermalis</em> and <em>D. radiodurans</em> are well-suited to defining a minimal set of conserved genes that could be responsible for extreme resistance. The two major findings of this analysis are (i) the characterization of the evolutionary trends that led to the emergence of extreme stress resistance in the <em>Deinococcus</em> lineage, in particular the finding that many families of paralogous genes, previously shown to be expanded in <em>D. radiodurans</em>, proliferated before the emergence of the common ancestor of the <em>Deinococci</em>, but were not present in the ancestor of the <em>Deinococcus-Thermus</em> group (<a href="#pone-0000955-t002">Table 2</a>); and (ii) delineation of a set of genes that comprise the predicted <em>Deinococcus</em> radiation and desiccation response regulon, which defines a new subgroup of targets for investigation in the <em>Deinococci</em> (<a href="#pone-0000955-t004">Table 4</a>). These findings have strengthened the view that <em>Deinococci</em> rely more heavily on the high efficiency of their detoxifying systems, including enzymic and nonenzymic ROS scavengers, than on the number and specificity of their DNA repair systems (<a href="#pone-0000955-t003">Table 3</a>). Our findings, however, do not rule out the possibility that the exceptional efficiency of DNA repair processes in both <em>Deinococcus</em> species is, at least in part, due to modifications of a set of universal repair genes. With respect to the impact of the whole-genome sequence of <em>D. geothermalis</em> on prevailing models of extreme IR resistance, the results of the comparative analysis weaken the arguments for a role of higher-order chromosome alignment structures (Hypothesis I); more clearly define and substantially revise downward the number of uncharacterized genes that might participate in DNA repair and contribute to resistance (Hypothesis II); and are consistent with the notion of a predominant role in resistance of systems involved in cellular protection and detoxification (cell-cleaning) (Hypothesis III).</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>In the hierarchy of DNA lesions caused <em>in vivo</em> by radiation, DSBs are the least frequent ones, but the most lethal <a href="#pone.0000955-vonSonntag1">[140]</a>. Since the number of genomic DSBs induced by a given dose of IR in resistant and sensitive bacteria is about the same <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Gerard1">[19]</a>, a legitimate question is whether resistant and sensitive bacteria are also equally susceptible to DNA base damage. Setlow and Duggan showed that <em>D. radiodurans</em> and <em>E. coli</em> are similarly susceptible to DNA thymine-dimers caused by UV <a href="#pone.0000955-Setlow1">[126]</a>. For IR and UV, the differences reported in resistance of DNA to radiation damage are not nearly sufficient to account for the relative resistance of <em>D. radiodurans</em>. Thus, it seems surprising that the recombination and excision repair systems of <em>D. geothermalis</em> and <em>D. radiodurans</em> did not proliferate compared to sensitive cells <a href="#pone.0000955-Daly1">[5]</a>. The DNA repair and damage signaling systems of these radiation reisistant bacteria appear quantitatively and qualitatively even less complex and diverse than those reported for some sensitive bacteria <a href="#pone.0000955-Daly1">[5]</a>, <a href="#pone.0000955-Qiu1">[144]</a>. Instead, the stress-resistance phenotypes of the <em>Deinococcus</em> lineage appear to have evolved progressively by accumulation of cell-cleaning systems which eliminate organic and inorganic cell components that become toxic under radiation or desiccation <a href="#pone.0000955-Ghosal1">[12]</a>, <a href="#pone.0000955-Daly5">[23]</a>, <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>. In <em>D. geothermalis</em> and <em>D. radiodurans</em>, this form of cell-cleaning appears to manifest itself as protein protection during exposure to IR <a href="#pone.0000955-Daly3">[17]</a> or desiccation [JFK, EKG, MJD, unpublished], where proteins in <em>Deinococci</em> are substantially more resistant to oxidative damage than proteins in sensitive bacteria <a href="#pone.0000955-Daly3">[17]</a>. Our finding that many genes in the predicted <em>Deinococcus</em> damage response regulon are the same as those found in SOS regulons of sensitive bacteria, but are regulated differently, is easily reconciled with the idea that enzymes and biochemical pathways in resistant bacteria survive and function more efficiently because they are less prone to interference from the toxic byproducts of IR and desiccation <a href="#pone.0000955-Ghosal1">[12]</a>, <a href="#pone.0000955-Daly3">[17]</a>, <a href="#pone.0000955-Daly5">[23]</a>.</p>
<a id="article1.body1.sec2.sec9.p3" name="article1.body1.sec2.sec9.p3"></a><p>More generally, our findings place constraints on the degree to which functional inferences can be made from whole-genome transcriptome analyses based on a single organism. For example, two independent analyses of gene induction in <em>D. radiodurans</em> recovering from different IR doses revealed numerous genes that are upregulated during the post-irradiation recovery, many of which were viewed as plausible candidates for a significant role in resistance <a href="#pone.0000955-Liu1">[46]</a>, <a href="#pone.0000955-Tanaka1">[92]</a>. The hierarchy of induced genes in both transcriptome analyses was very similar, however, most of the highly induced <em>D. radiodurans</em> genes have no orthologs in <em>D. geothermalis</em>, and knockout of many of the uncharacterized unique <em>D. radiodurans</em> genes that were strongly induced by IR had little effect on IR resistance. A similar paradigm is emerging from the analysis of other systems, where the cellular transcriptional response to stress was largely stochastic, frequently involving genes known to be unrelated to the mechanisms under investigation <a href="#pone.0000955-Fong1">[145]</a>-<a href="#pone.0000955-Stern1">[147]</a>. Thus, it stands to reason that any comprehensive bioinformatics effort aimed at deciphering a complex, multi-gene phenotype using whole-genome, transcriptome and proteome approaches should aim to study at least two closely-related species. In the present context of understanding the genomic basis of extreme resistance phenotypes and the nature of the common ancestor of the <em>Deinococcus</em>-<em>Thermus</em> group, we consider <em>Truepera radiovictrix</em> an appropriate next candidate for whole-genome sequencing. <em>T. radiovictrix</em> is a recently discovered, deeply branching representative of the <em>Deinococcus</em> branch that is both thermophilic and extremely IR-resistant <a href="#pone.0000955-Albuquerque1">[148]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The strains used were as follows: <em>Deinococcus radiodurans</em> (ATCC BAA-816), <em>Deinococcus geothermalis</em> (DSM 11300), and <em>Escherichia coli</em> (K-12) (MG1655).</p>


<h4>Cell Growth, Irradiation, Mutant Construction, and PCR</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p><em>D. radiodurans</em> strain ATCC BAA-816 was grown at 32°C in undefined liquid nutrient-rich medium TGY (1% tryptone/0.1% glucose/0.5% yeast extract) or on TGY solid medium <a href="#pone.0000955-Daly3">[17]</a>. In liquid culture, cell density was determined at 600&nbsp;nm by a Beckman Coulter spectrophotometer. For acute IR (<sup>60</sup>Co Gammacell irradiation unit, J. L. Shepard and Associates, Model 109) or UV (254 nm) exposures, late logarithmic-phase <em>D. radiodurans</em> cultures [OD<sub>600</sub> = 0.9, 1×10<sup>8</sup> colony-forming units (cfu)/ml] were irradiated to the indicated doses (<a href="#pone-0000955-g001">Figure 1</a>). Cell viability and cell numbers were determined by plate assay as described previously <a href="#pone.0000955-Daly3">[17]</a>. Three independent cell cultures and irradiation treatments of the same kind were performed and served as biological replicates for determining irradiation resistance profiles. To test the predicted involvement of the indicated genes, a mutant (<a href="#pone.0000955.s006">Figure S6</a>) was generated using previously developed <em>D. radiodurans</em> disruption protocols <a href="#pone.0000955-Markillie1">[75]</a>. PCR was carried out as described previously <a href="#pone.0000955-Liu1">[46]</a>.</p>


<h4>Whole-Genome Sequencing, Assembly and Structural Analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The complete genome of <em>D. geothermalis</em> (DSM 11300) was sequenced at the Joint Genome Institute (JGI) using a combination of 3 kb-, 8 kb- and fosmid- (40 kb) libraries. Library construction, sequencing, finishing, and automated annotation steps were carried out as follows.</p>
<h5>DNA shearing and sub-cloning.</h5><a id="article1.body1.sec3.sec3.sec1.p1" name="article1.body1.sec3.sec3.sec1.p1"></a><p>Approximately 3–5 µg of isolated DNA was randomly sheared to 3 kb fragments in a 100 µl volume using a HydroShear™ (Genomic Solutions, Ann Arbor, MI). The sheared DNA was immediately blunt end-repaired at room temperature for 40 min using 6 U of T4 DNA Polymerase (Roche Diagnostics, Indianapolis, IN), 30 U of DNA Polymerase I Klenow Fragment (NEB, Beverly, MA), 10 µl of 10 mM dNTP mix (GE Healthcare, Piscataway, NJ), and 13 µl of 10× Klenow Buffer in a 130 µl total volume. After incubation, the reaction was heat-inactivated for 15 min at 70°C, cooled to 4°C for 10 min, and then frozen at −20°C for storage. The end-repaired DNA was run on a 1% Tris/Borate/EDTA (TBE) agarose gel for ~60 min at 120 volts. Using ethidium bromide stain and UV illumination, 3 kb sheared fragments were extracted from the agarose gel and purified using QIAquick™ Gel Extraction Kit (QIAGEN, Valencia, CA). Approximately 300 ng of purified fragment was blunt-end-ligated overnight at 16°C into the <em>Sma</em> I site of 100 ng of pUC18 cloning vector (Roche) using 12 U T4 DNA Ligase, 3.2 µl 10× buffer (Roche), and 4.8 µl 30% PEG in a 32 µl total reaction volume. A very similar process was carried out to create an 8 kb library in pMCL200 with 10 µg of isolated genomic DNA.</p>
<a id="article1.body1.sec3.sec3.sec1.p2" name="article1.body1.sec3.sec3.sec1.p2"></a><p>Following standard protocols, 1 µl of each ligation product (3 kb or 8 kb) was electroporated into DH10B Electromax™ cells (Invitrogen, Carlsbad, CA) using the GENE PULSER® II electroporator (Bio-Rad, Hercules, CA). Transformed cells were transferred into 1 mL of SOC medium and incubated at 37°C in a rotating wheel for 1 h. Cells (usually 20–50 µl) were spread on 22×22 cm LB agar plates containing 100 µg/mL of ampicillin (pUC19) or 20 µg/mL of chloramphenicol (pMCL200), 120 µg/mL of IPTG, and 50 µg/mL of X-GAL. Colonies were grown for 16 h at 37°C. Individual white recombinant colonies were selected and picked into 384-well microtiter plates containing LB/glycerol (7.5% v/v) media containing 50 µg/mL of ampicillin or 20 µg/mL of chloramphenicol using the Q-Bot™ multitasking robot (Genetix, Dorset, U.K.). To test the quality of the library, 48 colonies were directly PCR-amplified with pUC m13–28 and –40 primers using standard protocols. Libraries passed PCR quality control if they had &gt;90% 3 kb inserts or 8 kb inserts, respectively. For more details, see research protocols at <a href="http://www.jgi.doe.gov">www.jgi.doe.gov</a>.</p>

<h5>Plasmid amplification.</h5><a id="article1.body1.sec3.sec3.sec2.p1" name="article1.body1.sec3.sec3.sec2.p1"></a><p>One µl-aliquots of saturated <em>E. coli</em> cultures (DH10B) containing (pUC19 vector with random 3 kb DNA inserts or pMCL200 vector with random 8 kb DNA inserts) were added to 5 µl of a 10 mM Tris-HCl pH 8.2 and 0.1 mM EDTA denaturation buffer. The mixtures were heat-lysed at 95°C for 5 min then placed at 4°C for 5 min. To these denatured products, 4 µl of a rolling circle amplification (RCA) reaction mixture (Templiphi™ DNA Sequencing Template Amplification Kit, GE Healthcare) were added. The amplification reactions were carried out at 30°C for 12–18 h. The amplified products were heat-inactivated at 65°C for 10 min then placed at 4°C until used as template for sequencing <a href="#pone.0000955-Detter1">[149]</a>.</p>

<h5>Plasmid sequencing.</h5><a id="article1.body1.sec3.sec3.sec3.p1" name="article1.body1.sec3.sec3.sec3.p1"></a><p>Aliquots of the 10 µl amplified plasmid RCA products were sequenced with standard pUC m13–28 or –40 primers. The reactions typically contained 1 µl of the RCA product, sequenced with 4 pmoles (1 µl) of standard M13–28 or –40 primers, 0.5 µl 5×buffer, 1.75 µl H<sub>2</sub>O, and 0.75 µl BigDye sequencing kit (Applied Biosystems) at 1 min denaturation and 25 cycles of 95°C for 30 sec, 50°C for 20 sec, 60°C for 4 min, and finally held at 4°C. The reactions were then purified by a magnetic bead protocol (see research protocols, <a href="http://www.jgi.doe.gov">www.jgi.doe.gov</a>) and run on an ABI PRISM 3730xl (Applied Biosystems) capillary DNA sequencer.</p>

<h5>Fosmid Library Construction.</h5><a id="article1.body1.sec3.sec3.sec4.p1" name="article1.body1.sec3.sec3.sec4.p1"></a><p>Approximately 15–20 µg of isolated DNA was randomly sheared to 40 kb fragments (25 cycles at speed code 17 using the large assembly, part # JHSH204007) in a 60 µL volume using a HydroShear™ (GeneMachines, San Carlos, CA). The sheared DNA was immediately blunt end-repaired at room temperature for 45 min using the End-It end-repair kit (Epicentre, Madison, WI). The end-repair reaction contained 60 µL sheared DNA, 8 µL of 10×End-It buffer, 8 µL of 2.5 mM End-It dNTP mix, 8 µL of 10 mM End-It ATP, and 4 µL of End-It Enzyme mix in a 80 µL total volume. After 45 min of incubation, the reaction was heat-inactivated for 10 min at 70°C, cooled to 4°C for 10 min and then frozen at −20°C for storage. The end-repaired DNA was run on a 1% TBE low melting point agarose gel for 13 hours using the following conditions (Temperature: 14°C, Voltage: 4.5 V/cm, Pulse initial: 1.0–final: 7.0 sec, Angle: 120°) on a BioRad Chef-DR III™ System PFGE system. Using standard procedures, the gel was stained with ethidium bromide, destained, and visualized under UV for less than 10 seconds while the 40 kb band was excised. DNA was extracted from the agarose gel and blunt-end ligated into pCC1FOS following the Copy Control Fosmid Kit (Epicentre) protocol. With minimal modifications to the Copy Control Fosmid Kit (Epicentre) protocol, the ligated DNA was packaged, infected and plated for picking and end-sequencing. For detailed JGI protocols used, please see research protocols at <a href="http://www.jgi.doe.gov">www.jgi.doe.gov</a>.</p>

<h5>Assembly and Structural Analysis.</h5><a id="article1.body1.sec3.sec3.sec5.p1" name="article1.body1.sec3.sec3.sec5.p1"></a><p>Draft assemblies were based on 34,919 total reads. The Phred/Phrap/Consed software package (<a href="http://www.phrap.com">http://www.phrap.com</a>) was used for sequence assembly and quality assessment <a href="#pone.0000955-Ewing1">[150]</a>, <a href="#pone.0000955-Ewing2">[151]</a>. After the whole-genome shotgun stage, sequence reads were assembled with parallel Phrap (High Performance Software, LLC). All mis-assemblies were corrected by editing in Consed <a href="#pone.0000955-Gordon1">[152]</a>, and gaps between contigs were closed by custom primer walk or PCR amplification (Roche Applied Science, Indianapolis, IN). The completed genome sequence of <em>D. geothermalis</em> (DSM 11300) contained 36,718 reads, achieving an average of 8-fold sequence coverage per base with an error rate less than 1 in 100,000. The <em>D. geothermalis</em> genome sequence can be accessed at GenBank, or at the JGI Integrated Microbial Genomes website (<a href="http://img.jgi.doe.gov">http://img.jgi.doe.gov</a>). Predicted coding sequences were manually analyzed and evaluated using an Integrated Microbial Genomes (IMG) annotation pipeline (<a href="http://img.jgi.doe.gov">http://img.jgi.doe.gov</a>). The general structure of the predicted <em>D. geothermalis</em> genome was examined by PFGE as described previously for <em>D. radiodurans</em> <a href="#pone.0000955-Daly7">[77]</a>, <a href="#pone.0000955-Daly8">[78]</a>. For structural analysis, <em>D. geothermalis</em> was exposed to 0.2 kGy, which introduces approximately 0.013 DSB/Gy per genome, and the cells were then embedded and lysed in agarose. For PFGE of genomic DNA subjected to restriction endonuclease analysis, non-irradiated <em>D. geothermalis</em> cells were used.</p>



<h4>Orthologous Clusters and Evolutionary Reconstructions</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Reconstructed clusters of orthologous genes for the <em>Deinococcus</em> and <em>Thermus</em> genomes (tdCOGs) were constructed using a technique based on the standard COG approach <a href="#pone.0000955-Tatusov1">[35]</a>, <a href="#pone.0000955-Tatusov2">[36]</a>, <a href="#pone.0000955-Tatusov3">[153]</a>. First, a coarse-grained classification was obtained by assigning predicted genes to the NCBI Clusters of Orthologous Groups of proteins (COGs) using the COGNITOR method <a href="#pone.0000955-Tatusov1">[35]</a>. Then, the genes were organized into tight clusters, based on triangles of best hits <a href="#pone.0000955-Tatusov2">[36]</a>. Proteins belonging to the same cluster were aligned using the MUSCLE program <a href="#pone.0000955-Edgar1">[154]</a>; alignments were converted into PSI-BLAST PSSMs <a href="#pone.0000955-Altschul1">[94]</a>. Subsequent PSI-BLAST searches using these PSSMs against a database of <em>Deinococcus</em> and <em>Thermus</em> proteins were used to merge homologous clusters and previously unclustered proteins into tdCOGs. Cases when proteins assigned to different COGs were automatically clustered into one tdCOG were resolved by manual curation (either COG or tdCOG assignment was changed to remove the contradiction).</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Evolutionary events in the history of the <em>Deinococcus-Thermus</em> group were reconstructed using an <em>ad hoc</em> parsimony approach <a href="#pone.0000955-Omelchenko1">[27]</a>, <a href="#pone.0000955-Makarova6">[155]</a>, <a href="#pone.0000955-Makarova7">[156]</a>. Presence/absence data from COG-based reconstruction of the deep ancestor of <em>Cyanobacteria, Actinobacteria</em> and <em>Deinococcus</em>-<em>Thermus</em> group <a href="#pone.0000955-Omelchenko1">[27]</a> were added to the tdCOG phyletic patterns. Simple parsimony rules were used to infer the ancestral states and the evolutionary events in the history of the <em>Deinococcus</em> and <em>Thermus</em> genomes (e.g. a gene present in both <em>Deinococci</em> and in the deep ancestor but absent in both <em>Thermus</em> species was considered to be present in the <em>Deinococcus</em>-<em>Thermus</em> group ancestor and in the <em>Deinococcus</em> genus ancestor, but lost by the <em>Thermus</em> genus ancestor). The only departure from the straightforward parsimony inference was made for homologous tdCOGs that form clade-specific expanded families, e.g. there are several tdCOGs, all assigned into the same ancestral COG, with genes present in both <em>Deinococci</em> but in neither of the <em>Thermus</em> species. In this case, contrary to the formal parsimony assumption of multiple losses in the <em>Thermus</em> ancestor, the scenario was interpreted as multiple gains (due to duplications) in the <em>Deinococcus</em> ancestor (<a href="#pone.0000955.s020">Table S10</a>).</p>


<h4>X-Ray Fluorescence</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>XRF microscopy measurements were made at beamline 2ID-D at the APS as described previously <a href="#pone.0000955-Daly3">[17]</a>. Briefly, the 2ID-D is an undulator beamline with Fresnel zone plates focusing optics that produced a focal spot with a FWHM (full width at half maximum) spatial resolution of approximately 120 nm for these experiments. For each pixel, the full XRF spectrum between approximately 2 keV and 10 keV was measured using a silicon drift detector. Thus, the distribution of elements between phosphorus and zinc on the periodic table of elements could be measured with 120-nm resolution throughout a cell and its periphery (<a href="#pone-0000955-g007">Figure 7</a>). XRF microprobe measurements were made on <em>D. geothermalis</em> cells grown in TGY to OD<sub>600</sub> 0.3 at 50°C; and <em>D. radiodurans</em> cells were grown in TGY to OD<sub>600</sub> 0.3 at 32°C. The cells were deposited on grids as suspensions in TGY liquid medium, which served to help maintain the structure and viability of the cells as they dried.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000955"></div><a name="pone.0000955.s001" id="pone.0000955.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Proposed evolutionary history of genome partitions in the Deinococcus-Thermus group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0000955.s002" id="pone.0000955.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Genome dot plots for homologous genome partitions of D. radiodurans and D. geothermalis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000955.s003" id="pone.0000955.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Guanine quadruplet repeats in D. radiodurans.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000955.s004" id="pone.0000955.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Verification of the presence of megaplasmid DG206 in D. geothermalis (DSM11300).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.12 MB DOC)</p>
<a name="pone.0000955.s005" id="pone.0000955.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Phylogenetic relationships of tdCOGs of the calcineurin-like phosphoesterase subfamily of COG0639 with proteins from other organisms represented by this COG.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000955.s006" id="pone.0000955.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s006">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Structure of D. radiodurans homozygous mutants.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.25 MB DOC)</p>
<a name="pone.0000955.s007" id="pone.0000955.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s007">Figure S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">The ESDSA model does not fully explain the early formation of covalently closed circular (ccc) derivatives of tandem duplications in irradiated D. radiodurans.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0000955.s008" id="pone.0000955.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s008">Figure S8. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Multiple alignment comparisons for RecA proteins of the Thermus-Deinococcus group with selected representatives of other bacteria.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000955.s009" id="pone.0000955.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s009">Figure S9. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Chrome azurol S agar plate assay for siderophore production.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.13 MB DOC)</p>
<a name="pone.0000955.s010" id="pone.0000955.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s010">Figure S10. </a></strong></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Whereas the nramp gene of D. radiodurans is essential, the fur gene is dispensable.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p2" name="article1.body1.sec4.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.13 MB DOC)</p>
<a name="pone.0000955.s011" id="pone.0000955.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s011">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material11.caption1.p1" name="article1.body1.sec4.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Homology between the D. radiodurans and D. geothermalis megaplasmids.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s011</p><a id="article1.body1.sec4.supplementary-material11.caption1.p2" name="article1.body1.sec4.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000955.s012" id="pone.0000955.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s012">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material12.caption1.p1" name="article1.body1.sec4.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Clusters of orthologous groups of proteins for Deinococcus and Thermus (tdCOGs).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s012</p><a id="article1.body1.sec4.supplementary-material12.caption1.p2" name="article1.body1.sec4.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.24 MB TXT)</p>
<a name="pone.0000955.s013" id="pone.0000955.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s013">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material13.caption1.p1" name="article1.body1.sec4.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Lineage specific expansion of selected families in D. geothermalis (DG), D. radiodurans (DR), T. thermophilus HB27 (TT27), and T. thermophilus HB8 (TT8).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s013</p><a id="article1.body1.sec4.supplementary-material13.caption1.p2" name="article1.body1.sec4.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000955.s014" id="pone.0000955.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s014">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material14.caption1.p1" name="article1.body1.sec4.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Protein families expanded in D. geothermalis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s014</p><a id="article1.body1.sec4.supplementary-material14.caption1.p2" name="article1.body1.sec4.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000955.s015" id="pone.0000955.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s015">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material15.caption1.p1" name="article1.body1.sec4.supplementary-material15.caption1.p1"></a><p class="preSiDOI">Protein families expanded in D. radiodurans.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s015</p><a id="article1.body1.sec4.supplementary-material15.caption1.p2" name="article1.body1.sec4.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0000955.s016" id="pone.0000955.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s016">Table S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material16.caption1.p1" name="article1.body1.sec4.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Gene context and motifs of predicted cytoplasmic proteins shared by two Deinococcus species, but for which homologs outside the lineage do not exist.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s016</p><a id="article1.body1.sec4.supplementary-material16.caption1.p2" name="article1.body1.sec4.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.17 MB DOC)</p>
<a name="pone.0000955.s017" id="pone.0000955.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s017">Table S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material17.caption1.p1" name="article1.body1.sec4.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Stress response-related genes in D. radiodurans (DR), D. geothermalis (DG) and T. thermophilus (TT).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s017</p><a id="article1.body1.sec4.supplementary-material17.caption1.p2" name="article1.body1.sec4.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(0.23 MB DOC)</p>
<a name="pone.0000955.s018" id="pone.0000955.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s018">Table S8. </a></strong></p><a id="article1.body1.sec4.supplementary-material18.caption1.p1" name="article1.body1.sec4.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Genes coding for replication, repair and recombination functions in E. coli, D. radiodurans and T. thermophilus.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s018</p><a id="article1.body1.sec4.supplementary-material18.caption1.p2" name="article1.body1.sec4.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(0.15 MB DOC)</p>
<a name="pone.0000955.s019" id="pone.0000955.s019"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s019">Table S9. </a></strong></p><a id="article1.body1.sec4.supplementary-material19.caption1.p1" name="article1.body1.sec4.supplementary-material19.caption1.p1"></a><p class="preSiDOI">Manganese- and iron-related homeostasis genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s019</p><a id="article1.body1.sec4.supplementary-material19.caption1.p2" name="article1.body1.sec4.supplementary-material19.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0000955.s020" id="pone.0000955.s020"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000955.s020">Table S10. </a></strong></p><a id="article1.body1.sec4.supplementary-material20.caption1.p1" name="article1.body1.sec4.supplementary-material20.caption1.p1"></a><p class="preSiDOI">Parsimony pattern rules for reconstruction of evolutionary events in the Deinococcus/Thermus lineage.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000955.s020</p><a id="article1.body1.sec4.supplementary-material20.caption1.p2" name="article1.body1.sec4.supplementary-material20.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Deb Ghosal at Uniformed Services University of the Health Sciences (USUHS) for conducting the chrome azurol S agar plate assay for siderophore production. We are also grateful to Susan Lucas and Tijana Glavina del Rio of the DOE-Joint Genome Institute for support in genome sequence quality control, production and assembly</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MD KM AL PR KK EG VM. Performed the experiments: BL KK CD ES EG VM AV MZ BR TB. Analyzed the data: EK MD MG YW KM MO AL AC ML BL KK JF CD ES EG VM BR TB AS AG SP EK KM. Contributed reagents/materials/analysis tools: EK MG. Wrote the paper: MD KM. Other: JGI Genome-sequencing project coordinator: AL. Genome annotation QC: EK. Genome annotation: ML KM. Data management/transfer, NCBI submission: SP. Genome Library construction: CD. Genome finishing: TB ES. Programmers: AS AG JF. Contributed to writing the paper: EK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000955-Gupta1" id="pone.0000955-Gupta1"></a>Gupta RS (1998) Protein phylogenies and signature sequences: A reappraisal of evolutionary relationships among archaebacteria, eubacteria, and eukaryotes. Microbiol Mol Biol Rev  62: 1435–1491.  <ul class="find" data-citedArticleID="997680"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Protein+phylogenies+and+signature+sequences%3A+A+reappraisal+of+evolutionary+relationships+among+archaebacteria%2C+eubacteria%2C+and+eukaryotes.&amp;auth=&amp;atitle=Protein+phylogenies+and+signature+sequences%3A+A+reappraisal+of+evolutionary+relationships+among+archaebacteria%2C+eubacteria%2C+and+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+phylogenies+and+signature+sequences%3A+A+reappraisal+of+evolutionary+relationships+among+archaebacteria%2C+eubacteria%2C+and+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+phylogenies+and+signature+sequences%3A+A+reappraisal+of+evolutionary+relationships+among+archaebacteria%2C+eubacteria%2C+and+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000955-Wolf1" id="pone.0000955-Wolf1"></a>Wolf YI, Rogozin IB, Grishin NV, Tatusov RL, Koonin EV (2001) Genome trees constructed using five different approaches suggest new major bacterial clades. BMC Evol Biol  1: 8.  <ul class="find" data-citedArticleID="997868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+trees+constructed+using+five+different+approaches+suggest+new+major+bacterial+clades.&amp;auth=&amp;atitle=Genome+trees+constructed+using+five+different+approaches+suggest+new+major+bacterial+clades." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+trees+constructed+using+five+different+approaches+suggest+new+major+bacterial+clades." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+trees+constructed+using+five+different+approaches+suggest+new+major+bacterial+clades.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000955-Lai1" id="pone.0000955-Lai1"></a>Lai WA, Kampfer P, Arun AB, Shen FT, Huber B, et al.  (2006) <em>Deinococcus ficus</em> sp. nov., isolated from the rhizosphere of <em>Ficus religiosa</em> L. Int J Syst Evol Microbiol  56: 787–791.  <ul class="find" data-citedArticleID="997752" data-doi="10.1099/ijs.0.64007-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.64007-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deinococcus+ficus+sp.+nov.%2C+isolated+from+the+rhizosphere+of+Ficus+religiosa+L." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deinococcus+ficus+sp.+nov.%2C+isolated+from+the+rhizosphere+of+Ficus+religiosa+L.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000955-Gutman1" id="pone.0000955-Gutman1"></a>Gutman PD, Fuchs P, Minton KW (1994) Restoration of the DNA damage resistance of <em>Deinococcus radiodurans</em> DNA polymerase mutants by <em>Escherichia coli</em> DNA polymerase I and Klenow fragment. Mutat Res  314: 87–97.  <ul class="find" data-citedArticleID="997684" data-doi="10.1016/0921-8777(94)90064-7"><li><a href="http://dx.doi.org/10.1016/0921-8777(94)90064-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Restoration+of+the+DNA+damage+resistance+of+Deinococcus+radiodurans+DNA+polymerase+mutants+by+Escherichia+coli+DNA+polymerase+I+and+Klenow+fragment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Restoration+of+the+DNA+damage+resistance+of+Deinococcus+radiodurans+DNA+polymerase+mutants+by+Escherichia+coli+DNA+polymerase+I+and+Klenow+fragment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000955-Daly1" id="pone.0000955-Daly1"></a>Daly MJ, Gaidamakova EK, Matrosova VY, Vasilenko A, Zhai M, et al.  (2004) Accumulation of Mn(II) in <em>Deinococcus radiodurans</em> facilitates gamma-radiation resistance. Science  306: 1025–1028.  <ul class="find" data-citedArticleID="997604" data-doi="10.1126/science.1103185"><li><a href="http://dx.doi.org/10.1126/science.1103185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accumulation+of+Mn%28II%29+in+Deinococcus+radiodurans+facilitates+gamma-radiation+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accumulation+of+Mn%28II%29+in+Deinococcus+radiodurans+facilitates+gamma-radiation+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000955-Daly2" id="pone.0000955-Daly2"></a>Daly MJ (2000) Engineering radiation-resistant bacteria for environmental biotechnology. Curr Opin Biotechnol  11: 280–285.  <ul class="find" data-citedArticleID="997608" data-doi="10.1016/s0958-1669(00)00096-3"><li><a href="http://dx.doi.org/10.1016/s0958-1669(00)00096-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineering+radiation-resistant+bacteria+for+environmental+biotechnology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineering+radiation-resistant+bacteria+for+environmental+biotechnology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000955-Ferreira1" id="pone.0000955-Ferreira1"></a>Ferreira AC, Nobre MF, Rainey FA, Silva MT, Wait R, et al.  (1997) <em>Deinococcus geothermalis</em> sp. nov. and <em>Deinococcus murrayi</em> sp. nov., two extremely radiation-resistant and slightly thermophilic species from hot springs. Int J Syst Bacteriol  47: 939–947.  <ul class="find" data-citedArticleID="997648" data-doi="10.1099/00207713-47-4-939"><li><a href="http://dx.doi.org/10.1099/00207713-47-4-939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deinococcus+geothermalis+sp.+nov.+and+Deinococcus+murrayi+sp.+nov.%2C+two+extremely+radiation-resistant+and+slightly+thermophilic+species+from+hot+springs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deinococcus+geothermalis+sp.+nov.+and+Deinococcus+murrayi+sp.+nov.%2C+two+extremely+radiation-resistant+and+slightly+thermophilic+species+from+hot+springs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000955-Saarimaa1" id="pone.0000955-Saarimaa1"></a>Saarimaa C, Peltola M, Raulio M, Neu TR, Salkinoja-Salonen MS, et al.  (2006) Characterization of adhesion threads of <em>Deinococcus geothermalis</em> as type IV pili. J Bacteriol  188: 7016–7021.  <ul class="find" data-citedArticleID="997824" data-doi="10.1128/jb.00608-06"><li><a href="http://dx.doi.org/10.1128/jb.00608-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+adhesion+threads+of+Deinococcus+geothermalis+as+type+IV+pili." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+adhesion+threads+of+Deinococcus+geothermalis+as+type+IV+pili.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000955-Kimura1" id="pone.0000955-Kimura1"></a>Kimura H, Asada R, Masta A, Naganuma T (2003) Distribution of microorganisms in the subsurface of the manus basin hydrothermal vent field in Papua New Guinea. Appl Environ Microbiol  69: 644–648.  <ul class="find" data-citedArticleID="997728" data-doi="10.1128/aem.69.1.644-648.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.1.644-648.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+microorganisms+in+the+subsurface+of+the+manus+basin+hydrothermal+vent+field+in+Papua+New+Guinea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+microorganisms+in+the+subsurface+of+the+manus+basin+hydrothermal+vent+field+in+Papua+New+Guinea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000955-Marteinsson1" id="pone.0000955-Marteinsson1"></a>Marteinsson VT, Hauksdottir S, Hobel CF, Kristmannsdottir H, Hreggvidsson GO, et al.  (2001) Phylogenetic diversity analysis of subterranean hot springs in Iceland. Appl Environ Microbiol  67: 4242–4248.  <ul class="find" data-citedArticleID="997784" data-doi="10.1128/aem.67.9.4242-4248.2001"><li><a href="http://dx.doi.org/10.1128/aem.67.9.4242-4248.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+diversity+analysis+of+subterranean+hot+springs+in+Iceland." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+diversity+analysis+of+subterranean+hot+springs+in+Iceland.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000955-Brim1" id="pone.0000955-Brim1"></a>Brim H, Venkateswaran A, Kostandarithes HM, Fredrickson JK, Daly MJ (2003) Engineering <em>Deinococcus geothermalis</em> for bioremediation of high-temperature radioactive waste environments. Appl Environ Microbiol  69: 4575–4582.  <ul class="find" data-citedArticleID="997580" data-doi="10.1128/aem.69.8.4575-4582.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.8.4575-4582.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineering+Deinococcus+geothermalis+for+bioremediation+of+high-temperature+radioactive+waste+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineering+Deinococcus+geothermalis+for+bioremediation+of+high-temperature+radioactive+waste+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000955-Ghosal1" id="pone.0000955-Ghosal1"></a>Ghosal D, Omelchenko MV, Gaidamakova EK, Matrosova VY, Vasilenko A, et al.  (2005) How radiation kills cells: survival of <em>Deinococcus radiodurans</em> and <em>Shewanella oneidensis</em> under oxidative stress. FEMS Microbiol Rev  29: 361–375.  <ul class="find" data-citedArticleID="997676" data-doi="10.1016/j.fmrre.2004.12.007"><li><a href="http://dx.doi.org/10.1016/j.fmrre.2004.12.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+radiation+kills+cells%3A+survival+of+Deinococcus+radiodurans+and+Shewanella+oneidensis+under+oxidative+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+radiation+kills+cells%3A+survival+of+Deinococcus+radiodurans+and+Shewanella+oneidensis+under+oxidative+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000955-Fredrickson1" id="pone.0000955-Fredrickson1"></a>Fredrickson JK, Kostandarithes HM, Li SW, Plymale AE, Daly MJ (2000) Reduction of Fe(III), Cr(VI), U(VI), and Tc(VII) by <em>Deinococcus radiodurans</em> R1. Appl Environ Microbiol  66: 2006–2011.  <ul class="find" data-citedArticleID="997656" data-doi="10.1128/aem.66.5.2006-2011.2000"><li><a href="http://dx.doi.org/10.1128/aem.66.5.2006-2011.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduction+of+Fe%28III%29%2C+Cr%28VI%29%2C+U%28VI%29%2C+and+Tc%28VII%29+by+Deinococcus+radiodurans+R1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduction+of+Fe%28III%29%2C+Cr%28VI%29%2C+U%28VI%29%2C+and+Tc%28VII%29+by+Deinococcus+radiodurans+R1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000955-Brim2" id="pone.0000955-Brim2"></a>Brim H, McFarlan SC, Fredrickson JK, Minton KW, Zhai M, et al.  (2000) Engineering <em>Deinococcus radiodurans</em> for metal remediation in radioactive mixed waste environments. Nat Biotechnol  18: 85–90.  <ul class="find" data-citedArticleID="997582"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Engineering+Deinococcus+radiodurans+for+metal+remediation+in+radioactive+mixed+waste+environments.&amp;auth=&amp;atitle=Engineering+Deinococcus+radiodurans+for+metal+remediation+in+radioactive+mixed+waste+environments." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineering+Deinococcus+radiodurans+for+metal+remediation+in+radioactive+mixed+waste+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineering+Deinococcus+radiodurans+for+metal+remediation+in+radioactive+mixed+waste+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000955-Brim3" id="pone.0000955-Brim3"></a>Brim H, Osborne JP, Kostandarithes HM, Fredrickson JK, Wackett LP, et al.  (2006) <em>Deinococcus radiodurans</em> engineered for complete toluene degradation facilitates Cr(VI) reduction. Microbiology  152: 2469–2477.  <ul class="find" data-citedArticleID="997584" data-doi="10.1099/mic.0.29009-0"><li><a href="http://dx.doi.org/10.1099/mic.0.29009-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deinococcus+radiodurans+engineered+for+complete+toluene+degradation+facilitates+Cr%28VI%29+reduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deinococcus+radiodurans+engineered+for+complete+toluene+degradation+facilitates+Cr%28VI%29+reduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000955-White1" id="pone.0000955-White1"></a>White O, Eisen JA, Heidelberg JF, Hickey EK, Peterson JD, et al.  (1999) Genome Sequence of the Radioresistant Bacterium <em>Deinococcus radiodurans</em> R1. Science  286: 1571–1577.  <ul class="find" data-citedArticleID="997866" data-doi="10.1126/science.286.5444.1571"><li><a href="http://dx.doi.org/10.1126/science.286.5444.1571" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+Sequence+of+the+Radioresistant+Bacterium+Deinococcus+radiodurans+R1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+Sequence+of+the+Radioresistant+Bacterium+Deinococcus+radiodurans+R1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000955-Daly3" id="pone.0000955-Daly3"></a>Daly MJ, Gaidamakova EK, Matrosova VY, Vasilenko A, Zhai M (2007) Protein Oxidation Implicated as the Primary Determinant of Bacterial Radioresistance. PLoS Biol  5: 769–779.  <ul class="find" data-citedArticleID="997610" data-doi="10.1371/journal.pbio.0050092"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0050092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+Oxidation+Implicated+as+the+Primary+Determinant+of+Bacterial+Radioresistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+Oxidation+Implicated+as+the+Primary+Determinant+of+Bacterial+Radioresistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000955-Daly4" id="pone.0000955-Daly4"></a>Daly MJ, Ouyang L, Fuchs P, Minton KW (1994) <em>In vivo</em> damage and <em>recA</em>-dependent repair of plasmid and chromosomal DNA in the radiation-resistant bacterium <em>Deinococcus radiodurans</em>. J Bacteriol  176: 3508–3517.  <ul class="find" data-citedArticleID="997612"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans.&amp;auth=&amp;atitle=In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000955-Gerard1" id="pone.0000955-Gerard1"></a>Gerard E, Jolivet E, Prieur D, Forterre P (2001) DNA protection mechanisms are not involved in the radioresistance of the hyperthermophilic archaea <em>Pyrococcus abyssi</em> and <em>P. furiosus</em>. Mol Genet Genomics  266: 72–78.  <ul class="find" data-citedArticleID="997670" data-doi="10.1007/s004380100520"><li><a href="http://dx.doi.org/10.1007/s004380100520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+protection+mechanisms+are+not+involved+in+the+radioresistance+of+the+hyperthermophilic+archaea+Pyrococcus+abyssi+and+P.+furiosus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+protection+mechanisms+are+not+involved+in+the+radioresistance+of+the+hyperthermophilic+archaea+Pyrococcus+abyssi+and+P.+furiosus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000955-Cox1" id="pone.0000955-Cox1"></a>Cox MM, Battista JR (2005) <em>Deinococcus radiodurans</em>-the consummate survivor. Nat Rev Microbiol  3: 882–892.  <ul class="find" data-citedArticleID="997602" data-doi="10.1038/nrmicro1264"><li><a href="http://dx.doi.org/10.1038/nrmicro1264" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deinococcus+radiodurans-the+consummate+survivor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deinococcus+radiodurans-the+consummate+survivor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000955-Makarova1" id="pone.0000955-Makarova1"></a>Makarova KS, Aravind L, Wolf YI, Tatusov RL, Minton KW, et al.  (2001) Genome of the extremely radiation-resistant bacterium <em>Deinococcus radiodurans</em> viewed from the perspective of comparative genomics. Microbiol Mol Biol Rev  65: 44–79.  <ul class="find" data-citedArticleID="997766" data-doi="10.1128/mmbr.65.1.44-79.2001"><li><a href="http://dx.doi.org/10.1128/mmbr.65.1.44-79.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+of+the+extremely+radiation-resistant+bacterium+Deinococcus+radiodurans+viewed+from+the+perspective+of+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+of+the+extremely+radiation-resistant+bacterium+Deinococcus+radiodurans+viewed+from+the+perspective+of+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000955-Zahradka1" id="pone.0000955-Zahradka1"></a>Zahradka K, Slade D, Bailone A, Sommer S, Averbeck D, et al.  (2006) Reassembly of shattered chromosomes in <em>Deinococcus radiodurans</em>. Nature  443: 569–573.  <ul class="find" data-citedArticleID="997880" data-doi="10.1038/nature05160"><li><a href="http://dx.doi.org/10.1038/nature05160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reassembly+of+shattered+chromosomes+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reassembly+of+shattered+chromosomes+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000955-Daly5" id="pone.0000955-Daly5"></a>Daly MJ (2006) Modulating radiation resistance: Insights based on defenses against reactive oxygen species in the radioresistant bacterium <em>Deinococcus radiodurans</em>. Clin Lab Med  26: 491–504, x.  <ul class="find" data-citedArticleID="997614" data-doi="10.1016/j.cll.2006.03.009"><li><a href="http://dx.doi.org/10.1016/j.cll.2006.03.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulating+radiation+resistance%3A+Insights+based+on+defenses+against+reactive+oxygen+species+in+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulating+radiation+resistance%3A+Insights+based+on+defenses+against+reactive+oxygen+species+in+the+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000955-Zimmerman1" id="pone.0000955-Zimmerman1"></a>Zimmerman JM, Battista JR (2005) A ring-like nucleoid is not necessary for radioresistance in the <em>Deinococcaceae</em>. BMC Microbiol  5: 17.  <ul class="find" data-citedArticleID="997884"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+ring-like+nucleoid+is+not+necessary+for+radioresistance+in+the+Deinococcaceae.&amp;auth=&amp;atitle=A+ring-like+nucleoid+is+not+necessary+for+radioresistance+in+the+Deinococcaceae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+ring-like+nucleoid+is+not+necessary+for+radioresistance+in+the+Deinococcaceae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+ring-like+nucleoid+is+not+necessary+for+radioresistance+in+the+Deinococcaceae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000955-Pavlov1" id="pone.0000955-Pavlov1"></a>Pavlov AK, Kalinin VL, Konstantinov AN, Shelegedin VN, Pavlov AA (2006) Was Earth ever infected by martian biota? Clues from radioresistant bacteria. Astrobiology  6: 911–918.  <ul class="find" data-citedArticleID="997814" data-doi="10.1089/ast.2006.6.911"><li><a href="http://dx.doi.org/10.1089/ast.2006.6.911" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Was+Earth+ever+infected+by+martian+biota%3F+Clues+from+radioresistant+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Was+Earth+ever+infected+by+martian+biota%3F+Clues+from+radioresistant+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000955-Lin1" id="pone.0000955-Lin1"></a>Lin J, Qi R, Aston C, Jing J, Anantharaman TS, et al.  (1999) Whole-genome shotgun optical mapping of <em>Deinococcus radiodurans</em>. Science  285: 1558–1562.  <ul class="find" data-citedArticleID="997758" data-doi="10.1126/science.285.5433.1558"><li><a href="http://dx.doi.org/10.1126/science.285.5433.1558" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+shotgun+optical+mapping+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+shotgun+optical+mapping+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000955-Omelchenko1" id="pone.0000955-Omelchenko1"></a>Omelchenko MV, Wolf YI, Gaidamakova EK, Matrosova VY, Vasilenko A, et al.  (2005) Comparative genomics of <em>Thermus thermophilus</em> and <em>Deinococcus radiodurans</em>: divergent routes of adaptation to thermophily and radiation resistance. BMC Evol Biol  5: 57.  <ul class="find" data-citedArticleID="997812"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+genomics+of+Thermus+thermophilus+and+Deinococcus+radiodurans%3A+divergent+routes+of+adaptation+to+thermophily+and+radiation+resistance.&amp;auth=&amp;atitle=Comparative+genomics+of+Thermus+thermophilus+and+Deinococcus+radiodurans%3A+divergent+routes+of+adaptation+to+thermophily+and+radiation+resistance." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics+of+Thermus+thermophilus+and+Deinococcus+radiodurans%3A+divergent+routes+of+adaptation+to+thermophily+and+radiation+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics+of+Thermus+thermophilus+and+Deinococcus+radiodurans%3A+divergent+routes+of+adaptation+to+thermophily+and+radiation+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000955-Eisen1" id="pone.0000955-Eisen1"></a>Eisen JA, Heidelberg JF, White O, Salzberg SL (2000) Evidence for symmetric chromosomal inversions around the replication origin in bacteria. Genome Biol  1: RESEARCH0011.  <ul class="find" data-citedArticleID="997634" data-doi="10.1186/gb-2000-1-6-research0011"><li><a href="http://dx.doi.org/10.1186/gb-2000-1-6-research0011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+symmetric+chromosomal+inversions+around+the+replication+origin+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+symmetric+chromosomal+inversions+around+the+replication+origin+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000955-Makarova2" id="pone.0000955-Makarova2"></a>Makarova KS, Wolf YI, White O, Minton K, Daly MJ (1999) Short repeats and IS elements in the extremely radiation-resistant bacterium <em>Deinococcus radiodurans</em> and comparison to other bacterial species. Res Microbiol  150: 711–724.  <ul class="find" data-citedArticleID="997768" data-doi="10.1016/s0923-2508(99)00121-7"><li><a href="http://dx.doi.org/10.1016/s0923-2508(99)00121-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Short+repeats+and+IS+elements+in+the+extremely+radiation-resistant+bacterium+Deinococcus+radiodurans+and+comparison+to+other+bacterial+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Short+repeats+and+IS+elements+in+the+extremely+radiation-resistant+bacterium+Deinococcus+radiodurans+and+comparison+to+other+bacterial+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000955-Makarova3" id="pone.0000955-Makarova3"></a>Makarova KS, Aravind L, Galperin MY, Grishin NV, Tatusov RL, et al.  (1999) Comparative genomics of the Archaea (Euryarchaeota): evolution of conserved protein families, the stable core, and the variable shell. Genome Res  9: 608–628.  <ul class="find" data-citedArticleID="997770"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+genomics+of+the+Archaea+%28Euryarchaeota%29%3A+evolution+of+conserved+protein+families%2C+the+stable+core%2C+and+the+variable+shell.&amp;auth=&amp;atitle=Comparative+genomics+of+the+Archaea+%28Euryarchaeota%29%3A+evolution+of+conserved+protein+families%2C+the+stable+core%2C+and+the+variable+shell." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics+of+the+Archaea+%28Euryarchaeota%29%3A+evolution+of+conserved+protein+families%2C+the+stable+core%2C+and+the+variable+shell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics+of+the+Archaea+%28Euryarchaeota%29%3A+evolution+of+conserved+protein+families%2C+the+stable+core%2C+and+the+variable+shell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000955-Sen1" id="pone.0000955-Sen1"></a>Sen D, Gilbert W (1988) Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis. Nature  334: 364–366.  <ul class="find" data-citedArticleID="997836" data-doi="10.1038/334364a0"><li><a href="http://dx.doi.org/10.1038/334364a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Formation+of+parallel+four-stranded+complexes+by+guanine-rich+motifs+in+DNA+and+its+implications+for+meiosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Formation+of+parallel+four-stranded+complexes+by+guanine-rich+motifs+in+DNA+and+its+implications+for+meiosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000955-Mojica1" id="pone.0000955-Mojica1"></a>Mojica FJ, Diez-Villasenor C, Soria E, Juez G (2000) Biological significance of a family of regularly spaced repeats in the genomes of Archaea, Bacteria and mitochondria. Mol Microbiol  36: 244–246.  <ul class="find" data-citedArticleID="997796" data-doi="10.1046/j.1365-2958.2000.01838.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01838.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biological+significance+of+a+family+of+regularly+spaced+repeats+in+the+genomes+of+Archaea%2C+Bacteria+and+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biological+significance+of+a+family+of+regularly+spaced+repeats+in+the+genomes+of+Archaea%2C+Bacteria+and+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000955-Barrangou1" id="pone.0000955-Barrangou1"></a>Barrangou R, Fremaux C, Deveau H, Richards M, Boyaval P, et al.  (2007) CRISPR provides acquired resistance against viruses in prokaryotes. Science  315: 1709–1712.  <ul class="find" data-citedArticleID="997558" data-doi="10.1126/science.1138140"><li><a href="http://dx.doi.org/10.1126/science.1138140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CRISPR+provides+acquired+resistance+against+viruses+in+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CRISPR+provides+acquired+resistance+against+viruses+in+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000955-Makarova4" id="pone.0000955-Makarova4"></a>Makarova KS, Grishin NV, Shabalina SA, Wolf YI, Koonin EV (2006) A putative RNA-interference-based immune system in prokaryotes: computational analysis of the predicted enzymatic machinery, functional analogies with eukaryotic RNAi, and hypothetical mechanisms of action. Biol Direct  1: 7.  <ul class="find" data-citedArticleID="997772"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+putative+RNA-interference-based+immune+system+in+prokaryotes%3A+computational+analysis+of+the+predicted+enzymatic+machinery%2C+functional+analogies+with+eukaryotic+RNAi%2C+and+hypothetical+mechanisms+of+action.&amp;auth=&amp;atitle=A+putative+RNA-interference-based+immune+system+in+prokaryotes%3A+computational+analysis+of+the+predicted+enzymatic+machinery%2C+functional+analogies+with+eukaryotic+RNAi%2C+and+hypothetical+mechanisms+of+action." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+putative+RNA-interference-based+immune+system+in+prokaryotes%3A+computational+analysis+of+the+predicted+enzymatic+machinery%2C+functional+analogies+with+eukaryotic+RNAi%2C+and+hypothetical+mechanisms+of+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+putative+RNA-interference-based+immune+system+in+prokaryotes%3A+computational+analysis+of+the+predicted+enzymatic+machinery%2C+functional+analogies+with+eukaryotic+RNAi%2C+and+hypothetical+mechanisms+of+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000955-Tatusov1" id="pone.0000955-Tatusov1"></a>Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, et al.  (2003) The COG database: an updated version includes eukaryotes. BMC Bioinformatics  4: 41.  <ul class="find" data-citedArticleID="997852"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+COG+database%3A+an+updated+version+includes+eukaryotes.&amp;auth=&amp;atitle=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+COG+database%3A+an+updated+version+includes+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000955-Tatusov2" id="pone.0000955-Tatusov2"></a>Tatusov RL, Koonin EV, Lipman DJ (1997) A genomic perspective on protein families. Science  278: 631–637.  <ul class="find" data-citedArticleID="997854" data-doi="10.1126/science.278.5338.631"><li><a href="http://dx.doi.org/10.1126/science.278.5338.631" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+perspective+on+protein+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+perspective+on+protein+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000955-Yokoyama1" id="pone.0000955-Yokoyama1"></a>Yokoyama K, Ishijima SA, Clowney L, Koike H, Aramaki H, et al.  (2006) Feast/famine regulatory proteins (FFRPs): <em>Escherichia coli</em> Lrp, AsnC and related archaeal transcription factors. FEMS Microbiol Rev  30: 89–108.  <ul class="find" data-citedArticleID="997876" data-doi="10.1111/j.1574-6976.2005.00005.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6976.2005.00005.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feast%2Ffamine+regulatory+proteins+%28FFRPs%29%3A+Escherichia+coli+Lrp%2C+AsnC+and+related+archaeal+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Feast%2Ffamine+regulatory+proteins+%28FFRPs%29%3A+Escherichia+coli+Lrp%2C+AsnC+and+related+archaeal+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000955-Gerischer1" id="pone.0000955-Gerischer1"></a>Gerischer U (2002) Specific and global regulation of genes associated with the degradation of aromatic compounds in bacteria. J Mol Microbiol Biotechnol  4: 111–121.  <ul class="find" data-citedArticleID="997674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Specific+and+global+regulation+of+genes+associated+with+the+degradation+of+aromatic+compounds+in+bacteria.&amp;auth=&amp;atitle=Specific+and+global+regulation+of+genes+associated+with+the+degradation+of+aromatic+compounds+in+bacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+and+global+regulation+of+genes+associated+with+the+degradation+of+aromatic+compounds+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+and+global+regulation+of+genes+associated+with+the+degradation+of+aromatic+compounds+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000955-MolinaHenares1" id="pone.0000955-MolinaHenares1"></a>Molina-Henares AJ, Krell T, Eugenia Guazzaroni M, Segura A, Ramos JL (2006) Members of the IclR family of bacterial transcriptional regulators function as activators and/or repressors. FEMS Microbiol Rev  30: 157–186.  <ul class="find" data-citedArticleID="997798" data-doi="10.1111/j.1574-6976.2005.00008.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6976.2005.00008.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Members+of+the+IclR+family+of+bacterial+transcriptional+regulators+function+as+activators+and%2For+repressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Members+of+the+IclR+family+of+bacterial+transcriptional+regulators+function+as+activators+and%2For+repressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000955-Ramos1" id="pone.0000955-Ramos1"></a>Ramos JL, Martinez-Bueno M, Molina-Henares AJ, Teran W, Watanabe K, et al.  (2005) The TetR family of transcriptional repressors. Microbiol Mol Biol Rev  69: 326–356.  <ul class="find" data-citedArticleID="997822" data-doi="10.1128/mmbr.69.2.326-356.2005"><li><a href="http://dx.doi.org/10.1128/mmbr.69.2.326-356.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+TetR+family+of+transcriptional+repressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+TetR+family+of+transcriptional+repressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000955-Hobman1" id="pone.0000955-Hobman1"></a>Hobman JL, Wilkie J, Brown NL (2005) A design for life: prokaryotic metal-binding MerR family regulators. Biometals  18: 429–436.  <ul class="find" data-citedArticleID="997700" data-doi="10.1007/s10534-005-3717-7"><li><a href="http://dx.doi.org/10.1007/s10534-005-3717-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+design+for+life%3A+prokaryotic+metal-binding+MerR+family+regulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+design+for+life%3A+prokaryotic+metal-binding+MerR+family+regulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000955-Bearson1" id="pone.0000955-Bearson1"></a>Bearson SM, Albrecht JA, Gunsalus RP (2002) Oxygen and nitrate-dependent regulation of <em>dmsABC</em> operon expression in <em>Escherichia coli</em>: sites for Fnr and NarL protein interactions. BMC Microbiol  2: 13.  <ul class="find" data-citedArticleID="997562"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Oxygen+and+nitrate-dependent+regulation+of+dmsABC+operon+expression+in+Escherichia+coli%3A+sites+for+Fnr+and+NarL+protein+interactions.&amp;auth=&amp;atitle=Oxygen+and+nitrate-dependent+regulation+of+dmsABC+operon+expression+in+Escherichia+coli%3A+sites+for+Fnr+and+NarL+protein+interactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxygen+and+nitrate-dependent+regulation+of+dmsABC+operon+expression+in+Escherichia+coli%3A+sites+for+Fnr+and+NarL+protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxygen+and+nitrate-dependent+regulation+of+dmsABC+operon+expression+in+Escherichia+coli%3A+sites+for+Fnr+and+NarL+protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000955-Sandrini1" id="pone.0000955-Sandrini1"></a>Sandrini MP, Clausen AR, Munch-Petersen B, Piskur J (2006) Thymidine kinase diversity in bacteria. Nucleosides Nucleotides Nucleic Acids  25: 1153–1158.  <ul class="find" data-citedArticleID="997826" data-doi="10.1080/15257770600894469"><li><a href="http://dx.doi.org/10.1080/15257770600894469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thymidine+kinase+diversity+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thymidine+kinase+diversity+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000955-Xi1" id="pone.0000955-Xi1"></a>Xi H, Schneider BL, Reitzer L (2000) Purine catabolism in <em>Escherichia coli</em> and function of xanthine dehydrogenase in purine salvage. J Bacteriol  182: 5332–5341.  <ul class="find" data-citedArticleID="997872" data-doi="10.1128/jb.182.19.5332-5341.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.19.5332-5341.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purine+catabolism+in+Escherichia+coli+and+function+of+xanthine+dehydrogenase+in+purine+salvage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purine+catabolism+in+Escherichia+coli+and+function+of+xanthine+dehydrogenase+in+purine+salvage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000955-Knofel1" id="pone.0000955-Knofel1"></a>Knofel T, Strater N (1999) X-ray structure of the <em>Escherichia coli</em> periplasmic 5′-nucleotidase containing a dimetal catalytic site. Nat Struct Biol  6: 448–453.  <ul class="find" data-citedArticleID="997736"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=X-ray+structure+of+the+Escherichia+coli+periplasmic+5%3F-nucleotidase+containing+a+dimetal+catalytic+site.&amp;auth=&amp;atitle=X-ray+structure+of+the+Escherichia+coli+periplasmic+5%3F-nucleotidase+containing+a+dimetal+catalytic+site." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=X-ray+structure+of+the+Escherichia+coli+periplasmic+5%E2%80%B2-nucleotidase+containing+a+dimetal+catalytic+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22X-ray+structure+of+the+Escherichia+coli+periplasmic+5%E2%80%B2-nucleotidase+containing+a+dimetal+catalytic+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000955-Liu1" id="pone.0000955-Liu1"></a>Liu Y, Zhou J, Omelchenko MV, Beliaev AS, Venkateswaran A, et al.  (2003) Transcriptome dynamics of <em>Deinococcus radiodurans</em> recovering from ionizing radiation. Proc Natl Acad Sci U S A  100: 4191–4196.  <ul class="find" data-citedArticleID="997764" data-doi="10.1073/pnas.0630387100"><li><a href="http://dx.doi.org/10.1073/pnas.0630387100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+dynamics+of+Deinococcus+radiodurans+recovering+from+ionizing+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+dynamics+of+Deinococcus+radiodurans+recovering+from+ionizing+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000955-Galperin1" id="pone.0000955-Galperin1"></a>Galperin MY, Moroz OV, Wilson KS, Murzin AG (2006) House cleaning, a part of good housekeeping. Mol Microbiol  59: 5–19.  <ul class="find" data-citedArticleID="997664" data-doi="10.1111/j.1365-2958.2005.04950.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04950.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=House+cleaning%2C+a+part+of+good+housekeeping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22House+cleaning%2C+a+part+of+good+housekeeping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000955-Shallom1" id="pone.0000955-Shallom1"></a>Shallom D, Shoham Y (2003) Microbial hemicellulases. Curr Opin Microbiol  6: 219–228.  <ul class="find" data-citedArticleID="997840" data-doi="10.1016/s1369-5274(03)00056-0"><li><a href="http://dx.doi.org/10.1016/s1369-5274(03)00056-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+hemicellulases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+hemicellulases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000955-Kolari1" id="pone.0000955-Kolari1"></a>Kolari M, Nuutinen J, Rainey FA, Salkinoja-Salonen MS (2003) Colored moderately thermophilic bacteria in paper-machine biofilms. J Ind Microbiol Biotechnol  30: 225–238.  <ul class="find" data-citedArticleID="997742"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Colored+moderately+thermophilic+bacteria+in+paper-machine+biofilms.&amp;auth=&amp;atitle=Colored+moderately+thermophilic+bacteria+in+paper-machine+biofilms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Colored+moderately+thermophilic+bacteria+in+paper-machine+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Colored+moderately+thermophilic+bacteria+in+paper-machine+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000955-Vaisanen1" id="pone.0000955-Vaisanen1"></a>Vaisanen OM, Weber A, Bennasar A, Rainey FA, Busse HJ, et al.  (1998) Microbial communities of printing paper machines. J Appl Microbiol  84: 1069–1084.  <ul class="find" data-citedArticleID="997862" data-doi="10.1046/j.1365-2672.1998.00447.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2672.1998.00447.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+communities+of+printing+paper+machines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+communities+of+printing+paper+machines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000955-Venkateswaran1" id="pone.0000955-Venkateswaran1"></a>Venkateswaran A, McFarlan SC, Ghostal D, Minton KW, Vasilenko A, et al.  (2000) Physiologic determinants of radiation resistance in <em>Deinococcus radiodurans</em>. ApplEnvironMicrobiol  66: 2620–2626.  <ul class="find" data-citedArticleID="997864" data-doi="10.1128/aem.66.6.2620-2626.2000"><li><a href="http://dx.doi.org/10.1128/aem.66.6.2620-2626.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physiologic+determinants+of+radiation+resistance+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physiologic+determinants+of+radiation+resistance+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000955-Holland1" id="pone.0000955-Holland1"></a>Holland AD, Rothfuss HM, Lidstrom ME (2006) Development of a defined medium supporting rapid growth for <em>Deinococcus radiodurans</em> and analysis of metabolic capacities. Appl Microbiol Biotechnol  72: 1074–1082.  <ul class="find" data-citedArticleID="997702" data-doi="10.1007/s00253-006-0399-1"><li><a href="http://dx.doi.org/10.1007/s00253-006-0399-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+defined+medium+supporting+rapid+growth+for+Deinococcus+radiodurans+and+analysis+of+metabolic+capacities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+defined+medium+supporting+rapid+growth+for+Deinococcus+radiodurans+and+analysis+of+metabolic+capacities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000955-Frankenberg1" id="pone.0000955-Frankenberg1"></a>Frankenberg N, Moser J, Jahn D (2003) Bacterial heme biosynthesis and its biotechnological application. Appl Microbiol Biotechnol  63: 115–127.  <ul class="find" data-citedArticleID="997654" data-doi="10.1007/s00253-003-1432-2"><li><a href="http://dx.doi.org/10.1007/s00253-003-1432-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+heme+biosynthesis+and+its+biotechnological+application." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+heme+biosynthesis+and+its+biotechnological+application.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000955-Pereira1" id="pone.0000955-Pereira1"></a>Pereira MM, Santana M, Teixeira M (2001) A novel scenario for the evolution of haem-copper oxygen reductases. Biochim Biophys Acta  1505: 185–208.  <ul class="find" data-citedArticleID="997818" data-doi="10.1016/s0005-2728(01)00169-4"><li><a href="http://dx.doi.org/10.1016/s0005-2728(01)00169-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+scenario+for+the+evolution+of+haem-copper+oxygen+reductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+scenario+for+the+evolution+of+haem-copper+oxygen+reductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000955-Junemann1" id="pone.0000955-Junemann1"></a>Junemann S (1997) Cytochrome bd terminal oxidase. Biochim Biophys Acta  1321: 107–127.  <ul class="find" data-citedArticleID="997720" data-doi="10.1016/s0005-2728(97)00046-7"><li><a href="http://dx.doi.org/10.1016/s0005-2728(97)00046-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytochrome+bd+terminal+oxidase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytochrome+bd+terminal+oxidase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000955-Harborne1" id="pone.0000955-Harborne1"></a>Harborne NR, Griffiths L, Busby SJ, Cole JA (1992) Transcriptional control, translation and function of the products of the five open reading frames of the <em>Escherichia coli nir</em> operon. Mol Microbiol  6: 2805–2813.  <ul class="find" data-citedArticleID="997692" data-doi="10.1111/j.1365-2958.1992.tb01460.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1992.tb01460.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+control%2C+translation+and+function+of+the+products+of+the+five+open+reading+frames+of+the+Escherichia+coli+nir+operon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+control%2C+translation+and+function+of+the+products+of+the+five+open+reading+frames+of+the+Escherichia+coli+nir+operon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000955-Cabello1" id="pone.0000955-Cabello1"></a>Cabello P, Pino C, Olmo-Mira MF, Castillo F, Roldan MD, et al.  (2004) Hydroxylamine assimilation by <em>Rhodobacter capsulatus</em> E1F1. requirement of the <em>hcp</em> gene (hybrid cluster protein) located in the nitrate assimilation <em>nas</em> gene region for hydroxylamine reduction. J Biol Chem  279: 45485–45494.  <ul class="find" data-citedArticleID="997592" data-doi="10.1074/jbc.m404417200"><li><a href="http://dx.doi.org/10.1074/jbc.m404417200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydroxylamine+assimilation+by+Rhodobacter+capsulatus+E1F1.+requirement+of+the+hcp+gene+%28hybrid+cluster+protein%29+located+in+the+nitrate+assimilation+nas+gene+region+for+hydroxylamine+reduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydroxylamine+assimilation+by+Rhodobacter+capsulatus+E1F1.+requirement+of+the+hcp+gene+%28hybrid+cluster+protein%29+located+in+the+nitrate+assimilation+nas+gene+region+for+hydroxylamine+reduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000955-Zumft1" id="pone.0000955-Zumft1"></a>Zumft WG (1997) Cell biology and molecular basis of denitrification. Microbiol Mol Biol Rev  61: 533–616.  <ul class="find" data-citedArticleID="997886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell+biology+and+molecular+basis+of+denitrification.&amp;auth=&amp;atitle=Cell+biology+and+molecular+basis+of+denitrification." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+biology+and+molecular+basis+of+denitrification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+biology+and+molecular+basis+of+denitrification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000955-Nakamura1" id="pone.0000955-Nakamura1"></a>Nakamura K, Kawabata T, Yura K, Go N (2003) Novel types of two-domain multi-copper oxidases: possible missing links in the evolution. FEBS Lett  553: 239–244.  <ul class="find" data-citedArticleID="997802" data-doi="10.1016/s0014-5793(03)01000-7"><li><a href="http://dx.doi.org/10.1016/s0014-5793(03)01000-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+types+of+two-domain+multi-copper+oxidases%3A+possible+missing+links+in+the+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+types+of+two-domain+multi-copper+oxidases%3A+possible+missing+links+in+the+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000955-Hille1" id="pone.0000955-Hille1"></a>Hille R (2002) Molybdenum and tungsten in biology. Trends Biochem Sci  27: 360–367.  <ul class="find" data-citedArticleID="997698" data-doi="10.1016/s0968-0004(02)02107-2"><li><a href="http://dx.doi.org/10.1016/s0968-0004(02)02107-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molybdenum+and+tungsten+in+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molybdenum+and+tungsten+in+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000955-Makarova5" id="pone.0000955-Makarova5"></a>Makarova KS, Aravind L, Daly MJ, Koonin EV (2000) Specific expansion of protein families in the radioresistant bacterium <em>Deinococcus radiodurans</em>. Genetica  108: 25–34.  <ul class="find" data-citedArticleID="997774" data-doi="10.1023/a:1004035424657"><li><a href="http://dx.doi.org/10.1023/a:1004035424657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+expansion+of+protein+families+in+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+expansion+of+protein+families+in+the+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000955-Fisher1" id="pone.0000955-Fisher1"></a>Fisher DI, Cartwright JL, McLennan AG (2006) Characterization of the Mn(2+)-stimulated (di)adenosine polyphosphate hydrolase encoded by the <em>Deinococcus radiodurans</em> DR2356 nudix gene. Arch Microbiol  186: 415–424.  <ul class="find" data-citedArticleID="997650" data-doi="10.1007/s00203-006-0155-z"><li><a href="http://dx.doi.org/10.1007/s00203-006-0155-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+Mn%282%2B%29-stimulated+%28di%29adenosine+polyphosphate+hydrolase+encoded+by+the+Deinococcus+radiodurans+DR2356+nudix+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+Mn%282%2B%29-stimulated+%28di%29adenosine+polyphosphate+hydrolase+encoded+by+the+Deinococcus+radiodurans+DR2356+nudix+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000955-Hou1" id="pone.0000955-Hou1"></a>Hou B, Xu ZW, Yang CW, Gao Y, Zhao SF, et al.  (2007) Protective effects of inosine on mice subjected to lethal total-body ionizing irradiation. J Radiat Res (Tokyo)  48: 57–62.  <ul class="find" data-citedArticleID="997704" data-doi="10.1269/jrr.06067"><li><a href="http://dx.doi.org/10.1269/jrr.06067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protective+effects+of+inosine+on+mice+subjected+to+lethal+total-body+ionizing+irradiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protective+effects+of+inosine+on+mice+subjected+to+lethal+total-body+ionizing+irradiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000955-Azeddoug1" id="pone.0000955-Azeddoug1"></a>Azeddoug H, Reysset G (1994) Cloning and sequencing of a chromosomal fragment from <em>Clostridium acetobutylicum</em> strain ABKn8 conferring chemical-damaging agents and UV resistance to <em>E. coli recA</em> strains. Curr Microbiol  29: 229–235.  <ul class="find" data-citedArticleID="997556" data-doi="10.1007/bf01570159"><li><a href="http://dx.doi.org/10.1007/bf01570159" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+sequencing+of+a+chromosomal+fragment+from+Clostridium+acetobutylicum+strain+ABKn8+conferring+chemical-damaging+agents+and+UV+resistance+to+E.+coli+recA+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+sequencing+of+a+chromosomal+fragment+from+Clostridium+acetobutylicum+strain+ABKn8+conferring+chemical-damaging+agents+and+UV+resistance+to+E.+coli+recA+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000955-Jobling1" id="pone.0000955-Jobling1"></a>Jobling MG, Ritchie DA (1988) The nucleotide sequence of a plasmid determinant for resistance to tellurium anions. Gene  66: 245–258.  <ul class="find" data-citedArticleID="997718" data-doi="10.1016/0378-1119(88)90361-7"><li><a href="http://dx.doi.org/10.1016/0378-1119(88)90361-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nucleotide+sequence+of+a+plasmid+determinant+for+resistance+to+tellurium+anions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nucleotide+sequence+of+a+plasmid+determinant+for+resistance+to+tellurium+anions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000955-Kitajima1" id="pone.0000955-Kitajima1"></a>Kitajima S, Sato F (1999) Plant pathogenesis-related proteins: molecular mechanisms of gene expression and protein function. J Biochem (Tokyo)  125: 1–8.  <ul class="find" data-citedArticleID="997730" data-doi="10.1093/oxfordjournals.jbchem.a022244"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a022244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+pathogenesis-related+proteins%3A+molecular+mechanisms+of+gene+expression+and+protein+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+pathogenesis-related+proteins%3A+molecular+mechanisms+of+gene+expression+and+protein+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000955-Miura1" id="pone.0000955-Miura1"></a>Miura S, Zou W, Ueda M, Tanaka A (2000) Screening of genes involved in isooctane tolerance in <em>Saccharomyces cerevisiae</em> by using mRNA differential display. Appl Environ Microbiol  66: 4883–4889.  <ul class="find" data-citedArticleID="997794" data-doi="10.1128/aem.66.11.4883-4889.2000"><li><a href="http://dx.doi.org/10.1128/aem.66.11.4883-4889.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Screening+of+genes+involved+in+isooctane+tolerance+in+Saccharomyces+cerevisiae+by+using+mRNA+differential+display." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Screening+of+genes+involved+in+isooctane+tolerance+in+Saccharomyces+cerevisiae+by+using+mRNA+differential+display.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000955-Brinkrolf1" id="pone.0000955-Brinkrolf1"></a>Brinkrolf K, Brune I, Tauch A (2006) Transcriptional regulation of catabolic pathways for aromatic compounds in <em>Corynebacterium glutamicum</em>. Genet Mol Res  5: 773–789.  <ul class="find" data-citedArticleID="997586"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptional+regulation+of+catabolic+pathways+for+aromatic+compounds+in+Corynebacterium+glutamicum.&amp;auth=&amp;atitle=Transcriptional+regulation+of+catabolic+pathways+for+aromatic+compounds+in+Corynebacterium+glutamicum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulation+of+catabolic+pathways+for+aromatic+compounds+in+Corynebacterium+glutamicum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulation+of+catabolic+pathways+for+aromatic+compounds+in+Corynebacterium+glutamicum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000955-Gury1" id="pone.0000955-Gury1"></a>Gury J, Barthelmebs L, Tran NP, Divies C, Cavin JF (2004) Cloning, deletion, and characterization of PadR, the transcriptional repressor of the phenolic acid decarboxylase-encoding <em>padA</em> gene of <em>Lactobacillus plantarum</em>. Appl Environ Microbiol  70: 2146–2153.  <ul class="find" data-citedArticleID="997682" data-doi="10.1128/aem.70.4.2146-2153.2004"><li><a href="http://dx.doi.org/10.1128/aem.70.4.2146-2153.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning%2C+deletion%2C+and+characterization+of+PadR%2C+the+transcriptional+repressor+of+the+phenolic+acid+decarboxylase-encoding+padA+gene+of+Lactobacillus+plantarum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning%2C+deletion%2C+and+characterization+of+PadR%2C+the+transcriptional+repressor+of+the+phenolic+acid+decarboxylase-encoding+padA+gene+of+Lactobacillus+plantarum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000955-Huillet1" id="pone.0000955-Huillet1"></a>Huillet E, Velge P, Vallaeys T, Pardon P (2006) LadR, a new PadR-related transcriptional regulator from <em>Listeria monocytogenes</em>, negatively regulates the expression of the multidrug efflux pump MdrL. FEMS Microbiol Lett  254: 87–94.  <ul class="find" data-citedArticleID="997712" data-doi="10.1111/j.1574-6968.2005.00014.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2005.00014.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LadR%2C+a+new+PadR-related+transcriptional+regulator+from+Listeria+monocytogenes%2C+negatively+regulates+the+expression+of+the+multidrug+efflux+pump+MdrL." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LadR%2C+a+new+PadR-related+transcriptional+regulator+from+Listeria+monocytogenes%2C+negatively+regulates+the+expression+of+the+multidrug+efflux+pump+MdrL.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000955-Kobayashi1" id="pone.0000955-Kobayashi1"></a>Kobayashi I, Tamura T, Sghaier H, Narumi I, Yamaguchi S, et al.  (2006) Characterization of monofunctional catalase KatA from radioresistant bacterium <em>Deinococcus radiodurans</em>. J Biosci Bioeng  101: 315–321.  <ul class="find" data-citedArticleID="997738" data-doi="10.1263/jbb.101.315"><li><a href="http://dx.doi.org/10.1263/jbb.101.315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+monofunctional+catalase+KatA+from+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+monofunctional+catalase+KatA+from+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000955-Yun1" id="pone.0000955-Yun1"></a>Yun EJ, Lee YN (2000) Production of two different catalase-peroxidases by <em>Deinococcus radiophilus</em>. FEMS Microbiol Lett  184: 155–159.  <ul class="find" data-citedArticleID="997878" data-doi="10.1111/j.1574-6968.2000.tb09007.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2000.tb09007.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+of+two+different+catalase-peroxidases+by+Deinococcus+radiophilus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+of+two+different+catalase-peroxidases+by+Deinococcus+radiophilus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000955-Imlay1" id="pone.0000955-Imlay1"></a>Imlay JA (2003) Pathways of oxidative damage. Annu Rev Microbiol  57: 395–418.  <ul class="find" data-citedArticleID="997714" data-doi="10.1146/annurev.micro.57.030502.090938"><li><a href="http://dx.doi.org/10.1146/annurev.micro.57.030502.090938" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathways+of+oxidative+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathways+of+oxidative+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000955-Nurizzo1" id="pone.0000955-Nurizzo1"></a>Nurizzo D, Shewry SC, Perlin MH, Brown SA, Dholakia JN, et al.  (2003) The crystal structure of aminoglycoside-3′-phosphotransferase-IIa​,an enzyme responsible for antibiotic resistance. J Mol Biol  327: 491–506.  <ul class="find" data-citedArticleID="997810" data-doi="10.1016/s0022-2836(03)00121-9"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00121-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+crystal+structure+of+aminoglycoside-3%E2%80%B2-phosphotransferase-IIa%2C+an+enzyme+responsible+for+antibiotic+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+crystal+structure+of+aminoglycoside-3%E2%80%B2-phosphotransferase-IIa%2C+an+enzyme+responsible+for+antibiotic+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000955-Markillie1" id="pone.0000955-Markillie1"></a>Markillie LM, Varnum SM, Hradecky P, Wong KK (1999) Targeted mutagenesis by duplication insertion in the radioresistant bacterium <em>Deinococcus radiodurans</em>: radiation sensitivities of catalase (<em>katA</em>) and superoxide dismutase (<em>sodA</em>) mutants. J Bacteriol  181: 666–669.  <ul class="find" data-citedArticleID="997782"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants.&amp;auth=&amp;atitle=Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+mutagenesis+by+duplication+insertion+in+the+radioresistant+bacterium+Deinococcus+radiodurans%3A+radiation+sensitivities+of+catalase+%28katA%29+and+superoxide+dismutase+%28sodA%29+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000955-Daly6" id="pone.0000955-Daly6"></a>Daly MJ, Ling O, Minton KW (1994) Interplasmidic recombination following irradiation of the radioresistant bacterium <em>Deinococcus radiodurans</em>. J Bacteriol  176: 7506–7515.  <ul class="find" data-citedArticleID="997616"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Interplasmidic+recombination+following+irradiation+of+the+radioresistant+bacterium+Deinococcus+radiodurans.&amp;auth=&amp;atitle=Interplasmidic+recombination+following+irradiation+of+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interplasmidic+recombination+following+irradiation+of+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interplasmidic+recombination+following+irradiation+of+the+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000955-Daly7" id="pone.0000955-Daly7"></a>Daly MJ, Minton KW (1995) Interchromosomal recombination in the extremely radioresistant bacterium <em>Deinococcus radiodurans</em>. J Bacteriol  177: 5495–5505.  <ul class="find" data-citedArticleID="997618"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Interchromosomal+recombination+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans.&amp;auth=&amp;atitle=Interchromosomal+recombination+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interchromosomal+recombination+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interchromosomal+recombination+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000955-Daly8" id="pone.0000955-Daly8"></a>Daly MJ, Minton KW (1996) An alternative pathway of recombination of chromosomal fragments precedes <em>recA</em>-dependent recombination in the radioresistant bacterium <em>Deinococcus radiodurans</em>. J Bacteriol  178: 4461–4471.  <ul class="find" data-citedArticleID="997620"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+alternative+pathway+of+recombination+of+chromosomal+fragments+precedes+recA-dependent+recombination+in+the+radioresistant+bacterium+Deinococcus+radiodurans.&amp;auth=&amp;atitle=An+alternative+pathway+of+recombination+of+chromosomal+fragments+precedes+recA-dependent+recombination+in+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+alternative+pathway+of+recombination+of+chromosomal+fragments+precedes+recA-dependent+recombination+in+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+alternative+pathway+of+recombination+of+chromosomal+fragments+precedes+recA-dependent+recombination+in+the+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000955-Daly9" id="pone.0000955-Daly9"></a>Daly MJ, Minton KW (1997) Recombination between a resident plasmid and the chromosome following irradiation of the radioresistant bacterium <em>Deinococcus radiodurans</em>. Gene  187: 225–229.  <ul class="find" data-citedArticleID="997622" data-doi="10.1016/s0378-1119(96)00755-x"><li><a href="http://dx.doi.org/10.1016/s0378-1119(96)00755-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+between+a+resident+plasmid+and+the+chromosome+following+irradiation+of+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+between+a+resident+plasmid+and+the+chromosome+following+irradiation+of+the+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000955-Gallegos1" id="pone.0000955-Gallegos1"></a>Gallegos MT, Schleif R, Bairoch A, Hofmann K, Ramos JL (1997) Arac/XylS family of transcriptional regulators. Microbiol Mol Biol Rev  61: 393–410.  <ul class="find" data-citedArticleID="997662"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arac%2FXylS+family+of+transcriptional+regulators.&amp;auth=&amp;atitle=Arac%2FXylS+family+of+transcriptional+regulators." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arac%2FXylS+family+of+transcriptional+regulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arac%2FXylS+family+of+transcriptional+regulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000955-Sorensen1" id="pone.0000955-Sorensen1"></a>Sorensen KI, Hove-Jensen B (1996) Ribose catabolism of <em>Escherichia coli</em>: characterization of the <em>rpiB</em> gene encoding ribose phosphate isomerase B and of the <em>rpiR</em> gene, which is involved in regulation of <em>rpiB</em> expression. J Bacteriol  178: 1003–1011.  <ul class="find" data-citedArticleID="997846"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Ribose+catabolism+of+Escherichia+coli%3A+characterization+of+the+rpiB+gene+encoding+ribose+phosphate+isomerase+B+and+of+the+rpiR+gene%2C+which+is+involved+in+regulation+of+rpiB+expression.&amp;auth=&amp;atitle=Ribose+catabolism+of+Escherichia+coli%3A+characterization+of+the+rpiB+gene+encoding+ribose+phosphate+isomerase+B+and+of+the+rpiR+gene%2C+which+is+involved+in+regulation+of+rpiB+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribose+catabolism+of+Escherichia+coli%3A+characterization+of+the+rpiB+gene+encoding+ribose+phosphate+isomerase+B+and+of+the+rpiR+gene%2C+which+is+involved+in+regulation+of+rpiB+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribose+catabolism+of+Escherichia+coli%3A+characterization+of+the+rpiB+gene+encoding+ribose+phosphate+isomerase+B+and+of+the+rpiR+gene%2C+which+is+involved+in+regulation+of+rpiB+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000955-Labie1" id="pone.0000955-Labie1"></a>Labie C, Bouche F, Bouche JP (1989) Isolation and mapping of <em>Escherichia coli</em> mutations conferring resistance to division inhibition protein DicB. J Bacteriol  171: 4315–4319.  <ul class="find" data-citedArticleID="997750"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+and+mapping+of+Escherichia+coli+mutations+conferring+resistance+to+division+inhibition+protein+DicB.&amp;auth=&amp;atitle=Isolation+and+mapping+of+Escherichia+coli+mutations+conferring+resistance+to+division+inhibition+protein+DicB." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+mapping+of+Escherichia+coli+mutations+conferring+resistance+to+division+inhibition+protein+DicB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+mapping+of+Escherichia+coli+mutations+conferring+resistance+to+division+inhibition+protein+DicB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000955-Pennella1" id="pone.0000955-Pennella1"></a>Pennella MA, Giedroc DP (2005) Structural determinants of metal selectivity in prokaryotic metal-responsive transcriptional regulators. Biometals  18: 413–428.  <ul class="find" data-citedArticleID="997816" data-doi="10.1007/s10534-005-3716-8"><li><a href="http://dx.doi.org/10.1007/s10534-005-3716-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+determinants+of+metal+selectivity+in+prokaryotic+metal-responsive+transcriptional+regulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+determinants+of+metal+selectivity+in+prokaryotic+metal-responsive+transcriptional+regulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000955-Buts1" id="pone.0000955-Buts1"></a>Buts L, Lah J, Dao-Thi MH, Wyns L, Loris R (2005) Toxin-antitoxin modules as bacterial metabolic stress managers. Trends Biochem Sci  30: 672–679.  <ul class="find" data-citedArticleID="997590" data-doi="10.1016/j.tibs.2005.10.004"><li><a href="http://dx.doi.org/10.1016/j.tibs.2005.10.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toxin-antitoxin+modules+as+bacterial+metabolic+stress+managers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toxin-antitoxin+modules+as+bacterial+metabolic+stress+managers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000955-Gerdes1" id="pone.0000955-Gerdes1"></a>Gerdes K, Christensen SK, Lobner-Olesen A (2005) Prokaryotic toxin-antitoxin stress response loci. Nat Rev Microbiol  3: 371–382.  <ul class="find" data-citedArticleID="997672" data-doi="10.1038/nrmicro1147"><li><a href="http://dx.doi.org/10.1038/nrmicro1147" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prokaryotic+toxin-antitoxin+stress+response+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prokaryotic+toxin-antitoxin+stress+response+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0000955-Bhoo1" id="pone.0000955-Bhoo1"></a>Bhoo SH, Davis SJ, Walker J, Karniol B, Vierstra RD (2001) Bacteriophytochromes are photochromic histidine kinases using a biliverdin chromophore. Nature  414: 776–779.  <ul class="find" data-citedArticleID="997568" data-doi="10.1038/414776a"><li><a href="http://dx.doi.org/10.1038/414776a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophytochromes+are+photochromic+histidine+kinases+using+a+biliverdin+chromophore." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophytochromes+are+photochromic+histidine+kinases+using+a+biliverdin+chromophore.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0000955-Davis1" id="pone.0000955-Davis1"></a>Davis SJ, Vener AV, Vierstra RD (1999) Bacteriophytochromes: phytochrome-like photoreceptors from nonphotosynthetic eubacteria. Science  286: 2517–2520.  <ul class="find" data-citedArticleID="997624" data-doi="10.1126/science.286.5449.2517"><li><a href="http://dx.doi.org/10.1126/science.286.5449.2517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophytochromes%3A+phytochrome-like+photoreceptors+from+nonphotosynthetic+eubacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophytochromes%3A+phytochrome-like+photoreceptors+from+nonphotosynthetic+eubacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0000955-Boyd1" id="pone.0000955-Boyd1"></a>Boyd A, Chakrabarty AM (1995) <em>Pseudomonas aeruginosa</em> biofilms: role of the alginate exopolysaccharide. J Ind Microbiol  15: 162–168.  <ul class="find" data-citedArticleID="997578" data-doi="10.1007/bf01569821"><li><a href="http://dx.doi.org/10.1007/bf01569821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudomonas+aeruginosa+biofilms%3A+role+of+the+alginate+exopolysaccharide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudomonas+aeruginosa+biofilms%3A+role+of+the+alginate+exopolysaccharide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0000955-Gacesa1" id="pone.0000955-Gacesa1"></a>Gacesa P (1998) Bacterial alginate biosynthesis–recent progress and future prospects. Microbiology  144 ( Pt 5): 1133–1143.  <ul class="find" data-citedArticleID="997660" data-doi="10.1099/00221287-144-5-1133"><li><a href="http://dx.doi.org/10.1099/00221287-144-5-1133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+alginate+biosynthesis%E2%80%93recent+progress+and+future+prospects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+alginate+biosynthesis%E2%80%93recent+progress+and+future+prospects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0000955-Agostini1" id="pone.0000955-Agostini1"></a>Agostini HJ, Carroll JD, Minton KW (1996) Identification and characterization of <em>uvrA</em>, a DNA repair gene of <em>Deinococcus radiodurans</em>. J Bacteriol  178: 6759–6765.  <ul class="find" data-citedArticleID="997550"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+and+characterization+of+uvrA%2C+a+DNA+repair+gene+of+Deinococcus+radiodurans.&amp;auth=&amp;atitle=Identification+and+characterization+of+uvrA%2C+a+DNA+repair+gene+of+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+uvrA%2C+a+DNA+repair+gene+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+uvrA%2C+a+DNA+repair+gene+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0000955-Lipton1" id="pone.0000955-Lipton1"></a>Lipton MS, Pasa-Tolic L, Anderson GA, Anderson DJ, Auberry DL, et al.  (2002) Global analysis of the <em>Deinococcus radiodurans</em> proteome by using accurate mass tags. Proc Natl Acad Sci U S A  99: 11049–11054.  <ul class="find" data-citedArticleID="997760" data-doi="10.1073/pnas.172170199"><li><a href="http://dx.doi.org/10.1073/pnas.172170199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+the+Deinococcus+radiodurans+proteome+by+using+accurate+mass+tags." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+the+Deinococcus+radiodurans+proteome+by+using+accurate+mass+tags.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0000955-Tanaka1" id="pone.0000955-Tanaka1"></a>Tanaka M, Earl AM, Howell HA, Park MJ, Eisen JA, et al.  (2004) Analysis of <em>Deinococcus radiodurans</em>'s transcriptional response to ionizing radiation and desiccation reveals novel proteins that contribute to extreme radioresistance. Genetics  168: 21–33.  <ul class="find" data-citedArticleID="997850" data-doi="10.1534/genetics.104.029249"><li><a href="http://dx.doi.org/10.1534/genetics.104.029249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Deinococcus+radiodurans%27s+transcriptional+response+to+ionizing+radiation+and+desiccation+reveals+novel+proteins+that+contribute+to+extreme+radioresistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Deinococcus+radiodurans%27s+transcriptional+response+to+ionizing+radiation+and+desiccation+reveals+novel+proteins+that+contribute+to+extreme+radioresistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0000955-Gutman2" id="pone.0000955-Gutman2"></a>Gutman PD, Carroll JD, Masters CI, Minton KW (1994) Sequencing, targeted mutagenesis and expression of a <em>recA</em> gene required for the extreme radioresistance of <em>Deinococcus radiodurans</em>. Gene  141: 31–37.  <ul class="find" data-citedArticleID="997686" data-doi="10.1016/0378-1119(94)90124-4"><li><a href="http://dx.doi.org/10.1016/0378-1119(94)90124-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing%2C+targeted+mutagenesis+and+expression+of+a+recA+gene+required+for+the+extreme+radioresistance+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing%2C+targeted+mutagenesis+and+expression+of+a+recA+gene+required+for+the+extreme+radioresistance+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0000955-Altschul1" id="pone.0000955-Altschul1"></a>Altschul SF, Koonin EV (1998) Iterated profile searches with PSI-BLAST–a tool for discovery in protein databases. Trends Biochem Sci  23: 444–447.  <ul class="find" data-citedArticleID="997554" data-doi="10.1016/s0968-0004(98)01298-5"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01298-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iterated+profile+searches+with+PSI-BLAST%E2%80%93a+tool+for+discovery+in+protein+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iterated+profile+searches+with+PSI-BLAST%E2%80%93a+tool+for+discovery+in+protein+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0000955-Schultz1" id="pone.0000955-Schultz1"></a>Schultz J, Copley RR, Doerks T, Ponting CP, Bork P (2000) SMART: a web-based tool for the study of genetically mobile domains. Nucleic Acids Res  28: 231–234.  <ul class="find" data-citedArticleID="997834" data-doi="10.1093/nar/28.1.231"><li><a href="http://dx.doi.org/10.1093/nar/28.1.231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SMART%3A+a+web-based+tool+for+the+study+of+genetically+mobile+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SMART%3A+a+web-based+tool+for+the+study+of+genetically+mobile+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0000955-MarchlerBauer1" id="pone.0000955-MarchlerBauer1"></a>Marchler-Bauer A, Panchenko AR, Shoemaker BA, Thiessen PA, Geer LY, et al.  (2002) CDD: a database of conserved domain alignments with links to domain three-dimensional structure. Nucleic Acids Res  30: 281–283.  <ul class="find" data-citedArticleID="997780" data-doi="10.1093/nar/30.1.281"><li><a href="http://dx.doi.org/10.1093/nar/30.1.281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CDD%3A+a+database+of+conserved+domain+alignments+with+links+to+domain+three-dimensional+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CDD%3A+a+database+of+conserved+domain+alignments+with+links+to+domain+three-dimensional+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0000955-Iyer1" id="pone.0000955-Iyer1"></a>Iyer LM, Koonin EV, Aravind L (2002) Classification and evolutionary history of the single-strand annealing proteins, RecT, Redbeta, ERF and RAD52. BMC Genomics  3: 8.  <ul class="find" data-citedArticleID="997716"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Classification+and+evolutionary+history+of+the+single-strand+annealing+proteins%2C+RecT%2C+Redbeta%2C+ERF+and+RAD52.&amp;auth=&amp;atitle=Classification+and+evolutionary+history+of+the+single-strand+annealing+proteins%2C+RecT%2C+Redbeta%2C+ERF+and+RAD52." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+and+evolutionary+history+of+the+single-strand+annealing+proteins%2C+RecT%2C+Redbeta%2C+ERF+and+RAD52." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+and+evolutionary+history+of+the+single-strand+annealing+proteins%2C+RecT%2C+Redbeta%2C+ERF+and+RAD52.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0000955-Harris1" id="pone.0000955-Harris1"></a>Harris DR, Tanaka M, Saveliev SV, Jolivet E, Earl AM, et al.  (2004) Preserving genome integrity: the DdrA protein of <em>Deinococcus radiodurans</em> R1. PLoS Biol  2: e304.  <ul class="find" data-citedArticleID="997694" data-doi="10.1371/journal.pbio.0020304"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preserving+genome+integrity%3A+the+DdrA+protein+of+Deinococcus+radiodurans+R1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preserving+genome+integrity%3A+the+DdrA+protein+of+Deinococcus+radiodurans+R1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0000955-Mironov1" id="pone.0000955-Mironov1"></a>Mironov AA, Koonin EV, Roytberg MA, Gelfand MS (1999) Computer analysis of transcription regulatory patterns in completely sequenced bacterial genomes. Nucleic Acids Res  27: 2981–2989.  <ul class="find" data-citedArticleID="997790" data-doi="10.1093/nar/27.14.2981"><li><a href="http://dx.doi.org/10.1093/nar/27.14.2981" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computer+analysis+of+transcription+regulatory+patterns+in+completely+sequenced+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computer+analysis+of+transcription+regulatory+patterns+in+completely+sequenced+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0000955-Mironov2" id="pone.0000955-Mironov2"></a>Mironov AA, Vinokurova NP, Gel'fand MS (2000) [Software for analyzing bacterial genomes]. Mol Biol (Mosk)  34: 253–262.  <ul class="find" data-citedArticleID="997792" data-doi="10.1007/bf02759643"><li><a href="http://dx.doi.org/10.1007/bf02759643" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BSoftware+for+analyzing+bacterial+genomes%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BSoftware+for+analyzing+bacterial+genomes%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0000955-Kunkel1" id="pone.0000955-Kunkel1"></a>Kunkel TA, Erie DA (2005) DNA mismatch repair. Annu Rev Biochem  74: 681–710.  <ul class="find" data-citedArticleID="997746" data-doi="10.1146/annurev.biochem.74.082803.133243"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.74.082803.133243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+mismatch+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+mismatch+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">102.
              </span><a name="pone.0000955-Kuzminov1" id="pone.0000955-Kuzminov1"></a>Kuzminov A (1999) Recombinational repair of DNA damage in <em>Escherichia coli</em> and bacteriophage lambda. Microbiol Mol Biol Rev  63: 751–813, table of contents.  <ul class="find" data-citedArticleID="997748"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda.&amp;auth=&amp;atitle=Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">103.
              </span><a name="pone.0000955-Touati1" id="pone.0000955-Touati1"></a>Touati E, Laurent-Winter C, Quillardet P, Hofnung M (1996) Global response of <em>Escherichia coli</em> cells to a treatment with 7-methoxy-2-nitronaphtho[2,1-b]furan (R7000), an extremely potent mutagen. Mutat Res  349: 193–200.  <ul class="find" data-citedArticleID="997858" data-doi="10.1016/0027-5107(95)00178-6"><li><a href="http://dx.doi.org/10.1016/0027-5107(95)00178-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+response+of+Escherichia+coli+cells+to+a+treatment+with+7-methoxy-2-nitronaphtho%5B2%2C1-b%5Dfuran+%28R7000%29%2C+an+extremely+potent+mutagen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+response+of+Escherichia+coli+cells+to+a+treatment+with+7-methoxy-2-nitronaphtho%5B2%2C1-b%5Dfuran+%28R7000%29%2C+an+extremely+potent+mutagen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">104.
              </span><a name="pone.0000955-Zhang1" id="pone.0000955-Zhang1"></a>Zhang YM, Liu JK, Wong TY (2003) The DNA excision repair system of the highly radioresistant bacterium <em>Deinococcus radiodurans</em> is facilitated by the pentose phosphate pathway. Mol Microbiol  48: 1317–1323.  <ul class="find" data-citedArticleID="997882" data-doi="10.1046/j.1365-2958.2003.03486.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03486.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DNA+excision+repair+system+of+the+highly+radioresistant+bacterium+Deinococcus+radiodurans+is+facilitated+by+the+pentose+phosphate+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DNA+excision+repair+system+of+the+highly+radioresistant+bacterium+Deinococcus+radiodurans+is+facilitated+by+the+pentose+phosphate+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">105.
              </span><a name="pone.0000955-Chasin1" id="pone.0000955-Chasin1"></a>Chasin LA, Magasanik B (1968) Induction and repression of the histidine-degrading enzymes of <em>Bacillus subtilis</em>. J Biol Chem  243: 5165–5178.  <ul class="find" data-citedArticleID="997596"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Induction+and+repression+of+the+histidine-degrading+enzymes+of+Bacillus+subtilis.&amp;auth=&amp;atitle=Induction+and+repression+of+the+histidine-degrading+enzymes+of+Bacillus+subtilis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+and+repression+of+the+histidine-degrading+enzymes+of+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+and+repression+of+the+histidine-degrading+enzymes+of+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">106.
              </span><a name="pone.0000955-Kannan1" id="pone.0000955-Kannan1"></a>Kannan K, Janiyani KL, Shivaji S, Ray MK (1998) Histidine utilisation operon (<em>hut</em>) is upregulated at low temperature in the antarctic psychrotrophic bacterium <em>Pseudomonas syringae</em>. FEMS Microbiol Lett  161: 7–14.  <ul class="find" data-citedArticleID="997722" data-doi="10.1111/j.1574-6968.1998.tb12922.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.1998.tb12922.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histidine+utilisation+operon+%28hut%29+is+upregulated+at+low+temperature+in+the+antarctic+psychrotrophic+bacterium+Pseudomonas+syringae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histidine+utilisation+operon+%28hut%29+is+upregulated+at+low+temperature+in+the+antarctic+psychrotrophic+bacterium+Pseudomonas+syringae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">107.
              </span><a name="pone.0000955-Bruce1" id="pone.0000955-Bruce1"></a>Bruce AK, Berner JD (1976) Respiratory activity as a determinant of radiation survival response. Can J Microbiol  22: 1336–1344.  <ul class="find" data-citedArticleID="997588" data-doi="10.1139/m76-197"><li><a href="http://dx.doi.org/10.1139/m76-197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Respiratory+activity+as+a+determinant+of+radiation+survival+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Respiratory+activity+as+a+determinant+of+radiation+survival+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">108.
              </span><a name="pone.0000955-Earl1" id="pone.0000955-Earl1"></a>Earl AM, Mohundro MM, Mian IS, Battista JR (2002) The IrrE protein of <em>Deinococcus radiodurans</em> R1 is a novel regulator of <em>recA</em> expression. J Bacteriol  184: 6216–6224.  <ul class="find" data-citedArticleID="997630" data-doi="10.1128/jb.184.22.6216-6224.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.22.6216-6224.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+IrrE+protein+of+Deinococcus+radiodurans+R1+is+a+novel+regulator+of+recA+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+IrrE+protein+of+Deinococcus+radiodurans+R1+is+a+novel+regulator+of+recA+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">109.
              </span><a name="pone.0000955-Hua1" id="pone.0000955-Hua1"></a>Hua Y, Narumi I, Gao G, Tian B, Satoh K, et al.  (2003) PprI: a general switch responsible for extreme radioresistance of <em>Deinococcus radiodurans</em>. Biochem Biophys Res Commun  306: 354–360.  <ul class="find" data-citedArticleID="997706" data-doi="10.1016/s0006-291x(03)00965-3"><li><a href="http://dx.doi.org/10.1016/s0006-291x(03)00965-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PprI%3A+a+general+switch+responsible+for+extreme+radioresistance+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PprI%3A+a+general+switch+responsible+for+extreme+radioresistance+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">110.
              </span><a name="pone.0000955-Gao1" id="pone.0000955-Gao1"></a>Gao G, Le D, Huang L, Lu H, Narumi I, et al.  (2006) Internal promoter characterization and expression of the <em>Deinococcus radiodurans pprI-folP</em> gene cluster. FEMS Microbiol Lett  257: 195–201.  <ul class="find" data-citedArticleID="997666" data-doi="10.1111/j.1574-6968.2006.00169.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2006.00169.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Internal+promoter+characterization+and+expression+of+the+Deinococcus+radiodurans+pprI-folP+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Internal+promoter+characterization+and+expression+of+the+Deinococcus+radiodurans+pprI-folP+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">111.
              </span><a name="pone.0000955-Carroll1" id="pone.0000955-Carroll1"></a>Carroll JD, Daly MJ, Minton KW (1996) Expression of <em>recA</em> in <em>Deinococcus radiodurans</em>. J Bacteriol  178: 130–135.  <ul class="find" data-citedArticleID="997594"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+of+recA+in+Deinococcus+radiodurans.&amp;auth=&amp;atitle=Expression+of+recA+in+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+recA+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+recA+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">112.
              </span><a name="pone.0000955-Cheo1" id="pone.0000955-Cheo1"></a>Cheo DL, Bayles KW, Yasbin RE (1991) Cloning and characterization of DNA damage-inducible promoter regions from <em>Bacillus subtilis</em>. J Bacteriol  173: 1696–1703.  <ul class="find" data-citedArticleID="997598"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cloning+and+characterization+of+DNA+damage-inducible+promoter+regions+from+Bacillus+subtilis.&amp;auth=&amp;atitle=Cloning+and+characterization+of+DNA+damage-inducible+promoter+regions+from+Bacillus+subtilis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+characterization+of+DNA+damage-inducible+promoter+regions+from+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+characterization+of+DNA+damage-inducible+promoter+regions+from+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">113.
              </span><a name="pone.0000955-Yasbin1" id="pone.0000955-Yasbin1"></a>Yasbin RE, Cheo D, Bayles KW (1991) The SOB system of <em>Bacillus subtilis</em>: a global regulon involved in DNA repair and differentiation. Res Microbiol  142: 885–892.  <ul class="find" data-citedArticleID="997874" data-doi="10.1016/0923-2508(91)90069-m"><li><a href="http://dx.doi.org/10.1016/0923-2508(91)90069-m" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SOB+system+of+Bacillus+subtilis%3A+a+global+regulon+involved+in+DNA+repair+and+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SOB+system+of+Bacillus+subtilis%3A+a+global+regulon+involved+in+DNA+repair+and+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">114.
              </span><a name="pone.0000955-Courcelle1" id="pone.0000955-Courcelle1"></a>Courcelle J, Khodursky A, Peter B, Brown PO, Hanawalt PC (2001) Comparative gene expression profiles following UV exposure in wild-type and SOS-deficient <em>Escherichia coli</em>. Genetics  158: 41–64.  <ul class="find" data-citedArticleID="997600"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli.&amp;auth=&amp;atitle=Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">115.
              </span><a name="pone.0000955-FernandezDeHenestrosa1" id="pone.0000955-FernandezDeHenestrosa1"></a>Fernandez De Henestrosa AR, Ogi T, Aoyagi S, Chafin D, Hayes JJ, et al.  (2000) Identification of additional genes belonging to the LexA regulon in <em>Escherichia coli</em>. Mol Microbiol  35: 1560–1572.  <ul class="find" data-citedArticleID="997646" data-doi="10.1046/j.1365-2958.2000.01826.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01826.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+additional+genes+belonging+to+the+LexA+regulon+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+additional+genes+belonging+to+the+LexA+regulon+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">116.
              </span><a name="pone.0000955-Narumi1" id="pone.0000955-Narumi1"></a>Narumi I, Satoh K, Kikuchi M, Funayama T, Yanagisawa T, et al.  (2001) The LexA protein from <em>Deinococcus radiodurans</em> is not involved in RecA induction following gamma irradiation. J Bacteriol  183: 6951–6956.  <ul class="find" data-citedArticleID="997804" data-doi="10.1128/jb.183.23.6951-6956.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.23.6951-6956.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+LexA+protein+from+Deinococcus+radiodurans+is+not+involved+in+RecA+induction+following+gamma+irradiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+LexA+protein+from+Deinococcus+radiodurans+is+not+involved+in+RecA+induction+following+gamma+irradiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">117.
              </span><a name="pone.0000955-Little1" id="pone.0000955-Little1"></a>Little JW, Mount DW, Yanisch-Perron CR (1981) Purified LexA protein is a repressor of the <em>recA</em> and <em>lexA</em> genes. Proc Natl Acad Sci U S A  78: 4199–4203.  <ul class="find" data-citedArticleID="997762" data-doi="10.1073/pnas.78.7.4199"><li><a href="http://dx.doi.org/10.1073/pnas.78.7.4199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purified+LexA+protein+is+a+repressor+of+the+recA+and+lexA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purified+LexA+protein+is+a+repressor+of+the+recA+and+lexA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">118.
              </span><a name="pone.0000955-Moseley1" id="pone.0000955-Moseley1"></a>Moseley BE, Mattingly A (1971) Repair of irradiation transforming deoxyribonucleic acid in wild type and a radiation-sensitive mutant of <em>Micrococcus radiodurans</em>. J Bacteriol  105: 976–983.  <ul class="find" data-citedArticleID="997800"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Repair+of+irradiation+transforming+deoxyribonucleic+acid+in+wild+type+and+a+radiation-sensitive+mutant+of+Micrococcus+radiodurans.&amp;auth=&amp;atitle=Repair+of+irradiation+transforming+deoxyribonucleic+acid+in+wild+type+and+a+radiation-sensitive+mutant+of+Micrococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repair+of+irradiation+transforming+deoxyribonucleic+acid+in+wild+type+and+a+radiation-sensitive+mutant+of+Micrococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repair+of+irradiation+transforming+deoxyribonucleic+acid+in+wild+type+and+a+radiation-sensitive+mutant+of+Micrococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">119.
              </span><a name="pone.0000955-Gutman3" id="pone.0000955-Gutman3"></a>Gutman PD, Yao HL, Minton KW (1991) Partial complementation of the UV sensitivity of <em>Deinococcus radiodurans</em> excision repair mutants by the cloned <em>denV</em> gene of bacteriophage T4. Mutat Res  254: 207–215.  <ul class="find" data-citedArticleID="997688" data-doi="10.1016/0921-8777(91)90058-w"><li><a href="http://dx.doi.org/10.1016/0921-8777(91)90058-w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Partial+complementation+of+the+UV+sensitivity+of+Deinococcus+radiodurans+excision+repair+mutants+by+the+cloned+denV+gene+of+bacteriophage+T4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Partial+complementation+of+the+UV+sensitivity+of+Deinococcus+radiodurans+excision+repair+mutants+by+the+cloned+denV+gene+of+bacteriophage+T4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">120.
              </span><a name="pone.0000955-Minton1" id="pone.0000955-Minton1"></a>Minton KW (1996) Repair of ionizing-radiation damage in the radiation resistant bacterium <em>Deinococcus radiodurans</em>. Mutat Res  363: 1–7.  <ul class="find" data-citedArticleID="997788" data-doi="10.1016/0921-8777(95)00014-3"><li><a href="http://dx.doi.org/10.1016/0921-8777(95)00014-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repair+of+ionizing-radiation+damage+in+the+radiation+resistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repair+of+ionizing-radiation+damage+in+the+radiation+resistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">121.
              </span><a name="pone.0000955-Schlesinger1" id="pone.0000955-Schlesinger1"></a>Schlesinger DJ (2007) Role of RecA in DNA damage repair in <em>Deinococcus radiodurans</em>. FEMS Microbiol Lett  274: 342–347.  <ul class="find" data-citedArticleID="997830" data-doi="10.1111/j.1574-6968.2007.00862.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2007.00862.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+RecA+in+DNA+damage+repair+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+RecA+in+DNA+damage+repair+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">122.
              </span><a name="pone.0000955-Daly10" id="pone.0000955-Daly10"></a>Daly MJ, Minton KW (1995) Resistance to radiation. Science  270: 1318.  <ul class="find" data-citedArticleID="997606" data-doi="10.1126/science.270.5240.1318"><li><a href="http://dx.doi.org/10.1126/science.270.5240.1318" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+to+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+to+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">123.
              </span><a name="pone.0000955-Eltsov1" id="pone.0000955-Eltsov1"></a>Eltsov M, Dubochet J (2005) Fine structure of the <em>Deinococcus radiodurans</em> nucleoid revealed by cryoelectron microscopy of vitreous sections. J Bacteriol  187: 8047–8054.  <ul class="find" data-citedArticleID="997636" data-doi="10.1128/jb.187.23.8047-8054.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.23.8047-8054.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine+structure+of+the+Deinococcus+radiodurans+nucleoid+revealed+by+cryoelectron+microscopy+of+vitreous+sections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine+structure+of+the+Deinococcus+radiodurans+nucleoid+revealed+by+cryoelectron+microscopy+of+vitreous+sections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">124.
              </span><a name="pone.0000955-Eltsov2" id="pone.0000955-Eltsov2"></a>Eltsov M, Dubochet J (2006) Study of the <em>Deinococcus radiodurans</em> nucleoid by cryoelectron microscopy of vitreous sections: Supplementary comments. J Bacteriol  188: 6053–6058; discussion 6059.  <ul class="find" data-citedArticleID="997638" data-doi="10.1128/jb.00230-06"><li><a href="http://dx.doi.org/10.1128/jb.00230-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Study+of+the+Deinococcus+radiodurans+nucleoid+by+cryoelectron+microscopy+of+vitreous+sections%3A+Supplementary+comments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Study+of+the+Deinococcus+radiodurans+nucleoid+by+cryoelectron+microscopy+of+vitreous+sections%3A+Supplementary+comments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">125.
              </span><a name="pone.0000955-Gao2" id="pone.0000955-Gao2"></a>Gao G, Lu H, Yin L, Hua Y (2007) Ring-like nucleoid does not play a key role in radioresistance of <em>Deinococcus radiodurans</em>. Sci China C Life Sci  50: 525–529.  <ul class="find" data-citedArticleID="997668" data-doi="10.1007/s11427-007-0061-x"><li><a href="http://dx.doi.org/10.1007/s11427-007-0061-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ring-like+nucleoid+does+not+play+a+key+role+in+radioresistance+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ring-like+nucleoid+does+not+play+a+key+role+in+radioresistance+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">126.
              </span><a name="pone.0000955-Setlow1" id="pone.0000955-Setlow1"></a>Setlow DM, Duggan DE (1964) The resistance of <em>Micrococcus radiodurans</em> to ultraviolet radiation: Ultraviolet-induced lesions in the cell's DNA. Biochim Biophys Acta  87: 664–668.  <ul class="find" data-citedArticleID="997838" data-doi="10.1016/0926-6550(64)90284-1"><li><a href="http://dx.doi.org/10.1016/0926-6550(64)90284-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+resistance+of+Micrococcus+radiodurans+to+ultraviolet+radiation%3A+Ultraviolet-induced+lesions+in+the+cell%27s+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+resistance+of+Micrococcus+radiodurans+to+ultraviolet+radiation%3A+Ultraviolet-induced+lesions+in+the+cell%27s+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">127.
              </span><a name="pone.0000955-Huang1" id="pone.0000955-Huang1"></a>Huang L, Hua X, Lu H, Gao G, Tian B, et al.  (2007) Three tandem HRDC domains have synergistic effect on the RecQ functions in <em>Deinococcus radiodurans</em>. DNA Repair (Amst)  6: 167–176.  <ul class="find" data-citedArticleID="997708" data-doi="10.1016/j.dnarep.2006.09.006"><li><a href="http://dx.doi.org/10.1016/j.dnarep.2006.09.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+tandem+HRDC+domains+have+synergistic+effect+on+the+RecQ+functions+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+tandem+HRDC+domains+have+synergistic+effect+on+the+RecQ+functions+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">128.
              </span><a name="pone.0000955-Kim1" id="pone.0000955-Kim1"></a>Kim JI, Sharma AK, Abbott SN, Wood EA, Dwyer DW, et al.  (2002) RecA Protein from the extremely radioresistant bacterium <em>Deinococcus radiodurans</em>: expression, purification, and characterization. J Bacteriol  184: 1649–1660.  <ul class="find" data-citedArticleID="997726" data-doi="10.1128/jb.184.6.1649-1660.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.6.1649-1660.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RecA+Protein+from+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans%3A+expression%2C+purification%2C+and+characterization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RecA+Protein+from+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans%3A+expression%2C+purification%2C+and+characterization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">129.
              </span><a name="pone.0000955-Narumi2" id="pone.0000955-Narumi2"></a>Narumi I, Satoh K, Kikuchi M, Funayama T, Kitayama S, et al.  (1999) Molecular analysis of the <em>Deinococcus radiodurans recA</em> locus and identification of a mutation site in a DNA repair-deficient mutant, rec30. Mutat Res  435: 233–243.  <ul class="find" data-citedArticleID="997806" data-doi="10.1016/s0921-8777(99)00048-8"><li><a href="http://dx.doi.org/10.1016/s0921-8777(99)00048-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+the+Deinococcus+radiodurans+recA+locus+and+identification+of+a+mutation+site+in+a+DNA+repair-deficient+mutant%2C+rec30." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+the+Deinococcus+radiodurans+recA+locus+and+identification+of+a+mutation+site+in+a+DNA+repair-deficient+mutant%2C+rec30.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">130.
              </span><a name="pone.0000955-Bernstein1" id="pone.0000955-Bernstein1"></a>Bernstein DA, Eggington JM, Killoran MP, Misic AM, Cox MM, et al.  (2004) Crystal structure of the <em>Deinococcus radiodurans</em> single-stranded DNA-binding protein suggests a mechanism for coping with DNA damage. Proc Natl Acad Sci U S A  101: 8575–8580.  <ul class="find" data-citedArticleID="997566" data-doi="10.1073/pnas.0401331101"><li><a href="http://dx.doi.org/10.1073/pnas.0401331101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+Deinococcus+radiodurans+single-stranded+DNA-binding+protein+suggests+a+mechanism+for+coping+with+DNA+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+Deinococcus+radiodurans+single-stranded+DNA-binding+protein+suggests+a+mechanism+for+coping+with+DNA+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">131.
              </span><a name="pone.0000955-Bowater1" id="pone.0000955-Bowater1"></a>Bowater R, Doherty AJ (2006) Making ends meet: repairing breaks in bacterial DNA by non-homologous end-joining. PLoS Genet  2: e8.  <ul class="find" data-citedArticleID="997576" data-doi="10.1371/journal.pgen.0020008"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Making+ends+meet%3A+repairing+breaks+in+bacterial+DNA+by+non-homologous+end-joining." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Making+ends+meet%3A+repairing+breaks+in+bacterial+DNA+by+non-homologous+end-joining.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">132.
              </span><a name="pone.0000955-Englander1" id="pone.0000955-Englander1"></a>Englander J, Klein E, Brumfeld V, Sharma AK, Doherty AJ, et al.  (2004) DNA toroids: framework for DNA repair in <em>Deinococcus radiodurans</em> and in germinating bacterial spores. J Bacteriol  186: 5973–5977.  <ul class="find" data-citedArticleID="997640" data-doi="10.1128/jb.186.18.5973-5977.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.18.5973-5977.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+toroids%3A+framework+for+DNA+repair+in+Deinococcus+radiodurans+and+in+germinating+bacterial+spores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+toroids%3A+framework+for+DNA+repair+in+Deinococcus+radiodurans+and+in+germinating+bacterial+spores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">133.
              </span><a name="pone.0000955-Kobayashi2" id="pone.0000955-Kobayashi2"></a>Kobayashi Y, Narumi I, Satoh K, Funayama T, Kikuchi M, et al.  (2004) Radiation response mechanisms of the extremely radioresistant bacterium <em>Deinococcus radiodurans</em>. Biol Sci Space  18: 134–135.  <ul class="find" data-citedArticleID="997740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Radiation+response+mechanisms+of+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans.&amp;auth=&amp;atitle=Radiation+response+mechanisms+of+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Radiation+response+mechanisms+of+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Radiation+response+mechanisms+of+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">134.
              </span><a name="pone.0000955-Lecointe1" id="pone.0000955-Lecointe1"></a>Lecointe F, Shevelev IV, Bailone A, Sommer S, Hubscher U (2004) Involvement of an X family DNA polymerase in double-stranded break repair in the radioresistant organism <em>Deinococcus radiodurans</em>. Mol Microbiol  53: 1721–1730.  <ul class="find" data-citedArticleID="997754" data-doi="10.1111/j.1365-2958.2004.04233.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04233.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Involvement+of+an+X+family+DNA+polymerase+in+double-stranded+break+repair+in+the+radioresistant+organism+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Involvement+of+an+X+family+DNA+polymerase+in+double-stranded+break+repair+in+the+radioresistant+organism+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">135.
              </span><a name="pone.0000955-LevinZaidman1" id="pone.0000955-LevinZaidman1"></a>Levin-Zaidman S, Englander J, Shimoni E, Sharma AK, Minton KW, et al.  (2003) Ringlike structure of the <em>Deinococcus radiodurans</em> genome: a key to radioresistance? Science  299: 254–256.  <ul class="find" data-citedArticleID="997756" data-doi="10.1126/science.1077865"><li><a href="http://dx.doi.org/10.1126/science.1077865" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ringlike+structure+of+the+Deinococcus+radiodurans+genome%3A+a+key+to+radioresistance%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ringlike+structure+of+the+Deinococcus+radiodurans+genome%3A+a+key+to+radioresistance%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">136.
              </span><a name="pone.0000955-Narumi3" id="pone.0000955-Narumi3"></a>Narumi I, Satoh K, Cui S, Funayama T, Kitayama S, et al.  (2004) PprA: a novel protein from <em>Deinococcus radiodurans</em> that stimulates DNA ligation. Mol Microbiol  54: 278–285.  <ul class="find" data-citedArticleID="997808" data-doi="10.1111/j.1365-2958.2004.04272.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04272.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PprA%3A+a+novel+protein+from+Deinococcus+radiodurans+that+stimulates+DNA+ligation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PprA%3A+a+novel+protein+from+Deinococcus+radiodurans+that+stimulates+DNA+ligation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">137.
              </span><a name="pone.0000955-Blasius1" id="pone.0000955-Blasius1"></a>Blasius M, Buob R, Shevelev IV, Hubscher U (2007) Enzymes involved in DNA ligation and end-healing in the radioresistant bacterium <em>Deinococcus radiodurans</em>. BMC Mol Biol  8: 69.  <ul class="find" data-citedArticleID="997570" data-doi="10.1186/1471-2199-8-69"><li><a href="http://dx.doi.org/10.1186/1471-2199-8-69" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymes+involved+in+DNA+ligation+and+end-healing+in+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymes+involved+in+DNA+ligation+and+end-healing+in+the+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">138.
              </span><a name="pone.0000955-Wright1" id="pone.0000955-Wright1"></a>Wright EG, Coates PJ (2006) Untargeted effects of ionizing radiation: implications for radiation pathology. Mutat Res  597: 119–132.  <ul class="find" data-citedArticleID="997870" data-doi="10.1016/j.mrfmmm.2005.03.035"><li><a href="http://dx.doi.org/10.1016/j.mrfmmm.2005.03.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Untargeted+effects+of+ionizing+radiation%3A+implications+for+radiation+pathology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Untargeted+effects+of+ionizing+radiation%3A+implications+for+radiation+pathology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">139.
              </span><a name="pone.0000955-Belyakov1" id="pone.0000955-Belyakov1"></a>Belyakov OV, Mitchell SA, Parikh D, Randers-Pehrson G, Marino SA, et al.  (2005) Biological effects in unirradiated human tissue induced by radiation damage up to 1 mm away. Proc Natl Acad Sci U S A  102: 14203–14208.  <ul class="find" data-citedArticleID="997564" data-doi="10.1073/pnas.0505020102"><li><a href="http://dx.doi.org/10.1073/pnas.0505020102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biological+effects+in+unirradiated+human+tissue+induced+by+radiation+damage+up+to+1+mm+away." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biological+effects+in+unirradiated+human+tissue+induced+by+radiation+damage+up+to+1+mm+away.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">140.
              </span><a name="pone.0000955-vonSonntag1" id="pone.0000955-vonSonntag1"></a>von Sonntag C (1987) The chemical basis of radiation biology. London, United Kingdom: Taylor &amp; Francis.   <ul class="find-nolinks"></ul></li><li><span class="label">141.
              </span><a name="pone.0000955-Du1" id="pone.0000955-Du1"></a>Du J, Gebicki JM (2004) Proteins are major initial cell targets of hydroxyl free radicals. Int J Biochem Cell Biol  36: 2334–2343.  <ul class="find" data-citedArticleID="997628" data-doi="10.1016/j.biocel.2004.05.012"><li><a href="http://dx.doi.org/10.1016/j.biocel.2004.05.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteins+are+major+initial+cell+targets+of+hydroxyl+free+radicals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteins+are+major+initial+cell+targets+of+hydroxyl+free+radicals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">142.
              </span><a name="pone.0000955-Kemner1" id="pone.0000955-Kemner1"></a>Kemner KM, Kelly SD, Lai B, Maser J, O'Loughlin E J, et al.  (2004) Elemental and redox analysis of single bacterial cells by x-ray microbeam analysis. Science  306: 686–687.  <ul class="find" data-citedArticleID="997724" data-doi="10.1126/science.1103524"><li><a href="http://dx.doi.org/10.1126/science.1103524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elemental+and+redox+analysis+of+single+bacterial+cells+by+x-ray+microbeam+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elemental+and+redox+analysis+of+single+bacterial+cells+by+x-ray+microbeam+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">143.
              </span><a name="pone.0000955-Heinz1" id="pone.0000955-Heinz1"></a>Heinz K, Marx A (2007) Lesion bypass activity of DNA polymerase A from the extremely radioresistant organism <em>Deinococcus radiodurans</em>. J Biol Chem  282: 10908–10914.  <ul class="find" data-citedArticleID="997696" data-doi="10.1074/jbc.m611404200"><li><a href="http://dx.doi.org/10.1074/jbc.m611404200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lesion+bypass+activity+of+DNA+polymerase+A+from+the+extremely+radioresistant+organism+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lesion+bypass+activity+of+DNA+polymerase+A+from+the+extremely+radioresistant+organism+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">144.
              </span><a name="pone.0000955-Qiu1" id="pone.0000955-Qiu1"></a>Qiu X, Daly MJ, Vasilenko A, Omelchenko MV, Gaidamakova EK, et al.  (2006) Transcriptome analysis applied to survival of <em>Shewanella oneidensis</em> MR-1 exposed to ionizing radiation. J Bacteriol  188: 1199–1204.  <ul class="find" data-citedArticleID="997820" data-doi="10.1128/jb.188.3.1199-1204.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.3.1199-1204.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+analysis+applied+to+survival+of+Shewanella+oneidensis+MR-1+exposed+to+ionizing+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+analysis+applied+to+survival+of+Shewanella+oneidensis+MR-1+exposed+to+ionizing+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">145.
              </span><a name="pone.0000955-Fong1" id="pone.0000955-Fong1"></a>Fong SS, Joyce AR, Palsson BO (2005) Parallel adaptive evolution cultures of <em>Escherichia coli</em> lead to convergent growth phenotypes with different gene expression states. Genome Res  15: 1365–1372.  <ul class="find" data-citedArticleID="997652" data-doi="10.1101/gr.3832305"><li><a href="http://dx.doi.org/10.1101/gr.3832305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+adaptive+evolution+cultures+of+Escherichia+coli+lead+to+convergent+growth+phenotypes+with+different+gene+expression+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+adaptive+evolution+cultures+of+Escherichia+coli+lead+to+convergent+growth+phenotypes+with+different+gene+expression+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">146.
              </span><a name="pone.0000955-Koonin1" id="pone.0000955-Koonin1"></a>Koonin EV (2007) Chance and necessity in cellular response to challenge. Molec Syst Biol: in press.   <ul class="find-nolinks"></ul></li><li><span class="label">147.
              </span><a name="pone.0000955-Stern1" id="pone.0000955-Stern1"></a>Stern S, Dror T, Stolovicki E, Brenner N, Braun E (2007) Genome-wide transcriptional plasticity underlies cellular adaptation to novel changes. Molec Syst Biol: in press.   <ul class="find-nolinks"></ul></li><li><span class="label">148.
              </span><a name="pone.0000955-Albuquerque1" id="pone.0000955-Albuquerque1"></a>Albuquerque L, Simoes C, Nobre MF, Pino NM, Battista JR, et al.  (2005) <em>Truepera radiovictrix</em> gen. nov., sp. nov., a new radiation resistant species and the proposal of <em>Trueperaceae</em> fam. nov. FEMS Microbiol Lett  247: 161–169.  <ul class="find" data-citedArticleID="997552" data-doi="10.1016/j.femsle.2005.05.002"><li><a href="http://dx.doi.org/10.1016/j.femsle.2005.05.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Truepera+radiovictrix+gen.+nov.%2C+sp.+nov.%2C+a+new+radiation+resistant+species+and+the+proposal+of+Trueperaceae+fam.+nov." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Truepera+radiovictrix+gen.+nov.%2C+sp.+nov.%2C+a+new+radiation+resistant+species+and+the+proposal+of+Trueperaceae+fam.+nov.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">149.
              </span><a name="pone.0000955-Detter1" id="pone.0000955-Detter1"></a>Detter JC, Jett JM, Lucas SM, Dalin E, Arellano AR, et al.  (2002) Isothermal strand-displacement amplification applications for high-throughput genomics. Genomics  80: 691–698.  <ul class="find" data-citedArticleID="997626" data-doi="10.1006/geno.2002.7020"><li><a href="http://dx.doi.org/10.1006/geno.2002.7020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isothermal+strand-displacement+amplification+applications+for+high-throughput+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isothermal+strand-displacement+amplification+applications+for+high-throughput+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">150.
              </span><a name="pone.0000955-Ewing1" id="pone.0000955-Ewing1"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res  8: 186–194.  <ul class="find" data-citedArticleID="997642" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">151.
              </span><a name="pone.0000955-Ewing2" id="pone.0000955-Ewing2"></a>Ewing B, Hillier L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using phred. I. Accuracy assessment. Genome Res  8: 175–185.  <ul class="find" data-citedArticleID="997644" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">152.
              </span><a name="pone.0000955-Gordon1" id="pone.0000955-Gordon1"></a>Gordon D, Abajian C, Green P (1998) Consed: a graphical tool for sequence finishing. Genome Res  8: 195–202.  <ul class="find" data-citedArticleID="997678" data-doi="10.1101/gr.8.3.195"><li><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consed%3A+a+graphical+tool+for+sequence+finishing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consed%3A+a+graphical+tool+for+sequence+finishing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">153.
              </span><a name="pone.0000955-Tatusov3" id="pone.0000955-Tatusov3"></a>Tatusov RL, Natale DA, Garkavtsev IV, Tatusova TA, Shankavaram UT, et al.  (2001) The COG database: new developments in phylogenetic classification of proteins from complete genomes. Nucleic Acids Res  29: 22–28.  <ul class="find" data-citedArticleID="997856" data-doi="10.1093/nar/29.1.22"><li><a href="http://dx.doi.org/10.1093/nar/29.1.22" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+COG+database%3A+new+developments+in+phylogenetic+classification+of+proteins+from+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+COG+database%3A+new+developments+in+phylogenetic+classification+of+proteins+from+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">154.
              </span><a name="pone.0000955-Edgar1" id="pone.0000955-Edgar1"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res  32: 1792–1797.  <ul class="find" data-citedArticleID="997632" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">155.
              </span><a name="pone.0000955-Makarova6" id="pone.0000955-Makarova6"></a>Makarova K, Slesarev A, Wolf Y, Sorokin A, Mirkin B, et al.  (2006) Comparative genomics of the lactic acid bacteria. Proc Natl Acad Sci U S A  103: 15611–15616.  <ul class="find" data-citedArticleID="997776" data-doi="10.1073/pnas.0607117103"><li><a href="http://dx.doi.org/10.1073/pnas.0607117103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics+of+the+lactic+acid+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics+of+the+lactic+acid+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">156.
              </span><a name="pone.0000955-Makarova7" id="pone.0000955-Makarova7"></a>Makarova KS, Wolf YI, Mekhedov SL, Mirkin BG, Koonin EV (2005) Ancestral paralogs and pseudoparalogs and their role in the emergence of the eukaryotic cell. Nucleic Acids Res  33: 4626–4638.  <ul class="find" data-citedArticleID="997778" data-doi="10.1093/nar/gki775"><li><a href="http://dx.doi.org/10.1093/nar/gki775" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancestral+paralogs+and+pseudoparalogs+and+their+role+in+the+emergence+of+the+eukaryotic+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancestral+paralogs+and+pseudoparalogs+and+their+role+in+the+emergence+of+the+eukaryotic+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">157.
              </span><a name="pone.0000955-Bordo1" id="pone.0000955-Bordo1"></a>Bordo D, Djinovic K, Bolognesi M (1994) Conserved patterns in the Cu,Zn superoxide dismutase family. J Mol Biol  238: 366–386.  <ul class="find" data-citedArticleID="997572" data-doi="10.1006/jmbi.1994.1298"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+patterns+in+the+Cu%2CZn+superoxide+dismutase+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+patterns+in+the+Cu%2CZn+superoxide+dismutase+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">158.
              </span><a name="pone.0000955-Singleton1" id="pone.0000955-Singleton1"></a>Singleton MR, Wentzell LM, Liu Y, West SC, Wigley DB (2002) Structure of the single-strand annealing domain of human RAD52 protein. Proc Natl Acad Sci U S A  99: 13492–13497.  <ul class="find" data-citedArticleID="997844" data-doi="10.1073/pnas.212449899"><li><a href="http://dx.doi.org/10.1073/pnas.212449899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+single-strand+annealing+domain+of+human+RAD52+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+single-strand+annealing+domain+of+human+RAD52+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">159.
              </span><a name="pone.0000955-Bouchard1" id="pone.0000955-Bouchard1"></a>Bouchard JD, Moineau S (2004) Lactococcal phage genes involved in sensitivity to AbiK and their relation to single-strand annealing proteins. J Bacteriol  186: 3649–3652.  <ul class="find" data-citedArticleID="997574" data-doi="10.1128/jb.186.11.3649-3652.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.11.3649-3652.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lactococcal+phage+genes+involved+in+sensitivity+to+AbiK+and+their+relation+to+single-strand+annealing+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lactococcal+phage+genes+involved+in+sensitivity+to+AbiK+and+their+relation+to+single-strand+annealing+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">160.
              </span><a name="pone.0000955-Schneider1" id="pone.0000955-Schneider1"></a>Schneider TD, Stephens RM (1990) Sequence logos: a new way to display consensus sequences. Nucleic Acids Res  18: 6097–6100.  <ul class="find" data-citedArticleID="997832" data-doi="10.1093/nar/18.20.6097"><li><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+logos%3A+a+new+way+to+display+consensus+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+logos%3A+a+new+way+to+display+consensus+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">161.
              </span><a name="pone.0000955-Kitayama1" id="pone.0000955-Kitayama1"></a>Kitayama S, Kohoroku M, Takagi A, Itoh H (1997) Mutation of <em>D. radiodurans</em> in a gene homologous to <em>ruvB</em> of <em>E. coli</em>. Mutat Res  385: 151–157.  <ul class="find" data-citedArticleID="997732" data-doi="10.1016/s0921-8777(97)00048-7"><li><a href="http://dx.doi.org/10.1016/s0921-8777(97)00048-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+of+D.+radiodurans+in+a+gene+homologous+to+ruvB+of+E.+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+of+D.+radiodurans+in+a+gene+homologous+to+ruvB+of+E.+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">162.
              </span><a name="pone.0000955-Udupa1" id="pone.0000955-Udupa1"></a>Udupa KS, O'Cain PA, Mattimore V, Battista JR (1994) Novel ionizing radiation-sensitive mutants of <em>Deinococcus radiodurans</em>. J Bacteriol  176: 7439–7446.  <ul class="find" data-citedArticleID="997860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Novel+ionizing+radiation-sensitive+mutants+of+Deinococcus+radiodurans.&amp;auth=&amp;atitle=Novel+ionizing+radiation-sensitive+mutants+of+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+ionizing+radiation-sensitive+mutants+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+ionizing+radiation-sensitive+mutants+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">163.
              </span><a name="pone.0000955-Battista1" id="pone.0000955-Battista1"></a>Battista JR, Park MJ, McLemore AE (2001) Inactivation of Two Homologues of Proteins Presumed to be Involved in the Desiccation Tolerance of Plants Sensitizes <em>Deinococcus radiodurans</em> R1 to Desiccation. Cryobiology  43: 133–139.  <ul class="find" data-citedArticleID="997560" data-doi="10.1006/cryo.2001.2357"><li><a href="http://dx.doi.org/10.1006/cryo.2001.2357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+Two+Homologues+of+Proteins+Presumed+to+be+Involved+in+the+Desiccation+Tolerance+of+Plants+Sensitizes+Deinococcus+radiodurans+R1+to+Desiccation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+Two+Homologues+of+Proteins+Presumed+to+be+Involved+in+the+Desiccation+Tolerance+of+Plants+Sensitizes+Deinococcus+radiodurans+R1+to+Desiccation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">164.
              </span><a name="pone.0000955-Kitayama2" id="pone.0000955-Kitayama2"></a>Kitayama S, Narumi I, Kikuchi M, Watanabe H (2000) Mutation in recR gene of <em>Deinococcus radiodurans</em> and possible involvement of its product in the repair of DNA interstrand cross-links. Mutat Res  461: 179–187.  <ul class="find" data-citedArticleID="997734" data-doi="10.1016/s0921-8777(00)00044-6"><li><a href="http://dx.doi.org/10.1016/s0921-8777(00)00044-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+in+recR+gene+of+Deinococcus+radiodurans+and+possible+involvement+of+its+product+in+the+repair+of+DNA+interstrand+cross-links." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+in+recR+gene+of+Deinococcus+radiodurans+and+possible+involvement+of+its+product+in+the+repair+of+DNA+interstrand+cross-links.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">165.
              </span><a name="pone.0000955-Funayama1" id="pone.0000955-Funayama1"></a>Funayama T, Narumi I, Kikuchi M, Kitayama S, Watanabe H, et al.  (1999) Identification and disruption analysis of the recN gene in the extremely radioresistant bacterium <em>Deinococcus radiodurans</em>. Mutat Res  435: 151–161.  <ul class="find" data-citedArticleID="997658" data-doi="10.1016/s0921-8777(99)00044-0"><li><a href="http://dx.doi.org/10.1016/s0921-8777(99)00044-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+disruption+analysis+of+the+recN+gene+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+disruption+analysis+of+the+recN+gene+in+the+extremely+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">166.
              </span><a name="pone.0000955-Gutman4" id="pone.0000955-Gutman4"></a>Gutman PD, Fuchs P, Ouyang L, Minton KW (1993) Identification, sequencing, and targeted mutagenesis of a DNA polymerase gene required for the extreme radioresistance of <em>Deinococcus radiodurans</em>. J Bacteriol  175: 3581–3590.  <ul class="find" data-citedArticleID="997690" data-doi="10.1016/0378-1119(94)90124-4"><li><a href="http://dx.doi.org/10.1016/0378-1119(94)90124-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification%2C+sequencing%2C+and+targeted+mutagenesis+of+a+DNA+polymerase+gene+required+for+the+extreme+radioresistance+of+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification%2C+sequencing%2C+and+targeted+mutagenesis+of+a+DNA+polymerase+gene+required+for+the+extreme+radioresistance+of+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">167.
              </span><a name="pone.0000955-Mattimore1" id="pone.0000955-Mattimore1"></a>Mattimore V, Battista JR (1996) Radioresistance of <em>Deinococcus radiodurans</em>: functions necessary to survive ionizing radiation are also necessary to survive prolonged desiccation. J Bacteriol  178: 633–637.  <ul class="find" data-citedArticleID="997786"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Radioresistance+of+Deinococcus+radiodurans%3A+functions+necessary+to+survive+ionizing+radiation+are+also+necessary+to+survive+prolonged+desiccation.&amp;auth=&amp;atitle=Radioresistance+of+Deinococcus+radiodurans%3A+functions+necessary+to+survive+ionizing+radiation+are+also+necessary+to+survive+prolonged+desiccation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Radioresistance+of+Deinococcus+radiodurans%3A+functions+necessary+to+survive+ionizing+radiation+are+also+necessary+to+survive+prolonged+desiccation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Radioresistance+of+Deinococcus+radiodurans%3A+functions+necessary+to+survive+ionizing+radiation+are+also+necessary+to+survive+prolonged+desiccation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">168.
              </span><a name="pone.0000955-Satoh1" id="pone.0000955-Satoh1"></a>Satoh K, Ohba H, Sghaier H, Narumi I (2006) Down-regulation of radioresistance by LexA2 in <em>Deinococcus radiodurans</em>. Microbiology  152: 3217–3226.  <ul class="find" data-citedArticleID="997828" data-doi="10.1099/mic.0.29139-0"><li><a href="http://dx.doi.org/10.1099/mic.0.29139-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Down-regulation+of+radioresistance+by+LexA2+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Down-regulation+of+radioresistance+by+LexA2+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">169.
              </span><a name="pone.0000955-Sheng1" id="pone.0000955-Sheng1"></a>Sheng D, Zheng Z, Tian B, Shen B, Hua Y (2004) LexA analog (dra0074) is a regulatory protein that is irrelevant to recA induction. J Biochem (Tokyo)  136: 787–793.  <ul class="find" data-citedArticleID="997842" data-doi="10.1093/jb/mvh188"><li><a href="http://dx.doi.org/10.1093/jb/mvh188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LexA+analog+%28dra0074%29+is+a+regulatory+protein+that+is+irrelevant+to+recA+induction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LexA+analog+%28dra0074%29+is+a+regulatory+protein+that+is+irrelevant+to+recA+induction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">170.
              </span><a name="pone.0000955-Huang2" id="pone.0000955-Huang2"></a>Huang LF, Zhang SW, Hua XT, Gao GJ, Hua YJ (2006) [Construction of the <em>recQ</em> double mutants and analysis of adversity in <em>Deinococcus radiodurans</em>]. Wei Sheng Wu Xue Bao  46: 205–209.  <ul class="find" data-citedArticleID="997710"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=%5BConstruction+of+the+recQ+double+mutants+and+analysis+of+adversity+in+Deinococcus+radiodurans%5D.&amp;auth=&amp;atitle=%5BConstruction+of+the+recQ+double+mutants+and+analysis+of+adversity+in+Deinococcus+radiodurans%5D." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BConstruction+of+the+recQ+double+mutants+and+analysis+of+adversity+in+Deinococcus+radiodurans%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BConstruction+of+the+recQ+double+mutants+and+analysis+of+adversity+in+Deinococcus+radiodurans%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.XML" value="347579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.PDF" value="870062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g001.PNG_L" value="865203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g001.PNG_M" value="64463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g001.PNG_S" value="13949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g001.TIF" value="841174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g001.PNG_I" value="77017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t001.PNG_L" value="90084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t001.PNG_M" value="120799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t001.PNG_S" value="15508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t001.TIF" value="341144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t001.PNG_I" value="40410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g002.PNG_L" value="143677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g002.PNG_M" value="38279"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g002.PNG_S" value="8633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g002.TIF" value="213446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g002.PNG_I" value="12446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t002.PNG_L" value="98815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t002.PNG_M" value="55965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t002.PNG_S" value="8748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t002.TIF" value="414700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t002.PNG_I" value="17313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g003.PNG_L" value="984310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g003.PNG_M" value="113277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g003.PNG_S" value="10587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g003.TIF" value="1782662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g003.PNG_I" value="37599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g004.PNG_L" value="527020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g004.PNG_M" value="131596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g004.PNG_S" value="10472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g004.TIF" value="853892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g004.PNG_I" value="46631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t003.PNG_L" value="878318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t003.PNG_M" value="206756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t003.PNG_S" value="19210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t003.TIF" value="3585540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t003.PNG_I" value="94871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g005.PNG_L" value="4333579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g005.PNG_M" value="408652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g005.PNG_S" value="17200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g005.TIF" value="5982072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g005.PNG_I" value="127096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g006.PNG_L" value="125969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g006.PNG_M" value="42674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g006.PNG_S" value="8823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g006.TIF" value="231846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g006.PNG_I" value="15324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t004.PNG_L" value="503060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t004.PNG_M" value="185064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t004.PNG_S" value="16594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t004.TIF" value="2115914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.t004.PNG_I" value="55453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g007.PNG_L" value="2110132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g007.PNG_M" value="193689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g007.PNG_S" value="30647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g007.TIF" value="4088810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.g007.PNG_I" value="294005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s001.DOC" value="78336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s002.DOC" value="59392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s003.DOC" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s004.DOC" value="118272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s005.DOC" value="60416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s006.DOC" value="245760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s007.DOC" value="82432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s008.DOC" value="48128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s009.DOC" value="133120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s010.DOC" value="134144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s011.DOC" value="36352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s012.TXT" value="235051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s013.DOC" value="52736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s014.DOC" value="47104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s015.DOC" value="65536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s016.DOC" value="169472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s017.DOC" value="228352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s018.DOC" value="151040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s019.DOC" value="82432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000955.s020.DOC" value="114176"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000955&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000955" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000955&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000955&volume=&issue=&title=Deinococcus geothermalis: The Pool of Extreme Radiation Resistance Genes Shrinks&author_name=Kira%20S.%20Makarova%2C%20Marina%20V.%20Omelchenko%2C%20Elena%20K.%20Gaidamakova%2C%20Vera%20Y.%20Matrosova%2C%20Alexander%20Vasilenko%2C%20Min%20Zhai%2C%20Alla%20Lapidus%2C%20Alex%20Copeland%2C%20Edwin%20Kim%2C%20Miriam%20Land%2C%20Konstantinos%20Mavromatis%2C%20Samuel%20Pitluck%2C%20Paul%20M.%20Richardson%2C%20Chris%20Detter%2C%20Thomas%20Brettin%2C%20Elizabeth%20Saunders%2C%20Barry%20Lai%2C%20Bruce%20Ravel%2C%20Kenneth%20M.%20Kemner%2C%20Yuri%20I.%20Wolf%2C%20Alexander%20Sorokin%2C%20Anna%20V.%20Gerasimova%2C%20Mikhail%20S.%20Gelfand%2C%20James%20K.%20Fredrickson%2C%20Eugene%20V.%20Koonin%2C%20Michael%20J.%20Daly&start_page=1&end_page=21" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000955" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000955" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000955" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000955&amp;t=Deinococcus%20geothermalis%3A%20The%20Pool%20of%20Extreme%20Radiation%20Resistance%20Genes%20Shrinks" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000955&title=Deinococcus%20geothermalis%3A%20The%20Pool%20of%20Extreme%20Radiation%20Resistance%20Genes%20Shrinks&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000955&amp;title=Deinococcus%20geothermalis%3A%20The%20Pool%20of%20Extreme%20Radiation%20Resistance%20Genes%20Shrinks" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000955" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000955&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Deinococcus geothermalis: The Pool of Extreme Radiation Resistance Genes Shrinks';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000955';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Deinococcus%20geothermalis%3A%20The%20Pool%20of%20Extreme%20Radiation%20Resistance%20Genes%20Shrinks http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000955" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000955" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="32801" data-articleid="24746"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="24746"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+repair%22" title="Search for articles in the subject area:'DNA repair'"><div class="flagText">DNA repair</div></a>
              <div data-categoryid="17453" data-articleid="24746"
                   data-categoryname="DNA repair"
                   class="flagImage" title="Flag 'DNA repair' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="24746"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="24746"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24746"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Infrared+radiation%22" title="Search for articles in the subject area:'Infrared radiation'"><div class="flagText">Infrared radiation</div></a>
              <div data-categoryid="23701" data-articleid="24746"
                   data-categoryname="Infrared radiation"
                   class="flagImage" title="Flag 'Infrared radiation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Regulator+genes%22" title="Search for articles in the subject area:'Regulator genes'"><div class="flagText">Regulator genes</div></a>
              <div data-categoryid="18579" data-articleid="24746"
                   data-categoryname="Regulator genes"
                   class="flagImage" title="Flag 'Regulator genes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9227'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7363'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4552&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
