

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000811"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000811" />

    <meta name="citation_title" content="Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses"/>
    <meta itemprop="name" content="Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses"/>

      <meta name="citation_author" content="Sanjay Chandriani"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, George Williams Hooper Foundation, University of California at San Francisco, San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Don Ganem"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, George Williams Hooper Foundation, University of California at San Francisco, San Francisco, California, United States of America"/>

    <meta name="citation_date" content="2007/8/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000811.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e811"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Kaposi's sarcoma-associated herpesvirus-like DNA sequences in AIDS-related body-cavity-based lymphomas.; citation_author=E Cesarman; citation_author=Y Chang; citation_author=PS Moore; citation_author=JW Said; citation_author=DM Knowles; citation_journal_title=N Engl J Med; citation_volume=332; citation_number=1; citation_pages=1186-1191; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Identification of herpesvirus-like DNA sequences in AIDS-associated Kaposi's sarcoma.; citation_author=Y Chang; citation_author=E Cesarman; citation_author=MS Pessin; citation_author=F Lee; citation_author=J Culpepper; citation_journal_title=Science; citation_volume=266; citation_number=2; citation_pages=1865-1869; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Kaposi's sarcoma-associated herpesvirus-like DNA sequences in multicentric Castleman's disease.; citation_author=J Soulier; citation_author=L Grollet; citation_author=E Oksenhendler; citation_author=P Cacoub; citation_author=D Cazals-Hatem; citation_journal_title=Blood; citation_volume=86; citation_number=3; citation_pages=1276-1280; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Detection of herpesvirus-like DNA sequences in Kaposi's sarcoma in patients with and without HIV infection.; citation_author=PS Moore; citation_author=Y Chang; citation_journal_title=N Engl J Med; citation_volume=332; citation_number=4; citation_pages=1181-1185; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Epidemiology of Kaposi's sarcoma-associated herpesvirus/human herpesvirus 8.; citation_author=TF Schulz; citation_journal_title=Adv Cancer Res; citation_volume=76; citation_number=5; citation_pages=121-160; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Characterization and cell cycle regulation of the major Kaposi's sarcoma-associated herpesvirus (human herpesvirus 8) latent genes and their promoter.; citation_author=R Sarid; citation_author=JS Wiezorek; citation_author=PS Moore; citation_author=Y Chang; citation_journal_title=J Virol; citation_volume=73; citation_number=6; citation_pages=1438-1446; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Restricted expression of Kaposi sarcoma-associated herpesvirus (human herpesvirus 8) genes in Kaposi sarcoma.; citation_author=W Zhong; citation_author=H Wang; citation_author=B Herndier; citation_author=D Ganem; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=7; citation_pages=6641-6646; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A cluster of latently expressed genes in Kaposi's sarcoma-associated herpesvirus.; citation_author=D Dittmer; citation_author=M Lagunoff; citation_author=R Renne; citation_author=K Staskus; citation_author=A Haase; citation_journal_title=J Virol; citation_volume=72; citation_number=8; citation_pages=8309-8315; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The size and conformation of Kaposi's sarcoma-associated herpesvirus (human herpesvirus 8) DNA in infected cells and virions.; citation_author=R Renne; citation_author=M Lagunoff; citation_author=W Zhong; citation_author=D Ganem; citation_journal_title=J Virol; citation_volume=70; citation_number=9; citation_pages=8151-8154; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Host range of Kaposi's sarcoma-associated herpesvirus in cultured cells.; citation_author=JT Bechtel; citation_author=Y Liang; citation_author=J Hvidding; citation_author=D Ganem; citation_journal_title=J Virol; citation_volume=77; citation_number=10; citation_pages=6474-6481; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A viral gene that activates lytic cycle expression of Kaposi's sarcoma-associated herpesvirus.; citation_author=R Sun; citation_author=SF Lin; citation_author=L Gradoville; citation_author=Y Yuan; citation_author=F Zhu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=11; citation_pages=10866-10871; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Lytic growth of Kaposi's sarcoma-associated herpesvirus (human herpesvirus 8) in culture.; citation_author=R Renne; citation_author=W Zhong; citation_author=B Herndier; citation_author=M McGrath; citation_author=N Abbey; citation_journal_title=Nat Med; citation_volume=2; citation_number=12; citation_pages=342-346; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Kaposi's sarcoma-associated herpesvirus.; citation_author=D Ganem; citation_number=13; citation_pages=2847-2888; citation_date=2007; citation_publisher=Lippincott, Williams & Wilkins; " />
      <meta name="citation_reference" content="citation_title=Reactivation of Kaposi's sarcoma-associated herpesvirus infection from latency by expression of the ORF 50 transactivator, a homolog of the EBV R protein.; citation_author=DM Lukac; citation_author=R Renne; citation_author=JR Kirshner; citation_author=D Ganem; citation_journal_title=Virology; citation_volume=252; citation_number=14; citation_pages=304-312; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Selective switch between latency and lytic replication of Kaposi's sarcoma herpesvirus and Epstein-Barr virus in dually infected body cavity lymphoma cells.; citation_author=G Miller; citation_author=L Heston; citation_author=E Grogan; citation_author=L Gradoville; citation_author=M Rigsby; citation_journal_title=J Virol; citation_volume=71; citation_number=15; citation_pages=314-324; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The lytic switch protein of KSHV activates gene expression via functional interaction with RBP-Jkappa (CSL), the target of the Notch signaling pathway.; citation_author=Y Liang; citation_author=J Chang; citation_author=SJ Lynch; citation_author=DM Lukac; citation_author=D Ganem; citation_journal_title=Genes Dev; citation_volume=16; citation_number=16; citation_pages=1977-1989; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Role of CCAAT/enhancer-binding protein alpha (C/EBPalpha) in activation of the Kaposi's sarcoma-associated herpesvirus (KSHV) lytic-cycle replication-associated protein (RAP) promoter in cooperation with the KSHV replication and transcription activator (RTA) and RAP.; citation_author=SE Wang; citation_author=FY Wu; citation_author=M Fujimuro; citation_author=J Zong; citation_author=SD Hayward; citation_journal_title=J Virol; citation_volume=77; citation_number=17; citation_pages=600-623; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CCAAT/enhancer-binding protein-alpha is induced during the early stages of Kaposi's sarcoma-associated herpesvirus (KSHV) lytic cycle reactivation and together with the KSHV replication and transcription activator (RTA) cooperatively stimulates the viral RTA, MTA, and PAN promoters.; citation_author=SE Wang; citation_author=FY Wu; citation_author=Y Yu; citation_author=GS Hayward; citation_journal_title=J Virol; citation_volume=77; citation_number=18; citation_pages=9590-9612; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Kaposi's Sarcoma-associated herpesvirus open reading frame 50 stimulates the transcriptional activity of STAT3.; citation_author=Y Gwack; citation_author=S Hwang; citation_author=C Lim; citation_author=YS Won; citation_author=CH Lee; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=19; citation_pages=6438-6442; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Kaposi's sarcoma-associated herpesvirus open reading frame 57 encodes a posttranscriptional regulator with multiple distinct activities.; citation_author=JR Kirshner; citation_author=DM Lukac; citation_author=J Chang; citation_author=D Ganem; citation_journal_title=J Virol; citation_volume=74; citation_number=20; citation_pages=3586-3597; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification of a cellular protein that interacts and synergizes with the RTA (ORF50) protein of Kaposi's sarcoma-associated herpesvirus in transcriptional activation.; citation_author=S Wang; citation_author=S Liu; citation_author=MH Wu; citation_author=Y Geng; citation_author=C Wood; citation_journal_title=J Virol; citation_volume=75; citation_number=21; citation_pages=11961-11973; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Oral ganciclovir for patients with cytomegalovirus retinitis treated with a ganciclovir implant. Roche Ganciclovir Study Group.; citation_author=DF Martin; citation_author=BD Kuppermann; citation_author=RA Wolitz; citation_author=AG Palestine; citation_author=H Li; citation_journal_title=N Engl J Med; citation_volume=340; citation_number=22; citation_pages=1063-1070; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Human gammaherpesvirus cytokines and chemokine receptors.; citation_author=J Nicholas; citation_journal_title=J Interferon Cytokine Res; citation_volume=25; citation_number=23; citation_pages=373-383; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human herpesvirus KSHV encodes a constitutively active G-protein-coupled receptor linked to cell proliferation.; citation_author=L Arvanitakis; citation_author=E Geras-Raaka; citation_author=A Varma; citation_author=MC Gershengorn; citation_author=E Cesarman; citation_journal_title=Nature; citation_volume=385; citation_number=24; citation_pages=347-350; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=G-protein-coupled receptor of Kaposi's sarcoma-associated herpesvirus is a viral oncogene and angiogenesis activator.; citation_author=C Bais; citation_author=B Santomasso; citation_author=O Coso; citation_author=L Arvanitakis; citation_author=EG Raaka; citation_journal_title=Nature; citation_volume=391; citation_number=25; citation_pages=86-89; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Transgenic expression of the chemokine receptor encoded by human herpesvirus 8 induces an angioproliferative disease resembling Kaposi's sarcoma.; citation_author=TY Yang; citation_author=SC Chen; citation_author=MW Leach; citation_author=D Manfra; citation_author=B Homey; citation_journal_title=J Exp Med; citation_volume=191; citation_number=26; citation_pages=445-454; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Lytic KSHV infection inhibits host gene expression by accelerating global mRNA turnover.; citation_author=B Glaunsinger; citation_author=D Ganem; citation_journal_title=Mol Cell; citation_volume=13; citation_number=27; citation_pages=713-723; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Highly selective escape from KSHV-mediated host mRNA shutoff and its implications for viral pathogenesis.; citation_author=B Glaunsinger; citation_author=D Ganem; citation_journal_title=J Exp Med; citation_volume=200; citation_number=28; citation_pages=391-398; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Induction of tubulogenesis in telomerase-immortalized human microvascular endothelial cells by glioblastoma cells.; citation_author=E Venetsanakos; citation_author=A Mirza; citation_author=C Fanton; citation_author=SR Romanov; citation_author=T Tlsty; citation_journal_title=Exp Cell Res; citation_volume=273; citation_number=29; citation_pages=21-33; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Open source clustering software.; citation_author=MJ de Hoon; citation_author=S Imoto; citation_author=J Nolan; citation_author=S Miyano; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=30; citation_pages=1453-1454; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Java Treeview–extensible visualization of microarray data.; citation_author=AJ Saldanha; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=31; citation_pages=3246-3248; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ARED: human AU-rich element-containing mRNA database reveals an unexpectedly diverse functional repertoire of encoded proteins.; citation_author=T Bakheet; citation_author=M Frevel; citation_author=BR Williams; citation_author=W Greer; citation_author=KS Khabar; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=32; citation_pages=246-254; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Kaposi's sarcoma-associated herpesvirus infects endothelial and spindle cells.; citation_author=C Boshoff; citation_author=TF Schulz; citation_author=MM Kennedy; citation_author=AK Graham; citation_author=C Fisher; citation_journal_title=Nat Med; citation_volume=1; citation_number=33; citation_pages=1274-1278; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Kaposi's sarcoma-associated herpesvirus gene expression in endothelial (spindle) tumor cells.; citation_author=KA Staskus; citation_author=W Zhong; citation_author=K Gebhard; citation_author=B Herndier; citation_author=H Wang; citation_journal_title=J Virol; citation_volume=71; citation_number=34; citation_pages=715-719; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Expression of HHV-8 latency-associated T0.7 RNA in spindle cells and endothelial cells of AIDS-associated, classical and African Kaposi's sarcoma.; citation_author=M Sturzl; citation_author=C Blasig; citation_author=A Schreier; citation_author=F Neipel; citation_author=C Hohenadl; citation_journal_title=Int J Cancer; citation_volume=72; citation_number=35; citation_pages=68-71; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Monitoring global messenger RNA changes in externally controlled microarray experiments.; citation_author=J van de Peppel; citation_author=P Kemmeren; citation_author=H van Bakel; citation_author=M Radonjic; citation_author=D van Leenen; citation_journal_title=EMBO Rep; citation_volume=4; citation_number=36; citation_pages=387-393; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Microarray data normalization and transformation.; citation_author=J Quackenbush; citation_journal_title=Nat Genet; citation_volume=32 Suppl; citation_number=37; citation_pages=496-501; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inhibition of p300 histone acetyltransferase by viral interferon regulatory factor.; citation_author=M Li; citation_author=B Damania; citation_author=X Alvarez; citation_author=V Ogryzko; citation_author=K Ozato; citation_journal_title=Mol Cell Biol; citation_volume=20; citation_number=38; citation_pages=8254-8263; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=HHV-8 encoded vIRF-1 represses the interferon antiviral response by blocking IRF-3 recruitment of the CBP/p300 coactivators.; citation_author=R Lin; citation_author=P Genin; citation_author=Y Mamane; citation_author=M Sgarbanti; citation_author=A Battistini; citation_journal_title=Oncogene; citation_volume=20; citation_number=39; citation_pages=800-811; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inhibition of interferon signaling by the Kaposi's sarcoma-associated herpesvirus full-length viral interferon regulatory factor 2 protein.; citation_author=S Fuld; citation_author=C Cunningham; citation_author=K Klucher; citation_author=AJ Davison; citation_author=DJ Blackbourn; citation_journal_title=J Virol; citation_volume=80; citation_number=40; citation_pages=3092-3097; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A Kaposi's sarcoma-associated herpesviral protein inhibits virus-mediated induction of type I interferon by blocking IRF-7 phosphorylation and nuclear accumulation.; citation_author=FX Zhu; citation_author=SM King; citation_author=EJ Smith; citation_author=DE Levy; citation_author=Y Yuan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=41; citation_pages=5573-5578; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inhibition of interferon regulatory factor 7 (IRF7)-mediated interferon signal transduction by the Kaposi's sarcoma-associated herpesvirus viral IRF homolog vIRF3.; citation_author=CH Joo; citation_author=YC Shin; citation_author=M Gack; citation_author=L Wu; citation_author=D Levy; citation_journal_title=J Virol; citation_volume=81; citation_number=42; citation_pages=8282-8292; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The role of suppressors of cytokine signaling (SOCS) proteins in regulation of the immune response.; citation_author=WS Alexander; citation_author=DJ Hilton; citation_journal_title=Annu Rev Immunol; citation_volume=22; citation_number=43; citation_pages=503-529; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=AU-rich elements and associated factors: are there unifying principles?; citation_author=C Barreau; citation_author=L Paillard; citation_author=HB Osborne; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=44; citation_pages=7138-7150; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The kaposin B protein of KSHV activates the p38/MK2 pathway and stabilizes cytokine mRNAs.; citation_author=C McCormick; citation_author=D Ganem; citation_journal_title=Science; citation_volume=307; citation_number=45; citation_pages=739-741; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers.; citation_author=TL Bailey; citation_author=C Elkan; citation_journal_title=Proc Int Conf Intell Syst Mol Biol; citation_volume=2; citation_number=46; citation_pages=28-36; citation_date=1994; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000811" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses"/>
    <meta name="twitter:description" content="Lytic infection by Kaposi's sarcoma-associated herpesvirus (KSHV) is associated with an extensive shutoff of host gene expression, mediated chiefly by accelerated mRNA turnover due to expression of the viral SOX protein. We have previously identified a small number of host mRNAs that can escape SOX-mediated degradation. Here we present a detailed, transcriptome-wide analysis of host shutoff, with careful microarray normalization to allow rigorous determination of the magnitude and extent of transcript loss. We find that the extent of transcript reduction represents a continuum of susceptibilities of transcripts to virus-mediated shutoff. Our results affirm that the levels of over 75% of host transcripts are substantially reduced during lytic infection, but also show that another &sim;20% of cellular mRNAs declines only slightly (less than 2-fold) during the course of infection. Approximately 2% of examined cellular genes are strongly upregulated during lytic infection, most likely due to transcriptional induction of mRNAs that display intrinsic SOX-resistance."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000811.g007"/>

  <meta property="og:title" content="Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000811" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6825'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4397'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6700&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000811">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000811" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000811&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sanjay Chandriani, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, George Williams Hooper Foundation, University of California at San Francisco, San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Don Ganem
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ganem@cgl.ucsf.edu">ganem@cgl.ucsf.edu</a></p>

                <p>Affiliation:
                  Howard Hughes Medical Institute, George Williams Hooper Foundation, University of California at San Francisco, San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 29, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000811</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000811" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000811" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000811" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000811" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000811-g001" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000811-g002" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000811-g003" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000811-g004" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000811-t001" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000811-g005" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000811-g006" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000811-g007" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000811">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000811" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Lytic infection by Kaposi's sarcoma-associated herpesvirus (KSHV) is associated with an extensive shutoff of host gene expression, mediated chiefly by accelerated mRNA turnover due to expression of the viral SOX protein. We have previously identified a small number of host mRNAs that can escape SOX-mediated degradation. Here we present a detailed, transcriptome-wide analysis of host shutoff, with careful microarray normalization to allow rigorous determination of the magnitude and extent of transcript loss. We find that the extent of transcript reduction represents a continuum of susceptibilities of transcripts to virus-mediated shutoff. Our results affirm that the levels of over 75% of host transcripts are substantially reduced during lytic infection, but also show that another ~20% of cellular mRNAs declines only slightly (less than 2-fold) during the course of infection. Approximately 2% of examined cellular genes are strongly upregulated during lytic infection, most likely due to transcriptional induction of mRNAs that display intrinsic SOX-resistance.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Chandriani S, Ganem D (2007) Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses. PLoS ONE 2(8):
          e811.
            doi:10.1371/journal.pone.0000811</p><p><strong>Academic Editor: </strong>Dong-Yan Jin, University of Hong Kong, China</p><p><strong>Received:</strong> June 13, 2007; <strong>Accepted:</strong> July 30, 2007; <strong>Published:</strong> August 29, 2007</p><p><strong>Copyright:</strong> © 2007 Chandriani, Ganem. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>All work was funded by the Howard Hughes Medical Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Kaposi's sarcoma-associated herpesvirus (KSHV) is a lymphotropic (gamma-2) herpesvirus that is etiologically linked to Kaposi's sarcoma (KS) as well as to several lymphoproliferative syndromes <a href="#pone.0000811-Cesarman1">[1]</a>–<a href="#pone.0000811-Schulz1">[5]</a>. Like all herpesviruses, KSHV displays two alternative genetic programs, latency and lytic replication. In latency, viral gene expression is restricted to a handful of genes <a href="#pone.0000811-Sarid1">[6]</a>–<a href="#pone.0000811-Dittmer1">[8]</a>, and the viral genome is maintained in the nucleus as a low copy-number episome <a href="#pone.0000811-Renne1">[9]</a>; host gene expression continues unabated, and no viral progeny are produced. Latent infection is the default program for viral infection in culture <a href="#pone.0000811-Bechtel1">[10]</a>. However, latently infected cells retain the full viral genome and, under the appropriate conditions, can be induced to enter the lytic cycle <a href="#pone.0000811-Sun1">[11]</a>, <a href="#pone.0000811-Renne2">[12]</a>. In this state, the majority of viral genes are expressed according to a temporally regulated program, with immediate-early (IE) genes expressed first, followed by delayed-early (DE) genes <a href="#pone.0000811-Ganem1">[13]</a>. IE proteins generally serve as regulators of the subsequent classes; the key IE protein is RTA (<span class="underline">r</span>eplication and <span class="underline">t</span>ranscription <span class="underline">a</span>ctivator), a transcription factor that is responsible for the switch from latent to lytic replication <a href="#pone.0000811-Sun1">[11]</a>, <a href="#pone.0000811-Lukac1">[14]</a>, <a href="#pone.0000811-Miller1">[15]</a>. Many DE genes are directly upregulated by RTA, which can activate transcription by direct DNA binding or by recruitment to other promoter sites through binding to cellular transcription factors (RBP-Jk, C/EBPα, and others)<a href="#pone.0000811-Liang1">[16]</a>–<a href="#pone.0000811-Wang3">[21]</a>. DE gene expression triggers lytic DNA replication, following which late (L) genes, predominantly encoding virion structural proteins, are expressed. During the late phase of lytic replcation, infectious viral progeny are assembled and released in large numbers.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Although most tumor cells of KS are latently infected, the lytic cycle is thought to play an important role in KS tumorigenesis, since ganciclovir treatment, which specifically blocks lytic replication, leads to a prompt and significant reduction in KS development even after many years of KSHV infection <a href="#pone.0000811-Martin1">[22]</a>. This suggests that the continuous operation of the lytic cycle in some fraction of the infected cells is necessary to sustain KS tumorigenesis. How the lytic cycle contributes to KS development has been a matter of debate <a href="#pone.0000811-Ganem1">[13]</a>. One model posits that growth and angiogenic factors released from lytically infected cells may influence tumor progression in a paracrine fashion. Many such factors are encoded by DE viral genes that have been identified, which include virally encoded cytokines and chemokines (e.g. v-GPCR, v-CCL1, v-CCL2, v-CCL3, v-IL6) (review,<a href="#pone.0000811-Nicholas1">[23]</a>). An important remaining question has been whether host genes encoding such factors can be induced by lytic infection. For example, the G-protein coupled receptor encoded by the DE gene ORF74 <a href="#pone.0000811-Arvanitakis1">[24]</a> can induce VEGF production when expressed in uninfected cells in culture <a href="#pone.0000811-Bais1">[25]</a>, and can result in angiogenesis in surrounding tissues when expressed as a murine transgene <a href="#pone.0000811-Yang1">[26]</a>. However, the relevance of these in vitro observations to in vivo infection depends on whether the indicated host genes can be expressed in the environment of lytic infection. Since some (but not all) herpesviruses induce a shutoff of host gene expression during their lytic cycle, investigation of the ability of KSHV-infected cells to support host gene expression is important to evaluating the potential contributions of host gene products to paracrine signaling.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>For this reason, our laboratory recently examined whether host gene expression is affected during the KSHV lytic cycle. Our results showed that KSHV infection leads to a rapid and extensive shutoff of host protein synthesis, as judged by pulse labeling with <sup>35</sup>S-methionine <a href="#pone.0000811-Glaunsinger1">[27]</a>. Further studies revealed that this is due to the action of a single viral gene, now termed SOX (<span class="underline">s</span>hut-<span class="underline">o</span>ff and e<span class="underline">x</span>onuclease), which leads to a large-scale degradation of host mRNAs <a href="#pone.0000811-Glaunsinger1">[27]</a>. Subsequently, we examined the profiles of host mRNAs in infected cells to see if any host transcripts could escape SOX-mediated decay <a href="#pone.0000811-Glaunsinger2">[28]</a>. In those studies, microarray data were normalized with the common assumption that the levels of relatively few transcripts change from condition to condition. Therefore, we were able to identify only the most highly expressed escapees that appear to be resistant to the inhibitory effects of SOX. Here, we re-examine host mRNA expression in lytically infected cells using a normalization algorithm that can reveal global gene expression changes if they occur; this allows a more comprehensive view of the magnitude and extent of host mRNA transcript loss. Our results show that circa 75% of transcripts are massively downregulated during lytic replication, with fewer than 2% undergoing active upregulation during the lytic cycle. Moreover, we identify a previously unrecognized class of mRNA whose levels decline only modestly (less than 2-fold) during lytic growth. This class is not small, representing approximately 20% of transcripts studied. Our results support the view that there is a gradient of susceptibility to SOX regulation among host transcripts, but affirm that <em>de novo</em> upregulation of host mRNAs in response to infection is limited to a very small subset of the transcriptome; this subset includes several paracrine signaling molecules, though VEGF is not prominent among them.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Tissue culture, virus preparation and infection</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>TIME cells were cultured as previously described<a href="#pone.0000811-Venetsanakos1">[29]</a>. Briefly, cells were expanded in EGM™-2 MV (Clonetics). KSHV virus stocks were prepared from BCBL-1 cells as previously described <a href="#pone.0000811-Bechtel1">[10]</a>. Virus titers were empirically determined by serial dilution and infection of TIME cells. For expression profiling experiments, cells were infected with KSHV (or mock infected) under conditions that yielded approximately 95% latently infected cells 48 hours post infection as judged by staining for the latency-associated nuclear antigen (LANA) (data not shown). After 6 hours of exposure to the KSHV inoculum, cells were washed with PBS and subjected to infection by Ad-RTA (adenovirus harboring the RTA cDNA) under conditions that yielded nearly 80% of KSHV-infected cells undergoing lytic cycle 48 hours later as judged by staining for the lytic marker, Orf59. Time = 0hr refers to the time at which Ad-RTA was applied to cells.</p>


<h4>RNA preparation, labeling, and microarray hybridization</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Total RNA was prepared from cells at times and conditions indicated in the text using the RNeasy Mini Kit (Qiagen) according to the manufacturer's protocol. Reference RNA was a mixture of RNAs purified from dividing untransduced human foreskin fibroblast, TIME cells and human umbilical vein endothelial cells (HUVEC). The integrity of the purified RNA's was analyzed using the 2100 Bioanalyzer (Agilent). RNAs were quantified using the ND1000 spectrophotometer (Nanodrop). The Low RNA Input Linear Amplification Kit (Agilent) was used according to the manufacturer's protocol to generate labeled cRNA from 240ng of total RNA. Experimental samples (labeled with Cy5) and reference samples (labeled with Cy3) were competitively hybridized to Whole Human Genome Oligo Microarrays (Agilent) according to the manufacturer's protocol. These microarrays have 41,000 probes that map to 20,087 unique unigene cluster IDs (Unigene Build #188). Cyanine 3-CTP and Cyanine 5-CTP were obtained from Perkin Elmer. Hybridizations and washes were performed according to the manufacturer's protocol. Washed arrays were scanned using the 48-slide DNA Microarray Scanner (Agilent) and feature intensities extracted using Feature Extraction Software version 8.5 (Agilent). All raw data were stored in the MIAME (minimum information about a microarray experiment)-compliant Princeton University MicroArray Database (PUMAdb) and can be accessed at <a href="http://puma.princeton.edu/">http://puma.princeton.edu/</a>. Fully normalized and processed data can be accessed at <a href="http://www.ucsf.edu/micro/faculty/ganem_folder/data/chandriani/shutoff/KSHV_lytic.html">http://www.ucsf.edu/micro/faculty/ganem_​folder/data/chandriani/shutoff/KSHV_lyti​c.html</a>.</p>


<h4>Microarray Data Analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Features on the array that were flagged as “Population outlier” or “Non-uniformity outlier” were removed from the data and not considered further. Both green and red channel intensities were required to be well above background. Furthermore, only features for which the green channel median intensity was greater than 85 and the red channel median intensity was greater than 110 were considered for further analysis. LOWESS normalized log<sub>2</sub>-ratios of features that passed these spot quality filters were downloaded from PUMAdb. For the second normalization step, we utilized signals from probes against the spiked transcripts that passed the following spot quality filters: were not feature or background “non-uniformity outliers”, were not saturated and were “well above background.” The LOWESS normalized log<sub>2</sub>-ratios for these features were averaged for each array. This average value constitutes the second normalization factor linearly applied to the remaining probes on the array.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The completely normalized data were then subject to zeroing and other filters. For a given probe in the time course series, the average log<sub>2</sub>-ratio across the two zero hour samples (mock KSHV) was subtracted from every log<sub>2</sub>-ratio expression value across the lytic replication time course experiments. In the time course data, probes that displayed at least a 1.86 fold change (from the 0 hr, mock KSHV) in at least two time points and had greater than 75% present data were considered responsive. As explained in the text, one unique cluster of 66 probes whose changes could not be validated by quantitative RT-PCR was removed for further consideration (the complete list of these probes is available in supplementary <a href="#pone.0000811.s003">Table S1</a>). These filters yielded a group of 10,006 probes.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Because we did not expect global changes in host gene expression upon over expression of vGPCR, RTA or EGFP, we only performed LOWESS normalization on these 9 arrays. A similar zeroing was conducted in this follow-up study except the three mock infected samples served as the zeroes. Probes were filtered for 80% present data and then filtered for at least two arrays displaying a 1.86 fold change over the average of the mock infected samples. These filters yielded a group of 2,392 probes.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Hierarchical clustering of data was performed with Cluster 3.0 <a href="#pone.0000811-deHoon1">[30]</a>. Clustered data are displayed at a heat map using Java TreeView <a href="#pone.0000811-Saldanha1">[31]</a>.</p>


<h4>ARE enrichment</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To identify ARE-containing transcripts, 3′UTRs for all available transcripts represented on the microarray were downloaded from the BioMart Project (<a href="http://www.biomart.org/biomart/martview">http://www.biomart.org/biomart/martview</a>). 3′ UTR sequences were available for 15,372 genes represented on the array. This information was used to query genes that contain the octomer ARE sequence, UAUUUAWW (W = A or U), at least twice in their 3′ UTRs <a href="#pone.0000811-Bakheet1">[32]</a>. The significance of enrichment of ARE-containing transcripts among the escapees over the background representation in all the genes on the array was calculated by assuming a hypergeometric distribution of ARE elements in 3′ UTRs. Analysis was performed in Matlab.</p>


<h4>qRT-PCR</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Quantitative RT-PCR was performed to corroborate microarray results for several genes. cDNA was generated using random primers with 2ug of total RNA using Superscript III reverse transcriptase according to the manufacturer's protocol (Invitrogen). A small fraction (1/50) of the reaction serves as the template for real-time PCR using specific primer/probe sets for each gene from Applied Biosystems. Quantitative PCR was performed using the 7300 Real Time PCR system (Applied Biosystems). Because host shutoff would likely affect commonly used transcripts for normalization, we also measured rRNA levels by RT-PCR and used this data for normalization of the qPCR results. The ddCt data (from qPCR) can be interpreted as normalized expression levels in log<sub>2</sub> space. This data is therefore easily compared to the microarray data, which is presented as log<sub>2</sub>-ratios.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Experimental design</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Many different cell types comprise the hallmark angioproliferative lesions in KS; however, only the spindle cells, which are of endothelial origin, are infected with KSHV <a href="#pone.0000811-Boshoff1">[33]</a>–<a href="#pone.0000811-Sturzl1">[35]</a>. Therefore, we designed our experiment to capture transcriptome-wide changes in a microvascular endothelial cell line called TIME cells <a href="#pone.0000811-Venetsanakos1">[29]</a>. Time cells were first treated with mock or KSHV inoculum for 6 hours. Inoculum was removed, cells were washed and then subjected to infection by an adenovirus construct encoding KSHV RTA (Ad-RTA). Total RNA was harvested at 0, 1.5, 3, 13, 24, 37 and 48 hours following Ad-RTA infection. Duplicate (for the mock KSHV infected cells) or triplicate (for the KSHV infected cells) zero time points were collected whereas single samples were collected for the ensuing time points. To evaluate host gene expression changes in response to isolated expression of either vGPCR or RTA, naïve TIME cells were infected in duplicate with adenovirus constructs encoding for vGPCR, RTA, EGFP or mock infected (in triplicate). Total RNA was harvested 48 hours post infection. Genome wide expression studies were performed using Whole Human Genome Oligo Microarrays from Agilent (see <a href="#s2">materials and methods</a>).</p>


<h4>Microarray normalization</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Commonly, microarray data normalization methods assume that relatively few transcripts change from sample to sample <a href="#pone.0000811-vandePeppel1">[36]</a>. Therefore, most normalization algorithms of two-color microarray data effectively move the distribution of log(R/G) of all the features on an array to zero <a href="#pone.0000811-Quackenbush1">[37]</a>. Even the more sophisticated normalization algorithms (such as LOWESS) that apply a non-linear correction to the features to accommodate intensity dependent biases also have an underlying assumption that relatively few transcripts change from sample to sample. When global changes in gene expression are predicted, external normalization controls can be used to better assess these changes<a href="#pone.0000811-vandePeppel1">[36]</a>. We used a modification of a method described by van de Peppel et al. <a href="#pone.0000811-vandePeppel1">[36]</a> to assess transcript changes during lytic replication; in this method, an equal amount of a mixture of 10 different in-vitro synthesized transcripts are added to an equal amount of total RNA of each sample. The Agilent microarray harbors probes that specifically recognize these spiked transcripts and can serve as critical internal standards for normalization. In so doing, the underlying assumption is that total RNA levels do not substantially change from sample to sample. (We experimentally validated this assumption by comparing total RNA yields from cells that are undergoing lytic replication and those which are not; data not shown). We used these exogenous transcripts to normalize the endogenous transcripts in a two-step normalization protocol. In the first normalization step, we utilized a LOWESS algorithm based on all the probes of the array. In the second step, we performed a linear normalization based only on the intensities of the probes for the spiked transcripts. We compared the microarray data normalized under the assumption that no global effects are taking place (LOWESS) versus data normalized without that assumption (two-step normalization) and observed starkly different expression profiles (supplementary <a href="#pone.0000811.s001">Figure S1</a>). To empirically validate the normalization protocol that better approximates changes in transcript levels, we used quantitative RT-PCR as an independent method to measure transcript level changes of more than 30 different mRNAs (see below). The subsequent RT-PCR data affirm (see below) that the microarray data normalization using spike-in controls more accurately reflects underlying changes to cellular transcripts during lytic replication.</p>


<h4>Transcript accumulation during KSHV lytic replication</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>To display transcripts that show even mild changes over the course of infection, a non-stringent fold filter (at least a 1.9-fold change in at least 2 time points) was applied to the expression data (<a href="#pone-0000811-g001">Figure 1</a>). Immediately, it is apparent that the levels of the vast preponderance of RNA transcripts decline substantially and progressively with infection, as predicted by our earlier studies of SOX-transfected cells. We sampled 16 representative examples of these RNAs by quantitative RT-PCR to independently confirm the downregulation seen on the array; the direction of transcript level change (i.e. down) was confirmed in every case (<a href="#pone-0000811-g002">Figure 2</a>). More importantly, as a validation of the two-step normalization, the magnitude of the changes as assessed by quantitative RT-PCR agreed very closely with the magnitude of changes suggested by the microarray data. Only a small number of mRNAs actually increase in abundance during infection; many, but by no means all, of these appear to be inducible by Ad-RTA alone (see below). <a href="#pone-0000811-g003">Figure 3</a> displays a schematic depiction of the array results, grouped according to the nature and magnitude of transcript level changes occurring during lytic growth. Over 75% of the host mRNAs whose signals met our stringent spot quality filters were strongly downregulated during lytic growth. To our surprise, another 22% of host transcripts declined only modestly in abundance during infection (mRNAs were assigned to this class if their levels declined by less than 1.9 fold during infection and are not displayed in the heat map of <a href="#pone-0000811-g001">Figure 1</a>). Again, quantitative RT-PCR analysis of 5 out of 5 randomly chosen members of this class also affirmed the array results (<a href="#pone-0000811-g002">Figure 2</a>). We think it likely that most of these transcripts are only weakly responsive to SOX-induced turnover; however, we cannot exclude that some of them are RNAs whose SOX-mediated turnover is balanced by transcriptional upregulation. Another small subset of mRNAs may be erroneously included in this class. We believe that some transcripts are so severely downregulated that the late time points fail to pass the spot quality filter of having spot intensity well above background; such RNAs would be assigned to this class by the above analysis. Overall, these results indicate that circa 98% of host transcripts remain at or below pre-infection levels during KSHV lytic growth.</p>
<div class="figure" id="pone-0000811-g001"><div class="img"><a name="pone-0000811-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Host (TIME cells) mRNA expression data during KSHV lytic infection.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Clustered microarray data are displayed for 10,006 probes exhibiting a 1.9 fold change (relative to the average of the 0hr timepoints of the mock KSHV/Ad-RTA infected cells) in at least two time points. Detailed description of data filters is available in the <a href="#s2">Materials and Methods</a>. The color bar describes the fold changes with respect to mock infected cells at time zero.</p>
<span>doi:10.1371/journal.pone.0000811.g001</span></div><div class="figure" id="pone-0000811-g002"><div class="img"><a name="pone-0000811-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Validation of microarray results by quantitative RT-PCR.</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p>Normalized log<sub>2</sub>-ratios of the microarray data and the average ddCt qPCR values of select genes are directly compared in colorimetric form. Probe names are given for the microarray data and the corresponding short gene names are given along side the qRT-PCR data. Asterisk (*) indicates microarray data that were not confirmed by qRT-PCR. As noted in the Results section, these probes are part of the cluster of microarray probes that was removed from subsequent analyses. The color bar refers to the fold changes with respect to mock infected cells at time zero.</p>
<span>doi:10.1371/journal.pone.0000811.g002</span></div><div class="figure" id="pone-0000811-g003"><div class="img"><a name="pone-0000811-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Summary of changes in transcript levels during KSHV lytic infection.</span></strong></p><a id="article1.body1.sec3.sec3.fig3.caption1.p1" name="article1.body1.sec3.sec3.fig3.caption1.p1"></a><p>Pie chart represents the total number (8,817) of unique unigene cluster IDs that pass microarray spot quality filters in 75% of arrays. The three sectors represent genes that display changes in the indicated fashion. The filters used to define the induced and repressed are the same as in <a href="#pone-0000811-g001">Figure 1</a>. The unresponsive sector represents genes that did not pass the fold filter, but did pass the spot quality filters.</p>
<span>doi:10.1371/journal.pone.0000811.g003</span></div>

<h4>mRNAs that escape KSHV mediated host shutoff</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Approximately 2% of assayed host mRNAs increased in abundance during infection. In our initial analyses, several small clusters of seemingly upregulated transcripts were observed (see supplemental <a href="#pone.0000811.s002">Figure S2</a>). However, for one of these clusters quantitative RT-PCR measurements failed to confirm the upregulation in 6 out of 6 cases (<a href="#pone-0000811-g002">Figure 2</a>). The reasons for this remain obscure, however, we suspect that some viral transcripts that are highly expressed during lytic replication may cross-hybridize to these microarray probes. Therefore, the 66 probes, which map to 54 genes, in this entire cluster were excluded from the summary analyses of <a href="#pone-0000811-g003">Figure 3</a>. Other upregulated mRNAs outside of this anomalous cluster were readily confirmed by qRT-PCR (9 out of 9 representative samples) (<a href="#pone-0000811-g002">Figure 2</a>). <a href="#pone-0000811-g004">Figure 4</a> shows the heat map of upregulated mRNAs in expanded form. As expected, several of these transcripts correspond to genes whose upregulation we had earlier observed by qualitative array analysis and whose upregulation at the protein level was confirmed by immunoassay or immunoblotting (e.g. IL-6) <a href="#pone.0000811-Glaunsinger2">[28]</a>. However, the present analysis greatly expands the list of identified upregulated RNAs, from 9 to 177. We attribute this to more accurate quantitation and normalization of transcript abundances, owing to the use of spiked transcript controls, as well as the use of larger arrays.</p>
<div class="figure" id="pone-0000811-g004"><div class="img"><a name="pone-0000811-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>mRNAs that escape KSHV mediated host shutoff.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>An expanded view of one cluster of mRNAs that exhibit elevated levels during lytic replication is displayed. Probes that map to select mRNAs are annotated. The color bar describes the fold changes with respect to mock infected cells at time zero.</p>
<span>doi:10.1371/journal.pone.0000811.g004</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>A full accounting of these upregulated transcripts is available in supplementary data (supplementary <a href="#pone.0000811.s004">Table S2</a>). While we do not yet understand the biology of all these gene products, there are tantalizing clusters of mRNAs with related functions. First, this endothelial cell line produces a number of proteins with activities in angiogenesis and vascular cell biology, including angiopoietin 2, angiopoietin-like polypeptide, ephrins A1 and B2, endothelin convertase and serpin B2 (plasminogen activator inhibitor 2). Second, several interferon-inducible transcripts accumulate (e.g. IFI 6-16, IFI 15, IFI 27, and GBP2), though the full complement of interferon-responsive mRNAs is not detectable. (SOX may degrade many of the latter.) In addition, however, we and others have recently found that KSHV encodes several lytic functions that blockade IFN signaling (Bisson, Page and Ganem, manuscript in preparation) <a href="#pone.0000811-Li1">[38]</a>–<a href="#pone.0000811-Joo1">[42]</a>. It therefore remains possible that the small set of upregulated transcripts labeled as IFN-responsive are in fact being upregulated by other, as yet unknown, cellular pathways. The arrays also reveal ample evidence of cytokine dysregulation, including the profound induction of IL6 and CXCL2. The upregulation of IL6 (and the production of v-IL6) may explain another observation, namely, the upregulation of the counter-regulatory molecule SOCS-1 (suppressor of cytokine signaling), which acts to impair STAT 3 function downstream of the IL6 receptor <a href="#pone.0000811-Alexander1">[43]</a>; however, many other explanations for SOCS1 upregulation are also possible. In addition, many molecules active in other cellular signal transduction pathways are upregulated, including many dual-specificity protein phosphatases, several orphan GPCRs, several ubiquitin ligases and components of the Notch signaling pathway (e.g. jagged 1, a ligand for Notch). Finally, numerous transcription factors are upregulated (e.g. FOXC1, ATF3, MADS-box TF1, HoxA5), but the significance of this remains to be established, since many of their induced transcripts will likely be subject to SOX-mediated decay.</p>


<h4>Enrichment of transcripts harboring ARE's among escapees</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>It is likely that there are multiple mechanisms by which transcripts escape SOX-mediated degradation. We have earlier shown that cis-acting sequences in transcripts play an important role in making transcripts refractory to SOX-mediated degradation <a href="#pone.0000811-Glaunsinger2">[28]</a>. Accordingly, we were interested to know if any cis-elements with known effects on RNA stability were over-represented in escapees of SOX-mediated decay. One well-known class of such elements are the so-called AU-rich elements (AREs). AREs bind multiple cellular factors, some of which stabilize and others of which destabilize the RNA <a href="#pone.0000811-Barreau1">[44]</a>. In the ground state, the net result of this is destabilization of the transcript. However, certain stimuli (notably p38 activation) can inactivate one or more of the destabilizing proteins (e.g. tristetraprolin or TTP), resulting in net stabilization of ARE-containing mRNAs. Interestingly, KSHV encodes a protein (Kaposin B) that upregulates the p38 pathway and stabilizes ARE transcripts, and this protein is highly expressed during lytic replication <a href="#pone.0000811-McCormick1">[45]</a>. Accordingly, we examined whether AREs might be enriched among the class of transcripts that escape shutoff. Indeed, when we examined the 3′ UTR sequences of the escapees, we found that this group of transcripts was enriched by 50% for transcripts that had two or more AREs in their 3′ UTRs (<a href="#pone-0000811-t001">Table 1</a>). Assuming a hypergeometric distribution, this enrichment is statistically significant (p-value = 1.0×10<sup>−3</sup>). However, we note that (i) many escapees lack AREs, and (ii) that many RNAs that are downregulated by SOX harbor AREs. Indeed, in one prominent SOX-resistant mRNA (IL6) that contains an ARE, mutation of the ARE does not restore SOX-susceptibility (Britt Glaunsinger, personal communication). Taken together, these data suggest that AREs cannot be the sole determinant of SOX resistance; multiple factors are likely at work.</p>
<div class="figure" id="pone-0000811-t001"><div class="img"><a name="pone-0000811-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1. </strong></p><span>doi:10.1371/journal.pone.0000811.t001</span></div>

<h4>Host gene expression changes induced by GPCR, RTA</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>What is the origin of the upregulation of these 177 mRNAs? Two viral gene products that have been much discussed in these contexts are RTA and the viral GPCR (orf74). As shown in <a href="#pone-0000811-g005">Figure 5</a>, when each is expressed individually in TIME cells by an adenovirus vector, large numbers of host transcripts are induced. The set of upregulated genes induced by each regulator is distinct but overlapping. <a href="#pone-0000811-g006">Figure 6</a> shows that when the set of GPCR-upregulated genes is considered, the vast majority are downregulated during lytic infection; only approximately 5% are upregulated during lytic growth. This result affirms the need for caution in extrapolating from results with GPCR-transfected cells to the circumstance of authentic infection.</p>
<div class="figure" id="pone-0000811-g005"><div class="img"><a name="pone-0000811-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Host mRNA expression data upon expression RTA, vGPCR and GFP.</span></strong></p><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1"></a><p>RTA, vGPCR or GFP were ectopically expressed in TIME cells. 48hrs after adenoviral delivery of these cDNAs, samples were harvested and gene expression analysis was performed. Clustered microarray data are displayed for 2,392 probes exhibiting a 1.9 fold change (relative to the average of the three mock infected samples) in at least two arrays. Detailed description of the data filters is available in the <a href="#s2">Methods</a>.</p>
<span>doi:10.1371/journal.pone.0000811.g005</span></div><div class="figure" id="pone-0000811-g006"><div class="img"><a name="pone-0000811-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>vGPCR responsive transcripts during lytic replication.</span></strong></p><a id="article1.body1.sec3.sec6.fig2.caption1.p1" name="article1.body1.sec3.sec6.fig2.caption1.p1"></a><p>Microarray data in <a href="#pone-0000811-g001">figures 1</a> and <a href="#pone-0000811-g003">3</a> were linked. Clustered data are displayed for 1,902 probes exhibiting at least a 1.9 fold change (relative to the average of the three mock infected samples) in both vGPCR arrays. Detailed description of data filters is available in the <a href="#s2">Methods</a>.</p>
<span>doi:10.1371/journal.pone.0000811.g006</span></div><a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p><a href="#pone-0000811-g007">Figure 7</a> shows another way to depict the potential contributions of GPCR and RTA to those few genes that are upregulated during lytic growth. Of the 169 probes analyzed, only 14% were uniquely induced by vGPCR expression; fully 30% were induced solely by RTA expression, and another 37% can be upregulated by either viral protein. Of course, this correlative analysis does not prove that these host genes are being upregulated by <em>these</em> viral proteins in vivo; this analysis is presented to outline the maximal potential contributions of each regulator to the observed patterns). Notably, nearly 18% of upregulated genes could be induced by neither viral factor, suggesting that other viral regulators likely also play important roles–candidates for these include the K1, K8, K14, MTA (orf 57) and K15 proteins. Dissecting the relative contributions of each of these factors to the overall pattern of host mRNA accumulation will likely require genetic and functional genomic analyses of each coding region, carried out in the context of lytic KSHV replication.</p>
<div class="figure" id="pone-0000811-g007"><div class="img"><a name="pone-0000811-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000811" data-uri="info:doi/10.1371/journal.pone.0000811.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000811.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000811.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000811.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000811.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Response of escapees to isolated expression of RTA, vGPCR or GFP.</span></strong></p><a id="article1.body1.sec3.sec6.fig3.caption1.p1" name="article1.body1.sec3.sec6.fig3.caption1.p1"></a><p>A. Expression data for mRNAs that escape KSHV mediated host shutoff (<a href="#pone-0000811-g001">Fig. 1</a>) are selected (169 probes) from the dataset presented in <a href="#pone-0000811-g003">Fig. 3</a>.</p>
<a id="article1.body1.sec3.sec6.fig3.caption1.p2" name="article1.body1.sec3.sec6.fig3.caption1.p2"></a><p>B. These data are summarized in a pie chart that depicts the influence of vGPCR and RTA on lytic escapee transcripts.</p>
<span>doi:10.1371/journal.pone.0000811.g007</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>These studies represent a comprehensive examination of the host endothelial transcriptome during lytic infection by KSHV, using methods that allow quantitative assessment of the magnitude and scope of mRNA abundance changes as infection progresses. The data reveal that while a large majority of host transcripts (~75%) succumb strongly to SOX-mediated degradation, a sizeable minority of transcripts (~20%) is only mildly affected by SOX. Furthermore, a small subset of transcripts (~2%) not only escapes host shutoff, but is in fact more highly expressed. We note here that although we have chosen (for clarity and simplicity) to categorize the responses to lytic infection as falling into 3 discrete classes, there is in fact a continuum of changes in host transcript levels ranging from strong induction to reduction below detection limits.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>What accounts for the variable responses of host transcripts to the lytic environment? At present, we do not have definitive answers to this question, but we can point out several possibilities. First, some transcripts may bear cis-acting sequences that confer resistance to SOX-mediated degradation. We have previously described one such sequence in the 3′ UTR of IL-6 mRNA, an RNA that strongly accumulates as lytic infection progresses. However, these sequences are not conserved in other RNAs whose abundance increases during infection, suggesting that additional cis-acting elements may exist that contribute to escape from SOX-mediated turnover. Our analysis shows that ARE elements are enriched in RNAs that escape degradation during infection. This is interesting since KSHV encodes a protein, kaposin B, that is strongly upregulated during lytic growth and that stabilizes ARE-containing messages via activation of the p38/MK2 pathway <a href="#pone.0000811-McCormick1">[45]</a>. However, this appealing model is complicated by the fact that many RNAs that contain AREs are nevertheless degraded during infection, indicating that other effects can override ARE-mediated stabilization. How such effects might operate is unknown. Stimulated by the observation that AREs are enriched in 3′ UTRs of transcripts that escape degradation, we searched for other elements that are conserved and enriched among this group of transcripts. Using MEME (Multiple Em for Motif Elicitation <a href="#pone.0000811-Bailey1">[46]</a>), several conserved elements were detected in these 3′ UTRs; however, none of these was enriched among the escapees when compared to the background representation in the 3′ UTRs of all the genes on the array (data not shown). This finding does not exclude that possibility that cis-acting elements are functional in restraining degradation since elements may be in other parts of the transcript or are in a form that the MEME algorithm could not detect.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Another possibility is that some cis elements can direct mRNAs to areas of the cytoplasm where SOX-mediated decay is more active; variations in the efficiency of such targeting could also lead to variations in transcript accumulation during infection. The idea that cytoplasmic compartmentation of transcripts plays a role in accessibility to the degradative machinery could also provide an attractive explanation for why viral mRNAs are not degraded during lytic replication. Finally, we note that even without regional or other variations in RNA turnover efficiency, other mechanisms can produce variation in the net accumulation of transcripts. Most simply, strongly enhanced transcription of some mRNAs could lead to their net accumulation by simply outrunning the degradative rate. Of course, none of these mechanisms is mutually exclusive, and the complex phenotype observed in vivo could well be due to contributions from all of these mechanisms. Clearly, much remains to be learned about the regulation of RNA stability during lytic KSHV growth. Resolution of these issues can be expected to inform our understanding of both KSHV pathogenesis and the control of host mRNA turnover more generally.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000811"></div><a name="pone.0000811.s001" id="pone.0000811.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000811.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Comparison of microarray data normalized using LOWESS or two step using spiked-in transcripts) algorithm. Microarray data for experiment in <a href="#pone-0000811-g001">Figure 1</a> were normalized using two different algorithms. These data were then linked together. The probes in <a href="#pone-0000811-g001">Figure 1</a> were extracted, clustered and displayed as this heat map. The color bar describes the fold changes with respect to mock infected cells at time zero.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000811.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.85 MB DOC)</p>
<a name="pone.0000811.s002" id="pone.0000811.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000811.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Microarray data including small cluster of probes which was removed from analysis. Clustered microarray data are displayed for probes exhibiting a 1.9 fold change (relative to the average of the 0hr timepoints of the mock KSHV/Ad-RTA infected cells) in at least two time points. These data include the small group of probes that were discarded from all analysis after qRT-PCR failed to validate this group. The color bar describes the fold changes with respect to mock infected cells at time zero.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000811.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.55 MB DOC)</p>
<a name="pone.0000811.s003" id="pone.0000811.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000811.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Pre-clustering file of microarray data presented in <a href="#pone-0000811-g001">Figure 1</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000811.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(2.32 MB TDS)</p>
<a name="pone.0000811.s004" id="pone.0000811.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000811.s004">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Pre-clustering file of microarray data of the escapees.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000811.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB TDS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Joseph Ziegelbauer, Andrew Dufresne and Britt Glaunsinger for critical reading of the manuscript, Anatoly Urisman for help with bioinformatics analysis and programming, and all the members of the Ganem laboratory for helpful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DG SC. Performed the experiments: SC. Analyzed the data: DG SC. Wrote the paper: DG SC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000811-Cesarman1" id="pone.0000811-Cesarman1"></a>Cesarman E, Chang Y, Moore PS, Said JW, Knowles DM (1995) Kaposi's sarcoma-associated herpesvirus-like DNA sequences in AIDS-related body-cavity-based lymphomas. N Engl J Med  332: 1186–1191.  <ul class="find" data-citedArticleID="984410" data-doi="10.1056/nejm199505043321802"><li><a href="http://dx.doi.org/10.1056/nejm199505043321802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kaposi%27s+sarcoma-associated+herpesvirus-like+DNA+sequences+in+AIDS-related+body-cavity-based+lymphomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kaposi%27s+sarcoma-associated+herpesvirus-like+DNA+sequences+in+AIDS-related+body-cavity-based+lymphomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000811-Chang1" id="pone.0000811-Chang1"></a>Chang Y, Cesarman E, Pessin MS, Lee F, Culpepper J, et al.  (1994) Identification of herpesvirus-like DNA sequences in AIDS-associated Kaposi's sarcoma. Science  266: 1865–1869.  <ul class="find" data-citedArticleID="984412" data-doi="10.1126/science.7997879"><li><a href="http://dx.doi.org/10.1126/science.7997879" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+herpesvirus-like+DNA+sequences+in+AIDS-associated+Kaposi%27s+sarcoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+herpesvirus-like+DNA+sequences+in+AIDS-associated+Kaposi%27s+sarcoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000811-Soulier1" id="pone.0000811-Soulier1"></a>Soulier J, Grollet L, Oksenhendler E, Cacoub P, Cazals-Hatem D, et al.  (1995) Kaposi's sarcoma-associated herpesvirus-like DNA sequences in multicentric Castleman's disease. Blood  86: 1276–1280.  <ul class="find" data-citedArticleID="984460"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Kaposi%27s+sarcoma-associated+herpesvirus-like+DNA+sequences+in+multicentric+Castleman%27s+disease.&amp;auth=&amp;atitle=Kaposi%27s+sarcoma-associated+herpesvirus-like+DNA+sequences+in+multicentric+Castleman%27s+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kaposi%27s+sarcoma-associated+herpesvirus-like+DNA+sequences+in+multicentric+Castleman%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kaposi%27s+sarcoma-associated+herpesvirus-like+DNA+sequences+in+multicentric+Castleman%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000811-Moore1" id="pone.0000811-Moore1"></a>Moore PS, Chang Y (1995) Detection of herpesvirus-like DNA sequences in Kaposi's sarcoma in patients with and without HIV infection. N Engl J Med  332: 1181–1185.  <ul class="find" data-citedArticleID="984444" data-doi="10.1056/nejm199505043321801"><li><a href="http://dx.doi.org/10.1056/nejm199505043321801" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+herpesvirus-like+DNA+sequences+in+Kaposi%27s+sarcoma+in+patients+with+and+without+HIV+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+herpesvirus-like+DNA+sequences+in+Kaposi%27s+sarcoma+in+patients+with+and+without+HIV+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000811-Schulz1" id="pone.0000811-Schulz1"></a>Schulz TF (1999) Epidemiology of Kaposi's sarcoma-associated herpesvirus/human herpesvirus 8. Adv Cancer Res  76: 121–160.  <ul class="find" data-citedArticleID="984458" data-doi="10.1016/s0065-230x(08)60775-7"><li><a href="http://dx.doi.org/10.1016/s0065-230x(08)60775-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epidemiology+of+Kaposi%27s+sarcoma-associated+herpesvirus%2Fhuman+herpesvirus+8." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epidemiology+of+Kaposi%27s+sarcoma-associated+herpesvirus%2Fhuman+herpesvirus+8.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000811-Sarid1" id="pone.0000811-Sarid1"></a>Sarid R, Wiezorek JS, Moore PS, Chang Y (1999) Characterization and cell cycle regulation of the major Kaposi's sarcoma-associated herpesvirus (human herpesvirus 8) latent genes and their promoter. J Virol  73: 1438–1446.  <ul class="find" data-citedArticleID="984456"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+and+cell+cycle+regulation+of+the+major+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+latent+genes+and+their+promoter.&amp;auth=&amp;atitle=Characterization+and+cell+cycle+regulation+of+the+major+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+latent+genes+and+their+promoter." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+and+cell+cycle+regulation+of+the+major+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+latent+genes+and+their+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+and+cell+cycle+regulation+of+the+major+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+latent+genes+and+their+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000811-Zhong1" id="pone.0000811-Zhong1"></a>Zhong W, Wang H, Herndier B, Ganem D (1996) Restricted expression of Kaposi sarcoma-associated herpesvirus (human herpesvirus 8) genes in Kaposi sarcoma. Proc Natl Acad Sci U S A  93: 6641–6646.  <ul class="find" data-citedArticleID="984478" data-doi="10.1073/pnas.93.13.6641"><li><a href="http://dx.doi.org/10.1073/pnas.93.13.6641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Restricted+expression+of+Kaposi+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+genes+in+Kaposi+sarcoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Restricted+expression+of+Kaposi+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+genes+in+Kaposi+sarcoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000811-Dittmer1" id="pone.0000811-Dittmer1"></a>Dittmer D, Lagunoff M, Renne R, Staskus K, Haase A, et al.  (1998) A cluster of latently expressed genes in Kaposi's sarcoma-associated herpesvirus. J Virol  72: 8309–8315.  <ul class="find" data-citedArticleID="984414"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+cluster+of+latently+expressed+genes+in+Kaposi%27s+sarcoma-associated+herpesvirus.&amp;auth=&amp;atitle=A+cluster+of+latently+expressed+genes+in+Kaposi%27s+sarcoma-associated+herpesvirus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cluster+of+latently+expressed+genes+in+Kaposi%27s+sarcoma-associated+herpesvirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cluster+of+latently+expressed+genes+in+Kaposi%27s+sarcoma-associated+herpesvirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000811-Renne1" id="pone.0000811-Renne1"></a>Renne R, Lagunoff M, Zhong W, Ganem D (1996) The size and conformation of Kaposi's sarcoma-associated herpesvirus (human herpesvirus 8) DNA in infected cells and virions. J Virol  70: 8151–8154.  <ul class="find" data-citedArticleID="984450"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+size+and+conformation+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+DNA+in+infected+cells+and+virions.&amp;auth=&amp;atitle=The+size+and+conformation+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+DNA+in+infected+cells+and+virions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+size+and+conformation+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+DNA+in+infected+cells+and+virions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+size+and+conformation+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+DNA+in+infected+cells+and+virions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000811-Bechtel1" id="pone.0000811-Bechtel1"></a>Bechtel JT, Liang Y, Hvidding J, Ganem D (2003) Host range of Kaposi's sarcoma-associated herpesvirus in cultured cells. J Virol  77: 6474–6481.  <ul class="find" data-citedArticleID="984406" data-doi="10.1128/jvi.77.11.6474-6481.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.11.6474-6481.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Host+range+of+Kaposi%27s+sarcoma-associated+herpesvirus+in+cultured+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Host+range+of+Kaposi%27s+sarcoma-associated+herpesvirus+in+cultured+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000811-Sun1" id="pone.0000811-Sun1"></a>Sun R, Lin SF, Gradoville L, Yuan Y, Zhu F, et al.  (1998) A viral gene that activates lytic cycle expression of Kaposi's sarcoma-associated herpesvirus. Proc Natl Acad Sci U S A  95: 10866–10871.  <ul class="find" data-citedArticleID="984466" data-doi="10.1073/pnas.95.18.10866"><li><a href="http://dx.doi.org/10.1073/pnas.95.18.10866" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+viral+gene+that+activates+lytic+cycle+expression+of+Kaposi%27s+sarcoma-associated+herpesvirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+viral+gene+that+activates+lytic+cycle+expression+of+Kaposi%27s+sarcoma-associated+herpesvirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000811-Renne2" id="pone.0000811-Renne2"></a>Renne R, Zhong W, Herndier B, McGrath M, Abbey N, et al.  (1996) Lytic growth of Kaposi's sarcoma-associated herpesvirus (human herpesvirus 8) in culture. Nat Med  2: 342–346.  <ul class="find" data-citedArticleID="984452" data-doi="10.1038/nm0396-342"><li><a href="http://dx.doi.org/10.1038/nm0396-342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lytic+growth+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+in+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lytic+growth+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28human+herpesvirus+8%29+in+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000811-Ganem1" id="pone.0000811-Ganem1"></a>Ganem D (2007)  Kaposi's sarcoma-associated herpesvirus. In: Knipe D, Howley P, editors. Fields' Virology. 5th ed. Philadelphia: Lippincott, Williams &amp; Wilkins. pp. 2847–2888.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000811-Lukac1" id="pone.0000811-Lukac1"></a>Lukac DM, Renne R, Kirshner JR, Ganem D (1998) Reactivation of Kaposi's sarcoma-associated herpesvirus infection from latency by expression of the ORF 50 transactivator, a homolog of the EBV R protein. Virology  252: 304–312.  <ul class="find" data-citedArticleID="984436" data-doi="10.1006/viro.1998.9486"><li><a href="http://dx.doi.org/10.1006/viro.1998.9486" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reactivation+of+Kaposi%27s+sarcoma-associated+herpesvirus+infection+from+latency+by+expression+of+the+ORF+50+transactivator%2C+a+homolog+of+the+EBV+R+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reactivation+of+Kaposi%27s+sarcoma-associated+herpesvirus+infection+from+latency+by+expression+of+the+ORF+50+transactivator%2C+a+homolog+of+the+EBV+R+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000811-Miller1" id="pone.0000811-Miller1"></a>Miller G, Heston L, Grogan E, Gradoville L, Rigsby M, et al.  (1997) Selective switch between latency and lytic replication of Kaposi's sarcoma herpesvirus and Epstein-Barr virus in dually infected body cavity lymphoma cells. J Virol  71: 314–324.  <ul class="find" data-citedArticleID="984442"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selective+switch+between+latency+and+lytic+replication+of+Kaposi%27s+sarcoma+herpesvirus+and+Epstein-Barr+virus+in+dually+infected+body+cavity+lymphoma+cells.&amp;auth=&amp;atitle=Selective+switch+between+latency+and+lytic+replication+of+Kaposi%27s+sarcoma+herpesvirus+and+Epstein-Barr+virus+in+dually+infected+body+cavity+lymphoma+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+switch+between+latency+and+lytic+replication+of+Kaposi%27s+sarcoma+herpesvirus+and+Epstein-Barr+virus+in+dually+infected+body+cavity+lymphoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+switch+between+latency+and+lytic+replication+of+Kaposi%27s+sarcoma+herpesvirus+and+Epstein-Barr+virus+in+dually+infected+body+cavity+lymphoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000811-Liang1" id="pone.0000811-Liang1"></a>Liang Y, Chang J, Lynch SJ, Lukac DM, Ganem D (2002) The lytic switch protein of KSHV activates gene expression via functional interaction with RBP-Jkappa (CSL), the target of the Notch signaling pathway. Genes Dev  16: 1977–1989.  <ul class="find" data-citedArticleID="984432" data-doi="10.1101/gad.996502"><li><a href="http://dx.doi.org/10.1101/gad.996502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+lytic+switch+protein+of+KSHV+activates+gene+expression+via+functional+interaction+with+RBP-Jkappa+%28CSL%29%2C+the+target+of+the+Notch+signaling+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+lytic+switch+protein+of+KSHV+activates+gene+expression+via+functional+interaction+with+RBP-Jkappa+%28CSL%29%2C+the+target+of+the+Notch+signaling+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000811-Wang1" id="pone.0000811-Wang1"></a>Wang SE, Wu FY, Fujimuro M, Zong J, Hayward SD, et al.  (2003) Role of CCAAT/enhancer-binding protein alpha (C/EBPalpha) in activation of the Kaposi's sarcoma-associated herpesvirus (KSHV) lytic-cycle replication-associated protein (RAP) promoter in cooperation with the KSHV replication and transcription activator (RTA) and RAP. J Virol  77: 600–623.  <ul class="find" data-citedArticleID="984470" data-doi="10.1128/jvi.77.1.600-623.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.1.600-623.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+CCAAT%2Fenhancer-binding+protein+alpha+%28C%2FEBPalpha%29+in+activation+of+the+Kaposi%27s+sarcoma-associated+herpesvirus+%28KSHV%29+lytic-cycle+replication-associated+protein+%28RAP%29+promoter+in+cooperation+with+the+KSHV+replication+and+transcription+activator+%28RTA%29+and+RAP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+CCAAT%2Fenhancer-binding+protein+alpha+%28C%2FEBPalpha%29+in+activation+of+the+Kaposi%27s+sarcoma-associated+herpesvirus+%28KSHV%29+lytic-cycle+replication-associated+protein+%28RAP%29+promoter+in+cooperation+with+the+KSHV+replication+and+transcription+activator+%28RTA%29+and+RAP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000811-Wang2" id="pone.0000811-Wang2"></a>Wang SE, Wu FY, Yu Y, Hayward GS (2003) CCAAT/enhancer-binding protein-alpha is induced during the early stages of Kaposi's sarcoma-associated herpesvirus (KSHV) lytic cycle reactivation and together with the KSHV replication and transcription activator (RTA) cooperatively stimulates the viral RTA, MTA, and PAN promoters. J Virol  77: 9590–9612.  <ul class="find" data-citedArticleID="984472" data-doi="10.1128/jvi.77.17.9590-9612.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.17.9590-9612.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CCAAT%2Fenhancer-binding+protein-alpha+is+induced+during+the+early+stages+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28KSHV%29+lytic+cycle+reactivation+and+together+with+the+KSHV+replication+and+transcription+activator+%28RTA%29+cooperatively+stimulates+the+viral+RTA%2C+MTA%2C+and+PAN+promoters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CCAAT%2Fenhancer-binding+protein-alpha+is+induced+during+the+early+stages+of+Kaposi%27s+sarcoma-associated+herpesvirus+%28KSHV%29+lytic+cycle+reactivation+and+together+with+the+KSHV+replication+and+transcription+activator+%28RTA%29+cooperatively+stimulates+the+viral+RTA%2C+MTA%2C+and+PAN+promoters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000811-Gwack1" id="pone.0000811-Gwack1"></a>Gwack Y, Hwang S, Lim C, Won YS, Lee CH, et al.  (2002) Kaposi's Sarcoma-associated herpesvirus open reading frame 50 stimulates the transcriptional activity of STAT3. J Biol Chem  277: 6438–6442.  <ul class="find" data-citedArticleID="984424" data-doi="10.1074/jbc.m108289200"><li><a href="http://dx.doi.org/10.1074/jbc.m108289200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kaposi%27s+Sarcoma-associated+herpesvirus+open+reading+frame+50+stimulates+the+transcriptional+activity+of+STAT3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kaposi%27s+Sarcoma-associated+herpesvirus+open+reading+frame+50+stimulates+the+transcriptional+activity+of+STAT3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000811-Kirshner1" id="pone.0000811-Kirshner1"></a>Kirshner JR, Lukac DM, Chang J, Ganem D (2000) Kaposi's sarcoma-associated herpesvirus open reading frame 57 encodes a posttranscriptional regulator with multiple distinct activities. J Virol  74: 3586–3597.  <ul class="find" data-citedArticleID="984428" data-doi="10.1128/jvi.74.8.3586-3597.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.8.3586-3597.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kaposi%27s+sarcoma-associated+herpesvirus+open+reading+frame+57+encodes+a+posttranscriptional+regulator+with+multiple+distinct+activities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kaposi%27s+sarcoma-associated+herpesvirus+open+reading+frame+57+encodes+a+posttranscriptional+regulator+with+multiple+distinct+activities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000811-Wang3" id="pone.0000811-Wang3"></a>Wang S, Liu S, Wu MH, Geng Y, Wood C (2001) Identification of a cellular protein that interacts and synergizes with the RTA (ORF50) protein of Kaposi's sarcoma-associated herpesvirus in transcriptional activation. J Virol  75: 11961–11973.  <ul class="find" data-citedArticleID="984474" data-doi="10.1128/jvi.75.24.11961-11973.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.24.11961-11973.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+cellular+protein+that+interacts+and+synergizes+with+the+RTA+%28ORF50%29+protein+of+Kaposi%27s+sarcoma-associated+herpesvirus+in+transcriptional+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+cellular+protein+that+interacts+and+synergizes+with+the+RTA+%28ORF50%29+protein+of+Kaposi%27s+sarcoma-associated+herpesvirus+in+transcriptional+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000811-Martin1" id="pone.0000811-Martin1"></a>Martin DF, Kuppermann BD, Wolitz RA, Palestine AG, Li H, et al.  (1999) Oral ganciclovir for patients with cytomegalovirus retinitis treated with a ganciclovir implant. Roche Ganciclovir Study Group. N Engl J Med  340: 1063–1070.  <ul class="find" data-citedArticleID="984438" data-doi="10.1056/nejm199904083401402"><li><a href="http://dx.doi.org/10.1056/nejm199904083401402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oral+ganciclovir+for+patients+with+cytomegalovirus+retinitis+treated+with+a+ganciclovir+implant.+Roche+Ganciclovir+Study+Group." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oral+ganciclovir+for+patients+with+cytomegalovirus+retinitis+treated+with+a+ganciclovir+implant.+Roche+Ganciclovir+Study+Group.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000811-Nicholas1" id="pone.0000811-Nicholas1"></a>Nicholas J (2005) Human gammaherpesvirus cytokines and chemokine receptors. J Interferon Cytokine Res  25: 373–383.  <ul class="find" data-citedArticleID="984446" data-doi="10.1089/jir.2005.25.373"><li><a href="http://dx.doi.org/10.1089/jir.2005.25.373" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+gammaherpesvirus+cytokines+and+chemokine+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+gammaherpesvirus+cytokines+and+chemokine+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000811-Arvanitakis1" id="pone.0000811-Arvanitakis1"></a>Arvanitakis L, Geras-Raaka E, Varma A, Gershengorn MC, Cesarman E (1997) Human herpesvirus KSHV encodes a constitutively active G-protein-coupled receptor linked to cell proliferation. Nature  385: 347–350.  <ul class="find" data-citedArticleID="984396" data-doi="10.1038/385347a0"><li><a href="http://dx.doi.org/10.1038/385347a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+herpesvirus+KSHV+encodes+a+constitutively+active+G-protein-coupled+receptor+linked+to+cell+proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+herpesvirus+KSHV+encodes+a+constitutively+active+G-protein-coupled+receptor+linked+to+cell+proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000811-Bais1" id="pone.0000811-Bais1"></a>Bais C, Santomasso B, Coso O, Arvanitakis L, Raaka EG, et al.  (1998) G-protein-coupled receptor of Kaposi's sarcoma-associated herpesvirus is a viral oncogene and angiogenesis activator. Nature  391: 86–89.  <ul class="find" data-citedArticleID="984400" data-doi="10.1097/00042560-199804010-00079"><li><a href="http://dx.doi.org/10.1097/00042560-199804010-00079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G-protein-coupled+receptor+of+Kaposi%27s+sarcoma-associated+herpesvirus+is+a+viral+oncogene+and+angiogenesis+activator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G-protein-coupled+receptor+of+Kaposi%27s+sarcoma-associated+herpesvirus+is+a+viral+oncogene+and+angiogenesis+activator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000811-Yang1" id="pone.0000811-Yang1"></a>Yang TY, Chen SC, Leach MW, Manfra D, Homey B, et al.  (2000) Transgenic expression of the chemokine receptor encoded by human herpesvirus 8 induces an angioproliferative disease resembling Kaposi's sarcoma. J Exp Med  191: 445–454.  <ul class="find" data-citedArticleID="984476" data-doi="10.1084/jem.191.3.445"><li><a href="http://dx.doi.org/10.1084/jem.191.3.445" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transgenic+expression+of+the+chemokine+receptor+encoded+by+human+herpesvirus+8+induces+an+angioproliferative+disease+resembling+Kaposi%27s+sarcoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transgenic+expression+of+the+chemokine+receptor+encoded+by+human+herpesvirus+8+induces+an+angioproliferative+disease+resembling+Kaposi%27s+sarcoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000811-Glaunsinger1" id="pone.0000811-Glaunsinger1"></a>Glaunsinger B, Ganem D (2004) Lytic KSHV infection inhibits host gene expression by accelerating global mRNA turnover. Mol Cell  13: 713–723.  <ul class="find" data-citedArticleID="984420" data-doi="10.1016/s1097-2765(04)00091-7"><li><a href="http://dx.doi.org/10.1016/s1097-2765(04)00091-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lytic+KSHV+infection+inhibits+host+gene+expression+by+accelerating+global+mRNA+turnover." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lytic+KSHV+infection+inhibits+host+gene+expression+by+accelerating+global+mRNA+turnover.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000811-Glaunsinger2" id="pone.0000811-Glaunsinger2"></a>Glaunsinger B, Ganem D (2004) Highly selective escape from KSHV-mediated host mRNA shutoff and its implications for viral pathogenesis. J Exp Med  200: 391–398.  <ul class="find" data-citedArticleID="984422" data-doi="10.1084/jem.20031881"><li><a href="http://dx.doi.org/10.1084/jem.20031881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+selective+escape+from+KSHV-mediated+host+mRNA+shutoff+and+its+implications+for+viral+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+selective+escape+from+KSHV-mediated+host+mRNA+shutoff+and+its+implications+for+viral+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000811-Venetsanakos1" id="pone.0000811-Venetsanakos1"></a>Venetsanakos E, Mirza A, Fanton C, Romanov SR, Tlsty T, et al.  (2002) Induction of tubulogenesis in telomerase-immortalized human microvascular endothelial cells by glioblastoma cells. Exp Cell Res  273: 21–33.  <ul class="find" data-citedArticleID="984468" data-doi="10.1006/excr.2001.5424"><li><a href="http://dx.doi.org/10.1006/excr.2001.5424" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+tubulogenesis+in+telomerase-immortalized+human+microvascular+endothelial+cells+by+glioblastoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+tubulogenesis+in+telomerase-immortalized+human+microvascular+endothelial+cells+by+glioblastoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000811-deHoon1" id="pone.0000811-deHoon1"></a>de Hoon MJ, Imoto S, Nolan J, Miyano S (2004) Open source clustering software. Bioinformatics  20: 1453–1454.  <ul class="find" data-citedArticleID="984482" data-doi="10.1093/bioinformatics/bth078"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Open+source+clustering+software." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Open+source+clustering+software.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000811-Saldanha1" id="pone.0000811-Saldanha1"></a>Saldanha AJ (2004) Java Treeview–extensible visualization of microarray data. Bioinformatics  20: 3246–3248.  <ul class="find" data-citedArticleID="984454" data-doi="10.1093/bioinformatics/bth349"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth349" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Java+Treeview%26%23x2013%3Bextensible+visualization+of+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Java+Treeview%26%23x2013%3Bextensible+visualization+of+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000811-Bakheet1" id="pone.0000811-Bakheet1"></a>Bakheet T, Frevel M, Williams BR, Greer W, Khabar KS (2001) ARED: human AU-rich element-containing mRNA database reveals an unexpectedly diverse functional repertoire of encoded proteins. Nucleic Acids Res  29: 246–254.  <ul class="find" data-citedArticleID="984402" data-doi="10.1093/nar/29.1.246"><li><a href="http://dx.doi.org/10.1093/nar/29.1.246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ARED%3A+human+AU-rich+element-containing+mRNA+database+reveals+an+unexpectedly+diverse+functional+repertoire+of+encoded+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ARED%3A+human+AU-rich+element-containing+mRNA+database+reveals+an+unexpectedly+diverse+functional+repertoire+of+encoded+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000811-Boshoff1" id="pone.0000811-Boshoff1"></a>Boshoff C, Schulz TF, Kennedy MM, Graham AK, Fisher C, et al.  (1995) Kaposi's sarcoma-associated herpesvirus infects endothelial and spindle cells. Nat Med  1: 1274–1278.  <ul class="find" data-citedArticleID="984408" data-doi="10.1038/nm1295-1274"><li><a href="http://dx.doi.org/10.1038/nm1295-1274" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kaposi%27s+sarcoma-associated+herpesvirus+infects+endothelial+and+spindle+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kaposi%27s+sarcoma-associated+herpesvirus+infects+endothelial+and+spindle+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000811-Staskus1" id="pone.0000811-Staskus1"></a>Staskus KA, Zhong W, Gebhard K, Herndier B, Wang H, et al.  (1997) Kaposi's sarcoma-associated herpesvirus gene expression in endothelial (spindle) tumor cells. J Virol  71: 715–719.  <ul class="find" data-citedArticleID="984462"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Kaposi%27s+sarcoma-associated+herpesvirus+gene+expression+in+endothelial+%28spindle%29+tumor+cells.&amp;auth=&amp;atitle=Kaposi%27s+sarcoma-associated+herpesvirus+gene+expression+in+endothelial+%28spindle%29+tumor+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kaposi%27s+sarcoma-associated+herpesvirus+gene+expression+in+endothelial+%28spindle%29+tumor+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kaposi%27s+sarcoma-associated+herpesvirus+gene+expression+in+endothelial+%28spindle%29+tumor+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000811-Sturzl1" id="pone.0000811-Sturzl1"></a>Sturzl M, Blasig C, Schreier A, Neipel F, Hohenadl C, et al.  (1997) Expression of HHV-8 latency-associated T0.7 RNA in spindle cells and endothelial cells of AIDS-associated, classical and African Kaposi's sarcoma. Int J Cancer  72: 68–71.  <ul class="find" data-citedArticleID="984464" data-doi="10.1002/(sici)1097-0215(19970703)72:1<68::aid-ijc10&gt;3.3.co;2-x"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0215(19970703)72:1<68::aid-ijc10&gt;3.3.co;2-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+HHV-8+latency-associated+T0.7+RNA+in+spindle+cells+and+endothelial+cells+of+AIDS-associated%2C+classical+and+African+Kaposi%27s+sarcoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+HHV-8+latency-associated+T0.7+RNA+in+spindle+cells+and+endothelial+cells+of+AIDS-associated%2C+classical+and+African+Kaposi%27s+sarcoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000811-vandePeppel1" id="pone.0000811-vandePeppel1"></a>van de Peppel J, Kemmeren P, van Bakel H, Radonjic M, van Leenen D, et al.  (2003) Monitoring global messenger RNA changes in externally controlled microarray experiments. EMBO Rep  4: 387–393.  <ul class="find" data-citedArticleID="984484" data-doi="10.1038/sj.embor.embor798"><li><a href="http://dx.doi.org/10.1038/sj.embor.embor798" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monitoring+global+messenger+RNA+changes+in+externally+controlled+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monitoring+global+messenger+RNA+changes+in+externally+controlled+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000811-Quackenbush1" id="pone.0000811-Quackenbush1"></a>Quackenbush J (2002) Microarray data normalization and transformation. Nat Genet  32 Suppl: 496–501.  <ul class="find" data-citedArticleID="984448" data-doi="10.1038/ng1032"><li><a href="http://dx.doi.org/10.1038/ng1032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+data+normalization+and+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+data+normalization+and+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000811-Li1" id="pone.0000811-Li1"></a>Li M, Damania B, Alvarez X, Ogryzko V, Ozato K, et al.  (2000) Inhibition of p300 histone acetyltransferase by viral interferon regulatory factor. Mol Cell Biol  20: 8254–8263.  <ul class="find" data-citedArticleID="984430" data-doi="10.1128/mcb.20.21.8254-8263.2000"><li><a href="http://dx.doi.org/10.1128/mcb.20.21.8254-8263.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+p300+histone+acetyltransferase+by+viral+interferon+regulatory+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+p300+histone+acetyltransferase+by+viral+interferon+regulatory+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000811-Lin1" id="pone.0000811-Lin1"></a>Lin R, Genin P, Mamane Y, Sgarbanti M, Battistini A, et al.  (2001) HHV-8 encoded vIRF-1 represses the interferon antiviral response by blocking IRF-3 recruitment of the CBP/p300 coactivators. Oncogene  20: 800–811.  <ul class="find" data-citedArticleID="984434" data-doi="10.1038/sj.onc.1204163"><li><a href="http://dx.doi.org/10.1038/sj.onc.1204163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HHV-8+encoded+vIRF-1+represses+the+interferon+antiviral+response+by+blocking+IRF-3+recruitment+of+the+CBP%2Fp300+coactivators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HHV-8+encoded+vIRF-1+represses+the+interferon+antiviral+response+by+blocking+IRF-3+recruitment+of+the+CBP%2Fp300+coactivators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000811-Fuld1" id="pone.0000811-Fuld1"></a>Fuld S, Cunningham C, Klucher K, Davison AJ, Blackbourn DJ (2006) Inhibition of interferon signaling by the Kaposi's sarcoma-associated herpesvirus full-length viral interferon regulatory factor 2 protein. J Virol  80: 3092–3097.  <ul class="find" data-citedArticleID="984416" data-doi="10.1128/jvi.80.6.3092-3097.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.6.3092-3097.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+interferon+signaling+by+the+Kaposi%27s+sarcoma-associated+herpesvirus+full-length+viral+interferon+regulatory+factor+2+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+interferon+signaling+by+the+Kaposi%27s+sarcoma-associated+herpesvirus+full-length+viral+interferon+regulatory+factor+2+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000811-Zhu1" id="pone.0000811-Zhu1"></a>Zhu FX, King SM, Smith EJ, Levy DE, Yuan Y (2002) A Kaposi's sarcoma-associated herpesviral protein inhibits virus-mediated induction of type I interferon by blocking IRF-7 phosphorylation and nuclear accumulation. Proc Natl Acad Sci U S A  99: 5573–5578.  <ul class="find" data-citedArticleID="984480" data-doi="10.1073/pnas.082420599"><li><a href="http://dx.doi.org/10.1073/pnas.082420599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Kaposi%27s+sarcoma-associated+herpesviral+protein+inhibits+virus-mediated+induction+of+type+I+interferon+by+blocking+IRF-7+phosphorylation+and+nuclear+accumulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Kaposi%27s+sarcoma-associated+herpesviral+protein+inhibits+virus-mediated+induction+of+type+I+interferon+by+blocking+IRF-7+phosphorylation+and+nuclear+accumulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000811-Joo1" id="pone.0000811-Joo1"></a>Joo CH, Shin YC, Gack M, Wu L, Levy D, et al.  (2007) Inhibition of interferon regulatory factor 7 (IRF7)-mediated interferon signal transduction by the Kaposi's sarcoma-associated herpesvirus viral IRF homolog vIRF3. J Virol  81: 8282–8292.  <ul class="find" data-citedArticleID="984426" data-doi="10.1128/jvi.00235-07"><li><a href="http://dx.doi.org/10.1128/jvi.00235-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+interferon+regulatory+factor+7+%28IRF7%29-mediated+interferon+signal+transduction+by+the+Kaposi%27s+sarcoma-associated+herpesvirus+viral+IRF+homolog+vIRF3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+interferon+regulatory+factor+7+%28IRF7%29-mediated+interferon+signal+transduction+by+the+Kaposi%27s+sarcoma-associated+herpesvirus+viral+IRF+homolog+vIRF3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000811-Alexander1" id="pone.0000811-Alexander1"></a>Alexander WS, Hilton DJ (2004) The role of suppressors of cytokine signaling (SOCS) proteins in regulation of the immune response. Annu Rev Immunol  22: 503–529.  <ul class="find" data-citedArticleID="984394"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+role+of+suppressors+of+cytokine+signaling+%28SOCS%29+proteins+in+regulation+of+the+immune+response.&amp;auth=&amp;atitle=The+role+of+suppressors+of+cytokine+signaling+%28SOCS%29+proteins+in+regulation+of+the+immune+response." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+suppressors+of+cytokine+signaling+%28SOCS%29+proteins+in+regulation+of+the+immune+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+suppressors+of+cytokine+signaling+%28SOCS%29+proteins+in+regulation+of+the+immune+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000811-Barreau1" id="pone.0000811-Barreau1"></a>Barreau C, Paillard L, Osborne HB (2005) AU-rich elements and associated factors: are there unifying principles? Nucleic Acids Res  33: 7138–7150.  <ul class="find" data-citedArticleID="984404" data-doi="10.1093/nar/gki1012"><li><a href="http://dx.doi.org/10.1093/nar/gki1012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AU-rich+elements+and+associated+factors%3A+are+there+unifying+principles%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AU-rich+elements+and+associated+factors%3A+are+there+unifying+principles%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000811-McCormick1" id="pone.0000811-McCormick1"></a>McCormick C, Ganem D (2005) The kaposin B protein of KSHV activates the p38/MK2 pathway and stabilizes cytokine mRNAs. Science  307: 739–741.  <ul class="find" data-citedArticleID="984440" data-doi="10.1126/science.1105779"><li><a href="http://dx.doi.org/10.1126/science.1105779" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+kaposin+B+protein+of+KSHV+activates+the+p38%2FMK2+pathway+and+stabilizes+cytokine+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+kaposin+B+protein+of+KSHV+activates+the+p38%2FMK2+pathway+and+stabilizes+cytokine+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000811-Bailey1" id="pone.0000811-Bailey1"></a>Bailey TL, Elkan C (1994) Fitting a mixture model by expectation maximization to discover motifs in biopolymers. Proc Int Conf Intell Syst Mol Biol  2: 28–36.  <ul class="find" data-citedArticleID="984398"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers.&amp;auth=&amp;atitle=Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.XML" value="89794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.PDF" value="857668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g001.PNG_L" value="125864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g001.PNG_M" value="87574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g001.PNG_S" value="6675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g001.TIF" value="546406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g001.PNG_I" value="35847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g002.PNG_L" value="205198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g002.PNG_M" value="153628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g002.PNG_S" value="8424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g002.TIF" value="707364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g002.PNG_I" value="59116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g003.PNG_L" value="44007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g003.PNG_M" value="75672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g003.PNG_S" value="5553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g003.TIF" value="246984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g003.PNG_I" value="29746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g004.PNG_L" value="210968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g004.PNG_M" value="122794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g004.PNG_S" value="6434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g004.TIF" value="607512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g004.PNG_I" value="48429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.t001.PNG_L" value="31765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.t001.PNG_M" value="40709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.t001.PNG_S" value="8943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.t001.TIF" value="145672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.t001.PNG_I" value="13104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g005.PNG_L" value="77838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g005.PNG_M" value="58617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g005.PNG_S" value="6067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g005.TIF" value="368634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g005.PNG_I" value="24781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g006.PNG_L" value="164125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g006.PNG_M" value="97427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g006.PNG_S" value="6088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g006.TIF" value="517240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g006.PNG_I" value="38473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g007.PNG_L" value="886338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g007.PNG_M" value="112055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g007.PNG_S" value="5835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g007.TIF" value="1422812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.g007.PNG_I" value="39588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.s001.EPS" value="847795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.s002.EPS" value="551083"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.s003.TDS" value="2315355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000811.s004.TDS" value="49479"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000811&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000811" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000811&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000811&volume=&issue=&title=Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses&author_name=Sanjay%20Chandriani%2C%20Don%20Ganem&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000811" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000811" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000811" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000811&amp;t=Host%20Transcript%20Accumulation%20during%20Lytic%20KSHV%20Infection%20Reveals%20Several%20Classes%20of%20Host%20Responses" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000811&title=Host%20Transcript%20Accumulation%20during%20Lytic%20KSHV%20Infection%20Reveals%20Several%20Classes%20of%20Host%20Responses&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000811&amp;title=Host%20Transcript%20Accumulation%20during%20Lytic%20KSHV%20Infection%20Reveals%20Several%20Classes%20of%20Host%20Responses" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000811" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000811&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Host Transcript Accumulation during Lytic KSHV Infection Reveals Several Classes of Host Responses';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000811';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Host%20Transcript%20Accumulation%20during%20Lytic%20KSHV%20Infection%20Reveals%20Several%20Classes%20of%20Host%20Responses http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000811" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000811" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24460"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lytic+cycle%22" title="Search for articles in the subject area:'Lytic cycle'"><div class="flagText">Lytic cycle</div></a>
              <div data-categoryid="27349" data-articleid="24460"
                   data-categoryname="Lytic cycle"
                   class="flagImage" title="Flag 'Lytic cycle' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="24460"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24460"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="24460"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="24460"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+replication%22" title="Search for articles in the subject area:'Viral replication'"><div class="flagText">Viral replication</div></a>
              <div data-categoryid="20015" data-articleid="24460"
                   data-categoryname="Viral replication"
                   class="flagImage" title="Flag 'Viral replication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Virus+effects+on+host+gene+expression%22" title="Search for articles in the subject area:'Virus effects on host gene expression'"><div class="flagText">Virus effects on host gene expression</div></a>
              <div data-categoryid="20527" data-articleid="24460"
                   data-categoryname="Virus effects on host gene expression"
                   class="flagImage" title="Flag 'Virus effects on host gene expression' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3771'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6079'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7955&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
