

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Differentiating between Near- and Non-Cognate Codons in Saccharomyces cerevisiae</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000517"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000517" />

    <meta name="citation_title" content="Differentiating between Near- and Non-Cognate Codons in Saccharomyces cerevisiae"/>
    <meta itemprop="name" content="Differentiating between Near- and Non-Cognate Codons in Saccharomyces cerevisiae"/>

      <meta name="citation_author" content="Ewan P. Plant"/>
            <meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America"/>
      <meta name="citation_author" content="Phuc Nguyen"/>
            <meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America"/>
      <meta name="citation_author" content="Jonathan R. Russ"/>
            <meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America"/>
      <meta name="citation_author" content="Yvette R. Pittman"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, Microbiology and Immunology, University of Medicine and Dentistry of New Jersey (UMDNJ) Robert Wood Johnson Medical School, Piscataway, New Jersey, United States of America"/>
      <meta name="citation_author" content="Thai Nguyen"/>
            <meta name="citation_author_institution" content="The Science and Technology Center at Eleanor Roosevelt High School, Greenbelt, Maryland, United States of America"/>
      <meta name="citation_author" content="Jack T. Quesinberry"/>
            <meta name="citation_author_institution" content="Huntington High School, Huntington, Maryland, United States of America"/>
      <meta name="citation_author" content="Terri Goss Kinzy"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, Microbiology and Immunology, University of Medicine and Dentistry of New Jersey (UMDNJ) Robert Wood Johnson Medical School, Piscataway, New Jersey, United States of America"/>
      <meta name="citation_author" content="Jonathan D. Dinman"/>
            <meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America"/>

    <meta name="citation_date" content="2007/6/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000517.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e517"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Codon-Anticodon Pairing - Wobble Hypothesis.; citation_author=FHC Crick; citation_journal_title=Journal of Molecular Biology; citation_volume=19; citation_number=1; citation_pages=548-555; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Wobble position modified nucleosides evolved to select transfer RNA codon recognition: a modified-wobble hypothesis.; citation_author=PF Agris; citation_journal_title=Biochimie; citation_volume=73; citation_number=2; citation_pages=1345-1349; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The frequency of translational misreading errors in E. coli is largely determined by tRNA competition.; citation_author=EB Kramer; citation_author=PJ Farabaugh; citation_journal_title=RNA; citation_volume=13; citation_number=3; citation_pages=87-96; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=tRNA's wobble decoding of the genome: 40 years of modification.; citation_author=PF Agris; citation_author=FA Vendeix; citation_author=WD Graham; citation_journal_title=J Mol Biol; citation_volume=366; citation_number=4; citation_pages=1-13; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structural Insights into Translational Fidelity.; citation_author=JM Ogle; citation_author=V Ramakrishnan; citation_journal_title=Annu Rev Biochem; citation_volume=74; citation_number=5; citation_pages=129-177; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The ribosome's response to codon-anticodon mismatches.; citation_author=T Daviter; citation_author=KB Gromadski; citation_author=MV Rodnina; citation_journal_title=Biochimie; citation_volume=88; citation_number=6; citation_pages=1001-1011; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Alignment/misalignment hypothesis for tRNA selection by the ribosome.; citation_author=KY Sanbonmatsu; citation_journal_title=Biochimie; citation_volume=88; citation_number=7; citation_pages=1075-1089; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis reveals two independent molecular requirements during transfer RNA selection on the ribosome.; citation_author=L Cochella; citation_author=JL Brunelle; citation_author=R Green; citation_journal_title=Nat Struct Mol Biol; citation_volume=14; citation_number=8; citation_pages=30-36; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Understanding discrimination by the ribosome: stability testing and groove measurement of codon-anticodon pairs.; citation_author=KY Sanbonmatsu; citation_author=S Joseph; citation_journal_title=J Mol Biol; citation_volume=328; citation_number=9; citation_pages=33-47; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Energy landscape of the ribosomal decoding center.; citation_author=KY Sanbonmatsu; citation_journal_title=Biochimie; citation_volume=88; citation_number=10; citation_pages=1053-1059; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Selection of tRNA by the Ribosome Requires a Transition from an Open to a Closed Form.; citation_author=JM Ogle; citation_author=FV Murphy; citation_author=MJ Tarry; citation_author=V Ramakrishnan; citation_journal_title=Cell; citation_volume=111; citation_number=11; citation_pages=721-732; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Fidelity in protein synthesis.; citation_author=L Cochella; citation_author=R Green; citation_journal_title=Curr Biol; citation_volume=15; citation_number=12; citation_pages=R536-R540; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structure of eEF3 and the mechanism of transfer RNA release from the E-site.; citation_author=CB Andersen; citation_author=T Becker; citation_author=M Blau; citation_author=M Anand; citation_author=M Halic; citation_journal_title=Nature; citation_volume=443; citation_number=13; citation_pages=663-668; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Ribosome-Bound Chaperones RAC and Ssb1/2p Are Required for Accurate Translation in Saccharomyces cerevisiae.; citation_author=M Rakwalska; citation_author=S Rospert; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=14; citation_pages=9186-9197; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Systematic analysis of bicistronic reporter assay data.; citation_author=JL Jacobs; citation_author=JD Dinman; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=15; citation_pages=e160-e170; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Codon-specific missense errors in vivo.; citation_author=F Bouadloun; citation_author=D Donner; citation_author=CG Kurland; citation_journal_title=EMBO J; citation_volume=2; citation_number=16; citation_pages=1351-1356; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Discrimination between defects in elongation fidelity and termination efficiency provides mechanistic insights into translational readthrough.; citation_author=J Salas-Marco; citation_author=DM Bedwell; citation_journal_title=J Mol Biol; citation_volume=348; citation_number=17; citation_pages=801-815; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Missense translation errors in Saccharomyces cerevisiae.; citation_author=I Stansfield; citation_author=KM Jones; citation_author=P Herbert; citation_author=A Lewendon; citation_author=WV Shaw; citation_journal_title=Journal of Molecular Biology; citation_volume=282; citation_number=18; citation_pages=13-24; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Ribosomal elongation cycle: energetic, kinetic and stereochemical aspects.; citation_author=VI Lim; citation_author=JF Curran; citation_author=MB Garber; citation_journal_title=J Mol Biol; citation_volume=351; citation_number=19; citation_pages=470-480; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mutations in elongation factor EF-1α affect the frequency of frameshifting and amino acid misincorporation in Saccharomyces cerevisiae.; citation_author=MG Sandbaken; citation_author=MR Culbertson; citation_journal_title=Genetics; citation_volume=120; citation_number=20; citation_pages=923-934; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Cloning and characterization of the elongation factor EF-1 beta homologue of Saccharomyces cerevisiae. EF-1 beta is essential for growth.; citation_author=K Hiraga; citation_author=K Suzuki; citation_author=E Tsuchiya; citation_author=T Miyakawa; citation_journal_title=FEBS Lett; citation_volume=316; citation_number=21; citation_pages=165-169; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Structural basis for nucleotide exchange and competition with tRNA in the yeast elongation factor complex eEF1A:eEF1Balpha.; citation_author=GR Andersen; citation_author=L Pedersen; citation_author=L Valente; citation_author=I Chatterjee; citation_author=TG Kinzy; citation_journal_title=Mol Cell; citation_volume=6; citation_number=22; citation_pages=1261-1266; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mutations in elongation factor 1beta, a guanine nucleotide exchange factor, enhance translational fidelity.; citation_author=A Carr-Schmid; citation_author=L Valente; citation_author=VI Loik; citation_author=T Williams; citation_author=LM Starita; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=23; citation_pages=5257-5266; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The crystal structure of the glutathione S-transferase-like domain of elongation factor 1Bgamma from Saccharomyces cerevisiae.; citation_author=MG Jeppesen; citation_author=P Ortiz; citation_author=W Shepard; citation_author=TG Kinzy; citation_author=J Nyborg; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=24; citation_pages=47190-47198; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The translation elongation factor, eEF1B,plays a role in the oxidative stress response pathway.; citation_author=O Olarewaju; citation_author=PA Ortiz; citation_author=W Chowdhury; citation_author=I Chatterjee; citation_author=TG Kinzy; citation_journal_title=RNA Biology; citation_volume=1; citation_number=25; citation_pages=12-17; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The allosteric three-site model for the ribosomal elongation cycle: features and future.; citation_author=KH Nierhaus; citation_journal_title=Biochemistry; citation_volume=29; citation_number=26; citation_pages=4997-5008; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=The elongation factor 3 unique in higher fungi and essential for protein biosynthesis is an E site factor.; citation_author=FJ Triana-Alonso; citation_author=K Chakraburtty; citation_author=KH Nierhaus; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=27; citation_pages=20473-20478; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Multiple regions of yeast ribosomal protein L1 are important for its interaction with 5 S rRNA and assembly into ribosomes.; citation_author=M Deshmukh; citation_author=J Stark; citation_author=LC Yeh; citation_author=JC Lee; citation_author=JL Woolford; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=28; citation_pages=30148-30156; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Ribosomal protein L5 helps anchor peptidyl-tRNA to the P-site in Saccharomyces cerevisiae.; citation_author=A Meskauskas; citation_author=JD Dinman; citation_journal_title=RNA; citation_volume=7; citation_number=29; citation_pages=1084-1096; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy.; citation_author=N Stoletzki; citation_author=A Eyre-Walker; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=30; citation_pages=374-381; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Error minimization explains the codon usage of highly expressed genes in Escherichia coli.; citation_author=HS Najafabadi; citation_author=J Lehmann; citation_author=M Omidi; citation_journal_title=Gene; citation_volume=387; citation_number=31; citation_pages=150-155; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=GCG Documentation.; citation_author=F Plewniak; citation_number=32; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Correlation between the abundance of yeast transfer RNAs and the occurrence of the respective codons in protein genes. Differences in synonymous codon choice patterns of yeast and Escherichia coli with reference to the abundance of isoaccepting transfer RNAs.; citation_author=T Ikemura; citation_journal_title=J Mol Biol; citation_volume=158; citation_number=33; citation_pages=573-597; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Transfer RNA gene redundancy and translational selection in Saccharomyces cerevisiae.; citation_author=R Percudani; citation_author=A Pavesi; citation_author=S Ottonello; citation_journal_title=J Mol Biol; citation_volume=268; citation_number=34; citation_pages=322-330; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=tRNA genes and retroelements in the yeast genome.; citation_author=J Hani; citation_author=H Feldmann; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=35; citation_pages=689-696; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Ribosomal frameshifting in the yeast retrotransposon Ty: tRNAs induce slippage on a 7 nucleotide minimal site.; citation_author=MF Belcourt; citation_author=PJ Farabaugh; citation_journal_title=Cell; citation_volume=62; citation_number=36; citation_pages=339-352; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Functional interactions between yeast translation elongation factors eEF1A and eEF3.; citation_author=M Anand; citation_author=K Chakraburtty; citation_author=MJ Marton; citation_author=AG Hinnebusch; citation_author=TG Kinzy; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=37; citation_pages=6985-6991; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Recognition and selection of tRNA in translation.; citation_author=MV Rodnina; citation_author=KB Gromadski; citation_author=U Kothe; citation_author=HJ Wieden; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=38; citation_pages=938-942; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A uniform response to mismatches in codon-anticodon complexes ensures ribosomal fidelity.; citation_author=KB Gromadski; citation_author=T Daviter; citation_author=MV Rodnina; citation_journal_title=Mol Cell; citation_volume=21; citation_number=39; citation_pages=369-377; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recognition of cognate transfer RNA by the 30S ribosomal subunit.; citation_author=JM Ogle; citation_author=DE Brodersen; citation_author=WM Clemons; citation_author=MJ Tarry; citation_author=AP Carter; citation_journal_title=Science; citation_volume=292; citation_number=40; citation_pages=897-902; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics.; citation_author=AP Carter; citation_author=WM Clemons; citation_author=DE Brodersen; citation_author=RJ Morgan-Warren; citation_author=BT Wimberly; citation_author=V Ramakrishnan; citation_journal_title=Nature; citation_volume=407; citation_number=41; citation_pages=340-348; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A genetic model to investigate drug-target interactions at the ribosomal decoding site.; citation_author=SN Hobbie; citation_author=C Bruell; citation_author=S Kalapala; citation_author=S Akshay; citation_author=S Schmidt; citation_journal_title=Biochimie; citation_volume=88; citation_number=42; citation_pages=1033-1043; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High efficiency transformation of Escherichia coli with plasmids.; citation_author=H Inoue; citation_author=H Nojima; citation_author=H Okayama; citation_journal_title=Gene; citation_volume=96; citation_number=43; citation_pages=23-28; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Translation elongation factor 1 functions in the yeast Saccharomyces cerevisiae.; citation_author=M Anand; citation_author=L Valente; citation_author=A Carr-Schmid; citation_author=R Munshi; citation_author=O Olarewaju; citation_journal_title=Cold Spring Harb Symp Quant Biol; citation_volume=66; citation_number=44; citation_pages=439-48; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mutations in a GTP-binding motif of eukaryotic elongation factor 1A reduce both translational fidelity and the requirement for nucleotide exchange.; citation_author=A Carr-Schmid; citation_author=N Durko; citation_author=J Cavallius; citation_author=WC Merrick; citation_author=TG Kinzy; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=45; citation_pages=30297-30302; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Translational maintenance of frame: mutants of Saccharomyces cerevisiae with altered -1 ribosomal frameshifting efficiencies.; citation_author=JD Dinman; citation_author=RB Wickner; citation_journal_title=Genetics; citation_volume=136; citation_number=46; citation_pages=75-86; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Transformation of intact yeast cells treated with alkali cations.; citation_author=H Ito; citation_author=Y Fukuda; citation_author=K Murata; citation_author=A Kimura; citation_journal_title=J Bacteriol; citation_volume=153; citation_number=47; citation_pages=163-168; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=An in vivo dual-luciferase assay system for studying translational recoding in the yeast Saccharomyces cerevisiae.; citation_author=JW Harger; citation_author=JD Dinman; citation_journal_title=RNA; citation_volume=9; citation_number=48; citation_pages=1019-1024; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Yeast vectors for the controlled expression of heterologous proteins in different genetic backgrounds.; citation_author=D Mumberg; citation_author=R Muller; citation_author=M Funk; citation_journal_title=Gene; citation_volume=156; citation_number=49; citation_pages=119-122; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of nucleotide exchange intermediates in the eEF1A- eEF1Balpha complex.; citation_author=GR Andersen; citation_author=L Valente; citation_author=L Pedersen; citation_author=TG Kinzy; citation_author=J Nyborg; citation_journal_title=Nat Struct Biol; citation_volume=8; citation_number=50; citation_pages=531-534; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the ternary complex of Phe-tRNAPhe, EF-Tu, and a GTP analog.; citation_author=P Nissen; citation_author=M Kjeldgaard; citation_author=S Thirup; citation_author=G Polekhina; citation_author=L Reshetnikova; citation_journal_title=Science; citation_volume=270; citation_number=51; citation_pages=1464-1472; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Pokeweed antiviral protein specifically inhibits Ty1 directed +1 ribosomal frameshifting and Ty1 retrotransposition in Saccharomyces cerevisiae.; citation_author=NE Tumer; citation_author=B Parikh; citation_author=P Li; citation_author=JD Dinman; citation_journal_title=J Virol; citation_volume=72; citation_number=52; citation_pages=1036-1042; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Translational misreading: Mutations in translation elongation factor 1α differentially affect programmed ribosomal frameshifting and drug sensitivity.; citation_author=JD Dinman; citation_author=TG Kinzy; citation_journal_title=RNA; citation_volume=3; citation_number=53; citation_pages=870-881; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000517" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Differentiating between Near- and Non-Cognate Codons in Saccharomyces cerevisiae"/>
    <meta name="twitter:description" content="BackgroundDecoding of mRNAs is performed by aminoacyl tRNAs (aa-tRNAs). This process is highly accurate, however, at low frequencies (10&minus;3 &ndash; 10&minus;4) the wrong aa-tRNA can be selected, leading to incorporation of aberrant amino acids. Although our understanding of what constitutes the correct or cognate aa-tRNA:mRNA interaction is well defined, a functional distinction between near-cognate or single mismatched, and unpaired or non-cognate interactions is lacking.Methodology/Principal FindingsMisreading of several synonymous codon substitutions at the catalytic site of firefly luciferase was assayed in Saccharomyces cerevisiae. Analysis of the results in the context of current kinetic and biophysical models of aa-tRNA selection suggests that the defining feature of near-cognate aa-tRNAs is their potential to form mini-helical structures with A-site codons, enabling stimulation of GTPase activity of eukaryotic Elongation Factor 1A (eEF1A). Paromomycin specifically stimulated misreading of near-cognate but not of non-cognate aa-tRNAs, providing a functional probe to distinguish between these two classes. Deletion of the accessory elongation factor eEF1B&gamma; promoted increased misreading of near-cognate, but hyperaccurate reading of non-cognate codons, suggesting that this factor also has a role in tRNA discrimination. A mutant of eEF1B&alpha;, the nucleotide exchange factor for eEF1A, promoted a general increase in fidelity, suggesting that the decreased rates of elongation may provide more time for discrimination between aa-tRNAs. A mutant form of ribosomal protein L5 promoted hyperaccurate decoding of both types of codons, even though it is topologically distant from the decoding center.Conclusions/SignficanceIt is important to distinguish between near-cognate and non-cognate mRNA:tRNA interactions, because such a definition may be important for informing therapeutic strategies for suppressing these two different categories of mutations underlying many human diseases. This study suggests that the defining feature of near-cognate aa-tRNAs is their potential to form mini-helical structures with A-site codons in the ribosomal decoding center. An aminoglycoside and a ribosomal factor can be used to distinguish between near-cognate and non-cognate interactions."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000517.g005"/>

  <meta property="og:title" content="Differentiating between Near- and Non-Cognate Codons in Saccharomyces cerevisiae" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000517" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7471'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5843'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4441&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000517">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000517" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000517&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Differentiating between Near- and Non-Cognate Codons in <i>Saccharomyces cerevisiae</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ewan P. Plant, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America
                </p>

                <p>
                    Current address: Laboratory of Hepatitis and Related Emerging Agents, Division of Emerging and Transfusion-Transmitted Diseases, Office of Blood Research and Review, Center for Biologics Evaluation and Research (CBER), FDA, Bethesda, Maryland, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Phuc Nguyen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America
                </p>

                <p>
                    Current address: National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonathan R. Russ, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yvette R. Pittman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics, Microbiology and Immunology, University of Medicine and Dentistry of New Jersey (UMDNJ) Robert Wood Johnson Medical School, Piscataway, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thai Nguyen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Science and Technology Center at Eleanor Roosevelt High School, Greenbelt, Maryland, United States of America
                </p>

                <p>
                    Current address: College of Agriculture, University of Maryland, College Park, Maryland, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jack T. Quesinberry, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Huntington High School, Huntington, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Terri Goss Kinzy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics, Microbiology and Immunology, University of Medicine and Dentistry of New Jersey (UMDNJ) Robert Wood Johnson Medical School, Piscataway, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonathan D. Dinman
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:dinman@umd.edu">dinman@umd.edu</a></p>

                <p>Affiliation:
                  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 13, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000517</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000517" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000517" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000517" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000517" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000517-g001" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-t001" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-g002" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-g003" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-t002" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-t003" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-g004" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-g005" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000517-t004" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000517">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000517" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Decoding of mRNAs is performed by aminoacyl tRNAs (aa-tRNAs). This process is highly accurate, however, at low frequencies (10<sup>−3</sup> – 10<sup>−4</sup>) the wrong aa-tRNA can be selected, leading to incorporation of aberrant amino acids. Although our understanding of what constitutes the correct or cognate aa-tRNA:mRNA interaction is well defined, a functional distinction between near-cognate or single mismatched, and unpaired or non-cognate interactions is lacking.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Misreading of several synonymous codon substitutions at the catalytic site of firefly luciferase was assayed in <em>Saccharomyces cerevisiae</em>. Analysis of the results in the context of current kinetic and biophysical models of aa-tRNA selection suggests that the defining feature of near-cognate aa-tRNAs is their potential to form mini-helical structures with A-site codons, enabling stimulation of GTPase activity of eukaryotic Elongation Factor 1A (eEF1A). Paromomycin specifically stimulated misreading of near-cognate but not of non-cognate aa-tRNAs, providing a functional probe to distinguish between these two classes. Deletion of the accessory elongation factor eEF1Bγ promoted increased misreading of near-cognate, but hyperaccurate reading of non-cognate codons, suggesting that this factor also has a role in tRNA discrimination. A mutant of eEF1Bα, the nucleotide exchange factor for eEF1A, promoted a general increase in fidelity, suggesting that the decreased rates of elongation may provide more time for discrimination between aa-tRNAs. A mutant form of ribosomal protein L5 promoted hyperaccurate decoding of both types of codons, even though it is topologically distant from the decoding center.</p>


<h3>Conclusions/Signficance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>It is important to distinguish between near-cognate and non-cognate mRNA:tRNA interactions, because such a definition may be important for informing therapeutic strategies for suppressing these two different categories of mutations underlying many human diseases. This study suggests that the defining feature of near-cognate aa-tRNAs is their potential to form mini-helical structures with A-site codons in the ribosomal decoding center. An aminoglycoside and a ribosomal factor can be used to distinguish between near-cognate and non-cognate interactions.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Plant EP, Nguyen P, Russ JR, Pittman YR, Nguyen T, et al.  (2007) Differentiating between Near- and Non-Cognate Codons in <em>Saccharomyces cerevisiae</em>. PLoS ONE 2(6):
          e517.
            doi:10.1371/journal.pone.0000517</p><p><strong>Academic Editor: </strong>Alfred Lewin, University of Florida, United States of America</p><p><strong>Received:</strong> April 10, 2007; <strong>Accepted:</strong> May 18, 2007; <strong>Published:</strong> June 13, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This work was supported by grants from the National Institutes of Health to JDD (GM058859 and AI064307), and to TGK (GM057483).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Accurate transmission of biological information is a central requirement at all levels of life. In cells, one aspect of this process is the faithful translation of the genetic code from DNA into protein. The intermediaries in the last stage of this process include mRNA, tRNAs, ribosomes and many <em>trans</em>-acting factors. The protein coding information of an mRNA is formatted as codons. The anticodon loops of aminoacyl tRNAs (aa-tRNAs) form base-pairing interactions with the codons. This enables ribosomes to add amino acids sequentially to the nascent protein. aa-tRNAs that can participate in standard Watson-Crick interactions with the first two bases in a codon and can form either canonical or non-Watson-Crick pairs at the third or “wobble” position are designated cognate-tRNAs <a href="#pone.0000517-Crick1">[1]</a>, <a href="#pone.0000517-Agris1">[2]</a>. In contrast, tRNAs that do not meet these requirements are commonly referred to as near- and non-cognate tRNAs. Utilization of near- and non-cognate tRNAs is called misreading or a missense error. Misreading occurs with low frequencies of 10<sup>−3</sup> and 10<sup>−4</sup> per codon (<a href="#pone.0000517-Kramer1">[3]</a> and references within).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The 64 codons encode 20 different amino acids and three termination signals. In cases where one amino acid is represented by multiple codons, some tRNAs can decode more than one codon. This redundancy is facilitated by tRNA modifications and by wobble base-pairing between the anticodon and the codon (reviewed in <a href="#pone.0000517-Agris2">[4]</a>). Our understanding of how the ribosome achieves such a high degree of specificity has been facilitated by both kinetic and structural analyses in bacteria (reviewed in <a href="#pone.0000517-Ogle1">[5]</a>–<a href="#pone.0000517-Sanbonmatsu1">[7]</a>). <em>In vitro</em> kinetic analyses using ribosomes, tRNAs, and the bacterial aa-tRNA binding factor EF-Tu have broken down the process of aa-tRNA selection into a series of discrete steps (reviewed in <a href="#pone.0000517-Daviter1">[6]</a> ). These studies have identified two stages (k<sub>−2</sub> and k<sub>7</sub>) in this process that favor rejection of aa-tRNAs whose anticodon loops cannot base-pair with codons. A mutational analysis demonstrated that two independent mechanisms corresponding to these two steps are required for utilization of cognate aa-tRNAs <a href="#pone.0000517-Cochella1">[8]</a>. Structural, biophysical, and computational analyses also show a mechanism for positive selection of cognate aa-tRNAs<a href="#pone.0000517-Sanbonmatsu1">[7]</a>–<a href="#pone.0000517-Sanbonmatsu3">[10]</a>} that emphasizes the geometry of base pairing at the ribosomal decoding center <a href="#pone.0000517-Ogle2">[11]</a>. Formation of an appropriately configured mini-helix in the decoding center generates an RNA minor-groove, enabling interaction with three critical bases of the small subunit rRNA. Formation of this mini-helix stimulates A1492 and A1493 of the small subunit rRNA to flip out into the minor groove forming a complex arrangement of hydrogen bonds with the tRNA/mRNA backbones in concert with G530 (<a href="#pone-0000517-g001">Figure 1A</a>). This in turn stimulates a conformational change in the aa-tRNA that transduces the information from the decoding center to activate the GTPase activity of EF-Tu (reviewed in <a href="#pone.0000517-Cochella2">[12]</a>). The energy barrier for flipping out of A1492 and A1493 is sufficiently small for correct binding of aa-tRNA to shift the equilibrium in favor of the subsequent steps <a href="#pone.0000517-Sanbonmatsu1">[7]</a>. Aminoglycoside antibiotics such as paromomycin stimulate misreading by binding to the decoding center, displacing A1492 and A1493. This forces these bases to mimic the “flipped out” conformations that they normally assume in response to the mini-helix formation by a cognate codon:anticodon pair (reviewed in <a href="#pone.0000517-Ogle1">[5]</a>). Together, these kinetic and biophysical mechanisms ensure the accurate utilization of cognate aa-tRNAs.</p>
<div class="figure" id="pone-0000517-g001"><div class="img"><a name="pone-0000517-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The decoding center and dual-luciferase reporters for determining rates of translational misreading in yeast.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Panel A. The codon:anticodon mini-helix in the decoding center is stabilized by base-pairing at all three positions of the mini-helix favoring A-minor interactions with flipped out bases G520, A1492 and A1493. PyMol (Delano Scientific, LLC) was used to generate this figure based on the coordinates 1IBM in the RSCN Protein Data Bank <a href="#pone.0000517-Ogle3">[40]</a>. Panel B. In all missense reporters, transcription is initiated from the yeast <em>ADH1</em> promoter, and terminated at a sequence from the <em>CYC1</em> 3′ UTR. The luciferase genes from <em>Renilla</em> and firefly are cloned in frame to produce a fusion of the two proteins. The sense reporter has the AGA codon encoding arginine at amino acid residue 218 in the catalytic site of firefly luciferase. Missense reporters contain the indicated mutations at this position, which encode the indicated amino acids. Efficiencies of missense suppression were calculated by dividing the ratio of firefly/Renilla luciferase generated from cells harboring the missense test vectors by the ratio of firefly to Renilla luciferase generated from cells harboring the sense control plasmid.</p>
<span>doi:10.1371/journal.pone.0000517.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>An unsettled issue remains the precise distinction between “near-cognate” and “non-cognate” tRNAs, especially in eukaryotes. This is important since the rational design or utilization of therapeutics may exploit the functional differences that exist between these two classes of tRNAs. One recent study has suggested that the relative abundances of bacterial aa-tRNAs plays a significant role in translational accuracy <a href="#pone.0000517-Kramer1">[3]</a>. By this model, highly abundant aa-tRNAs are more likely than low abundance aa-tRNAs to misread codons that are decoded by other low abundance aa-tRNAs. The current study using the yeast <em>Saccharomyces cerevisiae</em> supports this competition model. However it also suggests that this is not sufficient to explain the functional differences between near- and non-cognate aa-tRNAs. Using a series of seven substitutions of a codon in the catalytic site of firefly luciferase, we show that a second critical distinction lies in the ability to form hydrogen bonding interactions at all three positions between the aa-tRNA anticodon loop and the codon in the decoding center. This is likely the result of changes in formation of the codon:anticodon mini-helix. Thus, transient formation of the mini-helix allows the rRNA and tRNA conformational changes required for activation of the GTPase activity of eEF1A, the eukaryotic homolog of EF-Tu. This is supported by the demonstration of paromomycin stimulated misreading by aa-tRNAs that are capable of forming a transient interaction. This identifies paromomycin as a functional probe to distinguish between near- and non-cognate aa-tRNAs. The hypothetical roles in this process played by eEF1A and its associated factors, eEF1Bα and eEF1Bγ, were also investigated. The results suggest that the GTPase activity of eEF1A is preferentially stimulated by near-cognate codon:anticodon interactions, and point to discrete functional regions of the protein. Studies of eEF1Bα, the catalytic subunit of the guanine nucleotide exchange factor (GEF) required for recycling of GTP-bound eEF1A, are consistent with a kinetic model in which limiting concentrations of eEF1A:aa-tRNA:GTP ternary complex should decrease rates of protein synthesis, resulting in increased selection against both near- and non-cognate aa-tRNAs. Interestingly, deletion of the proposed regulatory subunit of the GEF complex, eEF1Bγ, tended to promote increased misreading of near-cognate codons and decreased misreading of non-cognate codons. This suggests eEF1Bγ may have a regulatory function. In a final series of experiments, the potential role of the fungus-specific elongation factor eEF3 in translational fidelity was indirectly assayed through analysis of a series of mutants in the ribosomal protein L5 (rpL5). rpL5 forms part of the ribosome binding site for this factor <a href="#pone.0000517-Andersen1">[13]</a>. Like the eEF1Bα mutant, the ability of an rpL5 mutant to promote enhanced fidelity at both non- and near-cognate codons suggests that slowing rates of elongation by disrupting the synergy between eEF1A and eEF3 results in increased selection against both near- and non-cognate aa-tRNAs.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Baseline and paromomycin-stimulated rates of missense suppression suggests a functional difference between near- and non-cognate tRNAs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Introduction of a missense mutation into the active site of an enzyme followed by quantitative measurement of the restoration of enzymatic activity can provide a basis to monitor translational error rates. The arginine at position 218 of firefly luciferase is located in the active site and is required for enzymatic activity. Mutation of the corresponding AGA arginine codon to either the AGC or UCU serine codon was previously used to monitor missense error rates of ribosome bound chaperone mutants in <em>S. cerevisiae</em> <a href="#pone.0000517-Rakwalska1">[14]</a>. The current study employed a bicistronic luciferase reporter system to monitor suppression of a series of missense mutations. This could functionally distinguish translational fidelity effects due to the inherent translatability of different codons and subsequently correlate these with <em>trans</em>-acting influences. In this assay, the gene encoding firefly luciferase is fused in frame with a downstream <em>Renilla</em> luciferase gene. Test plasmids harbored missense codons at position 218 of the firefly luciferase gene (<a href="#pone-0000517-g001">Figure 1B</a>, “Missense”). The control was identical except that it contained the wild-type AGA codon at this position (<a href="#pone-0000517-g001">Figure 1B</a>, “Sense”). Even though the two luciferase proteins are fused, the activity from each can be measured independently as they utilize different substrates. Rates of misreading were calculated by dividing the ratio of firefly luciferase activity to <em>Renilla</em> luciferase activity generated from the missense vector in strains harboring the indicated mutant allele by the ratio generated with the sense plasmid. The results were statistically tested as previously described <a href="#pone.0000517-Jacobs1">[15]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Previous studies have shown that missense errors occur with frequencies on the order of 10<sup>−4</sup> in both <em>E. coli</em> and in <em>S. cerevisiae</em> <a href="#pone.0000517-Kramer1">[3]</a>, <a href="#pone.0000517-Bouadloun1">[16]</a>–<a href="#pone.0000517-Stansfield1">[18]</a>. Consistent with the literature, the seven missense mutations assayed at position 218 demonstrated ratios of firefly to <em>Renilla</em> luciferase activities that were reduced by approximately 4 orders of magnitude in all cases (<a href="#pone-0000517-t001">Table 1</a>). Although rates of misreading are low and the differences between test and control samples are small, the sensitivity of the assay and rigor of the statistical methods enable meaningful analysis of the data. Inspection of these data revealed that rates of misreading varied over an approximately 4.5-fold range. The inability of any one amino acid (i.e. Ser, Cys, or Thr) to disproportionately influence apparent missense incorporation suggests that the assay monitored incorporation errors rather than altered luciferase activity arising from incorporation of any specific amino acid. The data reveal that the common element among the three most stimulatory codons is the potential to form a stable G•C base pair at the second position. The observation of different error rates among the four serine codons (which are decoded by different tRNA families) suggests that relative tRNA abundances are also important.</p>
<div class="figure" id="pone-0000517-t001"><div class="img"><a name="pone-0000517-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Baseline levels and effects of paromomycin on suppression of missense mutations at codon 218 of firefly luciferase.</span></strong></p><span>doi:10.1371/journal.pone.0000517.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Formation of the mini-helix in the decoding center stimulates flipping out of the small subunit bases A1492 and A1493 that in combination with G530 stabilizes this structure (<a href="#pone-0000517-g001">Figure 1A</a>). Paromomycin binding to the decoding center of the small ribosomal subunit displaces A1492 and A1493, thus enhancing the frequency of missense errors (reviewed in <a href="#pone.0000517-Daviter1">[6]</a>). It has been hypothesized that not only does the formation of the mini-helix displace A1492 and A1493, but once displaced they sterically position the interribose bonds to maintain A-form helices <a href="#pone.0000517-Lim1">[19]</a>. In the current study, translational misreading errors were significantly stimulated by paromomycin at codons that are capable of forming complete mini-helices with arginyl-tRNAs: UGU, AGU and AGC (<a href="#pone-0000517-g002">Figure 2</a>). In contrast, paromomycin did not stimulate misreading with anticodons unable to form the mini-helix, e.g. UCU, UCC, ACU and ACC. These findings suggest a functional definition for near- versus non-cognate codon:anticodon interactions. This potential to form base pairing interactions at all three positions, which is possibly nucleated by a strong Watson-Crick base pair at the second position would allow transient formation of the mRNA:tRNA mini-helix. We propose this as the defining feature of a near-cognate interaction. In contrast, non-cognate interactions are defined by their lack of potential to form the mini-helix.</p>
<div class="figure" id="pone-0000517-g002"><div class="img"><a name="pone-0000517-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Proposed basis for near-cognate codon-anticodon interactions.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Top. Base pairing between tRNA<sup>Arg2</sup> anticodon and UGU codon (left), and between tRNA<sup>Arg3</sup> anticodon and the AGC and AGU codons. Bottom. I•U, U•C, and U•U base pairing.</p>
<span>doi:10.1371/journal.pone.0000517.g002</span></div>

<h4>eEF1A mutants generally affect utilization of near-cognate aa-tRNAs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In eukaryotes, a ternary complex composed of eEF1A, aa-tRNA, and GTP delivers the aa-tRNA to the A-site of the ribosome. When an aa-tRNA containing the correct anticodon is sampled by the ribosome, a signal is transmitted through the body of the tRNA. This stimulates GTP hydrolysis by eEF1A and subsequent accommodation of the tRNA into the ribosomal A-site (reviewed in <a href="#pone.0000517-Ogle1">[5]</a>, <a href="#pone.0000517-Daviter1">[6]</a>). eEF1A is encoded by the essential <em>TEF1</em> and <em>TEF2</em> genes in <em>S. cerevisiae</em>. A set of <em>TEF2</em> mutants expressed in a <em>tef1Δ tef2Δ</em> genetic background <a href="#pone.0000517-Sandbaken1">[20]</a> were assayed with respect to their effects on misreading using the AGC and UGU serine codons to monitor misreading of near- and non-cognate codons, respectively. The results show allele-specific responses specifically to near-cognate codons. Strains bearing one of six alleles (E122K, E122Q, D156N, E286K, E295K, and E317K) promoted enhanced misreading of the near-cognate AGC Ser codon, but not of non-cognate UCU Ser (<a href="#pone-0000517-g003">Figure 3A</a>, <a href="#pone-0000517-t002">Table 2</a>, p&lt;0.01). In contrast, a strain expressing the T142I and to the lesser extent N153T/D156E mutant were better able to distinguish between the cognate AGA and near-cognate AGC codons. This enhanced fidelity did not extend to the non-cognate UCU codon (<a href="#pone-0000517-g003">Figure 3A</a>, <a href="#pone-0000517-t002">Table 2</a>, p&lt;0.01). In summary, it appears that near-cognate codons and not non-cognate are able to influence eEF1A activity in an allele-specific manner.</p>
<div class="figure" id="pone-0000517-g003"><div class="img"><a name="pone-0000517-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Effects of the eEF1 complex mutants on mis-reading of near- and non-cognate codons.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Misreading of the non-cognate UCU and near-cognate AGC codons by mutant forms of eEF1A (Panel A), or by isogenic strains with <em>tef3Δ, tef4Δ</em>, or <em>tef3Δ tef4Δ</em> double null mutants or <em>tef5Δ</em> strains expressing the K120R S121Δ I122Δ allele compared to isogenic wild-type strain (Panel B). Panel C. Misreading of all seven missense codons by cells lacking both forms of eEF1Bγ (<em>tef3Δ tef4Δ</em>). Effects of the indicated mutants are depicted as fold of isogenic wild-type cells. ** indicates <em>p</em> values of &lt;0.01; * indicates <em>p</em> values of &lt;0.05.</p>
<span>doi:10.1371/journal.pone.0000517.g003</span></div><div class="figure" id="pone-0000517-t002"><div class="img"><a name="pone-0000517-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Effects of selected alleles of genes on mis-reading of non-cognate and near-cognate mutations at codon 218 of firefly luciferase.</span></strong></p><span>doi:10.1371/journal.pone.0000517.t002</span></div>

<h4>A mutation in eEF1Bα promotes general hyperfidelity</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>After hydrolysis of GTP, the eEF1A:GDP complex is released from the ribosome. The eEF1Bα subunit (encoded by <em>TEF5</em>) is the essential nucleotide exchange factor responsible for catalytic activity in eEF1A recycling <a href="#pone.0000517-Hiraga1">[21]</a>. Analysis of the role of the eEF1Bα protein capitalized on the availability of the K120R S121Δ I122Δ mutant form of the protein that altered or deleted residues involved in critical interactions with the nucleotide binding pocket of eEF1A <a href="#pone.0000517-Andersen2">[22]</a>. This mutant strain was previously shown to enhance translational fidelity by promoting lower levels of readthrough at all three stop codons <a href="#pone.0000517-CarrSchmid1">[23]</a>. In the current study, the K120R S121Δ I122Δ eEF1Bα mutant showed modest but consistent enhanced fidelity at both the UCU and AGC non- and near-cognate codons respectively (<a href="#pone-0000517-g003">Figure 3B</a>, <a href="#pone-0000517-t002">Table 2</a>, p&lt;0.01). As discussed below, we hypothesize that this may be due to limiting concentrations of eEF1A:GTP.</p>


<h4>Codon-specific misreading in the absence of eEF1Bγ</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In yeast, the eEF1 complex contains a third non-essential subunit, eEF1Bγ. Yeast cells express two isoforms of eEF1Bγ, encoded by the <em>TEF3</em> and <em>TEF4</em> genes. The N-terminus of Tef3p exhibits structural similarity to gluthathione-S-transferases and is thought to participate in the regulation of elongation during stress <a href="#pone.0000517-Jeppesen1">[24]</a>, <a href="#pone.0000517-Olarewaju1">[25]</a>. Deletion of either eEF1Bγ isoform alone had no significant effects on misreading at either the AGC near-cognate or UGU non-cognate codons (<a href="#pone-0000517-g003">Figure 3B</a>, <a href="#pone-0000517-t002">Table 2</a>). However deletion of both eEF1Bγ-encoding genes caused a statistically significant increase in mis-incorporation at near- but not at non-cognate codons (<a href="#pone-0000517-g003">Figure 3B</a>, <a href="#pone-0000517-t003">Table 3</a>). These results suggest that these two proteins have redundant activities and that this factor functions in ensuring translational fidelity. To investigate this phenomenon further, misreading in the absence of eEF1Bγ (<em>tef3Δtef4Δ</em>) was assayed at the remaining 5 missense codons. Although misreading of the near-cognate AGU Ser codon was also enhanced, there was no effect on recognition of the ‘near-cognate’ UGU Cys codon (<a href="#pone-0000517-t003">Table 3</a>, <a href="#pone-0000517-g003">Figure 3C</a>). In contrast, although recognition of the UCU ‘non-cognate’ codon was not affected, recognition of the other non-cognate codons (ACC, ACU, and UCC) was significantly enhanced by the absence of eEF1Bγ.</p>
<div class="figure" id="pone-0000517-t003"><div class="img"><a name="pone-0000517-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Survey of isogenic wild-type and mutant pairs of strains with seven different codons at codon 218 of firefly luciferase.</span></strong></p><span>doi:10.1371/journal.pone.0000517.t003</span></div>

<h4>The K27E mutant of ribosomal protein L5 promotes a general enhancement of fidelity</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The three-site model of the ribosome posits that preventing simultaneous occupancy of the ribosomal A- and E-sites by aa-tRNA and deacylated-tRNAs respectively helps to coordinate the elongation cycle (reviewed in <a href="#pone.0000517-Nierhaus1">[26]</a>). In fungi, the unique essential elongation factor 3 (eEF3) facilitates eEF1A-dependent A-site binding of aa-tRNA and has ATP-dependent activity required for the release of deacylated tRNA from the E site <a href="#pone.0000517-TrianaAlonso1">[27]</a>. Although ribosomal protein L5 (rpL5) is far from the decoding center, it has been shown to interact with eEF3 <a href="#pone.0000517-Andersen1">[13]</a>. Previous studies have characterized five temperature-sensitive alleles of the yeast <em>RPL5</em> gene encoding rpL5 <a href="#pone.0000517-Deshmukh1">[28]</a>, <a href="#pone.0000517-Meskauskas1">[29]</a>. A preliminary assay of five <em>rpl5</em> mutant strains using the near- (AGC) and non- (UCU) cognate reporters indicated that the K27E mutant of L5 tended to be generally hyperaccurate (data not shown). Analysis of all 7 missense codons revealed that the K27E mutant generally promoted greater levels of translational accuracy (<a href="#pone-0000517-g004">Figure 4A</a>, <a href="#pone-0000517-t003">Table 3</a>). Dilution spot assays revealed that rpL5-K27E is resistant to paromomycin (<a href="#pone-0000517-g004">Figure 4B</a>), consistent with the notion that this mutant is antagonistic to the action of the drug. As discussed below, these findings suggest an indirect role for rpL5 in translational decoding. This may be through its association with eEF3 and account for some differences in A-site fidelity between bacteria and fungi</p>
<div class="figure" id="pone-0000517-g004"><div class="img"><a name="pone-0000517-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Characterization of alleles of <em>RPL5.</em></span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Panel A. Effects of the K27E rpl5 mutanton misreading of seven missense codons. Effects on the indicated missense reporters are depicted as fold of isogenic wild-type cells. ** indicates p values of &lt;0.01. Panel B. Paromomycin dilution spot assays. Ten-fold dilutions (10<sup>6</sup>→10<sup>1</sup> CFU) of logarithmically growing cells were arrayed onto H-leu medium containing paromomycin (1 mg/ml) or no drug control plates. Cells were grown at 25°C for 3 days.</p>
<span>doi:10.1371/journal.pone.0000517.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>A significant number of human diseases are caused by missense mutations. Understanding the functional differences between mutations and drugs that result in near- versus non-cognate codon usage may inform strategies for therapeutic interventions. In the current study, codon misreading was examined in the yeast eukaryotic model system using a dual luciferase reporter in which the AGA Arg codon in the firefly luciferase catalytic site was changed to AGC, AGU, UCU, UCC (serine); UGU (cysteine); and ACC or ACU (threonine). Each of these substitutions has a polar sidechain like arginine, but lacks the positive charge. The consistent 4-order of magnitude decrease in firefly luciferase activity with these substitutions agrees with previously characterized missense error rates. The finding no one amino acid alters firefly luciferase activity outside a range supports that the assay monitors misincorporation of arginine. The observation that the frequency of Arg misincorporation varied over an approximately 4.5-fold range independent of the identity of the encoded amino acid suggests that misincorporation frequency is determined by other factors.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In each species, some codons are used more frequently than others. The codon bias is especially clear in highly expressed genes. Studies in <em>E. coli</em> have led to the suggestion that codon bias minimizes the deleterious effects of aberrant decoding <a href="#pone.0000517-Stoletzki1">[30]</a>, <a href="#pone.0000517-Najafabadi1">[31]</a>. Examination of codon bias in highly expressed yeast genes (see <a href="#pone.0000517-Plewniak1">[32]</a>) reveals that both the AGU and AGC Ser codons occur at a low frequency of approximately 5% of serine in the yeast genome. All of the other codons employed in this study occur with significantly higher frequencies. UGU is the most frequently used Cys codon at 84% in highly transcribed genes. Its substitution promoted even higher rates of miscoding than AGU Ser at 5% codon frequency. Thus, it is clear that codon frequency is not the sole determinant of translational fidelity. A recent study in <em>E. coli</em> suggested that competition between different tRNAs is the underlying factor influencing misreading error rates <a href="#pone.0000517-Kramer1">[3]</a>. In yeast, gene copy number for individual tRNA species correlates with initial estimates of relative tRNA content in normally growing cells, allowing the number of functional genes encoding cognate tRNAs for each codon to be used as a proxy measure of tRNA abundance <a href="#pone.0000517-Ikemura1">[33]</a>, <a href="#pone.0000517-Percudani1">[34]</a>. Examination of yeast tRNA gene copy numbers <a href="#pone.0000517-Hani1">[35]</a> reveals that AGA Arg is decoded by the highly abundant tRNA<sup>Arg3</sup> (11 copies). In support of the tRNA competition model, the codons that intrinsically promoted higher levels of misreading were decoded by lower abundance tRNAs, while those at the lower end of the range are decoded by more abundant tRNAs. For example, the AGU and AGC codons are decoded by the relatively low abundance tRNA<sup>Ser3</sup> (4 copies), and all Cys codons are decoded by the 4 copy tRNA<sup>Cys</sup>. In contrast, the UCU and UCC codons are decoded by tRNA<sup>Ser2</sup> (11 copies), and the ACC and ACU codons are decoded by the high copy tRNA<sup>Thr1a</sup> (11 copies). However, tRNA competition alone cannot fully explain the observed differences. The pattern becomes more apparent when the ability of near-cognate codons to base pair with anticodons of different arginyl-tRNAs is considered (<a href="#pone-0000517-g002">Figure 2</a>). The first two bases of the AGC and AGU Ser codons could be recognized by the <sup>mcm5</sup>UCU anticodon of the highly abundant tRNA<sup>Arg3</sup>. Base pairing at the wobble positions of these two codons and <sup>mcm5</sup>U is also theoretically possible through N3-N3, 4-carbonyl-amino, and 2-carbonyl-N3, 4-carbonyl-N3 hydrogen bonding respectively (see <a href="#pone-0000517-g002">Figure 2</a>). Note that although there are three possible U•U base pairs and two possible C•U couples, the geometries of the N3-N3, 4-carbonyl-amino, and of the 2-carbonyl-N3, 4-carbonyl-N3 hydrogen pairing schemes provide the most energetically favorable topologies within the constraints of the tRNA:mRNA mini-helix. Further, it has been suggested that cmo<sup>5</sup> modification of U<sub>34</sub> stabilizes the shape of the anticodon loop (reviewed in <a href="#pone.0000517-Agris2">[4]</a>). Although the C1′-C1′ distance between pyrimidine-pyrimidine nucleotide pairs is short, and is thus destabilizing relative to that for pyrimidine-purine base pairs, biophysical analyses suggest that bridging water molecules could produce stable and planar U<sub>34</sub>•U3 and U<sub>34</sub>•C3 base pairs (reviewed in <a href="#pone.0000517-Agris2">[4]</a>). Of note, although biophysical analyses of RNA duplexes indicate that U•U base-pairing is more stable than U•C pairs at pH 7.0, the observation that the AGC codon promoted&gt;2-fold more misreading than the AGU codon suggests that this particular U•C base pair is more energetically permissible within the topological constraints imposed by the codon:anticodon mini-helix structure. Examination of the UGU Cys codon also reveals that it can potentially base pair at all three positions with the ICG anticodon of tRNA<sup>Arg2</sup>, which is encoded by 6 genes (<a href="#pone-0000517-g002">Figure 2</a>). Although G<sub>36</sub>•U1 pairing does not normally occur in cognate codon:anticodon interactions, this has been posited to occur at the P-site in Ty<em>1</em> promoted programmed +1 ribosomal frameshifting <a href="#pone.0000517-Belcourt1">[36]</a>. The lower abundance of this tRNA in combination with the less stable G<sub>36</sub>•U1 base pair at the first position of this pair may account for the lower frequency of misreading of this codon as compared to AGC Ser. tRNA modifications that facilitate wobble position interactions are present in both bacterial and eukaryotic systems, supporting the hypothesis that the formation of a mini-helix may be central for the ribosome to distinguish between near- and non-cognate interactions across the kingdoms. Kramer and Farabaugh also noted that misreading was enhanced in the presence of paromomycin for codons where there was potential for U•U base-pairing <a href="#pone.0000517-Kramer1">[3]</a>. However, the frequency of misreading varies between eukaryotes and bacteria indicating that other trans-acting factors or ribosomal components are involved. We extended our analysis to examine the effects of some of these factors.</p>

<h4>Influence of eEF1A and associated elongation factors in translational fidelity</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Because eEF1A delivers aa-tRNA to the ribosome and cognate codon:anticodon interactions stimulate its GTPase activity, mutants of this factor could alter translational fidelity in two ways. First, altered affinity for the ribosome could affect the initial binding step. Second, changes in intrinsic GTPase activity could affect aa-tRNA stimulation threshold. Because initial binding is independent of codon:anticodon interactions, mutants affecting the first step would be expected to alter fidelity in response to both near- and non-cognate codons. Since such an outcome was not observed for any of the eEF1A mutants, it is unlikely that any of the mutants affected initial binding rates. In contrast, since codon:anticodon interactions between near-cognate aa-tRNAs might be more likely to induce tRNA structural changes than those between non-cogante aa-tRNAs, eEF1A mutants with increased intrinsic GTPase activity or with decreased activation thresholds would more likely be stimulated by near-cognate as opposed to non-cognate aa-tRNAs. Similarly, those having decreased intrinsic GTPase activity or increased activation thresholds would be more discriminatory when presented with near-cognate aa-tRNAs.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Examination of the mutants within the context of the structure of the eEF1A•aa-tRNA•GTP modeled as the ternary complex provides some clues with regard to which mechanism may be altered in these mutants (<a href="#pone-0000517-g005">Figure 5A</a>). The charge reversal mutants in domain 2 proposed to be involved in binding the tRNA acceptor stem (E286K, E295K and E317K) all promoted enhanced misreading of the near-cognate AGC codon. It is tempting to speculate that these mutants may promote increased aa-tRNA dissociation rates by mimicking the structural change induced by correct tRNA:mRNA interactions and subsequently stimulating the GTPase activity of eEF1A. In contrast, the T142I mutant in domain 1 that interacts with the phosphate backbone of residue 61 at the base of the D-loop was more discriminatory, suggesting that this interaction is important for stimulation of GTP hydrolysis. Mutants in the vicinity of the GTP binding pocket had allele-specific effects on incorporation of the near-cognate tRNA. Charge reversal or neutralization of E122K or D156N strongly stimulated misreading. Perhaps the presence of additional positive charge in this region enhances GTP binding and/or GTPase activity. In contrast, loss of a positive charge in the N153T mutant slightly inhibited misreading. Curiously, this effect was enhanced in the N153T/D156N double mutation. Last, mutations of E40 and D130 which are not closely linked to either tRNA or GTP binding did not affect missense suppression even though they were initially isolated as +1 insertion suppressor mutants. It is also striking that genetic screens have never identified fidelity mutations in domain 3, which is also proposed to interact with aa-tRNA. This suggests that these interactions are either non-essential or irrelevant to presentation at the A-site.</p>
<div class="figure" id="pone-0000517-g005"><div class="img"><a name="pone-0000517-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Modeling of mutations in eEF1A and eEF1Bα that influence influence misincorporation of missense aa-tRNAs.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The nucleotide exchange factor eEF1Bα and the fitted aa-tRNA present clashes indicating that they do not interact with eEF1A simultaneously. PyMOL (Delano Scientific, LLC) was used with the coordinates 1G7C of yeast eEF1A:eEF1Bα (amino acids 114–206) in complex with GDPNP <a href="#pone.0000517-Andersen3">[50]</a>. The ribbon structure of eEF1A is shown in blue, eEF1A mutated bases are shown in cyan, and GDPNP is indicated in magenta. Panel A. tRNA (yellow) was fitted into the structure based on coordinates obtained from the crystal structure of the EF-Tu:Phe-tRNA<sup>Phe</sup>:GCPNP complex (1TTT in the RSCN Protein Data Bank, <a href="#pone.0000517-Nissen1">[51]</a>. Panel B. Ribbon structure of eEF1Bα from 1G7C is shown in red, and the KSI residues in the mutant form used in this study are indicated in salmon.</p>
<span>doi:10.1371/journal.pone.0000517.g005</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>The K120R S121Δ I122Δ (KSI) mutant of eEF1Bα (<em>tef5–7</em>) promoted increased accuracy in decoding both the ACG near- and UCU non-cognate codons. As noted above, mutants affecting fidelity in response to both near- and non-cognate codons are likely to be due to altered affinity at the initial binding step. Previous genetic analyses showed that this mutant also promoted increased fidelity at nonsense codons, slower rates of growth and protein synthesis, and hypersensitivity to translational inhibitors <a href="#pone.0000517-CarrSchmid1">[23]</a>. <a href="#pone-0000517-g005">Figure 5B</a> shows the X-ray crystal structure of eEF1A in complex with the catalytic terminus of eEF1Bα. eEF1Bα binds to domain I and II of eEF1A, where KSI amino acids are located close to the nucleotide-binding pocket in domain 1 of eEF1A. The binding site of eEF1Bα to domain II overlaps the proposed aa-tRNA binding site. This is significantly different from the bacterial homolog, and may be another way in which eukaryotes and bacteria differ in maintaining fidelity and promoting aa-tRNA delivery. We propose that these residues may facilitate exchange of GDP for GTP by eEF1A. Slowing this process would serve to limit the availability of functional ternary complexes, thus affecting binding of both near- and non-cognate codons. The effects of loss of eEF1Bγ shows a general trend of enhanced accuracy of non-cognate codons like an eEF1Bα mutant. However, the increased misreading of a near-cognate codon is more like an eEF1A mutant. Thus, although the precise function of eEF1Bγ has yet to be determined, it is reasonable to hypothesize that it may modulate eEF1Bα to affect fidelity.</p>


<h4>Ribosomal protein L5: Coordination of tRNA exit from the E-site with aa-tRNA entrance at the A-site</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The observation that the rpL5-K27E mutant generally promoted increased accuracy was initially surprising since this protein is located on the back of the central protuberance of the large subunit, far away from the ternary complex binding site, the ribosomal A-site, and the decoding center. In addition, a previous study implicated rpL5 in binding of peptidyl-tRNA, but not of aa-tRNAs <a href="#pone.0000517-Meskauskas1">[29]</a>. Nonetheless, the rpL5-K27E mutant generally promoted increased fidelity in response to almost all of the near- and non-cognate codons tested. An intriguing explanation for the allele-specific effects observed here may come from the observation that rpL5-L5 interacts with the fungal-specific elongation factor eEF3 <a href="#pone.0000517-Andersen1">[13]</a>. eEF3 is an ATPase that interacts with eEF1A and catalyzes release of deacylated-tRNA from the ribosomal E-site <a href="#pone.0000517-TrianaAlonso1">[27]</a>, <a href="#pone.0000517-Anand1">[37]</a>. It has been proposed that eEF3 and eEF1A work synergistically to remove deacylated tRNAs from the E-site and promote delivery of cognate aa-tRNA to the A-site <a href="#pone.0000517-TrianaAlonso1">[27]</a>. It is tempting to speculate that altered eEF3 binding to the K27E form of rpL5 upsets this synergy. Similar to the model proposed for the eEF1Bα KSI mutant, the rpL5-K27E mutant might promote enhanced accuracy at the codon recognition step.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>It should also be noted that a previous study also used a dual luciferase reporter system to examine missense suppression in yeast <a href="#pone.0000517-SalasMarco1">[17]</a>. The analyses applied in the current study cannot be directly applied to the data generated by Salas-Marco and Bedwell <a href="#pone.0000517-SalasMarco1">[17]</a> since that work employed eight different codons at two positions of firefly luciferase. Since most were non- synonymous, the effects of different amino acid substitutions and their specific locations on the activity of the enzyme cannot be controlled for. Since different strain backgrounds are used in the studies and even in different sets of mutants analyzed in this work, variations in wild-type values observed requires the use of specific statistical methodology for accurate comparisons between datasets <a href="#pone.0000517-Jacobs1">[15]</a>. Further, relative tRNA abundance issues complicated the two instances where synonymous codons were used. Despite these differences, independent analyses, reporter constructs and strain backgrounds showed similar levels of misreading. Furthermore, specific effects on misreading were observed for mutants of ribosomal components and key factors involved in elongation and termination. These data support the universal application of this approach to studies of translational fidelity.</p>


<h4>Kinetics: the difference between near- and non-cognate interactions may occur at the GTPase activation step</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>As described above, studies in the bacterial system shows accuracy during translation elongation is likely a two step process involving two distinct biophysical mechanisms: initial selection and proofreading (reviewed in <a href="#pone.0000517-Ogle1">[5]</a>, <a href="#pone.0000517-Rodnina1">[38]</a>). The first step, initial binding, is mostly determined by the interaction between the ribosome and EF-Tu, and forward and reverse rates (k<sub>1</sub> and k<sub>−1</sub>) are not affected by aa-tRNA identity <a href="#pone.0000517-Daviter1">[6]</a>. However, during the next step of codon recognition, the stabilizing effects due to interactions of G530, A1492 and A1493 with the mini-helix results in dissociation rates (k<sub>−2</sub>) of near-cognate aa-tRNAs being approximately 400-fold than those of cognate aa-tRNAs <a href="#pone.0000517-Gromadski1">[39]</a>. Computational modeling suggests the existence of two major energy minima at the decoding center corresponding to the flipped-in and flipped-out conformations of A1492, and A1493, and that fast flipping between the two states provides a kinetic means to discriminate at the level of codon:anticodon interactions <a href="#pone.0000517-Sanbonmatsu3">[10]</a>. Formation of the stable mini-helix results in the physical transduction of information to the ternary complex, thus activating the endogenous GTPase of EF-Tu (k<sub>3</sub>). This step acts as a kinetic trap to select for aa-tRNAs capable of forming the mini-helix. We propose that it is here that the difference between near- and non-cognate tRNA:mRNA interactions also occurs in eukaryotes. The potential of near-cognate interactions to form mini-helices, albeit at lower frequencies, provides the opportunity for stimulation of GTPase activation. The data also suggest that the presence of a canonical Watson-Crick base pair between N<sub>35</sub>•N′2 may aid in nucleating mini-helix formation, consistent with molecular dynamics modeling showing that stability testing by the kink in the mRNA between the P- and A-site codons destabilizes position 2 mismatches more severely than mismatches at the first position <a href="#pone.0000517-Sanbonmatsu2">[9]</a>. In contrast, non-cognate interactions cannot possibly form mini-helices, and thus are incapable of forming the kinetic trap and activating GTP hydrolysis. In a kinetic analysis comparing different codon:anticodon mismatches, one tRNA capable of participating in a non-cognate interaction was employed and stimulated GTPase activation approximately 6.7 fold less than the near-cognate codons <a href="#pone.0000517-Gromadski1">[39]</a>. This is consistent with the ~4.5 fold increased rates of misreading promoted by near-cognate codons in the current study.</p>


<h4>Using paromomycin to functionally distinguish between ‘near-’ and ‘non-cognate’ codon:anticodon interactions</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Binding of cognate aa-tRNA stimulates rearrangement of G530, A1492, and A1493 to establish A-minor interactions between themselves and the minor groove of the codon-anticodon helix <a href="#pone.0000517-Ogle2">[11]</a>, <a href="#pone.0000517-Ogle3">[40]</a>. Binding of paromomycin to the decoding center stimulates similar displacement of A1492 and A1493, positioning them to stabilize codon:anticodon interactions in a promiscuous manner <a href="#pone.0000517-Carter1">[41]</a>, perhaps trapping them in the flipped-out state <a href="#pone.0000517-Sanbonmatsu3">[10]</a>. In light of the data presented here, we suggest that mini-helix formation is a precondition for paromomycin-stimulated misreading. Furthermore, paromomycin-enhanced and the potential for mini-helix formation are coordinately maximized in the decoding center. Thus, paromomycin has the potential to be used as a tool to functionally distinguish between ‘near-’ and ‘non-cognate’ codons. However, this requires expansion of the system to utilize other codons for which the encoded amino acids do not result in a partially active luciferase protein. Kramer and Farabaugh observed changes in misreading with two different aminoglycosides, paromomycin and streptomycin <a href="#pone.0000517-Kramer1">[3]</a>. Although more information about aminoglycoside-ribosome interactions has recently become available (reviewed in <a href="#pone.0000517-Hobbie1">[42]</a>) more experimental work needs to be performed to determine if any aminoglycosides other than paromomycin may be better sensors of cognate status.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4><em>E. coli</em> and yeast strains and genetic methods</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>E. coli</em> strain DH5α was used to amplify plasmids. High efficiency transformations were performed as previously described <a href="#pone.0000517-Inoue1">[43]</a>. The <em>Saccharomyces cerevisiae</em> strains used in this study are listed in <a href="#pone-0000517-t004">Table 4</a>. The <em>RPL5</em> strains were generously provided by Dr. John Woolford. Isogenic <em>TEF2, TEF3</em>, and <em>TEF4</em> yeast strains were previously described <a href="#pone.0000517-Olarewaju1">[25]</a>, <a href="#pone.0000517-Anand2">[44]</a>, <a href="#pone.0000517-CarrSchmid2">[45]</a>. Strains were cultured on YPAD or synthetic complete medium (H-) <a href="#pone.0000517-Dinman1">[46]</a> and were freshly plated and incubated for two to five days at 30°C prior to transformation. Yeast were transformed with the alkali cation method <a href="#pone.0000517-Ito1">[47]</a>, plated on appropriate selective media, and incubated at 30°C for four days. To assay for paromomycin sensitivity, 10-fold dilutions of logarithmically growing cells were spotted onto H-leu containing drug at a concentration of 1 mg/ml, or onto no-drug control plates, and grown at 25°C for 3 days.</p>
<div class="figure" id="pone-0000517-t004"><div class="img"><a name="pone-0000517-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000517" data-uri="info:doi/10.1371/journal.pone.0000517.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000517.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000517.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000517.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000517.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Yeast Strains used in this study</span></strong></p><span>doi:10.1371/journal.pone.0000517.t004</span></div>

<h4>Plasmid Constructs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Plasmids used in this study contained a dual luciferase cassette on a yeast vector backbone with the <em>URA3</em> selectable marker. The parental pYDL-Control plasmid containing the wild-type <em>Renilla</em> and firefly luciferase genes has been described previously <a href="#pone.0000517-Harger1">[48]</a>. Missense mutations were introduced into the arginine codon (AGA) at position 218 in the catalytic site of firefly luciferase <a href="#pone.0000517-Rakwalska1">[14]</a> using variations on the following primers:</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>5′-ATGCGAGAANNNGACGCAGGCAGTTCTATG-3′ and 5′-GCCTGCGTCN′N′N′TTCTCGCATGCCAGAGATC-3′ (Integrated DNA Technology, Coralville, IA) where N denotes bases at codon 218 that were changed on the sense strand, and N′ are the corresponding bases mutagenized on the antisense strand. Oligonucleotide site directed mutagenesis reactions were performed using the StrataGene Quikchange II kit (La Jolla, CA) according to the manufacturer's instructions. The seven mutants thus created are listed in <a href="#pone-0000517-t001">Table 1</a>. A second set of plasmids containing the yeast <em>TRP1</em> reporter were constructed by transferring the dual luciferase cassettes from the resulting plasmids as Spe<em>I</em>–Xho<em>I</em> fragments into p414 ADH <a href="#pone.0000517-Mumberg1">[49]</a>.</p>


<h4>Dual Luceriferase Assays</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Transformed yeast cells were grown overnight in selective medium at 30°C to OD<sub>595</sub> of 0.8 to 1.0. Cells were pelleted by centrifugation, washed twice with 0.5 ml of cold lysis buffer (phosphate buffered saline containing 1 mM phenylmethylsulfonylfluoride), resuspended in cold lysis buffer and broken by agitation with glass beads (0.5 mm BioSpec). Lysates were clarified by centrifugation, and supernatants transferred to pre-chilled tubes. Luminescence reactions were initiated by addition of 50 µl of Promega DLR system to 5 µl of clarified cell lysates and measured using a Turner Design TD20/20 luminometer. At least three readings were taken for each assay and all assays were repeated (n = 3 – 12) until the data were normally distributed to enable statistical analyses both within and between experiments <a href="#pone.0000517-Jacobs1">[15]</a>. An unpaired two-sample <em>t</em>-test was used to test the hypothesis that two datasets came from the same population, a rejected hypothesis indicating that the datasets were significantly different. The <em>P</em>-values from this test is the estimation of the probability of an incorrect conclusion <a href="#pone.0000517-Jacobs1">[15]</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Ashton Trey Belew for assistance with data analysis, and Jennifer Baxter-Roshek and Rasa Rakauskaite for their technical and supervisory assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JD EP. Performed the experiments: EP PN JR YP TN JQ. Analyzed the data: TK JD EP JR. Contributed reagents/materials/analysis tools: TK JD PN. Wrote the paper: TK JD EP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000517-Crick1" id="pone.0000517-Crick1"></a>Crick FHC (1966) Codon-Anticodon Pairing - Wobble Hypothesis. Journal of Molecular Biology  19: 548–555.  <ul class="find" data-citedArticleID="957800" data-doi="10.1016/s0022-2836(66)80022-0"><li><a href="http://dx.doi.org/10.1016/s0022-2836(66)80022-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-Anticodon+Pairing+-+Wobble+Hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-Anticodon+Pairing+-+Wobble+Hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000517-Agris1" id="pone.0000517-Agris1"></a>Agris PF (1991) Wobble position modified nucleosides evolved to select transfer RNA codon recognition: a modified-wobble hypothesis. Biochimie  73: 1345–1349.  <ul class="find" data-citedArticleID="957772" data-doi="10.1016/0300-9084(91)90163-u"><li><a href="http://dx.doi.org/10.1016/0300-9084(91)90163-u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wobble+position+modified+nucleosides+evolved+to+select+transfer+RNA+codon+recognition%3A+a+modified-wobble+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wobble+position+modified+nucleosides+evolved+to+select+transfer+RNA+codon+recognition%3A+a+modified-wobble+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000517-Kramer1" id="pone.0000517-Kramer1"></a>Kramer EB, Farabaugh PJ (2007) The frequency of translational misreading errors in E. coli is largely determined by tRNA competition. RNA  13: 87–96.  <ul class="find" data-citedArticleID="957830" data-doi="10.1261/rna.294907"><li><a href="http://dx.doi.org/10.1261/rna.294907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+frequency+of+translational+misreading+errors+in+E.+coli+is+largely+determined+by+tRNA+competition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+frequency+of+translational+misreading+errors+in+E.+coli+is+largely+determined+by+tRNA+competition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000517-Agris2" id="pone.0000517-Agris2"></a>Agris PF, Vendeix FA, Graham WD (2007) tRNA's wobble decoding of the genome: 40 years of modification. J Mol Biol  366: 1–13.  <ul class="find" data-citedArticleID="957774" data-doi="10.1016/j.jmb.2006.11.046"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.11.046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNA%27s+wobble+decoding+of+the+genome%3A+40+years+of+modification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNA%27s+wobble+decoding+of+the+genome%3A+40+years+of+modification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000517-Ogle1" id="pone.0000517-Ogle1"></a>Ogle JM, Ramakrishnan V (2005) Structural Insights into Translational Fidelity. Annu Rev Biochem  74: 129–177.  <ul class="find" data-citedArticleID="957844" data-doi="10.1146/annurev.biochem.74.061903.155440"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.74.061903.155440" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+Insights+into+Translational+Fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+Insights+into+Translational+Fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000517-Daviter1" id="pone.0000517-Daviter1"></a>Daviter T, Gromadski KB, Rodnina MV (2006) The ribosome's response to codon-anticodon mismatches. Biochimie  88: 1001–1011.  <ul class="find" data-citedArticleID="957802" data-doi="10.1016/j.biochi.2006.04.013"><li><a href="http://dx.doi.org/10.1016/j.biochi.2006.04.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ribosome%27s+response+to+codon-anticodon+mismatches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ribosome%27s+response+to+codon-anticodon+mismatches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000517-Sanbonmatsu1" id="pone.0000517-Sanbonmatsu1"></a>Sanbonmatsu KY (2006) Alignment/misalignment hypothesis for tRNA selection by the ribosome. Biochimie  88: 1075–1089.  <ul class="find" data-citedArticleID="957862" data-doi="10.1016/j.biochi.2006.07.002"><li><a href="http://dx.doi.org/10.1016/j.biochi.2006.07.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alignment%2Fmisalignment+hypothesis+for+tRNA+selection+by+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alignment%2Fmisalignment+hypothesis+for+tRNA+selection+by+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000517-Cochella1" id="pone.0000517-Cochella1"></a>Cochella L, Brunelle JL, Green R (2007) Mutational analysis reveals two independent molecular requirements during transfer RNA selection on the ribosome. Nat Struct Mol Biol  14: 30–36.  <ul class="find" data-citedArticleID="957796" data-doi="10.1038/nsmb1183"><li><a href="http://dx.doi.org/10.1038/nsmb1183" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+reveals+two+independent+molecular+requirements+during+transfer+RNA+selection+on+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+reveals+two+independent+molecular+requirements+during+transfer+RNA+selection+on+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000517-Sanbonmatsu2" id="pone.0000517-Sanbonmatsu2"></a>Sanbonmatsu KY, Joseph S (2003) Understanding discrimination by the ribosome: stability testing and groove measurement of codon-anticodon pairs. J Mol Biol  328: 33–47.  <ul class="find" data-citedArticleID="957864" data-doi="10.1016/s0022-2836(03)00236-5"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00236-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+discrimination+by+the+ribosome%3A+stability+testing+and+groove+measurement+of+codon-anticodon+pairs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+discrimination+by+the+ribosome%3A+stability+testing+and+groove+measurement+of+codon-anticodon+pairs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000517-Sanbonmatsu3" id="pone.0000517-Sanbonmatsu3"></a>Sanbonmatsu KY (2006) Energy landscape of the ribosomal decoding center. Biochimie  88: 1053–1059.  <ul class="find" data-citedArticleID="957866" data-doi="10.1016/j.biochi.2006.06.012"><li><a href="http://dx.doi.org/10.1016/j.biochi.2006.06.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Energy+landscape+of+the+ribosomal+decoding+center." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Energy+landscape+of+the+ribosomal+decoding+center.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000517-Ogle2" id="pone.0000517-Ogle2"></a>Ogle JM, Murphy FV, Tarry MJ, Ramakrishnan V (2002) Selection of tRNA by the Ribosome Requires a Transition from an Open to a Closed Form. Cell  111: 721–732.  <ul class="find" data-citedArticleID="957846" data-doi="10.1016/s0092-8674(02)01086-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)01086-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+of+tRNA+by+the+Ribosome+Requires+a+Transition+from+an+Open+to+a+Closed+Form." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+of+tRNA+by+the+Ribosome+Requires+a+Transition+from+an+Open+to+a+Closed+Form.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000517-Cochella2" id="pone.0000517-Cochella2"></a>Cochella L, Green R (2005) Fidelity in protein synthesis. Curr Biol  15: R536–R540.  <ul class="find" data-citedArticleID="957798" data-doi="10.1016/j.cub.2005.07.018"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.07.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fidelity+in+protein+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fidelity+in+protein+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000517-Andersen1" id="pone.0000517-Andersen1"></a>Andersen CB, Becker T, Blau M, Anand M, Halic M, et al.  (2006) Structure of eEF3 and the mechanism of transfer RNA release from the E-site. Nature  443: 663–668.  <ul class="find" data-citedArticleID="957780" data-doi="10.1038/nature05126"><li><a href="http://dx.doi.org/10.1038/nature05126" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+eEF3+and+the+mechanism+of+transfer+RNA+release+from+the+E-site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+eEF3+and+the+mechanism+of+transfer+RNA+release+from+the+E-site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000517-Rakwalska1" id="pone.0000517-Rakwalska1"></a>Rakwalska M, Rospert S (2004) The Ribosome-Bound Chaperones RAC and Ssb1/2p Are Required for Accurate Translation in Saccharomyces cerevisiae. Mol Cell Biol  24: 9186–9197.  <ul class="find" data-citedArticleID="957856" data-doi="10.1128/mcb.24.20.9186-9197.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.20.9186-9197.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Ribosome-Bound+Chaperones+RAC+and+Ssb1%2F2p+Are+Required+for+Accurate+Translation+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Ribosome-Bound+Chaperones+RAC+and+Ssb1%2F2p+Are+Required+for+Accurate+Translation+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000517-Jacobs1" id="pone.0000517-Jacobs1"></a>Jacobs JL, Dinman JD (2004) Systematic analysis of bicistronic reporter assay data. Nucleic Acids Res  32: e160–e170.  <ul class="find" data-citedArticleID="957826" data-doi="10.1093/nar/gnh157"><li><a href="http://dx.doi.org/10.1093/nar/gnh157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+analysis+of+bicistronic+reporter+assay+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+analysis+of+bicistronic+reporter+assay+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000517-Bouadloun1" id="pone.0000517-Bouadloun1"></a>Bouadloun F, Donner D, Kurland CG (1983) Codon-specific missense errors in vivo. EMBO J  2: 1351–1356.  <ul class="find" data-citedArticleID="957788"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon-specific+missense+errors+in+vivo.&amp;auth=&amp;atitle=Codon-specific+missense+errors+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-specific+missense+errors+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-specific+missense+errors+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000517-SalasMarco1" id="pone.0000517-SalasMarco1"></a>Salas-Marco J, Bedwell DM (2005) Discrimination between defects in elongation fidelity and termination efficiency provides mechanistic insights into translational readthrough. J Mol Biol  348: 801–815.  <ul class="find" data-citedArticleID="957860" data-doi="10.1016/j.jmb.2005.03.025"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.03.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discrimination+between+defects+in+elongation+fidelity+and+termination+efficiency+provides+mechanistic+insights+into+translational+readthrough." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discrimination+between+defects+in+elongation+fidelity+and+termination+efficiency+provides+mechanistic+insights+into+translational+readthrough.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000517-Stansfield1" id="pone.0000517-Stansfield1"></a>Stansfield I, Jones KM, Herbert P, Lewendon A, Shaw WV, et al.  (1998) Missense translation errors in <em>Saccharomyces cerevisiae</em>. Journal of Molecular Biology  282: 13–24.  <ul class="find" data-citedArticleID="957870" data-doi="10.1006/jmbi.1998.1976"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1976" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Missense+translation+errors+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Missense+translation+errors+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000517-Lim1" id="pone.0000517-Lim1"></a>Lim VI, Curran JF, Garber MB (2005) Ribosomal elongation cycle: energetic, kinetic and stereochemical aspects. J Mol Biol  351: 470–480.  <ul class="find" data-citedArticleID="957832" data-doi="10.1016/j.jmb.2005.06.019"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.06.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+elongation+cycle%3A+energetic%2C+kinetic+and+stereochemical+aspects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+elongation+cycle%3A+energetic%2C+kinetic+and+stereochemical+aspects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000517-Sandbaken1" id="pone.0000517-Sandbaken1"></a>Sandbaken MG, Culbertson MR (1988) Mutations in elongation factor EF-1α affect the frequency of frameshifting and amino acid misincorporation in <em>Saccharomyces cerevisiae</em>. Genetics  120: 923–934.  <ul class="find" data-citedArticleID="957868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutations+in+elongation+factor+EF-1%26%23x3b1%3B+affect+the+frequency+of+frameshifting+and+amino+acid+misincorporation+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Mutations+in+elongation+factor+EF-1%26%23x3b1%3B+affect+the+frequency+of+frameshifting+and+amino+acid+misincorporation+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+elongation+factor+EF-1%26%23x3b1%3B+affect+the+frequency+of+frameshifting+and+amino+acid+misincorporation+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+elongation+factor+EF-1%26%23x3b1%3B+affect+the+frequency+of+frameshifting+and+amino+acid+misincorporation+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000517-Hiraga1" id="pone.0000517-Hiraga1"></a>Hiraga K, Suzuki K, Tsuchiya E, Miyakawa T (1993) Cloning and characterization of the elongation factor EF-1 beta homologue of Saccharomyces cerevisiae. EF-1 beta is essential for growth. FEBS Lett  316: 165–169.  <ul class="find" data-citedArticleID="957816" data-doi="10.1016/0014-5793(93)81208-h"><li><a href="http://dx.doi.org/10.1016/0014-5793(93)81208-h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+characterization+of+the+elongation+factor+EF-1+beta+homologue+of+Saccharomyces+cerevisiae.+EF-1+beta+is+essential+for+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+characterization+of+the+elongation+factor+EF-1+beta+homologue+of+Saccharomyces+cerevisiae.+EF-1+beta+is+essential+for+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000517-Andersen2" id="pone.0000517-Andersen2"></a>Andersen GR, Pedersen L, Valente L, Chatterjee I, Kinzy TG, et al.  (2000) Structural basis for nucleotide exchange and competition with tRNA in the yeast elongation factor complex eEF1A:eEF1Balpha. Mol Cell  6: 1261–1266.  <ul class="find" data-citedArticleID="957782" data-doi="10.1016/s1097-2765(00)00122-2"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)00122-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+nucleotide+exchange+and+competition+with+tRNA+in+the+yeast+elongation+factor+complex+eEF1A%3AeEF1Balpha." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+nucleotide+exchange+and+competition+with+tRNA+in+the+yeast+elongation+factor+complex+eEF1A%3AeEF1Balpha.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000517-CarrSchmid1" id="pone.0000517-CarrSchmid1"></a>Carr-Schmid A, Valente L, Loik VI, Williams T, Starita LM, et al.  (1999) Mutations in elongation factor 1beta, a guanine nucleotide exchange factor, enhance translational fidelity. Mol Cell Biol  19: 5257–5266.  <ul class="find" data-citedArticleID="957790"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutations+in+elongation+factor+1beta%2C+a+guanine+nucleotide+exchange+factor%2C+enhance+translational+fidelity.&amp;auth=&amp;atitle=Mutations+in+elongation+factor+1beta%2C+a+guanine+nucleotide+exchange+factor%2C+enhance+translational+fidelity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+elongation+factor+1beta%2C+a+guanine+nucleotide+exchange+factor%2C+enhance+translational+fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+elongation+factor+1beta%2C+a+guanine+nucleotide+exchange+factor%2C+enhance+translational+fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000517-Jeppesen1" id="pone.0000517-Jeppesen1"></a>Jeppesen MG, Ortiz P, Shepard W, Kinzy TG, Nyborg J, et al.  (2003) The crystal structure of the glutathione S-transferase-like domain of elongation factor 1Bgamma from <em>Saccharomyces cerevisiae</em>. J Biol Chem  278: 47190–47198.  <ul class="find" data-citedArticleID="957828" data-doi="10.1074/jbc.m306630200"><li><a href="http://dx.doi.org/10.1074/jbc.m306630200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+crystal+structure+of+the+glutathione+S-transferase-like+domain+of+elongation+factor+1Bgamma+from+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+crystal+structure+of+the+glutathione+S-transferase-like+domain+of+elongation+factor+1Bgamma+from+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000517-Olarewaju1" id="pone.0000517-Olarewaju1"></a>Olarewaju O, Ortiz PA, Chowdhury W, Chatterjee I, Kinzy TG (2004) The translation elongation factor, eEF1B,plays a role in the oxidative stress response pathway. RNA Biology  1: 12–17.  <ul class="find" data-citedArticleID="957850" data-doi="10.4161/rna.1.2.1033"><li><a href="http://dx.doi.org/10.4161/rna.1.2.1033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+translation+elongation+factor%2C+eEF1B%2Cplays+a+role+in+the+oxidative+stress+response+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+translation+elongation+factor%2C+eEF1B%2Cplays+a+role+in+the+oxidative+stress+response+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000517-Nierhaus1" id="pone.0000517-Nierhaus1"></a>Nierhaus KH (1990) The allosteric three-site model for the ribosomal elongation cycle: features and future. Biochemistry  29: 4997–5008.  <ul class="find" data-citedArticleID="957840" data-doi="10.1021/bi00473a001"><li><a href="http://dx.doi.org/10.1021/bi00473a001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+allosteric+three-site+model+for+the+ribosomal+elongation+cycle%3A+features+and+future." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+allosteric+three-site+model+for+the+ribosomal+elongation+cycle%3A+features+and+future.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000517-TrianaAlonso1" id="pone.0000517-TrianaAlonso1"></a>Triana-Alonso FJ, Chakraburtty K, Nierhaus KH (1995) The elongation factor 3 unique in higher fungi and essential for protein biosynthesis is an E site factor. J Biol Chem  270: 20473–20478.  <ul class="find" data-citedArticleID="957874" data-doi="10.1074/jbc.270.35.20473"><li><a href="http://dx.doi.org/10.1074/jbc.270.35.20473" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+elongation+factor+3+unique+in+higher+fungi+and+essential+for+protein+biosynthesis+is+an+E+site+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+elongation+factor+3+unique+in+higher+fungi+and+essential+for+protein+biosynthesis+is+an+E+site+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000517-Deshmukh1" id="pone.0000517-Deshmukh1"></a>Deshmukh M, Stark J, Yeh LC, Lee JC, Woolford JL Jr (1995) Multiple regions of yeast ribosomal protein L1 are important for its interaction with 5 S rRNA and assembly into ribosomes. J Biol Chem  270: 30148–30156.  <ul class="find" data-citedArticleID="957804" data-doi="10.1074/jbc.270.50.30148"><li><a href="http://dx.doi.org/10.1074/jbc.270.50.30148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+regions+of+yeast+ribosomal+protein+L1+are+important+for+its+interaction+with+5+S+rRNA+and+assembly+into+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+regions+of+yeast+ribosomal+protein+L1+are+important+for+its+interaction+with+5+S+rRNA+and+assembly+into+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000517-Meskauskas1" id="pone.0000517-Meskauskas1"></a>Meskauskas A, Dinman JD (2001) Ribosomal protein L5 helps anchor peptidyl-tRNA to the P-site in <em>Saccharomyces cerevisiae</em>. RNA  7: 1084–1096.  <ul class="find" data-citedArticleID="957834" data-doi="10.1017/s1355838201001480"><li><a href="http://dx.doi.org/10.1017/s1355838201001480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+protein+L5+helps+anchor+peptidyl-tRNA+to+the+P-site+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+protein+L5+helps+anchor+peptidyl-tRNA+to+the+P-site+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000517-Stoletzki1" id="pone.0000517-Stoletzki1"></a>Stoletzki N, Eyre-Walker A (2007) Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy. Mol Biol Evol  24: 374–381.  <ul class="find" data-citedArticleID="957872" data-doi="10.1093/molbev/msl166"><li><a href="http://dx.doi.org/10.1093/molbev/msl166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synonymous+codon+usage+in+Escherichia+coli%3A+selection+for+translational+accuracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synonymous+codon+usage+in+Escherichia+coli%3A+selection+for+translational+accuracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000517-Najafabadi1" id="pone.0000517-Najafabadi1"></a>Najafabadi HS, Lehmann J, Omidi M (2007) Error minimization explains the codon usage of highly expressed genes in <em>Escherichia coli</em>. Gene  387: 150–155.  <ul class="find" data-citedArticleID="957838" data-doi="10.1016/j.gene.2006.09.004"><li><a href="http://dx.doi.org/10.1016/j.gene.2006.09.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Error+minimization+explains+the+codon+usage+of+highly+expressed+genes+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Error+minimization+explains+the+codon+usage+of+highly+expressed+genes+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000517-Plewniak1" id="pone.0000517-Plewniak1"></a>Plewniak F (2007) GCG Documentation.   <a href="http://www.vet.gla.ac.uk/GCGdoc/Data_Files/codon_freq_tables.html">http://www.vet.gla.ac.uk/GCGdoc/Data_Fil​es/codon_freq_tables.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000517-Ikemura1" id="pone.0000517-Ikemura1"></a>Ikemura T (1982) Correlation between the abundance of yeast transfer RNAs and the occurrence of the respective codons in protein genes. Differences in synonymous codon choice patterns of yeast and <em>Escherichia coli</em> with reference to the abundance of isoaccepting transfer RNAs. J Mol Biol  158: 573–597.  <ul class="find" data-citedArticleID="957820" data-doi="10.1016/0022-2836(82)90250-9"><li><a href="http://dx.doi.org/10.1016/0022-2836(82)90250-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+the+abundance+of+yeast+transfer+RNAs+and+the+occurrence+of+the+respective+codons+in+protein+genes.+Differences+in+synonymous+codon+choice+patterns+of+yeast+and+Escherichia+coli+with+reference+to+the+abundance+of+isoaccepting+transfer+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+the+abundance+of+yeast+transfer+RNAs+and+the+occurrence+of+the+respective+codons+in+protein+genes.+Differences+in+synonymous+codon+choice+patterns+of+yeast+and+Escherichia+coli+with+reference+to+the+abundance+of+isoaccepting+transfer+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000517-Percudani1" id="pone.0000517-Percudani1"></a>Percudani R, Pavesi A, Ottonello S (1997) Transfer RNA gene redundancy and translational selection in <em>Saccharomyces cerevisiae</em>. J Mol Biol  268: 322–330.  <ul class="find" data-citedArticleID="957852" data-doi="10.1006/jmbi.1997.0942"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.0942" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+RNA+gene+redundancy+and+translational+selection+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+RNA+gene+redundancy+and+translational+selection+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000517-Hani1" id="pone.0000517-Hani1"></a>Hani J, Feldmann H (1998) tRNA genes and retroelements in the yeast genome. Nucleic Acids Res  26: 689–696.  <ul class="find" data-citedArticleID="957812" data-doi="10.1093/nar/26.3.689"><li><a href="http://dx.doi.org/10.1093/nar/26.3.689" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNA+genes+and+retroelements+in+the+yeast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNA+genes+and+retroelements+in+the+yeast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000517-Belcourt1" id="pone.0000517-Belcourt1"></a>Belcourt MF, Farabaugh PJ (1990) Ribosomal frameshifting in the yeast retrotransposon Ty: tRNAs induce slippage on a 7 nucleotide minimal site. Cell  62: 339–352.  <ul class="find" data-citedArticleID="957786" data-doi="10.1016/0092-8674(90)90371-k"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90371-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+frameshifting+in+the+yeast+retrotransposon+Ty%3A+tRNAs+induce+slippage+on+a+7+nucleotide+minimal+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+frameshifting+in+the+yeast+retrotransposon+Ty%3A+tRNAs+induce+slippage+on+a+7+nucleotide+minimal+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000517-Anand1" id="pone.0000517-Anand1"></a>Anand M, Chakraburtty K, Marton MJ, Hinnebusch AG, Kinzy TG (2002) Functional interactions between yeast translation elongation factors eEF1A and eEF3. J Biol Chem  278: 6985–6991.  <ul class="find" data-citedArticleID="957776" data-doi="10.1074/jbc.m209224200"><li><a href="http://dx.doi.org/10.1074/jbc.m209224200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+interactions+between+yeast+translation+elongation+factors+eEF1A+and+eEF3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+interactions+between+yeast+translation+elongation+factors+eEF1A+and+eEF3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000517-Rodnina1" id="pone.0000517-Rodnina1"></a>Rodnina MV, Gromadski KB, Kothe U, Wieden HJ (2005) Recognition and selection of tRNA in translation. FEBS Lett  579: 938–942.  <ul class="find" data-citedArticleID="957858" data-doi="10.1016/j.febslet.2004.11.048"><li><a href="http://dx.doi.org/10.1016/j.febslet.2004.11.048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+and+selection+of+tRNA+in+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+and+selection+of+tRNA+in+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000517-Gromadski1" id="pone.0000517-Gromadski1"></a>Gromadski KB, Daviter T, Rodnina MV (2006) A uniform response to mismatches in codon-anticodon complexes ensures ribosomal fidelity. Mol Cell  21: 369–377.  <ul class="find" data-citedArticleID="957810" data-doi="10.1016/j.molcel.2005.12.018"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.12.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+uniform+response+to+mismatches+in+codon-anticodon+complexes+ensures+ribosomal+fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+uniform+response+to+mismatches+in+codon-anticodon+complexes+ensures+ribosomal+fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000517-Ogle3" id="pone.0000517-Ogle3"></a>Ogle JM, Brodersen DE, Clemons WM Jr, Tarry MJ, Carter AP, et al.  (2001) Recognition of cognate transfer RNA by the 30S ribosomal subunit. Science  292: 897–902.  <ul class="find" data-citedArticleID="957848" data-doi="10.1126/science.1060612"><li><a href="http://dx.doi.org/10.1126/science.1060612" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+cognate+transfer+RNA+by+the+30S+ribosomal+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+cognate+transfer+RNA+by+the+30S+ribosomal+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000517-Carter1" id="pone.0000517-Carter1"></a>Carter AP, Clemons WM, Brodersen DE, Morgan-Warren RJ, Wimberly BT, Ramakrishnan V (2000) Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics. Nature  407: 340–348.  <ul class="find" data-citedArticleID="957794" data-doi="10.1038/35030019"><li><a href="http://dx.doi.org/10.1038/35030019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+insights+from+the+structure+of+the+30S+ribosomal+subunit+and+its+interactions+with+antibiotics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+insights+from+the+structure+of+the+30S+ribosomal+subunit+and+its+interactions+with+antibiotics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000517-Hobbie1" id="pone.0000517-Hobbie1"></a>Hobbie SN, Bruell C, Kalapala S, Akshay S, Schmidt S, et al.  (2006) A genetic model to investigate drug-target interactions at the ribosomal decoding site. Biochimie  88: 1033–1043.  <ul class="find" data-citedArticleID="957818" data-doi="10.1016/j.biochi.2006.04.008"><li><a href="http://dx.doi.org/10.1016/j.biochi.2006.04.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic+model+to+investigate+drug-target+interactions+at+the+ribosomal+decoding+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic+model+to+investigate+drug-target+interactions+at+the+ribosomal+decoding+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000517-Inoue1" id="pone.0000517-Inoue1"></a>Inoue H, Nojima H, Okayama H (1990) High efficiency transformation of <em>Escherichia coli</em> with plasmids. Gene  96: 23–28.  <ul class="find" data-citedArticleID="957822" data-doi="10.1016/0378-1119(90)90336-p"><li><a href="http://dx.doi.org/10.1016/0378-1119(90)90336-p" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+efficiency+transformation+of+Escherichia+coli+with+plasmids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+efficiency+transformation+of+Escherichia+coli+with+plasmids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000517-Anand2" id="pone.0000517-Anand2"></a>Anand M, Valente L, Carr-Schmid A, Munshi R, Olarewaju O, et al.  (2001) Translation elongation factor 1 functions in the yeast Saccharomyces cerevisiae. Cold Spring Harb Symp Quant Biol  66: 439–48.  <ul class="find" data-citedArticleID="957778" data-doi="10.1101/sqb.2001.66.439"><li><a href="http://dx.doi.org/10.1101/sqb.2001.66.439" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translation+elongation+factor+1+functions+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translation+elongation+factor+1+functions+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000517-CarrSchmid2" id="pone.0000517-CarrSchmid2"></a>Carr-Schmid A, Durko N, Cavallius J, Merrick WC, Kinzy TG (1999) Mutations in a GTP-binding motif of eukaryotic elongation factor 1A reduce both translational fidelity and the requirement for nucleotide exchange. J Biol Chem  274: 30297–30302.  <ul class="find" data-citedArticleID="957792" data-doi="10.1074/jbc.274.42.30297"><li><a href="http://dx.doi.org/10.1074/jbc.274.42.30297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+a+GTP-binding+motif+of+eukaryotic+elongation+factor+1A+reduce+both+translational+fidelity+and+the+requirement+for+nucleotide+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+a+GTP-binding+motif+of+eukaryotic+elongation+factor+1A+reduce+both+translational+fidelity+and+the+requirement+for+nucleotide+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000517-Dinman1" id="pone.0000517-Dinman1"></a>Dinman JD, Wickner RB (1994) Translational maintenance of frame: mutants of <em>Saccharomyces cerevisiae</em> with altered -1 ribosomal frameshifting efficiencies. Genetics  136: 75–86.  <ul class="find" data-citedArticleID="957806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+-1+ribosomal+frameshifting+efficiencies.&amp;auth=&amp;atitle=Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+-1+ribosomal+frameshifting+efficiencies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+-1+ribosomal+frameshifting+efficiencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+-1+ribosomal+frameshifting+efficiencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000517-Ito1" id="pone.0000517-Ito1"></a>Ito H, Fukuda Y, Murata K, Kimura A (1983) Transformation of intact yeast cells treated with alkali cations. J Bacteriol  153: 163–168.  <ul class="find" data-citedArticleID="957824"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transformation+of+intact+yeast+cells+treated+with+alkali+cations.&amp;auth=&amp;atitle=Transformation+of+intact+yeast+cells+treated+with+alkali+cations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transformation+of+intact+yeast+cells+treated+with+alkali+cations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transformation+of+intact+yeast+cells+treated+with+alkali+cations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000517-Harger1" id="pone.0000517-Harger1"></a>Harger JW, Dinman JD (2003) An in vivo dual-luciferase assay system for studying translational recoding in the yeast <em>Saccharomyces cerevisiae</em>. RNA  9: 1019–1024.  <ul class="find" data-citedArticleID="957814" data-doi="10.1261/rna.5930803"><li><a href="http://dx.doi.org/10.1261/rna.5930803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+in+vivo+dual-luciferase+assay+system+for+studying+translational+recoding+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+in+vivo+dual-luciferase+assay+system+for+studying+translational+recoding+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000517-Mumberg1" id="pone.0000517-Mumberg1"></a>Mumberg D, Muller R, Funk M (1995) Yeast vectors for the controlled expression of heterologous proteins in different genetic backgrounds. Gene  156: 119–122.  <ul class="find" data-citedArticleID="957836" data-doi="10.1016/0378-1119(95)00037-7"><li><a href="http://dx.doi.org/10.1016/0378-1119(95)00037-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+vectors+for+the+controlled+expression+of+heterologous+proteins+in+different+genetic+backgrounds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+vectors+for+the+controlled+expression+of+heterologous+proteins+in+different+genetic+backgrounds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000517-Andersen3" id="pone.0000517-Andersen3"></a>Andersen GR, Valente L, Pedersen L, Kinzy TG, Nyborg J (2001) Crystal structures of nucleotide exchange intermediates in the eEF1A- eEF1Balpha complex. Nat Struct Biol  8: 531–534.  <ul class="find" data-citedArticleID="957784" data-doi="10.1038/88598"><li><a href="http://dx.doi.org/10.1038/88598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+nucleotide+exchange+intermediates+in+the+eEF1A-+eEF1Balpha+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+nucleotide+exchange+intermediates+in+the+eEF1A-+eEF1Balpha+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000517-Nissen1" id="pone.0000517-Nissen1"></a>Nissen P, Kjeldgaard M, Thirup S, Polekhina G, Reshetnikova L, et al.  (1995) Crystal structure of the ternary complex of Phe-tRNAPhe, EF-Tu, and a GTP analog. Science  270: 1464–1472.  <ul class="find" data-citedArticleID="957842" data-doi="10.1126/science.270.5241.1464"><li><a href="http://dx.doi.org/10.1126/science.270.5241.1464" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+ternary+complex+of+Phe-tRNAPhe%2C+EF-Tu%2C+and+a+GTP+analog." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+ternary+complex+of+Phe-tRNAPhe%2C+EF-Tu%2C+and+a+GTP+analog.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000517-Tumer1" id="pone.0000517-Tumer1"></a>Tumer NE, Parikh B, Li P, Dinman JD (1998) Pokeweed antiviral protein specifically inhibits Ty<em>1</em> directed +1 ribosomal frameshifting and Ty<em>1</em> retrotransposition in <em>Saccharomyces cerevisiae</em>. J Virol  72: 1036–1042.  <ul class="find" data-citedArticleID="957876"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pokeweed+antiviral+protein+specifically+inhibits+Ty1+directed+%2B1+ribosomal+frameshifting+and+Ty1+retrotransposition+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Pokeweed+antiviral+protein+specifically+inhibits+Ty1+directed+%2B1+ribosomal+frameshifting+and+Ty1+retrotransposition+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pokeweed+antiviral+protein+specifically+inhibits+Ty1+directed+%2B1+ribosomal+frameshifting+and+Ty1+retrotransposition+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pokeweed+antiviral+protein+specifically+inhibits+Ty1+directed+%2B1+ribosomal+frameshifting+and+Ty1+retrotransposition+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000517-Dinman2" id="pone.0000517-Dinman2"></a>Dinman JD, Kinzy TG (1997) Translational misreading: Mutations in translation elongation factor 1α differentially affect programmed ribosomal frameshifting and drug sensitivity. RNA  3: 870–881.  <ul class="find" data-citedArticleID="957808"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Translational+misreading%3A+Mutations+in+translation+elongation+factor+1%26%23x3b1%3B+differentially+affect+programmed+ribosomal+frameshifting+and+drug+sensitivity.&amp;auth=&amp;atitle=Translational+misreading%3A+Mutations+in+translation+elongation+factor+1%26%23x3b1%3B+differentially+affect+programmed+ribosomal+frameshifting+and+drug+sensitivity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translational+misreading%3A+Mutations+in+translation+elongation+factor+1%26%23x3b1%3B+differentially+affect+programmed+ribosomal+frameshifting+and+drug+sensitivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translational+misreading%3A+Mutations+in+translation+elongation+factor+1%26%23x3b1%3B+differentially+affect+programmed+ribosomal+frameshifting+and+drug+sensitivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.XML" value="128833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.PDF" value="406411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g001.PNG_L" value="161162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g001.PNG_M" value="72662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g001.PNG_S" value="3358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g001.TIF" value="359538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g001.PNG_I" value="29714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t001.PNG_L" value="65670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t001.PNG_M" value="69288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t001.PNG_S" value="3888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t001.TIF" value="327094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t001.PNG_I" value="31448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g002.PNG_L" value="33270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g002.PNG_M" value="58453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g002.PNG_S" value="2447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g002.TIF" value="181556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g002.PNG_I" value="24798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g003.PNG_L" value="59121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g003.PNG_M" value="61511"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g003.PNG_S" value="6203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g003.TIF" value="367394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g003.PNG_I" value="83934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t002.PNG_L" value="170787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t002.PNG_M" value="61210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t002.PNG_S" value="3269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t002.TIF" value="891580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t002.PNG_I" value="26559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t003.PNG_L" value="93876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t003.PNG_M" value="84655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t003.PNG_S" value="5159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t003.TIF" value="524590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t003.PNG_I" value="44508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g004.PNG_L" value="188464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g004.PNG_M" value="112780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g004.PNG_S" value="4066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g004.TIF" value="285764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g004.PNG_I" value="42050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g005.PNG_L" value="905384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g005.PNG_M" value="335929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g005.PNG_S" value="8584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g005.TIF" value="1959124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.g005.PNG_I" value="117659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t004.PNG_L" value="365738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t004.PNG_M" value="130449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t004.PNG_S" value="5099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000517.t004.TIF" value="1353144"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000517&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000517" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000517&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000517&volume=&issue=&title=Differentiating between Near- and Non-Cognate Codons in Saccharomyces cerevisiae&author_name=Ewan%20P.%20Plant%2C%20Phuc%20Nguyen%2C%20Jonathan%20R.%20Russ%2C%20Yvette%20R.%20Pittman%2C%20Thai%20Nguyen%2C%20Jack%20T.%20Quesinberry%2C%20Terri%20Goss%20Kinzy%2C%20Jonathan%20D.%20Dinman&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000517" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000517" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000517" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000517&amp;t=Differentiating%20between%20Near-%20and%20Non-Cognate%20Codons%20in%20Saccharomyces%20cerevisiae" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000517&title=Differentiating%20between%20Near-%20and%20Non-Cognate%20Codons%20in%20Saccharomyces%20cerevisiae&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000517&amp;title=Differentiating%20between%20Near-%20and%20Non-Cognate%20Codons%20in%20Saccharomyces%20cerevisiae" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000517" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000517&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Differentiating between Near- and Non-Cognate Codons in Saccharomyces cerevisiae';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000517';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Differentiating%20between%20Near-%20and%20Non-Cognate%20Codons%20in%20Saccharomyces%20cerevisiae http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000517" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000517" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Anticodons%22" title="Search for articles in the subject area:'Anticodons'"><div class="flagText">Anticodons</div></a>
              <div data-categoryid="20289" data-articleid="23872"
                   data-categoryname="Anticodons"
                   class="flagImage" title="Flag 'Anticodons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="23872"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Elongation+factors%22" title="Search for articles in the subject area:'Elongation factors'"><div class="flagText">Elongation factors</div></a>
              <div data-categoryid="43301" data-articleid="23872"
                   data-categoryname="Elongation factors"
                   class="flagImage" title="Flag 'Elongation factors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Luciferase%22" title="Search for articles in the subject area:'Luciferase'"><div class="flagText">Luciferase</div></a>
              <div data-categoryid="31895" data-articleid="23872"
                   data-categoryname="Luciferase"
                   class="flagImage" title="Flag 'Luciferase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Luciferase+assay%22" title="Search for articles in the subject area:'Luciferase assay'"><div class="flagText">Luciferase assay</div></a>
              <div data-categoryid="21481" data-articleid="23872"
                   data-categoryname="Luciferase assay"
                   class="flagImage" title="Flag 'Luciferase assay' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomes%22" title="Search for articles in the subject area:'Ribosomes'"><div class="flagText">Ribosomes</div></a>
              <div data-categoryid="17985" data-articleid="23872"
                   data-categoryname="Ribosomes"
                   class="flagImage" title="Flag 'Ribosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Saccharomyces+cerevisiae%22" title="Search for articles in the subject area:'Saccharomyces cerevisiae'"><div class="flagText">Saccharomyces cerevisiae</div></a>
              <div data-categoryid="32473" data-articleid="23872"
                   data-categoryname="Saccharomyces cerevisiae"
                   class="flagImage" title="Flag 'Saccharomyces cerevisiae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transfer+RNA%22" title="Search for articles in the subject area:'Transfer RNA'"><div class="flagText">Transfer RNA</div></a>
              <div data-categoryid="21239" data-articleid="23872"
                   data-categoryname="Transfer RNA"
                   class="flagImage" title="Flag 'Transfer RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5012'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=803'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1255&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
