

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, Human Rhinovirus C (HRV C)</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001847"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001847" />

    <meta name="citation_title" content="Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, Human Rhinovirus C (HRV C)"/>
    <meta itemprop="name" content="Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, Human Rhinovirus C (HRV C)"/>

      <meta name="citation_author" content="Peter McErlean"/>
            <meta name="citation_author_institution" content="Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Laura A. Shackelton"/>
            <meta name="citation_author_institution" content="Mueller Laboratory, Center for Infectious Disease Dynamics, Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Emily Andrews"/>
            <meta name="citation_author_institution" content="Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Dale R. Webster"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of California San Francisco, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Biological and Medical Informatics Program, University of California San Francisco, San Francisco, California, United States of America"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of California, San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Stephen B. Lambert"/>
            <meta name="citation_author_institution" content="Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Michael D. Nissen"/>
            <meta name="citation_author_institution" content="Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Division of Microbiology, Queensland Health Pathology Service, Royal Brisbane Hospitals Campus, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Department of Paediatrics and Child Health, Royal Children's Hospitals, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Theo P. Sloots"/>
            <meta name="citation_author_institution" content="Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Division of Microbiology, Queensland Health Pathology Service, Royal Brisbane Hospitals Campus, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Department of Paediatrics and Child Health, Royal Children's Hospitals, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Ian M. Mackay"/>
            <meta name="citation_author_institution" content="Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia"/>
            <meta name="citation_author_institution" content="Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001847.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1847"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular characterization of a variant rhinovirus from an outbreak associated with uncommonly high mortality.; citation_author=D Kiang; citation_author=S Yagi; citation_author=KA Kantardjieff; citation_author=EJ Kim; citation_author=JK Louie; citation_journal_title=J Clin Virol; citation_volume=38; citation_number=1; citation_pages=227-237; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rhinovirus and the lower respiratory tract.; citation_author=FG Hayden; citation_journal_title=Reviews in Medical Virology; citation_volume=14; citation_number=2; citation_pages=17-31; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rhinoviruses infect the lower airways.; citation_author=NG Papadopoulos; citation_author=PJ Bates; citation_author=PG Bardin; citation_author=A Papi; citation_author=SH Leir; citation_journal_title=J Infect Dis; citation_volume=181; citation_number=3; citation_pages=1875-1884; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=VP1 sequencing of all human rhinovirus serotypes: Insights into genus phylogeny and susceptibility to antiviral capsid-binding compounds.; citation_author=RM Ledford; citation_author=NR Patel; citation_author=TM Demenczuk; citation_author=A Watanyar; citation_author=T Herbertz; citation_journal_title=Journal of Virology; citation_volume=78; citation_number=4; citation_pages=3663-3674; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Two groups of rhinoviruses revealed by a panel of antiviral compounds present sequence divergence and differential pathogenicity.; citation_author=K Andries; citation_author=B Dewindt; citation_author=J Snoeks; citation_author=L Wouters; citation_author=H Moereels; citation_journal_title=J Virol; citation_volume=64; citation_number=5; citation_pages=1117-1123; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Alignment of capsid protein VP1 sequences of all human rhinovirus prototype strains: conserved motifs and functional domains.; citation_author=P Laine; citation_author=S Blomqvist; citation_author=C Savolainen; citation_author=K Andries; citation_author=T Hovi; citation_journal_title=J Gen Virol; citation_volume=87; citation_number=6; citation_pages=129-138; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic clustering of all 102 human rhinovirus prototype strains: serotype 87 is close to human enterovirus 70.; citation_author=C Savolainen; citation_author=S Blomqvist; citation_author=MN Mulders; citation_author=T Hovi; citation_journal_title=J Gen Virol; citation_volume=83; citation_number=7; citation_pages=333-340; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cell recognition and entry by rhino- and enteroviruses.; citation_author=MG Rossmann; citation_author=J Bella; citation_author=PR Kolatkar; citation_author=Y He; citation_author=E Wimmer; citation_journal_title=Virology; citation_volume=269; citation_number=8; citation_pages=239-247; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Very-low-density lipoprotein receptor fragment shed from HeLa cells inhibits human rhinovirus infection.; citation_author=TC Marlovits; citation_author=C Abrahamsberg; citation_author=D Blaas; citation_journal_title=J Virol; citation_volume=72; citation_number=9; citation_pages=10246-10250; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Members of the low density lipoprotein receptor family mediate cell entry of a minor group common cold virus.; citation_author=F Hofer; citation_author=M Gruenberger; citation_author=H Kowalski; citation_author=H Machat; citation_author=M Huettinger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=10; citation_pages=1839-1842; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The Minor Receptor Group of Human Rhinovirus (HRV) Includes HRV23 and HRV25, but the Presence of a Lysine in the VP1 HI Loop Is Not Sufficient for Receptor Binding.; citation_author=M Vlasak; citation_author=B Roizman; citation_author=M Reithmayer; citation_author=I Goesler; citation_author=P Laine; citation_journal_title=J Virol; citation_volume=79; citation_number=11; citation_pages=7389-7395; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The cellular receptor to human rhinovirus 2 binds around the 5-fold axis and not in the canyon: a structural view.; citation_author=EA Hewat; citation_author=E Neumann; citation_author=JF Conway; citation_author=R Moser; citation_author=B Ronacher; citation_journal_title=EMBO J; citation_volume=19; citation_number=12; citation_pages=6317-6325; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human rhinovirus type 54 infection via heparan sulfate is less efficient and strictly dependent on low endosomal pH.; citation_author=AG Khan; citation_author=J Pichler; citation_author=A Rosemann; citation_author=D Blaas; citation_journal_title=J Virol; citation_volume=81; citation_number=13; citation_pages=4625-4632; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Structure of a human common cold virus and functional relationship to other picornaviruses.; citation_author=MG Rossmann; citation_author=E Arnold; citation_author=JW Erickson; citation_author=EA Frankenberger; citation_author=JP Griffith; citation_journal_title=Nature; citation_volume=317; citation_number=14; citation_pages=145-153; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Structure of human rhinovirus serotype 2 (HRV2).; citation_author=N Verdaguer; citation_author=D Blaas; citation_author=I Fita; citation_journal_title=Journal of Molecular Biology; citation_volume=300; citation_number=15; citation_pages=1179-1194; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comparison of the three-dimensional structure of two human rhinoviruses (HRV2 and HRV14).; citation_author=D Blaas; citation_author=E Kuechler; citation_author=G Vriend; citation_author=E Arnold; citation_author=M Luo; citation_journal_title=Proteins; citation_volume=2; citation_number=16; citation_pages=263-272; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Enterovirus 68 is associated with respiratory illness and shares biological features with both the enteroviruses and the rhinoviruses.; citation_author=MS Oberste; citation_author=K Maher; citation_author=D Schnurr; citation_author=MR Flemister; citation_author=JC Lovchik; citation_journal_title=J Gen Virol; citation_volume=85; citation_number=17; citation_pages=2577-2584; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Interaction of coxsackievirus A21 with its cellular receptor, ICAM-1.; citation_author=C Xiao; citation_author=CM Bator; citation_author=VD Bowman; citation_author=E Rieder; citation_author=Y He; citation_journal_title=J Virol; citation_volume=75; citation_number=18; citation_pages=2444-2451; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Determination of the structure of a decay accelerating factor-binding clinical isolate of echovirus 11 allows mapping of mutants with altered receptor requirements for infection.; citation_author=AD Stuart; citation_author=TA McKee; citation_author=PA Williams; citation_author=C Harley; citation_author=S Shen; citation_journal_title=J Virol; citation_volume=76; citation_number=19; citation_pages=7694-7704; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The structure of echovirus type 12 bound to a two-domain fragment of its cellular attachment protein decay-accelerating factor (CD 55).; citation_author=D Bhella; citation_author=IG Goodfellow; citation_author=P Roversi; citation_author=D Pettigrew; citation_author=Y Chaudhry; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=20; citation_pages=8325-8332; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional structure of poliovirus receptor bound to poliovirus.; citation_author=DM Belnap; citation_author=BM McDermott Jr; citation_author=DJ Filman; citation_author=N Cheng; citation_author=BL Trus; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=21; citation_pages=73-78; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Interaction of coxsackievirus B3 with the full length coxsackievirus-adenovirus receptor.; citation_author=Y He; citation_author=PR Chipman; citation_author=J Howitt; citation_author=CM Bator; citation_author=MA Whitt; citation_journal_title=Nat Struct Biol; citation_volume=8; citation_number=22; citation_pages=874-878; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Subclassification of picornaviruses.; citation_author=L Rosen; citation_journal_title=Bacteriological Reviews; citation_volume=29; citation_number=23; citation_pages=173-184; citation_date=1965; " />
      <meta name="citation_reference" content="citation_title=Picornaviridae: The viruses and their replication.; citation_author=VR Racaniello; citation_number=24; citation_pages=685-722; citation_date=2001; citation_publisher=Lippincott-Raven; " />
      <meta name="citation_reference" content="citation_title=New complete genome sequences of human rhinoviruses shed light on their phylogeny and genomic features.; citation_author=C Tapparel; citation_author=T Junier; citation_author=D Gerlach; citation_author=S Cordey; citation_author=S van Belle; citation_journal_title=BMC Genomics; citation_volume=10; citation_number=25; citation_pages=224; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-wide diversity and selective pressure in the human rhinovirus.; citation_author=A Kistler; citation_author=DR Webster; citation_author=S Rouskin; citation_author=V Magrini; citation_author=J Credle; citation_journal_title=Virol J; citation_volume=4; citation_number=26; citation_pages=40; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Recombination in circulating human enterovirus B: independent evolution of structural and non-structural genome regions.; citation_author=AN Lukashev; citation_author=VA Lashkevich; citation_author=OE Ivanova; citation_author=GA Koroleva; citation_author=AE Hinkkanen; citation_journal_title=J Gen Virol; citation_volume=86; citation_number=27; citation_pages=3281-3290; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Frequency and dynamics of recombination within different species of human enteroviruses.; citation_author=P Simmonds; citation_author=J Welch; citation_journal_title=Journal of Virology; citation_volume=80; citation_number=28; citation_pages=483-493; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Polymerase chain reaction and sequencing for typing rhinovirus RNA.; citation_author=J Mori; citation_author=JP Clewley; citation_journal_title=J Med Virol; citation_volume=44; citation_number=29; citation_pages=323-329; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A collaborative report: Rhinoviruses - extension of the numbering system from 89 to 100.; citation_author=VV Hamparian; citation_author=RJ Colonno; citation_author=MK Cooney; citation_author=EC Dick; citation_author=JM Gwaltney Jr; citation_journal_title=Virology; citation_volume=159; citation_number=30; citation_pages=191-192; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Characterisation of a newly identified human rhinovirus, HRV-QPM, discovered in infants with bronchiolitis.; citation_author=P McErlean; citation_author=LA Shackleton; citation_author=SB Lambert; citation_author=MD Nissen; citation_author=TP Sloots; citation_journal_title=J Clin Virol; citation_volume=39; citation_number=31; citation_pages=67-75; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Pan-viral screening of respiratory tract infections in adults with and without asthma reveals unexpected human coronavirus and human rhinovirus diversity.; citation_author=A Kistler; citation_author=PC Avila; citation_author=S Rouskin; citation_author=D Wang; citation_author=T Ward; citation_journal_title=J Infect Dis; citation_volume=196; citation_number=32; citation_pages=817-825; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Clinical and molecular epidemiology of human bocavirus in respiratory and fecal samples from children in Hong Kong.; citation_author=SKP Lau; citation_author=CCY Yip; citation_author=T-L Que; citation_author=RA Lee; citation_author=RKH Au-Yeung; citation_journal_title=J Infect Dis; citation_volume=196; citation_number=33; citation_pages=986-993; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Frequent detection of human rhinoviruses, paramyxoviruses, coronaviruses, and bocavirus during acute respiratory tract infections.; citation_author=KE Arden; citation_author=P McErlean; citation_author=MD Nissen; citation_author=TP Sloots; citation_author=IM Mackay; citation_journal_title=J Med Virol; citation_volume=78; citation_number=34; citation_pages=1232-1240; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence analysis of human rhinoviruses in the RNA-dependent RNA polymerase coding region reveals within-species variation.; citation_author=C Savolainen; citation_author=P Laine; citation_author=MN Mulders; citation_author=T Hovi; citation_journal_title=J Gen Virol; citation_volume=85; citation_number=35; citation_pages=2271-2277; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complete genomic sequencing shows that polioviruses and members of human enterovirus species C are closely related in the noncapsid coding region.; citation_author=B Brown; citation_author=MS Oberste; citation_author=K Maher; citation_author=MA Pallansch; citation_journal_title=J Virol; citation_volume=77; citation_number=36; citation_pages=8973-8984; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Full-length human iummunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination.; citation_author=KS Lole; citation_author=RC Bollinger; citation_author=RS Paranjape; citation_author=D Gadkari; citation_author=SS Kulkarni; citation_journal_title=Journal of Virology; citation_volume=73; citation_number=37; citation_pages=152-160; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Alignments and comparative profiles of picornavirus genera.; citation_author=AC Palmenberg; citation_author=J-Y Sgro; citation_volume=42; citation_number=38; citation_pages=149-155; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular relationships between 21 human rhinovirus serotypes.; citation_author=C Horsnell; citation_author=RE Gama; citation_author=PJ Hughes; citation_author=G Stanway; citation_journal_title=J Gen Virol; citation_volume=76; citation_number=39; citation_pages=2549-2555; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Structural studies of two rhinovirus serotypes complexed with fragments of their cellular receptor.; citation_author=PR Kolatkar; citation_author=J Bella; citation_author=NH Olson; citation_author=CM Bator; citation_author=TS Baker; citation_journal_title=The EMBO Journal; citation_volume=18; citation_number=40; citation_pages=6249-6259; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Presence of the newly discovered human polyomaviruses KI and WU in Australian patients with acute respiratory tract infection.; citation_author=S Bialasiewicz; citation_author=DM Whiley; citation_author=SB Lambert; citation_author=KC Jacob; citation_author=C Bletchly; citation_journal_title=J Clin Virol; citation_volume=41; citation_number=41; citation_pages=63-68; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Family Picornaviridae.; citation_author=G Stanway; citation_author=F Brown; citation_author=P Christian; citation_author=T Hovi; citation_author=T Hyypiä; citation_number=42; citation_pages=757-778; citation_date=2005; citation_publisher=Elsevier Academic Press; " />
      <meta name="citation_reference" content="citation_title=Insights into the genetic basis for natural phenotypic resistance of human rhinoviruses to pleconaril.; citation_author=RM Ledford; citation_author=MS Collett; citation_author=DC Pevear; citation_journal_title=Antiviral Res; citation_volume=68; citation_number=43; citation_pages=135-138; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cellular receptors of picornaviruses.; citation_author=E Rieder; citation_author=E Wimmer; citation_number=44; citation_pages=61-70; citation_date=2002; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=The BC loop in poliovirus coat protein VP1: an ideal acceptor site for major insertions.; citation_author=HJ Stirk; citation_author=JM Thornton; citation_journal_title=Protein Eng; citation_volume=7; citation_number=45; citation_pages=47-56; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A cellular receptor of human rhinovirus type 2, the very-low-density lipoprotein receptor, binds to two neighboring proteins of the viral capsid.; citation_author=E Neumann; citation_author=R Moser; citation_author=L Snyers; citation_author=D Blaas; citation_author=EA Hewat; citation_journal_title=J Virol; citation_volume=77; citation_number=46; citation_pages=8504-8511; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sequence and structure of human rhinoviruses reveal the basis of receptor discrimination.; citation_author=M Vlasak; citation_author=S Blomqvist; citation_author=T Hovi; citation_author=E Hewat; citation_author=D Blaas; citation_journal_title=J Virol; citation_volume=77; citation_number=47; citation_pages=6923-6930; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mouse models of rhinovirus-induced disease and exacerbation of allergic airway inflammation.; citation_author=NW Bartlett; citation_author=RP Walton; citation_author=MR Edwards; citation_author=J Aniscenko; citation_author=G Caramori; citation_journal_title=Nature Medicine; citation_volume=14; citation_number=48; citation_pages=199-204; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The structure of human rhinovirus 16.; citation_author=MA Oliveira; citation_author=R Zhao; citation_author=W-M Lee; citation_author=MJ Kremer; citation_author=I Minor; citation_journal_title=Structure; citation_volume=1; citation_number=49; citation_pages=51-68; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Amplification of rhinovirus specific nucleic acids from clinical samples using the polymerase chain reaction.; citation_author=RE Gama; citation_author=PR Horsnell; citation_author=PJ Hughes; citation_author=C North; citation_author=CB Bruce; citation_journal_title=J Med Virol; citation_volume=28; citation_number=50; citation_pages=73-77; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A diverse group of previously unrecognized human rhinoviruses are common causes of respiratory illness in infants.; citation_author=W-M Lee; citation_author=C Kiesner; citation_author=T Pappas; citation_author=I Lee; citation_author=K Grindle; citation_journal_title=PLoS One; citation_volume=e966; citation_number=51; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MassTag Polymerase-Chain-Reaction Detection of Respiratory Pathogens, Including a New Rhinovirus Genotype, That Caused Influenza-Like Illness in New York State during 2004–2005.; citation_author=D Lamson; citation_author=N Renwick; citation_author=V Kapoor; citation_author=Z Liu; citation_author=G Palacios; citation_journal_title=J Infect Dis; citation_volume=194; citation_number=52; citation_pages=1398-1402; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Role of respiratory viruses in acute upper and lower respiratory tract illness in the first year of life.; citation_author=MMH Kusel; citation_author=NH de Klerk; citation_author=PG Holt; citation_author=T Kebadze; citation_author=SL Johnston; citation_journal_title=Pediatr Infect Dis J; citation_volume=25; citation_number=53; citation_pages=680-686; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cleavage site analysis in picornaviral polyproteins: Discovering cellular targets by neural networks.; citation_author=N Blom; citation_author=J Hansen; citation_author=D Blaas; citation_author=S Brunak; citation_journal_title=Prot Sci; citation_volume=5; citation_number=54; citation_pages=2203-2216; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=UCSF chimera-A visualization system for exploratory research and analysis.; citation_author=EF Pettersen; citation_author=TD Goddard; citation_author=CC Huang; citation_author=GS Couch; citation_author=DM Greenblatt; citation_journal_title=Journal of computational chemistry; citation_volume=25; citation_number=55; citation_pages=1605-1612; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bioedit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT.; citation_author=TA Hall; citation_journal_title=Nucleic Acids Symp Ser; citation_volume=41; citation_number=56; citation_pages=98; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for molecular evolutionary genetic analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Briefings in bioinformatics; citation_volume=5; citation_number=57; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modelling.; citation_author=N Guex; citation_author=MC Peitsch; citation_journal_title=Electrophoresis; citation_volume=18; citation_number=58; citation_pages=2714-2723; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=VIPERdb: a relational database for structural virology.; citation_author=CM Shepherd; citation_author=IA Borelli; citation_author=G Lander; citation_author=P Natarajan; citation_author=V Siddavanahalli; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=59; citation_pages=D386-D389; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Human metapneumovirus, Australia, 2001–2004.; citation_author=TP Sloots; citation_author=IM Mackay; citation_author=S Bialasiewicz; citation_author=KC Jacob; citation_author=E McQueen; citation_journal_title=Emerg Infect Dis; citation_volume=12; citation_number=60; citation_pages=1263-1266; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=A multicenter, randomized, double-blind, controlled trial of nebulized epinephrine in infants with acute bronchiolitis.; citation_author=C Wainwright; citation_author=L Altamirano; citation_author=M Cheney; citation_author=J Cheney; citation_author=S Barber; citation_journal_title=The New England journal of medicine; citation_volume=349; citation_number=61; citation_pages=27-35; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001847" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, Human Rhinovirus C (HRV C)"/>
    <meta name="twitter:description" content="BackgroundHuman rhinoviruses (HRVs) are the most frequently detected pathogens in acute respiratory tract infections (ARTIs) and yet little is known about the prevalence, recurrence, structure and clinical impact of individual members. During 2007, the complete coding sequences of six previously unknown and highly divergent HRV strains were reported. To catalogue the molecular and clinical features distinguishing the divergent HRV strains, we undertook, for the first time, in silico analyses of all available polyprotein sequences and performed retrospective reviews of the medical records of cases in which variants of the prototype strain, HRV-QPM, had been detected.Methodology/Principle FindingsGenomic analyses revealed that the six divergent strains, residing within a clade we previously called HRV A2, had the shortest polyprotein of all picornaviruses investigated. Structure-based amino acid alignments identified conserved motifs shared among members of the genus Rhinovirus as well as substantive deletions and insertions unique to the divergent strains. Deletions mostly affected regions encoding proteins traditionally involved in antigenicity and serving as HRV and HEV receptor footprints. Because the HRV A2 strains cannot yet be cultured, we created homology models of predicted HRV-QPM structural proteins. In silico comparisons confirmed that HRV-QPM was most closely related to the major group HRVs. HRV-QPM was most frequently detected in infants with expiratory wheezing or persistent cough who had been admitted to hospital and required supplemental oxygen. It was the only virus detected in 65% of positive individuals. These observations contributed to an objective clinical impact ranging from mild to severe.ConclusionsThe divergent strains did not meet classification requirements for any existing species of the genus Rhinovirus or Enterovirus. HRV A2 strains should be partitioned into at least one new species, putatively called Human rhinovirus C, populated by members detected with high frequency, from individuals with respiratory symptoms requiring hospital admission."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001847.g006"/>

  <meta property="og:title" content="Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, Human Rhinovirus C (HRV C)" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001847" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4437'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7330'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5177&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001847">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001847" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001847&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, <i>Human Rhinovirus C</i> (HRV C)
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter McErlean, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia, 
                  Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura A. Shackelton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Mueller Laboratory, Center for Infectious Disease Dynamics, Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Emily Andrews, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia, 
                  Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dale R. Webster, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Biochemistry and Biophysics, University of California San Francisco, San Francisco, California, United States of America, 
                  Biological and Medical Informatics Program, University of California San Francisco, San Francisco, California, United States of America, 
                  Howard Hughes Medical Institute, University of California, San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen B. Lambert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia, 
                  Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael D. Nissen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia, 
                  Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia, 
                  Division of Microbiology, Queensland Health Pathology Service, Royal Brisbane Hospitals Campus, Brisbane, Queensland, Australia, 
                  Department of Paediatrics and Child Health, Royal Children's Hospitals, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Theo P. Sloots, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia, 
                  Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia, 
                  Division of Microbiology, Queensland Health Pathology Service, Royal Brisbane Hospitals Campus, Brisbane, Queensland, Australia, 
                  Department of Paediatrics and Child Health, Royal Children's Hospitals, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ian M. Mackay
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:ian.mackay@uq.edu.au">ian.mackay@uq.edu.au</a></p>

                <p>Affiliations:
                  Queensland Paediatric Infectious Diseases Laboratory, Sir Albert Sakzewski Virus Research Centre, Royal Children's Hospital, Brisbane, Queensland, Australia, 
                  Clinical and Medical Virology Centre, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001847</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001847" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001847" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001847" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001847" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001847-g001" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-g002" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-g003" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-g004" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-g005" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-g006" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-t001" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-t002" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001847-t003" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001847">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001847" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Human rhinoviruses (HRVs) are the most frequently detected pathogens in acute respiratory tract infections (ARTIs) and yet little is known about the prevalence, recurrence, structure and clinical impact of individual members. During 2007, the complete coding sequences of six previously unknown and highly divergent HRV strains were reported. To catalogue the molecular and clinical features distinguishing the divergent HRV strains, we undertook, for the first time, <em>in sili</em>co analyses of all available polyprotein sequences and performed retrospective reviews of the medical records of cases in which variants of the prototype strain, HRV-QPM, had been detected.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Genomic analyses revealed that the six divergent strains, residing within a clade we previously called HRV A2, had the shortest polyprotein of all picornaviruses investigated. Structure-based amino acid alignments identified conserved motifs shared among members of the genus <em>Rhinovirus</em> as well as substantive deletions and insertions unique to the divergent strains. Deletions mostly affected regions encoding proteins traditionally involved in antigenicity and serving as HRV and HEV receptor footprints. Because the HRV A2 strains cannot yet be cultured, we created homology models of predicted HRV-QPM structural proteins. <em>In silico</em> comparisons confirmed that HRV-QPM was most closely related to the major group HRVs. HRV-QPM was most frequently detected in infants with expiratory wheezing or persistent cough who had been admitted to hospital and required supplemental oxygen. It was the only virus detected in 65% of positive individuals. These observations contributed to an objective clinical impact ranging from mild to severe.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The divergent strains did not meet classification requirements for any existing species of the genus <em>Rhinovirus</em> or <em>Enterovirus</em>. HRV A2 strains should be partitioned into at least one new species, putatively called <em>Human rhinovirus C</em>, populated by members detected with high frequency, from individuals with respiratory symptoms requiring hospital admission.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>McErlean P, Shackelton LA, Andrews E, Webster DR, Lambert SB, et al.  (2008) Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, <em>Human Rhinovirus C</em> (HRV C). PLoS ONE 3(4):
          e1847.
            doi:10.1371/journal.pone.0001847</p><p><strong>Editor: </strong>Dong-Yan Jin, University of Hong Kong, China</p><p><strong>Received:</strong> December 4, 2007; <strong>Accepted:</strong> February 21, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 McErlean et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NHMRC Project grant 455905 and the Royal Children's Hospital Foundation grants 912-011 and 922-202 sponsored by the Woolworth's Fresh Futures Programme. PM was supported by a SASVRC PhD Scholarship.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Human rhinoviruses (HRVs) are the most frequently detected pathogens in acute respiratory tract infections (ARTIs). HRVs have been associated with lower respiratory tract (LRT) illness and more serious clinical outcomes within pediatric and other vulnerable populations <a href="#pone.0001847-Kiang1">[1]</a>. Despite this, HRV strains continue to be commonly defined, <em>en masse</em>, by their most prolific and currently most well-defined role in causing the ‘common cold’ <a href="#pone.0001847-Hayden1">[2]</a>, <a href="#pone.0001847-Papadopoulos1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Classified within the family <em>Picornaviridae</em>, the genus <em>Rhinovirus</em> consists of 100 serotyped strains divided into two species, <em>Human rhinovirus A</em> (HRV A; n = 75) and <em>Human rhinovirus B</em> (HRV B; n = 25) <a href="#pone.0001847-Ledford1">[4]</a>. Species classification was initially based on the susceptibility of strains to capsid binding antivirals <a href="#pone.0001847-Andries1">[5]</a>, <a href="#pone.0001847-Laine1">[6]</a> and subsequently confirmed by phylogenetic studies <a href="#pone.0001847-Ledford1">[4]</a>, <a href="#pone.0001847-Savolainen1">[7]</a>. HRVs are also subdivided by receptor usage; major group HRVs use an intercellular adhesion molecule (ICAM-1; n = 88 strains), which interacts within a depression of the viral surface known as the canyon <a href="#pone.0001847-Rossmann1">[8]</a>. Minor group HRVs (n = 12) receive a molecule from the low density lipoprotein receptor family (LDLR) at protrusions along the 5-fold axis of the capsid surface <a href="#pone.0001847-Marlovits1">[9]</a>–<a href="#pone.0001847-Hewat1">[12]</a>. Recently, heparan sulphate was identified as a pH dependent, low efficiency receptor for HRV-54 <a href="#pone.0001847-Khan1">[13]</a>. A relationship between receptor usage and species identity is only evident among minor group HRVs; all are HRV As. Major group HRVs reside in both species. Studies have identified four neutralizing immunogenic sites on the surface of a major group HRV (HRV-14; NImIA, NImIB, NImII and NImIII)<a href="#pone.0001847-Rossmann2">[14]</a> and three sites on a minor group HRV (HRV-2; site A, B and C) <a href="#pone.0001847-Verdaguer1">[15]</a>, with some overlap <a href="#pone.0001847-Blaas1">[16]</a>. These sites consist of discontinuous amino acid sequence within VP1, VP2 and VP3.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The human enteroviruses (HEVs) are the closest genetic relatives of the HRVs but they belong to the genus <em>Enterovirus</em>. HEV strains are classified into four species (A–D) and some have been implicated in ARTI <a href="#pone.0001847-Oberste1">[17]</a>. Similar to the major group HRVs, some HEVs have been shown to use ICAM-1 as their primary cellular receptor <a href="#pone.0001847-Xiao1">[18]</a>. Other host molecules employed by HEVs include the decay accelerating factor (DAF) <a href="#pone.0001847-Stuart1">[19]</a>, <a href="#pone.0001847-Bhella1">[20]</a>, poliovirus receptor (PVR) <a href="#pone.0001847-Belnap1">[21]</a> and coxsackie-adenovirus receptor (CAR) <a href="#pone.0001847-He1">[22]</a>. Under a proposal currently before the International Committee on Taxonomy of Viruses (ICTV), HRV and HEV strains will be combined into the genus <em>Enterovirus</em>, at the expense of <em>Rhinovirus</em>, an issue of contention for decades <a href="#pone.0001847-Rosen1">[23]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Picornavirus genomes typically consist of a 7.2 to 7.5 kb positive-sense, single-stranded RNA molecule located within a non-enveloped icosahedral capsid. The genomes contain a single open reading frame flanked by 5′and 3′untranslated regions (UTR). The polyprotein is first cleaved into three precursor polyproteins, P1, P2 and P3 (<a href="#pone-0001847-g001">Figure 1</a>). P1 is further cleaved into the structural proteins VP4, VP2, VP3 and VP1. Union of a single copy of each structural protein constitutes a viral protomer; the HRV capsid consists of 60 protomers arranged in a T = 1, pseudo T = 3 (i.e. T = p3) conformation. P2 and P3 are cleaved into seven non-structural proteins including two proteases (2A<sup>pro</sup> and 3C<sup>pro</sup>) and an RNA-dependant RNA polymerase (3D<sup>pol</sup>) <a href="#pone.0001847-Racaniello1">[24]</a>.</p>
<div class="figure" id="pone-0001847-g001"><div class="img"><a name="pone-0001847-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Characterization of HRV A2 genomes.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Scale representation of the coding region from prototype HRV A2 virus, HRV-QPM. Predicted protein products from the structural (VP4 to VP1) and non-structural (PRO-protease; POL-polymerase) regions are indicated. Regions are shaded alternatively for convenience and numbering indicates positions of predicted protease cleavage according to the start of the HRV-QPM polyprotein. Dashed lines indicate regions involved in species classifications. (B) HRV A2 amino acid similarity to HRV A (n = 34) and B (n = 13), or HEV (n = 71) strains as determined by SimPlot© analysis. Plots indicate the percentage similarity of a 50% consensus sequence from each species' polyprotein compared to the HRV A2 strains. Average similarity across the polyprotein is indicated by dotted line. Averaged values for the complete coding, P1 and 2C+3CD regions are indicated within the inset of each panel.</p>
<span>doi:10.1371/journal.pone.0001847.g001</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Recent studies have generated 27 additional HRV A and 12 additional HRV B genomes <a href="#pone.0001847-Tapparel1">[25]</a>, <a href="#pone.0001847-Kistler1">[26]</a> also resulting in the proposal that individual HRV genomes are under purifying selective pressure and that significant intra-strain variation occurs only at the antigenic sites <a href="#pone.0001847-Kistler1">[26]</a>. This is in stark contrast to the HEVs for which recombination in the non-structural regions is a significant contributor to viral diversification <a href="#pone.0001847-Lukashev1">[27]</a>, <a href="#pone.0001847-Simmonds1">[28]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Untypeable rhinoviruses have been described using PCR-based tools, since 1994 <a href="#pone.0001847-Mori1">[29]</a> but it was only in 2007 that the complete coding sequences of a number of novel and apparently divergent HRV strains were reported, permitting a for identification of the first new strains in two decades <a href="#pone.0001847-Hamparian1">[30]</a>. The initial strain described, HRV-QPM, was identified from children with presenting symptoms often suggesting bronchiolitis <a href="#pone.0001847-McErlean1">[31]</a>. Subsequently, ‘HRV-Xs’ were found in an adult asthma study in the United States <a href="#pone.0001847-Kistler2">[32]</a> and ‘HRV-Cs’ were described from a study of pediatric infection by human bocavirus in Hong Kong <a href="#pone.0001847-Lau1">[33]</a>. Some notable features associated with these six putative viruses include the reliance upon molecular methods for their detection, their apparent endeminicity, their failure to produce cytopathic effects in culture, their frequent detection in subjects with expiratory wheezing and their limited sequence identity with existing HRV strains but high shared sequence identity. Studies employing subgenomic sequences proposed that there were many of these putative viruses occupying a distinct position within the genus <em>Rhinovirus</em>, which we collectively called HRV A2 <a href="#pone.0001847-Arden1">[34]</a>. It has been proposed that these strains may constitute a new picornavirus species <a href="#pone.0001847-Lau1">[33]</a>. Data have not yet been presented which appropriately address classification using current ICTV criteria and the scope of the clinical impact of these divergent strains has not yet been studied in detail. Because it appears that the members of this clade are frequently associated with LRT illness we sought to better characterise them in the hope of finding features that might prove useful in predicting the outcome of HRV A2 infection.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Intensive genomic analysis and computer-based modelling resulted in comprehensive characterisation of all HRV A2 polyprotein sequences and, by superimposing the sites of known receptor contacts, we were able to propose a more robust taxonomic placement for these newly identified viruses (NIVs). Detailed medical chart reviews on HRV-QPM-positive (prototype HRV A2) individuals were augmented by a severity scoring system which quantified the outcome of infection by this apparently uncharacteristically pathogenic clade of HRV strains.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Genomic analyses</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We undertook a comparative analysis of all HRV A2 complete coding regions (n = 6; HRV-QPM, -X1, H2, -C024, -C025 and C026) against all HRV A (n = 34), HRV B (n = 13), most HEV sequences (<em>n</em> = 71, including species A–D) and two non-human enteroviruses (simian, SEV and porcine, PEV).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>HRV A2 sequences had a higher G+C content (average, 42.4%) than all other HRVs (average 38.7%) or HEV Ds and a shorter coding region (average, 2144 aa) than any other picornavirus investigated. The final residue of the HRV A2 genomes was an isoleucine, distinguishing them from all other picornavirus genomes we investigated, which ended at the preceding phenylalanine; the biological implications of this feature are unknown.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We identified 10 protease sites (data not shown) dividing the HRV A2 polyproteins into four structural and seven non-structural proteins (<a href="#pone-0001847-g001">Figure 1A</a>). However, unlike previous studies that relied on amino acid alignments to determine cleavage sites <a href="#pone.0001847-Lau1">[33]</a>, we obtained additional evidence that these sites were recognised by 2A<sup>pro</sup> and 3C<sup>pro</sup> by using a neural network protease cleavage-site prediction tool. A single cleavage by 2A<sup>pro</sup> was predicted between an A or L and a G residue at the VP1/2A junction. The remaining cleavages, predominantly occurring between Q and G residues, were mediated by 3C<sup>pro</sup>. Three motifs crucial for RNA polymerase binding (YGDD, TFLKR and SIRWT) were identified in all HRV A2 sequences <a href="#pone.0001847-Savolainen2">[35]</a>, <a href="#pone.0001847-Brown1">[36]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Using SimPlot© <a href="#pone.0001847-Lole1">[37]</a>, the HRV A2 sequences were found to be most similar to those from HRV A strains (54% identity) whereas identity with HEV species ranged from 46%–49% (<a href="#pone-0001847-g001">Figure 1B</a>). Dips in identity occurred across all regions but only the structural region was investigated further because its components contain antigenic sites and receptor contact residues, or ‘footprints’; the latter being significant domains for defining the major and minor group HRVs <a href="#pone.0001847-Palmenberg1">[38]</a> and other picornaviruses.</p>


<h4>Structural homology modelling</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Although identifiable using amino acid alignments<a href="#pone.0001847-Lau1">[33]</a>, the scope and impact of the many HRV A2 sequence deletions could not be fully qualified or quantified in this form. However, the inability to isolate the HRV A2 strains <em>in vitro</em> will hinder crystallography studies seeking to identify structural features, so we created structure-based alignments of the VP1–4 sequences from the prototype HRV A2 strain, HRV-QPM and used them to predict the presence of α-helices and β-sheets. These were identified in the eight, anti-parallel β-sheet ‘jellyroll’ conformation common among picornavirus proteins VP1, VP2 (including the EF ‘puff’) and VP3 <a href="#pone.0001847-Horsnell1">[39]</a> (<a href="#pone-0001847-g002">Figure 2</a>). Deletions in the VP1 protein imparted the most dramatic changes, reducing the protrusion of the BC, DE and HI loops compared with the other HRVs and introducing an additional C-terminal sheet-loop-sheet structure (arrow, <a href="#pone-0001847-g002">Figure 2B</a>). Similar deletions were noted in all HRV A2 strains but only HRV-C026 was predicted to encode a similar, C-terminal structure in VP1. We next assembled the HRV-QPM structural proteins into a protomer by associating them with the closest sequence match (HRV-16) and then replicated this structure <em>in silico</em> into a viral pentamer. RMSD values were derived from the VP1, VP2 and VP3 structures of all HRVs with crystallography-derived data available (HRV-1A, -2, -3, -14 and -16) and compared to the predicted HRV-QPM structure. HRV-QPM and HRV-16 shared the highest conformational similarity (0.145, 0.142 and 0.147 Å for the VP1, VP2 and VP3 structures respectively), reflecting the use of HRV-16 in predicting the structure HRV-QPM. The poorest agreements were between HRV-1A and HRV-3 in VP1 (0.793 Å) and VP3 (0.740 Å) and between HRV-2 and HRV-14 in VP2 (0.697 Å). At the ICAM-1 footprint, HRV-QPM values ranged from 0.064–0.106 Å and 0.709–0.779 Å in comparison to the same regions on HRV-16 and -14, respectively. Values could not be determined for HRV-2 because of the impact of deletions. We also examined the conformational agreement between an HEV footprint e.g. the CAR site employed by CV-B3, and the predicted HRV-QPM structure. A space filling model of the pentamer was next used to display SimPlot data in order to visualise regions of sequence conservation and diversification with a focus on the major (ICAM-1; <a href="#pone-0001847-g003">Figure 3A</a>) and minor (VLDL-R; <a href="#pone-0001847-g003">Figure 3C</a>) group domains. Overall, greater sequence identity was apparent between HRV-2 than HRV-14 reflecting the genomic similarity to HRV A strains, however within the domains defining HRV groups, HRV-QPM was most similar in its antigenic sites to the major group representative, HRV-14. Most sequence diversity was found in the VP2 protein while the most conserved protein in these comparisons was VP3.</p>
<div class="figure" id="pone-0001847-g002"><div class="img"><a name="pone-0001847-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Comparison of predicted protomers.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) A simplified depiction of two protomers in opposition on a viral capsid (shaded areas, adapted from <a href="#pone.0001847-Lole1">[37]</a>). As a guide for visualising loop length, a dashed and a dotted line are spaced equidistantly and represent proximal and distal positions from the virion core, respectively. (B) Ribbon depiction of two opposing viral protomers from HRV-QPM, HRV-2 (minor group) and HRV-14 (major group). HRV-QPM proteins were predicted by <em>in silico</em> matching to the empirically derived HRV-16 and HRV-14 structure (<a href="#s4">materials and methods</a>). Protomers consist of one copy each of VP1, VP2, VP3 and VP4. β-sheets are depicted as flat arrows and α-helices as coiled ribbons. The formation of VP1-VP3 into eight anti-parallel β-sheets is indicative of the ‘jellyroll’ conformation typical in picornaviruses. Major differences in the predicted HRV-QPM VP1 include the shortened external loops between β-sheets (asterisk) and an additional C-terminal sheet-loop-sheet formation (arrow indicates the same location on all protomers for comparison).</p>
<span>doi:10.1371/journal.pone.0001847.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To study these regions in greater detail, we mapped known HRV antigenic sites and receptor footprints onto ribbon depictions of pentamers derived from empirically determined structural data (major group, HRV-14, <a href="#pone-0001847-g003">Figure 3B</a>; minor group, HRV-2, <a href="#pone-0001847-g003">Figure 3D</a>) and compared their locations and structures to the predicted HRV-QPM pentamer. HRV-QPM and HRV-14 appeared to share a number of similar structures in the region of the ICAM-1 footprint <a href="#pone.0001847-Laine1">[6]</a>, <a href="#pone.0001847-Lau1">[33]</a>, <a href="#pone.0001847-Kolatkar1">[40]</a> (<a href="#pone-0001847-g003">Figure 3B</a>). The noteworthy structural disparities were two small helices present in the HRV-QPM sequences; one in the VP1 and the other in the VP3 ICAM-1 footprint (<a href="#pone-0001847-g003">Figure 3B</a>; VP1 and VP3, ICAM-1 arrows). Similar sequences were infrequent among the other HRV strains we examined. The HRV-QPM NImIA and IB sites appeared to differ structurally due to the shortening of the BC and DE loops in VP1 while the NImII and NImIII sites appeared structurally similar.</p>
<div class="figure" id="pone-0001847-g003"><div class="img"><a name="pone-0001847-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Predicted HRV-QPM pentamers compared to representative major (HRV-14) and minor (HRV-2) group HRV pentamers derived from crystallography data.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>(A) HRV-QPM versus HRV-14 SimPlot data projected onto a space filling depiction of the predicted HRV-QPM pentamer. Shading represents the amino acid identity (26–69%). The yellow dashed triangle represents a single icosahedral asymmetric unit (T = p3 conformation) composed of VP1 and VP2 from the same protomer and VP3 for an adjacent protomer. The major group domains of interest are divided between two asymmetric units for ease of viewing. Receptor (white) and antigenic (red) sites are shown in outline. (B) Top view ribbon depiction of a major group HRV pentamer (HRV-14; gray) with labelled antigenic neutralisation sites (NImIA-III, green) and combined HRV A (HRV-16) and B (HRV-14)ICAM-1 receptor footprints (red) <a href="#pone.0001847-Laine1">[6]</a>, <a href="#pone.0001847-Lole1">[37]</a>. Magnified areas of interest (boxed) highlight computer-based predictive comparisons to the equivalent HRV-QPM (orange) structures of interest. Arrows indicate structures and corresponding sequences of interest (refer to text). (C) HRV-QPM versus HRV-2 SimPlot data projected onto the HRV-QPM pentamer. The domains of interest are mostly shown within a single asymmetric unit. (D) A minor group pentamer (HRV-2; gray) including antigenic sites (sites A–C, green) and VLDL-R footprint (red) <a href="#pone.0001847-Hofer1">[10]</a>. Attachment of the VLDL-R involves adjacent VP1 molecules. Magnified VP1 area represents one half of a VLDL-R footprint <a href="#pone.0001847-Stanway1">[42]</a>. Amino acid substitutions (arrowed) contributed to the differences between minor group sites B and C.</p>
<span>doi:10.1371/journal.pone.0001847.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Comparison to HRV-2 (<a href="#pone-0001847-g003">Figure 3D</a>) also revealed the impact of VP1 deletions. The resulting loops were shorter, which ablated any analogue of antigenic site A and may hamper binding of the VLDL-R molecule, the only LDLR family member with a well-defined footprint. The area in VP1 forming site B was altered due to the presence of more hydrophilic amino acids at the equivalent positions in HRV-QPM (<a href="#pone-0001847-g003">Figure 3D</a>; VP1 site B, arrow). Deletions were apparent but did not confer obvious structural changes to the VP2 portion of site B. Furthermore, in antigenic site C (VP2), a single deletion and a hydrophobic to hydrophilic amino acid change contributed to a protrusion in HRV-QPM compared to the same region in HRV-2.</p>


<h4>Analysis of HRV-QPM in the context of HEV receptors</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Given the comparative differences we observed when locating and comparing the HRV domains, we also investigated the HRV-QPM capsid for the presence of footprints reportedly used by CV-A21 (which employs ICAM-1 as a receptor), Echovirus (EV)-11 (DAF), Poliovirus-1 (PVR) and CV-B3 (CAR).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The deletions in the VP1 BC loop had a less obvious impact on HEV receptor footprints which were mostly located on the β-sheets rather than the loop (<a href="#pone-0001847-g004">Figure 4</a>). The EF loop in HRV-QPM VP1 provided the greatest observable identity with any HEV footprint while the C-terminal sheet-loop-sheet structure may interfere with binding to the HEV receptors we investigated. All the HEV VP2 receptor footprints occured in the EF ‘puff’ but in this region of HRV-QPM a number of deletions and residue substitutions were apparent (<a href="#pone-0001847-g004">Figure 4</a>) that affected the appearance of predicted structural overlap. HEV footprints in VP3 differed from HRV-QPM at the N-terminus and the CD loop due to amino acid substitutions and loop-shortening deletions but they overlapped in the GH loop.</p>
<div class="figure" id="pone-0001847-g004"><div class="img"><a name="pone-0001847-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of HRV-QPM and HEV structures at the sites comprising HEV receptor footprints.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Figures indicate comparison of the predicted structures of HRV-QPM (orange) with representative HEVs (gray) in regions identified as HEV receptor footprints (red) across VP1, VP2 and VP3 proteins. RMSD values shown for conformational comparison of CAR and HRV-QPM structures, in angstroms.</p>
<span>doi:10.1371/journal.pone.0001847.g004</span></div>

<h4>HRV-QPM capsid prediction and visualization</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>HRV-QPM protomers were replicated and mapped onto an icosahedral lattice (T = 1 configuration; <a href="#pone-0001847-g005">Figure 5A</a>), rendered in 3D and depth cued in order to predict capsid structures. Since HRV-QPM protein and protomer structures were inferred by homology to their closest available relatives, these capsids (HRV-16 and HRV-14) were produced from the available empirical data, using the same approach. Visual comparisons revealed that the deletions in the HRV-QPM VP1 most obviously reduced the size of protrusions around the 5-fold axis of the capsid.</p>
<div class="figure" id="pone-0001847-g005"><div class="img"><a name="pone-0001847-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Predicted virion capsid structures.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Replication (×60) and mapping of the predicted HRV-QPM protomer onto a T = 1 icosahedral lattice (representing T = p3 configuration). (B) 3D rendering of predicted HRV capsids providing imagery similar to that obtained by cryo-electron microscopy reconstruction at 10 Å resolution. HRV-2, -14 and -16 predicted structures were derived from crystallography data. Each viral capsid has been depth cued to demonstrate canyon structure; yellow indicates surface detail and blue identifies areas of greatest depth. (C) Numbers superimposed over the lattice indicate the fold axis. The simplified position of the canyon is indicated by a circle on the capsid. All models were rendered and oriented identically, as determined by VIPERdb <a href="#pone.0001847-Shepherd1">[59]</a>.</p>
<span>doi:10.1371/journal.pone.0001847.g005</span></div>

<h4>Phylogenetics</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Previously, phylogenies had been estimated using subgenomic sequences from HRV A2-like strains <a href="#pone.0001847-McErlean1">[31]</a>–<a href="#pone.0001847-Lau1">[33]</a>. To test whether this approach accurately represented inter-strain relationships, we compared the entire polyprotein sequences from 120 previously described and, for the first time, all six newly-identified picornavirus strains (<a href="#pone-0001847-g006">Figure 6</a>). We confirmed that the newly-identified HRVs occupy a distinct phylogenetic position with the genus <em>Rhinovirus</em> <a href="#pone.0001847-Arden1">[34]</a> and are most closely related to the HRV A strains (also supported by SimPlot mapping; <a href="#pone-0001847-g003">Figure 3</a>). We also found support for our earlier data <a href="#pone.0001847-McErlean1">[31]</a> that this clade is divided into two distinct subgroups. In the 2C and 3CD regions, these subgroups share less than 70% amino acid identity, which also occurs among members of the existing HRV species.</p>
<div class="figure" id="pone-0001847-g006"><div class="img"><a name="pone-0001847-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Neighbour-joining phylogeny based on representative full-length picornavirus polyprotein sequences.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Trees are unrooted and relevant nodes are labelled with bootstrap values (%) (see <a href="#s4">materials and methods</a> for details). Species are indicated next to vertical bars. CV-Coxsackievirus A; EV-Echovirus; HEV-Enterovirus; HPV-Poliovirus.</p>
<span>doi:10.1371/journal.pone.0001847.g006</span></div>

<h4>Clinical impact of HRV-QPM variants</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Previously, preliminary and potentially subjective notes made by physicians upon first contact with ill individuals had been used for an approximate determination of the clinical impact of HRV-QPM <a href="#pone.0001847-McErlean1">[31]</a>. To better address clinical impact, a comprehensive review of available patient charts was undertaken and two objective severity scoring tools were applied (<a href="#pone-0001847-t001">Table 1</a>). Most HRV-QPM-positive individuals (76.5%) were admitted to hospital (<a href="#pone-0001847-t002">Table 2</a>). Five patients were admitted for 96 hours or longer; two with severe illness and only HRV-QPM detected (variants 005 and 012) and one with moderate illness positive for HRV-QPM (variant 017) and HCoV-NL63. The remaining two had underlying medical conditions and multiple infections (HRV-QPM variant 009 and HCoV-229E with cystic fibrosis deterioration and HRV-QPM variant 008, HBoV and HCoV-NL63 with a viral URTI during admission for cardiac surgery). Oxygen therapy was required by almost half of all HRV-QPM-positive individuals studied. In addition to those described previously <a href="#pone.0001847-McErlean1">[31]</a>, a new co-detection was identified with the newly identified polyomavirus, WUPyV <a href="#pone.0001847-Bialasiewicz1">[41]</a> in a patient exhibiting a persistent cough, already positive for HRV-QPM (variant 013) and HBoV.</p>
<div class="figure" id="pone-0001847-t001"><div class="img"><a name="pone-0001847-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Criteria used to determine the severity<a href="#pone.0001847-Sloots1">[60]</a> of respiratory illness upon discharge, in patients presenting to Queensland hospitals who tested positive for HRV-QPM.</span></strong></p><span>doi:10.1371/journal.pone.0001847.t001</span></div><div class="figure" id="pone-0001847-t002"><div class="img"><a name="pone-0001847-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>HRV-QPM detections and association of co-morbidities and severity of respiratory illness upon discharge.</span></strong></p><span>doi:10.1371/journal.pone.0001847.t002</span></div>

<h4>Applying species criteria to the HRV A2 strains</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p><em>In silico</em> data obtained from this study permitted an enhanced analysis of HRV A2 strains using the ICTV criteria which assign a member to the genus <em>Rhinovirus</em> or <em>Enterovirus</em> <a href="#pone.0001847-Stanway1">[42]</a> (<a href="#pone-0001847-t003">Table 3</a>). The requirement common to both genera was for &gt;70% amino acid identity in the P1 and 2C+3CD regions; this was not met by the HRV A2 genomes (<a href="#pone-0001847-g001">Figure 1B</a>). The remaining <em>Rhinovirus</em> species criterion was classification based on capsid-binding antiviral susceptibility and we found here, building upon earlier data <a href="#pone.0001847-McErlean1">[31]</a>, that all HRV A2 strains contain a Thr<sub>191</sub>. This residue reportedly contributes to conveying resistance to the capsid-binding antiviral Pleconaril <a href="#pone.0001847-Ledford1">[4]</a>, <a href="#pone.0001847-Ledford2">[43]</a>, placing the divergent HRVs into antiviral Group A <a href="#pone.0001847-Andries1">[5]</a>.</p>
<div class="figure" id="pone-0001847-t003"><div class="img"><a name="pone-0001847-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001847" data-uri="info:doi/10.1371/journal.pone.0001847.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001847.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001847.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001847.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001847.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001847.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Adherence of the newly identified HRV A2s to the ICTV species demarcation criteria.</span></strong></p><span>doi:10.1371/journal.pone.0001847.t003</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Comparison to the <em>Enterovirus</em> species criteria revealed several features that may also be applicable for classifying HRV A2 strains. Firstly, although we could not identify a specific receptor using these <em>in silico</em> methods, the HRV-QPM models most closely accommodated interaction with an ICAM-1-like molecule, similar to some HEV C viruses <a href="#pone.0001847-Rieder1">[44]</a>. Secondly, HRV A2 genomes exhibited ≤2.5% variation in G+C composition compared to HEV C and HEV D and ~5% compared to HEV A and HEV B. Thirdly, HRV- and HEV-like protease cleavage sites and locations suggested similar proteolytic processing (<a href="#pone-0001847-g001">Figure 1A</a>). Although we have been unable to find any indication of recombination in HRV-QPM <a href="#pone.0001847-McErlean1">[31]</a>, the differences in amino acid identity within the novel HRVs, particularly in the 2C and 3CD region of HRV-X2, -C024 and -C025 compared to HRV-QPM, -X1 and -C026 (data not shown), may indicate a relevant event requiring further study.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>In sum, these data indicate that the HRV A2 strains cannot be assigned to an existing HRV or HEV species and should be assigned to a new species, tentatively called <em>Human rhinovirus C</em>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In 2007, the sequences of a number of divergent HRV strains were reported as a result of worldwide molecular investigations into cases of suspected ARTI <a href="#pone.0001847-Kistler1">[26]</a>, <a href="#pone.0001847-McErlean1">[31]</a>, <a href="#pone.0001847-Lau1">[33]</a>. Because these strains formed a distinct clade within the existing HRV As, we previously assigned to all Australian and similar New York strains, the title of HRV A2 <a href="#pone.0001847-McErlean1">[31]</a>. Kistler <em>et al.</em> identified similarly divergent strains <a href="#pone.0001847-Kistler2">[32]</a> and Lau et al. used sequence-based criteria to propose that some HRV A2-like strains could be classified as a new species <a href="#pone.0001847-Lau1">[33]</a>. Our ongoing efforts to characterise additional HRVs led us to catalogue the distinguishing molecular and clinical features of these HRV A2-like strains and to better address their taxonomic placement. We undertook <em>in sili</em>co analyses, for the first time using all complete HRV A2 coding sequences and performed a retrospective review of the medical records of cases from which variants of the prototype strain, HRV-QPM, had been previously detected. To date the studies reported herein are the first of their kind to have been conducted on a single HRV strain.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>During our genomic analysis we found HRV A2 strains shared only 50%–53% average amino acid sequence identity with other HRV strains (<a href="#pone-0001847-g001">Figure 1</a>); less with any strain from an HEV species. Average HRV A2 G+C content (42.4%) fell between the rhinovirus (38.7%) and enterovirus (45.6%) genera, suggesting an intermediate evolutionary path for HRV A2 strains. Nonetheless, a large number of HRV motifs, including those recognized by the RNA-dependent RNA polymerase, were retained by HRV A2 strains. This supported the proposal that some HRV genomic regions are under purifying selection <a href="#pone.0001847-Kistler1">[26]</a>, even among the newly identified and divergent strains. HRV A2 strains were most similar to the HRV As overall, but amino acid sequences were as different from any traditional HRV or HEV species as the species in those genera were from each other. Previous subgenomic HRV A2 nucleotide phylogenies <a href="#pone.0001847-McErlean1">[31]</a>–<a href="#pone.0001847-Lau1">[33]</a> were confirmed and expanded herein reinforcing that these NIVs reside within a distinct clade of the genus <em>Rhinovirus</em>, branching from the existing HRV A species (<a href="#pone-0001847-g006">Figure 6</a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The inability to propagate an HRV A2 virus could be due to many factors including the age, storage, site of origin and amount of the inoculum, its handling during processing, how long after symptoms appear before sampling occurs and the culture conditions used. It is also possible that an unknown primary receptor is employed or that binding of a secondary receptor is required for successful viral attachment and entry; or a mix of both. We sought to approach receptor-related issues using homology models derived crystallography data which already exist for a small number of HRV strains. Models of the component structural proteins from the prototypical HRV A2 strain, HRV-QPM, predicted that the characteristic ‘jellyroll’ conformations were retained (<a href="#pone-0001847-g002">Figure 2</a>) despite frequent and distinguishing deletions and inter- and intra-genus diversity at the sites of receptor contact and antigenicity (<a href="#pone-0001847-g003">Figure 3</a>). This also supported findings that HRV variability is mostly localized to receptor and antigenic sites <a href="#pone.0001847-Racaniello1">[24]</a>, <a href="#pone.0001847-Lau1">[33]</a>, <a href="#pone.0001847-Stirk1">[45]</a> and begins to extend the data to include the more divergent strains.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The VP1 BC and HI loops receive the VLDL-R molecule among some minor group HRV strains <a href="#pone.0001847-Hewat1">[12]</a>, <a href="#pone.0001847-Neumann1">[46]</a>, <a href="#pone.0001847-Vlasak2">[47]</a>. The comparative reduction in size of protrusions along the 5-fold axis, resulting from deletions in the BC, DE and HI loops of the HRV-QPM VP1, together with structure-altering deletions and substitutions affecting some antigenic sites previously identified from a minor group strain (HRV-2), particularly site A, lead us to propose that HRV A2 viruses are unlikely to behave as members of the minor group. Mapping the known ICAM-1 contact sites onto the predicted HRV-QPM structure revealed that most of the footprint was retained in structure by HRV-QPM, although it differed in sequence compared to other major group strains. SimPlot data revealed that sequence variation was not uncommon among strains within the major groups. Two small α-helices predicted to form in the HRV-QPM VP1 and VP3 footprints may disrupt receptor binding but since ICAM-1 is predicted to attach at differing angles in different HRV strains <a href="#pone.0001847-Rossmann1">[8]</a>, it may be suitably flexible to overcome such obstacles. Although NImIA and NImIB, first identified in a major group virus (HRV-14), were affected by the deletions in VP1, additional support for HRV-QPM belonging to the major HRV group came from structural similarities in the NImII (VP2) and NImIII (VP3). Amino acid substitutions occurred at these sites (E→G and G→T respectively) rather than insertions or deletions. These variations may result in distinct immunogenicity.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Localisation of four known HEV receptor footprints to the predicted HRV-QPM structure identified at least one major sequence and/or structural divergence in each (<a href="#pone-0001847-g004">Figure 4</a>). However, like the ICAM-1 interactions with HRV-14 and -16, HEV receptors could still bind at the same locations but utilize different amino acids on the capsid of HRV A2 strains <a href="#pone.0001847-Bhella1">[20]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Recent findings by Bartlett et al have shown that once past the receptor, HRV replication can occur within non-human tissues <a href="#pone.0001847-Bartlett1">[48]</a> exemplifying the importance of receptors in moderating the potential for more widespread HRV replication and illness. The high detection frequency of HRV A2 strains from patients with acute LRT illness may be another indication that these divergent viruses employ a tissue-specific receptor or it may be that all HRV strains cause similar illness, they simply have not been subjected to sufficient individual study. The LDLR family of molecules are located on many tissues in many species <a href="#pone.0001847-Hofer1">[10]</a> and yet all these locations do not, to our knowledge, host HRV replication. Identifying an HRV A2 receptor <em>in vitro</em> cell culture system will be important for empirically addressing receptor- and replication-related issues and the outcomes may have broad implications for what is currently known of the attachment, entry and replication of HRV strains. Based on our predictive <em>in silico</em> studies of HRV-QPM, an ICAM-like molecule may be an early candidate for the role of receptor.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Future <em>in silico</em> studies could examine all available HRV polyprotein sequences to predict the extent of flexibility in receptor interactions ‘normally’ occurring in the major and minor groups. This would provide a context for the structural differences we observed in HRV-QPM. The sequence of most HRV genomes is unknown and our understanding of HRV structure is mostly due to crystallography data derived from less than 5% of HRV strains. There are limited structural data to support the wholesale extrapolation of receptor binding behaviour and capsid structure to the other HRV strains. Without data to the contrary, <em>in silico</em> analyses appear to provide a suitable surrogate to address this issue. By employing RMSD calculations to examine the conformational similarity between predicted HRV-QPM structures and those from HRV structures determined empirically, we could be confident that our predictions approximated experimental data. All comparative values (0.064–0.766 Å) were well below the maximum resolution used to originally determine HRV crystal structures (2–3 Å <a href="#pone.0001847-Rossmann2">[14]</a>, <a href="#pone.0001847-Verdaguer1">[15]</a>, <a href="#pone.0001847-Oliveira1">[49]</a>). Comparison to an HEV footprint also returned values below this threshold.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>While the inability to cultivate divergent HRV strains is a defining feature of the HRV A2s it is also a hindrance to classifying them. In our experience (data not shown), the earliest PCR-based methods <a href="#pone.0001847-Gama1">[50]</a> are already efficient at detecting these divergent strains. By extrapolating from published subgenomic sequence data, this clade is populated by a great number of divergent strains which are distributed both geographically and temporally, suggesting an endemic aspect to these previously uncharacterised viruses. No evidence exists to suggest that the HRV A2 strains are emerging viruses, rather it appears that they are newly identified strains that have been contributing to respiratory illness, in the absence of detection using culture-based diagnostic methods, for many years. There has now arisen a need for reliable protocols to characterise these putative viruses but in the absence of an HRV A2 isolate, neutralization, acid sensitivity and antiviral susceptibility studies, important for taxonomic placement, cannot be obtained. We have catalogued a series of features using methods which, while requiring further validation, contribute to our understanding of the biology and pathogenesis of HRV A2 strains. Additional molecular approaches will also be useful to classify other recently described divergent HRV strains <a href="#pone.0001847-Lee1">[51]</a>. Our data demonstrated that the newly identified HRVs, residing within the HRV A2 clade, do not satisfy the criteria for assignment to any existing HRV or HEV species and most likely constitute one or more novel species in the current genus <em>Rhinovirus</em>, tentatively called <em>Human rhinovirus C</em> (HRV C).</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Until recently, the genus <em>Rhinovirus</em> has been frequently neglected from clinical laboratory diagnosis and to date its members are not sought independently. Improved diagnostic methods, the broader application of existing PCR technology, new sequence data and many new insights have greatly enhanced our understanding of human rhinoviruses and the selective pressures impacting on their evolution <a href="#pone.0001847-Tapparel1">[25]</a>, <a href="#pone.0001847-Kistler1">[26]</a>. Similarly, the identification of a putative new HRV species, HRV C, the characterisation of HRV-QPM's molecular features and epidemiology and the identification of similar viruses around the world <a href="#pone.0001847-McErlean1">[31]</a>, <a href="#pone.0001847-Arden1">[34]</a>, <a href="#pone.0001847-Lamson1">[52]</a> have potential to reinvigorate HRV research. We found that the variants of one strain of the HRV Cs, HRV-QPM, sought in a well characterised population of specimens collected over all months of a single year, were more often associated with LRT illness than is commonly reported. The illness frequently presented as exacerbation of expiratory wheezing or persistent cough, with a requirement for supplemental oxygen, steroids and bronchodilator treatments. Despite exhaustive PCR-based investigation for other respiratory viral causes in our previous retrospective studies <a href="#pone.0001847-McErlean1">[31]</a>, <a href="#pone.0001847-Bialasiewicz1">[41]</a>, HRV-QPM detection, in the absence of any other pathogen, occurred among infants with mild, moderate and severe (in both instances) LRT illness and most cases (88.2%) were admitted to hospital. Associated illness was usually (92.3% of the time) scored as mild to moderate and 58.8% of the HRV-QPM positive cases were children aged 12 months or less. The same age group represented 43.2% of the total study population originally screened for HRV-QPM (n = 1,244). While it is to be expected that a paediatric hospital-based population would overestimate the severity of HRV C clinical impact compared to a community-based study, our study may in fact better represent the impact of first infection with HRV strains, the major pathogens detected during the first year of life <a href="#pone.0001847-Kusel1">[53]</a>.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>It will be important to further characterize these NIVs and determine whether their differences confer distinctive biological behaviours, such as unique growth properties, antiviral resistance and discrete clinical outcomes among infected individuals. Strain-focussed studies could also identify how many distinct HRVs circulate in a single respiratory season, how often a given strain recurs in a population and just how many HRV strains there are.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Nucleotide and amino acid comparisons</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Nucleotide and amino acid compositions were determined using BioEdit Sequence Alignment Editor© v7.0.5.3. Picornavirus coding sequences (obtained from GenBank or picornaviridae.com) were determined by multiple alignments and subsequent truncations of the 5′and 3′UTRs. Discrepant nucleotides were substituted with the most frequently employed nucleotides, as determined by consensus alignment. Alignments available upon request. Predicted picornavirus cleavage sites were identified by amino acid sequence submission to the NetPicoRNA World Wide Web server <a href="#pone.0001847-Blom1">[54]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Amino acid similarity was determined using SimPlot© v3.5 <a href="#pone.0001847-Lole1">[37]</a> employing the Hamming method on a 50% consensus of each group, a sliding window of 100 bp and a 1 bp step. Similarity data were mapped to the HRV-QPM capsid residues on the original sequence alignment and visualized using Chimera <a href="#pone.0001847-Pettersen1">[55]</a>.</p>


<h4>Phylogenetic analyses</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Picornavirus polyprotein sequences were compiled, translated, and aligned with the program BioEdit <a href="#pone.0001847-Hall1">[56]</a>. A neighbour-joining tree was generated using the Kimura two-parameter estimation in MEGA 3.1 <a href="#pone.0001847-Kumar1">[57]</a>. Nodal confidence values (%), noted at the relevant nodes, indicate the results of bootstrap resampling (<em>n</em> = 1000). Because of the sequence diversity among the viruses analyzed, we undertook an additional analysis on a subgroup of strains whereby we gap-stripped the alignment, removing all gaps/highly divergent regions and re-estimated the phylogeny (data not shown). This confirmed the phylogenetic position determined in the first instance.</p>


<h4>Structural homology modelling</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Secondary structures in the HRV A2 structural proteins were predicted using amino acid sequences submitted to the Jpred web server (<a href="http://www.compbio.dundee.ac.uk/">http://www.compbio.dundee.ac.uk/</a>~www-jpred/submit.html). The location of α-helices and β-sheets were revealed by comparison to other picornavirus sequences using Cn3D 4.1 (<a href="http://130.14.29.110/Structure/CN3D/cn3d.shtml">http://130.14.29.110/Structure/CN3D/cn3d​.shtml</a>) (data not shown).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>To determine the predicted protomer structure of HRV-QPM, DeepView/Swiss-Pdb Viewer v3.7, <a href="#pone.0001847-Guex1">[58]</a> were used to locate and individually thread HRV-QPM amino acid sequences into HRV reference structures. Of the five HRV strains with empirically determined structural data available, HRV-14 <a href="#pone.0001847-Rossmann2">[14]</a> and -16 <a href="#pone.0001847-Oliveira1">[49]</a> shared the highest sequence identities with HRV-QPM in the regions chosen. Proteins VP1 to VP3 were aligned and threaded through HRV-16 (1ayn1, 1ayn2, 1ayn3 respectively) and VP4 through HRV-14 (1na1). Models were submitted to the SWISS-MODEL server for final threading analysis. The resulting HRV-QPM protein data bank files (PDB) were then matched and structurally aligned to a template HRV protomer (HRV-16, 1AYN). Other picornavirus reference PDB files used in this study: HRV-2 (1FBN), HRV-3 (1RHI), HRV-14 (4RHV), CV-A21 (1Z7S), E-11 (1H8T), PV-1 (1HXS) and CV-B3 (1COV). Structural matching, alignments, ribbon figures, capsid predictions and root mean square deviation (RMSD) data were produced using Chimera <a href="#pone.0001847-Pettersen1">[55]</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Dr. Amy Kistler (USA) for the HRV-87 genome and helpful discussions and Prof. Patrick Woo (Hong Kong) for HRV C sequences. We thank Queensland Health Pathology Service for the provision of clinical specimens.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: IM PM. Performed the experiments: IM LS PM DW. Analyzed the data: SL IM LS PM EA MN DW. Contributed reagents/materials/analysis tools: TS IM DW. Wrote the paper: IM PM. Other: Statistical computations and manuscript co-author: SL. Creation of a clinical severity scoring system and medical chart review: MN. Ethics approvals: EA. Clinical chart collection and data collection: EA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001847-Kiang1" id="pone.0001847-Kiang1"></a>Kiang D, Yagi S, Kantardjieff KA, Kim EJ, Louie JK, et al.  (2007) Molecular characterization of a variant rhinovirus from an outbreak associated with uncommonly high mortality. J Clin Virol  38: 227–237.  <ul class="find" data-citedArticleID="1079020" data-doi="10.1016/j.jcv.2006.12.016"><li><a href="http://dx.doi.org/10.1016/j.jcv.2006.12.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+a+variant+rhinovirus+from+an+outbreak+associated+with+uncommonly+high+mortality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+a+variant+rhinovirus+from+an+outbreak+associated+with+uncommonly+high+mortality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001847-Hayden1" id="pone.0001847-Hayden1"></a>Hayden FG (2004) Rhinovirus and the lower respiratory tract. Reviews in Medical Virology  14: 17–31.  <ul class="find" data-citedArticleID="1079008" data-doi="10.1002/rmv.406"><li><a href="http://dx.doi.org/10.1002/rmv.406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rhinovirus+and+the+lower+respiratory+tract." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rhinovirus+and+the+lower+respiratory+tract.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001847-Papadopoulos1" id="pone.0001847-Papadopoulos1"></a>Papadopoulos NG, Bates PJ, Bardin PG, Papi A, Leir SH, et al.  (2000) Rhinoviruses infect the lower airways. J Infect Dis  181: 1875–1884.  <ul class="find" data-citedArticleID="1079062" data-doi="10.1086/315513"><li><a href="http://dx.doi.org/10.1086/315513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rhinoviruses+infect+the+lower+airways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rhinoviruses+infect+the+lower+airways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001847-Ledford1" id="pone.0001847-Ledford1"></a>Ledford RM, Patel NR, Demenczuk TM, Watanyar A, Herbertz T, et al.  (2004) VP1 sequencing of all human rhinovirus serotypes: Insights into genus phylogeny and susceptibility to antiviral capsid-binding compounds. Journal of Virology  78: 3663–3674.  <ul class="find" data-citedArticleID="1079038" data-doi="10.1128/jvi.78.7.3663-3674.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.7.3663-3674.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VP1+sequencing+of+all+human+rhinovirus+serotypes%3A+Insights+into+genus+phylogeny+and+susceptibility+to+antiviral+capsid-binding+compounds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VP1+sequencing+of+all+human+rhinovirus+serotypes%3A+Insights+into+genus+phylogeny+and+susceptibility+to+antiviral+capsid-binding+compounds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001847-Andries1" id="pone.0001847-Andries1"></a>Andries K, Dewindt B, Snoeks J, Wouters L, Moereels H, et al.  (1990) Two groups of rhinoviruses revealed by a panel of antiviral compounds present sequence divergence and differential pathogenicity. J Virol  64: 1117–1123.  <ul class="find" data-citedArticleID="1078982"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Two+groups+of+rhinoviruses+revealed+by+a+panel+of+antiviral+compounds+present+sequence+divergence+and+differential+pathogenicity.&amp;auth=&amp;atitle=Two+groups+of+rhinoviruses+revealed+by+a+panel+of+antiviral+compounds+present+sequence+divergence+and+differential+pathogenicity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+groups+of+rhinoviruses+revealed+by+a+panel+of+antiviral+compounds+present+sequence+divergence+and+differential+pathogenicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+groups+of+rhinoviruses+revealed+by+a+panel+of+antiviral+compounds+present+sequence+divergence+and+differential+pathogenicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001847-Laine1" id="pone.0001847-Laine1"></a>Laine P, Blomqvist S, Savolainen C, Andries K, Hovi T (2006) Alignment of capsid protein VP1 sequences of all human rhinovirus prototype strains: conserved motifs and functional domains. J Gen Virol  87: 129–138.  <ul class="find" data-citedArticleID="1079032" data-doi="10.1099/vir.0.81137-0"><li><a href="http://dx.doi.org/10.1099/vir.0.81137-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alignment+of+capsid+protein+VP1+sequences+of+all+human+rhinovirus+prototype+strains%3A+conserved+motifs+and+functional+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alignment+of+capsid+protein+VP1+sequences+of+all+human+rhinovirus+prototype+strains%3A+conserved+motifs+and+functional+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001847-Savolainen1" id="pone.0001847-Savolainen1"></a>Savolainen C, Blomqvist S, Mulders MN, Hovi T (2002) Genetic clustering of all 102 human rhinovirus prototype strains: serotype 87 is close to human enterovirus 70. J Gen Virol  83: 333–340.  <ul class="find" data-citedArticleID="1079076"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+clustering+of+all+102+human+rhinovirus+prototype+strains%3A+serotype+87+is+close+to+human+enterovirus+70.&amp;auth=&amp;atitle=Genetic+clustering+of+all+102+human+rhinovirus+prototype+strains%3A+serotype+87+is+close+to+human+enterovirus+70." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+clustering+of+all+102+human+rhinovirus+prototype+strains%3A+serotype+87+is+close+to+human+enterovirus+70." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+clustering+of+all+102+human+rhinovirus+prototype+strains%3A+serotype+87+is+close+to+human+enterovirus+70.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001847-Rossmann1" id="pone.0001847-Rossmann1"></a>Rossmann MG, Bella J, Kolatkar PR, He Y, Wimmer E, et al.  (2000) Cell recognition and entry by rhino- and enteroviruses. Virology  269: 239–247.  <ul class="find" data-citedArticleID="1079072" data-doi="10.1006/viro.2000.0258"><li><a href="http://dx.doi.org/10.1006/viro.2000.0258" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+recognition+and+entry+by+rhino-+and+enteroviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+recognition+and+entry+by+rhino-+and+enteroviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001847-Marlovits1" id="pone.0001847-Marlovits1"></a>Marlovits TC, Abrahamsberg C, Blaas D (1998) Very-low-density lipoprotein receptor fragment shed from HeLa cells inhibits human rhinovirus infection. J Virol  72: 10246–10250.  <ul class="find" data-citedArticleID="1079048"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Very-low-density+lipoprotein+receptor+fragment+shed+from+HeLa+cells+inhibits+human+rhinovirus+infection.&amp;auth=&amp;atitle=Very-low-density+lipoprotein+receptor+fragment+shed+from+HeLa+cells+inhibits+human+rhinovirus+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Very-low-density+lipoprotein+receptor+fragment+shed+from+HeLa+cells+inhibits+human+rhinovirus+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Very-low-density+lipoprotein+receptor+fragment+shed+from+HeLa+cells+inhibits+human+rhinovirus+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001847-Hofer1" id="pone.0001847-Hofer1"></a>Hofer F, Gruenberger M, Kowalski H, Machat H, Huettinger M, et al.  (1994) Members of the low density lipoprotein receptor family mediate cell entry of a minor group common cold virus. Proc Natl Acad Sci U S A  91: 1839–1842.  <ul class="find" data-citedArticleID="1079014" data-doi="10.1073/pnas.91.5.1839"><li><a href="http://dx.doi.org/10.1073/pnas.91.5.1839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Members+of+the+low+density+lipoprotein+receptor+family+mediate+cell+entry+of+a+minor+group+common+cold+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Members+of+the+low+density+lipoprotein+receptor+family+mediate+cell+entry+of+a+minor+group+common+cold+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001847-Vlasak1" id="pone.0001847-Vlasak1"></a>Vlasak M, Roizman B, Reithmayer M, Goesler I, Laine P, et al.  (2005) The Minor Receptor Group of Human Rhinovirus (HRV) Includes HRV23 and HRV25, but the Presence of a Lysine in the VP1 HI Loop Is Not Sufficient for Receptor Binding. J Virol  79: 7389–7395.  <ul class="find" data-citedArticleID="1079096" data-doi="10.1128/jvi.79.12.7389-7395.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.12.7389-7395.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Minor+Receptor+Group+of+Human+Rhinovirus+%28HRV%29+Includes+HRV23+and+HRV25%2C+but+the+Presence+of+a+Lysine+in+the+VP1+HI+Loop+Is+Not+Sufficient+for+Receptor+Binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Minor+Receptor+Group+of+Human+Rhinovirus+%28HRV%29+Includes+HRV23+and+HRV25%2C+but+the+Presence+of+a+Lysine+in+the+VP1+HI+Loop+Is+Not+Sufficient+for+Receptor+Binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001847-Hewat1" id="pone.0001847-Hewat1"></a>Hewat EA, Neumann E, Conway JF, Moser R, Ronacher B, et al.  (2000) The cellular receptor to human rhinovirus 2 binds around the 5-fold axis and not in the canyon: a structural view. EMBO J  19: 6317–6325.  <ul class="find" data-citedArticleID="1079012" data-doi="10.1093/emboj/19.23.6317"><li><a href="http://dx.doi.org/10.1093/emboj/19.23.6317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cellular+receptor+to+human+rhinovirus+2+binds+around+the+5-fold+axis+and+not+in+the+canyon%3A+a+structural+view." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cellular+receptor+to+human+rhinovirus+2+binds+around+the+5-fold+axis+and+not+in+the+canyon%3A+a+structural+view.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001847-Khan1" id="pone.0001847-Khan1"></a>Khan AG, Pichler J, Rosemann A, Blaas D (2008) Human rhinovirus type 54 infection via heparan sulfate is less efficient and strictly dependent on low endosomal pH. J Virol  81: 4625–4632.  <ul class="find" data-citedArticleID="1079018" data-doi="10.1128/jvi.02160-06"><li><a href="http://dx.doi.org/10.1128/jvi.02160-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+rhinovirus+type+54+infection+via+heparan+sulfate+is+less+efficient+and+strictly+dependent+on+low+endosomal+pH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+rhinovirus+type+54+infection+via+heparan+sulfate+is+less+efficient+and+strictly+dependent+on+low+endosomal+pH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001847-Rossmann2" id="pone.0001847-Rossmann2"></a>Rossmann MG, Arnold E, Erickson JW, Frankenberger EA, Griffith JP, et al.  (1985) Structure of a human common cold virus and functional relationship to other picornaviruses. Nature  317: 145–153.  <ul class="find" data-citedArticleID="1079074" data-doi="10.1038/317145a0"><li><a href="http://dx.doi.org/10.1038/317145a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+a+human+common+cold+virus+and+functional+relationship+to+other+picornaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+a+human+common+cold+virus+and+functional+relationship+to+other+picornaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001847-Verdaguer1" id="pone.0001847-Verdaguer1"></a>Verdaguer N, Blaas D, Fita I (2000) Structure of human rhinovirus serotype 2 (HRV2). Journal of Molecular Biology  300: 1179–1194.  <ul class="find" data-citedArticleID="1079094" data-doi="10.1006/jmbi.2000.3943"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3943" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+human+rhinovirus+serotype+2+%28HRV2%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+human+rhinovirus+serotype+2+%28HRV2%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001847-Blaas1" id="pone.0001847-Blaas1"></a>Blaas D, Kuechler E, Vriend G, Arnold E, Luo M, et al.  (1987) Comparison of the three-dimensional structure of two human rhinoviruses (HRV2 and HRV14). Proteins  2: 263–272.  <ul class="find" data-citedArticleID="1078994" data-doi="10.1002/prot.340020402"><li><a href="http://dx.doi.org/10.1002/prot.340020402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+the+three-dimensional+structure+of+two+human+rhinoviruses+%28HRV2+and+HRV14%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+the+three-dimensional+structure+of+two+human+rhinoviruses+%28HRV2+and+HRV14%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001847-Oberste1" id="pone.0001847-Oberste1"></a>Oberste MS, Maher K, Schnurr D, Flemister MR, Lovchik JC, et al.  (2004) Enterovirus 68 is associated with respiratory illness and shares biological features with both the enteroviruses and the rhinoviruses. J Gen Virol  85: 2577–2584.  <ul class="find" data-citedArticleID="1079056" data-doi="10.1099/vir.0.79925-0"><li><a href="http://dx.doi.org/10.1099/vir.0.79925-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enterovirus+68+is+associated+with+respiratory+illness+and+shares+biological+features+with+both+the+enteroviruses+and+the+rhinoviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enterovirus+68+is+associated+with+respiratory+illness+and+shares+biological+features+with+both+the+enteroviruses+and+the+rhinoviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001847-Xiao1" id="pone.0001847-Xiao1"></a>Xiao C, Bator CM, Bowman VD, Rieder E, He Y, et al.  (2001) Interaction of coxsackievirus A21 with its cellular receptor, ICAM-1. J Virol  75: 2444–2451.  <ul class="find" data-citedArticleID="1079102" data-doi="10.1128/jvi.75.5.2444-2451.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.5.2444-2451.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+coxsackievirus+A21+with+its+cellular+receptor%2C+ICAM-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+coxsackievirus+A21+with+its+cellular+receptor%2C+ICAM-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001847-Stuart1" id="pone.0001847-Stuart1"></a>Stuart AD, McKee TA, Williams PA, Harley C, Shen S, et al.  (2002) Determination of the structure of a decay accelerating factor-binding clinical isolate of echovirus 11 allows mapping of mutants with altered receptor requirements for infection. J Virol  76: 7694–7704.  <ul class="find" data-citedArticleID="1079090" data-doi="10.1128/jvi.76.15.7694-7704.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.15.7694-7704.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determination+of+the+structure+of+a+decay+accelerating+factor-binding+clinical+isolate+of+echovirus+11+allows+mapping+of+mutants+with+altered+receptor+requirements+for+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determination+of+the+structure+of+a+decay+accelerating+factor-binding+clinical+isolate+of+echovirus+11+allows+mapping+of+mutants+with+altered+receptor+requirements+for+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001847-Bhella1" id="pone.0001847-Bhella1"></a>Bhella D, Goodfellow IG, Roversi P, Pettigrew D, Chaudhry Y, et al.  (2004) The structure of echovirus type 12 bound to a two-domain fragment of its cellular attachment protein decay-accelerating factor (CD 55). J Biol Chem  279: 8325–8332.  <ul class="find" data-citedArticleID="1078990" data-doi="10.1074/jbc.m311334200"><li><a href="http://dx.doi.org/10.1074/jbc.m311334200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+echovirus+type+12+bound+to+a+two-domain+fragment+of+its+cellular+attachment+protein+decay-accelerating+factor+%28CD+55%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+echovirus+type+12+bound+to+a+two-domain+fragment+of+its+cellular+attachment+protein+decay-accelerating+factor+%28CD+55%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001847-Belnap1" id="pone.0001847-Belnap1"></a>Belnap DM, McDermott Jr BM, Filman DJ, Cheng N, Trus BL, et al.  (2000) Three-dimensional structure of poliovirus receptor bound to poliovirus. Proc Natl Acad Sci U S A  97: 73–78.  <ul class="find" data-citedArticleID="1078988" data-doi="10.1073/pnas.97.1.73"><li><a href="http://dx.doi.org/10.1073/pnas.97.1.73" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+structure+of+poliovirus+receptor+bound+to+poliovirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+structure+of+poliovirus+receptor+bound+to+poliovirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001847-He1" id="pone.0001847-He1"></a>He Y, Chipman PR, Howitt J, Bator CM, Whitt MA, et al.  (2001) Interaction of coxsackievirus B3 with the full length coxsackievirus-adenovirus receptor. Nat Struct Biol  8: 874–878.  <ul class="find" data-citedArticleID="1079010" data-doi="10.1038/nsb1001-874"><li><a href="http://dx.doi.org/10.1038/nsb1001-874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+coxsackievirus+B3+with+the+full+length+coxsackievirus-adenovirus+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+coxsackievirus+B3+with+the+full+length+coxsackievirus-adenovirus+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001847-Rosen1" id="pone.0001847-Rosen1"></a>Rosen L (1965) Subclassification of picornaviruses. Bacteriological Reviews  29: 173–184.  <ul class="find" data-citedArticleID="1079070"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Subclassification+of+picornaviruses.&amp;auth=&amp;atitle=Subclassification+of+picornaviruses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subclassification+of+picornaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subclassification+of+picornaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001847-Racaniello1" id="pone.0001847-Racaniello1"></a>Racaniello VR (2001)  Picornaviridae: The viruses and their replication. In: Knipe DM, Howley PM, editors. Fields Virology. Philadelphia: Lippincott-Raven. pp. 685–722.  <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0001847-Tapparel1" id="pone.0001847-Tapparel1"></a>Tapparel C, Junier T, Gerlach D, Cordey S, van Belle S, et al.  (2007) New complete genome sequences of human rhinoviruses shed light on their phylogeny and genomic features. BMC Genomics  10: 224.  <ul class="find" data-citedArticleID="1079092" data-doi="10.1186/1471-2164-8-224"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-224" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+complete+genome+sequences+of+human+rhinoviruses+shed+light+on+their+phylogeny+and+genomic+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+complete+genome+sequences+of+human+rhinoviruses+shed+light+on+their+phylogeny+and+genomic+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001847-Kistler1" id="pone.0001847-Kistler1"></a>Kistler A, Webster DR, Rouskin S, Magrini V, Credle J, et al.  (2007) Genome-wide diversity and selective pressure in the human rhinovirus. Virol J  4: 40.  <ul class="find" data-citedArticleID="1079022" data-doi="10.1186/1743-422x-4-40"><li><a href="http://dx.doi.org/10.1186/1743-422x-4-40" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+diversity+and+selective+pressure+in+the+human+rhinovirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+diversity+and+selective+pressure+in+the+human+rhinovirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001847-Lukashev1" id="pone.0001847-Lukashev1"></a>Lukashev AN, Lashkevich VA, Ivanova OE, Koroleva GA, Hinkkanen AE, et al.  (2005) Recombination in circulating human enterovirus B: independent evolution of structural and non-structural genome regions. J Gen Virol  86: 3281–3290.  <ul class="find" data-citedArticleID="1079046" data-doi="10.1099/vir.0.81264-0"><li><a href="http://dx.doi.org/10.1099/vir.0.81264-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+in+circulating+human+enterovirus+B%3A+independent+evolution+of+structural+and+non-structural+genome+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+in+circulating+human+enterovirus+B%3A+independent+evolution+of+structural+and+non-structural+genome+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001847-Simmonds1" id="pone.0001847-Simmonds1"></a>Simmonds P, Welch J (2006) Frequency and dynamics of recombination within different species of human enteroviruses. Journal of Virology  80: 483–493.  <ul class="find" data-citedArticleID="1079082" data-doi="10.1128/jvi.80.1.483-493.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.1.483-493.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequency+and+dynamics+of+recombination+within+different+species+of+human+enteroviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequency+and+dynamics+of+recombination+within+different+species+of+human+enteroviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001847-Mori1" id="pone.0001847-Mori1"></a>Mori J, Clewley JP (1994) Polymerase chain reaction and sequencing for typing rhinovirus RNA. J Med Virol  44: 323–329.  <ul class="find" data-citedArticleID="1079052" data-doi="10.1002/jmv.1890440403"><li><a href="http://dx.doi.org/10.1002/jmv.1890440403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymerase+chain+reaction+and+sequencing+for+typing+rhinovirus+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymerase+chain+reaction+and+sequencing+for+typing+rhinovirus+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001847-Hamparian1" id="pone.0001847-Hamparian1"></a>Hamparian VV, Colonno RJ, Cooney MK, Dick EC, Gwaltney Jr JM, et al.  (1987) A collaborative report: Rhinoviruses - extension of the numbering system from 89 to 100. Virology  159: 191–192.  <ul class="find" data-citedArticleID="1079006" data-doi="10.1016/0042-6822(87)90367-9"><li><a href="http://dx.doi.org/10.1016/0042-6822(87)90367-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+collaborative+report%3A+Rhinoviruses+-+extension+of+the+numbering+system+from+89+to+100." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+collaborative+report%3A+Rhinoviruses+-+extension+of+the+numbering+system+from+89+to+100.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001847-McErlean1" id="pone.0001847-McErlean1"></a>McErlean P, Shackleton LA, Lambert SB, Nissen MD, Sloots TP, et al.  (2007) Characterisation of a newly identified human rhinovirus, HRV-QPM, discovered in infants with bronchiolitis. J Clin Virol  39: 67–75.  <ul class="find" data-citedArticleID="1079050" data-doi="10.1016/j.jcv.2007.03.012"><li><a href="http://dx.doi.org/10.1016/j.jcv.2007.03.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterisation+of+a+newly+identified+human+rhinovirus%2C+HRV-QPM%2C+discovered+in+infants+with+bronchiolitis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterisation+of+a+newly+identified+human+rhinovirus%2C+HRV-QPM%2C+discovered+in+infants+with+bronchiolitis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001847-Kistler2" id="pone.0001847-Kistler2"></a>Kistler A, Avila PC, Rouskin S, Wang D, Ward T, et al.  (2007) Pan-viral screening of respiratory tract infections in adults with and without asthma reveals unexpected human coronavirus and human rhinovirus diversity. J Infect Dis  196: 817–825.  <ul class="find" data-citedArticleID="1079024" data-doi="10.1086/520816"><li><a href="http://dx.doi.org/10.1086/520816" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pan-viral+screening+of+respiratory+tract+infections+in+adults+with+and+without+asthma+reveals+unexpected+human+coronavirus+and+human+rhinovirus+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pan-viral+screening+of+respiratory+tract+infections+in+adults+with+and+without+asthma+reveals+unexpected+human+coronavirus+and+human+rhinovirus+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001847-Lau1" id="pone.0001847-Lau1"></a>Lau SKP, Yip CCY, Que T-L, Lee RA, Au-Yeung RKH, et al.  (2007) Clinical and molecular epidemiology of human bocavirus in respiratory and fecal samples from children in Hong Kong. J Infect Dis  196: 986–993.  <ul class="find" data-citedArticleID="1079036" data-doi="10.1086/521310"><li><a href="http://dx.doi.org/10.1086/521310" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clinical+and+molecular+epidemiology+of+human+bocavirus+in+respiratory+and+fecal+samples+from+children+in+Hong+Kong." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clinical+and+molecular+epidemiology+of+human+bocavirus+in+respiratory+and+fecal+samples+from+children+in+Hong+Kong.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001847-Arden1" id="pone.0001847-Arden1"></a>Arden KE, McErlean P, Nissen MD, Sloots TP, Mackay IM (2006) Frequent detection of human rhinoviruses, paramyxoviruses, coronaviruses, and bocavirus during acute respiratory tract infections. J Med Virol  78: 1232–1240.  <ul class="find" data-citedArticleID="1078984" data-doi="10.1002/jmv.20689"><li><a href="http://dx.doi.org/10.1002/jmv.20689" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+detection+of+human+rhinoviruses%2C+paramyxoviruses%2C+coronaviruses%2C+and+bocavirus+during+acute+respiratory+tract+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+detection+of+human+rhinoviruses%2C+paramyxoviruses%2C+coronaviruses%2C+and+bocavirus+during+acute+respiratory+tract+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001847-Savolainen2" id="pone.0001847-Savolainen2"></a>Savolainen C, Laine P, Mulders MN, Hovi T (2004) Sequence analysis of human rhinoviruses in the RNA-dependent RNA polymerase coding region reveals within-species variation. J Gen Virol  85: 2271–2277.  <ul class="find" data-citedArticleID="1079078" data-doi="10.1099/vir.0.79897-0"><li><a href="http://dx.doi.org/10.1099/vir.0.79897-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+analysis+of+human+rhinoviruses+in+the+RNA-dependent+RNA+polymerase+coding+region+reveals+within-species+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+analysis+of+human+rhinoviruses+in+the+RNA-dependent+RNA+polymerase+coding+region+reveals+within-species+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001847-Brown1" id="pone.0001847-Brown1"></a>Brown B, Oberste MS, Maher K, Pallansch MA (2003) Complete genomic sequencing shows that polioviruses and members of human enterovirus species C are closely related in the noncapsid coding region. J Virol  77: 8973–8984.  <ul class="find" data-citedArticleID="1078998" data-doi="10.1128/jvi.77.16.8973-8984.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.16.8973-8984.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genomic+sequencing+shows+that+polioviruses+and+members+of+human+enterovirus+species+C+are+closely+related+in+the+noncapsid+coding+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genomic+sequencing+shows+that+polioviruses+and+members+of+human+enterovirus+species+C+are+closely+related+in+the+noncapsid+coding+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001847-Lole1" id="pone.0001847-Lole1"></a>Lole KS, Bollinger RC, Paranjape RS, Gadkari D, Kulkarni SS, et al.  (1999) Full-length human iummunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination. Journal of Virology  73: 152–160.  <ul class="find" data-citedArticleID="1079044"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Full-length+human+iummunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination.&amp;auth=&amp;atitle=Full-length+human+iummunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full-length+human+iummunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full-length+human+iummunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001847-Palmenberg1" id="pone.0001847-Palmenberg1"></a>Palmenberg AC, Sgro J-Y (2002) Alignments and comparative profiles of picornavirus genera. 42: 149–155.  <ul class="find" data-citedArticleID="1079060"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alignments+and+comparative+profiles+of+picornavirus+genera.&amp;auth=&amp;atitle=Alignments+and+comparative+profiles+of+picornavirus+genera." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alignments+and+comparative+profiles+of+picornavirus+genera." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alignments+and+comparative+profiles+of+picornavirus+genera.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001847-Horsnell1" id="pone.0001847-Horsnell1"></a>Horsnell C, Gama RE, Hughes PJ, Stanway G (1995) Molecular relationships between 21 human rhinovirus serotypes. J Gen Virol  76: 2549–2555.  <ul class="find" data-citedArticleID="1079016" data-doi="10.1099/0022-1317-76-10-2549"><li><a href="http://dx.doi.org/10.1099/0022-1317-76-10-2549" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+relationships+between+21+human+rhinovirus+serotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+relationships+between+21+human+rhinovirus+serotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001847-Kolatkar1" id="pone.0001847-Kolatkar1"></a>Kolatkar PR, Bella J, Olson NH, Bator CM, Baker TS, et al.  (1999) Structural studies of two rhinovirus serotypes complexed with fragments of their cellular receptor. The EMBO Journal  18: 6249–6259.  <ul class="find" data-citedArticleID="1079026" data-doi="10.1093/emboj/18.22.6249"><li><a href="http://dx.doi.org/10.1093/emboj/18.22.6249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+studies+of+two+rhinovirus+serotypes+complexed+with+fragments+of+their+cellular+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+studies+of+two+rhinovirus+serotypes+complexed+with+fragments+of+their+cellular+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001847-Bialasiewicz1" id="pone.0001847-Bialasiewicz1"></a>Bialasiewicz S, Whiley DM, Lambert SB, Jacob KC, Bletchly C, et al.  (2008) Presence of the newly discovered human polyomaviruses KI and WU in Australian patients with acute respiratory tract infection. J Clin Virol  41: 63–68.  <ul class="find" data-citedArticleID="1078992" data-doi="10.1016/j.jcv.2007.11.001"><li><a href="http://dx.doi.org/10.1016/j.jcv.2007.11.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Presence+of+the+newly+discovered+human+polyomaviruses+KI+and+WU+in+Australian+patients+with+acute+respiratory+tract+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Presence+of+the+newly+discovered+human+polyomaviruses+KI+and+WU+in+Australian+patients+with+acute+respiratory+tract+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001847-Stanway1" id="pone.0001847-Stanway1"></a>Stanway G, Brown F, Christian P, Hovi T, Hyypiä T, et al.  (2005)  Family <em>Picornaviridae</em>. In: Fauquet CM, Mayo MA, Maniloff J, Desselberger U, Ball LA, editors. Virus taxonomy: eighth report of the international committee in taxonomy of viruses. San Diego: Elsevier Academic Press. pp. 757–778.  <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0001847-Ledford2" id="pone.0001847-Ledford2"></a>Ledford RM, Collett MS, Pevear DC (2005) Insights into the genetic basis for natural phenotypic resistance of human rhinoviruses to pleconaril. Antiviral Res  68: 135–138.  <ul class="find" data-citedArticleID="1079040" data-doi="10.1016/j.antiviral.2005.08.003"><li><a href="http://dx.doi.org/10.1016/j.antiviral.2005.08.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+the+genetic+basis+for+natural+phenotypic+resistance+of+human+rhinoviruses+to+pleconaril." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+the+genetic+basis+for+natural+phenotypic+resistance+of+human+rhinoviruses+to+pleconaril.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001847-Rieder1" id="pone.0001847-Rieder1"></a>Rieder E, Wimmer E (2002) Cellular receptors of picornaviruses. Molecular biology of picornaviruses. Washington DC: ASM Press. pp. 61–70.  <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0001847-Stirk1" id="pone.0001847-Stirk1"></a>Stirk HJ, Thornton JM (1994) The BC loop in poliovirus coat protein VP1: an ideal acceptor site for major insertions. Protein Eng  7: 47–56.  <ul class="find" data-citedArticleID="1079088" data-doi="10.1093/protein/7.1.47"><li><a href="http://dx.doi.org/10.1093/protein/7.1.47" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+BC+loop+in+poliovirus+coat+protein+VP1%3A+an+ideal+acceptor+site+for+major+insertions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+BC+loop+in+poliovirus+coat+protein+VP1%3A+an+ideal+acceptor+site+for+major+insertions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001847-Neumann1" id="pone.0001847-Neumann1"></a>Neumann E, Moser R, Snyers L, Blaas D, Hewat EA (2003) A cellular receptor of human rhinovirus type 2, the very-low-density lipoprotein receptor, binds to two neighboring proteins of the viral capsid. J Virol  77: 8504–8511.  <ul class="find" data-citedArticleID="1079054" data-doi="10.1128/jvi.77.15.8504-8511.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.15.8504-8511.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cellular+receptor+of+human+rhinovirus+type+2%2C+the+very-low-density+lipoprotein+receptor%2C+binds+to+two+neighboring+proteins+of+the+viral+capsid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cellular+receptor+of+human+rhinovirus+type+2%2C+the+very-low-density+lipoprotein+receptor%2C+binds+to+two+neighboring+proteins+of+the+viral+capsid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001847-Vlasak2" id="pone.0001847-Vlasak2"></a>Vlasak M, Blomqvist S, Hovi T, Hewat E, Blaas D (2003) Sequence and structure of human rhinoviruses reveal the basis of receptor discrimination. J Virol  77: 6923–6930.  <ul class="find" data-citedArticleID="1079098" data-doi="10.1128/jvi.77.12.6923-6930.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.12.6923-6930.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+structure+of+human+rhinoviruses+reveal+the+basis+of+receptor+discrimination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+structure+of+human+rhinoviruses+reveal+the+basis+of+receptor+discrimination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001847-Bartlett1" id="pone.0001847-Bartlett1"></a>Bartlett NW, Walton RP, Edwards MR, Aniscenko J, Caramori G, et al.  (2008) Mouse models of rhinovirus-induced disease and exacerbation of allergic airway inflammation. Nature Medicine  14: 199–204.  <ul class="find" data-citedArticleID="1078986" data-doi="10.1038/nm1713"><li><a href="http://dx.doi.org/10.1038/nm1713" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mouse+models+of+rhinovirus-induced+disease+and+exacerbation+of+allergic+airway+inflammation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mouse+models+of+rhinovirus-induced+disease+and+exacerbation+of+allergic+airway+inflammation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001847-Oliveira1" id="pone.0001847-Oliveira1"></a>Oliveira MA, Zhao R, Lee W-M, Kremer MJ, Minor I, et al.  (1993) The structure of human rhinovirus 16. Structure  1: 51–68.  <ul class="find" data-citedArticleID="1079058" data-doi="10.1016/0969-2126(93)90008-5"><li><a href="http://dx.doi.org/10.1016/0969-2126(93)90008-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+human+rhinovirus+16." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+human+rhinovirus+16.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001847-Gama1" id="pone.0001847-Gama1"></a>Gama RE, Horsnell PR, Hughes PJ, North C, Bruce CB, et al.  (1989) Amplification of rhinovirus specific nucleic acids from clinical samples using the polymerase chain reaction. J Med Virol  28: 73–77.  <ul class="find" data-citedArticleID="1079000" data-doi="10.1002/jmv.1890280204"><li><a href="http://dx.doi.org/10.1002/jmv.1890280204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amplification+of+rhinovirus+specific+nucleic+acids+from+clinical+samples+using+the+polymerase+chain+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amplification+of+rhinovirus+specific+nucleic+acids+from+clinical+samples+using+the+polymerase+chain+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001847-Lee1" id="pone.0001847-Lee1"></a>Lee W-M, Kiesner C, Pappas T, Lee I, Grindle K, et al.  (2007) A diverse group of previously unrecognized human rhinoviruses are common causes of respiratory illness in infants. PLoS One  e966:   <ul class="find" data-citedArticleID="1079042" data-doi="10.1371/journal.pone.0000966"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000966" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+diverse+group+of+previously+unrecognized+human+rhinoviruses+are+common+causes+of+respiratory+illness+in+infants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+diverse+group+of+previously+unrecognized+human+rhinoviruses+are+common+causes+of+respiratory+illness+in+infants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001847-Lamson1" id="pone.0001847-Lamson1"></a>Lamson D, Renwick N, Kapoor V, Liu Z, Palacios G, et al.  (2006) MassTag Polymerase-Chain-Reaction Detection of Respiratory Pathogens, Including a New Rhinovirus Genotype, That Caused Influenza-Like Illness in New York State during 2004–2005. J Infect Dis  194: 1398–1402.  <ul class="find" data-citedArticleID="1079034" data-doi="10.1086/508551"><li><a href="http://dx.doi.org/10.1086/508551" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MassTag+Polymerase-Chain-Reaction+Detection+of+Respiratory+Pathogens%2C+Including+a+New+Rhinovirus+Genotype%2C+That+Caused+Influenza-Like+Illness+in+New+York+State+during+2004%E2%80%932005." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MassTag+Polymerase-Chain-Reaction+Detection+of+Respiratory+Pathogens%2C+Including+a+New+Rhinovirus+Genotype%2C+That+Caused+Influenza-Like+Illness+in+New+York+State+during+2004%E2%80%932005.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001847-Kusel1" id="pone.0001847-Kusel1"></a>Kusel MMH, de Klerk NH, Holt PG, Kebadze T, Johnston SL, et al.  (2006) Role of respiratory viruses in acute upper and lower respiratory tract illness in the first year of life. Pediatr Infect Dis J  25: 680–686.  <ul class="find" data-citedArticleID="1079030" data-doi="10.1097/01.inf.0000226912.88900.a3"><li><a href="http://dx.doi.org/10.1097/01.inf.0000226912.88900.a3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+respiratory+viruses+in+acute+upper+and+lower+respiratory+tract+illness+in+the+first+year+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+respiratory+viruses+in+acute+upper+and+lower+respiratory+tract+illness+in+the+first+year+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001847-Blom1" id="pone.0001847-Blom1"></a>Blom N, Hansen J, Blaas D, Brunak S (1996) Cleavage site analysis in picornaviral polyproteins: Discovering cellular targets by neural networks. Prot Sci  5: 2203–2216.  <ul class="find" data-citedArticleID="1078996" data-doi="10.1002/pro.5560051107"><li><a href="http://dx.doi.org/10.1002/pro.5560051107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cleavage+site+analysis+in+picornaviral+polyproteins%3A+Discovering+cellular+targets+by+neural+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cleavage+site+analysis+in+picornaviral+polyproteins%3A+Discovering+cellular+targets+by+neural+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001847-Pettersen1" id="pone.0001847-Pettersen1"></a>Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, et al.  (2004) UCSF chimera-A visualization system for exploratory research and analysis. Journal of computational chemistry  25: 1605–1612.  <ul class="find" data-citedArticleID="1079064" data-doi="10.1002/jcc.20084"><li><a href="http://dx.doi.org/10.1002/jcc.20084" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UCSF+chimera-A+visualization+system+for+exploratory+research+and+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UCSF+chimera-A+visualization+system+for+exploratory+research+and+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001847-Hall1" id="pone.0001847-Hall1"></a>Hall TA (1999) Bioedit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucleic Acids Symp Ser  41: 98.  <ul class="find" data-citedArticleID="1079004"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bioedit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.&amp;auth=&amp;atitle=Bioedit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioedit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioedit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001847-Kumar1" id="pone.0001847-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for molecular evolutionary genetic analysis and sequence alignment. Briefings in bioinformatics  5: 150–163.  <ul class="find" data-citedArticleID="1079028" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+molecular+evolutionary+genetic+analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+molecular+evolutionary+genetic+analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001847-Guex1" id="pone.0001847-Guex1"></a>Guex N, Peitsch MC (1997) SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modelling. Electrophoresis  18: 2714–2723.  <ul class="find" data-citedArticleID="1079002" data-doi="10.1002/elps.1150181505"><li><a href="http://dx.doi.org/10.1002/elps.1150181505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SWISS-MODEL+and+the+Swiss-PdbViewer%3A+an+environment+for+comparative+protein+modelling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SWISS-MODEL+and+the+Swiss-PdbViewer%3A+an+environment+for+comparative+protein+modelling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001847-Shepherd1" id="pone.0001847-Shepherd1"></a>Shepherd CM, Borelli IA, Lander G, Natarajan P, Siddavanahalli V, et al.  (2006) VIPERdb: a relational database for structural virology. Nucleic Acids Res  34: D386–D389.  <ul class="find" data-citedArticleID="1079080" data-doi="10.1093/nar/gkj032"><li><a href="http://dx.doi.org/10.1093/nar/gkj032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VIPERdb%3A+a+relational+database+for+structural+virology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VIPERdb%3A+a+relational+database+for+structural+virology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001847-Sloots1" id="pone.0001847-Sloots1"></a>Sloots TP, Mackay IM, Bialasiewicz S, Jacob KC, McQueen E, et al.  (2008) Human metapneumovirus, Australia, 2001–2004. Emerg Infect Dis  12: 1263–1266.  <ul class="find" data-citedArticleID="1079084" data-doi="10.3201/eid1708.051239"><li><a href="http://dx.doi.org/10.3201/eid1708.051239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+metapneumovirus%2C+Australia%2C+2001%E2%80%932004." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+metapneumovirus%2C+Australia%2C+2001%E2%80%932004.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001847-Wainwright1" id="pone.0001847-Wainwright1"></a>Wainwright C, Altamirano L, Cheney M, Cheney J, Barber S, et al.  (2003) A multicenter, randomized, double-blind, controlled trial of nebulized epinephrine in infants with acute bronchiolitis. The New England journal of medicine  349: 27–35.  <ul class="find" data-citedArticleID="1079100" data-doi="10.1056/nejmoa022226"><li><a href="http://dx.doi.org/10.1056/nejmoa022226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+multicenter%2C+randomized%2C+double-blind%2C+controlled+trial+of+nebulized+epinephrine+in+infants+with+acute+bronchiolitis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+multicenter%2C+randomized%2C+double-blind%2C+controlled+trial+of+nebulized+epinephrine+in+infants+with+acute+bronchiolitis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.XML" value="130568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.PDF" value="735836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g001.PNG_L" value="411663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g001.PNG_M" value="40480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g001.PNG_S" value="7042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g001.TIF" value="680732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g001.PNG_I" value="13749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g002.PNG_L" value="5784841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g002.PNG_M" value="232867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g002.PNG_S" value="21373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g002.TIF" value="7890566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g002.PNG_I" value="194930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g003.PNG_L" value="6720850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g003.PNG_M" value="282598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g003.PNG_S" value="12882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g003.TIF" value="8206092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g003.PNG_I" value="88901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g004.PNG_L" value="1352389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g004.PNG_M" value="97219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g004.PNG_S" value="9255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g004.TIF" value="2397238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g004.PNG_I" value="33164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g005.PNG_L" value="4645603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g005.PNG_M" value="318442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g005.PNG_S" value="13163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g005.TIF" value="6905936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g005.PNG_I" value="98494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g006.PNG_L" value="682191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g006.PNG_M" value="26993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g006.PNG_S" value="12032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g006.TIF" value="1026568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.g006.PNG_I" value="122358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t001.PNG_L" value="78054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t001.PNG_M" value="47070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t001.PNG_S" value="8743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t001.TIF" value="373262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t001.PNG_I" value="14052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t002.PNG_L" value="254605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t002.PNG_M" value="141914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t002.PNG_S" value="13860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t002.TIF" value="1126416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t002.PNG_I" value="43528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t003.PNG_L" value="155045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t003.PNG_M" value="89385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t003.PNG_S" value="11271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t003.TIF" value="737008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001847.t003.PNG_I" value="26827"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001847&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001847" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001847&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001847&volume=&issue=&title=Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, Human Rhinovirus C (HRV C)&author_name=Peter%20McErlean%2C%20Laura%20A.%20Shackelton%2C%20Emily%20Andrews%2C%20Dale%20R.%20Webster%2C%20Stephen%20B.%20Lambert%2C%20Michael%20D.%20Nissen%2C%20Theo%20P.%20Sloots%2C%20Ian%20M.%20Mackay&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001847" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001847" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001847" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001847&amp;t=Distinguishing%20Molecular%20Features%20and%20Clinical%20Characteristics%20of%20a%20Putative%20New%20Rhinovirus%20Species%2C%20Human%20Rhinovirus%20C%20(HRV%20C)" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001847&title=Distinguishing%20Molecular%20Features%20and%20Clinical%20Characteristics%20of%20a%20Putative%20New%20Rhinovirus%20Species%2C%20Human%20Rhinovirus%20C%20(HRV%20C)&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001847&amp;title=Distinguishing%20Molecular%20Features%20and%20Clinical%20Characteristics%20of%20a%20Putative%20New%20Rhinovirus%20Species%2C%20Human%20Rhinovirus%20C%20(HRV%20C)" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001847" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001847&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Distinguishing Molecular Features and Clinical Characteristics of a Putative New Rhinovirus Species, Human Rhinovirus C (HRV C)';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001847';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Distinguishing%20Molecular%20Features%20and%20Clinical%20Characteristics%20of%20a%20Putative%20New%20Rhinovirus%20Species%2C%20Human%20Rhinovirus%20C%20(HRV%20C) http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001847" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001847" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+sequence+analysis%22" title="Search for articles in the subject area:'Amino acid sequence analysis'"><div class="flagText">Amino acid sequence analysis</div></a>
              <div data-categoryid="20173" data-articleid="26518"
                   data-categoryname="Amino acid sequence analysis"
                   class="flagImage" title="Flag 'Amino acid sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+sequencing%22" title="Search for articles in the subject area:'Protein sequencing'"><div class="flagText">Protein sequencing</div></a>
              <div data-categoryid="47503" data-articleid="26518"
                   data-categoryname="Protein sequencing"
                   class="flagImage" title="Flag 'Protein sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+comparison%22" title="Search for articles in the subject area:'Protein structure comparison'"><div class="flagText">Protein structure comparison</div></a>
              <div data-categoryid="20777" data-articleid="26518"
                   data-categoryname="Protein structure comparison"
                   class="flagImage" title="Flag 'Protein structure comparison' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+prediction%22" title="Search for articles in the subject area:'Protein structure prediction'"><div class="flagText">Protein structure prediction</div></a>
              <div data-categoryid="33203" data-articleid="26518"
                   data-categoryname="Protein structure prediction"
                   class="flagImage" title="Flag 'Protein structure prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rhinovirus+infection%22" title="Search for articles in the subject area:'Rhinovirus infection'"><div class="flagText">Rhinovirus infection</div></a>
              <div data-categoryid="49321" data-articleid="26518"
                   data-categoryname="Rhinovirus infection"
                   class="flagImage" title="Flag 'Rhinovirus infection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26518"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26518"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+packaging%22" title="Search for articles in the subject area:'Viral packaging'"><div class="flagText">Viral packaging</div></a>
              <div data-categoryid="21885" data-articleid="26518"
                   data-categoryname="Viral packaging"
                   class="flagImage" title="Flag 'Viral packaging' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3387'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4131'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5764&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=5623">Referee comments: Referee 3</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=3523">Referee comments: Referee 1 (Patrick CY Woo)</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=18035">Pentamer update</a><br>Posted by iamackay</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
