

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Selective Pharmacological Targeting of a DEAD Box RNA Helicase</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001583"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001583" />

    <meta name="citation_title" content="Selective Pharmacological Targeting of a DEAD Box RNA Helicase"/>
    <meta itemprop="name" content="Selective Pharmacological Targeting of a DEAD Box RNA Helicase"/>

      <meta name="citation_author" content="Lisa Lindqvist"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Monika Oberer"/>
            <meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Mikhail Reibarkh"/>
            <meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Regina Cencic"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Marie-Eve Bordeleau"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Emily Vogt"/>
            <meta name="citation_author_institution" content="Department of Biology, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Assen Marintchev"/>
            <meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Junichi Tanaka"/>
            <meta name="citation_author_institution" content="Department of Chemistry, Biology, and Marine Sciences, University of the Ryukyus, Nishihara, Okinawa, Japan"/>
      <meta name="citation_author" content="Francois Fagotto"/>
            <meta name="citation_author_institution" content="Department of Biology, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Michael Altmann"/>
            <meta name="citation_author_institution" content="Institut für Biochemie und Molekulare Medizin, Universität Bern, Bern, Switzerland"/>
      <meta name="citation_author" content="Gerhard Wagner"/>
            <meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Jerry Pelletier"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, McGill University, Montreal, Quebec, Canada"/>
            <meta name="citation_author_institution" content="McGill Cancer Center, McGill University, Montreal, Quebec, Canada"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001583.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1583"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Structure and mechanism of helicases and nucleic acid translocases.; citation_author=MR Singleton; citation_author=MS Dillingham; citation_author=DB wigley; citation_journal_title=Annual Review of Biochemistry; citation_volume=76; citation_number=1; citation_pages=23-50; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The DEAD-box protein family of RNA helicases.; citation_author=O Cordin; citation_author=J Banroques; citation_author=NK Tanner; citation_author=P Linder; citation_journal_title=Gene; citation_volume=367; citation_number=2; citation_pages=17-37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dead-box proteins: a family affair–active and passive players in RNP-remodeling.; citation_author=P Linder; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=3; citation_pages=4168-4180; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DEAD-box RNA helicase subunits of the Drosha complex are required for processing of rRNA and a subset of microRNAs.; citation_author=T Fukuda; citation_author=K Yamagata; citation_author=S Fujiyama; citation_author=T Matsumoto; citation_author=I Koshida; citation_journal_title=Nat Cell Biol; citation_volume=9; citation_number=4; citation_pages=604-611; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Characterization of eukaryotic initiation factor 4A, a protein involved in ATP-dependent binding of globin mRNA.; citation_author=JA Grifo; citation_author=SM Tahara; citation_author=JP Leis; citation_author=MA Morgan; citation_author=AJ Shatkin; citation_journal_title=J Biol Chem; citation_volume=257; citation_number=5; citation_pages=5246-5252; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Involvement of eukaryotic initiation factor 4A in the cap recognition process.; citation_author=I Edery; citation_author=M Humbelin; citation_author=A Darveau; citation_author=KA Lee; citation_author=S Milburn; citation_journal_title=J Biol Chem; citation_volume=258; citation_number=6; citation_pages=11398-11403; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Biochemical and kinetic characterization of the RNA helicase activity of eukaryotic initiation factor 4A.; citation_author=GW Rogers; citation_author=NJ Richter; citation_author=WC Merrick; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=7; citation_pages=12236-12244; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis of a DEAD box RNA helicase: the mammalian translation initiation factor eIF-4A.; citation_author=A Pause; citation_author=N Sonenberg; citation_journal_title=EMBO J; citation_volume=11; citation_number=8; citation_pages=2643-2654; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=ATP-dependent unwinding of messenger RNA structure by eukaryotic initiation factors.; citation_author=BK Ray; citation_author=TG Lawson; citation_author=JC Kramer; citation_author=MH Cladaras; citation_author=JA Grifo; citation_journal_title=J Biol Chem; citation_volume=260; citation_number=9; citation_pages=7651-7658; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Dominant negative mutants of mammalian translation initiation factor eIF-4A define a critical role for eIF-4F in cap-dependent and cap-independent initiation of translation.; citation_author=A Pause; citation_author=N Methot; citation_author=Y Svitkin; citation_author=WC Merrick; citation_author=N Sonenberg; citation_journal_title=EMBO J; citation_volume=13; citation_number=10; citation_pages=1205-1215; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The requirement for eukaryotic initiation factor 4A (elF4A) in translation is in direct proportion to the degree of mRNA 5' secondary structure.; citation_author=YV Svitkin; citation_author=A Pause; citation_author=A Haghighat; citation_author=S Pyronnet; citation_author=G Witherell; citation_journal_title=RNA; citation_volume=7; citation_number=11; citation_pages=382-394; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The p46 subunit of eukaryotic initiation factor (eIF)-4F exchanges with eIF-4A.; citation_author=J Yoder-Hill; citation_author=A Pause; citation_author=N Sonenberg; citation_author=WC Merrick; citation_journal_title=J Biol Chem; citation_volume=268; citation_number=12; citation_pages=5566-5573; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Characterization of the 46,000-dalton subunit of eIF-4F.; citation_author=SC Conroy; citation_author=TE Dever; citation_author=CL Owens; citation_author=WC Merrick; citation_journal_title=Arch Biochem Biophys; citation_volume=282; citation_number=13; citation_pages=363-371; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=eIF4A3 is a novel component of the exon junction complex.; citation_author=CC Chan; citation_author=J Dostie; citation_author=MD Diem; citation_author=W Feng; citation_author=M Mann; citation_journal_title=RNA; citation_volume=10; citation_number=14; citation_pages=200-209; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A nuclear translation-like factor eIF4AIII is recruited to the mRNA during splicing and functions in nonsense-mediated decay.; citation_author=MA Ferraiuolo; citation_author=CS Lee; citation_author=LW Ler; citation_author=JL Hsu; citation_author=M Costa-Mattioli; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=15; citation_pages=4118-4123; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The transformation suppressor Pdcd4 is a novel eukaryotic translation initiation factor 4A binding protein that inhibits translation.; citation_author=HS Yang; citation_author=AP Jansen; citation_author=AA Komar; citation_author=X Zheng; citation_author=WC Merrick; citation_journal_title=Mol Cell Biol; citation_volume=23; citation_number=16; citation_pages=26-37; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=S6K1- and betaTRCP-mediated degradation of PDCD4 promotes protein translation and cell growth.; citation_author=NV Dorrello; citation_author=A Peschiaroli; citation_author=D Guardavaccaro; citation_author=NH Colburn; citation_author=NE Sherman; citation_journal_title=Science; citation_volume=314; citation_number=17; citation_pages=467-471; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stimulation of mammalian translation initiation factor eIF4A activity by a small molecule inhibitor of eukaryotic translation.; citation_author=ME Bordeleau; citation_author=J Matthews; citation_author=JM Wojnar; citation_author=L Lindqvist; citation_author=O Novac; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=18; citation_pages=10460-10465; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inhibition of eukaryotic translation initiation by the marine natural product pateamine A.; citation_author=WK Low; citation_author=Y Dang; citation_author=T Schneider-Poetsch; citation_author=Z Shi; citation_author=NS Choi; citation_journal_title=Mol Cell; citation_volume=20; citation_number=19; citation_pages=709-722; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional Characterization of IRESes by an inhibitor of the RNA helicase eIF4A.; citation_author=M-E Bordeleau; citation_author=A Mori; citation_author=M Oberer; citation_author=L Lindqvist; citation_author=LS Chard; citation_journal_title=Nat Chem Biol; citation_volume=2; citation_number=20; citation_pages=213-220; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Substrate-Dependent Targeting of Eukaryotic Translation Initiation Factor 4A by Pateamine A: Negation of Domain-Linker Regulation of Activity.; citation_author=WK Low; citation_author=Y Dang; citation_author=S Bhat; citation_author=D Romo; citation_author=JO Liu; citation_journal_title=Chem Biol; citation_volume=14; citation_number=21; citation_pages=715-727; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structure of the exon junction core complex with a trapped DEAD-box ATPase bound to RNA.; citation_author=CB Andersen; citation_author=L Ballut; citation_author=JS Johansen; citation_author=H Chamieh; citation_author=KH Nielsen; citation_journal_title=Science; citation_volume=313; citation_number=22; citation_pages=1968-1972; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The crystal structure of the exon junction complex reveals how it maintains a stable grip on mRNA.; citation_author=F Bono; citation_author=J Ebert; citation_author=E Lorentzen; citation_author=E Conti; citation_journal_title=Cell; citation_volume=126; citation_number=23; citation_pages=713-725; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structural basis for the enhancement of eIF4A helicase activity by eIF4G.; citation_author=M Oberer; citation_author=A Marintchev; citation_author=G Wagner; citation_journal_title=Genes Dev; citation_volume=19; citation_number=24; citation_pages=2212-2223; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural basis for RNA unwinding by the DEAD-box protein Drosophila Vasa.; citation_author=T Sengoku; citation_author=O Nureki; citation_author=A Nakamura; citation_author=S Kobayashi; citation_author=S Yokoyama; citation_journal_title=Cell; citation_volume=125; citation_number=25; citation_pages=287-300; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic profiling of drug sensitivities via induced haploinsufficiency.; citation_author=G Giaever; citation_author=DD Shoemaker; citation_author=TW Jones; citation_author=H Liang; citation_author=EA Winzeler; citation_journal_title=Nat Genet; citation_volume=21; citation_number=26; citation_pages=278-283; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=RNA aptamers to initiation factor 4A helicase hinder cap-dependent translation by blocking ATP hydrolysis.; citation_author=A Oguro; citation_author=T Ohtsu; citation_author=YV Svitkin; citation_author=N Sonenberg; citation_author=Y Nakamura; citation_journal_title=RNA; citation_volume=9; citation_number=27; citation_pages=394-407; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The roles of individual eukaryotic translation initiation factors in ribosomal scanning and initiation codon selection.; citation_author=TV Pestova; citation_author=VG Kolupaeva; citation_journal_title=Genes Dev; citation_volume=16; citation_number=28; citation_pages=2906-2922; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis of human eIF4AIII identifies regions necessary for exon junction complex formation and nonsense-mediated mRNA decay.; citation_author=T Shibuya; citation_author=TO Tange; citation_author=ME Stroupe; citation_author=MJ Moore; citation_journal_title=RNA; citation_volume=12; citation_number=29; citation_pages=360-374; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=NMRPipe: a multidimensional spectral processing system based on UNIX pipes.; citation_author=F Delaglio; citation_author=S Grzesiek; citation_author=GW Vuister; citation_author=G Zhu; citation_author=J Pfeifer; citation_journal_title=J Biomol NMR; citation_volume=6; citation_number=30; citation_pages=277-293; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The program XEASY for computer-supported NMR spectral analysis of biological molecules.; citation_author=C Bartels; citation_author=T Xia; citation_author=M Billeter; citation_author=P Güntert; citation_author=K Wüthrich; citation_journal_title=J Biomol NMR; citation_volume=5; citation_number=31; citation_pages=1-10; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Structure of translation factor eIF4E bound to m7GDP and interaction with 4E-binding protein.; citation_author=H Matsuo; citation_author=H Li; citation_author=AM McGuire; citation_author=CM Fletcher; citation_author=AC Gingras; citation_journal_title=Nat Struct Biol; citation_volume=4; citation_number=32; citation_pages=717-724; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=eIF4AIII binds spliced mRNA in the exon junction complex and is essential for nonsense-mediated decay.; citation_author=T Shibuya; citation_author=TO Tange; citation_author=N Sonenberg; citation_author=MJ Moore; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=33; citation_pages=346-351; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The DEAD box protein eIF4A. 1. A minimal kinetic and thermodynamic framework reveals coupled binding of RNA and nucleotide.; citation_author=JR Lorsch; citation_author=D Herschlag; citation_journal_title=Biochemistry; citation_volume=37; citation_number=34; citation_pages=2180-2193; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=ATP/Mg++-dependent cross-linking of cap binding proteins to the 5' end of eukaryotic mRNA.; citation_author=N Sonenberg; citation_journal_title=Nucleic Acids Res; citation_volume=9; citation_number=35; citation_pages=1643-1656; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Bidirectional RNA helicase activity of eucaryotic translation initiation factors 4A and 4F.; citation_author=F Rozen; citation_author=I Edery; citation_author=K Meerovitch; citation_author=TE Dever; citation_author=WC Merrick; citation_journal_title=Mol Cell Biol; citation_volume=10; citation_number=36; citation_pages=1134-1144; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Cell Free Translation Systems; Spirin AS, editor.; citation_author=M Altmann; citation_author=H Trachsel; citation_number=37; citation_date=2002; citation_publisher=Springer-Verlag Berlin; " />
      <meta name="citation_reference" content="citation_title=A 250-nucleotide UA-rich element in the 3' untranslated region of Xenopus laevis Vg1 mRNA represses translation both in vivo and in vitro.; citation_author=LJ Otero; citation_author=A Devaux; citation_author=N Standart; citation_journal_title=RNA; citation_volume=7; citation_number=38; citation_pages=1753-1767; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Homogenous time resolved fluorescence assay to identify modulators of cap-dependent translation initiation.; citation_author=R Cencic; citation_author=Y Yan; citation_author=J Pelletier; citation_journal_title=Comb Chem High Throughput Screen; citation_volume=10; citation_number=39; citation_pages=181-188; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=DEAD-box proteins: the driving forces behind RNA metabolism.; citation_author=S Rocak; citation_author=P Linder; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=5; citation_number=40; citation_pages=232-241; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A new yeast translation initiation factor suppresses a mutation in the eIF-4A RNA helicase.; citation_author=R Coppolecchia; citation_author=P Buser; citation_author=A Stotz; citation_author=P Linder; citation_journal_title=EMBO J; citation_volume=12; citation_number=41; citation_pages=4005-4011; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001583" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Selective Pharmacological Targeting of a DEAD Box RNA Helicase"/>
    <meta name="twitter:description" content="RNA helicases represent a large family of proteins implicated in many biological processes including ribosome biogenesis, splicing, translation and mRNA degradation. However, these proteins have little substrate specificity, making inhibition of selected helicases a challenging problem. The prototypical DEAD box RNA helicase, eIF4A, works in conjunction with other translation factors to prepare mRNA templates for ribosome recruitment during translation initiation. Herein, we provide insight into the selectivity of a small molecule inhibitor of eIF4A, hippuristanol. This coral-derived natural product binds to amino acids adjacent to, and overlapping with, two conserved motifs present in the carboxy-terminal domain of eIF4A. Mutagenesis of amino acids within this region allowed us to alter the hippuristanol-sensitivity of eIF4A and undertake structure/function studies. Our results provide an understanding into how selective targeting of RNA helicases for pharmacological intervention can be achieved."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001583.g006"/>

  <meta property="og:title" content="Selective Pharmacological Targeting of a DEAD Box RNA Helicase" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001583" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5865'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5381'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3583&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001583">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001583" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001583&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Selective Pharmacological Targeting of a DEAD Box RNA Helicase
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lisa Lindqvist, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Monika Oberer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mikhail Reibarkh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Regina Cencic, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marie-Eve Bordeleau, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Emily Vogt, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Assen Marintchev, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Junichi Tanaka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Chemistry, Biology, and Marine Sciences, University of the Ryukyus, Nishihara, Okinawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Francois Fagotto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Altmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut für Biochemie und Molekulare Medizin, Universität Bern, Bern, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gerhard Wagner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jerry Pelletier
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:jerry.pelletier@mcgill.ca">jerry.pelletier@mcgill.ca</a></p>

                <p>Affiliations:
                  Department of Biochemistry, McGill University, Montreal, Quebec, Canada, 
                  McGill Cancer Center, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001583</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001583" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001583" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001583" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001583" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001583-g001" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001583-g002" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001583-g003" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001583-g004" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001583-g005" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001583-g006" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001583">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001583" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>RNA helicases represent a large family of proteins implicated in many biological processes including ribosome biogenesis, splicing, translation and mRNA degradation. However, these proteins have little substrate specificity, making inhibition of selected helicases a challenging problem. The prototypical DEAD box RNA helicase, eIF4A, works in conjunction with other translation factors to prepare mRNA templates for ribosome recruitment during translation initiation. Herein, we provide insight into the selectivity of a small molecule inhibitor of eIF4A, hippuristanol. This coral-derived natural product binds to amino acids adjacent to, and overlapping with, two conserved motifs present in the carboxy-terminal domain of eIF4A. Mutagenesis of amino acids within this region allowed us to alter the hippuristanol-sensitivity of eIF4A and undertake structure/function studies. Our results provide an understanding into how selective targeting of RNA helicases for pharmacological intervention can be achieved.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lindqvist L, Oberer M, Reibarkh M, Cencic R, Bordeleau M-E, et al.  (2008) Selective Pharmacological Targeting of a DEAD Box RNA Helicase. PLoS ONE 3(2):
          e1583.
            doi:10.1371/journal.pone.0001583</p><p><strong>Academic Editor: </strong>Thomas Preiss, Victor Chang Cardiac Research Institute, Australia</p><p><strong>Received:</strong> December 18, 2007; <strong>Accepted:</strong> January 13, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Lindqvist et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>LL was supported by a CIHR Chemical Biology and an NSERC CGSM fellowship. RC was supported by a CIHR Cancer Consortium Post-doctoral Training Grant Award. MO was supported by a post-doctoral fellowship from the Max Kade Foundation. This work was supported by a grant from NCIC (#017099) to JP and an NIH grant (#CA 68262) to GW.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Helicases and translocases are classified into 6 superfamilies (SF1–SF6) based on the arrangement of conserved sequence motifs, with many providing essential functions in nucleic acid metabolic processes <a href="#pone.0001583-Singleton1">[1]</a>. Members of the SF2 family consist of RNA helicases implicated in transcription, RNA export, splicing, translation, ribosome biogenesis, miRNA processing, and RNA decay <a href="#pone.0001583-Cordin1">[2]</a>–<a href="#pone.0001583-Fukuda1">[4]</a>. Eukaryotic initiation factor (eIF) 4A is one of the archetypical founding members of the DEAD box helicase family, the largest subclass of the SF2 family. eIF4A is an abundant translation factor that exists in free form (referred to herein as eIF4A<sub>f</sub>) or as a subunit of the heterotrimeric cap binding complex, eIF4F (referred to herein as eIF4A<sub>c</sub>) <a href="#pone.0001583-Grifo1">[5]</a>, <a href="#pone.0001583-Edery1">[6]</a>. It participates in the ribosome recruitment phase of translation and is delivered to the cap structure (m7GpppN, where N is any nucleotide) of mRNA templates as a subunit of eIF4F. It is thought to prepare the mRNA template for 43S pre-initiation complex (40S ribosome and associated factors) binding by unwinding local secondary structure. The helicase activity of eIF4A<sub>c</sub> is ~20-fold more efficient than eIF4A<sub>f </sub><a href="#pone.0001583-Rogers1">[7]</a>, <a href="#pone.0001583-Pause1">[8]</a> and during initiation eIF4A<sub>f</sub> is thought to cycle through the eIF4F complex <a href="#pone.0001583-Ray1">[9]</a>–<a href="#pone.0001583-YoderHill1">[12]</a>. There are two highly related isoforms, eIF4AI and eIF4AII (85–90% sequence identity) which are thought to be functionally interchangeable for translation initiation <a href="#pone.0001583-YoderHill1">[12]</a>, <a href="#pone.0001583-Conroy1">[13]</a>. A third protein, called eIF4AIII (DDX48), has ~65% sequence identity to eIF4AI and is part of the exon junction complex that participates in nonsense mediated decay <a href="#pone.0001583-Chan1">[14]</a>, <a href="#pone.0001583-Ferraiuolo1">[15]</a>. The helicase activity of eIF4A is inhibited when associated with the tumor suppressor gene product, Pdcd4, an event that is regulated by the mammalian target of rapamycin (mTOR) <a href="#pone.0001583-Yang1">[16]</a>, <a href="#pone.0001583-Dorrello1">[17]</a>. This underscores an important link between cellular homeostasis and translational control at the level of eIF4A availability.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In a screen aimed at identifying novel inhibitors of translation initiation, we identified and characterized two marine-derived natural products, pateamine and hippuristanol, that modulate eIF4A activity <a href="#pone.0001583-Bordeleau1">[18]</a>–<a href="#pone.0001583-Bordeleau2">[20]</a>. The binding site of pateamine on eIF4A is not defined, although its activity is dependent on the nature of the linker region joining the amino-terminal (NTD) and carboxy-terminal domains (CTD), a region with significant sequence variation among DEAD-box family members <a href="#pone.0001583-Low2">[21]</a>. On the other hand, hippuristanol interacts with eIF4AI-CTD (residues 237–406) and blocks the RNA-dependent ATPase, RNA binding, and helicase activities of eIF4AI <a href="#pone.0001583-Bordeleau2">[20]</a>. Herein, we define the hippuristanol-binding site on eIF4A. The site displays extensive sequence variation among DEAD box RNA helicases and provides a framework for understanding the selectivity of hippuristanol. We utilize this information to generate eIF4A alleles with reduced sensitivity to this small molecule and capable of rescuing hippuristanol-induced inhibition of translation. This allowed us to probe structure-function relationships of eIF4A in translation.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Defining the eIF4A hippuristanol binding site</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To identify the amino acids involved in hippuristanol binding, a series of NMR experiments were undertaken in which <sup>1</sup>H-<sup>15</sup>N-HSQC spectra of uniformly labelled eIF4AI-CTD were obtained in the absence or presence of compound (<a href="#pone-0001583-g001">Fig. 1A</a>). Residues that experienced significant chemical shift changes (&gt;mean plus standard deviation) are indicated in grey whereas those displaying direct NOE contacts (&lt;5Å) are highlighted in yellow (<a href="#pone-0001583-g001">Fig. 1B</a>). Hippuristanol binds directly (exhibits NOEs) to the N-terminal ends of β-strands E5 and E6, the C-terminal end of helix H4, as well as the loop regions adjacent to these secondary structural elements (<a href="#pone-0001583-g001">Figs. 1A–C</a>). Furthermore, adjacent regions undergo significant chemical shift changes (<a href="#pone-0001583-g001">Fig. 1C</a>; highlighted in blue).</p>
<div class="figure" id="pone-0001583-g001"><div class="img"><a name="pone-0001583-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Hippuristanol binds to eIF4AI-CTD.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Chemical shift changes of <sup>1</sup>H-<sup>15</sup>N-HSQC peaks, (Δδ(<sup>1</sup>H)+0.2 Δδ(<sup>15</sup>N), of eIF4A-I-CTD (52 µM) upon addition of hippuristanol (100 µM). Free and bound forms are in slow exchange and the resonances of eIF4AI-CTD had to be assigned in both states. The locations of secondary structures were identified by NMR and are indicated with magenta arrows (β-strands) and yellow rectangles (helices). (B) Primary amino acid sequence of eIF4AI indicating residues involved in hippuristanol binding. NOEs are highlighted in yellow, whereas those within 5Å are in grey and correspond to regions a, b, and c in A. Residues in bold denote conserved amino acids that define motifs V (ARGID) and VI (HRIGRGGRFG) of DEAD box family members <a href="#pone.0001583-Rocak1">[40]</a>. Arrows denote residues identified in Vasa that interact with ATP (red), RNA (blue), or are involved in interdomain interaction (green)<a href="#pone.0001583-Sengoku1">[25]</a>. (C) Surface and ribbon representations of the model for eIF4AI-CTD. The CTD is viewed from the position of the NTD. Residues of eIF4AI-CTD that show NOEs to hippuristanol are coloured yellow, those exhibiting major chemical shift changes but no NOEs are coloured blue. Residues contacting eIF4G are in red <a href="#pone.0001583-Oberer1">[24]</a>. The β-sheets (E1–E6) and α-helices (H1–H6) are labelled and refer to the locations marked in A. RNA and ADPNP are shown as sticks models. (D) Location of the hippuristanol-binding site in a model for eIF4AI complexed with RNA and ADPNP. The model is composed of the crystal structure of human eIF4AI-NTD (PDB #2G9N) and the homology model of the eIF4AI-CTD <a href="#pone.0001583-Oberer1">[24]</a>. The two domains are aligned to the structure of eIF4AIII from the EJC from which the RNA and ADPNP binding sites are adapted (PDB# 2HYI) <a href="#pone.0001583-Andersen1">[22]</a>. Color scheme of amino acid residues is as in C.</p>
<span>doi:10.1371/journal.pone.0001583.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We analyzed the position of the hippuristanol binding site in the context of a model based on the domain orientation of eIF4AIII in the exon junction complex (EJC) (PDB code 2HYI) <a href="#pone.0001583-Andersen1">[22]</a>, <a href="#pone.0001583-Bono1">[23]</a> (<a href="#pone-0001583-g001">Fig. 1D</a>). Here, the eIF4AI NTD crystal structure (PDB #2G9N) was used and the eIF4AIII-CTD was replaced with the homology model for eIF4AI <a href="#pone.0001583-Oberer1">[24]</a>. The positions of the RNA and ADPNP are taken from the EJC <a href="#pone.0001583-Andersen1">[22]</a>. Accordingly, the hippuristanol-binding site on the eIF4AI-CTD is directly adjacent to the ATP-binding site in the NTD. Since hippuristanol does not inhibit ATP crosslinking to eIF4A <a href="#pone.0001583-Bordeleau2">[20]</a>, it may perturb the interface between the NTD and CTD domains. The hippuristanol-binding site is far from the RNA-binding face and allows us to conclude that hippuristanol inhibits eIF4A RNA binding in an allosteric manner.</p>


<h4>Selectivity of hippuristanol for eIF4A</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>With the exception of R247<sup>eIF4AI</sup> and T328<sup>eIF4AI/II</sup>, all the hippuristanol binding residues are present in murine eIF4AI, eIF4AII, and the yeast eIF4A homolog Tif1/2p (<a href="#pone.0001583.s002">Fig. S1</a>). This hippuristanol binding site however is not conserved in eIF4AIII and we note 7 amino acid differences (<a href="#pone.0001583.s002">Fig. S1</a>; R247<sup>eIF4AI</sup> is changed to K252<sup>eIF4AIII</sup>, T328<sup>eIF4AI</sup> to S333<sup>eIF4AIII</sup>, L331<sup>eIF4AI</sup> to V336<sup>eIF4AIII</sup>, L332<sup>eIF4AI</sup> to W337<sup>eIF4AIII</sup>, I336<sup>eIF4AI</sup> to L341<sup>eIF4AIII</sup>, Q339<sup>eIF4AI</sup> to P344<sup>eIF4AIII</sup>, and V344<sup>eIF4AI</sup> to I349<sup>eIF4AIII</sup>). Among the residues that differ between eIF4AI and eIF4AIII, the L341<sup>eIF4AIII </sup>, P344<sup>eIF4AIII</sup> and I349<sup>eIF4AIII</sup> changes are expected to directly impact on hippuristanol binding since the corresponding side chains are part of the binding pocket (<a href="#pone.0001583.s002">Fig. S1</a>). The side chains of the other substitutions do not point towards the binding site and are not expected to impact on hippuristanol affinity.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We compared the relative sensitivities of murine eIF4AI, murine eIF4AII, and human eIF4AIII to hippuristanol in an RNA-dependent ATPase assay (<a href="#pone-0001583-g002">Fig. 2A</a>). eIF4AI and eIF4AII showed similar sensitivities to hippuristanol, whereas eIF4AIII required ~10-fold higher concentrations of compound to achieve equivalent inhibition (<a href="#pone-0001583-g002">Fig. 2A</a>). Tif1/2p ATPase activity displayed a similar sensitivity to hippuristanol as eIF4AI/II (data not shown), consistent with the conserved nature of the amino acids flanking motifs V and VI (<a href="#pone.0001583.s002">Fig. S1</a>). Consistent with these results, 10 µM hippuristanol inhibited RNA binding of eIF4AI, but not eIF4AIII (<a href="#pone-0001583-g002">Fig. 2B</a>, compare lane 2 to 1 and lane 4 to 3).</p>
<div class="figure" id="pone-0001583-g002"><div class="img"><a name="pone-0001583-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Selectivity of hippuristanol for eIF4A.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Inhibition of eIF4A RNA-dependent ATPase activity by hippuristanol. ATPase assays were performed with 0.1 µg of His<sub>6</sub>-eIF4AI or His<sub>6</sub>-eIF4AII at 25°C or with 0.1 µg of His<sub>6</sub>-eIF4AIII at 37°C for 2 h with 0.1 µCi γ-<sup>32</sup>P-ATP (10 Ci/mmol). Following analysis by TLC and quantitation using a Fuji BAS 2000 phosphoimager, the percent hydrolysis was determined and set relative to the DMSO vehicle control reactions. Each value represents the average of three measurements with the error shown as the standard deviation. (B) Crosslinking of recombinant proteins to RNA in the presence of hippuristanol. <sup>32</sup>P-labelled CAT RNA was cross-linked to 0.5–1 µg of the indicated recombinant protein in the presence or absence of hippuristanol, separated by SDS-PAGE, and visualized by autoradiography. [Note that in our hands, recombinant hDDX52 did not crosslink to RNA.] (C) Relative ATPase activity of eIF4AI, hDDX19, and hDDX52 in the presence of 50 µM hippuristanol. eIF4AI and hDDX19 where performed at 25°C for 5 minutes while hDDX52 was incubated for 60 minutes to allow for analysis to be in the linear range of ATP hydrolysis. The percent ATP hydrolysis was determined in the presence of hippuristanol and set relative to the DMSO vehicle control reactions. The results represent the average of 3 experiments with error bars signifying the standard deviation.</p>
<span>doi:10.1371/journal.pone.0001583.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Alignment of the region encompassing the hippuristanol binding site of murine (<a href="#pone.0001583.s003">Fig. S2A</a>) and human (<a href="#pone.0001583.s003">Fig. S2B</a>) DEAD box helicases indicates extensive sequence variation. Human DDX52 has a very high degree of conservation with eIF4AI in the hippuristanol binding region, with two changes present among the amino acids showing direct NOE contacts (<a href="#pone.0001583.s003">Fig. S2B</a>). Eleven of 17 amino acids (from regions encompassing motifs V and VI) showing weaker NOEs are also present in hDDX52 (<a href="#pone.0001583.s003">Fig. S2B</a>). hDDX19 contains five of the eight amino acids showing direct NOE contacts and 11 of the 17 amino acids showing weaker NOEs (<a href="#pone.0001583.s003">Fig. S2B</a>). Hippuristanol did not inhibit the RNA binding properties of hDDX19 (<a href="#pone-0001583-g002">Fig. 2B</a>) nor the ATPase activity of hDDX52 or hDDX19 (<a href="#pone-0001583-g002">Fig. 2C</a>). These results provide insight into why hippuristanol is selective for eIF4A since the amino acids that define the hippuristanol binding site are not well conserved among other DDX family members.</p>


<h4>Modulating eIF4A Hippuristanol Sensitivity</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Using the mapping information from these NMR studies, we addressed the feasibility of modulating hippuristanol sensitivity among eIF4A family members (<a href="#pone.0001583.s002">Fig. S1</a>). Given that Ded1p is resistant to inhibition by hippuristanol (Hipp<sup>R</sup>), we used information obtained from the sequence comparison of the hippuristanol binding site to guide us in our mutagenesis approach <a href="#pone.0001583-Bordeleau2">[20]</a> (<a href="#pone.0001583.s002">Fig. S1</a>). The <sup>338</sup>VQ<sup>339</sup> eIF4AI amino acid pair immediately downstream of motif V in eIF4A was altered to <sup>338</sup>IP<sup>339</sup> (present in Ded1p) or <sup>338</sup>IG<sup>339</sup> (for future NMR studies; since proline residues do not have amide protons and are not visible in <sup>1</sup>H-<sup>15</sup>N HSQC spectra). These mutants also harboured a G363T alteration in motif VI (eIF4AI<sup>IG/T</sup>), so we generated mutants harbouring only either a G363T or a <sup>338</sup>VQ<sup>339 </sup>to <sup>338</sup>IG<sup>339</sup> alteration (<a href="#pone.0001583.s002">Fig. S1</a>; eIF4AI<sup>IG</sup> and eIF4AI<sup>T</sup>). IP/T variants of eIF4AII and eIF4AIII were also generated (<a href="#pone.0001583.s002">Fig. S1</a>). We also addressed whether we could increase the sensitivity of eIF4AIII to hippuristanol by rebuilding a complete hippursitanol site (<a href="#pone.0001583.s002">Fig. S1</a>; eIF4AIII<sup>TLLQV</sup>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>eIF4AI<sup>IG/T</sup> and eIF4AI<sup>IP/T</sup> are more active than eIF4AI in an RNA-dependent ATPase assay (<a href="#pone-0001583-g003">Fig. 3A</a> and data not shown). The increased ATPase activity was principally a consequence of the <sup>338</sup>VQ<sup>339</sup> to <sup>338</sup>IG<sup>339 </sup>alteration (<a href="#pone.0001583.s004">Fig. S3</a>). Hippuristanol inhibited eIF4AI ATPase activity but had little effect on the ATPase activity of eIF4AI<sup>IG/T</sup> and eIF4AI<sup>IP/T</sup> (<a href="#pone-0001583-g003">Fig. 3A and B</a>). Similarly, eIF4AII was sensitive to inhibition by hippuristanol whereas eIF4AII<sup>IP/T</sup> was not (<a href="#pone-0001583-g003">Fig. 3B</a>). Titration of hippuristanol revealed that the ATPase activity of eIF4AIII<sup>IP/T </sup>was resistant to hippuristanol at concentrations up to 75 µM with a slight inhibition of activity at 100 µM (<a href="#pone-0001583-g003">Fig. 3C</a>). In contrast, eIF4AIII<sup>TLLQV</sup> was more sensitive to hippuristanol than eIF4AIII (<a href="#pone-0001583-g003">Fig. 3C</a>), showing a dose-response profile that resembled that of eIF4AI and eIF4AII (<a href="#pone-0001583-g002">Fig. 2A</a>). We further characterized the eIF4AI and eIF4AII hippuristanol-resistant alleles in RNA binding and helicase assays. As expected, hippuristanol reduced the ability of eIF4AI and eIF4AII to interact with RNA (<a href="#pone.0001583.s005">Fig. S4A</a>, compare lanes 2 and 8 to 1 and 7, respectively) but did not affect the RNA binding activities of eIF4AI<sup>IG/T</sup>, eIF4AI<sup>IP/T</sup>, and eIF4AII<sup>IP/T</sup> (<a href="#pone.0001583.s005">Fig. S4A</a>, compare lanes 4, 6, and 10 to 3, 5, and 9, respectively). The helicase activity of eIF4AI and eIF4AII is blocked by hippuristanol, whereas both eIF4AI<sup>IG/T </sup>and eIF4AII<sup>IP/T </sup>are resistant to inhibition (<a href="#pone.0001583.s005">Fig. S4B</a>). Taken together, these results demonstrate the feasibility of generating mutant alleles of DEAD-box helicase members with increased or reduced sensitivity to hippuristanol. This provides a powerful means by which to investigate the function of individual members of this family of proteins.</p>
<div class="figure" id="pone-0001583-g003"><div class="img"><a name="pone-0001583-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Characterization of eIF4A hippuristanol-resistant mutants.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Consequence of mutations in the eIF4AI hippuristanol binding site on ATP hydrolysis. ATP hydrolysis was monitored using 1 µg His<sub>6</sub>-eIF4AI or His<sub>6</sub>-eIF4AI<sup>IG/T</sup> in the presence or absence of 10 µM hippuristanol. Each value represents the average of three measurements with the standard deviation presented. (B) Relative ATPase activity of eIF4A mutants in the presence of hippuristanol. The percent ATP hydrolysis was determined in the presence of hippuristanol and set relative to the values obtained in the presence of control reactions containing vehicle (DMSO). The results represent the average of 3–7 experiments with error bars signifying the standard deviation. (C) Altered hippuristanol sensitivity of eIF4AIII. ATPase assays were performed with 0.5 µg recombinant protein with 0.1 µCi γ-<sup>32</sup>P-ATP (10 Ci/mmol). Following analysis by TLC and quantitation using a Fuji BAS 2000 phosphoimager, the percent hydrolysis was determined and set relative to the DMSO vehicle control reactions. Each value represents the average of three measurements with the standard deviation shown.</p>
<span>doi:10.1371/journal.pone.0001583.g003</span></div>

<h4>Structure/Function Studies of eIF4A</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We used the ability to generate hippuristanol-resistant alleles of eIF4A to probe structure-function relationships <em>in vitro</em>. Specifically, we asked: (i) if the helicase activity of eIF4AI is required for translation (or is its ATPase activity sufficient); (ii) if eIF4A:eIF4G interaction is essential for translation, and (iii) whether eIF4AI and eIF4AII are functionally interchangeable. The design of a helicase deficient mutant of eIF4A, eIF4AI<sup>Hel/IG/T</sup>, was guided by a previously described Vasa mutation in which this alteration abolished helicase activity but only reduced ATPase activity by 50% (<a href="#pone.0001583.s002">Fig. S1</a>) <a href="#pone.0001583-Sengoku1">[25]</a>. eIF4AI<sup>Quad/IG/T</sup> contains 4 missense mutations previously shown to inhibit interaction with eIF4G <a href="#pone.0001583-Oberer1">[24]</a>. eIF4AI<sup>Hel/IG/T</sup> showed a reduction in the rate of RNA-dependent ATP hydrolysis compared to eIF4AI<sup>IG/T</sup> - bringing it to levels similar to wild-type eIF4AI (<a href="#pone.0001583.s006">Fig. S5A</a>). As expected, eIF4AI<sup>Hel/IG/T</sup> does not display helicase activity (<a href="#pone.0001583.s006">Fig. S5B</a>, compare lanes 5 to 3), and its RNA binding activity is resistant to hippuristanol (<a href="#pone.0001583.s006">Fig. S5C</a>). eIF4AI<sup>Quad/IG/T</sup> has a similar rate of RNA-dependent ATP hydrolysis as eIF4AI<sup>IG/T</sup> (<a href="#pone.0001583.s006">Fig. S5A</a>), possesses helicase activity (<a href="#pone.0001583.s006">Fig. S5D</a>), and its RNA binding activity is resistant to hippuristanol (<a href="#pone.0001583.s006">Fig. S5C</a>). As predicted, it is impaired in its ability to interact with eIF4GI (<a href="#pone.0001583.s007">Fig. S6</a>).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We tested whether the Hipp<sup>R</sup> eIF4A alleles could rescue translation when this process is inhibited with hippuristanol (<a href="#pone-0001583-g004">Fig. 4</a>). <em>In vitro</em> translations were performed in rabbit reticulocyte lysate (RRL) programmed with the bicistronic reporter mRNA FF/HCV/Ren (<a href="#pone-0001583-g004">Fig. 4A</a>) <a href="#pone.0001583-Bordeleau2">[20]</a>. Here, <em>Renilla</em> (Ren) luciferase expression is HCV-driven and not eIF4A-dependent <a href="#pone.0001583-Bordeleau2">[20]</a>, thus serving as an internal control. Firefly (FF) luciferase expression is inhibited by &gt;90% in the presence of 5 µM hippuristanol, whereas that of <em>Renilla</em> is slightly reduced (<a href="#pone-0001583-g004">Fig. 4B</a>, compare lane 2 to 1). <em>Renilla</em> luciferase RLU readings from the translation products of this experiment are consistent with a 2-fold reduction in activity (LL, data not shown). Addition of wild-type eIF4AI does not rescue the inhibition by hippuristanol, whereas eIF4AI<sup>IG/T </sup>restored translation to ~60% of normal levels (<a href="#pone-0001583-g004">Fig. 4B</a>, compare lanes 6 and 4 to 5 and 3, respectively). These results are consistent with the idea that inhibition of translation by hippuristanol <em>in vitro</em> is a direct consequence of impaired eIF4A activity. Neither eIF4AI<sup>Quad/IG/T</sup> or eIF4AI<sup>Hel/IG/T</sup> are able to rescue translation inhibition by hippuristanol (<a href="#pone-0001583-g004">Fig. 4B</a>) indicating that eIF4A’s helicase activity and its ability to interact with eIF4G are necessary for its role in translation.</p>
<div class="figure" id="pone-0001583-g004"><div class="img"><a name="pone-0001583-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Functional requirements for eIF4A activity in translation.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Schematic representation of the reporter construct used in these studies is shown on top. (B) Left Panel: Rescue of hippuristanol-induced translation inhibition by eIF4AI<sup>IG/T</sup>. <em>In vitro</em> translations in RRL programmed with capped FF/HCV/Ren mRNA (8 µg/ml) and containing vehicle (0.1% DMSO) or 5 µM hippuristanol (in 0.1% DMSO) were supplemented with 0.5 µg recombinant protein. Protein synthesis was assessed by using <sup>35</sup>S-methionine incorporation as well as by monitoring luciferase assays. Protein products were separated by SDS-PAGE and visualized by autoradiography. The arrow indicates the position of migration of the firefly luciferase, whereas the arrowhead denotes the position of migration of <em>Renilla</em> luciferase. Right panel: Relative luciferase activity obtained in the presence of recombinant eIF4A. <em>Firefly</em> RLU readings obtained in the presence of recombinant eIF4A and hippuristanol were standardized to <em>Renilla</em> RLU values and set relative to the values obtained in the presence of vehicle (DMSO). The average of 3–8 experiments is shown with the standard deviations denoted. (C) eIF4AI and eIF4AII are functionally interchangeable. <em>In vitro</em> translations in RRL containing vehicle (DMSO) or 5 µM hippuristanol were supplemented with 0.8 µg recombinant eIF4A where indicated, and programmed with capped FF/HCV/Ren mRNA (8 µg/ml). Left panel: Protein products were separated by SDS-PAGE and visualized by autoradiography. The arrow indicates the position of migration of the firefly luciferase, whereas the arrowhead denotes the position of migration of <em>Renilla</em> luciferase. Right panel: Relative luciferase activity obtained in the presence of recombinant eIF4A and hippuristanol was standardized to <em>Renilla</em> Luciferase levels and set relative to the values obtained in the presence of vehicle (DMSO). The average of 4–5 experiments is shown with the standard deviations denoted. (D) Translational rescue by eIF4AI<sup>IG/T </sup>is selective for hippuristanol. <em>In vitro</em> translations in RRL containing vehicle (DMSO), 5 µM hippuristanaol, or 0.4 µM pateamine were supplemented with 0.8 µg recombinant eIF4A where indicated, and programmed with capped FF/HCV/Ren mRNA (8 µg/ml). Protein products were separated by SDS-PAGE and visualized by autoradiography. The arrow indicates the position of migration of the firefly luciferase, whereas the arrowhead denotes the position of migration of <em>Renilla</em> luciferase. The figure is a representative display of one of two experiments.</p>
<span>doi:10.1371/journal.pone.0001583.g004</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Next, we tested if eIF4AI and eIF4AII are functionally redundant for translation. To this end, we assessed the ability of eIF4AI<sup>IP/T </sup>and eIF4AII<sup>IP/T </sup>to rescue hippuristanol-induced translation inhibition (<a href="#pone-0001583-g004">Fig 4C</a>). Like eIF4AI<sup>IG/T </sup>(<a href="#pone-0001583-g004">Fig. 4B</a>), both eIF4AI<sup>IP/T </sup>and eIF4AII<sup>IP/T </sup>rescued to the same extent (<a href="#pone-0001583-g004">Fig. 4C</a>, compare lanes 10 and 6 to 8 and 4, respectively). The rescue by the Hipp<sup>R</sup> mutants in these experiments was specific for hippuristanol, since it was not observed when translation was inhibited by pateamine - another eIF4A small molecule activity modulator (<a href="#pone-0001583-g004">Fig. 4D</a>, compare lane 9 to 8).</p>


<h4>Hippuristanol targets eIF4A <em>in vivo</em></h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We used a genetic approach to demonstrate that hippuristanol targets eIF4A <em>in vivo</em>. <em>S. cerevisiae</em> contains two eIF4A orthologues of identical amino acid sequence, called Tif1 and Tif2 <a href="#pone.0001583-Giaever1">[26]</a>. We first assessed whether hippuristanol can block translation in an <em>in vitro S. cerevisiae</em> system programmed with <em>Renilla</em> mRNA (<a href="#pone-0001583-g005">Fig. 5A</a>). Concentrations of 1 µM hippuristanol were sufficient to inhibit protein synthesis (<a href="#pone-0001583-g005">Fig. 5A</a>). If Tif1/2p is the relevant biological target of hippuristanol <em>in vivo</em>, then <em>Saccharomyces cerevisiae</em> strains showing reduced activity of Tif1/2p should be more sensitive to growth inhibition by this compound than the wild-type (wt) strain <a href="#pone.0001583-Giaever1">[26]</a>. The growth of 4A-ts, which contains the temperature sensitive V69S allele of Tif1p, was more sensitive than the wild-type parental strain or strains containing a deletion of Tif1p (Δtif1) or Tif2p (Δtif2) (<a href="#pone-0001583-g005">Figs. 5B–C</a>). These results imply that hippuristanol targets the yeast homologue of eIF4A (Tif1/2p) <em>in vivo</em> to affect growth.</p>
<div class="figure" id="pone-0001583-g005"><div class="img"><a name="pone-0001583-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Hippuristanol targets eIF4A <em>in vivo</em>.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Inhibition of <em>Renilla</em> luciferase reporter in a yeast <em>in vitro</em> translation extract. Hippuristanol was added to a <em>S. cerevisiae</em> cytosolic translation extract programmed with 0.12 µg/ml capped <em>Renilla</em> luciferase mRNA. At various points following initiation of the translation reaction, aliquots were removed and the relative luciferase units (RLU) determined. (B) Haploinsufficiency for Tif1/2p leads to increased sensitivity to hippuristanol <em>in vivo.</em> Haploid wild type cells (strain CWO4) or an isogenic strain carrying the temperature-sensitive tif1V79A allele (strain SS13-3A/pSSC120) <a href="#pone.0001583-Coppolecchia1">[41]</a> were cultivated in rich medium (YPD) at 27°C to an O.D.<sub>600</sub> of 0.2, at which point hippuristanol (1 µM or 10 µM final concentration) or solvent (DMSO, 0.1%) was added and the growth of different cultures was monitored for several hours by measuring the O.D.<sub>600</sub>. (C) Serial dilutions of different haploid yeast strains were plated on YPD-plates containing the indicated concentrations of hippuristanol and incubated for 2–3 days at 27°C: wt (CWO4), wild type strain CWO4; 4A-ts, strain SS13-3A/pSSC120 carrying the tif1V79A allele; Δtif1, a BY4741-derivative strain carrying a tif1::kanX deletion; Δtif2, a BY4741-derivative strain carrying a tif2::kanX deletion; wt (BY4741), wild type strain BY4741.</p>
<span>doi:10.1371/journal.pone.0001583.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>These findings prompted us to use the <em>Xenopus laevis</em> translation system to assess if hippuristanol-induced inhibition of translation could be relieved by eIF4AI<sup>IG/T</sup> <em>in vivo</em> (<a href="#pone-0001583-g006">Fig. 6</a>). The hippuristanol binding site is 100% conserved between the murine and <em>X. laevis</em> eIF4AI proteins (data not shown). Addition of 5 µM hippuristanol inhibited cap-dependent translation of injected FF/HCV/Ren mRNA by 95% (<a href="#pone-0001583-g006">Fig. 6A</a>). Co-injection of recombinant eIF4AI slightly relieved the inhibition by hippuristanol to 30% of vehicle treated cells, whereas introduction of eIF4AI<sup>IG/T </sup>completely rescued the effect (<a href="#pone-0001583-g006">Fig. 6A</a>). Equivalent amounts of recombinant eIF4AI and eIF4AI<sup>IG/T </sup>were delivered to the cells upon micro-injection, as assessed by western blot analysis of extracts prepared from the injected eggs (<a href="#pone-0001583-g006">Fig. 6B</a>).</p>
<div class="figure" id="pone-0001583-g006"><div class="img"><a name="pone-0001583-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001583" data-uri="info:doi/10.1371/journal.pone.0001583.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001583.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001583.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001583.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001583.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span><em>In vivo</em> rescue of hippuristanol-induced translation inhibition by eIF4AI<sup>IG/T</sup>.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>(A) Rescue of translation in <em>Xenopus</em> oocytes by eIF4AI<sup>IG/T</sup>. The percent rescue was determined by normalizing the <em>Firefly</em> luciferase values to <em>Renilla</em> luciferase [to standardize for small variations in sample injection volumes], followed by dividing by the ratio obtained from the vehicle-treated samples (which was set at 100%). The data presented is the average of 9 independent sets of injections with the standard deviations denoted. (B) Western blot of extracts prepared from oocyte extracts. The equivalent of one oocyte was separated on a 10% SDS-PAGE, transferred to Immobilon-P, and probed with α-His<sub>6</sub> (to detect recombinant His<sub>6</sub>-eIF4AI) or α-tubulin antibodies.</p>
<span>doi:10.1371/journal.pone.0001583.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Herein, we demonstrate that hippuristanol interacts with amino acids within and adjacent to motifs V and VI of eIF4AI, two regions implicated in RNA, ATP, and interdomain contacts <a href="#pone.0001583-Bono1">[23]</a>, <a href="#pone.0001583-Sengoku1">[25]</a>. Hippuristanol inhibits the RNA-dependent ATPase activity and RNA binding ability of eIF4A, but does not prevent binding of ATP to eIF4A <a href="#pone.0001583-Bordeleau2">[20]</a>. The hippuristanol-binding site on eIF4A-CTD is adjacent to, or overlapping with, the NTD- and ATP-interacting surfaces. Since hippuristanol does not bind to the NTD, ATP would still be able to bind to eIF4A-NTD (its main binding site) in the presence of hippuristanol. The amino acid corresponding to Thr<sup>329</sup> of eIF4AI (<a href="#pone.0001583.s002">Fig. S1</a>) in Vasa and eIF4AIII is implicated in RNA binding via interaction with a phosphate residue on the RNA backbone. We therefore speculate that either: (i) hippuristanol interferes with proper interdomain interaction, which in turn abolishes RNA binding or (ii) affects alignment of Thr<sup>329</sup> with its target phosphate on RNA. We favor the former possibility as this mechanism of action has been documented by Nakamura and colleagues, who identified an RNA aptamer that inhibits eIF4A activity by also interfering with interdomain interaction <a href="#pone.0001583-Oguro1">[27]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Our results extend previous studies implicating eIF4A’s helicase activity and eIF4A:eIF4G interaction as being essential for translation (<a href="#pone-0001583-g004">Fig. 4</a>). eIF4AI<sup>IG/T</sup>, but not eIF4AI<sup>Quad/IG/T</sup>, was capable of rescuing hippuristanol-induced translation inhibition (<a href="#pone-0001583-g004">Fig. 4</a>), implying that eIF4AI<sup>IG/T</sup> can assemble into the eIF4F complex and does not rescue translation as a free subunit. Consistent with this interpretation, rescue of translation by eIF4AI<sup>IG/T</sup> is inhibited by the cap analog (m<sup>7</sup>GDP) (data not shown). Although eIF4A plays an accessory role in promoting 48S complex formation on unstructured mRNA templates, it is required for 48S complex formation and translation of mRNAs containing weakly structured (−13.6 kcal/mol) hairpins <a href="#pone.0001583-Pestova1">[28]</a>. eIF4A has been proposed to cycle through the eIF4F complex during initiation <a href="#pone.0001583-YoderHill1">[12]</a> and mutants of eIF4A have been previously described which appear to act in a dominant-negative manner to trap eIF4F in an inactive state, thus inhibiting translation <a href="#pone.0001583-Pause2">[10]</a>, <a href="#pone.0001583-Svitkin1">[11]</a>. The inverse relationship between 5′ secondary structure and sensitivity to inhibition by a dominant-negative mutant of eIF4A <a href="#pone.0001583-Svitkin1">[11]</a> is consistent with the idea that eIF4A functions to unwind secondary structure in the 5′ UTR during initiation to create a ribosome landing pad. However, not all potential activities of DEAD box proteins are necessarily involved in their function. Case in point is eIF4AIII where ATP hydrolysis is not required for its participation in NMD <a href="#pone.0001583-Shibuya1">[29]</a>. Therefore, we directly tested the requirement for eIF4A’s helicase activity in translation using a hippuristanol-resistant helicase-defective mutant and found it to be essential for eIF4A’s participation in initiation (<a href="#pone-0001583-g004">Fig. 4</a>). We also find that eIF4AI and eIF4AII are interchangeable in their ability to support translation <em>in vitro</em>. Previous reports documented that both eIF4AI and eIF4AII can associate with the eIF4F complex <a href="#pone.0001583-YoderHill1">[12]</a>, <a href="#pone.0001583-Conroy1">[13]</a>, and our studies extend these results by demonstrating that an eIF4A mutant lacking this function cannot participate in the translation process. The ability to generate hippuristanol-sensitive alleles of various DDX family members is a powerful tool with which to assign function and undertake structure-function studies.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>NMR Spectroscopy and modeling</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>NMR spectra were recorded at 298 K on a Varian Inova 600 and Inova 500 instruments equipped with cryogenic probes. Samples for NMR measurements typically contained 0.4–1 mM protein in buffer containing 20 mM Tris-HCl<sub>7.0</sub>, 300 mM NaCl, 5 mM DTT, 1 mM EDTA, 0.01% NaN<sub>3</sub>, 0.2 mM AEBSF, and 10% D<sub>2</sub>O. The spectra were processed with NMRPipe <a href="#pone.0001583-Delaglio1">[30]</a> and analyzed with XEASY<a href="#pone.0001583-Bartels1">[31]</a>. Sequential resonance assignments for eIF4AI-CTD and its complex with hippuristanol were obtained from standard triple-resonance NMR experiments [HNCA, HN(CA)CB, HN(CO)CA, HN(COCA)CB, HNCO, HN(CA)CO] on uniformly <sup>15</sup>N-/<sup>13</sup>C-labeled samples with 70% deuteration. <sup>15</sup>N-edited NOESY-HSQC and TOCSY-HSQC experiments were recorded on uniformly <sup>15</sup>N-labeled samples. Intermolecular NOEs between eIF4AI-CTD and hippuristanol were measured using uniformly <sup>15</sup>N/<sup>13</sup>C/<sup>2</sup>H labelled CTD (52 µM) complexed with 100 µM hippuristanol. <sup>15</sup>N-dispersed NOESY spectra were recorded in H<sub>2</sub>O, and exhibited intermolecular NOESY cross peaks between peptide HN and hippuristanol in an otherwise empty spectral region as described previously <a href="#pone.0001583-Matsuo1">[32]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Homology modeling of the eIF4AI-CTD was previously described <a href="#pone.0001583-Oberer1">[24]</a>. The model of full-length eIF4AI was constructed by manually superimposing the structure of human eIF4A-NTD (PDB# 2G9N) and the homology model of eIF4AI-CTD with the structure of eIF4AIII from the exon-junction complex (PDB# 2HYI) <a href="#pone.0001583-Andersen1">[22]</a>.</p>


<h4>Recombinant DNA Constructs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Site-directed mutagenesis was performed using a PCR based strategy. Oligonucleotides harboring the mutant sequence(s) were used in amplification reactions with either upstream or downstream oligonucleotides spanning unique restriction sites (<a href="#pone.0001583.s001">Table SI</a>). Following gel purification of PCR products, these were combined and used in amplification reactions in the absence of primer for 5 cycles, at which point primers targeting the ends of the fragments were added and amplifications continued for 30 cycles. DNA fragments were cloned into expression plasmids using convenient restrictions sites and all clones used in this study were sequenced to ensure the absence of secondary undesired mutations. The murine eIF4AI cDNA was subcloned from pET3b/eIF4AI into pET15b using NdeI and BamHI restriction enzymes <a href="#pone.0001583-Pause2">[10]</a>. The murine eIF4AII was excised from pACT2/4AII with BamHI (which was rendered blunt by Klenow repair) and XhoI, and inserted into the BamHI (blunt ended with Klenow)/XhoI sites of pET28a. Human eIF4AIII mutants were derived from pET-HTG-eIF4AIII <a href="#pone.0001583-Shibuya1">[29]</a> and subcloned into pET28a (BamHI site).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Cloning strategies to generate the various mutants utilized in this study were as follows: For pET15b/eIF4AI<sup>IG</sup>, two PCR products were produced using pET15b/eIF4AI as template and the primer pairs: (i) DraIII Forward and IG Reverse and (ii) IG Forward and BamHI Reverse (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers DraIII Forward and BamHI Reverse were used to amplify the full length mutant. The product was digested with DraIII and BamHI and inserted into pET15b/eIF4AI. For pET15b/eIF4AI<sup>T</sup>, two PCR products were produced using pET15b/eIF4AI as template and the primer pairs: (i) DraIII Forward and T Reverse and (ii) T Forward and BamHI Reverse (<a href="#pone.0001583.s001">Table S1</a>). The products were combined and cloned as described for pET15b/eIF4AI<sup>IG</sup>. For pET15b/eIF4AI<sup>IG/T</sup>, two PCR products were produced using pET15b/eIF4AI<sup>IG</sup> as template and the primer pairs: (i) DraIII Forward and T Reverse and (ii) T Forward and BamHI Reverse (<a href="#pone.0001583.s001">Table S1</a>). The products were combined and cloned as described for pET15b/eIF4AI<sup>IG</sup>. For pET15b/eIF4AI<sup>IP/T</sup>, two PCR products were produced using pET15b/eIF4AI as template and the primer pairs: (i) DraIII Forward and IP Reverse and (ii) IP Forward and BamHI Reverse (<a href="#pone.0001583.s001">Table S1</a>). The products were combined and cloned as described for pET15b/eIF4AI<sup>IG.</sup> Plasmid pET15b/eIF4AI<sup>IP</sup> was used as template for two PCR reactions, using either (i) primers DraIII Forward and T Reverse and (ii) T Forward and BamHI Reverse. The products were combined and cloned as described for pET15b/eIF4AI<sup>IG </sup>to generate pET15b/eIF4AI<sup>IP/T</sup>.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>For pET15b/eIF4AI<sup>Quad/IG/T</sup>, two PCR products were produced using pET15b/eIF4AI<sup>IG/T </sup>as template and the primer pairs: (i) primers D265R/E268Kforward and 4A(1220–1238)(AS) and (ii) D265R/E268Kreverse and pET15b Oligo (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers pET15b Oligo and 4A(1220–1238)(AS) were used to amplify the full length mutant. The product was digested with NdeI and BamHI and inserted into the same sites in pET15b to create pET15b/eIF4AI<sup>RK/IG/T</sup>. An additional round of mutagenesis was undertaken using this as template and the primer pairs: (i) D296A/T298Kreverse and pET15b Oligo, and (ii) D296A/T298Kforward and 4A(1220–1238)(AS) (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each other during 5 rounds of PCR amplification, after which the primers pET15b Oligo and 4A(1220–1238)(AS) were used to amplify the full length mutant. The product was digested with NdeI and BamHI and inserted into the same sites in pET15b to create pET15b/eIF4AI<sup>Quad/IG/T</sup>. For pET15b/eIF4AI<sup>Hel/IG/T</sup>, two PCR products were produced using pET15b/eIF4AI<sup>IG/T </sup>as template and the primer pairs: (i) AIGforward and 4A(1220–1238)(AS) and (ii) AIGreverse and pET15b Oligo (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers pET15b Oligo and 4A(1220–1238)(AS) were used to amplify the full length mutant. The product was digested with NdeI and BamHI and inserted into the same sites in pET15b.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>For pET28a/eIF4AII<sup>IP/T</sup>, two PCR products were produced using pET28a/eIF4AII as template and the primer pairs: (i) 4AII-IP(S) and 4AII(1278–1258) and (ii) 4AII-NheI-NTD and 4AII-IP(AS) (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers 4AII(1278-1258) and 4AII-NheI-NTD were used to amplify the full length mutant. The product was digested with NheI and XhoI and inserted into the same sites in pET28a to generate pET28a/eIF4AII<sup>IP</sup>. The T mutation was introduced (<a href="#pone.0001583.s002">Fig. S1</a>) using pET28a/eIF4AII<sup>IP</sup> as template and the primer pairs: (i) 4AII-T(S) and 4AII(1278–1258) and (ii) 4AII-NheI-NTD and 4AII-T(AS) (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers 4AII(1278–1258) and 4AII-NheI-NTD were used to amplify the full length mutant. The product was digested with NheI and XhoI and inserted into the same sites in pET28a.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>For pET28a/eIF4AIII<sup>IP/T</sup>, two PCR products were produced using pET28a/eIF4AII as template and the primer pairs: (i) Primer D and Primer F and (ii) Primer E and Primer A (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers Primer A and Primer D were used to amplify the full length mutant. The product was digested with BamHI and inserted into the same site in pET28a to generate pET28a/eIF4AIII<sup>IP</sup>. The T mutation was introduced (<a href="#pone.0001583.s002">Fig. S1</a>) using pET28a/eIF4AIII<sup>IP</sup> as template and the primer pairs: (i) Primer D and Primer C and (ii Primer B and Primer A (<a href="#pone.0001583.s001">Table S1</a>). Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers Primer A and Primer D were used to amplify the full length mutant. The product was digested with BamHI and inserted into the same site in pET28a. For pET28a/eIF4AIII<sup>TLLQV</sup>, two PCR products containing the TLLQV mutation were produced from pET28a/eIF4AIII: one with the primers Primer D and 4AIII(TLLQV)AS and the other with 4AIII(TLLQV)S and Primer A. Following gel purification, the fragments were used to extend off of each during 5 rounds of PCR amplification, after which the primers Primer A and Primer D were used to amplify the full length mutant. The product was digested with BamHI and inserted into the same site in pET28a.</p>


<h4>Purification of Hippuristanol</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Hippuristanol was extracted from the gorgonian <em>I. hippuris</em> as previously described<a href="#pone.0001583-Bordeleau2">[20]</a>.</p>


<h4>Recombinant Protein Expression and Purification</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Recombinant His<sub>6</sub>-eIF4AI and His<sub>6</sub>-eIF4AII were expressed in <em>E. Coli</em> BL21 (DE3) codon+. Bacteria were grown to an OD<sub>600 </sub>of 0.6 and induced with 1mM IPTG. Growth was continued an additional 3h at 37°C. Wild-type and mutant eIF4AI and eIF4AII proteins were resuspended in sonication buffer (20 mM Tris<sub>7.5</sub>, 10% glycerol, 0.1 mM EDTA, 200 mM KCl, 0.1% Triton X-100 and 3.4 mM β-mercaptoethanol). After sonication (9 pulses of 20 sec) and clarification by centrifugation (twice at 27,000×g for 30 min), the lysate was loaded on a Ni<sup>++</sup>-NTA agarose (Qiagen) column. The column was washed with 10 column volumes of wash 1 (20 mM Tris<sub>7.5</sub>, 10% glycerol, 0.1 mM EDTA, 800 mM KCl, 20 mM imidazole) and wash 2 (Wash 1 containing 300 mM KCl) buffer, and the His<sub>6</sub>-tagged proteins were eluted (Wash 1 buffer containing 300 mM KCl and 0.2 M imidazole), and dialyzed into A100 buffer (20 mM Tris<sub>7.5</sub>, 10% glycerol, 0.1 mM EDTA, 100 mM KCl, 2 mM DTT). The protein was loaded onto a Q-Sepharose Fast Flow (Amersham) column and eluted with a KCl gradient A100 to A500 (20 mM Tris<sub>7.5</sub>, 10% glycerol, 0.1 mM EDTA, 2 mM DTT ). The eluted protein was dialysed against 20 mM Tris<sub>7.5</sub>, 0.1 mM EDTA, and 10% glycerol.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Recombinant His<sub>6</sub>-eIF4AIII and mutants were purified through a Ni<sup>++</sup>-NTA agarose (Qiagen) column as previously described <a href="#pone.0001583-Shibuya2">[33]</a>. The eluent was diluted to a salt concentration of 100 mM and passed through a Q-Sepharose Fast Flow (Amersham) column and eIF4AIII collected in the flow through. His<sub>6</sub>-eIF4AIII was stored in 20 mM Tris<sub>7.5</sub>, 0.1 mM EDTA, 100 mM KCl, 10 % glycerol, and 2 mM DTT. His<sub>6</sub>-eIF4AIII<sup>TLLQV</sup> was further purified on a heparin-sepharose column.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>The cDNA encoding hDDX52 [NM007010.2] was obtained from Origene (Rockville, MD) and subcloned into the Not1 sites of pET28a. The purification procedure was as described above for eIF4AI using Ni<sup>++</sup>-NTA agarose and Q-Sepharose. Recombinant hDDX52 protein was stored in 20 mM Tris<sub>7.5</sub>, 10% glycerol, 0.1 mM EDTA, 2 mM DTT, 100mM KCl.</p>


<h4>ATPase, RNA binding, and Helicase Assays</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>ATPase assays were performed as described by Lorsch and Herschlag, using their “Condition B”<a href="#pone.0001583-Lorsch1">[34]</a>. Briefly, 0.1 µg or 1 µg protein (indicated in figure legend) was incubated with 2.5 µM poly (U) and 1 µM γ-<sup>32</sup>P-ATP (10 Ci/mmol) (0.01 µCi) at 25°C (except where specifically indicated) and time points taken at the indicated intervals by removing 2 µL aliquots and diluting into 2 µL of 25 mM EDTA. Inorganic phosphate and γ-<sup>32</sup>P-ATP were separated by TLC as described previously <a href="#pone.0001583-Lorsch1">[34]</a>. Results were quantitated using a Fujix BAS 2000 phosphoimager with a Fuji imaging screen. The fraction of P<sub>i</sub> at t = 0 was typically 1–3% and subtracted as background.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>For Tif1/2p, 0.5 µg Tif1/2p and 1 µg yeast eIF4G (aa542-883) was incubated with 12 nM 18S rRNA and 1 mM ATP at 25°C for 1h. The reaction was stopped with 50 mM EDTA and the formation of free phosphate determined as ammonium molybdate complex photometrically with malachite green (at A<sub>630</sub>) Calibration curves were prepared with increasing concentrations of Na<sub>3</sub>PO<sub>4</sub> (5–100 µM) (BioAssays Systems).</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Chemically cross-linking was performed with 0.5–1 µg recombinant protein and oxidized <sup>32</sup>P-labelled CAT mRNA (~18,000 cpm/µg) in the presence of 0.9 mM ATP with 10 µM hippuristanol or vehicle (DMSO) for 10 min at 30°C <a href="#pone.0001583-Bordeleau2">[20]</a>, <a href="#pone.0001583-Sonenberg1">[35]</a>, after which time sodium cyanoborohydride was added, and the incubation continued overnight at 4°C. The samples were then treated with RNAse A, separated on a 10% SDS-PAGE, and visualized by autoradiography.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>RNA helicase assays were performed as previously described <a href="#pone.0001583-Rogers1">[7]</a>, <a href="#pone.0001583-Rozen1">[36]</a>. Briefly, 0.4 µM recombinant His<sub>6</sub>-eIF4AI or His<sub>6</sub>-eIF4AII was incubated with 2 µM RNA-1/11 duplex in the presence of 1 mM ATP for 15 minutes at 35°C. Reactions were resolved on native 12% polyacrylamide gels, which were dried and exposed to X-Omat (Kodax) film at -70°C.</p>


<h4>Rescue of hippuristanol-induced translation inhibition</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The plasmid pKS/FF/HCV/Ren was linearized with BamHI and transcribed with T3 RNA polymerase to generate FF/HCV/Ren mRNA <a href="#pone.0001583-Bordeleau2">[20]</a>. <em>In vitro</em> translations were performed in rabbit reticulocyte lysates following the manufacturer’s instructions (Promega). Extracts were programmed with FF/HCV/Ren mRNA (8 µg/ml) and translations performed at a final concentration of 135 mM KCl. eIF4A rescue experiments were performed by the addition of 0.5 or 0.8 µg recombinant eIF4A (0.9–1.4 µM) to vehicle- or hippuristanol -treated extracts. Firefly luciferase activities were measured on a Berthold Lumat LB 9507 luminometer. Reactions performed in the presence of [<sup>35</sup>S]methionine were separated on a 10% SDS-polyacrylamide gel which was treated with EN<sup>3</sup>Hance, dried, and exposed to X-Omat (Kodax) film at −70°C. For <em>in vitro</em> translation reactions in yeast, wild type strain BY4741 was used to prepare an <em>in vitro</em> translation extract and assays were performed as previously reported <a href="#pone.0001583-Altmann1">[37]</a>.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Translations in <em>Xenopus</em> oocytes were performed essentially as previously described <a href="#pone.0001583-Otero1">[38]</a>. Briefly, collagenased <em>Xenopus</em> oocytes were sorted and incubated for 4 h at 16°C in 5 µM hippuristanol or 0.05% DMSO. Each oocyte was then injected with 50 nL of 0.94 mg/mL recombinant His<sub>6</sub>-eIF4AI or recombinant His<sub>6</sub>-eIF4AI<sup>IG/T</sup> or buffer alone immediately followed by 10 nL of 0.02 µM in vitro transcribed FF/HCV/Ren mRNA. Oocytes were then incubated for 4h at room temperature in fresh compound dilutions. Three oocytes were homogenized in 150 µL Passive Lysis Buffer (Promega). The cell lysates were cleared by centrifugation at 14000×g for 5 min. Ten microliters of the lysate was read per sample using the Dual Luciferase Assay system (Promega). Values were normalized to <em>Renilla</em> activity and the percent rescue determined as average value of compound challenged samples divided by the average value of DMSO challenged samples.</p>
<h5>Time resolved fluorescence resonance energy transfer (TR-FRET).</h5><a id="article1.body1.sec4.sec6.sec1.p1" name="article1.body1.sec4.sec6.sec1.p1"></a><p>Recombinant His<sub>6</sub>-eIF4AI protein (20 nM ) and GST-eIF4GI<sub>517-606 </sub>(40 nM) or GST-eIF4GI<sub>648–983 </sub>(200 nM) were incubated with Eu-W1024 labeled anti-6xHis antibody (1 nM) [Perkin Elmer] and anti-GST IgG antibody conjugated to SureLight-Allophycocyanin (100 nM) [Perkin Elmer] in TR-FRET buffer (20 mM Hepes<sub>7.4</sub>, 10 mM KCl, 1 mM DTT, 0.015% Tween 20, 1 µg/ml IgG). Reactions were performed at room temperature for 3 hrs. FRET signal was monitored using an Analyst HT reader (LJL Biosystems) <a href="#pone.0001583-Cencic1">[39]</a>. Data collection using the “Criterion Host v.2.0.1” software (LJL Biosystems) involved setting the Z height at 1 mm and utilizing 1 excitation filter (330/80) and 2 emission filters (620/7.5 and 665/7.5). A dichroic filter with a wavelength of 400 nm was used. For the measurement at 620 nm we employed 100 readings per well, with 10 ms between reading, integration time of 1000 µs, a delay time of 200 µs and 1000 µs integration time for the fluorescence emission recording. The parameters for the measurement at 665 nm were the same as for 620 nm, except for an integration time of 150 µs and a delay time of 50 µs. Due to the time delay, only the longer-lived FRET signal is detected, eliminating short-lived background fluorescence. The 665/620 ratio was calculated and normalized to the negative control reaction (containing His<sub>6</sub>-eIF4E<sub>W73A</sub> which does not interact with eIF4G or 4E-BP) to yield the S/B ratio. The 665 nm emissions are due to APC FRET and the 620 nm emissions are due to Eu-W1024 fluorescence.</p>


</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001583"></div><a name="pone.0001583.s001" id="pone.0001583.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001583.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001583.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001583.s002" id="pone.0001583.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001583.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Amino acid alignment of the hippuristanol binding site among murine eIF4AI, murine eIF4AII, and human eIF4AIII alleles used in this study. Direct protein-hippuristanol NOEs are highlighted in yellow, whereas those within 5{Angstrom} are in grey. Introduced amino acid changes are highlighted in red. The position of the first and last amino acid of the motif is indicated. Amino acids corresponding to the hippuristanol binding site in <em>S. cerevisiae</em> Ded1p is also shown.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001583.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001583.s003" id="pone.0001583.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001583.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Amino acid alignment of the hippuristanol binding site among DDX family members. Alignments shown are for members of the murine (A) or human (B) DEAD box family members. The Entrez Protein IDs are provided in parenthesis for each member. Direct protein-hippuristanol NOEs are highlighted in yellow, whereas those within 5{Angstrom} are in grey. The position of the first and last amino acid of the motif in the protein sequence is indicated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001583.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001583.s004" id="pone.0001583.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001583.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">RNA-dependent ATPase activity of eIF4AI<sup>IG/T</sup>, eIF4AI<sup>IG</sup>, and eIF4AI<sup>T</sup> mutant alleles. ATP hydrolysis was monitored using 1 µg recombinant protein. Each value represents the average of two measurements with the error of the mean presented. [Note in this assay, the protein preparation was different and not as active as the preparation used in <a href="#pone-0001583-g003">Fig. 3A</a>.]</p>
<p class="siDoi">doi:10.1371/journal.pone.0001583.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(10.00 MB TIF)</p>
<a name="pone.0001583.s005" id="pone.0001583.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001583.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Characterization of eIF4AI and eIF4AII hippuristanol-resistant mutants. (A) Crosslinking of recombinant proteins to RNA in the presence of hippuristanol. <sup>32</sup>P-labelled CAT RNA was cross-linked to 0.5–1 µg of the indicated recombinant protein in the presence or absence of hippuristanol, separated by SDS-PAGE, and visualized by autoradiography. (B) The helicase activities of the eIF4AI<sup>IG/T</sup> and eIF4AII<sup>IP/T</sup> mutants are resistant to hippuristanol. Helicase assays were performed with recombinant protein (0.4 µM) and duplexed RNA as described in the <a href="#s4">Materials and Methods</a>. Reactions were resolved on a native 12% acrylamide gel, which was dried, and exposed to BioMax XAR film (Kodak) film at −70°C. The position of migration of duplexed (ds) and single-stranded (ss) RNA are denoted to the right.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001583.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(5.78 MB TIF)</p>
<a name="pone.0001583.s006" id="pone.0001583.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001583.s006">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Characterization of eIF4A mutants. (A) RNA-dependent ATPase activity of eIF4AI<sup>Hel/IG/T</sup> and eIF4AI<sup>Quad/IG/T</sup> mutants. ATP hydrolysis was monitored using 1 µg recombinant protein. Each value represents the average of two measurements with the error of the mean presented. In this experiment, the protein preparations were different and not as active as the preparations used in <a href="#pone-0001583-g003">Fig. 3A</a>. (B) The helicase activity of eIF4AI<sup>Hel/IG/T</sup> is impaired. Recombinant protein (0.4 µM) was incubated with duplexed RNA as described in <a href="#s4">Materials and Methods</a>. Reactions were resolved on a native 12% acrylamide gel and visualized by autoradiography. The migration of duplexed and ssRNA are determined by the incubation of duplexed RNA alone at 35°C (lane 1) or boiling for 5 minutes (lane 2), respectively. (C) Crosslinking of eIF4AI<sup>Quad/IG/T</sup> and eIF4AI<sup>Hel/IG/T</sup> to RNA in the presence of hippuristanol. <sup>32</sup>P-labelled CAT RNA was cross-linked to 1 µg of the indicated recombinant protein in the presence or absence of hippuristanol, separated by SDS-PAGE, and visualized by autoradiography. (D) Helicase activity of eIF4AI<sup>Quad/IG/T</sup> is not impaired and resistant to hippuristanol. Helicase assays were performed with recombinant protein (0.4 µM) and duplexed RNA as described in the <a href="#s4">Materials and Methods</a>. Reactions were resolved on a native 12% acrylamide gel, which was dried, and exposed to BioMax XAR film (Kodak) film at −70°C. The position of migration of duplexed (ds) and single-stranded (ss) RNA are denoted to the right.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001583.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(4.08 MB TIF)</p>
<a name="pone.0001583.s007" id="pone.0001583.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001583.s007">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">The interaction of eIF4AI<sup>Quad/IG/T</sup> with eIF4GI is impaired. (A) Schematic representation of the various functional domains of eIF4GI. Protein and RNA binding sites on eIF4GI are indicated. The numbers below eIF4GI refer to the amino acid location of each binding site. A schematic of the recombinant eIF4GI fragments utilized and the regions they span are shown in grey boxes. (B) TR-FRET analysis of the interaction between eIF4AI, eIF4AI<sup>IG/T</sup>, eIF4AI<sup>Quad/IG/T</sup> with eIF4GI fragments. GST-eIF4GI fragments were incubated with recombinant His<sub>6</sub>-eIF4AI protein, as well as with Eu-W1024 labeled anti-6xHis antibody and anti-GST IgG antibody conjugated to SureLight-Allophycocyanin. The FRET signal (expressed as the signal to background ratio (S/B)) was monitored on an Analyst reader (LJL Biosystems) and represents the average of 4 experiments with the standard error of the mean shown. The signal obtained with eIF4AI and eIF4G<sub>517-606</sub> was equivalent to the background signal (S/B = 1).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001583.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(4.39 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Drs. Stanley Tahara (USC, California) and Lovisa Holmberg-Schiavone (Karolinska Institute, Sweden) for the gift of GST-eIF4GI expression plasmids and DDX19, respectively. We thank Drs. Melissa Moore (Brandeis University) and Nahum Sonenberg (McGill University) for plasmids encoding eIF4AIII and eIF4AII, respectively.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FF GW MA JP LL MO RC MB. Performed the experiments: FF MA JP LL MO MR RC MB EV. Analyzed the data: FF GW MA JP LL MO MR RC AM. Contributed reagents/materials/analysis tools: JT. Wrote the paper: JP LL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001583-Singleton1" id="pone.0001583-Singleton1"></a>Singleton MR, Dillingham MS, wigley DB (2007) Structure and mechanism of helicases and nucleic acid translocases. Annual Review of Biochemistry  76: 23–50.  <ul class="find" data-citedArticleID="1054316" data-doi="10.1146/annurev.biochem.76.052305.115300"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.76.052305.115300" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+mechanism+of+helicases+and+nucleic+acid+translocases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+mechanism+of+helicases+and+nucleic+acid+translocases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001583-Cordin1" id="pone.0001583-Cordin1"></a>Cordin O, Banroques J, Tanner NK, Linder P (2006) The DEAD-box protein family of RNA helicases. Gene  367: 17–37.  <ul class="find" data-citedArticleID="1054264" data-doi="10.1016/j.gene.2005.10.019"><li><a href="http://dx.doi.org/10.1016/j.gene.2005.10.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DEAD-box+protein+family+of+RNA+helicases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DEAD-box+protein+family+of+RNA+helicases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001583-Linder1" id="pone.0001583-Linder1"></a>Linder P (2006) Dead-box proteins: a family affair–active and passive players in RNP-remodeling. Nucleic Acids Res  34: 4168–4180.  <ul class="find" data-citedArticleID="1054280" data-doi="10.1093/nar/gkl468"><li><a href="http://dx.doi.org/10.1093/nar/gkl468" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dead-box+proteins%3A+a+family+affair%E2%80%93active+and+passive+players+in+RNP-remodeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dead-box+proteins%3A+a+family+affair%E2%80%93active+and+passive+players+in+RNP-remodeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001583-Fukuda1" id="pone.0001583-Fukuda1"></a>Fukuda T, Yamagata K, Fujiyama S, Matsumoto T, Koshida I, et al.  (2007) DEAD-box RNA helicase subunits of the Drosha complex are required for processing of rRNA and a subset of microRNAs. Nat Cell Biol  9: 604–611.  <ul class="find" data-citedArticleID="1054274" data-doi="10.1038/ncb1577"><li><a href="http://dx.doi.org/10.1038/ncb1577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DEAD-box+RNA+helicase+subunits+of+the+Drosha+complex+are+required+for+processing+of+rRNA+and+a+subset+of+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DEAD-box+RNA+helicase+subunits+of+the+Drosha+complex+are+required+for+processing+of+rRNA+and+a+subset+of+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001583-Grifo1" id="pone.0001583-Grifo1"></a>Grifo JA, Tahara SM, Leis JP, Morgan MA, Shatkin AJ, et al.  (1982) Characterization of eukaryotic initiation factor 4A, a protein involved in ATP-dependent binding of globin mRNA. J Biol Chem  257: 5246–5252.  <ul class="find" data-citedArticleID="1054278"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+eukaryotic+initiation+factor+4A%2C+a+protein+involved+in+ATP-dependent+binding+of+globin+mRNA.&amp;auth=&amp;atitle=Characterization+of+eukaryotic+initiation+factor+4A%2C+a+protein+involved+in+ATP-dependent+binding+of+globin+mRNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+eukaryotic+initiation+factor+4A%2C+a+protein+involved+in+ATP-dependent+binding+of+globin+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+eukaryotic+initiation+factor+4A%2C+a+protein+involved+in+ATP-dependent+binding+of+globin+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001583-Edery1" id="pone.0001583-Edery1"></a>Edery I, Humbelin M, Darveau A, Lee KA, Milburn S, et al.  (1983) Involvement of eukaryotic initiation factor 4A in the cap recognition process. J Biol Chem  258: 11398–11403.  <ul class="find" data-citedArticleID="1054270"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Involvement+of+eukaryotic+initiation+factor+4A+in+the+cap+recognition+process.&amp;auth=&amp;atitle=Involvement+of+eukaryotic+initiation+factor+4A+in+the+cap+recognition+process." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Involvement+of+eukaryotic+initiation+factor+4A+in+the+cap+recognition+process." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Involvement+of+eukaryotic+initiation+factor+4A+in+the+cap+recognition+process.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001583-Rogers1" id="pone.0001583-Rogers1"></a>Rogers GW Jr, Richter NJ, Merrick WC (1999) Biochemical and kinetic characterization of the RNA helicase activity of eukaryotic initiation factor 4A. J Biol Chem  274: 12236–12244.  <ul class="find" data-citedArticleID="1054306" data-doi="10.1074/jbc.274.18.12236"><li><a href="http://dx.doi.org/10.1074/jbc.274.18.12236" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+and+kinetic+characterization+of+the+RNA+helicase+activity+of+eukaryotic+initiation+factor+4A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+and+kinetic+characterization+of+the+RNA+helicase+activity+of+eukaryotic+initiation+factor+4A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001583-Pause1" id="pone.0001583-Pause1"></a>Pause A, Sonenberg N (1992) Mutational analysis of a DEAD box RNA helicase: the mammalian translation initiation factor eIF-4A. EMBO J  11: 2643–2654.  <ul class="find" data-citedArticleID="1054296"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutational+analysis+of+a+DEAD+box+RNA+helicase%3A+the+mammalian+translation+initiation+factor+eIF-4A.&amp;auth=&amp;atitle=Mutational+analysis+of+a+DEAD+box+RNA+helicase%3A+the+mammalian+translation+initiation+factor+eIF-4A." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+of+a+DEAD+box+RNA+helicase%3A+the+mammalian+translation+initiation+factor+eIF-4A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+of+a+DEAD+box+RNA+helicase%3A+the+mammalian+translation+initiation+factor+eIF-4A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001583-Ray1" id="pone.0001583-Ray1"></a>Ray BK, Lawson TG, Kramer JC, Cladaras MH, Grifo JA, et al.  (1985) ATP-dependent unwinding of messenger RNA structure by eukaryotic initiation factors. J Biol Chem  260: 7651–7658.  <ul class="find" data-citedArticleID="1054302"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=ATP-dependent+unwinding+of+messenger+RNA+structure+by+eukaryotic+initiation+factors.&amp;auth=&amp;atitle=ATP-dependent+unwinding+of+messenger+RNA+structure+by+eukaryotic+initiation+factors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATP-dependent+unwinding+of+messenger+RNA+structure+by+eukaryotic+initiation+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATP-dependent+unwinding+of+messenger+RNA+structure+by+eukaryotic+initiation+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001583-Pause2" id="pone.0001583-Pause2"></a>Pause A, Methot N, Svitkin Y, Merrick WC, Sonenberg N (1994) Dominant negative mutants of mammalian translation initiation factor eIF-4A define a critical role for eIF-4F in cap-dependent and cap-independent initiation of translation. EMBO J  13: 1205–1215.  <ul class="find" data-citedArticleID="1054298"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dominant+negative+mutants+of+mammalian+translation+initiation+factor+eIF-4A+define+a+critical+role+for+eIF-4F+in+cap-dependent+and+cap-independent+initiation+of+translation.&amp;auth=&amp;atitle=Dominant+negative+mutants+of+mammalian+translation+initiation+factor+eIF-4A+define+a+critical+role+for+eIF-4F+in+cap-dependent+and+cap-independent+initiation+of+translation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominant+negative+mutants+of+mammalian+translation+initiation+factor+eIF-4A+define+a+critical+role+for+eIF-4F+in+cap-dependent+and+cap-independent+initiation+of+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dominant+negative+mutants+of+mammalian+translation+initiation+factor+eIF-4A+define+a+critical+role+for+eIF-4F+in+cap-dependent+and+cap-independent+initiation+of+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001583-Svitkin1" id="pone.0001583-Svitkin1"></a>Svitkin YV, Pause A, Haghighat A, Pyronnet S, Witherell G, et al.  (2001) The requirement for eukaryotic initiation factor 4A (elF4A) in translation is in direct proportion to the degree of mRNA 5' secondary structure. RNA  7: 382–394.  <ul class="find" data-citedArticleID="1054320" data-doi="10.1017/s135583820100108x"><li><a href="http://dx.doi.org/10.1017/s135583820100108x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+requirement+for+eukaryotic+initiation+factor+4A+%28elF4A%29+in+translation+is+in+direct+proportion+to+the+degree+of+mRNA+5%27+secondary+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+requirement+for+eukaryotic+initiation+factor+4A+%28elF4A%29+in+translation+is+in+direct+proportion+to+the+degree+of+mRNA+5%27+secondary+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001583-YoderHill1" id="pone.0001583-YoderHill1"></a>Yoder-Hill J, Pause A, Sonenberg N, Merrick WC (1993) The p46 subunit of eukaryotic initiation factor (eIF)-4F exchanges with eIF-4A. J Biol Chem  268: 5566–5573.  <ul class="find" data-citedArticleID="1054324"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+p46+subunit+of+eukaryotic+initiation+factor+%28eIF%29-4F+exchanges+with+eIF-4A.&amp;auth=&amp;atitle=The+p46+subunit+of+eukaryotic+initiation+factor+%28eIF%29-4F+exchanges+with+eIF-4A." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+p46+subunit+of+eukaryotic+initiation+factor+%28eIF%29-4F+exchanges+with+eIF-4A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+p46+subunit+of+eukaryotic+initiation+factor+%28eIF%29-4F+exchanges+with+eIF-4A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001583-Conroy1" id="pone.0001583-Conroy1"></a>Conroy SC, Dever TE, Owens CL, Merrick WC (1990) Characterization of the 46,000-dalton subunit of eIF-4F. Arch Biochem Biophys  282: 363–371.  <ul class="find" data-citedArticleID="1054260" data-doi="10.1016/0003-9861(90)90130-q"><li><a href="http://dx.doi.org/10.1016/0003-9861(90)90130-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+46%2C000-dalton+subunit+of+eIF-4F." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+46%2C000-dalton+subunit+of+eIF-4F.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001583-Chan1" id="pone.0001583-Chan1"></a>Chan CC, Dostie J, Diem MD, Feng W, Mann M, et al.  (2004) eIF4A3 is a novel component of the exon junction complex. RNA  10: 200–209.  <ul class="find" data-citedArticleID="1054258" data-doi="10.1261/rna.5230104"><li><a href="http://dx.doi.org/10.1261/rna.5230104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=eIF4A3+is+a+novel+component+of+the+exon+junction+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22eIF4A3+is+a+novel+component+of+the+exon+junction+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001583-Ferraiuolo1" id="pone.0001583-Ferraiuolo1"></a>Ferraiuolo MA, Lee CS, Ler LW, Hsu JL, Costa-Mattioli M, et al.  (2004) A nuclear translation-like factor eIF4AIII is recruited to the mRNA during splicing and functions in nonsense-mediated decay. Proc Natl Acad Sci U S A  101: 4118–4123.  <ul class="find" data-citedArticleID="1054272" data-doi="10.1073/pnas.0400933101"><li><a href="http://dx.doi.org/10.1073/pnas.0400933101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+nuclear+translation-like+factor+eIF4AIII+is+recruited+to+the+mRNA+during+splicing+and+functions+in+nonsense-mediated+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+nuclear+translation-like+factor+eIF4AIII+is+recruited+to+the+mRNA+during+splicing+and+functions+in+nonsense-mediated+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001583-Yang1" id="pone.0001583-Yang1"></a>Yang HS, Jansen AP, Komar AA, Zheng X, Merrick WC, et al.  (2003) The transformation suppressor Pdcd4 is a novel eukaryotic translation initiation factor 4A binding protein that inhibits translation. Mol Cell Biol  23: 26–37.  <ul class="find" data-citedArticleID="1054322" data-doi="10.1128/mcb.23.1.26-37.2003"><li><a href="http://dx.doi.org/10.1128/mcb.23.1.26-37.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transformation+suppressor+Pdcd4+is+a+novel+eukaryotic+translation+initiation+factor+4A+binding+protein+that+inhibits+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transformation+suppressor+Pdcd4+is+a+novel+eukaryotic+translation+initiation+factor+4A+binding+protein+that+inhibits+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001583-Dorrello1" id="pone.0001583-Dorrello1"></a>Dorrello NV, Peschiaroli A, Guardavaccaro D, Colburn NH, Sherman NE, et al.  (2006) S6K1- and betaTRCP-mediated degradation of PDCD4 promotes protein translation and cell growth. Science  314: 467–471.  <ul class="find" data-citedArticleID="1054268" data-doi="10.1126/science.1130276"><li><a href="http://dx.doi.org/10.1126/science.1130276" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=S6K1-+and+betaTRCP-mediated+degradation+of+PDCD4+promotes+protein+translation+and+cell+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22S6K1-+and+betaTRCP-mediated+degradation+of+PDCD4+promotes+protein+translation+and+cell+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001583-Bordeleau1" id="pone.0001583-Bordeleau1"></a>Bordeleau ME, Matthews J, Wojnar JM, Lindqvist L, Novac O, et al.  (2005) Stimulation of mammalian translation initiation factor eIF4A activity by a small molecule inhibitor of eukaryotic translation. Proc Natl Acad Sci U S A  102: 10460–10465.  <ul class="find" data-citedArticleID="1054252" data-doi="10.1073/pnas.0504249102"><li><a href="http://dx.doi.org/10.1073/pnas.0504249102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stimulation+of+mammalian+translation+initiation+factor+eIF4A+activity+by+a+small+molecule+inhibitor+of+eukaryotic+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stimulation+of+mammalian+translation+initiation+factor+eIF4A+activity+by+a+small+molecule+inhibitor+of+eukaryotic+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001583-Low1" id="pone.0001583-Low1"></a>Low WK, Dang Y, Schneider-Poetsch T, Shi Z, Choi NS, et al.  (2005) Inhibition of eukaryotic translation initiation by the marine natural product pateamine A. Mol Cell  20: 709–722.  <ul class="find" data-citedArticleID="1054284" data-doi="10.1016/j.molcel.2005.10.008"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.10.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+eukaryotic+translation+initiation+by+the+marine+natural+product+pateamine+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+eukaryotic+translation+initiation+by+the+marine+natural+product+pateamine+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001583-Bordeleau2" id="pone.0001583-Bordeleau2"></a>Bordeleau M-E, Mori A, Oberer M, Lindqvist L, Chard LS, et al.  (2006) Functional Characterization of IRESes by an inhibitor of the RNA helicase eIF4A. Nat Chem Biol  2: 213–220.  <ul class="find" data-citedArticleID="1054254" data-doi="10.1038/nchembio776"><li><a href="http://dx.doi.org/10.1038/nchembio776" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+Characterization+of+IRESes+by+an+inhibitor+of+the+RNA+helicase+eIF4A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+Characterization+of+IRESes+by+an+inhibitor+of+the+RNA+helicase+eIF4A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001583-Low2" id="pone.0001583-Low2"></a>Low WK, Dang Y, Bhat S, Romo D, Liu JO (2007) Substrate-Dependent Targeting of Eukaryotic Translation Initiation Factor 4A by Pateamine A: Negation of Domain-Linker Regulation of Activity. Chem Biol  14: 715–727.  <ul class="find" data-citedArticleID="1054286" data-doi="10.1016/j.chembiol.2007.05.012"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2007.05.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Substrate-Dependent+Targeting+of+Eukaryotic+Translation+Initiation+Factor+4A+by+Pateamine+A%3A+Negation+of+Domain-Linker+Regulation+of+Activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Substrate-Dependent+Targeting+of+Eukaryotic+Translation+Initiation+Factor+4A+by+Pateamine+A%3A+Negation+of+Domain-Linker+Regulation+of+Activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001583-Andersen1" id="pone.0001583-Andersen1"></a>Andersen CB, Ballut L, Johansen JS, Chamieh H, Nielsen KH, et al.  (2006) Structure of the exon junction core complex with a trapped DEAD-box ATPase bound to RNA. Science  313: 1968–1972.  <ul class="find" data-citedArticleID="1054246" data-doi="10.1126/science.1131981"><li><a href="http://dx.doi.org/10.1126/science.1131981" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+exon+junction+core+complex+with+a+trapped+DEAD-box+ATPase+bound+to+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+exon+junction+core+complex+with+a+trapped+DEAD-box+ATPase+bound+to+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001583-Bono1" id="pone.0001583-Bono1"></a>Bono F, Ebert J, Lorentzen E, Conti E (2006) The crystal structure of the exon junction complex reveals how it maintains a stable grip on mRNA. Cell  126: 713–725.  <ul class="find" data-citedArticleID="1054250" data-doi="10.1016/j.cell.2006.08.006"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.08.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+crystal+structure+of+the+exon+junction+complex+reveals+how+it+maintains+a+stable+grip+on+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+crystal+structure+of+the+exon+junction+complex+reveals+how+it+maintains+a+stable+grip+on+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001583-Oberer1" id="pone.0001583-Oberer1"></a>Oberer M, Marintchev A, Wagner G (2005) Structural basis for the enhancement of eIF4A helicase activity by eIF4G. Genes Dev  19: 2212–2223.  <ul class="find" data-citedArticleID="1054290" data-doi="10.1101/gad.1335305"><li><a href="http://dx.doi.org/10.1101/gad.1335305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+the+enhancement+of+eIF4A+helicase+activity+by+eIF4G." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+the+enhancement+of+eIF4A+helicase+activity+by+eIF4G.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001583-Sengoku1" id="pone.0001583-Sengoku1"></a>Sengoku T, Nureki O, Nakamura A, Kobayashi S, Yokoyama S (2006) Structural basis for RNA unwinding by the DEAD-box protein Drosophila Vasa. Cell  125: 287–300.  <ul class="find" data-citedArticleID="1054310" data-doi="10.1016/j.cell.2006.01.054"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.01.054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+RNA+unwinding+by+the+DEAD-box+protein+Drosophila+Vasa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+RNA+unwinding+by+the+DEAD-box+protein+Drosophila+Vasa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001583-Giaever1" id="pone.0001583-Giaever1"></a>Giaever G, Shoemaker DD, Jones TW, Liang H, Winzeler EA, et al.  (1999) Genomic profiling of drug sensitivities via induced haploinsufficiency. Nat Genet  21: 278–283.  <ul class="find" data-citedArticleID="1054276" data-doi="10.1038/6791"><li><a href="http://dx.doi.org/10.1038/6791" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+profiling+of+drug+sensitivities+via+induced+haploinsufficiency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+profiling+of+drug+sensitivities+via+induced+haploinsufficiency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001583-Oguro1" id="pone.0001583-Oguro1"></a>Oguro A, Ohtsu T, Svitkin YV, Sonenberg N, Nakamura Y (2003) RNA aptamers to initiation factor 4A helicase hinder cap-dependent translation by blocking ATP hydrolysis. RNA  9: 394–407.  <ul class="find" data-citedArticleID="1054292" data-doi="10.1261/rna.2161303"><li><a href="http://dx.doi.org/10.1261/rna.2161303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+aptamers+to+initiation+factor+4A+helicase+hinder+cap-dependent+translation+by+blocking+ATP+hydrolysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+aptamers+to+initiation+factor+4A+helicase+hinder+cap-dependent+translation+by+blocking+ATP+hydrolysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001583-Pestova1" id="pone.0001583-Pestova1"></a>Pestova TV, Kolupaeva VG (2002) The roles of individual eukaryotic translation initiation factors in ribosomal scanning and initiation codon selection. Genes Dev  16: 2906–2922.  <ul class="find" data-citedArticleID="1054300" data-doi="10.1101/gad.1020902"><li><a href="http://dx.doi.org/10.1101/gad.1020902" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+roles+of+individual+eukaryotic+translation+initiation+factors+in+ribosomal+scanning+and+initiation+codon+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+roles+of+individual+eukaryotic+translation+initiation+factors+in+ribosomal+scanning+and+initiation+codon+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001583-Shibuya1" id="pone.0001583-Shibuya1"></a>Shibuya T, Tange TO, Stroupe ME, Moore MJ (2006) Mutational analysis of human eIF4AIII identifies regions necessary for exon junction complex formation and nonsense-mediated mRNA decay. RNA  12: 360–374.  <ul class="find" data-citedArticleID="1054312" data-doi="10.1261/rna.2190706"><li><a href="http://dx.doi.org/10.1261/rna.2190706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+of+human+eIF4AIII+identifies+regions+necessary+for+exon+junction+complex+formation+and+nonsense-mediated+mRNA+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+of+human+eIF4AIII+identifies+regions+necessary+for+exon+junction+complex+formation+and+nonsense-mediated+mRNA+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001583-Delaglio1" id="pone.0001583-Delaglio1"></a>Delaglio F, Grzesiek S, Vuister GW, Zhu G, Pfeifer J, et al.  (1995) NMRPipe: a multidimensional spectral processing system based on UNIX pipes. J Biomol NMR  6: 277–293.  <ul class="find" data-citedArticleID="1054266" data-doi="10.1007/bf00197809"><li><a href="http://dx.doi.org/10.1007/bf00197809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NMRPipe%3A+a+multidimensional+spectral+processing+system+based+on+UNIX+pipes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NMRPipe%3A+a+multidimensional+spectral+processing+system+based+on+UNIX+pipes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001583-Bartels1" id="pone.0001583-Bartels1"></a>Bartels C, Xia T, Billeter M, Güntert P, Wüthrich K (1995) The program XEASY for computer-supported NMR spectral analysis of biological molecules. J Biomol NMR  5: 1–10.  <ul class="find" data-citedArticleID="1054248" data-doi="10.1007/bf00417486"><li><a href="http://dx.doi.org/10.1007/bf00417486" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+program+XEASY+for+computer-supported+NMR+spectral+analysis+of+biological+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+program+XEASY+for+computer-supported+NMR+spectral+analysis+of+biological+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001583-Matsuo1" id="pone.0001583-Matsuo1"></a>Matsuo H, Li H, McGuire AM, Fletcher CM, Gingras AC, et al.  (1997) Structure of translation factor eIF4E bound to m7GDP and interaction with 4E-binding protein. Nat Struct Biol  4: 717–724.  <ul class="find" data-citedArticleID="1054288" data-doi="10.1038/nsb0997-717"><li><a href="http://dx.doi.org/10.1038/nsb0997-717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+translation+factor+eIF4E+bound+to+m7GDP+and+interaction+with+4E-binding+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+translation+factor+eIF4E+bound+to+m7GDP+and+interaction+with+4E-binding+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001583-Shibuya2" id="pone.0001583-Shibuya2"></a>Shibuya T, Tange TO, Sonenberg N, Moore MJ (2004) eIF4AIII binds spliced mRNA in the exon junction complex and is essential for nonsense-mediated decay. Nat Struct Mol Biol  11: 346–351.  <ul class="find" data-citedArticleID="1054314" data-doi="10.1038/nsmb750"><li><a href="http://dx.doi.org/10.1038/nsmb750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=eIF4AIII+binds+spliced+mRNA+in+the+exon+junction+complex+and+is+essential+for+nonsense-mediated+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22eIF4AIII+binds+spliced+mRNA+in+the+exon+junction+complex+and+is+essential+for+nonsense-mediated+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001583-Lorsch1" id="pone.0001583-Lorsch1"></a>Lorsch JR, Herschlag D (1998) The DEAD box protein eIF4A. 1. A minimal kinetic and thermodynamic framework reveals coupled binding of RNA and nucleotide. Biochemistry  37: 2180–2193.  <ul class="find" data-citedArticleID="1054282" data-doi="10.1021/bi972430g"><li><a href="http://dx.doi.org/10.1021/bi972430g" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DEAD+box+protein+eIF4A.+1.+A+minimal+kinetic+and+thermodynamic+framework+reveals+coupled+binding+of+RNA+and+nucleotide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DEAD+box+protein+eIF4A.+1.+A+minimal+kinetic+and+thermodynamic+framework+reveals+coupled+binding+of+RNA+and+nucleotide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001583-Sonenberg1" id="pone.0001583-Sonenberg1"></a>Sonenberg N (1981) ATP/Mg++-dependent cross-linking of cap binding proteins to the 5' end of eukaryotic mRNA. Nucleic Acids Res  9: 1643–1656.  <ul class="find" data-citedArticleID="1054318" data-doi="10.1093/nar/9.7.1643"><li><a href="http://dx.doi.org/10.1093/nar/9.7.1643" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATP%2FMg%2B%2B-dependent+cross-linking+of+cap+binding+proteins+to+the+5%27+end+of+eukaryotic+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATP%2FMg%2B%2B-dependent+cross-linking+of+cap+binding+proteins+to+the+5%27+end+of+eukaryotic+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001583-Rozen1" id="pone.0001583-Rozen1"></a>Rozen F, Edery I, Meerovitch K, Dever TE, Merrick WC, et al.  (1990) Bidirectional RNA helicase activity of eucaryotic translation initiation factors 4A and 4F. Mol Cell Biol  10: 1134–1144.  <ul class="find" data-citedArticleID="1054308"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bidirectional+RNA+helicase+activity+of+eucaryotic+translation+initiation+factors+4A+and+4F.&amp;auth=&amp;atitle=Bidirectional+RNA+helicase+activity+of+eucaryotic+translation+initiation+factors+4A+and+4F." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bidirectional+RNA+helicase+activity+of+eucaryotic+translation+initiation+factors+4A+and+4F." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bidirectional+RNA+helicase+activity+of+eucaryotic+translation+initiation+factors+4A+and+4F.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001583-Altmann1" id="pone.0001583-Altmann1"></a>Altmann M, Trachsel H (2002) Cell Free Translation Systems; Spirin AS, editor. Heidelberg: Springer-Verlag Berlin.   <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0001583-Otero1" id="pone.0001583-Otero1"></a>Otero LJ, Devaux A, Standart N (2001) A 250-nucleotide UA-rich element in the 3' untranslated region of Xenopus laevis Vg1 mRNA represses translation both in vivo and in vitro. RNA  7: 1753–1767.  <ul class="find" data-citedArticleID="1054294"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+250-nucleotide+UA-rich+element+in+the+3%27+untranslated+region+of+Xenopus+laevis+Vg1+mRNA+represses+translation+both+in+vivo+and+in+vitro.&amp;auth=&amp;atitle=A+250-nucleotide+UA-rich+element+in+the+3%27+untranslated+region+of+Xenopus+laevis+Vg1+mRNA+represses+translation+both+in+vivo+and+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+250-nucleotide+UA-rich+element+in+the+3%27+untranslated+region+of+Xenopus+laevis+Vg1+mRNA+represses+translation+both+in+vivo+and+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+250-nucleotide+UA-rich+element+in+the+3%27+untranslated+region+of+Xenopus+laevis+Vg1+mRNA+represses+translation+both+in+vivo+and+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001583-Cencic1" id="pone.0001583-Cencic1"></a>Cencic R, Yan Y, Pelletier J (2007) Homogenous time resolved fluorescence assay to identify modulators of cap-dependent translation initiation. Comb Chem High Throughput Screen  10: 181–188.  <ul class="find" data-citedArticleID="1054256" data-doi="10.2174/138620707780126688"><li><a href="http://dx.doi.org/10.2174/138620707780126688" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homogenous+time+resolved+fluorescence+assay+to+identify+modulators+of+cap-dependent+translation+initiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homogenous+time+resolved+fluorescence+assay+to+identify+modulators+of+cap-dependent+translation+initiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001583-Rocak1" id="pone.0001583-Rocak1"></a>Rocak S, Linder P (2004) DEAD-box proteins: the driving forces behind RNA metabolism. Nat Rev Mol Cell Biol  5: 232–241.  <ul class="find" data-citedArticleID="1054304" data-doi="10.1038/nrm1335"><li><a href="http://dx.doi.org/10.1038/nrm1335" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DEAD-box+proteins%3A+the+driving+forces+behind+RNA+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DEAD-box+proteins%3A+the+driving+forces+behind+RNA+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001583-Coppolecchia1" id="pone.0001583-Coppolecchia1"></a>Coppolecchia R, Buser P, Stotz A, Linder P (1993) A new yeast translation initiation factor suppresses a mutation in the eIF-4A RNA helicase. EMBO J  12: 4005–4011.  <ul class="find" data-citedArticleID="1054262"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+new+yeast+translation+initiation+factor+suppresses+a+mutation+in+the+eIF-4A+RNA+helicase.&amp;auth=&amp;atitle=A+new+yeast+translation+initiation+factor+suppresses+a+mutation+in+the+eIF-4A+RNA+helicase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+yeast+translation+initiation+factor+suppresses+a+mutation+in+the+eIF-4A+RNA+helicase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+yeast+translation+initiation+factor+suppresses+a+mutation+in+the+eIF-4A+RNA+helicase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.XML" value="109106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.PDF" value="428255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g001.PNG_L" value="3372667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g001.PNG_M" value="179867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g001.PNG_S" value="17474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g001.TIF" value="5664266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g001.PNG_I" value="121889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g002.PNG_L" value="453516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g002.PNG_M" value="57762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g002.PNG_S" value="8202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g002.TIF" value="530476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g002.PNG_I" value="22852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g003.PNG_L" value="423797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g003.PNG_M" value="50399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g003.PNG_S" value="8399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g003.TIF" value="670494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g003.PNG_I" value="17734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g004.PNG_L" value="1946827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g004.PNG_M" value="75207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g004.PNG_S" value="9609"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g004.TIF" value="2066174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g004.PNG_I" value="28119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g005.PNG_L" value="1520096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g005.PNG_M" value="70640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g005.PNG_S" value="8055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g005.TIF" value="1679844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g005.PNG_I" value="23359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g006.PNG_L" value="352639"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g006.PNG_M" value="25943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g006.PNG_S" value="5587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g006.TIF" value="465396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.g006.PNG_I" value="9231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.s001.DOC" value="53760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.s002.DOC" value="30208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.s003.DOC" value="68608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.s004.TIF" value="10002896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.s005.TIF" value="5783588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.s006.TIF" value="4084672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001583.s007.TIF" value="4393754"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001583&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001583" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001583&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001583&volume=&issue=&title=Selective Pharmacological Targeting of a DEAD Box RNA Helicase&author_name=Lisa%20Lindqvist%2C%20Monika%20Oberer%2C%20Mikhail%20Reibarkh%2C%20Regina%20Cencic%2C%20Marie-Eve%20Bordeleau%2C%20Emily%20Vogt%2C%20Assen%20Marintchev%2C%20Junichi%20Tanaka%2C%20Francois%20Fagotto%2C%20Michael%20Altmann%2C%20Gerhard%20Wagner%2C%20Jerry%20Pelletier&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001583" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001583" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001583" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001583&amp;t=Selective%20Pharmacological%20Targeting%20of%20a%20DEAD%20Box%20RNA%20Helicase" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001583&title=Selective%20Pharmacological%20Targeting%20of%20a%20DEAD%20Box%20RNA%20Helicase&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001583&amp;title=Selective%20Pharmacological%20Targeting%20of%20a%20DEAD%20Box%20RNA%20Helicase" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001583" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001583&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Selective Pharmacological Targeting of a DEAD Box RNA Helicase';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001583';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Selective%20Pharmacological%20Targeting%20of%20a%20DEAD%20Box%20RNA%20Helicase http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001583" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001583" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22ATP+hydrolysis%22" title="Search for articles in the subject area:'ATP hydrolysis'"><div class="flagText">ATP hydrolysis</div></a>
              <div data-categoryid="20131" data-articleid="25992"
                   data-categoryname="ATP hydrolysis"
                   class="flagImage" title="Flag 'ATP hydrolysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Luciferase%22" title="Search for articles in the subject area:'Luciferase'"><div class="flagText">Luciferase</div></a>
              <div data-categoryid="31895" data-articleid="25992"
                   data-categoryname="Luciferase"
                   class="flagImage" title="Flag 'Luciferase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="25992"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="25992"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+translation%22" title="Search for articles in the subject area:'Protein translation'"><div class="flagText">Protein translation</div></a>
              <div data-categoryid="24691" data-articleid="25992"
                   data-categoryname="Protein translation"
                   class="flagImage" title="Flag 'Protein translation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="25992"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="25992"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Xenopus+oocytes%22" title="Search for articles in the subject area:'Xenopus oocytes'"><div class="flagText">Xenopus oocytes</div></a>
              <div data-categoryid="42713" data-articleid="25992"
                   data-categoryname="Xenopus oocytes"
                   class="flagImage" title="Flag 'Xenopus oocytes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8469'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5681'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=535&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
