

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001778"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001778" />

    <meta name="citation_title" content="Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment"/>
    <meta itemprop="name" content="Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment"/>

      <meta name="citation_author" content="Paul Wilmes"/>
            <meta name="citation_author_institution" content="School of Environmental Sciences, University of East Anglia, Norwich, United Kingdom"/>
      <meta name="citation_author" content="Margaret Wexler"/>
            <meta name="citation_author_institution" content="School of Biological Sciences, University of East Anglia, Norwich, United Kingdom"/>
      <meta name="citation_author" content="Philip L. Bond"/>
            <meta name="citation_author_institution" content="School of Environmental Sciences, University of East Anglia, Norwich, United Kingdom"/>
            <meta name="citation_author_institution" content="School of Biological Sciences, University of East Anglia, Norwich, United Kingdom"/>

    <meta name="citation_date" content="2008/3/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001778.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1778"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Impact of culture-independent studies on the emerging phylogenetic view of bacterial diversity.; citation_author=P Hugenholtz; citation_author=BM Goebel; citation_author=NR Pace; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=1; citation_pages=6793-6793; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Community genomics among stratified microbial assemblages in the ocean's interior.; citation_author=EF DeLong; citation_author=CM Preston; citation_author=T Mincer; citation_author=V Rich; citation_author=SJ Hallam; citation_journal_title=Science; citation_volume=311; citation_number=2; citation_pages=496-503; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea.; citation_author=JC Venter; citation_author=K Remington; citation_author=JF Heidelberg; citation_author=AL Halpern; citation_author=D Rusch; citation_journal_title=Science; citation_volume=304; citation_number=3; citation_pages=66-74; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Metagenomic analysis of two enhanced biological phosphorus removal (EBPR) sludge communities.; citation_author=H García-Martín; citation_author=N Ivanovan; citation_author=V Kunin; citation_author=F Warnecke; citation_author=KW Barry; citation_journal_title=Nat Biotechnol; citation_volume=24; citation_number=4; citation_pages=1263-1269; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The application of two-dimensional polyacrylamide gel electrophoresis and downstream analyses to a mixed community of prokaryotic microorganisms.; citation_author=P Wilmes; citation_author=PL Bond; citation_journal_title=Environ Microbiol; citation_volume=6; citation_number=5; citation_pages=911-920; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Metaproteomics: studying functional gene expression in microbial ecosystems.; citation_author=P Wilmes; citation_author=PL Bond; citation_journal_title=Trends Microbiol; citation_volume=14; citation_number=6; citation_pages=92-97; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Metaproteomic analysis of Chesapeake Bay microbial communities.; citation_author=J Kan; citation_author=TE Hanson; citation_author=JM Ginter; citation_author=K Wang; citation_author=F Chen; citation_journal_title=Saline Systems; citation_volume=1; citation_number=7; citation_pages=7-19; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Metaproteomics approach to study the functionality of the microbiota in the human infant gastrointestinal tract.; citation_author=ES Klaassens; citation_author=WM de Vos; citation_author=EE Vaughan; citation_journal_title=Appl Environ Microbiol; citation_volume=73; citation_number=8; citation_pages=1388-1392; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Metaproteomics: a new approach for studying functional microbial ecology.; citation_author=PA Maron; citation_author=L Ranjard; citation_author=C Mougel; citation_author=P Lemanceau; citation_journal_title=Microb Ecol; citation_volume=53; citation_number=9; citation_pages=486-493; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Functional metaproteome analysis of protein extracts from contaminated soil and groundwater.; citation_author=D Benndorf; citation_author=GU Balcke; citation_author=H Harms; citation_author=M von Bergen; citation_journal_title=ISME Journal; citation_volume=1; citation_number=10; citation_pages=224-234; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A proteomic fingerprint of dissolved organic carbon and of soil particles.; citation_author=WX Schulze; citation_author=G Gleixner; citation_author=K Kaiser; citation_author=G Guggenberger; citation_author=M Mann; citation_journal_title=Oecologia; citation_volume=142; citation_number=11; citation_pages=335-343; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Metaproteomic analysis of a bacterial community response to cadmium exposure.; citation_author=CMR Lacerda; citation_author=LH Choe; citation_author=KF Reardon; citation_journal_title=J Proteome Res; citation_volume=6; citation_number=12; citation_pages=1145-1152; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Community proteomics of a natural microbial biofilm.; citation_author=RJ Ram; citation_author=NC VerBerkmoes; citation_author=MP Thelen; citation_author=GW Tyson; citation_author=BJ Baker; citation_journal_title=Science; citation_volume=308; citation_number=13; citation_pages=1915-1920; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The microbiology of biological phosphorus removal in activated sludge systems.; citation_author=RJ Seviour; citation_author=T Mino; citation_author=M Onuki; citation_journal_title=FEMS Microbiol Rev; citation_volume=27; citation_number=14; citation_pages=99-127; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A review and update of the microbiology of enhanced biological phosphorus removal in wastewater treatment plants.; citation_author=LL Blackall; citation_author=G Crocetti; citation_author=AM Saunders; citation_author=PL Bond; citation_journal_title=Antonie Van Leeuwenhoek; citation_volume=81; citation_number=15; citation_pages=681-691; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of some of the major groups of bacteria in efficient and nonefficient biological phosphorus removal activated sludge systems.; citation_author=PL Bond; citation_author=R Erhart; citation_author=M Wagner; citation_author=J Keller; citation_author=LL Blackall; citation_journal_title=Appl Environ Microbiol; citation_volume=65; citation_number=16; citation_pages=4077-4084; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Bacterial community structures of phosphate-removing and non- phosphate-removing activated sludges from sequencing batch reactors.; citation_author=PL Bond; citation_author=P Hugenholtz; citation_author=J Keller; citation_author=LL Blackall; citation_journal_title=Appl Environ Microbiol; citation_volume=61; citation_number=17; citation_pages=1910-1916; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Identification of polyphosphate-accumulating organisms and design of 16S rRNA-directed probes for their detection and quantitation.; citation_author=GR Crocetti; citation_author=P Hugenholtz; citation_author=PL Bond; citation_author=A Schuler; citation_author=J Keller; citation_journal_title=Appl Environ Microbiol; citation_volume=66; citation_number=18; citation_pages=1175-1182; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Enrichment, phylogenetic analysis and detection of a bacterium that performs enhanced biological phosphate removal in activated sludge.; citation_author=RPX Hesselmann; citation_author=C Werlen; citation_author=D Hahn; citation_author=JR van der Meer; citation_author=AJB Zehnder; citation_journal_title=Syst Appl Microbiol; citation_volume=22; citation_number=19; citation_pages=454-465; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Long-term population dynamics and in situ physiology in activated sludge systems with enhanced biological phosphorus removal operated with and without nitrogen removal.; citation_author=N Lee; citation_author=PH Nielsen; citation_author=H Aspegren; citation_author=M Henze; citation_author=KH Schleifer; citation_journal_title=Syst Appl Microbiol; citation_volume=26; citation_number=20; citation_pages=211-227; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stoichiometry and kinetics of acetate uptake under anaerobic conditions by an enriched culture of phosphorus-accumulating organisms at different pHs.; citation_author=CDM Filipe; citation_author=GT Daigger; citation_author=CPL Grady; citation_journal_title=Biotechnol Bioeng; citation_volume=76; citation_number=21; citation_pages=32-43; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparison of acetate and propionate uptake by polyphosphate accumulating organisms and glycogen accumulating organisms.; citation_author=A Oehmen; citation_author=ZG Yuan; citation_author=LL Blackall; citation_author=J Keller; citation_journal_title=Biotechnol Bioeng; citation_volume=91; citation_number=22; citation_pages=162-168; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microbiology of the ‘G-bacteria’ in activated sludge.; citation_author=RJ Seviour; citation_author=AM Maszenan; citation_author=JA Soddell; citation_author=V Tandoi; citation_author=BKC Patel; citation_journal_title=Environ Microbiol; citation_volume=2; citation_number=23; citation_pages=581-593; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Polyphosphate kinase from activated sludge performing enhanced biological phosphorus removal.; citation_author=KD McMahon; citation_author=MA Dojka; citation_author=NR Pace; citation_author=D Jenkins; citation_author=JD Keasling; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=24; citation_pages=4971-4978; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Towards exposure of elusive metabolic mixed-culture processes: the application of metaproteomic analyses to activated sludge.; citation_author=P Wilmes; citation_author=PL Bond; citation_journal_title=Water Sci Technol; citation_volume=54; citation_number=25; citation_pages=217-226; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Are some putative glycogen accumulating organisms (GAO) in anaerobic: aerobic activated sludge systems members of the alpha-Proteobacteria?; citation_author=M Beer; citation_author=YH Kong; citation_author=RJ Seviour; citation_journal_title=Microbiology; citation_volume=150; citation_number=26; citation_pages=2267-2275; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification and occurrence of tetrad-forming Alphaproteobacteria in anaerobic-aerobic activated sludge processes.; citation_author=MT Wong; citation_author=FM Tan; citation_author=WJ Ng; citation_author=WT Liu; citation_journal_title=Microbiology; citation_volume=150; citation_number=27; citation_pages=3741-3748; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular insight into extreme copper resistance in the extremophilic archaeon ‘Ferroplasma acidarmanus’ Fer1.; citation_author=C Baker-Austin; citation_author=M Dopson; citation_author=M Wexler; citation_author=RG Sawers; citation_author=PL Bond; citation_journal_title=Microbiology; citation_volume=151; citation_number=28; citation_pages=2637-2646; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluation of two-dimensional gel electrophoresis-based proteome analysis technology.; citation_author=SP Gygi; citation_author=GL Corthals; citation_author=Y Zhang; citation_author=Y Rochon; citation_author=R Aebersold; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=29; citation_pages=9390-9395; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Lipase and protease extraction from activated sludge.; citation_author=A Gessesse; citation_author=T Dueholm; citation_author=SB Petersen; citation_author=PH Nielsen; citation_journal_title=Water Res; citation_volume=37; citation_number=30; citation_pages=3652-3657; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The integrated microbial genomes (IMG) system.; citation_author=VM Markowitz; citation_author=F Korzeniewski; citation_author=K Palaniappan; citation_author=E Szeto; citation_author=G Werner; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=31; citation_pages=D344-D348; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Process design for microbial plastic factories: metabolic engineering of polyhydroxyalkanoates.; citation_author=IS Aldor; citation_author=JD Keasling; citation_journal_title=Curr Opin Biotechnol; citation_volume=14; citation_number=32; citation_pages=475-483; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetics and biochemistry of polyhydroxyalkanoate granule self-assembly: The key role of polyester synthases.; citation_author=BHA Rehm; citation_journal_title=Biotechnol Lett; citation_volume=28; citation_number=33; citation_pages=207-213; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Neutral lipid bodies in prokaryotes: Recent insights into structure, formation, and relationship to eukaryotic lipid depots.; citation_author=M Waltermann; citation_author=A Steinbuchel; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=34; citation_pages=3607-3619; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Poly [(R)-3-hydroxybutyrate] formation in Escherichia coli from glucose through an enoyl-CoA hydratase-mediated pathway.; citation_author=S Sato; citation_author=CT Nomura; citation_author=H Abe; citation_author=Y Doi; citation_author=T Tsuge; citation_journal_title=J Biosci Bioeng; citation_volume=103; citation_number=35; citation_pages=38-44; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Anaerobic metabolism of bacteria performing enhanced biological phosphate removal.; citation_author=RPX Hesselmann; citation_author=R Von Rummell; citation_author=SM Resnick; citation_author=R Hany; citation_author=AJB Zehnder; citation_journal_title=Water Res; citation_volume=34; citation_number=36; citation_pages=3487-3494; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Biochemical model for enhanced biological phosphorus removal.; citation_author=Y Comeau; citation_author=KJ Hall; citation_author=REW Hancock; citation_author=WK Oldham; citation_journal_title=Water Res; citation_volume=20; citation_number=37; citation_pages=1511-1521; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Evaluation of biochemical models for biological excess phosphorus removal.; citation_author=MC Wentzel; citation_author=LH Lötter; citation_author=GA Ekama; citation_author=RE Loewenthal; citation_author=GVR Marais; citation_journal_title=Water Sci Technol; citation_volume=23; citation_number=38; citation_pages=567-576; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Model for carbon metabolism in biological phosphorus removal processes based on in vivo 13C-NMR labelling experiments.; citation_author=H Pereira; citation_author=PC Lemos; citation_author=MAM Reis; citation_author=JPSG Crespo; citation_author=MJT Carrondo; citation_journal_title=Water Res; citation_volume=30; citation_number=39; citation_pages=2128-2138; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Effect of phosphorus accumulation on acetate metabolism in the biological phosphorus removal process.; citation_author=T Mino; citation_author=V Arun; citation_author=Y Tsuzuki; citation_author=T Matsuo; citation_number=40; citation_pages=27-38; citation_date=1987; citation_publisher=Pergamon Press, Oxford; " />
      <meta name="citation_reference" content="citation_title=Use of metabolic inhibitors and gas chromatography/mass spectrometry to study poly-beta-hydroxyalkanoates metabolism involving cryptic nutrients in enhanced biological phosphorus removal systems.; citation_author=TM Louie; citation_author=TJ Mah; citation_author=W Oldham; citation_author=WD Ramey; citation_journal_title=Water Res; citation_volume=34; citation_number=41; citation_pages=1507-1514; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Metabolic model for acetate uptake by a mixed culture of phosphate- and glycogen-accumulating organisms under anaerobic conditions.; citation_author=N Yagci; citation_author=N Artan; citation_author=EU Cokgor; citation_author=CW Randall; citation_author=D Orhon; citation_journal_title=Biotechnol Bioeng; citation_volume=84; citation_number=42; citation_pages=359-373; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phosphate transport in prokaryotes: molecules, mediators and mechanisms.; citation_author=HW van Veen; citation_journal_title=Antonie Van Leeuwenhoek; citation_volume=72; citation_number=43; citation_pages=299-315; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Fine-scale differences between Accumulibacter-like bacteria in enhanced biological phosphorus removal activated sludge.; citation_author=S He; citation_author=AZ Gu; citation_author=KD McMahon; citation_journal_title=Water Sci Technol; citation_volume=54; citation_number=44; citation_pages=111-117; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Strain-resolved community proteomics reveals recombining genomes of acidophilic bacteria.; citation_author=I Lo; citation_author=VJ Denef; citation_author=NC VerBerkmoes; citation_author=MB Shah; citation_author=D Goltsman; citation_journal_title=Nature; citation_volume=446; citation_number=45; citation_pages=537-541; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Orbitrap: a new mass spectrometer.; citation_author=QZ Hu; citation_author=RJ Noll; citation_author=HY Li; citation_author=A Makarov; citation_author=M Hardman; citation_journal_title=J Mass Spectrom; citation_volume=40; citation_number=46; citation_pages=430-443; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic oligodeoxynucleotide probes for the major subclasses of proteobacteria: problems and solutions.; citation_author=W Manz; citation_author=R Amann; citation_author=W Ludwig; citation_author=M Wagner; citation_author=KH Schleifer; citation_journal_title=Syst Appl Microbiol; citation_volume=15; citation_number=47; citation_pages=593-600; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=In situ probing of Gram-positive bacteria with high DNA G+C content using 23S rRNA-targeted oligonucleotides.; citation_author=C Roller; citation_author=M Wagner; citation_author=R Amann; citation_author=W Ludwig; citation_author=K-H Schleifer; citation_journal_title=Microbiology; citation_volume=140; citation_number=48; citation_pages=2849-2858; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Glycogen-accumulating organisms in laboratory-scale and full-scale wastewater treatment processes.; citation_author=GR Crocetti; citation_author=JF Banfield; citation_author=J Keller; citation_author=PL Bond; citation_author=LL Blackall; citation_journal_title=Microbiology; citation_volume=148; citation_number=49; citation_pages=3353-3364; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The domain-specific probe EUB338 is insufficient for the detection of all Bacteria: Development and evaluation of a more comprehensive probe set.; citation_author=H Daims; citation_author=A Bruhl; citation_author=R Amann; citation_author=KH Schleifer; citation_author=M Wagner; citation_journal_title=Syst Appl Microbiol; citation_volume=22; citation_number=50; citation_pages=434-444; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Population changes in a biofilm reactor for phosphorus removal as evidenced by the use of FISH.; citation_author=CM Falkentoft; citation_author=E Muller; citation_author=P Arnz; citation_author=P Harremoes; citation_author=H Mosbaek; citation_journal_title=Water Res; citation_volume=36; citation_number=51; citation_pages=491-500; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Primary and secondary metabolism, and post-translational protein modifications, as portrayed by proteomic analysis of Streptomyces coelicolor.; citation_author=AR Hesketh; citation_author=G Chandra; citation_author=AD Shaw; citation_author=JJ Rowland; citation_author=DB Kell; citation_journal_title=Mol Microbiol; citation_volume=46; citation_number=52; citation_pages=917-932; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Artemis: sequence visualization and annotation.; citation_author=K Rutherford; citation_author=J Parkhill; citation_author=J Crook; citation_author=T Horsnell; citation_author=P Rice; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=53; citation_pages=944-945; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001778" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment"/>
    <meta name="twitter:description" content="BackgroundThrough identification of highly expressed proteins from a mixed culture activated sludge system this study provides functional evidence of microbial transformations important for enhanced biological phosphorus removal (EBPR).Methodology/Principal FindingsA laboratory-scale sequencing batch reactor was successfully operated for different levels of EBPR, removing around 25, 40 and 55 mg/l P. The microbial communities were dominated by the uncultured polyphosphate-accumulating organism &ldquo;Candidatus Accumulibacter phosphatis&rdquo;. When EBPR failed, the sludge was dominated by tetrad-forming &alpha;-Proteobacteria. Representative and reproducible 2D gel protein separations were obtained for all sludge samples. 638 protein spots were matched across gels generated from the phosphate removing sludges. 111 of these were excised and 46 proteins were identified using recently available sludge metagenomic sequences. Many of these closely match proteins from &ldquo;Candidatus Accumulibacter phosphatis&rdquo; and could be directly linked to the EBPR process. They included enzymes involved in energy generation, polyhydroxyalkanoate synthesis, glycolysis, gluconeogenesis, glycogen synthesis, glyoxylate/TCA cycle, fatty acid &beta; oxidation, fatty acid synthesis and phosphate transport. Several proteins involved in cellular stress response were detected.Conclusions/SignificanceImportantly, this study provides direct evidence linking the metabolic activities of &ldquo;Accumulibacter&rdquo; to the chemical transformations observed in EBPR. Finally, the results are discussed in relation to current EBPR metabolic models."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001778.g004"/>

  <meta property="og:title" content="Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001778" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9685'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8949'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1367&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001778">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001778" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001778&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul Wilmes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Environmental Sciences, University of East Anglia, Norwich, United Kingdom
                </p>

                <p>
                    Current address: Department of Earth and Planetary Sciences, University of California, Berkeley, California, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Margaret Wexler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biological Sciences, University of East Anglia, Norwich, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philip L. Bond
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:phil.bond@uq.edu.au">phil.bond@uq.edu.au</a></p>

                <p>Affiliations:
                  School of Environmental Sciences, University of East Anglia, Norwich, United Kingdom, 
                  School of Biological Sciences, University of East Anglia, Norwich, United Kingdom
                </p>

                <p>
                    Current address: Advanced Water Management Centre, The University of Queensland, Brisbane, Queensland, Australia
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 12, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001778</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001778" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001778" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001778" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001778" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001778-t001" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001778-g001" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001778-g002" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001778-g003" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001778-t002" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001778-g004" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001778">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001778" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Through identification of highly expressed proteins from a mixed culture activated sludge system this study provides functional evidence of microbial transformations important for enhanced biological phosphorus removal (EBPR).</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>A laboratory-scale sequencing batch reactor was successfully operated for different levels of EBPR, removing around 25, 40 and 55 mg/l P. The microbial communities were dominated by the uncultured polyphosphate-accumulating organism “<em>Candidatus</em> Accumulibacter phosphatis”. When EBPR failed, the sludge was dominated by tetrad-forming α-<em>Proteobacteria</em>. Representative and reproducible 2D gel protein separations were obtained for all sludge samples. 638 protein spots were matched across gels generated from the phosphate removing sludges. 111 of these were excised and 46 proteins were identified using recently available sludge metagenomic sequences. Many of these closely match proteins from “<em>Candidatus</em> Accumulibacter phosphatis” and could be directly linked to the EBPR process. They included enzymes involved in energy generation, polyhydroxyalkanoate synthesis, glycolysis, gluconeogenesis, glycogen synthesis, glyoxylate/TCA cycle, fatty acid β oxidation, fatty acid synthesis and phosphate transport. Several proteins involved in cellular stress response were detected.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Importantly, this study provides direct evidence linking the metabolic activities of <em>“Accumulibacter”</em> to the chemical transformations observed in EBPR. Finally, the results are discussed in relation to current EBPR metabolic models.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wilmes P, Wexler M, Bond PL (2008) Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment. PLoS ONE 3(3):
          e1778.
            doi:10.1371/journal.pone.0001778</p><p><strong>Academic Editor: </strong>Anna-Louise Reysenbach, Portland State University, United States of America</p><p><strong>Received:</strong> August 23, 2007; <strong>Accepted:</strong> January 31, 2008; <strong>Published:</strong> March 12, 2008</p><p><strong>Copyright:</strong> © 2008 Wilmes et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This project received financial support from Anglian Water. PW was supported by a School of Environmental Sciences Studentship, a British Council Chevening Scholarship, a short-term research scholarship from the German Academic Exchange Service and a research scholarship from the Luxembourg Ministry of Culture, Higher Education and Research. PLB and MW received support from the Biotechnology and Biological Sciences Research Council, grant reference no. BB/D013348/1.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>There is increasing interest to understand microbial community compositions and functions directly within their respective environments. Molecular analysis of environmental samples, mostly by analysis of 16S rRNA genes, has greatly improved our knowledge of the vast microbial diversity <a href="#pone.0001778-Hugenholtz1">[1]</a>. More recently, large metagenomic sequencing projects that analyse genomic DNA directly from environmental samples, are providing much detail of the genetic diversity and potential within selected environments, e.g. seawater samples <a href="#pone.0001778-DeLong1">[2]</a>, <a href="#pone.0001778-Venter1">[3]</a> and activated sludge <a href="#pone.0001778-GarcaMartn1">[4]</a>. A huge challenge is to couple this improved knowledge of microbial diversity with functional details of these microbial ecosystems. As most of the microbial biomass in environmental samples is presently unobtainable as isolated pure cultures, this effort requires <em>in situ</em> approaches.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In recent work, transcriptomic and proteomic analyses, traditionally used for study of pure cultures, are being applied to detect expression profiles and provide functional insight directly from mixed microbial environmental samples. Our recent work established for the first time that a proteomics approach could be successfully applied to examine protein expression in environmental samples such as activated sludge <a href="#pone.0001778-Wilmes1">[5]</a>. Since then there has been only a handful of studies describing mixed culture proteomics (termed metaproteomics) <a href="#pone.0001778-Wilmes2">[6]</a>. These include examination of protein expression profiles from an estuary transect <a href="#pone.0001778-Kan1">[7]</a>, infant fecal samples <a href="#pone.0001778-Klaassens1">[8]</a>, freshwater samples following exposure to heavy metals <a href="#pone.0001778-Maron1">[9]</a> and contaminated soil and groundwater <a href="#pone.0001778-Benndorf1">[10]</a>. Proteomic analysis of soil and water was used to determine microbial taxonomic groups in those environments <a href="#pone.0001778-Schulze1">[11]</a>, and differentially expressed proteins from bacterial communities following exposure to cadmium were detected <a href="#pone.0001778-Lacerda1">[12]</a>. Notably, a high-throughput proteomic study of acid mine biofilms has been performed <a href="#pone.0001778-Ram1">[13]</a>, in which a large number of proteins (~2,000) were identified; one novel protein was confirmed as a key component of energy conservation in that environment <a href="#pone.0001778-Ram1">[13]</a>. Consequently, despite the limited number of investigations, the metaproteomic approach has already highlighted its potential for providing functional insight into overall microbial ecosystem function <a href="#pone.0001778-Wilmes2">[6]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Biological wastewater treatment plants (WWTPs) employing activated sludge represent the most widely used biotechnological process on Earth. The removal of organic carbon and other nutrients, mainly nitrogen and phosphorus (P), is essential to avoid the deterioration of receiving surface waters <a href="#pone.0001778-Seviour1">[14]</a>. WWTPs can be engineered to enable enhanced biological phosphorus removal (EBPR). These WWTPs are characterised by an anaerobic treatment phase that precedes an aerobic phase. These systems select for particular bacteria, which accumulate large amounts of intracellular polyphosphate (polyP), causing the desired P removal during wastewater treatment. While EBPR is used globally with success, the systems do suffer intermittent periods of poor performance, and improvements of operation and performance are pressing in view of future constraints on the water cycle enhanced by global climate change.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>There is great interest to understand the biochemistry of EBPR. However, most details of the process remain elusive, and this is partly because, in spite of many attempts, the polyphosphate accumulating organisms (PAOs) responsible for EBPR have not yet been isolated <a href="#pone.0001778-Blackall1">[15]</a>. Nevertheless, based on the EBPR transformations and general bacterial metabolism, metabolic models have been derived to describe the energetic and substrate requirements. During the initial anaerobic phase, PAOs degrade stored polyP and glycogen, and synthesise polyhydroxyalkanoates (PHAs) from short chain volatile fatty acids (VFAs). In the subsequent aerobic stage, they store polyP and glycogen, and degrade PHAs. With the advent of molecular techniques, dominant PAOs in laboratory-scale EBPR systems are found to be members of the β-<em>Proteobacteria</em> and close relatives of <em>Rhodocyclus</em> spp. <a href="#pone.0001778-Bond1">[16]</a>–<a href="#pone.0001778-Hesselmann1">[19]</a>. This group of PAOs are tentatively named “<em>Candidatus</em> Accumulibacter phosphatis” (herein described as <em>“A. phosphatis”</em>; <a href="#pone.0001778-Hesselmann1">[19]</a>).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Although <em>“A. phosphatis”</em> remains uncultured, the phylogenetic identity of these dominant PAOs has recently provided opportunity for <em>in situ</em> investigations of microbial function. These studies have been many, and include use of MAR-FISH <a href="#pone.0001778-Lee1">[20]</a>, and estimations of EBPR stoichiometry and kinetics <a href="#pone.0001778-Filipe1">[21]</a>, <a href="#pone.0001778-Oehmen1">[22]</a>. A recent major achievement has been the acquisition of EBPR metagenomic sequences <a href="#pone.0001778-GarcaMartn1">[4]</a>. These were obtained from two EBPR laboratory-scale reactors operated in Australia and the United States, henceforth referred to as the OZ and US sludge, respectively. Both reactors were enriched for <em>“A. phosphatis”</em> allowing García-Martín <em>et al.</em> <a href="#pone.0001778-GarcaMartn1">[4]</a> to subsequently assemble a composite <em>“A. phosphatis”</em> genome from the US metagenomic sequences and infer the metabolic pathways employed by <em>“A. phosphatis”</em> during EBPR. This provides much information on the genetic blueprint for EBPR, however, it does not provide direct functional information, speculated metabolisms still require verification, and system dynamics remain elusive.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Metaproteomics can be used to study protein expression from a complex system and provide direct evidence of metabolic and physiological activities. Recently, we used a proteomic approach through a combination of two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) for quantitative protein detection and mass spectrometry-based protein identification <a href="#pone.0001778-Wilmes1">[5]</a>. The proteomic approach is now more feasible as the metagenomic sequences provide increased opportunity to identify proteins. In the present study, we compared protein expression in sludge with differing EBPR performance. We focused on identification of highly expressed proteins that would be central to the EBPR metabolism. Numerous proteins were detected that could be directly linked to EBPR metabolism and to the dominant PAO, <em>“A. phosphatis”</em>. Furthermore, we demonstrate the presence of highly expressed proteins whose activities have not been previously linked to EBPR and which, consequently, may need to be included in future metabolic models.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Generation of sludge with differing EBPR performance</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A sequencing batch reactor (SBR) was operated for four different levels of EBPR over a period of more that 100 days, by alteration of the phosphorus levels (as phosphate) in the reactor feed. Three sludges with good EBPR performance were obtained. For these sludges, removal of phosphate-P from the influent was complete at 28.5, 42.4, and 55.2 mg/l, and this was stable for at least three sludge ages in each case. These sludges were termed EBPR<sub>28</sub>, EBPR<sub>42</sub> and EBPR<sub>55</sub>, respectively (<a href="#pone-0001778-t001">Table 1</a>). Reactor cycle studies demonstrate that the three sludges carried out metabolic transformations typical of EBPR (<a href="#pone-0001778-t001">Table 1</a>, <a href="#pone.0001778.s005">Fig. S1</a>). This included the release of orthophosphate into the medium during the anaerobic phase followed by the concomitant uptake of the excreted orthophosphate during the aerobic phase. In contrast to the sludges above, a fourth sludge did not perform EBPR. When the SBR phosphate-P feed was increased to 70.7 mg/l the EBPR performance failed, and an average of 66.7 mg/l phosphate-P remained in the reactor effluent. This sludge was termed nEBPR<sub>70</sub>. Low levels of P transformations detected in the nEBPR<sub>70</sub> sludge's cycle study were consistent with the absence of EBPR (<a href="#pone-0001778-t001">Table 1</a> and <a href="#pone.0001778.s005">Fig. S1D</a>). The carbon transformations of the nEBPR<sub>70</sub> were similar to those observed in EBPR sludges, with anaerobic PHA accumulation (<a href="#pone-0001778-t001">Table 1</a>), although anaerobic acetate uptake was not complete (<a href="#pone.0001778.s005">Fig. S1D</a>). This performance is characteristic of glycogen accumulating organisms (GAOs) that are implicated in EBPR failure in anaerobic/aerobic activated sludge systems <a href="#pone.0001778-Seviour2">[23]</a>.</p>
<div class="figure" id="pone-0001778-t001"><div class="img"><a name="pone-0001778-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001778.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001778.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of the EBPR transformations detected at various stages of the sequence batch reactor (SBR) performance.</span></strong></p><span>doi:10.1371/journal.pone.0001778.t001</span></div>

<h4>Microbial community analyses of the EBPR and nEBPR sludge</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The sludge microbial communities were analysed by fluorescent <em>in situ</em> hybridisation (FISH). Most of the cells that stained positively with DAPI were identified as bacteria with the EUBMIX probes (98 %). The EBPR sludges, at each P increase level, were all dominated by organisms forming distinct coccoid clusters of cells (<a href="#pone-0001778-g001">Fig. 1A,B&amp;C</a>). The EBPR<sub>28</sub>, EBPR<sub>42</sub> and EBPR<sub>55</sub> sludges gave positive signals for the β-<em>Proteobacteria</em> specific probe (63, 68 and 72 % of EUBMIX binding cells, respectively; <a href="#pone.0001778.s002">Table S1</a>) and the <em>“A. phosphatis”</em> specific probe PAO651 (61, 67 and 69 % of EUBMIX binding cells, respectively; <a href="#pone.0001778.s002">Table S1</a>). These results were similar to those found for sludges with similar P removal performances and biomass P contents <a href="#pone.0001778-Wilmes1">[5]</a>, <a href="#pone.0001778-Crocetti1">[18]</a>, <a href="#pone.0001778-McMahon1">[24]</a>, <a href="#pone.0001778-Wilmes3">[25]</a>. The number of cells belonging to both β-<em>Proteobacteria</em> and <em>“A. phosphatis”</em> were found to be significantly different in the three EBPR sludges (ANOVA, P&lt;0.05) and, consequently, provide a causative link between the dominance of the <em>Rhodocyclus</em>-type PAO and P removal performance. High abundances of α-<em>Proteobacteria</em> were also observed in the EBPR<sub>28</sub>, EBPR<sub>42</sub> and EBPR<sub>55</sub> sludges, at 34, 29 and 25 %, respectively (<a href="#pone.0001778.s002">Table S1</a>). These consisted mainly of tetrad-arranged coccoid cells, as described previously <a href="#pone.0001778-Wilmes3">[25]</a>. Thus, the numbers of <em>Rhodocyclus</em>-type PAOs increased as the tetrad-forming α-<em>Proteobacteria</em> decreased. A possible explanation is that these two groups of organisms are in direct competition with one another as suggested previously <a href="#pone.0001778-Blackall1">[15]</a>.</p>
<div class="figure" id="pone-0001778-g001"><div class="img"><a name="pone-0001778-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Representative FISH micrographs of the activated sludges analysed in this study.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) EBPR<sub>28</sub> sludge, (B) EBPR<sub>42</sub> sludge, (C) EBPR<sub>55 </sub>sludge and (D) nEBPR<sub>70</sub> sludge. Cells detected with probe EUBMIX only are green (A, B, C and D). Cells detected with both EUBMIX and PAO651 probes (A, B and C) and cells detected with both EUBMIX and ALF1b probes (D) are yellow-orange. Highlighted area in pane d corresponds to magnified region hybridised only with the ALF1b probe in the top right hand corner. Images taken under the different excitation wavelengths for CY3 and FITC were combined using Adobe Photoshop. Cells were observed under <em>x</em> 630 magnification, bars = 10 µm.</p>
<span>doi:10.1371/journal.pone.0001778.g001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In contrast to the EBPR sludges, the nEBPR<sub>70</sub> sludge was dominated by the tetrad-arranged coccoid cells already observed in the EBPR sludges (<a href="#pone-0001778-g001">Fig. 1D</a>), and again identified as α-<em>Proteobacteria</em> (53 % of EUBMIX binding cells). The nEBPR<sub>70</sub> sludge still revealed a rather high abundance of β-<em>Proteobacteria</em> (41 % of EUBMIX binding cells), but <em>“A. phosphatis”</em> was present in very low numbers (&lt;1%; <a href="#pone.0001778.s002">Table S1</a>). The dominance of the tetrad-arranged α-<em>Proteobacteria</em> concurs with other investigations of anaerobic:aerobic non-P removing sludges <a href="#pone.0001778-Seviour1">[14]</a>, <a href="#pone.0001778-Beer1">[26]</a>, <a href="#pone.0001778-Wong1">[27]</a>. Thus, our study again implicates this subphylum and cell type as GAOs that can dominate when EBPR fails.</p>


<h4>Metaproteomics of the EBPR sludges</h4>
<h5>2D-PAGE analysis.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>For each P increase level, 2D-PAGE separations of the proteins extracted from the four distinct sludges were carried out (<a href="#pone-0001778-g002">Fig. 2</a>), and from the gel separations metaproteomic maps were generated. Liquid isoelectric focussing prior to 2D-PAGE was employed to improve the gel separations. For each sludge highly reproducible metaproteomic maps were obtained, with &gt;98 % of protein spots being matched within replicate gels (n = 3; <a href="#pone.0001778.s003">Table S2</a>). Additionally, the proteomic maps generated from the EBPR<sub>28</sub>, EBPR<sub>42</sub> and EBPR<sub>55</sub> sludges exhibited high similarity. Overall, a total of 638 spots were matched between all the gels for the EBPR sludges, and these conserved spots are represented on the “Master” gel (<a href="#pone-0001778-g003">Fig. 3</a>). The proteomic maps produced from nEBPR<sub>70</sub> sludge samples were distinct compared to those generated from the EBPR sludges, and were analysed separately.</p>
<div class="figure" id="pone-0001778-g002"><div class="img"><a name="pone-0001778-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Representative 2D-PAGE separations of proteins extracted from (A) the EBPR<sub>28</sub> sludge, (B) the EBPR<sub>42</sub> sludge, (C) the EBPR<sub>55 </sub>sludge and (D) the nEBPR<sub>70</sub> sludge.</span></strong></p><a id="article1.body1.sec2.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec1.fig1.caption1.p1"></a><p>Approximate protein molecular mass ranges are provided on the left and isoelectric point ranges are provided on the bottom of the gel images.</p>
<span>doi:10.1371/journal.pone.0001778.g002</span></div><div class="figure" id="pone-0001778-g003"><div class="img"><a name="pone-0001778-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Master 2D-PAGE gel of the EBPR matchset with excised protein spots highlighted.</span></strong></p><a id="article1.body1.sec2.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec3.sec1.fig2.caption1.p1"></a><p>Spot numbering corresponds to the numbering used in <a href="#pone-0001778-t002">Table 2</a>, and supporting information <a href="#pone.0001778.s004">Table S3</a>.</p>
<span>doi:10.1371/journal.pone.0001778.g003</span></div><a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>Each of the matched spots was quantified (by intensity and size) across the matchset of the three EBPR sludges (see <a href="#pone.0001778.s004">Table S3</a> for quantitative comparison). The spots were then ranked according to their respective quantities in the EBPR<sub>55 </sub>gel set. 111 protein spots with the highest quantities were chosen for excision from a separate EBPR<sub>55</sub> gel.</p>

<h5>Mass Spectrometry (MS) analysis and protein identification.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>Excised protein spots for which positive identifications were obtained are highlighted on the 2D-PAGE master gel (<a href="#pone-0001778-g003">Fig. 3</a>). Details on the excised protein spots identified are listed in <a href="#pone-0001778-t002">Table 2</a>, and additional details are presented in <a href="#pone.0001778.s004">Table S3</a>.</p>
<div class="figure" id="pone-0001778-t002"><div class="img"><a name="pone-0001778-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001778.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001778.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Identification and putative function of proteins excised from the EBPR<sub>55</sub> 2D-PAGE gel and analysed by mass spectrometry (spot numbers refer to those in <a href="#pone-0001778-g003">Fig. 3</a>).</span></strong></p><span>doi:10.1371/journal.pone.0001778.t002</span></div><a id="article1.body1.sec2.sec3.sec2.p2" name="article1.body1.sec2.sec3.sec2.p2"></a><p>Initially, 38 of the 111 chosen proteins were positively identified from their respective peptide mass fingerprints (by MALDI-ToF MS/MASCOT) searched against the EBPR sludge metagenomic databases (numbered 1–38; <a href="#pone-0001778-t002">Table 2</a>; see <a href="#s3">Materials and Methods</a> for details on the databases). Protein digests, which did not result in positive identifications using MALDI-ToF MS/MASCOT and for which clear MS spectra had been obtained, were further analysed using Q-ToF MS/MS. The resulting peaklists were again searched using MASCOT. A further 8 protein spots were identified (numbered 39–46; <a href="#pone-0001778-t002">Table 2</a>). Overall, 41 % of the excised protein spots were identified. The rather low identification ratio may be due to strain variation between the sludges from which the metagenomic data were obtained (OZ and US sludges) and those from which the proteins were isolated (our EBPR<sub>55</sub> sludge), since the chosen stringent protein identification strategy relies on exact peptide matches. Nonetheless, the availability of metagenomic sequences allows for rapid protein identification compared to previous investigations relying on cost- and time-intensive <em>de novo</em> peptide sequencing <a href="#pone.0001778-Wilmes1">[5]</a>.</p>
<a id="article1.body1.sec2.sec3.sec2.p3" name="article1.body1.sec2.sec3.sec2.p3"></a><p>A number of proteins were identified several times from different individual spots, e.g. F<sub>0</sub>F<sub>1</sub>-type ATP synthase, beta subunit (spots 1, 4, 6, 8, 11, 12 and 14). Highly abundant proteins may swamp the 2D-PAGE gels, leading to multiple detection. This artefact of 2D-PAGE has been noted in pure culture studies and may be due to several reasons, including strain variation, differential protein processing, posttranslational modifications, and protein degradation <a href="#pone.0001778-BakerAustin1">[28]</a>, <a href="#pone.0001778-Gygi1">[29]</a>. Activated sludge is especially rich in lipases and proteases <a href="#pone.0001778-Gessesse1">[30]</a> and, despite inclusion of protease inhibitors in the protein extraction buffers (see <a href="#s3">Materials and Methods</a>), some protease activity may have been retained in the crude protein extracts. In total, 33 unique proteins were detected after excluding redundant identifications.</p>
<a id="article1.body1.sec2.sec3.sec2.p4" name="article1.body1.sec2.sec3.sec2.p4"></a><p>Protein identifications were deduced by matching the MS peaklist data to the metagenomic gene sequences using the MASCOT algorithm. The subsequent gene and protein functions were derived from the IMG/M annotation of the metagenome data <a href="#pone.0001778-Markowitz1">[31]</a>. Consequently, the putative protein functions are discussed here in relation to current EBPR metabolic models, and gene synteny information is included when relevant. Importantly, the present study highlights details of metabolic pathways active during EBPR, it reveals pathways previously not considered in metabolic models, and it provides direction for future investigations defining enzyme activities and regulatory events.</p>



<h4>Suggested protein functions in relation to EBPR</h4>
<h5>Fatty acid oxidation and PHA synthesis.</h5><a id="article1.body1.sec2.sec4.sec1.p1" name="article1.body1.sec2.sec4.sec1.p1"></a><p>The detected proteins along with possible functions are listed in <a href="#pone-0001778-t002">Table 2</a>. Several proteins putatively involved in PHA synthesis and fatty acid oxidation were highly expressed. These included acetyl-CoA acetyltransferase (PhaA; spot 41), which is involved in the formation of acetoacetyl-CoA, the first step of PHA synthesis, and poly (3-hydroxyalkanoate) synthetase (PhaC; spots 2, 5, 9, 27 and 28). The activity of PhaC links <em>(R)</em>-3-hydroxyacyl-CoA to an existing PHA molecule, the last step in the formation of PHA (<a href="#pone-0001778-g004">Fig. 4a</a>). These transformations would be an integral part of anaerobic EBPR metabolism.</p>
<div class="figure" id="pone-0001778-g004"><div class="img"><a name="pone-0001778-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001778" data-uri="info:doi/10.1371/journal.pone.0001778.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001778.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001778.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001778.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001778.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Proposed metabolic model for the (A) anaerobic and (B) aerobic phase of EBPR inferred from the proteomic data.</span></strong></p><a id="article1.body1.sec2.sec4.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec4.sec1.fig1.caption1.p1"></a><p>Identified proteins catalysing individual reactions are highlighted in green [best MASCOT metagenomic sequence match located on a scaffold source binned as <em>“A. phosphatis”</em>, i.e. strong association with the <em>“A. phosphatis”</em> composite genome], orange [best MASCOT sequence match located on a scaffold source binned as “other <em>Accumulibacter”</em> for which a strong BLAST hit (&gt;90 % identity) was obtained with a sequence binned as <em>“A. phosphatis”</em>, i.e. medium strong association with the <em>“A. phosphatis”</em> composite genome], and red [best MASCOT sequence match located on a scaffold source binned as “other <em>Accumulibacter”</em> for which a weak BLAST hit (&lt;90 % identity) was obtained with a sequence binned as <em>“A. phosphatis”</em>, i.e. weak association with the <em>“A. phosphatis”</em> composite genome]. Not all intermediate metabolites are shown. Abbreviations: ACC, acetyl-CoA carboxylase; ACD, acyl-CoA dehydrogenase; ACS, acyl-CoA synthetase; AGP, ADP-glucose pyrophosphorylase; ATPsyn, F<sub>0</sub>F<sub>1</sub>-type ATP synthase; CSY, citrate synthase; Fba, fructose bisphosphate aldolase; HpI, hydroxypyruvate isomerase; Ily, isocitrate lyase; Mdh, malate dehydrogenase; MalS, malate synthase; NADH, uncharacterised NAD(FAD)-dependent dehydrogenase; PhaA, acetyl-CoA acetyltransferase; PhaC, poly(3-hydroxyalkanoate) synthetase; PhaJ, enoyl-CoA hydratase; PpS, phosphoenolpyruvate synthase; Pst, ABC-type phosphate transport system; SCFA, short chain fatty acids; SuccDH, succinate dehydrogenase; TpI, triosephosphate isomerase.</p>
<span>doi:10.1371/journal.pone.0001778.g004</span></div><a id="article1.body1.sec2.sec4.sec1.p2" name="article1.body1.sec2.sec4.sec1.p2"></a><p>Enoyl-CoA hydratase (PhaJ) was also detected (spot 40). The activity of this enzyme may directly link PHA formation to fatty acid β oxidation (<a href="#pone-0001778-g004">Fig. 4a</a>) <a href="#pone.0001778-Aldor1">[32]</a>. Interestingly, all of the EBPR metagenomic contigs that contain <em>phaJ</em> also contain adjacent genes involved in PHA synthesis. From pure culture studies, the expression of <em>phaJ</em> is co-regulated with other PHA synthesis genes and in certain bacterial species <em>phaJ</em> is part of a PHA synthesis operon <a href="#pone.0001778-Rehm1">[33]</a>. However, based on nucleotide spacing, the genes described here are unlikely to represent an operon. Overall, three out of the five key enzymes involved in PHA formation <a href="#pone.0001778-Aldor1">[32]</a> were identified from the reactor sludge and are present on contigs and scaffolds linked to <em>“A. phosphatis”</em>. Consequently, our evidence of protein expression implicates PHA synthesis by <em>“A. phosphatis”</em> in the context of EBPR (<a href="#pone-0001778-g004">Fig. 4a</a>).</p>
<a id="article1.body1.sec2.sec4.sec1.p3" name="article1.body1.sec2.sec4.sec1.p3"></a><p>In addition to PhaJ, other proteins involved in fatty acid β oxidation were identified. Protein spots 22 and 44 were identified as acyl-CoA dehydrogenase and acyl-coenzyme A synthetase/AMP-(fatty) acid ligase respectively. These proteins are involved in the activation and the initial step of fatty acid β oxidation (<a href="#pone-0001778-g004">Fig. 4a</a>). Furthermore, protein spot 39 was identified as a biotin carboxylase assigned to <em>“A. phosphatis”</em> which forms part of the acetyl-CoA carboxylase complex. That complex catalyses the first committed and rate-limiting step of fatty acid synthesis. Furthermore, one of the <em>“A. phosphatis”</em> contigs that contains biotin carboxylase also contains a putative acyl dehydratase, an enzyme characteristic of aerobic fatty acid biosynthesis.</p>
<a id="article1.body1.sec2.sec4.sec1.p4" name="article1.body1.sec2.sec4.sec1.p4"></a><p>We hypothesise that fatty acid metabolism plays an important role in EBPR biochemistry beyond that of lipid metabolism for cell membranes (<a href="#pone-0001778-g004">Fig. 4</a>). One possibility is that it acts as an additional storage molecule in PAOs and fulfils a similar role compared to glycogen as suggested in previous metabolic models, e.g. <a href="#pone.0001778-Seviour1">[14]</a>. Fatty acid, if accumulated in the aerobic phase, could provide more reducing equivalents during the anaerobic phase, in comparison to glycogen. Neutral lipid storage molecules are widespread in eukaryotes, but have only been reported in relatively few bacteria including some actinomycetes and <em>Acinetobacter</em> species <a href="#pone.0001778-Waltermann1">[34]</a>. We postulate that anaerobic oxidation of stored fatty acids is important for contributing reducing equivalents. Furthermore, the utilisation of exogenous fatty acids may be relevant in full-scale EBPR systems that are not fed acetic acid based synthetic feed. In laboratory-scale reactors, propionate has been found to be a more favourable substrate for EBPR compared to acetate <a href="#pone.0001778-Oehmen1">[22]</a> and the provision of longer volatile fatty acids would require less energy expenditure for the accumulation of intracellular macromolecules (PHA and fatty acids). Consequently, fatty acid accumulation and degradation may have direct ramifications on the engineering of EBPR wastewater treatment systems. Finally, it should be noted that in some species PhaJ catalyses the formation of the PHB precursor (<em>R</em>)-3 hydroxyacyl-CoA <em>via</em> the intermediate crotonyl-CoA, from acetyl-CoA precursors <a href="#pone.0001778-Sato1">[35]</a>. Thus, another possible role for PhaJ detected in our study is PHB synthesis <em>via</em> this pathway. Overall, the exact role of fatty acid metabolism in EBPR warrants further investigation.</p>

<h5>Glycogen degradation and synthesis.</h5><a id="article1.body1.sec2.sec4.sec2.p1" name="article1.body1.sec2.sec4.sec2.p1"></a><p>One contentious issue regarding details of the EBPR metabolic model has been the nature of the glycolytic pathway used by PAOs in the anaerobic degradation of glycogen. Protein spot 7 was identified as triosephosphate isomerase. This supports previous suggestions that the Embden-Meyerhof is the key glycolytic pathway in EBPR <a href="#pone.0001778-GarcaMartn1">[4]</a>. However, this disagrees with metabolic evidence suggesting the Entner-Doudoroff pathway is used <a href="#pone.0001778-Hesselmann2">[36]</a>. Further evidence of key enzyme activity is required to determine the glycolytic pathway used by PAOs. In a recent unpublished investigation of another EBPR sludge we detected high expression of fructose bisphosphate aldolase, further supporting the Embden-Meyerhof pathway as the suggested glycolytic pathway. In regard to glycogen synthesis, protein spots 33 and 38 revealed phosphoenolpyruvate synthase and protein spot 37 was identified as ADP-glucose pyrophosphorylase. Thus, key enzymes involved in carbohydrate degradation and storage were identified (<a href="#pone-0001778-g004">Fig. 4</a>).</p>

<h5>The glyoxylate/TCA cycles.</h5><a id="article1.body1.sec2.sec4.sec3.p1" name="article1.body1.sec2.sec4.sec3.p1"></a><p>Our proteomic analysis identified hydroxypyruvate isomerase (spot 18) which exclusively catalyses the reversible isomerisation between hydroxypyruvate and tartonate semialdehyde (<a href="#pone-0001778-g004">Fig. 4b</a>). In bacteria the expression of this gene is induced by the presence of glyoxylate. Three other enzymes linked to the glyoxylate/tricarboxylic acid (TCA) cycle were highly expressed including malate synthase (MalS; spot 20) which catalyses the condensation of acetyl-CoA and glyoxylate with the formation of malate and CoA (<a href="#pone-0001778-g004">Fig. 4b</a>). Spot 21 revealed the Fe-S protein subunit of succinate dehydrogenase/fumarate reductase (SuccDH). On the corresponding <em>“A. phosphatis”</em> scaffold genes coding for other subunits of the enzyme preceded the gene encoding the Fe-S subunit. SuccDH catalyses the reversible conversion of succinate to fumarate, as part of the TCA or the glyoxylate cycles (<a href="#pone-0001778-g004">Fig. 4b</a>). In further unpublished work examining differential expression between the anaerobic and aerobic phases of EBPR, we detected increased expression of other glyoxylate cycle proteins, malate dehydrogenase and isocitrate lyase, in the aerobic phase of EBPR (<a href="#pone-0001778-g004">Fig. 4b</a>). Other enzymes of the TCA/glyoxylate cycle that are located on the <em>“A. phosphatis”</em> composite genome mediate the cyclic reactions, e.g. citrate synthase (protein spot 43). Taking these results into consideration, it is clear that the glyoxylate cycle is active in EBPR, and likely this is more so in the aerobic phase (<a href="#pone-0001778-g004">Fig. 4b</a>).</p>
<a id="article1.body1.sec2.sec4.sec3.p2" name="article1.body1.sec2.sec4.sec3.p2"></a><p>As alluded to earlier, the source of reducing power during the anaerobic PHA synthesis has long been a point of interest. This was originally proposed through oxidation of acetate via the TCA cycle <a href="#pone.0001778-Comeau1">[37]</a>–<a href="#pone.0001778-Pereira1">[39]</a> or through degradation of intracellular glycogen <a href="#pone.0001778-Mino1">[40]</a>. The glyoxylate cycle has also recently been proposed for producing reducing equivalents in the anaerobic phase <a href="#pone.0001778-Louie1">[41]</a>, <a href="#pone.0001778-Yagci1">[42]</a>, possibly in conjunction with a novel cytochrome <a href="#pone.0001778-GarcaMartn1">[4]</a>. Those previous suggestions were based on model bacterial pathways and the presence of genes on the metagenome. Our study provides evidence of high expression of proteins involved in the glyoxylate cycle, for the first time directly implicating its importance in EBPR metabolism. The glyoxylate shunt allows the production of reducing equivalents (for subsequent energy conservation) without the release of carbon dioxide in the conversion of isocitrate to succinate, in contrast to the reactions of the TCA cycle. Consequently, we argue that the operation of the glyoxylate shunt is more critical in the aerobic phase, where balancing carbon substrate utilisation for energy conservation and storage molecule (glycogen) synthesis is essential (<a href="#pone-0001778-g004">Fig. 4b</a>). This idea is supported by our recent detection of differential expression of other glyoxylate enzymes (unpublished data).</p>

<h5>Phosphate transport and bioenergetics.</h5><a id="article1.body1.sec2.sec4.sec4.p1" name="article1.body1.sec2.sec4.sec4.p1"></a><p>Two separate proteins were identified from protein spot 36. A hypothetical protein, and a periplasmic component of an ATP binding cassette (ABC)-type phosphate specific transport system (Pst). The gene producing the best MASCOT match was located on a contig assigned to “other <em>Accumulibacter”</em> that contained four other genes encoding the Pst system, namely three permease components and the ATPase component. In other bacteria the Pst transport system couples the hydrolysis of ATP to the translocation of phosphate across the inner membrane <a href="#pone.0001778-vanVeen1">[43]</a>. These are typically high affinity transporters that are only expressed during sub micromolar concentrations of extracellular phosphate and the specificity of Pst systems is for protonated phosphate species but not metal phosphates. Consequently, PAOs may use active transport for uptake and/or release of phosphate. It is proposed that the activity of this high affinity phosphate transport system may be especially pronounced in the later stages of the aerobic phase when P concentrations are limited (<a href="#pone-0001778-g004">Fig. 4b</a>), and that a low affinity inorganic phosphate transporter (Pit) is the more active system during the other stages <a href="#pone.0001778-GarcaMartn1">[4]</a>. Another possibility is that the Pst system detected here is active throughout the different stages of EBPR (<a href="#pone-0001778-g004">Fig. 4</a>) and the anaerobic phosphate efflux could mediate the production of ATP. Such activity would have direct ramifications for EBPR bioenergetics and biochemical characterisation of the PAO phosphate transport systems is required to reveal the exact physiological details.</p>
<a id="article1.body1.sec2.sec4.sec4.p2" name="article1.body1.sec2.sec4.sec4.p2"></a><p>The protein identified from the largest number of individual spots (7) was the β subunit of F<sub>0</sub>F<sub>1</sub>-type ATP synthase. The β subunit of F<sub>0</sub>F<sub>1</sub>-type ATP synthase is a non-membrane spanning protein and expression from <em>“A. phosphatis”</em> and “other <em>Accumulibacter</em>” were detected. All the genes coding for the F<sub>1</sub> unit of ATP synthase were contained on a single metagenome scaffold, i.e. γ, α and δ subunits, and subunit β. Protein spot 26 was identified as an uncharacterised NAD(FAD)-dependent dehydrogenase possibly forming part of the electron transport chain within <em>“A. phosphatis”</em>. Consequently, both detected proteins may be involved in ATP regeneration within <em>“A. phosphatis”</em> and, as such, fulfil an essential role in the EBPR metabolic model (<a href="#pone-0001778-g004">Fig. 4</a>).</p>

<h5>Stress response and other proteins.</h5><a id="article1.body1.sec2.sec4.sec5.p1" name="article1.body1.sec2.sec4.sec5.p1"></a><p>A number of stress response proteins were highly expressed in the EBPR sludge. Protein spots 23 and 32 were identified as separate peroxiredoxins, which protect cells against reactive oxygen species. Protein spot 30 revealed another oxidative stress induced protein, thiol-disulfide isomerase, thioredoxin. Thioredoxins are responsible for maintaining disulfide bonds within cytoplasmic proteins in a reduced state and, hence, are required for proper folding of proteins. Other proteins directly involved in protein folding were also detected, including molecular chaperone proteins. Proteins associated with cellular stress response mechanisms actually represented the largest fraction of proteins identified in this study. It is reasonable to expect that cells within the EBPR biomass would experience stress such as large changes in redox potential in short periods of time. Consequently, cells able to maintain protein function in the alternating anaerobic:aerobic sludge cycling are favoured.</p>
<a id="article1.body1.sec2.sec4.sec5.p2" name="article1.body1.sec2.sec4.sec5.p2"></a><p>Numerous housekeeping proteins not specific to EBPR were detected (<a href="#pone-0001778-t002">Table 2</a>). Only few 2D-PAGE gel spots were identified as proteins of unknown functions (protein spots 34, 35 and 46). This is in contrast to the metaproteomic investigation of the acid mine drainage biofilm in which the largest fraction of detected proteins were of unknown function <a href="#pone.0001778-Ram1">[13]</a>.</p>

<h5>Strain resolved community proteomics.</h5><a id="article1.body1.sec2.sec4.sec6.p1" name="article1.body1.sec2.sec4.sec6.p1"></a><p>It is evident that all the proteins' best MASCOT hits (which are a function of unique and shared peptide masses) were against sequences binned as <em>“Accumulibacter”</em> species but not specifically binned as <em>“A. phosphatis”</em> (<a href="#pone-0001778-t002">Table 2</a>). The logical explanation is that we are detecting proteins highly expressed by the dominating PAOs in our reactor and that these are distinct but closely related to the dominant <em>“A. phosphatis”</em> strain assembled by García-Martín <em>et al.</em> <a href="#pone.0001778-GarcaMartn1">[4]</a>. Overall, the vast majority of best MASCOT hits were obtained against the Phrap assembly of the US sludge (<a href="#pone-0001778-t002">Table 2</a>), followed by the Jazz assembly of the US sludge and, lastly, by the Phrap assembly of the OZ sludge. Consequently, the metagenomic sequences obtained from the US sludge better reflect the genetic make-up of the EBPR sludges described in this study.</p>
<a id="article1.body1.sec2.sec4.sec6.p2" name="article1.body1.sec2.sec4.sec6.p2"></a><p>Numerous other <em>“A. phosphatis”</em> strains (4 % divergent at the nucleotide level from the dominant strain) and <em>“Accumulibacter”</em> species (15 % divergent at the nucleotide level from the dominant <em>“A. phosphatis”</em> strain) were present in the US and OZ sludges (Kunin, V. and Hugenholtz, P.; unpublished). Furthermore, previous work has revealed extensive diversity among <em>“Accumulibacter”</em>-related organisms <a href="#pone.0001778-He1">[44]</a>. While our quantitative FISH analysis revealed dominance of <em>“Accumulibacter”</em>-type organisms, strain resolution was not obtained. Thus, it seems likely an <em>“Accumulibacter”</em> strain different from the assembled <em>“A. phosphatis”</em> dominated our reactor, and would have contributed to limiting protein identifications based on the MS hits against metagenome sequence. Consequently, the present study highlights the requirement for strain-resolved community proteomics in environmental microbiology research <a href="#pone.0001778-Lo1">[45]</a>. The use of advanced instrumentation, e.g. liquid chromatography electrospray two-dimensional linear ion trap mass spectrometry in conjunction with the Orbitrap detector <a href="#pone.0001778-Hu1">[46]</a>, will allow the differentiation of highly expressed proteins at the <em>“A. phosphatis”</em> strain and/or <em>“Accumulibacter”</em> species level.</p>



<h4>The metaproteomic approach</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The present study highlights the opportunity and power of applying proteomics to mixed culture systems for which metagenomic sequences are available. This is particularly applicable in systems that are well characterised with respect to biochemical transformations and have rather limited diversity. The classical 2D-PAGE proteomic approach was used in this study. In comparison, multidimensional liquid chromatography coupled to MS, such as that recently used to detect proteomes from a mixed culture biofilm in an acid drainage solution <a href="#pone.0001778-Ram1">[13]</a>, has potential for much higher throughput for protein identifications. However, the 2D-PAGE approach retains an advantage with regard to protein quantification since protein spot intensities and sizes on 2D-PAGE gels are a better reflection of protein abundance compared to abundances inferred from peptide MS data alone. Taking advantage of this characteristic of 2D-PAGE gels enabled us to focus on highly expressed proteins, and to monitor protein expression that increased with increasing P removal performance.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Numerous proteins that could be directly linked to the investigated metabolic mixed-culture process of EBPR were identified. The discovery of these functional enzymes is evidence for the described biochemical processes of EBPR metabolism. In addition, novel suggestions are made relating to PAO metabolism. These include the involvement of fatty acid metabolism and the glyoxylate shunt. Thus, we highlight potentially important functions and metabolic pathway details that have been overlooked in other EBPR studies. Additionally, this work provides important direction for future studies. Particular proteins detected here could be the focus of investigations for biochemical characterisation attempts to verify function, to examine regulatory details of expression, and to measure specific enzyme activities in full-scale EBPR systems.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sequencing batch reactor operation and sampling</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>A laboratory-scale SBR with alternating anaerobic/aerobic phases was operated as described previously <a href="#pone.0001778-Wilmes1">[5]</a>, <a href="#pone.0001778-Wilmes3">[25]</a>. Briefly, the reactor had a working volume of 2 l, and was operated on a 6 hr cycle consisting of a 120 min anaerobic phase, a 210 min aerobic phase and a 30 min settling/decant phase. Initially, the reactor was operated for approximately 1.5 months until stable EBPR performance, removing around 25 mg/l, was obtained. At this time-point (day of operation 0) an intensive sampling routine was started. This entailed measuring PO<sub>4</sub>-P in the influent (feed) and in the reactor at the end of the aerobic phase (t = 330 min) each day for a total of 102 days. On the tenth day of reactor operation, a cycle study was carried out. This involved taking samples for PO<sub>4</sub>-P, acetate, polyhydroxyalkanoate (PHA), mixed liquor suspended solids (MLSS) and total P analyses. At this time-point, the activated sludge within the reactor was completely removing 28.5 mg/l of PO<sub>4</sub>-P (no phosphate in the reactor at the end of the aerobic phase) and this constituted the EBPR<sub>28 </sub>sludge described in this study. On the 17<sup>th</sup> day of operation, the PO<sub>4</sub>-P concentration in the feed was increased to around 40 mg/l. A complete cycle study was carried out on day 49 and this sludge was completely removing 42.4 mg/l PO<sub>4</sub>-P. This constituted the EBPR<sub>42 </sub>sludge described in this study. On day 57 the PO<sub>4</sub>-P concentration in the feed was increased to around 55 mg/l and a cycle study was carried out on day 71. The sludge was completely removing 55.2 mg/l PO<sub>4</sub>-P and, consequently, this constituted the EBPR<sub>55 </sub>sludge described in this study. On the 78<sup>th</sup> day of reactor operation, the PO<sub>4</sub>-P concentration in the feed was increased to around 70 mg/l. Following this increase, the reactor's P removal performance started to fluctuate with 57.6 mg/l PO<sub>4</sub>-P present at the end of the aerobic phase on day 82. However, the reactor did regain its P removal performance on day 86. From day 92 onwards, the reactor gradually lost its EBPR performance with complete loss of P removal performance after 7 days, i.e. 1 sludge age. On day 99, another cycle study was carried out. 59.6 mg/l of extracellular PO<sub>4</sub>-P remained in the reactor at the end of the aerobic phase with 70.7 mg/l of PO<sub>4</sub>-P in the feed. This sludge constituted the nEBPR<sub>70 </sub>sludge described in this study.</p>


<h4>Chemical analyses</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Phosphate P, acetate, mixed liquor suspended solids (MLSS) and total P were analysed as described earlier <a href="#pone.0001778-Wilmes1">[5]</a>. Polyhydroxyalkanoates (PHAs) were quantified following acid methanolysis by gas chromatography <a href="#pone.0001778-Bond1">[16]</a> with modifications described in the Supporting <a href="#pone.0001778.s001">Material S1</a>.</p>


<h4>16S rRNA FISH with phylogenetic probes</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Sampling, cell fixation, hybridisation and image processing were carried out as reported earlier <a href="#pone.0001778-Wilmes1">[5]</a>. Samples were taken of each of the analysed sludges (EBPR<sub>28</sub>, EBPR<sub>42</sub>, EBPR<sub>55</sub> and nEBPR<sub>70</sub>) at the end of the aerobic phase. A range of broad and specific probes were employed and these were obtained from MWG Biotech (Ebersberg, Germany). Oligonucleotide probes ALF1b <a href="#pone.0001778-Manz1">[47]</a>, BET42a <a href="#pone.0001778-Manz1">[47]</a>, GAM42a <a href="#pone.0001778-Manz1">[47]</a>, HGC69a <a href="#pone.0001778-Roller1">[48]</a>, CF319a <a href="#pone.0001778-Manz1">[47]</a>, GAOQ431 <a href="#pone.0001778-Crocetti2">[49]</a> and PAO651 <a href="#pone.0001778-Crocetti1">[18]</a> were labelled with the sulfoindocyanide dye Cy3 and EUBMIX <a href="#pone.0001778-Daims1">[50]</a> probes were labelled with fluorescein isothiocyanate (FITC). uBET and uGAM were unlabeled <a href="#pone.0001778-Falkentoft1">[51]</a>. Single factor analysis of variance (ANOVA) was used to evaluate quantitative differences of FISH detected cells between the sludges.</p>


<h4>Protein extraction, purification and resuspension</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>100 ml samples were taken for each analysed sludge (EBPR<sub>28</sub>, EBPR<sub>42</sub>, EBPR<sub>55</sub> and nEBPR<sub>70</sub>) at the end of the aerobic phase (t = 330 min). Protein extractions and purifications were carried out as described earlier <a href="#pone.0001778-Wilmes1">[5]</a>, <a href="#pone.0001778-Wilmes3">[25]</a>. Following precipitation the protein pellets were resuspended in a resuspension buffer. The resuspension buffer consisted of 7 M urea, 2 M thiourea, 4 % (w/v) CHAPS, 40 mM Tris/1 mM EDTA, 50 mM dithiothreotol (DTT), 25 mM Pefabloc SC, 2 mM Pefabloc Protector (Roche, Welwyn Garden City, UK) and 1 % (v/v) ZOOM Carrier Ampholytes pH 3–10 (Invitrogen, Paisley, UK).</p>


<h4>Liquid isoelectric focussing</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Liquid isoelectric focusing was carried out in a ZOOM IEF Fractionator (Invitrogen) according to the manufacturer's instructions. 2.2 mg of total protein were fractionated in each run. The pH 5.4 and 6.2 ZOOM Disks were excluded from the assembly to obtain a single pH 4.6–7.0 fraction. The liquid fractionation was carried out according to the following conditions: 100 V for 20 min, 200 V for 80 min and 600 V for 140 min. After fractionation, the different fractions were removed from the fractionator. Immobilised pH gradient (IPG) buffer pH 4–7 (Amersham Biosciences–GE Healthcare, Chalfont St. Gilles, UK) was added to the pH 4.6–7.0 fraction to obtain a final concentration of around 2 % (v/v) carrier ampholytes. 2-D SDS-PAGE standards (Bio-Rad, Bath, UK) were mixed into the pH 4.6–7.0 fraction according to the manufacturer's instructions.</p>


<h4>2D-PAGE</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>The prepared pH 4.6–7.0 fractions were used to rehydrate 24 cm pH 4–7 IPG strips (Immobiline DryStrips, Amersham Biosciences–GE Healthcare) in an Immobiline DryStrip Reswelling Tray (Amersham Biosciences–GE Healthcare) for 16 hrs. For first-dimension separation, the strips were placed in an IPGphor ceramic manifold, covered with Plusone DryStrip cover fluid and focused for 100000 volt-hours in an Ettan IPGphor II isoelectric focusing system (Amersham Biosciences–GE Healthcare). The strips were then equilibrated <a href="#pone.0001778-Hesketh1">[52]</a> and applied to 14 % (v/v) Duracryl (Genomic Solutions, Huntingdon, UK) gels. Precision Plus Protein Standard Plugs (Bio-Rad) were layered onto the gels according to the manufacturer's instructions. Second dimension separation was carried out at 500 V in an Ettan DALT<em>six</em> electrophoresis system (Amersham Biosciences-GE Healthcare). The gels were stained overnight with SyproRuby (Bio-Rad) and scanned using a Molecular Imager FX (Bio-Rad). Triplicate 2D-PAGE separations were generated for each sludge sample.</p>


<h4><em>In silico</em> analysis of 2D-PAGE gels</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>The acquired gel images were processed and analysed using PDQuest, version 7.3.0 (Bio-Rad). The EBPR<sub>28</sub>, EBPR<sub>42</sub> and EBPR<sub>55</sub> replicate gels were placed within the same matchset, termed the EBPR matchset. Since the nEBPR<sub>70</sub> replicate gels were too dissimilar compared to the EBPR gels, they were placed in a separate matchset, i.e. the nEBPR matchset. Automated and manual spot detection and matching was performed, as well as spot densities determined for quantification. From the EBPR matchset a consensus gel (termed the “Master” gel) was produced.</p>


<h4>Spot excision</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>Following <em>in silico</em> analysis of the 2D-PAGE gels generated using PDQuest, spots were chosen for identification. The spots in the EBPR matchset were ranked according to their spot quantity as determined by PDQuest analysis. Protein spots that were highly expressed on the EBPR<sub>55</sub> replicate group were chosen for excision. The chosen spots were also present on the EBPR<sub>28</sub> and EBPR<sub>42</sub> gels. These were excised from an EBPR<sub>55</sub> 2D-PAGE gel using an Investigator Pro Pic (Genomic Solutions) spot-picking robot and placed in a 96 well plate prior to further processing.</p>


<h4>Matrix assisted laser desorption ionisation time-of-flight (MALDI-ToF) mass spectrometry (MS), Quadrupole-ToF (Q-ToF) MS/MS and protein identification</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>The excised spots were processed and digested as described earlier <a href="#pone.0001778-Wilmes1">[5]</a>. Details on mass spectrometry analyses and the protein identification strategy are provided in the Supporting <a href="#pone.0001778.s001">Material file S1</a>. Briefly, the digested protein samples were analysed on a Bruker UltraFlex MALDI-ToF/ToF mass spectrometer (Bruker Daltonics Ltd., Coventry, UK). Samples were further analysed using a Q-ToF-2 mass spectrometer (Micromass, Elstree, UK). The resulting peptide mass fingerprints were searched against the three metagenomic databases (OZ sludge, Phrap assembly; US sludge, Phrap assembly; US sludge, Jazz assembly) <a href="#pone.0001778-GarcaMartn1">[4]</a> using the MASCOT search tool (<a href="http://www.matrixscience.com">http://www.matrixscience.com</a>). Detailed information on the metagenomic sequences of the best MASCOT hits were retrieved using the integrated microbial genomes with microbiome samples (IMG/M) system (experimental version; 1 September 2006, <a href="http://www.jgi.doe.gov/">http://www.jgi.doe.gov/</a>) <a href="#pone.0001778-Markowitz1">[31]</a>. For contigs and scaffolds of interest, Genbank files were generated using IMG/M and analysed using Artemis release 8 <a href="#pone.0001778-Rutherford1">[53]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001778"></div><a name="pone.0001778.s001" id="pone.0001778.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001778.s001">Material S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supporting material and methods</p>
<p class="siDoi">doi:10.1371/journal.pone.0001778.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0001778.s002" id="pone.0001778.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001778.s002">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Results of the quantitative FISH analysis (standard deviations in brackets).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001778.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001778.s003" id="pone.0001778.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001778.s003">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Summary of the 2D-PAGE analysis of the EBPR matchset (standard deviations in brackets). Indicating numbers of spots detected on individual gels and those matched across the replicate gel sets (n = 3).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001778.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001778.s004" id="pone.0001778.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001778.s004">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Protein identification results obtained using MALDI-ToF MS (spot numbers 1–39), Q-ToF MS/MS (spot numbers 40–46) and MASCOT including additional information (spot numbers refer to those in <a href="#pone-0001778-g003">Fig. 3</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001778.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.88 MB DOC)</p>
<a name="pone.0001778.s005" id="pone.0001778.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001778.s005">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001778.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(4.07 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Jill Banfield (University of California at Berkeley, Berkeley, California, USA) and Joanne Peters (Anglian Water, Cambridge, UK) are acknowledged for support and encouragement. Phil Hugenholtz, Hector García Martín and Victor Kunin (Joint Genome Institute, Walnut Creek, California, USA) are thanked for provision of unpublished data. MALDI-ToF MS and Q-ToF MS/MS analyses were carried out by Mike Naldrett at the IFR and JIC Joint Proteomics Facility, Norwich, UK. Liz Rix (School of Environmental Sciences, University of East Anglia, Norwich, UK) and Fred Mellon (Institute of Food Research, Norwich, UK) are thanked for their help with acetate and PHA analyses, respectively.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PB PW. Performed the experiments: PW. Analyzed the data: PB PW. Wrote the paper: PB PW MW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001778-Hugenholtz1" id="pone.0001778-Hugenholtz1"></a>Hugenholtz P, Goebel BM, Pace NR (1998) Impact of culture-independent studies on the emerging phylogenetic view of bacterial diversity. J Bacteriol  180: 6793–6793.  <ul class="find" data-citedArticleID="1072706"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Impact+of+culture-independent+studies+on+the+emerging+phylogenetic+view+of+bacterial+diversity.&amp;auth=&amp;atitle=Impact+of+culture-independent+studies+on+the+emerging+phylogenetic+view+of+bacterial+diversity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+culture-independent+studies+on+the+emerging+phylogenetic+view+of+bacterial+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+culture-independent+studies+on+the+emerging+phylogenetic+view+of+bacterial+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001778-DeLong1" id="pone.0001778-DeLong1"></a>DeLong EF, Preston CM, Mincer T, Rich V, Hallam SJ, et al.  (2006) Community genomics among stratified microbial assemblages in the ocean's interior. Science  311: 496–503.  <ul class="find" data-citedArticleID="1072684" data-doi="10.1126/science.1120250"><li><a href="http://dx.doi.org/10.1126/science.1120250" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Community+genomics+among+stratified+microbial+assemblages+in+the+ocean%27s+interior." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Community+genomics+among+stratified+microbial+assemblages+in+the+ocean%27s+interior.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001778-Venter1" id="pone.0001778-Venter1"></a>Venter JC, Remington K, Heidelberg JF, Halpern AL, Rusch D, et al.  (2004) Environmental genome shotgun sequencing of the Sargasso Sea. Science  304: 66–74.  <ul class="find" data-citedArticleID="1072750" data-doi="10.1126/science.1093857"><li><a href="http://dx.doi.org/10.1126/science.1093857" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001778-GarcaMartn1" id="pone.0001778-GarcaMartn1"></a>García-Martín H, Ivanovan N, Kunin V, Warnecke F, Barry KW, et al.  (2006) Metagenomic analysis of two enhanced biological phosphorus removal (EBPR) sludge communities. Nat Biotechnol  24: 1263–1269.  <ul class="find" data-citedArticleID="1072690" data-doi="10.1038/nbt1247"><li><a href="http://dx.doi.org/10.1038/nbt1247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metagenomic+analysis+of+two+enhanced+biological+phosphorus+removal+%28EBPR%29+sludge+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metagenomic+analysis+of+two+enhanced+biological+phosphorus+removal+%28EBPR%29+sludge+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001778-Wilmes1" id="pone.0001778-Wilmes1"></a>Wilmes P, Bond PL (2004) The application of two-dimensional polyacrylamide gel electrophoresis and downstream analyses to a mixed community of prokaryotic microorganisms. Environ Microbiol  6: 911–920.  <ul class="find" data-citedArticleID="1072756" data-doi="10.1111/j.1462-2920.2004.00687.x"><li><a href="http://dx.doi.org/10.1111/j.1462-2920.2004.00687.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+application+of+two-dimensional+polyacrylamide+gel+electrophoresis+and+downstream+analyses+to+a+mixed+community+of+prokaryotic+microorganisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+application+of+two-dimensional+polyacrylamide+gel+electrophoresis+and+downstream+analyses+to+a+mixed+community+of+prokaryotic+microorganisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001778-Wilmes2" id="pone.0001778-Wilmes2"></a>Wilmes P, Bond PL (2006) Metaproteomics: studying functional gene expression in microbial ecosystems. Trends Microbiol  14: 92–97.  <ul class="find" data-citedArticleID="1072758" data-doi="10.1016/j.tim.2005.12.006"><li><a href="http://dx.doi.org/10.1016/j.tim.2005.12.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metaproteomics%3A+studying+functional+gene+expression+in+microbial+ecosystems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metaproteomics%3A+studying+functional+gene+expression+in+microbial+ecosystems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001778-Kan1" id="pone.0001778-Kan1"></a>Kan J, Hanson TE, Ginter JM, Wang K, Chen F (2005) Metaproteomic analysis of Chesapeake Bay microbial communities. Saline Systems  1: 7–19.  <ul class="find" data-citedArticleID="1072708" data-doi="10.1002/9781118010518.ch68"><li><a href="http://dx.doi.org/10.1002/9781118010518.ch68" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metaproteomic+analysis+of+Chesapeake+Bay+microbial+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metaproteomic+analysis+of+Chesapeake+Bay+microbial+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001778-Klaassens1" id="pone.0001778-Klaassens1"></a>Klaassens ES, de Vos WM, Vaughan EE (2007) Metaproteomics approach to study the functionality of the microbiota in the human infant gastrointestinal tract. Appl Environ Microbiol  73: 1388–1392.  <ul class="find" data-citedArticleID="1072710" data-doi="10.1128/aem.01921-06"><li><a href="http://dx.doi.org/10.1128/aem.01921-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metaproteomics+approach+to+study+the+functionality+of+the+microbiota+in+the+human+infant+gastrointestinal+tract." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metaproteomics+approach+to+study+the+functionality+of+the+microbiota+in+the+human+infant+gastrointestinal+tract.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001778-Maron1" id="pone.0001778-Maron1"></a>Maron PA, Ranjard L, Mougel C, Lemanceau P (2007) Metaproteomics: a new approach for studying functional microbial ecology. Microb Ecol  53: 486–493.  <ul class="find" data-citedArticleID="1072724" data-doi="10.1007/s00248-006-9196-8"><li><a href="http://dx.doi.org/10.1007/s00248-006-9196-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metaproteomics%3A+a+new+approach+for+studying+functional+microbial+ecology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metaproteomics%3A+a+new+approach+for+studying+functional+microbial+ecology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001778-Benndorf1" id="pone.0001778-Benndorf1"></a>Benndorf D, Balcke GU, Harms H, von Bergen M (2007) Functional metaproteome analysis of protein extracts from contaminated soil and groundwater. ISME Journal  1: 224–234.  <ul class="find" data-citedArticleID="1072668" data-doi="10.1038/ismej.2007.39"><li><a href="http://dx.doi.org/10.1038/ismej.2007.39" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+metaproteome+analysis+of+protein+extracts+from+contaminated+soil+and+groundwater." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+metaproteome+analysis+of+protein+extracts+from+contaminated+soil+and+groundwater.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001778-Schulze1" id="pone.0001778-Schulze1"></a>Schulze WX, Gleixner G, Kaiser K, Guggenberger G, Mann M, et al.  (2005) A proteomic fingerprint of dissolved organic carbon and of soil particles. Oecologia  142: 335–343.  <ul class="find" data-citedArticleID="1072744" data-doi="10.1007/s00442-004-1698-9"><li><a href="http://dx.doi.org/10.1007/s00442-004-1698-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+proteomic+fingerprint+of+dissolved+organic+carbon+and+of+soil+particles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+proteomic+fingerprint+of+dissolved+organic+carbon+and+of+soil+particles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001778-Lacerda1" id="pone.0001778-Lacerda1"></a>Lacerda CMR, Choe LH, Reardon KF (2007) Metaproteomic analysis of a bacterial community response to cadmium exposure. J Proteome Res  6: 1145–1152.  <ul class="find" data-citedArticleID="1072712" data-doi="10.1021/pr060477v"><li><a href="http://dx.doi.org/10.1021/pr060477v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metaproteomic+analysis+of+a+bacterial+community+response+to+cadmium+exposure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metaproteomic+analysis+of+a+bacterial+community+response+to+cadmium+exposure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001778-Ram1" id="pone.0001778-Ram1"></a>Ram RJ, VerBerkmoes NC, Thelen MP, Tyson GW, Baker BJ, et al.  (2005) Community proteomics of a natural microbial biofilm. Science  308: 1915–1920.  <ul class="find" data-citedArticleID="1072734"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Community+proteomics+of+a+natural+microbial+biofilm.&amp;auth=&amp;atitle=Community+proteomics+of+a+natural+microbial+biofilm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Community+proteomics+of+a+natural+microbial+biofilm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Community+proteomics+of+a+natural+microbial+biofilm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001778-Seviour1" id="pone.0001778-Seviour1"></a>Seviour RJ, Mino T, Onuki M (2003) The microbiology of biological phosphorus removal in activated sludge systems. FEMS Microbiol Rev  27: 99–127.  <ul class="find" data-citedArticleID="1072746" data-doi="10.1016/s0168-6445(03)00021-4"><li><a href="http://dx.doi.org/10.1016/s0168-6445(03)00021-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microbiology+of+biological+phosphorus+removal+in+activated+sludge+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microbiology+of+biological+phosphorus+removal+in+activated+sludge+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001778-Blackall1" id="pone.0001778-Blackall1"></a>Blackall LL, Crocetti G, Saunders AM, Bond PL (2002) A review and update of the microbiology of enhanced biological phosphorus removal in wastewater treatment plants. Antonie Van Leeuwenhoek  81: 681–691.  <ul class="find" data-citedArticleID="1072670" data-doi="10.1023/a:1020538429009"><li><a href="http://dx.doi.org/10.1023/a:1020538429009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+review+and+update+of+the+microbiology+of+enhanced+biological+phosphorus+removal+in+wastewater+treatment+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+review+and+update+of+the+microbiology+of+enhanced+biological+phosphorus+removal+in+wastewater+treatment+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001778-Bond1" id="pone.0001778-Bond1"></a>Bond PL, Erhart R, Wagner M, Keller J, Blackall LL (1999) Identification of some of the major groups of bacteria in efficient and nonefficient biological phosphorus removal activated sludge systems. Appl Environ Microbiol  65: 4077–4084.  <ul class="find" data-citedArticleID="1072672"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+some+of+the+major+groups+of+bacteria+in+efficient+and+nonefficient+biological+phosphorus+removal+activated+sludge+systems.&amp;auth=&amp;atitle=Identification+of+some+of+the+major+groups+of+bacteria+in+efficient+and+nonefficient+biological+phosphorus+removal+activated+sludge+systems." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+some+of+the+major+groups+of+bacteria+in+efficient+and+nonefficient+biological+phosphorus+removal+activated+sludge+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+some+of+the+major+groups+of+bacteria+in+efficient+and+nonefficient+biological+phosphorus+removal+activated+sludge+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001778-Bond2" id="pone.0001778-Bond2"></a>Bond PL, Hugenholtz P, Keller J, Blackall LL (1995) Bacterial community structures of phosphate-removing and non- phosphate-removing activated sludges from sequencing batch reactors. Appl Environ Microbiol  61: 1910–1916.  <ul class="find" data-citedArticleID="1072674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterial+community+structures+of+phosphate-removing+and+non-+phosphate-removing+activated+sludges+from+sequencing+batch+reactors.&amp;auth=&amp;atitle=Bacterial+community+structures+of+phosphate-removing+and+non-+phosphate-removing+activated+sludges+from+sequencing+batch+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+community+structures+of+phosphate-removing+and+non-+phosphate-removing+activated+sludges+from+sequencing+batch+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+community+structures+of+phosphate-removing+and+non-+phosphate-removing+activated+sludges+from+sequencing+batch+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001778-Crocetti1" id="pone.0001778-Crocetti1"></a>Crocetti GR, Hugenholtz P, Bond PL, Schuler A, Keller J, et al.  (2000) Identification of polyphosphate-accumulating organisms and design of 16S rRNA-directed probes for their detection and quantitation. Appl Environ Microbiol  66: 1175–1182.  <ul class="find" data-citedArticleID="1072678" data-doi="10.1128/aem.66.3.1175-1182.2000"><li><a href="http://dx.doi.org/10.1128/aem.66.3.1175-1182.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+polyphosphate-accumulating+organisms+and+design+of+16S+rRNA-directed+probes+for+their+detection+and+quantitation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+polyphosphate-accumulating+organisms+and+design+of+16S+rRNA-directed+probes+for+their+detection+and+quantitation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001778-Hesselmann1" id="pone.0001778-Hesselmann1"></a>Hesselmann RPX, Werlen C, Hahn D, van der Meer JR, Zehnder AJB (1999) Enrichment, phylogenetic analysis and detection of a bacterium that performs enhanced biological phosphate removal in activated sludge. Syst Appl Microbiol  22: 454–465.  <ul class="find" data-citedArticleID="1072700" data-doi="10.1016/s0723-2020(99)80055-1"><li><a href="http://dx.doi.org/10.1016/s0723-2020(99)80055-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enrichment%2C+phylogenetic+analysis+and+detection+of+a+bacterium+that+performs+enhanced+biological+phosphate+removal+in+activated+sludge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enrichment%2C+phylogenetic+analysis+and+detection+of+a+bacterium+that+performs+enhanced+biological+phosphate+removal+in+activated+sludge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001778-Lee1" id="pone.0001778-Lee1"></a>Lee N, Nielsen PH, Aspegren H, Henze M, Schleifer KH, et al.  (2003) Long-term population dynamics and <em>in situ</em> physiology in activated sludge systems with enhanced biological phosphorus removal operated with and without nitrogen removal. Syst Appl Microbiol  26: 211–227.  <ul class="find" data-citedArticleID="1072714" data-doi="10.1078/072320203322346065"><li><a href="http://dx.doi.org/10.1078/072320203322346065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-term+population+dynamics+and+in+situ+physiology+in+activated+sludge+systems+with+enhanced+biological+phosphorus+removal+operated+with+and+without+nitrogen+removal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-term+population+dynamics+and+in+situ+physiology+in+activated+sludge+systems+with+enhanced+biological+phosphorus+removal+operated+with+and+without+nitrogen+removal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001778-Filipe1" id="pone.0001778-Filipe1"></a>Filipe CDM, Daigger GT, Grady CPL (2001) Stoichiometry and kinetics of acetate uptake under anaerobic conditions by an enriched culture of phosphorus-accumulating organisms at different pHs. Biotechnol Bioeng  76: 32–43.  <ul class="find" data-citedArticleID="1072688" data-doi="10.1002/bit.1023"><li><a href="http://dx.doi.org/10.1002/bit.1023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stoichiometry+and+kinetics+of+acetate+uptake+under+anaerobic+conditions+by+an+enriched+culture+of+phosphorus-accumulating+organisms+at+different+pHs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stoichiometry+and+kinetics+of+acetate+uptake+under+anaerobic+conditions+by+an+enriched+culture+of+phosphorus-accumulating+organisms+at+different+pHs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001778-Oehmen1" id="pone.0001778-Oehmen1"></a>Oehmen A, Yuan ZG, Blackall LL, Keller J (2005) Comparison of acetate and propionate uptake by polyphosphate accumulating organisms and glycogen accumulating organisms. Biotechnol Bioeng  91: 162–168.  <ul class="find" data-citedArticleID="1072730" data-doi="10.1002/bit.20500"><li><a href="http://dx.doi.org/10.1002/bit.20500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+acetate+and+propionate+uptake+by+polyphosphate+accumulating+organisms+and+glycogen+accumulating+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+acetate+and+propionate+uptake+by+polyphosphate+accumulating+organisms+and+glycogen+accumulating+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001778-Seviour2" id="pone.0001778-Seviour2"></a>Seviour RJ, Maszenan AM, Soddell JA, Tandoi V, Patel BKC, et al.  (2000) Microbiology of the ‘G-bacteria’ in activated sludge. Environ Microbiol  2: 581–593.  <ul class="find" data-citedArticleID="1072748" data-doi="10.1046/j.1462-2920.2000.00153.x"><li><a href="http://dx.doi.org/10.1046/j.1462-2920.2000.00153.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbiology+of+the+%E2%80%98G-bacteria%E2%80%99+in+activated+sludge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbiology+of+the+%E2%80%98G-bacteria%E2%80%99+in+activated+sludge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001778-McMahon1" id="pone.0001778-McMahon1"></a>McMahon KD, Dojka MA, Pace NR, Jenkins D, Keasling JD (2002) Polyphosphate kinase from activated sludge performing enhanced biological phosphorus removal. Appl Environ Microbiol  68: 4971–4978.  <ul class="find" data-citedArticleID="1072726" data-doi="10.1128/aem.68.10.4971-4978.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.10.4971-4978.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polyphosphate+kinase+from+activated+sludge+performing+enhanced+biological+phosphorus+removal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polyphosphate+kinase+from+activated+sludge+performing+enhanced+biological+phosphorus+removal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001778-Wilmes3" id="pone.0001778-Wilmes3"></a>Wilmes P, Bond PL (2006) Towards exposure of elusive metabolic mixed-culture processes: the application of metaproteomic analyses to activated sludge. Water Sci Technol  54: 217–226.  <ul class="find" data-citedArticleID="1072760" data-doi="10.2166/wst.2006.390"><li><a href="http://dx.doi.org/10.2166/wst.2006.390" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+exposure+of+elusive+metabolic+mixed-culture+processes%3A+the+application+of+metaproteomic+analyses+to+activated+sludge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+exposure+of+elusive+metabolic+mixed-culture+processes%3A+the+application+of+metaproteomic+analyses+to+activated+sludge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001778-Beer1" id="pone.0001778-Beer1"></a>Beer M, Kong YH, Seviour RJ (2004) Are some putative glycogen accumulating organisms (GAO) in anaerobic: aerobic activated sludge systems members of the alpha-Proteobacteria? Microbiology  150: 2267–2275.  <ul class="find" data-citedArticleID="1072666" data-doi="10.1099/mic.0.26825-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26825-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+some+putative+glycogen+accumulating+organisms+%28GAO%29+in+anaerobic%3A+aerobic+activated+sludge+systems+members+of+the+alpha-Proteobacteria%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+some+putative+glycogen+accumulating+organisms+%28GAO%29+in+anaerobic%3A+aerobic+activated+sludge+systems+members+of+the+alpha-Proteobacteria%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001778-Wong1" id="pone.0001778-Wong1"></a>Wong MT, Tan FM, Ng WJ, Liu WT (2004) Identification and occurrence of tetrad-forming Alphaproteobacteria in anaerobic-aerobic activated sludge processes. Microbiology  150: 3741–3748.  <ul class="find" data-citedArticleID="1072762" data-doi="10.1099/mic.0.27291-0"><li><a href="http://dx.doi.org/10.1099/mic.0.27291-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+occurrence+of+tetrad-forming+Alphaproteobacteria+in+anaerobic-aerobic+activated+sludge+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+occurrence+of+tetrad-forming+Alphaproteobacteria+in+anaerobic-aerobic+activated+sludge+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001778-BakerAustin1" id="pone.0001778-BakerAustin1"></a>Baker-Austin C, Dopson M, Wexler M, Sawers RG, Bond PL (2005) Molecular insight into extreme copper resistance in the extremophilic archaeon ‘<em>Ferroplasma acidarmanus</em>’ Fer1. Microbiology  151: 2637–2646.  <ul class="find" data-citedArticleID="1072664" data-doi="10.1099/mic.0.28076-0"><li><a href="http://dx.doi.org/10.1099/mic.0.28076-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+insight+into+extreme+copper+resistance+in+the+extremophilic+archaeon+%E2%80%98Ferroplasma+acidarmanus%E2%80%99+Fer1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+insight+into+extreme+copper+resistance+in+the+extremophilic+archaeon+%E2%80%98Ferroplasma+acidarmanus%E2%80%99+Fer1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001778-Gygi1" id="pone.0001778-Gygi1"></a>Gygi SP, Corthals GL, Zhang Y, Rochon Y, Aebersold R (2000) Evaluation of two-dimensional gel electrophoresis-based proteome analysis technology. Proc Natl Acad Sci U S A  97: 9390–9395.  <ul class="find" data-citedArticleID="1072694" data-doi="10.1073/pnas.160270797"><li><a href="http://dx.doi.org/10.1073/pnas.160270797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+two-dimensional+gel+electrophoresis-based+proteome+analysis+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+two-dimensional+gel+electrophoresis-based+proteome+analysis+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001778-Gessesse1" id="pone.0001778-Gessesse1"></a>Gessesse A, Dueholm T, Petersen SB, Nielsen PH (2003) Lipase and protease extraction from activated sludge. Water Res  37: 3652–3657.  <ul class="find" data-citedArticleID="1072692" data-doi="10.1016/s0043-1354(03)00241-0"><li><a href="http://dx.doi.org/10.1016/s0043-1354(03)00241-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lipase+and+protease+extraction+from+activated+sludge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lipase+and+protease+extraction+from+activated+sludge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001778-Markowitz1" id="pone.0001778-Markowitz1"></a>Markowitz VM, Korzeniewski F, Palaniappan K, Szeto E, Werner G, et al.  (2006) The integrated microbial genomes (IMG) system. Nucleic Acids Res  34: D344–D348.  <ul class="find" data-citedArticleID="1072722" data-doi="10.1093/nar/gkj024"><li><a href="http://dx.doi.org/10.1093/nar/gkj024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+integrated+microbial+genomes+%28IMG%29+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+integrated+microbial+genomes+%28IMG%29+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001778-Aldor1" id="pone.0001778-Aldor1"></a>Aldor IS, Keasling JD (2003) Process design for microbial plastic factories: metabolic engineering of polyhydroxyalkanoates. Curr Opin Biotechnol  14: 475–483.  <ul class="find" data-citedArticleID="1072662" data-doi="10.1016/j.copbio.2003.09.002"><li><a href="http://dx.doi.org/10.1016/j.copbio.2003.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Process+design+for+microbial+plastic+factories%3A+metabolic+engineering+of+polyhydroxyalkanoates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Process+design+for+microbial+plastic+factories%3A+metabolic+engineering+of+polyhydroxyalkanoates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001778-Rehm1" id="pone.0001778-Rehm1"></a>Rehm BHA (2006) Genetics and biochemistry of polyhydroxyalkanoate granule self-assembly: The key role of polyester synthases. Biotechnol Lett  28: 207–213.  <ul class="find" data-citedArticleID="1072736" data-doi="10.1007/s10529-005-5521-4"><li><a href="http://dx.doi.org/10.1007/s10529-005-5521-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+and+biochemistry+of+polyhydroxyalkanoate+granule+self-assembly%3A+The+key+role+of+polyester+synthases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+and+biochemistry+of+polyhydroxyalkanoate+granule+self-assembly%3A+The+key+role+of+polyester+synthases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001778-Waltermann1" id="pone.0001778-Waltermann1"></a>Waltermann M, Steinbuchel A (2005) Neutral lipid bodies in prokaryotes: Recent insights into structure, formation, and relationship to eukaryotic lipid depots. J Bacteriol  187: 3607–3619.  <ul class="find" data-citedArticleID="1072752" data-doi="10.1128/jb.187.11.3607-3619.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.11.3607-3619.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutral+lipid+bodies+in+prokaryotes%3A+Recent+insights+into+structure%2C+formation%2C+and+relationship+to+eukaryotic+lipid+depots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutral+lipid+bodies+in+prokaryotes%3A+Recent+insights+into+structure%2C+formation%2C+and+relationship+to+eukaryotic+lipid+depots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001778-Sato1" id="pone.0001778-Sato1"></a>Sato S, Nomura CT, Abe H, Doi Y, Tsuge T (2007) Poly [(R)-3-hydroxybutyrate] formation in Escherichia coli from glucose through an enoyl-CoA hydratase-mediated pathway. J Biosci Bioeng  103: 38–44.  <ul class="find" data-citedArticleID="1072742" data-doi="10.1263/jbb.103.38"><li><a href="http://dx.doi.org/10.1263/jbb.103.38" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Poly+%5B%28R%29-3-hydroxybutyrate%5D+formation+in+Escherichia+coli+from+glucose+through+an+enoyl-CoA+hydratase-mediated+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Poly+%5B%28R%29-3-hydroxybutyrate%5D+formation+in+Escherichia+coli+from+glucose+through+an+enoyl-CoA+hydratase-mediated+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001778-Hesselmann2" id="pone.0001778-Hesselmann2"></a>Hesselmann RPX, Von Rummell R, Resnick SM, Hany R, Zehnder AJB (2000) Anaerobic metabolism of bacteria performing enhanced biological phosphate removal. Water Res  34: 3487–3494.  <ul class="find" data-citedArticleID="1072702" data-doi="10.1016/s0043-1354(00)00092-0"><li><a href="http://dx.doi.org/10.1016/s0043-1354(00)00092-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anaerobic+metabolism+of+bacteria+performing+enhanced+biological+phosphate+removal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anaerobic+metabolism+of+bacteria+performing+enhanced+biological+phosphate+removal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001778-Comeau1" id="pone.0001778-Comeau1"></a>Comeau Y, Hall KJ, Hancock REW, Oldham WK (1986) Biochemical model for enhanced biological phosphorus removal. Water Res  20: 1511–1521.  <ul class="find" data-citedArticleID="1072676" data-doi="10.1016/0043-1354(86)90115-6"><li><a href="http://dx.doi.org/10.1016/0043-1354(86)90115-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+model+for+enhanced+biological+phosphorus+removal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+model+for+enhanced+biological+phosphorus+removal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001778-Wentzel1" id="pone.0001778-Wentzel1"></a>Wentzel MC, Lötter LH, Ekama GA, Loewenthal RE, Marais GVR (1991) Evaluation of biochemical models for biological excess phosphorus removal. Water Sci Technol  23: 567–576.  <ul class="find" data-citedArticleID="1072754"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evaluation+of+biochemical+models+for+biological+excess+phosphorus+removal.&amp;auth=&amp;atitle=Evaluation+of+biochemical+models+for+biological+excess+phosphorus+removal." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+biochemical+models+for+biological+excess+phosphorus+removal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+biochemical+models+for+biological+excess+phosphorus+removal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001778-Pereira1" id="pone.0001778-Pereira1"></a>Pereira H, Lemos PC, Reis MAM, Crespo JPSG, Carrondo MJT, et al.  (1996) Model for carbon metabolism in biological phosphorus removal processes based on in vivo <sup>13</sup>C-NMR labelling experiments. Water Res  30: 2128–2138.  <ul class="find" data-citedArticleID="1072732" data-doi="10.1016/0043-1354(96)00035-8"><li><a href="http://dx.doi.org/10.1016/0043-1354(96)00035-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model+for+carbon+metabolism+in+biological+phosphorus+removal+processes+based+on+in+vivo+13C-NMR+labelling+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model+for+carbon+metabolism+in+biological+phosphorus+removal+processes+based+on+in+vivo+13C-NMR+labelling+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001778-Mino1" id="pone.0001778-Mino1"></a>Mino T, Arun V, Tsuzuki Y, Matsuo T (1987)  Effect of phosphorus accumulation on acetate metabolism in the biological phosphorus removal process. In: Ramadori R, editor. Biological Phosphate Removal From Wastewaters. Rome, Italy: Pergamon Press, Oxford. pp. 27–38.  <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001778-Louie1" id="pone.0001778-Louie1"></a>Louie TM, Mah TJ, Oldham W, Ramey WD (2000) Use of metabolic inhibitors and gas chromatography/mass spectrometry to study poly-beta-hydroxyalkanoates metabolism involving cryptic nutrients in enhanced biological phosphorus removal systems. Water Res  34: 1507–1514.  <ul class="find" data-citedArticleID="1072718" data-doi="10.1016/s0043-1354(99)00302-4"><li><a href="http://dx.doi.org/10.1016/s0043-1354(99)00302-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+metabolic+inhibitors+and+gas+chromatography%2Fmass+spectrometry+to+study+poly-beta-hydroxyalkanoates+metabolism+involving+cryptic+nutrients+in+enhanced+biological+phosphorus+removal+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+metabolic+inhibitors+and+gas+chromatography%2Fmass+spectrometry+to+study+poly-beta-hydroxyalkanoates+metabolism+involving+cryptic+nutrients+in+enhanced+biological+phosphorus+removal+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001778-Yagci1" id="pone.0001778-Yagci1"></a>Yagci N, Artan N, Cokgor EU, Randall CW, Orhon D (2003) Metabolic model for acetate uptake by a mixed culture of phosphate- and glycogen-accumulating organisms under anaerobic conditions. Biotechnol Bioeng  84: 359–373.  <ul class="find" data-citedArticleID="1072764" data-doi="10.1002/bit.10765"><li><a href="http://dx.doi.org/10.1002/bit.10765" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+model+for+acetate+uptake+by+a+mixed+culture+of+phosphate-+and+glycogen-accumulating+organisms+under+anaerobic+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+model+for+acetate+uptake+by+a+mixed+culture+of+phosphate-+and+glycogen-accumulating+organisms+under+anaerobic+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001778-vanVeen1" id="pone.0001778-vanVeen1"></a>van Veen HW (1997) Phosphate transport in prokaryotes: molecules, mediators and mechanisms. Antonie Van Leeuwenhoek  72: 299–315.  <ul class="find" data-citedArticleID="1072766" data-doi="10.1023/a:1000530927928"><li><a href="http://dx.doi.org/10.1023/a:1000530927928" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphate+transport+in+prokaryotes%3A+molecules%2C+mediators+and+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphate+transport+in+prokaryotes%3A+molecules%2C+mediators+and+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001778-He1" id="pone.0001778-He1"></a>He S, Gu AZ, McMahon KD (2006) Fine-scale differences between Accumulibacter-like bacteria in enhanced biological phosphorus removal activated sludge. Water Sci Technol  54: 111–117.  <ul class="find" data-citedArticleID="1072696" data-doi="10.2166/wst.2006.378"><li><a href="http://dx.doi.org/10.2166/wst.2006.378" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine-scale+differences+between+Accumulibacter-like+bacteria+in+enhanced+biological+phosphorus+removal+activated+sludge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine-scale+differences+between+Accumulibacter-like+bacteria+in+enhanced+biological+phosphorus+removal+activated+sludge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001778-Lo1" id="pone.0001778-Lo1"></a>Lo I, Denef VJ, VerBerkmoes NC, Shah MB, Goltsman D, et al.  (2007) Strain-resolved community proteomics reveals recombining genomes of acidophilic bacteria. Nature  446: 537–541.  <ul class="find" data-citedArticleID="1072716" data-doi="10.1038/nature05624"><li><a href="http://dx.doi.org/10.1038/nature05624" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strain-resolved+community+proteomics+reveals+recombining+genomes+of+acidophilic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strain-resolved+community+proteomics+reveals+recombining+genomes+of+acidophilic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001778-Hu1" id="pone.0001778-Hu1"></a>Hu QZ, Noll RJ, Li HY, Makarov A, Hardman M, et al.  (2005) The Orbitrap: a new mass spectrometer. J Mass Spectrom  40: 430–443.  <ul class="find" data-citedArticleID="1072704" data-doi="10.1002/jms.856"><li><a href="http://dx.doi.org/10.1002/jms.856" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Orbitrap%3A+a+new+mass+spectrometer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Orbitrap%3A+a+new+mass+spectrometer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001778-Manz1" id="pone.0001778-Manz1"></a>Manz W, Amann R, Ludwig W, Wagner M, Schleifer KH (1992) Phylogenetic oligodeoxynucleotide probes for the major subclasses of proteobacteria: problems and solutions. Syst Appl Microbiol  15: 593–600.  <ul class="find" data-citedArticleID="1072720" data-doi="10.1016/s0723-2020(11)80121-9"><li><a href="http://dx.doi.org/10.1016/s0723-2020(11)80121-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+oligodeoxynucleotide+probes+for+the+major+subclasses+of+proteobacteria%3A+problems+and+solutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+oligodeoxynucleotide+probes+for+the+major+subclasses+of+proteobacteria%3A+problems+and+solutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001778-Roller1" id="pone.0001778-Roller1"></a>Roller C, Wagner M, Amann R, Ludwig W, Schleifer K-H (1994) <em>In situ</em> probing of Gram-positive bacteria with high DNA G+C content using 23S rRNA-targeted oligonucleotides. Microbiology  140: 2849–2858.  <ul class="find" data-citedArticleID="1072738" data-doi="10.1099/00221287-140-10-2849"><li><a href="http://dx.doi.org/10.1099/00221287-140-10-2849" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+situ+probing+of+Gram-positive+bacteria+with+high+DNA+G%2BC+content+using+23S+rRNA-targeted+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+situ+probing+of+Gram-positive+bacteria+with+high+DNA+G%2BC+content+using+23S+rRNA-targeted+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001778-Crocetti2" id="pone.0001778-Crocetti2"></a>Crocetti GR, Banfield JF, Keller J, Bond PL, Blackall LL (2002) Glycogen-accumulating organisms in laboratory-scale and full-scale wastewater treatment processes. Microbiology  148: 3353–3364.  <ul class="find" data-citedArticleID="1072680"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Glycogen-accumulating+organisms+in+laboratory-scale+and+full-scale+wastewater+treatment+processes.&amp;auth=&amp;atitle=Glycogen-accumulating+organisms+in+laboratory-scale+and+full-scale+wastewater+treatment+processes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glycogen-accumulating+organisms+in+laboratory-scale+and+full-scale+wastewater+treatment+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glycogen-accumulating+organisms+in+laboratory-scale+and+full-scale+wastewater+treatment+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001778-Daims1" id="pone.0001778-Daims1"></a>Daims H, Bruhl A, Amann R, Schleifer KH, Wagner M (1999) The domain-specific probe EUB338 is insufficient for the detection of all Bacteria: Development and evaluation of a more comprehensive probe set. Syst Appl Microbiol  22: 434–444.  <ul class="find" data-citedArticleID="1072682" data-doi="10.1016/s0723-2020(99)80053-8"><li><a href="http://dx.doi.org/10.1016/s0723-2020(99)80053-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+domain-specific+probe+EUB338+is+insufficient+for+the+detection+of+all+Bacteria%3A+Development+and+evaluation+of+a+more+comprehensive+probe+set." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+domain-specific+probe+EUB338+is+insufficient+for+the+detection+of+all+Bacteria%3A+Development+and+evaluation+of+a+more+comprehensive+probe+set.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001778-Falkentoft1" id="pone.0001778-Falkentoft1"></a>Falkentoft CM, Muller E, Arnz P, Harremoes P, Mosbaek H, et al.  (2002) Population changes in a biofilm reactor for phosphorus removal as evidenced by the use of FISH. Water Res  36: 491–500.  <ul class="find" data-citedArticleID="1072686" data-doi="10.1016/s0043-1354(01)00231-7"><li><a href="http://dx.doi.org/10.1016/s0043-1354(01)00231-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+changes+in+a+biofilm+reactor+for+phosphorus+removal+as+evidenced+by+the+use+of+FISH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+changes+in+a+biofilm+reactor+for+phosphorus+removal+as+evidenced+by+the+use+of+FISH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001778-Hesketh1" id="pone.0001778-Hesketh1"></a>Hesketh AR, Chandra G, Shaw AD, Rowland JJ, Kell DB, et al.  (2002) Primary and secondary metabolism, and post-translational protein modifications, as portrayed by proteomic analysis of <em>Streptomyces coelicolor</em>. Mol Microbiol  46: 917–932.  <ul class="find" data-citedArticleID="1072698" data-doi="10.1046/j.1365-2958.2002.03219.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03219.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primary+and+secondary+metabolism%2C+and+post-translational+protein+modifications%2C+as+portrayed+by+proteomic+analysis+of+Streptomyces+coelicolor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primary+and+secondary+metabolism%2C+and+post-translational+protein+modifications%2C+as+portrayed+by+proteomic+analysis+of+Streptomyces+coelicolor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001778-Rutherford1" id="pone.0001778-Rutherford1"></a>Rutherford K, Parkhill J, Crook J, Horsnell T, Rice P, et al.  (2000) Artemis: sequence visualization and annotation. Bioinformatics  16: 944–945.  <ul class="find" data-citedArticleID="1072740" data-doi="10.1093/bioinformatics/16.10.944"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.10.944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Artemis%3A+sequence+visualization+and+annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Artemis%3A+sequence+visualization+and+annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.XML" value="132672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.PDF" value="589938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t001.PNG_L" value="67579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t001.PNG_M" value="91041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t001.PNG_S" value="13762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t001.TIF" value="328988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t001.PNG_I" value="30116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g001.PNG_L" value="1242575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g001.PNG_M" value="253065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g001.PNG_S" value="12590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g001.TIF" value="1773358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g001.PNG_I" value="83715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g002.PNG_L" value="837307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g002.PNG_M" value="105345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g002.PNG_S" value="8510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g002.TIF" value="965298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g002.PNG_I" value="33618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g003.PNG_L" value="269940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g003.PNG_M" value="120470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g003.PNG_S" value="8542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g003.TIF" value="291956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g003.PNG_I" value="38010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t002.PNG_L" value="526498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t002.PNG_M" value="152291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t002.PNG_S" value="19104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t002.TIF" value="2421102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.t002.PNG_I" value="92146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g004.PNG_L" value="886584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g004.PNG_M" value="163089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g004.PNG_S" value="20869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g004.TIF" value="1178268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.g004.PNG_I" value="167519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.s001.DOC" value="102912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.s002.DOC" value="33280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.s003.DOC" value="24576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.s004.DOC" value="883712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001778.s005.TIF" value="4059604"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001778&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001778" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001778&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001778&volume=&issue=&title=Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment&author_name=Paul%20Wilmes%2C%20Margaret%20Wexler%2C%20Philip%20L.%20Bond&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001778" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001778" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001778" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001778&amp;t=Metaproteomics%20Provides%20Functional%20Insight%20into%20Activated%20Sludge%20Wastewater%20Treatment" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001778&title=Metaproteomics%20Provides%20Functional%20Insight%20into%20Activated%20Sludge%20Wastewater%20Treatment&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001778&amp;title=Metaproteomics%20Provides%20Functional%20Insight%20into%20Activated%20Sludge%20Wastewater%20Treatment" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001778" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001778&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Metaproteomics Provides Functional Insight into Activated Sludge Wastewater Treatment';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001778';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Metaproteomics%20Provides%20Functional%20Insight%20into%20Activated%20Sludge%20Wastewater%20Treatment http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001778" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001778" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fatty+acids%22" title="Search for articles in the subject area:'Fatty acids'"><div class="flagText">Fatty acids</div></a>
              <div data-categoryid="18501" data-articleid="26380"
                   data-categoryname="Fatty acids"
                   class="flagImage" title="Flag 'Fatty acids' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glycogens%22" title="Search for articles in the subject area:'Glycogens'"><div class="flagText">Glycogens</div></a>
              <div data-categoryid="18143" data-articleid="26380"
                   data-categoryname="Glycogens"
                   class="flagImage" title="Flag 'Glycogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Metagenomics%22" title="Search for articles in the subject area:'Metagenomics'"><div class="flagText">Metagenomics</div></a>
              <div data-categoryid="18385" data-articleid="26380"
                   data-categoryname="Metagenomics"
                   class="flagImage" title="Flag 'Metagenomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphates%22" title="Search for articles in the subject area:'Phosphates'"><div class="flagText">Phosphates</div></a>
              <div data-categoryid="19751" data-articleid="26380"
                   data-categoryname="Phosphates"
                   class="flagImage" title="Flag 'Phosphates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+expression%22" title="Search for articles in the subject area:'Protein expression'"><div class="flagText">Protein expression</div></a>
              <div data-categoryid="17513" data-articleid="26380"
                   data-categoryname="Protein expression"
                   class="flagImage" title="Flag 'Protein expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+metabolism%22" title="Search for articles in the subject area:'Protein metabolism'"><div class="flagText">Protein metabolism</div></a>
              <div data-categoryid="19757" data-articleid="26380"
                   data-categoryname="Protein metabolism"
                   class="flagImage" title="Flag 'Protein metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomics%22" title="Search for articles in the subject area:'Proteomics'"><div class="flagText">Proteomics</div></a>
              <div data-categoryid="18223" data-articleid="26380"
                   data-categoryname="Proteomics"
                   class="flagImage" title="Flag 'Proteomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sludge%22" title="Search for articles in the subject area:'Sludge'"><div class="flagText">Sludge</div></a>
              <div data-categoryid="25329" data-articleid="26380"
                   data-categoryname="Sludge"
                   class="flagImage" title="Flag 'Sludge' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7815'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5652'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8505&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
