

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001420"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001420" />

    <meta name="citation_title" content="Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma"/>
    <meta itemprop="name" content="Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma"/>

      <meta name="citation_author" content="Yuka Nadai"/>
            <meta name="citation_author_institution" content="Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Lindsay M. Eyzaguirre"/>
            <meta name="citation_author_institution" content="Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Niel T. Constantine"/>
            <meta name="citation_author_institution" content="Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Anne M. Sill"/>
            <meta name="citation_author_institution" content="Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Farley Cleghorn"/>
            <meta name="citation_author_institution" content="Futures Group, Washington, D. C., United States of America"/>
      <meta name="citation_author" content="William A. Blattner"/>
            <meta name="citation_author_institution" content="Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Jean K. Carr"/>
            <meta name="citation_author_institution" content="Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America"/>

    <meta name="citation_date" content="2008/1/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001420.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1420"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Complete nucleotide sequence of the AIDS virus, HTLV-III.; citation_author=L Ratner; citation_author=W Haseltine; citation_author=R Patarca; citation_author=KJ Livak; citation_author=B Starcich; citation_journal_title=Nature; citation_volume=313; citation_number=1; citation_pages=277-284; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=HIV Sequence Database Web Site: Theoretical biology and biophysics group, Los Alamos National Laboratory.; citation_number=2; " />
      <meta name="citation_reference" content="citation_title=Recovery of virtually full-length HIV-1 provirus of diverse subtypes from primary virus cultures using the polymerase chain reaction.; citation_author=MO Salminen; citation_author=C Koch; citation_author=E Sanders-Buell; citation_author=PK Ehrenberg; citation_author=NL Michael; citation_journal_title=Virology; citation_volume=213; citation_number=3; citation_pages=80-86; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Characterization of subtype A HIV-1 from Africa by full genome sequencing.; citation_author=JK Carr; citation_author=T Laukkanen; citation_author=MO Salminen; citation_author=J Albert; citation_author=A Alaeus; citation_journal_title=Aids; citation_volume=13; citation_number=4; citation_pages=1819-1826; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=HIV type 1 subtypes among blood donors in the Mbeya region of southwest Tanzania.; citation_author=MA Arroyo; citation_author=M Hoelscher; citation_author=E Sanders-Buell; citation_author=KH Herbinger; citation_author=E Samky; citation_journal_title=AIDS Res Hum Retroviruses; citation_volume=20; citation_number=5; citation_pages=895-901; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Viral dynamics in human immunodeficiency virus type 1 infection.; citation_author=X Wei; citation_author=SK Ghosh; citation_author=ME Taylor; citation_author=VA Johnson; citation_author=EA Emini; citation_journal_title=Nature; citation_volume=373; citation_number=6; citation_pages=117-122; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Quantification of latent tissue reservoirs and total body viral load in HIV-1 infection.; citation_author=TW Chun; citation_author=L Carruth; citation_author=D Finzi; citation_author=X Shen; citation_author=JA DiGiuseppe; citation_journal_title=Nature; citation_volume=387; citation_number=7; citation_pages=183-188; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=HIV population dynamics in vivo: implications for genetic variation, pathogenesis, and therapy.; citation_author=JM Coffin; citation_journal_title=Science; citation_volume=267; citation_number=8; citation_pages=483-489; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Discontinuous sequence change of human immunodeficiency virus (HIV) type 1 env sequences in plasma viral and lymphocyte-associated proviral populations in vivo: implications for models of HIV pathogenesis.; citation_author=P Simmonds; citation_author=LQ Zhang; citation_author=F McOmish; citation_author=P Balfe; citation_author=CA Ludlam; citation_journal_title=J Virol; citation_volume=65; citation_number=9; citation_pages=6266-6276; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Methods for subtyping and molecular comparison of human viral genomes.; citation_author=M Arens; citation_journal_title=Clin Microbiol Rev; citation_volume=12; citation_number=10; citation_pages=612-626; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A genotypic assay for the amplification and sequencing of gag and protease from diverse human immunodeficiency virus type 1 group M subtypes.; citation_author=K Van Laethem; citation_author=Y Schrooten; citation_author=S Dedecker; citation_author=L Van Heeswijck; citation_author=K Deforche; citation_journal_title=J Virol Methods; citation_volume=132; citation_number=11; citation_pages=181-186; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Primalign HIV Sequence Database tool: Theoretical biology and biophysics group, Los Alamos National Laboratory.; citation_number=12; " />
      <meta name="citation_reference" content="citation_title=Quantitation of HIV-1 RNA in plasma predicts outcome after seroconversion.; citation_author=JW Mellors; citation_author=LA Kingsley; citation_author=CR Rinaldo; citation_author=JA Todd; citation_author=BS Hoo; citation_journal_title=Ann Intern Med; citation_volume=122; citation_number=13; citation_pages=573-579; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=High levels of HIV-1 in plasma during all stages of infection determined by competitive PCR.; citation_author=M Piatak; citation_author=MS Saag; citation_author=LC Yang; citation_author=SJ Clark; citation_author=JC Kappes; citation_journal_title=Science; citation_volume=259; citation_number=14; citation_pages=1749-1754; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Improved conditions for extraction and amplification of human immunodeficiency virus type 1 RNA from plasma samples with low viral load.; citation_author=ML Villahermosa; citation_author=M Thomson; citation_author=E Vazquez de Parga; citation_author=MT Cuevas; citation_author=G Contreras; citation_journal_title=J Hum Virol; citation_volume=3; citation_number=15; citation_pages=27-34; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Template-directed pausing of DNA synthesis by HIV-1 reverse transcriptase during polymerization of HIV-1 sequences in vitro.; citation_author=GJ Klarmann; citation_author=CA Schauber; citation_author=BD Preston; citation_journal_title=J Biol Chem; citation_volume=268; citation_number=16; citation_pages=9793-9802; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Pausing of reverse transcriptase on retroviral RNA templates is influenced by secondary structures both 5′ and 3′ of the catalytic site.; citation_author=GP Harrison; citation_author=MS Mayo; citation_author=E Hunter; citation_author=AM Lever; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=17; citation_pages=3433-3442; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Full-length cDNAs: more than just reaching the ends.; citation_author=M Das; citation_author=I Harvey; citation_author=LL Chu; citation_author=M Sinha; citation_author=J Pelletier; citation_journal_title=Physiol Genomics; citation_volume=6; citation_number=18; citation_pages=57-80; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Thermal effects on reverse transcription: improvement of accuracy and processivity in cDNA synthesis.; citation_author=CM Malboeuf; citation_author=SJ Isaacs; citation_author=NH Tran; citation_author=B Kim; citation_journal_title=Biotechniques; citation_volume=30; citation_number=19; citation_pages=1074-1078, 1080, 1082, passim; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Thermostabilization and thermoactivation of thermolabile enzymes by trehalose and its application for the synthesis of full length cDNA.; citation_author=P Carninci; citation_author=Y Nishiyama; citation_author=A Westover; citation_author=M Itoh; citation_author=S Nagaoka; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=20; citation_pages=520-524; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Non-enzymatic RNA hydrolysis promoted by the combined catalytic activity of buffers and magnesium ions.; citation_author=MG AbouHaidar; citation_author=IG Ivanov; citation_journal_title=Z Naturforsch [C]; citation_volume=54; citation_number=21; citation_pages=542-548; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Excess dNTPs minimize RNA hydrolysis during reverse transcription.; citation_author=GF Gerard; citation_author=S Collins; citation_author=MD Smith; citation_journal_title=Biotechniques; citation_volume=33; citation_number=22; citation_pages=984, 986, 988 passim; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection of high levels of recombination generated during PCR amplification of RNA templates.; citation_author=W Yu; citation_author=KJ Rusterholtz; citation_author=AT Krummel; citation_author=N Lehman; citation_journal_title=Biotechniques; citation_volume=40; citation_number=23; citation_pages=499-507; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Minimizing DNA recombination during long RT-PCR.; citation_author=G Fang; citation_author=G Zhu; citation_author=H Burger; citation_author=JS Keithly; citation_author=B Weiser; citation_journal_title=J Virol Methods; citation_volume=76; citation_number=24; citation_pages=139-148; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Stimulation and suppression of PCR-mediated recombination.; citation_author=MS Judo; citation_author=AB Wedel; citation_author=C Wilson; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=25; citation_pages=1819-1825; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Offset recombinant PCR: a simple but effective method for shuffling compact heterologous domains.; citation_author=DA Rozak; citation_author=PN Bryan; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=26; citation_pages=e82; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Viral quasispecies.; citation_author=M Eigen; citation_journal_title=Sci Am; citation_volume=269; citation_number=27; citation_pages=42-49; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Temporal fluctuations in HIV quasispecies in vivo are not reflected by sequential HIV isolations.; citation_author=A Meyerhans; citation_author=R Cheynier; citation_author=J Albert; citation_author=M Seth; citation_author=S Kwok; citation_journal_title=Cell; citation_volume=58; citation_number=28; citation_pages=901-910; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Fidelity of HIV-1 reverse transcriptase copying RNA in vitro.; citation_author=JP Ji; citation_author=LA Loeb; citation_journal_title=Biochemistry; citation_volume=31; citation_number=29; citation_pages=954-958; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of retroviral mutation.; citation_author=BD Preston; citation_author=JP Dougherty; citation_journal_title=Trends Microbiol; citation_volume=4; citation_number=30; citation_pages=16-21; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=From RNA to quasispecies: a DNA polymerase with proofreading activity is highly recommended for accurate assessment of viral diversity.; citation_author=I Malet; citation_author=M Belnard; citation_author=H Agut; citation_author=A Cahour; citation_journal_title=J Virol Methods; citation_volume=109; citation_number=31; citation_pages=161-170; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Broad antiretroviral defence by human APOBEC3G through lethal editing of nascent reverse transcripts.; citation_author=B Mangeat; citation_author=P Turelli; citation_author=G Caron; citation_author=M Friedli; citation_author=L Perrin; citation_journal_title=Nature; citation_volume=424; citation_number=32; citation_pages=99-103; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Hypermutation of HIV-1 DNA in the absence of the Vif protein.; citation_author=D Lecossier; citation_author=F Bouchonnet; citation_author=F Clavel; citation_author=AJ Hance; citation_journal_title=Science; citation_volume=300; citation_number=33; citation_pages=1112; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The cytidine deaminase CEM15 induces hypermutation in newly synthesized HIV-1 DNA.; citation_author=H Zhang; citation_author=B Yang; citation_author=RJ Pomerantz; citation_author=C Zhang; citation_author=SC Arunachalam; citation_journal_title=Nature; citation_volume=424; citation_number=34; citation_pages=94-98; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA deamination mediates innate immunity to retroviral infection.; citation_author=RS Harris; citation_author=KN Bishop; citation_author=AM Sheehy; citation_author=HM Craig; citation_author=SK Petersen-Mahrt; citation_journal_title=Cell; citation_volume=113; citation_number=35; citation_pages=803-809; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Global epidemiology of HIV.; citation_author=FE McCutchan; citation_journal_title=J Med Virol; citation_volume=78; citation_number=36; citation_pages=S7-S12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recombination in HIV-1.; citation_author=DL Robertson; citation_author=PM Sharp; citation_author=FE McCutchan; citation_author=BH Hahn; citation_journal_title=Nature; citation_volume=374; citation_number=37; citation_pages=124-126; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Assessment of RNA quality by semi-quantitative RT-PCR of multiple regions of a long ubiquitous mRNA.; citation_author=GH Swift; citation_author=MJ Peyton; citation_author=RJ MacDonald; citation_journal_title=Biotechniques; citation_volume=28; citation_number=38; citation_pages=524, 526, 528, 530-521; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Large-scale amplification, cloning and sequencing of near full-length HIV-1 subtype C genomes.; citation_author=CM Rousseau; citation_author=BA Birditt; citation_author=AR McKay; citation_author=JN Stoddard; citation_author=TC Lee; citation_journal_title=J Virol Methods; citation_volume=136; citation_number=39; citation_pages=118-125; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Optimization of in vitro transcription and full-length cDNA synthesis using the T4 bacteriophage gene 32 protein.; citation_author=C Piche; citation_author=JP Schernthaner; citation_journal_title=J Biomol Tech; citation_volume=16; citation_number=40; citation_pages=239-247; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dimethyl sulfoxide-mediated primer Tm reduction: a method for analyzing the role of renaturation temperature in the polymerase chain reaction.; citation_author=N Chester; citation_author=DR Marshak; citation_journal_title=Anal Biochem; citation_volume=209; citation_number=41; citation_pages=284-290; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Full-length cDNA synthesis for long-distance RT-PCR of large mRNA transcripts.; citation_author=PR Hawkins; citation_author=P Jin; citation_author=GK Fu; citation_journal_title=Biotechniques; citation_volume=34; citation_number=42; citation_pages=768-773; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001420" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma"/>
    <meta name="twitter:description" content="Nearly full-length genome sequencing of HIV-1 using peripheral blood mononuclear cells (PBMC) DNA as a template for PCR is now a relatively routine laboratory procedure. However, this has not been the case when using virion RNA as the template and this has made full genome analysis of circulating viruses difficult. Therefore, a well-developed procedure for sequencing of full-length HIV-1 RNA directly from plasma was needed. Plasma from U.S. donors representing a range of viral loads (VL) was used to develop the assay. RNA was extracted from plasma and reverse-transcribed. Two or three overlapping regions were PCR amplified to cover the entire viral genome and sequenced for verification. The success of the procedure was sensitive to VL but was routinely successful for VL greater than 105 and the rate declined in proportion to the VL. While the two-amplicon strategy had an advantage of increasing the possibility of amplifying a single species of HIV-1, the three-amplicon strategy was more successful in amplifying samples with low viral loads. This protocol provides a useful tool for molecular analysis to understand the HIV epidemic and pathogenesis, as well as diagnosis, therapy and future vaccine strategies."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001420.g001"/>

  <meta property="og:title" content="Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001420" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4488'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=969'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6987&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001420">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001420" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001420&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuka Nadai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lindsay M. Eyzaguirre, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Niel T. Constantine, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anne M. Sill, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Farley Cleghorn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Futures Group, Washington, D. C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William A. Blattner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean K. Carr
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jecarr@ihv.umaryland.edu">jecarr@ihv.umaryland.edu</a></p>

                <p>Affiliation:
                  Department of Epidemiology, Institute of Human Virology, University of Maryland, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001420</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001420" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001420" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001420" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001420" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001420-g001" data-doi="info:doi/10.1371/journal.pone.0001420" data-uri="info:doi/10.1371/journal.pone.0001420.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001420-t001" data-doi="info:doi/10.1371/journal.pone.0001420" data-uri="info:doi/10.1371/journal.pone.0001420.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001420-t002" data-doi="info:doi/10.1371/journal.pone.0001420" data-uri="info:doi/10.1371/journal.pone.0001420.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001420">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001420" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Nearly full-length genome sequencing of HIV-1 using peripheral blood mononuclear cells (PBMC) DNA as a template for PCR is now a relatively routine laboratory procedure. However, this has not been the case when using virion RNA as the template and this has made full genome analysis of circulating viruses difficult. Therefore, a well-developed procedure for sequencing of full-length HIV-1 RNA directly from plasma was needed. Plasma from U.S. donors representing a range of viral loads (VL) was used to develop the assay. RNA was extracted from plasma and reverse-transcribed. Two or three overlapping regions were PCR amplified to cover the entire viral genome and sequenced for verification. The success of the procedure was sensitive to VL but was routinely successful for VL greater than 10<sup>5</sup> and the rate declined in proportion to the VL. While the two-amplicon strategy had an advantage of increasing the possibility of amplifying a single species of HIV-1, the three-amplicon strategy was more successful in amplifying samples with low viral loads. This protocol provides a useful tool for molecular analysis to understand the HIV epidemic and pathogenesis, as well as diagnosis, therapy and future vaccine strategies.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nadai Y, Eyzaguirre LM, Constantine NT, Sill AM, Cleghorn F, et al.  (2008) Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma. PLoS ONE 3(1):
          e1420.
            doi:10.1371/journal.pone.0001420</p><p><strong>Academic Editor: </strong>Vineet KewalRamani, National Cancer Institute, United States of America</p><p><strong>Received:</strong> September 26, 2007; <strong>Accepted:</strong> December 7, 2007; <strong>Published:</strong> January 9, 2008</p><p><strong>Copyright:</strong> © 2008 Nadai et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The present work was supported by an NIH grant R21-A1057084-02 and a cooperative agreement between the Henry M. Jackson Foundation for the Advancement of Military Medicine and the U.S. Department of Defense.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Since the first complete sequence of the HIV-1 genome was published in 1985 <a href="#pone.0001420-Ratner1">[1]</a>, there have been numerous reports documenting the genetic variability of the virus based on sequencing. In the HIV sequence database <a href="#pone.0001420-1">[2]</a> there are currently 1404 HIV-1 sequences greater than 8 kb in length. However, the vast majority of these sequences have been generated using DNA from either peripheral blood mononuclear cells (PBMC) or cultured cells <a href="#pone.0001420-Ratner1">[1]</a>, <a href="#pone.0001420-Salminen1">[3]</a>–<a href="#pone.0001420-Arroyo1">[5]</a>. In fact, only 74 sequences from 47 individuals are identified as derived from RNA and all of them were generated using cloning; there are no full-length HIV-1 sequences that are clearly from direct PCR amplification and sequencing of virion RNA. The reason for this is that generating a single, nearly full-length PCR amplicon of HIV-1 using virion RNA as the template has been relatively difficult.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The ability to monitor the sequence of the full genome of HIV-1 in the virion is important for many reasons. Virion RNA reflects currently replicating virus, and thus has more pathogenic significance than the proviral DNA present in PBMC <a href="#pone.0001420-Wei1">[6]</a>–<a href="#pone.0001420-Simmonds1">[9]</a>. Proviral DNA contains high levels of integrated defective viral sequences <a href="#pone.0001420-Wei1">[6]</a>. Thus the analysis of virion RNA is ideal for studying the pathogenesis and epidemiology of HIV-1. Moreover, it is often the case that PBMC are not available from important study populations, and studies must be conducted using plasma or serum. Such studies have often been restricted to partial genome characterization because of the technical difficulty of using RNA as the template <a href="#pone.0001420-Arens1">[10]</a>, <a href="#pone.0001420-VanLaethem1">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A well-developed strategy for amplification and sequencing of nearly full-length HIV-1 directly from plasma viral RNA was needed. Standard sequencing methods require efficient PCR amplification and this is more difficult in viral RNA samples because of common problems in synthesizing full-length cDNA. Recent improvements in commercially available RNase H<sup>−</sup> free reverse transcriptase (RT) have made it possible to synthesize cDNA for nearly full-length HIV-1 RNA genomes (~9 kb). In this report, optimal conditions for reverse transcription and PCR amplification (RT-PCR) of HIV-1 RNA in plasma samples were defined, resulting in a simple, sensitive, and effective assay to sequence nearly full-length HIV-1 genomes from virion RNA.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Study Subjects</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Plasma samples were obtained from the Molecular Diagnostics Laboratory at the University of Maryland Medical Center in Baltimore, MD, a CAP-accredited facility. Unidentified, archived samples were selected based on the VL, ranging from 1,000 to over 75,0000 copies/ml. VL testing was performed using the COBAS Amplicor HIV-1 Monitor Test, version 1.5. (Roche Molecular Systems Inc., Durham, NC).</p>


<h4>RNA Extraction</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>RNA was extracted from 500 µl HIV-1 positive plasma specimens using QIAamp Viral RNA Mini Assay (Qiagen, Valencia, CA). Virions were initially concentrated from the plasma or serum by centrifugation at 21,000×g for 75 min. All but 140 µl of the supernatant was removed before lysis buffer was added. Extracted RNA (60 µl) was either immediately reverse transcribed or stored at −80°C until use.</p>


<h4>Reverse Transcription</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The methods for RT- PCR are illustrated in <a href="#pone-0001420-g001">Figure 1</a>. Reverse transcription of the RNA was performed by priming with UNINEF7′ (<a href="#pone-0001420-t001">Table 1</a>) close to the 3′ end of the viral RNA or by VIF-VPUoutR1 (<a href="#pone-0001420-t001">Table 1</a>) in the <em>vpu</em> gene. The extracted RNA (3 µl) was reverse transcribed in a total volume of 20 µl with 500 µM dNTP, 2.5 µM primer, 1× RT buffer, 5 mM MgCl2, 10 mM DTT, 40 U RnaseOUT, and 400 U SuperScript<sup>TM</sup> III RNase H<sup>−</sup> RT (Invitrogen, Carlsbad, CA). The RNA, primer and dNTPs were first incubated at 65°C for 5 minutes, then the remaining reagents were added for cDNA synthesis at 50°C for 2 hours, followed by 85°C for 5 minutes. Then 2 U E. coli RNase H (Invitrogen, Carlsbad, CA) was added, and the reaction tubes were incubated at 37°C for 20 minutes followed by 70°C for 15 minutes.</p>
<div class="figure" id="pone-0001420-g001"><div class="img"><a name="pone-0001420-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001420" data-uri="info:doi/10.1371/journal.pone.0001420.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001420.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001420.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Nearly full-length RT-PCR method.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Viral RNA was reverse-transcribed by priming with UNINEF 7′ or with VIF-VPUoutR1 using SuperScript<sup>TM</sup> III RNase H<sup>−</sup> RT. The locations of all the primers used for cDNA synthesis and nested PCR were depicted in this diagram with the map of the complete HIV-1 genome. Two different strategies were employed to amplify the nearly full-length genome: one amplified the 2.6-kb (<em>gag-pol</em>) and 7.0-kb (<em>pol-nef</em>) fragments with the overlap of 797-bp, and the other amplified three overlapping fragments of 2.6-kb (<em>gag-pol</em>), 3.7-kb (<em>pol-vpu</em>) and 3.3-kb (<em>env-nef</em>) with the 797-bp and 321-bp over lap regions, respectively.</p>
<span>doi:10.1371/journal.pone.0001420.g001</span></div><div class="figure" id="pone-0001420-t001"><div class="img"><a name="pone-0001420-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001420" data-uri="info:doi/10.1371/journal.pone.0001420.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001420.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001420.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Primers for the amplification of nearly full-length HIV-1 genome.</span></strong></p><span>doi:10.1371/journal.pone.0001420.t001</span></div>

<h4>PCR Amplification</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Two or three regions of the viral genome were independently amplified using nested PCR from the cDNA to contain a nearly full-length genome of HIV-1 (amplification primers listed in <a href="#pone-0001420-g001">Figure 1</a> and <a href="#pone-0001420-t001">Table 1</a>). The two-amplicon strategy consisted of one amplicon of about 2.6-kb consisting of <em>gag</em> and part of <em>pol</em> (nts 769–3338 HXB-2 (Genbank Acc No: K03455)) and another amplicon of approximately 7.0-kb stretching from the 5′ end of <em>pol</em> to the middle of <em>nef</em> (nts 2143–9181 HXB-2). The reaction volume was 50 µl, containing 1× PCR buffer, 350 µM dNTP mixture, 0.4 µM of each primer and 5 U Expand Long Template PCR enzyme mixture (Roche Diagnostics, Indianapolis, IN). Cycling conditions were 94°C for 2 minutes and then 10 cycles of (94°C 10 s, 60°C 30 s, 68°C 3 min), then 20 cycles of (94°C 10 s, 55°C 30 s, 68°C 3 min) followed by 68°C for 10 minutes. For the second amplicon, cycling conditions were the same, except that the annealing temperature for first 10 cycles was 65°C and the incubation at 68°C in each cycle was for 8 minutes. A hot-start using DynaWax (Finnzymes, Espoo, Finland) was employed in each PCR, with two layers separated by wax. In the bottom layer were the dNTPs and primers while the top layer contained the PCR buffer, DNA polymerase and template cDNA. Limiting dilution of template into the first round was used to minimize the chances of producing mosaic molecules. The amplicon that could be obtained with the lowest amount of cDNA input was applied to the following sequencing step.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>A three-amplicon strategy was used as needed. The first amplicon was the <em>gag</em>-<em>pol</em> amplicon described above, the second spanned <em>pol</em> to <em>vpu</em> (nts 2483–6231 HXB-2) and the third <em>env</em> to <em>nef</em> (nts 5861–9181 HXB-2) (<a href="#pone-0001420-g001">Figure 1</a> and <a href="#pone-0001420-t001">Table 1</a>). The cycling conditions for these two regions were the same as the 2.6-kb <em>gag-pol</em> region, except that the incubation at 68°C in each cycle was for 4 minutes rather than 3 minutes.</p>


<h4>Sequencing</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The PCR products for sequencing were filtered with Microcon YM-50 (Millipore Corporation, Bedford, MA, USA), and then the PCR products were directly sequenced using fluorescently labeled dideoxy chain terminators (BigDye® Terminator v3.1 Cycle Sequencing Assay, Applied Biosystems) and an automated 3130xl sequencer (Applied Biosystems). The primary set of sequencing primers used for each amplicon is listed in a supplemental material (<a href="#pone.0001420.s001">Table S1</a>). Sequencing reactions were repeated with alternate primers as needed to fill any gaps present to complete the sequencing of both DNA strands in each amplicon. The nearly full-length genome sequence was assembled by overlapping the sequences of the two or three amplicons and merging them into one sequence as long as the two were greater than 98% homologous.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Optimization of Extraction Method</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>QIAamp viral RNA mini assay was selected after comparisons with other commercially available extraction assays for its reliability and efficacy. RNA extraction using a volume of 500 µl was useful to improve sensitivity for low VL samples. To show the standardized correlation between VLs and RT-PCR efficacies, all the samples listed in <a href="#pone-0001420-t002">Table 2</a> were concentrated from the original volume of 500 µl.</p>
<div class="figure" id="pone-0001420-t002"><div class="img"><a name="pone-0001420-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001420" data-uri="info:doi/10.1371/journal.pone.0001420.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001420.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001420.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001420.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001420.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001420.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Comparative Sensitivities of HIV-1 Viral loads in Plasma samples for RT-PCR Detection of HIV-1 RNA.</span></strong></p><span>doi:10.1371/journal.pone.0001420.t002</span></div>

<h4>Optimization of Reverse Transcription</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Several different types of commercially available RT were compared, and SuperScript™ III RNase H<sup>−</sup> RT had the highest yield. The incubation temperature for cDNA synthesis was 50°C, which was recommended by the manufacturer. The yield did not increase when temperature was raised to 53°C and 55°C.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The following three primers were compared for the reverse transcription: Oligo dT, random hexamer and the specific primer for HIV-1 genome, UNINEF 7′. Generally, priming with Oligo dT and UNINEF 7′ resulted in much better yields for synthesizing full-length cDNA than random hexamer, and UNINEF 7′ tended to have more consistent results than Oligo dT. Therefore, UNINEF 7′ was used for all the reverse transcription reactions to obtain the data for the further experiments.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Originally 200 U of SuperScript™ III RT was employed in a 21 µL reaction volume; however, the increased input of the enzyme to 400 U significantly improved the efficacy of RT-PCR. In the samples with the VL range of 10,000–200,000 copies/ml, an increase of 30% was observed for the 7.3 kb <em>pol-nef</em> region and 10% for the 2.6 kb <em>gag-pol</em> region.</p>


<h4>Primer design and optimization of PCR</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The primers, listed in the legend to <a href="#pone-0001420-t001">Table 1</a>, were designed using the full-length genome alignment of diverse group M HIV-1 strains from the HIV sequence database <a href="#pone.0001420-1">[2]</a>. The conserved sites surrounding or within the target region of the HIV-1 sequences were examined for optimal PCR primer sequences. The primers were designed to achieve annealing at a temperature of &gt;55°C, and to avoid hairpins, self-priming, and primer-dimer formation. Primers were screened for specificity and evaluated to be able to amplify most of the strains within the diverse 9 subtypes of the group M HIV-1 using the Primalign HIV sequence database tool <a href="#pone.0001420-2">[12]</a>. In addition, ambiguous bases within the primers were used to represent the genetic diversity present at the primer binding sites.</p>


<h4>Comparative Sensitivities of HIV-1 VLs for successful PCR amplification</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The ability to successfully amplify and sequence HIV-1 RNA from plasma was dependent on VL. The frequency of positive amplifications of the four different fragments obtained in the 27 samples is shown in <a href="#pone-0001420-t002">Table 2</a>. The efficacy on the table is based on cDNA synthesis from the 3′end of viral RNA with primer, UNINEF 7′. In our initial strategy of amplifying two regions, a low efficiency of amplification of the 7.0-kb <em>pol-nef</em> fragment was observed in the samples with low VL. Therefore, the three-amplicon strategy was implemented for the samples that failed using the first approach. To measure the correlation between the VL and the efficacy of amplification, the PCR reactions were repeated at least 4 times with different dilutions to determine each as either positive or negative. The success rates for obtaining all amplicons for full-length were 100% in samples with a VL greater than 750,000 copies/ml, 75% in samples with a VL of 100,000–200,000 copies/ml, and 40% in samples with a VL of 10,000–100,000 copies/ml. Finally, there was a 76% success rate if the three-amplicon strategy were used for those with a VL of greater than 10,000 copies/ml.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Reverse transcription was also performed priming with VIF-VPUoutR1 for the samples with negative <em>vif-vpu</em> region amplification results. Approximately 60% of the previously negative samples were successfully amplified in the <em>pol-nef</em> and <em>gag-pol</em> region after priming with VIF-VPUoutR1, resulting in 90% efficiency for both regions for samples with a VL of greater than 10,000 copies/ml (data not shown).</p>


<h4>Analysis of Sequences of HIV-1 RNA</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The reverse-transcription and PCR methods were verified by directly sequencing the nested PCR products amplified in all three regions in 15 samples. The amplified material present in strong bands obtained with PCR was sufficient for good quality sequences in direct automated fluorescent sequencing. All the amplicons were sequenced separately and assembled for the nearly full-length HIV-1 genome (sequence data will be published separately). The resulting sequences were around 8500-bp (nts 769–9181 HXB-2) and contained the nearly entire HIV-1 coding region, but lacked the 3′end of the <em>nef</em> gene and U3 and R of LTR regions (nts 9181–9636 HXB-2). All of the reading frames were open without apparent insertion, deletion, or rearrangement.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>In the 2-amplicon and 3-amplicon strategies, there were regions of overlap between the adjacent amplicons. The overlapping regions of 797-bp and 321-bp were carefully analyzed to assemble the three fragments. Some overlapping regions displayed a small number of heterogeneous single-base substitutions, but the most common substitutions were G to A, which reflected the G to A hypermutation of one of the templates but not the other. To ensure that the sequence fragments were derived from the same genomic species, two criteria had to be met to represent a consensus sequence: the genetic distance had to be no greater than 2%, and the base differences had to be distributed evenly throughout the overlapping region. If these two criteria were met, a single sequence was generated; otherwise cDNA synthesis and PCR amplification were repeated.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The accuracy of direct sequencing of full-length virion RNA from HIV-1 relies on the ability to extract RNA from clinical materials, to conduct cDNA synthesis, and to amplify the cDNA successfully. The presence of an adequate number of intact RNA templates was essential for accurate and complete reverse transcription. An ultracentrifugation step to concentrate the viral particles before the extraction of RNA contributed to increased success especially for plasma samples with low VLs; a total of at least 250 virus copies (10,000 copies/ml×500 µl plasma÷60 µl RNA×3 µl used) had to be present for the procedure to be successful. Therefore the low abundance of viral RNA in some clinical samples remains a barrier to this application. However, most HIV-1 infected individuals have viral RNA titers that exceed 10,000 copies/ml in plasma if untreated or if they are on nonsuppressive antiretroviral therapy <a href="#pone.0001420-Mellors1">[13]</a>, <a href="#pone.0001420-Piatak1">[14]</a>, and thus this protocol can be employed for a broad range of plasma samples especially from infected individuals not under treatment, on suboptimal therapy, and in treatment failure.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Efficient cDNA synthesis is another important factor for successful RT-PCR. Increased efficiency of cDNA synthesis has been reported with a genetically engineered Moloney Murine Leukemia Virus (M-MLV) RT devoid of RNase H activity compared with unmodified M-MLV RT or AMV RT <a href="#pone.0001420-Villahermosa1">[15]</a>. SuperScript<sup>TM</sup> III RNase H<sup>−</sup> RT is M-MLV with such introduced point mutations. The manufacturer (Invitrogen) recommends 200 U of RT for sufficient cDNA synthesis; however increasing the input of RT to 400 U significantly improved the amplification results. It is probably because a significant amount of thermal-sensitive RT is degraded during the incubation. Since the original 200 U input is more cost-effective than 400 U, we recommend the use of 200 U RT for samples with VL &gt;750,000 copies/ml.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>To synthesize full-length cDNA efficiently has been difficult to achieve presumably because the RT tends to be interrupted at regions of secondary structure in mRNA templates <a href="#pone.0001420-Klarmann1">[16]</a>, <a href="#pone.0001420-Harrison1">[17]</a>, resulting in the generation of truncated cDNA. Even though the M-MLV RNase H<sup>−</sup> enzyme appears to be the best at negotiating regions of secondary structure compared to other available <em>in vitro</em> RT <a href="#pone.0001420-Das1">[18]</a>, as the length of mRNA increases, the RT is more likely to encounter such regions, decreasing the chances of achieving the 5′end of template RNA. At 55°C some secondary structural elements begin to melt, and many existing protocols aim to perform reactions at increased temperatures with minimum interference with the activity of RT <a href="#pone.0001420-Malboeuf1">[19]</a>, <a href="#pone.0001420-Carninci1">[20]</a>. However, the half-life of SuperScript<sup>TM</sup> III RT significantly declines as temperature is raised: 220 min at 50°C and 30 min at 55°C. More importantly, elevated temperatures augment RNA breakdown; RNA is thermolabile and susceptible to divalent Mg<sup>2+</sup>-catalyzed degradation <a href="#pone.0001420-Das1">[18]</a>, <a href="#pone.0001420-AbouHaidar1">[21]</a>, <a href="#pone.0001420-Gerard1">[22]</a>. The RT requires divalent cations, and the reaction condition of MgCl<sub>2</sub> with SuperScript III is 6 mM. It was reported that the rate of RNA breakdown is directly proportional to Mg<sup>2+</sup> concentration between 1 and 10 mM at above 37°C, and is increased 1.75-fold for every 5°C increment in temperature increase above 40°C in the presence of 3 mM Mg<sup>2+</sup> <a href="#pone.0001420-AbouHaidar1">[21]</a>, <a href="#pone.0001420-Gerard1">[22]</a>. Therefore, although high temperature contributes to resolving secondary structures of RNA, chances for a full-length mRNA template are much reduced. For those reasons, reverse transcription was carried out at 50°C in our protocol.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>A high DNA recombination frequency during long RT-PCR has been previously reported <a href="#pone.0001420-Yu1">[23]</a>, <a href="#pone.0001420-Fang1">[24]</a>. Recombination frequencies resulting from the combined procedure of long RT-PCR are especially higher than long PCR itself or RT-PCR of shorter molecules <a href="#pone.0001420-Yu1">[23]</a>–<a href="#pone.0001420-Rozak1">[26]</a>. For accurate characterization of viral RNA genomes derived from HIV-1 infected plasma, it is necessary to minimize artifactual recombination between quasispecies during RT-PCR. High recombination frequencies are assumed to be related to the presence of partial cDNA that results from incomplete cDNA synthesis as well as partial RNA templates caused by degradation. According to the previous studies <a href="#pone.0001420-Yu1">[23]</a>, <a href="#pone.0001420-Fang1">[24]</a>, the following conditions were recommended in order to decrease recombination frequencies during long RT-PCR: the minimum quantity of input HIV-1 RNA, use of mixtures of DNA polymerase and proofreading enzymes, increased incubation time for cDNA synthesis, and increased extension time and a minimum number of cycles for PCR. The last two are associated with achieving complete strand synthesis to avoid PCR-mediated recombination. A 120 min RT incubation time and an extension time of 1.5–2 min/kb of template DNA were suggested as suitable to minimize recombination in most long RT-PCR <a href="#pone.0001420-Fang1">[24]</a>. We employed a 120 min RT incubation time in this protocol after multiple experiments, and since the excess extension time may compromise yield and specificity, the optimal PCR extension time was determined based on template lengths and extension rates of the DNA polymerase.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>When the nearly full-length genome sequences were assembled, the overlapping regions displayed a small number of heterogeneous single-base substitutions. Most of them were probably because heterogeneous quasispecies exist in HIV-1 infected plasma samples. Any HIV sample contains a population of related but genetically diverse variants due to the high misincorporation rate of the HIV-1 RT <a href="#pone.0001420-Coffin1">[8]</a>, <a href="#pone.0001420-Eigen1">[27]</a>, <a href="#pone.0001420-Meyerhans1">[28]</a>. Heterogeneous bases could also result from mutations created during the <em>in vitro</em> reverse transcription step; the M-MLV RT is error prone for incorporation of a wrong nucleotide because of the lack of 3′→5′ proofreading exonuclease activity <a href="#pone.0001420-Malboeuf1">[19]</a>, <a href="#pone.0001420-Ji1">[29]</a>–<a href="#pone.0001420-Malet1">[31]</a>; mutations created during the reverse transcription are carried on to the final products to be analyzed, which may interfere with accurate analysis of viral RNA. The actual empirical error rates of the SuperScript<sup>TM</sup> III RT as used in our assay are unknown. The G to A single-base substitutions were the most commonly observed among the heterogeneous bases in the overlapping regions. Recent reports showed that a cytosine deaminase APOBEC3G (aplolipoprotein B mRNA-editing enzyme, catalytic polypeptide-like 3G), which is packaged in HIV-1 virions, induces G to A hypermutation to a nascent reverse transcript of HIV-1, which contributes in part to the innate antiviral activity <a href="#pone.0001420-Mangeat1">[32]</a>–<a href="#pone.0001420-Harris1">[35]</a>. Even though the sequence is not from an infectious virus, the hypermutated sequences can still be used for molecular subtyping.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>The evidence of dual infections and recombination between subtypes<a href="#pone.0001420-McCutchan1">[36]</a>, <a href="#pone.0001420-Robertson1">[37]</a> suggest that amplification of fewer fragments is desirable; while the two-amplicon strategy had an advantage of increasing the possibility of amplifying the single species of HIV-1, the three-amplicon strategy was more sensitive for amplification of samples with low VLs. In a study with SuperScript™ II MMLV RT, it was indicated that cDNA synthesis continued to 6-kb and decreased afterwards <a href="#pone.0001420-Swift1">[38]</a>. The 5′ end of an mRNA is generally represented to a lesser extent than the 3′ end in cDNA. This is in accordance with the increased sensitivity of amplification after cDNA synthesis with primer, VIF-VPUoutR1, compared to UNINEF 7′ in our study. The limiting dilution method for PCR was used because it is most likely to pick the predominant strain of the viral population. The relatively long overlap regions (797- and 321-bp) between the amplicons increase the likelihood of identifying and constructing full-length amplicons of the single molecular species.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Previous study has showed a method for the amplification and sequencing of nearly full-length HIV-1 genomes from RNA<a href="#pone.0001420-Rousseau1">[39]</a>. However, their primers were designed based on the sequences in their South African cohorts and amplify HIV-1 subtype C preferentially. The primers presented in this paper were designed to be able to pick up most of the strains from the group M HIV-1 available in the HIV sequence database <a href="#pone.0001420-1">[2]</a>. Additionally their PCR products need to be cloned with a commercially available cloning kit in comparison with our protocol with direct-PCR sequencing. Addition of a cloning step is more time-consuming and can introduce the possibility of bias. Therefore, the methods described here provide a more comprehensive protocol of sequencing nearly full-length HIV-1 RNA compared to currently described methods.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Since this protocol is still not applicable to a low VL range of HIV-1 infected plasma and serum specimens, further optimization of the protocol in the future will be necessary. Recent publications reported a number of reagents that are supposed to facilitate the disruption of RNA secondary structure and to improve an ability to achieve full-length cDNA synthesis, including EDTA to eliminate free Mg<sup>2+</sup>, addition of DMSO, RNA chaperone, formamide and glycerol, and pretreatment of the RNA with methylmercury hydrozide <a href="#pone.0001420-Das1">[18]</a>, <a href="#pone.0001420-Carninci1">[20]</a>, <a href="#pone.0001420-Gerard1">[22]</a>, <a href="#pone.0001420-Piche1">[40]</a>, <a href="#pone.0001420-Chester1">[41]</a>. Recently it has also been documented that an incorporation of a proofreading function to RT significantly increased the full-length content of cDNA up to 15-kb <a href="#pone.0001420-Hawkins1">[42]</a>. However, the effectiveness of these conditions has never been tested in controlled experiments harboring defined structural features capable of blocking RT activity, therefore supplementary testing will be required before implementing them.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>In summary, a new protocol for sequencing nearly full-length HIV-1 RNA genomes directly form plasma using long RT-PCR was described in this study. The success of the procedure was sensitive to VL but was routinely successful for VL greater than 10,000 copies/ml. The method used in this study overcomes some of the practical difficulties found when trying to sequence nearly full-genomic HIV-1 from virion RNA. Its importance is two-fold: first, it confers the ability to sequence the HIV-1 genomes of newly circulating viruses; second, it can be used to study the molecular epidemiology of HIV-1 in regions of the world where only serum or plasma is available.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001420"></div><a name="pone.0001420.s001" id="pone.0001420.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001420.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001420.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.14 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Linda Jagodzinski for her critical technical advice.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YN JC. Performed the experiments: YN LE. Analyzed the data: YN. Contributed reagents/materials/analysis tools: AS. Wrote the paper: YN. Other: Supervised the project as PI on the grant: WB. Initial PI on the grant: FC. Organized the research: FC. Contributed the viral load data of the samples: NC. Coordinated the laboratory: JC. Edited the manuscript: JC. Supervised the experiments: JC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001420-Ratner1" id="pone.0001420-Ratner1"></a>Ratner L, Haseltine W, Patarca R, Livak KJ, Starcich B, et al.  (1985) Complete nucleotide sequence of the AIDS virus, HTLV-III. Nature  313: 277–284.  <ul class="find" data-citedArticleID="1040166" data-doi="10.1038/313277a0"><li><a href="http://dx.doi.org/10.1038/313277a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+nucleotide+sequence+of+the+AIDS+virus%2C+HTLV-III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+nucleotide+sequence+of+the+AIDS+virus%2C+HTLV-III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001420-1" id="pone.0001420-1"></a>HIV Sequence Database Web Site: Theoretical biology and biophysics group, Los Alamos National Laboratory.  Available: <a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a> Accessed September 2006.  <ul class="find" data-citedArticleID="1040106"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=HIV+Sequence+Database+Web+Site%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory.&amp;auth=&amp;atitle=HIV+Sequence+Database+Web+Site%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+Sequence+Database+Web+Site%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+Sequence+Database+Web+Site%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001420-Salminen1" id="pone.0001420-Salminen1"></a>Salminen MO, Koch C, Sanders-Buell E, Ehrenberg PK, Michael NL, et al.  (1995) Recovery of virtually full-length HIV-1 provirus of diverse subtypes from primary virus cultures using the polymerase chain reaction. Virology  213: 80–86.  <ul class="find" data-citedArticleID="1040174" data-doi="10.1006/viro.1995.1548"><li><a href="http://dx.doi.org/10.1006/viro.1995.1548" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recovery+of+virtually+full-length+HIV-1+provirus+of+diverse+subtypes+from+primary+virus+cultures+using+the+polymerase+chain+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recovery+of+virtually+full-length+HIV-1+provirus+of+diverse+subtypes+from+primary+virus+cultures+using+the+polymerase+chain+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001420-Carr1" id="pone.0001420-Carr1"></a>Carr JK, Laukkanen T, Salminen MO, Albert J, Alaeus A, et al.  (1999) Characterization of subtype A HIV-1 from Africa by full genome sequencing. Aids  13: 1819–1826.  <ul class="find" data-citedArticleID="1040118" data-doi="10.1097/00002030-199910010-00003"><li><a href="http://dx.doi.org/10.1097/00002030-199910010-00003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+subtype+A+HIV-1+from+Africa+by+full+genome+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+subtype+A+HIV-1+from+Africa+by+full+genome+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001420-Arroyo1" id="pone.0001420-Arroyo1"></a>Arroyo MA, Hoelscher M, Sanders-Buell E, Herbinger KH, Samky E, et al.  (2004) HIV type 1 subtypes among blood donors in the Mbeya region of southwest Tanzania. AIDS Res Hum Retroviruses  20: 895–901.  <ul class="find" data-citedArticleID="1040114" data-doi="10.1089/0889222041725235"><li><a href="http://dx.doi.org/10.1089/0889222041725235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+type+1+subtypes+among+blood+donors+in+the+Mbeya+region+of+southwest+Tanzania." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+type+1+subtypes+among+blood+donors+in+the+Mbeya+region+of+southwest+Tanzania.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001420-Wei1" id="pone.0001420-Wei1"></a>Wei X, Ghosh SK, Taylor ME, Johnson VA, Emini EA, et al.  (1995) Viral dynamics in human immunodeficiency virus type 1 infection. Nature  373: 117–122.  <ul class="find" data-citedArticleID="1040184" data-doi="10.1038/373117a0"><li><a href="http://dx.doi.org/10.1038/373117a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+dynamics+in+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+dynamics+in+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001420-Chun1" id="pone.0001420-Chun1"></a>Chun TW, Carruth L, Finzi D, Shen X, DiGiuseppe JA, et al.  (1997) Quantification of latent tissue reservoirs and total body viral load in HIV-1 infection. Nature  387: 183–188.  <ul class="find" data-citedArticleID="1040122" data-doi="10.1038/387183a0"><li><a href="http://dx.doi.org/10.1038/387183a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantification+of+latent+tissue+reservoirs+and+total+body+viral+load+in+HIV-1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantification+of+latent+tissue+reservoirs+and+total+body+viral+load+in+HIV-1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001420-Coffin1" id="pone.0001420-Coffin1"></a>Coffin JM (1995) HIV population dynamics in vivo: implications for genetic variation, pathogenesis, and therapy. Science  267: 483–489.  <ul class="find" data-citedArticleID="1040124" data-doi="10.1126/science.7824947"><li><a href="http://dx.doi.org/10.1126/science.7824947" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+population+dynamics+in+vivo%3A+implications+for+genetic+variation%2C+pathogenesis%2C+and+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+population+dynamics+in+vivo%3A+implications+for+genetic+variation%2C+pathogenesis%2C+and+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001420-Simmonds1" id="pone.0001420-Simmonds1"></a>Simmonds P, Zhang LQ, McOmish F, Balfe P, Ludlam CA, et al.  (1991) Discontinuous sequence change of human immunodeficiency virus (HIV) type 1 env sequences in plasma viral and lymphocyte-associated proviral populations in vivo: implications for models of HIV pathogenesis. J Virol  65: 6266–6276.  <ul class="find" data-citedArticleID="1040176"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Discontinuous+sequence+change+of+human+immunodeficiency+virus+%28HIV%29+type+1+env+sequences+in+plasma+viral+and+lymphocyte-associated+proviral+populations+in+vivo%3A+implications+for+models+of+HIV+pathogenesis.&amp;auth=&amp;atitle=Discontinuous+sequence+change+of+human+immunodeficiency+virus+%28HIV%29+type+1+env+sequences+in+plasma+viral+and+lymphocyte-associated+proviral+populations+in+vivo%3A+implications+for+models+of+HIV+pathogenesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discontinuous+sequence+change+of+human+immunodeficiency+virus+%28HIV%29+type+1+env+sequences+in+plasma+viral+and+lymphocyte-associated+proviral+populations+in+vivo%3A+implications+for+models+of+HIV+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discontinuous+sequence+change+of+human+immunodeficiency+virus+%28HIV%29+type+1+env+sequences+in+plasma+viral+and+lymphocyte-associated+proviral+populations+in+vivo%3A+implications+for+models+of+HIV+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001420-Arens1" id="pone.0001420-Arens1"></a>Arens M (1999) Methods for subtyping and molecular comparison of human viral genomes. Clin Microbiol Rev  12: 612–626.  <ul class="find" data-citedArticleID="1040112"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Methods+for+subtyping+and+molecular+comparison+of+human+viral+genomes.&amp;auth=&amp;atitle=Methods+for+subtyping+and+molecular+comparison+of+human+viral+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methods+for+subtyping+and+molecular+comparison+of+human+viral+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methods+for+subtyping+and+molecular+comparison+of+human+viral+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001420-VanLaethem1" id="pone.0001420-VanLaethem1"></a>Van Laethem K, Schrooten Y, Dedecker S, Van Heeswijck L, Deforche K, et al.  (2006) A genotypic assay for the amplification and sequencing of gag and protease from diverse human immunodeficiency virus type 1 group M subtypes. J Virol Methods  132: 181–186.  <ul class="find" data-citedArticleID="1040180" data-doi="10.1016/j.jviromet.2005.10.008"><li><a href="http://dx.doi.org/10.1016/j.jviromet.2005.10.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genotypic+assay+for+the+amplification+and+sequencing+of+gag+and+protease+from+diverse+human+immunodeficiency+virus+type+1+group+M+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genotypic+assay+for+the+amplification+and+sequencing+of+gag+and+protease+from+diverse+human+immunodeficiency+virus+type+1+group+M+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001420-2" id="pone.0001420-2"></a>Primalign HIV Sequence Database tool: Theoretical biology and biophysics group, Los Alamos National Laboratory.  Available: <a href="http://hiv-web.lanl.gov/content/hiv-db/PRIMALIGN/PRIME.html">http://hiv-web.lanl.gov/content/hiv-db/P​RIMALIGN/PRIME.html</a> Accessed September 2006.  <ul class="find" data-citedArticleID="1040108"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Primalign+HIV+Sequence+Database+tool%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory.&amp;auth=&amp;atitle=Primalign+HIV+Sequence+Database+tool%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primalign+HIV+Sequence+Database+tool%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primalign+HIV+Sequence+Database+tool%3A+Theoretical+biology+and+biophysics+group%2C+Los+Alamos+National+Laboratory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001420-Mellors1" id="pone.0001420-Mellors1"></a>Mellors JW, Kingsley LA, Rinaldo CR Jr, Todd JA, Hoo BS, et al.  (1995) Quantitation of HIV-1 RNA in plasma predicts outcome after seroconversion. Ann Intern Med  122: 573–579.  <ul class="find" data-citedArticleID="1040156" data-doi="10.7326/0003-4819-122-8-199504150-00003"><li><a href="http://dx.doi.org/10.7326/0003-4819-122-8-199504150-00003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitation+of+HIV-1+RNA+in+plasma+predicts+outcome+after+seroconversion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitation+of+HIV-1+RNA+in+plasma+predicts+outcome+after+seroconversion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001420-Piatak1" id="pone.0001420-Piatak1"></a>Piatak M Jr, Saag MS, Yang LC, Clark SJ, Kappes JC, et al.  (1993) High levels of HIV-1 in plasma during all stages of infection determined by competitive PCR. Science  259: 1749–1754.  <ul class="find" data-citedArticleID="1040160" data-doi="10.1126/science.8096089"><li><a href="http://dx.doi.org/10.1126/science.8096089" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+levels+of+HIV-1+in+plasma+during+all+stages+of+infection+determined+by+competitive+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+levels+of+HIV-1+in+plasma+during+all+stages+of+infection+determined+by+competitive+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001420-Villahermosa1" id="pone.0001420-Villahermosa1"></a>Villahermosa ML, Thomson M, Vazquez de Parga E, Cuevas MT, Contreras G, et al.  (2000) Improved conditions for extraction and amplification of human immunodeficiency virus type 1 RNA from plasma samples with low viral load. J Hum Virol  3: 27–34.  <ul class="find" data-citedArticleID="1040182"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Improved+conditions+for+extraction+and+amplification+of+human+immunodeficiency+virus+type+1+RNA+from+plasma+samples+with+low+viral+load.&amp;auth=&amp;atitle=Improved+conditions+for+extraction+and+amplification+of+human+immunodeficiency+virus+type+1+RNA+from+plasma+samples+with+low+viral+load." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+conditions+for+extraction+and+amplification+of+human+immunodeficiency+virus+type+1+RNA+from+plasma+samples+with+low+viral+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+conditions+for+extraction+and+amplification+of+human+immunodeficiency+virus+type+1+RNA+from+plasma+samples+with+low+viral+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001420-Klarmann1" id="pone.0001420-Klarmann1"></a>Klarmann GJ, Schauber CA, Preston BD (1993) Template-directed pausing of DNA synthesis by HIV-1 reverse transcriptase during polymerization of HIV-1 sequences in vitro. J Biol Chem  268: 9793–9802.  <ul class="find" data-citedArticleID="1040144"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Template-directed+pausing+of+DNA+synthesis+by+HIV-1+reverse+transcriptase+during+polymerization+of+HIV-1+sequences+in+vitro.&amp;auth=&amp;atitle=Template-directed+pausing+of+DNA+synthesis+by+HIV-1+reverse+transcriptase+during+polymerization+of+HIV-1+sequences+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Template-directed+pausing+of+DNA+synthesis+by+HIV-1+reverse+transcriptase+during+polymerization+of+HIV-1+sequences+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Template-directed+pausing+of+DNA+synthesis+by+HIV-1+reverse+transcriptase+during+polymerization+of+HIV-1+sequences+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001420-Harrison1" id="pone.0001420-Harrison1"></a>Harrison GP, Mayo MS, Hunter E, Lever AM (1998) Pausing of reverse transcriptase on retroviral RNA templates is influenced by secondary structures both 5′ and 3′ of the catalytic site. Nucleic Acids Res  26: 3433–3442.  <ul class="find" data-citedArticleID="1040136" data-doi="10.1093/nar/26.14.3433"><li><a href="http://dx.doi.org/10.1093/nar/26.14.3433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pausing+of+reverse+transcriptase+on+retroviral+RNA+templates+is+influenced+by+secondary+structures+both+5%E2%80%B2+and+3%E2%80%B2+of+the+catalytic+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pausing+of+reverse+transcriptase+on+retroviral+RNA+templates+is+influenced+by+secondary+structures+both+5%E2%80%B2+and+3%E2%80%B2+of+the+catalytic+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001420-Das1" id="pone.0001420-Das1"></a>Das M, Harvey I, Chu LL, Sinha M, Pelletier J (2001) Full-length cDNAs: more than just reaching the ends. Physiol Genomics  6: 57–80.  <ul class="find" data-citedArticleID="1040126"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Full-length+cDNAs%3A+more+than+just+reaching+the+ends.&amp;auth=&amp;atitle=Full-length+cDNAs%3A+more+than+just+reaching+the+ends." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full-length+cDNAs%3A+more+than+just+reaching+the+ends." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full-length+cDNAs%3A+more+than+just+reaching+the+ends.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001420-Malboeuf1" id="pone.0001420-Malboeuf1"></a>Malboeuf CM, Isaacs SJ, Tran NH, Kim B (2001) Thermal effects on reverse transcription: improvement of accuracy and processivity in cDNA synthesis. Biotechniques  30: 1074–1078, 1080, 1082, passim.  <ul class="find" data-citedArticleID="1040148"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Thermal+effects+on+reverse+transcription%3A+improvement+of+accuracy+and+processivity+in+cDNA+synthesis.&amp;auth=&amp;atitle=Thermal+effects+on+reverse+transcription%3A+improvement+of+accuracy+and+processivity+in+cDNA+synthesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermal+effects+on+reverse+transcription%3A+improvement+of+accuracy+and+processivity+in+cDNA+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermal+effects+on+reverse+transcription%3A+improvement+of+accuracy+and+processivity+in+cDNA+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001420-Carninci1" id="pone.0001420-Carninci1"></a>Carninci P, Nishiyama Y, Westover A, Itoh M, Nagaoka S, et al.  (1998) Thermostabilization and thermoactivation of thermolabile enzymes by trehalose and its application for the synthesis of full length cDNA. Proc Natl Acad Sci U S A  95: 520–524.  <ul class="find" data-citedArticleID="1040116" data-doi="10.1073/pnas.95.2.520"><li><a href="http://dx.doi.org/10.1073/pnas.95.2.520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermostabilization+and+thermoactivation+of+thermolabile+enzymes+by+trehalose+and+its+application+for+the+synthesis+of+full+length+cDNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermostabilization+and+thermoactivation+of+thermolabile+enzymes+by+trehalose+and+its+application+for+the+synthesis+of+full+length+cDNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001420-AbouHaidar1" id="pone.0001420-AbouHaidar1"></a>AbouHaidar MG, Ivanov IG (1999) Non-enzymatic RNA hydrolysis promoted by the combined catalytic activity of buffers and magnesium ions. Z Naturforsch [C]  54: 542–548.  <ul class="find" data-citedArticleID="1040110"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Non-enzymatic+RNA+hydrolysis+promoted+by+the+combined+catalytic+activity+of+buffers+and+magnesium+ions.&amp;auth=&amp;atitle=Non-enzymatic+RNA+hydrolysis+promoted+by+the+combined+catalytic+activity+of+buffers+and+magnesium+ions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-enzymatic+RNA+hydrolysis+promoted+by+the+combined+catalytic+activity+of+buffers+and+magnesium+ions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-enzymatic+RNA+hydrolysis+promoted+by+the+combined+catalytic+activity+of+buffers+and+magnesium+ions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001420-Gerard1" id="pone.0001420-Gerard1"></a>Gerard GF, Collins S, Smith MD (2002) Excess dNTPs minimize RNA hydrolysis during reverse transcription. Biotechniques  33: 984, 986, 988 passim.  <ul class="find" data-citedArticleID="1040132"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Excess+dNTPs+minimize+RNA+hydrolysis+during+reverse+transcription.&amp;auth=&amp;atitle=Excess+dNTPs+minimize+RNA+hydrolysis+during+reverse+transcription." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Excess+dNTPs+minimize+RNA+hydrolysis+during+reverse+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Excess+dNTPs+minimize+RNA+hydrolysis+during+reverse+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001420-Yu1" id="pone.0001420-Yu1"></a>Yu W, Rusterholtz KJ, Krummel AT, Lehman N (2006) Detection of high levels of recombination generated during PCR amplification of RNA templates. Biotechniques  40: 499–507.  <ul class="find" data-citedArticleID="1040186" data-doi="10.2144/000112124"><li><a href="http://dx.doi.org/10.2144/000112124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+high+levels+of+recombination+generated+during+PCR+amplification+of+RNA+templates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+high+levels+of+recombination+generated+during+PCR+amplification+of+RNA+templates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001420-Fang1" id="pone.0001420-Fang1"></a>Fang G, Zhu G, Burger H, Keithly JS, Weiser B (1998) Minimizing DNA recombination during long RT-PCR. J Virol Methods  76: 139–148.  <ul class="find" data-citedArticleID="1040130" data-doi="10.1016/s0166-0934(98)00133-5"><li><a href="http://dx.doi.org/10.1016/s0166-0934(98)00133-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Minimizing+DNA+recombination+during+long+RT-PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Minimizing+DNA+recombination+during+long+RT-PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001420-Judo1" id="pone.0001420-Judo1"></a>Judo MS, Wedel AB, Wilson C (1998) Stimulation and suppression of PCR-mediated recombination. Nucleic Acids Res  26: 1819–1825.  <ul class="find" data-citedArticleID="1040142" data-doi="10.1093/nar/26.7.1819"><li><a href="http://dx.doi.org/10.1093/nar/26.7.1819" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stimulation+and+suppression+of+PCR-mediated+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stimulation+and+suppression+of+PCR-mediated+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001420-Rozak1" id="pone.0001420-Rozak1"></a>Rozak DA, Bryan PN (2005) Offset recombinant PCR: a simple but effective method for shuffling compact heterologous domains. Nucleic Acids Res  33: e82.  <ul class="find" data-citedArticleID="1040172" data-doi="10.1093/nar/gni081"><li><a href="http://dx.doi.org/10.1093/nar/gni081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Offset+recombinant+PCR%3A+a+simple+but+effective+method+for+shuffling+compact+heterologous+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Offset+recombinant+PCR%3A+a+simple+but+effective+method+for+shuffling+compact+heterologous+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001420-Eigen1" id="pone.0001420-Eigen1"></a>Eigen M (1993) Viral quasispecies. Sci Am  269: 42–49.  <ul class="find" data-citedArticleID="1040128" data-doi="10.1038/scientificamerican0793-42"><li><a href="http://dx.doi.org/10.1038/scientificamerican0793-42" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+quasispecies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+quasispecies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001420-Meyerhans1" id="pone.0001420-Meyerhans1"></a>Meyerhans A, Cheynier R, Albert J, Seth M, Kwok S, et al.  (1989) Temporal fluctuations in HIV quasispecies in vivo are not reflected by sequential HIV isolations. Cell  58: 901–910.  <ul class="find" data-citedArticleID="1040158" data-doi="10.1016/0092-8674(89)90942-2"><li><a href="http://dx.doi.org/10.1016/0092-8674(89)90942-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal+fluctuations+in+HIV+quasispecies+in+vivo+are+not+reflected+by+sequential+HIV+isolations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal+fluctuations+in+HIV+quasispecies+in+vivo+are+not+reflected+by+sequential+HIV+isolations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001420-Ji1" id="pone.0001420-Ji1"></a>Ji JP, Loeb LA (1992) Fidelity of HIV-1 reverse transcriptase copying RNA in vitro. Biochemistry  31: 954–958.  <ul class="find" data-citedArticleID="1040140" data-doi="10.1021/bi00119a002"><li><a href="http://dx.doi.org/10.1021/bi00119a002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fidelity+of+HIV-1+reverse+transcriptase+copying+RNA+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fidelity+of+HIV-1+reverse+transcriptase+copying+RNA+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001420-Preston1" id="pone.0001420-Preston1"></a>Preston BD, Dougherty JP (1996) Mechanisms of retroviral mutation. Trends Microbiol  4: 16–21.  <ul class="find" data-citedArticleID="1040164" data-doi="10.1016/0966-842x(96)81500-9"><li><a href="http://dx.doi.org/10.1016/0966-842x(96)81500-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+retroviral+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+retroviral+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001420-Malet1" id="pone.0001420-Malet1"></a>Malet I, Belnard M, Agut H, Cahour A (2003) From RNA to quasispecies: a DNA polymerase with proofreading activity is highly recommended for accurate assessment of viral diversity. J Virol Methods  109: 161–170.  <ul class="find" data-citedArticleID="1040150" data-doi="10.1016/s0166-0934(03)00067-3"><li><a href="http://dx.doi.org/10.1016/s0166-0934(03)00067-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+RNA+to+quasispecies%3A+a+DNA+polymerase+with+proofreading+activity+is+highly+recommended+for+accurate+assessment+of+viral+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+RNA+to+quasispecies%3A+a+DNA+polymerase+with+proofreading+activity+is+highly+recommended+for+accurate+assessment+of+viral+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001420-Mangeat1" id="pone.0001420-Mangeat1"></a>Mangeat B, Turelli P, Caron G, Friedli M, Perrin L, et al.  (2003) Broad antiretroviral defence by human APOBEC3G through lethal editing of nascent reverse transcripts. Nature  424: 99–103.  <ul class="find" data-citedArticleID="1040152" data-doi="10.1038/nature01709"><li><a href="http://dx.doi.org/10.1038/nature01709" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad+antiretroviral+defence+by+human+APOBEC3G+through+lethal+editing+of+nascent+reverse+transcripts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad+antiretroviral+defence+by+human+APOBEC3G+through+lethal+editing+of+nascent+reverse+transcripts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001420-Lecossier1" id="pone.0001420-Lecossier1"></a>Lecossier D, Bouchonnet F, Clavel F, Hance AJ (2003) Hypermutation of HIV-1 DNA in the absence of the Vif protein. Science  300: 1112.  <ul class="find" data-citedArticleID="1040146" data-doi="10.1126/science.1083338"><li><a href="http://dx.doi.org/10.1126/science.1083338" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hypermutation+of+HIV-1+DNA+in+the+absence+of+the+Vif+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hypermutation+of+HIV-1+DNA+in+the+absence+of+the+Vif+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001420-Zhang1" id="pone.0001420-Zhang1"></a>Zhang H, Yang B, Pomerantz RJ, Zhang C, Arunachalam SC, et al.  (2003) The cytidine deaminase CEM15 induces hypermutation in newly synthesized HIV-1 DNA. Nature  424: 94–98.  <ul class="find" data-citedArticleID="1040188" data-doi="10.1038/nature01707"><li><a href="http://dx.doi.org/10.1038/nature01707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cytidine+deaminase+CEM15+induces+hypermutation+in+newly+synthesized+HIV-1+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cytidine+deaminase+CEM15+induces+hypermutation+in+newly+synthesized+HIV-1+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001420-Harris1" id="pone.0001420-Harris1"></a>Harris RS, Bishop KN, Sheehy AM, Craig HM, Petersen-Mahrt SK, et al.  (2003) DNA deamination mediates innate immunity to retroviral infection. Cell  113: 803–809.  <ul class="find" data-citedArticleID="1040134" data-doi="10.1016/s0092-8674(03)00423-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00423-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+deamination+mediates+innate+immunity+to+retroviral+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+deamination+mediates+innate+immunity+to+retroviral+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001420-McCutchan1" id="pone.0001420-McCutchan1"></a>McCutchan FE (2006) Global epidemiology of HIV. J Med Virol  78: Suppl 1S7–S12.  <ul class="find" data-citedArticleID="1040154" data-doi="10.1002/jmv.20599"><li><a href="http://dx.doi.org/10.1002/jmv.20599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+epidemiology+of+HIV." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+epidemiology+of+HIV.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001420-Robertson1" id="pone.0001420-Robertson1"></a>Robertson DL, Sharp PM, McCutchan FE, Hahn BH (1995) Recombination in HIV-1. Nature  374: 124–126.  <ul class="find" data-citedArticleID="1040168" data-doi="10.1038/374124b0"><li><a href="http://dx.doi.org/10.1038/374124b0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+in+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+in+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001420-Swift1" id="pone.0001420-Swift1"></a>Swift GH, Peyton MJ, MacDonald RJ (2000) Assessment of RNA quality by semi-quantitative RT-PCR of multiple regions of a long ubiquitous mRNA. Biotechniques  28: 524, 526, 528, 530–521.  <ul class="find" data-citedArticleID="1040178"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Assessment+of+RNA+quality+by+semi-quantitative+RT-PCR+of+multiple+regions+of+a+long+ubiquitous+mRNA.&amp;auth=&amp;atitle=Assessment+of+RNA+quality+by+semi-quantitative+RT-PCR+of+multiple+regions+of+a+long+ubiquitous+mRNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+RNA+quality+by+semi-quantitative+RT-PCR+of+multiple+regions+of+a+long+ubiquitous+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+RNA+quality+by+semi-quantitative+RT-PCR+of+multiple+regions+of+a+long+ubiquitous+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001420-Rousseau1" id="pone.0001420-Rousseau1"></a>Rousseau CM, Birditt BA, McKay AR, Stoddard JN, Lee TC, et al.  (2006) Large-scale amplification, cloning and sequencing of near full-length HIV-1 subtype C genomes. J Virol Methods  136: 118–125.  <ul class="find" data-citedArticleID="1040170" data-doi="10.1016/j.jviromet.2006.04.009"><li><a href="http://dx.doi.org/10.1016/j.jviromet.2006.04.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+amplification%2C+cloning+and+sequencing+of+near+full-length+HIV-1+subtype+C+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+amplification%2C+cloning+and+sequencing+of+near+full-length+HIV-1+subtype+C+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001420-Piche1" id="pone.0001420-Piche1"></a>Piche C, Schernthaner JP (2005) Optimization of in vitro transcription and full-length cDNA synthesis using the T4 bacteriophage gene 32 protein. J Biomol Tech  16: 239–247.  <ul class="find" data-citedArticleID="1040162"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Optimization+of+in+vitro+transcription+and+full-length+cDNA+synthesis+using+the+T4+bacteriophage+gene+32+protein.&amp;auth=&amp;atitle=Optimization+of+in+vitro+transcription+and+full-length+cDNA+synthesis+using+the+T4+bacteriophage+gene+32+protein." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimization+of+in+vitro+transcription+and+full-length+cDNA+synthesis+using+the+T4+bacteriophage+gene+32+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimization+of+in+vitro+transcription+and+full-length+cDNA+synthesis+using+the+T4+bacteriophage+gene+32+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001420-Chester1" id="pone.0001420-Chester1"></a>Chester N, Marshak DR (1993) Dimethyl sulfoxide-mediated primer Tm reduction: a method for analyzing the role of renaturation temperature in the polymerase chain reaction. Anal Biochem  209: 284–290.  <ul class="find" data-citedArticleID="1040120" data-doi="10.1006/abio.1993.1121"><li><a href="http://dx.doi.org/10.1006/abio.1993.1121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dimethyl+sulfoxide-mediated+primer+Tm+reduction%3A+a+method+for+analyzing+the+role+of+renaturation+temperature+in+the+polymerase+chain+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dimethyl+sulfoxide-mediated+primer+Tm+reduction%3A+a+method+for+analyzing+the+role+of+renaturation+temperature+in+the+polymerase+chain+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001420-Hawkins1" id="pone.0001420-Hawkins1"></a>Hawkins PR, Jin P, Fu GK (2003) Full-length cDNA synthesis for long-distance RT-PCR of large mRNA transcripts. Biotechniques  34: 768–773.768-770, 772-773  <ul class="find" data-citedArticleID="1040138"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Full-length+cDNA+synthesis+for+long-distance+RT-PCR+of+large+mRNA+transcripts.&amp;auth=&amp;atitle=Full-length+cDNA+synthesis+for+long-distance+RT-PCR+of+large+mRNA+transcripts." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full-length+cDNA+synthesis+for+long-distance+RT-PCR+of+large+mRNA+transcripts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full-length+cDNA+synthesis+for+long-distance+RT-PCR+of+large+mRNA+transcripts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.XML" value="80272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.PDF" value="174084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.g001.PNG_L" value="1240275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.g001.PNG_M" value="37193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.g001.PNG_S" value="7173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.g001.TIF" value="2598628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.g001.PNG_I" value="13386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t001.PNG_L" value="166151"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t001.PNG_M" value="101208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t001.PNG_S" value="10560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t001.TIF" value="742442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t001.PNG_I" value="31178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t002.PNG_L" value="91096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t002.PNG_M" value="52120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t002.PNG_S" value="8758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t002.TIF" value="411270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.t002.PNG_I" value="16026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001420.s001.DOC" value="120320"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001420&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001420" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001420&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001420&volume=&issue=&title=Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma&author_name=Yuka%20Nadai%2C%20Lindsay%20M.%20Eyzaguirre%2C%20Niel%20T.%20Constantine%2C%20Anne%20M.%20Sill%2C%20Farley%20Cleghorn%2C%20William%20A.%20Blattner%2C%20Jean%20K.%20Carr&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001420" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001420" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001420" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001420&amp;t=Protocol%20for%20Nearly%20Full-Length%20Sequencing%20of%20HIV-1%20RNA%20from%20Plasma" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001420&title=Protocol%20for%20Nearly%20Full-Length%20Sequencing%20of%20HIV-1%20RNA%20from%20Plasma&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001420&amp;title=Protocol%20for%20Nearly%20Full-Length%20Sequencing%20of%20HIV-1%20RNA%20from%20Plasma" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001420" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001420&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Protocol for Nearly Full-Length Sequencing of HIV-1 RNA from Plasma';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001420';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Protocol%20for%20Nearly%20Full-Length%20Sequencing%20of%20HIV-1%20RNA%20from%20Plasma http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001420" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001420" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25670"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="25670"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25670"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="25670"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+sequencing%22" title="Search for articles in the subject area:'RNA sequencing'"><div class="flagText">RNA sequencing</div></a>
              <div data-categoryid="19765" data-articleid="25670"
                   data-categoryname="RNA sequencing"
                   class="flagImage" title="Flag 'RNA sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+synthesis%22" title="Search for articles in the subject area:'RNA synthesis'"><div class="flagText">RNA synthesis</div></a>
              <div data-categoryid="32935" data-articleid="25670"
                   data-categoryname="RNA synthesis"
                   class="flagImage" title="Flag 'RNA synthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="25670"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22cDNA+synthesis%22" title="Search for articles in the subject area:'cDNA synthesis'"><div class="flagText">cDNA synthesis</div></a>
              <div data-categoryid="41483" data-articleid="25670"
                   data-categoryname="cDNA synthesis"
                   class="flagImage" title="Flag 'cDNA synthesis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1386'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2598'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8449&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20743">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=9141">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
