

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Airborne Metagenome in an Indoor Urban Environment</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001862"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001862" />

    <meta name="citation_title" content="The Airborne Metagenome in an Indoor Urban Environment"/>
    <meta itemprop="name" content="The Airborne Metagenome in an Indoor Urban Environment"/>

      <meta name="citation_author" content="Susannah G. Tringe"/>
            <meta name="citation_author_institution" content="Department of Energy (DOE) Joint Genome Institute, Walnut Creek, California, United States of America"/>
            <meta name="citation_author_institution" content="Genomics Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Tao Zhang"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Xuguo Liu"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Yiting Yu"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Wah Heng Lee"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Jennifer Yap"/>
            <meta name="citation_author_institution" content="Environmental Health Institute, National Environment Agency, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Fei Yao"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Sim Tiow Suan"/>
            <meta name="citation_author_institution" content="Department of Microbiology, National University of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Seah Keng Ing"/>
            <meta name="citation_author_institution" content="Department of Microbiology, National University of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Matthew Haynes"/>
            <meta name="citation_author_institution" content="Biology Department and Center for Microbial Science, San Diego State University, North Life Sciences, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Forest Rohwer"/>
            <meta name="citation_author_institution" content="Biology Department and Center for Microbial Science, San Diego State University, North Life Sciences, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Chia Lin Wei"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="Patrick Tan"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>
      <meta name="citation_author" content="James Bristow"/>
            <meta name="citation_author_institution" content="Department of Energy (DOE) Joint Genome Institute, Walnut Creek, California, United States of America"/>
            <meta name="citation_author_institution" content="Genomics Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Edward M. Rubin"/>
            <meta name="citation_author_institution" content="Department of Energy (DOE) Joint Genome Institute, Walnut Creek, California, United States of America"/>
            <meta name="citation_author_institution" content="Genomics Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Yijun Ruan"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Republic of Singapore"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001862.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1862"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Indoor climate and air quality. Review of current and future topics in the field of ISB study group 10.; citation_author=P Hoppe; citation_author=I Martinac; citation_journal_title=Int J Biometeorol; citation_volume=42; citation_number=1; citation_pages=1-7; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Epidemiology of infectious diseases;; citation_author=M Osterholm; citation_author=C Hedberg; citation_number=2; citation_pages=161-173; citation_date=2005; citation_publisher=Churchill Livingstone; " />
      <meta name="citation_reference" content="citation_title=Design and maintenance of hospital ventilation systems and prevention of airborne nosocomial infections;; citation_author=A Streifel; citation_number=3; citation_pages=1577-1589; citation_date=2004; citation_publisher=Lippincott Williams & Wilkins; " />
      <meta name="citation_reference" content="citation_title=Survey of Culturable Airborne Bacteria at Four Diverse Locations in Oregon: Urban, Rural, Forest, and Coastal.; citation_author=BT Shaffer; citation_author=B Lighthart; citation_journal_title=Microb Ecol; citation_volume=34; citation_number=4; citation_pages=167-177; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Profiles of airborne fungi in buildings and outdoor environments in the United States.; citation_author=BG Shelton; citation_author=KH Kirkland; citation_author=WD Flanders; citation_author=GK Morris; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=5; citation_pages=1743-1753; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Changing microbial concentrations are associated with ventilation performance in Taiwan's air-conditioned office buildings.; citation_author=PC Wu; citation_author=YY Li; citation_author=CM Chiang; citation_author=CY Huang; citation_author=CC Lee; citation_journal_title=Indoor Air; citation_volume=15; citation_number=6; citation_pages=19-26; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development of a high-volume aerosol collection system for the identification of air-borne micro-organisms.; citation_author=JL Radosevich; citation_author=WJ Wilson; citation_author=JH Shinn; citation_author=TZ DeSantis; citation_author=GL Andersen; citation_journal_title=Lett Appl Microbiol; citation_volume=34; citation_number=7; citation_pages=162-167; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB.; citation_author=TZ DeSantis; citation_author=P Hugenholtz; citation_author=N Larsen; citation_author=M Rojas; citation_author=EL Brodie; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=8; citation_pages=5069-5072; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Prokaryotic diversity–magnitude, dynamics, and controlling factors.; citation_author=V Torsvik; citation_author=L Ovreas; citation_author=TF Thingstad; citation_journal_title=Science; citation_volume=296; citation_number=9; citation_pages=1064-1066; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Computational improvements reveal great bacterial diversity and high metal toxicity in soil.; citation_author=J Gans; citation_author=M Wolinsky; citation_author=J Dunbar; citation_journal_title=Science; citation_volume=309; citation_number=10; citation_pages=1387-1390; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea.; citation_author=JC Venter; citation_author=K Remington; citation_author=JF Heidelberg; citation_author=AL Halpern; citation_author=D Rusch; citation_journal_title=Science; citation_volume=304; citation_number=11; citation_pages=66-74; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative metagenomics of microbial communities.; citation_author=SG Tringe; citation_author=C von Mering; citation_author=A Kobayashi; citation_author=AA Salamov; citation_author=K Chen; citation_journal_title=Science; citation_volume=308; citation_number=12; citation_pages=554-557; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and polyphasic taxonomy of Caulobacter species. Proposal of Maricaulis gen. nov. with Maricaulis maris (Poindexter) comb. nov. as the type species, and emended description of the genera Brevundimonas and Caulobacter.; citation_author=WR Abraham; citation_author=C Strompl; citation_author=H Meyer; citation_author=S Lindholst; citation_author=ER Moore; citation_journal_title=Int J Syst Bacteriol; citation_volume=49 Pt 3; citation_number=13; citation_pages=1053-1073; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Bacterial population in Russian space station “Mir”.; citation_author=Y Kawamura; citation_author=Y Li; citation_author=H Liu; citation_author=X Huang; citation_author=Z Li; citation_journal_title=Microbiol Immunol; citation_volume=45; citation_number=14; citation_pages=819-828; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Hospital-acquired Brevundimonas vesicularis septicaemia following open-heart surgery: case report and literature review.; citation_author=J Gilad; citation_author=A Borer; citation_author=N Peled; citation_author=K Riesenberg; citation_author=S Tager; citation_journal_title=Scand J Infect Dis; citation_volume=32; citation_number=15; citation_pages=90-91; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Brevundimonas diminuta infections and its resistance to fluoroquinolones.; citation_author=XY Han; citation_author=RA Andrade; citation_journal_title=J Antimicrob Chemother; citation_volume=55; citation_number=16; citation_pages=853-859; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=High genetic diversity among Stenotrophomonas maltophilia strains despite their originating at a single hospital.; citation_author=S Valdezate; citation_author=A Vindel; citation_author=P Martin-Davila; citation_author=BS Del Saz; citation_author=F Baquero; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=17; citation_pages=693-699; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Employment of 16 S rDNA gene sequencing techniques to identify culturable environmental eubacteria in a tertiary referral hospital.; citation_author=J Xu; citation_author=CL Smyth; citation_author=JA Buchanan; citation_author=A Dolan; citation_author=PJ Rooney; citation_journal_title=J Hosp Infect; citation_volume=57; citation_number=18; citation_pages=52-58; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prevalent bacterial species and novel phylotypes in advanced noma lesions.; citation_author=BJ Paster; citation_author=WA Falkler; citation_author=CO Enwonwu; citation_author=EO Idigbe; citation_author=KO Savage; citation_journal_title=J Clin Microbiol; citation_volume=40; citation_number=19; citation_pages=2187-2191; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Beta-lactamases and outer membrane investigations in beta-lactam-resistant Comamonas acidovorans strains.; citation_author=M Ravaoarinoro; citation_author=C Therrien; citation_journal_title=Int J Antimicrob Agents; citation_volume=12; citation_number=20; citation_pages=27-31; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular identification of potential pathogens in water and air of a hospital therapy pool.; citation_author=LT Angenent; citation_author=ST Kelley; citation_author=A St Amand; citation_author=NR Pace; citation_author=MT Hernandez; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=21; citation_pages=4860-4865; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The closest BLAST hit is often not the nearest neighbor.; citation_author=LB Koski; citation_author=GB Golding; citation_journal_title=J Mol Evol; citation_volume=52; citation_number=22; citation_pages=540-542; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes.; citation_author=RL Tatusov; citation_author=ND Fedorova; citation_author=JD Jackson; citation_author=AR Jacobs; citation_author=B Kiryutin; citation_journal_title=BMC Bioinformatics; citation_volume=4; citation_number=23; citation_pages=41; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=STRING: known and predicted protein-protein associations, integrated and transferred across organisms.; citation_author=C von Mering; citation_author=LJ Jensen; citation_author=B Snel; citation_author=SD Hooper; citation_author=M Krupp; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=24; citation_pages=D433-437; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of the fimA gene encoding bundle-forming fimbriae of the plant pathogen Xanthomonas campestris pv. vesicatoria.; citation_author=T Ojanen-Reuhs; citation_author=N Kalkkinen; citation_author=B Westerlund-Wikstrom; citation_author=J van Doorn; citation_author=K Haahtela; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=25; citation_pages=1280-1290; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A factor produced by Escherichia coli K-12 inhibits the growth of E. coli mutants defective in the cytochrome bd quinol oxidase complex: enterochelin rediscovered.; citation_author=GM Cook; citation_author=C Loder; citation_author=B Soballe; citation_author=GP Stafford; citation_author=J Membrillo-Hernandez; citation_journal_title=Microbiology; citation_volume=144 (Pt 12); citation_number=26; citation_pages=3297-3308; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Impact of either elevated or decreased levels of cytochrome bd expression on Shigella flexneri virulence.; citation_author=SS Way; citation_author=S Sallustio; citation_author=RS Magliozzo; citation_author=MB Goldberg; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=27; citation_pages=1229-1237; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Interruption of the cydB locus in Brucella abortus attenuates intracellular survival and virulence in the mouse model of infection.; citation_author=S Endley; citation_author=D McMurray; citation_author=TA Ficht; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=28; citation_pages=2454-2462; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Urban aerosols harbor diverse and dynamic bacterial populations.; citation_author=EL Brodie; citation_author=TZ Desantis; citation_author=JP Parker; citation_author=IX Zubietta; citation_author=YM Piceno; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=29; citation_pages=299-304; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rapid determination of 16S ribosomal RNA sequences for phylogenetic analyses.; citation_author=DJ Lane; citation_author=B Pace; citation_author=GJ Olsen; citation_author=DA Stahl; citation_author=ML Sogin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=82; citation_number=30; citation_pages=6955-6959; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=NAST: a multiple sequence alignment server for comparative analysis of 16S rRNA genes.; citation_author=TZ DeSantis; citation_author=P Hugenholtz; citation_author=K Keller; citation_author=EL Brodie; citation_author=N Larsen; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=31; citation_pages=W394-399; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bellerophon: a program to detect chimeric sequences in multiple sequence alignments.; citation_author=T Huber; citation_author=G Faulkner; citation_author=P Hugenholtz; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=32; citation_pages=2317-2319; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB.; citation_author=TZ DeSantis; citation_author=P Hugenholtz; citation_author=N Larsen; citation_author=M Rojas; citation_author=EL Brodie; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=33; citation_pages=5069-5072; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=34; citation_pages=186-194; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment.; citation_author=B Ewing; citation_author=L Hillier; citation_author=MC Wendl; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=35; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_author=P Green; citation_number=36; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Comparative metagenomics of microbial communities.; citation_author=SG Tringe; citation_author=C von Mering; citation_author=A Kobayashi; citation_author=AA Salamov; citation_author=K Chen; citation_journal_title=Science; citation_volume=308; citation_number=37; citation_pages=554-557; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea.; citation_author=JC Venter; citation_author=K Remington; citation_author=JF Heidelberg; citation_author=AL Halpern; citation_author=D Rusch; citation_journal_title=Science; citation_volume=304; citation_number=38; citation_pages=66-74; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns.; citation_author=MB Eisen; citation_author=PT Spellman; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=39; citation_pages=14863-14868; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001862" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Airborne Metagenome in an Indoor Urban Environment"/>
    <meta name="twitter:description" content="The indoor atmosphere is an ecological unit that impacts on public health. To investigate the composition of organisms in this space, we applied culture-independent approaches to microbes harvested from the air of two densely populated urban buildings, from which we analyzed 80 megabases genomic DNA sequence and 6000 16S rDNA clones. The air microbiota is primarily bacteria, including potential opportunistic pathogens commonly isolated from human-inhabited environments such as hospitals, but none of the data contain matches to virulent pathogens or bioterror agents. Comparison of air samples with each other and nearby environments suggested that the indoor air microbes are not random transients from surrounding outdoor environments, but rather originate from indoor niches. Sequence annotation by gene function revealed specific adaptive capabilities enriched in the air environment, including genes potentially involved in resistance to desiccation and oxidative damage. This baseline index of air microbiota will be valuable for improving designs of surveillance for natural or man-made release of virulent pathogens."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001862.g003"/>

  <meta property="og:title" content="The Airborne Metagenome in an Indoor Urban Environment" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001862" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9792'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8565'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7676&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001862">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001862" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001862&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Airborne Metagenome in an Indoor Urban Environment
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susannah G. Tringe
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Susannah G. Tringe, Tao Zhang</p>

              
              

                <p>Affiliations:
                  Department of Energy (DOE) Joint Genome Institute, Walnut Creek, California, United States of America, 
                  Genomics Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tao Zhang
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Susannah G. Tringe, Tao Zhang</p>

              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xuguo Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yiting Yu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wah Heng Lee, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jennifer Yap, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Environmental Health Institute, National Environment Agency, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fei Yao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sim Tiow Suan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, National University of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Seah Keng Ing, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, National University of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew Haynes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biology Department and Center for Microbial Science, San Diego State University, North Life Sciences, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Forest Rohwer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biology Department and Center for Microbial Science, San Diego State University, North Life Sciences, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chia Lin Wei, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrick Tan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James Bristow, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Energy (DOE) Joint Genome Institute, Walnut Creek, California, United States of America, 
                  Genomics Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edward M. Rubin
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:ruanyj@gis.a-star.edu.sg">ruanyj@gis.a-star.edu.sg</a> (YR); <a href="mailto:emrubin@lbl.gov">emrubin@lbl.gov</a> (ER)</p>

                <p>Affiliations:
                  Department of Energy (DOE) Joint Genome Institute, Walnut Creek, California, United States of America, 
                  Genomics Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yijun Ruan
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:ruanyj@gis.a-star.edu.sg">ruanyj@gis.a-star.edu.sg</a> (YR); <a href="mailto:emrubin@lbl.gov">emrubin@lbl.gov</a> (ER)</p>

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Republic of Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001862</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001862" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001862" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001862" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001862" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001862-g001" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001862-g002" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001862-t001" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001862-g003" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001862-t002" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001862">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001862" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The indoor atmosphere is an ecological unit that impacts on public health. To investigate the composition of organisms in this space, we applied culture-independent approaches to microbes harvested from the air of two densely populated urban buildings, from which we analyzed 80 megabases genomic DNA sequence and 6000 16S rDNA clones. The air microbiota is primarily bacteria, including potential opportunistic pathogens commonly isolated from human-inhabited environments such as hospitals, but none of the data contain matches to virulent pathogens or bioterror agents. Comparison of air samples with each other and nearby environments suggested that the indoor air microbes are not random transients from surrounding outdoor environments, but rather originate from indoor niches. Sequence annotation by gene function revealed specific adaptive capabilities enriched in the air environment, including genes potentially involved in resistance to desiccation and oxidative damage. This baseline index of air microbiota will be valuable for improving designs of surveillance for natural or man-made release of virulent pathogens.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Tringe SG, Zhang T, Liu X, Yu Y, Lee WH, et al.  (2008) The Airborne Metagenome in an Indoor Urban Environment. PLoS ONE 3(4):
          e1862.
            doi:10.1371/journal.pone.0001862</p><p><strong>Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> November 22, 2007; <strong>Accepted:</strong> February 1, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Tringe et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Agency for Science, Technology and Research (A*STAR), Singapore, and under the auspices of the US Department of Energy's Office of Science, Biological and Environmental Research Program, and by the University of California, Lawrence Livermore National Laboratory under Contract No. W-7405-Eng-48, Lawrence Berkeley National Laboratory under contract No. DE-AC02-05CH11231, and Los Alamos National Laboratory under contract No. DE-AC02-06NA25396.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Modern humans spend 90% of their lives indoors <a href="#pone.0001862-Hoppe1">[1]</a>, and the air in these enclosed spaces contains a variety of microorganisms including bacteria, fungi, and viruses, some potentially harmful to human health. While air provides an extremely harsh environment for microbial survival, airborne transmission is the predominant route for disseminating microorganisms, and malicious dispersal of infectious agents represents a potential public health risk. The origins and composition of indoor air microbiota, however, are poorly understood. Determining the overall biological diversity in the indoor atmosphere, and assessing its dynamics, is essential to facilitating the rational development of public health policies.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Airborne microbes are often attached to dust particles or water droplets from sneezes and coughs or breezes over land or bodies of water. When the water in aerosols evaporates, the microbes become droplet nuclei and clumps, most of which can stay airborne indefinitely and drift with air flows <a href="#pone.0001862-Osterholm1">[2]</a>, <a href="#pone.0001862-Streifel1">[3]</a>. While studying airborne microbes is made challenging by their low concentrations, such particles enriched with microorganisms can be collected by sampling large volumes of air through air handling units (AHU) in modern building ventilation systems, without deployment of specialized sampling devices.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Studies of cultivable airborne microbes by a variety of sampling methods have revealed numerous bacteria and fungi to be present in air <a href="#pone.0001862-Shaffer1">[4]</a>–<a href="#pone.0001862-Wu1">[6]</a> but likely underestimate the diversity of the air microbial community as most environmental microbes are resistant to culture. One study of outdoor air found that only 0.08% of the microscopically evident motile cells was readily cultured <a href="#pone.0001862-Radosevich1">[7]</a>. Culture-independent techniques such as 16S ribosomal RNA gene coding DNA (16S rDNA) analysis and metagenomic sequencing provide a less biased perspective on environmental microbes because DNA is sampled directly from the environment. In this study, we have used an AHU filtration strategy for air sample collection and performed both 16S rDNA and metagenomic analyses to characterize the airborne biological diversity in an indoor urban environment.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We collected indoor air from two shopping centers in Singapore (<a href="#pone-0001862-g001">Figure 1A</a>), between March and April of 2005. Using an AHU (<a href="#pone-0001862-g001">Figure 1B</a>), approximately 6 million cubic meters of air (80% recycled and 20% fresh) were sampled from each of the two locations for analysis. A variety of observations support the premise that the time-averaged biological populations found on the filters constitute a true sampling of the air microbiota. This includes <sup>3</sup>H-thymidine and <sup>3</sup>H-leucine incorporation assays revealing the majority of the cells arrested in the filters not to be actively growing (Supplement <a href="#pone.0001862.s004">Tables S1</a> and <a href="#pone.0001862.s005">S2</a> in Supporting Information, SI), and physical and microscopic inspections revealing no indication of active bacterial growth (data not shown). Lastly, a culture-based survey of the filter contents revealed no abnormally high growth rates for particular microorganisms, and many of the isolates obtained were gram-positive species not found to be abundant in the DNA sequence data (<a href="#pone.0001862.s006">Tables S3</a> and <a href="#pone.0001862.s007">S4</a> in SI).</p>
<div class="figure" id="pone-0001862-g001"><div class="img"><a name="pone-0001862-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001862.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001862.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Air sampling location and device.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>A. The Singapore map and the two sampling locations (indicated by red stars) in Singapore. B. A schematic view of the air handling unit (AHU) filtration system used in the two buildings. C. An air panel filter (60×50×2 cm<sup>3</sup>) used in this study.</p>
<span>doi:10.1371/journal.pone.0001862.g001</span></div>
<h4>Indoor air microbial diversity assessed by 16S rDNA analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Phylogenetic diversity of the airborne bacteria in each location was first assessed with 16S rDNA clone sequencing (see <a href="#s4">Materials and Methods</a>). Among the 2,659 and 3,063 high-quality 16S rDNA sequences analyzed from the two air samples of locations 1 and 2 (Air-1 and Air-2), respectively, we observed 129 and 108 distinct ribotypes, most of which (78% for Air-1 and 91% for Air-2) had close (&gt;95% identity over &gt;1000 bp) relatives in the Greengenes database (<a href="http://greengenes.lbl.gov/">http://greengenes.lbl.gov/</a>)<a href="#pone.0001862-DeSantis1">[8]</a> (<a href="#pone-0001862-t001">Table 1</a>). Based on rarefaction curves constructed from these data, the samples are estimated to contain 170 to 300 distinct species in total (<a href="#pone-0001862-t001">Table 1</a>, <a href="#pone-0001862-g002">Figure 2A</a>). This stands in contrast to multiple surveys of outdoor environments where thousands – even millions - of different species are often estimated to be present, and a significant fraction of the 16S rDNA clone sequences are novel <a href="#pone.0001862-Torsvik1">[9]</a>–<a href="#pone.0001862-Tringe1">[12]</a>.</p>
<div class="figure" id="pone-0001862-g002"><div class="img"><a name="pone-0001862-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001862.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001862.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Microbial diversity in air and other local environmental samples.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A. Rarefaction curves of observed phylotype diversity for Air-1 (pink line), Air-2 (dark blue), water (green), Soil-1 (purple), and Soil-2 (red). The similarity threshold is 97%. Inset: the estimated diversity in the 5 samples using the Chao1 richness estimator. B. Distribution of microbial divisions in each of the environmental samples; circles represent, from the inside out, Air-1, Air-2, Water, Soil-1, and Soil-2. C. Bacterial orders within the Proteobacteria for each sample. D. A Venn diagram of all phylotypes observed in the air and dust samples from location 2.</p>
<span>doi:10.1371/journal.pone.0001862.g002</span></div><div class="figure" id="pone-0001862-t001"><div class="img"><a name="pone-0001862-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001862.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001862.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>16S rDNA analysis.</span></strong></p><span>doi:10.1371/journal.pone.0001862.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>It is commonly presumed that airborne microbes are a random assortment of aerosolized cells from nearby primary environments such as soil and water bodies, as the air environment is inadequate to sustain growth. To compare the biological contents of air to other local habitats, we collected two top soil samples from locations close to the air-sampled buildings (Soil-1 and Soil-2) and a water sample from the Singapore River that flows next to the shopping mall in location 1, and performed 16S rDNA clone sequencing on all the samples (<a href="#pone-0001862-t001">Table 1</a>).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Comparisons of these data to those from air showed the microbial diversity in air to be substantially lower than that in the aquatic and terrestrial environments, which were each estimated to contain thousands of phylotypes (<a href="#pone-0001862-t001">Table 1</a> and <a href="#pone-0001862-g002">Figure 2A</a>). The phylogenetic spectrum of organisms in the air is also very different from that of water and soil (<a href="#pone-0001862-g002">Figure 2B and 2C</a>; more details in <a href="#pone.0001862.s007">Table S4</a> in SI), suggesting that the organisms that are abundant in air are not those that dominate nearby terrestrial or aquatic environments. Significantly, the two air samples contained more phylotypes in common with each other than with the other environmental samples (<a href="#pone.0001862.s007">Table S4</a>).</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The most abundant airborne microbes included several species of <em>Brevundimonas</em> (56% and 44% of sequences in Air-1 and Air-2; <a href="#pone.0001862.s007">Table S4</a> in SI). This cosmopolitan genus within the <em>Caulobacterales</em> has been observed almost ubiquitously in fresh and salt water, soil, sludge and oil. <em>Brevundimonas</em> species have also been cultivated from nominally sterile environments such as the space station Mir, and in clinical settings where they have been implicated in opportunistic infections <a href="#pone.0001862-Abraham1">[13]</a>–<a href="#pone.0001862-Han1">[16]</a>. Interestingly, despite the prevalence of organisms from the typically aquatic <em>Caulobacterales</em> in both air samples, only one sequence affiliated with this family was observed among the 2546 16S rDNA sequences of from the Singapore river water sample (<a href="#pone.0001862.s007">Table S4</a> in SI).</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>16S rDNA sequence clusters closely related (many of them are ≥99% identity) to <em>Stenotrophomonas maltophilia</em> were also abundant in both the locations (29% of clones in Air-1 and 10% of clones in Air-2) (<a href="#pone.0001862.s007">Table S4</a> in SI). This species, which also showed up several times in the culture-based survey (<a href="#pone.0001862.s006">Table S3</a> in SI), is an environmentally widespread opportunistic pathogen frequently implicated in hospital-acquired infections <a href="#pone.0001862-Valdezate1">[17]</a>. There were also phylotype groups, such as <em>Brachybacterium</em>, <em>Acinetobacter</em>, and members of <em>Microbacteriaceae</em> and <em>Micrococcaceae</em>, which were frequently observed in one air sample, but not in the other (<a href="#pone.0001862.s007">Table S4</a> in SI). This differential representation of phylotypes suggests that each of the two shopping centers, though harboring a shared set of abundant microbes, has its own unique indoor microbiota.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>It is anticipated that air suspension particles would eventually settle to the floor. Therefore the floor sediment could reflect a fraction of the airborne organisms collected over time. We surveyed the microbial content of floor dust inside one of the buildings (Air-2) and compared it to dust immediately outside of the building using the same 16S rDNA analysis. Among the 566 and 624 clones of 16S rDNA sequences analyzed from the two samples, 5 and 7 clones from the indoor sample hit the dominant indoor air genera <em>Brevundimonas</em> and <em>Stenotrophomonas</em>, respectively, while none from the outside fell into these groups. Even more strikingly, 127 clones from the indoor dust and just 3 from the outdoor dust contained close matches to <em>Acinetobacter</em>, a genus found frequently (304 of 3063 clones) in the Air-2 clone library from this location but not that from the other location (<a href="#pone.0001862.s007">Table S4</a> in SI). Overall, some 26 of the 129 phylotypes observed in Air-2 were also among the 133 phylotypes in the indoor dust, but just 7 were seen in the outside dust (<a href="#pone-0001862-g002">Figure 2D</a>). These data further support the hypothesis that the indoor air is a distinct habitat, and that these dominant organisms were truly present in the air and not merely on the air filters.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>To identify potential origins for these airborne microorganisms, we inspected the two buildings and did not find obvious indoor habitats, such as planters or water fountains, which might serve as favorable reservoirs for these organisms. As some of the phylotype groups observed in both air samples included organisms previously observed in clinical specimens or human-inhabited environments, and included a number of human commensals and potential opportunistic pathogens (<a href="#pone.0001862.s010">Table S7</a> in SI) <a href="#pone.0001862-Abraham1">[13]</a>, <a href="#pone.0001862-Xu1">[18]</a>–<a href="#pone.0001862-Ravaoarinoro1">[20]</a>, it is reasonable to speculate that the human occupants of the shopping centers could contribute the airborne microbes through sneezing and coughing. We sampled a nasal swab and a saliva specimen from human subjects who had visited the shopping centers, and sequenced more than 1000 16S rDNA clones from each sample. Both of the samples were dominated by known human commensals and pathogens, such as <em>Corynebacteria</em> in the nasal swab and <em>Streptococcus</em> in saliva, but the nasal swab also contained 2 hits to the dominant air organism <em>Brevundimonas</em> (<a href="#pone.0001862.s007">Table S4</a> in SI). Though more human respiratory samples should be surveyed, this overlap of observed microbes in air samples and human nasal samples suggests an active interaction between the air microbiota and human occupants in the indoor space.</p>
<a id="article1.body1.sec2.sec1.p8" name="article1.body1.sec2.sec1.p8"></a><p>In combination, the 16S rDNA analysis of the air filter samples revealed that in comparison to local microbial communities in primary habitats such as soil and water, the microbial population in air from these two buildings is of limited diversity and unique composition, indicating that the air environment harbors a community that is not a random mix of visitors from other habitats. This is consistent with a previous study indicating that certain organisms may become aerosolized preferentially <a href="#pone.0001862-Angenent1">[21]</a>.</p>


<h4>Air metagenomic analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We hypothesized that some organisms, such as <em>Brevundimonas</em> spp., possess features that make them amenable to air dispersal. To gain insight into the molecular mechanisms allowing these specific species but not others to be enriched in the air, we then undertook genomic shotgun sequencing to further analyze the aerogenome.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>DNA isolated from the filter-trapped microbes was used to build small insert libraries for shotgun sequencing (<a href="#s4">Materials and Methods</a>). Roughly 80 million bases total of DNA sequence were generated from the two air samples (<a href="#pone.0001862.s008">Table S5</a> in SI). If a single bacterial species with a genome size of 4 Mb makes up 50% of each sample, this collection of sequences would provide a 5-fold genome coverage of the dominant bacterium in each sample, and therefore supply an informative glimpse of the previously unexplored air metagenome.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>A total of 66,702 and 74,018 DNA sequence reads with more than 100 contiguous high-quality bases were generated from samples Air-1 and Air-2, respectively. 52,303 (78%) and 58,587 (79%) of these had BLASTX hits in GenBank with an e-value less than 10<sup>−8</sup>. By comparison, less than half of the shotgun sequencing reads from soil in a recent study had hits at this threshold <a href="#pone.0001862-Tringe1">[12]</a>, indicating that most air microbes are likely to be closely related to fully or partially sequenced organisms. Considering the bias towards pathogenic species in current microbial DNA sequence databases, this may suggest that many indoor air microbes originate from human-associated habitats. 88% and 72% of the hits in samples Air-1 and Air-2 respectively are to bacteria, particularly Proteobacteria, while 0.26% and 2.0% of the hits are to eukaryotes; the remainder are to archaea and sequences without an associated taxonomy (<a href="#pone.0001862.s009">Table S6</a> in SI). By far the most frequent hits (28% for Air-1 and 35% for Air-2) were to the Alphaproteobacterium <em>Caulobacter crescentus</em>, a member of the same order, the <em>Caulobacterales</em>, that encompasses the <em>Brevundimonas</em> species observed repeatedly in the 16S rDNA clone library. <em>C. crescentus</em> was the only member of this order whose complete genome sequence was available in GenBank at the time of analysis. The next most abundant hits were to members of the order <em>Xanthomonadales</em> (22% for Air-1 and 8% for Air-2), which includes <em>Stenotrophomonas</em>; among the remaining hits, no more than a few percent were to any single genus. Though BLASTX hits are a poor means of species assignment <a href="#pone.0001862-Koski1">[22]</a>, the prevalence of hits to these families indicates that they make up a substantial portion of the organisms trapped from air on the AHU filters. While only 153 and 132 partial 16S ribosomal DNA sequences were found within the metagenomic data from Air-1 and Air-2, they confirmed an abundance of <em>Caulobacterales</em> and <em>Xanthomonadales</em> among the studied microbes (<a href="#pone.0001862.s009">Table S6</a> in SI).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Given the relatively low diversity in air, we expected some of the sequences to assemble into larger contigs. In each sample, roughly 60% of the reads assembled into contigs with reads from independent clones (<a href="#pone.0001862.s008">Table S5</a> in SI). While no individual genome was covered at sufficient depth in the air filter data to achieve genome assembly, 930 and 770 contigs longer than 3 kb emerged from the Air-1 and Air-2 library assemblies, respectively, and the average length of all contigs was ~1.5 kb (<a href="#pone.0001862.s001">Figure S1</a> in SI). In both Air-1 and Air-2, the longer and deeper contigs appeared to be affiliated with the <em>Caulobacterales</em> and <em>Xanthomonadales</em> (<a href="#pone.0001862.s002">Figure S2A</a> in SI). Based on the 16S rDNA amplicon clone analysis, rDNA sequences present in these contigs, and the results of BLASTN and BLASTX, we believe that these were derived primarily from the <em>Brevundimonas</em> and <em>Stenotrophomonas</em> spp. documented in the 16S clone libraries. Neither of these genera had a fully sequenced representative in the public database at the time of analysis.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>To further validate this observation, we conducted semi-quantitative PCR to assess the relative abundance of some of the larger contig sequences in air and other environmental samples (<a href="#pone.0001862.s003">Figure S3</a> in SI). The PCR results indeed suggested that these contig sequences were only abundant in the indoor air, but not in nearby terrestrial and water samples. Importantly, when signal was detected in the soil samples, it was of lesser magnitude than in air and present only in the soil sample matched to the location of the original sequence (see Air-1 contigs 15516 and 15369, and Air-2 contig 16031).</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>The pathogen burden carried by indoor air has not been extensively studied outside of hospitals, and it is unclear whether organisms related to potential bioterror agents are present in our atmosphere. Reassuringly, neither the 16S rDNA clone data nor the metagenomic data contained close sequence matches to virulent pathogens such as <em>Bacillus anthracis, Yersinia pestis</em> or other species on the CDC bioterror watch list (<a href="http://www.bt.cdc.gov/agent/agentlist.asp">http://www.bt.cdc.gov/agent/agentlist.as​p</a>). Some 1.5% of the assembled sequences aligned at 80–90% nucleotide identity to virulent and opportunistic pathogens, including pathogenic species of <em>Brucella</em>, <em>Bordetella</em>, <em>Burkholderia</em> and <em>Mycobacterium</em> (<a href="#pone.0001862.s010">Table S7</a> in SI), but this level of sequence identity is inadequate to predict pathogenicity.</p>


<h4>DNA sequence functional analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Relatively little is known about how microbes survive the stress of becoming airborne, and whether there are genetic contributors to aerosolization or airborne dissemination. We therefore examined the metagenomic data from the air samples to identify genes and functions that are overrepresented among the air microbiota, using the orthologous groups defined by the COG (<a href="http://www.ncbi.nlm.nih.gov/COG/">http://www.ncbi.nlm.nih.gov/COG/</a>) <a href="#pone.0001862-Tatusov1">[23]</a> and STRING (<a href="http://string.embl.de/">http://string.embl.de/</a>) <a href="#pone.0001862-vonMering1">[24]</a> databases. We first predicted the open reading frames (ORFs) from the air DNA sequences, and then annotated the functions and assessed the frequency of the ORFs using our previously described Environmental Gene Tag (EGT) analysis <a href="#pone.0001862-Tringe1">[12]</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Of the 34,984 and 44,021 predicted ORFs in the partially assembled Air-1 and Air-2 sequences, 24,733 (70.7%) and 26,897 (61.1%) of them have significant hits by BLASTP in the STRING database. On a broad functional level, genes participating in cell motility and secretion were more common in the air data than in previously characterized soil, ocean and whale fall community sequences (<a href="#pone-0001862-g003">Figure 3</a>). Specific gene families within these categories that were overrepresented in air filter DNA included cell membrane proteins participating in protein secretion, motility and conjugal transfer, particularly a large set of functionally related genes homologous to members of the VirB, VirD and Trb systems (<a href="#pone-0001862-t002">Table 2</a>). Fimbrial adhesins, whose homologs have been implicated in pathogenicity, motility and cell aggregation, also stood out as common “air community” genes; interestingly, the aggregation promoted by fimbriae in <em>Xanthomonas campestris</em> has been found to contribute to resistance to UV light and desiccation <a href="#pone.0001862-OjanenReuhs1">[25]</a>. We hypothesize that these cell surface protein genes are overrepresented in the air microbial community because they improve survival in the atmosphere, but they may also facilitate colonization of human building occupants or promote adhesion of cells to the fibers of air filters.</p>
<div class="figure" id="pone-0001862-g003"><div class="img"><a name="pone-0001862-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001862.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001862.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>COG analysis.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Predicted ORFs from the two air samples in this study and the other 3 environmental samples from previous studies were mapped to orthologous groups and broad functional categories according to the COG and STRING databases. Only functions that are more common in air than in other environments are depicted: N, Cell motility and secretion; U, Intracellular trafficking and vesicular transport; P, Inorganic ion transport and metabolism; K, Transcription; L, DNA replication, recombination and repair; G, Carbohydrate transport and metabolism; and O, Posttranslational modification.</p>
<span>doi:10.1371/journal.pone.0001862.g003</span></div><div class="figure" id="pone-0001862-t002"><div class="img"><a name="pone-0001862-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001862" data-uri="info:doi/10.1371/journal.pone.0001862.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001862.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001862.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001862.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001862.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001862.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Functionally associated gene clusters overrepresented in air and their relative abundance in other environments.</span></strong></p><span>doi:10.1371/journal.pone.0001862.t002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Also among the genes most overrepresented in the air sequences from both samples were a number of orthologous groups containing proteins involved in redox metabolism and inorganic ion metabolism (<a href="#pone-0001862-g003">Figure 3</a>, <a href="#pone-0001862-t002">Table 2</a>). These include a set of genes necessary for the biosynthesis of cytochrome bd, a terminal oxidase known to be important for survival of oxidative stress and iron deprivation <a href="#pone.0001862-Cook1">[26]</a> as well as virulence in some pathogens <a href="#pone.0001862-Way1">[27]</a>, <a href="#pone.0001862-Endley1">[28]</a>. Genes involved in iron transport and metabolism, particularly siderophore synthesis, sensing and uptake, were also prevalent in air microbes (<a href="#pone-0001862-t002">Table 2</a>). Not surprisingly, many poorly characterized and uncharacterized genes were found exclusively or predominantly in air data relative to sequence from previously examined outdoor environments.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>This metagenomic analysis of microbial species in a densely populated urban indoor atmosphere demonstrates that the organisms in air are distinct from those found in surrounding outdoor environments and possess unique genomic features. The considerable similarity observed in the indoor air organisms and metagenomes derived from two closely separated localities suggests that indoor air microbial communities share organisms and genetic features in common, although the primary habitats for these organisms are not entirely clear but likely include humans themselves.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Based on our functional assessments, it appears that the major stresses encountered by air microbiota may be iron limitation, oxidative damage and desiccation, all of which could reasonably be expected in indoor air. Our results also imply that the indoor air environment exerts a specific selective influence on microbes, so as to enrich certain organisms in the air. Sequences suggestive of opportunistic pathogens as well as virulence-associated genes are also common in the air DNA, indicating that infectious agents, but not virulent pathogens, are likely present in everyday air. These findings suggest that the resident microbes in the indoor atmosphere have been selected for an indoor life cycle, part of which is spent in the air.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>This baseline characterization of air microbiota provides an in-depth glimpse of an everyday environment closely encountered by the general public. However, the airborne microbiota may be dynamic and sensitive to changes by direct and indirect factors ranging from outside climate variation to indoor occupants and micro-niche establishment <a href="#pone.0001862-Brodie1">[29]</a>, and further analysis of sub-compartments of indoor air and different time points over a long period will be necessary to present a comprehensive picture of airborne microbiota. Our results indicate that air harbors a unique community that may originate from a variety of niches and is shaped by selective forces in the air environment. These findings will aid in formulating public polices guiding the quantification and measurement of indoor air composition for environmental and human health.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>1. Samples and DNA extraction</h4>
<h5>Air samples.</h5><a id="article1.body1.sec4.sec1.sec1.p1" name="article1.body1.sec4.sec1.sec1.p1"></a><p>We collected indoor air filters installed in two shopping centers at locations 1 and 2 in Singapore (<a href="#pone-0001862-g001">Figure 1A</a>), between March and April of 2005, for isolation of airborne microbes. The two sites are located 6.7 Km apart from one another; location 1 is situated along the Singapore River, which is connected to the Marina Bay ~960 meters down stream, and location 2 is ~1000 meters away from the east coast of Singapore. The retail areas in locations 1 and 2 are 23,111 and 52,144 square meters, respectively. The air supply to each building comprised 80% recycled air and 20% fresh air, processed in air handling units (AHU) to remove airborne dust and lower the temperature. In the AHU, the air flow velocity was on average 1.5 meters per second through the air filters (<a href="#pone-0001862-g001">Figure 1B</a>). There are 15 and 40 AHUs in location 1 and 2 respectively. On average, each AHU has 10–15 air filters mounted in parallel. Panel filters, NGB 290 (Libeltex, Belgium) (<a href="#pone-0001862-g001">Figure 1C</a>), were used in these buildings. The dimensions of the filter are 60×50×2 cm<sup>3</sup>, and the arrestance efficiency is 90% for 1 µm particles. The panel filters were installed for air circulation operation 14 hours per day, and for a total of 90 days before collection. Approximately 2 million cubic meters of air passed through each of the panel filters during the sampling period.</p>
<a id="article1.body1.sec4.sec1.sec1.p2" name="article1.body1.sec4.sec1.sec1.p2"></a><p>Three pieces of air filters from each site (6 million cubic meters of air per location) were soaked and washed with 10 liters of PBS buffer. The PBS contained no detectable bacterial contamination as determined by PCR using bacterial 16S specific primers (data not shown). The suspension was then filtered through Whatman® filter paper (#114) to remove big particles. We also tested Whatman 3MM Chr filter paper (Cat#3030917) for removing the dusts. Although the flow through was much clearer, the microbial yield was quite low. In both cases, the filtrates were then concentrated by 0.2 µm tangential flow filters (Amersham, CFP-2-E-4A). The microbes were pelleted by centrifugation (9,500 <em>g</em>) at 4°C for 30 min. The pellets were resuspended in 10 ml of PBS and stored at −80°C.</p>
<a id="article1.body1.sec4.sec1.sec1.p3" name="article1.body1.sec4.sec1.sec1.p3"></a><p>Microbial genomic DNA was extracted using the PUREGENE® DNA PURIFICATION KIT (Cat No. D-6000A, Gentra Systems, USA), following the manufacturer's protocol. Briefly, 500 µl of microbial pellet suspension were pelleted in a bench top centrifuge at full speed for 5 seconds. The pellets were resuspended in 600 µl of Cell Lysis Solution and incubated at 80°C for 5 min to lyse cells. The sample was further digested with 3 µl of RNAse A Solution at 37°C for 1 hour. Proteins were precipitated by adding 200 µl of Protein Precipitation Solution and spinning at full speed for 1 min at 4°C. The supernatants were transferred into a clean tube and mixed with 600 µl isopropanol. DNA was pelleted by spinning at full speed (13,200 rpm) for 1 min at 4°C. The DNA pellets were washed with 500 µl 70% ethanol and dissolved in 50 µl of DNA Hydrate Solution by incubating at 37°C for 1 hour. Typically, the yield of genomic DNA was 1 µg from one preparation.</p>
<a id="article1.body1.sec4.sec1.sec1.p4" name="article1.body1.sec4.sec1.sec1.p4"></a><p>The air sample genomic DNA prepared using a DNA Purification Kit (PUREGENE) still contained black particles from the air filters, which were found to be inhibitory to PCR and restriction enzyme digestion. We further used electrophoresis to separate the insoluble particles from the genomic DNA in a 1% agarose gel. The genomic DNA was excised from the gel and purified using the Qiaquick Gel Extraction Kit (Qiagen), following the protocol of the manufacturer.</p>

<h5>Soil samples.</h5><a id="article1.body1.sec4.sec1.sec2.p1" name="article1.body1.sec4.sec1.sec2.p1"></a><p>Soil samples were collected from the surface of open fields near locations 1 and 2, respectively. Plants and large sand particles were removed by passing through mesh sieves. The size selected soil samples were extracted directly using the <span class="underline">PUREGENE® DNA Purification Kit</span> (Cat No. D-6000A, Gentra, USA). The humic acids contamination was removed by 0.7% gel electrophoresis. The genomic DNA was excised from the agarose gel and purified by Qiagen gel extraction kit following the manufacturer's protocol. Typically, 300 mg of soil could yield 300–600 ng of genomic DNA.</p>

<h5>Water samples.</h5><a id="article1.body1.sec4.sec1.sec3.p1" name="article1.body1.sec4.sec1.sec3.p1"></a><p>The Singapore River is connected to Marina Bay at the southeast coast of Singapore. Essentially, it is an extension of the bay and contains salt water. Twenty liters of water were collected from the Singapore River right next to the shopping mall at location 1. The water was filtered through Whatman paper (#113), and further concentrated by tangential filtration using 0.2 µm filter column (Amersham, CFP-2-E-4A). The concentrated microbial fraction was centrifuged (9,500 g) for 30 min at 4°C. The microbial pellet was extracted for DNA using the PUREGENE® DNA Purification Kit (Cat No. D-6000A, Gentra, USA). The DNA yield is roughly 3 µg from 20 liters of water.</p>

<h5>Floor dust samples.</h5><a id="article1.body1.sec4.sec1.sec4.p1" name="article1.body1.sec4.sec1.sec4.p1"></a><p>In the location-2 building, the open floors are daily cleaned with high hygiene standard. Four (4) dead corners (corners next to escalators and open to all floors) in the building were identified with obvious accumulation of dust, representing average sedimentation of air suspension over times. Sterile Kimsweeper tissues were used to sweep the floor to collect the dust, and kept in sterile plastic bags. Dust samples of 3 dead corners immediately outside of the building were also collected similarly. In the lab, dust containing tissues were dissolved in 10 ml of PBS buffer followed by centrifugation to pellet the microbes. The microbial pellet was extracted for DNA using the PUREGENE® DNA Purification Kit (Cat No. D-6000A, Gentra, USA).</p>

<h5>Human fluid samples.</h5><a id="article1.body1.sec4.sec1.sec5.p1" name="article1.body1.sec4.sec1.sec5.p1"></a><p>Human nasal swab was collected and re-suspended in 50ul PBS buffer. 2 µl of the solution was directly used for broad range 16S rDNA PCR analysis. Approximately 50 µl of human saliva was collected. 2 µl of the homogenized solution was directly used for broad range 16S rDNA PCR analysis.</p>



<h4>2. Growth rate assessment of air filter-trapped microorganisms</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Filter samples collected from the two locations were tested for growth rate. Sections of the filters (2 cm×2 cm) were soaked in 10 ml PBS and shaken for 15 min at room temperature. The filters were placed in 50 ml syringes, and the liquid forced out using the plunger. From this sample 10 µl was taken for direct counts using an epifluorescent microscope. First, 1 ml of 2% paraformaldehyde was added to the 10 µl samples, which were then incubated at room temperature for 1 hour. The volume was increased to 5 ml in sterile water and the samples were deposited on 0.2 µm filters by vacuum filtration. The filters were stained with 100 µl 5× SYBR Gold (Molecular Probes) for 10 min in the dark, then rinsed in 100 µl H<sub>2</sub>O and mounted on glass slides. Bacterial counts were performed with Image-Pro Plus software (<a href="http://www.jknelectronics.com/ippage.htm">http://www.jknelectronics.com/ippage.htm</a>).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>From the 10 ml filtered microbial suspension, 1 ml for each of the three samples was incubated for 1 hour at room temperature in the presence of either 20 nM <sup>3</sup>H-Leucine or 20 nM <sup>3</sup>H-Thymidine (Amersham Biosciences). The incubation was terminated by addition of 75 µl 100% trichloroacetic acid (TCA). Blank reactions were set up by adding the TCA at the beginning of the incubation with the radioactive tracer. All reactions were chilled on ice for 10 min, and then pelleted at full speed in a microcentrifuge at room temperature for 5 min. The supernatants were aspirated, and the pellets resuspended in cold 5% TCA, vortexed, and pelleted again as above. This step was repeated with the substitution of 80% ethanol for TCA, and finally the pellets were resuspended in 1 ml scintillation fluid (Ultima Gold) and counted for 3 min in a scintillation counter. The amount of radiation incorporated into newly synthesized DNA was determined by subtracting the average radiation (dpm) in the controls from the average dpm found in the samples. The counting efficiency of the scintillation counter was first determined and used to calculate pmol thymidine incorporated into DNA ml<sup>−1</sup> hr<sup>−1</sup> ([adjustment for counting efficiency; 1 dpm per 0.85 cpm]×[1 Ci per 2.22×10<sup>12</sup> dpm]×[1 mmol per 84 Ci]). Each sample was counted for 3 min. For each sample the 3 replicate counts were averaged and “Carry-over Controls” were subtracted from this value to account for unincorporated radiation. This value was converted to pmol thymidine incorporation cell<sup>−1</sup> hr<sup>−1</sup>, using the mean direct count data (<a href="#pone.0001862.s004">Table S1</a>).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Based on the number of cells in the filters and the <sup>3</sup>H incorporation counts, the growth rates of microbial cells washed off from the filters were determined using standard conversion values. The growth rate of cells in sample 1 had an estimated doubling time of 244 (leucine) to 577 (thymidine) days. The incorporations of both leucine and thymidine in samples 2 and 3 were so low that no estimation of growth rates could be determined (i.e., the dpms in the control were equal to or greater than those observed in the samples). Overall, these results suggest essentially no growth of cells in all of the samples (<a href="#pone.0001862.s005">Table S2</a>).</p>


<h4>3. Cultivation and identification of air filter-trapped Microorganisms</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Airborne microbial concentrates in PBS buffer were diluted in 0.9% saline and plated in series on yeast peptone agar for fungal growth at 28°C, and tryptone soy agar and plate count agar for bacterial growth at 37°C. Visibly different colonies based on morphology were picked and streaked on nutrient agar for taxonomic identification.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>All the bacterial isolates were Gram stained and then subjected to standard analyses for classification as Gram positive (Catalase and Coagulase tests) and Gram negative (Oxidase test). After these 3 preliminary biochemical tests, a semi-automated bacterial identification instrument (VITEK Systems Inc., USA) was used with the programs for Vitek Bacillus Identification (BAC), the Gram Positive Identification (GPI) and the Gram Negative Identification (GNI) accordingly. For optimal conditions, Gram-positive isolates were diluted in saline to a turbidity concentration of 0.5 McFarland standard and Gram-negative isolates diluted to 1 McFarland standard or their optical equivalents, prior to loading onto the GPI and GNI test cards. Subsequently, these cards were incubated at 35.4°C +/− 2°C and processed automatically by the VITEK systems. Fungal and streptomycete isolates were identified microscopically after staining with lactophenol cotton blue (Sigma) and Gram reagents, respectively.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>307 bacterial isolates were classified based on morphology, Gram-staining and standard microbiological tests. 58% of the isolates were Gram-positive and 42% were Gram-negative. A collection of 74 isolates was further characterized using the VITEK system, of which 60 (81%) could be assigned to known taxonomic units of bacteria at genus level (<a href="#pone.0001862.s006">Table S3</a>). The methods used for isolation did not yield abundant fungal isolates. Only 3 <em>Penicillium</em> isolates were obtained.</p>


<h4>4. Broad range 16S rDNA sequence analyses</h4>
<h5>16S clone library Construction.</h5><a id="article1.body1.sec4.sec4.sec1.p1" name="article1.body1.sec4.sec4.sec1.p1"></a><p>The 16S rDNA was amplified from purified genomic DNA using bacterial universal primers; Bact-8F (<span class="gene">5′-AGAGTTTGATCCTGGCTCAG-3′</span>) and Bact-1391R (<span class="gene">5′-GACGGGCGGTGTGTRCA-3′</span>) <a href="#pone.0001862-Lane1">[30]</a>. Reaction conditions were as follows: 5.0 µl 10× AccuPrime™ PCR Buffer II (Invitrogen), 1.0 µl forward primer and 1.0 µl reverse primer (10 µM each), 0.2 µl AccuPrime™ <em>Taq</em> High Fidelity (5 U/µl; Invitrogen) and 1.0 µl of template genomic DNA (10 ng) in a total volume of 50 µl. The following cycling parameters were used: 5 min of initial denaturation at 95°C followed by 20 cycles of denaturation (30 s at 95°C), annealing (30 s at 56°C), and elongation (90 s at 72°C), with a final extension at 72°C for 8 min. Appropriately sized PCR products were agarose gel purified and cloned into the pCR-Blunt II-TOPO vector (Invitrogen). The libraries were transformed into TOP10 electrocompetent cells (Invitrogen) according to manufacturer's protocol.</p>

<h5>Sequencing and sequence data processing.</h5><a id="article1.body1.sec4.sec4.sec2.p1" name="article1.body1.sec4.sec4.sec2.p1"></a><p>16S rDNA clones were bidirectionally sequenced according to standard protocols (<a href="http://www.jgi.doe.gov">www.jgi.doe.gov</a>). Paired reads were assembled using phrap and all clones that failed to assemble, formed contigs less than 1250 bp in length or contained less than 80% Q20 bases (or 70% for nasal, saliva and dust samples) were removed from further processing. These clone sequences were then submitted for alignment by the NAST aligner at greengenes (greengenes.lbl.gov) <a href="#pone.0001862-DeSantis2">[31]</a>. During this process, any sequences that failed to align to known 16S sequences were removed. The aligned sequences were then checked for chimeric clones using the Bellerophon 3 server at greengenes <a href="#pone.0001862-Huber1">[32]</a> and likely chimeras were removed from further analysis. The percentage of chimeric sequences in each library was 0.1% or less for the air libraries and 5–10% for soil and water libraries. A total of 2659, 3063, 2408, 2648, and 2941 high-quality 16s rDNA sequences were generated from 16S rDNA clone libraries constructed from samples collected from air filters at locations 1 and 2, soils from locations 1 and 2, and water from the Singapore River near location 1. A total of 984, 688, 565, and 624 sequences were generated from the nasal swab, saliva, and indoor and outdoor dust samples. Cluster information was entered into the EstimateS program (Version 7, R. K. Colwell, <a href="http://purl.oclc.org/estimates">http://purl.oclc.org/estimates</a>) for rarefaction curve generation and species richness was estimated with the Chao1 estimator.</p>

<h5>Phylotype Determination.</h5><a id="article1.body1.sec4.sec4.sec3.p1" name="article1.body1.sec4.sec4.sec3.p1"></a><p>Cluster representatives were chosen based on phrap quality scores to use the highest quality sequences for further analysis; for large clusters, multiple representatives were chosen. These sequences were then BLAST against an in-house phylogenetically classified 16S sequence database based on greengenes <a href="#pone.0001862-DeSantis3">[33]</a>, and clusters with hits &gt;1000 bp and &gt;95% identity were assigned to the corresponding phylogenetic group. All other sequences were designated unknown.</p>



<h4>5. Metagenomic DNA shotgun sequencing analyses</h4>
<h5>Genomic DNA library construction.</h5><a id="article1.body1.sec4.sec5.sec1.p1" name="article1.body1.sec4.sec5.sec1.p1"></a><p>Approximately 1 µg of genomic DNA from each air sample was partially digested with <em>Alu</em>I (Fermentas, 0.1unit/µg DNA) at room temperature for 10 min. The partially digested genomic DNA fragments were loaded on a 0.7% agarose gel for electrophoresis. The 1–2 kb DNA fragments were excised and purified by QIAquick Gel Extraction Kit. The purified DNA fragments (100–500 ng) were blunted at both ends by Klenow enzyme (Fermentas) and de-phosphorylated by alkaline Phosphatase (New England Biolabs). The DNA inserts were ligated with <em>Eco</em>RV-digested pZErO-1 vector (Invitrogen) using T4 DNA ligase (Invitrogen) at 16°C overnight. The ligation products were transformed by electroporation into Electrocomp TOP10 cells (Invitrogen). The bacteria were plated on imMedia Zeo Agar plates (Invitrogen). Alternatively, ligations were done using pCR-Blunt II-TOPO (Invitrogen) cloning vector and transformed into Electrocomp TOP10 cells (Invitrogen) for selection by Kanamycin. Colonies were picked by Q-Bot (Applied Biosystem) for sequencing.</p>

<h5>DNA sequencing and assembly.</h5><a id="article1.body1.sec4.sec5.sec2.p1" name="article1.body1.sec4.sec5.sec2.p1"></a><p>Raw sequence reads were generated using ABI3730 DNA analyzers, and subjected to base calling and vector/adaptor trimming using PHRED <a href="#pone.0001862-Ewing1">[34]</a>, <a href="#pone.0001862-Ewing2">[35]</a>. We used PHRED score 15 as cutoff value for quality sequences because the most meaningful annotation analysis is BLASTX that has high tolerance to minor sequence errors. The average length of quality sequence (PHRED score ≥15) reads for further analysis is 532 bp. The statistics of the sequencing data is presented in <a href="#pone.0001862.s008">Table S5</a>. For each independent sample, all reads (from all libraries) were assembled using Phrap (-minmatch 30 -maxmatch 55 -minscore 55) <a href="#pone.0001862-Green1">[36]</a>. Contig length and read depth information was extracted from the phrap output to generate <a href="#pone.0001862.s001">Figure S1</a>.</p>
<a id="article1.body1.sec4.sec5.sec2.p2" name="article1.body1.sec4.sec5.sec2.p2"></a><p>Large contig sequences were individually analyzed using BLASTN to search for best matches in NCBI GenBank nr database. Though high score matches were found, the alignments of the contig sequences to subject sequences in the database were often patchy, suggesting that they are closely related, but were not derived from the same genome. Two examples are shown in <a href="#pone.0001862.s002">Figure S2</a>.</p>

<h5>Validation of air contig sequences.</h5><a id="article1.body1.sec4.sec5.sec3.p1" name="article1.body1.sec4.sec5.sec3.p1"></a><p>Large contig sequences from the two air libraries were selected for validation by semi-quantitative PCR. Contig specific PCR primers were designed to generate amplicon products around 200 bp for detection. Universal 16S rDNA primers were used as quantitative control. The quantity of input DNA templates for PCR from each environmental sample was adjusted based on the relative 16S rDNA PCR result. The PCR reaction conditions were as follows: 2.5 µl 10× Taq Buffer with KCl (Fermentas), 1.5 µl MgCl2 (25 mM, Fermentas), 0.5 µl dNTP mix (10 mM, eppendorf), 1.25 µl forward primer and 1.25 µl reverse primer (10 µM each), 0.25 µl Taq DNA polymerase (5 U/µl; Fermentas) and 1–10 ng of template genomic DNA (normalized by comparing the bands of 16S PCR products of different environmental samples) in a total volume of 25 µl. The following cycling parameters were used: 3 min of initial denaturation at 94°C followed by 35 cycles of denaturation (30 s at 94°C), annealing (30 s at 55°C), and elongation (30 s at 72°C), with a final extension at 72°C for 6 min. The PCR results are shown in <a href="#pone.0001862.s003">Figure S3</a>.</p>



<h4>6. DNA sequence homology analyses</h4>
<h5>BLASTN to rDNA database.</h5><a id="article1.body1.sec4.sec6.sec1.p1" name="article1.body1.sec4.sec6.sec1.p1"></a><p>To identify 16S rRNA genes within the genomic shotgun sequence data, vector sequence and low quality sequence with phred scores less than 15 was trimmed from the ends of all reads. The resulting sequences were then subjected to a BLASTN search against an in-house ribosomal RNA database based on Greengenes (greengenes.lbl.gov) <a href="#pone.0001862-DeSantis3">[33]</a> with an e-value threshold of 10<sup>−10</sup>. Hits to non-rRNA features such as internal tRNAs were removed manually. The taxonomic groups identified by this analysis are presented in <a href="#pone.0001862.s009">Table S6</a>.</p>

<h5>BLASTX to GenBank nr database.</h5><a id="article1.body1.sec4.sec6.sec2.p1" name="article1.body1.sec4.sec6.sec2.p1"></a><p>Quality-trimmed reads as described above were subjected to a BLASTX search against the GenBank nr database with an e-value threshold of 10<sup>−8</sup>. Gene IDs for all hits were collected and phylogenetically placed with an in-house script, TaxBreak. The taxonomic units assigned for each of the sequences are summarized in <a href="#pone.0001862.s009">Table S6</a>.</p>

<h5>BLASTN to GenBank nt database.</h5><a id="article1.body1.sec4.sec6.sec3.p1" name="article1.body1.sec4.sec6.sec3.p1"></a><p>Contigs from air assemblies were subjected to a BLASTN search against the GenBank nt database with an e-value threshold of 10<sup>−10</sup>. Hits ≥100 bp and ≥80% identity were analyzed for potential pathogens and commensal organisms.</p>



<h4>7. Metagenomic gene function annotation</h4>
<h5>Protein prediction.</h5><a id="article1.body1.sec4.sec7.sec1.p1" name="article1.body1.sec4.sec7.sec1.p1"></a><p>All assembled contigs as well as all singlet reads that failed to assemble were annotated using Fgenesb (<a href="http://www.softberry.com">www.softberry.com</a>). Proteins were predicted using general open reading frame (ORF) prediction parameters. In total 34,984 and 44,021 ORFs were predicted for the Air-1 and Air-2 libraries, respectively.</p>

<h5>Environmental Gene Tag (EGT) analysis.</h5><a id="article1.body1.sec4.sec7.sec2.p1" name="article1.body1.sec4.sec7.sec2.p1"></a><p>The predicted ORF sequences were subjected to BLASTP against the STRING database (<a href="http://string.embl.de/">http://string.embl.de/</a>) with an e-value of 10<sup>−8</sup> and assigned to the COG or NOG of the best hit. Proteins that did not have hits were then subjected to BLASTP against the NCBI GenBank nr database and annotated with the best hit. The abundance levels of COGs and NOGs within the air data were compared with previously published environmental datasets using Environmental Gene Tag (EGT) analysis <a href="#pone.0001862-Tringe2">[37]</a>. While the assembled data allows for more accurate gene predictions, the parameter of interest is the number of independent clones encoding homologs of a given functional group. To approximate this, each ORF assigned to a COG was weighted by the number of clones contributing to the contig and divided by the length of the contig to account for contig depth. The total number of clone-weighted homologs was then added up for each COG, and, to normalize for sample size, divided by the total number of clone-weighted homologs for all COGs. This gave the fraction of proteins with COG assignments that were assigned to that particular COG. Finally, when comparing among samples, this number was normalized by the sum across all samples, so that overrepresentation could be compared among different COGs.</p>

<h5>Supplementary KEGG analysis.</h5><a id="article1.body1.sec4.sec7.sec3.p1" name="article1.body1.sec4.sec7.sec3.p1"></a><p>The Fgenesb-predicted ORF sequences from the two air samples in this study, and other environmental sequences from other data sources <a href="#pone.0001862-Tringe2">[37]</a>, <a href="#pone.0001862-Venter2">[38]</a> such as seawater, soil and whale fall were subjected to BLASTP against the KEGG database (<a href="http://www.genome.ad.jp/kegg/">http://www.genome.ad.jp/kegg/</a>). At a cutoff of 60 bits, each environmental genomic sequence was mapped to at most one protein in the KEGG database (<a href="#pone.0001862.s011">Table S8</a> in SI).</p>
<a id="article1.body1.sec4.sec7.sec3.p2" name="article1.body1.sec4.sec7.sec3.p2"></a><p>The frequency of a KEGG pathway hit by an environmental sequence cluster is calculated by the number of individual sequences divided by the total number of sequences in each environmental dataset. The representations of KEGG pathways by the 2 air and 3 other environmental datasets were compared using a two-dimensional matrix clustering (<a href="#pone.0001862.s012">Table S9</a> in SI). Hierarchical clustering was performed using complete linkage clustering with uncentered correlation as the similarity metric in the CLUSTER package <a href="#pone.0001862-Eisen1">[39]</a>.</p>


</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001862"></div><a name="pone.0001862.s001" id="pone.0001862.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Length vs. depth plot of all the contigs</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001862.s002" id="pone.0001862.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">BLAST analysis of large air sequence contigs</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.09 MB DOC)</p>
<a name="pone.0001862.s003" id="pone.0001862.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Validation of relative abundance of air sequence contigs in different environments by semi-quantitative PCR</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.26 MB DOC)</p>
<a name="pone.0001862.s004" id="pone.0001862.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Direction of counts and 3H feeding of air filter associated microbes</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001862.s005" id="pone.0001862.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Cell doubling time (days) estimated by two isotope feeding assays</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001862.s006" id="pone.0001862.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s006">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Bacteria isolates from filters identified by culture-based assay</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001862.s007" id="pone.0001862.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s007">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Bacterial phylotypes identified by 16S rDNA clone sequence analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.18 MB DOC)</p>
<a name="pone.0001862.s008" id="pone.0001862.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s008">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Library sequences and assembly statistics</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.12 MB DOC)</p>
<a name="pone.0001862.s009" id="pone.0001862.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s009">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">16S phylotypes identified by BLASTN analysis and taxonomic units assigned by BLASTX analysis of shotgun genomic DNA sequences</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001862.s010" id="pone.0001862.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s010">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Pathogen-related sequences</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0001862.s011" id="pone.0001862.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s011">Table S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Mapping environmental ORFs to KEGG database</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001862.s012" id="pone.0001862.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001862.s012">Table S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">KEGG pathways overrepresented in air</p>
<p class="siDoi">doi:10.1371/journal.pone.0001862.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank Mr. Simon Chong for helping the collection of air filters used in this study, Xiaodong Zhao and Kunde R. Govindarajan for technical support, and the sequencing teams at the DOE Joint Genome Institute and the Genome Institute of Singapore for generating the DNA sequence data.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: ER YR. Performed the experiments: TZ MH FR ST XL FY SI. Analyzed the data: YR WL TZ PT MH FR JB ST XL YY SS. Contributed reagents/materials/analysis tools: ER CW SS JY. Wrote the paper: YR TZ PT ST.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001862-Hoppe1" id="pone.0001862-Hoppe1"></a>Hoppe P, Martinac I (1998) Indoor climate and air quality. Review of current and future topics in the field of ISB study group 10. Int J Biometeorol  42: 1–7.  <ul class="find" data-citedArticleID="1080320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Indoor+climate+and+air+quality.+Review+of+current+and+future+topics+in+the+field+of+ISB+study+group+10.&amp;auth=&amp;atitle=Indoor+climate+and+air+quality.+Review+of+current+and+future+topics+in+the+field+of+ISB+study+group+10." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Indoor+climate+and+air+quality.+Review+of+current+and+future+topics+in+the+field+of+ISB+study+group+10." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Indoor+climate+and+air+quality.+Review+of+current+and+future+topics+in+the+field+of+ISB+study+group+10.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001862-Osterholm1" id="pone.0001862-Osterholm1"></a>Osterholm M, Hedberg C (2005)  Epidemiology of infectious diseases;. In: Mandell G, Bennett J, Dolin R, editors. Philadelphia, PA: Churchill Livingstone. pp. 161–173.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001862-Streifel1" id="pone.0001862-Streifel1"></a>Streifel A (2004)  Design and maintenance of hospital ventilation systems and prevention of airborne nosocomial infections;. In: Mayhall C, editor. Philadelphia, PA: Lippincott Williams &amp; Wilkins. pp. 1577–1589.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001862-Shaffer1" id="pone.0001862-Shaffer1"></a>Shaffer BT, Lighthart B (1997) Survey of Culturable Airborne Bacteria at Four Diverse Locations in Oregon: Urban, Rural, Forest, and Coastal. Microb Ecol  34: 167–177.  <ul class="find" data-citedArticleID="1080340" data-doi="10.1007/s002489900046"><li><a href="http://dx.doi.org/10.1007/s002489900046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Survey+of+Culturable+Airborne+Bacteria+at+Four+Diverse+Locations+in+Oregon%3A+Urban%2C+Rural%2C+Forest%2C+and+Coastal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Survey+of+Culturable+Airborne+Bacteria+at+Four+Diverse+Locations+in+Oregon%3A+Urban%2C+Rural%2C+Forest%2C+and+Coastal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001862-Shelton1" id="pone.0001862-Shelton1"></a>Shelton BG, Kirkland KH, Flanders WD, Morris GK (2002) Profiles of airborne fungi in buildings and outdoor environments in the United States. Appl Environ Microbiol  68: 1743–1753.  <ul class="find" data-citedArticleID="1080342" data-doi="10.1128/aem.68.4.1743-1753.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.4.1743-1753.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profiles+of+airborne+fungi+in+buildings+and+outdoor+environments+in+the+United+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profiles+of+airborne+fungi+in+buildings+and+outdoor+environments+in+the+United+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001862-Wu1" id="pone.0001862-Wu1"></a>Wu PC, Li YY, Chiang CM, Huang CY, Lee CC, et al.  (2005) Changing microbial concentrations are associated with ventilation performance in Taiwan's air-conditioned office buildings. Indoor Air  15: 19–26.  <ul class="find" data-citedArticleID="1080362" data-doi="10.1111/j.1600-0668.2004.00313.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0668.2004.00313.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changing+microbial+concentrations+are+associated+with+ventilation+performance+in+Taiwan%27s+air-conditioned+office+buildings." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changing+microbial+concentrations+are+associated+with+ventilation+performance+in+Taiwan%27s+air-conditioned+office+buildings.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001862-Radosevich1" id="pone.0001862-Radosevich1"></a>Radosevich JL, Wilson WJ, Shinn JH, DeSantis TZ, Andersen GL (2002) Development of a high-volume aerosol collection system for the identification of air-borne micro-organisms. Lett Appl Microbiol  34: 162–167.  <ul class="find" data-citedArticleID="1080336" data-doi="10.1046/j.1472-765x.2002.01048.x"><li><a href="http://dx.doi.org/10.1046/j.1472-765x.2002.01048.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+high-volume+aerosol+collection+system+for+the+identification+of+air-borne+micro-organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+high-volume+aerosol+collection+system+for+the+identification+of+air-borne+micro-organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001862-DeSantis1" id="pone.0001862-DeSantis1"></a>DeSantis TZ, Hugenholtz P, Larsen N, Rojas M, Brodie EL, et al.  (2006) Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB. Appl Environ Microbiol  72: 5069–5072.  <ul class="find" data-citedArticleID="1080298" data-doi="10.1128/aem.03006-05"><li><a href="http://dx.doi.org/10.1128/aem.03006-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Greengenes%2C+a+chimera-checked+16S+rRNA+gene+database+and+workbench+compatible+with+ARB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Greengenes%2C+a+chimera-checked+16S+rRNA+gene+database+and+workbench+compatible+with+ARB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001862-Torsvik1" id="pone.0001862-Torsvik1"></a>Torsvik V, Ovreas L, Thingstad TF (2002) Prokaryotic diversity–magnitude, dynamics, and controlling factors. Science  296: 1064–1066.  <ul class="find" data-citedArticleID="1080348" data-doi="10.1126/science.1071698"><li><a href="http://dx.doi.org/10.1126/science.1071698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prokaryotic+diversity%E2%80%93magnitude%2C+dynamics%2C+and+controlling+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prokaryotic+diversity%E2%80%93magnitude%2C+dynamics%2C+and+controlling+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001862-Gans1" id="pone.0001862-Gans1"></a>Gans J, Wolinsky M, Dunbar J (2005) Computational improvements reveal great bacterial diversity and high metal toxicity in soil. Science  309: 1387–1390.  <ul class="find" data-citedArticleID="1080312" data-doi="10.1126/science.1112665"><li><a href="http://dx.doi.org/10.1126/science.1112665" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+improvements+reveal+great+bacterial+diversity+and+high+metal+toxicity+in+soil." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+improvements+reveal+great+bacterial+diversity+and+high+metal+toxicity+in+soil.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001862-Venter1" id="pone.0001862-Venter1"></a>Venter JC, Remington K, Heidelberg JF, Halpern AL, Rusch D, et al.  (2004) Environmental genome shotgun sequencing of the Sargasso Sea. Science  304: 66–74.  <ul class="find" data-citedArticleID="1080356" data-doi="10.1126/science.1093857"><li><a href="http://dx.doi.org/10.1126/science.1093857" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001862-Tringe1" id="pone.0001862-Tringe1"></a>Tringe SG, von Mering C, Kobayashi A, Salamov AA, Chen K, et al.  (2005) Comparative metagenomics of microbial communities. Science  308: 554–557.  <ul class="find" data-citedArticleID="1080350" data-doi="10.1126/science.1107851"><li><a href="http://dx.doi.org/10.1126/science.1107851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+metagenomics+of+microbial+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+metagenomics+of+microbial+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001862-Abraham1" id="pone.0001862-Abraham1"></a>Abraham WR, Strompl C, Meyer H, Lindholst S, Moore ER, et al.  (1999) Phylogeny and polyphasic taxonomy of Caulobacter species. Proposal of Maricaulis gen. nov. with Maricaulis maris (Poindexter) comb. nov. as the type species, and emended description of the genera Brevundimonas and Caulobacter. Int J Syst Bacteriol  49 Pt 3: 1053–1073.  <ul class="find" data-citedArticleID="1080290" data-doi="10.1099/00207713-49-3-1053"><li><a href="http://dx.doi.org/10.1099/00207713-49-3-1053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+and+polyphasic+taxonomy+of+Caulobacter+species.+Proposal+of+Maricaulis+gen.+nov.+with+Maricaulis+maris+%28Poindexter%29+comb.+nov.+as+the+type+species%2C+and+emended+description+of+the+genera+Brevundimonas+and+Caulobacter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+and+polyphasic+taxonomy+of+Caulobacter+species.+Proposal+of+Maricaulis+gen.+nov.+with+Maricaulis+maris+%28Poindexter%29+comb.+nov.+as+the+type+species%2C+and+emended+description+of+the+genera+Brevundimonas+and+Caulobacter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001862-Kawamura1" id="pone.0001862-Kawamura1"></a>Kawamura Y, Li Y, Liu H, Huang X, Li Z, et al.  (2001) Bacterial population in Russian space station “Mir”. Microbiol Immunol  45: 819–828.  <ul class="find" data-citedArticleID="1080324" data-doi="10.1111/j.1348-0421.2001.tb01321.x"><li><a href="http://dx.doi.org/10.1111/j.1348-0421.2001.tb01321.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+population+in+Russian+space+station+%E2%80%9CMir%E2%80%9D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+population+in+Russian+space+station+%E2%80%9CMir%E2%80%9D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001862-Gilad1" id="pone.0001862-Gilad1"></a>Gilad J, Borer A, Peled N, Riesenberg K, Tager S, et al.  (2000) Hospital-acquired Brevundimonas vesicularis septicaemia following open-heart surgery: case report and literature review. Scand J Infect Dis  32: 90–91.  <ul class="find" data-citedArticleID="1080314" data-doi="10.1080/00365540050164290"><li><a href="http://dx.doi.org/10.1080/00365540050164290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hospital-acquired+Brevundimonas+vesicularis+septicaemia+following+open-heart+surgery%3A+case+report+and+literature+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hospital-acquired+Brevundimonas+vesicularis+septicaemia+following+open-heart+surgery%3A+case+report+and+literature+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001862-Han1" id="pone.0001862-Han1"></a>Han XY, Andrade RA (2005) Brevundimonas diminuta infections and its resistance to fluoroquinolones. J Antimicrob Chemother  55: 853–859.  <ul class="find" data-citedArticleID="1080318" data-doi="10.1093/jac/dki139"><li><a href="http://dx.doi.org/10.1093/jac/dki139" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brevundimonas+diminuta+infections+and+its+resistance+to+fluoroquinolones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Brevundimonas+diminuta+infections+and+its+resistance+to+fluoroquinolones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001862-Valdezate1" id="pone.0001862-Valdezate1"></a>Valdezate S, Vindel A, Martin-Davila P, Del Saz BS, Baquero F, et al.  (2004) High genetic diversity among Stenotrophomonas maltophilia strains despite their originating at a single hospital. J Clin Microbiol  42: 693–699.  <ul class="find" data-citedArticleID="1080354" data-doi="10.1128/jcm.42.2.693-699.2003"><li><a href="http://dx.doi.org/10.1128/jcm.42.2.693-699.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+genetic+diversity+among+Stenotrophomonas+maltophilia+strains+despite+their+originating+at+a+single+hospital." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+genetic+diversity+among+Stenotrophomonas+maltophilia+strains+despite+their+originating+at+a+single+hospital.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001862-Xu1" id="pone.0001862-Xu1"></a>Xu J, Smyth CL, Buchanan JA, Dolan A, Rooney PJ, et al.  (2004) Employment of 16 S rDNA gene sequencing techniques to identify culturable environmental eubacteria in a tertiary referral hospital. J Hosp Infect  57: 52–58.  <ul class="find" data-citedArticleID="1080364" data-doi="10.1016/j.jhin.2004.01.011"><li><a href="http://dx.doi.org/10.1016/j.jhin.2004.01.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Employment+of+16+S+rDNA+gene+sequencing+techniques+to+identify+culturable+environmental+eubacteria+in+a+tertiary+referral+hospital." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Employment+of+16+S+rDNA+gene+sequencing+techniques+to+identify+culturable+environmental+eubacteria+in+a+tertiary+referral+hospital.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001862-Paster1" id="pone.0001862-Paster1"></a>Paster BJ, Falkler WA Jr, Jr, Enwonwu CO, Idigbe EO, Savage KO, et al.  (2002) Prevalent bacterial species and novel phylotypes in advanced noma lesions. J Clin Microbiol  40: 2187–2191.  <ul class="find" data-citedArticleID="1080334" data-doi="10.1128/jcm.40.6.2187-2191.2002"><li><a href="http://dx.doi.org/10.1128/jcm.40.6.2187-2191.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalent+bacterial+species+and+novel+phylotypes+in+advanced+noma+lesions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalent+bacterial+species+and+novel+phylotypes+in+advanced+noma+lesions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001862-Ravaoarinoro1" id="pone.0001862-Ravaoarinoro1"></a>Ravaoarinoro M, Therrien C (1999) Beta-lactamases and outer membrane investigations in beta-lactam-resistant Comamonas acidovorans strains. Int J Antimicrob Agents  12: 27–31.  <ul class="find" data-citedArticleID="1080338" data-doi="10.1016/s0924-8579(98)00095-8"><li><a href="http://dx.doi.org/10.1016/s0924-8579(98)00095-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beta-lactamases+and+outer+membrane+investigations+in+beta-lactam-resistant+Comamonas+acidovorans+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beta-lactamases+and+outer+membrane+investigations+in+beta-lactam-resistant+Comamonas+acidovorans+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001862-Angenent1" id="pone.0001862-Angenent1"></a>Angenent LT, Kelley ST, St Amand A, Pace NR, Hernandez MT (2005) Molecular identification of potential pathogens in water and air of a hospital therapy pool. Proc Natl Acad Sci U S A  102: 4860–4865.  <ul class="find" data-citedArticleID="1080292" data-doi="10.1073/pnas.0501235102"><li><a href="http://dx.doi.org/10.1073/pnas.0501235102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+identification+of+potential+pathogens+in+water+and+air+of+a+hospital+therapy+pool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+identification+of+potential+pathogens+in+water+and+air+of+a+hospital+therapy+pool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001862-Koski1" id="pone.0001862-Koski1"></a>Koski LB, Golding GB (2001) The closest BLAST hit is often not the nearest neighbor. J Mol Evol  52: 540–542.  <ul class="find" data-citedArticleID="1080326" data-doi="10.1007/s002390010184"><li><a href="http://dx.doi.org/10.1007/s002390010184" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+closest+BLAST+hit+is+often+not+the+nearest+neighbor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+closest+BLAST+hit+is+often+not+the+nearest+neighbor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001862-Tatusov1" id="pone.0001862-Tatusov1"></a>Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, et al.  (2003) The COG database: an updated version includes eukaryotes. BMC Bioinformatics  4: 41.  <ul class="find" data-citedArticleID="1080346"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+COG+database%3A+an+updated+version+includes+eukaryotes.&amp;auth=&amp;atitle=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+COG+database%3A+an+updated+version+includes+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001862-vonMering1" id="pone.0001862-vonMering1"></a>von Mering C, Jensen LJ, Snel B, Hooper SD, Krupp M, et al.  (2005) STRING: known and predicted protein-protein associations, integrated and transferred across organisms. Nucleic Acids Res  33: D433–437.  <ul class="find" data-citedArticleID="1080366" data-doi="10.1093/nar/gki005"><li><a href="http://dx.doi.org/10.1093/nar/gki005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=STRING%3A+known+and+predicted+protein-protein+associations%2C+integrated+and+transferred+across+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22STRING%3A+known+and+predicted+protein-protein+associations%2C+integrated+and+transferred+across+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001862-OjanenReuhs1" id="pone.0001862-OjanenReuhs1"></a>Ojanen-Reuhs T, Kalkkinen N, Westerlund-Wikstrom B, van Doorn J, Haahtela K, et al.  (1997) Characterization of the fimA gene encoding bundle-forming fimbriae of the plant pathogen Xanthomonas campestris pv. vesicatoria. J Bacteriol  179: 1280–1290.  <ul class="find" data-citedArticleID="1080330"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+the+fimA+gene+encoding+bundle-forming+fimbriae+of+the+plant+pathogen+Xanthomonas+campestris+pv.+vesicatoria.&amp;auth=&amp;atitle=Characterization+of+the+fimA+gene+encoding+bundle-forming+fimbriae+of+the+plant+pathogen+Xanthomonas+campestris+pv.+vesicatoria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+fimA+gene+encoding+bundle-forming+fimbriae+of+the+plant+pathogen+Xanthomonas+campestris+pv.+vesicatoria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+fimA+gene+encoding+bundle-forming+fimbriae+of+the+plant+pathogen+Xanthomonas+campestris+pv.+vesicatoria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001862-Cook1" id="pone.0001862-Cook1"></a>Cook GM, Loder C, Soballe B, Stafford GP, Membrillo-Hernandez J, et al.  (1998) A factor produced by Escherichia coli K-12 inhibits the growth of E. coli mutants defective in the cytochrome bd quinol oxidase complex: enterochelin rediscovered. Microbiology  144 (Pt 12): 3297–3308.  <ul class="find" data-citedArticleID="1080296" data-doi="10.1099/00221287-144-12-3297"><li><a href="http://dx.doi.org/10.1099/00221287-144-12-3297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+factor+produced+by+Escherichia+coli+K-12+inhibits+the+growth+of+E.+coli+mutants+defective+in+the+cytochrome+bd+quinol+oxidase+complex%3A+enterochelin+rediscovered." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+factor+produced+by+Escherichia+coli+K-12+inhibits+the+growth+of+E.+coli+mutants+defective+in+the+cytochrome+bd+quinol+oxidase+complex%3A+enterochelin+rediscovered.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001862-Way1" id="pone.0001862-Way1"></a>Way SS, Sallustio S, Magliozzo RS, Goldberg MB (1999) Impact of either elevated or decreased levels of cytochrome bd expression on Shigella flexneri virulence. J Bacteriol  181: 1229–1237.  <ul class="find" data-citedArticleID="1080360"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Impact+of+either+elevated+or+decreased+levels+of+cytochrome+bd+expression+on+Shigella+flexneri+virulence.&amp;auth=&amp;atitle=Impact+of+either+elevated+or+decreased+levels+of+cytochrome+bd+expression+on+Shigella+flexneri+virulence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+either+elevated+or+decreased+levels+of+cytochrome+bd+expression+on+Shigella+flexneri+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+either+elevated+or+decreased+levels+of+cytochrome+bd+expression+on+Shigella+flexneri+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001862-Endley1" id="pone.0001862-Endley1"></a>Endley S, McMurray D, Ficht TA (2001) Interruption of the cydB locus in Brucella abortus attenuates intracellular survival and virulence in the mouse model of infection. J Bacteriol  183: 2454–2462.  <ul class="find" data-citedArticleID="1080306" data-doi="10.1128/jb.183.8.2454-2462.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.8.2454-2462.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interruption+of+the+cydB+locus+in+Brucella+abortus+attenuates+intracellular+survival+and+virulence+in+the+mouse+model+of+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interruption+of+the+cydB+locus+in+Brucella+abortus+attenuates+intracellular+survival+and+virulence+in+the+mouse+model+of+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001862-Brodie1" id="pone.0001862-Brodie1"></a>Brodie EL, Desantis TZ, Parker JP, Zubietta IX, Piceno YM, et al.  (2007) Urban aerosols harbor diverse and dynamic bacterial populations. Proc Natl Acad Sci U S A  104: 299–304.  <ul class="find" data-citedArticleID="1080294" data-doi="10.1073/pnas.0608255104"><li><a href="http://dx.doi.org/10.1073/pnas.0608255104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Urban+aerosols+harbor+diverse+and+dynamic+bacterial+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Urban+aerosols+harbor+diverse+and+dynamic+bacterial+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001862-Lane1" id="pone.0001862-Lane1"></a>Lane DJ, Pace B, Olsen GJ, Stahl DA, Sogin ML, et al.  (1985) Rapid determination of 16S ribosomal RNA sequences for phylogenetic analyses. Proc Natl Acad Sci U S A  82: 6955–6959.  <ul class="find" data-citedArticleID="1080328" data-doi="10.1073/pnas.82.20.6955"><li><a href="http://dx.doi.org/10.1073/pnas.82.20.6955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+determination+of+16S+ribosomal+RNA+sequences+for+phylogenetic+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+determination+of+16S+ribosomal+RNA+sequences+for+phylogenetic+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001862-DeSantis2" id="pone.0001862-DeSantis2"></a>DeSantis TZ Jr, Hugenholtz P, Keller K, Brodie EL, Larsen N, et al.  (2006) NAST: a multiple sequence alignment server for comparative analysis of 16S rRNA genes. Nucleic Acids Res  34: W394–399.  <ul class="find" data-citedArticleID="1080300" data-doi="10.1093/nar/gkl244"><li><a href="http://dx.doi.org/10.1093/nar/gkl244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NAST%3A+a+multiple+sequence+alignment+server+for+comparative+analysis+of+16S+rRNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NAST%3A+a+multiple+sequence+alignment+server+for+comparative+analysis+of+16S+rRNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001862-Huber1" id="pone.0001862-Huber1"></a>Huber T, Faulkner G, Hugenholtz P (2004) Bellerophon: a program to detect chimeric sequences in multiple sequence alignments. Bioinformatics  20: 2317–2319.  <ul class="find" data-citedArticleID="1080322" data-doi="10.1093/bioinformatics/bth226"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bellerophon%3A+a+program+to+detect+chimeric+sequences+in+multiple+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bellerophon%3A+a+program+to+detect+chimeric+sequences+in+multiple+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001862-DeSantis3" id="pone.0001862-DeSantis3"></a>DeSantis TZ, Hugenholtz P, Larsen N, Rojas M, Brodie EL, et al.  (2006) Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB. Appl Environ Microbiol  72: 5069–5072.  <ul class="find" data-citedArticleID="1080302" data-doi="10.1128/aem.03006-05"><li><a href="http://dx.doi.org/10.1128/aem.03006-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Greengenes%2C+a+chimera-checked+16S+rRNA+gene+database+and+workbench+compatible+with+ARB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Greengenes%2C+a+chimera-checked+16S+rRNA+gene+database+and+workbench+compatible+with+ARB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001862-Ewing1" id="pone.0001862-Ewing1"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res  8: 186–194.  <ul class="find" data-citedArticleID="1080308" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001862-Ewing2" id="pone.0001862-Ewing2"></a>Ewing B, Hillier L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using phred. I. Accuracy assessment. Genome Res  8: 175–185.  <ul class="find" data-citedArticleID="1080310" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001862-Green1" id="pone.0001862-Green1"></a>Green P (1996)   <a href="http://bozemanmbtwashingtonedu/phrapdocs/phraphtml">http://bozemanmbtwashingtonedu/phrapdocs​/phraphtml</a>.  <ul class="find" data-citedArticleID="1080316" data-doi="10.1057/9780230376779"><li><a href="http://dx.doi.org/10.1057/9780230376779" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001862-Tringe2" id="pone.0001862-Tringe2"></a>Tringe SG, von Mering C, Kobayashi A, Salamov AA, Chen K, et al.  (2005) Comparative metagenomics of microbial communities. Science  308: 554–557.  <ul class="find" data-citedArticleID="1080352" data-doi="10.1126/science.1107851"><li><a href="http://dx.doi.org/10.1126/science.1107851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+metagenomics+of+microbial+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+metagenomics+of+microbial+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001862-Venter2" id="pone.0001862-Venter2"></a>Venter JC, Remington K, Heidelberg JF, Halpern AL, Rusch D, et al.  (2004) Environmental genome shotgun sequencing of the Sargasso Sea. Science  304: 66–74.  <ul class="find" data-citedArticleID="1080358" data-doi="10.1126/science.1093857"><li><a href="http://dx.doi.org/10.1126/science.1093857" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001862-Eisen1" id="pone.0001862-Eisen1"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A  95: 14863–14868.  <ul class="find" data-citedArticleID="1080304" data-doi="10.1073/pnas.95.25.14863"><li><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cluster+analysis+and+display+of+genome-wide+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cluster+analysis+and+display+of+genome-wide+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.XML" value="117603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.PDF" value="323532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g001.PNG_L" value="4434068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g001.PNG_M" value="453752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g001.PNG_S" value="18067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g001.TIF" value="5975016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g001.PNG_I" value="182495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g002.PNG_L" value="3332474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g002.PNG_M" value="174500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g002.PNG_S" value="12470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g002.TIF" value="4256264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g002.PNG_I" value="59332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t001.PNG_L" value="84016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t001.PNG_M" value="45768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t001.PNG_S" value="8939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t001.TIF" value="411464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t001.PNG_I" value="14226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g003.PNG_L" value="468286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g003.PNG_M" value="75932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g003.PNG_S" value="8537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g003.TIF" value="999366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.g003.PNG_I" value="27130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t002.PNG_L" value="362465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t002.PNG_M" value="208336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t002.PNG_S" value="19288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t002.TIF" value="1627120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.t002.PNG_I" value="63898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s001.DOC" value="78848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s002.DOC" value="93696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s003.DOC" value="256512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s004.DOC" value="34816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s005.DOC" value="31744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s006.DOC" value="38400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s007.DOC" value="180736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s008.DOC" value="119296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s009.DOC" value="84992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s010.DOC" value="113152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s011.DOC" value="33792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001862.s012.DOC" value="40448"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001862&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001862" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001862&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001862&volume=&issue=&title=The Airborne Metagenome in an Indoor Urban Environment&author_name=Susannah%20G.%20Tringe%2C%20Tao%20Zhang%2C%20Xuguo%20Liu%2C%20Yiting%20Yu%2C%20Wah%20Heng%20Lee%2C%20Jennifer%20Yap%2C%20Fei%20Yao%2C%20Sim%20Tiow%20Suan%2C%20Seah%20Keng%20Ing%2C%20Matthew%20Haynes%2C%20Forest%20Rohwer%2C%20Chia%20Lin%20Wei%2C%20Patrick%20Tan%2C%20James%20Bristow%2C%20Edward%20M.%20Rubin%2C%20Yijun%20Ruan&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001862" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001862" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001862" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001862&amp;t=The%20Airborne%20Metagenome%20in%20an%20Indoor%20Urban%20Environment" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001862&title=The%20Airborne%20Metagenome%20in%20an%20Indoor%20Urban%20Environment&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001862&amp;title=The%20Airborne%20Metagenome%20in%20an%20Indoor%20Urban%20Environment" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001862" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001862&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Airborne Metagenome in an Indoor Urban Environment';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001862';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Airborne%20Metagenome%20in%20an%20Indoor%20Urban%20Environment http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001862" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001862" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="26548"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26548"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="26548"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="26548"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26548"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="26548"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="26548"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="26548"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6094'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1112'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8270&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
