

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genetic Evidence for a Link Between Glycolysis and DNA Replication</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000447"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000447" />

    <meta name="citation_title" content="Genetic Evidence for a Link Between Glycolysis and DNA Replication"/>
    <meta itemprop="name" content="Genetic Evidence for a Link Between Glycolysis and DNA Replication"/>

      <meta name="citation_author" content="Laurent Jannière"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Danielle Canceill"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Catherine Suski"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Sophie Kanga"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Bérengère Dalmais"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Roxane Lestini"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Anne-Françoise Monnier"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Jérôme Chapuis"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Alexander Bolotin"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Marina Titok"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="Emmanuelle Le Chatelier"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>
      <meta name="citation_author" content="S. Dusko Ehrlich"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France"/>

    <meta name="citation_date" content="2007/5/16"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000447.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e447"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Chromosomal replicases as asymmetric dimers: studies of subunit arrangement and functional consequences.; citation_author=CS McHenry; citation_journal_title=Mol Microbiol; citation_volume=49; citation_number=1; citation_pages=1157-1165; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cellular DNA replicases: components and dynamics at the replication fork.; citation_author=A Johnson; citation_author=M O'Donnell; citation_journal_title=Annu Rev Biochem; citation_volume=74; citation_number=2; citation_pages=283-315; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA replication in the archaea.; citation_author=ER Barry; citation_author=SD Bell; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=70; citation_number=3; citation_pages=876-887; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The replicative polymerases PolC and DnaE are required for theta replication of the Bacillus subtilis plasmid pBS72.; citation_author=M Titok; citation_author=C Suski; citation_author=B Dalmais; citation_author=SD Ehrlich; citation_author=L Janniere; citation_journal_title=Microbiology; citation_volume=152; citation_number=4; citation_pages=1471-1478; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Two essential DNA polymerases at the bacterial replication fork.; citation_author=E Dervyn; citation_author=C Suski; citation_author=R Daniel; citation_author=C Bruand; citation_author=J Chapuis; citation_journal_title=Science; citation_volume=294; citation_number=5; citation_pages=1716-1719; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The DNA replication machine of a gram-positive organism.; citation_author=I Bruck; citation_author=M O'Donnell; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=6; citation_pages=28971-28983; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A two-protein strategy for the functional loading of a cellular replicative DNA helicase.; citation_author=M Velten; citation_author=S McGovern; citation_author=S Marsin; citation_author=SD Ehrlich; citation_author=P Noirot; citation_journal_title=Mol Cell; citation_volume=11; citation_number=7; citation_pages=1-20; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA replication in eukaryotic cells.; citation_author=SP Bell; citation_author=A Dutta; citation_journal_title=Annu Rev Biochem; citation_volume=71; citation_number=8; citation_pages=333-374; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Feedback controls restrain the initiation of Escherichia coli chromosomal replication.; citation_author=T Katayama; citation_journal_title=Mol Microbiol; citation_volume=41; citation_number=9; citation_pages=9-17; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DnaA: controlling the initiation of bacterial DNA replication and more.; citation_author=JM Kaguni; citation_journal_title=Annu Rev Microbiol; citation_volume=60; citation_number=10; citation_pages=351-371; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Timing of synthetic activities in the cell cycle.; citation_author=CE Helmstetter; citation_number=11; citation_pages=1627-1639; citation_date=1996; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Precise determinations of C and D periods by flow cytometry in Escherichia coli K-12 and B/r.; citation_author=O Michelsen; citation_author=MJ Teixeira de Mattos; citation_author=PR Jensen; citation_author=FG Hansen; citation_journal_title=Microbiology; citation_volume=149; citation_number=12; citation_pages=1001-1010; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA replication, the bacterial cell cycle, and cell growth.; citation_author=JW Zyskind; citation_author=DW Smith; citation_journal_title=Cell; citation_volume=69; citation_number=13; citation_pages=5-8; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=ppGpp-mediated regulation of DNA replication and cell division in Escherichia coli.; citation_author=G Schreiber; citation_author=EZ Ron; citation_author=G Glaser; citation_journal_title=Curr Microbiol; citation_volume=30; citation_number=14; citation_pages=27-32; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The stringent response blocks DNA replication outside the ori region in Bacillus subtilis and at the origin in Escherichia coli.; citation_author=A Levine; citation_author=F Vannier; citation_author=M Dehbi; citation_author=G Henckes; citation_author=SJ Seror; citation_journal_title=J Mol Biol; citation_volume=219; citation_number=15; citation_pages=605-613; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The stringent response.; citation_author=M Cashel; citation_author=DR Gentry; citation_author=VH Hernandez; citation_author=D Vinella; citation_number=16; citation_pages=1458-1496; citation_date=1996; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Stress responses and replication of plasmids in bacterial cells.; citation_author=G Wegrzyn; citation_author=A Wegrzyn; citation_journal_title=Microb Cell Fact; citation_volume=1; citation_number=17; citation_pages=2; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes.; citation_author=BP Tu; citation_author=A Kudlicki; citation_author=M Rowicka; citation_author=SL McKnight; citation_journal_title=Science; citation_volume=310; citation_number=18; citation_pages=1152-1158; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genomewide oscillation in transcription gates DNA replication and cell cycle.; citation_author=RR Klevecz; citation_author=J Bolen; citation_author=G Forrest; citation_author=DB Murray; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=19; citation_pages=1200-1205; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Oxygen-dependent regulation of in vivo replication of simian virus 40 DNA is modulated by glucose.; citation_author=HJ Riedinger; citation_author=M van Betteraey-Nikoleit; citation_author=U Hilfrich; citation_author=KH Eisele; citation_author=H Probst; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=20; citation_pages=47122-47130; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The yeast Mcm1 protein is regulated posttranscriptionally by the flux of glycolysis.; citation_author=Y Chen; citation_author=BK Tye; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=21; citation_pages=4631-4639; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mutants of S. cerevisiae defective in the maintenance of minichromosomes.; citation_author=GT Maine; citation_author=P Sinha; citation_author=BK Tye; citation_journal_title=Genetics; citation_volume=106; citation_number=22; citation_pages=365-385; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Mcm1 binds replication origins.; citation_author=VK Chang; citation_author=MJ Fitch; citation_author=JJ Donato; citation_author=TW Christensen; citation_author=AM Merchant; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=23; citation_pages=6093-6100; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mcm1 promotes replication initiation by binding specific elements at replication origins.; citation_author=VK Chang; citation_author=JJ Donato; citation_author=CS Chan; citation_author=BK Tye; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=24; citation_pages=6514-6524; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mcm7, a subunit of the presumptive MCM helicase, modulates its own expression in conjunction with Mcm1.; citation_author=MJ Fitch; citation_author=JJ Donato; citation_author=BK Tye; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=25; citation_pages=25408-25416; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cell cycle-regulated transcription of the CLB2 gene is dependent on Mcm1 and a ternary complex factor.; citation_author=M Maher; citation_author=F Cong; citation_author=D Kindelberger; citation_author=K Nasmyth; citation_author=S Dalton; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=26; citation_pages=3129-3137; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Characterization of the ECB binding complex responsible for the M/G(1)-specific transcription of CLN3 and SWI4.; citation_author=B Mai; citation_author=S Miles; citation_author=LL Breeden; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=27; citation_pages=430-441; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=S phase activation of the histone H2B promoter by OCA-S, a coactivator complex that contains GAPDH as a key component.; citation_author=L Zheng; citation_author=RG Roeder; citation_author=Y Luo; citation_journal_title=Cell; citation_volume=114; citation_number=28; citation_pages=255-266; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Regulation of Cell Division in Escherichia coli: Characterization of Temperature-Sensitive Division Mutants.; citation_author=JN Reeve; citation_author=DJ Groves; citation_author=DJ Clark; citation_journal_title=J Bacteriol; citation_volume=104; citation_number=29; citation_pages=1052-1064; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Process of cellular division in Escherichia coli: physiological study on thermosensitive mutants defective in cell division.; citation_author=M Ricard; citation_author=Y Hirota; citation_journal_title=J Bacteriol; citation_volume=116; citation_number=30; citation_pages=314-322; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of dnaX and dnaY temperature-sensitive mutants of Escherichia coli.; citation_author=JM Henson; citation_author=H Chu; citation_author=CA Irwin; citation_author=JR Walker; citation_journal_title=Genetics; citation_volume=92; citation_number=31; citation_pages=1041-1059; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Two glyceraldehyde-3-phosphate dehydrogenases with opposite physiological roles in a nonphotosynthetic bacterium.; citation_author=S Fillinger; citation_author=S Boschi-Muller; citation_author=S Azza; citation_author=E Dervyn; citation_author=G Branlant; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=32; citation_pages=14031-14037; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequential Peptide Affinity (SPA) system for the identification of mammalian and bacterial protein complexes.; citation_author=M Zeghouf; citation_author=J Li; citation_author=G Butland; citation_author=A Borkowska; citation_author=V Canadien; citation_journal_title=J Proteome Res; citation_volume=3; citation_number=33; citation_pages=463-468; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An expanded view of bacterial DNA replication.; citation_author=MF Noirot-Gros; citation_author=E Dervyn; citation_author=LJ Wu; citation_author=P Mervelet; citation_author=J Errington; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=34; citation_pages=8342-8347; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolution of glycolysis.; citation_author=LA Fothergill-Gilmore; citation_author=PA Michels; citation_journal_title=Prog Biophys Mol Biol; citation_volume=59; citation_number=35; citation_pages=105-235; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The global phylogeny of glycolytic enzymes.; citation_author=B Canback; citation_author=SG Andersson; citation_author=CG Kurland; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=36; citation_pages=6097-6102; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional profiling of the Saccharomyces cerevisiae genome.; citation_author=G Giaever; citation_author=AM Chu; citation_author=L Ni; citation_author=C Connelly; citation_author=L Riles; citation_journal_title=Nature; citation_volume=418; citation_number=37; citation_pages=387-391; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Experimental determination and system level analysis of essential genes in Escherichia coli MG1655.; citation_author=SY Gerdes; citation_author=MD Scholle; citation_author=JW Campbell; citation_author=G Balazsi; citation_author=E Ravasz; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=38; citation_pages=5673-5684; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Essential Bacillus subtilis genes.; citation_author=K Kobayashi; citation_author=SD Ehrlich; citation_author=A Albertini; citation_author=G Amati; citation_author=KK Andersen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=39; citation_pages=4678-4683; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multifaceted roles of glycolytic enzymes.; citation_author=JW Kim; citation_author=CV Dang; citation_journal_title=Trends Biochem Sci; citation_volume=30; citation_number=40; citation_pages=142-150; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multifunctional alpha-enolase: its role in diseases.; citation_author=V Pancholi; citation_journal_title=Cell Mol Life Sci; citation_volume=58; citation_number=41; citation_pages=902-920; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=New insights into an old protein: the functional diversity of mammalian glyceraldehyde-3-phosphate dehydrogenase.; citation_author=MA Sirover; citation_journal_title=Biochim Biophys Acta; citation_volume=1432; citation_number=42; citation_pages=159-184; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Functional identity of a primer recognition protein as phosphoglycerate kinase.; citation_author=HK Jindal; citation_author=JK Vishwanatha; citation_journal_title=J Biol Chem; citation_volume=265; citation_number=43; citation_pages=6540-6543; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A DEAD-box RNA helicase in the Escherichia coli RNA degradosome.; citation_author=B Py; citation_author=CF Higgins; citation_author=HM Krisch; citation_author=AJ Carpousis; citation_journal_title=Nature; citation_volume=381; citation_number=44; citation_pages=169-172; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Modulation of DNA polymerases alpha, delta and epsilon by lactate dehydrogenase and 3-phosphoglycerate kinase.; citation_author=O Popanda; citation_author=G Fox; citation_author=HW Thielmann; citation_journal_title=Biochim Biophys Acta; citation_volume=1397; citation_number=45; citation_pages=102-117; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Phosphoglycerate kinase acts in tumour angiogenesis as a disulphide reductase.; citation_author=AJ Lay; citation_author=XM Jiang; citation_author=O Kisker; citation_author=E Flynn; citation_author=A Underwood; citation_journal_title=Nature; citation_volume=408; citation_number=46; citation_pages=869-873; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Glyceraldehyde-3-phosphate dehydrogenase is a GABAA receptor kinase linking glycolysis to neuronal inhibition.; citation_author=JJ Laschet; citation_author=F Minier; citation_author=I Kurcewicz; citation_author=MH Bureau; citation_author=S Trottier; citation_journal_title=J Neurosci; citation_volume=24; citation_number=47; citation_pages=7614-7622; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Methylglyoxal modification of mSin3A links glycolysis to angiopoietin-2 transcription.; citation_author=D Yao; citation_author=T Taguchi; citation_author=T Matsumura; citation_author=R Pestell; citation_author=D Edelstein; citation_journal_title=Cell; citation_volume=124; citation_number=48; citation_pages=275-286; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=BAD and glucokinase reside in a mitochondrial complex that integrates glycolysis and apoptosis.; citation_author=NN Danial; citation_author=CF Gramm; citation_author=L Scorrano; citation_author=CY Zhang; citation_author=S Krauss; citation_journal_title=Nature; citation_volume=424; citation_number=49; citation_pages=952-956; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nature of the SOS-inducing signal in Escherichia coli. The involvement of DNA replication.; citation_author=M Sassanfar; citation_author=JW Roberts; citation_journal_title=J Mol Biol; citation_volume=212; citation_number=50; citation_pages=79-96; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Chaperonin-mediated protein folding.; citation_author=D Thirumalai; citation_author=GH Lorimer; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=30; citation_number=51; citation_pages=245-269; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multiple effects of trehalose on protein folding in vitro and in vivo.; citation_author=MA Singer; citation_author=S Lindquist; citation_journal_title=Mol Cell; citation_volume=1; citation_number=52; citation_pages=639-648; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Mechanism of substrate recognition by Hsp70 chaperones.; citation_author=A Erbse; citation_author=MP Mayer; citation_author=B Bukau; citation_journal_title=Biochem Soc Trans; citation_volume=32; citation_number=53; citation_pages=617-621; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A genetic selection for supercoiling mutants of Escherichia coli reveals proteins implicated in chromosome structure.; citation_author=CD Hardy; citation_author=NR Cozzarelli; citation_journal_title=Mol Microbiol; citation_volume=57; citation_number=54; citation_pages=1636-1652; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The organization and inheritance of the mitochondrial genome.; citation_author=XJ Chen; citation_author=RA Butow; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=55; citation_pages=815-825; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A bacterial G protein-mediated response to replication arrest.; citation_author=JJ Foti; citation_author=J Schienda; citation_author=VA Sutera; citation_author=ST Lovett; citation_journal_title=Mol Cell; citation_volume=17; citation_number=56; citation_pages=549-560; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple pathways process stalled replication forks.; citation_author=B Michel; citation_author=G Grompone; citation_author=MJ Flores; citation_author=V Bidnenko; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=57; citation_pages=12783-12788; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomic transcriptional response to loss of chromosomal supercoiling in Escherichia coli.; citation_author=BJ Peter; citation_author=J Arsuaga; citation_author=AM Breier; citation_author=AB Khodursky; citation_author=PO Brown; citation_journal_title=Genome Biol; citation_volume=5; citation_number=58; citation_pages=R87; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A novel regulatory mechanism couples deoxyribonucleotide synthesis and DNA replication in Escherichia coli.; citation_author=S Gon; citation_author=JE Camara; citation_author=HK Klungsoyr; citation_author=E Crooke; citation_author=K Skarstad; citation_journal_title=Embo J; citation_volume=25; citation_number=59; citation_pages=1137-1147; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Interactions of glutaredoxins, ribonucleotide reductase, and components of the DNA replication system of Escherichia coli.; citation_author=R Ortenberg; citation_author=S Gon; citation_author=A Porat; citation_author=J Beckwith; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=60; citation_pages=7439-7444; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional taxonomy of bacterial hyperstructures.; citation_author=V Norris; citation_author=T den Blaauwen; citation_author=A Cabin-Flaman; citation_author=RH Doi; citation_author=R Harshey; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=71; citation_number=61; citation_pages=230-253; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Replication fork and SeqA focus distributions in Escherichia coli suggest a replication hyperstructure dependent on nucleotide metabolism.; citation_author=F Molina; citation_author=K Skarstad; citation_journal_title=Mol Microbiol; citation_volume=52; citation_number=62; citation_pages=1597-1612; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ribonucleoside diphosphate reductase is a component of the replication hyperstructure in Escherichia coli.; citation_author=EC Guzman; citation_author=JL Caballero; citation_author=A Jimenez-Sanchez; citation_journal_title=Mol Microbiol; citation_volume=43; citation_number=63; citation_pages=487-495; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Reduced initiation frequency from oriC restores viability of a temperature-sensitive Escherichia coli replisome mutant.; citation_author=O Skovgaard; citation_author=A Lobner-Olesen; citation_journal_title=Microbiology; citation_volume=151; citation_number=64; citation_pages=963-973; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Interaction of the Escherichia coli dnaA initiation protein with the dnaZ polymerization protein in vivo.; citation_author=JR Walker; citation_author=JA Ramsey; citation_author=WG Haldenwang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=79; citation_number=65; citation_pages=3340-3344; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Suppression of temperature-sensitive chromosome replication of an Escherichia coli dnaX(Ts) mutant by reduction of initiation efficiency.; citation_author=A Blinkova; citation_author=MJ Hermandson; citation_author=JR Walker; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=66; citation_pages=3583-3595; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli DnaA protein: specific biochemical defects of mutant DnaAs reduce initiation frequency to suppress a temperature-sensitive dnaX mutation.; citation_author=JR Walker; citation_author=KA Severson; citation_author=MJ Hermandson; citation_author=A Blinkova; citation_author=KM Carr; citation_journal_title=Biochimie; citation_volume=88; citation_number=67; citation_pages=1-10; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The role of replication initiation control in promoting survival of replication fork damage.; citation_author=VA Sutera; citation_author=ST Lovett; citation_journal_title=Mol Microbiol; citation_volume=60; citation_number=68; citation_pages=229-239; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome engineering reveals large dispensable regions in Bacillus subtilis.; citation_author=H Westers; citation_author=R Dorenbos; citation_author=JM van Dijl; citation_author=J Kabel; citation_author=T Flanagan; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=69; citation_pages=2076-2090; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis.; citation_author=F Kunst; citation_author=N Ogasawara; citation_author=I Moszer; citation_author=AM Albertini; citation_author=G Alloni; citation_journal_title=Nature; citation_volume=390; citation_number=70; citation_pages=249-256; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Catabolite repression mediated by the CcpA protein in Bacillus subtilis: novel modes of regulation revealed by whole-genome analyses.; citation_author=MS Moreno; citation_author=BL Schneider; citation_author=RR Maile; citation_author=W Weyler; citation_author=MH Saier; citation_journal_title=Mol Microbiol; citation_volume=39; citation_number=71; citation_pages=1366-1381; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Bacillus subtilis metabolism and energetics in carbon-limited and excess-carbon chemostat culture.; citation_author=M Dauner; citation_author=T Storni; citation_author=U Sauer; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=72; citation_pages=7308-7317; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Invariance of the nucleoside triphosphate pools of Escherichia coli with growth rate.; citation_author=C Petersen; citation_author=LB Moller; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=73; citation_pages=3931-3935; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Relationship between growth rate and ATP concentration in Escherichia coli: a bioassay for available cellular ATP.; citation_author=DA Schneider; citation_author=RL Gourse; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=74; citation_pages=8262-8268; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Regulation of chromosome replication.; citation_author=TJ Kelly; citation_author=GW Brown; citation_journal_title=Annu Rev Biochem; citation_volume=69; citation_number=75; citation_pages=829-880; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Eukaryotic MCM proteins: beyond replication initiation.; citation_author=SL Forsburg; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=68; citation_number=76; citation_pages=109-131; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Common domains in the initiators of DNA replication in Bacteria, Archaea and Eukarya: combined structural, functional and phylogenetic perspectives.; citation_author=R Giraldo; citation_journal_title=FEMS Microbiol Rev; citation_volume=26; citation_number=77; citation_pages=533-554; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Bacterial single-stranded DNA-binding proteins are phosphorylated on tyrosine.; citation_author=I Mijakovic; citation_author=D Petranovic; citation_author=B Macek; citation_author=T Cepo; citation_author=M Mann; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=78; citation_pages=1588-1596; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Enzyme organization in DNA precursor biosynthesis.; citation_author=CK Mathews; citation_journal_title=Prog Nucleic Acid Res Mol Biol; citation_volume=44; citation_number=79; citation_pages=167-203; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Regulatory functions of nuclear hexokinase1 complex in glucose signaling.; citation_author=YH Cho; citation_author=SD Yoo; citation_author=J Sheen; citation_journal_title=Cell; citation_volume=127; citation_number=80; citation_pages=579-589; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiple mutations and cancer.; citation_author=LA Loeb; citation_author=KR Loeb; citation_author=JP Anderson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=81; citation_pages=776-781; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Why do cancers have high aerobic glycolysis?; citation_author=RA Gatenby; citation_author=RJ Gillies; citation_journal_title=Nat Rev Cancer; citation_volume=4; citation_number=82; citation_pages=891-899; citation_date=2004; " />
      <meta name="citation_reference" content="citation_number=83; citation_date=2000; citation_publisher=John Wiley & Sons, LTD; " />
      <meta name="citation_reference" content="citation_title=A vector for systematic gene inactivation in Bacillus subtilis.; citation_author=V Vagner; citation_author=E Dervyn; citation_author=SD Ehrlich; citation_journal_title=Microbiology; citation_volume=144; citation_number=84; citation_pages=3097-3104; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Development of a new integration site within the Bacillus subtilis chromosome and construction of compatible expression cassettes.; citation_author=B Hartl; citation_author=W Wehrl; citation_author=T Wiegert; citation_author=G Homuth; citation_author=W Schumann; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=85; citation_pages=2696-2699; citation_date=2001; " />
      <meta name="citation_reference" content="citation_number=86; citation_date=1990; citation_publisher=John Wiley & Sons, LTD; " />
      <meta name="citation_reference" content="citation_title=Regulation of the central glycolytic genes in Bacillus subtilis: binding of the repressor CggR to its single DNA target sequence is modulated by fructose-1,6-bisphosphate.; citation_author=T Doan; citation_author=S Aymerich; citation_journal_title=Mol Microbiol; citation_volume=47; citation_number=87; citation_pages=1709-1721; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Contributions of ATP, GTP, and redox state to nutritional stress activation of the Bacillus subtilis sigmaB transcription factor.; citation_author=S Zhang; citation_author=WG Haldenwang; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=88; citation_pages=7554-6750; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacillus subtilis functional genomics: global characterization of the stringent response by proteome and transcriptome analysis.; citation_author=C Eymann; citation_author=G Homuth; citation_author=C Scharf; citation_author=M Hecker; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=89; citation_pages=2500-2520; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Stress-induced activation of the sigma B transcription factor of Bacillus subtilis.; citation_author=SA Boylan; citation_author=AR Redfield; citation_author=MS Brody; citation_author=CW Price; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=90; citation_pages=7931-7937; citation_date=1993; " />
      <meta name="citation_reference" content="citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=91; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Tn10-derived transposons active in Bacillus subtilis.; citation_author=MA Petit; citation_author=C Bruand; citation_author=L Janniere; citation_author=SD Ehrlich; citation_journal_title=J Bacteriol; citation_volume=172; citation_number=92; citation_pages=6736-6740; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Involvement of DnaE, the second replicative DNA polymerase from Bacillus subtilis, in DNA mutagenesis.; citation_author=E Le Chatelier; citation_author=OJ Becherel; citation_author=E D'Alencon; citation_author=D CanceilI; citation_author=SD Ehrlich; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=93; citation_pages=1757-1767; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=T Bücher; citation_author=G Pfleiderer; citation_journal_title=Methods Enzymol; citation_volume=1; citation_number=94; citation_pages=435; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=Pyruvate kinase of Bacillus licheniformis.; citation_author=FW Tuominen; citation_author=RW Bernlohr; citation_journal_title=Methods Enzymol; citation_volume=42; citation_number=95; citation_pages=157-166; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=PcrA is an essential DNA helicase of Bacillus subtilis fulfilling functions both in repair and rolling-circle replication.; citation_author=MA Petit; citation_author=E Dervyn; citation_author=M Rose; citation_author=KD Entian; citation_author=S McGovern; citation_journal_title=Mol Microbiol; citation_volume=29; citation_number=96; citation_pages=261-273; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000447" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genetic Evidence for a Link Between Glycolysis and DNA Replication"/>
    <meta name="twitter:description" content="BackgroundA challenging goal in biology is to understand how the principal cellular functions are integrated so that cells achieve viability and optimal fitness in a wide range of nutritional conditions.Methodology/Principal FindingsWe report here a tight link between glycolysis and DNA synthesis. The link, discovered during an analysis of suppressors of thermosensitive replication mutants in bacterium Bacillus subtilis, is very strong as some metabolic alterations fully restore viability to replication mutants in which a lethal arrest of DNA synthesis otherwise occurs at a high, restrictive, temperature. Full restoration of viability by such alterations was limited to cells with mutations in three elongation factors (the lagging strand DnaE polymerase, the primase and the helicase) out of a large set of thermosensitive mutants affected in most of the replication proteins. Restoration of viability resulted, at least in part, from maintenance of replication protein activity at high temperature. Physiological studies suggested that this restoration depended on the activity of the three-carbon part of the glycolysis/gluconeogenesis pathway and occurred in both glycolytic and gluconeogenic regimens. Restoration took place abruptly over a narrow range of expression of genes in the three-carbon part of glycolysis. However, the absolute value of this range varied greatly with the allele in question. Finally, restoration of cell viability did not appear to be the result of a decrease in growth rate or an induction of major stress responses.Conclusions/SignificanceOur findings provide the first evidence for a genetic system that connects DNA chain elongation to glycolysis. Its role may be to modulate some aspect of DNA synthesis in response to the energy provided by the environment and the underlying mechanism is discussed. It is proposed that related systems are ubiquitous."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000447.g007"/>

  <meta property="og:title" content="Genetic Evidence for a Link Between Glycolysis and DNA Replication" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000447" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8068'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8561'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8256&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000447">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000447" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000447&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genetic Evidence for a Link Between Glycolysis and DNA Replication
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laurent Jannière
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:laurent.janniere@jouy.inra.fr">laurent.janniere@jouy.inra.fr</a></p>

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Danielle Canceill, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Catherine Suski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>

                <p>
                    Current address: Memorial Sloan-Kettering Cancer Center, New York, New York, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sophie Kanga, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>

                <p>
                    Current address: Laboratoire Régulation de l'Expression Génique chez les Microorganismes, CNRS-UPR9073, Institut de Biologie Physico-Chimique, Paris, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bérengère Dalmais, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roxane Lestini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>

                <p>
                    Current address: Centre de Génétique Moléculaire CNRS Bâtiment 26, Gif-sur-Yvette, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anne-Françoise Monnier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jérôme Chapuis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>

                <p>
                    Current address: Laboratoire de Virologie Immunologie Moléculaire, INRA, Jouy en Josas, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexander Bolotin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marina Titok, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>

                <p>
                    Current address: Belarussian State University, Biological Faculty, Department of Genetics and Biotechnology, Minsk, Belarus
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Emmanuelle Le Chatelier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            S. Dusko Ehrlich
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Microbienne, INRA, Jouy en Josas, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 16, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000447</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000447" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000447" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000447" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000447" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000447-g001" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-t001" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-t002" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-g002" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-g003" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-g004" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-g005" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-t003" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-g006" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-t004" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000447-g007" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000447">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000447" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>A challenging goal in biology is to understand how the principal cellular functions are integrated so that cells achieve viability and optimal fitness in a wide range of nutritional conditions.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We report here a tight link between glycolysis and DNA synthesis. The link, discovered during an analysis of suppressors of thermosensitive replication mutants in bacterium <em>Bacillus subtilis</em>, is very strong as some metabolic alterations fully restore viability to replication mutants in which a lethal arrest of DNA synthesis otherwise occurs at a high, restrictive, temperature. Full restoration of viability by such alterations was limited to cells with mutations in three elongation factors (the lagging strand DnaE polymerase, the primase and the helicase) out of a large set of thermosensitive mutants affected in most of the replication proteins. Restoration of viability resulted, at least in part, from maintenance of replication protein activity at high temperature. Physiological studies suggested that this restoration depended on the activity of the three-carbon part of the glycolysis/gluconeogenesis pathway and occurred in both glycolytic and gluconeogenic regimens. Restoration took place abruptly over a narrow range of expression of genes in the three-carbon part of glycolysis. However, the absolute value of this range varied greatly with the allele in question. Finally, restoration of cell viability did not appear to be the result of a decrease in growth rate or an induction of major stress responses.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our findings provide the first evidence for a genetic system that connects DNA chain elongation to glycolysis. Its role may be to modulate some aspect of DNA synthesis in response to the energy provided by the environment and the underlying mechanism is discussed. It is proposed that related systems are ubiquitous.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Jannière L, Canceill D, Suski C, Kanga S, Dalmais B, et al.  (2007) Genetic Evidence for a Link Between Glycolysis and DNA Replication. PLoS ONE 2(5):
          e447.
            doi:10.1371/journal.pone.0000447</p><p><strong>Academic Editor: </strong>Steve Sandler, University of Massachusetts, United States of America</p><p><strong>Received:</strong> January 23, 2007; <strong>Accepted:</strong> April 23, 2007; <strong>Published:</strong> May 16, 2007</p><p><strong>Copyright:</strong> © 2007 Janniere et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>We do not have any financial interest in this manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The replisome is a multiprotein machine that replicates DNA in living organisms <a href="#pone.0000447-McHenry1">[1]</a>–<a href="#pone.0000447-Barry1">[3]</a>. In the Gram-positive bacterium <em>Bacillus subtilis</em> and related micro-organisms, this machine contains two different polymerases which are thought to be specialized in leading (PolC) and lagging (DnaE) strand polymerization <a href="#pone.0000447-Titok1">[4]</a>, <a href="#pone.0000447-Dervyn1">[5]</a>. For processivity, the polymerases bind to a protein (the β clamp) that encircles and slides along the DNA. This protein is loaded on the duplex by proteins DnaX, δ and δ′ <a href="#pone.0000447-Bruck1">[6]</a>. The duplex is melted ahead of the polymerases by the DnaC helicase assembled on DNA by the combined action of the initiation factors DnaA (or PriA), DnaB, DnaD and DnaI <a href="#pone.0000447-Velten1">[7]</a>. RNA primers used by polymerases for strand synthesis are produced by the DnaG primase. While replisome assembly at origins is tightly regulated so that DNA synthesis initiates once per cell cycle <a href="#pone.0000447-Bell1">[8]</a>–<a href="#pone.0000447-Kaguni1">[10]</a>, its progression is generally thought to be unregulated, copying undamaged DNA from the origin to the terminus in an uncontrolled way (see however below).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Glycolysis, gluconeogenesis, the pentose phosphate pathway and the tricarboxylic acid pathway are major elements of the central carbon metabolism in which nutrients provided by the environment are converted into building blocks and used for generating energy and reducing power for biomass synthesis. Glycolysis is a nine reactions pathway that is conventionally split in two parts. In the first part, glucose 6-phosphate is converted into glyceraldehyde 3-phosphate. This set of reactions, directly fuelled by glycolytic nutrients, is efficiently shunted by the pentose phosphate pathway. In the second part, glyceraldehyde 3-phosphate is transformed into pyruvate. These reactions, termed thereafter the three-carbon part of glycolysis, are required for degradation of virtually all carbon sources and cannot be efficiently shunted in most organisms. Thus, they play a key role in cell metabolism. The gluconeogenesis pathway operates when carbon sources feed the bottom part of the central carbon metabolism. It uses most of the glycolytic reactions in the opposite direction to produce glucose 6-phosphate.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Several studies indicate that DNA replication might be linked to cell metabolism. First, the rate of replication is coupled to nutrient richness in bacteria. In <em>Escherichia coli</em>, this coupling is achieved by regulating the frequency of initiation of replication and, in slow-growing cells (generation time ≥70 mn), by modulating the rate of DNA chain elongation <a href="#pone.0000447-Helmstetter1">[11]</a>–<a href="#pone.0000447-Zyskind1">[13]</a>. Second, the stringent response induced by chemicals mimicking a nutrient starvation inhibits initiation of replication in <em>E. coli</em> and arrests DNA elongation at specific sites in the chromosome of <em>B. subtilis</em> (<a href="#pone.0000447-Schreiber1">[14]</a>, <a href="#pone.0000447-Levine1">[15]</a>, reviewed in <a href="#pone.0000447-Zyskind1">[13]</a>, <a href="#pone.0000447-Cashel1">[16]</a>). This response also interferes with plasmid replication (reviewed in <a href="#pone.0000447-Wegrzyn1">[17]</a>). Third, DNA synthesis takes place in the reductive phase of a metabolic respiration/reduction cycle in <em>Saccharomyces cerevisiae</em> <a href="#pone.0000447-Tu1">[18]</a>, <a href="#pone.0000447-Klevecz1">[19]</a>. Fourth, DNA synthesis is stimulated by glucose in SV40 and in HeLa cells grown in hypoxia <a href="#pone.0000447-Riedinger1">[20]</a>. Fifth, mutations in glycolytic genes encoding the enolase (termed thereafter Eno), the phosphoglycerate kinase (Pgk) or the glucokinase suppress a thermosensitive (Ts) mutation in the <em>S. cerevisiae</em> MCM1 protein <a href="#pone.0000447-Chen1">[21]</a>. This multifunctional protein, required for stable maintenance of (mini)-chromosomes, binds sequences closed to replication origins for stimulating initiation of DNA synthesis <a href="#pone.0000447-Maine1">[22]</a>–<a href="#pone.0000447-Chang2">[24]</a>. It also regulates transcription of genes involved in diverse cellular functions including replication and cell-cycle factors (see for instances <a href="#pone.0000447-Fitch1">[25]</a>–<a href="#pone.0000447-Mai1">[27]</a>). Finally, stimulation of histone H2B gene expression, which is essential for S phase progression, strictly depends on the glyceraldehyde 3-P dehydrogenase glycolytic enzyme (GapA) in human cells. In this task, GapA is complexed to a transcriptional co-activator of the H2B gene (ACO-S) and is thought to regulate the activity of the co-activator by sensing the NAD/NADH redox status <a href="#pone.0000447-Zheng1">[28]</a>. The H2B regulation pathway might also involve the lactate dehydrogenase (LDH) <a href="#pone.0000447-Zheng1">[28]</a>. While these observations argue for a functional link between replication and metabolism, the underlying key components and mechanism remain largely unknown. The findings reported here uncover for the first time a robust metabolism/replication link in the bacterium <em>B. subtilis</em>. Key elements of the link have been identified. They are the DnaE (the lagging strand polymerase), DnaG (primase) and DnaC (helicase) replisomal enzymes and the activity of the three-carbon part of the glycolysis/gluconeogenesis pathway. The mechanism and function of the link are discussed.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Genetic link between DNA replication and glycolysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To identify proteins functionally connected to the lagging strand DnaE polymerase, a search was conducted for extragenic suppressors of four <em>dnaE</em> thermosensitive (Ts) mutations (named <em>dnaE2.2</em>, <em>dnaE2.4</em>, <em>dnaE2.6</em> and <em>dnaE2.10</em>, see <a href="#pone-0000447-t001">Table 1</a> for mutation details). Shortly after a shift to restrictive temperature, the Ts mutants cease (or dramatically reduce) DNA synthesis, make filaments and die <a href="#pone.0000447-Dervyn1">[5]</a>. Among a collection of extragenic suppressors isolated at 45–49°C on a glycolytic rich medium (LB, see <a href="#s4">Materials and Methods</a> for details), seven were successively mapped. Surprisingly, rather than occurring in functions related to DNA metabolism, they fell into four metabolic genes (<em>pgk</em>, <em>pgm</em>, <em>eno</em> and <em>pykA</em>) ensuring the terminal reactions of glycolysis (<a href="#pone-0000447-g001">Fig. 1A</a>, the suppressed strains and metabolic mutations are listed <a href="#pone-0000447-t002">Table 2</a>). Three sets of data showed that these mutations were responsible for suppression. First, the genetic transfer of metabolic mutations (<em>pgkEP</em>, <em>pgm8, pgmIP</em> or <em>pykAJP</em>, labeled by a genetic marker) into parental <em>dnaE</em>(Ts) strains led to Tr transformants. Second, four <em>dnaE</em>(Ts) mutants carrying a suppressive mutation in <em>pgk</em>, <em>pgm</em>, <em>eno</em> or <em>pykA</em> became Ts when expressing a WT copy of the corresponding glycolytic gene from a xylose inducible promoter (as exemplified in <a href="#pone-0000447-g001">Fig. 1B</a> with the <em>pgm8</em> suppressor). Third, a deliberately generated deletion of <em>pykA</em> caused <em>dnaE</em>(Ts) suppression (see below).</p>
<div class="figure" id="pone-0000447-g001"><div class="img"><a name="pone-0000447-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Glycolytic mutations suppress <em>dnaE</em>(Ts) mutants.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Schematic representation of glycolysis. GapA: glyceraldehyde 3-phosphate dehydrogenase; Pgk: phosphoglycerate kinase; Pgm: phosphoglycerate mutase; Eno: enolase; PykA: pyruvate kinase. (B) Complementation assay. The <em>dnaE2.6</em> (1), <em>dnaE2.6 pgm8</em> (2) and <em>dnaE2.6 pgm8 pgm<sup>ind</sup></em> (3) strains were grown in LB at 30°C and plated on the same broth containing (grey bars) or not (white bars) 0.5% of xylose (the WT copy of <em>pgm</em> in strain 3 was expressed from a promoter induced by xylose). Upon incubation at permissive or restrictive temperatures, the concentration of cell forming unit (CFU/mL) was determined. (C–E) Growth of Ts and suppressed strains at various temperatures was analyzed by streaking (C; 1: WT TF8A strain; 2: <em>dnaE2.10</em>; 3: <em>dnaE2.10 pgm25</em>), plating (D) and optical microscopy after 2h of growth at 42°C (E). Cell pictures are as follows: cell: bright font; nucleoid: DAPI staining; membrane: FM5-95 staining. (F) Growth of four different <em>dnaE2.6</em> suppressed strains and the corresponding metabolic mutants at restrictive temperature (40°C) in LB liquid broth (followed by measurement of optical density).</p>
<span>doi:10.1371/journal.pone.0000447.g001</span></div><div class="figure" id="pone-0000447-t001"><div class="img"><a name="pone-0000447-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Mutations in <em>dnaE</em>(Ts) alleles</span></strong></p><span>doi:10.1371/journal.pone.0000447.t001</span></div><div class="figure" id="pone-0000447-t002"><div class="img"><a name="pone-0000447-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Spontaneously isolated suppressive mutations</span></strong></p><span>doi:10.1371/journal.pone.0000447.t002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Three sets of data demonstrated that glycolytic mutations are strong <em>dnaE</em>(Ts) suppressors. First, the seven suppressed strains carrying a mapped suppressive mutation grew vigorously at restrictive temperature on plates for at least 20 generations, as they formed thick, wild-type-like colonies and had a plating efficiency of 100% in these growth conditions (<a href="#pone-0000447-g001">Fig. 1C and D</a>). Second, the seven suppressed strains did not form filaments after 2–4 h of incubation at high temperature while <em>dnaE</em>(Ts) strains did (as illustrated for the <em>dnaE2.</em>6 and dnaE2<em>.6 pykAJP</em> strains <a href="#pone-0000447-g001">Fig. 1E</a>). Third, 4/4 spontaneously isolated suppressed strains (<em>dnaE2.4 pgkEP, dnaE2.4 pgmIP</em>, <em>dnaE2.4 pykAJP</em> and <em>dnaE2.6 pgm8</em>) grew as fast as the corresponding metabolic mutants at restrictive temperature (results with the <em>dnaE2.6 pgm8</em> strains are shown <a href="#pone-0000447-g001">Fig. 1F</a>).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To determine whether glycolytic mutations can suppress different <em>dnaE</em>(Ts) alleles and to search for other suppressive mutations, three of the above suppressors (<em>pgkEP</em>, <em>pgm8</em> and <em>ΔpykA</em>) and 16 other metabolic mutations (mainly deletions) in genes of the central carbon metabolism (see <a href="#pone-0000447-g002">Fig. 2</a> for a schematic representation of the central carbon metabolism and inactivated genes) were introduced into the four <em>dnaE</em>(Ts) strains. Control experiments showed that none of the metabolic mutations affect viability of DnaE WT cells at high temperature in LB. Most metabolic mutations (15/19) were not suppressive as they did not restore cell viability on plates at high temperature (<a href="#pone-0000447-g002">Fig. 2, yellow boxes</a>). However, mutations mapping in the three-carbon part of glycolysis were all suppressive (red boxes). Consistently, the corresponding 15 double mutants had a plating efficiency of 100%, formed thick, wild-type colonies and were not filamentous in LB broth at high temperature (the lack of filamentation of the <em>dnaE2.6 ΔgapA</em> strain is illustrated <a href="#pone-0000447-g001">Fig. 1E–F</a>). Moreover, 4/4 newly constructed double mutants grew like the corresponding metabolic mutants at restrictive temperature (as shown for strains <em>dnaE2.6 ΔgapA</em>, <em>dnaE2.6 pgkEP</em> and <em>dnaE2.6 ΔpykA</em>, <a href="#pone-0000447-g001">Fig. 1F</a>).</p>
<div class="figure" id="pone-0000447-g002"><div class="img"><a name="pone-0000447-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Suppressive mutations of <em>dnaE</em>(Ts) strains are clustered in the terminal reactions of glycolysis.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>To analyze the impact of metabolic mutations on the Ts character of <em>dnaE</em>(Ts) strains, the plating efficiency of double mutants at restrictive temperature was compared to that at permissive temperature in LB (boxed numbers). The level of filamentation was also analyzed for Tr strains. At least three independently constructed double mutants were tested. Metabolic mutations were deletions except for <em>pfk</em>, <em>pgk</em> and <em>pgm</em> were point mutations were used (<em>pfk1</em>, <em>pgkEP</em> and <em>pgm8</em>). Red boxes: strains fully suppressed (cells had a relative plating efficiency of 1, formed thick colonies, grew as fast as the corresponding metabolic mutant and did not make filament in liquid at restrictive temperature); yellow boxes: strains not suppressed (cells had a low plating efficiency at high temperature or, as in the <em>dnaE2.6 Δpdh</em> and <em>dnaE2.10 Δpta</em> strains, had a plating efficiency close to 1 but formed thin colonies with filamentous cells); white boxes: not tested. Thick lines and dots represent the central carbon metabolism reactions and their products, respectively. Main pathways are indicated. PDH: pyruvate dehydrogenase; O: overflow pathway; TCA: tricarboxylic acid cycle; PPP: pentose phosphate pathway; NEOG: gluconeogenesis.</p>
<span>doi:10.1371/journal.pone.0000447.g002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>To further circumscribe the glycolysis/replication relationship, the effect of 9 metabolic mutations on 14 Ts mutations mapping in initiation and elongation replication genes was assessed (93 strains were tested). The effect of <em>ΔpykA</em> on two Ts division genes (<em>div104</em> and <em>ts1</em>) was also investigated. Only the mutations in the three-carbon part of glycolysis were suppressive. Full suppression (i.e. on plates and in liquid) was detected for three new <em>dna</em>(Ts) mutations mapping in the elongation factors DnaG (primase, <em>dnaG20</em>) and DnaC (helicase, <em>dnaC14</em> and <em>dnaC30</em>) (<a href="#pone-0000447-g003">Fig. 3A, red boxes and Fig. B–D</a>). Partial suppression (seen on plates but not in liquid) of one DnaC (<em>dnaC8133</em>), two PolC (<em>dnaF33</em> and <em>dnaF69</em>) and two DnaX (<em>dnaX8132</em> and <em>dnaH51</em>) mutants was also observed in some strains (the reason for different suppression efficiency on plates and liquid is not fully understood) (<a href="#pone-0000447-g003">Fig. 3A, orange boxes</a>). The remaining cells were not suppressed (<a href="#pone-0000447-g003">Fig. 3A, yellow boxes</a>). Taken together, the results establish a tight link between the three-carbon part of glycolysis and DNA replication. The robust suppression phenotype of some <em>dna</em>(Ts) mutations suggest that the link is particularly strong with the DnaE, DnaG and DnaC enzymes.</p>
<div class="figure" id="pone-0000447-g003"><div class="img"><a name="pone-0000447-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Mutations in terminal reactions of glycolysis can suppress <em>dnaG</em>(Ts) and <em>dnaC</em>(Ts) mutants.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>(A) The effect of 9 metabolic mutations on Ts mutations carried by initiation (I) or elongation (E) factors of DNA replication was investigated by the plating and filamentation assays. Relative plating efficiencies (see legend to <a href="#pone-0000447-g002">Fig. 2</a>) are indicated inside the boxes. All the tested metabolic mutations were deletion except in the case of <em>pgk</em> where the point <em>pgkEP</em> mutation was used. At least two independent double mutants were tested. Red boxes: strains fully suppressed (cells had a plating efficiency of 100%, formed thick colonies and are not filamentous in liquid at restrictive temperature); orange boxes: strains partially suppressed (cells had a plating efficiency of 100%, formed thick colonies but were filamentous in liquid at high temperature); yellow boxes: strains not suppressed (cells had a low plating efficiency at high temperature or, in a few instances, had a plating efficiency close to 1 but formed thin colonies containing filamentous cells); white boxes: not tested. (B) Plating analysis of Ts and suppressed strains at various temperatures. (C) Growth analysis of <em>dnaG20 ΔpykA</em> and <em>ΔpykA</em> mutants at 42°C in LB liquid browth. (D) Optical microscopy analysis at restrictive temperature of Ts and suppressed mutants (see legend of <a href="#pone-0000447-g001">Fig. 1</a>).</p>
<span>doi:10.1371/journal.pone.0000447.g003</span></div>

<h4>Identification of metabolic alterations causing suppression</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Mutations in genes of the central carbon metabolism often reduce cell growth. To test whether this reduction correlated with suppression, the doubling time of 20 metabolic mutants, WT for replication functions, was measured at high temperature in LB. A large range of growth rates, extending from the optimal value (~3.8 doublings per hour) to a significantly lower rate (~1 doubling per hour) was observed (<a href="#pone-0000447-g004">Fig. 4</a>). Suppressive mutations (<em>pgm8</em>, <em>pgmIP</em>, <em>ΔgapA</em>, <em>pgkEP</em>, <em>pykAJP</em> and <em>ΔpykA</em>) had a moderate growth rate (1.5–2 doublings per hour). Interestingly, some non suppressive mutations grew at a similar (<em>pfk, ΔpdhB</em>) or even lower (~1 doubling per hour, <em>ΔpdhC</em> and <em>ΔpdhD</em>) rate. Moreover, suppression was observed in fast growing cells (2.5–3.5 doublings per hour, <a href="#pone-0000447-g004">Fig. 4</a>). These cells expressed glycolytic genes from inducible promoters (<em>Psapc-I</em> and <em>Pxyl</em>) or carried the <em>ΔpykA</em> mutation and were grown in a gluconeogenic medium (see below). Altogether, these data suggest that a mere decrease in growth rate is not a prerequisite for suppression.</p>
<div class="figure" id="pone-0000447-g004"><div class="img"><a name="pone-0000447-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Suppression does not depend on a growth rate decrease.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The doubling rate of metabolic mutants, WT for replication functions, was determined at 40–44°C by measurement of optical density of LB cultures. Values are positioned along a vertical axis. Mutations are grouped into suppressive, not suppressive, constitutive and inducible mutations. The growth rate of cells expressing glycolytic genes from the <em>Pspac-I</em> or <em>Pxyl</em> inducible promoters was positioned at the highest value allowing suppression for the following Ts alleles: <sup>a</sup>: <em>dnaC30</em>; <sup>b and d</sup>: <em>dnaE2.6</em>; <sup>c</sup>: <em>dnaG20</em>. <sup>e</sup>: Suppression of <em>dnaG20</em> by the <em>ΔpykA</em> mutation in the gluconeogenic medium CM (see <a href="#pone-0000447-g005">Fig. 5</a> and the text below).</p>
<span>doi:10.1371/journal.pone.0000447.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Mutations in genes of the central carbon metabolism cause unscheduled alterations in concentration of metabolites and/or energetic compounds. Because (i) unbalanced metabolism might be sensed by cells as a stress, (ii) some nutritional stresses interfere with DNA replication <a href="#pone.0000447-Schreiber1">[14]</a>, <a href="#pone.0000447-Levine1">[15]</a> and (iii) osmotic shocks can suppress Ts mutations in division and replication genes <a href="#pone.0000447-Reeve1">[29]</a>–<a href="#pone.0000447-Henson1">[31]</a>, the effect of various stresses on the Ts phenotype of replication mutants was assessed. For this purpose, the thermosensitivity of the <em>dnaE2.2, dnaE2.4, dnaE2.6, dnaG20</em> and <em>dnaC30</em> strains, WT for metabolic functions, was assessed on LB plates supplemented (separately or in combination) with chemicals causing osmotic (2% NaCl), energetic (125–500 µg/ml sodium azide) or nutritional (0.5 and 2 mg/ml DL-norvaline or 250 µg/ml arginine hydroxamate) stresses. In no case was suppression observed. Control experiments showed that, at the concentration used, the chemicals alter growth of WT cells but not their viability and plating efficiency. In case of arginine hydroxamate, WT cells did not survive to a 2 times higher concentration of the drug. This suggests that stress-like responses are not key determinants in suppression.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To further identify metabolic alterations required for suppression, strains expressing a gene (<em>pgk</em> or <em>pykA</em>) or a group of genes (the whole <em>gapA</em> operon, which includes the suppressive <em>gapA</em>, <em>pgk</em>, <em>pgm</em> and <em>eno</em> genes) of the bottom part of glycolysis under the control of an inducible promoter were constructed. As glycolytic enzymes are abundant in living cells, we first tested whether the selected promoters (<em>Pspac-I</em> - IPTG dependent - or <em>Pxyl</em> - xylose dependent) were strong enough to modulate gene expression and hence glycolysis throughout a substantial range at high temperature in LB. Results with the <em>Pspac-I::pykA</em> strain (WT for replication functions) showed a significant modulation of PykA activity (from about 0.04 to 0.45 unit per mg of protein) (<a href="#pone-0000447-g005">Fig. 5A</a>). However, the modulation did not cover the whole range of values observed with the <em>ΔpykA</em> (0.006 unit/mg) and wild-type PykA+ (2.03 units/mg) strains, presumably because of some leakage and insufficient strength of <em>Pspac-I</em> (the weak increase in PykA activity and possibly growth rate at 0–15 µM IPTG could result from a non linear response of the <em>Pspac-I</em> promoter at low IPTG concentrations). Despite this limitation, <em>Pspac-I</em> allowed a growth rate modulation extending from that of the PykA+ and <em>ΔpykA</em> strains (compare <a href="#pone-0000447-g004">Fig. 4</a> and <a href="#pone-0000447-g005">5A</a>). Similar effects on growth rate were observed with the <em>gapA</em> operon expressed from <em>Pspac-I</em> and with <em>pgk</em> or <em>pykA</em> expressed from <em>Pxyl</em> (not shown). This shows that the inducible promoters are strong enough to substantially modulate cell metabolism.</p>
<div class="figure" id="pone-0000447-g005"><div class="img"><a name="pone-0000447-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Suppression occurs abruptly at discrete and contrasted levels of PykA activity. (A) Modulation of PykA activity and growth rate.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A WT strain expressing <em>pykA</em> from the IPTG inducible <em>spac-I</em> promoter was cultivated at 45°C in LB supplemented with various concentrations of IPTG (µM). The number of doublings per hour (closed diamonds) was determined from optical density measurements. PykA activity was measured from crude extracts prepared from cells growing exponentially in LB broth supplemented with various IPTG concentrations (open diamonds). Typical experiments are presented. (B) Abrupt phenotypic shift of <em>dna</em>(Ts) <em>Pspac-I::pykA</em> strains on plates. Strains were spread on LB plates containing various IPTG concentrations (µM). The number of colony forming units was counted after plate incubation at restrictive temperature. Red, yellow and orange symbols pinpoints IPTG concentrations where cells were Tr (forming thick colonies at high frequency), Ts (low plating efficiency) or shifting between the two phenotypes (variable plating efficiency and thin, heterogeneous colonies), respectively. (C) Abrupt phenotypic shift of the <em>dnaE2.6</em> <em>Pspac-I::pykA</em> strain in liquid. The strain was grown at 30°C in LB broth supplemented with various concentrations of IPTG (µM). Exponentially growing cells were then shifted to restrictive temperature (time 0) and maintained at low concentration (OD<sub>600 nm</sub> &lt;0.4) by serial dilutions when required. Aliquots were withdrawn at different times and the number of viable cells was determined by growing cells on plates containing 100 µM IPTG at 30°C.</p>
<span>doi:10.1371/journal.pone.0000447.g005</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>To evaluate the impact of <em>pykA</em> expression on the Ts character of the <em>dnaE2.6</em> mutation, fresh <em>dnaE2.6 Pspac-I::pykA</em> cells grown at 30°C were spread on plates containing various IPTG concentrations and grown at restrictive temperature. Colony counting showed an abrupt shift from the Ts to Tr phenotype between 60 and 30 µM IPTG (<a href="#pone-0000447-g005">Fig. 5B</a>). This shift was also observed in liquid culture. Cells grew at ≤30 µM but not at ≥60 µM IPTG (<a href="#pone-0000447-g005">Fig. 5C</a>). They were filamentous at ≥60 µM but not at &lt;15 µM (not shown). Interestingly, the Ts/Tr shift occurred at a PykA activity (0.35–0.4 unit/mg) allowing a growth rate (~3.1 doublings per hour) which is only ~15% lower than the optimal value (<a href="#pone-0000447-g004">Fig. 4</a> and <a href="#pone-0000447-g005">5A</a>). A similar abrupt shift in thermosensitivity of the <em>dnaE2.6</em> mutation was observed at a high growth rate when the whole <em>gapA</em> operon was expressed from <em>Pspac-I</em> or when <em>pykA</em> or <em>pgk</em> were expressed from <em>Pxyl</em> (not shown). Taken together, these data suggest that the Ts/Tr shift of the <em>dnaE2.6</em> strain occurs abruptly upon a moderate reduction (~2 times) in expression of the genes encoding enzymes of the bottom part of glycolysis.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>The <em>Pspac-I::pykA</em> fusion was then introduced in two other DnaETs mutants (<em>dnaE2.2</em> and <em>dnaE2.4</em>), one DnaC mutant (<em>dnaC30</em>) and in the <em>dnaG20</em> strain and suppression was assessed on plates as above. All mutants shifted from the Ts to Tr phenotype upon modulation of PykA expression (<a href="#pone-0000447-g005">Fig. 5B</a>). In the range of linear response of promoter activity to inducer concentration (about 10–100 µM), the shift again occurred in a narrow (~2) window of IPTG concentration. Interestingly, despite the similar window size, the Ts mutants underwent the phenotypic shift at drastically different levels of PykA expression and growth rate. For instance, the <em>dnaE2.2</em> mutant shifted at low PykA concentration and growth rate while <em>dnaE2.6</em> and <em>dnaC30</em> shifted at high PykA concentrations and growth rates (<a href="#pone-0000447-g004">Fig. 4</a>). It is inferred from this that Ts mutants shift at discrete, specific and highly diverse levels of expression of genes of the three-carbon part of glycolysis. These results further support the notion that growth rate is not a key determinant in suppression.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>LB is a complex rich medium that likely contains glycolytic and gluconeogenic carbon sources. It however confers a glycolytic regimen to exponentially growing cells as mutants deleted for the glycolytic <em>pykA</em> or <em>gapA</em> genes grew at a slower (~2.5 times) rate than WT cells or strains deleted for the gluconeogenic <em>pckA</em> or <em>gapB</em> genes (<a href="#pone-0000447-g004">Fig. 4</a>). To further characterize the metabolic regimen (glycolytic or gluconeogenic) in suppressive and non suppressive contexts, the activity of the <em>gapB</em> promoter, an indicator of gluconeogenesis, in the WT strain and in metabolic mutants was measured. Using <em>lacZ</em> transcriptional fusion, it was previously shown that <em>gapB</em> expression is repressed (≤20 β-Galactosidase units) in glycolytic carbon sources and strongly induced (≥200 β-Galactosidase units) on gluconeogenic nutrients <a href="#pone.0000447-Fillinger1">[32]</a>. Using this construction, we observed a glycolytic regimen for both suppressive (<em>ΔpykA</em> and <em>pgkEP</em>) and non suppressive (WT, <em>Δpgi</em> and <em>ΔpdhC</em>) contexts (<a href="#pone-0000447-t003">Table 3</a>). This suggests that the suppression phenotype observed above occurs in a glycolytic regimen.</p>
<div class="figure" id="pone-0000447-t003"><div class="img"><a name="pone-0000447-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Suppression in LB occurred in glycolytic regimen</span></strong></p><span>doi:10.1371/journal.pone.0000447.t003</span></div><a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>In order to test whether suppression can also take place in gluconeogenesis, suppression was assessed in CM, a minimal medium supplemented with gluconeogenic carbon sources (casein hydrolysate and malate) that allows a high growth rate (~2.9 doublings per hour at 46°C) to WT cells. This medium is gluconeogenic as mutations in the three-carbon part of the gluconeogenesis pathway (including the reversible glycolytic reactions <em>pgm</em> and <em>pgk</em> and the gluconeogenic genes <em>pckA</em> and <em>gapB</em>) prevented (or strongly reduced) growth in CM, while inactivation of the glycolytic PykA enzyme had no effect on growth rate (not shown). The plating assay showed that <em>dnaE2.2</em>, <em>dnaE2.4</em>, <em>dnaE2.6</em>, <em>dnaG20</em> and <em>dnaC30</em> strains, WT for metabolic functions, were Ts in CM. The isogenic <em>ΔpykA</em> strains were all but one (the <em>dnaC30 ΔpykA</em> strain) Tr on plates. Taken together, these data showed that suppression of some Ts mutations can occur in the neoglucogenic regimen.</p>


<h4>Impact of metabolic alterations on DnaTs protein activity</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Suppression could have resulted from changes in the replisome that make Ts proteins dispensable for DNA synthesis. Two sets of data showed that this is not the case. First, most of the suppressed strains exhibited a residual thermosensitivity at extreme <em>B. subtilis</em> growth temperatures (cf <a href="#pone-0000447-g001">Fig. 1D</a> and <a href="#pone-0000447-g003">3B</a>). Second, disruption of the <em>dnaE</em> gene in strains carrying a suppressive mutation was not possible unless the recipient strain carried an additional copy of the gene (not shown). Hence, survival at high temperature of suppressed strains requires at least partially active Ts proteins.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Transcriptional fusions of the <em>dnaE</em>, <em>dnaC</em> and <em>polC</em> promoters with the <em>lacZ</em> reporter gene indicated that suppression does not result from <em>dna</em>(Ts) genes over-expression (<a href="#pone-0000447-t004">Table 4</a>). In order to test whether it might result from Ts protein accumulation, a set of PykA+ and PykA- strains encoding from <em>Pspac-I</em> the DnaE-2.2, -2.4 or –2.6 Ts proteins fused to the SPA tag <a href="#pone.0000447-Zeghouf1">[33]</a> at the C-terminus was constructed. The plating assay showed that the <em>dnaE</em>(Ts)-<em>SPA</em> PykA+ cells were Ts and that the isogenic PykA− strains were Tr at high IPTG concentrations (as illustrated in the top panel of <a href="#pone-0000447-g006">Fig. 6A</a> with the <em>dnaE2.2-SPA</em> allele). This suggested that DnaETs-SPA proteins respond to glycolytic mutations like the original DnaETs proteins. Despite the contrasted phenotype of the PykA+ and PykA− strains, both cell types contained similar amount of DnaETs-SPA fusion proteins at saturating amount of IPTG (1000 µM) as shown by Western blot analysis with anti-SPA antibodies (<a href="#pone-0000447-g006">Fig. 6A, bottom panel</a>). Moreover, the <em>dnaE2.2-SPA</em> <em>ΔpykA</em> strain exhibited a similar IPTG dependence and intracellular DnaETs-SPA concentration at permissive and restrictive temperature (<a href="#pone-0000447-g006">Fig. 6B</a>). Similar results were observed with the <em>dnaE2.4-SPA</em> <em>ΔpykA</em> strain (not shown). Taken together, these results suggest that (i) suppression does not depend on Ts protein accumulation and that (ii) the DnaTs proteins are Tr in the PykA− background.</p>
<div class="figure" id="pone-0000447-g006"><div class="img"><a name="pone-0000447-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Suppression is due to maintenance of DnaTs protein activity at high temperature.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Suppression is not due to DnaETs protein accumulation. Top panel: Plating efficiency of <em>PspacI::dnaE2.2-SPA</em> PykA+ and PykA− cells plated on medium supplemented with various IPTG concentrations and incubated at 47°C (restrictive temperature on plates). Bottom panel: SDS-PAGE (stained with SYPRO-Red for total protein staining) and Western blot analysis of crude extracts (10 µg of total proteins) prepared from cells grown ~5 generations in 1000 µM IPTG at 30 or 45°C (restrictive temperature in liquid cultures). (B) The DnaETs proteins are Tr in the PykA− context. Top panel: Plating efficiency of the <em>PspacI::dnaE2.2-SPA ΔpykA</em> strain grown in various IPTG concentrations at 30 and 47°C. Bottom panel: SDS-PAGE and Western blot analysis of crude extracts prepared from cells grown ~5 generations at 30 or 45°C in 250 or 1000 µM IPTG.</p>
<span>doi:10.1371/journal.pone.0000447.g006</span></div><div class="figure" id="pone-0000447-t004"><div class="img"><a name="pone-0000447-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Suppressive mutations do not stimulate promoters of replication genes</span></strong></p><span>doi:10.1371/journal.pone.0000447.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>DNA replication and carbon metabolism are fundamental to life. Here, we report a robust genetic link between them. This link, discovered during an extensive analysis of <em>dna</em>(Ts) suppressors, is very strong as some metabolic alterations fully restore growth of <em>dna</em>(Ts) mutants at a temperature at which a lethal arrest of DNA synthesis otherwise occurs. Two major lines of evidence show that this link is highly specific in terms of both replication and metabolism. In the case of replication, full viability is restored to mutants affected in 3 DNA elongation factors out of 9 replication functions tested (3 specific for DNA initiation and 6 mainly involved in DNA elongation). These factors are the lagging strand polymerase DnaE, the DnaC helicase and the DnaG primase. These enzymes probably act on the same DNA strand (the lagging strand template) in the replication fork and two of them (DnaG and DnaC) interact <a href="#pone.0000447-Dervyn1">[5]</a>, <a href="#pone.0000447-Velten1">[7]</a>, <a href="#pone.0000447-NoirotGros1">[34]</a>. In the case of metabolism, full viability is restored only by mutations in the 5 successive terminal reactions of the three-carbon part of glycolysis/gluconeogenesis out of 21 mutations leading to inactivity or absence of enzymes of the central carbon metabolism. Interestingly, two lines of observations suggest that these 5 terminal reactions are at a key position in the overall metabolism of the cell. First, these reactions provide a single route for the catabolism of nutrients (via both glycolysis and gluconeogenesis) and form therefore one of the most active parts of the metabolic network in any growth conditions. Second, the enzymes involved are highly abundant and conserved in the three kingdoms of life <a href="#pone.0000447-FothergillGilmore1">[35]</a>, <a href="#pone.0000447-Canback1">[36]</a>. For these reasons, the three-carbon part of the glycolysis/gluconeogenesis pathway can be viewed as an ancestral linker that organizes the metabolic pathway network into two domains including, in one case, the upper part of glycolysis and branched reactions, and, in the other case, the TCA cycle and branched reactions (<a href="#pone-0000447-g007">Fig. 7, left panel</a>). The linker might also serve non-metabolic roles, as its enzymes are often multifunctional and essential even in rich media containing nutrients that simultaneously enter the central carbon metabolism at various levels <a href="#pone.0000447-FothergillGilmore1">[35]</a>, <a href="#pone.0000447-Giaever1">[37]</a>–<a href="#pone.0000447-Sirover1">[42]</a>. Data reported here and elsewhere indicated that one of these new roles might be to coordinate principal cellular functions <a href="#pone.0000447-Chen1">[21]</a>, <a href="#pone.0000447-Zheng1">[28]</a>, <a href="#pone.0000447-Kim1">[40]</a>–<a href="#pone.0000447-Danial1">[49]</a>.</p>
<div class="figure" id="pone-0000447-g007"><div class="img"><a name="pone-0000447-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000447" data-uri="info:doi/10.1371/journal.pone.0000447.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000447.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000447.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000447.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000447.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Model of glycolysis and replication connection.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Signals generated according to the activity of the metabolic linker cause conformational changes in replication proteins to modulate replisome properties. The thick blue arrows pinpoint main sites of carbon diffusion.</p>
<span>doi:10.1371/journal.pone.0000447.g007</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The molecular basis of the Ts phenotype of the <em>dnaE</em>, <em>dnaG</em> and <em>dnaC</em> mutants has not been fully analyzed. However, two mutants (<em>dnaE2.10</em> and <em>dnaE2.6</em>) are defective in the elongation phase of DNA replication as they cease incorporation of labeled precursors shortly after a shift to restrictive temperature (47°C) <a href="#pone.0000447-Dervyn1">[5]</a> (unpublished data). The <em>dnaE2.2</em> and <em>danE2.4</em> mutations are also considered to affect elongation, as cells carrying these mutations exhibit a strong decrease (albeit, not an arrest) in the rate of precursor incorporation (unpublished data) and form long filaments at restrictive temperature (in bacteria, the latter response is induced when perturbations in DNA chain elongation lead to generation of single-stranded DNA <a href="#pone.0000447-Sassanfar1">[50]</a>). Cells containing <em>dnaG</em> and <em>dnaC</em> mutations also form filaments at restrictive temperature and are thus tentatively classified as elongation mutants as well. The likely high diversity in the processes leading to inhibition and rescue of replication in <em>dnaE</em>, <em>dnaG</em> and <em>dnaC</em> mutants contrasts sharply with the limited number of metabolic mutations, clustered in the three-carbon part of glycolysis, that lead to suppression. For this reason, we hypothesize that restoration of viability proceeds through a similar mechanism in all the suppressed strains.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Several hypotheses can account for the viability of <em>dna</em>(Ts) mutants at high temperature. In one hypothesis, growth restoration is due to DnaTs proteins being protected from heat-inactivation by a stress-like response induced by metabolic alterations. However, the following observations make this possibility unlikely. First, separate or combined induction of the general stress regulon and the stringent response by chemicals inducing nutritional, energetic or physical stresses was not suppressive. Second, efficient suppression is observed with Ts mutations occurring in only three replication proteins while a stress-like-induced mechanism would be expected to cause a more random profile of suppression because of the relaxed effect of thermoprotector agents on protein conformation <a href="#pone.0000447-Thirumalai1">[51]</a>–<a href="#pone.0000447-Erbse1">[53]</a>. Third, it is unexpected that thermoprotector agents would respond efficiently to numerous but sometimes limited changes in linker activity but not to any alteration elsewhere in central carbon metabolism.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>A second hypothesis is that the restoration of viability of <em>dna</em>(Ts) cells results from <em>dna</em>(Ts) genes over-expression and DnaTs protein accumulation. This hypothesis is based on recent evidence that several metabolic enzymes (aconitase, TktA…) play a role in the organization and packaging of bacterial and mitonchondrial genomes, and thereby, can modulate DNA supercoiling and expression of genes with promoters sensitive to DNA topology <a href="#pone.0000447-Hardy1">[54]</a>, <a href="#pone.0000447-Chen2">[55]</a>. This hypothesis is rejected as neither <em>dna</em> gene over-expression (the expression of <em>dnaE</em> and <em>dnaC</em> was analysed) nor DnaTs protein accumulation (3 <em>dnaE</em>(Ts) alleles were tested) are detected in suppressed cells. Topological changes might also rescue cell growth by stimulating expression of genes involved in the stabilisation, repair or restart of altered replication fork <a href="#pone.0000447-Foti1">[56]</a>, <a href="#pone.0000447-Michel1">[57]</a>. While attractive, this hypothesis is discarded as (i) promoters of the genes involved in these processes are not sensitive to DNA topology <a href="#pone.0000447-Hardy1">[54]</a>, <a href="#pone.0000447-Peter1">[58]</a> and (ii) it is unlikely that the stabilization, repair and restart pathways can rescue mutations that arrest DNA synthesis.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>A third hypothesis relies on the role played by the ribonucleotide reductase (RNR) and the dNTP pool on DNA replication. The synthesis of dNTP depends on RNR that converts NDPs into dNDPs. In <em>E. coli</em>, transcription of the RNR encoding gene is stimulated during initiation of DNA synthesis and this stimulation is required for DNA elongation <a href="#pone.0000447-Gon1">[59]</a>, <a href="#pone.0000447-Ortenberg1">[60]</a>. Moreover, RNR would be part of a large dynamic structure that contains the replication machinery and the dNTP synthesis apparatus <a href="#pone.0000447-Norris1">[61]</a>–<a href="#pone.0000447-Guzman1">[63]</a>. These functional and structural relationships between RNR, DNA elongation, and the replication machinery raise the possibility that RNR and/or the dNTP pool are involved in restoration of growth to Ts mutants. Obviously, this hypothesis makes sense in the case of mutations altering the polymerase DnaE. It however should not be involved in suppression of mutations in DnaC and DnaG whose activity (DNA unwinding and primer synthesis, respectively) depends on NTP but not on dNTP.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>A fourth hypothesis is that the restoration of viability is the result of a change in the characteristics of initiation or of chain elongation or of both. In <em>E. coli</em>, mutations that restore viability to a thermosensitive DNA elongation mutant (<em>dnaX2016</em>) have previously been isolated <a href="#pone.0000447-Skovgaard1">[64]</a>, <a href="#pone.0000447-Walker1">[65]</a>. They map to <em>dnaA</em>, a gene that encodes the major initiator of DNA replication. During initiation, DnaA binds to short repeated sequences located in the chromosomal <em>oriC</em> region, partially unwinds <em>oriC</em>, and recruits the helicase. The helicase then interacts with the primase that synthesizes primers at which the replisome is assembled for initiating DNA synthesis [reviewed in <a href="#pone.0000447-Johnson1">[2]</a>, <a href="#pone.0000447-Kaguni1">[10]</a>]. It was found that the restoration of viability to <em>dnaX</em>(Ts) <em>dnaA</em> double mutants at high temperature does not result from a strong stimulation of DNA elongation. Indeed, in 3/3 suppressed strains analyzed at high temperature, DNA synthesis remained strongly disturbed, leading to a high proportion of filamentous cells containing an incorrectly compacted and distributed nucleoid <a href="#pone.0000447-Blinkova1">[66]</a>. The lack of effect on DNA elongation was further supported by the fact that restoration of cell viability was only observed at temperatures allowing residual DNA synthesis (i.e. suppression did not happen at temperatures inhibiting DNA synthesis). Instead of depending on elongation stimulation, restoration of viability of <em>dnaX dnaA</em> double mutants was shown to result from a decrease in the efficiency of initiation. This decrease is a direct consequence of <em>dnaA</em> suppressive mutations that significantly weaken the initiation activity of DnaA <a href="#pone.0000447-Blinkova1">[66]</a>, <a href="#pone.0000447-Walker2">[67]</a>. It was thus proposed that these <em>dnaA</em> mutations restore viability at high temperature by reducing the number of chromosome per cell to a level that allows the partially active DnaX protein to support ongoing chromosomal replication and cell growth <a href="#pone.0000447-Blinkova1">[66]</a>. Survival of <em>E. coli</em> cells to mild inhibition of chain elongation by drugs also relies on mutations that reduce the efficiency of initiation <a href="#pone.0000447-Sutera1">[68]</a>. In contrast, our results suggest that the restoration of viability to <em>B. subtilis dna</em>(Ts) cells results mainly from stimulation of DNA elongation rather than from modifications in initiation efficiency. First, suppression can occur at temperatures at which DNA synthesis is arrested. This is at least the case for the <em>dnaE2.10</em> allele which can be suppressed at temperatures ranging from 47–50°C while DNA synthesis is already arrested at 47°C in Ts cells WT for glycolytic genes <a href="#pone.0000447-Dervyn1">[5]</a>. Second, DNA elongation seems to be normal in all (21/21) the suppressed strains identified here, as, unlike the original <em>dna</em>(Ts) mutants, these cells do not form filament at high temperature and their nucleoid is properly compacted and distributed. It should also be noted that despite the relatively large study conducted here, no suppressive mutations mapping in any of the four initiation genes (<em>dnaA</em>, <em>dnaB</em>, <em>dnaD</em> and <em>dnaI</em>) were isolated despite the total size of the DNA in these potential target genes being similar to that of the suppressive glycolytic genes (4.4 versus 6.7 kb, respectively). These observations therefore suggest that the restoration of cell viability in <em>E. coli</em> and <em>B. subtilis</em> elongation mutants depends on different mechanisms and that in <em>B. subtilis</em>, this mechanism mainly operates via stimulation of DNA chain elongation.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Stimulation of DNA elongation in <em>B. subtilis</em> suppressed cells might result from changes in replisome requirements. One possibility is that there is a change in the composition of the replisome that allows replication to occur despite the absence of the DnaTs proteins. There is no precedent for this, most likely because the function of DnaC, DnaG and DnaE (duplex unwinding, primer synthesis and lagging strand polymerization, respectively) are essential for DNA synthesis and cannot be carried out by other replisomal proteins. Nevertheless, they might theoretically be ensured by alternative replication factors as those encoded, for example, by prophages. While attractive, this possibility can be discounted because (i) studies on dnaE(Ts) mutants have all been carried out in cells devoid of the three main <em>B. subtilis</em> prophages (SPβ, PBSX and Skin) <a href="#pone.0000447-Westers1">[69]</a>and (ii) control experiments showed that the restoration of viability of <em>dnaC</em>(Ts) and <em>dnaG</em>(Ts) mutants, initially observed in prophage-containing cells, occurs with the same efficiency in prophage-free strains (not shown). It is also discarded that the putative alternative replication factors are of chromosomal origin since <em>B. subtilis</em> does not encode structural and/or functional homologues of the DnaC helicase and DnaG primase that could replace efficiently these proteins at the fork <a href="#pone.0000447-Kunst1">[70]</a>. In contrast to these hypothesis, two observations suggest that restoration of viability of <em>dna</em>(Ts) cells requires DnaTs proteins. First, most of the suppressed strains are thermosensitive at extreme temperatures. Second, the <em>dnaE</em> gene cannot be inactivated in cells carrying a suppressive mutation. Hence, restoration of viability cannot be explained satisfactorily by changes in the composition of the replisome.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>An alternative possibility for stimulating DNA elongation in <em>dna</em>(Ts) cells is a functional change in the replisome, a hypothesis consistent with all our results. Such a functional change might be due to suppressive metabolic mutations either making some activities of DnaTs proteins dispensable for DNA synthesis or protecting these activities from heat-inactivation. It may therefore be significant that (i) suppressed and non-suppressed strains contain the same amount of DnaETs protein at restrictive temperature and (ii) the minimal DnaETs concentration required for survival of (PykA−) suppressive strains is similar at permissive and restrictive temperature. As mentioned above, DnaC, DnaG and DnaE have catalytic activities that are essential for DNA synthesis and cannot be efficiently ensured by alternative proteins. Their heat inactivation can thus only be countered by a process allowing their thermo-protection. However, replication enzymes might also be required at the fork for structural and/or regulatory purposes. If the loss of these additional functions could be tolerated in certain conditions, this loss might be suppressed not only by thermo-protection but also by processes making these functions dispensable at the fork. The function(s) defective in DnaTs proteins at high temperature need to be identified to further dissect the molecular basis of stimulation of DNA elongation in suppressed cells. However, irrespective of the detailed mechanism, an attractive hypothesis is that suppression involves conformational changes in replisomal enzymes.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Our physiological studies identify the activity of the metabolic linker region as important in restoration of viability. This restoration occurs abruptly over a narrow range of metabolic linker activity and the absolute value of this range varies greatly with the Ts allele in question. However, viability can be restored without major change of the growth rate or the direction of carbon flow through the metabolic linker. The absence of effect of reversing the carbon flow rules out catabolite repression, a genetic system that regulates expression of numerous genes in highly energetic glycolytic carbon sources <a href="#pone.0000447-Moreno1">[71]</a>, as a major factor in suppression. Moreover, restoration of viability cannot be due to a reduction in ATP concentration since (i) inhibition of ATP synthesis is not suppressive and (ii) ATP is present at a constant level over a large range of metabolic activities and growth rates (from 2.3 to &lt;0.4 doublings per hour) <a href="#pone.0000447-Dauner1">[72]</a>–<a href="#pone.0000447-Schneider1">[74]</a>. We therefore propose that the postulated conformational changes in replisomal enzymes of suppressed cells depends on changes in the activity of the metabolic linker.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Our suppression assay provides the first evidence for a genetic system that connects DNA chain elongation to glycolysis. Its role in WT cells may be to modulate some aspects of DNA synthesis in response to the provided energy. The molecular mechanism of the linker effect on the postulated conformational changes in replisomal enzymes remains to be elucidated. We viewed the system as comprising the metabolic linker, a signal-generating activity of this linker, and signal-mediated conformational changes in replisomal proteins, possibly the lagging-strand DnaE polymerase, DnaC helicase and DnaG primase (<a href="#pone-0000447-g007">Fig. 7</a>). These conformational changes might result from protein binding to metabolites, post-translational modifications and/or protein-protein interactions. Metabolite binding (mainly ATP) and post-translational modifications (mainly phosphorylation) of replication proteins are frequently reported in the literature (see for instance) <a href="#pone.0000447-Bell1">[8]</a>, <a href="#pone.0000447-Kelly1">[75]</a>–<a href="#pone.0000447-Mijakovic1">[78]</a>. Examples of physical interactions between replication proteins and metabolic enzymes are on the contrary scarce. However, interactions between the primase DnaG and subunits of the pyruvate dehydrogenase, an enzyme that converts the glycolysis product (pyruvate) into acetyl-CoA, have been detected in <em>B. subtilis</em> using the yeast two hybrid technology <a href="#pone.0000447-NoirotGros1">[34]</a>. Moreover, evidence was also obtained in both prokaryotes and eukaryotes for an interaction between the replication machinery and a complex involved in dNTP synthesis. This interaction is proposed to form a dynamic hyperstructure in which a high concentration of dNTP is provided to support the very high rate (~700 nucleotide/s) of DNA synthesis in bacteria <a href="#pone.0000447-Norris1">[61]</a>, <a href="#pone.0000447-Mathews1">[79]</a>. It is also significant that glycolytic enzymes can be part of non-metabolic protein complexes that may integrate central carbon metabolism and various cellular functions (see for instance <a href="#pone.0000447-Zheng1">[28]</a>, <a href="#pone.0000447-Jindal1">[43]</a>, <a href="#pone.0000447-Py1">[44]</a>, <a href="#pone.0000447-Danial1">[49]</a>, <a href="#pone.0000447-Cho1">[80]</a>). Moreover, at least three enzymes of the metabolic linker seem to be endowed with a protein kinase activity (GapA, Pgk and PykA) <a href="#pone.0000447-Sirover1">[42]</a>, <a href="#pone.0000447-Popanda1">[45]</a>, <a href="#pone.0000447-Laschet1">[47]</a> (our unpublished observations). This later observation raises the possibility that the activity of the metabolic linker regulates the conformation of replication fork proteins by phosphorylation. Unexpected and fundamental regulation via phosphorylation by a metabolic linker enzyme has recently been shown in human cells, in which nervous flux can be regulated by the GAPDH-driven ATP-dependent phosphorylation of the GABA<sub>A</sub> neuronal receptor <a href="#pone.0000447-Laschet1">[47]</a>.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>Several reports in the literature suggest that the link between DNA replication and cell metabolism is ubiquitous (see above citations). Understanding the underlying mechanisms is thus of general interest. Such studies might also be of medical importance as early events in carcinogenesis, which generally include up-regulation of glycolysis (the Warburg effect) and a decrease in DNA stability and replication fidelity <a href="#pone.0000447-Loeb1">[81]</a>, <a href="#pone.0000447-Gatenby1">[82]</a>, might involve perturbations of the replication/metabolism link.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strain construction and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>B. subtilis</em> strains are listed in Supplementary <a href="#pone.0000447.s001">Table S1</a>, <a href="#pone.0000447.s002">S2</a>, <a href="#pone.0000447.s003">S3</a>. The <em>E. coli</em> strain used for plasmid constructions was DH5α (<em>supE</em>44 <em>supF</em>58 <em>hsdS</em>3(r<sub>B</sub><sup>−</sup>m<sub>B</sub><sup>−</sup>) <em>dapD</em>8 <em>lacY</em>1 <em>glnV</em>44 <em>Δ</em>(<em>gal-uvrB</em>)47 <em>tyrT</em>58 <em>gyrA</em>29 <em>tonA</em>53 <em>Δ</em>(<em>thyA</em>57)). <em>B. subtilis</em> strains carrying the <em>dna</em>(Ts) mutations in a prophage-free background (DGRM1-4) were constructed by transforming TF8A competent cells with total DNA of EDJ strains. Transformants were selected at 30°C on phleomycin (Pm) containing plates (the phleomycin gene - <em>pmr</em> - is located just upstream of <em>dnaE</em>) and the presence of the <em>dna</em>(Ts) mutation was checked by analyzing cell growth and filamentation at restrictive temperature by plating and optical microscopy analysis. To construct <em>dnaH51</em>, <em>dnaC8133</em> and <em>ts-6</em> prophage-free strains (DGRM12, 13 and 266, respectively), a similar strategy was used except that a Trp+ DNA was mixed to the transforming <em>dna</em>(Ts) DNA. Transformants were first selected for Trp+ at 30°C and then toothpicked to select Ts colonies (note that the procedure cured the spectinomycin (Sp) marker of BD54 ts-6). To construct strains (WT for DNA replication functions) lacking Zwf, Pps or PycA metabolic enzymes (DGRM34, 35 and 38), the corresponding genes, cloned into an <em>E. coli</em> vector, were interrupted by an erythromycin (Em) cassette. The mutations were then introduced in the <em>B. subtilis</em> genome by transforming TF8A cells with the plasmid cut in the vector backbone. Strains resulting from a double crossover event were selected by PCR. Metabolic mutations carried by strains provided by different laboratories were transferred in the TF8A context using transformation and appropriate marker selection (DGRM14-17, 25, 27–33, 36 and 37). To label spontaneously isolated suppressors mapping in genes of the <em>gapA</em> operon, the spectinomycin marker (<em>spc</em>) was inserted downstream of the operon (in <em>araE</em>) by transforming spontaneously suppressed strains (DGRM5, 6, 8–11) with a PCR product composed of <em>spc</em> flanked by the 5′ and 3′ region of <em>araE</em>. The yielded strains were named DGRM61, 62, 64, 83, 102 and 105. To label the <em>pykAJP</em> suppressive mutation, the corresponding suppressed strain (DGRM7) was transformed to erythromycin resistance with DNA of the BFS67 strains (from the European collection of <em>B. subtilis</em> mutants <a href="#pone.0000447-Schumann1">[83]</a>) carrying an EmR determinant (<em>ery</em>) in the vicinity of <em>pykA</em>. The presence of the PykA mutation in a representative transformant (DGRM65) was confirmed by showing that the constructed strain grew as slowly as the parental strain in LB and in minimal medium supplemented with glucose. To evaluate the impact of spontaneously isolated suppressive mutations on metabolic enzyme activity, suppressive mutations labeled by the <em>spc</em> or <em>ery</em> markers were transferred into the WT TF8A strain by transformation yielding strains DGRM18–24. Cell slow growth in LB at 45°C was used as a criteria for the presence of the metabolic mutation in the produced strains. A large (~200) collection of strains carrying a metabolic mutation and a replication or division Ts mutation (apart the spontaneously isolated strains) was generated by transforming Ts cells with total DNA extracted from strains containing a labeled metabolic mutation. The genotype of the constructed strains was checked by analyzing their Ts at 51°C (the maximal growth temperature of <em>B. subtilis</em>), growth rate at 30°C (metabolic mutants generally grow slower than WT strains in LB), back-cross studies, PCR analysis and/or Southern blotting. To construct strains (WT or Ts for replication functions) expressing PykA from the IPTG inducible promoter <em>spac-I</em>, a segment corresponding to the 5′ region of <em>pykA</em> (extending from position −42 to +412 according to the first base of the ORF) was generated by PCR and cloned between the <em>Hind</em>III-<em>Bam</em>HI sites of the <em>E. coli</em> (Em<sup>R</sup>) plasmid pMUTIN2 <a href="#pone.0000447-Vagner1">[84]</a>. Once validated by DNA sequencing, the yielded circular plasmid was introduced upstream of the chromosomal <em>pykA</em> gene of TF8A. This puts <em>pykA</em> under the control of <em>Pspac-I</em>. The resulted strain (DGRM26) was verified by PCR and by analyzing growth rate and PykA activity as a function of IPTG concentration. The fusion was then transferred to various <em>dna</em>(Ts) strains yielding DGRM260–265. For complementation studies, WT copies of the <em>pgk</em>, <em>pgm</em>, <em>eno</em> or <em>pykA</em> genes were PCR amplified and cloned downstream of the xylose inducible promoter <em>pxyl</em> of plasmid pAX01, at the <em>Bam</em>HI site <a href="#pone.0000447-Hartl1">[85]</a>. In the generated plasmids (checked by DNA sequencing), the transcriptional fusion is included in a cassette labeled by the <em>spc</em> marker and delimited by the front and back regions of the <em>B. subtilis</em> <em>lacA</em> locus. The labeled fusion was then introduced at the <em>lacA</em> locus of the WT (TF8A) strain by double crossover. The structure of the yielded strains (DGRM242–246) was controlled by PCR. The construction what then introduced by transformation into strains carrying a labeled glycolytic mutation or into <em>dnaE</em>(Ts) strains mutated in a glycolytic gene (DGRM247–259). Conditional expression of the metabolic genes was checked by following cell growth in LB complemented or not with 0.5% xylose. To measure the effect of metabolic mutations on carbon flux polarity, competent cells of the GM1514 strain that contains <em>lacZ</em> downstream from the <em>gapB</em> promoter at the <em>amyE</em> locus were transformed with the total DNA extracted from strains carrying metabolic mutations (168 Pgi::spc, 501–77, GM1501, GTD040 and DGRM18, 25). Transformants were selected on plates supplements with appropriate antibiotics. The presence of the metabolic mutations and of the fusion were verified by analyzing cell growth in LB at 37°C and by PCR, respectively. For strains carrying the <em>pgkEP</em> mutation, the presence of a functional <em>PgapB::lacZ</em> fusion was verified by measuring β-galactosidase activity upon transfer of the structure in the 168 strain. The constructed strains were termed DGRM239–241 and 285. To measure replication promoter activity in suppressed and non suppressed strains, we introduced by transformation the <em>ΔpykA::spc</em> mutation (from strain DGRM25) into strains carrying the <em>PdnaE::-</em>, <em>PpolC::-</em> or <em>PdnaC::lacZ</em> fusions and β-galactosidase activity was measured in the parental PykA+ (HVS597, 607 and 614) and yielded PykA− (DGRM322–324) strains. To investigate whether suppression depends on accumulation of DnaETs proteins, we first constructed PykA+ or PykA− strains carrying the <em>dnaE</em>(Ts) genes under the control of <em>Pspac-I</em>. For this, we transformed the <em>dnaE</em>(Ts) <em>pmr</em> strains (DGRM1-3) with the DNA extracted from cells carrying the <em>Pspac-I-dnaE</em> <em>ery</em> construction (HVS614). Transformants were selected at 30°C on plates containing EmR and 500 µM IPTG. A toothpicking analysis on 100–200 transformants allowed us to identify Ts EmR PmS cells. Growth analysis at 30°C (turbidity and optical microscopy analysis) in liquid and solid rich medium supplemented or not with 500 µM IPTG showed that the selected strains containas expected <em>dnaE</em> under the control of <em>Pspac-I</em>. The strains were termed DGRM230–232. To construct the isogenic PykA- strains, competent <em>spac-I-dnaE</em>(Ts) cells were transformed to SpR by the DNA extracted from the <em>pykA::spc</em> strain (DGRM25). Representative IPTG-dependent, slow growing EmR transformants were selected (DGRM233–235). That these cells carry the <em>dnaE</em>(Ts) mutation was controlled by transferring the EmR region into the 168 strain and showing that most of the transformants were Ts. We then fused the DnaETs proteins to the SPA tag <a href="#pone.0000447-Zeghouf1">[33]</a>. For this, competent cells of the constructed strains were transformed with the total DNA extracted from a strain (FLB5) containing the <em>dnaE-SPA</em> fusion labeled with a <em>neo</em> marker. A Western blot analysis of KmR transformants was carried out using the anti-SPA monoclonal anti-FLAG M2 antibody (Sigma, St Louis, MO). Clones encoding an IPTG inducible fusion protein of the expected size were thus identified. Among these, a genetic screen allowed the selection of representative strains containing the <em>dnaE</em>(Ts) mutation. The genetic analysis indicated that the SPA fusion decreases the activity of the DnaETs proteins as the corresponding cells were more IPTG dependent than the isogenic counterpart lacking the fusion. The structure of the <em>dnaE</em> region in the constructed strains (DGRM304–306 and 308–310) was checked by PCR. Plasmids used in the above mentioned constructions are available upon request.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Cells were grown in LB or minimal medium supplemented with glucose 0.2% or malate 0.5% <a href="#pone.0000447-Harwood1">[86]</a>. MM-CM is composed of minimal medium supplemented with caseinhydrolysate and malate (0,2% each). In case of strains carrying the <em>pta</em> mutation, acetate (1%) was added to LB in order to improve cell growth and prevent appearance of fast growing variants (control experiments showed that this complementation does not interfere with suppression). The <em>ΔgapA</em> strain was grown in the presence of 1 mM IPTG in order to allow <em>Pspac</em>-I-driven expression of the metabolic genes located downstream of <em>gapA</em> <a href="#pone.0000447-Doan1">[87]</a>. To induce the general stress regulon and/or the stringent response, NaCl (2%), sodium azide (125–250 µM), DL-norvaline (0.5 or 2 mg/ml) and/or arginine hydroxamate (250 µM) were added separately or in combination to LB as previously described <a href="#pone.0000447-Levine1">[15]</a>, <a href="#pone.0000447-Zhang1">[88]</a>–<a href="#pone.0000447-Boylan1">[90]</a>. <em>B. subtilis</em> and <em>E. coli</em> competent cells were prepared as previously described and antibiotics were added to the media at regular concentrations <a href="#pone.0000447-Harwood1">[86]</a>, <a href="#pone.0000447-Sambrook1">[91]</a>.</p>


<h4>Isolation and localization of spontaneously isolated suppressive mutations</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>In order to search for suppressive mutations, four <em>dnaE</em>(Ts) strains (DGRM1–4) cured of the three major prophages (SPβ, PBSX and SKIN) were used. Tr derivatives were selected at the lowest temperature reducing 10<sup>4</sup>–10<sup>5</sup> fold the platting efficiency of the Ts mutants (see below the temperature used). For each mutant, 20–40 cultures of 5 mL of LB were inoculated with 10<sup>4</sup> cells and grown at 30°C under shaking. At saturation, 10<sup>6</sup> to 10<sup>7</sup> cells were spread on LB plates and incubated at restrictive temperature. After 48–72 hours of growth, a large and a small colony from each plated culture were selected and streaked for further growth at restrictive temperature. An isolated colony was then cultivated at 30°C in liquid LB and stocked at −80°C. To identify strains carrying an extragenic suppressor, chromosomal DNA of Tr variants was extracted and used to transform the WT 168 strain to Pm<sup>R</sup>. Twenty to fifty transformants selected at 30°C were then toothpicked on Pm plates and grown at high or low temperature. Suppressive mutations were assumed to be extragenic when the linkage between the Tr marker and <em>pmr</em> was ≤85%. This procedure allowed us to identify 29 independent extragenic suppressors.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>To precisely map the suppressor (<em>JP</em>) carried by strain DGRM7 and located in the vicinity to <em>dnaE2.4</em> (genetic linkage of ~80%), a collection of cells carrying an Em<sup>R</sup> marker at various positions in the <em>dnaE</em> region was used (strains of the Bacillus functional analysis - BFA -, see Supplementary <a href="#pone.0000447.s002">Table S2</a>). The DNA of these cells was used to transform competent cells of the suppressed <em>dnaE2.4 JP</em> strain. Transformants were selected on Em + Pm plates at 30°C. For each transformation, the proportion of Tr colonies among 50 Em<sup>R</sup> Pm<sup>R</sup> transformants was determined. The highest Em<sup>R</sup>-<em>JP</em> linkage was observed with strains BFA93 (60%) and BFA91 (45%) suggesting that the <em>JP</em> mutation maps ~5 kb downstream of <em>dnaE</em>. This was confirmed by transforming the original <em>dnaE2.4</em> strain to thermoresistance with PCR fragments generated from the suppressed strain and containing overlapping sequences downstream of <em>dnaE</em>. DNA sequencing of the region revealed a 81 bp deletion in the central part of the glycolytic <em>pykA</em> gene. This mutation was termed <em>pykAJP</em>.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Mapping of <em>sup8</em>, a <em>dnaE2.6</em> suppressor unlinked to <em>dnaE</em> (strain DGRM9), was carried out by random transposition as described previously <a href="#pone.0000447-Petit1">[92]</a>. To enrich for cells carrying the transposon (<em>Tn</em>) in the vicinity to <em>sup8</em>, DNA of a large pool of cells that had suffered transposition was extracted, used to transform the parental <em>dnaE2.6</em> strain to Cm<sup>R</sup> (the <em>Tn</em> marker) at 30°C and Cm<sup>R</sup> Tr transformants were selected by replica-plating. This second generation Cm<sup>R</sup> Tr cells was then pooled and processed as above for further enrichment of <em>Tn</em>-<em>sup8</em> linked genomes. Genetic linkage analysis showed that a representative Cm<sup>R</sup> Tr strain carried <em>Tn</em> in the vicinity of <em>sup8</em> (linkage of 24%). To further map <em>Tn</em>, the DNA of this strain was restricted with several enzymes, run on agarose gel along with DNA ladders, and analyzed by Southern blot using as a probe a <sup>32</sup>P labeled DNA homologous to <em>Tn</em>. The size of the <em>Tn</em>-containing segments was then compared to the predicted restriction map of the <em>B. subtilis</em> genome using a software developed in the laboratory (available upon request). This allowed us to map <em>Tn</em> in a 585 bp segment located between coordinates 3,444 862 to 3,445 447 of the <em>B. subtilis</em> genome. This location was confirmed by PCR. To map <em>sup8</em>, mutants of the BFA collection carrying the Em<sup>R</sup> determinant in the <em>Tn</em> region were used (see <a href="#pone.0000447.s002">Supplementary Table 2</a>). The highest Em<sup>R</sup>-<em>sup8</em> linkage (90%) was observed with strain BFA1079 which carries the Em<sup>R</sup> determinant in <em>yvbK</em>. DNA sequencing of the region revealed a single mutation (termed <em>pgm8</em>) in the glycolytic <em>pgm</em> gene, ~8 kb downstream of <em>yvbK</em>.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>In order to determine whether additional suppressive mutations map in genes of the central carbon metabolism, the genetic linkage between suppressors and an Em<sup>R</sup> marker located in the vicinity of metabolic genes in BFA mutants (see <a href="#pone.0000447.s002">Supplementary Table S2</a>) was investigated. For this, competent cells of 5 suppressed strains (DGRM5, 6, 8, 10 and 11) were transformed to Em<sup>R</sup> at 30°C with DNA of BFA cells. The proportion of Ts/Tr transformants was then determined by toothpicking 20–50 colonies on selective plates incubated at permissive or restrictive temperature. Surprisingly, all the suppressive mutations were located nearby <em>yvbK</em> like <em>sup8</em> (linkage of 60–90%). As the suppressed strains grew poorly on minimal medium supplemented with glucose, it was speculated that they carry a mutation in the glycolytic genes located downstream from <em>yvbK</em>. Consistently, single mutations were found by sequencing in <em>pgk</em> (<em>pgkEP</em>), <em>pgm</em> (<em>pgmIP</em> and <em>pgm25</em>) and <em>eno</em> (<em>enoLP</em>). All the suppressive mutations are described in <a href="#pone-0000447-t002">Table 2</a>.</p>


<h4>Restrictive temperatures for <em>dna</em>(Ts) mutants</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Restrictive temperatures, defined as the lowest temperatures reducing 10<sup>4</sup>–10<sup>5</sup> fold the plating efficiency of Ts mutants on LB (for the plating assay) or reducing 10<sup>1</sup>–10<sup>3</sup> cell viability after 2.5 hours of incubation in LB broth (for the filamentation assay), were as follows (P: plating assay; F: filamentation assay): <em>dnaE2.2</em> and <em>dnaE2.4</em> (P: 47°C, F: 44°C); <em>dnaE2.6</em> (P: 42°C; F: 40°C); <em>dnaE2.10</em> (P: 49°C; F: 47°C); <em>dnaG20</em> (P: 45°C; F: 42°C); <em>dnaC14</em> (P: 49°C: F: 45°C); <em>dnaC30</em> (P: 45°C; F: 43°C); <em>dnaC8133</em> (P: 47°C; F: 45°C); <em>dnaX8132</em> (P: 45°C; F: 45°C); <em>dnaF33</em> (P: 37°C); <em>dnaF69</em> (P: 42°C); <em>dnaH51, ts-6</em>, <em>dnaB37</em> and <em>dnaD23</em> (P: 45°C); <em>dnaG5</em> and <em>dnaI2</em> (P: 47°C). For studies carried out in MM-CM, selective temperatures were as above except for <em>dnaE2.2</em> (P: 45°C; F: 42°C) and <em>dnaE2.4</em> (P: 47°C; F: 42°C).</p>


<h4>DNA/protein manipulations and enzymatic assays</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>DNA extraction and manipulations were carried out according to standard procedures <a href="#pone.0000447-Harwood1">[86]</a>, <a href="#pone.0000447-Sambrook1">[91]</a>. DNA sequencing was carried out on PCR products with the PRISM Sequencing Kit and the 377A sequencing apparatus from Applied Biosystem (Warrington) according to the manufacturer recommendation. Western blot analysis of DnaE-SPA production, β-Galactosidase activity and the amylase assay were carried out as previously described <a href="#pone.0000447-Schumann1">[83]</a>, <a href="#pone.0000447-LeChatelier1">[93]</a>. Protein concentrations were determined with a Bio-Rad Protein Assay using BSA as a standard. β-Galactosidase is expressed in Miller units per mg of protein. PykA activity was measured by employing the linked lactic dehydrogenase assay, as modified <a href="#pone.0000447-Bcher1">[94]</a>, <a href="#pone.0000447-Tuominen1">[95]</a>. Crude extracts were prepared as follows: cells from 25 ml cultures at OD<sub>600</sub> = 0.3 were centrifuged, resuspended in 75 µl of lysis buffer (60 mM Na<sub>2</sub>HPO<sub>4</sub>, 40 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM KCl, 1 mM MgSO<sub>4</sub>, 1mM DTT, 0.1 mg/ml lysozyme, 0.01 mg/ml Dnase), incubated 20 mn on ice, 5 mn at 37°C, 15 mn at 20°C, centrifuged 10 mn at 12000 g and the supernantant was stored at −20°C. The standard assay system contained in 1 ml: 50 mM Imidazole HCl pH 7.1, 10 mM MgCl<sub>2</sub>, 50 mM KCl, 5 mM ADP, 5 mM phosphoenolpyruvate, 0.2 mM NADH, 10 units lactate dehydrogenase. Reactions were initiated by the addition of 0.5 to 10 µl of crude extract in a 1-cm quartz cuvette, and kinetics of NADH oxidation was followed at 340 nm in a Lambda 20 Perkin Elmer spectrophotometer at 25°C. One unit of enzyme is defined as the amount required to convert 1 µmole of phosphoenolpyruvate to pyruvate per min in the standard assay.</p>


<h4>Microscopy analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>To check for cell filamentation, parental Ts strains incubated 2–3 hours at restrictive temperature and suppressed strains grown for at least 5 generations were stained 5 mn in the presence of 4′,6′-diamidino-2-phenylindole (DAPI) and FM5-95 (Molecular Probes) for visualizing the nucleoid and membrane, respectively. They were then deposited on glass slides covered with 1.2% agarose in minimal medium, and examined with a Leica (Leica DMRA2, Leica Microsystems GmbH, Wetzlar, Nl) microscope with a 100× magnification oil-immersion objective and a CDD camera (Photometrics CoolSNAP HQ, Roper Scientific Inc, Duluth, GA). Cell pictures were captured with METAMORPH V5.0 (Universal Imaging, Media, PA, USA).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000447"></div><a name="pone.0000447.s001" id="pone.0000447.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000447.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000447.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.05 MB XLS)</p>
<a name="pone.0000447.s002" id="pone.0000447.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000447.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000447.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0000447.s003" id="pone.0000447.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000447.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000447.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.05 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank undergraduate students who participated in this study and teams of the laboratory that supported this long-term project. We warmly thank Arthur L. Aronson, Stéphane Aymerich, Neal C. Brown, Michel Debarbouillé, Josef Deutsher, Jahn Dieter, Alessandro Galizzi, Elizabeth J. Harry, François Lecointe, Uwe Sauer, Simone J. Seror, Abraham L. Sonenshein and Jörge Stülke for providing us with various strains. Stéphane Aymerich, François Lecointe, Gradimir Misevic, Philippe Noirot, Victor Norris, Gérard Pierron and Uwe Sauer are warmly thanked for invaluable support, stimulating discussions and/or critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: LJ. Performed the experiments: LJ DC BD AM EL CS SK RL JC MT. Analyzed the data: LJ DC BD EL CS RL MT. Contributed reagents/materials/analysis tools: LJ DC BD AM AB EL CS SK RL JC MT. Wrote the paper: LJ DC EL SE. Other: Decided to submit the work for publication: LJ. Made useful comments on the paper: EL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000447-McHenry1" id="pone.0000447-McHenry1"></a>McHenry CS (2003) Chromosomal replicases as asymmetric dimers: studies of subunit arrangement and functional consequences. Mol Microbiol  49: 1157–1165.  <ul class="find" data-citedArticleID="951844" data-doi="10.1046/j.1365-2958.2003.03645.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03645.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+replicases+as+asymmetric+dimers%3A+studies+of+subunit+arrangement+and+functional+consequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+replicases+as+asymmetric+dimers%3A+studies+of+subunit+arrangement+and+functional+consequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000447-Johnson1" id="pone.0000447-Johnson1"></a>Johnson A, O'Donnell M (2005) Cellular DNA replicases: components and dynamics at the replication fork. Annu Rev Biochem  74: 283–315.  <ul class="find" data-citedArticleID="951810" data-doi="10.1146/annurev.biochem.73.011303.073859"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.73.011303.073859" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+DNA+replicases%3A+components+and+dynamics+at+the+replication+fork." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+DNA+replicases%3A+components+and+dynamics+at+the+replication+fork.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000447-Barry1" id="pone.0000447-Barry1"></a>Barry ER, Bell SD (2006) DNA replication in the archaea. Microbiol Mol Biol Rev  70: 876–887.  <ul class="find" data-citedArticleID="951738" data-doi="10.1128/mmbr.00029-06"><li><a href="http://dx.doi.org/10.1128/mmbr.00029-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+in+the+archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+in+the+archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000447-Titok1" id="pone.0000447-Titok1"></a>Titok M, Suski C, Dalmais B, Ehrlich SD, Janniere L (2006) The replicative polymerases PolC and DnaE are required for theta replication of the Bacillus subtilis plasmid pBS72. Microbiology  152: 1471–1478.  <ul class="find" data-citedArticleID="951902" data-doi="10.1099/mic.0.28693-0"><li><a href="http://dx.doi.org/10.1099/mic.0.28693-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+replicative+polymerases+PolC+and+DnaE+are+required+for+theta+replication+of+the+Bacillus+subtilis+plasmid+pBS72." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+replicative+polymerases+PolC+and+DnaE+are+required+for+theta+replication+of+the+Bacillus+subtilis+plasmid+pBS72.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000447-Dervyn1" id="pone.0000447-Dervyn1"></a>Dervyn E, Suski C, Daniel R, Bruand C, Chapuis J, et al.  (2001) Two essential DNA polymerases at the bacterial replication fork. Science  294: 1716–1719.  <ul class="find" data-citedArticleID="951768" data-doi="10.1126/science.1066351"><li><a href="http://dx.doi.org/10.1126/science.1066351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+essential+DNA+polymerases+at+the+bacterial+replication+fork." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+essential+DNA+polymerases+at+the+bacterial+replication+fork.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000447-Bruck1" id="pone.0000447-Bruck1"></a>Bruck I, O'Donnell M (2000) The DNA replication machine of a gram-positive organism. J Biol Chem  275: 28971–28983.  <ul class="find" data-citedArticleID="951748" data-doi="10.1074/jbc.m003565200"><li><a href="http://dx.doi.org/10.1074/jbc.m003565200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DNA+replication+machine+of+a+gram-positive+organism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DNA+replication+machine+of+a+gram-positive+organism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000447-Velten1" id="pone.0000447-Velten1"></a>Velten M, McGovern S, Marsin S, Ehrlich SD, Noirot P, et al.  (2003) A two-protein strategy for the functional loading of a cellular replicative DNA helicase. Mol Cell  11: 1–20.  <ul class="find" data-citedArticleID="951910" data-doi="10.1016/s1097-2765(03)00130-8"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00130-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+two-protein+strategy+for+the+functional+loading+of+a+cellular+replicative+DNA+helicase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+two-protein+strategy+for+the+functional+loading+of+a+cellular+replicative+DNA+helicase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000447-Bell1" id="pone.0000447-Bell1"></a>Bell SP, Dutta A (2002) DNA replication in eukaryotic cells. Annu Rev Biochem  71: 333–374.  <ul class="find" data-citedArticleID="951742" data-doi="10.1146/annurev.biochem.71.110601.135425"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.71.110601.135425" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+in+eukaryotic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+in+eukaryotic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000447-Katayama1" id="pone.0000447-Katayama1"></a>Katayama T (2001) Feedback controls restrain the initiation of Escherichia coli chromosomal replication. Mol Microbiol  41: 9–17.  <ul class="find" data-citedArticleID="951814" data-doi="10.1046/j.1365-2958.2001.02483.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02483.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feedback+controls+restrain+the+initiation+of+Escherichia+coli+chromosomal+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Feedback+controls+restrain+the+initiation+of+Escherichia+coli+chromosomal+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000447-Kaguni1" id="pone.0000447-Kaguni1"></a>Kaguni JM (2006) DnaA: controlling the initiation of bacterial DNA replication and more. Annu Rev Microbiol  60: 351–371.  <ul class="find" data-citedArticleID="951812" data-doi="10.1146/annurev.micro.60.080805.142111"><li><a href="http://dx.doi.org/10.1146/annurev.micro.60.080805.142111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DnaA%3A+controlling+the+initiation+of+bacterial+DNA+replication+and+more." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DnaA%3A+controlling+the+initiation+of+bacterial+DNA+replication+and+more.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000447-Helmstetter1" id="pone.0000447-Helmstetter1"></a>Helmstetter CE (1996)  Timing of synthetic activities in the cell cycle. In: Neidhart FC, Curtis R III, Ingraham JL, Lin ECC, Low KB, et al., editors. Escherichia coli and Salmonella. Washington, DC: ASM Press. pp. 1627–1639.  <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0000447-Michelsen1" id="pone.0000447-Michelsen1"></a>Michelsen O, Teixeira de Mattos MJ, Jensen PR, Hansen FG (2003) Precise determinations of C and D periods by flow cytometry in Escherichia coli K-12 and B/r. Microbiology  149: 1001–1010.  <ul class="find" data-citedArticleID="951848" data-doi="10.1099/mic.0.26058-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26058-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Precise+determinations+of+C+and+D+periods+by+flow+cytometry+in+Escherichia+coli+K-12+and+B%2Fr." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Precise+determinations+of+C+and+D+periods+by+flow+cytometry+in+Escherichia+coli+K-12+and+B%2Fr.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000447-Zyskind1" id="pone.0000447-Zyskind1"></a>Zyskind JW, Smith DW (1992) DNA replication, the bacterial cell cycle, and cell growth. Cell  69: 5–8.  <ul class="find" data-citedArticleID="951928" data-doi="10.1016/0092-8674(92)90112-p"><li><a href="http://dx.doi.org/10.1016/0092-8674(92)90112-p" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication%2C+the+bacterial+cell+cycle%2C+and+cell+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication%2C+the+bacterial+cell+cycle%2C+and+cell+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000447-Schreiber1" id="pone.0000447-Schreiber1"></a>Schreiber G, Ron EZ, Glaser G (1995) ppGpp-mediated regulation of DNA replication and cell division in Escherichia coli. Curr Microbiol  30: 27–32.  <ul class="find" data-citedArticleID="951888" data-doi="10.1007/bf00294520"><li><a href="http://dx.doi.org/10.1007/bf00294520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ppGpp-mediated+regulation+of+DNA+replication+and+cell+division+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ppGpp-mediated+regulation+of+DNA+replication+and+cell+division+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000447-Levine1" id="pone.0000447-Levine1"></a>Levine A, Vannier F, Dehbi M, Henckes G, Seror SJ (1991) The stringent response blocks DNA replication outside the ori region in Bacillus subtilis and at the origin in Escherichia coli. J Mol Biol  219: 605–613.  <ul class="find" data-citedArticleID="951832" data-doi="10.1016/0022-2836(91)90657-r"><li><a href="http://dx.doi.org/10.1016/0022-2836(91)90657-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+stringent+response+blocks+DNA+replication+outside+the+ori+region+in+Bacillus+subtilis+and+at+the+origin+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+stringent+response+blocks+DNA+replication+outside+the+ori+region+in+Bacillus+subtilis+and+at+the+origin+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000447-Cashel1" id="pone.0000447-Cashel1"></a>Cashel M, Gentry DR, Hernandez VH, Vinella D (1996)  The stringent response. In: Neidhart FC, Curtis R III, Ingraham JL, Lin ECC, Low KB, et al., editors. Escherichia coli and Salmonella. Washington, DC: ASM Press. pp. 1458–1496.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000447-Wegrzyn1" id="pone.0000447-Wegrzyn1"></a>Wegrzyn G, Wegrzyn A (2002) Stress responses and replication of plasmids in bacterial cells. Microb Cell Fact  1: 2.  <ul class="find" data-citedArticleID="951916" data-doi="10.1186/1475-2859-1-2"><li><a href="http://dx.doi.org/10.1186/1475-2859-1-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stress+responses+and+replication+of+plasmids+in+bacterial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stress+responses+and+replication+of+plasmids+in+bacterial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000447-Tu1" id="pone.0000447-Tu1"></a>Tu BP, Kudlicki A, Rowicka M, McKnight SL (2005) Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes. Science  310: 1152–1158.  <ul class="find" data-citedArticleID="951904" data-doi="10.1126/science.1120499"><li><a href="http://dx.doi.org/10.1126/science.1120499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Logic+of+the+yeast+metabolic+cycle%3A+temporal+compartmentalization+of+cellular+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Logic+of+the+yeast+metabolic+cycle%3A+temporal+compartmentalization+of+cellular+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000447-Klevecz1" id="pone.0000447-Klevecz1"></a>Klevecz RR, Bolen J, Forrest G, Murray DB (2004) A genomewide oscillation in transcription gates DNA replication and cell cycle. Proc Natl Acad Sci U S A  101: 1200–1205.  <ul class="find" data-citedArticleID="951820" data-doi="10.1073/pnas.0306490101"><li><a href="http://dx.doi.org/10.1073/pnas.0306490101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomewide+oscillation+in+transcription+gates+DNA+replication+and+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomewide+oscillation+in+transcription+gates+DNA+replication+and+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000447-Riedinger1" id="pone.0000447-Riedinger1"></a>Riedinger HJ, van Betteraey-Nikoleit M, Hilfrich U, Eisele KH, Probst H (2001) Oxygen-dependent regulation of in vivo replication of simian virus 40 DNA is modulated by glucose. J Biol Chem  276: 47122–47130.  <ul class="find" data-citedArticleID="951880" data-doi="10.1074/jbc.m106938200"><li><a href="http://dx.doi.org/10.1074/jbc.m106938200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxygen-dependent+regulation+of+in+vivo+replication+of+simian+virus+40+DNA+is+modulated+by+glucose." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxygen-dependent+regulation+of+in+vivo+replication+of+simian+virus+40+DNA+is+modulated+by+glucose.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000447-Chen1" id="pone.0000447-Chen1"></a>Chen Y, Tye BK (1995) The yeast Mcm1 protein is regulated posttranscriptionally by the flux of glycolysis. Mol Cell Biol  15: 4631–4639.  <ul class="find" data-citedArticleID="951758"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+yeast+Mcm1+protein+is+regulated+posttranscriptionally+by+the+flux+of+glycolysis.&amp;auth=&amp;atitle=The+yeast+Mcm1+protein+is+regulated+posttranscriptionally+by+the+flux+of+glycolysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+yeast+Mcm1+protein+is+regulated+posttranscriptionally+by+the+flux+of+glycolysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+yeast+Mcm1+protein+is+regulated+posttranscriptionally+by+the+flux+of+glycolysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000447-Maine1" id="pone.0000447-Maine1"></a>Maine GT, Sinha P, Tye BK (1984) Mutants of S. cerevisiae defective in the maintenance of minichromosomes. Genetics  106: 365–385.  <ul class="find" data-citedArticleID="951840"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutants+of+S.+cerevisiae+defective+in+the+maintenance+of+minichromosomes.&amp;auth=&amp;atitle=Mutants+of+S.+cerevisiae+defective+in+the+maintenance+of+minichromosomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutants+of+S.+cerevisiae+defective+in+the+maintenance+of+minichromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutants+of+S.+cerevisiae+defective+in+the+maintenance+of+minichromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000447-Chang1" id="pone.0000447-Chang1"></a>Chang VK, Fitch MJ, Donato JJ, Christensen TW, Merchant AM, et al.  (2003) Mcm1 binds replication origins. J Biol Chem  278: 6093–6100.  <ul class="find" data-citedArticleID="951754" data-doi="10.1074/jbc.m209827200"><li><a href="http://dx.doi.org/10.1074/jbc.m209827200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mcm1+binds+replication+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mcm1+binds+replication+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000447-Chang2" id="pone.0000447-Chang2"></a>Chang VK, Donato JJ, Chan CS, Tye BK (2004) Mcm1 promotes replication initiation by binding specific elements at replication origins. Mol Cell Biol  24: 6514–6524.  <ul class="find" data-citedArticleID="951756" data-doi="10.1128/mcb.24.14.6514-6524.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.14.6514-6524.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mcm1+promotes+replication+initiation+by+binding+specific+elements+at+replication+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mcm1+promotes+replication+initiation+by+binding+specific+elements+at+replication+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000447-Fitch1" id="pone.0000447-Fitch1"></a>Fitch MJ, Donato JJ, Tye BK (2003) Mcm7, a subunit of the presumptive MCM helicase, modulates its own expression in conjunction with Mcm1. J Biol Chem  278: 25408–25416.  <ul class="find" data-citedArticleID="951778" data-doi="10.1074/jbc.m300699200"><li><a href="http://dx.doi.org/10.1074/jbc.m300699200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mcm7%2C+a+subunit+of+the+presumptive+MCM+helicase%2C+modulates+its+own+expression+in+conjunction+with+Mcm1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mcm7%2C+a+subunit+of+the+presumptive+MCM+helicase%2C+modulates+its+own+expression+in+conjunction+with+Mcm1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000447-Maher1" id="pone.0000447-Maher1"></a>Maher M, Cong F, Kindelberger D, Nasmyth K, Dalton S (1995) Cell cycle-regulated transcription of the CLB2 gene is dependent on Mcm1 and a ternary complex factor. Mol Cell Biol  15: 3129–3137.  <ul class="find" data-citedArticleID="951836"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell+cycle-regulated+transcription+of+the+CLB2+gene+is+dependent+on+Mcm1+and+a+ternary+complex+factor.&amp;auth=&amp;atitle=Cell+cycle-regulated+transcription+of+the+CLB2+gene+is+dependent+on+Mcm1+and+a+ternary+complex+factor." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+cycle-regulated+transcription+of+the+CLB2+gene+is+dependent+on+Mcm1+and+a+ternary+complex+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+cycle-regulated+transcription+of+the+CLB2+gene+is+dependent+on+Mcm1+and+a+ternary+complex+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000447-Mai1" id="pone.0000447-Mai1"></a>Mai B, Miles S, Breeden LL (2002) Characterization of the ECB binding complex responsible for the M/G(1)-specific transcription of CLN3 and SWI4. Mol Cell Biol  22: 430–441.  <ul class="find" data-citedArticleID="951838" data-doi="10.1128/mcb.22.2.430-441.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.2.430-441.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+ECB+binding+complex+responsible+for+the+M%2FG%281%29-specific+transcription+of+CLN3+and+SWI4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+ECB+binding+complex+responsible+for+the+M%2FG%281%29-specific+transcription+of+CLN3+and+SWI4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000447-Zheng1" id="pone.0000447-Zheng1"></a>Zheng L, Roeder RG, Luo Y (2003) S phase activation of the histone H2B promoter by OCA-S, a coactivator complex that contains GAPDH as a key component. Cell  114: 255–266.  <ul class="find" data-citedArticleID="951926" data-doi="10.1016/s0092-8674(03)00552-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00552-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=S+phase+activation+of+the+histone+H2B+promoter+by+OCA-S%2C+a+coactivator+complex+that+contains+GAPDH+as+a+key+component." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22S+phase+activation+of+the+histone+H2B+promoter+by+OCA-S%2C+a+coactivator+complex+that+contains+GAPDH+as+a+key+component.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000447-Reeve1" id="pone.0000447-Reeve1"></a>Reeve JN, Groves DJ, Clark DJ (1970) Regulation of Cell Division in Escherichia coli: Characterization of Temperature-Sensitive Division Mutants. J Bacteriol  104: 1052–1064.  <ul class="find" data-citedArticleID="951876"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+Cell+Division+in+Escherichia+coli%3A+Characterization+of+Temperature-Sensitive+Division+Mutants.&amp;auth=&amp;atitle=Regulation+of+Cell+Division+in+Escherichia+coli%3A+Characterization+of+Temperature-Sensitive+Division+Mutants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+Cell+Division+in+Escherichia+coli%3A+Characterization+of+Temperature-Sensitive+Division+Mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+Cell+Division+in+Escherichia+coli%3A+Characterization+of+Temperature-Sensitive+Division+Mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000447-Ricard1" id="pone.0000447-Ricard1"></a>Ricard M, Hirota Y (1973) Process of cellular division in Escherichia coli: physiological study on thermosensitive mutants defective in cell division. J Bacteriol  116: 314–322.  <ul class="find" data-citedArticleID="951878"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Process+of+cellular+division+in+Escherichia+coli%3A+physiological+study+on+thermosensitive+mutants+defective+in+cell+division.&amp;auth=&amp;atitle=Process+of+cellular+division+in+Escherichia+coli%3A+physiological+study+on+thermosensitive+mutants+defective+in+cell+division." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Process+of+cellular+division+in+Escherichia+coli%3A+physiological+study+on+thermosensitive+mutants+defective+in+cell+division." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Process+of+cellular+division+in+Escherichia+coli%3A+physiological+study+on+thermosensitive+mutants+defective+in+cell+division.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000447-Henson1" id="pone.0000447-Henson1"></a>Henson JM, Chu H, Irwin CA, Walker JR (1979) Isolation and characterization of dnaX and dnaY temperature-sensitive mutants of Escherichia coli. Genetics  92: 1041–1059.  <ul class="find" data-citedArticleID="951806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+and+characterization+of+dnaX+and+dnaY+temperature-sensitive+mutants+of+Escherichia+coli.&amp;auth=&amp;atitle=Isolation+and+characterization+of+dnaX+and+dnaY+temperature-sensitive+mutants+of+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+dnaX+and+dnaY+temperature-sensitive+mutants+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+dnaX+and+dnaY+temperature-sensitive+mutants+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000447-Fillinger1" id="pone.0000447-Fillinger1"></a>Fillinger S, Boschi-Muller S, Azza S, Dervyn E, Branlant G, et al.  (2000) Two glyceraldehyde-3-phosphate dehydrogenases with opposite physiological roles in a nonphotosynthetic bacterium. J Biol Chem  275: 14031–14037.  <ul class="find" data-citedArticleID="951776" data-doi="10.1074/jbc.275.19.14031"><li><a href="http://dx.doi.org/10.1074/jbc.275.19.14031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+glyceraldehyde-3-phosphate+dehydrogenases+with+opposite+physiological+roles+in+a+nonphotosynthetic+bacterium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+glyceraldehyde-3-phosphate+dehydrogenases+with+opposite+physiological+roles+in+a+nonphotosynthetic+bacterium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000447-Zeghouf1" id="pone.0000447-Zeghouf1"></a>Zeghouf M, Li J, Butland G, Borkowska A, Canadien V, et al.  (2004) Sequential Peptide Affinity (SPA) system for the identification of mammalian and bacterial protein complexes. J Proteome Res  3: 463–468.  <ul class="find" data-citedArticleID="951922" data-doi="10.1021/pr034084x"><li><a href="http://dx.doi.org/10.1021/pr034084x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequential+Peptide+Affinity+%28SPA%29+system+for+the+identification+of+mammalian+and+bacterial+protein+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequential+Peptide+Affinity+%28SPA%29+system+for+the+identification+of+mammalian+and+bacterial+protein+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000447-NoirotGros1" id="pone.0000447-NoirotGros1"></a>Noirot-Gros MF, Dervyn E, Wu LJ, Mervelet P, Errington J, et al.  (2002) An expanded view of bacterial DNA replication. Proc Natl Acad Sci U S A  99: 8342–8347.  <ul class="find" data-citedArticleID="951856" data-doi="10.1073/pnas.122040799"><li><a href="http://dx.doi.org/10.1073/pnas.122040799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+expanded+view+of+bacterial+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+expanded+view+of+bacterial+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000447-FothergillGilmore1" id="pone.0000447-FothergillGilmore1"></a>Fothergill-Gilmore LA, Michels PA (1993) Evolution of glycolysis. Prog Biophys Mol Biol  59: 105–235.  <ul class="find" data-citedArticleID="951782" data-doi="10.1016/0079-6107(93)90001-z"><li><a href="http://dx.doi.org/10.1016/0079-6107(93)90001-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+glycolysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+glycolysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000447-Canback1" id="pone.0000447-Canback1"></a>Canback B, Andersson SG, Kurland CG (2002) The global phylogeny of glycolytic enzymes. Proc Natl Acad Sci U S A  99: 6097–6102.  <ul class="find" data-citedArticleID="951750" data-doi="10.1073/pnas.082112499"><li><a href="http://dx.doi.org/10.1073/pnas.082112499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+global+phylogeny+of+glycolytic+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+global+phylogeny+of+glycolytic+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000447-Giaever1" id="pone.0000447-Giaever1"></a>Giaever G, Chu AM, Ni L, Connelly C, Riles L, et al.  (2002) Functional profiling of the Saccharomyces cerevisiae genome. Nature  418: 387–391.  <ul class="find" data-citedArticleID="951790" data-doi="10.1038/nature00935"><li><a href="http://dx.doi.org/10.1038/nature00935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+profiling+of+the+Saccharomyces+cerevisiae+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+profiling+of+the+Saccharomyces+cerevisiae+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000447-Gerdes1" id="pone.0000447-Gerdes1"></a>Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, et al.  (2003) Experimental determination and system level analysis of essential genes in Escherichia coli MG1655. J Bacteriol  185: 5673–5684.  <ul class="find" data-citedArticleID="951788" data-doi="10.1128/jb.185.19.5673-5684.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.19.5673-5684.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+determination+and+system+level+analysis+of+essential+genes+in+Escherichia+coli+MG1655." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+determination+and+system+level+analysis+of+essential+genes+in+Escherichia+coli+MG1655.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000447-Kobayashi1" id="pone.0000447-Kobayashi1"></a>Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, et al.  (2003) Essential Bacillus subtilis genes. Proc Natl Acad Sci U S A  100: 4678–4683.  <ul class="find" data-citedArticleID="951822"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Essential+Bacillus+subtilis+genes.&amp;auth=&amp;atitle=Essential+Bacillus+subtilis+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+Bacillus+subtilis+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+Bacillus+subtilis+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000447-Kim1" id="pone.0000447-Kim1"></a>Kim JW, Dang CV (2005) Multifaceted roles of glycolytic enzymes. Trends Biochem Sci  30: 142–150.  <ul class="find" data-citedArticleID="951818" data-doi="10.1016/j.tibs.2005.01.005"><li><a href="http://dx.doi.org/10.1016/j.tibs.2005.01.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multifaceted+roles+of+glycolytic+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multifaceted+roles+of+glycolytic+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000447-Pancholi1" id="pone.0000447-Pancholi1"></a>Pancholi V (2001) Multifunctional alpha-enolase: its role in diseases. Cell Mol Life Sci  58: 902–920.  <ul class="find" data-citedArticleID="951862" data-doi="10.1007/pl00000910"><li><a href="http://dx.doi.org/10.1007/pl00000910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multifunctional+alpha-enolase%3A+its+role+in+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multifunctional+alpha-enolase%3A+its+role+in+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000447-Sirover1" id="pone.0000447-Sirover1"></a>Sirover MA (1999) New insights into an old protein: the functional diversity of mammalian glyceraldehyde-3-phosphate dehydrogenase. Biochim Biophys Acta  1432: 159–184.  <ul class="find" data-citedArticleID="951894" data-doi="10.1016/s0167-4838(99)00119-3"><li><a href="http://dx.doi.org/10.1016/s0167-4838(99)00119-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+insights+into+an+old+protein%3A+the+functional+diversity+of+mammalian+glyceraldehyde-3-phosphate+dehydrogenase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+insights+into+an+old+protein%3A+the+functional+diversity+of+mammalian+glyceraldehyde-3-phosphate+dehydrogenase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000447-Jindal1" id="pone.0000447-Jindal1"></a>Jindal HK, Vishwanatha JK (1990) Functional identity of a primer recognition protein as phosphoglycerate kinase. J Biol Chem  265: 6540–6543.  <ul class="find" data-citedArticleID="951808"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+identity+of+a+primer+recognition+protein+as+phosphoglycerate+kinase.&amp;auth=&amp;atitle=Functional+identity+of+a+primer+recognition+protein+as+phosphoglycerate+kinase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+identity+of+a+primer+recognition+protein+as+phosphoglycerate+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+identity+of+a+primer+recognition+protein+as+phosphoglycerate+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000447-Py1" id="pone.0000447-Py1"></a>Py B, Higgins CF, Krisch HM, Carpousis AJ (1996) A DEAD-box RNA helicase in the Escherichia coli RNA degradosome. Nature  381: 169–172.  <ul class="find" data-citedArticleID="951874" data-doi="10.1038/381169a0"><li><a href="http://dx.doi.org/10.1038/381169a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+DEAD-box+RNA+helicase+in+the+Escherichia+coli+RNA+degradosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+DEAD-box+RNA+helicase+in+the+Escherichia+coli+RNA+degradosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000447-Popanda1" id="pone.0000447-Popanda1"></a>Popanda O, Fox G, Thielmann HW (1998) Modulation of DNA polymerases alpha, delta and epsilon by lactate dehydrogenase and 3-phosphoglycerate kinase. Biochim Biophys Acta  1397: 102–117.  <ul class="find" data-citedArticleID="951872" data-doi="10.1016/s0167-4781(97)00229-7"><li><a href="http://dx.doi.org/10.1016/s0167-4781(97)00229-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+DNA+polymerases+alpha%2C+delta+and+epsilon+by+lactate+dehydrogenase+and+3-phosphoglycerate+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+DNA+polymerases+alpha%2C+delta+and+epsilon+by+lactate+dehydrogenase+and+3-phosphoglycerate+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000447-Lay1" id="pone.0000447-Lay1"></a>Lay AJ, Jiang XM, Kisker O, Flynn E, Underwood A, et al.  (2000) Phosphoglycerate kinase acts in tumour angiogenesis as a disulphide reductase. Nature  408: 869–873.  <ul class="find" data-citedArticleID="951828" data-doi="10.1038/35048596"><li><a href="http://dx.doi.org/10.1038/35048596" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphoglycerate+kinase+acts+in+tumour+angiogenesis+as+a+disulphide+reductase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphoglycerate+kinase+acts+in+tumour+angiogenesis+as+a+disulphide+reductase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000447-Laschet1" id="pone.0000447-Laschet1"></a>Laschet JJ, Minier F, Kurcewicz I, Bureau MH, Trottier S, et al.  (2004) Glyceraldehyde-3-phosphate dehydrogenase is a GABAA receptor kinase linking glycolysis to neuronal inhibition. J Neurosci  24: 7614–7622.  <ul class="find" data-citedArticleID="951826" data-doi="10.1523/jneurosci.0868-04.2004"><li><a href="http://dx.doi.org/10.1523/jneurosci.0868-04.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glyceraldehyde-3-phosphate+dehydrogenase+is+a+GABAA+receptor+kinase+linking+glycolysis+to+neuronal+inhibition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glyceraldehyde-3-phosphate+dehydrogenase+is+a+GABAA+receptor+kinase+linking+glycolysis+to+neuronal+inhibition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000447-Yao1" id="pone.0000447-Yao1"></a>Yao D, Taguchi T, Matsumura T, Pestell R, Edelstein D, et al.  (2006) Methylglyoxal modification of mSin3A links glycolysis to angiopoietin-2 transcription. Cell  124: 275–286.  <ul class="find" data-citedArticleID="951920" data-doi="10.1016/j.cell.2005.11.024"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.11.024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylglyoxal+modification+of+mSin3A+links+glycolysis+to+angiopoietin-2+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylglyoxal+modification+of+mSin3A+links+glycolysis+to+angiopoietin-2+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000447-Danial1" id="pone.0000447-Danial1"></a>Danial NN, Gramm CF, Scorrano L, Zhang CY, Krauss S, et al.  (2003) BAD and glucokinase reside in a mitochondrial complex that integrates glycolysis and apoptosis. Nature  424: 952–956.  <ul class="find" data-citedArticleID="951764" data-doi="10.1038/nature01825"><li><a href="http://dx.doi.org/10.1038/nature01825" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BAD+and+glucokinase+reside+in+a+mitochondrial+complex+that+integrates+glycolysis+and+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BAD+and+glucokinase+reside+in+a+mitochondrial+complex+that+integrates+glycolysis+and+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000447-Sassanfar1" id="pone.0000447-Sassanfar1"></a>Sassanfar M, Roberts JW (1990) Nature of the SOS-inducing signal in Escherichia coli. The involvement of DNA replication. J Mol Biol  212: 79–96.  <ul class="find" data-citedArticleID="951884" data-doi="10.1016/0022-2836(90)90306-7"><li><a href="http://dx.doi.org/10.1016/0022-2836(90)90306-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nature+of+the+SOS-inducing+signal+in+Escherichia+coli.+The+involvement+of+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nature+of+the+SOS-inducing+signal+in+Escherichia+coli.+The+involvement+of+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000447-Thirumalai1" id="pone.0000447-Thirumalai1"></a>Thirumalai D, Lorimer GH (2001) Chaperonin-mediated protein folding. Annu Rev Biophys Biomol Struct  30: 245–269.  <ul class="find" data-citedArticleID="951900" data-doi="10.1146/annurev.biophys.30.1.245"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.30.1.245" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chaperonin-mediated+protein+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chaperonin-mediated+protein+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000447-Singer1" id="pone.0000447-Singer1"></a>Singer MA, Lindquist S (1998) Multiple effects of trehalose on protein folding in vitro and in vivo. Mol Cell  1: 639–648.  <ul class="find" data-citedArticleID="951892" data-doi="10.1016/s1097-2765(00)80064-7"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80064-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+effects+of+trehalose+on+protein+folding+in+vitro+and+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+effects+of+trehalose+on+protein+folding+in+vitro+and+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000447-Erbse1" id="pone.0000447-Erbse1"></a>Erbse A, Mayer MP, Bukau B (2004) Mechanism of substrate recognition by Hsp70 chaperones. Biochem Soc Trans  32: 617–621.  <ul class="find" data-citedArticleID="951772" data-doi="10.1042/bst0320617"><li><a href="http://dx.doi.org/10.1042/bst0320617" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+substrate+recognition+by+Hsp70+chaperones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+substrate+recognition+by+Hsp70+chaperones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000447-Hardy1" id="pone.0000447-Hardy1"></a>Hardy CD, Cozzarelli NR (2005) A genetic selection for supercoiling mutants of Escherichia coli reveals proteins implicated in chromosome structure. Mol Microbiol  57: 1636–1652.  <ul class="find" data-citedArticleID="951798" data-doi="10.1111/j.1365-2958.2005.04799.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04799.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic+selection+for+supercoiling+mutants+of+Escherichia+coli+reveals+proteins+implicated+in+chromosome+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic+selection+for+supercoiling+mutants+of+Escherichia+coli+reveals+proteins+implicated+in+chromosome+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000447-Chen2" id="pone.0000447-Chen2"></a>Chen XJ, Butow RA (2005) The organization and inheritance of the mitochondrial genome. Nat Rev Genet  6: 815–825.  <ul class="find" data-citedArticleID="951760" data-doi="10.1038/nrg1708"><li><a href="http://dx.doi.org/10.1038/nrg1708" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+organization+and+inheritance+of+the+mitochondrial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+organization+and+inheritance+of+the+mitochondrial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000447-Foti1" id="pone.0000447-Foti1"></a>Foti JJ, Schienda J, Sutera VA Jr, Lovett ST (2005) A bacterial G protein-mediated response to replication arrest. Mol Cell  17: 549–560.  <ul class="find" data-citedArticleID="951784" data-doi="10.1016/j.molcel.2005.01.012"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.01.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+bacterial+G+protein-mediated+response+to+replication+arrest." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+bacterial+G+protein-mediated+response+to+replication+arrest.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000447-Michel1" id="pone.0000447-Michel1"></a>Michel B, Grompone G, Flores MJ, Bidnenko V (2004) Multiple pathways process stalled replication forks. Proc Natl Acad Sci U S A  101: 12783–12788.  <ul class="find" data-citedArticleID="951846" data-doi="10.1073/pnas.0401586101"><li><a href="http://dx.doi.org/10.1073/pnas.0401586101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+pathways+process+stalled+replication+forks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+pathways+process+stalled+replication+forks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000447-Peter1" id="pone.0000447-Peter1"></a>Peter BJ, Arsuaga J, Breier AM, Khodursky AB, Brown PO, et al.  (2004) Genomic transcriptional response to loss of chromosomal supercoiling in Escherichia coli. Genome Biol  5: R87.  <ul class="find" data-citedArticleID="951864" data-doi="10.1186/gb-2004-5-11-r87"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-11-r87" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+transcriptional+response+to+loss+of+chromosomal+supercoiling+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+transcriptional+response+to+loss+of+chromosomal+supercoiling+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000447-Gon1" id="pone.0000447-Gon1"></a>Gon S, Camara JE, Klungsoyr HK, Crooke E, Skarstad K, et al.  (2006) A novel regulatory mechanism couples deoxyribonucleotide synthesis and DNA replication in Escherichia coli. Embo J  25: 1137–1147.  <ul class="find" data-citedArticleID="951794" data-doi="10.1038/sj.emboj.7600990"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600990" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+regulatory+mechanism+couples+deoxyribonucleotide+synthesis+and+DNA+replication+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+regulatory+mechanism+couples+deoxyribonucleotide+synthesis+and+DNA+replication+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000447-Ortenberg1" id="pone.0000447-Ortenberg1"></a>Ortenberg R, Gon S, Porat A, Beckwith J (2004) Interactions of glutaredoxins, ribonucleotide reductase, and components of the DNA replication system of Escherichia coli. Proc Natl Acad Sci U S A  101: 7439–7444.  <ul class="find" data-citedArticleID="951860" data-doi="10.1073/pnas.0401965101"><li><a href="http://dx.doi.org/10.1073/pnas.0401965101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactions+of+glutaredoxins%2C+ribonucleotide+reductase%2C+and+components+of+the+DNA+replication+system+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactions+of+glutaredoxins%2C+ribonucleotide+reductase%2C+and+components+of+the+DNA+replication+system+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000447-Norris1" id="pone.0000447-Norris1"></a>Norris V, den Blaauwen T, Cabin-Flaman A, Doi RH, Harshey R, et al.  (2007) Functional taxonomy of bacterial hyperstructures. Microbiol Mol Biol Rev  71: 230–253.  <ul class="find" data-citedArticleID="951858" data-doi="10.1128/mmbr.00035-06"><li><a href="http://dx.doi.org/10.1128/mmbr.00035-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+taxonomy+of+bacterial+hyperstructures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+taxonomy+of+bacterial+hyperstructures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000447-Molina1" id="pone.0000447-Molina1"></a>Molina F, Skarstad K (2004) Replication fork and SeqA focus distributions in Escherichia coli suggest a replication hyperstructure dependent on nucleotide metabolism. Mol Microbiol  52: 1597–1612.  <ul class="find" data-citedArticleID="951852" data-doi="10.1111/j.1365-2958.2004.04097.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04097.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+fork+and+SeqA+focus+distributions+in+Escherichia+coli+suggest+a+replication+hyperstructure+dependent+on+nucleotide+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+fork+and+SeqA+focus+distributions+in+Escherichia+coli+suggest+a+replication+hyperstructure+dependent+on+nucleotide+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000447-Guzman1" id="pone.0000447-Guzman1"></a>Guzman EC, Caballero JL, Jimenez-Sanchez A (2002) Ribonucleoside diphosphate reductase is a component of the replication hyperstructure in Escherichia coli. Mol Microbiol  43: 487–495.  <ul class="find" data-citedArticleID="951796" data-doi="10.1046/j.1365-2958.2002.02761.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02761.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribonucleoside+diphosphate+reductase+is+a+component+of+the+replication+hyperstructure+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribonucleoside+diphosphate+reductase+is+a+component+of+the+replication+hyperstructure+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000447-Skovgaard1" id="pone.0000447-Skovgaard1"></a>Skovgaard O, Lobner-Olesen A (2005) Reduced initiation frequency from oriC restores viability of a temperature-sensitive Escherichia coli replisome mutant. Microbiology  151: 963–973.  <ul class="find" data-citedArticleID="951896" data-doi="10.1099/mic.0.27630-0"><li><a href="http://dx.doi.org/10.1099/mic.0.27630-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+initiation+frequency+from+oriC+restores+viability+of+a+temperature-sensitive+Escherichia+coli+replisome+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+initiation+frequency+from+oriC+restores+viability+of+a+temperature-sensitive+Escherichia+coli+replisome+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000447-Walker1" id="pone.0000447-Walker1"></a>Walker JR, Ramsey JA, Haldenwang WG (1982) Interaction of the Escherichia coli dnaA initiation protein with the dnaZ polymerization protein in vivo. Proc Natl Acad Sci U S A  79: 3340–3344.  <ul class="find" data-citedArticleID="951912" data-doi="10.1073/pnas.79.10.3340"><li><a href="http://dx.doi.org/10.1073/pnas.79.10.3340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+the+Escherichia+coli+dnaA+initiation+protein+with+the+dnaZ+polymerization+protein+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+the+Escherichia+coli+dnaA+initiation+protein+with+the+dnaZ+polymerization+protein+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000447-Blinkova1" id="pone.0000447-Blinkova1"></a>Blinkova A, Hermandson MJ, Walker JR (2003) Suppression of temperature-sensitive chromosome replication of an Escherichia coli dnaX(Ts) mutant by reduction of initiation efficiency. J Bacteriol  185: 3583–3595.  <ul class="find" data-citedArticleID="951744" data-doi="10.1128/jb.185.12.3583-3595.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.12.3583-3595.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+temperature-sensitive+chromosome+replication+of+an+Escherichia+coli+dnaX%28Ts%29+mutant+by+reduction+of+initiation+efficiency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+temperature-sensitive+chromosome+replication+of+an+Escherichia+coli+dnaX%28Ts%29+mutant+by+reduction+of+initiation+efficiency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000447-Walker2" id="pone.0000447-Walker2"></a>Walker JR, Severson KA, Hermandson MJ, Blinkova A, Carr KM, et al.  (2006) Escherichia coli DnaA protein: specific biochemical defects of mutant DnaAs reduce initiation frequency to suppress a temperature-sensitive dnaX mutation. Biochimie  88: 1–10.  <ul class="find" data-citedArticleID="951914" data-doi="10.1016/j.biochi.2005.08.009"><li><a href="http://dx.doi.org/10.1016/j.biochi.2005.08.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+DnaA+protein%3A+specific+biochemical+defects+of+mutant+DnaAs+reduce+initiation+frequency+to+suppress+a+temperature-sensitive+dnaX+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+DnaA+protein%3A+specific+biochemical+defects+of+mutant+DnaAs+reduce+initiation+frequency+to+suppress+a+temperature-sensitive+dnaX+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000447-Sutera1" id="pone.0000447-Sutera1"></a>Sutera VA Jr, Lovett ST (2006) The role of replication initiation control in promoting survival of replication fork damage. Mol Microbiol  60: 229–239.  <ul class="find" data-citedArticleID="951898" data-doi="10.1111/j.1365-2958.2006.05093.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05093.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+replication+initiation+control+in+promoting+survival+of+replication+fork+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+replication+initiation+control+in+promoting+survival+of+replication+fork+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000447-Westers1" id="pone.0000447-Westers1"></a>Westers H, Dorenbos R, van Dijl JM, Kabel J, Flanagan T, et al.  (2003) Genome engineering reveals large dispensable regions in Bacillus subtilis. Mol Biol Evol  20: 2076–2090.  <ul class="find" data-citedArticleID="951918" data-doi="10.1093/molbev/msg219"><li><a href="http://dx.doi.org/10.1093/molbev/msg219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+engineering+reveals+large+dispensable+regions+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+engineering+reveals+large+dispensable+regions+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000447-Kunst1" id="pone.0000447-Kunst1"></a>Kunst F, Ogasawara N, Moszer I, Albertini AM, Alloni G, et al.  (1997) The complete genome sequence of the gram-positive bacterium Bacillus subtilis. Nature  390: 249–256.  <ul class="find" data-citedArticleID="951824"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+complete+genome+sequence+of+the+gram-positive+bacterium+Bacillus+subtilis.&amp;auth=&amp;atitle=The+complete+genome+sequence+of+the+gram-positive+bacterium+Bacillus+subtilis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+genome+sequence+of+the+gram-positive+bacterium+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+genome+sequence+of+the+gram-positive+bacterium+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000447-Moreno1" id="pone.0000447-Moreno1"></a>Moreno MS, Schneider BL, Maile RR, Weyler W, Saier MH Jr (2001) Catabolite repression mediated by the CcpA protein in Bacillus subtilis: novel modes of regulation revealed by whole-genome analyses. Mol Microbiol  39: 1366–1381.  <ul class="find" data-citedArticleID="951854" data-doi="10.1046/j.1365-2958.2001.02328.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02328.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catabolite+repression+mediated+by+the+CcpA+protein+in+Bacillus+subtilis%3A+novel+modes+of+regulation+revealed+by+whole-genome+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catabolite+repression+mediated+by+the+CcpA+protein+in+Bacillus+subtilis%3A+novel+modes+of+regulation+revealed+by+whole-genome+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000447-Dauner1" id="pone.0000447-Dauner1"></a>Dauner M, Storni T, Sauer U (2001) Bacillus subtilis metabolism and energetics in carbon-limited and excess-carbon chemostat culture. J Bacteriol  183: 7308–7317.  <ul class="find" data-citedArticleID="951766" data-doi="10.1128/jb.183.24.7308-7317.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.24.7308-7317.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+subtilis+metabolism+and+energetics+in+carbon-limited+and+excess-carbon+chemostat+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+subtilis+metabolism+and+energetics+in+carbon-limited+and+excess-carbon+chemostat+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000447-Petersen1" id="pone.0000447-Petersen1"></a>Petersen C, Moller LB (2000) Invariance of the nucleoside triphosphate pools of Escherichia coli with growth rate. J Biol Chem  275: 3931–3935.  <ul class="find" data-citedArticleID="951866" data-doi="10.1074/jbc.275.6.3931"><li><a href="http://dx.doi.org/10.1074/jbc.275.6.3931" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Invariance+of+the+nucleoside+triphosphate+pools+of+Escherichia+coli+with+growth+rate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Invariance+of+the+nucleoside+triphosphate+pools+of+Escherichia+coli+with+growth+rate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000447-Schneider1" id="pone.0000447-Schneider1"></a>Schneider DA, Gourse RL (2004) Relationship between growth rate and ATP concentration in Escherichia coli: a bioassay for available cellular ATP. J Biol Chem  279: 8262–8268.  <ul class="find" data-citedArticleID="951886" data-doi="10.1074/jbc.m311996200"><li><a href="http://dx.doi.org/10.1074/jbc.m311996200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+growth+rate+and+ATP+concentration+in+Escherichia+coli%3A+a+bioassay+for+available+cellular+ATP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+growth+rate+and+ATP+concentration+in+Escherichia+coli%3A+a+bioassay+for+available+cellular+ATP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000447-Kelly1" id="pone.0000447-Kelly1"></a>Kelly TJ, Brown GW (2000) Regulation of chromosome replication. Annu Rev Biochem  69: 829–880.  <ul class="find" data-citedArticleID="951816" data-doi="10.1146/annurev.biochem.69.1.829"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.69.1.829" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+chromosome+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+chromosome+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000447-Forsburg1" id="pone.0000447-Forsburg1"></a>Forsburg SL (2004) Eukaryotic MCM proteins: beyond replication initiation. Microbiol Mol Biol Rev  68: 109–131.  <ul class="find" data-citedArticleID="951780" data-doi="10.1128/mmbr.68.1.109-131.2004"><li><a href="http://dx.doi.org/10.1128/mmbr.68.1.109-131.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eukaryotic+MCM+proteins%3A+beyond+replication+initiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eukaryotic+MCM+proteins%3A+beyond+replication+initiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000447-Giraldo1" id="pone.0000447-Giraldo1"></a>Giraldo R (2003) Common domains in the initiators of DNA replication in Bacteria, Archaea and Eukarya: combined structural, functional and phylogenetic perspectives. FEMS Microbiol Rev  26: 533–554.  <ul class="find" data-citedArticleID="951792" data-doi="10.1016/s0168-6445(02)00145-6"><li><a href="http://dx.doi.org/10.1016/s0168-6445(02)00145-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+domains+in+the+initiators+of+DNA+replication+in+Bacteria%2C+Archaea+and+Eukarya%3A+combined+structural%2C+functional+and+phylogenetic+perspectives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+domains+in+the+initiators+of+DNA+replication+in+Bacteria%2C+Archaea+and+Eukarya%3A+combined+structural%2C+functional+and+phylogenetic+perspectives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000447-Mijakovic1" id="pone.0000447-Mijakovic1"></a>Mijakovic I, Petranovic D, Macek B, Cepo T, Mann M, et al.  (2006) Bacterial single-stranded DNA-binding proteins are phosphorylated on tyrosine. Nucleic Acids Res  34: 1588–1596.  <ul class="find" data-citedArticleID="951850" data-doi="10.1093/nar/gkj514"><li><a href="http://dx.doi.org/10.1093/nar/gkj514" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+single-stranded+DNA-binding+proteins+are+phosphorylated+on+tyrosine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+single-stranded+DNA-binding+proteins+are+phosphorylated+on+tyrosine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000447-Mathews1" id="pone.0000447-Mathews1"></a>Mathews CK (1993) Enzyme organization in DNA precursor biosynthesis. Prog Nucleic Acid Res Mol Biol  44: 167–203.  <ul class="find" data-citedArticleID="951842" data-doi="10.1016/s0079-6603(08)60220-2"><li><a href="http://dx.doi.org/10.1016/s0079-6603(08)60220-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzyme+organization+in+DNA+precursor+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzyme+organization+in+DNA+precursor+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000447-Cho1" id="pone.0000447-Cho1"></a>Cho YH, Yoo SD, Sheen J (2006) Regulatory functions of nuclear hexokinase1 complex in glucose signaling. Cell  127: 579–589.  <ul class="find" data-citedArticleID="951762" data-doi="10.1016/j.cell.2006.09.028"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.09.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+functions+of+nuclear+hexokinase1+complex+in+glucose+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+functions+of+nuclear+hexokinase1+complex+in+glucose+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000447-Loeb1" id="pone.0000447-Loeb1"></a>Loeb LA, Loeb KR, Anderson JP (2003) Multiple mutations and cancer. Proc Natl Acad Sci U S A  100: 776–781.  <ul class="find" data-citedArticleID="951834" data-doi="10.1073/pnas.0334858100"><li><a href="http://dx.doi.org/10.1073/pnas.0334858100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+mutations+and+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+mutations+and+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000447-Gatenby1" id="pone.0000447-Gatenby1"></a>Gatenby RA, Gillies RJ (2004) Why do cancers have high aerobic glycolysis? Nat Rev Cancer  4: 891–899.  <ul class="find" data-citedArticleID="951786" data-doi="10.1038/nrc1478"><li><a href="http://dx.doi.org/10.1038/nrc1478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+do+cancers+have+high+aerobic+glycolysis%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+do+cancers+have+high+aerobic+glycolysis%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000447-Schumann1" id="pone.0000447-Schumann1"></a> Schumann W, Ehrlich SD, Ogasawara N, editors.  (2000) Functional analysis of bacterial genes: a pratical manual. Chichester: John Wiley &amp; Sons, LTD.   <ul class="find-nolinks"></ul></li><li><span class="label">84.
              </span><a name="pone.0000447-Vagner1" id="pone.0000447-Vagner1"></a>Vagner V, Dervyn E, Ehrlich SD (1998) A vector for systematic gene inactivation in Bacillus subtilis. Microbiology  144: 3097–3104.  <ul class="find" data-citedArticleID="951908" data-doi="10.1099/00221287-144-11-3097"><li><a href="http://dx.doi.org/10.1099/00221287-144-11-3097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+vector+for+systematic+gene+inactivation+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+vector+for+systematic+gene+inactivation+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000447-Hartl1" id="pone.0000447-Hartl1"></a>Hartl B, Wehrl W, Wiegert T, Homuth G, Schumann W (2001) Development of a new integration site within the Bacillus subtilis chromosome and construction of compatible expression cassettes. J Bacteriol  183: 2696–2699.  <ul class="find" data-citedArticleID="951800" data-doi="10.1128/jb.183.8.2696-2699.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.8.2696-2699.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+new+integration+site+within+the+Bacillus+subtilis+chromosome+and+construction+of+compatible+expression+cassettes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+new+integration+site+within+the+Bacillus+subtilis+chromosome+and+construction+of+compatible+expression+cassettes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0000447-Harwood1" id="pone.0000447-Harwood1"></a> Harwood CR, Cutting SM, editors.  (1990) Molecular biological methods for Bacillus. Chichester: John Wiley &amp; Sons, LTD.   <ul class="find-nolinks"></ul></li><li><span class="label">87.
              </span><a name="pone.0000447-Doan1" id="pone.0000447-Doan1"></a>Doan T, Aymerich S (2003) Regulation of the central glycolytic genes in Bacillus subtilis: binding of the repressor CggR to its single DNA target sequence is modulated by fructose-1,6-bisphosphate. Mol Microbiol  47: 1709–1721.  <ul class="find" data-citedArticleID="951770" data-doi="10.1046/j.1365-2958.2003.03404.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03404.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+the+central+glycolytic+genes+in+Bacillus+subtilis%3A+binding+of+the+repressor+CggR+to+its+single+DNA+target+sequence+is+modulated+by+fructose-1%2C6-bisphosphate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+the+central+glycolytic+genes+in+Bacillus+subtilis%3A+binding+of+the+repressor+CggR+to+its+single+DNA+target+sequence+is+modulated+by+fructose-1%2C6-bisphosphate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0000447-Zhang1" id="pone.0000447-Zhang1"></a>Zhang S, Haldenwang WG (2005) Contributions of ATP, GTP, and redox state to nutritional stress activation of the Bacillus subtilis sigmaB transcription factor. J Bacteriol  187: 7554–6750.  <ul class="find" data-citedArticleID="951924" data-doi="10.1128/jb.187.22.7554-7560.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.22.7554-7560.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contributions+of+ATP%2C+GTP%2C+and+redox+state+to+nutritional+stress+activation+of+the+Bacillus+subtilis+sigmaB+transcription+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contributions+of+ATP%2C+GTP%2C+and+redox+state+to+nutritional+stress+activation+of+the+Bacillus+subtilis+sigmaB+transcription+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0000447-Eymann1" id="pone.0000447-Eymann1"></a>Eymann C, Homuth G, Scharf C, Hecker M (2002) Bacillus subtilis functional genomics: global characterization of the stringent response by proteome and transcriptome analysis. J Bacteriol  184: 2500–2520.  <ul class="find" data-citedArticleID="951774" data-doi="10.1128/jb.184.9.2500-2520.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.9.2500-2520.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+subtilis+functional+genomics%3A+global+characterization+of+the+stringent+response+by+proteome+and+transcriptome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+subtilis+functional+genomics%3A+global+characterization+of+the+stringent+response+by+proteome+and+transcriptome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0000447-Boylan1" id="pone.0000447-Boylan1"></a>Boylan SA, Redfield AR, Brody MS, Price CW (1993) Stress-induced activation of the sigma B transcription factor of Bacillus subtilis. J Bacteriol  175: 7931–7937.  <ul class="find" data-citedArticleID="951746"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Stress-induced+activation+of+the+sigma+B+transcription+factor+of+Bacillus+subtilis.&amp;auth=&amp;atitle=Stress-induced+activation+of+the+sigma+B+transcription+factor+of+Bacillus+subtilis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stress-induced+activation+of+the+sigma+B+transcription+factor+of+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stress-induced+activation+of+the+sigma+B+transcription+factor+of+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0000447-Sambrook1" id="pone.0000447-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular Cloning: A Laboratory Manual. New York: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">92.
              </span><a name="pone.0000447-Petit1" id="pone.0000447-Petit1"></a>Petit MA, Bruand C, Janniere L, Ehrlich SD (1990) Tn10-derived transposons active in Bacillus subtilis. J Bacteriol  172: 6736–6740.  <ul class="find" data-citedArticleID="951868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tn10-derived+transposons+active+in+Bacillus+subtilis.&amp;auth=&amp;atitle=Tn10-derived+transposons+active+in+Bacillus+subtilis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tn10-derived+transposons+active+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tn10-derived+transposons+active+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0000447-LeChatelier1" id="pone.0000447-LeChatelier1"></a>Le Chatelier E, Becherel OJ, D'Alencon E, CanceilI D, Ehrlich SD, et al.  (2004) Involvement of DnaE, the second replicative DNA polymerase from Bacillus subtilis, in DNA mutagenesis. J Biol Chem  279: 1757–1767.  <ul class="find" data-citedArticleID="951830" data-doi="10.1074/jbc.m310719200"><li><a href="http://dx.doi.org/10.1074/jbc.m310719200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Involvement+of+DnaE%2C+the+second+replicative+DNA+polymerase+from+Bacillus+subtilis%2C+in+DNA+mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Involvement+of+DnaE%2C+the+second+replicative+DNA+polymerase+from+Bacillus+subtilis%2C+in+DNA+mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0000447-Bcher1" id="pone.0000447-Bcher1"></a>Bücher T, Pfleiderer G (1955)  Methods Enzymol  1: 435.  <ul class="find" data-citedArticleID="951740" data-doi="10.1016/0076-6879(55)01071-9"><li><a href="http://dx.doi.org/10.1016/0076-6879(55)01071-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methods+Enzymol" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methods+Enzymol%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0000447-Tuominen1" id="pone.0000447-Tuominen1"></a>Tuominen FW, Bernlohr RW (1975) Pyruvate kinase of Bacillus licheniformis. Methods Enzymol  42: 157–166.  <ul class="find" data-citedArticleID="951906" data-doi="10.1016/0076-6879(75)42110-3"><li><a href="http://dx.doi.org/10.1016/0076-6879(75)42110-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pyruvate+kinase+of+Bacillus+licheniformis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pyruvate+kinase+of+Bacillus+licheniformis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0000447-Petit2" id="pone.0000447-Petit2"></a>Petit MA, Dervyn E, Rose M, Entian KD, McGovern S, et al.  (1998) PcrA is an essential DNA helicase of Bacillus subtilis fulfilling functions both in repair and rolling-circle replication. Mol Microbiol  29: 261–273.  <ul class="find" data-citedArticleID="951870" data-doi="10.1046/j.1365-2958.1998.00927.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00927.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PcrA+is+an+essential+DNA+helicase+of+Bacillus+subtilis+fulfilling+functions+both+in+repair+and+rolling-circle+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PcrA+is+an+essential+DNA+helicase+of+Bacillus+subtilis+fulfilling+functions+both+in+repair+and+rolling-circle+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.XML" value="181884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.PDF" value="808172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g001.PNG_L" value="4278257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g001.PNG_M" value="211750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g001.PNG_S" value="5272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g001.TIF" value="7208054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g001.PNG_I" value="81663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t001.PNG_L" value="90068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t001.PNG_M" value="67917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t001.PNG_S" value="4266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t001.TIF" value="520976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t001.PNG_I" value="39381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t002.PNG_L" value="100410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t002.PNG_M" value="104603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t002.PNG_S" value="5203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t002.TIF" value="474928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t002.PNG_I" value="46084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g002.PNG_L" value="449841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g002.PNG_M" value="74233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g002.PNG_S" value="3237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g002.TIF" value="978056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g002.PNG_I" value="39876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g003.PNG_L" value="1355045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g003.PNG_M" value="167952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g003.PNG_S" value="8710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g003.TIF" value="4930024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g003.PNG_I" value="103041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g004.PNG_L" value="109762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g004.PNG_M" value="20813"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g004.PNG_S" value="2202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g004.TIF" value="224176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g004.PNG_I" value="10108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g005.PNG_L" value="523628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g005.PNG_M" value="46003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g005.PNG_S" value="2937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g005.TIF" value="1052896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g005.PNG_I" value="27040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t003.PNG_L" value="73930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t003.PNG_M" value="77108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t003.PNG_S" value="3053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t003.TIF" value="343354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t003.PNG_I" value="34171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g006.PNG_L" value="886841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g006.PNG_M" value="69057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g006.PNG_S" value="3025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g006.TIF" value="1166938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g006.PNG_I" value="25180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t004.PNG_L" value="83380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t004.PNG_M" value="87621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t004.PNG_S" value="3112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t004.TIF" value="376006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.t004.PNG_I" value="38777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g007.PNG_L" value="558475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g007.PNG_M" value="63936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g007.PNG_S" value="2598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g007.TIF" value="860752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.g007.PNG_I" value="27342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.s001.XLS" value="48640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.s002.XLS" value="32768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000447.s003.XLS" value="54272"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000447&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000447" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000447&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000447&volume=&issue=&title=Genetic Evidence for a Link Between Glycolysis and DNA Replication&author_name=Laurent%20Janni%C3%A8re%2C%20Danielle%20Canceill%2C%20Catherine%20Suski%2C%20Sophie%20Kanga%2C%20B%C3%A9reng%C3%A8re%20Dalmais%2C%20Roxane%20Lestini%2C%20Anne-Fran%C3%A7oise%20Monnier%2C%20J%C3%A9r%C3%B4me%20Chapuis%2C%20Alexander%20Bolotin%2C%20Marina%20Titok%2C%20Emmanuelle%20Le%20Chatelier%2C%20S.%20Dusko%20Ehrlich&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000447" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000447" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000447" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000447&amp;t=Genetic%20Evidence%20for%20a%20Link%20Between%20Glycolysis%20and%20DNA%20Replication" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000447&title=Genetic%20Evidence%20for%20a%20Link%20Between%20Glycolysis%20and%20DNA%20Replication&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000447&amp;title=Genetic%20Evidence%20for%20a%20Link%20Between%20Glycolysis%20and%20DNA%20Replication" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000447" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000447&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genetic Evidence for a Link Between Glycolysis and DNA Replication';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000447';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genetic%20Evidence%20for%20a%20Link%20Between%20Glycolysis%20and%20DNA%20Replication http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000447" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000447" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacillus+subtilis%22" title="Search for articles in the subject area:'Bacillus subtilis'"><div class="flagText">Bacillus subtilis</div></a>
              <div data-categoryid="32939" data-articleid="23732"
                   data-categoryname="Bacillus subtilis"
                   class="flagImage" title="Flag 'Bacillus subtilis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+metabolism%22" title="Search for articles in the subject area:'Cell metabolism'"><div class="flagText">Cell metabolism</div></a>
              <div data-categoryid="18083" data-articleid="23732"
                   data-categoryname="Cell metabolism"
                   class="flagImage" title="Flag 'Cell metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+metabolism%22" title="Search for articles in the subject area:'DNA metabolism'"><div class="flagText">DNA metabolism</div></a>
              <div data-categoryid="35473" data-articleid="23732"
                   data-categoryname="DNA metabolism"
                   class="flagImage" title="Flag 'DNA metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+replication%22" title="Search for articles in the subject area:'DNA replication'"><div class="flagText">DNA replication</div></a>
              <div data-categoryid="19773" data-articleid="23732"
                   data-categoryname="DNA replication"
                   class="flagImage" title="Flag 'DNA replication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+synthesis%22" title="Search for articles in the subject area:'DNA synthesis'"><div class="flagText">DNA synthesis</div></a>
              <div data-categoryid="40873" data-articleid="23732"
                   data-categoryname="DNA synthesis"
                   class="flagImage" title="Flag 'DNA synthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+metabolism%22" title="Search for articles in the subject area:'Enzyme metabolism'"><div class="flagText">Enzyme metabolism</div></a>
              <div data-categoryid="20267" data-articleid="23732"
                   data-categoryname="Enzyme metabolism"
                   class="flagImage" title="Flag 'Enzyme metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glycolysis%22" title="Search for articles in the subject area:'Glycolysis'"><div class="flagText">Glycolysis</div></a>
              <div data-categoryid="44115" data-articleid="23732"
                   data-categoryname="Glycolysis"
                   class="flagImage" title="Flag 'Glycolysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23732"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7411'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3090'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=612&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9527">Related findings in yeast</a><br>Posted by kupiec</p>
      <p><a href="/annotation/listThread.action?root=9167">Corrections in the Author Contributions section</a><br>Posted by janniere</p>
      <p><a href="/annotation/listThread.action?root=9089">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
