

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001036"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001036" />

    <meta name="citation_title" content="A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs"/>
    <meta itemprop="name" content="A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs"/>

      <meta name="citation_author" content="Jeremy L. Peirce"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
      <meta name="citation_author" content="Karl W. Broman"/>
            <meta name="citation_author_institution" content="Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland"/>
      <meta name="citation_author" content="Lu Lu"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
      <meta name="citation_author" content="Robert W. Williams"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>

    <meta name="citation_date" content="2007/10/17"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001036.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1036"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.; citation_author=A Darvasi; citation_author=M Soller; citation_journal_title=Genetics; citation_volume=141; citation_number=1; citation_pages=1199-1207; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait locus analysis using recombinant inbred intercrosses (RIX): theoretical and empirical considerations.; citation_author=F Zou; citation_author=JA Gelfond; citation_author=DC Airey; citation_author=L Lu; citation_author=KF Manly; citation_journal_title=Genetics; citation_number=2; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-tagged mice (GTM): two sets of genome-wide congenic strains.; citation_author=OA Iakoubova; citation_author=CL Olsson; citation_author=KM Dains; citation_author=DA Ross; citation_author=A Andalibi; citation_journal_title=Genomics; citation_volume=74; citation_number=3; citation_pages=89-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mapping of quantitative trait loci with knockout/congenic strains.; citation_author=VJ Bolivar; citation_author=MN Cook; citation_author=L Flaherty; citation_journal_title=Genome Res; citation_volume=11; citation_number=4; citation_pages=1549-1552; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Experimental strategies for the genetic dissection of complex traits in animal models.; citation_author=A Darvasi; citation_journal_title=Nat Genet; citation_volume=18; citation_number=5; citation_pages=19-24; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The replicability of QTLs for murine alcohol preference drinking behavior across eight independent studies.; citation_author=JK Belknap; citation_author=AL Atkins; citation_journal_title=Mamm Genome; citation_volume=12; citation_number=6; citation_pages=893-899; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Natural variation in neuron number in mice is linked to a major quantitative trait locus on Chr 11.; citation_author=RW Williams; citation_author=RC Strom; citation_author=D Goldowitz; citation_journal_title=J Neurosci; citation_volume=18; citation_number=7; citation_pages=138-146; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The significance of not finding a gene.; citation_author=MA Province; citation_journal_title=Am J Hum Genet; citation_volume=69; citation_number=8; citation_pages=660-663; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Combined analyses of data from quantitative trait loci mapping studies. Chromosome 4 effects on porcine growth and fatness.; citation_author=GA Walling; citation_author=PM Visscher; citation_author=L Andersson; citation_author=MF Rothschild; citation_author=L Wang; citation_journal_title=Genetics; citation_volume=155; citation_number=9; citation_pages=1369-1378; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multiple cross mapping (MCM) markedly improves the localization of a QTL for ethanol-induced activation.; citation_author=R Hitzemann; citation_author=B Malmanger; citation_author=S Cooper; citation_author=S Coulombe; citation_author=C Reed; citation_journal_title=Genes Brain Behav; citation_volume=1; citation_number=10; citation_pages=214-222; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multiple cross and inbred strain haplotype mapping of complex-trait candidate genes.; citation_author=YG Park; citation_author=R Clifford; citation_author=KH Buetow; citation_author=KW Hunter; citation_journal_title=Genome Res; citation_volume=13; citation_number=11; citation_pages=118-121; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A strategy for the integration of QTL, gene expression, and sequence analyses.; citation_author=R Hitzemann; citation_author=B Malmanger; citation_author=C Reed; citation_author=M Lawler; citation_author=B Hitzemann; citation_journal_title=Mamm Genome; citation_volume=14; citation_number=12; citation_pages=733-747; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Combining data from multiple inbred line crosses improves the power and resolution of quantitative trait loci mapping.; citation_author=R Li; citation_author=MA Lyons; citation_author=H Wittenburg; citation_author=B Paigen; citation_author=GA Churchill; citation_journal_title=Genetics; citation_volume=169; citation_number=13; citation_pages=1699-1709; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome Reshuffling for Advanced Intercross Line Permutation. submitted; citation_author=JL Peirce; citation_author=KW Broman; citation_author=L Lu; citation_author=EJ Chesler; citation_author=G Zhou; citation_number=14; " />
      <meta name="citation_reference" content="citation_title=Inbreeding and linkage.; citation_author=JBS Haldane; citation_author=CH Waddington; citation_journal_title=Genetics; citation_volume=16; citation_number=15; citation_pages=357-374; citation_date=1931; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for research workers.; citation_author=R Fisher; citation_number=16; citation_date=1925; citation_publisher=Oliver and Lloyd; " />
      <meta name="citation_reference" content="citation_title=A statistical framework for quantitative trait mapping.; citation_author=S Sen; citation_author=GA Churchill; citation_journal_title=Genetics; citation_volume=159; citation_number=17; citation_pages=371-387; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A new set of BXD recombinant inbred lines from advanced intercross populations in mice.; citation_author=JL Peirce; citation_author=L Lu; citation_author=J Gu; citation_author=LM Silver; citation_author=RW Williams; citation_journal_title=BMC Genet; citation_volume=5; citation_number=18; citation_pages=7; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complex trait analysis of the hippocampus: mapping and biometric analysis of two novel gene loci with specific effects on hippocampal structure in mice.; citation_author=L Lu; citation_author=DC Airey; citation_author=RW Williams; citation_journal_title=J Neurosci; citation_volume=21; citation_number=19; citation_pages=3503-3514; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.; citation_author=ES Lander; citation_author=D Botstein; citation_journal_title=Genetics; citation_volume=121; citation_number=20; citation_pages=185-199; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood from incomplete data via the EM algorithm.; citation_author=AP Dempster; citation_author=NM Laird; citation_author=DB Rubin; citation_journal_title=J Roy Statist Soc B; citation_volume=39; citation_number=21; citation_pages=1-38; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results.; citation_author=E Lander; citation_author=L Kruglyak; citation_journal_title=Nat Genet; citation_volume=11; citation_number=22; citation_pages=241-247; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=R/qtl: QTL mapping in experimental crosses.; citation_author=KW Broman; citation_author=H Wu; citation_author=S Sen; citation_author=GA Churchill; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=23; citation_pages=889-890; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=WebQTL: rapid exploratory analysis of gene expression and genetic networks for brain and behavior.; citation_author=EJ Chesler; citation_author=L Lu; citation_author=J Wang; citation_author=RW Williams; citation_author=KF Manly; citation_journal_title=Nat Neurosci; citation_volume=7; citation_number=24; citation_pages=485-486; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=WebQTL: web-based complex trait analysis.; citation_author=J Wang; citation_author=RW Williams; citation_author=KF Manly; citation_journal_title=Neuroinformatics; citation_volume=1; citation_number=25; citation_pages=299-308; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recombinant inbred strains.; citation_author=BA Taylor; citation_number=26; citation_pages=773-796; citation_date=1989; citation_publisher=Oxford UP; " />
      <meta name="citation_reference" content="citation_title=Genotyping new BXD recombinant inbred mouse strains and comparison of BXD and consensus maps.; citation_author=BA Taylor; citation_author=C Wnek; citation_author=BS Kotlus; citation_author=N Roemer; citation_author=T MacTaggart; citation_journal_title=Mamm Genome; citation_volume=10; citation_number=27; citation_pages=335-348; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Empirical threshold values for quantitative trait mapping.; citation_author=GA Churchill; citation_author=RW Doerge; citation_journal_title=Genetics; citation_volume=138; citation_number=28; citation_pages=963-971; citation_date=1994; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001036" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs"/>
    <meta name="twitter:description" content="BackgroundOver the past decade many linkage studies have defined chromosomal intervals containing polymorphisms that modulate a variety of traits. Many phenotypes are now associated with enough mapping data that meta-analysis could help refine locations of known QTLs and detect many novel QTLs.Methodology/Principal FindingsWe describe a simple approach to combining QTL mapping results for multiple studies and demonstrate its utility using two hippocampus weight loci. Using data taken from two populations, a recombinant inbred strain set and an advanced intercross population we demonstrate considerable improvements in significance and resolution for both loci. 1-LOD support intervals were improved 51% for Hipp1a and 37% for Hipp9a. We first generate locus-wise permuted P-values for association with the phenotype from multiple maps, which can be done using a permutation method appropriate to each population. These results are then assigned to defined physical positions by interpolation between markers with known physical and genetic positions. We then use Fisher's combination test to combine position-by-position probabilities among experiments. Finally, we calculate genome-wide combined P-values by generating locus-specific P-values for each permuted map for each experiment. These permuted maps are then sampled with replacement and combined. The distribution of best locus-specific P-values for each combined map is the null distribution of genome-wide adjusted P-values.Conclusions/SignificanceOur approach is applicable to a wide variety of segregating and non-segregating mapping populations, facilitates rapid refinement of physical QTL position, is complementary to other QTL fine mapping methods, and provides an appropriate genome-wide criterion of significance for combined mapping results."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001036.g002"/>

  <meta property="og:title" content="A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001036" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5835'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=514'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=904&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001036">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001036" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001036&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeremy L. Peirce
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jpeirce@utmem.edu">jpeirce@utmem.edu</a></p>

                <p>Affiliations:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America, 
                  Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karl W. Broman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lu Lu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America, 
                  Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert W. Williams
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America, 
                  Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 17, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001036</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001036" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001036" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001036" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001036" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001036-g001" data-doi="info:doi/10.1371/journal.pone.0001036" data-uri="info:doi/10.1371/journal.pone.0001036.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001036-g002" data-doi="info:doi/10.1371/journal.pone.0001036" data-uri="info:doi/10.1371/journal.pone.0001036.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001036">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001036" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Over the past decade many linkage studies have defined chromosomal intervals containing polymorphisms that modulate a variety of traits. Many phenotypes are now associated with enough mapping data that meta-analysis could help refine locations of known QTLs and detect many novel QTLs.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We describe a simple approach to combining QTL mapping results for multiple studies and demonstrate its utility using two hippocampus weight loci. Using data taken from two populations, a recombinant inbred strain set and an advanced intercross population we demonstrate considerable improvements in significance and resolution for both loci. 1-LOD support intervals were improved 51% for <em>Hipp1a</em> and 37% for <em>Hipp9a</em>. We first generate locus-wise permuted P-values for association with the phenotype from multiple maps, which can be done using a permutation method appropriate to each population. These results are then assigned to defined physical positions by interpolation between markers with known physical and genetic positions. We then use Fisher's combination test to combine position-by-position probabilities among experiments. Finally, we calculate genome-wide combined P-values by generating locus-specific P-values for each permuted map for each experiment. These permuted maps are then sampled with replacement and combined. The distribution of best locus-specific P-values for each combined map is the null distribution of genome-wide adjusted P-values.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our approach is applicable to a wide variety of segregating and non-segregating mapping populations, facilitates rapid refinement of physical QTL position, is complementary to other QTL fine mapping methods, and provides an appropriate genome-wide criterion of significance for combined mapping results.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Peirce JL, Broman KW, Lu L, Williams RW (2007) A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs. PLoS ONE 2(10):
          e1036.
            doi:10.1371/journal.pone.0001036</p><p><strong>Academic Editor: </strong>Aimee Zaas, Duke University, United States of America</p><p><strong>Received:</strong> February 4, 2007; <strong>Accepted:</strong> August 2, 2007; <strong>Published:</strong> October 17, 2007</p><p><strong>Copyright:</strong> © 2007 Peirce et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Our thanks for financial support from (1) P20-MH62009 from NIMH, NIDA, and NSF. (2) INIA grants U01AA13499 and U24AA135B from NIAAA, and INIA grant U01AA01442501 from NIAAA. The funding agencies had no role in design, conduct, analysis, and interpretation of data or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The majority of traits that show variation based on genetic influences are affected by multiple genes of small effect as well as by environment. When compelling candidates for direct investigation are not available, QTL mapping provides an effective approach to localizing regions of the genome that are likely to mediate genetic variation in the phenotype. Mapping a QTL to an interval of 20–60 Mb has become a relatively routine matter for traits with at least moderate heritability, but narrowing the interval to a small region that includes only a few candidate genes (1–5 Mb at most) is still a highly challenging task.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A variety of methods for accomplishing the task of narrowing a QTL interval have been proposed <a href="#pone.0001036-Darvasi1">[1]</a>–<a href="#pone.0001036-Darvasi2">[5]</a> and each has advantages and disadvantages. Our complementary approach combines the results for a variety of mapping studies, each of which should add a degree of precision and significance to the final result. In this paper we describe our method and provide an implementation, which handles the problem of combining data using different markers as well as converting from genetic to physical maps and regularizing positions to be combined.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Our method calculates locus-specific P-values using the distribution of alleles at each locus. For large populations of genetically independent animals, the negative log of these locus-specific P-values is very similar to the more typically reported LOD scores, but for smaller populations like recombinant inbred strains, and non-independent populations such as advanced intercross lines, and especially for populations with missing data, they are an important innovation. For these populations, converting to locus-specific P-values is more informative and allows us to apply Fisher's combination test to combination of QTL mapping results.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Since each QTL mapping effort originates from an independently generated population, the pattern of alleles inherited in one population is independent of the pattern of alleles inherited in another population. Therefore under the null hypothesis of no linkage, locus-specific P-values for any arbitrary point on the genome are strictly independent between data sets. Given the independence of our data sets, we chose to address the problem of combining multiple testing results by using Fisher's combination test.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Since Fisher's combination test operates on P-values, our approach first defines locus-specific P-values in the original and permuted QTL maps, then combines QTL maps on a point-wise basis using Fisher's combination test. We then convert sample permuted QTL maps from each data set to locus-specific P-values, combine them, and order the best genome-wide combined P-values from these samples to define the null distribution of genome-wide adjusted P-values.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Previous efforts to meta-analyze different mapping populations have included taking the best P-value for each QTL interval and combining these values using Fisher's combination test <a href="#pone.0001036-Belknap1">[6]</a> to assess the combined likelihood of ethanol drinking QTLs between C57BL/6J and DBA/2J. Unfortunately, this approach does not really increase the resolution of the QTL map, though it does improve confidence that a QTL exists in the region. Position-specific combination of QTL maps using Fisher's combination test was, however, used earlier to combine data from two different recombinant inbred (RI) strain sets <a href="#pone.0001036-Williams1">[7]</a> where the same markers were typed or could be inferred, though calculation of genome-wide adjusted significance was not considered.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Province <a href="#pone.0001036-Province1">[8]</a> describes a correction necessary to account for truncated LOD scores in combining QTL data with the clear application of position-specific data combination by simple addition of LOD scores. Other recent investigations have used a variety of approaches to integrating genetic mapping data including standardizing common phenotypes between crosses, <a href="#pone.0001036-Walling1">[9]</a> multiple cross mapping (MCM), a method of identifying haplotype blocks that segregate as expected in an initially defined QTL interval among multiple strains, <a href="#pone.0001036-Hitzemann1">[10]</a> and incorporation of additional data on gene density, identity <a href="#pone.0001036-Park1">[11]</a> and expression <a href="#pone.0001036-Hitzemann2">[12]</a>.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>A more recent paper by Li and colleagues <a href="#pone.0001036-Li1">[13]</a> describes combination of multiple crosses using cross as a covariate, and outlines a permutation method that retains stratification of phenotypes by cross for determining genome-wide significance, which should work well when simple shuffling of genotypes and phenotypes within cross is sufficient to provide appropriate randomization within data set. Our approach provides a general method of integrating data even when this is not the case, for instance with Advanced Intercross Lines (AILs), <a href="#pone.0001036-Peirce1">[14]</a>, recombinant inbred intercrosses (RIX), and heterogeneous stocks (HS). Our approach can also be used for intercrosses (F2), backcrosses (N2), RIs and other datasets and is a simple method of integrating results from studies using individually tailored approaches like composite interval mapping. These methods can be used in a complementary manner by combining appropriate crosses using the method of Li and co-workers <a href="#pone.0001036-Li1">[13]</a> and integrating other data sets using our method.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>Our simple method also has the advantage of not requiring adjustment for use with crosses that represent different numbers of effective tests across the genome. For instance, the number of independent tests represented by an RI strain set will be greater than the number represented by a standard F2, since the genetic length of the RI genome is four times that of the F2 genome. <a href="#pone.0001036-Haldane1">[15]</a> AIL and HS data sets may have even more extreme expansions of the genetic length of their genomes. Since a permuted QTL map is sampled from each permuted data set and combined in the same way as in the original combined data set, the effective number of tests in the permutations will be equivalent to that in the original data, so the genome-wide adjustment will be valid.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Distribution of locus-specific P-values</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Under the null hypothesis of no genetic effect at a locus, the distribution of P-values should be constant with a range of 0 to 1 <a href="#pone.0001036-Fisher1">[16]</a>. In fact, across permutations at a given locus this is enforced in the locus-specific P-value computation by ranking of values at each locus between permutations, which results in an even distribution of values at each position. In the permutations, where <strong>no</strong> locus should be associated with a real genetic effect, this should be the case among loci as well, and the distribution of P-values for each permutation should also be constant and range from 0 to 1. This is the case in a variety of tested permutations (RI, F2, and AIL populations) at tested markers. Interpolation of P-values between markers does seem to slightly reduce the number of P-values observed at the very low and high ends of the distribution, however.</p>


<h4>The value of locus-specific P-values</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In large data sets with independent observations—most F2 populations, for instance—the distribution of P-values under the null hypothesis of no linkage is approximately the same for each analyzed locus in the genome. The equivalent distribution of P-values by locus under the null hypothesis means that any two positions in the genome are equally likely to be the best P-value in the genome in a given permutation of an original data set. This assumption does not always hold in small data sets such as RI populations and populations with non-independent observations such as AILs. (<a href="#pone-0001036-g001">Fig. 1</a>) For consistency, however, we calculate locus-specific P-values for all data considered in this paper.</p>
<div class="figure" id="pone-0001036-g001"><div class="img"><a name="pone-0001036-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001036" data-uri="info:doi/10.1371/journal.pone.0001036.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001036.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001036.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001036.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001036.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001036.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001036.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001036.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001036.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The need for locus-specific P-values.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The 95% LOD score (the LOD score equivalent to a locus-specific P = 0.05) was calculated using 10,000 permutations for markers on Chr. 1 for body weight in several different populations. Each marker is indicated by a dot with connecting lines interpolated between adjacent markers. TJL BXD are BXD strains available from The Jackson Laboratory (The BXD strains developed by Taylor and colleagues <a href="#pone.0001036-Taylor1">[26]</a>, <a href="#pone.0001036-Taylor2">[27]</a>). New BXD are the recently developed BXD strains currently resident at UTHSC. <a href="#pone.0001036-Peirce2">[18]</a> Note that the maximum and minimum values of the 95<sup>th</sup> percentile LOD score vary considerably for the AIL population, somewhat for the RI (New BXD and TJL BXD) populations, (predicted by missing data pattern) and very little for the 183 member F2 population tested. (There are only three widely spaced markers genotyped for the F2 population on Chr. 1, so the interpolation between points should not be interpreted as a meaningful line. However, markers on all chromosomes were very similar, between a 95% LOD of 1.2 and 1.4.)</p>
<span>doi:10.1371/journal.pone.0001036.g001</span></div>

<h4>Locus-specific P-values and missing data</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>fEven when using Sen and Churchill's imputation <a href="#pone.0001036-Sen1">[17]</a> for missing genotypes, the effect of missing data is apparent in the permuted distribution of locus-specific P-values derived from simple reshuffling permutations such as SimplePerm.py, described above. There is a correlation (r = 0.52) between number of fraction of missing genotypes for both the classic C57BL/6J (B6) x DBA/2J (D2) RI (BXD RI) strains from The Jackson Laboratory and the new BXD RI strains <a href="#pone.0001036-Peirce2">[18]</a>. Dropping imputation for missing genotypes results in an even higher (r = 0.73) for correlation with the fraction of missing data. Where genotypes themselves are being permuted and reconstructed, for instance with AIL data, there is no significant correlation between permuted locus-specific LOD score distribution and missing data in the original data set.</p>


<h4>Composite significance and intervals of <em>Hipp1a</em> and <em>Hipp9a</em></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>As an example of the composite mapping method, we mapped bilateral hippocampus weight in two populations, each of which has already been individually reported <a href="#pone.0001036-Peirce1">[14]</a>, <a href="#pone.0001036-Lu1">[19]</a>. <em>Hipp1a</em> was originally reported as significant in the BXD strain set <a href="#pone.0001036-Lu1">[19]</a>. As the re-mapping of hippocampus weight QTLs in <a href="#pone-0001036-g002">Fig. 2</a> shows, we see a significant QTL on Chr. 1, but we also see a non-significant indication of a QTL on Chr. 9. (best locus-wise P = 0.001). This second QTL is individually significant in an AI population <a href="#pone.0001036-Peirce1">[14]</a>. (Note that, for each population the best possible P-value is P&lt;0.0002 since 5000 permutations were done. It may be possible to further improve composite resolution using much larger permutations.) In the case of <em>Hipp1a</em> the 1-LOD support interval (taken here as rough measure of confidence interval) is 34 Mb in the BXD strains, 17.5 Mb for the AIL, and the composite map is slightly better at 16.5 Mb, an average 51% improvement over the individual maps. For <em>Hipp9a</em> the interval was 24.5 Mb for the BXDs, 21.5 Mb for the AILs, and only 14.5 Mb for the composite map–an average improvement of 37% over the individual mapping efforts.</p>
<div class="figure" id="pone-0001036-g002"><div class="img"><a name="pone-0001036-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001036" data-uri="info:doi/10.1371/journal.pone.0001036.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001036.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001036.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001036.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001036.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001036.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001036.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001036.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001036.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Combined mapping for <em>Hipp1a</em> and <em>Hipp9a.</em></span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>This figure shows mapping data for the hippocampus weight loci <em>Hipp1a</em> and <em>Hipp9a</em> using 34 BXD strains (BXD; shaded line) and 679 advanced intercross animals (AIL, thin solid line) as well as the composite map using the described method (thick solid line). The genome-wide adjusted composite P = 0.05 threshold is −log P = 3.5 (dark solid horizontal line). Since 5000 permutations were used for each data set, the maximum −log P&lt;3.7 (graphed as −log P = 3.7 for convenience) for each individual data set, so increasing the number of permutations might increase the peak combined value and slightly improve the range of the combined interval. Bars underneath the peaks are labeled AIL, BXD, and combined to indicate the l-LOD support interval of these mapping populations.</p>
<span>doi:10.1371/journal.pone.0001036.g002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have developed and implemented a simple method for combining QTL mapping results from multiple QTL experiments. The ready availability of dense marker maps and physical positions as well as genetic positions for markers facilitates incorporation of positional information. This method can therefore be used not only to report combined probability of the existence of a QTL locus more accurately than simply combining the best P-values for multiple mapping approaches <a href="#pone.0001036-Belknap1">[6]</a>. It is also a valuable fine mapping technique and can be used in conjunction with almost any high-resolution mapping method <a href="#pone.0001036-Darvasi1">[1]</a>–<a href="#pone.0001036-Darvasi2">[5]</a> as well as for combination of multiple low-resolution mapping efforts.</p>

<h4>Narrowing intervals by combining mapping populations</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We have demonstrated that notable improvements in genome-wide significance and QTL confidence interval are possible using our method by using the example of two hippocampus weight loci. With only two mapping populations for this phenotype, we saw improvements of 37% to 51%, in 1-LOD confidence intervals. The extensible nature of the method means that any number of available mapping populations can be added to further improve the results.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In addition, combining loci that are barely significant or narrowly miss the criteria for significance in multiple populations can clearly lead to strong observations of new loci or improved confidence in previously known loci. For instance, combining mapping results at a locus significant at a point-wise P = 0.05 for seven mapping populations or P = 0.01 for three mapping populations results in a P<sub>combined</sub> = 0.0001, which is very likely significant genome-wide. Results from small mapping populations generated using the same parental strains can therefore profitably be combined using this method. Combining suggestive results (P = 0.001) with small additional results (P = 0.01) also generates much improved composite results. (P<sub>combined</sub> = 0.00012)</p>


<h4>Crosses in multiple strains and phenotypes</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>While we have concentrated on multiple crosses using the same progenitor strains, it would be simple to extend this method to include crosses in other strains or, for that matter, other phenotypes measured in independent crosses. The implicit assumption in these situations is that measurement of an equivalent trait is occurring despite the cross or phenotype difference. In the case of multiple strains, this means the allele difference causing the QTL occurs in the same gene between the strains crossed in each population. Populations such as the BXD, BXA/AXB, CXB, and BXH RI strain sets, when the common allele (B6) may differ from the other strains, could fall in this category. In the case of multiple phenotypes (which could also be in multiple strains) the assumption is that the phenotypes are measuring the effect of the same polymorphism between progenitor strains for the cross.</p>


<h4>The importance of physical position</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Genetic positions are necessary for generating QTL maps using methods such as interval mapping <a href="#pone.0001036-Lander1">[20]</a>, <a href="#pone.0001036-Dempster1">[21]</a>. Ultimately, however, the desired result of a QTL map is assignment of a probability that a given segment of DNA, a physical position, is involved in a phenotype. Additionally, genetic positions vary between crosses, while physical positions are constant—a desirable property when comparing between crosses is the fundamental goal.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Assigning a P-value to a physical position requires interpolation of physical positions for markers (and intermediate markers generated in interval mapping) with unknown physical positions. It also requires interpolation between these markers and the regular, set physical positions that are ultimately reported. For dense marker maps such as those typical of RI strains, the assumption that intermediate physical position can be linearly interpolated from genetic positions and flanking physical positions is reasonable. Some caution may be appropriate, however, when interpolating physical positions for widely spaced markers.</p>


<h4>Incorporating non-locus-specific P-values</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Since locus-specific P-values are not required in the case of sufficiently large RI sets, intercrosses (F2), and backcrosses (N2), the process of explicitly computing QTL maps for each data set and performing the permutation described could be approximated by deriving P-values for each position in some more generic manner, for example from the LOD score, and combining these values as described to generate composite QTL maps and genome-wide thresholds. This is particularly useful when a simple cross with highly significant loci is being considered since the software used to sort P-values (OutParser.py) runs in approximately N<sup>2</sup> time with respect to number of permutations. When converting LODs to P-values, however, under the null hypothesis of no linkage, approximately half the genome will be associated with LOD = 0 (because LODs are constrained to be positive in typical QTL maps) which should equate to P≅0.72 <a href="#pone.0001036-Province1">[8]</a>.</p>


<h4>Approximation of genome-wide P-value thresholds</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>When the distribution of permuted P-values across the genome is uniform, combination of multiple P-values for each locus should result in a uniform combined P-value distribution across the genome. We verified this empirically using a simple simulation, sampling and combining multiple observations from uniform distributions over 1000 permutations. Since in the ideal case each combined permuted QTL map will have a P-value distribution equivalent to a simple permuted QTL map, existing analytical <a href="#pone.0001036-Lander2">[22]</a> methods for estimating genome-wide P-value thresholds may be applicable where genetic lengths of the data sets being combined are similar. In most such cases, the distribution of permuted P-values is fairly uniform, so analytically determined cutoffs should provide reasonably good estimates. When a genome-wide permuted P-value is not required, the final step of combining P-values takes only a few seconds.</p>


<h4>Multiple mapping results from a single population</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>While any number of independent results can be combined using our method, it is problematic to include mapping results from the same reference population even when phenotyping was entirely independent. For instance, directly combining more than one QTL map from the same RI or congenic population is problematic. Instead it might be practical to combine the data gathered on the population as appropriate and generate a single QTL map from the combined data. This map could then be combined with any other independent mapping experiments using our method.</p>


<h4>Incorporation of congenic data</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>This method can be extended to incorporate data from a variety of data sources, including congenic and consomic data. For a polygenic trait, a test of whether a modifying locus from the introgressed parent is present in a given congenic segment addresses only the association of that region and does not contain information about the rest of the genome.</p>
<a id="article1.body1.sec3.sec7.p2" name="article1.body1.sec3.sec7.p2"></a><p>One simple approximate solution is to assume that the LOD = 0 for the remainder of the genome and apply Province's method (2001) to convert to P ≅ 0.72. This is a highly conservative assumption as our actual expectation is that there will be linkage in other parts of the genome as well for a complex trait. Fortunately, Fisher's combination test does not penalize too heavily for failure to observe association. P values for permutations could also be simulated by assigning a P-value, evenly distributed between 0 and 1, to each point outside the congenic interval. In situations where multiple congenics exist, combining the congenic interval results as a single data source might also be a reasonable approach.</p>


<h4>The importance of raw data archives</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>If meta-analysis of QTLs is limited to simple combination of maximum significance values for a QTL interval, the common practice of reporting QTL significance in terms of this maximum and of providing a figure representing the QTL interval is sufficient. For position specific analyses with joint determination of genome-wide significance thresholds, however, it is important to have access to the original data as well as the specific parameters used to perform the data analysis. Unfortunately, particularly in the older QTL literature, public access to the primary data was seldom a requirement of publication. The authors would strongly advocate that journals require public access to such data both to facilitate transparency and reproduction of results and to facilitate more sophisticated meta-analyses. All data used in generating the results for this paper is available at <a href="http://www.nervenet.org/papers/CombineQTL.html">http://www.nervenet.org/papers/CombineQT​L.html</a> or archived at <a href="http://www.genenetwork.org">www.genenetwork.org</a>.</p>


<h4>Summary</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>This method is a simple, easily extensible means of combining QTL mapping results from a wide variety of mapping efforts, and complements other fine mapping methods. The major advantages of this method are its simplicity, incorporation of physical position data and locus-specific P-values, and empirically estimated genome-wide P-value thresholds.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Assigning P-values associated with physical positions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The first step in generating a composite QTL map is to assign P-values to regular physical positions on the genome. This is done using permuted QTL maps processed to determine locus-specific P-values as described below. The details of the permutation required to generate the permuted maps are assumed to have been worked out before data is combined.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>We use locus-specific P-values because in certain data sources such as advanced intercrosses and small RI sets, the correspondence between the likelihood typically reported in mapping output (a LOD score for R/qtl, <a href="#pone.0001036-Broman1">[23]</a> for instance, or a LRS score in the case of WebQTL <a href="#pone.0001036-Chesler1">[24]</a>, <a href="#pone.0001036-Wang1">[25]</a>) and the P-value, as determined by generation of permuted QTL maps, is not constant with respect to position. This can be related to missing data patterns, distribution of alleles and non-syntenic association.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Once all of the permuted maps are generated, we assign locus-specific p-values to regularly spaced physical positions on each chromosome for the original data. In order to derive locus-specific P-values from raw LOD scores we let be the observed LOD at locus j. For each permutation, L<sub>perm</sub>(i,j) is the observed LOD at locus j in permutation replicate i. The locus-specific P-value for locus j is P(j) = proportion{ L<sub>perm</sub>(i,j)∃L<sub>obs</sub>(j)}. This is the proportion of the permutation replicates that have a LOD score greater than or equal to the observed LOD score at locus j.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>P-values at regular intervals were based on the known physical and genetic positions of markers and their P-values as computed above. First we linearly interpolated missing physical positions using flanking genetic positions for the nearest markers with complete information (assuming 0 cM and 0 Mb for the proximal end of the chromosome and assuming linear extension for the distal end of the chromosome using the ratio of physical to genetic distance of the nearest proximal markers.) After we obtained a complete set of physical positions, we linearly interpolated P values at regular intervals on a grid of physical positions, allowing values from the final marker to the end of the chromosome to equal those at the final marker.</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>For sparsely genotyped data sets it is more accurate to linearly interpolate using LOD scores based on an interval map and derive locus-specific P-values from the interpolated LOD scores, though this method is slower than the one described above. As genotype density increases the order of these steps has less effect. We tested the effect of this difference for our BXD RI data at 300 markers and the average percent difference in P-value between methods was only 12%. For all model data sets except our F2 data set, which is sparsely genotyped, we therefore calculated locus-specific P-values first. We provide an implementation for both methods.</p>


<h4>Sample data sets and phenotypes</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For calculating the LOD score equivalent to a locus-specific P = 0.05 (referred to as 95% LOD) in <a href="#pone-0001036-g001">Fig. 1</a>, we used raw body weight measurements from a previously described intercross <a href="#pone.0001036-Lu1">[19]</a>, as well as other previously described data <a href="#pone.0001036-Lu1">[19]</a> from the classic BXD recombinant inbred (RI) strains <a href="#pone.0001036-Taylor1">[26]</a>, <a href="#pone.0001036-Taylor2">[27]</a>, which are formed by repeated intercrossing and inbreeding the progeny of C57BL/6J and DBA/2J. In addition, we gathered new body weight data for a recently described <a href="#pone.0001036-Peirce2">[18]</a> set of BXD RI strains and from an advanced intercross <a href="#pone.0001036-Peirce1">[14]</a> using similar criteria.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For composite mapping of hippocampus weight QTLs shown in <a href="#pone-0001036-g002">Fig. 2</a>, we used previously described observations in the classic BXD RI strains <a href="#pone.0001036-Lu1">[19]</a> and observations made using the same protocol from an advanced intercross, the construction of which is described elsewhere <a href="#pone.0001036-Peirce1">[14]</a>. For this example of combined mapping data, we used the method and software described below.</p>


<h4>Combining QTL data sources</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>At the end of the process outlined above, each data source will be represented by a set of locus-specific P-values associated with regular and defined physical positions. For each of these positions, j, there will be k associated P-values, one from each data source. Since by Fisher's combination test −2 Σ ln P(k,j) for all k data sources at locus j follows a chi-squared distribution with 2k degrees of freedom in the case of the null hypothesis, (no association between phenotype and genotype) we can easily associate each position with a composite locus-wise P-value, P<sub>combined</sub>(j).</p>


<h4>Generating genome-wide P-value thresholds</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The combined map gives us a locus-specific combined P-value for any given locus, P<sub>combined</sub>(j), but it is also important to have a genome-wide threshold for error control at a specified genome-wide P-value. Genome-wide LOD score thresholds for QTL experiments are typically generated by permuting genotype or phenotype, re-running the mapping phase of the experiment for a large number of permutations <a href="#pone.0001036-Churchill1">[28]</a>, and creating an ordered list of the highest LOD score in each permutation. The genome-wide adjusted P-value is the proportion of this ordered list that is greater than or equal to a given LOD score in the original QTL map.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The approach here is similar except that we are first converting LOD scores to P-values for each permutation i. This is done as described above except that observed LOD score at locus j, is the value of the LOD score for a particular permutation in i, designated i<sub>fixed,</sub> for which locus-specific P-values are currently being calculated. We will term the LOD score for i<sub>fixed </sub>at locus j L(i<sub>fixed</sub>,j) and will designate the permuted LOD scores for all permutations except i<sub>fixed</sub> as L<sub>perm</sub>(i',j). This process is an excellent approximation of the locus-specific P-value that would result from generating a set of permutations of each i<sub>fixed</sub>, assuming that L<sub>perm</sub> (i',j) has a distribution equivalent to L<sub>perm</sub>(i,j), which should be true for large i.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>In other words, for each permutation in i, the locus-specific P-value at j is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.e001&amp;representation=PNG" class="inline-graphic"></span><br>After calculating locus specific P-values for all i permutations at all j positions, P(i,j), in all k data sets, we can estimate the genome-wide adjusted P-value. We do this by sampling a permuted map of locus specific P-values, P(i), for each of k data sets and calculating the combined P-value, P<sub>perm, combined</sub>(j), for each corresponding locus between sampled permuted maps as described above using Fisher's combination test.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>We then record the minimum P<sub>perm, ombined</sub>(j) value from each combined map. This permutation is repeated to create an ordered list of best P-values, P<sub>best</sub>. For a given P<sub>combined</sub>(j) value in the original data set, the genome-wide adjusted P-value, P<sub>adjusted</sub>, for the combined data is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001036.e002&amp;representation=PNG" class="inline-graphic"></span><br>Typically, however, the ordered list is simply used to calculate a genome-wide threshold, often set the 95<sup>th</sup> percentile value for P<sub>best</sub> (genome-wide adjusted P = 0.05).</p>


<h4>Software</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>All software described is available at <a href="http://www.nervenet.org/papers/CombineQTL.html">http://www.nervenet.org/papers/CombineQT​L.html</a>. For the customized analyses above, we wrote scripts using Python 2.4 as well as simple R scripts to automate use of R/qtl. Further integration with R/qtl as well as other QTL mapping programs should be relatively straightforward, providing that automation of the mapping functionality itself is achievable as it is with R/qtl. Permutations generated for advanced intercross lines were created using a custom python script designed for the purpose, the operation of which is described elsewhere <a href="#pone.0001036-Peirce1">[14]</a>. Permutation of populations with independent samples, such as F2, N2, and RI (phenotypic averages) populations can all be accomplished by simply shuffling the list of phenotypes. A Python script that does this and writes a file of permuted phenotypes, <strong>SimplePerm.py</strong>, is included.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>We wrote a set of two Python scripts, <strong>OutParser.py</strong>, <strong>and OutParserLI.py</strong> to parse the mapping output files from R/qtl and to generate a map of locus-specific P-values at regular, user defined intervals on a physical scale. These versions differ only in that OutParser.py calculates locus-specific P-values at markers then interpolates to regular positions, while OutParserLI.py (<strong>L</strong>OD <strong>I</strong>nterval) reverses the order of these steps. OutParserLI.py is somewhat slower but preferable for sparsely genotyped data sets. These scripts require a data file, <strong>markerphyspos.txt</strong> in the example files, which contains genetic positions for each marker (and/or interpolated marker if interval mapping has been performed) in the R/qtl output file as well as physical positions for as many markers as possible. Because the locus-specific P-values will often be used in combination with a similar set of values, defined at the same intervals, in another population (see description of <strong>CombineMaps.py</strong> below), the P values at the ends of each chromosome must be handled carefully. (If different data sources are associated with different markers, the beginning and ending points of each chromosome may well be different.) The assumption we have made is to repeat the most proximal actual marker value to the most proximal end of a given chromosome and likewise to repeat the most distal value to the distal end. Further, each chromosome is defined as 200 Mb in length (This is just slightly longer than the longest actual mouse chromosome. The user can adjust this value.) to ensure that values from each new data set can be easily accommodated and to eliminate the need for a separate file specifying length of each chromosome. This is also a convenient convention for plotting composite maps as the “unused” regions can be easily masked and graphs from multiple data sources, which may differ slightly in chromosome length and markers, can easily be lined up and overlaid.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p><strong>CombineMaps.py</strong> combines QTL maps of locus-specific P-values from different data sources. In order to generate combined maps of original data the program simply applies Fisher's combination test to the locus-specific P-values at equivalent physical positions for each defined position in the genome. The program also implements the method described above for finding genome-wide adjusted P-values. It does this by sampling with replacement from the set of permuted QTL maps for each data source. These maps are combined as described above and the best combined locus-specific P-value is recorded. This list of best P-values can be sorted (easily done in Excel or in Unix using “sort –n filename&gt;sortedfilename”) to allow genome-wide interpretation of locus-specific P-values.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Arthur Centeno for computer support, Amanda E. Birmingham for assistance in optimization and presentation of the associated software, and Pamela Franklin for administrative assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KB JP. Analyzed the data: KB RW JP. Contributed reagents/materials/analysis tools: LL. Wrote the paper: JP. Other: Provided support, advice, and editing: RW LL. Wrote code: JP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001036-Darvasi1" id="pone.0001036-Darvasi1"></a>Darvasi A, Soller M (1995) Advanced intercross lines, an experimental population for fine genetic mapping. Genetics  141: 1199–1207.  <ul class="find" data-citedArticleID="1005322" data-doi="10.1016/b978-0-12-374984-0.00021-8"><li><a href="http://dx.doi.org/10.1016/b978-0-12-374984-0.00021-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advanced+intercross+lines%2C+an+experimental+population+for+fine+genetic+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advanced+intercross+lines%2C+an+experimental+population+for+fine+genetic+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001036-Zou1" id="pone.0001036-Zou1"></a>Zou F, Gelfond JA, Airey DC, Lu L, Manly KF, et al.  (2005) Quantitative trait locus analysis using recombinant inbred intercrosses (RIX): theoretical and empirical considerations. Genetics.   <ul class="find" data-citedArticleID="1005366" data-doi="10.1534/genetics.104.035709"><li><a href="http://dx.doi.org/10.1534/genetics.104.035709" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+locus+analysis+using+recombinant+inbred+intercrosses+%28RIX%29%3A+theoretical+and+empirical+considerations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+locus+analysis+using+recombinant+inbred+intercrosses+%28RIX%29%3A+theoretical+and+empirical+considerations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001036-Iakoubova1" id="pone.0001036-Iakoubova1"></a>Iakoubova OA, Olsson CL, Dains KM, Ross DA, Andalibi A, et al.  (2001) Genome-tagged mice (GTM): two sets of genome-wide congenic strains. Genomics  74: 89–104.  <ul class="find" data-citedArticleID="1005336" data-doi="10.1006/geno.2000.6497"><li><a href="http://dx.doi.org/10.1006/geno.2000.6497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-tagged+mice+%28GTM%29%3A+two+sets+of+genome-wide+congenic+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-tagged+mice+%28GTM%29%3A+two+sets+of+genome-wide+congenic+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001036-Bolivar1" id="pone.0001036-Bolivar1"></a>Bolivar VJ, Cook MN, Flaherty L (2001) Mapping of quantitative trait loci with knockout/congenic strains. Genome Res  11: 1549–1552.  <ul class="find" data-citedArticleID="1005314" data-doi="10.1101/gr.194001"><li><a href="http://dx.doi.org/10.1101/gr.194001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+quantitative+trait+loci+with+knockout%2Fcongenic+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+quantitative+trait+loci+with+knockout%2Fcongenic+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001036-Darvasi2" id="pone.0001036-Darvasi2"></a>Darvasi A (1998) Experimental strategies for the genetic dissection of complex traits in animal models. Nat Genet  18: 19–24.  <ul class="find" data-citedArticleID="1005324" data-doi="10.1038/ng0198-19"><li><a href="http://dx.doi.org/10.1038/ng0198-19" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+strategies+for+the+genetic+dissection+of+complex+traits+in+animal+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+strategies+for+the+genetic+dissection+of+complex+traits+in+animal+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001036-Belknap1" id="pone.0001036-Belknap1"></a>Belknap JK, Atkins AL (2001) The replicability of QTLs for murine alcohol preference drinking behavior across eight independent studies. Mamm Genome  12: 893–899.  <ul class="find" data-citedArticleID="1005312" data-doi="10.1007/s00335-001-2074-2"><li><a href="http://dx.doi.org/10.1007/s00335-001-2074-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+replicability+of+QTLs+for+murine+alcohol+preference+drinking+behavior+across+eight+independent+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+replicability+of+QTLs+for+murine+alcohol+preference+drinking+behavior+across+eight+independent+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001036-Williams1" id="pone.0001036-Williams1"></a>Williams RW, Strom RC, Goldowitz D (1998) Natural variation in neuron number in mice is linked to a major quantitative trait locus on Chr 11. J Neurosci  18: 138–146.  <ul class="find" data-citedArticleID="1005364"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Natural+variation+in+neuron+number+in+mice+is+linked+to+a+major+quantitative+trait+locus+on+Chr+11.&amp;auth=&amp;atitle=Natural+variation+in+neuron+number+in+mice+is+linked+to+a+major+quantitative+trait+locus+on+Chr+11." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+variation+in+neuron+number+in+mice+is+linked+to+a+major+quantitative+trait+locus+on+Chr+11." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+variation+in+neuron+number+in+mice+is+linked+to+a+major+quantitative+trait+locus+on+Chr+11.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001036-Province1" id="pone.0001036-Province1"></a>Province MA (2001) The significance of not finding a gene. Am J Hum Genet  69: 660–663.  <ul class="find" data-citedArticleID="1005352" data-doi="10.1086/323316"><li><a href="http://dx.doi.org/10.1086/323316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+significance+of+not+finding+a+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+significance+of+not+finding+a+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001036-Walling1" id="pone.0001036-Walling1"></a>Walling GA, Visscher PM, Andersson L, Rothschild MF, Wang L, et al.  (2000) Combined analyses of data from quantitative trait loci mapping studies. Chromosome 4 effects on porcine growth and fatness. Genetics  155: 1369–1378.  <ul class="find" data-citedArticleID="1005360"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Combined+analyses+of+data+from+quantitative+trait+loci+mapping+studies.+Chromosome+4+effects+on+porcine+growth+and+fatness.&amp;auth=&amp;atitle=Combined+analyses+of+data+from+quantitative+trait+loci+mapping+studies.+Chromosome+4+effects+on+porcine+growth+and+fatness." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combined+analyses+of+data+from+quantitative+trait+loci+mapping+studies.+Chromosome+4+effects+on+porcine+growth+and+fatness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combined+analyses+of+data+from+quantitative+trait+loci+mapping+studies.+Chromosome+4+effects+on+porcine+growth+and+fatness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001036-Hitzemann1" id="pone.0001036-Hitzemann1"></a>Hitzemann R, Malmanger B, Cooper S, Coulombe S, Reed C, et al.  (2002) Multiple cross mapping (MCM) markedly improves the localization of a QTL for ethanol-induced activation. Genes Brain Behav  1: 214–222.  <ul class="find" data-citedArticleID="1005332" data-doi="10.1034/j.1601-183x.2002.10403.x"><li><a href="http://dx.doi.org/10.1034/j.1601-183x.2002.10403.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+cross+mapping+%28MCM%29+markedly+improves+the+localization+of+a+QTL+for+ethanol-induced+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+cross+mapping+%28MCM%29+markedly+improves+the+localization+of+a+QTL+for+ethanol-induced+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001036-Park1" id="pone.0001036-Park1"></a>Park YG, Clifford R, Buetow KH, Hunter KW (2003) Multiple cross and inbred strain haplotype mapping of complex-trait candidate genes. Genome Res  13: 118–121.  <ul class="find" data-citedArticleID="1005346" data-doi="10.1101/gr.786403"><li><a href="http://dx.doi.org/10.1101/gr.786403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+cross+and+inbred+strain+haplotype+mapping+of+complex-trait+candidate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+cross+and+inbred+strain+haplotype+mapping+of+complex-trait+candidate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001036-Hitzemann2" id="pone.0001036-Hitzemann2"></a>Hitzemann R, Malmanger B, Reed C, Lawler M, Hitzemann B, et al.  (2003) A strategy for the integration of QTL, gene expression, and sequence analyses. Mamm Genome  14: 733–747.  <ul class="find" data-citedArticleID="1005334" data-doi="10.1007/s00335-003-2277-9"><li><a href="http://dx.doi.org/10.1007/s00335-003-2277-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+strategy+for+the+integration+of+QTL%2C+gene+expression%2C+and+sequence+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+strategy+for+the+integration+of+QTL%2C+gene+expression%2C+and+sequence+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001036-Li1" id="pone.0001036-Li1"></a>Li R, Lyons MA, Wittenburg H, Paigen B, Churchill GA (2005) Combining data from multiple inbred line crosses improves the power and resolution of quantitative trait loci mapping. Genetics  169: 1699–1709.  <ul class="find" data-citedArticleID="1005342" data-doi="10.1534/genetics.104.033993"><li><a href="http://dx.doi.org/10.1534/genetics.104.033993" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combining+data+from+multiple+inbred+line+crosses+improves+the+power+and+resolution+of+quantitative+trait+loci+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combining+data+from+multiple+inbred+line+crosses+improves+the+power+and+resolution+of+quantitative+trait+loci+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001036-Peirce1" id="pone.0001036-Peirce1"></a>Peirce JL, Broman KW, Lu L, Chesler EJ, Zhou G, et al. Genome Reshuffling for Advanced Intercross Line Permutation. submitted  <ul class="find" data-citedArticleID="1005348" data-doi="10.1371/journal.pone.0001977"><li><a href="http://dx.doi.org/10.1371/journal.pone.0001977" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+Reshuffling+for+Advanced+Intercross+Line+Permutation.+submitted" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+Reshuffling+for+Advanced+Intercross+Line+Permutation.+submitted%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001036-Haldane1" id="pone.0001036-Haldane1"></a>Haldane JBS, Waddington CH (1931) Inbreeding and linkage. Genetics  16: 357–374.  <ul class="find" data-citedArticleID="1005330"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inbreeding+and+linkage.&amp;auth=&amp;atitle=Inbreeding+and+linkage." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inbreeding+and+linkage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inbreeding+and+linkage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001036-Fisher1" id="pone.0001036-Fisher1"></a>Fisher R (1925) Statistical methods for research workers. London: Oliver and Lloyd.   <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0001036-Sen1" id="pone.0001036-Sen1"></a>Sen S, Churchill GA (2001) A statistical framework for quantitative trait mapping. Genetics  159: 371–387.  <ul class="find" data-citedArticleID="1005354"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+statistical+framework+for+quantitative+trait+mapping.&amp;auth=&amp;atitle=A+statistical+framework+for+quantitative+trait+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+statistical+framework+for+quantitative+trait+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+statistical+framework+for+quantitative+trait+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001036-Peirce2" id="pone.0001036-Peirce2"></a>Peirce JL, Lu L, Gu J, Silver LM, Williams RW (2004) A new set of BXD recombinant inbred lines from advanced intercross populations in mice. BMC Genet  5: 7.  <ul class="find" data-citedArticleID="1005350"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice.&amp;auth=&amp;atitle=A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001036-Lu1" id="pone.0001036-Lu1"></a>Lu L, Airey DC, Williams RW (2001) Complex trait analysis of the hippocampus: mapping and biometric analysis of two novel gene loci with specific effects on hippocampal structure in mice. J Neurosci  21: 3503–3514.  <ul class="find" data-citedArticleID="1005344"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice.&amp;auth=&amp;atitle=Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001036-Lander1" id="pone.0001036-Lander1"></a>Lander ES, Botstein D (1989) Mapping mendelian factors underlying quantitative traits using RFLP linkage maps. Genetics  121: 185–199.  <ul class="find" data-citedArticleID="1005338"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mapping+mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps.&amp;auth=&amp;atitle=Mapping+mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001036-Dempster1" id="pone.0001036-Dempster1"></a>Dempster AP, Laird NM, Rubin DB (1977) Maximum likelihood from incomplete data via the EM algorithm. J Roy Statist Soc B  39: 1–38.  <ul class="find" data-citedArticleID="1005326"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum+likelihood+from+incomplete+data+via+the+EM+algorithm.&amp;auth=&amp;atitle=Maximum+likelihood+from+incomplete+data+via+the+EM+algorithm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+from+incomplete+data+via+the+EM+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+from+incomplete+data+via+the+EM+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001036-Lander2" id="pone.0001036-Lander2"></a>Lander E, Kruglyak L (1995) Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results. Nat Genet  11: 241–247.  <ul class="find" data-citedArticleID="1005340" data-doi="10.1038/ng1195-241"><li><a href="http://dx.doi.org/10.1038/ng1195-241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+dissection+of+complex+traits%3A+guidelines+for+interpreting+and+reporting+linkage+results." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+dissection+of+complex+traits%3A+guidelines+for+interpreting+and+reporting+linkage+results.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001036-Broman1" id="pone.0001036-Broman1"></a>Broman KW, Wu H, Sen S, Churchill GA (2003) R/qtl: QTL mapping in experimental crosses. Bioinformatics  19: 889–890.  <ul class="find" data-citedArticleID="1005316" data-doi="10.1093/bioinformatics/btg112"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=R%2Fqtl%3A+QTL+mapping+in+experimental+crosses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22R%2Fqtl%3A+QTL+mapping+in+experimental+crosses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001036-Chesler1" id="pone.0001036-Chesler1"></a>Chesler EJ, Lu L, Wang J, Williams RW, Manly KF (2004) WebQTL: rapid exploratory analysis of gene expression and genetic networks for brain and behavior. Nat Neurosci  7: 485–486.  <ul class="find" data-citedArticleID="1005318" data-doi="10.1038/nn0504-485"><li><a href="http://dx.doi.org/10.1038/nn0504-485" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebQTL%3A+rapid+exploratory+analysis+of+gene+expression+and+genetic+networks+for+brain+and+behavior." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebQTL%3A+rapid+exploratory+analysis+of+gene+expression+and+genetic+networks+for+brain+and+behavior.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001036-Wang1" id="pone.0001036-Wang1"></a>Wang J, Williams RW, Manly KF (2003) WebQTL: web-based complex trait analysis. Neuroinformatics  1: 299–308.  <ul class="find" data-citedArticleID="1005362" data-doi="10.1385/ni:1:4:299"><li><a href="http://dx.doi.org/10.1385/ni:1:4:299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebQTL%3A+web-based+complex+trait+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebQTL%3A+web-based+complex+trait+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001036-Taylor1" id="pone.0001036-Taylor1"></a>Taylor BA (1989)  Recombinant inbred strains. In: Lyon ML, Searle AG, editors. Genetic Variants and Strains of the Laboratory Mouse. Oxford: Oxford UP. pp. 773–796.  <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0001036-Taylor2" id="pone.0001036-Taylor2"></a>Taylor BA, Wnek C, Kotlus BS, Roemer N, MacTaggart T, et al.  (1999) Genotyping new BXD recombinant inbred mouse strains and comparison of BXD and consensus maps. Mamm Genome  10: 335–348.  <ul class="find" data-citedArticleID="1005358" data-doi="10.1007/s003359900998"><li><a href="http://dx.doi.org/10.1007/s003359900998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+new+BXD+recombinant+inbred+mouse+strains+and+comparison+of+BXD+and+consensus+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+new+BXD+recombinant+inbred+mouse+strains+and+comparison+of+BXD+and+consensus+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001036-Churchill1" id="pone.0001036-Churchill1"></a>Churchill GA, Doerge RW (1994) Empirical threshold values for quantitative trait mapping. Genetics  138: 963–971.  <ul class="find" data-citedArticleID="1005320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Empirical+threshold+values+for+quantitative+trait+mapping.&amp;auth=&amp;atitle=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+threshold+values+for+quantitative+trait+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.XML" value="66092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.PDF" value="220306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g001.PNG_L" value="46110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g001.PNG_M" value="46110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g001.PNG_S" value="11041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g001.TIF" value="61542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g001.PNG_I" value="31883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g002.PNG_L" value="506660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g002.PNG_M" value="71262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g002.PNG_S" value="9417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g002.TIF" value="709160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.g002.PNG_I" value="27946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.e001.PNG" value="10381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.e001.TIF" value="26596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.e002.PNG" value="9870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001036.e002.TIF" value="26076"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001036&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001036" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001036&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001036&volume=&issue=&title=A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs&author_name=Jeremy%20L.%20Peirce%2C%20Karl%20W.%20Broman%2C%20Lu%20Lu%2C%20Robert%20W.%20Williams&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001036" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001036" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001036" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001036&amp;t=A%20Simple%20Method%20for%20Combining%20Genetic%20Mapping%20Data%20from%20Multiple%20Crosses%20and%20Experimental%20Designs" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001036&title=A%20Simple%20Method%20for%20Combining%20Genetic%20Mapping%20Data%20from%20Multiple%20Crosses%20and%20Experimental%20Designs&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001036&amp;title=A%20Simple%20Method%20for%20Combining%20Genetic%20Mapping%20Data%20from%20Multiple%20Crosses%20and%20Experimental%20Designs" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001036" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001036&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Simple Method for Combining Genetic Mapping Data from Multiple Crosses and Experimental Designs';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001036';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Simple%20Method%20for%20Combining%20Genetic%20Mapping%20Data%20from%20Multiple%20Crosses%20and%20Experimental%20Designs http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001036" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001036" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="24906"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+mapping%22" title="Search for articles in the subject area:'Chromosome mapping'"><div class="flagText">Chromosome mapping</div></a>
              <div data-categoryid="19841" data-articleid="24906"
                   data-categoryname="Chromosome mapping"
                   class="flagImage" title="Flag 'Chromosome mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="24906"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="24906"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hippocampus%22" title="Search for articles in the subject area:'Hippocampus'"><div class="flagText">Hippocampus</div></a>
              <div data-categoryid="47345" data-articleid="24906"
                   data-categoryname="Hippocampus"
                   class="flagImage" title="Flag 'Hippocampus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Permutation%22" title="Search for articles in the subject area:'Permutation'"><div class="flagText">Permutation</div></a>
              <div data-categoryid="33641" data-articleid="24906"
                   data-categoryname="Permutation"
                   class="flagImage" title="Flag 'Permutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="24906"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Physical+mapping%22" title="Search for articles in the subject area:'Physical mapping'"><div class="flagText">Physical mapping</div></a>
              <div data-categoryid="49249" data-articleid="24906"
                   data-categoryname="Physical mapping"
                   class="flagImage" title="Flag 'Physical mapping' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1708'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2911'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2794&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
