

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast clpP1 Gene</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001386"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001386" />

    <meta name="citation_title" content="Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast clpP1 Gene"/>
    <meta itemprop="name" content="Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast clpP1 Gene"/>

      <meta name="citation_author" content="Per Erixon"/>
            <meta name="citation_author_institution" content="Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University, Uppsala, Sweden"/>
            <meta name="citation_author_institution" content="Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden"/>
      <meta name="citation_author" content="Bengt Oxelman"/>
            <meta name="citation_author_institution" content="Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University, Uppsala, Sweden"/>
            <meta name="citation_author_institution" content="Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001386.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1386"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Chloroplast genomes of plants.; citation_author=LA Raubeson; citation_author=RK Jansen; citation_number=1; citation_pages=45-68; citation_date=2005; citation_publisher=CABI Publishing; " />
      <meta name="citation_reference" content="citation_title=Rates of Nucleotide Substitution Vary Greatly among Plant Mitochondrial, Chloroplast, and Nuclear DNAs.; citation_author=KH Wolfe; citation_author=WH Li; citation_author=PM Sharp; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=84; citation_number=2; citation_pages=9054-9058; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Rates and patterns of chloroplast DNA evolution.; citation_author=MT Clegg; citation_author=BS Gaut; citation_author=GH Learn; citation_author=BR Morton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=3; citation_pages=6795-6801; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Comparing patterns of nucleotide substitution rates among chloroplast loci using the relative ratio test.; citation_author=SV Muse; citation_author=BS Gaut; citation_journal_title=Genetics; citation_volume=146; citation_number=4; citation_pages=393-399; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The Tortoise and the Hare II: Relative Utility of 21 Noncoding Chloroplast DNA Sequences for Phylogenetic Analysis.; citation_author=J Shaw; citation_author=EB Lickey; citation_author=JT Beck; citation_author=SB Farmer; citation_author=W Liu; citation_journal_title=Am J Bot; citation_volume=92; citation_number=5; citation_pages=142-166; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Endosymbiotic gene transfer: Organelle genomes forge eukaryotic chromosomes.; citation_author=JN Timmis; citation_author=MA Ayliffe; citation_author=CY Huang; citation_author=W Martin; citation_journal_title=Nature Rev Genet; citation_volume=5; citation_number=6; citation_pages=123-135; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation.; citation_author=ZH Yang; citation_author=JP Bielawski; citation_journal_title=Trends Ecol Evolut; citation_volume=15; citation_number=7; citation_pages=496-503; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Being Positive about Selection.; citation_author=C MacCallum; citation_author=E Hill; citation_journal_title=PLoS Biology; citation_volume=4; citation_number=8; citation_pages=e87; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular Adaption during Adaptive Radiation in the Hawaiian Endemic Genus Schiedea.; citation_author=MV Kapralov; citation_author=DA Filatov; citation_journal_title=PLoS ONE; citation_volume=1(1); citation_number=9; citation_pages=e8; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Widespread positive selection in the photosynthetic Rubisco enzyme.; citation_author=MV Kapralov; citation_author=DA Filatov; citation_journal_title=BMC Evolutionary Biology; citation_volume=7; citation_number=10; citation_pages=73; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Large-Scale Search for Genes on Which Positive Selection May Operate.; citation_author=T Endo; citation_author=K Ikeo; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=11; citation_pages=685-690; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Chloroplast and Mitochondrial Proteases in Arabidopsis. A Proposed Nomenclature.; citation_author=Z Adam; citation_author=I Adamska; citation_author=K Nakabayashi; citation_author=O Ostersetzer; citation_author=K Haussuhl; citation_journal_title=Plant Physiol; citation_volume=125; citation_number=12; citation_pages=1912-1918; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=ATP-dependent Clp Proteases in Photosynthetic Organisms-A Cut Above the Rest!; citation_author=AK Clarke; citation_journal_title=Ann Botany; citation_volume=83; citation_number=13; citation_pages=593-599; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The plastid clpP1 protease gene is essential for plant development.; citation_author=H Kuroda; citation_author=P Maliga; citation_journal_title=Nature; citation_volume=425; citation_number=14; citation_pages=86-89; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The chloroplast clpP gene, encoding a proteolytic subunit of ATP-dependent protease, is indispensable for chloroplast development in tobacco.; citation_author=T Shikanai; citation_author=K Shimizu; citation_author=K Ueda; citation_author=Y Nishimura; citation_author=T Kuroiwa; citation_journal_title=Plant Cell Physiol; citation_volume=42; citation_number=15; citation_pages=264-273; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The complete chloroplast genome sequence of the chlorophycean green alga Scenedesmus obliquus reveals a compact gene organization and a biased distribution of genes on the two DNA strands.; citation_author=J-C deCambiaire; citation_author=C Otis; citation_author=C Lemieux; citation_author=M Turmel; citation_journal_title=BMC Evol Biol; citation_volume=6; citation_number=16; citation_pages=37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A revised generic classification of the tribe Sileneae (Caryophyllaceae).; citation_author=B Oxelman; citation_author=M Lidén; citation_author=RK Rabeler; citation_author=M Popp; citation_journal_title=Nord J Bot; citation_volume=20; citation_number=17; citation_pages=743-748; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Reticulate or treelike chloroplast DNA evolution in Sileneae (Caryophyllaceae)?; citation_author=P Erixon; citation_author=B Oxelman; citation_number=18; citation_date=2006; citation_publisher=Uppsala University; " />
      <meta name="citation_reference" content="citation_title=Generic Boundaries in the Tribe Sileneae (Caryophyllaceae) as Inferred from Nuclear rDNA Sequences.; citation_author=B Oxelman; citation_author=M Lidén; citation_journal_title=Taxon; citation_volume=44; citation_number=19; citation_pages=525-542; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evolution of a RNA Polymerase Gene Family in Silene (Caryophyllaceae)-Incomplete Concerted Evolution and Topological Congruence Among Paralogues.; citation_author=M Popp; citation_author=B Oxelman; citation_journal_title=Syst Biol; citation_volume=53; citation_number=20; citation_pages=914-932; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=ZH Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=21; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial substitution rates are extraordinarily elevated and variable in a genus of flowering plants.; citation_author=Y Cho; citation_author=JP Mower; citation_author=Y-L Qiu; citation_author=JD Palmer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=22; citation_pages=17741-17746; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages.; citation_author=ZH Yang; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=23; citation_pages=908-917; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Frequent false detection of positive selection by the likelihood method with branch-site models.; citation_author=JZ Zhang; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=24; citation_pages=1332-1339; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Positive selection of a gene family during the emergence of humans and African apes.; citation_author=ME Johnson; citation_author=L Viggiano; citation_author=JA Bailey; citation_author=M Abdul-Rauf; citation_author=G Goodwin; citation_journal_title=Nature; citation_volume=413; citation_number=25; citation_pages=514-519; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evidence of functional selection pressure for alternative splicing events that accelerate evolution of protein subsequences.; citation_author=Y Xing; citation_author=C Lee; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=26; citation_pages=13526-13531; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A systematic search for positive selection in higher plants (Embryophytes).; citation_author=C Roth; citation_author=DA Liberles; citation_journal_title=BMC Plant Biology; citation_volume=6; citation_number=27; citation_pages=12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pattern of Nucleotide Substitutions in Growth Hormone-Prolactin Gene Family: A Paradigm for Evolution by Gene Duplication.; citation_author=T Ohta; citation_journal_title=Genetics; citation_volume=134; citation_number=28; citation_pages=1271-1276; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Positive Darwinian selection after gene duplication in primate ribonuclease genes.; citation_author=Z Zhang; citation_author=HF Rosenberg; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=29; citation_pages=3708-3713; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=GenBank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=30; citation_pages=D21-25; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Paraphyly in tribe Onagreae: Insights into phylogenetic relationships of Onagraceae based on nuclear and chloroplast sequence data.; citation_author=RA Levin; citation_author=WL Wagner; citation_author=PC Hoch; citation_author=WJ Hahn; citation_author=A Rodriguez; citation_journal_title=Syst Bot; citation_volume=29; citation_number=31; citation_pages=147-164; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=D Higgins; citation_author=J Thompson; citation_author=T Gibson; citation_author=JD Thompson; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=32; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Chloroplast rps16 intron phylogeny of the tribe Sileneae (Caryophyllaceae).; citation_author=B Oxelman; citation_author=M Lidén; citation_author=D Berglund; citation_journal_title=Plant Syst Evol; citation_volume=206; citation_number=33; citation_pages=393-410; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MrBayes: Bayesian inference of phylogeny.; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=34; citation_pages=754-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A Simple, Fast, and Accurate Algorithm to Estimate Large Phylogenies by Maximum Likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=35; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An update of the Angiosperm Phylogeny Group classification for the orders and families of flowering plants: APG II.; citation_author=II APG; citation_journal_title=Bot J Linnean Soc; citation_volume=141; citation_number=36; citation_pages=399-436; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A phylogeny of Legumes (Leguminosae) based on analysis of the plastid matK gene resolves many well-supported subclades within the family.; citation_author=MF Wojciechowski; citation_author=M Lavin; citation_author=MJ Sanderson; citation_journal_title=Am J Bot; citation_volume=91; citation_number=37; citation_pages=1846-1862; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and provisional classification of Solanaceae based on chloroplast DNA.; citation_author=RG Olmstead; citation_author=JA Sweere; citation_author=RE Spangler; citation_author=L Bohs; citation_author=JD Palmer; citation_number=38; citation_pages=111-137; citation_date=1999; citation_publisher=Royal Botanical Gardens; " />
      <meta name="citation_reference" content="citation_title=Analysis of the Amborella trichopoda Chloroplast Genome Sequence Suggests That Amborella Is Not a Basal Angiosperm.; citation_author=VV Goremykin; citation_author=KI Hirsch-Ernst; citation_author=S Wölfl; citation_author=FH Hellwig; citation_journal_title=Mol Bio Evol; citation_volume=20; citation_number=39; citation_pages=1499-1505; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees.; citation_author=A Rambaut; citation_author=NC Grassly; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=40; citation_pages=235-238; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001386" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast clpP1 Gene"/>
    <meta name="twitter:description" content="BackgroundSynonymous DNA substitution rates in the plant chloroplast genome are generally relatively slow and lineage dependent. Non-synonymous rates are usually even slower due to purifying selection acting on the genes. Positive selection is expected to speed up non-synonymous substitution rates, whereas synonymous rates are expected to be unaffected. Until recently, positive selection has seldom been observed in chloroplast genes, and large-scale structural rearrangements leading to gene duplications are hitherto supposed to be rare.Methodology/Principle FindingsWe found high substitution rates in the exons of the plastid clpP1 gene in Oenothera (the Evening Primrose family) and three separate lineages in the tribe Sileneae (Caryophyllaceae, the Carnation family). Introns have been lost in some of the lineages, but where present, the intron sequences have substitution rates similar to those found in other introns of their genomes. The elevated substitution rates of clpP1 are associated with statistically significant whole-gene positive selection in three branches of the phylogeny. In two of the lineages we found multiple copies of the gene. Neighboring genes present in the duplicated fragments do not show signs of elevated substitution rates or positive selection. Although non-synonymous substitutions account for most of the increase in substitution rates, synonymous rates are also markedly elevated in some lineages. Whereas plant clpP1 genes experiencing negative (purifying) selection are characterized by having very conserved lengths, genes under positive selection often have large insertions of more or less repetitive amino acid sequence motifs.Conclusions/SignificanceWe found positive selection of the clpP1 gene in various plant lineages to correlated with repeated duplication of the clpP1 gene and surrounding regions, repetitive amino acid sequences, and increase in synonymous substitution rates. The present study sheds light on the controversial issue of whether negative or positive selection is to be expected after gene duplications by providing evidence for the latter alternative. The observed increase in synonymous substitution rates in some of the lineages indicates that the detection of positive selection may be obscured under such circumstances. Future studies are required to explore the functional significance of the large inserted repeated amino acid motifs, as well as the possibility that synonymous substitution rates may be affected by positive selection."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001386.g005"/>

  <meta property="og:title" content="Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast clpP1 Gene" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001386" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4026'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2601'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7095&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001386">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001386" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001386&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast <i>clpP1</i> Gene
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Per Erixon
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:per.erixon@ebc.uu.se">per.erixon@ebc.uu.se</a></p>

                <p>Affiliations:
                  Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University, Uppsala, Sweden, 
                  Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bengt Oxelman
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University, Uppsala, Sweden, 
                  Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001386</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001386" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001386" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001386" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001386" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001386-g001" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001386-t001" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001386-g002" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001386-g003" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001386-g004" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001386-t002" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001386-g005" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001386">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001386" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Synonymous DNA substitution rates in the plant chloroplast genome are generally relatively slow and lineage dependent. Non-synonymous rates are usually even slower due to purifying selection acting on the genes. Positive selection is expected to speed up non-synonymous substitution rates, whereas synonymous rates are expected to be unaffected. Until recently, positive selection has seldom been observed in chloroplast genes, and large-scale structural rearrangements leading to gene duplications are hitherto supposed to be rare.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We found high substitution rates in the exons of the plastid <em>clpP1</em> gene in <em>Oenothera</em> (the Evening Primrose family) and three separate lineages in the tribe <em>Sileneae</em> (Caryophyllaceae, the Carnation family). Introns have been lost in some of the lineages, but where present, the intron sequences have substitution rates similar to those found in other introns of their genomes. The elevated substitution rates of <em>clpP1</em> are associated with statistically significant whole-gene positive selection in three branches of the phylogeny. In two of the lineages we found multiple copies of the gene. Neighboring genes present in the duplicated fragments do not show signs of elevated substitution rates or positive selection. Although non-synonymous substitutions account for most of the increase in substitution rates, synonymous rates are also markedly elevated in some lineages. Whereas plant <em>clpP1</em> genes experiencing negative (purifying) selection are characterized by having very conserved lengths, genes under positive selection often have large insertions of more or less repetitive amino acid sequence motifs.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We found positive selection of the <em>clpP1</em> gene in various plant lineages to correlated with repeated duplication of the <em>clpP1</em> gene and surrounding regions, repetitive amino acid sequences, and increase in synonymous substitution rates. The present study sheds light on the controversial issue of whether negative or positive selection is to be expected after gene duplications by providing evidence for the latter alternative. The observed increase in synonymous substitution rates in some of the lineages indicates that the detection of positive selection may be obscured under such circumstances. Future studies are required to explore the functional significance of the large inserted repeated amino acid motifs, as well as the possibility that synonymous substitution rates may be affected by positive selection.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Erixon P, Oxelman B (2008) Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast <em>clpP1</em> Gene. PLoS ONE 3(1):
          e1386.
            doi:10.1371/journal.pone.0001386</p><p><strong>Academic Editor: </strong>Jean-Nicolas Volff, Ecole Normale Supérieure de Lyon, France</p><p><strong>Received:</strong> March 12, 2007; <strong>Accepted:</strong> December 5, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Erixon, Oxelman. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the Linnaeus Centre for Bioinformatics, Uppsala University, and by grants from the Helge Ax:son Johnson foundation, the Nilsson-Ehle foundation, and the Royal Swedish Academy of Sciences to PE and partly by a grant from the Swedish Research Council to BO.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The circular chloroplast genome is in general expected to be a non-recombining unit where large within-genome duplications are rare. Most of its genes are single-copy, occurring in large and small single-copy regions, that are intervened by inverted repeat regions <a href="#pone.0001386-Raubeson1">[1]</a>. Substitution rates of chloroplast DNA (cpDNA) are held to be relatively slow and not very variable, although not constant, among lineages <a href="#pone.0001386-Wolfe1">[2]</a>–<a href="#pone.0001386-Shaw1">[5]</a>. The gene content is likewise thought to be well conserved <a href="#pone.0001386-Timmis1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Most reports of positive selection are from the human genome or other model organisms <a href="#pone.0001386-Yang1">[7]</a>, <a href="#pone.0001386-MacCallum1">[8]</a>, and documented significant positive bias of non-synonymous (dN) over synonymous (dS) substitutions from non-model organisms and/or entire genes are rare (but see <a href="#pone.0001386-Kapralov1">[9]</a> and <a href="#pone.0001386-Kapralov2">[10]</a>). From an evolutionary perspective it is to be expected that some genes (e.g. those involved in the immune system) can have specific sites that are under positive selection <a href="#pone.0001386-Endo1">[11]</a>, but genes that exhibit positive selection as a whole and with non-synonymous substitutions more or less evenly distributed over the entire length of the gene are clearly more enigmatic and of greater general evolutionary interest.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The chloroplast-encoded <em>clpP1</em> (caseinolytic peptidase, ATP-dependent, proteolytic subunit) is part of a gene family encoding <em>clpP</em> proteases with six members in <em>Arabidopsis</em> of the mustard family Brassicaceae <a href="#pone.0001386-Adam1">[12]</a>. The other five members are encoded in the nucleus (<em>clpP2</em>-<em>clpP6</em>) <a href="#pone.0001386-Adam1">[12]</a>. The main function of the protein is to degrade polypeptides, but the <em>clpP</em> proteases are involved in a variety of processes, ranging from developmental changes to stress tolerance <a href="#pone.0001386-Clarke1">[13]</a>. It has been suggested that the <em>clpP1</em> gene is essential for plant cell viability <a href="#pone.0001386-Kuroda1">[14]</a>, <a href="#pone.0001386-Shikanai1">[15]</a>. The gene is found in the chloroplast genome of all higher plants and most eukaryotic algae <a href="#pone.0001386-Adam1">[12]</a>. The structure of the gene and the amino acid composition are highly conserved with 196 amino acid residues distributed over three exons and with two intervening introns, but <em>Oenothera</em>, grasses, and the conifer genus <em>Pinus</em> lack the two introns. Most green algae also lack introns, but they have roughly the same number of amino acid residues as land plants. However, <em>Chlamydomonas</em> and <em>Scenedesmus</em> (both Chlorophyceae) have large insertions resulting in a total of 525 and 538 amino acids, respectively. It has been shown that the insertion sequence is not removed during transcription <a href="#pone.0001386-deCambiaire1">[16]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The present study makes a strong case for the existence of whole-gene positive selection on the chloroplast <em>clpP1</em> gene in at least three separate lineages of flowering plants. The positive selection appears to be linked to multiple gene-region duplications, elevation of synonymous substitution rates, and insertion of repetitive amino acid sequence motifs.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>In all 25 flowering plant species sequenced for this study (21 <em>Sileneae</em>, 4 <em>Oenothera</em>), there is one copy of the <em>clpP1</em> gene located between the <em>rps12</em> and <em>psbB</em> genes. None of these had frame shifts or premature stop codons, and thus appeared functional. This is in accordance with all published land plant chloroplast genomes. Additional <em>clpP1</em> copies were found in two of the species (<a href="#pone-0001386-g001">Fig. 1</a>). Several of the <em>clpP1</em> sequences lack introns (<a href="#pone-0001386-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001386-g001"><div class="img"><a name="pone-0001386-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Multiple Copies of the <em>clpP1</em> Gene Region Found in Two Species.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>DNA fragments sequenced from the <em>clpP1</em> region (thick black bars) for <em>Lychnis chalcedonica</em> (Lc) and <em>Silene fruticosa</em> (Sf). Shown in gray is the corresponding region, with the genes (boxes) and introns (dotted lines), in <em>Spinacia</em>. Thick white bars indicate non-homologous flanking regions (Lc2 is located differently in the chloroplast genome). Fragments with arrows on the left side consist of a continuous sequence from <em>rbcL</em> to <em>petB</em> (c. 18 kb). Thin lines mark lack of introns. Ψ indicates stop codon. ? indicates unknown flaking regions.</p>
<span>doi:10.1371/journal.pone.0001386.g001</span></div><div class="figure" id="pone-0001386-t001"><div class="img"><a name="pone-0001386-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001386.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001386.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Length variation<sup>a</sup>, PCR primers, and amino acid repeats in the <em>clpP1</em> region.</span></strong></p><span>doi:10.1371/journal.pone.0001386.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Six of 21 investigated species of <em>Sileneae</em> showed signs of elevated branch lengths in the <em>clpP1</em> exons (<a href="#pone-0001386-g002">Fig. 2B</a>). These taxa are distributed in three phylogenetically distinct lineages <a href="#pone.0001386-Oxelman1">[17]</a>–<a href="#pone.0001386-Popp1">[20]</a>: two closely related species in <em>Silene</em> subgenus <em>Behen</em> (<em>S. conica</em> and <em>S. conoidea</em>), one species in <em>Silene</em> subgenus <em>Silene</em> (<em>S. fruticosa</em>), and three of four investigated <em>Lychnis</em> species (<em>L. chalcedonica</em>, <em>L. flos-cuculi</em>, and <em>L. abyssinica</em>). Three of the six species (<em>S. conica</em>, <em>S. fruticosa</em>, and <em>L. chalcedonica</em>) have been extensively sequenced (&gt;25 kb) also for other, both coding and non-coding chloroplast regions, but do not display such extreme branch lengths elsewhere in the plastid genome <a href="#pone.0001386-Erixon1">[18]</a>. Surprisingly, the synonymous substitution rates in two of these three species are clearly elevated (up to five times) in <em>clpP1</em> when compared with other cpDNA genes (<a href="#pone-0001386-g003">Fig. 3</a>). All four investigated species of <em>Oenothera</em> had elevated substitution rates. Only <em>O. flava</em> possess introns of the four, and it is also the one with the shortest branch length (<a href="#pone-0001386-g002">Fig. 2B</a>). Synonymous substitution rates in the <em>clpP1</em> gene of <em>O. elata</em> are 2.5–3.2 times higher than other chloroplast genes (<a href="#pone-0001386-g003">Fig. 3</a>).</p>
<div class="figure" id="pone-0001386-g002"><div class="img"><a name="pone-0001386-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Positive Selection in the Chloroplast <em>clpP</em> Gene.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>A) dN/dS ratios calculated on the Eudicot tree topology. Numbers on nodes indicate classification as follows: 1 Myrtales, 2 Eurosids II, 5 Eurosids I, 6 Rosids, 13 Caryophyllales, 17 Euasterids I, 19 Euasterids II, 20 Asterids <a href="#pone.0001386-APG1">[36]</a>; 3 Hologalegina, 4 Fabaceae <a href="#pone.0001386-Wojciechowski1">[37]</a>; 7, 8, 9 subgenus <em>Behen</em>, 10 subgenus <em>Silene</em>, 11, 12 <em>Sileneae</em> <a href="#pone.0001386-Oxelman1">[17]</a>; 14 Solaneae, 15 Solanoideae, 16 Solanaceae <a href="#pone.0001386-Olmstead1">[38]</a>. Values on branches are the dN/dS ratios, S: dN = 0, N: dS = 0. Ratios significantly above one (Bonferroni-corrected) are indicated by asterisks: ***P&lt;0.001, *P&lt;0.05. B) dN- and dS-branch (in grey and black, respectively) lengths imposed on topology from Bayesian analysis of third codon positions from the <em>clpP1</em> exons Numbers on nodes are Bayesian posterior probabilities (Bpp). Only Bpp values&gt;0.50 are shown.</p>
<span>doi:10.1371/journal.pone.0001386.g002</span></div><div class="figure" id="pone-0001386-g003"><div class="img"><a name="pone-0001386-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>dS and dN Values of Four Chloroplast Genes.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>Diagram shows the pairwise dS values (top) and dN values (bottom) between eleven species of <em>Sileneae</em> and the outgroup (<em>Heliosperma alpestre</em>) for four chloroplast genes (<em>clpP1</em>; 591 bp, <em>psbB</em>; 1527 bp, <em>cemA</em>; 639 bp, and <em>petA</em>; 963 bp). To the right of the vertical gray line is the pairwise comparison between <em>Oenothera elata</em> and <em>Eucalyptus globulus</em>, for the same genes.</p>
<span>doi:10.1371/journal.pone.0001386.g003</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The position and length of the <em>clpP1</em> exons are generally conserved in <em>Sileneae</em>, like in most other angiosperms (<a href="#pone-0001386-t001">Table 1</a>), but exceptions were found in the six <em>Sileneae</em> species with long branches and the <em>Oenothera</em> sequences. <em>Silene conica</em> and <em>S. conoidea</em> share seven indels, of which the two largest insertions differ in length between the two species. Several of the species had long insertions partially consisting of amino acid repeats in their exons (<a href="#pone-0001386-t001">Table 1</a>).</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Analysis of all available seed plant <em>clpP1</em> sequences together with the five nuclear encoded gene family members (<em>clpP2</em>-<em>clpP6</em>) from <em>Arabidopsis</em> and <em>Oryza</em> strongly indicated (posterior probability 1.00) that the divergent sequences found in this study are of chloroplast origin (<a href="#pone.0001386.s001">Fig. S1</a>). The relationships within <em>Sileneae</em> in the gene family analysis, where all the long branches group together, are at odds with all previous cpDNA analyses (e.g. <a href="#pone.0001386-Erixon1">[18]</a>). The analysis of only third positions from the <em>clpP1</em> exons did, however, result in a topology compatible with other analyses of sequences from other cpDNA regions (<a href="#pone-0001386-g002">Fig. 2B</a>). For example, despite its very long branch, the <em>Silene conica</em> and <em>S. conoidea</em> clade grouped together with other members of <em>Silene</em> subgenus <em>Behen</em> (<a href="#pone-0001386-g002">Fig. 2B</a>).</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>The <em>Sileneae</em> phylogenies based on <em>clpP1</em> exon (<a href="#pone-0001386-g004">Fig. 4A</a>) and intron (<a href="#pone-0001386-g004">Fig. 4B</a>) sequences were substantially different. Intron sequences from <em>Lychnis chalcedonica</em> and <em>Silene fruticosa</em> did not show signs of elevated substitution rates relative to other cpDNA regions, and the phylogeny (<a href="#pone-0001386-g004">Fig. 4B</a>) is compatible with other <em>Sileneae</em> phylogenies based on cpDNA <a href="#pone.0001386-Erixon1">[18]</a>.</p>
<div class="figure" id="pone-0001386-g004"><div class="img"><a name="pone-0001386-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Bayesian Consensus Phylograms of the Tribe <em>Sileneae.</em></span></strong></p><a id="article1.body1.sec2.fig4.caption1.p1" name="article1.body1.sec2.fig4.caption1.p1"></a><p>A) The <em>clpP1</em> exon sequences, and B) the <em>clpP1</em> intron sequences. Branches in bold have Bayesian posterior probabilities (Bpp) of 1.00. Only Bpp values above 0.90 are shown at nodes. Note that A and B are drawn at different scales. ** indicates taxa that lack introns in <em>clpP1</em>.</p>
<span>doi:10.1371/journal.pone.0001386.g004</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>We used the method of Yang <a href="#pone.0001386-Yang2">[21]</a> to investigate the ratio of non-synonymous to synonymous substitutions (dN/dS, ω) and found that this ratio varies significantly among lineages (P&lt;0.0001), both under a generally accepted eudicot phylogeny (<a href="#pone-0001386-g002">Fig. 2A</a>) and under the tree topology from Bayesian analysis of only third positions from the <em>clpP1</em> exons (<a href="#pone-0001386-g002">Fig. 2B</a>). Several branches in both <em>Sileneae</em> and <em>Oenothera</em> resulted in ω&gt;1 (<a href="#pone-0001386-g002">Fig. 2</a>). In addition, the branches in Solanaceae have high ω-values, despite shorter branch lengths. Three branches have ω that are significantly higher than one after Bonferroni correction (<a href="#pone-0001386-t002">table 2</a>, <a href="#pone-0001386-g002">Fig. 2A</a>). The second topology (<a href="#pone-0001386-g002">Fig. 2B</a>) gives very similar ω-values, but the values of the three branches with significant positive selection are higher still, and two internal branches not resolved in the first topology (<a href="#pone-0001386-g002">Fig. 2A</a>) receive ω-values&gt;2.0 (<a href="#pone-0001386-g002">Fig. 2B</a>).</p>
<div class="figure" id="pone-0001386-t002"><div class="img"><a name="pone-0001386-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001386.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001386.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Branches with dN&gt;dS in the phylogeny in <a href="#pone-0001386-g001">figure 1A</a>.</span></strong></p><span>doi:10.1371/journal.pone.0001386.t002</span></div><a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>We reject the idea that the observed open reading frames in the <em>clpP1</em> sequences of <em>Silene conica</em> and <em>S. conoidea</em> have persisted by chance alone under a neutral model of evolution, because only 0.69% of simulated sequences with a branch length of 0.25 from the <em>S. latifolia</em> sequence appear to have remained functional (98.66% had premature stop-codons and an additional 0.65% lacked a proper start-codon). The average number of premature stop codons per simulated sequence was 4.3.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our results indicate highly elevated substitution rates in the chloroplast <em>clpP1</em> exons in <em>Sileneae</em> and in <em>Oenothera</em>. Although long branch lengths also could indicate high ages, we argue that this is unlikely because the comparisons with other genes (<a href="#pone-0001386-g003">Fig. 3</a>) which, unless acquired by horizontal transfer, must be of the same age. Even if the topological relationships sometimes are ambiguous, any resolution of the trees in <a href="#pone-0001386-g002">Fig. 2</a> will show that the sister group has a significantly shorter branch. As sister groups by definition has the same age, either the short branch has undergone a substitution rate decrease or vice versa. This supports the explanation for the long <em>clpP1</em> branches as likely to have been the result of a substitution rate increase.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In <em>Sileneae</em>, there are at least three independent increases in substitution rates. Loss of introns in <em>clpP1</em> appears correlated with elevated substitution rates in several lineages. In two of the <em>Sileneae</em> lineages, high substitution rates are accompanied with both gene duplications and significant positive selection.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Even if the rate of synonymous substitutions in the <em>clpP1</em> gene for the <em>Sileneae</em> species with generally elevated substitution rates appears higher than that of other chloroplast genes for the same taxa (<a href="#pone-0001386-g003">Fig. 3</a>), most of the total substitution rate increase can be ascribed to non-synonymous substitutions. A generally elevated substitution rate for a whole genome such as the mitochondrial genome of some species of <em>Plantago</em> could be explained by, e.g. an increased amount of oxygen free-radicals <a href="#pone.0001386-Cho1">[22]</a>, but elevated substitution rates in, and exclusive to, a specific gene are harder to explain. The very long branch leading to <em>Silene conica</em> and <em>S. conoidea</em> is puzzling in this context, because the rates of both synonymous and non-synonymous substitutions are very high, and there is no significant positive selection. The pattern is similar, but less extreme, in <em>L. chalcedonica</em> (<a href="#pone-0001386-g003">Fig. 3</a>). The length of the branches leading to <em>Solanum</em>/<em>Lycopersicon</em> (Solanaceae), within the Fabaceae, to <em>Vitis</em>, and to <em>Cucumis</em> might also indicate elevated substitution rates, but this is much less striking than in the long <em>Sileneae</em> branches and in <em>Oenothera</em> (<a href="#pone.0001386.s001">Fig. S1</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>By comparing synonymous and non-synonymous substitutions, we were able to detect statistically significant positive selection at the gene level on three branches. However, we anticipate that the actual duration of the episodes of positive selection in the tree might be larger. The power of the tests employed here is relatively low, i.e. positive selection is difficult to detect even if it exists at many sites <a href="#pone.0001386-Yang1">[7]</a>. Recently, methods have been developed to detect positive selection on individual codons for specific branches (e.g. the branch-site likelihood method <a href="#pone.0001386-Yang3">[23]</a>). These methods are generally more powerful and their utilization has resulted in a marked increase in the number of published reports of positive selection <a href="#pone.0001386-Zhang1">[24]</a>. However, simulations by Zhang <a href="#pone.0001386-Zhang1">[24]</a> showed that the power of these methods comes at a cost in the form of high levels of false positives.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Under the assumption that substitutions in non-coding sequences are selection-wise neutral, elevated substitution rates in exon sequences compared to introns provide an indication of positive selection <a href="#pone.0001386-Johnson1">[25]</a>. By comparing the branch lengths of the exon tree (<a href="#pone-0001386-g004">Fig. 4A</a>) and the intron tree (<a href="#pone-0001386-g004">Fig. 4B</a>) it is apparent that this is the case in <em>S. fruticosa</em> and probably also in <em>Lychnis chalcedonica.</em> For example, the uncorrected pairwise base distance between <em>S. fruticosa</em> (Sf1) and <em>S. schafta</em> is 0.23 in exons, but 0.05 in the introns, and for <em>L. chalcedonica</em> (exon: Lc1, intron: Lc3/Lc4 combined) and <em>L. flos-jovis</em> these figures are 0.30 and 0.04, respectively. Finally, <em>Oenothera flava</em> (the only <em>Oenothera</em> species in this study to have introns) shows more variability in exons than in introns compared to <em>Eucalyptus</em> (0.26 and 0.18, respectively), although the difference is less striking for this taxon.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The <em>clpP1</em> exon sequences that show the most extreme substitution rates are those of <em>Silene conica</em> and <em>S. conoidea</em>, but because they lack introns in the gene, the exon/intron comparison cannot be made. The variability in the gene is approximately an order of magnitude higher (synonymous and non-synonymous substitutions contributing roughly equally to the increase) than that of spacer-regions in the cpDNA of <em>S. conica</em> (see below). The pairwise base difference between <em>Silene conica</em> and <em>Silene latifolia</em> in intergenic spacers is on average 0.03 (<em>rbcL</em>/<em>accD</em>: 0.037, <em>petA</em>/<em>psbJ</em>: 0.035, <em>psbE</em>/<em>petL</em>: 0.028, <em>rpl20</em>/<em>rps12</em>: 0.022, data from <a href="#pone.0001386-Erixon1">[18]</a>), but the difference in the <em>clpP1</em> gene is 0.31. Despite the very divergent sequences, the dN/dS ratios did not indicate positive selection acting within this group. Because ratios around 1 implicate absence of both positive and purifying selection, this indicates that the <em>S. conica</em> and <em>S. conoidea</em> sequences may have lost their function. However, the simulation experiment strongly rejected the hypothesis that the absence of stop codons can be explained by chance alone. Further support for this is given by the fact that <em>Silene conica</em> and <em>S. conoidea</em> have seven indels in the <em>clpP1</em> sequence, all of lengths that are multiples of three. The existence of these indels that do not distort the reading frame is in itself strong evidence for maintained gene function. Finally, even if <em>S. conica</em> appears to have a somewhat elevated cpDNA substitution rate in general <a href="#pone.0001386-Erixon1">[18]</a>, it seems unlikely that the very high substitution rates in <em>clpP1</em> would be the effect of lost function. Xing and Lee <a href="#pone.0001386-Xing1">[26]</a> found that alternative splicing could greatly relax selection pressure (measured as dN/dS). This effect was accompanied by a strong decrease in synonymous substitutions. Because we observe a strong increase in synonymous substitutions in our data (<a href="#pone-0001386-g003">Fig. 3</a>), this explanation too, seems unlikely in this particular case.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Whether there is a causal relationship between the increase in synonymous and non-synonymous rates in <em>Silene conica/conoidea</em> remains unclear. However, there are other indications that positive selection of <em>clpP1</em> is correlated with increased synonymous substitution rates; <em>Lychnis chalcedonica</em> and <em>Oenothera elata</em> also have elevated synonymous substitution rates (<a href="#pone-0001386-g003">Fig. 3</a>). Some of the other branches in the eudicot <em>clpP1</em> tree (<a href="#pone-0001386-g002">Fig. 2</a>) have combinations of branch lengths and dN/dS ratios that indicate a more widespread occurrence of positive selection of the gene. In particular the branch leading to <em>Solanum</em>/<em>Lycopersicon</em> (node 14 in <a href="#pone-0001386-g002">Fig. 2A</a>) that has a dN/dS ratio significantly higher than 1 before Bonferroni-correction (<a href="#pone-0001386-t002">Table 2</a>), but also the branches leading to node 6 (<a href="#pone-0001386-g002">Fig. 2A</a>), <em>Medicago</em>, <em>Vitis</em>, and <em>Cucumis</em> seem interesting targets for further investigations.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In a systematic search for positive selection in higher plants based on almost 140,000 embryophyte gene sequences from GenBank, very few cases of ω values above one were found when averaging over whole genes <a href="#pone.0001386-Roth1">[27]</a>. Only in two cases were ω&gt;2, and both of these were sequence pairs of nuclear encoded genes <a href="#pone.0001386-Roth1">[27]</a>. This illustrates how unusual our findings are. The recent report on positive selection in the chloroplast gene <em>rbcL</em> <a href="#pone.0001386-Kapralov2">[10]</a> clearly shows that specific sites in that gene are under positive selection in a wide range of land plants. Our study gives indications that positive selection in the <em>clpP1</em> gene might be widespread in flowering plants. In <em>rbcL</em> it is only a small proportion of the sites that appear to be under positive selection <a href="#pone.0001386-Kapralov2">[10]</a>, whereas in <em>clpP1</em> a very large proportion of the sites are affected.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>In the present study, the rates of synonymous substitutions are rather conserved with respect to different taxa and genes, with the important exception of the species undergoing rapid evolution in the <em>clpP1</em> gene (<a href="#pone-0001386-g003">Fig. 3</a>). None of the “normal” taxa or genes shows as high dS rates as the <em>clpP1</em> gene from those three species. The degree of elevated dS rates also indicates an interesting pattern: the species with the strongest estimated positive selection has the least elevated dS rate and vice versa, i.e. the rate of non-synonymous substitutions are roughly the same among the three species.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Elevated evolutionary rates due to positive selection or relaxed selective constraints are often preceded by gene duplication <a href="#pone.0001386-Ohta1">[28]</a>. We detected extra <em>clpP1</em> gene copies only in <em>Lychnis chalcedonica</em> and <em>Silene fruticosa</em>. Indeed, the completely sequenced chloroplast genome of <em>Oenothera elata</em> (NC_002693) contains only one copy of <em>clpP1</em>.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>Only one of the four <em>clpP1</em> copies in <em>L. chalcedonica</em> is potentially functional (Lc1), i.e. the others contain stop codons or are incomplete. The intron-containing Lc4 fragment shows obvious signs of elevated substitution rates in <em>clpP1</em>, although less so than Lc1. The Lc3 copy, apparently a pseudogene, is less divergent than the other copies in <em>Lychnis chalcedonica</em>. This does not seem to be an artifact due to missing data, because in the region where sequence information for Lc1, Lc2, and Lc3 overlap the uncorrected distance between Lc1/Lc2 and <em>L. flos-jovis</em> (the “normal” <em>Lychnis</em> species in this study) is 30.3%/34.6%, whereas between Lc3 and <em>L. flos-jovis</em> it is 3.9%. Thus, in absence of a formal phylogenetic analysis of the <em>clpP1</em> copies in <em>L. chalcedonica</em>, we may speculate that at least the duplication leading to Lc3 preceded the onset of positive selection. In <em>S. fruticosa</em>, Sf2 is markedly less divergent than the two other copies, and thus also probably the result of an ancient duplication preceding the non-synonymous rates increase. In the region where sequence information for Sf1 and Sf2 overlaps the uncorrected distance between Sf1 and <em>S. schafta</em> is 22.6%, whereas between Sf2 and <em>S. schafta</em> it is 5.2%. Both these cases may thus agree with one of few documented cases where gene duplication precedes the onset of positive selection <a href="#pone.0001386-Zhang2">[29]</a>. It may be that positive selection, under some circumstances, can be triggered by duplication rather than being an expected outcome of it.</p>
<a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>The very large insertions (174 to 197 amino acids) found in the <em>clpP1</em> exon 1 of <em>Silene fruticosa</em> (Sf1), <em>Lychnis flos-cuculi</em>, and <em>L. abyssinica</em> do not cause frame shifts or stop codons. To our knowledge, the effect of indel evolution has not been studied in relation to positive selection. It is possible that repetitive insertions are beneficial, because given that the repeats are in multiples of three nucleotides, they reduce the probability of stop codons, while possibly fostering new phenotypic variants.</p>

<h4>-Conclusions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In our study, four distantly related taxa or groups of taxa (<em>Oenothera</em>, <em>Silene fruticosa</em>, <em>Silene conica/conoidea</em>, and <em>Lychnis chalcedonica/flos-cuculi/abyssinica</em>) exhibit substitution rates in <em>clpP1</em> exon sequences that are hitherto unprecedented in the chloroplast genome. We conclude that these high evolutionary rates are correlated with positive selection of <em>clpP1</em> in the evolutionary histories of at least three of these four groups. In the case of <em>Lychnis</em>, this was probably preceded by a duplication of a segment including <em>clpP1</em>, <em>psbB</em>, <em>psbT</em>, <em>psbN</em>, and <em>psbH</em>, but only the <em>clpP1</em> gene shows signs of positive selection. We cannot rule out the possibility of gene duplications as a causal agent in the other three cases, because duplicates may either be ambiguous, extinct, or undetected. One of the major aspects of the present results is that they indicate that positive selection may be accompanied by elevated synonymous substitution rates in some cases. If this indeed proves to be the case it will have far-reaching consequences for our ability to detect positive selection. Also, the fact that positive selection appears to have originated in at least three closely related lineages of <em>Sileneae</em> calls for caution when interpreting plastid data at population and phylogenetic levels (cf. <a href="#pone.0001386-Kapralov1">[9]</a>). The relationship between cpDNA duplications, increased substitution rates, positive selection, and indel evolution in the chloroplast genome needs further scrutiny, and plant <em>clpP1</em> appears to constitute an excellent model system for such studies.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>-Taxon sampling</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All seed plant <em>clpP1</em> sequences on GenBank <a href="#pone.0001386-Benson1">[30]</a> as of May 15th 2006, as well as the nuclear <em>clpP2-6</em> from <em>Oryza</em> and <em>Arabidopsis</em>, were downloaded; in addition, 21 species of <em>Sileneae</em> and four species of <em>Oenothera</em> were sequenced for the gene (<a href="#pone.0001386.s002">Table S1</a>). The sampling of <em>Sileneae</em> species follows that of Erixon and Oxelman <a href="#pone.0001386-Erixon1">[18]</a>, but <em>Silene conoidea</em> and <em>Lychnis abyssinica</em> were added after the observation that closely related taxa exhibited extremely high substitution rates. The inclusion of <em>Oenothera</em> in the study was deemed important because a survey of complete chloroplast genomes on GenBank revealed that <em>Oenothera elata</em> (NC_002693) lacks introns and shows signs of elevated substitution rates. The four species of <em>Oenothera</em> were chosen to represent different sections in the genus <a href="#pone.0001386-Levin1">[31]</a>.</p>


<h4>-DNA Amplification and Sequencing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>All 21 <em>Sileneae</em> species (except <em>Silene conoidea</em>, <em>Lychnis flos-cuculi</em>, <em>Lychnis flos-jovis</em>, and <em>Lychnis abyssinica</em>) were amplified and sequenced in a continuous region from the end of <em>rbcL</em> to the beginning of <em>petB</em> corresponding to c. 18 kb in <em>Spinacia</em>. DNA preparation, amplification, sequencing, and general primers follow Erixon and Oxelman <a href="#pone.0001386-Erixon1">[18]</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The <em>clpP1</em> region in <em>Oenothera</em> was amplified with the PCR primers rps12-F and clpP/psbH-R6 (<a href="#pone-0001386-g005">Fig. 5</a>). For sequencing of this product, eight primers were constructed (<a href="#pone.0001386.s003">Table S2</a>). In addition, some specific primers were made for <em>Silene fruticosa</em> and <em>Lychnis chalcedonica</em>, either to amplify a specific sequence copy or to sequence through large insertions (<a href="#pone.0001386.s003">Table S2</a>).</p>
<div class="figure" id="pone-0001386-g005"><div class="img"><a name="pone-0001386-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001386" data-uri="info:doi/10.1371/journal.pone.0001386.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001386.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001386.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001386.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001386.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The <em>clpP1</em> Gene and Flanking Regions with Primer Sites.</span></strong></p><a id="article1.body1.sec4.sec2.fig1.caption1.p1" name="article1.body1.sec4.sec2.fig1.caption1.p1"></a><p>Approximate position of general primers for <em>Sileneae</em> <a href="#pone.0001386-Erixon1">[18]</a> with reference to <em>Spinacia</em>. Boxes correspond to protein coding genes with their name and position in <em>Spinacia</em>. Boxes above the line denote genes that are transcribed from left to right, and those below are transcribed from right to left. Dotted lines represent introns.</p>
<span>doi:10.1371/journal.pone.0001386.g005</span></div>

<h4>-Alignment</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The amino acid sequences from <em>Arabidopsis</em> and <em>Oryza</em> for all five nuclear members of the <em>clpP</em> gene family were aligned using ClustalW version 1.83 <a href="#pone.0001386-Higgins1">[32]</a>, with default settings, together with chloroplast <em>clpP1</em> amino acid sequences. The nuclear sequences corresponding to the 21 first amino acids in the chloroplast <em>clpP1</em> gene of <em>Spinacia</em> were excluded prior the analysis, because the alignment responded strongly even to small changes in the parameters. All sequences corresponding to the six last amino acids of <em>Spinacia clpP1</em> were also excluded, due to extreme length variation both within <em>clpP1</em> and among the other gene family members. The amino acids were only used in the alignment process; all analyses were done on nucleotide sequences.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>All non-<em>clpP1</em> exon DNA sequences were manually aligned, using the principles of Oxelman et al. <a href="#pone.0001386-Oxelman3">[33]</a>, in the sequence alignment editor Se-Al version 2.0a11 (<a href="http://evolve.zoo.ox.ac.uk/">http://evolve.zoo.ox.ac.uk/</a>).</p>


<h4>-Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Bayesian phylogenetic analyses were performed with MrBayes version 3.1.2 <a href="#pone.0001386-Huelsenbeck1">[34]</a>. For both the gene family data set (72 terminals) and a restricted eudicot <em>clpP1</em> data set (38 terminals, see below), substitutions were modeled with the GTR+Γ model, which received highest AIC scores according to MrAIC.pl version 1.4 (<a href="http://www.abc.se/">http://www.abc.se/</a>~nylander/) together with PHYML version 2.4.4 <a href="#pone.0001386-Guindon1">[35]</a>. The large data set ran for 50 million generations with six MCMC chains (temperature (t) = 0.2) and four independent runs with trees sampled every 1000th generation. The eudicot data set ran for 20 million generations with four MCMC chains (t = 0.2) and two independent runs with trees sampled every 100th generation. The first 50% of the sampled trees were, in both analyses, discarded as burn-in.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Bayesian analyses, with the same settings as above except that number of generations were 10 million, were performed on: a) a matrix of only third positions from the Eudicot data set, b) a matrix of <em>Sileneae clpP1</em> intron sequences, and c) a matrix of <em>Sileneae clpP1</em> exon sequences.</p>


<h4>-Detection of positive selection</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>PAML version 3.14 <a href="#pone.0001386-Yang2">[21]</a> was used to calculate the non-synonymous (dN) and synonymous (dS) substitutions rates, and the ratio (ω) between them. To test for variation in ω among the branches, the likelihood for the data under a model with fixed ω (estimated from data) for the entire tree (m0) was compared to a model allowing for the branches to have different ω (m1). To test the null hypothesis that addition of a ω parameter for each branch does not increase likelihood of the data, a likelihood-ratio test was performed, where the test parameter is assumed to follow the chi-square distribution with the degrees of freedom equal the number of tree branches −1. The null hypothesis of absence of selection on individual branches was tested by comparing the maximum likelihoods from the m1 analysis to maximum likelihoods for models with free ω for all branches except that the ω of the branch under consideration was set to 1 (m2). This test has one degree of freedom. Only branches with ω&gt;1.0 were tested, but because these were detected <em>a posteriori</em>, the probabilities were Bonferroni corrected <a href="#pone.0001386-Yang1">[7]</a>.</p>


<h4>-Topologies used for estimates of dN/dS</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Two different tree topologies were used for calculations of dN and dS values. The first topology (<a href="#pone-0001386-g002">Fig. 2A</a>) was based on the classification of the Angiosperm Phylogeny Group II <a href="#pone.0001386-APG1">[36]</a> at the family level and on strongly supported within-family relationships published elsewhere <a href="#pone.0001386-Oxelman1">[17]</a>–<a href="#pone.0001386-Popp1">[20]</a>, <a href="#pone.0001386-Wojciechowski1">[37]</a>, <a href="#pone.0001386-Olmstead1">[38]</a> . The sistergroup relationship between <em>Silene conica</em> and <em>S. conoidea</em> has not been published before, but is based on their very similar morphology, i.e. they belong to the section <em>Conoimorpha</em> Otth, which is characterized by a calyx morphology and a basic chromosome number, both of which are unique in <em>Sileneae</em>. Five <em>Silene</em> sequences were excluded from this analysis; these were identical or almost identical (≤4 bases different) to at least one of the non-excluded sequences. Only sequences with the entire <em>clpP1</em> coding region were included in the analyses detecting for positive selection.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>To explore the effect of the tree topology on the outcome a second substitution rate analysis was performed using the 50% majority-rule consensus phylogeny from the Bayesian analysis of third positions (<a href="#pone-0001386-g002">Fig. 2B</a>). This topology is incompatible with the first topology with respect to the relations within Fabaceae. Another substantial difference is in the level of resolution. Eight more branches are unresolved in the second phylogeny compared to the first and two relationships are resolved only in the second (in <em>Oenothera</em> and in <em>Lychnis</em>, see <a href="#pone-0001386-g002">Fig. 2</a>).</p>


<h4>-Synonymous substitution rate comparison</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>In order to obtain an estimate of the synonymous substitution rates in the <em>clpP1</em> exons in <em>Sileneae</em> we made pairwise comparisons between eleven <em>Silene</em>/<em>Lychnis</em> species and <em>Heliosperma</em> (outgroup). These estimates were compared with data from three other chloroplast genes (<em>psbB</em>, 1527 bp; <em>cemA</em>, 639 bp; <em>petA</em>, 963 bp). The reason for choosing these particular genes was that they are the only large genes present in the 18 kb fragment sequenced for all the twelve taxa <a href="#pone.0001386-Erixon1">[18]</a>, and when Goremykin et al <a href="#pone.0001386-Goremykin1">[39]</a> compared the synonymous substitution rates for these genes of ten angiosperms relative to <em>Pinus</em>, the average dS values were similar and <em>clpP1</em> had the lowest value (<em>clpP1</em>, 0.86; <em>cemA</em>, 0.89; <em>psbB</em>, 1.00; <em>petA</em>, 1.53). We also made a pairwise comparison of dS values for the same genes from the complete chloroplast genomes of <em>Oenothera elata</em> and <em>Eucalyptus globulus</em>. PAML version 3.14 <a href="#pone.0001386-Yang2">[21]</a> was used for all estimates.</p>


<h4>-Indirect test for loss of function</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>In the absence of expression studies, we conducted a simple test to evaluate if the observed open reading frames of sequences with highly elevated rates could have persisted by chance. Ten thousand sequences were generated with SeqGen version 1.3.2 <a href="#pone.0001386-Rambaut1">[40]</a> under the JC69 model of evolution using the <em>Silene latifolia</em> sequence as ancestral and a branch length of 0.25. The branch length was chosen to be considerably smaller than the longest branch observed in the substitution rate analysis, namely the one leading to <em>S. conica</em>/<em>S. conoidea</em>, i.e. 0.47 (<a href="#pone-0001386-g002">Figures 2A</a>). A simple substitution model was chosen to simulate neutral evolution. Since the chloroplast DNA sequences have an AT-bias, the JC69 model will result in a conservative test, because an excess of A and T will increase the number of simulated stop codons simply because these are AT-rich (TAA, TAG, and TGA).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001386"></div><a name="pone.0001386.s001" id="pone.0001386.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001386.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><em>clpP</em> gene family phylogeny. A) Bayesian consensus phylogram of all seed plant <em>clpP1</em> sequences and the nuclear <em>clpP2-clpP6</em> for <em>Arabidopsis</em> and <em>Oryza</em>, and B) without <em>Oenothera</em> and the long <em>Sileneae</em> branches. Numbers on nodes are Bayesian posterior probabilities (Bpp). Branches in bold have Bpp = 1.00. Only Bpp&gt;0.90 are shown. Each * indicates a missing intron. Y indicates pseudogene. ? indicates incomplete gene (only exon 1) without stop codon.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001386.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(8.35 MB TIF)</p>
<a name="pone.0001386.s002" id="pone.0001386.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001386.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Plant taxa, vouchers for the sequences obtained in this study, and GenBank accession numbers</p>
<p class="siDoi">doi:10.1371/journal.pone.0001386.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001386.s003" id="pone.0001386.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001386.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Specific primers</p>
<p class="siDoi">doi:10.1371/journal.pone.0001386.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Katarina Andreasen, Adrian Clarke, Martin Lascoux, Henrik Nilsson, Mats Thulin, Mikael Thollesson, Niklas Wikström, and an anonymous reviewer for valuable comments on an earlier draft of this manuscript. Nahid Heidari is gratefully acknowledged for laboratory support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PE BO. Performed the experiments: PE. Analyzed the data: PE. Contributed reagents/materials/analysis tools: PE BO. Wrote the paper: PE BO.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001386-Raubeson1" id="pone.0001386-Raubeson1"></a>Raubeson LA, Jansen RK (2005)  Chloroplast genomes of plants. In: Henry RJ, editor. Plant diversity and evolution: Genotypic and phenotypic variation in higher plants. Wallingford: CABI Publishing. pp. 45–68.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001386-Wolfe1" id="pone.0001386-Wolfe1"></a>Wolfe KH, Li WH, Sharp PM (1987) Rates of Nucleotide Substitution Vary Greatly among Plant Mitochondrial, Chloroplast, and Nuclear DNAs. Proc Natl Acad Sci U S A  84: 9054–9058.  <ul class="find" data-citedArticleID="1037060" data-doi="10.1073/pnas.84.24.9054"><li><a href="http://dx.doi.org/10.1073/pnas.84.24.9054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+Nucleotide+Substitution+Vary+Greatly+among+Plant+Mitochondrial%2C+Chloroplast%2C+and+Nuclear+DNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+Nucleotide+Substitution+Vary+Greatly+among+Plant+Mitochondrial%2C+Chloroplast%2C+and+Nuclear+DNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001386-Clegg1" id="pone.0001386-Clegg1"></a>Clegg MT, Gaut BS, Learn GH, Morton BR (1994) Rates and patterns of chloroplast DNA evolution. Proc Natl Acad Sci U S A  91: 6795–6801.  <ul class="find" data-citedArticleID="1037006" data-doi="10.1073/pnas.91.15.6795"><li><a href="http://dx.doi.org/10.1073/pnas.91.15.6795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+and+patterns+of+chloroplast+DNA+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+and+patterns+of+chloroplast+DNA+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001386-Muse1" id="pone.0001386-Muse1"></a>Muse SV, Gaut BS (1997) Comparing patterns of nucleotide substitution rates among chloroplast loci using the relative ratio test. Genetics  146: 393–399.  <ul class="find" data-citedArticleID="1037032"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparing+patterns+of+nucleotide+substitution+rates+among+chloroplast+loci+using+the+relative+ratio+test.&amp;auth=&amp;atitle=Comparing+patterns+of+nucleotide+substitution+rates+among+chloroplast+loci+using+the+relative+ratio+test." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparing+patterns+of+nucleotide+substitution+rates+among+chloroplast+loci+using+the+relative+ratio+test." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparing+patterns+of+nucleotide+substitution+rates+among+chloroplast+loci+using+the+relative+ratio+test.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001386-Shaw1" id="pone.0001386-Shaw1"></a>Shaw J, Lickey EB, Beck JT, Farmer SB, Liu W, et al.  (2005) The Tortoise and the Hare II: Relative Utility of 21 Noncoding Chloroplast DNA Sequences for Phylogenetic Analysis. Am J Bot  92: 142–166.  <ul class="find" data-citedArticleID="1037052" data-doi="10.3732/ajb.92.1.142"><li><a href="http://dx.doi.org/10.3732/ajb.92.1.142" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Tortoise+and+the+Hare+II%3A+Relative+Utility+of+21+Noncoding+Chloroplast+DNA+Sequences+for+Phylogenetic+Analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Tortoise+and+the+Hare+II%3A+Relative+Utility+of+21+Noncoding+Chloroplast+DNA+Sequences+for+Phylogenetic+Analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001386-Timmis1" id="pone.0001386-Timmis1"></a>Timmis JN, Ayliffe MA, Huang CY, Martin W (2004) Endosymbiotic gene transfer: Organelle genomes forge eukaryotic chromosomes. Nature Rev Genet  5: 123–135.  <ul class="find" data-citedArticleID="1037056" data-doi="10.1038/nrg1271"><li><a href="http://dx.doi.org/10.1038/nrg1271" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endosymbiotic+gene+transfer%3A+Organelle+genomes+forge+eukaryotic+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endosymbiotic+gene+transfer%3A+Organelle+genomes+forge+eukaryotic+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001386-Yang1" id="pone.0001386-Yang1"></a>Yang ZH, Bielawski JP (2000) Statistical methods for detecting molecular adaptation. Trends Ecol Evolut  15: 496–503.  <ul class="find" data-citedArticleID="1037064" data-doi="10.1016/s0169-5347(00)01994-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)01994-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+methods+for+detecting+molecular+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+methods+for+detecting+molecular+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001386-MacCallum1" id="pone.0001386-MacCallum1"></a>MacCallum C, Hill E (2006) Being Positive about Selection. PLoS Biology  4: e87.  <ul class="find" data-citedArticleID="1037030" data-doi="10.1371/journal.pbio.0040087"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040087" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Being+Positive+about+Selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Being+Positive+about+Selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001386-Kapralov1" id="pone.0001386-Kapralov1"></a>Kapralov MV, Filatov DA (2006) Molecular Adaption during Adaptive Radiation in the Hawaiian Endemic Genus Schiedea. PLoS ONE  1(1): e8.  <ul class="find" data-citedArticleID="1037022" data-doi="10.1371/journal.pone.0000008"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+Adaption+during+Adaptive+Radiation+in+the+Hawaiian+Endemic+Genus+Schiedea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+Adaption+during+Adaptive+Radiation+in+the+Hawaiian+Endemic+Genus+Schiedea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001386-Kapralov2" id="pone.0001386-Kapralov2"></a>Kapralov MV, Filatov DA (2007) Widespread positive selection in the photosynthetic Rubisco enzyme. BMC Evolutionary Biology  7: 73.  <ul class="find" data-citedArticleID="1037024" data-doi="10.1186/1471-2148-7-73"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-73" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+positive+selection+in+the+photosynthetic+Rubisco+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+positive+selection+in+the+photosynthetic+Rubisco+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001386-Endo1" id="pone.0001386-Endo1"></a>Endo T, Ikeo K, Gojobori T (1996) Large-Scale Search for Genes on Which Positive Selection May Operate. Mol Biol Evol  13: 685–690.  <ul class="find" data-citedArticleID="1037008" data-doi="10.1093/oxfordjournals.molbev.a025629"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025629" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-Scale+Search+for+Genes+on+Which+Positive+Selection+May+Operate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-Scale+Search+for+Genes+on+Which+Positive+Selection+May+Operate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001386-Adam1" id="pone.0001386-Adam1"></a>Adam Z, Adamska I, Nakabayashi K, Ostersetzer O, Haussuhl K, et al.  (2001) Chloroplast and Mitochondrial Proteases in <em>Arabidopsis</em>. A Proposed Nomenclature. Plant Physiol  125: 1912–1918.  <ul class="find" data-citedArticleID="1036998" data-doi="10.1104/pp.125.4.1912"><li><a href="http://dx.doi.org/10.1104/pp.125.4.1912" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chloroplast+and+Mitochondrial+Proteases+in+Arabidopsis.+A+Proposed+Nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chloroplast+and+Mitochondrial+Proteases+in+Arabidopsis.+A+Proposed+Nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001386-Clarke1" id="pone.0001386-Clarke1"></a>Clarke AK (1999) ATP-dependent Clp Proteases in Photosynthetic Organisms-A Cut Above the Rest! Ann Botany  83: 593–599.  <ul class="find" data-citedArticleID="1037004"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=ATP-dependent+Clp+Proteases+in+Photosynthetic+Organisms-A+Cut+Above+the+Rest%21&amp;auth=&amp;atitle=ATP-dependent+Clp+Proteases+in+Photosynthetic+Organisms-A+Cut+Above+the+Rest%21" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATP-dependent+Clp+Proteases+in+Photosynthetic+Organisms-A+Cut+Above+the+Rest%21" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATP-dependent+Clp+Proteases+in+Photosynthetic+Organisms-A+Cut+Above+the+Rest%21%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001386-Kuroda1" id="pone.0001386-Kuroda1"></a>Kuroda H, Maliga P (2003) The plastid <em>clpP1</em> protease gene is essential for plant development. Nature  425: 86–89.  <ul class="find" data-citedArticleID="1037026" data-doi="10.1038/nature01909"><li><a href="http://dx.doi.org/10.1038/nature01909" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+plastid+clpP1+protease+gene+is+essential+for+plant+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+plastid+clpP1+protease+gene+is+essential+for+plant+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001386-Shikanai1" id="pone.0001386-Shikanai1"></a>Shikanai T, Shimizu K, Ueda K, Nishimura Y, Kuroiwa T, et al.  (2001) The chloroplast <em>clpP</em> gene, encoding a proteolytic subunit of ATP-dependent protease, is indispensable for chloroplast development in tobacco. Plant Cell Physiol  42: 264–273.  <ul class="find" data-citedArticleID="1037054" data-doi="10.1093/pcp/pce031"><li><a href="http://dx.doi.org/10.1093/pcp/pce031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chloroplast+clpP+gene%2C+encoding+a+proteolytic+subunit+of+ATP-dependent+protease%2C+is+indispensable+for+chloroplast+development+in+tobacco." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chloroplast+clpP+gene%2C+encoding+a+proteolytic+subunit+of+ATP-dependent+protease%2C+is+indispensable+for+chloroplast+development+in+tobacco.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001386-deCambiaire1" id="pone.0001386-deCambiaire1"></a>deCambiaire J-C, Otis C, Lemieux C, Turmel M (2006) The complete chloroplast genome sequence of the chlorophycean green alga <em>Scenedesmus obliquus</em> reveals a compact gene organization and a biased distribution of genes on the two DNA strands. BMC Evol Biol  6: 37.  <ul class="find" data-citedArticleID="1037074"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+complete+chloroplast+genome+sequence+of+the+chlorophycean+green+alga+Scenedesmus+obliquus+reveals+a+compact+gene+organization+and+a+biased+distribution+of+genes+on+the+two+DNA+strands.&amp;auth=&amp;atitle=The+complete+chloroplast+genome+sequence+of+the+chlorophycean+green+alga+Scenedesmus+obliquus+reveals+a+compact+gene+organization+and+a+biased+distribution+of+genes+on+the+two+DNA+strands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+chloroplast+genome+sequence+of+the+chlorophycean+green+alga+Scenedesmus+obliquus+reveals+a+compact+gene+organization+and+a+biased+distribution+of+genes+on+the+two+DNA+strands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+chloroplast+genome+sequence+of+the+chlorophycean+green+alga+Scenedesmus+obliquus+reveals+a+compact+gene+organization+and+a+biased+distribution+of+genes+on+the+two+DNA+strands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001386-Oxelman1" id="pone.0001386-Oxelman1"></a>Oxelman B, Lidén M, Rabeler RK, Popp M (2001) A revised generic classification of the tribe <em>Sileneae</em> (Caryophyllaceae). Nord J Bot  20: 743–748.  <ul class="find" data-citedArticleID="1037038" data-doi="10.1111/j.1756-1051.2000.tb00760.x"><li><a href="http://dx.doi.org/10.1111/j.1756-1051.2000.tb00760.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+revised+generic+classification+of+the+tribe+Sileneae+%28Caryophyllaceae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+revised+generic+classification+of+the+tribe+Sileneae+%28Caryophyllaceae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001386-Erixon1" id="pone.0001386-Erixon1"></a>Erixon P, Oxelman B (2006) Reticulate or treelike chloroplast DNA evolution in <em>Sileneae</em> (Caryophyllaceae)? Erixon P. Phylogenetic Support and Chloroplast Genome Evolution in <em>Sileneae</em> (Caryophyllaceae). Uppsala University.   PhD Thesis.  <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0001386-Oxelman2" id="pone.0001386-Oxelman2"></a>Oxelman B, Lidén M (1995) Generic Boundaries in the Tribe <em>Sileneae</em> (Caryophyllaceae) as Inferred from Nuclear rDNA Sequences. Taxon  44: 525–542.  <ul class="find" data-citedArticleID="1037040" data-doi="10.2307/1223498"><li><a href="http://dx.doi.org/10.2307/1223498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generic+Boundaries+in+the+Tribe+Sileneae+%28Caryophyllaceae%29+as+Inferred+from+Nuclear+rDNA+Sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generic+Boundaries+in+the+Tribe+Sileneae+%28Caryophyllaceae%29+as+Inferred+from+Nuclear+rDNA+Sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001386-Popp1" id="pone.0001386-Popp1"></a>Popp M, Oxelman B (2004) Evolution of a RNA Polymerase Gene Family in <em>Silene</em> (Caryophyllaceae)-Incomplete Concerted Evolution and Topological Congruence Among Paralogues. Syst Biol  53: 914–932.  <ul class="find" data-citedArticleID="1037044"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+a+RNA+Polymerase+Gene+Family+in+Silene+%28Caryophyllaceae%29-Incomplete+Concerted+Evolution+and+Topological+Congruence+Among+Paralogues.&amp;auth=&amp;atitle=Evolution+of+a+RNA+Polymerase+Gene+Family+in+Silene+%28Caryophyllaceae%29-Incomplete+Concerted+Evolution+and+Topological+Congruence+Among+Paralogues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+a+RNA+Polymerase+Gene+Family+in+Silene+%28Caryophyllaceae%29-Incomplete+Concerted+Evolution+and+Topological+Congruence+Among+Paralogues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+a+RNA+Polymerase+Gene+Family+in+Silene+%28Caryophyllaceae%29-Incomplete+Concerted+Evolution+and+Topological+Congruence+Among+Paralogues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001386-Yang2" id="pone.0001386-Yang2"></a>Yang ZH (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="1037066" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001386-Cho1" id="pone.0001386-Cho1"></a>Cho Y, Mower JP, Qiu Y-L, Palmer JD (2004) Mitochondrial substitution rates are extraordinarily elevated and variable in a genus of flowering plants. Proc Natl Acad Sci U S A  101: 17741–17746.  <ul class="find" data-citedArticleID="1037002" data-doi="10.1073/pnas.0408302101"><li><a href="http://dx.doi.org/10.1073/pnas.0408302101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+substitution+rates+are+extraordinarily+elevated+and+variable+in+a+genus+of+flowering+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+substitution+rates+are+extraordinarily+elevated+and+variable+in+a+genus+of+flowering+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001386-Yang3" id="pone.0001386-Yang3"></a>Yang ZH, Nielsen R (2002) Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages. Mol Biol Evol  19: 908–917.  <ul class="find" data-citedArticleID="1037068" data-doi="10.1093/oxfordjournals.molbev.a004148"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001386-Zhang1" id="pone.0001386-Zhang1"></a>Zhang JZ (2004) Frequent false detection of positive selection by the likelihood method with branch-site models. Mol Biol Evol  21: 1332–1339.  <ul class="find" data-citedArticleID="1037070" data-doi="10.1093/molbev/msh117"><li><a href="http://dx.doi.org/10.1093/molbev/msh117" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+false+detection+of+positive+selection+by+the+likelihood+method+with+branch-site+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+false+detection+of+positive+selection+by+the+likelihood+method+with+branch-site+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001386-Johnson1" id="pone.0001386-Johnson1"></a>Johnson ME, Viggiano L, Bailey JA, Abdul-Rauf M, Goodwin G, et al.  (2001) Positive selection of a gene family during the emergence of humans and African apes. Nature  413: 514–519.  <ul class="find" data-citedArticleID="1037020" data-doi="10.1038/35097067"><li><a href="http://dx.doi.org/10.1038/35097067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+selection+of+a+gene+family+during+the+emergence+of+humans+and+African+apes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+selection+of+a+gene+family+during+the+emergence+of+humans+and+African+apes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001386-Xing1" id="pone.0001386-Xing1"></a>Xing Y, Lee C (2005) Evidence of functional selection pressure for alternative splicing events that accelerate evolution of protein subsequences. Proc Natl Acad Sci U S A  102: 13526–13531.  <ul class="find" data-citedArticleID="1037062" data-doi="10.1073/pnas.0501213102"><li><a href="http://dx.doi.org/10.1073/pnas.0501213102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+functional+selection+pressure+for+alternative+splicing+events+that+accelerate+evolution+of+protein+subsequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+functional+selection+pressure+for+alternative+splicing+events+that+accelerate+evolution+of+protein+subsequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001386-Roth1" id="pone.0001386-Roth1"></a>Roth C, Liberles DA (2006) A systematic search for positive selection in higher plants (Embryophytes). BMC Plant Biology  6: 12.  <ul class="find" data-citedArticleID="1037050"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+systematic+search+for+positive+selection+in+higher+plants+%28Embryophytes%29.&amp;auth=&amp;atitle=A+systematic+search+for+positive+selection+in+higher+plants+%28Embryophytes%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systematic+search+for+positive+selection+in+higher+plants+%28Embryophytes%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systematic+search+for+positive+selection+in+higher+plants+%28Embryophytes%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001386-Ohta1" id="pone.0001386-Ohta1"></a>Ohta T (1993) Pattern of Nucleotide Substitutions in Growth Hormone-Prolactin Gene Family: A Paradigm for Evolution by Gene Duplication. Genetics  134: 1271–1276.  <ul class="find" data-citedArticleID="1037034"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pattern+of+Nucleotide+Substitutions+in+Growth+Hormone-Prolactin+Gene+Family%3A+A+Paradigm+for+Evolution+by+Gene+Duplication.&amp;auth=&amp;atitle=Pattern+of+Nucleotide+Substitutions+in+Growth+Hormone-Prolactin+Gene+Family%3A+A+Paradigm+for+Evolution+by+Gene+Duplication." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pattern+of+Nucleotide+Substitutions+in+Growth+Hormone-Prolactin+Gene+Family%3A+A+Paradigm+for+Evolution+by+Gene+Duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pattern+of+Nucleotide+Substitutions+in+Growth+Hormone-Prolactin+Gene+Family%3A+A+Paradigm+for+Evolution+by+Gene+Duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001386-Zhang2" id="pone.0001386-Zhang2"></a>Zhang Z, Rosenberg HF, Nei M (1998) Positive Darwinian selection after gene duplication in primate ribonuclease genes. Proc Natl Acad Sci U S A  95: 3708–3713.  <ul class="find" data-citedArticleID="1037072" data-doi="10.1073/pnas.95.7.3708"><li><a href="http://dx.doi.org/10.1073/pnas.95.7.3708" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+Darwinian+selection+after+gene+duplication+in+primate+ribonuclease+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+Darwinian+selection+after+gene+duplication+in+primate+ribonuclease+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001386-Benson1" id="pone.0001386-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2007) GenBank. Nucleic Acids Res  35: D21–25.  <ul class="find" data-citedArticleID="1037000" data-doi="10.1093/nar/gkl986"><li><a href="http://dx.doi.org/10.1093/nar/gkl986" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenBank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenBank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001386-Levin1" id="pone.0001386-Levin1"></a>Levin RA, Wagner WL, Hoch PC, Hahn WJ, Rodriguez A, et al.  (2004) Paraphyly in tribe Onagreae: Insights into phylogenetic relationships of Onagraceae based on nuclear and chloroplast sequence data. Syst Bot  29: 147–164.  <ul class="find" data-citedArticleID="1037028" data-doi="10.1600/036364404772974293"><li><a href="http://dx.doi.org/10.1600/036364404772974293" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paraphyly+in+tribe+Onagreae%3A+Insights+into+phylogenetic+relationships+of+Onagraceae+based+on+nuclear+and+chloroplast+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paraphyly+in+tribe+Onagreae%3A+Insights+into+phylogenetic+relationships+of+Onagraceae+based+on+nuclear+and+chloroplast+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001386-Higgins1" id="pone.0001386-Higgins1"></a>Higgins D, Thompson J, Gibson T, Thompson JD, Higgins DG, et al.  (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1037016" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001386-Oxelman3" id="pone.0001386-Oxelman3"></a>Oxelman B, Lidén M, Berglund D (1997) Chloroplast <em>rps16</em> intron phylogeny of the tribe <em>Sileneae</em> (Caryophyllaceae). Plant Syst Evol  206: 393–410.  <ul class="find" data-citedArticleID="1037042" data-doi="10.1007/bf00987959"><li><a href="http://dx.doi.org/10.1007/bf00987959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chloroplast+rps16+intron+phylogeny+of+the+tribe+Sileneae+%28Caryophyllaceae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chloroplast+rps16+intron+phylogeny+of+the+tribe+Sileneae+%28Caryophyllaceae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001386-Huelsenbeck1" id="pone.0001386-Huelsenbeck1"></a>Huelsenbeck JP, Ronquist F (2001) MrBayes: Bayesian inference of phylogeny. Bioinformatics  17: 754–755.  <ul class="find" data-citedArticleID="1037018" data-doi="10.1093/bioinformatics/17.8.754"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes%3A+Bayesian+inference+of+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes%3A+Bayesian+inference+of+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001386-Guindon1" id="pone.0001386-Guindon1"></a>Guindon S, Gascuel O (2003) A Simple, Fast, and Accurate Algorithm to Estimate Large Phylogenies by Maximum Likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="1037014"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+Simple%2C+Fast%2C+and+Accurate+Algorithm+to+Estimate+Large+Phylogenies+by+Maximum+Likelihood.&amp;auth=&amp;atitle=A+Simple%2C+Fast%2C+and+Accurate+Algorithm+to+Estimate+Large+Phylogenies+by+Maximum+Likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Simple%2C+Fast%2C+and+Accurate+Algorithm+to+Estimate+Large+Phylogenies+by+Maximum+Likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Simple%2C+Fast%2C+and+Accurate+Algorithm+to+Estimate+Large+Phylogenies+by+Maximum+Likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001386-APG1" id="pone.0001386-APG1"></a>APG II (2003) An update of the Angiosperm Phylogeny Group classification for the orders and families of flowering plants: APG II. Bot J Linnean Soc  141: 399–436.  <ul class="find" data-citedArticleID="1036996" data-doi="10.1046/j.1095-8339.2003.t01-1-00158.x"><li><a href="http://dx.doi.org/10.1046/j.1095-8339.2003.t01-1-00158.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+update+of+the+Angiosperm+Phylogeny+Group+classification+for+the+orders+and+families+of+flowering+plants%3A+APG+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+update+of+the+Angiosperm+Phylogeny+Group+classification+for+the+orders+and+families+of+flowering+plants%3A+APG+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001386-Wojciechowski1" id="pone.0001386-Wojciechowski1"></a>Wojciechowski MF, Lavin M, Sanderson MJ (2004) A phylogeny of Legumes (Leguminosae) based on analysis of the plastid <em>matK</em> gene resolves many well-supported subclades within the family. Am J Bot  91: 1846–1862.  <ul class="find" data-citedArticleID="1037058" data-doi="10.3732/ajb.91.11.1846"><li><a href="http://dx.doi.org/10.3732/ajb.91.11.1846" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogeny+of+Legumes+%28Leguminosae%29+based+on+analysis+of+the+plastid+matK+gene+resolves+many+well-supported+subclades+within+the+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogeny+of+Legumes+%28Leguminosae%29+based+on+analysis+of+the+plastid+matK+gene+resolves+many+well-supported+subclades+within+the+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001386-Olmstead1" id="pone.0001386-Olmstead1"></a>Olmstead RG, Sweere JA, Spangler RE, Bohs L, Palmer JD (1999)  Phylogeny and provisional classification of Solanaceae based on chloroplast DNA. In: Nee M, Symon DE, Lester RN, Jessop JP, editors. Solanaceae IV. Kew: Royal Botanical Gardens. pp. 111–137.  <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001386-Goremykin1" id="pone.0001386-Goremykin1"></a>Goremykin VV, Hirsch-Ernst KI, Wölfl S, Hellwig FH (2003) Analysis of the <em>Amborella trichopoda</em> Chloroplast Genome Sequence Suggests That <em>Amborella</em> Is Not a Basal Angiosperm. Mol Bio Evol  20: 1499–1505.  <ul class="find" data-citedArticleID="1037012" data-doi="10.1093/molbev/msg159"><li><a href="http://dx.doi.org/10.1093/molbev/msg159" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+Amborella+trichopoda+Chloroplast+Genome+Sequence+Suggests+That+Amborella+Is+Not+a+Basal+Angiosperm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+Amborella+trichopoda+Chloroplast+Genome+Sequence+Suggests+That+Amborella+Is+Not+a+Basal+Angiosperm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001386-Rambaut1" id="pone.0001386-Rambaut1"></a>Rambaut A, Grassly NC (1997) Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees. Comput Appl Biosci  13: 235–238.  <ul class="find" data-citedArticleID="1037046" data-doi="10.1093/bioinformatics/13.3.235"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.3.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seq-Gen%3A+an+application+for+the+Monte+Carlo+simulation+of+DNA+sequence+evolution+along+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seq-Gen%3A+an+application+for+the+Monte+Carlo+simulation+of+DNA+sequence+evolution+along+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.XML" value="110451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.PDF" value="448453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g001.PNG_L" value="96971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g001.PNG_M" value="19196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g001.PNG_S" value="5830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g001.TIF" value="152140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g001.PNG_I" value="6908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t001.PNG_L" value="364936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t001.PNG_M" value="189711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t001.PNG_S" value="19059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t001.TIF" value="1620668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t001.PNG_I" value="62136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g002.PNG_L" value="376000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g002.PNG_M" value="53357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g002.PNG_S" value="8639"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g002.TIF" value="527440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g002.PNG_I" value="32664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g003.PNG_L" value="241314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g003.PNG_M" value="31990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g003.PNG_S" value="7673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g003.TIF" value="430200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g003.PNG_I" value="15361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g004.PNG_L" value="136420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g004.PNG_M" value="42528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g004.PNG_S" value="6344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g004.TIF" value="218942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g004.PNG_I" value="14379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t002.PNG_L" value="154401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t002.PNG_M" value="86164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t002.PNG_S" value="11557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t002.TIF" value="748620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.t002.PNG_I" value="25624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g005.PNG_L" value="53543"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g005.PNG_M" value="23491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g005.PNG_S" value="4710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g005.TIF" value="87160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.g005.PNG_I" value="8490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.s001.TIF" value="8338532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.s002.DOC" value="41472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001386.s003.DOC" value="24064"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001386&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001386" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001386&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001386&volume=&issue=&title=Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast clpP1 Gene&author_name=Per%20Erixon%2C%20Bengt%20Oxelman&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001386" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001386" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001386" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001386&amp;t=Whole-Gene%20Positive%20Selection%2C%20Elevated%20Synonymous%20Substitution%20Rates%2C%20Duplication%2C%20and%20Indel%20Evolution%20of%20the%20Chloroplast%20clpP1%20Gene" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001386&title=Whole-Gene%20Positive%20Selection%2C%20Elevated%20Synonymous%20Substitution%20Rates%2C%20Duplication%2C%20and%20Indel%20Evolution%20of%20the%20Chloroplast%20clpP1%20Gene&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001386&amp;title=Whole-Gene%20Positive%20Selection%2C%20Elevated%20Synonymous%20Substitution%20Rates%2C%20Duplication%2C%20and%20Indel%20Evolution%20of%20the%20Chloroplast%20clpP1%20Gene" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001386" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001386&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Whole-Gene Positive Selection, Elevated Synonymous Substitution Rates, Duplication, and Indel Evolution of the Chloroplast clpP1 Gene';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001386';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Whole-Gene%20Positive%20Selection%2C%20Elevated%20Synonymous%20Substitution%20Rates%2C%20Duplication%2C%20and%20Indel%20Evolution%20of%20the%20Chloroplast%20clpP1%20Gene http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001386" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001386" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chloroplasts%22" title="Search for articles in the subject area:'Chloroplasts'"><div class="flagText">Chloroplasts</div></a>
              <div data-categoryid="32715" data-articleid="25602"
                   data-categoryname="Chloroplasts"
                   class="flagImage" title="Flag 'Chloroplasts' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25602"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="32847" data-articleid="25602"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="17653" data-articleid="25602"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25602"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46091" data-articleid="25602"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="25602"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Silene%22" title="Search for articles in the subject area:'Silene'"><div class="flagText">Silene</div></a>
              <div data-categoryid="43617" data-articleid="25602"
                   data-categoryname="Silene"
                   class="flagImage" title="Flag 'Silene' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4071'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7595'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6735&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
