

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000436"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000436" />

    <meta name="citation_title" content="A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia"/>
    <meta itemprop="name" content="A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia"/>

      <meta name="citation_author" content="Bora E. Baysal"/>
            <meta name="citation_author_institution" content="Department of Obstetrics, Gynecology and Reproductive Sciences, University of Pittsburgh School of Medicine, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America"/>
            <meta name="citation_author_institution" content="Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2007/5/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000436.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e436"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_number=1; citation_date=2006; citation_publisher=John Wiley; " />
      <meta name="citation_reference" content="citation_title=Mutations in SDHD, a mitochondrial complex II gene, in hereditary paraganglioma.; citation_author=BE Baysal; citation_author=RE Ferrell; citation_author=JE Wilett-Brozick; citation_author=EC Lawrence; citation_author=D Myssiorek; citation_journal_title=Science; citation_volume=287; citation_number=2; citation_pages=848-851; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The SDH mutation database: an online resource for succinate dehydrogenase sequence variants involved in pheochromocytoma, paraganglioma and mitochondrial complex II deficiency.; citation_author=JP Bayley; citation_author=P Devilee; citation_author=PE Taschner; citation_journal_title=BMC Med Genet; citation_volume=6; citation_number=3; citation_pages=39; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A HIF1alpha regulatory loop links hypoxia and mitochondrial signals in pheochromocytomas.; citation_author=PL Dahia; citation_author=KN Ross; citation_author=ME Wright; citation_author=CY Hayashida; citation_author=S Santagata; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=4; citation_pages=72-80; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Altitude is a phenotypic modifier in hereditary paraganglioma type 1: evidence for an oxygen-sensing defect.; citation_author=K Astrom; citation_author=JE Cohen; citation_author=JE Willett-Brozick; citation_author=CE Aston; citation_author=BE Baysal; citation_journal_title=Hum Genet; citation_volume=113; citation_number=5; citation_pages=228-237; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The mitochondrial SDHD gene is required for early embryogenesis, and its partial deficiency results in persistent carotid body glomus cell activation with full responsiveness to hypoxia.; citation_author=JI Piruat; citation_author=CO Pintado; citation_author=P Ortega-Saenz; citation_author=M Roche; citation_author=J Lopez-Barneo; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=6; citation_pages=10933-10940; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mutation of succinate dehydrogenase subunit C results in increased O2.-, oxidative stress, and genomic instability.; citation_author=BG Slane; citation_author=N Aykin-Burns; citation_author=BJ Smith; citation_author=AL Kalen; citation_author=PC Goswami; citation_journal_title=Cancer Res; citation_volume=66; citation_number=7; citation_pages=7615-7620; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial tumour suppressors: a genetic and biochemical update.; citation_author=E Gottlieb; citation_author=IP Tomlinson; citation_journal_title=Nat Rev Cancer; citation_volume=5; citation_number=8; citation_pages=857-866; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Krebs cycle enzymes as tumor suppressors.; citation_author=BE Baysal; citation_journal_title=Drug Discovery Today: Disease Mechanisms; citation_volume=2; citation_number=9; citation_pages=247-254; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Isolation and Immortalization of Lymphocytes.; citation_author=PD Ling; citation_author=HM Huls; citation_number=10; citation_pages=28.2.2-28.2.5; citation_date=2005; citation_publisher=John Wiley & Sons; " />
      <meta name="citation_reference" content="citation_title=Somatic mutations and intraclonal variations in the rearranged Vkappa genes of B-non-Hodgkin's lymphoma cell lines.; citation_author=C Gabay; citation_author=H Ben Bassat; citation_author=M Schlesinger; citation_author=R Laskov; citation_journal_title=Eur J Haematol; citation_volume=63; citation_number=11; citation_pages=180-191; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Prevalence of SDHB, SDHC, and SDHD germline mutations in clinic patients with head and neck paragangliomas.; citation_author=BE Baysal; citation_author=JE Willett-Brozick; citation_author=EC Lawrence; citation_author=CM Drovdlic; citation_author=SA Savul; citation_journal_title=J Med Genet; citation_volume=39; citation_number=12; citation_pages=178-183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hot-stop PCR: a simple and general assay for linear quantitation of allele ratios.; citation_author=H Uejima; citation_author=MP Lee; citation_author=H Cui; citation_author=AP Feinberg; citation_journal_title=Nat Genet; citation_volume=25; citation_number=13; citation_pages=375-376; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli DNA polymerase III epsilon subunit increases Moloney murine leukemia virus reverse transcriptase fidelity and accuracy of RT-PCR procedures.; citation_author=B Arezi; citation_author=HH Hogrefe; citation_journal_title=Anal Biochem; citation_volume=360; citation_number=14; citation_pages=84-91; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=In vivo somatic mutations in humans: measurement and analysis.; citation_author=RJ Albertini; citation_author=JA Nicklas; citation_author=JP O'Neill; citation_author=SH Robison; citation_journal_title=Annu Rev Genet; citation_volume=24; citation_number=15; citation_pages=305-326; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=C→U editing of neurofibromatosis 1 mRNA occurs in tumors that express both the type II transcript and apobec-1, the catalytic subunit of the apolipoprotein B mRNA-editing enzyme.; citation_author=D Mukhopadhyay; citation_author=S Anant; citation_author=RM Lee; citation_author=S Kennedy; citation_author=D Viskochil; citation_journal_title=Am J Hum Genet; citation_volume=70; citation_number=16; citation_pages=38-50; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanism of class switch recombination: linkage with somatic hypermutation.; citation_author=T Honjo; citation_author=K Kinoshita; citation_author=M Muramatsu; citation_journal_title=Annu Rev Immunol; citation_volume=20; citation_number=17; citation_pages=165-196; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Messenger RNA editing in mammals: new members of the APOBEC family seeking roles in the family business.; citation_author=JE Wedekind; citation_author=GS Dance; citation_author=MP Sowden; citation_author=HC Smith; citation_journal_title=Trends Genet; citation_volume=19; citation_number=18; citation_pages=207-216; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Somatic hypermutation: activation-induced deaminase for C/G followed by polymerase eta for A/T.; citation_author=MS Neuberger; citation_author=C Rada; citation_journal_title=J Exp Med; citation_volume=204; citation_number=19; citation_pages=7-10; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Early nonsense: mRNA decay solves a translational problem.; citation_author=N Amrani; citation_author=MS Sachs; citation_author=A Jacobson; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=7; citation_number=20; citation_pages=415-425; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence variation in human succinate dehydrogenase genes: evidence for long-term balancing selection on SDHA.; citation_author=BE Baysal; citation_author=EC Lawrence; citation_author=RE Ferrell; citation_journal_title=BMC Biol; citation_volume=5; citation_number=21; citation_pages=12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mechanisms regulating the recruitment of macrophages into hypoxic areas of tumors and other ischemic tissues.; citation_author=C Murdoch; citation_author=A Giannoudis; citation_author=CE Lewis; citation_journal_title=Blood; citation_volume=104; citation_number=22; citation_pages=2224-2234; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hypoxia, leukocytes, and the pulmonary circulation.; citation_author=KR Stenmark; citation_author=NJ Davie; citation_author=JT Reeves; citation_author=MG Frid; citation_journal_title=J Appl Physiol; citation_volume=98; citation_number=23; citation_pages=715-721; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000436" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia"/>
    <meta name="twitter:description" content="BackgroundSomatic cytidine mutations in normal mammalian nuclear genes occur during antibody diversification in B lymphocytes and generate an isoform of apolipoprotein B in intestinal cells by RNA editing. Here, I describe that succinate dehydrogenase (SDH; mitochondrial complex II) subunit B gene (SDHB) is somatically mutated at a cytidine residue in normal peripheral blood mononuclear cells (PBMCs) and T-cell acute leukemia. Germ line mutations in the SDHB, SDHC or SDHD genes cause hereditary paraganglioma (PGL) tumors which show constitutive activation of homeostatic mechanisms induced by oxygen deprivation (hypoxia).Principal FindingsTo determine the prevalence of a mutation identified in the SDHB mRNA, 180 samples are tested. An SDHB stop-codon mutation c.136C&gt;T (R46X) is present in a significant fraction (average&#8202;=&#8202;5.8%, range&#8202;=&#8202;less than 1 to 30%, n&#8202;=&#8202;52) of the mRNAs obtained from PBMCs. In contrast, the R46X mutation is present in the genomic DNA of PBMCs at very low levels. Examination of the PBMC cell-type subsets identifies monocytes and natural killer (NK) cells as primary sources of the mutant transcript, although lesser contributions also come from B and T lymphocytes. Transcript sequence analyses in leukemic cell lines derived from monocyte, NK, T and B cells indicate that the mutational mechanism targeting SDHB is operational in T-cell acute leukemia. Accordingly, substantial levels (more than 3%) of the mutant SDHB transcripts are detected in five of 20 primary childhood T-cell acute lymphoblastic leukemia (T-ALL) bone marrow samples, but in none of 20 B-ALL samples. In addition, distinct heterozygous SDHB missense DNA mutations are identified in Jurkat and TALL-104 cell lines which are derived from T-ALLs.ConclusionsThe identification of a recurrent, inactivating stop-codon mutation in the SDHB gene in normal blood cells suggests that SDHB is targeted by a cytidine deaminase enzyme. The SDHB mutations in normal PBMCs and leukemic T cells might play a role in cellular pre-adaptation to hypoxia."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000436.g004"/>

  <meta property="og:title" content="A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000436" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2960'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=238'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=569&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000436">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000436" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000436&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bora E. Baysal
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:baysalb@mwri.magee.edu">baysalb@mwri.magee.edu</a></p>

                <p>Affiliations:
                  Department of Obstetrics, Gynecology and Reproductive Sciences, University of Pittsburgh School of Medicine, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America, 
                  Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 09, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000436</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000436" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000436" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000436" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000436" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000436-g001" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000436-g002" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000436-t001" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000436-g003" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000436-g004" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000436">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000436" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Somatic cytidine mutations in normal mammalian nuclear genes occur during antibody diversification in B lymphocytes and generate an isoform of apolipoprotein B in intestinal cells by RNA editing. Here, I describe that succinate dehydrogenase (SDH; mitochondrial complex II) subunit B gene (<em>SDHB</em>) is somatically mutated at a cytidine residue in normal peripheral blood mononuclear cells (PBMCs) and T-cell acute leukemia. Germ line mutations in the <em>SDHB</em>, <em>SDHC</em> or <em>SDHD</em> genes cause hereditary paraganglioma (PGL) tumors which show constitutive activation of homeostatic mechanisms induced by oxygen deprivation (hypoxia).</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To determine the prevalence of a mutation identified in the <em>SDHB</em> mRNA, 180 samples are tested. An <em>SDHB</em> stop-codon mutation c.136C&gt;T (R46X) is present in a significant fraction (average = 5.8%, range = less than 1 to 30%, n = 52) of the mRNAs obtained from PBMCs. In contrast, the R46X mutation is present in the genomic DNA of PBMCs at very low levels. Examination of the PBMC cell-type subsets identifies monocytes and natural killer (NK) cells as primary sources of the mutant transcript, although lesser contributions also come from B and T lymphocytes. Transcript sequence analyses in leukemic cell lines derived from monocyte, NK, T and B cells indicate that the mutational mechanism targeting <em>SDHB</em> is operational in T-cell acute leukemia. Accordingly, substantial levels (more than 3%) of the mutant <em>SDHB</em> transcripts are detected in five of 20 primary childhood T-cell acute lymphoblastic leukemia (T-ALL) bone marrow samples, but in none of 20 B-ALL samples. In addition, distinct heterozygous <em>SDHB</em> missense DNA mutations are identified in Jurkat and TALL-104 cell lines which are derived from T-ALLs.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The identification of a recurrent, inactivating stop-codon mutation in the <em>SDHB</em> gene in normal blood cells suggests that <em>SDHB</em> is targeted by a cytidine deaminase enzyme. The <em>SDHB</em> mutations in normal PBMCs and leukemic T cells might play a role in cellular pre-adaptation to hypoxia.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Baysal BE (2007) A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia. PLoS ONE 2(5):
          e436.
            doi:10.1371/journal.pone.0000436</p><p><strong>Academic Editor: </strong>Stefan Maas, Lehigh University, United States of America</p><p><strong>Received:</strong> March 29, 2007; <strong>Accepted:</strong> April 15, 2007; <strong>Published:</strong> May 9, 2007</p><p><strong>Copyright:</strong> © 2007 Bora Baysal. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NIH grants (CA11236 to B.E.B, CA98543 and CA114766 to Children's Oncology Group). The funders had no role in the design and conduct of the study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The author has declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Oxygen deprivation (hypoxia) induces adaptive responses in most organisms and cell types <a href="#pone.0000436-Chadwick1">[1]</a> and may contribute to pathogeneses of common human diseases. Several adaptive responses to hypoxia are mimicked by hereditary paraganglioma (PGL), a human genetic disorder characterized by the development of tumors from the hypoxia-sensitive paraganglionic tissues and caused by germ line heterozygous inactivating mutations in the nuclear-encoded <em>SDHB</em>, <em>SDHC</em> or <em>SDHD</em> subunit genes of mitochondrial complex II (succinate dehydrogenase; SDH; succinate-ubiquinone oxidoreductase) <a href="#pone.0000436-Baysal1">[2]</a>, <a href="#pone.0000436-Bayley1">[3]</a>. PGL mutations or chronic hypoxic exposure predispose to carotid body (CB) paragangliomas <a href="#pone.0000436-Baysal1">[2]</a>. The PGL tumors are highly vascular and show global gene expression profiles of angiogenesis and hypoxic-pathway activation <a href="#pone.0000436-Dahia1">[4]</a>. Lower altitudes are associated with decreased penetrance/expressivity and increased prevalence of <em>SDHD</em> mutations <a href="#pone.0000436-Astrom1">[5]</a>. Heterozygous inactivation of the mouse <em>Sdhd</em> gene leads to persistent hypoxic stimulation of the CB paraganglionic cells <a href="#pone.0000436-Piruat1">[6]</a>. These data strongly suggest that constitutive activation of hypoxia-sensing and signaling pathways, presumably as a consequence of accumulation of reactive oxygen species <a href="#pone.0000436-Slane1">[7]</a>, succinate <a href="#pone.0000436-Gottlieb1">[8]</a> and/or another messenger molecule, underlies PGL pathogenesis. Whether PGL gene mutations predispose to non-paraganglionic tumors or system abnormalities is a subject of ongoing investigations <a href="#pone.0000436-Gottlieb1">[8]</a>, <a href="#pone.0000436-Baysal2">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In contrast to <em>SDHB</em> and <em>SDHC</em>, mutations in <em>SDHD</em> display a transmission pattern of maternal genomic imprinting (inactivation). <em>SDHD</em> mutations cause PGL type 1 (PGL1) after a paternal but not a maternal transmission. Although no maternal transmission of PGL1 has ever been documented, its molecular basis remains unclear. Non-paraganglionic tissues display bi-allelic expression of <em>SDHD</em>, suggesting that imprinting may be confined to paraganglia <a href="#pone.0000436-Baysal1">[2]</a>. In this report, I demonstrate that the <em>SDHB</em> subunit gene is targeted by a somatic mutational mechanism in normal peripheral blood and childhood T-cell acute leukemia.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Samples</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Peripheral bloods (10–30 mL) were obtained by venipuncture into EDTA-containing tubes from consenting adults of both sexes and shipped to author's laboratory at room temperature. Peripheral blood mononuclear cells (PBMCs) were isolated by centrifugation using BD vacutainer CPT™ tubes (Beckton-Dickinson) and stored at −135°C or in liquid nitrogen in freezing media, which was composed of 90% growth media containing RPMI-1640 (CellGro) and 15% fetal bovine serum (CellGro) and 10% DMSO (Sigma). A subset of PBMCs was used to immortalize B cells by Epstein-Barr virus using standard protocols <a href="#pone.0000436-Ling1">[10]</a>. Diagnostic bone marrow samples from acute leukemia patients were received in freezing media from ALL cell bank of Children's Oncology Group (COG) and stored at −80°C. Fetal brain and thymus tissues were collected from 20 week or older fetuses during necropsy and stored at −80°C. cDNAs synthesized from column-purified normal PBMC cell-type subsets, including monocytes (CD14+), NK cells (CD56+, negative selection), B (CD19+) and T lymphocytes (CD3+), were purchased from AllCells, LLC Inc. through StemCell Technologies. Each subset was purified from blood obtained from a different adult subject. The purity and the viability of the each cell-type subset were on average 94% and 97%, respectively, as confirmed by FACScan analyses. The leukemia cell lines were purchased from American Type Culture Collection (ATCC) and grown in the recommended culture conditions to extract DNA and RNA. The B cell lymphoblastic leukemia lines included DB, RL and Pfeiffer. The Pfeiffer cell line contains somatic hypermutations in the rearranged immunoglobulin variable genes <a href="#pone.0000436-Gabay1">[11]</a>. T cell lines included Jurkat, SUP-T1, TALL-104, MOLT-3, MOLT-4, CEM/C2, which were derived from child/adolescent acute leukemia, Loucy, which was derived from adult acute leukemia and HuT 78, HH, which were derived from cutaneous T cell leukemia. NK-92 is an interleukin-2 (IL-2) dependent cell line derived from natural killer cell. U-937, THP-1 and AML-193 were derived from acute monocytic leukemia/lymphoma. All samples were obtained under research protocols reviewed and approved by the University of Pittsburgh Institutional Review Board (IRB) committee.</p>


<h4>Stimulation of PBMCs by phytohemagglutinin (PHA)</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>PBMCs were freshly isolated from 30 mL peripheral blood drawn into EDTA tubes by Ficoll-Hypaq density-gradient centrifugation. One to two million PBMCs were added to three ml of RPMI 1640 media containing 10% fetal bovine serum, 20 U/ml IL-2 (Pepro Tech) and 2.5 or 5.0 µg/ml of selective T-cell stimulant phytohemagglutinin PHA-L (Roche Diagnostics) and incubated at 37°C and 5% CO<sub>2</sub> concentration. These recommended PHA concentrations showed robust T cell stimulation, as assessed by MTT cell proliferation kit I (Roche Diagnostics), by inducing activation-induced cell death in the Jurkat cell line. PHA-stimulated PBMCs were sampled at six hours, days 2, 5 and 8 to extract RNA after determining total cell number and fraction of dead cells by trypan blue staining in a hemocytometer. The number of PHA stimulated PBMCs remained unchanged at six hours and day 2 but increased ~10-fold at days 5 and 8.</p>


<h4>Isolation of CD4+ PBMCs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>CD4+ cells were isolated from fresh monocyte-depleted PBMCs, obtained from 30 mL blood, using a positive selection column following the recommended protocol (EasySep™, StemCell Technologies). Monocyte-depleted PBMCs were prepared by Ficoll-Hypaque density separation using RosetteSep™ monocyte (CD36) depletion cocktail (StemCell Technologies).</p>


<h4>RNA isolation, RT-PCR, PCR amplification, plasmid cloning, sequencing</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Total RNA was isolated by the single-step method of RNA extraction (RNA-Bee™, Tel-Test). cDNA was synthesized from 0.1–5 µg of total RNA using Maloney Murine Leukemia Virus reverse transcriptase (Life Technologies) and oligo(dT) primers following the recommended protocol. One fifth to one tenth of the freshly synthesized cDNA was used for nested RT-PCR amplification. The steady-state fraction of mutant transcripts was estimated by three semi-independent RT-PCR reactions. For 11 of 179 (6.1%) tested samples, the fraction of mutant transcripts was estimated by two RT-PCR reactions because of assay failure in one of the three replicates. The first round of the nested RT-PCR was performed in three separate reactions using a common forward primer located in exon 1 (F1A) and a unique reverse primer located in exons 3 (R13), 5 (R15) or 8 (R10), respectively (<a href="#pone.0000436.s005">Table S1</a>). The first round PCR products were diluted 1 to 50 in Tris-EDTA (TE) buffer (pH = 8.0) and subjected to second round PCR employing nested primers F1C and R14, which are located in exons 1 and 3, respectively. The first-and second round of RT-PCR employed 30 cycles at 60°C and 61°C annealing temperatures, respectively. By not designing the RT-PCR primers from exon 2, where the R46X mutation is located, co-amplification of the gDNA was prevented. Genomic PCR of <em>SDHB</em> exon 2 was performed using intronic primers FN2A, RN2A and 2R. RT-PCR amplification of the other SDH subunit genes, <em>SDHA</em>, <em>SDHC</em> and <em>SDHD</em> were also performed by two rounds of nested PCR using the primers listed in <a href="#pone.0000436.s005">Table S1</a>. To estimate transcript mutation rates (<a href="#pone.0000436.s006">Table S2</a>), RT-PCR products were cloned into plasmid following a commercial protocol (PCR-Script™ Amp, Stratagene) and sequenced by standard automated sequencing after isolating DNAs from plasmids by alkaline miniprep kits (Qiagene). <em>SDHB</em> DNA mutation analyses of the leukemic samples were performed by genomic PCR amplification of each of 8 exons and direct sequencing as described <a href="#pone.0000436-Baysal3">[12]</a>. In addition to the novel <em>SDHB</em> gene mutations in TALL-104 and Jurkat, we also identified a previously reported silent polymorphism, c.24C&gt;T (Ser8Ser) in the SUP-T1 cell line. The DNA mutations were confirmed by alternative methods including restriction enzyme digestion (for A15T) and cDNA sequencing (for R217H).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Standard PCR amplifications were performed by <em>Taq</em> polymerase (GeneChoice). All PCR amplifications were confirmed by gel electrophoresis before further analyses. Each PCR reaction contained 20 pmol of oligonucleotide primers, 1.5 mM of MgCl<sub>2</sub> in standard 1× PCR buffer, 0.2 mM of each of the dNTPs and 1 Unit of <em>Taq</em> polymerase. Each amplification plate had water and TE buffer negative control samples. Both rounds of the RT-PCRs for the assessment of transcript mutation rates (<a href="#pone.0000436.s006">Table S2</a>) and the first round of mutation enrichment RT-PCRs (RT-<span class="underline">e</span>PCRs) employed PfuUltra2 enzyme, a high fidelity <em>Taq</em> polymerase enzyme, following the recommended protocol (Stratagene).</p>


<h4>Enrichment PCR for the R46X mutation</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To enrich for the mutant DNA molecules, we employed enrichment RT-PCR (RT-<span class="underline">e</span>PCR) and enrichment genomic PCR (<span class="underline">e</span>PCR). RT-<span class="underline">e</span>PCR involved overnight-digestion of 10 µL of the first-round PfuUltra-amplified RT-PCR products by 10 U of <em>Taq</em>I (New England Biolabs) in a 20 µL reaction volume. The digestion products were then diluted 1 to 10 in TE buffer and PCR amplified in the second round. <span class="underline">e</span>PCR started with overnight-digestion of 0.5–1.0 µg of genomic DNA (gDNA) by 10 U of <em>Taq</em>I (New England Biolabs) in a 20 µL reaction volume. Four µL of the RE digestion product were then directly used for nested PCR amplifications in two rounds. The first round <span class="underline">e</span>PCR products amplified by the intronic primers FN2A and 2R were diluted 1 to 50 in TE buffer and subjected to second round PCR amplification by FN2A and the nested intronic primer RN2A (<a href="#pone.0000436.s005">Table S1</a>).</p>


<h4>Quantification of the R46X mutation by TaqI RE digestion and capillary gel electrophoresis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The quantification experiments employed the hot-stop PCR method <a href="#pone.0000436-Uejima1">[13]</a>. We removed the PCR plate at the end of the 29<sup>th</sup> cycle of the second round PCR and added 20 pmol of a HEX-labeled R14 primer (Sigma-Genosys) in a 4 µL 1× PCR buffer. The 30<sup>th</sup> (last) cycle of the PCR was completed to produce fluorescently-labeled homo-duplex extension products. Fifteen µL of the fluorescently-labeled PCR products, which contained ~200–300 ng of DNA, were directly digested by 10 U of <em>Taq</em>I RE, 1× <em>Taq</em>I RE buffer and 1× Bovine Serum Albumin (all from New England Biolabs) overnight in a 25 µL reaction volume. To ensure complete digestion, the reaction was re-dosed with another 10 U of <em>Taq</em>I RE in a 5 µL volume and continued for at least another four hours in the following day. All RE digestion products were inspected by gel electrophoresis (<a href="#pone.0000436.s005">Fig. S1</a>). The fluorescently-labeled and <em>Taq</em>I-RE digested PCR products were run in capillary gel electrophoresis (3100 Avant Genetic Analyzer, ABI Prism, Applied Biosystems/HITACHI). To quantify the amount of the mutant fraction, the area under the peak for the mutant undigested sequence (285 bp) was divided by the sum of the areas under the peaks for the mutant (undigested) and wild-type (digested, 126 bp) sequences (See <a href="#pone-0000436-g001">Fig. 1C,D</a>).</p>
<div class="figure" id="pone-0000436-g001"><div class="img"><a name="pone-0000436-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Analysis of <em>SDHB</em> R46X mRNA mutation.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>A. The <em>SDHB</em> gene has 843 bp coding nucleotides spread to 8 exons within ~35 kb at chromosome band 1p36.13. The 5′- portion of the gene shows the position of the R46X mutation and the mitochondrial signal peptide cleavage site (filled circle). B. Sequence chromatograms of RT-PCR products show samples that have high (top), low (middle), or undetectable (bottom) amounts of mutant R46X sequences. C. Quantification of the mutant fraction involved <em>Taq</em>I RE digestion of fluorescently-labeled RT-PCR products and capillary gel electrophoresis. The red peaks denote the molecular weight marker. D. The agarose gel electrophoresis shows variable fractions of mutant RT-PCR products from normal PBMCs detected by <em>Taq</em>I RE digestion, which releases two bands 159 and 126 bp in size from the wild-type sequence (also see <a href="#pone.0000436.s001">Fig. S1A</a>). E. Fractions of <em>Taq</em>I RE resistant transcripts in the purified PBMC cell types are shown in an agarose gel. The negative image is presented to enhance the visibility of mutant transcripts.</p>
<span>doi:10.1371/journal.pone.0000436.g001</span></div>

<h4>Analyses of somatic mutation rates</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Nucleotide misincorporation occurs during <em>in vitro</em> synthesis of cDNA by reverse transcriptase and PCR amplification. The nucleotide misincorporation rates by Moloney Murine Leukemia virus (MMLV)/SuperScript III reverse transcriptases, <em>Taq</em> and PfuUltra DNA polymerases were assumed 6.0×10<sup>−5</sup>, 8×10<sup>−6</sup> and 0.4×10<sup>−6</sup> per bp per duplication, respectively <a href="#pone.0000436-Arezi1">[14]</a>. We estimated the highest effective duplication number of <em>SDHB</em> cDNAs during the two rounds of nested PCR in PBMCs using control plasmids containing the wild-type gene. Using serial dilutions, we found that after 30 cycles of the first-round PCR, as low as 0.05 fg (0.05×10<sup>−15</sup> g) of <em>SDHB</em> cDNA plasmids did not produce bands detectable in the ethidium bromide-stained agarose gel, similar to the PBMC cDNAs. However, 1 to 50 dilutions of the first-round PCR products yielded ~400 ng of DNA after the 30 cycle amplification of the second round nested PCR. (<em>Taq</em>I RE digestion and gel electrophoreses of the second-round wild-type plasmid PCR products showed no evidence of mutations within the 4-bp RE recognition site introduced during amplification by <em>Taq</em> polymerase.) This enrichment corresponds to ~4×10<sup>11</sup>-fold or ~38.5 effective duplications by the nested PCR amplification. Consequently, the nested RT-PCR by <em>Taq</em> polymerase is expected to generate errors during cDNA synthesis (6.0×10<sup>−5</sup> per bp) and PCR amplification (38.5×8×10<sup>−6</sup> per bp), which sums to ~36.8×10<sup>−5</sup> per bp (i.e., one error in 2,717 bps). Similarly, nested RT-PCR by PfuUltra is expected to generate a total error rate of ~7.54×10<sup>−5</sup> per bp (i.e., one error in 13,263 bp), which was used to test for excess mutations observed in the cloned-sequenced RT-PCR products assuming Poisson distribution (<a href="#pone.0000436.s006">Table S2</a>). Statistical analyses were performed using SISA, a Simple Interactive Package for Statistical Analysis (<a href="http://home.clara.net/sisa/index.htm">http://home.clara.net/sisa/index.htm</a>).</p>


<h4>Control experiments on the R46X mutation</h4>
<h5>(a) Overall random mutation rate in SDHB transcripts is not increased in PBMCs.</h5><a id="article1.body1.sec2.sec8.sec1.p1" name="article1.body1.sec2.sec8.sec1.p1"></a><p>We cloned and sequenced <em>SDHB</em> RT-PCR products obtained from total and monocyte-depleted CD4+ PBMCs, leukemic cell lines derived from a post-germinal center B lymphocyte (Pfeifer), a T lymphocyte (Jurkat) and a natural killer cell (NK-92) (<a href="#pone.0000436.s006">Table S2</a>). The R46X mutation was observed among the cDNA clones obtained from total and CD4+ PBMCs and the Jurkat cell line. The overall <em>SDHB</em> transcript mutation rate was increased only in the Jurkat line. The mutation rates in the control genes <em>SDHA</em>, <em>SDHD</em> and <em>SDHC</em> were not significantly increased in Jurkat.</p>

<h5>(b) The R46X mutation is not introduced during enzymatic amplification.</h5><a id="article1.body1.sec2.sec8.sec2.p1" name="article1.body1.sec2.sec8.sec2.p1"></a><p>Although absence of the R46X mutation in the EBV-transformed lymphoblastoid cell lines strongly suggests that the mutation is not introduced during <em>in vitro</em> amplification, to further rule out this possibility, we also tested wild-type <em>SDHB</em> plasmids. On the basis of nucleotide misincorporation rates, enzymatic amplification of the wild-type plasmids for 38.5 effective duplications during nested PCR (above) are expected to generate random mutations in ~0.15% of the PCR products within the 4 bp-long <em>Taq</em>I RE recognition site. Accordingly, agarose and capillary gel electrophoreses showed no evidence of mutations in the <em>Taq</em>I RE recognition site in the PCR products amplified from 10<sup>6</sup>-fold range of starting wild-type plasmid DNA amounts (<a href="#pone.0000436.s001">Fig. S1</a>). When wild-type <em>SDHB</em> plasmids were PCR amplified in the first-round, digested by <em>Taq</em>I RE and subjected to a second-round PCR (i.e., mutation enrichment PCR, <span class="underline">e</span>PCR), no plasmid template (n = 19) showed the R46X mutation by direct sequencing. In contrast, all PBMC cDNAs (n = 35) tested by RT-<span class="underline">e</span>PCR showed the R46X mutation by direct sequencing (<a href="#pone.0000436.s001">Fig. S1B</a>). Finally, the R46X mutation was observed after RT-PCR amplification either by <em>Taq</em> polymerase or by PfuUltra (<a href="#pone.0000436.s006">Table S2</a>) and after reverse transcription either by MMLV or by SuperScript III (Life Technologies) reverse transcriptase enzymes.</p>

<h5>(c) Nested PCR amplification does not significantly distort the starting fractions of mutant cDNAs.</h5><a id="article1.body1.sec2.sec8.sec3.p1" name="article1.body1.sec2.sec8.sec3.p1"></a><p>To test whether nested PCR amplification of <em>SDHB</em> significantly distorts the starting fraction of mutant cDNAs, we amplified control plasmid templates which contained pre-determined fractions of mutant DNAs and quantified the percentage of mutant molecules by <em>Taq</em>I RE digestion and capillary gel electrophoresis. Primers F1C-R10, F1C-R15 and F1C-R13 were used in the first-round of the nested PCR to generate three replicates for each sample. Each replicate was diluted 1 to 50 in T.E. buffer and then PCR amplified by F1C-R14 in the second-round, in the last cycle of which a fluorescently-labeled R14 primer was added. The confidence intervals of the expected and the observed percentages of mutant DNAs overlapped over a 1,000-fold range of starting fractions of the mutant DNA (Fig. S2<a href="#pone.0000436.s002">pone.0000436.s002</a>) suggesting that nested PCR does not significantly skew the fraction of mutant DNAs.</p>

<h5>(d) The R46X mutation targets the full-length SDHB coding mRNA sequences.</h5><a id="article1.body1.sec2.sec8.sec4.p1" name="article1.body1.sec2.sec8.sec4.p1"></a><p>Northern analysis indicated that <em>SDHB</em> is ubiquitously expressed and its transcripts cluster at a single band at ~1.1 kb size (<a href="#pone.0000436.s003">Fig. S3</a>). RT-PCR, <em>Taq</em>I RE digestion and gel electrophoresis of PBMC mRNAs showed that R46X occurs in the full-length <em>SDHB</em> coding mRNA. This is also confirmed by direct sequencing of 23 clones that have the R46X mutation.</p>

<h5>(e) No processed SDHB pseudogene exists in the gDNA.</h5><a id="article1.body1.sec2.sec8.sec5.p1" name="article1.body1.sec2.sec8.sec5.p1"></a><p>Processed pseudogenes in the gDNA could co-amplify during RT-PCR and potentially complicate interpretation of cDNA-based mutation analyses. Sequence of the human genome does not indicate presence of a pseudogene for <em>SDHB</em> (<a href="http://www.genome.ucsc.edu">http://www.genome.ucsc.edu</a>). Accordingly, PCR amplification of gDNA by <em>SDHB</em> RT-PCR primers F1A and R13 did not yield any products in 23 samples (data not shown).</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Discovery of SDHB R46X mRNA mutation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>While investigating consequences of a PGL-associated <em>SDHB</em> splice-site mutation IVS1 +1 G&gt;T on transcript splicing in an affected subject by reverse transcription, nested PCR amplification (RT-PCR) and direct sequencing we found that a fraction of the correctly-spliced <em>SDHB</em> complementary DNAs (cDNAs) synthesized from peripheral blood mononuclear cells (PBMCs) contained a premature termination codon mutation, c.136C&gt;T (R46X) (<a href="#pone-0000436-g001">Fig. 1A</a>) that was absent in the genomic DNA (gDNA). The R46X transcript mutation was subsequently identified in several unaffected subjects in variable fractions by direct sequencing of PBMC cDNAs (<a href="#pone-0000436-g001">Fig. 1B</a>). The R46X, a pathogenic mutation in exon 2, has been previously identified in the germ lines of multiple PGL cases <a href="#pone.0000436-Bayley1">[3]</a> and is predicted to truncate the <em>SDHB</em> protein product shortly after the mitochondrial signal peptide sequence (<a href="#pone-0000436-g001">Fig. 1A</a>). Further control experiments, which are explained in detail in Methods, showed that (a) overall random mutation rate in <em>SDHB</em> transcripts is not increased in PBMCs (b) the R46X mutation is not introduced by <em>Taq</em> polymerase; (c) Nested PCR amplification does not significantly distort the starting fractions of mutant cDNAs; (d) the R46X mutation targets the full-length <em>SDHB</em> coding mRNA sequences; (e) no <em>SDHB</em> pseudogene exists in the gDNA.</p>


<h4>Quantification of the R46X mutation in PBMCs</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>To quantify the fraction of <em>SDHB</em> transcripts containing R46X, we screened a unique <em>Taq</em>I restriction enzyme (RE) recognition site (T<span class="underline">C</span>GA) destroyed by the mutation using fluorescent “hot-stop” RT-PCR, <em>Taq</em>I RE digestion and capillary gel electrophoresis (<a href="#pone-0000436-g001">Fig. 1C, D</a>). The average fraction of the R46X-containing <em>SDHB</em> transcripts obtained from PBMCs of normal controls and of <em>SDHD/C</em> germ line mutation carriers were 5.7% and 5.9% (range less than 1 to 30%), respectively (<a href="#pone-0000436-g002">Fig. 2</a>). Cloning-sequencing confirmed that ~99% of the <em>Taq</em>I-RE-digestion-resistant RT-PCR clones from PBMCs were composed of the R46X mutation (<a href="#pone-0000436-t001">Table 1</a>). When the second round RT-PCR was performed after removing the wild-type sequences from the first round products by <em>Taq</em>I RE digestion (i.e., enrichment RT-PCR; RT-<span class="underline">e</span>PCR), all tested PBMCs showed the R46X mutation by direct sequencing (see <a href="#s2">Methods</a>). Thus, the average inactivation rate of <em>SDHB</em> mRNAs in PBMCs is ~10,000-fold higher than would be expected from all background <em>in vivo</em> somatic inactivating DNA mutations as measured in the <em>HGPRT</em> gene in normal adult peripheral blood T lymphocytes <a href="#pone.0000436-Albertini1">[15]</a>. The levels of the mutated <em>SDHB</em> transcripts in PBMCs are comparable to the C-to-U-edited neurofibromatosis 1 (<em>NF1</em>) mRNAs that occur in substantial levels (&gt;3%) in certain NF1-related tumors, but not in normal tissues, and are thought to be catalyzed by the apolipoprotein B mRNA-editing enzyme APOBEC-1 <a href="#pone.0000436-Mukhopadhyay1">[16]</a>.</p>
<div class="figure" id="pone-0000436-g002"><div class="img"><a name="pone-0000436-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The steady-state fraction of mutant <em>SDHB</em> cDNAs resistant to <em>Taq</em>I RE digestion.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Each circle represents a sample from a different subject (total indicated by n) except for those in the PHA stimulated PBMCs which were obtained from two normal donors, stimulated by 2.5 µg/ml and 5.0 µg/ml concentrations of PHA and tested at days 2, 5 and 8 for a total of 12 samples. The value corresponding to each circle was derived from three RT-PCR reactions. Two outlier PBMC values were shown at the top with their mutant transcript fractions in parentheses. Boxes and the vertical lines denote the means and their 95% confidence intervals of samples sets. “PBMC mutation carrier” group contains 5 <em>SDHC</em> and 24 <em>SDHD</em> mutation carriers. The leukemic cell lines were derived from B cells (n = 3), T cells (n = 9), NK cells (n = 1) and monocytes (n = 2).</p>
<span>doi:10.1371/journal.pone.0000436.g002</span></div><div class="figure" id="pone-0000436-t001"><div class="img"><a name="pone-0000436-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000436.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000436.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Sequence profiles of <em>Taq</em>I-RE-digestion-resistant mutant <em>SDHB</em> clones</span></strong></p><span>doi:10.1371/journal.pone.0000436.t001</span></div>

<h4>Analyses of the R46X mutation in PBMC cell-type subsets</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Because PBMCs are primarily composed of lymphocytes and monocytes, we analyzed cDNAs obtained from purified peripheral blood monocytes, natural killer (NK) cells, T and B lymphocytes. The fraction of mutant transcripts was higher in monocytes and NK cells than in T and B lymphocytes (<a href="#pone-0000436-g001">Figure 1E</a>). Cloning-sequencing of <em>Taq</em>I-RE-digestion-resistant band from the monocytes confirmed the R46X mutation in 32 of 37 (86%) clones (<a href="#pone-0000436-t001">Table 1</a>). Mutation enrichment RT-PCR (RT-<span class="underline">e</span>PCR) analyses confirmed the presence of R46X mutation in each of the PBMC cell-type subset (data not shown).</p>


<h4>Analyses of the R46X mutation in leukemic cell lines</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>To test presence of the R46X transcript mutation in leukemic cell lines derived from PBMCs, we amplified <em>SDHB</em> cDNAs mutated at the <em>Taq</em>I RE site by RT-<span class="underline">e</span>PCR. Cloning-sequencing of <em>Taq</em>I-RE-digestion-resistant products showed the R46X mutation in 27 of 54 (50%) clones in the T cell leukemia cell lines but only in 2 of 26 (7.7%) clones in NK-92, which is derived from an NK cell, in 2 of 23 (8.7%) in Pfeiffer, which is derived from a post-germinal center B lymphocyte, and in 9 of 56 (16.1%) of THP-1, which is derived from a monocyte (<a href="#pone-0000436-t001">Table 1</a>). The number of clones with the R46X mutation was statistically significantly overrepresented only in the T-ALL cell lines relative to the random somatic mutation profiles <a href="#pone.0000436-Albertini1">[15]</a> (<a href="#pone-0000436-t001">Table 1</a>). Furthermore, the overall transcription mutation rate in <em>SDHB</em> but not <em>SDHA</em>, <em>SDHC</em> and <em>SDHD</em> was increased in the Jurkat T cell line (<a href="#pone.0000436.s006">Table S2</a>).</p>


<h4>Analyses of the R46X mutation in primary leukemic and various other samples</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Next, we compared the <em>Taq</em>I-resistant mutant transcript fractions in a total of 175 samples including EBV-transformed lymphoblastoid cell lines, which are derived from peripheral blood B cells, various commercial leukemic cell lines (see <a href="#s2">Methods</a>), diagnostic bone marrow samples from childhood T- and B-ALL, PBMCs treated by T cell-stimulant phytohemagglutinin (PHA), fetal thymus and brain tissues and non-mutant wild-type plasmid control DNAs. Substantial levels (&gt;3%) of <em>Taq</em>I-RE-digestion-resistant mutant transcripts were observed in five of 20 diagnostic primary T-ALL samples (<a href="#pone-0000436-g002">Fig. 2</a>). Cloning-sequencing of the <em>Taq</em>I-digestion resistant bands from T-ALL primary samples uncovered the R46X mutation in 27 of 43 (62%) clones (<a href="#pone-0000436-t001">Table 1</a>). Because primary diagnostic leukemic bone marrow samples are significantly enriched for the lineage-specific lymphocytes, these results further suggest that the mutational mechanism targeting the <em>SDHB</em> mRNA remains functional in leukemic T cells.</p>


<h4>Analyses of the R46X mutation in genomic DNA (gDNA)</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>To test whether the R46X mutation also occurs in the gDNA of PBMCs, we initially PCR-amplified <em>SDHB</em> exon 2 from 22 samples and digested it with <em>Taq</em>I RE. Gel electrophoresis of the RE digestion products showed no evidence of the mutation (<a href="#pone-0000436-g003">Fig. 3A</a>), including in the sample that showed the highest fraction of the mutant cDNA (30.2% in <a href="#pone-0000436-g002">Fig. 2</a>). To test whether the R46X mutation exists in the gDNA at very low levels, we employed enrichment PCR (<span class="underline">e</span>PCR), which involved removal of the non-mutant wild-type gDNA by <em>Taq</em>I RE digestion before nested PCR amplification. Analyses of the <span class="underline">e</span>PCR products by <em>Taq</em>I RE digestion and gel electrophoresis showed variable fractions of mutant molecules in PBMC gDNAs (<a href="#pone-0000436-g003">Fig. 3B</a>). Direct sequencing of <span class="underline">e</span>PCR products showed the R46X mutation in 3 of 24 samples (<a href="#pone-0000436-g003">Fig. 3D</a>). Cloning-sequencing of the <em>Taq</em>I RE digestion-resistant <span class="underline">e</span>PCR products from these three samples showed the R46X mutation in 27 of 30 clones (90%) confirming that the <span class="underline">e</span>PCR method statistically significantly enriched for the R46X mutation (<a href="#pone-0000436-t001">Table 1</a>). These findings indicate that the R46X mutation identified in <em>SDHB</em> cDNA also occurs in the gDNA at a much lower frequency.</p>
<div class="figure" id="pone-0000436-g003"><div class="img"><a name="pone-0000436-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span><em>SDHB</em> mutations in genomic DNAs (gDNAs).</span></strong></p><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1"></a><p>A. <em>Taq</em>I RE digestion of PCR-amplified exon 2 shows no evidence of undigested mutant DNAs at 260 bp size (pointed by an arrow head). <em>Taq</em>I RE digestion products of the wild-type DNA co-migrate at sizes of 128 and 132 bps (denoted by a star). B. Gel electrophoresis of <em>Taq</em>I RE-digested (mutation) enrichment PCR (<span class="underline">e</span>PCR) products shows variable amounts of mutant gDNAs at 233 bp size (pointed by an arrow head), which are confirmed to be primarily composed of c.136C&gt;T by sequencing (<a href="#pone-0000436-t001">Table 1</a>). <em>Taq</em>I digestion of wild-type <span class="underline">e</span>PCR products gives two bands at sizes, 101 and 132 bp (shown by a star). C. <span class="underline">e</span>PCR analyses of leukemic cell lines demonstrate the presence of mutant gDNA in the MOLT-4 cell line (pointed by an arrow head) but not in other leukemic cell lines in this experiment. D. Direct sequence analysis of a gDNA <span class="underline">e</span>PCR product shows selective enrichment of the R46X mutation. E, F. Sequence chromatograms of the heterozygous DNA mutations detected in two T-ALL cell lines are shown.</p>
<span>doi:10.1371/journal.pone.0000436.g003</span></div><a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>To test whether genomic R46X mutation also occurs in the leukemic cell lines, we employed <span class="underline">e</span>PCR on the gDNAs of 11 leukemic cell lines including four originating from T cells (Jurkat, CEM/C2, Loucy, MOLT-4), three from B cells (Pfeiffer, DB and RL), three from monocytes (AML-93, U-937, THP-1) and one from NK cells (NK-92). Analyses of the <span class="underline">e</span>PCR products by <em>Taq</em>I RE digestion and gel electrophoresis uncovered the presence of mutant bands in the CD4+ T-ALL cell lines MOLT-4 and Jurkat in three replicate experiments (<a href="#pone-0000436-g003">Fig. 3C</a>). Cloning-sequencing of the <em>Taq</em>I RE digestion-resistant bands from MOLT-4 and Jurkat uncovered R46X in 43 of 54 clones (~80%) (<a href="#pone-0000436-t001">Table 1</a>), demonstrating a statistically significant enrichment of the mutation. These results further suggest that the R46X mutation occurs in leukemic T lymphocytes.</p>


<h4>SDHB gene sequencing in PBMC-derived leukemic cell lines</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>We sequenced all eight <em>SDHB</em> gene exons in 16 leukemic cell line DNAs, including those derived from T, B, NK cells and monocytes, and found novel expressed DNA sequence alterations in two of six childhood T-ALL cell lines. The Arg217His mutation in TALL-104 (<a href="#pone-0000436-g003">Fig. 3E</a>) targets an amino acid residue conserved in 497 of 500 <em>SDHB</em> sequence entries from diverse organisms in GenBank and its distinct germ line missense mutation, Arg217Cys, is previously linked to PGL susceptibility <a href="#pone.0000436-Bayley1">[3]</a>. The Ala15Thr mutation in Jurkat (<a href="#pone-0000436-g003">Fig. 3F</a>) decreases hydrophocity of the mitochondrial signal peptide by converting a non-polar to a polar residue. The mutated alanine is conserved in 20 of 23 multicellular eukaryotic organisms which have the orthologous signal peptide sequences including fly, chicken and mammals (<a href="#pone.0000436.s004">Fig. S4</a>). These two missense variants were not present in the <em>SDHB</em> orthologous amino acid positions in other organisms in GenBank, in over 200 human chromosomes that we previously sequenced, and have not been reported as population variants in extensive mutation surveys of PGL <a href="#pone.0000436-Bayley1">[3]</a>. No <em>SDHB</em> gDNA mutations were detected in 20 T-ALL primary diagnostic samples.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The current study describes identification of substantial levels of <em>SDHB</em> transcripts carrying a recurrent R46X mutation in normal PBMCs. Analyses of the purified PBMC cell-types suggest monocytes and NK cells as the major sources of the mutant transcripts, although T and B lymphocytes also contribute at lower levels. In contrast, analyses of leukemic cell lines and bone marrow samples uncover the R46X transcripts only in T-ALL. The R46X-mRNAs constitute on average 5.8% of all <em>SDHB</em> transcripts in PBMCs, strongly suggesting that a cytidine deaminase enzyme(s) catalyzes this somatic mutation.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Programmed somatic mutations targeting cytidine residues in protein-encoding mammalian nuclear genes were previously observed in two normal cell types: the post-germinal center B lymphocytes during diversification of immunoglobulins by the process of somatic hypermutation <a href="#pone.0000436-Honjo1">[17]</a> and in intestinal cells to generate a shorter protein isoform (ApoB48) of apolipoprotein B by RNA editing <a href="#pone.0000436-Wedekind1">[18]</a>. The mutations in the immunoglobulin and apolipoprotein B loci confer new properties to the targeted gene products (i.e., gain-of-function). The R46X mutation in <em>SDHB</em> is unusual because it predicts loss-of-function, it targets a classical tumor-suppressor gene, and it occurs in PBMCs.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The results described here prompt important questions on mechanism and significance of <em>SDHB</em> mRNA mutations identified in normal PBMCs and leukemic T cells. The current results show that the R46X mutation is present at much higher proportions in mRNA than in DNA. This finding could be explained by two models. First, the putative mutator enzyme may target both DNA and mRNA, the latter with a higher efficiency. Alternatively, the mutator enzyme may target only DNA, which is then repaired after the transcription. The presence of sequence heterogeneity and A/T mutations in the <em>Taq</em>I RE recognition site (e.g., in the T-ALL bone marrow samples in <a href="#pone-0000436-t001">Table 1</a>) supports the second model because such sequence variants suggest operation of an error-prone repair mechanism in the leukemic T cells. The error-prone DNA repair of uracil residues is thought to be responsible for the generation of A/T mutations during antibody diversification in B cells by the activation-induced cytosine deaminase (AID) enzyme <a href="#pone.0000436-Neuberger1">[19]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Another important question concerns the causes of variation in the fraction of R46X-mRNAs obtained from total PBMCs. Heritable or acquired conditions such as infections, medications, exercise and diurnal rhythm might influence the relative abundance of PBMC cell-type subsets and, consequently, the fraction of the R46X-mRNAs. Alternatively, technical factors such as the length of time span and the storage conditions between the blood draw to the PBMC isolation procedure might affect the cell viability. Furthermore, if the R46X-mRNAs are variably targeted by nonsense-mediated decay, an intra-cellular surveillance mechanism that removes the transcripts containing premature termination codons <a href="#pone.0000436-Amrani1">[20]</a>, the steady-state fraction of mutant <em>SDHB</em> mRNAs would underestimate the true fraction of mutated transcripts in certain samples or PBMC cell-type subsets. Although each of these factors might influence the proportion of mutated <em>SDHB</em> mRNAs in a given sample, the present results indicate that the R46X mRNA mutation occurs in every tested PBMC sample at significantly higher levels compared to the control lymphoblastoid cell lines or wild-type plasmid templates. The identification of distinct heterozygous missense <em>SDHB</em> gene mutations in two T-ALL cell lines further suggests that the R46X mutation might play a functional role in the biology of PBMCs and leukemic T cells.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The genetic regulation of SDH subunits appears to be emerging as a common motif in several normal cell types. First, the transmission pattern of PGL1 tumors suggests that the <em>SDHD</em> subunit gene is suppressed on the maternal allele by genomic imprinting in the paraganglionic tissues <a href="#pone.0000436-Baysal1">[2]</a>. The <em>SDHD</em> mutations or chronic hypoxic exposure cause paragangliomas in the carotid body (CB). Notably, the CB is the major oxygen sensor organ in mammals and stimulates the cardiopulmonary system within seconds of exposure to hypoxia. Second, the <em>SDHA</em> gene, which encodes the major catalytical subunit (flavoprotein) of SDH, carries a strong signature of balancing selection in the African chromosomes <a href="#pone.0000436-Baysal4">[21]</a>. Distinct <em>SDHA</em> alleles, including two missense variants at amino acid residues conserved in mammals, have been maintained at intermediate frequencies for long evolutionary periods. Thus, certain <em>SDHA</em> allelic variants that have slight functional deficits might confer protection against pathogens or toxins that are prevalent in Africa. Finally, the present results demonstrate a recurrent <em>SDHB</em> mutation in normal PBMCs and leukemic T cells. Although many gaps exist in our knowledge on the mechanisms of imprinting on <em>SDHD</em>, balancing selection on <em>SDHA</em> and transcript mutations in <em>SDHB</em>, including their impact on SDH enzyme activity, the extant data on PGL suggest that these regulatory mechanisms might downregulate SDH function and facilitate early detection of and pre-adaptation to hypoxia (<a href="#pone-0000436-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0000436-g004"><div class="img"><a name="pone-0000436-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000436" data-uri="info:doi/10.1371/journal.pone.0000436.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000436.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000436.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000436.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000436.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>SDH genetic downregulation as a mechanism for early detection of hypoxia.</span></strong></p><a id="article1.body1.sec4.fig1.caption1.p1" name="article1.body1.sec4.fig1.caption1.p1"></a><p>Data from PGL research suggest that SDH normally inactivates a messenger(s) critical for downstream hypoxia-signaling. Consequently, heterozygous SDH germ line mutations over-activate the hypoxia signaling pathways in PGL. Loss of the wild-type allele by loss-of-heterozygosity (LOH) leads to constitutive hypoxic signaling and tumorigenesis in PGL. Transcript mutations, genomic imprinting, balancing selection or germ line mutations are postulated to reduce SDH activity and change the hypoxia-detection threshold from a lower oxygen concentration (<em>a</em>) to higher oxygen concentrations (<em>b, c</em>), thus leading to earlier detection of impending hypoxia.</p>
<span>doi:10.1371/journal.pone.0000436.g004</span></div><a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>It is well-known that blood monocytes differentiate into macrophages in solid tissues and accumulate in large numbers in chronically hypoxic avascular and necrotic areas in infected and neoplastic organs <a href="#pone.0000436-Murdoch1">[22]</a>, <a href="#pone.0000436-Stenmark1">[23]</a>. The <em>SDHB</em> somatic transcript mutations might enable earlier activation of hypoxia inducible pathways and increase survival of monocytes and natural killer cells in these hypoxic areas. A similar survival advantage might be conferred to leukemic T cells as they shuttle between the well-oxygenated peripheral blood and the more hypoxic immune-system organs. In conclusion, the present results suggest that the <em>SDHB</em> gene is targeted by a somatic mutational mechanism that likely involves the activity of a cytidine deaminase enzyme expressed in PBMCs and leukemic T cells. If the somatic <em>SDHB</em> mutations are proven to play a role in oxygen homeostasis in PBMCs and leukemic T cells, targeting of the SDH complex might provide new therapeutic tools against infections, leukemia and cancer.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000436"></div><a name="pone.0000436.s001" id="pone.0000436.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000436.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Detecting the R46X mutation in agarose gel electrophoresis. Each test set contains 16 different samples. The second-round RT-PCR products remaining undigested after the TaqI RE digestion at 285 bp (examples shown by arrows) indicate the presence of mutations within the 4-bp RE recognition. Highest levels of mutations were observed in the PBMC and T-ALL samples (also see <a href="#pone-0000436-g002">Figure 2</a>). Plasmid controls demonstrate outcome of the experiment when plasmids (~4.0 kb) containing the wild-type SDHB cDNA were used for nested PCR amplification at various starting amounts shown by numbers 1 to 4. Total plasmid input amounts: 1 = 28 ng, 2 = 0.28 ng, 3 = 2.8 pg, 4 = 28 fg. B. Enrichment of R46X by enrichment RT-PCR (RT-ePCR) in PBMCs. RT-ePCR involved removal of the wild-type sequences by TaqI digestion before the second-round PCR. Direct sequencing of second-round RT-ePCR products confirmed presence of the R46X mutation in all tested PBMCs (n = 35) but in none of the control wild-type plasmid templates (n = 19) nor in the lymhoblastoid cell lines (n = 14).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000436.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.18 MB TIF)</p>
<a name="pone.0000436.s002" id="pone.0000436.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000436.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Fraction of mutant cDNAs before and after PCR amplification. Mutant and wild-type plasmid DNAs that have full-length SDHB cDNAs were mixed in variable amounts to generate control template sets (denoted by letters A–I) for nested PCR. (Image for set E was re-positioned at the end of other sets from the lower half of the gel.) Each template set was composed of two samples (shown by numbers 1 and 2) that have the same fraction of mutant DNAs but different starting amounts of total plasmid [~20 and 5 fg (10–15 g), respectively]. The top of the figure shows Taq I RE digestion results of the second round PCR products that were amplified by F1C-R10, F1C-R15 (shown under the gel pictures) in the first-round and F1C-R14 in the second round. Graph at the bottom shows, on a logarithmic scale, the average percentages (denoted in parentheses) of the expected (starting) and observed (measured after nested PCR) ratios of mutant/wild type plasmids in the test sets. The confidence intervals (delimited by plus signs) for the expected percentages were derived from the most extreme values of plasmid DNA concentrations that were obtained from multiple (n = 7) spectrophotometric measurements. For the observed percentages, 95% confidence intervals were derived from quantification of the six replicates in each set. The lower boundaries of the observed confidence intervals for sets G, H were zero. Set I, which is not shown in the graph, has 0.02% expected and 0% observed mutant DNAs, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000436.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.09 MB TIF)</p>
<a name="pone.0000436.s003" id="pone.0000436.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000436.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Northern analysis of SDHB mRNA. SDHB is ubiquitously expressed and its transcripts cluster at a single band of ~1.1 kb size. Multiple Tissue Northern (MTN™, CLONTECH) Blot contained ~2 microgram of mRNA in each lane. The hybridization probe was generated by RT-PCR amplification of the full-length SDHB gene by primers F1A-R9 (<a href="#pone.0000436.s005">Table S1</a>) and labeled by 32P following the random priming method using a commercial protocol (High Prime, Roche).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000436.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.05 MB TIF)</p>
<a name="pone.0000436.s004" id="pone.0000436.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000436.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The Ala15Thr mutation in Jurkat cell line. Multiple sequence alignment (Clustal W 1.83) of N-terminal sequences of SDHB gene products demonstrates that human Ala15 is conserved (shown by red fonts) in most organisms which are denoted by their genus names. All sequences detected by the default parameters of BLAST analyses (<a href="http://www.ncbi.nlm.nih.gov/BLAST">http://www.ncbi.nlm.nih.gov/BLAST</a>) are shown.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000436.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB TIF)</p>
<a name="pone.0000436.s005" id="pone.0000436.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000436.s005">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Oligonucleotide PCR primers used to amplify SDH subunit genes by nested PCR. * Primers used in both rounds of the nested PCR.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000436.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000436.s006" id="pone.0000436.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000436.s006">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Transcript mutations in SDH subunit genes. *1 misincorporation per 13,263 bp on the basis of RT-PCR amplification by PfuUltra. Ns = not significant</p>
<p class="siDoi">doi:10.1371/journal.pone.0000436.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>I thank J.E. Willett-Brozick, G. Olshansky-Feddes, M. Hasipek, E.C. Lawrence for technical assistance, Drs. R.E. Ferrell, S.P. Hunger for helpful discussions, Drs. S.P. Hunger, M. Devidas and ALL Cell Bank Committee of the Children's Oncology Group (COG) for providing the diagnostic leukemia samples, patients and volunteers for blood donation.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BB. Performed the experiments: BB. Analyzed the data: BB. Contributed reagents/materials/analysis tools: BB. Wrote the paper: BB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000436-Chadwick1" id="pone.0000436-Chadwick1"></a> Chadwick DJ, Goode J, editors.  (2006) Signaling pathways in acute oxygen sensing; Novartis Foundation symposium. Hoboken: John Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000436-Baysal1" id="pone.0000436-Baysal1"></a>Baysal BE, Ferrell RE, Wilett-Brozick JE, Lawrence EC, Myssiorek D, et al.  (2000) Mutations in SDHD, a mitochondrial complex II gene, in hereditary paraganglioma. Science  287: 848–851.  <ul class="find" data-citedArticleID="950726" data-doi="10.1126/science.287.5454.848"><li><a href="http://dx.doi.org/10.1126/science.287.5454.848" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+SDHD%2C+a+mitochondrial+complex+II+gene%2C+in+hereditary+paraganglioma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+SDHD%2C+a+mitochondrial+complex+II+gene%2C+in+hereditary+paraganglioma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000436-Bayley1" id="pone.0000436-Bayley1"></a>Bayley JP, Devilee P, Taschner PE (2005) The SDH mutation database: an online resource for succinate dehydrogenase sequence variants involved in pheochromocytoma, paraganglioma and mitochondrial complex II deficiency. BMC Med Genet  6: 39.  <ul class="find" data-citedArticleID="950724"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+SDH+mutation+database%3A+an+online+resource+for+succinate+dehydrogenase+sequence+variants+involved+in+pheochromocytoma%2C+paraganglioma+and+mitochondrial+complex+II+deficiency.&amp;auth=&amp;atitle=The+SDH+mutation+database%3A+an+online+resource+for+succinate+dehydrogenase+sequence+variants+involved+in+pheochromocytoma%2C+paraganglioma+and+mitochondrial+complex+II+deficiency." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SDH+mutation+database%3A+an+online+resource+for+succinate+dehydrogenase+sequence+variants+involved+in+pheochromocytoma%2C+paraganglioma+and+mitochondrial+complex+II+deficiency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SDH+mutation+database%3A+an+online+resource+for+succinate+dehydrogenase+sequence+variants+involved+in+pheochromocytoma%2C+paraganglioma+and+mitochondrial+complex+II+deficiency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000436-Dahia1" id="pone.0000436-Dahia1"></a>Dahia PL, Ross KN, Wright ME, Hayashida CY, Santagata S, et al.  (2005) A HIF1alpha regulatory loop links hypoxia and mitochondrial signals in pheochromocytomas. PLoS Genet  1: 72–80.  <ul class="find" data-citedArticleID="950736" data-doi="10.1371/journal.pgen.0010008"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+HIF1alpha+regulatory+loop+links+hypoxia+and+mitochondrial+signals+in+pheochromocytomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+HIF1alpha+regulatory+loop+links+hypoxia+and+mitochondrial+signals+in+pheochromocytomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000436-Astrom1" id="pone.0000436-Astrom1"></a>Astrom K, Cohen JE, Willett-Brozick JE, Aston CE, Baysal BE (2003) Altitude is a phenotypic modifier in hereditary paraganglioma type 1: evidence for an oxygen-sensing defect. Hum Genet  113: 228–237.  <ul class="find" data-citedArticleID="950722" data-doi="10.1007/s00439-003-0969-6"><li><a href="http://dx.doi.org/10.1007/s00439-003-0969-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altitude+is+a+phenotypic+modifier+in+hereditary+paraganglioma+type+1%3A+evidence+for+an+oxygen-sensing+defect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Altitude+is+a+phenotypic+modifier+in+hereditary+paraganglioma+type+1%3A+evidence+for+an+oxygen-sensing+defect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000436-Piruat1" id="pone.0000436-Piruat1"></a>Piruat JI, Pintado CO, Ortega-Saenz P, Roche M, Lopez-Barneo J (2004) The mitochondrial SDHD gene is required for early embryogenesis, and its partial deficiency results in persistent carotid body glomus cell activation with full responsiveness to hypoxia. Mol Cell Biol  24: 10933–10940.  <ul class="find" data-citedArticleID="950752" data-doi="10.1128/mcb.24.24.10933-10940.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.24.10933-10940.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mitochondrial+SDHD+gene+is+required+for+early+embryogenesis%2C+and+its+partial+deficiency+results+in+persistent+carotid+body+glomus+cell+activation+with+full+responsiveness+to+hypoxia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mitochondrial+SDHD+gene+is+required+for+early+embryogenesis%2C+and+its+partial+deficiency+results+in+persistent+carotid+body+glomus+cell+activation+with+full+responsiveness+to+hypoxia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000436-Slane1" id="pone.0000436-Slane1"></a>Slane BG, Aykin-Burns N, Smith BJ, Kalen AL, Goswami PC, et al.  (2006) Mutation of succinate dehydrogenase subunit C results in increased O2.-, oxidative stress, and genomic instability. Cancer Res  66: 7615–7620.  <ul class="find" data-citedArticleID="950754" data-doi="10.1158/0008-5472.can-06-0833"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-0833" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+of+succinate+dehydrogenase+subunit+C+results+in+increased+O2.-%2C+oxidative+stress%2C+and+genomic+instability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+of+succinate+dehydrogenase+subunit+C+results+in+increased+O2.-%2C+oxidative+stress%2C+and+genomic+instability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000436-Gottlieb1" id="pone.0000436-Gottlieb1"></a>Gottlieb E, Tomlinson IP (2005) Mitochondrial tumour suppressors: a genetic and biochemical update. Nat Rev Cancer  5: 857–866.  <ul class="find" data-citedArticleID="950740" data-doi="10.1038/nrc1737"><li><a href="http://dx.doi.org/10.1038/nrc1737" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+tumour+suppressors%3A+a+genetic+and+biochemical+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+tumour+suppressors%3A+a+genetic+and+biochemical+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000436-Baysal2" id="pone.0000436-Baysal2"></a>Baysal BE (2005) Krebs cycle enzymes as tumor suppressors. Drug Discovery Today: Disease Mechanisms  2: 247–254.  <ul class="find" data-citedArticleID="950728" data-doi="10.1016/j.ddmec.2005.05.012"><li><a href="http://dx.doi.org/10.1016/j.ddmec.2005.05.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Krebs+cycle+enzymes+as+tumor+suppressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Krebs+cycle+enzymes+as+tumor+suppressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000436-Ling1" id="pone.0000436-Ling1"></a>Ling PD, Huls HM (2005)  Isolation and Immortalization of Lymphocytes. In: Ausubel FM, Brent R, Kingston RE, Moore DD, Seidman JG, Smith JA, Struhl K, editors. Current Protocols in Molecular Biology. New Jersey: John Wiley &amp; Sons. pp. 28.2.2–28.2.5.  <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0000436-Gabay1" id="pone.0000436-Gabay1"></a>Gabay C, Ben Bassat H, Schlesinger M, Laskov R (1999) Somatic mutations and intraclonal variations in the rearranged Vkappa genes of B-non-Hodgkin's lymphoma cell lines. Eur J Haematol  63: 180–191.  <ul class="find" data-citedArticleID="950738" data-doi="10.1111/j.1600-0609.1999.tb01766.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0609.1999.tb01766.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Somatic+mutations+and+intraclonal+variations+in+the+rearranged+Vkappa+genes+of+B-non-Hodgkin%27s+lymphoma+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Somatic+mutations+and+intraclonal+variations+in+the+rearranged+Vkappa+genes+of+B-non-Hodgkin%27s+lymphoma+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000436-Baysal3" id="pone.0000436-Baysal3"></a>Baysal BE, Willett-Brozick JE, Lawrence EC, Drovdlic CM, Savul SA, et al.  (2002) Prevalence of SDHB, SDHC, and SDHD germline mutations in clinic patients with head and neck paragangliomas. J Med Genet  39: 178–183.  <ul class="find" data-citedArticleID="950730" data-doi="10.1136/jmg.39.3.178"><li><a href="http://dx.doi.org/10.1136/jmg.39.3.178" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+SDHB%2C+SDHC%2C+and+SDHD+germline+mutations+in+clinic+patients+with+head+and+neck+paragangliomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+SDHB%2C+SDHC%2C+and+SDHD+germline+mutations+in+clinic+patients+with+head+and+neck+paragangliomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000436-Uejima1" id="pone.0000436-Uejima1"></a>Uejima H, Lee MP, Cui H, Feinberg AP (2000) Hot-stop PCR: a simple and general assay for linear quantitation of allele ratios. Nat Genet  25: 375–376.  <ul class="find" data-citedArticleID="950758" data-doi="10.1038/ng0501-97a"><li><a href="http://dx.doi.org/10.1038/ng0501-97a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hot-stop+PCR%3A+a+simple+and+general+assay+for+linear+quantitation+of+allele+ratios." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hot-stop+PCR%3A+a+simple+and+general+assay+for+linear+quantitation+of+allele+ratios.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000436-Arezi1" id="pone.0000436-Arezi1"></a>Arezi B, Hogrefe HH (2007) Escherichia coli DNA polymerase III epsilon subunit increases Moloney murine leukemia virus reverse transcriptase fidelity and accuracy of RT-PCR procedures. Anal Biochem  360: 84–91.  <ul class="find" data-citedArticleID="950720" data-doi="10.1016/j.ab.2006.10.009"><li><a href="http://dx.doi.org/10.1016/j.ab.2006.10.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+DNA+polymerase+III+epsilon+subunit+increases+Moloney+murine+leukemia+virus+reverse+transcriptase+fidelity+and+accuracy+of+RT-PCR+procedures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+DNA+polymerase+III+epsilon+subunit+increases+Moloney+murine+leukemia+virus+reverse+transcriptase+fidelity+and+accuracy+of+RT-PCR+procedures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000436-Albertini1" id="pone.0000436-Albertini1"></a>Albertini RJ, Nicklas JA, O'Neill JP, Robison SH (1990) In vivo somatic mutations in humans: measurement and analysis. Annu Rev Genet  24: 305–326.  <ul class="find" data-citedArticleID="950716" data-doi="10.1146/annurev.ge.24.120190.001513"><li><a href="http://dx.doi.org/10.1146/annurev.ge.24.120190.001513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+somatic+mutations+in+humans%3A+measurement+and+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+somatic+mutations+in+humans%3A+measurement+and+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000436-Mukhopadhyay1" id="pone.0000436-Mukhopadhyay1"></a>Mukhopadhyay D, Anant S, Lee RM, Kennedy S, Viskochil D, et al.  (2002) C→U editing of neurofibromatosis 1 mRNA occurs in tumors that express both the type II transcript and apobec-1, the catalytic subunit of the apolipoprotein B mRNA-editing enzyme. Am J Hum Genet  70: 38–50.  <ul class="find" data-citedArticleID="950746" data-doi="10.1086/337952"><li><a href="http://dx.doi.org/10.1086/337952" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=C%26%23x2192%3BU+editing+of+neurofibromatosis+1+mRNA+occurs+in+tumors+that+express+both+the+type+II+transcript+and+apobec-1%2C+the+catalytic+subunit+of+the+apolipoprotein+B+mRNA-editing+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22C%26%23x2192%3BU+editing+of+neurofibromatosis+1+mRNA+occurs+in+tumors+that+express+both+the+type+II+transcript+and+apobec-1%2C+the+catalytic+subunit+of+the+apolipoprotein+B+mRNA-editing+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000436-Honjo1" id="pone.0000436-Honjo1"></a>Honjo T, Kinoshita K, Muramatsu M (2002) Molecular mechanism of class switch recombination: linkage with somatic hypermutation. Annu Rev Immunol  20: 165–196.  <ul class="find" data-citedArticleID="950742" data-doi="10.1146/annurev.immunol.20.090501.112049"><li><a href="http://dx.doi.org/10.1146/annurev.immunol.20.090501.112049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanism+of+class+switch+recombination%3A+linkage+with+somatic+hypermutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanism+of+class+switch+recombination%3A+linkage+with+somatic+hypermutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000436-Wedekind1" id="pone.0000436-Wedekind1"></a>Wedekind JE, Dance GS, Sowden MP, Smith HC (2003) Messenger RNA editing in mammals: new members of the APOBEC family seeking roles in the family business. Trends Genet  19: 207–216.  <ul class="find" data-citedArticleID="950760" data-doi="10.1016/s0168-9525(03)00054-4"><li><a href="http://dx.doi.org/10.1016/s0168-9525(03)00054-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Messenger+RNA+editing+in+mammals%3A+new+members+of+the+APOBEC+family+seeking+roles+in+the+family+business." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Messenger+RNA+editing+in+mammals%3A+new+members+of+the+APOBEC+family+seeking+roles+in+the+family+business.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000436-Neuberger1" id="pone.0000436-Neuberger1"></a>Neuberger MS, Rada C (2007) Somatic hypermutation: activation-induced deaminase for C/G followed by polymerase eta for A/T. J Exp Med  204: 7–10.  <ul class="find" data-citedArticleID="950750" data-doi="10.1084/jem.20062409"><li><a href="http://dx.doi.org/10.1084/jem.20062409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Somatic+hypermutation%3A+activation-induced+deaminase+for+C%2FG+followed+by+polymerase+eta+for+A%2FT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Somatic+hypermutation%3A+activation-induced+deaminase+for+C%2FG+followed+by+polymerase+eta+for+A%2FT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000436-Amrani1" id="pone.0000436-Amrani1"></a>Amrani N, Sachs MS, Jacobson A (2006) Early nonsense: mRNA decay solves a translational problem. Nat Rev Mol Cell Biol  7: 415–425.  <ul class="find" data-citedArticleID="950718" data-doi="10.1038/nrm1942"><li><a href="http://dx.doi.org/10.1038/nrm1942" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+nonsense%3A+mRNA+decay+solves+a+translational+problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+nonsense%3A+mRNA+decay+solves+a+translational+problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000436-Baysal4" id="pone.0000436-Baysal4"></a>Baysal BE, Lawrence EC, Ferrell RE (2007) Sequence variation in human succinate dehydrogenase genes: evidence for long-term balancing selection on SDHA. BMC Biol  5: 12.  <ul class="find" data-citedArticleID="950732" data-doi="10.1186/1741-7007-5-12"><li><a href="http://dx.doi.org/10.1186/1741-7007-5-12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+variation+in+human+succinate+dehydrogenase+genes%3A+evidence+for+long-term+balancing+selection+on+SDHA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+variation+in+human+succinate+dehydrogenase+genes%3A+evidence+for+long-term+balancing+selection+on+SDHA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000436-Murdoch1" id="pone.0000436-Murdoch1"></a>Murdoch C, Giannoudis A, Lewis CE (2004) Mechanisms regulating the recruitment of macrophages into hypoxic areas of tumors and other ischemic tissues. Blood  104: 2224–2234.  <ul class="find" data-citedArticleID="950748" data-doi="10.1182/blood-2004-03-1109"><li><a href="http://dx.doi.org/10.1182/blood-2004-03-1109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+regulating+the+recruitment+of+macrophages+into+hypoxic+areas+of+tumors+and+other+ischemic+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+regulating+the+recruitment+of+macrophages+into+hypoxic+areas+of+tumors+and+other+ischemic+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000436-Stenmark1" id="pone.0000436-Stenmark1"></a>Stenmark KR, Davie NJ, Reeves JT, Frid MG (2005) Hypoxia, leukocytes, and the pulmonary circulation. J Appl Physiol  98: 715–721.  <ul class="find" data-citedArticleID="950756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hypoxia%2C+leukocytes%2C+and+the+pulmonary+circulation.&amp;auth=&amp;atitle=Hypoxia%2C+leukocytes%2C+and+the+pulmonary+circulation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hypoxia%2C+leukocytes%2C+and+the+pulmonary+circulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hypoxia%2C+leukocytes%2C+and+the+pulmonary+circulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.XML" value="86294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.PDF" value="434009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g001.PNG_L" value="1095754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g001.PNG_M" value="196391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g001.PNG_S" value="6183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g001.TIF" value="2059830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g001.PNG_I" value="72048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g002.PNG_L" value="22744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g002.PNG_M" value="23634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g002.PNG_S" value="2363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g002.TIF" value="67982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g002.PNG_I" value="10537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.t001.PNG_L" value="148694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.t001.PNG_M" value="34884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.t001.PNG_S" value="2453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.t001.TIF" value="713356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.t001.PNG_I" value="15721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g003.PNG_L" value="197996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g003.PNG_M" value="98053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g003.PNG_S" value="4342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g003.TIF" value="487220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g003.PNG_I" value="39238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g004.PNG_L" value="206528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g004.PNG_M" value="51976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g004.PNG_S" value="2827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g004.TIF" value="344656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.g004.PNG_I" value="21213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.s001.TIF" value="179054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.s002.TIF" value="87860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.s003.TIF" value="50164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.s004.TIF" value="53752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.s005.DOC" value="39424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000436.s006.DOC" value="38400"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000436&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000436" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000436&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000436&volume=&issue=&title=A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia&author_name=Bora%20E.%20Baysal&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000436" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000436" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000436" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000436&amp;t=A%20Recurrent%20Stop-Codon%20Mutation%20in%20Succinate%20Dehydrogenase%20Subunit%20B%20Gene%20in%20Normal%20Peripheral%20Blood%20and%20Childhood%20T-Cell%20Acute%20Leukemia" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000436&title=A%20Recurrent%20Stop-Codon%20Mutation%20in%20Succinate%20Dehydrogenase%20Subunit%20B%20Gene%20in%20Normal%20Peripheral%20Blood%20and%20Childhood%20T-Cell%20Acute%20Leukemia&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000436&amp;title=A%20Recurrent%20Stop-Codon%20Mutation%20in%20Succinate%20Dehydrogenase%20Subunit%20B%20Gene%20in%20Normal%20Peripheral%20Blood%20and%20Childhood%20T-Cell%20Acute%20Leukemia" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000436" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000436&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Recurrent Stop-Codon Mutation in Succinate Dehydrogenase Subunit B Gene in Normal Peripheral Blood and Childhood T-Cell Acute Leukemia';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000436';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Recurrent%20Stop-Codon%20Mutation%20in%20Succinate%20Dehydrogenase%20Subunit%20B%20Gene%20in%20Normal%20Peripheral%20Blood%20and%20Childhood%20T-Cell%20Acute%20Leukemia http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000436" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000436" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22B+cells%22" title="Search for articles in the subject area:'B cells'"><div class="flagText">B cells</div></a>
              <div data-categoryid="31979" data-articleid="23710"
                   data-categoryname="B cells"
                   class="flagImage" title="Flag 'B cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complementary+DNA%22" title="Search for articles in the subject area:'Complementary DNA'"><div class="flagText">Complementary DNA</div></a>
              <div data-categoryid="32053" data-articleid="23710"
                   data-categoryname="Complementary DNA"
                   class="flagImage" title="Flag 'Complementary DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="23710"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Monocytes%22" title="Search for articles in the subject area:'Monocytes'"><div class="flagText">Monocytes</div></a>
              <div data-categoryid="40523" data-articleid="23710"
                   data-categoryname="Monocytes"
                   class="flagImage" title="Flag 'Monocytes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23710"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23710"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="23710"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cells%22" title="Search for articles in the subject area:'T cells'"><div class="flagText">T cells</div></a>
              <div data-categoryid="40147" data-articleid="23710"
                   data-categoryname="T cells"
                   class="flagImage" title="Flag 'T cells' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1329'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6022'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1710&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
