

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001997"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001997" />

    <meta name="citation_title" content="Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images"/>
    <meta itemprop="name" content="Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images"/>

      <meta name="citation_author" content="Alfredo Rodriguez"/>
            <meta name="citation_author_institution" content="Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Laboratory of Biomathematics, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America"/>
      <meta name="citation_author" content="Douglas B. Ehlenberger"/>
            <meta name="citation_author_institution" content="Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Laboratory of Biomathematics, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America"/>
      <meta name="citation_author" content="Dara L. Dickstein"/>
            <meta name="citation_author_institution" content="Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America"/>
      <meta name="citation_author" content="Patrick R. Hof"/>
            <meta name="citation_author_institution" content="Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America"/>
      <meta name="citation_author" content="Susan L. Wearne"/>
            <meta name="citation_author_institution" content="Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Laboratory of Biomathematics, Mount Sinai School of Medicine, New York, New York, United States of America"/>
            <meta name="citation_author_institution" content="Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America"/>

    <meta name="citation_date" content="2008/4/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001997.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1997"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Rayburst sampling, an algorithm for automated three-dimensional shape analysis from laser-scanning microsopy images.; citation_author=A Rodriguez; citation_author=D Ehlenberger; citation_author=PR Hof; citation_author=SL Wearne; citation_journal_title=Nat Protocols; citation_volume=1; citation_number=1; citation_pages=2156-2161; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Morphological variations in the dendritic spines of the neocortex.; citation_author=EG Jones; citation_author=TPS Powell; citation_journal_title=J Cell Sci; citation_volume=5; citation_number=2; citation_pages=509-529; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=The small pyramidal neuron of the rat cerebral cortex. The perikaryon, dendrites and spines.; citation_author=A Peters; citation_author=IR Kaiserman-Abramof; citation_journal_title=Am J Anat; citation_volume=127; citation_number=3; citation_pages=321-355; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional structure of dendritic spines and synapses in rat hippocampus (CA1) at postnatal day 15 and adult ages: implications for the maturation of synaptic physiology and long-term potentiation.; citation_author=KM Harris; citation_author=FE Jensen; citation_author=B Tsao; citation_journal_title=J Neurosci; citation_volume=12; citation_number=4; citation_pages=2685-2705; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Structure and function of dendritic spines.; citation_author=EA Nimchinsky; citation_author=BL Sabatini; citation_author=K Svoboda; citation_journal_title=Annu Rev Physiol; citation_volume=64; citation_number=5; citation_pages=313-353; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Do thin spines learn to be mushroom spines that remember?; citation_author=J Bourne; citation_author=KM Harris; citation_journal_title=Curr Opin Neurobiol; citation_volume=17; citation_number=6; citation_pages=381-386; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Dendritic spines: structure, dynamics and regulation.; citation_author=H Hering; citation_author=M Sheng; citation_journal_title=Nat Rev Neurosci; citation_volume=2; citation_number=7; citation_pages=880-888; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Experience-dependent plasticity of dendritic spines in the developing rat barrel cortex in vivo.; citation_author=B Lendval; citation_author=EA Stern; citation_author=B Chen; citation_author=K Svoboda; citation_journal_title=Nature; citation_volume=404; citation_number=8; citation_pages=876-881; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The dynamics of dendritic structure in developing hippocampal slices.; citation_author=ME Dailey; citation_author=SJ Smith; citation_journal_title=J Neurosci; citation_volume=16; citation_number=9; citation_pages=2983-2994; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Rapid dendritic morphogenesis in CA1 hippocampal dendrites induced by synaptic activity.; citation_author=M Maletic-Savatic; citation_author=R Malinow; citation_author=K Svoboda; citation_journal_title=Science; citation_volume=283; citation_number=10; citation_pages=1923-1927; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Reversal and consolidation of activity-induced synaptic modifications.; citation_author=Q Zhou; citation_author=MM Poo; citation_journal_title=Trends Neurosci; citation_volume=27; citation_number=11; citation_pages=378-383; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Development of long-term dendritic spine stability in diverse regions of cerebral cortex.; citation_author=Y Zuo; citation_author=A Lin; citation_author=P Chang; citation_author=WB Gan; citation_journal_title=Neuron; citation_volume=46; citation_number=12; citation_pages=181-189; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Experience-dependent and cell-type-specific spine growth in the neocortex.; citation_author=A Holtmaat; citation_author=L Wilbrecht; citation_author=GW Knott; citation_author=E Welker; citation_author=K Svoboda; citation_journal_title=Nature; citation_volume=44; citation_number=13; citation_pages=979-983; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Long-term in vivo imaging of experience-dependent synaptic plasticity in adult cortex.; citation_author=JT Trachtenberg; citation_author=BE Chen; citation_author=GW Knott; citation_author=G Feng; citation_author=JR Sanes; citation_journal_title=Nature; citation_volume=420; citation_number=14; citation_pages=788-794; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structure-stability-function relationships of dendritic spines.; citation_author=H Kasai; citation_author=M Matsuzaki; citation_author=J Noguchi; citation_author=N Yasumatsu; citation_author=H Nakahara; citation_journal_title=Trends Neurosci; citation_volume=26; citation_number=15; citation_pages=360-368; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transient and persistent dendritic spines in the neocortex in vivo.; citation_author=A Holtmaat; citation_author=JT Trachtenberg; citation_author=L Wilbrecht; citation_author=GM Shepherd; citation_author=X Zhang; citation_journal_title=Neuron; citation_volume=45; citation_number=16; citation_pages=279-291; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Long-term dendritic spine stability in the adult cortex.; citation_author=J Grutzendler; citation_author=N Kasthuri; citation_author=WB Gan; citation_journal_title=Nature; citation_volume=420; citation_number=17; citation_pages=812-816; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Quantitative fine-structural analysis of olfactory cortical synapses.; citation_author=T Shikorski; citation_author=CF Stevens; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=96; citation_number=18; citation_pages=4107-4112; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cell type and pathway dependence of synaptic AMPA receptor number and variability in the hippocampus.; citation_author=Z Nusser; citation_author=R Lujan; citation_author=G Laube; citation_author=J Roberts; citation_author=E Molnar; citation_journal_title=Neuron; citation_volume=21; citation_number=19; citation_pages=545-559; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Morphological changes in dendritic spines associated with long-term synaptic plasticity.; citation_author=R Yuste; citation_author=T Bonhoeffer; citation_journal_title=Ann Rev Neurosci; citation_volume=24; citation_number=20; citation_pages=1071-1089; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ca2+-dependent regulation in neuronal gene expression.; citation_author=H Bito; citation_author=K Deisseroth; citation_author=RW Tsien; citation_journal_title=Curr Opin Neurobiol; citation_volume=7; citation_number=21; citation_pages=419-429; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Anatomical and physiological plasticity of dendritic spines.; citation_author=VA Alvarez; citation_author=BL Sabatini; citation_journal_title=Annu Rev Neurosci; citation_volume=30; citation_number=22; citation_pages=79-97; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Calcium dynamics of spines depend on their dendritic location.; citation_author=K Holthoff; citation_author=D Tsay; citation_author=R Yuste; citation_journal_title=Neuron; citation_volume=33; citation_number=23; citation_pages=425-437; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ca2+ signaling in dendritic spines.; citation_author=BL Bloodgood; citation_author=BL Sabatini; citation_journal_title=Curr Opin Neurobiol; citation_volume=17; citation_number=24; citation_pages=1-7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Regulation of spine calcium dynamics by rapid spine motility.; citation_author=A Majewska; citation_author=A Tashiro; citation_author=R Yuste; citation_journal_title=J Neurosci; citation_volume=20; citation_number=25; citation_pages=8262-8268; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=From form to function: calcium compartmentalization in dendritic spines.; citation_author=R Yuste; citation_author=A Majewska; citation_author=K Holthoff; citation_journal_title=Nat Neurosci; citation_volume=3; citation_number=26; citation_pages=653-659; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Direct measurement of coupling between dendritic spines and shafts.; citation_author=K Svoboda; citation_author=DW Tank; citation_author=W Denk; citation_journal_title=Science; citation_volume=272; citation_number=27; citation_pages=716-719; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Neuronal activity regulates diffusion across the neck of dendritic spines.; citation_author=BL Bloodgood; citation_author=BL Sabatini; citation_journal_title=Science; citation_volume=310; citation_number=28; citation_pages=866-869; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Anomalous diffusion in Purkinje cell dendrites caused by spines.; citation_author=F Santamaria; citation_author=S Wils; citation_author=E De Schutter; citation_author=GJ Augustine; citation_journal_title=Neuron; citation_volume=52; citation_number=29; citation_pages=635-648; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Neuronal vulnerability and informatics in human disease.; citation_author=FE Bloom; citation_author=WG Young; citation_author=EA Nimchinsky; citation_author=PR Hof; citation_author=JH Morrison; citation_title=Neuroinformatics - An Overview of the Human Brain Project; citation_number=30; citation_pages=83-123; citation_date=1997; citation_publisher=Lawrence Erlbaum; " />
      <meta name="citation_reference" content="citation_title=Neuron imaging with Neurolucida - A PC-based system for image combining microscropy.; citation_author=JR Glaser; citation_author=E Glaser; citation_journal_title=Comput Med Imaging Graphics; citation_volume=14; citation_number=31; citation_pages=307-317; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=New techniques for imaging, digitization and analysis of three-dimensional neuronal morphology on multiple scales.; citation_author=SL Wearne; citation_author=A Rodriguez; citation_author=D Ehlenberger; citation_author=AB Rocher; citation_author=SC Henderson; citation_journal_title=Neuroscience; citation_volume=136; citation_number=32; citation_pages=661-680; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rapid automated three-dimensional tracing of neurons from confocal image stacks.; citation_author=KA Al-Kofahi; citation_author=S Lasek; citation_author=DH Szarowski; citation_author=CJ Pace; citation_author=G Nagy; citation_journal_title=IEEE Trans Inform Tech Biomed; citation_volume=6; citation_number=33; citation_pages=171-187; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Computing skeletons in three dimensions.; citation_author=G Borgefors; citation_author=I Nystrom; citation_author=G Sanniti Di Baja; citation_journal_title=Pattern Recognition; citation_volume=32; citation_number=34; citation_pages=1225-1236; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Analysis of tubular structures in three-dimensional confocal images.; citation_author=GJ Streekstra; citation_author=J van Pelt; citation_journal_title=Network: Comput Neural Systems; citation_volume=13; citation_number=35; citation_pages=381-395; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=New methods for the computer-assisted 3-D reconstruction of neurons from confocal image stacks.; citation_author=S Schmitt; citation_author=JF Evers; citation_author=C Duch; citation_author=M Scholz; citation_author=K Obermayer; citation_journal_title=NeuroImage; citation_volume=23; citation_number=36; citation_pages=1283-1298; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Automated three-dimensional tracing of neurons in confocal and brightfield images.; citation_author=W He; citation_author=TA Hamilton; citation_author=AR Cohen; citation_author=TJ Holmes; citation_author=C Pace; citation_journal_title=Microsc Microanal; citation_volume=9; citation_number=37; citation_pages=296-310; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Algorithmic extraction of morphological statistics from electronic archives of neuroanatomy.; citation_author=R Scorcioni; citation_author=GA Ascoli; citation_title=IWANN 2001, Lecture Notes in Computer Science; citation_number=38; citation_pages=30-37; citation_date=2001; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Picture thresholding using an interative selection method.; citation_author=TW Ridler; citation_author=S Calvard; citation_journal_title=IEEE Trans Syst Man Cybern; citation_volume=SMC-8; citation_number=39; citation_pages=630-632; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Oct-trees and their use in representing three-dimensional objects.; citation_author=CL Jackins; citation_author=SL Tanimoto; citation_journal_title=Computer Graphics Image Process; citation_volume=14; citation_number=40; citation_pages=249-270; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=An image analysis algorithm for the fine structure of neuronal dendrites.; citation_author=IYY Koh; citation_author=WB Lindquist; citation_author=K Zito; citation_author=EA Nimchinsky; citation_author=K Svoboda; citation_journal_title=Neural Comput; citation_volume=14; citation_number=41; citation_pages=1283-1310; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Building skeleton models via 3-D medial surface/axis thinning algorithms.; citation_author=TC Lee; citation_author=RL Kashyap; citation_author=C Chu; citation_journal_title=CVGIP: Graph Models Image Process; citation_volume=56; citation_number=42; citation_pages=462-478; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Dendritic spine detection using curvilinear structure detector and LDA classifier.; citation_author=Y Zhang; citation_author=X Zhou; citation_author=RM Witt; citation_author=BL Sabatini; citation_author=D Adjeroh; citation_journal_title=NeuroImage; citation_volume=36; citation_number=43; citation_pages=346-360; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A novel computational approach for automatic dendrite spines detection in two-photon laser scan microscopy.; citation_author=J Cheng; citation_author=X Zhou; citation_author=E Miller; citation_author=RM Witt; citation_author=J Zhu; citation_journal_title=J Neurosci Methods; citation_volume=165; citation_number=44; citation_pages=122-134; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Age-related morphologic alterations in dendrites and spine densities of corticocortically projecting neurons in macaque monkeys.; citation_author=H Duan; citation_author=SL Wearne; citation_author=AB Rocher; citation_author=A Macedo; citation_author=JH Morrison; citation_journal_title=Cereb Cortex; citation_volume=13; citation_number=45; citation_pages=950-961; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Influence of dendritic structure on firing pattern in model neocortical neurons.; citation_author=ZF Mainen; citation_author=TJ Sejnowski; citation_journal_title=Nature; citation_volume=382; citation_number=46; citation_pages=363-366; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Effects of dendritic morphology on CA3 pyramidal cell electrophysiology: a simulation study.; citation_author=JL Krichmar; citation_author=SJ Nasuto; citation_author=R Scorcioni; citation_author=SD Washington; citation_author=GA Ascoli; citation_journal_title=Brain Res; citation_volume=941; citation_number=47; citation_pages=11-28; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The modeling of pyramidal neurons in the visual cortex.; citation_author=K Stratford; citation_author=A Mason; citation_author=A Larkman; citation_author=G Major; citation_author=J Jack; citation_title=The Computing Neuron; citation_number=48; citation_pages=296-321; citation_date=1989; citation_publisher=Addison-Wesley; " />
      <meta name="citation_reference" content="citation_title=Morphological alterations in neurons forming corticocortical projections in the neocortex of aged patas monkeys.; citation_author=TL Page; citation_author=M Einstein; citation_author=H Duan; citation_author=Y He; citation_author=T Flores; citation_journal_title=Neurosci Lett; citation_volume=317; citation_number=49; citation_pages=37-41; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001997" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images"/>
    <meta name="twitter:description" content="A fundamental challenge in understanding how dendritic spine morphology controls learning and memory has been quantifying three-dimensional (3D) spine shapes with sufficient precision to distinguish morphologic types, and sufficient throughput for robust statistical analysis. The necessity to analyze large volumetric data sets accurately, efficiently, and in true 3D has been a major bottleneck in deriving reliable relationships between altered neuronal function and changes in spine morphology. We introduce a novel system for automated detection, shape analysis and classification of dendritic spines from laser scanning microscopy (LSM) images that directly addresses these limitations. The system is more accurate, and at least an order of magnitude faster, than existing technologies. By operating fully in 3D the algorithm resolves spines that are undetectable with standard two-dimensional (2D) tools. Adaptive local thresholding, voxel clustering and Rayburst Sampling generate a profile of diameter estimates used to classify spines into morphologic types, while minimizing optical smear and quantization artifacts. The technique opens new horizons on the objective evaluation of spine changes with synaptic plasticity, normal development and aging, and with neurodegenerative disorders that impair cognitive function."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001997.g007"/>

  <meta property="og:title" content="Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001997" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8030'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7165'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1820&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001997">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001997" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001997&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alfredo Rodriguez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Laboratory of Biomathematics, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Douglas B. Ehlenberger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Laboratory of Biomathematics, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dara L. Dickstein, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrick R. Hof, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susan L. Wearne
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:susan.wearne@mssm.edu">susan.wearne@mssm.edu</a></p>

                <p>Affiliations:
                  Department of Neuroscience, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Laboratory of Biomathematics, Mount Sinai School of Medicine, New York, New York, United States of America, 
                  Computational Neurobiology and Imaging Center, Mount Sinai School of Medicine, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001997</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001997" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001997" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001997" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001997" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001997-g001" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-g002" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-g003" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-g004" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-g005" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-g006" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-t001" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-t002" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-t003" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-g007" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-t004" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-t005" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001997-t006" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001997">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001997" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>A fundamental challenge in understanding how dendritic spine morphology controls learning and memory has been quantifying three-dimensional (3D) spine shapes with sufficient precision to distinguish morphologic types, and sufficient throughput for robust statistical analysis. The necessity to analyze large volumetric data sets accurately, efficiently, and in true 3D has been a major bottleneck in deriving reliable relationships between altered neuronal function and changes in spine morphology. We introduce a novel system for automated detection, shape analysis and classification of dendritic spines from laser scanning microscopy (LSM) images that directly addresses these limitations. The system is more accurate, and at least an order of magnitude faster, than existing technologies. By operating fully in 3D the algorithm resolves spines that are undetectable with standard two-dimensional (2D) tools. Adaptive local thresholding, voxel clustering and Rayburst Sampling generate a profile of diameter estimates used to classify spines into morphologic types, while minimizing optical smear and quantization artifacts. The technique opens new horizons on the objective evaluation of spine changes with synaptic plasticity, normal development and aging, and with neurodegenerative disorders that impair cognitive function.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rodriguez A, Ehlenberger DB, Dickstein DL, Hof PR, Wearne SL (2008) Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images. PLoS ONE 3(4):
          e1997.
            doi:10.1371/journal.pone.0001997</p><p><strong>Editor: </strong>Bernado Sabatini, Harvard University, United States of America</p><p><strong>Received:</strong> October 9, 2007; <strong>Accepted:</strong> March 4, 2008; <strong>Published:</strong> April 23, 2008</p><p><strong>Copyright:</strong> © 2008 Rodriguez et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Supported by NIH grants MH71818 and DC05669 (SLW) and MH58911 and AG05138 (PRH).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Because of their capacity for plasticity and their key role as the location of excitatory synapses in the cerebral cortex, accurately characterizing the structure of dendritic spines is of profound biological significance. Spine morphology determines the strength, stability and function of excitatory synaptic connections that subserve the neuronal networks underlying cognitive function. Precise quantification of spine morphology in three dimensions (3D) is essential to understanding the structural determinants of normal neuronal function, its development, plasticity, and its dysfunction in neurodegenerative disorders. To date however, this level of precision has been restricted to time-intensive electron microscopy (EM) approaches with limited throughput, that are impractical for comparative population studies. In this paper we introduce a novel computational approach for detection and shape analysis of neuronal dendritic spines from confocal and multiphoton laser scanning microscopy (CLSM and MPLSM) images, that operates fully in 3D, and is faster and more accurate than existing semi-automated technologies. The algorithm is a module of our NeuronStudio software application <a href="#pone.0001997-Rodriguez1">[1]</a>, an integrated system for semi-automated digitization, morphometry and analysis of complex neuronal morphology at high resolution.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Spines come in multiple shapes and sizes, which subserve a diversity of function <a href="#pone.0001997-Jones1">[2]</a>–<a href="#pone.0001997-Bourne1">[6]</a>. Although spine shapes in fixed tissue form a continuum rather than distinct categories, broad subclasses (e.g., thin, stubby, mushroom) have traditionally been distinguished on the basis of morphology (see refs <a href="#pone.0001997-Peters1">[3]</a>, <a href="#pone.0001997-Hering1">[7]</a> for review). Spine morphology also varies dynamically in response to synaptic activity <a href="#pone.0001997-Lendval1">[8]</a>–<a href="#pone.0001997-Holtmaat1">[13]</a>. Smaller spines are less stable and more motile <a href="#pone.0001997-Trachtenberg1">[14]</a>–<a href="#pone.0001997-Holtmaat2">[16]</a>, and as a result, more plastic than large spines <a href="#pone.0001997-Grutzendler1">[17]</a>. Recent data indicate that the size and morphology of the spine head are correlated with numbers of docked presynaptic vesicles <a href="#pone.0001997-Shikorski1">[18]</a>, numbers of postsynaptic receptors <a href="#pone.0001997-Nusser1">[19]</a>, and hence with synaptic strength. From a biophysical viewpoint, these effects give rise to increased synaptic currents and reduced time constants for calcium compartmentalization in larger spine heads (see refs <a href="#pone.0001997-Nimchinsky1">[5]</a>, <a href="#pone.0001997-Yuste1">[20]</a> for review), modulating postsynaptic mechanisms that determine functions such as learning and memory <a href="#pone.0001997-Bito1">[21]</a>–<a href="#pone.0001997-Bloodgood1">[24]</a>. Spine neck length and diameter also affects diffusional coupling between dendrite and spine <a href="#pone.0001997-Majewska1">[25]</a>–<a href="#pone.0001997-Bloodgood2">[28]</a>, and spine density and shape regulate the degree of anomalous diffusion of chemical signals within the dendrite <a href="#pone.0001997-Santamaria1">[29]</a>. As more precise data from these studies emerges, the need for accurate spine morphometry in true 3D, and on large enough scales for robust statistical analyses becomes increasingly critical.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Digital representation of neuronal morphology using light microscopy has traditionally relied on manual tracing from a computer screen <a href="#pone.0001997-Bloom1">[30]</a>, <a href="#pone.0001997-Glaser1">[31]</a>, and is prone to subjective errors. Despite the recent introduction of semi-automated tracing methods (e.g., AutoNeuron, MBF Bioscience, Williston, VT), the problem of detecting and characterizing spine shapes automatically, in 3D, remains unsolved. In existing image analysis-based tools, precision at the finest scales is limited by the skeletonization methods used and by quantization errors. Even when imaged at the limits of CLSM resolution, single spines may span as few as 3–10 voxels, and neck diameters may be subvoxel resolution. Accurate recovery of spine geometry thus requires new analysis algorithms capable of using subvoxel information, and the full three dimensions of structural information from a LSM image stack. The spine detection and analysis method presented here is compatible with many methods for tracing the dendritic tree and thus can be implemented as an add-on module to existing manual or automated tracing packages. Spine voxels are clustered based on connectivity, analyzed with subvoxel precision using Rayburst-based shape analysis routines <a href="#pone.0001997-Rodriguez1">[1]</a>, and classified into three morphologic types, mushroom, stubby and thin. Procedures for declumping of merged spines and for combining detached spine heads with their stems are implemented to ensure accurate spine counts. The method is at least an order of magnitude faster than previous algorithms, and as a 64-bit application, can handle multi-gigabyte datasets. These algorithms have been tested and validated in our freely distributed software application, NeuronStudio <a href="#pone.0001997-Rodriguez1">[1]</a>, <a href="#pone.0001997-Wearne1">[32]</a> (<a href="http://www.mssm.edu/cnic">http://www.mssm.edu/cnic</a>), which provides visual verification of spine classification and the ability for manual spine editing through interactive 2D and 3D displays.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Image Segmentation and Voxel Processing</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The spine analysis module described here utilizes a previously computed model of the dendritic tree, comprising a series of nodes of specified diameter forming conical frusta (<a href="#pone-0001997-g001">Fig. 1A,B</a>), which could be obtained by any of several existing methods <a href="#pone.0001997-AlKofahi1">[33]</a>–<a href="#pone.0001997-He1">[37]</a>, and which is compatible with many existing neuronal morphometry applications <a href="#pone.0001997-Wearne1">[32]</a>, <a href="#pone.0001997-Scorcioni1">[38]</a>. Because fluorescence intensity can vary with adequacy of filling, imaging depth and XY spatial extent in CLSM and MPLSM image stacks, segmentation based on a globally selected intensity threshold is in general not feasible, and requires a dynamically adjusted, local threshold.</p>
<div class="figure" id="pone-0001997-g001"><div class="img"><a name="pone-0001997-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Processing of Candidate Voxels for Spine Detection.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A) Dendritic model rendered as green balls superimposed on volume-rendered data (light grey voxels), augmented by the maximum spine height (<em>MSH</em>, dark grey envelope surrounding dendrites that contains all spine candidate voxels). Four corners of the axis-aligned bounding box (AABB) are shown, with the length of each axis segment representing 10 µm to depict the scale in 3D. B) Close-up view of a spiny dendritic section, volume rendered in grayscale, with superimposed 3D model formed by sequential green frustra. A single capsule formed by a cylindrical dendritic segment capped by two hemispheres at consecutive nodes is outlined in blue. Spine candidate voxels V<sub>1</sub> and V<sub>2</sub> are shown with their corresponding distance to surface (<em>DTS</em>) values (red arrows). The dark grey envelope around the dendrite represents the <em>MSH</em>, measured from the surface of the 3D model, as indicated by the white dashed line. C) Octree calculated for the top fork of the dendritic section shown in (A). The root node is outlined in red. Recursive subdivision of the 3D space results in increasingly smaller cubes surrounding the model. The inset on the right illustrates bulk rejection of voxels in a single leaf (red shaded cube) that does not intersect the <em>MSH</em> zone (grey envelope surrounding model) created by a small dendritic section. D) Volume-rendered spiny dendrite with voxels representing exterior maxima drawn as red cubes (voxel size exaggerated to enhance visibility). The number of exterior maxima is usually much greater than the number of spines, since multiple maxima may occur on a single spine (white arrow), or along the surface of the dendrites (red arrow).</p>
<span>doi:10.1371/journal.pone.0001997.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Although the spine analysis method presented here is technically independent of the segmentation technique used, its performance will be enhanced as the efficacy of segmentation increases. In the case of threshold-based segmentation, faint or very thin spines may appear too small or be missed completely if the chosen threshold is too high. Likewise, too low a threshold will cause background noise to be segmented as part of the object, while spine shapes may appear larger, and distorted. For the current implementation we use an adaptation of the ISODATA method <a href="#pone.0001997-Ridler1">[39]</a> to compute a local threshold at each node along the dendritic model. This method is appropriate for datasets exhibiting a bimodal distribution of intensity values, such as the grayscale images characteristic of deconvolved LSM image stacks, placing the threshold midway between the centroids of the two peaks of the distribution. To ensure an adequate sample of spine and background voxels for the ISODATA distributions regardless of the orientation of the dendrite, at each node along the dendritic model we define a cubic section of data centered at the node and having X, Y, and Z dimensions equal to 2.5 times the node's diameter. Voxels inside that domain are tested for intersection with the dendritic model, and only the intensities of voxels that do not intersect the model are used to compute the ISODATA threshold. It should be noted that excluding voxels based on intersection with the model is a non-trivial computation, requiring optimization in order to avoid significant overhead.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Once each node has been assigned a threshold value, any voxel in the dataset may be segmented by linearly interpolating a local threshold value between nodes along the closest dendritic segment. We use the term <em>spine candidates</em> for voxels with intensity values at or above their local threshold value, whose distance to the closest point on the surface of the model (<em>distance to surface</em>: <em>DTS</em>) is less than or equal to a user-defined parameter, the <em>maximum spine height</em> (<em>MSH</em>) expected for the dataset. <a href="#pone-0001997-g001">Figure 1B</a> shows <em>DTS</em> values (red arrows) measured for two sample spine candidate voxels in the data (<em>V<sub>1</sub></em> and <em>V<sub>2</sub></em>).</p>


<h4>Octree Partitioning to Optimize Voxel Processing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Neurons are sparsely distributed within an image volume and spine candidate voxels occupy only a small fraction of the dataset. To identify and process spine candidate voxels efficiently we utilize a data structure known as an octree <a href="#pone.0001997-Jackins1">[40]</a>, which recursively organizes objects distributed inside a 3D volume, allowing fast searching by spatial location (<a href="#pone-0001997-g001">Fig. 1C</a>). We first define an axis-aligned bounding box (AABB) around the entire dendritic model, which becomes the root of the octree structure (outer cube drawn in red, <a href="#pone-0001997-g001">Fig. 1C</a>), and contains references to all dendritic segments. An AABB is a bounding volume used to optimize the computation of intersection tests in 3D. It comprises a rectangular box having faces aligned with the major axes rather than with the geometry of the object (Wikipedia article at: <a href="http://en.wikipedia.org/wiki/Bounding_volume">http://en.wikipedia.org/wiki/Bounding_vo​lume</a>). The root AABB is subsequently partitioned into eight octants that are each further subdivided, recursively (inset, <a href="#pone-0001997-g001">Fig. 1C</a>). Each octant becomes a child of its parent in the resulting octree structure. At successive subdivisions, the dendritic segments in the parent are tested for intersection with each of its children. Each child then receives a reference only to those dendritic segments that intersect it, and therefore receives a subset of the segments in the parent. Children with less than a pre-specified number of references are not divided further. These are termed the leaves of the octree.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The octree improves execution in three ways: (i) by allowing bulk rejection of voxels whose <em>DTS</em> exceeds the <em>MSH</em> (e.g., the red shaded cube in inset, <a href="#pone-0001997-g001">Fig. 1C</a>); (ii) by allowing rapid identification of model segments within the neighborhood of each spine candidate voxel; (iii) by rapid rejection of voxels failing to meet a minimum threshold established by the dendritic segments intersecting that leaf.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Before partitioning, the radii of the model's dendritic segments are augmented by the <em>MSH</em>, creating an envelope around the dendrites that includes all candidate spine voxels (dark grey envelope surrounding the model, <a href="#pone-0001997-g001">Fig. 1A–C</a>). Sections of the imaged volume that do not intersect these augmented segments are not divided further and voxels inside these sections can be excluded from our list of candidates (inset, <a href="#pone-0001997-g001">Fig. 1C</a>). Without the octree optimizations, the computational expense of the algorithm would be O(<em>M</em>×<em>N</em>), where <em>M</em> is the number of voxels in the data, and <em>N</em> is the number of edges in the model. As an example, a dataset where the spine analysis took 22 seconds with the octree in place, required over five minutes to process with the octree disabled.</p>


<h4>Spine Detection Using Voxel Clustering</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Individual spines are detected by clustering candidate spine voxels, starting from the tips and moving towards the dendrite. Voxels lacking 26-connected candidate neighbors with a greater <em>DTS</em> value are termed <em>exterior maxima</em>, and represent local high points that can occur at the tips of spines as well as surface irregularities and imaging noise along the surface of the dendrite (red voxels in <a href="#pone-0001997-g001">Fig. 1D</a>). Exterior maxima are processed in decreasing <em>DTS</em> order to ensure that each cluster uses the highest maximum. The cluster-building algorithm can be described as an iterative 3D flood-fill of the structure starting from an exterior maximum. Each iteration builds a layer by establishing a floor value that limits cluster growth towards the dendritic segment but does not constrain sideways growth for that iteration (<a href="#pone-0001997-g002">Fig. 2</a>). Successive layers are created by adding all 26-connected neighbors of the previous layer, and establishing the minimum <em>DTS</em> of these voxels as the floor value for the layer. All unvisited voxels connected to the current layer with a <em>DTS</em> greater than or equal to the floor value, are then added in a number of subiterations. For the first layer, the exterior maximum and its immediate neighbors establish its floor. Pseudo-code implementing an optimized version of the cluster building algorithm is provided in <a href="#pone.0001997.s001">Box S1</a> (Supporting Information, online).</p>
<div class="figure" id="pone-0001997-g002"><div class="img"><a name="pone-0001997-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Cluster-Building by Iterative Addition of Layers.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Individual voxels are rendered as wireframe cubes in successive layers of different colors, superimposed on a volume-rendered dendritic spine. A) Voxels of first layer shown in red. White frame shows each corner of the layer's AABB. B) Fourth iteration of the cluster-building algorithm produces the green layer, and correspondingly larger bounding box. C) In the last iteration, the diagonal of the bounding box exceeds the user-provided maximum spine width (<em>MSW</em>), as the green layer floods into the dendrite (green arrow). The yellow layer is the first to exhibit a spread ratio exceeding <em>SR</em><sub>(crit)</sub> (yellow arrow). Hence, the layer immediately preceding this becomes the spine base (orange layer, orange arrow). Scale bar shown as red horizontal edge of lower left corner of AABB represents 0.45 µm. D) Schematic of cluster layer-building, showing exterior maximum (red cube at maximum point); floor of layer 6 (proximal end of purple layer) and depth of layer 6 (distance from exterior maximum to floor).</p>
<span>doi:10.1371/journal.pone.0001997.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As voxels are added to individual layers, we maintain an AABB for the layer by updating the minimum and maximum voxel coordinates in all three dimensions (white bounding box, <a href="#pone-0001997-g002">Fig. 2A–C</a>). We estimate the size of the layer continuously as voxels are added by computing the diagonal of the bounding box, which we term the <em>spread of the layer</em>. If at any point during layer-building the spread exceeds a user-provided <em>maximum spine width</em>, the cluster building stops (see green layer flooding into dendrite, <a href="#pone-0001997-g002">Fig. 2C</a>).</p>


<h4>Calculating Spine Profiles</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>For each layer built in a particular cluster, we maintain a profile of measures for later use by the shape classification routine. We first compute the center of mass of each layer and run the 2D variant of our Rayburst Sampling algorithm <a href="#pone.0001997-Rodriguez1">[1]</a> from that center of mass to estimate the diameter at that layer. The <em>Rayburst diameter</em> is the diameter obtained by measuring the minimum surface-to-surface span inside a tubular structure (see Rodriguez et al. <a href="#pone.0001997-Rodriguez1">[1]</a> for full mathematical details). For structures assumed to have an approximately radially symmetric cross section, 2D Rayburst run in the XY plane is insensitive to residual Z axis smear from incomplete deconvolution, yielding a reliable estimate of the structure's diameter irrespective of its orientation within the image stack. The spine profile contains three values: the <em>spread of the layer</em>; the <em>Rayburst diameter</em>, as defined above, and the <em>depth of the layer</em>, computed as the distance from the exterior maximum to the floor of that layer (<a href="#pone-0001997-g002">Fig. 2D</a>). The base of the spine is the last layer in the cluster that should be considered part of the spine. For detached clusters the algorithm produces a final layer containing no voxels. This layer is always considered the base and its depth is set to the <em>DTS</em> of the maximum, since no floor was defined. To determine the base of an attached cluster we search its layers sequentially, from first to last, for the first layer that displays a sudden increase in spread (e.g., <a href="#pone-0001997-g002">Fig. 2C</a>, yellow arrow, yellow layer). The base is the layer immediately preceding this (e.g., <a href="#pone-0001997-g002">Fig. 2C</a>, orange arrow, orange layer). We quantify this spread increase, for each layer <em>i</em> in a cluster, in terms of the <em>spread ratio</em> (<em>SR</em>), given by the expression:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>S<sub>i</sub></em> is the length of the diagonal of the AABB encompassing all voxels in the <em>i</em>th (current) layer; <em>S<sub>j</sub></em> is the length of the diagonal of the AABB encompassing all voxels in the <em>j</em>th layer, for all <em>j</em>≤<em>i</em>. The layer preceding the first layer encountered with an <em>SR</em> value greater than a critical value <em>SR<sub>(crit)</sub></em> is selected as the base. An empirically determined value of <em>SR</em><sub>(crit)</sub> = 1.5 was optimal for the data analyzed in the present study. All layers in the cluster after the base are then discarded, and their voxels may later enter into another cluster. For attached clusters, the last layer has a spread larger that the maximum spine width allowed (final green layer, <a href="#pone-0001997-g002">Fig. 2C</a>), therefore the spread and Rayburst diameter are set to infinity to ensure that this layer is never selected as the base. Once the base layer is identified, we compute the aspect ratio (<em>AR</em>) of the cluster, defined as the ratio of the depth to the spread of the base layer (<a href="#pone-0001997-g002">Fig. 2D</a>). Clusters with low aspect ratio values (less than 0.25 in our empirical datasets) represent dendritic surface irregularities that are too flat to be considered spines, and can be safely discarded. Clusters may also be discarded based on user-supplied parameters for minimum voxel count and minimum spine height (depth of base layer). All remaining clusters are classified as spines.</p>


<h4>Cluster Declumping</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Cluster shape is calculated in 3D and used both to remove non-spine clusters and to classify spines into types. Inadequate image resolution, inaccurate thresholding, or sheer physical proximity can cause adjacent spines to appear merged (e.g., <a href="#pone-0001997-g003">Fig. 3A</a>). We introduce a declumping method that effectively separates merged spines using intensity gradient information. In spiny dendrites imaged with CLSM or MPLSM, apparently merged spines are relatively common, making cluster declumping an essential precursor to spine shape analysis (<a href="#pone-0001997-g003">Fig. 3B</a>). When two spines are in contact, layers built from the exterior maximum of the first spine may extend into the adjoining spine. Voxel intensities are naturally brighter at the center of spines and dimmer at the edges, forming an intensity landscape that can be used to delimit individual hills (spines) and valleys (edges) (<a href="#pone-0001997-g003">Fig. 3C</a>). Declumping uses intensity gradients, computed as 3D vectors at each spine voxel, to detect the brighter centers during the layer building (red and blue vectors, <a href="#pone-0001997-g003">Fig. 3C</a>). As each layer is built, the center of mass of the voxels used to establish the layer's floor is calculated. For the first layer, we compute the center of mass of the exterior maximum and its immediate neighbors. We then define a <em>layer attachment line</em> from this center of mass to the closest point along the medial axis of the dendritic tree. To prevent the layer from crossing an intensity valley into an adjacent spine, only those voxels whose intensity gradient points towards the layer attachment line are considered for inclusion in the spine layer.</p>
<div class="figure" id="pone-0001997-g003"><div class="img"><a name="pone-0001997-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Cluster Declumping and Spine Stem Reattachment.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A) Volumetric data showing cluster of three spines that appear merged due to limited image resolution. B) Result of clustering algorithm using the cluster declumping routine described in the text. The merged spines have been properly detected as separate clusters indicated by differently colored voxels. C) Local 3D gradient vectors used to declump two merged spines. Vector heads are colored red or blue; black tails point in direction of increasing intensity. At the valley marked by the white arrow, the gradients reverse direction: red gradients point leftward, toward the center of mass of the red spine; blue gradients point rightward, toward the center of mass of the blue spine. D) Volumetric data showing an apparently detached spine head, and its spine stem attached to the dendrite. Because of poor resolution of the spine neck, this thin spine is detected as two separate spines. E) Bell-shaped region (transparent red) used to detect the stem of a spine whose neck cannot be adequately resolved. The voxels of the attached (red) and detached (yellow) spines are represented as cubes within a volume rendering of the dataset. Because the tip of the attached spine falls within the bell-shaped region, the algorithm merges these two clusters into a single spine. White line between yellow detached and red attached spines represents an approximate scale of 0.8 µm. F) Schematic showing parameters of the spine stem reattachment routine. <em>p<sub>0</sub></em>: lowest-<em>DTS</em> voxel on a detached spine; <em>p<sub>1</sub></em>: closest point to surface of the dendritic model: <em>m</em>: maximum point on an attached spine within the bell-shaped domain shown in transparent red; <em>p<sub>m</sub></em>: projection of <em>m</em> onto line segment [<em>p<sub>0</sub></em>, <em>p<sub>1</sub></em>]; <em>D<sub>max</sub></em><sub>:</sub>: limit of the bell-shaped domain that encloses the spine stem.</p>
<span>doi:10.1371/journal.pone.0001997.g003</span></div>

<h4>Spine Shape Classification</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>After clustering, spine shapes are classified into three types, mushroom, stubby and thin, using the profile of 2D Rayburst diameters computed in consecutive layers along the length of the spine. Use of 2D Rayburst within each layer avoids the effects of residual optical smearing in the Z direction that arise from incomplete deconvolution <a href="#pone.0001997-Rodriguez1">[1]</a>, providing a reliable, high resolution profile of spine shape. <a href="#pone-0001997-g004">Figure 4A,B</a> shows a 2D Rayburst run from the center of mass of the third layer in a typical mushroom spine. The blue horizontal lines superimposed on the red Rayburst vectors in the XY view (<a href="#pone-0001997-g004">Fig. 4A</a>) and ZY view (<a href="#pone-0001997-g004">Fig. 4B</a>) indicate the resulting diameter in layer 3. Consecutive Rayburst diameters are plotted in <a href="#pone-0001997-g004">Figure 4C</a> versus layer number as bar graphs. These profiles of Rayburst diameters are used to classify spine shapes. Spine shape classification is controlled by critical values of the following three parameters: <em>AR</em> (defined in <em>Calculating Spine Profiles</em>), head to neck ratio (<em>HNR</em>) and head diameter (<em>HD</em>). These three parameters, together with the <em>spread ratio</em> defined in Equation (1), control the decision tree of <a href="#pone-0001997-g005">Figure 5</a> that classifies spine shapes. First the s<em>pread ratio</em> is used to determine the base of the spine. Then the <em>AR</em> is computed, and clusters are separated into valid spines (those that are sufficiently tall and narrow; <em>AR</em>&gt;<em>AR_spine</em><sub>(<em>crit</em>)</sub>) or invalid spines (those that are too wide and flat). For valid spines, existence of a neck is the first decision point in the scheme. The measurements <em>HNR</em>, <em>neck_position</em> and <em>head_position</em> are returned by the following algorithm, which iterates sequentially through the layers of the spine profile:</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p><strong>SET</strong> HNR to 0</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p><strong>SET</strong> neck_position to 0</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p><strong>SET</strong> head_position to 0</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p><strong>FOR</strong> every layer i in the profile</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p><strong>FOR</strong> every layer j &lt; i</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p><strong>SET</strong> r to rayburst_diameter[j] / rayburst_diameter[i]</p>
<a id="article1.body1.sec2.sec6.p8" name="article1.body1.sec2.sec6.p8"></a><p><strong>IF</strong> r &gt; HNR <strong>THEN</strong></p>
<a id="article1.body1.sec2.sec6.p9" name="article1.body1.sec2.sec6.p9"></a><p><strong>SET</strong> HNR to r</p>
<a id="article1.body1.sec2.sec6.p10" name="article1.body1.sec2.sec6.p10"></a><p><strong>SET</strong> neck_position to i</p>
<a id="article1.body1.sec2.sec6.p11" name="article1.body1.sec2.sec6.p11"></a><p><strong>SET</strong> head_position to j</p>
<a id="article1.body1.sec2.sec6.p12" name="article1.body1.sec2.sec6.p12"></a><p><strong>END IF</strong></p>
<a id="article1.body1.sec2.sec6.p13" name="article1.body1.sec2.sec6.p13"></a><p><strong>END FOR</strong></p>
<a id="article1.body1.sec2.sec6.p14" name="article1.body1.sec2.sec6.p14"></a><p><strong>END FOR</strong></p>
<div class="figure" id="pone-0001997-g004"><div class="img"><a name="pone-0001997-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Shape Classification Using Rayburst Diameters of Consecutive Layers.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>A) XY view showing the rays (red lines) of a 2D Rayburst run at the center of mass (green square) of a single layer. The thick blue line indicates the resulting width of the structure as calculated by Rayburst, and provides an approximate scale of 0.7 µm. B) Side profile of the Rayburst core, demonstrating how the rays extend in the XY plane only, avoiding the effects of optical smearing in the Z direction. C) Bar graphs showing the blue Rayburst diameters calculated at each layer as a function of the layer number, for three representative spines of type, mushroom, thin and stubby. These profiles of Rayburst diameters are used to classify spine types. <em>HD</em>: head diameter; <em>ND</em>: neck diameter. For attached spines, the last layer's Rayburst diameter (black bar) is set to infinity.</p>
<span>doi:10.1371/journal.pone.0001997.g004</span></div><div class="figure" id="pone-0001997-g005"><div class="img"><a name="pone-0001997-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Flowchart of Algorithm Used for Spine Classification.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>The spread profile of the cluster is used to determine the base of the spine. From this computation an aspect ratio (<em>AR</em>) is calculated for the cluster to determine whether a spine has been found. For valid spines, the presence of a neck indicates a mushroom or thin spine, whereas absence of a neck indicates a stubby or thin spine. The aspect ratio and the width of the head (head diameter; <em>HD</em>) are used to resolve the final spine types.</p>
<span>doi:10.1371/journal.pone.0001997.g005</span></div><a id="article1.body1.sec2.sec6.p15" name="article1.body1.sec2.sec6.p15"></a><p>Spines with <em>HNR</em> greater than a critical value, <em>HNR</em><sub>(<em>crit</em>)</sub>, are considered to have a neck. For detached spines, the above algorithm is not needed since we simply select the last layer as the neck, and the largest layer before the neck as the head. Spines with a neck can be either thin or mushroom types. If any layer above the neck has a Rayburst diameter exceeding a critical value (<em>HD</em><sub>(crit)</sub>), the spine is classified as a mushroom, otherwise it is a thin (see flowchart, <a href="#pone-0001997-g005">Fig. 5</a>). For spines lacking significant necks, an aspect ratio less than <em>AR_thin</em><sub>(<em>crit</em>)</sub> indicates a stubby, otherwise the spine is labeled as thin. By optimizing against manually classified data, we empirically determined <em>AR_thin</em><sub>(<em>crit</em>)</sub> to be 2.5; <em>HNR</em><sub>(<em>crit</em>)</sub> to be 1.1 and <em>HD</em><sub>(<em>crit</em>)</sub> to be 0.35 µm for the data analyzed in this study.</p>


<h4>Spine Stem Reattachment</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Extremely thin necks that occur on some spines cannot be resolved under LSM, leading to apparently detached spine heads (e.g., <a href="#pone-0001997-g003">Fig. 3D</a>). The end of the neck closest to the dendrite, termed the <em>spine stem</em>, may protrude far enough for it to be detected as a cluster and classified as a separate spine (red voxels, <a href="#pone-0001997-g003">Fig. 3E</a>), artificially increasing spine counts. Before spine counting and classification, we search for potential spine stems by determining if any attached spines without necks are located directly below a detached spine. Given the location of the lowest-<em>DTS</em> voxel in a detached spine, <em>p<sub>0</sub></em>, we compute the closest point on the surface of the dendritic model, <em>p<sub>1</sub></em>, (<a href="#pone-0001997-g003">Fig. 3F</a>). These two points define a line, <em>L</em>. For any attached spine in a prespecified neighborhood of <em>p<sub>0</sub></em>, we project its maximum voxel, <em>m</em>, onto <em>L</em>. The attached spine is identified as the stem if the projection point, <em>p<sub>m</sub></em>, lies within the line segment [<em>p<sub>0</sub></em>, <em>p</em><sub>1</sub>], and its perpendicular distance to the line is less than the value <em>D</em><sub>max</sub> given by the expression for a bell-shaped domain (<a href="#pone-0001997-g003">Fig. 3F</a>):<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Here <em>r<sub>bell</sub></em> represents the maximum radius at the base of a bell-shaped domain around the line segment [<em>p</em><sub>0</sub>, <em>p</em><sub>1</sub>] in micrometers, obtained empirically for our data. If more than one attached spine meets these criteria, the spine with the smallest perpendicular distance is chosen as the stem, and the two clusters are combined into a single spine.</p>


<h4>Validation of Automated Spine Detection and Shape Classification</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>We validated the detection and classification algorithms by comparison with trained human operators on the same data. Seven CLSM image stacks, each containing at least one spiny dendritic segment such as that shown in <a href="#pone-0001997-g006">Figure 6</a>, were collected according to the procedure described in <a href="#s4">Methods</a>. NeuronStudio was used to detect and classify spines automatically in these data, and the results were compared to those obtained independently by four skilled operators using manual methods. Because the problem of classifying spine shapes is independent of the detection method used, we evaluate these two processes separately. <a href="#pone-0001997-g006">Figure 6</a> shows XY and ZY maximal projections of approximately half the field of view of a typical image stack. Before deconvolution (<a href="#pone-0001997-g006">Fig. 6A</a>), the spines are visible in XY, but substantially obscured by the brighter dendrite in the ZY view. Following deconvolution (<a href="#pone-0001997-g006">Fig. 6B</a>), relative intensities of spines and dendrite are closer, and individual spines are clearly discernible in the ZY projection (<a href="#pone-0001997-g006">Fig. 6B</a>).</p>
<div class="figure" id="pone-0001997-g006"><div class="img"><a name="pone-0001997-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Manual and Automated Results for Typical Image Data.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>A,B) XY and ZY maximal projections of half the field of view of a typical image stack before (A) and after (B) deconvolution. Compared to the raw data (A), the deconvolved data exhibit good relative intensity equalization of spines and dendrites, and significantly reduced Z-axis “stretching” from optical smear, in the ZY projection (B). Note the difference in scale in XY vs ZY projections, reflecting the fact that voxel dimensions are twice as big in Z as in X and Y; (voxels are [0.05 µm, 0.05 µm, 0.1 µm]). C, D) Comparison of spines detected by one of the manual operators (red circles, C) and automatically by NeuronStudio (green circles, D). E,F) Close-up XY view of a spiny branch comparing spines detected manually (red circles, E) and automatically (green circles, D). Examples of typical mismatches between the manual and automatic detection methods are highlighted with double circles (double red circles: Manual-only detection; double green circles: <em>NS-only</em> detection), and by white arrows (misses). The short, stubby spine projecting downward from the dendrite is missed by the manual operator (white arrow, E), but detected by NeuronStudio (double green circles, F) A very faint, thin spine that is missed by NeuronStudio (white arrow, F) is detected by the manual operator only (double red circles, E). G,H) Close-up ZY view of a spiny branch comparing manual with automated spine detection. Example of a stubby spine projecting in the Z direction, that is typically missed by the manual operator (white arrow, G) but detected by <em>NS-only</em> (double green circles, H).</p>
<span>doi:10.1371/journal.pone.0001997.g006</span></div>

<h4>Comparison of Automated and Manual Spine Detection</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>A skilled operator examined each of the confocal image stacks interactively, first using a 2D slice viewer and then a 3D volume rendering. In the manual procedure, the operator labeled each identified spine with a marker labeled ‘manual’. NeuronStudio then identified spines in the same dataset automatically, placing a marker labeled ‘auto’ at the center of mass of each spine. Automatically and manually detected spines were compared on a spine-by-spine basis and the counts for both methods, for the manual method only, or for NeuronStudio only, were tabulated for each stack (<a href="#pone-0001997-t001">Table 1</a>). Across the seven image stacks, the manual operator detected 447 spines, and NeuronStudio detected 492. Of these 492, 401 were detected by both the manual and automated methods.</p>
<div class="figure" id="pone-0001997-t001"><div class="img"><a name="pone-0001997-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Validation of Automated Spine Detection.</span></strong></p><span>doi:10.1371/journal.pone.0001997.t001</span></div><a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>Spines detected by both manual and automated methods were considered correct hits. Spines detected by the manual method only (<em>Manual-only</em>, <a href="#pone-0001997-t001">Table 1</a>), or by NeuronStudio only (<em>NS-only</em>, <a href="#pone-0001997-t001">Table 1</a>), were potential errors. Those detected by <em>NS-only</em> could be either false positives, or correct detections that were missed by the human operator. Examples of spines detected manually (red circles), by NeuronStudio (green circles), <em>Manual-only</em> (double red circles) and <em>NS-only</em> (double green circles) are shown in XY and ZY projections of typical image stacks, in <a href="#pone-0001997-g006">Figure 6</a>. <em>NS-only</em> and <em>Manual-only</em> spines were subsequently analyzed by another three independent human operators. This showed that most <em>NS-only</em> spines were small, flat stubby types, or short thin types, projecting primarily above and below the dendrite, along the Z direction where they are more difficult for a human operator to distinguish (e.g., white arrow, <a href="#pone-0001997-g006">Fig. 6G</a> and double green circles, <a href="#pone-0001997-g006">Fig. 6H</a>). A small percentage of <em>NS-only</em> spines were judged actual false positives – dendritic surface bumps too small to be considered actual spines or non-reattached spine stems. <em>Manual-only</em> spines represent potential misses by NeuronStudio. Subsequent examination of M<em>anual-only</em> spines showed that about half were from clusters of two or three adjacent spines that did not declump completely. Of the remainder, about half were very dim voxel clusters, below the local threshold established by NeuronStudio, but just detectable by the human operator (e.g., double red circle, <a href="#pone-0001997-g006">Fig. 6E</a> and white arrow, <a href="#pone-0001997-g006">Fig. 6F</a>), while the rest were judged actual false positives.</p>


<h4>Comparison of Automated and Manual Spine Shape Classification</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Following detection, spine shapes were classified by NeuronStudio. Three skilled human operators (A, B, and C) then examined the same set of spines in 3D with default markers superimposed on the volume rendered data, using NeuronStudio interactively to rotate, zoom, and inspect the data from any angle. The three operators classified each spine independently by setting the default marker type to ‘stubby’, ‘thin’, or ‘mushroom’, according to the observed spine shape and a set of previously agreed classification criteria. A <em>human consensus</em> (HC) classification was also established as the type designated by two or more of the three human operators for each spine. To assess intra-operator variability, one operator (Operator C) performed the classification twice, evaluating the same datasets on different days. After adjusting NeuronStudio's shape classification parameters to fit the HC, spine types were classified automatically. <a href="#pone-0001997-t002">Table 2</a> shows the type counts for each operator, and the HC and NeuronStudio classifications. For each spine type, variability between operators was evaluated by measuring the percent match between each pair of operators on a spine-by-spine basis. All pairwise percent matches on the same 442 detected spines are shown in <a href="#pone-0001997-t003">Table 3</a> (higher percent match indicates lower variability between operators).</p>
<div class="figure" id="pone-0001997-t002"><div class="img"><a name="pone-0001997-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Table of Spine Classification Counts by Morphologic Type.</span></strong></p><span>doi:10.1371/journal.pone.0001997.t002</span></div><div class="figure" id="pone-0001997-t003"><div class="img"><a name="pone-0001997-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Intra- and Inter-Operator Percent Match in Shape Classification by Morphologic Type.</span></strong></p><span>doi:10.1371/journal.pone.0001997.t003</span></div><a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2"></a><p>Because many spine types are ambiguous, shape classification by human operators is subjective and prone to high variability both within an operator on the same spines (intra-operator variability), and between trained operators using the same classification criteria (inter-operator variability). Pairwise percent match between operators ranged from 78.3% to 85.8% (<a href="#pone-0001997-t003">Table 3</a>). Percent match within human operator C on two different days was 82.9% (C<sub>1</sub>–C<sub>2</sub>, <a href="#pone-0001997-t003">Table 3</a>). NeuronStudio's classification matched the HC classification 85.8% of the time after adjusting the shape parameters (<a href="#pone-0001997-t003">Table 3</a>). Within each spine class, NeuronStudio matched 79.1% of mushroom spines; 82.8% of stubby and 92.1% of thin spines classified by the human consensus. In general NeuronStudio matched the HC standard equally, or better than, the best inter-human match rates (Operator A to Operator B, 85.8% overall), and better than the intra-operator match (Operator C<sub>1</sub>–C<sub>2</sub>, 82.9% overall, <a href="#pone-0001997-t003">Table 3</a>). Automated classification by NeuronStudio has the advantage of removing human subjectivity and intra-operator variability, and the parameters can be optimized to match a particular classification criterion.</p>


<h4>Symmetry of Detection and Classification</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>To evaluate the algorithm's performance on spines of different orientations with respect to the image plane and around the dendritic structure, for each spine we record the angle formed by the spine's primary axis with the image plane. Spines were divided into two groups based on this angle: <em>mostly horizontal</em>, those with absolute values of angle≤45 degrees from the image plane, and <em>mostly vertical</em>, those with absolute values of angles&gt;45 degrees from the plane. For dendrites running approximately parallel with the image plane, the <em>mostly horizontal</em> spines are visible on the sides of the dendrite, in the XY view (e.g., <a href="#pone-0001997-g006">Fig. 6E,F</a> and <a href="#pone-0001997-g007">Fig. 7A,B,C</a>). The <em>mostly vertical</em> spines tend to project orthogonal to the image plane, being more visible in the ZX and ZY views (e.g., <a href="#pone-0001997-g006">Fig. 6G,H</a> and spines marked 1,2, and 3 in <a href="#pone-0001997-g007">Fig. 7D</a>).</p>
<div class="figure" id="pone-0001997-g007"><div class="img"><a name="pone-0001997-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>3D Information is Required for Accurate Spine Detection.</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>Two- and 3D renderings of a spiny dendrite imaged with CLSM and reconstructed with NeuronStudio with results of spine detection algorithm. A) Image stack is shown as a 2D maximal projection with arrows indicating positions of three representative spines (numbered 1,2,3) oriented perpendicular to the image plane, that cannot be detected from a 2D projection. B) The same dataset shown as a 3D volume rendering. The three spines are somewhat more visible in the 3D XY view. C) XY view of spines detected in 3D by NeuronStudio with spine voxels represented in different colors, superimposed on volume rendered data. Arrows point to spines 1, 2, and 3, not visible in 2D, that can be detected by NeuronStudio in 3D. D) ZY view of the same dataset as shown in (C), rotated through 90° to show the prominence of these spines in 3D.</p>
<span>doi:10.1371/journal.pone.0001997.g007</span></div>

<h4>Symmetry of Spine Detection</h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>To evaluate the symmetry of detection for all orientations, we compared the number of spines detected by the automated and manual methods in the <em>mostly horizontal</em> and <em>mostly vertical</em> groups. The results are shown in <a href="#pone-0001997-t004">Table 4</a>. Both the manual and automated methods reported more spines in the <em>mostly horizontal</em> orientation than the <em>mostly vertical</em> orientation (<a href="#pone-0001997-t004">Table 4</a>). This result confirms that when using light microscopy, spines with long axes pointing along the optic axis, and particularly those positioned directly above and below the dendrite are significantly less visible than those parallel with the image plane. This asymmetry was greater in the manual method (65.0% <em>mostly horizontal</em>:35.0% <em>mostly vertical</em>, <a href="#pone-0001997-t004">Table 4</a>) than the automated method (58.5% <em>mostly horizontal</em>:41.5% <em>mostly vertical</em>, <a href="#pone-0001997-t004">Table 4</a>), because the automated method was better at detecting sharp bumps projecting mostly vertically, above and below the dendrite, while the manual method was better at detecting very faint spines on the sides of the dendrite (see “Comparison of Automated and Manual Spine Detection”).</p>
<div class="figure" id="pone-0001997-t004"><div class="img"><a name="pone-0001997-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Symmetry of Spine Detection with respect to Orientation.</span></strong></p><span>doi:10.1371/journal.pone.0001997.t004</span></div>

<h4>Symmetry of Spine Classification</h4>
<a id="article1.body1.sec2.sec13.p1" name="article1.body1.sec2.sec13.p1"></a><p>To evaluate the symmetry of spine shape classification, we compared the type composition (measured as percentages of mushroom, stubby and thin types relative to total spine numbers) of the automated and manual methods in the <em>mostly horizontal</em> and <em>mostly vertical</em> groups. The results are summarized in <a href="#pone-0001997-t005">Table 5</a>. Spine type compositions were very similar in the <em>mostly horizontal</em> and <em>mostly vertical</em> orientations, for both manual and automated classification methods. In each method, substantially fewer spines were classified as mushroom and thin types, and substantially more spines were classified as stubby, in the <em>mostly vertical</em> relative to the <em>mostly horizontal</em> group (<a href="#pone-0001997-t005">Table 5</a>). The alteration in type composition moving from <em>mostly horizontal</em> to <em>mostly vertical</em> orientations was very similar in the automated and manual classification methods. We infer from this that the altered type composition in the vertical orientation arose primarily from shape artifacts due to effects such as poorer resolution, and residual optical smear, in the Z direction. Future research will focus on developing methods to adjust shape classification parameters depending on spine orientation, and degree of residual optical smear in the images.</p>
<div class="figure" id="pone-0001997-t005"><div class="img"><a name="pone-0001997-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Symmetry of Spine Classification with Respect to Orientation.</span></strong></p><span>doi:10.1371/journal.pone.0001997.t005</span></div>

<h4>Imaging Requirements and Algorithm Performance</h4>
<a id="article1.body1.sec2.sec14.p1" name="article1.body1.sec2.sec14.p1"></a><p>For this study the data were imaged at a resolution of 0.05 µm in XY (lateral resolution) with 0.1 µm steps along the optical axis (axial resolution). We have found that the algorithm can successfully detect spines at resolutions as low as 0.2 µm in any direction. For reliable shape classification, we recommend that voxel resolution be maintained at 0.1 µm or higher in all directions. Whatever the chosen resolution, our method requires that the LSM data be properly deconvolved in order to reduce the optical smearing introduced by the point spread function (PSF) of the microscope as well as to filter out any shot noise created by the CCD camera during digitization.</p>
<a id="article1.body1.sec2.sec14.p2" name="article1.body1.sec2.sec14.p2"></a><p>We tested the algorithm's performance on a Windows workstation with an Intel Xeon 1.0 GHz processor and 1GB of RAM. The execution time for a representative image stack of size 512×512×100 at a resolution of 0.05×0.05×0.10 µm containing a single branch with about 70 spines was 12.8 s. When the algorithm was run on a composite of 7 stacks of the same resolution and similar spine distribution as above, the running time increased to 90 s, representing a linear increase in execution time with input size. Because some overhead is incurred in spine management, the observed linearity can be affected by the spine density in each dataset. In general the execution time should remain proportional to the number of voxels examined, which ultimately depends on the size of the dataset. Output of the spine analysis can be saved as a text file (see <a href="#pone-0001997-t006">Table 6</a>).</p>
<div class="figure" id="pone-0001997-t006"><div class="img"><a name="pone-0001997-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001997" data-uri="info:doi/10.1371/journal.pone.0001997.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001997.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001997.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001997.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001997.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Spine Analysis Output Format.</span></strong></p><span>doi:10.1371/journal.pone.0001997.t006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The emerging appreciation within Neuroscience generally, that spine morphology is a sensitive index of functional and structural plasticity, has generated a rapidly increasing demand for tools that can reconstruct, classify and quantify spine shapes. The ability to analyze large volumetric data sets accurately, efficiently, and in true 3D has been a major bottleneck in deriving robust relationships between altered neuronal function and changes in spine morphology. Traditional computer-assisted manual methods for digitizing spines remain time-consuming, inaccurate and subjective (e.g., NeuroZoom <a href="#pone.0001997-Bloom1">[30]</a>, Neurolucida <a href="#pone.0001997-Glaser1">[31]</a> [MBF Bioscience, Williston, VT]). Even with the advent of semi-automated tracing methods, characterization of fine dendritic and spine structures in true 3D remains a difficult challenge.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To the best of our knowledge, three methods for automated dendritic spine analysis from light microscopy images have been reported in the literature, only one of which operates on 3D data. This 3D method, by Koh et al. <a href="#pone.0001997-Koh1">[41]</a>, uses the “grassfire” propagation technique for assigning each dendritic voxel a distance to the medial axis of the dendritic structure <a href="#pone.0001997-Lee1">[42]</a>. Tips and protrusions on the surface of the dendrite are identified as local maxima in this metric. While the method produces good results on a number of spine configurations, the iterative nature of the grassfire algorithm results in slow execution times on modestly sized datasets, on the order of half an hour for a 512×512×512 stack, the typical size to reconstruct a single spiny branch. The use of a global user-defined segmentation threshold further limits the applicability to small image stacks in which relatively uniform fluorescence levels can be attained.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>More recently, Zhang et al. <a href="#pone.0001997-Zhang1">[43]</a> have used a curvilinear structure detector to detect spines in 2D projections of LSM image stacks, and linear discriminant analysis to differentiate true spines from pseudospines. The method improves upon earlier techniques by using a local adaptive threshold for spine and dendrite detection, but is implemented only in 2D, from a maximal projection. In such 2D methods, spines above and below the dendrite projecting along the optic axis cannot be detected reliably, and spine shapes, lengths and other measurements are necessarily distorted by the absence of information in the Z direction. Nor is sufficient information available to discretize adjacent spines that appear merged in the 2D segmented images, artificially altering computed spine counts and densities. Recently, the same group has addressed some of the problems with the Koh method <a href="#pone.0001997-Koh1">[41]</a> by introducing various forms of adaptive thresholding and using a more efficient method of detecting spine tips <a href="#pone.0001997-Cheng1">[44]</a>. This implementation, however, remains 2D-based and its application to spine counts and shape analysis is accordingly limited.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The automated spine detection and shape analysis algorithms presented in this paper directly address these limitations, providing significant advances over existing techniques. By operating on minimal subsets of voxels defined by the octree, our algorithm avoids many of the computational constraints encountered by previous spine analysis techniques <a href="#pone.0001997-Koh1">[41]</a>, <a href="#pone.0001997-Zhang1">[43]</a>. Although the current implementation uses a threshold-based segmentation method, the method presented here is technically independent of the segmentation method used. Future work will focus on identifying and adapting other methods to allow greater flexibility in data segmentation. Use of the Rayburst Sampling algorithm optimizes accuracy in quantifying 3D spine morphology, by avoiding residual optical image smear that can distort spine shapes, and by minimizing quantization error that limits the accuracy of digitized images. Most importantly, the ability to operate in 3D is a fundamental requirement of a spine analysis tool. As demonstrated in <a href="#pone-0001997-g007">Figure 7</a>, existing 2D detection methods can substantially underestimate spine counts <a href="#pone.0001997-Duan1">[45]</a>, misrepresenting spine densities in morphometric studies. Nor can spine morphologies, volumes or surface areas, essential parameters in biophysical models that relate neuronal firing patterns to their structure <a href="#pone.0001997-Mainen1">[46]</a>–<a href="#pone.0001997-Stratford1">[48]</a>, be quantified accurately with 2D methods.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Extending the method to different imaging modalities is a direction for further research. Live neuron imaging requires the ability to work in significantly lower axial resolutions, with highly asymmetrical voxel dimensions. Our future research will focus on evaluating the performance of the method under such conditions and developing new techniques to allow proper detection and classification of spines in live imaging. The automated spine analysis algorithms presented in this study provide a much-needed tool for the objective evaluation of morphometric changes that occur with synaptic plasticity, normal development and aging, and with neurodegenerative disorders that impair normal cognitive function.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Animals and Data Acquisition and Preprocessing</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Four 9 month old male mice (C57Bl/SJL) were used. Animals were anesthetized with choral hydrate (15% aqueous solution, i.p.) and were perfused transcardially with 4% paraformaldehyde and 0.125% glutaraldehyde in phosphate buffer saline (PBS; pH 7.4). The brains were then carefully removed from the skull and postfixed for 6 hours. All procedures were conducted in accordance with the National Institute of Health Guide for the Care and Use of Laboratory Animals and were approved by the Mount Sinai School of Medicine Institutional Animal Care and Uses Committee.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>For intracellular injections, brains were coronally sectioned at 200 µm on a Vibratome (Leica, Nussloch, Germany). The sections were then incubated in 4,6-diamidino-2-phenylindole (DAPI; Sigma, St. Louis, MO, USA), a fluorescent nucleic acid stain, for 5 minutes, mounted on nitrocellulose filter paper and immersed in PBS. Using DAPI as a staining guide, individual layer II/III pyramidal neurons of the frontal cortex were loaded with 5% Lucifer Yellow (Molecular Probes, Eugene, OR, USA) in distilled water under a DC current of 3–8 nA for 10 minutes, or until the dye had filled distal processes and no further loading was observed <a href="#pone.0001997-Duan1">[45]</a>, <a href="#pone.0001997-Page1">[49]</a>. Tissue slices were then mounted and coverslipped in Permafluor. Dendritic segment and spine imaging was performed using a Zeiss 410 confocal laser scanning microscope (Zeiss, Thornwood, NY, USA) using a 488 nm excitation wavelength, using a 1.4 N.A. Plan-Apochromat 100× objective with a working distance of 170 µm and a 5× digital zoom. After gain and offset settings were optimized, segments were digitally imaged at 0.1 µm increments, along the optical axis. The confocal stacks were then deconvolved with AutoDeblur (MediaCybernetics, Bethesda, MD, USA).</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p><em>Supporting Information is available online</em> (<a href="#pone.0001997.s001">Box S1</a>)</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001997"></div><a name="pone.0001997.s001" id="pone.0001997.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001997.s001">Box S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Pseudo-Code for Spine Cluster Building Algorithm</p>
<p class="siDoi">doi:10.1371/journal.pone.0001997.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Mr. William Janssen, Dr. Farid Hamzei-Sichani and Mr. Kevin Kelliher for technical assistance, and colleagues in the Wearne and Hof laboratories for their participation.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SW AR DE. Performed the experiments: DD SW AR DE. Analyzed the data: SW AR DE. Contributed reagents/materials/analysis tools: PH DD. Wrote the paper: PH SW AR DE. Other: Developed Software: DE AR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001997-Rodriguez1" id="pone.0001997-Rodriguez1"></a>Rodriguez A, Ehlenberger D, Hof PR, Wearne SL (2006) Rayburst sampling, an algorithm for automated three-dimensional shape analysis from laser-scanning microsopy images. Nat Protocols  1: 2156–2161.  <ul class="find" data-citedArticleID="1091836" data-doi="10.1038/nprot.2006.313"><li><a href="http://dx.doi.org/10.1038/nprot.2006.313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rayburst+sampling%2C+an+algorithm+for+automated+three-dimensional+shape+analysis+from+laser-scanning+microsopy+images." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rayburst+sampling%2C+an+algorithm+for+automated+three-dimensional+shape+analysis+from+laser-scanning+microsopy+images.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001997-Jones1" id="pone.0001997-Jones1"></a>Jones EG, Powell TPS (1969) Morphological variations in the dendritic spines of the neocortex. J Cell Sci  5: 509–529.  <ul class="find" data-citedArticleID="1091808"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Morphological+variations+in+the+dendritic+spines+of+the+neocortex.&amp;auth=&amp;atitle=Morphological+variations+in+the+dendritic+spines+of+the+neocortex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morphological+variations+in+the+dendritic+spines+of+the+neocortex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Morphological+variations+in+the+dendritic+spines+of+the+neocortex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001997-Peters1" id="pone.0001997-Peters1"></a>Peters A, Kaiserman-Abramof IR (1970) The small pyramidal neuron of the rat cerebral cortex. The perikaryon, dendrites and spines. Am J Anat  127: 321–355.  <ul class="find" data-citedArticleID="1091832" data-doi="10.1002/aja.1001270402"><li><a href="http://dx.doi.org/10.1002/aja.1001270402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+small+pyramidal+neuron+of+the+rat+cerebral+cortex.+The+perikaryon%2C+dendrites+and+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+small+pyramidal+neuron+of+the+rat+cerebral+cortex.+The+perikaryon%2C+dendrites+and+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001997-Harris1" id="pone.0001997-Harris1"></a>Harris KM, Jensen FE, Tsao B (1992) Three-dimensional structure of dendritic spines and synapses in rat hippocampus (CA1) at postnatal day 15 and adult ages: implications for the maturation of synaptic physiology and long-term potentiation. J Neurosci  12: 2685–2705.  <ul class="find" data-citedArticleID="1091794"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Three-dimensional+structure+of+dendritic+spines+and+synapses+in+rat+hippocampus+%28CA1%29+at+postnatal+day+15+and+adult+ages%3A+implications+for+the+maturation+of+synaptic+physiology+and+long-term+potentiation.&amp;auth=&amp;atitle=Three-dimensional+structure+of+dendritic+spines+and+synapses+in+rat+hippocampus+%28CA1%29+at+postnatal+day+15+and+adult+ages%3A+implications+for+the+maturation+of+synaptic+physiology+and+long-term+potentiation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+structure+of+dendritic+spines+and+synapses+in+rat+hippocampus+%28CA1%29+at+postnatal+day+15+and+adult+ages%3A+implications+for+the+maturation+of+synaptic+physiology+and+long-term+potentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+structure+of+dendritic+spines+and+synapses+in+rat+hippocampus+%28CA1%29+at+postnatal+day+15+and+adult+ages%3A+implications+for+the+maturation+of+synaptic+physiology+and+long-term+potentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001997-Nimchinsky1" id="pone.0001997-Nimchinsky1"></a>Nimchinsky EA, Sabatini BL, Svoboda K (2002) Structure and function of dendritic spines. Annu Rev Physiol  64: 313–353.  <ul class="find" data-citedArticleID="1091826" data-doi="10.1146/annurev.physiol.64.081501.160008"><li><a href="http://dx.doi.org/10.1146/annurev.physiol.64.081501.160008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+dendritic+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+dendritic+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001997-Bourne1" id="pone.0001997-Bourne1"></a>Bourne J, Harris KM (2007) Do thin spines learn to be mushroom spines that remember? Curr Opin Neurobiol  17: 381–386.  <ul class="find" data-citedArticleID="1091782" data-doi="10.1016/j.conb.2007.04.009"><li><a href="http://dx.doi.org/10.1016/j.conb.2007.04.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Do+thin+spines+learn+to+be+mushroom+spines+that+remember%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Do+thin+spines+learn+to+be+mushroom+spines+that+remember%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001997-Hering1" id="pone.0001997-Hering1"></a>Hering H, Sheng M (2001) Dendritic spines: structure, dynamics and regulation. Nat Rev Neurosci  2: 880–888.  <ul class="find" data-citedArticleID="1091798" data-doi="10.1038/35104061"><li><a href="http://dx.doi.org/10.1038/35104061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dendritic+spines%3A+structure%2C+dynamics+and+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dendritic+spines%3A+structure%2C+dynamics+and+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001997-Lendval1" id="pone.0001997-Lendval1"></a>Lendval B, Stern EA, Chen B, Svoboda K (2000) Experience-dependent plasticity of dendritic spines in the developing rat barrel cortex <em>in vivo</em>. Nature  404: 876–881.  <ul class="find" data-citedArticleID="1091818" data-doi="10.1038/35009107"><li><a href="http://dx.doi.org/10.1038/35009107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experience-dependent+plasticity+of+dendritic+spines+in+the+developing+rat+barrel+cortex+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experience-dependent+plasticity+of+dendritic+spines+in+the+developing+rat+barrel+cortex+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001997-Dailey1" id="pone.0001997-Dailey1"></a>Dailey ME, Smith SJ (1996) The dynamics of dendritic structure in developing hippocampal slices. J Neurosci  16: 2983–2994.  <ul class="find" data-citedArticleID="1091786"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+dynamics+of+dendritic+structure+in+developing+hippocampal+slices.&amp;auth=&amp;atitle=The+dynamics+of+dendritic+structure+in+developing+hippocampal+slices." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamics+of+dendritic+structure+in+developing+hippocampal+slices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamics+of+dendritic+structure+in+developing+hippocampal+slices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001997-MaleticSavatic1" id="pone.0001997-MaleticSavatic1"></a>Maletic-Savatic M, Malinow R, Svoboda K (1999) Rapid dendritic morphogenesis in CA1 hippocampal dendrites induced by synaptic activity. Science  283: 1923–1927.  <ul class="find" data-citedArticleID="1091824" data-doi="10.1126/science.283.5409.1923"><li><a href="http://dx.doi.org/10.1126/science.283.5409.1923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+dendritic+morphogenesis+in+CA1+hippocampal+dendrites+induced+by+synaptic+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+dendritic+morphogenesis+in+CA1+hippocampal+dendrites+induced+by+synaptic+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001997-Zhou1" id="pone.0001997-Zhou1"></a>Zhou Q, Poo MM (2004) Reversal and consolidation of activity-induced synaptic modifications. Trends Neurosci  27: 378–383.  <ul class="find" data-citedArticleID="1091862" data-doi="10.1016/j.tins.2004.05.006"><li><a href="http://dx.doi.org/10.1016/j.tins.2004.05.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reversal+and+consolidation+of+activity-induced+synaptic+modifications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reversal+and+consolidation+of+activity-induced+synaptic+modifications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001997-Zuo1" id="pone.0001997-Zuo1"></a>Zuo Y, Lin A, Chang P, Gan WB (2005) Development of long-term dendritic spine stability in diverse regions of cerebral cortex. Neuron  46: 181–189.  <ul class="find" data-citedArticleID="1091864" data-doi="10.1016/j.neuron.2005.04.001"><li><a href="http://dx.doi.org/10.1016/j.neuron.2005.04.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+long-term+dendritic+spine+stability+in+diverse+regions+of+cerebral+cortex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+long-term+dendritic+spine+stability+in+diverse+regions+of+cerebral+cortex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001997-Holtmaat1" id="pone.0001997-Holtmaat1"></a>Holtmaat A, Wilbrecht L, Knott GW, Welker E, Svoboda K (2006) Experience-dependent and cell-type-specific spine growth in the neocortex. Nature  44: 979–983.  <ul class="find" data-citedArticleID="1091802" data-doi="10.1038/nature04783"><li><a href="http://dx.doi.org/10.1038/nature04783" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experience-dependent+and+cell-type-specific+spine+growth+in+the+neocortex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experience-dependent+and+cell-type-specific+spine+growth+in+the+neocortex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001997-Trachtenberg1" id="pone.0001997-Trachtenberg1"></a>Trachtenberg JT, Chen BE, Knott GW, Feng G, Sanes JR, et al.  (2002) Long-term in vivo imaging of experience-dependent synaptic plasticity in adult cortex. Nature  420: 788–794.  <ul class="find" data-citedArticleID="1091852" data-doi="10.1038/nature01273"><li><a href="http://dx.doi.org/10.1038/nature01273" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-term+in+vivo+imaging+of+experience-dependent+synaptic+plasticity+in+adult+cortex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-term+in+vivo+imaging+of+experience-dependent+synaptic+plasticity+in+adult+cortex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001997-Kasai1" id="pone.0001997-Kasai1"></a>Kasai H, Matsuzaki M, Noguchi J, Yasumatsu N, Nakahara H (2003) Structure-stability-function relationships of dendritic spines. Trends Neurosci  26: 360–368.  <ul class="find" data-citedArticleID="1091810" data-doi="10.1016/s0166-2236(03)00162-0"><li><a href="http://dx.doi.org/10.1016/s0166-2236(03)00162-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-stability-function+relationships+of+dendritic+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-stability-function+relationships+of+dendritic+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001997-Holtmaat2" id="pone.0001997-Holtmaat2"></a>Holtmaat A, Trachtenberg JT, Wilbrecht L, Shepherd GM, Zhang X, et al.  (2005) Transient and persistent dendritic spines in the neocortex in vivo. Neuron  45: 279–291.  <ul class="find" data-citedArticleID="1091804" data-doi="10.1016/j.neuron.2005.01.003"><li><a href="http://dx.doi.org/10.1016/j.neuron.2005.01.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transient+and+persistent+dendritic+spines+in+the+neocortex+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transient+and+persistent+dendritic+spines+in+the+neocortex+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001997-Grutzendler1" id="pone.0001997-Grutzendler1"></a>Grutzendler J, Kasthuri N, Gan WB (2002) Long-term dendritic spine stability in the adult cortex. Nature  420: 812–816.  <ul class="find" data-citedArticleID="1091792" data-doi="10.1038/nature01276"><li><a href="http://dx.doi.org/10.1038/nature01276" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-term+dendritic+spine+stability+in+the+adult+cortex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-term+dendritic+spine+stability+in+the+adult+cortex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001997-Shikorski1" id="pone.0001997-Shikorski1"></a>Shikorski T, Stevens CF (1999) Quantitative fine-structural analysis of olfactory cortical synapses. Proc Natl Acad Sci USA  96: 4107–4112.  <ul class="find" data-citedArticleID="1091844" data-doi="10.1073/pnas.96.7.4107"><li><a href="http://dx.doi.org/10.1073/pnas.96.7.4107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+fine-structural+analysis+of+olfactory+cortical+synapses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+fine-structural+analysis+of+olfactory+cortical+synapses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001997-Nusser1" id="pone.0001997-Nusser1"></a>Nusser Z, Lujan R, Laube G, Roberts J, Molnar E, et al.  (1998) Cell type and pathway dependence of synaptic AMPA receptor number and variability in the hippocampus. Neuron  21: 545–559.  <ul class="find" data-citedArticleID="1091828" data-doi="10.1016/s0896-6273(00)80565-6"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)80565-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+type+and+pathway+dependence+of+synaptic+AMPA+receptor+number+and+variability+in+the+hippocampus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+type+and+pathway+dependence+of+synaptic+AMPA+receptor+number+and+variability+in+the+hippocampus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001997-Yuste1" id="pone.0001997-Yuste1"></a>Yuste R, Bonhoeffer T (2001) Morphological changes in dendritic spines associated with long-term synaptic plasticity. Ann Rev Neurosci  24: 1071–1089.  <ul class="find" data-citedArticleID="1091856"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Morphological+changes+in+dendritic+spines+associated+with+long-term+synaptic+plasticity.&amp;auth=&amp;atitle=Morphological+changes+in+dendritic+spines+associated+with+long-term+synaptic+plasticity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morphological+changes+in+dendritic+spines+associated+with+long-term+synaptic+plasticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Morphological+changes+in+dendritic+spines+associated+with+long-term+synaptic+plasticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001997-Bito1" id="pone.0001997-Bito1"></a>Bito H, Deisseroth K, Tsien RW (1997) Ca<sup>2+</sup>-dependent regulation in neuronal gene expression. Curr Opin Neurobiol  7: 419–429.  <ul class="find" data-citedArticleID="1091772" data-doi="10.1016/s0959-4388(97)80072-4"><li><a href="http://dx.doi.org/10.1016/s0959-4388(97)80072-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ca2%2B-dependent+regulation+in+neuronal+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ca2%2B-dependent+regulation+in+neuronal+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001997-Alvarez1" id="pone.0001997-Alvarez1"></a>Alvarez VA, Sabatini BL (2007) Anatomical and physiological plasticity of dendritic spines. Annu Rev Neurosci  30: 79–97.  <ul class="find" data-citedArticleID="1091770" data-doi="10.1146/annurev.neuro.30.051606.094222"><li><a href="http://dx.doi.org/10.1146/annurev.neuro.30.051606.094222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anatomical+and+physiological+plasticity+of+dendritic+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anatomical+and+physiological+plasticity+of+dendritic+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001997-Holthoff1" id="pone.0001997-Holthoff1"></a>Holthoff K, Tsay D, Yuste R (2002) Calcium dynamics of spines depend on their dendritic location. Neuron  33: 425–437.  <ul class="find" data-citedArticleID="1091800" data-doi="10.1016/s0896-6273(02)00576-7"><li><a href="http://dx.doi.org/10.1016/s0896-6273(02)00576-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Calcium+dynamics+of+spines+depend+on+their+dendritic+location." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Calcium+dynamics+of+spines+depend+on+their+dendritic+location.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001997-Bloodgood1" id="pone.0001997-Bloodgood1"></a>Bloodgood BL, Sabatini BL (2007) Ca<sup>2+</sup> signaling in dendritic spines. Curr Opin Neurobiol  17: 1–7.  <ul class="find" data-citedArticleID="1091774" data-doi="10.1016/j.conb.2007.04.003"><li><a href="http://dx.doi.org/10.1016/j.conb.2007.04.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ca2%2B+signaling+in+dendritic+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ca2%2B+signaling+in+dendritic+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001997-Majewska1" id="pone.0001997-Majewska1"></a>Majewska A, Tashiro A, Yuste R (2000) Regulation of spine calcium dynamics by rapid spine motility. J Neurosci  20: 8262–8268.  <ul class="find" data-citedArticleID="1091822"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+spine+calcium+dynamics+by+rapid+spine+motility.&amp;auth=&amp;atitle=Regulation+of+spine+calcium+dynamics+by+rapid+spine+motility." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+spine+calcium+dynamics+by+rapid+spine+motility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+spine+calcium+dynamics+by+rapid+spine+motility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001997-Yuste2" id="pone.0001997-Yuste2"></a>Yuste R, Majewska A, Holthoff K (2000) From form to function: calcium compartmentalization in dendritic spines. Nat Neurosci  3: 653–659.  <ul class="find" data-citedArticleID="1091858" data-doi="10.1038/76609"><li><a href="http://dx.doi.org/10.1038/76609" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+form+to+function%3A+calcium+compartmentalization+in+dendritic+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+form+to+function%3A+calcium+compartmentalization+in+dendritic+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001997-Svoboda1" id="pone.0001997-Svoboda1"></a>Svoboda K, Tank DW, Denk W (1996) Direct measurement of coupling between dendritic spines and shafts. Science  272: 716–719.  <ul class="find" data-citedArticleID="1091850" data-doi="10.1126/science.272.5262.716"><li><a href="http://dx.doi.org/10.1126/science.272.5262.716" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+measurement+of+coupling+between+dendritic+spines+and+shafts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+measurement+of+coupling+between+dendritic+spines+and+shafts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001997-Bloodgood2" id="pone.0001997-Bloodgood2"></a>Bloodgood BL, Sabatini BL (2005) Neuronal activity regulates diffusion across the neck of dendritic spines. Science  310: 866–869.  <ul class="find" data-citedArticleID="1091776" data-doi="10.1126/science.1114816"><li><a href="http://dx.doi.org/10.1126/science.1114816" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuronal+activity+regulates+diffusion+across+the+neck+of+dendritic+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuronal+activity+regulates+diffusion+across+the+neck+of+dendritic+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001997-Santamaria1" id="pone.0001997-Santamaria1"></a>Santamaria F, Wils S, De Schutter E, Augustine GJ (2006) Anomalous diffusion in Purkinje cell dendrites caused by spines. Neuron  52: 635–648.  <ul class="find" data-citedArticleID="1091838" data-doi="10.1016/j.neuron.2006.10.025"><li><a href="http://dx.doi.org/10.1016/j.neuron.2006.10.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anomalous+diffusion+in+Purkinje+cell+dendrites+caused+by+spines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anomalous+diffusion+in+Purkinje+cell+dendrites+caused+by+spines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001997-Bloom1" id="pone.0001997-Bloom1"></a>Bloom FE, Young WG, Nimchinsky EA, Hof PR, Morrison JH (1997)  Neuronal vulnerability and informatics in human disease. In: Koslow SH, Huerta MF, editors. Neuroinformatics - An Overview of the Human Brain Project. Mahwah: Lawrence Erlbaum. pp. 83–123.  <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0001997-Glaser1" id="pone.0001997-Glaser1"></a>Glaser JR, Glaser E (1990) Neuron imaging with Neurolucida - A PC-based system for image combining microscropy. Comput Med Imaging Graphics  14: 307–317.  <ul class="find" data-citedArticleID="1091790" data-doi="10.1016/0895-6111(90)90105-k"><li><a href="http://dx.doi.org/10.1016/0895-6111(90)90105-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuron+imaging+with+Neurolucida+-+A+PC-based+system+for+image+combining+microscropy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuron+imaging+with+Neurolucida+-+A+PC-based+system+for+image+combining+microscropy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001997-Wearne1" id="pone.0001997-Wearne1"></a>Wearne SL, Rodriguez A, Ehlenberger D, Rocher AB, Henderson SC, et al.  (2005) New techniques for imaging, digitization and analysis of three-dimensional neuronal morphology on multiple scales. Neuroscience  136: 661–680.  <ul class="find" data-citedArticleID="1091854" data-doi="10.1016/j.neuroscience.2005.05.053"><li><a href="http://dx.doi.org/10.1016/j.neuroscience.2005.05.053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+techniques+for+imaging%2C+digitization+and+analysis+of+three-dimensional+neuronal+morphology+on+multiple+scales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+techniques+for+imaging%2C+digitization+and+analysis+of+three-dimensional+neuronal+morphology+on+multiple+scales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001997-AlKofahi1" id="pone.0001997-AlKofahi1"></a>Al-Kofahi KA, Lasek S, Szarowski DH, Pace CJ, Nagy G, et al.  (2002) Rapid automated three-dimensional tracing of neurons from confocal image stacks. IEEE Trans Inform Tech Biomed  6: 171–187.  <ul class="find" data-citedArticleID="1091768" data-doi="10.1109/titb.2002.1006304"><li><a href="http://dx.doi.org/10.1109/titb.2002.1006304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+automated+three-dimensional+tracing+of+neurons+from+confocal+image+stacks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+automated+three-dimensional+tracing+of+neurons+from+confocal+image+stacks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001997-Borgefors1" id="pone.0001997-Borgefors1"></a>Borgefors G, Nystrom I, Sanniti Di Baja G (1999) Computing skeletons in three dimensions. Pattern Recognition  32: 1225–1236.  <ul class="find" data-citedArticleID="1091780" data-doi="10.1016/s0031-3203(98)00082-x"><li><a href="http://dx.doi.org/10.1016/s0031-3203(98)00082-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computing+skeletons+in+three+dimensions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computing+skeletons+in+three+dimensions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001997-Streekstra1" id="pone.0001997-Streekstra1"></a>Streekstra GJ, van Pelt J (2002) Analysis of tubular structures in three-dimensional confocal images. Network: Comput Neural Systems  13: 381–395.  <ul class="find" data-citedArticleID="1091848" data-doi="10.1088/0954-898x/13/3/308"><li><a href="http://dx.doi.org/10.1088/0954-898x/13/3/308" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+tubular+structures+in+three-dimensional+confocal+images." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+tubular+structures+in+three-dimensional+confocal+images.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001997-Schmitt1" id="pone.0001997-Schmitt1"></a>Schmitt S, Evers JF, Duch C, Scholz M, Obermayer K (2004) New methods for the computer-assisted 3-D reconstruction of neurons from confocal image stacks. NeuroImage  23: 1283–1298.  <ul class="find" data-citedArticleID="1091840" data-doi="10.1016/j.neuroimage.2004.06.047"><li><a href="http://dx.doi.org/10.1016/j.neuroimage.2004.06.047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+methods+for+the+computer-assisted+3-D+reconstruction+of+neurons+from+confocal+image+stacks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+methods+for+the+computer-assisted+3-D+reconstruction+of+neurons+from+confocal+image+stacks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001997-He1" id="pone.0001997-He1"></a>He W, Hamilton TA, Cohen AR, Holmes TJ, Pace C, et al.  (2003) Automated three-dimensional tracing of neurons in confocal and brightfield images. Microsc Microanal  9: 296–310.  <ul class="find" data-citedArticleID="1091796" data-doi="10.1017/s143192760303040x"><li><a href="http://dx.doi.org/10.1017/s143192760303040x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+three-dimensional+tracing+of+neurons+in+confocal+and+brightfield+images." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+three-dimensional+tracing+of+neurons+in+confocal+and+brightfield+images.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001997-Scorcioni1" id="pone.0001997-Scorcioni1"></a>Scorcioni R, Ascoli GA (2001)  Algorithmic extraction of morphological statistics from electronic archives of neuroanatomy. In: Mira J, Prieto A, editors. IWANN 2001, Lecture Notes in Computer Science. Berlin, Heidelberg: Springer-Verlag. pp. 30–37.  <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001997-Ridler1" id="pone.0001997-Ridler1"></a>Ridler TW, Calvard S (1978) Picture thresholding using an interative selection method. IEEE Trans Syst Man Cybern  SMC-8: 630–632.  <ul class="find" data-citedArticleID="1091834" data-doi="10.1109/tsmc.1978.4310039"><li><a href="http://dx.doi.org/10.1109/tsmc.1978.4310039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Picture+thresholding+using+an+interative+selection+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Picture+thresholding+using+an+interative+selection+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001997-Jackins1" id="pone.0001997-Jackins1"></a>Jackins CL, Tanimoto SL (1980) Oct-trees and their use in representing three-dimensional objects. Computer Graphics Image Process  14: 249–270.  <ul class="find" data-citedArticleID="1091806" data-doi="10.1016/0146-664x(80)90055-6"><li><a href="http://dx.doi.org/10.1016/0146-664x(80)90055-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oct-trees+and+their+use+in+representing+three-dimensional+objects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oct-trees+and+their+use+in+representing+three-dimensional+objects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001997-Koh1" id="pone.0001997-Koh1"></a>Koh IYY, Lindquist WB, Zito K, Nimchinsky EA, Svoboda K (2002) An image analysis algorithm for the fine structure of neuronal dendrites. Neural Comput  14: 1283–1310.  <ul class="find" data-citedArticleID="1091812" data-doi="10.1162/089976602753712945"><li><a href="http://dx.doi.org/10.1162/089976602753712945" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+image+analysis+algorithm+for+the+fine+structure+of+neuronal+dendrites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+image+analysis+algorithm+for+the+fine+structure+of+neuronal+dendrites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001997-Lee1" id="pone.0001997-Lee1"></a>Lee TC, Kashyap RL, Chu C (1994) Building skeleton models via 3-D medial surface/axis thinning algorithms. CVGIP: Graph Models Image Process  56: 462–478.  <ul class="find" data-citedArticleID="1091816" data-doi="10.1006/cgip.1994.1042"><li><a href="http://dx.doi.org/10.1006/cgip.1994.1042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Building+skeleton+models+via+3-D+medial+surface%2Faxis+thinning+algorithms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Building+skeleton+models+via+3-D+medial+surface%2Faxis+thinning+algorithms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001997-Zhang1" id="pone.0001997-Zhang1"></a>Zhang Y, Zhou X, Witt RM, Sabatini BL, Adjeroh D, et al.  (2007) Dendritic spine detection using curvilinear structure detector and LDA classifier. NeuroImage  36: 346–360.  <ul class="find" data-citedArticleID="1091860" data-doi="10.1016/j.neuroimage.2007.02.044"><li><a href="http://dx.doi.org/10.1016/j.neuroimage.2007.02.044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dendritic+spine+detection+using+curvilinear+structure+detector+and+LDA+classifier." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dendritic+spine+detection+using+curvilinear+structure+detector+and+LDA+classifier.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001997-Cheng1" id="pone.0001997-Cheng1"></a>Cheng J, Zhou X, Miller E, Witt RM, Zhu J, et al.  (2007) A novel computational approach for automatic dendrite spines detection in two-photon laser scan microscopy. J Neurosci Methods  165: 122–134.  <ul class="find" data-citedArticleID="1091784" data-doi="10.1016/j.jneumeth.2007.05.020"><li><a href="http://dx.doi.org/10.1016/j.jneumeth.2007.05.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+computational+approach+for+automatic+dendrite+spines+detection+in+two-photon+laser+scan+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+computational+approach+for+automatic+dendrite+spines+detection+in+two-photon+laser+scan+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001997-Duan1" id="pone.0001997-Duan1"></a>Duan H, Wearne SL, Rocher AB, Macedo A, Morrison JH, et al.  (2003) Age-related morphologic alterations in dendrites and spine densities of corticocortically projecting neurons in macaque monkeys. Cereb Cortex  13: 950–961.  <ul class="find" data-citedArticleID="1091788" data-doi="10.1093/cercor/13.9.950"><li><a href="http://dx.doi.org/10.1093/cercor/13.9.950" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Age-related+morphologic+alterations+in+dendrites+and+spine+densities+of+corticocortically+projecting+neurons+in+macaque+monkeys." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Age-related+morphologic+alterations+in+dendrites+and+spine+densities+of+corticocortically+projecting+neurons+in+macaque+monkeys.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001997-Mainen1" id="pone.0001997-Mainen1"></a>Mainen ZF, Sejnowski TJ (1996) Influence of dendritic structure on firing pattern in model neocortical neurons. Nature  382: 363–366.  <ul class="find" data-citedArticleID="1091820" data-doi="10.1038/382363a0"><li><a href="http://dx.doi.org/10.1038/382363a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+dendritic+structure+on+firing+pattern+in+model+neocortical+neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+dendritic+structure+on+firing+pattern+in+model+neocortical+neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001997-Krichmar1" id="pone.0001997-Krichmar1"></a>Krichmar JL, Nasuto SJ, Scorcioni R, Washington SD, Ascoli GA (2002) Effects of dendritic morphology on CA3 pyramidal cell electrophysiology: a simulation study. Brain Res  941: 11–28.  <ul class="find" data-citedArticleID="1091814" data-doi="10.1016/s0006-8993(02)02488-5"><li><a href="http://dx.doi.org/10.1016/s0006-8993(02)02488-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+dendritic+morphology+on+CA3+pyramidal+cell+electrophysiology%3A+a+simulation+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+dendritic+morphology+on+CA3+pyramidal+cell+electrophysiology%3A+a+simulation+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001997-Stratford1" id="pone.0001997-Stratford1"></a>Stratford K, Mason A, Larkman A, Major G, Jack J (1989)  The modeling of pyramidal neurons in the visual cortex. In: Durbin R, Miall C, Mitchison G, editors. The Computing Neuron. London, U.K.: Addison-Wesley. pp. 296–321.  <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0001997-Page1" id="pone.0001997-Page1"></a>Page TL, Einstein M, Duan H, He Y, Flores T, et al.  (2002) Morphological alterations in neurons forming corticocortical projections in the neocortex of aged patas monkeys. Neurosci Lett  317: 37–41.  <ul class="find" data-citedArticleID="1091830" data-doi="10.1016/s0304-3940(01)02428-4"><li><a href="http://dx.doi.org/10.1016/s0304-3940(01)02428-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morphological+alterations+in+neurons+forming+corticocortical+projections+in+the+neocortex+of+aged+patas+monkeys." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Morphological+alterations+in+neurons+forming+corticocortical+projections+in+the+neocortex+of+aged+patas+monkeys.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.XML" value="129636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.PDF" value="538970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g001.PNG_L" value="3174945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g001.PNG_M" value="289266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g001.PNG_S" value="17617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g001.TIF" value="4410068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g001.PNG_I" value="96614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g002.PNG_L" value="1179980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g002.PNG_M" value="184602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g002.PNG_S" value="16370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g002.TIF" value="1914712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g002.PNG_I" value="62677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.e001.PNG" value="10129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.e001.TIF" value="25800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g003.PNG_L" value="1830557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g003.PNG_M" value="167783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g003.PNG_S" value="19935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g003.TIF" value="3055956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g003.PNG_I" value="111081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g004.PNG_L" value="1199520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g004.PNG_M" value="117809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g004.PNG_S" value="13869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g004.TIF" value="2133580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g004.PNG_I" value="41377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g005.PNG_L" value="1013098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g005.PNG_M" value="84381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g005.PNG_S" value="13244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g005.TIF" value="1778616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g005.PNG_I" value="31040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.e002.PNG" value="10660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.e002.TIF" value="26388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g006.PNG_L" value="1296365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g006.PNG_M" value="149380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g006.PNG_S" value="19814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g006.TIF" value="2484914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g006.PNG_I" value="101250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t001.PNG_L" value="95145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t001.PNG_M" value="120683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t001.PNG_S" value="17505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t001.TIF" value="433952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t001.PNG_I" value="42390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t002.PNG_L" value="99025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t002.PNG_M" value="59763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t002.PNG_S" value="8772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t002.TIF" value="427780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t002.PNG_I" value="18787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t003.PNG_L" value="114007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t003.PNG_M" value="156067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t003.PNG_S" value="18632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t003.TIF" value="471078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t003.PNG_I" value="54796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g007.PNG_L" value="1692007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g007.PNG_M" value="203388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g007.PNG_S" value="18879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g007.TIF" value="2854192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.g007.PNG_I" value="74977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t004.PNG_L" value="123109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t004.PNG_M" value="75723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t004.PNG_S" value="9567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t004.TIF" value="515700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t004.PNG_I" value="23684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t005.PNG_L" value="123936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t005.PNG_M" value="73304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t005.PNG_S" value="9300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t005.TIF" value="537258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t005.PNG_I" value="22869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t006.PNG_L" value="135180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t006.PNG_M" value="83549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t006.PNG_S" value="10990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t006.TIF" value="594428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.t006.PNG_I" value="26194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001997.s001.DOC" value="22016"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001997&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001997" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001997&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001997&volume=&issue=&title=Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images&author_name=Alfredo%20Rodriguez%2C%20Douglas%20B.%20Ehlenberger%2C%20Dara%20L.%20Dickstein%2C%20Patrick%20R.%20Hof%2C%20Susan%20L.%20Wearne&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001997" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001997" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001997" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001997&amp;t=Automated%20Three-Dimensional%20Detection%20and%20Shape%20Classification%20of%20Dendritic%20Spines%20from%20Fluorescence%20Microscopy%20Images" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001997&title=Automated%20Three-Dimensional%20Detection%20and%20Shape%20Classification%20of%20Dendritic%20Spines%20from%20Fluorescence%20Microscopy%20Images&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001997&amp;title=Automated%20Three-Dimensional%20Detection%20and%20Shape%20Classification%20of%20Dendritic%20Spines%20from%20Fluorescence%20Microscopy%20Images" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001997" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001997&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Automated Three-Dimensional Detection and Shape Classification of Dendritic Spines from Fluorescence Microscopy Images';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001997';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Automated%20Three-Dimensional%20Detection%20and%20Shape%20Classification%20of%20Dendritic%20Spines%20from%20Fluorescence%20Microscopy%20Images http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001997" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001997" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Algorithms%22" title="Search for articles in the subject area:'Algorithms'"><div class="flagText">Algorithms</div></a>
              <div data-categoryid="32621" data-articleid="26818"
                   data-categoryname="Algorithms"
                   class="flagImage" title="Flag 'Algorithms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Confocal+laser+microscopy%22" title="Search for articles in the subject area:'Confocal laser microscopy'"><div class="flagText">Confocal laser microscopy</div></a>
              <div data-categoryid="17403" data-articleid="26818"
                   data-categoryname="Confocal laser microscopy"
                   class="flagImage" title="Flag 'Confocal laser microscopy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Dendritic+structure%22" title="Search for articles in the subject area:'Dendritic structure'"><div class="flagText">Dendritic structure</div></a>
              <div data-categoryid="19507" data-articleid="26818"
                   data-categoryname="Dendritic structure"
                   class="flagImage" title="Flag 'Dendritic structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Head%22" title="Search for articles in the subject area:'Head'"><div class="flagText">Head</div></a>
              <div data-categoryid="48905" data-articleid="26818"
                   data-categoryname="Head"
                   class="flagImage" title="Flag 'Head' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Imaging+techniques%22" title="Search for articles in the subject area:'Imaging techniques'"><div class="flagText">Imaging techniques</div></a>
              <div data-categoryid="2295" data-articleid="26818"
                   data-categoryname="Imaging techniques"
                   class="flagImage" title="Flag 'Imaging techniques' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Neck%22" title="Search for articles in the subject area:'Neck'"><div class="flagText">Neck</div></a>
              <div data-categoryid="49119" data-articleid="26818"
                   data-categoryname="Neck"
                   class="flagImage" title="Flag 'Neck' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Neuronal+dendrites%22" title="Search for articles in the subject area:'Neuronal dendrites'"><div class="flagText">Neuronal dendrites</div></a>
              <div data-categoryid="17499" data-articleid="26818"
                   data-categoryname="Neuronal dendrites"
                   class="flagImage" title="Flag 'Neuronal dendrites' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Neuronal+morphology%22" title="Search for articles in the subject area:'Neuronal morphology'"><div class="flagText">Neuronal morphology</div></a>
              <div data-categoryid="18153" data-articleid="26818"
                   data-categoryname="Neuronal morphology"
                   class="flagImage" title="Flag 'Neuronal morphology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6608'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4142'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9671&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=11655">Publisher's Note: Correction to Academic Editor's name</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=3603">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
