

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001195"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001195" />

    <meta name="citation_title" content="Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets"/>
    <meta itemprop="name" content="Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets"/>

      <meta name="citation_author" content="Yujin Hoshida"/>
            <meta name="citation_author_institution" content="The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="Pediatric Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Jean-Philippe Brunet"/>
            <meta name="citation_author_institution" content="The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Pablo Tamayo"/>
            <meta name="citation_author_institution" content="The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Todd R. Golub"/>
            <meta name="citation_author_institution" content="The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="Pediatric Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Jill P. Mesirov"/>
            <meta name="citation_author_institution" content="The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001195.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1195"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Prediction of cancer outcome with microarrays: a multiple random validation strategy.; citation_author=S Michiels; citation_author=S Koscielny; citation_author=C Hill; citation_journal_title=Lancet; citation_volume=365; citation_number=1; citation_pages=488-492; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Independence and reproducibility across microarray platforms.; citation_author=JE Larkin; citation_author=BC Frank; citation_author=H Gavras; citation_author=R Sultana; citation_author=J Quackenbush; citation_journal_title=Nat Methods; citation_volume=2; citation_number=2; citation_pages=337-344; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.; citation_author=A Subramanian; citation_author=P Tamayo; citation_author=VK Mootha; citation_author=S Mukherjee; citation_author=BL Ebert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=3; citation_pages=15545-15550; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Statistical Methods for Research Workers.; citation_author=RA Fisher; citation_number=4; citation_date=1932; citation_publisher=Oliver and Boyd; " />
      <meta name="citation_reference" content="citation_title=Large-scale analysis of the human and mouse transcriptomes.; citation_author=AI Su; citation_author=MP Cooke; citation_author=KA Ching; citation_author=Y Hakak; citation_author=JR Walker; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=5; citation_pages=4465-4470; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multiclass cancer diagnosis using tumor gene expression signatures.; citation_author=S Ramaswamy; citation_author=P Tamayo; citation_author=R Rifkin; citation_author=S Mukherjee; citation_author=CH Yeang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=6; citation_pages=15149-15154; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular profiling of diffuse large B-cell lymphoma identifies robust subtypes including one characterized by host inflammatory response.; citation_author=S Monti; citation_author=KJ Savage; citation_author=JL Kutok; citation_author=F Feuerhake; citation_author=P Kurtin; citation_journal_title=Blood; citation_volume=105; citation_number=7; citation_pages=1851-1861; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma.; citation_author=A Rosenwald; citation_author=G Wright; citation_author=WC Chan; citation_author=JM Connors; citation_author=E Campo; citation_journal_title=N Engl J Med; citation_volume=346; citation_number=8; citation_pages=1937-1947; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Consensus Clustering: A resampling-based method for class discovery and visualization of gene expression microarray data.; citation_author=S Monti; citation_author=P Tamayo; citation_author=J Mesirov; citation_author=T Golub; citation_journal_title=Machine Learning Journal; citation_volume=52; citation_number=9; citation_pages=91-118; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Are clusters found in one dataset present in another dataset?; citation_author=AV Kapp; citation_author=R Tibshirani; citation_journal_title=Biostatistics; citation_volume=8; citation_number=10; citation_pages=9-31; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling predicts clinical outcome of breast cancer.; citation_author=LJ van't Veer; citation_author=H Dai; citation_author=MJ van de Vijver; citation_author=YD He; citation_author=AA Hart; citation_journal_title=Nature; citation_volume=415; citation_number=11; citation_pages=530-536; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Predicting the clinical status of human breast cancer by using gene expression profiles.; citation_author=M West; citation_author=C Blanchette; citation_author=H Dressman; citation_author=E Huang; citation_author=S Ishida; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=12; citation_pages=11462-11467; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multifactorial analysis of differences between sporadic breast cancers and cancers involving BRCA1 and BRCA2 mutations.; citation_author=SR Lakhani; citation_author=J Jacquemier; citation_author=JP Sloane; citation_author=BA Gusterson; citation_author=TJ Anderson; citation_journal_title=J Natl Cancer Inst; citation_volume=90; citation_number=13; citation_pages=1138-1145; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Diffuse large B-cell lymphoma outcome prediction by gene-expression profiling and supervised machine learning.; citation_author=MA Shipp; citation_author=KN Ross; citation_author=P Tamayo; citation_author=AP Weng; citation_author=JL Kutok; citation_journal_title=Nat Med; citation_volume=8; citation_number=14; citation_pages=68-74; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Missing value estimation methods for DNA microarrays.; citation_author=O Troyanskaya; citation_author=M Cantor; citation_author=G Sherlock; citation_author=P Brown; citation_author=T Hastie; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=15; citation_pages=520-525; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns.; citation_author=MB Eisen; citation_author=PT Spellman; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=16; citation_pages=14863-14868; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular classification of cancer: class discovery and class prediction by gene expression monitoring.; citation_author=TR Golub; citation_author=DK Slonim; citation_author=P Tamayo; citation_author=C Huard; citation_author=M Gaasenbeek; citation_journal_title=Science; citation_volume=286; citation_number=17; citation_pages=531-537; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response.; citation_author=VG Tusher; citation_author=R Tibshirani; citation_author=G Chu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=18; citation_pages=5116-5121; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Controlling the false discovery rate in behavior genetics research.; citation_author=Y Benjamini; citation_author=D Drai; citation_author=G Elmer; citation_author=N Kafkafi; citation_author=I Golani; citation_journal_title=Behav Brain Res; citation_volume=125; citation_number=19; citation_pages=279-284; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Combining significance levels.; citation_author=BJ Becker; citation_number=20; citation_pages=215-230; citation_date=1994; citation_publisher=Russell Sage Foundation; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001195" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets"/>
    <meta name="twitter:description" content="Whole genome expression profiles are widely used to discover molecular subtypes of diseases. A remaining challenge is to identify the correspondence or commonality of subtypes found in multiple, independent data sets generated on various platforms. While model-based supervised learning is often used to make these connections, the models can be biased to the training data set and thus miss inherent, relevant substructure in the test data. Here we describe an unsupervised subclass mapping method (SubMap), which reveals common subtypes between independent data sets. The subtypes within a data set can be determined by unsupervised clustering or given by predetermined phenotypes before applying SubMap. We define a measure of correspondence for subtypes and evaluate its significance building on our previous work on gene set enrichment analysis. The strength of the SubMap method is that it does not impose the structure of one data set upon another, but rather uses a bi-directional approach to highlight the common substructures in both. We show how this method can reveal the correspondence between several cancer-related data sets. Notably, it identifies common subtypes of breast cancer associated with estrogen receptor status, and a subgroup of lymphoma patients who share similar survival patterns, thus improving the accuracy of a clinical outcome predictor."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001195.g005"/>

  <meta property="og:title" content="Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001195" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4007'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=616'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3152&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001195">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001195" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001195&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yujin Hoshida
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:hoshida@broad.mit.edu">hoshida@broad.mit.edu</a></p>

                <p>Affiliations:
                  The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America, 
                  Pediatric Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Philippe Brunet, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pablo Tamayo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Todd R. Golub
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Todd R. Golub, Jill P. Mesirov</p>

              
              

                <p>Affiliations:
                  The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America, 
                  Pediatric Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jill P. Mesirov
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Todd R. Golub, Jill P. Mesirov</p>

              
              

                <p>Affiliation:
                  The Eli and Edythe L. Broad Institute, Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001195</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001195" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001195" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001195" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001195" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001195-g001" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001195-t001" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001195-g002" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001195-g003" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001195-g004" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001195-g005" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001195">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001195" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Whole genome expression profiles are widely used to discover molecular subtypes of diseases. A remaining challenge is to identify the correspondence or commonality of subtypes found in multiple, independent data sets generated on various platforms. While model-based supervised learning is often used to make these connections, the models can be biased to the training data set and thus miss inherent, relevant substructure in the test data. Here we describe an unsupervised subclass mapping method (SubMap), which reveals common subtypes between independent data sets. The subtypes within a data set can be determined by unsupervised clustering or given by predetermined phenotypes before applying SubMap. We define a measure of correspondence for subtypes and evaluate its significance building on our previous work on gene set enrichment analysis. The strength of the SubMap method is that it does not impose the structure of one data set upon another, but rather uses a bi-directional approach to highlight the common substructures in both. We show how this method can reveal the correspondence between several cancer-related data sets. Notably, it identifies common subtypes of breast cancer associated with estrogen receptor status, and a subgroup of lymphoma patients who share similar survival patterns, thus improving the accuracy of a clinical outcome predictor.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hoshida Y, Brunet J-P, Tamayo P, Golub TR, Mesirov JP (2007) Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets. PLoS ONE 2(11):
          e1195.
            doi:10.1371/journal.pone.0001195</p><p><strong>Academic Editor: </strong>Oliver Hofmann, South African National Bioinformatics Institute, South Africa</p><p><strong>Received:</strong> June 7, 2007; <strong>Accepted:</strong> October 24, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Hoshida et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Yujin Hoshida is supported by Charles A King Trust Fellowship. There was no contribution from the funder in any aspects of the current study including the design, analysis, interpretation, and preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>DNA microarray-based whole genome expression profiling is subject to poor reproducibility of discovered molecular disease subtypes and can lead to biomarkers that do not generalize <a href="#pone.0001195-Michiels1">[1]</a>. This problem arises from various technical and biological sources including platform differences <a href="#pone.0001195-Larkin1">[2]</a>, and has been a major obstacle to moving microarrays into the clinic as a tool to uncover as yet unrecognized disease subtypes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Comparison and integration of the molecular disease subtypes, independently defined in different data sets, has been a highly challenging problem. Subtypes are often based on subtle differences in gene expression, which can be dominated by the measurement variation between different experiments and/or platforms. A widely used method to connect such independent data sets, supervised learning, does not completely solve this problem. Subtype models depend on one particular “training” data set with its own platform-specific data structure. This structure may not be present in new “test” data sets.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we describe subclass mapping (SubMap), an unsupervised method that reveals common subtypes observed in independent data sets without relying on, or being biased by, one model data set. By bi-directionally evaluating association of predetermined subtypes between independent data sets, our method identifies more reliable molecular disease subtypes and subpopulations of cohorts that share similar clinical behavior. These results indicate the great potential of this method to maximize the use of the vast amount of accumulating genomics data and to improve current clinical practice through the development of better diagnostics and therapeutics.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Overview of Subclass mapping (SubMap)</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>SubMap is an unsupervised method, which estimates the significance of an association between subclasses observed in two independent data sets. The subclass labels are predetermined as manually assigned phenotypes or by clustering prior to the application of the SubMap algorithm (i.e., the SubMap algorithm does not assign a <em>de novo</em> class label to each sample). Because the subclass correspondence is evaluated for all pairs of subclasses, one subclass drawn from each data set, the number of subclasses or subtypes in the two data sets does not need to be the same or even similar.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The mapping information obtained by SubMap can be used to reveal general subclasses common to both data sets and thus expose the likelihood that they share the same or similar underlying biological property. Details of our approach are described in the <a href="#s4">Materials and Methods</a> section and <a href="#pone-0001195-g001">Figure 1</a>. Here we review the three key steps in the SubMap method:</p>
<div class="figure" id="pone-0001195-g001"><div class="img"><a name="pone-0001195-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Subclass mapping (SubMap) methodology. Two independent data sets, A and B, are clustered separately, compared and integrated.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(a) Candidate subclasses are defined by clustering A and B (predetermined phenotype can also be used). Marker genes of each candidate subclass in A (A<em><sub>i</sub></em>) are selected, and mapped onto a gene list ranked according to their differential expression with respect to a subclass of B (B<em><sub>j</sub></em>). Their over-representation at the top of the ranking is evaluated using the enrichment score (ES<sub>A<em>i</em>B<em>j</em></sub>), and significance is assessed as a nominal p-value, p<sub>A<em>i</em>B<em>j</em></sub>, by randomly permuting sample class labels in B. This process is repeated by interchanging the role of A and B to compute ES<sub>B<em>j</em>A<em>i</em></sub> and p<sub>B<em>j</em>A<em>i</em></sub>. (b) Mutual enrichment information, p<sub>A<em>i</em>B<em>j</em></sub> and p<sub>B<em>j</em>A<em>i</em></sub>, are combined using the Fisher inverse chi-square statistic, <em>F<sub>ij</sub></em>. Its significance is estimated based on a null distribution for the <em>F<sub>ij</sub></em> generated by randomly picking the nominal-p from corresponding null distributions for ES<sub>A<em>i</em>B<em>j</em></sub> and ES<sub>B<em>j</em>A<em>i</em></sub>. After multiple hypothesis testing (MHT) correction, p-values for <em>F<sub>ij</sub></em> are summarized in the subclass association (SA) matrix. Clustering of the SA matrix reveals subclasses common to A and B.</p>
<span>doi:10.1371/journal.pone.0001195.g001</span></div><h5>Step 1: Measure similarity between subclasses.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>We start with two independent data sets, A and B, with candidate subclasses independently determined in both. For simplicity, suppose we have two candidate subclasses in each data set: A1 and A2 in data set A; B1 and B2 in data set B. We define a set of marker genes, <em>marker</em>(A1) for subclass A1, and similarly for A2, using any suitable subclass discrimination metric. Using the same metric, genes in data set B are rank-ordered according to their correlation with B1 vs. B2 to yield a gene list, <em>ranking</em>(B1). Association between A1 and B1 is evaluated by quantifying the over-representation of <em>marker(</em>A1) in the up-regulated end of the list <em>ranking</em>(B1) using Gene Set Enrichment Analysis (GSEA) as described in <a href="#pone.0001195-Subramanian1">[3]</a>. The latter provides both an enrichment score (ES) and a corresponding nominal p-value by means of an empirical permutation test. We repeat this process, interchanging the roles of A1 and B1, to compute a nominal p-value for the enrichment of <em>marker</em>(B1) in <em>ranking</em>(A1). We can generalize this process straightforwardly when there are more than two candidate subclasses in either data set and compute a pair of nominal p-values for every possible pair-wise combination of subclasses. Thus if there are <em>n<sub>A</sub></em> subclasses in A and <em>n<sub>B</sub></em> subclasses in B, then we have mutual association information for each of the <em>n<sub>A</sub></em>*<em>n<sub>B</sub></em> pairs of candidate subclasses from the two independent data sets.</p>

<h5>Step 2: Estimate significance of association between subclasses.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>To summarize and estimate the significance of the mutual association information, i.e., the pair of nominal p-values for the ES, we use the Fisher inverse chi-square statistic, <em>F</em> <a href="#pone.0001195-Fisher1">[4]</a>. To estimate a nominal p-value for <em>F</em>, we generate an appropriate null distribution by randomly picking ES scores from the null distributions corresponding to each direction of the enrichment analysis (e.g., <em>marker</em>(A1) in <em>ranking</em>(B1) and <em>marker</em>(B1) in <em>ranking</em>(A1)) and generating the corresponding Fisher inverse chi-square statistic.</p>

<h5>Step 3: Construct and cluster subclass association matrix.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>We use a Bonferonni adjustment to account for multiple hypotheses testing, and summarize the adjusted p-values in a matrix called the subclass association matrix (SA matrix). Two-way clustering of this matrix reveals general subclasses common to these data sets.</p>
<a id="article1.body1.sec2.sec1.sec3.p2" name="article1.body1.sec2.sec1.sec3.p2"></a><p>We applied our method to four pairs of publicly available data sets (<a href="#pone-0001195-t001">Table 1</a>). We first validated our method by correctly recovering expected associations of subclasses using a pair of data sets comprised of multiple tissue types (Example 1). Next we analyzed diffuse large B cell lymphoma (DLBCL) data sets to show the superiority of our method to standard supervised prediction methods (Example 2). We then applied our method to breast cancer data sets to identify a common subtype we found to be associated with the estrogen receptor status (Example 3). The final example shows that our method has the potential to improve the performance of molecular marker-based patient outcome prediction (Example 4). We note that all of these analyses were performed on data sets acquired on a wide variety of DNA microarray platforms.</p>
<div class="figure" id="pone-0001195-t001"><div class="img"><a name="pone-0001195-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001195.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001195.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Data sets</span></strong></p><span>doi:10.1371/journal.pone.0001195.t001</span></div>


<h4>Example 1. Multiple tissue types</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>For a straightforward validation of subclass mapping, we used two data sets of multiple normal tissue types acquired on two different generations of the Affymetrix GeneChip oligonucleotide microarray platform, Multi-A <a href="#pone.0001195-Su1">[5]</a> and Multi-B <a href="#pone.0001195-Ramaswamy1">[6]</a>. This represents the case where subtypes are determined by ancillary phenotype information rather than “discovered” by clustering. Both data sets include four distinct tissue types, i.e., breast, prostate, lung, and colon. The only cross-dataset pairs of subtypes showing significant association corresponded to the same tissue type (<a href="#pone-0001195-g002">Figure 2a</a>).</p>
<div class="figure" id="pone-0001195-g002"><div class="img"><a name="pone-0001195-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Example 1: Multiple tissue types.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(a) SubMap was applied to two data sets, Multi-A and Multi-B, containing multiple tissue types: breast (Br), prostate (Pr), lung (Lu), and colon (Co). Bonferroni-corrected p-values for breast, prostate, lung, and colon tissues were 0.002, 0.002, 0.002, and 0.002, respectively. (b) Each tissue type in Multi-B was removed before applying SubMap. Only subsets of the same tissue type were significantly associated (Bonferoni-corrected p&lt;0.05). The p-values for “Multi-A-Br and Multi-B-Lu (left-upper)”, “Multi-A-Lu and Multi-B-Br (left-bottom)”, and “Multi-A-Co and Multi-B-Br (right-bottom)” are 0.330, 0.547, and 0.517, respectively</p>
<span>doi:10.1371/journal.pone.0001195.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In actual biological or clinical data sets that nominally include the same or similar samples, the entire range of sample diversity in one data set may not be represented in the other data sets, i.e., a subclass in one data set may not be represented in others. To assess whether such a situation would lead to false positive associations, we removed the samples corresponding to one tissue type in Multi-B and re-ran SubMap with the Multi-A (<a href="#pone-0001195-g002">Figure 2b</a>). Importantly, only subsets of the same tissue type were significantly associated, and there were no false positive calls.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We next investigated whether the choice of the number of marker genes to be mapped might affect the result of SubMap. The number of strongly differentially expressed genes for cross-dataset subclass pairs might differ. Since we are using the same number of marker genes for all comparisons we sought to confirm that the enrichment score was robust. We believed this would be the case based on the robustness of the GSEA score. We tested the effect of using 20, 50, and 200 ranked marker genes from a list of differentially expressed genes, as they represent the number of markers commonly used in classification and prediction studies for microarray data, as well as the level at which we might expect to begin to see noise. The resulting observed significance calls were quite stable for pairs of same tissue type irrespective of the number of the marker genes used (<a href="#pone.0001195.s004">Figure S1</a>).</p>


<h4>Example 2. Diffuse large B cell lymphoma: comparison with other methods</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Here we show the superiority of SubMap in directly highlighting all three of the corresponding, common subtypes in two independent Diffuse Large B-Cell Lymphoma (DLBCL) data sets. We first compare with our previous work in <a href="#pone.0001195-Monti1">[7]</a> where two of the three common subtypes could only be associated after removing the third. Importantly, we also show how supervised approaches, building a model from one data set to identify the subtypes in the other, also fail to identify all three shared subtypes.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We analyzed two DLBCL data sets, DLBCL-A <a href="#pone.0001195-Monti1">[7]</a> and DLBCL-B <a href="#pone.0001195-Rosenwald1">[8]</a>. The data sets were generated using one-channel oligonucleotide microarrays and 2-channel custom cDNA microarrays, respectively. In previous work we found three robust subtypes in both DLBCL-A and DLBCL-B using resampling-based multiple clustering trials and three different clustering algorithms <a href="#pone.0001195-Monti1">[7]</a>, <a href="#pone.0001195-Monti2">[9]</a>. The three clustered subtypes were designated as “oxidative phosphorylation (OxPhos)”, “B-cell response (BCR)”, and “host response (HR)” according to relevant molecular mechanisms.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In this precedent analysis, we associated these subtypes between the data sets by assessing the overlap of each subtype's marker genes in both data sets one-by-one using the Fisher test for a 2×2 table. However, unlike SubMap, this approach is highly sensitive to the criteria for marker gene selection and the total number of genes common to both platforms. Notably, initially only the HR clusters from the two datasets showed a significant association, and only after their removal could we identify the significant association for the OxPhos and BCR subtypes <a href="#pone.0001195-Monti1">[7]</a>. However, our SubMap method immediately and automatically recovered the three-subclass structure common to both data sets without removing any samples (<a href="#pone-0001195-g003">Figure 3</a>). This is likely due to the method's ability to capture subtle, but reproducible, subclass associations by incorporating bi-directional marker gene enrichment information.</p>
<div class="figure" id="pone-0001195-g003"><div class="img"><a name="pone-0001195-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Example 2: Common subtypes of Diffuse Large B-cell Lymphoma (DLBCL).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>SubMap was applied for three subclasses of DLBCL pre-determined in DLBCL-A and DLBCL-B data sets. Bonferroni-corrected p-values for “oxidative phosphorylation (OxPhos)”, “B-cell response (BCR)”, and “host response (HR)” subtypes were 0.008, 0.001, and 0.001, respectively. The association for the pair of DLBCL-A-BCR and DLBCL-B-OxPhos was not significant (p = 0.362).</p>
<span>doi:10.1371/journal.pone.0001195.g003</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Next we compared SubMap with supervised methods for their ability to identify similar subtypes from distinct data sets using DLBCL-A and DLBCL-B. Supervised learning methods are widely used to associate two independent data sets; the model of the subtype is defined in a “training” data set, and assessed in a “test” data set. We employed two standard supervised methods, k-nearest neighbors (k = 7, and 10 marker genes) and support vector machines (using all genes in common) and interchanged the roles of the two sets as train and test (details are described in the <a href="#s4">Materials and Methods</a>). Both the k-NN and SVM predictors failed to reliably predict all three DLBCL subtypes. When the DLBCL-B was used to train the model, only the HR clusters were associated by prediction (accuracy≥90%). When the DLBCL-A was used to train the model, only the BCR clusters were associated by the SVM classifier (accuracy≥90%), and none by the k-NN predictor (all accuracies≤51%). (See <a href="#pone.0001195.s001">Table S1</a> for full results). In addition, we tested a recently reported method, called clusterRepro <a href="#pone.0001195-Kapp1">[10]</a>. This method assumes a model of subclassification in one data set, and evaluates it in another data set. Again we only observed significant reproducibility (accuracy≥90%) for the HR cluster, in this case when DLBCL-A was used as the model set (<a href="#pone.0001195.s002">Table S2</a>).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>In this example we see the difficulty of identifying common subclasses and how SubMap outperforms four other approaches.</p>


<h4>Example 3. Breast cancer: estrogen receptor status</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We next compared SubMap method with another more straightforward approach, i.e., to merge the two independent data sets into a single data set, and and then to run clustering to find subtypes observed in the combined data. To this end, we employed two breast cancer data sets, Breast-A <a href="#pone.0001195-vantVeer1">[11]</a> and Breast-B <a href="#pone.0001195-West1">[12]</a> generated using one-channel oligonucleotide and two-channel microarrays, respectively. Hierarchical clustering, after simply merging the two data sets, only revealed the dominant experiment-of-origin-specific structure despite several attempts of normalization (<a href="#pone.0001195.s005">Figure S2</a>), whereas SubMap identified common subtypes associated with a key molecular determinant of a disease as described below.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Using hierarchical clustering as described in <a href="#s4">Materials and Methods</a>, we identified three (A<sub>1</sub>, A<sub>2</sub>, and A<sub>3</sub>) and four (B<sub>1</sub>, B<sub>2</sub>, B<sub>3</sub>, and B<sub>4</sub>) candidate subclasses in Breast-A and Breast-B, respectively (<a href="#pone-0001195-g004">Figure 4a</a>) as described in the <a href="#s4">Materials and Methods</a> section. SubMap revealed the common subtypes as two sets of significant associations, “A<sub>1</sub> ∪ A<sub>2</sub> and B<sub>1</sub> ∪ B<sub>3</sub>“ and “A<sub>3</sub> and B<sub>2</sub> ∪ B<sub>4</sub>“ (<a href="#pone-0001195-g004">Figure 4b</a>). We note the dominant substructure (i.e., first splitting in the dendrogram in <a href="#pone-0001195-g004">Figure 4a</a>) in Breast-B is B<sub>1</sub> ∪ B<sub>2</sub> and B<sub>3</sub> ∪ B<sub>4</sub>. Thus, the subgrouping of samples from Breast-B that associate with Breast-A might not be evident by looking at Breast-B alone.</p>
<div class="figure" id="pone-0001195-g004"><div class="img"><a name="pone-0001195-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Example 3: Common subtypes of breast cancer associated with estrogen receptor (ER) status.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(a) Candidate subclass labels were assigned using hierarchical clustering in Breast-A and Breast-B data sets independently. (b) Subclass association (SA) matrix for Breast-A and Breast-B. Bonferroni-corrected p-values for the combinations of “A<sub>1</sub> and B<sub>2</sub>”, “A<sub>1</sub> and B<sub>4</sub>“, “A<sub>2</sub> and B<sub>1</sub>”, “A<sub>3</sub> and B<sub>1</sub>”, and “A<sub>3</sub> and B<sub>3</sub>” were 0.070, 0.002, 0.023, 0.001, and 0.055, respectively (FDR-corrected p-values of 0.014, 0.001, 0.008, 0.001, and 0.014, respectively). *: ER status is missing for one case.</p>
<span>doi:10.1371/journal.pone.0001195.g004</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Both data sets have immunohistochemistry data of estrogen receptor (ER) status, which is known to play important role in breast cancer biology, e.g., BRCA1 mutation, lymphocytic infiltration, and early occurrence of distant metastasis <a href="#pone.0001195-vantVeer1">[11]</a>, <a href="#pone.0001195-Lakhani1">[13]</a>. In fact, this was the only sample-phenotype information common to both data sets. Thus, we sought to evaluate if the common subtypes obtained by our method captured this ancillary phenotypic information. We found that the common subtypes defined by SubMap were significantly associated with the ER status in each data set (Breast-A: ER was positive in 50/61 cases of A<sub>1</sub> ∪ A<sub>2</sub>, and 3/36 cases of A<sub>3</sub>. p = 4.2e-13. Breast-B: ER was positive in 18/19 cases of B<sub>1</sub> ∪ B<sub>3</sub>, and 7/23 cases of B<sub>2</sub> ∪ B<sub>4</sub>. p = 8.0e-7. Fisher's exact test). This result leads us to speculate that ER status may indeed be a significant factor in determining the subclasses and subclass associations.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>To investigate this hypothesis further, we evaluated the expression status of ER signaling-related genes in the common subtypes to confirm that the subtypes are relevant to the biology of ER signaling. In both input data sets, we ranked genes by the extent of their up-regulation in the subtypes with more ER positive cases (A<sub>1</sub> ∪ A<sub>2</sub> and B<sub>1</sub> ∪ B<sub>3</sub>), and evaluated the over-representation of the ER-related gene set (<a href="#pone.0001195.s003">Table S3</a>) using Gene Set Enrichment Analysis <a href="#pone.0001195-Subramanian1">[3]</a>. The ER-related gene set was significantly over-expressed with p-values of 0.033 and 0.010 in A<sub>1</sub> ∪ A<sub>2</sub> within Breast-A and B<sub>1</sub> ∪ B<sub>3</sub> within Breast-B, respectively.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>The association between A<sub>1</sub> and B<sub>3</sub> was not statistically significant (<a href="#pone-0001195-g004">Figure 4b</a>). In fact, the number of cases with lymph node metastasis in B<sub>1</sub> (8/12, 67%) is larger than that of B<sub>3</sub> (2/7, 29%). Although this difference is not statistically significant (p = 0.170, Fisher's exact test), this might suggest the existence of some biological heterogeneity among the ER-positive subclasses, and cause such “step-like” shape.</p>


<h4>Example 4. Diffuse large B-cell lymphoma: patient survival</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In this final example, we applied SubMap to see if identifying the corresponding subtypes in two diffuse large B-cell lymphoma data sets would enable us to distinguish subpopulations relevant to patient survival. We analyzed two independent DLBCL data sets, DLBCL-C (n = 58) <a href="#pone.0001195-Shipp1">[14]</a> and DLBCL-D (n = 129) <a href="#pone.0001195-Monti1">[7]</a>. DLBCL-D contains the samples in DLBCL-A from Example 2 for which survival information was available. DLBCL-C is a third independent data set. These data sets were generated using different generations of the Affymetrix GeneChip oligonucleotide microarray.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Using hierarchical clustering, four candidate subclasses were defined in each data set (C<sub>1</sub>, C<sub>2</sub>, C<sub>3</sub>, and C<sub>4</sub> in DLBCL-C, and D<sub>1</sub>, D<sub>2</sub>, D<sub>3</sub>, and D<sub>4</sub> in DLBCL-D). There was no statistically significant difference in the survival curves among these subclasses (data not shown). SubMap identified two significantly corresponding pairs of subclasses between these two data sets, “C<sub>3</sub> and D<sub>2</sub>” and “C<sub>4</sub>, and D<sub>3</sub>” (<a href="#pone-0001195-g005">Figure 5a</a>).</p>
<div class="figure" id="pone-0001195-g005"><div class="img"><a name="pone-0001195-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001195" data-uri="info:doi/10.1371/journal.pone.0001195.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001195.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001195.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001195.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001195.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Example 4: Survival prediction in Diffuse Large B-cell lymphoma (DLBCL) data sets.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(a) Subclass association (SA) matrix for the comparison between DLBCL-C and DLBCL-D data sets. Bonferroni-corrected p-values for the pairs of “C<sub>3</sub> and D<sub>2</sub>” and “C<sub>4</sub> and D<sub>3</sub>” were 0.002 and 0.002, respectively, (b) Survival prediction models were built using DLBCL-C and applied to DLBCL-D. Kaplan-Meier survival curves for the predicted groups in DLBCL-D are shown. Left: Prediction model was trained using all cases in DLBCL-C (n = 58), and tested on all cases in DLBCL-D (n = 129). Middle: Survival prediction using only cases from “matched” subclasses. Model was trained using C<sub>3</sub> ∪ C<sub>4</sub> samples (n = 25) and tested in D<sub>2</sub> ∪ D<sub>3</sub> (n = 61). The survival separation was better than that in the left panel in spite of having fewer samples. Right: survival prediction using only cases from “unmatched” subclasses. Model was trained using C<sub>1</sub> ∪ C<sub>2</sub> (n = 33) and tested in D<sub>1</sub> ∪ D<sub>4</sub> (n = 68). The numbers of events were 61, 22, and 39 for all (D<sub>1</sub> ∪ D<sub>2</sub> ∪ D<sub>3</sub> ∪ D<sub>4</sub>), “matched” (D<sub>2</sub> ∪ D<sub>3</sub>), and “unmatched” (D<sub>1 </sub>∪ D<sub>4</sub>) patients, respectively. p-values were calculated using the log-rank test. DFS: disease free survival.</p>
<span>doi:10.1371/journal.pone.0001195.g005</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>We next sought to determine whether we could improve the survival distinction between samples by restricting our consideration to the corresponding subclasses. Based on the sample labeling of poor prognosis in DLBCL-C defined in <a href="#pone.0001195-Shipp1">[14]</a>, we built a survival predictive model using a supervised learning method, k-nearest neighbor algorithm (k = 3, and 10 marker genes), and tested it in DLBCL-D (<a href="#pone-0001195-g005">Figure 5b</a>). When the model was built using only matched subclasses, C<sub>3</sub> ∪ C<sub>4</sub> (n = 25), and tested in their counterparts, D<sub>2</sub> ∪ D<sub>3</sub> (n = 61), the predictive model yielded better separation of survival than a model built using all of the samples (<a href="#pone-0001195-g005">Figure 5b</a>) in spite of having many fewer, nearly half, the number of samples to build and test the predictor. On the other hand, the predictor trained using unmatched subclasses, C<sub>1</sub> ∪ C<sub>2</sub> (n = 33) did not work on D<sub>1</sub> ∪ D<sub>4</sub> (n = 68) This suggests that the predictive survival signature common to both of the data sets is carried by the “corresponding” cases, which our method identified.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Genomic profiling is a powerful approach to discovering molecular disease subtypes. However, the uncovered subtypes may also reflect idiosyncrasies, technical variation, representational and measurement biases that limit the ability of the model and results to generalize to other studies or platforms. In this paper, we have shown how SubMap can address these issues and be an effective approach to recognizing subtypes across different data sets.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the four examples presented above the method clearly outperforms several more traditional, alternative approaches such as, i) using supervised models (e.g., k-NN and SVM) to learn the subclasses obtained in one of the datasets and then predicting the subclass membership of the samples in the second set, ii) merging the two data sets and clustering samples to search for common subtypes, iii) associating the subtypes between the data sets by assessing the overlap of each subtype's marker genes in both data sets, or iv) creating an explicit model of subclassification in one data set, and evaluating it in another data set (e.g., clusterRepro). All of these alternative methods have serious limitations and do not always produce reproducible results. While two different datasets may claim to represent the same biological phenotype, they often contain different molecular subtypes or biased representations and samplings with different dynamic ranges, measurement biases, probe efficiencies etc. These differences violate the common underlying assumption, of these other approaches-that two instances of the data, such as the train and test set, are samplings from the same probability distribution.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>SubMap associates the subclasses between datasets by using a more abstract, higher-level similarity measure (i.e., the enrichment score of ranked subclass markers). SubMap focuses on the concordant collective behavior of groups of genes rather than specific gene expression values, and does not rely on one particular data set to define a “model” like supervised learning methods. The result of this approach, which concurrently and bi-directionally evaluates similarity by treating the two input data sets equally within a self-contained unit, is that SubMap is able to provide solutions to the mapping problem that are more robust, more generalizable and more tolerant of the platform idiosyncrasies and biases.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>SubMap performs marker gene selection multiple times. For example, in the case where data set A and B have 3 and 4 subclasses, respectively, it performs maker gene selection 7 times, 3 times in data set A and 4 times in data set B. We cannot assume the same number of statistically “significant” marker genes for all of them, and to compute the significance for each round of marker selection by, e.g., permutation testing is highly computationally intensive. The GSEA enrichment score (ES) is weighted by the correlation between gene expression signals and the subclass. Therefore, genes having small correlation with phenotype (i.e., less significant genes) show negligible contribution in the computation of the ES. For this reason, there is no need to set significance thresholds to select the marker genes. The fixed number of marker gene set only needs to include sufficient significant marker genes to produce a robust ES (see <a href="#pone.0001195.s004">Figure S1</a>).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Another important and useful property of SubMap is that it helps to identify an appropriate level of subclass granularity to select the best overall concordance. Typically in unsupervised clustering, the depth of splitting of the clustering solution is somewhat arbitrary. Even when a model selection method is used, there is the possibility of over-fitting to a dataset structure that represents batch effects or platform idiosyncrasies. As a consequence, a dominant structure in one data set might not necessarily show the best concordance with the other data set. A concrete example of this was shown in Example 3, where the second dominant splitting in Breast-B, rather than the first, showed the best concordance with Breast-A. The clustering of the subclass association matrix gives an indication of which candidate subclasses show the best concordance between the data sets.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The use of more robust and generalized subclasses enables the better utilization of microarray data and the construction of better classification models. For example, the possibility of mapping the corresponding classes across a large collection of cross-platform or cross-laboratory datasets provides a method to increases the number of available samples related to a given phenotype. This translates to a better signal to noise ratio and thus improves the selection of biomarkers and the training of more accurate, robust, and generalizable supervised classifiers. Moreover, SubMap can provide a more precise stratification of the data for the application of predictive models. In Example 4, our method identified subpopulations in two independent data sets, where a clinical outcome predictor is more accurate than one modeled by all the samples. This additional resolution might help explain why an outcome predictor works only on a subset of the population. The integration and utilization of multiple datasets, representing a wider span of the biological space of interest, can both increase the accuracy of models as described above and provide models that are more interpretable, realistic, and biologically meaningful.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The ability of SubMap to find common subtypes across data sets is, of course, dependent on the representation of the subtypes in those data sets, and the quality and granularity of their labeling. This is true whether the samples are labeled manually by, for example, a pathologist, or computationally by clustering. At a coarser granularity (i.e., smaller number of larger candidate subclasses), a biological subclass of interest might be missed. On the other hand, a finer granularity (i.e., larger number of smaller candidate subclasses) could yield an unbalanced comparison in selecting the marker genes. In addition, too fine a class splitting might yield multiple similar candidate subclasses, which might cause a failure to detect distinct marker genes resulting in lower sensitivity in detecting subclass association. However, we note here that, aside from the issue of increasing runtime, there is no <em>a priori</em> limit on the number of the candidate subclasses that can be associated as long as they have robust corresponding marker genes.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>SubMap is an exploratory method, and the optimal (or biologically reasonable) number of the candidate subclasses would depend on the particular data sets with complex trade-offs. One possible approach is to define multiple possible candidate subclassifications in each data set, and evaluate the subclass association structure in all combinations of candidate subclassifications (<a href="#pone.0001195.s007">Figure S4</a>). To define an appropriate measure or statistic to evaluate such a search for optimal candidate subclasses will require further consideration and investigation.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>In summary, SubMap is a method that provides more general molecular subclass identification and correspondence among a collection of microarray data sets. By allowing the integration of multiple data sets measured on different platforms, from different laboratories, it increases the possibility of extracting more meaningful biological subclasses and their corresponding biomarkers and classifiers. This has the potential to improve gene expression-based clinical classifiers where robustness, generalization, and reproducibility across platforms are paramount.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>The entire methodology is implemented as the SubMap module of the GenePattern software and is freely available from <a href="http://www.broad.mit.edu/genepattern/">http://www.broad.mit.edu/genepattern/</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data preprocessing</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We started from data sets that were already normalized for their respective study without any additional normalization procedure to account for different platform derivation. For the signal intensity data generated by one-channel oligonucleotide microarrays, Affymetrix's GeneChip, we applied a lower threshold of 20U and a upper threshold of 16,000U. For the log2 transformed ratio data generated by cDNA microarrays, we first removed genes whose values were missing in more than 5% of the samples, and then imputed the missing values for the rest of the genes using a k-nearest neighbor algorithm <a href="#pone.0001195-Troyanskaya1">[15]</a> (ImputeMissingValues.KNN, in the GenePattern software package, <a href="http://www.broad.mit.edu/genepattern/">http://www.broad.mit.edu/genepattern/</a>).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Before marker gene selection, we used following gene filtering. For the oligonucleotide array data, only genes exhibiting at least 3-fold differential expression and an absolute difference of at least 100 units across the samples in the experiment were included. For the cDNA array data, only genes with an absolute log2 ratio greater than one and whose difference in log2 ratio across all the samples in the data set was greater than one were included.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Before applying the SubMap, each microarray probe ID was converted into its corresponding HUGO gene symbol (<a href="http://www.gene.ucl.ac.uk/nomenclature/">http://www.gene.ucl.ac.uk/nomenclature/</a>), and multiple probe data corresponding to a single gene symbol was averaged. The number of genes remaining for our analyses of multiple tissue types, DLBCL, breast cancer, and DLBCL (with survival data) data sets were 5565, 661, 1213, and 3795, respectively.</p>


<h4>Subclass mapping (SubMap)</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The algorithm and pseudocode for SubMap is shown in <a href="#pone.0001195.s008">Box S1</a>. We describe the algorithm in more detail below.</p>
<h5>i) Candidate subclasses.</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p>We emphasize here that SubMap is an approach to evaluate the relationship between otherwise determined subclass structures in independently generated data sets. Thus, any clustering method can be used to generate candidate subclass labels. Alternatively, biologically/clinically known phenotype information can be used as candidate subclass labels. For the specific examples described in the results section, we chose to use agglomerative hierarchical clustering to generate candidate subclass labels (<a href="#pone-0001195-g001">Figure 1a</a>) <a href="#pone.0001195-Eisen1">[16]</a>. We used the Pearson correlation coefficient as a distance metric and the average linkage method from the dChip software (<a href="http://www.biostat.harvard.edu/complab/dchip">www.biostat.harvard.edu/complab/dchip</a>). We determined subclass splitting by tracing from the root of the dendrogram. A subclass that contained at least 10% of the samples was considered as a possible candidate subclass. We set the maximum number of the candidate subclasses as four. Any other criteria could be used to define a candidate subclass.</p>

<h5>ii) Similarity between candidate subclasses in each data set: enrichment score (ES).</h5><a id="article1.body1.sec4.sec2.sec2.p1" name="article1.body1.sec4.sec2.sec2.p1"></a><p>Suppose data set A and B have <em>n<sub>A</sub></em> and <em>n<sub>B</sub></em> candidate subclasses, respectively. First, we take the top 100 differentially expressed genes between a candidate subclass (<em>CS</em>) and the rest of the candidate subclasses (<em>nonCS</em>) the data set A. Such marker genes are identified for each candidate subclass in A. In data set B, genes are ordered according to their differential expression with respect to <em>CS</em> vs. <em>nonCS</em>. To evaluate differential gene expression between <em>CS</em> and <em>nonCS</em>, any measure, e.g., <em>t</em>-statistic, can be used. In this study, we used the signal-to-noise ratio (SNR), (<em>μ<sub>CS</sub></em>-<em>μ<sub>nonCS</sub></em>)/(<em>σ<sub>CS</sub></em>+<em>σ<sub>nonCS</sub></em>), where <em>μ<sub>i</sub></em> and <em>σ<sub>i</sub></em> denote, respectively, the sample mean and sample standard deviation within class <em>CS</em> or <em>nonCS</em> <a href="#pone.0001195-Golub1">[17]</a>. Thus, every candidate subclass in A has a set of marker genes, and every candidate subclass in B has a gene list ranked by differential expression between the candidate subclass and the remaining candidate subclasses in B. Alternatively, other methods for gene selection and ranking such as neighborhood analysis <a href="#pone.0001195-Golub1">[17]</a>, Significance Analysis of Microarrays <a href="#pone.0001195-Tusher1">[18]</a>, and the false discovery rate (FDR)-based approaches <a href="#pone.0001195-Benjamini1">[19]</a> can easily be used.</p>
<a id="article1.body1.sec4.sec2.sec2.p2" name="article1.body1.sec4.sec2.sec2.p2"></a><p>For every pair-wise combination of <em>n<sub>A</sub></em> marker gene sets and <em>n<sub>B</sub></em> gene rankings, class similarity is measured by calculating a gene set enrichment score (ES) and estimating its significance as previously described in <a href="#pone.0001195-Subramanian1">[3]</a>. Briefly, in each ranked gene list for B<em><sub>j</sub></em> (<em>j</em>Є{1,…,<em>n<sub>B</sub></em>}), a marker gene set of a candidate subclass A<em><sub>i</sub></em> (<em>i</em>Є{1,…,<em>n<sub>A</sub></em>}) is identified, and its enrichment or over-representation at the top of the list is measured by computing a ES (observed ES<sub>A<em>i</em>B<em>j</em></sub>). The ES is essentially a Kolmogorov-Smirnov score weighted by the SNR.</p>
<a id="article1.body1.sec4.sec2.sec2.p3" name="article1.body1.sec4.sec2.sec2.p3"></a><p>We recompute the ES<sub>A<em>i</em>B<em>j</em></sub> for 1000 random assignments of the subclass labels in B<em><sub>j</sub></em> to generate a null distribution for ES<sub>A<em>i</em>B<em>j</em></sub>. We sort the resulting ES scores in the null distribution, and estimate the significance of the observed ES<sub>A<em>i</em>B<em>j</em></sub> from its rank in the list as a nominal p-value (p<sub>A<em>i</em>B<em>j</em></sub>). We repeat this process by interchanging the role of the data sets A and B. Thus, we compute p<sub>A<em>i</em>B<em>j</em></sub> and p<sub>B<em>j</em>A<em>i</em></sub>.</p>

<h5>iii) Significance of association between candidate subclasses.</h5><a id="article1.body1.sec4.sec2.sec3.p1" name="article1.body1.sec4.sec2.sec3.p1"></a><p>We next sought a summary statistic for p<sub>A<em>i</em>B<em>j</em></sub> and p<sub>B<em>j</em>A<em>i</em></sub> in order to obtain combined mutual marker gene enrichment information as a measure of similarity of subclasses in the two input data sets. We use the Fisher inverse chi-square statistic, <em>F<sub>ij</sub> = -2</em>(log(<em>p<sub>AiBj</sub></em>)<em>+</em>log(<em>p<sub>BjAi</sub></em>)) to summarize the two p-values (<a href="#pone-0001195-g001">Figure 1b</a>) <a href="#pone.0001195-Fisher1">[4]</a>, <a href="#pone.0001195-Becker1">[20]</a>. The significance of <em>F<sub>ij</sub></em> is estimated as a one-sided nominal p-value in a null distribution for <em>F<sub>ij</sub></em> generated by randomly picking one rank p-value from the null distribution for each of ES<sub>A<em>i</em>B<em>j</em></sub> and ES<sub>B<em>j</em>A<em>i</em></sub> and evaluating their Fisher statistic 10,000 times.</p>

<h5>iv) Subclass association (SA) matrix.</h5><a id="article1.body1.sec4.sec2.sec4.p1" name="article1.body1.sec4.sec2.sec4.p1"></a><p>We now have <em>n<sub>A</sub></em>*<em>n<sub>B</sub></em> nominal p-values for all <em>F<sub>ij</sub></em> (<em>i</em>Є{1,…,<em>n<sub>A</sub></em>}, <em>j</em>Є{1,…,<em>n<sub>B</sub></em>}). To correct for multiple hypothesis testing, a Bonferroni correction is applied. Alternatively, for cases with a larger number of candidate subclasses, we can estimate a FDR by multiplying each nominal p-value by <em>n<sub>A</sub></em>*<em>n<sub>B</sub></em> and dividing by the rank of the nominal p-value. Using these corrected p-values, we obtain a subclass association (SA) matrix, which represents the global association structure of the candidate subclasses in the two independent data sets. By applying two-way clustering on the SA matrix (we used hierarchical clustering), it becomes possible to evaluate whether there are common subclasses that exist in both data sets.</p>



<h4>Prediction analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For prediction analysis, we used k-nearest neighbor (k-NN) and support vector machine (SVM) modules from the GenePattern software. For k-NN, briefly, genes in the training data set were rank-ordered by the SNR. Using the selected marker genes, prediction was performed based upon a majority vote of the class membership of its k nearest neighbors in the training set weighted by the reciprocal of the cosine distance. The k and the number of marker genes were chosen so as to minimize the leave-one-out cross-validation (LOOCV) error rate in the training set.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>In Example 4, the prediction models were built based on LOOCV. Fractional use of each gene in the LOOCV models was calculated, and top 5 most frequently used genes were used for class prediction in the test data set (<a href="#pone.0001195.s006">Figure S3</a>).</p>


<h4>Survival analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Kaplan-Meir analysis and the log-rank test were performed using GenePattern SurvivalCurve and SurvivalDifference modules, respectively.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001195"></div><a name="pone.0001195.s001" id="pone.0001195.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001195.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001195.s002" id="pone.0001195.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001195.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001195.s003" id="pone.0001195.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001195.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.01 MB XLS)</p>
<a name="pone.0001195.s004" id="pone.0001195.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s004">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Effect of the number of marker genes on the result of SubMap.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001195.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.10 MB TIF)</p>
<a name="pone.0001195.s005" id="pone.0001195.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s005">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Two breast cancer data sets, Breast-A and Breast-B, are directly merged and clustered. (a) Each column was normalized by subtracting the column mean and divided by the column SD before clustering. (b) Gene expression data are converted to their rank in each column, and clustering was performed using the rank to compute the distance. Pearson correlation and the average linkage method were used for the hierarchical clustering.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001195.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.08 MB TIF)</p>
<a name="pone.0001195.s006" id="pone.0001195.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s006">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Genes used for the survival prediction in Example 4. Leave-one-out cross-validation (LOOCV) is performed using C<sub>3</sub> ∪ C<sub>4</sub> in DLBCL-C as described in the Method section. Bar indicates fractional use of each gene in LOOCV models. Box indicates top 5 most frequently used genes in LOOCV that is used for prediction in D<sub>2</sub> ∪ D<sub>3</sub> in DLBCL-D. Genes shown in red are also included in a prediction model built using all samples.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001195.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB TIF)</p>
<a name="pone.0001195.s007" id="pone.0001195.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s007">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Effect of granularity of the candidate subclasses on SubMap result. In Breast-A and Breast-B data sets in Example 3, the finest granularity (i.e., the largest number of candidate subclasses) was defined as subclasses having at least 10% of the cohort. Each subclass was labeled by “number of subclasses”-“data set”-“subclass number”. In Breast-A, we defined sets of two (2-A<sub>1</sub> and 2-A<sub>2</sub>) and three (3-A<sub>1</sub>, 3-A<sub>2</sub>, and 3-A<sub>3</sub>) candidate subclasses. In Breast-B, we defined sets of two (2-B<sub>1</sub> and 2-B<sub>2</sub>), four (4-B<sub>1</sub>, 4-B<sub>2</sub>, 4-B<sub>3</sub>, and 4-B<sub>4</sub>), and six (6-B<sub>1</sub>, 6-B<sub>2</sub>, 6-B<sub>3</sub>, 6-B<sub>4</sub>, 6-B<sub>5</sub>, and 6-B<sub>6</sub>) candidate subclasses. SubMap was performed on all combinations of sets of the candidate subclasses. When the coarsest granularity (i.e., the smallest number of candidate subclasses) was assumed in Breast-B, we observed no significant subclass association (left heatmaps). When finer granularity was assumed for Breast-B (middle heatmaps), significant “two-class” correspondence started to appear, indicating the coarsest granularity in Breast-B was not appropriate to find significant subclass association. The finest granularity for Breast-A derived more significant associations (middle bottom heatmap). When the finest granularity was assumed in Breast-B, a small fraction of samples (6-B<sub>6</sub>) showed no association with any subclasses in Breast-A (right heatmaps), suggesting that this is too fine a granularity yielding weaker marker genes and lower sensitivity to capture a counterpart of 6-B<sub>6</sub>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001195.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.13 MB TIF)</p>
<a name="pone.0001195.s008" id="pone.0001195.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001195.s008">Box S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Algorithm to generate a SA matrix.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001195.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.29 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Aravind Subramanian, Joshua Gould, Marc-Danie Nazaire, and Michael Reich for technical help, and Ken Ross and Stefano Monti for critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YH. Analyzed the data: YH. Contributed reagents/materials/analysis tools: YH JB. Wrote the paper: JM PT TG YH JB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001195-Michiels1" id="pone.0001195-Michiels1"></a>Michiels S, Koscielny S, Hill C (2005) Prediction of cancer outcome with microarrays: a multiple random validation strategy. Lancet  365: 488–492.  <ul class="find" data-citedArticleID="1019476" data-doi="10.1016/s0140-6736(05)17866-0"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)17866-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+cancer+outcome+with+microarrays%3A+a+multiple+random+validation+strategy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+cancer+outcome+with+microarrays%3A+a+multiple+random+validation+strategy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001195-Larkin1" id="pone.0001195-Larkin1"></a>Larkin JE, Frank BC, Gavras H, Sultana R, Quackenbush J (2005) Independence and reproducibility across microarray platforms. Nat Methods  2: 337–344.  <ul class="find" data-citedArticleID="1019474" data-doi="10.1038/nmeth757"><li><a href="http://dx.doi.org/10.1038/nmeth757" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independence+and+reproducibility+across+microarray+platforms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independence+and+reproducibility+across+microarray+platforms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001195-Subramanian1" id="pone.0001195-Subramanian1"></a>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, et al.  (2005) Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A  102: 15545–15550.  <ul class="find" data-citedArticleID="1019490" data-doi="10.1073/pnas.0506580102"><li><a href="http://dx.doi.org/10.1073/pnas.0506580102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001195-Fisher1" id="pone.0001195-Fisher1"></a>Fisher RA (1932) Statistical Methods for Research Workers. London: Oliver and Boyd.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0001195-Su1" id="pone.0001195-Su1"></a>Su AI, Cooke MP, Ching KA, Hakak Y, Walker JR, et al.  (2002) Large-scale analysis of the human and mouse transcriptomes. Proc Natl Acad Sci U S A  99: 4465–4470.  <ul class="find" data-citedArticleID="1019488" data-doi="10.1073/pnas.012025199"><li><a href="http://dx.doi.org/10.1073/pnas.012025199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+analysis+of+the+human+and+mouse+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+analysis+of+the+human+and+mouse+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001195-Ramaswamy1" id="pone.0001195-Ramaswamy1"></a>Ramaswamy S, Tamayo P, Rifkin R, Mukherjee S, Yeang CH, et al.  (2001) Multiclass cancer diagnosis using tumor gene expression signatures. Proc Natl Acad Sci U S A  98: 15149–15154.  <ul class="find" data-citedArticleID="1019482" data-doi="10.1073/pnas.211566398"><li><a href="http://dx.doi.org/10.1073/pnas.211566398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiclass+cancer+diagnosis+using+tumor+gene+expression+signatures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiclass+cancer+diagnosis+using+tumor+gene+expression+signatures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001195-Monti1" id="pone.0001195-Monti1"></a>Monti S, Savage KJ, Kutok JL, Feuerhake F, Kurtin P, et al.  (2005) Molecular profiling of diffuse large B-cell lymphoma identifies robust subtypes including one characterized by host inflammatory response. Blood  105: 1851–1861.  <ul class="find" data-citedArticleID="1019478" data-doi="10.1182/blood-2004-07-2947"><li><a href="http://dx.doi.org/10.1182/blood-2004-07-2947" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+profiling+of+diffuse+large+B-cell+lymphoma+identifies+robust+subtypes+including+one+characterized+by+host+inflammatory+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+profiling+of+diffuse+large+B-cell+lymphoma+identifies+robust+subtypes+including+one+characterized+by+host+inflammatory+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001195-Rosenwald1" id="pone.0001195-Rosenwald1"></a>Rosenwald A, Wright G, Chan WC, Connors JM, Campo E, et al.  (2002) The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma. N Engl J Med  346: 1937–1947.  <ul class="find" data-citedArticleID="1019484" data-doi="10.1056/nejmoa012914"><li><a href="http://dx.doi.org/10.1056/nejmoa012914" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+molecular+profiling+to+predict+survival+after+chemotherapy+for+diffuse+large-B-cell+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+molecular+profiling+to+predict+survival+after+chemotherapy+for+diffuse+large-B-cell+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001195-Monti2" id="pone.0001195-Monti2"></a>Monti S, Tamayo P, Mesirov J, Golub T (2003) Consensus Clustering: A resampling-based method for class discovery and visualization of gene expression microarray data. Machine Learning Journal  52: 91–118.  <ul class="find" data-citedArticleID="1019480"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Consensus+Clustering%3A+A+resampling-based+method+for+class+discovery+and+visualization+of+gene+expression+microarray+data.&amp;auth=&amp;atitle=Consensus+Clustering%3A+A+resampling-based+method+for+class+discovery+and+visualization+of+gene+expression+microarray+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consensus+Clustering%3A+A+resampling-based+method+for+class+discovery+and+visualization+of+gene+expression+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consensus+Clustering%3A+A+resampling-based+method+for+class+discovery+and+visualization+of+gene+expression+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001195-Kapp1" id="pone.0001195-Kapp1"></a>Kapp AV, Tibshirani R (2007) Are clusters found in one dataset present in another dataset? Biostatistics  8: 9–31.  <ul class="find" data-citedArticleID="1019470" data-doi="10.1093/biostatistics/kxj029"><li><a href="http://dx.doi.org/10.1093/biostatistics/kxj029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+clusters+found+in+one+dataset+present+in+another+dataset%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+clusters+found+in+one+dataset+present+in+another+dataset%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001195-vantVeer1" id="pone.0001195-vantVeer1"></a>van't Veer LJ, Dai H, van de Vijver MJ, He YD, Hart AA, et al.  (2002) Gene expression profiling predicts clinical outcome of breast cancer. Nature  415: 530–536.  <ul class="find" data-citedArticleID="1019498" data-doi="10.1038/415530a"><li><a href="http://dx.doi.org/10.1038/415530a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001195-West1" id="pone.0001195-West1"></a>West M, Blanchette C, Dressman H, Huang E, Ishida S, et al.  (2001) Predicting the clinical status of human breast cancer by using gene expression profiles. Proc Natl Acad Sci U S A  98: 11462–11467.  <ul class="find" data-citedArticleID="1019496" data-doi="10.1073/pnas.201162998"><li><a href="http://dx.doi.org/10.1073/pnas.201162998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+the+clinical+status+of+human+breast+cancer+by+using+gene+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+the+clinical+status+of+human+breast+cancer+by+using+gene+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001195-Lakhani1" id="pone.0001195-Lakhani1"></a>Lakhani SR, Jacquemier J, Sloane JP, Gusterson BA, Anderson TJ, et al.  (1998) Multifactorial analysis of differences between sporadic breast cancers and cancers involving BRCA1 and BRCA2 mutations. J Natl Cancer Inst  90: 1138–1145.  <ul class="find" data-citedArticleID="1019472" data-doi="10.1093/jnci/90.15.1138"><li><a href="http://dx.doi.org/10.1093/jnci/90.15.1138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multifactorial+analysis+of+differences+between+sporadic+breast+cancers+and+cancers+involving+BRCA1+and+BRCA2+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multifactorial+analysis+of+differences+between+sporadic+breast+cancers+and+cancers+involving+BRCA1+and+BRCA2+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001195-Shipp1" id="pone.0001195-Shipp1"></a>Shipp MA, Ross KN, Tamayo P, Weng AP, Kutok JL, et al.  (2002) Diffuse large B-cell lymphoma outcome prediction by gene-expression profiling and supervised machine learning. Nat Med  8: 68–74.  <ul class="find" data-citedArticleID="1019486" data-doi="10.1038/nm0102-68"><li><a href="http://dx.doi.org/10.1038/nm0102-68" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diffuse+large+B-cell+lymphoma+outcome+prediction+by+gene-expression+profiling+and+supervised+machine+learning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diffuse+large+B-cell+lymphoma+outcome+prediction+by+gene-expression+profiling+and+supervised+machine+learning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001195-Troyanskaya1" id="pone.0001195-Troyanskaya1"></a>Troyanskaya O, Cantor M, Sherlock G, Brown P, Hastie T, et al.  (2001) Missing value estimation methods for DNA microarrays. Bioinformatics  17: 520–525.  <ul class="find" data-citedArticleID="1019492" data-doi="10.1093/bioinformatics/17.6.520"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.6.520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Missing+value+estimation+methods+for+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Missing+value+estimation+methods+for+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001195-Eisen1" id="pone.0001195-Eisen1"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A  95: 14863–14868.  <ul class="find" data-citedArticleID="1019464" data-doi="10.1073/pnas.95.25.14863"><li><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cluster+analysis+and+display+of+genome-wide+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cluster+analysis+and+display+of+genome-wide+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001195-Golub1" id="pone.0001195-Golub1"></a>Golub TR, Slonim DK, Tamayo P, Huard C, Gaasenbeek M, et al.  (1999) Molecular classification of cancer: class discovery and class prediction by gene expression monitoring. Science  286: 531–537.  <ul class="find" data-citedArticleID="1019468" data-doi="10.1126/science.286.5439.531"><li><a href="http://dx.doi.org/10.1126/science.286.5439.531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+classification+of+cancer%3A+class+discovery+and+class+prediction+by+gene+expression+monitoring." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+classification+of+cancer%3A+class+discovery+and+class+prediction+by+gene+expression+monitoring.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001195-Tusher1" id="pone.0001195-Tusher1"></a>Tusher VG, Tibshirani R, Chu G (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc Natl Acad Sci U S A  98: 5116–5121.  <ul class="find" data-citedArticleID="1019494" data-doi="10.1073/pnas.091062498"><li><a href="http://dx.doi.org/10.1073/pnas.091062498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001195-Benjamini1" id="pone.0001195-Benjamini1"></a>Benjamini Y, Drai D, Elmer G, Kafkafi N, Golani I (2001) Controlling the false discovery rate in behavior genetics research. Behav Brain Res  125: 279–284.  <ul class="find" data-citedArticleID="1019462" data-doi="10.1016/s0166-4328(01)00297-2"><li><a href="http://dx.doi.org/10.1016/s0166-4328(01)00297-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+false+discovery+rate+in+behavior+genetics+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+false+discovery+rate+in+behavior+genetics+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001195-Becker1" id="pone.0001195-Becker1"></a>Becker BJ (1994)  Combining significance levels. In: Cooper H, Hedges LV, editors. The Handbook of Research Synthesis. New York: Russell Sage Foundation. pp. 215–230.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.XML" value="81578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.PDF" value="338679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g001.PNG_L" value="131762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g001.PNG_M" value="79422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g001.PNG_S" value="14707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g001.TIF" value="280212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g001.PNG_I" value="75898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.t001.PNG_L" value="184744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.t001.PNG_M" value="104712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.t001.PNG_S" value="11457"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.t001.TIF" value="767250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.t001.PNG_I" value="31857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g002.PNG_L" value="36441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g002.PNG_M" value="49115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g002.PNG_S" value="10917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g002.TIF" value="155366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g002.PNG_I" value="18730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g003.PNG_L" value="22662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g003.PNG_M" value="44055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g003.PNG_S" value="9986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g003.TIF" value="57616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g003.PNG_I" value="22416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g004.PNG_L" value="49115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g004.PNG_M" value="60812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g004.PNG_S" value="9214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g004.TIF" value="186434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g004.PNG_I" value="23592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g005.PNG_L" value="64942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g005.PNG_M" value="78706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g005.PNG_S" value="10469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g005.TIF" value="180672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.g005.PNG_I" value="29617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s001.XLS" value="17408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s002.XLS" value="16384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s003.XLS" value="14848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s004.TIF" value="98483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s005.TIF" value="97166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s006.TIF" value="72483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s007.TIF" value="162836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001195.s008.DOC" value="285696"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001195&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001195" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001195&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001195&volume=&issue=&title=Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets&author_name=Yujin%20Hoshida%2C%20Jean-Philippe%20Brunet%2C%20Pablo%20Tamayo%2C%20Todd%20R.%20Golub%2C%20Jill%20P.%20Mesirov&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001195" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001195" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001195" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001195&amp;t=Subclass%20Mapping%3A%20Identifying%20Common%20Subtypes%20in%20Independent%20Disease%20Data%20Sets" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001195&title=Subclass%20Mapping%3A%20Identifying%20Common%20Subtypes%20in%20Independent%20Disease%20Data%20Sets&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001195&amp;title=Subclass%20Mapping%3A%20Identifying%20Common%20Subtypes%20in%20Independent%20Disease%20Data%20Sets" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001195" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001195&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Subclass Mapping: Identifying Common Subtypes in Independent Disease Data Sets';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001195';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Subclass%20Mapping%3A%20Identifying%20Common%20Subtypes%20in%20Independent%20Disease%20Data%20Sets http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001195" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001195" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Breast+cancer%22" title="Search for articles in the subject area:'Breast cancer'"><div class="flagText">Breast cancer</div></a>
              <div data-categoryid="18763" data-articleid="25222"
                   data-categoryname="Breast cancer"
                   class="flagImage" title="Flag 'Breast cancer' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Breast+tissue%22" title="Search for articles in the subject area:'Breast tissue'"><div class="flagText">Breast tissue</div></a>
              <div data-categoryid="47845" data-articleid="25222"
                   data-categoryname="Breast tissue"
                   class="flagImage" title="Flag 'Breast tissue' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Diffuse+large+B-cell+lymphoma%22" title="Search for articles in the subject area:'Diffuse large B-cell lymphoma'"><div class="flagText">Diffuse large B-cell lymphoma</div></a>
              <div data-categoryid="37973" data-articleid="25222"
                   data-categoryname="Diffuse large B-cell lymphoma"
                   class="flagImage" title="Flag 'Diffuse large B-cell lymphoma' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Forecasting%22" title="Search for articles in the subject area:'Forecasting'"><div class="flagText">Forecasting</div></a>
              <div data-categoryid="34789" data-articleid="25222"
                   data-categoryname="Forecasting"
                   class="flagImage" title="Flag 'Forecasting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25222"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Marker+genes%22" title="Search for articles in the subject area:'Marker genes'"><div class="flagText">Marker genes</div></a>
              <div data-categoryid="23481" data-articleid="25222"
                   data-categoryname="Marker genes"
                   class="flagImage" title="Flag 'Marker genes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25222"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="25222"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1872'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7954'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9090&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
