

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, Nematostella vectensis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000153"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000153" />

    <meta name="citation_title" content="Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, Nematostella vectensis"/>
    <meta itemprop="name" content="Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, Nematostella vectensis"/>

      <meta name="citation_author" content="Joseph F. Ryan"/>
            <meta name="citation_author_institution" content="Bioinformatics Program, Boston University, Boston, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="Genome Technology Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Maureen E. Mazza"/>
            <meta name="citation_author_institution" content="Department of Biology, Boston University, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Kevin Pang"/>
            <meta name="citation_author_institution" content="Kewalo Marine Laboratory, Pacific Bioscience Research Center, University of Hawaii, Honolulu, Hawaii, United States of America"/>
      <meta name="citation_author" content="David Q. Matus"/>
            <meta name="citation_author_institution" content="Kewalo Marine Laboratory, Pacific Bioscience Research Center, University of Hawaii, Honolulu, Hawaii, United States of America"/>
      <meta name="citation_author" content="Andreas D. Baxevanis"/>
            <meta name="citation_author_institution" content="Genome Technology Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Mark Q. Martindale"/>
            <meta name="citation_author_institution" content="Kewalo Marine Laboratory, Pacific Bioscience Research Center, University of Hawaii, Honolulu, Hawaii, United States of America"/>
      <meta name="citation_author" content="John R. Finnerty"/>
            <meta name="citation_author_institution" content="Bioinformatics Program, Boston University, Boston, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biology, Boston University, Boston, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/1/24"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000153.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e153"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The branchial Hox code and its implications for gene regulation, patterning of the nervous system and head evolution.; citation_author=P Hunt; citation_author=J Whiting; citation_author=S Nonchev; citation_author=MH Sham; citation_author=H Marshall; citation_journal_title=Development Suppl; citation_volume=2; citation_number=1; citation_pages=63-77; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The zootype and the phylotypic stage.; citation_author=JM Slack; citation_author=PW Holland; citation_author=CF Graham; citation_journal_title=Nature; citation_volume=361; citation_number=2; citation_pages=490-492; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Hox genes and the evolution of vertebrate axial morphology.; citation_author=AC Burke; citation_author=CE Nelson; citation_author=BA Morgan; citation_author=C Tabin; citation_journal_title=Development; citation_volume=121; citation_number=3; citation_pages=333-346; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Hox genes and the diversification of insect and crustacean body plans.; citation_author=M Averof; citation_author=M Akam; citation_journal_title=Nature; citation_volume=376; citation_number=4; citation_pages=420-423; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Crustacean appendage evolution associated with changes in Hox gene expression.; citation_author=M Averof; citation_author=NH Patel; citation_journal_title=Nature; citation_volume=388; citation_number=5; citation_pages=682-686; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Exploring the myriapod body plan: expression patterns of the ten Hox genes in a centipede.; citation_author=CL Hughes; citation_author=TC Kaufman; citation_journal_title=Development; citation_volume=129; citation_number=6; citation_pages=1225-1238; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Insights into the urbilaterian brain: conserved genetic patterning mechanisms in insect and vertebrate brain development.; citation_author=R Lichtneckert; citation_author=H Reichert; citation_journal_title=Heredity; citation_volume=94; citation_number=7; citation_pages=465-477; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The murine Hox-2 genes display dynamic dorsoventral patterns of expression during central nervous system development.; citation_author=A Graham; citation_author=M Maden; citation_author=R Krumlauf; citation_journal_title=Development; citation_volume=112; citation_number=8; citation_pages=255-264; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Region-specific gastrointestinal Hox code during murine embryonal gut development.; citation_author=Y Kawazoe; citation_author=T Sekimoto; citation_author=M Araki; citation_author=K Takagi; citation_author=K Araki; citation_journal_title=Dev Growth Differ; citation_volume=44; citation_number=9; citation_pages=77-84; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=HoxA and HoxB cluster genes subdivide the digestive tract into morphological domains during chick development.; citation_author=J Sakiyama; citation_author=Y Yokouchi; citation_author=A Kuroiwa; citation_journal_title=Mech Dev; citation_volume=101; citation_number=10; citation_pages=233-236; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Characterization of the Hox cluster from the mosquito Anopheles gambiae (Diptera: Culicidae).; citation_author=TP Powers; citation_author=J Hogan; citation_author=Z Ke; citation_author=K Dymbrowski; citation_author=X Wang; citation_journal_title=Evol Dev; citation_volume=2; citation_number=11; citation_pages=311-325; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Organization of the Hox gene cluster in the grasshopper, Schistocerca gregaria.; citation_author=DE Ferrier; citation_author=M Akam; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=12; citation_pages=13024-13029; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Characterization of the Hox gene cluster in the malaria vector mosquito, Anopheles gambiae.; citation_author=MP Devenport; citation_author=C Blass; citation_author=P Eggleston; citation_journal_title=Evol Dev; citation_volume=2; citation_number=13; citation_pages=326-339; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequence of the Tribolium castaneum homeotic complex: the region corresponding to the Drosophila melanogaster antennapedia complex.; citation_author=SJ Brown; citation_author=JP Fellers; citation_author=TD Shippy; citation_author=EA Richardson; citation_author=M Maxwell; citation_journal_title=Genetics; citation_volume=160; citation_number=14; citation_pages=1067-1074; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Homeobox genes in the ribbonworm Lineus sanguineus: evolutionary implications.; citation_author=M Kmita-Cunisse; citation_author=F Loosli; citation_author=J Bierne; citation_author=WJ Gehring; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=15; citation_pages=3030-3035; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Archetypal organization of the amphioxus Hox gene cluster.; citation_author=J Garcia-Fernandez; citation_author=PW Holland; citation_journal_title=Nature; citation_volume=370; citation_number=16; citation_pages=563-566; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=No more than 14: the end of the amphioxus Hox cluster.; citation_author=C Minguillon; citation_author=J Gardenyes; citation_author=E Serra; citation_author=LF Castro; citation_author=A Hill-Force; citation_journal_title=Int J Biol Sci; citation_volume=1; citation_number=17; citation_pages=19-23; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Why are Hox genes clustered?; citation_author=RS Mann; citation_journal_title=Bioessays; citation_volume=19; citation_number=18; citation_pages=661-664; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Ciona intestinalis ParaHox genes: evolution of Hox/ParaHox cluster integrity, developmental mode, and temporal colinearity.; citation_author=DE Ferrier; citation_author=PW Holland; citation_journal_title=Mol Phylogenet Evol; citation_volume=24; citation_number=19; citation_pages=412-417; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Conservation of regulatory sequences and gene expression patterns in the disintegrating Drosophila Hox gene complex.; citation_author=B Negre; citation_author=S Casillas; citation_author=M Suzanne; citation_author=E Sanchez-Herrero; citation_author=M Akam; citation_journal_title=Genome Res; citation_volume=15; citation_number=20; citation_pages=692-700; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for a dispersed Hox gene cluster in the platyhelminth parasite Schistosoma mansoni.; citation_author=RJ Pierce; citation_author=W Wu; citation_author=H Hirai; citation_author=A Ivens; citation_author=LD Murphy; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=21; citation_pages=2491-2503; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Remodelling of the homeobox gene complement in the tunicate Oikopleura dioica.; citation_author=RB Edvardsen; citation_author=HC Seo; citation_author=MF Jensen; citation_author=A Mialon; citation_author=J Mikhaleva; citation_journal_title=Curr Biol; citation_volume=15; citation_number=22; citation_pages=R12-13; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Unusual gene order and organization of the sea urchin hox cluster.; citation_author=RA Cameron; citation_author=L Rowen; citation_author=R Nesbitt; citation_author=S Bloom; citation_author=JP Rast; citation_journal_title=J Exp Zoolog B Mol Dev Evol; citation_volume=306; citation_number=23; citation_pages=45-58; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A new split of the Hox gene complex in Drosophila: relocation and evolution of the gene labial.; citation_author=B Negre; citation_author=JM Ranz; citation_author=F Casals; citation_author=M Caceres; citation_author=A Ruiz; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=24; citation_pages=2042-2054; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Splitting the bithorax complex of Drosophila.; citation_author=G Struhl; citation_journal_title=Nature; citation_volume=308; citation_number=25; citation_pages=454-456; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Splits in fruitfly hox gene complexes [letter].; citation_author=G VonAllmen; citation_author=I Hogga; citation_author=A Spierer; citation_author=F Karch; citation_author=W Bender; citation_journal_title=Nature; citation_volume=380; citation_number=26; citation_pages=116; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Structural changes in the antennapedia complex of Drosophila pseudoobscura.; citation_author=FM Randazzo; citation_author=MA Seeger; citation_author=CA Huss; citation_author=MA Sweeney; citation_author=JK Cecil; citation_journal_title=Genetics; citation_volume=134; citation_number=27; citation_pages=319-330; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Homeobox genes in the developing central nervous system.; citation_author=E Boncinelli; citation_author=A Simeone; citation_author=D Acampora; citation_author=M Gulisano; citation_journal_title=Ann Genet; citation_volume=36; citation_number=28; citation_pages=30-37; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Hox genes are not always Colinear.; citation_author=AS Monteiro; citation_author=DE Ferrier; citation_journal_title=Int J Biol Sci; citation_volume=2; citation_number=29; citation_pages=95-103; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Organizing axes in time and space; 25 years of colinear tinkering.; citation_author=M Kmita; citation_author=D Duboule; citation_journal_title=Science; citation_volume=301; citation_number=30; citation_pages=331-333; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Chromatin decondensation and nuclear reorganization of the HoxB locus upon induction of transcription.; citation_author=S Chambeyron; citation_author=WA Bickmore; citation_journal_title=Genes Dev; citation_volume=18; citation_number=31; citation_pages=1119-1130; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Inversion-induced disruption of the Hoxd cluster leads to the partition of regulatory landscapes.; citation_author=F Spitz; citation_author=C Herkenne; citation_author=MA Morris; citation_author=D Duboule; citation_journal_title=Nat Genet; citation_volume=37; citation_number=32; citation_pages=889-893; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chromosomal continuity in the abdominal region of the bithorax complex of Drosophila is not essential for its contribution to metameric identity.; citation_author=SY Tiong; citation_author=JR Whittle; citation_author=MC Gribbin; citation_journal_title=Development; citation_volume=101; citation_number=33; citation_pages=135-142; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Evolution of a transcriptional repression domain in an insect Hox protein.; citation_author=R Galant; citation_author=SB Carroll; citation_journal_title=Nature; citation_volume=415; citation_number=34; citation_pages=910-913; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ultrabithorax regulates genes at several levels of the wing-patterning hierarchy to shape the development of the Drosophila haltere.; citation_author=SD Weatherbee; citation_author=G Halder; citation_author=J Kim; citation_author=A Hudson; citation_author=S Carroll; citation_journal_title=Genes Dev; citation_volume=12; citation_number=35; citation_pages=1474-1482; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Origins of bilateral symmetry: Hox and dpp expression in a sea anemone.; citation_author=JR Finnerty; citation_author=K Pang; citation_author=P Burton; citation_author=D Paulson; citation_author=MQ Martindale; citation_journal_title=Science; citation_volume=304; citation_number=36; citation_pages=1335-1337; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Homeodomain Proteins.; citation_author=TR Bürglin; citation_number=37; citation_date=2005; citation_publisher=Wiley-VCH Verlag; " />
      <meta name="citation_reference" content="citation_title=NK-2 homeobox genes and heart development.; citation_author=RP Harvey; citation_journal_title=Dev Biol; citation_volume=178; citation_number=38; citation_pages=203-216; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Pax genes in eye development and evolution.; citation_author=Z Kozmik; citation_journal_title=Curr Opin Genet Dev; citation_volume=15; citation_number=39; citation_pages=430-438; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Morphological and developmental macroevolution: a paleontological perspective.; citation_author=JW Valentine; citation_author=D Jablonski; citation_journal_title=Int J Dev Biol; citation_volume=47; citation_number=40; citation_pages=517-522; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Hox, ParaHox, ProtoHox: facts and guesses.; citation_author=J Garcia-Fernandez; citation_journal_title=Heredity; citation_volume=94; citation_number=41; citation_pages=145-152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluating hypotheses of basal animal phylogeny using complete sequences of large and small subunit rRNA.; citation_author=M Medina; citation_author=AG Collins; citation_author=JD Silberman; citation_author=ML Sogin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=42; citation_pages=9707-9712; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evaluating multiple alternative hypotheses for the origin of Bilateria: an analysis of 18S rRNA molecular evidence.; citation_author=AG Collins; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=43; citation_pages=15458-15463; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Homeoboxes in sea anemones (Cnidaria:Anthozoa): a PCR-based survey of Nematostella vectensis and Metridium senile.; citation_author=JR Finnerty; citation_author=MQ Martindale; citation_journal_title=Biol Bull; citation_volume=193; citation_number=44; citation_pages=62-76; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Homeoboxes in sea anemones and other nonbilaterian animals: implications for the evolution of the Hox cluster and the zootype.; citation_author=JR Finnerty; citation_journal_title=Curr Top Dev Biol; citation_volume=40; citation_number=45; citation_pages=211-254; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Ancient origins of axial patterning genes: Hox genes and ParaHox genes in the Cnidaria.; citation_author=JR Finnerty; citation_author=MQ Martindale; citation_journal_title=Evol Dev; citation_volume=1; citation_number=46; citation_pages=16-23; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Combined-method phylogenetic analysis of Hox and ParaHox genes of the metazoa.; citation_author=MJ Kourakis; citation_author=MQ Martindale; citation_journal_title=J Exp Zool; citation_volume=288; citation_number=47; citation_pages=175-191; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cnidarian homeoboxes and the zootype.; citation_author=DE Martinez; citation_author=D Bridge; citation_author=LM Masuda-Nakagawa; citation_author=P Cartwright; citation_journal_title=Nature; citation_volume=393; citation_number=48; citation_pages=748-749; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Axial patterning and diversification in the cnidaria predate the Hox system.; citation_author=K Kamm; citation_author=B Schierwater; citation_author=W Jakob; citation_author=SL Dellaporta; citation_author=DJ Miller; citation_journal_title=Curr Biol; citation_volume=16; citation_number=49; citation_pages=920-926; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ancient complexity of the non-Hox ANTP gene complement in the anthozoan Nematostella vectensis. Implications for the evolution of the ANTP superclass.; citation_author=K Kamm; citation_author=B Schierwater; citation_journal_title=J Exp Zoolog B Mol Dev Evol; citation_number=50; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Minimal ProtoHox cluster inferred from bilaterian and cnidarian Hox complements.; citation_author=D Chourrout; citation_author=F Delsuc; citation_author=P Chourrout; citation_author=RB Edvardsen; citation_author=F Rentzsch; citation_journal_title=Nature; citation_volume=442; citation_number=51; citation_pages=684-687; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The cnidarian-bilaterian ancestor possessed at least 56 homeoboxes. Evidence from the starlet sea anemone, Nematostella vectensis.; citation_author=JF Ryan; citation_author=PM Burton; citation_author=M Mazza; citation_author=GK Kwong; citation_author=JC Mullikin; citation_journal_title=Genome Biology In press; citation_number=52; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=StellaBase: the Nematostella vectensis Genomics Database.; citation_author=JC Sullivan; citation_author=JF Ryan; citation_author=JA Watson; citation_author=J Webb; citation_author=JC Mullikin; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=53; citation_pages=D495-499; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=StellaBase—The Nematostella vectensis Genomics Database; citation_number=54; " />
      <meta name="citation_reference" content="citation_title=Joint Genome Institute—Nematostella vectensis genome portal, v1.0; citation_number=55; " />
      <meta name="citation_reference" content="citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=56; citation_pages=D501-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dorso/ventral genes are asymmetrically expressed and involved in germ-layer demarcation during cnidarian gastrulation.; citation_author=DQ Matus; citation_author=GH Thomsen; citation_author=MQ Martindale; citation_journal_title=Curr Biol; citation_volume=16; citation_number=57; citation_pages=499-505; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=TA Stephenson; citation_number=58; citation_pages=148 p; citation_date=1928; citation_publisher=The Ray Society.; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence for deep evolutionary roots of bilaterality in animal development.; citation_author=DQ Matus; citation_author=K Pang; citation_author=H Marlow; citation_author=CW Dunn; citation_author=GH Thomsen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=59; citation_pages=11195-11200; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Localized expression of a dpp/BMP2/4 ortholog in a coral embryo.; citation_author=DC Hayward; citation_author=G Samuel; citation_author=PC Pontynen; citation_author=J Catmull; citation_author=R Saint; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=60; citation_pages=8106-8111; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Asymmetric expression of the BMP antagonists chordin and gremlin in the sea anemone Nematostella vectensis: Implications for the evolution of axial patterning.; citation_author=F Rentzsch; citation_author=R Anton; citation_author=M Saina; citation_author=M Hammerschmidt; citation_author=TW Holstein; citation_journal_title=Dev Biol; citation_number=61; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Early evolution of a homeobox gene: the parahox gene Gsx in the Cnidaria and the Bilateria.; citation_author=JR Finnerty; citation_author=D Paulson; citation_author=P Burton; citation_author=K Pang; citation_author=MQ Martindale; citation_journal_title=Evol Dev; citation_volume=5; citation_number=62; citation_pages=331-345; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution of Antp-class genes and differential expression of Hydra Hox/paraHox genes in anterior patterning.; citation_author=D Gauchat; citation_author=F Mazet; citation_author=C Berney; citation_author=M Schummer; citation_author=S Kreger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=63; citation_pages=4493-4498; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Conserved and divergent genes in apex and axis development of cnidarians.; citation_author=B Galliot; citation_journal_title=Curr Opin Genet Dev; citation_volume=10; citation_number=64; citation_pages=629-637; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Gene structure and larval expression of cnox-2Am from the coral Acropora millepora.; citation_author=DC Hayward; citation_author=J Catmull; citation_author=JS Reece-Hoyes; citation_author=H Berghammer; citation_author=H Dodd; citation_journal_title=Dev Genes Evol; citation_volume=211; citation_number=65; citation_pages=10-19; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Homeobox genes in the cnidarian Eleutheria dichotoma: evolutionary implications for the origin of Antennapedia-class (HOM/Hox) genes.; citation_author=K Kuhn; citation_author=B Streit; citation_author=B Schierwater; citation_journal_title=Mol Phylogenet Evol; citation_volume=6; citation_number=66; citation_pages=30-38; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Isolation of Hox genes from the scyphozoan Cassiopeia xamachana: implications for the early evolution of Hox genes.; citation_author=K Kuhn; citation_author=B Streit; citation_author=B Schierwater; citation_journal_title=J Exp Zool; citation_volume=285; citation_number=67; citation_pages=63-75; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The HOX-like gene Cnox2-Pc is expressed at the anterior region in all life cycle stages of the jellyfish Podocoryne carnea.; citation_author=LM Masuda-Nakagawa; citation_author=H Groer; citation_author=BL Aerne; citation_author=V Schmid; citation_journal_title=Dev Genes Evol; citation_volume=210; citation_number=68; citation_pages=151-156; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Presence of eight distinct homeobox-containing genes in cnidarians.; citation_author=M Naito; citation_author=H Ishiguro; citation_author=T Fujisawa; citation_author=Y Kurosawa; citation_journal_title=FEBS Lett; citation_volume=333; citation_number=69; citation_pages=271-274; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Detection of homeobox genes in development and evolution.; citation_author=MT Murtha; citation_author=JF Leckman; citation_author=FH Ruddle; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=70; citation_pages=10711-10715; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Homeoboxes in cnidarians.; citation_author=B Schierwater; citation_author=M Murtha; citation_author=M Dick; citation_author=FH Ruddle; citation_author=LW Buss; citation_journal_title=J Exp Zool; citation_volume=260; citation_number=71; citation_pages=413-416; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=HOM/HOX homeobox genes are present in hydra (Chlorohydra viridissima) and are differentially expressed during regeneration.; citation_author=M Schummer; citation_author=I Scheurlen; citation_author=C Schaller; citation_author=B Galliot; citation_journal_title=Embo J; citation_volume=11; citation_number=72; citation_pages=1815-1823; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Expression of Cnox-2, a HOM/HOX homeobox gene in hydra, is correlated with axial pattern formation.; citation_author=MA Shenk; citation_author=HR Bode; citation_author=RE Steele; citation_journal_title=Development; citation_volume=117; citation_number=73; citation_pages=657-667; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Expression of Cnox-2, a HOM/HOX gene, is suppressed during head formation in hydra.; citation_author=MA Shenk; citation_author=L Gee; citation_author=RE Steele; citation_author=HR Bode; citation_journal_title=Dev Biol; citation_volume=160; citation_number=74; citation_pages=108-118; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Life stage and tissue-specific expression of the homeobox gene cnox1-Pc of the hydrozoan Podocoryne carnea.; citation_author=BL Aerne; citation_author=CD Baader; citation_author=V Schmid; citation_journal_title=Dev Biol; citation_volume=169; citation_number=75; citation_pages=547-556; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The ParaHox gene cluster is an evolutionary sister of the Hox gene cluster.; citation_author=NM Brooke; citation_author=J Garcia-Fernandez; citation_author=PW Holland; citation_journal_title=Nature; citation_volume=392; citation_number=76; citation_pages=920-922; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=77; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Neighbor-net: an agglomerative method for the construction of phylogenetic networks.; citation_author=D Bryant; citation_author=V Moulton; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=78; citation_pages=255-265; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Application of phylogenetic networks in evolutionary studies.; citation_author=DH Huson; citation_author=D Bryant; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=79; citation_pages=254-267; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Theoretical approaches to the analysis of homeobox gene evolution.; citation_author=C Kappen; citation_journal_title=Comput Chem; citation_volume=20; citation_number=80; citation_pages=49-59; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The Antennapedia-type homeobox genes have evolved from three precursors separated early in metazoan evolution.; citation_author=FR Schubert; citation_author=K Nieselt-Struwe; citation_author=P Gruss; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=81; citation_pages=143-147; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Tgfbeta signaling acts on a Hox response element to confer specificity and diversity to Hox protein function.; citation_author=A Grienenberger; citation_author=S Merabet; citation_author=J Manak; citation_author=I Iltis; citation_author=A Fabre; citation_journal_title=Development; citation_volume=130; citation_number=82; citation_pages=5445-5455; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Homeodomain proteins.; citation_author=WJ Gehring; citation_author=M Affolter; citation_author=T Burglin; citation_journal_title=Annu Rev Biochem; citation_volume=63; citation_number=83; citation_pages=487-526; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Evolution of Antennapedia-class homeobox genes.; citation_author=J Zhang; citation_author=M Nei; citation_journal_title=Genetics; citation_volume=142; citation_number=84; citation_pages=295-303; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=EST analysis of the cnidarian Acropora millepora reveals extensive gene loss and rapid sequence divergence in the model invertebrates.; citation_author=RD Kortschak; citation_author=G Samuel; citation_author=R Saint; citation_author=DJ Miller; citation_journal_title=Curr Biol; citation_volume=13; citation_number=85; citation_pages=2190-2195; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Maintenance of ancestral complexity and non-metazoan genes in two basal cnidarians.; citation_author=U Technau; citation_author=S Rudd; citation_author=P Maxwell; citation_author=PM Gordon; citation_author=M Saina; citation_journal_title=Trends Genet; citation_volume=21; citation_number=86; citation_pages=633-639; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Region-specific expression of murine Hox genes implies the Hox code-mediated patterning of the digestive tract.; citation_author=T Sekimoto; citation_author=K Yoshinobu; citation_author=M Yoshida; citation_author=S Kuratani; citation_author=S Fujimoto; citation_journal_title=Genes Cells; citation_volume=3; citation_number=87; citation_pages=51-64; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genetic and developmental bases of serial homology in vertebrate limb evolution.; citation_author=I Ruvinsky; citation_author=JJ Gibson-Brown; citation_journal_title=Development; citation_volume=127; citation_number=88; citation_pages=5233-5244; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A conserved Hox axis in the mouse and human female reproductive system: late establishment and persistent adult expression of the Hoxa cluster genes.; citation_author=HS Taylor; citation_author=GB Vanden Heuvel; citation_author=P Igarashi; citation_journal_title=Biol Reprod; citation_volume=57; citation_number=89; citation_pages=1338-1345; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Gene structure and developmental expression of an otx gene in the sea anemone Nematostella; citation_author=ME Mazza; citation_author=K Pang; citation_author=M Martindale; citation_author=J Finnerty; citation_number=90; citation_date=submitted; " />
      <meta name="citation_reference" content="citation_title=Unexpected complexity of the Wnt gene family in a sea anemone.; citation_author=A Kusserow; citation_author=K Pang; citation_author=C Sturm; citation_author=M Hrouda; citation_author=J Lentfer; citation_journal_title=Nature; citation_volume=433; citation_number=91; citation_pages=156-160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record.; citation_author=KJ Peterson; citation_author=NJ Butterfield; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=92; citation_pages=9547-9552; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Medusozoan phylogeny and character evolution clarified by new large and small subunit rDNA data and an assessment of the utility of phylogenetic mixture models.; citation_author=AG Collins; citation_author=P Schuchert; citation_author=AC Marques; citation_author=T Jankowski; citation_author=M Medina; citation_journal_title=Syst Biol; citation_volume=55; citation_number=93; citation_pages=97-115; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Spatial expression of Hox cluster genes in the ontogeny of a sea urchin.; citation_author=C Arenas-Mena; citation_author=AR Cameron; citation_author=EH Davidson; citation_journal_title=Development; citation_volume=127; citation_number=94; citation_pages=4631-4643; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cephalopod Hox genes and the origin of morphological novelties.; citation_author=PN Lee; citation_author=P Callaerts; citation_author=HG De Couet; citation_author=MQ Martindale; citation_journal_title=Nature; citation_volume=424; citation_number=95; citation_pages=1061-1065; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=[Extreme divergence of a homeotic gene: the bicoid case].; citation_author=F Bonneton; citation_journal_title=Med Sci (Paris); citation_volume=19; citation_number=96; citation_pages=1265-1270; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Expression patterns of the rogue Hox genes Hox3/zen and fushi tarazu in the apterygote insect Thermobia domestica.; citation_author=CL Hughes; citation_author=PZ Liu; citation_author=TC Kaufman; citation_journal_title=Evol Dev; citation_volume=6; citation_number=97; citation_pages=393-401; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Maternal expression and early zygotic regulation of the Hox3/zen gene in the grasshopper Schistocerca gregaria.; citation_author=P Dearden; citation_author=M Grbic; citation_author=F Falciani; citation_author=M Akam; citation_journal_title=Evol Dev; citation_volume=2; citation_number=98; citation_pages=261-270; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The anterior determinant bicoid of Drosophila is a derived Hox class 3 gene.; citation_author=M Stauber; citation_author=H Jackle; citation_author=U Schmidt-Ott; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=99; citation_pages=3786-3789; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A single Hox3 gene with composite bicoid and zerknullt expression characteristics in non-Cyclorrhaphan flies.; citation_author=M Stauber; citation_author=A Prell; citation_author=U Schmidt-Ott; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=100; citation_pages=274-279; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Conservation and elaboration of Hox gene regulation during evolution of the vertebrate head.; citation_author=M Manzanares; citation_author=H Wada; citation_author=N Itasaki; citation_author=PA Trainor; citation_author=R Krumlauf; citation_journal_title=Nature; citation_volume=408; citation_number=101; citation_pages=854-857; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Conserved anterior boundaries of Hox gene expression in the central nervous system of the leech Helobdella.; citation_author=MJ Kourakis; citation_author=VA Master; citation_author=DK Lokhorst; citation_author=D Nardelli-Haefliger; citation_author=CJ Wedeen; citation_journal_title=Dev Biol; citation_volume=190; citation_number=102; citation_pages=284-300; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Collinear activation of Hoxb genes during gastrulation is linked to mesoderm cell ingression.; citation_author=T Iimura; citation_author=O Pourquie; citation_journal_title=Nature; citation_volume=442; citation_number=103; citation_pages=568-571; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The vertebrate limb: a model system to study the Hox/HOM gene network during development and evolution.; citation_author=D Duboule; citation_journal_title=Bioessays; citation_volume=14; citation_number=104; citation_pages=375-384; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Cdx1 and Cdx2 have overlapping functions in anteroposterior patterning and posterior axis elongation.; citation_author=E van den Akker; citation_author=S Forlani; citation_author=K Chawengsaksophak; citation_author=W de Graaff; citation_author=F Beck; citation_journal_title=Development; citation_volume=129; citation_number=105; citation_pages=2181-2193; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ParaHox gene expression in the polychaete annelid Capitella sp. I.; citation_author=AC Frobius; citation_author=EC Seaver; citation_journal_title=Dev Genes Evol; citation_volume=216; citation_number=106; citation_pages=81-88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Epithelial-mesenchymal signaling during the regionalization of the chick gut.; citation_author=DJ Roberts; citation_author=DM Smith; citation_author=DJ Goff; citation_author=CJ Tabin; citation_journal_title=Development; citation_volume=125; citation_number=107; citation_pages=2791-2801; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Investigating the origins of triploblasty: 'mesodermal' gene expression in a diploblastic animal, the sea anemone Nematostella vectensis (phylum, Cnidaria; class, Anthozoa).; citation_author=MQ Martindale; citation_author=K Pang; citation_author=JR Finnerty; citation_journal_title=Development; citation_volume=131; citation_number=108; citation_pages=2463-2474; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bayesian hypothesis testing of four-taxon topologies using molecular sequence data.; citation_author=JS Sinsheimer; citation_author=JA Lake; citation_author=RJ Little; citation_journal_title=Biometrics; citation_volume=52; citation_number=109; citation_pages=193-210; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Bayesian phylogenetic inference using DNA sequences: a Markov Chain Monte Carlo Method.; citation_author=Z Yang; citation_author=B Rannala; citation_journal_title=Mol Biol Evol; citation_volume=14; citation_number=110; citation_pages=717-724; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach.; citation_author=J Felsenstein; citation_journal_title=J Mol Evol; citation_volume=17; citation_number=111; citation_pages=368-376; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Expansion of the Hox gene family and the evolution of chordates.; citation_author=JW Pendleton; citation_author=BK Nagai; citation_author=MT Murtha; citation_author=FH Ruddle; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=112; citation_pages=6300-6304; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A Hox class 3 orthologue from the spider Cupiennius salei is expressed in a Hox-gene-like fashion.; citation_author=WG Damen; citation_author=D Tautz; citation_journal_title=Dev Genes Evol; citation_volume=208; citation_number=113; citation_pages=586-590; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=PHYLIP - Phylogeny Inference Package: Felsenstein, J; citation_author=J Felsenstein; citation_number=114; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences.; citation_author=DT Jones; citation_author=WR Taylor; citation_author=JM Thornton; citation_journal_title=Computer Applications in the Biosciences; citation_volume=8; citation_number=115; citation_pages=275-282; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=ProtTest: selection of best-fit models of protein evolution.; citation_author=F Abascal; citation_author=R Zardoya; citation_author=D Posada; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=116; citation_pages=2104-2105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Confidence limits on phylogenies: an approach using the bootstrap.; citation_author=J Felsenstein; citation_journal_title=Evolution; citation_volume=39; citation_number=117; citation_pages=783-791; citation_date=1985; " />
      <meta name="citation_reference" content="citation_author=WP Maddison; citation_author=DR Maddison; citation_number=118; citation_date=1992; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=119; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=DL Swofford; citation_number=120; citation_date=2003; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=The phusion assembler.; citation_author=JC Mullikin; citation_author=Z Ning; citation_journal_title=Genome Res; citation_volume=13; citation_number=121; citation_pages=81-90; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomic inventory and expression of Sox and Fox genes in the cnidarian Nematostella vectensis.; citation_author=CR Magie; citation_author=K Pang; citation_author=MQ Martindale; citation_journal_title=Dev Genes Evol; citation_volume=215; citation_number=122; citation_pages=618-630; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=TA Stephenson; citation_number=123; citation_pages=426 p; citation_date=1935; citation_publisher=The Ray Society; " />
      <meta name="citation_reference" content="citation_author=DR Maddison; citation_author=WP Maddison; citation_number=124; citation_date=2000; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=Caenorhabditis elegans embryonic axial patterning requires two recently discovered posterior-group Hox genes.; citation_author=K Van Auken; citation_author=DC Weaver; citation_author=LG Edgar; citation_author=WB Wood; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=125; citation_pages=4499-4503; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cnidarians reveal intermediate stages in the evolution of Hox clusters and axial complexity.; citation_author=JR Finnerty; citation_journal_title=Am Zool; citation_volume=41; citation_number=126; citation_pages=608-620; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Homeobox gene clusters and the human paralogy map.; citation_author=C Popovici; citation_author=M Leveugle; citation_author=D Birnbaum; citation_author=F Coulier; citation_journal_title=FEBS Lett; citation_volume=491; citation_number=127; citation_pages=237-242; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genesis and evolution of the Evx and Mox genes and the extended Hox and ParaHox gene clusters.; citation_author=C Minguillon; citation_author=J Garcia-Fernandez; citation_journal_title=Genome Biol; citation_volume=4; citation_number=128; citation_pages=R12; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A model of evolutionary change in proteins.; citation_author=MO Dayhoff; citation_author=RM Schwartz; citation_author=BC Orcutt; citation_number=129; citation_pages=pp. 345-352; citation_date=1978; citation_publisher=National Biomedical Research Foundation.; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000153" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, Nematostella vectensis"/>
    <meta name="twitter:description" content="BackgroundHox genes were critical to many morphological innovations of bilaterian animals. However, early Hox evolution remains obscure. Phylogenetic, developmental, and genomic analyses on the cnidarian sea anemone Nematostella vectensis challenge recent claims that the Hox code is a bilaterian invention and that no &ldquo;true&rdquo; Hox genes exist in the phylum Cnidaria.Methodology/Principal FindingsPhylogenetic analyses of 18 Hox-related genes from Nematostella identify putative Hox1, Hox2, and Hox9+ genes. Statistical comparisons among competing hypotheses bolster these findings, including an explicit consideration of the gene losses implied by alternate topologies. In situ hybridization studies of 20 Hox-related genes reveal that multiple Hox genes are expressed in distinct regions along the primary body axis, supporting the existence of a pre-bilaterian Hox code. Additionally, several Hox genes are expressed in nested domains along the secondary body axis, suggesting a role in &ldquo;dorsoventral&rdquo; patterning.Conclusions/SignificanceA cluster of anterior and posterior Hox genes, as well as ParaHox cluster of genes evolved prior to the cnidarian-bilaterian split. There is evidence to suggest that these clusters were formed from a series of tandem gene duplication events and played a role in patterning both the primary and secondary body axes in a bilaterally symmetrical common ancestor. Cnidarians and bilaterians shared a common ancestor some 570 to 700 million years ago, and as such, are derived from a common body plan. Our work reveals several conserved genetic components that are found in both of these diverse lineages. This finding is consistent with the hypothesis that a set of developmental rules established in the common ancestor of cnidarians and bilaterians is still at work today."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000153.g011"/>

  <meta property="og:title" content="Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, Nematostella vectensis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000153" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4679'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6419'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1313&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000153">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000153" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000153&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, <i>Nematostella vectensis</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph F. Ryan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Bioinformatics Program, Boston University, Boston, Massachusetts, United States of America, 
                  Genome Technology Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maureen E. Mazza, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Boston University, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kevin Pang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Kewalo Marine Laboratory, Pacific Bioscience Research Center, University of Hawaii, Honolulu, Hawaii, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Q. Matus, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Kewalo Marine Laboratory, Pacific Bioscience Research Center, University of Hawaii, Honolulu, Hawaii, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andreas D. Baxevanis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Technology Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Q. Martindale, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Kewalo Marine Laboratory, Pacific Bioscience Research Center, University of Hawaii, Honolulu, Hawaii, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John R. Finnerty
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jrf3@bu.edu">jrf3@bu.edu</a>
            </p>

                <p>Affiliations:
                  Bioinformatics Program, Boston University, Boston, Massachusetts, United States of America, 
                  Department of Biology, Boston University, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 24, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000153</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000153" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000153" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000153" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000153" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000153-t001" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g001" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g002" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g003" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g004" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g005" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g006" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g007" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g008" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g009" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g010" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000153-g011" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000153">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000153" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Hox genes were critical to many morphological innovations of bilaterian animals. However, early Hox evolution remains obscure. Phylogenetic, developmental, and genomic analyses on the cnidarian sea anemone <em>Nematostella vectensis</em> challenge recent claims that the Hox code is a bilaterian invention and that no “true” Hox genes exist in the phylum Cnidaria.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Phylogenetic analyses of 18 Hox-related genes from <em>Nematostella</em> identify putative Hox1, Hox2, and Hox9+ genes. Statistical comparisons among competing hypotheses bolster these findings, including an explicit consideration of the gene losses implied by alternate topologies. <em>In situ</em> hybridization studies of 20 Hox-related genes reveal that multiple Hox genes are expressed in distinct regions along the primary body axis, supporting the existence of a pre-bilaterian Hox code. Additionally, several Hox genes are expressed in nested domains along the secondary body axis, suggesting a role in “dorsoventral” patterning.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>A cluster of anterior and posterior Hox genes, as well as ParaHox cluster of genes evolved prior to the cnidarian-bilaterian split. There is evidence to suggest that these clusters were formed from a series of tandem gene duplication events and played a role in patterning both the primary and secondary body axes in a bilaterally symmetrical common ancestor. Cnidarians and bilaterians shared a common ancestor some 570 to 700 million years ago, and as such, are derived from a common body plan. Our work reveals several conserved genetic components that are found in both of these diverse lineages. This finding is consistent with the hypothesis that a set of developmental rules established in the common ancestor of cnidarians and bilaterians is still at work today.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ryan JF, Mazza ME, Pang K, Matus DQ, Baxevanis AD, et al.  (2007) Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, <em>Nematostella vectensis</em>
            . PLoS ONE 2(1):
          e153.
            doi:10.1371/journal.pone.0000153</p><p><strong>Academic Editor: </strong>Justin Fay, Washington University in St. Louis School of Medicine, United States of America</p><p><strong>Received:</strong> September 12, 2006; <strong>Accepted:</strong> November 30, 2006; <strong>Published:</strong> January 24, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This research was supported by the National Science Foundation and the National Atmospheric and Space Administration (NSF grant number DEB51 9727244 to M.Q.M. and J.R.F.; NSF grant number IBN-0212773 to J.R.F.; NASA grant number NAG2-1374 and NAG2-1519s to M.Q.M.) and by the Intramural Research Program of the National Human Genome Research Institute, National Institutes of Health. M.E.M. received financial support from the Boston University Women's Guild.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Thousands of papers have been written about Hox genes over the last twenty years, and cross-species comparisons using this particular family of homeodomain transcription factors have provided the critical momentum sparking the recent resurgence of the field of evolutionary developmental biology. Three remarkable phenomena that have fueled enormous interest in Hox genes are the Hox code, Hox clusters, and Hox colinearity.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In a phylogenetically diverse range of animals, a conserved “Hox code” is partially responsible for patterning the primary body axis. The term Hox code was first applied to the segmentally-restricted expression of Hox genes in the branchial system of the developing mouse <a href="#pone.0000153-Hunt1">[1]</a>. However, extensive similarity among Hox expression patterns in a wide range of taxa soon led to the recognition that a Hox code might be a fundamental developmental mechanism of animals <a href="#pone.0000153-Slack1">[2]</a>. In all bilaterian animals that have been studied, multiple Hox genes are found in their genomes and, over the course of development, different regions along the primary body axis come to express different Hox genes or different combinations of Hox genes. Appropriate Hox expression is required to confer the appropriate regional identity upon these Hox-expressing body regions — <em>ergo</em>, a Hox code. Furthermore, comparable body regions are patterned by orthologous Hox genes in distantly related taxa, so a similar Hox code appears to be widely conserved. However, this does not mean that the “Hox code” is static over evolutionary time. Hox expression patterns can vary substantially with respect to how much overlap exists between expression domains, what fraction of the primary body axis is accounted for by Hox expression, the precise axial order of different Hox orthologs, the degree of dorsal-ventral asymmetry in Hox expression, and the germ layer in which Hox genes are expressed. With regards to this last point, while Hox genes are generally regarded as exhibiting ectodermal and mesodermal expression, they are also expressed in endoderm <a href="#pone.0000153-Burke1">[3]</a>–<a href="#pone.0000153-Sakiyama1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In addition, across a range of bilaterian animals, Hox genes are located in conserved genomic clusters. The relative genomic organization of orthologous Hox genes is well conserved among select Ecdysozoa such as <em>Anopheles</em>, <em>Schistocerca</em>, and <em>Tribolium</em> 
            <a href="#pone.0000153-Powers1">[11]</a>–<a href="#pone.0000153-Brown1">[14]</a>; Lophotrochozoa, such as <em>Lineus</em> 
            <a href="#pone.0000153-KmitaCunisse1">[15]</a>; and Deuterstomia, such as vertebrates and <em>Branchiostoma</em> 
            <a href="#pone.0000153-GarciaFernandez1">[16]</a>, <a href="#pone.0000153-Minguillon1">[17]</a>. The origin of Hox clusters is not especially remarkable, since the Hox clusters would have arisen as a natural outgrowth of the process of tandem gene duplication. However, the persistence of Hox clusters over hundreds of millions of years in diverse metazoan lineages suggests that strong stabilizing selection must be operating.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>One explanation for the conservation of genomic organization is that the proper regulation of these genes may depend upon their close physical linkage (reviewed in <a href="#pone.0000153-Mann1">[18]</a>. However, in diverse bilaterian taxa (for example, <em>Ciona intestinalis</em>, <em>Caenorhabditis elegans</em>, <em>Drosophila melanogaster</em>, <em>D. pseudoobscura</em>, <em>D. repleta</em>, <em>D. virilis</em>, <em>Oikopleura dioica</em>, <em>Schistosoma mansoni</em>, and <em>Strongylocentrotus purpuratus</em>), the Hox cluster has experienced breaks, undergone extensive rearrangements, or even degenerated to the point where a cluster cannot be recognized or identified <a href="#pone.0000153-Ferrier2">[19]</a>–<a href="#pone.0000153-Randazzo1">[27]</a>. The degeneration of the Hox cluster does not necessarily imply that the Hox code has been abandoned, as Hox genes may continue to specify the same axial territories even after a Hox cluster has undergone extensive rearrangements. For example, it appears that all insects employ the same Hox code, but some insects have intact Hox clusters (for example, grasshopper), while others have partially degraded Hox clusters (for example, fruit flies).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>It has also been observed in a phylogenetically widespread range of taxa that the relative spatial and/or temporal expression of Hox genes is correlated with their relative position within Hox clusters. This correspondence between gene expression and cluster organization has been termed <em>colinearity</em> 
            <a href="#pone.0000153-Boncinelli1">[28]</a>. The existence of colinearity implies that linkage impacts gene regulation <a href="#pone.0000153-Mann1">[18]</a>. However, Hox colinearity is not universal <a href="#pone.0000153-Monteiro1">[29]</a>, and no single mechanism has been identified that can explain Hox colinearity or the persistence of Hox clusters in diverse metazoan lineages <a href="#pone.0000153-Kmita1">[30]</a>. Rather, it seems that several different regulatory mechanisms may contribute to the stability of Hox clusters. For example, both higher-order chromatin structure and local <em>cis</em>-regulatory elements may result in coordinated regulation of neighboring Hox loci <a href="#pone.0000153-Chambeyron1">[31]</a>, <a href="#pone.0000153-Spitz1">[32]</a>. Furthermore, in some taxa, most notably <em>Drosophila</em>, Hox linkage does not appear to be required for appropriate Hox expression <a href="#pone.0000153-Tiong1">[33]</a>. The general correspondence between the genomic organization and spatial expression of Hox genes in <em>Drosophila</em> may be attributable to phylogenetic inertia.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Over evolutionary time, the functional diversification of Hox genes has clearly contributed to the diversification of animal body plans <a href="#pone.0000153-Averof2">[5]</a>, <a href="#pone.0000153-Galant1">[34]</a>–<a href="#pone.0000153-Valentine1">[40]</a>. For this reason, understanding the origin and early evolution of Hox genes could prove critical to understanding the metazoan radiation. A Hox cluster consisting of seven genes evolved prior to the divergence of protostomes and deuterostomes <a href="#pone.0000153-GarciaFernandez2">[41]</a> and, as both insects and vertebrates utilize Hox genes to pattern a portion of their primary body axes, the Hox code can be said to predate the diversification of crown bilaterians. The phylum Cnidaria can provide unique insights into early Hox evolution since cnidarians constitute an outgroup to the Bilateria <a href="#pone.0000153-Medina1">[42]</a>, <a href="#pone.0000153-Collins1">[43]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>It is currently a matter of debate whether cnidarians possess <em>bona fide</em> Hox genes, and if so, whether the Hox code originated prior to the cnidarian-bilaterian divergence. In the last few years, several studies have suggested that cnidarians possess both anterior and posterior Hox genes, but they lack group 3 and central Hox genes <a href="#pone.0000153-Finnerty1">[36]</a>, <a href="#pone.0000153-Finnerty2">[44]</a>–<a href="#pone.0000153-Martinez1">[48]</a> (see <a href="#pone-0000153-t001">Table 1</a> for varying gene nomenclature). More recently, Kamm and co-workers have advocated two seemingly contradictory hypotheses: (1) that cnidarians possess anterior Hox genes, but instead of <em>bona fide</em> posterior Hox genes, they possess a posterior Hox/Cdx like gene <a href="#pone.0000153-Kamm1">[49]</a>, and (2) that cnidarian genes related to bilaterian Hox genes “should be regarded as Hox-like but not as true Hox genes” <a href="#pone.0000153-Kamm2">[50]</a>. A more recent study by Chourrout and co-workers suggests that the cnidarian-bilaterian ancestor possessed two to three ParaHox genes as well as an Anterior and group 3-like Hox gene, each of which subsequently underwent independent radiations within the bilaterian and cnidarian lineages <a href="#pone.0000153-Chourrout1">[51]</a>.</p>
<div class="figure" id="pone-0000153-t001"><div class="img"><a name="pone-0000153-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Nomenclature of Nematostella Hox and Hox-related genes.</span></strong></p><span>doi:10.1371/journal.pone.0000153.t001</span></div><a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>With respect to the Hox code, a 2004 study on the expression of five candidate Hox genes in the anthozoan sea anemone <em>Nematostella vectensis</em> found support for the existence of a Hox code in the cnidarian-bilaterian ancestor; multiple Hox genes appeared to be present, and they were found to be expressed in distinct territories along the primary body axis <a href="#pone.0000153-Finnerty1">[36]</a>. In contrast, the aforementioned 2006 study by Kamm et al. <a href="#pone.0000153-Kamm1">[49]</a> concluded that the Hox code was a bilaterian invention based on what they regarded as the absence of central, group 3, and posterior Hox genes in Cnidaria, and on differences between the Hox expression patterns between the anthozoan <em>Nematostella</em> and the colonial hydrozoan <em>Eleutheria</em>.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>In the current study, we employ novel analytical methods and present an extensive battery of new evidence from the sea anemone <em>Nematostella vectensis</em> to address the origin and early evolution of Hox genes and the Hox code. This evidence includes phylogenetic analysis of eighteen distinct Hox-related loci from <em>Nematostella</em> 
            <a href="#pone.0000153-Ryan1">[52]</a>, linkage analysis of these eighteen loci based on an assembly of the <em>Nematostella</em> genome <a href="#pone.0000153-Sullivan1">[53]</a> and extensive corroborating gene mapping studies; and developmental gene expression assays for 20 <em>Nematostella</em> Hox-related genes, 12 of which have never been described before. For those genes whose expression has been previously described, we reveal previously unknown aspects of the spatiotemporal expression that are critical to interpreting Hox evolution.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>Contrary to some recent reports <a href="#pone.0000153-Kamm1">[49]</a>, <a href="#pone.0000153-Kamm2">[50]</a>, multiple lines of phylogenetic evidence support the hypothesis that both anterior and posterior Hox genes and two ParaHox genes were present in the cnidarian-bilaterian ancestor. Seven <em>Nematostella</em> genes appear to be descended from the founding members of the Hox1, Hox2, and Hox9+ families. We provide more detailed transcriptional annotation of a genomic cluster comprising two ParaHox genes as well as one Hox1 family member, three Hox2 family members, an even-skipped ortholog, an HlxB9 ortholog, and a Rough ortholog <a href="#pone.0000153-Chourrout1">[51]</a>.</p>
<a id="article1.body1.sec1.p11" name="article1.body1.sec1.p11"></a><p>During larval development, the putative Hox1, Hox2, and Hox9+ homologs are expressed in a number of distinct spatial domains that collectively account for practically the entire primary body axis, from the aboral to the oral extremity. Five of these candidate Hox genes (anthox7, anthox8, anthox8a, anthox6a, anthox1a) and one candidate ParaHox gene (NVHD065) are also differentially expressed along the secondary body axis, known as the directive axis. These genes are expressed in nested subsets along the directive axis, suggesting that <em>Nematostella</em> may be employing Hox genes to pattern both its primary and secondary axes. Phylogenetic mapping of gene expression patterns on a molecular phylogeny suggests that differential expression along the primary body axis is a primitive feature of the <em>Nematostella</em> Hox-related genes, while differential expression along the secondary body axis evolved afterwards.</p>
<a id="article1.body1.sec1.p12" name="article1.body1.sec1.p12"></a><p>Collectively, these data suggest that at least a rudimentary Hox code was operative in the cnidarian-bilaterian ancestor and that it played a role in patterning the animal's primary body axis (and possibly the secondary body axis as well). Moreover, strong stabilizing selection has been operating on this Hox code that has maintained certain core characteristics despite being deployed in a bewildering array of animal forms for over half a billion years.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Phylogenetic Analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Overall, there is substantial agreement among three different phylogenetic methods (neighbor-joining, maximum-likelihood, and Bayesian analysis) regarding the phylogenetic relationships of 18 Hox-related genes from the sea anemone <em>Nematostella</em> (phylum Cnidaria) and 43 Hox-related genes from representative Bilateria (<a href="#pone-0000153-g001">Figure 1</a>). On all three trees (<a href="#pone-0000153-g002">Figure 2</a>, <a href="#pone.0000153.s001">S1</a>, and <a href="#pone.0000153.s002">S2</a>), presumed protostome and deuterostome orthologs are grouped together with robust statistical support. In addition, all three trees indicate that the HlxB9, Gbx, Evx, and Rough families emerge basal to a clade that contains the Hox and ParaHox genes <a href="#pone.0000153-Kamm2">[50]</a>, <a href="#pone.0000153-Ryan1">[52]</a>. On the neighbor-joining (<a href="#pone-0000153-g002">Figure 2</a>) and Bayesian trees (<a href="#pone.0000153.s001">Figure S1</a>), the Mox family also emerges basal to the Hox-ParaHox clade, but on the maximum-likelihood tree, Mox is nested within the Hox-ParaHox clade.</p>
<div class="figure" id="pone-0000153-g001"><div class="img"><a name="pone-0000153-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Alignment of homeodomains included in the phylogenetic analyses. All sequences are aligned to the <em>Drosophila</em> Antennapedia homeodomain. Each <em>Nematostella</em> sequence is grouped with putative bilaterian homologs. The degree of statistical support (bootstrap proporation [BP] or posterior probability [PP]) for each of these homology assignments is indicated separately for the neighbor-joining (NJ), Bayesian (Bayes), and maximum-likelihood (ML) trees. The average statistical support for each grouping is indicated in the far right column (NJ-BP+Bayes-PP+ML-BP/3). The dataset is available in Phylip format as <a href="#pone.0000153.s012">Figure S12</a>.</p>
<span>doi:10.1371/journal.pone.0000153.g001</span></div><div class="figure" id="pone-0000153-g002"><div class="img"><a name="pone-0000153-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Homeodomain phylogeny based on neighbor-joining. The cladogram is rooted using the dll sequences. <em>Nematostella</em> sequences are shown in red. Bilaterian sequences are shown in black. Bootstrap proportions are presented at each node. Open circles depict implied gene losses for <em>Nematostella</em> (red) and <em>Branchiostoma</em> (black). Closed circles depict implied lineage-specific gene duplications for <em>Nematostella</em> (red) and <em>Branchiostoma</em> (black). The dataset used in this analysis is available as <a href="#pone.0000153.s012">Figure S12</a>.</p>
<span>doi:10.1371/journal.pone.0000153.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>On all three trees, and in agreement with two recent studies, nine of <em>Nematostella's</em> sequences can be confidently assigned to the following six homeodomain families: HlxB9 (1), Gbx (1), Evx (1), Rough (1), Mox (4), Gsx (1) <a href="#pone.0000153-Kamm2">[50]</a>, <a href="#pone.0000153-Ryan1">[52]</a>. The mean statistical support for these groupings, averaged over all three phylogenetic analyses, ranges from 0.65 to 1.00 (<a href="#pone-0000153-g001">Figure 1</a>).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Other groupings are also recovered on all three trees, but with somewhat lower statistical support. The anthox1 and anthox1a homeodomains cluster together on all three trees, suggesting that these two sequences were produced by a gene duplication that is specific to the Cnidaria. The anthox1/1a lineage appears most closely related to the posterior Hox genes (Hox9–14) on all three trees, with mean statistical support of 0.39. Anthox6 appears most closely related to the Hox1 family on all three trees, in agreement with previous studies <a href="#pone.0000153-Finnerty3">[45]</a>, <a href="#pone.0000153-Finnerty4">[46]</a>, <a href="#pone.0000153-Kamm1">[49]</a>, <a href="#pone.0000153-Ryan1">[52]</a>. The mean statistical support for this grouping is 0.23 (<a href="#pone-0000153-g001">Figure 1</a>).</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The precise placement of six <em>Nematostella</em> homeodomains varies between trees. Homeodomain NVHD065 groups with the Cdx family on the neighbor-joining and Bayesian trees. However, on the maximum-likelihood tree, NVHD065 groups with the Xlox family. The mean statistical support for the grouping of NVHD065 with Cdx (0.42) is higher than the mean statistical support for its grouping with Xlox (0.20). However, NVHD065 shares more identical amino acids with Xlox homeodomains (39/60 versus amphiXlox) than with Cdx homeodomains (30/60 versus amphiCdx). It also shares one distinctive residue with Xlox homeodomains (a histidine at position 44) and no distinctive residues with Cdx homeodomains. Chourrout and co-workers <a href="#pone.0000153-Chourrout1">[51]</a> have suggested that the NVHD065 gene (which they refer to as XLOX/CDX ) may be related to both XLOX and CDX.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Anthox6a <a href="#pone.0000153-Kamm1">[49]</a> (known as NVHD060 in <a href="#pone.0000153-Ryan1">[52]</a>) groups with anthox6 and appears most closely related to the Hox1 family on both the neighbor-joining and Bayesian trees (<a href="#pone-0000153-g002">Figure 2</a>; <a href="#pone.0000153.s001">Figure S1</a>). However, on the maximum-likelihood tree, anthox6a appears more closely related to the Gsx family (<a href="#pone.0000153.s002">Figure S2</a>). However, bootstrap support for the grouping of anthox6a with Gsx is very low; the bootstrap proportion (BP) is only 0.04. The mean statistical support (MS) for the node uniting anthox6, anthox6a, and Hox1 is substantially higher (MS = 0.16), suggesting that anthox6a could be a quickly evolving paralog of anthox6.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>Anthox9 <a href="#pone.0000153-Kamm1">[49]</a> (known as NVHD117 in <a href="#pone.0000153-Ryan1">[52]</a>) appears most closely related to Gsx on the neighbor-joining and Bayesian trees (MS = 0.24). However, on the maximum-likelihood tree, this sequence emerges as the sister group to the Mox clade, albeit with very low bootstrap support (BP = 0.02). Kamm et. al. <a href="#pone.0000153-Kamm1">[49]</a> point out that this predicted protein is peculiar in possessing an isoleucine residue at position 16 of the homeodomain, and they suggested that it may be a pseudogene. In support of this hypothesis, anthox9 is the only one of 20 Hox-related genes in the <em>Nematostella</em> genome for which we have failed to detect expression by <em>in situ</em> hybridization. On all three trees, anthox7, anthox8a, and anthox8b form a well-supported clade exclusive of any bilaterian sequences, which suggests that these three genes arose via two gene duplications within the Cnidaria. The precise placement of this anthox7/8a/8b lineage varies among all three trees. On the neighbor-joining tree (<a href="#pone-0000153-g002">Figure 2</a>), anthox7, 8a, and 8b group with the Hox2 family, a finding that is consistent with previous studies <a href="#pone.0000153-Finnerty3">[45]</a>, <a href="#pone.0000153-Finnerty4">[46]</a>. However, on the Bayesian tree, anthox7/8a/8b appears more closely related to the Hox1 family. On the maximum-likelihood tree, anthox7/8a/8b emerges as an independent lineage, immediately after the emergence of a bilaterian Hox2 lineage.</p>


<h4>Structure and genomic arrangement of Hox and Hox-related genes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Based on two publicly available genome assemblies <a href="#pone.0000153-Sullivan1">[53]</a>–<a href="#pone.0000153-2">[55]</a> and an abundance of corroborating spot-sequencing, the eighteen Hox and Hox-related genes of <em>Nematostella</em> are distributed among seven different genomic clusters (<a href="#pone-0000153-g003">Figure 3</a>). This finding is consistent with a recent report by Chourrout and co-workers <a href="#pone.0000153-Chourrout1">[51]</a>. Overall, the agreement between our spot sequencing and the genome traces is extremely high. For example, an anthox2 cDNA is a precise match for JGI scaffold 27 over its entire length of 1,019 nucleotides <a href="#pone.0000153-Finnerty4">[46]</a>, and a 3,794 nucleotide stretch of the anthox1a locus that we isolated by ligation-mediated PCR is a 99.9% match to JGI scaffold 3, differing by four single nucleotide insertions.</p>
<div class="figure" id="pone-0000153-g003"><div class="img"><a name="pone-0000153-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Clusters of Hox-related homeobox genes in the <em>Nematostella</em> genome. Based on the current genomic assemblies, thirty ANTP class genes of <em>Nematostella</em> are distributed among seven homeobox clusters <a href="#pone.0000153-Ryan1">[52]</a>. The location of the PRD class gene Dmbx is also shown. The arrangement of the <em>Nematostella</em> genes is shown in relation to the hypothesized "extended Hox cluster," that is presumed to have existed in the most recent common ancestor of protostomes and deuterostomes (gray box; <a href="#pone.0000153-Popovici1">[127]</a>, <a href="#pone.0000153-Minguillon2">[128]</a>). Horizontal lines connecting <em>Nematostella</em> genes indicate known genomic linkage. Double-arrows connect <em>Nematostella</em> homeodomains to their putative bilaterian homologs based on phylogenetic analyses of homeodomain sequences (<a href="#pone-0000153-g001">Figure 1</a>; <a href="#pone.0000153.s001">supp figs. 1</a>–<a href="#pone.0000153.s002">2</a>). Detailed diagrams of each of the eight <em>Nematostella</em> homeodomain clusters are presented in <a href="#pone.0000153.s003">supplemental figures 3</a>
                     <a href="#pone.0000153.s004">pone.0000153.s004</a>
                     <a href="#pone.0000153.s005">pone.0000153.s005</a>
                     <a href="#pone.0000153.s006">pone.0000153.s006</a>
                     <a href="#pone.0000153.s007">pone.0000153.s007</a>
                     <a href="#pone.0000153.s008">pone.0000153.s008</a>
                     <a href="#pone.0000153.s009">pone.0000153.s009</a>–<a href="#pone.0000153.s010">10</a>.</p>
<span>doi:10.1371/journal.pone.0000153.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The largest cluster comprises seven Hox-related genes, as follows: Rough—HlxB9—anthox6—Evx—anthox8b—anthox8​a—anthox7(<a href="#pone-0000153-g003">Figure 3</a>; <a href="#pone.0000153.s003">Figure S3</a>). The intergenic distances among these genes ranges from 2,868 nucleotides between anthox8a and anthox8b to 21,487 nucleotides between anthox8b and eve (<a href="#pone.0000153.s003">Figure S3</a>). With the exception of HlxB9, all of these genes exhibit the same transcriptional orientation. Except for Evx, which consists of three exons, these genes all contain two exons. In no case is the homeodomain interrupted by an intron. The PRD class homeobox gene Dmbx <a href="#pone.0000153-Ryan1">[52]</a> is located more than 98,000 nucleotides upstream of anthox7, and the NK-related homeodomain HLXd is located more than 415,000 nucleotides upstream of Dmbx. There are no other homeodomains within 42,385 nucleotides downstream of Rough. Based on BLASTX searches against the NCBI Reference Sequence (RefSeq) database <a href="#pone.0000153-Pruitt1">[56]</a>, a single non-homeobox gene resembling a hypothetical mouse protein (accession number XP_001000509.1; E-value = 9×10<sup>−6</sup>) is interposed between anthox8b and eve, while a sequence similar to a predicted <em>Staphylococcus</em> gene protein (accession number XP_251955; E-value = 2×10<sup>−20</sup>) is interposed between eve and anthox6 (<a href="#pone.0000153.s003">Figure S3</a>). Twenty non-homeobox genes are predicted to lie within the nearly 300 kilobases that separates anthox6 from HlxB9.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>All four Mox genes reside in a compact genomic cluster, each having the same transcriptional orientation (<a href="#pone.0000153.s004">Figure S4</a>). From the N-terminal exon of MoxB to the C-terminal exon of MoxA, the cluster spans 23,210 nucleotides. No other homeobox genes reside within 48 kilobases upstream or 40 kilobases downstream of the Mox cluster. Roughly nine kilobases upstream of the Mox cluster, BLASTX identifies a protein with significant similarity to the PF20 protein of <em>Chlamydomonas</em> (accession number T08180; E-value = 8×10<sup>−24</sup>). Roughly 5 kilobases downstream of the Mox cluster, BLASTX identifies a sequence with significant similarity to a retrotransposon (accession number BAD86655.1; E-value 2×10<sup>−27</sup>). The BLASTX search returned no significant hits within the intergenic regions (cutoff E-value&lt;0.0001).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The two putative paraHox genes (anthox2 and NVHD065) are closely linked on a single genomic scaffold in the Phusion assembly (<a href="#pone-0000153-g003">Figure 3</a>; <a href="#pone.0000153.s005">Figure S5</a>). Anthox2, a clear Gsx ortholog <a href="#pone.0000153-Finnerty4">[46]</a>, is separated by 12,977 nucleotides from NVHD065, a sequence that appears most closely related to Cdx on the neighbor-joining and Bayesian trees (<a href="#pone-0000153-g002">Figure 2</a>; <a href="#pone.0000153.s001">Figure S1</a>) but most closely related to Xlox on the maximum-likelihood tree (<a href="#pone.0000153.s002">Figure S2</a>). The two genes exhibit the same transcriptional orientation. Both genes possess two exons, with the homeodomain encoded entirely by the second exon. Anthox2 has a short intron (176 nucleotides), while NVHD065 has a long intron (10,529 nucleotides). A 1,580,677 nucleotide scaffold encompassing this two-gene cluster was recovered from the JGI assembly of the <em>Nematostella</em> genome. A predicted gene with apparent homology to the largest subunit of RNA Polymerase II (accession number XP_001056421.1) is located approximately 5,500 nucleotides upstream of Gsx, while a gene with apparent homology to a predicted DNA-polymerase (accession number XP_785333.1) is located approximately 7,000 nucleotides downstream of NVHD065.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Anthox9 <a href="#pone.0000153-Chourrout1">[51]</a>, is very tightly linked to the posterior Hox gene anthox1a (<a href="#pone.0000153.s006">Figure S6</a>). The two genes exhibit the opposite orientation. There are no other genes predicted to occur in the 3,865-nucleotide intergenic region. Anthox1, anthox6a, and Gbx do not appear closely linked to other Hox-related genes, although all three genes are distantly linked to homeobox genes from the HLX family. Anthox1 is over 300,000 nucleotides distant from HLXa (<a href="#pone.0000153.s007">Figure S7</a>). Anthox6a is over 263,000 nucleotides distant from HLXb (<a href="#pone.0000153.s008">Figure S8</a>). Gbx is over 600,000 nucleotides distant from HLXc (<a href="#pone.0000153.s009">Figure S9</a>). Interestingly, different HLX genes are also distantly linked to the anterior Hox cluster (HLXd; <a href="#pone.0000153.s003">Figure S3</a>) and the anthox1a/anthox9 cluster (HLXc-like; <a href="#pone.0000153.s006">Figure S6</a>). Finally, the Dlx gene is only 6,988 nucleotides distant from NVHD021 (<a href="#pone.0000153.s010">Figure S10</a>). NVHD021 (accession number DQ206308) is an ortholog to the <em>Drosophila</em> ANTP class gene, CG13424, and is more distantly related to the Hox genes than most other genes included in this study <a href="#pone.0000153-Kamm2">[50]</a>, <a href="#pone.0000153-Ryan1">[52]</a>.</p>


<h4>Gene Expression of basal Hox-related genes in Nematostella</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The developmental expression of 20 ANTP class homeobox genes is reported here (<a href="#pone-0000153-g004">Figure 4</a> and <a href="#pone-0000153-g005">5</a>), including seven putative Hox genes (anthox1, anthox1a, anthox6, anthox6a, anthox7, anthox8a, and anthox8b), two putative ParaHox genes (anthox2 and NVHD065), eight basal Hox-related genes (HlxB9, Gbx, Rough, Evx, and four Mox genes) and three more distantly related ANTP homeobox genes (Dlx, HLXb, and a NVHD021). Expression of another Hox-related gene (anthox9) could not be detected. The expression of 13 of these genes has not previously been described. For seven of these genes, we present new views or new developmental stages and describe novel aspects of their developmental expression.</p>
<div class="figure" id="pone-0000153-g004"><div class="img"><a name="pone-0000153-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Developmental Expression of Hox-ParaHox related genes in <em>Nematostella</em>. Gene expression was assayed throughout embryonic and larval development using <em>in situ</em> hybridization. All images are optical sections that permit visualization of the endodermal tissue layer. Panels M and V are transverse sections, but all other images are longitudinal sections with the future oral end of the animal facing left. The blastopore (site of the future mouth) is indicated by an asterisk. Abbreviations are as follows: apical tuft (at); coelenterone (coe); bodywall ectoderm (ecbw); pharyngeal ectoderm (ecph); bodywall endoderm (enbw); pharyngeal endoderm (enph); mesentery (mes); pharynx (pha); tentacle (tn).</p>
<span>doi:10.1371/journal.pone.0000153.g004</span></div><div class="figure" id="pone-0000153-g005"><div class="img"><a name="pone-0000153-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Developmental Expression of Hox and ParaHox homologs in <em>Nematostella</em>. Gene expression was assayed throughout embryonic and larval development using <em>in situ</em> hybridization. All images are optical sections that permit visualization of the endodermal tissue layer. Panels A, J, M, P, S, and Y are transverse sections, but all other images are longitudinal sections with the future oral end of the animal facing left. The blastopore, which becomes the mouth, is indicated by an asterisk. Abbreviations are as follows: apical tuft (at); coelenterone (coe); bodywall ectoderm (ecbw); pharyngeal ectoderm (ecph); bodywall endoderm (enbw); pharyngeal endoderm (enph); mesentery (mes); pharynx (pha); tentacle (tn).</p>
<span>doi:10.1371/journal.pone.0000153.g005</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>HlxB9 transcripts are not detected until after gastrulation (<a href="#pone-0000153-g004">Figure 4A–C</a>). At planula stages, the majority of expression is in cells of the pharyngeal ectoderm. In addition, a small number of cells scattered throughout the body wall ectoderm are also expressing HlxB9. By polyp stages, HlxB9 expressing cells are concentrated in a ring around the pharynx (<a href="#pone-0000153-g004">Figure 4C</a>) and in the ectoderm between the tentacles (<a href="#pone-0000153-g004">Figure 4C</a>, inset).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Rough is first detected by <em>in situ</em> hybridization during mid-gastrulation in cells that span the ectodermal epithelium (<a href="#pone-0000153-g004">Figure 4D</a>). In the planula larva and the juvenile polyp, transcripts can be seen in both ectodermal and body wall endoderm in the center of the body column and in the aboral region (<a href="#pone-0000153-g004">Figure 4E and 4F</a>). Rough-expressing cells are not seen at the oral pole or in the developing tentacles. The basal location of the mature cell bodies suggests that these cells may be a subset of neurons.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Evx expression first becomes apparent in the planula larva, in scattered columnar cells located along the basal surface of the ectodermal epithelium, where the outer ectodermal layer meets the mesoglea (<a href="#pone-0000153-g004">Figure 4H</a>). No Evx expression is observed at the oral or aboral extremities, <em>i.e.</em>, around the mouth or in the foot. The identity of the Evx-expressing cells is not known with certainty, but their morphology and location are consistent with them being a subset of sensory neurons (Meg Daly, personal communication). In the juvenile polyp, Evx-expressing cells appear exclusively in the endoderm at the base of the tentacles (<a href="#pone-0000153-g004">Figure 4I, inset</a>).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>The expression patterns of the four Mox genes are indistinguishable. MoxB expression is depicted in <a href="#pone-0000153-g004">Figures 4J–4L</a>. As with Evx, expression is not evident prior to gastrulation (data not shown). Following gastrulation, Mox is expressed in a ring of endoderm in the pharyngeal region of the planula. Expression persists in this ring of pharyngeal endoderm after the tentacles have emerged and the planula has begun to assume the form of the adult polyp (<a href="#pone-0000153-g004">Figure 4L</a>). Mox is not expressed in the body wall endoderm or the ectoderm. Expression of the other three Mox genes is illustrated in <a href="#pone.0000153.s011">Figure S11</a>.</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>Gbx expression has been described previously <a href="#pone.0000153-Matus1">[57]</a>. Expression begins during early planula stages on the left and right sides of the directive axis in body wall endoderm (<a href="#pone-0000153-g004">Figure 4M and 4N</a>). During later stages, expression is down regulated in body wall endoderm and initiates, and persists in pharyngeal endoderm on the left and right sides (<a href="#pone-0000153-g004">Figure 4O</a>).</p>


<h4>Gene Expression of ANTP genes more distantly related to Hox in Nematostella</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Dlx expression first becomes apparent during the later stages of gastrulation, as a circumferential ring of ectodermal cells in the center of the body column (<a href="#pone-0000153-g004">Figure 4P</a>). Expression persists throughout planula stages but continues to be excluded from the oral and aboral poles (<a href="#pone-0000153-g004">Figure 4Q</a>). In the juvenile polyp, a layer of Dlx expressing cells resides in a basal position within the ectodermal epithelium, at the level of the pharynx (<a href="#pone-0000153-g004">Figure 4R</a>). Some or all of the Dlx expressing cells may be neural precursors based on their initial and final basal nuclear position in the ectodermal epithelium.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>NVHD021 expression first becomes apparent in the oral ectodermal region of the planula larva (<a href="#pone-0000153-g004">Figure 4T</a>). Its expression persists in a more concentrated ring around the mouth of the juvenile polyp (<a href="#pone-0000153-g004">Figure 4U</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>HLXb transcripts accumulate during late planula stages in body wall ectoderm overlying the pharyngeal region (capitulum), and in eight domains of pharyngeal endoderm that correspond to the eight prospective mesenteries (<a href="#pone-0000153-g004">Figure 4V and 4W</a>). The mesenteries will eventually grow out to meet mesenterial precursors in the body wall endoderm. Expression in both of these regions persists throughout polyp stages and is upregulated in oral tissues in the juvenile polyp (<a href="#pone-0000153-g004">Figure 4X</a>). HLXb (accession number DQ206303) is an ortholog to the Human ANTP class gene, HLX1, and is more distantly related to the Hox genes than most other genes included in this study <a href="#pone.0000153-Ryan1">[52]</a>.</p>


<h4>Expression of Hox and ParaHox genes in Nematostella</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The terms dorsal and ventral have been applied to the directive axis of cnidarians, specifically as an aid to naming mesenteries <a href="#pone.0000153-Stephenson1">[58]</a>. However, the use of these terms does not imply definitive homology between the directive axis of cnidarians and the dorsal-ventral axis of bilaterians. It has recently been shown that many genes responsible for dorsoventral patterning in Bilateria are also expressed asymmetrically about the directive axis in Cnidaria <a href="#pone.0000153-Finnerty1">[36]</a>, <a href="#pone.0000153-Matus1">[57]</a>, <a href="#pone.0000153-Matus2">[59]</a>–<a href="#pone.0000153-Rentzsch1">[61]</a>. However, because the expression patterns of dorsoventral patterning genes are not entirely consistent between Cnidaria and Bilateria, the molecular data do not make clear whether one side of the directive axis corresponds to the dorsal of Bilateria, and the other side to ventral. Here, we will adopt the use of these terms to describe gene expression patterns and designate specific mesenteries without implying homology with the “dorsal” and “ventral” of Bilateria. We will refer to the side opposite the siphonoglyph <a href="#pone.0000153-Stephenson1">[58]</a>, a ciliated groove in the pharynx of <em>Nematostella</em>, as ventral.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Anthox6a is expressed in the bodywall endoderm and the presumptive mesenteries of the early planula (<a href="#pone-0000153-g005">Figure 5A and 5B</a>), but it is not expressed in the juvenile polyp (<a href="#pone-0000153-g005">Figure 5C</a>). A transverse section reveals that the expression is restricted along the secondary or directive axis; about 2/3 of the endodermal tissue expresses anthox6a. Six out of the eight presumptive mesenteries express anthox6a, and the sharp expression boundary appears to coincide with one pair of mesenteries. Double <em>in situ</em> labeling reveals that anthox6a is expressed on the ventral side of the directive axis (data not shown).</p>
<div class="figure" id="pone-0000153-g006"><div class="img"><a name="pone-0000153-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Mean statistical support for select phylogenetic groupings. The mean statistical support for hypothetical clades linking each of six <em>Nematostella</em> Hox/ParaHox lineages with potential homologs in the Bilateria is depicted graphically. The mean statistical support is the average of the neighbor-joining bootstrap proportion, the Bayesian posterior probability, and the maximum-likelihood bootstrap proportion. It is expressed as a percent of trees in which the given grouping was recovered. Individual <em>Nematostella</em> genes were grouped into lineages (for example, anthox1-anthox1a) when the mean statistical support for the clade uniting them exceeded the mean statistical support for any other competing relationship. The final column depicts the highest support obtained for any pairwise relationship with another cnidarian homeodomain.</p>
<span>doi:10.1371/journal.pone.0000153.g006</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The early expression of anthox6 has been described previously <a href="#pone.0000153-Finnerty1">[36]</a>. Expression first becomes visible during early planula stages as a ring in the pharyngeal endoderm (<a href="#pone-0000153-g005">Figure 5E</a>). This pattern persists in the juvenile polyp, and it extends orally to the mouth opening (<a href="#pone-0000153-g005">Figure 5D</a>). The region expressing anthox6 comes to include the endoderm at the base of the tentacles and scattered cells at the tips of the elongating tentacles (<a href="#pone-0000153-g005">Figure 5D, inset</a>).</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Anthox1 expression begins at late blastula stages, in the vegetal hemisphere of the embryo and earlier than any other Hox gene in <em>Nematostella</em>; expression then persists through gastrulation (<a href="#pone-0000153-g005">Figure 5G and 5H</a>). In the planula, anthox1 expression persists and becomes refined to the apical tuft of sensory cilia at the leading swimming end of the planula (aboral end of the adult polyp). Expression wanes in the polyp (<a href="#pone-0000153-g005">Figure 5I</a>). This putative posterior Hox gene is expressed at the opposite side of the oral-aboral axis from the anterior Hox gene anthox6, as described previously <a href="#pone.0000153-Finnerty1">[36]</a>.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Anthox1a is the sister gene of the putative posterior Hox gene anthox1, but its expression is more similar to putative anterior Hox genes, such as anthox6a. Like anthox6a, anthox1a exhibits extremely restricted expression along the directive axis (<a href="#pone-0000153-g005">Figure 5J–K</a>). This gene is expressed in a thin stripe of body wall endoderm flanking the ventral midline. This region gives rise to the ventral surfaces of the ventral pair of mesenteries in the adult. Expression persists in these two mesenteries in the juvenile polyp stage (<a href="#pone-0000153-g005">Figure 5L</a>). Expression is also seen in the endoderm at base of the tentacles (<a href="#pone-0000153-g005">Figure 5L, inset</a>).</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>Anthox7, anthox8a, and anthox8b are expressed in the body wall endoderm, in nested domains centered about the ventral midline (<a href="#pone-0000153-g005">Figure 5M–5U</a>). The expression of these genes first becomes visible in the early larva, and it persists into the juvenile polyp stage. Anthox7 is expressed in a pair of narrow, bilaterally symmetrical bands of body-wall endoderm (<a href="#pone-0000153-g005">Figure 5M</a>). Anthox8a is expressed in a single broad band along the ventral midline that encompasses the expression territories of both anthox1a and anthox7. Anthox8a is expressed along the ventral midline in the same basic expression territory as its sister gene anthox8b (encompassing the expression territories of both anthox7 and anthox1a) (<a href="#pone-0000153-g005">Figure 5P–5U</a>). However, anthox8a also exhibits an arc of expression in the pharyngeal endoderm along the dorsal surface of the pharynx. This expression corresponds to a subset of the pharyngeal ring of expression exhibited by the anterior Hox gene anthox6 (<a href="#pone-0000153-g005">Figure 5S and 5T</a>). Anthox8a and anthox8b continue to be expressed in the ventral pair of mesenteries in the juvenile polyp (<a href="#pone-0000153-g005">Figure 5R and 5U</a>), while anthox7 is expressed in the adjacent pair of ventral-lateral mesenteries (<a href="#pone-0000153-g005">Figure 5O</a>). Both anthox8a and anthox8b are expressed in the endoderm at the base of the tentacles, with anthox8a being expressed earlier, before tentacle outgrowth is initiated. The expression of anthox7 and anthox8a, but not anthox8b, have been described previously <a href="#pone.0000153-Finnerty1">[36]</a>, <a href="#pone.0000153-Matus2">[59]</a>.</p>
<a id="article1.body1.sec2.sec5.p7" name="article1.body1.sec2.sec5.p7"></a><p>Anthox2 is a Gsx (anterior ParaHox) ortholog <a href="#pone.0000153-Finnerty4">[46]</a> whose expression has been described elsewhere <a href="#pone.0000153-Finnerty5">[62]</a>. Transcripts accumulate in scattered cells around the oral pole in the ectoderm of the body column, the pharynx, and the tentacles (<a href="#pone-0000153-g005">Figure 5V–5X</a>). These cells appear to be neural precursors based on cell morphology. No aboral expression is observed. Anthox2 does not exhibit any asymmetry about the directive axis.</p>
<a id="article1.body1.sec2.sec5.p8" name="article1.body1.sec2.sec5.p8"></a><p>NVHD065 is expressed during early planula stages in two thin stripes along the ventral midline (<a href="#pone-0000153-g005">Figure 5Y–5Z</a>). These two stripes persist into the polyp stage (<a href="#pone-0000153-g005">Figure 5AA</a>). The cells expressing NVHD065 are located in the same region as the cells that express anthox1a, anthox8a, and anthox8b. These cells contribute to the development of the ventral pair of mesenteries.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Do cnidarians possess “true” Hox genes?</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Numerous Hox-related genes have been recovered from several cnidarian model systems over the last fifteen years <a href="#pone.0000153-Finnerty2">[44]</a>, <a href="#pone.0000153-Finnerty4">[46]</a>, <a href="#pone.0000153-Kamm1">[49]</a>, <a href="#pone.0000153-Kamm2">[50]</a>, <a href="#pone.0000153-Gauchat1">[63]</a>–<a href="#pone.0000153-Murtha1">[70]</a>, but the precise identity of these genes has been a contentious issue. Some of the earliest homeobox sequences recovered from the phylum Cnidaria were initially identified as anterior and central Hox genes (lab/pb-like and deformed-like) <a href="#pone.0000153-Naito1">[69]</a>–<a href="#pone.0000153-Shenk2">[74]</a>. Murtha and co-workers identified two homeobox fragments from <em>Sarsia</em> as likely Hox1 (lab) and Hox2 (pb) homologs <a href="#pone.0000153-Murtha1">[70]</a>. In addition, they suggested that a primitive axial patterning system featuring Hox genes could have evolved prior to the evolutionary split between Cnidaria and Bilateria; they also suggested that the anterior Hox genes might have evolved before the more posterior Hox genes <a href="#pone.0000153-Murtha1">[70]</a>. Both of these hypotheses are consistent with the findings of the present study. However, the orthology assignments made in these early studies were of limited reliability since they were based on pairwise alignments <a href="#pone.0000153-Naito1">[69]</a>, <a href="#pone.0000153-Murtha1">[70]</a>, <a href="#pone.0000153-Schummer1">[72]</a>–<a href="#pone.0000153-Shenk2">[74]</a> or, in one case, on a phylogenetic analysis that encompassed too narrow a sample of bilaterian homeobox genes <a href="#pone.0000153-Schierwater1">[71]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Subsequent studies relied increasingly on phylogenetic analysis of homeodomain sequences for orthology assignments. These studies consistently reported the existence of anterior (hox1 and/or hox2 related) and posterior (hox9–14 related) homeodomains in the Cnidaria <a href="#pone.0000153-Finnerty2">[44]</a>, <a href="#pone.0000153-Finnerty4">[46]</a>, <a href="#pone.0000153-Martinez1">[48]</a>, <a href="#pone.0000153-Gauchat1">[63]</a>, <a href="#pone.0000153-Galliot1">[64]</a>, <a href="#pone.0000153-Aerne1">[75]</a>. However, clear homologs of the other bilaterian Hox families (hox3–hox8) were not identified. In addition, a clear ortholog of the bilaterian Gsx gene, an “anterior” ParaHox gene was identified in the Cnidaria <a href="#pone.0000153-Finnerty4">[46]</a>. At the same time, a less convincing possible ortholog of Cdx, a “posterior” ParaHox gene was identified <a href="#pone.0000153-Finnerty4">[46]</a>. The discovery of ParaHox genes in Cnidaria is important given that Hox genes, by themselves, do not constitute a monophyletic group. The most recent common ancestor of the Hox genes appears to have given rise also to some or all of the ParaHox genes (Gsx, Xlox, and Cdx) <a href="#pone.0000153-Brooke1">[76]</a>, although the exact nature of the relationship of the Hox and ParaHox genes is still unclear <a href="#pone.0000153-GarciaFernandez2">[41]</a>, <a href="#pone.0000153-Kourakis1">[47]</a>, <a href="#pone.0000153-Chourrout1">[51]</a>. Regardless of their exact relationship, the evolution of Hox genes cannot be considered in isolation from that of the ParaHox genes.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>With the sequencing of the complete genome of the sea anemone <em>Nematostella vectensis</em>, the Hox complement of cnidarians is being re-evaluated. Kamm and co-workers <a href="#pone.0000153-Kamm1">[49]</a> culled nine Hox-related homeodomains from the genome of <em>Nematostella</em> (six of which had been previously described), and compared these, along with four Hox-related genes from the colonial hydrozoan <em>Eleutheria</em>, to bilaterian Hox and ParaHox genes using phylogenetic analysis. (The Kamm et al. 2006 study did not include the NVHD065 homeodomain included in the current study <a href="#pone.0000153-Kamm1">[49]</a>.) In agreement with numerous previous studies <a href="#pone.0000153-Finnerty2">[44]</a>, <a href="#pone.0000153-Finnerty4">[46]</a>, <a href="#pone.0000153-Martinez1">[48]</a>, <a href="#pone.0000153-Gauchat1">[63]</a>, <a href="#pone.0000153-Galliot1">[64]</a>, <a href="#pone.0000153-Aerne1">[75]</a>, the Kamm et al. phylogenetic analyses <a href="#pone.0000153-Kamm1">[49]</a> supported the existence of a cnidarian Gsx gene, the existence of a cnidarian Hox1 gene, the existence of a cnidarian Cdx gene, and the absence of central Hox genes (Hox3–8) in Cnidaria. However, Kamm and co-workers <a href="#pone.0000153-Kamm1">[49]</a> differ from previous studies in concluding that cnidarians lack any other “true” Hox genes, aside from Hox1.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>More recently, Chourrout and co-workers came to a similar conclusion; they contend that anterior Hox genes likely predated the cnidarian-bilaterian split, but that “non-anterior genes could have appeared independently in the Hox and ParaHox clusters, possibly after the separation of bilaterians and cnidarians” <a href="#pone.0000153-Chourrout1">[51]</a>. This conclusion was largely based on an unorthodox phylogenetic treatment of cnidarian and bilaterian homeodomain sequences that employed Neighbor-net analysis. Neighbor-net analysis is based on the neighbor-joining method <a href="#pone.0000153-Saitou1">[77]</a>, but it is explicitly designed to investigate and visualize “complex evolutionary scenarios” that cannot be accurately modeled by a bifurcating phylogenetic tree, <em>for example</em>, scenarios involving reticulate evolution such as gene recombination, hybridization, and horizontal gene transfer <a href="#pone.0000153-Bryant1">[78]</a>, <a href="#pone.0000153-Huson1">[79]</a>. The authors do not explicitly justify their use of Neighbor-net analysis rather than more traditional methods that produce bifurcating trees. The Hox radiation is thought to have occurred via a bifurcating process that involved repeated rounds of gene duplication and divergence. In support of this, there are many well-supported nodes in published Hox phylogenies. The Neighbour-network diagram presented by Chourrout et al. can make it more difficult to visualize these well-supported nodes because it is a two dimensional projection of a three-dimensional graph. We suggest that Neighbour-net analysis would have been more appropriate as a supplement to rather than a replacement for a traditional tree-building algorithm.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Below, we argue that recent studies systematically underestimate the phylogenetic support for the existence of multiple Hox genes in the cnidarian-bilaterian ancestor due to three important logical shortcomings. (1) These studies do not explicitly consider the weakness of the statistical support for the competing phylogenetic hypotheses that are implied or explicitly stated; (2) they do not account for the number of independent gene gains and losses that would be required by competing phylogenetic hypotheses; and (3) they do not properly root the Hox-ParaHox radiation with a succession of closely related outgroup genes (<em>for example</em>, HlxB9, Gbx, Evx, Rough, and Mox). When these issues are addressed, it becomes apparent that the support for three or more “true” Hox genes in the cnidarian-bilaterian ancestor is considerably greater than the support for only two <a href="#pone.0000153-Chourrout1">[51]</a>, one <a href="#pone.0000153-Kamm1">[49]</a>, <a href="#pone.0000153-Chourrout1">[51]</a>, or even none <a href="#pone.0000153-Kamm2">[50]</a>.</p>


<h4>Statistical support for competing hypotheses</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>In absolute terms, the statistical support for the nodes uniting cnidarian and bilaterian Hox and ParaHox genes is generally modest. The reasons for this modest statistical support include the large number of taxa involved, the small number of phylogenetic characters, the occurrence of lineage specific Hox gene duplications, and the (likely) short evolutionary interval between the initial Hox-ParaHox radiation and the cnidarian-bilaterian divergence. However, the support for these critical nodes is substantially higher than the support for the hypothetical nodes that are predicted by competing hypotheses (<a href="#pone-0000153-g006">Figure 6</a>). For example, while the mean statistical support for the clade linking anthox1/1a with Hox9+ is modest (0.39), the mean statistical support for any other pairing is practically non-existent: the pairing of anthox1/1a with Cdx receives a mean statistical support of 0.04, the pairing with Hox3/zen receives a mean statistical support of 0.02, and the pairing with Xlox receives a mean statistical support of 0.01. The mean statistical support for all other pairwise groupings between anthox1/1a and bilaterian families was 0.00. Furthermore, the highest support for a clade linking anthox1/1a to another <em>Nematostella</em> lineage was only 0.01. Clearly, the best-supported relationship for anthox1/1a is to the Hox9+ family. A similar approach bolsters the support for a close relationship between anthox2 and Gsx, between anthox6 and Hox1, and between NvHD065 and Cdx. The only case where competing pairings receive comparable support is anthox7/8/8a. The mean statistical support grouping this <em>Nematostella</em> lineage with the Hox2 family (0.13) is only slightly higher than the mean statistical support for grouping it with the Hox1 family (0.09). However, the analysis provides no support for the hypothesis that anthox7/8/8a is orthologous to Hox3, as recently proposed <a href="#pone.0000153-Chourrout1">[51]</a>—the mean statistical support for this grouping is 0.00.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>
               <a href="#pone-0000153-g007">Figure 7</a> compares the statistical support for larger phylogenetic groupings that are predicted by competing evolutionary scenarios. For example, if the hypothesis that cnidarians lack “true” Hox genes recently put forth by Kamm and Schierwater <a href="#pone.0000153-Kamm2">[50]</a> is strictly interpreted from a phylogenetic standpoint, this would imply that all of the bilaterian Hox genes share a most recent common ancestor to the exclusion of any cnidarian genes (as in <a href="#pone-0000153-g007">Figure 7A–B</a>). However, the mean statistical support for the clade that unites all bilaterian Hox genes (Hox1–Hox14) to the exclusion of all cnidarian genes is 0.00. Contrast this with the mean statistical support for the clade uniting anthox1/1a with Hox9+ (0.39), the clade uniting anthox6 with Hox1 (0.23), and the clade uniting anthox7/8a/8b with Hox2 (0.13). Viewed in this way, the phylogenetic analyses provide substantially greater support for the existence of five or six Hox/ParaHox genes in the cnidarian-bilaterian ancestor (<a href="#pone-0000153-g007">Figure 7E–F</a>) than for the existence of only one, two, or three Hox/ParaHox genes in the cnidarian-bilaterian ancestor (<a href="#pone-0000153-g007">Figure 7A–C</a>).</p>
<div class="figure" id="pone-0000153-g007"><div class="img"><a name="pone-0000153-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7. </strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Hox/ParaHox evolutionary scenarios. The phylogenies drawn here depict six mutually exclusive scenarios regarding the evolution of the Hox and ParaHox genes. Ten distinct Hox and ParaHox lineages are thought to have been present in the ancestral bilaterian (Hox1, Hox2, Hox3, Hox4, Hox5, Hox6–8, Hox9+, Cdx, Gsx, and Xlox). Five distinct Hox/ParaHox lineages are recognized for <em>Nematostella</em>. (<em>Nematostella</em> homeodomains that tend to cluster together in the phylogenetic analyses are grouped together here: anthox1/1a; anthox2/9; anthox6/6a; anthox7/8a/8b.) Assuming no gene loss in the Cnidaria, then the existence of five Hox/ParaHox lineages in <em>Nematostella</em> implies that the cnidarian-bilaterian ancestor (CBA) could have possessed as few as one Hox/ParaHox gene (scenario A) or as many as five (scenario E). There is some indication that a central class Hox gene was lost in the Cnidaria <a href="#pone.0000153-Kourakis1">[47]</a>, and that the CBA may have possessed six distinct Hox/ParaHox genes (scenario F). The ancestral Hox/ParaHox genes present in the CBA are indicated by solid squares. If a particular hypothetical clade is recovered on one or more of the phylogenetic analyses presented here, this is indicated above the relevant branch (NJ = neighbor-joining, <a href="#pone-0000153-g002">Figure 2</a>; Ba = Bayesian inference, <a href="#pone.0000153.s001">Figure S1</a>; ML = maximum- likelihood, <a href="#pone.0000153.s002">Figure S2</a>; φ = none). Below each branch, the average statistical support is indicated (NJ-bootstrap proportion+Bayes-posterior probability+ML-boostrap proportion/3). Each scenario implies a different number of lineage-specific gene losses.</p>
<span>doi:10.1371/journal.pone.0000153.g007</span></div>

<h4>Implied gene gains and losses</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Nearly all homeodomain phylogenies that have been published recently are derived from alignments of amino acid sequences. Branch lengths and topologies are optimized in order to minimize the amount of amino acid substitution that is inferred. However, preferred tree topologies do not account for gene duplication or gene loss. Compared to amino acid substitutions, gene losses and gene gains are rare evolutionary events, and they ought to be considered when attempting to choose among alternate hypotheses. Given two competing topologies that require roughly equivalent amounts of amino acid evolution, the topology that requires fewer gene gains and losses is preferable because it presupposes fewer of these relatively rare and typically unsubstantiated evolutionary events.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Among the analyses presented here, the neighbor-joining tree requires fewer lineage-specific gene losses and gains (20) than either the Bayesian tree (25) or the maximum-likelihood tree (23). The Bayesian tree is particularly unparsimonious with respect to implied gene losses by <em>Nematostella</em> (<a href="#pone.0000153.s001">Figure S1</a>); it requires that <em>Nematostella</em> underwent secondary loss of Hox2, a Hox3/Xlox precursor, Hox4, Hox5, and a Hox6–8 precursor. The maximum-likelihood tree requires that <em>Nematostella</em> lost Hox2, Hox3, and a Hox4–8/Cdx precursor (<a href="#pone.0000153.s002">Figure S2</a>). By contrast, the neighbor-joining tree requires that <em>Nematostella</em> lost a single gene (a Hox3–Hox8/Xlox precursor).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The issue of gene loss is particularly pertinent for reconstructing the evolution of the Hox2 family (<a href="#pone-0000153-g008">Figure 8</a>). On the neighbor-joining tree, anthox7/8a/8b is paired with Hox2/pb (<a href="#pone-0000153-g008">Figure 8A</a>). This topology suggests that Hox2 was present in the cnidarian-bilaterian ancestor. No gene loss is implied. However, on the maximum-likelihood tree, anthox7/8a/8b and Hox2/pb appear as independent gene lineages that emerged one after the other (<a href="#pone-0000153-g008">Figure 8B</a>). This arrangement implies two gene losses: the loss of a Hox2 gene in <em>Nematostella</em> and the loss of an anthox7/8 gene in the Bilateria. Finally, on the Bayesian tree, anthox7/8a/8b is grouped with Hox1/labial (<a href="#pone-0000153-g008">Figure 8C</a>). Anthox6/6a and Hox2/pb appear as successive outgroups to this putative Hox1 clade. This topology implies that three anterior Hox genes were present in the cnidarian-bilaterian ancestor: Hox1, Hox2, and an anthox6/6a precursor. Subsequently, the Hox2 gene was lost in the line leading to <em>Nematostella</em> and the anthox6/6a precursor was lost in the line leading to bilaterians. The scenario implied by the neighbor-joining tree is the most parsimonious, and based on this rationale, we favor the phylogenetic hypothesis generated by the neighbor-joining analysis and its implications that the cnidarian-bilaterian ancestor possessed a Hox2 gene.</p>
<div class="figure" id="pone-0000153-g008"><div class="img"><a name="pone-0000153-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8. </strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Reconstruction of Hox2 evolution. Portions of the neighbor-joining tree, the maximum-likelihood tree, and the Bayesian tree are redrawn here. The neighbor-joining tree implies that anthox7, 8a, and 8b are direct descendants of the ancestral Hox2 gene in the cnidarian-bilaterian ancestor. No gene loss is required. The maximum-likelihood tree implies that both a Hox2 precursor (square) and an anthox7/8a/8b precursor (circle) were present in the cnidarian-bilaterian ancestor. Hox2 was lost in the line leading to <em>Nematostella</em>, while anthox7/8a/8b ortholog was lost in the line leading to Bilateria. The Bayesian tree implies that a Hox2 precursor (square), a Hox1 parecursor (triangle), and an anthox6/6a precursor were present in the cnidarian-bilaterian ancestor. Hox2 was lost in the line leading to <em>Nematostella</em>, while anthox6/6a was lost in the line leading to Bilateria.</p>
<span>doi:10.1371/journal.pone.0000153.g008</span></div>

<h4>Absence of Central Hox Genes in the Phylum Cnidaria</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Numerous studies have commented on the apparent absence of central Hox genes (Hox4–Hox8) in the phylum Cnidaria <a href="#pone.0000153-Finnerty2">[44]</a>–<a href="#pone.0000153-Finnerty4">[46]</a>, <a href="#pone.0000153-Chourrout1">[51]</a>, <a href="#pone.0000153-Ryan1">[52]</a>. The lack of central genes in the Cnidaria could be explained (1) if the central Hox genes arose in the bilaterian lineage, or (2) if the central Hox genes originated prior to the cnidarian-bilaterian divergence, but they were subsequently lost in the Cnidaria. The phylogenetic analyses performed in this study tend to support the latter. On the neighbor-joining tree (<a href="#pone-0000153-g002">Fig. 2</a>), the central Hox genes appear as the sister group to a Hox3-Xlox clade. No <em>Nematostella</em> sequences fall within this clade. However, the tree implies that the common ancestor of this clade evolved prior to the cnidarian-bilaterian divergence, and that direct descendants of this ancestral gene were lost in the line leading to <em>Nematostella</em>. The maximum likelihood tree (<a href="#pone.0000153.s002">Fig. S2</a>) groups the central Hox genes with the Cdx genes. This tree likewise implies that a single gene was lost in the line leading to <em>Nematostella</em>. The Bayesian tree is least parsimonious with respect to the absence of central Hox genes in the Cnidaria (<a href="#pone.0000153.s001">Fig. S1</a>). The Bayesian tree implies that three central Hox genes were present in the cnidarian-bilaterian ancestor (Hox4, Hox5, and Hox6–8), and all three were lost in the lineage leading to <em>Nematostella</em>. Recently, Ryan and co-workers suggested a hypothesis that could explain the absence of central Hox genes in the Cnidaria without requiring gene loss in the Cnidaria. A Bayesian analysis of 455 homeodomains from human, fruit fly, and <em>Nematostella</em> placed anthox1 and anthox1a as the sister group to a clade comprising the central Hox genes and the posterior Hox genes. If this were true, then the central Hox genes evolved within the Bilateria, and no gene loss occurred in the Cnidaria. Unfortunately, the data at hand do not decisively distinguish among these plausible scenarios.</p>


<h4>Considerations regarding dataset construction</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>In contrast to the present study, Kamm and co-workers concluded that cnidarians diverged from bilaterians prior to the evolution of “a definitive Hox system” <a href="#pone.0000153-Kamm1">[49]</a> and that cnidarians lack “true Hox genes” <a href="#pone.0000153-Kamm2">[50]</a>. One basis for these conclusions was a phylogenetic analysis in which cnidarian homeodomains did not generally pair up with specific bilaterian Hox families (except for Hox1). The Kamm et al. dataset differs substantially from the current study, in that it only includes 41 Hox and ParaHox related homeodomains from Cnidaria and Bilateria <a href="#pone.0000153-Kamm1">[49]</a>. Three different cnidarian species are represented: <em>Nematostella</em>, the colonial hydrozoa <em>Eleutheria</em>, and the scleractinian coral <em>Acropora</em>. However, several important gene families that appear to be close outgroups to the Hox/ParaHox clade were not represented: HlxB9, Gbx, Evx, Rough, and Mox. In addition, the authors did not include <em>Nematostella</em> homeodomain NVHD065 which, based on our own analyses, is clearly nested among the Hox and ParaHox genes of the Bilateria. Finally, the authors do not consider their data in a rooted framework.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>To determine whether differences in the composition of the two datasets might account for differences in our conclusions, we re-analyzed the Kamm et al. dataset. We used the same evolutionary model (Dayhoff) and the same phylogenetic method (maximum-likelihood) employed in the original study, but we performed a more thorough search. Where the original study performed local rearrangements on a single starting tree, we performed global rearrangements on ten randomly generated starting trees (see <a href="#s4">Methods</a> for details). In contrast to the published study, our re-analysis groups specific cnidarian homeodomains with the Hox1, Hox2, and Hox9+ families (<a href="#pone-0000153-g009">Figure 9</a>). Eight out of ten trees that were identified in our analysis exhibited a higher likelihood than the single tree presented by Kamm and co-workers (<a href="#pone-0000153-g001">Figure 1</a> in <a href="#pone.0000153-Kamm1">[49]</a>). All eight of these trees grouped one or more cnidarian genes with Hox9–10, and three trees, including the tree with the highest likelihood (<a href="#pone-0000153-g009">Figure 9B</a>), grouped one or more cnidarian genes with Hox2. This result suggests that the Hox9+ family and possibly the Hox2 family were represented in the cnidarian-bilaterian ancestor.</p>
<div class="figure" id="pone-0000153-g009"><div class="img"><a name="pone-0000153-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9. </strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p>Re-analysis of the Kamm et al. maximum-likelihood phylogeny. (A) The redrawn maximum-likelihood phylogeny presented by Kamm et al. <a href="#pone.0000153-Kamm1">[49]</a>. The tree is unrooted. Based on the Dayhoff-PAM1 substitution matrix <a href="#pone.0000153-Dayhoff1">[129]</a> the tree's overall likelihood is −2288.08809. Bootstrap proportions determined in the original study are shown above the relevant nodes. Bootstrap proportions determined in the current study are shown below the relevant nodes. Inferred lineage specific gene losses are represented by open circles (black for Bilateria and red for Cnidaria). Inferred lineage specific gene duplications are represented by solid circles (black for Bilateria and red for Cnidaria). Sequences inferred to have been present in the common ancestor are indicated by lettered squares. The names of gene families that contain cnidarian representatives are enclosed by red lines. (B) A tree we identified using global instead of local rearrangements with the Kamm et al. data and the Dayhoff substitution matrix. The tree's overall likelihood is −2280.76406. Bootstrap proportions determined in the present study are shown below the relevant nodes. The dataset used in this analysis is available as <a href="#pone.0000153.s013">Figure S13</a>.</p>
<span>doi:10.1371/journal.pone.0000153.g009</span></div><a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>Ironically, even if the phylogeny presented by Kamm et al. <a href="#pone.0000153-Kamm1">[49]</a> accurately reflects the evolutionary relationships among cnidarian and bilaterian homeodomains, it would not support the authors' contention that the Hox system originated after the cnidarian-bilaterian split. The tree actually implies that a much more extensive Hox superfamily was present in the cnidarian-bilaterian ancestor than is currently found in either extant Cnidaria or Bilateria (<a href="#pone-0000153-g009">Figure 9a</a>). If the tree is rooted at Gsx (the most logical place to root their tree based on a recently published analysis of the homeodomain superclass <a href="#pone.0000153-Ryan1">[52]</a>), the topology implies that the twelve distinct Hox-ParaHox genes were present in the CBA (Gsx, Hox1, Hox2, Xlox/Hox3, Hox4–8, Hox9+, and Cdx), as well as several hypothetical homeodomain lineages defined by cnidarian representatives (anthox1, anthox6a, anthox7/8a/8b, anthox9, and cnox-3). Clearly, this particular phylogeny cannot be used to argue that the Hox “system” is unique to bilaterians.</p>


<h4>Evolution of the Hox and ParaHox clusters</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Originally it was thought that the Hox cluster was a result of a series of ancient tandem duplications <a href="#pone.0000153-Kappen1">[80]</a>–<a href="#pone.0000153-Zhang1">[84]</a>. In 1998, Brooke et al. <a href="#pone.0000153-Brooke1">[76]</a> put forth the theory that a four-gene ProtoHox cluster (consisting of an anterior, group 3, central and posterior ProtoHox gene) duplicated to form the Hox and ParaHox sister clusters <a href="#pone.0000153-GarciaFernandez2">[41]</a>. Recently several authors have proposed theories in which a two-gene ProtoHox cluster duplicated and the resulting clusters later independently underwent tandem gene duplications to form the extant Hox and ParaHox clusters <a href="#pone.0000153-GarciaFernandez2">[41]</a>, <a href="#pone.0000153-Chourrout1">[51]</a>.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>The seven <em>Nematostella</em> Hox genes appear to trace their origins to three genes in the cnidarian-bilaterian ancestor: two anterior Hox genes and one posterior Hox gene (<a href="#pone-0000153-g010">Figure 10</a>). In support of the original Hox-ParaHox hypothesis <a href="#pone.0000153-Brooke1">[76]</a>, the neighbor-joining and Bayesian trees group the group3 Hox and ParaHox genes (Hox3, XLOX), and the neighbor-joining tree groups the posterior Hox and ParaHox genes (Hox9+, Cdx). However, none of our analyses groups Hox1, Hox2, and Gsx into a single clade as required by the original Hox-ParaHox hypothesis <a href="#pone.0000153-Brooke1">[76]</a>, and the mean statistical support for this relationship is 0.00 (see <a href="#s4">methods</a> for 8 evolutionary scenarios that were tested). Furthermore, posterior Hox genes and Cdx do not form a monophyletic group in our maximum-likelihood or Bayesian analyses, and the mean statistical support for a sister-group relationship between Xlox and Hox3 is not overwhelming (0.41).</p>
<div class="figure" id="pone-0000153-g010"><div class="img"><a name="pone-0000153-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10. </strong></p><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1"></a><p>Phylogenetic mapping of Hox expression. The neighbor-joining and Bayesian phylogenies (<a href="#pone-0000153-g002">Figure 2</a> and <a href="#pone.0000153.s001">S1</a>) were pared to remove all bilaterian sequences. The strict consensus topology shown here depicts the relative relationships among <em>Nematostella</em> sequences. Each of the <em>Nematostella</em> Hox-related sequences is coded according to whether its expression is restricted along the primary (O/A) body axis or the secondary (directive) body axis (Y = yes; N = no). A yellow Y in the directive column signifies that the expression is bilateral (both sides of the directive axis), and a red Y indicates that the expression is unilateral. The character state found in the terminal taxon is indicated in the colored boxes. The internal nodes are shaded to indicate the character states found in hypothetical ancestors. For each gene, the spatial expression is depicted on a diagram of the juvenile polyp. In the case of Dlx, anthox6a and anthox1, the expression pattern that is depicted actually occurs earlier, in the larval stage, but it is represented on a diagram of the polyp to facilitate spatial comparisons with the other genes. The polyp is drawn in lateral view with the overlying ectoderm (dark gray) partially peeled away to reveal the underlying endoderm of the body column (light gray) and the lumen of the pharynx (white). The pharynx is drawn as though everted. Only one representative tentacle is shown. The mesoglea, a largely acellular layer of connective tissue that separates the endoderm from the ectoderm, is depicted as a thin black line. Gene expression is depicted as black shading in the endoderm or ectoderm. The major regions along the primary body axis are demarcated with dotted lines: Ph = pharynx; H = head; C = column; F = foot. Cross-sectional views through the body column (at the arrowheads) are shown for Gbx, anthox7, anthox8a, anthox8b, anthox6a, anthox1a, and NVHD065.</p>
<span>doi:10.1371/journal.pone.0000153.g010</span></div><a id="article1.body1.sec3.sec6.p3" name="article1.body1.sec3.sec6.p3"></a><p>Our analysis, unlike the original Brooke et al. paper <a href="#pone.0000153-Brooke1">[76]</a> and subsequent Hox/ParaHox cluster analyses <a href="#pone.0000153-GarciaFernandez2">[41]</a>, benefits from being able to have a full genome from which to determine reliable outgroups to root our phylogeny. The fact that Gsx forms an independent lineage in all of our analyses (<a href="#pone-0000153-g002">Figure 2</a>, <a href="#pone.0000153.s001">S1</a>, and <a href="#pone.0000153.s002">S2</a>), poses a major stumbling block for any theory that considers the Hox and ParaHox sister clusters.</p>
<a id="article1.body1.sec3.sec6.p4" name="article1.body1.sec3.sec6.p4"></a><p>Several lines of evidence in our data suggest a novel hypothesis—that this two-gene ParaHox cluster may have been formed as a result of a tandem duplication rather than a cluster duplication. First and foremost, Gsx consistently emerges as an independent lineage, which is inconsistent with a cluster-duplication scenario, and characteristic of a lineage that was formed by a tandem duplication event. Also, a Hox3 gene has never been recovered from a cnidarian despite numerous PCR surveys (for example <a href="#pone.0000153-Kuhn1">[66]</a>, <a href="#pone.0000153-Kuhn2">[67]</a>, <a href="#pone.0000153-Schierwater1">[71]</a>), EST studies (for example <a href="#pone.0000153-Kortschak1">[85]</a>, <a href="#pone.0000153-Technau1">[86]</a> ) and two full genome scans (<em>Hydra</em> 
               <a href="#pone.0000153-Chourrout1">[51]</a> and <em>Nematostella</em> 
               <a href="#pone.0000153-Kamm1">[49]</a>–<a href="#pone.0000153-Ryan1">[52]</a>). The moderate affinity of Hox3 for Xlox in our analyses may be due to convergence or it may represent a bilaterian-specific duplication. Finally, any evidence uniting Cdx and posterior Hox genes or Hox3 and Xlox could simply have been a result of those genes being the last in a series of tandem duplications. It is not difficult to imagine a scenario in which a series of tandem duplication events, occurring at both the 5′ and 3′ ends of a primordial ProtoHox cluster, could have led to the ParaHox genes Gsx and NVHD065 being clustered adjacent to one another at one end of the cluster and, subsequently, detached via a translocation event (<a href="#pone-0000153-g011">Figure 11I</a>). If this scenario is correct, the ParaHox genes would represent detached Hox genes rather than a sister cluster.</p>
<div class="figure" id="pone-0000153-g011"><div class="img"><a name="pone-0000153-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000153" data-uri="info:doi/10.1371/journal.pone.0000153.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000153.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000153.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000153.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000153.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11. </strong></p><a id="article1.body1.sec3.sec6.fig2.caption1.p1" name="article1.body1.sec3.sec6.fig2.caption1.p1"></a><p>Origins of the Hox and ParaHox clusters. (A) The tandem duplication model of Hox and ParaHox clusters. Our evidence conflicts with the current theory that a multigene duplication event formed the Hox and ParaHox cluster. Under this alternative model, each Hox and ParaHox gene was formed through a series of tandem duplications either in the 3′ or 5′ direction and a translocation event separated the two ParaHox genes from the Hox cluster. Hox3 and Central Hox genes are not included in this model because it is not clear whether they were present in the cnidarian-bilaterian ancestor and subsequently lost in the Cnidaria or alternatively, were derived in the Bilateria from subsequent tandem duplication events of another Hox/ParaHox gene (for example anthox7/8a/8b or anthox1/1a). The general scenario presented here is robust enough to easily accommodate either event. (B) In this scenario adapted from Chourrout et al. 2006 <a href="#pone-0000153-g003">Figure 3</a>, a two-gene ProtoHox cluster is duplicated to form two sister clusters, the Hox and ParaHox clusters <a href="#pone.0000153-Chourrout1">[51]</a>. Chourrout et al. found that NVHD065 (Xlox/Cdx in their analysis) showed conflicting affinities for Xlox and Cdx; similarly our neighbor-joining and Bayesian trees had NVHD065 grouping with Cdx while the maximum-likelihood tree had NVHD065 grouping with Xlox. Unlike our study, Chourrout et al. considered anthox1 and anthox1a (HoxE and HoxF in their analysis) to be non-Hox/ParaHox lineages, hence their omission from their model <a href="#pone.0000153-Chourrout1">[51]</a>.</p>
<span>doi:10.1371/journal.pone.0000153.g011</span></div>

<h4>Do cnidarians employ a Hox code?</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>To say that cnidarians utilize a Hox code, as this concept is generally understood, they must possess multiple Hox genes, and these genes must specify distinct regions along a body axis. Most authors refer specifically to the primary body axis when discussing the Hox code, but other conserved Hox axes have been described, for example in the vertebrate urogenital system, digestive system, and paired appendages <a href="#pone.0000153-Sekimoto1">[87]</a>–<a href="#pone.0000153-Taylor1">[89]</a>. Evidence suggests that the ancestor to bilaterians had a seven gene Hox cluster and at least a three-gene ParaHox cluster <a href="#pone.0000153-GarciaFernandez2">[41]</a>. However, as is evident in several disparately related species, the existence of a Hox code does not require the existence of intact Hox clusters or Hox colinearity. Despite claims to the contrary <a href="#pone.0000153-Kamm1">[49]</a>, <a href="#pone.0000153-Kamm2">[50]</a>, our findings suggest that <em>Nematostella</em> has at least three “true” Hox genes and two “true” ParaHox genes, and their Hox genes are expressed in distinct regions along the primary body axis. In addition, there is clear evidence for both a Hox cluster and ParaHox cluster in <em>Nematostella</em> 
               <a href="#pone.0000153-Chourrout1">[51]</a>.</p>


<h4>Nematostella Hox Linkage and the “Hox-code”</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>Recently, Kamm and co-workers <a href="#pone.0000153-Kamm1">[49]</a> concluded that, “with the exception of independently duplicated genes, the cnidarian [Hox-related] genes are unlinked.” Unfortunately, their study relied on contigs generated by searching the NCBI trace archive with MegaBlast rather than a full genome assembly, and this conclusion proved to be incorrect <a href="#pone.0000153-Kamm1">[49]</a>. With the benefit of additional data from two publicly available genome assemblies and gene mapping experiments, we can clearly recognize linked clusters of Hox and Hox-related genes that were not detected by Kamm et al. <a href="#pone.0000153-Kamm1">[49]</a>. Our linkage findings corroborate those of Chourrout and co-workers <a href="#pone.0000153-Chourrout1">[51]</a> who used another genome sub-assembly method, described only by name (Marche à Droite (Màd)), to produce their own set of <em>Nematostella</em> contigs.</p>


<h4>Nematostella Hox gene expression and the “Hox-code”</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>The seven putative Hox genes of <em>Nematostella</em> are expressed in three principal domains along the primary body axis and, collectively, these account for practically the entire axis: the pharynx, the body column, and the aboral extremity. Anthox6 is expressed in an endodermal ring lining of the pharynx at the oral end of the body, adjacent to <em>Nv-otx</em> expression <a href="#pone.0000153-Mazza1">[90]</a>. Anthox1a, anthox7, anthox8a, and anthox8b are expressed widely throughout the endoderm of the body column, from the tentacle zone to the foot region. Anthox1 is expressed in a discrete spot of ectoderm at the aboral extremity.</p>
<a id="article1.body1.sec3.sec9.p2" name="article1.body1.sec3.sec9.p2"></a><p>We hypothesize that multiple Hox genes are involved in patterning the primary body axis of <em>Nematostella</em>. This conclusion is clearly supported by the expression data, though functional studies will be necessary to prove that Hox genes are <em>required</em> for axial patterning in <em>Nematostella</em>. If Hox genes are involved in patterning the primary body axis of <em>Nematostella</em>, this suggests that a simple “Hox code” existed prior to the divergence of Cnidaria and Bilateria. To presume that distinct Hox expression domains have independently evolved to differentiate axial regions along the primary axis in both the Cnidaria and the Bilateria is clearly less parsimonious. Importantly, we should not expect a cnidarian “Hox code” to be as elaborate as that seen in the model bilaterians because the Cnidaria have not achieved the same degree of axial complexity as fruit flies or vertebrates. Nor should we expect a precise correspondence between the Hox expression territories of Cnidaria and Bilateria, as independent evolution within the long diverged cnidarian and bilaterian lineages will tend to obscure evidence of homologous pattering mechanisms. Furthermore, to say that a Hox code existed prior to the cnidarian-bilaterian split is not to say that the Hox code was the original metazoan axial patterning system. Other possible axial patterning systems, such as the Wnt genes <a href="#pone.0000153-Kusserow1">[91]</a>, may have been in place prior to the Hox code. Studies on additional basal metazoan taxa may shed light on the origin of animal body axes.</p>
<a id="article1.body1.sec3.sec9.p3" name="article1.body1.sec3.sec9.p3"></a><p>Kamm and coworkers <a href="#pone.0000153-Kamm1">[49]</a> have argued against the existence of a cnidarian Hox code because the larval expression of <em>Nematostella</em> anthox6 differs from its ortholog in the colonial hydrozoan Eleutheria <a href="#pone.0000153-Kamm1">[49]</a>. However, the Anthozoa and Hydrozoa are separated by more than 500 million years of evolutionary time <a href="#pone.0000153-Peterson1">[92]</a>, and the compound life history of the Hydrozoa, which features two major adult body plans (polyp and medusa), is believed to be a derived feature within the Cnidaria <a href="#pone.0000153-Collins2">[93]</a>. Furthermore, the polyp of medusozoan cnidarians (including Hydrozoa), are believed to be secondarily reduced <a href="#pone.0000153-Collins2">[93]</a>. It would not be surprising if the origin of a novel adult body plan, accompanied by the simplification of the ancestral adult body plan, would be associated with changes in the spatial deployment of Hox genes. The origin of novel body plans has been accompanied by radical alterations of Hox expression in both echinoderms <a href="#pone.0000153-ArenasMena1">[94]</a> and cephalopods <a href="#pone.0000153-Lee1">[95]</a>. In addition, Hox gene expression patterns are known to change radically even in the absence of major alterations to animal body plans; for example, the Hox3 homologs of <em>Drosophila</em> have abandoned their ancestral role in the Hox code <a href="#pone.0000153-Bonneton1">[96]</a>–<a href="#pone.0000153-Stauber2">[100]</a>.</p>
<a id="article1.body1.sec3.sec9.p4" name="article1.body1.sec3.sec9.p4"></a><p>Kamm and coworkers <a href="#pone.0000153-Kamm1">[49]</a> argue that the expression of <em>Nematostella</em>'s Hox genes in different germ layers is additional evidence that the “Hox system” originated after the cnidarian-bilaterian split. It is quite common for Hox genes to be expressed in multiple germ layers in the Bilateria. For example, while Hox genes are most often thought of as being expressed in the central nervous system <a href="#pone.0000153-Hunt1">[1]</a>, <a href="#pone.0000153-Lichtneckert1">[7]</a>, <a href="#pone.0000153-Graham1">[8]</a>, <a href="#pone.0000153-Manzanares1">[101]</a>, <a href="#pone.0000153-Kourakis2">[102]</a>, Hox genes are also coordinately expressed in axial <a href="#pone.0000153-Burke1">[3]</a>, <a href="#pone.0000153-Iimura1">[103]</a> and limb bud mesoderm <a href="#pone.0000153-Duboule1">[104]</a>, as well as in endodermal derivatives <a href="#pone.0000153-Kawazoe1">[9]</a>, <a href="#pone.0000153-Sakiyama1">[10]</a>, <a href="#pone.0000153-vandenAkker1">[105]</a>, <a href="#pone.0000153-Frobius1">[106]</a>. Interestingly, Hox gene expression in mesoderm has been shown to be important for endodermal patterning <a href="#pone.0000153-Roberts1">[107]</a> and endodermal expression of Hox genes in anthozoans may be related to the bifunctional mesendodermal status of anthozoan gastrodermis <a href="#pone.0000153-Martindale1">[108]</a>.</p>
<a id="article1.body1.sec3.sec9.p5" name="article1.body1.sec3.sec9.p5"></a><p>Ironically, despite Kamm and coworkers <a href="#pone.0000153-Kamm1">[49]</a> using the expression patterns of “Hox-like” genes in the hydrozoan <em>Eleutheria</em> as counter evidence for the existence a “Hox system”, the genes show clear signs of axial expression <a href="#pone.0000153-Kamm1">[49]</a>. The “posterior Hox/Cdx-like” gene Cnox-4ed is expressed at the aboral pole (<a href="#pone-0000153-g003">Figure 3H</a> in <a href="#pone.0000153-Kamm1">[49]</a>) and the “anterior Hox-like” gene Cnox-5ed is expressed at the opposite oral pole of primary polyps (<a href="#pone-0000153-g003">Figure 3E and 3M</a> in <a href="#pone.0000153-Kamm1">[49]</a>). This evidence suggests that at least some components of the “Hox system” are evident in this highly derived hydrozoan.</p>
<a id="article1.body1.sec3.sec9.p6" name="article1.body1.sec3.sec9.p6"></a><p>In contrast to Kamm et al., we have the advantage of surveying the expression of the entire compliment of <em>Nematostella</em> Hox genes. One interesting feature gleaned from this survey is the potential role of <em>Nematostella</em> Hox genes in patterning the secondary (directive) body axis. Anthox1a shows a highly restricted pattern along the ventral midline of the endodermal body wall (<a href="#pone-0000153-g005">Figure 5J</a>) and anthox7 shows a complimentary pattern on its dorsal boundary (<a href="#pone-0000153-g005">Figure 5N</a>), while anthox8a and anthox8b encompasses the entire anthox1a and anthox7 expression domain (<a href="#pone-0000153-g005">Figure 5J–5U</a>). Anthox6a shows the broadest domain of expression, extending across approximately 2/3 of the dorso-ventral axis (<a href="#pone-0000153-g005">Figure 5E</a>). Only the midline of the dorsal body wall fails to express Hox genes, although the dorsal pharynx expresses anthox8a (<a href="#pone-0000153-g005">Figure 5P and 5R</a>).</p>


<h4>Evolution of axial patterning</h4>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1"></a><p>Considering the evidence presented here, it appears possible that both the Cnidaria and the Bilateria utilize Hox genes to pattern both the primary and secondary body axes. It is therefore interesting to speculate which role evolved first. Mapping the expression patterns of the Hox-related genes of <em>Nematostella</em> onto the neighbor-joining phylogeny suggests that restricted expression along the primary body axis is a primitive trait of the larger Hox-related clade (<a href="#pone-0000153-g010">Figure 10</a>). Of the basal Hox-related lineages (Gbx, HlxB, Rough, Eve, Mox) all exhibit restricted expression along the primary body axis except for Rough. None exhibit restricted expression along the secondary axis, except for Gbx. In contrast, the majority of the genes that make up the Hox and ParaHox clade exhibit restricted expression along the secondary body axis. Additionally, the restricted expression of Gbx along the secondary axis differs from the restricted expression of Hox/ParaHox genes along the same axis in that Gbx expression is not offset to the ventral side. Gbx is expressed in a broad swath in the middle of the directive axis.</p>
<a id="article1.body1.sec3.sec10.p2" name="article1.body1.sec3.sec10.p2"></a><p>The most parsimonious explanation for these data is that restricted expression along the directive axis evolved twice in <em>Nematostella</em> Hox genes, once in the Gbx lineage (see triangular node on <a href="#pone-0000153-g010">Figure 10</a>), and once near the base of the Hox-ParaHox radiation (see star-shaped node on <a href="#pone-0000153-g010">Figure 10</a>). A less parsimonious competing hypothesis would be that directive patterning was lost in three lineages (Rough/Eve, Mox, and anthox2) and then became restricted to a single side of the embryo in the Hox/ParaHox lineage.</p>
<a id="article1.body1.sec3.sec10.p3" name="article1.body1.sec3.sec10.p3"></a><p>Regardless of which of these hypotheses is favored, the analysis suggests that the ancestral gene that gave rise to all Hox and ParaHox genes (besides Gsx) might have been differentially expressed along both the primary and secondary body axis. Later radiations of this ancestral gene led to (at least) four main lineages: anthox6/6a, anthox7/8a/8b, anthox1/1a, and NVHD065. At least three of these ancestral genes (<em>i.e.</em>, anthox6/6a, anthox7/8a/8b, and anthox1/1a) subsequently radiated independently in the lineage leading to <em>Nematostella</em>. In the anthox6/6a and anthox1/1a lineages (see circular nodes on <a href="#pone-0000153-g010">Figure 10</a>), the directive-patterning component would appear to have been lost in one of the resulting paralogs (i.e. anthox6 and anthox1). While this analysis is sensitive to tree topology, it offers novel predictions about the evolution of Hox-gene expression. A thorough test of these predictions will require expression data for the full complement of Hox-related genes from other basal animal models, including other Anthozoa, medusozoan cnidarians, ctenophores, and sponges.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1"></a><p>Cnidarians (in particular, anthozoan cnidarians) have the potential to provide important insight into the evolution of metazoan body plans. Although not morphologically complex, these animals have a surprisingly sophisticated genomic repertoire that might link the axial organization of anthozoans with bilaterians. The current multi-dimensional study takes an in-depth look at the phylogenetic relationships, patterns of gene expression, and genomic arrangements of Hox genes and ParaHox genes of the starlet sea anemone, <em>Nematostella vectensis</em>. Our results demonstrate the following. (1) <em>Nematostella</em> has a seven-gene cluster comprising three <em>bona fide</em> Hox genes and four Hox-related genes and a separate two-gene ParaHox cluster (previously reported in <a href="#pone.0000153-Chourrout1">[51]</a>). (2) There appear to be seven <em>bona fide</em> Hox genes in <em>Nematostella</em> belonging to the Hox1, Hox2, and Hox9–14 subfamilies, and they are expressed in three distinct domains that collectively span almost the entire primary body axis. The expression of multiple Hox genes in distinct domains along the primary body axis suggests the existence of a rudimentary “Hox code.” (3) Hox genes are also expressed in nested spatial domains along the secondary (directive) axis, which suggests that they could be playing a role in patterning this body axis. (4) Phylogenetic evidence suggests that the Hox and ParaHox clusters may have formed through a series of independent tandem gene duplications and not necessarily via a cluster duplication. Additional data from other basal metazoan groups will help to elucidate which of <em>Nematostella's</em> traits are ancestral and which might be derived. The additional taxa that should prove especially useful would be (1) another anthozoan cnidarian such as a scleractinian coral, a sea pen, or a ceriathian tube anemone: (2) one or more medusozan cnidarians such as <em>Hydra</em>, <em>Hydractinia</em>, <em>Eleutheria</em>, or <em>Aurelia</em>; (3) a ctenophore, such as <em>Mnemiopsis</em>; (4) the placozoan <em>Trichoplax</em>; (5) a sponge, such as <em>Amphimedon</em> (formerly <em>Reniera</em>); (6) and an acoel flatworm.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Phylogenetic Analysis</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The evolutionary relationships among 61 Hox-related genes were estimated by performing three separate phylogenetic analyses (neighbor-joining <a href="#pone.0000153-Saitou1">[77]</a>, Bayesian inference <a href="#pone.0000153-Sinsheimer1">[109]</a>, <a href="#pone.0000153-Yang1">[110]</a>, and maximum-likelihood <a href="#pone.0000153-Felsenstein1">[111]</a> on the 60 amino acids of the homeodomain (<a href="#pone-0000153-g001">Figure 1</a>). The dataset utilized in this study is as compact as possible to minimize computer run times while still representing all of the known Hox-related gene families of Cnidaria, Deuterostomia, and Protostomia. To represent cnidarians, eighteen Hox-related genes were chosen from the sea anemone <em>Nematostella vectensis</em>. These Hox-related genes were identified in a previous study that compared complete or near-complete Hox complements from the sequenced genomes of human, fruit fly, <em>Nematostella</em> 
               <a href="#pone.0000153-Ryan1">[52]</a>. Unlike this previous study, we selected 20 homeodomains from the lancelet <em>Branchiostoma floridae</em> and only one homeodomain from <em>Homo sapiens</em> to represent the deuterostomes. In general, the lancelet was preferred over the human because its lineage has not experienced the Hox cluster duplications that have occurred during the ancestry of vertebrates <a href="#pone.0000153-Pendleton1">[112]</a>. However, a human sequence was chosen to represent the HlxB9 gene family because HlxB9 has not yet been recovered in <em>Branchiostoma</em>. To represent the protostomes, we selected 16 homeodomains from <em>Drosophila melanogaster,</em> one from the spider <em>Cupiennius salei</em> 
               <a href="#pone.0000153-Damen1">[<em>113</em>]</a>, and one from the polychaete worm <em>Capitella</em> 
               <a href="#pone.0000153-Frobius1">[106]</a>. The Hox3 gene of <em>Cupiennius</em> was selected because the three Hox3-related genes of Drosophila (<em>zen1</em>, <em>zen2</em>, and <em>bicoid</em>) are known to be highly derived relative to the ancestral Hox3 sequence <a href="#pone.0000153-Bonneton1">[96]</a>–<a href="#pone.0000153-Stauber2">[100]</a>. The XLOX homeodomain of <em>Capitella</em> was selected because XLOX has not been identified in the fruitfly. Distalless (dll) sequences from <em>Nematostella</em>, <em>Branchiostoma</em>, and <em>Drosophila</em> were used as an outgroup—this outgroup designation is based on the results of a much more extensive previous analysis <a href="#pone.0000153-Ryan1">[52]</a>. The phylogenetic dataset is available as a supplemental file (<a href="#pone.0000153.s012">Figure S12</a>).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Neighbor-joining <a href="#pone.0000153-Saitou1">[77]</a> analysis was performed using the computer package Phylip (version 3.6.1; <a href="#pone.0000153-Felsenstein2">[114]</a>). Distances among homeodomains were calculated using the ProtDist program of Phylip and the James-Taylor-Thorton (JTT) distance matrix <a href="#pone.0000153-Jones1">[115]</a>. The JTT matrix was determined to be superior to the other available distance matrices within ProtDist using ProtTest 1.3 <a href="#pone.0000153-Abascal1">[116]</a>. Support for clades on the neighbor-joining tree was assessed by 1000-replicates of bootstrap <a href="#pone.0000153-Felsenstein3">[117]</a>. The neighbor-joining tree, like the other two trees, was re-drawn and re-rooted using the Dll sequences as an outgroup with the computer program MacClade, version 4.03 <a href="#pone.0000153-Maddison1">[118]</a>.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Maximum-likelihood <a href="#pone.0000153-Felsenstein1">[111]</a> analysis was performed using the Proml program of the computer package Phylip, version 3.6.1 <a href="#pone.0000153-Felsenstein2">[114]</a>. The best model of protein evolution available within Proml was determined using ProtTest 1.3 <a href="#pone.0000153-Abascal1">[116]</a>: JTT plus Gamma distributed rates plus invariant residues. Five rate categories were specified, including one category to accommodate invariant residues. The proportion of invariant residues was determined empirically (0.119). The coefficient of variation of substitution rates among sites was set to 0.893. Global rearrangements were performed on 1000 starting trees produced by random addition of taxa. Support for particular clades was assessed by 100-replicates of the bootstrap <a href="#pone.0000153-Felsenstein3">[117]</a>.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>Bayesian analysis was performed using MrBayes version 3.1.2-MPI under a mixed rate model (aamodelpr = mix) <a href="#pone.0000153-Ronquist1">[119]</a>. Two simultaneous, completely independent Markov chain Monte Carlo searches were run, starting from different random trees (Nruns = 2). The search was conducted for 20,000,000 generations with trees being sampled every 100 generations and printed every 1000. Each Markov chain generated 200,000 trees. The two resulting treefiles were meshed and the first 80,000 trees were discarded as “burnin.” The Consense program of Phylip <a href="#pone.0000153-Felsenstein2">[114]</a> was used to build a “Majority rule (extended)” tree from the remaining 320,000 trees.</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>To facilitate direct comparisons among studies, we also re-analyzed the homeodomain dataset of Kamm et al. <a href="#pone.0000153-Kamm1">[49]</a>. In our re-analysis of the Kamm et al. dataset, we used the same basic phylogenetic methodology that generated <a href="#pone-0000153-g001">Figure 1</a> of the original study: we performed a maximum-likelihood analysis using the Dayhoff amino acid substitution matrix. However, where the original study utilized local rearrangements on a single starting tree, we conducted a more thorough search using global rearrangements on ten randomly generated starting trees. The phylogenetic dataset is available as a supplemental file (<a href="#pone.0000153.s013">Figure S13</a>).</p>


<h4>Statistical support for alternative hypothesis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Statistical support for alternative hypotheses was determined by constructing a constraint trees for each alternate hypothesis and using the constraints to filter trees from each analysis (100 bootstrap trees from maximum-likelihood, 1000 trees from neighbor-joining analysis, and 320,000 trees from Bayes) in Paup* <a href="#pone.0000153-Swofford1">[120]</a>. The mean statistical support is the average of the neighbor-joining bootstrap proportion, the Bayesian posterior probability, and the maximum-likelihood bootstrap proportion. It is expressed as a percent of trees in which the given grouping was recovered. Because of rounding, 0.00 does not mean absolutely no support, rather it means less than 0.05.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For the relationship of Gsx genes with the Hox1 and Hox2 lineages, 8 constraints with various <em>Nematostella</em> members were constructed. The bilaterian Hox1, Hox2, and Gsx genes and the <em>Nematostella</em> gene anthox2 were the core members of each constraint. One set of analysis included anthox6 and anthox6a in addition to the core members. Another set included anthox7, anthox8a, and anthox8b along with anthox6 and anthox6a. In addition, anthox2a was added to each of the previous constraints.</p>


<h4>Determination of Gene Linkage and Gene Structure</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Clusters of Hox-related homeobox genes were identified by screening a draft assembly of the <em>Nematostella</em> genome with a dataset of 123 <em>Nematostella</em> homeobox sequences <a href="#pone.0000153-Ryan1">[52]</a>. The assembly was produced using the Phusion program <a href="#pone.0000153-Mullikin1">[121]</a> and is available at StellaBase <a href="#pone.0000153-Sullivan1">[53]</a>, <a href="#pone.0000153-1">[54]</a> Clusters were verified and extended by comparison with the genome assembly produced by the Joint Genome Institute <a href="#pone.0000153-2">[55]</a>. Additional experimental confirmation was provided by the following: (1) isolation and sequencing of clones from a genomic library prepared in the Lambda FixII vector (Stratagene), (2) genomic walking using the techniques of ligation-mediated PCR and long PCR, and (3) isolation and sequencing of transcripts obtained by screening a cDNA library or by RACE. Comparisons between the transcripts and the genomic sequence were used to determine gene structure.</p>


<h4>Isolation of transcripts</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Gene specific primers were designed to amplify partial transcripts using 5′ and 3′ RACE. Where possible, full-length transcripts were assembled by conceptually splicing overlapping 5′ and 3′ RACE products. Primer design was based on smaller gene fragments originally obtained by degenerate PCR in the case of anthox1, anthox1a, anthox7, anthox8, and evx, <a href="#pone.0000153-Finnerty2">[44]</a>, <a href="#pone.0000153-Finnerty3">[45]</a>. Full length transcripts were isolated from a cDNA library for anthox2 and anthox6 <a href="#pone.0000153-Finnerty4">[46]</a>. Primer design was based on consulting genomic trace sequences for Dll, HlxB9, Gbx, Mox, Rough, anthox2a, anthox9, and NvHD065. All sequences have been deposited in the GenBank database (Accession numbers are provided in <a href="#pone-0000153-g001">Figure 1</a>.). Primer sequences are available upon request from M. Q. Martindale <a href="mailto:(mqmartin@hawaii.edu)">(mqmartin@hawaii.edu)</a>
            </p>


<h4>In Situ Hybridization<a href="#pone.0000153-Martindale1">[108]</a>
            </h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>For each gene under study, digoxygenin-labeled sense and anti-sense probes were produced using the 3′ RACE products as templates according to published protocols <a href="#pone.0000153-Matus1">[57]</a>, <a href="#pone.0000153-Finnerty5">[62]</a>, <a href="#pone.0000153-Martindale1">[108]</a>, <a href="#pone.0000153-Magie1">[122]</a>. All probes spanned a small region of the homeobox, the coding sequence downstream of the homeobox and the 3′ untranslated region. Probe length ranged from 323–1200 nucleotides. Hybridizations were performed as previously described at 65°C for 20–44 hours at probe concentrations of 1.0 ng/ml.</p>


<h4>Phylogenetic analysis of gene losses, gene gains, and gene expression changes</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Gene expression patterns for 14 <em>Nematostella</em> genes were scored with respect to whether their expression is axially restricted along the primary axis, the oral-aboral axis, and the secondary axis, the directive axis <a href="#pone.0000153-Stephenson2">[123]</a>. These character states were then mapped onto neighbor-joining, Bayesian, and maximum-likelihood phylogenies of the 14 <em>Nematostella</em> genes. The phylogenies were obtained by removing non-<em>Nematostella</em> taxa from the neighbor-joining analysis in <a href="#pone-0000153-g002">figure 2</a>, the Bayesian phylogeny in <a href="#pone.0000153.s001">supplemental figure 1</a>, and the maximum-likelihood phylogeny in <a href="#pone.0000153.s002">supplemental figure 2</a>. Character mapping was performed using the ACCTRAN character state optimization method implemented in the computer program MacClade, version 4.03; <a href="#pone.0000153-Maddison2">[124]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000153"></div><a name="pone.0000153.s001" id="pone.0000153.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Homeodomain phylogeny based on Bayesian inference. The tree is rooted and labeled in the same manner as <a href="#pone-0000153-g002">Figure 2</a>. Posterior probabilities are shown at each node. The dataset used in this analysis is available as <a href="#pone.0000153.s012">Figure S12</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.27 MB EPS)</p>
<a name="pone.0000153.s002" id="pone.0000153.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Homeodomain phylogeny based on maximum-likelihood. The tree is rooted and labeled in the same manner as <a href="#pone-0000153-g002">Figure 2</a>. Bootstrap proportions are shown at each node. The overall likelihood of the tree is −3103.08424. The dataset used in this analysis is available as <a href="#pone.0000153.s012">Figure S12</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.27 MB EPS)</p>
<a name="pone.0000153.s003" id="pone.0000153.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Annotated anterior-Hox cluster of <em>Nematostella</em>. Two overlapping genomic scaffolds (C408300840.Contig2 and C402800703.Conting1) that contain seven ANTP class genes (anthox6, anthox7, anthox8a, anthox8b, Evx, Rough, and HlxB9) were identified in an assembly of the <em>Nematostella</em> genome produced using the computer program Phusion <a href="#pone.0000153-Mullikin1">[121]</a>. A PRD class homeobox belonging to the Dmbx family is also linked to this cluster. These eight homeobox genes are clustered within a 329,244-nucleotide span. A 4,930-nucleotide region containing part of the second exon of anthox8b is common to both genomic scaffolds (double dotted line). The cluster was confirmed computationally by comparing it to the independently generated <em>Nematostella</em> genome assembly produced by the Joint Genome Institute (U. S. Department of Energy) <a href="#pone.0000153-2">[55]</a>. The two overlapping genomic scaffolds in the Phusion assembly align to a single scaffold in the JGI assembly spanning 1073713 nucleotides (scaffold-61). The larger JGI scaffold also encompasses an additional ANTP class homeodomain belonging to the HLX family (HLXd; <a href="#pone.0000153-Ryan1">[52]</a>). Portions of the cluster were corroborated experimentally through the independent sequencing of RACE products, cDNA clones, genomic clones, and PCR fragments. The locations of these corroborating segments are indicated below the cluster. The sequence of the anthox7, anthox8a, anthox8b, Evx, Rough, and HlxB9 transcripts were determined by conceptually splicing overlapping 5′ and 3′ RACE products. The sequence of the anthox6 transcript was previously determined by isolating and sequencing a full-length clone from a cDNA library <a href="#pone.0000153-Finnerty4">[46]</a>. Transcriptional orientation is indicated by an arrow above each locus. Coding regions of homeobox genes are indicated by boxes: red = homeobox sequences; gray = other protein coding regions; black = untranslated regions.; Gray spheres indicate approximate location of non homeodomain genes predicted from BLASTx search of the RefSeq database (NCBI: latest release_07.17.06). Lengths of segments flanking homeodomain genes (in nucleotides) are given.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.09 MB EPS)</p>
<a name="pone.0000153.s004" id="pone.0000153.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Annotated Mox cluster of <em>Nematostella</em>. Four Mox loci exhibiting the same transcriptional orientation (5′ to 3′ from left to right), are linked within a span of 23,212 nucleotides on a single genomic scaffold in the Phusion assembly 112,563 nucleotides long. A 345,744 nucleotide scaffold encompassing this cluster is contained in the JGI assembly of the <em>Nematostella</em> genome (scaffold-248) <a href="#pone.0000153-2">[55]</a>. A TBLASTN analysis of the scaffold against the homeodomain dataset did not identify any additonal homeodomains. The sequence of moxD and moxC transcripts were determined by conceptually splicing overlapping 5′ and 3′ RACE products. 3′ RACE products were used to annotate the homeodomain containing exon and the 3′ UTR of moxB and moxA. The figure is labeled as in <a href="#pone.0000153.s003">Figure S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.63 MB EPS)</p>
<a name="pone.0000153.s005" id="pone.0000153.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Annotated ParaHox cluster of <em>Nematostella</em>. The Gsx ortholog, anthox2 is linked to the possible Cdx/Xlox homolog, NVHD065. A single genomic scaffold (402901197.Contig1) containing the Gsx ortholog and NVHD065 was identified in an assembly of the <em>Nematostella genome</em> produced using Phusion <a href="#pone.0000153-Mullikin1">[121]</a>. The structures of Gsx and NVHD065 were coroborated experimentally through independent sequencing of RACE products and cDNA clones. The sequence of the Gsx transcript was previously determined by isolating and sequencing a full-length clone from a cDNA library<a href="#pone.0000153-Finnerty4">[46]</a>, while the NVHD065 transcript was determined by conceptually splicing overlapping 5′ and 3′ RACE products. A 1,580,679 nucleotide scaffold encompassing this cluster is contained in the JGI assembly of the <em>Nematostella</em> genome (scaffold-27) <a href="#pone.0000153-2">[55]</a>. A TBLASTN analysis of the scaffold against the homeodomain dataset did not identify any additional homeodomains. The figure is labeled as <a href="#pone.0000153.s003">Figure S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.59 MB EPS)</p>
<a name="pone.0000153.s006" id="pone.0000153.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Annotated anthox1a cluster of <em>Nematostella</em>. Two Hox family genes were mapped to a single 98,121 nucleotide genomic scaffold in the Phusion assembly (c401800829.Contig2). The sequence of the anthox 9 transcript was determined by conceptually splicing overlapping 5′ and 3′ RACE fragments while the sequence of anthox1a was previously determined by isolating and sequencing a full-length clone from a cDNA library which was further extended using ligation mediated PCR. A 2,832,129 nucleotide scaffold (scaffold-03) was recovered from the JGI assembly that encompassed the Phusion scaffold <a href="#pone.0000153-2">[55]</a>. The anthox9 - anthox1a locus begins approximately 58,8435 nucleotides from the start of the scaffold. A subsequent TBLASTN analysis of the JGI_03 scaffold against the homeodomain database recovered three additional ANTP class genes. MsxlxA and MsxlxB were mapped approximately 343,185 bases downstream of anthox1a and are separated from each other by approximately 8945 nucleotides. An EST (DV089324) was recovered from the NCBI database that mapped to the MsxlxA homeodomain. An HLXc-like homeodomain was identifed approximately 1,809,900 nucleotides away from MsxlxB. The figure is labeled as <a href="#pone.0000153.s003">Figure S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.63 MB EPS)</p>
<a name="pone.0000153.s007" id="pone.0000153.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Annotated anthox1-containing genomic scaffold of <em>Nematostella</em>. Anthox1 was identified on a single 359,083 nucleotide scaffold in the Phusion assembly (c407500859.Contig1). A previously recovered 5′ RACE product that was further extended using ligation-mediated PCR was used to determine the sequence of anthox1. This RACE product was used to probe the <em>Nematostella</em> genomic library and a single genomic clone was identified and sequenced (AA8a). A 2,783,717 nucleotide scaffold (scaffold-04) was recovered from the JGI assembly that encompassed the Phusion scaffold <a href="#pone.0000153-2">[55]</a>. The anthox1 locus begins approximately 1,711,804 nucleotides from the start of the scaffold. A subsequent TBLASTN analysis of the JGI_04 scaffold against the homeodomain database recovered three additional ANTP class genes. Emxlx is located approximately 549,540 nucleotides, and HLXa approximately 306,200 nucleotides upstream of anthox1, while an ANTP class ambiguous family homeodomain is approximately 434,972 nucleotides downstream of anthox1. The figure is labeled as <a href="#pone.0000153.s003">Figure S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.61 MB EPS)</p>
<a name="pone.0000153.s008" id="pone.0000153.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s008">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Annotated anthox6a cluster of <em>Nematostella</em>. Anthox6a and HLXb were identified from a single genomic scaffold (c403601169.Contig1) from the Phusion assembly of the <em>Nematostella</em> genome. The sequence of anthox6a and HLXb was determined by conceptually splicing overlapping 5′ and 3′ RACE products. A 1,571,464 nucleotide scaffold encompassing this cluster is contained in the JGI assembly of the <em>Nematostella</em> genome (scaffold-26) <a href="#pone.0000153-2">[55]</a>. A TBLASTN analysis of the scaffold against the homeodomain dataset did not identify any additional homeodomains. The figure is labeled as <a href="#pone.0000153.s003">Figure S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.59 MB EPS)</p>
<a name="pone.0000153.s009" id="pone.0000153.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s009">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Annotated Gbx cluster of <em>Nematostella</em>. Gbx was localized to a single 117,312 nucleotide genomic scaffold (c402901116.Contig1) from the Phusion assembly of the <em>Nematostella</em> genome. The sequence of Gbx was determined by conceptually splicing overlapping 5′ and 3′ RACE products. A 1,068,422 nucleotide scaffold encompassing this cluster is contained in the JGI assembly of the <em>Nematostella</em> genome (scaffold-58) <a href="#pone.0000153-2">[55]</a>. A TBLASTN analysis of the scaffold against the homeodomain dataset identified an additional homeodomain, HLXc located approximately 601,580 nucleotides upstream of Gbx. The figure is labeled as <a href="#pone.0000153.s003">Figure S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.59 MB EPS)</p>
<a name="pone.0000153.s010" id="pone.0000153.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s010">Figure S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Annotated Dlx-NVHD021 cluster of <em>Nematostella</em>. Dlx and NVHD021 are located 6988 nucleotides apart on a single genomic scaffold (c413501012.Contig1) from the Phusion assembly of the <em>Nematostella</em> genome. The sequence of Dlx was was determined by conceptually splicing overlapping 5′ and 3′ RACE products, while the longest open reading frame downstream of NVHD021 homeodomain was inferred using MacVector 7.2.3 (Accelrys Inc.). A 1,601,021 nucleotide scaffold encompassing this cluster is contained in the JGI assembly of the <em>Nematostella</em> genome (scaffold-20) <a href="#pone.0000153-2">[55]</a>. A TBLASTN analysis of the scaffold against the homeodomain dataset did not identify any additional homeodomains. The figure is labeled as <a href="#pone.0000153.s003">Figure S3</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.58 MB EPS)</p>
<a name="pone.0000153.s011" id="pone.0000153.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s011">Figure S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Expression of MoxB, MoxC, and MoxD in <em>Nematostella</em>. <em>In situ</em> hybridization reveals the spatial expression of MoxA (A–D), MoxC (E–F), and MoxD (G–H) during larval development. Photos F and H depict optical cross-sections through the primary body axis, the oral-aboral axis, at the level of the developing pharynx. All other photos shown are optical longitudinal sections along the principal body axis. The blastopore, which will give rise to the mouth, is indicated by an asterisk. The division between ectoderm (ec) and endoderm (en) is clearly visible in all photos. At later stages (for example, panel D), body wall endoderm (enbw) can be distinguished from pharyngeal endoderm (enph). Tentacles (tn) emerge around the blastopore/mouth at the late larval stage (visible in C, D, and G). Prior to settlement at approximately 5–7 days, the first two mesenteries (mes) and pharynx (pha) have developed, and the gastrovascular cavity, or coelenteron (coe), is clearly visible (D).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(4.63 MB EPS)</p>
<a name="pone.0000153.s012" id="pone.0000153.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s012">Figure S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Alignment of homeodomains used in our phylogenetic analysis in phylip format.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.73 MB EPS)</p>
<a name="pone.0000153.s013" id="pone.0000153.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000153.s013">Figure S13. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Alignment of homeodomains used in the re-evaluation of Kamm et al. dataset in phylip format <a href="#pone.0000153-Kamm1">[49]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000153.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.45 MB EPS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are extremely grateful to the Joint Genome Institute (U. S. Department of Energy) for sequencing the genome of <em>Nematostella</em>, and to the researchers who carried out the project under the direction of principal investigator Daniel Rokhsar. The authors are equally grateful to the many researchers who published on <em>Nematostella</em> long before it entered the genomic age (for a list, see <a href="http://nematostella.org">http://nematostella.org</a> 
            <a href="#pone.0000153-VanAuken1">[125]</a>), particularly Cadet Hand and Kevin Uhlinger who introduced us to this species.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MM JF JR MM. Performed the experiments: JR MM KP DM. Analyzed the data: MM JF JR MM KP DM AB. Contributed reagents/materials/analysis tools: JR MM KP DM. Wrote the paper: MM JF JR AB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000153-Hunt1" id="pone.0000153-Hunt1"></a>Hunt P, Whiting J, Nonchev S, Sham MH, Marshall H, et al.  (1991) The branchial Hox code and its implications for gene regulation, patterning of the nervous system and head evolution. Development Suppl  2: 63–77.  <ul class="find" data-citedArticleID="925584"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+branchial+Hox+code+and+its+implications+for+gene+regulation%2C+patterning+of+the+nervous+system+and+head+evolution.&amp;auth=&amp;atitle=The+branchial+Hox+code+and+its+implications+for+gene+regulation%2C+patterning+of+the+nervous+system+and+head+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+branchial+Hox+code+and+its+implications+for+gene+regulation%2C+patterning+of+the+nervous+system+and+head+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+branchial+Hox+code+and+its+implications+for+gene+regulation%2C+patterning+of+the+nervous+system+and+head+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000153-Slack1" id="pone.0000153-Slack1"></a>Slack JM, Holland PW, Graham CF (1993) The zootype and the phylotypic stage. Nature  361: 490–492.  <ul class="find" data-citedArticleID="925704" data-doi="10.1038/361490a0"><li><a href="http://dx.doi.org/10.1038/361490a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+zootype+and+the+phylotypic+stage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+zootype+and+the+phylotypic+stage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000153-Burke1" id="pone.0000153-Burke1"></a>Burke AC, Nelson CE, Morgan BA, Tabin C (1995) Hox genes and the evolution of vertebrate axial morphology. Development  121: 333–346.  <ul class="find" data-citedArticleID="925510"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hox+genes+and+the+evolution+of+vertebrate+axial+morphology.&amp;auth=&amp;atitle=Hox+genes+and+the+evolution+of+vertebrate+axial+morphology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hox+genes+and+the+evolution+of+vertebrate+axial+morphology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hox+genes+and+the+evolution+of+vertebrate+axial+morphology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000153-Averof1" id="pone.0000153-Averof1"></a>Averof M, Akam M (1995) Hox genes and the diversification of insect and crustacean body plans. Nature  376: 420–423.  <ul class="find" data-citedArticleID="925494" data-doi="10.1038/376420a0"><li><a href="http://dx.doi.org/10.1038/376420a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hox+genes+and+the+diversification+of+insect+and+crustacean+body+plans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hox+genes+and+the+diversification+of+insect+and+crustacean+body+plans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000153-Averof2" id="pone.0000153-Averof2"></a>Averof M, Patel NH (1997) Crustacean appendage evolution associated with changes in Hox gene expression. Nature  388: 682–686.  <ul class="find" data-citedArticleID="925496" data-doi="10.1038/41786"><li><a href="http://dx.doi.org/10.1038/41786" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crustacean+appendage+evolution+associated+with+changes+in+Hox+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crustacean+appendage+evolution+associated+with+changes+in+Hox+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000153-Hughes1" id="pone.0000153-Hughes1"></a>Hughes CL, Kaufman TC (2002) Exploring the myriapod body plan: expression patterns of the ten Hox genes in a centipede. Development  129: 1225–1238.  <ul class="find" data-citedArticleID="925580"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Exploring+the+myriapod+body+plan%3A+expression+patterns+of+the+ten+Hox+genes+in+a+centipede.&amp;auth=&amp;atitle=Exploring+the+myriapod+body+plan%3A+expression+patterns+of+the+ten+Hox+genes+in+a+centipede." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+myriapod+body+plan%3A+expression+patterns+of+the+ten+Hox+genes+in+a+centipede." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+myriapod+body+plan%3A+expression+patterns+of+the+ten+Hox+genes+in+a+centipede.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000153-Lichtneckert1" id="pone.0000153-Lichtneckert1"></a>Lichtneckert R, Reichert H (2005) Insights into the urbilaterian brain: conserved genetic patterning mechanisms in insect and vertebrate brain development. Heredity  94: 465–477.  <ul class="find" data-citedArticleID="925620" data-doi="10.1038/sj.hdy.6800664"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800664" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+the+urbilaterian+brain%3A+conserved+genetic+patterning+mechanisms+in+insect+and+vertebrate+brain+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+the+urbilaterian+brain%3A+conserved+genetic+patterning+mechanisms+in+insect+and+vertebrate+brain+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000153-Graham1" id="pone.0000153-Graham1"></a>Graham A, Maden M, Krumlauf R (1991) The murine Hox-2 genes display dynamic dorsoventral patterns of expression during central nervous system development. Development  112: 255–264.  <ul class="find" data-citedArticleID="925570"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+murine+Hox-2+genes+display+dynamic+dorsoventral+patterns+of+expression+during+central+nervous+system+development.&amp;auth=&amp;atitle=The+murine+Hox-2+genes+display+dynamic+dorsoventral+patterns+of+expression+during+central+nervous+system+development." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+murine+Hox-2+genes+display+dynamic+dorsoventral+patterns+of+expression+during+central+nervous+system+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+murine+Hox-2+genes+display+dynamic+dorsoventral+patterns+of+expression+during+central+nervous+system+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000153-Kawazoe1" id="pone.0000153-Kawazoe1"></a>Kawazoe Y, Sekimoto T, Araki M, Takagi K, Araki K, et al.  (2002) Region-specific gastrointestinal Hox code during murine embryonal gut development. Dev Growth Differ  44: 77–84.  <ul class="find" data-citedArticleID="925598" data-doi="10.1046/j.1440-169x.2002.00623.x"><li><a href="http://dx.doi.org/10.1046/j.1440-169x.2002.00623.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Region-specific+gastrointestinal+Hox+code+during+murine+embryonal+gut+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Region-specific+gastrointestinal+Hox+code+during+murine+embryonal+gut+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000153-Sakiyama1" id="pone.0000153-Sakiyama1"></a>Sakiyama J, Yokouchi Y, Kuroiwa A (2001) HoxA and HoxB cluster genes subdivide the digestive tract into morphological domains during chick development. Mech Dev  101: 233–236.  <ul class="find" data-citedArticleID="925688" data-doi="10.1016/s0925-4773(00)00564-5"><li><a href="http://dx.doi.org/10.1016/s0925-4773(00)00564-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HoxA+and+HoxB+cluster+genes+subdivide+the+digestive+tract+into+morphological+domains+during+chick+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HoxA+and+HoxB+cluster+genes+subdivide+the+digestive+tract+into+morphological+domains+during+chick+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000153-Powers1" id="pone.0000153-Powers1"></a>Powers TP, Hogan J, Ke Z, Dymbrowski K, Wang X, et al.  (2000) Characterization of the Hox cluster from the mosquito Anopheles gambiae (Diptera: Culicidae). Evol Dev  2: 311–325.  <ul class="find" data-citedArticleID="925670" data-doi="10.1046/j.1525-142x.2000.00072.x"><li><a href="http://dx.doi.org/10.1046/j.1525-142x.2000.00072.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+Hox+cluster+from+the+mosquito+Anopheles+gambiae+%28Diptera%3A+Culicidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+Hox+cluster+from+the+mosquito+Anopheles+gambiae+%28Diptera%3A+Culicidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000153-Ferrier1" id="pone.0000153-Ferrier1"></a>Ferrier DE, Akam M (1996) Organization of the Hox gene cluster in the grasshopper, Schistocerca gregaria. Proc Natl Acad Sci U S A  93: 13024–13029.  <ul class="find" data-citedArticleID="925540" data-doi="10.1073/pnas.93.23.13024"><li><a href="http://dx.doi.org/10.1073/pnas.93.23.13024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organization+of+the+Hox+gene+cluster+in+the+grasshopper%2C+Schistocerca+gregaria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organization+of+the+Hox+gene+cluster+in+the+grasshopper%2C+Schistocerca+gregaria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000153-Devenport1" id="pone.0000153-Devenport1"></a>Devenport MP, Blass C, Eggleston P (2000) Characterization of the Hox gene cluster in the malaria vector mosquito, Anopheles gambiae. Evol Dev  2: 326–339.  <ul class="find" data-citedArticleID="925528" data-doi="10.1046/j.1525-142x.2000.00074.x"><li><a href="http://dx.doi.org/10.1046/j.1525-142x.2000.00074.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+Hox+gene+cluster+in+the+malaria+vector+mosquito%2C+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+Hox+gene+cluster+in+the+malaria+vector+mosquito%2C+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000153-Brown1" id="pone.0000153-Brown1"></a>Brown SJ, Fellers JP, Shippy TD, Richardson EA, Maxwell M, et al.  (2002) Sequence of the Tribolium castaneum homeotic complex: the region corresponding to the Drosophila melanogaster antennapedia complex. Genetics  160: 1067–1074.  <ul class="find" data-citedArticleID="925506"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence+of+the+Tribolium+castaneum+homeotic+complex%3A+the+region+corresponding+to+the+Drosophila+melanogaster+antennapedia+complex.&amp;auth=&amp;atitle=Sequence+of+the+Tribolium+castaneum+homeotic+complex%3A+the+region+corresponding+to+the+Drosophila+melanogaster+antennapedia+complex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+of+the+Tribolium+castaneum+homeotic+complex%3A+the+region+corresponding+to+the+Drosophila+melanogaster+antennapedia+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+of+the+Tribolium+castaneum+homeotic+complex%3A+the+region+corresponding+to+the+Drosophila+melanogaster+antennapedia+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000153-KmitaCunisse1" id="pone.0000153-KmitaCunisse1"></a>Kmita-Cunisse M, Loosli F, Bierne J, Gehring WJ (1998) Homeobox genes in the ribbonworm Lineus sanguineus: evolutionary implications. Proc Natl Acad Sci U S A  95: 3030–3035.  <ul class="find" data-citedArticleID="925602" data-doi="10.1073/pnas.95.6.3030"><li><a href="http://dx.doi.org/10.1073/pnas.95.6.3030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeobox+genes+in+the+ribbonworm+Lineus+sanguineus%3A+evolutionary+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeobox+genes+in+the+ribbonworm+Lineus+sanguineus%3A+evolutionary+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000153-GarciaFernandez1" id="pone.0000153-GarciaFernandez1"></a>Garcia-Fernandez J, Holland PW (1994) Archetypal organization of the amphioxus Hox gene cluster. Nature  370: 563–566.  <ul class="find" data-citedArticleID="925562" data-doi="10.1038/370563a0"><li><a href="http://dx.doi.org/10.1038/370563a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Archetypal+organization+of+the+amphioxus+Hox+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Archetypal+organization+of+the+amphioxus+Hox+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000153-Minguillon1" id="pone.0000153-Minguillon1"></a>Minguillon C, Gardenyes J, Serra E, Castro LF, Hill-Force A, et al.  (2005) No more than 14: the end of the amphioxus Hox cluster. Int J Biol Sci  1: 19–23.  <ul class="find" data-citedArticleID="925646" data-doi="10.7150/ijbs.1.19"><li><a href="http://dx.doi.org/10.7150/ijbs.1.19" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=No+more+than+14%3A+the+end+of+the+amphioxus+Hox+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22No+more+than+14%3A+the+end+of+the+amphioxus+Hox+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000153-Mann1" id="pone.0000153-Mann1"></a>Mann RS (1997) Why are Hox genes clustered? Bioessays  19: 661–664.  <ul class="find" data-citedArticleID="925628" data-doi="10.1002/bies.950190804"><li><a href="http://dx.doi.org/10.1002/bies.950190804" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+are+Hox+genes+clustered%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+are+Hox+genes+clustered%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000153-Ferrier2" id="pone.0000153-Ferrier2"></a>Ferrier DE, Holland PW (2002) Ciona intestinalis ParaHox genes: evolution of Hox/ParaHox cluster integrity, developmental mode, and temporal colinearity. Mol Phylogenet Evol  24: 412–417.  <ul class="find" data-citedArticleID="925542" data-doi="10.1016/s1055-7903(02)00204-x"><li><a href="http://dx.doi.org/10.1016/s1055-7903(02)00204-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ciona+intestinalis+ParaHox+genes%3A+evolution+of+Hox%2FParaHox+cluster+integrity%2C+developmental+mode%2C+and+temporal+colinearity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ciona+intestinalis+ParaHox+genes%3A+evolution+of+Hox%2FParaHox+cluster+integrity%2C+developmental+mode%2C+and+temporal+colinearity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000153-Negre1" id="pone.0000153-Negre1"></a>Negre B, Casillas S, Suzanne M, Sanchez-Herrero E, Akam M, et al.  (2005) Conservation of regulatory sequences and gene expression patterns in the disintegrating Drosophila Hox gene complex. Genome Res  15: 692–700.  <ul class="find" data-citedArticleID="925658" data-doi="10.1101/gr.3468605"><li><a href="http://dx.doi.org/10.1101/gr.3468605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+of+regulatory+sequences+and+gene+expression+patterns+in+the+disintegrating+Drosophila+Hox+gene+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+of+regulatory+sequences+and+gene+expression+patterns+in+the+disintegrating+Drosophila+Hox+gene+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000153-Pierce1" id="pone.0000153-Pierce1"></a>Pierce RJ, Wu W, Hirai H, Ivens A, Murphy LD, et al.  (2005) Evidence for a dispersed Hox gene cluster in the platyhelminth parasite Schistosoma mansoni. Mol Biol Evol  22: 2491–2503.  <ul class="find" data-citedArticleID="925666" data-doi="10.1093/molbev/msi239"><li><a href="http://dx.doi.org/10.1093/molbev/msi239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+a+dispersed+Hox+gene+cluster+in+the+platyhelminth+parasite+Schistosoma+mansoni." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+a+dispersed+Hox+gene+cluster+in+the+platyhelminth+parasite+Schistosoma+mansoni.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000153-Edvardsen1" id="pone.0000153-Edvardsen1"></a>Edvardsen RB, Seo HC, Jensen MF, Mialon A, Mikhaleva J, et al.  (2005) Remodelling of the homeobox gene complement in the tunicate Oikopleura dioica. Curr Biol  15: R12–13.  <ul class="find" data-citedArticleID="925532" data-doi="10.1016/j.cub.2004.12.010"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.12.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Remodelling+of+the+homeobox+gene+complement+in+the+tunicate+Oikopleura+dioica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Remodelling+of+the+homeobox+gene+complement+in+the+tunicate+Oikopleura+dioica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000153-Cameron1" id="pone.0000153-Cameron1"></a>Cameron RA, Rowen L, Nesbitt R, Bloom S, Rast JP, et al.  (2006) Unusual gene order and organization of the sea urchin hox cluster. J Exp Zoolog B Mol Dev Evol  306: 45–58.  <ul class="find" data-citedArticleID="925512" data-doi="10.1002/jez.b.21070"><li><a href="http://dx.doi.org/10.1002/jez.b.21070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unusual+gene+order+and+organization+of+the+sea+urchin+hox+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unusual+gene+order+and+organization+of+the+sea+urchin+hox+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000153-Negre2" id="pone.0000153-Negre2"></a>Negre B, Ranz JM, Casals F, Caceres M, Ruiz A (2003) A new split of the Hox gene complex in Drosophila: relocation and evolution of the gene labial. Mol Biol Evol  20: 2042–2054.  <ul class="find" data-citedArticleID="925660" data-doi="10.1093/molbev/msg238"><li><a href="http://dx.doi.org/10.1093/molbev/msg238" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+split+of+the+Hox+gene+complex+in+Drosophila%3A+relocation+and+evolution+of+the+gene+labial." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+split+of+the+Hox+gene+complex+in+Drosophila%3A+relocation+and+evolution+of+the+gene+labial.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000153-Struhl1" id="pone.0000153-Struhl1"></a>Struhl G (1984) Splitting the bithorax complex of Drosophila. Nature  308: 454–456.  <ul class="find" data-citedArticleID="925716" data-doi="10.1038/308454a0"><li><a href="http://dx.doi.org/10.1038/308454a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Splitting+the+bithorax+complex+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Splitting+the+bithorax+complex+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000153-VonAllmen1" id="pone.0000153-VonAllmen1"></a>VonAllmen G, Hogga I, Spierer A, Karch F, Bender W, et al.  (1996) Splits in fruitfly hox gene complexes [letter]. Nature  380: 116.  <ul class="find" data-citedArticleID="925732" data-doi="10.1038/380116a0"><li><a href="http://dx.doi.org/10.1038/380116a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Splits+in+fruitfly+hox+gene+complexes+%5Bletter%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Splits+in+fruitfly+hox+gene+complexes+%5Bletter%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000153-Randazzo1" id="pone.0000153-Randazzo1"></a>Randazzo FM, Seeger MA, Huss CA, Sweeney MA, Cecil JK, et al.  (1993) Structural changes in the antennapedia complex of Drosophila pseudoobscura. Genetics  134: 319–330.  <ul class="find" data-citedArticleID="925674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Structural+changes+in+the+antennapedia+complex+of+Drosophila+pseudoobscura.&amp;auth=&amp;atitle=Structural+changes+in+the+antennapedia+complex+of+Drosophila+pseudoobscura." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+changes+in+the+antennapedia+complex+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+changes+in+the+antennapedia+complex+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000153-Boncinelli1" id="pone.0000153-Boncinelli1"></a>Boncinelli E, Simeone A, Acampora D, Gulisano M (1993) Homeobox genes in the developing central nervous system. Ann Genet  36: 30–37.  <ul class="find" data-citedArticleID="925498"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Homeobox+genes+in+the+developing+central+nervous+system.&amp;auth=&amp;atitle=Homeobox+genes+in+the+developing+central+nervous+system." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeobox+genes+in+the+developing+central+nervous+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeobox+genes+in+the+developing+central+nervous+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000153-Monteiro1" id="pone.0000153-Monteiro1"></a>Monteiro AS, Ferrier DE (2006) Hox genes are not always Colinear. Int J Biol Sci  2: 95–103.  <ul class="find" data-citedArticleID="925650" data-doi="10.7150/ijbs.2.95"><li><a href="http://dx.doi.org/10.7150/ijbs.2.95" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hox+genes+are+not+always+Colinear." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hox+genes+are+not+always+Colinear.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000153-Kmita1" id="pone.0000153-Kmita1"></a>Kmita M, Duboule D (2003) Organizing axes in time and space; 25 years of colinear tinkering. Science  301: 331–333.  <ul class="find" data-citedArticleID="925600" data-doi="10.1126/science.1085753"><li><a href="http://dx.doi.org/10.1126/science.1085753" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organizing+axes+in+time+and+space%3B+25+years+of+colinear+tinkering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organizing+axes+in+time+and+space%3B+25+years+of+colinear+tinkering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000153-Chambeyron1" id="pone.0000153-Chambeyron1"></a>Chambeyron S, Bickmore WA (2004) Chromatin decondensation and nuclear reorganization of the HoxB locus upon induction of transcription. Genes Dev  18: 1119–1130.  <ul class="find" data-citedArticleID="925514" data-doi="10.1101/gad.292104"><li><a href="http://dx.doi.org/10.1101/gad.292104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+decondensation+and+nuclear+reorganization+of+the+HoxB+locus+upon+induction+of+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+decondensation+and+nuclear+reorganization+of+the+HoxB+locus+upon+induction+of+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000153-Spitz1" id="pone.0000153-Spitz1"></a>Spitz F, Herkenne C, Morris MA, Duboule D (2005) Inversion-induced disruption of the Hoxd cluster leads to the partition of regulatory landscapes. Nat Genet  37: 889–893.  <ul class="find" data-citedArticleID="925706" data-doi="10.1038/ng1597"><li><a href="http://dx.doi.org/10.1038/ng1597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inversion-induced+disruption+of+the+Hoxd+cluster+leads+to+the+partition+of+regulatory+landscapes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inversion-induced+disruption+of+the+Hoxd+cluster+leads+to+the+partition+of+regulatory+landscapes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000153-Tiong1" id="pone.0000153-Tiong1"></a>Tiong SY, Whittle JR, Gribbin MC (1987) Chromosomal continuity in the abdominal region of the bithorax complex of Drosophila is not essential for its contribution to metameric identity. Development  101: 135–142.  <ul class="find" data-citedArticleID="925726"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chromosomal+continuity+in+the+abdominal+region+of+the+bithorax+complex+of+Drosophila+is+not+essential+for+its+contribution+to+metameric+identity.&amp;auth=&amp;atitle=Chromosomal+continuity+in+the+abdominal+region+of+the+bithorax+complex+of+Drosophila+is+not+essential+for+its+contribution+to+metameric+identity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+continuity+in+the+abdominal+region+of+the+bithorax+complex+of+Drosophila+is+not+essential+for+its+contribution+to+metameric+identity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+continuity+in+the+abdominal+region+of+the+bithorax+complex+of+Drosophila+is+not+essential+for+its+contribution+to+metameric+identity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000153-Galant1" id="pone.0000153-Galant1"></a>Galant R, Carroll SB (2002) Evolution of a transcriptional repression domain in an insect Hox protein. Nature  415: 910–913.  <ul class="find" data-citedArticleID="925558" data-doi="10.1038/nature717"><li><a href="http://dx.doi.org/10.1038/nature717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+a+transcriptional+repression+domain+in+an+insect+Hox+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+a+transcriptional+repression+domain+in+an+insect+Hox+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000153-Weatherbee1" id="pone.0000153-Weatherbee1"></a>Weatherbee SD, Halder G, Kim J, Hudson A, Carroll S (1998) Ultrabithorax regulates genes at several levels of the wing-patterning hierarchy to shape the development of the Drosophila haltere. Genes Dev  12: 1474–1482.  <ul class="find" data-citedArticleID="925734" data-doi="10.1101/gad.12.10.1474"><li><a href="http://dx.doi.org/10.1101/gad.12.10.1474" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrabithorax+regulates+genes+at+several+levels+of+the+wing-patterning+hierarchy+to+shape+the+development+of+the+Drosophila+haltere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrabithorax+regulates+genes+at+several+levels+of+the+wing-patterning+hierarchy+to+shape+the+development+of+the+Drosophila+haltere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000153-Finnerty1" id="pone.0000153-Finnerty1"></a>Finnerty JR, Pang K, Burton P, Paulson D, Martindale MQ (2004) Origins of bilateral symmetry: Hox and dpp expression in a sea anemone. Science  304: 1335–1337.  <ul class="find" data-citedArticleID="925544" data-doi="10.1126/science.1091946"><li><a href="http://dx.doi.org/10.1126/science.1091946" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origins+of+bilateral+symmetry%3A+Hox+and+dpp+expression+in+a+sea+anemone." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origins+of+bilateral+symmetry%3A+Hox+and+dpp+expression+in+a+sea+anemone.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000153-Brglin1" id="pone.0000153-Brglin1"></a>Bürglin TR (2005)  Homeodomain Proteins. In: Meyers RA, editor. Encyclopedia or Molecular Cell Biology and Molecular Medicine. Weinheim: Wiley-VCH Verlag.   2nd ed. <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0000153-Harvey1" id="pone.0000153-Harvey1"></a>Harvey RP (1996) NK-2 homeobox genes and heart development. Dev Biol  178: 203–216.  <ul class="find" data-citedArticleID="925574" data-doi="10.1006/dbio.1996.0212"><li><a href="http://dx.doi.org/10.1006/dbio.1996.0212" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NK-2+homeobox+genes+and+heart+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NK-2+homeobox+genes+and+heart+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000153-Kozmik1" id="pone.0000153-Kozmik1"></a>Kozmik Z (2005) Pax genes in eye development and evolution. Curr Opin Genet Dev  15: 430–438.  <ul class="find" data-citedArticleID="925610" data-doi="10.1016/j.gde.2005.05.001"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.05.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pax+genes+in+eye+development+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pax+genes+in+eye+development+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000153-Valentine1" id="pone.0000153-Valentine1"></a>Valentine JW, Jablonski D (2003) Morphological and developmental macroevolution: a paleontological perspective. Int J Dev Biol  47: 517–522.  <ul class="find" data-citedArticleID="925728"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Morphological+and+developmental+macroevolution%3A+a+paleontological+perspective.&amp;auth=&amp;atitle=Morphological+and+developmental+macroevolution%3A+a+paleontological+perspective." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morphological+and+developmental+macroevolution%3A+a+paleontological+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Morphological+and+developmental+macroevolution%3A+a+paleontological+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000153-GarciaFernandez2" id="pone.0000153-GarciaFernandez2"></a>Garcia-Fernandez J (2005) Hox, ParaHox, ProtoHox: facts and guesses. Heredity  94: 145–152.  <ul class="find" data-citedArticleID="925564" data-doi="10.1038/sj.hdy.6800621"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800621" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hox%2C+ParaHox%2C+ProtoHox%3A+facts+and+guesses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hox%2C+ParaHox%2C+ProtoHox%3A+facts+and+guesses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000153-Medina1" id="pone.0000153-Medina1"></a>Medina M, Collins AG, Silberman JD, Sogin ML (2001) Evaluating hypotheses of basal animal phylogeny using complete sequences of large and small subunit rRNA. Proc Natl Acad Sci U S A  98: 9707–9712.  <ul class="find" data-citedArticleID="925644" data-doi="10.1073/pnas.171316998"><li><a href="http://dx.doi.org/10.1073/pnas.171316998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+hypotheses+of+basal+animal+phylogeny+using+complete+sequences+of+large+and+small+subunit+rRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+hypotheses+of+basal+animal+phylogeny+using+complete+sequences+of+large+and+small+subunit+rRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000153-Collins1" id="pone.0000153-Collins1"></a>Collins AG (1998) Evaluating multiple alternative hypotheses for the origin of Bilateria: an analysis of 18S rRNA molecular evidence. Proc Natl Acad Sci USA  95: 15458–15463.  <ul class="find" data-citedArticleID="925518" data-doi="10.1073/pnas.95.26.15458"><li><a href="http://dx.doi.org/10.1073/pnas.95.26.15458" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+multiple+alternative+hypotheses+for+the+origin+of+Bilateria%3A+an+analysis+of+18S+rRNA+molecular+evidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+multiple+alternative+hypotheses+for+the+origin+of+Bilateria%3A+an+analysis+of+18S+rRNA+molecular+evidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000153-Finnerty2" id="pone.0000153-Finnerty2"></a>Finnerty JR, Martindale MQ (1997) Homeoboxes in sea anemones (Cnidaria:Anthozoa): a PCR-based survey of Nematostella vectensis and Metridium senile. Biol Bull  193: 62–76.  <ul class="find" data-citedArticleID="925546" data-doi="10.2307/1542736"><li><a href="http://dx.doi.org/10.2307/1542736" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeoboxes+in+sea+anemones+%28Cnidaria%3AAnthozoa%29%3A+a+PCR-based+survey+of+Nematostella+vectensis+and+Metridium+senile." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeoboxes+in+sea+anemones+%28Cnidaria%3AAnthozoa%29%3A+a+PCR-based+survey+of+Nematostella+vectensis+and+Metridium+senile.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000153-Finnerty3" id="pone.0000153-Finnerty3"></a>Finnerty JR (1998) Homeoboxes in sea anemones and other nonbilaterian animals: implications for the evolution of the Hox cluster and the zootype. Curr Top Dev Biol  40: 211–254.  <ul class="find" data-citedArticleID="925548" data-doi="10.1016/s0070-2153(08)60368-3"><li><a href="http://dx.doi.org/10.1016/s0070-2153(08)60368-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeoboxes+in+sea+anemones+and+other+nonbilaterian+animals%3A+implications+for+the+evolution+of+the+Hox+cluster+and+the+zootype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeoboxes+in+sea+anemones+and+other+nonbilaterian+animals%3A+implications+for+the+evolution+of+the+Hox+cluster+and+the+zootype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000153-Finnerty4" id="pone.0000153-Finnerty4"></a>Finnerty JR, Martindale MQ (1999) Ancient origins of axial patterning genes: Hox genes and ParaHox genes in the Cnidaria. Evol Dev  1: 16–23.  <ul class="find" data-citedArticleID="925550" data-doi="10.1046/j.1525-142x.1999.99010.x"><li><a href="http://dx.doi.org/10.1046/j.1525-142x.1999.99010.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+origins+of+axial+patterning+genes%3A+Hox+genes+and+ParaHox+genes+in+the+Cnidaria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+origins+of+axial+patterning+genes%3A+Hox+genes+and+ParaHox+genes+in+the+Cnidaria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000153-Kourakis1" id="pone.0000153-Kourakis1"></a>Kourakis MJ, Martindale MQ (2000) Combined-method phylogenetic analysis of Hox and ParaHox genes of the metazoa. J Exp Zool  288: 175–191.  <ul class="find" data-citedArticleID="925606" data-doi="10.1002/1097-010x(20000815)288:2<175::aid-jez8&gt;3.0.co;2-n"><li><a href="http://dx.doi.org/10.1002/1097-010x(20000815)288:2<175::aid-jez8&gt;3.0.co;2-n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combined-method+phylogenetic+analysis+of+Hox+and+ParaHox+genes+of+the+metazoa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combined-method+phylogenetic+analysis+of+Hox+and+ParaHox+genes+of+the+metazoa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000153-Martinez1" id="pone.0000153-Martinez1"></a>Martinez DE, Bridge D, Masuda-Nakagawa LM, Cartwright P (1998) Cnidarian homeoboxes and the zootype. Nature  393: 748–749.  <ul class="find" data-citedArticleID="925634" data-doi="10.1038/31616"><li><a href="http://dx.doi.org/10.1038/31616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cnidarian+homeoboxes+and+the+zootype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cnidarian+homeoboxes+and+the+zootype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000153-Kamm1" id="pone.0000153-Kamm1"></a>Kamm K, Schierwater B, Jakob W, Dellaporta SL, Miller DJ (2006) Axial patterning and diversification in the cnidaria predate the Hox system. Curr Biol  16: 920–926.  <ul class="find" data-citedArticleID="925592" data-doi="10.1016/j.cub.2006.03.036"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.03.036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Axial+patterning+and+diversification+in+the+cnidaria+predate+the+Hox+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Axial+patterning+and+diversification+in+the+cnidaria+predate+the+Hox+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000153-Kamm2" id="pone.0000153-Kamm2"></a>Kamm K, Schierwater B (2006) Ancient complexity of the non-Hox ANTP gene complement in the anthozoan Nematostella vectensis. Implications for the evolution of the ANTP superclass. J Exp Zoolog B Mol Dev Evol.   <ul class="find" data-citedArticleID="925594" data-doi="10.1002/jez.b.21123"><li><a href="http://dx.doi.org/10.1002/jez.b.21123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+complexity+of+the+non-Hox+ANTP+gene+complement+in+the+anthozoan+Nematostella+vectensis.+Implications+for+the+evolution+of+the+ANTP+superclass." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+complexity+of+the+non-Hox+ANTP+gene+complement+in+the+anthozoan+Nematostella+vectensis.+Implications+for+the+evolution+of+the+ANTP+superclass.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000153-Chourrout1" id="pone.0000153-Chourrout1"></a>Chourrout D, Delsuc F, Chourrout P, Edvardsen RB, Rentzsch F, et al.  (2006) Minimal ProtoHox cluster inferred from bilaterian and cnidarian Hox complements. Nature  442: 684–687.  <ul class="find" data-citedArticleID="925516" data-doi="10.1038/nature04863"><li><a href="http://dx.doi.org/10.1038/nature04863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Minimal+ProtoHox+cluster+inferred+from+bilaterian+and+cnidarian+Hox+complements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Minimal+ProtoHox+cluster+inferred+from+bilaterian+and+cnidarian+Hox+complements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000153-Ryan1" id="pone.0000153-Ryan1"></a>Ryan JF, Burton PM, Mazza M, Kwong GK, Mullikin JC, et al.  (2006) The cnidarian-bilaterian ancestor possessed at least 56 homeoboxes. Evidence from the starlet sea anemone, <em>Nematostella vectensis</em>. Genome Biology In press.   <ul class="find" data-citedArticleID="925684"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+cnidarian-bilaterian+ancestor+possessed+at+least+56+homeoboxes.+Evidence+from+the+starlet+sea+anemone%2C+Nematostella+vectensis.&amp;auth=&amp;atitle=The+cnidarian-bilaterian+ancestor+possessed+at+least+56+homeoboxes.+Evidence+from+the+starlet+sea+anemone%2C+Nematostella+vectensis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cnidarian-bilaterian+ancestor+possessed+at+least+56+homeoboxes.+Evidence+from+the+starlet+sea+anemone%2C+Nematostella+vectensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cnidarian-bilaterian+ancestor+possessed+at+least+56+homeoboxes.+Evidence+from+the+starlet+sea+anemone%2C+Nematostella+vectensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000153-Sullivan1" id="pone.0000153-Sullivan1"></a>Sullivan JC, Ryan JF, Watson JA, Webb J, Mullikin JC, et al.  (2006) StellaBase: the Nematostella vectensis Genomics Database. Nucleic Acids Res  34: D495–499.  <ul class="find" data-citedArticleID="925718" data-doi="10.1093/nar/gkj020"><li><a href="http://dx.doi.org/10.1093/nar/gkj020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=StellaBase%3A+the+Nematostella+vectensis+Genomics+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22StellaBase%3A+the+Nematostella+vectensis+Genomics+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000153-1" id="pone.0000153-1"></a>StellaBase—The Nematostella vectensis Genomics Database.   <ul class="find-nolinks"></ul></li><li><span class="label">55.
              </span><a name="pone.0000153-2" id="pone.0000153-2"></a>Joint Genome Institute—Nematostella vectensis genome portal, v1.0.   <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0000153-Pruitt1" id="pone.0000153-Pruitt1"></a>Pruitt KD, Tatusova T, Maglott DR (2005) NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res  33: D501–504.  <ul class="find" data-citedArticleID="925672" data-doi="10.1093/nar/gkl842"><li><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000153-Matus1" id="pone.0000153-Matus1"></a>Matus DQ, Thomsen GH, Martindale MQ (2006) Dorso/ventral genes are asymmetrically expressed and involved in germ-layer demarcation during cnidarian gastrulation. Curr Biol  16: 499–505.  <ul class="find" data-citedArticleID="925638" data-doi="10.1016/j.cub.2006.01.052"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.01.052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dorso%2Fventral+genes+are+asymmetrically+expressed+and+involved+in+germ-layer+demarcation+during+cnidarian+gastrulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dorso%2Fventral+genes+are+asymmetrically+expressed+and+involved+in+germ-layer+demarcation+during+cnidarian+gastrulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000153-Stephenson1" id="pone.0000153-Stephenson1"></a>Stephenson TA (1928) British Sea Anemones. London: The Ray Society..   <ul class="find-nolinks"></ul></li><li><span class="label">59.
              </span><a name="pone.0000153-Matus2" id="pone.0000153-Matus2"></a>Matus DQ, Pang K, Marlow H, Dunn CW, Thomsen GH, et al.  (2006) Molecular evidence for deep evolutionary roots of bilaterality in animal development. Proc Natl Acad Sci U S A  103: 11195–11200.  <ul class="find" data-citedArticleID="925640" data-doi="10.1073/pnas.0601257103"><li><a href="http://dx.doi.org/10.1073/pnas.0601257103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+for+deep+evolutionary+roots+of+bilaterality+in+animal+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+for+deep+evolutionary+roots+of+bilaterality+in+animal+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000153-Hayward1" id="pone.0000153-Hayward1"></a>Hayward DC, Samuel G, Pontynen PC, Catmull J, Saint R, et al.  (2002) Localized expression of a dpp/BMP2/4 ortholog in a coral embryo. Proc Natl Acad Sci U S A  99: 8106–8111.  <ul class="find" data-citedArticleID="925576" data-doi="10.1073/pnas.112021499"><li><a href="http://dx.doi.org/10.1073/pnas.112021499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localized+expression+of+a+dpp%2FBMP2%2F4+ortholog+in+a+coral+embryo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localized+expression+of+a+dpp%2FBMP2%2F4+ortholog+in+a+coral+embryo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000153-Rentzsch1" id="pone.0000153-Rentzsch1"></a>Rentzsch F, Anton R, Saina M, Hammerschmidt M, Holstein TW, et al.  (2006) Asymmetric expression of the BMP antagonists chordin and gremlin in the sea anemone Nematostella vectensis: Implications for the evolution of axial patterning. Dev Biol.   <ul class="find" data-citedArticleID="925676" data-doi="10.1016/j.ydbio.2006.06.003"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2006.06.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asymmetric+expression+of+the+BMP+antagonists+chordin+and+gremlin+in+the+sea+anemone+Nematostella+vectensis%3A+Implications+for+the+evolution+of+axial+patterning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asymmetric+expression+of+the+BMP+antagonists+chordin+and+gremlin+in+the+sea+anemone+Nematostella+vectensis%3A+Implications+for+the+evolution+of+axial+patterning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000153-Finnerty5" id="pone.0000153-Finnerty5"></a>Finnerty JR, Paulson D, Burton P, Pang K, Martindale MQ (2003) Early evolution of a homeobox gene: the parahox gene Gsx in the Cnidaria and the Bilateria. Evol Dev  5: 331–345.  <ul class="find" data-citedArticleID="925552" data-doi="10.1046/j.1525-142x.2003.03041.x"><li><a href="http://dx.doi.org/10.1046/j.1525-142x.2003.03041.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+evolution+of+a+homeobox+gene%3A+the+parahox+gene+Gsx+in+the+Cnidaria+and+the+Bilateria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+evolution+of+a+homeobox+gene%3A+the+parahox+gene+Gsx+in+the+Cnidaria+and+the+Bilateria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000153-Gauchat1" id="pone.0000153-Gauchat1"></a>Gauchat D, Mazet F, Berney C, Schummer M, Kreger S, et al.  (2000) Evolution of Antp-class genes and differential expression of Hydra Hox/paraHox genes in anterior patterning. Proc Natl Acad Sci U S A  97: 4493–4498.  <ul class="find" data-citedArticleID="925566" data-doi="10.1073/pnas.97.9.4493"><li><a href="http://dx.doi.org/10.1073/pnas.97.9.4493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+Antp-class+genes+and+differential+expression+of+Hydra+Hox%2FparaHox+genes+in+anterior+patterning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+Antp-class+genes+and+differential+expression+of+Hydra+Hox%2FparaHox+genes+in+anterior+patterning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000153-Galliot1" id="pone.0000153-Galliot1"></a>Galliot B (2000) Conserved and divergent genes in apex and axis development of cnidarians. Curr Opin Genet Dev  10: 629–637.  <ul class="find" data-citedArticleID="925560" data-doi="10.1016/s0959-437x(00)00141-6"><li><a href="http://dx.doi.org/10.1016/s0959-437x(00)00141-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+and+divergent+genes+in+apex+and+axis+development+of+cnidarians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+and+divergent+genes+in+apex+and+axis+development+of+cnidarians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000153-Hayward2" id="pone.0000153-Hayward2"></a>Hayward DC, Catmull J, Reece-Hoyes JS, Berghammer H, Dodd H, et al.  (2001) Gene structure and larval expression of cnox-2Am from the coral Acropora millepora. Dev Genes Evol  211: 10–19.  <ul class="find" data-citedArticleID="925578" data-doi="10.1007/s004270000112"><li><a href="http://dx.doi.org/10.1007/s004270000112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+structure+and+larval+expression+of+cnox-2Am+from+the+coral+Acropora+millepora." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+structure+and+larval+expression+of+cnox-2Am+from+the+coral+Acropora+millepora.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000153-Kuhn1" id="pone.0000153-Kuhn1"></a>Kuhn K, Streit B, Schierwater B (1996) Homeobox genes in the cnidarian Eleutheria dichotoma: evolutionary implications for the origin of Antennapedia-class (HOM/Hox) genes. Mol Phylogenet Evol  6: 30–38.  <ul class="find" data-citedArticleID="925612"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Homeobox+genes+in+the+cnidarian+Eleutheria+dichotoma%3A+evolutionary+implications+for+the+origin+of+Antennapedia-class+%28HOM%2FHox%29+genes.&amp;auth=&amp;atitle=Homeobox+genes+in+the+cnidarian+Eleutheria+dichotoma%3A+evolutionary+implications+for+the+origin+of+Antennapedia-class+%28HOM%2FHox%29+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeobox+genes+in+the+cnidarian+Eleutheria+dichotoma%3A+evolutionary+implications+for+the+origin+of+Antennapedia-class+%28HOM%2FHox%29+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeobox+genes+in+the+cnidarian+Eleutheria+dichotoma%3A+evolutionary+implications+for+the+origin+of+Antennapedia-class+%28HOM%2FHox%29+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000153-Kuhn2" id="pone.0000153-Kuhn2"></a>Kuhn K, Streit B, Schierwater B (1999) Isolation of Hox genes from the scyphozoan Cassiopeia xamachana: implications for the early evolution of Hox genes. J Exp Zool  285: 63–75.  <ul class="find" data-citedArticleID="925614" data-doi="10.1002/(sici)1097-010x(19990415)285:1<63::aid-jez8&gt;3.0.co;2-x"><li><a href="http://dx.doi.org/10.1002/(sici)1097-010x(19990415)285:1<63::aid-jez8&gt;3.0.co;2-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+Hox+genes+from+the+scyphozoan+Cassiopeia+xamachana%3A+implications+for+the+early+evolution+of+Hox+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+Hox+genes+from+the+scyphozoan+Cassiopeia+xamachana%3A+implications+for+the+early+evolution+of+Hox+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000153-MasudaNakagawa1" id="pone.0000153-MasudaNakagawa1"></a>Masuda-Nakagawa LM, Groer H, Aerne BL, Schmid V (2000) The HOX-like gene Cnox2-Pc is expressed at the anterior region in all life cycle stages of the jellyfish Podocoryne carnea. Dev Genes Evol  210: 151–156.  <ul class="find" data-citedArticleID="925636" data-doi="10.1007/s004270050022"><li><a href="http://dx.doi.org/10.1007/s004270050022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+HOX-like+gene+Cnox2-Pc+is+expressed+at+the+anterior+region+in+all+life+cycle+stages+of+the+jellyfish+Podocoryne+carnea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+HOX-like+gene+Cnox2-Pc+is+expressed+at+the+anterior+region+in+all+life+cycle+stages+of+the+jellyfish+Podocoryne+carnea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000153-Naito1" id="pone.0000153-Naito1"></a>Naito M, Ishiguro H, Fujisawa T, Kurosawa Y (1993) Presence of eight distinct homeobox-containing genes in cnidarians. FEBS Lett  333: 271–274.  <ul class="find" data-citedArticleID="925656" data-doi="10.1016/0014-5793(93)80668-k"><li><a href="http://dx.doi.org/10.1016/0014-5793(93)80668-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Presence+of+eight+distinct+homeobox-containing+genes+in+cnidarians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Presence+of+eight+distinct+homeobox-containing+genes+in+cnidarians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000153-Murtha1" id="pone.0000153-Murtha1"></a>Murtha MT, Leckman JF, Ruddle FH (1991) Detection of homeobox genes in development and evolution. Proc Natl Acad Sci U S A  88: 10711–10715.  <ul class="find" data-citedArticleID="925654" data-doi="10.1073/pnas.88.23.10711"><li><a href="http://dx.doi.org/10.1073/pnas.88.23.10711" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+homeobox+genes+in+development+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+homeobox+genes+in+development+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000153-Schierwater1" id="pone.0000153-Schierwater1"></a>Schierwater B, Murtha M, Dick M, Ruddle FH, Buss LW (1991) Homeoboxes in cnidarians. J Exp Zool  260: 413–416.  <ul class="find" data-citedArticleID="925690" data-doi="10.1002/jez.1402600316"><li><a href="http://dx.doi.org/10.1002/jez.1402600316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeoboxes+in+cnidarians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeoboxes+in+cnidarians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000153-Schummer1" id="pone.0000153-Schummer1"></a>Schummer M, Scheurlen I, Schaller C, Galliot B (1992) HOM/HOX homeobox genes are present in hydra (Chlorohydra viridissima) and are differentially expressed during regeneration. Embo J  11: 1815–1823.  <ul class="find" data-citedArticleID="925694" data-doi="10.1016/0168-9525(92)90382-e"><li><a href="http://dx.doi.org/10.1016/0168-9525(92)90382-e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HOM%2FHOX+homeobox+genes+are+present+in+hydra+%28Chlorohydra+viridissima%29+and+are+differentially+expressed+during+regeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HOM%2FHOX+homeobox+genes+are+present+in+hydra+%28Chlorohydra+viridissima%29+and+are+differentially+expressed+during+regeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000153-Shenk1" id="pone.0000153-Shenk1"></a>Shenk MA, Bode HR, Steele RE (1993) Expression of Cnox-2, a HOM/HOX homeobox gene in hydra, is correlated with axial pattern formation. Development  117: 657–667.  <ul class="find" data-citedArticleID="925698" data-doi="10.1006/dbio.1993.1290"><li><a href="http://dx.doi.org/10.1006/dbio.1993.1290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+Cnox-2%2C+a+HOM%2FHOX+homeobox+gene+in+hydra%2C+is+correlated+with+axial+pattern+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+Cnox-2%2C+a+HOM%2FHOX+homeobox+gene+in+hydra%2C+is+correlated+with+axial+pattern+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000153-Shenk2" id="pone.0000153-Shenk2"></a>Shenk MA, Gee L, Steele RE, Bode HR (1993) Expression of Cnox-2, a HOM/HOX gene, is suppressed during head formation in hydra. Dev Biol  160: 108–118.  <ul class="find" data-citedArticleID="925700" data-doi="10.1006/dbio.1993.1290"><li><a href="http://dx.doi.org/10.1006/dbio.1993.1290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+Cnox-2%2C+a+HOM%2FHOX+gene%2C+is+suppressed+during+head+formation+in+hydra." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+Cnox-2%2C+a+HOM%2FHOX+gene%2C+is+suppressed+during+head+formation+in+hydra.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000153-Aerne1" id="pone.0000153-Aerne1"></a>Aerne BL, Baader CD, Schmid V (1995) Life stage and tissue-specific expression of the homeobox gene cnox1-Pc of the hydrozoan Podocoryne carnea. Dev Biol  169: 547–556.  <ul class="find" data-citedArticleID="925490" data-doi="10.1006/dbio.1995.1168"><li><a href="http://dx.doi.org/10.1006/dbio.1995.1168" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life+stage+and+tissue-specific+expression+of+the+homeobox+gene+cnox1-Pc+of+the+hydrozoan+Podocoryne+carnea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life+stage+and+tissue-specific+expression+of+the+homeobox+gene+cnox1-Pc+of+the+hydrozoan+Podocoryne+carnea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000153-Brooke1" id="pone.0000153-Brooke1"></a>Brooke NM, Garcia-Fernandez J, Holland PW (1998) The ParaHox gene cluster is an evolutionary sister of the Hox gene cluster. Nature  392: 920–922.  <ul class="find" data-citedArticleID="925504"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+ParaHox+gene+cluster+is+an+evolutionary+sister+of+the+Hox+gene+cluster.&amp;auth=&amp;atitle=The+ParaHox+gene+cluster+is+an+evolutionary+sister+of+the+Hox+gene+cluster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ParaHox+gene+cluster+is+an+evolutionary+sister+of+the+Hox+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ParaHox+gene+cluster+is+an+evolutionary+sister+of+the+Hox+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000153-Saitou1" id="pone.0000153-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="925686"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000153-Bryant1" id="pone.0000153-Bryant1"></a>Bryant D, Moulton V (2004) Neighbor-net: an agglomerative method for the construction of phylogenetic networks. Mol Biol Evol  21: 255–265.  <ul class="find" data-citedArticleID="925508" data-doi="10.1093/molbev/msh018"><li><a href="http://dx.doi.org/10.1093/molbev/msh018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neighbor-net%3A+an+agglomerative+method+for+the+construction+of+phylogenetic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neighbor-net%3A+an+agglomerative+method+for+the+construction+of+phylogenetic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000153-Huson1" id="pone.0000153-Huson1"></a>Huson DH, Bryant D (2006) Application of phylogenetic networks in evolutionary studies. Mol Biol Evol  23: 254–267.  <ul class="find" data-citedArticleID="925586" data-doi="10.1093/molbev/msj030"><li><a href="http://dx.doi.org/10.1093/molbev/msj030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+phylogenetic+networks+in+evolutionary+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+phylogenetic+networks+in+evolutionary+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000153-Kappen1" id="pone.0000153-Kappen1"></a>Kappen C (1996) Theoretical approaches to the analysis of homeobox gene evolution. Comput Chem  20: 49–59.  <ul class="find" data-citedArticleID="925596" data-doi="10.1016/s0097-8485(96)80007-6"><li><a href="http://dx.doi.org/10.1016/s0097-8485(96)80007-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theoretical+approaches+to+the+analysis+of+homeobox+gene+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theoretical+approaches+to+the+analysis+of+homeobox+gene+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000153-Schubert1" id="pone.0000153-Schubert1"></a>Schubert FR, Nieselt-Struwe K, Gruss P (1993) The Antennapedia-type homeobox genes have evolved from three precursors separated early in metazoan evolution. Proc Natl Acad Sci U S A  90: 143–147.  <ul class="find" data-citedArticleID="925692" data-doi="10.1073/pnas.90.1.143"><li><a href="http://dx.doi.org/10.1073/pnas.90.1.143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Antennapedia-type+homeobox+genes+have+evolved+from+three+precursors+separated+early+in+metazoan+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Antennapedia-type+homeobox+genes+have+evolved+from+three+precursors+separated+early+in+metazoan+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000153-Grienenberger1" id="pone.0000153-Grienenberger1"></a>Grienenberger A, Merabet S, Manak J, Iltis I, Fabre A, et al.  (2003) Tgfbeta signaling acts on a Hox response element to confer specificity and diversity to Hox protein function. Development  130: 5445–5455.  <ul class="find" data-citedArticleID="925572" data-doi="10.1242/dev.00760"><li><a href="http://dx.doi.org/10.1242/dev.00760" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tgfbeta+signaling+acts+on+a+Hox+response+element+to+confer+specificity+and+diversity+to+Hox+protein+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tgfbeta+signaling+acts+on+a+Hox+response+element+to+confer+specificity+and+diversity+to+Hox+protein+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000153-Gehring1" id="pone.0000153-Gehring1"></a>Gehring WJ, Affolter M, Burglin T (1994) Homeodomain proteins. Annu Rev Biochem  63: 487–526.  <ul class="find" data-citedArticleID="925568" data-doi="10.1146/annurev.bi.63.070194.002415"><li><a href="http://dx.doi.org/10.1146/annurev.bi.63.070194.002415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeodomain+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeodomain+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000153-Zhang1" id="pone.0000153-Zhang1"></a>Zhang J, Nei M (1996) Evolution of Antennapedia-class homeobox genes. Genetics  142: 295–303.  <ul class="find" data-citedArticleID="925738"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+Antennapedia-class+homeobox+genes.&amp;auth=&amp;atitle=Evolution+of+Antennapedia-class+homeobox+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+Antennapedia-class+homeobox+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+Antennapedia-class+homeobox+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000153-Kortschak1" id="pone.0000153-Kortschak1"></a>Kortschak RD, Samuel G, Saint R, Miller DJ (2003) EST analysis of the cnidarian Acropora millepora reveals extensive gene loss and rapid sequence divergence in the model invertebrates. Curr Biol  13: 2190–2195.  <ul class="find" data-citedArticleID="925604" data-doi="10.1016/j.cub.2003.11.030"><li><a href="http://dx.doi.org/10.1016/j.cub.2003.11.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EST+analysis+of+the+cnidarian+Acropora+millepora+reveals+extensive+gene+loss+and+rapid+sequence+divergence+in+the+model+invertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EST+analysis+of+the+cnidarian+Acropora+millepora+reveals+extensive+gene+loss+and+rapid+sequence+divergence+in+the+model+invertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0000153-Technau1" id="pone.0000153-Technau1"></a>Technau U, Rudd S, Maxwell P, Gordon PM, Saina M, et al.  (2005) Maintenance of ancestral complexity and non-metazoan genes in two basal cnidarians. Trends Genet  21: 633–639.  <ul class="find" data-citedArticleID="925724" data-doi="10.1016/j.tig.2005.09.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.09.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+ancestral+complexity+and+non-metazoan+genes+in+two+basal+cnidarians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+ancestral+complexity+and+non-metazoan+genes+in+two+basal+cnidarians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0000153-Sekimoto1" id="pone.0000153-Sekimoto1"></a>Sekimoto T, Yoshinobu K, Yoshida M, Kuratani S, Fujimoto S, et al.  (1998) Region-specific expression of murine Hox genes implies the Hox code-mediated patterning of the digestive tract. Genes Cells  3: 51–64.  <ul class="find" data-citedArticleID="925696" data-doi="10.1046/j.1365-2443.1998.00167.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2443.1998.00167.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Region-specific+expression+of+murine+Hox+genes+implies+the+Hox+code-mediated+patterning+of+the+digestive+tract." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Region-specific+expression+of+murine+Hox+genes+implies+the+Hox+code-mediated+patterning+of+the+digestive+tract.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0000153-Ruvinsky1" id="pone.0000153-Ruvinsky1"></a>Ruvinsky I, Gibson-Brown JJ (2000) Genetic and developmental bases of serial homology in vertebrate limb evolution. Development  127: 5233–5244.  <ul class="find" data-citedArticleID="925682"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+and+developmental+bases+of+serial+homology+in+vertebrate+limb+evolution.&amp;auth=&amp;atitle=Genetic+and+developmental+bases+of+serial+homology+in+vertebrate+limb+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+and+developmental+bases+of+serial+homology+in+vertebrate+limb+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+and+developmental+bases+of+serial+homology+in+vertebrate+limb+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0000153-Taylor1" id="pone.0000153-Taylor1"></a>Taylor HS, Vanden Heuvel GB, Igarashi P (1997) A conserved Hox axis in the mouse and human female reproductive system: late establishment and persistent adult expression of the Hoxa cluster genes. Biol Reprod  57: 1338–1345.  <ul class="find" data-citedArticleID="925722" data-doi="10.1095/biolreprod57.6.1338"><li><a href="http://dx.doi.org/10.1095/biolreprod57.6.1338" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+conserved+Hox+axis+in+the+mouse+and+human+female+reproductive+system%3A+late+establishment+and+persistent+adult+expression+of+the+Hoxa+cluster+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+conserved+Hox+axis+in+the+mouse+and+human+female+reproductive+system%3A+late+establishment+and+persistent+adult+expression+of+the+Hoxa+cluster+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0000153-Mazza1" id="pone.0000153-Mazza1"></a>Mazza ME, Pang K, Martindale M, Finnerty J (submitted) Gene structure and developmental expression of an otx gene in the sea anemone Nematostella  <ul class="find" data-citedArticleID="925642"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+structure+and+developmental+expression+of+an+otx+gene+in+the+sea+anemone+Nematostella&amp;auth=&amp;atitle=Gene+structure+and+developmental+expression+of+an+otx+gene+in+the+sea+anemone+Nematostella" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+structure+and+developmental+expression+of+an+otx+gene+in+the+sea+anemone+Nematostella" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+structure+and+developmental+expression+of+an+otx+gene+in+the+sea+anemone+Nematostella%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0000153-Kusserow1" id="pone.0000153-Kusserow1"></a>Kusserow A, Pang K, Sturm C, Hrouda M, Lentfer J, et al.  (2005) Unexpected complexity of the Wnt gene family in a sea anemone. Nature  433: 156–160.  <ul class="find" data-citedArticleID="925616" data-doi="10.1038/nature03158"><li><a href="http://dx.doi.org/10.1038/nature03158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unexpected+complexity+of+the+Wnt+gene+family+in+a+sea+anemone." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unexpected+complexity+of+the+Wnt+gene+family+in+a+sea+anemone.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0000153-Peterson1" id="pone.0000153-Peterson1"></a>Peterson KJ, Butterfield NJ (2005) Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record. Proc Natl Acad Sci U S A  102: 9547–9552.  <ul class="find" data-citedArticleID="925664" data-doi="10.1073/pnas.0503660102"><li><a href="http://dx.doi.org/10.1073/pnas.0503660102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0000153-Collins2" id="pone.0000153-Collins2"></a>Collins AG, Schuchert P, Marques AC, Jankowski T, Medina M, et al.  (2006) Medusozoan phylogeny and character evolution clarified by new large and small subunit rDNA data and an assessment of the utility of phylogenetic mixture models. Syst Biol  55: 97–115.  <ul class="find" data-citedArticleID="925520"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Medusozoan+phylogeny+and+character+evolution+clarified+by+new+large+and+small+subunit+rDNA+data+and+an+assessment+of+the+utility+of+phylogenetic+mixture+models.&amp;auth=&amp;atitle=Medusozoan+phylogeny+and+character+evolution+clarified+by+new+large+and+small+subunit+rDNA+data+and+an+assessment+of+the+utility+of+phylogenetic+mixture+models." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Medusozoan+phylogeny+and+character+evolution+clarified+by+new+large+and+small+subunit+rDNA+data+and+an+assessment+of+the+utility+of+phylogenetic+mixture+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Medusozoan+phylogeny+and+character+evolution+clarified+by+new+large+and+small+subunit+rDNA+data+and+an+assessment+of+the+utility+of+phylogenetic+mixture+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0000153-ArenasMena1" id="pone.0000153-ArenasMena1"></a>Arenas-Mena C, Cameron AR, Davidson EH (2000) Spatial expression of Hox cluster genes in the ontogeny of a sea urchin. Development  127: 4631–4643.  <ul class="find" data-citedArticleID="925492"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Spatial+expression+of+Hox+cluster+genes+in+the+ontogeny+of+a+sea+urchin.&amp;auth=&amp;atitle=Spatial+expression+of+Hox+cluster+genes+in+the+ontogeny+of+a+sea+urchin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+expression+of+Hox+cluster+genes+in+the+ontogeny+of+a+sea+urchin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+expression+of+Hox+cluster+genes+in+the+ontogeny+of+a+sea+urchin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0000153-Lee1" id="pone.0000153-Lee1"></a>Lee PN, Callaerts P, De Couet HG, Martindale MQ (2003) Cephalopod Hox genes and the origin of morphological novelties. Nature  424: 1061–1065.  <ul class="find" data-citedArticleID="925618" data-doi="10.1038/nature01872"><li><a href="http://dx.doi.org/10.1038/nature01872" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cephalopod+Hox+genes+and+the+origin+of+morphological+novelties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cephalopod+Hox+genes+and+the+origin+of+morphological+novelties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0000153-Bonneton1" id="pone.0000153-Bonneton1"></a>Bonneton F (2003) [Extreme divergence of a homeotic gene: the bicoid case]. Med Sci (Paris)  19: 1265–1270.  <ul class="find" data-citedArticleID="925500" data-doi="10.1051/medsci/200319121265"><li><a href="http://dx.doi.org/10.1051/medsci/200319121265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BExtreme+divergence+of+a+homeotic+gene%3A+the+bicoid+case%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BExtreme+divergence+of+a+homeotic+gene%3A+the+bicoid+case%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0000153-Hughes2" id="pone.0000153-Hughes2"></a>Hughes CL, Liu PZ, Kaufman TC (2004) Expression patterns of the rogue Hox genes Hox3/zen and fushi tarazu in the apterygote insect Thermobia domestica. Evol Dev  6: 393–401.  <ul class="find" data-citedArticleID="925582" data-doi="10.1111/j.1525-142x.2004.04048.x"><li><a href="http://dx.doi.org/10.1111/j.1525-142x.2004.04048.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+patterns+of+the+rogue+Hox+genes+Hox3%2Fzen+and+fushi+tarazu+in+the+apterygote+insect+Thermobia+domestica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+patterns+of+the+rogue+Hox+genes+Hox3%2Fzen+and+fushi+tarazu+in+the+apterygote+insect+Thermobia+domestica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0000153-Dearden1" id="pone.0000153-Dearden1"></a>Dearden P, Grbic M, Falciani F, Akam M (2000) Maternal expression and early zygotic regulation of the Hox3/zen gene in the grasshopper Schistocerca gregaria. Evol Dev  2: 261–270.  <ul class="find" data-citedArticleID="925526" data-doi="10.1046/j.1525-142x.2000.00065.x"><li><a href="http://dx.doi.org/10.1046/j.1525-142x.2000.00065.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maternal+expression+and+early+zygotic+regulation+of+the+Hox3%2Fzen+gene+in+the+grasshopper+Schistocerca+gregaria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maternal+expression+and+early+zygotic+regulation+of+the+Hox3%2Fzen+gene+in+the+grasshopper+Schistocerca+gregaria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0000153-Stauber1" id="pone.0000153-Stauber1"></a>Stauber M, Jackle H, Schmidt-Ott U (1999) The anterior determinant bicoid of Drosophila is a derived Hox class 3 gene. Proc Natl Acad Sci U S A  96: 3786–3789.  <ul class="find" data-citedArticleID="925708" data-doi="10.1073/pnas.96.7.3786"><li><a href="http://dx.doi.org/10.1073/pnas.96.7.3786" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+anterior+determinant+bicoid+of+Drosophila+is+a+derived+Hox+class+3+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+anterior+determinant+bicoid+of+Drosophila+is+a+derived+Hox+class+3+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0000153-Stauber2" id="pone.0000153-Stauber2"></a>Stauber M, Prell A, Schmidt-Ott U (2002) A single Hox3 gene with composite bicoid and zerknullt expression characteristics in non-Cyclorrhaphan flies. Proc Natl Acad Sci U S A  99: 274–279.  <ul class="find" data-citedArticleID="925710" data-doi="10.1073/pnas.012292899"><li><a href="http://dx.doi.org/10.1073/pnas.012292899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+Hox3+gene+with+composite+bicoid+and+zerknullt+expression+characteristics+in+non-Cyclorrhaphan+flies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+Hox3+gene+with+composite+bicoid+and+zerknullt+expression+characteristics+in+non-Cyclorrhaphan+flies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0000153-Manzanares1" id="pone.0000153-Manzanares1"></a>Manzanares M, Wada H, Itasaki N, Trainor PA, Krumlauf R, et al.  (2000) Conservation and elaboration of Hox gene regulation during evolution of the vertebrate head. Nature  408: 854–857.  <ul class="find" data-citedArticleID="925630" data-doi="10.1038/35048570"><li><a href="http://dx.doi.org/10.1038/35048570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+and+elaboration+of+Hox+gene+regulation+during+evolution+of+the+vertebrate+head." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+and+elaboration+of+Hox+gene+regulation+during+evolution+of+the+vertebrate+head.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">102.
              </span><a name="pone.0000153-Kourakis2" id="pone.0000153-Kourakis2"></a>Kourakis MJ, Master VA, Lokhorst DK, Nardelli-Haefliger D, Wedeen CJ, et al.  (1997) Conserved anterior boundaries of Hox gene expression in the central nervous system of the leech Helobdella. Dev Biol  190: 284–300.  <ul class="find" data-citedArticleID="925608" data-doi="10.1006/dbio.1997.8689"><li><a href="http://dx.doi.org/10.1006/dbio.1997.8689" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+anterior+boundaries+of+Hox+gene+expression+in+the+central+nervous+system+of+the+leech+Helobdella." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+anterior+boundaries+of+Hox+gene+expression+in+the+central+nervous+system+of+the+leech+Helobdella.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">103.
              </span><a name="pone.0000153-Iimura1" id="pone.0000153-Iimura1"></a>Iimura T, Pourquie O (2006) Collinear activation of Hoxb genes during gastrulation is linked to mesoderm cell ingression. Nature  442: 568–571.  <ul class="find" data-citedArticleID="925588" data-doi="10.1038/nature04838"><li><a href="http://dx.doi.org/10.1038/nature04838" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collinear+activation+of+Hoxb+genes+during+gastrulation+is+linked+to+mesoderm+cell+ingression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collinear+activation+of+Hoxb+genes+during+gastrulation+is+linked+to+mesoderm+cell+ingression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">104.
              </span><a name="pone.0000153-Duboule1" id="pone.0000153-Duboule1"></a>Duboule D (1992) The vertebrate limb: a model system to study the Hox/HOM gene network during development and evolution. Bioessays  14: 375–384.  <ul class="find" data-citedArticleID="925530" data-doi="10.1002/bies.950140606"><li><a href="http://dx.doi.org/10.1002/bies.950140606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+vertebrate+limb%3A+a+model+system+to+study+the+Hox%2FHOM+gene+network+during+development+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+vertebrate+limb%3A+a+model+system+to+study+the+Hox%2FHOM+gene+network+during+development+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">105.
              </span><a name="pone.0000153-vandenAkker1" id="pone.0000153-vandenAkker1"></a>van den Akker E, Forlani S, Chawengsaksophak K, de Graaff W, Beck F, et al.  (2002) Cdx1 and Cdx2 have overlapping functions in anteroposterior patterning and posterior axis elongation. Development  129: 2181–2193.  <ul class="find" data-citedArticleID="925740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cdx1+and+Cdx2+have+overlapping+functions+in+anteroposterior+patterning+and+posterior+axis+elongation.&amp;auth=&amp;atitle=Cdx1+and+Cdx2+have+overlapping+functions+in+anteroposterior+patterning+and+posterior+axis+elongation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cdx1+and+Cdx2+have+overlapping+functions+in+anteroposterior+patterning+and+posterior+axis+elongation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cdx1+and+Cdx2+have+overlapping+functions+in+anteroposterior+patterning+and+posterior+axis+elongation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">106.
              </span><a name="pone.0000153-Frobius1" id="pone.0000153-Frobius1"></a>Frobius AC, Seaver EC (2006) ParaHox gene expression in the polychaete annelid Capitella sp. I. Dev Genes Evol  216: 81–88.  <ul class="find" data-citedArticleID="925556" data-doi="10.1007/s00427-005-0049-0"><li><a href="http://dx.doi.org/10.1007/s00427-005-0049-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ParaHox+gene+expression+in+the+polychaete+annelid+Capitella+sp.+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ParaHox+gene+expression+in+the+polychaete+annelid+Capitella+sp.+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">107.
              </span><a name="pone.0000153-Roberts1" id="pone.0000153-Roberts1"></a>Roberts DJ, Smith DM, Goff DJ, Tabin CJ (1998) Epithelial-mesenchymal signaling during the regionalization of the chick gut. Development  125: 2791–2801.  <ul class="find" data-citedArticleID="925678"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Epithelial-mesenchymal+signaling+during+the+regionalization+of+the+chick+gut.&amp;auth=&amp;atitle=Epithelial-mesenchymal+signaling+during+the+regionalization+of+the+chick+gut." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epithelial-mesenchymal+signaling+during+the+regionalization+of+the+chick+gut." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epithelial-mesenchymal+signaling+during+the+regionalization+of+the+chick+gut.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">108.
              </span><a name="pone.0000153-Martindale1" id="pone.0000153-Martindale1"></a>Martindale MQ, Pang K, Finnerty JR (2004) Investigating the origins of triploblasty: 'mesodermal' gene expression in a diploblastic animal, the sea anemone Nematostella vectensis (phylum, Cnidaria; class, Anthozoa). Development  131: 2463–2474.  <ul class="find" data-citedArticleID="925632" data-doi="10.1242/dev.01119"><li><a href="http://dx.doi.org/10.1242/dev.01119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigating+the+origins+of+triploblasty%3A+%27mesodermal%27+gene+expression+in+a+diploblastic+animal%2C+the+sea+anemone+Nematostella+vectensis+%28phylum%2C+Cnidaria%3B+class%2C+Anthozoa%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigating+the+origins+of+triploblasty%3A+%27mesodermal%27+gene+expression+in+a+diploblastic+animal%2C+the+sea+anemone+Nematostella+vectensis+%28phylum%2C+Cnidaria%3B+class%2C+Anthozoa%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">109.
              </span><a name="pone.0000153-Sinsheimer1" id="pone.0000153-Sinsheimer1"></a>Sinsheimer JS, Lake JA, Little RJ (1996) Bayesian hypothesis testing of four-taxon topologies using molecular sequence data. Biometrics  52: 193–210.  <ul class="find" data-citedArticleID="925702" data-doi="10.2307/2533156"><li><a href="http://dx.doi.org/10.2307/2533156" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+hypothesis+testing+of+four-taxon+topologies+using+molecular+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+hypothesis+testing+of+four-taxon+topologies+using+molecular+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">110.
              </span><a name="pone.0000153-Yang1" id="pone.0000153-Yang1"></a>Yang Z, Rannala B (1997) Bayesian phylogenetic inference using DNA sequences: a Markov Chain Monte Carlo Method. Mol Biol Evol  14: 717–724.  <ul class="find" data-citedArticleID="925736" data-doi="10.1093/oxfordjournals.molbev.a025811"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+phylogenetic+inference+using+DNA+sequences%3A+a+Markov+Chain+Monte+Carlo+Method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+phylogenetic+inference+using+DNA+sequences%3A+a+Markov+Chain+Monte+Carlo+Method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">111.
              </span><a name="pone.0000153-Felsenstein1" id="pone.0000153-Felsenstein1"></a>Felsenstein J (1981) Evolutionary trees from DNA sequences: a maximum likelihood approach. J Mol Evol  17: 368–376.  <ul class="find" data-citedArticleID="925534" data-doi="10.1007/bf01734359"><li><a href="http://dx.doi.org/10.1007/bf01734359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+trees+from+DNA+sequences%3A+a+maximum+likelihood+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+trees+from+DNA+sequences%3A+a+maximum+likelihood+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">112.
              </span><a name="pone.0000153-Pendleton1" id="pone.0000153-Pendleton1"></a>Pendleton JW, Nagai BK, Murtha MT, Ruddle FH (1993) Expansion of the Hox gene family and the evolution of chordates. Proc Natl Acad Sci U S A  90: 6300–6304.  <ul class="find" data-citedArticleID="925662" data-doi="10.1073/pnas.90.13.6300"><li><a href="http://dx.doi.org/10.1073/pnas.90.13.6300" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expansion+of+the+Hox+gene+family+and+the+evolution+of+chordates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expansion+of+the+Hox+gene+family+and+the+evolution+of+chordates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">113.
              </span><a name="pone.0000153-Damen1" id="pone.0000153-Damen1"></a>Damen WG, Tautz D (1998) A Hox class 3 orthologue from the spider Cupiennius salei is expressed in a Hox-gene-like fashion. Dev Genes Evol  208: 586–590.  <ul class="find" data-citedArticleID="925522" data-doi="10.1007/s004270050218"><li><a href="http://dx.doi.org/10.1007/s004270050218" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Hox+class+3+orthologue+from+the+spider+Cupiennius+salei+is+expressed+in+a+Hox-gene-like+fashion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Hox+class+3+orthologue+from+the+spider+Cupiennius+salei+is+expressed+in+a+Hox-gene-like+fashion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">114.
              </span><a name="pone.0000153-Felsenstein2" id="pone.0000153-Felsenstein2"></a>Felsenstein J (2004) PHYLIP - Phylogeny Inference Package: Felsenstein, J  <ul class="find" data-citedArticleID="925536" data-doi="10.1002/0471650129.dob0534"><li><a href="http://dx.doi.org/10.1002/0471650129.dob0534" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PHYLIP+-+Phylogeny+Inference+Package%3A+Felsenstein%2C+J" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PHYLIP+-+Phylogeny+Inference+Package%3A+Felsenstein%2C+J%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">115.
              </span><a name="pone.0000153-Jones1" id="pone.0000153-Jones1"></a>Jones DT, Taylor WR, Thornton JM (1992) The rapid generation of mutation data matrices from protein sequences. Computer Applications in the Biosciences  8: 275–282.  <ul class="find" data-citedArticleID="925590" data-doi="10.1093/bioinformatics/8.3.275"><li><a href="http://dx.doi.org/10.1093/bioinformatics/8.3.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rapid+generation+of+mutation+data+matrices+from+protein+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rapid+generation+of+mutation+data+matrices+from+protein+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">116.
              </span><a name="pone.0000153-Abascal1" id="pone.0000153-Abascal1"></a>Abascal F, Zardoya R, Posada D (2005) ProtTest: selection of best-fit models of protein evolution. Bioinformatics  21: 2104–2105.  <ul class="find" data-citedArticleID="925488" data-doi="10.1093/bioinformatics/bti263"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ProtTest%3A+selection+of+best-fit+models+of+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ProtTest%3A+selection+of+best-fit+models+of+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">117.
              </span><a name="pone.0000153-Felsenstein3" id="pone.0000153-Felsenstein3"></a>Felsenstein J (1985) Confidence limits on phylogenies: an approach using the bootstrap. Evolution  39: 783–791.  <ul class="find" data-citedArticleID="925538" data-doi="10.2307/2408678"><li><a href="http://dx.doi.org/10.2307/2408678" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Confidence+limits+on+phylogenies%3A+an+approach+using+the+bootstrap." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Confidence+limits+on+phylogenies%3A+an+approach+using+the+bootstrap.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">118.
              </span><a name="pone.0000153-Maddison1" id="pone.0000153-Maddison1"></a>Maddison WP, Maddison DR (1992) MacClade: Analysis of phylogeny and character evolution. 3.07 ed. Sunderland, , MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">119.
              </span><a name="pone.0000153-Ronquist1" id="pone.0000153-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="925680" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">120.
              </span><a name="pone.0000153-Swofford1" id="pone.0000153-Swofford1"></a>Swofford DL (2003) Paup*. Phylogenetic Analysis Using Parsimony (*and Other Methods). 4.0b10 for Unix ed. Sunderland, , Massachusetts: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">121.
              </span><a name="pone.0000153-Mullikin1" id="pone.0000153-Mullikin1"></a>Mullikin JC, Ning Z (2003) The phusion assembler. Genome Res  13: 81–90.  <ul class="find" data-citedArticleID="925652" data-doi="10.1101/gr.731003"><li><a href="http://dx.doi.org/10.1101/gr.731003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phusion+assembler." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phusion+assembler.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">122.
              </span><a name="pone.0000153-Magie1" id="pone.0000153-Magie1"></a>Magie CR, Pang K, Martindale MQ (2005) Genomic inventory and expression of Sox and Fox genes in the cnidarian Nematostella vectensis. Dev Genes Evol  215: 618–630.  <ul class="find" data-citedArticleID="925626" data-doi="10.1007/s00427-005-0022-y"><li><a href="http://dx.doi.org/10.1007/s00427-005-0022-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+inventory+and+expression+of+Sox+and+Fox+genes+in+the+cnidarian+Nematostella+vectensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+inventory+and+expression+of+Sox+and+Fox+genes+in+the+cnidarian+Nematostella+vectensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">123.
              </span><a name="pone.0000153-Stephenson2" id="pone.0000153-Stephenson2"></a>Stephenson TA (1935) The British Sea Anemones. Volume II. London: The Ray Society.   <ul class="find-nolinks"></ul></li><li><span class="label">124.
              </span><a name="pone.0000153-Maddison2" id="pone.0000153-Maddison2"></a>Maddison DR, Maddison WP (2000) MacClade 4: Analysis of Phylogeny and Character Evolution. Sunderland, , MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">125.
              </span><a name="pone.0000153-VanAuken1" id="pone.0000153-VanAuken1"></a>Van Auken K, Weaver DC, Edgar LG, Wood WB (2000) Caenorhabditis elegans embryonic axial patterning requires two recently discovered posterior-group Hox genes. Proc Natl Acad Sci U S A  97: 4499–4503.  <ul class="find" data-citedArticleID="925730" data-doi="10.1073/pnas.97.9.4499"><li><a href="http://dx.doi.org/10.1073/pnas.97.9.4499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caenorhabditis+elegans+embryonic+axial+patterning+requires+two+recently+discovered+posterior-group+Hox+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Caenorhabditis+elegans+embryonic+axial+patterning+requires+two+recently+discovered+posterior-group+Hox+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">126.
              </span><a name="pone.0000153-Finnerty6" id="pone.0000153-Finnerty6"></a>Finnerty JR (2001) Cnidarians reveal intermediate stages in the evolution of Hox clusters and axial complexity. Am Zool  41: 608–620.  <ul class="find" data-citedArticleID="925554" data-doi="10.1093/icb/41.3.608"><li><a href="http://dx.doi.org/10.1093/icb/41.3.608" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cnidarians+reveal+intermediate+stages+in+the+evolution+of+Hox+clusters+and+axial+complexity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cnidarians+reveal+intermediate+stages+in+the+evolution+of+Hox+clusters+and+axial+complexity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">127.
              </span><a name="pone.0000153-Popovici1" id="pone.0000153-Popovici1"></a>Popovici C, Leveugle M, Birnbaum D, Coulier F (2001) Homeobox gene clusters and the human paralogy map. FEBS Lett  491: 237–242.  <ul class="find" data-citedArticleID="925668" data-doi="10.1016/s0014-5793(01)02187-1"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)02187-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeobox+gene+clusters+and+the+human+paralogy+map." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeobox+gene+clusters+and+the+human+paralogy+map.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">128.
              </span><a name="pone.0000153-Minguillon2" id="pone.0000153-Minguillon2"></a>Minguillon C, Garcia-Fernandez J (2003) Genesis and evolution of the Evx and Mox genes and the extended Hox and ParaHox gene clusters. Genome Biol  4: R12.  <ul class="find" data-citedArticleID="925648"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genesis+and+evolution+of+the+Evx+and+Mox+genes+and+the+extended+Hox+and+ParaHox+gene+clusters.&amp;auth=&amp;atitle=Genesis+and+evolution+of+the+Evx+and+Mox+genes+and+the+extended+Hox+and+ParaHox+gene+clusters." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genesis+and+evolution+of+the+Evx+and+Mox+genes+and+the+extended+Hox+and+ParaHox+gene+clusters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genesis+and+evolution+of+the+Evx+and+Mox+genes+and+the+extended+Hox+and+ParaHox+gene+clusters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">129.
              </span><a name="pone.0000153-Dayhoff1" id="pone.0000153-Dayhoff1"></a>Dayhoff MO, Schwartz RM, Orcutt BC (1978)  A model of evolutionary change in proteins. In: Dayhoff MO, editor. Atlas of Protein Sequence and Structure. Silver Spring, Maryland: National Biomedical Research Foundation.. pp. pp. 345–352.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.XML" value="309921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.PDF" value="1354289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.t001.PNG_L" value="114045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.t001.PNG_M" value="16153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.t001.PNG_S" value="1839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.t001.TIF" value="7476746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.t001.PNG_I" value="6925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g001.PNG_L" value="3113579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g001.PNG_M" value="219282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g001.PNG_S" value="8886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g001.TIF" value="3862166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g001.PNG_I" value="180051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g002.PNG_L" value="1023087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g002.PNG_M" value="85673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g002.PNG_S" value="4652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g002.TIF" value="1831616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g002.PNG_I" value="69483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g003.PNG_L" value="748550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g003.PNG_M" value="89050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g003.PNG_S" value="4162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g003.TIF" value="1067064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g003.PNG_I" value="34762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g004.PNG_L" value="5647239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g004.PNG_M" value="289022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g004.PNG_S" value="20900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g004.TIF" value="7267028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g004.PNG_I" value="306103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g005.PNG_L" value="5511787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g005.PNG_M" value="272936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g005.PNG_S" value="24670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g005.TIF" value="6906852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g005.PNG_I" value="351696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g006.PNG_L" value="954746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g006.PNG_M" value="91852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g006.PNG_S" value="3374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g006.TIF" value="1452790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g006.PNG_I" value="33238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g007.PNG_L" value="1389290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g007.PNG_M" value="124560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g007.PNG_S" value="6343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g007.TIF" value="2143374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g007.PNG_I" value="87471"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g008.PNG_L" value="278168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g008.PNG_M" value="132789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g008.PNG_S" value="4168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g008.TIF" value="439652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g008.PNG_I" value="50981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g009.PNG_L" value="573967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g009.PNG_M" value="135826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g009.PNG_S" value="4580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g009.TIF" value="942648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g009.PNG_I" value="55221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g010.PNG_L" value="709486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g010.PNG_M" value="137606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g010.PNG_S" value="6961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g010.TIF" value="1148824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g010.PNG_I" value="63917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g011.PNG_L" value="641679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g011.PNG_M" value="66073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g011.PNG_S" value="3398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g011.TIF" value="942534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.g011.PNG_I" value="26962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s001.EPS" value="1273491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s002.EPS" value="1265275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s003.EPS" value="1089531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s004.EPS" value="626307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s005.EPS" value="590503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s006.EPS" value="627879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s007.EPS" value="608875"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s008.EPS" value="591891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s009.EPS" value="587203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s010.EPS" value="580719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s011.EPS" value="4628767"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s012.EPS" value="726951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000153.s013.EPS" value="447819"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000153&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000153" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000153&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000153&volume=&issue=&title=Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, Nematostella vectensis&author_name=Joseph%20F.%20Ryan%2C%20Maureen%20E.%20Mazza%2C%20Kevin%20Pang%2C%20David%20Q.%20Matus%2C%20Andreas%20D.%20Baxevanis%2C%20Mark%20Q.%20Martindale%2C%20John%20R.%20Finnerty&start_page=1&end_page=23" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000153" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000153" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000153" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000153&amp;t=Pre-Bilaterian%20Origins%20of%20the%20Hox%20Cluster%20and%20the%20Hox%20Code%3A%20Evidence%20from%20the%20Sea%20Anemone%2C%20Nematostella%20vectensis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000153&title=Pre-Bilaterian%20Origins%20of%20the%20Hox%20Cluster%20and%20the%20Hox%20Code%3A%20Evidence%20from%20the%20Sea%20Anemone%2C%20Nematostella%20vectensis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000153&amp;title=Pre-Bilaterian%20Origins%20of%20the%20Hox%20Cluster%20and%20the%20Hox%20Code%3A%20Evidence%20from%20the%20Sea%20Anemone%2C%20Nematostella%20vectensis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000153" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000153&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Pre-Bilaterian Origins of the Hox Cluster and the Hox Code: Evidence from the Sea Anemone, Nematostella vectensis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000153';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Pre-Bilaterian%20Origins%20of%20the%20Hox%20Cluster%20and%20the%20Hox%20Code%3A%20Evidence%20from%20the%20Sea%20Anemone%2C%20Nematostella%20vectensis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000153" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000153" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cnidaria%22" title="Search for articles in the subject area:'Cnidaria'"><div class="flagText">Cnidaria</div></a>
              <div data-categoryid="43947" data-articleid="23144"
                   data-categoryname="Cnidaria"
                   class="flagImage" title="Flag 'Cnidaria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Endoderm%22" title="Search for articles in the subject area:'Endoderm'"><div class="flagText">Endoderm</div></a>
              <div data-categoryid="20437" data-articleid="23144"
                   data-categoryname="Endoderm"
                   class="flagImage" title="Flag 'Endoderm' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="23144"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23144"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23144"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23144"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23144"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="23144"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9701'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2939'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1743&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4417">Phylogenetic networks</a><br>Posted by fdelsuc</p>
      <p><a href="/annotation/listThread.action?root=3505">aamodel estimation for homeodomain</a><br>Posted by Ferdi</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
