

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000630"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000630" />

    <meta name="citation_title" content="Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus"/>
    <meta itemprop="name" content="Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus"/>

      <meta name="citation_author" content="Joanne Miles"/>
            <meta name="citation_author_institution" content="Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Jennifer A. Mitchell"/>
            <meta name="citation_author_institution" content="Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Lyubomira Chakalova"/>
            <meta name="citation_author_institution" content="Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Beatriz Goyenechea"/>
            <meta name="citation_author_institution" content="Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Cameron S. Osborne"/>
            <meta name="citation_author_institution" content="Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Laura O'Neill"/>
            <meta name="citation_author_institution" content="Institute of Biomedical Research, The Medical School, University of Birmingham, Birmingham, United Kingdom"/>
      <meta name="citation_author" content="Keiji Tanimoto"/>
            <meta name="citation_author_institution" content="Department of Cell and Developmental Biology, University of Michigan Medical School, Ann Arbor, Michigan, United States of America"/>
      <meta name="citation_author" content="James Douglas Engel"/>
            <meta name="citation_author_institution" content="Department of Cell and Developmental Biology, University of Michigan Medical School, Ann Arbor, Michigan, United States of America"/>
      <meta name="citation_author" content="Peter Fraser"/>
            <meta name="citation_author_institution" content="Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom"/>

    <meta name="citation_date" content="2007/7/18"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000630.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e630"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome.; citation_author=P Carninci; citation_author=T Kasukawa; citation_author=S Katayama; citation_author=J Gough; citation_author=MC Frith; citation_journal_title=Science; citation_volume=309; citation_number=1; citation_pages=1559-1563; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution.; citation_author=J Cheng; citation_author=P Kapranov; citation_author=J Drenkow; citation_author=S Dike; citation_author=S Brubaker; citation_journal_title=Science; citation_volume=308; citation_number=2; citation_pages=1149-1154; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22.; citation_author=D Kampa; citation_author=J Cheng; citation_author=P Kapranov; citation_author=M Yamanaka; citation_author=S Brubaker; citation_journal_title=Genome Res; citation_volume=14; citation_number=3; citation_pages=331-342; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Large-scale transcriptional activity in chromosomes 21 and 22.; citation_author=P Kapranov; citation_author=SE Cawley; citation_author=J Drenkow; citation_author=S Bekiranov; citation_author=RL Strausberg; citation_journal_title=Science; citation_volume=296; citation_number=4; citation_pages=916-919; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The transcriptional activity of human Chromosome 22.; citation_author=JL Rinn; citation_author=G Euskirchen; citation_author=P Bertone; citation_author=R Martone; citation_author=NM Luscombe; citation_journal_title=Genes Dev; citation_volume=17; citation_number=5; citation_pages=529-540; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Corfu deltabeta thalassemia deletion disrupts gamma-globin gene silencing and reveals post-transcriptional regulation of HbF expression.; citation_author=L Chakalova; citation_author=CS Osborne; citation_author=YF Dai; citation_author=B Goyenechea; citation_author=A Metaxotou-Mavromati; citation_journal_title=Blood; citation_volume=105; citation_number=6; citation_pages=2154-2160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcription defines the embryonic domains of cis-regulatory activity at the Drosophila bithorax complex.; citation_author=RA Drewell; citation_author=E Bae; citation_author=J Burr; citation_author=EB Lewis; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=99; citation_number=7; citation_pages=16853-16858; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Intergenic transcription through a polycomb group response element counteracts silencing.; citation_author=S Schmitt; citation_author=M Prestel; citation_author=R Paro; citation_journal_title=Genes Dev; citation_volume=19; citation_number=8; citation_pages=697-708; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chromatin remodeling and extragenic transcription at the MHC class II locus control region.; citation_author=K Masternak; citation_author=N Peyraud; citation_author=M Krawczyk; citation_author=E Barras; citation_author=W Reith; citation_journal_title=Nature Immunology; citation_volume=4; citation_number=9; citation_pages=132-137; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Locus control region transcription plays an active role in long-range gene activation.; citation_author=Y Ho; citation_author=F Elefant; citation_author=SA Liebhaber; citation_author=NE Cooke; citation_journal_title=Mol Cell; citation_volume=23; citation_number=10; citation_pages=365-375; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Analysis of intergenic transcription in the human IL-4/IL-13 gene cluster.; citation_author=DF Rogan; citation_author=DJ Cousins; citation_author=S Santangelo; citation_author=PA Ioannou; citation_author=M Antoniou; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=11; citation_pages=2446-2451; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Distal enhancer elements transcribe intergenic RNA in the IL-10 family gene cluster.; citation_author=EA Jones; citation_author=RA Flavell; citation_journal_title=J Immunol; citation_volume=175; citation_number=12; citation_pages=7437-7446; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Antisense intergenic transcription in V(D)J recombination.; citation_author=DJ Bolland; citation_author=AL Wood; citation_author=CM Johnston; citation_author=SF Bunting; citation_author=G Morgan; citation_journal_title=Nat Immunol; citation_volume=5; citation_number=13; citation_pages=630-637; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Xite, X-inactivation Intergenic transcription elements that regulate the probability of choice.; citation_author=Y Ogawa; citation_author=JT Lee; citation_journal_title=Molecular Cell; citation_volume=11; citation_number=14; citation_pages=731-743; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Intergenic transcription and transinduction of the human beta-globin locus.; citation_author=HL Ashe; citation_author=J Monks; citation_author=M Wijgerde; citation_author=P Fraser; citation_author=NJ Proudfoot; citation_journal_title=Genes & Development; citation_volume=11; citation_number=15; citation_pages=2494-2509; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Intergenic transcription and developmental remodeling of chromatin subdomains in the human beta-globin locus.; citation_author=J Gribnau; citation_author=K Diderich; citation_author=S Pruzina; citation_author=R Calzolari; citation_author=P Fraser; citation_journal_title=Mol Cell; citation_volume=5; citation_number=16; citation_pages=377-386; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Deletion of a region that is a candidate for the difference between the deletion forms of hereditary persistence of fetal hemoglobin and deltabeta-thalassemia affects beta- but not gamma-globin gene expression.; citation_author=R Calzolari; citation_author=T McMorrow; citation_author=N Yannoutsos; citation_author=A Langeveld; citation_author=F Grosveld; citation_journal_title=Embo J; citation_volume=18; citation_number=17; citation_pages=949-958; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Methylation of histone H3 Lys 4 in coding regions of active genes.; citation_author=BE Bernstein; citation_author=EL Humphrey; citation_author=RL Erlich; citation_author=R Schneider; citation_author=P Bouman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=18; citation_pages=8695-8700; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genomic maps and comparative analysis of histone modifications in human and mouse.; citation_author=BE Bernstein; citation_author=M Kamal; citation_author=K Lindblad-Toh; citation_author=S Bekiranov; citation_author=DK Bailey; citation_journal_title=Cell; citation_volume=120; citation_number=19; citation_pages=169-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The histone modification pattern of active genes revealed through genome-wide chromatin analysis of a higher eukaryote.; citation_author=D Schubeler; citation_author=DM MacAlpine; citation_author=D Scalzo; citation_author=C Wirbelauer; citation_author=C Kooperberg; citation_journal_title=Genes & Development; citation_volume=18; citation_number=20; citation_pages=1263-1271; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Developmentally dynamic histone acetylation pattern of a tissue-specific chromatin domain.; citation_author=EC Forsberg; citation_author=KM Downs; citation_author=HM Christensen; citation_author=H Im; citation_author=PA Nuzzi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=21; citation_pages=14494-14499; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Developmental stage differences in chromatin subdomains of the beta-globin locus.; citation_author=A Kim; citation_author=A Dean; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=101; citation_number=22; citation_pages=7028-7033; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nuclear localization and histone acetylation: a pathway for chromatin opening and transcriptional activation of the human beta-globin locus.; citation_author=D Schubeler; citation_author=C Francastel; citation_author=DM Cimbora; citation_author=A Reik; citation_author=DI Martin; citation_journal_title=Genes Dev; citation_volume=14; citation_number=23; citation_pages=940-950; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The murine beta-globin locus control region regulates the rate of transcription but not the hyperacetylation of histones at the active genes.; citation_author=D Schubeler; citation_author=M Groudine; citation_author=MA Bender; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=98; citation_number=24; citation_pages=11432-11437; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Dicer-dependent turnover of intergenic transcripts from the human beta-globin gene cluster.; citation_author=D Haussecker; citation_author=NJ Proudfoot; citation_journal_title=Mol Cell Biol; citation_volume=25; citation_number=25; citation_pages=9724-9733; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A complex chromatin landscape revealed by patterns of nuclease sensitivity and histone modification within the mouse beta-globin locus.; citation_author=M Bulger; citation_author=D Schubeler; citation_author=MA Bender; citation_author=J Hamilton; citation_author=CM Farrell; citation_journal_title=Mol Cell Biol; citation_volume=23; citation_number=26; citation_pages=5234-5244; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Effects of altered gene order or orientation of the locus control region on human beta-globin gene expression in mice.; citation_author=K Tanimoto; citation_author=Q Liu; citation_author=J Bungert; citation_author=JD Engel; citation_journal_title=Nature; citation_volume=398; citation_number=27; citation_pages=344-348; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Each hypersensitive site of the human beta-globin locus control region confers a different developmental pattern of expression on the globin genes.; citation_author=P Fraser; citation_author=S Pruzina; citation_author=M Antoniou; citation_author=F Grosveld; citation_journal_title=Genes Dev; citation_volume=7; citation_number=28; citation_pages=106-113; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Effect of deletion of 5′HS3 or 5′HS2 of the human beta-globin locus control region on the developmental regulation of globin gene expression in beta-globin locus yeast artificial chromosome transgenic mice.; citation_author=KR Peterson; citation_author=CH Clegg; citation_author=PA Navas; citation_author=EJ Norton; citation_author=TG Kimbrough; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=93; citation_number=29; citation_pages=6605-6609; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Reconstitution of human beta-globin locus control region hypersensitive sites in the absence of chromatin assembly.; citation_author=KM Leach; citation_author=K Nightingale; citation_author=K Igarashi; citation_author=PP Levings; citation_author=JD Engel; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=30; citation_pages=2629-2640; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A long terminal repeat of the human endogenous retrovirus ERV-9 is located in the 5′ boundary area of the human beta-globin locus control region.; citation_author=QM Long; citation_author=C Bengra; citation_author=CH Li; citation_author=F Kutlar; citation_author=D Tuan; citation_journal_title=Genomics; citation_volume=54; citation_number=31; citation_pages=542-555; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Definition of transcriptional promoters in the human beta globin locus control region.; citation_author=SJ Routledge; citation_author=NJ Proudfoot; citation_journal_title=J Mol Biol; citation_volume=323; citation_number=32; citation_pages=601-611; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Transcription of the Hypersensitive Site Hs2 Enhancer in Erythroid-Cells.; citation_author=D Tuan; citation_author=SM Kong; citation_author=K Hu; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=89; citation_number=33; citation_pages=11219-11223; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Proliferation and maturation of human erythroid progenitors in liquid culture.; citation_author=E Fibach; citation_author=D Manor; citation_author=A Oppenheim; citation_author=EA Rachmilewitz; citation_journal_title=Blood; citation_volume=73; citation_number=34; citation_pages=100-103; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Proliferation and differentiation of erythroid progenitors in liquid culture: analysis of progenitors derived from patients with polycythemia vera.; citation_author=E Fibach; citation_author=EA Rachmilewitz; citation_journal_title=Am J Hematol; citation_volume=35; citation_number=35; citation_pages=151-156; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Non-coding transcripts far upstream of the epsilon-globin gene are distinctly expressed in human primary tissues and erythroleukemia cell lines.; citation_author=P Xiang; citation_author=X Fang; citation_author=W Yin; citation_author=G Barkess; citation_author=Q Li; citation_journal_title=Biochem Biophys Res Commun; citation_volume=344; citation_number=36; citation_pages=623-630; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Intranuclear Colocation of Newly Replicated DNA and Pcna by Simultaneous Immunofluorescent Labeling and Confocal Microscopy in Mcf-7 Cells.; citation_author=C Humbert; citation_author=MS Santisteban; citation_author=Y Usson; citation_author=M Robertnicoud; citation_journal_title=Journal of Cell Science; citation_volume=103; citation_number=37; citation_pages=97-103; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Elongation by RNA polymerase II: the short and long of it.; citation_author=RJ Sims; citation_author=R Belotserkovskaya; citation_author=D Reinberg; citation_journal_title=Genes Dev; citation_volume=18; citation_number=38; citation_pages=2437-2468; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Histone variants, nucleosome assembly and epigenetic inheritance.; citation_author=S Henikoff; citation_author=T Furuyama; citation_author=K Ahmad; citation_journal_title=Trends Genet; citation_volume=20; citation_number=39; citation_pages=320-326; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Long-range chromatin regulatory interactions in vivo.; citation_author=D Carter; citation_author=L Chakalova; citation_author=CS Osborne; citation_author=YF Dai; citation_author=P Fraser; citation_journal_title=Nat Genet; citation_volume=32; citation_number=40; citation_pages=623-626; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Looping and interaction between hypersensitive sites in the active beta-globin locus.; citation_author=B Tolhuis; citation_author=RJ Palstra; citation_author=E Splinter; citation_author=F Grosveld; citation_author=W de Laat; citation_journal_title=Mol Cell; citation_volume=10; citation_number=41; citation_pages=1453-1465; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNA polymerase II transcription is concentrated outside replication domains throughout S-phase.; citation_author=DG Wansink; citation_author=EE Manders; citation_author=I van der Kraan; citation_author=JA Aten; citation_author=R van Driel; citation_journal_title=J Cell Sci 107 ( Pt; citation_volume=6); citation_number=42; citation_pages=1449-1456; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Active genes dynamically colocalize to shared sites of ongoing transcription.; citation_author=CS Osborne; citation_author=L Chakalova; citation_author=KE Brown; citation_author=D Carter; citation_author=A Horton; citation_journal_title=Nat Genet; citation_volume=36; citation_number=43; citation_pages=1065-1071; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The locus control region is required for association of the murine beta-globin locus with engaged transcription factories during erythroid maturation.; citation_author=T Ragoczy; citation_author=MA Bender; citation_author=A Telling; citation_author=R Byron; citation_author=M Groudine; citation_journal_title=Genes Dev; citation_volume=20; citation_number=44; citation_pages=1447-1457; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Replication and transcription: Forces that shape the landscape of the genome.; citation_author=L Chakalova; citation_author=JA Mitchell; citation_author=E Debrand; citation_author=CS Osborne; citation_author=P Fraser; citation_journal_title=Nature Reviews Genetics in press; citation_number=45; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The erythropoietic mouse spleen-a model system of development.; citation_author=HW Dickerman; citation_author=TC Cheng; citation_author=HH Kazazian; citation_author=JL Spivak; citation_journal_title=Arch Biochem Biophys; citation_volume=177; citation_number=46; citation_pages=1-9; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=RNA fluorescence in situ hybridization tagging and recovery of associated proteins to analyze in vivo chromatin interactions.; citation_author=L Chakalova; citation_author=D Carter; citation_author=P Fraser; citation_journal_title=Methods Enzymol; citation_volume=375; citation_number=47; citation_pages=479-493; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Immunoprecipitation of native chromatin: NChIP.; citation_author=LP O'Neill; citation_author=BM Turner; citation_journal_title=Methods; citation_volume=31; citation_number=48; citation_pages=76-82; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(T)(-Delta Delta C) method.; citation_author=KJ Livak; citation_author=TD Schmittgen; citation_journal_title=Methods; citation_volume=25; citation_number=49; citation_pages=402-408; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000630" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus"/>
    <meta name="twitter:description" content="Several lines of evidence have established strong links between transcriptional activity and specific post-translation modifications of histones. Here we show using RNA FISH that in erythroid cells, intergenic transcription in the human &beta;-globin locus occurs over a region of greater than 250 kb including several genes in the nearby olfactory receptor gene cluster. This entire region is transcribed during S phase of the cell cycle. However, within this region there are &sim;20 kb sub-domains of high intergenic transcription that occurs outside of S phase. These sub-domains are developmentally regulated and enriched with high levels of active modifications primarily to histone H3. The sub-domains correspond to the &beta;-globin locus control region, which is active at all developmental stages in erythroid cells, and the region flanking the developmentally regulated, active globin genes. These results correlate high levels of non-S phase intergenic transcription with domain-wide active histone modifications to histone H3."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000630.g006"/>

  <meta property="og:title" content="Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000630" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2666'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1045'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6320&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000630">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000630" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000630&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joanne Miles, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jennifer A. Mitchell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lyubomira Chakalova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Beatriz Goyenechea, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cameron S. Osborne, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura O'Neill, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biomedical Research, The Medical School, University of Birmingham, Birmingham, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Keiji Tanimoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell and Developmental Biology, University of Michigan Medical School, Ann Arbor, Michigan, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James Douglas Engel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell and Developmental Biology, University of Michigan Medical School, Ann Arbor, Michigan, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter Fraser
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:peter.fraser@bbsrc.ac.uk">peter.fraser@bbsrc.ac.uk</a></p>

                <p>Affiliation:
                  Laboratory of Chromatin and Gene Expression, The Babraham Institute, Babraham Research Campus, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 18, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000630</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000630" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000630" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000630" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000630" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000630-g001" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000630-g002" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000630-g003" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000630-g004" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000630-g005" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000630-g006" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000630">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000630" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Several lines of evidence have established strong links between transcriptional activity and specific post-translation modifications of histones. Here we show using RNA FISH that in erythroid cells, intergenic transcription in the human <em>β</em>-globin locus occurs over a region of greater than 250 kb including several genes in the nearby olfactory receptor gene cluster. This entire region is transcribed during S phase of the cell cycle. However, within this region there are ~20 kb sub-domains of high intergenic transcription that occurs outside of S phase. These sub-domains are developmentally regulated and enriched with high levels of active modifications primarily to histone H3. The sub-domains correspond to the <em>β</em>-globin locus control region, which is active at all developmental stages in erythroid cells, and the region flanking the developmentally regulated, active globin genes. These results correlate high levels of non-S phase intergenic transcription with domain-wide active histone modifications to histone H3.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Miles J, Mitchell JA, Chakalova L, Goyenechea B, Osborne CS, et al.  (2007) Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus. PLoS ONE 2(7):
          e630.
            doi:10.1371/journal.pone.0000630</p><p><strong>Academic Editor: </strong>Edith Heard, Institut Curie, France</p><p><strong>Received:</strong> April 23, 2007; <strong>Accepted:</strong> June 16, 2007; <strong>Published:</strong> July 18, 2007</p><p><strong>Copyright:</strong> © 2007 Miles et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Medical Research Council and the Biotechnology and Biological Sciences Research Council UK.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Far more transcription takes place in the cell nucleus than can be accounted for by protein-coding gene transcription alone<a href="#pone.0000630-Carninci1">[1]</a>–<a href="#pone.0000630-Rinn1">[5]</a>. Transcriptome studies have revealed a plethora of non-coding RNAs, some of which have been implicated in diverse regulatory processes such as dosage compensation, genomic imprinting and RNAi. However, most non-genic transcripts seem to fall into a category characterized by several studies investigating intergenic transcription<a href="#pone.0000630-Chakalova1">[6]</a>. Intergenic transcripts are often produced from regions flanking active genes and their long-range regulatory elements. Their occurrence in correlation with gene activity and chromatin structural alterations, suggests a role in the regulation of gene expression. Such intergenic transcription has been most extensively studied in the <em>Drosophila</em> bithorax complex and human <em>β</em>-globin loci. In the bithorax complex it has been suggested that intergenic transcription plays a role in initiating activation of the locus<a href="#pone.0000630-Drewell1">[7]</a> and in regulating cellular memory<a href="#pone.0000630-Schmitt1">[8]</a>. Intergenic transcription has also been proposed to be involved in regulation of the major histocompatibility complex locus<a href="#pone.0000630-Masternak1">[9]</a>, the human growth hormone locus<a href="#pone.0000630-Ho1">[10]</a> the IL4 locus<a href="#pone.0000630-Rogan1">[11]</a>, the IL10 gene cluster<a href="#pone.0000630-Jones1">[12]</a>, regulating chromatin accessibility during VDJ recombination<a href="#pone.0000630-Bolland1">[13]</a>, and in regulating probability of choice of X chromosome inactivation<a href="#pone.0000630-Ogawa1">[14]</a>. Thus intergenic transcription is common and may be a part of varied mechanisms for the regulation of gene expression in eukaryotes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In the human <em>β</em>-globin locus intergenic transcripts can be found throughout the locus in erythroid cells<a href="#pone.0000630-Ashe1">[15]</a>, <a href="#pone.0000630-Gribnau1">[16]</a> at very low abundance relative to gene transcripts. RNA FISH (fluorescent <em>in situ</em> hybridisation) showed that intergenic transcripts are only detected in a proportion of cells in an unsynchronized population. They appear to be generated in a cell-cycle-dependent manner; detectable predominantly during G1 phase with a small percentage of loci showing RNA FISH signals in early S-phase<a href="#pone.0000630-Gribnau1">[16]</a>. The highest percentage of loci with positive signals are detected with probes homologous to the DNase I sensitive sub-domains containing the locus control region (LCR) and active adult <em>β</em>-globin genes (<em>HBD</em>, haemoglobin delta; <em>HBB</em>, hemoglobin beta). Deletion of a 2.5 kb region containing the putative adult sub-domain intergenic promoter results in a sub-domain-wide failure to adopt the characteristic DNase I sensitive chromatin conformation during development, and an abnormally low and variegated expression of the adult <em>HBB</em> gene<a href="#pone.0000630-Gribnau1">[16]</a>, <a href="#pone.0000630-Calzolari1">[17]</a>. These results suggest that intergenic transcription could play a role in decondensation of chromatin domains and gene activation.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Genome scale studies have revealed that histone H3 di- and tri-methylation on lysine 4 (H3K4me2, H3K4me3) as well as H3 and H4 hyperacetylation (H3ac and H4ac, respectively) are enriched in the regions surrounding active genes<a href="#pone.0000630-Bernstein1">[18]</a>–<a href="#pone.0000630-Schubeler1">[20]</a>. Tri-methylation on H3K4 correlates most strongly with the promoter regions of expressed genes, often residing within 1 kb of the transcription start site. Hyperacetylation of both H3 and H4 also correlate with promoter regions while di-methylation on H3K4 occurs across wider regions in the vicinity of active genes. In contrast to the punctate patterns of H3K4 methylation detected in most active regions, several of the <em>Hox</em> clusters contain large domains of enriched H3K4 methylation encompassing multiple genes and their flanking sequences. <em>Hox</em> H3K4 methylated domains appear to be tissue-specific and correlate significantly with intergenic transcription<a href="#pone.0000630-Bernstein2">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>A complex pattern of histone modifications has been detected in the human and mouse <em>β</em>-globin loci<a href="#pone.0000630-Forsberg1">[21]</a>–<a href="#pone.0000630-Bulger1">[26]</a>. Collectively, these experiments have suggested a correlation between H3 and H4 acetylation and H3K4 methylation modifications associated with active genes, and DNase I hypersensitivity sites in the LCR as well as areas of high general DNase I sensitivity. Here we present a high-resolution, locus-wide analysis of intergenic transcripts and histone modifications across the human <em>β</em>-globin locus during development in yeast artificial chromosome (YAC) transgenic mice and in primary human erythroid cells. We show that the intergenic transcription pattern is complex and extensive and that active histone modifications strongly correlate with areas of non-S phase intergenic transcription linking the cell-cycle-specific timing of intergenic transcription with large chromatin domains of modified histones.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Intergenic transcription throughout the human <em>β</em>-globin locus</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We previously identified regions of intergenic transcription in the human <em>β</em>-globin locus in transgenic mice via RNA FISH<a href="#pone.0000630-Gribnau1">[16]</a>. Though rather crudely mapped, regions of increased intergenic transcription corresponded to domains of increased general DNase I sensitivity. To verify our FISH data and obtain higher resolution we used quantitative real-time reverse transcription PCR (RT-PCR) with several primer pairs encompassing nearly all of the non-repetitive, non-genic sequences across the entire <em>β</em>-globin locus. Total RNA was prepared for analysis from embryonic day E11.5 red cells and adult anemic spleen from transgenic 264W which contains a single copy human <em>β</em>-globin locus YAC<a href="#pone.0000630-Tanimoto1">[27]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The results show that at both stages of development the LCR is transcribed (<a href="#pone-0000630-g001">Figure 1</a>), consistent with our previous RNA FISH findings in which a relatively large percentage of loci were positive for transcript signals in this region. The abundance of LCR transcripts appears to be greater in embryonic blood cells compared to adult anemic spleen cells, which may reflect the fact that embryonic blood is composed of nearly 100% erythroid cells, whereas adult anemic spleen is 80–90% erythroid. However, we cannot rule out the possibility, and in fact the data suggest, that the intergenic transcription pattern in the LCR region may change during development. This is consistent with the LCR's role as a complex regulatory region<a href="#pone.0000630-Fraser1">[28]</a>, <a href="#pone.0000630-Peterson1">[29]</a> with many intergenic promoters<a href="#pone.0000630-Leach1">[30]</a>–<a href="#pone.0000630-Tuan1">[33]</a>. In embryonic cells intergenic transcript levels are high throughout the majority of the εγ domain whilst relatively low in the δβ domain. The region of low transcript levels immediately upstream of the <em>ΗΒΕ</em> (hemoglobin epsilon) gene suggests that the majority of LCR transcripts are not contiguous with εγ domain transcripts.</p>
<div class="figure" id="pone-0000630-g001"><div class="img"><a name="pone-0000630-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distinct domains of intergenic transcription in the human <em>β</em>-globin locus are developmentally regulated.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Intergenic transcription in the human <em>β</em>-globin locus in 264W transgenic mice measured by quantitative RT-PCR. Total RNA was prepared from erythroid tissues at two developmental stages – E11.5 embryonic blood (Embryonic) and adult anemic spleen (Adult). cDNA was produced and quantified by real-time PCR using primer pairs across the human <em>β</em>-globin locus. Bar plots represent relative transcript quantities normalised to the most 5′ primer pair in the olfactory receptor gene region, which shows low levels of transcription at both developmental stages. Primer pair positions are relative to the <em>HBE</em> gene transcription start site at position 1; they are aligned with a map of the locus shown below the graphs. Shaded regions of the graphs correspond to the locus control region (LTR promoter to downstream of LCR HS1) and the δβ domain (δβ intergenic promoter to 3′HS1). Map features: red arrowheads, globin genes; pink rectangle, <em>β</em>-like pseudogene; white arrowheads, olfactory receptor genes (<em>HOR</em>, human olfactory receptor); vertical blue lines, hypersensitive sites; blue arrows, intergenic transcription start sites; LTR, long terminal repeat; δβ, δβ intergenic promoter.</p>
<span>doi:10.1371/journal.pone.0000630.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>In adult cells transcript levels in the εγ domain are reduced whilst the LCR and δβ domains are relatively highly transcribed. The exceptions to this pattern are the regions immediately upstream and downstream of the δβ promoter where similar levels are seen at both stages of development. In addition to sense transcription in this region, strand-specific RT-PCR detects significant levels of antisense transcription, which appears to initiate somewhere downstream of the δβ promoter and terminate somewhere in the vicinity of the <em>β</em>-like pseudogene (unpublished observations). These data showing high levels of intergenic transcription in the LCR and the δβ domain in adult cells are consistent with our RNA FISH data in which a larger percentage of cells displayed RNA FISH signals in these regions, compared to the εγ domain<a href="#pone.0000630-Gribnau1">[16]</a>.</p>


<h4>Developmentally regulated histone modifications in the <em>β</em>-globin YAC transgene locus</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To compare histone modifications within the human <em>β</em>-globin locus to levels of intergenic transcription we assessed the pattern of histone modifications across the human <em>β</em>-globin locus in the 264W line using native chromatin immunoprecipitation (ChIP) on erythroid cells from E11.5 embryonic blood and adult anemic spleen. We sought to generate a high-resolution map of active histone modifications in the human <em>β</em>-globin locus in transgenic mice. In E11.5 embryonic blood, the LCR and the εγ domain are highly enriched for histone H3K4 tri-methylation and histone H3 hyperacetylation (<a href="#pone-0000630-g002">Figure 2</a>, top and middle panel). Conversely, the δβ domain is relatively devoid of these histone modifications. The 5′ and 3′ boundaries of enrichment for these active modifications occur near the long terminal repeat (LTR) promoter upstream of hypersensitive site 5 of the LCR (HS5) and the region upstream of the δβ intergenic promoter. This H3K4me3 and H3ac domain correlates well with the domain of increased intergenic transcription at E11.5 (<a href="#pone-0000630-g001">Figure 1</a>, top panel). In contrast to H3K4me3 and H3 hyperacetylation, H4 acetylation appears to be moderately enriched across the entire locus and does not appear to follow the same domain structure (<a href="#pone-0000630-g002">Figure 2</a>, bottom panel).</p>
<div class="figure" id="pone-0000630-g002"><div class="img"><a name="pone-0000630-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Histone modifications throughout the human <em>β</em>-globin locus in embryonic cells reflect intergenic transcription patterns.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Histone modifications in the human <em>β</em>-globin locus in embryonic blood cells from 264W transgenic embryos assayed by ChIP. Chromatin from E11.5 embryonic blood cells was immunoprecipitated with antibodies specific for trimethylated lysine 4 of histone H3 (H3K4me3), acetylated histone H3 (K9/14, H3ac), and acetylated histone H4 (ChIP grade antibody, K5/18/12/16, H4ac<sub>Ch</sub>). The fold-enrichment of antibody-bound sequences over input was analysed by real-time PCR using primer pairs across the <em>β</em>-globin locus. Bar plots represent enrichment normalised to the most 5′ primer pair in the olfactory receptor gene region, which shows low enrichment for all antibodies; horizontal dashed lines mark the level of the normalisation data point (value 1). Primer pair positions are relative to the <em>HBE</em> gene transcription start site at position 1; they are aligned with a map of the locus shown below the graphs. Shaded regions of the graphs correspond to the locus control region (LTR promoter to downstream of LCR HS1) and the δβ domain (δβ intergenic promoter to 3′HS1). Map features: red arrowheads, globin genes; pink rectangle, <em>β</em>-like pseudogene; white arrowheads, olfactory receptor genes (<em>HOR</em>, human olfactory receptor); vertical blue lines, hypersensitive sites; blue arrows, intergenic transcription start sites; LTR, long terminal repeat; δβ, δβ intergenic promoter.</p>
<span>doi:10.1371/journal.pone.0000630.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In adult anemic spleen we took advantage of the hugely increased numbers of available erythroid cells and used additional antibodies against di-methylated H3K4 and penta-acetylated H4 (H4ac<sub>p</sub>) in addition to those above. The results show that the histone modification profile in adult erythroid cells is dramatically different compared to E11.5 cells. The LCR and the δβ domain are highly enriched for H3K4me3, H3K4me2 and H3ac (<a href="#pone-0000630-g003">Figure 3</a>), while the εγ domain lacks these histone modifications. The presence of these histone modifications in the LCR and δβ domains correlates with increased intergenic transcription in these regions in adult cells (<a href="#pone-0000630-g001">Figure 1</a>, bottom panel) and increased general sensitivity to DNase I <a href="#pone.0000630-Gribnau1">[16]</a>. A sub-domain pattern of H4 acetylation is discernable with both H4 antibodies (<a href="#pone-0000630-g003">Figure 3</a>) but appears to be less well defined compared to H3 modifications, as in embryonic cells. The LCR and δβ sub-domains do appear enriched for H4 acetylated chromatin over the inactive εγ domain and the upstream olfactory receptor gene (<em>ORG</em>) region.</p>
<div class="figure" id="pone-0000630-g003"><div class="img"><a name="pone-0000630-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Histone modifications throughout the human <em>β</em>-globin locus in adult cells reflect intergenic transcription patterns.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Histone modifications in the human <em>β</em>-globin locus in 264W transgenic adult erythroid cells assayed by ChIP. Chromatin from anemic spleen was immunoprecipitated with antibodies specific for trimethylated lysine 4 of histone H3 (H3K4me3), dimethylated lysine 4 of histone H3 (H3K4me2), acetylated histone H3 (K9/14, H3ac), and acetylated histone H4 (two different antibodies: ChIP grade antibody, K5/18/12/16, H4ac<sub>Ch</sub>; penta lysine, H4ac<sub>P</sub>). The fold-enrichment of antibody-bound sequences over input was analysed by real-time PCR using primer pairs across the <em>β</em>-globin locus. Bar plots represent enrichment normalised to the most 5′ primer pair in the olfactory receptor gene region, which shows low enrichment for all antibodies; horizontal dashed lines mark the level of the normalisation data point (value 1). Primer pair positions are relative to the <em>HBE</em> gene transcription start site at position 1; they are aligned with a map of the locus shown below the graphs. Shaded regions of the graphs correspond to the locus control region (LTR promoter to downstream of LCR HS1) and the δβ domain (δβ intergenic promoter to 3′HS1). Map features: red arrowheads, globin genes; pink rectangle, <em>β</em>-like pseudogene; white arrowheads, olfactory receptor genes (<em>HOR</em>, human olfactory receptor); vertical blue lines, hypersensitive sites; blue arrows, intergenic transcription start sites; LTR, long terminal repeat; δβ, δβ intergenic promoter.</p>
<span>doi:10.1371/journal.pone.0000630.g003</span></div>

<h4>Histone modification profile of the endogenous <em>β</em>-globin locus in primary erythroid precursor cells</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The human <em>β</em>-globin locus has been studied extensively in transgenic mice. Despite the fact that the <em>β</em>-globin genes are fully expressed and the locus is developmentally regulated in transgenic mice there have been noticeable differences in the way the human locus is regulated in mice. We were therefore interested in characterising the epigenetic profile in the endogenous human <em>β</em>-globin locus at high resolution. Studying the locus in its endogenous chromosomal location provided the additional advantage of extending our analyses further into the 5′<em>ORG</em> region. We obtained nucleated, buffy coat cells from human peripheral blood from a local blood bank and used the two-step liquid culture system <a href="#pone.0000630-Fibach1">[34]</a>, <a href="#pone.0000630-Fibach2">[35]</a> to generate large quantities of adult erythroid precursor cells. We previously showed by RNA FISH that upon maturation, these cells transcribe primarily <em>HBB</em> with a small percentage of loci (approximately 10%) positive for <em>HBG</em> (hemoglobin gamma) transcription<a href="#pone.0000630-Chakalova1">[6]</a>. We used the same antibodies as in the adult anemic spleen transgenic studies above (<a href="#pone-0000630-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0000630-g004"><div class="img"><a name="pone-0000630-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Histone modifications throughout the human <em>β</em>-globin locus in primary erythroid precursor cells.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Histone modifications in the human <em>β</em>-globin locus in human primary erythroid precursor cells assayed by ChIP. Chromatin from erythroid precursor cells was immunoprecipitated with antibodies specific for trimethylated lysine 4 of histone H3 (H3K4me3), dimethylated lysine 4 of histone H3 (H3K4me2), acetylated histone H3 (K9/14, H3ac), and acetylated histone H4 (two different antibodies: ChIP grade antibody, K5/18/12/16, H4ac<sub>Ch</sub>; penta lysine, H4ac<sub>P</sub>). The fold-enrichment of antibody-bound sequences over input was analysed by real-time PCR using primer pairs across the <em>β</em>-globin locus. Bar plots represent enrichment normalised to the same primer pair in the olfactory receptor gene region as in previous figures (second from left to right; an additional primer pair upstream of the normalisation point is included in this experiment). Horizontal dashed lines mark the level of the normalisation data point (value 1). Primer pair positions are relative to the <em>HBE</em> gene transcription start site at position 1; they are aligned with a map of the locus shown below the graphs. Shaded regions of the graphs correspond to the locus control region (LTR promoter to downstream of LCR HS1) and the δβ domain (δβ intergenic promoter to 3′HS1). Map features: red arrowheads, globin genes; pink rectangle, <em>β</em>-like pseudogene; white arrowheads, olfactory receptor genes (<em>HOR</em>, human olfactory receptor); vertical blue lines, hypersensitive sites; blue arrows, intergenic transcription start sites; LTR, long terminal repeat; δβ, δβ intergenic promoter.</p>
<span>doi:10.1371/journal.pone.0000630.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As in the transgenic mice, there are clearly detectable domains of active histone modifications in the LCR and <em>δβ</em> domains indicating that in general the adult <em>β</em>-globin transgene locus has a domain structure similar to the endogenous <em>β</em>-globin locus. However, notable exceptions are the increase in active histone modifications in the region upstream of the <em>δβ</em> promoter to the <em>β</em>-like pseudogene region. This is apparent for both H3K4me2 and H3K4me3 as well as H3ac and H4ac. This corresponds roughly to the areas of antisense intergenic transcription seen in transgenic mice (unpublished observations). Antisense transcription in this region has also been detected in primary erythroid precursor cells<a href="#pone.0000630-Haussecker1">[25]</a>. Also noteworthy is the fact that the εγ domain and region upstream of the <em>β</em>-globin locus containing the olfactory receptor gene cluster have low levels of acetylated histones.</p>


<h4>Intergenic transcription through the olfactory receptor gene cluster in erythroid cells</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Analysis of ESTs and mRNAs in the vicinity of the <em>β</em>-globin locus has shown a number of spliced non-coding transcripts with 5′ exons far upstream of the <em>β</em>-globin locus and extending into the <em>β</em>-globin locus<a href="#pone.0000630-Xiang1">[36]</a>. Many of these transcripts initiate within a 50 bp region within an LTR-like element located just downstream of the <em>UBQLN3</em> gene, located approximately 236 kb upstream of the <em>HBE</em> gene (<a href="#pone-0000630-g005">Figure 5A and B</a>). We decided to investigate transcription across this region in greater detail using RNA FISH.</p>
<div class="figure" id="pone-0000630-g005"><div class="img"><a name="pone-0000630-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Non-coding transcripts originate 230 kb upstream of the <em>β</em>-globin locus.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) A schematic representation of the human <em>β</em>-globin locus. Red arrowheads, globin genes; pink rectangle, <em>β</em>-like pseudogene; white arrowheads, olfactory receptor genes; black arrowhead, <em>UBQLN3</em> gene; vertical blue lines, hypersensitive sites; blue arrows, intergenic transcription start sites; LTR, long terminal repeats; δβ, δβ intergenic promoter; coordinates in kilobases (kb) are relative to the <em>HBE</em> gene transcription start site at position 1. (B) ESTs originating from the LTR promoter 235 kb upstream of the <em>ε</em> gene. Intervening spliced regions are depicted along with the location of exons. Coordinates (in base pairs) of the start sites (arrows) are indicated; NCBI Nucleotide database accession numbers are shown on the right. (C) RNA FISH analysis of intergenic transcription in the 5′ olfactory receptor gene cluster region in human primary erythroid precursor cells. Intergenic transcription signals are detected in green, <em>HBB</em> signals are detected in red, DAPI staining of nuclear DNA is in blue. Examples for 3 individual intergenic probes and one probe cocktail are shown, as indicated; for probe labels and positions see (D) and <a href="#s4">Materials and Methods</a>; scale bar, 5 µm. Each intergenic probe detects signals only in a proportion of all <em>β</em>-globin loci in the cell population. The percentage of cells with intergenic signals was scored for individual probes or cocktail; results from the quantitative analysis are presented in (D). (D) Bar plot representing percentages of cells with intergenic signals. Bar positions on the x axis correspond to probe positions and are aligned with the map below the graph; coordinates in kilobases (kb) are relative to the <em>HBE</em> gene transcription start site at position 1. Probes are depicted as green boxes below the map. Probe cocktails are indicated by open boxes. Probes from 5′ to 3′: 5′UBQLN3, 3′UBQLN3, 5′ORG14, 3′ORG14, 3′ORG8, 3′ORG7A, 3′ORG7B, 5′ORG6, HS5, HS3, 3′ε, 5′γ, 5′δ, and 5′β. Map features as in (A). Relatively high levels of intergenic transcription are detected in the <em>β</em>-globin LCR and the active δβ domain.</p>
<span>doi:10.1371/journal.pone.0000630.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We performed double-label RNA FISH in human primary cultured erythroid cells obtained from the two-step liquid culture system described above. We used single-stranded <em>HBB</em> intron probes and single-stranded probes to various regions in the <em>ORG</em> cluster and <em>β</em>-globin locus (<a href="#pone-0000630-g005">Figure 5C</a>). <em>HBB</em> intron probes detect gene transcription signals at 90–95% of <em>β</em>-globin loci in erythroid cells, and therefore serve as an excellent internal control to identify <em>HBB</em> expressing erythroid cells and to mark the position of the <em>β</em>-globin locus in the nucleus. RNA FISH with probes immediately 5′ or 3′ of the <em>UBQULN3</em> gene, upstream of the -236 LTR, detect little or no transcription signals (<a href="#pone-0000630-g005">Figure 5D</a>). In contrast, probes to the region immediately downstream of the -236 LTR detects significant sense transcripts (7% of loci; p&lt;0.05) (<a href="#pone-0000630-g005">Figure 5D</a>). Similar percentages of positive loci are detected with other probes in the <em>ORG</em> region. These results show that the most 5′ transcription start site of sense transcription in the <em>ORG</em> cluster is located in the vicinity of the -236 LTR element in primary erythroid cells in agreement with the results of Xiang <em>et al</em>.<a href="#pone.0000630-Xiang1">[36]</a> and EST databases. Probes in the LCR and δβ sub-domains detect intergenic transcript signals in three- to five-fold more loci (16–20% versus 4–8%) than the <em>ORG</em> probes (<a href="#pone-0000630-g005">Figure 5D</a>), consistent with our previous data in transgenic mice<a href="#pone.0000630-Gribnau1">[16]</a> and the RT-PCR data above showing higher levels of RNA transcripts in these regions. Probes in the εγ domain detect signals at approximately 6% of loci in adult erythroid cells, consistent with our earlier observations in transgenic mice. These results show that the percentage of loci in which transcription is occurring in the <em>ORG</em> cluster and εγ domains is low while probes in the active domains (LCR and δβ) detect transcription at significantly more loci. Thus, domains of high or frequent intergenic transcription in the human <em>β</em>-globin locus correlate strongly with chromatin domains of highly modified chromatin.</p>


<h4>Cell cycle specificity of transcribed domains in the human <em>β</em>-globin locus</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We previously observed that intergenic transcription is cell-cycle regulated occurring predominantly in G1 phase, but also with a minority of cells showing signals in S phase of the cell cycle<a href="#pone.0000630-Gribnau1">[16]</a>. We had previously noted transcription in the <em>ORG</em> cluster in erythroid cells derived from human cord blood using RNA FISH probes homologous to a region upstream of the LCR (Gribnau and Fraser, unpublished). However, the transcription signals in this region were unusual. Only a small fraction of cells had signals and many of them appeared as doublets suggesting that the region had been duplicated, indicating that transcription in this area occurs preferentially in S-phase cells<a href="#pone.0000630-Gribnau1">[16]</a>. Therefore, we considered the possibility that not only intergenic transcription occurs during restricted stages of the cell cycle, but that different regions are transcribed at different stages. We were interested to determine the cell-cycle timing of transcripts in the upstream <em>ORG</em> region compared to the sub-domains of the <em>β</em>-globin locus. We used PCNA immuno-staining to mark cells in S phase in conjunction with RNA FISH<a href="#pone.0000630-Gribnau1">[16]</a> with intergenic probes in primary cultured human erythroid cells to determine whether these transcripts occur predominantly in S phase or in non-S-phase cells (<a href="#pone-0000630-g006">Figure 6A</a>). The data show that transcription throughout the <em>ORG</em> region occurs predominantly in PCNA positive, S-phase cells (<a href="#pone-0000630-g006">Figure 6B</a>). The percentage of signals occurring in S-phase ranges from 64 to 72 in the <em>ORG</em> region upstream of the <em>β</em>-globin locus. The timing of intergenic transcription in the LCR and δβ sub-domains is markedly different. The majority of signals in these active domains occur in non-S-phase cells. There are still a small percentage of cell nuclei with signals in S-phase nuclei in the active domains and these levels are comparable to the percentage of nuclei with S-phase signals in the <em>ORG</em> region. Signals in the εγ sub-domain occur at nearly equal frequencies in S- and non-S-phase nuclei (<a href="#pone-0000630-g006">Figure 6B</a>). These results suggest the possibility that a large transcript that initiates at the -236 LTR element continues through the entire <em>ORG</em> region and into the <em>β</em>-globin locus in S-phase cells. The existence of these types of transcripts is supported by and consistent with EST data. The increased transcription of the LCR and δβ sub-domains in non-S-phase nuclei shows that the majority of transcription in these regions is controlled independently of transcription in the <em>ORG</em> region and εγ sub-domain. Furthermore these results show that the majority of LCR and δβ sub-domain transcripts are not contiguous with each other or the <em>ORG</em> and εγ sub-domain transcripts. We noted that the ratio of G1 to S phase intergenic transcription is slightly higher in the εγ domain compared to the <em>ORG</em> cluster. We previously showed that a small percentage of cells in the human cell cultures are not fully differentiated, still transcribe the <em>HBG</em> genes and would be expected to have an active εγ domain <a href="#pone.0000630-Chakalova1">[6]</a>. This may account for the slightly increased ratio of G1/S intergenic transcription in this region.</p>
<div class="figure" id="pone-0000630-g006"><div class="img"><a name="pone-0000630-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000630" data-uri="info:doi/10.1371/journal.pone.0000630.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000630.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000630.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000630.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000630.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Upstream transcripts occur predominantly in S phase.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Cell cycle analysis of intergenic transcription in the 5′ olfactory receptor gene cluster region in human primary erythroid precursor cells by RNA immuno-FISH. Intergenic transcription signals are detected in green, <em>HBB</em> transcript signals are red, PCNA is pseudocoloured in blue; scale bar, 2 µm. Shown are single focal planes from several representative PCNA-positive cells and one PCNA-negative cell with various probe combinations. For all probe positions, see <a href="#s4">Materials and Methods</a>. PCNA-negative cell (bottom right): the brightness of the blue channel is enhanced to demonstrate the absence of PCNA staining. (B) Percentage of PCNA-positive intergenic signal-containing cells was scored for individual probes or the indicated probe cocktails. Bar plot representing percentages of cells with intergenic signals (as in <a href="#pone-0000630-g005">Figure 5D</a>) with the proportion of PCNA-positive cells (S phase) shown in blue. The percentage of intergenic signal-containing S-phase cells is relatively uniform throughout the 5′ <em>ORG</em> and <em>β</em>-globin loci (average 3.3% marked by dashed line). Bar positions on the x axis correspond to probe positions and are aligned with the map below the graph; coordinates in kilobases (kb) are relative to the <em>HBE</em> gene transcription start site at position 1. Probes are depicted as green boxes below the map. Probe cocktails are indicated by open boxes. Probes from 5′ to 3′: 3′ORG14, 3′ORG8, 3′ORG7A, 3′ORG7B, 5′ORG6, HS5, HS3, 3′ε, 5′γ, 5′δ, and 5′β. Map features: red arrowheads, globin genes; pink rectangle, <em>β</em>-like pseudogene; white arrowheads, olfactory receptor genes; black arrowhead, <em>UBQLN3</em> gene; vertical blue lines, hypersensitive sites; blue arrows, intergenic transcription start sites; LTR, long terminal repeats; δβ, δβ intergenic promoter. The high level of intergenic transcription in the <em>β</em>-globin LCR and the active δβ domain is accounted for by higher percentages of non-S-phase cells with intergenic signals.</p>
<span>doi:10.1371/journal.pone.0000630.g006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>PCNA localization patterns change as S phase progresses permitting the discrimination of early, middle and late S-phase stages<a href="#pone.0000630-Gribnau1">[16]</a>, <a href="#pone.0000630-Humbert1">[37]</a>. We noted that with probes to the most 5′ region of the transcribed <em>ORG</em> domain we observed a higher proportion of cells in early S phase compared to cells with middle or late PCNA patterns. Probes located more 3′ of the -236 LTR in the <em>ORG</em> region detected signals in progressively later S-phase cells indicating an S phase-specific wave of transcription through the upstream <em>ORG</em> cluster, consistent with a large continuous transcript into the <em>β</em>-globin locus. Combined with the EST data, these results suggest that large transcripts initiate from the LTR promoter early in S phase, immediately after the locus has replicated and process through the entire <em>ORG</em> cluster, LCR and into the <em>β</em>-globin locus. These transcripts are distinct from the bulk of transcription that occurs in the active LCR and δβ sub-domains that occurs predominantly in G1 phase. These results link high levels of non-S-phase intergenic transcription with chromatin domains that are highly enriched in active histone modifications.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have shown that the human <em>β</em>-globin locus is composed of multiple chromatin sub-domains that are developmentally regulated. The individual sub-domains can be distinguished by differential general sensitivity to DNase I, intergenic transcription and active histone modifications primarily to H3. Our data show that H3K4 di- and tri-methylation and H3 hyperacetylation clearly mark domains with high levels of G1 phase intergenic transcription. H4 hyperacetylation also marks the active domains but appears at moderate levels in the inactive sub-domains in embryonic and adult erythroid cells in transgenic mice. We show that a very large transcript that initiates approximately 236 kb upstream of the human <em>β</em>-globin locus and extends through the locus is produced primarily in S phase. Transcription of the active sub-domains, containing the LCR and the active genes at each developmental stage, occurs primarily in G1 phase. These results correlate high levels of G1 phase-specific intergenic transcription with high levels of active histone modifications, namely, H3K4 di- and tri-methylation, and H3 acetylation, across the transcribed sub-domains, suggesting that the timing of intergenic transcription and/or the level of transcription may play a role in propagating these marks. There are clear indications that the elongating form of RNA polymerase II (RNAPII) is associated with histone modifying and chromatin remodelling activities<a href="#pone.0000630-Sims1">[38]</a>, which could account for the modified domains we observe in areas of high intergenic transcription. Additional possibilities are suggested by recent studies indicating that transcription by RNAPII outside of S phase could promote replication independent histone exchange leading to the deposition of variant histones such as H3.3<a href="#pone.0000630-Henikoff1">[39]</a>. Although non-genic transcription is widespread in vertebrate genomes it is unlikely that all of this transcription leads to deposition of variant histones and active histone modifications. Controlling the timing of intergenic transcription may be a strategy adopted to modify specified domains.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In adult erythroid cells the LCR and the adult <em>β</em>-globin genes engage in long-range interactions, essentially forming a chromatin loop<a href="#pone.0000630-Carter1">[40]</a>, <a href="#pone.0000630-Tolhuis1">[41]</a>. The mechanism by which these distal sequences find each other has been the subject of intense speculation and debate. Although a loop is formed, a “looping mechanism” implies that the two sequences find each other via diffusion-mediated random collisions. The discovery of intergenic transcripts initiating in the LCR and proceeding in the direction of the globin genes have suggested a tracking mechanism of enhancer-gene contact in which the LCR and associated factors including RNAPII track through the locus in search of an activatable gene promoter. Our data indicate that the majority of LCR transcripts are not contiguous with δβ domain transcripts, implying that a continuous scanning mechanism is unlikely. However, a low level of transcription through the εγ domain may be contiguous with LCR and δβ domain transcripts. If the chromatin loop was established through a scanning mechanism one might expect that minimally it would need to operate at least two times per cell cycle. First, when cells exit mitosis, long-range contacts may need to be re-established after decondensation of the inactive mitotic chromosome structure prior to gene transcription. Second, after DNA replication, which is known to temporarily disrupt transcription, long-range contacts may need to be re-established on individual daughter alleles in early S-phase. Although we do not know the precise timing of G1-specific intergenic transcripts in the εγ domain (i.e. early or late G1) intergenic transcripts are clearly present there in early S-phase just after replication of the locus, suggesting that a limited tracking mechanism of LCR-gene interaction would be compatible with our data. However, a diffusion-mediated looping mechanism of LCR-gene contact is not ruled out by our results.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>What could be the role of the very long transcripts that initiate at −236 kb upstream of the locus? The null hypothesis is that it has no role at all and is merely the result of transcriptional noise. However, the transcript appears to be very tightly regulated occurring predominantly in S phase, strand-specific and apparently initiating from a single discrete site. It is possible that histones in the <em>ORG</em> and εγ domain regions are modified by passage of the RNAPII complex as part of decondensation of the globin locus, but that active marks in these regions are rapidly or more thoroughly turned over due to the rarity and very low level at which these transcripts occur. We estimate that there may be as little as a single RNAPII complex transcribing these regions in loci with a positive RNA FISH signal. ChIP analysis on a sorted or synchronized population of early S-phase erythroid cells could address the question of whether this low level transcription is linked to transient histone modifications.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Another potential role of intergenic transcription is to facilitate the re-entry of the globin locus into a transcription factory after DNA replication. DNA replication takes place in replication factories which form in proximity to transcription factories in early S phase<a href="#pone.0000630-Wansink1">[42]</a>. In the latter case active genes may need to disengage from RNAPII factories to shuttle to a nearby replication factory. Highly-expressed genes like globin, which are nearly always associated with transcription factories in expressing cells<a href="#pone.0000630-Osborne1">[43]</a>, <a href="#pone.0000630-Ragoczy1">[44]</a>, could be reeled back into a transcription factory after replication by the processive action of RNAPII localized in a factory<a href="#pone.0000630-Chakalova2">[45]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In summary, our data strengthen the link between intergenic transcription and modification of histones over wide chromatin domains, and suggest that developmental regulation of expression the human <em>β</em>-globin genes occurs in part through epigenetic changes to chromatin structural domains.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Animals and Human primary cell culture</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Experimental procedures were conducted in compliance with an animal protocol approved by the home office and local ethical review committee. Transgenic mice homozygous for a wild type 150 kb human <em>β</em>-globin locus YAC were previously described by Tanimoto <em>et al</em>.<a href="#pone.0000630-Tanimoto1">[27]</a>. Adult mice were made anemic as previously described<a href="#pone.0000630-Dickerman1">[46]</a>. Human peripheral blood from healthy individuals was obtained from a local blood bank, prepared and cultured as described in Chakalova <em>et al</em>.<a href="#pone.0000630-Chakalova1">[6]</a>, and harvested on day 2 post-hemoglobinization.</p>


<h4>RNA-FISH</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>RNA FISH was performed as previously described<a href="#pone.0000630-Gribnau1">[16]</a>. The following probes were used to visualise intergenic transcription signals (coordinates relative to the <em>ε</em> gene transcription start site at position +1 are given in brackets): 5′UBQLN3 (-243679 to -241290), 3′UBQLN3 (-236996 to -236060), 5′ORG14 (-235344 to -232998), 3′ORG14 (-219479 to -221543), 3′ORG8 (-131754 to -130106), 3′ORG7A (-113405 to -111608), 3′ORG7B (-100614 to -99051), 5′ORG6 (-94181 to -92378), 5′ORG3 (-57154 to -55376), 3′ORG1 (-27315 to -25831), HS5 (-22844 to -19501), HS3 (-16236 to -14330), 3′ε (3126 to 5919), 5′γ (10881 to 12866), 5′δ (32851 to 35239), 5′β (38504 to 42384). Digoxigenin-labeled single-stranded DNA probes were generated from the cloned sequences as described<a href="#pone.0000630-Chakalova3">[47]</a> and detected with FITC-labeled antibodies. Intergenic probes were used in combination with a probe to the <em>ΗΒΒ</em> gene intron 2; in double-label RNA FISH experiments, a cocktail of four dinitrophenol-labeled <em>ΗΒΒ</em> intron 2 30-mer oligonucleotides were used (described in <a href="#pone.0000630-Chakalova1">[6]</a>), and detected with Texas Red-labeled antibodies; for three-colour cell cycle analysis, a longer biotin-labeled single-stranded DNA probe (coordinates 43126 to 44043) was applied and detected with Alexa Fluor 350-labeled Streptavidin (Molecular Probes Invitrogen). In all cases, slides were examined on an Olympus BX41 epifluorescence microscope. To determine the percentage of loci associated with intergenic transcription signals, a minimum of 100 loci were counted for each data point. PCNA was visualised with a primary mouse monoclonal anti-PCNA antibody (Santa Cruz) followed by Texas Red-conjugated goat anti-mouse secondary antibody (Jackson ImmunoResearch). To determine the percentage of PCNA-positive intergenic transcription signal-containing cells, cells with intergenic signals were identified, and digital images were captured for each of the three colour channels to record gene and intergenic FISH signals as well as PCNA patterns. 40 cells were analysed for each data point.</p>


<h4>Quantitative RT-PCR</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Total RNA was extracted from adult anemic spleen and day 11 embryonic blood of homozygous transgenic mice. RNA was isolated according to the manufacturer's instructions from frozen cell pellets using 4 ml of RNA-Bee (AMS Biotechnology) per 10<sup>7</sup> cells. 1 µg of total RNA was mixed with Random hexanucleotide mix (5 ng/µl final concentration, Promega) and RNase-free water in a final volume of 20 µl. Reverse transcription was carried out with Superscript II reverse transcriptase (Invitrogen) following the protocol provided by the manufacturer in the presence of 2 u/µl RNasin. RT negative controls in which the reverse transcriptase enzyme omitted were set up in parallel. Real-time PCR was performed with an ABI PRISM 7000 Sequence Detection System using SYBR green PCR Master Mix (Applied Biosystems). 2 µl of cDNA were used in real-time PCR, in duplicate, with the following thermal cycling conditions: 50°C for 2 minutes and 95°C for 5 minutes, followed by 40 cycles of 95°C for 30 seconds and 62°C for 2 minutes. For primer sequences, see <a href="#pone.0000630.s001">Table S1</a> in Supporting Information. The relative amount of cDNA amplification for each primer pair was calculated by comparing to transgenic genomic DNA standards. Data were normalised to the most 5′ data point in the olfactory receptor region, which shows low level of transcription.</p>


<h4>Chromatin immunoprecipitation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Histone modification profiles across the human <em>β</em>-globin locus were assessed by native chromatin immunoprecipitation (NChIP)<a href="#pone.0000630-ONeill1">[48]</a>. Single-cell suspensions of erythroid or mouse embryo fibroblast cells were resuspended to 2×10<sup>7</sup> cells/ml in ice cold 1×RSB (10 mM Tris-HCl, pH7.5, 10 mM NaCl, 3 mM MgCl<sub>2</sub>), 0.1% Triton X-100, 0.5 mM DTT, 0.1 M sucrose, 0.1 mM PMSF (Sigma), 5 mM Na butyrate and 1/50<sup>th</sup> volume protease inhibitor cocktail (Sigma). The cells were dounced in a cold glass homogeniser and diluted with an equal volume of the same buffer with 0.25 M sucrose. The suspension was layered onto a sucrose cushion consisting of a half volume of 0.33 M sucrose, 5 mM MgCl<sub>2</sub>, 10 mM Tris pH8, 0.5 mM DTT, 0.1 mM PMSF, 5 mM Na butyrate. This was then centrifuged at 800 x g for 5 min at 4<sup>o</sup>C to obtain the nuclear pellet.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>After preparing nuclei, chromatin was digested with micrococcal nuclease generating DNA predominantly mononucleosomal in length. NChIP was carried out using the following rabbit polyclonal antibodies: anti-trimethyl-histone H3 (K4) (Abcam), anti-dimethyl-histone H3 (K4), anti-acetyl-histone H3 (K9/K14), anti-acetyl-histone H4 (K5/18/12/16), ChIP-grade anti-hyperacetylated histone H4, penta lysine (all from Upstate Biotechnology)</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>DNA from the Input chromatin fractions was quantified by standard spectrophotometry. DNA concentrations in the antibody-bound fractions were determined by PicoGreen (Invitrogen) fluorescence quantification, using Input DNA for standards. Real-time PCR was performed in an ABI PRISM 7000 Sequence Detection System using SYBR green PCR Master Mix (Applied Biosystems). All PCR reactions were carried out in duplicate on 3 ng DNA at 50°C for 2 minutes and 95°C for 5 minutes, followed by 40 cycles of 95°C for 30 seconds and 62°C for 2 minutes. For primer sequences, see <a href="#pone.0000630.s001">Table S1</a> in Supporting Information. The ratio of Bound to Input DNA was calculated using the comparative C<sub>T</sub> method, Bound/Input = 2<sup>(Input Ct – Bound Ct)</sup><a href="#pone.0000630-Livak1">[49]</a>. Data were normalised to the most 5′ data point in the olfactory receptor region, which shows low enrichment for all antibodies. The promoter of the ubiquitously expressed mouse <em>Actb</em> (<em>β</em>-actin) gene was used as an internal positive control (Supporting Information <a href="#pone.0000630.s002">Figure S1</a>). The ChIP experiments were repeated several times with similar results and domain patterns. Shown are the results of a single representative ChIP experiment.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000630"></div><a name="pone.0000630.s001" id="pone.0000630.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000630.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Histone modifications at the mouse Actb (beta actin) promoter. Histone modifications were assayed by ChIP in 264W transgenic mice at two developmental stages as a positive control for the ChIP procedure. The ChIP material is the same as in <a href="#pone-0000630-g002">Figures 2</a> and <a href="#pone-0000630-g003">3</a> (Embryonic and Adult, respectively). Briefly, chromatin from erythroid cells was immunoprecipitated with antibodies specific for trimethylated lysine 4 of histone H3 (H3K4me3), dimethylated lysine 4 of histone H3 (H3K4me2), acetylated histone H3 (K9/14, H3ac), and acetylated histone H4 (two different antibodies: ChIP grade antibody, K5/18/12/16, H4acCh; penta lysine, H4acP). The fold-enrichment of antibody-bound sequences was analysed by real-time PCR (Bound/Input) using a primer pair in the mouse Actb promoter.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000630.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.19 MB TIF)</p>
<a name="pone.0000630.s002" id="pone.0000630.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000630.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Primer pairs used for real-time PCR. Primer pairs used to amplify sequences in the human HBB gene cluster and flanking regions; primer names reflect the position of the amplicon relative to the HBE gene transcription start site at position +1 or known genomic elements. Actb Pr is the mouse β-actin promoter region amplicon. For, forward primer; Rev, reverse primer.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000630.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Drs. E. Debrand and A. Horton for helpful advice and assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PF LC JM JM. Performed the experiments: LC JM JM. Analyzed the data: LC JM JM. Contributed reagents/materials/analysis tools: DE KT CO LO BG TK. Wrote the paper: PF LC JM JM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000630-Carninci1" id="pone.0000630-Carninci1"></a>Carninci P, Kasukawa T, Katayama S, Gough J, Frith MC, et al.  (2005) The transcriptional landscape of the mammalian genome. Science  309: 1559–1563.  <ul class="find" data-citedArticleID="967644" data-doi="10.1126/science.1112014"><li><a href="http://dx.doi.org/10.1126/science.1112014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+landscape+of+the+mammalian+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+landscape+of+the+mammalian+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000630-Cheng1" id="pone.0000630-Cheng1"></a>Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, et al.  (2005) Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution. Science  308: 1149–1154.  <ul class="find" data-citedArticleID="967654" data-doi="10.1126/science.1108625"><li><a href="http://dx.doi.org/10.1126/science.1108625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000630-Kampa1" id="pone.0000630-Kampa1"></a>Kampa D, Cheng J, Kapranov P, Yamanaka M, Brubaker S, et al.  (2004) Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22. Genome Res  14: 331–342.  <ul class="find" data-citedArticleID="967680" data-doi="10.1101/gr.2094104"><li><a href="http://dx.doi.org/10.1101/gr.2094104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+RNAs+identified+from+an+in-depth+analysis+of+the+transcriptome+of+human+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+RNAs+identified+from+an+in-depth+analysis+of+the+transcriptome+of+human+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000630-Kapranov1" id="pone.0000630-Kapranov1"></a>Kapranov P, Cawley SE, Drenkow J, Bekiranov S, Strausberg RL, et al.  (2002) Large-scale transcriptional activity in chromosomes 21 and 22. Science  296: 916–919.  <ul class="find" data-citedArticleID="967682" data-doi="10.1126/science.1068597"><li><a href="http://dx.doi.org/10.1126/science.1068597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+transcriptional+activity+in+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+transcriptional+activity+in+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000630-Rinn1" id="pone.0000630-Rinn1"></a>Rinn JL, Euskirchen G, Bertone P, Martone R, Luscombe NM, et al.  (2003) The transcriptional activity of human Chromosome 22. Genes Dev  17: 529–540.  <ul class="find" data-citedArticleID="967704" data-doi="10.1101/gad.1055203"><li><a href="http://dx.doi.org/10.1101/gad.1055203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+activity+of+human+Chromosome+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+activity+of+human+Chromosome+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000630-Chakalova1" id="pone.0000630-Chakalova1"></a>Chakalova L, Osborne CS, Dai YF, Goyenechea B, Metaxotou-Mavromati A, et al.  (2005) The Corfu deltabeta thalassemia deletion disrupts gamma-globin gene silencing and reveals post-transcriptional regulation of HbF expression. Blood  105: 2154–2160.  <ul class="find" data-citedArticleID="967648" data-doi="10.1182/blood-2003-11-4069"><li><a href="http://dx.doi.org/10.1182/blood-2003-11-4069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Corfu+deltabeta+thalassemia+deletion+disrupts+gamma-globin+gene+silencing+and+reveals+post-transcriptional+regulation+of+HbF+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Corfu+deltabeta+thalassemia+deletion+disrupts+gamma-globin+gene+silencing+and+reveals+post-transcriptional+regulation+of+HbF+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000630-Drewell1" id="pone.0000630-Drewell1"></a>Drewell RA, Bae E, Burr J, Lewis EB (2002) Transcription defines the embryonic domains of cis-regulatory activity at the Drosophila bithorax complex. Proceedings of the National Academy of Sciences of the United States of America  99: 16853–16858.  <ul class="find" data-citedArticleID="967658" data-doi="10.1073/pnas.222671199"><li><a href="http://dx.doi.org/10.1073/pnas.222671199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+defines+the+embryonic+domains+of+cis-regulatory+activity+at+the+Drosophila+bithorax+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+defines+the+embryonic+domains+of+cis-regulatory+activity+at+the+Drosophila+bithorax+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000630-Schmitt1" id="pone.0000630-Schmitt1"></a>Schmitt S, Prestel M, Paro R (2005) Intergenic transcription through a polycomb group response element counteracts silencing. Genes Dev  19: 697–708.  <ul class="find" data-citedArticleID="967710" data-doi="10.1101/gad.326205"><li><a href="http://dx.doi.org/10.1101/gad.326205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intergenic+transcription+through+a+polycomb+group+response+element+counteracts+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intergenic+transcription+through+a+polycomb+group+response+element+counteracts+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000630-Masternak1" id="pone.0000630-Masternak1"></a>Masternak K, Peyraud N, Krawczyk M, Barras E, Reith W (2003) Chromatin remodeling and extragenic transcription at the MHC class II locus control region. Nature Immunology  4: 132–137.  <ul class="find" data-citedArticleID="967692" data-doi="10.1038/ni883"><li><a href="http://dx.doi.org/10.1038/ni883" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+remodeling+and+extragenic+transcription+at+the+MHC+class+II+locus+control+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+remodeling+and+extragenic+transcription+at+the+MHC+class+II+locus+control+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000630-Ho1" id="pone.0000630-Ho1"></a>Ho Y, Elefant F, Liebhaber SA, Cooke NE (2006) Locus control region transcription plays an active role in long-range gene activation. Mol Cell  23: 365–375.  <ul class="find" data-citedArticleID="967674" data-doi="10.1016/j.molcel.2006.05.041"><li><a href="http://dx.doi.org/10.1016/j.molcel.2006.05.041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locus+control+region+transcription+plays+an+active+role+in+long-range+gene+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locus+control+region+transcription+plays+an+active+role+in+long-range+gene+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000630-Rogan1" id="pone.0000630-Rogan1"></a>Rogan DF, Cousins DJ, Santangelo S, Ioannou PA, Antoniou M, et al.  (2004) Analysis of intergenic transcription in the human IL-4/IL-13 gene cluster. Proc Natl Acad Sci U S A  101: 2446–2451.  <ul class="find" data-citedArticleID="967706" data-doi="10.1073/pnas.0308327100"><li><a href="http://dx.doi.org/10.1073/pnas.0308327100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+intergenic+transcription+in+the+human+IL-4%2FIL-13+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+intergenic+transcription+in+the+human+IL-4%2FIL-13+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000630-Jones1" id="pone.0000630-Jones1"></a>Jones EA, Flavell RA (2005) Distal enhancer elements transcribe intergenic RNA in the IL-10 family gene cluster. J Immunol  175: 7437–7446.  <ul class="find" data-citedArticleID="967678"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Distal+enhancer+elements+transcribe+intergenic+RNA+in+the+IL-10+family+gene+cluster.&amp;auth=&amp;atitle=Distal+enhancer+elements+transcribe+intergenic+RNA+in+the+IL-10+family+gene+cluster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distal+enhancer+elements+transcribe+intergenic+RNA+in+the+IL-10+family+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distal+enhancer+elements+transcribe+intergenic+RNA+in+the+IL-10+family+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000630-Bolland1" id="pone.0000630-Bolland1"></a>Bolland DJ, Wood AL, Johnston CM, Bunting SF, Morgan G, et al.  (2004) Antisense intergenic transcription in V(D)J recombination. Nat Immunol  5: 630–637.  <ul class="find" data-citedArticleID="967638" data-doi="10.1038/ni1068"><li><a href="http://dx.doi.org/10.1038/ni1068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+intergenic+transcription+in+V%28D%29J+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+intergenic+transcription+in+V%28D%29J+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000630-Ogawa1" id="pone.0000630-Ogawa1"></a>Ogawa Y, Lee JT (2003) Xite, X-inactivation Intergenic transcription elements that regulate the probability of choice. Molecular Cell  11: 731–743.  <ul class="find" data-citedArticleID="967696" data-doi="10.1016/s1097-2765(03)00063-7"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00063-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Xite%2C+X-inactivation+Intergenic+transcription+elements+that+regulate+the+probability+of+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Xite%2C+X-inactivation+Intergenic+transcription+elements+that+regulate+the+probability+of+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000630-Ashe1" id="pone.0000630-Ashe1"></a>Ashe HL, Monks J, Wijgerde M, Fraser P, Proudfoot NJ (1997) Intergenic transcription and transinduction of the human beta-globin locus. Genes &amp; Development  11: 2494–2509.  <ul class="find" data-citedArticleID="967632" data-doi="10.1101/gad.11.19.2494"><li><a href="http://dx.doi.org/10.1101/gad.11.19.2494" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intergenic+transcription+and+transinduction+of+the+human+beta-globin+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intergenic+transcription+and+transinduction+of+the+human+beta-globin+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000630-Gribnau1" id="pone.0000630-Gribnau1"></a>Gribnau J, Diderich K, Pruzina S, Calzolari R, Fraser P (2000) Intergenic transcription and developmental remodeling of chromatin subdomains in the human beta-globin locus. Mol Cell  5: 377–386.  <ul class="find" data-citedArticleID="967668" data-doi="10.1016/s1097-2765(00)80432-3"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80432-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intergenic+transcription+and+developmental+remodeling+of+chromatin+subdomains+in+the+human+beta-globin+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intergenic+transcription+and+developmental+remodeling+of+chromatin+subdomains+in+the+human+beta-globin+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000630-Calzolari1" id="pone.0000630-Calzolari1"></a>Calzolari R, McMorrow T, Yannoutsos N, Langeveld A, Grosveld F (1999) Deletion of a region that is a candidate for the difference between the deletion forms of hereditary persistence of fetal hemoglobin and deltabeta-thalassemia affects beta- but not gamma-globin gene expression. Embo J  18: 949–958.  <ul class="find" data-citedArticleID="967642" data-doi="10.1093/emboj/18.4.949"><li><a href="http://dx.doi.org/10.1093/emboj/18.4.949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletion+of+a+region+that+is+a+candidate+for+the+difference+between+the+deletion+forms+of+hereditary+persistence+of+fetal+hemoglobin+and+deltabeta-thalassemia+affects+beta-+but+not+gamma-globin+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletion+of+a+region+that+is+a+candidate+for+the+difference+between+the+deletion+forms+of+hereditary+persistence+of+fetal+hemoglobin+and+deltabeta-thalassemia+affects+beta-+but+not+gamma-globin+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000630-Bernstein1" id="pone.0000630-Bernstein1"></a>Bernstein BE, Humphrey EL, Erlich RL, Schneider R, Bouman P, et al.  (2002) Methylation of histone H3 Lys 4 in coding regions of active genes. Proc Natl Acad Sci U S A  99: 8695–8700.  <ul class="find" data-citedArticleID="967634" data-doi="10.1073/pnas.082249499"><li><a href="http://dx.doi.org/10.1073/pnas.082249499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation+of+histone+H3+Lys+4+in+coding+regions+of+active+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation+of+histone+H3+Lys+4+in+coding+regions+of+active+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000630-Bernstein2" id="pone.0000630-Bernstein2"></a>Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, et al.  (2005) Genomic maps and comparative analysis of histone modifications in human and mouse. Cell  120: 169–181.  <ul class="find" data-citedArticleID="967636" data-doi="10.1016/j.cell.2005.01.001"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000630-Schubeler1" id="pone.0000630-Schubeler1"></a>Schubeler D, MacAlpine DM, Scalzo D, Wirbelauer C, Kooperberg C, et al.  (2004) The histone modification pattern of active genes revealed through genome-wide chromatin analysis of a higher eukaryote. Genes &amp; Development  18: 1263–1271.  <ul class="find" data-citedArticleID="967712" data-doi="10.1101/gad.1198204"><li><a href="http://dx.doi.org/10.1101/gad.1198204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+histone+modification+pattern+of+active+genes+revealed+through+genome-wide+chromatin+analysis+of+a+higher+eukaryote." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+histone+modification+pattern+of+active+genes+revealed+through+genome-wide+chromatin+analysis+of+a+higher+eukaryote.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000630-Forsberg1" id="pone.0000630-Forsberg1"></a>Forsberg EC, Downs KM, Christensen HM, Im H, Nuzzi PA, et al.  (2000) Developmentally dynamic histone acetylation pattern of a tissue-specific chromatin domain. Proc Natl Acad Sci U S A  97: 14494–14499.  <ul class="find" data-citedArticleID="967664" data-doi="10.1073/pnas.97.26.14494"><li><a href="http://dx.doi.org/10.1073/pnas.97.26.14494" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmentally+dynamic+histone+acetylation+pattern+of+a+tissue-specific+chromatin+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmentally+dynamic+histone+acetylation+pattern+of+a+tissue-specific+chromatin+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000630-Kim1" id="pone.0000630-Kim1"></a>Kim A, Dean A (2004) Developmental stage differences in chromatin subdomains of the beta-globin locus. Proceedings of the National Academy of Sciences of the United States of America  101: 7028–7033.  <ul class="find" data-citedArticleID="967684" data-doi="10.1073/pnas.0307985101"><li><a href="http://dx.doi.org/10.1073/pnas.0307985101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+stage+differences+in+chromatin+subdomains+of+the+beta-globin+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+stage+differences+in+chromatin+subdomains+of+the+beta-globin+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000630-Schubeler2" id="pone.0000630-Schubeler2"></a>Schubeler D, Francastel C, Cimbora DM, Reik A, Martin DI, et al.  (2000) Nuclear localization and histone acetylation: a pathway for chromatin opening and transcriptional activation of the human beta-globin locus. Genes Dev  14: 940–950.  <ul class="find" data-citedArticleID="967714"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Nuclear+localization+and+histone+acetylation%3A+a+pathway+for+chromatin+opening+and+transcriptional+activation+of+the+human+beta-globin+locus.&amp;auth=&amp;atitle=Nuclear+localization+and+histone+acetylation%3A+a+pathway+for+chromatin+opening+and+transcriptional+activation+of+the+human+beta-globin+locus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+localization+and+histone+acetylation%3A+a+pathway+for+chromatin+opening+and+transcriptional+activation+of+the+human+beta-globin+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+localization+and+histone+acetylation%3A+a+pathway+for+chromatin+opening+and+transcriptional+activation+of+the+human+beta-globin+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000630-Schubeler3" id="pone.0000630-Schubeler3"></a>Schubeler D, Groudine M, Bender MA (2001) The murine beta-globin locus control region regulates the rate of transcription but not the hyperacetylation of histones at the active genes. Proceedings of the National Academy of Sciences of the United States of America  98: 11432–11437.  <ul class="find" data-citedArticleID="967716" data-doi="10.1073/pnas.201394698"><li><a href="http://dx.doi.org/10.1073/pnas.201394698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+murine+beta-globin+locus+control+region+regulates+the+rate+of+transcription+but+not+the+hyperacetylation+of+histones+at+the+active+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+murine+beta-globin+locus+control+region+regulates+the+rate+of+transcription+but+not+the+hyperacetylation+of+histones+at+the+active+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000630-Haussecker1" id="pone.0000630-Haussecker1"></a>Haussecker D, Proudfoot NJ (2005) Dicer-dependent turnover of intergenic transcripts from the human beta-globin gene cluster. Mol Cell Biol  25: 9724–9733.  <ul class="find" data-citedArticleID="967670" data-doi="10.1128/mcb.25.21.9724-9733.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.21.9724-9733.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dicer-dependent+turnover+of+intergenic+transcripts+from+the+human+beta-globin+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dicer-dependent+turnover+of+intergenic+transcripts+from+the+human+beta-globin+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000630-Bulger1" id="pone.0000630-Bulger1"></a>Bulger M, Schubeler D, Bender MA, Hamilton J, Farrell CM, et al.  (2003) A complex chromatin landscape revealed by patterns of nuclease sensitivity and histone modification within the mouse beta-globin locus. Mol Cell Biol  23: 5234–5244.  <ul class="find" data-citedArticleID="967640" data-doi="10.1128/mcb.23.15.5234-5244.2003"><li><a href="http://dx.doi.org/10.1128/mcb.23.15.5234-5244.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+complex+chromatin+landscape+revealed+by+patterns+of+nuclease+sensitivity+and+histone+modification+within+the+mouse+beta-globin+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+complex+chromatin+landscape+revealed+by+patterns+of+nuclease+sensitivity+and+histone+modification+within+the+mouse+beta-globin+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000630-Tanimoto1" id="pone.0000630-Tanimoto1"></a>Tanimoto K, Liu Q, Bungert J, Engel JD (1999) Effects of altered gene order or orientation of the locus control region on human beta-globin gene expression in mice. Nature  398: 344–348.  <ul class="find" data-citedArticleID="967720" data-doi="10.1038/18698"><li><a href="http://dx.doi.org/10.1038/18698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+altered+gene+order+or+orientation+of+the+locus+control+region+on+human+beta-globin+gene+expression+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+altered+gene+order+or+orientation+of+the+locus+control+region+on+human+beta-globin+gene+expression+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000630-Fraser1" id="pone.0000630-Fraser1"></a>Fraser P, Pruzina S, Antoniou M, Grosveld F (1993) Each hypersensitive site of the human beta-globin locus control region confers a different developmental pattern of expression on the globin genes. Genes Dev  7: 106–113.  <ul class="find" data-citedArticleID="967666" data-doi="10.1101/gad.7.1.106"><li><a href="http://dx.doi.org/10.1101/gad.7.1.106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Each+hypersensitive+site+of+the+human+beta-globin+locus+control+region+confers+a+different+developmental+pattern+of+expression+on+the+globin+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Each+hypersensitive+site+of+the+human+beta-globin+locus+control+region+confers+a+different+developmental+pattern+of+expression+on+the+globin+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000630-Peterson1" id="pone.0000630-Peterson1"></a>Peterson KR, Clegg CH, Navas PA, Norton EJ, Kimbrough TG, et al.  (1996) Effect of deletion of 5′HS3 or 5′HS2 of the human beta-globin locus control region on the developmental regulation of globin gene expression in beta-globin locus yeast artificial chromosome transgenic mice. Proceedings of the National Academy of Sciences of the United States of America  93: 6605–6609.  <ul class="find" data-citedArticleID="967700" data-doi="10.1073/pnas.93.13.6605"><li><a href="http://dx.doi.org/10.1073/pnas.93.13.6605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+deletion+of+5%26%23x2032%3BHS3+or+5%26%23x2032%3BHS2+of+the+human+beta-globin+locus+control+region+on+the+developmental+regulation+of+globin+gene+expression+in+beta-globin+locus+yeast+artificial+chromosome+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+deletion+of+5%26%23x2032%3BHS3+or+5%26%23x2032%3BHS2+of+the+human+beta-globin+locus+control+region+on+the+developmental+regulation+of+globin+gene+expression+in+beta-globin+locus+yeast+artificial+chromosome+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000630-Leach1" id="pone.0000630-Leach1"></a>Leach KM, Nightingale K, Igarashi K, Levings PP, Engel JD, et al.  (2001) Reconstitution of human beta-globin locus control region hypersensitive sites in the absence of chromatin assembly. Mol Cell Biol  21: 2629–2640.  <ul class="find" data-citedArticleID="967686" data-doi="10.1128/mcb.21.8.2629-2640.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.8.2629-2640.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstitution+of+human+beta-globin+locus+control+region+hypersensitive+sites+in+the+absence+of+chromatin+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstitution+of+human+beta-globin+locus+control+region+hypersensitive+sites+in+the+absence+of+chromatin+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000630-Long1" id="pone.0000630-Long1"></a>Long QM, Bengra C, Li CH, Kutlar F, Tuan D (1998) A long terminal repeat of the human endogenous retrovirus ERV-9 is located in the 5′ boundary area of the human beta-globin locus control region. Genomics  54: 542–555.  <ul class="find" data-citedArticleID="967690" data-doi="10.1006/geno.1998.5608"><li><a href="http://dx.doi.org/10.1006/geno.1998.5608" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+long+terminal+repeat+of+the+human+endogenous+retrovirus+ERV-9+is+located+in+the+5%26%23x2032%3B+boundary+area+of+the+human+beta-globin+locus+control+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+long+terminal+repeat+of+the+human+endogenous+retrovirus+ERV-9+is+located+in+the+5%26%23x2032%3B+boundary+area+of+the+human+beta-globin+locus+control+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000630-Routledge1" id="pone.0000630-Routledge1"></a>Routledge SJ, Proudfoot NJ (2002) Definition of transcriptional promoters in the human beta globin locus control region. J Mol Biol  323: 601–611.  <ul class="find" data-citedArticleID="967708" data-doi="10.1016/s0022-2836(02)01011-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)01011-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+of+transcriptional+promoters+in+the+human+beta+globin+locus+control+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+of+transcriptional+promoters+in+the+human+beta+globin+locus+control+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000630-Tuan1" id="pone.0000630-Tuan1"></a>Tuan D, Kong SM, Hu K (1992) Transcription of the Hypersensitive Site Hs2 Enhancer in Erythroid-Cells. Proceedings of the National Academy of Sciences of the United States of America  89: 11219–11223.  <ul class="find" data-citedArticleID="967724" data-doi="10.1073/pnas.89.23.11219"><li><a href="http://dx.doi.org/10.1073/pnas.89.23.11219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+of+the+Hypersensitive+Site+Hs2+Enhancer+in+Erythroid-Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+of+the+Hypersensitive+Site+Hs2+Enhancer+in+Erythroid-Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000630-Fibach1" id="pone.0000630-Fibach1"></a>Fibach E, Manor D, Oppenheim A, Rachmilewitz EA (1989) Proliferation and maturation of human erythroid progenitors in liquid culture. Blood  73: 100–103.  <ul class="find" data-citedArticleID="967660"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Proliferation+and+maturation+of+human+erythroid+progenitors+in+liquid+culture.&amp;auth=&amp;atitle=Proliferation+and+maturation+of+human+erythroid+progenitors+in+liquid+culture." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proliferation+and+maturation+of+human+erythroid+progenitors+in+liquid+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proliferation+and+maturation+of+human+erythroid+progenitors+in+liquid+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000630-Fibach2" id="pone.0000630-Fibach2"></a>Fibach E, Rachmilewitz EA (1990) Proliferation and differentiation of erythroid progenitors in liquid culture: analysis of progenitors derived from patients with polycythemia vera. Am J Hematol  35: 151–156.  <ul class="find" data-citedArticleID="967662" data-doi="10.1002/ajh.2830350303"><li><a href="http://dx.doi.org/10.1002/ajh.2830350303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proliferation+and+differentiation+of+erythroid+progenitors+in+liquid+culture%3A+analysis+of+progenitors+derived+from+patients+with+polycythemia+vera." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proliferation+and+differentiation+of+erythroid+progenitors+in+liquid+culture%3A+analysis+of+progenitors+derived+from+patients+with+polycythemia+vera.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000630-Xiang1" id="pone.0000630-Xiang1"></a>Xiang P, Fang X, Yin W, Barkess G, Li Q (2006) Non-coding transcripts far upstream of the epsilon-globin gene are distinctly expressed in human primary tissues and erythroleukemia cell lines. Biochem Biophys Res Commun  344: 623–630.  <ul class="find" data-citedArticleID="967728" data-doi="10.1016/j.bbrc.2006.03.189"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.03.189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-coding+transcripts+far+upstream+of+the+epsilon-globin+gene+are+distinctly+expressed+in+human+primary+tissues+and+erythroleukemia+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-coding+transcripts+far+upstream+of+the+epsilon-globin+gene+are+distinctly+expressed+in+human+primary+tissues+and+erythroleukemia+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000630-Humbert1" id="pone.0000630-Humbert1"></a>Humbert C, Santisteban MS, Usson Y, Robertnicoud M (1992) Intranuclear Colocation of Newly Replicated DNA and Pcna by Simultaneous Immunofluorescent Labeling and Confocal Microscopy in Mcf-7 Cells. Journal of Cell Science  103: 97–103.  <ul class="find" data-citedArticleID="967676"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Intranuclear+Colocation+of+Newly+Replicated+DNA+and+Pcna+by+Simultaneous+Immunofluorescent+Labeling+and+Confocal+Microscopy+in+Mcf-7+Cells.&amp;auth=&amp;atitle=Intranuclear+Colocation+of+Newly+Replicated+DNA+and+Pcna+by+Simultaneous+Immunofluorescent+Labeling+and+Confocal+Microscopy+in+Mcf-7+Cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intranuclear+Colocation+of+Newly+Replicated+DNA+and+Pcna+by+Simultaneous+Immunofluorescent+Labeling+and+Confocal+Microscopy+in+Mcf-7+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intranuclear+Colocation+of+Newly+Replicated+DNA+and+Pcna+by+Simultaneous+Immunofluorescent+Labeling+and+Confocal+Microscopy+in+Mcf-7+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000630-Sims1" id="pone.0000630-Sims1"></a>Sims RJ 3rd, Belotserkovskaya R, Reinberg D (2004) Elongation by RNA polymerase II: the short and long of it. Genes Dev  18: 2437–2468.  <ul class="find" data-citedArticleID="967718" data-doi="10.1101/gad.1235904"><li><a href="http://dx.doi.org/10.1101/gad.1235904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elongation+by+RNA+polymerase+II%3A+the+short+and+long+of+it." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elongation+by+RNA+polymerase+II%3A+the+short+and+long+of+it.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000630-Henikoff1" id="pone.0000630-Henikoff1"></a>Henikoff S, Furuyama T, Ahmad K (2004) Histone variants, nucleosome assembly and epigenetic inheritance. Trends Genet  20: 320–326.  <ul class="find" data-citedArticleID="967672" data-doi="10.1016/j.tig.2004.05.004"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.05.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+variants%2C+nucleosome+assembly+and+epigenetic+inheritance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+variants%2C+nucleosome+assembly+and+epigenetic+inheritance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000630-Carter1" id="pone.0000630-Carter1"></a>Carter D, Chakalova L, Osborne CS, Dai YF, Fraser P (2002) Long-range chromatin regulatory interactions in vivo. Nat Genet  32: 623–626.  <ul class="find" data-citedArticleID="967646" data-doi="10.1038/ng1051"><li><a href="http://dx.doi.org/10.1038/ng1051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-range+chromatin+regulatory+interactions+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-range+chromatin+regulatory+interactions+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000630-Tolhuis1" id="pone.0000630-Tolhuis1"></a>Tolhuis B, Palstra RJ, Splinter E, Grosveld F, de Laat W (2002) Looping and interaction between hypersensitive sites in the active beta-globin locus. Mol Cell  10: 1453–1465.  <ul class="find" data-citedArticleID="967722" data-doi="10.1016/s1097-2765(02)00781-5"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00781-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Looping+and+interaction+between+hypersensitive+sites+in+the+active+beta-globin+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Looping+and+interaction+between+hypersensitive+sites+in+the+active+beta-globin+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000630-Wansink1" id="pone.0000630-Wansink1"></a>Wansink DG, Manders EE, van der Kraan I, Aten JA, van Driel R, et al.  (1994) RNA polymerase II transcription is concentrated outside replication domains throughout S-phase. J Cell Sci 107 ( Pt  6): 1449–1456.  <ul class="find" data-citedArticleID="967726"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RNA+polymerase+II+transcription+is+concentrated+outside+replication+domains+throughout+S-phase.&amp;auth=&amp;atitle=RNA+polymerase+II+transcription+is+concentrated+outside+replication+domains+throughout+S-phase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+polymerase+II+transcription+is+concentrated+outside+replication+domains+throughout+S-phase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+polymerase+II+transcription+is+concentrated+outside+replication+domains+throughout+S-phase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000630-Osborne1" id="pone.0000630-Osborne1"></a>Osborne CS, Chakalova L, Brown KE, Carter D, Horton A, et al.  (2004) Active genes dynamically colocalize to shared sites of ongoing transcription. Nat Genet  36: 1065–1071.  <ul class="find" data-citedArticleID="967698" data-doi="10.1038/ng1423"><li><a href="http://dx.doi.org/10.1038/ng1423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Active+genes+dynamically+colocalize+to+shared+sites+of+ongoing+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Active+genes+dynamically+colocalize+to+shared+sites+of+ongoing+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000630-Ragoczy1" id="pone.0000630-Ragoczy1"></a>Ragoczy T, Bender MA, Telling A, Byron R, Groudine M (2006) The locus control region is required for association of the murine beta-globin locus with engaged transcription factories during erythroid maturation. Genes Dev  20: 1447–1457.  <ul class="find" data-citedArticleID="967702" data-doi="10.1101/gad.1419506"><li><a href="http://dx.doi.org/10.1101/gad.1419506" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+locus+control+region+is+required+for+association+of+the+murine+beta-globin+locus+with+engaged+transcription+factories+during+erythroid+maturation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+locus+control+region+is+required+for+association+of+the+murine+beta-globin+locus+with+engaged+transcription+factories+during+erythroid+maturation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000630-Chakalova2" id="pone.0000630-Chakalova2"></a>Chakalova L, Mitchell JA, Debrand E, Osborne CS, Fraser P (2005) Replication and transcription: Forces that shape the landscape of the genome. Nature Reviews Genetics in press.   <ul class="find" data-citedArticleID="967650" data-doi="10.1038/nrg1673"><li><a href="http://dx.doi.org/10.1038/nrg1673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+and+transcription%3A+Forces+that+shape+the+landscape+of+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+and+transcription%3A+Forces+that+shape+the+landscape+of+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000630-Dickerman1" id="pone.0000630-Dickerman1"></a>Dickerman HW, Cheng TC, Kazazian HH Jr, Spivak JL (1976) The erythropoietic mouse spleen-a model system of development. Arch Biochem Biophys  177: 1–9.  <ul class="find" data-citedArticleID="967656" data-doi="10.1016/0003-9861(76)90408-2"><li><a href="http://dx.doi.org/10.1016/0003-9861(76)90408-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+erythropoietic+mouse+spleen-a+model+system+of+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+erythropoietic+mouse+spleen-a+model+system+of+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000630-Chakalova3" id="pone.0000630-Chakalova3"></a>Chakalova L, Carter D, Fraser P (2004) RNA fluorescence in situ hybridization tagging and recovery of associated proteins to analyze in vivo chromatin interactions. Methods Enzymol  375: 479–493.  <ul class="find" data-citedArticleID="967652" data-doi="10.1016/s0076-6879(03)75029-0"><li><a href="http://dx.doi.org/10.1016/s0076-6879(03)75029-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+fluorescence+in+situ+hybridization+tagging+and+recovery+of+associated+proteins+to+analyze+in+vivo+chromatin+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+fluorescence+in+situ+hybridization+tagging+and+recovery+of+associated+proteins+to+analyze+in+vivo+chromatin+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000630-ONeill1" id="pone.0000630-ONeill1"></a>O'Neill LP, Turner BM (2003) Immunoprecipitation of native chromatin: NChIP. Methods  31: 76–82.  <ul class="find" data-citedArticleID="967694" data-doi="10.1016/s1046-2023(03)00090-2"><li><a href="http://dx.doi.org/10.1016/s1046-2023(03)00090-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunoprecipitation+of+native+chromatin%3A+NChIP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunoprecipitation+of+native+chromatin%3A+NChIP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000630-Livak1" id="pone.0000630-Livak1"></a>Livak KJ, Schmittgen TD (2001) Analysis of relative gene expression data using real-time quantitative PCR and the 2(T)(-Delta Delta C) method. Methods  25: 402–408.  <ul class="find" data-citedArticleID="967688" data-doi="10.1006/meth.2001.1262"><li><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28T%29%28-Delta+Delta+C%29+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28T%29%28-Delta+Delta+C%29+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.XML" value="103996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.PDF" value="591264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g001.PNG_L" value="297870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g001.PNG_M" value="66144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g001.PNG_S" value="6503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g001.TIF" value="1178078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g001.PNG_I" value="26777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g002.PNG_L" value="369332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g002.PNG_M" value="75718"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g002.PNG_S" value="8645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g002.TIF" value="1656160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g002.PNG_I" value="39145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g003.PNG_L" value="497211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g003.PNG_M" value="66182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g003.PNG_S" value="11372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g003.TIF" value="2070020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g003.PNG_I" value="66803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g004.PNG_L" value="599876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g004.PNG_M" value="80648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g004.PNG_S" value="9998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g004.TIF" value="2425126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g004.PNG_I" value="55302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g005.PNG_L" value="986168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g005.PNG_M" value="105933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g005.PNG_S" value="9165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g005.TIF" value="2018890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g005.PNG_I" value="55354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g006.PNG_L" value="1793258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g006.PNG_M" value="163847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g006.PNG_S" value="10006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g006.TIF" value="3097582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.g006.PNG_I" value="80591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.s001.TIF" value="565646"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000630.s002.DOC" value="35328"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000630&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000630" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000630&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000630&volume=&issue=&title=Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus&author_name=Joanne%20Miles%2C%20Jennifer%20A.%20Mitchell%2C%20Lyubomira%20Chakalova%2C%20Beatriz%20Goyenechea%2C%20Cameron%20S.%20Osborne%2C%20Laura%20O'Neill%2C%20Keiji%20Tanimoto%2C%20James%20Douglas%20Engel%2C%20Peter%20Fraser&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000630" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000630" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000630" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000630&amp;t=Intergenic%20Transcription%2C%20Cell-Cycle%20and%20the%20Developmentally%20Regulated%20Epigenetic%20Profile%20of%20the%20Human%20Beta-Globin%20Locus" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000630&title=Intergenic%20Transcription%2C%20Cell-Cycle%20and%20the%20Developmentally%20Regulated%20Epigenetic%20Profile%20of%20the%20Human%20Beta-Globin%20Locus&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000630&amp;title=Intergenic%20Transcription%2C%20Cell-Cycle%20and%20the%20Developmentally%20Regulated%20Epigenetic%20Profile%20of%20the%20Human%20Beta-Globin%20Locus" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000630" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000630&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Intergenic Transcription, Cell-Cycle and the Developmentally Regulated Epigenetic Profile of the Human Beta-Globin Locus';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000630';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Intergenic%20Transcription%2C%20Cell-Cycle%20and%20the%20Developmentally%20Regulated%20Epigenetic%20Profile%20of%20the%20Human%20Beta-Globin%20Locus http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000630" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000630" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin%22" title="Search for articles in the subject area:'Chromatin'"><div class="flagText">Chromatin</div></a>
              <div data-categoryid="31933" data-articleid="24098"
                   data-categoryname="Chromatin"
                   class="flagImage" title="Flag 'Chromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40169" data-articleid="24098"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="24098"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histone+modification%22" title="Search for articles in the subject area:'Histone modification'"><div class="flagText">Histone modification</div></a>
              <div data-categoryid="19555" data-articleid="24098"
                   data-categoryname="Histone modification"
                   class="flagImage" title="Flag 'Histone modification' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histones%22" title="Search for articles in the subject area:'Histones'"><div class="flagText">Histones</div></a>
              <div data-categoryid="18573" data-articleid="24098"
                   data-categoryname="Histones"
                   class="flagImage" title="Flag 'Histones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Immunoprecipitation%22" title="Search for articles in the subject area:'Immunoprecipitation'"><div class="flagText">Immunoprecipitation</div></a>
              <div data-categoryid="19963" data-articleid="24098"
                   data-categoryname="Immunoprecipitation"
                   class="flagImage" title="Flag 'Immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Locus+control+region%22" title="Search for articles in the subject area:'Locus control region'"><div class="flagText">Locus control region</div></a>
              <div data-categoryid="21391" data-articleid="24098"
                   data-categoryname="Locus control region"
                   class="flagImage" title="Flag 'Locus control region' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Synthesis+phase%22" title="Search for articles in the subject area:'Synthesis phase'"><div class="flagText">Synthesis phase</div></a>
              <div data-categoryid="19095" data-articleid="24098"
                   data-categoryname="Synthesis phase"
                   class="flagImage" title="Flag 'Synthesis phase' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4831'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3457'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5215&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=12919">Referee Comments: Referee #1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
