

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001806"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001806" />

    <meta name="citation_title" content="Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions"/>
    <meta itemprop="name" content="Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions"/>

      <meta name="citation_author" content="Sergio Rojas-Galeano"/>
            <meta name="citation_author_institution" content="Division of Parasitology, National Institute for Medical Research, London, United Kingdom"/>
            <meta name="citation_author_institution" content="Department of Computer Science, University College London, London, United Kingdom"/>
      <meta name="citation_author" content="Emily Hsieh"/>
            <meta name="citation_author_institution" content="Division of Cellular and Molecular Medicine, Centre for Infection, St George's University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Dan Agranoff"/>
            <meta name="citation_author_institution" content="Department of Infectious Diseases and Immunity, Faculty of Medicine, Imperial College London, London, United Kingdom"/>
      <meta name="citation_author" content="Sanjeev Krishna"/>
            <meta name="citation_author_institution" content="Division of Cellular and Molecular Medicine, Centre for Infection, St George's University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Delmiro Fernandez-Reyes"/>
            <meta name="citation_author_institution" content="Division of Parasitology, National Institute for Medical Research, London, United Kingdom"/>
            <meta name="citation_author_institution" content="Department of Computer Science, University College London, London, United Kingdom"/>

    <meta name="citation_date" content="2008/3/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001806.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1806"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_author=P Baldi; citation_author=W Hatfield; citation_number=1; citation_date=2002; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Protocols for disease classification from mass spectrometry data.; citation_author=M Wagner; citation_author=D Naik; citation_author=A Pothen; citation_journal_title=Proteomics; citation_number=2; citation_pages=1692-1698; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The SELDI-TOF MS approach to proteomics: protein profiling and biomarker identification.; citation_author=HJ Issaq; citation_author=TD Veenstra; citation_author=TP Conrads; citation_author=D Felschow; citation_journal_title=Biochem Biophys Res Comm; citation_volume=292; citation_number=3; citation_pages=587-592; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=S Davies; citation_author=S Russell; citation_number=4; citation_date=1994; " />
      <meta name="citation_reference" content="citation_author=M Garey; citation_author=D Johnson; citation_number=5; citation_date=1979; citation_publisher=W.H. Freeman and Company; " />
      <meta name="citation_reference" content="citation_title=An introduction to variable and feature selection.; citation_author=I Guyon; citation_author=A Elisseeff; citation_journal_title=Journal of Machine Learning Research; citation_volume=3; citation_number=6; citation_pages=1157-1182; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of diagnostic markers for tuberculosis by proteomic fingerprinting of serum.; citation_author=D Agranoff; citation_author=D Fernandez-Reyes; citation_author=M Papadopoulos; citation_author=S Rojas Galeano; citation_author=M Herbster; citation_journal_title=Lancet; citation_volume=368; citation_number=7; citation_pages=1012-1021; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Computational protein biomarker prediction: a case study for prostate cancer.; citation_author=M Wagner; citation_author=D Naik; citation_author=A Pothen; citation_author=S Kasukurti; citation_author=R Devineni; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A novel and accurate test for Human African Trypanosomiasis.; citation_author=MC Papadopoulos; citation_author=PM Abel; citation_author=D Agranoff; citation_author=A Stich; citation_author=E Tarelli; citation_journal_title=Lancet; citation_volume=363; citation_number=9; citation_pages=1358-1363; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High-resolution serum proteomic features for ovarian cancer detection.; citation_author=T Conrads; citation_author=V Fusaro; citation_author=S Ross; citation_author=D Johann; citation_author=V Rajapakse; citation_journal_title=Endocrine-Related Cancer; citation_number=10; citation_pages=163-178; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A robust hybrid between genetic algorithm and support vector machine for extracting an optimal feature gene subset.; citation_author=L Li; citation_author=W Jian; citation_author=X Li; citation_author=K Moser; citation_author=Z Guo; citation_journal_title=Genomics; citation_number=11; citation_pages=16-23; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiclass cancer classification and biomarker discovery using GA-based algorithms.; citation_author=J Liu; citation_author=G Gutler; citation_author=W Li; citation_author=Z Pan; citation_author=S Peng; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=12; citation_pages=2691-2697; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=H Fröhlich; citation_author=O Chapelle; citation_author=B Scholkopf; citation_number=13; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene Selection for Cancer Classification using Support Vector Machines.; citation_author=I Guyon; citation_author=J Weston; citation_author=S Barnhill; citation_author=V Vapnik; citation_journal_title=Machine Learning; citation_volume=46; citation_number=14; citation_pages=389-422; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Recursive SVM feature selection and sample classification for mass-spectometry and microarray data.; citation_author=X Zhang; citation_author=X Lu; citation_author=Q Shi; citation_author=X Xu; citation_author=H Leung; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=15; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=H Fröhlich; citation_author=A Zell; citation_number=16; citation_pages=2041-2046; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Improving the performance of SVM-RFE to select genes in microarray data.; citation_author=Y Ding; citation_author=D Wilkins; citation_journal_title=BMC Bioinformatics; citation_number=17; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=J Bedo; citation_author=C Sanderson; citation_author=A Kowalczyk; citation_number=18; citation_date=2006; citation_publisher=Springer Berlin / Heidelberg; " />
      <meta name="citation_reference" content="citation_title=Choosing Multiple Parameters for Support Vector Machines.; citation_author=O Chapelle; citation_author=V Vapnik; citation_author=O Bousquet; citation_author=S Mukherjee; citation_journal_title=Machine Learning; citation_volume=46; citation_number=19; citation_pages=131-159; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=T Van Gestel; citation_author=JAK Suykens; citation_author=B De Moor; citation_author=J Vandewalle; citation_number=20; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sparse Bayesian learning and the Relevance Vector Machine.; citation_author=ME Tipping; citation_journal_title=Journal of Machine Learning Research; citation_volume=1; citation_number=21; citation_pages=211-244; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=S Rojas Galeano; citation_author=D Fernandez-Reyes; citation_number=22; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionary tuning of multiple SVM parameters.; citation_author=F Friedrichs; citation_author=C Igel; citation_journal_title=Neurocomputing; citation_number=23; citation_pages=107-117; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=P Pelikan; citation_author=K Sastry; citation_author=E Cantu-Paz; citation_number=24; citation_date=2006; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Large Margin Classification Using the Perceptron Algorithm.; citation_author=Y Freund; citation_author=R Schapire; citation_journal_title=Machine Learning; citation_volume=37; citation_number=25; citation_pages=277-296; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A genetic algorithm tutorial.; citation_author=D Whitley; citation_journal_title=Statistics and Computing; citation_number=26; citation_pages=65-85; citation_date=1994; " />
      <meta name="citation_reference" content="citation_author=J Shawe-Taylor; citation_author=N Cristianini; citation_number=27; citation_date=2004; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_author=J Weston; citation_author=S Mukherjee; citation_author=O Chapelle; citation_author=M Pontil; citation_author=T Poggio; citation_number=28; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays.; citation_author=U Alon; citation_author=N Barkai; citation_author=DA Notterman; citation_author=K Gish; citation_author=S Ybarra; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=29; citation_pages=6745-6750; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Gene expression-based classification of malignant gliomas correlates better with survival than histological classification.; citation_author=C Nutt; citation_author=D Mani; citation_author=R Betensky; citation_author=P Tamayo; citation_author=G Cairncross; citation_journal_title=Cancer Research; citation_number=30; citation_pages=1602-1607; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=L Davies; citation_number=31; citation_date=1991; citation_publisher=Van Nostrand Reinhold Company; " />
      <meta name="citation_reference" content="citation_title=Functions of positive and negative type and their connection with the theory of integral equations.; citation_author=J Mercer; citation_journal_title=Philos Trans Roy Soc London; citation_number=32; citation_date=1909; " />
      <meta name="citation_reference" content="citation_title=Support vector networks.; citation_author=C Cortes; citation_author=V Vapnik; citation_journal_title=Machine Learning; citation_volume=20; citation_number=33; citation_pages=273-297; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The Perceptron: A Probabilistic Model for Information Storage and Organization in the Brain.; citation_author=F Rosenblatt; citation_journal_title=Psychological Review; citation_volume=65; citation_number=34; citation_pages=386-408; citation_date=1958; " />
      <meta name="citation_reference" content="citation_author=M Minsky; citation_author=S Papert; citation_number=35; citation_date=1969; citation_publisher=MIT Press; " />
      <meta name="citation_reference" content="citation_author=M Herbster; citation_number=36; citation_pages=444-460; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=On convergence proofs on perceptrons.; citation_author=A Novikoff; citation_journal_title=Symposium on the Mathematical Theory of Automata; citation_number=37; citation_pages=615-622; citation_date=1962; " />
      <meta name="citation_reference" content="citation_author=N Cristianini; citation_author=J Shawe-Taylor; citation_number=38; citation_date=2000; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Making large-scale SVM learning practical.; citation_author=T Joachims; citation_number=39; citation_date=1999; citation_publisher=MIT Press; " />
      <meta name="citation_reference" content="citation_title=MATLAB/MEX Interface to SVMlight.; citation_author=T Briggs; citation_number=40; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001806" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions"/>
    <meta name="twitter:description" content="BackgroundThe analysis of complex proteomic and genomic profiles involves the identification of significant markers within a set of hundreds or even thousands of variables that represent a high-dimensional problem space. The occurrence of noise, redundancy or combinatorial interactions in the profile makes the selection of relevant variables harder.Methodology/Principal FindingsHere we propose a method to select variables based on estimated relevance to hidden patterns. Our method combines a weighted-kernel discriminant with an iterative stochastic probability estimation algorithm to discover the relevance distribution over the set of variables. We verified the ability of our method to select predefined relevant variables in synthetic proteome-like data and then assessed its performance on biological high-dimensional problems. Experiments were run on serum proteomic datasets of infectious diseases. The resulting variable subsets achieved classification accuracies of 99% on Human African Trypanosomiasis, 91% on Tuberculosis, and 91% on Malaria serum proteomic profiles with fewer than 20% of variables selected. Our method scaled-up to dimensionalities of much higher orders of magnitude as shown with gene expression microarray datasets in which we obtained classification accuracies close to 90% with fewer than 1% of the total number of variables.ConclusionsOur method consistently found relevant variables attaining high classification accuracies across synthetic and biological datasets. Notably, it yielded very compact subsets compared to the original number of variables, which should simplify downstream biological experimentation."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001806.g004"/>

  <meta property="og:title" content="Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001806" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6549'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2445'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9296&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001806">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001806" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001806&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sergio Rojas-Galeano, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Division of Parasitology, National Institute for Medical Research, London, United Kingdom, 
                  Department of Computer Science, University College London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Emily Hsieh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Cellular and Molecular Medicine, Centre for Infection, St George's University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dan Agranoff, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Infectious Diseases and Immunity, Faculty of Medicine, Imperial College London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sanjeev Krishna, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Cellular and Molecular Medicine, Centre for Infection, St George's University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Delmiro Fernandez-Reyes
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:dfernan@nimr.mrc.ac.uk">dfernan@nimr.mrc.ac.uk</a></p>

                <p>Affiliations:
                  Division of Parasitology, National Institute for Medical Research, London, United Kingdom, 
                  Department of Computer Science, University College London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 26, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001806</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001806" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001806" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001806" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001806" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001806-g001" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001806-t001" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001806-g002" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001806-g003" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001806-g004" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001806-t002" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001806-t003" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001806">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001806" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The analysis of complex proteomic and genomic profiles involves the identification of significant markers within a set of hundreds or even thousands of variables that represent a high-dimensional problem space. The occurrence of noise, redundancy or combinatorial interactions in the profile makes the selection of relevant variables harder.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we propose a method to select variables based on estimated relevance to hidden patterns. Our method combines a weighted-kernel discriminant with an iterative stochastic probability estimation algorithm to discover the relevance distribution over the set of variables. We verified the ability of our method to select predefined relevant variables in synthetic proteome-like data and then assessed its performance on biological high-dimensional problems. Experiments were run on serum proteomic datasets of infectious diseases. The resulting variable subsets achieved classification accuracies of 99% on Human African Trypanosomiasis, 91% on Tuberculosis, and 91% on Malaria serum proteomic profiles with fewer than 20% of variables selected. Our method scaled-up to dimensionalities of much higher orders of magnitude as shown with gene expression microarray datasets in which we obtained classification accuracies close to 90% with fewer than 1% of the total number of variables.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our method consistently found relevant variables attaining high classification accuracies across synthetic and biological datasets. Notably, it yielded very compact subsets compared to the original number of variables, which should simplify downstream biological experimentation.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rojas-Galeano S, Hsieh E, Agranoff D, Krishna S, Fernandez-Reyes D (2008) Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions. PLoS ONE 3(3):
          e1806.
            doi:10.1371/journal.pone.0001806</p><p><strong>Editor: </strong>Gustavo Stolovitzky, IBM Thomas J. Watson Research Center, United States of America</p><p><strong>Received:</strong> October 1, 2007; <strong>Accepted:</strong> February 11, 2008; <strong>Published:</strong> March 26, 2008</p><p><strong>Copyright:</strong> © 2008 Rojas-Galeano et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The study was funded by The Medical Research Council, United Kingdom. The sponsor of the study had no direct role in study design, data collection, data analysis, data interpretation, or writing of the report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>High-throughput genomic and proteomic screening of biological samples produces large data arrays <a href="#pone.0001806-Baldi1">[1]</a>–<a href="#pone.0001806-Issaq1">[3]</a> characterizing instances of two different conditions in a very high dimensional space; that is, the space consisting of a vast number of observations or variables that are acquired for each sample. This is the case for mass spectrometry profiles of complex protein mixtures with hundreds of measures of mass-to-charge ratios for polypeptide chains detected in samples such as serum, or genomic microarray studies profiling tens of thousands of genes expressed in tissue samples. The computational analysis of these biological datasets involves the discovery of informative patterns between sample instances and the identification of the specific biomarkers of disease. These analyses facilitate the design of new diagnostic tests or can be used to focus further biological research on specific drug or vaccine candidate molecules. Intuitively, such patterns should not span the entire spectrum of observations but ought to be encoded in a few relevant variables, with the remainder representing noise. The search for such a subset of relevant variables would imply an exhaustive test of all possible combinations, a task that even for the dimensionality of serum proteomic datasets would prove unfeasible. The computational complexity of such searches increases exponentially with the number of variables; it is known as a NP-complete problem and hence computationally intractable <a href="#pone.0001806-Davies1">[4]</a>, <a href="#pone.0001806-Garey1">[5]</a>. Consequentially heuristic methods with the aim of selecting an approximate-best variable subset must be considered.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>There are two approaches to variable selection: filter and wrapper methods <a href="#pone.0001806-Guyon1">[6]</a>. Filter methods rank the complete set of variables with a given criterion, independently from the applied classifier. They have been widely-used in the analysis of proteomic signatures of diseases such as prostate cancer, sleeping sickness and tuberculosis <a href="#pone.0001806-Agranoff1">[7]</a>–<a href="#pone.0001806-Papadopoulos1">[9]</a>. Several variants which have also been applied to genomic cancer datasets include lists of permutations of significant variables that are filtered by genetic algorithms (GA) coupled with support vector machines (SVMs) <a href="#pone.0001806-Conrads1">[10]</a>–<a href="#pone.0001806-Frhlich1">[13]</a>. Wrapper methodologies on the other hand, implicitly use the classifier to evaluate variables according to their contribution to its predictive power. Although variable selection using wrapper strategies may incur extra computational costs, this is compensated by the ability to explore complex associations between variables detected within the intrinsic patterns incorporated in the discrimination rules. Recursive feature elimination (RFE) uses SVM functions to iteratively rank and discard relevant variables via a greedy search and has been applied to cancer microarray datasets <a href="#pone.0001806-Guyon2">[14]</a>–<a href="#pone.0001806-Bedo1">[18]</a>. The main drawback of this approach lies in the greedy strategy that may disrupt relationships between variables discarded in different stages of the algorithm, leading to sub-optimal selected subsets. To sidestep this difficulty, an alternative approach combines weighted kernels with SVMs <a href="#pone.0001806-Chapelle1">[19]</a>–<a href="#pone.0001806-RojasGaleano1">[22]</a>; this approach assigns a weight to each variable to indicate its relevance. In <a href="#pone.0001806-Chapelle1">[19]</a> the weight vector is computed using a gradient-descent formulation, which uses bounds on the expected generalization error of the SVM. However, the applicability of this method is restricted by assumptions requiring the kernel and objective functions to be continuous and differentiable, as well as the dataset being separable. In a previous work <a href="#pone.0001806-RojasGaleano1">[22]</a> we proposed to adapt the weighted-kernel SVM using a GA instead of the gradient descent algorithm to improve model selection on weighted radial basis kernels rather than to select variables. In a similar direction, a recent technique using evolutionary strategies to adjust both scaling and orientation of generalized Gaussian kernels in SVMs has been reported <a href="#pone.0001806-Friedrichs1">[23]</a>; the evolved matrices, however, must be constrained to meet the requirements of proper kernels and, similarly, the aim is to improve the performance of classification instead of selecting variables.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The wrapper method we describe in this paper focuses on estimating a relevance distribution encoded by the weight vector; such a distribution becomes instrumental in the selection of significant variables. For this end, the <em>weighted Kernel-based Iterative Estimation of Relevance Algorithm</em> (wKIERA) combines a stochastic-search estimation of distribution algorithm with a kernel pattern-recognition method. The motivation behind using a stochastic estimation of distribution algorithm <a href="#pone.0001806-Pelikan1">[24]</a> is three-fold: (i) the ability to derive the parameters of the weighted kernel directly from the resulting relevance distribution; (ii) its capability of avoiding premature poor convergence on optimization of multiple-minima cost functions; and (iii) the low memory-space requirements arising from its compact representation, which is advantageous in the case of dimensionalities of hundreds or thousands of variables. The advantage of employing kernel-based classification is its ability to handle nonlinear decision surfaces in data generated from high-throughput experiments while still adhering to the simplicities of linear classifiers. We reduced the computational cost of the iterative estimation algorithm by using a kernel perceptron <a href="#pone.0001806-Freund1">[25]</a> as an alternative to SVM, since it provides fast operation with guarantees on upper bounds of misclassification errors. Consequently, wKIERA combines the exploration-exploitation trade-off exhibited by probabilistic model-building stochastic search algorithms for combinatorics <a href="#pone.0001806-Whitley1">[26]</a> with robustness to nonlinear concepts in high-dimensional spaces provided by kernel-based pattern analysis <a href="#pone.0001806-ShaweTaylor1">[27]</a>. Our framework successfully selects relevant variables in high-dimensional proteomic and genomic profiles of complex biological processes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We performed experiments with wKIERA (<a href="#pone-0001806-g001">Fig. 1</a>) on a variety of synthetic and biological datasets (<a href="#pone-0001806-t001">Table 1</a>). First, wKIERA was run <em>N</em> times with different random training/test splits, obtaining an average relevance vector <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e001&amp;representation=PNG" class="inline-graphic"></span>. This vector was then scaled to the interval [0, 1] and its components were sorted in descending order with highest values representing relevant variables. We selected relevant variables by defining a cutoff threshold on <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e002&amp;representation=PNG" class="inline-graphic"></span>. We then used SVMs to evaluate the performance of selected variables in 100 classification experiments using random training/test splits of the dataset. We visualized the classification performance of the subsets of variables obtained by applying a threshold with a step size of 0.1 to the wKIERA relevance vector <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e003&amp;representation=PNG" class="inline-graphic"></span> (<a href="#pone-0001806-g002">Figs. 2</a>, <a href="#pone-0001806-g003">3</a> and <a href="#pone-0001806-g004">4 top</a>). We then compared the subset of best performing variables from the threshold plot, with the least relevant ranked variables by wKIERA, as well as with the complete set of original variables and with those rated as relevant according to rank correlation coefficients (<a href="#pone-0001806-g002">Figs. 2</a>, <a href="#pone-0001806-g003">3</a> and <a href="#pone-0001806-g004">4 middle</a>). The performance in ROC space for the same subsets of variables is also shown (<a href="#pone-0001806-g002">Figures 2</a>, <a href="#pone-0001806-g003">3</a> and <a href="#pone-0001806-g004">4 bottom</a>).</p>
<div class="figure" id="pone-0001806-g001"><div class="img"><a name="pone-0001806-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>High level flow chart of the <em>wKIERA</em> Algorithm.</span></strong></p><span>doi:10.1371/journal.pone.0001806.g001</span></div><div class="figure" id="pone-0001806-t001"><div class="img"><a name="pone-0001806-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001806.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001806.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Description of simulated and biological datasets used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001806.t001</span></div><div class="figure" id="pone-0001806-g002"><div class="img"><a name="pone-0001806-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Performance of variable subsets on simulated datasets.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>A) LOI dataset (wKIERA settings: <em>poolsize</em> = 10, <em>maxiter</em> = 400, <em>rep</em> = 2000, wkRBF ρ = 0.1); B) NLG dataset (<em>poolsize</em> = 10, <em>maxiter</em> = 400, <em>rep</em> = 2000, wkPoly d = 2). <span class="underline">Top</span>: Average SVM accuracy on 100 randomly train/test splits using subsets of variables obtained by thresholding the estimated factors of a weighted kernel with the corresponding cutoff on horizontal axis. Resulting subset size (number of variables) is shown in brackets. <span class="underline">Middle</span>: Comparison of classification accuracy of SVM trained using variables selected by best-wKIERA-ranked (red); worst-wKIERA-ranked (black); rank correlation coefficients (blue) and using all variables (green). Results are averaged over 100 randomly training/test splits. <span class="underline">Bottom</span>: ROC-space analysis of the SVM classifiers shown in the mid plot.</p>
<span>doi:10.1371/journal.pone.0001806.g002</span></div><div class="figure" id="pone-0001806-g003"><div class="img"><a name="pone-0001806-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Performance of variable subsets on proteomic datasets.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>A) HAT dataset (wKIERA settings: <em>poolsize</em> = 10, <em>maxiter</em> = 400, <em>rep</em> = 2000, wkRBF ρ = 0.01); B) TB dataset (<em>poolsize</em> = 10, <em>maxiter</em> = 400, <em>rep</em> = 2000, wkRBF ρ = 1). C) MALARIA dataset (<em>poolsize</em> = 10, <em>maxiter</em> = 400, <em>rep</em> = 2000, wkRBF ρ = 1). <span class="underline">Top</span>, <span class="underline">Middle</span> and <span class="underline">Bottom</span>: See legend on <a href="#pone-0001806-g002">Figure 2</a>.</p>
<span>doi:10.1371/journal.pone.0001806.g003</span></div><div class="figure" id="pone-0001806-g004"><div class="img"><a name="pone-0001806-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001806.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Performance of variable subsets on gene expression microarray datasets.</span></strong></p><a id="article1.body1.sec2.fig4.caption1.p1" name="article1.body1.sec2.fig4.caption1.p1"></a><p>A) COLON CANCER dataset (wKIERA settings: <em>poolsize</em> = 100, <em>maxiter</em> = 1000, <em>rep</em> = 1000, wkRBF ρ = 0.1); B) GLIAL CANCER dataset (<em>poolsize</em> = 100, <em>maxiter</em> = 1000, <em>rep</em> = 1000, wkRBF ρ = 1×10<sup>−5</sup>). <span class="underline">Top</span>, <span class="underline">Middle</span> and <span class="underline">Bottom</span>: See legend on <a href="#pone-0001806-g002">Figure 2</a>.</p>
<span>doi:10.1371/journal.pone.0001806.g004</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>To assess the framework reliability we designed experiments using linear and non-linear simulated proteomic-like datasets with predefined sets of relevant variables. For all of the synthetic datasets wKIERA selected the correct relevant variables among the first top-ranked components of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e008&amp;representation=PNG" class="inline-graphic"></span> except for the LH dataset where one irrelevant variable was ranked before another relevant (<a href="#pone-0001806-t002">Table 2</a>). <a href="#pone-0001806-g002">Figure 2</a> shows the classification performance of two representative proteomic-like artificial datasets: one with outlier instances (LOI) and the other sampled from a mixture of Gaussians (NLG). On the LOI dataset, the performance of wKIERA is comparable to that of the rank correlation coefficients but with a smaller set of relevant variables (<a href="#pone-0001806-g002">Fig. 2A middle</a>). The accuracy obtained with the worst-wKIERA-ranked variables is close to random classification as expected and shows that the best-ranked variables were not selected by chance. Moreover, classification using all variables is poor because excessive noise is introduced by the non-relevant variables (<a href="#pone-0001806-g002">Fig. 2A middle</a>). Similarly, in the NLG dataset, classification with selected variables by wKIERA outperformed that of bottom-ranked or all variables (<a href="#pone-0001806-g002">Fig. 2B middle</a>). On this dataset our method clearly outperformed rank correlation coefficients and indeed it is known that the latter method is sensitive to non-linear labeling functions (<a href="#pone-0001806-g002">Fig. 2B middle</a>). Experimental results for the other synthetic datasets followed similar trends (not shown); in all cases the predefined relevant variables were successfully selected by wKIERA (<a href="#pone-0001806-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001806-t002"><div class="img"><a name="pone-0001806-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001806.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001806.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Selected variables in synthetic datasets by wKIERA (<em>poolsize</em> = 10, <em>maxiter</em> = 400).</span></strong></p><span>doi:10.1371/journal.pone.0001806.t002</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>We assessed the performance of wKIERA on real data using a panel of high-dimensional biological patterns. We focused our experiments on proprietary proteomic datasets of infectious diseases such as Human African Trypanosomiasis (HAT) <a href="#pone.0001806-Papadopoulos1">[9]</a>, Tuberculosis (TB) <a href="#pone.0001806-Agranoff1">[7]</a> and Malaria (<a href="#pone-0001806-t001">Table 1</a>). On the HAT dataset, classifiers trained with variables selected by wKIERA achieved an accuracy of 99% with comparable performance to using those selected by rank correlation coefficients (<a href="#pone-0001806-g003">Fig. 3A</a>). However, the number of variables selected by wKIERA was much smaller (21% of the total (44) compared to 55% (114)). Interestingly, classifiers trained with all variables or the worst-wKIERA-ranked subset of variables showed accuracies above 90%, which indicates that discrimination patterns are widely distributed across all variables in this dataset. Analyses of the TB dataset show that wKIERA selected variables yielding an accuracy of 91% while for those selected with rank correlation coefficients the accuracy was 89% and using all variables 87% (<a href="#pone-0001806-g003">Fig. 3B</a>). As on the previous dataset the wKIERA subset was the smallest (17% of total size (37)) compared to 52% (113) and 100% (219). A 74% accuracy obtained by the worst-wKIERA-ranked may indicate the occurrence of noise in this dataset. Lastly, results for the Malaria dataset were wKIERA: 91%, rank correlation coefficients: 89%, and all-variables: 88% (<a href="#pone-0001806-g003">Fig. 3C</a>). Consistently, the subset obtained with wKIERA is much smaller (11 compared to 58 from a total of 170 variables). Once more, the 65% obtained with the worst-wKIERA-ranked may also suggest the presence of noise in this dataset.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>In order to assess the scalability of our method to higher numbers of variables, we subsequently conducted experiments on publicly available microarray datasets (<a href="#pone-0001806-t001">Table 1</a>) where dimensionality was increased between two and three orders of magnitude compared to the proteomic datasets described above. On the COLON CANCER dataset, the wKIERA subset of variables achieved 88% accuracy with only 2.5% (50) of the total variables, whereas rank correlation coefficients achieved 82% with a size of 8.5% (171) (<a href="#pone-0001806-g004">Fig. 4A</a>). On the other hand, in the GLIAL CANCER dataset the wKIERA subset attained 91% accuracy, outperforming all the other subsets of variables that achieved accuracies below 60% (<a href="#pone-0001806-g004">Fig. 4B</a>). Again, the small number of variables selected by wKIERA (just 0.4% or 48 out of 12626) is noteworthy. The poor performance obtained with rank correlation coefficients indicates that labeled-correlated variables are insufficient to solve the possibly non-linear separation surfaces contained in this dataset.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>In summary, our wKIERA method consistently found relevant variables attaining high classification accuracies in synthetic and biological datasets, and yielded subsets that were very compact compared to the original number of variables. This is highly desirable for the feasibility of downstream biological experimentation. The method reliably scaled-up to dimensionalities of much higher orders of magnitude even when few instances were available, as shown with the cancer microarray datasets.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We propose an iterative framework for weighted kernel-based relevance estimation for high dimensional biological patterns. Variable relevance estimation assuming variable independence was achieved using a kernel perceptron classifier coupled with a probabilistic-model-building stochastic optimizer. We have shown the viability of such a configuration in controlled synthetic experiments. In a set of experiments involving proteomic profiles for infectious diseases our method found sets of significant protein clusters that achieved high classification accuracies but which were three times smaller than sets derived using classic correlation coefficients. The dimensionality of the overall datasets varied between 170 and 219. We also tried our method in problems with much larger dimensionalities such as cancer expression microarrays with 2000 and 12625 genes where only a handful of instances are available. The method scaled-up remarkably well in these situations, revealing significant patterns.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Weighted polynomial or RBF data-pattern kernel representations can be used within the wKIERA framework. Use of weighted RBF kernels was preferred for biological datasets because they are considered to be polynomial kernels of infinite degree <a href="#pone.0001806-ShaweTaylor1">[27]</a>. For synthetic datasets such as LH, NLG and NLC we experimented with polynomial weighted kernels in accordance with previous studies in the literature where the non-weighted versions were used <a href="#pone.0001806-Frhlich1">[13]</a>, <a href="#pone.0001806-Zhang1">[15]</a>, <a href="#pone.0001806-Weston1">[28]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The wKIERA framework modularity admits different configurations where faster online learning algorithms and more complex probabilistic-based search models can be used. This might allow us to analyze complex patterns of composite variable interactions and multivariate dependencies. We are currently investigating new mistake-driven algorithms with better generalization performance than the kernel perceptron but still showing fast execution. We are also considering refining the estimation of distribution algorithm by using probabilistic graphical models to represent higher-degree, nonlinear, conditional, or even time dependencies between variables. This research path may further improve the ability of our method to find informative pattern distributions that are likely to emerge given the dynamic nature of protein interactions.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Datasets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Proteome-like synthetic datasets were designed in order to perform controlled experiments using dimensionalities of two hundred variables, from which two to six were relevant. We encoded linear and non-linear labeling functions into the relevant variables. A few hundred samples were included, resulting in square-shaped data matrices. Sampling and labeling mechanisms are described in <a href="#pone-0001806-t001">Table 1</a>. We generated four linear datasets: LR, where some relevant variables can be discarded as redundant without disturbing classification accuracy; LOV, where noise was introduced to particular loci in randomly selected instances simulating artifacts generated during array processing; LOI, where noise was imposed on all variables in randomly selected instances, simulating inaccurate collection of samples; and LH, where a predefined linear discriminant for relevant variables was used to label the instances. In addition, two nonlinear datasets were generated: NLG, where clusters of mixtures of Gaussians were generated for each class, and NLC, where the clusters follow a tighter checkers-patterned distribution. The last two datasets also included redundancy.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Experiments were also conducted on real biological datasets. We tested proprietary proteomic profiles of infectious diseases (HAT <a href="#pone.0001806-Papadopoulos1">[9]</a>, TB <a href="#pone.0001806-Agranoff1">[7]</a> and MALARIA [Unpublished]). These high dimensional datasets are almost square, i.e. the number of variables and instances are similar (<a href="#pone-0001806-t001">Table 1</a>). We also used two publicly available gene expression microarray datasets COLON CANCER <a href="#pone.0001806-Alon1">[29]</a> and GLIAL CANCER <a href="#pone.0001806-Nutt1">[30]</a>). These datasets have a much higher dimensionality (2000 and 12625 respectively) and fewer instances (66 and 50 respectively). Compared to the proteomic datasets, the latter two datasets are rectangular in shape posing a more challenging obstacle to variable selection because of the curse of dimensionality phenomenon, i.e. shortage of sufficient instances to correctly sample high dimensional spaces.</p>


<h4>Notation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We denote <em>D</em> = {(<span class="underline">x</span><sub>1</sub>,<em>y</em><sub>1</sub>),…,(<span class="underline">x</span><em><sub>m</sub></em>,<em>y<sub>m</sub></em>)} a collection of <em>m</em> instance/label pairs where each instance <span class="underline">x</span><em><sub>i</sub></em> = (<em>x<sub>i</sub></em><sub>1</sub>,<em>x<sub>i</sub></em><sub>2</sub>…,<em>x<sub>in</sub></em>) consists of <em>n</em> observations representing one sample in an <em>n</em>-dimensional space, <em>y<sub>i</sub></em>Є{1, −1} specifying its binary class label, and 1≤<em>i</em>≤<em>m</em>. The coordinates of such a space are related to variables; each one associated with a factor ω ˆ<em><sub>i</sub></em>Є{0,1} to indicate its relevance. The vector <span class="underline">ω</span> approximates these factors using continuous weights ω<em><sub>i</sub></em>Є[0,1]. The set of instance indexes is denoted by <em>J</em> = {1,2,…,<em>m</em>}. Instances are randomly split into a training subset <em>S</em> and a test subset <em>U</em> to be used by a learning classifier. The kernel matrix of all instances in <em>D</em> is denoted by <em>K</em>, the kernel matrix of training instances by <em>K<sub>S</sub></em> and the kernel matrix of training versus test instances by <em>K<sub>U</sub></em>. The class labels for training and test sets are denoted by <em>y<sub>S</sub></em> and <em>y<sub>U</sub></em> respectively. A candidate weight vector that approximates the optimal <span class="underline">ω</span> is termed <em><span class="underline">w</span></em>. The collection of all such vectors <em><span class="underline">w</span></em> is denoted <em>W</em> while the collection of vectors <em><span class="underline">w</span></em> with best classification performance is <em>B</em>.</p>


<h4>weighted Kernel-based Iterative Estimation of Relevance Algorithm (wKIERA)</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A high level depiction of wKIERA is shown in <a href="#pone-0001806-g001">Fig. 1</a>. The method iteratively optimizes the parameters (<span class="underline">ω</span>,<span class="underline">α</span>) of Eq. (13) by executing the components marked as <em>learn</em> and <em>estimate</em>. We used a kernel perceptron as a supervised learner <a href="#pone.0001806-Freund1">[25]</a> and an estimation of distribution algorithm for the <em>estimate</em> component <a href="#pone.0001806-Pelikan1">[24]</a>. However, the modular design of the wKIERA allows plugging of any linear-threshold kernel classifier and any stochastic optimization algorithm into these components.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The stochastic optimization module for estimation of <span class="underline">ω</span> was designed with a probabilistic model-building strategy known as estimation of distribution algorithm <a href="#pone.0001806-Pelikan1">[24]</a> and is summarized in <a href="#pone-0001806-t003">Table 3</a>. Inputs are a dataset <em>D</em>, the number of candidate weight vectors <em><span class="underline">w</span></em> (<em>poolsize</em>), the maximum number of iterations (<em>maxiter</em>), and the parameters of a base kernel. Depending on the kernel type, this can be the degree of a polynomial kernel <em>d</em> or the width of a RBF kernel ρ. This base kernel will be transformed to a weighted version using every candidate weight vector <em><span class="underline">w</span></em>.</p>
<div class="figure" id="pone-0001806-t003"><div class="img"><a name="pone-0001806-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001806" data-uri="info:doi/10.1371/journal.pone.0001806.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001806.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001806.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001806.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001806.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Weighted Kernel-based Iterative Estimation of Relevance Algorithm (wKIERA).</span></strong></p><span>doi:10.1371/journal.pone.0001806.t003</span></div><a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>First, the pool of weight candidates <em>W</em> is uniformly randomly initialized and the main loop (<a href="#pone-0001806-t003">Table 3</a>) is executed <em>maxiter</em> number of iterations. The variables <em>top</em> and <em>best<span class="underline">w</span></em> are used to trace the candidate with best score across all iterations. On each iteration the set of instance indexes <em>J</em> = {1,2,…,<em>m</em>} is split into two subsets of randomly permuted indexes, <em>S</em> and <em>U</em>. Then a weighted kernel matrix <em>K</em> is computed using the corresponding weight vector <em><span class="underline">w</span></em>, the input vectors <strong><span class="underline">x</span></strong><em><sub>J</sub></em> and the base kernel. The kernel matrix <em>K<sub>S</sub></em> is fed into a kernel perceptron to learn a discriminant function <em>h</em> that classifies the examples in <em>S</em> within a supervised learning framework using the corresponding labels <em>y<sub>S</sub></em>. The fitness of the candidate weight vector <em><span class="underline">w</span></em> is then evaluated with the multi-objective scoring function of Eq. (1) which depends on classification accuracy in the test set using <em>K<sub>U</sub></em> and <em>y<sub>U</sub></em>, and a measure of its length. A matrix <em>B</em> is then created with half the best-scoring weight vectors from <em>W</em>. The matrix <em>B</em> is now used to estimate a uniformly and independently multivariate Gaussian distribution by computing the mean and standard deviation vectors μ and σ. Two additional parameters for noise δ and skewness ξ are set using a predefined schedule of the current iteration number and the <em>top</em> score. At this point a new pool of weight vector candidates <em>W</em> is generated using the estimated probability distribution with added perturbations. A skewed multivariate normally distributed matrix <em>W<sup>new</sup></em>~<em>N</em><sub>δ</sub>(μ,σ+ξ) is used for this purpose. Negative values generated by this distribution are set to zero since only positive values are valid weights ω<em><sub>k</sub></em> in Eq. (10) and Eq. (11). Finally, the best candidate <em>best<span class="underline">w</span></em> is carried over to the next iteration by assigning it to the first slot of the new pool <em>W</em> (as suggested in <a href="#pone.0001806-Davies2">[31]</a>). These steps are repeated a maximum number of iterations or until the algorithm halts for a maximum period of consecutive iterations. At the end of the loop the best candidate <em>best<span class="underline">w</span></em> containing the estimated vector of weights <span class="underline">ω</span> is returned.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>The [δ,ξ]-schedule was defined according to the best parameters found in preliminary experiments. The amount of noise δ added by the random number generator was initialized in 0.2 and linearly declines to zero by the final iteration. This is intended to encourage a broader exploration of the search space at the beginning stages of the algorithm while further exploitation of the feasible subspace is performed in the later stages. On the other hand, the skewness of the distribution ξ is set to zero up to the point where <em>top</em> score achieves a safety-net value of 0.9 when it starts to decrease towards a value of −1. When this happens, the random number generator becomes biased to produce negative weight values which in turn will be set to zero. This is meant to promote downscaling of irrelevant variables in classifiers obtaining high classification scores. A safety-net value of 0.9 will ensure that classifiers with less than 90% accuracy are penalized.</p>


<h4>Scoring function</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The score function guides the search of the wKIERA algorithm. It is defined as a multi-objective function made of an estimate of the accuracy of a weighted kernel classifier and a measure of the size of the weight vector:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The first term in Eq. (1), corresponding to the accuracy of a classifier, computes the proportion of correctly classified examples in an unseen test set. Classifiers with higher rates of accuracy get values close to 1. The second term in Eq. (1) is intended to solve ties between candidates with the same accuracy, in which case those with lower scale factors are preferred. For this purpose the average of <strong><span class="underline">w</span></strong> is used to calculate <em>LEN</em>(<strong><span class="underline">w</span></strong>) = 1−<em>AVG</em>(<strong><span class="underline">w</span></strong>); thus candidates comprising plenty of null weights get length values approaching to 1. We weight the first term of the multi-objective function with 0.99 as classification accuracy should be the dominant criterion of the search.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>We consider other measures of classification performance, including sensitivity (SE) and specificity (SP) of a classifier. They are defined in Eq. (2) and Eq. (3), where TP and TN denote the number of positive and negative correctly classified cases, and FP and FN denote the positive and negative misclassified cases. The accuracy, then, can be computed as Eq. (4). We used TP and TN to plot classifiers in a receiver operator characteristic (ROC) space where the performance (positive diagnostic likelihood ratio) of a classifier is expressed by its true positive rate (TPR = SE) and false positive rate (FPR = 1−SP).<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br></p>


<h4>Linear-Threshold Discriminants</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>A linear-threshold discriminant corresponds to a hyperplane in the space of instances in <em>D</em>, that is, an <em>n</em>-dimensional plane defining two half-spaces. An instance is hence classified as positive or negative depending on the side of the hyperplane it lies on. A hyperplane is characterized by its normal <em>n</em>-dimensional weight vector <strong><span class="underline">w</span></strong> and a bias term <em>b</em> (<em>b</em>≠0 refers to a non-centered hyperplane). A linear discriminant function can be specified as a rule to discriminate instances in <em>D</em>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where 〈·,·〉 denotes inner product. By weighting input variables in <strong><span class="underline">x</span></strong> with <span class="underline">ω</span>, the contribution of variables with non-significant factors to the inner product in Eq. (5) is diminished. The linear discriminant therefore becomes:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>where * denotes element-wise product. The parameters (<strong><span class="underline">w</span></strong>,<em>b</em>) are obtained by solving an optimization problem on the misclassification error incurred by <em>h</em><span class="underline"><sub>ω</sub></span>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>here <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e025&amp;representation=PNG" class="inline-graphic"></span> measures the discrepancies between the predicted and the real label on every instance in <em>D</em>.</p>


<h4>Weighted kernels</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>A kernel is a continuous, symmetrical and positive semi-definite function between two vectors in a given Hilbert space <em>H</em>. Mercer's theorem <a href="#pone.0001806-Mercer1">[32]</a> states that such a function corresponds to the inner product between images of the input vectors in a transformed feature space (usually of a larger dimensionality). Therefore, when vectors from the input space are mapped to a feature space <strong>x</strong><em><sub>i</sub></em>↦φ(<strong><span class="underline">x</span></strong><em><sub>i</sub></em>) using the nonlinear transformation φ(·), their inner products in the feature space becomes 〈φ(<strong><span class="underline">x</span></strong><em><sub>i</sub></em>), φ(<strong><span class="underline">x</span></strong><em><sub>j</sub></em>)〉↦<em>k</em>(<strong><span class="underline">x</span></strong><em><sub>i</sub></em>,<strong><span class="underline">x</span></strong><em><sub>j</sub></em>) where <em>k</em> : <em>H</em>×<em>H</em>↦ℜ is a function mapping a pair of points in <em>H</em> to the real set ℜ. By means of φ(·), nonlinearities in the input space can be solved with linear discriminants in the feature space if a proper function <em>k</em>(·,·) is used. In the present study <em>H</em> is defined by ℜ<em><sup>n</sup></em>.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Two widely-used kernel functions are the Radial Basis Function (RBF) and polynomial kernels defined in Eq. (8) and Eq. (9) respectively:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>where the parameter ρ&gt;0 is the width of a symmetric radial function similar to a Gaussian bell centered in one of the input patterns and the parameter <em>d</em>&gt;0 is the polynomial degree. A weighted version of these kernels assigns a scale factor, 0&lt;ω<em><sub>k</sub></em>&lt;1, for each input dimension as shown in Eq. (10) and Eq. (11) respectively <a href="#pone.0001806-Chapelle1">[19]</a>. In the weighted polynomial kernel the scale factors ω<em><sub>k</sub></em> adjust the contribution of each variable to the inner product. In the weighted RBF kernel ω<em><sub>k</sub></em> shape the width of the radial function in every dimension. Null scale factors prevent the corresponding variables affecting the kernel computation, making them irrelevant in practice.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br></p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>Any kernel defines a so-called Reproducing Kernel Hilbert Space (RKHS) where an inner product between two arbitrary vectors amounts to the evaluation of the correspondfoing kernel function. In this way a hyperplane in the RKHS can be characterized by replacing inner products with kernel functions and hence the linear discriminant of Eq. (5) becomes <a href="#pone.0001806-Cortes1">[33]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e020&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br>and the weighted version of Eq. (6) corresponding to:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e021&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br></p>
<a id="article1.body1.sec4.sec6.p4" name="article1.body1.sec4.sec6.p4"></a><p>The expression <em>k</em>(<strong><span class="underline">x</span></strong><em><sub>i</sub></em> * <span class="underline">ω</span>,<strong><span class="underline">x</span></strong><em><sub>j</sub></em> * <span class="underline">ω</span>) in Eq. (13) matches one of the above-defined weighted kernels which we have denoted <em>k</em><span class="underline"><sub>ω</sub></span>(<strong><span class="underline">x</span></strong><em><sub>i</sub></em>,<strong><span class="underline">x</span></strong><sub>j</sub>). Note that a kernel matrix <em>K</em><span class="underline"><sub>ω</sub></span> can be computed off-line for every pair of instances in <em>D</em>, i.e. as ⌊<em>K</em><span class="underline"><sub>ω</sub></span>⌋<em><sub>ij</sub></em> = <em>k</em><span class="underline"><sub>ω</sub></span>(<strong><span class="underline">x</span></strong><em><sub>i</sub></em>,<strong><span class="underline">x</span></strong><em><sub>j</sub></em>).</p>


<h4>Kernel Perceptron</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>A Perceptron classifier <a href="#pone.0001806-Rosenblatt1">[34]</a>, <a href="#pone.0001806-Minsky1">[35]</a> uses a hyperplane to separate examples from a dataset <em>D</em> onto different half-spaces corresponding to binary classes. The hyperplane is represented by the parameters (<strong><span class="underline">w</span></strong>,<em>b</em>) of Eq. (5) which are learned by a mistake-driven algorithm conducting incremental updates from a stream of instances. It has been shown <a href="#pone.0001806-Herbster1">[36]</a>, <a href="#pone.0001806-Novikoff1">[37]</a> that given two separable sets of positive and negative examples in a Hilbert space, the Perceptron algorithm converges to a discriminant hyperplane with a number of mistakes theoretically bounded in terms of the distance of separation between the sets (also know as their <em>margin</em>). The linear separability constraint which is certainly difficult to ensure in realistic situations, can be solved by using kernel functions to transform the input space to a higher dimensional RHKS <a href="#pone.0001806-Cortes1">[33]</a>. The resulting Kernel Perceptron algorithm <a href="#pone.0001806-Freund1">[25]</a>, is able to learn a linear discriminant with implicit kernel representations as in Eq. (12). Additional advantages of this algorithm include ease of implementation and fast computation; given its incremental character, the number of updates grows as <em>O</em>(<em>n</em>) where <em>n</em> is the number of examples in <em>D</em>.</p>


<h4>Support Vector Machines</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The SVM <a href="#pone.0001806-ShaweTaylor1">[27]</a>, <a href="#pone.0001806-Cortes1">[33]</a>, <a href="#pone.0001806-Cristianini1">[38]</a> is a kernel machine that learns a hyperplane with the maximal margin of separation between vectors of two distinctive classes in a RKHS. The discrimination function of an SVM is similar to that of the Kernel Perceptron and takes the form showed in Eq. (14),<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e022&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>where the coefficients α<em><sub>i</sub></em> and the bias term <em>b</em> are found by solving a constrained quadratic optimization problem aimed to minimize the misclassification rate and the complexity of the classifier while maximizing the margin. Notice that only those patterns whose α<em><sub>i</sub></em>≠0, participate in the computation of Eq. (14) and hence they are called the support vectors. The motivation for maximizing the margin is rooted in the theory of Structural Risk Minimization <a href="#pone.0001806-Cristianini1">[38]</a> and its aim is to maximize the generalization ability of the discriminant by reducing its capacity. In this sense, the SVM learns the optimal separating hyperplane whereas the Kernel Perceptron learns an approximation to that optimum. However the computational complexity of the SVM is quadratic in time since it requires <em>O</em>(<em>n</em><sup>2</sup>) computations to solve the quadratic optimization problem.</p>


<h4>Rank correlation coefficients</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>The Pearson correlation coefficients are computed using Eq. (15) where <em>X<sub>k</sub></em> represents the random variable corresponding to the <em>k</em>-th component of the input instance vectors (<em>k</em> = 1,2,…<em>n</em>) and <em>Y</em> is the random variable representing the class labels.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br></p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>Since only a finite sample of the input instances is available, the estimate of <em>R</em>(<em>k</em>) is given by Eq. (16) where <em>x<sub>ik</sub></em> corresponds to the <em>k</em>-th variable value of the <em>i</em>-th sample and <em>y<sub>i</sub></em> is its class label.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001806.e024&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br></p>


<h4>Source code</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>The proposed method was implemented in Matlab 7.0 including scripts for wKIERA, kernel perceptron, scoring and evaluation functions. The source code is available upon request. For evaluation of SVM classifiers we used the SVMLight <a href="#pone.0001806-Joachims1">[39]</a> library with the MEX-SVMLight interface for Matlab <a href="#pone.0001806-Briggs1">[40]</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Dr. Mark Herbster, Department of Computer Science, UCL, London, UK, for valuable comments and revision of the work. Dr. A. A. Holder, Division of Parasitology, NIMR, London, UK for support and valuable comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DF SR. Performed the experiments: DF DA SR EH. Analyzed the data: DF SR. Contributed reagents/materials/analysis tools: SK DF DA SR. Wrote the paper: DF SR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001806-Baldi1" id="pone.0001806-Baldi1"></a>Baldi P, Hatfield W (2002) DNA Microarrays and Gene Expression:. Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001806-Wagner1" id="pone.0001806-Wagner1"></a>Wagner M, Naik D, Pothen A (2003) Protocols for disease classification from mass spectrometry data. Proteomics 1692–1698.  <ul class="find" data-citedArticleID="1075582" data-doi="10.1002/pmic.200300519"><li><a href="http://dx.doi.org/10.1002/pmic.200300519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protocols+for+disease+classification+from+mass+spectrometry+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protocols+for+disease+classification+from+mass+spectrometry+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001806-Issaq1" id="pone.0001806-Issaq1"></a>Issaq HJ, Veenstra TD, Conrads TP, Felschow D (2002) The SELDI-TOF MS approach to proteomics: protein profiling and biomarker identification. Biochem Biophys Res Comm  292: 587–592.  <ul class="find" data-citedArticleID="1075552" data-doi="10.1006/bbrc.2002.6678"><li><a href="http://dx.doi.org/10.1006/bbrc.2002.6678" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SELDI-TOF+MS+approach+to+proteomics%3A+protein+profiling+and+biomarker+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SELDI-TOF+MS+approach+to+proteomics%3A+protein+profiling+and+biomarker+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001806-Davies1" id="pone.0001806-Davies1"></a>Davies S, Russell S (1994) NP-completeness of searches for smallest possible feature sets.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0001806-Garey1" id="pone.0001806-Garey1"></a>Garey M, Johnson D (1979) Computers and Intractability: A Guide to the Theory of NP-Completeness. W.H. Freeman and Company.   <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0001806-Guyon1" id="pone.0001806-Guyon1"></a>Guyon I, Elisseeff A (2003) An introduction to variable and feature selection. Journal of Machine Learning Research  3: 1157–1182.  <ul class="find" data-citedArticleID="1075546"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+introduction+to+variable+and+feature+selection.&amp;auth=&amp;atitle=An+introduction+to+variable+and+feature+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+introduction+to+variable+and+feature+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+introduction+to+variable+and+feature+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001806-Agranoff1" id="pone.0001806-Agranoff1"></a>Agranoff D, Fernandez-Reyes D, Papadopoulos M, Rojas Galeano S, Herbster M, et al.  (2006) Identification of diagnostic markers for tuberculosis by proteomic fingerprinting of serum. Lancet  368: 1012–1021.  <ul class="find" data-citedArticleID="1075512" data-doi="10.1016/s0140-6736(06)69342-2"><li><a href="http://dx.doi.org/10.1016/s0140-6736(06)69342-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+diagnostic+markers+for+tuberculosis+by+proteomic+fingerprinting+of+serum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+diagnostic+markers+for+tuberculosis+by+proteomic+fingerprinting+of+serum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001806-Wagner2" id="pone.0001806-Wagner2"></a>Wagner M, Naik D, Pothen A, Kasukurti S, Devineni R, et al.  (2004) Computational protein biomarker prediction: a case study for prostate cancer. BMC Bioinformatics  5:   <ul class="find" data-citedArticleID="1075584" data-doi="10.1186/1471-2105-5-26"><li><a href="http://dx.doi.org/10.1186/1471-2105-5-26" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+protein+biomarker+prediction%3A+a+case+study+for+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+protein+biomarker+prediction%3A+a+case+study+for+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001806-Papadopoulos1" id="pone.0001806-Papadopoulos1"></a>Papadopoulos MC, Abel PM, Agranoff D, Stich A, Tarelli E, et al.  (2004) A novel and accurate test for Human African Trypanosomiasis. Lancet  363: 1358–1363.  <ul class="find" data-citedArticleID="1075568" data-doi="10.1016/s0140-6736(04)16046-7"><li><a href="http://dx.doi.org/10.1016/s0140-6736(04)16046-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+and+accurate+test+for+Human+African+Trypanosomiasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+and+accurate+test+for+Human+African+Trypanosomiasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001806-Conrads1" id="pone.0001806-Conrads1"></a>Conrads T, Fusaro V, Ross S, Johann D, Rajapakse V, et al.  (2004) High-resolution serum proteomic features for ovarian cancer detection. Endocrine-Related Cancer 163–178.  <ul class="find" data-citedArticleID="1075524" data-doi="10.1677/erc.0.0110163"><li><a href="http://dx.doi.org/10.1677/erc.0.0110163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+serum+proteomic+features+for+ovarian+cancer+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+serum+proteomic+features+for+ovarian+cancer+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001806-Li1" id="pone.0001806-Li1"></a>Li L, Jian W, Li X, Moser K, Guo Z, et al.  (2005) A robust hybrid between genetic algorithm and support vector machine for extracting an optimal feature gene subset. Genomics 16–23.  <ul class="find" data-citedArticleID="1075556" data-doi="10.1016/j.ygeno.2004.09.007"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2004.09.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+robust+hybrid+between+genetic+algorithm+and+support+vector+machine+for+extracting+an+optimal+feature+gene+subset." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+robust+hybrid+between+genetic+algorithm+and+support+vector+machine+for+extracting+an+optimal+feature+gene+subset.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001806-Liu1" id="pone.0001806-Liu1"></a>Liu J, Gutler G, Li W, Pan Z, Peng S, et al.  (2005) Multiclass cancer classification and biomarker discovery using GA-based algorithms. Bioinformatics  21: 2691–2697.  <ul class="find" data-citedArticleID="1075558" data-doi="10.1093/bioinformatics/bti419"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti419" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiclass+cancer+classification+and+biomarker+discovery+using+GA-based+algorithms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiclass+cancer+classification+and+biomarker+discovery+using+GA-based+algorithms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001806-Frhlich1" id="pone.0001806-Frhlich1"></a>Fröhlich H, Chapelle O, Scholkopf B (2003) Feature Selection for Support Vector Machines by Means of Genetic Algorithms.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001806-Guyon2" id="pone.0001806-Guyon2"></a>Guyon I, Weston J, Barnhill S, Vapnik V (2002) Gene Selection for Cancer Classification using Support Vector Machines. Machine Learning  46: 389–422.  <ul class="find" data-citedArticleID="1075548" data-doi="10.1023/a:1012487302797"><li><a href="http://dx.doi.org/10.1023/a:1012487302797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+Selection+for+Cancer+Classification+using+Support+Vector+Machines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+Selection+for+Cancer+Classification+using+Support+Vector+Machines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001806-Zhang1" id="pone.0001806-Zhang1"></a>Zhang X, Lu X, Shi Q, Xu X, Leung H, et al.  (2006) Recursive SVM feature selection and sample classification for mass-spectometry and microarray data. BMC Bioinformatics  7:   <ul class="find" data-citedArticleID="1075590"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recursive+SVM+feature+selection+and+sample+classification+for+mass-spectometry+and+microarray+data.&amp;auth=&amp;atitle=Recursive+SVM+feature+selection+and+sample+classification+for+mass-spectometry+and+microarray+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recursive+SVM+feature+selection+and+sample+classification+for+mass-spectometry+and+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recursive+SVM+feature+selection+and+sample+classification+for+mass-spectometry+and+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001806-Frhlich2" id="pone.0001806-Frhlich2"></a>Fröhlich H, Zell A (2004) Feature Subset Selection for Support Vector Machines by Incremental Regularized Risk Minimization. pp. 2041–2046.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0001806-Ding1" id="pone.0001806-Ding1"></a>Ding Y, Wilkins D (2006) Improving the performance of SVM-RFE to select genes in microarray data. BMC Bioinformatics.   <ul class="find" data-citedArticleID="1075534" data-doi="10.1186/1471-2105-7-s2-s12"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-s2-s12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improving+the+performance+of+SVM-RFE+to+select+genes+in+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improving+the+performance+of+SVM-RFE+to+select+genes+in+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001806-Bedo1" id="pone.0001806-Bedo1"></a>Bedo J, Sanderson C, Kowalczyk A (2006) An Efficient Alternative to SVM Based Recursive Feature Elimination with Applications in Natural Language Processing and Bioinformatics. Springer Berlin / Heidelberg.   <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0001806-Chapelle1" id="pone.0001806-Chapelle1"></a>Chapelle O, Vapnik V, Bousquet O, Mukherjee S (2002) Choosing Multiple Parameters for Support Vector Machines. Machine Learning  46: 131–159.  <ul class="find" data-citedArticleID="1075522" data-doi="10.1023/a:1012450327387"><li><a href="http://dx.doi.org/10.1023/a:1012450327387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Choosing+Multiple+Parameters+for+Support+Vector+Machines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Choosing+Multiple+Parameters+for+Support+Vector+Machines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001806-VanGestel1" id="pone.0001806-VanGestel1"></a>Van Gestel T, Suykens JAK, De Moor B, Vandewalle J (2001) Automatic Relevance Determination for Least Squares Suppor Vector Machine regression.   <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0001806-Tipping1" id="pone.0001806-Tipping1"></a>Tipping ME (2001) Sparse Bayesian learning and the Relevance Vector Machine. Journal of Machine Learning Research  1: 211–244.  <ul class="find" data-citedArticleID="1075578"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sparse+Bayesian+learning+and+the+Relevance+Vector+Machine.&amp;auth=&amp;atitle=Sparse+Bayesian+learning+and+the+Relevance+Vector+Machine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sparse+Bayesian+learning+and+the+Relevance+Vector+Machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sparse+Bayesian+learning+and+the+Relevance+Vector+Machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001806-RojasGaleano1" id="pone.0001806-RojasGaleano1"></a>Rojas Galeano S, Fernandez-Reyes D (2005) Adapting Multiple Kernel Parameters for Support Vector Machines using Genetic Algorithms.   <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001806-Friedrichs1" id="pone.0001806-Friedrichs1"></a>Friedrichs F, Igel C (2005) Evolutionary tuning of multiple SVM parameters. Neurocomputing 107–117.  <ul class="find" data-citedArticleID="1075542" data-doi="10.1016/j.neucom.2004.11.022"><li><a href="http://dx.doi.org/10.1016/j.neucom.2004.11.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+tuning+of+multiple+SVM+parameters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+tuning+of+multiple+SVM+parameters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001806-Pelikan1" id="pone.0001806-Pelikan1"></a>Pelikan P, Sastry K, Cantu-Paz E (2006) Scalable Optimization via Probabilistic Modeling: From Algorithms to Applications. Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0001806-Freund1" id="pone.0001806-Freund1"></a>Freund Y, Schapire R (1999) Large Margin Classification Using the Perceptron Algorithm. Machine Learning  37: 277–296.  <ul class="find" data-citedArticleID="1075536" data-doi="10.1145/279943.279985"><li><a href="http://dx.doi.org/10.1145/279943.279985" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large+Margin+Classification+Using+the+Perceptron+Algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large+Margin+Classification+Using+the+Perceptron+Algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001806-Whitley1" id="pone.0001806-Whitley1"></a>Whitley D (1994) A genetic algorithm tutorial. Statistics and Computing 65–85.  <ul class="find" data-citedArticleID="1075588" data-doi="10.1007/bf00175354"><li><a href="http://dx.doi.org/10.1007/bf00175354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic+algorithm+tutorial." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic+algorithm+tutorial.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001806-ShaweTaylor1" id="pone.0001806-ShaweTaylor1"></a>Shawe-Taylor J, Cristianini N (2004) Kernel Methods for Pattern Analysis. Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0001806-Weston1" id="pone.0001806-Weston1"></a>Weston J, Mukherjee S, Chapelle O, Pontil M, Poggio T, et al.  (2000) Feature Selection for SVMs.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0001806-Alon1" id="pone.0001806-Alon1"></a>Alon U, Barkai N, Notterman DA, Gish K, Ybarra S, et al.  (1999) Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays. Proc Natl Acad Sci U S A  96: 6745–6750.  <ul class="find" data-citedArticleID="1075514"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays.&amp;auth=&amp;atitle=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001806-Nutt1" id="pone.0001806-Nutt1"></a>Nutt C, Mani D, Betensky R, Tamayo P, Cairncross G, et al.  (2003) Gene expression-based classification of malignant gliomas correlates better with survival than histological classification. Cancer Research 1602–1607.  <ul class="find" data-citedArticleID="1075566"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification.&amp;auth=&amp;atitle=Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001806-Davies2" id="pone.0001806-Davies2"></a>Davies L (1991) Handbook of Genetic Algortihms. Van Nostrand Reinhold Company.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001806-Mercer1" id="pone.0001806-Mercer1"></a>Mercer J (1909) Functions of positive and negative type and their connection with the theory of integral equations. Philos Trans Roy Soc London.   <ul class="find" data-citedArticleID="1075560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functions+of+positive+and+negative+type+and+their+connection+with+the+theory+of+integral+equations.&amp;auth=&amp;atitle=Functions+of+positive+and+negative+type+and+their+connection+with+the+theory+of+integral+equations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functions+of+positive+and+negative+type+and+their+connection+with+the+theory+of+integral+equations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functions+of+positive+and+negative+type+and+their+connection+with+the+theory+of+integral+equations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001806-Cortes1" id="pone.0001806-Cortes1"></a>Cortes C, Vapnik V (1995) Support vector networks. Machine Learning  20: 273–297.  <ul class="find" data-citedArticleID="1075526" data-doi="10.1007/bf00994018"><li><a href="http://dx.doi.org/10.1007/bf00994018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Support+vector+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Support+vector+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001806-Rosenblatt1" id="pone.0001806-Rosenblatt1"></a>Rosenblatt F (1958) The Perceptron: A Probabilistic Model for Information Storage and Organization in the Brain. Psychological Review  65: 386–408.  <ul class="find" data-citedArticleID="1075574" data-doi="10.1037/h0042519"><li><a href="http://dx.doi.org/10.1037/h0042519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Perceptron%3A+A+Probabilistic+Model+for+Information+Storage+and+Organization+in+the+Brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Perceptron%3A+A+Probabilistic+Model+for+Information+Storage+and+Organization+in+the+Brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001806-Minsky1" id="pone.0001806-Minsky1"></a>Minsky M, Papert S (1969) Perceptrons. MIT Press.   <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001806-Herbster1" id="pone.0001806-Herbster1"></a>Herbster M (2001) Learning Additive Models Online with Fast Evaluating Kernels. pp. 444–460.  <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0001806-Novikoff1" id="pone.0001806-Novikoff1"></a>Novikoff A (1962) On convergence proofs on perceptrons. Symposium on the Mathematical Theory of Automata 615–622.  <ul class="find" data-citedArticleID="1075564"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+convergence+proofs+on+perceptrons.&amp;auth=&amp;atitle=On+convergence+proofs+on+perceptrons." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+convergence+proofs+on+perceptrons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+convergence+proofs+on+perceptrons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001806-Cristianini1" id="pone.0001806-Cristianini1"></a>Cristianini N, Shawe-Taylor J (2000) An Introduction to Support Vector Machines and other kernel-based learning methods. Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001806-Joachims1" id="pone.0001806-Joachims1"></a>Joachims T (1999)  Making large-scale SVM learning practical. In: Sholkopf B, Burges C, Smola A, editors. Advances in Kernel Methods - Support Vector Learning. MIT Press.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001806-Briggs1" id="pone.0001806-Briggs1"></a>Briggs T (2005) MATLAB/MEX Interface to SVMlight.  <ul class="find" data-citedArticleID="1075520"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MATLAB%2FMEX+Interface+to+SVMlight.&amp;auth=&amp;atitle=MATLAB%2FMEX+Interface+to+SVMlight." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MATLAB%2FMEX+Interface+to+SVMlight." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MATLAB%2FMEX+Interface+to+SVMlight.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.XML" value="100201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.PDF" value="444983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e001.PNG" value="5043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e001.TIF" value="20952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e002.PNG" value="5043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e002.TIF" value="20952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e003.PNG" value="5043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e003.TIF" value="20952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g001.PNG_L" value="227441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g001.PNG_M" value="88647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g001.PNG_S" value="13589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g001.TIF" value="675792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g001.PNG_I" value="38901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t001.PNG_L" value="420978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t001.PNG_M" value="252556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t001.PNG_S" value="18442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t001.TIF" value="1722752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t001.PNG_I" value="77014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e004.PNG" value="6247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e004.TIF" value="22192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e005.PNG" value="6336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e005.TIF" value="22280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e006.PNG" value="6375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e006.TIF" value="22348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e007.PNG" value="6506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e007.TIF" value="22480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g002.PNG_L" value="523823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g002.PNG_M" value="75175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g002.PNG_S" value="14266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g002.TIF" value="1637042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g002.PNG_I" value="56179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g003.PNG_L" value="681434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g003.PNG_M" value="89587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g003.PNG_S" value="13886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g003.TIF" value="1987874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g003.PNG_I" value="45840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g004.PNG_L" value="534714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g004.PNG_M" value="78634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g004.PNG_S" value="14460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g004.TIF" value="1665878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.g004.PNG_I" value="58632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e008.PNG" value="5043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e008.TIF" value="20952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t002.PNG_L" value="100038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t002.PNG_M" value="56314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t002.PNG_S" value="9888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t002.TIF" value="452234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t002.PNG_I" value="17341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t003.PNG_L" value="120436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t003.PNG_M" value="101933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t003.PNG_S" value="15614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t003.TIF" value="714550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.t003.PNG_I" value="65200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e009.PNG" value="9900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e009.TIF" value="25940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e010.PNG" value="7890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e010.TIF" value="23796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e011.PNG" value="7890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e011.TIF" value="23804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e012.PNG" value="10455"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e012.TIF" value="26168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e013.PNG" value="8630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e013.TIF" value="24508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e014.PNG" value="9197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e014.TIF" value="25104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e015.PNG" value="9880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e015.TIF" value="25632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e025.PNG" value="6801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e025.TIF" value="22700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e016.PNG" value="9584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e016.TIF" value="25524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e017.PNG" value="8086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e017.TIF" value="23952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e018.PNG" value="11458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e018.TIF" value="26980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e019.PNG" value="11244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e019.TIF" value="26844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e020.PNG" value="10893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e020.TIF" value="26560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e021.PNG" value="11081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e021.TIF" value="26828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e022.PNG" value="11196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e022.TIF" value="26896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e023.PNG" value="12654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e023.TIF" value="28460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e024.PNG" value="15691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001806.e024.TIF" value="31380"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001806&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001806" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001806&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001806&volume=&issue=&title=Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions&author_name=Sergio%20Rojas-Galeano%2C%20Emily%20Hsieh%2C%20Dan%20Agranoff%2C%20Sanjeev%20Krishna%2C%20Delmiro%20Fernandez-Reyes&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001806" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001806" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001806" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001806&amp;t=Estimation%20of%20Relevant%20Variables%20on%20High-Dimensional%20Biological%20Patterns%20Using%20Iterated%20Weighted%20Kernel%20Functions" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001806&title=Estimation%20of%20Relevant%20Variables%20on%20High-Dimensional%20Biological%20Patterns%20Using%20Iterated%20Weighted%20Kernel%20Functions&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001806&amp;title=Estimation%20of%20Relevant%20Variables%20on%20High-Dimensional%20Biological%20Patterns%20Using%20Iterated%20Weighted%20Kernel%20Functions" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001806" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001806&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Estimation of Relevant Variables on High-Dimensional Biological Patterns Using Iterated Weighted Kernel Functions';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001806';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Estimation%20of%20Relevant%20Variables%20on%20High-Dimensional%20Biological%20Patterns%20Using%20Iterated%20Weighted%20Kernel%20Functions http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001806" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001806" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Algorithms%22" title="Search for articles in the subject area:'Algorithms'"><div class="flagText">Algorithms</div></a>
              <div data-categoryid="32621" data-articleid="26436"
                   data-categoryname="Algorithms"
                   class="flagImage" title="Flag 'Algorithms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Kernel+functions%22" title="Search for articles in the subject area:'Kernel functions'"><div class="flagText">Kernel functions</div></a>
              <div data-categoryid="36449" data-articleid="26436"
                   data-categoryname="Kernel functions"
                   class="flagImage" title="Flag 'Kernel functions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Kernel+methods%22" title="Search for articles in the subject area:'Kernel methods'"><div class="flagText">Kernel methods</div></a>
              <div data-categoryid="36465" data-articleid="26436"
                   data-categoryname="Kernel methods"
                   class="flagImage" title="Flag 'Kernel methods' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="26436"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polynomials%22" title="Search for articles in the subject area:'Polynomials'"><div class="flagText">Polynomials</div></a>
              <div data-categoryid="32347" data-articleid="26436"
                   data-categoryname="Polynomials"
                   class="flagImage" title="Flag 'Polynomials' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomic+databases%22" title="Search for articles in the subject area:'Proteomic databases'"><div class="flagText">Proteomic databases</div></a>
              <div data-categoryid="43483" data-articleid="26436"
                   data-categoryname="Proteomic databases"
                   class="flagImage" title="Flag 'Proteomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Support+vector+machines%22" title="Search for articles in the subject area:'Support vector machines'"><div class="flagText">Support vector machines</div></a>
              <div data-categoryid="23025" data-articleid="26436"
                   data-categoryname="Support vector machines"
                   class="flagImage" title="Flag 'Support vector machines' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tuberculosis%22" title="Search for articles in the subject area:'Tuberculosis'"><div class="flagText">Tuberculosis</div></a>
              <div data-categoryid="47969" data-articleid="26436"
                   data-categoryname="Tuberculosis"
                   class="flagImage" title="Flag 'Tuberculosis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9505'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8454'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8800&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=18713">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
