

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Correction of Population Stratification in Large Multi-Ethnic Association Studies</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001382"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001382" />

    <meta name="citation_title" content="Correction of Population Stratification in Large Multi-Ethnic Association Studies"/>
    <meta itemprop="name" content="Correction of Population Stratification in Large Multi-Ethnic Association Studies"/>

      <meta name="citation_author" content="David Serre"/>
            <meta name="citation_author_institution" content="Genome Quebec Innovation Centre, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Alexandre Montpetit"/>
            <meta name="citation_author_institution" content="Genome Quebec Innovation Centre, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Guillaume Paré"/>
            <meta name="citation_author_institution" content="Genome Quebec Innovation Centre, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="James C. Engert"/>
            <meta name="citation_author_institution" content="Department of Medicine, McGill University, Montreal, Quebec, Canada"/>
      <meta name="citation_author" content="Salim Yusuf"/>
            <meta name="citation_author_institution" content="Department of Medicine, McMaster University, Hamilton, Ontario, Canada"/>
      <meta name="citation_author" content="Bernard Keavney"/>
            <meta name="citation_author_institution" content="Institute of Human Genetics, Newcastle upon Tyne, United Kingdom"/>
      <meta name="citation_author" content="Thomas J. Hudson"/>
            <meta name="citation_author_institution" content="Ontario Institute for Cancer Research, Toronto, Ontario, Canada"/>
      <meta name="citation_author" content="Sonia Anand"/>
            <meta name="citation_author_institution" content="Department of Human Genetics, McGill University, Montreal, Quebec, Canada"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001382.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1382"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=1; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=MR Nelson; citation_author=M Klotsman; citation_author=AM McNeill; citation_author=Y Maruyama; citation_author=CE Bowman; citation_number=2; " />
      <meta name="citation_reference" content="citation_title=Genes, environment and the value of prospective cohort studies.; citation_author=TA Manolio; citation_author=JE Bailey-Wilson; citation_author=FS Collins; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=3; citation_pages=812-820; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic epidemiology and public health: hope, hype, and future prospects.; citation_author=G Davey Smith; citation_author=S Ebrahim; citation_author=S Lewis; citation_author=AL Hansell; citation_author=LJ Palmer; citation_journal_title=Lancet; citation_volume=366; citation_number=4; citation_pages=1484-1498; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Effect of potentially modifiable risk factors associated with myocardial infarction in 52 countries (the INTERHEART study): case-control study.; citation_author=S Yusuf; citation_author=S Hawken; citation_author=S Ounpuu; citation_author=T Dans; citation_author=A Avezum; citation_journal_title=Lancet; citation_volume=364; citation_number=5; citation_pages=937-952; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A human genome diversity cell line panel.; citation_author=HM Cann; citation_author=C de Toma; citation_author=L Cazes; citation_author=MF Legrand; citation_author=V Morel; citation_journal_title=Science; citation_volume=296; citation_number=6; citation_pages=261-262; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetic Analysis of 103 Candidate Genes for Coronary Artery Disease and Associated Phenotypes in a Founder Population Reveals a New Association between Endothelin-1 and High-Density Lipoprotein Cholesterol.; citation_author=G Pare; citation_author=D Serre; citation_author=D Brisson; citation_author=SS Anand; citation_author=A Montpetit; citation_journal_title=Am J Hum Genet; citation_volume=80; citation_number=7; citation_pages=673-682; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.; citation_author=CS Carlson; citation_author=MA Eberle; citation_author=MJ Rieder; citation_author=Q Yi; citation_author=L Kruglyak; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=8; citation_pages=106-120; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Highly parallel SNP genotyping.; citation_author=JB Fan; citation_author=A Oliphant; citation_author=R Shen; citation_author=BG Kermani; citation_author=CA Garcia; citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology; citation_volume=68; citation_number=9; citation_pages=69-78; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ML-Relate: a computer program for maximum likelihood estimation of relatedness and relationship.; citation_author=ST Kalinowski; citation_author=AP Wagner; citation_author=LM Taper; citation_journal_title=Molecular Ecology Notes; citation_volume=6; citation_number=10; citation_pages=576-579; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=155; citation_number=11; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evidence for gradients of human genetic diversity within and among continents.; citation_author=D Serre; citation_author=S Paabo; citation_journal_title=Genome Res; citation_volume=14; citation_number=12; citation_pages=1679-1685; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic data analysis.; citation_author=BS Weir; citation_number=13; citation_date=1996; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=Genomic control for association studies.; citation_author=B Devlin; citation_author=K Roeder; citation_journal_title=Biometrics; citation_volume=55; citation_number=14; citation_pages=997-1004; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Confounding from Cryptic Relatedness in Case-Control Association Studies.; citation_author=BF Voight; citation_author=JK Pritchard; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=15; citation_pages=e32; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Obesity and the risk of myocardial infarction in 27,000 participants from 52 countries: a case-control study.; citation_author=S Yusuf; citation_author=S Hawken; citation_author=S Ounpuu; citation_author=L Bautista; citation_author=MG Franzosi; citation_journal_title=Lancet; citation_volume=366; citation_number=16; citation_pages=1640-1649; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Association of psychosocial risk factors with risk of acute myocardial infarction in 11119 cases and 13648 controls from 52 countries (the INTERHEART study): case-control study.; citation_author=A Rosengren; citation_author=S Hawken; citation_author=S Ounpuu; citation_author=K Sliwa; citation_author=M Zubaid; citation_journal_title=Lancet; citation_volume=364; citation_number=17; citation_pages=953-962; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Familial hypercholesterolemia and coronary heart disease: a HuGE association review.; citation_author=MA Austin; citation_author=CM Hutter; citation_author=RL Zimmern; citation_author=SE Humphries; citation_journal_title=Am J Epidemiol; citation_volume=160; citation_number=18; citation_pages=421-429; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Apolipoprotein E: far more than a lipid transport protein.; citation_author=RW Mahley; citation_author=SC Rall; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=1; citation_number=19; citation_pages=507-537; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Unbiased methods for population-based association studies.; citation_author=B Devlin; citation_author=K Roeder; citation_author=SA Bacanu; citation_journal_title=Genet Epidemiol; citation_volume=21; citation_number=20; citation_pages=273-284; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic dissection of complex traits.; citation_author=ES Lander; citation_author=NJ Schork; citation_journal_title=Science; citation_volume=265; citation_number=21; citation_pages=2037-2048; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Assessing the impact of population stratification on genetic association studies.; citation_author=ML Freedman; citation_author=D Reich; citation_author=KL Penney; citation_author=GJ McDonald; citation_author=AA Mignault; citation_journal_title=Nat Genet; citation_volume=36; citation_number=22; citation_pages=388-393; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The effects of human population structure on large genetic association studies.; citation_author=J Marchini; citation_author=LR Cardon; citation_author=MS Phillips; citation_author=P Donnelly; citation_journal_title=Nat Genet; citation_volume=36; citation_number=23; citation_pages=512-517; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic relatedness analysis: modern data and new challenges.; citation_author=BS Weir; citation_author=AD Anderson; citation_author=AB Hepler; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=24; citation_pages=771-780; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA-based methods for pedigree reconstruction and kinship analysis in natural population.; citation_author=MS Blouin; citation_journal_title=Trends in Ecology and Evolution; citation_volume=18; citation_number=25; citation_pages=503-511; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Variability of the genetic contribution of Quebec population founders associated to some deleterious genes.; citation_author=E Heyer; citation_author=M Tremblay; citation_journal_title=Am J Hum Genet; citation_volume=56; citation_number=26; citation_pages=970-978; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Population stratification and spurious allelic association.; citation_author=LR Cardon; citation_author=LJ Palmer; citation_journal_title=Lancet; citation_volume=361; citation_number=27; citation_pages=598-604; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Case-control studies of association in structured or admixed populations.; citation_author=JK Pritchard; citation_author=P Donnelly; citation_journal_title=Theor Popul Biol; citation_volume=60; citation_number=28; citation_pages=227-237; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A genome-wide association study identifies novel risk loci for type 2 diabetes.; citation_author=R Sladek; citation_author=G Rocheleau; citation_author=J Rung; citation_author=C Dina; citation_author=L Shen; citation_journal_title=Nature; citation_volume=445; citation_number=29; citation_pages=881-885; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=An evolutionary framework for common diseases: the ancestral-susceptibility model.; citation_author=A Di Rienzo; citation_author=RR Hudson; citation_journal_title=Trends Genet; citation_volume=21; citation_number=30; citation_pages=596-601; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Differential susceptibility to hypertension is due to selection during the out-of-Africa expansion.; citation_author=JH Young; citation_author=YP Chang; citation_author=JD Kim; citation_author=JP Chretien; citation_author=MJ Klag; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=31; citation_pages=e82; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CYP3A variation and the evolution of salt-sensitivity variants.; citation_author=EE Thompson; citation_author=H Kuttab-Boulos; citation_author=D Witonsky; citation_author=L Yang; citation_author=BA Roe; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=32; citation_pages=1059-1069; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Natural selection and population history in the human angiotensinogen gene (AGT): 736 complete AGT sequences in chromosomes from around the world.; citation_author=T Nakajima; citation_author=S Wooding; citation_author=T Sakagami; citation_author=M Emi; citation_author=K Tokunaga; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=33; citation_pages=898-916; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Population Structure and Eigenanalysis.; citation_author=N Patterson; citation_author=AL Price; citation_author=D Reich; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=34; citation_pages=e190; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic structure, self-identified race/ethnicity, and confounding in case-control association studies.; citation_author=H Tang; citation_author=T Quertermous; citation_author=B Rodriguez; citation_author=SL Kardia; citation_author=X Zhu; citation_journal_title=Am J Hum Genet; citation_volume=76; citation_number=35; citation_pages=268-275; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analyses of genetic structure of Tibeto-Burman populations reveals sex-biased admixture in southern Tibeto-Burmans.; citation_author=B Wen; citation_author=X Xie; citation_author=S Gao; citation_author=H Li; citation_author=H Shi; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=36; citation_pages=856-865; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA polymorphism reveals hidden heterogeneity within some Asian populations.; citation_author=R Chakraborty; citation_journal_title=Am J Hum Genet; citation_volume=47; citation_number=37; citation_pages=87-94; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Color and genomic ancestry in Brazilians.; citation_author=FC Parra; citation_author=RC Amado; citation_author=JR Lambertucci; citation_author=J Rocha; citation_author=CM Antunes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=38; citation_pages=177-182; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Demonstrating stratification in a European American population.; citation_author=CD Campbell; citation_author=EL Ogburn; citation_author=KL Lunetta; citation_author=HN Lyon; citation_author=ML Freedman; citation_journal_title=Nat Genet; citation_volume=37; citation_number=39; citation_pages=868-872; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Population stratification in epidemiologic studies of common genetic variants and cancer: quantification of bias.; citation_author=S Wacholder; citation_author=N Rothman; citation_author=N Caporaso; citation_journal_title=J Natl Cancer Inst; citation_volume=92; citation_number=40; citation_pages=1151-1158; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Counterpoint: bias from population stratification is not a major threat to the validity of conclusions from epidemiological studies of common polymorphisms and cancer.; citation_author=S Wacholder; citation_author=N Rothman; citation_author=N Caporaso; citation_journal_title=Cancer Epidemiol Biomarkers Prev; citation_volume=11; citation_number=41; citation_pages=513-520; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Diabetes mellitus in the Pima Indians: genetic and evolutionary considerations.; citation_author=WC Knowler; citation_author=DJ Pettitt; citation_author=PH Bennett; citation_author=RC Williams; citation_journal_title=Am J Phys Anthropol; citation_volume=62; citation_number=42; citation_pages=107-114; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Population stratification confounds genetic association studies among Latinos.; citation_author=S Choudhry; citation_author=NE Coyle; citation_author=H Tang; citation_author=K Salari; citation_author=D Lind; citation_journal_title=Hum Genet; citation_volume=118; citation_number=43; citation_pages=652-664; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Convergent adaptation of human lactase persistence in Africa and Europe.; citation_author=SA Tishkoff; citation_author=FA Reed; citation_author=A Ranciaro; citation_author=BF Voight; citation_author=CC Babbitt; citation_journal_title=Nat Genet; citation_volume=39; citation_number=44; citation_pages=31-40; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001382" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Correction of Population Stratification in Large Multi-Ethnic Association Studies"/>
    <meta name="twitter:description" content="BackgroundThe vast majority of genetic risk factors for complex diseases have, taken individually, a small effect on the end phenotype. Population-based association studies therefore need very large sample sizes to detect significant differences between affected and non-affected individuals. Including thousands of affected individuals in a study requires recruitment in numerous centers, possibly from different geographic regions. Unfortunately such a recruitment strategy is likely to complicate the study design and to generate concerns regarding population stratification.Methodology/Principal FindingsWe analyzed 9,751 individuals representing three main ethnic groups - Europeans, Arabs and South Asians - that had been enrolled from 154 centers involving 52 countries for a global case/control study of acute myocardial infarction. All individuals were genotyped at 103 candidate genes using 1,536 SNPs selected with a tagging strategy that captures most of the genetic diversity in different populations. We show that relying solely on self-reported ethnicity is not sufficient to exclude population stratification and we present additional methods to identify and correct for stratification.Conclusions/SignificanceOur results highlight the importance of carefully addressing population stratification and of carefully &ldquo;cleaning&rdquo; the sample prior to analyses to obtain stronger signals of association and to avoid spurious results."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001382.g003"/>

  <meta property="og:title" content="Correction of Population Stratification in Large Multi-Ethnic Association Studies" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001382" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3366'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4658'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9507&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001382">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001382" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001382&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Correction of Population Stratification in Large Multi-Ethnic Association Studies
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Serre, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Quebec Innovation Centre, McGill University, Montreal, Quebec, Canada
                </p>

                <p>
                    Current address: Genomic Medicine Institute, Lerner Research Institute, Cleveland Clinic, Cleveland, Ohio, United States of America,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexandre Montpetit, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Quebec Innovation Centre, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guillaume Paré, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Quebec Innovation Centre, McGill University, Montreal, Quebec, Canada
                </p>

                <p>
                    Current address: Division of Preventive Medicine, Brigham and Women's Hospital, Boston, Massachusetts, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James C. Engert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medicine, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Salim Yusuf, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medicine, McMaster University, Hamilton, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernard Keavney, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Human Genetics, Newcastle upon Tyne, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas J. Hudson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:tom.hudson@oicr.on.ca">tom.hudson@oicr.on.ca</a></p>

                <p>Affiliation:
                  Ontario Institute for Cancer Research, Toronto, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sonia Anand
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, McGill University, Montreal, Quebec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001382</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001382" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001382" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001382" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001382" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001382-t001" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001382-g001" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001382-g002" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001382-g003" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001382-t002" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001382">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001382" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The vast majority of genetic risk factors for complex diseases have, taken individually, a small effect on the end phenotype. Population-based association studies therefore need very large sample sizes to detect significant differences between affected and non-affected individuals. Including thousands of affected individuals in a study requires recruitment in numerous centers, possibly from different geographic regions. Unfortunately such a recruitment strategy is likely to complicate the study design and to generate concerns regarding population stratification.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We analyzed 9,751 individuals representing three main ethnic groups - Europeans, Arabs and South Asians - that had been enrolled from 154 centers involving 52 countries for a global case/control study of acute myocardial infarction. All individuals were genotyped at 103 candidate genes using 1,536 SNPs selected with a tagging strategy that captures most of the genetic diversity in different populations. We show that relying solely on self-reported ethnicity is not sufficient to exclude population stratification and we present additional methods to identify and correct for stratification.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results highlight the importance of carefully addressing population stratification and of carefully “cleaning” the sample prior to analyses to obtain stronger signals of association and to avoid spurious results.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Serre D, Montpetit A, Paré G, Engert JC, Yusuf S, et al.  (2008) Correction of Population Stratification in Large Multi-Ethnic Association Studies. PLoS ONE 3(1):
          e1382.
            doi:10.1371/journal.pone.0001382</p><p><strong>Academic Editor: </strong>Wim Crusio, Centre National de la Recherche Scientifique, France</p><p><strong>Received:</strong> August 29, 2007; <strong>Accepted:</strong> December 7, 2007; <strong>Published:</strong> January 2, 2008</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>The INTERHEART study is funded by the Canadian Institutes of Health Research, the Heart and Stroke Foundation of Ontario, the International Clinical Epidemiology Network and through unrestricted grants from several pharmaceutical companies. This work was supported by Genome Canada and Genome Quebec. TJH is the recipient of a Clinician-Scientist Award in Translational Research by the Burroughs Wellcome Fund and an Investigator Award from CIHR. SY holds an endowed chair from the Heart and Stroke foundation of Ontario. SA holds a clinican scientist award from the CIHR. JCE is a research scholar of the Fonds de la Recherche en Santé du Québec (FRSQ).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Complex diseases result from the intricate interactions of multiple environmental and genetic factors. In most cases, common genetic risk factors explain, individually, only a small proportion of the variance of quantitative traits and show modest associations between affected and non-affected individuals. Currently, most association studies include several hundred cases and controls from one single population, but the sample sizes are out of necessity increasing as a result of the expected relatively modest associations. In addition, the recent release of detailed descriptions of genetic diversity in non-European populations, such as those provided by the International HapMap project <a href="#pone.0001382-The1">[1]</a>, will shift the focus from mostly Caucasian-centered studies to diverse populations from various geographic origins. For example, GlaxoSmithKline recently started an initiative to generate and publicly release large-scale genotype information from samples collected around the world <a href="#pone.0001382-Nelson1">[2]</a>. This is appropriate since the majority of the global health burden is in low and middle income countries that include many individuals of non-European origins. Therefore studies are needed to examine the association of genetic markers for various diseases in multiple ethnic groups. Another trend affecting the recruitment strategy of genetic/epidemiologic studies is the collection of biological materials (i.e. blood and DNA) from a very large number of individuals (i.e. several hundreds of thousands) regardless of their health status. These prospective cohort studies will later allow designing nested case/control studies for any disease that is relatively common in the population <a href="#pone.0001382-Manolio1">[3]</a>, <a href="#pone.0001382-DaveySmith1">[4]</a>. All these changes in recruitment strategies will require the development of specific methods for analyzing multi-ethnic and/or multi-center samples. Here, we describe practical methods for adequately designing and conducting population-based association studies with multi-center recruitment in which a large number of markers are genotyped. We use as an example more than 9,000 individuals (about half of whom are cases of first acute myocardial infarction and half are matched controls) from three ethnic groups recruited from 154 centers in the INTERHEART study and genotyped at 1,536 SNPs in 103 candidate genes. We describe an approach to efficiently select a set of tagging SNPs that captures most of the genetic diversity in populations with different allele frequencies and linkage disequilibrium patterns, and present several methods to efficiently identify and correct possible problems arising from population stratification and relatedness among subjects.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Samples</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We analyzed individuals recruited for the INTERHEART study <a href="#pone.0001382-Yusuf1">[5]</a>, a global case/control study of risk factors for acute myocardial infarction (MI) involving 29,972 individuals recruited from 262 centers in 52 countries. Informed written consent to obtain the baseline information and to collect and store the genetic and other biologic specimens was obtained from 21,508 individuals (including all individuals analyzed in this study). INTERHEART was approved by appropriate regulatory and ethics committees in all participating countries and centers and by the Institutional Review Board of McGill University Faculty of Medicine. To identify incident cases of acute MI, all patients, irrespective of age, admitted to the coronary care unit (or an equivalent cardiology ward) within 24 hours of symptom onset were screened. Cases were eligible if they had characteristic symptoms plus electrocardiogram changes indicative of a new MI (new pathologic Q waves, at least 1 mm ST elevation in any 2 or more contiguous limb leads or a new left bundle branch block, or new persistent ST-T wave changes diagnostic of a non-Q wave MI) or a plasma level of cardiac troponin level above that considered normal in the hospital/institution where the patient was registered. For each case, at least one control of the same age (±5 years) and sex was recruited from the same centre. Controls were defined as individuals who had no previous diagnosis of heart disease or history of exertional chest pain. Eligible controls were classified as i) <span class="underline">hospital-based,</span> defined as patients attending the hospital or outpatient clinics for the following reasons: refraction and cataracts, physical check-up, routine pap smear, routine breast exam, elective minor surgery for conditions that were not obviously related to CHD or its risk factors, elective orthopedic surgery (eligibility dependent on ability to complete physical measures), or ii. patients attending the hospital or outpatient clinics for: outpatient fractures, arthritic complaints, plastic surgery, hemorrhoids, hernias, hydroceles, routine colon cancer screening, endoscopy, minor dermatologic disorders; or ii) <span class="underline">community-based,</span> defined as visitors or relatives of a patient from a non-cardiac ward, or an unrelated (not first-degree relative) visitor of a cardiac patient. 58% of controls in INTERHEART were hospital-based and 36% of controls were community-based, and results were similar with both types of controls. In the remainder of the controls, 3% were from an undocumented source, and 3% were recruited through the WHO MONICA study in Göteborg, Sweden. Exclusion criteria for controls were identical to those described for cases. Structured questionnaires were administered to all cases and controls to obtain information on demographic factors (including self-reported ethnicity) as well as socioeconomic and health status. Non-fasting blood samples (20 mL) were drawn within 24 hours of hospital admission from each individual and centrifuged. These were separated into 6 aliquots (2 serum, 2 plasma, 1 citrate and 1 buffy coat) and frozen immediately at −20°C or −70°C after processing. Samples were shipped by courier to the National Blood Storage Site where they are stored in liquid nitrogen (−196°C). Finally, nitrogen vapor tanks were shipped to the Core Laboratory at the Population Health Research Institute (PHRI), Hamilton Canada for central long term storage. Samples collected among Chinese had to remain in China for legal reasons, and were shipped to the core lab in Beijing at the Fu Wai Hospital. We extracted DNA from blood samples using the Gentra Autopure LS isolation system (Gentra Systems Inc, Minneapolis, USA) according to the manufacturer's instructions. For this project, we analyzed 8,975 individuals with self-reported ethnicity defined as “Arab”, “South Asian” or “European” regardless of their geographic locations as well as 316 individuals from Nepal and 460 individuals from Iran who self-reported their ethnicity as “Other Asian”. <a href="#pone-0001382-t001">Table 1</a> shows the countries in which the individuals genotyped have been recruited (see also Supplemental <a href="#pone.0001382.s001">Figure S1</a>). Following the approach used in the original INTERHEART analysis of nine modifiable risk factors and acute MI, we initially grouped people recruited from Nepal who reported their ethnicity as “other Asian” together with South Asian individuals and people recruited from Iran who reported their ethnicity as “other Asian” with Arabs. For sake of simplicity, we will refer to these three datasets as the European, South Asian and Arab population samples throughout the manuscript.</p>
<div class="figure" id="pone-0001382-t001"><div class="img"><a name="pone-0001382-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001382.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001382.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Origin of the individuals used in the study.</span></strong></p><span>doi:10.1371/journal.pone.0001382.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In addition, we genotyped the same SNPs in 1,062 individuals from the HGDP-CEPH Human Genome Diversity Cell Line Panel <a href="#pone.0001382-Cann1">[6]</a> later referred to as HGDP-CEPH panel. These individuals come from 52 populations representing most of the inhabited geographic areas of the world.</p>


<h4>Gene selection</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Candidate genes were selected according to previous reports of association with MI or with one of the nine modifiable risk factors associated with MI <a href="#pone.0001382-Yusuf1">[5]</a>, with a particular emphasis on lipid metabolism (see <a href="#pone.0001382-Pare1">[7]</a> for details).</p>


<h4>SNP selection</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We retrieved the chromosome coordinates of each selected gene according to its refSeq annotation and included 10 kb of upstream and downstream DNA sequence to capture possible <em>cis</em>-regulatory variants. Overlapping gene regions (such as the APOA1-APOA4-APOC3 gene cluster) were concatenated into a single locus. We then retrieved the genotypes for all SNPs genotyped in these regions by the International HapMap project <a href="#pone.0001382-The1">[1]</a> (release 16) for all unrelated individuals from the following populations: individuals from Utah, USA, with northern and western European ancestry (CEU), individuals from the Yoruba people in Ibadan, Nigeria (YRI) and Han Chinese from Beijing, China (CHB). We used LD-select <a href="#pone.0001382-Carlson1">[8]</a> separately for each region (i.e. gene or locus containing several genes) and each population (CEU, CHB and YRI) and identified possible tagging SNPs using a linkage disequilibrium (LD) cut-off of r<sup>2</sup>&gt;0.8 and a minor allele frequency (MAF) of 5%. Finally, based on these results we selected the minimal set of tagging SNPs such that, in each population, every SNP (with MAF&gt;5%) is either directly genotyped or in LD (r<sup>2</sup>&gt;0.8) with one of the genotyped SNPs (see Supplemental <a href="#pone.0001382.s011">Text S1</a> on-line).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In addition, we included all coding non-synonymous SNPs with a MAF larger than 5% (109 cSNPs, including 54 non-tSNPs) as well as SNPs that have been shown in the literature to be directly associated with MI, lipid metabolism or one of the other intermediate phenotypes relevant for the study of MI (145 SNPs, including 81 non-tSNPs). The final list of SNPs genotyped is shown in Supplemental <a href="#pone.0001382.s008">Table S1</a>.</p>


<h4>Genotyping</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>1,536 SNPs were genotyped using Illumina's GoldenGate technology based on allele-specific primer extension followed by highly multiplex PCR using universal primers <a href="#pone.0001382-Fan1">[9]</a>. 1,453 SNPs were successfully genotyped in more than 95% of the individuals of each population sample and are analyzed here (Supplemental <a href="#pone.0001382.s008">Table S1</a>). Individuals genotyped at less than 95% of the SNPs and those with genotypes at markers located on the sex chromosomes incompatible with their reported sex were excluded from further analyses (N = 387, see Supplemental <a href="#pone.0001382.s009">Table S2</a> for a detailed breakdown).</p>


<h4>Estimation of relatedness</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>For each SNP, we determined whether two individuals from the same population sample shared 0, 1 or 2 allele(s) and averaged the allele sharing over all genotyped SNPs. We then compared the proportion of shared alleles for every pair-wise comparison within one population sample to a normal distribution and displayed the results in a quantile-quantile (QQ) plot.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>After excluding identical, or nearly identical, samples (i.e., more than 99% of alleles shared, N = 170), we randomly selected 88 individuals from pairs that shared more than 83% of their alleles. This value corresponds to the relatedness cut-off empirically estimated (see <a href="#s3">Results</a> for details). We successfully genotyped 87 of the individuals at 99 microsatellite loci. We performed a kinship analysis using the ML-relate program <a href="#pone.0001382-Kalinowski1">[10]</a> that uses a Bayesian approach to estimate relationship between pairs of individuals.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>To detect whether cases were significantly more related to each other than the controls to each other (or inversely), we tested in each population sample the distribution of allele sharing among cases to the distribution of allele sharing among controls. We calculated all pair-wise comparisons of allele sharing between two cases and all pair-wise allele sharing between two controls and tested the difference of the means of the two distributions by a Welch Two Sample t-test. We assessed the significance of the t-statistic by 300 permutations: for each population sample, we randomly assigned the individuals into two groups (i.e. regardless of the disease status) and tested the difference between the mean of the two distributions consisting of all possible pair-wise comparisons within each group. To evaluate the power of these analyses, we used unrelated individuals from the Saguenay-Lac St-Jean region (SLSJ, Quebec, Canada) that have been genotyped at the same SNPs <a href="#pone.0001382-Pare1">[7]</a>. We calculated every pair-wise comparison of two individuals from this population and tested this distribution against all pair-wise comparisons of two European controls from the INTERHEART study. We controlled for possible population differentiation by testing the distribution of pair-wise comparisons between one individual from the SLSJ region and one INTERHEART European individual against pair-wise comparisons of the Europeans controls.</p>


<h4>Assessment of population stratification</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To estimate population stratification at a gross level, we used the program STRUCTURE <a href="#pone.0001382-Pritchard1">[11]</a>. This program uses a Bayesian approach to assign individuals into a pre-specified number (<em>K</em>) of “populations” according to their genotypes. These populations are determined such that linkage disequilibrium among unlinked markers and deviations from Hardy-Weinberg are minimized in each of them. We allowed the individuals to be admixed from two or more populations and used a model of correlated allele frequencies which yields stronger clustering <a href="#pone.0001382-Serre1">[12]</a>. Every analysis was replicated thrice and consisted of 200,000 burn-in steps followed by 200,000 Markov Chain Monte Carlo steps. We selected genotypes from SNPs distant from at least 50,000 bp to decrease the chance that they are in LD with each other. Two sets of SNPs were generated: a first set of 133 SNPs randomly selected according to our distance criteria, and a second set composed of the 127 SNPs highly differentiated across populations (based on Fst estimates calculated after grouping the HDGP-CEPH individuals by continents, <a href="#pone.0001382-Weir1">[13]</a>. This second set of SNPs led to higher discrimination power (Supplemental <a href="#pone.0001382.s002">Figure S2</a>) and only results obtained with this set are presented in further analyses. The analyses with STRUCTURE were performed i) separately on each population sample after addition of all individuals from the HGDP-CEPH panel (except for Native American and Oceanian individuals since prior studies of these populations have emphasized the importance of genetic drift leading to large differences in allele frequencies) or ii) on the entire dataset combining all European, Arab and South-Asian INTERHEART individuals.</p>


<h4>Second generation population samples</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We generated second generation population samples by first removing problematic samples and centers: 1) we randomly excluded one individual from each pair of related individuals (N = 131), 2) all individuals that were clustered by STRUCTURE among sub-Saharan Africans or East Asians (N = 104), and 3) all individuals from two centers that showed a very high proportion of problematic samples (including more than 10% discrepancies between reported and genetically-inferred sex, N = 719). In addition, all Nepalese and Iranian individuals were removed from, respectively, the South Asian and the Arab population sample (N = 776). Supplemental <a href="#pone.0001382.s009">Table S2</a> shows the detailed breakdown per population sample.</p>


<h4>Associations between genotypes and Apolipoprotein B concentrations</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>We tested, separately in each population sample, the association between genotypes and ApoB levels in blood for each SNP by an analysis of variance (ANOVA). We used sex, age and waist circumference as covariates in these analyses and excluded individuals with diabetes (defined as self-reported diabetes, on medication pre-admission for diabetes, oral hypoglycemics, insulin or with HbA1c&gt;7%) or on pre-admission medication for lowering cholesterol or blood pressure (inclusion of diabetic individuals led to the same strong associations with ApoB, data not shown). We also included as covariates for some of the analyses the recruitment center and the coefficients of ancestry inferred by STRUCTURE for each individual (using the results obtained by analyzing all individuals from the three population samples together). To estimate whether multiple significant associations from the same region were independent or simply due to LD, we tested hierarchically the associations by successively including the genotypes of stronger associations as covariates.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Identification of related individuals</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To estimate whether the datasets made of individuals of a same self-reported ethnicity were roughly genetically homogenous, we calculated in each population sample the proportion of shared alleles between every pair of individuals. If individuals are sampled randomly from a homogeneous random-mating population, we expect every individual to be, on average, equally distant genetically from everybody else (since information from many unlinked loci is summarized). We thus plotted the distribution of allele sharing for all pair-wise comparisons within each population sample against a normal distribution (see <a href="#pone-0001382-g001">Figure 1</a> for the European individuals and Supplemental <a href="#pone.0001382.s003">Figure S3</a> for the other two datasets). Overall, the distributions appear roughly normal (i.e., we obtain a straight line on the QQ-plot for most of the range) but with significant deviations on both extremes. We observed a dramatic deviation on the right-hand side of the graph for the pairs of individuals with a proportion of allele sharing larger than 0.83 that could indicate sampling of related individuals. The most extreme case in the European sample consists of identical or nearly identical (&gt;99%) genotypes obtained from 16 pairs of supposedly different individuals. The great majority of the pairs with a high proportion of shared alleles (i.e. larger than 0.83) are composed of individuals recruited in the same center. Overall we identified 71 likely related individuals (39 pairs) in the European population sample, 75 (41 pairs) in the South Asian sample and 97 (60 pairs) in the Arab sample. To test whether these individuals were actually related, we randomly selected 87 individuals from pairs with a very high proportion of allele sharing (&gt;0.83), after exclusion of identical or nearly identical DNAs (&gt;0.99), and genotyped them at 99 microsatellite loci. Kinship analyses using the Bayesian approach implemented in ML-relate <a href="#pone.0001382-Kalinowski1">[10]</a> identified the same pairs of related individuals, with different degrees of relatedness: 71 pairs of parent/offspring, 28 full-siblings and 12 half-siblings.</p>
<div class="figure" id="pone-0001382-g001"><div class="img"><a name="pone-0001382-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001382.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001382.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distribution of pair-wise allele sharing among the INTERHEART European individuals.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The graph shows the QQ plot of the distribution of all pair-wise measures of allele sharing against a normal distribution (the red line displays the expectation). The green line shows to the empirical cut-off used to identify related individuals (correspond to an allele sharing larger than 83%). The deviation on the left-hand side of the graph (i.e. low allele sharing) corresponds to pairs of individuals originating from different sub-populations.</p>
<span>doi:10.1371/journal.pone.0001382.g001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The presence of closely related individuals can generate spurious results but is unlikely to strongly influence association studies unless they make up a large proportion of the dataset. On the other hand, the possibility that the cases are, on average, more closely related to each others than are the controls (or inversely) is particularly worrying since this difference in genealogy depth could potentially generate large numbers of false positives <a href="#pone.0001382-Devlin1">[14]</a>, <a href="#pone.0001382-Voight1">[15]</a>. We examined this possibility in our study by testing, in each population sample, the distribution of pair-wise allele sharing among cases against the distribution obtained by pair-wise comparisons among controls. We assessed the significance of the t-statistic obtained by 300 permutations (see Supplemental <a href="#pone.0001382.s004">Figure S4</a> for the Europeans). The difference in allele sharing between cases and controls was not significant in any population sample (p = 0.75 for the European individuals, p = 0.77 in South Asians and p = 0.75 in Arabs). We evaluated the power of these analyses by estimating the distribution of allele sharing among unrelated individuals recruited from a founder population of the Saguenay Lac S<sup>t</sup>-Jean region of Quebec, Canada. For these individuals, we observed a significant increase in allele sharing compared to the European individuals from the INTERHEART study (p&lt;0.005). We validated that this difference resulted from higher average relatedness and not from population differentiation by comparing pair-wise allele sharing between one individual from the SLSJ region and one European from the INTERHEART study and testing this distribution against the within-European distribution of allele sharing (p = 0.16). Non-parametric testing (Two-sample Kolmogorov-Smirnov) yielded, qualitatively, similar results (data not shown).</p>


<h4>Analysis of gross population stratification</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p><a href="#pone-0001382-g001">Figure 1</a> also shows an excess of pairs on the left-hand side of the graph relative to a normal expectation. These pairs of individuals are more different genetically (i.e. less allele shared) than the vast majority of the pairs and this could indicate population stratification in the sample or the presence of individuals with an incorrect self-reported ethnicity. As a first attempt to identify individuals genetically different from the rest of the samples, we analyzed each population sample separately using the program STRUCTURE <a href="#pone.0001382-Pritchard1">[11]</a>. STRUCTURE is a Bayesian algorithm that uses genotype information from all individuals without considering their origins and assigns them into a chosen number of populations (see <a href="#s2">Materials and Methods</a> for details). Since this algorithm relies on the estimation of allele frequencies in different populations, it is easier to identify groups of individuals than a few outliers in a relatively homogenous population <a href="#pone.0001382-Pritchard1">[11]</a>, <a href="#pone.0001382-Serre1">[12]</a>. We thus spiked each population sample before analysis with 914 individuals from the HGDP-CEPH panel <a href="#pone.0001382-Cann1">[6]</a> originating from several geographic locations in Africa, Europe and Asia (see <a href="#s2">Materials and Methods</a>). These individuals also enable us to estimate the discrimination power of our analysis and the level of population differentiation that can be identified. Using 127 SNPs highly differentiated among populations from different continents (see <a href="#s2">Materials and Methods</a>), we were able to pin-point a few individuals whose genotypes were more compatible with an ancestry from Sub-Saharan Africa or South-East Asia than their “European”, “Arab” or “South-Asian” self-reported origin (Supplemental <a href="#pone.0001382.s010">Table S3</a>). Interestingly, in many of these cases, the ancestry inferred from the genotypes best fitted the individual's geographic origin than his/her self-reported ethnicity. For example, three individuals from Zimbabwe self-described as Europeans displayed very high coefficient of ancestry from the African population (i.e. larger than 90%). In addition, a large proportion of the individuals recruited from Nepalese centers cluster with South-East Asian individuals from the HGDP-CEPH (Supplemental <a href="#pone.0001382.s010">Table S3</a>). All Nepalese individuals self-reported their ethnicity as “Other Asian” but were analyzed together with “South Asians” based on similarity in cultural practices in previous publications of the INTERHEART study <a href="#pone.0001382-Yusuf1">[5]</a>, <a href="#pone.0001382-Yusuf2">[16]</a>, <a href="#pone.0001382-Rosengren1">[17]</a>. The analysis of each population sample separately lacked power to separate individuals from Europe, the Middle East and South Asia as indicated by the assignments of the HGDP-CEPH individuals (i.e., we did not identify any clustering of the HGDP-CEPH individuals at the sub-continental level). We thus reanalyzed with STRUCTURE all INTERHEART individuals pooled together (after exclusion of the few outliers with Sub-Saharan African or East Asian ancestry). <a href="#pone-0001382-g002">Figure 2</a> shows the results of this analysis using K = 3 populations. With enough individuals from each group, STRUCTURE is able to better estimate the allele frequencies corresponding to the three main self-reported ethnicities (i.e. Europeans, South Asians and Arabs) and consequently, assigns more than 90% of the individuals in the population corresponding to their self-reported ethnicity with a coefficient of ancestry larger than 0.85 (see <a href="#pone-0001382-g002">Figure 2</a> and Supplemental <a href="#pone.0001382.s002">Figure S2</a>). The remaining individuals could represent random fluctuations due to our limited power (only 127 SNPs were used in this analysis) or differences between genetically-inferred and self-reported ethnicity). We did not detect any clear correlation between the assignment coefficients estimated by STRUCTURE and, either the geographic origin of the samples or their case/control status. However, we observed that a large proportion of the individuals recruited in Iranian centers that self-described their ethnicity as ‘Other-Asian’ (but were gathered in the Arab population sample in this study) were assigned among “Europeans”.</p>
<div class="figure" id="pone-0001382-g002"><div class="img"><a name="pone-0001382-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001382.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001382.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Genetic clustering of the INTERHEART individuals inferred by STRUCTURE.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>“European” (blue dots), “Arabs” (green dots) and “South Asian” (pink dots) individuals are displayed according to their coefficients of ancestry in three populations (K = 3) as estimated by STRUCTURE using 127 SNPs. The coefficients of ancestry display separately for each population samples were inferred from a single analysis (i.e. all individuals combined) and are represented using the same axes. See also Supplemental <a href="#pone.0001382.s002">Figure S2</a> for the distribution of the coefficients of ancestry.</p>
<span>doi:10.1371/journal.pone.0001382.g002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Based on the results of these analyses we generated second generation datasets after exclusion of problematic samples. We randomly excluded one individual from each pair of related individuals, all individuals that were clustered by STRUCTURE among sub-Saharan Africans or East Asians and all Nepalese and Iranian individuals. In addition, we excluded all individuals from two centers that showed a very high proportion of problematic samples (including more than 10% discrepancies between reported and genetically-inferred sex). This consequently reduced our sample sizes to 4,069 individuals in the European population sample (starting from 4,292), 2,450 in the South Asian sample (out of 2900, including 316 Nepalese) and 1,399 individuals in the Arab sample (out of 2559, including 460 Iranians).</p>


<h4>Correcting association analysis for possible residual stratification</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>We tested separately in each population sample the association between genotypes and Apolipoprotein B (ApoB) concentration (see <a href="#s2">Materials and Methods</a> for details). <a href="#pone-0001382-g003">Figure 3</a> shows the distribution of the p-value obtained for each SNP in the South-Asian dataset. The figure shows a global deviation (towards more significant associations) from the pattern expected by chance if there is no association between genotypes and ApoB concentration. This deviation is not limited to a few outliers but affects the entire distribution. This could be an indication that many SNPs (i.e. several hundred) in our panel are significantly associated with ApoB level or, alternatively, that a previously undetected stratification in the dataset affects the results. We first tried to correct this global deviation by using the coefficients of ancestry estimated by STRUCTURE for each individual as covariates in the ANOVA. This did not lead to any significant difference in the distribution of the p-values (see Supplemental <a href="#pone.0001382.s005">Figure S5</a>). We then tested whether the geographic origin of the individuals could influence the associations. After using the recruitment centers as covariates of the analyses, the distribution of the p-values for the South-Asian individuals fitted much better the distribution expected under no association, and only five SNPs (most notably rs429358 in APOE) showed significant deviation from the expectation and strong association with ApoB concentration (<a href="#pone-0001382-g003">Figure 3</a>). Similar patterns were observed in the Arab and, to a lesser extent, in the European datasets (data not shown). Correcting the association tests for the recruitment centers thus led to a dramatic change in the overall distribution of the associations with some of the SNPs showing up to two orders of magnitude decrease in statistical significance. It is important to note here that the stratification observed among centers is not due to a systematic difference in DNA preparation or storage between centers. The INTERHEART protocol requires that, for every case recruited, at least one control (same sex, same age) is recruited from the same center. Blood samples (or buffy coats) from cases and controls are then shipped to Canada and treated identically (after randomization). However, due to stochastic failures at different stages (e.g. DNA extractions, genotyping) some centers included more cases than controls (or inversely) at the end of the study which contributes to the observed stratification effect (in combination with allele frequency differences among centers).</p>
<div class="figure" id="pone-0001382-g003"><div class="img"><a name="pone-0001382-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001382.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001382.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Distribution of the p-values of the associations between genotypes at 1,453 SNPs and ApoB level in South-Asians.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>The plot shows the observed distribution of the p-values (y-axis) against the expectation under a model without any association (grey crosses and x-axis). The axes are in logarithmic scales. Red crosses correspond to the association between ApoB and the genotypes at one SNP without any correction. Blue crosses stand for the same tests using recruitment centers used as additional covariates.</p>
<span>doi:10.1371/journal.pone.0001382.g003</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In the European dataset (but not in the South-Asian and Arab datasets), the distribution of p-values shows a bump with a higher significance level for the SNPs with p&lt;0.05 (74 SNPs) than we would expect by chance (Supplemental <a href="#pone.0001382.s006">Figure S6</a>). Interestingly, most of the strongest associations come from SNPs located in a few genes. We thus tested whether this deviation was due to the carry-over of a limited number of signals to many SNPs in strong LD with each others. We reanalyzed the associations between the genotypes and ApoB level conditional on the genotypes of SNPs with the strongest associations (see <a href="#s2">Materials and Methods</a>). After correcting for the signal of the five strongest associations, the entire distribution becomes indistinguishable from the expected distribution (Supplemental <a href="#pone.0001382.s006">Figure S6</a>).</p>


<h4>Effect of cleaning the dataset</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>To estimate the influence of stratification on the results obtained and the loss of power resulting from the reduction in sample size, we contrasted the results of the associations with ApoB concentration prior to and after “cleaning” in each dataset. In all population samples, we observe a reduction in the deviation of the p-value distribution from the expectation after removing outlier individuals and/or centers (i.e. in the second generation population samples). The changes are more dramatic in the Arab dataset than in the South-Asian and European datasets (Supplementary <a href="#pone.0001382.s007">Figure S7</a>). The effect of cleaning the datasets does not affect evenly all markers and some of the associations between genotypes and ApoB concentration changed more dramatically than others. Consequently, the markers most strongly associated with ApoB concentration differ (<a href="#pone-0001382-t002">Table 2</a>). Interestingly, we note that after cleaning, the SNP most strongly associated with ApoB level in all three population samples (rs429358 in APOE) replicates well supported associations <a href="#pone.0001382-Austin1">[18]</a>, <a href="#pone.0001382-Mahley1">[19]</a>. In addition, the p-values for the most significant associations are only moderately changed (decreased or actually improved) despite the loss of 3.6 to 40% of the samples (<a href="#pone-0001382-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001382-t002"><div class="img"><a name="pone-0001382-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001382" data-uri="info:doi/10.1371/journal.pone.0001382.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001382.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001382.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001382.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001382.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001382.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Five strongest associations with ApoB level in each population sample using raw data and second generation datasets.</span></strong></p><span>doi:10.1371/journal.pone.0001382.t002</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3>
<h4>Detecting and correction for population substructure</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>One of the main drawbacks of population-based association studies (in comparison to family-based association studies) are their susceptibility to population stratification <a href="#pone.0001382-Devlin2">[20]</a>–<a href="#pone.0001382-Marchini1">[23]</a>. The presence of differing levels of relatedness among the samples or the existence of unnoticed sub-populations can induce both a loss of power in detecting true associations and generate spurious associations <a href="#pone.0001382-Devlin1">[14]</a>, <a href="#pone.0001382-Voight1">[15]</a>, <a href="#pone.0001382-Freedman1">[22]</a>, <a href="#pone.0001382-Marchini1">[23]</a>. These issues are likely to become even more crucial in the future since the effect of stratification increases with the sample size and since recruitment criteria are widening to obtain larger and multi-purpose cohorts. For example, UK Biobank, one of the largest on-going prospective studies, only excludes first-degree relatives and aims to obtain a global representation of the UK population including its ethnic minorities. We describe in our study a handful of simple methods that can be applied to any large scale genotyping projects (i.e. more than 1,000 SNPs) to identify and address possible stratification problems in the sample.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The INTERHEART study was originally designed as a “matched” case-control study but was unmatched in the analysis of nine modifiable risk factors <a href="#pone.0001382-Yusuf1">[5]</a> to minimize the loss of cases and controls for whom matching was not possible, given that there was general agreement for key results among the matched and unmatched data analyses. In this genetic analysis matching was not used since we often lacked genotypes of one of the two members of the matched pair (due to a failure in blood collection, DNA extraction or genotyping). As a consequence, the INTERHEART protocol, while specifically excluding the recruitment of related individuals as cases, authorized using a relative of one case as a control for another case. Numerous methods have been developed in the past to assess the degree of relationship among individuals (see <a href="#pone.0001382-Weir2">[24]</a>, <a href="#pone.0001382-Blouin1">[25]</a> for reviews). We showed here that, when enough markers are genotyped, a simple QQ-plot of allele sharing, as the one in <a href="#pone-0001382-g001">Figure 1</a>, allowed us to identify related individuals who almost always consisted of a case and one or more control individual(s). A more problematic issue arises if the cases are on average more related to each other than are the controls (or inversely): the global difference in the depth of the genealogies of each group can lead to differences in alleles frequencies and thus generate spurious associations. Since many of the individuals analyzed here were recruited from non-cosmopolitan areas, we were concerned that this could be an issue in the INTERHEART study. By comparing the mean pair-wise allele sharing observed among cases to that observed among controls, we showed that none of the differences observed was significantly larger than the difference observed by randomly assigning individuals into two groups. By contrast, the same analysis performed on unrelated individuals from a founder population of the Saguenay-Lac St-Jean from Quebec, Canada <a href="#pone.0001382-Heyer1">[26]</a> revealed an overall shorter genealogy than the INTERHEART Europeans consistent with their demographic history. This result clearly indicates that our analysis has sufficient power to identify slight differences in relatedness and excludes differential average relatedness between cases and controls as a major issue in our population samples.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>One of the most common arguments advanced to explain the lack of reproducibility in population-based association studies is the presence of undetected subpopulations in the sample, leading to spurious results (e.g. <a href="#pone.0001382-Cardon1">[27]</a>). We expected this issue to be especially problematic in this study since the individuals (both cases and controls) were recruited in more than a hundred centers across the world and later grouped together based on their self-reported ethnicity. Several methods have been developed to address population stratification based either on correcting the test statistic to account for genetic heterogeneity in the sample, or on performing structured associations after the identification of subpopulations <a href="#pone.0001382-Devlin2">[20]</a>, <a href="#pone.0001382-Pritchard2">[28]</a>. Devlin and Roeder <a href="#pone.0001382-Devlin1">[14]</a> proposed to use random SNPs as “genomic controls” to estimate the average effect of population substructure in the sample and then correct the test statistics accordingly. One limitation of this approach is that it assumes a constant effect of stratification or admixture over all loci and thus does not correct appropriately for markers located in regions of adaptive selection (i.e. loci where natural selection acted or is acting differently on different populations). This is a major drawback for whole genome scans: they include many SNPs in such regions that will not be sufficiently corrected (see <a href="#pone.0001382-Sladek1">[29]</a> as an example). Even for candidate gene studies, this effect can critically hamper the association analyses since natural selection can greatly affect genetic diversity at disease genes <a href="#pone.0001382-DiRienzo1">[30]</a>. In the INTERHEART study, several genes under investigation have been shown to differ drastically among populations due to the effect of natural selection <a href="#pone.0001382-Young1">[31]</a>–<a href="#pone.0001382-Nakajima1">[33]</a>. We thus discarded using genomic controls (GC) to correct for stratification since GC selected randomly would not correct sufficiently for stratification in genes under selection, while selecting GC in genes under selection would inflate the correction coefficient and over-correct all other loci, resulting in a large loss of power. Instead, we opted to use the program STRUCTURE <a href="#pone.0001382-Pritchard1">[11]</a> that uses genotypes to group individuals according to their genetic ancestry. An alternative program, EIGENSTRAT <a href="#pone.0001382-Patterson1">[34]</a>, performs similar analyses but does not incorporate any defined genetic model and thus is not as efficient as Structure with only a few hundreds of independent markers (it is, on the other hand, computationally more interesting for genome-wide or other large dataset). We spiked our dataset with individuals of known ancestry genotyped at the same loci to better identify possible individuals with ancestry from Africa and South East Asia present in the INTERHEART dataset. This procedure, coupled with the use of highly differentiated SNPs (i.e. SNPs with high Fst) yielded better clustering and thus, a more powerful identification of outliers (see also <a href="#pone.0001382-Sladek1">[29]</a>). Overall, the great majority of the individuals were gathered into the population corresponding to their self-reported ethnicity, consistent with previous reports showing high correspondence between self-reported ethnicity and genetic estimates of ancestry <a href="#pone.0001382-Tang1">[35]</a>. In addition, many individuals from Nepal, who all reported an “Other Asian” ethnicity but were grouped with “South Asian” individuals in previous analyses of the INTERHEART study, display high coefficients of ancestry from South East Asian populations as well as high heterogeneity in their assignments. This observation is consistent with previous reports of genetic heterogeneity in Tibeto-Burman populations <a href="#pone.0001382-Wen1">[36]</a>, <a href="#pone.0001382-Chakraborty1">[37]</a> and shows that the self-reported ethnicity correctly captured the genetic information but the later grouping of these individuals with South-Asian individuals lead to genetic heterogeneity. In contrast, we identified several clear outliers in each dataset, with in some cases the genetically inferred ancestry corresponding better to the geographic location than their self-reported ethnicities. These discrepancies could be due to clerical errors or sample mislabeling, or alternatively, represent true differences between self-reported ethnicity and genetic ancestry. In agreement with previous studies <a href="#pone.0001382-Parra1">[38]</a>, we identified several Brazilian individuals self-described as “Europeans” that show high level of African ancestry which illustrates some of the limitations of using self-reported ethnicity. Unfortunately, we were underpowered to identify (or rule out) with STRUCTURE and the reduced number of markers available (~130 selected SNPs) more subtle stratification levels due to intra-continental differences (e.g. <a href="#pone.0001382-Campbell1">[39]</a>). The deviations in the distribution of the p-values observed in at least two datasets (i.e. South Asian and Arab) clearly indicate that the exclusion of the outliers identified by STRUCTURE was not sufficient to remove all stratification and illustrates that self-reported ethnicity on its own is not sufficient to protect against population stratification. However, we successfully corrected this overall inflation in the significance of the associations using the recruitment centers as covariates. This approach can be easily applied to other scenarios when, for example, the controls are recruited in different centers. In such cases, a simple test such as a QQ plot of the p-value distribution will indicate if the use of additional covariates is useful and if using the dataset is appropriate for drawing biological/medical conclusions.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>Several studies have looked at the effect of stratification from a theoretical perspective and sometimes reached contradicting conclusions <a href="#pone.0001382-Freedman1">[22]</a>, <a href="#pone.0001382-Marchini1">[23]</a>, <a href="#pone.0001382-Wacholder1">[40]</a>, <a href="#pone.0001382-Wacholder2">[41]</a> but few concrete examples have shown its influence on the results of a real association study <a href="#pone.0001382-Campbell1">[39]</a>, <a href="#pone.0001382-Knowler1">[42]</a>, <a href="#pone.0001382-Choudhry1">[43]</a>. Here we empirically show that cleaning-up the datasets to remove as much stratification as possible does influence the overall distribution of the association p-values. In particular, we demonstrate that even the strongest associations (i.e. the SNPs that are most likely to be reported as “significantly associated”) can differ according to the “state” of the dataset: while we observe significant differences among the results of each dataset prior to cleaning, the strongest association in all three cleaned datasets is due to one SNP in the ApoE gene (rs429358) known to be strongly involved in Apolipoprotein B concentration <a href="#pone.0001382-Mahley1">[19]</a>. Interestingly, the loss of power resulting from a reduction in sample size (up to 40% in the Arab dataset) is almost completely compensated by the cleaner signal obtained: the strength of the confirmed associations is very similar or even improved in the cleaned datasets relative to the analysis performed with the raw data. This shows that cleaning up the datasets to obtain un-stratified samples, even at the cost of reduced sample size, is crucial to obtain reliable results. In our study, the genetic risk factors associated with the phenotype investigated seem to be similar in the different populations (e.g. APOE shows strong association with ApoB in all three population samples). In addition, the strongest signal comes from the presumably functional allele that has been directly genotyped. This represents the best case scenario to identify true associations in a stratified sample (even if the stratification will still generate spurious associations). If on the contrary, the risk factors associated with a particular trait differ among populations (e.g. if one would look at lactose tolerance, <a href="#pone.0001382-Tishkoff1">[44]</a>, or if the causative polymorphism is not directly genotyped and the LD patterns differ among populations, the power to detect true associations in a stratified population sample will be greatly decreased, resulting in both spurious associations and false negatives. This also illustrates a potential drawback of combining cohorts for different ethnicities in a single analysis: if the LD patterns surrounding the causative polymorphism(s) are different among populations or if the genetic risk factors are not shared across ethnicities, pooling individuals from diverse origin could lead to a loss of power (by diluting the effect observed at a given marker) instead of an increase due to the larger sample size.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001382"></div><a name="pone.0001382.s001" id="pone.0001382.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Map showing the geographic origin of each INTERHEART individual analyzed in this study. Each pie graph shows if at least one individual with self-reported ethnicity defined as “European” (blue section), “South-Asian” (pink section) or “Arabs” (green section) has been recruited in the country (regardless of the number of individuals recruited, see Supplemental <a href="#pone.0001382.s008">Table S1</a> for details). All individuals from Nepal and Iran reported their ethnicity as “Other Asian” and are displayed by a yellow section.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.50 MB TIF)</p>
<a name="pone.0001382.s002" id="pone.0001382.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">The graphs show the distribution of individuals according to their coefficients of ancestry from each population (K = 3). The left panel correspond to the assignments using 127 SNPs highly differentiated across population, the right panel to the assignments using 133 SNPs randomly selected.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.20 MB TIF)</p>
<a name="pone.0001382.s003" id="pone.0001382.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">QQ plot of the distribution of pair-wise allele sharing among the South Asian (left panel) and Arab (right panel) individuals against a normal distribution.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.06 MB TIF)</p>
<a name="pone.0001382.s004" id="pone.0001382.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Estimation of cryptic relatedness in Europeans. The graph displays the distribution of the t-statistic obtained in 300 tests of the difference in means between the distributions of allele sharing within two groups of randomly assigned individuals (Welch Two Sample t-test). The red arrow shows the t-statistic obtained by testing the INTERHEART Europeans cases vs. controls. The green arrow corresponds to the comparison of the distribution of pair-wise allele sharing among the Saguenay Lac St-Jean (SLSJ) individuals vs. the allele sharing observed in Europeans from the INTERHEART study. The pink arrow shows the t-statistic obtained in the comparison of inter-sample allele sharing (i.e., one SLSJ individual compared to one European individual from INTERHEART) vs. the distribution of allele sharing in Europeans.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.07 MB TIF)</p>
<a name="pone.0001382.s005" id="pone.0001382.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Effect of STRUCTURE on the distribution of the p-values for the associations between the genotypes and ApoB level in South-Asians. The plot shows the observed distribution of the p-values against the expectation under a model without any association (axes in logarithmic scales). Red crosses correspond to the association between ApoB and the genotypes at one SNP without any correction. Light blue crosses stand for the same tests using the coefficients of ancestry from STRUCTURE used as additional covariates</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.08 MB TIF)</p>
<a name="pone.0001382.s006" id="pone.0001382.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Distribution of the p-values for the associations between the genotypes and ApoB level in Europeans. Red crosses correspond to the non-corrected association between ApoB and the genotypes at one SNP. Blue crosses stands for the same tests after correcting for the signal of the five strongest associations (i.e. by conditioning the analyses on the genotypes at the five strongest associations).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB TIF)</p>
<a name="pone.0001382.s007" id="pone.0001382.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Distribution of the p-values for the associations between the genotypes and ApoB level in raw and cleaned datasets. Crosses correspond to the association between ApoB and the genotypes at one SNP using the raw (x-axis) and the cleaned datasets (y-axis). Green, Pink and Blue crosses stand for respectively the tests in the Arab, South-Asian and European datasets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.10 MB TIF)</p>
<a name="pone.0001382.s008" id="pone.0001382.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s008">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Description of the SNPs included in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0001382.s009" id="pone.0001382.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s009">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Excluded samples</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0001382.s010" id="pone.0001382.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s010">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Outliers identified by STRUCTURE with substantial ancestry from South-East Asia or Sub-Saharan Africa</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0001382.s011" id="pone.0001382.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001382.s011">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Tagging Efficiency</p>
<p class="siDoi">doi:10.1371/journal.pone.0001382.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.42 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Matthew McQueen and Michael Phillips for technical assistance and sample management, Fanny Chagnon for organizational guidance, Sumathy Rangarajan for helping with sample and questionnaire issues and the INTERHEART coordinators for sample collection.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DS SY SA TH JE. Performed the experiments: DS. Analyzed the data: DS AM TH GP. Contributed reagents/materials/analysis tools: DS SY BK SA AM TH GP JE. Wrote the paper: DS SA TH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001382-The1" id="pone.0001382-The1"></a>The International HapMap Consortium (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="1036622" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001382-Nelson1" id="pone.0001382-Nelson1"></a>Nelson MR, Klotsman M, McNeill AM, Maruyama Y, Bowman CE, et al.   Development of a densely genotyped population reference sample: a resource for population, disease, and pharmacological genetics research; 2006; Brisbane, Australia.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001382-Manolio1" id="pone.0001382-Manolio1"></a>Manolio TA, Bailey-Wilson JE, Collins FS (2006) Genes, environment and the value of prospective cohort studies. Nat Rev Genet  7: 812–820.  <ul class="find" data-citedArticleID="1036596" data-doi="10.1038/nrg1919"><li><a href="http://dx.doi.org/10.1038/nrg1919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes%2C+environment+and+the+value+of+prospective+cohort+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes%2C+environment+and+the+value+of+prospective+cohort+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001382-DaveySmith1" id="pone.0001382-DaveySmith1"></a>Davey Smith G, Ebrahim S, Lewis S, Hansell AL, Palmer LJ, et al.  (2005) Genetic epidemiology and public health: hope, hype, and future prospects. Lancet  366: 1484–1498.  <ul class="find" data-citedArticleID="1036574" data-doi="10.1016/s0140-6736(05)67601-5"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)67601-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+epidemiology+and+public+health%3A+hope%2C+hype%2C+and+future+prospects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+epidemiology+and+public+health%3A+hope%2C+hype%2C+and+future+prospects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001382-Yusuf1" id="pone.0001382-Yusuf1"></a>Yusuf S, Hawken S, Ounpuu S, Dans T, Avezum A, et al.  (2004) Effect of potentially modifiable risk factors associated with myocardial infarction in 52 countries (the INTERHEART study): case-control study. Lancet  364: 937–952.  <ul class="find" data-citedArticleID="1036642" data-doi="10.1016/s0140-6736(04)17018-9"><li><a href="http://dx.doi.org/10.1016/s0140-6736(04)17018-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+potentially+modifiable+risk+factors+associated+with+myocardial+infarction+in+52+countries+%28the+INTERHEART+study%29%3A+case-control+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+potentially+modifiable+risk+factors+associated+with+myocardial+infarction+in+52+countries+%28the+INTERHEART+study%29%3A+case-control+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001382-Cann1" id="pone.0001382-Cann1"></a>Cann HM, de Toma C, Cazes L, Legrand MF, Morel V, et al.  (2002) A human genome diversity cell line panel. Science  296: 261–262.  <ul class="find" data-citedArticleID="1036564" data-doi="10.1126/science.296.5566.261b"><li><a href="http://dx.doi.org/10.1126/science.296.5566.261b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+human+genome+diversity+cell+line+panel." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+human+genome+diversity+cell+line+panel.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001382-Pare1" id="pone.0001382-Pare1"></a>Pare G, Serre D, Brisson D, Anand SS, Montpetit A, et al.  (2007) Genetic Analysis of 103 Candidate Genes for Coronary Artery Disease and Associated Phenotypes in a Founder Population Reveals a New Association between Endothelin-1 and High-Density Lipoprotein Cholesterol. Am J Hum Genet  80: 673–682.  <ul class="find" data-citedArticleID="1036604" data-doi="10.1086/513286"><li><a href="http://dx.doi.org/10.1086/513286" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+Analysis+of+103+Candidate+Genes+for+Coronary+Artery+Disease+and+Associated+Phenotypes+in+a+Founder+Population+Reveals+a+New+Association+between+Endothelin-1+and+High-Density+Lipoprotein+Cholesterol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+Analysis+of+103+Candidate+Genes+for+Coronary+Artery+Disease+and+Associated+Phenotypes+in+a+Founder+Population+Reveals+a+New+Association+between+Endothelin-1+and+High-Density+Lipoprotein+Cholesterol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001382-Carlson1" id="pone.0001382-Carlson1"></a>Carlson CS, Eberle MA, Rieder MJ, Yi Q, Kruglyak L, et al.  (2004) Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. Am J Hum Genet  74: 106–120.  <ul class="find" data-citedArticleID="1036568" data-doi="10.1086/381000"><li><a href="http://dx.doi.org/10.1086/381000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selecting+a+maximally+informative+set+of+single-nucleotide+polymorphisms+for+association+analyses+using+linkage+disequilibrium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selecting+a+maximally+informative+set+of+single-nucleotide+polymorphisms+for+association+analyses+using+linkage+disequilibrium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001382-Fan1" id="pone.0001382-Fan1"></a>Fan JB, Oliphant A, Shen R, Kermani BG, Garcia CA, et al.  (2003) Highly parallel SNP genotyping. Cold Spring Harbor Symposia on Quantitative Biology  68: 69–78.  <ul class="find" data-citedArticleID="1036582" data-doi="10.1101/sqb.2003.68.69"><li><a href="http://dx.doi.org/10.1101/sqb.2003.68.69" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+parallel+SNP+genotyping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+parallel+SNP+genotyping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001382-Kalinowski1" id="pone.0001382-Kalinowski1"></a>Kalinowski ST, Wagner AP, Taper LM (2006) ML-Relate: a computer program for maximum likelihood estimation of relatedness and relationship. Molecular Ecology Notes  6: 576–579.  <ul class="find" data-citedArticleID="1036588" data-doi="10.1111/j.1471-8286.2006.01256.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2006.01256.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ML-Relate%3A+a+computer+program+for+maximum+likelihood+estimation+of+relatedness+and+relationship." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ML-Relate%3A+a+computer+program+for+maximum+likelihood+estimation+of+relatedness+and+relationship.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001382-Pritchard1" id="pone.0001382-Pritchard1"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics  155: 945–959.  <ul class="find" data-citedArticleID="1036610"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data.&amp;auth=&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+population+structure+using+multilocus+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001382-Serre1" id="pone.0001382-Serre1"></a>Serre D, Paabo S (2004) Evidence for gradients of human genetic diversity within and among continents. Genome Res  14: 1679–1685.  <ul class="find" data-citedArticleID="1036616" data-doi="10.1101/gr.2529604"><li><a href="http://dx.doi.org/10.1101/gr.2529604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+gradients+of+human+genetic+diversity+within+and+among+continents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+gradients+of+human+genetic+diversity+within+and+among+continents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001382-Weir1" id="pone.0001382-Weir1"></a>Weir BS (1996) Genetic data analysis. Sunderland, MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001382-Devlin1" id="pone.0001382-Devlin1"></a>Devlin B, Roeder K (1999) Genomic control for association studies. Biometrics  55: 997–1004.  <ul class="find" data-citedArticleID="1036576" data-doi="10.1111/j.0006-341x.1999.00997.x"><li><a href="http://dx.doi.org/10.1111/j.0006-341x.1999.00997.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+control+for+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+control+for+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001382-Voight1" id="pone.0001382-Voight1"></a>Voight BF, Pritchard JK (2005) Confounding from Cryptic Relatedness in Case-Control Association Studies. PLoS Genet  1: e32.  <ul class="find" data-citedArticleID="1036628" data-doi="10.1371/journal.pgen.0010032.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010032.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Confounding+from+Cryptic+Relatedness+in+Case-Control+Association+Studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Confounding+from+Cryptic+Relatedness+in+Case-Control+Association+Studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001382-Yusuf2" id="pone.0001382-Yusuf2"></a>Yusuf S, Hawken S, Ounpuu S, Bautista L, Franzosi MG, et al.  (2005) Obesity and the risk of myocardial infarction in 27,000 participants from 52 countries: a case-control study. Lancet  366: 1640–1649.  <ul class="find" data-citedArticleID="1036644" data-doi="10.1016/s0140-6736(05)67663-5"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)67663-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Obesity+and+the+risk+of+myocardial+infarction+in+27%2C000+participants+from+52+countries%3A+a+case-control+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Obesity+and+the+risk+of+myocardial+infarction+in+27%2C000+participants+from+52+countries%3A+a+case-control+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001382-Rosengren1" id="pone.0001382-Rosengren1"></a>Rosengren A, Hawken S, Ounpuu S, Sliwa K, Zubaid M, et al.  (2004) Association of psychosocial risk factors with risk of acute myocardial infarction in 11119 cases and 13648 controls from 52 countries (the INTERHEART study): case-control study. Lancet  364: 953–962.  <ul class="find" data-citedArticleID="1036614" data-doi="10.1016/s0140-6736(04)17019-0"><li><a href="http://dx.doi.org/10.1016/s0140-6736(04)17019-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+psychosocial+risk+factors+with+risk+of+acute+myocardial+infarction+in+11119+cases+and+13648+controls+from+52+countries+%28the+INTERHEART+study%29%3A+case-control+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+psychosocial+risk+factors+with+risk+of+acute+myocardial+infarction+in+11119+cases+and+13648+controls+from+52+countries+%28the+INTERHEART+study%29%3A+case-control+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001382-Austin1" id="pone.0001382-Austin1"></a>Austin MA, Hutter CM, Zimmern RL, Humphries SE (2004) Familial hypercholesterolemia and coronary heart disease: a HuGE association review. Am J Epidemiol  160: 421–429.  <ul class="find" data-citedArticleID="1036558" data-doi="10.1093/aje/kwh237"><li><a href="http://dx.doi.org/10.1093/aje/kwh237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Familial+hypercholesterolemia+and+coronary+heart+disease%3A+a+HuGE+association+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Familial+hypercholesterolemia+and+coronary+heart+disease%3A+a+HuGE+association+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001382-Mahley1" id="pone.0001382-Mahley1"></a>Mahley RW, Rall SC Jr (2000) Apolipoprotein E: far more than a lipid transport protein. Annu Rev Genomics Hum Genet  1: 507–537.  <ul class="find" data-citedArticleID="1036594" data-doi="10.1146/annurev.genom.1.1.507"><li><a href="http://dx.doi.org/10.1146/annurev.genom.1.1.507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Apolipoprotein+E%3A+far+more+than+a+lipid+transport+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Apolipoprotein+E%3A+far+more+than+a+lipid+transport+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001382-Devlin2" id="pone.0001382-Devlin2"></a>Devlin B, Roeder K, Bacanu SA (2001) Unbiased methods for population-based association studies. Genet Epidemiol  21: 273–284.  <ul class="find" data-citedArticleID="1036578" data-doi="10.1002/gepi.1034"><li><a href="http://dx.doi.org/10.1002/gepi.1034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unbiased+methods+for+population-based+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unbiased+methods+for+population-based+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001382-Lander1" id="pone.0001382-Lander1"></a>Lander ES, Schork NJ (1994) Genetic dissection of complex traits. Science  265: 2037–2048.  <ul class="find" data-citedArticleID="1036592" data-doi="10.1126/science.8091226"><li><a href="http://dx.doi.org/10.1126/science.8091226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+dissection+of+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+dissection+of+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001382-Freedman1" id="pone.0001382-Freedman1"></a>Freedman ML, Reich D, Penney KL, McDonald GJ, Mignault AA, et al.  (2004) Assessing the impact of population stratification on genetic association studies. Nat Genet  36: 388–393.  <ul class="find" data-citedArticleID="1036584" data-doi="10.1038/ng1333"><li><a href="http://dx.doi.org/10.1038/ng1333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+the+impact+of+population+stratification+on+genetic+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+the+impact+of+population+stratification+on+genetic+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001382-Marchini1" id="pone.0001382-Marchini1"></a>Marchini J, Cardon LR, Phillips MS, Donnelly P (2004) The effects of human population structure on large genetic association studies. Nat Genet  36: 512–517.  <ul class="find" data-citedArticleID="1036598" data-doi="10.1038/ng1337"><li><a href="http://dx.doi.org/10.1038/ng1337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+human+population+structure+on+large+genetic+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+human+population+structure+on+large+genetic+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001382-Weir2" id="pone.0001382-Weir2"></a>Weir BS, Anderson AD, Hepler AB (2006) Genetic relatedness analysis: modern data and new challenges. Nat Rev Genet  7: 771–780.  <ul class="find" data-citedArticleID="1036636" data-doi="10.1038/nrg1960"><li><a href="http://dx.doi.org/10.1038/nrg1960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+relatedness+analysis%3A+modern+data+and+new+challenges." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+relatedness+analysis%3A+modern+data+and+new+challenges.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001382-Blouin1" id="pone.0001382-Blouin1"></a>Blouin MS (2003) DNA-based methods for pedigree reconstruction and kinship analysis in natural population. Trends in Ecology and Evolution  18: 503–511.  <ul class="find" data-citedArticleID="1036560" data-doi="10.1016/s0169-5347(03)00225-8"><li><a href="http://dx.doi.org/10.1016/s0169-5347(03)00225-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA-based+methods+for+pedigree+reconstruction+and+kinship+analysis+in+natural+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA-based+methods+for+pedigree+reconstruction+and+kinship+analysis+in+natural+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001382-Heyer1" id="pone.0001382-Heyer1"></a>Heyer E, Tremblay M (1995) Variability of the genetic contribution of Quebec population founders associated to some deleterious genes. Am J Hum Genet  56: 970–978.  <ul class="find" data-citedArticleID="1036586"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Variability+of+the+genetic+contribution+of+Quebec+population+founders+associated+to+some+deleterious+genes.&amp;auth=&amp;atitle=Variability+of+the+genetic+contribution+of+Quebec+population+founders+associated+to+some+deleterious+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variability+of+the+genetic+contribution+of+Quebec+population+founders+associated+to+some+deleterious+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variability+of+the+genetic+contribution+of+Quebec+population+founders+associated+to+some+deleterious+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001382-Cardon1" id="pone.0001382-Cardon1"></a>Cardon LR, Palmer LJ (2003) Population stratification and spurious allelic association. Lancet  361: 598–604.  <ul class="find" data-citedArticleID="1036566" data-doi="10.1016/s0140-6736(03)12520-2"><li><a href="http://dx.doi.org/10.1016/s0140-6736(03)12520-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+stratification+and+spurious+allelic+association." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+stratification+and+spurious+allelic+association.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001382-Pritchard2" id="pone.0001382-Pritchard2"></a>Pritchard JK, Donnelly P (2001) Case-control studies of association in structured or admixed populations. Theor Popul Biol  60: 227–237.  <ul class="find" data-citedArticleID="1036612" data-doi="10.1006/tpbi.2001.1543"><li><a href="http://dx.doi.org/10.1006/tpbi.2001.1543" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Case-control+studies+of+association+in+structured+or+admixed+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Case-control+studies+of+association+in+structured+or+admixed+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001382-Sladek1" id="pone.0001382-Sladek1"></a>Sladek R, Rocheleau G, Rung J, Dina C, Shen L, et al.  (2007) A genome-wide association study identifies novel risk loci for type 2 diabetes. Nature  445: 881–885.  <ul class="find" data-citedArticleID="1036618" data-doi="10.1038/nature05616"><li><a href="http://dx.doi.org/10.1038/nature05616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001382-DiRienzo1" id="pone.0001382-DiRienzo1"></a>Di Rienzo A, Hudson RR (2005) An evolutionary framework for common diseases: the ancestral-susceptibility model. Trends Genet  21: 596–601.  <ul class="find" data-citedArticleID="1036580" data-doi="10.1016/j.tig.2005.08.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.08.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionary+framework+for+common+diseases%3A+the+ancestral-susceptibility+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionary+framework+for+common+diseases%3A+the+ancestral-susceptibility+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001382-Young1" id="pone.0001382-Young1"></a>Young JH, Chang YP, Kim JD, Chretien JP, Klag MJ, et al.  (2005) Differential susceptibility to hypertension is due to selection during the out-of-Africa expansion. PLoS Genet  1: e82.  <ul class="find" data-citedArticleID="1036640" data-doi="10.1371/journal.pgen.0010082.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010082.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+susceptibility+to+hypertension+is+due+to+selection+during+the+out-of-Africa+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+susceptibility+to+hypertension+is+due+to+selection+during+the+out-of-Africa+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001382-Thompson1" id="pone.0001382-Thompson1"></a>Thompson EE, Kuttab-Boulos H, Witonsky D, Yang L, Roe BA, et al.  (2004) CYP3A variation and the evolution of salt-sensitivity variants. Am J Hum Genet  75: 1059–1069.  <ul class="find" data-citedArticleID="1036624" data-doi="10.1086/426406"><li><a href="http://dx.doi.org/10.1086/426406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CYP3A+variation+and+the+evolution+of+salt-sensitivity+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CYP3A+variation+and+the+evolution+of+salt-sensitivity+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001382-Nakajima1" id="pone.0001382-Nakajima1"></a>Nakajima T, Wooding S, Sakagami T, Emi M, Tokunaga K, et al.  (2004) Natural selection and population history in the human angiotensinogen gene (AGT): 736 complete AGT sequences in chromosomes from around the world. Am J Hum Genet  74: 898–916.  <ul class="find" data-citedArticleID="1036600" data-doi="10.1086/420793"><li><a href="http://dx.doi.org/10.1086/420793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001382-Patterson1" id="pone.0001382-Patterson1"></a>Patterson N, Price AL, Reich D (2006) Population Structure and Eigenanalysis. PLoS Genet  2: e190.  <ul class="find" data-citedArticleID="1036608" data-doi="10.1371/journal.pgen.0020190"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020190" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+Structure+and+Eigenanalysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+Structure+and+Eigenanalysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001382-Tang1" id="pone.0001382-Tang1"></a>Tang H, Quertermous T, Rodriguez B, Kardia SL, Zhu X, et al.  (2005) Genetic structure, self-identified race/ethnicity, and confounding in case-control association studies. Am J Hum Genet  76: 268–275.  <ul class="find" data-citedArticleID="1036620" data-doi="10.1086/427888"><li><a href="http://dx.doi.org/10.1086/427888" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure%2C+self-identified+race%2Fethnicity%2C+and+confounding+in+case-control+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure%2C+self-identified+race%2Fethnicity%2C+and+confounding+in+case-control+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001382-Wen1" id="pone.0001382-Wen1"></a>Wen B, Xie X, Gao S, Li H, Shi H, et al.  (2004) Analyses of genetic structure of Tibeto-Burman populations reveals sex-biased admixture in southern Tibeto-Burmans. Am J Hum Genet  74: 856–865.  <ul class="find" data-citedArticleID="1036638" data-doi="10.1086/386292"><li><a href="http://dx.doi.org/10.1086/386292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analyses+of+genetic+structure+of+Tibeto-Burman+populations+reveals+sex-biased+admixture+in+southern+Tibeto-Burmans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analyses+of+genetic+structure+of+Tibeto-Burman+populations+reveals+sex-biased+admixture+in+southern+Tibeto-Burmans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001382-Chakraborty1" id="pone.0001382-Chakraborty1"></a>Chakraborty R (1990) Mitochondrial DNA polymorphism reveals hidden heterogeneity within some Asian populations. Am J Hum Genet  47: 87–94.  <ul class="find" data-citedArticleID="1036570"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mitochondrial+DNA+polymorphism+reveals+hidden+heterogeneity+within+some+Asian+populations.&amp;auth=&amp;atitle=Mitochondrial+DNA+polymorphism+reveals+hidden+heterogeneity+within+some+Asian+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+polymorphism+reveals+hidden+heterogeneity+within+some+Asian+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+polymorphism+reveals+hidden+heterogeneity+within+some+Asian+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001382-Parra1" id="pone.0001382-Parra1"></a>Parra FC, Amado RC, Lambertucci JR, Rocha J, Antunes CM, et al.  (2003) Color and genomic ancestry in Brazilians. Proc Natl Acad Sci U S A  100: 177–182.  <ul class="find" data-citedArticleID="1036606" data-doi="10.1073/pnas.0126614100"><li><a href="http://dx.doi.org/10.1073/pnas.0126614100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Color+and+genomic+ancestry+in+Brazilians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Color+and+genomic+ancestry+in+Brazilians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001382-Campbell1" id="pone.0001382-Campbell1"></a>Campbell CD, Ogburn EL, Lunetta KL, Lyon HN, Freedman ML, et al.  (2005) Demonstrating stratification in a European American population. Nat Genet  37: 868–872.  <ul class="find" data-citedArticleID="1036562" data-doi="10.1038/ng1607"><li><a href="http://dx.doi.org/10.1038/ng1607" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Demonstrating+stratification+in+a+European+American+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Demonstrating+stratification+in+a+European+American+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001382-Wacholder1" id="pone.0001382-Wacholder1"></a>Wacholder S, Rothman N, Caporaso N (2000) Population stratification in epidemiologic studies of common genetic variants and cancer: quantification of bias. J Natl Cancer Inst  92: 1151–1158.  <ul class="find" data-citedArticleID="1036630" data-doi="10.1093/jnci/92.14.1151"><li><a href="http://dx.doi.org/10.1093/jnci/92.14.1151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+stratification+in+epidemiologic+studies+of+common+genetic+variants+and+cancer%3A+quantification+of+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+stratification+in+epidemiologic+studies+of+common+genetic+variants+and+cancer%3A+quantification+of+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001382-Wacholder2" id="pone.0001382-Wacholder2"></a>Wacholder S, Rothman N, Caporaso N (2002) Counterpoint: bias from population stratification is not a major threat to the validity of conclusions from epidemiological studies of common polymorphisms and cancer. Cancer Epidemiol Biomarkers Prev  11: 513–520.  <ul class="find" data-citedArticleID="1036632"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Counterpoint%3A+bias+from+population+stratification+is+not+a+major+threat+to+the+validity+of+conclusions+from+epidemiological+studies+of+common+polymorphisms+and+cancer.&amp;auth=&amp;atitle=Counterpoint%3A+bias+from+population+stratification+is+not+a+major+threat+to+the+validity+of+conclusions+from+epidemiological+studies+of+common+polymorphisms+and+cancer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Counterpoint%3A+bias+from+population+stratification+is+not+a+major+threat+to+the+validity+of+conclusions+from+epidemiological+studies+of+common+polymorphisms+and+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Counterpoint%3A+bias+from+population+stratification+is+not+a+major+threat+to+the+validity+of+conclusions+from+epidemiological+studies+of+common+polymorphisms+and+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001382-Knowler1" id="pone.0001382-Knowler1"></a>Knowler WC, Pettitt DJ, Bennett PH, Williams RC (1983) Diabetes mellitus in the Pima Indians: genetic and evolutionary considerations. Am J Phys Anthropol  62: 107–114.  <ul class="find" data-citedArticleID="1036590" data-doi="10.1002/ajpa.1330620114"><li><a href="http://dx.doi.org/10.1002/ajpa.1330620114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diabetes+mellitus+in+the+Pima+Indians%3A+genetic+and+evolutionary+considerations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diabetes+mellitus+in+the+Pima+Indians%3A+genetic+and+evolutionary+considerations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001382-Choudhry1" id="pone.0001382-Choudhry1"></a>Choudhry S, Coyle NE, Tang H, Salari K, Lind D, et al.  (2006) Population stratification confounds genetic association studies among Latinos. Hum Genet  118: 652–664.  <ul class="find" data-citedArticleID="1036572" data-doi="10.1007/s00439-005-0071-3"><li><a href="http://dx.doi.org/10.1007/s00439-005-0071-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+stratification+confounds+genetic+association+studies+among+Latinos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+stratification+confounds+genetic+association+studies+among+Latinos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001382-Tishkoff1" id="pone.0001382-Tishkoff1"></a>Tishkoff SA, Reed FA, Ranciaro A, Voight BF, Babbitt CC, et al.  (2007) Convergent adaptation of human lactase persistence in Africa and Europe. Nat Genet  39: 31–40.  <ul class="find" data-citedArticleID="1036626" data-doi="10.1038/ng1946"><li><a href="http://dx.doi.org/10.1038/ng1946" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Convergent+adaptation+of+human+lactase+persistence+in+Africa+and+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Convergent+adaptation+of+human+lactase+persistence+in+Africa+and+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.XML" value="124948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.PDF" value="240404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t001.PNG_L" value="109497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t001.PNG_M" value="56732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t001.PNG_S" value="14502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t001.TIF" value="782442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t001.PNG_I" value="61497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g001.PNG_L" value="120151"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g001.PNG_M" value="48073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g001.PNG_S" value="9895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g001.TIF" value="317924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g001.PNG_I" value="16796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g002.PNG_L" value="633462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g002.PNG_M" value="79651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g002.PNG_S" value="8127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g002.TIF" value="1394932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g002.PNG_I" value="25492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g003.PNG_L" value="111514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g003.PNG_M" value="50267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g003.PNG_S" value="10144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g003.TIF" value="228876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.g003.PNG_I" value="17146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t002.PNG_L" value="291618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t002.PNG_M" value="121568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t002.PNG_S" value="13084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t002.TIF" value="1346756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.t002.PNG_I" value="36031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s001.TIF" value="439295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s002.TIF" value="252098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s003.TIF" value="73919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s004.TIF" value="80943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s005.TIF" value="93756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s006.TIF" value="68573"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s007.TIF" value="123794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s008.PDF" value="41498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s009.PDF" value="10125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s010.PDF" value="12408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001382.s011.DOC" value="423930"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001382&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001382" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001382&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001382&volume=&issue=&title=Correction of Population Stratification in Large Multi-Ethnic Association Studies&author_name=David%20Serre%2C%20Alexandre%20Montpetit%2C%20Guillaume%20Par%C3%A9%2C%20James%20C.%20Engert%2C%20Salim%20Yusuf%2C%20Bernard%20Keavney%2C%20Thomas%20J.%20Hudson%2C%20Sonia%20Anand&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001382" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001382" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001382" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001382&amp;t=Correction%20of%20Population%20Stratification%20in%20Large%20Multi-Ethnic%20Association%20Studies" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001382&title=Correction%20of%20Population%20Stratification%20in%20Large%20Multi-Ethnic%20Association%20Studies&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001382&amp;title=Correction%20of%20Population%20Stratification%20in%20Large%20Multi-Ethnic%20Association%20Studies" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001382" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001382&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Correction of Population Stratification in Large Multi-Ethnic Association Studies';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001382';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Correction%20of%20Population%20Stratification%20in%20Large%20Multi-Ethnic%20Association%20Studies http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001382" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001382" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabs%22" title="Search for articles in the subject area:'Arabs'"><div class="flagText">Arabs</div></a>
              <div data-categoryid="41103" data-articleid="25594"
                   data-categoryname="Arabs"
                   class="flagImage" title="Flag 'Arabs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ethnic+epidemiology%22" title="Search for articles in the subject area:'Ethnic epidemiology'"><div class="flagText">Ethnic epidemiology</div></a>
              <div data-categoryid="42349" data-articleid="25594"
                   data-categoryname="Ethnic epidemiology"
                   class="flagImage" title="Flag 'Ethnic epidemiology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ethnicities%22" title="Search for articles in the subject area:'Ethnicities'"><div class="flagText">Ethnicities</div></a>
              <div data-categoryid="40419" data-articleid="25594"
                   data-categoryname="Ethnicities"
                   class="flagImage" title="Flag 'Ethnicities' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Myocardial+infarction%22" title="Search for articles in the subject area:'Myocardial infarction'"><div class="flagText">Myocardial infarction</div></a>
              <div data-categoryid="24301" data-articleid="25594"
                   data-categoryname="Myocardial infarction"
                   class="flagImage" title="Flag 'Myocardial infarction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogeography%22" title="Search for articles in the subject area:'Phylogeography'"><div class="flagText">Phylogeography</div></a>
              <div data-categoryid="32107" data-articleid="25594"
                   data-categoryname="Phylogeography"
                   class="flagImage" title="Flag 'Phylogeography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="25594"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Stratigraphy%22" title="Search for articles in the subject area:'Stratigraphy'"><div class="flagText">Stratigraphy</div></a>
              <div data-categoryid="20919" data-articleid="25594"
                   data-categoryname="Stratigraphy"
                   class="flagImage" title="Flag 'Stratigraphy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="25594"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5358'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5292'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7849&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=16883">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
