

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000419"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000419" />

    <meta name="citation_title" content="Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population"/>
    <meta itemprop="name" content="Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population"/>

      <meta name="citation_author" content="Baochuan Lin"/>
            <meta name="citation_author_institution" content="Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America"/>
      <meta name="citation_author" content="Anthony P. Malanoski"/>
            <meta name="citation_author_institution" content="Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America"/>
      <meta name="citation_author" content="Zheng Wang"/>
            <meta name="citation_author_institution" content="NOVA Research Incorporated, Alexandria, Virginia, United States of America"/>
      <meta name="citation_author" content="Kate M. Blaney"/>
            <meta name="citation_author_institution" content="NOVA Research Incorporated, Alexandria, Virginia, United States of America"/>
      <meta name="citation_author" content="Adam G. Ligler"/>
            <meta name="citation_author_institution" content="NOVA Research Incorporated, Alexandria, Virginia, United States of America"/>
      <meta name="citation_author" content="Robb K. Rowley"/>
            <meta name="citation_author_institution" content="Headquarters United States Air Force/Surgeon General, Falls Church, Virginia, United States of America"/>
      <meta name="citation_author" content="Eric H. Hanson"/>
            <meta name="citation_author_institution" content="Headquarters United States Air Force/Surgeon General, Falls Church, Virginia, United States of America"/>
      <meta name="citation_author" content="Erik von Rosenvinge"/>
            <meta name="citation_author_institution" content="Malcolm Grow USAF Medical Center, Andrews Air Force Base, Maryland, United States of America"/>
      <meta name="citation_author" content="Frances S. Ligler"/>
            <meta name="citation_author_institution" content="Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America"/>
      <meta name="citation_author" content="Anne W. Kusterbeck"/>
            <meta name="citation_author_institution" content="Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America"/>
      <meta name="citation_author" content="David Metzgar"/>
            <meta name="citation_author_institution" content="Department of Defense Center for Deployment Health Research, Naval Health Research Center, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Christopher P. Barrozo"/>
            <meta name="citation_author_institution" content="Department of Defense Center for Deployment Health Research, Naval Health Research Center, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Kevin L. Russell"/>
            <meta name="citation_author_institution" content="Department of Defense Center for Deployment Health Research, Naval Health Research Center, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Clark Tibbetts"/>
            <meta name="citation_author_institution" content="Headquarters United States Air Force/Surgeon General, Falls Church, Virginia, United States of America"/>
      <meta name="citation_author" content="Joel M. Schnur"/>
            <meta name="citation_author_institution" content="Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America"/>
      <meta name="citation_author" content="David A. Stenger"/>
            <meta name="citation_author_institution" content="Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America"/>

    <meta name="citation_date" content="2007/5/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000419.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e419"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Using automated medical records for rapid identification of illness syndromes (syndromic surveillance): the example of lower respiratory infection.; citation_author=R Lazarus; citation_author=KP Kleinman; citation_author=I Dashevsky; citation_author=A DeMaria; citation_author=R Platt; citation_journal_title=BMC Public Health; citation_volume=1; citation_number=1; citation_pages=9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Diagnostic system for rapid and sensitive differential detection of pathogens.; citation_author=T Briese; citation_author=G Palacios; citation_author=M Kokoris; citation_author=O Jabado; citation_author=Z Liu; citation_journal_title=Emerg Infect Dis; citation_volume=11; citation_number=2; citation_pages=310-313; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rapid identification and strain-typing of respiratory pathogens for epidemic surveillance.; citation_author=DJ Ecker; citation_author=R Sampath; citation_author=LB Blyn; citation_author=MW Eshoo; citation_author=C Ivy; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=3; citation_pages=8012-8017; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Applications of Luminex xMAP technology for rapid, high-throughput multiplexed nucleic acid detection.; citation_author=SA Dunbar; citation_journal_title=Clin Chim Acta; citation_volume=363; citation_number=4; citation_pages=71-82; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Viral discovery and sequence recovery using DNA microarrays.; citation_author=D Wang; citation_author=A Urisman; citation_author=YT Liu; citation_author=M Springer; citation_author=TG Ksiazek; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=5; citation_pages=E2; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High-density microarray of small-subunit ribosomal DNA probes.; citation_author=KH Wilson; citation_author=WJ Wilson; citation_author=JL Radosevich; citation_author=TZ DeSantis; citation_author=VS Viswanathan; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=6; citation_pages=2535-2541; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sequence-specific identification of 18 pathogenic microorganisms using microarray technology.; citation_author=WJ Wilson; citation_author=CL Strout; citation_author=TZ DeSantis; citation_author=JL Stilwell; citation_author=AV Carrano; citation_journal_title=Mol Cell Probes; citation_volume=16; citation_number=7; citation_pages=119-127; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Use of oligonucleotide microarrays for rapid detection and serotyping of acute respiratory disease-associated adenoviruses.; citation_author=B Lin; citation_author=GJ Vora; citation_author=D Thach; citation_author=E Walter; citation_author=D Metzgar; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=8; citation_pages=3232-3239; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microarray-based detection of genetic heterogeneity, antimicrobial resistance, and the viable but nonculturable state in human pathogenic Vibrio spp.; citation_author=GJ Vora; citation_author=CE Meador; citation_author=MM Bird; citation_author=CA Bopp; citation_author=JD Andreadis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=9; citation_pages=19109-19114; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarray-based detection and genotyping of viral pathogens.; citation_author=D Wang; citation_author=L Coscoy; citation_author=M Zylberberg; citation_author=PC Avila; citation_author=HA Boushey; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=10; citation_pages=15687-15692; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Co-infections of adenovirus species in previously vaccinated patients.; citation_author=GJ Vora; citation_author=B Lin; citation_author=K Gratwick; citation_author=C Meador; citation_author=C Hansen; citation_journal_title=Emerg Infect Dis; citation_volume=12; citation_number=11; citation_pages=921-930; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Extensive polymorphisms observed in HIV-1 clade B protease gene using high-density oligonucleotide arrays.; citation_author=MJ Kozal; citation_author=N Shah; citation_author=N Shen; citation_author=R Yang; citation_author=R Fucini; citation_journal_title=Nat Med; citation_volume=2; citation_number=12; citation_pages=753-759; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Resequencing and mutational analysis using oligonucleotide microarrays.; citation_author=JG Hacia; citation_journal_title=Nat Genet; citation_volume=21; citation_number=13; citation_pages=42-47; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Use of resequencing oligonucleotide microarrays for identification of Streptococcus pyogenes and associated antibiotic resistance determinants.; citation_author=L Davignon; citation_author=EA Walter; citation_author=KM Mueller; citation_author=CP Barrozo; citation_author=DA Stenger; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=14; citation_pages=5690-5695; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Using a resequencing microarray as a multiple respiratory pathogen detection assay.; citation_author=B Lin; citation_author=KM Blaney; citation_author=AP Malanoski; citation_author=AG Ligler; citation_author=JM Schnur; citation_journal_title=J Clin Microbiol; citation_volume=45; citation_number=15; citation_pages=443-452; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Broad-spectrum respiratory tract pathogen identification using resequencing DNA microarrays.; citation_author=B Lin; citation_author=Z Wang; citation_author=GJ Vora; citation_author=JA Thornton; citation_author=JM Schnur; citation_journal_title=Genome Res; citation_volume=16; citation_number=16; citation_pages=527-535; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identifying influenza viruses with resequencing microarrays.; citation_author=Z Wang; citation_author=LT Daum; citation_author=GJ Vora; citation_author=D Metzgar; citation_author=EA Walter; citation_journal_title=Emerg Infect Dis; citation_volume=12; citation_number=17; citation_pages=638-646; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Disease outbreak detection system using syndromic data in the greater Washington DC area.; citation_author=MD Lewis; citation_author=JA Pavlin; citation_author=JL Mansfield; citation_author=S O'Brien; citation_author=LG Boomsma; citation_journal_title=Am J Prev Med; citation_volume=23; citation_number=18; citation_pages=180-186; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Automated identification of multiple micro-organisms from resequencing DNA microarrays.; citation_author=AP Malanoski; citation_author=B Lin; citation_author=Z Wang; citation_author=JM Schnur; citation_author=DA Stenger; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=19; citation_pages=5300-5311; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Using Resequencing Microarray as a Multiple Respiratory Pathogen Detection Assay.; citation_author=B Lin; citation_author=KM Blaney; citation_author=AP Malanoski; citation_author=AG Ligler; citation_author=JM Schnur; citation_journal_title=J Clin Microbio; citation_number=20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A role for Streptococcus pneumoniae in virus-associated pneumonia.; citation_author=SA Madhi; citation_author=KP Klugman; citation_journal_title=Nat Med; citation_volume=10; citation_number=21; citation_pages=811-813; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Respiratory viruses predisposing to bacterial infections: role of neuraminidase.; citation_author=VT Peltola; citation_author=JA McCullers; citation_journal_title=Pediatr Infect Dis J; citation_volume=23; citation_number=22; citation_pages=S87-97; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Development of three multiplex RT-PCR assays for the detection of 12 respiratory RNA viruses.; citation_author=S Bellau-Pujol; citation_author=A Vabret; citation_author=L Legrand; citation_author=J Dina; citation_author=S Gouarin; citation_journal_title=J Virol Methods; citation_volume=126; citation_number=23; citation_pages=53-63; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000419" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population"/>
    <meta name="twitter:description" content="A broad spectrum detection platform that provides sequence level resolution of target regions would have a significant impact in public health, case management, and means of expanding our understanding of the etiology of diseases. A previously developed respiratory pathogen microarray (RPM v.1) demonstrated the capability of this platform for this purpose. This newly developed RPM v.1 was used to analyze 424 well-characterized nasal wash specimens from patients presenting with febrile respiratory illness in the Washington, D. C. metropolitan region. For each specimen, the RPM v.1 results were compared against composite reference assay (viral and bacterial culture and, where appropriate, RT-PCR/PCR) results. Across this panel, the RPM assay showed &ge;98% overall agreement for all the organisms detected compared with reference methods. Additionally, the RPM v.1 results provide sequence information which allowed phylogenetic classification of circulating influenza A viruses in &sim;250 clinical specimens, and allowed monitoring the genetic variation as well as antigenic variability prediction. Multiple pathogens (2&ndash;4) were detected in 58 specimens (13.7%) with notably increased abundances of respiratory colonizers (esp. S. pneumoniae) during viral infection. This first-ever comparison of a broad-spectrum viral and bacterial identification technology of this type against a large battery of conventional &ldquo;gold standard&rdquo; assays confirms the utility of the approach for both medical surveillance and investigations of complex etiologies of illness caused by respiratory co-infections."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000419.g001"/>

  <meta property="og:title" content="Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000419" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6651'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9360'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2588&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000419">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000419" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000419&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Baochuan Lin
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:baochuan.lin@nrl.navy.mil">baochuan.lin@nrl.navy.mil</a></p>

                <p>Affiliation:
                  Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anthony P. Malanoski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zheng Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  NOVA Research Incorporated, Alexandria, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kate M. Blaney, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  NOVA Research Incorporated, Alexandria, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adam G. Ligler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  NOVA Research Incorporated, Alexandria, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robb K. Rowley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Headquarters United States Air Force/Surgeon General, Falls Church, Virginia, United States of America
                </p>

                <p>
                    Current address: Amigenics, Las Vegas, Nevada, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eric H. Hanson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Headquarters United States Air Force/Surgeon General, Falls Church, Virginia, United States of America
                </p>

                <p>
                    Current address: Amigenics, Las Vegas, Nevada, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Erik von Rosenvinge, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Malcolm Grow USAF Medical Center, Andrews Air Force Base, Maryland, United States of America
                </p>

                <p>
                    Current address: Division of Gastroenterology and Hepatology, Department of Medicine, University of Maryland School of Medicine, Baltimore, Maryland, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frances S. Ligler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anne W. Kusterbeck, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Metzgar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Defense Center for Deployment Health Research, Naval Health Research Center, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christopher P. Barrozo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Defense Center for Deployment Health Research, Naval Health Research Center, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kevin L. Russell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Defense Center for Deployment Health Research, Naval Health Research Center, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Clark Tibbetts, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Headquarters United States Air Force/Surgeon General, Falls Church, Virginia, United States of America
                </p>

                <p>
                    Current address: Tessarae Inc., McLean, Virginia, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joel M. Schnur, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David A. Stenger
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Bio/Molecular Science and Engineering, Code 6900, Naval Research Laboratory, Washington, D. C., United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 09, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000419</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000419" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000419" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000419" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000419" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000419-t001" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000419-t002" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000419-t003" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000419-g001" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000419-t004" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000419">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000419" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>A broad spectrum detection platform that provides sequence level resolution of target regions would have a significant impact in public health, case management, and means of expanding our understanding of the etiology of diseases. A previously developed respiratory pathogen microarray (RPM v.1) demonstrated the capability of this platform for this purpose. This newly developed RPM v.1 was used to analyze 424 well-characterized nasal wash specimens from patients presenting with febrile respiratory illness in the Washington, D. C. metropolitan region. For each specimen, the RPM v.1 results were compared against composite reference assay (viral and bacterial culture and, where appropriate, RT-PCR/PCR) results. Across this panel, the RPM assay showed ≥98% overall agreement for all the organisms detected compared with reference methods. Additionally, the RPM v.1 results provide sequence information which allowed phylogenetic classification of circulating influenza A viruses in ~250 clinical specimens, and allowed monitoring the genetic variation as well as antigenic variability prediction. Multiple pathogens (2–4) were detected in 58 specimens (13.7%) with notably increased abundances of respiratory colonizers (esp. <em>S. pneumoniae</em>) during viral infection. This first-ever comparison of a broad-spectrum viral and bacterial identification technology of this type against a large battery of conventional “gold standard” assays confirms the utility of the approach for both medical surveillance and investigations of complex etiologies of illness caused by respiratory co-infections.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lin B, Malanoski AP, Wang Z, Blaney KM, Ligler AG, et al.  (2007) Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population. PLoS ONE 2(5):
          e419.
            doi:10.1371/journal.pone.0000419</p><p><strong>Academic Editor: </strong>Neil Hall, University of Liverpool, United Kingdom</p><p><strong>Received:</strong> December 7, 2006; <strong>Accepted:</strong> April 12, 2007; <strong>Published:</strong> May 9, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>The funding for this research is provided by the NRL base program and the Office of Naval Research. This project would not be possible without the support from the Air Force Surgeon General Office, the Joint Program Executive Office and Defense Threat Reduction Agency during the Silent Guardian Demonstration Project.</p><p><strong>Competing interests:</strong> There are two pending patent applications, one for amplification method and the other is for anlaysis algorithm, that are related to this manusccipt.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Emerging infections with nonspecific symptoms have enormous impact on public health and global economics (e.g. severe acute respiratory syndrome outbreaks, and avian (H5N1) influenza). Such outbreaks emphasize the need for efficient simultaneous detection of multiple organisms that will facilitate timely and judicious implementation of countermeasures for outbreak containment, treatment interventions, and case management <a href="#pone.0000419-Lazarus1">[1]</a>. Ideally, a system capable of detecting all potentially relevant organisms is needed. There are many technologies being developed with a view toward addressing this issue, for example MasscodeTM multiplex RT-PCR system <a href="#pone.0000419-Briese1">[2]</a>, electrospray ionization mass spectrometry analysis of PCR amplicons <a href="#pone.0000419-Ecker1">[3]</a>, Luminex® xMAP™ <a href="#pone.0000419-Dunbar1">[4]</a>, and other microarray based approaches <a href="#pone.0000419-Wang1">[5]</a>, <a href="#pone.0000419-Wilson1">[6]</a>, <a href="#pone.0000419-Wilson2">[7]</a>, <a href="#pone.0000419-Lin1">[8]</a>, <a href="#pone.0000419-Vora1">[9]</a>, <a href="#pone.0000419-Wang2">[10]</a>, <a href="#pone.0000419-Vora2">[11]</a>. While promising, these technologies have only been tested for detection of a few pathogens or particular classes of pathogens simultaneously. In addition, these technologies provide a very limited range of genetic resolution and require additional testing for determination of detailed mutation or strain-variations in detected pathogens. High density resequencing arrays provide resolution of individual nucleotides in long target sequences (hundreds to thousands of nucleotides) <a href="#pone.0000419-Kozal1">[12]</a>, <a href="#pone.0000419-Hacia1">[13]</a>. We have demonstrated discrimination of mutations in targeted pathogens as well as sensitivities and specificities that are similar to or improved over those of other technologies based on a subset of the organisms the microarray is designed to detect. The respiratory pathogen microarray version 1 (RPM v.1), while not providing comprehensive coverage of all potential causes of respiratory illness, targets a much broader range of organisms (including bacteria and DNA and RNA viruses) in a single test than other potential methods <a href="#pone.0000419-Davignon1">[14]</a>, <a href="#pone.0000419-Lin2">[15]</a>, <a href="#pone.0000419-Lin3">[16]</a>, <a href="#pone.0000419-Wang3">[17]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Our previous work on the RPM v.1 demonstrated detection of pathogens individually at 10<sup>1</sup>–10<sup>3</sup> genome copies using spiked samples as well as select organisms in clinical samples. We have also demonstrated the use of culture isolates of clinical samples in which a more detailed analysis of strain distribution in influenza viruses was possible <a href="#pone.0000419-Wang3">[17]</a>. Unfortunately, when directly using clinical samples, only the successful attribution of flu samples to the correct flu season was accomplished <a href="#pone.0000419-Lin2">[15]</a>. The accuracy of the base calls from the array has been compared with only <em>de novo</em> sequencing using culture isolates of clinical samples. Our studies demonstrate that one of the challenges in developing multiplex detection systems for a broad range of very different organisms is the validation of the test results. Few samples are available that have been subjected to testing for a large number of diverse pathogens. Similarly, the archived clinical specimens used in our previous studies were originally only tested for a few select targets. When multiple organisms were detected by the RPM v.1, it was not always possible to perform confirmatory testing due to a lack of available materials. This is a limitation of using archived samples for validating newly developed multiplex assays as it is very time-consuming and costly thus making multiple confirmatory assays in clinical specimens very difficult to achieve.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this study, we validate the potential of the respiratory pathogen microarray (RPM v.1) for providing broad-spectrum pathogen detection. A set of 424 clinical samples collected in the Washington, DC metropolitan region from December 2004 to February 2005 were subjected to a full panel of conventional microbial analysis (culture and/or RT-PCR/PCR) as well as analysis by the RPM v.1 microarray. This set of samples demonstrated good agreement across the panel of tests making authentication of multiple detection events by the RPM v.1 microarray possible. Many specimens from this set represented co-infections providing a range of multi-target samples for analysis. Here we use clinical samples rather than culture isolates and demonstrate detailed analysis of strain distributions of influenza viruses within a flu season as well as providing a phylogenetic measure of circulating influenza strains. The accuracy of the base calls was confirmed by comparison to <em>de novo</em> sequencing using representative clinical samples. This comparison of a broad-spectrum viral and bacterial identification technology with a large battery of conventional “gold standard” assays represents more rigorous testing of our approach for both medical surveillance and investigation of complex etiologies than has been carried out previously. This study provides a previously unavailable fundamental assessment of a multiple pathogen detection assay and paves the way to improvements for a microarray-based platform which would provide more comprehensive multiple pathogen detection in a single test. The implications and limitations of the assay are also discussed.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Specimen collection and processing</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Specimens were collected at six military treatment facilities in the Washington, DC metropolitan region between December 2004 and February 2005. The demographics of the Washington, DC metropolitan region have previously been shown to have a broad age, gender, and geographic distribution <a href="#pone.0000419-Lewis1">[18]</a>. Patients were recruited from both emergency department and clinic settings, after the nature and possible consequences of the study were explained and informed consents were obtained from participants. Patient samples were stored in CVM media (Hardy Diagnostics, Santa Maria, CA) for viral culture processing, or stored in tryptic soy broth with glycerol for bacterial analysis, and were shipped frozen to destination laboratories for analysis. These samples were collected, and this research has been conducted in compliance with all applicable federal and international regulations governing the protection of human subjects in research, under Naval Medical Center protocols #B05LHOOOOO-018.</p>


<h4>Microarray hybridization and analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The RPM v.1 design and specimen process protocols for microarray analysis were described in detail in previous publications <a href="#pone.0000419-Davignon1">[14]</a>, <a href="#pone.0000419-Lin2">[15]</a>, <a href="#pone.0000419-Lin3">[16]</a>, <a href="#pone.0000419-Wang3">[17]</a>. Microarray hybridization and processing were carried out according to the manufacturer's recommended protocol (Affymetrix Inc., Santa Clara, CA) with the following modifications: Purified PCR products were fragmented at 37°C for 5 minutes, and then labeled with Biotin-N6-ddATP at 37°C for 30 minutes. Hybridization was carried out in the hybridization oven at 45°C and 60 rev/min for 4 hours. The image scanning and processing to produce FASTA output file were performed as previously described <a href="#pone.0000419-Lin3">[16]</a>. Final pathogen identification for the RPM v.1 assay was performed using Computer-Implemented Biological Sequence Identifier (CIBSI) Version 2.0 software <a href="#pone.0000419-Malanoski1">[19]</a>, an automatic pathogen identification algorithm (based on nucleic acid sequence) that was developed and tested in detail in previous studies <a href="#pone.0000419-Lin3">[16]</a>, <a href="#pone.0000419-Lin4">[20]</a>.</p>


<h4>Reference assays</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Independent assays were performed using bacterial and viral culture except for difficult-to-culture pathogens, for which PCR methods were used. Additional assays, primarily conventional and/or real-time PCR and RT/PCR, were performed on culture negative samples to test for pathogens occurring at titers too low for culture and/or loss of viability from transportation. Overall, a specimen was defined as “positive” for a pathogen if the composite assay results (culture and/or duplicate PCR assays) were positive.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Viral analyses were performed by FOCUS Diagnostics, Inc., a CAP-certified reference laboratory (Cypress, CA) or by Air Force Institute for Operational Health (Brooks City Base, TX). Viral processing included: inoculation of sample, monitoring for cytopathic effect (7–15 days of monitoring), sub-passage as necessary and confirmatory testing as needed (i.e., direct monoclonal immunofluorescence assay, indirect immunofluorescence assay or PCR).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Bacterial analyses were performed at the Naval Health Research Center (NHRC, San Diego, CA) Respiratory Disease Laboratory. Samples were then analyzed for the presence of bacterial pathogens utilizing classic culture techniques with selective agar plates. For all culture methods, if growth resembled colonies appropriate to a targeted pathogen, confirmatory tests (e.g., susceptibility disks, biochemical testing) were conducted to verify the identity of the suspected colony.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Initial PCR analyses of difficult-to-culture pathogens (<em>Bordetella pertussis, Chlamydia pneumoniae/psittaci</em>, coronavirus OC43 and 229E, and <em>Mycoplasma pneumoniae)</em> were performed at the NHRC Respiratory Disease Laboratory or FOCUS Diagnostics, Inc. (Cypress, CA). Further species-specific RT/PCR amplification assays were performed on culture-negative but RPM v.1-positive samples with positive (culture-positive, RPM v.1-positive) and negative (culture-negative, RPM v.1-negative) controls. These assays were performed at NRL using previously published primers (<a href="#pone.0000419.s001">Table S1</a>). In general, 25 µl PCR reactions were run on MyiQ™ real-time PCR detection systems (Bio-Rad, Hercules, CA, USA). Conventional PCR assays were analyzed by electrophoresis thorugh1.5% TAE agarose gels. Sequencing reactions were performed by SequeTech (Mountain View, CA).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Sequence alignments for influenza-positive samples were performed on <em>HA3 hemagglutinin</em> sequences using MagAlign function of Lasergene version 6 (DNASTAR, Inc. Madison, WI). Rooted phylogenetic trees were generated by using neighbor-joining method in PAUP* 4.0 (Sinauer Associates, Inc., Publishers, Sunderland, MA) and rooted to A/Panama/2007/99. Reliability estimates were assessed using 1,000 bootstrap replicates.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>A total of 424 samples were tested using RPM v.1 assays and parallel reference assays (culture and/or PCR tests). The sensitivity and specificity of the RPM v.1 assay were determined using various prototype strains and clinical samples in a previous study <a href="#pone.0000419-Lin2">[15]</a>. <a href="#pone-0000419-t001">Table 1</a> shows the collective number of positive results obtained using culture assays only, all reference assays, and RPM v.1. Of the 424 samples, 269 (63.4%) were identified as positive for influenza A by RPM v.1, compared to 176 (41.5%) by culture (30 to 42-day). Similarly, 46 (10.8%) were positive for influenza B using RPM v.1, while culture identified 28 (6.6%) as influenza B-positive. RPM v.1 identified 24 (5.6%) as positive for coronavirus (229E or OC43), while PCR identified 28 (6.6%) samples as coronavirus (229E and OC43) positive. Additional pathogens were identified in low incidence–shown here as [RPM v.1 vs. culture], including adenovirus-[9 (2.1%) vs. 2 (0.5%)], <em>Streptococcus pneumoniae</em>-[38 (8.9%) vs. 8 (1.9%)], and <em>S. pyogenes</em>-[13 (3.1%) vs. 9 (2.1%)]. RPM v.1 also identified type I and III parainfluenza (PIV) virus, PIV1 (1, 0.2%), PIV3 (2, 0.5%), rhinovirus type 89 (2, 0.5%), <em>M. pneumoniae</em> (3, 0.7%), and <em>Neisseria meningitides</em> (14, 3.3%) which were not identified by culture. The RPM v.1 did not contain resequencing tiles for PIV2 or <em>Haemophilus influenzae,</em> both of which were detected by culture, PIV2 (2, 0.5%) and <em>H. influenzae</em> (7; 1.7%).</p>
<div class="figure" id="pone-0000419-t001"><div class="img"><a name="pone-0000419-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Pathogens identified for 424 matched specimens-overall microarray vs. reference methods.</span></strong></p><span>doi:10.1371/journal.pone.0000419.t001</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In this study, influenza virus was the most commonly identified respiratory pathogen by all methods. With respect to all reference assays, the RPM v.1 method showed a detection sensitivity of 99% and a specificity of 96%, and an overall agreement of 98% (<a href="#pone-0000419-t002">Table 2</a>) for influenza A virus. For influenza B, the RPM v.1 detection sensitivity and specificity were 98% with an overall agreement of 97% (<a href="#pone-0000419-t002">Table 2</a>). All but three influenza culture-positive specimens were also positive on microarrays. However, two specimens identified as influenza A-positive by culture were clearly detected as influenza B on microarrays. Real-time RT-PCR later concurred with the microarray results and not culture that these 2 specimens were indeed influenza B-positive. The RPM v.1 also demonstrated excellent detection sensitivity, specificity, and overall agreement with respect to the reference assay results for other pathogens detected (<a href="#pone.0000419.s002">Table S2</a>–<a href="#pone.0000419.s003">S3</a>).</p>
<div class="figure" id="pone-0000419-t002"><div class="img"><a name="pone-0000419-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Evaluation of the detection efficiency for influenza A and B viruses in clinical samples</span></strong></p><span>doi:10.1371/journal.pone.0000419.t002</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The capability of the RPM v.1 system for the identification of complex mixtures of pathogens <a href="#pone.0000419-Lin2">[15]</a>, <a href="#pone.0000419-Lin3">[16]</a>, <a href="#pone.0000419-Wang3">[17]</a> were further demonstrated through assessment of the incidence of co-infections in the 424 samples. Of these samples, 58 (13.7%) showed viral or viral/bacterial co-infections as determined by RPM and/or culture (<a href="#pone-0000419-t004">Table 4</a>). These co-infections were further verified using published type-specific PCR assays and in-house specific PCR primers (<a href="#pone.0000419.s001">Table S1</a>). It is well known that <em>S. pneumoniae</em> and <em>N. meningitidis</em> colonize the mouth and upper respiratory system, so it is not surprising that these were common co-infections found in clinical samples. However, consistent with our previous study <a href="#pone.0000419-Lin2">[15]</a>, quantitative real-time PCR data showed that most of the <em>S. pneumoniae</em> present in influenza-positive samples harbored a high titer (≥10<sup>4</sup> genome copies/µl) as compared to influenza-negative samples (data not shown). The high titer bacteria present in these clinical samples was possibly virally induced bacterial superinfection, as first suggested by the findings of Madhi et al. <a href="#pone.0000419-Madhi1">[21]</a> and Peltola and McCullers <a href="#pone.0000419-Peltola1">[22]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>A critical aspect for influenza epidemiology is to track genetic changes within influenza strains, since antigenic drift is the mechanism by which influenza viruses escape from immunological pressure induced by previous natural exposures and vaccinations. Analysis of the key amino acids (deduced from nucleotide sequence) in the HA3 sequences of all influenza A H3N2 positive isolates revealed two major circulating strains: A/New York/258/2005 (Group I) and A/Aichi/133/2005 (Group II) (<a href="#pone-0000419-t003">Table 3</a>). Group I belongs to the A/California/7/2004 lineage and carries signature amino acids substitutions in antigenic site D: valine to isoleucine at position 226 (V226I) and serine to proline at position 227 (S227P). Group II showed an A/Wellington/1/04 lineage signature amino acid substitution at position 227 (S227P), and serine to asparagine at position 216 (S216N), the key amino acid at antigenic site B. Surprisingly, the only outlier sample was identified as A/Wyoming/3/03 (with IS)-like isolate. Amino acids position at 216 correspond to 188, 226 corresponds to 198, and 227 corresponds to 199 on the tiled prototype sequence (<a href="#pone-0000419-t003">Table 3</a>).</p>
<div class="figure" id="pone-0000419-t003"><div class="img"><a name="pone-0000419-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Nucleotides difference in <em>hemagglutinin</em> (HA3) genes identified by RPM v.1 from 250 influenza A/H3N2 isolates.</span></strong></p><span>doi:10.1371/journal.pone.0000419.t003</span></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Using 15 representative samples, phylogenetic analysis comparison between the sequences generated from RPM v.1 (68 to 96% resolved bases) (<a href="#pone-0000419-g001">Fig. 1A</a>) and the sequences generated via conventional sequencing (100% resolved bases) (<a href="#pone-0000419-g001">Fig. 1B</a>), showed similar results, indicating that ambiguous base calls from the microarray did not affect phylogeny determination for influenza A viruses (<a href="#pone-0000419-t003">Table 3</a>, <a href="#pone-0000419-g001">Fig. 1</a>). The representative samples were the A/Wyoming/3/03 (with IS)-like isolate and 14 other samples randomly selected from the two groups. The resulting phylogenetic trees clearly confirmed that the influenza A/H3N2 positive samples consisted of two major groups: A/New York/258/2005 (A/California/7/2004 lineage) and A/Aichi/133/2005 (A/Wellington/1/04 lineage) with one A/Wyoming/3/03 (with IS)-like isolate as an outlier. Further confirmation using all HA3 sequences produced the same groups that were identified by tracking key amino acid substitutions (data not shown). Both groups apparently originated from the common ancestor A/Fujian/411/2002 lineage. The Wellington-like strains had longer distances than New York-like strains suggesting that Wellington-like strains had more genetic variations and they might be evolved from California lineage. For influenza B, the results showed that all the isolates were close to the B/Texas/3/2002 strain, which belongs to B/Yamagata/16/88 lineage and was the circulating strain from the 2004–2005 influenza season.</p>
<div class="figure" id="pone-0000419-g001"><div class="img"><a name="pone-0000419-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000419.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000419.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Rooted phylogenetic analysis of the <em>hemagglutinin</em> (HA3) genes of 15 representative influenza A/H3N2 isolates and reference strains.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(A) phylogenetic tree generated using sequences obtained from RPM v.1, (B) phylogenetic tree generated using de novo sequences from the same set of the isolates. Reference sequences were obtained from GenBank and indicated in underlined, bold font. Numbers above branches indicate bootstrap values from 1000 replicates. Note: ***-the A/Wyoming/3/03 (with IS)-like isolate. Bootstrap values above 50% were shown at branches. Scale near the bottom of each panel relates the length of a branch to the number of nucleic acid substitutions.</p>
<span>doi:10.1371/journal.pone.0000419.g001</span></div></div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Despite the effort to establish completely blind independent testing for every pathogen (viral and bacterial culture and, where appropriate, RT-PCR/PCR), differences in detection sensitivities of the methods and transportation issues necessitates using further confirmatory testing. This is because a measure of uncertainty is introduced in the quality of the reference assays as the services of a local reference laboratory could not be arranged (samples were transported to San Diego, CA). Furthermore, although large sample volumes are set aside for reference assays, a reduction in detection sensitivity is still inevitable because these samples are split and frozen before testing for each viral and bacterial agent in separate cultures. To reduce this uncertainty, archived samples stored at the time of the original testing are retested using real-time RT-PCR/PCR in house as an additional reference method on RPM-positive but culture-negative samples, using positive (positive for both RPM and culture) and negative controls (negative for both RPM and culture). Combining the results of all confirmatory tests demonstrates an excellent concordance with RPM v.1 (<a href="#pone-0000419-t002">Table 2</a> and Table S2–S3) with ≥98% overall agreement for all the organisms detected. The false positive/negative rate is within a reasonable range expected when testing any two methods such as is seen when PCR assay methods are compared to culture methods for all but two of the organisms. The detection sensitivity for Coronaviruses and <em>S. pneumoniae</em> by RPM v.1 is lower compared to the PCR methods and indicates the need to improve and/or optimize the primers sets used in the RPM v.1 assay for these organisms. However, the specificity remains very high for all organisms.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The ability to test for multiple pathogens is increasingly recognized as a necessity, especially in cases where differential diagnosis of etiological agents is difficult. Broad-spectrum diagnostics are particularly useful when different etiological agents that cause similar symptoms (such as respiratory illness) co-circulate in the same populations, or when co-infections of multiple organisms are of potential importance to epidemiology, symptomology, or treatment. However, it is difficult, cumbersome and costly to detect multiple pathogens and co-infections using culture methods or single test PCR methods. Currently, the reported coinfection rate among respiratory pathogens varies (4.5–23%) due to the different breadth and sensitivities of the detection methods <a href="#pone.0000419-BellauPujol1">[23]</a>. Using RPM v.1 (and confirmed by real-time RT-PCR/PCR), this study found that 58 (13.7%) of the tested samples showed viral or viral/bacterial co-infections (<a href="#pone-0000419-t004">Table 4</a>). It is a challenge that will only increase in difficulty as more organisms are covered to establish the validity of any broad spectrum platform against “gold” standards as few existing archive samples have been tested for such a large number of organisms.</p>
<div class="figure" id="pone-0000419-t004"><div class="img"><a name="pone-0000419-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000419" data-uri="info:doi/10.1371/journal.pone.0000419.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000419.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000419.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000419.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000419.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Comparative results for specimens positive for multiple pathogens.</span></strong></p><span>doi:10.1371/journal.pone.0000419.t004</span></div><a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>In addition to pathogen detection, sequences obtained using the RPM v.1 provide useful additional information for tracking strain variation without additional testing. This study shows that the majority of the clinical samples tested are positive for influenza A/B virus (74%), which is not surprising for the samples collected during the flu season. Comparison of conventional sequencing for representative influenza A positive samples demonstrates the utility of this information from RPM v.1. The isolates belong to two major lineages of influenza A/H3N2 and all isolates are evolved from the A/Fujian/411/02 strain, one component of the influenza vaccine for 2004–2005 season. Antigenic drift selected by vaccination apparently leads to new strains of influenza A/H3N2 during 2004–2005. As anticipated, the RPM v.1 identifies the A/California/7/2004 lineage as the dominant circulating strain. The interesting feature is the emergence of Wellington lineage strains at the same time in significant numbers, which are of the same clade as the A/Wisconsin/67/2005 strain, a suggested vaccine component for 2006–2007 season. These results confirm that the RPM v.1 is a useful tool itself not only for influenza surveillance but also for predicting antigenic variation that will be beneficial for vaccine candidate predication. Considering all of these results, a high level of confidence in the general reliability of the resequencing approach for simultaneous pathogen detection and strain/variant identification directly from clinical samples has been established.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The results show excellent correlation with reference assays, but neither the RPM v.1 nor the tests conducted in the reference assays represent complete coverage of all respiratory pathogens. Consequently, no pathogen is detected for 14% of the specimens, but the patients still exhibit flu-like symptoms. Furthermore, the RPM v.1 is designed as a proof-of-concept microarray for the detection of more than 20 common respiratory pathogens encountered among military basic trainees with large section of the chip dedicated to adenoviruses and influenza viruses, but in this work it is used for surveying a slightly different population within Washington, D.C. metropolitan region, which have a broad age, gender, and geographic distribution and may have a different mix of common pathogens. In addition, the major causes of the common cold, rhinoviruses and enteroviruses, are under represented on the RPM v.1. Subsequently, only a few strains of the recognized 99 strains of rhinovirus and none of the enteroviruses can be detected by RPM v.1. The confirmatory assays do identify several organisms (i.e. PIV2, <em>H. influenzae</em>) in clinical specimens that the RPM v.1 is not designed to detect. Additionally, preliminary testing of a new chip design with more thorough coverage (&gt;75 pathogens) identifies pathogens that the confirmatory assays detect but which the RPM v.1 cannot detect (unpublished data). This indicates the potential of this technology to reduce the number of samples not attributed to a tested organism to a significantly smaller number than is currently possible.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The major difficulties of the current technology are associated with the primer selection for amplification of the chosen targets. The design of the multiplex amplification strategy is a time consuming effort. The current system remains somewhat vulnerable to the rapid mutation of RNA viruses and each new resequencing array design that increases the number of pathogens would require recalibrating the multiplex primers mix. One solution that already mitigates this is the grouping of pathogens into subsets for amplifications that are recombined for chip analysis. In the future, the more variable pathogens can be partitioned into their own mix so that the performance for the conserved pathogens i.e. bacteria are not affected when primers are changed and only mixes that have targets added to them require recalibration. Simplified redesign and alternative amplification methods that provide the necessary sensitivity with more comprehensive coverage are also currently being investigated. Despite these limitations, the positive results of this study lead us to believe the resequencing microarray is an excellent candidate for the next generation pathogen detection tools and provide a modern, broad-spectrum infectious disease surveillance solution for critical decision makers, including healthcare providers, patients, public health authorities, and framers of biosecurity policy.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000419"></div><a name="pone.0000419.s001" id="pone.0000419.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000419.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000419.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.12 MB DOC)</p>
<a name="pone.0000419.s002" id="pone.0000419.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000419.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000419.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000419.s003" id="pone.0000419.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000419.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000419.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>This project would not be possible without the access to the samples collected during the Silent Guardian Demonstration Project.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DM FL BL AM JS KR DS RR EH Ev AK CT. Performed the experiments: KB AL CB. Analyzed the data: BL AM ZW RR EH. Contributed reagents/materials/analysis tools: CB. Wrote the paper: BL AM DS ZW. Other: Revision of the first draft: FL CT. Wrote sections of the paper: RR EH Ev DM KB. Revision of the paper: JS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000419-Lazarus1" id="pone.0000419-Lazarus1"></a>Lazarus R, Kleinman KP, Dashevsky I, DeMaria A, Platt R (2001) Using automated medical records for rapid identification of illness syndromes (syndromic surveillance): the example of lower respiratory infection. BMC Public Health  1: 9.  <ul class="find" data-citedArticleID="948838"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Using+automated+medical+records+for+rapid+identification+of+illness+syndromes+%28syndromic+surveillance%29%3A+the+example+of+lower+respiratory+infection.&amp;auth=&amp;atitle=Using+automated+medical+records+for+rapid+identification+of+illness+syndromes+%28syndromic+surveillance%29%3A+the+example+of+lower+respiratory+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+automated+medical+records+for+rapid+identification+of+illness+syndromes+%28syndromic+surveillance%29%3A+the+example+of+lower+respiratory+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+automated+medical+records+for+rapid+identification+of+illness+syndromes+%28syndromic+surveillance%29%3A+the+example+of+lower+respiratory+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000419-Briese1" id="pone.0000419-Briese1"></a>Briese T, Palacios G, Kokoris M, Jabado O, Liu Z, et al.  (2005) Diagnostic system for rapid and sensitive differential detection of pathogens. Emerg Infect Dis  11: 310–313.  <ul class="find" data-citedArticleID="948826" data-doi="10.3201/eid1102.040492"><li><a href="http://dx.doi.org/10.3201/eid1102.040492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diagnostic+system+for+rapid+and+sensitive+differential+detection+of+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diagnostic+system+for+rapid+and+sensitive+differential+detection+of+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000419-Ecker1" id="pone.0000419-Ecker1"></a>Ecker DJ, Sampath R, Blyn LB, Eshoo MW, Ivy C, et al.  (2005) Rapid identification and strain-typing of respiratory pathogens for epidemic surveillance. Proc Natl Acad Sci U S A  102: 8012–8017.  <ul class="find" data-citedArticleID="948832" data-doi="10.1073/pnas.0409920102"><li><a href="http://dx.doi.org/10.1073/pnas.0409920102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+identification+and+strain-typing+of+respiratory+pathogens+for+epidemic+surveillance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+identification+and+strain-typing+of+respiratory+pathogens+for+epidemic+surveillance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000419-Dunbar1" id="pone.0000419-Dunbar1"></a>Dunbar SA (2006) Applications of Luminex xMAP technology for rapid, high-throughput multiplexed nucleic acid detection. Clin Chim Acta  363: 71–82.  <ul class="find" data-citedArticleID="948830" data-doi="10.1016/j.cccn.2005.06.023"><li><a href="http://dx.doi.org/10.1016/j.cccn.2005.06.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applications+of+Luminex+xMAP+technology+for+rapid%2C+high-throughput+multiplexed+nucleic+acid+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applications+of+Luminex+xMAP+technology+for+rapid%2C+high-throughput+multiplexed+nucleic+acid+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000419-Wang1" id="pone.0000419-Wang1"></a>Wang D, Urisman A, Liu YT, Springer M, Ksiazek TG, et al.  (2003) Viral discovery and sequence recovery using DNA microarrays. PLoS Biol  1: E2.  <ul class="find" data-citedArticleID="948860" data-doi="10.1371/journal.pbio.0000002"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+discovery+and+sequence+recovery+using+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+discovery+and+sequence+recovery+using+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000419-Wilson1" id="pone.0000419-Wilson1"></a>Wilson KH, Wilson WJ, Radosevich JL, DeSantis TZ, Viswanathan VS, et al.  (2002) High-density microarray of small-subunit ribosomal DNA probes. Appl Environ Microbiol  68: 2535–2541.  <ul class="find" data-citedArticleID="948866" data-doi="10.1128/aem.68.5.2535-2541.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.5.2535-2541.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-density+microarray+of+small-subunit+ribosomal+DNA+probes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-density+microarray+of+small-subunit+ribosomal+DNA+probes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000419-Wilson2" id="pone.0000419-Wilson2"></a>Wilson WJ, Strout CL, DeSantis TZ, Stilwell JL, Carrano AV, et al.  (2002) Sequence-specific identification of 18 pathogenic microorganisms using microarray technology. Mol Cell Probes  16: 119–127.  <ul class="find" data-citedArticleID="948868" data-doi="10.1006/mcpr.2001.0397"><li><a href="http://dx.doi.org/10.1006/mcpr.2001.0397" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence-specific+identification+of+18+pathogenic+microorganisms+using+microarray+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence-specific+identification+of+18+pathogenic+microorganisms+using+microarray+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000419-Lin1" id="pone.0000419-Lin1"></a>Lin B, Vora GJ, Thach D, Walter E, Metzgar D, et al.  (2004) Use of oligonucleotide microarrays for rapid detection and serotyping of acute respiratory disease-associated adenoviruses. J Clin Microbiol  42: 3232–3239.  <ul class="find" data-citedArticleID="948842" data-doi="10.1128/jcm.42.7.3232-3239.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.7.3232-3239.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+oligonucleotide+microarrays+for+rapid+detection+and+serotyping+of+acute+respiratory+disease-associated+adenoviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+oligonucleotide+microarrays+for+rapid+detection+and+serotyping+of+acute+respiratory+disease-associated+adenoviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000419-Vora1" id="pone.0000419-Vora1"></a>Vora GJ, Meador CE, Bird MM, Bopp CA, Andreadis JD, et al.  (2005) Microarray-based detection of genetic heterogeneity, antimicrobial resistance, and the viable but nonculturable state in human pathogenic Vibrio spp. Proc Natl Acad Sci U S A  102: 19109–19114.  <ul class="find" data-citedArticleID="948856" data-doi="10.1073/pnas.0505033102"><li><a href="http://dx.doi.org/10.1073/pnas.0505033102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray-based+detection+of+genetic+heterogeneity%2C+antimicrobial+resistance%2C+and+the+viable+but+nonculturable+state+in+human+pathogenic+Vibrio+spp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray-based+detection+of+genetic+heterogeneity%2C+antimicrobial+resistance%2C+and+the+viable+but+nonculturable+state+in+human+pathogenic+Vibrio+spp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000419-Wang2" id="pone.0000419-Wang2"></a>Wang D, Coscoy L, Zylberberg M, Avila PC, Boushey HA, et al.  (2002) Microarray-based detection and genotyping of viral pathogens. Proc Natl Acad Sci U S A  99: 15687–15692.  <ul class="find" data-citedArticleID="948862" data-doi="10.1073/pnas.242579699"><li><a href="http://dx.doi.org/10.1073/pnas.242579699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray-based+detection+and+genotyping+of+viral+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray-based+detection+and+genotyping+of+viral+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000419-Vora2" id="pone.0000419-Vora2"></a>Vora GJ, Lin B, Gratwick K, Meador C, Hansen C, et al.  (2006) Co-infections of adenovirus species in previously vaccinated patients. Emerg Infect Dis  12: 921–930.  <ul class="find" data-citedArticleID="948858" data-doi="10.3201/eid1206.050245"><li><a href="http://dx.doi.org/10.3201/eid1206.050245" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Co-infections+of+adenovirus+species+in+previously+vaccinated+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Co-infections+of+adenovirus+species+in+previously+vaccinated+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000419-Kozal1" id="pone.0000419-Kozal1"></a>Kozal MJ, Shah N, Shen N, Yang R, Fucini R, et al.  (1996) Extensive polymorphisms observed in HIV-1 clade B protease gene using high-density oligonucleotide arrays. Nat Med  2: 753–759.  <ul class="find" data-citedArticleID="948836" data-doi="10.1038/nm0796-753"><li><a href="http://dx.doi.org/10.1038/nm0796-753" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+polymorphisms+observed+in+HIV-1+clade+B+protease+gene+using+high-density+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+polymorphisms+observed+in+HIV-1+clade+B+protease+gene+using+high-density+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000419-Hacia1" id="pone.0000419-Hacia1"></a>Hacia JG (1999) Resequencing and mutational analysis using oligonucleotide microarrays. Nat Genet  21: 42–47.  <ul class="find" data-citedArticleID="948834" data-doi="10.1038/4469"><li><a href="http://dx.doi.org/10.1038/4469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resequencing+and+mutational+analysis+using+oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resequencing+and+mutational+analysis+using+oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000419-Davignon1" id="pone.0000419-Davignon1"></a>Davignon L, Walter EA, Mueller KM, Barrozo CP, Stenger DA, et al.  (2005) Use of resequencing oligonucleotide microarrays for identification of Streptococcus pyogenes and associated antibiotic resistance determinants. J Clin Microbiol  43: 5690–5695.  <ul class="find" data-citedArticleID="948828" data-doi="10.1128/jcm.43.11.5690-5695.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.11.5690-5695.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+resequencing+oligonucleotide+microarrays+for+identification+of+Streptococcus+pyogenes+and+associated+antibiotic+resistance+determinants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+resequencing+oligonucleotide+microarrays+for+identification+of+Streptococcus+pyogenes+and+associated+antibiotic+resistance+determinants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000419-Lin2" id="pone.0000419-Lin2"></a>Lin B, Blaney KM, Malanoski AP, Ligler AG, Schnur JM, et al.  (2007) Using a resequencing microarray as a multiple respiratory pathogen detection assay. J Clin Microbiol  45: 443–452.  <ul class="find" data-citedArticleID="948844" data-doi="10.1128/jcm.01870-06"><li><a href="http://dx.doi.org/10.1128/jcm.01870-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+a+resequencing+microarray+as+a+multiple+respiratory+pathogen+detection+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+a+resequencing+microarray+as+a+multiple+respiratory+pathogen+detection+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000419-Lin3" id="pone.0000419-Lin3"></a>Lin B, Wang Z, Vora GJ, Thornton JA, Schnur JM, et al.  (2006) Broad-spectrum respiratory tract pathogen identification using resequencing DNA microarrays. Genome Res  16: 527–535.  <ul class="find" data-citedArticleID="948846" data-doi="10.1101/gr.4337206"><li><a href="http://dx.doi.org/10.1101/gr.4337206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad-spectrum+respiratory+tract+pathogen+identification+using+resequencing+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad-spectrum+respiratory+tract+pathogen+identification+using+resequencing+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000419-Wang3" id="pone.0000419-Wang3"></a>Wang Z, Daum LT, Vora GJ, Metzgar D, Walter EA, et al.  (2006) Identifying influenza viruses with resequencing microarrays. Emerg Infect Dis  12: 638–646.  <ul class="find" data-citedArticleID="948864" data-doi="10.3201/eid1204.051441"><li><a href="http://dx.doi.org/10.3201/eid1204.051441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+influenza+viruses+with+resequencing+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+influenza+viruses+with+resequencing+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000419-Lewis1" id="pone.0000419-Lewis1"></a>Lewis MD, Pavlin JA, Mansfield JL, O'Brien S, Boomsma LG, et al.  (2002) Disease outbreak detection system using syndromic data in the greater Washington DC area. Am J Prev Med  23: 180–186.  <ul class="find" data-citedArticleID="948840" data-doi="10.1016/s0749-3797(02)00490-7"><li><a href="http://dx.doi.org/10.1016/s0749-3797(02)00490-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disease+outbreak+detection+system+using+syndromic+data+in+the+greater+Washington+DC+area." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disease+outbreak+detection+system+using+syndromic+data+in+the+greater+Washington+DC+area.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000419-Malanoski1" id="pone.0000419-Malanoski1"></a>Malanoski AP, Lin B, Wang Z, Schnur JM, Stenger DA (2006) Automated identification of multiple micro-organisms from resequencing DNA microarrays. Nucleic Acids Res  34: 5300–5311.  <ul class="find" data-citedArticleID="948852" data-doi="10.1093/nar/gkl565"><li><a href="http://dx.doi.org/10.1093/nar/gkl565" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+identification+of+multiple+micro-organisms+from+resequencing+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+identification+of+multiple+micro-organisms+from+resequencing+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000419-Lin4" id="pone.0000419-Lin4"></a>Lin B, Blaney KM, Malanoski AP, Ligler AG, Schnur JM, et al.  (2006) Using Resequencing Microarray as a Multiple Respiratory Pathogen Detection Assay. J Clin Microbio.   <ul class="find" data-citedArticleID="948848" data-doi="10.1128/jcm.01870-06"><li><a href="http://dx.doi.org/10.1128/jcm.01870-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+Resequencing+Microarray+as+a+Multiple+Respiratory+Pathogen+Detection+Assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+Resequencing+Microarray+as+a+Multiple+Respiratory+Pathogen+Detection+Assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000419-Madhi1" id="pone.0000419-Madhi1"></a>Madhi SA, Klugman KP (2004) A role for Streptococcus pneumoniae in virus-associated pneumonia. Nat Med  10: 811–813.  <ul class="find" data-citedArticleID="948850" data-doi="10.1038/nm1077"><li><a href="http://dx.doi.org/10.1038/nm1077" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+Streptococcus+pneumoniae+in+virus-associated+pneumonia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+Streptococcus+pneumoniae+in+virus-associated+pneumonia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000419-Peltola1" id="pone.0000419-Peltola1"></a>Peltola VT, McCullers JA (2004) Respiratory viruses predisposing to bacterial infections: role of neuraminidase. Pediatr Infect Dis J  23: S87–97.  <ul class="find" data-citedArticleID="948854" data-doi="10.1097/01.inf.0000108197.81270.35"><li><a href="http://dx.doi.org/10.1097/01.inf.0000108197.81270.35" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Respiratory+viruses+predisposing+to+bacterial+infections%3A+role+of+neuraminidase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Respiratory+viruses+predisposing+to+bacterial+infections%3A+role+of+neuraminidase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000419-BellauPujol1" id="pone.0000419-BellauPujol1"></a>Bellau-Pujol S, Vabret A, Legrand L, Dina J, Gouarin S, et al.  (2005) Development of three multiplex RT-PCR assays for the detection of 12 respiratory RNA viruses. J Virol Methods  126: 53–63.  <ul class="find" data-citedArticleID="948824" data-doi="10.1016/j.jviromet.2005.01.020"><li><a href="http://dx.doi.org/10.1016/j.jviromet.2005.01.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+three+multiplex+RT-PCR+assays+for+the+detection+of+12+respiratory+RNA+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+three+multiplex+RT-PCR+assays+for+the+detection+of+12+respiratory+RNA+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.XML" value="82163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.PDF" value="168984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t001.PNG_L" value="78587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t001.PNG_M" value="75381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t001.PNG_S" value="3412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t001.TIF" value="423484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t001.PNG_I" value="33647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t002.PNG_L" value="40577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t002.PNG_M" value="39839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t002.PNG_S" value="2538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t002.TIF" value="239468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t002.PNG_I" value="18298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t003.PNG_L" value="228277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t003.PNG_M" value="87311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t003.PNG_S" value="3374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t003.TIF" value="1030094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t003.PNG_I" value="37248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.g001.PNG_L" value="60363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.g001.PNG_M" value="56265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.g001.PNG_S" value="2442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.g001.TIF" value="523596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.g001.PNG_I" value="26056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t004.PNG_L" value="140889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t004.PNG_M" value="96586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t004.PNG_S" value="6533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t004.TIF" value="726682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.t004.PNG_I" value="69903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.s001.DOC" value="116736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.s002.DOC" value="30208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000419.s003.DOC" value="29696"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000419&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000419" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000419&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000419&volume=&issue=&title=Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population&author_name=Baochuan%20Lin%2C%20Anthony%20P.%20Malanoski%2C%20Zheng%20Wang%2C%20Kate%20M.%20Blaney%2C%20Adam%20G.%20Ligler%2C%20Robb%20K.%20Rowley%2C%20Eric%20H.%20Hanson%2C%20Erik%20von%20Rosenvinge%2C%20Frances%20S.%20Ligler%2C%20Anne%20W.%20Kusterbeck%2C%20David%20Metzgar%2C%20Christopher%20P.%20Barrozo%2C%20Kevin%20L.%20Russell%2C%20Clark%20Tibbetts%2C%20Joel%20M.%20Schnur%2C%20David%20A.%20Stenger&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000419" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000419" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000419" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000419&amp;t=Application%20of%20Broad-Spectrum%2C%20Sequence-Based%20Pathogen%20Identification%20in%20an%20Urban%20Population" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000419&title=Application%20of%20Broad-Spectrum%2C%20Sequence-Based%20Pathogen%20Identification%20in%20an%20Urban%20Population&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000419&amp;title=Application%20of%20Broad-Spectrum%2C%20Sequence-Based%20Pathogen%20Identification%20in%20an%20Urban%20Population" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000419" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000419&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Application of Broad-Spectrum, Sequence-Based Pathogen Identification in an Urban Population';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000419';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Application%20of%20Broad-Spectrum%2C%20Sequence-Based%20Pathogen%20Identification%20in%20an%20Urban%20Population http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000419" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000419" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+pathogens%22" title="Search for articles in the subject area:'Bacterial pathogens'"><div class="flagText">Bacterial pathogens</div></a>
              <div data-categoryid="42717" data-articleid="23676"
                   data-categoryname="Bacterial pathogens"
                   class="flagImage" title="Flag 'Bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Co-infections%22" title="Search for articles in the subject area:'Co-infections'"><div class="flagText">Co-infections</div></a>
              <div data-categoryid="21527" data-articleid="23676"
                   data-categoryname="Co-infections"
                   class="flagImage" title="Flag 'Co-infections' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Influenza%22" title="Search for articles in the subject area:'Influenza'"><div class="flagText">Influenza</div></a>
              <div data-categoryid="48257" data-articleid="23676"
                   data-categoryname="Influenza"
                   class="flagImage" title="Flag 'Influenza' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Influenza+A+virus%22" title="Search for articles in the subject area:'Influenza A virus'"><div class="flagText">Influenza A virus</div></a>
              <div data-categoryid="43725" data-articleid="23676"
                   data-categoryname="Influenza A virus"
                   class="flagImage" title="Flag 'Influenza A virus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="23676"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23676"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="23676"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+pathogens%22" title="Search for articles in the subject area:'Viral pathogens'"><div class="flagText">Viral pathogens</div></a>
              <div data-categoryid="42945" data-articleid="23676"
                   data-categoryname="Viral pathogens"
                   class="flagImage" title="Flag 'Viral pathogens' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3653'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7545'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6886&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
