

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: MamuSNP: A Resource for Rhesus Macaque (Macaca mulatta) Genomics</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000438"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000438" />

    <meta name="citation_title" content="MamuSNP: A Resource for Rhesus Macaque (Macaca mulatta) Genomics"/>
    <meta itemprop="name" content="MamuSNP: A Resource for Rhesus Macaque (Macaca mulatta) Genomics"/>

      <meta name="citation_author" content="Ripan S. Malhi"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of Illinois Urbana-Champaign, Urbana, Illinois, United States of America"/>
            <meta name="citation_author_institution" content="Institute of Genomic Biology, University of Illinois, Urbana-Champaign, Urbana, Illinois, United States of America"/>
      <meta name="citation_author" content="Brad Sickler"/>
            <meta name="citation_author_institution" content="Bioinformatics Core, Genome Center, University of California Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Dawei Lin"/>
            <meta name="citation_author_institution" content="Bioinformatics Core, Genome Center, University of California Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Jessica Satkoski"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of California, Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Raul Y. Tito"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of Illinois Urbana-Champaign, Urbana, Illinois, United States of America"/>
      <meta name="citation_author" content="Debbie George"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of California, Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Sreetharan Kanthaswamy"/>
            <meta name="citation_author_institution" content="California National Primate Research Center, University of California, Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="David Glenn Smith"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of California, Davis, Davis, California, United States of America"/>
            <meta name="citation_author_institution" content="California National Primate Research Center, University of California, Davis, Davis, California, United States of America"/>

    <meta name="citation_date" content="2007/5/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000438.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e438"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Evolutionary and biomedical insights from the rhesus macaque genome.; citation_journal_title=Science; citation_volume=316; citation_number=1; citation_pages=222-234; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Single nucleotide polymorphisms (SNPs) distinguish Indian-origin and Chinese-origin rhesus macaques (Macaca mulatta).; citation_author=B Ferguson; citation_author=SL Street; citation_author=H Wright; citation_author=C Pearson; citation_author=Y Jia; citation_journal_title=BMC Genomics; citation_volume=8; citation_number=2; citation_pages=43; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Demographic Histories and Patterns of Linkage Disequilibrium for Chinese and Indian Rhesus Macaques.; citation_author=RD Hernandez; citation_author=MJ Hubisz; citation_author=D Wheeler; citation_author=DG Smith; citation_author=B Ferguson; citation_journal_title=Science; citation_volume=316; citation_number=3; citation_pages=240-243; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Scanning human gene deserts for long-range enhancers.; citation_author=MA Nobrega; citation_author=I Ovcharenko; citation_author=V Afzal; citation_author=EM Rubin; citation_journal_title=Science; citation_volume=302; citation_number=4; citation_pages=413; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Extensive sharing of MHC class II alleles between rhesus and cynomolgus macaques.; citation_author=GG Doxiadis; citation_author=AJ Rouweler; citation_author=NH deGroot; citation_author=A Louwerse; citation_author=N Otting; citation_journal_title=Immunogenetics; citation_volume=58; citation_number=5; citation_pages=259-68; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fundamentals of molecular evolution Sunderland,; citation_author=D Graur; citation_author=WH Li; citation_number=6; citation_date=2000; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=Sequencing and analysis of Neanderthal genomic DNA.; citation_author=JP Noonan; citation_author=G Coop; citation_author=S Kudaravalli; citation_author=D Smith; citation_author=J Krause; citation_journal_title=Science; citation_volume=314; citation_number=7; citation_pages=1113-8; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microsatellite markers for standardized genetic management of captive colonies of rhesus macaques (Macaca mulatta).; citation_author=S Kanthaswamy; citation_author=A von Dollen; citation_author=JD Kurushima; citation_author=O Alminas; citation_author=J Roger; citation_journal_title=Am J Primatol; citation_volume=68; citation_number=8; citation_pages=73-95; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA Variation Within and Among Regional Populations of Longtail macaques (Macaca fascicularis in Relation to Other Species of the Fascicularis Group of Macaques.; citation_author=DG Smith; citation_author=JW McDonough; citation_author=DA George; citation_journal_title=Am J Primatol; citation_volume=69; citation_number=9; citation_pages=182-198; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic characterization of wild and captive rhesus macaques in China.; citation_author=JA Satkoski; citation_author=DA George; citation_author=DG Smith; citation_author=S Kanthaswamy; citation_journal_title=J Med Primatol.; citation_number=10; citation_date=in press; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=25; citation_number=11; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A sequencing method based on Real-time pyrophosphate.; citation_author=M Ronaghi; citation_author=M AhlZn; citation_author=P NyrZn; citation_journal_title=Science; citation_volume=281; citation_number=12; citation_pages=363-365; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000438" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="MamuSNP: A Resource for Rhesus Macaque (Macaca mulatta) Genomics"/>
    <meta name="twitter:description" content="We developed a novel method for identifying SNPs widely distributed throughout the coding and non-coding regions of a genome. The method uses large-scale parallel pyrosequencing technology in combination with bioinformatics tools. We used this method to generate approximately 23,000 candidate SNPs throughout the Macaca mulatta genome. We estimate that over 60% of the SNPs will be of high frequency and useful for mapping QTLs, genetic management, and studies of individual relatedness, whereas other less frequent SNPs may be useful as population specific markers for ancestry identification. We have created a web resource called MamuSNP to view the SNPs and associated information online. This resource will also be useful for researchers using a wide variety of Macaca species in their research."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000438.g003"/>

  <meta property="og:title" content="MamuSNP: A Resource for Rhesus Macaque (Macaca mulatta) Genomics" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000438" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4822'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6805'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7638&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000438">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000438" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000438&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    <i>Mamu</i>SNP: A Resource for Rhesus Macaque (<i>Macaca mulatta</i>) Genomics
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ripan S. Malhi
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:malhi@uiuc.edu">malhi@uiuc.edu</a></p>

                <p>Affiliations:
                  Department of Anthropology, University of Illinois Urbana-Champaign, Urbana, Illinois, United States of America, 
                  Institute of Genomic Biology, University of Illinois, Urbana-Champaign, Urbana, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Brad Sickler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioinformatics Core, Genome Center, University of California Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dawei Lin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioinformatics Core, Genome Center, University of California Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jessica Satkoski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Anthropology, University of California, Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Raul Y. Tito, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Anthropology, University of Illinois Urbana-Champaign, Urbana, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Debbie George, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Anthropology, University of California, Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sreetharan Kanthaswamy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  California National Primate Research Center, University of California, Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Glenn Smith
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Anthropology, University of California, Davis, Davis, California, United States of America, 
                  California National Primate Research Center, University of California, Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 09, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000438</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000438" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000438" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000438" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000438" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000438-g001" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000438-g002" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000438-g003" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000438-t001" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000438">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000438" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>We developed a novel method for identifying SNPs widely distributed throughout the coding and non-coding regions of a genome. The method uses large-scale parallel pyrosequencing technology in combination with bioinformatics tools. We used this method to generate approximately 23,000 candidate SNPs throughout the <em>Macaca mulatta</em> genome. We estimate that over 60% of the SNPs will be of high frequency and useful for mapping QTLs, genetic management, and studies of individual relatedness, whereas other less frequent SNPs may be useful as population specific markers for ancestry identification. We have created a web resource called <em>Mamu</em>SNP to view the SNPs and associated information online. This resource will also be useful for researchers using a wide variety of <em>Macaca</em> species in their research.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Malhi RS, Sickler B, Lin D, Satkoski J, Tito RY, et al.  (2007) <em>Mamu</em>SNP: A Resource for Rhesus Macaque (<em>Macaca mulatta</em>) Genomics. PLoS ONE 2(5):
          e438.
            doi:10.1371/journal.pone.0000438</p><p><strong>Academic Editor: </strong>Jean-Nicolas Volff, Ecole Normale Supérieure de Lyon, France</p><p><strong>Received:</strong> March 9, 2007; <strong>Accepted:</strong> April 19, 2007; <strong>Published:</strong> May 9, 2007</p><p><strong>Copyright:</strong> © 2007 Malhi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>NIH grant RR05090 to DGS</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Rhesus macaques (<em>Macaca mulatta</em>) are a widely used and valuable model for biomedical research and understanding the etiology of human diseases. With the completion of a draft sequence of the rhesus macaque genome <a href="#pone.0000438-Rhesus1">[1]</a>, the ability to identify genetic polymorphisms throughout the entire genome will vastly improve the utility of rhesus macaques for investigating the heritable components of complex diseases. In particular, the identification of SNPs (Single Nucleotide Polymorphisms) will contribute to a high-density linkage map that can be used to perform QTL (Quantitative Trait Loci) and association studies to identify genes that contribute to a phenotype of interest. Research to begin finding diagnostic SNPs within or near genes in the <em>M. mulatta</em> genome has already begun <a href="#pone.0000438-Ferguson1">[2]</a>, <a href="#pone.0000438-Hernandez1">[3]</a>. In addition SNPs are useful markers for genetic management in captive colonies because they are widely distributed and abundant in the genome and are amenable to automated high-throughput analyses.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Large-scale parallel pyrosequencing technology from 454 Life Science™ is a new method that can quickly obtain hundreds of thousands of short (approximately 100bp) DNA sequences. Due to the nature of the sequencing method, these sequences are distributed randomly across the entire genome. This wide range of coverage makes it advantageous to discover unlinked common SNPs in coding and non-coding regions by screening one or only a few animals. This method of SNP identification is unique because it can be used to identify SNPs distant from any genes. Therefore, this method is useful for identifying regulatory regions located in genomic deserts far from a gene, but which still contribute to variation in a phenotype (e.g. <a href="#pone.0000438-Nobrega1">[4]</a>).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we use large-scale parallel pyrosequencing from 454 Life Science™ to generate approximately 23,000 candidate SNPs throughout the rhesus macaque genome; the candidate SNPs are distributed across all 20 autosomes and the X chromosome. Each candidate SNP is mapped to an exact location on the current draft of the genome (07/06) and flanking DNA sequence for each SNP is provided so that primers can be easily designed to assay the SNP. Over 1,500 SNPs were sequenced at least twice through the pyosequencing process and a subset of these SNPs have been confirmed via Sanger sequencing in a panel of animals from geographically diverse origins. All candidate SNPs and associated information can be accessed at the website <a href="http://mamusnp.ucdavis.edu">http://mamusnp.ucdavis.edu</a>. Updated SNP frequencies in populations as well as additional verified SNPs will be routinely uploaded to the website.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Cynomolgus macaques (also known as longtail or crab-eating macaques), <em>Macaca fascicularis,</em> are becoming an important non-human primate model and may soon outnumber rhesus macaques in biomedical research. Doxiadis et al. <a href="#pone.0000438-Doxiadis1">[5]</a> has recently shown a relatively high level of allele sharing at class II MHC loci between rhesus and cynomolgus macaques. In addition, a high level of allele sharing has also been reported at STR loci between the two species (Satkoski pers com). Therefore many of the SNPs identified in rhesus macaques and reported on the <em>Mamu</em>SNP website may also be polymorphic in cynomolgus macaques and other closely related primates. Although these SNPs would exhibit a significant ascertainment bias in cynomolgus macaques, they may still be useful as markers for QTL analyses.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The average read length received from 454 Life Science™ was 103.92 bases. The DNA sequence was of high quality except within regions of homopolymer tracts, a known limitation of pyrosequencing. We obtained 339,967 reads from a single run. These reads were blasted against a repeat masked version of the macaque genome. Of the reads 42% exhibited a high match score with the macaque genome. It is likely that the remaining reads aligned to repeat masked regions of the genome or represent artifacts or DNA sequence from exogenous organisms in the original blood sample from which DNA was extracted. Of the total number of reads, 58% did not match the macaque genome with a ≥98% identity, 20% exhibited a perfect match with the DNA sequence from the macaque genome and 8% contained the information for candidate SNPs (<a href="#pone-0000438-g001">Figure 1</a>). After filtering, we identified 22,892 candidate SNPs from a single run (<a href="#pone.0000438.s001">Table S1</a>). The SNPs are evenly distributed across all chromosomes in the genome (<a href="#pone-0000438-g002">Figure 2</a>). However, the expected (based on simulations) and observed numbers of sequence overlaps were statistically significantly different from each other (p = 0.0001). The excess number of overlaps is likely due to an “inefficiency” in the 454 Life Science™ system resulting in beads with the same DNA segment being amplified and sequenced multiple times (454 Life Science™ pers com). This excess number of overlaps in the sequencing system is desirable for SNP identification and verification and as a result we were able to obtain at least 2×coverage on 1,947 SNPs. We trimmed this overlap SNP set by removing SNPs that exhibited a Sanger quality score below 60 in the draft sequence or a pyrosequencing score below 20. This resulted in an overlap set comprising 1,559 SNPs distributed across the genome (<a href="#pone-0000438-g003">Figure 3</a>). The transition/transversion ratio is within the expected limits (2.36) <a href="#pone.0000438-Graur1">[6]</a> and provides confidence that the SNPs in the overlap are not due to artifact or error. Previous studies using the same sequencing technology also report statistics consistent with a low level of error once the sequences have been appropriately screened for quality <a href="#pone.0000438-Noonan1">[7]</a>.</p>
<div class="figure" id="pone-0000438-g001"><div class="img"><a name="pone-0000438-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000438.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000438.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Proportion of 454 pyrosequencing DNA fragments that exhibited SNPs.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>No match indicates a DNA fragment not from <em>M. mulatta</em>. A near match indicates a DNA fragment with at least a 98% match to <em>M. mulatta</em> but unsuitable for SNP detection. A perfect match indicates a DNA fragment with a 100% match with <em>M. mulatta</em>. A SNP indicates a DNA fragment with a near perfect match with <em>M. mulatta</em> and suitable sequence for SNP detection.</p>
<span>doi:10.1371/journal.pone.0000438.g001</span></div><div class="figure" id="pone-0000438-g002"><div class="img"><a name="pone-0000438-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000438.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000438.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution of Candidate SNPs across genome.</span></strong></p><span>doi:10.1371/journal.pone.0000438.g002</span></div><div class="figure" id="pone-0000438-g003"><div class="img"><a name="pone-0000438-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000438.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000438.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Distribution of overlap SNPs across the genome.</span></strong></p><span>doi:10.1371/journal.pone.0000438.g003</span></div>
<h4>Confirmation of Candidate SNPs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We were able to confirm the presence of 26 high-frequency SNPs (63.4%) out of the 41 loci sequenced in a pooled sample of four rhesus macaques (<a href="#pone-0000438-t001">Table 1</a>). Five of the 21 high-frequency SNPs were found in regions characterized as homologues to genes in humans. Based on the small number of individuals in the pooled sample the SNP frequencies should be considered as approximations. It is likely that most of the remaining 15 SNPs are of low-frequency but some may be useful as population specific markers for ancestry identification. A subset of three SNPs confirmed in the pooled sample was also confirmed with 100% accuracy in the DNA sequences of individuals used in the pooled sample. These results suggest that a majority of the candidate SNPs identified using this method will be high frequency and will be useful genetic markers for mapping QTLs, genetic management <a href="#pone.0000438-Kanthaswamy1">[8]</a>, and determining relatedness.</p>
<div class="figure" id="pone-0000438-t001"><div class="img"><a name="pone-0000438-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000438" data-uri="info:doi/10.1371/journal.pone.0000438.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000438.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000438.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000438.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000438.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000438.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Candidate SNPs confirmed through Sanger sequencing.</span></strong></p><span>doi:10.1371/journal.pone.0000438.t001</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Experimental and Computational Information Integration</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>It is a non-trivial task to manage and analyze hundreds of thousands of sequences and their associate computational results, especially in a complete genome context. We created a database that has integrated 454 Life Science™ DNA sequences, their corresponding quality scores, BLAST outputs, rhesus macaque genome sequences and their corresponding Sanger quality scores. The database is built on top of MySQL with a web interface that allows dynamic queries through Perl CGI (Common Gateway Interface). The website for this database dubbed <em>Mamu</em>SNP, can be accessed at <a href="http://mamusnp.ucdavis.edu">http://mamusnp.ucdavis.edu</a>. <em>Mamu</em>SNP supports queries on one or more 454 sequence IDs to create a summary report with 454 sequences, relative positions of each base pair, quality scores in a table format and with mismatch sequences high-lighted by a different color. The details of BLAST outputs including BLAST scores and the sequence alignments with reference are also shown in the report. The query feature is a useful tool for verifying computational results and SNP candidate selection. Since queries concerning chromosome number and map positions are supported, this tool is particularly useful to search for novel SNPs in a particular genomic region of interest, for screening possible cases of overlaps between 454 sequences within a genomic region and for the retrieval of sequences flanking the target template. The latter facilitates primer design for the experimental verification of candidate SNPs.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We chose to pyrosequence an animal from western China to maximize diversity when compared to the draft sequence from a rhesus macaque of Indian ancestry. Sample Sch00R1684 was obtained from Sichuan Province, China and the provenience of this sample was confirmed through mitochondrial DNA (mtDNA) sequencing (see Genbank Accession # DQ373245: <a href="#pone.0000438-Smith1">[9]</a>). This sample was chosen for analysis because there is a high probability, based on its nuclear (STR) and mitochondrial genome, that this animal has no admixture with eastern Chinese or Indian rhesus macaques in its ancestry <a href="#pone.0000438-Satkoski1">[10]</a>. Three separate DNA extractions were performed from <em>M. mulatta</em> blood sample Sch00R1684 using a QiaAmp Blood Extraction Mini Kit. The three extracts were pooled, purified, and concentrated using a Millipore Microcon YM-100 centrifugal Filter Device. Five micrograms of concentrated extract (at&gt;300 ng/ul) was submitted to 454 Life Sciences in Branford, CT for pyrosequencing.</p>

<h4>Analysis and alignment of Raw Sequences</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All DNA sequences once received from 454 were blasted against the MMUL 1.0 genome assembly (<a href="http://www.genome.wustl.edu/genome.cgi?">http://www.genome.wustl.edu/genome.cgi?</a>GENOME = Macaca%20mulatta&amp;SECTION = assemblies) from the Baylor College of Human Medicine. Prior to the search, repetitive sequences and low quality regions were masked out of the database at Baylor. The alignment was performed with NCBI's BLASTN variant of the BLASTALL program <a href="#pone.0000438-Altschul1">[11]</a> without any low-complexity filters and an expectation value of 10.0. Chromosome, 454 sequence ID, alignment location, mismatch information, and gap information was parsed with a Perl script using BioPerl modules. To standardize conventions, alignments and mismatch locations were relabeled relative to the 5′ strand of the chromosome.</p>


<h4>Identification of SNP Candidates</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>To avoid reporting non-informative sequence polymorphisms, sequence alignments between 454 sequences and the reference genome that corresponded to variable MHC regions, and regions showing a statistically improbable number of sequence overlaps (&gt;4 overlaps) were removed from further analysis. Only alignments with ≥98% identity or only one mismatch, with the second alignment scoring worst than the first were accepted for analysis. An identity match of 98% was selected to allow one or two mismatches on a single sequence. To control for pyrosequencing read errors in homopolymeric regions due to non-linear light response <a href="#pone.0000438-Ronaghi1">[12]</a>, any mismatch that was an extension/truncation of an existing homopolymeric repeat was removed from further analysis. All mismatches occurring within 5 bp of a gap were screened out. If overlapping alignments from 454 re-sequencing artifacts contain conflicting reports of mismatches the mismatch site was screened out. Any mismatch passing the aforementioned filters was considered as a potential SNP candidate.</p>


<h4>Confirmation of SNP Candidates</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We screened a subset of candidate SNPs using Sanger sequencing of a pooled sample. The putative SNPs were first verified and checked for paralogous sequences by comparing the 454-generated fragment sequence to the rhesus macaque genome, using NCBI's rhesus macaque genome megaBLAST resource. (<a href="http://www.ncbi.nlm.nih.gov/projects/genome/seq/BlastGen/BlastGen.cgi?">http://www.ncbi.nlm.nih.gov/projects/gen​ome/seq/BlastGen/BlastGen.cgi?</a>taxid = 9544). Fragments that matched multiple regions of the genome were discarded to avoid complications during primer-design and genotyping. Forty-one SNP loci in forty-one independent fragments were chosen for Sanger sequencing confirmation. After identifying the candidate loci, 400 bp of flanking sequence on either side of the SNP were retrieved using the ‘Sequence Query’ feature of the <em>Mamu</em>SNP web interface (<a href="http://mamusnp.ucdavis.edu">http://mamusnp.ucdavis.edu</a>); these sequences were used to generate oligonucleotide primers.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>We created a pool of four <em>M. mulatta</em> samples exhibiting different mitochondrial DNA haplogroups and sequenced the forward and reverse strand of DNA fragments encompassing the candidate SNPs. By sequencing eight chromosomes in a pooled sample we were able to identify SNPs exhibited in the total population at greater than 12.5% in frequency. We define SNPs at this frequency or greater as “high-frequency SNPs.” The peaks of the electropherograms for the polymorphic site were visually inspected and if more than one peak was present at this position in both forward and reverse sequences it was considered to be a high-frequency SNP. Three loci that exhibited high-frequency SNPs in the pooled sample were randomly sequenced for all four individuals to confirm the SNP in individuals as well as the pooled sample. The frequencies of the major and minor alleles for these SNPs are given in <a href="#pone-0000438-t001">Table 1</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000438"></div><a name="pone.0000438.s001" id="pone.0000438.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000438.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of candidate SNPs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000438.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(6.91 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Monalyn Umali for assistance with screening candidate SNPs.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RM DL SK DS. Performed the experiments: JS RT DG. Analyzed the data: RM BS DL JS RT. Contributed reagents/materials/analysis tools: RM DS. Wrote the paper: RM BS JS DS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000438-Rhesus1" id="pone.0000438-Rhesus1"></a>Rhesus Macaque Genome Sequencing and Analysis Consortium (2007) Evolutionary and biomedical insights from the rhesus macaque genome. Science  316: 222–234.  <ul class="find" data-citedArticleID="950866"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolutionary+and+biomedical+insights+from+the+rhesus+macaque+genome.&amp;auth=&amp;atitle=Evolutionary+and+biomedical+insights+from+the+rhesus+macaque+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+and+biomedical+insights+from+the+rhesus+macaque+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+and+biomedical+insights+from+the+rhesus+macaque+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000438-Ferguson1" id="pone.0000438-Ferguson1"></a>Ferguson B, Street SL, Wright H, Pearson C, Jia Y, et al.  (2007) Single nucleotide polymorphisms (SNPs) distinguish Indian-origin and Chinese-origin rhesus macaques (Macaca mulatta). BMC Genomics  8: 43.  <ul class="find" data-citedArticleID="950854" data-doi="10.1186/1471-2164-8-480"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+nucleotide+polymorphisms+%28SNPs%29+distinguish+Indian-origin+and+Chinese-origin+rhesus+macaques+%28Macaca+mulatta%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+nucleotide+polymorphisms+%28SNPs%29+distinguish+Indian-origin+and+Chinese-origin+rhesus+macaques+%28Macaca+mulatta%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000438-Hernandez1" id="pone.0000438-Hernandez1"></a>Hernandez RD, Hubisz MJ, Wheeler D, Smith DG, Ferguson B, et al.  (2007) Demographic Histories and Patterns of Linkage Disequilibrium for Chinese and Indian Rhesus Macaques. Science  316: 240–243.  <ul class="find" data-citedArticleID="950858" data-doi="10.1126/science.1140462"><li><a href="http://dx.doi.org/10.1126/science.1140462" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Demographic+Histories+and+Patterns+of+Linkage+Disequilibrium+for+Chinese+and+Indian+Rhesus+Macaques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Demographic+Histories+and+Patterns+of+Linkage+Disequilibrium+for+Chinese+and+Indian+Rhesus+Macaques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000438-Nobrega1" id="pone.0000438-Nobrega1"></a>Nobrega MA, Ovcharenko I, Afzal V, Rubin EM (2003) Scanning human gene deserts for long-range enhancers. Science  302: 413.  <ul class="find" data-citedArticleID="950862" data-doi="10.1126/science.1088328"><li><a href="http://dx.doi.org/10.1126/science.1088328" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scanning+human+gene+deserts+for+long-range+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scanning+human+gene+deserts+for+long-range+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000438-Doxiadis1" id="pone.0000438-Doxiadis1"></a>Doxiadis GG, Rouweler AJ, deGroot NH, Louwerse A, Otting N, et al.  (2006) Extensive sharing of MHC class II alleles between rhesus and cynomolgus macaques. Immunogenetics  58: 259–68.  <ul class="find" data-citedArticleID="950852" data-doi="10.1007/s00251-006-0083-8"><li><a href="http://dx.doi.org/10.1007/s00251-006-0083-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+sharing+of+MHC+class+II+alleles+between+rhesus+and+cynomolgus+macaques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+sharing+of+MHC+class+II+alleles+between+rhesus+and+cynomolgus+macaques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000438-Graur1" id="pone.0000438-Graur1"></a>Graur D, Li WH (2000) Fundamentals of molecular evolution Sunderland,. MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0000438-Noonan1" id="pone.0000438-Noonan1"></a>Noonan JP, Coop G, Kudaravalli S, Smith D, Krause J, et al.  (2006) Sequencing and analysis of Neanderthal genomic DNA. Science  314: 1113–8.  <ul class="find" data-citedArticleID="950864" data-doi="10.1126/science.1131412"><li><a href="http://dx.doi.org/10.1126/science.1131412" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+analysis+of+Neanderthal+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+analysis+of+Neanderthal+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000438-Kanthaswamy1" id="pone.0000438-Kanthaswamy1"></a>Kanthaswamy S, von Dollen A, Kurushima JD, Alminas O, Roger J, et al.  (2006) Microsatellite markers for standardized genetic management of captive colonies of rhesus macaques (Macaca mulatta). Am J Primatol  68: 73–95.  <ul class="find" data-citedArticleID="950860" data-doi="10.1002/ajp.20207"><li><a href="http://dx.doi.org/10.1002/ajp.20207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellite+markers+for+standardized+genetic+management+of+captive+colonies+of+rhesus+macaques+%28Macaca+mulatta%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellite+markers+for+standardized+genetic+management+of+captive+colonies+of+rhesus+macaques+%28Macaca+mulatta%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000438-Smith1" id="pone.0000438-Smith1"></a>Smith DG, McDonough JW, George DA (2007) Mitochondrial DNA Variation Within and Among Regional Populations of Longtail macaques (Macaca fascicularis in Relation to Other Species of the Fascicularis Group of Macaques. Am J Primatol  69: 182–198.  <ul class="find" data-citedArticleID="950872" data-doi="10.1002/ajp.20337"><li><a href="http://dx.doi.org/10.1002/ajp.20337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+Variation+Within+and+Among+Regional+Populations+of+Longtail+macaques+%28Macaca+fascicularis+in+Relation+to+Other+Species+of+the+Fascicularis+Group+of+Macaques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+Variation+Within+and+Among+Regional+Populations+of+Longtail+macaques+%28Macaca+fascicularis+in+Relation+to+Other+Species+of+the+Fascicularis+Group+of+Macaques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000438-Satkoski1" id="pone.0000438-Satkoski1"></a>Satkoski JA, George DA, Smith DG, Kanthaswamy S (in press) Genetic characterization of wild and captive rhesus macaques in China. <span class="underline">J Med Primatol</span>..   <ul class="find" data-citedArticleID="950870" data-doi="10.1111/j.1600-0684.2007.00228.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0684.2007.00228.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+characterization+of+wild+and+captive+rhesus+macaques+in+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+characterization+of+wild+and+captive+rhesus+macaques+in+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000438-Altschul1" id="pone.0000438-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLST: a new generation of protein database search programs. Nucleic Acids Res.  25: 3389–3402.  <ul class="find" data-citedArticleID="950850" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000438-Ronaghi1" id="pone.0000438-Ronaghi1"></a>Ronaghi M, AhlZn M, NyrZn P (1998) A sequencing method based on Real-time pyrophosphate. Science  281: 363–365.  <ul class="find" data-citedArticleID="950868" data-doi="10.1126/science.281.5375.363"><li><a href="http://dx.doi.org/10.1126/science.281.5375.363" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sequencing+method+based+on+Real-time+pyrophosphate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sequencing+method+based+on+Real-time+pyrophosphate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.XML" value="44765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.PDF" value="142921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g001.PNG_L" value="279969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g001.PNG_M" value="20445"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g001.PNG_S" value="1825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g001.TIF" value="736998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g001.PNG_I" value="10940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g002.PNG_L" value="137946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g002.PNG_M" value="8818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g002.PNG_S" value="1519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g002.TIF" value="680032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g002.PNG_I" value="4862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g003.PNG_L" value="122216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g003.PNG_M" value="7688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g003.PNG_S" value="1471"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g003.TIF" value="730554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.g003.PNG_I" value="4395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.t001.PNG_L" value="191493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.t001.PNG_M" value="92317"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.t001.PNG_S" value="8074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.t001.TIF" value="916036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.t001.PNG_I" value="92859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000438.s001.XLS" value="6909440"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000438&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000438" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000438&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000438&volume=&issue=&title=MamuSNP: A Resource for Rhesus Macaque (Macaca mulatta) Genomics&author_name=Ripan%20S.%20Malhi%2C%20Brad%20Sickler%2C%20Dawei%20Lin%2C%20Jessica%20Satkoski%2C%20Raul%20Y.%20Tito%2C%20Debbie%20George%2C%20Sreetharan%20Kanthaswamy%2C%20David%20Glenn%20Smith&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000438" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000438" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000438" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000438&amp;t=MamuSNP%3A%20A%20Resource%20for%20Rhesus%20Macaque%20(Macaca%20mulatta)%20Genomics" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000438&title=MamuSNP%3A%20A%20Resource%20for%20Rhesus%20Macaque%20(Macaca%20mulatta)%20Genomics&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000438&amp;title=MamuSNP%3A%20A%20Resource%20for%20Rhesus%20Macaque%20(Macaca%20mulatta)%20Genomics" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000438" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000438&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'MamuSNP: A Resource for Rhesus Macaque (Macaca mulatta) Genomics';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000438';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20MamuSNP%3A%20A%20Resource%20for%20Rhesus%20Macaque%20(Macaca%20mulatta)%20Genomics http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000438" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000438" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="23714"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23714"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+analysis%22" title="Search for articles in the subject area:'Genome analysis'"><div class="flagText">Genome analysis</div></a>
              <div data-categoryid="40539" data-articleid="23714"
                   data-categoryname="Genome analysis"
                   class="flagImage" title="Flag 'Genome analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23714"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23714"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23714"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="23714"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23714"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7740'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3022'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4480&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
