

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001834"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001834" />

    <meta name="citation_title" content="Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System"/>
    <meta itemprop="name" content="Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System"/>

      <meta name="citation_author" content="Létitia Jean"/>
            <meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Chiu Fan Lee"/>
            <meta name="citation_author_institution" content="Clarendon Laboratory, Department of Physics, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Michael Shaw"/>
            <meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="David J. Vaux"/>
            <meta name="citation_author_institution" content="Sir William Dunn School of Pathology, University of Oxford, Oxford, United Kingdom"/>

    <meta name="citation_date" content="2008/3/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001834.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1834"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Models of amyloid seeding in Alzheimer's disease and scrapie: mechanistic truths and physiological consequences of the time-dependent solubility of amyloid proteins.; citation_author=JD Harper; citation_author=PT Lansbury; citation_journal_title=Annu Rev Biochem; citation_volume=66; citation_number=1; citation_pages=385-407; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Aspects on human amyloid forms and their fibril polypeptides.; citation_author=P Westermark; citation_journal_title=Febs J; citation_volume=272; citation_number=2; citation_pages=5942-5949; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quantitative evaluation of congo red binding to amyloid-like proteins with a beta-pleated sheet conformation.; citation_author=WE Klunk; citation_author=JW Pettegrew; citation_author=DJ Abraham; citation_journal_title=J Histochem Cytochem; citation_volume=37; citation_number=3; citation_pages=1273-1281; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Thioflavine T interaction with synthetic Alzheimer's disease beta-amyloid peptides: detection of amyloid aggregation in solution.; citation_author=H LeVine; citation_journal_title=Protein Sci; citation_volume=2; citation_number=4; citation_pages=404-410; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Amyloid fibrillogenesis: themes and variations.; citation_author=JC Rochet; citation_author=PT Lansbury; citation_journal_title=Curr Opin Struct Biol; citation_volume=10; citation_number=5; citation_pages=60-68; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Review: history of the amyloid fibril.; citation_author=JD Sipe; citation_author=AS Cohen; citation_journal_title=J Struct Biol; citation_volume=130; citation_number=6; citation_pages=88-98; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Guidelines for protein design: the energetics of beta sheet side chain interactions.; citation_author=CK Smith; citation_author=L Regan; citation_journal_title=Science; citation_volume=270; citation_number=7; citation_pages=980-982; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=De novo amyloid proteins from designed combinatorial libraries.; citation_author=MW West; citation_author=W Wang; citation_author=J Patterson; citation_author=JD Mancias; citation_author=JR Beasley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=8; citation_pages=11211-11216; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Dissecting the assembly of Abeta16-22 amyloid peptides into antiparallel beta sheets.; citation_author=DK Klimov; citation_author=D Thirumalai; citation_journal_title=Structure; citation_volume=11; citation_number=9; citation_pages=295-307; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Side chain interactions determine the amyloid organization: a single layer beta-sheet molecular structure of the calcitonin peptide segment 15-19.; citation_author=D Zanuy; citation_author=N Haspel; citation_author=HH Tsai; citation_author=B Ma; citation_author=K Gunasekaran; citation_journal_title=Phys Biol; citation_volume=1; citation_number=10; citation_pages=89-99; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of the minimal amyloid-forming fragment of the islet amyloid polypeptide. An experimental support for the key role of the phenylalanine residue in amyloid formation.; citation_author=R Azriel; citation_author=E Gazit; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=11; citation_pages=34156-34161; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Charge states rather than propensity for beta-structure determine enhanced fibrillogenesis in wild-type Alzheimer's beta-amyloid peptide compared to E22Q Dutch mutant.; citation_author=F Massi; citation_author=D Klimov; citation_author=D Thirumalai; citation_author=JE Straub; citation_journal_title=Protein Sci; citation_volume=11; citation_number=12; citation_pages=1639-1647; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Amyloid fibril formation by pentapeptide and tetrapeptide fragments of human calcitonin.; citation_author=M Reches; citation_author=Y Porat; citation_author=E Gazit; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=13; citation_pages=35475-35480; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Elucidation of primary structure elements controlling early amyloid beta-protein oligomerization.; citation_author=G Bitan; citation_author=SS Vollers; citation_author=DB Teplow; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=14; citation_pages=34882-34889; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular basis for amyloid fibril formation and stability.; citation_author=OS Makin; citation_author=E Atkins; citation_author=P Sikorski; citation_author=J Johansson; citation_author=LC Serpell; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=15; citation_pages=315-320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Soluble protein oligomers in neurodegeneration: lessons from the Alzheimer's amyloid beta-peptide.; citation_author=C Haass; citation_author=DJ Selkoe; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=8; citation_number=16; citation_pages=101-112; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Senile plaque composition and posttranslational modification of amyloid-beta peptide and associated proteins.; citation_author=CS Atwood; citation_author=RN Martins; citation_author=MA Smith; citation_author=G Perry; citation_journal_title=Peptides; citation_volume=23; citation_number=17; citation_pages=1343-1350; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular isoform distribution and glycosylation of acetylcholinesterase are altered in brain and cerebrospinal fluid of patients with Alzheimer's disease.; citation_author=J Saez-Valero; citation_author=G Sberna; citation_author=CA McLean; citation_author=DH Small; citation_journal_title=J Neurochem; citation_volume=72; citation_number=18; citation_pages=1600-1608; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Memory deficits correlating with acetylcholinesterase splice shift and amyloid burden in doubly transgenic mice.; citation_author=TM Rees; citation_author=A Berson; citation_author=EH Sklan; citation_author=L Younkin; citation_author=S Younkin; citation_journal_title=Curr Alzheimer Res; citation_volume=2; citation_number=19; citation_pages=291-300; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Butyrylcholinesterase attenuates amyloid fibril formation in vitro.; citation_author=S Diamant; citation_author=E Podoly; citation_author=A Friedler; citation_author=H Ligumsky; citation_author=O Livnah; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=20; citation_pages=8628-8633; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Amyloid fibril formation by a synthetic peptide from a region of human acetylcholinesterase that is homologous to the Alzheimer's amyloid-beta peptide.; citation_author=MG Cottingham; citation_author=MS Hollinshead; citation_author=DJ Vaux; citation_journal_title=Biochemistry; citation_volume=41; citation_number=21; citation_pages=13539-13547; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detecting hidden sequence propensity for amyloid fibril formation.; citation_author=S Yoon; citation_author=WJ Welsh; citation_journal_title=Protein Sci; citation_volume=13; citation_number=22; citation_pages=2149-2160; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The intact human acetylcholinesterase C-terminal oligomerization domain is alpha-helical in situ and in isolation, but a shorter fragment forms beta-sheet-rich amyloid fibrils and protofibrillar oligomers.; citation_author=MG Cottingham; citation_author=JL Voskuil; citation_author=DJ Vaux; citation_journal_title=Biochemistry; citation_volume=42; citation_number=23; citation_pages=10863-10873; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Heterologous amyloid seeding: revisiting the role of acetylcholinesterase in Alzheimer's disease.; citation_author=L Jean; citation_author=B Thomas; citation_author=A Tahiri-Alaoui; citation_author=M Shaw; citation_author=DJ Vaux; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=24; citation_pages=e652; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The organization of aromatic side groups in an amyloid fibril probed by solid-state 2H and 19F NMR spectroscopy.; citation_author=E Jack; citation_author=M Newsome; citation_author=PG Stockley; citation_author=SE Radford; citation_author=DA Middleton; citation_journal_title=J Am Chem Soc; citation_volume=128; citation_number=25; citation_pages=8098-8099; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Arrest of beta-amyloid fibril formation by a pentapeptide ligand.; citation_author=LO Tjernberg; citation_author=J Naslund; citation_author=F Lindqvist; citation_author=J Johansson; citation_author=AR Karlstrom; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=26; citation_pages=8545-8548; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Controlling polymerization of beta-amyloid and prion-derived peptides with synthetic small molecule ligands.; citation_author=P Kuner; citation_author=B Bohrmann; citation_author=LO Tjernberg; citation_author=J Naslund; citation_author=G Huber; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=27; citation_pages=1673-1678; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Surfactant properties of Alzheimer's A beta peptides and the mechanism of amyloid aggregation.; citation_author=B Soreghan; citation_author=J Kosmoski; citation_author=C Glabe; citation_journal_title=J Biol Chem; citation_volume=269; citation_number=28; citation_pages=28551-28554; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Rapid method for measurement of surface tension in multiwell plates.; citation_author=MG Cottingham; citation_author=CD Bain; citation_author=DJ Vaux; citation_journal_title=Lab Invest; citation_volume=84; citation_number=29; citation_pages=523-529; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chemistry for the analysis of protein-protein interactions: rapid and efficient cross-linking triggered by long wavelength light.; citation_author=DA Fancy; citation_author=T Kodadek; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=30; citation_pages=6020-6024; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The toxicity of the Alzheimer's beta-amyloid peptide correlates with a distinct fiber morphology.; citation_author=B Seilheimer; citation_author=B Bohrmann; citation_author=L Bondolfi; citation_author=F Muller; citation_author=D Stuber; citation_journal_title=J Struct Biol; citation_volume=119; citation_number=31; citation_pages=59-71; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Alpha-synuclein, especially the Parkinson's disease-associated mutants, forms pore-like annular and tubular protofibrils.; citation_author=HA Lashuel; citation_author=BM Petre; citation_author=J Wall; citation_author=M Simon; citation_author=RJ Nowak; citation_journal_title=J Mol Biol; citation_volume=322; citation_number=32; citation_pages=1089-1102; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sequence determinants of amyloid fibril formation.; citation_author=M Lopez de la Paz; citation_author=L Serrano; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=33; citation_pages=87-92; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sequence dependence of amyloid fibril formation: insights from molecular dynamics simulations.; citation_author=M Lopez de la Paz; citation_author=GM de Mori; citation_author=L Serrano; citation_author=G Colombo; citation_journal_title=J Mol Biol; citation_volume=349; citation_number=34; citation_pages=583-596; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A structural model for Alzheimer's beta -amyloid fibrils based on experimental constraints from solid state NMR.; citation_author=AT Petkova; citation_author=Y Ishii; citation_author=JJ Balbach; citation_author=ON Antzutkin; citation_author=RD Leapman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=35; citation_pages=16742-16747; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Charge attraction and beta propensity are necessary for amyloid fibril formation from tetrapeptides.; citation_author=L Tjernberg; citation_author=W Hosia; citation_author=N Bark; citation_author=J Thyberg; citation_author=J Johansson; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=36; citation_pages=43243-43246; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Probing the initial stage of aggregation of the Abeta(10-35)-protein: assessing the propensity for peptide dimerization.; citation_author=B Tarus; citation_author=JE Straub; citation_author=D Thirumalai; citation_journal_title=J Mol Biol; citation_volume=345; citation_number=37; citation_pages=1141-1156; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Spontaneous assembly of a self-complementary oligopeptide to form a stable macroscopic membrane.; citation_author=S Zhang; citation_author=T Holmes; citation_author=C Lockshin; citation_author=A Rich; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=38; citation_pages=3334-3338; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=An analysis of side chain interactions and pair correlations within antiparallel beta-sheets: the differences between backbone hydrogen-bonded and non-hydrogen-bonded residue pairs.; citation_author=MA Wouters; citation_author=PM Curmi; citation_journal_title=Proteins; citation_volume=22; citation_number=39; citation_pages=119-131; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The ‘Arctic’ APP mutation (E693G) causes Alzheimer's disease by enhanced Abeta protofibril formation.; citation_author=C Nilsberth; citation_author=A Westlind-Danielsson; citation_author=CB Eckman; citation_author=MM Condron; citation_author=K Axelman; citation_journal_title=Nat Neurosci; citation_volume=4; citation_number=40; citation_pages=887-893; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of a penta- and hexapeptide of islet amyloid polypeptide (IAPP) with amyloidogenic and cytotoxic properties.; citation_author=K Tenidis; citation_author=M Waldner; citation_author=J Bernhagen; citation_author=W Fischle; citation_author=M Bergmann; citation_journal_title=J Mol Biol; citation_volume=295; citation_number=41; citation_pages=1055-1071; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rationalization of the effects of mutations on peptide and protein aggregation rates.; citation_author=F Chiti; citation_author=M Stefani; citation_author=N Taddei; citation_author=G Ramponi; citation_author=CM Dobson; citation_journal_title=Nature; citation_volume=424; citation_number=42; citation_pages=805-808; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A possible role for pi-stacking in the self-assembly of amyloid fibrils.; citation_author=E Gazit; citation_journal_title=Faseb J; citation_volume=16; citation_number=43; citation_pages=77-83; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=p-p interactions in self-assembly.; citation_author=CG Claessens; citation_author=JF Stoddart; citation_journal_title=J Phys Org Chem; citation_volume=10; citation_number=44; citation_pages=254-272; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Controlling self-assembly.; citation_author=RE Gillard; citation_author=FM Raymo; citation_author=JF Stoddart; citation_journal_title=Chem Eur J; citation_volume=3; citation_number=45; citation_pages=1933-1940; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Aromatic interactions in model systems.; citation_author=ML Waters; citation_journal_title=Curr Opin Chem Biol; citation_volume=6; citation_number=46; citation_pages=736-741; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The role of Phe in the formation of well-ordered oligomers of amyloidogenic hexapeptide (NFGAIL) observed in molecular dynamics simulations with explicit solvent.; citation_author=C Wu; citation_author=H Lei; citation_author=Y Duan; citation_journal_title=Biophys J; citation_volume=88; citation_number=47; citation_pages=2897-2906; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Assessing the role of aromatic residues in the amyloid aggregation of human muscle acylphosphatase.; citation_author=F Bemporad; citation_author=N Taddei; citation_author=M Stefani; citation_author=F Chiti; citation_journal_title=Protein Sci; citation_volume=15; citation_number=48; citation_pages=862-870; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structure and texture of fibrous crystals formed by Alzheimer's abeta(11-25) peptide fragment.; citation_author=P Sikorski; citation_author=ED Atkins; citation_author=LC Serpell; citation_journal_title=Structure; citation_volume=11; citation_number=49; citation_pages=915-926; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural diversity of amyloid fibril formed in human calcitonin as revealed by site-directed 13C solid-state NMR spectroscopy.; citation_author=A Naito; citation_author=M Kamihira; citation_author=R Inoue; citation_author=H Saito; citation_journal_title=Magn Reson Chem; citation_volume=42; citation_number=50; citation_pages=247-257; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Aromatic -p Stacking in Solution as Revealed through the Aggregation of Phenylacetylene Macrocycles.; citation_author=AS Shetty; citation_author=J Zhang; citation_author=JS Moore; citation_journal_title=J Am Chem Soc; citation_volume=118; citation_number=51; citation_pages=1019-1027; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=pi-Stacking interactions. Alive and well in proteins.; citation_author=GB McGaughey; citation_author=M Gagne; citation_author=AK Rappe; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=52; citation_pages=15458-15463; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=On the role of surface tension in the stabilization of globular proteins.; citation_author=TY Lin; citation_author=SN Timasheff; citation_journal_title=Protein Sci; citation_volume=5; citation_number=53; citation_pages=372-381; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cation-pi interactions in protein-protein interfaces.; citation_author=PB Crowley; citation_author=A Golovin; citation_journal_title=Proteins; citation_volume=59; citation_number=54; citation_pages=231-239; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Interactions between cytochrome c2 and the photosynthetic reaction center from Rhodobacter sphaeroides: the cation-pi interaction.; citation_author=ML Paddock; citation_author=KH Weber; citation_author=C Chang; citation_author=MY Okamura; citation_journal_title=Biochemistry; citation_volume=44; citation_number=55; citation_pages=9619-9625; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Cationminus signpi Interaction.; citation_author=JC Ma; citation_author=DA Dougherty; citation_journal_title=Chem Rev; citation_volume=97; citation_number=56; citation_pages=1303-1324; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Analysis of cation π interactions to the structural stability of RNA binding proteins.; citation_author=S Chakkaravarthi; citation_author=MM Gromiha; citation_journal_title=Polymer; citation_volume=47; citation_number=57; citation_pages=709-721; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=NMR studies of model peptides of PHGGGWGQ repeats within the N-terminus of prion proteins: a loop conformation with histidine and tryptophan in close proximity.; citation_author=H Yoshida; citation_author=N Matsushima; citation_author=Y Kumaki; citation_author=M Nakata; citation_author=K Hikichi; citation_journal_title=J Biochem (Tokyo); citation_volume=128; citation_number=58; citation_pages=271-281; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The octapeptide repeats in mammalian prion protein constitute a pH-dependent folding and aggregation site.; citation_author=R Zahn; citation_journal_title=J Mol Biol; citation_volume=334; citation_number=59; citation_pages=477-488; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Non-hydrogen bond interactions involving the methionine sulfur atom.; citation_author=D Pal; citation_author=P Chakrabarti; citation_journal_title=J Biomol Struct Dyn; citation_volume=19; citation_number=60; citation_pages=115-128; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Environment of tryptophan side chains in proteins.; citation_author=U Samanta; citation_author=D Pal; citation_author=P Chakrabarti; citation_journal_title=Proteins; citation_volume=38; citation_number=61; citation_pages=288-300; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Experimental constraints on quaternary structure in Alzheimer's beta-amyloid fibrils.; citation_author=AT Petkova; citation_author=WM Yau; citation_author=R Tycko; citation_journal_title=Biochemistry; citation_volume=45; citation_number=62; citation_pages=498-512; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein stabilization by engineered metal chelation.; citation_author=JT Kellis; citation_author=RJ Todd; citation_author=FH Arnold; citation_journal_title=Biotechnology (N Y); citation_volume=9; citation_number=63; citation_pages=994-995; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Dramatic aggregation of Alzheimer abeta by Cu(II) is induced by conditions representing physiological acidosis.; citation_author=CS Atwood; citation_author=RD Moir; citation_author=X Huang; citation_author=RC Scarpa; citation_author=NM Bacarra; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=64; citation_pages=12817-12826; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Histidine-13 is a crucial residue in the zinc ion-induced aggregation of the A beta peptide of Alzheimer's disease.; citation_author=ST Liu; citation_author=G Howlett; citation_author=CJ Barrow; citation_journal_title=Biochemistry; citation_volume=38; citation_number=65; citation_pages=9373-9378; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Metal binding modes of Alzheimer's amyloid beta-peptide in insoluble aggregates and soluble complexes.; citation_author=T Miura; citation_author=K Suzuki; citation_author=N Kohata; citation_author=H Takeuchi; citation_journal_title=Biochemistry; citation_volume=39; citation_number=66; citation_pages=7024-7031; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Examining the zinc binding site of the amyloid-beta peptide.; citation_author=DS Yang; citation_author=J McLaurin; citation_author=K Qin; citation_author=D Westaway; citation_author=PE Fraser; citation_journal_title=Eur J Biochem; citation_volume=267; citation_number=67; citation_pages=6692-6698; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Preferential Cu2+ coordination by His96 and His111 induces beta-sheet formation in the unstructured amyloidogenic region of the prion protein.; citation_author=CE Jones; citation_author=SR Abdelraheim; citation_author=DR Brown; citation_author=JH Viles; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=68; citation_pages=32018-32027; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Conformational and geometrical properties of beta-sheets in proteins. I. Parallel beta-sheets.; citation_author=FR Salemme; citation_author=DW Weatherford; citation_journal_title=J Mol Biol; citation_volume=146; citation_number=69; citation_pages=101-117; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Atomic structures of amyloid cross-beta spines reveal varied steric zippers.; citation_author=MR Sawaya; citation_author=S Sambashivan; citation_author=R Nelson; citation_author=MI Ivanova; citation_author=SA Sievers; citation_journal_title=Nature; citation_volume=447; citation_number=70; citation_pages=453-457; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Amyloid fibril formation by an SH3 domain.; citation_author=JI Guijarro; citation_author=M Sunde; citation_author=JA Jones; citation_author=ID Campbell; citation_author=CM Dobson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=71; citation_pages=4224-4228; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Amyloid fibrils from muscle myoglobin.; citation_author=M Fandrich; citation_author=MA Fletcher; citation_author=CM Dobson; citation_journal_title=Nature; citation_volume=410; citation_number=72; citation_pages=165-166; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Protein aggregation and aggregate toxicity: new insights into protein folding, misfolding diseases and biological evolution.; citation_author=M Stefani; citation_author=CM Dobson; citation_journal_title=J Mol Med; citation_volume=81; citation_number=73; citation_pages=678-699; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Conformational constraints for amyloid fibrillation: the importance of being unfolded.; citation_author=VN Uversky; citation_author=AL Fink; citation_journal_title=Biochim Biophys Acta; citation_volume=1698; citation_number=74; citation_pages=131-153; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling.; citation_author=N Guex; citation_author=MC Peitsch; citation_journal_title=Electrophoresis; citation_volume=18; citation_number=75; citation_pages=2714-2723; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001834" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System"/>
    <meta name="twitter:description" content="Polymerization into amyloid fibrils is a crucial step in the pathogenesis of neurodegenerative syndromes. Amyloid assembly is governed by properties of the sequence backbone and specific side-chain interactions, since fibrils from unrelated sequences possess similar structures and morphologies. Therefore, characterization of the structural determinants driving amyloid aggregation is of fundamental importance. We investigated the forces involved in the amyloid assembly of a model peptide derived from the oligomerization domain of acetylcholinesterase (AChE), AChE586-599, through the effect of single point mutations on &beta;-sheet propensity, conformation, fibrilization, surfactant activity, oligomerization and fibril morphology. AChE586-599 was chosen due to its fibrilization tractability and AChE involvement in Alzheimer's disease. The results revealed how specific regions and residues can control AChE586-599 assembly. Hydrophobic and/or aromatic residues were crucial for maintaining a high &beta;-strand propensity, for the conformational transition to &beta;-sheet, and for the first stage of aggregation. We also demonstrated that positively charged side-chains might be involved in electrostatic interactions, which could control the transition to &beta;-sheet, the oligomerization and assembly stability. Further interactions were also found to participate in the assembly. We showed that some residues were important for AChE586-599 surfactant activity and that amyloid assembly might preferentially occur at an air-water interface. Consistently with the experimental observations and assembly models for other amyloid systems, we propose a model for AChE586-599 assembly in which a steric-zipper formed through specific interactions (hydrophobic, electrostatic, cation-&pi;, SH-aromatic, metal chelation and polar-polar) would maintain the &beta;-sheets together. We also propose that the stacking between the strands in the &beta;-sheets along the fiber axis could be stabilized through &pi;-&pi; interactions and metal chelation. The dissection of the specific molecular recognition driving AChE586-599 amyloid assembly has provided further knowledge on such poorly understood and complicated process, which could be applied to protein folding and the targeting of amyloid diseases."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001834.g011"/>

  <meta property="og:title" content="Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001834" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7308'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9798'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1630&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001834">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001834" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001834&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Létitia Jean, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Sir William Dunn School of Pathology, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chiu Fan Lee, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Clarendon Laboratory, Department of Physics, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Shaw, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Sir William Dunn School of Pathology, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David J. Vaux
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:david.vaux@path.ox.ac.uk">david.vaux@path.ox.ac.uk</a></p>

                <p>Affiliation:
                  Sir William Dunn School of Pathology, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 19, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001834</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001834" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001834" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001834" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001834" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001834-g001" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g002" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g003" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g004" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g005" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g006" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g007" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g008" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g009" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g010" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001834-g011" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001834">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001834" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Polymerization into amyloid fibrils is a crucial step in the pathogenesis of neurodegenerative syndromes. Amyloid assembly is governed by properties of the sequence backbone and specific side-chain interactions, since fibrils from unrelated sequences possess similar structures and morphologies. Therefore, characterization of the structural determinants driving amyloid aggregation is of fundamental importance. We investigated the forces involved in the amyloid assembly of a model peptide derived from the oligomerization domain of acetylcholinesterase (AChE), AChE<sub>586-599</sub>, through the effect of single point mutations on β-sheet propensity, conformation, fibrilization, surfactant activity, oligomerization and fibril morphology. AChE<sub>586-599</sub> was chosen due to its fibrilization tractability and AChE involvement in Alzheimer's disease. The results revealed how specific regions and residues can control AChE<sub>586-599</sub> assembly. Hydrophobic and/or aromatic residues were crucial for maintaining a high β-strand propensity, for the conformational transition to β-sheet, and for the first stage of aggregation. We also demonstrated that positively charged side-chains might be involved in electrostatic interactions, which could control the transition to β-sheet, the oligomerization and assembly stability. Further interactions were also found to participate in the assembly. We showed that some residues were important for AChE<sub>586-599</sub> surfactant activity and that amyloid assembly might preferentially occur at an air-water interface. Consistently with the experimental observations and assembly models for other amyloid systems, we propose a model for AChE<sub>586-599</sub> assembly in which a steric-zipper formed through specific interactions (hydrophobic, electrostatic, cation-π, SH-aromatic, metal chelation and polar-polar) would maintain the β-sheets together. We also propose that the stacking between the strands in the β-sheets along the fiber axis could be stabilized through π-π interactions and metal chelation. The dissection of the specific molecular recognition driving AChE<sub>586-599</sub> amyloid assembly has provided further knowledge on such poorly understood and complicated process, which could be applied to protein folding and the targeting of amyloid diseases.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Jean L, Lee CF, Shaw M, Vaux DJ (2008) Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System. PLoS ONE 3(3):
          e1834.
            doi:10.1371/journal.pone.0001834</p><p><strong>Academic Editor: </strong>Suzannah Rutherford, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received:</strong> December 18, 2007; <strong>Accepted:</strong> February 19, 2008; <strong>Published:</strong> March 19, 2008</p><p><strong>Copyright:</strong> © 2008 Jean et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a research grant from Synaptica Ltd.</p><p><strong>Competing interests:</strong> Synaptica Ltd holds patents on the use of the AChE586-599 peptide and related peptides within T40 as potential biomarkers for neurodegenerative disease. The University of Oxford holds patents on the method of plate-based surface tension measurement.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Protein misfolding can be deleterious by triggering aggregation and insolubilization. In turn, the aggregation can lead to toxic conformation during which polymerization by folding and stacking of cross-β sheets result in the formation of amyloid fibrils. Fibril formation is a multiple kinetic event during which an energetically unfavourable nucleated polymerization (characterized by a lag phase) initiates the formation of a minimal self-assembled complex (nucleus or seed) serving as a structural template for a cooperative amyloid elongation <a href="#pone.0001834-Harper1">[1]</a>. Amyloid fibrilization is proposed to be the molecular basis of and the common link between a variety of pathological conditions and human neurodegenerative syndromes, such as type II diabetes, Alzheimer's, Parkinson's, Huntington's and prion diseases <a href="#pone.0001834-Westermark1">[2]</a>. Although amyloid formation and deposition is a common feature of these diseases, the amyloid fibrils originate from different and distinct proteins or peptides that do not appear to share any sequence homology or function. However, fibrils formed from these amyloid-related sequences possess similar structural, physical and chemical properties, including formation of β-sheets whose strands run perpendicular to the fibril axis, fibril morphology and typical X-ray diffraction pattern, kinetic pattern of fibril formation and staining with dyes such as Congo red and Thioflavin T (ThT) <a href="#pone.0001834-Klunk1">[3]</a>, <a href="#pone.0001834-LeVine1">[4]</a>, <a href="#pone.0001834-Harper1">[1]</a>, <a href="#pone.0001834-Rochet1">[5]</a>, <a href="#pone.0001834-Sipe1">[6]</a>. Therefore, amyloid formation involves more than non-specific aggregation and non-specific hydrophobic interactions and it is recognized that some levels of structural complexity and specific pattern of interactions are important <a href="#pone.0001834-Smith1">[7]</a>, <a href="#pone.0001834-West1">[8]</a>, <a href="#pone.0001834-Klimov1">[9]</a>, <a href="#pone.0001834-Zanuy1">[10]</a>. Indeed, certain types of residues characterized by high β-sheet propensity, and/or fastest kinetics of aggregation, and/or stabilizing and assembling properties have been found to be commonly present in amyloid-related sequences <a href="#pone.0001834-Azriel1">[11]</a>, <a href="#pone.0001834-Massi1">[12]</a>, <a href="#pone.0001834-Reches1">[13]</a>, <a href="#pone.0001834-Bitan1">[14]</a>, <a href="#pone.0001834-Makin1">[15]</a>. Consequently in recent years, a great deal of attention has focussed on determining the factors and interactions resulting in fibrilization and finding common rules that govern the assembly. Such detailed dissections of the specific molecular recognition and self-assembly during amyloid formation could provide invaluable knowledge for the targeting and control of diseases involving toxic protein aggregation and deposition.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>During Alzheimer's disease (AD) pathogenesis, the accumulation in the brain of extracellular amyloid-β-peptide (Aβ) in senile plaques and of intracellular hyperphosphorylated Tau in neurofibrillary tangles are thought to represent the hallmarks of the disease <a href="#pone.0001834-Haass1">[16]</a>. However, other proteins have also been implicated in the pathology, with one example being acetylcholinesterase (AChE) <a href="#pone.0001834-Atwood1">[17]</a>. AChE is associated with senile plaques, promotes Aβ fibrilisation, and triggers early disease and increases plaque burden in double transgenic mice expressing human amyloid precursor protein (hAPP, from which Aβ is proteolytically cleaved) and hAChE when compared to single transgenic hAPP mice <a href="#pone.0001834-SaezValero1">[18]</a>, <a href="#pone.0001834-Rees1">[19]</a>, <a href="#pone.0001834-Diamant1">[20]</a>. We have studied a 14 residue peptide named AChE<sub>586-599</sub>, which corresponds to a region within the C-terminal oligomerization domain of human AChE. The region encompassing AChE<sub>586-599</sub> shares homology with Aβ and possesses high propensity for conversion to non-native (hidden) β-strand, a property associated with amyloidogenicity <a href="#pone.0001834-Cottingham1">[21]</a>, <a href="#pone.0001834-Yoon1">[22]</a>. Moreover, AChE<sub>586-599</sub> adopts a β-sheet conformation, self-assembles into amyloid fibrils and promotes Aβ fibrilization <a href="#pone.0001834-Cottingham2">[23]</a>, <a href="#pone.0001834-Jean1">[24]</a>. This peptide represents a tractable model for studying amyloid formation because its fibrilization is highly dependent upon pH. This allows a total control of the start of the polymerization process, which is triggered by the addition of physiological buffer to an acid solution. Moreover, AChE<sub>586-599</sub> is an attractive model due to its residue composition with alternating charged, polar and hydrophobic amino acids most of which have been previously shown to be implicated in amyloid formation <a href="#pone.0001834-Azriel1">[11]</a>, <a href="#pone.0001834-Massi1">[12]</a>, <a href="#pone.0001834-Makin1">[15]</a>, <a href="#pone.0001834-Jack1">[25]</a>. Understanding how AChE<sub>586-599</sub> residue composition and chemical nature affect the polymerization process should provide insights and strengthen current knowledge into complex networks of interactions leading to amyloid fibril formation.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this study, we examined the role of each residue within AChE<sub>586-599</sub> through the effect of single point mutations on the β-sheet propensity, conformation, fibrilization, surfactant activity, oligomer formation and fibril morphology of AChE<sub>586-599</sub>. We determined the importance of residues and the potential molecular interactions underlying AChE<sub>586-599</sub> assembly into β-sheets and during the stacking of these sheets. Non-covalent side chain-side chain interactions, such as hydrophobic, cation-π and π-π interactions, were found to be critical for fibrilization and assembly stabilization. This detailed analysis allowed us to propose a model for the amyloid polymerization of AChE<sub>586-599</sub> in which specific interactions between residue side-chains lead to the formation of a steric-zipper maintaining the β-sheets together, and π-π interactions allow the stacking and arrangement of strands within a β-sheet.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>To determine the role of each residue of AChE<sub>586-599</sub> in the process of amyloid formation, a library of alanine scanning mutants along with a structurally conserved substitution mutant (Tyr to Phe) and a truncation mutant (missing the last residue) were used.</p>

<h4>Identification of the residues important for the β-sheet propensity of AChE<sub>586-599</sub> and its conformational transition from random coil to β-sheet upon neutralization</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We performed secondary structure prediction in term of high propensity for conversion to non-native (hidden) β-strand, using the method described by Yoon and Welsh <a href="#pone.0001834-Yoon1">[22]</a>. Previously, Yoon and Welsh have predicted the minimal amyloidogenic regions for Aβ and α-synuclein, and have also identified AChE<sub>586-599</sub> to be a region of AChE with high non-native (hidden) β-strand propensity <a href="#pone.0001834-Yoon1">[22]</a>. Since the amyloidogenicity of a peptide has been associated with its β-sheet forming propensity, such analysis could provide an insight on the importance of certain residues in the fibrilogenicity of AChE<sub>586-599</sub>. When we applied the algorithm to AChE<sub>586-599</sub>, the whole peptide (with the exceptions of the N-terminal Ala and Glu, and the C-terminal Lys) possessed propensity for conversion to β-strand with the strongest propensity for the sequence YMVH (<a href="#pone-0001834-g001">Figure 1</a>). Some mutations (E<sub>2</sub> and R<sub>5</sub>) increased the β-strand propensity. The H<sub>12</sub>/A mutant strengthened the propensity for V<sub>11</sub> albeit decreasing it slightly for Y<sub>9</sub> and M<sub>10</sub>. By contrast, some mutations drastically impaired the β-strand propensity in some part of the sequence: W<sub>6</sub>, Y<sub>9</sub>, M<sub>10</sub>, V<sub>11</sub> and W<sub>13</sub>. The W<sub>6</sub>/A mutant created a break in the continuity of β-strand propensity, whereas the Y<sub>9</sub>/A, M<sub>10</sub>/A, V<sub>11</sub>/A and W<sub>13</sub>/A mutants drastically decreased it in the YMVH region. Four other mutations also impaired the β-strand propensity but moderately: F<sub>3</sub> triggering a stronger random coil propensity for A<sub>1</sub>; S<sub>7</sub>, S<sub>8</sub> and K<sub>14</sub> slightly decreasing the β-strand propensity in the YMVH region. Only the mutation to Ala did not affect the β-strand propensity. Thus, W<sub>6</sub>, Y<sub>9</sub>, M<sub>10</sub>, V<sub>11</sub> and W<sub>13</sub> appeared to be crucial for maintaining a high β-strand propensity along the entire sequence.</p>
<div class="figure" id="pone-0001834-g001"><div class="img"><a name="pone-0001834-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Secondary structure propensity of AChE<sub>586-599</sub> mutants as predicted by hidden β-propensity method (available at <a href="http://opal.umdnj.edu">http://opal.umdnj.edu</a>).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Propensities for helices (red squares), β-strands (blue squares) and random coil (green squares) are presented numerically using a 0-1 scale, with low values indicating zero to low propensity and high values indicating high propensity to near certainty. Hydrophobic residues are shown in green and aromatic residues in bold with a bigger font size.</p>
<span>doi:10.1371/journal.pone.0001834.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Far-UV circular dichroism (CD) studies were performed to establish the conformation of AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants before and after neutralization, allowing us to follow early conformational changes that preceded and occurred during aggregation. AChE<sub>586-599</sub> was previously shown to be random coil when non-aggregated and to switch to a β-sheet structure upon neutralization <a href="#pone.0001834-Cottingham2">[23]</a>. Representatives of the conformations and changes in conformation observed are presented in <a href="#pone-0001834-g002">Figure 2A</a> and the conformations found for all the peptides at the different pHs are summarized in <a href="#pone-0001834-g002">Figure 2B</a>. CD spectra typical for a random coil structure (negative molar ellipticity at 200 nm or below) were observed for all peptides at acidic pH, indicating their non-aggregated status under acidic conditions. Nine mutant peptides still displayed random coil spectra after neutralization (e.g. W<sub>6</sub>/A, top left panel in <a href="#pone-0001834-g002">Figure 2A</a>) (<a href="#pone-0001834-g002">Figure 2B</a>). Therefore, these mutants were not able to adopt a β-sheet conformation under the conditions of the assay, which suggested the importance of the hydrophobicity and/or aromaticity of F<sub>3</sub>, W<sub>6</sub>, S<sub>7</sub>, S<sub>8</sub>, Y<sub>9</sub>, M<sub>10</sub>, V<sub>11</sub> and W<sub>13</sub> in the conformational transition upon neutralization. By contrast, negative molar ellipticity around 215 nm was found at neutral pH for 6 peptides: AChE<sub>586-599</sub>, E<sub>2</sub>/A, H<sub>4</sub>/A, R<sub>5</sub>/A, K<sub>14</sub>/A and ΔK<sub>14</sub> mutants (<a href="#pone-0001834-g002">Figure 2A</a>, bottom panels, and <a href="#pone-0001834-g002">Figure 2B</a>). Such a negative ellipticity is typically assigned to β-sheet structures. Some peptides started to adopt a partial (e.g. ΔK<sub>14</sub> mutant, <a href="#pone-0001834-g002">Figure 2A</a>, top right panel with double negative ellipticities at 200 and 215 nm) or a complete β-sheet structure (e.g. R<sub>5</sub>/A mutant, <a href="#pone-0001834-g002">Figure 2A</a> bottom right panel) after only 10 min at neutral pH. These results indicated that such mutants were faster than AChE<sub>586-599</sub> at adopting a β-sheet conformation, and therefore that the Arg and the Lys residues are not crucially involved in the conformational change during neutralization and may even have a negative effect. After 24 hours at neutral pH, visual inspection of the solutions for the peptides adopting a β-sheet structure revealed the presence of insoluble aggregates, indicating tertiary or quaternary arrangements possibly leading to the formation of intermolecular-stacked β-sheets (see below).</p>
<div class="figure" id="pone-0001834-g002"><div class="img"><a name="pone-0001834-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Conformation of AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>(A) Far UV spectra (250 to 190 nm) before and after pH neutralization (50 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.2) of 4 AChE<sub>586-599</sub> mutants (100 µM). These spectra are representatives of the different structures and different changes in structure observed for the wild-type and mutant peptides. (B) Conformation and changes in conformation after pH neutralization (50 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.2) for AChE<sub>586-599</sub> and all AChE<sub>586-599</sub> mutants (100 µM). (C) Near UV spectra (320 to 240 nm) before and after pH neutralization (50 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.2) of AChE<sub>586-599</sub> and 4 mutants (100 µM). In all panels, the mutation within AChE<sub>586-599</sub> is indicated in bold and italics.</p>
<span>doi:10.1371/journal.pone.0001834.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We then applied near-UV CD to follow the behavior of the aromatic side-chains during aggregation. AChE<sub>586-599</sub> does not possess Cys residues, therefore any bands in the near-UV spectrum can only be attributed to constraints on the side-chains of aromatic residues. Before neutralization, the AChE<sub>586-599</sub> spectrum did not exhibit any strong positive bands, indicating the absence of conformational restriction of the aromatic side-chains (<a href="#pone-0001834-g002">Figure 2C</a>, left panel). By contrast, the near-UV spectrum of AChE<sub>586-599</sub> after 24 hours at neutral pH showed three positive bands (below 260 nm, at 287 nm and 297 nm), which were consistent with conformational constraints on the aromatic rings of Phe and Tyr and/or Trp. By contrast the spectra of mutants, in which a single aromatic residue was substituted to Ala (the F<sub>3</sub>/A, W<sub>6</sub>/A, Y<sub>9</sub>/A and W<sub>13</sub>/A mutants), did not display any positive bands (<a href="#pone-0001834-g002">Figure 2C</a>, right panel). Therefore, the formation of the insoluble aggregates of AChE<sub>586-599</sub> was associated with tertiary or quaternary interactions involving aromatic residues.</p>


<h4>Identification of the residues important for the fibrilization properties of AChE<sub>586-599</sub></h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The ability to form amyloid and the fibrilization kinetics of AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants were determined by changes in ThT fluorescence emission in shaking conditions. Shaking was used to accelerate fibrilization, which was necessary at least for some mutants. After a lag phase of 0.09 hours, AChE<sub>586-599</sub> rapidly self-assembled into amyloid aggregates (<a href="#pone-0001834-g003">Figure 3A</a>). It has to be noted that the kinetics of AChE<sub>586-599</sub> assembly involved 2 phases, with a first rapid assembly (&lt;1 hour) followed by a short plateau and decrease of ThT signal, and a second assembly with a slower rate (between 4.6 and 12 hours) also followed by a short plateau and decrease of ThT signal (<a href="#pone-0001834-g003">Figure 3</a> A). However, AChE<sub>586-599</sub> assembly performed in quiescent conditions did not display this biphasic behavior (data not shown). Instead, it revealed that the assembly started immediately, without the first assembly, but instead resembling the second assembly observed during the shaking conditions. This assembly in quiescent conditions was also followed by a short plateau and a decrease of ThT signal identical to the ones observed during shaking conditions. Therefore, AChE<sub>586-599</sub> biphasic behavior was triggered by the shaking environment, which might either affect the assembly susceptibility to breakage by increasing shear forces, or affect the peptide surfactant activity by increasing the surface area and peptide recruitment. Collectively, the decay of the ThT signal after plateau in both conditions (shaking or not) suggested that the assembly of AChE<sub>586-599</sub> was not stable.</p>
<div class="figure" id="pone-0001834-g003"><div class="img"><a name="pone-0001834-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Fibrilization properties of AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) 100 µM peptide was incubated with 165 µM ThT in PBS. Changes in ThT fluorescence were monitored (A, with the right panel showing scale-up of the left panel to visualize the rapid fibrilization of some peptides) with the lag phase of fibrilization (B, left panel) and plateau height (B, right panel) depicted. A black star signifies p&lt;0.003 (B, left panel) and p&lt;0.03 (B, right panel) when compared to the wild type peptide (WT). The peptides shown are representatives of the different fibrilization properties observed. (C) Fibrilization properties for AChE<sub>586-599</sub> and all AChE<sub>586-599</sub> mutants (100 µM). The properties are divided into 4 categories: ability to fibrilize, duration of lag phase, height of plateau and stability of the amyloid products (indicated by stability or decay of the ThT fluorescence after plateau). The mutation within AChE<sub>586-599</sub> is indicated in bold and italics. The peptides that do not fibrilize and/or the peptides which amyloid products are not stable are indicated by grey boxes. The lag phase and plateau height for the mutant peptides are shown as fold ratio of AChE<sub>586-599</sub> (e.g. ‘1’ represents equal value to AChE<sub>586-599</sub> and ‘100’ for the lag phase represents 100 times longer than AChE<sub>586-599</sub>). ‘N.D.’ means ‘not detectable’.</p>
<span>doi:10.1371/journal.pone.0001834.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Most of the substitutions affected both the lag phase and plateau height (<a href="#pone-0001834-g003">Figure 3A, B and C</a>). The exceptions were the E<sub>2</sub>/A and H<sub>4</sub>/A mutants, which showed a similar lag phase to AChE<sub>586-599</sub>, however their plateau height was affected with a drastic decrease for the E<sub>2</sub>/A mutant and an increase for the H<sub>4</sub>/A mutant. For both mutants, the substitution triggered stability of the assembly. The R<sub>5</sub>/A, S<sub>7</sub>/A, S<sub>8</sub>/A and W<sub>13</sub>/A mutants showed a similar plateau height to AChE<sub>586-599</sub>. However, their lag phases were increased, except for the R<sub>5</sub>/A mutant.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Substitution of H<sub>12</sub> and K<sub>14</sub> reduced the kinetics of fibrilization but did not abolish it completely. Indeed, slight increases of the lag phase were observed (5 and 3 times longer than AChE<sub>586-599</sub>, respectively), however their plateau heights were reduced on average by half (<a href="#pone-0001834-g003">Figure 3C</a>). Aggregation of the R<sub>5</sub>/A and ΔK<sub>14</sub> mutants exhibited a significantly shorter lag phase than AChE<sub>586-599</sub>, which indicated a lower kinetic solubility and higher kinetic rate of fibrilization (<a href="#pone-0001834-g003">Figure 3C</a>). In fact, the R<sub>5</sub>/A and ΔK<sub>14</sub> mutants immediately aggregated. These results suggested that positive charges from the side-chains of R<sub>5</sub> and K<sub>14</sub> might induce repulsion not favorable to a rapid aggregation and therefore might control assembly through specific interactions. By contrast, substitution of F<sub>3</sub> to Ala and Y<sub>9</sub> to Ala or Phe resulted in a dramatically decreased aggregation rate, with a very long lag phase (respectively 100, 64 or 160 times slower than AChE<sub>586-599</sub>) and a lower plateau height (except for the Y<sub>9</sub>/A mutant) (<a href="#pone-0001834-g003">Figure 3A and B</a>). The W<sub>13</sub>/A mutant was also affected with a lag phase 21 times slower than AChE<sub>586-599</sub> (<a href="#pone-0001834-g003">Figure 3C</a>). These results suggested the importance of aromatic residues in the first stage of aggregation of AChE<sub>586-599</sub>. Interestingly, the substitution of Y<sub>9</sub> to Phe affected the rate of aggregation even more than the substitution to Ala (<a href="#pone-0001834-g003">Figure 3C</a>). The most dramatic effect was that of the substitution of W<sub>6</sub> and M<sub>10</sub> to Ala, leading to a total loss of the peptide ability to form any aggregates as far as could be detected in the assay (<a href="#pone-0001834-g003">Figure 3C</a>). Finally, the substitutions to Ala of E<sub>2</sub>, F<sub>3</sub>, H<sub>4</sub>, S<sub>7</sub> and S<sub>8</sub> triggered the AChE<sub>586-599</sub> assembly to be stable (<a href="#pone-0001834-g003">Figure 3A and C</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Some mutants had very low level of fibrilization (very low plateau height) (e.g. the F<sub>3</sub>/A mutant), which may explain the absence of β-sheet conformation by CD for these mutants. Furthermore, the ThT assay done in similar conditions to the CD (no shaking, 24 hours) led to very poor fibrilization (long lag phase and low plateau height) or absence of fibrilization for the S<sub>7</sub>/A, S<sub>8</sub>/A, Y<sub>9</sub>/A and W<sub>13</sub>/A mutants (data not shown), which would also explain the absence of β-sheet conformation by CD.</p>


<h4>Effect of electrostatic interactions and ionic strength on the stability of AChE<sub>586-599</sub> aggregates</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To examine the effect of charge neutralization on AChE<sub>586-599</sub> fibril formation, we used increasing salt concentrations (NaCl and KCl) in a ThT assay, and the effects on both the E<sub>2</sub>/A and K<sub>14</sub>/A mutants were analysed. The strategy for selecting these mutants was based upon the presence of opposite charges on their side-chain at neutral pH, their position at the AChE<sub>586-599</sub> termini and their fibrilization properties (see above). Moreover, removing the side-chain charge from one of them was potentially leaving the side-chain charge from the other one uncompensated.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Increasing the salt concentration, therefore the ionic strength of the solute, significantly reduced the lag phase and increased the plateau height of the E<sub>2</sub>/A mutant (<a href="#pone-0001834-g004">Figure 4A and B</a>). Similarly, the salt concentration increase was able to significantly increase the plateau height of the K<sub>14</sub>/A mutant, in a concentration-dependent manner (<a href="#pone-0001834-g004">Figure 4C</a>). Furthermore, increasing the ionic strength stabilized the assembly of the K<sub>14</sub>/A mutant. Collectively these data strongly suggested that an increase in ionic strength of the solute successfully shielded the uncompensated charges in either mutant. In turn, this would indicate that E<sub>2</sub> and/or K<sub>14</sub> were likely to be involved in electrostatic interactions, such as salt bridges.</p>
<div class="figure" id="pone-0001834-g004"><div class="img"><a name="pone-0001834-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Effect of ionic strength on the fibrilization properties of mutant peptide with unstable amyloid products.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Mutant peptide E<sub>2</sub>/A (50 µM) (A and B) or K<sub>14</sub>/A (100 µM) (C) were incubated with 165 µM ThT in 1.8 mM KH<sub>2</sub>PO<sub>4</sub> and 10.1 mM NaH<sub>2</sub>PO<sub>4</sub> with varying concentration of NaCl and KCl. The concentrations of NaCl and KCl were respectively: 0 and 0 (0 X), 42.8 mM and 0.84 mM (0.31 X), 85.6 mM and 1.7 mM (0.63 X), 136.9 mM and 2.7 mM (1 X), 171.1 mM and 3.4 mM (1.25 X), 350 mM and 6.7 mM (2.5 X), 700 mM and 13.5 mM (5 X), and 1.4 M and 27 mM (10 X). Changes in ThT fluorescence were monitored (A, with the right panel showing a scale-up of the left panel to visualize the rapid fibrilizations; and C). The lag phase of fibrilization (B, left panel) and plateau height (B, right panel; and C, right panel) are depicted. A black star signifies p&lt;0.02 (B, left panel) and p&lt;0.03 (B, right panel; C, right panel) when compared to 1 X NaCl and KCl.</p>
<span>doi:10.1371/journal.pone.0001834.g004</span></div>

<h4>A lack of fibrilization does not preclude peptide interaction with AChE<sub>586-599</sub></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The very poor fibrilization or the absence of fibrilization of the F<sub>3</sub>/A, W<sub>6</sub>/A and M<sub>10</sub>/A mutants prompted us to investigate the effect of these mutants in co-fibrilization assays. Indeed if F<sub>3</sub>, W<sub>6</sub> and M<sub>10</sub> are crucial, as suggested by the CD and ThT assays (see <a href="#pone-0001834-g002">Figures 2</a> and <a href="#pone-0001834-g003">3</a>), for the formation of β-sheets and amyloid aggregation, it is important to determine whether the corresponding mutants could interact with the wild-type peptide AChE<sub>586-599</sub> and if they could affect AChE<sub>586-599</sub> fibrilization. It was previously shown that short sequences of Aβ containing Phe residues were able to bind specifically to the full length Aβ and to inhibit its fibrilization <a href="#pone.0001834-Tjernberg1">[26]</a>, <a href="#pone.0001834-Kuner1">[27]</a>. Such findings could be applied to the prevention of critical interactions occurring during amyloidogenesis.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>When AChE<sub>586-599</sub> (50 µM) was fibrilized in the presence of various concentrations of the F3/A mutant (ranging from 1.56 µM to equimolar), some of AChE<sub>586-599</sub> fibrilization parameters were affected. First, the F<sub>3</sub>/A mutant was able to stabilize the final aggregation stage of AChE<sub>586-599</sub> (<a href="#pone-0001834-g005">Figure 5A</a>). Indeed, the assembly of AChE<sub>586-599</sub> on its own was not stable with the plateau reaching a maximum before decreasing (<a href="#pone-0001834-g005">Figure 5A</a>, black open squares). By contrast, the plateau remained stable when AChE<sub>586-599</sub> was co-fibrilized with substoichiometric amounts of the F3/A mutant (from 6.25 to 50 µM). Second, the presence of the F<sub>3</sub>/A mutant (from 6.25 to 50 µM) also shortened the lag phase and increased the plateau height of AChE<sub>586-599</sub> (p&lt;0.005 and p&lt;0.014 respectively) (<a href="#pone-0001834-g005">Figure 5B and 5C</a>). Thus, AChE<sub>586-599</sub> and the F<sub>3</sub>/A mutant were clearly able to interact with one another.</p>
<div class="figure" id="pone-0001834-g005"><div class="img"><a name="pone-0001834-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Effect of the mutant peptide F<sub>3</sub>/A on AChE<sub>586-599</sub> fibrilization.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Varying concentrations of the mutant F<sub>3</sub>/A were incubated with 165 µM ThT, with or without 50 µM AChE<sub>586-599</sub>. Changes in ThT fluorescence were monitored (A) with the lag phase of fibrilization (B) and the plateau height (C) depicted. The mutant peptide F<sub>3</sub>/A decreases the lag phase of AChE<sub>586-599</sub> (A, with the inset showing a scale-up to visualize the fibrilization of the F<sub>3</sub>/A mutant on its own; and B), and increases the plateau height of AChE<sub>586-599</sub> (A and C). A black star signifies p&lt;0.03 (B) and p&lt;0.05 (C) when compared to both 50 µM AChE<sub>586-599</sub> and the equivalent concentration of the F<sub>3</sub>/A mutant. A white star signifies p&lt;0.05 (C) when compared to the equivalent concentration of the F<sub>3</sub>/A mutant. The double bar in B indicates the absence of fibrilization (i.e. an indeterminably long lag phase).</p>
<span>doi:10.1371/journal.pone.0001834.g005</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>In contrast to the F<sub>3</sub>/A mutant, the W<sub>6</sub>/A and M<sub>10</sub>/A mutants did not affect AChE<sub>586-599</sub> fibrilization properties. Indeed in the presence of either mutant, both the lag phase and plateau height were similar to the values for AChE<sub>586-599</sub> fibrilized on its own (<a href="#pone-0001834-g006">Figures 6A, B, D and E</a>). However, both mutants clearly interacted with AChE<sub>586-599</sub>, as indicated by the far-UV CD of the mixture, which was not entirely the arithmetic addition of the two separate spectra (<a href="#pone-0001834-g006">Figures 6 C and F</a>). The spectra of the AChE<sub>586-599</sub> and mutant mixtures after 24 hours under neutral conditions indicated the presence of both random coil (200 nm) and β-sheet (215 nm) structures. The random coil signals for the mixtures were statistically different to the signal of the arithmetic sums of AChE<sub>586-599</sub> with the W<sub>6</sub>/A mutant or with the M<sub>10</sub>/A mutant (p&lt;0.014 and p&lt;0.01 respectively) (<a href="#pone-0001834-g006">Figures 6C and F</a>, insets).</p>
<div class="figure" id="pone-0001834-g006"><div class="img"><a name="pone-0001834-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Effect of the mutant peptides W<sub>6</sub>/A and M<sub>10</sub>/A on AChE<sub>586-599</sub> fibrilization.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Varying concentrations of the mutant peptides were incubated with 165 µM ThT, with or without 50 µM AChE<sub>586-599</sub>. Changes in ThT fluorescence were measured and plotted as the lag phase of fibrilization (A and D) and the plateau height (B and E). The double bar in A and D indicates the absence of fibrilization (i.e.an indeterminably long lag phase). Far UV spectra (250 to 190 nm) before and after pH neutralization (50 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.2) of 75 µM AChE<sub>586-599</sub> with 75 µM mutant peptide (C and F). The insets show the mean residue ellipticity at 200 nm (random coil) and 215 nm (β-sheet) for 75 µM AChE<sub>586-599</sub>, 75 µM mutant peptide, 75 µM AChE<sub>586-599</sub> with 75 µM mutant peptide, and the arithmetic addition of AChE<sub>586-599</sub> to the mutant. (A and B) The mutant peptide W<sub>6</sub>/A does not affect either the lag phase (A, a black star signifies p&lt;0.002 when compared to W<sub>6</sub>/A at the equivalent concentration) or the plateau height of AChE<sub>586-599</sub> (B, a black star signifies p&lt;0.045 when compared to W<sub>6</sub>/A at the equivalent concentration; a white star signifies p&lt;0.05 when compared to AChE<sub>586-599</sub>). (C) The mutant peptide W<sub>6</sub>/A interacts with AChE<sub>586-599</sub>. A black star signifies p&lt;0.014 when compared to the arithmetic addition of 75 µM W<sub>6</sub>/A mutant to 75 µM AChE<sub>586-599</sub>. (D and E) The mutant peptide M<sub>10</sub>/A does not affect either the lag phase (D, a black star signifies p&lt;0.0007 when compared to M<sub>10</sub>/A at the equivalent concentration) or the plateau height of AChE<sub>586-599</sub> (E, a black star signifies p&lt;0.045 when compared to M<sub>10</sub>/A at the equivalent concentration). (F) The mutant peptide M<sub>10</sub>/A interacts with AChE<sub>586-599</sub>. A black star signifies p&lt;0.01 when compared to the arithmetic addition of 75 µM M<sub>10</sub>/A mutant and 75 µM AChE<sub>586-599</sub>.</p>
<span>doi:10.1371/journal.pone.0001834.g006</span></div>

<h4>Identification of the residues important for the surfactant properties of AChE<sub>586-599</sub></h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Both AChE<sub>586-599</sub> and Aβ possess surfactant properties. Similarly to detergents, both peptides reduce the surface tension of water by orientating their hydrophobic moiety away from the aqueous phase and ordering their amphiphilic moiety at the air-water interface <a href="#pone.0001834-Soreghan1">[28]</a>, <a href="#pone.0001834-Cottingham3">[29]</a>. The surfactant activity of AChE<sub>586-599</sub> was previously shown to be highly pH dependent <a href="#pone.0001834-Cottingham3">[29]</a>. All the substitution mutants were analyzed for surfactant activity by measuring differential absorbance, as described in <a href="#s4">Material and Methods</a> (<a href="#pone-0001834-g007">Figure 7</a>). AChE<sub>586-599</sub> showed a differential absorbance of 0.29±0.02 ΔOD (<a href="#pone-0001834-g007">Figure 7A and C</a>). Only one mutant (R<sub>5</sub>/A) displayed surfactant activity similar to AChE<sub>586-599</sub> after 2 min neutralization and only one showed a reduced surfactant effect under these conditions (K<sub>14</sub>/A) (<a href="#pone-0001834-g007">Figure 7A and C</a>). The remaining mutants showed a larger increase of ΔOD after 2 min at neutral pH, which demonstrated that their effect on surface tension was bigger than AChE<sub>586-599</sub> and also strongly pH dependent (p&lt;0.035) (<a href="#pone-0001834-g007">Figure 7C</a>).</p>
<div class="figure" id="pone-0001834-g007"><div class="img"><a name="pone-0001834-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Surfactant properties of AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Surface tension was measured before and after neutralization (1M NaH<sub>2</sub>PO<sub>4</sub>, pH 7.2). (A) Representative surfactant activity of the AChE<sub>586-599</sub> mutants (50 µM). ΔOD calculations were as described in <a href="#s4">Methods</a>. A black star signifies p&lt;0.035 when compared to AChE<sub>586-599</sub>. (B) Temporal pattern of the surfactant properties for AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants. The peptides shown are representatives of the different surfactant properties observed. A black star signifies p&lt;0.05 when compared to the same peptide after 2 min at neutral pH. (C) Surfactant properties for AChE<sub>586-599</sub> and all AChE<sub>586-599</sub> mutants (50 µM). The properties are divided into 2 categories: surfactant activity dependent on pH (depicted by subtracting the value at acidic pH to the value at neutral pH after 2 min) and stability of the surfactant activity (indicated by stability or decay of the OD signal). The mutation within AChE<sub>586-599</sub> is indicated in bold and italics. The peptides with unstable surfactant activity are indicated by grey boxes. ‘*’ indicates peptides which activity remains stable over the time course, albeit one time point. The activity for the mutant peptides is shown as fold ratio of AChE<sub>586-599</sub> activity (e.g. ‘1’ represents equal value to AChE<sub>586-599</sub>).</p>
<span>doi:10.1371/journal.pone.0001834.g007</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>When the temporal pattern of the surfactant activity of the mutants was analyzed, it was realized that some mutants were not stably surface active (<a href="#pone-0001834-g007">Figure 7B and C</a>). Indeed, the surfactant activity of the E<sub>2</sub>/A, F<sub>3</sub>/A, V<sub>11</sub>/A, H<sub>12</sub>/A and W<sub>13</sub>/A mutants, displayed after 2 min at neutral pH, significantly decreased with time (p&lt;0.05). This result indicates that these mutants were able to quickly segregate at the air-water interface after neutralization. The subsequent loss of surfactant effect suggests that these peptides were then either promptly leaving the air-water interface to return to the bulk, or were undergoing a conformational change at the air-water interface to reduce their effect on surface energy. By contrast, the surfactant activity of AChE<sub>586-599</sub> and the other mutants was stable, which suggests that they remained stably associated with the air-water interface during the length of the time course.</p>


<h4>Identification of the residues important for the early steps of AChE<sub>586-599</sub> oligomerization</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The absence of ThT signal during the fibrilization assay did not preclude the presence of small oligomers that could be ThT negative. Therefore, we analyzed the involvement and role of the side-chains of each residue within AChE<sub>586-599</sub> in oligomer formation and oligomer size distribution by performing photo-induced cross-linking of unlabeled AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants (PICUP) <a href="#pone.0001834-Fancy1">[30]</a>. It was previously demonstrated that the concentration of monomeric peptide during PICUP had to remain below 15 µM to avoid non-specific collision (Kenneth Baker and David J Vaux, unpublished data). In our assay, it was not possible to detect the cross-linked oligomers via conventional protein stains, presumably because the starting mass of peptide is then distributed across many individually low abundance oligomeric species. Therefore, the oligomers were detected by western-blot using the specific mouse Mab 105A, anti-AChE<sub>586-599</sub> in a β-sheet conformation <a href="#pone.0001834-Cottingham2">[23]</a>, which also provided additional information about the conformation of the cross-linked products. However, it was impossible to assay the F<sub>3</sub>/A, H<sub>4</sub>/A and R<sub>5</sub>/A mutants since these mutations were shown to abolish 105A immunoreactivity <a href="#pone.0001834-Cottingham2">[23]</a>.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>AChE<sub>586-599</sub> oligomers had a distinct size distribution ranging from ~3 to 16 kDa, with the oligomer intensity decreasing with the increase in size (<a href="#pone-0001834-g008">Figure 8</a>, top panel). These oligomeric forms may range from dimers to nonamers according to their observed molecular weights (AChE<sub>586-599</sub> being 1.86 kDa). The most intense oligomer band was ~5 kDa, which may correspond to trimers. Substitutions of S<sub>7</sub>, S<sub>8</sub>, Y<sub>9</sub> and M<sub>10</sub> to Ala yielded a qualitative distribution of oligomers similar to that of AChE<sub>586-599</sub>. However, the relative total amounts formed were different with S<sub>7</sub>/A much weaker than AChE<sub>586-599</sub>, M<sub>10</sub>/A weaker, S<sub>8</sub>/A similar (except the ~5 kDa oligomers) and Y<sub>9</sub>/A much stronger. Therefore, the side-chains of S<sub>7</sub>, S<sub>8</sub>, Y<sub>9</sub> and M<sub>10</sub> were not essential for normal oligomer distribution. When Y<sub>9</sub> was substituted to Phe, both the oligomer distribution and amount decreased significantly, when compared to the Y<sub>9</sub>/A mutant. The only oligomer bands detected were at ~5 and 6.5 kDa, which could be trimers and tetramers. Phe differs from Tyr by missing the phenolic oxygen, which suggests that the importance of Y<sub>9</sub> in the formation of oligomers bigger than tetramers may reside in the phenolic oxygen rather than the phenolic ring itself.</p>
<div class="figure" id="pone-0001834-g008"><div class="img"><a name="pone-0001834-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Formation of amyloid oligomers by AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Oligomers of AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants (12 µM) were cross-linked by photo-induced cross-linking. Cross-linked products were resolved (16.5% Tris-Tricine SDS-PAGE), electro-blotted onto nitrocellulose and probed with Mab 105A (specific for AChE<sub>586-599</sub> in β-sheet conformation). Marker proteins are indicated. Arrows indicate low abundance oligomeric species. Due to the strength of the signal for the oligomeric species, Y<sub>9</sub>/A was loaded at a third of the amount of the other peptides (Y<sub>9</sub>/A*). The signal resulting from loading equal amount to the other peptides can be seen on the individual lane on the right hand side of the top panel (Y<sub>9</sub>/A). On the right hand side of the bottom panel, an overexposure of the signal for W<sub>13</sub>/A shows multiple oligomeric species not seen at normal exposure.</p>
<span>doi:10.1371/journal.pone.0001834.g008</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The substitution of W<sub>6</sub> and W<sub>13</sub> to Ala severely affected both the amount and size distribution of the oligomers, with the oligomers ranging from ~3 to 9 kDa (dimers to pentamers) for W<sub>6</sub>/A and from ~5 to 9 kDa (trimers to pentamers) for W<sub>13</sub>/A, and the oligomer amount being more reduced for W<sub>13</sub>/A. Thus, the side-chains of the W<sub>6</sub> and W<sub>13</sub> appeared to be essential and a driving force in the association into oligomers.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>For the V<sub>11</sub>/A, K<sub>14</sub>/A and ΔK<sub>14</sub> mutants, oligomers at ~78 kDa (circa 42-mers) were detected. For V<sub>11</sub>/A and K<sub>14</sub>/A, the amount of these high molecular weight oligomers was reduced relative to that of the ΔK<sub>14</sub> mutant. As the abundance of the band at ~78 kDa increased, the low molecular weight oligomers were fewer and less abundant, suggesting a precursor-product relationship. Although, the high molecular weight oligomer band at ~78 kDa was less intense for K<sub>14</sub>/A relative to ΔK<sub>14</sub>, a prominent smear of oligomers was present between ~12 and 40 kDa. Along with the ~78 kDa oligomers, 2 other types of oligomers ~5 to 6.5 kDa (trimers and tetramers) were prominent for V<sub>11</sub>/A. In addition to the normal oligomer size distribution, substitution of H<sub>12</sub> to Ala also yielded a smear of oligomers ranging from ~12 to 20 kDa (heptamers to dodecamers). These results suggested that the side-chains of V<sub>11</sub>, H<sub>12</sub> and K<sub>14</sub> within AChE<sub>586-599</sub> restrained the formation of high molecular weight oligomers and therefore controlled AChE<sub>586-599</sub> oligomerization.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>Mutation of E<sub>2</sub> to Ala abolished the immediate formation of cross-linked oligomers larger than ~3 kDa (dimers), suggesting that E<sub>2</sub> was essential for any oligomer formation above dimers at this low concentration during this short time-course experiment.</p>


<h4>Morphology of the amyloid aggregates of AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The ThT assay has allowed the determination of the aggregation potential and kinetics of the various peptides, and also provided a clue about the stability and quantity of the final amyloid products (plateau height). However, it did not give direct information about the size and morphology of the aggregates formed. Moreover, the analysis of the oligomer formation and oligomer size distribution by PICUP did not allow the study of the F<sub>3</sub>/A, H<sub>4</sub>/A and R<sub>5</sub>/A mutants. Thus, we used negative staining electron microscopy (EM) to examine the ultrastructure of the aggregates for the F<sub>3</sub>/A, H<sub>4</sub>/A and R<sub>5</sub>/A mutants and also to determine whether mutants, which were faster than AChE<sub>586-599</sub> at adopting a β-sheet conformation and/or at fibrilizing (e.g. K<sub>14</sub>/A and ΔK<sub>14</sub>), could form different types of amyloid aggregates. Representative images of the morphologies observed are presented in <a href="#pone-0001834-g009">Figure 9</a>.</p>
<div class="figure" id="pone-0001834-g009"><div class="img"><a name="pone-0001834-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>T40/IDE digestion products form amyloid protofibrils.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Electron micrographs of negatively stained AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants showing fibrils (A) and protofibrils (B). The white arrows in the left panel indicate thinner fibrils, and the black arrows indicate twists in the fibrils (A).</p>
<span>doi:10.1371/journal.pone.0001834.g009</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Appearance of unbranched fibrils was observed for AChE<sub>586-599</sub> and mutants E<sub>2</sub>/A, H<sub>4</sub>/A, R<sub>5</sub>/A, K<sub>14</sub>/A and ΔK<sub>14</sub> (<a href="#pone-0001834-g009">Figure 9A</a>). The fibrils formed by the mutants were different to the ones derived from the wild-type peptide. AChE<sub>586-599</sub> fibrils were generally aligned, very long (&gt;2 µm) and broad (10–15 nm) (<a href="#pone-0001834-g009">Figure 9A</a>, top panel) and showing typical helical twists (inset, black arrows). The periodicity of AChE<sub>586-599</sub> fibrils is 26–34 nm. Some thinner (5–8 nm) and sometimes shorter fibrils were also observed for AChE<sub>586-599</sub> (white arrows). By contrast, the fibrils from the E<sub>2</sub>/A mutant were as broad as the ones from AChE<sub>586-599</sub> (~15 nm), however they were shorter and less abundant (<a href="#pone-0001834-g009">Figure 9A</a>). The R<sub>5</sub>/A mutant formed large fibrilar aggregates, from which laterally associated (top inset) and tangled (bottom inset) fibrils were emerging (<a href="#pone-0001834-g009">Figure 9A</a>). Fibrils from H<sub>4</sub>/A mutant were abundant, thin (4–6 nm) and tangled with each other (<a href="#pone-0001834-g009">Figure 9A</a>), often resulting in structures resembling ‘plates’ (inset). Similarly, fibrils from ΔK<sub>14</sub> mutant were more abundant, slightly thinner (8–13 nm) and more tangled than AChE<sub>586-599</sub> (<a href="#pone-0001834-g009">Figure 9A</a>). ΔK<sub>14</sub> also displayed some very short fibrils (&gt;50 nm) (inset). The helical twists were also observed for ΔK<sub>14</sub> mutant (black arrow), however this twisted morphology was less frequent than for the wild-type AChE<sub>586-599</sub> fibrils. The K<sub>14</sub>/A mutant displayed a similar fibril morphology to the ΔK<sub>14</sub> mutant, albeit the fibrils were thinner (5.5 to 8.5 nm) and fewer (<a href="#pone-0001834-g009">Figure 9A</a>).</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Examination of F<sub>3</sub>/A mutant revealed predominantly spherical structures (diameter 10–16 nm), “rods” (8–13 nm wide, &gt;50 nm long) (protofibrils) and amorphous aggregates of various sizes (some being over 50 nm wide, inset) (<a href="#pone-0001834-g009">Figure 9B</a>). The spherical and “rod” structures are consistent with the presence of amyloid precursors (oligomers) <a href="#pone.0001834-Seilheimer1">[31]</a>, <a href="#pone.0001834-Lashuel1">[32]</a>, <a href="#pone.0001834-Jean1">[24]</a>. However, all of the “rods” appeared branched with the branching not following any particular dimension or orientation.</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>The morphology of the amyloid aggregates formed by the various peptides was consistent with the results from the fibrilization assay (e.g. the highly tangled and abundant fibrilar structures that are observed for the ΔK<sub>14</sub> mutant are consistent with the very fast fibrilization kinetics determined in the ThT assay).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The goal of this study was to investigate the driving forces involved in the organization of a model peptide, AChE<sub>586-599</sub>, into β-sheet oligomers and subsequently into amyloid fibrils. The choice of AChE<sub>586-599</sub> as a model peptide for amyloid assembly was guided by two criteria, which are the tractability of AChE<sub>586-599</sub> assembly upon neutralization (AChE<sub>586-599</sub> remains monomeric and random coil at acidic pH) and the involvement of AChE or related products in Alzheimer's disease and in the promotion of Aβ fibrilization <a href="#pone.0001834-SaezValero1">[18]</a>, <a href="#pone.0001834-Cottingham2">[23]</a>, <a href="#pone.0001834-Rees1">[19]</a>, <a href="#pone.0001834-Diamant1">[20]</a>, <a href="#pone.0001834-Jean1">[24]</a>. Amyloid fibrils originating from unrelated proteins or peptides possess similar structures and morphologies <a href="#pone.0001834-Harper1">[1]</a>, <a href="#pone.0001834-Rochet1">[5]</a>. While overall amino acid composition is an important factor determining amyloid formation, the details of primary sequence also plays an important role. Therefore, a common view is that fibril assembly is governed by properties of the sequence backbone <em>and</em> specific side-chain interactions <a href="#pone.0001834-Smith1">[7]</a>, <a href="#pone.0001834-West1">[8]</a>, <a href="#pone.0001834-Klimov1">[9]</a>, <a href="#pone.0001834-LopezdelaPaz1">[33]</a>, <a href="#pone.0001834-Zanuy1">[10]</a>. This would explain that a simple hydrophobic collapse would not be sufficient to drive the aggregation of any polypeptide chains under physiological conditions. On the contrary, a final ordered amyloid assembly would be determined by, and highly dependent on, a number of specific interactions between side-chains at specific positions within the sequence, which is supported by our results.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The effect of the positional scanning mutations suggested that there was a position dependence of AChE<sub>586-599</sub> assembly. Indeed, some positions within the AChE<sub>586-599</sub> sequence were tolerant to alanine substitutions, whereas others were restrictive. The termini of AChE<sub>586-599</sub> appeared to be more permissive to mutations (e.g. E<sub>2</sub>, H<sub>4</sub>, R<sub>5</sub> at the N-terminus, and H<sub>12</sub> and K<sub>14</sub> at the C-terminus) since they did not preclude fibrilization. By contrast, mutations within the core of AChE<sub>586-599</sub> sequence were very restrictive since they sharply abolished or affected fibrilization (e.g. W<sub>6</sub>, S<sub>7</sub>, S<sub>8</sub>, Y<sub>9</sub>, M<sub>10</sub> and V<sub>11</sub>). The only exceptions were two terminal aromatic residues, F<sub>3</sub> and W<sub>13</sub>, which drastically affected fibrilization. Thus, one could speculate that only the positions within the AChE<sub>586-599</sub> sequence providing maximal and optimal stabilizing interactions, would be affected by mutations. On one hand, the dependence of the observed amyloidogenic potentials on the position of the mutation could correlate with the β-sheet propensity and the hydrophobicity of the side-chain at this position. For example, the side-chains of A<sub>1</sub> and F<sub>3</sub> were optimally fitted with the side-chains of V<sub>11</sub> and W<sub>13</sub> to create a hydrophobic motif at the edges of the assembly (see below and the model in <a href="#pone-0001834-g010">Figure 10B</a>). Similarly, the side-chain of W<sub>6</sub> was optimally fitted with the side-chain of M<sub>10</sub> to create a hydrophobic motif at the core of the structure (see <a href="#pone-0001834-g010">Figure 10B</a>). It was also observed that substitutions within the core of AChE<sub>586-599</sub>, or modifying the hydrophobicity and/or aromaticity, affected to some extend the β-sheet propensity, with the strongest effect for F<sub>3</sub>, W<sub>6</sub>, Y<sub>9</sub>, M<sub>10</sub>, V<sub>11</sub> and W<sub>13</sub>. Since for the H<sub>4</sub>/A and R<sub>5</sub>/A mutants, there was no perturbation of the hydrophobic patch, the stabilizing effect of A<sub>1</sub>-W<sub>13</sub>, F<sub>3</sub>-V<sub>11</sub> and W<sub>6</sub>-M<sub>10</sub> side-chain interactions could still occur. Moreover, substitutions of H<sub>4</sub> and R<sub>5</sub> did not affect the β-sheet propensity within the AChE<sub>586-599</sub> sequence (see <a href="#pone-0001834-g001">Figure 1</a>). This could explain why the fibrilization properties of these mutants were at least as good as those of AChE<sub>586-599</sub>. Although substitution of E<sub>2</sub> and K<sub>14</sub> did not disrupt the hydrophobic patches, the mutations would abolish a putative salt bridge (as discussed below). On the other hand, previous studies have shown that during amyloid formation water molecules barely interact with the hydrophobic patches but instead cluster around the terminal charged residues <a href="#pone.0001834-LopezdelaPaz2">[34]</a>. This solvent effect could also contribute to the position dependence of AChE<sub>586-599</sub> aggregation.</p>
<div class="figure" id="pone-0001834-g010"><div class="img"><a name="pone-0001834-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Structural model for AChE<sub>586-599</sub> amyloid assembly based on conformation, fibrilization and surfactant properties of the wild type and mutant peptides.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(A) Summary of the conformation, fibrilization and surfactant properties of AChE<sub>586-599</sub> and mutant peptides. Mutant peptides with similar properties to AChE<sub>586-599</sub> (white boxes), with enhanced properties (green boxes), with diminished properties (red boxes, with low decrease indicated by light red and strong decrease by dark red). ‘N.D.’ indicates ‘non-detectable’. (B) Model of interactions between residues of AChE<sub>586-599</sub> β-strands, which form a steric-zipper interface between the fibril forming β-sheets. The steric-zipper interface is shown as an antiparallel assembly of three AChE<sub>586-599</sub> β-strands. AChE<sub>586-599</sub> is represented at the primary amino acid sequence level (left panel) or at the carbon backbone structure level (right panel). On the left panel, hydrophobic interactions are represented as green shaded boxes, electrostatic interactions as blue and red shaded boxes, cation-π interactions as pink boxes and potential metal binding sites as brown boxes. The grey arrows indicate the direction of the strands. On the right panel, the chains are colored by residue type with hydrophobic residues (A, F, W, M and V) in green, negatively charged (E) in red, positively charged (H, R and K) in blue, and polar (S and Y) in cyan. (C) Model of quaternary interactions within β-sheets of AChE<sub>586-599</sub> within a fibril. Each β-sheet is represented with only 3 copies of AChE<sub>586-599</sub> for clarity: sheet 1 colored in different shades of green, sheet 2 in different shades of blue, and sheet 3 in different shades of red. The fibril is growing from the lighter to the darker color. On the carbon backbone structure, only the side chains of aromatic residues (F<sub>3</sub>, H<sub>4</sub>, W<sub>6</sub>, Y<sub>9</sub>, H<sub>12</sub> and W<sub>13</sub>) are represented for clarity. The boxes highlight possible aromatic interactions (π-π) between strands within a β-sheet. Within a β-sheet, AChE<sub>586-599</sub> strands are stacking in a parallel arrangement. Within AChE<sub>586-599</sub> fibril, the β-sheets are antiparallel, have the same sides facing each other (‘face-to-face’) and the orientation of the sheet edges facing up (‘up-up’). According to the nomenclature of Sawaya <em>et al</em>., this type of arrangement and orientation corresponds to a class 1 steric-zipper <a href="#pone.0001834-Sawaya1">[70]</a>.</p>
<span>doi:10.1371/journal.pone.0001834.g010</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>There are criteria that an amyloid structure must meet to be stable, such as the need to place charged residues outside the core amyloid structure. Moreover, electrostatic interactions, such as salt bridges, may contribute to the orientation and stability of the amyloid assembly <a href="#pone.0001834-Petkova1">[35]</a>, <a href="#pone.0001834-Tjernberg2">[36]</a>, <a href="#pone.0001834-Makin1">[15]</a>, <a href="#pone.0001834-Tarus1">[37]</a>. Furthermore, Lys and Glu residues have been often found in neighboring β-strands, and peptides rich in these two residues formed fibrils <a href="#pone.0001834-Zhang1">[38]</a>, <a href="#pone.0001834-Wouters1">[39]</a>, <a href="#pone.0001834-Tjernberg2">[36]</a>. Massi et al. predicted that an equilibrium between electrostatic interactions and hydration determines the stability of amyloid-forming peptides, therefore that the fibrilization kinetics would be affected by pH and ionic strength <a href="#pone.0001834-Massi1">[12]</a>. We have demonstrated that by providing additional ionic strength, we successfully shielded the uncompensated charges of the E<sub>2</sub>/A and K<sub>14</sub>/A mutants resulting in a shorter lag phase for the assembly and an increase of the plateau height. For the K<sub>14</sub>/A mutant, the resulting aggregates were also more stable. Thus, these results suggested that specific Coulombic interactions, such as a salt bridge between E<sub>2</sub> and K<sub>14</sub>, might occur during AChE<sub>586-599</sub> oligomerization. This hypothesis is reinforced by the rapid kinetics of fibrilization and the presence of high molecular weight oligomers upon substitution of the charged K<sub>14</sub> with an uncharged side-chain (Ala), which demonstrated the unequivocal involvement of the positive charge of the K<sub>14</sub> side-chain. A similar effect was observed for Aβ, in which the mutation of D<sub>23</sub> to Asn yielded higher oligomers, suggested to be due to the removal of the salt bridge between D<sub>23</sub> and K<sub>28</sub> <a href="#pone.0001834-Bitan1">[14]</a>. Nilsberth et al. also found that for the Aβ arctic mutation (E<sub>22</sub> to Gly), the substitution increased the rate of protofibril formation <a href="#pone.0001834-Nilsberth1">[40]</a>. Furthermore, substitutions of K<sub>14</sub>, by affecting early kinetics and assembly, could destabilize the structure of the final assembly or drive the assembly into a different pathway, which would lead to different morphologies for the aggregates. In accordance with this hypothesis, the fibrils observed for the K<sub>14</sub>/A and ΔK<sub>14</sub> mutants were shorter, and thinner and more tangled than AChE<sub>586-599</sub>. Similarly, the presence of an Ala instead of the N-terminal Asn, or the absence of Asn, in the peptide NFGAILSS of human islet amyloid polypeptide (IAPP) accelerated the kinetics of aggregation and modified the morphology of the fibrils, which were thinner and more tangled <a href="#pone.0001834-Tenidis1">[41]</a>, <a href="#pone.0001834-Azriel1">[11]</a>. Tenidis et al proposed that Asn directed self-assembly and lateral packing of the filaments <a href="#pone.0001834-Tenidis1">[41]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Interestingly, the E<sub>2</sub>/A mutant had a stable plateau height (albeit drastically decreased) and similar lag phase of fibrilization to that of AChE<sub>586-599</sub>. The increase of β-sheet propensity triggered by this mutation (see <a href="#pone-0001834-g001">Figure 1</a>) could compensate the absence of the salt bridge with K<sub>14</sub> during early assembly, which would explain the similarity of lag phase. Moreover, E<sub>2</sub> was found to be essential for the formation of normal amount and normal length fibrils, which could explain the very low plateau height observed in the ThT assay for this mutant. The substitutions of E<sub>2</sub> and K<sub>14</sub> also altered the general properties of AChE<sub>586-599</sub>, such as its net charge and isoelectric point (pI). At acidic pH, the net charge of AChE<sub>586-599</sub> is +4, with the side-chains of H<sub>4</sub>, R<sub>5</sub>, H<sub>12</sub> and K<sub>14</sub> being protonated. This high density of positive charges would prevent interactions between AChE<sub>586-599</sub> molecules, which would explain the random coil conformation. By contrast, at neutral pH the net charge of AChE<sub>586-599</sub> is +1, with the carboxylic group of E<sub>2</sub> side-chain being deprotonated (−1), the amino group of R<sub>5</sub> and K<sub>14</sub> side-chains protonated (+2). By substituting E<sub>2</sub> to Ala, the net charge of AChE<sub>586-599</sub> became +2. By substituting R<sub>5</sub> or K<sub>14</sub> to Ala, the net charge of AChE<sub>586-599</sub> became 0. According to Chiti et al., an increase of the net charge would trigger intermolecular repulsion, whereas a low net charge would favor aggregation <a href="#pone.0001834-Chiti1">[42]</a>. Therefore, the E<sub>2</sub>/A mutant would be less prone to fibrilization, and the R<sub>5</sub>/A and K<sub>14</sub>/A and ΔK<sub>14</sub> mutants would fibrilize more rapidly than AChE<sub>586-599</sub>, just as we observed. The pI of AChE<sub>586-599</sub> is 8.65 and substitution decreasing it could facilitate aggregation at neutral pH. Substitution of E<sub>2</sub> to Ala raises the pI to 10.00, which could explain why the E<sub>2</sub>/A mutant was no faster than AChE<sub>586-599</sub>. By contrast, substitution of R<sub>5</sub> to Ala, or K<sub>14</sub> to Ala or removal of K<sub>14</sub> decreases the pI to 6.96. These observations are in perfect agreement with our CD and fibrilization results, in which each of these mutants was faster than AChE<sub>586-599</sub> by at least one experimental measure.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Single point mutations, which were not affecting the net charge of AChE<sub>586-599</sub>, were found to drastically affect or to completely abolish AChE<sub>586-599</sub> conformational change and fibrilization (e.g. F<sub>3</sub>/A and W<sub>6</sub>/A). Therefore, the minimization of Coulombic repulsion is not the only factor involved in amyloid assembly and interactions between side-chains, particularly of hydrophobic and aromatic nature, could provide additional energy for stabilization. The frequency of aromatic residues is low in proteins in general, however they occur very frequently in amyloid-related sequences <a href="#pone.0001834-Azriel1">[11]</a>, <a href="#pone.0001834-Gazit1">[43]</a>. Interactions between aromatic ring planes that are parallel to each other, referred as π-π interactions or π-stackings, play a key role in molecular recognition and self-assembly, which could be the function that the aromatic residues play during amyloid formation <a href="#pone.0001834-Claessens1">[44]</a>, <a href="#pone.0001834-Gillard1">[45]</a>, <a href="#pone.0001834-Gazit1">[43]</a>, <a href="#pone.0001834-Waters1">[46]</a>. Moreover, aromatic residues are characterized by both a high hydrophobicity and a high β-sheet propensity. Aromatic residues are abundant in AChE<sub>586-599</sub> (29% of the total residues) and we assessed their involvement by a complementary approach, including far- and near-UV CD. These assays showed that upon neutralization a conformational transition from random coil to β-sheet occurred for AChE<sub>586-599</sub> and involved strong interactions between aromatic residues for the formation of tertiary or quaternary structures. Indeed, the near-UV CD clearly demonstrated that the aromatic rings had restrained mobility, as when buried, which is consistent with these residues stacking during β-sheet formation. Similar π-stacking was described for several amyloid-forming peptides, such as IAPP and Aβ <a href="#pone.0001834-Makin1">[15]</a>, <a href="#pone.0001834-Jack1">[25]</a>. Additionally to the CD studies, the influence of the aromatic residues was also observed in the fibrilization and oligomerization assays. Indeed F<sub>3</sub>, W<sub>6</sub>, Y<sub>9</sub> and W<sub>13</sub>, when mutated, appeared to significantly influence the first stage of aggregation with either no fibrilization observed or a drastically longer lag phase. This result was also confirmed by the study on oligomer formation where the F<sub>3</sub>/A, W<sub>6</sub>/A, Y<sub>9</sub>/F and W<sub>13</sub>/A mutants led to a very poor oligomerization. The effect of the substitutions on the early stage of aggregation might be related to a decrease in hydrophobicity and β-sheet propensity rather than a lack of aromatic ring, as it was proposed for other systems <a href="#pone.0001834-Wu1">[47]</a>, <a href="#pone.0001834-Bemporad1">[48]</a>. However in later stages of the aggregation, the role of the aromatic rings of F<sub>3</sub>, W<sub>6</sub>, Y<sub>9</sub> and W<sub>13</sub> within AChE<sub>586-599</sub> might involve π-stacking to stabilize the cross-β structure. Indeed in a number of models, the rings of Phe residues were proposed to cement together the β-strands in a β-sheet, creating a Phe zipper <a href="#pone.0001834-Sikorski1">[49]</a>, <a href="#pone.0001834-Naito1">[50]</a>, <a href="#pone.0001834-Zanuy1">[10]</a>, <a href="#pone.0001834-Makin1">[15]</a>. Furthermore, the substitution of F<sub>3</sub> or W<sub>13</sub> to Ala may have affected the tight association between strands, at the paired hydrophobic patches within the steric-zipper (see green box in <a href="#pone-0001834-g010">Figure 10B</a>). The smaller side-chain of Ala instead of the bulky side-chain of F<sub>3</sub> or W<sub>13</sub> may permit increased flexibility that could have opposed an ordered assembly. Additionally to an effect on early aggregation, the F<sub>3</sub> mutation to Ala resulted in spherical oligomers, short, wide and branched protofibrils, and amorphous aggregates. Similarly, a Phe to Ala substitution in the peptide NFGAILSS of human IAPP, resulted in the formation of amorphous aggregates <a href="#pone.0001834-Azriel1">[11]</a>. It was previously described that the specificity and the directionality of the amyloid assembly could be provided by the specific orientation of aromatic side-chains <a href="#pone.0001834-Shetty1">[51]</a>, <a href="#pone.0001834-McGaughey1">[52]</a>, <a href="#pone.0001834-Gazit1">[43]</a>. Without F<sub>3</sub>, AChE<sub>586-599</sub> may be lacking important interactions involved in the directionality of the assembly. This could result in a correct minimal assembly that failed to orientate for further linear stacking, leading to multiple branching and finally aggregation into amorphous structures. Therefore, the branched “protofibrils” and amorphous aggregates would be an amyloid dead end and no fibrils would be formed, as our EM results suggested.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The F<sub>3</sub>/A mutant, as mentioned above, would be less hydrophobic at the extremities of the assembly and therefore would be more exposed to the solvent. Thus, the F<sub>3</sub>/A mutant might not segregate at the air-water interface as stably as AChE<sub>586-599</sub> could, due to its amphipathicity. By analyzing the temporal pattern of surfactant activity, we demonstrated that indeed AChE<sub>586-599</sub> stably remained associated with the air-water interface, whereas the F<sub>3</sub>/A mutant quickly left the interface to relocate to the bulk solution. However, the F<sub>3</sub>/A mutant possessed a stronger surfactant activity than AChE<sub>586-599</sub>, after 2 min at neutral pH. This result suggested that soon after neutralization, the F<sub>3</sub>/A mutant was able to relocate and to aggregate faster than AChE<sub>586-599</sub> at the air-water interface. This could explain the shorter lag phase observed when AChE<sub>586-599</sub> was co-fibrilized with the F<sub>3</sub>/A mutant, as compared to AChE<sub>586-599</sub> alone. Indeed if faster at aggregating, only small amount of the F<sub>3</sub>/A mutant would be sufficient to create nuclei to accelerate an assembly. Above a certain threshold of assembly, AChE<sub>586-599</sub> aggregates would eventually relocate to the bulk solution to free the air-water interface for monomer recruitment and further assembly. Once in the bulk, AChE<sub>586-599</sub> assembly would possibly be prone to dissociation or “shedding”. The F<sub>3</sub>/A mutant formed branched and amorphous aggregates, which by being an amyloid dead-end might be more stable than those from AChE<sub>586-599</sub>. This hypothesis could explain the difference of aggregate stability between the F<sub>3</sub>/A mutant (stable) and AChE<sub>586-599</sub> (unstable), and also the fact that the F<sub>3</sub>/A mutant was able to stabilize AChE<sub>586-599</sub> during co-fibrilization assays (e.g. by capping and slowing dissociation). Furthermore, López de la Paz et al. proposed that water molecules could act as a cement to bring strands and side-chains close enough via water-mediated hydrogen bonds, which would stabilize the amyloidogenic organization <a href="#pone.0001834-LopezdelaPaz2">[34]</a>. A similar effect of water molecules could stabilize the F<sub>3</sub>/A mutant in the bulk solution. Similarly to the F<sub>3</sub>/A mutant, the assembly stability at the air water-interface of the V<sub>11</sub>/A, H<sub>12</sub>/A and W<sub>13</sub>/A mutants was affected, which correlates with the negative effect of the substitutions on their fibrilization potential. It was previously proposed that surface tension could play an important role in the stabilization of proteins <a href="#pone.0001834-Lin1">[53]</a>. Collectively, these results suggest that AChE<sub>586-599</sub> amyloid assembly could preferentially occur at an air-water interface rather than in the bulk solution, potentially due to a stabilization effect.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In addition to the non-covalent interactions described above, cation-π interactions have been also found to play a role in molecular association in biological systems <a href="#pone.0001834-Crowley1">[54]</a>, <a href="#pone.0001834-Paddock1">[55]</a>. A cation-π interaction is a short-range electrostatic interaction between π electrons in an aromatic ring and a positively charged cation, most commonly between Arg and Tyr <a href="#pone.0001834-Ma1">[56]</a>, <a href="#pone.0001834-Crowley1">[54]</a>. The formation of a cation-π interaction could lower the cost of desolvating the charge of the cation and could provide a mean for burying the positively charged group within a solvent-excluding domain. Moreover, cation-π interactions are important for specificity and stability during protein association <a href="#pone.0001834-Crowley1">[54]</a>, <a href="#pone.0001834-Chakkaravarthi1">[57]</a>. In protein complexes, Arg involved in cation-π interactions were also found to be involved in cation-anion interactions, which provide long range attraction for the guanidium group and ensure the specificity of binding <a href="#pone.0001834-Crowley1">[54]</a>. The importance of cation-π interaction was demonstrated for other amyloidogenic peptide <a href="#pone.0001834-Yoshida1">[58]</a>, <a href="#pone.0001834-Zahn1">[59]</a>, <a href="#pone.0001834-Bemporad1">[48]</a>. Thus, it is possible that a cation-π interaction between R<sub>5</sub> and Y<sub>9</sub> occurred during AChE<sub>586-599</sub> fibrilization (see <a href="#pone-0001834-g010">Figure 10B</a>, pink boxes), allowing the burial of the polar Arg group within the core of AChE<sub>586-599</sub>. Without R<sub>5</sub>, AChE<sub>586-599</sub> rapidly formed large fibrilar aggregates composed of laterally associated and tangled fibrils, which could be due to the absence of specificity and stability provided by an R<sub>5</sub>-Y<sub>9</sub> interaction. In addition to cation-π interaction, amino-aromatic interaction could also contribute to the formation of R<sub>5</sub>-Y<sub>9</sub> side-chain interactions, and Y<sub>9</sub> could be involved in long-range interaction with the negatively charged side-chain of E<sub>2</sub> providing further specificity to the assembly. The substitution of Y<sub>9</sub> with Phe, conserving only the aromatic ring, had a more deleterious effect on fibrilization than the Ala substitution and drastically impaired oligomer formation, suggesting a role of Y<sub>9</sub> OH group during stacking rather than during very early assembly (formation of the nuclei).</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Additionally to π-π and cation-π interactions, aromatic residues can also be involved in SH-π interactions. The Met sulfur can favorably and strongly interact with the non-polar surfaces on binding partners (specifically the aromatic face of residues), and can also engage oxygen atoms through S–O interactions and N–H groups through hydrogen bonding. Such interactions would be useful in the association between different sub-units in oligomeric proteins and could be a stabilizing force in holding two β-strands together <a href="#pone.0001834-Pal1">[60]</a>. Met has been found to preferentially associate with Trp, due to hydrogen bonding and S-aromatic interactions <a href="#pone.0001834-Samanta1">[61]</a>. When M<sub>10</sub> was substituted to Ala, AChE<sub>586-599</sub> lost the ability to switch to a β-sheet conformation and to fibrilize. However, the M<sub>10</sub>/A mutant was able to form oligomers with a size distribution identical to AChE<sub>586-599</sub> but less abundant, which reinforces that it is not merely the hydrophobicity of the side-chain that drives AChE<sub>586-599</sub> oligomerization. It is possible that these oligomers did not bind ThT and were not able to further assemble into larger species, resulting in an absence of signal during the ThT assays. A role of M<sub>35</sub> has been described in the dimerization of the Aβ protofibril <a href="#pone.0001834-Petkova2">[62]</a>. Therefore, it is possible that M<sub>10</sub> interacted with W<sub>6</sub>, through hydrophobic or S-aromatic interactions, to stabilize the AChE<sub>586-599</sub> assembly. The similar effects observed for the substitutions of both W<sub>6</sub> and M<sub>10</sub> on fibrilization and oligomerization reinforce this hypothesis.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Another important factor able to stabilize proteins is metal chelation <a href="#pone.0001834-Kellis1">[63]</a>. Studies on the binding of metal ions on amyloid proteins or peptides demonstrated that Cu(II) ions induce β-sheet formation of the unstructured amyloidogenic region of the prion protein, and Cu(II) and Zn(II) ions strongly induce Aβ fibrilization <a href="#pone.0001834-Atwood2">[64]</a>, <a href="#pone.0001834-Liu1">[65]</a>, <a href="#pone.0001834-Miura1">[66]</a>, <a href="#pone.0001834-Yang1">[67]</a>, <a href="#pone.0001834-Jones1">[68]</a>. In the case of AChE<sub>586-599</sub>, the substitution of H<sub>12</sub> to Ala affected both the fibrilization rate and the oligomerization, with a slower lag phase than AChE<sub>586-599</sub>, a decrease in plateau height and an unbalanced distribution of oligomers. Moreover, the substitution of H<sub>4</sub> to Ala affected the morphology of the fibrils, thinner than AChE<sub>586-599</sub> and tangled. Thus, it appeared that H<sub>12</sub> was more “important” than H<sub>4</sub>, which was in agreement with the significant loss of propensity for conversion to β-strand within the sequence YMVH (the strongest propensity within AChE<sub>586-599</sub>), when H<sub>12</sub> was mutated to Ala. A putative role for H<sub>4</sub> and H<sub>12</sub> in metal chelation will be the subject of further studies.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>AChE<sub>586-599</sub> is amphiphilic due to an alternating pattern of polar and non-polar residues, which would trigger the burial of the non-polar faces by aggregating into β-sheet structure. Although nature and evolution have disfavored such alternating pattern, sequences containing it were shown to self-assemble <a href="#pone.0001834-West1">[8]</a>. Polar side-chains have the advantage of forming hydrogen bonds, in addition to the van der Waals interactions. In general polar residues interact with the solvent or other polar residues. The effects of substitutions of S<sub>7</sub> and S<sub>8</sub> on all the AChE<sub>586-599</sub> properties tested were similar and consistent. Indeed, the decrease oligomer amount for the S<sub>7</sub>/A and S<sub>8</sub>/A mutants were in agreement with the absence of fibrilization or their longer lag phases of fibrilization. The similarity and consistency upon substitution suggests that the two Ser residues might interact together rather than with other residues, creating a polar-polar interaction through hydrogen bonding, which would be in agreement with the strong correlation found between Ser-Ser pairing in β-sheets <a href="#pone.0001834-Wouters1">[39]</a>.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>It is thought that extended parallel β-sheets are less stable than antiparallel ones <a href="#pone.0001834-Salemme1">[69]</a>, <a href="#pone.0001834-LopezdelaPaz2">[34]</a>. This is based on the fact that in antiparallel β-sheets, most contacts along the fibril axis are between non-identical and complementary residues, which allow more variability in the hydrogen bonds and side-chain interactions, and also in the geometry of the interactions. This variability would allow a greater number of possible conformations and alignments for the strands. By contrast, in a parallel arrangement, the contacts are in between identical residues and the optimal geometry of the hydrogen bonding would therefore be linear. The presence of uncompensated opposite charges on each peptide plays a fundamental role in favoring an arrangement in which the distance between identical charges is maximized. In the case of AChE<sub>586-599</sub>, this would favor an antiparallel organization to avoid high electrostatic repulsion due to E<sub>2</sub> and K<sub>14</sub>. This is a conclusion already tentatively suggested by ELISA on these mutants, using the MAb 105A <a href="#pone.0001834-Cottingham2">[23]</a>. A good agreement and correlation were found between all the assays and properties for AChE<sub>586-599</sub> aggregation, and are summarized in <a href="#pone-0001834-g010">Figure 10A</a>. In accord with the experimental observations, and taking all the previous arguments and interactions into consideration, we attempted to model the assembly of AChE<sub>586-599</sub>, in which we have considered only an antiparallel arrangement for the β-sheets (<a href="#pone-0001834-g010">Figure 10B and C</a>). Our model fits with the position dependence of assembly (restrictive and permissive positions for substitutions) and the putative interactions described above. Indeed, <a href="#pone-0001834-g010">figure 10B</a> shows an electrostatic interaction between E<sub>2</sub> and K<sub>14</sub> (blue and red shaded box); two hydrophobic patches, the first one at the edges of the assembly (A<sub>1</sub>-W<sub>13</sub> and F<sub>3</sub>-V<sub>11</sub>) and the second within the core of the assembly (W<sub>6</sub>-M<sub>10</sub>, which could also include an S-aromatic interaction) (green shaded boxes); cation-π interactions between R<sub>5</sub> and Y<sub>9</sub> (pink boxes); putative site for metal chelation between H<sub>4</sub> and H<sub>12</sub> (brown boxes); and polar-polar interaction between S<sub>7</sub> and S<sub>8</sub> (non boxed). All these interactions fit with the recent proposal of a steric-zipper forming the basic surface from which the β-sheet stacking occurs and the fibril elongates <a href="#pone.0001834-Sawaya1">[70]</a>. <a href="#pone-0001834-g010">Figure 10C</a> represents the formation of the β-sheets along the fiber axis and the putative quaternary interactions stabilizing and reinforcing such assembly. Such interactions during stacking of the strands within the β-sheet would be π-π between aromatic rings (Phe, Tyr and Trp residues) and metal chelation (His and Tyr residues). Our experiments were not able to ascertain the orientation of the strand edges (both edges ‘up’, or one ‘up’ and one ‘down’,) or of the strand faces (face-to-face or face-to-back, with the same or different faces adjacent to one another) during the stacking. The differences between the types of orientation would see the assignment to different classes of steric-zipper, according to the nomenclature of Sawaya et al., and the identity of the aromatic residues involved in the π-π interactions <a href="#pone.0001834-Sawaya1">[70]</a>. Class 2, 3, 6, 7 and 8 of steric zippers were ruled out due to the parallel arrangement of their β-sheets. We found that optimal π-π stacking, as represented and highlighted on <a href="#pone-0001834-g010">Figure 10C</a>, could be achieved only when AChE<sub>586-599</sub> β-sheets have the same sides facing each other (‘face-to-face’) and the orientation of the sheet edges facing up (‘up-up’). According to the nomenclature of Sawaya et al., this type of arrangement and orientation corresponds to a class 1 steric-zipper <a href="#pone.0001834-Sawaya1">[70]</a>. The strands within a β-sheet are stacking in a parallel arrangement, whereas the β-sheets are antiparallel. We noted that the only other class with similar side-chain interaction within the steric-zipper, class 5 (the strands within a β-sheet are stacking in an antiparallel arrangement; the β-sheets are antiparallel, and have the same sides facing each other, ‘face-to-face’), led to a model with poor π-π stacking (<a href="#pone-0001834-g011">Figure 11</a>). However, the orientation of the stacking remains to be determined.</p>
<div class="figure" id="pone-0001834-g011"><div class="img"><a name="pone-0001834-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001834" data-uri="info:doi/10.1371/journal.pone.0001834.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001834.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001834.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001834.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001834.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11.  <span>Model of quaternary interactions within β-sheets of AChE<sub>586-599</sub> within a fibril of class 5.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>Each β-sheet is represented with only 3 copies of AChE<sub>586-599</sub> for clarity: sheet 1 colored in different shades of green, sheet 2 in different shades of blue, and sheet 3 in different shades of red. On the carbon backbone structure, only the side chains of aromatic residues (F<sub>3</sub>, H<sub>4</sub>, W<sub>6</sub>, Y<sub>9</sub>, H<sub>12</sub> and W<sub>13</sub>) are represented for clarity. The boxes highlight possible aromatic interactions (π-π) between strands within a β-sheet. Within a β-sheet, AChE<sub>586-599</sub> strands are stacking in an antiparallel arrangement. Within AChE<sub>586-599</sub> fibril, the β-sheets are antiparallel, have the same sides facing each other (‘face-to-face’), however the orientation of the strand edges within a sheet alternate between up and down (‘up = down’). According to the nomenclature of Sawaya <em>et al</em>., this type of arrangement and orientation corresponds to a class 5 steric-zipper <a href="#pone.0001834-Sawaya1">[70]</a>.</p>
<span>doi:10.1371/journal.pone.0001834.g011</span></div><a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>In summary, the analysis of stabilizing or destabilizing effects of residue substitutions on the amyloid assembly of AChE<sub>586-599</sub> has provided evidence for the critical role of specific side-chain interactions in the stabilization of nascent aggregates and for the position dependence of these side-chains upon polymerization and fibril formation. Systematic dissections of the critical residues and interactions driving amyloid assembly, and of the chemical details underlying the molecular recognition process could provide invaluable information on such a poorly understood and complicated process. The benefits of such an understanding could be applied to the wider field of protein folding since an increasing number of non-pathogenic polypeptides have been shown to form amyloid fibrils under certain conditions <a href="#pone.0001834-Guijarro1">[71]</a>, <a href="#pone.0001834-Fandrich1">[72]</a>, <a href="#pone.0001834-Stefani1">[73]</a>, <a href="#pone.0001834-Uversky1">[74]</a>. Another important application would be in the biological and medical fields by helping in the design of synthetic molecules to prevent the critical interactions occurring during amyloidogenesis (e.g. capping peptides abolishing Aβ fibrilization and blocking of π-stacking interactions) <a href="#pone.0001834-Tjernberg1">[26]</a>, <a href="#pone.0001834-Kuner1">[27]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Synthetic peptides and antibodies</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants were prepared as described <a href="#pone.0001834-Cottingham2">[23]</a>. Specific mouse Mab (105A) anti-AChE<sub>586-599</sub> in a β-sheet conformation was previously described <a href="#pone.0001834-Cottingham2">[23]</a>.</p>


<h4>Preparation of amyloid oligomers</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants oligomers (12 µM) were covalently cross-linked by photo-activation using the photo-induced cross-linking of unlabelled proteins (PICUP) as previously described <a href="#pone.0001834-Jean1">[24]</a>. However, the light was filtered through a 450 nm UV filter and the reaction mixture was exposed to 5 flashes of light (xenon lamp).</p>


<h4>SDS-PAGE and Western-blot</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Amyloid oligomers were resolved on 10% Tris-Tricine SDS-PAGE and electro-blotted onto nitrocellulose. Nitrocellulose membranes were blocked with 5% (w/v) non-fat milk in PBS and incubated with the Mab 105A recognizing AChE<sub>586-599</sub> in β-sheet conformation, followed by anti-mouse IgG conjugated to horseradish peroxidase (HRP). Products were visualized by enhanced chemiluminescence.</p>


<h4>Circular dichroism</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>CD-spectra were recorded from 250 to 190 nm (far-UV) and from 320 to 240 nm (near-UV) at 20°C in a quartz cuvette (1 mm path length) using a Jasco J-720 spectropolarimeter. The mean spectra of multiple scans (scan speed of 50 nm min<sup>−1</sup> and response time 4 sec) were collected. The spectra were blank subtracted and normalized to molar ellipticity. At least three independent assays were performed and analyzed with the two-sample Student's t-test.</p>


<h4>Fibrilization experiments</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The assays were performed in a 96-well plate (black wall, clear bottom; Greiner, UK) with 165 µM ThT in PBS. ThT fluorescence (excitation 450 nm, emission 480 nm) was measured at 37°C every 6 min, with 5 min shaking after every measurement, on a BMG Polarstar plate reader. The values of buffer-ThT were subtracted from the values of peptide-ThT. At least three independent assays were performed and analyzed with the two-sample Student's t-test.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>To investigate the effect of ionic strength, the fibrilization experiment were carried as described above except that the peptides were incubated with 165 µM ThT in 1.8 mM KH<sub>2</sub>PO<sub>4</sub> and 10.1 mM NaH<sub>2</sub>PO<sub>4</sub> with varying concentration of NaCl (from 0 to 1.4 M) and KCl (from 0 to 27 mM).</p>


<h4>Surface tension measurement</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Analyses were performed in a 96-well plate format, as described <a href="#pone.0001834-Cottingham3">[29]</a>, <a href="#pone.0001834-Jean1">[24]</a>. Briefly, AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants were re-suspended in 80 µL 200 mM sodium acetate pH 3 and surface tension measured at 450 nm (BMG Polarstar plate reader) before and at various time points after neutralization (20 µL 1M NaH<sub>2</sub>PO<sub>4</sub>, pH7.2). ΔOD = (OD<sub>offset position</sub>–OD<sub>central position</sub>)<sub>neutral pH 2min</sub>–(OD<sub>offset position</sub>–OD<sub>central position</sub>)<sub>acidic pH</sub>. At least three independent assays were performed and analyzed with the two-sample Student's t-test.</p>


<h4>Electron microscopy</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>200 µM AChE<sub>586-599</sub> and AChE<sub>586-599</sub> mutants in 50 mM NaH<sub>2</sub>PO<sub>4</sub> pH 7.2 were incubated for 36 hours. Then the samples were adsorbed onto Formvar-coated 400 mesh copper grids, air dried, washed with distilled water, negatively stained with 2% aqueous uranyl acetate and viewed with a Zeiss Omega 912 microscope.</p>


<h4>Structural Model Building</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The fibrillar model was built with the DeepView program <a href="#pone.0001834-Guex1">[75]</a>. Starting with one β-strand aligned along the x-axis, a second β-strand, which is a copy of the first strand rotated by 180° along the z-axis, was placed next to the first strand along the y-axis separated by 10 Å. A third strand, an exact copy of the first strand is placed next to the second strand along the y-axis, again separated by 10 Å. Adjustment was made to the second and third strands to ensure that the side-chains on separate strands are intercalating with each other. These three strands constitute a layer or steric-zipper. For Class 1 fibril, the second and third layers were added by translating the original layer by 4.8 Å and 9.6 Å along the z-axis. For the Class 5 fibrils, the second layer was obtained by rotating the original layer by 180° along the y-axis and then by translation so that it laid on top of the original layer along the z-axis with a separation of 4.8 Å. The third layer was a translation of the first layer by 9.6 Å along the z-axis. Adjustments were made to the second and third layers to allow for the correct configuration for hydrogen bonding. Computations for energy minimization were done <em>in vacuo</em> using the GROMOS96 43B1 parameter set without reaction field, as implemented within Swiss-PdbViewer.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are indebted to W. Welsh and C. Davison for invaluable help and thank K Baker for sharing unpublished data. This work was supported by a research grant from Synaptica Ltd.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DV LJ. Performed the experiments: LJ MS. Analyzed the data: DV LJ CL. Contributed reagents/materials/analysis tools: MS CL. Wrote the paper: DV LJ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001834-Harper1" id="pone.0001834-Harper1"></a>Harper JD, Lansbury PT Jr (1997) Models of amyloid seeding in Alzheimer's disease and scrapie: mechanistic truths and physiological consequences of the time-dependent solubility of amyloid proteins. Annu Rev Biochem  66: 385–407.  <ul class="find" data-citedArticleID="1077714" data-doi="10.1146/annurev.biochem.66.1.385"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.66.1.385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Models+of+amyloid+seeding+in+Alzheimer%27s+disease+and+scrapie%3A+mechanistic+truths+and+physiological+consequences+of+the+time-dependent+solubility+of+amyloid+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Models+of+amyloid+seeding+in+Alzheimer%27s+disease+and+scrapie%3A+mechanistic+truths+and+physiological+consequences+of+the+time-dependent+solubility+of+amyloid+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001834-Westermark1" id="pone.0001834-Westermark1"></a>Westermark P (2005) Aspects on human amyloid forms and their fibril polypeptides. Febs J  272: 5942–5949.  <ul class="find" data-citedArticleID="1077806" data-doi="10.1111/j.1742-4658.2005.05024.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2005.05024.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aspects+on+human+amyloid+forms+and+their+fibril+polypeptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aspects+on+human+amyloid+forms+and+their+fibril+polypeptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001834-Klunk1" id="pone.0001834-Klunk1"></a>Klunk WE, Pettegrew JW, Abraham DJ (1989) Quantitative evaluation of congo red binding to amyloid-like proteins with a beta-pleated sheet conformation. J Histochem Cytochem  37: 1273–1281.  <ul class="find" data-citedArticleID="1077726" data-doi="10.1177/37.8.2666510"><li><a href="http://dx.doi.org/10.1177/37.8.2666510" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+evaluation+of+congo+red+binding+to+amyloid-like+proteins+with+a+beta-pleated+sheet+conformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+evaluation+of+congo+red+binding+to+amyloid-like+proteins+with+a+beta-pleated+sheet+conformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001834-LeVine1" id="pone.0001834-LeVine1"></a>LeVine H 3rd (1993) Thioflavine T interaction with synthetic Alzheimer's disease beta-amyloid peptides: detection of amyloid aggregation in solution. Protein Sci  2: 404–410.  <ul class="find" data-citedArticleID="1077732" data-doi="10.1002/pro.5560020312"><li><a href="http://dx.doi.org/10.1002/pro.5560020312" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thioflavine+T+interaction+with+synthetic+Alzheimer%27s+disease+beta-amyloid+peptides%3A+detection+of+amyloid+aggregation+in+solution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thioflavine+T+interaction+with+synthetic+Alzheimer%27s+disease+beta-amyloid+peptides%3A+detection+of+amyloid+aggregation+in+solution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001834-Rochet1" id="pone.0001834-Rochet1"></a>Rochet JC, Lansbury PT Jr (2000) Amyloid fibrillogenesis: themes and variations. Curr Opin Struct Biol  10: 60–68.  <ul class="find" data-citedArticleID="1077768" data-doi="10.1016/s0959-440x(99)00049-4"><li><a href="http://dx.doi.org/10.1016/s0959-440x(99)00049-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amyloid+fibrillogenesis%3A+themes+and+variations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amyloid+fibrillogenesis%3A+themes+and+variations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001834-Sipe1" id="pone.0001834-Sipe1"></a>Sipe JD, Cohen AS (2000) Review: history of the amyloid fibril. J Struct Biol  130: 88–98.  <ul class="find" data-citedArticleID="1077784" data-doi="10.1006/jsbi.2000.4221"><li><a href="http://dx.doi.org/10.1006/jsbi.2000.4221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Review%3A+history+of+the+amyloid+fibril." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Review%3A+history+of+the+amyloid+fibril.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001834-Smith1" id="pone.0001834-Smith1"></a>Smith CK, Regan L (1995) Guidelines for protein design: the energetics of beta sheet side chain interactions. Science  270: 980–982.  <ul class="find" data-citedArticleID="1077786" data-doi="10.1126/science.270.5238.980"><li><a href="http://dx.doi.org/10.1126/science.270.5238.980" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guidelines+for+protein+design%3A+the+energetics+of+beta+sheet+side+chain+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Guidelines+for+protein+design%3A+the+energetics+of+beta+sheet+side+chain+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001834-West1" id="pone.0001834-West1"></a>West MW, Wang W, Patterson J, Mancias JD, Beasley JR, et al.  (1999) De novo amyloid proteins from designed combinatorial libraries. Proc Natl Acad Sci U S A  96: 11211–11216.  <ul class="find" data-citedArticleID="1077804" data-doi="10.1073/pnas.96.20.11211"><li><a href="http://dx.doi.org/10.1073/pnas.96.20.11211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=De+novo+amyloid+proteins+from+designed+combinatorial+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22De+novo+amyloid+proteins+from+designed+combinatorial+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001834-Klimov1" id="pone.0001834-Klimov1"></a>Klimov DK, Thirumalai D (2003) Dissecting the assembly of Abeta16-22 amyloid peptides into antiparallel beta sheets. Structure  11: 295–307.  <ul class="find" data-citedArticleID="1077724" data-doi="10.1016/s0969-2126(03)00031-5"><li><a href="http://dx.doi.org/10.1016/s0969-2126(03)00031-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissecting+the+assembly+of+Abeta16-22+amyloid+peptides+into+antiparallel+beta+sheets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissecting+the+assembly+of+Abeta16-22+amyloid+peptides+into+antiparallel+beta+sheets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001834-Zanuy1" id="pone.0001834-Zanuy1"></a>Zanuy D, Haspel N, Tsai HH, Ma B, Gunasekaran K, et al.  (2004) Side chain interactions determine the amyloid organization: a single layer beta-sheet molecular structure of the calcitonin peptide segment 15-19. Phys Biol  1: 89–99.  <ul class="find" data-citedArticleID="1077820" data-doi="10.1088/1478-3967/1/2/005"><li><a href="http://dx.doi.org/10.1088/1478-3967/1/2/005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Side+chain+interactions+determine+the+amyloid+organization%3A+a+single+layer+beta-sheet+molecular+structure+of+the+calcitonin+peptide+segment+15-19." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Side+chain+interactions+determine+the+amyloid+organization%3A+a+single+layer+beta-sheet+molecular+structure+of+the+calcitonin+peptide+segment+15-19.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001834-Azriel1" id="pone.0001834-Azriel1"></a>Azriel R, Gazit E (2001) Analysis of the minimal amyloid-forming fragment of the islet amyloid polypeptide. An experimental support for the key role of the phenylalanine residue in amyloid formation. J Biol Chem  276: 34156–34161.  <ul class="find" data-citedArticleID="1077678" data-doi="10.1074/jbc.m102883200"><li><a href="http://dx.doi.org/10.1074/jbc.m102883200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+minimal+amyloid-forming+fragment+of+the+islet+amyloid+polypeptide.+An+experimental+support+for+the+key+role+of+the+phenylalanine+residue+in+amyloid+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+minimal+amyloid-forming+fragment+of+the+islet+amyloid+polypeptide.+An+experimental+support+for+the+key+role+of+the+phenylalanine+residue+in+amyloid+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001834-Massi1" id="pone.0001834-Massi1"></a>Massi F, Klimov D, Thirumalai D, Straub JE (2002) Charge states rather than propensity for beta-structure determine enhanced fibrillogenesis in wild-type Alzheimer's beta-amyloid peptide compared to E22Q Dutch mutant. Protein Sci  11: 1639–1647.  <ul class="find" data-citedArticleID="1077746" data-doi="10.1110/ps.3150102"><li><a href="http://dx.doi.org/10.1110/ps.3150102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charge+states+rather+than+propensity+for+beta-structure+determine+enhanced+fibrillogenesis+in+wild-type+Alzheimer%27s+beta-amyloid+peptide+compared+to+E22Q+Dutch+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Charge+states+rather+than+propensity+for+beta-structure+determine+enhanced+fibrillogenesis+in+wild-type+Alzheimer%27s+beta-amyloid+peptide+compared+to+E22Q+Dutch+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001834-Reches1" id="pone.0001834-Reches1"></a>Reches M, Porat Y, Gazit E (2002) Amyloid fibril formation by pentapeptide and tetrapeptide fragments of human calcitonin. J Biol Chem  277: 35475–35480.  <ul class="find" data-citedArticleID="1077764" data-doi="10.1074/jbc.m206039200"><li><a href="http://dx.doi.org/10.1074/jbc.m206039200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amyloid+fibril+formation+by+pentapeptide+and+tetrapeptide+fragments+of+human+calcitonin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amyloid+fibril+formation+by+pentapeptide+and+tetrapeptide+fragments+of+human+calcitonin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001834-Bitan1" id="pone.0001834-Bitan1"></a>Bitan G, Vollers SS, Teplow DB (2003) Elucidation of primary structure elements controlling early amyloid beta-protein oligomerization. J Biol Chem  278: 34882–34889.  <ul class="find" data-citedArticleID="1077682" data-doi="10.1074/jbc.m300825200"><li><a href="http://dx.doi.org/10.1074/jbc.m300825200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elucidation+of+primary+structure+elements+controlling+early+amyloid+beta-protein+oligomerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elucidation+of+primary+structure+elements+controlling+early+amyloid+beta-protein+oligomerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001834-Makin1" id="pone.0001834-Makin1"></a>Makin OS, Atkins E, Sikorski P, Johansson J, Serpell LC (2005) Molecular basis for amyloid fibril formation and stability. Proc Natl Acad Sci U S A  102: 315–320.  <ul class="find" data-citedArticleID="1077744" data-doi="10.1073/pnas.0406847102"><li><a href="http://dx.doi.org/10.1073/pnas.0406847102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+basis+for+amyloid+fibril+formation+and+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+basis+for+amyloid+fibril+formation+and+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001834-Haass1" id="pone.0001834-Haass1"></a>Haass C, Selkoe DJ (2007) Soluble protein oligomers in neurodegeneration: lessons from the Alzheimer's amyloid beta-peptide. Nat Rev Mol Cell Biol  8: 101–112.  <ul class="find" data-citedArticleID="1077712" data-doi="10.1038/nrm2101"><li><a href="http://dx.doi.org/10.1038/nrm2101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Soluble+protein+oligomers+in+neurodegeneration%3A+lessons+from+the+Alzheimer%27s+amyloid+beta-peptide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Soluble+protein+oligomers+in+neurodegeneration%3A+lessons+from+the+Alzheimer%27s+amyloid+beta-peptide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001834-Atwood1" id="pone.0001834-Atwood1"></a>Atwood CS, Martins RN, Smith MA, Perry G (2002) Senile plaque composition and posttranslational modification of amyloid-beta peptide and associated proteins. Peptides  23: 1343–1350.  <ul class="find" data-citedArticleID="1077674" data-doi="10.1016/s0196-9781(02)00070-0"><li><a href="http://dx.doi.org/10.1016/s0196-9781(02)00070-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Senile+plaque+composition+and+posttranslational+modification+of+amyloid-beta+peptide+and+associated+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Senile+plaque+composition+and+posttranslational+modification+of+amyloid-beta+peptide+and+associated+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001834-SaezValero1" id="pone.0001834-SaezValero1"></a>Saez-Valero J, Sberna G, McLean CA, Small DH (1999) Molecular isoform distribution and glycosylation of acetylcholinesterase are altered in brain and cerebrospinal fluid of patients with Alzheimer's disease. J Neurochem  72: 1600–1608.  <ul class="find" data-citedArticleID="1077770" data-doi="10.1046/j.1471-4159.1999.721600.x"><li><a href="http://dx.doi.org/10.1046/j.1471-4159.1999.721600.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+isoform+distribution+and+glycosylation+of+acetylcholinesterase+are+altered+in+brain+and+cerebrospinal+fluid+of+patients+with+Alzheimer%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+isoform+distribution+and+glycosylation+of+acetylcholinesterase+are+altered+in+brain+and+cerebrospinal+fluid+of+patients+with+Alzheimer%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001834-Rees1" id="pone.0001834-Rees1"></a>Rees TM, Berson A, Sklan EH, Younkin L, Younkin S, et al.  (2005) Memory deficits correlating with acetylcholinesterase splice shift and amyloid burden in doubly transgenic mice. Curr Alzheimer Res  2: 291–300.  <ul class="find" data-citedArticleID="1077766" data-doi="10.2174/1567205054367847"><li><a href="http://dx.doi.org/10.2174/1567205054367847" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Memory+deficits+correlating+with+acetylcholinesterase+splice+shift+and+amyloid+burden+in+doubly+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Memory+deficits+correlating+with+acetylcholinesterase+splice+shift+and+amyloid+burden+in+doubly+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001834-Diamant1" id="pone.0001834-Diamant1"></a>Diamant S, Podoly E, Friedler A, Ligumsky H, Livnah O, et al.  (2006) Butyrylcholinesterase attenuates amyloid fibril formation in vitro. Proc Natl Acad Sci U S A  103: 8628–8633.  <ul class="find" data-citedArticleID="1077698" data-doi="10.1073/pnas.0602922103"><li><a href="http://dx.doi.org/10.1073/pnas.0602922103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Butyrylcholinesterase+attenuates+amyloid+fibril+formation+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Butyrylcholinesterase+attenuates+amyloid+fibril+formation+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001834-Cottingham1" id="pone.0001834-Cottingham1"></a>Cottingham MG, Hollinshead MS, Vaux DJ (2002) Amyloid fibril formation by a synthetic peptide from a region of human acetylcholinesterase that is homologous to the Alzheimer's amyloid-beta peptide. Biochemistry  41: 13539–13547.  <ul class="find" data-citedArticleID="1077690" data-doi="10.1021/bi0260334"><li><a href="http://dx.doi.org/10.1021/bi0260334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amyloid+fibril+formation+by+a+synthetic+peptide+from+a+region+of+human+acetylcholinesterase+that+is+homologous+to+the+Alzheimer%27s+amyloid-beta+peptide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amyloid+fibril+formation+by+a+synthetic+peptide+from+a+region+of+human+acetylcholinesterase+that+is+homologous+to+the+Alzheimer%27s+amyloid-beta+peptide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001834-Yoon1" id="pone.0001834-Yoon1"></a>Yoon S, Welsh WJ (2004) Detecting hidden sequence propensity for amyloid fibril formation. Protein Sci  13: 2149–2160.  <ul class="find" data-citedArticleID="1077814" data-doi="10.1110/ps.04790604"><li><a href="http://dx.doi.org/10.1110/ps.04790604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+hidden+sequence+propensity+for+amyloid+fibril+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+hidden+sequence+propensity+for+amyloid+fibril+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001834-Cottingham2" id="pone.0001834-Cottingham2"></a>Cottingham MG, Voskuil JL, Vaux DJ (2003) The intact human acetylcholinesterase C-terminal oligomerization domain is alpha-helical in situ and in isolation, but a shorter fragment forms beta-sheet-rich amyloid fibrils and protofibrillar oligomers. Biochemistry  42: 10863–10873.  <ul class="find" data-citedArticleID="1077692" data-doi="10.1021/bi034768i"><li><a href="http://dx.doi.org/10.1021/bi034768i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+intact+human+acetylcholinesterase+C-terminal+oligomerization+domain+is+alpha-helical+in+situ+and+in+isolation%2C+but+a+shorter+fragment+forms+beta-sheet-rich+amyloid+fibrils+and+protofibrillar+oligomers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+intact+human+acetylcholinesterase+C-terminal+oligomerization+domain+is+alpha-helical+in+situ+and+in+isolation%2C+but+a+shorter+fragment+forms+beta-sheet-rich+amyloid+fibrils+and+protofibrillar+oligomers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001834-Jean1" id="pone.0001834-Jean1"></a>Jean L, Thomas B, Tahiri-Alaoui A, Shaw M, Vaux DJ (2007) Heterologous amyloid seeding: revisiting the role of acetylcholinesterase in Alzheimer's disease. PLoS ONE  2: e652.  <ul class="find" data-citedArticleID="1077718" data-doi="10.1371/journal.pone.0000652"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000652" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterologous+amyloid+seeding%3A+revisiting+the+role+of+acetylcholinesterase+in+Alzheimer%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterologous+amyloid+seeding%3A+revisiting+the+role+of+acetylcholinesterase+in+Alzheimer%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001834-Jack1" id="pone.0001834-Jack1"></a>Jack E, Newsome M, Stockley PG, Radford SE, Middleton DA (2006) The organization of aromatic side groups in an amyloid fibril probed by solid-state 2H and 19F NMR spectroscopy. J Am Chem Soc  128: 8098–8099.  <ul class="find" data-citedArticleID="1077716" data-doi="10.1021/ja0581898"><li><a href="http://dx.doi.org/10.1021/ja0581898" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+organization+of+aromatic+side+groups+in+an+amyloid+fibril+probed+by+solid-state+2H+and+19F+NMR+spectroscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+organization+of+aromatic+side+groups+in+an+amyloid+fibril+probed+by+solid-state+2H+and+19F+NMR+spectroscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001834-Tjernberg1" id="pone.0001834-Tjernberg1"></a>Tjernberg LO, Naslund J, Lindqvist F, Johansson J, Karlstrom AR, et al.  (1996) Arrest of beta-amyloid fibril formation by a pentapeptide ligand. J Biol Chem  271: 8545–8548.  <ul class="find" data-citedArticleID="1077796" data-doi="10.1074/jbc.271.15.8545"><li><a href="http://dx.doi.org/10.1074/jbc.271.15.8545" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arrest+of+beta-amyloid+fibril+formation+by+a+pentapeptide+ligand." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arrest+of+beta-amyloid+fibril+formation+by+a+pentapeptide+ligand.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001834-Kuner1" id="pone.0001834-Kuner1"></a>Kuner P, Bohrmann B, Tjernberg LO, Naslund J, Huber G, et al.  (2000) Controlling polymerization of beta-amyloid and prion-derived peptides with synthetic small molecule ligands. J Biol Chem  275: 1673–1678.  <ul class="find" data-citedArticleID="1077728" data-doi="10.1074/jbc.275.3.1673"><li><a href="http://dx.doi.org/10.1074/jbc.275.3.1673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+polymerization+of+beta-amyloid+and+prion-derived+peptides+with+synthetic+small+molecule+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+polymerization+of+beta-amyloid+and+prion-derived+peptides+with+synthetic+small+molecule+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001834-Soreghan1" id="pone.0001834-Soreghan1"></a>Soreghan B, Kosmoski J, Glabe C (1994) Surfactant properties of Alzheimer's A beta peptides and the mechanism of amyloid aggregation. J Biol Chem  269: 28551–28554.  <ul class="find" data-citedArticleID="1077788"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Surfactant+properties+of+Alzheimer%27s+A+beta+peptides+and+the+mechanism+of+amyloid+aggregation.&amp;auth=&amp;atitle=Surfactant+properties+of+Alzheimer%27s+A+beta+peptides+and+the+mechanism+of+amyloid+aggregation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surfactant+properties+of+Alzheimer%27s+A+beta+peptides+and+the+mechanism+of+amyloid+aggregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surfactant+properties+of+Alzheimer%27s+A+beta+peptides+and+the+mechanism+of+amyloid+aggregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001834-Cottingham3" id="pone.0001834-Cottingham3"></a>Cottingham MG, Bain CD, Vaux DJ (2004) Rapid method for measurement of surface tension in multiwell plates. Lab Invest  84: 523–529.  <ul class="find" data-citedArticleID="1077694" data-doi="10.1038/labinvest.3700054"><li><a href="http://dx.doi.org/10.1038/labinvest.3700054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+method+for+measurement+of+surface+tension+in+multiwell+plates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+method+for+measurement+of+surface+tension+in+multiwell+plates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001834-Fancy1" id="pone.0001834-Fancy1"></a>Fancy DA, Kodadek T (1999) Chemistry for the analysis of protein-protein interactions: rapid and efficient cross-linking triggered by long wavelength light. Proc Natl Acad Sci U S A  96: 6020–6024.  <ul class="find" data-citedArticleID="1077700" data-doi="10.1073/pnas.96.11.6020"><li><a href="http://dx.doi.org/10.1073/pnas.96.11.6020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemistry+for+the+analysis+of+protein-protein+interactions%3A+rapid+and+efficient+cross-linking+triggered+by+long+wavelength+light." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemistry+for+the+analysis+of+protein-protein+interactions%3A+rapid+and+efficient+cross-linking+triggered+by+long+wavelength+light.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001834-Seilheimer1" id="pone.0001834-Seilheimer1"></a>Seilheimer B, Bohrmann B, Bondolfi L, Muller F, Stuber D, et al.  (1997) The toxicity of the Alzheimer's beta-amyloid peptide correlates with a distinct fiber morphology. J Struct Biol  119: 59–71.  <ul class="find" data-citedArticleID="1077778" data-doi="10.1006/jsbi.1997.3859"><li><a href="http://dx.doi.org/10.1006/jsbi.1997.3859" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+toxicity+of+the+Alzheimer%27s+beta-amyloid+peptide+correlates+with+a+distinct+fiber+morphology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+toxicity+of+the+Alzheimer%27s+beta-amyloid+peptide+correlates+with+a+distinct+fiber+morphology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001834-Lashuel1" id="pone.0001834-Lashuel1"></a>Lashuel HA, Petre BM, Wall J, Simon M, Nowak RJ, et al.  (2002) Alpha-synuclein, especially the Parkinson's disease-associated mutants, forms pore-like annular and tubular protofibrils. J Mol Biol  322: 1089–1102.  <ul class="find" data-citedArticleID="1077730" data-doi="10.1016/s0022-2836(02)00735-0"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00735-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alpha-synuclein%2C+especially+the+Parkinson%27s+disease-associated+mutants%2C+forms+pore-like+annular+and+tubular+protofibrils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alpha-synuclein%2C+especially+the+Parkinson%27s+disease-associated+mutants%2C+forms+pore-like+annular+and+tubular+protofibrils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001834-LopezdelaPaz1" id="pone.0001834-LopezdelaPaz1"></a>Lopez de la Paz M, Serrano L (2004) Sequence determinants of amyloid fibril formation. Proc Natl Acad Sci U S A  101: 87–92.  <ul class="find" data-citedArticleID="1077738" data-doi="10.1073/pnas.2634884100"><li><a href="http://dx.doi.org/10.1073/pnas.2634884100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+determinants+of+amyloid+fibril+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+determinants+of+amyloid+fibril+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001834-LopezdelaPaz2" id="pone.0001834-LopezdelaPaz2"></a>Lopez de la Paz M, de Mori GM, Serrano L, Colombo G (2005) Sequence dependence of amyloid fibril formation: insights from molecular dynamics simulations. J Mol Biol  349: 583–596.  <ul class="find" data-citedArticleID="1077740" data-doi="10.1016/j.jmb.2005.03.081"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.03.081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+dependence+of+amyloid+fibril+formation%3A+insights+from+molecular+dynamics+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+dependence+of+amyloid+fibril+formation%3A+insights+from+molecular+dynamics+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001834-Petkova1" id="pone.0001834-Petkova1"></a>Petkova AT, Ishii Y, Balbach JJ, Antzutkin ON, Leapman RD, et al.  (2002) A structural model for Alzheimer's beta -amyloid fibrils based on experimental constraints from solid state NMR. Proc Natl Acad Sci U S A  99: 16742–16747.  <ul class="find" data-citedArticleID="1077760" data-doi="10.1073/pnas.262663499"><li><a href="http://dx.doi.org/10.1073/pnas.262663499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+structural+model+for+Alzheimer%27s+beta+-amyloid+fibrils+based+on+experimental+constraints+from+solid+state+NMR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+structural+model+for+Alzheimer%27s+beta+-amyloid+fibrils+based+on+experimental+constraints+from+solid+state+NMR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001834-Tjernberg2" id="pone.0001834-Tjernberg2"></a>Tjernberg L, Hosia W, Bark N, Thyberg J, Johansson J (2002) Charge attraction and beta propensity are necessary for amyloid fibril formation from tetrapeptides. J Biol Chem  277: 43243–43246.  <ul class="find" data-citedArticleID="1077798" data-doi="10.1074/jbc.m205570200"><li><a href="http://dx.doi.org/10.1074/jbc.m205570200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charge+attraction+and+beta+propensity+are+necessary+for+amyloid+fibril+formation+from+tetrapeptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Charge+attraction+and+beta+propensity+are+necessary+for+amyloid+fibril+formation+from+tetrapeptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001834-Tarus1" id="pone.0001834-Tarus1"></a>Tarus B, Straub JE, Thirumalai D (2005) Probing the initial stage of aggregation of the Abeta(10-35)-protein: assessing the propensity for peptide dimerization. J Mol Biol  345: 1141–1156.  <ul class="find" data-citedArticleID="1077792" data-doi="10.1016/j.jmb.2004.11.022"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.11.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+the+initial+stage+of+aggregation+of+the+Abeta%2810-35%29-protein%3A+assessing+the+propensity+for+peptide+dimerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+the+initial+stage+of+aggregation+of+the+Abeta%2810-35%29-protein%3A+assessing+the+propensity+for+peptide+dimerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001834-Zhang1" id="pone.0001834-Zhang1"></a>Zhang S, Holmes T, Lockshin C, Rich A (1993) Spontaneous assembly of a self-complementary oligopeptide to form a stable macroscopic membrane. Proc Natl Acad Sci U S A  90: 3334–3338.  <ul class="find" data-citedArticleID="1077822" data-doi="10.1073/pnas.90.8.3334"><li><a href="http://dx.doi.org/10.1073/pnas.90.8.3334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spontaneous+assembly+of+a+self-complementary+oligopeptide+to+form+a+stable+macroscopic+membrane." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spontaneous+assembly+of+a+self-complementary+oligopeptide+to+form+a+stable+macroscopic+membrane.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001834-Wouters1" id="pone.0001834-Wouters1"></a>Wouters MA, Curmi PM (1995) An analysis of side chain interactions and pair correlations within antiparallel beta-sheets: the differences between backbone hydrogen-bonded and non-hydrogen-bonded residue pairs. Proteins  22: 119–131.  <ul class="find" data-citedArticleID="1077808" data-doi="10.1002/prot.340220205"><li><a href="http://dx.doi.org/10.1002/prot.340220205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+analysis+of+side+chain+interactions+and+pair+correlations+within+antiparallel+beta-sheets%3A+the+differences+between+backbone+hydrogen-bonded+and+non-hydrogen-bonded+residue+pairs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+analysis+of+side+chain+interactions+and+pair+correlations+within+antiparallel+beta-sheets%3A+the+differences+between+backbone+hydrogen-bonded+and+non-hydrogen-bonded+residue+pairs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001834-Nilsberth1" id="pone.0001834-Nilsberth1"></a>Nilsberth C, Westlind-Danielsson A, Eckman CB, Condron MM, Axelman K, et al.  (2001) The ‘Arctic’ APP mutation (E693G) causes Alzheimer's disease by enhanced Abeta protofibril formation. Nat Neurosci  4: 887–893.  <ul class="find" data-citedArticleID="1077754" data-doi="10.1038/nn0901-887"><li><a href="http://dx.doi.org/10.1038/nn0901-887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+%E2%80%98Arctic%E2%80%99+APP+mutation+%28E693G%29+causes+Alzheimer%27s+disease+by+enhanced+Abeta+protofibril+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+%E2%80%98Arctic%E2%80%99+APP+mutation+%28E693G%29+causes+Alzheimer%27s+disease+by+enhanced+Abeta+protofibril+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001834-Tenidis1" id="pone.0001834-Tenidis1"></a>Tenidis K, Waldner M, Bernhagen J, Fischle W, Bergmann M, et al.  (2000) Identification of a penta- and hexapeptide of islet amyloid polypeptide (IAPP) with amyloidogenic and cytotoxic properties. J Mol Biol  295: 1055–1071.  <ul class="find" data-citedArticleID="1077794" data-doi="10.1006/jmbi.1999.3422"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3422" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+penta-+and+hexapeptide+of+islet+amyloid+polypeptide+%28IAPP%29+with+amyloidogenic+and+cytotoxic+properties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+penta-+and+hexapeptide+of+islet+amyloid+polypeptide+%28IAPP%29+with+amyloidogenic+and+cytotoxic+properties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001834-Chiti1" id="pone.0001834-Chiti1"></a>Chiti F, Stefani M, Taddei N, Ramponi G, Dobson CM (2003) Rationalization of the effects of mutations on peptide and protein aggregation rates. Nature  424: 805–808.  <ul class="find" data-citedArticleID="1077686" data-doi="10.1038/nature01891"><li><a href="http://dx.doi.org/10.1038/nature01891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rationalization+of+the+effects+of+mutations+on+peptide+and+protein+aggregation+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rationalization+of+the+effects+of+mutations+on+peptide+and+protein+aggregation+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001834-Gazit1" id="pone.0001834-Gazit1"></a>Gazit E (2002) A possible role for pi-stacking in the self-assembly of amyloid fibrils. Faseb J  16: 77–83.  <ul class="find" data-citedArticleID="1077704" data-doi="10.1096/fj.01-0442hyp"><li><a href="http://dx.doi.org/10.1096/fj.01-0442hyp" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+possible+role+for+pi-stacking+in+the+self-assembly+of+amyloid+fibrils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+possible+role+for+pi-stacking+in+the+self-assembly+of+amyloid+fibrils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001834-Claessens1" id="pone.0001834-Claessens1"></a>Claessens CG, Stoddart JF (1997) p-p interactions in self-assembly. J Phys Org Chem  10: 254–272.  <ul class="find" data-citedArticleID="1077688" data-doi="10.1002/(sici)1099-1395(199705)10:5<254::aid-poc875&gt;3.0.co;2-3"><li><a href="http://dx.doi.org/10.1002/(sici)1099-1395(199705)10:5<254::aid-poc875&gt;3.0.co;2-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p-p+interactions+in+self-assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p-p+interactions+in+self-assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001834-Gillard1" id="pone.0001834-Gillard1"></a>Gillard RE, Raymo FM, Stoddart JF (1997) Controlling self-assembly. Chem Eur J  3: 1933–1940.  <ul class="find" data-citedArticleID="1077706" data-doi="10.1002/chem.19970031208"><li><a href="http://dx.doi.org/10.1002/chem.19970031208" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+self-assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+self-assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001834-Waters1" id="pone.0001834-Waters1"></a>Waters ML (2002) Aromatic interactions in model systems. Curr Opin Chem Biol  6: 736–741.  <ul class="find" data-citedArticleID="1077802"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Aromatic+interactions+in+model+systems.&amp;auth=&amp;atitle=Aromatic+interactions+in+model+systems." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aromatic+interactions+in+model+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aromatic+interactions+in+model+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001834-Wu1" id="pone.0001834-Wu1"></a>Wu C, Lei H, Duan Y (2005) The role of Phe in the formation of well-ordered oligomers of amyloidogenic hexapeptide (NFGAIL) observed in molecular dynamics simulations with explicit solvent. Biophys J  88: 2897–2906.  <ul class="find" data-citedArticleID="1077810" data-doi="10.1529/biophysj.104.055574"><li><a href="http://dx.doi.org/10.1529/biophysj.104.055574" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+Phe+in+the+formation+of+well-ordered+oligomers+of+amyloidogenic+hexapeptide+%28NFGAIL%29+observed+in+molecular+dynamics+simulations+with+explicit+solvent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+Phe+in+the+formation+of+well-ordered+oligomers+of+amyloidogenic+hexapeptide+%28NFGAIL%29+observed+in+molecular+dynamics+simulations+with+explicit+solvent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001834-Bemporad1" id="pone.0001834-Bemporad1"></a>Bemporad F, Taddei N, Stefani M, Chiti F (2006) Assessing the role of aromatic residues in the amyloid aggregation of human muscle acylphosphatase. Protein Sci  15: 862–870.  <ul class="find" data-citedArticleID="1077680" data-doi="10.1110/ps.051915806"><li><a href="http://dx.doi.org/10.1110/ps.051915806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+the+role+of+aromatic+residues+in+the+amyloid+aggregation+of+human+muscle+acylphosphatase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+the+role+of+aromatic+residues+in+the+amyloid+aggregation+of+human+muscle+acylphosphatase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001834-Sikorski1" id="pone.0001834-Sikorski1"></a>Sikorski P, Atkins ED, Serpell LC (2003) Structure and texture of fibrous crystals formed by Alzheimer's abeta(11-25) peptide fragment. Structure  11: 915–926.  <ul class="find" data-citedArticleID="1077782" data-doi="10.1016/s0969-2126(03)00149-7"><li><a href="http://dx.doi.org/10.1016/s0969-2126(03)00149-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+texture+of+fibrous+crystals+formed+by+Alzheimer%27s+abeta%2811-25%29+peptide+fragment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+texture+of+fibrous+crystals+formed+by+Alzheimer%27s+abeta%2811-25%29+peptide+fragment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001834-Naito1" id="pone.0001834-Naito1"></a>Naito A, Kamihira M, Inoue R, Saito H (2004) Structural diversity of amyloid fibril formed in human calcitonin as revealed by site-directed 13C solid-state NMR spectroscopy. Magn Reson Chem  42: 247–257.  <ul class="find" data-citedArticleID="1077752" data-doi="10.1002/mrc.1323"><li><a href="http://dx.doi.org/10.1002/mrc.1323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+diversity+of+amyloid+fibril+formed+in+human+calcitonin+as+revealed+by+site-directed+13C+solid-state+NMR+spectroscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+diversity+of+amyloid+fibril+formed+in+human+calcitonin+as+revealed+by+site-directed+13C+solid-state+NMR+spectroscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001834-Shetty1" id="pone.0001834-Shetty1"></a>Shetty AS, Zhang J, Moore JS (1996) Aromatic -p Stacking in Solution as Revealed through the Aggregation of Phenylacetylene Macrocycles. J Am Chem Soc  118: 1019–1027.  <ul class="find" data-citedArticleID="1077780" data-doi="10.1021/ja9528893"><li><a href="http://dx.doi.org/10.1021/ja9528893" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aromatic+-p+Stacking+in+Solution+as+Revealed+through+the+Aggregation+of+Phenylacetylene+Macrocycles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aromatic+-p+Stacking+in+Solution+as+Revealed+through+the+Aggregation+of+Phenylacetylene+Macrocycles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001834-McGaughey1" id="pone.0001834-McGaughey1"></a>McGaughey GB, Gagne M, Rappe AK (1998) pi-Stacking interactions. Alive and well in proteins. J Biol Chem  273: 15458–15463.  <ul class="find" data-citedArticleID="1077748" data-doi="10.1074/jbc.273.25.15458"><li><a href="http://dx.doi.org/10.1074/jbc.273.25.15458" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=pi-Stacking+interactions.+Alive+and+well+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22pi-Stacking+interactions.+Alive+and+well+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001834-Lin1" id="pone.0001834-Lin1"></a>Lin TY, Timasheff SN (1996) On the role of surface tension in the stabilization of globular proteins. Protein Sci  5: 372–381.  <ul class="find" data-citedArticleID="1077734" data-doi="10.1002/pro.5560050222"><li><a href="http://dx.doi.org/10.1002/pro.5560050222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+role+of+surface+tension+in+the+stabilization+of+globular+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+role+of+surface+tension+in+the+stabilization+of+globular+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001834-Crowley1" id="pone.0001834-Crowley1"></a>Crowley PB, Golovin A (2005) Cation-pi interactions in protein-protein interfaces. Proteins  59: 231–239.  <ul class="find" data-citedArticleID="1077696" data-doi="10.1002/prot.20417"><li><a href="http://dx.doi.org/10.1002/prot.20417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cation-pi+interactions+in+protein-protein+interfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cation-pi+interactions+in+protein-protein+interfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001834-Paddock1" id="pone.0001834-Paddock1"></a>Paddock ML, Weber KH, Chang C, Okamura MY (2005) Interactions between cytochrome c2 and the photosynthetic reaction center from Rhodobacter sphaeroides: the cation-pi interaction. Biochemistry  44: 9619–9625.  <ul class="find" data-citedArticleID="1077756" data-doi="10.1021/bi050651d"><li><a href="http://dx.doi.org/10.1021/bi050651d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactions+between+cytochrome+c2+and+the+photosynthetic+reaction+center+from+Rhodobacter+sphaeroides%3A+the+cation-pi+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactions+between+cytochrome+c2+and+the+photosynthetic+reaction+center+from+Rhodobacter+sphaeroides%3A+the+cation-pi+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001834-Ma1" id="pone.0001834-Ma1"></a>Ma JC, Dougherty DA (1997) The Cationminus signpi Interaction. Chem Rev  97: 1303–1324.  <ul class="find" data-citedArticleID="1077742" data-doi="10.1021/cr9603744"><li><a href="http://dx.doi.org/10.1021/cr9603744" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Cationminus+signpi+Interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Cationminus+signpi+Interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001834-Chakkaravarthi1" id="pone.0001834-Chakkaravarthi1"></a>Chakkaravarthi S, Gromiha MM (2006) Analysis of cation π interactions to the structural stability of RNA binding proteins. Polymer  47: 709–721.  <ul class="find" data-citedArticleID="1077684" data-doi="10.1016/j.polymer.2005.11.059"><li><a href="http://dx.doi.org/10.1016/j.polymer.2005.11.059" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+cation+%CF%80+interactions+to+the+structural+stability+of+RNA+binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+cation+%CF%80+interactions+to+the+structural+stability+of+RNA+binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001834-Yoshida1" id="pone.0001834-Yoshida1"></a>Yoshida H, Matsushima N, Kumaki Y, Nakata M, Hikichi K (2000) NMR studies of model peptides of PHGGGWGQ repeats within the N-terminus of prion proteins: a loop conformation with histidine and tryptophan in close proximity. J Biochem (Tokyo)  128: 271–281.  <ul class="find" data-citedArticleID="1077816" data-doi="10.1093/oxfordjournals.jbchem.a022750"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a022750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NMR+studies+of+model+peptides+of+PHGGGWGQ+repeats+within+the+N-terminus+of+prion+proteins%3A+a+loop+conformation+with+histidine+and+tryptophan+in+close+proximity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NMR+studies+of+model+peptides+of+PHGGGWGQ+repeats+within+the+N-terminus+of+prion+proteins%3A+a+loop+conformation+with+histidine+and+tryptophan+in+close+proximity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001834-Zahn1" id="pone.0001834-Zahn1"></a>Zahn R (2003) The octapeptide repeats in mammalian prion protein constitute a pH-dependent folding and aggregation site. J Mol Biol  334: 477–488.  <ul class="find" data-citedArticleID="1077818" data-doi="10.1016/j.jmb.2003.09.048"><li><a href="http://dx.doi.org/10.1016/j.jmb.2003.09.048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+octapeptide+repeats+in+mammalian+prion+protein+constitute+a+pH-dependent+folding+and+aggregation+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+octapeptide+repeats+in+mammalian+prion+protein+constitute+a+pH-dependent+folding+and+aggregation+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001834-Pal1" id="pone.0001834-Pal1"></a>Pal D, Chakrabarti P (2001) Non-hydrogen bond interactions involving the methionine sulfur atom. J Biomol Struct Dyn  19: 115–128.  <ul class="find" data-citedArticleID="1077758" data-doi="10.1080/07391102.2001.10506725"><li><a href="http://dx.doi.org/10.1080/07391102.2001.10506725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-hydrogen+bond+interactions+involving+the+methionine+sulfur+atom." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-hydrogen+bond+interactions+involving+the+methionine+sulfur+atom.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001834-Samanta1" id="pone.0001834-Samanta1"></a>Samanta U, Pal D, Chakrabarti P (2000) Environment of tryptophan side chains in proteins. Proteins  38: 288–300.  <ul class="find" data-citedArticleID="1077774" data-doi="10.1002/(sici)1097-0134(20000215)38:3<288::aid-prot5&gt;3.0.co;2-7"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(20000215)38:3<288::aid-prot5&gt;3.0.co;2-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environment+of+tryptophan+side+chains+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environment+of+tryptophan+side+chains+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001834-Petkova2" id="pone.0001834-Petkova2"></a>Petkova AT, Yau WM, Tycko R (2006) Experimental constraints on quaternary structure in Alzheimer's beta-amyloid fibrils. Biochemistry  45: 498–512.  <ul class="find" data-citedArticleID="1077762" data-doi="10.1021/bi051952q"><li><a href="http://dx.doi.org/10.1021/bi051952q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+constraints+on+quaternary+structure+in+Alzheimer%27s+beta-amyloid+fibrils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+constraints+on+quaternary+structure+in+Alzheimer%27s+beta-amyloid+fibrils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001834-Kellis1" id="pone.0001834-Kellis1"></a>Kellis JT Jr, Todd RJ, Arnold FH (1991) Protein stabilization by engineered metal chelation. Biotechnology (N Y)  9: 994–995.  <ul class="find" data-citedArticleID="1077722" data-doi="10.1038/nbt1091-994"><li><a href="http://dx.doi.org/10.1038/nbt1091-994" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+stabilization+by+engineered+metal+chelation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+stabilization+by+engineered+metal+chelation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001834-Atwood2" id="pone.0001834-Atwood2"></a>Atwood CS, Moir RD, Huang X, Scarpa RC, Bacarra NM, et al.  (1998) Dramatic aggregation of Alzheimer abeta by Cu(II) is induced by conditions representing physiological acidosis. J Biol Chem  273: 12817–12826.  <ul class="find" data-citedArticleID="1077676" data-doi="10.1074/jbc.273.21.12817"><li><a href="http://dx.doi.org/10.1074/jbc.273.21.12817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dramatic+aggregation+of+Alzheimer+abeta+by+Cu%28II%29+is+induced+by+conditions+representing+physiological+acidosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dramatic+aggregation+of+Alzheimer+abeta+by+Cu%28II%29+is+induced+by+conditions+representing+physiological+acidosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001834-Liu1" id="pone.0001834-Liu1"></a>Liu ST, Howlett G, Barrow CJ (1999) Histidine-13 is a crucial residue in the zinc ion-induced aggregation of the A beta peptide of Alzheimer's disease. Biochemistry  38: 9373–9378.  <ul class="find" data-citedArticleID="1077736" data-doi="10.1021/bi990205o"><li><a href="http://dx.doi.org/10.1021/bi990205o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histidine-13+is+a+crucial+residue+in+the+zinc+ion-induced+aggregation+of+the+A+beta+peptide+of+Alzheimer%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histidine-13+is+a+crucial+residue+in+the+zinc+ion-induced+aggregation+of+the+A+beta+peptide+of+Alzheimer%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001834-Miura1" id="pone.0001834-Miura1"></a>Miura T, Suzuki K, Kohata N, Takeuchi H (2000) Metal binding modes of Alzheimer's amyloid beta-peptide in insoluble aggregates and soluble complexes. Biochemistry  39: 7024–7031.  <ul class="find" data-citedArticleID="1077750" data-doi="10.1021/bi0002479"><li><a href="http://dx.doi.org/10.1021/bi0002479" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metal+binding+modes+of+Alzheimer%27s+amyloid+beta-peptide+in+insoluble+aggregates+and+soluble+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metal+binding+modes+of+Alzheimer%27s+amyloid+beta-peptide+in+insoluble+aggregates+and+soluble+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001834-Yang1" id="pone.0001834-Yang1"></a>Yang DS, McLaurin J, Qin K, Westaway D, Fraser PE (2000) Examining the zinc binding site of the amyloid-beta peptide. Eur J Biochem  267: 6692–6698.  <ul class="find" data-citedArticleID="1077812" data-doi="10.1046/j.1432-1327.2000.01767.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1327.2000.01767.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Examining+the+zinc+binding+site+of+the+amyloid-beta+peptide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Examining+the+zinc+binding+site+of+the+amyloid-beta+peptide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001834-Jones1" id="pone.0001834-Jones1"></a>Jones CE, Abdelraheim SR, Brown DR, Viles JH (2004) Preferential Cu2+ coordination by His96 and His111 induces beta-sheet formation in the unstructured amyloidogenic region of the prion protein. J Biol Chem  279: 32018–32027.  <ul class="find" data-citedArticleID="1077720" data-doi="10.1074/jbc.m403467200"><li><a href="http://dx.doi.org/10.1074/jbc.m403467200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preferential+Cu2%2B+coordination+by+His96+and+His111+induces+beta-sheet+formation+in+the+unstructured+amyloidogenic+region+of+the+prion+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preferential+Cu2%2B+coordination+by+His96+and+His111+induces+beta-sheet+formation+in+the+unstructured+amyloidogenic+region+of+the+prion+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001834-Salemme1" id="pone.0001834-Salemme1"></a>Salemme FR, Weatherford DW (1981) Conformational and geometrical properties of beta-sheets in proteins. I. Parallel beta-sheets. J Mol Biol  146: 101–117.  <ul class="find" data-citedArticleID="1077772" data-doi="10.1016/0022-2836(81)90368-5"><li><a href="http://dx.doi.org/10.1016/0022-2836(81)90368-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+and+geometrical+properties+of+beta-sheets+in+proteins.+I.+Parallel+beta-sheets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+and+geometrical+properties+of+beta-sheets+in+proteins.+I.+Parallel+beta-sheets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001834-Sawaya1" id="pone.0001834-Sawaya1"></a>Sawaya MR, Sambashivan S, Nelson R, Ivanova MI, Sievers SA, et al.  (2007) Atomic structures of amyloid cross-beta spines reveal varied steric zippers. Nature  447: 453–457.  <ul class="find" data-citedArticleID="1077776" data-doi="10.1038/nature05695"><li><a href="http://dx.doi.org/10.1038/nature05695" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atomic+structures+of+amyloid+cross-beta+spines+reveal+varied+steric+zippers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atomic+structures+of+amyloid+cross-beta+spines+reveal+varied+steric+zippers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001834-Guijarro1" id="pone.0001834-Guijarro1"></a>Guijarro JI, Sunde M, Jones JA, Campbell ID, Dobson CM (1998) Amyloid fibril formation by an SH3 domain. Proc Natl Acad Sci U S A  95: 4224–4228.  <ul class="find" data-citedArticleID="1077710" data-doi="10.1073/pnas.95.8.4224"><li><a href="http://dx.doi.org/10.1073/pnas.95.8.4224" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amyloid+fibril+formation+by+an+SH3+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amyloid+fibril+formation+by+an+SH3+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001834-Fandrich1" id="pone.0001834-Fandrich1"></a>Fandrich M, Fletcher MA, Dobson CM (2001) Amyloid fibrils from muscle myoglobin. Nature  410: 165–166.  <ul class="find" data-citedArticleID="1077702" data-doi="10.1038/35065514"><li><a href="http://dx.doi.org/10.1038/35065514" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amyloid+fibrils+from+muscle+myoglobin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amyloid+fibrils+from+muscle+myoglobin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001834-Stefani1" id="pone.0001834-Stefani1"></a>Stefani M, Dobson CM (2003) Protein aggregation and aggregate toxicity: new insights into protein folding, misfolding diseases and biological evolution. J Mol Med  81: 678–699.  <ul class="find" data-citedArticleID="1077790" data-doi="10.1007/s00109-003-0464-5"><li><a href="http://dx.doi.org/10.1007/s00109-003-0464-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+aggregation+and+aggregate+toxicity%3A+new+insights+into+protein+folding%2C+misfolding+diseases+and+biological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+aggregation+and+aggregate+toxicity%3A+new+insights+into+protein+folding%2C+misfolding+diseases+and+biological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001834-Uversky1" id="pone.0001834-Uversky1"></a>Uversky VN, Fink AL (2004) Conformational constraints for amyloid fibrillation: the importance of being unfolded. Biochim Biophys Acta  1698: 131–153.  <ul class="find" data-citedArticleID="1077800" data-doi="10.1016/j.bbapap.2003.12.008"><li><a href="http://dx.doi.org/10.1016/j.bbapap.2003.12.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+constraints+for+amyloid+fibrillation%3A+the+importance+of+being+unfolded." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+constraints+for+amyloid+fibrillation%3A+the+importance+of+being+unfolded.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001834-Guex1" id="pone.0001834-Guex1"></a>Guex N, Peitsch MC (1997) SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling. Electrophoresis  18: 2714–2723.  <ul class="find" data-citedArticleID="1077708" data-doi="10.1002/elps.1150181505"><li><a href="http://dx.doi.org/10.1002/elps.1150181505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SWISS-MODEL+and+the+Swiss-PdbViewer%3A+an+environment+for+comparative+protein+modeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SWISS-MODEL+and+the+Swiss-PdbViewer%3A+an+environment+for+comparative+protein+modeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.XML" value="162012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.PDF" value="1844057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g001.PNG_L" value="466671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g001.PNG_M" value="113964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g001.PNG_S" value="16054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g001.TIF" value="1244194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g001.PNG_I" value="44409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g002.PNG_L" value="3788550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g002.PNG_M" value="228466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g002.PNG_S" value="11755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g002.TIF" value="5510212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g002.PNG_I" value="127323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g003.PNG_L" value="1267000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g003.PNG_M" value="68981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g003.PNG_S" value="9989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g003.TIF" value="2532682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g003.PNG_I" value="31240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g004.PNG_L" value="1823411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g004.PNG_M" value="167927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g004.PNG_S" value="13261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g004.TIF" value="3605096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g004.PNG_I" value="63904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g005.PNG_L" value="1540084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g005.PNG_M" value="121505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g005.PNG_S" value="9705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g005.TIF" value="3038808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g005.PNG_I" value="43266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g006.PNG_L" value="1294881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g006.PNG_M" value="91860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g006.PNG_S" value="11212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g006.TIF" value="2423722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g006.PNG_I" value="31901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g007.PNG_L" value="2320464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g007.PNG_M" value="98113"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g007.PNG_S" value="9530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g007.TIF" value="3706818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g007.PNG_I" value="40105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g008.PNG_L" value="1980751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g008.PNG_M" value="147276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g008.PNG_S" value="11437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g008.TIF" value="1804908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g008.PNG_I" value="84678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g009.PNG_L" value="9789454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g009.PNG_M" value="135399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g009.PNG_S" value="14843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g009.TIF" value="9770538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g009.PNG_I" value="172505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g010.PNG_L" value="2463300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g010.PNG_M" value="217116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g010.PNG_S" value="21809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g010.TIF" value="3648470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g010.PNG_I" value="146723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g011.PNG_L" value="3212028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g011.PNG_M" value="247893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g011.PNG_S" value="16535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g011.TIF" value="4675600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001834.g011.PNG_I" value="83308"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001834&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001834" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001834&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001834&volume=&issue=&title=Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System&author_name=L%C3%A9titia%20Jean%2C%20Chiu%20Fan%20Lee%2C%20Michael%20Shaw%2C%20David%20J.%20Vaux&start_page=1&end_page=17" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001834" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001834" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001834" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001834&amp;t=Structural%20Elements%20Regulating%20Amyloidogenesis%3A%20A%20Cholinesterase%20Model%20System" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001834&title=Structural%20Elements%20Regulating%20Amyloidogenesis%3A%20A%20Cholinesterase%20Model%20System&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001834&amp;title=Structural%20Elements%20Regulating%20Amyloidogenesis%3A%20A%20Cholinesterase%20Model%20System" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001834" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001834&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Structural Elements Regulating Amyloidogenesis: A Cholinesterase Model System';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001834';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Structural%20Elements%20Regulating%20Amyloidogenesis%3A%20A%20Cholinesterase%20Model%20System http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001834" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001834" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Electrostatics%22" title="Search for articles in the subject area:'Electrostatics'"><div class="flagText">Electrostatics</div></a>
              <div data-categoryid="17995" data-articleid="26492"
                   data-categoryname="Electrostatics"
                   class="flagImage" title="Flag 'Electrostatics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hydrogen+bonding%22" title="Search for articles in the subject area:'Hydrogen bonding'"><div class="flagText">Hydrogen bonding</div></a>
              <div data-categoryid="23891" data-articleid="26492"
                   data-categoryname="Hydrogen bonding"
                   class="flagImage" title="Flag 'Hydrogen bonding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligomers%22" title="Search for articles in the subject area:'Oligomers'"><div class="flagText">Oligomers</div></a>
              <div data-categoryid="20627" data-articleid="26492"
                   data-categoryname="Oligomers"
                   class="flagImage" title="Flag 'Oligomers' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="26492"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plateaus%22" title="Search for articles in the subject area:'Plateaus'"><div class="flagText">Plateaus</div></a>
              <div data-categoryid="27385" data-articleid="26492"
                   data-categoryname="Plateaus"
                   class="flagImage" title="Flag 'Plateaus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40551" data-articleid="26492"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signal+peptides%22" title="Search for articles in the subject area:'Signal peptides'"><div class="flagText">Signal peptides</div></a>
              <div data-categoryid="19701" data-articleid="26492"
                   data-categoryname="Signal peptides"
                   class="flagImage" title="Flag 'Signal peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Surfactants%22" title="Search for articles in the subject area:'Surfactants'"><div class="flagText">Surfactants</div></a>
              <div data-categoryid="26369" data-articleid="26492"
                   data-categoryname="Surfactants"
                   class="flagImage" title="Flag 'Surfactants' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8636'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3912'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2522&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6475">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=17373">Referee Comments: Referee 1 (William Welsh)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
