

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000095"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000095" />

    <meta name="citation_title" content="Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses"/>
    <meta itemprop="name" content="Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses"/>

      <meta name="citation_author" content="Nader Pourmand"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Lisa Diamond"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Rebecca Garten"/>
            <meta name="citation_author_institution" content="Molecular Virology and Vaccines Branch, Influenza Division, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Julianna P. Erickson"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Jochen Kumm"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Ruben O. Donis"/>
            <meta name="citation_author_institution" content="Molecular Virology and Vaccines Branch, Influenza Division, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Ronald W. Davis"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000095.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e95"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genetic characterization of the pathogenic influenza A/Goose/Guangdong/1/96 (H5N1) virus: similarity of its hemagglutinin gene to those of H5N1 viruses from the 1997 outbreaks in Hong Kong.; citation_author=X Xu; citation_author= Subbarao; citation_author=NJ Cox; citation_author=Y Guo; citation_journal_title=Virology; citation_volume=261; citation_number=1; citation_pages=15-19; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Characterization of an avian influenza A (H5N1) virus isolated from a child with a fatal respiratory illness.; citation_author=K Subbarao; citation_author=A Klimov; citation_author=J Katz; citation_author=H Regnery; citation_author=W Lim; citation_journal_title=Science; citation_volume=279; citation_number=2; citation_pages=393-396; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Emergence and predominance of an H5N1 influenza variant in China.; citation_author=GJ Smith; citation_author=XH Fan; citation_author=J Wang; citation_author=KS Li; citation_author=K Qin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=3; citation_pages=16936-16941; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=World Health Organization Global Influenza Program Surveillance Network, “Evolution of H5N1 avian influenza viruses in Asia,”; citation_journal_title=Emerg Infect Dis; citation_number=4; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of a human H5N1 influenza A virus isolated in 2003.; citation_author=K Shinya; citation_author=M Hatta; citation_author=S Yamada; citation_author=A Takada; citation_author=S Watanabe; citation_journal_title=J Virol; citation_volume=79; citation_number=5; citation_pages=9926-9932; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Enhancing public health surveillance for influenza virus by incorporating newly available rapid diagnostic tests.; citation_author=PV Effler; citation_author=MC Ieong; citation_author=T Tom; citation_author=M Nakata; citation_journal_title=Emerg Infect Dis; citation_volume=8; citation_number=6; citation_pages=23-28; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=NJ Schmidt; citation_author=RW Emmons; citation_number=7; citation_pages=xv, 1225; citation_date=1989; citation_publisher=American Public Health Association; " />
      <meta name="citation_reference" content="citation_title=Rapid and sensitive method using multiplex real-time PCR for diagnosis of infections by influenza a and influenza B viruses, respiratory syncytial virus, and parainfluenza viruses 1, 2, 3, and 4.; citation_author=KE Templeton; citation_author=SA Scheltinga; citation_author=MF Beersma; citation_author=AC Kroes; citation_author=EC Claas; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=8; citation_pages=1564-1569; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rapid simultaneous diagnosis of infections with respiratory syncytial viruses A and B, influenza viruses A and B, and human parainfluenza virus types 1, 2, and 3 by multiplex quantitative reverse transcription-polymerase chain reaction-enzyme hybridization assay (Hexaplex).; citation_author=J Fan; citation_author=KJ Henrickson; citation_author=LL Savatski; citation_journal_title=Clin Infect Dis; citation_volume=26; citation_number=9; citation_pages=1397-1402; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A 5′-nuclease real-time reverse transcriptase-polymerase chain reaction assay for the detection of a broad range of influenza A subtypes, including H5N1.; citation_author=DM Whiley; citation_author=TP Sloots; citation_journal_title=Diagn Microbiol Infect Dis; citation_number=10; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparison of SmartCycler real-time reverse transcription-PCR assay in a public health laboratory with direct immunofluorescence and cell culture assays in a medical center for detection of influenza A virus.; citation_author=NF Habib-Bein; citation_author=WH Beckwith; citation_author=D Mayo; citation_author=ML Landry; citation_journal_title=J Clin Microbiol; citation_volume=41; citation_number=11; citation_pages=3597-3601; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=New laboratory assay for diagnostic testing of avian influenza A/H5 (Asian Lineage).; citation_journal_title=MMWR Morb Mortal Wkly Rep; citation_volume=10; citation_number=12; citation_pages=127; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=JC Paulson; citation_number=13; citation_pages=131-219; citation_date=1985; citation_publisher=Academic; " />
      <meta name="citation_reference" content="citation_title=Receptor specificity in human, avian, and equine H2 and H3 influenza virus isolates.; citation_author=RJ Connor; citation_author=Y Kawaoka; citation_author=RG Webster; citation_author=JC Paulson; citation_journal_title=Virology; citation_volume=205; citation_number=14; citation_pages=17-23; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Early alterations of the receptor-binding properties of H1, H2, and H3 avian influenza virus hemagglutinins after their introduction into mammals.; citation_author=M Matrosovich; citation_author=A Tuzikov; citation_author=N Bovin; citation_author=A Gambaryan; citation_author=A Klimov; citation_journal_title=J Virol; citation_volume=74; citation_number=15; citation_pages=8502-8512; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=H5 avian and H9 swine influenza virus haemagglutinin structures: possible origin of influenza subtypes.; citation_author=Y Ha; citation_author=DJ Stevens; citation_author=JJ Skehel; citation_author=DC Wiley; citation_journal_title=Embo J; citation_volume=21; citation_number=16; citation_pages=865-875; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structure and receptor specificity of the hemagglutinin from an H5N1 influenza virus.; citation_author=J Stevens; citation_author=O Blixt; citation_author=TM Tumpey; citation_author=JK Taubenberger; citation_author=JC Paulson; citation_journal_title=Science; citation_volume=312; citation_number=17; citation_pages=404-410; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The role of influenza A virus hemagglutinin residues 226 and 228 in receptor specificity and host range restriction.; citation_author=A Vines; citation_author=K Wells; citation_author=M Matrosovich; citation_author=MR Castrucci; citation_author=T Ito; citation_journal_title=J Virol; citation_volume=72; citation_number=18; citation_pages=7626-7631; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Glycan microarray analysis of the hemagglutinins from modern and pandemic influenza viruses reveals different receptor specificities.; citation_author=J Stevens; citation_author=O Blixt; citation_author=L Glaser; citation_author=JK Taubenberger; citation_author=P Palese; citation_journal_title=J Mol Biol; citation_volume=355; citation_number=19; citation_pages=1143-1155; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Origin and evolution of the 1918 “Spanish” influenza virus hemagglutinin gene.; citation_author=AH Reid; citation_author=TG Fanning; citation_author=JV Hultin; citation_author=JK Taubenberger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=20; citation_pages=1651-1656; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Additional glycosylation at the receptor binding site of the hemagglutinin (HA) for H5 and H7 viruses may be an adaptation to poultry hosts, but does it influence pathogenicity?; citation_author=J Banks; citation_author=L Plowright; citation_journal_title=Avian Dis; citation_volume=47; citation_number=21; citation_pages=942-950; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Proteolytic cleavage of influenza virus hemagglutinins: primary structure of the connecting peptide between HA1 and HA2 determines proteolytic cleavability and pathogenicity of Avian influenza viruses.; citation_author=FX Bosch; citation_author=W Garten; citation_author=HD Klenk; citation_author=R Rott; citation_journal_title=Virology; citation_volume=113; citation_number=22; citation_pages=725-735; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Mutations at the cleavage site of the hemagglutinin after the pathogenicity of influenza virus A/chick/Penn/83 (H5N2).; citation_author=M Ohuchi; citation_author=M Orlich; citation_author=R Ohuchi; citation_author=BE Simpson; citation_author=W Garten; citation_journal_title=Virology; citation_volume=168; citation_number=23; citation_pages=274-280; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A sequencing method based on real-time pyrophosphate.; citation_author=M Ronaghi; citation_author=M Uhlen; citation_author=P Nyren; citation_journal_title=Science; citation_volume=281; citation_number=24; citation_pages=363; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Incidence of adamantane resistance among influenza A (H3N2) viruses isolated worldwide from 1994 to 2005: a cause for concern.; citation_author=RA Bright; citation_author=MJ Medina; citation_author=X Xu; citation_author=G Perez-Oronoz; citation_author=TR Wallis; citation_journal_title=Lancet; citation_volume=366; citation_number=25; citation_pages=1175-1181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sentinel-base DNA genotyping using multiple sequencing primers for high-risk human papillomaviruses.; citation_author=B Gharizadeh; citation_author=B Zheng; citation_author=M Akhras; citation_author=M Ghaderi; citation_author=O Jejelowo; citation_journal_title=Mol Cell Probes; citation_volume=20; citation_number=26; citation_pages=230-238; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ubiquitin, E6-AP, and their role in p53 inactivation.; citation_author=M Scheffner; citation_journal_title=Pharmacol Ther; citation_volume=78; citation_number=27; citation_pages=129-139; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genesis of a highly pathogenic and potentially pandemic H5N1 influenza virus in eastern Asia.; citation_author=KS Li; citation_author=Y Guan; citation_author=J Wang; citation_author=GJD Smith; citation_author=KM Xu; citation_journal_title=Nature; citation_volume=430; citation_number=28; citation_pages=209-213; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mapping of functional domains on the influenza A virus RNA polymerase PB2 molecule using monoclonal antibodies.; citation_author=M Hatta; citation_author=Y Asano; citation_author=K Masunaga; citation_author=T Ito; citation_author=K Okazaki; citation_journal_title=Arch Virol; citation_volume=145; citation_number=29; citation_pages=1947-1961; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Large-scale sequence analysis of avian influenza isolates.; citation_author=JC Obenauer; citation_author=J Denson; citation_author=PK Mehta; citation_author=XP Su; citation_author=S Mukatira; citation_journal_title=Science; citation_volume=311; citation_number=30; citation_pages=1576-1580; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=31; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Hemagglutination inhibition.; citation_author=AP Kendal; citation_author=MS Pereira; citation_author=JJ Skehel; citation_number=32; citation_pages=B17-35; citation_date=1982; " />
      <meta name="citation_reference" content="citation_author=TM Cover; citation_author=JA Thomas; citation_number=33; citation_pages=xxii, 542; citation_date=1991; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Detection of gyrA mutations associated with ciprofloxacin resistance in Neisseria gonorrhoeae by rapid and reliable pre-programmed short DNA sequencing.; citation_author=B Gharizadeh; citation_author=M Akhras; citation_author=M Unemo; citation_author=B Wretlind; citation_author=P Nyren; citation_journal_title=Int J Antimicrob Agents; citation_volume=26; citation_number=34; citation_pages=486-490; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=N Cristianini; citation_author=J Shawe-Taylor; citation_number=35; citation_pages=xiii, 189; citation_date=2000; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_number=36; " />
      <meta name="citation_reference" content="citation_author=RO Duda; citation_author=PE Hart; citation_author=DG Stork; citation_number=37; citation_pages=xx, 654; citation_date=2001; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_author=D Meyer; citation_number=38; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000095" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses"/>
    <meta name="twitter:description" content="A highly discriminative and information-rich diagnostic assay for H5N1 avian influenza would meet immediate patient care needs and provide valuable information for public health interventions, e.g., tracking of new and more dangerous variants by geographic area as well as avian-to-human or human-to-human transmission. In the present study, we have designed a rapid assay based on multilocus nucleic acid sequencing that focuses on the biologically significant regions of the H5N1 hemagglutinin gene. This allows the prediction of viral strain, clade, receptor binding properties, low- or high-pathogenicity cleavage site and glycosylation status. H5 HA genes were selected from nine known high-pathogenicity avian influenza subtype H5N1 viruses, based on their diversity in biologically significant regions of hemagglutinin and/or their ability to cause infection in humans. We devised a consensus pre-programmed pyrosequencing strategy, which may be used as a faster, more accurate alternative to de novo sequencing. The available data suggest that the assay described here is a reliable, rapid, information-rich and cost-effective approach for definitive diagnosis of H5N1 avian influenza. Knowledge of the predicted functional sequences of the HA will enhance H5N1 avian influenza surveillance efforts."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000095.g005"/>

  <meta property="og:title" content="Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000095" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5186'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9391'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2008&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000095">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000095" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000095&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nader Pourmand
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pourmand@stanford.edu">pourmand@stanford.edu</a>
            </p>

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lisa Diamond, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rebecca Garten, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular Virology and Vaccines Branch, Influenza Division, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julianna P. Erickson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jochen Kumm, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ruben O. Donis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular Virology and Vaccines Branch, Influenza Division, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ronald W. Davis
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000095</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000095" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000095" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000095" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000095" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000095-g001" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000095-g002" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000095-g003" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000095-g004" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000095-t001" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000095-t002" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000095-g005" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000095">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000095" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>A highly discriminative and information-rich diagnostic assay for H5N1 avian influenza would meet immediate patient care needs and provide valuable information for public health interventions, e.g., tracking of new and more dangerous variants by geographic area as well as avian-to-human or human-to-human transmission. In the present study, we have designed a rapid assay based on multilocus nucleic acid sequencing that focuses on the biologically significant regions of the H5N1 hemagglutinin gene. This allows the prediction of viral strain, clade, receptor binding properties, low- or high-pathogenicity cleavage site and glycosylation status. H5 HA genes were selected from nine known high-pathogenicity avian influenza subtype H5N1 viruses, based on their diversity in biologically significant regions of hemagglutinin and/or their ability to cause infection in humans. We devised a consensus pre-programmed pyrosequencing strategy, which may be used as a faster, more accurate alternative to <em>de novo</em> sequencing. The available data suggest that the assay described here is a reliable, rapid, information-rich and cost-effective approach for definitive diagnosis of H5N1 avian influenza. Knowledge of the predicted functional sequences of the HA will enhance H5N1 avian influenza surveillance efforts.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Pourmand N, Diamond L, Garten R, Erickson JP, Kumm J, et al.  (2006) Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses. PLoS ONE 1(1):
          e95.
            doi:10.1371/journal.pone.0000095</p><p><strong>Academic Editor: </strong>Paul Digard, University of Cambridge, United Kingdom</p><p><strong>Received:</strong> October 18, 2006; <strong>Accepted:</strong> November 20, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> © 2006 Pourmand et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>National Institutes of Health, 1R21 A1059499-01 and PO1-HG000205</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The worldwide spread of high-pathogenicity H5N1 avian influenza A virus in poultry and wild birds has resulted in many human infections, with high fatality rates. Although sustained human-to-human transmission has not yet occurred, concern about a potential pandemic continues to mount. The current HA lineage of H5N1 avian influenza was first found among domestic poultry populations in 1996 in southern China <a href="#pone.0000095-Xu1">[1]</a>. A similar H5N1 influenza virus spread directly from poultry to humans in Hong Kong in 1997, causing death in 6 out of 18 persons diagnosed with infection with this virus <a href="#pone.0000095-Subbarao1">[2]</a>. While the massive culling of poultry in 1997 temporarily eradicated the virus in Hong Kong, the virus has continued to spread across Asia, causing human deaths in Thailand, Vietnam, Indonesia, China and elsewhere <a href="#pone.0000095-Subbarao1">[2]</a>, <a href="#pone.0000095-Smith1">[3]</a>. The rapid spread of H5N1 in birds from Asia into Europe and Africa in recent months has intensified efforts to control the virus and avert a pandemic. To address the recognized need for rapid, low-cost diagnosis, tracking critically important genetic changes in the virus among animal and human host populations, and identifying specific viral clades <a href="#pone.0000095-WHO1">[4]</a>, we have developed high-throughput methods for monitoring viral mutations that may control virulence and transmissibility in humans <a href="#pone.0000095-Shinya1">[5]</a>. Accurate and rapid detection and tracking of H5N1 will be critical to prevent or control a potential pandemic.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Diagnosis of influenza type A infections in clinical microbiology laboratories has traditionally been performed using cell culture and/or direct fluorescent antibody assays <a href="#pone.0000095-Shinya1">[5]</a>–<a href="#pone.0000095-Schmidt1">[7]</a>. These methods are time-consuming and require biosafety level 3 enhanced biocontainment facilities and equipment to protect laboratory personnel from exposure to H5N1 cultured in the laboratory. Because these facilities not widely available, culture-based assays are increasingly being replaced in clinical settings by the various polymerase chain reaction (PCR) methods <a href="#pone.0000095-Templeton1">[8]</a>–<a href="#pone.0000095-HabibBein1">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>PCR is more sensitive than traditional tests and detection does not require viable virus or morphologically intact infected cells in the sample. A PCR-based molecular diagnostic test is currently the most widely used by public health laboratories to diagnose the presence of H5N1 in clinical specimens <a href="#pone.0000095-CDC1">[12]</a>. We hypothesized that coupling a PCR assay to a rapid sequencing method would further increase the value of molecular techniques for virus identification and characterization, especially if implemented into automated robotic platforms in the near future. Nucleic acid sequencing is considered the most reliable and highest-resolution method for virus identification, but is typically too slow and costly to use as a primary assay. Samples can be prepared sequentially for PCR diagnosis of H5N1 influenza virus, and pyrosequencing, yielding results in approximately 90 minutes, with immediate availability of the viral sequence data. The speed, sensitivity, precision, low cost, and high throughput of this method give it substantial advantages in H5N1 influenza characterization. We have designed an assay that focuses on three biologically significant regions of the H5N1 hemagglutinin gene (<a href="#pone-0000095-g001">Figure 1</a>), including sites informative of viral ancestry.</p>
<div class="figure" id="pone-0000095-g001"><div class="img"><a name="pone-0000095-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Primers, markers and key sites on H5N1 influenza A hemagglutinin gene. A region of 768 bases of cDNA of the hemagglutinin gene of nine strains of H5N1 was assayed to obtain sequence information for three key biologically significant sites (glycosylation, receptor specificity, and HA1/HA2 cleavage; (sites indicated in red)). Two markers (green) distinguish clades of H5N1. Three additional polymorphic sites (yellow) provide unambiguous strain identification. The PCR primers used to bracket this region are indicated in purple; internal primers are blue.</p>
<span>doi:10.1371/journal.pone.0000095.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Influenza type A viruses have an eight-segment negative-sense RNA genome complexed with nucleoprotein and polymerase surrounded by the matrix protein and a lipid envelope that contains two integral membrane glycoproteins, hemagglutinin (HA) and neuraminidase (NA), protruding from the virion surface. The HA binds to sialic acid-terminated glycan receptors on the host cell surface, triggering virion uptake by endocytosis. Human respiratory tract cells have predominantly alpha 2-6 linked sialic acid receptors, whereas duck intestine has predominantly alpha 2-3 linked sialic acid receptors. Host-adapted human and avian influenza viruses selectively bind to homologous variant sialoside structures <a href="#pone.0000095-Paulson1">[13]</a>–<a href="#pone.0000095-Matrosovich1">[15]</a>. The HA receptor binding site is located at the convergence of one helix, two loops and three single residues near the top of the molecule; amino acid residues in the 184-186 (188-190 H3) helix as well as the 130–134 (134-138 H3) and 217–224 (221-228 H3) loops make up the rims, whereas residues 91 (98 H3), 149 (153 H3) and 179 (183 H3) form the floor of the cavity (H5 numbering; references: <a href="#pone.0000095-Ha1">[16]</a>, <a href="#pone.0000095-Stevens1">[17]</a>). In the H3 subtype of HA, amino acids 222 and 224 (226 and 228 H3) are major determinants of human or avian host sialic acid specificity of the virus; alpha 2-6 linkages in human viruses and alpha 2-3 linkages in avian viruses <a href="#pone.0000095-Vines1">[18]</a>. Similar changes have been documented for the H1 HA that caused the 1918 so-called Spanish influenza pandemic <a href="#pone.0000095-Stevens2">[19]</a>, <a href="#pone.0000095-Reid1">[20]</a>. Mutation at amino acid 223 from serine-to-asparagine (227 in H3) in a 2003 human H5N1 isolate (A/Hong Kong/213/03) maintained avian-like binding specificity of alpha 2-3 linked sialic acids but also had increased binding to alpha 2-6 sialic acid linkages <a href="#pone.0000095-Stevens1">[17]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The HA is also responsible for cell entry by mediating fusion of the endosomal and viral membranes. The HA requires proteolytic cleavage to become functional in membrane fusion. Cleavage, mediated by host proteases, results in two disulfide-linked subunits, HA1 and HA2. The HA1 region of the HA gene encompasses three sites of known biological significance:</p>


<ul class="bulletlist">

<li>A glycosylation sequon at the top of the receptor binding site at amino acids 154-156 (of mature H5 HA1) (158-160 H3) has been linked to viral adaptation from waterfowl to domesticated poultry. The majority of H5N1 viruses isolated from humans since 2004 have this motif <a href="#pone.0000095-WHO1">[4]</a>, <a href="#pone.0000095-Banks1">[21]</a>.</li>

<li>The receptor binding site. Changes in the rims or floor of the receptor-binding site thought to favor binding of human forms of cell surface sialosides are considered critical for sustained transmission in the human population.</li>

<li>Finally, the number of basic amino acids (lysine or arginine) in the cleavage site between HA1 and HA2 determines whether the virus is highly pathogenic for birds <a href="#pone.0000095-Bosch1">[22]</a>, <a href="#pone.0000095-Ohuchi1">[23]</a>. All Eurasian H5N1 viruses currently circulating are highly pathogenic and contain either five or six consecutive basic amino acids at this site <a href="#pone.0000095-WHO1">[4]</a>.</li>

</ul><a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Recent phylogenetic analysis of H5N1 evolution indicates that distinct major and minor clades have emerged among HA gene lineages; the two most important ones of these from a public health perspective are termed clades 1 and 2. These two clades are identifiable with two amino acids, 124 and 212 (129 and 216 in H3) <a href="#pone.0000095-WHO1">[4]</a>. These clade distinctions may be antigenically significant; therefore clade identification may aid selection of appropriate vaccines. Rapid H5N1 identification will assist in pinpointing the source of particularly virulent outbreaks, and in targeting limited supplies of vaccines and anti-virals to key regions.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Here, we report the results of a proof-of-principle experiment based on the application of pyrosequencing technology targeting the hemagglutinin gene of H5N1 influenza. The assay uses RT-PCR to amplify a known H5N1-specific region of 768 nucleotides. Subsequent pyrosequencing <a href="#pone.0000095-Ronaghi1">[24]</a> of strains that are H5N1-positive with ten specific sequencing primers is used to determine HA clade and strain, receptor binding preference, low- or high-pathogenicity cleavage site, and glycosylation status. The assay presented here is considerably more informative than traditional techniques such as real-time PCR, as it not only identifies the H5N1 lineage but also predicts receptor binding properties that could herald the development of human-human transmissibility. This assay would be most advantageous as part of a screening method during high-volumes of H5N1 activity. Also, pyrosequencing has some advantages over Sanger sequencing-based methods, specifically the availability of sequence data directly following the sequencing primer, and the accessibility of the results in real time. Moreover, this assay is specific, rapid and cost-effective.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>HA Amplification by PCR</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The first step in the analysis of a clinical specimen or a viral isolate in our assay is the generation of a DNA copy of the viral RNA, which is accomplished by reverse transcription coupled to PCR (RT-PCR). This was performed using two different biotinylated combinations of PCR primers specific to the H5N1 regions of interest in order to achieve optimal sequencing flexibility of H5N1 isolates, as listed in <a href="#pone-0000095-g002">Figure 2</a>. As shown in <a href="#pone-0000095-g003">Figure 3</a>, either pair of primers provided reliable amplification of H5N1, and neither generated products when used in PCRs with negative controls (total genomic DNA from unrelated human cell lines) (confirmed by Sanger sequencing).</p>
<div class="figure" id="pone-0000095-g002"><div class="img"><a name="pone-0000095-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Phylogenetic relationships of H5N1 hemagglutinin (HA) genes from highly pathogenic H5N1 avian influenza viruses used in this study. Phylogenetic trees were inferred from nucleotide sequences by the neighbor joining method in the MEGA program. Horizontal distances are proportional to the number of nucleotide changes between the viruses. HA clade determinations are shown on the right.</p>
<span>doi:10.1371/journal.pone.0000095.g002</span></div><div class="figure" id="pone-0000095-g003"><div class="img"><a name="pone-0000095-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Amplicon DNA yield from PCR with different combinations of biotinylated primers. All PCR products were Sanger-sequenced to confirm the identity of the sample. A) The nine H5N1 samples amplified with a biotinylated forward primer (B-F-H5N1-1) and nonbiotinylated reverse primer (R-H5N1-2). From left to right, the samples are A/goose/Guangdong/1/96 (AF144305), A/Hong Kong/156/97 (AF036356), A/Hong Kong/483/97 (AF046097), A/Hong Kong/213/2003 (AY575869), A/chicken/Korea/ES/03 (AY676035), A/Vietnam/1203/2004 vaccine candidate with deletion of polybasic cleavage site (AY651334), A/Vietnam/JP14/2005 (ISDN117778), A/Vietnam/HN30408/2005 (ISDN119678), A/Indonesia/5/05 (ISDN125873). B) The H5N1 PCR products amplified with a biotinylated reverse primer (B-R-H5N1-3) and a nonbiotinylated forward primer (F-H5N1-4). Samples are in the order stated for part (A).</p>
<span>doi:10.1371/journal.pone.0000095.g003</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>All of the H5N1 PCR products were sequenced at least twice both by Sanger dideoxy sequencing and pyrosequencing; PCR-positive amplicons generated correct sequence results independent of sequencing method or fragment size. Furthermore, no loops or primer-dimers were observed when primers were pyrosequenced in the absence of template. Representative and typical pyrograms of sample A/Vietnam/HN30408/ 2005, obtained by pyrosequencing, are shown in <a href="#pone-0000095-g004">Figure 4</a>; pyrograms for the remaining eight samples are available as supplementary information. The pyrosequencing run covering the required 14 bases took approximately 15 minutes.</p>
<div class="figure" id="pone-0000095-g004"><div class="img"><a name="pone-0000095-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Panels A–I: Pyrograms obtained by pyrosequencing of sample A/Vietnam/HN30408/2005 with the HS 96 system. Peaks above a given nucleotide sample indicate nucleotide incorporation (with height proportional to number of consecutive incorporation events). Initial pyrosequencing was performed with systematic nucleotide dispensation for <em>de novo</em> sequencing (pre-programmed dispensation runs can also be performed). Panel J: Map of the HA sequence of A/Vietnam/HN30408/2005 obtained by Sanger sequencing; pyrosequencing data is overlaid and color-coded to show correlation between the sequence data obtained by the two methods. Reverse-primed pyrosequencing results should be read as reverse complements. For clarity, primers and biologically significant sequence segments are colored to correspond to the sequence map presented in <a href="#pone-0000095-g001">Figure 1</a>.</p>
<span>doi:10.1371/journal.pone.0000095.g004</span></div>

<h4>HA Pyrosequencing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The results of our pyrosequencing assay clearly distinguished the nine different strains of H5N1 avian influenza, based on eight sites, as shown in <a href="#pone-0000095-t001">Table 1</a>. Furthermore, this approach provided accurate sequencing of regions of known biological significance (for supplementary data, see <a href="http://bioel.stanford.edu/avianflu/index.html">http://bioel.stanford.edu/avianflu/index​.html</a>). The results of this assay can be seen in <a href="#pone-0000095-t002">Table 2</a>, which shows the amino acid sequences characteristic of each strain tested.</p>
<div class="figure" id="pone-0000095-t001"><div class="img"><a name="pone-0000095-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000095.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000095.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Sequences of H5N1 clade markers and active sites, as determined by pyrosequencing.</span></strong></p><span>doi:10.1371/journal.pone.0000095.t001</span></div><div class="figure" id="pone-0000095-t002"><div class="img"><a name="pone-0000095-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000095.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000095.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Characterization of the hemagglutinin from the H5N1 influenza viruses used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0000095.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In order to characterize the strains based on the HA sequence, pyrograms obtained via pyrosequencing were visually inspected to verify the sequences of functionally relevant sites and the presence of polymorphisms that could serve as lineage markers. These sequences were then compared to known H5N1 sequences for clade and strain identification. Certain positions provided essential information for the identification of signatures important for public health, such as changes in the receptor binding site which could signal an increase in the ability of the virus to transmit from human- to-human. The prototype receptor binding site for avian isolates in amino acids 221-224 of the HA1 is GQSG. One human isolate, Hong Kong/213/03, has a mutation at amino acid 223, S-&gt;N, which switches the affinity of the hemagglutinin from alpha 2-3 linked sialic acid to the alpha 2-6 linked sialic acid preferred by human influenza viruses. This mutation was detected by a single nucleotide change, G-&gt;A, in the sequence of the receptor binding active site. Rapid characterization of the receptor binding active site and other active sites of the hemagglutinin is critical in identifying viruses with increased pandemic potential.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our initial results indicate that targeted pyrosequencing approach can clearly distinguish among different strains of H5N1, and can accurately sequence regions of known biological significance such as receptor binding (supplementary data: <a href="http://bioel.stanford.edu/avianflu/index.html">http://bioel.stanford.edu/avianflu/index​.html</a>). The HA sequence results obtained by pyrosequencing were 100% identical to those obtained by the Sanger method (two replicates were performed to cross-verify results). Three pyrosequencing replicates, with nine samples at ten sites, were performed: two with a <em>de novo</em> dispensation and one with a pre-programmed dispensation order. Of the three pyrosequencing replicates performed, all successful runs verified one another. Because Sanger sequencing relies on molecular separation of the polymerase-mediated extension of the primer, the first ~20–50 bases 3′ of the primer are not discernible. In contrast, pyrosequencing provides unambiguous sequence from the first nucleotide 3′ to the primer, increasing the sequence information yield per assay. This feature of pyrosequencing was exploited by designing H5N1-specific primers adjacent to critical polymorphic sites such that the initial base-callings are usually sufficient to determine the lineage of a given HA gene.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>A simpler detection assay giving a positive or negative identification of H5N1 will be helpful, but a more information-rich method is necessary to provide additional information to guide patient care or public health measures, especially as new strains emerge. Indeed, some laboratories are already using pyrosequencing to identify drug-resistant viruses <a href="#pone.0000095-Bright1">[25]</a>. As H5N1 influenza becomes more prevalent, it will be impractical to perform full sequencing of all isolates in time for analysis of rapidly changing epidemiological trends. Our assay could be a valuable complement to full sequencing at public health laboratories. We have developed a rapid and inexpensive assay based on DNA sequencing for early detection of virus present in host cells. This assay permits rapid, simplified and accurate identification of avian H5N1 influenza A, and consists of simple procedures maintaining high sensitivity and specificity. As new sites of functional relevance are identified, new site-specific primers can be added to broaden the utility of the assay. Expanded knowledge of H5N1 sequence and evolution is likely to contribute to more effective diagnostic methods and treatments that require less investment of time and money.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>An additional advantage of pyrosequencing over the Sanger method is its application in genotyping single and multiple HPV infections <a href="#pone.0000095-Gharizadeh1">[26]</a>. This technology is based on the use of multiple sequencing primers in the same pyrosequencing reaction. Pools of primers are chosen based on their position on the target, the sequence information they yield, and their cross-reactivity with other primers in the multiplex assay. Clinically relevant samples containing multiple infections can then be identified and characterized.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>These early robust results have validated our approach and selection of specific primers. We have moved toward a further refinement of the assay based on these results. After an initial <em>de novo</em> pyrosequencing run was performed on the H5N1 samples, sequence data for each strain were compiled for each nucleotide site. This information was integrated to produce consensus pre-programmed dispensation orders of nucleotides. This type of pyrosequencing would allow us to obtain sequence results of similarly high quality in 10 to 15 minutes (<a href="#pone-0000095-g005">Figure 5</a>). With this approach, all of the H5N1 subtypes included in this study could be positively identified. New variants would not be fully characterized, but would be heralded by truncated sequences.</p>
<div class="figure" id="pone-0000095-g005"><div class="img"><a name="pone-0000095-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000095" data-uri="info:doi/10.1371/journal.pone.0000095.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000095.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000095.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000095.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000095.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>A representative pyrogram illustration. Comparison of pyrosequencing results obtained using the <em>de novo</em> (top) and consensus pre-programmed (bottom) sequencing methods for site 3 (a clade marker) in sample A1 (goose/Guangdong/1/96). Both methods give the sequence <strong>GACAAAGCTCTATCAAAAC</strong> for our laboratory stock. In contrast the GenBank sequence (accession # AF144305) reads <strong>GACAAAGCTATATCAAAAC. Database information was found to be based off wild-type sequence information, while the sample we analyzed was obtained via reverse genetics.</strong> A longer read length is obtained via the consensus pre-programmed dispensation method (22 versus 12 nucleotides).</p>
<span>doi:10.1371/journal.pone.0000095.g005</span></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In summary, we have developed a reliable, rapid, cost-effective, and information-rich diagnostic assay for H5N1 influenza. This sequence-based method could be extended to include amplicons from other genes of interest; for example, drug targets such as NA and the M2 ion channel <a href="#pone.0000095-Scheffner1">[27]</a>, <a href="#pone.0000095-Li1">[28]</a>, or the postulated virulence motifs on NS1 or PB2 <a href="#pone.0000095-Hatta1">[29]</a>, <a href="#pone.0000095-Obenauer1">[30]</a>. Pyrosequencing is user-friendly, and permits significantly more efficient and rapid genotyping than traditional techniques. This method detects a wide range of influenza A H5N1 subtypes based on sequence information. The assay will allow further development of technology to directly detect H5N1 or specific strains of influenza A in clinical specimens without extensive sample preparation.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Avian Influenza A Subtypes and H5N1-Specific Primer Design</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Sequences of 362 avian influenza A virus species were acquired from GenBank at the National Center for Biotechnology Information and Influenza Sequence Database at Los Alamos National Laboratory and aligned using Clustal X version 1.83 <a href="#pone.0000095-Thompson1">[31]</a>. A specific sequencing primer was designed for H5N1, with no sequence similarity to the other influenza virus HA subtypes based on the database searches and alignments (<a href="#pone-0000095-g001">Figure 1</a>).</p>


<h4>H5N1 Avian Influenza Virus Isolates</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The viruses that formed the test set for this study were chosen on the basis of their diversity in biologically significant regions of hemagglutinin and/or their ability to cause infection in humans. Virus strain names are shown in <a href="#pone-0000095-g002">Figure 2</a> and accession numbers are as follows: A/goose/Guangdong/1/96 (AF144305), A/Hong Kong/156/97 (AF036356), A/Hong Kong/483/97 (AF046097), A/Hong Kong/213/2003 (AY575869), A/chicken/Korea/ES/03 (AY676035), A/Vietnam/1203/2004 vaccine candidate with deletion of polybasic cleavage site (AY651334), A/Vietnam/JP14/2005 (ISDN117778), A/Vietnam/HN30408/2005 (ISDN119678), A/Indonesia/5/05 (ISDN125873).</p>


<h4>RNA Extraction, RT-PCR and PCR Amplification</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Viruses were isolated and propagated by inoculation into the allantoic sac of 10-day old chicken embryos, as described previously <a href="#pone.0000095-Kendal1">[32]</a>. Viral infectivity was determined by endpoint dilution, injection into chicken embryos and hemagglutination assay. All work involving infectious H5N1 influenza was performed in government-approved biosafety level 3-enhanced containment facilities as required by the U.S. Department of Agriculture and the Select Agent Program (see guidance at <a href="http://www.cdc.gov/flu/h2n2bsl3.htm">http://www.cdc.gov/flu/h2n2bsl3.htm</a>).Viral RNA was extracted using the QIAmp vRNA Kit (Qiagen, Valencia CA). Extractions were performed according to manufacturer's instructions. QIAGEN Onestep RT-PCR kit (Qiagen, Valencia CA) was used to perform RT-PCR from 30 ng of RNA (approximate concentration of 10 ng/µl by spectrophotometer) in a 50 µl reaction volume. The RT-PCR amplification primers, which are biotinylated F-H5N1-1/3 (5′-TCAAYGACTATGAAGAAYTGAAACA-3′), and R-H5N1-2/4 (5′- AACCATCTACCATTCCCTGCCATCC-3) were synthesized by IDT (Coralville, IA, USA). RT-PCR was performed with a DNA Engine (PTC-200) Peltier Thermal Cycler (BIO-RAD, Hercules, Calif.) as follows: 50°C for 10 minutes, 95°C for 15 minutes, 35 cycles of 95°C, 55°C, and 72°C for 1 minute each, and finally a 10 minute final extension at 72°C. To confirm proper amplification, PCR products were electrophoresed in a 1% agarose gel and visualized by ethidium bromide staining under UV illumination.</p>


<h4>Sanger dideoxy DNA Sequencing of PCR Products</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The amplified DNA from all isolates was cycle sequenced in both directions using the BigDye Terminators Reaction Kit v. 3.1 (Applied Biosystems, Foster City, CA) on an ABI automated DNA sequencer (3730 XL DNA Analyzer).</p>


<h4>Hemagglutinin Sequencing Primers</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>An entropy-based analysis of genetic variation among H5N1 strains (<a href="#pone-0000095-g001">Figure 1</a>) was used to design PCR primers <a href="#pone.0000095-Cover1">[33]</a>. The approach focused on amplicons comprising regions for receptor binding sites, cleavage site and glycosylation sites, along with markers to identify clade and individual strain (<a href="#pone-0000095-g001">Figure 1</a>). Highly conserved sites were selected as internal sequencing primers to sequence each region of interest. The selected region is bracketed by sequences unique to and highly conserved in the H5N1 subtype. Based on sequence alignments of H5N1 and sequencing results (data not shown) from all the isolates, a set of sequencing primers were designed spanning a region of the HA informative with regards to clade, strain, receptor binding motif, cleavability and glycosylation (<a href="#pone-0000095-g001">Figure 1</a>) and which specifically hybridize to H5N1. Sequences for relevant influenza viruses with known properties and virulence status were compiled for each sample from pyrograms. These sequences were then compared to known H5N1 sequences by alignment and visual inspection.</p>


<h4>Pyroseqencing</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Biotinylated PCR product (5 ng/µl) from RT step from viral RNA (<a href="#pone-0000095-g001">Figure 1</a>) was immobilized onto 2.5 µl streptavidin-coated High Performance Sepharose beads (Amersham Biosciences, Piscataway, NJ) by incubation at room temperature for at least 10 minutes with agitation at 1400 rpm. Single-stranded DNA was obtained by washing the immobilized PCR product with 70% EtOH, denatured with 0.2 M NaOH, and washed with TA-Buffer (0.1 M Tris-Acetate, pH 7.6) using a Vacuum Prep Tool and Vacuum Prep Worktable (Biotage, Uppsala, Sweden). The beads were then suspended in 12 µl annealing buffer (10 mM Tris-acetate pH 7.75, 5 mM Mg-acetate) containing 0.3 pmol sequencing primer. Single-stranded DNA was hybridized to the sequencing primer by incubation at 90°C for 2 minutes, at 60°C for 5 minutes and at room temperature for 5 minutes.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Primed single-stranded PCR products were sequenced using PSQ™ HS96A System (Biotage). Sequencing was performed in a total volume of 12 µl using the PSQ 96 Gold kit (Biotage). Consensus pre-programmed dispensation orders were determined by integrating sequence information for each strain at a given sequencing site <a href="#pone.0000095-Gharizadeh2">[34]</a>. See <a href="#pone-0000095-g005">figure 5</a> for a comparison of <em>de novo</em> and pre-programmed sequencing strategies. As shown, approximately 10 more nucleotides of sequence information are obtained with the pre-programmed method, because nucleotide incorporations are anticipated. This reduces the amount of time required as well as the accumulation of sequencing byproducts. Negative-control nucleotide dispensations were also included in the program to check for insertions and to measure background signals. The identity and number of nucleotide extension events were determined by automated measurement of the amount of light generated after incorporation of each dNTP.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>Raw data were interpreted using software developed specifically for this purpose,“Classifier.” Classification of samples by strain is straightforward using the short sequence segments obtained. We use a Support Vector Machine <a href="#pone.0000095-Cristianini1">[35]</a> approach implemented in the statistical programming language R <a href="#pone.0000095-R1">[36]</a> to classify a given sample with statistical accuracy. This provides an automated sample identification tool, designed for eventual use with large numbers of source sequences <a href="#pone.0000095-Duda1">[37]</a>, <a href="#pone.0000095-Meyer1">[38]</a>.</p>


<h4>Supplementary information</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>See <a href="http://bioel.stanford.edu/avianflu/index.html">http://bioel.stanford.edu/avianflu/index​.html</a> for detailed pyrosequencing results for all nine strains.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Alexander Klimov, Amanda Balish, Rick Bright, Catherine Smith, Michael Shaw, Yumi Matsuoka, and Nancy Cox for their invaluable help. Viruses and sequences were kindly provided through the WHO Global Influenza Program, and its affiliates, including Nyoman Kandun, Endang Sedyaningsih, Patrick Blair, Huluan Chen, Wilina Lim, Peter Cheng and Le Thi Quynh Mai.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p>Reference herein to any specific commercial products, process, or service by trade name, trademark, manufacturer, or otherwise, is provided for information only and does not constitute or imply its endorsement, recommendation, or favoring by the Centers for Disease Control and Prevention and the U.S. Department of Health and Human Services.</p>
<a id="article1.back1.ack1.p3" name="article1.back1.ack1.p3"></a><p>The views and opinions expressed herein are those of the authors and do not necessarily reflect the views and opinions of the Centers for Disease Control and Prevention and the U.S. Department of Health and Human Services.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RD NP. Performed the experiments: JE. Analyzed the data: NP LD JE. Contributed reagents/materials/analysis tools: JK LD RG RD. Wrote the paper: NP LD RG JE RD.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000095-Xu1" id="pone.0000095-Xu1"></a>Xu X, Subbarao , Cox NJ, Guo Y (1999) Genetic characterization of the pathogenic influenza A/Goose/Guangdong/1/96 (H5N1) virus: similarity of its hemagglutinin gene to those of H5N1 viruses from the 1997 outbreaks in Hong Kong. Virology  261: 15–19.  <ul class="find" data-citedArticleID="920528" data-doi="10.1006/viro.1999.9820"><li><a href="http://dx.doi.org/10.1006/viro.1999.9820" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+characterization+of+the+pathogenic+influenza+A%2FGoose%2FGuangdong%2F1%2F96+%28H5N1%29+virus%3A+similarity+of+its+hemagglutinin+gene+to+those+of+H5N1+viruses+from+the+1997+outbreaks+in+Hong+Kong." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+characterization+of+the+pathogenic+influenza+A%2FGoose%2FGuangdong%2F1%2F96+%28H5N1%29+virus%3A+similarity+of+its+hemagglutinin+gene+to+those+of+H5N1+viruses+from+the+1997+outbreaks+in+Hong+Kong.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000095-Subbarao1" id="pone.0000095-Subbarao1"></a>Subbarao K, Klimov A, Katz J, Regnery H, Lim W, et al.  (1998) Characterization of an avian influenza A (H5N1) virus isolated from a child with a fatal respiratory illness. Science  279: 393–396.  <ul class="find" data-citedArticleID="920516" data-doi="10.1126/science.279.5349.393"><li><a href="http://dx.doi.org/10.1126/science.279.5349.393" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+an+avian+influenza+A+%28H5N1%29+virus+isolated+from+a+child+with+a+fatal+respiratory+illness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+an+avian+influenza+A+%28H5N1%29+virus+isolated+from+a+child+with+a+fatal+respiratory+illness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000095-Smith1" id="pone.0000095-Smith1"></a>Smith GJ, Fan XH, Wang J, Li KS, Qin K, et al.  (2006) Emergence and predominance of an H5N1 influenza variant in China. Proc Natl Acad Sci U S A  103: 16936–16941.  <ul class="find" data-citedArticleID="920510" data-doi="10.1073/pnas.0608157103"><li><a href="http://dx.doi.org/10.1073/pnas.0608157103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+and+predominance+of+an+H5N1+influenza+variant+in+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+and+predominance+of+an+H5N1+influenza+variant+in+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000095-WHO1" id="pone.0000095-WHO1"></a>WHO (2005) World Health Organization Global Influenza Program Surveillance Network, “Evolution of H5N1 avian influenza viruses in Asia,” Emerg Infect Dis.   11, 2005. (<a href="http://www.cdc.gov/ncidod/eid/vol11no10/pdfs/05-0644.pdf">http://www.cdc.gov/ncidod/eid/vol11no10/​pdfs/05-0644.pdf</a>).  <ul class="find" data-citedArticleID="920524" data-doi="10.3201/eid1110.050644"><li><a href="http://dx.doi.org/10.3201/eid1110.050644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=World+Health+Organization+Global+Influenza+Program+Surveillance+Network%2C+%26%23x201c%3BEvolution+of+H5N1+avian+influenza+viruses+in+Asia%2C%26%23x201d%3B" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22World+Health+Organization+Global+Influenza+Program+Surveillance+Network%2C+%26%23x201c%3BEvolution+of+H5N1+avian+influenza+viruses+in+Asia%2C%26%23x201d%3B%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000095-Shinya1" id="pone.0000095-Shinya1"></a>Shinya K, Hatta M, Yamada S, Takada A, Watanabe S, et al.  (2005) Characterization of a human H5N1 influenza A virus isolated in 2003. J Virol  79: 9926–9932.  <ul class="find" data-citedArticleID="920508" data-doi="10.1128/jvi.79.15.9926-9932.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.15.9926-9932.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+human+H5N1+influenza+A+virus+isolated+in+2003." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+human+H5N1+influenza+A+virus+isolated+in+2003.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000095-Effler1" id="pone.0000095-Effler1"></a>Effler PV, Ieong MC, Tom T, Nakata M (2002) Enhancing public health surveillance for influenza virus by incorporating newly available rapid diagnostic tests. Emerg Infect Dis  8: 23–28.  <ul class="find" data-citedArticleID="920470" data-doi="10.3201/eid0801.010067"><li><a href="http://dx.doi.org/10.3201/eid0801.010067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancing+public+health+surveillance+for+influenza+virus+by+incorporating+newly+available+rapid+diagnostic+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancing+public+health+surveillance+for+influenza+virus+by+incorporating+newly+available+rapid+diagnostic+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000095-Schmidt1" id="pone.0000095-Schmidt1"></a>Schmidt NJ, Emmons RW, American Public Health Association. Committee on Laboratory Standards and Practices (1989) Diagnostic procedures for viral, rickettsial, and chlamydial infections. Washington, DC: American Public Health Association.   <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0000095-Templeton1" id="pone.0000095-Templeton1"></a>Templeton KE, Scheltinga SA, Beersma MF, Kroes AC, Claas EC (2004) Rapid and sensitive method using multiplex real-time PCR for diagnosis of infections by influenza a and influenza B viruses, respiratory syncytial virus, and parainfluenza viruses 1, 2, 3, and 4. J Clin Microbiol  42: 1564–1569.  <ul class="find" data-citedArticleID="920518" data-doi="10.1128/jcm.42.4.1564-1569.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.4.1564-1569.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+and+sensitive+method+using+multiplex+real-time+PCR+for+diagnosis+of+infections+by+influenza+a+and+influenza+B+viruses%2C+respiratory+syncytial+virus%2C+and+parainfluenza+viruses+1%2C+2%2C+3%2C+and+4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+and+sensitive+method+using+multiplex+real-time+PCR+for+diagnosis+of+infections+by+influenza+a+and+influenza+B+viruses%2C+respiratory+syncytial+virus%2C+and+parainfluenza+viruses+1%2C+2%2C+3%2C+and+4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000095-Fan1" id="pone.0000095-Fan1"></a>Fan J, Henrickson KJ, Savatski LL (1998) Rapid simultaneous diagnosis of infections with respiratory syncytial viruses A and B, influenza viruses A and B, and human parainfluenza virus types 1, 2, and 3 by multiplex quantitative reverse transcription-polymerase chain reaction-enzyme hybridization assay (Hexaplex). Clin Infect Dis  26: 1397–1402.  <ul class="find" data-citedArticleID="920472" data-doi="10.1086/516357"><li><a href="http://dx.doi.org/10.1086/516357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+simultaneous+diagnosis+of+infections+with+respiratory+syncytial+viruses+A+and+B%2C+influenza+viruses+A+and+B%2C+and+human+parainfluenza+virus+types+1%2C+2%2C+and+3+by+multiplex+quantitative+reverse+transcription-polymerase+chain+reaction-enzyme+hybridization+assay+%28Hexaplex%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+simultaneous+diagnosis+of+infections+with+respiratory+syncytial+viruses+A+and+B%2C+influenza+viruses+A+and+B%2C+and+human+parainfluenza+virus+types+1%2C+2%2C+and+3+by+multiplex+quantitative+reverse+transcription-polymerase+chain+reaction-enzyme+hybridization+assay+%28Hexaplex%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000095-Whiley1" id="pone.0000095-Whiley1"></a>Whiley DM, Sloots TP (2005) A 5′-nuclease real-time reverse transcriptase-polymerase chain reaction assay for the detection of a broad range of influenza A subtypes, including H5N1. Diagn Microbiol Infect Dis.   <ul class="find" data-citedArticleID="920526" data-doi="10.1016/j.diagmicrobio.2005.08.002"><li><a href="http://dx.doi.org/10.1016/j.diagmicrobio.2005.08.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+5%26%23x2032%3B-nuclease+real-time+reverse+transcriptase-polymerase+chain+reaction+assay+for+the+detection+of+a+broad+range+of+influenza+A+subtypes%2C+including+H5N1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+5%26%23x2032%3B-nuclease+real-time+reverse+transcriptase-polymerase+chain+reaction+assay+for+the+detection+of+a+broad+range+of+influenza+A+subtypes%2C+including+H5N1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000095-HabibBein1" id="pone.0000095-HabibBein1"></a>Habib-Bein NF, Beckwith WH 3rd, Mayo D, Landry ML (2003) Comparison of SmartCycler real-time reverse transcription-PCR assay in a public health laboratory with direct immunofluorescence and cell culture assays in a medical center for detection of influenza A virus. J Clin Microbiol  41: 3597–3601.  <ul class="find" data-citedArticleID="920480" data-doi="10.1128/jcm.41.8.3597-3601.2003"><li><a href="http://dx.doi.org/10.1128/jcm.41.8.3597-3601.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+SmartCycler+real-time+reverse+transcription-PCR+assay+in+a+public+health+laboratory+with+direct+immunofluorescence+and+cell+culture+assays+in+a+medical+center+for+detection+of+influenza+A+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+SmartCycler+real-time+reverse+transcription-PCR+assay+in+a+public+health+laboratory+with+direct+immunofluorescence+and+cell+culture+assays+in+a+medical+center+for+detection+of+influenza+A+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000095-CDC1" id="pone.0000095-CDC1"></a>(CDC) CfDCaP (2006) New laboratory assay for diagnostic testing of avian influenza A/H5 (Asian Lineage). MMWR Morb Mortal Wkly Rep  10: 127.  <ul class="find" data-citedArticleID="920460"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=New+laboratory+assay+for+diagnostic+testing+of+avian+influenza+A%2FH5+%28Asian+Lineage%29.&amp;auth=&amp;atitle=New+laboratory+assay+for+diagnostic+testing+of+avian+influenza+A%2FH5+%28Asian+Lineage%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+laboratory+assay+for+diagnostic+testing+of+avian+influenza+A%2FH5+%28Asian+Lineage%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+laboratory+assay+for+diagnostic+testing+of+avian+influenza+A%2FH5+%28Asian+Lineage%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000095-Paulson1" id="pone.0000095-Paulson1"></a>Paulson JC (1985) The Receptors.  Conn M, editor. Orlando, FL: Academic. pp. 131–219.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000095-Connor1" id="pone.0000095-Connor1"></a>Connor RJ, Kawaoka Y, Webster RG, Paulson JC (1994) Receptor specificity in human, avian, and equine H2 and H3 influenza virus isolates. Virology  205: 17–23.  <ul class="find" data-citedArticleID="920462" data-doi="10.1006/viro.1994.1615"><li><a href="http://dx.doi.org/10.1006/viro.1994.1615" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Receptor+specificity+in+human%2C+avian%2C+and+equine+H2+and+H3+influenza+virus+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Receptor+specificity+in+human%2C+avian%2C+and+equine+H2+and+H3+influenza+virus+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000095-Matrosovich1" id="pone.0000095-Matrosovich1"></a>Matrosovich M, Tuzikov A, Bovin N, Gambaryan A, Klimov A, et al.  (2000) Early alterations of the receptor-binding properties of H1, H2, and H3 avian influenza virus hemagglutinins after their introduction into mammals. J Virol  74: 8502–8512.  <ul class="find" data-citedArticleID="920488" data-doi="10.1128/jvi.74.18.8502-8512.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.18.8502-8512.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+alterations+of+the+receptor-binding+properties+of+H1%2C+H2%2C+and+H3+avian+influenza+virus+hemagglutinins+after+their+introduction+into+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+alterations+of+the+receptor-binding+properties+of+H1%2C+H2%2C+and+H3+avian+influenza+virus+hemagglutinins+after+their+introduction+into+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000095-Ha1" id="pone.0000095-Ha1"></a>Ha Y, Stevens DJ, Skehel JJ, Wiley DC (2002) H5 avian and H9 swine influenza virus haemagglutinin structures: possible origin of influenza subtypes. Embo J  21: 865–875.  <ul class="find" data-citedArticleID="920478" data-doi="10.1093/emboj/21.5.865"><li><a href="http://dx.doi.org/10.1093/emboj/21.5.865" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=H5+avian+and+H9+swine+influenza+virus+haemagglutinin+structures%3A+possible+origin+of+influenza+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22H5+avian+and+H9+swine+influenza+virus+haemagglutinin+structures%3A+possible+origin+of+influenza+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000095-Stevens1" id="pone.0000095-Stevens1"></a>Stevens J, Blixt O, Tumpey TM, Taubenberger JK, Paulson JC, et al.  (2006) Structure and receptor specificity of the hemagglutinin from an H5N1 influenza virus. Science  312: 404–410.  <ul class="find" data-citedArticleID="920512" data-doi="10.1126/science.1124513"><li><a href="http://dx.doi.org/10.1126/science.1124513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+receptor+specificity+of+the+hemagglutinin+from+an+H5N1+influenza+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+receptor+specificity+of+the+hemagglutinin+from+an+H5N1+influenza+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000095-Vines1" id="pone.0000095-Vines1"></a>Vines A, Wells K, Matrosovich M, Castrucci MR, Ito T, et al.  (1998) The role of influenza A virus hemagglutinin residues 226 and 228 in receptor specificity and host range restriction. J Virol  72: 7626–7631.  <ul class="find" data-citedArticleID="920522"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+role+of+influenza+A+virus+hemagglutinin+residues+226+and+228+in+receptor+specificity+and+host+range+restriction.&amp;auth=&amp;atitle=The+role+of+influenza+A+virus+hemagglutinin+residues+226+and+228+in+receptor+specificity+and+host+range+restriction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+influenza+A+virus+hemagglutinin+residues+226+and+228+in+receptor+specificity+and+host+range+restriction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+influenza+A+virus+hemagglutinin+residues+226+and+228+in+receptor+specificity+and+host+range+restriction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000095-Stevens2" id="pone.0000095-Stevens2"></a>Stevens J, Blixt O, Glaser L, Taubenberger JK, Palese P, et al.  (2006) Glycan microarray analysis of the hemagglutinins from modern and pandemic influenza viruses reveals different receptor specificities. J Mol Biol  355: 1143–1155.  <ul class="find" data-citedArticleID="920514" data-doi="10.1016/j.jmb.2005.11.002"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.11.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glycan+microarray+analysis+of+the+hemagglutinins+from+modern+and+pandemic+influenza+viruses+reveals+different+receptor+specificities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glycan+microarray+analysis+of+the+hemagglutinins+from+modern+and+pandemic+influenza+viruses+reveals+different+receptor+specificities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000095-Reid1" id="pone.0000095-Reid1"></a>Reid AH, Fanning TG, Hultin JV, Taubenberger JK (1999) Origin and evolution of the 1918 “Spanish” influenza virus hemagglutinin gene. Proc Natl Acad Sci U S A  96: 1651–1656.  <ul class="find" data-citedArticleID="920500" data-doi="10.1073/pnas.96.4.1651"><li><a href="http://dx.doi.org/10.1073/pnas.96.4.1651" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+and+evolution+of+the+1918+%26%23x201c%3BSpanish%26%23x201d%3B+influenza+virus+hemagglutinin+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+and+evolution+of+the+1918+%26%23x201c%3BSpanish%26%23x201d%3B+influenza+virus+hemagglutinin+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000095-Banks1" id="pone.0000095-Banks1"></a>Banks J, Plowright L (2003) Additional glycosylation at the receptor binding site of the hemagglutinin (HA) for H5 and H7 viruses may be an adaptation to poultry hosts, but does it influence pathogenicity? Avian Dis  47: 942–950.  <ul class="find" data-citedArticleID="920454" data-doi="10.1637/0005-2086-47.s3.942"><li><a href="http://dx.doi.org/10.1637/0005-2086-47.s3.942" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Additional+glycosylation+at+the+receptor+binding+site+of+the+hemagglutinin+%28HA%29+for+H5+and+H7+viruses+may+be+an+adaptation+to+poultry+hosts%2C+but+does+it+influence+pathogenicity%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Additional+glycosylation+at+the+receptor+binding+site+of+the+hemagglutinin+%28HA%29+for+H5+and+H7+viruses+may+be+an+adaptation+to+poultry+hosts%2C+but+does+it+influence+pathogenicity%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000095-Bosch1" id="pone.0000095-Bosch1"></a>Bosch FX, Garten W, Klenk HD, Rott R (1981) Proteolytic cleavage of influenza virus hemagglutinins: primary structure of the connecting peptide between HA1 and HA2 determines proteolytic cleavability and pathogenicity of Avian influenza viruses. Virology  113: 725–735.  <ul class="find" data-citedArticleID="920456" data-doi="10.1016/0042-6822(81)90201-4"><li><a href="http://dx.doi.org/10.1016/0042-6822(81)90201-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteolytic+cleavage+of+influenza+virus+hemagglutinins%3A+primary+structure+of+the+connecting+peptide+between+HA1+and+HA2+determines+proteolytic+cleavability+and+pathogenicity+of+Avian+influenza+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteolytic+cleavage+of+influenza+virus+hemagglutinins%3A+primary+structure+of+the+connecting+peptide+between+HA1+and+HA2+determines+proteolytic+cleavability+and+pathogenicity+of+Avian+influenza+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000095-Ohuchi1" id="pone.0000095-Ohuchi1"></a>Ohuchi M, Orlich M, Ohuchi R, Simpson BE, Garten W, et al.  (1989) Mutations at the cleavage site of the hemagglutinin after the pathogenicity of influenza virus A/chick/Penn/83 (H5N2). Virology  168: 274–280.  <ul class="find" data-citedArticleID="920494" data-doi="10.1016/0042-6822(89)90267-5"><li><a href="http://dx.doi.org/10.1016/0042-6822(89)90267-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+at+the+cleavage+site+of+the+hemagglutinin+after+the+pathogenicity+of+influenza+virus+A%2Fchick%2FPenn%2F83+%28H5N2%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+at+the+cleavage+site+of+the+hemagglutinin+after+the+pathogenicity+of+influenza+virus+A%2Fchick%2FPenn%2F83+%28H5N2%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000095-Ronaghi1" id="pone.0000095-Ronaghi1"></a>Ronaghi M, Uhlen M, Nyren P (1998) A sequencing method based on real-time pyrophosphate. Science  281: 363.365.  <ul class="find" data-citedArticleID="920502" data-doi="10.1126/science.281.5375.363"><li><a href="http://dx.doi.org/10.1126/science.281.5375.363" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sequencing+method+based+on+real-time+pyrophosphate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sequencing+method+based+on+real-time+pyrophosphate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000095-Bright1" id="pone.0000095-Bright1"></a>Bright RA, Medina MJ, Xu X, Perez-Oronoz G, Wallis TR, et al.  (2005) Incidence of adamantane resistance among influenza A (H3N2) viruses isolated worldwide from 1994 to 2005: a cause for concern. Lancet  366: 1175–1181.  <ul class="find" data-citedArticleID="920458" data-doi="10.1016/s0140-6736(05)67338-2"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)67338-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incidence+of+adamantane+resistance+among+influenza+A+%28H3N2%29+viruses+isolated+worldwide+from+1994+to+2005%3A+a+cause+for+concern." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incidence+of+adamantane+resistance+among+influenza+A+%28H3N2%29+viruses+isolated+worldwide+from+1994+to+2005%3A+a+cause+for+concern.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000095-Gharizadeh1" id="pone.0000095-Gharizadeh1"></a>Gharizadeh B, Zheng B, Akhras M, Ghaderi M, Jejelowo O, et al.  (2006) Sentinel-base DNA genotyping using multiple sequencing primers for high-risk human papillomaviruses. Mol Cell Probes  20: 230–238.  <ul class="find" data-citedArticleID="920474" data-doi="10.1016/j.mcp.2006.01.002"><li><a href="http://dx.doi.org/10.1016/j.mcp.2006.01.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sentinel-base+DNA+genotyping+using+multiple+sequencing+primers+for+high-risk+human+papillomaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sentinel-base+DNA+genotyping+using+multiple+sequencing+primers+for+high-risk+human+papillomaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000095-Scheffner1" id="pone.0000095-Scheffner1"></a>Scheffner M (1998) Ubiquitin, E6-AP, and their role in p53 inactivation. Pharmacol Ther  78: 129–139.  <ul class="find" data-citedArticleID="920504" data-doi="10.1016/s0163-7258(98)00003-5"><li><a href="http://dx.doi.org/10.1016/s0163-7258(98)00003-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ubiquitin%2C+E6-AP%2C+and+their+role+in+p53+inactivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ubiquitin%2C+E6-AP%2C+and+their+role+in+p53+inactivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000095-Li1" id="pone.0000095-Li1"></a>Li KS, Guan Y, Wang J, Smith GJD, Xu KM, et al.  (2004) Genesis of a highly pathogenic and potentially pandemic H5N1 influenza virus in eastern Asia. Nature  430: 209–213.  <ul class="find" data-citedArticleID="920486" data-doi="10.1038/nature02746"><li><a href="http://dx.doi.org/10.1038/nature02746" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genesis+of+a+highly+pathogenic+and+potentially+pandemic+H5N1+influenza+virus+in+eastern+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genesis+of+a+highly+pathogenic+and+potentially+pandemic+H5N1+influenza+virus+in+eastern+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000095-Hatta1" id="pone.0000095-Hatta1"></a>Hatta M, Asano Y, Masunaga K, Ito T, Okazaki K, et al.  (2000) Mapping of functional domains on the influenza A virus RNA polymerase PB2 molecule using monoclonal antibodies. Arch Virol  145: 1947–1961.  <ul class="find" data-citedArticleID="920482" data-doi="10.1007/s007050070068"><li><a href="http://dx.doi.org/10.1007/s007050070068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+functional+domains+on+the+influenza+A+virus+RNA+polymerase+PB2+molecule+using+monoclonal+antibodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+functional+domains+on+the+influenza+A+virus+RNA+polymerase+PB2+molecule+using+monoclonal+antibodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000095-Obenauer1" id="pone.0000095-Obenauer1"></a>Obenauer JC, Denson J, Mehta PK, Su XP, Mukatira S, et al.  (2006) Large-scale sequence analysis of avian influenza isolates. Science  311: 1576–1580.  <ul class="find" data-citedArticleID="920492" data-doi="10.1126/science.1121586"><li><a href="http://dx.doi.org/10.1126/science.1121586" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+sequence+analysis+of+avian+influenza+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+sequence+analysis+of+avian+influenza+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000095-Thompson1" id="pone.0000095-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="920520" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000095-Kendal1" id="pone.0000095-Kendal1"></a>Kendal AP, Pereira MS, Skehel JJ (1982)  Hemagglutination inhibition. In: Kendal AP PM, Skehel JJ, editors. Concepts and procedures for laboratory-based influenza surveillance. pp. B17–35.  Centers for Disease Control and Prevention and Pan-American Health Organization, Atlanta.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000095-Cover1" id="pone.0000095-Cover1"></a>Cover TM, Thomas JA (1991) Elements of information theory. New York: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0000095-Gharizadeh2" id="pone.0000095-Gharizadeh2"></a>Gharizadeh B, Akhras M, Unemo M, Wretlind B, Nyren P, et al.  (2005) Detection of gyrA mutations associated with ciprofloxacin resistance in Neisseria gonorrhoeae by rapid and reliable pre-programmed short DNA sequencing. Int J Antimicrob Agents  26: 486–490.  <ul class="find" data-citedArticleID="920476" data-doi="10.1016/j.ijantimicag.2005.08.017"><li><a href="http://dx.doi.org/10.1016/j.ijantimicag.2005.08.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+gyrA+mutations+associated+with+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae+by+rapid+and+reliable+pre-programmed+short+DNA+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+gyrA+mutations+associated+with+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae+by+rapid+and+reliable+pre-programmed+short+DNA+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000095-Cristianini1" id="pone.0000095-Cristianini1"></a>Cristianini N, Shawe-Taylor J (2000) An introduction to support vector machines and other kernel-based learning methods. Cambridge, U.K.; New York: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0000095-R1" id="pone.0000095-R1"></a>  R Statistical Package <a href="http://www.r-project.org/">http://www.r-project.org/</a>
               .  <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0000095-Duda1" id="pone.0000095-Duda1"></a>Duda RO, Hart PE, Stork DG (2001) Pattern classification. New York; Chichester [England]: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0000095-Meyer1" id="pone.0000095-Meyer1"></a>Meyer D (2006) Support Vector Machines.   Technische Universit¨at Wien, Austria. pp. Interface to libsvm in package e1071.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.XML" value="107167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.PDF" value="463464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g001.PNG_L" value="2233941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g001.PNG_M" value="241554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g001.PNG_S" value="5900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g001.TIF" value="2865842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g001.PNG_I" value="71909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g002.PNG_L" value="11557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g002.PNG_M" value="15144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g002.PNG_S" value="1764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g002.TIF" value="34558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g002.PNG_I" value="7944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g003.PNG_L" value="204632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g003.PNG_M" value="81798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g003.PNG_S" value="3354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g003.TIF" value="179730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g003.PNG_I" value="29753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g004.PNG_L" value="1893248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g004.PNG_M" value="209594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g004.PNG_S" value="8509"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g004.TIF" value="2531580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g004.PNG_I" value="159833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t001.PNG_L" value="259777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t001.PNG_M" value="33672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t001.PNG_S" value="2307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t001.TIF" value="11490828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t001.PNG_I" value="14238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t002.PNG_L" value="113284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t002.PNG_M" value="16089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t002.PNG_S" value="1849"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t002.TIF" value="5214750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.t002.PNG_I" value="6776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g005.PNG_L" value="360047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g005.PNG_M" value="27377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g005.PNG_S" value="2173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g005.TIF" value="447728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000095.g005.PNG_I" value="10322"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000095&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000095" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000095&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000095&volume=&issue=&title=Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses&author_name=Nader%20Pourmand%2C%20Lisa%20Diamond%2C%20Rebecca%20Garten%2C%20Julianna%20P.%20Erickson%2C%20Jochen%20Kumm%2C%20Ruben%20O.%20Donis%2C%20Ronald%20W.%20Davis&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000095" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000095" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000095" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000095&amp;t=Rapid%20and%20Highly%20Informative%20Diagnostic%20Assay%20for%20H5N1%20Influenza%20Viruses" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000095&title=Rapid%20and%20Highly%20Informative%20Diagnostic%20Assay%20for%20H5N1%20Influenza%20Viruses&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000095&amp;title=Rapid%20and%20Highly%20Informative%20Diagnostic%20Assay%20for%20H5N1%20Influenza%20Viruses" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000095" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000095&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Rapid and Highly Informative Diagnostic Assay for H5N1 Influenza Viruses';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000095';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Rapid%20and%20Highly%20Informative%20Diagnostic%20Assay%20for%20H5N1%20Influenza%20Viruses http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000095" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000095" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Avian+influenza+A+viruses%22" title="Search for articles in the subject area:'Avian influenza A viruses'"><div class="flagText">Avian influenza A viruses</div></a>
              <div data-categoryid="43251" data-articleid="23028"
                   data-categoryname="Avian influenza A viruses"
                   class="flagImage" title="Flag 'Avian influenza A viruses' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23028"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22H5N1%22" title="Search for articles in the subject area:'H5N1'"><div class="flagText">H5N1</div></a>
              <div data-categoryid="47163" data-articleid="23028"
                   data-categoryname="H5N1"
                   class="flagImage" title="Flag 'H5N1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Influenza%22" title="Search for articles in the subject area:'Influenza'"><div class="flagText">Influenza</div></a>
              <div data-categoryid="48257" data-articleid="23028"
                   data-categoryname="Influenza"
                   class="flagImage" title="Flag 'Influenza' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23028"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+sequencing%22" title="Search for articles in the subject area:'RNA sequencing'"><div class="flagText">RNA sequencing</div></a>
              <div data-categoryid="19765" data-articleid="23028"
                   data-categoryname="RNA sequencing"
                   class="flagImage" title="Flag 'RNA sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="23028"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23028"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2783'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5547'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2770&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
