

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000059"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000059" />

    <meta name="citation_title" content="Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective"/>
    <meta itemprop="name" content="Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective"/>

      <meta name="citation_author" content="R. Henrik Nilsson"/>
            <meta name="citation_author_institution" content="Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden"/>
      <meta name="citation_author" content="Martin Ryberg"/>
            <meta name="citation_author_institution" content="Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden"/>
      <meta name="citation_author" content="Erik Kristiansson"/>
            <meta name="citation_author_institution" content="Department of Mathematical Statistics, Chalmers University of Technology, Göteborg, Sweden"/>
      <meta name="citation_author" content="Kessy Abarenkov"/>
            <meta name="citation_author_institution" content="Institute of Botany and Ecology, University of Tartu, Tartu, Estonia"/>
      <meta name="citation_author" content="Karl-Henrik Larsson"/>
            <meta name="citation_author_institution" content="Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden"/>
      <meta name="citation_author" content="Urmas Kõljalg"/>
            <meta name="citation_author_institution" content="Institute of Botany and Ecology, University of Tartu, Tartu, Estonia"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000059.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e59"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Biological identifications through DNA barcodes.; citation_author=PDN Hebert; citation_author=A Cywinska; citation_author=SL Ball; citation_author=JR deWaard; citation_journal_title=Proc R Soc Lond B; citation_volume=270; citation_number=1; citation_pages=313-321; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Towards writing the encyclopaedia of life: an introduction to DNA barcoding.; citation_author=V Savolainen; citation_author=RB Cowan; citation_author=AP Vogler; citation_author=GK Roderick; citation_author=R Lane; citation_journal_title=Philos Transact R Soc B; citation_volume=360; citation_number=2; citation_pages=1805-1811; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=GenBank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=3; citation_pages=D16-D20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=galaxie-CGI scripts for sequence identification through automated phylogenetic analysis.; citation_author=RH Nilsson; citation_author=K-H Larsson; citation_author=BM Ursing; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=4; citation_pages=1447-1452; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Defining operational taxonomic units using DNA barcode data.; citation_author=M Blaxter; citation_author=J Mann; citation_author=T Chapman; citation_author=F Thomas; citation_author=C Whitton; citation_journal_title=Philos Transact R Soc B; citation_volume=360; citation_number=5; citation_pages=1935-1943; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The magnitude of fungal diversity: the 1.5 million species estimate revisited.; citation_author=DL Hawksworth; citation_journal_title=Mycol Res; citation_volume=105; citation_number=6; citation_pages=1422-1432; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Approaching the taxonomic affiliation of unidentified sequences in public databases–an example from the mycorrhizal fungi.; citation_author=RH Nilsson; citation_author=E Kristiansson; citation_author=M Ryberg; citation_author=K-H Larsson; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=7; citation_pages=178; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the unreliability of published DNA sequences.; citation_author=PD Bridge; citation_author=PJ Roberts; citation_author=BM Spooner; citation_author=G Panchal; citation_journal_title=New Phytol; citation_volume=160; citation_number=8; citation_pages=43-48; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=9; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evolutionary studies of mycorrhizal fungi: recent advances and future directions.; citation_author=TD Bruns; citation_author=RP Shefferson; citation_journal_title=Can J Bot; citation_volume=82; citation_number=10; citation_pages=1122-1132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=TaxI: a software tool for DNA barcoding using distance methods.; citation_author=D Steinke; citation_author=M Vences; citation_author=W Salzburger; citation_author=A Meyer; citation_journal_title=Philos Transact R Soc B; citation_volume=360; citation_number=11; citation_pages=1975-1980; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Glomales rRNA gene diversity-all that glistens is not necessarily glomalean?; citation_author=JP Clapp; citation_author=A Rodriguez; citation_author=JC Dodd; citation_journal_title=Mycorrhiza; citation_volume=12; citation_number=12; citation_pages=269-270; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Glomeromycota rRNA genes–the diversity of myths?; citation_author=A Schüßler; citation_author=D Schwarzott; citation_author=C Walker; citation_journal_title=Mycorrhiza; citation_volume=13; citation_number=13; citation_pages=233-236; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics.; citation_author=TJ White; citation_author=TD Bruns; citation_author=S Lee; citation_author=JW Taylor; citation_number=14; citation_pages=315-322; citation_date=1990; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Ribosomal DNA: molecular evolution and phylogenetic inference.; citation_author=DM Hillis; citation_author=MT Dixon; citation_journal_title=Q Rev Biol; citation_volume=66; citation_number=15; citation_pages=411-453; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Ribosomal ITS sequences and plant phylogenetic inference.; citation_author=I Álvarez; citation_author=JF Wendel; citation_journal_title=Mol Phylogenet Evol; citation_volume=29; citation_number=16; citation_pages=417-434; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The molecular revolution in ectomycorrhizal ecology: peeking into the black-box.; citation_author=TR Horton; citation_author=TD Bruns; citation_journal_title=Mol Ecol; citation_volume=10; citation_number=17; citation_pages=1855-1871; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Automated phylogenetic taxonomy: an example in the homobasidiomycetes (mushroom-forming fungi).; citation_author=DS Hibbett; citation_author=RH Nilsson; citation_author=M Snyder; citation_author=M Fonseca; citation_author=J Costanzo; citation_journal_title=Syst Biol; citation_volume=54; citation_number=18; citation_pages=660-668; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=UNITE: a database providing web-based methods for the molecular identification of ectomycorrhizal fungi.; citation_author=U Kõljalg; citation_author=KH Larsson; citation_author=K Abarenkov; citation_author=RH Nilsson; citation_author=IJ Alexander; citation_journal_title=New Phytol; citation_volume=166; citation_number=19; citation_pages=1063-1068; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=What are bacterial species?; citation_author=FM Cohan; citation_journal_title=Annu Rev of Microbiol; citation_volume=56; citation_number=20; citation_pages=457-487; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection of plot-level changes in ectomycorrhizal communities across years in an old-growth mixed-conifer forest.; citation_author=A Izzo; citation_author=J Agbowo; citation_author=TD Bruns; citation_journal_title=New Phytol; citation_volume=166; citation_number=21; citation_pages=619-630; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Critical factors for assembling a high volume of DNA barcodes.; citation_author=M Hajibabaei; citation_author=JR deWaard; citation_author=NV Ivanova; citation_author=S Ratnasingham; citation_author=RT Dooh; citation_journal_title=Philos Transact R Soc B; citation_volume=360; citation_number=22; citation_pages=1959-1967; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA barcoding: error rates based on comprehensive sampling.; citation_author=CP Meyer; citation_author=G Paulay; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=23; citation_pages=e422; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA barcoding is no substitute for taxonomy.; citation_author=MC Ebach; citation_author=C Holdrege; citation_journal_title=Nature; citation_volume=434; citation_number=24; citation_pages=697; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA barcoding: promise and pitfalls.; citation_author=C Moritz; citation_author=C Cicero; citation_journal_title=PLoS Biology; citation_volume=2; citation_number=25; citation_pages=e354; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000059" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective"/>
    <meta name="twitter:description" content="BackgroundDNA sequences are increasingly seen as one of the primary information sources for species identification in many organism groups. Such approaches, popularly known as barcoding, are underpinned by the assumption that the reference databases used for comparison are sufficiently complete and feature correctly and informatively annotated entries.Methodology/Principal FindingsThe present study uses a large set of fungal DNA sequences from the inclusive International Nucleotide Sequence Database to show that the taxon sampling of fungi is far from complete, that about 20% of the entries may be incorrectly identified to species level, and that the majority of entries lack descriptive and up-to-date annotations.ConclusionsThe problems with taxonomic reliability and insufficient annotations in public DNA repositories form a tangible obstacle to sequence-based species identification, and it is manifest that the greatest challenges to biological barcoding will be of taxonomical, rather than technical, nature."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000059.t001"/>

  <meta property="og:title" content="Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000059" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7594'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1144'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9816&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000059">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000059" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000059&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            R. Henrik Nilsson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:henrik.nilsson@botany.gu.se">henrik.nilsson@botany.gu.se</a>
            </p>

                <p>Affiliation:
                  Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martin Ryberg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Erik Kristiansson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Mathematical Statistics, Chalmers University of Technology, Göteborg, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kessy Abarenkov, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Botany and Ecology, University of Tartu, Tartu, Estonia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karl-Henrik Larsson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant and Environmental Sciences, Göteborg University, Göteborg, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Urmas Kõljalg
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Botany and Ecology, University of Tartu, Tartu, Estonia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000059</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000059" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000059" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000059" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000059" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000059-t001" data-doi="info:doi/10.1371/journal.pone.0000059" data-uri="info:doi/10.1371/journal.pone.0000059.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000059.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000059">Reader Comments (7)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000059" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>DNA sequences are increasingly seen as one of the primary information sources for species identification in many organism groups. Such approaches, popularly known as barcoding, are underpinned by the assumption that the reference databases used for comparison are sufficiently complete and feature correctly and informatively annotated entries.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The present study uses a large set of fungal DNA sequences from the inclusive International Nucleotide Sequence Database to show that the taxon sampling of fungi is far from complete, that about 20% of the entries may be incorrectly identified to species level, and that the majority of entries lack descriptive and up-to-date annotations.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The problems with taxonomic reliability and insufficient annotations in public DNA repositories form a tangible obstacle to sequence-based species identification, and it is manifest that the greatest challenges to biological barcoding will be of taxonomical, rather than technical, nature.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nilsson RH, Ryberg M, Kristiansson E, Abarenkov K, Larsson K-H, et al.  (2006) Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective. PLoS ONE 1(1):
          e59.
            doi:10.1371/journal.pone.0000059</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> October 16, 2006; <strong>Accepted:</strong> October 26, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> © 2006 Nilsson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the foundations of Helge Ax:son Johnson, Wilhelm and Martina Lundgren, and Lars Hierta.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Species identification relies heavily on DNA sequence comparison in many groups of organisms, particularly those in which distinguishing morphological characteristics come thinly seeded. Such processes, increasingly known as <em>barcoding</em>, hold great promise for simplifying and standardizing the identification of biological specimens <a href="#pone.0000059-Hebert1">[1]</a>–<a href="#pone.0000059-Savolainen1">[2]</a>. The course of action is straightforward: some predefined DNA region of the organism is sequenced and compared for similarity in an inclusive database for sequence data such as the longstanding International Nucleotide Sequence Database <a href="#pone.0000059-Benson1">[3]</a> (INSD: GenBank, EMBL, and DDBJ) which is the most widely used sequence repository in the field. The result is used in the taxonomic annotation of the new sequence, which usually is submitted to the database under the inferred name. Such a procedure leans on three central but, interestingly, somewhat implicit assumptions <a href="#pone.0000059-Nilsson1">[4]</a>–<a href="#pone.0000059-Blaxter1">[5]</a>:</p>


<ol class="order">

<li>that the reference database features a satisfactory taxonomic sampling of sequences</li>

<li>that the sequences in the reference database are correctly identified and annotated</li>

<li>that the process of translating the comparison into species names is standardized, universally adopted, and not easily misunderstood</li>

</ol><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In the case of fungi, none of these criteria are met to any satisfactory extent:</p>


<ol class="order">

<li>Less than 1% of the estimated 1.5 million extant species of fungi have been sequenced for the ITS region, the most widely used locus for species identification in the fungi <a href="#pone.0000059-Hawksworth1">[6]</a>–<a href="#pone.0000059-Nilsson2">[7]</a>.</li>

<li>It has been suggested that a considerable portion-perhaps as much as 20%-of all fungal sequences deposited in INSD may be incorrectly annotated to species level <a href="#pone.0000059-Bridge1">[8]</a>, though rigorous statistics are lacking.</li>

<li>Newly generated sequences are typically identified using DNA-similarity searches like BLAST <a href="#pone.0000059-Altschul1">[9]</a>. These are bound by criteria 1 and 2 and are associated with a range of additional complications such that their use for taxonomic identification has been cautioned in recent years <a href="#pone.0000059-Blaxter1">[5]</a>, <a href="#pone.0000059-Bruns1">[10]</a>–<a href="#pone.0000059-Steinke1">[11]</a>.</li>

</ol><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>One does not have to stretch ones imagination to see how unfortunate decisions and circumstances, once effectuated, will not only remain in but also propagate through the various public sequence repositories through subsequent searches and submissions. Indeed, contemporary scientific literature is strewn with cases of mistaken species identities resulting from compromised DNA sequence comparison <a href="#pone.0000059-Bruns1">[10]</a>, <a href="#pone.0000059-Clapp1">[12]</a>–<a href="#pone.0000059-Schler1">[13]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>But exactly how much reliance could be placed on the taxonomic annotations of publicly available sequences–how large a proportion of these are disputable? The present study aims to generalize previous sectional estimates by <em>in silico</em> analysis of a large set of fungal DNA sequences from INSD for various statistics. On the basis of the odd 51,000 fungal ITS sequences currently available, we carried out serial sequence similarity analysis and, for a subset of the sequences, external comparison to present objective statistics on the taxonomic reliability of fungal ITS sequences in INSD. Fungi form a large and ubiquitous group of organisms where species identification on morphological grounds often falls short and where the use of DNA sequence analysis for eukaryote species identification was once pioneered <a href="#pone.0000059-White1">[14]</a>. They therefore constitute an appealing model group for estimation of taxonomic reliability in public sequence databases under authentic circumstances.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The ITS region is a multi-copy, transcribed but non-coding and easily amplified region of the ribosomal DNA <a href="#pone.0000059-Hillis1">[15]</a>–<a href="#pone.0000059-lvarez1">[16]</a>. It has become the standard locus for species identification–often even delimitation-in the fungi due to its high variability <a href="#pone.0000059-Horton1">[17]</a>–<a href="#pone.0000059-Hibbett1">[18]</a>. A local copy of all 51354 INSD fungal ITS sequences was created and kept up-to-date through weekly synchronization (<a href="#pone.0000059.s001">Supporting Information</a>). To nuance the representation of fungal diversity, the sequences were divided into those <em>fully identified</em> (identified to species level) and those <em>insufficiently identified</em> (not identified to species level) using regular expressions on the INSD organism specification field <a href="#pone.0000059-Nilsson2">[7]</a>. All sequences were compared against each other for similarity using NCBI-BLAST and the results were analyzed for various statistics (<a href="#pone-0000059-t001">Table 1</a>). As a second reference point, for the cases where the fungal taxonomic reference database UNITE <a href="#pone.0000059-Kljalg1">[19]</a> featured fully identified but independent ITS sequences from species also represented in the INSD dataset, the UNITE sequences were run against the latter to estimate ones chances of obtaining the correct name as the topmost BLAST match in INSD. For purposes of sequence comparison with BLAST, a <em>thorough match</em> was conservatively defined (<a href="#pone.0000059.s001">Supporting Information</a>) as to be far more stringent than the informal 3% rule of sequence dissimilarity sometimes evoked for species delimitation among bacteria and other organisms <a href="#pone.0000059-Cohan1">[20]</a>–<a href="#pone.0000059-Izzo1">[21]</a>. Sequences in match-pairs that satisfy the thorough match-criteria are referred to as <em>applicable</em>.</p>
<div class="figure" id="pone-0000059-t001"><div class="img"><a name="pone-0000059-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000059.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000059" data-uri="info:doi/10.1371/journal.pone.0000059.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000059.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000059.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000059.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000059.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000059.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000059.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000059.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000059.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000059.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>A fungal perspective on data reliability in INSD.</span></strong></p><span>doi:10.1371/journal.pone.0000059.t001</span></div></div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Proportion of compromised taxonomic annotations</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The results are summarized in <a href="#pone-0000059-t001">Table 1</a>, which portrays a variegated picture of the taxonomic status of publicly indexed fungal sequences. Based on the conservative criteria defined for a thorough BLAST match and the discriminative variability of the ITS region, one would expect any such thoroughly matching pair to be conspecific. Yet 11% of all 15491 applicable sequences find thorough matches in other congeneric but heterospecific sequences, and another 7% among species of a different genus. When synonyms are accounted for, these correspond to 3231 distinct accession numbers such that a minimum of 10% and a maximum of 21% of the applicable sequences have compromised taxonomic annotations (<a href="#pone.0000059.s001">Supporting Information</a>). These entries form, in turn, the best matches of 5% of all insufficiently identified sequences, such that in a worst-case scenario, one in every twenty insufficiently identified sequences finds its most similar counterpart among entries whose taxonomic annotation can be questioned.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>That 10–21% of the INSD sequences have incorrect or unsatisfactory taxonomic annotations translates into a matter of concern for the researcher seeking to establish the taxonomic affiliation of newly generated sequences. To obtain a clearer picture of the extent to which this process will be hampered by the compromised entries, the sequence identification procedure was reproduced through the use of UNITE, a highly filtered, closed-submission taxonomic database for reliable ITS-based identification of mycorrhizal fungi (<a href="http://unite.ut.ee">http://unite.ut.ee</a>). We employed the 240 species present in both INSD and the UNITE databases such that the UNITE sequences were used as input for comparison in INSD (<a href="#pone.0000059.s001">Supporting Information</a>). As the taxonomic affiliations of the UNITE sequences are well-known and -documented, the proportion of times a different taxonomic affiliation is suggested by INSD-even though a conspecific ITS sequence is present therein-represents a rational estimate of the impact of taxonomically compromised annotations in INSD. We found that one has on average a 20% (49/240) chance of obtaining a different species name on top of the INSD BLAST hit list, each such case hinting at a compromised annotation of either the topmost match or the purportedly conspecific INSD sequence (or even both). In a further 8% (20/240) of the cases, the correct species name was present in the topmost region of the hit list but was obscured by the presence of insufficiently identified sequences, such that one would be reluctant to annotate ones sequence after the best fully identified match. Jointly these estimates imply that the taxonomic and nomenclatural problems in public sequence databases are more far-reaching than previously assumed and that this has considerable repercussion on sequence-based species identification.</p>


<h4>Insufficiently identified sequences, orphans, and other compounding factors</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>More than 27% of all fungal ITS sequences in INSD are insufficiently identified, and the majority (74%) of these find their best match in other insufficiently identified-rather than fully identified-sequences. Similarly, over 90% of the fully identified sequences find their best matches in other fully identified sequences. In other words, the two sequence classes constitute two largely separate entities, both of which convey information not present in the other.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Six percent of all sequences over 350 bp lack good BLAST matches altogether (i.e., have an E-value of &gt;0.0 as reported by BLAST). These outliers probably represent a mix of species whose closest relatives have not been sequenced and species that lack close, extant relatives. Two thirds of these sequences are fully identified; the oldest sequence with an unsatisfactory BLAST match has resided in INSD for a full 14 years. Interestingly, 85% of the fully identified sequences that fail to find a thorough match do so in the presence of other purportedly congeneric sequences, and 35% even in the presence of other purportedly conspecific sequences.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The observation that a comparatively small set of sequences explains a disproportionally large part of the results (<a href="#pone-0000059-t001">Table 1</a>) is probably best viewed as an indication of a highly patchy and non-random taxonomic distribution of species sampled. Roughly half of both the identified and the insufficiently identified sequences do not constitute the best BLAST match of any other sequence. Similarly, 76% of all mycological studies account for 100% of all best BLAST matches, such that there are over 1000 studies in INSD whose sequences do not constitute the best match of any other sequence (a <em>study</em> is defined as a distinct combination of the INSD AUTHORS and TITLE fields as to correspond to a published or unpublished scientific manuscript). A full 55% of all sequences are best matched by another sequence from the same study.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Sequence annotations play an important role for the researcher trying to verify alleged names and taxonomic integrities. However, many entries in INSD prove to be both devoid of vital information and outdated (<a href="#pone-0000059-t001">Table 1</a>). For example, 82% of the sequences lack explicit reference to a voucher specimen, 63% are not tagged with specimen country of origin, and 42% of all sequences are marked as not having been published in spite of the fact that about 40% of these indeed have been (<a href="#pone.0000059.s001">Supporting Information</a>). Although 14% of all sequences contain DNA ambiguities, less than 1% of all sequences have ever been updated. That these issues pose a further obstacle to sequence identification needs little iteration.</p>


<h4>Primary data - a challenge for biological barcoding</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The present study suggests that the taxonomic reliability in public databases is not satisfactory, and that the problem shows little tendency for self-amelioration over time (<a href="#pone-0000059-t001">Table 1</a>). This is worrisome, particularly since DNA sequences have been opined as the primary information source in barcoding-type approaches to species identification (where reference DNA sequences serve as arbiters-barcodes-of conspecificity). It is apparent from <a href="#pone-0000059-t001">Table 1</a> that the major sequence databases are not optimally suited to serve as barcoding engines as they presently stand; new techniques and strategies for data indexation and verification will have to be explored to address the above shortcomings <a href="#pone.0000059-Blaxter1">[5]</a>, <a href="#pone.0000059-Hajibabaei1">[22]</a>. It is, however, not in technology that the greatest challenge to barcoding lies; rather, it is in the integrity of the primary data itself <a href="#pone.0000059-Meyer1">[23]</a>–<a href="#pone.0000059-Moritz1">[25]</a> (<a href="#pone-0000059-t001">Table 1</a>). As the results presented herein suggest, the relation of species and species names-taxonomy - to barcoding could be only one: that of the <em>primus motor</em>. No technical feats could ever make up for compromised primary data or lack of such data altogether.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The large body of insufficiently identified fungi in INSD constitutes a silent plea for a wide and generalized sequencing effort of well-identified and -annotated [type] specimens residing in herbaria worldwide to form the basis for such barcoding initiatives. This will without doubt be a painstaking undertaking involving taxonomic experts in all groups of fungi. The approach taken by the UNITE database has been to cover as many genera of fungi as possible at the temporary expense of intrageneric completeness. That approach finds support in the present study: in order to avoid the current situation where insufficiently identified sequences amass and obscure similarity searches in the public sequence databases, select reference sequences covering the whole range of fungal diversity need be made available as early on as possible.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The species is in many ways the basic unit in biology, and the ever-increasing rate at which DNA sequences are released and used for scientific research prompts us to make any effort to verify that these are tagged with correct names. Sadly, more than 10% of all publicly available fungal ITS sequences have compromised taxonomic annotations, and the information needed to evaluate whether any given name is reasonable is in many cases simply not there. The inherent difficulty in species identification in the fungi, however, suggests that these estimates need not necessarily reflect the status of the total body of DNA sequences. Even so, caution and patience should be attributes of anyone seeking to identify species through DNA sequence data alone.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Barcoding-type approaches will doubtlessly be a central and most valuable element in future species identification, though contemporary major sequence repositories are not optimally suited for such operation. While we can expect technological advancements to eliminate many of the problems faced at present, the taxonomical aspect of the DNA sequences remains a substantial concern. Taxonomy lays at the heart of sequence-mediated species identification, and unlike the latter it forms a poor candidate for automation. Sadly the declining number of taxonomists is a problem for which no shortcuts exist and moreover one whose immediate resolution does not seem to be looming on the horizon.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000059"></div><a name="pone.0000059.s001" id="pone.0000059.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000059.s001">Technical Information. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Detailed technical description of estimates used in the manuscript</p>
<p class="siDoi">doi:10.1371/journal.pone.0000059.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.10 MB PDF)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HN MR EK. Performed the experiments: HN MR EK KA. Analyzed the data: HN MR EK KA. Contributed reagents/materials/analysis tools: KA UK. Wrote the paper: HN MR EK KL UK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000059-Hebert1" id="pone.0000059-Hebert1"></a>Hebert PDN, Cywinska A, Ball SL, deWaard JR (2003) Biological identifications through DNA barcodes. Proc R Soc Lond B  270: 313–321.  <ul class="find" data-citedArticleID="917770" data-doi="10.1098/rspb.2002.2218"><li><a href="http://dx.doi.org/10.1098/rspb.2002.2218" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biological+identifications+through+DNA+barcodes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biological+identifications+through+DNA+barcodes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000059-Savolainen1" id="pone.0000059-Savolainen1"></a>Savolainen V, Cowan RB, Vogler AP, Roderick GK, Lane R (2005) Towards writing the encyclopaedia of life: an introduction to DNA barcoding. Philos Transact R Soc B  360: 1805–1811.  <ul class="find" data-citedArticleID="917790" data-doi="10.1098/rstb.2005.1730"><li><a href="http://dx.doi.org/10.1098/rstb.2005.1730" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+writing+the+encyclopaedia+of+life%3A+an+introduction+to+DNA+barcoding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+writing+the+encyclopaedia+of+life%3A+an+introduction+to+DNA+barcoding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000059-Benson1" id="pone.0000059-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2006) GenBank. Nucleic Acids Res  34: D16–D20.  <ul class="find" data-citedArticleID="917752" data-doi="10.1093/nar/gkm929"><li><a href="http://dx.doi.org/10.1093/nar/gkm929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenBank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenBank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000059-Nilsson1" id="pone.0000059-Nilsson1"></a>Nilsson RH, Larsson K-H, Ursing BM (2004) galaxie-CGI scripts for sequence identification through automated phylogenetic analysis. Bioinformatics  20: 1447–1452.  <ul class="find" data-citedArticleID="917786" data-doi="10.1093/bioinformatics/bth119"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=galaxie-CGI+scripts+for+sequence+identification+through+automated+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22galaxie-CGI+scripts+for+sequence+identification+through+automated+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000059-Blaxter1" id="pone.0000059-Blaxter1"></a>Blaxter M, Mann J, Chapman T, Thomas F, Whitton C, et al.  (2005) Defining operational taxonomic units using DNA barcode data. Philos Transact R Soc B  360: 1935–1943.  <ul class="find" data-citedArticleID="917754" data-doi="10.1098/rstb.2005.1725"><li><a href="http://dx.doi.org/10.1098/rstb.2005.1725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defining+operational+taxonomic+units+using+DNA+barcode+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defining+operational+taxonomic+units+using+DNA+barcode+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000059-Hawksworth1" id="pone.0000059-Hawksworth1"></a>Hawksworth DL (2001) The magnitude of fungal diversity: the 1.5 million species estimate revisited. Mycol Res  105: 1422–1432.  <ul class="find" data-citedArticleID="917768" data-doi="10.1017/s0953756201004725"><li><a href="http://dx.doi.org/10.1017/s0953756201004725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+magnitude+of+fungal+diversity%3A+the+1.5+million+species+estimate+revisited." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+magnitude+of+fungal+diversity%3A+the+1.5+million+species+estimate+revisited.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000059-Nilsson2" id="pone.0000059-Nilsson2"></a>Nilsson RH, Kristiansson E, Ryberg M, Larsson K-H (2005) Approaching the taxonomic affiliation of unidentified sequences in public databases–an example from the mycorrhizal fungi. BMC Bioinformatics  6: 178.  <ul class="find" data-citedArticleID="917788"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Approaching+the+taxonomic+affiliation+of+unidentified+sequences+in+public+databases%26%23x2013%3Ban+example+from+the+mycorrhizal+fungi.&amp;auth=&amp;atitle=Approaching+the+taxonomic+affiliation+of+unidentified+sequences+in+public+databases%26%23x2013%3Ban+example+from+the+mycorrhizal+fungi." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approaching+the+taxonomic+affiliation+of+unidentified+sequences+in+public+databases%26%23x2013%3Ban+example+from+the+mycorrhizal+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approaching+the+taxonomic+affiliation+of+unidentified+sequences+in+public+databases%26%23x2013%3Ban+example+from+the+mycorrhizal+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000059-Bridge1" id="pone.0000059-Bridge1"></a>Bridge PD, Roberts PJ, Spooner BM, Panchal G (2003) On the unreliability of published DNA sequences. New Phytol  160: 43–48.  <ul class="find" data-citedArticleID="917756" data-doi="10.1046/j.1469-8137.2003.00861.x"><li><a href="http://dx.doi.org/10.1046/j.1469-8137.2003.00861.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+unreliability+of+published+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+unreliability+of+published+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000059-Altschul1" id="pone.0000059-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="917750" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000059-Bruns1" id="pone.0000059-Bruns1"></a>Bruns TD, Shefferson RP (2004) Evolutionary studies of mycorrhizal fungi: recent advances and future directions. Can J Bot  82: 1122–1132.  <ul class="find" data-citedArticleID="917758" data-doi="10.1139/b04-021"><li><a href="http://dx.doi.org/10.1139/b04-021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+studies+of+mycorrhizal+fungi%3A+recent+advances+and+future+directions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+studies+of+mycorrhizal+fungi%3A+recent+advances+and+future+directions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000059-Steinke1" id="pone.0000059-Steinke1"></a>Steinke D, Vences M, Salzburger W, Meyer A (2005) TaxI: a software tool for DNA barcoding using distance methods. Philos Transact R Soc B  360: 1975–1980.  <ul class="find" data-citedArticleID="917794" data-doi="10.1098/rstb.2005.1729"><li><a href="http://dx.doi.org/10.1098/rstb.2005.1729" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TaxI%3A+a+software+tool+for+DNA+barcoding+using+distance+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TaxI%3A+a+software+tool+for+DNA+barcoding+using+distance+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000059-Clapp1" id="pone.0000059-Clapp1"></a>Clapp JP, Rodriguez A, Dodd JC (2002) Glomales rRNA gene diversity-all that glistens is not necessarily glomalean? Mycorrhiza  12: 269–270.  <ul class="find" data-citedArticleID="917760" data-doi="10.1007/s00572-002-0175-8"><li><a href="http://dx.doi.org/10.1007/s00572-002-0175-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glomales+rRNA+gene+diversity-all+that+glistens+is+not+necessarily+glomalean%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glomales+rRNA+gene+diversity-all+that+glistens+is+not+necessarily+glomalean%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000059-Schler1" id="pone.0000059-Schler1"></a>Schüßler A, Schwarzott D, Walker C (2003) Glomeromycota rRNA genes–the diversity of myths? Mycorrhiza  13: 233–236.  <ul class="find" data-citedArticleID="917792" data-doi="10.1007/s00572-003-0250-9"><li><a href="http://dx.doi.org/10.1007/s00572-003-0250-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glomeromycota+rRNA+genes%26%23x2013%3Bthe+diversity+of+myths%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glomeromycota+rRNA+genes%26%23x2013%3Bthe+diversity+of+myths%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000059-White1" id="pone.0000059-White1"></a>White TJ, Bruns TD, Lee S, Taylor JW (1990)  Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics. In: Innis MA, Gelfand DH, Sninsky JJ, editors. PCR Protocols: a Guide to Methods and Applications. New York: Academic Press. pp. 315–322.  <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000059-Hillis1" id="pone.0000059-Hillis1"></a>Hillis DM, Dixon MT (1991) Ribosomal DNA: molecular evolution and phylogenetic inference. Q Rev Biol  66: 411–453.  <ul class="find" data-citedArticleID="917774" data-doi="10.1086/417338"><li><a href="http://dx.doi.org/10.1086/417338" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+DNA%3A+molecular+evolution+and+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+DNA%3A+molecular+evolution+and+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000059-lvarez1" id="pone.0000059-lvarez1"></a>Álvarez I, Wendel JF (2003) Ribosomal ITS sequences and plant phylogenetic inference. Mol Phylogenet Evol  29: 417–434.  <ul class="find" data-citedArticleID="917798" data-doi="10.1016/s1055-7903(03)00208-2"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00208-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+ITS+sequences+and+plant+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+ITS+sequences+and+plant+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000059-Horton1" id="pone.0000059-Horton1"></a>Horton TR, Bruns TD (2001) The molecular revolution in ectomycorrhizal ecology: peeking into the black-box. Mol Ecol  10: 1855–1871.  <ul class="find" data-citedArticleID="917776" data-doi="10.1046/j.0962-1083.2001.01333.x"><li><a href="http://dx.doi.org/10.1046/j.0962-1083.2001.01333.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+revolution+in+ectomycorrhizal+ecology%3A+peeking+into+the+black-box." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+revolution+in+ectomycorrhizal+ecology%3A+peeking+into+the+black-box.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000059-Hibbett1" id="pone.0000059-Hibbett1"></a>Hibbett DS, Nilsson RH, Snyder M, Fonseca M, Costanzo J, et al.  (2005) Automated phylogenetic taxonomy: an example in the homobasidiomycetes (mushroom-forming fungi). Syst Biol  54: 660–668.  <ul class="find" data-citedArticleID="917772"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Automated+phylogenetic+taxonomy%3A+an+example+in+the+homobasidiomycetes+%28mushroom-forming+fungi%29.&amp;auth=&amp;atitle=Automated+phylogenetic+taxonomy%3A+an+example+in+the+homobasidiomycetes+%28mushroom-forming+fungi%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+phylogenetic+taxonomy%3A+an+example+in+the+homobasidiomycetes+%28mushroom-forming+fungi%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+phylogenetic+taxonomy%3A+an+example+in+the+homobasidiomycetes+%28mushroom-forming+fungi%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000059-Kljalg1" id="pone.0000059-Kljalg1"></a>Kõljalg U, Larsson KH, Abarenkov K, Nilsson RH, Alexander IJ, et al.  (2005) UNITE: a database providing web-based methods for the molecular identification of ectomycorrhizal fungi. New Phytol  166: 1063–1068.  <ul class="find" data-citedArticleID="917780" data-doi="10.1111/j.1469-8137.2005.01376.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.2005.01376.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UNITE%3A+a+database+providing+web-based+methods+for+the+molecular+identification+of+ectomycorrhizal+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UNITE%3A+a+database+providing+web-based+methods+for+the+molecular+identification+of+ectomycorrhizal+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000059-Cohan1" id="pone.0000059-Cohan1"></a>Cohan FM (2002) What are bacterial species? Annu Rev of Microbiol  56: 457–487.  <ul class="find" data-citedArticleID="917762" data-doi="10.1146/annurev.micro.56.012302.160634"><li><a href="http://dx.doi.org/10.1146/annurev.micro.56.012302.160634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+are+bacterial+species%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+are+bacterial+species%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000059-Izzo1" id="pone.0000059-Izzo1"></a>Izzo A, Agbowo J, Bruns TD (2005) Detection of plot-level changes in ectomycorrhizal communities across years in an old-growth mixed-conifer forest. New Phytol  166: 619–630.  <ul class="find" data-citedArticleID="917778" data-doi="10.1111/j.1469-8137.2005.01354.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.2005.01354.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+plot-level+changes+in+ectomycorrhizal+communities+across+years+in+an+old-growth+mixed-conifer+forest." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+plot-level+changes+in+ectomycorrhizal+communities+across+years+in+an+old-growth+mixed-conifer+forest.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000059-Hajibabaei1" id="pone.0000059-Hajibabaei1"></a>Hajibabaei M, deWaard JR, Ivanova NV, Ratnasingham S, Dooh RT, et al.  (2005) Critical factors for assembling a high volume of DNA barcodes. Philos Transact R Soc B  360: 1959–1967.  <ul class="find" data-citedArticleID="917766" data-doi="10.1098/rstb.2005.1727"><li><a href="http://dx.doi.org/10.1098/rstb.2005.1727" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Critical+factors+for+assembling+a+high+volume+of+DNA+barcodes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Critical+factors+for+assembling+a+high+volume+of+DNA+barcodes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000059-Meyer1" id="pone.0000059-Meyer1"></a>Meyer CP, Paulay G (2005) DNA barcoding: error rates based on comprehensive sampling. PLoS Biol  3: e422.  <ul class="find" data-citedArticleID="917782" data-doi="10.1371/journal.pbio.0030422"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030422" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+barcoding%3A+error+rates+based+on+comprehensive+sampling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+barcoding%3A+error+rates+based+on+comprehensive+sampling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000059-Ebach1" id="pone.0000059-Ebach1"></a>Ebach MC, Holdrege C (2005) DNA barcoding is no substitute for taxonomy. Nature  434: 697.  <ul class="find" data-citedArticleID="917764" data-doi="10.1038/434697b"><li><a href="http://dx.doi.org/10.1038/434697b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+barcoding+is+no+substitute+for+taxonomy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+barcoding+is+no+substitute+for+taxonomy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000059-Moritz1" id="pone.0000059-Moritz1"></a>Moritz C, Cicero C (2004) DNA barcoding: promise and pitfalls. PLoS Biology  2: e354.  <ul class="find" data-citedArticleID="917784" data-doi="10.1371/journal.pbio.0020354"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+barcoding%3A+promise+and+pitfalls." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+barcoding%3A+promise+and+pitfalls.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.XML" value="63958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.PDF" value="64334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.t001.PNG_L" value="336543"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.t001.PNG_M" value="50773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.t001.PNG_S" value="2885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.t001.TIF" value="12602676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.t001.PNG_I" value="19789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000059.s001.PDF" value="100253"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000059&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000059" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000059&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000059&volume=&issue=&title=Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective&author_name=R.%20Henrik%20Nilsson%2C%20Martin%20Ryberg%2C%20Erik%20Kristiansson%2C%20Kessy%20Abarenkov%2C%20Karl-Henrik%20Larsson%2C%20Urmas%20K%C3%B5ljalg&start_page=1&end_page=4" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000059" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000059" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000059" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000059&amp;t=Taxonomic%20Reliability%20of%20DNA%20Sequences%20in%20Public%20Sequence%20Databases%3A%20A%20Fungal%20Perspective" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000059&title=Taxonomic%20Reliability%20of%20DNA%20Sequences%20in%20Public%20Sequence%20Databases%3A%20A%20Fungal%20Perspective&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000059&amp;title=Taxonomic%20Reliability%20of%20DNA%20Sequences%20in%20Public%20Sequence%20Databases%3A%20A%20Fungal%20Perspective" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000059" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000059&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000059';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Taxonomic%20Reliability%20of%20DNA%20Sequences%20in%20Public%20Sequence%20Databases%3A%20A%20Fungal%20Perspective http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000059" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000059" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22BLAST+algorithm%22" title="Search for articles in the subject area:'BLAST algorithm'"><div class="flagText">BLAST algorithm</div></a>
              <div data-categoryid="31827" data-articleid="22958"
                   data-categoryname="BLAST algorithm"
                   class="flagImage" title="Flag 'BLAST algorithm' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="22958"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="22958"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+classification%22" title="Search for articles in the subject area:'Fungal classification'"><div class="flagText">Fungal classification</div></a>
              <div data-categoryid="24953" data-articleid="22958"
                   data-categoryname="Fungal classification"
                   class="flagImage" title="Flag 'Fungal classification' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungi%22" title="Search for articles in the subject area:'Fungi'"><div class="flagText">Fungi</div></a>
              <div data-categoryid="43047" data-articleid="22958"
                   data-categoryname="Fungi"
                   class="flagImage" title="Flag 'Fungi' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="22958"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Spacer+regions%22" title="Search for articles in the subject area:'Spacer regions'"><div class="flagText">Spacer regions</div></a>
              <div data-categoryid="41167" data-articleid="22958"
                   data-categoryname="Spacer regions"
                   class="flagImage" title="Flag 'Spacer regions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Species+delimitation%22" title="Search for articles in the subject area:'Species delimitation'"><div class="flagText">Species delimitation</div></a>
              <div data-categoryid="39023" data-articleid="22958"
                   data-categoryname="Species delimitation"
                   class="flagImage" title="Flag 'Species delimitation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8247'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3768'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3806&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=11027">This paper is cited by the below peer-reviewed papers</a><br>Posted by RHNi</p>
      <p><a href="/annotation/listThread.action?root=9725">Article discussed at the Barcode of Life Blog</a><br>Posted by RHNi</p>
      <p><a href="/annotation/listThread.action?root=41503">Well written paper, but...</a><br>Posted by sandersson</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
