

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001184"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001184" />

    <meta name="citation_title" content="Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome"/>
    <meta itemprop="name" content="Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome"/>

      <meta name="citation_author" content="Shicai Fan"/>
            <meta name="citation_author_institution" content="MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China"/>
            <meta name="citation_author_institution" content="School of Automation Engineering, University of Electronic Science and Technology of China, Chengdu, China"/>
      <meta name="citation_author" content="Fang Fang"/>
            <meta name="citation_author_institution" content="MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China"/>
      <meta name="citation_author" content="Xuegong Zhang"/>
            <meta name="citation_author_institution" content="MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China"/>
      <meta name="citation_author" content="Michael Q. Zhang"/>
            <meta name="citation_author_institution" content="MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China"/>
            <meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001184.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1184"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=CpG islands as gene markers in the vertebrate nucleus.; citation_author=AP Bird; citation_journal_title=Trends Genet; citation_volume=3; citation_number=1; citation_pages=342-347; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Escape from gene silencing in ICF syndrome: evidence for advanced replication time as a major determinant.; citation_author=RS Hansen; citation_author=R Stoger; citation_author=C Wijmenga; citation_author=AM Stanek; citation_author=TK Canfield; citation_journal_title=Hum Mol Genet; citation_volume=9; citation_number=2; citation_pages=2575-2587; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Recent advances in X-chromosome inactivation.; citation_author=E Heard; citation_journal_title=Curr Opin Cell Biol; citation_volume=16; citation_number=3; citation_pages=247-255; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The fundamental role of epigenetic events in cancer.; citation_author=PA Jones; citation_author=SB Baylin; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=4; citation_pages=415-428; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mammalian epigenomics: reprogramming the genome for development and therapy.; citation_author=W Reik; citation_author=F Santos; citation_author=W Dean; citation_journal_title=Theriogenology; citation_volume=59; citation_number=5; citation_pages=21-32; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA methylation represses transcription in vivo.; citation_author=Z Siegfried; citation_author=S Eden; citation_author=M Mendelsohn; citation_author=X Feng; citation_author=BZ Tsuberi; citation_journal_title=Nat Genet; citation_volume=22; citation_number=6; citation_pages=203-206; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Epigenetic reprogramming in mammalian development.; citation_author=W Reik; citation_author=W Dean; citation_author=J Walter; citation_journal_title=Science; citation_volume=293; citation_number=7; citation_pages=1089-1093; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Germline epimutation of MLH1 in individuals with multiple cancers.; citation_author=CM Suter; citation_author=DI Martin; citation_author=RL Ward; citation_journal_title=Nat Genet; citation_volume=36; citation_number=8; citation_pages=497-501; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Amount and distribution of 5-methylcytosine in human DNA from different types of tissues or cells.; citation_author=E Melanie; citation_author=AG-S Miguel; citation_author=H Lan-Hsiang; citation_author=M Rose Marie; citation_author=CK Kenneth; citation_journal_title=Nucleic Acids Res; citation_volume=10; citation_number=9; citation_pages=2709-2721; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=CpG islands in vertebrate genomes.; citation_author=M Gardiner-Garden; citation_author=M Frommer; citation_journal_title=J Mol Biol; citation_volume=196; citation_number=10; citation_pages=261-282; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Large-scale methylation analysis of human genomic DNA reveals tissue-specific differences between the methylation profiles of genes and pseudogenes.; citation_author=C Grunau; citation_author=W Hindermann; citation_author=A Rosenthal; citation_journal_title=Hum Mol Genet; citation_volume=9; citation_number=11; citation_pages=2651-2663; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Association of tissue-specific differentially methylated regions (TDMs) with differential gene expression.; citation_author=F Song; citation_author=JF Smith; citation_author=MT Kimura; citation_author=AD Morrow; citation_author=T Matsuyama; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=12; citation_pages=3336-3341; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Large-scale human promoter mapping using CpG islands.; citation_author=IP Ioshikhes; citation_author=MQ Zhang; citation_journal_title=Nat Genet; citation_volume=26; citation_number=13; citation_pages=61-63; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sp1 elements protect a CpG island from de novo methylation.; citation_author=M Brandeis; citation_author=D Frank; citation_author=I Keshet; citation_author=Z Siegfried; citation_author=M Mendelsohn; citation_journal_title=Nature; citation_volume=371; citation_number=14; citation_pages=435-438; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The establishment and maintenance of DNA methylation patterns in mouse somatic cells.; citation_author=MS Turker; citation_journal_title=Semin Cancer Biol; citation_volume=9; citation_number=15; citation_pages=329-337; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A distinct sequence (ATAAA)n separates methylated and unmethylated domains at the 5′-end of the GSTP1 CpG island.; citation_author=DS Millar; citation_author=CL Paul; citation_author=PL Molloy; citation_author=SJ Clark; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=16; citation_pages=24893-24899; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=DNA binding sites for putative methylation boundaries in the unmethylated region of the BRCA1 promoter.; citation_author=DT Butcher; citation_author=DN Mancini-DiNardo; citation_author=TK Archer; citation_author=DI Rodenhiser; citation_journal_title=Int J Cancer; citation_volume=111; citation_number=17; citation_pages=669-678; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcription factor Sp1 is essential for early embryonic development but dispensable for cell growth and differentiation.; citation_author=M Marin; citation_author=A Karis; citation_author=P Visser; citation_author=F Grosveld; citation_author=S Philipsen; citation_journal_title=Cell; citation_volume=89; citation_number=18; citation_pages=619-628; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Prediction of methylated CpGs in DNA sequences using a support vector machine.; citation_author=M Bhasin; citation_author=H Zhang; citation_author=EL Reinherz; citation_author=PA Reche; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=19; citation_pages=4302-4308; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CpG island methylation in human lymphocytes is highly correlated with DNA sequence, repeats, and predicted DNA structure.; citation_author=C Bock; citation_author=M Paulsen; citation_author=S Tierling; citation_author=T Mikeska; citation_author=T Lengauer; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=20; citation_pages=e26; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Predicting aberrant CpG island methylation.; citation_author=FA Feltus; citation_author=EK Lee; citation_author=JF Costello; citation_author=C Plass; citation_author=PM Vertino; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=21; citation_pages=12253-12258; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computational prediction of methylation status in human genomic sequences.; citation_author=R Das; citation_author=N Dimitrova; citation_author=Z Xuan; citation_author=RA Rollins; citation_author=F Haghighi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=22; citation_pages=10713-10716; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Predicting methylation status of CpG islands in the human brain.; citation_author=F Fang; citation_author=S Fan; citation_author=X Zhang; citation_author=MQ Zhang; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=23; citation_pages=2204-2209; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Large-scale structure of genomic methylation patterns.; citation_author=RA Rollins; citation_author=F Haghighi; citation_author=JR Edwards; citation_author=R Das; citation_author=MQ Zhang; citation_journal_title=Genome Res; citation_volume=16; citation_number=24; citation_pages=157-163; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene silencing in mammalian cells and the spread of DNA methylation.; citation_author=MS Turker; citation_journal_title=Oncogene; citation_volume=21; citation_number=25; citation_pages=5388-5393; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mining ChIP-chip data for transcription factor and cofactor binding sites.; citation_author=AD Smith; citation_author=P Sumazin; citation_author=D Das; citation_author=MQ Zhang; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=26; citation_pages=i403-i412; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes.; citation_author=V Matys; citation_author=OV Kel-Margoulis; citation_author=E Fricke; citation_author=I Liebich; citation_author=S Land; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=27; citation_pages=D108-D110; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A comprehensive analysis of allelic methylation status of CpG islands on human chromosome 21q.; citation_author=Y Yamada; citation_author=H Watanabe; citation_author=F Miura; citation_author=H Soejima; citation_author=M Uchiyama; citation_journal_title=Genome Res; citation_volume=14; citation_number=28; citation_pages=247-266; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microarray-based DNA methylation profiling: technology and applications.; citation_author=A Schumacher; citation_author=P Kapranov; citation_author=Z Kaminsky; citation_author=J Flanagan; citation_author=A Assadzadeh; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=29; citation_pages=528-542; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structure and mechanism of the RNA polymerase II transcription machinery.; citation_author=S Hahn; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=30; citation_pages=394-403; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The role of chromatin during transcription.; citation_author=B Li; citation_author=M Carey; citation_author=JL Workman; citation_journal_title=Cell; citation_volume=128; citation_number=31; citation_pages=707-719; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=High-resolution profiling of histone methylations in the human genome.; citation_author=A Barski; citation_author=S Cuddapah; citation_author=K Cui; citation_author=TY Roh; citation_author=DE Schones; citation_journal_title=Cell; citation_volume=129; citation_number=32; citation_pages=823-837; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Methylation of histone H3 Lys 4 in coding regions of active genes.; citation_author=BE Bernstein; citation_author=EL Humphrey; citation_author=RL Erlich; citation_author=R Schneider; citation_author=P Bouman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=33; citation_pages=8695-8700; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genomic maps and comparative analysis of histone modifications in human and mouse.; citation_author=BE Bernstein; citation_author=M Kamal; citation_author=K Lindblad-Toh; citation_author=S Bekiranov; citation_author=DK Bailey; citation_journal_title=Cell; citation_volume=120; citation_number=34; citation_pages=169-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The primary function of a redundant Sp1 binding site in the mouse aprt gene promoter is to block epigenetic gene inactivation.; citation_author=P Mummaneni; citation_author=P Yates; citation_author=J Simpson; citation_author=J Rose; citation_author=MS Turker; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=35; citation_pages=5163-5169; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=CTCF mediates methylation-sensitive enhancer-blocking activity at the H19/Igf2 locus.; citation_author=AT Hark; citation_author=CJ Schoenherr; citation_author=DJ Katz; citation_author=RS Ingram; citation_author=JM Levorse; citation_journal_title=Nature; citation_volume=405; citation_number=36; citation_pages=408-409; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Boundaries between chromosomal domains of X inactivation and escape bind CTCF and lack CpG methylation during early development.; citation_author=GN Filippova; citation_author=MK Cheng; citation_author=JM Moore; citation_author=JP Truong; citation_author=YJ Hu; citation_journal_title=Dev Cell; citation_volume=8; citation_number=37; citation_pages=31-42; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Promoter CpG hypomethylation and transcription factor EGR1 hyperactivate heparanase expression in bladder cancer.; citation_author=T Ogishima; citation_author=H Shiina; citation_author=JE Breault; citation_author=M Terashima; citation_author=S Honda; citation_journal_title=Oncogene; citation_volume=24; citation_number=38; citation_pages=6765-6772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=HEN1 recognizes 21-24 nt small RNA duplexes and deposits a methyl group onto the 2′ OH of the 3′ terminal nucleotide.; citation_author=Z Yang; citation_author=YW Ebright; citation_author=B Yu; citation_author=X Chen; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=39; citation_pages=667-675; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA methylation patterns and epigenetic memory.; citation_author=AP Bird; citation_journal_title=Genes Dev; citation_volume=16; citation_number=40; citation_pages=6-21; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=DNA methylation and chromatin - unraveling the tangled web.; citation_author=KD Robertson; citation_journal_title=Oncogene; citation_volume=21; citation_number=41; citation_pages=5361-5379; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A bivalent chromatin structure marks key developmental genes in embryonic stem cells.; citation_author=BE Bernstein; citation_author=TS Mikkelsen; citation_author=X Xie; citation_author=M Kamal; citation_author=DJ Huebert; citation_journal_title=Cell; citation_volume=125; citation_number=42; citation_pages=315-326; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genes VIII: Person Prentice Hall.; citation_author=B Lewin; citation_number=43; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The mammalian epigenome.; citation_author=BE Bernstein; citation_author=A Meissner; citation_author=ES Lander; citation_journal_title=Cell; citation_volume=128; citation_number=44; citation_pages=669-681; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Temporal and regional changes in DNA methylation in the embryonic, extraembryonic and germ cell lineages during mouse embryo development.; citation_author=M Monk; citation_author=M Boubelik; citation_author=S Lehnert; citation_journal_title=Development; citation_volume=99; citation_number=45; citation_pages=371-382; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Developmental pattern of gene-specific DNA methylation in the mouse embryo and germ line.; citation_author=T Kafri; citation_author=M Ariel; citation_author=M Brandeis; citation_author=R Shemer; citation_author=L Urven; citation_journal_title=Genes Dev; citation_volume=6; citation_number=46; citation_pages=705-714; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Polymerase chain reaction-aided genomic sequencing of an X chromosome-linked CpG island: Methylation patterns suggest clonal inheritance, CpG site autonomy, and an explanation of activity state stability.; citation_author=GP Pfeifer; citation_author=SD Steigerwald; citation_author=RS Hansen; citation_author=SM Gartler; citation_author=AD Riggs; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=87; citation_number=47; citation_pages=8252-8256; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Comprehensive analysis of CpG islands in human chromosomes 21 and 22.; citation_author=D Takai; citation_author=PA Jones; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=48; citation_pages=3740-3745; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Aging, DNA methylation and cancer.; citation_author=JP Issa; citation_journal_title=Crit Rev Oncol Hematol; citation_volume=32; citation_number=49; citation_pages=31-43; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Microarray-based survey of CpG islands identifies concurrent hyper- and hypomethylation patterns in tissues derived from patients with breast cancer.; citation_author=A Piotrowski; citation_author=M Benetkiewicz; citation_author=U Menzel; citation_author=TD de Stahl; citation_author=K Mantripragada; citation_journal_title=Genes Chromosomes Cancer; citation_volume=45; citation_number=50; citation_pages=656-667; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Methylation profiling of twenty four genes and the concordant methylation behaviours of nineteen genes that may contribute to hepatocellular carcinogenesis.; citation_author=J Yu; citation_author=HY Zhang; citation_author=ZZ Ma; citation_author=W Lu; citation_author=YF Wang; citation_journal_title=Cell Res; citation_volume=13; citation_number=51; citation_pages=319-333; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA methylation and chromatin structure: the puzzling CpG islands.; citation_author=P Caiafa; citation_author=M Zampieri; citation_journal_title=J Cell Biochem; citation_volume=94; citation_number=52; citation_pages=257-265; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The logic of inductive inference.; citation_author=RA Fisher; citation_journal_title=Journal of the Royal Statistical Society; citation_volume=98; citation_number=53; citation_pages=39-82; citation_date=1935; " />
      <meta name="citation_reference" content="citation_title=Similarity of position frequency matrices for transcription factor binding sites.; citation_author=DE Schones; citation_author=P Sumazin; citation_author=MQ Zhang; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=54; citation_pages=307-313; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=55; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MONKEY: identifying conserved transcription-factor binding sites in multiple alignments using a binding site-specific evolutionary model.; citation_author=AM Moses; citation_author=DY Chiang; citation_author=DA Pollard; citation_author=VN Iyer; citation_author=MB Eisen; citation_journal_title=Genome Biol; citation_volume=5; citation_number=56; citation_pages=R98; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001184" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome"/>
    <meta name="twitter:description" content="BackgroundMajority of CpG dinucleotides in mammalian genomes tend to undergo DNA methylation, but most CpG islands are resistant to such epigenetic modification. Understanding about mechanisms that may lead to the methylation resistance of CpG islands is still very poor.Methodology/Principal FindingsUsing the genome-scale in vivo DNA methylation data from human brain, we investigated the flanking sequence features of methylation-resistant CpG islands, and discovered that there are several over-represented putative Transcription Factor Binding Sites (TFBSs) in methylation-resistant CpG islands, and a specific group of zinc finger protein binding sites are over-represented in boundary regions (&sim;400 bp) flanking such CpG islands. About 77% of the over-represented putative TFBSs are conserved among human, mouse and rat. We also observed the enrichment of 4 histone methylations in methylation-resistant CpG islands or their boundaries.Conclusions/SignificanceOur results suggest a possible mechanism that certain putative zinc finger protein binding sites over-represented in the boundary regions of the methylation-resistant CpG islands may block the spreading of methylation into these islands, and those TFBSs over-represented within the islands may both reinforce the methylation blocking and promote transcription. Some histone modifications may also enhance the immunity of the CpG islands against DNA methylation by augmenting these TFs' binding. We speculate that the dynamical equilibrium between methylation spreading and blocking is likely to be responsible for the establishment and maintenance of the relatively stable DNA methylation pattern in human somatic cells."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001184.g004"/>

  <meta property="og:title" content="Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001184" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1345'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4862'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9510&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001184">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001184" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001184&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shicai Fan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China, 
                  School of Automation Engineering, University of Electronic Science and Technology of China, Chengdu, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fang Fang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xuegong Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Q. Zhang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mzhang@cshl.edu">mzhang@cshl.edu</a></p>

                <p>Affiliations:
                  MOE Key Laboratory of Bioinformatics and Bioinformatics Division, TNLIST, Department of Automation, Tsinghua University, Beijing 100084, China, 
                  Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001184</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001184" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001184" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001184" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001184" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001184-g001" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001184-g002" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001184-t001" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001184-t002" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001184-t003" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001184-t004" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001184-g003" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001184-g004" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001184">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001184" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Majority of CpG dinucleotides in mammalian genomes tend to undergo DNA methylation, but most CpG islands are resistant to such epigenetic modification. Understanding about mechanisms that may lead to the methylation resistance of CpG islands is still very poor.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using the genome-scale <em>in vivo</em> DNA methylation data from human brain, we investigated the flanking sequence features of methylation-resistant CpG islands, and discovered that there are several over-represented putative Transcription Factor Binding Sites (TFBSs) in methylation-resistant CpG islands, and a specific group of zinc finger protein binding sites are over-represented in boundary regions (~400 bp) flanking such CpG islands. About 77% of the over-represented putative TFBSs are conserved among human, mouse and rat. We also observed the enrichment of 4 histone methylations in methylation-resistant CpG islands or their boundaries.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results suggest a possible mechanism that certain putative zinc finger protein binding sites over-represented in the boundary regions of the methylation-resistant CpG islands may block the spreading of methylation into these islands, and those TFBSs over-represented within the islands may both reinforce the methylation blocking and promote transcription. Some histone modifications may also enhance the immunity of the CpG islands against DNA methylation by augmenting these TFs' binding. We speculate that the dynamical equilibrium between methylation spreading and blocking is likely to be responsible for the establishment and maintenance of the relatively stable DNA methylation pattern in human somatic cells.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Fan S, Fang F, Zhang X, Zhang MQ (2007) Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome. PLoS ONE 2(11):
          e1184.
            doi:10.1371/journal.pone.0001184</p><p><strong>Academic Editor: </strong>Jörg Hoheisel, Deutsches Krebsforschungszentrum, Germany</p><p><strong>Received:</strong> June 23, 2007; <strong>Accepted:</strong> October 26, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Fan et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is supported in part by NSFC grants 60540420569 (MQZ and XZ) and 30625012 (XZ), the National Basic Research Program of China (2004CB518605) (XZ and MQZ), NIH grant HG01696 (MQZ) and the Changjiang Scholar Award of China (MQZ).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>DNA methylation is one of the most important epigenetic modifications. In mammalian genomes, DNA methylation occurs at the cytosine residue in the context 5′-CG-3′ (CpG dinucleotide) by virtue of DNA methyltransferases <a href="#pone.0001184-Bird1">[1]</a>. It has been reported that DNA methylation plays many important functional roles such as X-chromosome inactivation <a href="#pone.0001184-Hansen1">[2]</a>, <a href="#pone.0001184-Heard1">[3]</a>, the establishment and maintenance of tissue-specific gene expression profiles <a href="#pone.0001184-Jones1">[4]</a>–<a href="#pone.0001184-Siegfried1">[6]</a> as well as developmental programming regulation <a href="#pone.0001184-Reik2">[7]</a>, <a href="#pone.0001184-Suter1">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In human somatic cells, about 70–80% of CpG dinucleotides undergo methylation <a href="#pone.0001184-Melanie1">[9]</a>. Unmethylated CpG dinucleotides largely reside in genomic regions called CpG islands (CGIs) <a href="#pone.0001184-Bird1">[1]</a>. A CGI is a region where the CpGs are more dense than the genome average, commonly defined as a region with length of 200 bp or longer in which the G+C content is no less than 50% and the ratio of observed/expected CpGs is larger than 0.6 <a href="#pone.0001184-GardinerGarden1">[10]</a>. Although some CGIs could be methylated in the imprinted regions <a href="#pone.0001184-Jones1">[4]</a> or in the inactive X-chromosome <a href="#pone.0001184-Heard1">[3]</a>, <a href="#pone.0001184-Grunau1">[11]</a>, <a href="#pone.0001184-Song1">[12]</a>, most CGIs are generally resistant to DNA methylation <a href="#pone.0001184-Ioshikhes1">[13]</a>. But so far it is still poorly understood what mechanisms may lead to the methylation-resistance of these CGIs.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A few biological experiments focusing on some specific genes have indicated that certain <em>cis</em>-acting elements could act as boundaries to protect CGIs from methylation (by binding to the corresponding TFs), such as Sp1 elements in mouse <em>aprt</em> gene <a href="#pone.0001184-Brandeis1">[14]</a>, <a href="#pone.0001184-Turker1">[15]</a>, the (ATAAA)<sub>n</sub> repeated sequences in human <em>GSTP1</em> gene <a href="#pone.0001184-Millar1">[16]</a>, and Sp1 and CTCF elements in <em>BRCA1</em> gene <a href="#pone.0001184-Butcher1">[17]</a>. These elements are not universal, and it has been observed that the deletion of Sp1 in mouse <em>aprt</em> gene would not cause aberrant methylation of CGIs <a href="#pone.0001184-Marin1">[18]</a>. Therefore there must be some other <em>cis</em>-acting elements performing similar boundary roles and/or some other mechanisms that may help to resist methylation. In addition, there is little knowledge about location and length, if exist, of the protective boundaries around unmethylated CGIs.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In recent years, there emerged some computational works on the prediction of the methylation status of CpG dinucleotides <a href="#pone.0001184-Bhasin1">[19]</a>, CGIs <a href="#pone.0001184-Bock1">[20]</a>, <a href="#pone.0001184-Feltus1">[21]</a>, and CGI fragments <a href="#pone.0001184-Das1">[22]</a>, <a href="#pone.0001184-Fang1">[23]</a> based solely on DNA sequence features . They could reach prediction accuracies around 80%, which partially confirms people's speculation that there may exit sequence propensity for genomic DNA methylation.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Inspired by these success, we conducted a further investigation on the potential flanking sequence features of methylation-resistant CGIs (referred to as un-methylated or U-CGIs for simplicity) with a genome-scale methylation profiling dataset from human brain <a href="#pone.0001184-Rollins1">[24]</a>. We found that certain zinc finger protein binding sites, including Sp1 and CTCF sites, are over-represented in the boundary sequences of U-CGIs, and the core region of such boundaries appears to extend ~400 bp upstream or downstream from the island. There are also some over-represented <em>cis</em>-acting elements within U-CGIs, which may have dual functions of blocking DNA methylation spreading and promoting transcription. These results were validated to a certain degree on two other independent large-scale datasets. The study on a recent genome-scale histone modification dataset showed the enrichment of the four histone methylations in the U-CGIs or their boundaries, which implies that many of the TFs indeed bind to some of their putative binding sites. Based on our observations on these genome-scale data, we believe that the dynamical equilibrium hypothesis proposed for mouse <a href="#pone.0001184-Turker2">[25]</a> could be generalized to human: during the <em>de novo</em> methylation process in the genome, repetitive sequences (such as <em>Alu</em> sequences) act as DNA methylation center. As methylation spreads towards a CGI, several <em>cis</em>-acting elements in the boundaries of the CGI can recruit specific DNA binding proteins, such as Sp1 and CTCF, as battlefronts to prevent DNA methylation from encroaching into the CGI, and additional <em>cis</em>-elements within the U-CGI regions can further help reinforcing the methylation blocking activities. And these enriched histone methylations will also enhance the immunity of U-CGIs from methylation by augmenting the TFs' binding. These opposing (spreading and blocking) activities would come to a dynamical equilibrium under a given physiological condition of the cells to establish the observed genomic methylation patterns.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We got 304 U-CGIs and their corresponding flanking sequences from the methylation-resistant domains of Rollins <em>et al</em>'s genome-scale methylation data from human brain (see <a href="#s4">Materials and methods</a> for details). Another 210 methylation-prone CGIs (referred to as methylated CGIs or M-CGIs) and their flanking sequences were extracted from the methylation-prone domains of the same data set to serve as the background control samples. We used these data to explore the <em>cis</em>-element features of U-CGIs and their flanking sequences that are distinct from those of M-CGIs and their flanking sequences.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>For studying the sequence features, we split U-CGIs, M-CGIs and their corresponding flanking sequences into several 200 bp-long fragments (motivated by the average inter-nucleosome distance) as illustrated in <a href="#pone-0001184-g001">Figure 1</a>, and all samples were aligned by the two ends of the CpG islands (see <a href="#s4">Materials and Methods</a> for details). Then MOTIFCLASS <a href="#pone.0001184-Smith1">[26]</a> was applied to investigate the Transcription Factor Binding Site (TFBS) frequencies both in the fragments of U-CGIs and their flanking sequences. TFBSs used here are from TRANSFAC 9.4 <a href="#pone.0001184-Matys1">[27]</a>. We used a two-step hypothesis test for this study (see <a href="#s4">Materials and Methods</a> for details), and obtained all the significantly enriched TFBSs (Bonferroni-adjusted <em>p</em>-value cutoff 0.01) for each fragment of U-CGIs and flanking sequences. The enrichment of TFBSs is shown in <a href="#pone-0001184-g002">Figure 2</a>. In the figure, the region between the two ‘0’s on the <em>x</em>-axis represents the U-CGI region, the regions outside the two ‘0’s are the flanking regions, and the <em>y</em>-axis is the number of significant TFBSs in each fragment. From <a href="#pone-0001184-g002">Figure 2</a>, one can see that there are many enriched TFBSs in and around the U-CGIs, and they reach “plateaus” around the two ends of U-CGIs. These regions extend ~400 bp upstream and downstream from the U-CGIs.</p>
<div class="figure" id="pone-0001184-g001"><div class="img"><a name="pone-0001184-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Sketch of the division strategies.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>We split U&amp;M-CGIs into several fragments with equal width of 200 bp. As the median length of U-CGIs are much longer than M-CGIs, there are 6 fragments of U-CGIs (A1, B1, C1, D1, E1, F1) whereas there are only 2 fragments of M-CGIs (A2 and F2), and such fragments could cover most of the sequences of U&amp;M-CGIs. Applying the same division strategy to the flanking sequences corresponding to both U&amp;M-CGIs, we extended 800 bp upstream and downstream of CGIs, and got fragments of J1, I1, H1, G1, K1, L1, M1, N1, and J2, I2, H2, G2, K2, L2, M2, N2 respectively. We further define 400 bp upstream and downstream of U-CGIs (O1, P1) as the boundary sequences since there are few significant TFBSs outside such regions.</p>
<span>doi:10.1371/journal.pone.0001184.g001</span></div><div class="figure" id="pone-0001184-g002"><div class="img"><a name="pone-0001184-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The number of significantly over-represented TFBSs in each fragment of U-CGIs and their flanking sequences.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>The <em>x</em>-axis indicates the location of fragment relative to the 5′ or 3′ end of U-CGIs. Fragments insides the two ‘0’s correspond to A1, B1, C1, D1, E1 and F1 of <a href="#pone-0001184-g001">Figure 1</a>, and fragments that are in the 800 bp upstream and downstream of U-CGIs represent J1, I1, H1, G1 and K1, L1, M1, N1 of <a href="#pone-0001184-g001">Figure 1</a> respectively. The <em>y</em>-axis is the number of significantly enriched TFBSs in each fragment. We applied the Bonferroni-adjusted <em>p</em>-value cutoff 0.01 here. The sequences that are 400 bp upstream and downstream of U-CGIs are defined as the boundary sequences.</p>
<span>doi:10.1371/journal.pone.0001184.g002</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>To study whether these results are sensitive to the choice of <em>p</em>-value cutoff, we also experimented with other cutoffs. The number and distribution of significant TFBSs under <em>p</em>-value cutoffs of 0.05 and 0.001 (Bonferroni-adjusted) are shown in <a href="#pone.0001184.s001">Figures S1</a> and <a href="#pone.0001184.s002">S2</a> in Supplementary Materials. One can see that although the specific number of significant TFBSs in each fragment may vary with the different cutoffs, the fact that there are some common enriched TFBSs in U-CGIs and in their nearby flanking sequences remains to be true. One may further define the 400 bp upstream and downstream of U-CGIs as the boundary sequences (shown as O1 and P1 regions in <a href="#pone-0001184-g001">Figure 1</a>), as such flanking regions appear to be more robustly containing the enriched TFBSs.</p>

<h4>Over-represented TFBSs in methylation-resistant CGIs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>For extracting the specific TFBSs that are enriched in U-CGIs, we took TFBSs that are simultaneously significantly enriched in more than three U-CGI fragments (i.e., half of the counted fragments) as over-represented TFBSs in U-CGIs. Ten TFBSs satisfy this criterion and they are listed in <a href="#pone-0001184-t001">Table 1</a> (their corresponding motif logos are given in <a href="#pone.0001184.s004">Table S1</a> in Supplementary Materials). The <em>p</em>-values of these TFBSs by the two-step test were also listed in <a href="#pone-0001184-t001">Table 1</a>. We looked up the tissue expression pattern of the transcription factors (TFs) corresponding to these TFBSs in the database TissueDistributionDBs (<a href="http://genius.embnet.dkfz-heidelberg.de/menu/tissue_db/">http://genius.embnet.dkfz-heidelberg.de/​menu/tissue_db/</a>). Of the 10 over-represented TFBSs, 8 have been reported to be expressed in the brain (shown in the 2nd column of <a href="#pone-0001184-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001184-t001"><div class="img"><a name="pone-0001184-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Over-represented TFBSs and other novel motifs in U-CGI fragments.</span></strong></p><span>doi:10.1371/journal.pone.0001184.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We checked the annotations of these TFs in Swiss-Prot (<a href="http://cn.expasy.org/">http://cn.expasy.org/</a>), and found that among the 8 TFs reported to be expressed in human brain, the basic helix-loop-helix domain is shared by 3 TFs. To study the significance of this observation, we took the rest non-redundant vertebrate TFs in TRANSFAC 9.4 that are expressed in human brain as control and compared the number of these TFs that contain the basic helix-loop-helix domain (<a href="#pone-0001184-t002">Table 2</a>). One can see that the fraction of basic helix-loop-helix domain in the U-CGIs-over-represented TFs is marginally higher than that in the control TFs (<em>p</em>-value = 0.0544 by Fisher's Exact Test).</p>
<div class="figure" id="pone-0001184-t002"><div class="img"><a name="pone-0001184-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>The number of TFs with specific domains.</span></strong></p><span>doi:10.1371/journal.pone.0001184.t002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To investigate whether there are some novel motifs over-represented in the U-CGIs, we applied the program DME-b <a href="#pone.0001184-Smith1">[26]</a> to search for new motifs and applied the two-step hypothesis test to get the over-represented ones. After eliminating motifs that were similar to over-represented TFBSs, we got two over-represented novel motifs. They are also listed at the bottom of <a href="#pone-0001184-t001">Table 1</a>, and their logos are listed in supplementary <a href="#pone.0001184.s006">Table S3</a> together with the corresponding TRANSFAC TFBSs that are most similar to.</p>


<h4>Over-represented TFBSs in the boundaries of U-CGIs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In the boundary regions of U-CGIs, we took the TFBSs that are both significantly enriched in O1 segment and P1 segment as the over-represented TFBSs in the boundaries. There are 13 such TFBSs and they are listed in <a href="#pone-0001184-t003">Table 3</a> with their corresponding <em>p</em>-values. Their motif logos are in the supplementary <a href="#pone.0001184.s005">Table S2</a>. The corresponding TFs for 10 of these TFBSs are reported as expressed in human brain according to TissueDistributionDBs (also shown in <a href="#pone-0001184-t003">Table 3</a>). By checking the annotation in Swiss-Prot, 7 of these 10 TFs possess zinc finger (C2H2-type) domains (<a href="#pone-0001184-t004">Table 4</a>). Also the rest non-redundant vertebrate TFs expressed in human brain are chosen as the control and the number of TFs with such domain in control set is shown in <a href="#pone-0001184-t004">Table 4</a>. One can see that the fraction of zinc finger domain in these TFs corresponding to the over-represented TFBSs in U-CGI boundaries is much higher than that in the control set (<em>p</em>-value = 8.63e-4 by Fisher's Exact Test).</p>
<div class="figure" id="pone-0001184-t003"><div class="img"><a name="pone-0001184-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Over-represented TFBSs and other novel motifs in boundary sequences.</span></strong></p><span>doi:10.1371/journal.pone.0001184.t003</span></div><div class="figure" id="pone-0001184-t004"><div class="img"><a name="pone-0001184-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The number of TFs with specific domains.</span></strong></p><span>doi:10.1371/journal.pone.0001184.t004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We also looked for possible new motifs in the boundaries in the same way as mentioned in the last section, and got two over-represented novel motifs in the U-CGIs boundaries. They are listed at the bottom of <a href="#pone-0001184-t003">Table 3</a>, and their logos with the most similar known TFBSs can be found in the supplementary <a href="#pone.0001184.s007">Table S4</a>.</p>


<h4>Conservation of the putative TFBSs over-represented in U-CGIs and in the boundaries</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We investigated the sequence conservation of the over-represented putative TFBSs across human, mouse and rat to examine how many of these sites are potentially functional (see <a href="#s4">Materials and Methods</a> for details). We call a putative binding site significantly conserved if its <em>p</em>-value is less than 0.01. The proportion of significantly conserved binding sites among all the binding sites of each transcription factor is shown in supplementary <a href="#pone.0001184.s008">Tables S5</a> and <a href="#pone.0001184.s009">S6</a>. One can see that the average proportion of significantly conserved binding sites of all over-represented putative TFBSs is 77.39% (with standard deviation 6.7%), which suggests that most of these TFBSs are under functional (negative) selection.</p>


<h4>Validation on two independent datasets</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Yamada <em>et al</em> <a href="#pone.0001184-Yamada1">[28]</a> profiled the methylation status of CGIs on human chromosome 21q from peripheral blood leukocytes. From their data, we got 104 U-CGIs and applied the same procedure to check the significance of the identified over-represented TFBSs based on the Rollins <em>et al</em>'s data (see <a href="#s2">results</a> in <a href="#pone.0001184.s010">Tables S7</a> and <a href="#pone.0001184.s011">S8</a> in Supplementary Materials). It can be seen that 7 of the 10 TFBSs over-represented in U-CGIs and 10 of the 13 TFBSs over-represented in boundaries obtained in Rollins <em>et al</em>'s data are again significantly enriched in this dataset.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Recently, Schumacher <em>et al</em> <a href="#pone.0001184-Schumacher1">[29]</a> reported the profile of unmethylated sites on human chromosomes 21 and 22 in the brain tissue of eight adults. We extracted 61 U-CGIs from their unmethylated regions and applied our method (see <a href="#s2">results</a> in <a href="#pone.0001184.s012">Tables S9</a> and <a href="#pone.0001184.s013">S10</a> in Supplementary Materials). It can be seen that 8 of the 10 over-represented TFBSs in U-CGIs and 9 of the 13 over-represented TFBSs in boundaries obtained in Rollins <em>et al</em>'s data are also significantly enriched.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>From the results on these two independent datasets, we can see that most over-represented TFBSs identified in Rollins <em>et al</em>'s data are indeed enriched in all three independent data sets, suggesting that the existence of those putatively functional <em>cis</em>-acting elements in and around U-CGIs is robust and ubiquitous (largely independent of the examined tissue types).</p>


<h4>The influence of U-CGI localization preference</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>As is known, most of the U-CGIs are positioned at the 5′ end of human genes<a href="#pone.0001184-Bird1">[1]</a>, which is also true for our dataset: 297 of the 304 U-CGIs are located in the promoter regions, while only 79 of the 210 M-CGIs are in promoter regions. It is necessary to check whether these over-represented <em>cis</em>-elements may simply reflect the difference between promoter-CGI (CGI located in promoter) and non-promoter-CGI (CGI not located in promoter) (see <a href="#s4">Materials and Methods</a> for details). Results show that there is no clear boundary in the flanking region of the promoter related CGIs (<a href="#pone-0001184-g003">Figure 3</a>). There are still 3 over-represented TFBSs (V$HEN1_01, V$AP4 and V$HEB_Q6) in the promoter related CGIs in at least 5 of the experiments. However, when we changed the definition of over-represented TFBSs to be TFBSs that are significantly enriched in all the 6 U-CGI fragments, the first three TFBSs in <a href="#pone-0001184-t001">Table 1</a> are still over-represented in the U-CGIs, while there is no over-represented TFBS in the 10 groups of promoter related CGIs. In summary, it is unlikely that the observed TFBSs over-represented in and around U-CGIs are merely caused by the difference between the CGIs in promoter and non-promoter, although this possibility could not be completely ruled out. As a matter of fact, positioning in the promoter region is one important feature of U-CGIs themselves. It is possible that more U-CGIs could be promoter related as many more promoters are still being discovered each day.</p>
<div class="figure" id="pone-0001184-g003"><div class="img"><a name="pone-0001184-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The number of significantly over-represented TFBSs in each fragment of promoter related CGIs and their flanking sequences.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The <em>x</em>-axis indicates the location of fragment relative to the 5′ or 3′ end of CGIs. The <em>y</em>-axis is the average number of significant TFBSs in each fragment in the 10 experiments. One can see that there is no clear boundary in the flanking region of promoter related CGIs.</p>
<span>doi:10.1371/journal.pone.0001184.g003</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>For our promoter related U-CGIs, one could see that the number of over-represented TFBSs in the 5′ and 3′ end of U-CGIs is nearly symmetrical (<a href="#pone-0001184-g002">Figure 2</a>). This is reasonable because we found that 72.4% (220 out of 304) of U-CGIs are located in bidirectional promoters (promoters shared by pairs of genes that flank them on the opposite strands). To study whether the symmetry also exists for U-CGIs located in the unidirectional promoters, we extracted the 71 U-CGIs located in unidirectional promoters on the positive strand and observed the similar symmetric distribution of the over-represented TFBSs (See <a href="#pone.0001184.s003">Figure S3</a> in Supplementary Materials). Therefore, without loss of statistical power, we analyzed all the U-CGIs without subdividing the directions of their related promoters.</p>


<h4>Other chromatin marks mapped to the U-CGIs and their boundaries</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>A large part of the TFs corresponding to the over-represented TFBSs act as activators, which could recruit co-activator complexes (such as chromatin-remodeling complexes, histone-modification enzymes) to modify the chromatin structure<a href="#pone.0001184-Hahn1">[30]</a>, <a href="#pone.0001184-Li1">[31]</a>. It is interesting to study whether there are any correlated chromatin marks inside the U-CGIs and their boundaries. Currently there is only one high-resolution genome-scale profiling about histone modifications in human, which detected 20 histone methylations in T cells<a href="#pone.0001184-Barski1">[32]</a>. It has been reported that many tissues have similar DNA methylation landscapes, especially in CGIs<a href="#pone.0001184-Grunau1">[11]</a>, <a href="#pone.0001184-Song1">[12]</a>, <a href="#pone.0001184-Fang1">[23]</a>. Therefore, we investigated the intensities of the 20 histone modification marks in each fragment of the U- and M-CGIs and their flanking sequences. The average differences of the 20 histone marks between U-CGIs and M-CGIs are shown in <a href="#pone-0001184-g004">Figure 4</a>. One could see that H3K4me1, H3K4me2, H3K4me3 and H3K9me1, all of which were reported to be positively correlated with transcriptional levels<a href="#pone.0001184-Barski1">[32]</a>–<a href="#pone.0001184-Bernstein2">[34]</a>, are enriched in boundary regions (H3K4me3 is also enriched inside U-CGIs). A more recent ChIP-seq study has further confirmed that CTCF marks boundaries of histone methylation domains <a href="#pone.0001184-Barski1">[32]</a>. The results imply that the U-CGIs and their boundaries may physically correspond to active chromatin domains and their barriers, many of the enriched ZF TFs may help recruiting chromatin-remodeling factors during the establishment and/or maintenance of the U-CGIs that are refractory to DNA methylation.</p>
<div class="figure" id="pone-0001184-g004"><div class="img"><a name="pone-0001184-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001184" data-uri="info:doi/10.1371/journal.pone.0001184.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001184.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001184.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001184.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001184.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The average differences of histone modifications between U and M fragments.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The <em>x</em>-axis indicates the location of fragment relative to the 5′ or 3′ end of CGIs. The <em>y</em>-axis is the differential average intensity in each fragment. One can see that H3K4me1, H3K4me2 and H3K9me1are enriched in boundary regions, and H3K4me3 is enriched both in the U-CGIs and their boundaries.</p>
<span>doi:10.1371/journal.pone.0001184.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Of all the over-represented <em>cis</em>-acting elements in the boundaries of the U-CGIs, some of the corresponding TFs have been reported to prevent DNA methylation from spreading; while others still need to be further experimentally validated. In <a href="#pone.0001184-Brandeis1">[14]</a>, <a href="#pone.0001184-Mummaneni1">[35]</a>, Brandeis <em>et al</em> and Mummaneni <em>et al</em> reported that Sp1 elements act as a barrier to impede the spreading of DNA methylation, later Sp1 and CTCF elements were observed to be boundary elements to maintain the methylation free state of <em>BRCA1</em> promoter in normal breast tissue <a href="#pone.0001184-Butcher1">[17]</a>. CTCF is a transcriptional regulator with 11 zinc finger domains which exerts versatile function including repression, activation and chromatin insulation. Its methylation-dependent binding plays important roles in the regulation of imprinted genes (<em>H19/Igf2</em>) <a href="#pone.0001184-Hark1">[36]</a> and in X-chromosome inactivation <a href="#pone.0001184-Filippova1">[37]</a>. Consistent with these observations, Sp1 and CTCF elements are both over-represented in our boundary regions. However the previously reported (ATAAA)<sub>n</sub> boundary elements are not observed in our data, they are mostly found in repetitive heterochromatin regions that are under represented by the current human genome assembly. For the rest 11 TFBSs besides Sp1 and CTCF elements, there have been no reports about their potential roles in preventing methylation (notice that some of the TFBSs may be related, such as V$SP1, V$GC_01 and V$KROX_Q6 (reverse complement)). Furthermore, the Egr family corresponding to matrix V$KROX_Q6 has been recognized to be sensitive to methylation <a href="#pone.0001184-Ogishima1">[38]</a>, and Sp3 corresponding to V$SP3 belongs to the same family with Sp1. Thus, it is possible that such TFBSs may have similar boundary functions. What is most surprising is that, 70% of the <em>trans</em>-acting factors corresponding to over-represented TFBSs in the boundaries of U-CGIs possess zinc finger (C2H2-type) DNA binding domains comparing to only 17% in control set (<a href="#pone-0001184-t004">Table 4</a>), and 74% of these putative binding sites are found in the evolutionarily conserved regions. Therefore, we suggest that these zinc finger DNA binding proteins are likely to play essential roles in blocking the spreading of DNA methylation and protecting the U-CGIs from being methylated.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Among the over-represented TFBSs within the U-CGIs, some of them are also present in the boundary sequences, e.g. V$SP1_01. HEN1 which is related to V$HEN1_01 and the latter is known to methylate <em>micro</em>RNAs in plants <a href="#pone.0001184-Yang1">[39]</a>, and thus may in turn protect nearby sequences from methylation in plants since DNA methylation in plants could be directed by such <em>micro</em>RNAs <a href="#pone.0001184-Bird2">[40]</a>. But till now, there is neither a clear demonstration that RNA-directed <em>de novo</em> methylation exists in mammals <a href="#pone.0001184-Bird2">[40]</a>, nor is there an evidence that HEN1 could methylate <em>micro</em>RNAs in mammals. Although many <em>cis</em>-acting elements in U-CGIs had been reported to be methylation sensitive, the understanding of their specific function is still controversial. Some reported that Sp1 elements in U-CGIs mask CGIs from <em>de novo</em> methylation directly <a href="#pone.0001184-Brandeis1">[14]</a> which would enhance the blocking function; others reported that Sp1 elements in U-CGIs are required for transcription and may only assist the unmethylation status indirectly <a href="#pone.0001184-Mummaneni1">[35]</a>. We investigated the function annotations of TFs corresponding to the over-represented TFBSs in U-CGIs from Swiss-Prot. It turned out that 75% of them are annotated as activator, comparing to 50% in those over-represented in boundaries and 43% in the control TFs. Such TFs in U-CGIs should be closely related to activation of gene expression, which is consistent with earlier reports <a href="#pone.0001184-Jones1">[4]</a>, <a href="#pone.0001184-Robertson1">[41]</a>. Combining with the results that some over-represented TFBSs in U-CGIs are also present in boundary regions, we believe that these over-represented <em>cis</em>-acting elements in U-CGIs may have dual functions: they may help protecting U-CGIs against methylation, at the same time can also promote transcription <a href="#pone.0001184-Mummaneni1">[35]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>It has been reported that H3K4 methylation coincides with CGIs to remarkable extent <a href="#pone.0001184-Bernstein3">[42]</a>. The targeting of the modification enzymes to the specific sites is largely dependent on gene-specific transcription factors <a href="#pone.0001184-Lewin1">[43]</a>. The zinc finger proteins corresponding to the over-represented TFBSs might be co-regulators of the 4 enriched histone methylations. On the other hand, the histone modifications positively correlated with transcriptional levels have the potential to unfold chromatins, which can further strengthen the TFs' binding. The higher the probability of TFs' binding is, the stronger the methylation-protection function will be. DNA methylation and histone modifications are two important factors in the complex regulatory network modulating chromatin structure and genome function<a href="#pone.0001184-Bernstein4">[44]</a>. The correlation between the transcription factors, histone modification and CGI methylation observed in this study may reveal a clue on how the factors interact with each other in the complex network.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We also investigated whether there were some over-represented TFBSs in M-CGIs and their 400 bp flanking regions. Applying the same two-step hypothesis test on A2 vs. A1, F2 vs. F1, O2 vs. O1 and P2 vs. P1, we also got some over-represented TFBSs (see supplementary <a href="#pone.0001184.s014">Tables S11</a> and <a href="#pone.0001184.s015">S12</a>). But there is no enriched chromatin marks in or around M-CGIs from <a href="#pone-0001184-g004">Figure 4</a>. Whether it is possible that the lacking of these TFBSs in the U-CGIs and in the boundaries may also contribute to methylation resistance can be an interesting topic for further investigations.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In mammalian genomes, methylation pattern is established by several steps during development <a href="#pone.0001184-Reik2">[7]</a>. Firstly, most of methyl groups are erased after fertilization <a href="#pone.0001184-Monk1">[45]</a>. Gametes will then undergo <em>de novo</em> methylation at about the time of implantation <a href="#pone.0001184-Kafri1">[46]</a>, and herefrom methylation pattern will be preserved by maintenance methylation during lifetime <a href="#pone.0001184-Pfeifer1">[47]</a>. In the mouse, a hypothesis based on the <em>aprt</em> gene was proposed that methylation preferentially targets to specific sequences called methylation center in the <em>de novo</em> methylation stage <a href="#pone.0001184-Turker1">[15]</a>, such as B1 repetitive elements <a href="#pone.0001184-Turker2">[25]</a>; then methylation will spread to their surrounding sequences till some <em>cis</em>-acting sequences block its spreading <a href="#pone.0001184-Turker1">[15]</a>, and such barrier would explain the existence of the U-CGIs <a href="#pone.0001184-Bird2">[40]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In the genome-wide U-CGIs obtained from human brain DNA, sequences adjacent to the unmethylation domains are occupied by <em>Alu</em>S and <em>Alu</em>Y families which are highly methylated <a href="#pone.0001184-Rollins1">[24]</a>. In addition, we found that there are boundary regions of 400 bp both in the 5′ and 3′ end of U-CGIs over-represented with a group of zinc finger protein binding sites. Thus, we infer that the protection mechanism in the formation of U-CGIs based on mouse <em>aprt</em> gene is also applicable to human: <em>Alu</em> elements may function as the methylation centers <a href="#pone.0001184-Takai1">[48]</a>; when methylation moves forward their surrounding sequences, <em>cis</em>-acting elements bound with specific zinc finger proteins will block its spreading, and methylation pattern stabilizes when such counter-act reaches to a dynamical equilibrium. When some external condition changes, the dynamical equilibrium may be reestablished and the range of the methylation-resistant regions would move back and forth along the DNA sequences. For example, if some of the <em>cis</em>-acting elements are mutated (or methylation enzymes are mutated or their concentration levels are changed), the blocking capacity may be weakened or even dysfunctional, which would cause the U-CGIs to be gradually methylated. Similar observations have been reported in <a href="#pone.0001184-Brandeis1">[14]</a>, <a href="#pone.0001184-Mummaneni1">[35]</a>. Therefore, aberrant methylation of tumor-suppressor genes and progressive methylation in some tissue during aging may be explained by the weakening of the protection mechanism <a href="#pone.0001184-Issa1">[49]</a>. In our data, one U-CGI is in the promoter region of <em>MT1A</em> with <em>Alu</em> sequences in the 5′ flanking region. <em>MT1A</em> gene was reported to be methylation-free in normal tissue but hypermethylated to varying extent in some cancers <a href="#pone.0001184-Piotrowski1">[50]</a>, <a href="#pone.0001184-Yu1">[51]</a>. We speculate that a shifting of the dynamical equilibrium may contribute to the aberrant methylation.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Datasets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Our dataset is from Rollins <em>et al</em> <a href="#pone.0001184-Rollins1">[24]</a>, which detects the <em>in vivo</em> DNA methylation landscape of human brain. By digestion the sequences with McrBC (Rm<sup>5</sup>C-N<sub>40–500</sub>-Rm<sup>5</sup>C) and another five restriction endonucleases (REs): Tail (ACGT), BstUI (CGCG), HhaI (GCGC), HpaII (CCGG), and Acil (CCGC and GCGG), they identified 4240 methylation-resistant domains and 3518 methylation-prone domains respectively. According to the definition of CGI in UCSC Genome Browser: minimum length 200 bp, G+C content ≥50% and observed/expected CpG ratio ≥0.6 <a href="#pone.0001184-GardinerGarden1">[10]</a>, we extracted 304 U-CGIs from methylation-resistant domains and 210 M-CGIs in methylation-prone domains as our background sequences. The median lengths of U-CGIs and M-CGIs are 886 bp and 275 bp respectively. To identify over-represented TFBSs, we used 146 non-redundant TFBSs corresponding to vertebrate TFs of TRANSFAC9.4 <a href="#pone.0001184-Matys1">[27]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>For validation, we obtained another two independent datasets. One is from Yamada <em>et al</em> <a href="#pone.0001184-Yamada1">[28]</a>. They developed a simple HpaII-McrBC PCR method to discriminate the methylation status of CGIs on human chromosome 21q from peripheral blood leukocytes, and got 103 U-CGIs. As their definition of CGI (minimum length ≥400 bp, G+C content ≥50%, observed/expected CpG ratio ≥0.6) is different from ours, we applied our definition on their U-CGIs and obtained 104 U-CGIs. The other data is from Schumacher <em>et al</em> <a href="#pone.0001184-Schumacher1">[29]</a>. They applied tiling microarray technology to investigate the profiling of unmethylated sites on chromosome 21 and 22 in the brain tissue of eight adults respectively. Combining all the unmethylated regions of eight samples, we define such CGIs as U-CGIs if there are more than 100 bp overlap between the CGIs and these unmethylated regions. Finally, we extracted 61 U-CGIs from their data. The background sequences in the validation are also from M-CGIs as the methylation-prone data in the original datasets are too insufficient.</p>


<h4>Division strategy</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>In order to identify the sequence features in the U-CGIs, we divided the U-CGIs and the background CGIs into several 200 bp-long fragments, and all samples were aligned by the two ends of the CGIs (<a href="#pone-0001184-g001">Figure 1</a>). Such division strategy is based on the following reasons: firstly, the median length of the U-CGIs is much longer than median length of the M-CGIs, in order to avoid bias when comparing the foreground sequences (U) with the background sequences (M), we need to divide them into fragments with the same length; secondly, all of the CGIs are longer than 200 bp, setting the fragments to 200 bp would make full use of the samples; thirdly, we divided the U and the M-CGIs into 6 fragments and 2 fragments respectively because it would cover most of the sequences although some U-CGIs are longer than 1200 bp and some M-CGIs longer than 400 bp. Before this study, we knew that there should be some sequence features related to the formation of the U-CGIs in their flanking sequences <a href="#pone.0001184-Caiafa1">[52]</a>, but we were unclear about how far the informative flanking sequences would extend. Therefore, we just arbitrarily extended 1200 bp around the U-CGIs at first, and also divided them into several 200 bp-long fragments. As there is no over-represented TFBS between the 800–1200 bp flanking regions, we only show the fragments in 800 bp flanking regions (<a href="#pone-0001184-g001">Figure 1</a>). It should be noted that some U-CGIs are less than 1200 bp and some M-CGIs are less than 400 bp. If any defined CGI fragment extends out of CGI, such fragment would be eliminated; likewise, if any fragment of flanking sequences surpasses the given domain, such fragment would be eliminated as well.</p>


<h4>Two-step hypothesis test</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>After the division processes, we applied MOTIFCLASS <a href="#pone.0001184-Smith1">[26]</a> to get the putative TFBSs in every fragment of both types of the CGIs and their flanking sequences respectively, and counted the number of samples with given TFBS. Then Fisher's exact test <a href="#pone.0001184-Fisher1">[53]</a> was implemented in the U&amp;M-CGIs and their flanking fragments by comparing (A1, B1, C1) with A2, (F1, E1, D1) with F2, J1 with J2, I1 with I2, H1 with H2, G1 with G2, K1 with K2, L1 with L2, M1 with M2, and N1 with N2. A TFBS with <em>p</em>-value less than Bonferroni-adjusted cutoff (0.01) is regarded as a statistically significant one. In order to filter the influence of the biased G+C content between U and M, we randomly shuffled fragments of both the U-CGIs and their boundary sequences for 1000 times, and compared the shuffled U fragments with M fragments to get the proportion of samples significantly enriched with given TFBS in the shuffled U fragments. Similar to a permutation test, we regard the ratio that the proportion with TFBS in shuffled U fragments is at least as extreme as the proportion in U fragments in 1000 times as the <em>p</em>-value in the shuffle test. And only the TFBS with <em>p</em>-value less than 0.01 would be regarded to have no biased G+C content influence. We define the two step processes as a two-step hypothesis test. Only the TFBS that is significant in both hypothesis tests is reported as the significant one.</p>


<h4>Novel motif identification</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The processing described above only explores the known motifs (TFBSs). It is also meaningful to identify over-represented novel motifs in each of the fragment of the U-CGIs and the boundary sequences. Therefore, we applied DME-b <a href="#pone.0001184-Smith1">[26]</a> to hunt the novel motifs with width from 6 to 9 bp. For each motif width, we got the top 50 enriched motifs as candidate significant motifs. Considering that there would be some redundancies both between the novel motifs with different width themselves and between motifs and the over-represented TFBSs, we used the program Matcompare <a href="#pone.0001184-Schones1">[54]</a> to filter all the redundant novel motifs. The parameters of Matcompare are set as below: the greatest overhang when comparing motifs is 2, the Kullback-Leibler (K-L) divergence threshold is set to 1. If two motifs with different width are similar, the motif with larger width would be kept. In order to filter the influence of biased G+C content, we also implemented the two-step hypothesis test on the non-redundant motifs, and the <em>p</em>-value cutoff in the first step is also Bonferroni-adjusted (0.01).</p>


<h4>Promoter influence analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The promoter database is from CSHLmpd (<a href="http://rulai.cshl.edu/cshlmpd/">http://rulai.cshl.edu/cshlmpd/</a>). We defined the promoter to be the region between 1 kb upstream and 200 bp downstream of TSS (Transcription Starting Site). If a CGI overlaps with any promoter region, such CGI is regarded as promoter-CGI. In the process of identifying whether the over-represented TFBSs are only the difference between CGIs in promoter and non-promoter, the ideal method may be to implement the two-step hypothesis test on promoter-U-CGIs and promoter-M-CGIs. However, the strongly biased sample sizes between the promoter-U-CGIs and the promoter-M-CGIs as well as the small size of the promoter-M-CGIs compelled us to apply some other methods. We extracted all the CGIs in human genome and divided them into promoter-CGIs and non-promoter-CGIs. Then, by sampling data from the CGIs randomly, we constructed 10 groups of foreground sequences (promoter related CGIs) and background sequences (non-promoter related CGIs) with the same sample size and composition as U-CGIs and M-CGIs respectively, i.e. 304 foreground sequences (297 promoter-CGIs, 7 non-promoter-CGIs) and 210 background sequences (79 promoter-CGIs, 131 non-promoter-CGIs). The comparison process between foreground and background sequences is the same as U-CGIs vs. M-CGIs.</p>


<h4>Conservation analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>We investigated the conservation information of the over-represented putative TFBSs among human, mouse and rat. With the multiple alignment results of human, mouse and rat on UCSC Genome Browser, we used the 100 bp upstream and downstream sequences of all the putative binding sites as the input sequences of PAML <a href="#pone.0001184-Yang2">[55]</a> to get their phylogenetic tree. Based on the phylogenetic tree, we applied MONKEY <a href="#pone.0001184-Moses1">[56]</a> to get the significance that each putative binding sites are more conserved among the three species than their flanking sequences. With the <em>p</em>-value cutoff of 0.01, we calculated the proportion of the significantly conserved binding sites among the over-represented putative binding sites.</p>


<h4>Chromatin marks information</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Barski <em>et al</em><a href="#pone.0001184-Barski1">[32]</a> generated a large scale high-resolution profiling of 20 histone methylations in human resting CD4<sup>+</sup> T cells. We counted the number of each modification in a 200 bp window sliding with 10 bp offset in the U- and M-CGIs and their flanking sequences. The average intensity differences of the 20 chromatin marks in CGIs and 1000 bp flanking sequences between U and M are calculated at each location. The results are shown in <a href="#pone-0001184-g004">Figure 4</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001184"></div><a name="pone.0001184.s001" id="pone.0001184.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The number of over-represented TFBSs in each fragment of U-CGIs and their flanking sequences. The x-axis indicates the location of fragment relative to the 5′ or 3′ end of U-CGIs. Thus, fragments insides the two ‘0’s correspond to A1, B1, C1, D1, E1 and F1 of <a href="#pone-0001184-g001">Figure 1</a> in the article, and fragments that are in the 800 bp upstream and downstream of U-CGIs represent J1, I1, H1, G1 and K1, L1, M1, N1 of <a href="#pone-0001184-g001">Figure 1</a> in the article respectively. We applied the Bonferroni-adjusted p-value cutoff 0.05 here.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(6.81 MB TIF)</p>
<a name="pone.0001184.s002" id="pone.0001184.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">The number of over-represented TFBSs in each fragment of U-CGIs and their flanking sequences. The x-axis indicates the location of fragment relative to the 5′ or 3′ end of U-CGIs. Thus, fragments insides the two ‘0’s correspond to A1, B1, C1, D1, E1 and F1 of <a href="#pone-0001184-g001">Figure 1</a> in the article, and fragments that are in the 800 bp upstream and downstream of U-CGIs represent J1, I1, H1, G1 and K1, L1, M1, N1 of <a href="#pone-0001184-g001">Figure 1</a> in the article respectively. We applied the Bonferroni-adjusted p-value cutoff 0.001 here.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(6.89 MB TIF)</p>
<a name="pone.0001184.s003" id="pone.0001184.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The number of over-represented TFBSs in each fragment of U-CGIs located in promoters of unidirectional genes and their flanking sequences. The x-axis indicates the location of fragment relative to the 5′ or 3′ end of U-CGIs. Thus, fragments insides the two ‘0’s correspond to A1, B1, C1, D1, E1 and F1 of <a href="#pone-0001184-g001">Figure 1</a> in article, and fragments that are in the 800 bp upstream and downstream of U-CGIs represent J1, I1, H1, G1 and K1, L1, M1, N1 of <a href="#pone-0001184-g001">Figure 1</a> in article respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(6.84 MB TIF)</p>
<a name="pone.0001184.s004" id="pone.0001184.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The logos of over-represented TFBSs in U-CGIs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001184.s005" id="pone.0001184.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">The logos of over-represented TFBSs in boundaries of U-CGIs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0001184.s006" id="pone.0001184.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s006">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Motifs that are over-represented in U-CGIs. Also listed are the most similar TFBS to the motif, logo of the TFBS and their K-L divergence.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001184.s007" id="pone.0001184.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s007">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Motifs that are over-represented in the boundary regions. Also listed are the most similar TFBS to the motif, logo of the TFBS and their K-L divergence.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001184.s008" id="pone.0001184.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s008">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">The proportion of the significantly conserved binding sites in all the over-represented putative binding sites of every TF in each U-CGI fragment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001184.s009" id="pone.0001184.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s009">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">The proportion of the significantly conserved binding sites in all the over-represented putative binding sites of every TF in boundary sequences.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001184.s010" id="pone.0001184.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s010">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Validation results in the U-CGIs of Yamada et al's data. The check mark in the table indicates that the TFBS is significantly enriched in the specific U-CGI fragment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001184.s011" id="pone.0001184.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s011">Table S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Validation results in boundaries of U-CGIs of Yamada et al's data. The check mark in the table indicates that the TFBS is significantly enriched in the specific boundary region.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001184.s012" id="pone.0001184.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s012">Table S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Validation results in the U-CGIs of the Schumacher et al's data. The check mark in the table indicates that the TFBS is significantly enriched in the specific U-CGI fragment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001184.s013" id="pone.0001184.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s013">Table S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Validation results in the boundaries of the U-CGIs of the Schumacher et al's data. The check mark in the table indicates that the TFBS is significantly enriched in the specific boundary region.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001184.s014" id="pone.0001184.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s014">Table S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Over-represented TFBSs in M-CGI fragments. The second column indicates whether the TFs corresponding to the over-represented TFBSs are expressed in human brain. The logos of the TFBSs are also given. The two numbers in the 4th-5th columns are the p-values in the two-step hypothesis test in the corresponding fragment (we use p1 and p2 to represent the upper and lower value respectively). Only the TFBS with p1 less than Bonferroni-adjusted p-value cutoff 0.01 and p2 less than 0.01 is regarded as a significant TFBS in the fragment, and is marked in bold. Here we regard TFBSs that are both significant in A2 and F2 fragments as the over-represented TFBSs in M-CGIs. The redundant TFBSs are eliminated according to MatCompare.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.12 MB DOC)</p>
<a name="pone.0001184.s015" id="pone.0001184.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001184.s015">Table S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">Over-represented TFBSs in the 400 bp flanking regions of M-CGI. The second column indicates whether the TFs corresponding to the over-represented TFBSs are expressed in human brain. The logos of the TFBSs are also given. The two numbers in the 4th-5th columns are the p-values in the two-step hypothesis test in the corresponding fragment (we use p1 and p2 to represent the upper and lower value respectively). Only the TFBS with p1 less than Bonferroni-adjusted p-value cutoff 0.01 and p2 less than 0.01 is regarded as a significant TFBS in the fragment, and is marked in bold. Here we regard TFBSs that are both significant in O2 and P2 fragments as the over-represented TFBSs in flanking sequences of M-CGIs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001184.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Xiaowo Wang for his help on the conservation analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MZ XZ SF. Performed the experiments: SF. Analyzed the data: MZ SF FF. Contributed reagents/materials/analysis tools: MZ FF. Wrote the paper: MZ XZ SF.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001184-Bird1" id="pone.0001184-Bird1"></a>Bird AP (1987) CpG islands as gene markers in the vertebrate nucleus. Trends Genet  3: 342–347.  <ul class="find" data-citedArticleID="1018190" data-doi="10.1016/0168-9525(87)90294-0"><li><a href="http://dx.doi.org/10.1016/0168-9525(87)90294-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CpG+islands+as+gene+markers+in+the+vertebrate+nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CpG+islands+as+gene+markers+in+the+vertebrate+nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001184-Hansen1" id="pone.0001184-Hansen1"></a>Hansen RS, Stoger R, Wijmenga C, Stanek AM, Canfield TK, et al.  (2000) Escape from gene silencing in ICF syndrome: evidence for advanced replication time as a major determinant. Hum Mol Genet  9: 2575–2587.  <ul class="find" data-citedArticleID="1018218" data-doi="10.1093/hmg/9.18.2575"><li><a href="http://dx.doi.org/10.1093/hmg/9.18.2575" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escape+from+gene+silencing+in+ICF+syndrome%3A+evidence+for+advanced+replication+time+as+a+major+determinant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escape+from+gene+silencing+in+ICF+syndrome%3A+evidence+for+advanced+replication+time+as+a+major+determinant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001184-Heard1" id="pone.0001184-Heard1"></a>Heard E (2004) Recent advances in X-chromosome inactivation. Curr Opin Cell Biol  16: 247–255.  <ul class="find" data-citedArticleID="1018222" data-doi="10.1016/j.ceb.2004.03.005"><li><a href="http://dx.doi.org/10.1016/j.ceb.2004.03.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+advances+in+X-chromosome+inactivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+advances+in+X-chromosome+inactivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001184-Jones1" id="pone.0001184-Jones1"></a>Jones PA, Baylin SB (2002) The fundamental role of epigenetic events in cancer. Nat Rev Genet  3: 415–428.  <ul class="find" data-citedArticleID="1018228"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+fundamental+role+of+epigenetic+events+in+cancer.&amp;auth=&amp;atitle=The+fundamental+role+of+epigenetic+events+in+cancer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fundamental+role+of+epigenetic+events+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fundamental+role+of+epigenetic+events+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001184-Reik1" id="pone.0001184-Reik1"></a>Reik W, Santos F, Dean W (2003) Mammalian epigenomics: reprogramming the genome for development and therapy. Theriogenology  59: 21–32.  <ul class="find" data-citedArticleID="1018256" data-doi="10.1016/s0093-691x(02)01269-4"><li><a href="http://dx.doi.org/10.1016/s0093-691x(02)01269-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+epigenomics%3A+reprogramming+the+genome+for+development+and+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+epigenomics%3A+reprogramming+the+genome+for+development+and+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001184-Siegfried1" id="pone.0001184-Siegfried1"></a>Siegfried Z, Eden S, Mendelsohn M, Feng X, Tsuberi BZ, et al.  (1999) DNA methylation represses transcription in vivo. Nat Genet  22: 203–206.  <ul class="find" data-citedArticleID="1018268" data-doi="10.1038/9727"><li><a href="http://dx.doi.org/10.1038/9727" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+represses+transcription+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+represses+transcription+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001184-Reik2" id="pone.0001184-Reik2"></a>Reik W, Dean W, Walter J (2001) Epigenetic reprogramming in mammalian development. Science  293: 1089–1093.  <ul class="find" data-citedArticleID="1018258" data-doi="10.1126/science.1063443"><li><a href="http://dx.doi.org/10.1126/science.1063443" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+reprogramming+in+mammalian+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+reprogramming+in+mammalian+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001184-Suter1" id="pone.0001184-Suter1"></a>Suter CM, Martin DI, Ward RL (2004) Germline epimutation of MLH1 in individuals with multiple cancers. Nat Genet  36: 497–501.  <ul class="find" data-citedArticleID="1018274" data-doi="10.1038/ng1342"><li><a href="http://dx.doi.org/10.1038/ng1342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Germline+epimutation+of+MLH1+in+individuals+with+multiple+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Germline+epimutation+of+MLH1+in+individuals+with+multiple+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001184-Melanie1" id="pone.0001184-Melanie1"></a>Melanie E, Miguel AG-S, Lan-Hsiang H, Rose Marie M, Kenneth CK, et al.  (1982) Amount and distribution of 5-methylcytosine in human DNA from different types of tissues or cells. Nucleic Acids Res  10: 2709–2721.  <ul class="find" data-citedArticleID="1018240" data-doi="10.1093/nar/10.8.2709"><li><a href="http://dx.doi.org/10.1093/nar/10.8.2709" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amount+and+distribution+of+5-methylcytosine+in+human+DNA+from+different+types+of+tissues+or+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amount+and+distribution+of+5-methylcytosine+in+human+DNA+from+different+types+of+tissues+or+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001184-GardinerGarden1" id="pone.0001184-GardinerGarden1"></a>Gardiner-Garden M, Frommer M (1987) CpG islands in vertebrate genomes. J Mol Biol  196: 261–282.  <ul class="find" data-citedArticleID="1018212" data-doi="10.1016/0022-2836(87)90689-9"><li><a href="http://dx.doi.org/10.1016/0022-2836(87)90689-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CpG+islands+in+vertebrate+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CpG+islands+in+vertebrate+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001184-Grunau1" id="pone.0001184-Grunau1"></a>Grunau C, Hindermann W, Rosenthal A (2000) Large-scale methylation analysis of human genomic DNA reveals tissue-specific differences between the methylation profiles of genes and pseudogenes. Hum Mol Genet  9: 2651–2663.  <ul class="find" data-citedArticleID="1018214" data-doi="10.1093/hmg/9.18.2651"><li><a href="http://dx.doi.org/10.1093/hmg/9.18.2651" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+methylation+analysis+of+human+genomic+DNA+reveals+tissue-specific+differences+between+the+methylation+profiles+of+genes+and+pseudogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+methylation+analysis+of+human+genomic+DNA+reveals+tissue-specific+differences+between+the+methylation+profiles+of+genes+and+pseudogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001184-Song1" id="pone.0001184-Song1"></a>Song F, Smith JF, Kimura MT, Morrow AD, Matsuyama T, et al.  (2005) Association of tissue-specific differentially methylated regions (TDMs) with differential gene expression. Proc Natl Acad Sci U S A  102: 3336–3341.  <ul class="find" data-citedArticleID="1018272" data-doi="10.1073/pnas.0408436102"><li><a href="http://dx.doi.org/10.1073/pnas.0408436102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+tissue-specific+differentially+methylated+regions+%28TDMs%29+with+differential+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+tissue-specific+differentially+methylated+regions+%28TDMs%29+with+differential+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001184-Ioshikhes1" id="pone.0001184-Ioshikhes1"></a>Ioshikhes IP, Zhang MQ (2000) Large-scale human promoter mapping using CpG islands. Nat Genet  26: 61–63.  <ul class="find" data-citedArticleID="1018224"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Large-scale+human+promoter+mapping+using+CpG+islands.&amp;auth=&amp;atitle=Large-scale+human+promoter+mapping+using+CpG+islands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+human+promoter+mapping+using+CpG+islands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+human+promoter+mapping+using+CpG+islands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001184-Brandeis1" id="pone.0001184-Brandeis1"></a>Brandeis M, Frank D, Keshet I, Siegfried Z, Mendelsohn M, et al.  (1994) Sp1 elements protect a CpG island from de novo methylation. Nature  371: 435–438.  <ul class="find" data-citedArticleID="1018196" data-doi="10.1038/371435a0"><li><a href="http://dx.doi.org/10.1038/371435a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sp1+elements+protect+a+CpG+island+from+de+novo+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sp1+elements+protect+a+CpG+island+from+de+novo+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001184-Turker1" id="pone.0001184-Turker1"></a>Turker MS (1999) The establishment and maintenance of DNA methylation patterns in mouse somatic cells. Semin Cancer Biol  9: 329–337.  <ul class="find" data-citedArticleID="1018278" data-doi="10.1006/scbi.1999.0133"><li><a href="http://dx.doi.org/10.1006/scbi.1999.0133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+establishment+and+maintenance+of+DNA+methylation+patterns+in+mouse+somatic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+establishment+and+maintenance+of+DNA+methylation+patterns+in+mouse+somatic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001184-Millar1" id="pone.0001184-Millar1"></a>Millar DS, Paul CL, Molloy PL, Clark SJ (2000) A distinct sequence (ATAAA)n separates methylated and unmethylated domains at the 5′-end of the GSTP1 CpG island. J Biol Chem  275: 24893–24899.  <ul class="find" data-citedArticleID="1018242" data-doi="10.1074/jbc.m906538199"><li><a href="http://dx.doi.org/10.1074/jbc.m906538199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+distinct+sequence+%28ATAAA%29n+separates+methylated+and+unmethylated+domains+at+the+5%E2%80%B2-end+of+the+GSTP1+CpG+island." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+distinct+sequence+%28ATAAA%29n+separates+methylated+and+unmethylated+domains+at+the+5%E2%80%B2-end+of+the+GSTP1+CpG+island.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001184-Butcher1" id="pone.0001184-Butcher1"></a>Butcher DT, Mancini-DiNardo DN, Archer TK, Rodenhiser DI (2004) DNA binding sites for putative methylation boundaries in the unmethylated region of the BRCA1 promoter. Int J Cancer  111: 669–678.  <ul class="find" data-citedArticleID="1018198" data-doi="10.1002/ijc.20324"><li><a href="http://dx.doi.org/10.1002/ijc.20324" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+binding+sites+for+putative+methylation+boundaries+in+the+unmethylated+region+of+the+BRCA1+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+binding+sites+for+putative+methylation+boundaries+in+the+unmethylated+region+of+the+BRCA1+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001184-Marin1" id="pone.0001184-Marin1"></a>Marin M, Karis A, Visser P, Grosveld F, Philipsen S (1997) Transcription factor Sp1 is essential for early embryonic development but dispensable for cell growth and differentiation. Cell  89: 619–628.  <ul class="find" data-citedArticleID="1018236" data-doi="10.1016/s0092-8674(00)80243-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80243-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+factor+Sp1+is+essential+for+early+embryonic+development+but+dispensable+for+cell+growth+and+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+factor+Sp1+is+essential+for+early+embryonic+development+but+dispensable+for+cell+growth+and+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001184-Bhasin1" id="pone.0001184-Bhasin1"></a>Bhasin M, Zhang H, Reinherz EL, Reche PA (2005) Prediction of methylated CpGs in DNA sequences using a support vector machine. FEBS Lett  579: 4302–4308.  <ul class="find" data-citedArticleID="1018188" data-doi="10.1016/j.febslet.2005.07.002"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.07.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+methylated+CpGs+in+DNA+sequences+using+a+support+vector+machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+methylated+CpGs+in+DNA+sequences+using+a+support+vector+machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001184-Bock1" id="pone.0001184-Bock1"></a>Bock C, Paulsen M, Tierling S, Mikeska T, Lengauer T, et al.  (2006) CpG island methylation in human lymphocytes is highly correlated with DNA sequence, repeats, and predicted DNA structure. PLoS Genet  2: e26.  <ul class="find" data-citedArticleID="1018194" data-doi="10.1371/journal.pgen.0020026"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CpG+island+methylation+in+human+lymphocytes+is+highly+correlated+with+DNA+sequence%2C+repeats%2C+and+predicted+DNA+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CpG+island+methylation+in+human+lymphocytes+is+highly+correlated+with+DNA+sequence%2C+repeats%2C+and+predicted+DNA+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001184-Feltus1" id="pone.0001184-Feltus1"></a>Feltus FA, Lee EK, Costello JF, Plass C, Vertino PM (2003) Predicting aberrant CpG island methylation. Proc Natl Acad Sci U S A  100: 12253–12258.  <ul class="find" data-citedArticleID="1018206" data-doi="10.1073/pnas.2037852100"><li><a href="http://dx.doi.org/10.1073/pnas.2037852100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+aberrant+CpG+island+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+aberrant+CpG+island+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001184-Das1" id="pone.0001184-Das1"></a>Das R, Dimitrova N, Xuan Z, Rollins RA, Haghighi F, et al.  (2006) Computational prediction of methylation status in human genomic sequences. Proc Natl Acad Sci U S A  103: 10713–10716.  <ul class="find" data-citedArticleID="1018202" data-doi="10.1073/pnas.0602949103"><li><a href="http://dx.doi.org/10.1073/pnas.0602949103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+prediction+of+methylation+status+in+human+genomic+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+prediction+of+methylation+status+in+human+genomic+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001184-Fang1" id="pone.0001184-Fang1"></a>Fang F, Fan S, Zhang X, Zhang MQ (2006) Predicting methylation status of CpG islands in the human brain. Bioinformatics  22: 2204–2209.  <ul class="find" data-citedArticleID="1018204" data-doi="10.1093/bioinformatics/btl377"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl377" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+methylation+status+of+CpG+islands+in+the+human+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+methylation+status+of+CpG+islands+in+the+human+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001184-Rollins1" id="pone.0001184-Rollins1"></a>Rollins RA, Haghighi F, Edwards JR, Das R, Zhang MQ, et al.  (2006) Large-scale structure of genomic methylation patterns. Genome Res  16: 157–163.  <ul class="find" data-citedArticleID="1018262" data-doi="10.1101/gr.4362006"><li><a href="http://dx.doi.org/10.1101/gr.4362006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+structure+of+genomic+methylation+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+structure+of+genomic+methylation+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001184-Turker2" id="pone.0001184-Turker2"></a>Turker MS (2002) Gene silencing in mammalian cells and the spread of DNA methylation. Oncogene  21: 5388–5393.  <ul class="find" data-citedArticleID="1018280" data-doi="10.1038/sj.onc.1205599"><li><a href="http://dx.doi.org/10.1038/sj.onc.1205599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+silencing+in+mammalian+cells+and+the+spread+of+DNA+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+silencing+in+mammalian+cells+and+the+spread+of+DNA+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001184-Smith1" id="pone.0001184-Smith1"></a>Smith AD, Sumazin P, Das D, Zhang MQ (2005) Mining ChIP-chip data for transcription factor and cofactor binding sites. Bioinformatics  21: i403–i412.  <ul class="find" data-citedArticleID="1018270" data-doi="10.1093/bioinformatics/bti1043"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti1043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mining+ChIP-chip+data+for+transcription+factor+and+cofactor+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mining+ChIP-chip+data+for+transcription+factor+and+cofactor+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001184-Matys1" id="pone.0001184-Matys1"></a>Matys V, Kel-Margoulis OV, Fricke E, Liebich I, Land S, et al.  (2006) TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes. Nucleic Acids Res  34: D108–D110.  <ul class="find" data-citedArticleID="1018238" data-doi="10.1093/nar/gkj143"><li><a href="http://dx.doi.org/10.1093/nar/gkj143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRANSFAC+and+its+module+TRANSCompel%3A+transcriptional+gene+regulation+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRANSFAC+and+its+module+TRANSCompel%3A+transcriptional+gene+regulation+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001184-Yamada1" id="pone.0001184-Yamada1"></a>Yamada Y, Watanabe H, Miura F, Soejima H, Uchiyama M, et al.  (2004) A comprehensive analysis of allelic methylation status of CpG islands on human chromosome 21q. Genome Res  14: 247–266.  <ul class="find" data-citedArticleID="1018282" data-doi="10.1101/gr.1351604"><li><a href="http://dx.doi.org/10.1101/gr.1351604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+analysis+of+allelic+methylation+status+of+CpG+islands+on+human+chromosome+21q." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+analysis+of+allelic+methylation+status+of+CpG+islands+on+human+chromosome+21q.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001184-Schumacher1" id="pone.0001184-Schumacher1"></a>Schumacher A, Kapranov P, Kaminsky Z, Flanagan J, Assadzadeh A, et al.  (2006) Microarray-based DNA methylation profiling: technology and applications. Nucleic Acids Res  34: 528–542.  <ul class="find" data-citedArticleID="1018266" data-doi="10.1093/nar/gkj461"><li><a href="http://dx.doi.org/10.1093/nar/gkj461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray-based+DNA+methylation+profiling%3A+technology+and+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray-based+DNA+methylation+profiling%3A+technology+and+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001184-Hahn1" id="pone.0001184-Hahn1"></a>Hahn S (2004) Structure and mechanism of the RNA polymerase II transcription machinery. Nat Struct Mol Biol  11: 394–403.  <ul class="find" data-citedArticleID="1018216" data-doi="10.1038/nsmb763"><li><a href="http://dx.doi.org/10.1038/nsmb763" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+mechanism+of+the+RNA+polymerase+II+transcription+machinery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+mechanism+of+the+RNA+polymerase+II+transcription+machinery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001184-Li1" id="pone.0001184-Li1"></a>Li B, Carey M, Workman JL (2007) The role of chromatin during transcription. Cell  128: 707–719.  <ul class="find" data-citedArticleID="1018234" data-doi="10.1016/j.cell.2007.01.015"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.01.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+chromatin+during+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+chromatin+during+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001184-Barski1" id="pone.0001184-Barski1"></a>Barski A, Cuddapah S, Cui K, Roh TY, Schones DE, et al.  (2007) High-resolution profiling of histone methylations in the human genome. Cell  129: 823–837.  <ul class="find" data-citedArticleID="1018178" data-doi="10.1016/j.cell.2007.05.009"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.05.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+profiling+of+histone+methylations+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+profiling+of+histone+methylations+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001184-Bernstein1" id="pone.0001184-Bernstein1"></a>Bernstein BE, Humphrey EL, Erlich RL, Schneider R, Bouman P, et al.  (2002) Methylation of histone H3 Lys 4 in coding regions of active genes. Proc Natl Acad Sci U S A  99: 8695–8700.  <ul class="find" data-citedArticleID="1018180" data-doi="10.1073/pnas.082249499"><li><a href="http://dx.doi.org/10.1073/pnas.082249499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation+of+histone+H3+Lys+4+in+coding+regions+of+active+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation+of+histone+H3+Lys+4+in+coding+regions+of+active+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001184-Bernstein2" id="pone.0001184-Bernstein2"></a>Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, et al.  (2005) Genomic maps and comparative analysis of histone modifications in human and mouse. Cell  120: 169–181.  <ul class="find" data-citedArticleID="1018182" data-doi="10.1016/j.cell.2005.01.001"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001184-Mummaneni1" id="pone.0001184-Mummaneni1"></a>Mummaneni P, Yates P, Simpson J, Rose J, Turker MS (1998) The primary function of a redundant Sp1 binding site in the mouse aprt gene promoter is to block epigenetic gene inactivation. Nucleic Acids Res  26: 5163–5169.  <ul class="find" data-citedArticleID="1018248" data-doi="10.1093/nar/26.22.5163"><li><a href="http://dx.doi.org/10.1093/nar/26.22.5163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+primary+function+of+a+redundant+Sp1+binding+site+in+the+mouse+aprt+gene+promoter+is+to+block+epigenetic+gene+inactivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+primary+function+of+a+redundant+Sp1+binding+site+in+the+mouse+aprt+gene+promoter+is+to+block+epigenetic+gene+inactivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001184-Hark1" id="pone.0001184-Hark1"></a>Hark AT, Schoenherr CJ, Katz DJ, Ingram RS, Levorse JM, et al.  (2000) CTCF mediates methylation-sensitive enhancer-blocking activity at the H19/Igf2 locus. Nature  405: 408–409.  <ul class="find" data-citedArticleID="1018220"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CTCF+mediates+methylation-sensitive+enhancer-blocking+activity+at+the+H19%2FIgf2+locus.&amp;auth=&amp;atitle=CTCF+mediates+methylation-sensitive+enhancer-blocking+activity+at+the+H19%2FIgf2+locus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CTCF+mediates+methylation-sensitive+enhancer-blocking+activity+at+the+H19%2FIgf2+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CTCF+mediates+methylation-sensitive+enhancer-blocking+activity+at+the+H19%2FIgf2+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001184-Filippova1" id="pone.0001184-Filippova1"></a>Filippova GN, Cheng MK, Moore JM, Truong JP, Hu YJ, et al.  (2005) Boundaries between chromosomal domains of X inactivation and escape bind CTCF and lack CpG methylation during early development. Dev Cell  8: 31–42.  <ul class="find" data-citedArticleID="1018208" data-doi="10.1016/j.devcel.2004.10.018"><li><a href="http://dx.doi.org/10.1016/j.devcel.2004.10.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Boundaries+between+chromosomal+domains+of+X+inactivation+and+escape+bind+CTCF+and+lack+CpG+methylation+during+early+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Boundaries+between+chromosomal+domains+of+X+inactivation+and+escape+bind+CTCF+and+lack+CpG+methylation+during+early+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001184-Ogishima1" id="pone.0001184-Ogishima1"></a>Ogishima T, Shiina H, Breault JE, Terashima M, Honda S, et al.  (2005) Promoter CpG hypomethylation and transcription factor EGR1 hyperactivate heparanase expression in bladder cancer. Oncogene  24: 6765–6772.  <ul class="find" data-citedArticleID="1018250" data-doi="10.1038/sj.onc.1208811"><li><a href="http://dx.doi.org/10.1038/sj.onc.1208811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Promoter+CpG+hypomethylation+and+transcription+factor+EGR1+hyperactivate+heparanase+expression+in+bladder+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Promoter+CpG+hypomethylation+and+transcription+factor+EGR1+hyperactivate+heparanase+expression+in+bladder+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001184-Yang1" id="pone.0001184-Yang1"></a>Yang Z, Ebright YW, Yu B, Chen X (2006) HEN1 recognizes 21-24 nt small RNA duplexes and deposits a methyl group onto the 2′ OH of the 3′ terminal nucleotide. Nucleic Acids Res  34: 667–675.  <ul class="find" data-citedArticleID="1018284" data-doi="10.1093/nar/gkj474"><li><a href="http://dx.doi.org/10.1093/nar/gkj474" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HEN1+recognizes+21-24+nt+small+RNA+duplexes+and+deposits+a+methyl+group+onto+the+2%E2%80%B2+OH+of+the+3%E2%80%B2+terminal+nucleotide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HEN1+recognizes+21-24+nt+small+RNA+duplexes+and+deposits+a+methyl+group+onto+the+2%E2%80%B2+OH+of+the+3%E2%80%B2+terminal+nucleotide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001184-Bird2" id="pone.0001184-Bird2"></a>Bird AP (2002) DNA methylation patterns and epigenetic memory. Genes Dev  16: 6–21.  <ul class="find" data-citedArticleID="1018192" data-doi="10.1101/gad.947102"><li><a href="http://dx.doi.org/10.1101/gad.947102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+patterns+and+epigenetic+memory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+patterns+and+epigenetic+memory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001184-Robertson1" id="pone.0001184-Robertson1"></a>Robertson KD (2002) DNA methylation and chromatin - unraveling the tangled web. Oncogene  21: 5361–5379.  <ul class="find" data-citedArticleID="1018260" data-doi="10.1038/sj.onc.1205609"><li><a href="http://dx.doi.org/10.1038/sj.onc.1205609" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+and+chromatin+-+unraveling+the+tangled+web." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+and+chromatin+-+unraveling+the+tangled+web.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001184-Bernstein3" id="pone.0001184-Bernstein3"></a>Bernstein BE, Mikkelsen TS, Xie X, Kamal M, Huebert DJ, et al.  (2006) A bivalent chromatin structure marks key developmental genes in embryonic stem cells. Cell  125: 315–326.  <ul class="find" data-citedArticleID="1018184" data-doi="10.1016/j.cell.2006.02.041"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.02.041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+bivalent+chromatin+structure+marks+key+developmental+genes+in+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+bivalent+chromatin+structure+marks+key+developmental+genes+in+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001184-Lewin1" id="pone.0001184-Lewin1"></a>Lewin B (2004) Genes VIII: Person Prentice Hall.  <ul class="find" data-citedArticleID="1018232"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genes+VIII%3A+Person+Prentice+Hall.&amp;auth=&amp;atitle=Genes+VIII%3A+Person+Prentice+Hall." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+VIII%3A+Person+Prentice+Hall." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+VIII%3A+Person+Prentice+Hall.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001184-Bernstein4" id="pone.0001184-Bernstein4"></a>Bernstein BE, Meissner A, Lander ES (2007) The mammalian epigenome. Cell  128: 669–681.  <ul class="find" data-citedArticleID="1018186" data-doi="10.1016/j.cell.2007.01.033"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.01.033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mammalian+epigenome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mammalian+epigenome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001184-Monk1" id="pone.0001184-Monk1"></a>Monk M, Boubelik M, Lehnert S (1987) Temporal and regional changes in DNA methylation in the embryonic, extraembryonic and germ cell lineages during mouse embryo development. Development  99: 371–382.  <ul class="find" data-citedArticleID="1018244"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Temporal+and+regional+changes+in+DNA+methylation+in+the+embryonic%2C+extraembryonic+and+germ+cell+lineages+during+mouse+embryo+development.&amp;auth=&amp;atitle=Temporal+and+regional+changes+in+DNA+methylation+in+the+embryonic%2C+extraembryonic+and+germ+cell+lineages+during+mouse+embryo+development." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal+and+regional+changes+in+DNA+methylation+in+the+embryonic%2C+extraembryonic+and+germ+cell+lineages+during+mouse+embryo+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal+and+regional+changes+in+DNA+methylation+in+the+embryonic%2C+extraembryonic+and+germ+cell+lineages+during+mouse+embryo+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001184-Kafri1" id="pone.0001184-Kafri1"></a>Kafri T, Ariel M, Brandeis M, Shemer R, Urven L, et al.  (1992) Developmental pattern of gene-specific DNA methylation in the mouse embryo and germ line. Genes Dev  6: 705–714.  <ul class="find" data-citedArticleID="1018230" data-doi="10.1101/gad.6.5.705"><li><a href="http://dx.doi.org/10.1101/gad.6.5.705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+pattern+of+gene-specific+DNA+methylation+in+the+mouse+embryo+and+germ+line." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+pattern+of+gene-specific+DNA+methylation+in+the+mouse+embryo+and+germ+line.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001184-Pfeifer1" id="pone.0001184-Pfeifer1"></a>Pfeifer GP, Steigerwald SD, Hansen RS, Gartler SM, Riggs AD (1990) Polymerase chain reaction-aided genomic sequencing of an X chromosome-linked CpG island: Methylation patterns suggest clonal inheritance, CpG site autonomy, and an explanation of activity state stability. Proc Natl Acad Sci U S A  87: 8252–8256.  <ul class="find" data-citedArticleID="1018252" data-doi="10.1073/pnas.87.21.8252"><li><a href="http://dx.doi.org/10.1073/pnas.87.21.8252" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymerase+chain+reaction-aided+genomic+sequencing+of+an+X+chromosome-linked+CpG+island%3A+Methylation+patterns+suggest+clonal+inheritance%2C+CpG+site+autonomy%2C+and+an+explanation+of+activity+state+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymerase+chain+reaction-aided+genomic+sequencing+of+an+X+chromosome-linked+CpG+island%3A+Methylation+patterns+suggest+clonal+inheritance%2C+CpG+site+autonomy%2C+and+an+explanation+of+activity+state+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001184-Takai1" id="pone.0001184-Takai1"></a>Takai D, Jones PA (2002) Comprehensive analysis of CpG islands in human chromosomes 21 and 22. Proc Natl Acad Sci U S A  99: 3740–3745.  <ul class="find" data-citedArticleID="1018276" data-doi="10.1073/pnas.052410099"><li><a href="http://dx.doi.org/10.1073/pnas.052410099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+analysis+of+CpG+islands+in+human+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+analysis+of+CpG+islands+in+human+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001184-Issa1" id="pone.0001184-Issa1"></a>Issa JP (1999) Aging, DNA methylation and cancer. Crit Rev Oncol Hematol  32: 31–43.  <ul class="find" data-citedArticleID="1018226" data-doi="10.1016/s1040-8428(99)00019-0"><li><a href="http://dx.doi.org/10.1016/s1040-8428(99)00019-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aging%2C+DNA+methylation+and+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aging%2C+DNA+methylation+and+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001184-Piotrowski1" id="pone.0001184-Piotrowski1"></a>Piotrowski A, Benetkiewicz M, Menzel U, de Stahl TD, Mantripragada K, et al.  (2006) Microarray-based survey of CpG islands identifies concurrent hyper- and hypomethylation patterns in tissues derived from patients with breast cancer. Genes Chromosomes Cancer  45: 656–667.  <ul class="find" data-citedArticleID="1018254" data-doi="10.1002/gcc.20331"><li><a href="http://dx.doi.org/10.1002/gcc.20331" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray-based+survey+of+CpG+islands+identifies+concurrent+hyper-+and+hypomethylation+patterns+in+tissues+derived+from+patients+with+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray-based+survey+of+CpG+islands+identifies+concurrent+hyper-+and+hypomethylation+patterns+in+tissues+derived+from+patients+with+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001184-Yu1" id="pone.0001184-Yu1"></a>Yu J, Zhang HY, Ma ZZ, Lu W, Wang YF, et al.  (2003) Methylation profiling of twenty four genes and the concordant methylation behaviours of nineteen genes that may contribute to hepatocellular carcinogenesis. Cell Res  13: 319–333.  <ul class="find" data-citedArticleID="1018288" data-doi="10.1038/sj.cr.7290177"><li><a href="http://dx.doi.org/10.1038/sj.cr.7290177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation+profiling+of+twenty+four+genes+and+the+concordant+methylation+behaviours+of+nineteen+genes+that+may+contribute+to+hepatocellular+carcinogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation+profiling+of+twenty+four+genes+and+the+concordant+methylation+behaviours+of+nineteen+genes+that+may+contribute+to+hepatocellular+carcinogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001184-Caiafa1" id="pone.0001184-Caiafa1"></a>Caiafa P, Zampieri M (2005) DNA methylation and chromatin structure: the puzzling CpG islands. J Cell Biochem  94: 257–265.  <ul class="find" data-citedArticleID="1018200" data-doi="10.1002/jcb.20325"><li><a href="http://dx.doi.org/10.1002/jcb.20325" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+and+chromatin+structure%3A+the+puzzling+CpG+islands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+and+chromatin+structure%3A+the+puzzling+CpG+islands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001184-Fisher1" id="pone.0001184-Fisher1"></a>Fisher RA (1935) The logic of inductive inference. Journal of the Royal Statistical Society  98: 39–82.  <ul class="find" data-citedArticleID="1018210" data-doi="10.2307/2342435"><li><a href="http://dx.doi.org/10.2307/2342435" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+logic+of+inductive+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+logic+of+inductive+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001184-Schones1" id="pone.0001184-Schones1"></a>Schones DE, Sumazin P, Zhang MQ (2005) Similarity of position frequency matrices for transcription factor binding sites. Bioinformatics  21: 307–313.  <ul class="find" data-citedArticleID="1018264" data-doi="10.1093/bioinformatics/bth480"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Similarity+of+position+frequency+matrices+for+transcription+factor+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Similarity+of+position+frequency+matrices+for+transcription+factor+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001184-Yang2" id="pone.0001184-Yang2"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="1018286" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001184-Moses1" id="pone.0001184-Moses1"></a>Moses AM, Chiang DY, Pollard DA, Iyer VN, Eisen MB (2004) MONKEY: identifying conserved transcription-factor binding sites in multiple alignments using a binding site-specific evolutionary model. Genome Biol  5: R98.  <ul class="find" data-citedArticleID="1018246"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MONKEY%3A+identifying+conserved+transcription-factor+binding+sites+in+multiple+alignments+using+a+binding+site-specific+evolutionary+model.&amp;auth=&amp;atitle=MONKEY%3A+identifying+conserved+transcription-factor+binding+sites+in+multiple+alignments+using+a+binding+site-specific+evolutionary+model." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MONKEY%3A+identifying+conserved+transcription-factor+binding+sites+in+multiple+alignments+using+a+binding+site-specific+evolutionary+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MONKEY%3A+identifying+conserved+transcription-factor+binding+sites+in+multiple+alignments+using+a+binding+site-specific+evolutionary+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.XML" value="134071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.PDF" value="324559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g001.PNG_L" value="80084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g001.PNG_M" value="24529"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g001.PNG_S" value="4810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g001.TIF" value="122998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g001.PNG_I" value="8932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g002.PNG_L" value="182683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g002.PNG_M" value="26340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g002.PNG_S" value="8422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g002.TIF" value="526442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g002.PNG_I" value="8307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t001.PNG_L" value="314696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t001.PNG_M" value="193053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t001.PNG_S" value="17607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t001.TIF" value="1414010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t001.PNG_I" value="59412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t002.PNG_L" value="38716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t002.PNG_M" value="51176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t002.PNG_S" value="10061"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t002.TIF" value="168566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t002.PNG_I" value="17712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t003.PNG_L" value="208874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t003.PNG_M" value="120032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t003.PNG_S" value="20549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t003.TIF" value="1060250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t003.PNG_I" value="141385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t004.PNG_L" value="41017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t004.PNG_M" value="52992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t004.PNG_S" value="10186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t004.TIF" value="172340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.t004.PNG_I" value="18362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g003.PNG_L" value="210708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g003.PNG_M" value="28512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g003.PNG_S" value="8453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g003.TIF" value="584626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g003.PNG_I" value="8873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g004.PNG_L" value="1313111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g004.PNG_M" value="182255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g004.PNG_S" value="15360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g004.TIF" value="2009292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.g004.PNG_I" value="62005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s001.TIF" value="1497628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s002.TIF" value="1528470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s003.TIF" value="1534226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s004.DOC" value="76288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s005.DOC" value="98304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s006.DOC" value="44032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s007.DOC" value="42496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s008.DOC" value="74752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s009.DOC" value="81408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s010.DOC" value="44544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s011.DOC" value="36864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s012.DOC" value="44544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s013.DOC" value="35840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s014.DOC" value="116736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001184.s015.DOC" value="45056"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001184&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001184" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001184&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001184&volume=&issue=&title=Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome&author_name=Shicai%20Fan%2C%20Fang%20Fang%2C%20Xuegong%20Zhang%2C%20Michael%20Q.%20Zhang&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001184" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001184" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001184" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001184&amp;t=Putative%20Zinc%20Finger%20Protein%20Binding%20Sites%20Are%20Over-Represented%20in%20the%20Boundaries%20of%20Methylation-Resistant%20CpG%20Islands%20in%20the%20Human%20Genome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001184&title=Putative%20Zinc%20Finger%20Protein%20Binding%20Sites%20Are%20Over-Represented%20in%20the%20Boundaries%20of%20Methylation-Resistant%20CpG%20Islands%20in%20the%20Human%20Genome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001184&amp;title=Putative%20Zinc%20Finger%20Protein%20Binding%20Sites%20Are%20Over-Represented%20in%20the%20Boundaries%20of%20Methylation-Resistant%20CpG%20Islands%20in%20the%20Human%20Genome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001184" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001184&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Putative Zinc Finger Protein Binding Sites Are Over-Represented in the Boundaries of Methylation-Resistant CpG Islands in the Human Genome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001184';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Putative%20Zinc%20Finger%20Protein%20Binding%20Sites%20Are%20Over-Represented%20in%20the%20Boundaries%20of%20Methylation-Resistant%20CpG%20Islands%20in%20the%20Human%20Genome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001184" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001184" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin%22" title="Search for articles in the subject area:'Chromatin'"><div class="flagText">Chromatin</div></a>
              <div data-categoryid="31933" data-articleid="25200"
                   data-categoryname="Chromatin"
                   class="flagImage" title="Flag 'Chromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="25200"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25200"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="25200"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="25200"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="25200"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="25200"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="25200"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=366'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3131'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=41&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
