

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Detection of a MicroRNA Signal in an In Vivo Expression Set of mRNAs</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000804"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000804" />

    <meta name="citation_title" content="Detection of a MicroRNA Signal in an In Vivo Expression Set of mRNAs"/>
    <meta itemprop="name" content="Detection of a MicroRNA Signal in an In Vivo Expression Set of mRNAs"/>

      <meta name="citation_author" content="Tsunglin Liu"/>
            <meta name="citation_author_institution" content="Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Thales Papagiannakopoulos"/>
            <meta name="citation_author_institution" content="Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Kathy Puskar"/>
            <meta name="citation_author_institution" content="Department of Computer Science, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Shuping Qi"/>
            <meta name="citation_author_institution" content="Departments of Psychiatry, Molecular and Medical Pharmacology, the Semel Institute and the Jonsson Comprehensive Cancer Center, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Fernando Santiago"/>
            <meta name="citation_author_institution" content="Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="William Clay"/>
            <meta name="citation_author_institution" content="Department of Mathematics, Jackson State University, Jackson, Mississippi, United States of America"/>
      <meta name="citation_author" content="Kaiqin Lao"/>
            <meta name="citation_author_institution" content="Applied Biosystems, Foster City, California, United States of America"/>
      <meta name="citation_author" content="Yohan Lee"/>
            <meta name="citation_author_institution" content="Department of Human Genetics, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Stanley F. Nelson"/>
            <meta name="citation_author_institution" content="Department of Human Genetics, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Harley I. Kornblum"/>
            <meta name="citation_author_institution" content="Departments of Psychiatry, Molecular and Medical Pharmacology, the Semel Institute and the Jonsson Comprehensive Cancer Center, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Frank Doyle"/>
            <meta name="citation_author_institution" content="Department of Chemical Engineering, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Linda Petzold"/>
            <meta name="citation_author_institution" content="Department of Computer Science, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Boris Shraiman"/>
            <meta name="citation_author_institution" content="Kavli Institute of Theoretical Physics, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Kenneth S. Kosik"/>
            <meta name="citation_author_institution" content="Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>

    <meta name="citation_date" content="2007/8/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000804.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e804"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mechanisms of microRNA-mediated gene regulation in animal cells.; citation_author=TW Nilsen; citation_journal_title=Trends Genet.; citation_number=1; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MicroRNA expression profiles classify human cancers.; citation_author=J Lu; citation_author=G Getz; citation_author=EA Miska; citation_author=E Alvarez-Saavedra; citation_author=J Lamb; citation_journal_title=Nature; citation_volume=435; citation_number=2; citation_pages=834-838; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene expression-based classification of malignant gliomas correlates better with survival than histological classification.; citation_author=CL Nutt; citation_author=DR Mani; citation_author=RA Betensky; citation_author=P Tamayo; citation_author=JG Cairncross; citation_journal_title=Cancer Res; citation_volume=63; citation_number=3; citation_pages=1602-1607; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Prediction of central nervous system embryonal tumour outcome based on gene expression.; citation_author=SL Pomeroy; citation_author=P Tamayo; citation_author=M Gaasenbeek; citation_author=LM Sturla; citation_author=M Angelo; citation_journal_title=Nature; citation_volume=415; citation_number=4; citation_pages=436-442; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Real time PCR profiling of 330 human mircoRNAs.; citation_author=K Lao; citation_author=NL Xu; citation_author=YA Sun; citation_author=KJ Livak; citation_author=NA Straus; citation_journal_title=Biotechnol J; citation_volume=2; citation_number=5; citation_pages=33-35; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are mircoRNA targets.; citation_author=BP Lewis; citation_author=CB Burge; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=120; citation_number=6; citation_pages=15-16; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The evolving role of microRNAs in animal gene expression.; citation_author=KB Massirer; citation_author=AE Pasquinelli; citation_journal_title=Bioessays; citation_volume=28; citation_number=7; citation_pages=449-452; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microarray profiling of microRNAs reveals frequent coexpression with neighboring miRNAs and host genes.; citation_author=S Baskerville; citation_author=DP Bartel; citation_journal_title=RNA; citation_volume=11; citation_number=8; citation_pages=241-247; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Acetylation of general transcription factors by histone acetyltransferases.; citation_author=A Imhof; citation_author=XJ Yang; citation_author=VV Ogryzko; citation_author=Y Nakatani; citation_author=AP Wolffe; citation_journal_title=Curr Biol; citation_volume=7; citation_number=9; citation_pages=689-692; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=p53 sites acetylated in vitro by PCAF and p300 are acetylated in vivo in response to DNA damage.; citation_author=L Liu; citation_author=DM Scolnick; citation_author=RC Trievel; citation_author=HB Zhang; citation_author=R Marmorstein; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=10; citation_pages=1202-1209; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Acetylation regulates the differentiation-specific functions of the retinoblastoma protein.; citation_author=DX Nguyen; citation_author=LA Baglia; citation_author=SM Huang; citation_author=CM Baker; citation_author=DJ McCance; citation_journal_title=EMBOJ; citation_volume=23; citation_number=11; citation_pages=1609-1618; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A global map of p53 transcription-factor binding sites in the human genome.; citation_author=CL Wei; citation_author=Q Wu; citation_author=VB Vega; citation_author=KP Chiu; citation_author=P Ng; citation_journal_title=Cell; citation_volume=124; citation_number=12; citation_pages=207-219; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Theacetyltransferase p300/CBP-associated factor is a p53 target gene in breast tumor cells.; citation_author=GS Watts; citation_author=MM Oshiro; citation_author=DJ Junk; citation_author=RJ Wozniak; citation_author=S Watterson; citation_journal_title=Neoplasia; citation_volume=6; citation_number=13; citation_pages=187-194; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cell-type-specific signatures of microRNAs on target mRNA expression.; citation_author=P Sood; citation_author=A Krek; citation_author=M Zavolan; citation_author=G Macino; citation_author=N Rajewsky; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=14; citation_pages=2746-2751; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs.; citation_author=LP Lim; citation_author=NC Lau; citation_author=P Garrett-Engele; citation_author=A Grimson; citation_author=JM Schelter; citation_journal_title=Nature; citation_volume=433; citation_number=15; citation_pages=769-73; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regression of the human dihydrofolate reductase gene by a non-coding interfering transcript.; citation_author=I Martianov; citation_author=A Ramadass; citation_author=A Serra Barros; citation_author=N Chow; citation_author=A Akoulitchev; citation_journal_title=Nature; citation_volume=445; citation_number=16; citation_pages=666-670; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Intrinsic ubiquitination activity of PCAF controls the stability of the oncoprotein Hdm2.; citation_author=LK Linares; citation_author=R Kiernan; citation_author=R Triboulet; citation_author=C Chable-Bessia; citation_author=D Latreille; citation_journal_title=Nat Cell Biol; citation_volume=9; citation_number=17; citation_pages=331-338; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Extensive modulation of a set of microRNAs in primary glioblastoma.; citation_author=SA Ciafre; citation_author=S Galardi; citation_author=A Mangiola; citation_author=M Ferracin; citation_author=CG Liu; citation_journal_title=Biochem Biophys Res Commun; citation_volume=334; citation_number=18; citation_pages=1351-1358; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Antisense inhibition of human miRNAs and indications for an involvement of miRNA in cell growth and apoptosis.; citation_author=AM Cheng; citation_author=MW Byrom; citation_author=J Shelton; citation_author=LP Ford; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=19; citation_pages=1290-1297; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling of gliomas strongly predicts survival.; citation_author=WA Freije; citation_author=FE Castro-Vargas; citation_author=Z Fang; citation_author=S Horvath; citation_author=T Cloughesy; citation_journal_title=Cancer Res; citation_volume=64; citation_number=20; citation_pages=6503-6510; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Optimized high-throughput microRNA expression profiling provides novel biomarker assessment of clinical prostate and breast cancer biopsies.; citation_author=MD Mattie; citation_author=CC Benz; citation_author=J Bowers; citation_author=K Sensinger; citation_author=L Wong; citation_journal_title=Mol Cancer; citation_volume=5; citation_number=21; citation_pages=24; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Model-based analysis of oligonucleotide arrays: Expression index computation and outlier detection.; citation_author=C Li; citation_author=WH Wong; citation_journal_title=Proc Natl Acad SciUSA; citation_volume=98; citation_number=22; citation_pages=31-36; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2007.; citation_author=TJ Hubbard; citation_author=BL Aken; citation_author=K Beal; citation_author=B Ballester; citation_author=M Caccamo; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=23; citation_pages=D610-617; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Biomart and bioconductor: a powerful link between biological databases and microarry data analysis.; citation_author=S Durinck; citation_author=Y Moreau; citation_author=A Kasprzyk; citation_author=S Daivis; citation_author=B De Moor; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=24; citation_pages=3439-3440; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the mathematical foundations of theoretical statistics.; citation_author=RA Fisher; citation_journal_title=Phil Trans Loyal Soc; citation_volume=A222; citation_number=25; citation_pages=309-368; citation_date=1922; " />
      <meta name="citation_reference" content="citation_title=miRBase: microRNA sequences, targets, and gene nomenclature.; citation_author=S Griffiths-Jones; citation_author=RJ Grocock; citation_author=S van Dongen; citation_author=A Bateman; citation_author=AJ Enright; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=26; citation_pages=D140-D144; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000804" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Detection of a MicroRNA Signal in an In Vivo Expression Set of mRNAs"/>
    <meta name="twitter:description" content="BackgroundmicroRNAs (miRNAs) are approximately 21 nucleotide non-coding transcripts capable of regulating gene expression. The most widely studied mechanism of regulation involves binding of a miRNA to the target mRNA. As a result, translation of the target mRNA is inhibited and the mRNA may be destabilized. The inhibitory effects of miRNAs have been linked to diverse cellular processes including malignant proliferation, apoptosis, development, differentiation, and metabolic processes. We asked whether endogenous fluctuations in a set of mRNA and miRNA profiles contain correlated changes that are statistically distinguishable from the many other fluctuations in the data set.Methodology/Principal FindingsRNA was extracted from 12 human primary brain tumor biopsies. These samples were used to determine genome-wide mRNA expression levels by microarray analysis and a miRNA profile by real-time reverse transcription PCR. Correlation coefficients were determined for all possible mRNA-miRNA pairs and the distribution of these correlations compared to the random distribution. An excess of high positive and negative correlation pairs were observed at the tails of these distributions. Most of these highest correlation pairs do not contain sufficiently complementary sequences to predict a target relationship; nor do they lie in physical proximity to each other. However, by examining pairs in which the significance of the correlation coefficients is modestly relaxed, negative correlations do tend to predict targets and positive correlations tend to predict physically proximate pairs. A subset of high correlation pairs were experimentally validated by over-expressing or suppressing a miRNA and measuring the correlated mRNAs.Conclusions/SignificanceSufficient information exists within a set of tumor samples to detect endogenous correlations between miRNA and mRNA levels. Based on the validations the causal arrow for these correlations is likely to be directed from the miRNAs to the mRNAs. From these data sets, we inferred and validated a tumor suppression pathway linked to miR-181c."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000804.g009"/>

  <meta property="og:title" content="Detection of a MicroRNA Signal in an In Vivo Expression Set of mRNAs" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000804" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4514'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2003'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3991&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000804">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000804" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000804&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Detection of a MicroRNA Signal in an <i>In Vivo</i> Expression Set of mRNAs
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tsunglin Liu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Tsunglin Liu, Thales Papagiannakopoulos</p>

              
              

                <p>Affiliations:
                  Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America, 
                  Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thales Papagiannakopoulos
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Tsunglin Liu, Thales Papagiannakopoulos</p>

              
              

                <p>Affiliations:
                  Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America, 
                  Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kathy Puskar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Computer Science, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shuping Qi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departments of Psychiatry, Molecular and Medical Pharmacology, the Semel Institute and the Jonsson Comprehensive Cancer Center, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fernando Santiago, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America, 
                  Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William Clay, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Mathematics, Jackson State University, Jackson, Mississippi, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kaiqin Lao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Applied Biosystems, Foster City, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yohan Lee, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stanley F. Nelson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harley I. Kornblum, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departments of Psychiatry, Molecular and Medical Pharmacology, the Semel Institute and the Jonsson Comprehensive Cancer Center, David Geffen School of Medicine, University of California at Los Angeles, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frank Doyle, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Chemical Engineering, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Linda Petzold, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Computer Science, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Boris Shraiman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Kavli Institute of Theoretical Physics, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth S. Kosik
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kosik@lifesci.ucsb.edu">kosik@lifesci.ucsb.edu</a></p>

                <p>Affiliations:
                  Neuroscience Research Institute, University of California at Santa Barbara, Santa Barbara, California, United States of America, 
                  Department of Molecular, Cellular and Developmental Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 29, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000804</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000804" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000804" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000804" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000804" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000804-g001" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-t001" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g002" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g003" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-t002" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g004" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g005" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-t003" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g006" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g007" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g008" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000804-g009" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000804">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000804" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>microRNAs (miRNAs) are approximately 21 nucleotide non-coding transcripts capable of regulating gene expression. The most widely studied mechanism of regulation involves binding of a miRNA to the target mRNA. As a result, translation of the target mRNA is inhibited and the mRNA may be destabilized. The inhibitory effects of miRNAs have been linked to diverse cellular processes including malignant proliferation, apoptosis, development, differentiation, and metabolic processes. We asked whether endogenous fluctuations in a set of mRNA and miRNA profiles contain correlated changes that are statistically distinguishable from the many other fluctuations in the data set.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>RNA was extracted from 12 human primary brain tumor biopsies. These samples were used to determine genome-wide mRNA expression levels by microarray analysis and a miRNA profile by real-time reverse transcription PCR. Correlation coefficients were determined for all possible mRNA-miRNA pairs and the distribution of these correlations compared to the random distribution. An excess of high positive and negative correlation pairs were observed at the tails of these distributions. Most of these highest correlation pairs do not contain sufficiently complementary sequences to predict a target relationship; nor do they lie in physical proximity to each other. However, by examining pairs in which the significance of the correlation coefficients is modestly relaxed, negative correlations do tend to predict targets and positive correlations tend to predict physically proximate pairs. A subset of high correlation pairs were experimentally validated by over-expressing or suppressing a miRNA and measuring the correlated mRNAs.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Sufficient information exists within a set of tumor samples to detect endogenous correlations between miRNA and mRNA levels. Based on the validations the causal arrow for these correlations is likely to be directed from the miRNAs to the mRNAs. From these data sets, we inferred and validated a tumor suppression pathway linked to <em>miR-181c</em>.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Liu T, Papagiannakopoulos T, Puskar K, Qi S, Santiago F, et al.  (2007) Detection of a MicroRNA Signal in an <em>In Vivo</em> Expression Set of mRNAs. PLoS ONE 2(8):
          e804.
            doi:10.1371/journal.pone.0000804</p><p><strong>Academic Editor: </strong>Laszlo Tora, Institute of Genetics and Molecular and Cellular Biology, France</p><p><strong>Received:</strong> April 17, 2007; <strong>Accepted:</strong> August 1, 2007; <strong>Published:</strong> August 29, 2007</p><p><strong>Copyright:</strong> © 2007 Liu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the W.M. Keck Foundation, the NIH Neuroscience Microarray Consortium U24NSNS052108, and NS052563.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>miRNAs regulate both the stability and translatability of their mRNA targets <a href="#pone.0000804-Nilsen1">[1]</a>. This regulation occurs by binding of the ~21 nucleotide mature miRNA to an imperfectly matched sequence within the target mRNA in animals. Most descriptions of miRNA function focus either on their role as post-transcriptional regulators of target mRNAs or at a much higher level on their cell biological processes and organismal roles. Between these two roles lies a poorly understood landscape through which the miRNA signal must travel to implement its cell biological function at the phenotypic level. Following reductions in the level of a protein encoded by a target mRNA, subsequent effects may alter the levels of other mRNAs and thus changes in miRNA expression will ramify through the transcriptional profile. As the networked link to miRNA fluctuation becomes more distant and diluted, correlative changes in mRNAs may no longer be discernable among the many other factors that control change in mRNA levels across a set of transcriptional profiles. However, changes in mRNAs that may not be direct targets, but lie closer to the consequences of miRNA inhibitory effects may be discernable in the transcriptional profile.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Often the effects of miRNAs on mRNAs are studied by expressing or suppressing the miRNA in cells. Because it is very likely that non-physiological levels of miRNAs will alter the target field, i.e. the specific set of mRNAs targeted, a more accurate measure of miRNA effects requires endogenous determinations. To discover cases in which miRNAs exert detectably strong relationship to the level of an mRNA within a set of transcriptional profiles we collected a set of human glioma samples and performed broad-scale analyses of miRNA levels and mRNA levels. Broad based gene expression in cancer data sets, which are known to vary in a manner that is predictive of tumor grade and patient prognosis <a href="#pone.0000804-Lu1">[2]</a>–<a href="#pone.0000804-Pomeroy1">[4]</a>, were used to find significant correlations between endogenous fluctuations in the miRNA and mRNA expression.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>mRNAs were measured with oligonucleotide arrays (Affymetrix). The miRNA profile was measured by multiplex RT-PCR (Applied Biosystems) which can discern the specificity of miRNAs even for very similar sequences from the same family <a href="#pone.0000804-Lao1">[5]</a>. These methods are more reliable when comparing the expression of the same miRNA, which was done in this study. The statistical analysis applied to the data set revealed an excess of high correlation miRNA-mRNA pairs beyond the number expected by chance. A subset of these high correlation pairs was experimentally validated and revealed a network of linked genes related to carcinogenesis.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Detection of highly correlated of miRNA-mRNA pairs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We measured the expression levels of 330 miRNAs and about 14,500 genes in 12 brain tumor biopsies from humans using multiplex RT-PCR and Affymetrix oligonucleotide microarrays, respectively. Before correlation analysis, we filtered data with substantial noise due to low abundance in order to gain more confidence in the resulting correlations (see <a href="#s4">Methods</a>). We also filtered those data with little variation across the 12 samples because the absence of high variation will result in correlations mainly due to noise. These criteria left 93 miRNAs and 7,089 probe sets for the analysis. This approach was chosen because we intended to ask whether there existed significant correlation within a highly accurate data set, instead of extracting lower reliability data from the entire data set. Both the mRNA and the miRNA data sets were normalized by quantile normalization (see <a href="#s4">Methods</a>). After normalization, we calculated the Pearson's correlation between the expression profiles of the miRNAs and the mRNAs, and compared these correlations with the random case. The random case is referred to as any result obtained from randomly generated or permuted expression profiles. Here, we asked whether the distribution of 93*7,089 = 659,277 correlation coefficients, r's, from the data set was significantly different from the random case.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The distribution of r's between our data set and the random case (see <a href="#s4">Methods</a> for a derived closed form solution of the random case) clearly differed (<a href="#pone-0000804-g001">Figure 1A</a>). To ensure a truly random control, we randomly permuted the order of samples in miRNA (or mRNA) profiles and repeated the analysis 100 times. Such random permutations preserved the autocorrelations among miRNAs and mRNAs, and generated results close to the random case. Quantitatively, our data set output 74 and 1631 correlations below −0.9 and −0.8 respectively, while the output from the random cases was 23.65 and 615.12 on average. The binomial probability to obtain at least 74 and 1631 correlations from 659277 random trials with a probability of success (23.65/659277) and (615.12/659277) was 1.4*10<sup>−11</sup> and 1.6*10<sup>−11</sup> respectively. Similarly, the binomial probability to obtain at least 75 and 1735 correlations above 0.9 and 0.8 from 659277 random trials with a probability of success (23.73/659277) and (628.19/659277) was 3.6*10<sup>−11</sup> and 1.1*10<sup>−11</sup> respectively. These small p-values indicated the significant difference between our data set and the random cases. Thus, the correlations between miRNAs and mRNAs contained information regarding interactions, but did not suggest a causal direction of the interaction. The difference between the distribution of correlations in the data set and that of the random distributions, was re-plotted with the ratio between the number of r's in each range of experimentally derived coefficients to the random case (<a href="#pone-0000804-g001">Figure1B</a>). At both positive and negative correlation tails, the miRNA-mRNA correlations were stronger and more numerous than expected in the random case.</p>
<div class="figure" id="pone-0000804-g001"><div class="img"><a name="pone-0000804-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distribution of correlation coefficients.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) The histogram consists of 40 bins ranging from −1 to 1. (B) The ratio of number of correlation coefficients in the experimental data to the random case for bin in <a href="#pone-0000804-g001">Figureô 1A</a>. At high correlations, the number of microRNA-mRNA pairs is larger than expected in the random case. This result suggests information exists in the correlation coefficients between miRNAs and mRNAs.</p>
<span>doi:10.1371/journal.pone.0000804.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>As seen in <a href="#pone-0000804-g001">Figure1B</a> the high positive and negative correlations at the tails indicated a set of miRNA-mRNA correlations that were stronger and more numerous than expected in the random case. To define this set we defined a cutoff r<sub>c</sub> according to the equality<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.e001&amp;representation=PNG" class="inline-graphic"></span><br>This says that for a given miRNA, the expected number of mRNAs with correlation greater than r<sub>c</sub> (~0.916 with 7089 mRNAs) is only 0.1 in the random case. A correlation coefficient was considered high when the absolute value was above the 0.916 cutoff. With a total of 93 miRNAs, we would expect 9.3 pairs with a high negative correlation by chance; however we obtained 35 such pairs from our experimental data (<a href="#pone-0000804-t001">Table1</a>), of which 27% were expected to occur by chance. Similarly, for the positive correlations, we obtained 37 such pairs, of which 25% were expected to occur by chance. In other words, approximately one quarter of the total high correlation pairs—72 positive and negative correlations (<a href="#pone-0000804-t001">Table1</a>)—could be false positives. Nevertheless, the binomial probability in obtaining at least 72 coefficients above the cutoff with the probability of success 0.1 is very low (P~5*10<sup>−21</sup>).</p>
<div class="figure" id="pone-0000804-t001"><div class="img"><a name="pone-0000804-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>List of miRNA-mRNA high correlation pairs and number of predicted target sites.</span></strong></p><span>doi:10.1371/journal.pone.0000804.t001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The clear correlation signal arising from just 12 samples was somewhat surprising given the very large number of correlations tested. However, the finding was validated by demonstrating a decline in excess strong correlations over the random case with fewer samples (<a href="#pone.0000804.s002">Figure S2</a>, where the ratio in number of coefficients, similar to <a href="#pone-0000804-g001">Figure1B</a>, was plotted). The signal declined with fewer samples because the correlation between a miRNA and an mRNA in the actual data did not increase as fast as the probability for high correlations in the random case. This difference from the random case increased our confidence that the signal was bona fide.</p>


<h4>Target Predictions among Correlated Pairs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Among the explanations for high negative correlation pairs is direct targeting of the mRNA by the paired miRNA. Therefore, we checked the percentage of miRNA-mRNA high correlation pairs that were predicted as target pairs by TargetScan <a href="#pone.0000804-Lewis1">[6]</a>. Among the 35 high negative correlation pairs four were predicted to have one target site (<a href="#pone-0000804-t001">Table1</a>). This number of predicted targets may represent chance because among 37 high positive correlation pairs five were predicted to have one target site. No high positive correlation pair has two predicted sites, but among the high correlation negative correlation pairs two pairs (in addition to the four single target site pairs) had two predicted target sites (<a href="#pone-0000804-t001">Table1</a>). Among the 659277 miRNA-mRNA pairs, 84238 (~12.8%) were predicted targets. Thus, with 35 (or 37) pairs, we would expect about 4~5 predicted target pairs by chance, which was the case for the highly correlated miRNA-mRNA pairs. These findings demonstrate that the dominant feature of the miRNA-mRNA relationship in terms of the magnitude of change in the mRNAs is indirectly related to the binding of the miRNA to its mRNA targets. miRNA binding to a target usually has only a modest effect on the level of the target mRNA and a greater effect on the encoded protein. Therefore, direct binding effects are more difficult to detect in an endogenous data set of profiles. However, the ability to detect specific endogenous changes in an mRNA profile associated with miRNA fluctuations has not been previously reported.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Although most of the strongest miRNA-mRNA correlations are not likely to be due to direct targeting, it is possible to detect in the data set a weak effect that is consistent with miRNA targeting to specific mRNAs. The coefficients of all miRNA-mRNA pairs were split into 40 bins, ranging from −1 to 1 with a bin size 0.05 and the percentage of predicted target pairs for each bin calculated (<a href="#pone-0000804-g002">Figure2</a>). A clear trend was observed as the correlation went from negative to positive: the percentage of predicted target pairs decreased. As a control, we repeated the target analysis for the 100 random permutations mentioned above and compared their slopes (see <a href="#s4">Methods</a>). None of the 100 random permutations gave a slope as negative as our observation, which indicated the significance of the trend. This finding is consistent with data that miRNAs can down regulate their targeted mRNAs <a href="#pone.0000804-Massirer1">[7]</a>, and therefore more predicted target pairs should lie in the region of negative correlation. This trend was apparent in the data even though about 87% of the miRNA-mRNA pairs were not predicted as target pairs by TargetScan.</p>
<div class="figure" id="pone-0000804-g002"><div class="img"><a name="pone-0000804-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Percentage of predicted targets across a range of correlation coefficients.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The percentage of targets over the range of correlation defined in <a href="#pone-0000804-g001">Figureô 1</a> decreases with the increasing correlation between miRNAs and mRNAs. This implies that the correlation indeed captures information about target prediction, and the negative slope might be related to direct targeting effect between miRNAs and mRNAs.</p>
<span>doi:10.1371/journal.pone.0000804.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Having detected a relationship between negative correlation values and the percentage of predicted targets, we next asked whether for a given miRNA, do its predicted targets show more negative correlations. To this end, we split the 7,089 correlations of each miRNA into two groups, those with and without predicted targets. We then applied one sided Wilcoxon Rank Sum (WRS) test to check the null hypothesis, that the correlation of the first group is not lower than the second group. For each miRNA, the one sided WRS test gave a P-value, and a P-value smaller than a given cutoff indicated that the correlation with the predicted targets was more negative than that with non-predicted targets. Of 91 miRNAs (two miRNAs from the set of 93 were not in TargetScan.), 28 (~30%) miRNAs fell below the significance level P = 0.05 (<a href="#pone-0000804-g003">Figure3</a>). Using the Bonferroni correction for multiple tests, 7 miRNAs were found to show significantly more negative correlations with the predicted targets, while only two were found to show more positive correlation (<a href="#pone-0000804-g003">Figure3</a>). No relationship was found between miRNA abundance and this difference in the rank sum. Thus, those pairs which are predicted targets, but showed a more positive correlation may be indirectly related.</p>
<div class="figure" id="pone-0000804-g003"><div class="img"><a name="pone-0000804-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Histogram of P values from the WRS test for each of the miRNAs.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The WRS test compared the correlations of the predicted targets with that of the non-predicted ones for each of the 91 miRNAs. As the correlation went from negative to positive the percentage of predicted target pairs decreased. Interestingly, some highly positively correlated miRNAs had high P values, which suggests an indirect relationship with the predicted target.</p>
<span>doi:10.1371/journal.pone.0000804.g003</span></div>

<h4>Positive Correlations between Proximate Pairs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Some miRNAs are intronic, and these miRNAs show correlated expression patterns with their host genes <a href="#pone.0000804-Baskerville1">[8]</a>. This observation was replicated in our data where almost all the miRNAs and their host genes showed positive correlations (<a href="#pone-0000804-t002">Table2</a>). Three miRNA-mRNA pairs, <em>miR-25/MCM7, miR-15b/SMC4L1</em>, and <em>miR-30c/NFYC</em> also appeared in the previous study. The reported correlation coefficients were 0.838, 0.509, and 0.406, respectively, values which are consistent with our data (<a href="#pone-0000804-t002">Table2</a>). We also observed the co-expression of neighboring miRNAs as reported in the same study (<a href="#pone.0000804.s001">Figure S1</a>).</p>
<div class="figure" id="pone-0000804-t002"><div class="img"><a name="pone-0000804-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Correlations between intronic miRNAs and their host genes.</span></strong></p><span>doi:10.1371/journal.pone.0000804.t002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>A positive correlation signal was also detected between mRNAs and those miRNAs that were close to, but not within the gene encoding the mRNA (<a href="#pone-0000804-g004">Figure4</a>). Although weak, the positive correlation signal persisted beyond the 1 Mb shown in the figure. Thus, co-expression comprises part of the positively correlated signal. However, the strongest co-expression signal from the intronic miRNAs and the host genes did not contain a correlation greater than 0.8 (<a href="#pone-0000804-g004">Figure4</a>). Therefore, proximity did not explain the very highly positive correlations.</p>
<div class="figure" id="pone-0000804-g004"><div class="img"><a name="pone-0000804-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution of correlations between miRNAs and proximate mRNAs.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>This figure showed the distributions for 3 groups of miRNA-mRNA pairs, classified by the distance between a miRNA and an mRNA on the same chromosome. The y-axis shows the percentage of miRNA-mRNA pairs in each range of correlation (−1 to 1 with binsize 0.1). An intronic miRNA had a distance zero to its host gene. The average length of a human gene, 55Kb (<a href="http://www.ncbi.nlm.nih.gov/Web/Newsltr/Spring03/human.html">http://www.ncbi.nlm.nih.gov/Web/Newsltr/​Spring03/human.html</a>), was chosen as one distance boundary. As the distance between the correlated pair increased, the distribution increasingly shifted away from positive correlations to a normal distribution.</p>
<span>doi:10.1371/journal.pone.0000804.g004</span></div>

<h4>Validation of High Correlation Pairs</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Three miRNAs (<em>miR-181c, miR-182, miR-19a</em>) were selected from among the miRNA-mRNA high correlation pairs to test whether the correlations predicted changes that could be reproduced experimentally. <em>miR-181c</em> and <em>miR-182</em> were individually transfected as pre-miRNAs into U251 glioblastoma cells to increase their levels 2–4 fold (data not shown). <em>miR-19a</em> was downregulated by addition of 2′<em>-O-</em>methyl (2′<em>-O-</em>Me)-<em>miR-19a</em>. The U251 cells were derived from a high-grade glioblastoma; therefore, they represent a reasonable cell culture system for experimental validations of data derived from glioblastoma biopsies. We determined whether the effects of each miRNA on the mRNAs with which it was highly correlated corresponded to the relationship predicted in the correlation data set. These experiments also served to suggest the direction of causality between the correlated pair.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Twenty-four hours after the transfections of pre-<em>miR-182</em>, pre-<em>miR-181c</em>, 2′<em>-O-</em>Me-<em>miR-19a</em>, as well as a pre-miR scramble or a 2′<em>-O-</em>Me scramble, we collected total RNA for real-time RT-PCR to determine the relative changes in the levels of the correlated mRNAs (<a href="#pone-0000804-g005">Figure5</a>). We computed a delta Ct by taking the difference in the mRNA values in cells exposed to the precursor miRNA or the 2′<em>-O-</em>Me-miRNA and cells exposed to the appropriate scrambled control. As negative controls we determined the levels of ten mRNAs that did not show any significant correlation with the miRNA being tested. For each miRNA, the expected outcomes are shown in the left box of <a href="#pone-0000804-t003">Table3</a> as either positive (+) or negative (−) correlations based on the high correlation pairs in <a href="#pone-0000804-t001">Table1</a>, or no change (blank space) because the measured mRNA is not highly correlated with endogenous fluctuations in the test miRNA. The actual outcome of the experiment, i.e. change in mRNA level relative to the scrambled, is shown in the right box (<a href="#pone-0000804-t003">Table3</a>). Up or down changes of more than two-fold in the mRNA level are indicated by an up or down arrow; a difference less than two fold was denoted by a blank space. Most boxes represent control cases in which no significant correlation existed in the data set, nor did any change in the mRNA occur in the experiment (<a href="#pone-0000804-t003">Table3</a>). Thus, all of these cases bore out the predictions. Five cases with predicted correlative changes were borne out by experiment. In four cases, the data set predicted a correlation that did not occur experimentally and in one case the experiment and the prediction went in opposite directions (<a href="#pone-0000804-t003">Table3</a>). Given that we expect 26% to be false positives and the imprecision of U251 cells to replicate the in vivo fluctuations among the more diverse tumor samples this result indicates a very high degree of predictability.</p>
<div class="figure" id="pone-0000804-g005"><div class="img"><a name="pone-0000804-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Experimental Validation of Highly correlated miRNA/transcript pairs.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The above figure represents Ct<sub>scrambled</sub>-Ct<sub>pre-mir/2′<em>-O-</em>Me</sub> from three individual transfection experiments of pre-<em>miR-181c</em>, pre-<em>miR-182</em>, 2′<em>-O-</em>Me-<em>miR-19a</em>, 2′<em>-O-</em>Me-scrambled and pre-mir-scrambled. <em>PCAF, ASCC1, FADS2, HPS1, ANXA4, GSTO1, IFITM1, DDX11, TTC3</em> and <em>PRR13</em> levels were determined after transfection.</p>
<span>doi:10.1371/journal.pone.0000804.g005</span></div><div class="figure" id="pone-0000804-t003"><div class="img"><a name="pone-0000804-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Validation of highly correlated pairs.</span></strong></p><span>doi:10.1371/journal.pone.0000804.t003</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Among the validated high correlation pairs was a positive correlation between <em>miR-181c</em> and <em>p300/CBP-associated factor</em> (<em>PCAF</em>). <em>PCAF</em> is a histone acetyltransferase that induces transcriptional activation by histone <em>H3</em> acetylation of target promoters <a href="#pone.0000804-Imhof1">[9]</a>. Additionally, <em>PCAF</em> can acetylate lysines on non-histone proteins such as <em>tumor protein p53</em> and <em>retinoblastoma 1</em>, activating their tumor suppressor activity <a href="#pone.0000804-Liu1">[10]</a>, <a href="#pone.0000804-Nguyen1">[11]</a>. Abrogating <em>PCAF</em> function by lowering its transcript levels is a mechanism by which the transforming growth factor-beta, retinoblastoma, and <em>p53</em> tumor suppressive pathways are negatively modulated in cancer. The validation experiment suggested that the direction of causality very likely went from <em>miR-181c</em> to <em>PCAF</em> (although if the pair is networked as a negative feedback loop in cis, causality <em>in vivo</em> could be bi-directional).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>We performed western blot analysis to verify the increase of <em>PCAF</em> protein levels in response to <em>miR-181c</em> upregulation in U251 cells and in an additional glioma line, U87. Twenty-four hours post-transfection we observed a significant increase in <em>PCAF</em> in the pre-<em>miR-181c</em> treated U87 and U251 cells when compared to the negative control scramble (<a href="#pone-0000804-g006">Figure6A,B</a>). This observation suggested that the increase of <em>PCAF</em> observed in the qPCR experiments was translated into an increase in the protein/enzyme levels in both glioma lines. The upregulation of <em>PCAF</em> in response to increased <em>miR-181c</em> could occur through a known transactivator of <em>PCAF</em>, <em>p53</em> <a href="#pone.0000804-Wei1">[12]</a>, <a href="#pone.0000804-Watts1">[13]</a>. We checked the levels of <em>p53</em> in both U251 and U87 treated cells and observed a significant increase of <em>p53</em> protein levels in the <em>miR-181c</em> treated U251 cells (<a href="#pone-0000804-g006">Figure6C,D</a>); however we could not detect <em>p53</em> in the U87 cells.</p>
<div class="figure" id="pone-0000804-g006"><div class="img"><a name="pone-0000804-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Immunoblot analysis of <em>miR-181c</em> treated cells.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Immunoblots of U251 and U87 transfected cells transfected with either <em>miR-181c</em> pre-miRNA or a scrambled sequence. <em>PCAF</em> (A) and <em>p53</em> (C) were immunodetected and actin was used as a loading control. Quantification of the immunoblots showed a significant difference in B and C (p&lt;0.01, t-test in all three comparisons).</p>
<span>doi:10.1371/journal.pone.0000804.g006</span></div><a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>We then tested how <em>miR-181c</em> upregulation affected cell proliferation and apoptosis, because the upregulation of <em>PCAF</em> and <em>p53</em> protein levels in U251 cells could lead to activation of a plethora of other downstream tumor suppressive genes that can suppress the oncogenic state of these glioma cells. We used the Cell-Titer Glo assay to determine the number of metabolically viable cells upon addition of <em>miR-181c</em> as compared to the negative control scramble. We observed ~30% (p&lt;0.01, t-test) decrease in viable cells in the <em>miR-181c</em> treated sample, relative to the negative control (<a href="#pone-0000804-g007">Figure7A</a>). This decrease could occur due to increased apoptosis, therefore we checked the activity of Caspase-3/7 in cells that were treated under the same conditions. There was ~12% (p&lt;0.05, t-test) increase in Caspase-3/7 activity in response to <em>miR-181c</em> upregulation relative to the negative control (<a href="#pone-0000804-g007">Figure7B</a>). These results suggest that the decrease of cell numbers with <em>miR-181c</em> addition could be partially explained by increased apoptosis. However, the level of apoptosis that we observed does not fully account for the much higher difference in the viable cell numbers, suggesting that <em>miR-181c</em> is also having an anti-proliferative effect which could be mediated through <em>p53</em> and <em>PCAF</em> (<a href="#pone-0000804-g008">Figure8</a>).</p>
<div class="figure" id="pone-0000804-g007"><div class="img"><a name="pone-0000804-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Cell Growth and Apoptosis Assays.</span></strong></p><a id="article1.body1.sec2.sec4.fig3.caption1.p1" name="article1.body1.sec2.sec4.fig3.caption1.p1"></a><p>U251 cells were treated with pre-<em>miR-181c</em> and negative control scramble. 24 hrs post-transfection, a) cell growth (p&lt;0.01, t-test) and b) apoptosis were assayed (p&lt;0.05, t-test).</p>
<span>doi:10.1371/journal.pone.0000804.g007</span></div><div class="figure" id="pone-0000804-g008"><div class="img"><a name="pone-0000804-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Tumor suppressive pathway involving <em>miR-181c.</em></span></strong></p><span>doi:10.1371/journal.pone.0000804.g008</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>RNA from 12 brain tumor samples with some variation in their histological diagnoses was sufficient to detect a set of miRNA-mRNA pairs whose fluctuations are highly correlated. The merit of this statistical detection method is that it avoids non-physiological conditions required to alter gene expression. The number of such high correlation pairs was significantly greater than expected in the random case (<a href="#pone-0000804-g001">Figure1</a>). A weak positive correlative effect due to distance between the pairs could be detected (<a href="#pone-0000804-g004">Figure4</a>) as well as a weak negative correlative effect due to direct miRNA targeting of mRNAs (<a href="#pone-0000804-g002">Figure2</a>,<a href="#pone-0000804-g003">3</a>). However, among the very high correlation pairs only a small number bore a predicted target relationship (11 of 72, <a href="#pone-0000804-t001">Table1</a>) or were located on the same chromosome (7 of 72). For the remaining high correlation pairs the data does not imply any direction of casualty—is the correlation due to an effect of the miRNA on the mRNA or vice versa? Key to this approach was the endogeneous variations among the samples. In other words, the correlations might stem from individual differences and are not tumor specific. By restricting the samples to gliomas rather than including healthy controls, these data in general cannot directly address carcinogenesis. The interpretation of the correlations is limited to miRNA/mRNA relationships solely within our sample set.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Validation of the correlations was obtained by over-expressing or suppressing the implicated miRNA and measuring the mRNAs predicted to correlate. These experiments frequently matched the prediction and suggested that in these cases, the correlation was due to an effect of the miRNA on the mRNA (<a href="#pone-0000804-t003">Table3</a>). As expected, some pairs were not validated. Among the reasons for failure to validate are: (a) some of the high correlation pairs are false positives; (b) the U251 cells do not exactly replicate the tumors; (c) in some cases more than one miRNA contributes to the effect; (d) the direction of causality may pass from mRNA to miRNA; (e) direct causality does not underlie the relationship, rather the high correlation is the consequence that both the miRNA and the mRNA have a true biological relationship with a third element. Nevertheless, the statistical approach to endogenous fluctuations can reveal <em>in vivo</em> effects of miRNAs on the transcriptional profile.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Global effects of miRNAs upon the transcriptional profile and tissue specificity of mRNA expression have been reported <a href="#pone.0000804-Sood1">[14]</a>, <a href="#pone.0000804-Lim1">[15]</a>. However, these studies have generally focused on miRNA targets which show more modest effects compared to the non-target effects described here. Furthermore, in one of the cited studies <a href="#pone.0000804-Lim1">[15]</a> the over-expression of a miRNA by transfection can create non-physiological changes upon the mRNA target field. Because direct miRNA targeting is unlikely in the absence of a seed region, as observed in most of the high correlation pairs, other explanations for how the high correlation pairs might arise were schematized (<a href="#pone-0000804-g009">Figure9</a>). Among the possible explanations for high correlation pairing is the binding of the miRNA to the promoter region of the paired gene with the result that transcription increases in those sets with positive correlations. A non-coding RNA binding to a promoter has been described <a href="#pone.0000804-Martianov1">[16]</a>, albeit in this case suppression occurred. Also possible is that the mRNA drives the effect on the miRNA.</p>
<div class="figure" id="pone-0000804-g009"><div class="img"><a name="pone-0000804-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000804" data-uri="info:doi/10.1371/journal.pone.0000804.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000804.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000804.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000804.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Mechanisms through which high miRNA-mRNA correlation pairs could operate.</span></strong></p><span>doi:10.1371/journal.pone.0000804.g009</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Validation experiments allowed us to infer a tumor suppression pathway involving <em>miR-181c</em> (<a href="#pone-0000804-g008">Figure8</a>). <em>MiR-181c</em> and <em>PCAF</em> were among the endogenously highly correlated transcripts. We determined that <em>miR-181c</em> leads to an increase in <em>PCAF</em> and <em>p53</em> protein levels consistent with the known positive interactions between <em>PCAF</em> and <em>p53</em> (<a href="#pone-0000804-g006">Figure6</a>). At least two possible mechanisms could explain this result. The increase in <em>p53</em> protein levels could drive the increase in <em>PCAF</em> <a href="#pone.0000804-Wei1">[12]</a>, <a href="#pone.0000804-Watts1">[13]</a>. Alternatively, the increase in <em>p53</em> could occur through <em>PCAF</em>, which was recently shown to stabilize <em>p53</em> by inactivating <em>MDM2</em>, an E3 ubiquitin ligase that negatively controls <em>p53</em> stability <a href="#pone.0000804-Linares1">[17]</a>. Concluding, the positive correlation of <em>miR-181c</em> with these two tumor suppressor genes can potentially explain the inhibitory effect of <em>miR-181c</em> on cell growth and the more subtle increase in apoptosis observed in the glioma cells (<a href="#pone-0000804-g008">Figure8</a>). Interestingly, <em>miR-181c</em> has been reported to be low in a panel of high-grade glioblastomas tumors and glioblastomas cell lines <a href="#pone.0000804-Ciafre1">[18]</a> and inhibition of another member of the <em>miR-181</em> family, <em>miR-181a</em>, in A549 lung carcinoma cells induced cell growth <a href="#pone.0000804-Cheng1">[19]</a>. Lower levels of <em>miR-181c</em> in glioblastomas could indirectly decrease the transcript levels of the positively correlated <em>PCAF</em>, resulting in decreased activation of major tumor suppressive pathways in glioblastoma cells.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Sample preparation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Tissue from glioma resections were obtained and collected at UCLA under approval from the Institutional Review Board. Categorization into high or low grade was based on the formal pathology report, with WHO grades III and IV being termed “high grade” and grades I and II termed “low grade”. Histological diagnoses fell into the following categories: two low grade anaplastic mixed gliomas, one low grade oligodendroglioma, six high grade glioblastomas, two high grade gliosarcomas, and one high grade glioblastoma/gliosarcoma. From each the 12 tumor samples mRNAs and miRNAs were extracted.</p>


<h4>mRNA Microarrays</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Microarray data were obtained from the 12 tumors using Affymetrics HG-U133A array platforms as described previously <a href="#pone.0000804-Freije1">[20]</a>. Briefly, total RNA was isolated from tumor samples using a TRIzol reagent (Invitrogen Life Technologies, Carlsbad, CA) and was followed by a cleanup on a RNeasy column (Qiagen, Hilden, Germany). cDNA was generated and cRNA probes were generated using standard protocols. Aliquots of each sample were hybridized to U133A oligonucleotide microarray (GeneChip Human Genome U133A, Affymetrix, Santa Clara, CA), which represents ~14,500 human transcripts. The chips were scanned using the GeneArray scanner (Affymetrix). The CEL files generated by the Affymetrix Microarray Suite (MAS 5.0) were converted into DCP files using the DNA-Chip Analyzer (dChip 1.3, <a href="http://biosun1.harvard.edu/complab/dchip/">http://biosun1.harvard.edu/complab/dchip​/</a>). The processed data and the raw CEL files had been deposited in NCBI Gene Expression Omnibus database (series record GSE8692).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>All arrays were performed using standard protocols recommended by the manufacturer within the UCLA DNA Microarray Facility. Briefly, all RNA samples had a 28S/18S ratio over 1.5 with no evidence of degradation. All scanned images were visually inspected for surface defects. Quality Control assessment of array data was performed using GCOS v1.4 (Affymetrix, Santa Clara, Ca) and Array Assist v5.0 build 30237 (Stratagene, La Jolla, Ca). Hybridization and Poly-A controls were within expected parameters, and 3′/5′ ratios for actin and GAPDH were below 3 (mean Beta Actin 3′/5′ = 1.53 and mean GAPDH 3′/5′ = 1.02). GCOS v1.4 Expression Reports also revealed expected Call percentages (Present: 48% to 62%; Absent: 36% to 49%; Marginal: 1.5% to 1.9%). All arrays were within 1.5 fold of each other in overall intensity.</p>


<h4>miRNA expression profile</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Using the same 12 tumor samples from which the microarray data was obtained, miRNA profiling was performed by real-time multiplex RT-PCR <a href="#pone.0000804-Lao1">[5]</a>. Briefly, multiplexed reactions were performed with 330 sets of reverse transcriptase and second strand synthesis primers followed by individual singleplex TaqMan reaction in four 96-well reaction plates to measure the abundance of each of the 330 miRNAs after the multiplexed RT-PCR. Each primer and probe contained zip-coded sequences specifically assigned to each miRNA to increase the specificity of each reaction so that even small sequence differences in miRNA were amplified and detected. This approach is reliable as miRNA measurements on RT-PCR and microarray are concordant <a href="#pone.0000804-Mattie1">[21]</a>.</p>


<h4>Cell Culture and Reagents</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>U251 and U87 glioblastoma cells were plated out in DMEM containing 10% DMEM, 100 units/ml penicillin and streptomycin. Cells were cultured in a humidified incubator with 5% CO<sub>2</sub> at 37°C. Transfections were performed using siPORT NeoFX (Ambion, Austin, TX). Transfection complexes were prepared according to the manufacturer's instructions and 1×10<sup>5</sup>/ml cells were added directly to the transfection complexes on the plate to a final pre-mir concentration of 50 nmol/L. The transfection medium was replaced 8 hours post-transfection. Three independent transfections were performed for each pre-mir and 2′<em>-O-</em>Me oligoncleotide of <em>miR-181c, miR-182, miR-19a</em> and a scrambled pre-miR or 2′<em>-O-</em>Me as a negative control (Ambion, TX).</p>


<h4>Real-Time PCR analyses of mRNAs</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Total RNA was harvested 24 h after the transfection of U251 cells with the pre-mir by using the mirVana kit (Ambion) and 1.2 µg of total RNA was converted to cDNA by Superscript II first strand synthesis kit (Invitrogen, Carlsbad, CA), both according to manufacturers protocol. SYBR green real time PCR was performed by using the 7500 real-time PCR system (Applied Biosystems, Foster City, CA), POWER SYBR green master mix (Applied Biosystems) and gene specific primers for <em>PCAF, ASCC1, FADS2, HPS1, ANXA4, GSTO1, IFITM1, DDX11, TTC3, PRR13, ENTPD1, NCF2</em> and <em>GAPDH</em> (Applied Biosystems, <a href="http://www.allgenes.com">http://www.allgenes.com</a>). All the real-time PCR data was normalized to <em>GAPDH</em>. Melting temperatures for individual amplicons were determined, and proper PCR product sizes were confirmed using agarose gel electrophoresis. PCR assays were run in triplicate, and expression data were averaged. To calculate the relative expression for each gene, the 2<sup>−ΔΔCt</sup> method was used.</p>


<h4>Western Blot</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Total protein was isolated from U251 and U87 transfected cells in mRIPA cell lysis buffer. Protein concentration was measured using the BCA assay kit (Pierce, Rockford, IL). The membrane was first probed with antibodies against <em>PCAF</em> (H-369) (Santa Cruz Biotechnology, Santa Cruz, CA), anti-<em>p53</em> (BD Biosciences, San Jose, CA) and then with anti-beta-actin antibody (Sigma-Aldrich St. Louis, MO). Secondary antibodies were labeled with either Alexa Fluor 680 (Invitrogen) or IRDye800 (Li-Cor Biosciences, Lincoln, NE). Signals were visualized using the Odyssey infrared imaging system (Li-Cor Biosciences).</p>


<h4>Cell Growth and Apoptosis Assay</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>U251 cells were transfected with 50 nM of pre-<em>miR-181c</em> (Ambion) and 50 nM of negative control scramble (Ambion) and seeded in 96-well plates at 10,000 cells/well. Caspase-3/7 Glo (Promega, Madison, WI) reagent was used to determine the Relative Apoptosis 24 hours post-transfection. Cell Titer Glo (Promega) reagent was used to determine relative cell growth 24 hours post-transfection. Relative Apoptosis/Cell growth was determined by comparing <em>miR-181c</em> with scramble treated cells. Results represent the means of three separate experiments.</p>


<h4>Data processing</h4>
<h5>mRNA data.</h5><a id="article1.body1.sec4.sec8.sec1.p1" name="article1.body1.sec4.sec8.sec1.p1"></a><p>We computed the perfect match minus mismatch model-based expression values and performed quantile normalization implemented in dChip <a href="#pone.0000804-Li1">[22]</a>. During normalization, we checked the single, array, and probe outliers such that these less reliable data were excluded. After normalization, we were able to compare the expression levels of all the genes in 12 samples. The presence-call was also determined for each probe set. This call indicates whether an expression level is reliable by labeling it as Present/Marginal/Absent through comparing the perfect match with the mis-match values for each probe set. Probe sets were linked to gene symbols using two annotation files, “HG_U133A_annot.csv” obtained from Affymetrix website and Ensembl (Release 43 February 2007) <a href="#pone.0000804-Hubbard1">[23]</a> through BioMart <a href="#pone.0000804-Durinck1">[24]</a>. Due to the overlap between transcripts, a single probe set might detect more than one gene. Alternatively, different probe sets might be mapped into the same gene because many of them were designed to detect different splice variants of the same gene. We found that probes for the same gene usually showed very high correlation among themselves. For example, in the filtered data mentioned below, there were 2,563 sets of probes that detect the splice variants of the same genes. Probe sets that detected potentially alternatively splice variants were highly correlated with over 80% with correlation coefficients over 0.5.</p>
<a id="article1.body1.sec4.sec8.sec1.p2" name="article1.body1.sec4.sec8.sec1.p2"></a><p>The HG-U133A microarray chip contains 22,283 probe sets, from which a subset was selected for correlation analysis. We set up three criteria in selecting the subset: 1) to exclude less reliable data, the presence-call of a probe set through out 12 tumor samples had to be all Present. 2) to ensure high variability that permits the investigation of correlation, we selected genes which the maximum intensity among 12 samples was at least two fold higher than the minimum one. 3) a probe set had to have a gene symbol such that the biological relevance can be checked further. The rationale behind these criteria were: 1) Because correlation was critical to the analysis, we sought to reduce the uncertainty in the data as much as possible. In fact, when the level in one or more samples was labeled “Absent”, the overall levels in all samples were usually relatively low. Because low levels were usually accompanied with relatively high fluctuations, we tended to filter such cases. 2) The absence of high variability for a miRNA across samples implied that the observed variation was more likelyto be noise, and the resulting correlation spurious. 3) a gene symbol was needed for the target analysis and the output of highly correlated miRNA-mRNA pairs. 12542, 2547, and 105 probe sets were filtered away by criteria 1,2, and 3, respectively, and we were left with 7,089 probe sets for correlation analysis, which correspond to 5,822 distinct genes.</p>

<h5>miRNA data.</h5><a id="article1.body1.sec4.sec8.sec2.p1" name="article1.body1.sec4.sec8.sec2.p1"></a><p>Multiplex RT-PCR data for miRNAs were normalized and reliability determined. The accuracy of measurement determinations with different amounts of miRNA samples demonstrated that a cut-off of 32 for the 30 ng sample was reliable <a href="#pone.0000804-Lao1">[5]</a>. Bias was clearly detected when comparing the levels of each miRNA from two samples before normalization (<a href="#pone.0000804.s003">Figure S3</a>). Without normalization, the fluctuation in overall concentration of miRNAs resulted in a strong artifactual auto-correlation among miRNAs themselves (<a href="#pone.0000804.s004">Figure S4</a>). Therefore, similar to the normalization of microarray data, quantile normalization on the multiplex RT-PCR results was performed. In brief, with N ( = 12 here) samples, each contains Ct values of m microRNAs, quantile normalization projects the N k<sup>th</sup>-tile's into the same value which is the average of the N k<sup>th</sup>-tile's (k = 1~m). After normalization, the Ct distributions of the N samples overlap exactly by definition.</p>
<a id="article1.body1.sec4.sec8.sec2.p2" name="article1.body1.sec4.sec8.sec2.p2"></a><p>In order to increase the confidence, we selected a subset of miRNAs in which all the Ct's in 12 samples were reliable, i.e., less than the cutoff 32, and obtained Ct's for 94 miRNAs in each sample. This subset was then used for quantile normalization. After normalization, we further excluded one miRNA such that the remaining miRNAs all had strong signals, i.e., the maximum Ct among 12 samples was greater than the minimum by at least 1, a definition similar to the criteria used for mRNA data processing. The Ct's of these 93 miRNAs were then used for correlation analysis.</p>



<h4>Correlation coefficient and the random case</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Before correlation analysis, we transformed the microarray intensity data into log scale of base 2 such that they were comparable with the Ct data of miRNAs. In addition, we converted a Ct value into 32-Ct such that the latter value was positively proportional to the log of copy number.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>The distribution of correlation coefficients of two normally distributed random variables was utilized <a href="#pone.0000804-Fisher1">[25]</a>. The derivation was recapped as follows: Given two profiles x<sub>i</sub> and y<sub>i</sub> with i = 1~N, where N is the number of data points (N = 12 in our case), the Pearson's correlation coefficient r is defined as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.e002&amp;representation=PNG" class="inline-graphic"></span><br>where dx<sub>i</sub> and dy<sub>i</sub> are the normalized profiles with mean equals zero and norm equals one (Σdx<sub>i</sub> = 0 and Σdx<sub>i</sub><sup>2</sup> = 1). We can view dx<sub>i</sub> and dy<sub>i</sub> as two vectors on a uni-sphere in high dimensional space, then r is simply the inner product of these two vectors. For example, in three dimensions, without loss of generality we set z-axis as the direction of one vector, the inner product is simply the angle between the two vectors, i.e., r = cosθ. The probability density of r can be derived through <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.e003&amp;representation=PNG" class="inline-graphic"></span><br>where A is the surface area in three dimensional space. For a random vector, the distribution on the surface should be uniform such that p(A) = 1/A = 1/4π, a constant. In n-dimensions, the probability density of θ is known as p(θ)∝sinθ<sup>(n−2)</sup>. Thus,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.e004&amp;representation=PNG" class="inline-graphic"></span><br>Since the vector is constrained to have its mean as zero, given N data points, the vectors stay in N-1 dimensions. Thus, n equals N-1 in the last equation; after normalization such that the probability adds up to one, we obtained the probability distribution as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000804.e005&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec9.p3" name="article1.body1.sec4.sec9.p3"></a><p>To confirm the probability distribution, we ran the following simulations. We used our miRNA/mRNA data and randomly generated a set of mRNA/miRNA data of the same size and compared the resulting distribution of correlations with the formula. All the 100 trials resulted in a distribution very close to the random case. In addition, we randomly permuted the sample order in the miRNA (or mRNA) profiles and repeated the analysis. The clear signals shown in figures of this work disappeared in all 100 such trials (data not shown).</p>


<h4>TargetScan prediction</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>Bulk data of the predicted mRNA targets of miRNAs in TargetScan (Release 3.1 November 2006) was downloaded from the website. We considered miRNAs in both conserved and non-conserved families (conservation across human, mouse, rat, dog, and sometimes chicken). The predicted targets only in human were extracted. Two binding sites were considered distinct if their seed region loci in the multiply aligned 3′UTR sequences were different. The 3′UTR length for each gene was determined from the 3′UTR sequence from TargetScan. We removed the masked bases and counted the remaining bases as the 3′UTR length.</p>


<h4>Estimation of slope in target percentage</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>We selected the correlation between −0.8 and 0.8 for slope estimation to avoid the large fluctuations due to small amount of miRNA-mRNA pairs outside the range. In <a href="#pone-0000804-g002">Figure2</a>, the slope of points inside this range was obtained by fitting the curve to a straight line.</p>


<h4>miRNA host genes and their loci</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>Host gene information was obtained from miRBase (Release 9.1 February 2007) <a href="#pone.0000804-GriffithsJones1">[26]</a>. Gene loci were obtained from Ensembl (Release 43 February 2007) <a href="#pone.0000804-Hubbard1">[23]</a> through BioMart <a href="#pone.0000804-Durinck1">[24]</a> using dataset <em>Homo sapiens</em> NCBI36. Each gene loci is a range of base pairs and the distance between two genes was defined as the head to tail distance except when a miRNA is in the intron of an mRNA.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000804"></div><a name="pone.0000804.s001" id="pone.0000804.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000804.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Relationship between the correlation of two miRNAs on the same chromosome strand and the distance separating the two miRNA loci (small points). The distances for all pairs of miRNAs were ranked, and the distance rank is used as the x-axis. For each distance rank, the physical distance on the chromosome strand in log-scale is shown (circles). To view the coarse-grained relationship, five correlations for each distance rank are averaged. For example, for distance rank 10, we averaged the five correlations of distance ranks 8 to 12.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000804.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.45 MB TIF)</p>
<a name="pone.0000804.s002" id="pone.0000804.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000804.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Ratio of number of correlation coefficients between our experiments and the random case with 12, 11, 10, 9, 8, and 7 samples.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000804.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.45 MB TIF)</p>
<a name="pone.0000804.s003" id="pone.0000804.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000804.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Scatter plots of the Ct values of two samples for all miRNAs. Two pairs of two samples were shown as an illustration.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000804.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.45 MB TIF)</p>
<a name="pone.0000804.s004" id="pone.0000804.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000804.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Distribution of correlation coefficients among miRNAs before and after normalization.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000804.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.45 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Thanks to Nickolaus Rajewsky for his insightful suggestions and to the unstinting help from members of the Kosik lab.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KK. Performed the experiments: TP SQ FS YL. Analyzed the data: BS FD TL KP WC LP. Contributed reagents/materials/analysis tools: SN HK KL. Wrote the paper: KK TL TP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000804-Nilsen1" id="pone.0000804-Nilsen1"></a>Nilsen TW (2007) Mechanisms of microRNA-mediated gene regulation in animal cells. Trends Genet..   <ul class="find" data-citedArticleID="983884" data-doi="10.1016/j.tig.2007.02.011"><li><a href="http://dx.doi.org/10.1016/j.tig.2007.02.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+microRNA-mediated+gene+regulation+in+animal+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+microRNA-mediated+gene+regulation+in+animal+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000804-Lu1" id="pone.0000804-Lu1"></a>Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al.  (2005) MicroRNA expression profiles classify human cancers. Nature  435: 834–838.  <ul class="find" data-citedArticleID="983874" data-doi="10.1038/nature03702"><li><a href="http://dx.doi.org/10.1038/nature03702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+expression+profiles+classify+human+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+expression+profiles+classify+human+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000804-Nutt1" id="pone.0000804-Nutt1"></a>Nutt CL, Mani DR, Betensky RA, Tamayo P, Cairncross JG, et al.  (2003) Gene expression-based classification of malignant gliomas correlates better with survival than histological classification. Cancer Res  63: 1602–1607.  <ul class="find" data-citedArticleID="983886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification.&amp;auth=&amp;atitle=Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression-based+classification+of+malignant+gliomas+correlates+better+with+survival+than+histological+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000804-Pomeroy1" id="pone.0000804-Pomeroy1"></a>Pomeroy SL, Tamayo P, Gaasenbeek M, Sturla LM, Angelo M, et al.  (2002) Prediction of central nervous system embryonal tumour outcome based on gene expression. Nature  415: 436–442.  <ul class="find" data-citedArticleID="983888" data-doi="10.1038/415436a"><li><a href="http://dx.doi.org/10.1038/415436a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+central+nervous+system+embryonal+tumour+outcome+based+on+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+central+nervous+system+embryonal+tumour+outcome+based+on+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000804-Lao1" id="pone.0000804-Lao1"></a>Lao K, Xu NL, Sun YA, Livak KJ, Straus NA (2007) Real time PCR profiling of 330 human mircoRNAs. Biotechnol J  2: 33–35.  <ul class="find" data-citedArticleID="983862" data-doi="10.1002/biot.200600119"><li><a href="http://dx.doi.org/10.1002/biot.200600119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Real+time+PCR+profiling+of+330+human+mircoRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Real+time+PCR+profiling+of+330+human+mircoRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000804-Lewis1" id="pone.0000804-Lewis1"></a>Lewis BP, Burge CB, Bartel DP (2005) Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are mircoRNA targets. Cell  120: 15–16.  <ul class="find" data-citedArticleID="983864" data-doi="10.1016/j.cell.2004.12.035"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.12.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+mircoRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+mircoRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000804-Massirer1" id="pone.0000804-Massirer1"></a>Massirer KB, Pasquinelli AE (2006) The evolving role of microRNAs in animal gene expression. Bioessays  28: 449–452.  <ul class="find" data-citedArticleID="983878" data-doi="10.1002/bies.20406"><li><a href="http://dx.doi.org/10.1002/bies.20406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolving+role+of+microRNAs+in+animal+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolving+role+of+microRNAs+in+animal+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000804-Baskerville1" id="pone.0000804-Baskerville1"></a>Baskerville S, Bartel DP (2005) Microarray profiling of microRNAs reveals frequent coexpression with neighboring miRNAs and host genes. RNA  11: 241–247.  <ul class="find" data-citedArticleID="983844" data-doi="10.1261/rna.7240905"><li><a href="http://dx.doi.org/10.1261/rna.7240905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+profiling+of+microRNAs+reveals+frequent+coexpression+with+neighboring+miRNAs+and+host+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+profiling+of+microRNAs+reveals+frequent+coexpression+with+neighboring+miRNAs+and+host+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000804-Imhof1" id="pone.0000804-Imhof1"></a>Imhof A, Yang XJ, Ogryzko VV, Nakatani Y, Wolffe AP, et al.  (1997) Acetylation of general transcription factors by histone acetyltransferases. Curr Biol  7: 689–692.  <ul class="find" data-citedArticleID="983860" data-doi="10.1016/s0960-9822(06)00296-x"><li><a href="http://dx.doi.org/10.1016/s0960-9822(06)00296-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acetylation+of+general+transcription+factors+by+histone+acetyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acetylation+of+general+transcription+factors+by+histone+acetyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000804-Liu1" id="pone.0000804-Liu1"></a>Liu L, Scolnick DM, Trievel RC, Zhang HB, Marmorstein R, et al.  (1999) p53 sites acetylated in vitro by PCAF and p300 are acetylated in vivo in response to DNA damage. Mol Cell Biol  19: 1202–1209.  <ul class="find" data-citedArticleID="983872"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=p53+sites+acetylated+in+vitro+by+PCAF+and+p300+are+acetylated+in+vivo+in+response+to+DNA+damage.&amp;auth=&amp;atitle=p53+sites+acetylated+in+vitro+by+PCAF+and+p300+are+acetylated+in+vivo+in+response+to+DNA+damage." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p53+sites+acetylated+in+vitro+by+PCAF+and+p300+are+acetylated+in+vivo+in+response+to+DNA+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p53+sites+acetylated+in+vitro+by+PCAF+and+p300+are+acetylated+in+vivo+in+response+to+DNA+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000804-Nguyen1" id="pone.0000804-Nguyen1"></a>Nguyen DX, Baglia LA, Huang SM, Baker CM, McCance DJ (2004) Acetylation regulates the differentiation-specific functions of the retinoblastoma protein. EMBOJ  23: 1609–1618.  <ul class="find" data-citedArticleID="983882" data-doi="10.1038/sj.emboj.7600176"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acetylation+regulates+the+differentiation-specific+functions+of+the+retinoblastoma+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acetylation+regulates+the+differentiation-specific+functions+of+the+retinoblastoma+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000804-Wei1" id="pone.0000804-Wei1"></a>Wei CL, Wu Q, Vega VB, Chiu KP, Ng P, et al.  (2004) A global map of p53 transcription-factor binding sites in the human genome. Cell  124: 207–219.  <ul class="find" data-citedArticleID="983894" data-doi="10.1016/j.cell.2005.10.043"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.10.043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+map+of+p53+transcription-factor+binding+sites+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+map+of+p53+transcription-factor+binding+sites+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000804-Watts1" id="pone.0000804-Watts1"></a>Watts GS, Oshiro MM, Junk DJ, Wozniak RJ, Watterson S, et al.  (2004) Theacetyltransferase p300/CBP-associated factor is a p53 target gene in breast tumor cells. Neoplasia  6: 187–194.  <ul class="find" data-citedArticleID="983892" data-doi="10.1593/neo.03292"><li><a href="http://dx.doi.org/10.1593/neo.03292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theacetyltransferase+p300%2FCBP-associated+factor+is+a+p53+target+gene+in+breast+tumor+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theacetyltransferase+p300%2FCBP-associated+factor+is+a+p53+target+gene+in+breast+tumor+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000804-Sood1" id="pone.0000804-Sood1"></a>Sood P, Krek A, Zavolan M, Macino G, Rajewsky N (2006) Cell-type-specific signatures of microRNAs on target mRNA expression. Proc Natl Acad Sci USA  103: 2746–2751.  <ul class="find" data-citedArticleID="983890" data-doi="10.1073/pnas.0511045103"><li><a href="http://dx.doi.org/10.1073/pnas.0511045103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-type-specific+signatures+of+microRNAs+on+target+mRNA+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-type-specific+signatures+of+microRNAs+on+target+mRNA+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000804-Lim1" id="pone.0000804-Lim1"></a>Lim LP, Lau NC, Garrett-Engele P, Grimson A, Schelter JM, et al.  (2005) Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs. Nature  433: 769–73.  <ul class="find" data-citedArticleID="983868" data-doi="10.1038/nature03315"><li><a href="http://dx.doi.org/10.1038/nature03315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+shows+that+some+microRNAs+downregulate+large+numbers+of+target+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+shows+that+some+microRNAs+downregulate+large+numbers+of+target+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000804-Martianov1" id="pone.0000804-Martianov1"></a>Martianov I, Ramadass A, Serra Barros A, Chow N, Akoulitchev A (2007) Regression of the human dihydrofolate reductase gene by a non-coding interfering transcript. Nature  445: 666–670.  <ul class="find" data-citedArticleID="983876" data-doi="10.1038/nature05519"><li><a href="http://dx.doi.org/10.1038/nature05519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regression+of+the+human+dihydrofolate+reductase+gene+by+a+non-coding+interfering+transcript." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regression+of+the+human+dihydrofolate+reductase+gene+by+a+non-coding+interfering+transcript.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000804-Linares1" id="pone.0000804-Linares1"></a>Linares LK, Kiernan R, Triboulet R, Chable-Bessia C, Latreille D, et al.  (2007) Intrinsic ubiquitination activity of PCAF controls the stability of the oncoprotein Hdm2. Nat Cell Biol  9: 331–338.  <ul class="find" data-citedArticleID="983870" data-doi="10.1038/ncb1545"><li><a href="http://dx.doi.org/10.1038/ncb1545" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intrinsic+ubiquitination+activity+of+PCAF+controls+the+stability+of+the+oncoprotein+Hdm2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intrinsic+ubiquitination+activity+of+PCAF+controls+the+stability+of+the+oncoprotein+Hdm2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000804-Ciafre1" id="pone.0000804-Ciafre1"></a>Ciafre SA, Galardi S, Mangiola A, Ferracin M, Liu CG, et al.  (2005) Extensive modulation of a set of microRNAs in primary glioblastoma. Biochem Biophys Res Commun  334: 1351–1358.  <ul class="find" data-citedArticleID="983848" data-doi="10.1016/j.bbrc.2005.07.030"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.07.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+modulation+of+a+set+of+microRNAs+in+primary+glioblastoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+modulation+of+a+set+of+microRNAs+in+primary+glioblastoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000804-Cheng1" id="pone.0000804-Cheng1"></a>Cheng AM, Byrom MW, Shelton J, Ford LP (2005) Antisense inhibition of human miRNAs and indications for an involvement of miRNA in cell growth and apoptosis. Nucleic Acids Res  33: 1290–1297.  <ul class="find" data-citedArticleID="983846" data-doi="10.1093/nar/gki200"><li><a href="http://dx.doi.org/10.1093/nar/gki200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+inhibition+of+human+miRNAs+and+indications+for+an+involvement+of+miRNA+in+cell+growth+and+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+inhibition+of+human+miRNAs+and+indications+for+an+involvement+of+miRNA+in+cell+growth+and+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000804-Freije1" id="pone.0000804-Freije1"></a>Freije WA, Castro-Vargas FE, Fang Z, Horvath S, Cloughesy T, et al.  (2004) Gene expression profiling of gliomas strongly predicts survival. Cancer Res  64: 6503–6510.  <ul class="find" data-citedArticleID="983854" data-doi="10.1158/0008-5472.can-04-0452"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-04-0452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+of+gliomas+strongly+predicts+survival." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+of+gliomas+strongly+predicts+survival.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000804-Mattie1" id="pone.0000804-Mattie1"></a>Mattie MD, Benz CC, Bowers J, Sensinger K, Wong L, et al.  (2006) Optimized high-throughput microRNA expression profiling provides novel biomarker assessment of clinical prostate and breast cancer biopsies. Mol Cancer  5: 24.  <ul class="find" data-citedArticleID="983880"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Optimized+high-throughput+microRNA+expression+profiling+provides+novel+biomarker+assessment+of+clinical+prostate+and+breast+cancer+biopsies.&amp;auth=&amp;atitle=Optimized+high-throughput+microRNA+expression+profiling+provides+novel+biomarker+assessment+of+clinical+prostate+and+breast+cancer+biopsies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimized+high-throughput+microRNA+expression+profiling+provides+novel+biomarker+assessment+of+clinical+prostate+and+breast+cancer+biopsies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimized+high-throughput+microRNA+expression+profiling+provides+novel+biomarker+assessment+of+clinical+prostate+and+breast+cancer+biopsies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000804-Li1" id="pone.0000804-Li1"></a>Li C, Wong WH (2001) Model-based analysis of oligonucleotide arrays: Expression index computation and outlier detection. Proc Natl Acad SciUSA  98: 31–36.  <ul class="find" data-citedArticleID="983866" data-doi="10.1073/pnas.98.1.31"><li><a href="http://dx.doi.org/10.1073/pnas.98.1.31" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model-based+analysis+of+oligonucleotide+arrays%3A+Expression+index+computation+and+outlier+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model-based+analysis+of+oligonucleotide+arrays%3A+Expression+index+computation+and+outlier+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000804-Hubbard1" id="pone.0000804-Hubbard1"></a>Hubbard TJ, Aken BL, Beal K, Ballester B, Caccamo M (2007) Ensembl 2007. Nucleic Acids Res  35: D610–617.  <ul class="find" data-citedArticleID="983858" data-doi="10.1093/nar/gkl996"><li><a href="http://dx.doi.org/10.1093/nar/gkl996" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2007." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2007.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000804-Durinck1" id="pone.0000804-Durinck1"></a>Durinck S, Moreau Y, Kasprzyk A, Daivis S, De Moor B, et al.  (2005) Biomart and bioconductor: a powerful link between biological databases and microarry data analysis. Bioinformatics  21: 3439–3440.  <ul class="find" data-citedArticleID="983850" data-doi="10.1093/bioinformatics/bti525"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biomart+and+bioconductor%3A+a+powerful+link+between+biological+databases+and+microarry+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biomart+and+bioconductor%3A+a+powerful+link+between+biological+databases+and+microarry+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000804-Fisher1" id="pone.0000804-Fisher1"></a>Fisher RA (1922) On the mathematical foundations of theoretical statistics. Phil Trans Loyal Soc  A222: 309–368.  <ul class="find" data-citedArticleID="983852"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+mathematical+foundations+of+theoretical+statistics.&amp;auth=&amp;atitle=On+the+mathematical+foundations+of+theoretical+statistics." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+mathematical+foundations+of+theoretical+statistics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+mathematical+foundations+of+theoretical+statistics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000804-GriffithsJones1" id="pone.0000804-GriffithsJones1"></a>Griffiths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ (2006) miRBase: microRNA sequences, targets, and gene nomenclature. Nucleic Acids Res  34: D140–D144.  <ul class="find" data-citedArticleID="983856" data-doi="10.1093/nar/gkj112"><li><a href="http://dx.doi.org/10.1093/nar/gkj112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRBase%3A+microRNA+sequences%2C+targets%2C+and+gene+nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRBase%3A+microRNA+sequences%2C+targets%2C+and+gene+nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.XML" value="114737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.PDF" value="663249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g001.PNG_L" value="62925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g001.PNG_M" value="73010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g001.PNG_S" value="6816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g001.TIF" value="104122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g001.PNG_I" value="25060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e001.PNG" value="12456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e001.TIF" value="29024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t001.PNG_L" value="285581"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t001.PNG_M" value="157551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t001.PNG_S" value="16665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t001.TIF" value="1147308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t001.PNG_I" value="47444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g002.PNG_L" value="111714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g002.PNG_M" value="110503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g002.PNG_S" value="9338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g002.TIF" value="163912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g002.PNG_I" value="37919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g003.PNG_L" value="87342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g003.PNG_M" value="87447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g003.PNG_S" value="8728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g003.TIF" value="144608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g003.PNG_I" value="30512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t002.PNG_L" value="80802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t002.PNG_M" value="98455"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t002.PNG_S" value="14434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t002.TIF" value="418772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t002.PNG_I" value="32285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g004.PNG_L" value="111147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g004.PNG_M" value="107742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g004.PNG_S" value="10380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g004.TIF" value="242490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g004.PNG_I" value="36654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g005.PNG_L" value="8845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g005.PNG_M" value="51058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g005.PNG_S" value="7512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g005.TIF" value="52088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g005.PNG_I" value="18972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t003.PNG_L" value="109698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t003.PNG_M" value="144148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t003.PNG_S" value="17251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t003.TIF" value="514306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.t003.PNG_I" value="50519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g006.PNG_L" value="205266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g006.PNG_M" value="64781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g006.PNG_S" value="9070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g006.TIF" value="691644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g006.PNG_I" value="25221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g007.PNG_L" value="81483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g007.PNG_M" value="26344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g007.PNG_S" value="6513"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g007.TIF" value="318712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g007.PNG_I" value="10182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g008.PNG_L" value="204055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g008.PNG_M" value="112205"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g008.PNG_S" value="8411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g008.TIF" value="341370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g008.PNG_I" value="55459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g009.PNG_L" value="24715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g009.PNG_M" value="68128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g009.PNG_S" value="7567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g009.TIF" value="76152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.g009.PNG_I" value="25576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e002.PNG" value="15607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e002.TIF" value="31140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e003.PNG" value="11069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e003.TIF" value="27292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e004.PNG" value="10524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e004.TIF" value="26420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e005.PNG" value="11587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.e005.TIF" value="27428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.s001.TIF" value="487213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.s002.TIF" value="1457353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.s003.TIF" value="486795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000804.s004.TIF" value="1457313"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000804&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000804" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000804&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000804&volume=&issue=&title=Detection of a MicroRNA Signal in an In Vivo Expression Set of mRNAs&author_name=Tsunglin%20Liu%2C%20Thales%20Papagiannakopoulos%2C%20Kathy%20Puskar%2C%20Shuping%20Qi%2C%20Fernando%20Santiago%2C%20William%20Clay%2C%20Kaiqin%20Lao%2C%20Yohan%20Lee%2C%20Stanley%20F.%20Nelson%2C%20Harley%20I.%20Kornblum%2C%20Frank%20Doyle%2C%20Linda%20Petzold%2C%20Boris%20Shraiman%2C%20Kenneth%20S.%20Kosik&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000804" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000804" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000804" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000804&amp;t=Detection%20of%20a%20MicroRNA%20Signal%20in%20an%20In%20Vivo%20Expression%20Set%20of%20mRNAs" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000804&title=Detection%20of%20a%20MicroRNA%20Signal%20in%20an%20In%20Vivo%20Expression%20Set%20of%20mRNAs&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000804&amp;title=Detection%20of%20a%20MicroRNA%20Signal%20in%20an%20In%20Vivo%20Expression%20Set%20of%20mRNAs" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000804" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000804&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Detection of a MicroRNA Signal in an In Vivo Expression Set of mRNAs';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000804';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Detection%20of%20a%20MicroRNA%20Signal%20in%20an%20In%20Vivo%20Expression%20Set%20of%20mRNAs http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000804" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000804" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Apoptosis%22" title="Search for articles in the subject area:'Apoptosis'"><div class="flagText">Apoptosis</div></a>
              <div data-categoryid="17189" data-articleid="24446"
                   data-categoryname="Apoptosis"
                   class="flagImage" title="Flag 'Apoptosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="47601" data-articleid="24446"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glioblastoma+multiforme%22" title="Search for articles in the subject area:'Glioblastoma multiforme'"><div class="flagText">Glioblastoma multiforme</div></a>
              <div data-categoryid="36741" data-articleid="24446"
                   data-categoryname="Glioblastoma multiforme"
                   class="flagImage" title="Flag 'Glioblastoma multiforme' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="24446"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22MicroRNA%22" title="Search for articles in the subject area:'MicroRNA'"><div class="flagText">MicroRNA</div></a>
              <div data-categoryid="19821" data-articleid="24446"
                   data-categoryname="MicroRNA"
                   class="flagImage" title="Flag 'MicroRNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24446"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="24446"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transfection%22" title="Search for articles in the subject area:'Transfection'"><div class="flagText">Transfection</div></a>
              <div data-categoryid="47399" data-articleid="24446"
                   data-categoryname="Transfection"
                   class="flagImage" title="Flag 'Transfection' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8699'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7298'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2469&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10041">Referee Comments: Referee 1 (Elia Stupka)</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=5433">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
