

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000776"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000776" />

    <meta name="citation_title" content="Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions"/>
    <meta itemprop="name" content="Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions"/>

      <meta name="citation_author" content="Zhen Xuan Yeo"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Sum Thai Wong"/>
            <meta name="citation_author_institution" content="Bioinformatics Institute, Singapore, Singapore"/>
      <meta name="citation_author" content="Satya Nanda Vel Arjunan"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
      <meta name="citation_author" content="Vincent Piras"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
      <meta name="citation_author" content="Masaru Tomita"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
      <meta name="citation_author" content="Kumar Selvarajoo"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>
      <meta name="citation_author" content="Alessandro Giuliani"/>
            <meta name="citation_author_institution" content="Department of Environment and Health, Instituto Superiore di Sanita', Roma, Italy"/>
      <meta name="citation_author" content="Masa Tsuchiya"/>
            <meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan"/>

    <meta name="citation_date" content="2007/8/22"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000776.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e776"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Transcriptional regulation by the numbers: applications.; citation_author=L Bintu; citation_author=NE Buchler; citation_author=HG Garcia; citation_author=U Gerland; citation_author=T Hwa; citation_journal_title=Curr Opin Genet Dev; citation_volume=15; citation_number=1; citation_pages=125-135; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulation by the numbers: models.; citation_author=L Bintu; citation_author=NE Buchler; citation_author=HG Garcia; citation_author=U Gerland; citation_author=T Hwa; citation_journal_title=Curr Opin Genet Dev; citation_volume=15; citation_number=2; citation_pages=116-124; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the relationship between genomic regulatory element organization and gene regulatory dynamics.; citation_author=DM Wolf; citation_author=FH Eeckman; citation_journal_title=J Theor Biol; citation_volume=195; citation_number=3; citation_pages=167-186; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A synthetic oscillatory network of transcriptional regulators.; citation_author=MB Elowitz; citation_author=S Leibler; citation_journal_title=Nature; citation_volume=403; citation_number=4; citation_pages=335-338; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Computational studies of gene regulatory networks: in numero molecular biology.; citation_author=J Hasty; citation_author=D McMillen; citation_author=F Isaacs; citation_author=JJ Collins; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=5; citation_pages=268-279; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The evolution of transcriptional regulation in eukaryotes.; citation_author=GA Wray; citation_author=MW Hahn; citation_author=E Abouheif; citation_author=JP Balhoff; citation_author=M Pizer; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=6; citation_pages=1377-1419; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory code of a eukaryotic genome.; citation_author=CT Harbison; citation_author=DB Gordon; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=KD Macisaac; citation_journal_title=Nature; citation_volume=431; citation_number=7; citation_pages=99-104; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Detailed map of a cis-regulatory input function.; citation_author=Y Setty; citation_author=AE Mayo; citation_author=MG Surette; citation_author=U Alon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=8; citation_pages=7702-7707; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomic cis-regulatory logic: experimental and computational analysis of a sea urchin gene.; citation_author=CH Yuh; citation_author=H Bolouri; citation_author=EH Davidson; citation_journal_title=Science; citation_volume=279; citation_number=9; citation_pages=1896-1902; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cis-regulatory logic in the endo16 gene: switching from a specification to a differentiation mode of control.; citation_author=CH Yuh; citation_author=H Bolouri; citation_author=EH Davidson; citation_journal_title=Development; citation_volume=128; citation_number=10; citation_pages=617-629; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A genomic regulatory network for development.; citation_author=EH Davidson; citation_author=JP Rast; citation_author=P Oliveri; citation_author=A Ransick; citation_author=C Calestani; citation_journal_title=Science; citation_volume=295; citation_number=11; citation_pages=1669-1678; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Modelling DNA sequence-based cis-regulatory gene networks.; citation_author=H Bolouri; citation_author=EH Davidson; citation_journal_title=Dev Biol; citation_volume=246; citation_number=12; citation_pages=2-13; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=On schemes of combinatorial transcription logic.; citation_author=NE Buchler; citation_author=U Gerland; citation_author=T Hwa; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=13; citation_pages=5136-5141; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Control of gene expression during T cell activation: alternate regulation of mRNA transcription and mRNA stability.; citation_author=C Cheadle; citation_author=J Fan; citation_author=YS Cho-Chung; citation_author=T Werner; citation_author=J Ray; citation_journal_title=BMC Genomics; citation_volume=6(1); citation_number=14; citation_pages=75; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Extensive Association of Functionally and Cytotopically Related mRNAs with Puf Family RNA-Binding Proteins in Yeast.; citation_author=AP Gerber; citation_author=D Herschlag; citation_author=PO Brown; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=15; citation_pages=E79; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomic regulatory networks and animal development.; citation_author=A Stathopoulos; citation_author=M Levine; citation_journal_title=Dev Cell; citation_volume=9; citation_number=16; citation_pages=449-462; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Filtering transcriptional noise during development: concepts and mechanisms.; citation_author=AM Arias; citation_author=P Hayward; citation_journal_title=Nature Rev Genet; citation_volume=7; citation_number=17; citation_pages=34-44; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Brn1/2/4, the predicted midgut regulator of the endo16 gene of the sea urchin embryo.; citation_author=CH Yuh; citation_author=ER Dorman; citation_author=EH Davidson; citation_journal_title=Dev Biol; citation_volume=281; citation_number=18; citation_pages=286-298; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An otx cis-regulatory module: a key node in the sea urchin endomesoderm gene regulatory network.; citation_author=CH Yuh; citation_author=ER Dorman; citation_author=ML Howard; citation_author=EH Davidson; citation_journal_title=Dev Biol; citation_volume=269; citation_number=19; citation_pages=536-551; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recursive network approach can identify constitutive regulatory circuits in gene expression data.; citation_author=MF Blasi; citation_author=I Casorellia; citation_author=A Colosimob; citation_author=FS Blasic; citation_author=M Bignamia; citation_journal_title=Physica A; citation_volume=348; citation_number=20; citation_pages=349-370; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A quantization method based on threshold optimization for microarray short time series.; citation_author=B Di Camillo; citation_author=F Sanchez-Cabo; citation_author=G Toffolo; citation_author=SK Nair; citation_author=Z Trajanoski; citation_journal_title=BMC Bioinformatics; citation_volume=1; citation_number=21; citation_pages=S11; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The global transcriptional regulatory network for metabolism in Escherichia coli exhibits few dominant functional states.; citation_author=CL Barrett; citation_author=CD Herring; citation_author=JL Reed; citation_author=BO Palsson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=22; citation_pages=19103-19108; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory cascades in development: initial rates, not steady state, determine network kinetics.; citation_author=H Bolouri; citation_author=EH Davidson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=23; citation_pages=9371-9376; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Logic functions of the genomic cis-regulatory code.; citation_author=S Istrail; citation_author=EH Davidson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=24; citation_pages=4954-4959; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nucleotides of transcription factor binding sites exert interdependent effects on the binding affinities of transcription factors.; citation_author=ML Bulyk; citation_author=PLF Johnson; citation_author=GM Church; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=25; citation_pages=1255-1261; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hysteresis in a synthetic mammalian gene network.; citation_author=BP Kramer; citation_author=M Fussenegger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=26; citation_pages=9517-9522; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detection of multistability, bifurcations, and hysteresis in a large class of biological positive-feedback systems.; citation_author=D Angeli; citation_author=JE Ferrell; citation_author=ED Sontag; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=27; citation_pages=1822-1827; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A Markovian formalization of heart rate dynamics evinces a quantum-like hypothesis.; citation_author=A Giuliani; citation_author=P Lo Giudice; citation_author=AM Mancini; citation_author=G Quatrini; citation_author=L Pacifici; citation_author=CL Webber; citation_author=M Zak; citation_author=JP Zbilut; citation_journal_title=Biol Cybern; citation_volume=74; citation_number=28; citation_pages=181-187; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Boolean dynamics of genetic regulatory networks inferred from microarray time series data.; citation_author=S Martin; citation_author=Z Zhang; citation_author=A Martino; citation_author=JL Faulon; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=29; citation_pages=866-74; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A biophysical approach to transcription factor binding site discovery.; citation_author=M Djordjevic; citation_author=AM Sengupta; citation_author=BI Shraiman; citation_journal_title=Genome Res; citation_volume=13; citation_number=30; citation_pages=2381-2390; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computational identification of cis-regulatory elements associated with groups of functionally related genes in Saccharomyces cerevisiae.; citation_author=JD Hughes; citation_author=PW Estep; citation_author=S Tavazoie; citation_author=GM Church; citation_journal_title=J Mol Biol; citation_volume=296; citation_number=31; citation_pages=1205-14; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Estimating gene networks from gene expression data by combining Bayesian network model with promoter element detection.; citation_author=Y Tamada; citation_author=S Kim; citation_author=H Bannai; citation_author=S Imoto; citation_author=K Tashiro; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=32; citation_pages=II227-II236; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Reconstruction of Gene Regulatory Networks under the Finite State Linear Model.; citation_author=D Ruklisa; citation_author=A Brazma; citation_author=J Viksna; citation_journal_title=Genome Informatics; citation_volume=16; citation_number=33; citation_pages=225-236; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome Biology Reverse engineering of gene regulatory networks: a finite state linear model.; citation_author=A Brazma; citation_author=T Schlitt; citation_number=34; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=MM Mano; citation_number=35; citation_pages=516; citation_date=2001; citation_publisher=Prentice Hall PTR; " />
      <meta name="citation_reference" content="citation_title=Network motifs: simple building blocks of complex networks.; citation_author=R Milo; citation_author=S Shen-Orr; citation_author=S Itzkovitz; citation_author=N Kashtan; citation_author=D Chklovskii; citation_journal_title=Science; citation_volume=298; citation_number=36; citation_pages=824-827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Dynamic properties of network motifs contribute to biological network organization.; citation_author=RJ Prill; citation_author=PA Iglesias; citation_author=A Levchenko; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=37; citation_pages=1881-1892; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Network motifs in integrated cellular networks of transcription–regulation and protein–protein interaction.; citation_author=E Yeger-Lotem; citation_author=S Sattath; citation_author=N Kashtan; citation_author=S Itzkovitz; citation_author=R Milo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=38; citation_pages=5934-5939; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modular cis-regulatory organization of Endo16, a gut-specific gene of the sea urchin embryo.; citation_author=CH Yuh; citation_author=EH Davidson; citation_journal_title=Development; citation_volume=122; citation_number=39; citation_pages=1069-1082; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Quantitative functional interrelations within the cis-regulatory system of the S. purpuratus Endo16 gene.; citation_author=CH Yuh; citation_author=JG Moore; citation_author=EH Davidson; citation_journal_title=Development; citation_volume=122; citation_number=40; citation_pages=4045-4056; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000776" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions"/>
    <meta name="twitter:description" content="BackgroundCellular signaling involves a sequence of events from ligand binding to membrane receptors through transcription factors activation and the induction of mRNA expression. The transcriptional-regulatory system plays a pivotal role in the control of gene expression. A novel computational approach to the study of gene regulation circuits is presented here.MethodologyBased on the concept of finite state machine, which provides a discrete view of gene regulation, a novel sequential logic model (SLM) is developed to decipher control mechanisms of dynamic transcriptional regulation of gene expressions. The SLM technique is also used to systematically analyze the dynamic function of transcriptional inputs, the dependency and cooperativity, such as synergy effect, among the binding sites with respect to when, how much and how fast the gene of interest is expressed.Principal FindingsSLM is verified by a set of well studied expression data on endo16 of Strongylocentrotus purpuratus (sea urchin) during the embryonic midgut development. A dynamic regulatory mechanism for endo16 expression controlled by three binding sites, UI, R and Otx is identified and demonstrated to be consistent with experimental findings. Furthermore, we show that during transition from specification to differentiation in wild type endo16 expression profile, SLM reveals three binary activities are not sufficient to explain the transcriptional regulation of endo16 expression and additional activities of binding sites are required. Further analyses suggest detailed mechanism of R switch activity where indirect dependency occurs in between UI activity and R switch during specification to differentiation stage.Conclusions/SignificanceThe sequential logic formalism allows for a simplification of regulation network dynamics going from a continuous to a discrete representation of gene activation in time. In effect our SLM is non-parametric and model-independent, yet providing rich biological insight. The demonstration of the efficacy of this approach in endo16 is a promising step for further application of the proposed method."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000776.g009"/>

  <meta property="og:title" content="Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000776" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2010'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7251'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3189&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000776">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000776" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000776&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhen Xuan Yeo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sum Thai Wong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioinformatics Institute, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Satya Nanda Vel Arjunan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vincent Piras, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Masaru Tomita, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kumar Selvarajoo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alessandro Giuliani, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Environment and Health, Instituto Superiore di Sanita', Roma, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Masa Tsuchiya
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:tsuchiya@ttck.keio.ac.jp">tsuchiya@ttck.keio.ac.jp</a></p>

                <p>Affiliation:
                  Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 22, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000776</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000776" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000776" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000776" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000776" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000776-g001" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t001" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g002" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t002" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t003" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g003" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t004" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t005" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g004" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g005" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g006" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t006" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t007" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t008" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t008" title="Table 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g007" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g008" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-g009" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000776-t009" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t009" title="Table 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000776">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000776" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Cellular signaling involves a sequence of events from ligand binding to membrane receptors through transcription factors activation and the induction of mRNA expression. The transcriptional-regulatory system plays a pivotal role in the control of gene expression. A novel computational approach to the study of gene regulation circuits is presented here.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Based on the concept of finite state machine, which provides a discrete view of gene regulation, a novel sequential logic model (SLM) is developed to decipher control mechanisms of dynamic transcriptional regulation of gene expressions. The SLM technique is also used to systematically analyze the dynamic function of transcriptional inputs, the dependency and cooperativity, such as synergy effect, among the binding sites with respect to when, how much and how fast the gene of interest is expressed.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>SLM is verified by a set of well studied expression data on <em>endo16</em> of Strongylocentrotus purpuratus (sea urchin) during the embryonic midgut development. A dynamic regulatory mechanism for <em>endo16</em> expression controlled by three binding sites, UI, R and Otx is identified and demonstrated to be consistent with experimental findings. Furthermore, we show that during transition from specification to differentiation in wild type <em>endo16</em> expression profile, SLM reveals three binary activities are not sufficient to explain the transcriptional regulation of <em>endo16</em> expression and additional activities of binding sites are required. Further analyses suggest detailed mechanism of R switch activity where indirect dependency occurs in between UI activity and R switch during specification to differentiation stage.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec4.p1" name="article1.front1.article-meta1.abstract1.sec4.p1"></a><p>The sequential logic formalism allows for a simplification of regulation network dynamics going from a continuous to a discrete representation of gene activation in time. In effect our SLM is non-parametric and model-independent, yet providing rich biological insight. The demonstration of the efficacy of this approach in <em>endo16</em> is a promising step for further application of the proposed method.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Yeo ZX, Wong ST, Arjunan SNV, Piras V, Tomita M, et al.  (2007) Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions. PLoS ONE 2(8):
          e776.
            doi:10.1371/journal.pone.0000776</p><p><strong>Academic Editor: </strong>Andrea Califano, Columbia University, United States of America</p><p><strong>Received:</strong> March 12, 2007; <strong>Accepted:</strong> August 2, 2007; <strong>Published:</strong> August 22, 2007</p><p><strong>Copyright:</strong> © 2007 Yeo et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Agency for Science, Technology and Research of Singapore (A*STAR), Japan Science and Technology Agency/Core Research for Evolutional Science and Technology (JST CREST), Ministry of Education, Culture, Sports, Science and Technology of Japan (MEXT), and Research fund by Yamagata Prefecture and Tsuruoka City, Japan.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Understanding of dynamic control of gene regulatory networks is a prime challenge in molecular biology. As gene regulatory network is underpinned by dynamical interaction of transcriptional-regulatory systems through transcriptional activation, transcriptional-regulatory system can be considered as an elementary component of gene regulatory networks <a href="#pone.0000776-Bintu1">[1]</a>, <a href="#pone.0000776-Bintu2">[2]</a>, <a href="#pone.0000776-Wolf1">[3]</a>, <a href="#pone.0000776-Elowitz1">[4]</a>, <a href="#pone.0000776-Hasty1">[5]</a>. Moreover, substantial evidence supports that evolution and regulation of transcriptional-regulatory systems are major contributing factor on the variation and selection of biological phenotypes <a href="#pone.0000776-Wray1">[6]</a>. A transcriptional-regulatory system is based on the presence of transcription factor binding sites of genes which are responsible for receiving temporal regulatory input signals, integrating these signals and producing output in terms of gene expression <a href="#pone.0000776-Harbison1">[7]</a>. The relationship between regulatory input signals and gene expression profile is a complex mapping <a href="#pone.0000776-Setty1">[8]</a>, <a href="#pone.0000776-Yuh1">[9]</a>, <a href="#pone.0000776-Yuh2">[10]</a>, <a href="#pone.0000776-Davidson1">[11]</a>, <a href="#pone.0000776-Bolouri1">[12]</a> and combinatorial regulatory inputs add further complexity to the entire framework <a href="#pone.0000776-Setty1">[8]</a>, <a href="#pone.0000776-Yuh2">[10]</a>, <a href="#pone.0000776-Buchler1">[13]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>To decipher the dynamic regulation mechanism of a transcriptional-regulatory system, a sequential logic model (SLM) is used to demonstrate the existence of dynamical logical mapping between trans-activation and temporal mRNA expression profiles. Our efforts are driven by the following goals: 1. Generalization of dynamical transcription for control and prediction of gene expression at mRNA level using SLM. 2. Identify a formalism that allows the extraction of the functional information that is associated with transcriptional-regulatory components that control mRNA expression (e.g. dynamic-function of <em>cis</em>-acting sites and dynamic-dependency among the sites including cooperative effects).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>On the basis of this SLM, both <em>characteristic equation analysis</em> and <em>time-simulation analysis</em> (<a href="#s4">Methods</a>) have been developed to investigate the dynamics of transcriptional-regulation circuits. Characteristic equation is employed for systematically extracting the dynamic function of <em>cis</em>-acting sites and their relationship in regulating gene expression. Time-simulation analysis is performed to simulate gene expression profile in mutagenesis analysis (<em>in silico</em> mutagenesis), to predict novel gene expression profiles under different activity of <em>cis</em>-acting sites (forward mapping) and to identify specific binding activity when a particular expression profile is given (reverse mapping). SLM is part of the logical model family and has a long and established tradition in engineering and systems analysis; it is distinctive from classical Boolean model by its capability for modeling of non-binary expression levels as well as for consideration of dynamic behaviour. The state transition from present to next state at a given time interval is expressed as an AND logic for activation of <em>cis</em>-acting sites and present state conditions. Therefore, expression of AND logic terms at a given time interval (<em>characteristic equation</em>) manifests <em>when</em> the effect of activation of <em>cis</em>-acting sites, such as enhancer and silencer, has occurred as well as logical dependence among <em>cis</em>-acting sites. Construction and analysis of the SLM only requires two input information: temporal gene expression profiles and activity of <em>cis</em>-acting sites. Therefore, SLM is non-parametric and model-independent, yet providing rich biological insight. Moreover it can in principle incorporate other elements of gene regulation different from TF like post-transcriptional regulation by means of differential mRNA stability <a href="#pone.0000776-Cheadle1">[14]</a>, <a href="#pone.0000776-Gerber1">[15]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Our model is verified using a set of well studied expression data on <em>endo16</em>, a marker for sea urchin gut development, provided by Davidson's group <a href="#pone.0000776-Yuh1">[9]</a>, <a href="#pone.0000776-Yuh2">[10]</a>. The choice of an embryonic development gene regulation circuit allows us to circumvent all the problems linked to the vagaries of gene expression noise and modulating signals: in fact, it is well known that developing embryo filters out the inherent genetic noise in order to follow a specific temporal development scheme <a href="#pone.0000776-Stathopoulos1">[16]</a>, <a href="#pone.0000776-Arias1">[17]</a>, which makes it possible to treat the gene regulation as a deterministic logic machine. In our verification model, three regulatory sites: Otx, R and UI are selected due to their significant functions in controlling the onset as well as transition of specification and differentiation for the gut development of the species. Our model allows to account for the entire <em>endo16</em> gene expression dynamics as controlled by Otx, R and UI <em>cis</em>-acting sites during transition from specification of endomesoderm to differentiation of midgut in sea urchin. For instance, our <em>endo16</em> SLM clearly reveals <em>when</em> repressive effect of R on the Otx <em>cis</em>-acting site occurs in order to prepare for the transition from specification to differentiation of the mid gut development. In addition, our model has the utility to demonstrate that the activation of binding site of R and Otx is distinct from the resultant functional activation of <em>endo16</em> expression (e.g. <a href="#pone-0000776-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0000776-g001"><div class="img"><a name="pone-0000776-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Identify time period required for preparation from specification to differentiation during midgut development.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p><em>Left</em>: the characteristic analysis at 29–40 pfh and 40–50 pfh. The characteristic equation (<a href="#pone-0000776-t001">Table 1</a>) shows when repressive effect of R and Otx together occurs; <em>Right</em>: 29–40 pfh(red bar) and 40–50 pfh (blue bar) are corresponding time periods in the control profile. Given that UI, R and Otx are activated at 29–40 pfh, only UI drives the state transition from 01<sub>2</sub> state: R and Otx are independent of the state transition. However, at 40–50 pfh (UI, R and Otx remain activated), the characteristic equation consists of the AND logic (<em>ROtx</em>)' which indicates that <em>(ROtx)'</em> is repressive: activation of <em>R</em> and <em>Otx</em> cooperatively prevent the state rises from 10<sub>2</sub> to 11<sub>2</sub> since (<em>ROtx</em>)' generates <em>q<sub>0,t+1</sub></em> = 0 when <em>R</em> = 1 and <em>Otx</em> = 1. Further analysis of Otx mutation (<a href="#pone-0000776-t003">Table 3</a>) shows R has no effect on any state transition suggesting that R is a silencer to Otx at 40–50 pfh (main text).</p>
<span>doi:10.1371/journal.pone.0000776.g001</span></div><div class="figure" id="pone-0000776-t001"><div class="img"><a name="pone-0000776-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of characteristic equation analysis for <em>endo16</em> SLM with Otx, R and UI as variable of input conditions.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t001</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this report, the construction and analysis of <em>endo16</em> SLM are first introduced to provide validation over our approach. Subsequently, a mechanism of transcriptional-regulatory control on <em>endo16</em> expression is proposed on the basis of the analyses. This is followed by Conclusion and <a href="#s4">Methods</a> sections. In <a href="#s4">Methods</a> section, apart of the mathematical framework for our approach, three synthetic models are included to further illustrate the features of SLM analysis. Moreover, we also demonstrate the construction of gene expression control at network level by combining multiple SLMs that are representing different gene components of a gene network (<a href="#s4">Methods</a>).</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Generation and first test of the model</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Temporal activation of transcription factor binding sites is defined as the occupation of transcription factors on these sites over time in which causing the regulation mRNA expression of a particular gene. For verification purposes, our approach is applied to decipher <em>cis</em>-regulation of sea urchin <em>endo16</em> temporal gene expression data <a href="#pone.0000776-Yuh1">[9]</a>, <a href="#pone.0000776-Yuh2">[10]</a> to construct a sequential logical model; although our approach can handle general transactivity, in this section we only focus on <em>cis</em>-regulation due to the fact that we are using mutation data on <em>cis</em>-acting sites of <em>endo16</em>. The strict deterministic control exerted on gene expression during embryo development makes this model system almost ideal for testing the SLM approach. Recent work by Yuh <em>et al</em> <a href="#pone.0000776-Yuh2">[10]</a> demonstrated that the so called A and B modules are the main <em>cis</em>-regulatory regions that control the expression of <em>endo16</em> for specification and differentiation of sea urchin gut development. In our analysis, UI (Unique Factor I, characterized as Brn1/2/4 Yuh <em>et al</em> <a href="#pone.0000776-Yuh3">[18]</a>) and Otx (<em>Orthodenticle-like</em>) <a href="#pone.0000776-Yuh4">[19]</a> are selected because they are the key-drivers of module B and A respectively <a href="#pone.0000776-Yuh2">[10]</a>. R is chosen as another input since it is suggested to be critically involved in the switching from specification of sea urchin endomesoderm to its differentiation into embryonic gut. It has been shown that UI, R and Otx are sufficient to resemble the regulation caused by module A and B during the same developmental period <a href="#pone.0000776-Yuh2">[10]</a>. A set of temporal gene expression profiles related to UI, Otx and R sites are elected for modeling. All temporal gene expression levels are normalized with respect to the control BA-Bp•CAT expression profile in term of concentration of CAT (chloramphenicol acetyl transferase) acting as reporter (i)<a href="#pone.0000776.s003">Figure S1(A,B,D–F)</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Our modelling approach involves the discretization and digitization of temporal mRNA expression profiles into a finite number of levels (states), which are defined by discretization of continuous/analogue gene expression levels (Step 1 &amp; 2, <a href="#pone-0000776-g002">Figure 2</a> and <a href="#s4">Methods</a>). During the mapping process (Step 3&amp;4), we consider the variation in mRNA expression level (<em>state</em>) to exhibit a behaviour in which the transition from present to the next state only depends on the present state and the corresponding input conditions. The input condition is defined as the activation state of <em>cis</em>-acting sites (Step 2, <a href="#s4">Methods</a>).From the BA-Bp•CAT construct (<a href="#pone.0000776.s003">Figure S1(A)</a>) the Otx input is considered as ‘on’ (represented by 1) if the sampling time falls between 18–48 <em>post fertilization hours</em> (pfh), and ‘off’ (represented by 0) otherwise <a href="#pone.0000776-Yuh2">[10]</a>. If the sampling time falls between 24–72 pfh, we assign 1 to <em>UI</em> input only in this period. R is always considered to be ‘on’ at all intervals within 0–72 pfh (refer <a href="#pone.0000776.s003">Figure S1(A)</a>). Hence, the temporal input condition of UI, R and Otx is given as (010<sub>2</sub>, 010<sub>2</sub>, 010<sub>2</sub>, 010<sub>2</sub>, 010<sub>2</sub>, 011<sub>2</sub>, 011<sub>2</sub>, 111<sub>2</sub>, 111<sub>2</sub>, 111<sub>2</sub>, 111<sub>2</sub>, 111<sub>2</sub>, 111<sub>2</sub>, 111<sub>2</sub>, 110<sub>2</sub>, 110<sub>2</sub>, 110<sub>2</sub>, 110<sub>2</sub>, 110<sub>2</sub>, 110<sub>2</sub>), where a unit time interval is about 3.6 hours. The optimum unit interval is chosen such that maximum number of state transitions is obtained and minimum consecutively identical states transitions are generated. Consecutively identical states transitions occur at multiple unit time intervals due to over sampling. Since such state transitions can be viewed as a single state transition over a longer time interval, they are lumped locally together to represent one state transition. The truth table for <em>endo16</em> only consists of state transitions extracted from results of CAT reporter provided from the publications (Step 3). State transitions that are not available are regarded as <em>don't care</em> conditions (<a href="#s4">Methods</a>).</p>
<div class="figure" id="pone-0000776-g002"><div class="img"><a name="pone-0000776-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Overviews of SLM and analyses.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Step 1. Obtain temporal transcriptional activation and corresponding time-series expression data; Step 2. SLM Mapping: Dicretization and digitization into time-series bar chart; Step 3. Truth table construction: tabulate the digitized data into present states, input conditions and next states; Step 4. Mathematical mapping: Characteristic equation analysis and time simulation analysis; Step 5. Network motifs construction: integration of multiple SLMs to form gene network model.</p>
<span>doi:10.1371/journal.pone.0000776.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We present a single valued function of sequential logic mapping (Eq. (1)) which suggests the involvement of extra number of <em>cis</em>-acting sites in the transactivation control of gene expression BA-Bp•CAT during 50–61 pfh. The sequential logic mapping is a single valued function since only a unique next state is given by a particular pair of present state and input condition. However, we found out that there are one-to-many correspondences occurring in some of the temporal expression profiles. In a temporal expression profile, B(UIm)A(Otxm)-Bp•CAT (UIm: UI mutation; Otxm: Otx mutation)(refers to <a href="#pone.0000776.s003">Figure S1(B)</a>) two different state transition occurs from a unique pair of present state and input condition at 25–40 and 50–61 pfh respectively. Given 10<sub>2</sub> as present state and 010<sub>2</sub> as input condition of UI, R and Otx, two different next states, 10<sub>2</sub> at 25–40 pfh and 01<sub>2</sub> at 50–61 pfh are mapped. State transition at 25–40 pfh is selected base on two selection criteria: 1) The selected transition is due to the combined activities of three <em>cis</em>-acting sites: UI, R and Otx; 2) one-to-many correspondence for the chosen state transition is not found in other expression profiles.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Beyond 25 pfh, in order to check the involvement of UI (first criterion), we compare the next state of B(UIm)A(Otxm)-Bp•CAT (UI mutant) (refer to <a href="#pone.0000776.s003">Figure S1(B)</a>) and BA(Otxm)-Bp•CAT (non-UI mutant) (refer to <a href="#pone.0000776.s004">Figure S2(B)</a>) at time interval 25–40 pfh. Under the same present state (10<sub>2</sub>), no transition (10<sub>2</sub>) is induced in the UI mutant, whereas in the non-UI mutant, state transition to 11<sub>2</sub> level occurs. Therefore, UI is required for the state transition of non-UI mutant. However, we are unable to conclude the UI involvement at 50–61 pfh since the present state of BA(Otxm)-Bp•CAT is different from B(UIm)A(Otxm)-Bp•CAT: two state transitions are comparable if their present states are identical. Criterion 2 is applied which reveals another one-to-many correspondence occurring in a different profile when state transition at 50–61 pfh is selected. In B(Rm)A-Bp•CAT (Rm: R mutation)(refer to <a href="#pone.0000776.s003">Figure S1(B)</a>) during 50–61 pfh and 74–80 pfh, for the same present state (11<sub>2</sub>) and the same input condition of UI, R and Otx, 100<sub>2</sub>, two different next states are observed: 10<sub>2</sub> at 50–61 pfh and 11<sub>2</sub> at 74–80 pfh. This conflict (one-to-many correspondence for the chosen state transition) suggests that the binary activities of the three <em>cis</em>-acting sites, UI, R and Otx are not enough to understand state transition activity at 50–61 pfh in the wild type expression profile and additional <em>cis</em>-acting sites are required in sequential logic modeling. The findings not only specified the requirement of extra regulatory site, it also identified when such regulation is occurring.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>A simple analysis of sequential logic mapping reveals cooperativity and conditional effect of three transcription factors, UI, R and Otx. Pertaining to the case of <em>endo16</em>, present states are divided into four levels: 00<sub>2</sub>; 01<sub>2</sub>; 10<sub>2</sub>; 11<sub>2</sub>, corresponding to basal (4×10<sup>5</sup> mol per embryo), threshold (8×10<sup>5</sup> mol per embryo), specification (12×10<sup>5</sup> mol per embryo) and differentiation (16 ×10<sup>5</sup> mol per embryo) respectively from analysis in Yuh <em>et al</em>. <a href="#pone.0000776-Yuh1">[9]</a>, <a href="#pone.0000776-Yuh2">[10]</a>. These states are used in both characteristic equation and simulation analysis (Step4, <a href="#s4">Methods</a>). The simplified logical equation (sequential logic equation, SLE) of <em>endo16</em> SLM is derived and shown in Eq. (1) (<a href="#pone.0000776.s002">Table S1</a> shows the corresponding truth table).<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>q<sub>1,t</sub> q<sub>0,t</sub></em> and <em>q<sub>1,t+1</sub> q<sub>0,t+1</sub></em> are binary variables representing present and next state respectively; <em>UI, R</em> and <em>Otx</em> (<em>italic</em> for variable) are the binary variables for the input condition of the three <em>cis</em>-acting sites.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>The characteristic equation analysis is achieved by substituting input conditions into variables <em>UI</em>, <em>R</em> and <em>Otx</em> in Eq. (1). (<a href="#s4">Methods</a>). The equations given in <a href="#pone-0000776-t001">Table 1</a> allow for the derivation of some important insights about the role played by the different actors in time:</p>


<ol class="order">

<li>When present state equals to basal (00<sub>2</sub>) level, Otx functions as an activating site since state increase only if <em>Otx</em> = 1, provided that <em>UI</em> = 0. Otx activation occurs between 0–25 pfh of the control expression profile whereas UI is not activated during this time period. The result suggests Otx activation function as a ‘kick off’ switch for <em>endo16</em> expression.</li>

<li>At the threshold (01<sub>2</sub>) level, the characteristic equation suggests that state transition is only dependent on UI (<a href="#pone-0000776-t001">Table 1</a>). This is clearly shown during 25–40 pfh of the control expression profile that equally shows that, at this level, the state transition is independent of R and Otx. Thus, UI alone plays a role as gene driver to increase <em>endo16</em> expression level beyond threshold level at 25–40 pfh.</li>

<li>If present state has reached the specification (10<sub>2</sub>) level, activation of R and Otx (<em>R</em> = 1 and <em>Otx</em> = 1) will initiate a repressive effect on gene expression (refer to <a href="#pone.0000776.s003">Figure S1(B)</a>). This situation can be found during 40–50 pfh of control expression profile (<a href="#pone-0000776-g001">Figure 1</a>). As long as both R and Otx are activated, the state will not increase from specification to differentiation (11<sub>2</sub>) level. The characteristic equation contains the repressive term: (<em>ROtx</em>)' ((<em>ROtx</em>)' = 0 for <em>R</em> = 1 and <em>Otx</em> = 1) at specification level (<a href="#pone-0000776-t002">Table 2</a>). This implies that Otx and R sites operate together to introduce a repressive effect on gene expression such that the state cannot achieve differentiation level. One interesting point to note is that although <em>cis</em>-acting sites, R and Otx are activated during 29–50 pfh, repressive activation occurs only during 40–50 pfh. This result clearly demonstrates that activation of binding site of R and Otx is distinct from the resultant functional activation of <em>endo16</em> expression (<a href="#pone-0000776-g001">Figure 1</a>).</li>

<li>Previous experimental analyses from Yuh <em>et al</em> <a href="#pone.0000776-Yuh2">[10]</a> suggested activation of R switch depends on UI activation. Characteristic equation analysis further explores and specifies that whether the dependency is a direct or indirect. A direct dependency is due to physiochemical interaction of two TFs (formation of functional complex) whereas indirect dependency occurs when two TFs are interacting via a third party (no complex formation) (see <a href="#s4">Methods</a>: <em>Prediction of interactions between transcriptional binding sites from the transition map</em>). When R switch repression occurs (at 30–48 pfh, state transition from 10<sub>2</sub> to 10<sub>2</sub>), <em>UI</em> = 1, <em>R</em> = 1 and <em>Otx</em> = 1 (<a href="#pone-0000776-t002">Table 2</a>), suggests the function of R switch depends on the activation of UI. Since, the <em>UI</em>(<em>ROtx</em>)' term consists UI outside the parentheses of (<em>ROtx</em>)', therefore, UI is not considered as part of the repressive complex, hence indicating R switch is indirectly dependent on UI activation.</li>

</ol><div class="figure" id="pone-0000776-t002"><div class="img"><a name="pone-0000776-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Functional effect of combinatorial input condition in state transition for <em>endo16</em> SLM.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t002</span></div><a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>From the above considerations, it is cleared that how we can easily derive from the characteristic equation a detailed tale of the roles played by the different regulators at subsequent times and their mutual interactions in a way formally similar to sensitivity analysis in differential equation style but allowing for a much greater flexibility (<a href="#pone-0000776-t002">Table 2</a>).</p>


<h4><em>In-silico</em> mutagenesis of <em>endo16 cis</em>-regulatory region</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Having obtained a reliable SLM describing <em>cis</em>-regulation by means of detailed analysis of the correspondence between activation states of the transcription factors and the reporter gene temporal profile, we tested the model by means of a sort of <em>in-silico</em> mutagenesis (<a href="#s4">Methods</a>). Since the state transitions in the truth table only carry information within the time periods 0–50 pfh and 61–s80 pfh, the model derived from the truth table is not supposed to perform simulation with time period of 0–80 pfh. Hence, a truncated-time-simulation for 0–50 pfh and 61–80 pfh are executed separately with different initial state conditions. The initial condition for present state in all expression profiles within 0–50 pfh is equal to 00<sub>2</sub> basal level by default whereas the initial state for expression profiles within 61–80 pfh is varying. The initial state of BA·Bp-CAT for 61–80 pfh, for instance, is found to be threshold level in the discretized profiles of the raw data (refer to <a href="#pone.0000776.s003">Figure S1(A)</a>). Same procedure is applied for forward and reverse mapping mentioned in the next section.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The above procedure allows for the derivation of the functional consequences of different mutations of the system:</p>


<ol class="order">

<li>Unlike wild type, the R mutant (<em>R</em> = 0, <a href="#pone-0000776-t003">Table 3</a>) reaches a premature differentiation level early at about 40–50 pfh. This differentiation can be achieved even if Otx is activated since (<em>ROtx</em>)' = 1 (<a href="#pone-0000776-t001">Table 1</a>).</li>

<li>UI mutant is investigated by setting <em>UI</em> = 0. Threshold level is the maximum state that can be achieved and there is no differentiation level in UI mutant. UI mutant can establish a threshold level only when Otx is activated. Subsequently, the mutation of Otx (<em>Otx</em> = 0) introduces a profile similar to R mutant where an early differentiation expression level at 40–50 pfh is identified when Otx is mutated.</li>

</ol><div class="figure" id="pone-0000776-t003"><div class="img"><a name="pone-0000776-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Combinatorial mutant logic equations for <em>in silico</em> mutagenesis of <em>endo16</em> SLM.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The operational consequences of simultaneous mutations of different sites can be investigated as well: three different mutant couples (ROtx)m , (UIOtx)m and (UIR)m are generated by setting (<em>R</em> = 0, <em>Otx</em> = 0), (<em>UI</em> = 0, <em>Otx</em> = 0) and (<em>UI</em> = 0, <em>R</em> = 0) respectively. Some consequences of these double mutations can be derived:</p>


<ol class="order">

<li>If initial state is equal to basal level, (ROtx)m will always remain in the same state.</li>

<li>In the case of (UIOtx)m, from any state different from threshold level, R activation has no effect and the expression profile always remains at basal level.</li>

<li>Expression profile generated by (UIR)m is restricted to threshold level and the only non-mutated input, Otx, is functional at low state (00<sub>2</sub>, 01<sub>2</sub>).</li>

</ol><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>These results highlight other dependencies among the <em>cis</em>-acting sites not evident from the simple characteristic equation analysis. The identical logic equations of (OtxR)m and Otxm indicate that R has no effect. <a href="#pone-0000776-t001">Table 1</a> shows that the AND logic (<em>OtxR</em>)' (repression) is found in the characteristic equations that are derived from setting 01<sub>2</sub> and 10<sub>2</sub> as present state. Hence, R functions as <em>silencer</em> in which the repressive effect is dependent on the activation of Otx (<a href="#pone-0000776-g001">Figure 1</a> and <a href="#pone.0000776.s003">Figure S1(C)</a>).</p>


<h4>Forward and Reverse Mapping: predict novel expression and profile specific binding activities of UI, R and Otx</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Forward mapping is applied on <em>endo16</em> sequential logic equation to <em>infer</em> a novel temporal gene expression profile with a given temporal activity of <em>cis</em>-acting sites (input condition), whereas reverse mapping can be used to <em>deduce</em> possible input conditions from a given temporal gene expression profile (details in <a href="#s4">Methods</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Forward mapping is performed to replicate the expression profile of BA-Bp•CAT for verification of <em>endo16</em> SLM. The state transitions during 50–61 pfh are excluded in <em>endo16</em> sequential logic modelling. Truncated-time-simulation is performed where forward mappings for 0–50 pfh and 61–80 pfh are carried out separately with different initial present state (<a href="#pone-0000776-g003">Figure 3</a>). To simulate the profile beyond 61 pfh the present state is reset at 47–50 pfh.</p>
<div class="figure" id="pone-0000776-g003"><div class="img"><a name="pone-0000776-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Forward mapping and <em>in silico</em> mutagenesis of <em>endo16</em> SLM.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Discretized profile of BA-Bp·CAT at 0–50 and 61–72 pfh are reproduced by forward mapping. (B) <em>In silico</em> mutagenesis of R is achieved by providing input series with <em>R</em> is always set to 0. State transition to differentiation expression level occurred early at 40–50 pfh. The rest of the state transition beside 40–50 pfh is same as <a href="#pone-0000776-g003">Figure 3(A)</a>. (C) <em>In silico</em> mutagenesis with Otx as the only input condition; the state transitions are generated to show similar expression profile to that of A-Bp·CAT ((<a href="#pone.0000776.s003">Figure S1 (A)</a>)</p>
<span>doi:10.1371/journal.pone.0000776.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Next, forward mapping with a novel input series is used to prove the statement Otx is the driver of Module A, to do so the following series: input condition (<em>UI<sub>t</sub> R<sub>t</sub> Otx<sub>t</sub></em>) (000<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 001<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>, 000<sub>2</sub>) is substituted into Eq. (1). This simulation indicates that the state level is 01<sub>2</sub> during 18–50 pfh and is 00<sub>2</sub> otherwise. The results generate good approximation to experimentally observed Otx-Bp•CAT (refers to <a href="#pone.0000776.s004">Figure S2(B)</a>). This confirms Otx as the <em>main driver</em> for module A.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Reverse mapping is applied on a specific expression profile in which the differentiation level (11<sub>2</sub>) is achieved before 21 pfh, for the estimation of a series of input condition that establish such an expression profile. The particular state level for reverse mapping during 0–21 pfh is given as 00<sub>2</sub> 01<sub>2</sub> 10<sub>2</sub> 10<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub>. The results of reverse mapping provide 216 possible combinations of temporal input conditions in total. However, it is known that R activation is time-invariant in <em>endo16</em> mRNA expression. As a result, only 2 out of the 216 possible combinations are selected under the condition where R is always being activated (<a href="#pone-0000776-t004">Table 4</a>). Reverse mapping analysis suggests that activation of UI and Otx in advance and then followed by earlier deactivation of Otx can result in the expression reaching differentiation level before 21 pfh.</p>
<div class="figure" id="pone-0000776-t004"><div class="img"><a name="pone-0000776-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Reverse mapping of given <em>endo16</em> expression profile when R is considered to be always activated.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t004</span></div>

<h4>Regulatory mechanism for <em>endo16</em> expression during gut specification to differentiation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The sequential logic analyses (characteristic equation analysis and simulation analysis) are able to model the regulatory mechanism for <em>endo16</em> expression during gut specification to differentiation as follows.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>During 0–25 pfh, Otx functions as an activation site that drives the <em>endo16</em> expression level from basal to threshold level. The threshold level holds a role of temporal checkpoint for specification level to occur since UI is unable to drive the expression further (to reach specification level) if threshold level is not reached. Beyond 25 pfh (transition from threshold to specification level), Otx has no activating effect on <em>endo16</em> expression even though the site is still in activation. Just above threshold level, UI is activated and has begun to drive the expression profile to the specification level. Although activation of UI is able to drive the <em>endo16</em> expression beyond differentiation level during 70–80 pfh <a href="#pone.0000776-Yuh2">[10]</a>, it is found that the expression does not exceed specification level between 25–70 pfh even UI is activated. Hence, a repressive effect must be present to prevent UI from driving the expression beyond specification level. In fact, apart from earlier (0–25 pfh) activation, Otx exhibits a repressive effect at the period of 40–50 pfh when R is activated. When present state is 10<sub>2</sub> (specification level), the characteristic equation derived from Eq. (1) consists of (<em>ROtx</em>)' (<a href="#pone-0000776-t001">Table 1</a>) term, which indicates that presence of both R and Otx prevents a state jump from 10<sub>2</sub> to 11<sub>2</sub> state within 40–50 pfh. The repressive effect plays an important role for specifying collaborative effect of ROtx as an essential switching control from specification to differentiation. The maintenance of specification level at 40–50 pfh is required such that gene expression does not reach differentiation level (state greater than 10<sub>2</sub>) at the earlier developmental stage. During 50–61 pfh, additional <em>cis</em>-acting sites other than UI, R and Otx are responsible to turn off the specification expression (from 10<sub>2</sub> to 01<sub>2</sub>) as the only committed cells are preparing for differentiation expression. Upon removal of Otx activation at later stage, R repression is deactivated, i.e. differentiation expression is allowed to begin only when Otx control is absent.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The analysis based on the sequential logic modeling consists of characteristic equation and simulation analyses. Analysis of SLM systematically extracts functional information on transcriptional activities hidden in mutation on <em>cis</em>-acting sites and corresponding gene expression data. Nevertheless, to attain input condition, SLM construction is not restricted on the usage of mutation data. For instance a model can be derived by the observations coming from a pharmacological intervention on specific trans-activating inducers.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The discrete nature of SLM operates noise suppression on the experimental data by collapsing the continuous value of expression into modal classes. Characteristic equation analysis allows for systematic identification of the key function of <em>cis</em>-acting sites and the logical relationship between the sites. Our model reveals that the function of a <em>cis</em>-acting site can vary under different present state instead of solely determined by input condition. The variation of the function of <em>cis</em>-acting site due to different present state is defined as <em>conditional effect</em> of state transition. This conditional effect can be due to several factors, including the consequence of DNA structure remodeling, co-factor activity or other mechanical and chemical process such as covalent modification during transcription. These results can provide the mean for hypothesis formulation and experimental design in understanding the mechanism of transcription.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Three time-simulation analyses, <em>in silico</em> mutagenesis, forward and reverse mapping can be achieved based on sequential logic. <em>In silico</em> mutagenesis, performed by deactivating <em>cis</em>-acting sites, provides prediction of mutant expression profiles and identifies global functional activity of <em>cis</em>-acting sites in the gene expression. In contrast, characteristic equation analysis reveals local functional activities with given present state and input condition. Forward mapping is employed for prediction of novel or uncharacterized temporal expression profile under different input conditions. The forward mapping simulation establishes a platform to investigate the variation in dynamics of the activation of <em>cis</em>-acting sites. Lastly, reverse mapping is a useful feature to suggest possible input condition in a dynamical manner, which leads to a particular expression profile. The resulting time series of input conditions provide indication for the dynamics of transcription factors that are involved in transcriptional regulation. As a potential approach for constructing gene regulatory network, reverse mapping could be used to extract active signaling pathways. These signaling pathways are associated with the activity of these transcription factors from the given temporal RNA expressions.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In summary, sequential logic modelling has provided a non-parametric, model independent, dynamic and quantitative approach to facilitate systematic analysis of <em>cis</em>-regulatory system. At current stage, only the expression level is non-Boolean whereas the activation of <em>cis</em>-acting site is still considered as binary. Recent study on embryonic development also shows that the possibility of representing gene relative activation by means of few states is totally consistent with recent computational and experimental findings <a href="#pone.0000776-Yuh3">[18]</a>, <a href="#pone.0000776-Blasi1">[20]</a>, <a href="#pone.0000776-DiCamillo1">[21]</a>, <a href="#pone.0000776-Barrett1">[22]</a>, <a href="#pone.0000776-Bolouri2">[23]</a>, <a href="#pone.0000776-Istrail1">[24]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The concentration effect of transcription factor during transcription can be incorporated in our approach as non-binary activation of <em>cis</em>-acting sites <a href="#pone.0000776-Bolouri2">[23]</a>, <a href="#pone.0000776-Bulyk1">[25]</a>. In our subsequent work, we are focusing on dealing with the binding effect (defined as occupancy by Bolouri <em>et al</em> <a href="#pone.0000776-Bolouri2">[23]</a> and Istrail <em>et al</em> <a href="#pone.0000776-Istrail1">[24]</a>) of transcription factors on the activation of <em>cis</em>-acting sites to generalise the simple on/off model of activation. This can be simply carried out by assigning two (or more) binary number for activation of each <em>cis</em>-acting site (e.g., <em>UI</em>→<em>UI<sub>1</sub>UI<sub>0</sub></em>) which is same as dealing with the expression state, <em>q<sub>1</sub>q<sub>0</sub></em>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>A fully differentiated biological system may display a more analogical behaviour of gene regulation. We handle this situation in i) <em>cis</em>-acting sites activation as well as ii) gene mRNA induction. In i), activation level of <em>cis</em>-acting sites is discretized (beyond on/off) based on transcription factor concentration (see above). In ii), gene regulatory networks have often been considered to be naturally ‘discretized processes’ <a href="#pone.0000776-Kramer1">[26]</a>, <a href="#pone.0000776-Angeli1">[27]</a>. Bistable and multistable (and hence discrete) hysteretic switches, enabling cells to adopt multiple internal expression states in response to an external input have a pivotal impact on biological systems, ranging from cell-fate decisions to cell-cycle control <a href="#pone.0000776-Kramer1">[26]</a>. This has to do with the modular and hierarchical characteristics of biological systems. For example, the existence of ‘modules’ implies some form of discretization occurs while any form of hierarchy implies the possibility to define ‘discrete layers’. The choice of the ‘optimal discretization’ when in presence of a sufficient amount of data can be based on the maximization of explained variance by a cluster analysis procedure <a href="#pone.0000776-Giuliani1">[28]</a>, <a href="#pone.0000776-Martin1">[29]</a>, a well studied statistical problem. A specific observation is assigned to its ‘discrete class’ on the basis the minimum distance to the k centroid values (average values for the studied variables, in this case the clustering variable is only one and corresponds to the expression level of the correspondent gene) relative to the best k-means cluster solution. The optimal number of clusters k (discrete classes) is maximizing the model explained variance (R-square), namely, the ratio between the variance of the distribution of the single statistical units coded by the relative cluster centroid value and the original distribution total variance.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Our discretization procedure is based on the analysis of Yuh <em>et al</em>. <a href="#pone.0000776-Yuh1">[9]</a>, <a href="#pone.0000776-Yuh2">[10]</a>, which utilizes experimental replicates for noise averaging effect. However, we can extend our discussion on the effect of noise to the construction of SLM, after the optimal discretization is obtained, by the following 2 points: a) discretization is a well known technique to reduce noise in engineering field. With sufficient data, using basic statistical analysis such as looking at standard deviation and mean values, we can appropriately discretize the state levels in both transcription factor binding activation and mRNA expression. In the case of a sufficiently high number of observations, a data driven discretization process can be performed by means of k-means cluster analysis <a href="#pone.0000776-Giuliani1">[28]</a>, <a href="#pone.0000776-Martin1">[29]</a> by assigning each observation to the nearest cluster. The noise is not a major issue for this approach given that substituting actual value with the cluster centroid value (cluster = discrete classes) facilitates the noise filtration. b) If a wrong transition state occurs due to the noise, this will be seen in simulation analysis, e.g. i) result in conflicting binding activation (in comparison to experimental data for modelling) by <em>Reverse mapping</em>, Step 4b, ii) multiple state transitions occurs for same input and present state by <em>Forward mapping</em>, Step 4b; however, this type of errors are not expected to be frequent if clusters are well formed (e.g. k-means cluster analysis is bounded to generate clusters with the most separate as possible). <em>Endo</em> 16 data is particularly a favourable example i) as it based on a very good previous experimentation and ii) clearly because developmental system is under strict control and consequently the classes of concentrations (discretization) are easy to detect.</p>

<h4>Conclusion</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We have developed the model on a particularly well known regulation system, acting in a quasi-deterministic matter so to provide a reliable development scheme. This promises to be potentially fruitful avenue for the systematic analysis of gene regulation networks in general. The main strength of the model resides in its flexibility: it allows for different roles played by the same factors in different instants of times and different initial conditions such as in understanding of time-delay in <em>cis</em>-regulation of gene expression. One interesting feature of SLM can be traced back to the possibility of dramatically reducing the complexity of parameterization in regulatory circuits that until now have hampered the classical differential equation approach in time-course modeling of regulatory networks. Moreover, the proposed method ends up with a description of the modeled regulation circuits into easily understandable way to the mainframe biologists without the need of mathematical formalisms which are relatively difficult to grasp.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>The sequential logic models (SLM)</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The SLM constitutes of the following 5 operations (<a href="#pone-0000776-g002">Figure 2</a>):</p>


<ol class="order">

<li>Obtaining data for temporal transcriptional activation and corresponding temporal gene expression data from experiments/literatures. Additionally, the information of active transcriptional regulatory sites can also be estimated computationally <a href="#pone.0000776-Djordjevic1">[30]</a>, <a href="#pone.0000776-Hughes1">[31]</a>, <a href="#pone.0000776-Tamada1">[32]</a></li>

<li>Discretization of gene expression level and expression time: The full range (maximum level–minimum level among data) of continuous gene expression levels is subdivided into multiple discrete and equally separated states. Concurrently, the time axis of the profile has also been discretized by sampling the gene expression data at fixed time intervals. b) Digitization of gene expression level and transcriptional activity associated with unit time interval: Each discretized expression state is encoded with binary value. There are maximally <em>2<sup>n</sup></em> states can be coded by n number of binary bit. For transcriptional activity, a binary input value representing the activation of single transcriptional-regulatory site that associates to the current state transition is assigned to each interval. The set of binary input values (multiple activations) is called input condition. Input condition is defined as the activity of transactivation on <em>cis</em>-acting sites. A value of 1 (of binary variable) denotes activation, whereas a value of 0 denotes deactivation. Binary variable, <em>x'</em> is defined as complementary representation of the binary variable, <em>x</em>.</li>

<li>Truth table construction: A truth table is a tabulated representation that illustrates the mapping of possible state (expression level) transitions under various input conditions, which obtained from digitized temporal gene expression profiles. The table includes the information of the states at time t (present states), input conditions at time <em>t</em> and states at time <em>t</em>+1 (next states) (three main columns). Present state and next state are represented by (<em>q<sub>1</sub></em>,<sub>t</sub> <em>q<sub>0,t</sub></em>)and (<em>q<sub>1,t+1</sub> q<sub>0,t+1</sub></em>) respectively, where <em>q<sub>n,t</sub></em> and <em>q<sub>n,t+1</sub></em> refer to the <em>n</em><sup>th</sup> bit (n = 0,1) at present state (time interval <em>t</em>) and next state (time interval <em>t</em>+1) respectively (<a href="#pone-0000776-t005">Table 5</a>). When binary transcriptional activation is considered, there are <em>2<sup>m</sup></em> of possible input conditions, where m is number of inputs (e.g. number of transcription factor binding sites). If there are <em>k</em> = <em>2<sup>n</sup></em> states and m <em>cis</em>-acting sites, the total number of row for a complete table is equal to <em>2<sup>k+m</sup></em>.</li>

<li>Mathematical Mapping by SL (<a href="#pone.0000776.s001">Text S1</a>): The mapping can be represented as a finite state machine (finite state automata) <a href="#pone.0000776-Ruklisa1">[33]</a>, <a href="#pone.0000776-Brazma1">[34]</a> which consists of a finite number of states (expression states), transitions between those states, and inputs (transcriptional activity). The output of the ‘machines’ is equal to the next state. In general, the next state is a function (<strong><em>F</em></strong>) of present input and present state (refers to Mealy model <a href="#pone.0000776-Mano1">[35]</a>). <strong><em>F</em></strong> can be represented as sum of AND logics in binary system <a href="#pone.0000776-Mano1">[35]</a>, this function is called sequential logic equation (SLE). On the basis of SLE, both characteristic equation analysis and time-simulation analysis have been developed to investigate the dynamics of transcriptional-regulation. a) <em>Characteristic equation analysis</em> involves the substitution of present state into its state variables in SLE and forming a simplified equation that characterised the transcriptional regulation by input variables at the particular present state. The simplified characteristic equations determine the dynamic functions and interactions among transcriptional binding sites. b) <em>Time-simulation analysis</em> is performed to simulate gene expression profile in mutagenesis analysis (<em>in silico</em> mutagenesis), to predict novel gene expression profiles under different activity of <em>cis</em>-acting sites (forward mapping) and to identify specific binding activity when a particular expression profile is given (reverse mapping).</li>

<li>Gene regulatory network motif construction: When more than two genes are regulated by identical inputs, we construct each corresponding SLE and combine then and consider as co-regulated gene network of SLMs to describe gene regulatory network motifs.</li>

</ol><div class="figure" id="pone-0000776-t005"><div class="img"><a name="pone-0000776-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Truth table representation of state transition information for Enhancer-activator model.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t005</span></div><a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>To further illustrate our approach conceptually, three synthetic models are introduced at the following sections. In order to develop binary <em>cis</em>-regulatory system through synthetic models, we consider 2 <em>cis</em>-acting sites with binary activation and 2 bits expression state discretization (<em>n</em> = 2 and <em>m</em> = 2 in Eq. (2)).</p>


<h4>Synthetic Model: Enhancer-activator (EA) sequential logic model</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>To illustrate the utility of our SLM, we developed a synthetic model for EA binding (<a href="#pone-0000776-g004">Figure 4</a>) consisting of four temporal gene expression output profiles (<a href="#pone-0000776-g002">Figure 2, Step 1</a>). The gene expression level is discretized into four states by two binary bits (Step 2). Input conditions, <em>E</em> and <em>A</em> are binary variables that represent the activation (on/off) of the two <em>cis</em>-acting sites E and A. E and A are specifically designed as non-synergistic enhancer and activating site respectively. The complete truth table (<a href="#pone-0000776-t005">Table 5</a>) is constructed from 16 possible state transition that represents the mapping between activation of <em>cis</em>-acting sites and temporal mRNA expression profiles described in <a href="#pone-0000776-g002">Figure 2 (Step 3)</a>. Present state and next state are represented by two binary variables, <em>q<sub>1,t</sub> q<sub>0,t</sub></em> and <em>q<sub>1,t+1</sub> q<sub>0,t+1</sub></em> respectively. The current state, <em>q<sub>1,t</sub> q<sub>0,t</sub></em> is simplified as <em>q<sub>1</sub> q<sub>0</sub></em>. Construction of SL equation is a standard procedure in digital design <a href="#pone.0000776-Mano1">[35]</a>. For instance, from 8<sup>th </sup>row at <a href="#pone-0000776-t005">Table 5</a>, the next state level, <em>q<sub>1,t+</sub>1 q<sub>0,t+1</sub></em> is 10<sub>2</sub>, where <em>q<sub>1,t+1</sub></em> = 1 and <em>q<sub>0,t+1</sub></em> = 0. The first bit (Least Significant Bit) of the next state, <em>q<sub>0,t+1</sub></em> produces 1 only when <em>q<sub>1</sub></em> = 0, <em>q<sub>0</sub></em> = 1, <em>E</em> = 1 and <em>A</em> = 1. This condition is AND logic. Therefore, the corresponding logical term (<em>minterm</em>) of <em>q<sub>0 t+1</sub></em> is given as <em>q<sub>1</sub>'q<sub>0</sub>EA</em> (<a href="#pone-0000776-g005">Figure 5</a>). Subsequently, the sequential logic equation corresponding to <a href="#pone-0000776-t005">Table 5</a> is constructed (Step 4):<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Eq. (2) can be further simplified (factorized) into Eq. (3) computationally by Quine-McCluskey algorithm or identified graphically by Karnaugh Map <a href="#pone.0000776-Mano1">[35]</a>.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br></p>
<div class="figure" id="pone-0000776-g004"><div class="img"><a name="pone-0000776-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Synthetic model for activation of E and A inputs.</span></strong></p><a id="article1.body1.sec4.sec2.fig1.caption1.p1" name="article1.body1.sec4.sec2.fig1.caption1.p1"></a><p>(A) Only exhibits basal expression without binding of transcription factors. (B) Binding of transcriptional activator on activation site increase the expression level. (C) Binding of another activator on enhancer does not alter expression level. (D) Expression level is highly elevated in comparing to (B) when both activator site and enhancer are bound.</p>
<span>doi:10.1371/journal.pone.0000776.g004</span></div><div class="figure" id="pone-0000776-g005"><div class="img"><a name="pone-0000776-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Construction of sequential logic mapping between dynamic transactivity and a state transition.</span></strong></p><a id="article1.body1.sec4.sec2.fig2.caption1.p1" name="article1.body1.sec4.sec2.fig2.caption1.p1"></a><p>State (expression level) is represented by the two binary numbers, <em>q<sub>1,t+1</sub>q<sub>0,t+1</sub>.</em> (A) Activation of E and A inputs (<em>E</em> = 1, <em>A</em> = 1) generates the state transition from 01<sub>2</sub> to 10<sub>2</sub>. The mapping between the activation of EA sites (Transcriptional activity) and the state transition (Gene expression)) is established by <em>q<sub>1,t+1</sub></em> = <em>q<sub>1</sub>'q<sub>0</sub>EA</em> and <em>q<sub>1,t+1</sub></em> = 0, where minterm (AND logic), <em>q<sub>1</sub>'q<sub>0</sub>EA</em> is 1 only if <em>q<sub>1</sub></em> = 0, <em>q<sub>0</sub></em> = 1, <em>E</em> = 1 and <em>A</em> = 1. (B) The mapping between input (<em>E</em> = 0, <em>A</em> = 1) and the state transition from 11<sub>2</sub> to 10<sub>2</sub> is established by <em>q<sub>1,t+1</sub> = q<sub>1</sub>q<sub>0</sub>E'A</em> and <em>q<sub>0,t+1</sub></em> = 0, where minterm <em>q<sub>1</sub>q<sub>0</sub>E'A</em> is 1 only if <em>q<sub>1</sub></em> = 1, <em>q<sub>0</sub></em> = 1, <em>E</em> = 0 and <em>A</em> = 1.</p>
<span>doi:10.1371/journal.pone.0000776.g005</span></div><h5>Determination of dynamic function of transcriptional binding sites using Eq. 3: cooperativity and physiochemical variation (Step 4a).</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p>The construction of characteristic equation for Eq. (3) requires setting present state, <em>q<sub>1,t</sub> q <sub>0,t</sub></em> equal to either of 00<sub>2</sub>, 01<sub>2</sub>, 10<sub>2</sub> or 11<sub>2</sub>. As a result, four simpler mapping equations, which are now functions of <em>E</em> and <em>A</em>, represent four characteristic equations corresponding to the four present states respectively (<a href="#pone-0000776-t006">Table 6</a>). By setting <em>E</em> and <em>A</em> to either 1 or 0, the next state is obtained under these input conditions. For example, if present state, <em>q<sub>1</sub>q<sub>0</sub></em> = 00<sub>2</sub>, then the characteristic equation derived from Eq. (3) becomes <em>q<sub>1,t+1</sub></em> = 0 and <em>q<sub>0,t+1</sub></em> = <em>A</em> (<a href="#pone-0000776-t006">Table 6</a>). Firstly, this shows that the state transition from present state, 00<sub>2</sub> to next state only depend on <em>A</em> and it is independent from <em>E</em>. Secondly, if <em>A</em> is set to 1 (A is activated), the state will rise from 00<sub>2</sub> (present state) to 01<sub>2</sub> (next state). Otherwise, if <em>A</em> = 0 then there is no effect. Hence, A is concluded to be an activating site at 00<sub>2</sub> state and E has <em>no effect</em> under the same condition. Characteristic equation at 10<sub>2</sub> present state, <em>q<sub>1,t+1</sub></em> = <em>A</em> and <em>q<sub>0,t+1</sub></em> = <em>EA</em>, shows that activation of E site act as an <em>enhancer</em> to A as followed: 1) E has no effect if <em>A</em> = 0; 2) State transition remains at 10<sub>2</sub> state if <em>E</em> = 0; 3) State is transited to 11<sub>2</sub> state (1 level increment) if both A and E are activated. Moreover, comparison of different characteristic equation with identical present states can reveals <em>when</em> the enhancer function of E <em>cis</em>-acting site occurs (<a href="#pone-0000776-g006">Figure 6</a>).</p>
<div class="figure" id="pone-0000776-g006"><div class="img"><a name="pone-0000776-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Identification of dynamic functional activities of transcriptional inputs for EA model (</span></strong></p><a id="article1.body1.sec4.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec4.sec2.sec1.fig1.caption1.p1"></a><p>Eq. (3)). Function of identical transcription factor binding activity is not unique over time due to the dynamic nature. <em>Left</em>: characteristic equation analyses; <em>Right</em>: Colours (red and blue) indicate state transitions corresponding to the analyses. From Left, characteristic equation of the model at 01<sub>2</sub> present state indicates that while E and A sites are activated, E has no effect in the state transition from t<sub>3</sub> to t<sub>4</sub>. The next state is generated by the characteristic equation <em>q<sub>1,t+1</sub></em> = <em>A</em> and <em>q<sub>0,t+1</sub></em> = 0, in which there is no <em>E</em> variable, i.e., E is independent of the state transition. The characteristic equation at 10<sub>2</sub> present state, <em>q<sub>1,t+1</sub></em> = <em>E</em> and <em>q<sub>0,t+1</sub></em> = <em>EA</em> shows that E is only functioning as enhancer at time <em>t</em> = t<sub>8</sub> (see Right) when present state equals to 10<sub>2</sub> provided that <em>A</em> = 1. This example shows characteristic equation analysis can reveal <em>when</em> the enhancer function of E site occurs. (see conditional effect in <a href="#s4">Methods</a>).</p>
<span>doi:10.1371/journal.pone.0000776.g006</span></div><div class="figure" id="pone-0000776-t006"><div class="img"><a name="pone-0000776-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Summary of characteristic equation analysis for Enhancer-activator model.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t006</span></div>
<h5>Prediction of interactions between transcriptional binding sites from the transition map (Step 4a).</h5><a id="article1.body1.sec4.sec2.sec2.p1" name="article1.body1.sec4.sec2.sec2.p1"></a><p>Using the Boolean conditions of each state transition generated from a given characteristic equation defining one state level (Eq. S4), we can determine possible functions and interactions between binding sites for this state level. All possible Boolean combinations between two Boolean variables (minterms) <em>X</em> and <em>Y</em> describing the activation state of two binding sites X and Y can have a different interpretation to determine dependency between two binding sites, cooperativity and anti-cooperativity, and also binding sites functions (repression, enhancement). For example, dependency of two binding sites, X and Y can be determined in minterms motifs such as <em>XY</em>, (<em>XY</em>)<em>'</em>, <em>XY'</em> or (<em>XY'</em>)<em>'</em>, whereas non-dependency of X and Y are determined for motifs <em>X+Y</em> and (<em>X+Y</em>)<em>'</em> (manuscript in preparation). In <em>endo16</em> model, the (<em>ROtx</em>)<em>'</em> motif was found in the characteristic equations, determining dependency of R and Otx binding sites, and describing R-Otx complex activity as one repressor. Moreover, conditional effect can be determined by observing the different interpretation we can find for some given binding sites for the other characteristic equations at different state levels.</p>

<h5>SLE for Synergistic EA (SEA) and Conditional EA (CEA) model (Step 4a).</h5><a id="article1.body1.sec4.sec2.sec3.p1" name="article1.body1.sec4.sec2.sec3.p1"></a><p>A non-linear binding activity event is modeled via slight modification of minterm on Eq. (3). Eq. (4) demonstrate SLE for synergy effect on EA binding sites and Eq. (5) for conditional EA (when the same transcription activation causes different state transitions, there is the presence of a so-called <em>conditional effect</em>).<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>Synergistic effect, from characteristic equation (<a href="#pone-0000776-t007">Table 7</a>), is observed in (01<sub>2</sub>→11<sub>2</sub>) state transition. The activation of A alone increases the state by one step (10<sub>2</sub>): A is an activating site. On the other hand, the deactivation of A alone decreases the state by one step (00<sub>2</sub>): the activation of E alone has no effect. This suggests that E and A act in <em>synergy</em> at 01<sub>2</sub> state.</p>
<div class="figure" id="pone-0000776-t007"><div class="img"><a name="pone-0000776-t007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t007.TIF"></span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Synergistic Enhancer-activator model.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t007</span></div><a id="article1.body1.sec4.sec2.sec3.p2" name="article1.body1.sec4.sec2.sec3.p2"></a><p>In conditional Enhancer-activator model (Eq. (5)), the activation of E and A show synergistic as well as repressive effect. Eq. (5) illustrates that E and A are synergistic at 01<sub>2</sub> state (same characteristic equation derived from Eq. (4), <a href="#pone-0000776-t007">Table 7</a> and <a href="#pone-0000776-t008">Table 8</a>). At present state, 11<sub>2</sub>, characteristic equation (<em>q<sub>1,t+1</sub></em> = <em>A</em> and <em>q<sub>0,t+1</sub></em> = <em>E'A</em>) shows repressive effect: activation of both E and A generates (11<sub>2</sub>→10<sub>2</sub>) transition. The repressive effect is clearly shown by the AND logic, <em>E'A</em>, since <em>E'A</em> is 1 only if <em>E</em> = 0 (off) and <em>A</em> = 1 (on). This suggests that some physiochemical change occur in the activation of E site as compared to activation of the synergistic transition. Therefore, our SLM can point changes arising from <em>conditional effects</em> such as physicochemical regulation of gene expression.</p>
<div class="figure" id="pone-0000776-t008"><div class="img"><a name="pone-0000776-t008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t008.TIF"></span>)
                </a></li></ul></div><p><strong>Table 8.  <span>Conditional Enhancer-activator model.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t008</span></div>
<h5>Control of temporal gene expression by varying E and A dynamically (Step 4b).</h5><a id="article1.body1.sec4.sec2.sec4.p1" name="article1.body1.sec4.sec2.sec4.p1"></a><p><em>In silico</em> mutagenesis of EA (Eq. (3)), SEA (Eq. (4)) and CEA model (Eq. (5)) are performed to demonstrate the effect of E and A site mutants on temporal gene expression profile (<a href="#pone-0000776-g007">Figure 7</a>). For <em>in-silico</em> mutagenesis of EA model, we consider the deactivation of binding sites by setting <em>E</em>, <em>A</em> or both to zero in Eq. (3). Setting <em>A</em> = 0 implies that all states remain at 00<sub>2</sub>. However, setting <em>E</em> = 0 means that 11<sub>2</sub> state cannot occur in the mutant expression profile (<a href="#pone-0000776-g007">Figure 7(A)</a>). This result shows that the temporal functional activity of the site E have an enhancer effect only at 10<sub>2</sub> state and above. Similarly, for <em>in-silico</em> mutagenesis of SEA model and CEA model, <a href="#pone-0000776-g007">Figure 7(B)</a> &amp; <a href="#pone-0000776-g007">Figure 7(C)</a> show the differential role of E (not always an enhancer) at different time points. This suggests <em>when</em> conditional effect occurs on E acting site through the 6. activity of transcription factor. Furthermore, <em>in-silico</em> mutagenesis of the three models clearly indicates <em>when</em> the mutant state transitions are occurred, which implies changes in rate of gene expression between the wild-type and mutant conditions.</p>
<div class="figure" id="pone-0000776-g007"><div class="img"><a name="pone-0000776-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span><em>In silico</em> mutagenesis: comparison of wild type and mutant expression profiles with 3 models.</span></strong></p><a id="article1.body1.sec4.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec4.sec2.sec4.fig1.caption1.p1"></a><p>(A) EA model shows E is an enhancer; (B) SEA model shows E is a synergistic enhancer and (C) CEA model reveals E is a synergistic enhancer as well as silencer. <a href="#pone-0000776-g007">Figure 7</a> also shows <em>when</em> those effects of the E site occur. Wild type profiles: (A)–(C) are generated by Eq. (3), (4) and (5) with given input condition: <em>E<sub>t</sub>A<sub>t</sub></em> = (10<sub>2</sub> 00<sub>2</sub> 01<sub>2</sub> 11<sub>2</sub> 10<sub>2</sub> 00<sub>2</sub> 01<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 01<sub>2</sub> 00<sub>2</sub> 00<sub>2</sub>), where <em>t</em> = t<sub>0</sub>–t<sub>11</sub>. The corresponding mutant profiles are obtained by setting <em>E<sub>t</sub></em> = 0 in input condition.</p>
<span>doi:10.1371/journal.pone.0000776.g007</span></div>
<h5>Forward and Reverse Mapping (Step 4b).</h5><a id="article1.body1.sec4.sec2.sec5.p1" name="article1.body1.sec4.sec2.sec5.p1"></a><p>Dynamic simulation for forward mapping can be performed with novel input conditions to infer a temporal gene expression (Step 4). For example, given the starting state is 00<sub>2</sub>, and if an arbitrary temporal input series (<em>E<sub>t</sub>A<sub>t</sub></em>) (t = 0, 1,..11) is given: (00<sub>2</sub> 01<sub>2</sub> 01<sub>2</sub> 01<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 00<sub>2</sub> 10<sub>2</sub> 10<sub>2</sub> 10<sub>2</sub>), substituting these conditions into Eq. (4), a new series of next state (<em>q<sub>1,t+1</sub> q<sub>0,t+1</sub></em>) (<em>t</em> = 0,1,..11) is produced: (00<sub>2</sub> 01<sub>2</sub> 10<sub>2</sub> 10<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 00<sub>2</sub> 00<sub>2</sub> 00<sub>2</sub> 00<sub>2</sub>). Forward mapping based on a complete SLM (a complete truth table) could provide a complete picture of how dynamic activation of <em>cis</em>-acting sites via temporal activity of transcription factors is able to control temporal gene expression. In order to illustrate the process of extraction of input condition by reverse mapping (Step 4), a state transition map representation for <a href="#pone-0000776-t005">Table 5</a> is constructed in <a href="#pone-0000776-g008">Figure 8(A)</a>. The state transition map describes the transition from a state at one time point to another state at the following time point under a particular input condition. It is possible that forward mapping points to more than one time series of input conditions being generated by reverse mapping. In fact, in our case, there are 64 possible combinations of input series are suggested (<a href="#pone-0000776-g008">Figure 8(B)</a>), each one able to generate expression profiles given by forward mapping. The multiple input series could account for system robustness, given that it allows for more than one time series of input conditions to achieve the same temporal state transition. Reverse mapping can be used to infer transactivity on <em>cis</em>-acting sites from temporal gene expression at a time interval of interest.</p>
<div class="figure" id="pone-0000776-g008"><div class="img"><a name="pone-0000776-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Construction of state map representation and reverse mapping of EA model.</span></strong></p><a id="article1.body1.sec4.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec4.sec2.sec5.fig1.caption1.p1"></a><p>(A) The state transition map (finite state machine), constructed using truth table (<a href="#pone-0000776-t005">Table 5</a>), consists of four states (purple circles), 10 state transitions (in arrow) and 16 input conditions (binary values associated to each arrow). (B) Using the state transition map in (a) the temporal binding activity corresponding digitized gene expression profile (00<sub>2</sub> 01<sub>2</sub> 10<sub>2</sub> 10<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 11<sub>2</sub> 00<sub>2</sub> 00<sub>2</sub> 00<sub>2</sub> 00<sub>2</sub>) is inferred (reverse mapping). There are 11 state transitions in 12 time steps. Following the 12 arrows in state transition map, there are 2 possible inputs (01, 11) for each of the first two transitions (00<sub>2</sub>→01<sub>2</sub>→10<sub>2</sub>), 1 possible input for the next five transitions and 2 possible inputs for the final four transitions. In total, 64 possible combinations of input series are suggested where these input series are able to generate expression profile given by forward mapping.</p>
<span>doi:10.1371/journal.pone.0000776.g008</span></div>


<h4>Incomplete sequential logic model: <em>don't care condition</em></h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Thus far four binary variables: <em>q<sub>1</sub></em>, <em>q<sub>0</sub></em>, <em>E</em> and <em>A</em> are considered in truth table construction and sixteen different state transitions are required for the construction of complete truth table. In practical, there will be cases where some of the state transitions are yet to be obtained from experimental mRNA expression profiles and are regarded as don't care conditions <a href="#pone.0000776-Mano1">[35]</a>. For don't care condition, the next state of the state transitions is conventionally set to 00<sub>2</sub>. Existing state transitions from incomplete truth table, however, are not affected by additional state transition in don't care condition: additional information in don't care condition only gives rise to state transition with a new set of present state and input condition, which does not overlapped with present state and input condition of existing state transitions. Therefore, characteristic equations corresponding to existing state transitions are invariant under additional information in don't care condition. However, for forward mapping and <em>in silico</em> mutagenesis, if present state and input condition for don't care condition become current mapping condition, then the remaining time series is unreliable. In reverse mapping, if next state is 00<sub>2</sub>, it is also not possible to distinguish present input conditions derived from don't care condition and existing state transition.</p>


<h4>Synthetic network motifs (Step 5)</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We have discussed the systematic construction and analyses of single SLM. In this section, we demonstrate that combination of two connected SLMs which termed as <em>network motif</em> shows the control of gene regulation at network level (<a href="#pone-0000776-g009">Figure 9</a> and <a href="#pone-0000776-t009">Table 9</a>). A network motif is defined as pattern or architecture of connectivity in gene network that consists of specific regulatory function and recur significantly more often than randomized network <a href="#pone.0000776-Milo1">[36]</a>, <a href="#pone.0000776-Prill1">[37]</a>, <a href="#pone.0000776-YegerLotem1">[38]</a>. Furthermore, we consider the possibility where the input condition (e.g. <em>E</em> and <em>A</em>) of SLM is a function of its output state (e.g. <em>q<sub>1,t+1</sub></em>,<em>q<sub>0,t+1</sub></em>), i.e. modeling of auto-regulative motif. For example, if we consider a positive feed-forward motif where one of the inputs of the second EA model is the output of the first EA model, we have two model equations systems for each EA model defined as the following:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e028&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e029&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>In the case where <em>E<sub>2</sub></em> is a function of the output of the first EA model, <em>E<sub>2</sub></em> is written as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e030&amp;representation=PNG" class="inline-graphic"></span>. For the 1-bit input <em>E<sub>2</sub></em> has 2 values: 0 and 1; and for 2-bits input of the first EA model can have 4 values: 00, 01, 10, and 11. The function <em>f</em> determines the output state of the first EA model which corresponds to the input state of E<sub>2</sub>. M1 and M2 can be considered as one single block (M1–M2) via <em>f</em> mapping. And we can determine the equation model for the composite of the two EA models:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.e031&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br> In general, the same principle can be applied to construct various complexities of gene network motifs. (<a href="#pone-0000776-g009">Figure 9</a> and <a href="#pone-0000776-t009">Table 9</a>).</p>
<div class="figure" id="pone-0000776-g009"><div class="img"><a name="pone-0000776-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Construction of gene network via SLMs: Synthetic network motifs.</span></strong></p><a id="article1.body1.sec4.sec4.fig1.caption1.p1" name="article1.body1.sec4.sec4.fig1.caption1.p1"></a><p>(A)–(D) Four sets of SLMs that resemble four fundamental units are considered for construction of multiple gene network motifs (<a href="#pone-0000776-t009">Table 9</a>): EA/SEA models (positive regulation, set 1); Silencer-activator (SA)/Synergistic SA models (repressive regulation, set 2); By considering input condition of E site for set 1 as a function of its output state, an auto-regulative positive feedback is constructed; we denote this set as Simple positive feedback (set 3). Similarly, a silencer version of unit network motif for auto-regulative feedback, Simple negative feedback, is formed by considering input condition of S site for set 2 as a function of its output state (set 4). (E)–(I) To extend the single SLM approach to genes network, several combinations of two SLMs representing two genes, each regulated by two <em>cis</em>-acting sites are constructed based on the four set of fundamental units albeit with auto-regulative feedback replaced by inter-genetic feedback control (<a href="#pone-0000776-t009">Table 9</a>). In addition, all of these combinations are co-regulated networks with common activator site (A/A1). A co-regulated network is defined as set of genes that contained at least one common active <em>cis</em>-acting site. The simplest form of co-regulated network occurred when no direct connectivity found between two SLMs where Both <em>Gene1</em> and <em>Gene2</em> are linked by their common regulatory sites: E and A ((E)). Non-auto-regulative feedforward and feedback control can occur if <em>Gene1</em> and <em>Gene2</em> are connected by at least one of the gene product. Thus, in addition to intra-genetic-feedback control that happened in single SLM ((C)–(D)), network motifs constructed by multiple SLMs can exhibit inter-genetic-feedforward and feedback control. In inter-genetic-feedback control, the input condition of a SLM is now a function of output state from another SLM. Given that output signal from <em>Gene2</em> is function as feedback signal onto <em>Gene1</em>, the input condition of <em>Gene1</em>, <em>X</em>1, is defined as a function of output state of <em>Gene2</em>: <em>X</em>1 = <em>X</em>1(<em>q<sub>1,t+1,Gene2</sub></em>, <em>q<sub>0,t+1, Gene2</sub></em>) where <em>X1</em> can be either an enhancer or an silencer <em>Gene1</em> ((H)–(I)). Inter-genetic-feedforward control is defined similarly to inter-feedback control where <em>X</em>2 = <em>X</em>2(<em>q<sub>1,t+1,Gene1</sub></em>, <em>q<sub>0,t+1, Gene1</sub></em>) since we consider output signal from <em>Gene1</em> to be the feedforward signal onto <em>Gene2</em> ((F)–(G)).</p>
<span>doi:10.1371/journal.pone.0000776.g009</span></div><div class="figure" id="pone-0000776-t009"><div class="img"><a name="pone-0000776-t009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000776" data-uri="info:doi/10.1371/journal.pone.0000776.t009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000776.t009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000776.t009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000776.t009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000776.t009.TIF"></span>)
                </a></li></ul></div><p><strong>Table 9.  <span>Synthetic network motifs constructed by SLMs.</span></strong></p><span>doi:10.1371/journal.pone.0000776.t009</span></div>

<h4>Biological data</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The temporal <em>endo16</em> expression profiles were quantitated via CAT (chloramphenicol acetyltransferase) reporter assays <a href="#pone.0000776-Yuh1">[9]</a>, <a href="#pone.0000776-Yuh2">[10]</a>, <a href="#pone.0000776-Yuh5">[39]</a>, <a href="#pone.0000776-Yuh6">[40]</a>. The method for the generation of <em>endo16</em> expression profile had been well described <a href="#pone.0000776-Yuh2">[10]</a>, <a href="#pone.0000776-Yuh5">[39]</a>, <a href="#pone.0000776-Yuh6">[40]</a>. In these experiments, a 2.3 kb region in the promoter of <em>endo16</em> is cloned along with a CAT reporter gene. This promoter region is chosen as it is able to represent the full expression pattern of <em>endo16</em> gene in sea urchin development <a href="#pone.0000776-Yuh2">[10]</a>, <a href="#pone.0000776-Yuh5">[39]</a>, <a href="#pone.0000776-Yuh6">[40]</a>. The function and organisation of <em>endo16</em> regulatory sequences in the region were well characterized <a href="#pone.0000776-Yuh1">[9]</a>, <a href="#pone.0000776-Yuh2">[10]</a>, <a href="#pone.0000776-Yuh3">[18]</a>, <a href="#pone.0000776-Yuh5">[39]</a>, <a href="#pone.0000776-Yuh6">[40]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000776"></div><a name="pone.0000776.s001" id="pone.0000776.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000776.s001">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000776.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000776.s002" id="pone.0000776.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000776.s002">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000776.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000776.s003" id="pone.0000776.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000776.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Figures obtained from <a href="#pone.0000776-Yuh2">[10]</a>. (A) Effect of mutation of the R site in BA-Bp⋅CAT. The timecourse of expression of this construct (B(Rm)A-Bp⋅CAT, orange curve) is compared with that of the BA-Bp⋅CAT control (red), and to that of B-Bp⋅CAT (green) and A-Bp⋅CAT (blue). (B) Same input condition and present state generates different state transitions by B(UIm)A(Otxm)-Bp⋅CAT. The timecourse of expression of the double mutation B(UIm)A(Otxm)-Bp⋅CAT (black) is compared with BA-Bp⋅CAT (red) and the triple mutation B(UIm,CB2m)A(Otxm)-Bp⋅CAT (magenta). From the profile of B(UIm)A(Otxm)-Bp⋅CAT, given that 10<sub>2</sub> as present state and 010<sub>2</sub> as input condition of UI, R and Otx, two different next states, 10<sub>2</sub> at 25–40 pfh and 01<sub>2</sub> at 50–61 pfh are mapped. (C) Successive pathways of spatial and temporal control within the <em>endo16 cis</em>-regulatory system. The diagram summarizes results from several previous studies (Yuh <em>et al</em>., 1996; Yuh <em>et al</em>., 1998; Yuh and Davidson, 1996). Module (Mod) A functions are shown in red; Module B functions in blue. Early in development the <em>endo16</em> gene responds to a ubiquitous activator (SpOtx1) binding in Module A, but in order to achieve accurate spatial expression, activity must be extinguished outside the veg2 endomesodermal domain by repressors binding in the upstream modules (F, E and DC). Later in development, the activity of a transcriptional regulator (UI) binding in Module B rises and the internal BA intermodule input switch shuts off Otx input so that the system is now driven only by Module B input. This input is amplified in Module A, which provides the sole communication with the basal transcription apparatus. (D) Direct demonstration of repression function mediated by the R target site. The timecourse generated by a construct consisting of three copies of an oligonucleotide that represents the R target site (<a href="#pone-0000776-g003">Figure 3</a>) ligated to A-Bp⋅CAT ([R3]A-Bp⋅CAT, orange curve) is compared with the timecourse of A-Bp⋅CAT, blue. (E) Additional mutation of UI in a BA construct carrying the R mutation also negates the effect of the R mutation. An average of two experiments is shown in which the output of B(UIm,Rm)A-Bp⋅CAT (green) is seen to be very similar to that of B(UIm)A-Bp⋅CAT (purple). That is, though the R sites is intact in B(UIm)A-Bp⋅CAT, it fails to repress the activity of Module A in this context. As controls, BA-Bp⋅CAT (red) and A-Bp⋅CAT (blue) are shown in normal embryos of the same batch of eggs. (F) The UI site alone produces the late rise in expression. Output kinetics are shown from an experiment in which only the UI site has been left intact (construct [UI]-Bp+⋅CAT of <a href="#pone-0000776-t001">Table 1</a>) so that it alone provides regulatory input into the enhanced basal promoter (black). This construct produces exactly the same output as does [UI-R-CB2]-Bp+⋅CAT (orange), again generating the late rise in expression. However, the output is of low magnitude relative to that of BA-Bp⋅CAT: the inset shows these data (i.e. for [UI]-Bp+⋅CAT) at reduced scale, to also indicate BA-Bp⋅CAT output in the same experiment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000776.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(2.59 MB DOC)</p>
<a name="pone.0000776.s004" id="pone.0000776.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000776.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Figures obtained from <a href="#pone.0000776-Yuh1">[9]</a>. (A) Effect of BA modules during the absent of Otx. BA(Otx)-Bp⋅CAT (black) (note that (Otx) is equivalent to (Otxm) in <a href="#pone.0000776-Yuh2">[10]</a>) is compared with BA-Bp⋅CAT (red). (B) Effect of Otx on temporal expression. A(Otx)-Bp⋅CAT (red) is compared with A-Bp⋅CAT (dark blue) and OtxZ-Bp⋅CAT (marine blue).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000776.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.15 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Prof. Eric Davidson for discussion in the <em>endo16 cis</em>-regulatory system. We thank Prof Kanaga Sabapathy and Dr. Kenneth Lee for their critical review of our work in both computational and biological aspects. We also thank Mr. Li Bin for sharing his time for discussion.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SW MT ZY. Performed the experiments: SA MT ZY. Analyzed the data: SA AG SW MT ZY VP KS MT. Contributed reagents/materials/analysis tools: SA MT ZY VP MT. Wrote the paper: AG SW MT ZY VP KS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000776-Bintu1" id="pone.0000776-Bintu1"></a>Bintu L, Buchler NE, Garcia HG, Gerland U, Hwa T, et al.  (2005) Transcriptional regulation by the numbers: applications. Curr Opin Genet Dev  15: 125–135.  <ul class="find" data-citedArticleID="980984" data-doi="10.1016/j.gde.2005.02.006"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.02.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulation+by+the+numbers%3A+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulation+by+the+numbers%3A+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000776-Bintu2" id="pone.0000776-Bintu2"></a>Bintu L, Buchler NE, Garcia HG, Gerland U, Hwa T, et al.  (2005) Transcriptional regulation by the numbers: models. Curr Opin Genet Dev  15: 116–124.  <ul class="find" data-citedArticleID="980986" data-doi="10.1016/j.gde.2005.02.007"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.02.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulation+by+the+numbers%3A+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulation+by+the+numbers%3A+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000776-Wolf1" id="pone.0000776-Wolf1"></a>Wolf DM, Eeckman FH (1998) On the relationship between genomic regulatory element organization and gene regulatory dynamics. J Theor Biol  195: 167–186.  <ul class="find" data-citedArticleID="981040" data-doi="10.1006/jtbi.1998.0790"><li><a href="http://dx.doi.org/10.1006/jtbi.1998.0790" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+relationship+between+genomic+regulatory+element+organization+and+gene+regulatory+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+relationship+between+genomic+regulatory+element+organization+and+gene+regulatory+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000776-Elowitz1" id="pone.0000776-Elowitz1"></a>Elowitz MB, Leibler S (2000) A synthetic oscillatory network of transcriptional regulators. Nature  403: 335–338.  <ul class="find" data-citedArticleID="981008" data-doi="10.1038/35002125"><li><a href="http://dx.doi.org/10.1038/35002125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+synthetic+oscillatory+network+of+transcriptional+regulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+synthetic+oscillatory+network+of+transcriptional+regulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000776-Hasty1" id="pone.0000776-Hasty1"></a>Hasty J, McMillen D, Isaacs F, Collins JJ (2001) Computational studies of gene regulatory networks: in numero molecular biology. Nat Rev Genet  2: 268–279.  <ul class="find" data-citedArticleID="981016" data-doi="10.1038/35066056"><li><a href="http://dx.doi.org/10.1038/35066056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+studies+of+gene+regulatory+networks%3A+in+numero+molecular+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+studies+of+gene+regulatory+networks%3A+in+numero+molecular+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000776-Wray1" id="pone.0000776-Wray1"></a>Wray GA, Hahn MW, Abouheif E, Balhoff JP, Pizer M, et al.  (2003) The evolution of transcriptional regulation in eukaryotes. Mol Biol Evol  20: 1377–1419.  <ul class="find" data-citedArticleID="981042"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+evolution+of+transcriptional+regulation+in+eukaryotes.&amp;auth=&amp;atitle=The+evolution+of+transcriptional+regulation+in+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+transcriptional+regulation+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+transcriptional+regulation+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000776-Harbison1" id="pone.0000776-Harbison1"></a>Harbison CT, Gordon DB, Lee TI, Rinaldi NJ, Macisaac KD, et al.  (2004) Transcriptional regulatory code of a eukaryotic genome. Nature  431: 99–104.  <ul class="find" data-citedArticleID="981014" data-doi="10.1038/nature02800"><li><a href="http://dx.doi.org/10.1038/nature02800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+code+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+code+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000776-Setty1" id="pone.0000776-Setty1"></a>Setty Y, Mayo AE, Surette MG, Alon U (2003) Detailed map of a cis-regulatory input function. Proc Natl Acad Sci U S A  100: 7702–7707.  <ul class="find" data-citedArticleID="981034" data-doi="10.1073/pnas.1230759100"><li><a href="http://dx.doi.org/10.1073/pnas.1230759100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detailed+map+of+a+cis-regulatory+input+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detailed+map+of+a+cis-regulatory+input+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000776-Yuh1" id="pone.0000776-Yuh1"></a>Yuh CH, Bolouri H, Davidson EH (1998) Genomic cis-regulatory logic: experimental and computational analysis of a sea urchin gene. Science  279: 1896–1902.  <ul class="find" data-citedArticleID="981046" data-doi="10.1126/science.279.5358.1896"><li><a href="http://dx.doi.org/10.1126/science.279.5358.1896" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+cis-regulatory+logic%3A+experimental+and+computational+analysis+of+a+sea+urchin+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+cis-regulatory+logic%3A+experimental+and+computational+analysis+of+a+sea+urchin+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000776-Yuh2" id="pone.0000776-Yuh2"></a>Yuh CH, Bolouri H, Davidson EH (2001) Cis-regulatory logic in the endo16 gene: switching from a specification to a differentiation mode of control. Development  128: 617–629.  <ul class="find" data-citedArticleID="981048"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cis-regulatory+logic+in+the+endo16+gene%3A+switching+from+a+specification+to+a+differentiation+mode+of+control.&amp;auth=&amp;atitle=Cis-regulatory+logic+in+the+endo16+gene%3A+switching+from+a+specification+to+a+differentiation+mode+of+control." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cis-regulatory+logic+in+the+endo16+gene%3A+switching+from+a+specification+to+a+differentiation+mode+of+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cis-regulatory+logic+in+the+endo16+gene%3A+switching+from+a+specification+to+a+differentiation+mode+of+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000776-Davidson1" id="pone.0000776-Davidson1"></a>Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, et al.  (2002) A genomic regulatory network for development. Science  295: 1669–1678.  <ul class="find" data-citedArticleID="981002" data-doi="10.1126/science.1069883"><li><a href="http://dx.doi.org/10.1126/science.1069883" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+regulatory+network+for+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+regulatory+network+for+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000776-Bolouri1" id="pone.0000776-Bolouri1"></a>Bolouri H, Davidson EH (2002) Modelling DNA sequence-based cis-regulatory gene networks. Dev Biol  246: 2–13.  <ul class="find" data-citedArticleID="980990" data-doi="10.1006/dbio.2002.0617"><li><a href="http://dx.doi.org/10.1006/dbio.2002.0617" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modelling+DNA+sequence-based+cis-regulatory+gene+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modelling+DNA+sequence-based+cis-regulatory+gene+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000776-Buchler1" id="pone.0000776-Buchler1"></a>Buchler NE, Gerland U, Hwa T (2003) On schemes of combinatorial transcription logic. Proc Natl Acad Sci U S A  100: 5136–5141.  <ul class="find" data-citedArticleID="980996" data-doi="10.1073/pnas.0930314100"><li><a href="http://dx.doi.org/10.1073/pnas.0930314100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+schemes+of+combinatorial+transcription+logic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+schemes+of+combinatorial+transcription+logic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000776-Cheadle1" id="pone.0000776-Cheadle1"></a>Cheadle C, Fan J, Cho-Chung YS, Werner T, Ray J, et al.  (2005) Control of gene expression during T cell activation: alternate regulation of mRNA transcription and mRNA stability. BMC Genomics  6(1): 75.  <ul class="find" data-citedArticleID="981000"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Control+of+gene+expression+during+T+cell+activation%3A+alternate+regulation+of+mRNA+transcription+and+mRNA+stability.&amp;auth=&amp;atitle=Control+of+gene+expression+during+T+cell+activation%3A+alternate+regulation+of+mRNA+transcription+and+mRNA+stability." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+gene+expression+during+T+cell+activation%3A+alternate+regulation+of+mRNA+transcription+and+mRNA+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+gene+expression+during+T+cell+activation%3A+alternate+regulation+of+mRNA+transcription+and+mRNA+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000776-Gerber1" id="pone.0000776-Gerber1"></a>Gerber AP, Herschlag D, Brown PO (2004) Extensive Association of Functionally and Cytotopically Related mRNAs with Puf Family RNA-Binding Proteins in Yeast. PLoS Biol  2(3): E79.  <ul class="find" data-citedArticleID="981010" data-doi="10.1371/journal.pbio.0020079"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+Association+of+Functionally+and+Cytotopically+Related+mRNAs+with+Puf+Family+RNA-Binding+Proteins+in+Yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+Association+of+Functionally+and+Cytotopically+Related+mRNAs+with+Puf+Family+RNA-Binding+Proteins+in+Yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000776-Stathopoulos1" id="pone.0000776-Stathopoulos1"></a>Stathopoulos A, Levine M (2005) Genomic regulatory networks and animal development. Dev Cell  9(4): 449–462.  <ul class="find" data-citedArticleID="981036" data-doi="10.1016/j.devcel.2005.09.005"><li><a href="http://dx.doi.org/10.1016/j.devcel.2005.09.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+regulatory+networks+and+animal+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+regulatory+networks+and+animal+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000776-Arias1" id="pone.0000776-Arias1"></a>Arias AM, Hayward P (2006) Filtering transcriptional noise during development: concepts and mechanisms. Nature Rev Genet  7(1): 34–44.  <ul class="find" data-citedArticleID="980980" data-doi="10.1038/nrg1750"><li><a href="http://dx.doi.org/10.1038/nrg1750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filtering+transcriptional+noise+during+development%3A+concepts+and+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Filtering+transcriptional+noise+during+development%3A+concepts+and+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000776-Yuh3" id="pone.0000776-Yuh3"></a>Yuh CH, Dorman ER, Davidson EH (2005) Brn1/2/4, the predicted midgut regulator of the endo16 gene of the sea urchin embryo. Dev Biol  281: 286–298.  <ul class="find" data-citedArticleID="981050" data-doi="10.1016/j.ydbio.2005.02.034"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2005.02.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brn1%2F2%2F4%2C+the+predicted+midgut+regulator+of+the+endo16+gene+of+the+sea+urchin+embryo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Brn1%2F2%2F4%2C+the+predicted+midgut+regulator+of+the+endo16+gene+of+the+sea+urchin+embryo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000776-Yuh4" id="pone.0000776-Yuh4"></a>Yuh CH, Dorman ER, Howard ML, Davidson EH (2004) An otx cis-regulatory module: a key node in the sea urchin endomesoderm gene regulatory network. Dev Biol  269: 536–551.  <ul class="find" data-citedArticleID="981052" data-doi="10.1016/j.ydbio.2004.02.025"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2004.02.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+otx+cis-regulatory+module%3A+a+key+node+in+the+sea+urchin+endomesoderm+gene+regulatory+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+otx+cis-regulatory+module%3A+a+key+node+in+the+sea+urchin+endomesoderm+gene+regulatory+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000776-Blasi1" id="pone.0000776-Blasi1"></a>Blasi MF, Casorellia I, Colosimob A, Blasic FS, Bignamia M (2005) Recursive network approach can identify constitutive regulatory circuits in gene expression data. Physica A  348: 349–370.  <ul class="find" data-citedArticleID="980988" data-doi="10.1016/j.physa.2004.09.005"><li><a href="http://dx.doi.org/10.1016/j.physa.2004.09.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recursive+network+approach+can+identify+constitutive+regulatory+circuits+in+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recursive+network+approach+can+identify+constitutive+regulatory+circuits+in+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000776-DiCamillo1" id="pone.0000776-DiCamillo1"></a>Di Camillo B, Sanchez-Cabo F, Toffolo G, Nair SK, Trajanoski Z, et al.  (2005) A quantization method based on threshold optimization for microarray short time series. BMC Bioinformatics  1: 6 Suppl 4S11.  <ul class="find" data-citedArticleID="981004" data-doi="10.1186/1471-2105-6-s4-s11"><li><a href="http://dx.doi.org/10.1186/1471-2105-6-s4-s11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+quantization+method+based+on+threshold+optimization+for+microarray+short+time+series." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+quantization+method+based+on+threshold+optimization+for+microarray+short+time+series.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000776-Barrett1" id="pone.0000776-Barrett1"></a>Barrett CL, Herring CD, Reed JL, Palsson BO (2005) The global transcriptional regulatory network for metabolism in Escherichia coli exhibits few dominant functional states. Proc Natl Acad Sci U S A  102: 19103–19108.  <ul class="find" data-citedArticleID="980982" data-doi="10.1073/pnas.0505231102"><li><a href="http://dx.doi.org/10.1073/pnas.0505231102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+global+transcriptional+regulatory+network+for+metabolism+in+Escherichia+coli+exhibits+few+dominant+functional+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+global+transcriptional+regulatory+network+for+metabolism+in+Escherichia+coli+exhibits+few+dominant+functional+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000776-Bolouri2" id="pone.0000776-Bolouri2"></a>Bolouri H, Davidson EH (2003) Transcriptional regulatory cascades in development: initial rates, not steady state, determine network kinetics. Proc Natl Acad Sci U S A  100: 9371–9376.  <ul class="find" data-citedArticleID="980992" data-doi="10.1073/pnas.1533293100"><li><a href="http://dx.doi.org/10.1073/pnas.1533293100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+cascades+in+development%3A+initial+rates%2C+not+steady+state%2C+determine+network+kinetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+cascades+in+development%3A+initial+rates%2C+not+steady+state%2C+determine+network+kinetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000776-Istrail1" id="pone.0000776-Istrail1"></a>Istrail S, Davidson EH (2005) Logic functions of the genomic cis-regulatory code. Proc Natl Acad Sci U S A  102: 4954–4959.  <ul class="find" data-citedArticleID="981020" data-doi="10.1073/pnas.0409624102"><li><a href="http://dx.doi.org/10.1073/pnas.0409624102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Logic+functions+of+the+genomic+cis-regulatory+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Logic+functions+of+the+genomic+cis-regulatory+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000776-Bulyk1" id="pone.0000776-Bulyk1"></a>Bulyk ML, Johnson PLF, Church GM (2002) Nucleotides of transcription factor binding sites exert interdependent effects on the binding affinities of transcription factors. Nucleic Acids Res  30: 1255–1261.  <ul class="find" data-citedArticleID="980998" data-doi="10.1093/nar/30.5.1255"><li><a href="http://dx.doi.org/10.1093/nar/30.5.1255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotides+of+transcription+factor+binding+sites+exert+interdependent+effects+on+the+binding+affinities+of+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotides+of+transcription+factor+binding+sites+exert+interdependent+effects+on+the+binding+affinities+of+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000776-Kramer1" id="pone.0000776-Kramer1"></a>Kramer BP, Fussenegger M (2005) Hysteresis in a synthetic mammalian gene network. Proc Natl Acad Sci U S A  102: 9517–9522.  <ul class="find" data-citedArticleID="981022" data-doi="10.1073/pnas.0500345102"><li><a href="http://dx.doi.org/10.1073/pnas.0500345102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hysteresis+in+a+synthetic+mammalian+gene+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hysteresis+in+a+synthetic+mammalian+gene+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000776-Angeli1" id="pone.0000776-Angeli1"></a>Angeli D, Ferrell JE, Sontag ED (2004) Detection of multistability, bifurcations, and hysteresis in a large class of biological positive-feedback systems. Proc Natl Acad Sci U S A  101: 1822–1827.  <ul class="find" data-citedArticleID="980978" data-doi="10.1073/pnas.0308265100"><li><a href="http://dx.doi.org/10.1073/pnas.0308265100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+multistability%2C+bifurcations%2C+and+hysteresis+in+a+large+class+of+biological+positive-feedback+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+multistability%2C+bifurcations%2C+and+hysteresis+in+a+large+class+of+biological+positive-feedback+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000776-Giuliani1" id="pone.0000776-Giuliani1"></a>Giuliani A, Lo Giudice P, Mancini AM, Quatrini G, Pacifici L, Webber CL, Zak M, Zbilut JP (1996) A Markovian formalization of heart rate dynamics evinces a quantum-like hypothesis. Biol Cybern  74: 181–187.  <ul class="find" data-citedArticleID="981012" data-doi="10.1007/bf00204206"><li><a href="http://dx.doi.org/10.1007/bf00204206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Markovian+formalization+of+heart+rate+dynamics+evinces+a+quantum-like+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Markovian+formalization+of+heart+rate+dynamics+evinces+a+quantum-like+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000776-Martin1" id="pone.0000776-Martin1"></a>Martin S, Zhang Z, Martino A, Faulon JL (2007) Boolean dynamics of genetic regulatory networks inferred from microarray time series data. Bioinformatics  23: 866–74.  <ul class="find" data-citedArticleID="981026" data-doi="10.1093/bioinformatics/btm021"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Boolean+dynamics+of+genetic+regulatory+networks+inferred+from+microarray+time+series+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Boolean+dynamics+of+genetic+regulatory+networks+inferred+from+microarray+time+series+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000776-Djordjevic1" id="pone.0000776-Djordjevic1"></a>Djordjevic M, Sengupta AM, Shraiman BI (2003) A biophysical approach to transcription factor binding site discovery. Genome Res  13: 2381–2390.  <ul class="find" data-citedArticleID="981006" data-doi="10.1101/gr.1271603"><li><a href="http://dx.doi.org/10.1101/gr.1271603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+biophysical+approach+to+transcription+factor+binding+site+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+biophysical+approach+to+transcription+factor+binding+site+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000776-Hughes1" id="pone.0000776-Hughes1"></a>Hughes JD, Estep PW, Tavazoie S, Church GM (2000) Computational identification of cis-regulatory elements associated with groups of functionally related genes in Saccharomyces cerevisiae. J Mol Biol  296: 1205–14.  <ul class="find" data-citedArticleID="981018" data-doi="10.1006/jmbi.2000.3519"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+cis-regulatory+elements+associated+with+groups+of+functionally+related+genes+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+cis-regulatory+elements+associated+with+groups+of+functionally+related+genes+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000776-Tamada1" id="pone.0000776-Tamada1"></a>Tamada Y, Kim S, Bannai H, Imoto S, Tashiro K, et al.  (2003) Estimating gene networks from gene expression data by combining Bayesian network model with promoter element detection. Bioinformatics  19: Suppl 2II227–II236.  <ul class="find" data-citedArticleID="981038" data-doi="10.1093/bioinformatics/btg1082"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg1082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+gene+networks+from+gene+expression+data+by+combining+Bayesian+network+model+with+promoter+element+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+gene+networks+from+gene+expression+data+by+combining+Bayesian+network+model+with+promoter+element+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000776-Ruklisa1" id="pone.0000776-Ruklisa1"></a>Ruklisa D, Brazma A, Viksna J (2005) Reconstruction of Gene Regulatory Networks under the Finite State Linear Model. Genome Informatics  16(2): 225–236.  <ul class="find" data-citedArticleID="981032"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Reconstruction+of+Gene+Regulatory+Networks+under+the+Finite+State+Linear+Model.&amp;auth=&amp;atitle=Reconstruction+of+Gene+Regulatory+Networks+under+the+Finite+State+Linear+Model." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstruction+of+Gene+Regulatory+Networks+under+the+Finite+State+Linear+Model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstruction+of+Gene+Regulatory+Networks+under+the+Finite+State+Linear+Model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000776-Brazma1" id="pone.0000776-Brazma1"></a>Brazma A, Schlitt T (2003) Genome Biology Reverse engineering of gene regulatory networks: a finite state linear model.  4:P5 doi:10.1186/gb-2003-4-6-p5.  <ul class="find" data-citedArticleID="980994"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+Biology+Reverse+engineering+of+gene+regulatory+networks%3A+a+finite+state+linear+model.&amp;auth=&amp;atitle=Genome+Biology+Reverse+engineering+of+gene+regulatory+networks%3A+a+finite+state+linear+model." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+Biology+Reverse+engineering+of+gene+regulatory+networks%3A+a+finite+state+linear+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+Biology+Reverse+engineering+of+gene+regulatory+networks%3A+a+finite+state+linear+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000776-Mano1" id="pone.0000776-Mano1"></a>Mano MM (2001) Digital Design. Prentice Hall PTR.   <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0000776-Milo1" id="pone.0000776-Milo1"></a>Milo R, Shen-Orr S, Itzkovitz S, Kashtan N, Chklovskii D, et al.  (2002) Network motifs: simple building blocks of complex networks. Science  298: 824–827.  <ul class="find" data-citedArticleID="981028" data-doi="10.1126/science.298.5594.824"><li><a href="http://dx.doi.org/10.1126/science.298.5594.824" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+motifs%3A+simple+building+blocks+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+motifs%3A+simple+building+blocks+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000776-Prill1" id="pone.0000776-Prill1"></a>Prill RJ, Iglesias PA, Levchenko A (2005) Dynamic properties of network motifs contribute to biological network organization. PLoS Biol  3: 1881–1892.  <ul class="find" data-citedArticleID="981030" data-doi="10.1371/journal.pbio.0030343"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+properties+of+network+motifs+contribute+to+biological+network+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+properties+of+network+motifs+contribute+to+biological+network+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000776-YegerLotem1" id="pone.0000776-YegerLotem1"></a>Yeger-Lotem E, Sattath S, Kashtan N, Itzkovitz S, Milo R (2004) Network motifs in integrated cellular networks of transcription–regulation and protein–protein interaction. Proc Natl Acad Sci U S A  101: 5934–5939.  <ul class="find" data-citedArticleID="981044" data-doi="10.1073/pnas.0306752101"><li><a href="http://dx.doi.org/10.1073/pnas.0306752101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+motifs+in+integrated+cellular+networks+of+transcription%26%23x2013%3Bregulation+and+protein%26%23x2013%3Bprotein+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+motifs+in+integrated+cellular+networks+of+transcription%26%23x2013%3Bregulation+and+protein%26%23x2013%3Bprotein+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000776-Yuh5" id="pone.0000776-Yuh5"></a>Yuh CH, Davidson EH (1996) Modular cis-regulatory organization of Endo16, a gut-specific gene of the sea urchin embryo. Development  122: 1069–1082.  <ul class="find" data-citedArticleID="981054"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modular+cis-regulatory+organization+of+Endo16%2C+a+gut-specific+gene+of+the+sea+urchin+embryo.&amp;auth=&amp;atitle=Modular+cis-regulatory+organization+of+Endo16%2C+a+gut-specific+gene+of+the+sea+urchin+embryo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modular+cis-regulatory+organization+of+Endo16%2C+a+gut-specific+gene+of+the+sea+urchin+embryo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modular+cis-regulatory+organization+of+Endo16%2C+a+gut-specific+gene+of+the+sea+urchin+embryo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000776-Yuh6" id="pone.0000776-Yuh6"></a>Yuh CH, Moore JG, Davidson EH (1996) Quantitative functional interrelations within the cis-regulatory system of the S. purpuratus Endo16 gene. Development  122: 4045–4056.  <ul class="find" data-citedArticleID="981056"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quantitative+functional+interrelations+within+the+cis-regulatory+system+of+the+S.+purpuratus+Endo16+gene.&amp;auth=&amp;atitle=Quantitative+functional+interrelations+within+the+cis-regulatory+system+of+the+S.+purpuratus+Endo16+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+functional+interrelations+within+the+cis-regulatory+system+of+the+S.+purpuratus+Endo16+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+functional+interrelations+within+the+cis-regulatory+system+of+the+S.+purpuratus+Endo16+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.XML" value="169944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.PDF" value="1031505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g001.PNG_L" value="160329"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g001.PNG_M" value="122185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g001.PNG_S" value="9301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g001.TIF" value="440830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g001.PNG_I" value="43803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t001.PNG_L" value="129526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t001.PNG_M" value="73441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t001.PNG_S" value="10307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t001.TIF" value="595326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t001.PNG_I" value="22531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e001.PNG" value="8598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e001.TIF" value="25352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e002.PNG" value="7446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e002.TIF" value="23672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e003.PNG" value="8222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e003.TIF" value="24688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e004.PNG" value="9545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e004.TIF" value="25804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g002.PNG_L" value="281199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g002.PNG_M" value="136507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g002.PNG_S" value="13858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g002.TIF" value="1243150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g002.PNG_I" value="70993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e005.PNG" value="15747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e005.TIF" value="32756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t002.PNG_L" value="91624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t002.PNG_M" value="107172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t002.PNG_S" value="15975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t002.TIF" value="494770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t002.PNG_I" value="41271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t003.PNG_L" value="156589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t003.PNG_M" value="86937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t003.PNG_S" value="11143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t003.TIF" value="788682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t003.PNG_I" value="26133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e006.PNG" value="11766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e006.TIF" value="29460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e007.PNG" value="9664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e007.TIF" value="26240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e008.PNG" value="9939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e008.TIF" value="26060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e009.PNG" value="11336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e009.TIF" value="28392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e010.PNG" value="8342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e010.TIF" value="24852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e011.PNG" value="9666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e011.TIF" value="26240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g003.PNG_L" value="190027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g003.PNG_M" value="126160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g003.PNG_S" value="15823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g003.TIF" value="554666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g003.PNG_I" value="74837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t004.PNG_L" value="40919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t004.PNG_M" value="52502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t004.PNG_S" value="10747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t004.TIF" value="208608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t004.PNG_I" value="17788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t005.PNG_L" value="58558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t005.PNG_M" value="67681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t005.PNG_S" value="14586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t005.TIF" value="400910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t005.PNG_I" value="24458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e012.PNG" value="17827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e012.TIF" value="35008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e013.PNG" value="11609"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e013.TIF" value="27612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g004.PNG_L" value="122081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g004.PNG_M" value="129883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g004.PNG_S" value="11215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g004.TIF" value="278788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g004.PNG_I" value="49443"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g005.PNG_L" value="168730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g005.PNG_M" value="115160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g005.PNG_S" value="10277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g005.TIF" value="590378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g005.PNG_I" value="41875"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g006.PNG_L" value="147586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g006.PNG_M" value="106501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g006.PNG_S" value="8873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g006.TIF" value="419870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g006.PNG_I" value="38383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t006.PNG_L" value="122772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t006.PNG_M" value="71671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t006.PNG_S" value="10539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t006.TIF" value="647772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t006.PNG_I" value="22145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e014.PNG" value="7761"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e014.TIF" value="24140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e015.PNG" value="7790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e015.TIF" value="24200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e016.PNG" value="7378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e016.TIF" value="23540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e017.PNG" value="7390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e017.TIF" value="23540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e018.PNG" value="11490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e018.TIF" value="27392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e019.PNG" value="12233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e019.TIF" value="28212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t007.PNG_L" value="133663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t007.PNG_M" value="79548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t007.PNG_S" value="10800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t007.TIF" value="697878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t007.PNG_I" value="24082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e020.PNG" value="7733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e020.TIF" value="24220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e021.PNG" value="8057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e021.TIF" value="24552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e022.PNG" value="8087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e022.TIF" value="24548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e023.PNG" value="8094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e023.TIF" value="24544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t008.PNG_L" value="132760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t008.PNG_M" value="79463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t008.PNG_S" value="10890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t008.TIF" value="693102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t008.PNG_I" value="24047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e024.PNG" value="8163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e024.TIF" value="24772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e025.PNG" value="8087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e025.TIF" value="24584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e026.PNG" value="7791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e026.TIF" value="24360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e027.PNG" value="7758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e027.TIF" value="24416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g007.PNG_L" value="406312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g007.PNG_M" value="210098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g007.PNG_S" value="16805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g007.TIF" value="1289950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g007.PNG_I" value="82372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g008.PNG_L" value="189706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g008.PNG_M" value="116682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g008.PNG_S" value="9446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g008.TIF" value="377240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g008.PNG_I" value="43141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e028.PNG" value="11166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e028.TIF" value="27340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e029.PNG" value="11404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e029.TIF" value="27424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e030.PNG" value="8781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e030.TIF" value="24788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e031.PNG" value="17375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e031.TIF" value="34000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g009.PNG_L" value="1353993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g009.PNG_M" value="193157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g009.PNG_S" value="17375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g009.TIF" value="2965066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.g009.PNG_I" value="96098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t009.PNG_L" value="368057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t009.PNG_M" value="207776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t009.PNG_S" value="17366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t009.TIF" value="1543940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.t009.PNG_I" value="64792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e032.PNG" value="14906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e032.TIF" value="34732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e033.PNG" value="12080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e033.TIF" value="29060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e034.PNG" value="14795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e034.TIF" value="34704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e035.PNG" value="15132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e035.TIF" value="34028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e036.PNG" value="14567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e036.TIF" value="34648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e037.PNG" value="12425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e037.TIF" value="29900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e038.PNG" value="12657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e038.TIF" value="29956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e039.PNG" value="15699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e039.TIF" value="35676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e040.PNG" value="15613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e040.TIF" value="35224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e041.PNG" value="12441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e041.TIF" value="29924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e042.PNG" value="12081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e042.TIF" value="29760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e043.PNG" value="15520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e043.TIF" value="35076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e044.PNG" value="15810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.e044.TIF" value="35776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.s001.DOC" value="33280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.s002.DOC" value="40960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.s003.DOC" value="2591204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000776.s004.DOC" value="1148000"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000776&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000776" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000776&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000776&volume=&issue=&title=Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions&author_name=Zhen%20Xuan%20Yeo%2C%20Sum%20Thai%20Wong%2C%20Satya%20Nanda%20Vel%20Arjunan%2C%20Vincent%20Piras%2C%20Masaru%20Tomita%2C%20Kumar%20Selvarajoo%2C%20Alessandro%20Giuliani%2C%20Masa%20Tsuchiya&start_page=1&end_page=18" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000776" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000776" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000776" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000776&amp;t=Sequential%20Logic%20Model%20Deciphers%20Dynamic%20Transcriptional%20Control%20of%20Gene%20Expressions" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000776&title=Sequential%20Logic%20Model%20Deciphers%20Dynamic%20Transcriptional%20Control%20of%20Gene%20Expressions&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000776&amp;title=Sequential%20Logic%20Model%20Deciphers%20Dynamic%20Transcriptional%20Control%20of%20Gene%20Expressions" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000776" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000776&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Sequential Logic Model Deciphers Dynamic Transcriptional Control of Gene Expressions';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000776';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Sequential%20Logic%20Model%20Deciphers%20Dynamic%20Transcriptional%20Control%20of%20Gene%20Expressions http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000776" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000776" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="24390"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24390"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="24390"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="24390"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulatory+networks%22" title="Search for articles in the subject area:'Gene regulatory networks'"><div class="flagText">Gene regulatory networks</div></a>
              <div data-categoryid="17555" data-articleid="24390"
                   data-categoryname="Gene regulatory networks"
                   class="flagImage" title="Flag 'Gene regulatory networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutagenesis%22" title="Search for articles in the subject area:'Mutagenesis'"><div class="flagText">Mutagenesis</div></a>
              <div data-categoryid="21421" data-articleid="24390"
                   data-categoryname="Mutagenesis"
                   class="flagImage" title="Flag 'Mutagenesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="24390"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="24390"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=375'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6726'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8799&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=1833">Referee Comments: Referee 1 (Timothy Gardner and Ilaria Mogno)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
