

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001700"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001700" />

    <meta name="citation_title" content="Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium"/>
    <meta itemprop="name" content="Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium"/>

      <meta name="citation_author" content="Denise M. Wolf"/>
            <meta name="citation_author_institution" content="Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Lisa Fontaine-Bodin"/>
            <meta name="citation_author_institution" content="Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Bioengineering, University of California, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Ilka Bischofs"/>
            <meta name="citation_author_institution" content="Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Bioengineering, University of California, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Gavin Price"/>
            <meta name="citation_author_institution" content="Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Bioengineering, University of California, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Jay Keasling"/>
            <meta name="citation_author_institution" content="Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Bioengineering, University of California, Berkeley, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Chemical Engineering, University of California, Berkeley, California, United States of America"/>
      <meta name="citation_author" content="Adam P. Arkin"/>
            <meta name="citation_author_institution" content="Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Bioengineering, University of California, Berkeley, California, United States of America"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001700.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1700"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The effect of the growth environment on the lag phase of Listeria monocytogenes.; citation_author=TP Robinson; citation_author=MJ Ocio; citation_author=A Kaloti; citation_author=BM Mackey; citation_journal_title=Int J Food Microbiol; citation_volume=44; citation_number=1; citation_pages=83-92; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Growth history influences starvation-induced expression of uspA, grpE, and rpoS and subsequent cryotolerance in Escherichia coli O157:H7.; citation_author=PV Gawande; citation_author=MW Griffiths; citation_journal_title=J Food Prot; citation_volume=68; citation_number=2; citation_pages=1154-1158; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Survival of enteric bacteria in seawater.; citation_author=Y Rozen; citation_author=S Belkin; citation_journal_title=FEMS Microbiol Rev; citation_volume=25; citation_number=3; citation_pages=513-529; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Memory in bacteria and phage.; citation_author=J Casadesus; citation_author=R D'Ari; citation_journal_title=Bioessays; citation_volume=24; citation_number=4; citation_pages=512-518; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Autoamplification of a two-component regulatory system results in “learning” behavior.; citation_author=SM Hoffer; citation_author=HV Westerhoff; citation_author=KJ Hellingwerf; citation_author=PW Postma; citation_author=J Tommassen; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=5; citation_pages=4914-4917; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Bacterial observations: a rudimentary form of intelligence?; citation_author=KJ Hellingwerf; citation_journal_title=Trends Microbiol; citation_volume=13; citation_number=6; citation_pages=152-158; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A response regulator model in a simple sensory system.; citation_author=DE Koshland; citation_journal_title=Science; citation_volume=196; citation_number=7; citation_pages=1055-1063; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Playing Dr Jekyll and Mr Hyde: combined mechanisms of phase variation in bacteria.; citation_author=B Hallet; citation_journal_title=Curr Opin Microbiol; citation_volume=4; citation_number=8; citation_pages=570-581; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Switches, cross-talk and memory in Escherichia coli adherence.; citation_author=NJ Holden; citation_author=DL Gally; citation_journal_title=J Med Microbiol; citation_volume=53; citation_number=9; citation_pages=585-593; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bistability in bacteria.; citation_author=D Dubnau; citation_author=R Losick; citation_journal_title=Mol Microbiol; citation_volume=61; citation_number=10; citation_pages=564-572; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Signalling network with a bistable hysteretic switch controls developmental activation of the sigma transcription factor in Bacillus subtilis.; citation_author=OA Igoshin; citation_author=CW Price; citation_author=MA Savageau; citation_journal_title=Mol Microbiol; citation_volume=61; citation_number=11; citation_pages=165-184; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Motifs, modules and games in bacteria.; citation_author=DM Wolf; citation_author=AP Arkin; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=12; citation_pages=125-134; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stochastic kinetic analysis of developmental pathway bifurcation in phage lambda-infected Escherichia coli cells.; citation_author=A Arkin; citation_author=J Ross; citation_author=HH McAdams; citation_journal_title=Genetics; citation_volume=149; citation_number=13; citation_pages=1633-1648; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Transcriptional dynamics of the embryonic stem cell switch.; citation_author=V Chickarmane; citation_author=C Troein; citation_author=UA Nuber; citation_author=HM Sauro; citation_author=C Peterson; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=14; citation_pages=e123; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bistability in the JNK cascade.; citation_author=CP Bagowski; citation_author=JE Ferrell; citation_journal_title=Curr Biol; citation_volume=11; citation_number=15; citation_pages=1176-1182; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Self-perpetuating states in signal transduction: positive feedback, double-negative feedback and bistability.; citation_author=JE Ferrell; citation_journal_title=Curr Opin Cell Biol; citation_volume=14; citation_number=16; citation_pages=140-148; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Network dynamics and cell physiology.; citation_author=JJ Tyson; citation_author=K Chen; citation_author=B Novak; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=2; citation_number=17; citation_pages=908-916; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mathematical model of the fission yeast cell cycle with checkpoint controls at the G1/S, G2/M and metaphase/anaphase transitions.; citation_author=B Novak; citation_author=A Csikasz-Nagy; citation_author=B Gyorffy; citation_author=K Chen; citation_author=JJ Tyson; citation_journal_title=Biophys Chem; citation_volume=72; citation_number=18; citation_pages=185-200; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Adaptive response of a gene network to environmental changes by fitness-induced attractor selection.; citation_author=A Kashiwagi; citation_author=I Urabe; citation_author=K Kaneko; citation_author=T Yomo; citation_journal_title=PLoS ONE; citation_volume=1; citation_number=19; citation_pages=e49; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Positive feedback in eukaryotic gene networks: cell differentiation by graded to binary response conversion.; citation_author=A Becskei; citation_author=B Seraphin; citation_author=L Serrano; citation_journal_title=Embo J; citation_volume=20; citation_number=20; citation_pages=2528-2535; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Construction of a genetic toggle switch in Escherichia coli.; citation_author=TS Gardner; citation_author=CR Cantor; citation_author=JJ Collins; citation_journal_title=Nature; citation_volume=403; citation_number=21; citation_pages=339-342; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rational design of memory in eukaryotic cells.; citation_author=CM Ajo-Franklin; citation_author=DA Drubin; citation_author=JA Eskin; citation_author=EP Gee; citation_author=D Landgraf; citation_journal_title=Genes Dev; citation_volume=21; citation_number=22; citation_pages=2271-2276; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mammalian synthetic biology: engineering of sophisticated gene networks.; citation_author=D Greber; citation_author=M Fussenegger; citation_journal_title=J Biotechnol; citation_volume=130; citation_number=23; citation_pages=329-345; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Bifurcation and singularity analysis of a molecular network for the induction of long-term memory.; citation_author=H Song; citation_author=P Smolen; citation_author=E Av-Ron; citation_author=DA Baxter; citation_author=JH Byrne; citation_journal_title=Biophys J; citation_volume=90; citation_number=24; citation_pages=2309-2325; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Short-term and long-term memory in single cells.; citation_author=BH Morimoto; citation_author=DE Koshland; citation_journal_title=Faseb J; citation_volume=5; citation_number=25; citation_pages=2061-2067; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Enhancement of cellular memory by reducing stochastic transitions.; citation_author=M Acar; citation_author=A Becskei; citation_author=A van Oudenaarden; citation_journal_title=Nature; citation_volume=435; citation_number=26; citation_pages=228-232; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution and the theory of games.; citation_author=J Maynard Smith; citation_number=27; citation_pages=viii, 224; citation_date=1982; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Diversity in times of adversity: probabilistic strategies in microbial survival games.; citation_author=DM Wolf; citation_author=VJ Vazirani; citation_author=AP Arkin; citation_journal_title=J Theor Biol in press; citation_number=28; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A microbial modified Prisoner's Dilemma game: how frequency-dependent selection can lead to random phase variation.; citation_author=DM Wolf; citation_author=VJ Vazirani; citation_author=AP Arkin; citation_journal_title=J Theor Biol in press; citation_number=29; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phenotypic diversity, population growth, and information in fluctuating environments.; citation_author=E Kussell; citation_author=S Leibler; citation_journal_title=Science; citation_volume=309; citation_number=30; citation_pages=2075-2078; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Repeated Games with Incomplete Information.; citation_author=R Aumann; citation_author=M Maschler; citation_number=31; citation_date=1995; citation_publisher=MIT Press; " />
      <meta name="citation_reference" content="citation_title=Dynamic Noncooperative Game Theory.; citation_author=T Basar; citation_author=GJ Olsder; citation_number=32; citation_date=1995; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=A strategy of win-stay, lose-shift that outperforms tit-for-tat in the Prisoner's Dilemma game.; citation_author=M Nowak; citation_author=K Sigmund; citation_journal_title=Nature; citation_volume=364; citation_number=33; citation_pages=56-58; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The evolution of cooperation.; citation_author=R Axelrod; citation_author=WD Hamilton; citation_journal_title=Science; citation_volume=211; citation_number=34; citation_pages=1390-1396; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Tit for tat in heterogeneous populations.; citation_author=M Nowak; citation_author=K Sigmund; citation_journal_title=Nature; citation_volume=355; citation_number=35; citation_pages=250-252; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Phosphatases modulate the bistable sporulation gene expression pattern in Bacillus subtilis.; citation_author=JW Veening; citation_author=LW Hamoen; citation_author=OP Kuipers; citation_journal_title=Mol Microbiol; citation_volume=56; citation_number=36; citation_pages=1481-1494; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic networks controlling the initiation of sporulation and the development of genetic competence in Bacillus subtilis.; citation_author=AD Grossman; citation_journal_title=Annu Rev Genet; citation_volume=29; citation_number=37; citation_pages=477-508; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=When the going gets tough: survival strategies and environmental signaling networks in Bacillus subtilis.; citation_author=T Msadek; citation_journal_title=Trends Microbiol; citation_volume=7; citation_number=38; citation_pages=201-207; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The B. subtilis SIN Operon: An Evolvable Network Motif.; citation_author=CA Voigt; citation_author=D Wolf; citation_author=AP Arkin; citation_journal_title=Genetics; citation_number=39; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An excitable gene regulatory circuit induces transient cellular differentiation.; citation_author=GM Suel; citation_author=J Garcia-Ojalvo; citation_author=LM Liberman; citation_author=MB Elowitz; citation_journal_title=Nature; citation_volume=440; citation_number=40; citation_pages=545-550; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional profiling of the Bacillus anthracis life cycle in vitro and an implied model for regulation of spore formation.; citation_author=NH Bergman; citation_author=EC Anderson; citation_author=EE Swenson; citation_author=MM Niemeyer; citation_author=AD Miyoshi; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=41; citation_pages=6092-6100; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stable isotope ratios as a tool in microbial forensics--Part 2. Isotopic variation among different growth media as a tool for sourcing origins of bacterial cells or spores.; citation_author=HW Kreuzer-Martin; citation_author=LA Chesson; citation_author=MJ Lott; citation_author=JV Dorigan; citation_author=JR Ehleringer; citation_journal_title=J Forensic Sci; citation_volume=49; citation_number=42; citation_pages=961-967; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Stable isotope ratios as a tool in microbial forensics--Part 1. Microbial isotopic composition as a function of growth medium.; citation_author=HW Kreuzer-Martin; citation_author=LA Chesson; citation_author=MJ Lott; citation_author=JV Dorigan; citation_author=JR Ehleringer; citation_journal_title=J Forensic Sci; citation_volume=49; citation_number=43; citation_pages=954-960; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A multistep epigenetic switch enables the stable inheritance of DNA methylation states.; citation_author=HN Lim; citation_author=A van Oudenaarden; citation_journal_title=Nat Genet; citation_volume=39; citation_number=44; citation_pages=269-275; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The effects of stress on the thermoelectric quality of metals.; citation_author=JA Ewing; citation_journal_title=Proceedings of the Royal Society of London; citation_volume=32; citation_number=45; citation_pages=399-402; citation_date=1881; " />
      <meta name="citation_reference" content="citation_title=Systems with Hysteresis.; citation_author=MA Krasnosel'skij; citation_author=JA Lifshits; citation_author=AV Sobolev; citation_number=46; citation_date=1989; citation_publisher=Springer Verlag; " />
      <meta name="citation_reference" content="citation_title=Hysteresis drives cell-cycle transitions in Xenopus laevis egg extracts.; citation_author=W Sha; citation_author=J Moore; citation_author=K Chen; citation_author=AD Lassaletta; citation_author=CS Yi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=47; citation_pages=975-980; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A mathematical theory of cummunication.; citation_author=CE Shannon; citation_journal_title=Bell Systems Technical Journal; citation_volume=27; citation_number=48; citation_pages=379-656; citation_date=1948; " />
      <meta name="citation_reference" content="citation_title=How much do people remember? Some estimates of the quantity of learned information in long-term memory.; citation_author=TK Landauer; citation_journal_title=Cognitive Science; citation_volume=10; citation_number=49; citation_pages=477-493; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Information theory and neural coding.; citation_author=A Borst; citation_author=FE Theunissen; citation_journal_title=Nat Neurosci; citation_volume=2; citation_number=50; citation_pages=947-957; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=How to measure the information gained from one symbol.; citation_author=MR DeWeese; citation_author=M Meister; citation_journal_title=Network; citation_volume=10; citation_number=51; citation_pages=325-340; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=How much information is associated with a particular stimulus?; citation_author=DA Butts; citation_journal_title=Network; citation_volume=14; citation_number=52; citation_pages=177-187; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Analyzing neural responses to natural signals: maximally informative dimensions.; citation_author=T Sharpee; citation_author=NC Rust; citation_author=W Bialek; citation_journal_title=Neural Comput; citation_volume=16; citation_number=53; citation_pages=223-250; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Information Theory: Dover Publications.; citation_author=RB Ash; citation_number=54; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=SpoIIE governs the phosphorylation state of a protein regulating transcription factor sigma F during sporulation in Bacillus subtilis.; citation_author=F Arigoni; citation_author=L Duncan; citation_author=S Alper; citation_author=R Losick; citation_author=P Stragier; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=55; citation_pages=3238-3242; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular genetics of sporulation in Bacillus subtilis.; citation_author=P Stragier; citation_author=R Losick; citation_journal_title=Annu Rev Genet; citation_volume=30; citation_number=56; citation_pages=297-241; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Replacement of the Bacillus subtilis subtilisin structural gene with an In vitro-derived deletion mutation.; citation_author=ML Stahl; citation_author=E Ferrari; citation_journal_title=J Bacteriol; citation_volume=158; citation_number=57; citation_pages=411-418; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Molecular Biology Methods for Bacillus.; citation_author=C Harwood; citation_author=SM Cutting; citation_journal_title=Wiley-Intersciences; citation_number=58; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Phenotypic consequences of 1,000 generations of selection at elevated CO2 in a green alga.; citation_author=S Collins; citation_author=G Bell; citation_journal_title=Nature; citation_volume=431; citation_number=59; citation_pages=566-569; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fighting change with change: adaptive variation in an uncertain world.; citation_author=L Ancel Meyers; citation_author=JJ Bull; citation_journal_title=Trends in Ecology & Evolution; citation_volume=17; citation_number=60; citation_pages=551-557; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Cognitive Neurosciences III: MIT Press.; citation_author=MS Gazzaniga; citation_number=61; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Externally controlled attractor selection in a high-dimensional system.; citation_author=FH Willeboordse; citation_author=K Kaneko; citation_journal_title=Phys Rev E Stat Nonlin Soft Matter Phys; citation_volume=72; citation_number=62; citation_pages=026207; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple attractors in immunology: theory and experiment.; citation_author=LA Segel; citation_journal_title=Biophys Chem; citation_volume=72; citation_number=63; citation_pages=223-230; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cell fates as high-dimensional attractor states of a complex gene regulatory network.; citation_author=S Huang; citation_author=G Eichler; citation_author=Y Bar-Yam; citation_author=DE Ingber; citation_journal_title=Phys Rev Lett; citation_volume=94; citation_number=64; citation_pages=128701; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Output regulation and Internal models: a frequency domain approach.; citation_author=G Bengtsson; citation_journal_title=Automatica; citation_volume=13; citation_number=65; citation_pages=333-345; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Extinction dynamics of age-structured populations in a fluctuating environment.; citation_author=R Lande; citation_author=SH Orzack; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=85; citation_number=66; citation_pages=7418-7421; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Stochastic gene expression in fluctuating environments.; citation_author=M Thattai; citation_author=A van Oudenaarden; citation_journal_title=Genetics; citation_volume=167; citation_number=67; citation_pages=523-530; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual.; citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=68; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=FACS-optimized mutants of the green fluorescent protein (GFP).; citation_author=BP Cormack; citation_author=RH Valdivia; citation_author=S Falkow; citation_journal_title=Gene; citation_volume=173; citation_number=69; citation_pages=33-38; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=An investigation into the compartmentalization of the sporulation transcription factor sigmaE in Bacillus subtilis.; citation_author=M Fujita; citation_author=R Losick; citation_journal_title=Mol Microbiol; citation_volume=43; citation_number=70; citation_pages=27-38; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Control of development by altered localization of a transcription factor in B. subtilis.; citation_author=JD Quisel; citation_author=DC Lin; citation_author=AD Grossman; citation_journal_title=Mol Cell; citation_volume=4; citation_number=71; citation_pages=665-672; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Construction of protein overproducer strains in Bacillus subtilis by an integrative approach.; citation_author=J Jan; citation_author=F Valle; citation_author=F Bolivar; citation_author=E Merino; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=55; citation_number=72; citation_pages=69-75; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Silhouettes: A graphical aid to the interpretation and validation of cluster analysis.; citation_author=PJ Rousseeuw; citation_journal_title=Journal of Computational and Applied Mathematics; citation_volume=20; citation_number=73; citation_pages=53-65; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Information transport in spatiotemporal systems.; citation_author=JA Vastano; citation_author=HL Swinney; citation_journal_title=Physical Review Letters; citation_volume=60; citation_number=74; citation_pages=1773-1776; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=On the deduction of chemical reaction pathways from measurements of time series of concentrations.; citation_author=M Samoilov; citation_author=A Arkin; citation_author=J Ross; citation_journal_title=Chaos; citation_volume=11; citation_number=75; citation_pages=108-114; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA transport during transformation.; citation_author=I Chen; citation_author=D Dubnau; citation_journal_title=Front Biosci; citation_volume=8; citation_number=76; citation_pages=s544-556; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The DegS/DegU and ComP/ComA two-component systems are part of a network controlling degradative enzyme synthesis and competence in Bacillus subtilis.; citation_author=F Kunst; citation_author=T Msadek; citation_author=J Bignon; citation_author=G Rapoport; citation_journal_title=Res Microbiol; citation_volume=145; citation_number=77; citation_pages=393-402; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Chemotaxis in Bacillus subtilis: how bacteria monitor environmental signals.; citation_author=LF Garrity; citation_author=GW Ordal; citation_journal_title=Pharmacol Ther; citation_volume=68; citation_number=78; citation_pages=87-104; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Regulation of peptide antibiotic production in Bacillus.; citation_author=MA Marahiel; citation_author=MM Nakano; citation_author=P Zuber; citation_journal_title=Mol Microbiol; citation_volume=7; citation_number=79; citation_pages=631-636; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Genetics of subtilin and nisin biosyntheses: biosynthesis of lantibiotics.; citation_author=KD Entian; citation_author=WM de Vos; citation_journal_title=Antonie Van Leeuwenhoek; citation_volume=69; citation_number=80; citation_pages=109-117; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A master regulator for biofilm formation by Bacillus subtilis.; citation_author=DB Kearns; citation_author=F Chu; citation_author=SS Branda; citation_author=R Kolter; citation_author=R Losick; citation_journal_title=Mol Microbiol; citation_volume=55; citation_number=81; citation_pages=739-749; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacillus subtilis transcriptional regulators interaction.; citation_author=A Sanchez; citation_author=J Olmos; citation_journal_title=Biotechnol Lett; citation_volume=26; citation_number=82; citation_pages=403-407; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001700" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium"/>
    <meta name="twitter:description" content="Memory is usually associated with higher organisms rather than bacteria. However, evidence is mounting that many regulatory networks within bacteria are capable of complex dynamics and multi-stable behaviors that have been linked to memory in other systems. Moreover, it is recognized that bacteria that have experienced different environmental histories may respond differently to current conditions. These &ldquo;memory&rdquo; effects may be more than incidental to the regulatory mechanisms controlling acclimation or to the status of the metabolic stores. Rather, they may be regulated by the cell and confer fitness to the organism in the evolutionary game it participates in. Here, we propose that history-dependent behavior is a potentially important manifestation of memory, worth classifying and quantifying. To this end, we develop an information-theory based conceptual framework for measuring both the persistence of memory in microbes and the amount of information about the past encoded in history-dependent dynamics. This method produces a phenomenological measure of cellular memory without regard to the specific cellular mechanisms encoding it. We then apply this framework to a strain of Bacillus subtilis engineered to report on commitment to sporulation and degradative enzyme (AprE) synthesis and estimate the capacity of these systems and growth dynamics to &lsquo;remember&rsquo; 10 distinct cell histories prior to application of a common stressor. The analysis suggests that B. subtilis remembers, both in short and long term, aspects of its cell history, and that this memory is distributed differently among the observables. While this study does not examine the mechanistic bases for memory, it presents a framework for quantifying memory in cellular behaviors and is thus a starting point for studying new questions about cellular regulation and evolutionary strategy."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001700.g007"/>

  <meta property="og:title" content="Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001700" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9978'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8421'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=106&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001700">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001700" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001700&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Denise M. Wolf
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Denise M. Wolf, Lisa Fontaine-Bodin</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:dmwolf@lbl.gov">dmwolf@lbl.gov</a> (DW); <a href="mailto:aparkin@lbl.gov">aparkin@lbl.gov</a> (AA)</p>

                <p>Affiliation:
                  Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lisa Fontaine-Bodin
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Denise M. Wolf, Lisa Fontaine-Bodin</p>

              
              

                <p>Affiliations:
                  Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America, 
                  Department of Bioengineering, University of California, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ilka Bischofs, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America, 
                  Department of Bioengineering, University of California, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gavin Price, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America, 
                  Department of Bioengineering, University of California, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jay Keasling, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America, 
                  Department of Bioengineering, University of California, Berkeley, California, United States of America, 
                  Department of Chemical Engineering, University of California, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adam P. Arkin
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:dmwolf@lbl.gov">dmwolf@lbl.gov</a> (DW); <a href="mailto:aparkin@lbl.gov">aparkin@lbl.gov</a> (AA)</p>

                <p>Affiliations:
                  Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, California, United States of America, 
                  Department of Bioengineering, University of California, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001700</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001700" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001700" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001700" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001700" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001700-g001" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001700-g002" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001700-g003" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001700-g004" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001700-t001" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001700-g005" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001700-g006" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001700-g007" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001700">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001700" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Memory is usually associated with higher organisms rather than bacteria. However, evidence is mounting that many regulatory networks within bacteria are capable of complex dynamics and multi-stable behaviors that have been linked to memory in other systems. Moreover, it is recognized that bacteria that have experienced different environmental histories may respond differently to current conditions. These “memory” effects may be more than incidental to the regulatory mechanisms controlling acclimation or to the status of the metabolic stores. Rather, they may be regulated by the cell and confer fitness to the organism in the evolutionary game it participates in. Here, we propose that history-dependent behavior is a potentially important manifestation of memory, worth classifying and quantifying. To this end, we develop an information-theory based conceptual framework for measuring both the persistence of memory in microbes and the amount of information about the past encoded in history-dependent dynamics. This method produces a phenomenological measure of cellular memory without regard to the specific cellular mechanisms encoding it. We then apply this framework to a strain of <em>Bacillus subtilis</em> engineered to report on commitment to sporulation and degradative enzyme (AprE) synthesis and estimate the capacity of these systems and growth dynamics to ‘remember’ 10 distinct cell histories prior to application of a common stressor. The analysis suggests that <em>B. subtilis</em> remembers, both in short and long term, aspects of its cell history, and that this memory is distributed differently among the observables. While this study does not examine the mechanistic bases for memory, it presents a framework for quantifying memory in cellular behaviors and is thus a starting point for studying new questions about cellular regulation and evolutionary strategy.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wolf DM, Fontaine-Bodin L, Bischofs I, Price G, Keasling J, et al.  (2008) Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium. PLoS ONE 3(2):
          e1700.
            doi:10.1371/journal.pone.0001700</p><p><strong>Academic Editor: </strong>Pamela A. Silver, Harvard Medical School, United States of America</p><p><strong>Received:</strong> November 15, 2007; <strong>Accepted:</strong> January 28, 2008; <strong>Published:</strong> February 27, 2008</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>The authors would like to acknowledge the National Institutes of Health (R01 GM073010-01), the Department of Energy and the Howard Hughes Medical Institute for support during the period of this project.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Your average bacterium is unlikely to recite π to 15 places or compose a symphony. Yet evidence is mounting that these ‘simple’ cells contain complex control circuitry capable of generating multi-stable behaviors and other complex dynamics that have been conceptually linked to memory in other systems. And though few would call this phenomenon memory in the ‘human’ sense, it has long been known that bacterial cells that have experienced different environmental histories may respond differently to current conditions <a href="#pone.0001700-Robinson1">[1]</a>–<a href="#pone.0001700-Rozen1">[3]</a>. Though some of these history-dependent behavioral differences may be physically necessary consequences of the prior history, and thus some might argue insignificant, other behavioral differences may be controllable and therefore selectable and even fitness enhancing manifestations of memory.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In this paper we take the potentially controversial view that history-dependent behavior, whether short or long term, controlled or incidental, reflects a form of memory <a href="#pone.0001700-Casadesus1">[4]</a>–<a href="#pone.0001700-Hellingwerf1">[6]</a>. Because bacterial dynamics at every level of resolution operate within the limitations and potentials of nonlinear physical and biochemical dynamical systems, they must exhibit at least very short-term transient memory, and potentially longer term memory. The type of memory (and its significance) depends on which features of cell history are ‘remembered’, and at what resolution; whether or not the system eventually ‘forgets’ its past, and if so, how long this forgetting takes; the mechanisms in the cell responsible for memory storage, encoding, and retrieval; and whether or not this memory provides a fitness advantage in a natural environment. In cellular systems, environmental memory has been noted to be inherent in everything from the selective history of mutation, epigenetic inheritance via chromatin modification in neurons and DNA methylation in chemotaxing bacteria <a href="#pone.0001700-Koshland1">[7]</a>, genetic and epigenetic phase variation mechanisms controlling surface features of pathogenic bacteria <a href="#pone.0001700-Hallet1">[8]</a>, <a href="#pone.0001700-Holden1">[9]</a>, cellular proliferation and survival in the immune system, and in switch-like feedback systems in regulatory networks spanning signal transduction, metabolism and gene expression <a href="#pone.0001700-Dubnau1">[10]</a>–<a href="#pone.0001700-Gardner1">[21]</a>. There is also a growing body of work focusing on synthetically designing and constructing network motifs and systems that are capable of showing some types of dynamic memory <a href="#pone.0001700-AjoFranklin1">[22]</a>, <a href="#pone.0001700-Greber1">[23]</a>. These and many other studies in synthetic and natural systems suggest that even the simplest first-order chemical reactions have at least transient memory of initial conditions, and more complex mechanisms involving history-dependent changes in the concentrations, states and localization of proteins and other regulatory network elements can encode a wide range of input information and store it for amounts of time ranging from minutes to days or longer <a href="#pone.0001700-Casadesus1">[4]</a>, <a href="#pone.0001700-Ferrell1">[16]</a>, <a href="#pone.0001700-Song1">[24]</a>, <a href="#pone.0001700-Morimoto1">[25]</a>. The state dynamics of such systems contain the memory of past controlling inputs, and even of past environmental conditions if one is to interpret more broadly <a href="#pone.0001700-Hoffer1">[5]</a>, <a href="#pone.0001700-Acar1">[26]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In metazoans, the ability of somatic cells to remember their fates is key to development and thus to organismal fitness. The same can be said for other types of metazoan cells like those found in the immune system that use a memory of past states to modify future behavior. In principle at least, memory, whether short- or long-term, can feasibly confer an evolutionary advantage in microbes as well. For instance, Hoffer <em>et. al.</em> suggest that in <em>E. coli</em> a form of ‘memory’ of past phosphate limitation leads to a faster response to successive periods of phosphate limitation, and that this faster response may be survival enhancing <a href="#pone.0001700-Hoffer1">[5]</a>. It has also been suggested that pathogenic bacteria use cross-talk encoded memory to balance the demands of immune avoidance with a sequential, compartment to compartment infection lifecycle <a href="#pone.0001700-Hallet1">[8]</a>, <a href="#pone.0001700-Holden1">[9]</a>. More abstractly, the dynamic implementation of cellular behaviors can be viewed as a selected, ‘winning’ (or at least stable) strategy in an evolutionary game <a href="#pone.0001700-Wolf1">[12]</a>, <a href="#pone.0001700-MaynardSmith1">[27]</a>. In game theory, information creates advantage <a href="#pone.0001700-Wolf2">[28]</a>–<a href="#pone.0001700-Kussell1">[30]</a>, and information about the past as well as the present creates even greater advantage. Thus if bacterial cells are able to store information about past experience in some type of memory, and use this memory to modulate their behavior, this opens up the possibility of playing game strategies with memory, a provably superior family of strategies compared to those without memory <a href="#pone.0001700-Aumann1">[31]</a>–<a href="#pone.0001700-Nowak2">[35]</a>. Even if the memory capacity of the system is short term, but on the order of environmental fluctuations, it could conceivably impact fitness and therefore play a role in an evolved adaptive behaviour <a href="#pone.0001700-Wolf2">[28]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Given the potential ubiquity and significance of bacterial memory, we propose that quantifying history dependent behavior in microbes could be an important piece of the puzzle of bacterial regulation, survival strategy, and evolution. To this end, we developed an information-theory based conceptual framework for thinking about and measuring both the persistence of memory in microbes and the amount of information about the past encoded in these dynamics. This method produces a phenomenological measure of cellular memory <em>without</em> regard to the specific cellular mechanisms encoding it. We then applied this framework to the bacterium <em>B. subtilis</em>. <em>B. subtilis</em> presents an excellent model organism for this study because of its exquisite sensitivity to environmental conditions, its known mechanisms of bistability and other hysteretic switch-like regulatory stress response mechanisms and architectures, and its developmental decision to sporulate that strongly resembles eukaryotic memory-associated processes determining developmental cell fate (<a href="#pone.0001700-Dubnau1">[10]</a>, <a href="#pone.0001700-Veening1">[36]</a>–<a href="#pone.0001700-Suel1">[40]</a>, <a href="#pone-0001700-g001">Fig. 1</a>). Also, certain aspects of <em>B. subtilis</em> behavior, such as spore coat composition, have already been associated with environmental memory <a href="#pone.0001700-Bergman1">[41]</a>–<a href="#pone.0001700-KreuzerMartin2">[43]</a>, and though much suggests that there should be memory, how these response dynamics depend on past conditions prior to application of a stress has not been systematically examined.</p>
<div class="figure" id="pone-0001700-g001"><div class="img"><a name="pone-0001700-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The <em>B. subtilis</em> stress response meta-network, where each oval represents both a stress response and the regulatory network of 100 or so interacting molecular species that regulates it.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Among the many ingenious genetic and biochemical programs employed by <em>B. subtilis</em> to cope with environmentally adverse conditions are its ability to take up extracellular DNA, competence <a href="#pone.0001700-Suel1">[40]</a>, <a href="#pone.0001700-Chen1">[76]</a>; differentiate into an inert heat-, chemical, and UV-resistant spore <a href="#pone.0001700-Grossman1">[37]</a>; secrete degradative enzymes to identify and digest new food sources <a href="#pone.0001700-Kunst1">[77]</a>; become motile and chemotax toward possibly better surroundings <a href="#pone.0001700-Garrity1">[78]</a>; synthesize antibiotics to eliminate competitors in the same ecosystem <a href="#pone.0001700-Marahiel1">[79]</a>, <a href="#pone.0001700-Entian1">[80]</a>; turn on alternative metabolic pathways, and form biofilms (<a href="#pone.0001700-Kearns1">[81]</a>, not shown), just to name a few <a href="#pone.0001700-Msadek1">[38]</a>. The cross-repressive feedback between sporulation and competence, and the many positive feedback loops within each large ‘individual’ stress response pathway <a href="#pone.0001700-Dubnau1">[10]</a>, <a href="#pone.0001700-Veening1">[36]</a>–<a href="#pone.0001700-Suel1">[40]</a>, <a href="#pone.0001700-Sanchez1">[82]</a>, are suggestive of switches and other elements that could potentially encode memory. The two stress response pathways monitored in our experiments, sporulation and synthesis of the degradative enzyme subtilisin, are denoted by bold-faced ovals. The fluorescent reporters (GFP and DsRed) fused to the respective promoters P<em>spoIIE</em> and P<em>aprE</em> are indicated (see <a href="#s4">Materials and Methods</a>).</p>
<span>doi:10.1371/journal.pone.0001700.g001</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In our experiments, we quantified the ability of three <em>B. subtilis</em> stress response systems–sporulation, degradative enzyme synthesis, and growth-to ‘remember’ 10 distinct cell histories prior to application of a common stressor. We chose to observe commitment to sporulation (via reporter fusion to P<sub>SPOIIE</sub>) because the sporulation decision is bistable, and bistability is associated with memory <a href="#pone.0001700-Holden1">[9]</a>, <a href="#pone.0001700-Igoshin1">[11]</a>, <a href="#pone.0001700-Ferrell1">[16]</a>, <a href="#pone.0001700-Lim1">[44]</a>. We added the reporter for degradative enzyme synthesis (measured by a fluorescent reporter fused to the AprE promoter) because though it shares many common controllers with sporulation, its expression pattern is quite different and not believed to be bistable or probabilistic. We wondered whether any history-dependence in sporulation control would be mirrored in AprE control. Finally, we chose to observe growth (as measured by OD<sub>600</sub>) because it is perhaps the most accessible measure of cellular health and fitness and is an integrator of many other aspects of cell function, thus it may show interesting differences depending on cell history. One can imagine that there might be a strong fitness incentive toward memory in <em>B. subtilis</em>. If cells could use a memory of past conditions to ‘predict’ future conditions, and delay sporulation, an expensive process, if the environment is likely to improve or accelerate sporulation if the starvation period is likely to be long, they might improve their odds for long-term survival.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Information Theoretic Memory Framework</h4>
<h5>‘Adaptive’ memory experiment.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>A complete quantification of biologically relevant memory would involve first perturbing the cell with all possible sequences of complex environmental inputs it might experience in the wild in each of its growth modes, then measuring all cellular responses to these perturbations, and, finally, quantifying the degree and distribution of history-dependence in these responses.</p>
<a id="article1.body1.sec2.sec1.sec1.p2" name="article1.body1.sec2.sec1.sec1.p2"></a><p>Here we assume a simple approximation of this scenario, in which each sample of a biological system is subjected to one of many conditions prior to time t0, and then observed in a common condition after t0 (see <a href="#pone-0001700-g002">Fig. 2</a> and Definition (1) in <a href="#pone.0001700.s001">Appendix S1</a> in Supplementary Information). We call this an ‘adaptive’ memory experiment because it roughly simulates a temporal shift in the environment requiring adaptation or acclimation, and to differentiate it from the more classical memory experiments in physics, engineering and cell biology designed to identify hysteretic loops <a href="#pone.0001700-Ewing1">[45]</a>–<a href="#pone.0001700-Sha1">[47]</a>. While we do not identify such loops here, multistability is suggested by the appearance of long term memory in our experiments. More complex environmental history trajectories could feasibly unravel more memory effects.</p>
<div class="figure" id="pone-0001700-g002"><div class="img"><a name="pone-0001700-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>An ‘adaptive’ memory experiment.</span></strong></p><a id="article1.body1.sec2.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec1.fig1.caption1.p1"></a><p>In an adaptive memory experiment, each (identical) sample of a biological system is subjected to one of several conditions prior to time t0, and then observed in a common condition after t0. If different past histories lead to different short-term behaviors in current conditions, the system can be said to exhibit short-term memory. If different past histories lead to different long-term behaviors, the system can be said to exhibit long-term memory.</p>
<span>doi:10.1371/journal.pone.0001700.g002</span></div><a id="article1.body1.sec2.sec1.sec1.p3" name="article1.body1.sec2.sec1.sec1.p3"></a><p>We are interested in whether past conditions can be inferred from observations of behavior in current conditions. <em>The assumption here is that history-dependent behavior is a manifestation of memory, and that the better the possible inference about prior conditions from current measurements, the more memory there is within the system.</em></p>

<h5>Adapting communication metrics to memory.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>To quantify this intuitive concept of history-dependence as memory, we use concepts from information theory <a href="#pone.0001700-Shannon1">[48]</a> in the tradition of Landauer's use of informational entropy to estimate human memory capacity <a href="#pone.0001700-Landauer1">[49]</a>, and the extensive body of work characterizing memory in individual neurons <a href="#pone.0001700-Borst1">[50]</a>–<a href="#pone.0001700-Sharpee1">[53]</a>.</p>
<a id="article1.body1.sec2.sec1.sec2.p2" name="article1.body1.sec2.sec1.sec2.p2"></a><p>By interpreting the random variable <em>Y</em> as behavior in current conditions, and the random variable <em>M</em> as past cellular history prior to time t0, the mutual information <em>I</em>(<em>M</em>;<em>Y</em>) = <em>H</em>(<em>M</em>)−<em>H</em>(<em>M</em>|<em>Y</em>) of <em>M</em> relative to <em>Y</em> provides a measure of memory in informational entropy bits (see <a href="#pone.0001700-Shannon1">[48]</a>, <a href="#pone-0001700-g003">Fig. 3</a>, and Definition (2) in <a href="#pone.0001700.s001">Appendix S1</a> for details, including the definition of informational entropy <em>H</em>). Roughly speaking, from this perspective <em>I(M;Y)</em> captures how much uncertainty about past conditions can be reduced by observations of behavior in current conditions. Worded differently, <em>I(M;Y)</em> captures how much information about past conditions can be inferred from observations of behavior in current conditions. <em>The better the possible inference about prior conditions (and thus the higher the bit count of I(M;Y)), the more memory there is within the system.</em></p>
<div class="figure" id="pone-0001700-g003"><div class="img"><a name="pone-0001700-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Information-based conceptual schema for measuring memory in microbes.</span></strong></p><a id="article1.body1.sec2.sec1.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec2.fig1.caption1.p1"></a><p>In communication theory (top), the informational entropy of the signal space H(<em>X</em>) captures the number of different messages <em>X</em> that can be communicated and their probabilistic dispersal; the mutual information I(<em>X,Y</em>) between transmitted and received signals quantifies the amount of information actually communicated. A memory experiment, in contrast, involves subjecting cells to distinct treatments <em>M</em> prior to time t0, followed by an identical treatment <em>S</em> after time t0, with cell behavior from t0 on monitored through temporal sampling of one or more observable variables <em>Y</em>. As applied to bacterial memory (bottom), the informational entropy of the cell history space H(<em>M</em>) captures the number of different cell histories prior to time t0 tested by the experimental compendium and their probabilistic dispersal; the mutual information I<sub>trans</sub>(<em>M;Y;t<sub>trans</sub></em>) between the transient response of the observable variable <em>Y</em> after time t0 and the cell history prior to time t0 captures the short-term memory of cell history exhibited by <em>Y</em> over the cell history space in response to treatment <em>S</em>. Likewise, the mutual information I<sub>asym</sub>(<em>M,Y</em>) between the long-term response of <em>Y</em> and cell history prior to t0 captures the long-term memory of cell history exhibited by <em>Y</em>.</p>
<span>doi:10.1371/journal.pone.0001700.g003</span></div>
<h5>Short term vs. long term memory.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>Memory, or history-dependent behavior, can manifest across multiple time scales. Short term, or transient, memory is stored by the system for some time, and then ‘forgotten’ (see <a href="#pone-0001700-g004">Fig. 4a,d</a>). Systems may also have either ‘effective’ long term memory if the transient dynamics are long compared to environmental fluctuations, or ‘true’ asymptotic memory if the stationary state of the system depends on initial conditions, as occurs in nonlinear systems with multiple attractors (see <a href="#pone-0001700-g004">Figs. 4b,c,and e</a>). For an example of the latter, the state of a bistable switch encodes an asymptotic memory of the last switching event.</p>
<div class="figure" id="pone-0001700-g004"><div class="img"><a name="pone-0001700-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Different types of history-dependent behavior one might observe.</span></strong></p><a id="article1.body1.sec2.sec1.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec3.fig1.caption1.p1"></a><p>a) Short-term deterministic memory. State trajectories ‘remember’ their initial condition for some time, and then converge to a common asymptotic behavior. b) Long-term deterministic memory. State trajectories of multi-stable systems ‘remember’ which basin of attraction their initial condition started in indefinitely (the basin containing X0<sub>1</sub> vs. the basin containing X0<sub>2</sub> and X0<sub>3</sub>), but retain a memory of the exact initial condition within a basin of attraction only transiently (X0<sub>2</sub> vs. X0<sub>3</sub>). c) Short-term and Long-term memory in a system with unobservable states. The state space of the cell is two dimensional (X,Y), but only one of the two dimensions, X, is observed. Though all four initial conditions are distinct in the larger space, the unobserved Y component renders them identical to the observer. Thus the trajectories appear to diverge from a common starting point and approach one of two asymptotic states. This gives the observer the impression of first an increase in information and memory and then a decrease as the trajectories approach their long-term values. d,e) If measurements are made on single cells rather than on averaged populations (as we did in this paper), history-dependent distributions may be observed. d) Short-term stochastic memory. State trajectories are probabilistic in individual cells, with a distribution over the population that initially retains a ‘memory’ of the initial condition of the population. In the long-term, this memory degrades as the distribution approaches a global attractor. e) Long-term stochastic memory. The distribution over the population retains a ‘memory’ of the initial condition indefinitely, or at least over the time-horizon of the experiment.</p>
<span>doi:10.1371/journal.pone.0001700.g004</span></div><a id="article1.body1.sec2.sec1.sec3.p2" name="article1.body1.sec2.sec1.sec3.p2"></a><p>Because in many systems the significance, mechanistic origin, and function of memory likely depends on how long it lasts, and in particular whether it can be classified as short-term or long-term, we distinguish between the two types of memory and quantify them separately. From an information perspective, we say that an external observer of an adaptive memory experiment with <em>a priori</em> knowledge of the probability distribution over cell histories detects short-term memory in this system if observing measurements of some fraction of the short-term behaviour of the system after time t0 leads to a reduction in uncertainty about the history of the system prior to time t0. In this case, we say that the cells exhibit I<sub>trans</sub>(<em>M</em>;<em>Y</em>; <em>t<sub>trans</sub></em>)≡I(<em>M;Y</em>(t = t0:t0+<em>t<sub>trans</sub></em>)) bits of <em>short term memory</em> in the observable <em>Y</em> over the period from t0 to t0+trans, where t<sub>trans</sub> is a time before the signal approaches its steady state (Definition (4) in <a href="#pone.0001700.s001">Appendix S1</a>). Likewise, long-term memory is detected if observing measurements of the system behavior near an apparent steady state after time t0 leads to a reduction in uncertainty about the history of the system. Here we say the cells exhibit I<sub>asym</sub>(<em>M</em>;<em>Y</em>)≡I(<em>M;Y</em>(t = t0+<em>t<sub>asym</sub></em>:∞)) bits of <em>long term memory</em> in the observable response <em>Y</em> during the experiment, where t<sub>asym</sub> is the time it takes for the signal to settle (Definition (3) in <a href="#pone.0001700.s001">Appendix S1</a>).</p>

<h5>Memory quantification normalized.</h5><a id="article1.body1.sec2.sec1.sec4.p1" name="article1.body1.sec2.sec1.sec4.p1"></a><p>The above metrics for short term and long term memory are absolute measures, in that they give a bit count for an answer. Though these absolute numbers can be useful, it is also useful to measure memory in relative terms, compared to the total amount of memory that <em>could</em> be observed in a perfectly retentive system given the limitations of the experiments. To address this issue, we define <em>short-term memory fidelity</em> to be P<sub>trans</sub>(<em>M; Y; t<sub>trans</sub></em>)≡I(<em>M;Y</em>(t = t0:t0+<em>t<sub>trans</sub></em>))/H(<em>M</em>) and l<em>ong-term memory fidelity</em> to be P<sub>asym</sub>(<em>M;Y</em>)≡I(<em>M;Y</em>(t = t0+<em>t<sub>asym</sub></em> :∞))/H(<em>M</em>), where H(M) is the entropy over all the past conditions that were applied in the experiment. These normalized mutual information metrics, measures between 0 and 1 of the fraction of uncertainty about the past conditions tested that is reduced by knowledge of future cellular response, have also been called the coefficients of constraint <a href="#pone.0001700-Ash1">[54]</a> (see Definition (5) in <a href="#pone.0001700.s001">Appendix S1</a>).</p>

<h5>Quantifying memory in higher dimensions.</h5><a id="article1.body1.sec2.sec1.sec5.p1" name="article1.body1.sec2.sec1.sec5.p1"></a><p>In addition to analyzing each observable individually, we are interested in calculating the short and long term memory exhibited by the combined behavior of multiple observables. To do so, the above definitions are easily extended to the case of multiple observables by letting Y be a vector <em>Y</em>̅ = (<em>Y</em><sub>1</sub>,…,<em>Y<sub>n</sub></em>) and calculating I<sub>asym</sub>(<em>M</em>;(<em>Y<sub>1</sub>,…,Y<sub>n</sub></em>)) and I<sub>trans</sub>(<em>M</em>;(<em>Y<sub>1</sub>,…,Y<sub>n</sub></em>); <em>t<sub>trans</sub></em>) and the memory fidelity of each. This combined-memory estimation is interesting because it allows one to address the question of whether combining information from multiple read-outs leads to extra memory beyond what is present in any of the individual read-outs, and if so, how much. This issue is related to the size of the memory, and the dimension it occupies within a cell's state space.</p>
<a id="article1.body1.sec2.sec1.sec5.p2" name="article1.body1.sec2.sec1.sec5.p2"></a><p>An inequality governing the informational entropy of a vector pair of variables (X,Y) is as follows: max(<em>H</em>(<em>X</em>),<em>H</em>(<em>Y</em>))≤<em>H</em>(<em>X</em>,<em>Y</em>)≤<em>H</em>(<em>X</em>)+<em>H</em>(<em>Y</em>) <a href="#pone.0001700-Ash1">[54]</a>. Thus, we know that the memory exhibited by any pair of observables must be greater or equal to the bit count of the most retentive pathway of the pair, and less than or equal to the sum of the bit counts of the two pathways. If two pathways are controlled independently, their combined behavior could produce the upper limit on memory in the higher-dimensional space, whereas if the pathways are controlled by a common signal or if one pathway hierarchically controls the other, the lower limit might be realized. To quantify this concept, we define <em>memory orthogonality</em> between two pathway readouts <em>Y1</em> and <em>Y2</em> to be: <em>Mem<sub>orth</sub>(M;(Y1,Y2)) ≡ (I(M;(Y1,Y2))-max(I(M;Y1),I(M;Y2)))/min(​I(M;Y1),I(M;Y2))</em>, where <em>M</em> is cell history and <em>I</em> is mutual information. <em>Mem<sub>orth</sub></em> equals 1 if the two variables combined as a vector yield the upper bound of memory, and 0 if the two variables in combination yield the lower bound (see Definition (6) in <a href="#pone.0001700.s001">Appendix S1</a>).</p>

<h5>Implementation.</h5><a id="article1.body1.sec2.sec1.sec6.p1" name="article1.body1.sec2.sec1.sec6.p1"></a><p>For the calculations above, listed more formally in <a href="#pone.0001700.s001">Appendix S1</a> in Supplemental Information, we need to estimate probability distributions over the past cell histories being tested and the responses of the cells to each history. For past conditions/histories, we enforce a uniform probability of observation of each condition by running each experiment (condition i = &gt; response i) a fixed number of times. For responses, we cluster trajectories from the different conditions and the probability of a response is simply the histogram of trajectories over clusters. The probability of prior environment given cluster membership is enumerated in a similar way. Details of the entire analysis algorithm can be found in <a href="#s4">Materials and Methods</a>.</p>

<h5>Caveats.</h5><a id="article1.body1.sec2.sec1.sec7.p1" name="article1.body1.sec2.sec1.sec7.p1"></a><p>The above information-based metrics and simple associated analysis algorithm (see <a href="#s4">Materials and Methods</a>) are useful in that they transform the ‘lay’ questions–“Do cells ‘remember’ past experiences and use these memories to modify future stress response dynamics?” and “If so, is this ‘memory’ short term or long term, and how much is there?”–into well-defined queries about information and uncertainty yielding quantitative estimates of microbial memory in informational entropy <em>bits</em>.</p>
<a id="article1.body1.sec2.sec1.sec7.p2" name="article1.body1.sec2.sec1.sec7.p2"></a><p>However, any attempt to quantify or qualify memory is fundamentally limited by the possibility of unobservable states (see <a href="#pone-0001700-g004">Fig. 4c</a>), uncontrolled and unobservable inputs, poor choice of input combinations and sequences, and measurement errors and distortions. Here we assume most such limitations, discussed in more detail in Supplementary Information (<a href="#pone.0001700.s002">Section S1</a>), are inherent in the estimation of memory processes and most likely to result in information loss and thus <em>underestimates</em> of the ability of the system to ‘remember’ the cell histories tested by the experimental compendium. Therefore we interpret quantifications of memory within our <em>B. subtilis</em> compendium as lower bound estimates.</p>



<h4>Experiment and Overview of Analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Memory experiment on <em>B. subtilis</em>: To test for history dependent behavior–‘memory’-in <em>B. subtilis</em>, we engineered a fluorescently labeled strain of <em>Bacillus subtilis</em> to report on commitment to sporulation and degradative enzyme synthesis: the KEE strain (P<em>spoIIE-gfp</em>, P<em>aprE-dsred cmp</em>, see <a href="#s4">Materials and Methods</a> for details on strain construction). The <em>spoIIE</em> promoter (P<em>spoIIE</em>), our sporulation reporter, controls expression of <em>spoIIE</em>, a gene encoding a serine phosphatase specifically expressed upon commitment to sporulation and therefore considered a good sporulation commitment signal <a href="#pone.0001700-Arigoni1">[55]</a>, <a href="#pone.0001700-Stragier1">[56]</a>. The <em>aprE</em> promoter (P<em>aprE</em>), our degradative enzyme synthesis reporter, controls expression of the extracellular protease subtilisin naturally produced by <em>B. subtilis</em> cells at the end of exponential growth <a href="#pone.0001700-Stahl1">[57]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>With the KEE reporter strain, we used our framework to estimate, in informational entropy bits, the capacity of these stress response pathways and of the cell growth dynamics to ‘remember’ 10 distinct cell histories prior to application of a common stressor. Specifically, we first grew three replicate cultures in one of two media, Luria Broth medium (LB) or growth medium (GM) <a href="#pone.0001700-Harwood1">[58]</a>, to one of five different densities (all still in exponential growth, ranging from OD<sub>600</sub> = [0.1:1], see <a href="#pone-0001700-t001">Table 1</a>, where OD<sub>600</sub> is the optical density of the culture at 600nM), for a total of ten cell histories. Thus in the first stage of the experiment, a clonal population of cells was divided into 30 groups, each of which experienced one of the 10 cell histories consisting of growth in one of two media to one of five cell densities over a fixed period of time (see <a href="#s4">Materials and Methods</a> for details).</p>
<div class="figure" id="pone-0001700-t001"><div class="img"><a name="pone-0001700-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Cell history table.</span></strong></p><span>doi:10.1371/journal.pone.0001700.t001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We chose to combine different media with growth to different densities as our set of cell histories because growth media can impact cell state, as can growth of cultures to different densities over a fixed period of time. Cells deplete nutrients and respond to the environment and its dynamics with changes in metabolic fluxes, post-translational modifications, gene expression, quorum signaling and synthesis of storage compounds. GM medium (also called CH medium) is a rich medium with casein hydrolysate as the sole carbon source <a href="#pone.0001700-Harwood1">[58]</a>. LB medium is a much richer and more complex medium than GM and therefore sustains more rapid growth. We assumed that any resulting history-dependent differences in cell state at time t0 might lead to different history-dependent behaviors in the common medium after t0.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>After experiencing one of the 10 different cell histories, cells were then pelleted and resuspended at an intermediate density (OD<sub>600</sub> = 0.5) in a common stress medium, in this case, sporulation salts starvation medium (SM) <a href="#pone.0001700-Harwood1">[58]</a>. The resuspension time is denoted t0. Thus, regardless of past experiences, all cells observed after t0 were subjected to starvation conditions starting at t0 in a fixed-density, fixed-size population.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Our three observables <em>Y</em> after t0 consisted of two fluorescent reporters, one for sporulation initiation and another for degradative enzyme synthesis (strain KEE (P<em>spoIIE-gfp</em>, P<em>aprE-dsred cmp</em>)), and optical density of the culture as a proxy for cell growth (OD<sub>600</sub>), measured at the bulk population level every 15 minutes for 24 hours starting at time t0 (see <a href="#pone-0001700-g005">Fig. 5</a> for time series, and <a href="#s4">Materials and Methods</a> for details on strain construction and experiments). Thus, with 30 cultures–three for each of the 10 cell histories– and three observables per culture measured every 15 minutes for 24 hours in the common stress medium starting at t0, the memory data compendium for this set of experiments consists of 30×3×96 = 8,640 measurements arranged in a 90 by 96 matrix.</p>
<div class="figure" id="pone-0001700-g005"><div class="img"><a name="pone-0001700-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span><em>B. subtilis</em> memory data compendium.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>These plots show the dynamics of the sporulation initiation reporter P<em>spoIIE-gfp expression</em> (a), the degradative enzyme synthesis reporter P<em>aprE-dsred</em> expression (b), and cell growth (c) of <em>B. subtilis</em> KEE after the onset of starvation (resuspension in SM) as a function of cell history prior to starvation, as measured by fluorescence (GFP, and DsRed) and OD<sub>600</sub> time series measurements taken every 15 minutes for 24 hours, respectively. The 10 cell histories tested consisted of growth in either rich LB medium or poorer GM medium to one of five densities D, −1, −2, −3, −4, (see experimental overview section and <a href="#s4">Materials and Methods</a> for details). Fluorescent intensities in (a–b) were divided by OD600 (c) and then normalized to a [0 1] scale by dividing by the maximum. The error bars show standard deviation over replicates at each time point.</p>
<span>doi:10.1371/journal.pone.0001700.g005</span></div><a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>Data analysis overview: The resulting memory data compendium was then analyzed for short- and long-term memory in each output signal individually and in all possible combinations of the three signals by applying the memory quantification algorithm described in detail in <a href="#s4">Materials and Methods</a> and illustrated in the flow chart in Supplementary Information <a href="#pone.0001700.s003">Section S2</a>.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>To briefly summarize, in order to estimate how much short-term and long-term memory was manifested in the behavior of the reporters, we sought to calculate the mutual information between the behavior of the cells after t0 and the history of the cells before t0. This calculation required that we estimate the joint probability density between cellular behavior after t0 and cell history prior to t0. Given constraints on the amount of data and other considerations described in detail in <a href="#pone.0001700.s004">Section S3</a> of Supplementary Information, we took a clustering approach to this problem. That is, we first clustered the response of the pathway reporter as a way of dividing the trajectories into groups with common, distinct behaviors. The resulting assignment of each trajectory to a cluster was then used to calculate the frequency of co-occurrence of each behavioral class and each possible cell history. From this histogram we estimated the requisite joint probability distribution, which was then used to calculate the mutual information between cell history and the behavior of the observable, and thus arrive at an estimate for memory.</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>We performed this procedure on the 30 trajectories (3 replicates for each of the 10 cell histories tested) of each of the three observables, using both the short term (first 11 hours of measurements, during which the signal was still dynamically varying-see <a href="#s4">Materials and Methods</a> for more details on our choice of analysis intervals) and long-term response (last three hours of measurements, from 21 to 24 hours, by which time the signals have remained flat for several hours) in order to estimate short-term and long-term memories manifested in each individual signal. To calculate the short-term and long-term memory in the combined activities of multiple signals, we took the same approach, with the one difference being that the clustering step captured the combined behavior of multiple readouts (Step 3 in the algorithm in <a href="#s4">Materials and Methods</a>). All bit counts were then normalized to calculate memory fidelities and orthogonalities, as defined in <a href="#pone.0001700.s001">Appendix S1</a>, in order to estimate in relative terms how much of the total possible memory each system ‘remembers’, and how much ‘extra’ memory is embedded in the higher-dimensional spaces formed by multiple pathways.</p>
<a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>Since the 30 populations were subjected to 10 different (within error) past conditions <em>M</em> = (<em>Medium1, Density1</em>) in equal proportions, the informational entropy of the cell history space M is H(<em>M</em>) = −log<sub>2</sub>(1/10) = 3.3219 bits. Thus, without prior knowledge there are 3.3219 bits of information about cell history at most that can be recovered from observation of these three outputs, either individually or in combination and on any time scale.</p>


<h4>Experimental Results</h4>
<h5>A qualitative overview of history-dependence.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>The <em>B. subtilis</em> stress responses measured by the three observables (<a href="#pone-0001700-g005">Figure 5</a>) appear neither memoryless nor in possession of a perfect memory of the cell histories tested. They do not appear to be memoryless because not all signals from a given observable follow a common trajectory (within noise bounds) irrespective of past history of the cells. Nor does the memory of any observable appear to be perfect, because though there are ten distinct cell histories prior to time t0, there appear to be fewer than ten distinct dynamics per observable in response to the starvation stressor administered at time t0. By eye, there appear to be more distinct behaviors in the short term than in the long term. Also, different cell histories group together for different observables. This means that we expect a higher bit count estimate of short term memory than long term memory, and different amounts of memory and of different aspects of cell history in the three pathway observables.</p>

<h5>All observables exhibit short-term memory of cell history, with sporulation exhibiting the most and growth dynamics the least.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>The transient behavior (first 11 hours) of the SpoIIE (sporulation) reporter clusters into five distinct classes of behavior (different onset times and sigmoidal vs. more pulsatile expression), whereas the transient behavior of the AprE (degradative enzyme synthesis) reporter clusters into three classes (different onset times and different expression levels) and the growth reporter into just two classes (some vs. almost no growth) (see left panels of <a href="#pone-0001700-g006">Fig. 6a,b,c</a>). The mutual information between the resulting clustering vectors and the cell history vector captures how well the different behavioral classes of each observable correspond to different cell histories. Performing this calculation, we estimate I<sub>trans(spo)</sub> = <em>1.96 bits</em> of short-term memory in the sporulation reporter; I<sub>trans(AprE)</sub> = <em>1.4855 bits</em> of short-term memory in the degradative enzyme synthesis reporter, and I<sub>trans(OD)</sub> = <em>1 bit</em> of short-term memory in the growth dynamics reporter OD<sub>600</sub>. Thus, all three observables exhibit short-term memory of the cell histories tested, with the sporulation reporter exhibiting the most memory and growth dynamics the least.</p>
<div class="figure" id="pone-0001700-g006"><div class="img"><a name="pone-0001700-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>The map from cell history to <em>B. subtilis</em> stress response clusters.</span></strong></p><a id="article1.body1.sec2.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec2.fig1.caption1.p1"></a><p>The transient dynamics and long-term levels of the sporulation initiation (P<em>spoIIE-gfp</em> expression), AprE synthesis (P<em>aprE-dsred</em> expression), and growth (OD<sub>600</sub>) signals were clustered using the automatic method in <a href="#s4">Materials and Methods</a>. This figure shows the heat maps for each signal in <a href="#pone-0001700-g005">Figure 5</a> (dark red indicates maximum, and dark blue minimum), the number of behavioral classes for each signal, and which subset of the ten cell histories in our test set corresponds to each cluster. For example, the asymptotic sporulation initiation signal from P<em>spoIIE</em>-<em>gfp</em> fusion clustered into two classes, one (top, 1) corresponding to a history of growth in rich LB medium to the three highest densities, D, −1, and −2, and the other class (bottom, 2) corresponding to all other cell histories.</p>
<span>doi:10.1371/journal.pone.0001700.g006</span></div><a id="article1.body1.sec2.sec3.sec2.p2" name="article1.body1.sec2.sec3.sec2.p2"></a><p>Dividing these absolute bit counts by the entropy of the cell history space, we estimate the short-term memory fidelities of sporulation initiation, degradative enzyme synthesis, and growth dynamics to be P<sub>trans(spo)</sub> = I<sub>trans(spo)</sub>/H(M) = 1.96/3.3219 = 0.59, P<sub>trans(AprE)</sub> = I<sub>trans(AprE)</sub>/H(M) = 1.48/3.3219 = 0.45, and P<sub>trans(OD)</sub> = I<sub>trans(OD)</sub>/H(<em>M</em>) = 1/3.32≈0.3, respectively. This means that if one were to observe all 30 short-term responses of one of the three reporters after t0 but not told which history corresponds to which trajectory, 59% of the uncertainty about cell history prior to time t0 could be reduced by observation of the transient sporulation reporter dynamics after time t0, 45% of this uncertainty about the past could be reduced by observation of the degradative enzyme synthesis reporter dynamics after t0, and only 30% of this uncertainty could be reduced by observation of the growth dynamics after t0. More intuitively, one could say that 59%, 45% and 30% of the cell histories tested are ‘remembered’ by the short-term dynamics of the sporulation, degradative enzyme synthesis, and growth reporters, respectively (see <a href="#pone-0001700-g007">Fig. 7</a> and <a href="#pone.0001700.s005">Table S1</a>).</p>
<div class="figure" id="pone-0001700-g007"><div class="img"><a name="pone-0001700-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001700" data-uri="info:doi/10.1371/journal.pone.0001700.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001700.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001700.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001700.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001700.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Estimates of cell-history memory and mutual information in <em>B. subtilis.</em></span></strong></p><a id="article1.body1.sec2.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec3.sec2.fig2.caption1.p1"></a><p>The upper left bar plot shows our estimate of long-term (blue bars) and short-term (first 11 hours, red bars) memory fidelity (% of the maximum recoverable information about cell history) exhibited in starvation medium SM by sporulation initiation (P<em>spoIIE-gfp</em> expression), degradative enzyme synthesis (P<em>aprE-dsred</em> expression), and growth dynamics (OD<sub>600</sub>), and over all vector pairs of observable read-outs and the vector triple, with respect to the cell history space tested by our compendium. The lower right bar plot shows our estimate of the number of bits of mutual information shared by all pairs of short-term (red bars) and long-term (blue bars) observable signals in our memory data compendium. The surrounding flow diagram circuit illustrates the experimental and analytical scenario.</p>
<span>doi:10.1371/journal.pone.0001700.g007</span></div>
<h5>All observables exhibit long-term memory of cell history, though at a lower bit count than short-term memory.</h5><a id="article1.body1.sec2.sec3.sec3.p1" name="article1.body1.sec2.sec3.sec3.p1"></a><p>Though short term memory can be important—because even short term behavioral differences may have fitness consequences <a href="#pone.0001700-Collins1">[59]</a>, especially if they are on the order of environmental fluctuations <a href="#pone.0001700-Wolf2">[28]</a>, <a href="#pone.0001700-AncelMeyers1">[60]</a>—long term memory is generally the first thing that comes to mind when memory is discussed <a href="#pone.0001700-Gazzaniga1">[61]</a>–<a href="#pone.0001700-Huang1">[64]</a>. One might expect long term memory in <em>B. subtilis</em> stress responses-sporulation control especially-because of the feedback topologies in their regulatory circuitry and reportedly bistable behaviors <a href="#pone.0001700-Dubnau1">[10]</a>, <a href="#pone.0001700-Veening1">[36]</a>–<a href="#pone.0001700-Voigt1">[39]</a>.</p>
<a id="article1.body1.sec2.sec3.sec3.p2" name="article1.body1.sec2.sec3.sec3.p2"></a><p>To estimate the long term memory in each individual pathway we first clustered the final segment of the 30 time series of each reporter (from 21 to 24 hours after t0) to estimate the number of distinct long-term behaviors for each of the three pathway reporters (results = <em>2</em> unequal-sized clusters for each reporter, as shown in <a href="#pone-0001700-g006">Fig. 6</a>, though the cluster sizes and associated cell histories differ across reporters). We then calculated the mutual information between the clustering results and the cell history vector to arrive at lower bound estimates of I<sub>asym(spo)</sub> = <em>0.8813 bits,</em> I<sub>asym(AprE)</sub> = <em>0.72 bits</em>, and I<sub>asym(OD)</sub> = <em>0.97 bits</em> of long-term memory in the networks controlling sporulation initiation, AprE synthesis and growth dynamics, respectively. Thus, like a switch, there appear to be two, stable, long term behaviors for each pathway reporter, though the probability of converging to each is not equal or the same across reporters, as is reflected by distinct bit counts of less than 1 (if half the past histories lead to one attractor and the other half of the histories lead to the other, there would be 1 bit of asymptotic memory).</p>
<a id="article1.body1.sec2.sec3.sec3.p3" name="article1.body1.sec2.sec3.sec3.p3"></a><p>Dividing these absolute numbers by the entropy of the cell history space, we estimate the long-term memory fidelities of sporulation initiation, degradative enzyme synthesis, and growth to be P<sub>asym(spo)</sub> = I<sub>asym(spo)</sub>/H(<em>M</em>) = 0.8813/3.3219 = 0.265, P<sub>asym(AprE)</sub> = 0.22, and P<sub>asym(OD)</sub> = 0.29, respectively. Thus, approximately 25% of the uncertainty about cell history prior to the onset of starvation is reduced by knowledge of any one of the three long-term reporter dynamics in the starvation environment. To summarize, all three observables exhibit around 1 bit of long-term memory of the histories tested, though of different aspects of cell history as will be shown below. One bit is a significant amount but much less than the nearly 2 bits of memory seen in the most retentive short-term response.</p>

<h5>Different observables remember different aspects of cell history to different degrees.</h5><a id="article1.body1.sec2.sec3.sec4.p1" name="article1.body1.sec2.sec3.sec4.p1"></a><p>The above memory estimates are in a sense high-level, because each of the 10 distinct cell histories is treated identically. By drilling down a level of resolution to the component parts of the cell histories–initial nutrient composition of the media and cell density reached in that media (which can also feasibly affect both the nutritional composition of the medium and cell state while in log phase)–we can investigate which aspects of cell history are remembered by the observables and for how long.</p>
<a id="article1.body1.sec2.sec3.sec4.p2" name="article1.body1.sec2.sec3.sec4.p2"></a><p>In the short term, all three observables have a perfect memory of whether they were grown in LB or GM, and only a partial memory of their density in this medium. Put more formally, if we consider growth medium in isolation and calculate the mutual information between growth medium prior to time t0 and transient response of the three reporters to starvation after time t0, we see that a history of growth in LB can be distinguished from a history of growth in GM with 100% memory fidelity (P<sub>trans(Spo)</sub>(<em>Medium1</em>;<em>Y</em>;<em>t<sub>tran</sub></em><sub>s</sub> = 11 hrs) = P<sub>trans(AprE)</sub>(<em>Medium1</em>;<em>Y</em>;<em>t<sub>tran</sub></em><sub>s</sub> = 11 hrs) = P<sub>trans(OD)</sub>(<em>Medium1</em>;<em>Y</em>; <em>t<sub>tran</sub></em><sub>s</sub> = 11 hrs) = 1, where <em>Medium1</em> is a random variable representing growth medium prior to time t0, and can take on the values GM or LB). In contrast, the ability of the pathways to remember the population density reached prior to t0 (and any changes in cell state these differences in cell density create) is less simple. With a history of growth in GM, the cell density prior to the onset of starvation at t0 is not ‘remembered’ by the short-term <em>B. subtilis</em> sporulation, degradative enzyme synthesis, or growth dynamics responses, even transiently (0% memory fidelity), as all responses are indistinguishable within noise (I<sub>trans</sub>(<em>Density1/Medium1</em> = GM;<em>Y</em>; <em>t<sub>tran</sub></em><sub>s</sub> = 11hrs) = 0). However, when grown in LB, the cell density prior to t0 is remembered with 80% memory fidelity by the transient sporulation dynamics and with 60% memory fidelity by the transient AprE dynamics (P<sub>trans(Spo)</sub>(<em>Density1</em>/<em>Medium1</em> = LB;<em>Y</em>; <em>t<sub>tran</sub></em><sub>s</sub> = 11 hrs) = 0.8; P<sub>trans(AprE)</sub>(<em>Density1</em>/<em>Medium1</em> = LB;<em>Y</em>; <em>t<sub>tran</sub></em><sub>s</sub> = 11 hrs) = 0.6).</p>
<a id="article1.body1.sec2.sec3.sec4.p3" name="article1.body1.sec2.sec3.sec4.p3"></a><p>In the long term, all three observables have only a partial memory of which medium they were grown in, and to what density. Like in the transient memory case, past growth medium is remembered better than past cell density, but unlike in the transient memory case, there is no perfectly clean dividing line separating out the long-term responses to the two growth media histories. For example, given observations of the long-term behavior of the sporulation reporter, a history of growth in LB can be distinguished from a history of growth in GM with only <em>39% memory fidelity</em> (I<sub>asym(spo)</sub>(<em>Medium1</em>;<em>Y</em>)/H(<em>Medium1</em>) = 0.39), whereas cell densities (grouped into five classes, (D,-1,-2,-3 and -4)) prior to t0 are remembered even less well, with only 12.1% memory fidelity (I<sub>asym(spo)</sub>(<em>Density1</em>;<em>Y</em>)/H(<em>Density1</em>) = 0.121). A similar pattern can be seen in the long-term memories of the other two reporters. Interestingly, though each reporter exhibits two possible long-term behaviors, the clusters are different sizes and the histories that correspond to each behavioral cluster are different for different pathways. As will be shown in the next section, these differences lead to the possibility of an increased memory capacity in the higher dimensional space defined by the combined activities of multiple pathways.</p>

<h5>There is more long-term memory in the combined activity of the observables than is present in any individual observable.</h5><a id="article1.body1.sec2.sec3.sec5.p1" name="article1.body1.sec2.sec3.sec5.p1"></a><p>Interestingly, analysis of the <em>transient memory</em> of the pairs of pathway readouts (Spo, AprE), (Spo, OD<sub>600</sub>), and (AprE, OD<sub>600</sub>)) shows <em>no increase in memory in the higher dimensional space</em> than is found in the most retentive pathway in the dyad (see <a href="#pone-0001700-g007">Figure 7</a>). For example, we estimate the transient memory found in the pair (AprE, OD<sub>600</sub>) to be 1.4855 bits, which is the same bit count found in AprE alone (<em>Mem<sub>orth(trans)</sub></em>(<em>M</em>;(<em>AprE, OD<sub>600</sub></em>)) = (1.4855-1.4855)/(1.4855) = 0). Likewise, the three-dimensional readout (Spo, AprE, OD<sub>600</sub>) shows no more transient memory than is found in the sporulation pathway (1.96 bits), its most retentive member.</p>
<a id="article1.body1.sec2.sec3.sec5.p2" name="article1.body1.sec2.sec3.sec5.p2"></a><p>However, the same conclusion does not follow for asymptotic memory. <em>Every pair of pathway readouts contains more asymptotic memory than either constituent signal, and the triple pathway readout contains more asymptotic memory (at 1.57 bits) than any of the constituent pairs</em> (see <a href="#pone-0001700-g007">Figure 7</a>). This implies that the long term behavior of our three observables occupies a relatively high dimensional space, with each subsystem responding differently to aspects of past conditions. For example, though the AprE pathway is estimated to have only 0.7219 bits of asymptotic memory and the growth measure OD<sub>600</sub> has only 0.971 bits of asymptotic memory, the pair (AprE, OD<sub>600</sub>) has 1.371 bits of asymptotic memory (<em>Mem<sub>orth(asym)</sub></em>(<em>M</em>;(<em>AprE</em>, <em>OD<sub>600</sub></em>)) = (1.371-0.971)/(0.7219) = 0.554, or 55.4% of the maximum). Put more concretely, the asymptotic behavior of the AprE signal alone ‘remembers’ two classes of cell history: the first a history of growth in rich medium to higher densities and the second all other histories in the compendium. Whereas observations of the asymptotic behavior of the growth signal allow distinction between two <em>different</em> classes of cell history; the first growth in rich medium to all densities greater than the lowest tested (-4), and the second all other histories in the compendium. Viewed together as a combined vector in a higher dimensional space, the asymptotes of the pair (AprE, OD<sub>600</sub>) permit distinction between three classes of cell history: growth in rich medium to higher cell densities, growth in rich medium to low (but not lowest) and intermediate cell densities, and, finally, growth in rich medium to the lowest density or growth in poorer medium to any density. Adding the sporulation signal increases the information storage yet again, by adding another discernable class, leading to a total long-term combinatorial storage of 1.57 bits. Thus, because the different cellular systems in <em>B. subtilis</em> remember different aspects of prior history, the combined activity of multiple pathways is able to combinatorially store more information about the past than can any individual pathway. However, the total asymptotic memory is still somewhat less than the total transient memory (1.57 vs. 1.96 bits). (For a complete accounting of cell history memory over all signal combinations, and for the mutual information between all pairs of signals, including the transient and asymptotic responses of each signal, see <a href="#pone-0001700-g007">Figure 7</a> and <a href="#pone.0001700.s005">Table S1</a>.)</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Though evidence that bacterial cells are able to remember their histories and use these memories to alter their behavior in a fitness enhancing manner would not raise expectations that bacteria could recite π or write music, it <em>would</em> enrich the motifs-modules-games view of bacterial regulation <a href="#pone.0001700-Wolf1">[12]</a> by adding game strategies <em>with memory</em> to the repertoire of microbes. This exploratory paper does not provide evidence that <em>B. subtilis</em>, or any other microbe, is intelligent or is playing an evolved, fitness-enhancing memory strategy. Rather, in this work we propose that the familiar phenomenon of history-dependent behavior in microbes reflects a form of memory worth studying systematically and quantifying, and that doing so sets the foundation for understanding both the mechanisms and function of memory in cell behavior and fitness. To this end we formulated a conceptual information-theory based framework for measuring microbial memory, thereby introducing tools that begin to observe and quantify the relationship between past cell history and future cell behavior from a new angle. This method produces a phenomenological measure of cellular memory without regard to the specific cellular mechanisms encoding it.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We then applied these tools to a simple set of medium-shift experiments on <em>B. subtilis</em>, in the process demonstrating that <em>B. subtilis</em> does ‘remember’, both in the short and long term, aspects of its cell history, and that this memory is distributed differently among the observables. More short term than long term memory was evident, with short-term sporulation dynamics exhibiting the most memory and long-term degradative enzyme AprE synthesis dynamics the least. As expected, some but not all of the history-dependence between the sporulation and AprE reporters is shared (AprE has 75% of the short-term and 80% of the long-term memory estimated for sporulation). We also illustrated how to quantify memory in multiple combined variables, in the process showing that because the different cellular systems in <em>B. subtilis</em> remember different aspects of prior history to different degrees, the combined activity of multiple pathways is able to combinatorially store more information about the past than can any individual pathway. Of the two components of cell history varied in our compendium–past growth medium and the cell density reached in this medium, which can alter cell state even in log phase–growth medium appeared to be better remembered by <em>B. subtilis</em>, with past density remembered best when originally grown in the medium richest with nutrients, LB. Admittedly we do not yet know whether the memory we have observed is fitness enhancing and evolved or just incidental, or what molecular mechanisms or artifacts are responsible for the observed pattern of memory storage. Rather, these simple experiments and the surrounding analysis and framework demonstrate what could be the beginning of a larger memory program, and indicate that memory in cellular behaviors may be a rich area for further exploration.</p>

<h4>Ideas for a more complete memory-in-microbes research program</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>A more complete program for investigating memory in bacteria would encompass at least three lines of inquiry, essentially the ‘what’, ‘how’, and ‘why’ of bacterial memory. The first line of inquiry (what), for which this study is an example, is the quantification of environmental memory in a microbe. This study could be extended by resolving the population-averaged behavior analyzed in this paper into single-cell measurements and memory classification and quantification. Given that sporulation is thought to be a stochastically triggered bistable developmental process <a href="#pone.0001700-Dubnau1">[10]</a>, <a href="#pone.0001700-Veening1">[36]</a>–<a href="#pone.0001700-Voigt1">[39]</a>, one might expect the population-averaged measurements (<a href="#pone-0001700-g005">Figure 5.b</a>) to resolve into bimodal distributions of high and low GFP-expressing cells. And since AprE synthesis control is believed to be more deterministic and analog, one might expect more monomodal distributions. Preliminary data from flow cytometry analysis support this expectation, at least for some histories and time points (see <a href="#pone.0001700.s007">Figure S1</a> in Supplemental Information), but further work is needed to determine for what conditions and pathways memory at the single-cell level can be classified as stochastic, and the form and quantification of this stochasticity. An exploration of the memory characteristics of other cellular players active in these and interacting networks, and the space of their environmental sensitivity, with the goal of estimating the ‘true’ memory capacity of the system, are other possible extensions of this work.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>A second line of inquiry (how) would build upon the first by elucidating the causal basis for any observed environmental memory. Though many genetic and epigenetic bacterial switching mechanisms have been elucidated <a href="#pone.0001700-Hallet1">[8]</a>, <a href="#pone.0001700-Dubnau1">[10]</a>, <a href="#pone.0001700-Ferrell1">[16]</a>, still unclear is exactly how different types of environmental and intercellular signals might be encoded and remembered within cellular circuitry for varying lengths of time, a question addressable through mutant studies and modeling. On the ‘meta’ level one could ask whether memory is stored within single cells, population distributions, or in the larger state space defined by the cell-environment interaction through distributions of nutrients, waste products, enzymes, signaling molecules, biofilm generating conditions, and so on. A third line of inquiry could focus on the ‘why’s' of environmental memory. Is environmental memory, if it exists, controlled or incidental: evolutionarily advantageous, deleterious, or neutral? Is there evidence that memory-modulation of phenotype expression control does not provide a fitness advantage in the <em>present</em> but rather in a <em>future</em> implicitly anticipated from past experiences, thus implying an internal model of environmental dynamics (in analogy to the internal model principle in control <a href="#pone.0001700-Bengtsson1">[65]</a>)? We suspect that answers to these ‘why’ questions could be key to whether the others are worth deeply pursuing.</p>


<h4>What do the <em>B. subtilis</em> memory observations in this case study mean?</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Though we do not yet know whether the memory we have observed is fitness enhancing and evolved, or merely incidental, we can speculate. Looking qualitatively at the three behavioral observables together, we notice that when cells are grown to low density in the less rich GM medium prior to the onset of starvation conditions, they on average grow very fast after resuspension in starvation media, and after a brief lag start turning on their degradative enzyme synthesis and their probabilistic sporulation machinery, even as the population continues to grow. Whereas when cells are grown in richer, nutrient filled LB medium to the same low density prior to the onset of starvation conditions, they take a quite different approach. In this case, cells seem to adopt a wait-and-see strategy, forgoing growth and delaying sporulation and AprE synthesis for many hours.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>A game strategy with memory?: The most tempting speculation is that <em>B. subtilis</em> is playing a memory strategy in an evolutionary game. From a game perspective, one could take these observations as a sign that after transitioning from a less rich medium to starvation, <em>B. subtilis</em> uses its memory of past nutrient-limited growth in the context of an implicit internal model of environmental dynamics to ‘predict’ how long starvation conditions will last. If the cells expect starvation to last a long time, a rational course of action might be to create as many spores as possible, as fast as possible, to maximize the spore count that will lie dormant until the next period of nutritional plenty. On the other hand, if growth in a rich environment prior to starvation in the context of this internal model produces a prediction of a short period of starvation, the rational action might be to <em>delay</em> sporulation, thereby decreasing the chances of having committed irreversibly to an unnecessary, costly 8 hour developmental program during which conditions could improve and the cells could be growing. Viewed in this way, <em>B. subtilis</em>'s cell-history dependent behavior might constitute an evolved probabilistic memory strategy in its game of survival. Such a strategy would trump diversification strategies without memory <a href="#pone.0001700-Wolf2">[28]</a>–<a href="#pone.0001700-Kussell1">[30]</a>, <a href="#pone.0001700-Nowak2">[35]</a>, <a href="#pone.0001700-Lande1">[66]</a>, <a href="#pone.0001700-Thattai1">[67]</a>, and be analogous to adaptive model-based bet hedging over a diversified portfolio in the stock market <a href="#pone.0001700-Wolf2">[28]</a>.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>If the above scenario is true, one would expect sporulation initiation <em>delay</em> to be a likely feature of the sporulation regulation strategy of <em>B. subtilis</em> to exhibit memory. Within our experimental compendium, the delay in turning on the sporulation machinery, as estimated by the amount of time it takes for GFP to start being noticeably expressed from the SpoIIE promoter by a population (normalized GFP intensity &gt;0.035, after which GFP rapidly increases), ranges from a relatively short 1.5 hours to a much longer nearly 8 or more hours after the onset of starvation (<a href="#pone-0001700-g005">Figs. 5b</a> and <a href="#pone.0001700.s008">S2</a> in Supplemental Information). Calculating the mutual information between GFP expression delays and cell history, we see that most (86%) of the short-term memory in the sporulation reporter can be recapitulated by reducing the trajectories to this single number (I(<em>M; Initiation Delay</em>)/I<sub>trans</sub>(<em>M;Y;t<sub>trans</sub></em> = 11hrs) = 1.685/1.96 ≈ 0.86). This calculation does not prove that the history-dependence we have observed is an evolved and fitness enhancing memory strategy in a game, but it is suggestive.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>…or an artifact of metabolism?: Then again, the explanation could have little to do with evolutionary games. It could be that differences in metabolic stores, housekeeping apparatus, or metabolic state induced by the different media and different biomass of the culture simply represent initial conditions from which entry into sporulation and other stress responses is more or less easy <a href="#pone.0001700-Robinson1">[1]</a>. For example there might be more ribosomes after growth in LB than there are after growth in GM, forcing cells coming from the latter to stop growth and initiate sporulation sooner. Or it could be that growth in GM, a medium that while not nutrient-limited is lacking the excess of simple carbon and nitrogen sources and readily available amino acids found in LB, activates metabolic pathways that can facilitate growth and spore formation in stress conditions. Then, when transferred to starvation conditions, cells might be able to use this metabolic machinery (and perhaps some form of intracellular nutrient storage) to scavenge whatever scarce nutrients are to be found in the new medium in order to grow and turn on their sporulation and degradative enzyme pathways nearly immediately. Whereas with a history of growth in rich, complex LB medium, cells might enter starvation conditions of SM without enzymatic or other reserves necessary for a near-immediate response to severely limited conditions, and thus require a delay while the cells construct the necessary metabolic machinery to acclimate to their environment.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>These possibilities are not mutually exclusive; history-dependent behaviors could stem from some combination of evolved diversification game strategy and artifactual adaptive metabolic processes. Experiments comparing the fitness of wildtype bacteria to mutants with disrupted memory mechanisms coupled to a game theoretic analysis will be necessary to distinguish among the possibilities, and would identify the mechanistic source of memory behaviors in the process. In any case, we hope that this conceptual framework and analytical approach to quantifying memory in cellular behaviors will be a useful point of departure for studying a new set of questions about cellular regulation and evolutionary strategy in microbes.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains and culture media</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>Bacillus subtilis</em> 168 <em>trpC</em> (Bacillus Genetic Stock center) was used as the wild-type strain. <em>Escherichia coli</em> strain DH5α was used for all plasmid amplifications and isolations. <em>Escherichia coli</em> was grown at 37°C in LB supplemented, when necessary, with ampicillin at a final concentration of 100 µg/ml. <em>B. subtilis</em> was cultured at 37°C in either LB, growth medium (GM) or sporulation medium (SM). GM and SM media are commonly used in the ‘induction of sporulation by resuspension protocol’ described by Harwood and Cutting <a href="#pone.0001700-Harwood1">[58]</a> and were supplemented with 50 µg/ml and 20 µg/ml L-tryptophan respectively. Antibiotics were added, with the following final concentrations: chloramphenicol, 5 µg/ml; spectinomycin, 100 µg/ml.</p>


<h4>DNA isolation and manipulation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Total genomic DNA from <em>B. subtilis</em> 168 was isolated with DNeasy Blood &amp; Tissue Kit (Qiagen) following manufacturer's protocol for Gram positive bacteria. Plasmid DNA was extracted from <em>E. coli</em> with the QIAprep kit (Qiagen). DNA restriction and cloning were performed according to standard procedures <a href="#pone.0001700-Sambrook1">[68]</a>. Restriction enzymes and T4 DNA ligase were obtained from New England BioLabs and used according to the manufacturer's instructions. DNA fragments were purified from agarose gels with the QIAquick gel purification kit (Qiagen). <em>Vent</em> DNA polymerase (New England Biolabs) was used for PCRs.</p>


<h4><em>B. subtilis</em> reporter strain construction</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Strains and plasmids are listed in <a href="#pone.0001700.s006">Table S2</a> in Supplemental Information. To integrate the fluorescent reporter fusions in the <em>B. subtilis</em> genome the pLFKEE integration vector was constructed as followed. The GFP variant GFPmut2 <a href="#pone.0001700-Cormack1">[69]</a> was excised from pMF19 <a href="#pone.0001700-Fujita1">[70]</a> by digestion with <em>BamHI/EcoRI</em> enzymes and ligated into pEA18 (a gift from Antje Hofmeister) digested with the same enzymes, to give pLF22. The plasmid pEA18 (<em>cmp</em>, <em>spc</em>) is a vector <a href="#pone.0001700-Quisel1">[71]</a> allowing integration by double cross-over at the <em>amyE</em> locus, with a chloramphenicol selection. The <em>spoIIE</em> promoter (P<em><sub>spoIIE</sub></em>) was amplified by PCR from <em>B. subtilis</em> 168 genomic DNA using primers PspoIIE-D/<em>EcoRI</em> (atcacggaattcaaatcggtttctcttgcagaagccg) and PspoIIEM-R/<em>HindIII</em> (atacaaagcttttatattcgttgcctgtcattatagcg)​,and digested with <em>EcoRI and HindIII</em>, then ligated 5′ of <em>gfpmut2</em> on pLF22 that had been digested with the same enzymes to give pLF25 (P<em><sub>spoIIE</sub></em>-<em>gfp</em>, <em>cmp</em>). The transcriptional profile of the <em>spoIIE</em> gene was verified by total RNA dot blot before and after induction of sporulation to confirm its early and specific expression induction at the onset of sporulation (see <a href="#pone.0001700.s009">Figure S3</a>).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>To obtain the P<em><sub>aprE</sub></em>-<em>dsred</em> fusion, the <em>dsredexpress</em> coding sequence was amplified by PCR from pDsRed-Express (Clontech) using primers DsRed-D/<em>FseI</em> (tacggccggcctaaggaggaactacaaatggcgagcagt​gaggacatcatcaagg)and DsRed-X/<em>EcoRV</em> (agatatcgatcagatctacaggaacaggtggtggcg). The PCR fragment obtained was digested with <em>FseI</em> and <em>EcoRV</em>. A modified version of the <em>aprE</em> promoter (P<em><sub>aprE</sub></em>) (developed and tested in <a href="#pone.0001700-Jan1">[72]</a>) was amplified by PCR from pSG-TTGACA <a href="#pone.0001700-Jan1">[72]</a> using primers PaprESG-D/<em>AgeI</em> (tgaaccggttgtcaaacatgagaattcagcg) and PaprE-R/<em>FseI</em> (caaggccggccaaattcagagtagacttacttaaaagac​).The resulting PCR fragment was digested with <em>AgeI</em> and <em>FseI</em> and ligated with <em>FseI</em>/<em>EcoRV-</em>digested <em>dsredexpress</em> into <em>AgeI/EcoRV</em>-digested pLF25 in a three-point ligation to give pLFKEE (<em>P<sub>spoIIE</sub>-gfp, P<sub>aprE</sub>-dsred, cmp spc</em>). Selection of plasmid constructions in <em>E. coli</em> clones was done by adding ampicillin as described above and correct fusions were verified by sequencing.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>To construct <em>B. subtilis</em> KEE, pLFKEE was transformed into <em>B. subtilis</em> 168 competent cells as previously described <a href="#pone.0001700-Harwood1">[58]</a> and selected on LB solid medium supplemented with chloramphenicol. Integration clones were screened for their <em>amyE</em> phenotype on LB+1% starch solid medium <a href="#pone.0001700-Harwood1">[58]</a>. The inability of the clones obtained to grow on spectinomycin was checked to eliminate single cross-over plasmid integration events. Correct integration of the fusion at the <em>amyE</em> locus was verified by PCR analysis.</p>


<h4>Medium-shift experimental protocol</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Before each experiment, cells were streaked from −80°C glycerol stocks on LB plates with chloramphenicol and grown overnight. One colony was picked and inoculated in 5 ml liquid LB medium with chloramphenicol in a series of dilution tubes and grown overnight at 37°C. The culture the closest to OD<sub>600</sub> of 1.0 was used to inoculate 60 ml of LB or GM in 250-ml flasks to a final OD<sub>600</sub> of 0.05 (flask D) after elimination of the culture medium by centrifugation of the cells (6,000×g, 3 min). The culture was split in two, and successive dilutions of 1:2 were performed to a total of 5 flasks of 30 ml culture (flask D and dilution flasks: -1, -2, -3, -4). Cells in all four flasks were grown simultaneously at 37°C, 200 rpm, until the most concentrated culture grew to an OD<sub>600</sub> of 1.0 (Flask D). Then, 25 ml of each culture were harvested by centrifugation (8,000×g, 5 min) and resuspended in a pre-warmed SM medium volume calculated to obtain a final OD of 0.5 (medium density). Three aliquots of 200 μl from each flask were transferred to a sterile Costar 96-well black plate with flat clear bottom (Corning). Cells in the plate were grown in a Tecan Safire microplate spectrophotometer at 37°C medium linear shaking setting (395 rpm). Culture turbidity (OD<sub>600</sub>) and fluorescence were measured at 15 minutes intervals for a total time of 24 hours. GFPmut2 was read at wavelengths of 481 nm (excitation) and 507 nm (emission), and DsRedexpress was read at 557 nm (excitation) and 579 nm (emission).</p>


<h4>Memory and mutual information analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>There are a number of ways to translate the memory quantification definitions in <a href="#pone.0001700.s001">Appendix S1</a> into an analysis algorithm. We took a simple fixed-interval, clustering-based approach executed as a five-step algorithm implemented the MATLAB© (<a href="http://www.mathworks.com/">http://www.mathworks.com/</a>) analysis environment, as follows (see Supplemental Information <a href="#pone.0001700.s003">Section S2</a> for schematic):</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>(step 0–select time intervals): The first step in analyzing the data is to select time intervals to analyze. We parsed the time series data (30 trajectories measured over 24 hours for each of three observables) into a ‘short-term’ set taken well before steady-state is reached (first 11 hours after t0, the onset of starvation–though we could have taken any endpoint between 8 and 19 hours and obtained the same result (see panel (b) in <a href="#pone.0001700.s004">Section S3</a>)) and an ‘long-term’ set. For our purposes, we take as our ‘proxy’ for long-term, asymptotic behavior the last three hours of our measurements, from 21 to 24 hours after t0, because by then all signals have remained flat for several hours. Experiments run for longer periods of time indicate that these signals remain flat for as long as we have measured them (36 hours, data not shown). However, we view this long-term data set as only a proxy for asymptotic behavior because though these signals remain constant for at least 36 hours, cells are forming spores and might be physiologically changing in other respects during this period and beyond.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>(step 1–cluster data): We used the Matlab script in S2.2 to hierarchically cluster the 30 short-term and 30 long-term trajectories of each observable (10 cell histories×3 replicates) and to select ‘optimal’ clustering partitions for each. The assumption here is that the behavior of the observable (e.g., GFP intensity) falls into distinct classes, for example, increasing or decreasing. This script a) constructs a Euclidean distance matrix with the Matlab function pdist.m, b) constructs dendrograms using ward and average linkage with the function dendrogram.m, c) performs silhouette analysis on all tree cuts of both trees from (b) with the Matlab function silhouette.m <a href="#pone.0001700-Rousseeuw1">[73]</a>, and d) ‘optimizes’ data clustering by selecting the partition that maximizes the mean silhouette, a measure of the compactness and separation of the clusters in the partition <a href="#pone.0001700-Rousseeuw1">[73]</a>. This step produced six 30×1 cluster vectors, one short-term and one long-term cluster vector for each of the three observables (i.e., ClustSPO_short, ClustSPO_long, ClustAprE_short, ClustAprE_long, ClustOD_short, ClustOD_short).</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>(step 2–estimate memory): Next we estimated the short-term and long-term memory in bits of each individual observable with the Matlab program Entropy_MutualInfo.m in S2.1. This program accepts two input vectors, A and B, and calculates from them individual informational entropies H(A) and H(B), the entropy of the pair H(A,B), and the mutual information between A and B, I(A;B) = H(A)+H(B)-H(A,B). H(X) is defined in Supplementary Information (<a href="#pone.0001700.s001">Appendix S1</a>), and H(X,Y) is calculated by first calculating the joint probability distribution over (X,Y) and then calculating the entropy H over this joint distribution. Thus, memory is estimated to be the mutual information between cell history and cell behavior and calculated by calling Entropy_MutualInfo.m with input vectors A = M = [1 1 1 2 2 2 …10 10 10], the cell history vector , and B equal to one of the six cluster vectors from step 1. To calculate memory fidelities, we normalized these memory estimates by dividing by H(M) = 3.32, the entropy of the cell history space.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>(step 3–estimate memory in higher dimensions): The third step of the algorithm is to estimate the short and long-term memory exhibited by the combined activities of pairs of observables and by the triple of observables. To do this, we first used the script in S2.3 to combine cluster vectors from multiple read-outs. This script takes as its input two cluster vectors Clust1 and Clust2 and outputs a combined cluster vector Clust3 (e.g., if Clust1 = ClustSpo_short; and Clust2 = ClustAprE_short; then the output Clust3 is a vector capturing all combined short-term behaviors of Spo and AprE, for example (Spo,AprE) = (increasing, decreasing), (increasing, increasing) or (decreasing, decreasing)). Next, by calling Entropy_MutualInfo.m with inputs A = (the cell history vector M), and B = (the combined cluster vector Clust3), we calculate the mutual information between cell history and cell behavior, and thus the memory exhibited by the combined activity of the vector of observables contributing to Clust3. After computing short- and long-term memory for all four possible vector combinations of the observables, these estimates were divided by H(M) to estimate memory fidelities and normalized according to Definition (6) in Methods to estimate memory orthogonalities. Finally, we (step 4) calculated the mutual information between all pairs of observables using the cluster vectors from (step 1) as inputs to Entropy_MutualInfo.m.</p>
<a id="article1.body1.sec4.sec5.p6" name="article1.body1.sec4.sec5.p6"></a><p>We took this fixed-interval, clustering-based approach because of our desire to focus on how different cell histories can lead to qualitatively different stress response behaviors, and because a much larger data set would be required to use algorithms such as that suggested by Swinney to estimate mutual information at measurement intervals short enough to avoid excessive ‘blurring’ of the time series dynamics <a href="#pone.0001700-Vastano1">[74]</a>, <a href="#pone.0001700-Samoilov1">[75]</a>. See <a href="#pone.0001700.s004">Section S3</a> in Supplemental Information for a detailed discussion of alternative approaches and why we chose the one we did, and <a href="#pone.0001700.s003">Section S2</a> for Matlab scripts and programs, including a note on a bootstrap method for calculating confidence intervals that one could apply to data sets with a sufficient number of replicates (not present in this data set).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001700"></div><a name="pone.0001700.s001" id="pone.0001700.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s001">Appendix S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Memory quantification definitions.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.19 MB PDF)</p>
<a name="pone.0001700.s002" id="pone.0001700.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s002">Section S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Fundamental limitations of memory experiments.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.27 MB PDF)</p>
<a name="pone.0001700.s003" id="pone.0001700.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s003">Section S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Matlab programs and flow chart for memory analysis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.27 MB PDF)</p>
<a name="pone.0001700.s004" id="pone.0001700.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s004">Section S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Alternative strategies for memory calculations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.16 MB PDF)</p>
<a name="pone.0001700.s005" id="pone.0001700.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s005">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Complete set of memory and mutual information calculations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.72 MB PDF)</p>
<a name="pone.0001700.s006" id="pone.0001700.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s006">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Bacillus strain and plasmid table.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.12 MB PDF)</p>
<a name="pone.0001700.s007" id="pone.0001700.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s007">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Example histograms from flow cytometry analysis of <em>B. subtilis</em> strain KEE.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.84 MB PDF)</p>
<a name="pone.0001700.s008" id="pone.0001700.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s008">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Sporulation initiation delay as a function of cell history.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.13 MB PDF)</p>
<a name="pone.0001700.s009" id="pone.0001700.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001700.s009">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">The P<em>spoIIE-gfp</em> fusion activity tracks <em>spoIIE</em> gene expression.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001700.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.25 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank our undergraduate laboratory assistant Tina Tipawan Mahatdejkul, Antje Hofmeister for strains and vectors, and the Arkin and Keasling labs.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DW JK AA LF IB GP. Performed the experiments: LF. Analyzed the data: DW. Wrote the paper: DW AA LF. Other: Developed framework: DW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001700-Robinson1" id="pone.0001700-Robinson1"></a>Robinson TP, Ocio MJ, Kaloti A, Mackey BM (1998) The effect of the growth environment on the lag phase of Listeria monocytogenes. Int J Food Microbiol  44: 83–92.  <ul class="find" data-citedArticleID="1065146" data-doi="10.1016/s0168-1605(98)00120-2"><li><a href="http://dx.doi.org/10.1016/s0168-1605(98)00120-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+the+growth+environment+on+the+lag+phase+of+Listeria+monocytogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+the+growth+environment+on+the+lag+phase+of+Listeria+monocytogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001700-Gawande1" id="pone.0001700-Gawande1"></a>Gawande PV, Griffiths MW (2005) Growth history influences starvation-induced expression of uspA, grpE, and rpoS and subsequent cryotolerance in Escherichia coli O157:H7. J Food Prot  68: 1154–1158.  <ul class="find" data-citedArticleID="1065084"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Growth+history+influences+starvation-induced+expression+of+uspA%2C+grpE%2C+and+rpoS+and+subsequent+cryotolerance+in+Escherichia+coli+O157%3AH7.&amp;auth=&amp;atitle=Growth+history+influences+starvation-induced+expression+of+uspA%2C+grpE%2C+and+rpoS+and+subsequent+cryotolerance+in+Escherichia+coli+O157%3AH7." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Growth+history+influences+starvation-induced+expression+of+uspA%2C+grpE%2C+and+rpoS+and+subsequent+cryotolerance+in+Escherichia+coli+O157%3AH7." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Growth+history+influences+starvation-induced+expression+of+uspA%2C+grpE%2C+and+rpoS+and+subsequent+cryotolerance+in+Escherichia+coli+O157%3AH7.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001700-Rozen1" id="pone.0001700-Rozen1"></a>Rozen Y, Belkin S (2001) Survival of enteric bacteria in seawater. FEMS Microbiol Rev  25: 513–529.  <ul class="find" data-citedArticleID="1065150" data-doi="10.1111/j.1574-6976.2001.tb00589.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6976.2001.tb00589.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Survival+of+enteric+bacteria+in+seawater." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Survival+of+enteric+bacteria+in+seawater.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001700-Casadesus1" id="pone.0001700-Casadesus1"></a>Casadesus J, D'Ari R (2002) Memory in bacteria and phage. Bioessays  24: 512–518.  <ul class="find" data-citedArticleID="1065058" data-doi="10.1002/bies.10102"><li><a href="http://dx.doi.org/10.1002/bies.10102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Memory+in+bacteria+and+phage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Memory+in+bacteria+and+phage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001700-Hoffer1" id="pone.0001700-Hoffer1"></a>Hoffer SM, Westerhoff HV, Hellingwerf KJ, Postma PW, Tommassen J (2001) Autoamplification of a two-component regulatory system results in “learning” behavior. J Bacteriol  183: 4914–4917.  <ul class="find" data-citedArticleID="1065098" data-doi="10.1128/jb.183.16.4914-4917.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.16.4914-4917.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autoamplification+of+a+two-component+regulatory+system+results+in+%E2%80%9Clearning%E2%80%9D+behavior." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autoamplification+of+a+two-component+regulatory+system+results+in+%E2%80%9Clearning%E2%80%9D+behavior.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001700-Hellingwerf1" id="pone.0001700-Hellingwerf1"></a>Hellingwerf KJ (2005) Bacterial observations: a rudimentary form of intelligence? Trends Microbiol  13: 152–158.  <ul class="find" data-citedArticleID="1065096" data-doi="10.1016/j.tim.2005.02.001"><li><a href="http://dx.doi.org/10.1016/j.tim.2005.02.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+observations%3A+a+rudimentary+form+of+intelligence%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+observations%3A+a+rudimentary+form+of+intelligence%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001700-Koshland1" id="pone.0001700-Koshland1"></a>Koshland DE Jr (1977) A response regulator model in a simple sensory system. Science  196: 1055–1063.  <ul class="find" data-citedArticleID="1065112" data-doi="10.1126/science.870969"><li><a href="http://dx.doi.org/10.1126/science.870969" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+response+regulator+model+in+a+simple+sensory+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+response+regulator+model+in+a+simple+sensory+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001700-Hallet1" id="pone.0001700-Hallet1"></a>Hallet B (2001) Playing Dr Jekyll and Mr Hyde: combined mechanisms of phase variation in bacteria. Curr Opin Microbiol  4: 570–581.  <ul class="find" data-citedArticleID="1065092" data-doi="10.1016/s1369-5274(00)00253-8"><li><a href="http://dx.doi.org/10.1016/s1369-5274(00)00253-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Playing+Dr+Jekyll+and+Mr+Hyde%3A+combined+mechanisms+of+phase+variation+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Playing+Dr+Jekyll+and+Mr+Hyde%3A+combined+mechanisms+of+phase+variation+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001700-Holden1" id="pone.0001700-Holden1"></a>Holden NJ, Gally DL (2004) Switches, cross-talk and memory in Escherichia coli adherence. J Med Microbiol  53: 585–593.  <ul class="find" data-citedArticleID="1065100" data-doi="10.1099/jmm.0.05491-0"><li><a href="http://dx.doi.org/10.1099/jmm.0.05491-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Switches%2C+cross-talk+and+memory+in+Escherichia+coli+adherence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Switches%2C+cross-talk+and+memory+in+Escherichia+coli+adherence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001700-Dubnau1" id="pone.0001700-Dubnau1"></a>Dubnau D, Losick R (2006) Bistability in bacteria. Mol Microbiol  61: 564–572.  <ul class="find" data-citedArticleID="1065070" data-doi="10.1111/j.1365-2958.2006.05249.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05249.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bistability+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bistability+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001700-Igoshin1" id="pone.0001700-Igoshin1"></a>Igoshin OA, Price CW, Savageau MA (2006) Signalling network with a bistable hysteretic switch controls developmental activation of the sigma transcription factor in Bacillus subtilis. Mol Microbiol  61: 165–184.  <ul class="find" data-citedArticleID="1065104" data-doi="10.1111/j.1365-2958.2006.05212.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05212.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signalling+network+with+a+bistable+hysteretic+switch+controls+developmental+activation+of+the+sigma+transcription+factor+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signalling+network+with+a+bistable+hysteretic+switch+controls+developmental+activation+of+the+sigma+transcription+factor+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001700-Wolf1" id="pone.0001700-Wolf1"></a>Wolf DM, Arkin AP (2003) Motifs, modules and games in bacteria. Curr Opin Microbiol  6: 125–134.  <ul class="find" data-citedArticleID="1065186" data-doi="10.1016/s1369-5274(03)00033-x"><li><a href="http://dx.doi.org/10.1016/s1369-5274(03)00033-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Motifs%2C+modules+and+games+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Motifs%2C+modules+and+games+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001700-Arkin1" id="pone.0001700-Arkin1"></a>Arkin A, Ross J, McAdams HH (1998) Stochastic kinetic analysis of developmental pathway bifurcation in phage lambda-infected Escherichia coli cells. Genetics  149: 1633–1648.  <ul class="find" data-citedArticleID="1065036"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Stochastic+kinetic+analysis+of+developmental+pathway+bifurcation+in+phage+lambda-infected+Escherichia+coli+cells.&amp;auth=&amp;atitle=Stochastic+kinetic+analysis+of+developmental+pathway+bifurcation+in+phage+lambda-infected+Escherichia+coli+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+kinetic+analysis+of+developmental+pathway+bifurcation+in+phage+lambda-infected+Escherichia+coli+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+kinetic+analysis+of+developmental+pathway+bifurcation+in+phage+lambda-infected+Escherichia+coli+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001700-Chickarmane1" id="pone.0001700-Chickarmane1"></a>Chickarmane V, Troein C, Nuber UA, Sauro HM, Peterson C (2006) Transcriptional dynamics of the embryonic stem cell switch. PLoS Comput Biol  2: e123.  <ul class="find" data-citedArticleID="1065062" data-doi="10.1371/journal.pcbi.0020123.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020123.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+dynamics+of+the+embryonic+stem+cell+switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+dynamics+of+the+embryonic+stem+cell+switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001700-Bagowski1" id="pone.0001700-Bagowski1"></a>Bagowski CP, Ferrell JE Jr (2001) Bistability in the JNK cascade. Curr Biol  11: 1176–1182.  <ul class="find" data-citedArticleID="1065044" data-doi="10.1016/s0960-9822(01)00330-x"><li><a href="http://dx.doi.org/10.1016/s0960-9822(01)00330-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bistability+in+the+JNK+cascade." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bistability+in+the+JNK+cascade.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001700-Ferrell1" id="pone.0001700-Ferrell1"></a>Ferrell JE Jr (2002) Self-perpetuating states in signal transduction: positive feedback, double-negative feedback and bistability. Curr Opin Cell Biol  14: 140–148.  <ul class="find" data-citedArticleID="1065076" data-doi="10.1016/s0955-0674(02)00314-9"><li><a href="http://dx.doi.org/10.1016/s0955-0674(02)00314-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-perpetuating+states+in+signal+transduction%3A+positive+feedback%2C+double-negative+feedback+and+bistability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-perpetuating+states+in+signal+transduction%3A+positive+feedback%2C+double-negative+feedback+and+bistability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001700-Tyson1" id="pone.0001700-Tyson1"></a>Tyson JJ, Chen K, Novak B (2001) Network dynamics and cell physiology. Nat Rev Mol Cell Biol  2: 908–916.  <ul class="find" data-citedArticleID="1065176" data-doi="10.1038/35103078"><li><a href="http://dx.doi.org/10.1038/35103078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+dynamics+and+cell+physiology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+dynamics+and+cell+physiology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001700-Novak1" id="pone.0001700-Novak1"></a>Novak B, Csikasz-Nagy A, Gyorffy B, Chen K, Tyson JJ (1998) Mathematical model of the fission yeast cell cycle with checkpoint controls at the G1/S, G2/M and metaphase/anaphase transitions. Biophys Chem  72: 185–200.  <ul class="find" data-citedArticleID="1065138" data-doi="10.1016/s0301-4622(98)00133-1"><li><a href="http://dx.doi.org/10.1016/s0301-4622(98)00133-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mathematical+model+of+the+fission+yeast+cell+cycle+with+checkpoint+controls+at+the+G1%2FS%2C+G2%2FM+and+metaphase%2Fanaphase+transitions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mathematical+model+of+the+fission+yeast+cell+cycle+with+checkpoint+controls+at+the+G1%2FS%2C+G2%2FM+and+metaphase%2Fanaphase+transitions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001700-Kashiwagi1" id="pone.0001700-Kashiwagi1"></a>Kashiwagi A, Urabe I, Kaneko K, Yomo T (2006) Adaptive response of a gene network to environmental changes by fitness-induced attractor selection. PLoS ONE  1: e49.  <ul class="find" data-citedArticleID="1065108" data-doi="10.1371/journal.pone.0000049"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+response+of+a+gene+network+to+environmental+changes+by+fitness-induced+attractor+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+response+of+a+gene+network+to+environmental+changes+by+fitness-induced+attractor+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001700-Becskei1" id="pone.0001700-Becskei1"></a>Becskei A, Seraphin B, Serrano L (2001) Positive feedback in eukaryotic gene networks: cell differentiation by graded to binary response conversion. Embo J  20: 2528–2535.  <ul class="find" data-citedArticleID="1065048" data-doi="10.1093/emboj/20.10.2528"><li><a href="http://dx.doi.org/10.1093/emboj/20.10.2528" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+feedback+in+eukaryotic+gene+networks%3A+cell+differentiation+by+graded+to+binary+response+conversion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+feedback+in+eukaryotic+gene+networks%3A+cell+differentiation+by+graded+to+binary+response+conversion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001700-Gardner1" id="pone.0001700-Gardner1"></a>Gardner TS, Cantor CR, Collins JJ (2000) Construction of a genetic toggle switch in Escherichia coli. Nature  403: 339–342.  <ul class="find" data-citedArticleID="1065080"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Construction+of+a+genetic+toggle+switch+in+Escherichia+coli.&amp;auth=&amp;atitle=Construction+of+a+genetic+toggle+switch+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+a+genetic+toggle+switch+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+a+genetic+toggle+switch+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001700-AjoFranklin1" id="pone.0001700-AjoFranklin1"></a>Ajo-Franklin CM, Drubin DA, Eskin JA, Gee EP, Landgraf D, et al.  (2007) Rational design of memory in eukaryotic cells. Genes Dev  21: 2271–2276.  <ul class="find" data-citedArticleID="1065030" data-doi="10.1101/gad.1586107"><li><a href="http://dx.doi.org/10.1101/gad.1586107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rational+design+of+memory+in+eukaryotic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rational+design+of+memory+in+eukaryotic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001700-Greber1" id="pone.0001700-Greber1"></a>Greber D, Fussenegger M (2007) Mammalian synthetic biology: engineering of sophisticated gene networks. J Biotechnol  130: 329–345.  <ul class="find" data-citedArticleID="1065088" data-doi="10.1016/j.jbiotec.2007.05.014"><li><a href="http://dx.doi.org/10.1016/j.jbiotec.2007.05.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+synthetic+biology%3A+engineering+of+sophisticated+gene+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+synthetic+biology%3A+engineering+of+sophisticated+gene+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001700-Song1" id="pone.0001700-Song1"></a>Song H, Smolen P, Av-Ron E, Baxter DA, Byrne JH (2006) Bifurcation and singularity analysis of a molecular network for the induction of long-term memory. Biophys J  90: 2309–2325.  <ul class="find" data-citedArticleID="1065166" data-doi="10.1529/biophysj.105.074500"><li><a href="http://dx.doi.org/10.1529/biophysj.105.074500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bifurcation+and+singularity+analysis+of+a+molecular+network+for+the+induction+of+long-term+memory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bifurcation+and+singularity+analysis+of+a+molecular+network+for+the+induction+of+long-term+memory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001700-Morimoto1" id="pone.0001700-Morimoto1"></a>Morimoto BH, Koshland DE Jr (1991) Short-term and long-term memory in single cells. Faseb J  5: 2061–2067.  <ul class="find" data-citedArticleID="1065134"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Short-term+and+long-term+memory+in+single+cells.&amp;auth=&amp;atitle=Short-term+and+long-term+memory+in+single+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Short-term+and+long-term+memory+in+single+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Short-term+and+long-term+memory+in+single+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001700-Acar1" id="pone.0001700-Acar1"></a>Acar M, Becskei A, van Oudenaarden A (2005) Enhancement of cellular memory by reducing stochastic transitions. Nature  435: 228–232.  <ul class="find" data-citedArticleID="1065028" data-doi="10.1038/nature03524"><li><a href="http://dx.doi.org/10.1038/nature03524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancement+of+cellular+memory+by+reducing+stochastic+transitions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancement+of+cellular+memory+by+reducing+stochastic+transitions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001700-MaynardSmith1" id="pone.0001700-MaynardSmith1"></a>Maynard Smith J (1982) Evolution and the theory of games. Cambridge, New York: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0001700-Wolf2" id="pone.0001700-Wolf2"></a>Wolf DM, Vazirani VJ, Arkin AP (2005) Diversity in times of adversity: probabilistic strategies in microbial survival games. J Theor Biol in press.   <ul class="find" data-citedArticleID="1065188" data-doi="10.1016/j.jtbi.2004.11.020"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2004.11.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+in+times+of+adversity%3A+probabilistic+strategies+in+microbial+survival+games." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+in+times+of+adversity%3A+probabilistic+strategies+in+microbial+survival+games.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001700-Wolf3" id="pone.0001700-Wolf3"></a>Wolf DM, Vazirani VJ, Arkin AP (2005) A microbial modified Prisoner's Dilemma game: how frequency-dependent selection can lead to random phase variation. J Theor Biol in press.   <ul class="find" data-citedArticleID="1065190" data-doi="10.1016/j.jtbi.2004.11.021"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2004.11.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microbial+modified+Prisoner%27s+Dilemma+game%3A+how+frequency-dependent+selection+can+lead+to+random+phase+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microbial+modified+Prisoner%27s+Dilemma+game%3A+how+frequency-dependent+selection+can+lead+to+random+phase+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001700-Kussell1" id="pone.0001700-Kussell1"></a>Kussell E, Leibler S (2005) Phenotypic diversity, population growth, and information in fluctuating environments. Science  309: 2075–2078.  <ul class="find" data-citedArticleID="1065122" data-doi="10.1126/science.1114383"><li><a href="http://dx.doi.org/10.1126/science.1114383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+diversity%2C+population+growth%2C+and+information+in+fluctuating+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+diversity%2C+population+growth%2C+and+information+in+fluctuating+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001700-Aumann1" id="pone.0001700-Aumann1"></a>Aumann R, Maschler M (1995) Repeated Games with Incomplete Information. Cambridge: MIT Press.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001700-Basar1" id="pone.0001700-Basar1"></a>Basar T, Olsder GJ (1995) Dynamic Noncooperative Game Theory. London and San Diego: Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0001700-Nowak1" id="pone.0001700-Nowak1"></a>Nowak M, Sigmund K (1993) A strategy of win-stay, lose-shift that outperforms tit-for-tat in the Prisoner's Dilemma game. Nature  364: 56–58.  <ul class="find" data-citedArticleID="1065140" data-doi="10.1038/364056a0"><li><a href="http://dx.doi.org/10.1038/364056a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+strategy+of+win-stay%2C+lose-shift+that+outperforms+tit-for-tat+in+the+Prisoner%27s+Dilemma+game." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+strategy+of+win-stay%2C+lose-shift+that+outperforms+tit-for-tat+in+the+Prisoner%27s+Dilemma+game.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001700-Axelrod1" id="pone.0001700-Axelrod1"></a>Axelrod R, Hamilton WD (1981) The evolution of cooperation. Science  211: 1390–1396.  <ul class="find" data-citedArticleID="1065042" data-doi="10.1126/science.7466396"><li><a href="http://dx.doi.org/10.1126/science.7466396" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+cooperation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+cooperation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001700-Nowak2" id="pone.0001700-Nowak2"></a>Nowak M, Sigmund K (1992) Tit for tat in heterogeneous populations. Nature  355: 250–252.  <ul class="find" data-citedArticleID="1065142" data-doi="10.1038/355250a0"><li><a href="http://dx.doi.org/10.1038/355250a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tit+for+tat+in+heterogeneous+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tit+for+tat+in+heterogeneous+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001700-Veening1" id="pone.0001700-Veening1"></a>Veening JW, Hamoen LW, Kuipers OP (2005) Phosphatases modulate the bistable sporulation gene expression pattern in Bacillus subtilis. Mol Microbiol  56: 1481–1494.  <ul class="find" data-citedArticleID="1065180" data-doi="10.1111/j.1365-2958.2005.04659.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04659.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphatases+modulate+the+bistable+sporulation+gene+expression+pattern+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphatases+modulate+the+bistable+sporulation+gene+expression+pattern+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001700-Grossman1" id="pone.0001700-Grossman1"></a>Grossman AD (1995) Genetic networks controlling the initiation of sporulation and the development of genetic competence in Bacillus subtilis. Annu Rev Genet  29: 477–508.  <ul class="find" data-citedArticleID="1065090" data-doi="10.1146/annurev.ge.29.120195.002401"><li><a href="http://dx.doi.org/10.1146/annurev.ge.29.120195.002401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+networks+controlling+the+initiation+of+sporulation+and+the+development+of+genetic+competence+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+networks+controlling+the+initiation+of+sporulation+and+the+development+of+genetic+competence+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001700-Msadek1" id="pone.0001700-Msadek1"></a>Msadek T (1999) When the going gets tough: survival strategies and environmental signaling networks in Bacillus subtilis. Trends Microbiol  7: 201–207.  <ul class="find" data-citedArticleID="1065136" data-doi="10.1016/s0966-842x(99)01479-1"><li><a href="http://dx.doi.org/10.1016/s0966-842x(99)01479-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=When+the+going+gets+tough%3A+survival+strategies+and+environmental+signaling+networks+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22When+the+going+gets+tough%3A+survival+strategies+and+environmental+signaling+networks+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001700-Voigt1" id="pone.0001700-Voigt1"></a>Voigt CA, Wolf D, Arkin AP (2004) The B. subtilis SIN Operon: An Evolvable Network Motif. Genetics.   <ul class="find" data-citedArticleID="1065182" data-doi="10.1534/genetics.104.031955"><li><a href="http://dx.doi.org/10.1534/genetics.104.031955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+B.+subtilis+SIN+Operon%3A+An+Evolvable+Network+Motif." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+B.+subtilis+SIN+Operon%3A+An+Evolvable+Network+Motif.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001700-Suel1" id="pone.0001700-Suel1"></a>Suel GM, Garcia-Ojalvo J, Liberman LM, Elowitz MB (2006) An excitable gene regulatory circuit induces transient cellular differentiation. Nature  440: 545–550.  <ul class="find" data-citedArticleID="1065172" data-doi="10.1038/nature04588"><li><a href="http://dx.doi.org/10.1038/nature04588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+excitable+gene+regulatory+circuit+induces+transient+cellular+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+excitable+gene+regulatory+circuit+induces+transient+cellular+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001700-Bergman1" id="pone.0001700-Bergman1"></a>Bergman NH, Anderson EC, Swenson EE, Niemeyer MM, Miyoshi AD, et al.  (2006) Transcriptional profiling of the Bacillus anthracis life cycle in vitro and an implied model for regulation of spore formation. J Bacteriol  188: 6092–6100.  <ul class="find" data-citedArticleID="1065052" data-doi="10.1128/jb.00723-06"><li><a href="http://dx.doi.org/10.1128/jb.00723-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+profiling+of+the+Bacillus+anthracis+life+cycle+in+vitro+and+an+implied+model+for+regulation+of+spore+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+profiling+of+the+Bacillus+anthracis+life+cycle+in+vitro+and+an+implied+model+for+regulation+of+spore+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001700-KreuzerMartin1" id="pone.0001700-KreuzerMartin1"></a>Kreuzer-Martin HW, Chesson LA, Lott MJ, Dorigan JV, Ehleringer JR (2004) Stable isotope ratios as a tool in microbial forensics--Part 2. Isotopic variation among different growth media as a tool for sourcing origins of bacterial cells or spores. J Forensic Sci  49: 961–967.  <ul class="find" data-citedArticleID="1065116" data-doi="10.1520/jfs2003227"><li><a href="http://dx.doi.org/10.1520/jfs2003227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stable+isotope+ratios+as+a+tool+in+microbial+forensics--Part+2.+Isotopic+variation+among+different+growth+media+as+a+tool+for+sourcing+origins+of+bacterial+cells+or+spores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stable+isotope+ratios+as+a+tool+in+microbial+forensics--Part+2.+Isotopic+variation+among+different+growth+media+as+a+tool+for+sourcing+origins+of+bacterial+cells+or+spores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001700-KreuzerMartin2" id="pone.0001700-KreuzerMartin2"></a>Kreuzer-Martin HW, Chesson LA, Lott MJ, Dorigan JV, Ehleringer JR (2004) Stable isotope ratios as a tool in microbial forensics--Part 1. Microbial isotopic composition as a function of growth medium. J Forensic Sci  49: 954–960.  <ul class="find" data-citedArticleID="1065118" data-doi="10.1520/jfs2003226"><li><a href="http://dx.doi.org/10.1520/jfs2003226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stable+isotope+ratios+as+a+tool+in+microbial+forensics--Part+1.+Microbial+isotopic+composition+as+a+function+of+growth+medium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stable+isotope+ratios+as+a+tool+in+microbial+forensics--Part+1.+Microbial+isotopic+composition+as+a+function+of+growth+medium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001700-Lim1" id="pone.0001700-Lim1"></a>Lim HN, van Oudenaarden A (2007) A multistep epigenetic switch enables the stable inheritance of DNA methylation states. Nat Genet  39: 269–275.  <ul class="find" data-citedArticleID="1065128" data-doi="10.1038/ng1956"><li><a href="http://dx.doi.org/10.1038/ng1956" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+multistep+epigenetic+switch+enables+the+stable+inheritance+of+DNA+methylation+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+multistep+epigenetic+switch+enables+the+stable+inheritance+of+DNA+methylation+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001700-Ewing1" id="pone.0001700-Ewing1"></a>Ewing JA (1881) The effects of stress on the thermoelectric quality of metals. Proceedings of the Royal Society of London  32: 399–402.  <ul class="find" data-citedArticleID="1065074" data-doi="10.1098/rspl.1881.0041"><li><a href="http://dx.doi.org/10.1098/rspl.1881.0041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+stress+on+the+thermoelectric+quality+of+metals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+stress+on+the+thermoelectric+quality+of+metals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001700-Krasnoselskij1" id="pone.0001700-Krasnoselskij1"></a>Krasnosel'skij MA, Lifshits JA, Sobolev AV (1989) Systems with Hysteresis. Berlin: Springer Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0001700-Sha1" id="pone.0001700-Sha1"></a>Sha W, Moore J, Chen K, Lassaletta AD, Yi CS, et al.  (2003) Hysteresis drives cell-cycle transitions in Xenopus laevis egg extracts. Proc Natl Acad Sci U S A  100: 975–980.  <ul class="find" data-citedArticleID="1065160" data-doi="10.1073/pnas.0235349100"><li><a href="http://dx.doi.org/10.1073/pnas.0235349100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hysteresis+drives+cell-cycle+transitions+in+Xenopus+laevis+egg+extracts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hysteresis+drives+cell-cycle+transitions+in+Xenopus+laevis+egg+extracts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001700-Shannon1" id="pone.0001700-Shannon1"></a>Shannon CE (1948) A mathematical theory of cummunication. Bell Systems Technical Journal  27: 379–656.379-423, 623-656  <ul class="find" data-citedArticleID="1065162" data-doi="10.1002/j.1538-7305.1948.tb00917.x"><li><a href="http://dx.doi.org/10.1002/j.1538-7305.1948.tb00917.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mathematical+theory+of+cummunication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mathematical+theory+of+cummunication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001700-Landauer1" id="pone.0001700-Landauer1"></a>Landauer TK (1986) How much do people remember? Some estimates of the quantity of learned information in long-term memory. Cognitive Science  10: 477–493.  <ul class="find" data-citedArticleID="1065124" data-doi="10.1207/s15516709cog1004_4"><li><a href="http://dx.doi.org/10.1207/s15516709cog1004_4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+much+do+people+remember%3F+Some+estimates+of+the+quantity+of+learned+information+in+long-term+memory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+much+do+people+remember%3F+Some+estimates+of+the+quantity+of+learned+information+in+long-term+memory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001700-Borst1" id="pone.0001700-Borst1"></a>Borst A, Theunissen FE (1999) Information theory and neural coding. Nat Neurosci  2: 947–957.  <ul class="find" data-citedArticleID="1065054" data-doi="10.1038/14731"><li><a href="http://dx.doi.org/10.1038/14731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Information+theory+and+neural+coding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Information+theory+and+neural+coding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001700-DeWeese1" id="pone.0001700-DeWeese1"></a>DeWeese MR, Meister M (1999) How to measure the information gained from one symbol. Network  10: 325–340.  <ul class="find" data-citedArticleID="1065068" data-doi="10.1088/0954-898x/10/4/303"><li><a href="http://dx.doi.org/10.1088/0954-898x/10/4/303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+to+measure+the+information+gained+from+one+symbol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+to+measure+the+information+gained+from+one+symbol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001700-Butts1" id="pone.0001700-Butts1"></a>Butts DA (2003) How much information is associated with a particular stimulus? Network  14: 177–187.  <ul class="find" data-citedArticleID="1065056" data-doi="10.1088/0954-898x/14/2/301"><li><a href="http://dx.doi.org/10.1088/0954-898x/14/2/301" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+much+information+is+associated+with+a+particular+stimulus%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+much+information+is+associated+with+a+particular+stimulus%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001700-Sharpee1" id="pone.0001700-Sharpee1"></a>Sharpee T, Rust NC, Bialek W (2004) Analyzing neural responses to natural signals: maximally informative dimensions. Neural Comput  16: 223–250.  <ul class="find" data-citedArticleID="1065164" data-doi="10.1162/089976604322742010"><li><a href="http://dx.doi.org/10.1162/089976604322742010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analyzing+neural+responses+to+natural+signals%3A+maximally+informative+dimensions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analyzing+neural+responses+to+natural+signals%3A+maximally+informative+dimensions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001700-Ash1" id="pone.0001700-Ash1"></a>Ash RB (1990) Information Theory: Dover Publications.   <ul class="find-nolinks"></ul></li><li><span class="label">55.
              </span><a name="pone.0001700-Arigoni1" id="pone.0001700-Arigoni1"></a>Arigoni F, Duncan L, Alper S, Losick R, Stragier P (1996) SpoIIE governs the phosphorylation state of a protein regulating transcription factor sigma F during sporulation in Bacillus subtilis. Proc Natl Acad Sci U S A  93: 3238–3242.  <ul class="find" data-citedArticleID="1065034" data-doi="10.1073/pnas.93.8.3238"><li><a href="http://dx.doi.org/10.1073/pnas.93.8.3238" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SpoIIE+governs+the+phosphorylation+state+of+a+protein+regulating+transcription+factor+sigma+F+during+sporulation+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SpoIIE+governs+the+phosphorylation+state+of+a+protein+regulating+transcription+factor+sigma+F+during+sporulation+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001700-Stragier1" id="pone.0001700-Stragier1"></a>Stragier P, Losick R (1996) Molecular genetics of sporulation in Bacillus subtilis. Annu Rev Genet  30: 297–241.  <ul class="find" data-citedArticleID="1065170" data-doi="10.1146/annurev.genet.30.1.297"><li><a href="http://dx.doi.org/10.1146/annurev.genet.30.1.297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetics+of+sporulation+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetics+of+sporulation+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001700-Stahl1" id="pone.0001700-Stahl1"></a>Stahl ML, Ferrari E (1984) Replacement of the Bacillus subtilis subtilisin structural gene with an In vitro-derived deletion mutation. J Bacteriol  158: 411–418.  <ul class="find" data-citedArticleID="1065168"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Replacement+of+the+Bacillus+subtilis+subtilisin+structural+gene+with+an+In+vitro-derived+deletion+mutation.&amp;auth=&amp;atitle=Replacement+of+the+Bacillus+subtilis+subtilisin+structural+gene+with+an+In+vitro-derived+deletion+mutation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replacement+of+the+Bacillus+subtilis+subtilisin+structural+gene+with+an+In+vitro-derived+deletion+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replacement+of+the+Bacillus+subtilis+subtilisin+structural+gene+with+an+In+vitro-derived+deletion+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001700-Harwood1" id="pone.0001700-Harwood1"></a>Harwood C, Cutting SM (1990) Molecular Biology Methods for Bacillus. Wiley-Intersciences.   <ul class="find" data-citedArticleID="1065094"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+Biology+Methods+for+Bacillus.&amp;auth=&amp;atitle=Molecular+Biology+Methods+for+Bacillus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+Biology+Methods+for+Bacillus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+Biology+Methods+for+Bacillus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001700-Collins1" id="pone.0001700-Collins1"></a>Collins S, Bell G (2004) Phenotypic consequences of 1,000 generations of selection at elevated CO2 in a green alga. Nature  431: 566–569.  <ul class="find" data-citedArticleID="1065064" data-doi="10.1038/nature02945"><li><a href="http://dx.doi.org/10.1038/nature02945" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+consequences+of+1%2C000+generations+of+selection+at+elevated+CO2+in+a+green+alga." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+consequences+of+1%2C000+generations+of+selection+at+elevated+CO2+in+a+green+alga.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001700-AncelMeyers1" id="pone.0001700-AncelMeyers1"></a>Ancel Meyers L, Bull JJ (2002) Fighting change with change: adaptive variation in an uncertain world. Trends in Ecology &amp; Evolution  17: 551–557.  <ul class="find" data-citedArticleID="1065032" data-doi="10.1016/s0169-5347(02)02633-2"><li><a href="http://dx.doi.org/10.1016/s0169-5347(02)02633-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fighting+change+with+change%3A+adaptive+variation+in+an+uncertain+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fighting+change+with+change%3A+adaptive+variation+in+an+uncertain+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001700-Gazzaniga1" id="pone.0001700-Gazzaniga1"></a>Gazzaniga MS (2004) The Cognitive Neurosciences III: MIT Press.   <ul class="find-nolinks"></ul></li><li><span class="label">62.
              </span><a name="pone.0001700-Willeboordse1" id="pone.0001700-Willeboordse1"></a>Willeboordse FH, Kaneko K (2005) Externally controlled attractor selection in a high-dimensional system. Phys Rev E Stat Nonlin Soft Matter Phys  72: 026207.  <ul class="find" data-citedArticleID="1065184" data-doi="10.1103/physreve.72.026207"><li><a href="http://dx.doi.org/10.1103/physreve.72.026207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Externally+controlled+attractor+selection+in+a+high-dimensional+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Externally+controlled+attractor+selection+in+a+high-dimensional+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001700-Segel1" id="pone.0001700-Segel1"></a>Segel LA (1998) Multiple attractors in immunology: theory and experiment. Biophys Chem  72: 223–230.  <ul class="find" data-citedArticleID="1065158" data-doi="10.1016/s0301-4622(98)00136-7"><li><a href="http://dx.doi.org/10.1016/s0301-4622(98)00136-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+attractors+in+immunology%3A+theory+and+experiment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+attractors+in+immunology%3A+theory+and+experiment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001700-Huang1" id="pone.0001700-Huang1"></a>Huang S, Eichler G, Bar-Yam Y, Ingber DE (2005) Cell fates as high-dimensional attractor states of a complex gene regulatory network. Phys Rev Lett  94: 128701.  <ul class="find" data-citedArticleID="1065102" data-doi="10.1103/physrevlett.94.128701"><li><a href="http://dx.doi.org/10.1103/physrevlett.94.128701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+fates+as+high-dimensional+attractor+states+of+a+complex+gene+regulatory+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+fates+as+high-dimensional+attractor+states+of+a+complex+gene+regulatory+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001700-Bengtsson1" id="pone.0001700-Bengtsson1"></a>Bengtsson G (1977) Output regulation and Internal models: a frequency domain approach. Automatica  13: 333–345.  <ul class="find" data-citedArticleID="1065050" data-doi="10.1016/0005-1098(77)90016-4"><li><a href="http://dx.doi.org/10.1016/0005-1098(77)90016-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Output+regulation+and+Internal+models%3A+a+frequency+domain+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Output+regulation+and+Internal+models%3A+a+frequency+domain+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001700-Lande1" id="pone.0001700-Lande1"></a>Lande R, Orzack SH (1988) Extinction dynamics of age-structured populations in a fluctuating environment. Proc Natl Acad Sci U S A  85: 7418–7421.  <ul class="find" data-citedArticleID="1065126" data-doi="10.1073/pnas.85.19.7418"><li><a href="http://dx.doi.org/10.1073/pnas.85.19.7418" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extinction+dynamics+of+age-structured+populations+in+a+fluctuating+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extinction+dynamics+of+age-structured+populations+in+a+fluctuating+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001700-Thattai1" id="pone.0001700-Thattai1"></a>Thattai M, van Oudenaarden A (2004) Stochastic gene expression in fluctuating environments. Genetics  167: 523–530.  <ul class="find" data-citedArticleID="1065174" data-doi="10.1534/genetics.167.1.523"><li><a href="http://dx.doi.org/10.1534/genetics.167.1.523" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+gene+expression+in+fluctuating+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+gene+expression+in+fluctuating+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001700-Sambrook1" id="pone.0001700-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular Cloning: A Laboratory Manual. NY: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">69.
              </span><a name="pone.0001700-Cormack1" id="pone.0001700-Cormack1"></a>Cormack BP, Valdivia RH, Falkow S (1996) FACS-optimized mutants of the green fluorescent protein (GFP). Gene  173: 33–38.  <ul class="find" data-citedArticleID="1065066" data-doi="10.1016/0378-1119(95)00685-0"><li><a href="http://dx.doi.org/10.1016/0378-1119(95)00685-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FACS-optimized+mutants+of+the+green+fluorescent+protein+%28GFP%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FACS-optimized+mutants+of+the+green+fluorescent+protein+%28GFP%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001700-Fujita1" id="pone.0001700-Fujita1"></a>Fujita M, Losick R (2002) An investigation into the compartmentalization of the sporulation transcription factor sigmaE in Bacillus subtilis. Mol Microbiol  43: 27–38.  <ul class="find" data-citedArticleID="1065078" data-doi="10.1046/j.1365-2958.2002.02732.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02732.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+investigation+into+the+compartmentalization+of+the+sporulation+transcription+factor+sigmaE+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+investigation+into+the+compartmentalization+of+the+sporulation+transcription+factor+sigmaE+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001700-Quisel1" id="pone.0001700-Quisel1"></a>Quisel JD, Lin DC, Grossman AD (1999) Control of development by altered localization of a transcription factor in B. subtilis. Mol Cell  4: 665–672.  <ul class="find" data-citedArticleID="1065144" data-doi="10.1016/s1097-2765(00)80377-9"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80377-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+development+by+altered+localization+of+a+transcription+factor+in+B.+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+development+by+altered+localization+of+a+transcription+factor+in+B.+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001700-Jan1" id="pone.0001700-Jan1"></a>Jan J, Valle F, Bolivar F, Merino E (2001) Construction of protein overproducer strains in Bacillus subtilis by an integrative approach. Appl Microbiol Biotechnol  55: 69–75.  <ul class="find" data-citedArticleID="1065106" data-doi="10.1007/s002530000448"><li><a href="http://dx.doi.org/10.1007/s002530000448" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+protein+overproducer+strains+in+Bacillus+subtilis+by+an+integrative+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+protein+overproducer+strains+in+Bacillus+subtilis+by+an+integrative+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001700-Rousseeuw1" id="pone.0001700-Rousseeuw1"></a>Rousseeuw PJ (1987) Silhouettes: A graphical aid to the interpretation and validation of cluster analysis. Journal of Computational and Applied Mathematics  20: 53–65.  <ul class="find" data-citedArticleID="1065148" data-doi="10.1016/0377-0427(87)90125-7"><li><a href="http://dx.doi.org/10.1016/0377-0427(87)90125-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Silhouettes%3A+A+graphical+aid+to+the+interpretation+and+validation+of+cluster+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Silhouettes%3A+A+graphical+aid+to+the+interpretation+and+validation+of+cluster+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001700-Vastano1" id="pone.0001700-Vastano1"></a>Vastano JA, Swinney HL (1988) Information transport in spatiotemporal systems. Physical Review Letters  60: 1773–1776.  <ul class="find" data-citedArticleID="1065178" data-doi="10.1103/physrevlett.60.1773"><li><a href="http://dx.doi.org/10.1103/physrevlett.60.1773" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Information+transport+in+spatiotemporal+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Information+transport+in+spatiotemporal+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001700-Samoilov1" id="pone.0001700-Samoilov1"></a>Samoilov M, Arkin A, Ross J (2001) On the deduction of chemical reaction pathways from measurements of time series of concentrations. Chaos  11: 108–114.  <ul class="find" data-citedArticleID="1065154" data-doi="10.1063/1.1336499"><li><a href="http://dx.doi.org/10.1063/1.1336499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+deduction+of+chemical+reaction+pathways+from+measurements+of+time+series+of+concentrations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+deduction+of+chemical+reaction+pathways+from+measurements+of+time+series+of+concentrations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001700-Chen1" id="pone.0001700-Chen1"></a>Chen I, Dubnau D (2003) DNA transport during transformation. Front Biosci  8: s544–556.  <ul class="find" data-citedArticleID="1065060" data-doi="10.2741/1047"><li><a href="http://dx.doi.org/10.2741/1047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+transport+during+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+transport+during+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001700-Kunst1" id="pone.0001700-Kunst1"></a>Kunst F, Msadek T, Bignon J, Rapoport G (1994) The DegS/DegU and ComP/ComA two-component systems are part of a network controlling degradative enzyme synthesis and competence in Bacillus subtilis. Res Microbiol  145: 393–402.  <ul class="find" data-citedArticleID="1065120" data-doi="10.1016/0923-2508(94)90087-6"><li><a href="http://dx.doi.org/10.1016/0923-2508(94)90087-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DegS%2FDegU+and+ComP%2FComA+two-component+systems+are+part+of+a+network+controlling+degradative+enzyme+synthesis+and+competence+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DegS%2FDegU+and+ComP%2FComA+two-component+systems+are+part+of+a+network+controlling+degradative+enzyme+synthesis+and+competence+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001700-Garrity1" id="pone.0001700-Garrity1"></a>Garrity LF, Ordal GW (1995) Chemotaxis in Bacillus subtilis: how bacteria monitor environmental signals. Pharmacol Ther  68: 87–104.  <ul class="find" data-citedArticleID="1065082" data-doi="10.1016/0163-7258(95)00027-5"><li><a href="http://dx.doi.org/10.1016/0163-7258(95)00027-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemotaxis+in+Bacillus+subtilis%3A+how+bacteria+monitor+environmental+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemotaxis+in+Bacillus+subtilis%3A+how+bacteria+monitor+environmental+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001700-Marahiel1" id="pone.0001700-Marahiel1"></a>Marahiel MA, Nakano MM, Zuber P (1993) Regulation of peptide antibiotic production in Bacillus. Mol Microbiol  7: 631–636.  <ul class="find" data-citedArticleID="1065130" data-doi="10.1111/j.1365-2958.1993.tb01154.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1993.tb01154.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+peptide+antibiotic+production+in+Bacillus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+peptide+antibiotic+production+in+Bacillus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001700-Entian1" id="pone.0001700-Entian1"></a>Entian KD, de Vos WM (1996) Genetics of subtilin and nisin biosyntheses: biosynthesis of lantibiotics. Antonie Van Leeuwenhoek  69: 109–117.  <ul class="find" data-citedArticleID="1065072" data-doi="10.1007/bf00399416"><li><a href="http://dx.doi.org/10.1007/bf00399416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+subtilin+and+nisin+biosyntheses%3A+biosynthesis+of+lantibiotics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+subtilin+and+nisin+biosyntheses%3A+biosynthesis+of+lantibiotics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001700-Kearns1" id="pone.0001700-Kearns1"></a>Kearns DB, Chu F, Branda SS, Kolter R, Losick R (2005) A master regulator for biofilm formation by Bacillus subtilis. Mol Microbiol  55: 739–749.  <ul class="find" data-citedArticleID="1065110" data-doi="10.1111/j.1365-2958.2004.04440.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04440.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+master+regulator+for+biofilm+formation+by+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+master+regulator+for+biofilm+formation+by+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001700-Sanchez1" id="pone.0001700-Sanchez1"></a>Sanchez A, Olmos J (2004) Bacillus subtilis transcriptional regulators interaction. Biotechnol Lett  26: 403–407.  <ul class="find" data-citedArticleID="1065156" data-doi="10.1023/b:bile.0000018259.66762.ed"><li><a href="http://dx.doi.org/10.1023/b:bile.0000018259.66762.ed" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+subtilis+transcriptional+regulators+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+subtilis+transcriptional+regulators+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.XML" value="158670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.PDF" value="565270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g001.PNG_L" value="476410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g001.PNG_M" value="125767"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g001.PNG_S" value="12976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g001.TIF" value="1093388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g001.PNG_I" value="41228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g002.PNG_L" value="45403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g002.PNG_M" value="122607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g002.PNG_S" value="14048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g002.TIF" value="275222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g002.PNG_I" value="43945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g003.PNG_L" value="323891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g003.PNG_M" value="137895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g003.PNG_S" value="15924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g003.TIF" value="688320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g003.PNG_I" value="49260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g004.PNG_L" value="30515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g004.PNG_M" value="104069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g004.PNG_S" value="13258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g004.TIF" value="116950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g004.PNG_I" value="37379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.t001.PNG_L" value="115503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.t001.PNG_M" value="156927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.t001.PNG_S" value="19438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.t001.TIF" value="496054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.t001.PNG_I" value="52479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g005.PNG_L" value="939600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g005.PNG_M" value="89088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g005.PNG_S" value="9212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g005.TIF" value="1376028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g005.PNG_I" value="29022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g006.PNG_L" value="587790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g006.PNG_M" value="116432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g006.PNG_S" value="12905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g006.TIF" value="1599176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g006.PNG_I" value="40725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g007.PNG_L" value="160698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g007.PNG_M" value="118951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g007.PNG_S" value="15389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g007.TIF" value="517260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.g007.PNG_I" value="40817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s001.PDF" value="186908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s002.PDF" value="268611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s003.PDF" value="267299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s004.PDF" value="155644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s005.PDF" value="715175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s006.PDF" value="123161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s007.PDF" value="841756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s008.PDF" value="126403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001700.s009.PDF" value="253071"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001700&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001700" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001700&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001700&volume=&issue=&title=Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium&author_name=Denise%20M.%20Wolf%2C%20Lisa%20Fontaine-Bodin%2C%20Ilka%20Bischofs%2C%20Gavin%20Price%2C%20Jay%20Keasling%2C%20Adam%20P.%20Arkin&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001700" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001700" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001700" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001700&amp;t=Memory%20in%20Microbes%3A%20Quantifying%20History-Dependent%20Behavior%20in%20a%20Bacterium" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001700&title=Memory%20in%20Microbes%3A%20Quantifying%20History-Dependent%20Behavior%20in%20a%20Bacterium&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001700&amp;title=Memory%20in%20Microbes%3A%20Quantifying%20History-Dependent%20Behavior%20in%20a%20Bacterium" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001700" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001700&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Memory in Microbes: Quantifying History-Dependent Behavior in a Bacterium';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001700';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Memory%20in%20Microbes%3A%20Quantifying%20History-Dependent%20Behavior%20in%20a%20Bacterium http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001700" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001700" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacillus+subtilis%22" title="Search for articles in the subject area:'Bacillus subtilis'"><div class="flagText">Bacillus subtilis</div></a>
              <div data-categoryid="32939" data-articleid="26226"
                   data-categoryname="Bacillus subtilis"
                   class="flagImage" title="Flag 'Bacillus subtilis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cellular+stress+responses%22" title="Search for articles in the subject area:'Cellular stress responses'"><div class="flagText">Cellular stress responses</div></a>
              <div data-categoryid="20429" data-articleid="26226"
                   data-categoryname="Cellular stress responses"
                   class="flagImage" title="Flag 'Cellular stress responses' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="26226"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Long-term+memory%22" title="Search for articles in the subject area:'Long-term memory'"><div class="flagText">Long-term memory</div></a>
              <div data-categoryid="20517" data-articleid="26226"
                   data-categoryname="Long-term memory"
                   class="flagImage" title="Flag 'Long-term memory' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Memory%22" title="Search for articles in the subject area:'Memory'"><div class="flagText">Memory</div></a>
              <div data-categoryid="31849" data-articleid="26226"
                   data-categoryname="Memory"
                   class="flagImage" title="Flag 'Memory' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+density%22" title="Search for articles in the subject area:'Population density'"><div class="flagText">Population density</div></a>
              <div data-categoryid="43529" data-articleid="26226"
                   data-categoryname="Population density"
                   class="flagImage" title="Flag 'Population density' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sporulation%22" title="Search for articles in the subject area:'Sporulation'"><div class="flagText">Sporulation</div></a>
              <div data-categoryid="48041" data-articleid="26226"
                   data-categoryname="Sporulation"
                   class="flagImage" title="Flag 'Sporulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Starvation%22" title="Search for articles in the subject area:'Starvation'"><div class="flagText">Starvation</div></a>
              <div data-categoryid="21519" data-articleid="26226"
                   data-categoryname="Starvation"
                   class="flagImage" title="Flag 'Starvation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7743'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2529'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8313&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=18503">Evaluation at F1000 Biology</a><br>Posted by NiyazAhmed</p>
      <p><a href="/annotation/listThread.action?root=3193">Memory in Bacteria</a><br>Posted by Cingham</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
