

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genesis of a Fungal Non-Self Recognition Repertoire</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000283"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000283" />

    <meta name="citation_title" content="Genesis of a Fungal Non-Self Recognition Repertoire"/>
    <meta itemprop="name" content="Genesis of a Fungal Non-Self Recognition Repertoire"/>

      <meta name="citation_author" content="Mathieu Paoletti"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Moléculaire des Champignons, UMR-5095 Centre National de la Recherche Scientifique (CNRS) et Université Bordeaux 2, Institut de Biochimie et Génétique Cellulaires (IBGC), Bordeaux, France"/>
      <meta name="citation_author" content="Sven J. Saupe"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Moléculaire des Champignons, UMR-5095 Centre National de la Recherche Scientifique (CNRS) et Université Bordeaux 2, Institut de Biochimie et Génétique Cellulaires (IBGC), Bordeaux, France"/>
      <meta name="citation_author" content="Corinne Clavé"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Moléculaire des Champignons, UMR-5095 Centre National de la Recherche Scientifique (CNRS) et Université Bordeaux 2, Institut de Biochimie et Génétique Cellulaires (IBGC), Bordeaux, France"/>

    <meta name="citation_date" content="2007/3/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000283.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e283"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Decoding the patterns of self and nonself by the innate immune system.; citation_author=R Medzhitov; citation_author=CA, Janeway; citation_journal_title=Science; citation_volume=296; citation_number=1; citation_pages=298-300; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The many selves of social insects.; citation_author=DC Queller; citation_author=JE Strassmann; citation_journal_title=Science; citation_volume=296; citation_number=2; citation_pages=311-313; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Recognition and rejection of self in plant self-incompatibility: comparisons to animal histocompatibility.; citation_author= Nasrallah; citation_journal_title=Trends in Immunology; citation_volume=26; citation_number=3; citation_pages=412-418; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Somatic cell parasitism and the evolution of somatic tissue compatibility.; citation_author=LW Buss; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=79; citation_number=4; citation_pages=5337-5341; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of a protochordate histocompatibility locus.; citation_author=AW De Tomaso; citation_author=SV Nyholm; citation_author=KJ Palmeri; citation_author=KJ Ishizuka; citation_author=WB Ludington; citation_journal_title=Nature; citation_volume=438; citation_number=5; citation_pages=454-459; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of balanced genetic polymorphism.; citation_author=A Richman; citation_journal_title=Mol Ecol; citation_volume=9; citation_number=6; citation_pages=1953-1963; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The genetics of hyphal fusion and vegetative incompatibility in filamentous ascomycete fungi.; citation_author=NL Glass; citation_author=DJ Jacobson; citation_author=PK Shiu; citation_journal_title=Annu Rev Genet; citation_volume=34; citation_number=7; citation_pages=165-186; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fatal attraction: nonself recognition and heterokaryon incompatibility in filamentous fungi.; citation_author=N Glass; citation_author=I Kaneko; citation_journal_title=Eukaryotic Cell; citation_volume=2; citation_number=8; citation_pages=1-8; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular genetics of heterokaryon incompatibility in filamentous ascomycetes.; citation_author=SJ Saupe; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=64; citation_number=9; citation_pages=489-502; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Hypovirulence: mycoviruses at the fungal-plant interface.; citation_author=DL Nuss; citation_journal_title=Nat Rev Microbiol; citation_volume=3; citation_number=10; citation_pages=632-642; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Programmed cell death correlates with virus transmission in a filamentous fungus.; citation_author=S Biella; citation_author=ML Smith; citation_author=JR Aist; citation_author=P Cortesi; citation_author=MG Milgroom; citation_journal_title=Proc Biol Sci; citation_volume=269; citation_number=11; citation_pages=2269-2276; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=HET-E and HET-D belong to a new subfamily of WD40 proteins involved in vegetative incompatibility specificity in the fungus Podospora anserina.; citation_author=E Espagne; citation_author=P Balhadere; citation_author=ML Penin; citation_author=C Barreau; citation_author=B Turcq; citation_journal_title=Genetics; citation_volume=161; citation_number=12; citation_pages=71-81; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Contribution à l'étude des systèmes non alléliques d'incompatibilité chez le champignon filamenteux Podospora anserina.; citation_author=E Espagne; citation_journal_title=Thèse, Université de Bordeaux; citation_volume=II; citation_number=13; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reactivity in vegetative incompatibility of the HET-E protein of the fungus Podospora anserina is dependent on GTP-binding activity and a WD40 repeated domain.; citation_author=E Espagne; citation_author=P Balhadere; citation_author=J Begueret; citation_author=B Turcq; citation_journal_title=Mol Gen Genet; citation_volume=256; citation_number=14; citation_pages=620-627; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A gene responsible for vegetative incompatibility in the fungus Podospora anserina encodes a protein with a GTP-binding motif and G beta homologous domain.; citation_author=S Saupe; citation_author=B Turcq; citation_author=J Begueret; citation_journal_title=Gene; citation_volume=162; citation_number=15; citation_pages=135-139; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Sequence diversity and unusual variability at the het-c locus involved in vegetative incompatibility in the fungus Podospora anserina.; citation_author=S Saupe; citation_author=B Turcq; citation_author=J Begueret; citation_journal_title=Curr Genet; citation_volume=27; citation_number=16; citation_pages=466-471; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=[Systems of incompatibility in Podospora anserina].; citation_author=J Bernet; citation_journal_title=C R Acad Sci Hebd Seances Acad Sci D; citation_volume=265; citation_number=17; citation_pages=1330-1333; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=Polypeptide synthesis during protoplasmic incompatibility in the fungus Podospora anserina.; citation_author=H Boucherie; citation_author=CH Dupont; citation_author=J Bernet; citation_journal_title=Biochim Biophys Acta; citation_volume=653; citation_number=18; citation_pages=18-26; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Autophagy is induced during cell death by incompatibility and is essential for differentiation in the filamentous fungus Podospora anserina.; citation_author=B Pinan-Lucarre; citation_author=M Paoletti; citation_author=K Dementhon; citation_author=B Coulary-Salin; citation_author=C Clave; citation_journal_title=Mol Microbiol; citation_volume=47; citation_number=19; citation_pages=321-333; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Accelerated cell death in Podospora autophagy mutants.; citation_author=B Pinan-Lucarre; citation_author=A Balguerie; citation_author=C Clave; citation_journal_title=Eukaryot Cell; citation_volume=4; citation_number=20; citation_pages=1765-1774; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inactivation of the Podospora anserina vegetative incompatibility locus het-c, whose product resembles a glycolipid transfer protein, drastically impairs ascospore production.; citation_author=S Saupe; citation_author=C Descamps; citation_author=B Turcq; citation_author=J Begueret; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=21; citation_pages=5927-5931; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Vegetative incompatibility in the het-6 region of Neurospora crassa is mediated by two linked genes.; citation_author=ML Smith; citation_author=OC Micali; citation_author=SP Hubbard; citation_author=N Mir-Rashed; citation_author=DJ Jacobson; citation_journal_title=Genetics; citation_volume=155; citation_number=22; citation_pages=1095-1104; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Non-self recognition and programmed cell death in filamentous fungi.; citation_author=NL Glass; citation_author=K Dementhon; citation_journal_title=Curr Opin Microbiol; citation_number=23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=STAND, a class of P-loop NTPases including animal and plant regulators of programmed cell death: multiple, complex domain architectures, unusual phyletic patterns, and evolution by horizontal gene transfer.; citation_author=DD Leipe; citation_author=EV Koonin; citation_author=L Aravind; citation_journal_title=J Mol Biol; citation_volume=343; citation_number=24; citation_pages=1-28; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=NACHT-LRR proteins (NLRs) in bacterial infection and immunity.; citation_author=TA Kufer; citation_author=JH Fritz; citation_author=DJ Philpott; citation_journal_title=Trends Microbiol; citation_volume=13; citation_number=25; citation_pages=381-388; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RIP: the evolutionary cost of genome defense.; citation_author=JE Galagan; citation_author=EU Selker; citation_journal_title=Trends Genet; citation_volume=20; citation_number=26; citation_pages=417-423; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of the filamentous fungus Neurospora crassa.; citation_author=JE Galagan; citation_author=SE Calvo; citation_author=KA Borkovich; citation_author=EU Selker; citation_author=ND Read; citation_journal_title=Nature; citation_volume=422; citation_number=27; citation_pages=859-868; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of programmed cell death pathways in filamentous fungi.; citation_author=ND Fedorova; citation_author=JH Badger; citation_author=GD Robson; citation_author=JR Wortman; citation_author=WC Nierman; citation_journal_title=BMC Genomics; citation_volume=6; citation_number=28; citation_pages=177; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Altering a gene involved in nuclear distribution increases the repeat-induced point mutation process in the fungus Podospora anserina.; citation_author=K Bouhouche; citation_author=D Zickler; citation_author=R Debuchy; citation_author=S Arnaise; citation_journal_title=Genetics; citation_volume=167; citation_number=29; citation_pages=151-159; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Repeat-induced G-C to A-T mutations in Neurospora.; citation_author=EB Cambareri; citation_author=BC Jensen; citation_author=E Schabtach; citation_author=EU Selker; citation_journal_title=Science; citation_volume=244; citation_number=30; citation_pages=1571-1575; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The WD repeat: a common architecture for diverse functions.; citation_author=TF Smith; citation_author=C Gaitatzes; citation_author=K Saxena; citation_author=EJ Neer; citation_journal_title=Trends Biochem Sci; citation_volume=24; citation_number=31; citation_pages=181-185; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Concerted and Birth-and-Death Evolution of Multigene Families.; citation_author=M Nei; citation_author=AP Rooney; citation_journal_title=Annu Rev Genet; citation_number=32; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis in molecular evolutionary genetics.; citation_author=M Nei; citation_journal_title=Annu Rev Genet; citation_volume=30; citation_number=33; citation_pages=371-403; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular signatures of natural selection.; citation_author=R Nielsen; citation_journal_title=Annu Rev Genet; citation_volume=39; citation_number=34; citation_pages=197-218; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selectionism and neutralism in molecular evolution.; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=35; citation_pages=2318-2342; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Not so different after all: a comparison of methods for detecting amino acid sites under selection.; citation_author=SL Kosakovsky Pond; citation_author=SD Frost; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=36; citation_pages=1208-1222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structure of the C-terminal domain of Tup1, a corepressor of transcription in yeast.; citation_author=ER Sprague; citation_author=MJ Redd; citation_author=AD Johnson; citation_author=C Wolberger; citation_journal_title=Embo J; citation_volume=19; citation_number=37; citation_pages=3016-3027; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Folding of proteins with WD-repeats: comparison of six members of the WD-repeat superfamily to the G protein beta subunit.; citation_author=I Garcia-Higuera; citation_author=J Fenoglio; citation_author=Y Li; citation_author=C Lewis; citation_author=MP Panchenko; citation_journal_title=Biochemistry; citation_volume=35; citation_number=38; citation_pages=13985-13994; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The WD-repeats of Net2p interact with Dnm1p and Fis1p to regulate division of mitochondria.; citation_author=KL Cerveny; citation_author=RE Jensen; citation_journal_title=Mol Biol Cell; citation_volume=14; citation_number=39; citation_pages=4126-4139; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural basis for glycosphingolipid transfer specificity.; citation_author=L Malinina; citation_author=ML Malakhova; citation_author=A Teplov; citation_author=RE Brown; citation_author=DJ Patel; citation_journal_title=Nature; citation_volume=430; citation_number=40; citation_pages=1048-1053; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Point mutational analysis of the liganding site in human glycolipid transfer protein. Functionality of the complex.; citation_author=ML Malakhova; citation_author=L Malinina; citation_author=HM Pike; citation_author=AT Kanack; citation_author=DJ Patel; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=41; citation_pages=26312-26320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rates of spontaneous mutation.; citation_author=JW Drake; citation_author=B Charlesworth; citation_author=D Charlesworth; citation_author=JF Crow; citation_journal_title=Genetics; citation_volume=148; citation_number=42; citation_pages=1667-1686; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Diminishing returns from mutation supply rate in asexual populations.; citation_author=JA Arjan; citation_author=M Visser; citation_author=CW Zeyl; citation_author=PJ Gerrish; citation_author=JL Blanchard; citation_journal_title=Science; citation_volume=283; citation_number=43; citation_pages=404-406; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The degenerate DNA transposon Pat and repeat-induced point mutation (RIP) in Podospora anserina.; citation_author=A Hamann; citation_author=F Feller; citation_author=HD Osiewacz; citation_journal_title=Mol Gen Genet; citation_volume=263; citation_number=44; citation_pages=1061-1069; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Lost in the middle of nowhere: the AvrLm1 avirulence gene of the Dothideomycete Leptosphaeria maculans.; citation_author=L Gout; citation_author=I Fudal; citation_author=ML Kuhn; citation_author=F Blaise; citation_author=M Eckert; citation_journal_title=Mol Microbiol; citation_volume=60; citation_number=45; citation_pages=67-80; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence of RIP (repeat-induced point mutation) in transposase sequences of Aspergillus oryzae.; citation_author=MD Montiel; citation_author=HA Lee; citation_author=DB Archer; citation_journal_title=Fungal Genet Biol; citation_volume=43; citation_number=46; citation_pages=439-445; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Repeat-modulated population genetic effects in fungal proteins.; citation_author=FN Braun; citation_author=DA Liberles; citation_journal_title=J Mol Evol; citation_volume=59; citation_number=47; citation_pages=97-102; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Selective acquisition of novel mating type and vegetative incompatibility genes via interspecies gene transfer in the globally invading eukaryote Ophiostoma novo-ulmi.; citation_author=M Paoletti; citation_author=KW Buck; citation_author=CM Brasier; citation_journal_title=Mol Ecol; citation_volume=15; citation_number=48; citation_pages=249-262; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence for balancing selection operating at the het-c heterokaryon incompatibility locus in a group of filamentous fungi.; citation_author=J Wu; citation_author=SJ Saupe; citation_author=NL Glass; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=49; citation_pages=12398-12403; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A primate-specific acceleration in the evolution of the caspase-dependent apoptosis pathway.; citation_author=EJ Vallender; citation_author=BT Lahn; citation_journal_title=Hum Mol Genet; citation_volume=15; citation_number=50; citation_pages=3034-3040; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of tol, a mediator of mating-type-associated vegetative incompatibility in Neurospora crassa.; citation_author=PK Shiu; citation_author=NL Glass; citation_journal_title=Genetics; citation_volume=151; citation_number=51; citation_pages=545-555; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Nonallelic interactions between het-c and a polymorphic locus, pin-c, are essential for nonself recognition and programmed cell death in Neurospora crassa.; citation_author=I Kaneko; citation_author=K Dementhon; citation_author=Q Xiang; citation_author=NL Glass; citation_journal_title=Genetics; citation_volume=172; citation_number=52; citation_pages=1545-1555; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=[Properties of an incompatibility system in Podospora anserina fungus and value of this system for the study of incompatibility].; citation_author=J Labarere; citation_journal_title=C R Acad Sci Hebd Seances Acad Sci D; citation_volume=276; citation_number=53; citation_pages=1301-1304; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=L'incompatibilite cellulaire chez le champignon Podospora anserina: etude physiologique de la reaction.; citation_author=J Begueret; citation_journal_title=These de l'Universite de Bordeaux; citation_volume=II; citation_number=54; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Immunoglobulin gene diversification.; citation_author=N Maizels; citation_journal_title=Annu Rev Genet; citation_volume=39; citation_number=55; citation_pages=23-46; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Antigenic variation in trypanosomes: enhanced phenotypic variation in a eukaryotic parasite.; citation_author=JD Barry; citation_author=R McCulloch; citation_journal_title=Adv Parasitol; citation_volume=49; citation_number=56; citation_pages=1-70; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gene conversion and the evolution of three leucine-rich repeat gene families in Arabidopsis thaliana.; citation_author=M Mondragon-Palomino; citation_author=BS Gaut; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=57; citation_pages=2444-2456; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for the adaptive evolution of mutation rates.; citation_author=D Metzgar; citation_author=C Wills; citation_journal_title=Cell; citation_volume=101; citation_number=58; citation_pages=581-584; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=HyPhy: hypothesis testing using phylogenies.; citation_author=SL Pond; citation_author=SD Frost; citation_author=SV Muse; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=59; citation_pages=676-679; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A method for detecting positive selection at single amino acid sites.; citation_author=Y Suzuki; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=60; citation_pages=1315-1328; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.; citation_author=M Nei; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=3; citation_number=61; citation_pages=418-426; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=62; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=63; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=A simple method for estimating evolutionary rate of base substitutions through comparative studies of nucleotide sequences.; citation_author=M Kimura; citation_journal_title=Journal of Molecular Evolution; citation_volume=16; citation_number=64; citation_pages=111-120; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Geno3D: automatic comparative molecular modelling of protein.; citation_author=C Combet; citation_author=M Jambon; citation_author=G Deleage; citation_author=C Geourjon; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=65; citation_pages=213-214; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000283" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genesis of a Fungal Non-Self Recognition Repertoire"/>
    <meta name="twitter:description" content="Conspecific allorecognition, the ability for an organism to discriminate its own cells from those of another individual of the same species, has been developed by many organisms. Allorecognition specificities are determined by highly polymorphic genes. The processes by which this extreme polymorphism is generated remain largely unknown. Fungi are able to form heterokaryons by fusion of somatic cells, and somatic non self-recognition is controlled by heterokaryon incompatibility loci (het loci). Herein, we have analyzed the evolutionary features of the het-d and het-e fungal allorecognition genes. In these het genes, allorecognition specificity is determined by a polymorphic WD-repeat domain. We found that het-d and het-e belong to a large gene family with 10 members that all share the WD-repeat domain and show that repeats of all members of the family undergo concerted evolution. It follows that repeat units are constantly exchanged both within and between members of the gene family. As a consequence, high mutation supply in the repeat domain is ensured due to the high total copy number of repeats. We then show that in each repeat four residues located at the protein/protein interaction surface of the WD-repeat domain are under positive diversifying selection. Diversification of het-d and het-e is thus ensured by high mutation supply, followed by reshuffling of the repeats and positive selection for favourable variants. We also propose that RIP, a fungal specific hypermutation process acting specifically on repeated sequences might further enhance mutation supply. The combination of these evolutionary mechanisms constitutes an original process for generating extensive polymorphism at loci that require rapid diversification."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000283.g004"/>

  <meta property="og:title" content="Genesis of a Fungal Non-Self Recognition Repertoire" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000283" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4196'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3302'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2952&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000283">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000283" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000283&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genesis of a Fungal Non-Self Recognition Repertoire
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mathieu Paoletti
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mathieu.paoletti@ibgc.u-bordeaux2.fr">mathieu.paoletti@ibgc.u-bordeaux2.fr</a></p>

                <p>Affiliation:
                  Laboratoire de Génétique Moléculaire des Champignons, UMR-5095 Centre National de la Recherche Scientifique (CNRS) et Université Bordeaux 2, Institut de Biochimie et Génétique Cellulaires (IBGC), Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sven J. Saupe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Moléculaire des Champignons, UMR-5095 Centre National de la Recherche Scientifique (CNRS) et Université Bordeaux 2, Institut de Biochimie et Génétique Cellulaires (IBGC), Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Corinne Clavé
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Génétique Moléculaire des Champignons, UMR-5095 Centre National de la Recherche Scientifique (CNRS) et Université Bordeaux 2, Institut de Biochimie et Génétique Cellulaires (IBGC), Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000283</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000283" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000283" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000283" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000283" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000283-g001" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000283-t001" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000283-g002" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000283-t002" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000283-g003" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000283-g004" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000283">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000283" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Conspecific allorecognition, the ability for an organism to discriminate its own cells from those of another individual of the same species, has been developed by many organisms. Allorecognition specificities are determined by highly polymorphic genes. The processes by which this extreme polymorphism is generated remain largely unknown. Fungi are able to form heterokaryons by fusion of somatic cells, and somatic non self-recognition is controlled by heterokaryon incompatibility loci (<em>het</em> loci). Herein, we have analyzed the evolutionary features of the <em>het-d</em> and <em>het-e</em> fungal allorecognition genes. In these <em>het</em> genes, allorecognition specificity is determined by a polymorphic WD-repeat domain. We found that <em>het-d</em> and <em>het-e</em> belong to a large gene family with 10 members that all share the WD-repeat domain and show that repeats of all members of the family undergo concerted evolution. It follows that repeat units are constantly exchanged both within and between members of the gene family. As a consequence, high mutation supply in the repeat domain is ensured due to the high total copy number of repeats. We then show that in each repeat four residues located at the protein/protein interaction surface of the WD-repeat domain are under positive diversifying selection. Diversification of <em>het-d</em> and <em>het-e</em> is thus ensured by high mutation supply, followed by reshuffling of the repeats and positive selection for favourable variants. We also propose that RIP, a fungal specific hypermutation process acting specifically on repeated sequences might further enhance mutation supply. The combination of these evolutionary mechanisms constitutes an original process for generating extensive polymorphism at loci that require rapid diversification.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Paoletti M, Saupe SJ, Clavé C (2007) Genesis of a Fungal Non-Self Recognition Repertoire. PLoS ONE 2(3):
          e283.
            doi:10.1371/journal.pone.0000283</p><p><strong>Academic Editor: </strong>James Fraser, University of Queensland, Australia</p><p><strong>Received:</strong> January 8, 2007; <strong>Accepted:</strong> February 10, 2007; <strong>Published:</strong> March 14, 2007</p><p><strong>Copyright:</strong> © 2007 Paoletti et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>M.P. is funded by a European commission grant (Transdeath Contract 511983).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Most living organisms have developed genetic systems to discriminate self from non-self. This ability is crucial for instance for the immune response in vertebrates <a href="#pone.0000283-Medzhitov1">[1]</a>, social organisation in insects <a href="#pone.0000283-Queller1">[2]</a> or efficient out-breeding in flowering plants <a href="#pone.0000283-Nasrallah1">[3]</a>. It has been long recognized that self/non-self discrimination is particularly critical for organisms that spontaneously form somatic chimeras, such as protists (dictyostelids and myxomycota), sponges, tunicates, ascidians and filamentous fungi <a href="#pone.0000283-Buss1">[4]</a>. In these organisms the ability to recognize and reject conspecific non-self is essential for maintenance of biological integrity. Because of their life style, these organisms are highly vulnerable to somatic cell parasitism that can even lead to germ line invasion, “the evolutionary equivalent of death” <a href="#pone.0000283-Buss1">[4]</a>. Such primitive allorecognition systems might be the evolutionary ancestor of acquired immunity in mammals as suggested by the fact that the recently isolated gene controlling somatic histocompatibility in the prochordate <em>Botryllus</em> belongs to the immunoglobulin superfamily <a href="#pone.0000283-DeTomaso1">[5]</a>. When known, the genes involved in self/non-self recognition share a particular evolutionary signature as they appear to be under positive Darwinian selection that favours accumulation and maintenance of allelic diversity (diversifying selection) <a href="#pone.0000283-Richman1">[6]</a>. The study of the molecular evolution of this type of loci raises keen interest as their evolutionary behaviour is in sharp contrast with the vast majority of the other genes, and the mechanisms leading to high allelic polymorphism remain unclear.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In filamentous fungi, somatic cell fusion occurs spontaneously between filaments of different isolates leading to formation of a heterokaryotic structure <a href="#pone.0000283-Glass1">[7]</a> <a href="#pone.0000283-Glass2">[8]</a> <a href="#pone.0000283-Saupe1">[9]</a>. In most cases however, these heterokaryotic cells are compartmentalised and destroyed by a cell death reaction. The genes controlling this somatic non-self recognition are known as heterokaryon incompatibility genes (<em>het</em> genes). Each fungal species possesses about a dozen of such incompatibility loci and a genetic difference at any one of them is sufficient to trigger destruction of the mixed cell and abortion of the fusion event. As a consequence, in most fungal species, pairs of isolates taken at random are generally incompatible. In fungi, the destruction of heterokaryotic fusion cells might also serve to limit the horizontal spreading of mycoviruses. Mycoviruses are not transmitted via the extracellular route but exclusively through cell fusion events so that incompatibility is able to limit horizontal transfer of these infectious elements <a href="#pone.0000283-Nuss1">[10]</a> <a href="#pone.0000283-Biella1">[11]</a> .</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A number of <em>het</em> genes have been isolated and characterized in two model species <em>Neurospora crassa</em> and <em>Podospora anserina</em> <a href="#pone.0000283-Glass2">[8]</a>. The <em>het-c, het-d</em> and <em>het-e</em> incompatibility loci of <em>P. anserina</em> are multi-allelic and define two non-allelic incompatibility systems <a href="#pone.0000283-Espagne1">[12]</a>–<a href="#pone.0000283-Boucherie1">[18]</a> Each <em>het-c</em> allele is incompatible with a subset of <em>het-d</em> and <em>het-e</em> alleles. Interactions between incompatible <em>het-c/het-d</em> or <em>het-c/het-e</em> allele pairs lead to a cell death reaction characterized by vacuole enlargement and bursting and a simultaneous, massive induction of autophagy <a href="#pone.0000283-PinanLucarre1">[19]</a>. Autophagy is believed to prevent spreading of the cell death reaction to cells adjacent to the fusion cell <a href="#pone.0000283-PinanLucarre2">[20]</a>. <em>het-c</em> encodes a glycolipid transfer protein (GLTP) and displays a cellular function in addition to its role in allorecognition since it is required for proper ascospore formation <a href="#pone.0000283-Saupe4">[21]</a>. Comparisons of different <em>het-c</em> alleles have suggested that this gene undergoes rapid evolution <a href="#pone.0000283-Saupe3">[16]</a>. <em>het-d</em> and <em>het-e</em> encode large paralogous proteins that comprise three distinct functional domains. Both proteins display a N-terminal HET domain, a central NACHT domain and a C-terminal WD-repeat domain. The HET domain is a fungal-specific protein domain found in a number of proteins involved in incompatibility <a href="#pone.0000283-Smith1">[22]</a>. All fungal incompatibility gene systems-with the exception of the <em>P. anserina het-s/het-S</em> prion system-possess at least one constituent bearing a HET domain, suggesting that this domain might be involved in mediating cell death <a href="#pone.0000283-Glass3">[23]</a>. The NACHT domain is a regulatory NTP-binding domain involved in oligomerization and found in a variety of proteins controlling programmed cell death and pathogen recognition phenomena <a href="#pone.0000283-Leipe1">[24]</a>, <a href="#pone.0000283-Kufer1">[25]</a>. Finally, the WD-repeat domain, made of a variable number of WD-40 repeat units, is a frequent repeated domain involved in protein-protein interaction. WD-repeat proteins fold into a circular β-propeller structure forming a protein-protein interaction platform. Comparison of different <em>het-d</em> and <em>het-e</em> alleles have indicated that this WD-repeat region is responsible for recognition specificity <a href="#pone.0000283-Espagne1">[12]</a>, <a href="#pone.0000283-Espagne3">[14]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the present paper, we describe the evolutionary mechanisms responsible for diversification of the <em>het-d</em> and <em>het-e</em> allorecognition genes. We found that <em>het-d</em> and <em>het-e</em> belong to a large gene family of 10 members sharing the NACHT/WD-repeat domain organisation. We then show that the WD-repeat regions of these genesexchange repeat units and undergo concerted evolution. 4 codon positions of the WD-repeat unit are subjected to positive diversifying selection. These 4 codon positions correspond to residues forming the protein-protein interaction surface in the β-propeller structure of the WD-repeat domain. Finally, we show that two genes of the family have been inactivated by a fungal specific mutagenesis process termed RIP <a href="#pone.0000283-Galagan1">[26]</a>. We propose that extensive diversification of the <em>het-d</em> and <em>het-e</em> genes is achieved by a unique combination of high mutation supply, concerted evolution and positive Darwinian selection.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4><em>het-D</em> and <em>het-E</em> belong to a large gene family</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Using HET-D and HET-E protein sequences in tBlastn searches on the <em>P. anserina</em> genome (<a href="http://podospora.igmors.u-psud.fr/">http://podospora.igmors.u-psud.fr/</a>), we identified a number of loci encoding proteins with high sequence similarities to HET-D and HET-E. These loci were named according to their domain organisation (<a href="#pone.0000283.s005">Table S1</a>), and a schematic representation of the family is presented <a href="#pone-0000283-g001">Figure 1</a>. In addition to <em>het-D</em> and <em>het-E</em>, three more loci comprise genes encoding for the same three domain proteins. They will be grouped under the generic name HNWD genes. We also identified two genes encoding for proteins associating only the NACHT and the WD-repeat domains, named <em>NWD1</em> and <em>NWD2</em>. Three pseudogene copies of <em>NWD</em> genes were also found. Two of them (<em>NWDp1</em> and <em>NWDp2</em>) contain numerous non sense mutations, whilst the third pseudogene (<em>NWDp3</em>) is lacking a start codon. <em>NWD3p</em> is immediately preceded by the relic of a transposable element. 60 kbp upstream, a gene encoding for a single stand alone HET domain, transcribed in the opposite direction to <em>NWDp3</em>, is present. It is conceivable that these two sequences constitute the remains of an ancestral <em>HNWD</em> gene that was split in two by the inversion of a this 60 kbp fragment. We also found 120 additional loci encoding proteins displaying a HET domain. These loci will not be analyzed further here, but note that the presence of multiple loci encoding HET proteins was already described in a number of fungal species, including <em>N. crassa</em> and <em>Aspergillus fumigatus</em> <a href="#pone.0000283-Galagan2">[27]</a>, <a href="#pone.0000283-Fedorova1">[28]</a>.</p>
<div class="figure" id="pone-0000283-g001"><div class="img"><a name="pone-0000283-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic representation of the <em>NWD</em> gene family.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The structural domains of the 10 members of the <em>NWD</em> gene family of the sequenced S strain are represented by blue boxes (HET domain), pink boxes (NACHT domain) and orange boxes (WD-repeat unit). Arrowheads represent stop codons in the pseudogenes. Numbers in brackets indicate the range of repeat number found in wild type isolates (<a href="#pone-0000283-t002">Table 2</a>). Note that <em>NWDp-3</em> pseudogene was apparently inactivated by inversion of a 60 kbp fragment splitting an ancestral <em>HNWD</em> gene in two segments with opposite orientations (arrows).</p>
<span>doi:10.1371/journal.pone.0000283.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We conclude that <em>het-D</em> and <em>het-E</em> belong to a large gene family containing both active and pseudogene members that share NACHT and WD domains.</p>


<h4><em>NWDp1</em> and <em>NWDp2</em> pseudogenes have been inactivated by RIP</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Repeat Induced Point Mutation (RIP) is a process first described in <em>N. crassa</em> that detects and mutates both copies of a duplicated sequence. Providing that the nucleotide sequences are over 80% identical, RIP is active on linked duplicated copies of a sequence over 400 bp in length, or over 1 kbp if the copies are unlinked. RIP is thought to have evolved as a defence mechanism against transposable elements. Evidence of RIP have been collected from a number of species, including <em>P. anserina</em> <a href="#pone.0000283-Bouhouche1">[29]</a>. RIP incorporates C:G to T:A mutations, preferentially in CpA dinucleotides <a href="#pone.0000283-Galagan1">[26]</a>, <a href="#pone.0000283-Cambareri1">[30]</a>. RIP reduces GC content of mutated sequences and results in skewed CA/TA and TG/TA ratios.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The two pseudogenes <em>NWDp1</em> and <em>NWDp</em>2 are heavily mutated compared to wild type <em>NWD1</em> sequence. To assess the hypothesis that <em>NWDp1</em> and <em>NWDp2</em> pseudogenes are RIP inactivated, we first determined the GC content of NWD family members. As reported in <a href="#pone.0000283.s006">Table S2</a>, GC content of pseudogenes <em>NWDp1</em> and <em>NWDp2</em> is significantly lower than for other family members. We then measured CA/TA and TG/TA ratios at all the loci of interest in the regions encoding the three functional HET, NACHT and WD domains (<a href="#pone-0000283-t001">Table 1</a>). Also, as RIP is known to be a leaky process that introduces mutations in and around the target duplicated sequences, we also estimated the CA/TA ratio in the regions surrounding the genes of interest. Reference values (CA/TA = 2.244 and TG/TA = 2.225) were obtained by analysing 20 randomly chosen 750 bp long genomic sequences. CA/TA and TG/TA ratios for the NACHT domain of the pseudogenes <em>NWDp1</em> and <em>NWDp2</em> are significantly skewed compared to NACHT domains of other family members. We also observe that low CA/TA and TG/TA ratios extend at least 750 bp upstream and downstream of the pseudogenes. From the above observations, we conclude that pseudogenes <em>NWDp1</em> and <em>NWDp2</em> have been inactivated through the RIP process. Note that considering the relatively low efficiency of RIP in <em>P. anserina</em> <a href="#pone.0000283-Bouhouche1">[29]</a>, RIP levels in <em>NWDp1</em> and <em>NWDp2</em> are very high. For example no CpA dinucleotides are found in the 750 bp upstream of the <em>NWDp2</em> pseudogene, compared to 44 CpA dinucleotides found in the corresponding region of functional <em>NWD1</em>, strongly suggesting that multiple rounds of RIP acted on the pseudogenes.</p>
<div class="figure" id="pone-0000283-t001"><div class="img"><a name="pone-0000283-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000283.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000283.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>CA/TA and TG/TA ratios at <em>NWD</em> loci.</span></strong></p><span>doi:10.1371/journal.pone.0000283.t001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Intriguingly, the CA/TA and TG/TA ratios in the WD domain of <em>NWDp1</em> and <em>NWDp2</em> loci are comparable to those observed in the WD domains of all the active loci, and to that of the rest of the genome. Translation of the NACHT domains of the pseudogenes reveals an accumulation of non sense mutations. In contrast, the WD domain of the <em>NWDp2</em> locus is devoid of stop codons over a total length of 5 repeats of the WD40 unit, whilst in the <em>NWDp1</em> WD domain, a single stop codon occurs at the beginning of the 7<sup>th</sup> of the WD-40 unit (<a href="#pone-0000283-g001">Figure 1</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>In conclusion, the <em>NWDp1</em> and <em>NWDp2</em> pseudogenes have been subjected to extensive RIP. However, this high level of RIP is not observed in the WD-40 sequences of the pseudogenes. This observation might be explained by hypothesizing that WD-repeats of the pseudogenes undergo concerted evolution with the WD repeats of the active loci so that repeats in the pseudogenes can be replaced by repeats originating from the active loci.</p>


<h4>The WD-40 repeats undergo concerted evolution both within and between loci</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In addition to the presence of intact WD-40 repeats in the pseudogenes loci outlined above, a second observation suggested concerted evolution of the WD repeats of the <em>NWD</em> gene family. Individual WD-40 repeats of the gene family are extremely conserved within a gene as well as between genes with up to 100% identity. This conservation level is striking considering that the WD-40 motif is loosely defined with only a few key residues critical for the β-propeller fold. In other WD-repeat proteins in <em>P. anserina</em> and other species, conservation of repeat units rarely exceeds 10 to 20% at the protein level <a href="#pone.0000283-Smith2">[31]</a> and is in most cases barely detectable at the nucleotide level. Concerted evolution is a process that leads to homogenization of repeated sequences. Concerted evolution is driven by unequal crossing overs and/or gene conversion events in repeat arrays. During concerted evolution a mutation in any given repeat can ultimately either spread to all of the repeats or be eliminated, as a result all the repeats evolve as a unit. Concerted evolution is detected in phylogenetic analyses by sequences grouping as species of origin rather than as orthologues <a href="#pone.0000283-Nei1">[32]</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To perform this type of phylogenetic analysis we searched for sequences similar to HET-D and HET-E in the fungal databases. Interestingly, <em>HNWD</em> type of sequences were not found in any other organism, even in closely related species such as <em>N. crassa</em> or <em>Chaetomium globosum</em>. However, <em>NWD</em> orthologues were identified in the genomes of 5 additional species, <em>C. globosum, Giberella zeae, Aspergillus nidulans, Aspergillus oryzae</em> and <em>Aspergillus fumigatus</em>. An ITS based phylogenetic tree of these species is presented on <a href="#pone.0000283.s001">Figure S1</a>. All available <em>NWD</em> sequences were collected (<a href="#pone.0000283.s007">Table S3</a>) and a phylogenetic tree of all the individual WD40 units was constructed using Kimura two parameter distances. We conducted bootstrap and interior branch tests available in the MEGA 3.1 package to assess statistical significance of the phylogenetic trees. Both tests resulted in different statistical support of the tree produced as already reported <a href="#pone.0000283-Nei2">[33]</a>. In this WD-repeat phylogenetic tree, all <em>P. anserina</em> WD-40 repeat sequences cluster together, including repeats from the pseudogenes (<a href="#pone-0000283-g002">Figure 2A and S2</a>). The <em>P. anserina</em> clade also includes WD-40 sequences from a single <em>F. graminearum</em> gene, possibly revealing a common ancestral origin. Similarly, all WD40 sequences from <em>Ch. globosum</em> group together. Other WD-40 sequences are mixed up. The clustering of <em>P. anserina</em> sequences was well supported by the interior branch test, but only the two main internal nodes were well supported by bootstrapping analysis. Note that interior branch test is more reliable for closely related sequences <a href="#pone.0000283-Nei2">[33]</a>. Another striking point is that even if WD-40 sequences from <em>het-D</em> and <em>HNWD1</em>, and from <em>het-E</em> and <em>HNWD3</em> are mixed (<a href="#pone.0000283.s003">Figure S3</a>), WD-40 sequences tend to cluster by gene of origin, suggesting preferential intralocus recombinations. Note that parsimony trees lead to the same conclusions (<a href="#pone.0000283.s004">Figure S4</a>)</p>
<div class="figure" id="pone-0000283-g002"><div class="img"><a name="pone-0000283-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Concerted evolution of the WD-40 repeats.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A/Schematic representation of a Neighbor-Joining phylogenetic tree of individual WD-40 sequences. Each box represents a single WD-40 sequence. Asterisks indicate Internal Branch Length tast values over 80. The corresponding full size tree is presented on <a href="#pone.0000283.s003">figure S3</a>. B/Neighbor-Joining phylogenetic tree of NACHT domain sequences. Each WD-40 unit is designated by its gene of origin and position in the domain. Cyan branches indicate sequences associated to NACHT domains belonging to N-I clade of the NACHT phylogeny. Internal Branch Length test values over 80 are indicated. For both trees, species of origin are colour coded.</p>
<span>doi:10.1371/journal.pone.0000283.g002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>“Whole-gene” and NACHT phylogenetic trees were also constructed with the same genes. In the NACHT domain phylogeny the RIP inactivated NACHT domains were excluded because they are too divergent for proper alignment. The “whole-gene” and NACHT phylogenetic trees resulted in similar topologies which was to be expected as the NACHT domains are more divergent than the WD-repeat domains and hence outweight the influence of the WD-repeat on the tree topology. Both phylogenies gave evidence for two well supported main clades (N-I and N-II, <a href="#pone-0000283-g002">Figure 2b</a>). The smallest clade N-I contains <em>P. anserina</em> <em>NWD2</em>, as well as at least one representative from all the other species. N-II includes the remaining sequences. Within N-II, <em>P. anserina</em> sequences are spread over two well supported sub-clades each also containing sequences from other species. The statistical support by the bootstrap or the internal branch method for both the “whole-gene” phylogeny and the NACHT phylogeny are roughly in agreement.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The fact that all <em>Ch. globosum</em> and <em>P. anserina</em> WD-40 sequences group as species rather than orthologs is proof of concerted evolution of the WD-40 sequences. An alternative hypothesis to concerted evolution of the WD40 sequences would be a recent duplication of the genes in Podospora or a closely related ancestor. However, the grouping of WD-40 sequences from the N-I and N-II NACHT domain clades allows to reject this hypothesis. Additional data support the hypothesis of concerted evolution. The presence in <em>P. anserina</em> of functional WD domains at the pseudogene loci is readily accounted for by concerted evolution of the repeats but not in the recent gene expansion hypothesis. From the above observations, we conclude that the WD-40 sequences from <em>P. anserina</em> (and incidently <em>Ch. globosum</em>) are undergoing concerted evolution. In other words, the repeat units of the entire gene family undergo genetic exchanges by unequal crossing-over and/or gene conversion and as a result they evolve in concert. These phylogenetic analyses indicate that potentially each repeat unit at any given locus from the gene family can be recombined with any repeat of any locus of the family. It follows that any given repeat is a potential target for mutations that can then be introduced in repeat arrays of other family members. As a consequence, mutation supply in the repeat region is significantly increased. The total number of templates for mutation is 69 repeats in the sequenced <em>S</em> strain.</p>


<h4>WD repeat number is highly variable in all members of the family</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>If driven by unequal crossing-overs, concerted evolution will result in repeat array size polymorphism. Repeat number polymorphism in wild <em>P. anserina</em> isolates was already reported for <em>het-D</em> and <em>het-E</em> alleles <a href="#pone.0000283-Espagne1">[12]</a>. We determined the number of WD-40 repeats by PCR amplification of the WD-repeat domain from each <em>NWD</em> locus in wild <em>P. anserina</em> isolates. We observed an extensive repeat number polymorphism (<a href="#pone-0000283-t002">Table 2</a>). No two isolates have the same WD-40 unit number distribution pattern suggesting unequal crossing-overs contribute to WD-40 sequence redistribution. Note however that the total number of WD repeats is constantly high, with over 50 repeats of the WD-40 unit in each isolate.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In agreement with the detection of concerted evolution in the WD-40 repeats, we show that all members of the gene family show extensive repeat number polymorphism suggesting that repeats are actively exchanging in wild populations.</p>
<div class="figure" id="pone-0000283-t002"><div class="img"><a name="pone-0000283-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000283.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000283.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>WD-repeat number polymorphism.</span></strong></p><span>doi:10.1371/journal.pone.0000283.t002</span></div>

<h4>4 codon positions of the WD-40 repeats are under positive diversifying selection</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Positive Darwinian selection is common in self/non self recognition genes <a href="#pone.0000283-Richman1">[6]</a> and describes the fact that acquisition of non synonymous mutations is favoured hence promoting evolution of the encoded protein <a href="#pone.0000283-Nei1">[32]</a>, <a href="#pone.0000283-Nielsen1">[34]</a>, <a href="#pone.0000283-Nei3">[35]</a>. Since the WD-repeats control recognition specificity, we assessed the possibility that they might be under positive Darwinian selection. The most common test for detecting direction of selection is to determine the Dn/Ds ratio, i. e. comparing the ratio of non synonymous mutation per non synonymous site to the ratio of synonymous mutations per synonymous sites. A Dn/Ds ratio close to 1 is indicative of neutral evolution. Dn/Ds&lt;1 reveals purifying selection whereas Dn/Ds&gt;1 indicates positive selection.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Since the WD-40 of the family evolve as a unit, in concert, their comparison is evolutionary meaningful. We performed two different types of analysis using the whole set of 69 <em>P. anserina</em> WD40 sequences from the sequenced <em>S</em> strain, the Suzuki and Nei counting model (1999), and the Fixed Effect Likelyhood (FEL) implemented by HyPhy <a href="#pone.0000283-KosakovskyPond1">[36]</a>. Both models identified 4 codons (codons number 7, 9 25 and 27) under strong positive selection, and a number of less strongly selected positions (<a href="#pone-0000283-g003">Figure 3</a>). Sites under strong positive selection work in pairs, 2 positively selected codons surrounding a site under strong purifying selection (codons 8 and 26). Purifying selection at position 8 and 26 is fully consistent with the fact that these positions are part of the WD-repeat consensus <a href="#pone.0000283-Smith2">[31]</a> .</p>
<div class="figure" id="pone-0000283-g003"><div class="img"><a name="pone-0000283-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Positive selection on the WD-40 repeat sequences.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Dn/Ds ratio calculated by the FEL method (p-value&lt;0.05) represented along a consensus of the WD-40 amino acid sequence (u = A, G or S, o = S or T). Black arrows indicate regions of the WD-40 sequence forming β-strands.</p>
<span>doi:10.1371/journal.pone.0000283.g003</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Together the 69 WD-40 repeat units present in the genome of the sequenced <em>S</em> strain form 55 different combinations of amino acids at the 4 positions under positive selection (<a href="#pone.0000283.s008">Table S4</a>). This observation suggests that the WD-40 sequences are evolving under diversifying selection rather than adaptive-directional-selection <a href="#pone.0000283-Nielsen1">[34]</a>.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>We conclude that selection promotes acquisition and maintenance of a high number of polymorphism at these specific positions in the pool of WD-40 sequences of the <em>NWD</em> gene family.</p>


<h4>The residues under positive diversifying selection in the WD-40 repeats group at the protein-protein interaction platform</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The WD repeats adopt a circular structure with a central pore called the β-propeller that assembles 7–8 WD units, each repeat being composed of 4 short antiparallel β-strands (<em>a, b, c</em> and <em>d, a</em> being the closest to the central pore). In each subdomain, strand <em>d</em> from one repeat associates with strands <em>a, b</em> and <em>c</em> from the next repeat. The β-propeller displays a flat rigid protein-protein interaction platform at the top of the structure. Surface residues of this platform are responsible for protein/protein interactions in WD proteins such as Tup1 <a href="#pone.0000283-Sprague1">[37]</a>, the Gβ subunits <a href="#pone.0000283-GarciaHiguera1">[38]</a> and net2 <a href="#pone.0000283-Cerveny1">[39]</a> (For a review <a href="#pone.0000283-Smith2">[31]</a>). A structural model of the WD-40 repeat domain of HET-E was generated by homology modelling to WD-40 domains of known structure. When positioned on this homology-based three dimensional model of the HET-E WD domain, the residues corresponding to the 4 codon positions under positive selection are all located on the top of the β-propeller on the surface of the interaction platform (<a href="#pone-0000283-g004">fig. 4a</a>). Pair (7, 9) is located in the loop connecting strand <em>d</em> from one repeat to strand <em>a</em> in the next repeat for and pair (25, 27) in a β-turn linking strand <em>b</em> and <em>c</em> within a repeat (<a href="#pone-0000283-g004">fig. 4a</a>). This positioning in the WD domain of the residues under selection strongly suggests that they are involved in protein-protein interactions.</p>
<div class="figure" id="pone-0000283-g004"><div class="img"><a name="pone-0000283-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000283" data-uri="info:doi/10.1371/journal.pone.0000283.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000283.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000283.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000283.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000283.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Analogy based modelling of interacting partners.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>A/Top and side view of the predicted structure of the HET-E WD repeat domain. Positions under strong positive selection are coloured. B/Predicted structure of the <em>het-C</em> gene product encoding a glycolipid transfer protein. Red amino acids correspond to positions experimentally shown to be involved in recognition specificities, yellow amino acids represent highly polymorphic positions between <em>het-C</em> alleles. Note that residue 126 is located in a unstructured loop, so its positioning is uncertain. Residues forming the sugar binding site are coloured in cyan, residues forming the lipid binding tunnel are coloured in blue. Polymorphic residues are located outside the sugar binding site and the lipid binding hydrophobic tunnel.</p>
<span>doi:10.1371/journal.pone.0000283.g004</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p><em>het-C</em> is the interacting partner of <em>het-D</em> and <em>het-E</em> in incompatibility. Incompatibility is triggered by interaction of specific allele pairs of <em>het-C</em> and <em>het-D</em>, and <em>het-C</em> and <em>het-E</em>. <em>het-C</em> encodes for a glycolipid transfer protein that also assumes developmental functions <a href="#pone.0000283-Saupe4">[21]</a>. Although the restricted number of <em>het-C</em> allele sequences prevents genesis of statistically significant results, comparison of the four known alleles led to two important observations. First, <em>het-C</em> alleles are more variable in the open reading frame than in 5′ or 3′ non coding regions and introns with about twice as many mutations in coding regions as in non coding regions. Second, polymorphism in the open reading frame results in an excess of non synonymous over synonymous mutations, with only two synonymous mutations observed for 15 non synonymous mutations between the two most divergent alleles <a href="#pone.0000283-Saupe3">[16]</a>. These observations are suggestive of positive selection acting on <em>het-C</em>. At least 2 codons in positions 133 and 153 involved in determining allelic specificities with <em>het-D</em> and <em>het-E</em> were identified, as well as a highly polymorphic region comprising codons 126, 140/141 and 144/145 <a href="#pone.0000283-Saupe3">[16]</a>. As for the HET-E WD repeat domain, we generated a structure model of HET-C by homology modelling to human GLTP <a href="#pone.0000283-Malinina1">[40]</a>. On this structure model, residues 126, 133, 153, 140/141 and 144/145 are all located at the surface of the HET-C protein roughly in the same region, and are not part of the hydrophobic tunnel nor the sugar binding site involved in glycolipid binding (<a href="#pone-0000283-g004">Figure 4B</a>) <a href="#pone.0000283-Malinina1">[40]</a>, <a href="#pone.0000283-Malakhova1">[41]</a>. These data suggest that variable positions in HET-C could form an interaction interface with WD domains of HET-D and HET-E, and potentially that the WD sequences and <em>het-C</em> are co-evolving.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Genesis of new allorecognition alleles</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Most living organisms, from bacteria to higher eukaryotes, have developed allorecognition systems to discriminate self from non self tissues and cells. A remarkable property of recognition genes is their ability to accumulate polymorphism, they are often under positive Darwinian selection <a href="#pone.0000283-Nielsen1">[34]</a>. It is generally accepted that balanced selection that necessarily involves some type of rare allele advantage contributes to fixation of rare advantageous alleles <a href="#pone.0000283-Richman1">[6]</a>. However, although highly polymorphic sequences are central for allorecognition, the mechanisms of their genesis are often unclear. The low mutation rate in most cellular organisms imposes a serious limitation on diversification and adaptation in general <a href="#pone.0000283-Drake1">[42]</a>, and particularly so in small or well adapted populations <a href="#pone.0000283-Arjan1">[43]</a>. In fungi, allorecognition functions are ensured by vegetative incompatibility loci.We show here that the combination of high WD-40 repeat sequence number, concerted evolution and positive selection provides the <em>P. anserina NWD</em> gene family-which includes several allorecognition genes-with a dynamic system by which mutations in WD40 repeats can be acquired and spread to the whole family and positively selected if advantageous. All the 70 or so WD-40 units of the <em>NWD</em> family members, the basic elements of the WD recognition domain, are multiple potential targets for mutagenesis which <em>per se</em> might already be a significant enhancement of mutation supply. The fact that RIP, as a repeat-induced hypermutation process might also contribute to increase mutation supply and diversification of the WD-40 repertoire is discussed in a following section. Concerted evolution will then allow redistribution of repeat variants from this large WD-40 sequence repertoire between family members. Finally and importantly, positive selection will filter mutations in the loops of the interaction platform of the β-propeller structure to retain favourable mutations. In addition, the concerted evolution process will also ensure constant homogenization of the remaining of the WD-40 sequence thereby maintaining competency for further rounds of redistribution.</p>


<h4>To what extent RIP contributes to WD-repeat sequence diversification?</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Two pseudogene members of the NWD family have been inactivated by the fungal specific hypermutation process called RIP. RIP introduces CT to TA and TG to TA mutations, increasing chances of introducing amber and ochre stop codons. RIP was first described as a defence mechanism against transposable elements. Indeed, relics of transposable elements riddled with stop codons have been found in a number of species including <em>P. anserina</em> <a href="#pone.0000283-Hamann1">[44]</a>, <em>Leptospharia maculans</em> <a href="#pone.0000283-Gout1">[45]</a> and Aspergillus species <a href="#pone.0000283-Montiel1">[46]</a>. In <em>N. crassa</em>, RIP mutated sequences are methylated, resulting in their transcriptional silencing <a href="#pone.0000283-Galagan1">[26]</a>. However, despite thorough investigation, DNA methylation could not be associated to RIP in Aspergillus species <a href="#pone.0000283-Montiel1">[46]</a>, and to the best of our knowledge RIP associated methylation has never been reported for any organism other than <em>N. crassa</em>. Also, RIP mutagenesis is by far the most efficient in <em>N. crassa</em>. Consequently, one function of low efficiency RIP might be to promote evolution of repeat sequences <a href="#pone.0000283-Galagan1">[26]</a>, <a href="#pone.0000283-Braun1">[47]</a>, while high mutagenesis and transcriptional silencing add extra strength to genome defence in <em>N. crassa</em>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>When excluding the WD repeat sequences, RIP inactivated <em>NWDp1</em> and <em>NWDp2</em> pseudogene sequences are about 80% identical, corresponding to the threshold level beyond which RIP is believed to become ineffective <a href="#pone.0000283-Galagan1">[26]</a>. <em>NWDp1</em> and <em>NWDp2</em> are most likely the products of a gene duplication, both duplicated copies being subsequently inactivated by multiple rounds of RIP. This inactivation must have introduced RIP mutations within the WD-repeat sequences as well, thereby contributing to WD sequence diversification. It can thus be stated that at least at the time of the initial RIPping of the <em>NWDp1</em> and <em>NWDp2</em> genes, RIP has provided a mutation input in the WD-40 repeat repertoire. Sequences over 80% identical and over 1 kbp long are found in WD-repeat arrays of <em>P. anserina NWD</em> gene family, thus-if these repeats are no refractory to RIP for some unknown reason-, the WD-repeats should <em>a priori</em> trigger RIP. We have counted the number of RIP characteristic mutations (C to T and G to A transitions in a CA or TG context) at silent positions only (third base of each codon) to avoid any effect of selection. The sample of such potential selection-free RIP targets is limited and prevents convincing statistical analysis, however as expected under RIP, we noted that most CpA dinucleotides (8/9) are mutated, and number of C to T transitions are more frequent at CpA dinucleotides (78%) as compared to other dinucleotides (around 40%) (<a href="#pone.0000283.s009">Table S5</a>). These observations are compatible with the hypothesis that RIP might be occurring in WD-repeat sequences, so that these repeats might be constantly promoting their own diversification. Clear footprints of RIP might well be blurred by the purifying and positive selection acting on the repeats as well as the concerted evolution process which <em>per se</em> promotes homogenisation. Only experimental determination of the levels of RIP in the WD-repeats in the absence of selection will allow to determine to what extent RIP participates to mutation supply in this gene family. The two extreme possibilities are that RIP introduced mutations in the repeat pool only at the time of duplication of <em>NWD1p</em> and <em>NWDp2</em> or that WD-repeat triggered RIP constantly allows locus specific hypermutation of the repeat arrays.</p>


<h4>Positive selection on <em>het</em> genes</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Acquisition/maintenance of new allele specificities at <em>het</em> loci in different fungal species rely on extremely unusual processes. In <em>O. novo-ulmi</em>, the dutch elm disease pathogen, acquisition of new <em>het</em> alleles (as well as mating type alleles) occurs through introgression from a closely related species <a href="#pone.0000283-Paoletti1">[48]</a>. In <em>N. crassa</em>, balancing selection is acting at the <em>het-C</em> locus to maintain three functional alleles, and evidences for ancient trans-species polymorphism were presented <a href="#pone.0000283-Wu1">[49]</a>. These observations highlight the essential role that vegetative incompatibility must play in fungal biology. This conclusion is now comforted by the fact that we identified positive Darwinian selection acting on positions forming the protein/protein interface of a recognition gene family. Note that genes controlling programmed cell death seem to be a target of choice for selection to act on. A<em>paf-1</em> encoded protein controlling apoptosis in mammals display a structural organisation similar to proteins of the <em>HNWD</em> gene family with a N-terminal CARD effector domain, a central NACHT domain and a C-terminal domain containing 12 WD-repeats Recently, it was shown that <em>apaf-1</em>, is also subjected to positive selection acting on the WD domain <a href="#pone.0000283-Vallender1">[50]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Among the NWD genes, only <em>het-D</em> and <em>het-E</em> have been characterized as vegetative incompatibility genes so far <a href="#pone.0000283-Espagne1">[12]</a>. It is likely that other members of the family also correspond to incompatibility genes, in particular those displaying a HET domain characteristic of proteins related to incompatibility <a href="#pone.0000283-Espagne1">[12]</a>, <a href="#pone.0000283-Saupe2">[15]</a>, <a href="#pone.0000283-Glass3">[23]</a>, <a href="#pone.0000283-Shiu1">[51]</a>, <a href="#pone.0000283-Kaneko1">[52]</a>. Molecular genetic analyses indicate that <em>HNWD2</em> in either tightly linked or congruent to the <em>het-R</em> incompatibility gene <a href="#pone.0000283-Labarere1">[53]</a>, (D. Chevanne and M.P., unpublished results).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Clearly positive diversifying selection is acting on the WD-repeats of this set of genes, but what is actually selected? Combining the large number of different WD-repeats to the ability to re-shuffle them by concerted evolution has the potential to generate very large numbers of non self recognition alleles. However, an incompatibility system is defined by the interaction of two partners and in the case of <em>het-d</em> and <em>het-e,</em> this partner is the <em>het-c</em> gene. Four alleles of <em>het-C</em> have been identified in a sample of 16 wild isolates, which is not in terms with the level of variability in the WD-40 sequences. This leads to an apparent paradox : what is the use of being able to generate very large numbers of <em>het-d</em> and <em>het-e</em> alleles when there are only 4 <em>het-c</em> alleles around. Several hypotheses can be envisaged to explain this discrepancy. Using chimeric <em>het-C</em> alleles it was shown that new interaction specificities can be obtained experimentally <a href="#pone.0000283-Saupe3">[16]</a>. So, one may hypothesize that other yet uncharacterized natural <em>het-C</em> alleles exist in wild populations. Alternatively, <em>het-C</em> may not be free to diverge extensively because it assumes a cellular function in spore formation <a href="#pone.0000283-Saupe4">[21]</a>. Thus, it might be the number of <em>het-C</em> alleles is indeed limited to the four known alleles. Then how can one explain the diversity in <em>het-D</em> and <em>het-E</em>? As stated above, the other <em>HNWD</em> genes are likely additional incompatibility genes whose interacting partners are unknown, so that incompatible interactions with <em>het-c</em> may not be the only interactions that promote divergence of the repeats. Indeed the <em>het-R</em>/<em>het-V</em> system is independent of <em>het-C</em> (unpublished).</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>An altogether different view would be that selection is not acting on emergence of novel phenotypic classes but on repeated emergence of incompatibility towards the same four <em>het-c</em> alleles. Mutation and re-shuffling of the WD-40 repeats would perpetually restore already existing phenotypes. One striking characteristic of the incompatibility system we describe here is its dynamic character, in contrast to incompatibility systems such as the <em>N. crassa het-C</em> system that has maintained the same three alleles in different species over tens of million years of evolution (Wu et al 1998). As a consequence of this dynamic character, <em>het-e</em> and <em>het-d</em> alleles seem to be genetically unstable. In spite of the strong positive selection in the repeats-which strongly argues for an adaptive advantage associated to incompatibility-numerous wild isolates have inactive <em>het-d</em> and <em>het-e</em> alleles as a result of repeat loss, as for instance in the sequenced <em>S</em> strain. Similarly, in genetic screens selecting for loss of incompatibility, <em>het-d</em> and <em>het-e</em> inactivation through WD-repeat loss is by far the most frequent event <a href="#pone.0000283-Begueret1">[54]</a>. As a flipside of the dynamic character of the concerted evolution process, the <em>het-d</em> and <em>het-e</em> incompatibility genes appear to be genetically fragile. It appears however that in every analysed wild isolate at least two <em>HNWD</em> genes contain a minimum of 10 repeats, the apparent lower limit for activity in incompatibility <a href="#pone.0000283-Espagne3">[14]</a>, <a href="#pone.0000283-Saupe2">[15]</a>. It might be that the signature of positive selection in the WD-repeats results from reoccurring episodes of loss and subsequent reacquisition of the incompatibility function.</p>


<h4>General implications</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Fast evolving sets of genes have been found in various organisms. For instance, immunoglobulin genes in mammals evolved through a combination of high gene copy number, DNA shuffling between exons of the genes into expressed sites, and high mutation rate to optimise recognition (for a review, <a href="#pone.0000283-Maizels1">[55]</a>. In parasites such as <em>P. falciparum</em>, antigen surface encoding genes need to evolve rapidly to avoid recognition by the immune system, and this is achieved by DNA shuffling between a single expressed site and numerous silent loci, introducing mutations in the process <a href="#pone.0000283-Barry1">[56]</a>. In plants, production of a variety of resistance genes to pathogens is promoted through shuffling of sets of resistance genes <a href="#pone.0000283-MondragonPalomino1">[57]</a>, and in the fungus <em>L. maculans</em>, RIP could promote evolution of an <em>avr</em> gene located in a gene desert heavily subjected to RIP <a href="#pone.0000283-Gout1">[45]</a>. In bacteria, high mutation rate associated to mutator genes allow rapid adaptation to new selective conditions <a href="#pone.0000283-Metzgar1">[58]</a>. In <em>P. anserina</em> genesis of new allorecognition specificities relies on the combination of the DNA shuffling of a large repertoire of repeats between expressed and non expressed loci and a step of selection for advantageous new variants. RIP as a repeat specific hypermutation process could further contribute significantly to diversification. It would then seem that organisms have evolved or exploited various mechanisms to achieve rapid gene diversification through strategically similar processes involving high mutation input, DNA sequence re-shuffling and positive selection.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sequence mining</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Sequences were identified by tblastn searching of the <em>P. anserina</em> database (<a href="http://podospora.igmors.u-psud.fr/">http://podospora.igmors.u-psud.fr/</a>), the NCBI database (<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>), the Broad Institute Fungal Genome Initiative database (<a href="http://www.broad.mit.edu/cgi-bin/annotation/fgi/blast_page.cgi">http://www.broad.mit.edu/cgi-bin/annotat​ion/fgi/blast_page.cgi</a>), the TIGR <em>Aspergillus fumigatus</em> database (<a href="http://www.tigr.org/tdb/e2k1/afu1/">http://www.tigr.org/tdb/e2k1/afu1/</a>), and the <em>A. oryzae</em> database (<a href="http://www.bio.nite.go.jp/dogan/Top">http://www.bio.nite.go.jp/dogan/Top</a>) with <em>het-D</em> and <em>het-E</em> sequences. NACHT and WD-repeat domains from non annotated sequences were identified using the Prosite program (<a href="http://www.expasy.org/prosite/">http://www.expasy.org/prosite/</a>). For <em>P. anserina</em> genes, HET domain <a href="#pone.0000283-Smith1">[22]</a> corresponds to the extension N-terminal to the NACHT domain. All sequences are presented in <a href="#pone.0000283.s005">Tables S1</a> and <a href="#pone.0000283.s007">S3</a>.</p>


<h4>Positive selection analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For positive selection detection, the Fixed Effect Likelihood model (FEL) implemented in HyPhy <a href="#pone.0000283-KosakovskyPond1">[36]</a>, <a href="#pone.0000283-Pond1">[59]</a> was used with the HKY85+G substitution model found to be the best matrix by HyPhy's model selection program. We also ran a SLAC method derived from the Suzuki and Gojobori model <a href="#pone.0000283-Suzuki1">[60]</a>, implemented as a Web interface on the <a href="http://www.datamonkey.org">www.datamonkey.org</a> web site. Consensus protein sequence of the WD-40 unit was determined by the Consensus program (<a href="http://coot.embl.de/Alignment/consensus.html">http://coot.embl.de/Alignment/consensus.​html</a>). Cumulative behaviour of the <em>het-C</em> alleles was determined using the program SNAP implemented as a web site interface (<a href="http://www.hiv.lanl.gov/content/hiv-db/SNAP/WEBSNAP/SNAP.html">http://www.hiv.lanl.gov/content/hiv-db/S​NAP/WEBSNAP/SNAP.html</a>) based on Nei and Gojobori method <a href="#pone.0000283-Nei4">[61]</a>.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Sequence alignments were performed with ClustalW, and phylogenetic analysis were computed using the MEGA3.1 package <a href="#pone.0000283-Kumar1">[62]</a>. Neighbor-Joining trees <a href="#pone.0000283-Saitou1">[63]</a> were constructed based on Kimura 2-parameters distances <a href="#pone.0000283-Kimura1">[64]</a>. Internal Branch Tests were conducted using 1000 replicates. Parsimony phylogenies were also determined and bootstrap analysis conducted with 1000 replicates. RIP inactivated NACHT sequences as well as <em>A. oryzae</em> WD sequences from locus Ao0041 were excluded because too divergent for proper alignment. Neighbor-joining is a distance based method of phylogeny reconstruction. In the Parsimony approach, for any given topology the sum of the minimum possible substitutions over all sites is known as the tree length for that topology. The topology with the minimum tree length is known as the Maximum Parsimony tree. Merits and demerits of these methods were discussed <a href="#pone.0000283-Nei2">[33]</a>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Highly conserved gene families have been shown to evolve according to either of two processes, “Birth and Death with Strong purifying Selection” or “concerted evolution” <a href="#pone.0000283-Nei1">[32]</a>, <a href="#pone.0000283-Nei3">[35]</a>. In the Birth and Death with purifying selection model duplicated sequences evolve separately in a genome, and purifying selection will ensure conservation of the sequences. In this case, genes of a family will cluster as orthologues. In contrast, in the concerted evolution process, members of a family evolve as a unit within a genome by unequal crossing-overs and/or gene conversion and sequences will then group as species of origin rather than as orthologues.</p>


<h4>Structure modelling</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>HET-E1 and HET-C2 structure models were generated by homology modelling using the Geno3D web interface (<a href="http://geno3d-pbil.ibcp.fr/">http://geno3d-pbil.ibcp.fr/</a>) <a href="#pone.0000283-Combet1">[65]</a>, graphic representations were made with Pymol.</p>


<h4>PCR</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>PCR amplifications were performed with 10 ng genomic DNA, 100 ng of each primer, 100 µM each dNTP, 1×Taq polymerase buffer containing magnesium chloride, 1U Taq DNA polymerase (Qbiogen). After 5 min denaturation at 95°C, 35 cycles of 30 sec at 95°C, 30 sec at 55°C and 3 min at 72°C were performed before gel electrophoresis. Primers are listed on <a href="#pone.0000283.s010">Table S6</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000283"></div><a name="pone.0000283.s001" id="pone.0000283.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogenetic relationship of the species under study. An ITS sequence Neighbor-Joining phylogeny was constructed and bootstrap tests conducted with 1000 replicates (bold). Genetic distances are indicated. Accession numbers for the ITS sequences are: AY278557 (P. anserina), DQ336707 (C. globosum), DQ453701 (F. graminearum), AJ937756 (A. nidulans), DQ401534 (A. fumigatus) and DQ411551 (A. oryzae).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000283.s002" id="pone.0000283.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Close up view of the P. anserina clade presented <a href="#pone-0000283-g002">Fig. 2B</a>. WD-40 repeats are designated by the gene of origin and their position from the first WD-40 repeat at the N-terminal end of the WD-repeat domain. Genes of origin are colour coded. Cyan branches indicate sequences associated to NACHT domains grouping in the N-I clade of the NACHT phylogeny. Internal Branch Length test values over 80 are indicated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000283.s003" id="pone.0000283.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Full size WD-40 unit Neighbor-Joining phylogenetic tree identical to <a href="#pone-0000283-g002">figure 2A</a>. Loci of origin are noted, and species of origin are colour coded. Each WD-40 unit is designated by the gene of origin and the number of the WD-40 repeat from the N-terminal end of the domain. Cyan branches indicate sequences associated to NACHT domains grouping in the N-I clade of the NACHT phylogeny. Internal Branch Length test values over 80 are indicated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000283.s004" id="pone.0000283.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Maximum Parsimony phylogenetic trees constructed with the same data set as Neighbor-Joining tree reported <a href="#pone-0000283-g002">Fig. 2</a>. A/NACHT phylogeny, B/WD-40 phylogeny. Loci of origin are noted, and species of origin are colour coded. Each WD-40 unit is designated by the gene of origin and the number of the WD-40 repeat from the N-terminal end of the domain. Cyan branches of the WD-40 phylogeny indicate sequences associated to NACHT domains grouping in the N-I clade of the NACHT phylogeny. Bootstrap values over 50 are indicated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0000283.s005" id="pone.0000283.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s005">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Accession numbers of P. anserina NWD family genes Genes newly identified were named according to their structural composition. H = HET domain, N = NACHT domain, WD = WD-repeat domain. Pseudogenes are indicated by the suffixe p.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000283.s006" id="pone.0000283.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s006">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">GC content of the NWD family GC content of the NWD loci were determined for each member of the gene family. Control values (CTL) were defined as the GC and TA contents averaged over twenty randomly chosen 750bp control sequences. Shaded values are significantly lower than the rest of the dataset at a 95% confidence interval.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000283.s007" id="pone.0000283.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s007">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Genes of the NWD family identified in other fungal species.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000283.s008" id="pone.0000283.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s008">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">55 combinations of amino acids at the positions under positive selection in the WD-40 repeats of the P. anserina NWD gene family. WD sequences are named as in <a href="#pone-0000283-g002">figure 2</a>. Identical combinations are colour shaded, and the number of different amino acids at each position is indicated</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0000283.s009" id="pone.0000283.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s009">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Mutation frequencies at silent positions in the WD-40 sequences. WD-40 sequences were analysed as two pools according to their belonging to one of the two main clades in the WD40-phylogeny presented <a href="#pone-0000283-g002">figure 2</a>. One pool comprised WD-40 sequences from loci NWD-1, het-E, NWDp1, NWDp2, NWDp3 and HNWD3, the other pool comprised sequences from loci NWD2, Het-D, HNWD1 and HNWD2. C to T transitions in the pool of WD-40 sequences compared to the consensus sequences were counted on both strands and are reported according to their context.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000283.s010" id="pone.0000283.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000283.s010">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Primers used to PCR amplify WD repeat domains from P. anserina NWD gene family members.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000283.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors wish to thank the <em>P. anserina</em> consortium and the Genoscope for sequencing <em>P. anserina</em> genome.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MP. Performed the experiments: SS. Analyzed the data: MP. Wrote the paper: SS MP CC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000283-Medzhitov1" id="pone.0000283-Medzhitov1"></a>Medzhitov R, Janeway CA, Jr. (2002) Decoding the patterns of self and nonself by the innate immune system. Science  296: 298–300.  <ul class="find" data-citedArticleID="936484" data-doi="10.1126/science.1068883"><li><a href="http://dx.doi.org/10.1126/science.1068883" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Decoding+the+patterns+of+self+and+nonself+by+the+innate+immune+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Decoding+the+patterns+of+self+and+nonself+by+the+innate+immune+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000283-Queller1" id="pone.0000283-Queller1"></a>Queller DC, Strassmann JE (2002) The many selves of social insects. Science  296: 311–313.  <ul class="find" data-citedArticleID="936514" data-doi="10.1126/science.1070671"><li><a href="http://dx.doi.org/10.1126/science.1070671" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+many+selves+of+social+insects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+many+selves+of+social+insects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000283-Nasrallah1" id="pone.0000283-Nasrallah1"></a>Nasrallah  (2005) Recognition and rejection of self in plant self-incompatibility: comparisons to animal histocompatibility. Trends in Immunology  26: 412–418.  <ul class="find" data-citedArticleID="936492" data-doi="10.1016/j.it.2005.06.005"><li><a href="http://dx.doi.org/10.1016/j.it.2005.06.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+and+rejection+of+self+in+plant+self-incompatibility%3A+comparisons+to+animal+histocompatibility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+and+rejection+of+self+in+plant+self-incompatibility%3A+comparisons+to+animal+histocompatibility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000283-Buss1" id="pone.0000283-Buss1"></a>Buss LW (1982) Somatic cell parasitism and the evolution of somatic tissue compatibility. Proc Natl Acad Sci U S A  79: 5337–5341.  <ul class="find" data-citedArticleID="936428" data-doi="10.1073/pnas.79.17.5337"><li><a href="http://dx.doi.org/10.1073/pnas.79.17.5337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Somatic+cell+parasitism+and+the+evolution+of+somatic+tissue+compatibility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Somatic+cell+parasitism+and+the+evolution+of+somatic+tissue+compatibility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000283-DeTomaso1" id="pone.0000283-DeTomaso1"></a>De Tomaso AW, Nyholm SV, Palmeri KJ, Ishizuka KJ, Ludington WB, et al.  (2005) Isolation and characterization of a protochordate histocompatibility locus. Nature  438: 454–459.  <ul class="find" data-citedArticleID="936436" data-doi="10.1038/nature04150"><li><a href="http://dx.doi.org/10.1038/nature04150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+a+protochordate+histocompatibility+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+a+protochordate+histocompatibility+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000283-Richman1" id="pone.0000283-Richman1"></a>Richman A (2000) Evolution of balanced genetic polymorphism. Mol Ecol  9: 1953–1963.  <ul class="find" data-citedArticleID="936516" data-doi="10.1046/j.1365-294x.2000.01125.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.01125.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+balanced+genetic+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+balanced+genetic+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000283-Glass1" id="pone.0000283-Glass1"></a>Glass NL, Jacobson DJ, Shiu PK (2000) The genetics of hyphal fusion and vegetative incompatibility in filamentous ascomycete fungi. Annu Rev Genet  34: 165–186.  <ul class="find" data-citedArticleID="936454"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+genetics+of+hyphal+fusion+and+vegetative+incompatibility+in+filamentous+ascomycete+fungi.&amp;auth=&amp;atitle=The+genetics+of+hyphal+fusion+and+vegetative+incompatibility+in+filamentous+ascomycete+fungi." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetics+of+hyphal+fusion+and+vegetative+incompatibility+in+filamentous+ascomycete+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetics+of+hyphal+fusion+and+vegetative+incompatibility+in+filamentous+ascomycete+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000283-Glass2" id="pone.0000283-Glass2"></a>Glass N, Kaneko I (2003) Fatal attraction: nonself recognition and heterokaryon incompatibility in filamentous fungi. Eukaryotic Cell  2: 1–8.  <ul class="find" data-citedArticleID="936456" data-doi="10.1128/ec.2.1.1-8.2003"><li><a href="http://dx.doi.org/10.1128/ec.2.1.1-8.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fatal+attraction%3A+nonself+recognition+and+heterokaryon+incompatibility+in+filamentous+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fatal+attraction%3A+nonself+recognition+and+heterokaryon+incompatibility+in+filamentous+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000283-Saupe1" id="pone.0000283-Saupe1"></a>Saupe SJ (2000) Molecular genetics of heterokaryon incompatibility in filamentous ascomycetes. Microbiol Mol Biol Rev  64: 489–502.  <ul class="find" data-citedArticleID="936520" data-doi="10.1128/mmbr.64.3.489-502.2000"><li><a href="http://dx.doi.org/10.1128/mmbr.64.3.489-502.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetics+of+heterokaryon+incompatibility+in+filamentous+ascomycetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetics+of+heterokaryon+incompatibility+in+filamentous+ascomycetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000283-Nuss1" id="pone.0000283-Nuss1"></a>Nuss DL (2005) Hypovirulence: mycoviruses at the fungal-plant interface. Nat Rev Microbiol  3: 632–642.  <ul class="find" data-citedArticleID="936504" data-doi="10.1038/nrmicro1206"><li><a href="http://dx.doi.org/10.1038/nrmicro1206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hypovirulence%3A+mycoviruses+at+the+fungal-plant+interface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hypovirulence%3A+mycoviruses+at+the+fungal-plant+interface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000283-Biella1" id="pone.0000283-Biella1"></a>Biella S, Smith ML, Aist JR, Cortesi P, Milgroom MG (2002) Programmed cell death correlates with virus transmission in a filamentous fungus. Proc Biol Sci  269: 2269–2276.  <ul class="find" data-citedArticleID="936420" data-doi="10.1098/rspb.2002.2148"><li><a href="http://dx.doi.org/10.1098/rspb.2002.2148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Programmed+cell+death+correlates+with+virus+transmission+in+a+filamentous+fungus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Programmed+cell+death+correlates+with+virus+transmission+in+a+filamentous+fungus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000283-Espagne1" id="pone.0000283-Espagne1"></a>Espagne E, Balhadere P, Penin ML, Barreau C, Turcq B (2002) HET-E and HET-D belong to a new subfamily of WD40 proteins involved in vegetative incompatibility specificity in the fungus Podospora anserina. Genetics  161: 71–81.  <ul class="find" data-citedArticleID="936440"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=HET-E+and+HET-D+belong+to+a+new+subfamily+of+WD40+proteins+involved+in+vegetative+incompatibility+specificity+in+the+fungus+Podospora+anserina.&amp;auth=&amp;atitle=HET-E+and+HET-D+belong+to+a+new+subfamily+of+WD40+proteins+involved+in+vegetative+incompatibility+specificity+in+the+fungus+Podospora+anserina." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HET-E+and+HET-D+belong+to+a+new+subfamily+of+WD40+proteins+involved+in+vegetative+incompatibility+specificity+in+the+fungus+Podospora+anserina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HET-E+and+HET-D+belong+to+a+new+subfamily+of+WD40+proteins+involved+in+vegetative+incompatibility+specificity+in+the+fungus+Podospora+anserina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000283-Espagne2" id="pone.0000283-Espagne2"></a>Espagne E (2000) Contribution à l'étude des systèmes non alléliques d'incompatibilité chez le champignon filamenteux Podospora anserina. Thèse, Université de Bordeaux  II:   <ul class="find" data-citedArticleID="936442"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Contribution+%26%23xe0%3B+l%27%26%23xe9%3Btude+des+syst%26%23xe8%3Bmes+non+all%26%23xe9%3Bliques+d%27incompatibilit%26%23xe9%3B+chez+le+champignon+filamenteux+Podospora+anserina.&amp;auth=&amp;atitle=Contribution+%26%23xe0%3B+l%27%26%23xe9%3Btude+des+syst%26%23xe8%3Bmes+non+all%26%23xe9%3Bliques+d%27incompatibilit%26%23xe9%3B+chez+le+champignon+filamenteux+Podospora+anserina." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contribution+%26%23xe0%3B+l%27%26%23xe9%3Btude+des+syst%26%23xe8%3Bmes+non+all%26%23xe9%3Bliques+d%27incompatibilit%26%23xe9%3B+chez+le+champignon+filamenteux+Podospora+anserina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contribution+%26%23xe0%3B+l%27%26%23xe9%3Btude+des+syst%26%23xe8%3Bmes+non+all%26%23xe9%3Bliques+d%27incompatibilit%26%23xe9%3B+chez+le+champignon+filamenteux+Podospora+anserina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000283-Espagne3" id="pone.0000283-Espagne3"></a>Espagne E, Balhadere P, Begueret J, Turcq B (1997) Reactivity in vegetative incompatibility of the HET-E protein of the fungus Podospora anserina is dependent on GTP-binding activity and a WD40 repeated domain. Mol Gen Genet  256: 620–627.  <ul class="find" data-citedArticleID="936444" data-doi="10.1007/s004380050610"><li><a href="http://dx.doi.org/10.1007/s004380050610" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reactivity+in+vegetative+incompatibility+of+the+HET-E+protein+of+the+fungus+Podospora+anserina+is+dependent+on+GTP-binding+activity+and+a+WD40+repeated+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reactivity+in+vegetative+incompatibility+of+the+HET-E+protein+of+the+fungus+Podospora+anserina+is+dependent+on+GTP-binding+activity+and+a+WD40+repeated+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000283-Saupe2" id="pone.0000283-Saupe2"></a>Saupe S, Turcq B, Begueret J (1995) A gene responsible for vegetative incompatibility in the fungus Podospora anserina encodes a protein with a GTP-binding motif and G beta homologous domain. Gene  162: 135–139.  <ul class="find" data-citedArticleID="936522" data-doi="10.1016/0378-1119(95)00272-8"><li><a href="http://dx.doi.org/10.1016/0378-1119(95)00272-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+responsible+for+vegetative+incompatibility+in+the+fungus+Podospora+anserina+encodes+a+protein+with+a+GTP-binding+motif+and+G+beta+homologous+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+responsible+for+vegetative+incompatibility+in+the+fungus+Podospora+anserina+encodes+a+protein+with+a+GTP-binding+motif+and+G+beta+homologous+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000283-Saupe3" id="pone.0000283-Saupe3"></a>Saupe S, Turcq B, Begueret J (1995) Sequence diversity and unusual variability at the het-c locus involved in vegetative incompatibility in the fungus Podospora anserina. Curr Genet  27: 466–471.  <ul class="find" data-citedArticleID="936524" data-doi="10.1007/bf00311217"><li><a href="http://dx.doi.org/10.1007/bf00311217" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+diversity+and+unusual+variability+at+the+het-c+locus+involved+in+vegetative+incompatibility+in+the+fungus+Podospora+anserina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+diversity+and+unusual+variability+at+the+het-c+locus+involved+in+vegetative+incompatibility+in+the+fungus+Podospora+anserina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000283-Bernet1" id="pone.0000283-Bernet1"></a>Bernet J (1967) [Systems of incompatibility in Podospora anserina]. C R Acad Sci Hebd Seances Acad Sci D  265: 1330–1333.  <ul class="find" data-citedArticleID="936418"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=%5BSystems+of+incompatibility+in+Podospora+anserina%5D.&amp;auth=&amp;atitle=%5BSystems+of+incompatibility+in+Podospora+anserina%5D." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BSystems+of+incompatibility+in+Podospora+anserina%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BSystems+of+incompatibility+in+Podospora+anserina%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000283-Boucherie1" id="pone.0000283-Boucherie1"></a>Boucherie H, Dupont CH, Bernet J (1981) Polypeptide synthesis during protoplasmic incompatibility in the fungus Podospora anserina. Biochim Biophys Acta  653: 18–26.  <ul class="find" data-citedArticleID="936422" data-doi="10.1016/0005-2787(81)90100-3"><li><a href="http://dx.doi.org/10.1016/0005-2787(81)90100-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polypeptide+synthesis+during+protoplasmic+incompatibility+in+the+fungus+Podospora+anserina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polypeptide+synthesis+during+protoplasmic+incompatibility+in+the+fungus+Podospora+anserina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000283-PinanLucarre1" id="pone.0000283-PinanLucarre1"></a>Pinan-Lucarre B, Paoletti M, Dementhon K, Coulary-Salin B, Clave C (2003) Autophagy is induced during cell death by incompatibility and is essential for differentiation in the filamentous fungus Podospora anserina. Mol Microbiol  47: 321–333.  <ul class="find" data-citedArticleID="936508" data-doi="10.1046/j.1365-2958.2003.03208.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03208.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autophagy+is+induced+during+cell+death+by+incompatibility+and+is+essential+for+differentiation+in+the+filamentous+fungus+Podospora+anserina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autophagy+is+induced+during+cell+death+by+incompatibility+and+is+essential+for+differentiation+in+the+filamentous+fungus+Podospora+anserina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000283-PinanLucarre2" id="pone.0000283-PinanLucarre2"></a>Pinan-Lucarre B, Balguerie A, Clave C (2005) Accelerated cell death in Podospora autophagy mutants. Eukaryot Cell  4: 1765–1774.  <ul class="find" data-citedArticleID="936510" data-doi="10.1128/ec.4.11.1765-1774.2005"><li><a href="http://dx.doi.org/10.1128/ec.4.11.1765-1774.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accelerated+cell+death+in+Podospora+autophagy+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accelerated+cell+death+in+Podospora+autophagy+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000283-Saupe4" id="pone.0000283-Saupe4"></a>Saupe S, Descamps C, Turcq B, Begueret J (1994) Inactivation of the Podospora anserina vegetative incompatibility locus het-c, whose product resembles a glycolipid transfer protein, drastically impairs ascospore production. Proc Natl Acad Sci U S A  91: 5927–5931.  <ul class="find" data-citedArticleID="936526" data-doi="10.1073/pnas.91.13.5927"><li><a href="http://dx.doi.org/10.1073/pnas.91.13.5927" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+the+Podospora+anserina+vegetative+incompatibility+locus+het-c%2C+whose+product+resembles+a+glycolipid+transfer+protein%2C+drastically+impairs+ascospore+production." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+the+Podospora+anserina+vegetative+incompatibility+locus+het-c%2C+whose+product+resembles+a+glycolipid+transfer+protein%2C+drastically+impairs+ascospore+production.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000283-Smith1" id="pone.0000283-Smith1"></a>Smith ML, Micali OC, Hubbard SP, Mir-Rashed N, Jacobson DJ, et al.  (2000) Vegetative incompatibility in the het-6 region of Neurospora crassa is mediated by two linked genes. Genetics  155: 1095–1104.  <ul class="find" data-citedArticleID="936530"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Vegetative+incompatibility+in+the+het-6+region+of+Neurospora+crassa+is+mediated+by+two+linked+genes.&amp;auth=&amp;atitle=Vegetative+incompatibility+in+the+het-6+region+of+Neurospora+crassa+is+mediated+by+two+linked+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vegetative+incompatibility+in+the+het-6+region+of+Neurospora+crassa+is+mediated+by+two+linked+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vegetative+incompatibility+in+the+het-6+region+of+Neurospora+crassa+is+mediated+by+two+linked+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000283-Glass3" id="pone.0000283-Glass3"></a>Glass NL, Dementhon K (2006) Non-self recognition and programmed cell death in filamentous fungi. Curr Opin Microbiol.   <ul class="find" data-citedArticleID="936458" data-doi="10.1016/j.mib.2006.09.001"><li><a href="http://dx.doi.org/10.1016/j.mib.2006.09.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-self+recognition+and+programmed+cell+death+in+filamentous+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-self+recognition+and+programmed+cell+death+in+filamentous+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000283-Leipe1" id="pone.0000283-Leipe1"></a>Leipe DD, Koonin EV, Aravind L (2004) STAND, a class of P-loop NTPases including animal and plant regulators of programmed cell death: multiple, complex domain architectures, unusual phyletic patterns, and evolution by horizontal gene transfer. J Mol Biol  343: 1–28.  <ul class="find" data-citedArticleID="936476" data-doi="10.1016/j.jmb.2004.08.023"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.08.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=STAND%2C+a+class+of+P-loop+NTPases+including+animal+and+plant+regulators+of+programmed+cell+death%3A+multiple%2C+complex+domain+architectures%2C+unusual+phyletic+patterns%2C+and+evolution+by+horizontal+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22STAND%2C+a+class+of+P-loop+NTPases+including+animal+and+plant+regulators+of+programmed+cell+death%3A+multiple%2C+complex+domain+architectures%2C+unusual+phyletic+patterns%2C+and+evolution+by+horizontal+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000283-Kufer1" id="pone.0000283-Kufer1"></a>Kufer TA, Fritz JH, Philpott DJ (2005) NACHT-LRR proteins (NLRs) in bacterial infection and immunity. Trends Microbiol  13: 381–388.  <ul class="find" data-citedArticleID="936470" data-doi="10.1016/j.tim.2005.06.004"><li><a href="http://dx.doi.org/10.1016/j.tim.2005.06.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NACHT-LRR+proteins+%28NLRs%29+in+bacterial+infection+and+immunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NACHT-LRR+proteins+%28NLRs%29+in+bacterial+infection+and+immunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000283-Galagan1" id="pone.0000283-Galagan1"></a>Galagan JE, Selker EU (2004) RIP: the evolutionary cost of genome defense. Trends Genet  20: 417–423.  <ul class="find" data-citedArticleID="936448" data-doi="10.1016/j.tig.2004.07.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.07.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RIP%3A+the+evolutionary+cost+of+genome+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RIP%3A+the+evolutionary+cost+of+genome+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000283-Galagan2" id="pone.0000283-Galagan2"></a>Galagan JE, Calvo SE, Borkovich KA, Selker EU, Read ND, et al.  (2003) The genome sequence of the filamentous fungus Neurospora crassa. Nature  422: 859–868.  <ul class="find" data-citedArticleID="936450" data-doi="10.1038/nature01554"><li><a href="http://dx.doi.org/10.1038/nature01554" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+the+filamentous+fungus+Neurospora+crassa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+the+filamentous+fungus+Neurospora+crassa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000283-Fedorova1" id="pone.0000283-Fedorova1"></a>Fedorova ND, Badger JH, Robson GD, Wortman JR, Nierman WC (2005) Comparative analysis of programmed cell death pathways in filamentous fungi. BMC Genomics  6: 177.  <ul class="find" data-citedArticleID="936446" data-doi="10.1186/1471-2164-6-177"><li><a href="http://dx.doi.org/10.1186/1471-2164-6-177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+programmed+cell+death+pathways+in+filamentous+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+programmed+cell+death+pathways+in+filamentous+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000283-Bouhouche1" id="pone.0000283-Bouhouche1"></a>Bouhouche K, Zickler D, Debuchy R, Arnaise S (2004) Altering a gene involved in nuclear distribution increases the repeat-induced point mutation process in the fungus Podospora anserina. Genetics  167: 151–159.  <ul class="find" data-citedArticleID="936424" data-doi="10.1534/genetics.167.1.151"><li><a href="http://dx.doi.org/10.1534/genetics.167.1.151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altering+a+gene+involved+in+nuclear+distribution+increases+the+repeat-induced+point+mutation+process+in+the+fungus+Podospora+anserina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Altering+a+gene+involved+in+nuclear+distribution+increases+the+repeat-induced+point+mutation+process+in+the+fungus+Podospora+anserina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000283-Cambareri1" id="pone.0000283-Cambareri1"></a>Cambareri EB, Jensen BC, Schabtach E, Selker EU (1989) Repeat-induced G-C to A-T mutations in Neurospora. Science  244: 1571–1575.  <ul class="find" data-citedArticleID="936430" data-doi="10.1126/science.2544994"><li><a href="http://dx.doi.org/10.1126/science.2544994" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repeat-induced+G-C+to+A-T+mutations+in+Neurospora." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repeat-induced+G-C+to+A-T+mutations+in+Neurospora.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000283-Smith2" id="pone.0000283-Smith2"></a>Smith TF, Gaitatzes C, Saxena K, Neer EJ (1999) The WD repeat: a common architecture for diverse functions. Trends Biochem Sci  24: 181–185.  <ul class="find" data-citedArticleID="936532" data-doi="10.1016/s0968-0004(99)01384-5"><li><a href="http://dx.doi.org/10.1016/s0968-0004(99)01384-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+WD+repeat%3A+a+common+architecture+for+diverse+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+WD+repeat%3A+a+common+architecture+for+diverse+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000283-Nei1" id="pone.0000283-Nei1"></a>Nei M, Rooney AP (2005) Concerted and Birth-and-Death Evolution of Multigene Families. Annu Rev Genet.   <ul class="find" data-citedArticleID="936494" data-doi="10.1146/annurev.genet.39.073003.112240"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concerted+and+Birth-and-Death+Evolution+of+Multigene+Families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concerted+and+Birth-and-Death+Evolution+of+Multigene+Families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000283-Nei2" id="pone.0000283-Nei2"></a>Nei M (1996) Phylogenetic analysis in molecular evolutionary genetics. Annu Rev Genet  30: 371–403.  <ul class="find" data-citedArticleID="936496" data-doi="10.1146/annurev.genet.30.1.371"><li><a href="http://dx.doi.org/10.1146/annurev.genet.30.1.371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+in+molecular+evolutionary+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+in+molecular+evolutionary+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000283-Nielsen1" id="pone.0000283-Nielsen1"></a>Nielsen R (2005) Molecular signatures of natural selection. Annu Rev Genet  39: 197–218.  <ul class="find" data-citedArticleID="936502" data-doi="10.1146/annurev.genet.39.073003.112420"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112420" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+signatures+of+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+signatures+of+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000283-Nei3" id="pone.0000283-Nei3"></a>Nei M (2005) Selectionism and neutralism in molecular evolution. Mol Biol Evol  22: 2318–2342.  <ul class="find" data-citedArticleID="936498" data-doi="10.1093/molbev/msi242"><li><a href="http://dx.doi.org/10.1093/molbev/msi242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selectionism+and+neutralism+in+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selectionism+and+neutralism+in+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000283-KosakovskyPond1" id="pone.0000283-KosakovskyPond1"></a>Kosakovsky Pond SL, Frost SD (2005) Not so different after all: a comparison of methods for detecting amino acid sites under selection. Mol Biol Evol  22: 1208–1222.  <ul class="find" data-citedArticleID="936468" data-doi="10.1093/molbev/msi105"><li><a href="http://dx.doi.org/10.1093/molbev/msi105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000283-Sprague1" id="pone.0000283-Sprague1"></a>Sprague ER, Redd MJ, Johnson AD, Wolberger C (2000) Structure of the C-terminal domain of Tup1, a corepressor of transcription in yeast. Embo J  19: 3016–3027.  <ul class="find" data-citedArticleID="936534" data-doi="10.1093/emboj/19.12.3016"><li><a href="http://dx.doi.org/10.1093/emboj/19.12.3016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+C-terminal+domain+of+Tup1%2C+a+corepressor+of+transcription+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+C-terminal+domain+of+Tup1%2C+a+corepressor+of+transcription+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000283-GarciaHiguera1" id="pone.0000283-GarciaHiguera1"></a>Garcia-Higuera I, Fenoglio J, Li Y, Lewis C, Panchenko MP, et al.  (1996) Folding of proteins with WD-repeats: comparison of six members of the WD-repeat superfamily to the G protein beta subunit. Biochemistry  35: 13985–13994.  <ul class="find" data-citedArticleID="936452" data-doi="10.1021/bi9612879"><li><a href="http://dx.doi.org/10.1021/bi9612879" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Folding+of+proteins+with+WD-repeats%3A+comparison+of+six+members+of+the+WD-repeat+superfamily+to+the+G+protein+beta+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Folding+of+proteins+with+WD-repeats%3A+comparison+of+six+members+of+the+WD-repeat+superfamily+to+the+G+protein+beta+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000283-Cerveny1" id="pone.0000283-Cerveny1"></a>Cerveny KL, Jensen RE (2003) The WD-repeats of Net2p interact with Dnm1p and Fis1p to regulate division of mitochondria. Mol Biol Cell  14: 4126–4139.  <ul class="find" data-citedArticleID="936432" data-doi="10.1091/mbc.e03-02-0092"><li><a href="http://dx.doi.org/10.1091/mbc.e03-02-0092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+WD-repeats+of+Net2p+interact+with+Dnm1p+and+Fis1p+to+regulate+division+of+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+WD-repeats+of+Net2p+interact+with+Dnm1p+and+Fis1p+to+regulate+division+of+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000283-Malinina1" id="pone.0000283-Malinina1"></a>Malinina L, Malakhova ML, Teplov A, Brown RE, Patel DJ (2004) Structural basis for glycosphingolipid transfer specificity. Nature  430: 1048–1053.  <ul class="find" data-citedArticleID="936482" data-doi="10.1038/nature02856"><li><a href="http://dx.doi.org/10.1038/nature02856" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+glycosphingolipid+transfer+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+glycosphingolipid+transfer+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000283-Malakhova1" id="pone.0000283-Malakhova1"></a>Malakhova ML, Malinina L, Pike HM, Kanack AT, Patel DJ, et al.  (2005) Point mutational analysis of the liganding site in human glycolipid transfer protein. Functionality of the complex. J Biol Chem  280: 26312–26320.  <ul class="find" data-citedArticleID="936480" data-doi="10.1074/jbc.m500481200"><li><a href="http://dx.doi.org/10.1074/jbc.m500481200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Point+mutational+analysis+of+the+liganding+site+in+human+glycolipid+transfer+protein.+Functionality+of+the+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Point+mutational+analysis+of+the+liganding+site+in+human+glycolipid+transfer+protein.+Functionality+of+the+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000283-Drake1" id="pone.0000283-Drake1"></a>Drake JW, Charlesworth B, Charlesworth D, Crow JF (1998) Rates of spontaneous mutation. Genetics  148: 1667–1686.  <ul class="find" data-citedArticleID="936438"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rates+of+spontaneous+mutation.&amp;auth=&amp;atitle=Rates+of+spontaneous+mutation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+spontaneous+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+spontaneous+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000283-Arjan1" id="pone.0000283-Arjan1"></a>Arjan JA, Visser M, Zeyl CW, Gerrish PJ, Blanchard JL, et al.  (1999) Diminishing returns from mutation supply rate in asexual populations. Science  283: 404–406.  <ul class="find" data-citedArticleID="936412" data-doi="10.1126/science.283.5400.404"><li><a href="http://dx.doi.org/10.1126/science.283.5400.404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diminishing+returns+from+mutation+supply+rate+in+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diminishing+returns+from+mutation+supply+rate+in+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000283-Hamann1" id="pone.0000283-Hamann1"></a>Hamann A, Feller F, Osiewacz HD (2000) The degenerate DNA transposon Pat and repeat-induced point mutation (RIP) in Podospora anserina. Mol Gen Genet  263: 1061–1069.  <ul class="find" data-citedArticleID="936462" data-doi="10.1007/s004380050035"><li><a href="http://dx.doi.org/10.1007/s004380050035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+degenerate+DNA+transposon+Pat+and+repeat-induced+point+mutation+%28RIP%29+in+Podospora+anserina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+degenerate+DNA+transposon+Pat+and+repeat-induced+point+mutation+%28RIP%29+in+Podospora+anserina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000283-Gout1" id="pone.0000283-Gout1"></a>Gout L, Fudal I, Kuhn ML, Blaise F, Eckert M, et al.  (2006) Lost in the middle of nowhere: the AvrLm1 avirulence gene of the Dothideomycete Leptosphaeria maculans. Mol Microbiol  60: 67–80.  <ul class="find" data-citedArticleID="936460" data-doi="10.1111/j.1365-2958.2006.05076.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05076.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lost+in+the+middle+of+nowhere%3A+the+AvrLm1+avirulence+gene+of+the+Dothideomycete+Leptosphaeria+maculans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lost+in+the+middle+of+nowhere%3A+the+AvrLm1+avirulence+gene+of+the+Dothideomycete+Leptosphaeria+maculans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000283-Montiel1" id="pone.0000283-Montiel1"></a>Montiel MD, Lee HA, Archer DB (2006) Evidence of RIP (repeat-induced point mutation) in transposase sequences of Aspergillus oryzae. Fungal Genet Biol  43: 439–445.  <ul class="find" data-citedArticleID="936490" data-doi="10.1016/j.fgb.2006.01.011"><li><a href="http://dx.doi.org/10.1016/j.fgb.2006.01.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+RIP+%28repeat-induced+point+mutation%29+in+transposase+sequences+of+Aspergillus+oryzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+RIP+%28repeat-induced+point+mutation%29+in+transposase+sequences+of+Aspergillus+oryzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000283-Braun1" id="pone.0000283-Braun1"></a>Braun FN, Liberles DA (2004) Repeat-modulated population genetic effects in fungal proteins. J Mol Evol  59: 97–102.  <ul class="find" data-citedArticleID="936426" data-doi="10.1007/s00239-004-2608-9"><li><a href="http://dx.doi.org/10.1007/s00239-004-2608-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repeat-modulated+population+genetic+effects+in+fungal+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repeat-modulated+population+genetic+effects+in+fungal+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000283-Paoletti1" id="pone.0000283-Paoletti1"></a>Paoletti M, Buck KW, Brasier CM (2006) Selective acquisition of novel mating type and vegetative incompatibility genes via interspecies gene transfer in the globally invading eukaryote Ophiostoma novo-ulmi. Mol Ecol  15: 249–262.  <ul class="find" data-citedArticleID="936506" data-doi="10.1111/j.1365-294x.2005.02728.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02728.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+acquisition+of+novel+mating+type+and+vegetative+incompatibility+genes+via+interspecies+gene+transfer+in+the+globally+invading+eukaryote+Ophiostoma+novo-ulmi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+acquisition+of+novel+mating+type+and+vegetative+incompatibility+genes+via+interspecies+gene+transfer+in+the+globally+invading+eukaryote+Ophiostoma+novo-ulmi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000283-Wu1" id="pone.0000283-Wu1"></a>Wu J, Saupe SJ, Glass NL (1998) Evidence for balancing selection operating at the het-c heterokaryon incompatibility locus in a group of filamentous fungi. Proc Natl Acad Sci U S A  95: 12398–12403.  <ul class="find" data-citedArticleID="936540" data-doi="10.1073/pnas.95.21.12398"><li><a href="http://dx.doi.org/10.1073/pnas.95.21.12398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+balancing+selection+operating+at+the+het-c+heterokaryon+incompatibility+locus+in+a+group+of+filamentous+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+balancing+selection+operating+at+the+het-c+heterokaryon+incompatibility+locus+in+a+group+of+filamentous+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000283-Vallender1" id="pone.0000283-Vallender1"></a>Vallender EJ, Lahn BT (2006) A primate-specific acceleration in the evolution of the caspase-dependent apoptosis pathway. Hum Mol Genet  15: 3034–3040.  <ul class="find" data-citedArticleID="936538" data-doi="10.1093/hmg/ddl245"><li><a href="http://dx.doi.org/10.1093/hmg/ddl245" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+primate-specific+acceleration+in+the+evolution+of+the+caspase-dependent+apoptosis+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+primate-specific+acceleration+in+the+evolution+of+the+caspase-dependent+apoptosis+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000283-Shiu1" id="pone.0000283-Shiu1"></a>Shiu PK, Glass NL (1999) Molecular characterization of tol, a mediator of mating-type-associated vegetative incompatibility in Neurospora crassa. Genetics  151: 545–555.  <ul class="find" data-citedArticleID="936528"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+characterization+of+tol%2C+a+mediator+of+mating-type-associated+vegetative+incompatibility+in+Neurospora+crassa.&amp;auth=&amp;atitle=Molecular+characterization+of+tol%2C+a+mediator+of+mating-type-associated+vegetative+incompatibility+in+Neurospora+crassa." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+tol%2C+a+mediator+of+mating-type-associated+vegetative+incompatibility+in+Neurospora+crassa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+tol%2C+a+mediator+of+mating-type-associated+vegetative+incompatibility+in+Neurospora+crassa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000283-Kaneko1" id="pone.0000283-Kaneko1"></a>Kaneko I, Dementhon K, Xiang Q, Glass NL (2006) Nonallelic interactions between het-c and a polymorphic locus, pin-c, are essential for nonself recognition and programmed cell death in Neurospora crassa. Genetics  172: 1545–1555.  <ul class="find" data-citedArticleID="936464" data-doi="10.1534/genetics.105.051490"><li><a href="http://dx.doi.org/10.1534/genetics.105.051490" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonallelic+interactions+between+het-c+and+a+polymorphic+locus%2C+pin-c%2C+are+essential+for+nonself+recognition+and+programmed+cell+death+in+Neurospora+crassa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonallelic+interactions+between+het-c+and+a+polymorphic+locus%2C+pin-c%2C+are+essential+for+nonself+recognition+and+programmed+cell+death+in+Neurospora+crassa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000283-Labarere1" id="pone.0000283-Labarere1"></a>Labarere J (1973) [Properties of an incompatibility system in Podospora anserina fungus and value of this system for the study of incompatibility]. C R Acad Sci Hebd Seances Acad Sci D  276: 1301–1304.  <ul class="find" data-citedArticleID="936474"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=%5BProperties+of+an+incompatibility+system+in+Podospora+anserina+fungus+and+value+of+this+system+for+the+study+of+incompatibility%5D.&amp;auth=&amp;atitle=%5BProperties+of+an+incompatibility+system+in+Podospora+anserina+fungus+and+value+of+this+system+for+the+study+of+incompatibility%5D." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BProperties+of+an+incompatibility+system+in+Podospora+anserina+fungus+and+value+of+this+system+for+the+study+of+incompatibility%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BProperties+of+an+incompatibility+system+in+Podospora+anserina+fungus+and+value+of+this+system+for+the+study+of+incompatibility%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000283-Begueret1" id="pone.0000283-Begueret1"></a>Begueret J (1973) L'incompatibilite cellulaire chez le champignon Podospora anserina: etude physiologique de la reaction. These de l'Universite de Bordeaux  II:   <ul class="find" data-citedArticleID="936416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=L%27incompatibilite+cellulaire+chez+le+champignon+Podospora+anserina%3A+etude+physiologique+de+la+reaction.&amp;auth=&amp;atitle=L%27incompatibilite+cellulaire+chez+le+champignon+Podospora+anserina%3A+etude+physiologique+de+la+reaction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=L%27incompatibilite+cellulaire+chez+le+champignon+Podospora+anserina%3A+etude+physiologique+de+la+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22L%27incompatibilite+cellulaire+chez+le+champignon+Podospora+anserina%3A+etude+physiologique+de+la+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000283-Maizels1" id="pone.0000283-Maizels1"></a>Maizels N (2005) Immunoglobulin gene diversification. Annu Rev Genet  39: 23–46.  <ul class="find" data-citedArticleID="936478" data-doi="10.1146/annurev.genet.39.073003.110544"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.110544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunoglobulin+gene+diversification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunoglobulin+gene+diversification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000283-Barry1" id="pone.0000283-Barry1"></a>Barry JD, McCulloch R (2001) Antigenic variation in trypanosomes: enhanced phenotypic variation in a eukaryotic parasite. Adv Parasitol  49: 1–70.  <ul class="find" data-citedArticleID="936414" data-doi="10.1016/s0065-308x(01)49037-3"><li><a href="http://dx.doi.org/10.1016/s0065-308x(01)49037-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antigenic+variation+in+trypanosomes%3A+enhanced+phenotypic+variation+in+a+eukaryotic+parasite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antigenic+variation+in+trypanosomes%3A+enhanced+phenotypic+variation+in+a+eukaryotic+parasite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000283-MondragonPalomino1" id="pone.0000283-MondragonPalomino1"></a>Mondragon-Palomino M, Gaut BS (2005) Gene conversion and the evolution of three leucine-rich repeat gene families in Arabidopsis thaliana. Mol Biol Evol  22: 2444–2456.  <ul class="find" data-citedArticleID="936488" data-doi="10.1093/molbev/msi241"><li><a href="http://dx.doi.org/10.1093/molbev/msi241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+conversion+and+the+evolution+of+three+leucine-rich+repeat+gene+families+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+conversion+and+the+evolution+of+three+leucine-rich+repeat+gene+families+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000283-Metzgar1" id="pone.0000283-Metzgar1"></a>Metzgar D, Wills C (2000) Evidence for the adaptive evolution of mutation rates. Cell  101: 581–584.  <ul class="find" data-citedArticleID="936486" data-doi="10.1016/s0092-8674(00)80869-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80869-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+the+adaptive+evolution+of+mutation+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+the+adaptive+evolution+of+mutation+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000283-Pond1" id="pone.0000283-Pond1"></a>Pond SL, Frost SD, Muse SV (2005) HyPhy: hypothesis testing using phylogenies. Bioinformatics  21: 676–679.  <ul class="find" data-citedArticleID="936512" data-doi="10.1093/bioinformatics/bti079"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HyPhy%3A+hypothesis+testing+using+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HyPhy%3A+hypothesis+testing+using+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000283-Suzuki1" id="pone.0000283-Suzuki1"></a>Suzuki Y, Gojobori T (1999) A method for detecting positive selection at single amino acid sites. Mol Biol Evol  16: 1315–1328.  <ul class="find" data-citedArticleID="936536" data-doi="10.1093/oxfordjournals.molbev.a026042"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+detecting+positive+selection+at+single+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+detecting+positive+selection+at+single+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000283-Nei4" id="pone.0000283-Nei4"></a>Nei M, Gojobori T (1986) Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions. Mol Biol Evol  3: 418–426.  <ul class="find" data-citedArticleID="936500"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.&amp;auth=&amp;atitle=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000283-Kumar1" id="pone.0000283-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="936472" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000283-Saitou1" id="pone.0000283-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="936518"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000283-Kimura1" id="pone.0000283-Kimura1"></a>Kimura M (1980) A simple method for estimating evolutionary rate of base substitutions through comparative studies of nucleotide sequences. Journal of Molecular Evolution  16: 111–120.  <ul class="find" data-citedArticleID="936466" data-doi="10.1007/bf01731581"><li><a href="http://dx.doi.org/10.1007/bf01731581" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+method+for+estimating+evolutionary+rate+of+base+substitutions+through+comparative+studies+of+nucleotide+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+method+for+estimating+evolutionary+rate+of+base+substitutions+through+comparative+studies+of+nucleotide+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000283-Combet1" id="pone.0000283-Combet1"></a>Combet C, Jambon M, Deleage G, Geourjon C (2002) Geno3D: automatic comparative molecular modelling of protein. Bioinformatics  18: 213–214.  <ul class="find" data-citedArticleID="936434" data-doi="10.1093/bioinformatics/18.1.213"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.1.213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geno3D%3A+automatic+comparative+molecular+modelling+of+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geno3D%3A+automatic+comparative+molecular+modelling+of+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.XML" value="137768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.PDF" value="306057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g001.PNG_L" value="258437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g001.PNG_M" value="116499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g001.PNG_S" value="4175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g001.TIF" value="354966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g001.PNG_I" value="41681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t001.PNG_L" value="222585"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t001.PNG_M" value="53495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t001.PNG_S" value="2997"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t001.TIF" value="926928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t001.PNG_I" value="22374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g002.PNG_L" value="1102770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g002.PNG_M" value="67100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g002.PNG_S" value="8347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g002.TIF" value="1845376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g002.PNG_I" value="99651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t002.PNG_L" value="100656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t002.PNG_M" value="98531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t002.PNG_S" value="5298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t002.TIF" value="510664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.t002.PNG_I" value="45409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g003.PNG_L" value="211397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g003.PNG_M" value="96673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g003.PNG_S" value="3817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g003.TIF" value="433456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g003.PNG_I" value="36364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g004.PNG_L" value="795181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g004.PNG_M" value="132253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g004.PNG_S" value="4139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g004.TIF" value="931114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.g004.PNG_I" value="47884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s001.PDF" value="13061"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s002.PDF" value="21794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s003.PDF" value="24474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s004.PDF" value="27150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s005.PDF" value="16193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s006.PDF" value="16587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s007.PDF" value="17425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s008.PDF" value="51379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s009.PDF" value="19063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000283.s010.PDF" value="17767"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000283&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000283" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000283&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000283&volume=&issue=&title=Genesis of a Fungal Non-Self Recognition Repertoire&author_name=Mathieu%20Paoletti%2C%20Sven%20J.%20Saupe%2C%20Corinne%20Clav%C3%A9&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000283" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000283" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000283" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000283&amp;t=Genesis%20of%20a%20Fungal%20Non-Self%20Recognition%20Repertoire" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000283&title=Genesis%20of%20a%20Fungal%20Non-Self%20Recognition%20Repertoire&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000283&amp;title=Genesis%20of%20a%20Fungal%20Non-Self%20Recognition%20Repertoire" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000283" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000283&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genesis of a Fungal Non-Self Recognition Repertoire';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000283';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genesis%20of%20a%20Fungal%20Non-Self%20Recognition%20Repertoire http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000283" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000283" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Concerted+evolution%22" title="Search for articles in the subject area:'Concerted evolution'"><div class="flagText">Concerted evolution</div></a>
              <div data-categoryid="25351" data-articleid="23404"
                   data-categoryname="Concerted evolution"
                   class="flagImage" title="Flag 'Concerted evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="23404"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23404"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23404"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="23404"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23404"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="23404"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pseudogenes%22" title="Search for articles in the subject area:'Pseudogenes'"><div class="flagText">Pseudogenes</div></a>
              <div data-categoryid="33795" data-articleid="23404"
                   data-categoryname="Pseudogenes"
                   class="flagImage" title="Flag 'Pseudogenes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7947'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8074'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8314&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
