

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Covert Genetic Selections to Optimize Phenotypes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001200"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001200" />

    <meta name="citation_title" content="Covert Genetic Selections to Optimize Phenotypes"/>
    <meta itemprop="name" content="Covert Genetic Selections to Optimize Phenotypes"/>

      <meta name="citation_author" content="Di Wu"/>
            <meta name="citation_author_institution" content="Monash Institute of Medical Research, Monash University, Monash Medical Centre, Melbourne, Victoria, Australia"/>
      <meta name="citation_author" content="Elizabeth Townsley"/>
            <meta name="citation_author_institution" content="Department of Pathology, Cell Biology Program, Case Western Reserve University School of Medicine, Cleveland, Ohio, United States of America"/>
      <meta name="citation_author" content="Alan Michael Tartakoff"/>
            <meta name="citation_author_institution" content="Department of Pathology, Cell Biology Program, Case Western Reserve University School of Medicine, Cleveland, Ohio, United States of America"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001200.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1200"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Identification of a novel serine/threonine kinase and a novel 15-kD protein as potential mediators of the gamma interferon-induced cell death.; citation_author=LP Deiss; citation_author=E Feinstein; citation_author=H Berissi; citation_author=O Cohen; citation_author=A Kimchi; citation_journal_title=Genes Dev; citation_volume=9; citation_number=1; citation_pages=15-30; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A lentiviral RNAi library for human and mouse genes applied to an arrayed viral high-content screen.; citation_author=J Moffat; citation_author=DA Grueneberg; citation_author=X Yang; citation_author=SY Kim; citation_author=AM Kloepfer; citation_journal_title=Cell; citation_volume=124; citation_number=2; citation_pages=1283-1298; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Use of genetic suppressor elements to dissect distinct biological effects of separate p53 domains.; citation_author=VS Ossovskaya; citation_author=IA Mazo; citation_author=MV Chernov; citation_author=OB Chernova; citation_author=Z Strezoska; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=3; citation_pages=10309-10314; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genetic selection of growth-inhibitory sequences in mammalian cells.; citation_author=DG Pestov; citation_author=LF Lau; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=4; citation_pages=12549-12553; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Selection-subtraction approach (SSA): a universal genetic screening technique that enables negative selection.; citation_author=AD Singhi; citation_author=RV Kondratov; citation_author=N Neznanov; citation_author=MV Chernov; citation_author=AV Gudkov; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=5; citation_pages=9327-9332; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chemogenomic profiling: identifying the functional interactions of small molecules in yeast.; citation_author=G Giaever; citation_author=P Flaherty; citation_author=J Kumm; citation_author=M Proctor; citation_author=C Nislow; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=6; citation_pages=793-798; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integration of chemical-genetic and genetic interaction data links bioactive compounds to cellular target pathways.; citation_author=AB Parsons; citation_author=RL Brost; citation_author=H Ding; citation_author=Z Li; citation_author=C Zhang; citation_journal_title=Nat Biotechnol; citation_volume=22; citation_number=7; citation_pages=62-69; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mapping pathways and phenotypes by systematic gene overexpression.; citation_author=R Sopko; citation_author=D Huang; citation_author=N Preston; citation_author=G Chua; citation_author=B Papp; citation_journal_title=Mol Cell; citation_volume=21; citation_number=8; citation_pages=319-330; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global analysis of protein activities using proteome chips.; citation_author=H Zhu; citation_author=M Bilgin; citation_author=R Bangham; citation_author=D Hall; citation_author=A Casamayor; citation_journal_title=Science; citation_volume=293; citation_number=9; citation_pages=2101-2105; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A microarray-based protocol for monitoring the growth of yeast overexpression strains.; citation_author=RA Butcher; citation_author=SL Schreiber; citation_journal_title=Nat Protoc; citation_volume=1; citation_number=10; citation_pages=569-576; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A novel plasmid-based microarray screen identifies suppressors of rrp6Delta in Saccharomyces cerevisiae.; citation_author=K Abruzzi; citation_author=S Denome; citation_author=JR Olsen; citation_author=J Assenholt; citation_author=LL Haaning; citation_journal_title=Mol Cell Biol; citation_volume=27; citation_number=11; citation_pages=1044-1055; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Biochemical and genetic analysis of the yeast proteome with a movable ORF collection.; citation_author=DM Gelperin; citation_author=MA White; citation_author=ML Wilkinson; citation_author=Y Kon; citation_author=LA Kung; citation_journal_title=Genes Dev; citation_volume=19; citation_number=12; citation_pages=2816-2826; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=eIF4E–from translation to transformation.; citation_author=Y Mamane; citation_author=E Petroulakis; citation_author=L Rong; citation_author=K Yoshida; citation_author=LW Ler; citation_journal_title=Oncogene; citation_volume=23; citation_number=13; citation_pages=3172-3179; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Initiator tRNA binding by e/aIF5B, the eukaryotic/archaeal homologue of bacterial initiation factor IF2.; citation_author=L Guillon; citation_author=E Schmitt; citation_author=S Blanquet; citation_author=Y Mechulam; citation_journal_title=Biochemistry; citation_volume=44; citation_number=14; citation_pages=15594-15601; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Initial characterization of the nascent polypeptide-associated complex in yeast.; citation_author=B Reimann; citation_author=J Bradsher; citation_author=J Franke; citation_author=E Hartmann; citation_author=M Wiedmann; citation_journal_title=Yeast; citation_volume=15; citation_number=15; citation_pages=397-407; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Missense translation errors in Saccharomyces cerevisiae.; citation_author=I Stansfield; citation_author=KM Jones; citation_author=P Herbert; citation_author=A Lewendon; citation_author=WV Shaw; citation_journal_title=J Mol Biol; citation_volume=282; citation_number=16; citation_pages=13-24; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A large nucleolar U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis.; citation_author=F Dragon; citation_author=JE Gallagher; citation_author=PA Compagnone-Post; citation_author=BM Mitchell; citation_author=KA Porwancher; citation_journal_title=Nature; citation_volume=417; citation_number=17; citation_pages=967-970; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multiple regions of yeast ribosomal protein L1 are important for its interaction with 5 S rRNA and assembly into ribosomes.; citation_author=M Deshmukh; citation_author=J Stark; citation_author=LC Yeh; citation_author=JC Lee; citation_author=JL Woolford; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=18; citation_pages=30148-30156; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Lsm proteins and RNA processing.; citation_author=JD Beggs; citation_journal_title=Biochem Soc Trans; citation_volume=33; citation_number=19; citation_pages=433-438; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Roles of N-linked glycans in the endoplasmic reticulum.; citation_author=A Helenius; citation_author=M Aebi; citation_journal_title=Annu Rev Biochem; citation_volume=73; citation_number=20; citation_pages=1019-1049; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cell wall 1,6-beta-glucan synthesis in Saccharomyces cerevisiae depends on ER glucosidases I and II, and the molecular chaperone BiP/Kar2p.; citation_author=JF Simons; citation_author=M Ebersold; citation_author=A Helenius; citation_journal_title=Embo J; citation_volume=17; citation_number=21; citation_pages=396-405; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Dependence of Ypt1 and Sec4 membrane attachment on Bet2.; citation_author=G Rossi; citation_author=JA Yu; citation_author=AP Newman; citation_author=S Ferro-Novick; citation_journal_title=Nature; citation_volume=351; citation_number=22; citation_pages=158-161; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evidence for overlapping and distinct functions in protein transport of coat protein Sec24p family members.; citation_author=R Peng; citation_author=A De Antoni; citation_author=D Gallwitz; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=23; citation_pages=11521-11528; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The pleckstrin homology domain proteins Slm1 and Slm2 are required for actin cytoskeleton organization in yeast and bind phosphatidylinositol-4,5-bisphosphate and TORC2.; citation_author=M Fadri; citation_author=A Daquinag; citation_author=S Wang; citation_author=T Xue; citation_author=J Kunz; citation_journal_title=Mol Biol Cell; citation_volume=16; citation_number=24; citation_pages=1883-1900; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification and functional analysis of the essential and regulatory light chains of the only type II myosin Myo1p in Saccharomyces cerevisiae.; citation_author=J Luo; citation_author=EA Vallen; citation_author=C Dravis; citation_author=SE Tcheperegine; citation_author=B Drees; citation_journal_title=J Cell Biol; citation_volume=165; citation_number=25; citation_pages=843-855; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Screening and identification of yeast sequences that cause growth inhibition when overexpressed.; citation_author=R Akada; citation_author=J Yamamoto; citation_author=I Yamashita; citation_journal_title=Mol Gen Genet; citation_volume=254; citation_number=26; citation_pages=267-274; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Large-scale exploration of growth inhibition caused by overexpression of genomic fragments in Saccharomyces cerevisiae.; citation_author=J Boyer; citation_author=G Badis; citation_author=C Fairhead; citation_author=E Talla; citation_author=F Hantraye; citation_journal_title=Genome Biol; citation_volume=5; citation_number=27; citation_pages=R72; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An efficient method to isolate yeast genes causing overexpression-mediated growth arrest.; citation_author=C Espinet; citation_author=MA de la Torre; citation_author=M Aldea; citation_author=E Herrero; citation_journal_title=Yeast; citation_volume=11; citation_number=28; citation_pages=25-32; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Construction of a GAL1-regulated yeast cDNA expression library and its application to the identification of genes whose overexpression causes lethality in yeast.; citation_author=H Liu; citation_author=J Krizek; citation_author=A Bretscher; citation_journal_title=Genetics; citation_volume=132; citation_number=29; citation_pages=665-673; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Dominant genetics using a yeast genomic library under the control of a strong inducible promoter.; citation_author=SW Ramer; citation_author=SJ Elledge; citation_author=RW Davis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=30; citation_pages=11589-11593; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A large-scale overexpression screen in Saccharomyces cerevisiae identifies previously uncharacterized cell cycle genes.; citation_author=LF Stevenson; citation_author=BK Kennedy; citation_author=E Harlow; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=31; citation_pages=3946-3951; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Saccharomyces cerevisiae colony growth and ageing: biphasic growth accompanied by changes in gene expression.; citation_author=JR Meunier; citation_author=M Choder; citation_journal_title=Yeast; citation_volume=15; citation_number=32; citation_pages=1159-1169; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Dosage sensitivity and the evolution of gene families in yeast.; citation_author=B Papp; citation_author=C Pal; citation_author=LD Hurst; citation_journal_title=Nature; citation_volume=424; citation_number=33; citation_pages=194-197; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Temperature Dependence of the Spontaneous Mutation Rate to Respiration Deficiency in Saccharomyces.; citation_author=M Ogur; citation_author=S Ogur; citation_author=R St John; citation_journal_title=Genetics; citation_volume=45; citation_number=34; citation_pages=189-194; citation_date=1960; " />
      <meta name="citation_reference" content="citation_title=A comparison of prediction accurary, complexity, and training time of thirty-three old and new classification algorithms.; citation_author=T Lim; citation_author=W Loh; citation_author=Y Shih; citation_journal_title=Machine Learning; citation_volume=40; citation_number=35; citation_pages=203-228; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mutations that are synthetically lethal with a gas1Delta allele cause defects in the cell wall of Saccharomyces cerevisiae.; citation_author=N Tomishige; citation_author=Y Noda; citation_author=H Adachi; citation_author=H Shimoi; citation_author=A Takatsuki; citation_journal_title=Mol Genet Genomics; citation_volume=269; citation_number=36; citation_pages=562-573; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Vectors for the inducible overexpression of glutathione S-transferase fusion proteins in yeast.; citation_author=DA Mitchell; citation_author=TK Marshall; citation_author=RJ Deschenes; citation_journal_title=Yeast; citation_volume=9; citation_number=37; citation_pages=715-722; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Preparation of high molecular weight RNA.; citation_author=K Kohrer; citation_author=H Domdey; citation_journal_title=Methods Enzymol; citation_volume=194; citation_number=38; citation_pages=398-405; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.; citation_author=BM Bolstad; citation_author=RA Irizarry; citation_author=M Astrand; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=39; citation_pages=185-193; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Statistical analysis of gene expression microarray data.; citation_author=TP Speed; citation_number=40; citation_date=2003; citation_publisher=Chapman & Hall/CRC; " />
      <meta name="citation_reference" content="citation_title=limmaGUI: a graphical user interface for linear modeling of microarray data.; citation_author=JM Wettenhall; citation_author=GK Smyth; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=41; citation_pages=3705-3706; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Green fluorescent protein as a marker for gene expression and subcellular localization in budding yeast.; citation_author=RK Niedenthal; citation_author=L Riles; citation_author=M Johnston; citation_author=JH Hegemann; citation_journal_title=Yeast; citation_volume=12; citation_number=42; citation_pages=773-786; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001200" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Covert Genetic Selections to Optimize Phenotypes"/>
    <meta name="twitter:description" content="In many high complexity systems (cells, organisms, institutions, societies, economies, etc.), it is unclear which components should be regulated to affect overall performance. To identify and prioritize molecular targets which impact cellular phenotypes, we have developed a selection procedure (&ldquo;SPI&rdquo;&ndash;single promoting/inhibiting target identification) which monitors the abundance of ectopic cDNAs. We have used this approach to identify growth regulators. For this purpose, complex pools of S. cerevisiae cDNA transformants were established and we quantitated the evolution of the spectrum of cDNAs which was initially present. These data emphasized the importance of translation initiation and ER-Golgi traffic for growth. SPI provides functional insight into the stability of cellular phenotypes under circumstances in which established genetic approaches cannot be implemented. It provides a functional &ldquo;synthetic genetic signature&rdquo; for each state of the cell (i.e. genotype and environment) by surveying complex genetic libraries, and does not require specialized arrays of cDNAs/shRNAs, deletion strains, direct assessment of clonal growth or even a conditional phenotype. Moreover, it establishes a hierarchy of importance of those targets which can contribute, either positively or negatively, to modify the prevailing phenotype. Extensions of these proof-of-principle experiments to other cell types should provide a novel and powerful approach to analyze multiple aspects of the basic biology of yeast and animal cells as well as clinically-relevant issues."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001200.g004"/>

  <meta property="og:title" content="Covert Genetic Selections to Optimize Phenotypes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001200" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2248'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9190'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2755&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001200">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001200" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001200&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Covert Genetic Selections to Optimize Phenotypes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Di Wu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Monash Institute of Medical Research, Monash University, Monash Medical Centre, Melbourne, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elizabeth Townsley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Cell Biology Program, Case Western Reserve University School of Medicine, Cleveland, Ohio, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alan Michael Tartakoff
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:amt10@case.edu">amt10@case.edu</a></p>

                <p>Affiliation:
                  Department of Pathology, Cell Biology Program, Case Western Reserve University School of Medicine, Cleveland, Ohio, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001200</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001200" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001200" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001200" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001200" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001200-g001" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-g002" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-t001" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-t002" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-g003" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-t003" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-t004" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-g004" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-t005" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-t006" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001200-t007" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001200">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001200" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>In many high complexity systems (cells, organisms, institutions, societies, economies, etc.), it is unclear which components should be regulated to affect overall performance. To identify and prioritize molecular targets which impact cellular phenotypes, we have developed a selection procedure (“SPI”–<span class="underline">s</span>ingle <span class="underline">p</span>romoting/<span class="underline">i</span>nhibiting target identification) which monitors the abundance of ectopic cDNAs. We have used this approach to identify growth regulators. For this purpose, complex pools of <em>S. cerevisiae</em> cDNA transformants were established and we quantitated the evolution of the spectrum of cDNAs which was initially present. These data emphasized the importance of translation initiation and ER-Golgi traffic for growth. SPI provides functional insight into the stability of cellular phenotypes under circumstances in which established genetic approaches cannot be implemented. It provides a functional “synthetic genetic signature” for each state of the cell (i.e. genotype and environment) by surveying complex genetic libraries, and does not require specialized arrays of cDNAs/shRNAs, deletion strains, direct assessment of clonal growth or even a conditional phenotype. Moreover, it establishes a hierarchy of importance of those targets which can contribute, either positively or negatively, to modify the prevailing phenotype. Extensions of these proof-of-principle experiments to other cell types should provide a novel and powerful approach to analyze multiple aspects of the basic biology of yeast and animal cells as well as clinically-relevant issues.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wu D, Townsley E, Tartakoff AM (2007) Covert Genetic Selections to Optimize Phenotypes. PLoS ONE 2(11):
          e1200.
            doi:10.1371/journal.pone.0001200</p><p><strong>Academic Editor: </strong>Alfred Lewin, University of Florida, United States of America</p><p><strong>Received:</strong> August 20, 2007; <strong>Accepted:</strong> October 8, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Wu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NSF MCB04311496; NIH P30CA43703, Case Western Presidential Research Initiative Grant. None of the above played any active role in the design, review, or approval of this manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Cell growth, migration, and ability to resist stress and infection depend on many factors which have been the subject of focused investigations. Further elucidation of these known determinants is of intrinsic interest and could lead to the design of corresponding molecular therapies. Nevertheless, when multiple components are involved–only some of which are known-it generally is unclear which should be targeted in order to have maximal effects on cellular performance. This optimization task becomes especially daunting when the goal is to manipulate performance–not with cells in culture–but in the full complexity of the animal. Moreover, the task of identifying growth inhibitory factors from high complexity cDNA or shRNA libraries is problematic since these are the components which disappear and therefore can be identified only by implementation of specialized strategies, e.g. <a href="#pone.0001200-Deiss1">[1]</a>, <a href="#pone.0001200-Moffat1">[2]</a>, <a href="#pone.0001200-Ossovskaya1">[3]</a>, <a href="#pone.0001200-Pestov1">[4]</a>, <a href="#pone.0001200-Singhi1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The search procedure described below (SPI) starts with a complex library and selects–in one step and without requiring assays which depend on clonal growth-single cDNAs which can have positive or negative impact on growth or on other complex phenotypes. It thus identifies groups of “contributory” cDNAs, and establishes an approximate hierarchy of their importance. Moreover, it makes possible the identification of relevant genes without requiring that they be able to overtly correct a given phenotype. These features of SPI distinguish it from most classical cloning strategies.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To exemplify this approach–which could be extended to shRNAs and to animal cells using viral libraries at low multiplicity-we have used a bank of yeast transformants which differ from each other with regard to single ectopic cDNAs. We have then quantitated the abundance of each cDNA/transformant as the pool of cells grows in liquid culture (<a href="#pone-0001200-g001">Figure 1A</a>). The central premise is that cDNAs which promote growth or survival will cause the corresponding cells and their cDNAs to become more abundant, while those which impair growth will have inverse effects. The selection is “covert,” since the critical data pertain to relative abundance of cDNAs over time rather than to gross alteration of phenotype.</p>
<div class="figure" id="pone-0001200-g001"><div class="img"><a name="pone-0001200-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Overview A: Covert Selection Strategy Each color symbolizes the presence of a distinct ectopic cDNA.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Significant changes occur when the cells are grown. cDNAs which sensitize cells to the culture conditions are expected to become depleted, while those which promote growth are expected to become enriched. B: cDNA Quantitation Upper: A typical yeast plasmid for cDNA expression, indicating the flanking targets (red) for PCR amplification. Lower: Comparison of transcriptional profiling to SPI. Both procedures generate biotinylated cRNAs using T7 RNA polymerase. In SPI, 30 cycles with the reverse primer are designed to yield linear single-stranded products, which are converted into a double-stranded copy in a single final step upon addition of the forward primer. C: GC Content of Subsets of cDNAs Having determined which of 933 cDNAs are detected on the microarrays, we subdivide the group into true positives, false positives, true negatives and false negatives. As shown, each group has nearly the same GC content. False negatives therefore are not due to difficulties in copying sequences of high GC content. D: Proportionality of Readout Two groups of 1000 strains were mixed in different proportions and then analyzed. The mean signal ratio between the groups is a near linear function of the relative DNA input, with a Pearson Correlation Coefficient of 0.986. The rectangular insert at the bottom of the figure represents the strategy of mixing different proportions of two pools of transformants.</p>
<span>doi:10.1371/journal.pone.0001200.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>By identifying the universe of enriched and depleted cDNAs which are characteristic of a given cell, SPI establishes a signature of positive and negative synthetic genetic relations, which is the equivalent of what a mathematician would refer to as a “sensitivity analysis.” Such information will allow inroads into the understanding of seemingly asymptomatic knockout cells. Moreover, adaptations of SPI should provide novel access to optimization in the context of malignant growth, cell migration and susceptibility to infection.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>SPI can be effective without requiring panels of deletion strains, “bar-coded” strains or plasmids, or further design of novel expression systems, e.g. <a href="#pone.0001200-Giaever1">[6]</a>, <a href="#pone.0001200-Parsons1">[7]</a>. Since it is also likely to be able to make use of conventional cDNA libraries, it should be able to encompass the full diversity of splice variants of transcripts, e.g. for investigation of animal cells. The present study–in addition to identifying cDNAs which govern growth of yeast-provides a proof of principle for such undertakings.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Preliminaries</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The pool of yeast strains which we have studied carries a defined synthetic library of 5885 plasmids in which each cDNA is under control of a galactose-inducible promoter <a href="#pone.0001200-Sopko1">[8]</a>, <a href="#pone.0001200-Zhu1">[9]</a>. The availability of these constructs made it possible to generate an approximately uniform mixture of corresponding transformants without induction, and then to test the impact of their expression when galactose was added. To develop procedures for plasmid recovery, copying of cDNA inserts, and probing of microarrays, we have first worked with small subsets of transformants.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The cDNA inserts are all present in a constant context in the library plasmids. To copy them without introducing the length and abundance bias which is characteristic of conventional PCR, we developed an efficient extraction procedure (<a href="#pone.0001200.s001">Figure S1A</a>) and a linear PCR procedure (“ds-Linear PCR”) (<a href="#pone-0001200-g001">Figure 1B</a>, <a href="#pone.0001200.s001">S1B</a>, <a href="#pone.0001200.s002">S2</a>) which in a final step generates double-stranded products. The rest of the procedure flows directly into the established chemistry, microarray probing with biotinylated cRNAs, and analytic algorithms developed for transcriptional profiling.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>By processing a pooled subset of 933 defined transformants (before growth), we were able to tabulate the number of loci for which microarray signals were both expected and were detected (true positives), loci for which signals were expected but were not detected (false negatives), loci for which signals were not expected but were detected (false positives), and loci for which signals were not expected and were not detected (true negatives). In this situation, there were 94% real positives and 6% false negatives. We tabulated the GC content of cDNAs in each in these four categories and observe that false negatives were not enriched in cDNAs of high CG content, as might have been expected if they were difficult to copy-<a href="#pone-0001200-g001">Figure 1C</a>–and there was only modest depletion of longer cDNAs.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>When DNA was recovered from two distinct pools of ~1000 defined transformants (before growth) and different proportions were mixed and processed, the ratio of the mean signal values for each pool was nearly proportional to its relative abundance over the range examined–<a href="#pone-0001200-g001">Figure 1D</a>. In biological experiments, a two-fold change in relative signal intensity therefore corresponds to a comparable relative change in cDNA input.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>We have inquired whether the pool of 5885 transformants retains its initial cDNA spectrum (i.e. the distribution of fluorescent intensities among all cDNAs) in liquid culture in selective medium under several conditions. Those which change to the greatest extent–the tails of the distributions–are referred to as “SPI Extremes.”</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>When quadruplicate cultures were maintained for 30 generations at 30°C without induction (i.e. in glucose medium), there was little change of relative abundance of most cDNAs; however, some SPI Extremes could already be identified, with negative changes being obvious. Increases appear very modest (<a href="#pone-0001200-g002">Figure 2A</a>), in part because the algorithm which was used for these calculations places upper (and lower) limits on the numerical estimates. <a href="#pone-0001200-g002">Figures 2B and 2C</a> show that the spectrum which persisted at 37°C in glucose medium was nearly identical to that which persisted at 30°C in glucose medium. (Full data sets for all cDNAs are provided in <a href="#pone.0001200.s006">Table S1</a>, <a href="#pone.0001200.s007">S2</a>, <a href="#pone.0001200.s008">S3</a>, <a href="#pone.0001200.s009">S4</a>, <a href="#pone.0001200.s010">S5</a> and <a href="#pone.0001200.s011">S6</a>.)</p>
<div class="figure" id="pone-0001200-g002"><div class="img"><a name="pone-0001200-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Differential cDNA Enrichment and Depletion (Fold Change) Upon Growth Replicate data sets were used to calculate fold-enrichment for six comparisons (see <a href="#s4">Methods</a>).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>In each case, the horizontal axis (gene index) displays all cDNAs in order of their relative fold change. A: Cells grown at 30°C in glucose vs the initial sample. B: Cells grown at 37°C in glucose vs the initial sample. C: Fold-changes (B) divided by fold-changes (A). D: Cells grown at 30°C in galactose vs 30°C in glucose. E: Cells grown at 37°C in galactose vs 37°C in glucose. F: Fold-changes (E) divided by fold-changes (D).</p>
<span>doi:10.1371/journal.pone.0001200.g002</span></div>

<h4>Analysis of Growth</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To identify cDNAs which affect growth upon deliberate overexpression, we have cultured samples of the full mixture of 5885 transformants in galactose medium for 30 generations at 30°C and compared their cDNA spectra to that of cells cultured in glucose medium-<a href="#pone-0001200-g002">Figure 2D</a>. Small numbers of cDNAs showed distinct differential enrichment or depletion (<a href="#pone-0001200-t001">Table 1</a> and <a href="#pone-0001200-t002">2</a>). Their identity is discussed below. For the 100 most enriched or depleted cDNAs, the quantitative estimates of fold change have a normalized standard deviation of ~10% (see <a href="#s4">Methods</a>). Although SPI can identify growth inhibitory cDNAs, there is little reason to expect functional coherence within this group since interference with many functions can be detrimental. No candidates are obviously related to plasmid maintenance.</p>
<div class="figure" id="pone-0001200-t001"><div class="img"><a name="pone-0001200-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>cDNAs Among the Top 100 at Both Tempratures</span></strong></p><span>doi:10.1371/journal.pone.0001200.t001</span></div><div class="figure" id="pone-0001200-t002"><div class="img"><a name="pone-0001200-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>cDNAs Among the Bottom 100 at Both Temperatures</span></strong></p><span>doi:10.1371/journal.pone.0001200.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>By comparing data sets for cells cultured in galactose medium <em>vs</em> glucose medium, we minimized the possible contribution to SPI Extremes of host cell mutations which affect growth. To validate the impact of single SPI Extreme cDNAs we have used three growth tests at 30°C, with the understanding that the SPI protocol of growth for 30 generations greatly amplifies differences which are expected to be only modest after briefer growth intervals.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In the first test, we evaluated growth on solid media including galactose <em>vs</em> glucose (e.g. <a href="#pone-0001200-g003">Figure 3A/B</a>). All transformants gave colonies of comparable size on glucose plates. Of 70 strains which showed SPI depletion at 30°C less than −10 or SPI enrichment greater than +10, ~80% were confirmed as being stimulatory or inhibitory by these plate tests (In <a href="#pone-0001200-g003">Figure 3B</a>, 9/13 of the transformants which showed SPI depletion were smaller than the average of the five controls, while 5/13 of the transformants which showed SPI enrichment generated colonies larger than the average size of the controls.) None showed growth characteristics on plates opposite those which are registered by SPI fold-change calculations.</p>
<div class="figure" id="pone-0001200-g003"><div class="img"><a name="pone-0001200-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Validations on Solid Media Five control strains (#1–3, 18–19) which show little or no SPI enrichment/depletion and 27 SPI extreme strains including representatives from <a href="#pone-0001200-t003">Table 3</a> were streaked on glucose dropout plates (Glc: A) or galactose dropout plates (Gal: B) and allowed to grow at 30°C.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Strains which show SPI depletion are in the inner circle, while strains which show enrichment are in the outer circle. Growth of all strains is comparable on the glucose plates. On the galactose plate, the colony sizes of the strains often parallels expectations, both for increases and decreases. Among those illustrated, the concordance is modest, but more extensive surveys raise it to ~80% (see text). The strains which do not conform to expectations could reflect differences between the requirements for growth in liquid vs solid media. The identities of the strains are given in <a href="#pone-0001200-t003">Table 3</a>. Strains #1–3 and #18–19 are controls whose SPI values range from −0.6 to 1.19</p>
<span>doi:10.1371/journal.pone.0001200.g003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The second test involved growth over 24 hr in liquid galactose <em>vs</em> glucose medium. This test appeared to be more sensitive for detection of the inhibitory cDNAs than the plate test, in that 11/13 of the strains in <a href="#pone-0001200-g003">Figure 3</a> which showed SPI depletion grew more slowly in liquid culture than the controls. The normalized growth rate of the entire group of 13 strains was 0.57+/−0.15–<a href="#pone-0001200-t003">Table 3</a>. The liquid test appears less sensitive than the plate test for transformants which had consistently shown SPI enrichment. We observe, for example, a normalized growth rate of 0.93+/−0.12 for a set of 22 such strains (<a href="#pone-0001200-t003">Table 3</a>), i.e. comparable to the controls. As illustrated in the enrichment plot in <a href="#pone-0001200-g002">Figure 2D</a>, the positive SPI extremes at 30°C were less pronounced than for the depletions.</p>
<div class="figure" id="pone-0001200-t003"><div class="img"><a name="pone-0001200-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Validation Tests</span></strong></p><span>doi:10.1371/journal.pone.0001200.t003</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>In a final liquid medium validation test, we have mixed small pools of transformants, grown them in glucose or galactose medium and assessed the relative abundance of their cDNAs both at t<sub>0</sub> and after ten generations, using ds-Linear PCR. As shown in <a href="#pone.0001200.s003">Figure S3</a>, those which had shown enrichment persisted, while those which had been depleted vanished.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>Thus, the predictive power of SPI is high. SPI therefore could be applicable under circumstances in which clonal validation is not possible.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>Since transformants can be cultured in galactose medium at temperatures up to 37°C without loss of viability (as judged by staining with FUN1), parallel experiments have also been conducted at this modestly stressful temperature (<a href="#pone-0001200-g002">Figure 2E</a>). Under these conditions, there were more examples of strong enrichment than at 30°C, while decreases remained modest. <a href="#pone-0001200-t001">Table 1</a> lists those which are most enriched at both 30°C and at 37°C.</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>SPI can identify cDNAs which show differential enrichment upon exposure to stress. It therefore can also make predictions as to how to protect cells against stress or sensitize them to stress. For this purpose, we have evaluated the fold-enrichment of cDNAs after growth at 37°C <em>vs</em> 30°C in galactose–<a href="#pone-0001200-g002">Figure 2F</a>. Several groups of cDNAs were among the 100 which showed the highest differential enrichment (<a href="#pone-0001200-t004">Table 4</a>).</p>
<div class="figure" id="pone-0001200-t004"><div class="img"><a name="pone-0001200-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>cDNAs Differentially Enriched at 37°C</span></strong></p><span>doi:10.1371/journal.pone.0001200.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Gene complementation strategies can be used to investigate diseases of monogenic origin but are seldom applied to diseases of more complex origin. The present study shows that related strategies can be extended to the analysis of both normal growth and growth at elevated temperature, providing an example of a complex phenotype. A fundamental difference between SPI and classical complementation cloning is that the goal is to identify a spectrum of “contributory genes” (or cDNAs), rather than a single entity (“command gene”), which by definition have graded impact on cellular phenotypes. Such genes are not identified by classical genetic strategies unless they overtly change the phenotype. A second fundamental difference is a loosening of the concept of genetic selection, since in SPI the selection is at the level of differential cDNA enrichment, rather than overt phenotypic correction. The two attendant characteristics of SPI which are integrally part of this strategy are the expression of single ectopic cDNAs, and the inclusive quantitative readout which is afforded by microarrays.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Two related plasmid-enrichment strategies have recently been described for yeast, both of which used microarrays and were based on two-way (two-color) comparisons. The first used a centromeric galactose-inducible cDNA library to investigate aspects of drug resistance. Plasmids were recovered and transcribed to yield single-stranded fluorescent probes which identified at least one logical target, but appeared also to generate non-specific products <a href="#pone.0001200-Butcher1">[10]</a>. The second study used a high copy yeast genomic library to identify proteins which are related to the exosome. Plasmids recovered after growth were amplified through <em>E. coli</em> and then copied to yield fluorescent probes <a href="#pone.0001200-Abruzzi1">[11]</a>. In neither study was attention given to the information content of depleted cDNAs.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The realization that multiple cDNAs can promote cell growth may seem at variance with the concept of there being a single rate-limiting-step. Nevertheless, such multiplicity is characteristic of optimization strategies for complex systems, reflecting the intricate interactions among numerous components which contribute to the sustenance of the whole. Unlike other complex systems, when working with cells the availability of gene libraries make it possible to learn which components are most critical. Clearly, the optimization strategy of SPI could be made more stringent by prolonging the growth/selection interval. Moreover, it could be iterated to identify dependent groups of functionally significant genes, e.g. by starting with a cell in which one SPI extreme cDNA is already overexpressed (or depleted).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Our control experiments (<a href="#pone-0001200-g001">Figure 1</a>) using defined mixtures of cDNA transformants show that SPI can yield data sets in which real positives overwhelmingly dominate the readout. Moreover, their recovery does not depend on GC content, the accuracy of replicates appears satisfactory (fold changes having a normalized standard deviation are ~10%), and signal strength on the microarrays is a monotonic approximately linear function of input. Analysis of pools of transformants grown without induction already identified some differential depletion of cDNAs, presumably because unspecified factors were sequestered by the inserts.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Upon induction, considering that the library which we have used results in significant production of most of the proteins encoded by cDNAs <a href="#pone.0001200-Zhu1">[9]</a>, <a href="#pone.0001200-Gelperin1">[12]</a>, it is notable that yeast tolerates overexpression of a large fraction of its genome.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>To our knowledge, there has been no previous identification of single proteins which can increase the rate of growth or survival of yeast under standard growth conditions at 30°C. The cDNAs which become enriched could either directly accelerate the cell cycle or possibly bypass checkpoints or other events which delay rapid growth. Strikingly, twenty-nine cDNAs among the 100 which were most enriched at 30°C were also among the 100 most enriched at 37°C (<a href="#pone-0001200-t001">Table 1</a>). Several of these pertain to events which have previously been recognized as important control points: ribosome synthesis and translation initiation, for example, are generally considered to limit the speed of cell growth and are implicated in oncogenesis <a href="#pone.0001200-Mamane1">[13]</a>. Moreover, censorship of glycoprotein exit from the ER is certainly needed for growth and expansion of the cell wall.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The six of these which encode proteins which are critical for protein synthesis or ribosome genesis-and therefore potentially affect the titer of many other proteins-are:</p>


<ul class="bulletlist">

<li><span class="underline">Fun12p/eIF5B</span>, a GTPase which functions in translation initiation by promoting binding of Met-tRNA<sub>i</sub><sup>Met</sup> to small ribosomal subunits and in subunit joining <a href="#pone.0001200-Guillon1">[14]</a>.</li>

<li><span class="underline">Egd2p</span>, a subunit of the NAC complex which binds nascent polypeptide chains and is thought to influence their delivery to the ER <a href="#pone.0001200-Reimann1">[15]</a>.</li>

<li><span class="underline">Rps9A/B</span>, a conserved small ribosomal subunit protein which is a major determinant of translational fidelity <a href="#pone.0001200-Stansfield1">[16]</a>.</li>

<li><span class="underline">Utp10p</span>, a protein associated with U3 snoRNA which is required for 18S rRNA synthesis <a href="#pone.0001200-Dragon1">[17]</a>.</li>

<li><span class="underline">Rpl5p/L1</span>, which is required for assembly of large ribosomal subunits <a href="#pone.0001200-Deshmukh1">[18]</a>.</li>

<li><span class="underline">Lsm3p</span>, which functions in mRNA decay <a href="#pone.0001200-Beggs1">[19]</a>.</li>

</ul><a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Three cDNAs encode proteins which are relevant to ER-Golgi transport:</p>


<ul class="bulletlist">

<li><span class="underline">Gls2p/Rot2p</span>, one of the subunits of the endoplasmic reticulum glucosidase II. This enzyme trims the N-glycans of newly-synthesized glycoproteins after their folding in the ER and prior to exit to the Golgi. It is also required for efficient ER-associated degradation <a href="#pone.0001200-Helenius1">[20]</a>, <a href="#pone.0001200-Simons1">[21]</a>. Suboptimal maturation and expression of one or more of its glycoprotein substrates–or perhaps a cell wall component such as 1,6-β-glucan–could normally limit cell growth.</li>

<li><span class="underline">Bet2p</span>, a prenyl transferase which is required to anchor Ypt1p and Sec4p, which function in ER-Golgi transport <a href="#pone.0001200-Rossi1">[22]</a>.</li>

<li><span class="underline">Sfb2p</span>, a probable component of COP II vesicles <a href="#pone.0001200-Peng1">[23]</a>.</li>

</ul><a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Two encode components which impact the actin cytoskeleton</p>


<ul class="bulletlist">

<li><span class="underline">Slm1p</span>, which regulates actin organization in response to stress <a href="#pone.0001200-Fadri1">[24]</a>.</li>

<li><span class="underline">Mlc2p</span>, the regulatory light chain of myosin I <a href="#pone.0001200-Luo1">[25]</a>.</li>

</ul><a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>SPI readily identifies cDNAs which become depleted upon growth. As mentioned above, many of these can be validated qualitatively, both by following the colony size of single transformants on solid media (<a href="#pone-0001200-g003">Figure 3</a>) and by monitoring the growth of pure cultures in liquid medium (<a href="#pone-0001200-t003">Table 3</a>).</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>Previous studies have used panels of <em>S. cerevisiae</em> transformants to evaluate the ability of individual cDNAs, GST fusions and gene fragments to inhibit growth (<a href="#pone-0001200-t005">Table 5</a>, <a href="#pone.0001200.s012">Table S7</a>) <a href="#pone.0001200-Sopko1">[8]</a>, <a href="#pone.0001200-Akada1">[26]</a>, <a href="#pone.0001200-Boyer1">[27]</a>, <a href="#pone.0001200-Espinet1">[28]</a>, <a href="#pone.0001200-Liu1">[29]</a>, <a href="#pone.0001200-Ramer1">[30]</a>, <a href="#pone.0001200-Stevenson1">[31]</a>. These mostly qualitative assays monitor clonal growth on solid media, which places distinct demands on cells (e.g. <a href="#pone.0001200-Meunier1">[32]</a>), while SPI is based on a competition assay in liquid culture and lends itself to quantitative comparisons. <a href="#pone-0001200-t005">Table 5</a> summarizes the vectors used and numbers of inhibitory cDNAs which were identified. <a href="#pone-0001200-t006">Table 6</a> tabulates the extent of overlap with SPI. From the SPI data set we have used the 600 most depleted cDNAs for this comparison since the two recent large-scale studies based on colony growth of cDNA transformants have identified 454 and 759 inhibitory cDNAs, respectively <a href="#pone.0001200-Sopko1">[8]</a>, <a href="#pone.0001200-Gelperin1">[12]</a>. Judging from this comparison, SPI shows the greatest overlap with the one study which has used the same GST-ORF expression library that we have used <a href="#pone.0001200-Sopko1">[8]</a>–although it uses a different host cell. The Venn diagram in <a href="#pone-0001200-g004">Figure 4</a> illustrates the overlap between SPI and this study as well as a recent study which used an ORF-GST fusion library <a href="#pone.0001200-Gelperin1">[12]</a>.</p>
<div class="figure" id="pone-0001200-g004"><div class="img"><a name="pone-0001200-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Venn Diagram comparison of data sets which have identified growth-inhibitory cDNAs.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>See <a href="#pone-0001200-t005">Table 5</a> and <a href="#pone-0001200-t006">6</a> for detail.</p>
<span>doi:10.1371/journal.pone.0001200.g004</span></div><div class="figure" id="pone-0001200-t005"><div class="img"><a name="pone-0001200-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Screens for Inhibitory cDNAs</span></strong></p><span>doi:10.1371/journal.pone.0001200.t005</span></div><div class="figure" id="pone-0001200-t006"><div class="img"><a name="pone-0001200-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Coincidence of Inhibitory cDNAs</span></strong></p><span>doi:10.1371/journal.pone.0001200.t006</span></div><a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p><a href="#pone-0001200-t007">Table 7</a> enumerates the 27 cDNAs which are shared by the SPI 600, the second study which used the same library, and the study which is based on expression of ORF-GST fusions <a href="#pone.0001200-Gelperin1">[12]</a>. Of this group, only 5 are also shared by the investigation which surveys random transcriptional fragments <a href="#pone.0001200-Boyer1">[27]</a>. Proteins which affect growth can do so as part of complexes. Since the functionality of such complexes may be perturbed by alterations of their stoichiometry <a href="#pone.0001200-Papp1">[33]</a>, we have asked whether the group of 27 depleted SPI extremes enumerated in <a href="#pone-0001200-t007">Table 7</a> is enriched in proteins which are known to be in stable complexes. Judging from the database [<a href="http://portal.curagen.com/cgi-bin/com.curagen.portal.servlet.PortalYeastListmodeInList">http://portal.curagen.com/cgi-bin/com.cu​ragen.portal.servlet.PortalYeastListmode​InList</a>], there is no obvious enrichment. A similar conclusion has been reached in the second study which surveys the same GST-ORF library <a href="#pone.0001200-Sopko1">[8]</a>.</p>
<div class="figure" id="pone-0001200-t007"><div class="img"><a name="pone-0001200-t007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001200" data-uri="info:doi/10.1371/journal.pone.0001200.t007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001200.t007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001200.t007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001200.t007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001200.t007.TIF"></span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Inhibitory cDNAs in Multiple Studies</span></strong></p><span>doi:10.1371/journal.pone.0001200.t007</span></div><a id="article1.body1.sec3.p13" name="article1.body1.sec3.p13"></a><p>With regard to stress resistance, among the 100 which show the greatest differential enrichment at 37<strong>°</strong>C (<em>vs</em> 30°C) are 21 cDNAs which encode mitochondrial proteins, reminiscent of longstanding observations of the importance of functional mitochondria for survival at elevated temperature <a href="#pone.0001200-Ogur1">[34]</a>. Additionally, six contribute to expression of GPI-anchored proteins, which concentrate at the cell surface <a href="#pone.0001200-Lim1">[35]</a>, <a href="#pone.0001200-Tomishige1">[36]</a>. <a href="#pone-0001200-t004">Table 4</a> also lists groups of differentially enriched cDNAs which are implicated in protein turnover or stress responses.</p>
<a id="article1.body1.sec3.p14" name="article1.body1.sec3.p14"></a><p>Calculations of differential enrichment at 37°C are clearly a composite, including cDNAs which were enriched at 37°C as well as those which were depleted at 30°C. <a href="#pone.0001200.s004">Figure S4</a> separates out these two subsets by comparing the 37°C <em>vs</em> 30°C data to both the enrichment at 37°C and the depletion at 30°C. The subgroup encoding mitochondrial proteins accounts for ~40% of those which show an increment at 37°C.</p>
<a id="article1.body1.sec3.p15" name="article1.body1.sec3.p15"></a><p>One might expect that the SPI Extreme (enriched) cDNAs would correspond to essential genes; however, as for the total genome, only about 20% of the cDNAs which are most enriched at 30°C (or 37°C) correspond to genes which are essential under standard growth conditions (<em>Saccharomyces</em> Genome Database). Thus, survival of the organism under laboratory conditions cannot require those genes which have the potential to be most beneficial. These “accessory” beneficial genes represent an evolutionary opportunity (or therapeutic opportunity) which can be detected by SPI.</p>
<a id="article1.body1.sec3.p16" name="article1.body1.sec3.p16"></a><p>It is also of interest to ask whether SPI Extreme enriched cDNAs correspond to mRNAs which are upregulated at 37°C (in either glucose or galactose medium). The comparison is greatly simplified since the same microarrays can be used for both purposes. As shown in <a href="#pone.0001200.s005">Figure S5</a>, there is minimal concordance. This discrepancy could signify that the normal circuitry of gene expression seldom allows the cell to manipulate the level of single transcripts, i.e. bystander transcripts which would be co-modulated would sabotage any attempt to up- or down-regulate those which–by themselves–could be most useful.</p>
<a id="article1.body1.sec3.p17" name="article1.body1.sec3.p17"></a><p>We expect that the greatest prospect for implementation of SPI will be in the context of animal cell biology, where it should again exemplify the utility of genetic approaches outside the normal realm of genetic inquiry. This is especially because of the difficulty of choosing optimal therapeutic targets for many diseases. In each case, subtle selective events are surely always at work. SPI makes it possible to use a covert selection “<em>in situ</em>” to identify the single genes which should be manipulated to influence such phenotypes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cells and cDNA Library</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>These materials were obtained from M. Snyder and D. Gelperin (Yale University) <a href="#pone.0001200-Zhu1">[9]</a>. The haploid host cell was YC123 = SF657-2D = Snyder strain 258 (MATa <em>pep4-3 his4-580 ura3-52 leu2-3, 112</em>). The 10 kb pEG(KG) 2μ vector used for cDNA expression carried both <em>URA3</em> and <em>leu2-d</em> selectable markers and appends GST to the N-terminus of each product <a href="#pone.0001200-Mitchell1">[37]</a>. Frozen stocks of single transformants and pools of transformants were prepared by standard methods.</p>


<h4>Cell Growth, Plasmid and RNA Recovery</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Liquid cultures were established from single colonies. After growth in uracil dropout medium at room temperature, aliquots were diluted to A600 = 0.1 using uracil drop-out glycerol-lactate medium (2% glycerol, 2% lactate, 0.05% glucose, 0.67% bacto-yeast nitrogen base without amino acids, pH 5.5), grown overnight at room temperature to A600 = 1−2 and then harvested by sedimentation.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>5 ml samples of cells were washed with water, broken by vortexing with glass beads, and extracted with a Qiagen DNA extraction kit.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>To evaluate cell viability, samples were stained with FUN1 (Molecular Probes (F-7030)) and examined by epifluorescence. Living cells showed bar-shaped orange structures in the vacuole while dead cells lacked this signal and were predominantly green.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>To study growth at 30°C and 37°C, frozen pools including all 5885 strains were thawed, washed, and then adjusted to A600 = 0.05−0.1 in glycerol-lactate medium. 5 ml at OD600 = 1−2 was set aside and refrozen to provide a t<sub>0</sub> sample. Duplicate cultures were supplemented with 2% glucose or 2% galactose and then shaken at 30 and 37°C. Growth was monitored at 600 nm and aliquots of each culture were rediluted to A600 = 0.05−0.1 so that the A600 never exceeded 1.5. Duplicate 5 ml cultures were snap frozen in liquid nitrogen after a total of 30 generations.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>For RNA analysis, we used hot phenol <a href="#pone.0001200-Kohrer1">[38]</a> to extract logarithmic cultures growing in glycerol-lactate medium supplemented with 2% glucose- or 2% galactose and processed the samples in accordance with Affymetrix protocols.</p>


<h4>Ds-Linear PCR Amplification</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Linear amplification of cDNA inserts was performed using a mixture of DNA polymerases (Taq and pfu) and the reverse primer (5′-TGTAATACGACTCACTATAGGGGATCCCCGGGAATT​GCCATG-3)which includes the T7 phage RNA polymerase sequences (5 min pre-denaturation at 94°C, followed by 30 cycles of 1 min at 94°C, 1 min at 58°C and 7 min at 67°C, and concluding with 10 min at 70°C). The concluding step with addition of forward primer (5′-TGGTGGTGGTGGAATTCCAGCTGACCACC-3′) and Hotstart taq polymerase consisted of 6 min at 94°C, 5 sec at 95°C, 1 min at 58°C, 7 min at 67°C and 10 min at 70°C. The primers were complementary to sequences which flank the inserts and do not include the regions which encode GST or the GAL promoter.</p>


<h4>Microarray analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Ds-linear PCR products were pooled, concentrated and purified using a Qiagen column. In general, 200 ng of PCR product was used to generate biotinylated cRNA probes for Affymetrix S98 DNA microarrays. Samples were processed at the University Affymetrix facility, scanning the microarrays with a GeneArray scanner and preprocessing data using RMA (Robust multiarray average) <a href="#pone.0001200-Bolstad1">[39]</a>, <a href="#pone.0001200-Speed1">[40]</a>. For background-correction, normalization, and signal intensity calculations. In all experiments described below we studied independent triplicates or quadruplicates and based analysis on those cDNAs for which signals are classified as “Present” in each sample, using the MAS5 algorithm. Normalized signals from totally independent replicates have a mean correlation coefficient of 0.963014.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>To learn whether cDNAs which give only a weak signal at t = 0 can be studied reproducibly, we asked whether there is any correlation between initial signal intensity and the consistency of their presence (or absence) after 30 generations of growth. We did not detect any such correlation using Student's t-test. Signal intensity data from replicate samples were used to perform differential expression analysis by fitting a linear model (one way ANOVA) using the limma package. The P values were adjusted with an empirical Bayesian method and the Benjamini and Hochberg False Discover Rate. Fold changes of &gt;2 with p&lt;0.05 were considered meaningful. Average fold-changes were then used to produce the enrichment or “S-plots” and to identify the most enriched and most depleted cDNAs. Fractional values reflect depletion and are represented according to the following convention: 0.1 = −10; 0.2 = −5 etc. Standard deviations of the logarithm of fold change values were also estimated using the limma (Linear Models for Microarray Analysis) package <a href="#pone.0001200-Wettenhall1">[41]</a>. For the groups of fold changes listed in <a href="#pone.0001200.s006">Table S1</a>, <a href="#pone.0001200.s007">S2</a>, <a href="#pone.0001200.s008">S3</a>, <a href="#pone.0001200.s009">S4</a>, <a href="#pone.0001200.s010">S5</a> and <a href="#pone.0001200.s011">S6</a>, the means +/− standard deviation are −6.43+/−0.60 (30°C decreases); 4.24+/−0.48 (30°C increases); −6.24+/−0.47 (37°C depletion); 4.90+/−0.47 (37°C increases). Given the normalization procedures of RMA, the fold change estimates are relative.</p>


<h4>Validation</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Growth of individual transformants was studied by streaking single colonies onto solid media (2% glycerol, 2% lactate in uracil drop-out medium, pH5.5+2% glucose or+0.05% raffinose and 2% galactose) at 30°C and following their growth for increasing periods of time. We also have monitored the growth of duplicate cultures of single transformants in liquid uracil drop-out glycerol-lactate medium supplemented with 2% glucose <em>vs</em> 2% galactose at 30°C over 24 hrs. Cultures were initiated with an A600 = 0.03. Light scattering at 620 nm was measured with a Coulter-Beckman plate reader as a function of time. Kinetic rate constants were estimated using Origin and the galactose data were compared to the rate of growth of parallel cultures of each strain in glucose medium. To provide a uniform point of comparison between experiments, the five control strains (#1–3 and #18–19 in <a href="#pone-0001200-g003">Figure 3</a>) were included in each experiment, their average rate of growth was normalized to 1.0 and individual strains were then compared to them.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Alternatively, mixtures of transformants were cultured in liquid media identical to those used for the initial experiment, processed using ds-Linear PCR and the resulting double-stranded DNA products were then resolved on Agarose gels.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>To monitor growth in the absence of galactose, selected cDNAs (without the GST moiety) were copied by conventional PCR and subcloned into a <em>URA3</em> vector in which transcription was under control of a <em>MET25</em>, methionine-represssible, promoter <a href="#pone.0001200-Niedenthal1">[42]</a>. Corresponding transformants were then tested on plates made with complete medium <em>vs</em> methionine-dropout medium.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001200"></div><a name="pone.0001200.s001" id="pone.0001200.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Plasmid Extraction and Examples of ds-Linear PCR Products A)DNA was extracted from a set of six transformants, restricted with SmaI, and fractionated on a gel and stained with ethidium bromide. Note the progressively increasing sizes of the inserts, whose abundance is comparable to that of the endogenous 2-micron circle. B) An equivalent pool of eight plasmids was used for ds-Linear PCR. The fractionated products are illustrated adjacent to size standards (*)</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0001200.s002" id="pone.0001200.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Determination of the End-Point for ds-Linear PCR <a href="#pone.0001200.s001">Figure S1</a> shows that the products of ds-Linear PCR yield sharp bands, which is surprising. To determine the approximate point of arrest of the linear cycles, the products of the first 30 cycles (30R)-which included only the reverse primer-were incubated with any of three forward primers for the final step of the reaction. Our conventional primer (F)-not shown-and one of the test primers (F1) caused an obvious increase of intensity of the product bands. A second test primer (F2) did not. The point of arrest is therefore between sites complementary to F1 and F2. Primer F2 is 103 base upstream of primer F1, which is 234 bases upstream of primer F. The sequence of F1 is 5′-ATGTGCCTGGATGCGTTCC-3′. The sequence of F2 is 5′-TGAAAATGTTCGAAGATCGTTTATGTC-3′.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.14 MB DOC)</p>
<a name="pone.0001200.s003" id="pone.0001200.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Validation by PCR. A small pool of transformants (a–d) was either extracted at once or allowed to grow for ten generations at 30°C in glucose or galactose medium, before DNA extraction. Their degree of enrichment or depletion (fold-change) in the initial experiment with 5885 transformants is indicated in parentheses. Triplicate samples of the initial and final pools were copied by ds-Linear PCR and the products were analyzed. Note that the enriched cDNA becomes dominant and that those which had been depleted vanish, by comparison to a control. The control plasmid had become neither enriched nor depleted when part of the complete pool of strains in the initial experiment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.18 MB TIF)</p>
<a name="pone.0001200.s004" id="pone.0001200.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Analysis of Differential Enrichment (37°C vs 30°C) cDNAs which show strong differential enrichment in <a href="#pone-0001200-t002">Table 2</a> can do so either because of enrichment at 37°C or depletion at 30°C. Correspondingly, cDNAs which show strong differential depletion at 37°C can do so either because of depletion at 37°C or enrichment at 30°C. The three panels to the left concern the 100 cDNAs which show the greatest relative depletion at 37°C, while those at the right concern the 100 which show the greatest enrichment. Note, in each case, the presence of cDNAs which show both types of behavior.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB TIF)</p>
<a name="pone.0001200.s005" id="pone.0001200.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Comparison of 30°C SPI Data to Transcriptional Profiles. Panel A represents SPI second order fold data which are calculated by dividing the 37°C vs 30°C fold change in galactose by the 37°C vs 30°C fold change in glucose. In panels B–D, the second order SPI data are compared to RNA transcript profiles of the same host cell (without plasmid) cultured at 30°C or 37°C in glucose or galactose medium. In panel B the RNA signals at 37°C in glucose are compared to RNA data at 30°C in glucose. In panel C the RNA signals at 37°C in galactose are compared to 30°C in galactose. In panel D the (37°C galactose/30°C galactose) ratio is compared to the (37°C glucose/30°C glucose) ratio. As can be readily seen, there is no widespread correspondence between the levels of mRNAs and SPI data. The transcripts which do show strong induction upon addition of galactose include the familiar set of genes GAL1, GAL2, etc.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.07 MB TIF)</p>
<a name="pone.0001200.s006" id="pone.0001200.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s006">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">glc30 vs control. For <a href="#pone.0001200.s006">Tables S1</a>, <a href="#pone.0001200.s007">S2</a>, <a href="#pone.0001200.s008">S3</a>, <a href="#pone.0001200.s009">S4</a>, <a href="#pone.0001200.s010">S5</a> and <a href="#pone.0001200.s011">S6</a>: Full SPI Data Sets. The averaged data are divided into six groups as in <a href="#pone-0001200-g002">Figure 2</a>, comparing S1) Cells cultured in glucose medium at 30°C to t<sub>0</sub> samples, S2) Cells cultured in glucose medium at 37°C to t<sub>0</sub> samples, S3) Cells cultured in glucose medium at 37°C to cells cultured in glucose medium at 30°C, S4) Cells grown at 30°C in galactose vs 30°C in glucose, S5) Cells grown at 37°C in galactose vs 37°C in glucose, and S6) Cells grown at 37°C in galactose vs 30°C in galactose. The successive columns indicate the Affymetrix identification number interrogated, the Systematic and Gene names, the logarithm of fold change (FC), the t statistic (1/normalized standard deviation), the P value, adjusted P value, and B statistic.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.82 MB XLS)</p>
<a name="pone.0001200.s007" id="pone.0001200.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s007">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">glc37 vs control</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(1.82 MB XLS)</p>
<a name="pone.0001200.s008" id="pone.0001200.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s008">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">gal30 vs glc30</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(1.82 MB RTF)</p>
<a name="pone.0001200.s009" id="pone.0001200.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s009">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">glc37 vs glc30</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(1.82 MB XLS)</p>
<a name="pone.0001200.s010" id="pone.0001200.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s010">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">gal37 vs glc37</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(1.82 MB XLS)</p>
<a name="pone.0001200.s011" id="pone.0001200.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s011">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">gal37 vs gal30</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(1.82 MB XLS)</p>
<a name="pone.0001200.s012" id="pone.0001200.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001200.s012">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Inhibitory cDNAs, Comparison to Previous Studies. The citations of earlier investigations which have identified yeast cDNAs and gene fragments which inhibit growth are included in the text. In the study of Sopko et al. the magnitude of growth inhibition is designated 1–4, with 1 being the strongest inhibition.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001200.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.28 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Drs. D. Gelperin and M. Snyder for materials and B. Dujon, T. Hattier, P. Leahy, S. Lemmon, C. Li, G. Smyth, M. Sy, M. Veigl, H. Weinstein, C. Widnell, P. Zhang and Y. Zhang for information and advice.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AT. Performed the experiments: DW ET. Analyzed the data: AT DW ET. Wrote the paper: AT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001200-Deiss1" id="pone.0001200-Deiss1"></a>Deiss LP, Feinstein E, Berissi H, Cohen O, Kimchi A (1995) Identification of a novel serine/threonine kinase and a novel 15-kD protein as potential mediators of the gamma interferon-induced cell death. Genes Dev  9: 15–30.  <ul class="find" data-citedArticleID="1019774" data-doi="10.1101/gad.9.1.15"><li><a href="http://dx.doi.org/10.1101/gad.9.1.15" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+novel+serine%2Fthreonine+kinase+and+a+novel+15-kD+protein+as+potential+mediators+of+the+gamma+interferon-induced+cell+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+novel+serine%2Fthreonine+kinase+and+a+novel+15-kD+protein+as+potential+mediators+of+the+gamma+interferon-induced+cell+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001200-Moffat1" id="pone.0001200-Moffat1"></a>Moffat J, Grueneberg DA, Yang X, Kim SY, Kloepfer AM, et al.  (2006) A lentiviral RNAi library for human and mouse genes applied to an arrayed viral high-content screen. Cell  124: 1283–1298.  <ul class="find" data-citedArticleID="1019806" data-doi="10.1016/j.cell.2006.01.040"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.01.040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+lentiviral+RNAi+library+for+human+and+mouse+genes+applied+to+an+arrayed+viral+high-content+screen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+lentiviral+RNAi+library+for+human+and+mouse+genes+applied+to+an+arrayed+viral+high-content+screen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001200-Ossovskaya1" id="pone.0001200-Ossovskaya1"></a>Ossovskaya VS, Mazo IA, Chernov MV, Chernova OB, Strezoska Z, et al.  (1996) Use of genetic suppressor elements to dissect distinct biological effects of separate p53 domains. Proc Natl Acad Sci U S A  93: 10309–10314.  <ul class="find" data-citedArticleID="1019812" data-doi="10.1073/pnas.93.19.10309"><li><a href="http://dx.doi.org/10.1073/pnas.93.19.10309" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+genetic+suppressor+elements+to+dissect+distinct+biological+effects+of+separate+p53+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+genetic+suppressor+elements+to+dissect+distinct+biological+effects+of+separate+p53+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001200-Pestov1" id="pone.0001200-Pestov1"></a>Pestov DG, Lau LF (1994) Genetic selection of growth-inhibitory sequences in mammalian cells. Proc Natl Acad Sci U S A  91: 12549–12553.  <ul class="find" data-citedArticleID="1019820" data-doi="10.1073/pnas.91.26.12549"><li><a href="http://dx.doi.org/10.1073/pnas.91.26.12549" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+selection+of+growth-inhibitory+sequences+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+selection+of+growth-inhibitory+sequences+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001200-Singhi1" id="pone.0001200-Singhi1"></a>Singhi AD, Kondratov RV, Neznanov N, Chernov MV, Gudkov AV (2004) Selection-subtraction approach (SSA): a universal genetic screening technique that enables negative selection. Proc Natl Acad Sci U S A  101: 9327–9332.  <ul class="find" data-citedArticleID="1019830" data-doi="10.1073/pnas.0403080101"><li><a href="http://dx.doi.org/10.1073/pnas.0403080101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection-subtraction+approach+%28SSA%29%3A+a+universal+genetic+screening+technique+that+enables+negative+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection-subtraction+approach+%28SSA%29%3A+a+universal+genetic+screening+technique+that+enables+negative+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001200-Giaever1" id="pone.0001200-Giaever1"></a>Giaever G, Flaherty P, Kumm J, Proctor M, Nislow C, et al.  (2004) Chemogenomic profiling: identifying the functional interactions of small molecules in yeast. Proc Natl Acad Sci U S A  101: 793–798.  <ul class="find" data-citedArticleID="1019786" data-doi="10.1073/pnas.0307490100"><li><a href="http://dx.doi.org/10.1073/pnas.0307490100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemogenomic+profiling%3A+identifying+the+functional+interactions+of+small+molecules+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemogenomic+profiling%3A+identifying+the+functional+interactions+of+small+molecules+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001200-Parsons1" id="pone.0001200-Parsons1"></a>Parsons AB, Brost RL, Ding H, Li Z, Zhang C, et al.  (2004) Integration of chemical-genetic and genetic interaction data links bioactive compounds to cellular target pathways. Nat Biotechnol  22: 62–69.  <ul class="find" data-citedArticleID="1019816" data-doi="10.1038/nbt919"><li><a href="http://dx.doi.org/10.1038/nbt919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integration+of+chemical-genetic+and+genetic+interaction+data+links+bioactive+compounds+to+cellular+target+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integration+of+chemical-genetic+and+genetic+interaction+data+links+bioactive+compounds+to+cellular+target+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001200-Sopko1" id="pone.0001200-Sopko1"></a>Sopko R, Huang D, Preston N, Chua G, Papp B, et al.  (2006) Mapping pathways and phenotypes by systematic gene overexpression. Mol Cell  21: 319–330.  <ul class="find" data-citedArticleID="1019832" data-doi="10.1016/j.molcel.2005.12.011"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.12.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+pathways+and+phenotypes+by+systematic+gene+overexpression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+pathways+and+phenotypes+by+systematic+gene+overexpression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001200-Zhu1" id="pone.0001200-Zhu1"></a>Zhu H, Bilgin M, Bangham R, Hall D, Casamayor A, et al.  (2001) Global analysis of protein activities using proteome chips. Science  293: 2101–2105.  <ul class="find" data-citedArticleID="1019844" data-doi="10.1126/science.1062191"><li><a href="http://dx.doi.org/10.1126/science.1062191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+protein+activities+using+proteome+chips." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+protein+activities+using+proteome+chips.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001200-Butcher1" id="pone.0001200-Butcher1"></a>Butcher RA, Schreiber SL (2006) A microarray-based protocol for monitoring the growth of yeast overexpression strains. Nat Protoc  1: 569–576.  <ul class="find" data-citedArticleID="1019772" data-doi="10.1038/nprot.2006.80"><li><a href="http://dx.doi.org/10.1038/nprot.2006.80" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microarray-based+protocol+for+monitoring+the+growth+of+yeast+overexpression+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microarray-based+protocol+for+monitoring+the+growth+of+yeast+overexpression+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001200-Abruzzi1" id="pone.0001200-Abruzzi1"></a>Abruzzi K, Denome S, Olsen JR, Assenholt J, Haaning LL, et al.  (2007) A novel plasmid-based microarray screen identifies suppressors of rrp6Delta in Saccharomyces cerevisiae. Mol Cell Biol  27: 1044–1055.  <ul class="find" data-citedArticleID="1019762" data-doi="10.1128/mcb.01299-06"><li><a href="http://dx.doi.org/10.1128/mcb.01299-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+plasmid-based+microarray+screen+identifies+suppressors+of+rrp6Delta+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+plasmid-based+microarray+screen+identifies+suppressors+of+rrp6Delta+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001200-Gelperin1" id="pone.0001200-Gelperin1"></a>Gelperin DM, White MA, Wilkinson ML, Kon Y, Kung LA, et al.  (2005) Biochemical and genetic analysis of the yeast proteome with a movable ORF collection. Genes Dev  19: 2816–2826.  <ul class="find" data-citedArticleID="1019784" data-doi="10.1101/gad.1362105"><li><a href="http://dx.doi.org/10.1101/gad.1362105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+and+genetic+analysis+of+the+yeast+proteome+with+a+movable+ORF+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+and+genetic+analysis+of+the+yeast+proteome+with+a+movable+ORF+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001200-Mamane1" id="pone.0001200-Mamane1"></a>Mamane Y, Petroulakis E, Rong L, Yoshida K, Ler LW, et al.  (2004) eIF4E–from translation to transformation. Oncogene  23: 3172–3179.  <ul class="find" data-citedArticleID="1019800" data-doi="10.1038/sj.onc.1207549"><li><a href="http://dx.doi.org/10.1038/sj.onc.1207549" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=eIF4E%E2%80%93from+translation+to+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22eIF4E%E2%80%93from+translation+to+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001200-Guillon1" id="pone.0001200-Guillon1"></a>Guillon L, Schmitt E, Blanquet S, Mechulam Y (2005) Initiator tRNA binding by e/aIF5B, the eukaryotic/archaeal homologue of bacterial initiation factor IF2. Biochemistry  44: 15594–15601.  <ul class="find" data-citedArticleID="1019788" data-doi="10.1021/bi051514j"><li><a href="http://dx.doi.org/10.1021/bi051514j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initiator+tRNA+binding+by+e%2FaIF5B%2C+the+eukaryotic%2Farchaeal+homologue+of+bacterial+initiation+factor+IF2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initiator+tRNA+binding+by+e%2FaIF5B%2C+the+eukaryotic%2Farchaeal+homologue+of+bacterial+initiation+factor+IF2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001200-Reimann1" id="pone.0001200-Reimann1"></a>Reimann B, Bradsher J, Franke J, Hartmann E, Wiedmann M, et al.  (1999) Initial characterization of the nascent polypeptide-associated complex in yeast. Yeast  15: 397–407.  <ul class="find" data-citedArticleID="1019824"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+characterization+of+the+nascent+polypeptide-associated+complex+in+yeast.&amp;auth=&amp;atitle=Initial+characterization+of+the+nascent+polypeptide-associated+complex+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+characterization+of+the+nascent+polypeptide-associated+complex+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+characterization+of+the+nascent+polypeptide-associated+complex+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001200-Stansfield1" id="pone.0001200-Stansfield1"></a>Stansfield I, Jones KM, Herbert P, Lewendon A, Shaw WV, et al.  (1998) Missense translation errors in Saccharomyces cerevisiae. J Mol Biol  282: 13–24.  <ul class="find" data-citedArticleID="1019836" data-doi="10.1006/jmbi.1998.1976"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1976" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Missense+translation+errors+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Missense+translation+errors+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001200-Dragon1" id="pone.0001200-Dragon1"></a>Dragon F, Gallagher JE, Compagnone-Post PA, Mitchell BM, Porwancher KA, et al.  (2002) A large nucleolar U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis. Nature  417: 967–970.  <ul class="find" data-citedArticleID="1019778" data-doi="10.1038/nature00769"><li><a href="http://dx.doi.org/10.1038/nature00769" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+large+nucleolar+U3+ribonucleoprotein+required+for+18S+ribosomal+RNA+biogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+large+nucleolar+U3+ribonucleoprotein+required+for+18S+ribosomal+RNA+biogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001200-Deshmukh1" id="pone.0001200-Deshmukh1"></a>Deshmukh M, Stark J, Yeh LC, Lee JC, Woolford JL Jr (1995) Multiple regions of yeast ribosomal protein L1 are important for its interaction with 5 S rRNA and assembly into ribosomes. J Biol Chem  270: 30148–30156.  <ul class="find" data-citedArticleID="1019776" data-doi="10.1074/jbc.270.50.30148"><li><a href="http://dx.doi.org/10.1074/jbc.270.50.30148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+regions+of+yeast+ribosomal+protein+L1+are+important+for+its+interaction+with+5+S+rRNA+and+assembly+into+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+regions+of+yeast+ribosomal+protein+L1+are+important+for+its+interaction+with+5+S+rRNA+and+assembly+into+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001200-Beggs1" id="pone.0001200-Beggs1"></a>Beggs JD (2005) Lsm proteins and RNA processing. Biochem Soc Trans  33: 433–438.  <ul class="find" data-citedArticleID="1019766" data-doi="10.1042/bst0330433"><li><a href="http://dx.doi.org/10.1042/bst0330433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lsm+proteins+and+RNA+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lsm+proteins+and+RNA+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001200-Helenius1" id="pone.0001200-Helenius1"></a>Helenius A, Aebi M (2004) Roles of N-linked glycans in the endoplasmic reticulum. Annu Rev Biochem  73: 1019–1049.  <ul class="find" data-citedArticleID="1019790" data-doi="10.1146/annurev.biochem.73.011303.073752"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.73.011303.073752" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+of+N-linked+glycans+in+the+endoplasmic+reticulum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+of+N-linked+glycans+in+the+endoplasmic+reticulum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001200-Simons1" id="pone.0001200-Simons1"></a>Simons JF, Ebersold M, Helenius A (1998) Cell wall 1,6-beta-glucan synthesis in Saccharomyces cerevisiae depends on ER glucosidases I and II, and the molecular chaperone BiP/Kar2p. Embo J  17: 396–405.  <ul class="find" data-citedArticleID="1019828" data-doi="10.1093/emboj/17.2.396"><li><a href="http://dx.doi.org/10.1093/emboj/17.2.396" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+wall+1%2C6-beta-glucan+synthesis+in+Saccharomyces+cerevisiae+depends+on+ER+glucosidases+I+and+II%2C+and+the+molecular+chaperone+BiP%2FKar2p." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+wall+1%2C6-beta-glucan+synthesis+in+Saccharomyces+cerevisiae+depends+on+ER+glucosidases+I+and+II%2C+and+the+molecular+chaperone+BiP%2FKar2p.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001200-Rossi1" id="pone.0001200-Rossi1"></a>Rossi G, Yu JA, Newman AP, Ferro-Novick S (1991) Dependence of Ypt1 and Sec4 membrane attachment on Bet2. Nature  351: 158–161.  <ul class="find" data-citedArticleID="1019826" data-doi="10.1038/351158a0"><li><a href="http://dx.doi.org/10.1038/351158a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dependence+of+Ypt1+and+Sec4+membrane+attachment+on+Bet2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dependence+of+Ypt1+and+Sec4+membrane+attachment+on+Bet2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001200-Peng1" id="pone.0001200-Peng1"></a>Peng R, De Antoni A, Gallwitz D (2000) Evidence for overlapping and distinct functions in protein transport of coat protein Sec24p family members. J Biol Chem  275: 11521–11528.  <ul class="find" data-citedArticleID="1019818" data-doi="10.1074/jbc.275.15.11521"><li><a href="http://dx.doi.org/10.1074/jbc.275.15.11521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+overlapping+and+distinct+functions+in+protein+transport+of+coat+protein+Sec24p+family+members." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+overlapping+and+distinct+functions+in+protein+transport+of+coat+protein+Sec24p+family+members.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001200-Fadri1" id="pone.0001200-Fadri1"></a>Fadri M, Daquinag A, Wang S, Xue T, Kunz J (2005) The pleckstrin homology domain proteins Slm1 and Slm2 are required for actin cytoskeleton organization in yeast and bind phosphatidylinositol-4,5-bisphosphate and TORC2. Mol Biol Cell  16: 1883–1900.  <ul class="find" data-citedArticleID="1019782" data-doi="10.1091/mbc.e04-07-0564"><li><a href="http://dx.doi.org/10.1091/mbc.e04-07-0564" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pleckstrin+homology+domain+proteins+Slm1+and+Slm2+are+required+for+actin+cytoskeleton+organization+in+yeast+and+bind+phosphatidylinositol-4%2C5-bisphosphate+and+TORC2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pleckstrin+homology+domain+proteins+Slm1+and+Slm2+are+required+for+actin+cytoskeleton+organization+in+yeast+and+bind+phosphatidylinositol-4%2C5-bisphosphate+and+TORC2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001200-Luo1" id="pone.0001200-Luo1"></a>Luo J, Vallen EA, Dravis C, Tcheperegine SE, Drees B, et al.  (2004) Identification and functional analysis of the essential and regulatory light chains of the only type II myosin Myo1p in Saccharomyces cerevisiae. J Cell Biol  165: 843–855.  <ul class="find" data-citedArticleID="1019798" data-doi="10.1083/jcb.200401040"><li><a href="http://dx.doi.org/10.1083/jcb.200401040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+functional+analysis+of+the+essential+and+regulatory+light+chains+of+the+only+type+II+myosin+Myo1p+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+functional+analysis+of+the+essential+and+regulatory+light+chains+of+the+only+type+II+myosin+Myo1p+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001200-Akada1" id="pone.0001200-Akada1"></a>Akada R, Yamamoto J, Yamashita I (1997) Screening and identification of yeast sequences that cause growth inhibition when overexpressed. Mol Gen Genet  254: 267–274.  <ul class="find" data-citedArticleID="1019764" data-doi="10.1007/s004380050415"><li><a href="http://dx.doi.org/10.1007/s004380050415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Screening+and+identification+of+yeast+sequences+that+cause+growth+inhibition+when+overexpressed." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Screening+and+identification+of+yeast+sequences+that+cause+growth+inhibition+when+overexpressed.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001200-Boyer1" id="pone.0001200-Boyer1"></a>Boyer J, Badis G, Fairhead C, Talla E, Hantraye F, et al.  (2004) Large-scale exploration of growth inhibition caused by overexpression of genomic fragments in Saccharomyces cerevisiae. Genome Biol  5: R72.  <ul class="find" data-citedArticleID="1019770"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Large-scale+exploration+of+growth+inhibition+caused+by+overexpression+of+genomic+fragments+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Large-scale+exploration+of+growth+inhibition+caused+by+overexpression+of+genomic+fragments+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+exploration+of+growth+inhibition+caused+by+overexpression+of+genomic+fragments+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+exploration+of+growth+inhibition+caused+by+overexpression+of+genomic+fragments+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001200-Espinet1" id="pone.0001200-Espinet1"></a>Espinet C, de la Torre MA, Aldea M, Herrero E (1995) An efficient method to isolate yeast genes causing overexpression-mediated growth arrest. Yeast  11: 25–32.  <ul class="find" data-citedArticleID="1019780"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+efficient+method+to+isolate+yeast+genes+causing+overexpression-mediated+growth+arrest.&amp;auth=&amp;atitle=An+efficient+method+to+isolate+yeast+genes+causing+overexpression-mediated+growth+arrest." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+efficient+method+to+isolate+yeast+genes+causing+overexpression-mediated+growth+arrest." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+efficient+method+to+isolate+yeast+genes+causing+overexpression-mediated+growth+arrest.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001200-Liu1" id="pone.0001200-Liu1"></a>Liu H, Krizek J, Bretscher A (1992) Construction of a GAL1-regulated yeast cDNA expression library and its application to the identification of genes whose overexpression causes lethality in yeast. Genetics  132: 665–673.  <ul class="find" data-citedArticleID="1019796"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Construction+of+a+GAL1-regulated+yeast+cDNA+expression+library+and+its+application+to+the+identification+of+genes+whose+overexpression+causes+lethality+in+yeast.&amp;auth=&amp;atitle=Construction+of+a+GAL1-regulated+yeast+cDNA+expression+library+and+its+application+to+the+identification+of+genes+whose+overexpression+causes+lethality+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+a+GAL1-regulated+yeast+cDNA+expression+library+and+its+application+to+the+identification+of+genes+whose+overexpression+causes+lethality+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+a+GAL1-regulated+yeast+cDNA+expression+library+and+its+application+to+the+identification+of+genes+whose+overexpression+causes+lethality+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001200-Ramer1" id="pone.0001200-Ramer1"></a>Ramer SW, Elledge SJ, Davis RW (1992) Dominant genetics using a yeast genomic library under the control of a strong inducible promoter. Proc Natl Acad Sci U S A  89: 11589–11593.  <ul class="find" data-citedArticleID="1019822" data-doi="10.1073/pnas.89.23.11589"><li><a href="http://dx.doi.org/10.1073/pnas.89.23.11589" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominant+genetics+using+a+yeast+genomic+library+under+the+control+of+a+strong+inducible+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dominant+genetics+using+a+yeast+genomic+library+under+the+control+of+a+strong+inducible+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001200-Stevenson1" id="pone.0001200-Stevenson1"></a>Stevenson LF, Kennedy BK, Harlow E (2001) A large-scale overexpression screen in Saccharomyces cerevisiae identifies previously uncharacterized cell cycle genes. Proc Natl Acad Sci U S A  98: 3946–3951.  <ul class="find" data-citedArticleID="1019838" data-doi="10.1073/pnas.051013498"><li><a href="http://dx.doi.org/10.1073/pnas.051013498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+large-scale+overexpression+screen+in+Saccharomyces+cerevisiae+identifies+previously+uncharacterized+cell+cycle+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+large-scale+overexpression+screen+in+Saccharomyces+cerevisiae+identifies+previously+uncharacterized+cell+cycle+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001200-Meunier1" id="pone.0001200-Meunier1"></a>Meunier JR, Choder M (1999) Saccharomyces cerevisiae colony growth and ageing: biphasic growth accompanied by changes in gene expression. Yeast  15: 1159–1169.  <ul class="find" data-citedArticleID="1019802"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Saccharomyces+cerevisiae+colony+growth+and+ageing%3A+biphasic+growth+accompanied+by+changes+in+gene+expression.&amp;auth=&amp;atitle=Saccharomyces+cerevisiae+colony+growth+and+ageing%3A+biphasic+growth+accompanied+by+changes+in+gene+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saccharomyces+cerevisiae+colony+growth+and+ageing%3A+biphasic+growth+accompanied+by+changes+in+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Saccharomyces+cerevisiae+colony+growth+and+ageing%3A+biphasic+growth+accompanied+by+changes+in+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001200-Papp1" id="pone.0001200-Papp1"></a>Papp B, Pal C, Hurst LD (2003) Dosage sensitivity and the evolution of gene families in yeast. Nature  424: 194–197.  <ul class="find" data-citedArticleID="1019814" data-doi="10.1038/nature01771"><li><a href="http://dx.doi.org/10.1038/nature01771" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dosage+sensitivity+and+the+evolution+of+gene+families+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dosage+sensitivity+and+the+evolution+of+gene+families+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001200-Ogur1" id="pone.0001200-Ogur1"></a>Ogur M, Ogur S, St John R (1960) Temperature Dependence of the Spontaneous Mutation Rate to Respiration Deficiency in Saccharomyces. Genetics  45: 189–194.  <ul class="find" data-citedArticleID="1019810"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Temperature+Dependence+of+the+Spontaneous+Mutation+Rate+to+Respiration+Deficiency+in+Saccharomyces.&amp;auth=&amp;atitle=Temperature+Dependence+of+the+Spontaneous+Mutation+Rate+to+Respiration+Deficiency+in+Saccharomyces." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temperature+Dependence+of+the+Spontaneous+Mutation+Rate+to+Respiration+Deficiency+in+Saccharomyces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temperature+Dependence+of+the+Spontaneous+Mutation+Rate+to+Respiration+Deficiency+in+Saccharomyces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001200-Lim1" id="pone.0001200-Lim1"></a>Lim T, Loh W, Shih Y (2000) A comparison of prediction accurary, complexity, and training time of thirty-three old and new classification algorithms. Machine Learning  40: 203–228.  <ul class="find" data-citedArticleID="1019794"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+comparison+of+prediction+accurary%2C+complexity%2C+and+training+time+of+thirty-three+old+and+new+classification+algorithms.&amp;auth=&amp;atitle=A+comparison+of+prediction+accurary%2C+complexity%2C+and+training+time+of+thirty-three+old+and+new+classification+algorithms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+prediction+accurary%2C+complexity%2C+and+training+time+of+thirty-three+old+and+new+classification+algorithms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+prediction+accurary%2C+complexity%2C+and+training+time+of+thirty-three+old+and+new+classification+algorithms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001200-Tomishige1" id="pone.0001200-Tomishige1"></a>Tomishige N, Noda Y, Adachi H, Shimoi H, Takatsuki A, et al.  (2003) Mutations that are synthetically lethal with a gas1Delta allele cause defects in the cell wall of Saccharomyces cerevisiae. Mol Genet Genomics  269: 562–573.  <ul class="find" data-citedArticleID="1019840" data-doi="10.1007/s00438-003-0864-9"><li><a href="http://dx.doi.org/10.1007/s00438-003-0864-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+that+are+synthetically+lethal+with+a+gas1Delta+allele+cause+defects+in+the+cell+wall+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+that+are+synthetically+lethal+with+a+gas1Delta+allele+cause+defects+in+the+cell+wall+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001200-Mitchell1" id="pone.0001200-Mitchell1"></a>Mitchell DA, Marshall TK, Deschenes RJ (1993) Vectors for the inducible overexpression of glutathione S-transferase fusion proteins in yeast. Yeast  9: 715–722.  <ul class="find" data-citedArticleID="1019804"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Vectors+for+the+inducible+overexpression+of+glutathione+S-transferase+fusion+proteins+in+yeast.&amp;auth=&amp;atitle=Vectors+for+the+inducible+overexpression+of+glutathione+S-transferase+fusion+proteins+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vectors+for+the+inducible+overexpression+of+glutathione+S-transferase+fusion+proteins+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vectors+for+the+inducible+overexpression+of+glutathione+S-transferase+fusion+proteins+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001200-Kohrer1" id="pone.0001200-Kohrer1"></a>Kohrer K, Domdey H (1991) Preparation of high molecular weight RNA. Methods Enzymol  194: 398–405.  <ul class="find" data-citedArticleID="1019792" data-doi="10.1016/0076-6879(91)94030-g"><li><a href="http://dx.doi.org/10.1016/0076-6879(91)94030-g" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preparation+of+high+molecular+weight+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preparation+of+high+molecular+weight+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001200-Bolstad1" id="pone.0001200-Bolstad1"></a>Bolstad BM, Irizarry RA, Astrand M, Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics  19: 185–193.  <ul class="find" data-citedArticleID="1019768" data-doi="10.1093/bioinformatics/19.2.185"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001200-Speed1" id="pone.0001200-Speed1"></a>Speed TP (2003) Statistical analysis of gene expression microarray data. Boca Raton, FL: Chapman &amp; Hall/CRC.   xiii, 222 p., [224] p. of plates p.  <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001200-Wettenhall1" id="pone.0001200-Wettenhall1"></a>Wettenhall JM, Smyth GK (2004) limmaGUI: a graphical user interface for linear modeling of microarray data. Bioinformatics  20: 3705–3706.  <ul class="find" data-citedArticleID="1019842" data-doi="10.1093/bioinformatics/bth449"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=limmaGUI%3A+a+graphical+user+interface+for+linear+modeling+of+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22limmaGUI%3A+a+graphical+user+interface+for+linear+modeling+of+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001200-Niedenthal1" id="pone.0001200-Niedenthal1"></a>Niedenthal RK, Riles L, Johnston M, Hegemann JH (1996) Green fluorescent protein as a marker for gene expression and subcellular localization in budding yeast. Yeast  12: 773–786.  <ul class="find" data-citedArticleID="1019808"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Green+fluorescent+protein+as+a+marker+for+gene+expression+and+subcellular+localization+in+budding+yeast.&amp;auth=&amp;atitle=Green+fluorescent+protein+as+a+marker+for+gene+expression+and+subcellular+localization+in+budding+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Green+fluorescent+protein+as+a+marker+for+gene+expression+and+subcellular+localization+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Green+fluorescent+protein+as+a+marker+for+gene+expression+and+subcellular+localization+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.XML" value="145027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.PDF" value="393716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g001.PNG_L" value="1480081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g001.PNG_M" value="107275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g001.PNG_S" value="11248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g001.TIF" value="3465034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g001.PNG_I" value="37208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g002.PNG_L" value="843602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g002.PNG_M" value="55815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g002.PNG_S" value="7465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g002.TIF" value="2140036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g002.PNG_I" value="19519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t001.PNG_L" value="163423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t001.PNG_M" value="115690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t001.PNG_S" value="19110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t001.TIF" value="844118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t001.PNG_I" value="97380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t002.PNG_L" value="150432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t002.PNG_M" value="100331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t002.PNG_S" value="17975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t002.TIF" value="835934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t002.PNG_I" value="89647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g003.PNG_L" value="2941574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g003.PNG_M" value="377678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g003.PNG_S" value="11797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g003.TIF" value="4077780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g003.PNG_I" value="115061"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t003.PNG_L" value="397473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t003.PNG_M" value="105302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t003.PNG_S" value="17676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t003.TIF" value="2158382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t003.PNG_I" value="72516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t004.PNG_L" value="71799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t004.PNG_M" value="69080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t004.PNG_S" value="12348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t004.TIF" value="490112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t004.PNG_I" value="33556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g004.PNG_L" value="413865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g004.PNG_M" value="94493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g004.PNG_S" value="10507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g004.TIF" value="831048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.g004.PNG_I" value="38006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t005.PNG_L" value="123006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t005.PNG_M" value="68214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t005.PNG_S" value="9296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t005.TIF" value="491828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t005.PNG_I" value="20729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t006.PNG_L" value="98296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t006.PNG_M" value="55731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t006.PNG_S" value="9129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t006.TIF" value="498594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t006.PNG_I" value="17577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t007.PNG_L" value="130919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t007.PNG_M" value="93534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t007.PNG_S" value="14776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t007.TIF" value="750192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.t007.PNG_I" value="72863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s001.TIF" value="103771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s002.TIF" value="141206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s003.TIF" value="181682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s004.TIF" value="55496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s005.TIF" value="72808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s006.XLS" value="1817600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s007.XLS" value="1817600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s008.XLS" value="1817600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s009.XLS" value="1817600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s010.XLS" value="1817600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s011.XLS" value="1817600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001200.s012.XLS" value="282624"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001200&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001200" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001200&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001200&volume=&issue=&title=Covert Genetic Selections to Optimize Phenotypes&author_name=Di%20Wu%2C%20Elizabeth%20Townsley%2C%20Alan%20Michael%20Tartakoff&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001200" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001200" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001200" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001200&amp;t=Covert%20Genetic%20Selections%20to%20Optimize%20Phenotypes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001200&title=Covert%20Genetic%20Selections%20to%20Optimize%20Phenotypes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001200&amp;title=Covert%20Genetic%20Selections%20to%20Optimize%20Phenotypes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001200" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001200&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Covert Genetic Selections to Optimize Phenotypes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001200';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Covert%20Genetic%20Selections%20to%20Optimize%20Phenotypes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001200" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001200" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complementary+DNA%22" title="Search for articles in the subject area:'Complementary DNA'"><div class="flagText">Complementary DNA</div></a>
              <div data-categoryid="32053" data-articleid="25232"
                   data-categoryname="Complementary DNA"
                   class="flagImage" title="Flag 'Complementary DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Galactose%22" title="Search for articles in the subject area:'Galactose'"><div class="flagText">Galactose</div></a>
              <div data-categoryid="20227" data-articleid="25232"
                   data-categoryname="Galactose"
                   class="flagImage" title="Flag 'Galactose' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+pool%22" title="Search for articles in the subject area:'Gene pool'"><div class="flagText">Gene pool</div></a>
              <div data-categoryid="35161" data-articleid="25232"
                   data-categoryname="Gene pool"
                   class="flagImage" title="Flag 'Gene pool' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glucose%22" title="Search for articles in the subject area:'Glucose'"><div class="flagText">Glucose</div></a>
              <div data-categoryid="20225" data-articleid="25232"
                   data-categoryname="Glucose"
                   class="flagImage" title="Flag 'Glucose' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25232"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="25232"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25232"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22cDNA+libraries%22" title="Search for articles in the subject area:'cDNA libraries'"><div class="flagText">cDNA libraries</div></a>
              <div data-categoryid="35307" data-articleid="25232"
                   data-categoryname="cDNA libraries"
                   class="flagImage" title="Flag 'cDNA libraries' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3237'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9754'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9372&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
