

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen Candida albicans</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000996"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000996" />

    <meta name="citation_title" content="A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen Candida albicans"/>
    <meta itemprop="name" content="A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen Candida albicans"/>

      <meta name="citation_author" content="Isabel Miranda"/>
            <meta name="citation_author_institution" content="Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Rita Rocha"/>
            <meta name="citation_author_institution" content="Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Maria C. Santos"/>
            <meta name="citation_author_institution" content="Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Denisa D. Mateus"/>
            <meta name="citation_author_institution" content="Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Gabriela R. Moura"/>
            <meta name="citation_author_institution" content="Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Laura Carreto"/>
            <meta name="citation_author_institution" content="Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Manuel A. S. Santos"/>
            <meta name="citation_author_institution" content="Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal"/>

    <meta name="citation_date" content="2007/10/3"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000996.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e996"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Driving change: the evolution of alternative genetic codes.; citation_author=MA Santos; citation_author=G Moura; citation_author=SE Massey; citation_author=MF Tuite; citation_journal_title=Trends Genet; citation_volume=20; citation_number=1; citation_pages=95-102; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution of the genetic code in yeasts.; citation_author=I Miranda; citation_author=R Silva; citation_author=MA Santos; citation_journal_title=Yeast; citation_volume=23; citation_number=2; citation_pages=203-213; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=How mitochondria redefine the code.; citation_author=RD Knight; citation_author=LF Landweber; citation_author=M Yarus; citation_journal_title=J Mol Evol; citation_volume=53; citation_number=3; citation_pages=299-313; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Codon reassignment and amino acid composition in hemichordate mitochondria.; citation_author=J Castresana; citation_author=G Feldmaier-Fuchs; citation_author=S Paabo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=4; citation_pages=3703-3707; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Rewiring the keyboard: evolvability of the genetic code.; citation_author=RD Knight; citation_author=SJ Freeland; citation_author=LF Landweber; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=5; citation_pages=49-58; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=UGA can be decoded as tryptophan at low efficiency in Bacillus subtilis.; citation_author=PS Lovett; citation_author=NP Ambulos; citation_author=W Mulbry; citation_author=N Noguchi; citation_author=EJ Rogers; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=6; citation_pages=1810-1812; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Does Paramecium primaurelia use a different genetic code in its macronucleus?; citation_author=F Caron; citation_author=E Meyer; citation_journal_title=Nature; citation_volume=314; citation_number=7; citation_pages=185-188; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Differential use of termination codons in ciliated protozoa.; citation_author=DS Harper; citation_author=CL Jahn; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=8; citation_pages=3252-3256; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A new noncanonical nuclear genetic code: translation of UAA into glutamate.; citation_author=R Sanchez-Silva; citation_author=E Villalobo; citation_author=L Morin; citation_author=A Torres; citation_journal_title=Curr Biol; citation_volume=13; citation_number=9; citation_pages=442-447; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Developmentally regulated chromosome fragmentation linked to imprecise elimination of repeated sequences in paramecia.; citation_author=A Le Mouel; citation_author=A Butler; citation_author=F Caron; citation_author=E Meyer; citation_journal_title=Eukaryot Cell; citation_volume=2; citation_number=10; citation_pages=1076-1090; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Codon reassignment in Candida species: an evolutionary conundrum.; citation_author=MF Tuite; citation_author=MA Santos; citation_journal_title=Biochimie; citation_volume=78; citation_number=11; citation_pages=993-9; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=CUG codons in Candida spp.; citation_author=TH Jukes; citation_author=S Osawa; citation_journal_title=J Mol Evol; citation_volume=42; citation_number=12; citation_pages=321-322; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Non-universal usage of the leucine CUG codon and the molecular phylogeny of the genus Candida.; citation_author=T Sugita; citation_author=T Nakase; citation_journal_title=Syst Appl Microbiol; citation_volume=22; citation_number=13; citation_pages=79-86; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Dramatic events in ciliate evolution: alteration of UAA and UAG termination codons to glutamine codons due to anticodon mutations in two Tetrahymena tRNAs.; citation_author=N Hanyu; citation_author=Y Kuchino; citation_author=S Nishimura; citation_author=H Beier; citation_journal_title=EMBO J; citation_volume=5; citation_number=14; citation_pages=1307-1311; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=A new UAG-encoded residue in the structure of a methanogen methyltransferase.; citation_author=B Hao; citation_author=W Gong; citation_author=TK Ferguson; citation_author=CM James; citation_author=JA Krzycki; citation_journal_title=Science; citation_volume=296; citation_number=15; citation_pages=1462-1466; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of a selenocysteyl-tRNA(Ser) in mammalian cells that recognizes the nonsense codon, UGA.; citation_author=BJ Lee; citation_author=PJ Worland; citation_author=JN Davis; citation_author=TC Stadtman; citation_author=DL Hatfield; citation_journal_title=J Biol Chem; citation_volume=264; citation_number=16; citation_pages=9724-9727; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Selection and characterization of Escherichia coli variants capable of growth on an otherwise toxic tryptophan analogue.; citation_author=JM Bacher; citation_author=AD Ellington; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=17; citation_pages=5414-5425; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inhibited cell growth and protein functional changes from an editing-defective tRNA synthetase.; citation_author=JM Bacher; citation_author=V Crecy-Lagard; citation_author=PR Schimmel; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=18; citation_pages=1697-1701; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Global effects of mistranslation from an editing defect in mammalian cells.; citation_author=LA Nangle; citation_author=CM Motta; citation_author=P Schimmel; citation_journal_title=Chem Biol; citation_volume=13; citation_number=19; citation_pages=1091-1100; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An expanded eukaryotic genetic code.; citation_author=JW Chin; citation_author=TA Cropp; citation_author=JC Anderson; citation_author=M Mukherji; citation_author=Z Zhang; citation_journal_title=Science; citation_volume=301; citation_number=20; citation_pages=964-967; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Incorporation of nonnatural amino acids into proteins.; citation_author=TL Hendrickson; citation_author=V Crecy-Lagard; citation_author=P Schimmel; citation_journal_title=Annu Rev Biochem; citation_volume=73; citation_number=21; citation_pages=147-176; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transfer RNA structural change is a key element in the reassignment of the CUG codon in Candida albicans.; citation_author=MA Santos; citation_author=VM Perreau; citation_author=MF Tuite; citation_journal_title=Embo J; citation_volume=15; citation_number=22; citation_pages=5060-8; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The CUG codon is decoded in vivo as serine and not leucine in Candida albicans.; citation_author=MA Santos; citation_author=MF Tuite; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=23; citation_pages=1481-1486; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Non-universal decoding of the leucine codon CUG in several Candida species.; citation_author=T Ohama; citation_author=T Suzuki; citation_author=M Mori; citation_author=S Osawa; citation_author=T Ueda; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=24; citation_pages=4039-45; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The ‘polysemous’ codon--a codon with multiple amino acid assignment caused by dual specificity of tRNA identity.; citation_author=T Suzuki; citation_author=T Ueda; citation_author=K Watanabe; citation_journal_title=Embo J; citation_volume=16; citation_number=25; citation_pages=1122-34; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Non-standard translational events in Candida albicans mediated by an unusual seryl-tRNA with a 5′-CAG-3′ (leucine) anticodon.; citation_author=MA Santos; citation_author=G Keith; citation_author=MF Tuite; citation_journal_title=Embo J; citation_volume=12; citation_number=26; citation_pages=607-16; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The Candida albicans CUG-decoding ser-tRNA has an atypical anticodon stem-loop structure.; citation_author=VM Perreau; citation_author=G Keith; citation_author=WM Holmes; citation_author=A Przykorska; citation_author=MA Santos; citation_journal_title=J Mol Biol; citation_volume=293; citation_number=27; citation_pages=1039-1053; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=On malleability in the genetic code.; citation_author=DW Schultz; citation_author=M Yarus; citation_journal_title=J Mol Evol; citation_volume=42; citation_number=28; citation_pages=597-601; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Transfer RNA mutation and the malleability of the genetic code.; citation_author=DW Schultz; citation_author=M Yarus; citation_journal_title=J Mol Biol; citation_volume=235; citation_number=29; citation_pages=1377-1380; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Comparative evolutionary genomics unveils the molecular mechanism of reassignment of the CTG codon in Candida spp.; citation_author=SE Massey; citation_author=G Moura; citation_author=P Beltrao; citation_author=R Almeida; citation_author=JR Garey; citation_journal_title=Genome Res; citation_volume=13; citation_number=30; citation_pages=544-557; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Direct analysis of aminoacylation levels of tRNAs in vivo. Application to studying recognition of Escherichia coli initiator tRNA mutants by glutaminyl-tRNA synthetase.; citation_author=U Varshney; citation_author=CP Lee; citation_author=UL RajBhandary; citation_journal_title=J Biol Chem; citation_volume=266; citation_number=31; citation_pages=24712-24718; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Codon reassignment and the evolving genetic code: problems and pitfalls in post-genome analysis.; citation_author=JM O'Sullivan; citation_author=JB Davenport; citation_author=MF Tuite; citation_journal_title=Trends Genet; citation_volume=17; citation_number=32; citation_pages=20-22; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Seryl-tRNA synthetase is not responsible for the evolution of CUG codon reassignment in Candida albicans.; citation_author=JM O'Sullivan; citation_author=MJ Mihr; citation_author=MA Santos; citation_author=MF Tuite; citation_journal_title=Yeast; citation_volume=18; citation_number=33; citation_pages=313-322; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Phenotypic Switching.; citation_author=DR Soll; citation_journal_title=Candida and Candidiasis; citation_number=34; citation_pages=123-142; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The ALS gene family of Candida albicans.; citation_author=LL Hoyer; citation_journal_title=Trends Microbiol; citation_volume=9; citation_number=35; citation_pages=176-80; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Expression of the Candida albicans gene ALS1 in Saccharomyces cerevisiae induces adherence to endothelial and epithelial cells.; citation_author=Y Fu; citation_author=G Rieg; citation_author=WA Fonzi; citation_author=PH Belanger; citation_author=JE Edwards; citation_journal_title=Infect Immun; citation_volume=66; citation_number=36; citation_pages=1783-1786; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Global cell surface conformational shift mediated by a Candida albicans adhesin.; citation_author=JM Rauceo; citation_author=NK Gaur; citation_author=KG Lee; citation_author=JE Edwards; citation_author=SA Klotz; citation_journal_title=Infect Immun; citation_volume=72; citation_number=37; citation_pages=4948-4955; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Candida albicans and Saccharomyces cerevisiae expressing ALA1/ALS5 adhere to accessible threonine, serine, or alanine patches.; citation_author=NK Gaur; citation_author=RL Smith; citation_author=SA Klotz; citation_journal_title=Cell Commun Adhes; citation_volume=9; citation_number=38; citation_pages=45-57; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Overexpression of the Candida albicans ALA1 gene in Saccharomyces cerevisiae results in aggregation following attachment of yeast cells to extracellular matrix proteins, adherence properties similar to those of Candida albicans.; citation_author=NK Gaur; citation_author=SA Klotz; citation_author=RL Henderson; citation_journal_title=Infect Immun; citation_volume=67; citation_number=39; citation_pages=6040-6047; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Expression, cloning, and characterization of a Candida albicans gene, ALA1, that confers adherence properties upon Saccharomyces cerevisiae for extracellular matrix proteins.; citation_author=NK Gaur; citation_author=SA Klotz; citation_journal_title=Infect Immun; citation_volume=65; citation_number=40; citation_pages=5289-5294; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=EAP1, a Candida albicans gene involved in binding human epithelial cells.; citation_author=F Li; citation_author=SP Palecek; citation_journal_title=Eukaryot Cell; citation_volume=2; citation_number=41; citation_pages=1266-1273; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evidence implicating phospholipase as a virulence factor of Candida albicans.; citation_author=AS Ibrahim; citation_author=F Mirbod; citation_author=SG Filler; citation_author=Y Banno; citation_author=GT Cole; citation_journal_title=Infect Immun; citation_volume=63; citation_number=42; citation_pages=1993-8; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Candida albicans Secreted Aspartyl Proteinases in Virulence and Pathogenesis.; citation_author=JR Naglik; citation_author=SJ Challacombe; citation_author=B Hube; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=67; citation_number=43; citation_pages=400-428; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Adhesion in Candida spp.; citation_author=P Sundstrom; citation_journal_title=Cell Microbiol; citation_volume=4; citation_number=44; citation_pages=461-469; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Potential role of phospholipases in virulence and fungal pathogenesis.; citation_author=MA Ghannoum; citation_journal_title=Clin Microbiol Rev; citation_volume=13; citation_number=45; citation_pages=122-43; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Epigenetic properties of white-opaque switching in Candida albicans are based on a self-sustaining transcriptional feedback loop.; citation_author=RE Zordan; citation_author=DJ Galgoczy; citation_author=AD Johnson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=46; citation_pages=12807-12812; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bistable expression of WOR1, a master regulator of white-opaque switching in Candida albicans.; citation_author=G Huang; citation_author=H Wang; citation_author=S Chou; citation_author=X Nie; citation_author=J Chen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=47; citation_pages=12813-12818; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adhesive and mammalian transglutaminase substrate properties of Candida albicans Hwp1.; citation_author=JF Staab; citation_author=SD Bradway; citation_author=PL Fidel; citation_author=P Sundstrom; citation_journal_title=Science; citation_volume=283; citation_number=48; citation_pages=1535-8; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A screen in Saccharomyces cerevisiae identified CaMCM1, an essential gene in Candida albicans crucial for morphogenesis.; citation_author=M Rottmann; citation_author=S Dieter; citation_author=H Brunner; citation_author=S Rupp; citation_journal_title=Mol Microbiol; citation_volume=47; citation_number=49; citation_pages=943-959; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Hgc1, a novel hypha-specific G1 cyclin-related protein regulates Candida albicans hyphal morphogenesis.; citation_author=X Zheng; citation_author=Y Wang; citation_author=Y Wang; citation_journal_title=EMBO J; citation_volume=23; citation_number=50; citation_pages=1845-1856; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of a mating type-like locus in the asexual pathogenic yeast Candida albicans.; citation_author=CM Hull; citation_author=AD Johnson; citation_journal_title=Science; citation_volume=285; citation_number=51; citation_pages=1271-5; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=White-opaque switching in Candida albicans is controlled by mating-type locus homeodomain proteins and allows efficient mating.; citation_author=MG Miller; citation_author=AD Johnson; citation_journal_title=Cell; citation_volume=110; citation_number=52; citation_pages=293-302; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=High-frequency occurrence of chromosome translocation in a mutant strain of Candida albicans by a suppressor mutation of ploidy shift.; citation_author=SI Iwaguchi; citation_author=T Kanbe; citation_author=T Tohne; citation_author=PT Magee; citation_author=T Suzuki; citation_journal_title=Yeast; citation_volume=16; citation_number=53; citation_pages=411-422; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Correlation between polyploidy and auxotrophic segregation in the imperfect yeast Candida albicans.; citation_author=T Suzuki; citation_author=A Hitomi; citation_author=PT Magee; citation_author=S Sakaguchi; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=54; citation_pages=3345-3353; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Completion of a parasexual cycle in Candida albicans by induced chromosome loss in tetraploid strains.; citation_author=RJ Bennett; citation_author=AD Johnson; citation_journal_title=EMBO J; citation_volume=22; citation_number=55; citation_pages=2505-2515; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Induction of mating in Candida albicans by construction of MTLa and MTLalpha strains.; citation_author=BB Magee; citation_author=PT Magee; citation_journal_title=Science; citation_volume=289; citation_number=56; citation_pages=310-3; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evidence for mating of the “asexual” yeast Candida albicans in a mammalian host.; citation_author=CM Hull; citation_author=RM Raisner; citation_author=AD Johnson; citation_journal_title=Science; citation_volume=289; citation_number=57; citation_pages=307-10; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comparison of the separation of Candida albicans chromosome-sized DNA by pulsed-field gel electrophoresis techniques.; citation_author=BA Lasker; citation_author=GF Carle; citation_author=GS Kobayashi; citation_author=G Medoff; citation_journal_title=Nucleic Acids Res; citation_volume=17; citation_number=58; citation_pages=3783-3793; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Electrophoretic karyotypes and chromosome numbers in Candida species.; citation_author=BB Magee; citation_author=PT Magee; citation_journal_title=J Gen Microbiol; citation_volume=133; citation_number=59; citation_pages=425-430; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=The non-standard genetic code of Candida spp.: an evolving genetic code or a novel mechanism for adaptation?; citation_author=MA Santos; citation_author=T Ueda; citation_author=K Watanabe; citation_author=MF Tuite; citation_journal_title=Mol Microbiol; citation_volume=26; citation_number=60; citation_pages=423-431; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Standard and nonstandard mRNA decoding in Candida.; citation_author=JM O'Sullivan; citation_author=MA Santos; citation_author=MF Tuite; citation_journal_title=Candida and Candidiasis; citation_number=61; citation_pages=279-292; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The exchange of the discriminator base A73 for G is alone sufficient to convert human tRNA(Leu) into a serine-acceptor in vitro.; citation_author=K Breitschopf; citation_author=HJ Gross; citation_journal_title=EMBO J; citation_volume=13; citation_number=62; citation_pages=3166-3167; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Transfer RNA structural change is a key element in the reassignment of the CUG codon in Candida albicans.; citation_author=MA Santos; citation_author=VM Perreau; citation_author=MF Tuite; citation_journal_title=EMBO J; citation_volume=15; citation_number=63; citation_pages=5060-5068; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Comparative genome hybridization reveals widespread aneuploidy in Candida albicans laboratory strains.; citation_author=A Selmecki; citation_author=S Bergmann; citation_author=J Berman; citation_journal_title=Mol Microbiol; citation_volume=55; citation_number=64; citation_pages=1553-1565; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple chromosomal and phenotypic changes in spontaneous mutants of Candida albicans.; citation_author=EP Rustchenko-Bulgac; citation_author=DH Howard; citation_journal_title=J Gen Microbiol; citation_volume=139 Pt 6; citation_number=65; citation_pages=1195-1207; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Chromosomal rearrangements associated with morphological mutants provide a means for genetic variation of Candida albicans.; citation_author=EP Rustchenko-Bulgac; citation_author=F Sherman; citation_author=JB Hicks; citation_journal_title=J Bacteriol; citation_volume=172; citation_number=66; citation_pages=1276-83; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=High frequency variation of colony morphology and chromosome reorganization in the pathogenic yeast Candida albicans.; citation_author=T Suzuki; citation_author=I Kobayashi; citation_author=T Kanbe; citation_author=K Tanaka; citation_journal_title=J Gen Microbiol; citation_volume=135; citation_number=67; citation_pages=425-434; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Specific chromosome alterations in fluconazole-resistant mutants of Candida albicans.; citation_author=V Perepnikhatka; citation_author=FJ Fischer; citation_author=M Niimi; citation_author=RA Baker; citation_author=RD Cannon; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=68; citation_pages=4041-4049; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Variation in assimilating functions occurs in spontaneous Candida albicans mutants having chromosomal alterations.; citation_author=EP Rustchenko; citation_author=DH Howard; citation_author=F Sherman; citation_journal_title=Microbiology; citation_volume=143 ( Pt 5); citation_number=69; citation_pages=1765-1778; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Monosomy of a specific chromosome determines L-sorbose utilization: a novel regulatory mechanism in Candida albicans.; citation_author=G Janbon; citation_author=F Sherman; citation_author=E Rustchenko; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=70; citation_pages=5150-5155; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Homozygosity at the MTL locus in clinical strains of Candida albicans: karyotypic rearrangements and tetraploid formation.; citation_author=M Legrand; citation_author=P Lephart; citation_author=A Forche; citation_author=FM Mueller; citation_author=T Walsh; citation_journal_title=Mol Microbiol; citation_volume=52; citation_number=71; citation_pages=1451-1462; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Expression of mutant alanine tRNAs increases spontaneous mutagenesis in Escherichia coli.; citation_author=R Dorazi; citation_author=JJ Lingutla; citation_author=MZ Humayun; citation_journal_title=Mol Microbiol; citation_volume=44; citation_number=72; citation_pages=131-141; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The diploid genome sequence of Candida albicans.; citation_author=T Jones; citation_author=NA Federspiel; citation_author=H Chibana; citation_author=J Dungan; citation_author=S Kalman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=73; citation_pages=7329-7334; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Editing-defective tRNA synthetase causes protein misfolding and neurodegeneration.; citation_author=JW Lee; citation_author=K Beebe; citation_author=LA Nangle; citation_author=J Jang; citation_author=CM Longo-Guess; citation_journal_title=Nature; citation_volume=443; citation_number=74; citation_pages=50-55; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Selective advantages created by codon ambiguity allowed for the evolution of an alternative genetic code in Candida spp.; citation_author=MA Santos; citation_author=C Cheesman; citation_author=V Costa; citation_author=P Moradas-Ferreira; citation_author=MF Tuite; citation_journal_title=Mol Microbiol; citation_volume=31; citation_number=75; citation_pages=937-947; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Understanding Candida albicans at the molecular level.; citation_author=J Pla; citation_author=C Gil; citation_author=L Monteoliva; citation_author=F Navarro-Garcia; citation_author=M Sanchez; citation_journal_title=Yeast; citation_volume=12; citation_number=76; citation_pages=1677-702; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning: a laboratory manual.; citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=77; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Coexpression of eukaryotic tRNASer and yeast seryl-tRNA synthetase leads to functional amber suppression in Escherichia coli.; citation_author=I Weygand-Durasevic; citation_author=M Nalaskowska; citation_author=D Soll; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=78; citation_pages=232-239; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Use of a dot blot hybridization method for identification of pure tRNA species on different membranes.; citation_author=J Heitzler; citation_author=L Marechal-Drouard; citation_author=G Dirheimer; citation_author=G Keith; citation_journal_title=Biochim Biophys Acta; citation_volume=1129; citation_number=79; citation_pages=273-277; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=PCR protocols: A Guide to Methods and Applications.; citation_author=MA Innis; citation_author=DH Gelfand; citation_author=JJ Sninsky; citation_author=TJ White; citation_number=80; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Preparation of high molecular weight RNA.; citation_author=K Kohrer; citation_author=H Domdey; citation_journal_title=Methods Enzymol; citation_volume=194; citation_number=81; citation_pages=398-405; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Statistical Treatment for Rejection of Deviant Values - Critical-Values of Dixon Q Parameter and Related Subrange Ratios at the 95-Percent Confidence Level.; citation_author=DB Rorabacher; citation_journal_title=Analytical Chemistry; citation_volume=63; citation_number=82; citation_pages=139-146; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Factors affecting the phospholipase activity of Candida species in vitro.; citation_author=LP Samaranayake; citation_author=JM Raeside; citation_author=TW MacFarlane; citation_journal_title=Sabouraudia; citation_volume=22; citation_number=83; citation_pages=201-207; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Cell Cycle Analysis of Yeasts.; citation_author=M Fortuna; citation_author=MJ Sousa; citation_author=M Côrte-Real; citation_author=C Leão; citation_author=A Salvador; citation_journal_title=Current Protocols in Cytometry; citation_number=84; citation_pages=11.13.1-11.13.9; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Construction of an SfiI macrorestriction map of the Candida albicans genome.; citation_author=WS Chu; citation_author=BB Magee; citation_author=PT Magee; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=85; citation_pages=6637-6651; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000996" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen Candida albicans"/>
    <meta name="twitter:description" content="BackgroundThe discovery of genetic code alterations and expansions in both prokaryotes and eukaryotes abolished the hypothesis of a frozen and universal genetic code and exposed unanticipated flexibility in codon and amino acid assignments. It is now clear that codon identity alterations involve sense and non-sense codons and can occur in organisms with complex genomes and proteomes. However, the biological functions, the molecular mechanisms of evolution and the diversity of genetic code alterations remain largely unknown. In various species of the genus Candida, the leucine CUG codon is decoded as serine by a unique serine tRNA that contains a leucine 5&prime;-CAG-3&prime;anticodon (tRNACAGSer). We are using this codon identity redefinition as a model system to elucidate the evolution of genetic code alterations.Methodology/Principal FindingsWe have reconstructed the early stages of the Candida genetic code alteration by engineering tRNAs that partially reverted the identity of serine CUG codons back to their standard leucine meaning. Such genetic code manipulation had profound cellular consequences as it exposed important morphological variation, altered gene expression, re-arranged the karyotype, increased cell-cell adhesion and secretion of hydrolytic enzymes.Conclusion/SignificanceOur study provides the first experimental evidence for an important role of genetic code alterations as generators of phenotypic diversity of high selective potential and supports the hypothesis that they speed up evolution of new phenotypes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000996.g008"/>

  <meta property="og:title" content="A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen Candida albicans" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000996" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6010'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9387'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2738&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000996">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000996" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000996&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen <i>Candida albicans</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Isabel Miranda, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rita Rocha, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria C. Santos, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Denisa D. Mateus, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gabriela R. Moura, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura Carreto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Manuel A. S. Santos
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:msantos@bio.ua.pt">msantos@bio.ua.pt</a></p>

                <p>Affiliation:
                  Department of Biology, Centro de Estudos do Ambiente e do Mar (CESAM), University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 03, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000996</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000996" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000996" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000996" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000996" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000996-g001" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000996-g002" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000996-g003" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000996-g004" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000996-g005" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000996-g006" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000996-g007" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000996-g008" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000996">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000996" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The discovery of genetic code alterations and expansions in both prokaryotes and eukaryotes abolished the hypothesis of a frozen and universal genetic code and exposed unanticipated flexibility in codon and amino acid assignments. It is now clear that codon identity alterations involve sense and non-sense codons and can occur in organisms with complex genomes and proteomes. However, the biological functions, the molecular mechanisms of evolution and the diversity of genetic code alterations remain largely unknown. In various species of the genus <em>Candida</em>, the leucine CUG codon is decoded as serine by a unique serine tRNA that contains a leucine 5′-CAG-3′anticodon (tRNA<sub>CAG</sub><sup>Ser</sup>). We are using this codon identity redefinition as a model system to elucidate the evolution of genetic code alterations.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have reconstructed the early stages of the <em>Candida</em> genetic code alteration by engineering tRNAs that partially reverted the identity of serine CUG codons back to their standard leucine meaning. Such genetic code manipulation had profound cellular consequences as it exposed important morphological variation, altered gene expression, re-arranged the karyotype, increased cell-cell adhesion and secretion of hydrolytic enzymes.</p>


<h3>Conclusion/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our study provides the first experimental evidence for an important role of genetic code alterations as generators of phenotypic diversity of high selective potential and supports the hypothesis that they speed up evolution of new phenotypes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Miranda I, Rocha R, Santos MC, Mateus DD, Moura GR, et al.  (2007) A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen <em>Candida albicans</em>. PLoS ONE 2(10):
          e996.
            doi:10.1371/journal.pone.0000996</p><p><strong>Academic Editor: </strong>Rosemary Redfield, University of British Columbia, Canada</p><p><strong>Received:</strong> July 9, 2007; <strong>Accepted:</strong> September 18, 2007; <strong>Published:</strong> October 3, 2007</p><p><strong>Copyright:</strong> © 2007 Miranda et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by FCT/FEDER projects POCI/BIA-MIC/55466/04, POCI/BIA-PRO/55472/2004 and POCI/SAU-MMO/55476/2004. IM, RR are supported by FCT/FEDER, BD/19807/99 and BD/8296/2002 PhD grants, respectively. MASS was supported by an EMBO YIP and a Human Frontier Science Programme Grant (REF: RGP45/2005).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A number of exceptions to the standard genetic code have been discovered in prokaryotic and eukaryotic organisms, involving nonsense-to-sense and sense-to-sense codon identity changes <a href="#pone.0000996-Santos1">[1]</a>, <a href="#pone.0000996-Miranda1">[2]</a>. Twenty five of such alterations have been recorded in mitochondrial genetic codes of metazoan, fungi, red algae, green plants, alveolates, stramenopiles, haptophtes and euglenozoans <a href="#pone.0000996-Knight1">[3]</a>. The most remarkable alterations involve metazoan arginine AGA and AGG (AGR) codons, which changed their identity to serine at the base of the phylogenetic tree, and later on to translation-stop in vertebrates and to glycine in urochordates <a href="#pone.0000996-Castresana1">[4]</a>, <a href="#pone.0000996-Knight2">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In bacteria and eukaryotic cytoplasmic systems, 18 genetic code alterations have also been recorded, but unlike in mitochondria, they involve, with one exception, nonsense-to-sense codon identity changes or codon unassignments (codons that vanished from genomes). For example, in <em>Micrococcus</em> spp., <em>Mycoplasma</em> spp. and <em>Pseudomicrothorax dubius</em> the AGA/AUA, CGG and UGA codons are unassigned, respectively. In <em>Bacillus subtilis</em> UGA codons are used to terminate mRNA translation (stop codons) and to insert tryptophan, creating readthrough proteins <a href="#pone.0000996-Lovett1">[6]</a>. In various species of ciliates, 1 or 2 stop codons changed their identity to either glutamine (UAA and UAG), glutamate (UAA) or cysteine (UGA). Interestingly, these genetic code alterations apparently minimize nonsense errors arising from re-assembly of the ciliates fragmented genome <a href="#pone.0000996-Caron1">[7]</a>–<a href="#pone.0000996-LeMouel1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Those genetic code alterations show that certain codons, namely codons that start with uridine (UNN) or adenosine (ANN), in particular stop (UAA, UAG or UGA), arginine (AGR), serine (AGY) and isoleucine (AUA) codons are more prone to identity changes than others. The only exception to this first codon base rule occurs in yeast mitochondria where cytosine starting codons (CNN), namely leucine CUN codons, changed their identity to threonine. Also, the leucine CUG codon altered its identity to serine in the cytoplasm of various <em>Candida</em> and <em>Debaryomyces</em> species <a href="#pone.0000996-Tuite1">[11]</a>–<a href="#pone.0000996-Sugita1">[13]</a>. These findings suggest that the strength of the interaction of the first codon-anticodon base pair plays an important role in the evolution of genetic code alterations. Indeed, the change of identity of UAA and UAG stop codons to glutamine in various ciliates involved first base misreading by glutamine tRNAs, which decode glutamine CAA and CAG codons <a href="#pone.0000996-Harper1">[8]</a>, <a href="#pone.0000996-Hanyu1">[14]</a>. But, it also indicates that other forces beyond codon-anticodon interaction play important roles in codon identity redefinition.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The unexpected flexibility of the genetic code described above is further highlighted by insertion of selenocystein (21<sup>st</sup> amino acid) in the active sites of prokaryotic and eukaryotic selenoproteins and insertion of pyrrolysin (22<sup>sd</sup> amino acid) in the active site of monomethylamine methyltransferase of <em>Methanosarcina barkeri</em> <a href="#pone.0000996-Hao1">[15]</a>, <a href="#pone.0000996-Lee1">[16]</a>. These expansions involving reprogramming of UGA and UAG stop codons, respectively, highlight the potential of genetic code alterations/expansions to generate functional innovation. This hypothesis is supported by recent artificial expansion of the genetic code through synthetic biology methodologies. Indeed, 49 non-natural amino acids have already been incorporated into <em>E. coli</em>, yeast and mammalian cells <a href="#pone.0000996-Bacher1">[17]</a>–<a href="#pone.0000996-Chin1">[20]</a>, to produce novel proteins of biotechnological and biomedical interest. This dramatic demonstration of genetic code flexibility also unveiled an extraordinary capacity of complex organisms to tolerate partial codon identity redefinition <a href="#pone.0000996-Hendrickson1">[21]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We are using <em>C. albicans</em> as a model system to elucidate the evolution of genetic code alterations. In this case, a unique serine tRNA (tRNA<sub>CAG</sub><sup>Ser</sup>) decodes leucine CUG codons as serine <a href="#pone.0000996-Santos2">[22]</a>–<a href="#pone.0000996-Ohama1">[24]</a>. However, the tRNA<sub>CAG</sub><sup>Ser</sup> is recognized by both leucyl- and seryl-tRNA synthetases (LeuRS and SerRS) and it is aminoacylated <em>in vitro</em> with both serine (97%) and leucine (3%) <a href="#pone.0000996-Suzuki1">[25]</a>. This unusual dual aminoacylation of the tRNA<sub>CAG</sub><sup>Ser</sup> has been preserved to the present day <a href="#pone.0000996-Santos4">[26]</a>, <a href="#pone.0000996-Perreau1">[27]</a>, raising the intriguing possibility that it may play a role in <em>C. albicans</em> biology. It also provides strong support for the hypothesis that codon identity redefinition is driven through codon decoding ambiguity <a href="#pone.0000996-Schultz1">[28]</a>, <a href="#pone.0000996-Schultz2">[29]</a>. In here, we have reconstructed the early stages of the <em>C. albicans</em> genetic code alteration to shed new light into those questions. This partial reversion of CUG identity from serine back to leucine triggered morphogenesis, phenotypic switching, and up-regulated expression of genes involved in cell adhesion and hyphal development and increased secretion of proteinases and phospholipases. Important karyotype alterations were also observed. The overall data suggest that <em>C. albicans</em> CUG ambiguity is an important phenotypic diversity generator and highlight important and yet overlooked functional roles for genetic code alterations.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Reverting CUG identity from serine back to leucine</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The CUG identity alteration from leucine to serine was initiated 272±25My ago through a mutant serine tRNA (tRNA<sub>CAG</sub><sup>Ser</sup>), containing a 5′-CAG-3′ anticodon, which could decode CUG codons (see <a href="#s1">introduction</a>) <a href="#pone.0000996-Santos4">[26]</a>, <a href="#pone.0000996-Santos2">[22]</a>. Initially, this unique tRNA<sub>CAG</sub><sup>Ser</sup> competed with endogenous tRNA<sup>Leu</sup>, which decoded CUG codons as leucine <a href="#pone.0000996-Massey1">[30]</a>, creating a new situation where both leucines and serines were inserted at CUG positions on a genome wide scale (<a href="#pone-0000996-g001">Figure 1</a>). For reasons not yet fully understood, the wild type tRNA<sup>Leu</sup> disappeared from the <em>Candida</em> ancestor genome leaving CUG decoding exclusively to the mutant tRNA<sub>CAG</sub><sup>Ser</sup>. Disappearance of the tRNA<sup>Leu</sup> should have abolished the ambiguous status of CUG codons, however the tRNA<sub>CAG</sub><sup>Ser</sup> is recognized by both LeuRS and SerRS (see above), creating a serine tRNA that exists in 2 distinct forms, namely ser-tRNA<sub>CAG</sub><sup>Ser</sup> (charged with serine) and leu-tRNA<sub>CAG</sub><sup>Ser</sup> (charged with leucine). This ambiguous tRNA still exists in <em>C. albicans</em> <a href="#pone.0000996-Suzuki1">[25]</a>.</p>
<div class="figure" id="pone-0000996-g001"><div class="img"><a name="pone-0000996-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Reconstruction model for the <em>Candida</em> genetic code alteration.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The ancestor of <em>Candida</em> decoded the CUG codon as leucine using a single leucine tRNA (tRNA<sup>Leu</sup>). This situation changed dramatically with appearance 272±25My of a mutant serine tRNA that acquired a 5′-CAG-3′anticodon (tRNA<sub>CAG</sub><sup>Ser</sup>). The latter competed with the tRNA<sup>Leu</sup> for decoding of CUG codons, inserting both leucine and serine, at high level, at CUG positions, on a proteome wide scale. Such ambiguity decreased over time due to disappearance of the tRNA<sup>Leu</sup> gene, however charging of the tRNA<sub>CAG</sub><sup>Ser</sup> with leucine and serine prevented complete change of identity of the CUG codon from leucine to serine. In order to elucidate why CUG ambiguity was preserved in <em>C. albicans</em> and clarify whether CUG identity could be partially reverted from serine back to leucine, we have reconstructed the early stages of CUG identity change (high level of ambiguity) in <em>C. albicans</em> using <em>S. cerevisiae</em> tRNAs that decode CUG codons as leucine.</p>
<span>doi:10.1371/journal.pone.0000996.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We have re-created in <em>C. albicans</em> (CAI-4 strain) the high ambiguity status of CUG codons that existed 272±25 My ago in the <em>Candida</em> ancestor (<a href="#pone-0000996-g001">Figure 1</a>). For this, we have expressed <em>Saccharomyces cerevisiae</em> wild type and mutant tRNAs, which decode CUG codons as leucine, in <em>C. albicans</em> (<a href="#pone-0000996-g002">Figure 2A–D</a>). These tRNAs<sup>Leu</sup> competed with the novel tRNA<sub>CAG</sub><sup>Ser</sup> for CUG codons at the ribosome A-site, but were not lethal. We have hypothesized that such genetic manipulation would increase CUG ambiguity and could uncover phenotypes associated to the residual ambiguity (3%) <a href="#pone.0000996-Suzuki1">[25]</a> of CUG codons in <em>C. albicans</em>.</p>
<div class="figure" id="pone-0000996-g002"><div class="img"><a name="pone-0000996-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Transfer RNAs used in this study.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>In order to reconstruct the early stages of the CUG identity alteration in <em>C. albicans</em>, <em>S. cerevisiae</em> leucine tRNAs containing the anticodons UAG or CAG were expressed in C. <em>albicans</em>. A) The respective tRNA genes were cloned into plasmid pUA12, which is based on the <em>C. albicans</em> pRM1 vector. B) A leucine tRNA gene containing the near-cognate anticodon (5′-UAG-3′) for the CUG codon was used as a low decoding efficiency tRNA (pUA13). C) Two tRNA<sub>CAG</sub><sup>Leu</sup> genes, containing anticodons cognate for the CUG codon were used for higher CUG decoding efficiency, one contained G<sub>33</sub> (pUA14; medium decoding efficiency) and the other contained U<sub>33</sub> (pUA15; high decoding efficiency), in the anticodon-loop. D) The <em>S. cerevisiae</em> tRNA<sub>AGA</sub><sup>Ser</sup> gene was used as negative control (pUA16).</p>
<span>doi:10.1371/journal.pone.0000996.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Three <em>S. cerevisiae</em> tRNA<sup>Leu</sup> genes, namely a wild type tRNA<sub>UAG</sub><sup>Leu</sup> and two mutant tRNA<sub>CAG</sub><sup>Leu</sup> (containing U<sub>33</sub> or G<sub>33</sub> in the anticodon-loop), plus a control tRNA<sub>AGA</sub><sup>Ser</sup> (<a href="#pone-0000996-g002">Figure 2B–D</a>), were successfully expressed in <em>C. albicans</em> CAI-4 cells, as shown by Northern blot analysis of tRNAs fractionated by acidic-PAGE, which separates charged from uncharged tRNAs <a href="#pone.0000996-Varshney1">[31]</a> (<a href="#pone-0000996-g003">Figure 3A</a>). This is in line with previous experiments that showed that the identity elements of <em>C. albicans</em> and <em>S. cerevisiae</em> LeuRS and SerRS are identical <a href="#pone.0000996-OSullivan1">[32]</a>, <a href="#pone.0000996-OSullivan2">[33]</a>. Transformation efficiency of plasmids carrying the <em>S. cerevisiae</em> tRNA<sup>Leu</sup> genes was lower than that of control plasmids (<a href="#pone-0000996-g003">Figure 3B</a>) containing no tRNA gene or containing a cognate serine decoder tRNA (pUA16) (<a href="#pone-0000996-g002">Figure 2D</a>). In other words, the tRNA<sup>Leu</sup> were slightly toxic to <em>C. albicans</em>, which supported the hypothesis that they were fully functional and could incorporate leucine at CUG positions. Remarkably, clones that survived transformation showed no decrease in growth rate (<a href="#pone-0000996-g003">Figure 3C</a>), suggesting that they adapted to increased CUG ambiguity.</p>
<div class="figure" id="pone-0000996-g003"><div class="img"><a name="pone-0000996-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Expression of <em>S. cerevisiae</em> tRNA<sup>leu</sup> in <em>C. albicans.</em></span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>A) Aminoacylation <em>in vivo</em> in <em>C. albicans</em> of <em>S. cerevisiae</em> tRNA<sub>UAG/CAG</sub><sup>Leu</sup> and tRNA<sub>AGA</sub><sup>Ser</sup> was monitored by Acidic Page and Northern Blot analysis. For this, total tRNAs were extracted under acidic conditions from pUA13, pUA14, pUA15, and pUA16 clones and fractionated on an acidic polyacrylamide gel, as described in <a href="#s4">materials and methods</a>. These gels separated deacylated (-AA) from aminoacylated tRNAs (+AA), which were detected using a tDNA<sub>Leu/Ser</sub> probe labeled with [<sup>32</sup>P]. B) Transformation efficiencies of plasmids encoding tRNA<sup>Leu</sup>, which decoded the <em>C. albicans</em> serine CUG codons as leucine, was significantly lower that that of control plasmids (pUA12 and pUA16), indicating that the leucine tRNAs were slightly toxic. C) However, clones that survived the transformation procedure adapted readily to CUG ambiguity and showed growth rates similar to control clones (pUA12).</p>
<span>doi:10.1371/journal.pone.0000996.g003</span></div>

<h4>Ambiguous CUG decoding generated important phenotypic diversity</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Interestingly, expression of those <em>S. cerevisiae</em> tRNA<sup>Leu</sup> genes in <em>C. albicans</em> triggered morphogenesis in both solid and liquid media (<a href="#pone-0000996-g004">Figure 4</a>). The pUA15 clones displayed extensive morphological variation (<a href="#pone-0000996-g004">Figure 4B–C</a>) and produced highly heterogenous cell populations containing elongated-ovoid cells, pseudohypha and true hypha (not shown). Some pUA15 clones produced hypha that occupied sectors or entire colonies. Notably, morphological events that gave rise to these phenotypes happened spontaneously without external inducing factors. As expected, control pUA12 and pUA16 clones had homogeneous morphology and formed smooth-white colonies similar to those of untransformed <em>C. albicans</em> CAI4 (<a href="#pone-0000996-g004">Figure 4A</a>). Similar results were obtained with clones pUA13 and pUA14 (data not shown). Apart from morphogenesis, CUG ambiguity also induced phenotypic switching, which is a <em>C. albicans</em> phenotype characterized by reversible induction of opaque or myceliated sectors in white smooth colonies <a href="#pone.0000996-Soll1">[34]</a>. High frequency of phenotypic switching (63–88%) was obtained for all clones expressing <em>S. cerevisiae</em> leucine tRNAs (pUA13-15), but not for control clones (pUA12 and pUA16) (<a href="#pone-0000996-g004">Figure 4D</a>).</p>
<div class="figure" id="pone-0000996-g004"><div class="img"><a name="pone-0000996-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Ambiguous CUG decoding triggered morphogenesis and phenotypic switching.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) Smooth colony morphology of control clones growing on MM-uri-phloxin B (50µg/ml) agar plates. B) Ambiguous pUA15 clones formed long hyphae, even in absence of external inducers, just growing in MM-uri agar plates at 30°C. Similar results were obtained for pUA13 and pUA14 ambiguous clones (data not shown). D). Expression of <em>S. cerevisiae</em> tRNA<sup>Leu</sup> in <em>C. albicans</em> also induced phenotypic switching, which is characterized by transition between different cell-phase forms, namely white-opaque and myceliated-unmyceliated, giving rise to sectored colonies. D) Phenotypic switching was quantified by counting sectored colonies grown in MM-uri, after 7 days of incubation at 30°C. For each plasmid, up to 10 clones were plated and 3000 colonies were screened.</p>
<span>doi:10.1371/journal.pone.0000996.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>CUG ambiguity also increased cell adhesion in liquid and solid media (<a href="#pone-0000996-g005">Figure 5A</a>), and once more, this phenotype was exacerbated in pUA15 clones, as they displayed strong flocculation in liquid media (<a href="#pone-0000996-g005">Figure 5A</a>). Interestingly, more than 50% of the genes involved in adhesion contain CUG codons. For example, the <em>ALS</em> gene family which encodes cell-surface glycoproteins that mediate adhesion to host surfaces <a href="#pone.0000996-Hoyer1">[35]</a>, contain various CUG codons (3CUGs-<em>ALS2, ALS3, ALS8</em>; 4CUGs-<em>ALS4</em>; 5CUGs-<em>ALS1</em>; 11CUGs-<em>ALS6</em>; 12CUGs-<em>ALS9</em>; 18CUGs-<em>ALS7</em>). It is not yet clear whether the change of serine (polar) for leucine (hydrophobic) at CUG positions in the Als proteins is responsible for the flocculation and exacerbated agar adhesion observed in pUA15 clones. But, the strong adhesion phenotypes resulting from expression of <em>ALA1, EAP1</em> and members of the <em>C. albicans ALS</em> gene family, namely <em>ALS1</em> and <em>ALS5</em> in <em>S. cerevisiae</em>, support the hypothesis that the replacement of serines with leucines at CUG positions increases adhesion <a href="#pone.0000996-Fu1">[36]</a>–<a href="#pone.0000996-Li1">[41]</a>.</p>
<div class="figure" id="pone-0000996-g005"><div class="img"><a name="pone-0000996-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Increased CUG ambiguity resulted in higher hydrolytic activity and increased adhesion.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A) Highly ambiguous cells (pUA15) exhibited strong adhesion phenotypes both in solid and liquid media. Adhesion to the solid agar surface resulted from cell-cell and cell-agar adhesion. In liquid media, cells showed a strong flocculation phenotype and sedimented even when grown with agitation (30°C for 2 days). B–C) Cells transformed with pUA13-14 (data not shown) and with pUA15 plasmids, had higher SAP and phospholipase activity than control cells, as determined by hydrolysis of BSA and egg yolk, respectively. Hydrolytic activity was quantified by measuring precipitation zones formed around the colonies, corrected by the colony diameter, in order to obtain Pz values.</p>
<span>doi:10.1371/journal.pone.0000996.g005</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Finally, expression of <em>S. cerevisiae</em> tRNA<sup>Leu</sup> (pUA15) in <em>C. albicans</em> increased production of extracellular hydrolases, namely secreted aspartic proteinases (SAP) and phospholipases, as determined on agar plates supplemented with BSA and egg yolk, respectively (<a href="#pone-0000996-g005">Figure 5B,C</a>). Hydrolysis of these substrates lead to formation of an halo of precipitated peptides around the colonies, indicative of SAP and phospholipase production <a href="#pone.0000996-Ibrahim1">[42]</a>. Since adhesion, SAPs and phospholipases are important virulence attributes <a href="#pone.0000996-Naglik1">[43]</a>–<a href="#pone.0000996-Ghannoum1">[45]</a>, those phenotypes are relevant to <em>C. albicans</em> pathogenesis and indicate, for the first time, that codon ambiguity and the <em>Candida</em> genetic code alteration may play a role in infection. It will be most interesting to put this hypothesis to experimental test as a positive result would clearly show that the negative impact of codon decoding ambiguity could be overcome by high selective potential of novel phenotypic diversity.</p>


<h4>Gene expression alterations</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In order to elucidate how CUG ambiguity generated phenotypic diversity we have carried out gene expression profiling of <em>C. albicans</em> cells expressing the <em>S. cerevisiae</em> tRNA<sup>Leu</sup> (pUA15 clones), using DNA microarrays. However, the diversity of phenotypes and permanent switching between phenotypes in culture (<a href="#pone-0000996-g004">Figure 4</a>) created cell variability and culture instability that prevented us from obtaining meaningful mRNA expression profiles (data not shown). To overcome this limitation, we have tried to stabilize some of the phenotypes on solid media, but, once again, this turned out to be very difficult to achieve due to very high switching between morphological forms. As the mRNA profiles represented average values of a variety of phenotypes, reproducibility was low and most genes failed to pass our microarray statistical filters. Despite this, we were able to detect 4 genes whose expression was consistently altered in the microarray data sets and were relevant to the pUA15 phenotypes, namely the hypha-specific G1 cyclin-related protein <em>HGC1</em> (2.64 fold), the hypha specific gene <em>HWP1</em> (41,76 fold), the white-opaque switch regulator <em>WOR1</em> (7 fold) and the transcription factor <em>MCM1</em> regulator of hyphal growth (-1.84 fold) (<a href="#pone-0000996-g006">Figure 6</a>). Expression of these genes was confirmed by Real Time PCR, as described in <a href="#s4">materials and methods</a>.</p>
<div class="figure" id="pone-0000996-g006"><div class="img"><a name="pone-0000996-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Increased CUG ambiguity up-regulated morphogenesis genes.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Cells of pUA15 clones showed significant up-regulation of the <em>WOR1</em> (7.0±2.5) gene and hyphal-specific genes <em>CaHWP1</em> (41.76±9.96) and <em>HGC1</em> (2.64±1.12). Since <em>WOR1</em> increases the frequency of the white-opaque transition the very high percentage of opaque cells found in transformed clones may be a consequence of <em>WOR1</em> up-regulation. On the other hand, expression of the hypha-specific genes, <em>CaHWP1</em> and <em>CaHGC1,</em> supported the hypothesis that morphogenesis and hyphal growth triggered by CUG ambiguity resulted from expression of morphogenesis regulators. Induction of the <em>CaHWP1</em> gene was accompanied by repression of the <em>CaMCM1</em> (−1.84±0.44) gene, which controls cell morphology through the recruitment of other morphogenesis regulatory factors.</p>
<span>doi:10.1371/journal.pone.0000996.g006</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The <em>WOR1</em> gene (7 fold up-regulated) is a master regulator of white-opaque switching and its expression induced the white-opaque phase transition <a href="#pone.0000996-Zordan1">[46]</a>, <a href="#pone.0000996-Huang1">[47]</a>. This provides a likely explanation for the strong induction of white-opaque switching in pUA15 clones, whose cells switch at high frequency. The <em>HWP1</em> gene (41,76±9,96 up-regulated) encodes a hyphal-specific cell wall mannoprotein, which is a substrate for mammalian transglutaminases and plays a crucial role in adhesion of <em>C. albicans</em> to epithelial cells <a href="#pone.0000996-Staab1">[48]</a>. Interestingly, <em>HWP1</em> expression was correlated with <em>MCM1</em> repression (-1.84 fold), confirming previous results that showed that expression of one of these genes represses expression of the other <a href="#pone.0000996-Rottmann1">[49]</a>. <em>MCM1</em> plays an important role in cell morphology and its expression is auto-regulated by a feedback control mechanism. Since both low and high Mcm1p levels lead to hyphal formation, it may act as a recruiting regulatory factor for morphogenesis in <em>C. albicans</em>. Depletion of Mcm1p induced transcription of <em>HWP1</em>, however no Mcm1p binding site was found in the <em>HWP1</em> promotor and it is not yet clear how the former activates transcription of the later. Finally, the <em>HGC1</em> gene was also up-regulated (2.64 fold). This gene is crucial for hyphal formation by promoting apical bud elongation and it is strongly induced during morphogenesis. It is not required for expression of hypha-specific genes (HSGs), like <em>HWP1,</em> but is positively regulated by the cAMP/PKA pathways and repressed by Tup1 and Nrg1 morphogenesis repressors <a href="#pone.0000996-Zheng1">[50]</a>.</p>


<h4>Increased CUG ambiguity increased <em>C. albicans</em> ploidy</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Up-regulation of the <em>WOR1</em> gene and the high percentage of opaque cells (mating competent) observed in pUA15 clones prompted the question of whether CUG ambiguity would induce mating in <em>C. albicans</em>. Indeed, pUA15 opaque cells formed conjugation tubes and mating figures which were readily observed by optical microscopy (<a href="#pone-0000996-g007">Figure 7A</a>). Furthermore, flow cytometry analysis of pUA15 clones identified sub-populations of 4N, 6N and 8N cells (<a href="#pone-0000996-g007">Figure 7B</a>), supporting the hypothesis that ambiguous <em>C. albicans</em> mated at high frequency. Since <em>C. albicans</em> is a diploid organism with a sexual life cycle <a href="#pone.0000996-Hull1">[51]</a>, and considering that its mating locus (MTL) is heterozygotic in mating incompetent white cells (MTLa/α) and homozygotic in mating competent opaque cells (MTLa/a or MTLα/α) <a href="#pone.0000996-Miller1">[52]</a>, the latter were isolated from pUA15 clones and were analysed for MTL homozygozity. All clones analysed were MTLα/α, suggesting that expression of tRNA<sup>Leu</sup> (pUA15) induced biased MTLα/α homozygoty (<a href="#pone-0000996-g007">Figure 7C</a>), creating an excess of MTLα/α over MTLa/a cells. These results were in good agreement with up-regulation of the <em>WOR1</em> gene as its expression is controlled by the MTLa1-α2 heterodimer, which, in turn, controls white-opaque transition and mating competence <a href="#pone.0000996-Zordan1">[46]</a>, <a href="#pone.0000996-Huang1">[47]</a>. That is, switching from white-to-opaque phase required conversion of heterozygotic MTL to homozygotic MTL inducing mating competence <a href="#pone.0000996-Miller1">[52]</a>. Therefore, it is likely that MTL homozygoty induced by tRNA<sup>Leu</sup> derepressed the <em>WOR1</em> gene, which, in turn, promoted the white-to-opaque transition and mating.</p>
<div class="figure" id="pone-0000996-g007"><div class="img"><a name="pone-0000996-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Ambiguous CUG decoding induced mating.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A) In pUA15 transformed cells, the number of opaque cells was very high. This phenotype is most likely explained by up-regulation of the white-opaque master regulator <em>WOR1</em> gene (<a href="#pone-0000996-g006">Figure 6</a>). Since <em>C. albicans</em> white cells (most common form) are mating incompetent and opaque cells (rare cells) are mating competent, we have verified whether pUA15 opaque cells formed conjugation tubes and mating figures in liquid culture. Both were readily observed using optical microscopy (white arrows). B) In order to confirm that mating occurred, the DNA content of pUA15 cells was analyzed by flow cytometry. Since <em>C. albicans</em> is diploid, 4N cells were expected. Surprisingly, higher ploidies (6N, 8N) were also observed suggesting that the cultures had significant number of aneuploid and poliploid cells. C) White to opaque transition and mating induced by CUG ambiguity occurs due to MTL homozygosity. Since mating requires transition from the heterozygotic mating locus (MTL a/α) found in white cells, to the homozygotic configuration (MTLa/a or MTLα/α) found in opaque cells, detection of αα/αα cells supported the hypothesis that CUG ambiguity induced mating.</p>
<span>doi:10.1371/journal.pone.0000996.g007</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The presence of cells with high ploidy (6N and 8N) in pUA15 opaque cultures (<a href="#pone-0000996-g007">Figure 7B</a>), prompted us to monitor ploidy variability. Most clones showed increased ploidy ranging from 4N to 8N (data not shown), however very large cells with remarkably high ploidy (&gt;64N) were also observed (<a href="#pone-0000996-g008">Figure 8A</a>). In general, ploidy variability between clones was higher than previously described <a href="#pone.0000996-Iwaguchi1">[53]</a>, <a href="#pone.0000996-Suzuki2">[54]</a> and raised the question of whether those high chromosome numbers could return to 2N over time. Since wild type <em>C. albicans</em> undergoes a process of chromosome loss after mating, which decreases its ploidy from 4N back to 2N <a href="#pone.0000996-Bennett1">[55]</a>–<a href="#pone.0000996-Hull2">[57]</a>, we hypothesized that high ploidy in pUA15 clones could also be reduced. To test this hypothesis, clones with very high ploidy (large cells) were successively re-plated on fresh agar plates and their ploidy was monitored over time by flow cytometry. Indeed, ploidy reverted to 2N or 4N after several passages on fresh agar, confirming the above hypothesis (<a href="#pone-0000996-g008">Figure 8B</a>).</p>
<div class="figure" id="pone-0000996-g008"><div class="img"><a name="pone-0000996-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000996" data-uri="info:doi/10.1371/journal.pone.0000996.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000996.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000996.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000996.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000996.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Ambiguous CUG decoding induced karyotype rearrangements and ploidy-shift.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>A) In ambiguous cell lines (pUA15) polyploidy was predominant and very high ploidy was often detected (&gt;32N). Aneuploidy was also observed (6N). B) However, after plating cells several consecutive times on fresh agar chromosome numbers were reduced indicating that most cells returned to low ploidy (2N or 4N). Ploidy reduction after mating normally occurs by chromosome loss in <em>C. albicans</em> and it is likely that such mechanism also played a role in ploidy reduction in pUA15 transformed cells. C) CUG ambiguity also promoted extensive rearrangements of the R-chromosome (highlighted in white circles). Chromosomes were separated by PFGE on 0.6% agarose gels under the following conditions: 120–300 s for 24h at 80 V, then 420–900 s for 48 h at 80 V. The numbers 1–7 and R identify <em>C. albicans</em> chromosomes.</p>
<span>doi:10.1371/journal.pone.0000996.g008</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The above results also prompted us to check whether transformation of <em>C. albicans</em> with the pUA15 plasmid destabilized its karyotype. The <em>C. albicans</em> karyotype is characterized by frequent chromosome rearrangements, in particular of the chromosome R, which contains rDNA cistrons <a href="#pone.0000996-Lasker1">[58]</a>, <a href="#pone.0000996-Magee2">[59]</a>. We wondered whether the tRNA<sup>Leu</sup> would affect rRNA metabolism and protein synthesis. Indeed, various rearrangements of the R-chromosome were readily observed (<a href="#pone-0000996-g008">Figure 8C</a>). In particular, the size of R-chromosomes increased in some clones, decreased in others and these rearrangements affected most cells (<a href="#pone-0000996-g008">Figure 8C</a>). It will now be most interesting to investigate whether CUG ambiguity affects ribosome assembly and the rate of protein synthesis.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The identity of CUG codons is variable in the genus <em>Candida</em>. Indeed, <em>C. glabrata</em> decodes CUGs as leucine, <em>C. cylindracea</em> changed their identity completely to serine and the other <em>Candida</em> species decode them ambiguously <a href="#pone.0000996-Sugita1">[13]</a>, <a href="#pone.0000996-Suzuki1">[25]</a>, <a href="#pone.0000996-Santos5">[60]</a>, <a href="#pone.0000996-OSullivan3">[61]</a>. These differences in CUG decoding arose from differences in the structure of the tRNA<sub>CAG</sub><sup>Ser</sup>, which is the only cognate tRNA for CUG codons in <em>Candida</em>. Indeed, the various tRNA<sub>CAG</sub><sup>Ser</sup> have identity determinants for both SerRS and LeuRS <a href="#pone.0000996-Santos1">[1]</a>. For example, the <em>C. albicans</em> tRNA<sub>CAG</sub><sup>Ser</sup> contains identity elements for the LeuRS, namely m<sup>1</sup>G<sub>37</sub> and the middle base of the anticodon (A<sub>35</sub>), but the discriminator base at position 73 (G<sub>73</sub>) is specific for SerRS and not for LeuRS (A<sub>73</sub> discriminator) <a href="#pone.0000996-Breitschopf1">[62]</a>. This base is critical for the recognition of tRNAs by aminoacyl-tRNA synthetases (aaRSs) and one would expect that the LeuRS would not recognize tRNAs with G<sub>73</sub>. Therefore, charging the tRNA<sub>CAG</sub><sup>Ser</sup> with leucine <a href="#pone.0000996-Suzuki1">[25]</a> suggests that the <em>Candida</em> LeuRS may have evolved a unique mode of recognition of its cognate tRNA<sup>Leu</sup>. Interestingly, the presence of a unique guanosine in the turn of the anticodon-loop (G-turn) of the tRNA<sub>CAG</sub><sup>Ser</sup> (G<sub>33</sub>), a conserved position occupied by U<sub>33</sub> (U-turn), reduced leucylation efficiency of the tRNA<sub>CAG</sub><sup>Ser</sup> <a href="#pone.0000996-Suzuki1">[25]</a>. That is, recognition of the ancestral tRNA<sub>CAG</sub><sup>Ser</sup> by the LeuRS was efficient and G<sub>33</sub> acted as a leucine identity anti-determinant. The reason for G<sub>33</sub> selection is not yet clear, but one possibility is that it may have decreased the toxicity of the mutant tRNA<sub>CAG</sub><sup>Ser</sup> during the early stages of CUG identity alteration <a href="#pone.0000996-Perreau1">[27]</a>, <a href="#pone.0000996-Santos6">[63]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The dual recognition of the tRNA<sub>CAG</sub><sup>Ser</sup> by the LeuRS and SerRS indicates that there are two forms of the tRNA<sub>CAG</sub><sup>Ser</sup> in the cytoplasm of <em>C. albicans</em>, namely Ser-tRNA<sub>CAG</sub><sup>Ser</sup> and Leu-tRNA<sub>CAG</sub><sup>Ser</sup>, which are charged with serine and leucine, respectively. These 2 tRNAs generate ambiguity at CUG codons since they compete with each other for CUGs at the ribosome A-site. Interestingly, such CUG ambiguity was not constant over the 272±25My of evolution of the genetic code alteration (see <a href="#s1">introduction</a>) <a href="#pone.0000996-Massey1">[30]</a>. It was high during the early stages of CUG identity change (when the tRNA<sub>CAG</sub><sup>Ser</sup> gene appeared), and decreased gradually due to elimination of the tRNA<sup>Leu</sup> gene from the genome of the <em>Candida</em> ancestor <a href="#pone.0000996-Massey1">[30]</a>. Reconstruction of the high level of CUG ambiguity, which existed during the early stages of CUG identity alteration, provided the first insight on how the genetic chaos created at the onset of CUG identity change may have generated phenotypic diversity of evolutionary and adaptive relevance. In extant <em>C. albicans</em>, morphological variation alters cell surface antigens and it is likely that this pathogen uses its sophisticated capacity to generate morphological variation as a strategy to escape the immune system. Furthermore, secreted proteinases and phospolipases are important <em>C. albicans</em> virulence attributes and their increased activity in pUA15 clones may also be relevant to virulence. It will now be interesting to engineer stable high level mistranslation in <em>C. albicans</em> and test the virulence of the recombinant strains in mice models.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The high phenotypic diversity of pUA13, pUA14 and pUA15 clones and constant transition between phenotypes prevented us from carrying out a detailed study of the impact of CUG ambiguity on gene expression. However, the strong up-regulation of the hyphal specific gene (<em>HWP1</em>) and of the master regulator of the white-opaque transition (<em>WOR1</em>), may explain, at least in part, some of the phenotypes observed. As a hyphal specific gene (HSG), high <em>HWP1</em> expression, induced by CUG ambiguity, is in agreement with spontaneous morphogenesis events that generate filamentous cell populations. Furthermore, <em>HWP1</em> up-regulation may also increase adhesion since Hwp1p is a glycosylphosphatidylinositol cell wall adhesin (GPI-CWP) and mediates attachment of <em>C. albicans</em> cells to human endothelial and epithelial cells <a href="#pone.0000996-Sundstrom1">[44]</a>. The observed adhesion phenotype (<a href="#pone-0000996-g005">Figure 5A</a>) may have also resulted from the combined up-regulation of <em>HWP1</em> and <em>WOR1</em> genes, since the later, previously known as <em>EAP2</em> (enhanced adhesion to polystyrene), mediates <em>C. albicans</em> and <em>S. cerevisiae</em> adhesion to polystyrene and epithelial cells <a href="#pone.0000996-Li1">[41]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Apart from its putative role in adhesion, up-regulation of <em>WOR1</em> may also explain the white-opaque phenotype since Wor1p is a transcriptional regulator of white-opaque switching <a href="#pone.0000996-Zordan1">[46]</a>, <a href="#pone.0000996-Huang1">[47]</a>. Indeed, Wor1p is present in very low amounts in white cells and accumulates in opaque cells. <em>WOR1</em> is repressed by the heterodimer MTL a1/α2 and is activated by Wor1p itself when cells become homozygous MTL aa/aa or MTL αα/αα. Increased accumulation of Wor1p triggered white-opaque switching and repressed its own transcription by a feedback regulatory mechanism <a href="#pone.0000996-Zordan1">[46]</a>, <a href="#pone.0000996-Huang1">[47]</a>. The up-regulation of the <em>HGC1</em> gene (<a href="#pone-0000996-g006">Figure 6</a>), which is a hypha-specific gene encoding a G1 cyclin-related protein, that plays a role in hyphal morphogenesis, was also significant. Since it is transcriptionally regulated by hypha-inducing rather than cell cycle signals <a href="#pone.0000996-Zheng1">[50]</a>, its up-regulation in ambiguous cells supports the hypothesis that it functions independently of other cell cycle cyclins.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>CUG ambiguity also generated karyotype alterations and formation of polyploids and aneuploids. Ploidy-shift has been associated with chromosomal rearrangements <a href="#pone.0000996-Iwaguchi1">[53]</a>, <a href="#pone.0000996-Suzuki2">[54]</a>, <a href="#pone.0000996-Selmecki1">[64]</a> which also generates morphological variation <a href="#pone.0000996-RustchenkoBulgac1">[65]</a>–<a href="#pone.0000996-Suzuki3">[67]</a>, antifungal resistance <a href="#pone.0000996-Perepnikhatka1">[68]</a>, adaptation to alternative carbon sources <a href="#pone.0000996-Rustchenko1">[69]</a>, <a href="#pone.0000996-Janbon1">[70]</a>, or even with homozigosity of chromosome-V <a href="#pone.0000996-Legrand1">[71]</a>. This suggests that part of the phenotypic diversity observed in pUA13, pUA14 and pUA15 clones may have resulted from karyotype alterations, or from a combination of up-regulation of the genes described above and karyotype destabilization. Since some clones did not have karyotype alterations (<a href="#pone-0000996-g008">Figure 8C</a>), but still displayed phenotypic variability, it is likely that the former does not play a main role in expression of phenotypic diversity. However, one should not exclude the hypothesis that genome destabilization contributes to exposure of hidden phenotypes through ambiguous CUG decoding. Finally, in <em>E. coli</em>, genetic code ambiguity induced by misreading tRNAs triggered translational stress-induced mutagenesis (TSM), due to synthesis of error-prone DNA polymerases <a href="#pone.0000996-Dorazi1">[72]</a>. This general mistranslation resulted in increased global error rates during DNA replication leading to heritable genetic changes. Since these hypermutagenic phenotypes result in rapid adaptation, an unexpected consequence of genetic code ambiguity (and genetic code alterations) is acceleration of genome variability and fast evolution of new phenotypes. This may explain evolution of the high plasticity of <em>Candida</em> morphology and the very high heterozigosity of its genome <a href="#pone.0000996-Jones1">[73]</a>.</p>

<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Genetic code alterations pose important new biological questions whose answers remain elusive. It is now clear that a number of them evolved through codon decoding ambiguity, required significant structural change of protein synthesis machineries and reprogrammed codon usage <a href="#pone.0000996-Santos1">[1]</a>, <a href="#pone.0000996-Knight2">[5]</a>, <a href="#pone.0000996-Massey1">[30]</a>. However, codon decoding ambiguity is toxic, decreases fitness and may ultimately lead to cell death, as is the case in multicellular organisms <a href="#pone.0000996-Nangle1">[19]</a>, <a href="#pone.0000996-Lee2">[74]</a>. For these reasons, evolution of genetic code alterations through codon ambiguity is most puzzling. This study unveiled possible ways of overcoming the negative impact of codon ambiguity by high selective potential generated through generation of phenotypic diversity. The molecular mechanism used to generate such phenotypic diversity remains to be elucidated. However, the adaptive potential of the unveiled phenotypes strongly suggests that CUG ambiguity may have been preserved in <em>Candida</em> spp. as a novel generator phenotypic diversity. We have previously shown that codon ambiguity in <em>S. cerevisiae</em> creates a competitive edge under stress by inducing the general stress response and pre-adapting cells to sudden environmental changes <a href="#pone.0000996-Santos7">[75]</a>. Therefore, the toxicity of codon ambiguity is not an impediment to codon identity redefinition, supporting the hypothesis that codon misreading plays a critical role in the evolution of genetic code alterations and genetic code expansion.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Escherichia coli strain JM109 (recA1 SupE44 endA1 hsdR17 gyrA96 relA1 thi Δ(Lac-proAB) F'[traD36 proAB-lacI lacZ ΔM15] was used has a host for all DNA manipulations. Candida albicans CAI4 (ura3Δ::imm434/ura3::imm434) was grown at 30°C in YEPD (2% glucose; 1% yeast extract, 1% peptone). After transformation with pUA12, pUA13, pUA14, pUA15, and pUA16 plasmids, cells were grown in minimal medium lacking uridine (MM-uri) (0.67% yeast nitrogen base without amino acids, 2% glucose, 2% agar and 100µg/ml of the required amino acids). Solid MM-uri was supplemented with phloxin B (50µg/ml) in order to detect macroscopic growth of opaque cells.</p>


<h4>Plasmids</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A multi-cloning site was inserted (<em>NruI/EcoRV</em>) in the low-copy <em>C. albicans</em> vector pRM1 <a href="#pone.0000996-Pla1">[76]</a>. The resulting vector was named pUA12. For heterologous expression of the <em>S. cerevisiae</em> tRNAs genes in CAI4, a genomic DNA fragment containing <em>S. cerevisiae</em> tRNA gene (700 bp), previously amplified by PCR, was cloned into the multi cloning site of pUA12 plasmid. <em>S. cerevisiae</em> Leu-tRNA<sub>UAG</sub> and a 300 bp flanking region, upstream and downstream of the gene, was amplified with the following set of primers: 5′-CCGCTCGAGCGGCGACTGTCCAGACTTAGTAAAG CT-3′ and 5′-GCTCTAGAGCCCGCTGTCGCCAGCGTTAGC-3′. Genomic DNA containing <em>S. cerevisiae</em> tRNA<sub>GAG</sub><sup>Leu</sup> gene was used as a template for PCR amplification using the forward primers: 5′-GCTATGGGCCCGCCTCCGGGTAGTTGCAACGGTACTC​TGGCCGAGTGGTCTAAGGCG-3′ and 5′-GCTATGGGCCCTAGTTGCAACGG TATCTGGCCGAGTGGTCTAAGGCGTCAGGTTCAGGTCC-3​′;and reverse primer 5′-ATGCATAAAAACAAAATTTGTTGAAA-3′. These primers introduced a mutation in the first position of the anticodon changing it from 5′-GAG-3′ to 5′-CAG-3′ and allowed insertion of G or T at position 33 of the tRNA anticodon-loop. Upstream of this gene, a 250 bp fragment with the same sequence of the 5′ flanking <em>C. albicans</em> Ser-tRNA<sub>CAG</sub> gene, was also inserted. This fragment was amplified by PCR from a <em>XhoI/ApaI</em> genomic DNA fragment, with the following primers: 5′-CCGCTCGAGCGGGTA TGCAATCGTTGTCTGTAATGTA-3′ and 5′-GCTATGGGCCCAAGCACAAA TGGTTATGACAATTG ATG-3′. The pUA16 control plasmid was constructed by inserting a <em>XhoI/AvaIII</em> genomic DNA fragment (600 bp), containing <em>S. cerevisiae</em> Ser-tRNA<sub>AGA</sub> gene amplified by PCR with the following pair of primers 5′-CCGC TCGAGCGGGAGGATTCCTATATCCTTGAGGAG-3′ and 5′-GGCTCGATGCATG CCAGGAAGAAATACACTGC-3′, into the multicloning site. All DNA amplifications were carried out using a Mastercycle gradient (Eppendorf) and standard PCR protocols.</p>


<h4>Plasmid transformation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Transformation of <em>E. coli</em> was carried out as described by <a href="#pone.0000996-Sambrook1">[77]</a> CAI4 transformation was performed by the spheroplast method as described in the Manual for Preparation and Transformation of <em>Pichia pastoris</em> Spheroplasts (version A, Invitrogen).</p>


<h4>Northern Blot analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Acidic Northern Blot analysis was performed as described by Santos <em>et al.</em> (1996). For total tRNA extractions, 250 ml cultures grown overnight in YEPD or MM-uri medium were harvested at an OD<sub>600</sub> of 0.7–0.9 and the pellets were frozen at −70°C overnight. Cells were resuspended in 5 ml lysis buffer (0.3M sodium acetate, pH 4.5, 10 mM EDTA), 1 vol. phenol equilibrated with sodium citrate pH 4.5 and baked glass beads. Cell suspension was vigorously shaken &gt;30 seconds and incubated on ice for periods longer than 30 seconds, this procedure was repeated 8 times <a href="#pone.0000996-WeygandDurasevic1">[78]</a>. The aqueous phase containing RNAs was separated from the phenolic phase by centrifugation at 3200×<em>g</em> for 20 min at 4°C and then transferred to a new Falcon tube and re-extracted with fresh phenol. Aqueous phases containing RNAs were harvested by centrifugation at 3200×<em>g</em> for 20 min at 4°C and applied to a 20 ml DEAE-cellulose column equilibrated with 0.1 M sodium acetate pH 4.5. tRNAs were eluted with 0.1 M sodium acetate/1 M sodium chloride and precipitated with 2.5 vol. ethanol, resuspended in 10 mM sodium acetate pH 5.0/1 mM EDTA, and stored a −20°C. The deacylated tRNAs were obtained by incubation at 37°C for 1 h in 1 M Tris pH 8, 1 mM EDTA buffer <a href="#pone.0000996-Santos4">[26]</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>tRNAs were fractionated at 4°C in 7.5% acrylamidde-8 M urea (30 cm long, 0.8 mm thick), buffered with 0.1 M sodium acetate pH 5.0. The 7.5% acidic gels were run at 300 V until bromophenol blue dye reached the bottom <a href="#pone.0000996-Varshney1">[31]</a>. Fractionated tRNAs were transferred to nitrocellulose membranes (Hybond N, Amersham). Membranes were pre-hybridized for 6 h in a Hybridization oven at 50°C in 50% formamide, 5×SSC, 1% SDS, 0.04% Ficoll, 0.04% polyvinylpyrrolidone and 250 μg/ml sheared salmon sperm DNA <a href="#pone.0000996-Heitzler1">[79]</a>. Hybridization was performed overnight in the above buffer using probes labelled with [α-<sup>32</sup>P]dCTP by PCR, using standard protocols <a href="#pone.0000996-Innis1">[80]</a>, except that the amount of dCTP was reduced from 100 to 50 mM and 5 nmol (30 μCi) 6000 Ci/mmol [α-<sup>32</sup>P]dCTP was added to the reaction mixture. In order to decrease the background level of free radioactivity, 50 PCR cycles were performed to decrease the amount of non-incorporated [α-<sup>32</sup>P]dATP. Membranes were washed at low stringency in 1×SSC, 0.5% SDS at 50°C or at high stringency in 0.1×SSC, 0.5% SDS at 65°C for 1 h. The membranes were exposed overnight with intensifying screens and developed using a Molecular Imager FX (Biorad).</p>


<h4>Switching frequencies and phenotypic characterization</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p><em>C. albicans</em> grown overnight at 30°C in MM-uri were serially diluted to 1000 cells per ml. Approximately 50 cells were plated onto fresh agar plates and then allowed to grow at 30°C for 7 days in a humidified incubator to prevent drying of the agar surface. Sectored colonies displaying atypical colonies were scored and the data was analyzed for statistical significance using ANOVA. Colonies were photographed using a Stemi 2000-C dissecting microscope equipped with AxioCam HRc camera and Axio Vision Software from Zeiss. Cells were photographed using a Zeiss MC80 Axioplan 2 light microscope.</p>


<h4>Real Time RT-PCR</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Total RNA was prepared from <em>C. albicans</em> using hot acid phenol <a href="#pone.0000996-Kohrer1">[81]</a>. First-strand cDNA synthesis was carried out using the Superscript II RT kit from Invitrogen and its quantification was carried out in an Applied Biosystems 7500 Real Time PCR system using the SYBR Green I dye quantification assay (Power SYBR Green PCR master Mix). Primer concentrations were tested (0,2 to 0,4 µM) to ensure the lowest threshold cycle (C<sub>T</sub>) and the highest signal magnitude against the target template and to ensure non-specific product formation, resulting from primer dimmerization. After amplification, reactions were checked for presence of non-specific products through dissociation curve analysis. Each gene was quantified using 9 replicas of both control (pUA12) and ambiguous (pUA15) clones and a mean value was calculated. Outliers were rejected using critical values of Dixon's “Q” parameter at 95% confidence level <a href="#pone.0000996-Rorabacher1">[82]</a>.</p>


<h4>Determination of extra cellular hydrolytic activity</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p><em>C. albicans</em> strains were screened for the production of extra cellular phospholipase and secreted aspartic proteinase activity by growing cells on MM-uri agar supplemented with 10% egg yolk (Merck) and 10% bovine albumin (Sigma), respectively. A 3×2 µl suspension of 10<sup>7</sup>cells/ml in PBS was plated on the surface of the agar medium and left to dry at room temperature. The culture was then incubated at 30°C for 3 days, after which the diameter of the colony and the precipitation zone around the colony was determined. Three different clones of pUA12, pUA13, pUA14 and pUA15 transformed cells were tested twice. The experiment was carried out on two different occasions. The extra cellular hydrolase activity was calculated using the formula [(1/Pz)-1], where Pz value represents the hydrolyse zone, i. e., the cloudy-zone-around-plus-colony diameter divided by the colony diameter <a href="#pone.0000996-Samaranayake1">[83]</a>. Data obtained was submitted to an ANOVA statistical test.</p>


<h4>Genome analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>DNA content of <em>C. albicans</em> cells was determined using FACS analysis <a href="#pone.0000996-Fortuna1">[84]</a>. Karyotype analysis was performed using Pulsed-Field Gel Electrophoresis (PFGE) <a href="#pone.0000996-Chu1">[85]</a>.</p>


<h4>MTL analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>PCR analysis of the MTL configuration was carried out in pUA12 and pUA15 cells, through amplification of <em>oBPα</em> and <em>MTLa1</em> genes, respectively, using the following pairs of primers: 5′ GTGGTCAATGGAGCTGATAC 3′and 5′ ACATGTGGTCGCCCAACTCC 3′; 5′ TTGAAGCGTGAGAGGCAGGAG 3′ and 5′ ATCAATTCCCTTTCTCTTCGATTAGG 3′.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are most grateful to Jorge Rino for helping with the light microscopy studies, Alexander Johnson for providing oBP primers and Concha Gil for the pRM1 plasmid.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MS. Performed the experiments: GM RR IM MS DM. Analyzed the data: MS GM IM LC. Contributed reagents/materials/analysis tools: MS. Wrote the paper: MS IM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000996-Santos1" id="pone.0000996-Santos1"></a>Santos MA, Moura G, Massey SE, Tuite MF (2004) Driving change: the evolution of alternative genetic codes. Trends Genet  20: 95–102.  <ul class="find" data-citedArticleID="1001876" data-doi="10.1016/j.tig.2003.12.009"><li><a href="http://dx.doi.org/10.1016/j.tig.2003.12.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Driving+change%3A+the+evolution+of+alternative+genetic+codes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Driving+change%3A+the+evolution+of+alternative+genetic+codes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000996-Miranda1" id="pone.0000996-Miranda1"></a>Miranda I, Silva R, Santos MA (2006) Evolution of the genetic code in yeasts. Yeast  23: 203–213.  <ul class="find" data-citedArticleID="1001838"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+the+genetic+code+in+yeasts.&amp;auth=&amp;atitle=Evolution+of+the+genetic+code+in+yeasts." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+genetic+code+in+yeasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+genetic+code+in+yeasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000996-Knight1" id="pone.0000996-Knight1"></a>Knight RD, Landweber LF, Yarus M (2001) How mitochondria redefine the code. J Mol Evol  53: 299–313.  <ul class="find" data-citedArticleID="1001810" data-doi="10.1007/s002390010220"><li><a href="http://dx.doi.org/10.1007/s002390010220" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+mitochondria+redefine+the+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+mitochondria+redefine+the+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000996-Castresana1" id="pone.0000996-Castresana1"></a>Castresana J, Feldmaier-Fuchs G, Paabo S (1998) Codon reassignment and amino acid composition in hemichordate mitochondria. Proc Natl Acad Sci U S A  95: 3703–3707.  <ul class="find" data-citedArticleID="1001760" data-doi="10.1073/pnas.95.7.3703"><li><a href="http://dx.doi.org/10.1073/pnas.95.7.3703" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+reassignment+and+amino+acid+composition+in+hemichordate+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+reassignment+and+amino+acid+composition+in+hemichordate+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000996-Knight2" id="pone.0000996-Knight2"></a>Knight RD, Freeland SJ, Landweber LF (2001) Rewiring the keyboard: evolvability of the genetic code. Nat Rev Genet  2: 49–58.  <ul class="find" data-citedArticleID="1001812" data-doi="10.1038/35047500"><li><a href="http://dx.doi.org/10.1038/35047500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rewiring+the+keyboard%3A+evolvability+of+the+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rewiring+the+keyboard%3A+evolvability+of+the+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000996-Lovett1" id="pone.0000996-Lovett1"></a>Lovett PS, Ambulos NP Jr, Mulbry W, Noguchi N, Rogers EJ (1991) UGA can be decoded as tryptophan at low efficiency in Bacillus subtilis. J Bacteriol  173: 1810–1812.  <ul class="find" data-citedArticleID="1001828"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=UGA+can+be+decoded+as+tryptophan+at+low+efficiency+in+Bacillus+subtilis.&amp;auth=&amp;atitle=UGA+can+be+decoded+as+tryptophan+at+low+efficiency+in+Bacillus+subtilis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UGA+can+be+decoded+as+tryptophan+at+low+efficiency+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UGA+can+be+decoded+as+tryptophan+at+low+efficiency+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000996-Caron1" id="pone.0000996-Caron1"></a>Caron F, Meyer E (1985) Does Paramecium primaurelia use a different genetic code in its macronucleus? Nature  314: 185–188.  <ul class="find" data-citedArticleID="1001758" data-doi="10.1038/314185a0"><li><a href="http://dx.doi.org/10.1038/314185a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Does+Paramecium+primaurelia+use+a+different+genetic+code+in+its+macronucleus%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Does+Paramecium+primaurelia+use+a+different+genetic+code+in+its+macronucleus%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000996-Harper1" id="pone.0000996-Harper1"></a>Harper DS, Jahn CL (1989) Differential use of termination codons in ciliated protozoa. Proc Natl Acad Sci U S A  86: 3252–3256.  <ul class="find" data-citedArticleID="1001784" data-doi="10.1073/pnas.86.9.3252"><li><a href="http://dx.doi.org/10.1073/pnas.86.9.3252" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+use+of+termination+codons+in+ciliated+protozoa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+use+of+termination+codons+in+ciliated+protozoa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000996-SanchezSilva1" id="pone.0000996-SanchezSilva1"></a>Sanchez-Silva R, Villalobo E, Morin L, Torres A (2003) A new noncanonical nuclear genetic code: translation of UAA into glutamate. Curr Biol  13: 442–447.  <ul class="find" data-citedArticleID="1001874" data-doi="10.1016/s0960-9822(03)00126-x"><li><a href="http://dx.doi.org/10.1016/s0960-9822(03)00126-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+noncanonical+nuclear+genetic+code%3A+translation+of+UAA+into+glutamate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+noncanonical+nuclear+genetic+code%3A+translation+of+UAA+into+glutamate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000996-LeMouel1" id="pone.0000996-LeMouel1"></a>Le Mouel A, Butler A, Caron F, Meyer E (2003) Developmentally regulated chromosome fragmentation linked to imprecise elimination of repeated sequences in paramecia. Eukaryot Cell  2: 1076–1090.  <ul class="find" data-citedArticleID="1001818" data-doi="10.1128/ec.2.5.1076-1090.2003"><li><a href="http://dx.doi.org/10.1128/ec.2.5.1076-1090.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmentally+regulated+chromosome+fragmentation+linked+to+imprecise+elimination+of+repeated+sequences+in+paramecia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmentally+regulated+chromosome+fragmentation+linked+to+imprecise+elimination+of+repeated+sequences+in+paramecia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000996-Tuite1" id="pone.0000996-Tuite1"></a>Tuite MF, Santos MA (1996) Codon reassignment in Candida species: an evolutionary conundrum. Biochimie  78: 993–9.  <ul class="find" data-citedArticleID="1001910" data-doi="10.1016/s0300-9084(97)86722-3"><li><a href="http://dx.doi.org/10.1016/s0300-9084(97)86722-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+reassignment+in+Candida+species%3A+an+evolutionary+conundrum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+reassignment+in+Candida+species%3A+an+evolutionary+conundrum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000996-Jukes1" id="pone.0000996-Jukes1"></a>Jukes TH, Osawa S (1996) CUG codons in <em>Candida</em> spp. J Mol Evol  42: 321–322.  <ul class="find" data-citedArticleID="1001808" data-doi="10.1007/bf02198859"><li><a href="http://dx.doi.org/10.1007/bf02198859" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CUG+codons+in+Candida+spp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CUG+codons+in+Candida+spp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000996-Sugita1" id="pone.0000996-Sugita1"></a>Sugita T, Nakase T (1999) Non-universal usage of the leucine CUG codon and the molecular phylogeny of the genus Candida. Syst Appl Microbiol  22: 79–86.  <ul class="find" data-citedArticleID="1001900" data-doi="10.1016/s0723-2020(99)80030-7"><li><a href="http://dx.doi.org/10.1016/s0723-2020(99)80030-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-universal+usage+of+the+leucine+CUG+codon+and+the+molecular+phylogeny+of+the+genus+Candida." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-universal+usage+of+the+leucine+CUG+codon+and+the+molecular+phylogeny+of+the+genus+Candida.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000996-Hanyu1" id="pone.0000996-Hanyu1"></a>Hanyu N, Kuchino Y, Nishimura S, Beier H (1986) Dramatic events in ciliate evolution: alteration of UAA and UAG termination codons to glutamine codons due to anticodon mutations in two Tetrahymena tRNAs. EMBO J  5: 1307–1311.  <ul class="find" data-citedArticleID="1001780"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dramatic+events+in+ciliate+evolution%3A+alteration+of+UAA+and+UAG+termination+codons+to+glutamine+codons+due+to+anticodon+mutations+in+two+Tetrahymena+tRNAs.&amp;auth=&amp;atitle=Dramatic+events+in+ciliate+evolution%3A+alteration+of+UAA+and+UAG+termination+codons+to+glutamine+codons+due+to+anticodon+mutations+in+two+Tetrahymena+tRNAs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dramatic+events+in+ciliate+evolution%3A+alteration+of+UAA+and+UAG+termination+codons+to+glutamine+codons+due+to+anticodon+mutations+in+two+Tetrahymena+tRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dramatic+events+in+ciliate+evolution%3A+alteration+of+UAA+and+UAG+termination+codons+to+glutamine+codons+due+to+anticodon+mutations+in+two+Tetrahymena+tRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000996-Hao1" id="pone.0000996-Hao1"></a>Hao B, Gong W, Ferguson TK, James CM, Krzycki JA, et al.  (2002) A new UAG-encoded residue in the structure of a methanogen methyltransferase. Science  296: 1462–1466.  <ul class="find" data-citedArticleID="1001782" data-doi="10.1126/science.1069556"><li><a href="http://dx.doi.org/10.1126/science.1069556" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+UAG-encoded+residue+in+the+structure+of+a+methanogen+methyltransferase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+UAG-encoded+residue+in+the+structure+of+a+methanogen+methyltransferase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000996-Lee1" id="pone.0000996-Lee1"></a>Lee BJ, Worland PJ, Davis JN, Stadtman TC, Hatfield DL (1989) Identification of a selenocysteyl-tRNA(Ser) in mammalian cells that recognizes the nonsense codon, UGA. J Biol Chem  264: 9724–9727.  <ul class="find" data-citedArticleID="1001820"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+a+selenocysteyl-tRNA%28Ser%29+in+mammalian+cells+that+recognizes+the+nonsense+codon%2C+UGA.&amp;auth=&amp;atitle=Identification+of+a+selenocysteyl-tRNA%28Ser%29+in+mammalian+cells+that+recognizes+the+nonsense+codon%2C+UGA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+selenocysteyl-tRNA%28Ser%29+in+mammalian+cells+that+recognizes+the+nonsense+codon%2C+UGA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+selenocysteyl-tRNA%28Ser%29+in+mammalian+cells+that+recognizes+the+nonsense+codon%2C+UGA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000996-Bacher1" id="pone.0000996-Bacher1"></a>Bacher JM, Ellington AD (2001) Selection and characterization of Escherichia coli variants capable of growth on an otherwise toxic tryptophan analogue. J Bacteriol  183: 5414–5425.  <ul class="find" data-citedArticleID="1001750" data-doi="10.1128/jb.183.18.5414-5425.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.18.5414-5425.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+and+characterization+of+Escherichia+coli+variants+capable+of+growth+on+an+otherwise+toxic+tryptophan+analogue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+and+characterization+of+Escherichia+coli+variants+capable+of+growth+on+an+otherwise+toxic+tryptophan+analogue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000996-Bacher2" id="pone.0000996-Bacher2"></a>Bacher JM, Crecy-Lagard V, Schimmel PR (2005) Inhibited cell growth and protein functional changes from an editing-defective tRNA synthetase. Proc Natl Acad Sci U S A  102: 1697–1701.  <ul class="find" data-citedArticleID="1001752" data-doi="10.1073/pnas.0409064102"><li><a href="http://dx.doi.org/10.1073/pnas.0409064102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibited+cell+growth+and+protein+functional+changes+from+an+editing-defective+tRNA+synthetase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibited+cell+growth+and+protein+functional+changes+from+an+editing-defective+tRNA+synthetase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000996-Nangle1" id="pone.0000996-Nangle1"></a>Nangle LA, Motta CM, Schimmel P (2006) Global effects of mistranslation from an editing defect in mammalian cells. Chem Biol  13: 1091–1100.  <ul class="find" data-citedArticleID="1001842" data-doi="10.1016/j.chembiol.2006.08.011"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2006.08.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+effects+of+mistranslation+from+an+editing+defect+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+effects+of+mistranslation+from+an+editing+defect+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000996-Chin1" id="pone.0000996-Chin1"></a>Chin JW, Cropp TA, Anderson JC, Mukherji M, Zhang Z, et al.  (2003) An expanded eukaryotic genetic code. Science  301: 964–967.  <ul class="find" data-citedArticleID="1001762" data-doi="10.1126/science.1084772"><li><a href="http://dx.doi.org/10.1126/science.1084772" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+expanded+eukaryotic+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+expanded+eukaryotic+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000996-Hendrickson1" id="pone.0000996-Hendrickson1"></a>Hendrickson TL, Crecy-Lagard V, Schimmel P (2004) Incorporation of nonnatural amino acids into proteins. Annu Rev Biochem  73: 147–176.  <ul class="find" data-citedArticleID="1001788" data-doi="10.1146/annurev.biochem.73.012803.092429"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.73.012803.092429" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incorporation+of+nonnatural+amino+acids+into+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incorporation+of+nonnatural+amino+acids+into+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000996-Santos2" id="pone.0000996-Santos2"></a>Santos MA, Perreau VM, Tuite MF (1996) Transfer RNA structural change is a key element in the reassignment of the CUG codon in Candida albicans. Embo J  15: 5060–8.  <ul class="find" data-citedArticleID="1001878"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans.&amp;auth=&amp;atitle=Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000996-Santos3" id="pone.0000996-Santos3"></a>Santos MA, Tuite MF (1995) The CUG codon is decoded in vivo as serine and not leucine in Candida albicans. Nucleic Acids Res  23: 1481–1486.  <ul class="find" data-citedArticleID="1001880" data-doi="10.1093/nar/23.9.1481"><li><a href="http://dx.doi.org/10.1093/nar/23.9.1481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CUG+codon+is+decoded+in+vivo+as+serine+and+not+leucine+in+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CUG+codon+is+decoded+in+vivo+as+serine+and+not+leucine+in+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000996-Ohama1" id="pone.0000996-Ohama1"></a>Ohama T, Suzuki T, Mori M, Osawa S, Ueda T, et al.  (1993) Non-universal decoding of the leucine codon CUG in several Candida species. Nucleic Acids Res  21: 4039–45.  <ul class="find" data-citedArticleID="1001850" data-doi="10.1093/nar/21.17.4039"><li><a href="http://dx.doi.org/10.1093/nar/21.17.4039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-universal+decoding+of+the+leucine+codon+CUG+in+several+Candida+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-universal+decoding+of+the+leucine+codon+CUG+in+several+Candida+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000996-Suzuki1" id="pone.0000996-Suzuki1"></a>Suzuki T, Ueda T, Watanabe K (1997) The ‘polysemous’ codon--a codon with multiple amino acid assignment caused by dual specificity of tRNA identity. Embo J  16: 1122–34.  <ul class="find" data-citedArticleID="1001904" data-doi="10.1093/emboj/16.5.1122"><li><a href="http://dx.doi.org/10.1093/emboj/16.5.1122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+%E2%80%98polysemous%E2%80%99+codon--a+codon+with+multiple+amino+acid+assignment+caused+by+dual+specificity+of+tRNA+identity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+%E2%80%98polysemous%E2%80%99+codon--a+codon+with+multiple+amino+acid+assignment+caused+by+dual+specificity+of+tRNA+identity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000996-Santos4" id="pone.0000996-Santos4"></a>Santos MA, Keith G, Tuite MF (1993) Non-standard translational events in Candida albicans mediated by an unusual seryl-tRNA with a 5′-CAG-3′ (leucine) anticodon. Embo J  12: 607–16.  <ul class="find" data-citedArticleID="1001882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Non-standard+translational+events+in+Candida+albicans+mediated+by+an+unusual+seryl-tRNA+with+a+5%3F-CAG-3%3F+%28leucine%29+anticodon.&amp;auth=&amp;atitle=Non-standard+translational+events+in+Candida+albicans+mediated+by+an+unusual+seryl-tRNA+with+a+5%3F-CAG-3%3F+%28leucine%29+anticodon." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-standard+translational+events+in+Candida+albicans+mediated+by+an+unusual+seryl-tRNA+with+a+5%E2%80%B2-CAG-3%E2%80%B2+%28leucine%29+anticodon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-standard+translational+events+in+Candida+albicans+mediated+by+an+unusual+seryl-tRNA+with+a+5%E2%80%B2-CAG-3%E2%80%B2+%28leucine%29+anticodon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000996-Perreau1" id="pone.0000996-Perreau1"></a>Perreau VM, Keith G, Holmes WM, Przykorska A, Santos MA, et al.  (1999) The Candida albicans CUG-decoding ser-tRNA has an atypical anticodon stem-loop structure. J Mol Biol  293: 1039–1053.  <ul class="find" data-citedArticleID="1001854" data-doi="10.1006/jmbi.1999.3209"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3209" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Candida+albicans+CUG-decoding+ser-tRNA+has+an+atypical+anticodon+stem-loop+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Candida+albicans+CUG-decoding+ser-tRNA+has+an+atypical+anticodon+stem-loop+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000996-Schultz1" id="pone.0000996-Schultz1"></a>Schultz DW, Yarus M (1996) On malleability in the genetic code. J Mol Evol  42: 597–601.  <ul class="find" data-citedArticleID="1001890" data-doi="10.1007/bf02352290"><li><a href="http://dx.doi.org/10.1007/bf02352290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+malleability+in+the+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+malleability+in+the+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000996-Schultz2" id="pone.0000996-Schultz2"></a>Schultz DW, Yarus M (1994) Transfer RNA mutation and the malleability of the genetic code. J Mol Biol  235: 1377–1380.  <ul class="find" data-citedArticleID="1001892" data-doi="10.1006/jmbi.1994.1094"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+RNA+mutation+and+the+malleability+of+the+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+RNA+mutation+and+the+malleability+of+the+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000996-Massey1" id="pone.0000996-Massey1"></a>Massey SE, Moura G, Beltrao P, Almeida R, Garey JR, et al.  (2003) Comparative evolutionary genomics unveils the molecular mechanism of reassignment of the CTG codon in Candida spp. Genome Res  13: 544–557.  <ul class="find" data-citedArticleID="1001834" data-doi="10.1101/gr.811003"><li><a href="http://dx.doi.org/10.1101/gr.811003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+evolutionary+genomics+unveils+the+molecular+mechanism+of+reassignment+of+the+CTG+codon+in+Candida+spp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+evolutionary+genomics+unveils+the+molecular+mechanism+of+reassignment+of+the+CTG+codon+in+Candida+spp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000996-Varshney1" id="pone.0000996-Varshney1"></a>Varshney U, Lee CP, RajBhandary UL (1991) Direct analysis of aminoacylation levels of tRNAs in vivo. Application to studying recognition of Escherichia coli initiator tRNA mutants by glutaminyl-tRNA synthetase. J Biol Chem  266: 24712–24718.  <ul class="find" data-citedArticleID="1001912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Direct+analysis+of+aminoacylation+levels+of+tRNAs+in+vivo.+Application+to+studying+recognition+of+Escherichia+coli+initiator+tRNA+mutants+by+glutaminyl-tRNA+synthetase.&amp;auth=&amp;atitle=Direct+analysis+of+aminoacylation+levels+of+tRNAs+in+vivo.+Application+to+studying+recognition+of+Escherichia+coli+initiator+tRNA+mutants+by+glutaminyl-tRNA+synthetase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+analysis+of+aminoacylation+levels+of+tRNAs+in+vivo.+Application+to+studying+recognition+of+Escherichia+coli+initiator+tRNA+mutants+by+glutaminyl-tRNA+synthetase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+analysis+of+aminoacylation+levels+of+tRNAs+in+vivo.+Application+to+studying+recognition+of+Escherichia+coli+initiator+tRNA+mutants+by+glutaminyl-tRNA+synthetase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000996-OSullivan1" id="pone.0000996-OSullivan1"></a>O'Sullivan JM, Davenport JB, Tuite MF (2001) Codon reassignment and the evolving genetic code: problems and pitfalls in post-genome analysis. Trends Genet  17: 20–22.  <ul class="find" data-citedArticleID="1001844" data-doi="10.1016/s0168-9525(00)02144-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02144-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+reassignment+and+the+evolving+genetic+code%3A+problems+and+pitfalls+in+post-genome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+reassignment+and+the+evolving+genetic+code%3A+problems+and+pitfalls+in+post-genome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000996-OSullivan2" id="pone.0000996-OSullivan2"></a>O'Sullivan JM, Mihr MJ, Santos MA, Tuite MF (2001) Seryl-tRNA synthetase is not responsible for the evolution of CUG codon reassignment in Candida albicans. Yeast  18: 313–322.  <ul class="find" data-citedArticleID="1001846" data-doi="10.1016/s0378-1119(01)00632-1"><li><a href="http://dx.doi.org/10.1016/s0378-1119(01)00632-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seryl-tRNA+synthetase+is+not+responsible+for+the+evolution+of+CUG+codon+reassignment+in+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seryl-tRNA+synthetase+is+not+responsible+for+the+evolution+of+CUG+codon+reassignment+in+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000996-Soll1" id="pone.0000996-Soll1"></a>Soll DR (2002) Phenotypic Switching. Candida and Candidiasis 123–142.  <ul class="find" data-citedArticleID="1001896"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phenotypic+Switching.&amp;auth=&amp;atitle=Phenotypic+Switching." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+Switching." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+Switching.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000996-Hoyer1" id="pone.0000996-Hoyer1"></a>Hoyer LL (2001) The ALS gene family of Candida albicans. Trends Microbiol  9: 176–80.  <ul class="find" data-citedArticleID="1001790" data-doi="10.1016/s0966-842x(01)01984-9"><li><a href="http://dx.doi.org/10.1016/s0966-842x(01)01984-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ALS+gene+family+of+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ALS+gene+family+of+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000996-Fu1" id="pone.0000996-Fu1"></a>Fu Y, Rieg G, Fonzi WA, Belanger PH, Edwards JE, et al.  (1998) Expression of the Candida albicans gene ALS1 in Saccharomyces cerevisiae induces adherence to endothelial and epithelial cells. Infect Immun  66: 1783–1786.  <ul class="find" data-citedArticleID="1001770"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+of+the+Candida+albicans+gene+ALS1+in+Saccharomyces+cerevisiae+induces+adherence+to+endothelial+and+epithelial+cells.&amp;auth=&amp;atitle=Expression+of+the+Candida+albicans+gene+ALS1+in+Saccharomyces+cerevisiae+induces+adherence+to+endothelial+and+epithelial+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+Candida+albicans+gene+ALS1+in+Saccharomyces+cerevisiae+induces+adherence+to+endothelial+and+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+Candida+albicans+gene+ALS1+in+Saccharomyces+cerevisiae+induces+adherence+to+endothelial+and+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000996-Rauceo1" id="pone.0000996-Rauceo1"></a>Rauceo JM, Gaur NK, Lee KG, Edwards JE, Klotz SA, et al.  (2004) Global cell surface conformational shift mediated by a Candida albicans adhesin. Infect Immun  72: 4948–4955.  <ul class="find" data-citedArticleID="1001858" data-doi="10.1128/iai.72.9.4948-4955.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.9.4948-4955.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+cell+surface+conformational+shift+mediated+by+a+Candida+albicans+adhesin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+cell+surface+conformational+shift+mediated+by+a+Candida+albicans+adhesin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000996-Gaur1" id="pone.0000996-Gaur1"></a>Gaur NK, Smith RL, Klotz SA (2002) Candida albicans and Saccharomyces cerevisiae expressing ALA1/ALS5 adhere to accessible threonine, serine, or alanine patches. Cell Commun Adhes  9: 45–57.  <ul class="find" data-citedArticleID="1001772" data-doi="10.1080/15419060212187"><li><a href="http://dx.doi.org/10.1080/15419060212187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Candida+albicans+and+Saccharomyces+cerevisiae+expressing+ALA1%2FALS5+adhere+to+accessible+threonine%2C+serine%2C+or+alanine+patches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Candida+albicans+and+Saccharomyces+cerevisiae+expressing+ALA1%2FALS5+adhere+to+accessible+threonine%2C+serine%2C+or+alanine+patches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000996-Gaur2" id="pone.0000996-Gaur2"></a>Gaur NK, Klotz SA, Henderson RL (1999) Overexpression of the Candida albicans ALA1 gene in Saccharomyces cerevisiae results in aggregation following attachment of yeast cells to extracellular matrix proteins, adherence properties similar to those of Candida albicans. Infect Immun  67: 6040–6047.  <ul class="find" data-citedArticleID="1001774"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Overexpression+of+the+Candida+albicans+ALA1+gene+in+Saccharomyces+cerevisiae+results+in+aggregation+following+attachment+of+yeast+cells+to+extracellular+matrix+proteins%2C+adherence+properties+similar+to+those+of+Candida+albicans.&amp;auth=&amp;atitle=Overexpression+of+the+Candida+albicans+ALA1+gene+in+Saccharomyces+cerevisiae+results+in+aggregation+following+attachment+of+yeast+cells+to+extracellular+matrix+proteins%2C+adherence+properties+similar+to+those+of+Candida+albicans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+the+Candida+albicans+ALA1+gene+in+Saccharomyces+cerevisiae+results+in+aggregation+following+attachment+of+yeast+cells+to+extracellular+matrix+proteins%2C+adherence+properties+similar+to+those+of+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+the+Candida+albicans+ALA1+gene+in+Saccharomyces+cerevisiae+results+in+aggregation+following+attachment+of+yeast+cells+to+extracellular+matrix+proteins%2C+adherence+properties+similar+to+those+of+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000996-Gaur3" id="pone.0000996-Gaur3"></a>Gaur NK, Klotz SA (1997) Expression, cloning, and characterization of a Candida albicans gene, ALA1, that confers adherence properties upon Saccharomyces cerevisiae for extracellular matrix proteins. Infect Immun  65: 5289–5294.  <ul class="find" data-citedArticleID="1001776"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression%2C+cloning%2C+and+characterization+of+a+Candida+albicans+gene%2C+ALA1%2C+that+confers+adherence+properties+upon+Saccharomyces+cerevisiae+for+extracellular+matrix+proteins.&amp;auth=&amp;atitle=Expression%2C+cloning%2C+and+characterization+of+a+Candida+albicans+gene%2C+ALA1%2C+that+confers+adherence+properties+upon+Saccharomyces+cerevisiae+for+extracellular+matrix+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression%2C+cloning%2C+and+characterization+of+a+Candida+albicans+gene%2C+ALA1%2C+that+confers+adherence+properties+upon+Saccharomyces+cerevisiae+for+extracellular+matrix+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression%2C+cloning%2C+and+characterization+of+a+Candida+albicans+gene%2C+ALA1%2C+that+confers+adherence+properties+upon+Saccharomyces+cerevisiae+for+extracellular+matrix+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000996-Li1" id="pone.0000996-Li1"></a>Li F, Palecek SP (2003) EAP1, a Candida albicans gene involved in binding human epithelial cells. Eukaryot Cell  2: 1266–1273.  <ul class="find" data-citedArticleID="1001826" data-doi="10.1128/ec.2.6.1266-1273.2003"><li><a href="http://dx.doi.org/10.1128/ec.2.6.1266-1273.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EAP1%2C+a+Candida+albicans+gene+involved+in+binding+human+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EAP1%2C+a+Candida+albicans+gene+involved+in+binding+human+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000996-Ibrahim1" id="pone.0000996-Ibrahim1"></a>Ibrahim AS, Mirbod F, Filler SG, Banno Y, Cole GT, et al.  (1995) Evidence implicating phospholipase as a virulence factor of Candida albicans. Infect Immun  63: 1993–8.  <ul class="find" data-citedArticleID="1001798"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+implicating+phospholipase+as+a+virulence+factor+of+Candida+albicans.&amp;auth=&amp;atitle=Evidence+implicating+phospholipase+as+a+virulence+factor+of+Candida+albicans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+implicating+phospholipase+as+a+virulence+factor+of+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+implicating+phospholipase+as+a+virulence+factor+of+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000996-Naglik1" id="pone.0000996-Naglik1"></a>Naglik JR, Challacombe SJ, Hube B (2003) Candida albicans Secreted Aspartyl Proteinases in Virulence and Pathogenesis. Microbiol Mol Biol Rev  67: 400–428.  <ul class="find" data-citedArticleID="1001840" data-doi="10.1128/mmbr.67.3.400-428.2003"><li><a href="http://dx.doi.org/10.1128/mmbr.67.3.400-428.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Candida+albicans+Secreted+Aspartyl+Proteinases+in+Virulence+and+Pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Candida+albicans+Secreted+Aspartyl+Proteinases+in+Virulence+and+Pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000996-Sundstrom1" id="pone.0000996-Sundstrom1"></a>Sundstrom P (2002) Adhesion in Candida spp. Cell Microbiol  4: 461–469.  <ul class="find" data-citedArticleID="1001902" data-doi="10.1046/j.1462-5822.2002.00206.x"><li><a href="http://dx.doi.org/10.1046/j.1462-5822.2002.00206.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adhesion+in+Candida+spp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adhesion+in+Candida+spp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000996-Ghannoum1" id="pone.0000996-Ghannoum1"></a>Ghannoum MA (2000) Potential role of phospholipases in virulence and fungal pathogenesis. Clin Microbiol Rev  13: 122–43.  <ul class="find" data-citedArticleID="1001778" data-doi="10.1128/cmr.13.1.122-143.2000"><li><a href="http://dx.doi.org/10.1128/cmr.13.1.122-143.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Potential+role+of+phospholipases+in+virulence+and+fungal+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Potential+role+of+phospholipases+in+virulence+and+fungal+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000996-Zordan1" id="pone.0000996-Zordan1"></a>Zordan RE, Galgoczy DJ, Johnson AD (2006) Epigenetic properties of white-opaque switching in Candida albicans are based on a self-sustaining transcriptional feedback loop. Proc Natl Acad Sci U S A  103: 12807–12812.  <ul class="find" data-citedArticleID="1001918" data-doi="10.1073/pnas.0605138103"><li><a href="http://dx.doi.org/10.1073/pnas.0605138103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+properties+of+white-opaque+switching+in+Candida+albicans+are+based+on+a+self-sustaining+transcriptional+feedback+loop." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+properties+of+white-opaque+switching+in+Candida+albicans+are+based+on+a+self-sustaining+transcriptional+feedback+loop.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000996-Huang1" id="pone.0000996-Huang1"></a>Huang G, Wang H, Chou S, Nie X, Chen J, et al.  (2006) Bistable expression of WOR1, a master regulator of white-opaque switching in Candida albicans. Proc Natl Acad Sci U S A  103: 12813–12818.  <ul class="find" data-citedArticleID="1001792" data-doi="10.1073/pnas.0605270103"><li><a href="http://dx.doi.org/10.1073/pnas.0605270103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bistable+expression+of+WOR1%2C+a+master+regulator+of+white-opaque+switching+in+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bistable+expression+of+WOR1%2C+a+master+regulator+of+white-opaque+switching+in+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000996-Staab1" id="pone.0000996-Staab1"></a>Staab JF, Bradway SD, Fidel PL, Sundstrom P (1999) Adhesive and mammalian transglutaminase substrate properties of Candida albicans Hwp1. Science  283: 1535–8.  <ul class="find" data-citedArticleID="1001898" data-doi="10.1126/science.283.5407.1535"><li><a href="http://dx.doi.org/10.1126/science.283.5407.1535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adhesive+and+mammalian+transglutaminase+substrate+properties+of+Candida+albicans+Hwp1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adhesive+and+mammalian+transglutaminase+substrate+properties+of+Candida+albicans+Hwp1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000996-Rottmann1" id="pone.0000996-Rottmann1"></a>Rottmann M, Dieter S, Brunner H, Rupp S (2003) A screen in Saccharomyces cerevisiae identified CaMCM1, an essential gene in Candida albicans crucial for morphogenesis. Mol Microbiol  47: 943–959.  <ul class="find" data-citedArticleID="1001862" data-doi="10.1046/j.1365-2958.2003.03358.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03358.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+screen+in+Saccharomyces+cerevisiae+identified+CaMCM1%2C+an+essential+gene+in+Candida+albicans+crucial+for+morphogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+screen+in+Saccharomyces+cerevisiae+identified+CaMCM1%2C+an+essential+gene+in+Candida+albicans+crucial+for+morphogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000996-Zheng1" id="pone.0000996-Zheng1"></a>Zheng X, Wang Y, Wang Y (2004) Hgc1, a novel hypha-specific G1 cyclin-related protein regulates Candida albicans hyphal morphogenesis. EMBO J  23: 1845–1856.  <ul class="find" data-citedArticleID="1001916" data-doi="10.1038/sj.emboj.7600195"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hgc1%2C+a+novel+hypha-specific+G1+cyclin-related+protein+regulates+Candida+albicans+hyphal+morphogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hgc1%2C+a+novel+hypha-specific+G1+cyclin-related+protein+regulates+Candida+albicans+hyphal+morphogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000996-Hull1" id="pone.0000996-Hull1"></a>Hull CM, Johnson AD (1999) Identification of a mating type-like locus in the asexual pathogenic yeast Candida albicans. Science  285: 1271–5.  <ul class="find" data-citedArticleID="1001794" data-doi="10.1126/science.285.5431.1271"><li><a href="http://dx.doi.org/10.1126/science.285.5431.1271" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+mating+type-like+locus+in+the+asexual+pathogenic+yeast+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+mating+type-like+locus+in+the+asexual+pathogenic+yeast+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000996-Miller1" id="pone.0000996-Miller1"></a>Miller MG, Johnson AD (2002) White-opaque switching in Candida albicans is controlled by mating-type locus homeodomain proteins and allows efficient mating. Cell  110: 293–302.  <ul class="find" data-citedArticleID="1001836" data-doi="10.1016/s0092-8674(02)00837-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00837-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=White-opaque+switching+in+Candida+albicans+is+controlled+by+mating-type+locus+homeodomain+proteins+and+allows+efficient+mating." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22White-opaque+switching+in+Candida+albicans+is+controlled+by+mating-type+locus+homeodomain+proteins+and+allows+efficient+mating.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000996-Iwaguchi1" id="pone.0000996-Iwaguchi1"></a>Iwaguchi SI, Kanbe T, Tohne T, Magee PT, Suzuki T (2000) High-frequency occurrence of chromosome translocation in a mutant strain of Candida albicans by a suppressor mutation of ploidy shift. Yeast  16: 411–422.  <ul class="find" data-citedArticleID="1001802"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High-frequency+occurrence+of+chromosome+translocation+in+a+mutant+strain+of+Candida+albicans+by+a+suppressor+mutation+of+ploidy+shift.&amp;auth=&amp;atitle=High-frequency+occurrence+of+chromosome+translocation+in+a+mutant+strain+of+Candida+albicans+by+a+suppressor+mutation+of+ploidy+shift." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-frequency+occurrence+of+chromosome+translocation+in+a+mutant+strain+of+Candida+albicans+by+a+suppressor+mutation+of+ploidy+shift." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-frequency+occurrence+of+chromosome+translocation+in+a+mutant+strain+of+Candida+albicans+by+a+suppressor+mutation+of+ploidy+shift.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000996-Suzuki2" id="pone.0000996-Suzuki2"></a>Suzuki T, Hitomi A, Magee PT, Sakaguchi S (1994) Correlation between polyploidy and auxotrophic segregation in the imperfect yeast Candida albicans. J Bacteriol  176: 3345–3353.  <ul class="find" data-citedArticleID="1001906"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Correlation+between+polyploidy+and+auxotrophic+segregation+in+the+imperfect+yeast+Candida+albicans.&amp;auth=&amp;atitle=Correlation+between+polyploidy+and+auxotrophic+segregation+in+the+imperfect+yeast+Candida+albicans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+polyploidy+and+auxotrophic+segregation+in+the+imperfect+yeast+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+polyploidy+and+auxotrophic+segregation+in+the+imperfect+yeast+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000996-Bennett1" id="pone.0000996-Bennett1"></a>Bennett RJ, Johnson AD (2003) Completion of a parasexual cycle in Candida albicans by induced chromosome loss in tetraploid strains. EMBO J  22: 2505–2515.  <ul class="find" data-citedArticleID="1001754" data-doi="10.1093/emboj/cdg235"><li><a href="http://dx.doi.org/10.1093/emboj/cdg235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Completion+of+a+parasexual+cycle+in+Candida+albicans+by+induced+chromosome+loss+in+tetraploid+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Completion+of+a+parasexual+cycle+in+Candida+albicans+by+induced+chromosome+loss+in+tetraploid+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000996-Magee1" id="pone.0000996-Magee1"></a>Magee BB, Magee PT (2000) Induction of mating in Candida albicans by construction of MTLa and MTLalpha strains. Science  289: 310–3.  <ul class="find" data-citedArticleID="1001830" data-doi="10.1126/science.289.5477.310"><li><a href="http://dx.doi.org/10.1126/science.289.5477.310" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+mating+in+Candida+albicans+by+construction+of+MTLa+and+MTLalpha+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+mating+in+Candida+albicans+by+construction+of+MTLa+and+MTLalpha+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000996-Hull2" id="pone.0000996-Hull2"></a>Hull CM, Raisner RM, Johnson AD (2000) Evidence for mating of the “asexual” yeast Candida albicans in a mammalian host. Science  289: 307–10.  <ul class="find" data-citedArticleID="1001796" data-doi="10.1126/science.289.5477.307"><li><a href="http://dx.doi.org/10.1126/science.289.5477.307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+mating+of+the+%E2%80%9Casexual%E2%80%9D+yeast+Candida+albicans+in+a+mammalian+host." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+mating+of+the+%E2%80%9Casexual%E2%80%9D+yeast+Candida+albicans+in+a+mammalian+host.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000996-Lasker1" id="pone.0000996-Lasker1"></a>Lasker BA, Carle GF, Kobayashi GS, Medoff G (1989) Comparison of the separation of Candida albicans chromosome-sized DNA by pulsed-field gel electrophoresis techniques. Nucleic Acids Res  17: 3783–3793.  <ul class="find" data-citedArticleID="1001816" data-doi="10.1093/nar/17.10.3783"><li><a href="http://dx.doi.org/10.1093/nar/17.10.3783" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+the+separation+of+Candida+albicans+chromosome-sized+DNA+by+pulsed-field+gel+electrophoresis+techniques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+the+separation+of+Candida+albicans+chromosome-sized+DNA+by+pulsed-field+gel+electrophoresis+techniques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000996-Magee2" id="pone.0000996-Magee2"></a>Magee BB, Magee PT (1987) Electrophoretic karyotypes and chromosome numbers in Candida species. J Gen Microbiol  133: 425–430.  <ul class="find" data-citedArticleID="1001832" data-doi="10.1099/00221287-133-2-425"><li><a href="http://dx.doi.org/10.1099/00221287-133-2-425" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Electrophoretic+karyotypes+and+chromosome+numbers+in+Candida+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Electrophoretic+karyotypes+and+chromosome+numbers+in+Candida+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000996-Santos5" id="pone.0000996-Santos5"></a>Santos MA, Ueda T, Watanabe K, Tuite MF (1997) The non-standard genetic code of Candida spp.: an evolving genetic code or a novel mechanism for adaptation? Mol Microbiol  26: 423–431.  <ul class="find" data-citedArticleID="1001884" data-doi="10.1046/j.1365-2958.1997.5891961.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.5891961.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+non-standard+genetic+code+of+Candida+spp.%3A+an+evolving+genetic+code+or+a+novel+mechanism+for+adaptation%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+non-standard+genetic+code+of+Candida+spp.%3A+an+evolving+genetic+code+or+a+novel+mechanism+for+adaptation%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000996-OSullivan3" id="pone.0000996-OSullivan3"></a>O'Sullivan JM, Santos MA, Tuite MF (2002) Standard and nonstandard mRNA decoding in Candida. Candida and Candidiasis 279–292.  <ul class="find" data-citedArticleID="1001848"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Standard+and+nonstandard+mRNA+decoding+in+Candida.&amp;auth=&amp;atitle=Standard+and+nonstandard+mRNA+decoding+in+Candida." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Standard+and+nonstandard+mRNA+decoding+in+Candida." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Standard+and+nonstandard+mRNA+decoding+in+Candida.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000996-Breitschopf1" id="pone.0000996-Breitschopf1"></a>Breitschopf K, Gross HJ (1994) The exchange of the discriminator base A73 for G is alone sufficient to convert human tRNA(Leu) into a serine-acceptor in vitro. EMBO J  13: 3166–3167.  <ul class="find" data-citedArticleID="1001756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+exchange+of+the+discriminator+base+A73+for+G+is+alone+sufficient+to+convert+human+tRNA%28Leu%29+into+a+serine-acceptor+in+vitro.&amp;auth=&amp;atitle=The+exchange+of+the+discriminator+base+A73+for+G+is+alone+sufficient+to+convert+human+tRNA%28Leu%29+into+a+serine-acceptor+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+exchange+of+the+discriminator+base+A73+for+G+is+alone+sufficient+to+convert+human+tRNA%28Leu%29+into+a+serine-acceptor+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+exchange+of+the+discriminator+base+A73+for+G+is+alone+sufficient+to+convert+human+tRNA%28Leu%29+into+a+serine-acceptor+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000996-Santos6" id="pone.0000996-Santos6"></a>Santos MA, Perreau VM, Tuite MF (1996) Transfer RNA structural change is a key element in the reassignment of the CUG codon in Candida albicans. EMBO J  15: 5060–5068.  <ul class="find" data-citedArticleID="1001886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans.&amp;auth=&amp;atitle=Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+RNA+structural+change+is+a+key+element+in+the+reassignment+of+the+CUG+codon+in+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000996-Selmecki1" id="pone.0000996-Selmecki1"></a>Selmecki A, Bergmann S, Berman J (2005) Comparative genome hybridization reveals widespread aneuploidy in Candida albicans laboratory strains. Mol Microbiol  55: 1553–1565.  <ul class="find" data-citedArticleID="1001894" data-doi="10.1111/j.1365-2958.2005.04492.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04492.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genome+hybridization+reveals+widespread+aneuploidy+in+Candida+albicans+laboratory+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genome+hybridization+reveals+widespread+aneuploidy+in+Candida+albicans+laboratory+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000996-RustchenkoBulgac1" id="pone.0000996-RustchenkoBulgac1"></a>Rustchenko-Bulgac EP, Howard DH (1993) Multiple chromosomal and phenotypic changes in spontaneous mutants of Candida albicans. J Gen Microbiol  139 Pt 6: 1195–1207.  <ul class="find" data-citedArticleID="1001866" data-doi="10.1099/00221287-139-6-1195"><li><a href="http://dx.doi.org/10.1099/00221287-139-6-1195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+chromosomal+and+phenotypic+changes+in+spontaneous+mutants+of+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+chromosomal+and+phenotypic+changes+in+spontaneous+mutants+of+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000996-RustchenkoBulgac2" id="pone.0000996-RustchenkoBulgac2"></a>Rustchenko-Bulgac EP, Sherman F, Hicks JB (1990) Chromosomal rearrangements associated with morphological mutants provide a means for genetic variation of Candida albicans. J Bacteriol  172: 1276–83.  <ul class="find" data-citedArticleID="1001868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chromosomal+rearrangements+associated+with+morphological+mutants+provide+a+means+for+genetic+variation+of+Candida+albicans.&amp;auth=&amp;atitle=Chromosomal+rearrangements+associated+with+morphological+mutants+provide+a+means+for+genetic+variation+of+Candida+albicans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+rearrangements+associated+with+morphological+mutants+provide+a+means+for+genetic+variation+of+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+rearrangements+associated+with+morphological+mutants+provide+a+means+for+genetic+variation+of+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000996-Suzuki3" id="pone.0000996-Suzuki3"></a>Suzuki T, Kobayashi I, Kanbe T, Tanaka K (1989) High frequency variation of colony morphology and chromosome reorganization in the pathogenic yeast Candida albicans. J Gen Microbiol  135: 425–434.  <ul class="find" data-citedArticleID="1001908" data-doi="10.1099/00221287-135-2-425"><li><a href="http://dx.doi.org/10.1099/00221287-135-2-425" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+frequency+variation+of+colony+morphology+and+chromosome+reorganization+in+the+pathogenic+yeast+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+frequency+variation+of+colony+morphology+and+chromosome+reorganization+in+the+pathogenic+yeast+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000996-Perepnikhatka1" id="pone.0000996-Perepnikhatka1"></a>Perepnikhatka V, Fischer FJ, Niimi M, Baker RA, Cannon RD, et al.  (1999) Specific chromosome alterations in fluconazole-resistant mutants of Candida albicans. J Bacteriol  181: 4041–4049.  <ul class="find" data-citedArticleID="1001852"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Specific+chromosome+alterations+in+fluconazole-resistant+mutants+of+Candida+albicans.&amp;auth=&amp;atitle=Specific+chromosome+alterations+in+fluconazole-resistant+mutants+of+Candida+albicans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+chromosome+alterations+in+fluconazole-resistant+mutants+of+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+chromosome+alterations+in+fluconazole-resistant+mutants+of+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000996-Rustchenko1" id="pone.0000996-Rustchenko1"></a>Rustchenko EP, Howard DH, Sherman F (1997) Variation in assimilating functions occurs in spontaneous Candida albicans mutants having chromosomal alterations. Microbiology  143 ( Pt 5): 1765–1778.  <ul class="find" data-citedArticleID="1001864" data-doi="10.1099/00221287-143-5-1765"><li><a href="http://dx.doi.org/10.1099/00221287-143-5-1765" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+assimilating+functions+occurs+in+spontaneous+Candida+albicans+mutants+having+chromosomal+alterations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+assimilating+functions+occurs+in+spontaneous+Candida+albicans+mutants+having+chromosomal+alterations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000996-Janbon1" id="pone.0000996-Janbon1"></a>Janbon G, Sherman F, Rustchenko E (1998) Monosomy of a specific chromosome determines L-sorbose utilization: a novel regulatory mechanism in Candida albicans. Proc Natl Acad Sci U S A  95: 5150–5155.  <ul class="find" data-citedArticleID="1001804" data-doi="10.1073/pnas.95.9.5150"><li><a href="http://dx.doi.org/10.1073/pnas.95.9.5150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monosomy+of+a+specific+chromosome+determines+L-sorbose+utilization%3A+a+novel+regulatory+mechanism+in+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monosomy+of+a+specific+chromosome+determines+L-sorbose+utilization%3A+a+novel+regulatory+mechanism+in+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000996-Legrand1" id="pone.0000996-Legrand1"></a>Legrand M, Lephart P, Forche A, Mueller FM, Walsh T, et al.  (2004) Homozygosity at the MTL locus in clinical strains of Candida albicans: karyotypic rearrangements and tetraploid formation. Mol Microbiol  52: 1451–1462.  <ul class="find" data-citedArticleID="1001824" data-doi="10.1111/j.1365-2958.2004.04068.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04068.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homozygosity+at+the+MTL+locus+in+clinical+strains+of+Candida+albicans%3A+karyotypic+rearrangements+and+tetraploid+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homozygosity+at+the+MTL+locus+in+clinical+strains+of+Candida+albicans%3A+karyotypic+rearrangements+and+tetraploid+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000996-Dorazi1" id="pone.0000996-Dorazi1"></a>Dorazi R, Lingutla JJ, Humayun MZ (2002) Expression of mutant alanine tRNAs increases spontaneous mutagenesis in Escherichia coli. Mol Microbiol  44: 131–141.  <ul class="find" data-citedArticleID="1001766" data-doi="10.1046/j.1365-2958.2002.02847.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02847.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+mutant+alanine+tRNAs+increases+spontaneous+mutagenesis+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+mutant+alanine+tRNAs+increases+spontaneous+mutagenesis+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000996-Jones1" id="pone.0000996-Jones1"></a>Jones T, Federspiel NA, Chibana H, Dungan J, Kalman S, et al.  (2004) The diploid genome sequence of Candida albicans. Proc Natl Acad Sci U S A  101: 7329–7334.  <ul class="find" data-citedArticleID="1001806" data-doi="10.1073/pnas.0401648101"><li><a href="http://dx.doi.org/10.1073/pnas.0401648101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+diploid+genome+sequence+of+Candida+albicans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+diploid+genome+sequence+of+Candida+albicans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000996-Lee2" id="pone.0000996-Lee2"></a>Lee JW, Beebe K, Nangle LA, Jang J, Longo-Guess CM, et al.  (2006) Editing-defective tRNA synthetase causes protein misfolding and neurodegeneration. Nature  443: 50–55.  <ul class="find" data-citedArticleID="1001822" data-doi="10.1038/nature05096"><li><a href="http://dx.doi.org/10.1038/nature05096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Editing-defective+tRNA+synthetase+causes+protein+misfolding+and+neurodegeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Editing-defective+tRNA+synthetase+causes+protein+misfolding+and+neurodegeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000996-Santos7" id="pone.0000996-Santos7"></a>Santos MA, Cheesman C, Costa V, Moradas-Ferreira P, Tuite MF (1999) Selective advantages created by codon ambiguity allowed for the evolution of an alternative genetic code in Candida spp. Mol Microbiol  31: 937–947.  <ul class="find" data-citedArticleID="1001888" data-doi="10.1046/j.1365-2958.1999.01233.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01233.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+advantages+created+by+codon+ambiguity+allowed+for+the+evolution+of+an+alternative+genetic+code+in+Candida+spp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+advantages+created+by+codon+ambiguity+allowed+for+the+evolution+of+an+alternative+genetic+code+in+Candida+spp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000996-Pla1" id="pone.0000996-Pla1"></a>Pla J, Gil C, Monteoliva L, Navarro-Garcia F, Sanchez M, et al.  (1996) Understanding Candida albicans at the molecular level. Yeast  12: 1677–702.  <ul class="find" data-citedArticleID="1001856"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Understanding+Candida+albicans+at+the+molecular+level.&amp;auth=&amp;atitle=Understanding+Candida+albicans+at+the+molecular+level." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+Candida+albicans+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+Candida+albicans+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000996-Sambrook1" id="pone.0000996-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular cloning: a laboratory manual. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">78.
              </span><a name="pone.0000996-WeygandDurasevic1" id="pone.0000996-WeygandDurasevic1"></a>Weygand-Durasevic I, Nalaskowska M, Soll D (1994) Coexpression of eukaryotic tRNASer and yeast seryl-tRNA synthetase leads to functional amber suppression in Escherichia coli. J Bacteriol  176: 232–239.  <ul class="find" data-citedArticleID="1001914"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Coexpression+of+eukaryotic+tRNASer+and+yeast+seryl-tRNA+synthetase+leads+to+functional+amber+suppression+in+Escherichia+coli.&amp;auth=&amp;atitle=Coexpression+of+eukaryotic+tRNASer+and+yeast+seryl-tRNA+synthetase+leads+to+functional+amber+suppression+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coexpression+of+eukaryotic+tRNASer+and+yeast+seryl-tRNA+synthetase+leads+to+functional+amber+suppression+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coexpression+of+eukaryotic+tRNASer+and+yeast+seryl-tRNA+synthetase+leads+to+functional+amber+suppression+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000996-Heitzler1" id="pone.0000996-Heitzler1"></a>Heitzler J, Marechal-Drouard L, Dirheimer G, Keith G (1992) Use of a dot blot hybridization method for identification of pure tRNA species on different membranes. Biochim Biophys Acta  1129: 273–277.  <ul class="find" data-citedArticleID="1001786" data-doi="10.1016/0167-4781(92)90503-r"><li><a href="http://dx.doi.org/10.1016/0167-4781(92)90503-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+a+dot+blot+hybridization+method+for+identification+of+pure+tRNA+species+on+different+membranes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+a+dot+blot+hybridization+method+for+identification+of+pure+tRNA+species+on+different+membranes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000996-Innis1" id="pone.0000996-Innis1"></a>Innis MA, Gelfand DH, Sninsky JJ, White TJ (1990) PCR protocols: A Guide to Methods and Applications.   <ul class="find-nolinks"></ul></li><li><span class="label">81.
              </span><a name="pone.0000996-Kohrer1" id="pone.0000996-Kohrer1"></a>Kohrer K, Domdey H (1991) Preparation of high molecular weight RNA. Methods Enzymol  194: 398–405.  <ul class="find" data-citedArticleID="1001814" data-doi="10.1016/0076-6879(91)94030-g"><li><a href="http://dx.doi.org/10.1016/0076-6879(91)94030-g" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preparation+of+high+molecular+weight+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preparation+of+high+molecular+weight+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000996-Rorabacher1" id="pone.0000996-Rorabacher1"></a>Rorabacher DB (1991) Statistical Treatment for Rejection of Deviant Values - Critical-Values of Dixon Q Parameter and Related Subrange Ratios at the 95-Percent Confidence Level. Analytical Chemistry  63: 139–146.  <ul class="find" data-citedArticleID="1001860" data-doi="10.1021/ac00002a010"><li><a href="http://dx.doi.org/10.1021/ac00002a010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+Treatment+for+Rejection+of+Deviant+Values+-+Critical-Values+of+Dixon+Q+Parameter+and+Related+Subrange+Ratios+at+the+95-Percent+Confidence+Level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+Treatment+for+Rejection+of+Deviant+Values+-+Critical-Values+of+Dixon+Q+Parameter+and+Related+Subrange+Ratios+at+the+95-Percent+Confidence+Level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000996-Samaranayake1" id="pone.0000996-Samaranayake1"></a>Samaranayake LP, Raeside JM, MacFarlane TW (1984) Factors affecting the phospholipase activity of Candida species in vitro. Sabouraudia  22: 201–207.  <ul class="find" data-citedArticleID="1001870" data-doi="10.1080/00362178485380331"><li><a href="http://dx.doi.org/10.1080/00362178485380331" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Factors+affecting+the+phospholipase+activity+of+Candida+species+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Factors+affecting+the+phospholipase+activity+of+Candida+species+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000996-Fortuna1" id="pone.0000996-Fortuna1"></a>Fortuna M, Sousa MJ, Côrte-Real M, Leão C, Salvador A, et al.  (2000) Cell Cycle Analysis of Yeasts. Current Protocols in Cytometry 11.13.1–11.13.9.  <ul class="find" data-citedArticleID="1001768" data-doi="10.1002/0471142956.cy1113s13"><li><a href="http://dx.doi.org/10.1002/0471142956.cy1113s13" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+Cycle+Analysis+of+Yeasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+Cycle+Analysis+of+Yeasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000996-Chu1" id="pone.0000996-Chu1"></a>Chu WS, Magee BB, Magee PT (1993) Construction of an SfiI macrorestriction map of the Candida albicans genome. J Bacteriol  175: 6637–6651.  <ul class="find" data-citedArticleID="1001764"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Construction+of+an+SfiI+macrorestriction+map+of+the+Candida+albicans+genome.&amp;auth=&amp;atitle=Construction+of+an+SfiI+macrorestriction+map+of+the+Candida+albicans+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+an+SfiI+macrorestriction+map+of+the+Candida+albicans+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+an+SfiI+macrorestriction+map+of+the+Candida+albicans+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.XML" value="131335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.PDF" value="477012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g001.PNG_L" value="364121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g001.PNG_M" value="46584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g001.PNG_S" value="8507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g001.TIF" value="584678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g001.PNG_I" value="16249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g002.PNG_L" value="93129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g002.PNG_M" value="115468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g002.PNG_S" value="10174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g002.TIF" value="898654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g002.PNG_I" value="40752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g003.PNG_L" value="719877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g003.PNG_M" value="48135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g003.PNG_S" value="7012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g003.TIF" value="1025356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g003.PNG_I" value="16215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g004.PNG_L" value="3725174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g004.PNG_M" value="178246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g004.PNG_S" value="11553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g004.TIF" value="5643034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g004.PNG_I" value="59483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g005.PNG_L" value="2753782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g005.PNG_M" value="87137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g005.PNG_S" value="8367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g005.TIF" value="2950422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g005.PNG_I" value="27367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g006.PNG_L" value="238117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g006.PNG_M" value="18769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g006.PNG_S" value="6651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g006.TIF" value="390720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g006.PNG_I" value="19919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g007.PNG_L" value="1134493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g007.PNG_M" value="139857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g007.PNG_S" value="9911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g007.TIF" value="2822500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g007.PNG_I" value="48820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g008.PNG_L" value="364322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g008.PNG_M" value="145201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g008.PNG_S" value="10551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g008.TIF" value="2277992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000996.g008.PNG_I" value="52353"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000996&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000996" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000996&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000996&volume=&issue=&title=A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen Candida albicans&author_name=Isabel%20Miranda%2C%20Rita%20Rocha%2C%20Maria%20C.%20Santos%2C%20Denisa%20D.%20Mateus%2C%20Gabriela%20R.%20Moura%2C%20Laura%20Carreto%2C%20Manuel%20A.%20S.%20Santos&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000996" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000996" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000996" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000996&amp;t=A%20Genetic%20Code%20Alteration%20Is%20a%20Phenotype%20Diversity%20Generator%20in%20the%20Human%20Pathogen%20Candida%20albicans" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000996&title=A%20Genetic%20Code%20Alteration%20Is%20a%20Phenotype%20Diversity%20Generator%20in%20the%20Human%20Pathogen%20Candida%20albicans&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000996&amp;title=A%20Genetic%20Code%20Alteration%20Is%20a%20Phenotype%20Diversity%20Generator%20in%20the%20Human%20Pathogen%20Candida%20albicans" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000996" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000996&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Genetic Code Alteration Is a Phenotype Diversity Generator in the Human Pathogen Candida albicans';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000996';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Genetic%20Code%20Alteration%20Is%20a%20Phenotype%20Diversity%20Generator%20in%20the%20Human%20Pathogen%20Candida%20albicans http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000996" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000996" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Candida+albicans%22" title="Search for articles in the subject area:'Candida albicans'"><div class="flagText">Candida albicans</div></a>
              <div data-categoryid="18657" data-articleid="24826"
                   data-categoryname="Candida albicans"
                   class="flagImage" title="Flag 'Candida albicans' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="24826"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="24826"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leucine%22" title="Search for articles in the subject area:'Leucine'"><div class="flagText">Leucine</div></a>
              <div data-categoryid="40727" data-articleid="24826"
                   data-categoryname="Leucine"
                   class="flagImage" title="Flag 'Leucine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="24826"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Saccharomyces+cerevisiae%22" title="Search for articles in the subject area:'Saccharomyces cerevisiae'"><div class="flagText">Saccharomyces cerevisiae</div></a>
              <div data-categoryid="32473" data-articleid="24826"
                   data-categoryname="Saccharomyces cerevisiae"
                   class="flagImage" title="Flag 'Saccharomyces cerevisiae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Serine%22" title="Search for articles in the subject area:'Serine'"><div class="flagText">Serine</div></a>
              <div data-categoryid="40197" data-articleid="24826"
                   data-categoryname="Serine"
                   class="flagImage" title="Flag 'Serine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transfer+RNA%22" title="Search for articles in the subject area:'Transfer RNA'"><div class="flagText">Transfer RNA</div></a>
              <div data-categoryid="21239" data-articleid="24826"
                   data-categoryname="Transfer RNA"
                   class="flagImage" title="Flag 'Transfer RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3996'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5621'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2172&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=15209">Candida genus is not monophyletic</a><br>Posted by jstajich</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
