

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic Mdx Mouse</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000427"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000427" />

    <meta name="citation_title" content="Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic Mdx Mouse"/>
    <meta itemprop="name" content="Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic Mdx Mouse"/>

      <meta name="citation_author" content="Massimo Buvoli"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of Colorado at Boulder, Boulder, Colorado, United States of America"/>
      <meta name="citation_author" content="Ada Buvoli"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of Colorado at Boulder, Boulder, Colorado, United States of America"/>
      <meta name="citation_author" content="Leslie A. Leinwand"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of Colorado at Boulder, Boulder, Colorado, United States of America"/>

    <meta name="citation_date" content="2007/5/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000427.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e427"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Function and genetics of dystrophin and dystrophin-related proteins in muscle.; citation_author=DJ Blake; citation_author=A Weir; citation_author=SE Newey; citation_author=KE Davies; citation_journal_title=Physiol Rev; citation_volume=82; citation_number=1; citation_pages=291-329; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanisms of muscular dystrophies: old and new players.; citation_author=KE Davies; citation_author=KJ Nowak; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=7; citation_number=2; citation_pages=762-773; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Construction of a 2.6-Mb contig in yeast artificial chromosomes spanning the human dystrophin gene using an STS-based approach.; citation_author=AJ Coffey; citation_author=RG Roberts; citation_author=ED Green; citation_author=CG Cole; citation_author=R Butler; citation_journal_title=Genomics; citation_volume=12; citation_number=3; citation_pages=474-484; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A yeast artificial chromosome contig containing the complete Duchenne muscular dystrophy gene.; citation_author=AP Monaco; citation_author=AP Walker; citation_author=I Millwood; citation_author=Z Larin; citation_author=H Lehrach; citation_journal_title=Genomics; citation_volume=12; citation_number=4; citation_pages=465-473; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The complete sequence of dystrophin predicts a rod-shaped cytoskeletal protein.; citation_author=M Koenig; citation_author=AP Monaco; citation_author=LM Kunkel; citation_journal_title=Cell; citation_volume=53; citation_number=5; citation_pages=219-226; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Pre-mRNA splicing: awash in a sea of proteins.; citation_author=MS Jurica; citation_author=MJ Moore; citation_journal_title=Mol Cell; citation_volume=12; citation_number=6; citation_pages=5-14; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Insights into the mechanisms of splicing: more lessons from the ribosome.; citation_author=MM Konarska; citation_author=CC Query; citation_journal_title=Genes Dev; citation_volume=19; citation_number=7; citation_pages=2255-2260; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exon recognition in vertebrate splicing.; citation_author=SM Berget; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=8; citation_pages=2411-2414; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Multiple splicing defects in an intronic false exon.; citation_author=H Sun; citation_author=LA Chasin; citation_journal_title=Mol Cell Biol; citation_volume=20; citation_number=9; citation_pages=6414-6425; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases.; citation_author=BJ Blencowe; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=10; citation_pages=106-110; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A general role for splicing enhancers in exon definition.; citation_author=BJ Lam; citation_author=KJ Hertel; citation_journal_title=RNA; citation_volume=8; citation_number=11; citation_pages=1233-1241; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The role of exon sequences in splice site selection.; citation_author=A Watakabe; citation_author=K Tanaka; citation_author=Y Shimura; citation_journal_title=Genes Dev; citation_volume=7; citation_number=12; citation_pages=407-418; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A splicing enhancer in the human fibronectin alternate ED1 exon interacts with SR proteins and stimulates U2 snRNP binding.; citation_author=A Lavigueur; citation_author=H La Branche; citation_author=AR Kornblihtt; citation_author=B Chabot; citation_journal_title=Genes Dev; citation_volume=7; citation_number=13; citation_pages=2405-2417; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The cardiac troponin T alternative exon contains a novel purine-rich positive splicing element.; citation_author=R Xu; citation_author=J Teng; citation_author=TA Cooper; citation_journal_title=Mol Cell Biol; citation_volume=13; citation_number=14; citation_pages=3660-3674; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=General splicing factor SF2/ASF promotes alternative splicing by binding to an exonic splicing enhancer.; citation_author=Q Sun; citation_author=A Mayeda; citation_author=RK Hampson; citation_author=AR Krainer; citation_author=FM Rottman; citation_journal_title=Genes Dev; citation_volume=7; citation_number=15; citation_pages=2598-2608; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Selection of novel exon recognition elements from a pool of random sequences.; citation_author=H Tian; citation_author=R Kole; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=16; citation_pages=6291-6298; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Identification of a new class of exonic splicing enhancers by in vivo selection.; citation_author=LR Coulter; citation_author=MA Landree; citation_author=TA Cooper; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=17; citation_pages=2143-2150; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Selection and characterization of pre-mRNA splicing enhancers: identification of novel SR protein-specific enhancer sequences.; citation_author=TD Schaal; citation_author=T Maniatis; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=18; citation_pages=1705-1719; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Strong RNA splicing enhancers identified by a modified method of cycled selection interact with SR protein.; citation_author=H Tian; citation_author=R Kole; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=19; citation_pages=33833-33839; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sorting out the complexity of SR protein functions.; citation_author=BR Graveley; citation_journal_title=RNA; citation_volume=6; citation_number=20; citation_pages=1197-1211; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SR proteins are ‘locators’ of the RNA splicing machinery.; citation_author=BR Graveley; citation_author=KJ Hertel; citation_author=T Maniatis; citation_journal_title=Curr Biol; citation_volume=9; citation_number=21; citation_pages=R6-7; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Alternative pre-mRNA splicing and proteome expansion in metazoans.; citation_author=T Maniatis; citation_author=B Tasic; citation_journal_title=Nature; citation_volume=418; citation_number=22; citation_pages=236-243; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Building specificity with nonspecific RNA-binding proteins.; citation_author=R Singh; citation_author=J Valcarcel; citation_journal_title=Nat Struct Mol Biol; citation_volume=12; citation_number=23; citation_pages=645-653; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RS domains contact the pre-mRNA throughout spliceosome assembly.; citation_author=KJ Hertel; citation_author=BR Graveley; citation_journal_title=Trends Biochem Sci; citation_volume=30; citation_number=24; citation_pages=115-118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RS domains contact splicing signals and promote splicing by a common mechanism in yeast through humans.; citation_author=H Shen; citation_author=MR Green; citation_journal_title=Genes Dev; citation_volume=20; citation_number=25; citation_pages=1755-1765; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=SR proteins are sufficient for exon bridging across an intron.; citation_author=JM Stark; citation_author=DP Bazett-Jones; citation_author=M Herfort; citation_author=MB Roth; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=26; citation_pages=2163-2168; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The relative strengths of SR protein-mediated associations of alternative and constitutive exons can influence alternative splicing.; citation_author=JM Stark; citation_author=TA Cooper; citation_author=MB Roth; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=27; citation_pages=29838-29842; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Disruption of the splicing enhancer sequence within exon 27 of the dystrophin gene by a nonsense mutation induces partial skipping of the exon and is responsible for Becker muscular dystrophy.; citation_author=N Shiga; citation_author=Y Takeshima; citation_author=H Sakamoto; citation_author=K Inoue; citation_author=Y Yokota; citation_journal_title=J Clin Invest; citation_volume=100; citation_number=28; citation_pages=2204-2210; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Multiple distinct splicing enhancers in the protein-coding sequences of a constitutively spliced pre-mRNA.; citation_author=TD Schaal; citation_author=T Maniatis; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=29; citation_pages=261-273; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Substrate specificities of SR proteins in constitutive splicing are determined by their RNA recognition motifs and composite pre-mRNA exonic elements.; citation_author=A Mayeda; citation_author=GR Screaton; citation_author=SD Chandler; citation_author=XD Fu; citation_author=AR Krainer; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=30; citation_pages=1853-1863; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Purine-rich exon sequences are not necessarily splicing enhancer sequence in the dystrophin gene.; citation_author=T Ito; citation_author=Y Takeshima; citation_author=H Sakamoto; citation_author=H Nakamura; citation_author=M Matsuo; citation_journal_title=Kobe J Med Sci; citation_volume=47; citation_number=31; citation_pages=193-202; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A mechanism for exon skipping caused by nonsense or missense mutations in BRCA1 and other genes.; citation_author=HX Liu; citation_author=L Cartegni; citation_author=MQ Zhang; citation_author=AR Krainer; citation_journal_title=Nat Genet; citation_volume=27; citation_number=32; citation_pages=55-58; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Disruption of an SF2/ASF-dependent exonic splicing enhancer in SMN2 causes spinal muscular atrophy in the absence of SMN1.; citation_author=L Cartegni; citation_author=AR Krainer; citation_journal_title=Nat Genet; citation_volume=30; citation_number=33; citation_pages=377-384; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of functional exonic splicing enhancer motifs recognized by individual SR proteins.; citation_author=HX Liu; citation_author=M Zhang; citation_author=AR Krainer; citation_journal_title=Genes Dev; citation_volume=12; citation_number=34; citation_pages=1998-2012; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Predictive identification of exonic splicing enhancers in human genes.; citation_author=WG Fairbrother; citation_author=RF Yeh; citation_author=PA Sharp; citation_author=CB Burge; citation_journal_title=Science; citation_volume=297; citation_number=35; citation_pages=1007-1013; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Distribution of SR protein exonic splicing enhancer motifs in human protein-coding genes.; citation_author=J Wang; citation_author=PJ Smith; citation_author=AR Krainer; citation_author=MQ Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=36; citation_pages=5053-5062; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Distribution of exonic splicing enhancer elements in human genes.; citation_author=Y Wu; citation_author=Y Zhang; citation_author=J Zhang; citation_journal_title=Genomics; citation_volume=86; citation_number=37; citation_pages=329-336; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A perfect message: RNA surveillance and nonsense-mediated decay.; citation_author=MW Hentze; citation_author=AE Kulozik; citation_journal_title=Cell; citation_volume=96; citation_number=38; citation_pages=307-310; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A premature termination codon in either exon of minute virus of mice P4 promoter-generated pre-mRNA can inhibit nuclear splicing of the intervening intron in an open reading frame-dependent manner.; citation_author=A Gersappe; citation_author=L Burger; citation_author=DJ Pintel; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=39; citation_pages=22452-22458; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Listening to silence and understanding nonsense: exonic mutations that affect splicing.; citation_author=L Cartegni; citation_author=SL Chew; citation_author=AR Krainer; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=40; citation_pages=285-298; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=NASty effects on fibrillin pre-mRNA splicing: another case of ESE does it, but proposals for translation-dependent splice site choice live on.; citation_author=LE Maquat; citation_journal_title=Genes Dev; citation_volume=16; citation_number=41; citation_pages=1743-1753; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Alternatively spliced TCR mRNA induced by disruption of reading frame.; citation_author=J Wang; citation_author=JI Hamilton; citation_author=MS Carter; citation_author=S Li; citation_author=MF Wilkinson; citation_journal_title=Science; citation_volume=297; citation_number=42; citation_pages=108-110; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Nonsense-associated altered splicing: a frame-dependent response distinct from nonsense-mediated decay.; citation_author=J Wang; citation_author=YF Chang; citation_author=JI Hamilton; citation_author=MF Wilkinson; citation_journal_title=Mol Cell; citation_volume=10; citation_number=43; citation_pages=951-957; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mechanistic links between nonsense-mediated mRNA decay and pre-mRNA splicing in mammalian cells.; citation_author=F Lejeune; citation_author=LE Maquat; citation_journal_title=Curr Opin Cell Biol; citation_volume=17; citation_number=44; citation_pages=309-315; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Early nonsense: mRNA decay solves a translational problem.; citation_author=N Amrani; citation_author=MS Sachs; citation_author=A Jacobson; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=7; citation_number=45; citation_pages=415-425; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Quality control of gene expression: a stepwise assembly pathway for the surveillance complex that triggers nonsense-mediated mRNA decay.; citation_author=I Behm-Ansmant; citation_author=E Izaurralde; citation_journal_title=Genes Dev; citation_volume=20; citation_number=46; citation_pages=391-398; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The molecular basis of muscular dystrophy in the mdx mouse: a point mutation.; citation_author=P Sicinski; citation_author=Y Geng; citation_author=AS Ryder-Cook; citation_author=EA Barnard; citation_author=MG Darlison; citation_journal_title=Science; citation_volume=244; citation_number=47; citation_pages=1578-1580; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Expression of the murine Duchenne muscular dystrophy gene in muscle and brain.; citation_author=JS Chamberlain; citation_author=JA Pearlman; citation_author=DM Muzny; citation_author=RA Gibbs; citation_author=JE Ranier; citation_journal_title=Science; citation_volume=239; citation_number=48; citation_pages=1416-1418; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Scanning from an independently specified branch point defines the 3′ splice site of mammalian introns.; citation_author=CW Smith; citation_author=EB Porro; citation_author=JG Patton; citation_author=B Nadal-Ginard; citation_journal_title=Nature; citation_volume=342; citation_number=49; citation_pages=243-247; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The organization of 3′ splice-site sequences in mammalian introns.; citation_author=R Reed; citation_journal_title=Genes Dev; citation_volume=3; citation_number=50; citation_pages=2113-2123; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Beta-thalassemia due to two novel nucleotide substitutions in consensus acceptor splice sequences of the beta-globin gene.; citation_author=C Wong; citation_author=SE Antonarakis; citation_author=SC Goff; citation_author=SH Orkin; citation_author=BG Forget; citation_journal_title=Blood; citation_volume=73; citation_number=51; citation_pages=914-918; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Scanning and competition between AGs are involved in 3′ splice site selection in mammalian introns.; citation_author=CW Smith; citation_author=TT Chu; citation_author=B Nadal-Ginard; citation_journal_title=Mol Cell Biol; citation_volume=13; citation_number=52; citation_pages=4939-4952; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Unexpected inactivation of acceptor consensus splice sequence by a -3 C to T transition in intron 2 of the CFTR gene.; citation_author=T Bienvenu; citation_author=D Hubert; citation_author=N Fonknechten; citation_author=D Dusser; citation_author=JC Kaplan; citation_journal_title=Hum Genet; citation_volume=94; citation_number=53; citation_pages=65-68; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Identification of a novel X-linked gene responsible for Emery-Dreifuss muscular dystrophy.; citation_author=S Bione; citation_author=E Maestrini; citation_author=S Rivella; citation_author=M Mancini; citation_author=S Regis; citation_journal_title=Nat Genet; citation_volume=8; citation_number=54; citation_pages=323-327; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=RNA splice junctions of different classes of eukaryotes: sequence statistics and functional implications in gene expression.; citation_author=MB Shapiro; citation_author=P Senapathy; citation_journal_title=Nucleic Acids Res; citation_volume=15; citation_number=55; citation_pages=7155-7174; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Modification of pre-mRNA processing: application to dystrophin expression.; citation_author=SD Wilton; citation_author=S Fletcher; citation_journal_title=Curr Opin Mol Ther; citation_volume=8; citation_number=56; citation_pages=130-135; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modulation of in vitro splicing of the upstream intron by modifying an intra-exon sequence which is deleted from the dystrophin gene in dystrophin Kobe.; citation_author=Y Takeshima; citation_author=H Nishio; citation_author=H Sakamoto; citation_author=H Nakamura; citation_author=M Matsuo; citation_journal_title=J Clin Invest; citation_volume=95; citation_number=57; citation_pages=515-520; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Splicing analysis disclosed a determinant single nucleotide for exon skipping caused by a novel intraexonic four-nucleotide deletion in the dystrophin gene.; citation_author=VK Tran; citation_author=Y Takeshima; citation_author=Z Zhang; citation_author=M Yagi; citation_author=A Nishiyama; citation_journal_title=J Med Genet; citation_volume=43; citation_number=58; citation_pages=924-930; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An exon skipping-associated nonsense mutation in the dystrophin gene uncovers a complex interplay between multiple antagonistic splicing elements.; citation_author=A Disset; citation_author=CF Bourgeois; citation_author=N Benmalek; citation_author=M Claustres; citation_author=J Stevenin; citation_journal_title=Hum Mol Genet; citation_volume=15; citation_number=59; citation_pages=999-1013; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Enhancer elements activate the weak 3′ splice site of alpha-tropomyosin exon 2.; citation_author=BT Dye; citation_author=M Buvoli; citation_author=SA Mayer; citation_author=CH Lin; citation_author=JG Patton; citation_journal_title=RNA; citation_volume=4; citation_number=60; citation_pages=1523-1536; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=ESEfinder: A web resource to identify exonic splicing enhancers.; citation_author=L Cartegni; citation_author=J Wang; citation_author=Z Zhu; citation_author=MQ Zhang; citation_author=AR Krainer; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=61; citation_pages=3568-3571; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An increased specificity score matrix for the prediction of SF2/ASF-specific exonic splicing enhancers.; citation_author=PJ Smith; citation_author=C Zhang; citation_author=J Wang; citation_author=SL Chew; citation_author=MQ Zhang; citation_journal_title=Hum Mol Genet; citation_volume=15; citation_number=62; citation_pages=2490-2508; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=SR proteins: a conserved family of pre-mRNA splicing factors.; citation_author=AM Zahler; citation_author=WS Lane; citation_author=JA Stolk; citation_author=MB Roth; citation_journal_title=Genes Dev; citation_volume=6; citation_number=63; citation_pages=837-847; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The function of multisite splicing enhancers.; citation_author=KJ Hertel; citation_author=T Maniatis; citation_journal_title=Mol Cell; citation_volume=1; citation_number=64; citation_pages=449-455; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A systematic analysis of the factors that determine the strength of pre-mRNA splicing enhancers.; citation_author=BR Graveley; citation_author=KJ Hertel; citation_author=T Maniatis; citation_journal_title=Embo J; citation_volume=17; citation_number=65; citation_pages=6747-6756; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Exon definition may facilitate splice site selection in RNAs with multiple exons.; citation_author=BL Robberson; citation_author=GJ Cote; citation_author=SM Berget; citation_journal_title=Mol Cell Biol; citation_volume=10; citation_number=66; citation_pages=84-94; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Alternative splicing of the adenylyl cyclase stimulatory G-protein G alpha(s) is regulated by SF2/ASF and heterogeneous nuclear ribonucleoprotein A1 (hnRNPA1) and involves the use of an unusual TG 3′-splice Site.; citation_author=AJ Pollard; citation_author=AR Krainer; citation_author=SC Robson; citation_author=GN Europe-Finner; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=67; citation_pages=15241-15251; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural and functional analysis of an mRNP complex that mediates the high stability of human beta-globin mRNA.; citation_author=J Yu; citation_author=JE Russell; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=68; citation_pages=5879-5888; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cationic liposome-mediated RNA transfection.; citation_author=RW Malone; citation_author=PL Felgner; citation_author=IM Verma; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=69; citation_pages=6077-6081; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Alternative splicing of beta-tropomyosin pre-mRNA: multiple cis-elements can contribute to the use of the 5′- and 3′-splice sites of the nonmuscle/smooth muscle exon 6.; citation_author=T Tsukahara; citation_author=C Casciato; citation_author=DM Helfman; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=70; citation_pages=2318-2325; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Human SR proteins and isolation of a cDNA encoding SRp75.; citation_author=AM Zahler; citation_author=KM Neugebauer; citation_author=JA Stolk; citation_author=MB Roth; citation_journal_title=Mol Cell Biol; citation_volume=13; citation_number=71; citation_pages=4023-4028; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The human dystrophin gene requires 16 hours to be transcribed and is cotranscriptionally spliced.; citation_author=CN Tennyson; citation_author=HJ Klamut; citation_author=RG Worton; citation_journal_title=Nat Genet; citation_volume=9; citation_number=72; citation_pages=184-190; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=An extensive network of coupling among gene expression machines.; citation_author=T Maniatis; citation_author=R Reed; citation_journal_title=Nature; citation_volume=416; citation_number=73; citation_pages=499-506; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Human transformer 2beta and SRp55 interact with a calcitonin-specific splice enhancer.; citation_author=Q Tran; citation_author=TP Coleman; citation_author=JR Roesser; citation_journal_title=Biochim Biophys Acta; citation_volume=1625; citation_number=74; citation_pages=141-152; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Order of intron removal during splicing of endogenous adenine phosphoribosyltransferase and dihydrofolate reductase pre-mRNA.; citation_author=O Kessler; citation_author=Y Jiang; citation_author=LA Chasin; citation_journal_title=Mol Cell Biol; citation_volume=13; citation_number=75; citation_pages=6211-6222; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Localization of an exonic splicing enhancer responsible for mammalian natural trans-splicing.; citation_author=C Caudevilla; citation_author=C Codony; citation_author=D Serra; citation_author=G Plasencia; citation_author=R Roman; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=76; citation_pages=3108-3115; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Modulation of exon skipping by high-affinity hnRNP A1-binding sites and by intron elements that repress splice site utilization.; citation_author=M Blanchette; citation_author=B Chabot; citation_journal_title=Embo J; citation_volume=18; citation_number=77; citation_pages=1939-1952; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=High-affinity hnRNP A1 binding sites and duplex-forming inverted repeats have similar effects on 5′ splice site selection in support of a common looping out and repression mechanism.; citation_author=FU Nasim; citation_author=S Hutchison; citation_author=M Cordeau; citation_author=B Chabot; citation_journal_title=RNA; citation_volume=8; citation_number=78; citation_pages=1078-1089; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Intronic binding sites for hnRNP A/B and hnRNP F/H proteins stimulate pre-mRNA splicing.; citation_author=R Martinez-Contreras; citation_author=JF Fisette; citation_author=FU Nasim; citation_author=R Madden; citation_author=M Cordeau; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=79; citation_pages=e21; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A simple and rapid method for generating a deletion by PCR.; citation_author=Y Imai; citation_author=Y Matsushima; citation_author=T Sugimura; citation_author=M Terada; citation_journal_title=Nucleic Acids Res; citation_volume=19; citation_number=80; citation_pages=2785; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Human Upf proteins target an mRNA for nonsense-mediated decay when bound downstream of a termination codon.; citation_author=J Lykke-Andersen; citation_author=MD Shu; citation_author=JA Steitz; citation_journal_title=Cell; citation_volume=103; citation_number=81; citation_pages=1121-1131; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Suppression of nonsense mutations in cell culture and mice by multimerized suppressor tRNA genes.; citation_author=M Buvoli; citation_author=A Buvoli; citation_author=LA Leinwand; citation_journal_title=Mol Cell Biol; citation_volume=20; citation_number=82; citation_pages=3116-3124; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Effects of nonsense mutations on nuclear and cytoplasmic adenine phosphoribosyltransferase RNA.; citation_author=O Kessler; citation_author=LA Chasin; citation_journal_title=Mol Cell Biol; citation_volume=16; citation_number=83; citation_pages=4426-4435; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Functional crosstalk between exon enhancers, polypyrimidine tracts and branchpoint sequences.; citation_author=M Buvoli; citation_author=SA Mayer; citation_author=JG Patton; citation_journal_title=Embo J; citation_volume=16; citation_number=84; citation_pages=7174-7183; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000427" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic Mdx Mouse"/>
    <meta name="twitter:description" content="BackgroundPre-mRNA splicing, the removal of introns from RNA, takes place within the spliceosome, a macromolecular complex composed of five small nuclear RNAs and a large number of associated proteins. Spliceosome assembly is modulated by the 5&prime; and 3&prime; splice site consensus sequences situated at the ends of each intron, as well as by exonic and intronic splicing enhancers/silencers recognized by SR and hnRNP proteins. Nonsense mutations introducing a premature termination codon (PTC) often result in the activation of cellular quality control systems that reduce mRNA levels or alter the mRNA splicing pattern. The mdx mouse, a commonly used genetic model for Duchenne muscular dystrophy (DMD), lacks dystrophin by virtue of a premature termination codon (PTC) in exon 23 that also severely reduces the level of dystrophin mRNA. However, the effect of the mutation on dystrophin RNA processing has not yet been described.Methodology/Principal FindingUsing combinations of different biochemical and cellular assays, we found that the mdx mutation partially disrupts a multisite exonic splicing enhancer (ESE) that is recognized by a 40 kDa SR protein. In spite of the presence of an inefficient intron 22 3&prime; splice site containing the rare GAG triplet, the mdx mutation does not activate nonsense-associated altered splicing (NAS), but induces exclusively nonsense-mediated mRNA decay (NMD). Functional binding sites for SR proteins were also identified in exon 22 and 24, and in vitro experiments show that SR proteins can mediate direct association between exon 22, 23, and 24.Conclusions/SignificanceOur findings highlight the complex crosstalk between trans-acting factors, cis-elements and the RNA surveillance machinery occurring during dystrophin mRNA processing. Moreover, they suggest that dystrophin exon&ndash;exon interactions could play an important role in preventing mdx exon 23 skipping, as well as in facilitating the pairing of committed splice sites."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000427.g006"/>

  <meta property="og:title" content="Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic Mdx Mouse" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000427" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=338'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5740'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1073&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000427">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000427" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000427&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic <i>Mdx</i> Mouse
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Massimo Buvoli, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular, Cellular and Developmental Biology, University of Colorado at Boulder, Boulder, Colorado, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ada Buvoli, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular, Cellular and Developmental Biology, University of Colorado at Boulder, Boulder, Colorado, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leslie A. Leinwand
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Leslie.Leinwand@Colorado.edu">Leslie.Leinwand@Colorado.edu</a></p>

                <p>Affiliation:
                  Department of Molecular, Cellular and Developmental Biology, University of Colorado at Boulder, Boulder, Colorado, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 09, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000427</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000427" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000427" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000427" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000427" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000427-g001" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000427-g002" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000427-g003" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000427-g004" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000427-g005" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000427-g006" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000427">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000427" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Pre-mRNA splicing, the removal of introns from RNA, takes place within the spliceosome, a macromolecular complex composed of five small nuclear RNAs and a large number of associated proteins. Spliceosome assembly is modulated by the 5′ and 3′ splice site consensus sequences situated at the ends of each intron, as well as by exonic and intronic splicing enhancers/silencers recognized by SR and hnRNP proteins. Nonsense mutations introducing a premature termination codon (PTC) often result in the activation of cellular quality control systems that reduce mRNA levels or alter the mRNA splicing pattern. The <em>mdx</em> mouse, a commonly used genetic model for Duchenne muscular dystrophy (DMD), lacks dystrophin by virtue of a premature termination codon (PTC) in exon 23 that also severely reduces the level of dystrophin mRNA. However, the effect of the mutation on dystrophin RNA processing has not yet been described.</p>


<h3>Methodology/Principal Finding</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using combinations of different biochemical and cellular assays, we found that the <em>mdx</em> mutation partially disrupts a multisite exonic splicing enhancer (ESE) that is recognized by a 40 kDa SR protein. In spite of the presence of an inefficient intron 22 3′ splice site containing the rare GAG triplet, the <em>mdx</em> mutation does not activate nonsense-associated altered splicing (NAS), but induces exclusively nonsense-mediated mRNA decay (NMD). Functional binding sites for SR proteins were also identified in exon 22 and 24, and <em>in vitro</em> experiments show that SR proteins can mediate direct association between exon 22, 23, and 24.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our findings highlight the complex crosstalk between trans-acting factors, cis-elements and the RNA surveillance machinery occurring during dystrophin mRNA processing. Moreover, they suggest that dystrophin exon–exon interactions could play an important role in preventing <em>mdx</em> exon 23 skipping, as well as in facilitating the pairing of committed splice sites.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Buvoli M, Buvoli A, Leinwand LA (2007) Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic <em>Mdx</em> Mouse. PLoS ONE 2(5):
          e427.
            doi:10.1371/journal.pone.0000427</p><p><strong>Academic Editor: </strong>Janet Kelso, Max Planck Institute for Evolutionary Anthropology, Germany</p><p><strong>Received:</strong> February 7, 2007; <strong>Accepted:</strong> April 15, 2007; <strong>Published:</strong> May 9, 2007</p><p><strong>Copyright:</strong> © 2007 Buvoli et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work has been supported by a grant from the NIH (GM29090) to L.A.L.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Dystrophin is a 427 kDa structural protein that connects the actin cytoskeleton with the basal lamina and protects the myofibers from mechanical stress produced during muscle contraction <a href="#pone.0000427-Blake1">[1]</a>, <a href="#pone.0000427-Davies1">[2]</a>. Mutations in the very large dystrophin gene (2.4 million bases) are associated with both Duchenne muscular dystrophy (DMD), characterized by severe and progressive muscle degeneration, and the milder Becker muscular dystrophy (BMD). Dystrophin gene expression is controlled by several independent promoters, alternative splicing, and the use of different polyA<sup>+</sup> signals. Although the gene covers about 2.4 megabases on the short arm of the X chromosome <a href="#pone.0000427-Coffey1">[3]</a>, <a href="#pone.0000427-Monaco1">[4]</a>, the 79 exons only encode a 14 kb mRNA <a href="#pone.0000427-Koenig1">[5]</a>. Thus, 99.4% of hnRNA corresponds to intervening sequences that have to be removed during the splicing reaction. Pre-mRNA splicing consists of two consecutive trans-esterification reactions, and is carried out by the spliceosome, an RNA-protein complex composed of five small ribonucleoparticles called snRNPs and a large number of non-snRNP splicing factors <a href="#pone.0000427-Jurica1">[6]</a>, <a href="#pone.0000427-Konarska1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Splice site selection occurs by precise identification of poorly conserved consensus sequences located at the intron-exon boundaries. Commitment for a given set of splice sites appears to be enhanced by protein cross-talk across each exon, a process called exon definition <a href="#pone.0000427-Berget1">[8]</a>, and by repression of numerous pseudosplice sites that are usually present in primary transcripts <a href="#pone.0000427-Sun1">[9]</a>. Specific cis-elements located in the exons, known as exonic splicing enhancers (ESEs), promote both constitutive and regulated splicing <a href="#pone.0000427-Blencowe1">[10]</a> and appear to be involved in exon definition <a href="#pone.0000427-Lam1">[11]</a>. Although ESEs were initially identified in alternative spliced exons as purine-rich regions <a href="#pone.0000427-Watakabe1">[12]</a>–<a href="#pone.0000427-Sun2">[15]</a>, different motifs have been identified by different iterative selection strategies <a href="#pone.0000427-Tian1">[16]</a>–<a href="#pone.0000427-Tian2">[19]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>ESEs are recognized by SR proteins, a group of highly conserved modular splicing factors containing a N-terminal RNA-recognition domain (RRM), that mediates RNA binding, and a C-terminal arginine/serine rich domain, required for protein-protein interactions <a href="#pone.0000427-Graveley1">[20]</a>. When bound to ESEs, SR proteins recruit and facilitate the binding of different components of the spliceosome to the nearby intron <a href="#pone.0000427-Blencowe1">[10]</a>, <a href="#pone.0000427-Graveley2">[21]</a>, <a href="#pone.0000427-Maniatis1">[22]</a>. Moreover, they promote spliceosome assembly by directly contacting the pre-mRNA <a href="#pone.0000427-Singh1">[23]</a>–<a href="#pone.0000427-Shen1">[25]</a>, as well as splice site selection by bringing exons into proximity <a href="#pone.0000427-Stark1">[26]</a>, <a href="#pone.0000427-Stark2">[27]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Even though the majority of ESEs have been found in enhancer-dependent introns containing a weak splice site, their discovery in constitutive exons <a href="#pone.0000427-Shiga1">[28]</a>–<a href="#pone.0000427-Cartegni1">[33]</a>, together with the finding that protein-coding sequences are enriched for potential SR binding sites <a href="#pone.0000427-Liu2">[34]</a>–<a href="#pone.0000427-Wu1">[37]</a>, suggests that they could be present in the majority of exons. Fortuitous disruption of ESEs by missense, silent, and nonsense mutations, promotes exon skipping, intron retention, or exon size reduction. This process is different from the controversial nonsense–associated altered splicing (NAS), a post-transcriptional control pathway that detects disruptions in the open reading frame of precursor mRNAs. By regulating the synthesis of alternatively spliced isoforms skipping the nonsense mutation, NAS could prevent the production of deleterious truncated proteins <a href="#pone.0000427-Hentze1">[38]</a>–<a href="#pone.0000427-Wang3">[43]</a>. While NAS affects only the splicing process, nonsense-mediated mRNA decay (NMD), the primary cellular response to mRNA carrying a premature termination codon (PTC), rapidly degrades fully spliced transcripts. NMD is activated during the first round of translation by PTCs placed at&gt;50–55 nucleotides from the last exon–exon junction that is marked, after splicing, with an exon junction complex (EJC) that interacts with the termination factors <a href="#pone.0000427-Lejeune1">[44]</a>–<a href="#pone.0000427-BehmAnsmant1">[46]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The <em>mdx</em> mouse, the most widely used animal model for Duchenne muscular dystrophy, carries an ochre nonsense mutation in dystrophin exon 23 <a href="#pone.0000427-Sicinski1">[47]</a> that drastically reduces the abundance of dystrophin mRNA and consequently abolishes dystrophin expression <a href="#pone.0000427-Chamberlain1">[48]</a>. Interestingly, a rare G residue precedes the terminal dinucleotide AG of intron 22, a configuration that is also present in the wild type strain. <em>In vitro</em>, GAG does not appear to function efficiently for the second step of splicing, and CAG to GAG mutations often result in exon skipping <em>in vivo</em> <a href="#pone.0000427-Smith1">[49]</a>–<a href="#pone.0000427-Bione1">[54]</a>. These findings explain the very low presence of GAG trinucleotides at natural 3′ splice sites <a href="#pone.0000427-Shapiro1">[55]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>A promising RNA “re-framing” strategy based on antisense oligonucleotides that promote skipping of exons carrying nonsense mutations has recently been developed and tested in the <em>mdx</em> mouse system <a href="#pone.0000427-Wilton1">[56]</a>. However, the potential impact of the <em>mdx</em> mutation on dystrophin mRNA processing has not yet been examined. In this study we investigated the molecular events triggered by the <em>mdx</em> mutation on mRNA metabolism, the functional activity of the GAG present in intron 22, and the potential relevance of SR proteins on dystrophin mRNA splicing.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Identification of dystrophin exon 23 splicing enhancers</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Since it has been reported that ESEs are present in several dystrophin exons and their inactivation can cause perturbations in the gene splicing pattern <a href="#pone.0000427-Shiga1">[28]</a>, <a href="#pone.0000427-Takeshima1">[57]</a>–<a href="#pone.0000427-Disset1">[59]</a>, we began our analysis by determining whether exon 23 contains ESEs. To this aim, different regions of the exon 23 were cloned into the enhancer-dependent Dpy2 pre-mRNA <a href="#pone.0000427-Dye1">[60]</a>, an heterologous construct that carries a weak polypyrimidine tract and does not splice <em>in vitro</em> (<a href="#pone-0000427-g001">Figure 1</a>, Dpy2). A minimal region, capable of activating Dpy2 splicing <em>in vitro,</em> was mapped to the first 90 nucleotides of exon 23 (<a href="#pone-0000427-g001">Figure 1</a>, D90). The <em>mdx</em> C→T transversion located in a 12 nucleotides conserved region (M1element), did not disrupt any of the potential high-score SR motifs for SF2/ASF, SC35, SRp40 and SRp55 <a href="#pone.0000427-Cartegni3">[61]</a>, <a href="#pone.0000427-Smith3">[62]</a> but reduced D90 splicing more than 40% (<a href="#pone-0000427-g001">Figure 1</a>, D90<em>mdx</em>). Mutagenesis of a purine-rich region (R1 element) matching a splicing enhancer previously identified by <em>in vitro</em> selection <a href="#pone.0000427-Tian1">[16]</a>, <a href="#pone.0000427-Schaal1">[18]</a>, completely abolished splicing activation (<a href="#pone-0000427-g001">Figure 1</a>, D90R1M) that was not restored by downstream exonic sequences (data not shown). To define further the ESE, we determined that the first 40 nucleotides of exon 23 containing only M1, did not activate Dpy2 splicing (data not shown). However, their deletion abolished Dpy2 activation (<a href="#pone-0000427-g001">Figure 1</a>, D1-40Δ) suggesting that the exon 23 ESE is a multipartite enhancer consisting of two elements, M1 and R1. Finally, when compared with the well-characterized α-tropomyosin exon 2 enhancer elements (Dpy2E construct), exon 23 ESE stimulated Dpy2 splicing less efficiently (2.5 fold reduction).</p>
<div class="figure" id="pone-0000427-g001"><div class="img"><a name="pone-0000427-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Exon 23 contains purine-rich elements that activate splicing of a heterologous substrate containing a weak 3′ splice site.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The sequence of the first 90 nucleotides of dystrophin exon 23 is shown at the top of the figure. Arrows identify the natural mutation (C→T) occurring in the <em>mdx</em> mouse and the mutations that disrupt the purine-rich region R1. The borders of the M1 element are defined by the homology (12 of 12 positions) shared with the human dystrophin exon 23. Different regions of dystrohin exon 23 were inserted in the second exon of Dpy2, an enhancer-dependent mRNA containing a weak pyrimidine stretch <a href="#pone.0000427-Dye1">[60]</a>. The structure of the chimeric pre-mRNA substrates subjected to <em>in vitro</em> splicing reactions is shown at the bottom right of the figure. White boxes indicate Dpy2 exons, black boxes indicate different regions of the dystrophin exon 23, dashed boxes indicate portions of exon 23 not included in the substrates, white vertical lines indicate the <em>mdx</em> and R1 mutations. The gray box in Dpy2E identifies the two splicing enhancers found in the α-tropomyosin exon 2 gene <a href="#pone.0000427-Dye1">[60]</a>. The structures and the mobility of the products and intermediates of splicing are shown on the sides of the gel. D172-212Δ RNA was used as a positive control for D1-40Δ to rule out any exon size inhibitory effect on splicing. Asterisks indicate the position of the spliced RNAs. Splicing reactions, carried out for 70 min at 30°C, were resolved on a 12% denaturing polyacrylamide gel. Comparable results were obtained in three independent experiments.</p>
<span>doi:10.1371/journal.pone.0000427.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>These data indicate that the constitutive dystrophin exon 23 contains a multisite-ESE that is partially inactivated by the <em>mdx</em> nonsense mutation. Thus, reduction of dystrophin mRNA levels observed in the <em>mdx</em> mouse could result from aberrant splicing caused by the disruption of this splicing cis-element.</p>


<h4>SR proteins interact with dystrophin exon 23 splicing enhancer</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p><strong>S</strong>R proteins bound to ESEs promote spliceosome assembly and pre-mRNA processing. To determine whether SR proteins could form a stable complex with the exon 23 ESE, an RNA mobility shift assay was performed. As shown in <a href="#pone-0000427-g002">Figure 2</a>, a retarded RNA-protein complex was resolved on a native gel following incubation of <sup>32</sup>P-labeled D90 RNA and calf thymus purified SR proteins (second lane from left). An excess of D90 cold competitor successfully prevented the efficient formation of the complex, whereas the competitor carrying the <em>mdx</em> mutation (D90<em>mdx</em>) was much less effective. In contrast, the double mutant D90<em>mdx</em>R1M was unable to significantly inhibit complex formation even at the highest concentration of competitor tested. Thus, SR proteins specifically bind exon 23 ESE and could mediate the enhancing activity of D90 observed in the heterologous Dpy2 construct. As judged by the effect of the D90<em>mdx</em> competitor on complex formation, it appears that the <em>mdx</em> mutation interferes with SR assembly on exon 23 ESE.</p>
<div class="figure" id="pone-0000427-g002"><div class="img"><a name="pone-0000427-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>SR proteins bind exon 23 enhancer elements.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p><sup>32</sup>P labeled D90 RNA was incubated at 30°C for 10 min with purified SR proteins prepared from calf thymus as previously described <a href="#pone.0000427-Zahler1">[63]</a>. Binding reactions performed in the absence or in the presence of increasing concentrations of cold competitor RNAs (with molar excess of 30, 60, 90 and 120 fold, as indicated by the gradients above the gel lanes) were fractionated on a 4% non-denaturing polyacrylamide gel. The schematic representation of the RNAs is depicted as in <a href="#pone-0000427-g001">Figure 1</a>. The positions of free RNA and bound complexes are shown. Comparable results were obtained in two independent experiments.</p>
<span>doi:10.1371/journal.pone.0000427.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>UV cross-linking experiments were then carried out to determine which SR protein(s) might directly contact the M1 and/or the R1 elements. Uniformly <sup>32</sup>P-labeled D90 RNA was incubated with purified SR proteins, UV irradiated and RNase digested as described in the <a href="#s4">Materials and Methods</a> section. The resulting radioactive complexes were then resolved by SDS-PAGE. As shown in <a href="#pone-0000427-g003">Figure 3A</a>, a 40 kDa SR protein was efficiently cross-linked to the first 90 nucleotides of exon 23 containing both M1 and R1 (D90). A reduction in complex formation was observed after deletion of R1 (D90ΔR1), confirming the functional relationship between M1 and R1 previously observed in the <em>in vitro</em> splicing reaction. A 40 kDa protein was also the only polypeptide cross-linked to the whole exon 23 RNA (D23). Nevertheless, the comparison between the intensity of the D90ΔR1 and D23ΔR1 signals (with a difference of about 2.5 fold) strongly suggests that another 40 kDa protein binding site is located downstream of D90. Binding specificity for the M1 and R1 sites was then demonstrated by the addition to the reactions of the mutant cold competitors D90<em>mdx</em> and D90R1M. As shown in <a href="#pone-0000427-g003">Figure 3B</a>, both RNAs had a similar effect in reducing the 40 kDa cross-linking when compared with the wild type D90. Selective enrichment of the 40 kDa protein in our SR preparation was ruled out by Coomassie-stained SDS-PAGE gel that showed the expected SR protein relative ratio (<a href="#pone-0000427-g003">Figure 3C</a>, SR lane) <a href="#pone.0000427-Zahler1">[63]</a>. Thus, the natural <em>mdx</em> mutation could impair splicing efficiency by reducing the binding of a 40 kDa SR protein to exon 23 ESE.</p>
<div class="figure" id="pone-0000427-g003"><div class="img"><a name="pone-0000427-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>UV cross-linking of a 40 kDa SR protein to exon 23.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A) Different <sup>32</sup>P labeled RNAs represented as in <a href="#pone-0000427-g001">Figure 1</a> were incubated at 30°C for 10 min with purified SR proteins. Following UV cross-linking and RNAse A/T1 digestion, <sup>32</sup>P labeled proteins were analyzed by SDS-PAGE. Molecular weight markers are shown on the left. B) Cross-linking of <sup>32</sup>P labeled D90 RNA was performed in the presence of increasing amounts of cold competitor RNAs (10 and 30 fold molar excess, as indicated by the gradients above the gel lanes). C) Coomassie–stained SDS-PAGE separated SR proteins (7 µg). Comparable results were obtained in three independent experiments.</p>
<span>doi:10.1371/journal.pone.0000427.g003</span></div>

<h4><em>In vivo</em> analysis of mutations affecting exon 23 ESE activity</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The relevance of the <em>mdx</em> and R1 mutations and their potential effects on dystrophin mRNA metabolism were next studied in cell culture. COS 7 cells were transfected with model substrates carrying different portions of the dystrophin gene. Two-exon constructs, containing the genomic sequence spanning from exon 22 to exon 23, were utilized to study the effect of the mutations on mRNA splicing. Three-exon constructs, containing the genomic sequence spanning from exon 22 to exon 24 and therefore sensitive to mRNA surveillance, were employed to examine both NAS and NMD activities. Translation start and termination codons were inserted in frame with the dystrophin exons at the beginning of exon 22 and at the end of exon 23 or 24 respectively. Twenty-four hours after transfection, total RNA was isolated and quantitative analyses were carried out on Northern blots normalized to the internal control neomycin (see <a href="#s4">Materials and Methods</a>). As shown in <a href="#pone-0000427-g004">Figure 4A</a>, the <em>mdx</em> mutation decreased the levels of RNA to ~50% (D22-23 <em>mdx</em>), while the R1 mutation (D22-23 R1M) had a milder phenotype than predicted from the <em>in vitro</em> splicing experiments (<a href="#pone-0000427-g001">Figure 1</a>, D90R1M), reducing the processing of the D22-23 R1M RNA to ~85%. The observation that the simultaneous presence of the two mutations had an additive negative effect on splicing (D22-23 <em>mdx</em>R1M construct, reduction to ~35%) probably reflect the previous finding that only one splicing enhancer complex can interact with the spliceosome components at any given time <a href="#pone.0000427-Hertel2">[64]</a>, <a href="#pone.0000427-Graveley3">[65]</a>.</p>
<div class="figure" id="pone-0000427-g004"><div class="img"><a name="pone-0000427-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Splicing of dystrophin mini-genes <em>in vivo.</em></span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Cos 7 cells were transfected with plasmids expressing different portions of the genomic region of the dystrophin gene spanning from exon 22 to exon 23/24. Northern blot analysis was carried out on total RNA prepared 24 hr after transfection, as described in <a href="#s4">Materials and Methods</a>. Band intensities, quantified by phosphorimager analysis, were normalized for transfection efficiency and RNA recovery to the level of the co-expressed neomycin mRNA (neo). Normalized values were expressed as a percentage of wt mRNAs that were defined as 100. The schematic structure of the substrates is shown on the right part of both panels A and B. Dystrophin exons are identified as in <a href="#pone-0000427-g001">Figure 1</a>. β-globin exons are shown as gray boxes while the 5′ and 3′ UTRs as white thinner boxes. (A) Two exon mini-genes carrying dystrophin exons 22 and 23. (B) Three exon mini-genes carrying dystrophin exon 22 and 23 and 24, chimeric constructs, and the β-globin system (β−globin wt and β−globin 39 carrying a nonsense mutation at codon 39) used as a reference <a href="#pone.0000427-LykkeAndersen1">[81]</a>. Comparable results were obtained in three independent experiments. (C) Left panel: autoradiogram of a representative quantitative RT-PCR performed as described in <a href="#s4">Materials and Methods</a>. The upper bands correspond to RT-PCR products from dystrophin mRNA obtained with primers located in exons 22 and 24; the lower bands correspond to the neomycin gene used as an internal control. −and+lanes correspond to reactions carried out in absence or presence of reverse transcriptase. Right panel: non-quantitative RT-PCR performed as described in <a href="#s4">Materials and Methods</a>. Mobility of the predicted splicing products is shown on the right. The PCR product deriving from skipping of exon 23 was not detected.</p>
<span>doi:10.1371/journal.pone.0000427.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Since it has been shown that the trinucleotide GAG severely slows down the second step of splicing <em>in vitro</em> <a href="#pone.0000427-Smith1">[49]</a>, <a href="#pone.0000427-Smith2">[52]</a>, and in different human genetic diseases is associated with aberrant splicing phenotypes <a href="#pone.0000427-Wong1">[51]</a>, <a href="#pone.0000427-Bienvenu1">[53]</a>, <a href="#pone.0000427-Bione1">[54]</a>, we next investigated if the presence of the ESE in the constitutive exon 23 is required to activate the 3′ splice site of intron 22 containing the rare G residue at position −3. Mutant mini-genes carrying the canonical CAG trinucleotide at the 3′ splice site of intron 22 were generated and their activity examined. As shown in <a href="#pone-0000427-g004">Figure 4A</a>, splicing increased about 2 fold for both the wild-type and the <em>mdx</em> RNA (CAG exon 23 wt, CAG exon 23 <em>mdx</em>), but their relative abundance perfectly matched the ratio observed in the GAG constructs (~50% reduction for the <em>mdx</em> RNA). The possibility that intron 23 5′ splice site could enhance GAG recognition <a href="#pone.0000427-Robberson1">[66]</a> was ruled out by the analysis of the three-exon constructs carrying the trinucleotide CAG. In this case, splicing stimulation, as well as comparative RNA ratios, perfectly matched the results previously obtained with the two-exon constructs (data not shown). Thus, as opposed to other more specialized ESEs involved in the activation of non-canonical 3′ splice sites <a href="#pone.0000427-Pollard1">[67]</a>, exon 23 ESE appears to be a constitutive element, not functionally dedicated to activate the GAG located at the 3′ splice site of intron 22.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Surprisingly, the presence of the <em>mdx</em> mutation had essentially no effect on RNA abundance of the three-exon constructs as determined by both Northern blot (<a href="#pone-0000427-g004">Figure 4B</a> D22-23-24 wt, D22-23-24 <em>mdx</em>) and quantitative RT-PCR (<a href="#pone-0000427-g004">Figure 4C left panel</a>). In addition, no levels of exon 23 skipped RNA were detected when the radioactive PCR was carried out for 34 cycles (data not shown). Moreover, Western blot analysis revealed that exon 22-23-24 mRNA was competent for translation, producing a polypeptide of about 20 kDa (data not shown). The discovery that the levels of expression of the three-exon mini-genes (comparable, after normalization, to the two exon constructs) were about 4-5 fold lower than those of the β-globin constructs (<a href="#pone-0000427-g004">Figure 4B</a>, compare D22-23-24 wt with βwt), prompted us to investigate if mRNA instability was interfering with NMD detection. To address this issue, we built new hybrid constructs fused to the β-globin 3′ UTR, that carries stabilizing cis-elements recognized by cytoplasmatic RNP complexes <a href="#pone.0000427-Yu1">[68]</a>. To boost the efficiency of translation we also linked the β-globin 5′UTR and exon 1 to dystrophin exon 22 <a href="#pone.0000427-Malone1">[69]</a>. Examination of these constructs revealed a 3–4 fold increase in RNA levels, with a significant reduction in the abundance of the mRNA carrying the <em>mdx</em> mutation to ~60% (<a href="#pone-0000427-g004">Figure 4B</a>, β22-23-24 wt and β22-23-24 <em>mdx</em>). Interestingly, similar results were obtained in the presence of the β-globin 3′UTR alone (22-23-24 3′β constructs), while no effect on dystrophin mRNA metabolism was detected in constructs carrying the β-globin 5′UTR alone (<a href="#pone-0000427-g004">Figure 4B</a>, 5′β22-23-24wt and <em>mdx</em>). These results, together with the finding that NAS activity was not detectable in the β-globin hybrid constructs either (<a href="#pone-0000427-g004">Figure 4C, right panel</a>), support the hypothesis that reduction of dystrophin mRNA observed <em>in vivo</em> is probably due to NMD.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The finding that the D22-23-24, but not the D22-23 mini-genes constructs, were insensitive to the presence of the <em>mdx</em> mutation, also indicates that the presence of intron 23 and exon 24 compensate for the partial inactivation of exon 23 ESE. The hypothesis that splicing between exons 23 and 24 could compensate for the <em>mdx</em> mutation by creating a new ESE <a href="#pone.0000427-Tsukahara1">[70]</a> was then examinated by testing RNA substrates containing juxtaposed exons 23 and 24. Although RNA abundance was still reduced by the <em>mdx</em> mutation, only a mild compensatory effect (~10–15%) was observed with these constructs (data not shown).</p>


<h4>SR proteins promote dystrophin exon bridging <em>in vitro</em></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In addition to their involvement in the splicing reaction, SR proteins and ESEs appear to mediate exon/exon association, a process that, bringing exons into proximity, could facilitate selection of the correct 5′ and 3′ splice sites <a href="#pone.0000427-Stark1">[26]</a>, <a href="#pone.0000427-Stark2">[27]</a>. In a gene like dystrophin, characterized by large introns, exon-bridging could play an important role before mRNA catalysis. Moreover, if not affected by the <em>mdx</em> mutation, associations between exon 22, 23 and 24, could force the inclusion of exon 23 and prevent its skipping.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To determine whether SR proteins interact with the constitutive exons 22 and 24, we performed UV crosslinking experiments with purified SR proteins, uniformly <sup>32</sup>P-labeled exon 22 and 24 RNAs and nonspecific cold competitor RNA,. This analysis revealed that, in addition to a 40 kDa protein(s), exons 22 and 24 efficiently interacted with two other SR proteins, migrating at ~34 kDa (<a href="#pone-0000427-g005">Figure 5A</a>). Neither of these SR proteins cross-linked to the antisense-exons used as controls for binding specificity (exon 22A and exon 24A). Of the known SR proteins, ASF/SF2, SC35 and 9G8 migrate as 34 kDa peptides.</p>
<div class="figure" id="pone-0000427-g005"><div class="img"><a name="pone-0000427-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>SR proteins interact with dystrophin exon 22 and 24.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A) Different SR proteins associate with exons 22 and 24. <sup>32</sup>P labeled exon 22 and 24 RNAs, incubated with SR proteins were subjected to UV-crosslinking as described in <a href="#pone-0000427-g003">Figure 3</a>. Molecular weight markers are shown at the left; exon 22A and 24A correspond to the antisense RNAs used as controls. B) Exon 24, but not exon 22 activates splicing of the enhancer-dependent Dpy2 RNA. Different regions of dystrohin exon 22 (145 nt) and 24 (113 nt) were cloned in the second exon of the enhancer-dependent Dpy2 substrate. The structure of the chimeric constructs subjected to <em>in vitro</em> splicing reactions is shown on the right of the panel, with black boxes indicating exon 22 and 24 and dashed boxes the portion of the corresponding exon not included in the pre-mRNA. A indicates the anti-sense orientation of each region, while numbers after the slash indicate their nucleotide length. Dpy2 and Dpy2E are defined as in <a href="#pone-0000427-g001">Figure 1</a>. The structures and the mobility of the products and intermediates of splicing are shown on both sides of the panel. Asterisks indicate the position of the spliced RNAs. The molecular weigh of the band detected below the D24/60 RNA precursor does not correspond to the expected splicing product. <em>In vitro</em> splicing reactions were carried out as in <a href="#pone-0000427-g001">Figure 1</a>. Comparable results were obtained in two independent experiments. C) Schematic representation of SR binding sites within dystrophin exon 22 and 24 computed by ESEfinder <a href="#pone.0000427-Cartegni3">[61]</a>, <a href="#pone.0000427-Smith3">[62]</a>.</p>
<span>doi:10.1371/journal.pone.0000427.g005</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>To determine whether SR proteins bound to the constitutive exons 22 and 24 could also activate splicing in an enhancer-dependent context, we cloned different regions of the two exons in the Dpy2 pre-mRNA. As shown in <a href="#pone-0000427-g005">Figure 5B</a>, only the entire exon 24 was able to stimulate Dpy2 splicing quite efficiently. However, it is important to keep in mind that the strength of each ESE is determined by different parameters <a href="#pone.0000427-Graveley3">[65]</a>. Thus, association of particular SR proteins with constitutive exons can result in the formation of complexes involved only in the basic splicing reaction and not having enough potency for activating weak splice sites (see <a href="#s3">Discussion</a>).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>We next asked whether SR proteins and ESEs could directly promote dystrophin exon bridging <em>in vitro</em>. Biotinylated <sup>32</sup>P-labeled exon RNAs were incubated with non-biotinylated <sup>32</sup>P-labeled exon RNAs and purified SR proteins diluted to the same concentration measured in splicing HeLa nuclear extract (~30 µg/ml) <a href="#pone.0000427-Zahler2">[71]</a>. Possible ternary complexes (biotinylated exon/SR proteins/non-biotinylated exon) were purified with streptavidin magnetic beads and, after extensive washing, RNA was extracted and resolved on denaturing gels. This assay revealed that exon/exon associations were efficiently promoted in the presence of SR proteins. In fact, a substantial enrichment of co-purified non-biotinylated exons (~10 fold over the experimental background), was consistently detected after addition of SR proteins to the reactions (<a href="#pone-0000427-g006">Figure 6</a>, compare lanes 8–11 with lane 12). However, neither the presence of the <em>mdx</em> mutation nor the deletion of the R1 element affected the efficient formation of complexes between exon 22, 24 and biotinylated exon 23 (<a href="#pone-0000427-g006">Figure 6</a>, compare lanes 10 and 11 with lane 9). The specific formation of ternary complexes is demonstrated by the findings that (i) in the absence of SR proteins only biotinylated RNAs were purified (<a href="#pone-0000427-g006">Figure 6</a>, lanes 5–7) and (ii) antisense exons 22 or 24, used as negative controls and whose stability was not influenced by the presence of the SR proteins (<a href="#pone-0000427-g006">Figure 6</a>, lanes 13 and 14), did not associate with biotinylated exon 23. Thus, the bridging activity of SR proteins, that in not impaired by the <em>mdx</em> mutation, could be involved in counterweighing exon 23 skipping predisposition caused by the combination of the mutation and the unusual trinucleotide GAG.</p>
<div class="figure" id="pone-0000427-g006"><div class="img"><a name="pone-0000427-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000427" data-uri="info:doi/10.1371/journal.pone.0000427.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000427.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000427.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000427.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000427.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>SR proteins mediate associations of exons 22, 23 and 24.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Biotinylated and non-biotinylated dystrophin exons labeled with <sup>32</sup>P-ATP (sense exons) or <sup>32</sup>P-UTP (antisense exons), were mixed with purified SR proteins. Following incubation, biotinylated RNAs were purified with streptavidin magnetic beads and the formation of RNA complexes was analyzed on 10% denaturing polyacrylamide gel. Asterisks and A indicate biotinylated and antisense exons respectively. Lanes 1 and 2, showing the relative migration of exons 22, 23 and 24, contain one-fourth of the RNA inputs assayed in lanes 8–11 and 13–14 respectively. Lane 15 and 16 contain one-fourth of the supernatants collected from reactions loaded in lanes 13–14. Background levels, probably generated by non-specific trapping of RNA or RNA/protein complexes into the magnetic beads, are shown in lanes 3, 4, and 12 and 17 respectively. Comparable results were obtained in three independent experiments.</p>
<span>doi:10.1371/journal.pone.0000427.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The physical characteristics of the dystrophin gene provide a unique model for expanding our knowledge of the mechanisms and factors involved in splice site selection and RNA processing. Dystrophin introns, some of which span more than 100 kb, appear to be spliced during the 16 hours required to transcribe the entire gene <a href="#pone.0000427-Tennyson1">[72]</a>. This coordinate activity probably provides an efficient way to increase the local concentration of splicing factors nearby the hnRNA, reduces the number of splice sites available and ensures that exons are spliced in the correct 5′ to 3′ order <a href="#pone.0000427-Maniatis2">[73]</a>. However, little is known about the factors and the cis-elements involved in dystrophin mRNA splicing.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In this study, we report that in the <em>mdx</em> mouse the presence of a nonsense mutation in exon 23 partially inactivates a multisite exonic splicing enhancer. Although this event could cause exon skipping, we found that splicing occurs normally and the mRNA carrying the nonsense mutation is degraded by activation of the NMD pathway. Here we provide evidence that exon 23 contains multiple binding sites for a 40 kDa SR protein. The M1 element is highly conserved between both human and mouse dystrophin genes (12 out of 12 positions), while R1 matches a splicing enhancer that was previously identified by <em>in vitro</em> functional iterative strategies <a href="#pone.0000427-Tian1">[16]</a>, <a href="#pone.0000427-Schaal1">[18]</a>. Several SR binding sites having enhancing activity have also been identified and studied in the constitutive β-globin exon 2 <a href="#pone.0000427-Schaal2">[29]</a>, and other observations have demonstrated a linear correlation between splicing efficiency and the number of enhancer elements present <a href="#pone.0000427-Graveley3">[65]</a>. Our data are in agreement with this model and also show the functional flexibility of adjacent ESEs. In fact, while both M1 and R1 were necessary to activate splicing of the enhancer dependent Dpy2 construct, in the context of the dystrophin gene, intron 22 removal is mainly dependent on the activity of M1. Nonetheless, in the presence of the <em>mdx</em> mutation, R1 could play an important role as a back-up element, which partially compensates for the lack of M1 activity. By UV cross-linking and band shift experiments, we have shown that both M1 and R1 can be specifically recognized by a 40 kDa SR. Although both Tra 2α/β and SRp40 migrate at ~40 kDa, Tra 2 is not present in purified preparations of SR proteins <a href="#pone.0000427-Tran2">[74]</a>, therefore SRp40 should be the only protein that binds exon 23 ESE.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In this paper we also show that exon 23 ESE activity is not functionally linked to the rare GAG present at the 3′ splice site of intron 22. Two lines of evidence support this conclusion: i) two exon mini-genes carrying the most efficient CAG were still equally sensitive to the <em>mdx</em> mutation, and ii) in the human dystrophin gene M1 is conserved, while the GAG is replaced by CAG. By quantitative RT-PCR employing different exon/intron primers <a href="#pone.0000427-Kessler1">[75]</a>, we found that splicing of a mini-gene carrying exons 22, 23, and 24 did not follow the first-come, first-served model. Nevertheless, intron 23 was spliced before intron 22 in only 50% of the molecules. At an RNA polymerase average elongation rate of 2.4 kb min<sup>−1</sup> <a href="#pone.0000427-Tennyson1">[72]</a>, intron 22 3′ splice site becomes available for splicing just a minute before intron 23 3′ splice site synthesis. Therefore, we believe that lack of linear progression in intron removal is not caused by a severe delay promoted by the GAG. Our mini-gene analysis also rules out the possibility that the 16 hours necessary to transcribe a gene as large as dystrophin could provide enough time for intron 22 removal, even with its slow kinetic of cleavage. Our results indicate that in vivo, removal of dystrophin intron 22 is not subjected to the same functional limitations observed previously in vitro <a href="#pone.0000427-Smith1">[49]</a>, and support the hypothesis that the use of natural GAG 3′ splice sites could be forced by the absence of downstream competing CAGs <a href="#pone.0000427-Smith2">[52]</a>, as in the case of dystrophin exon 23 (data not shown). The lack of cross-talk between ESEs and GAG could explain why just a small minority of natural 3′ splice sites contain a GAG while ESEs appear to be present in most exons.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Here we also show that the SR proteins can directly promote <em>in vitro</em> exon/exon association between the constitutive dystrophin exons 22, 23 and 24. Alternative splicing of the chicken cardiac troponin T (cTnT) gene appears to be modulated through exon/exon interactions mediated by SR proteins <a href="#pone.0000427-Stark2">[27]</a>, and ESEs are required for mammalian trans-splicing <a href="#pone.0000427-Caudevilla1">[76]</a>. Our data show that <em>in vitro</em>, the formation of a ternary complex between SR proteins and different dystrophin exons occurs efficiently and is not influenced by the presence of the <em>mdx</em> mutation. Interactions of exon 22, 23 and 24 mediated by SR proteins, could hold the dystrophin pre-mRNA in a topological configuration that imposes the use of intron 22 3′ splice site. Pre-splicing physical interactions between exon 23 and 24 could increase the local concentration of ESEs around intron 22 3′ splice site <a href="#pone.0000427-Graveley3">[65]</a> and compensate for the damaging effects of the <em>mdx</em> mutation. Thus, the splicing stimulation observed with the three-exon constructs (<a href="#pone-0000427-g004">Figure 4A</a>) could derive from the combined action of the ESEs contained in the two unspliced exons.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In the dystrophin gene, selection of committed splice sites often separated by long distances (&gt;100 kb) could be achieved by intron looping promoted by redundant ESEs situated in constitutive exons that are not functionally subordinated to splice sites strength or distance. Interestingly, this mechanism has been also proposed for several hnRNP proteins bound to intronic sequences <a href="#pone.0000427-Blanchette1">[77]</a>–<a href="#pone.0000427-MartinezContreras1">[79]</a>. The finding that purine rich regions with a wide spectrum of enhancing activity have been identified in several other dystrophin exons <a href="#pone.0000427-Ito1">[31]</a> indicates that ESEs could be common elements along the gene and could share the same bridging properties of the ones we have analyzed. Thus, optimization of therapeutic antisense oligonucleotides promoting exon skipping and preventing the incorporation of nonsense mutations in the mature dystrophin RNA should take into account both the complex crosstalk between ESEs, SR and pre-mRNA and potential exon/exon associations.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>DNA constructs</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The dystrophin genomic region spanning exons 22, 23 and 24 was amplified from wild type and <em>mdx</em> genomic DNAs with ExpandTM Long template PCR system (Roche) and cloned in the <em>EcoR</em>V site of both pcDNA3.1(+) and pcDNA3.1/Myc-His(−) A (<em>Invitrogen</em>). The translation start and termination codons were inserted in frame with the dystrophin exons at the beginning of exon 22 and at the end of exon 23 or 24 during the PCR reaction. Minigenes were fused to the 3′ UTR of bovine growth hormone carried by the cloning vectors. Single dystrophin exons were cloned into the <em>Pst</em>I site of pUC18-Dpy2 <a href="#pone.0000427-Dye1">[60]</a>, generating Dpy2/22, Dpy2/23wt, Dpy2/23<em>mdx</em> and Dpy2/24 plasmids. Mutant constructs were prepared by inverse PCR <a href="#pone.0000427-Imai1">[80]</a> with appropriate oligonucleotide pairs. Templates for <em>in vitro</em> RNA transcription were generated by PCR using primers containing the T7 transcription initiation sequence in the 5′ oligonucleotides. Human β-globin reporter plasmids βwt and β39 were as described <a href="#pone.0000427-LykkeAndersen1">[81]</a>. All oligonucleotide sequences and cloning strategies are available upon request.</p>


<h4>DNA transfection</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>COS 7 cells were grown in 100 mm dishes containing DMEM (Gibco-BRL) supplemented with 10% fetal bovine serum (Gibco-BRL). Transfections were carried out with 3 µg of each construct using Lipofectamine reagent (Gibco-BRL) according to the manufacturer's instructions. RNA was isolated 20 hr after transfection.</p>


<h4>RNA extraction, Northern Blot and RT-PCR</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Total RNA was isolated from cultured cells using TRI REAGENT (Molecular Research Center, Inc.) according to the manufacturer's instructions. Northern Blot analysis was performed as previously described on 10 µg of total RNA <a href="#pone.0000427-Buvoli1">[82]</a>. Blots shown in <a href="#pone-0000427-g004">Figure 4</a> were hybridized with oligonucleotide 24E complementary to dystrophin exon 24 (5′-TCTGCATTGTTTGAGCTGTTTTT-3′), and oligonucleotide 2glo complementary to β-globin exon 2 (5′-CACTAAAGGCACCGAGCACTTT-3′). Oligonucleotide 22E complementary to dystrophin exon 22 (5′-CTGTAATTTCCCGAGTCTCTCC-3′), and oligonucleotide 23E complementary to dystrophin exon 23 (<span class="gene">5′-CTGAAATTTTCGAAG</span> TTTATTC-3′) were also used to confirm the results obtained with the previous set of oligonucleotides. Oligonucleotide Neo3 (5′-AAGGTGAGATGACAGGAGATC-3′) was used to normalize the blots to the internal control neomycin. Band intensities were quantified by phosphorimager analysis (Storm 860 image analyzer, Molecular Dynamics). For RT-PCR analysis, total RNA was treated with DNase I Amp Grade (Gibco-BRL) and cDNA synthesized with random hexamers (Gibco BRL) and SUPERSCRIPT II reverse transcriptase (Gibco BRL) according to the manufacturer's instructions. For quantitative amplification (<a href="#pone-0000427-g004">Figure 4C, left panel</a>) PCR reactions were carried out in the presence of α <sup>32</sup>P dCTP as previously described <a href="#pone.0000427-Kessler2">[83]</a> with primers 22E, 22E5 (<span class="gene">5′-TTTTTGACACTTTACCACC</span> AAT-3′), 24E, Neo5 (<span class="gene">5′-TGAATGAACTGCA</span> GGACGAGG-3′) and Neo3. PCR reactions were analyzed on 8% acrylamide gels after 10, 15, 20 35 cycles and bands quantified by phosphorimager analysis. The non-quantitative PCR carried out to monitor exon 23 skipping (<a href="#pone-0000427-g004">Figure 4C, right panel</a>), was performed with oligonucleotides 22E and 24E (35 cycles). PCR reactions were assessed on a 1.2% agarose gel and stained with EtBr.</p>


<h4><em>In vitro</em> transcription and splicing reactions</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Capped <sup>32</sup>P–labeled RNAs were synthesized with T7 RNA polymerase, as previously described <a href="#pone.0000427-Buvoli2">[84]</a>. Unlabelled competitor RNAs were prepared using the MEGAscript ™ T7 Kit (Ambion) following the manufacturer's instructions. Standard splicing reactions contained 20–40 fmol of <sup>32</sup>P-labelled RNA transcript, 2.2 mM MgCl<sub>2</sub>, 0.5 mM ATP, 20 mM creatine phosphate, 1000 U/ml Rnasin (Promega) and 40% nuclear extract. Splicing products, fractionated on 12% polyacrylamide (acrylamide:bisacrylamide 20:1)/8M urea gels, were quantified by phoshorimager analysis.</p>


<h4>SR proteins</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>SR proteins were purified as previously described <a href="#pone.0000427-Zahler1">[63]</a>. Gel-shift, UV cross-linking, and exon bridging assays were carried out with a final SR protein concentration of ~30 µg/ml, corresponding to the one measured in HeLa splicing nuclear extracts <a href="#pone.0000427-Zahler2">[71]</a>.</p>


<h4>Gel-shift assay</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Binding reactions for the gel-shift assays, containing purified SR proteins <a href="#pone.0000427-Zahler1">[63]</a>, 100 fmol RNA, 5 mg/ml heparin, 20 mM HEPES pH 8, 5% glycerol, 70 mM KCL and 3 mM MgCl<sub>2</sub>, were incubated at 30°C for 10 minutes. Cold competitor RNAs were pre-incubated with SR proteins for 5 minutes prior to the addition of radiolabeled substrates. Complexes were fractionated on 4% polyacrylamide gels (acrylamide/bisacrylamide 80:1), 5% glycerol, 0.5×TBE. Signals were quantified by phoshorimager analysis.</p>


<h4>UV cross-linking</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>UV cross-linking experiments were carried out as for the gel shift-assay, except that heparin was omitted from the reactions and RNA samples were labeled with ATP. After 10 minutes incubation at 30°C, reactions were transferred in a 96-well microtiter dish (Falcon Micro Test III™), placed directly under a 254-nm UV light at 5 cm from the bulb, and irradiated for 10 minutes. RNAse A (1mg/ml) and T1 (10U) were then added and the reactions were incubated for 15 minutes at 37°C. Samples, adjusted to 40 mM Tris (pH 6.8), 7% glycerol, 1.5% SDS and 400 mM 2-mercaptoethanol, were then fractionated on 12% SDS-PAGE gel. Signals were quantified by phoshorimager analysis.</p>


<h4>Exon bridging assay</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Biotinylated RNAs were prepared as capped <sup>32</sup>P–labeled RNAs, except that ATP or UTP were used as radiolabeled nucleotides for sense or antisense exons respectively and the concentrations of both CTP and Biotin-14-CTP (Gibco-BRL) were 250 µM. Before use, 0.6 mg of Streptavidin MagneSphere® Paramagnetic Particles (Promega) were both washed (3 times with 1ml) and resuspended (100 µl) in buffer W (20 mM HEPES, 100 KCL, 3 mM MgCl<sub>2</sub>, 5 µg/ml tRNA, and 20 µg/ml BSA). Exon/exon interactions were carried out in 10 µl reactions containing 20 mM HEPES, 100 KCL, 3 mM MgCl<sub>2</sub>, 20 units of RNasin (Promega), equimolar amounts of biotinylated and nonbiotinylated RNAs (200 fmoles each) and purified SR proteins (~30 µg/ml). After incubation at 30°C for 10 minutes, streptavidin beads were added and reactions incubated at 4°C for 1 hour with nutation. Beads were then collected by magnetic stand (Promega), and while the aqueous fractions were phenolized and then ethanol precipitated, beads were washed 4 times with 1 ml of buffer W containing 0.05% Triton-X-100 and then resuspended in 100 µl of elution buffer (1% SDS, 1 mM EDTA, 50 µg/ml tRNA). Samples were incubated at 90°C for 5 minutes, beads collected and supernatants, containing the released RNAs, extracted with phenol/chloroform and ethanol precipitated. RNAs were analyzed on 10% polyacrylamide-8M urea gels. Signals were quantified by phoshorimager analysis.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank L. Cartegni and J. Patton for their critical reading of the manuscript and helpful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MB. Performed the experiments: MB AB. Analyzed the data: LL MB AB. Wrote the paper: MB. Other: Principal investigator: LL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000427-Blake1" id="pone.0000427-Blake1"></a>Blake DJ, Weir A, Newey SE, Davies KE (2002) Function and genetics of dystrophin and dystrophin-related proteins in muscle. Physiol Rev  82: 291–329.  <ul class="find" data-citedArticleID="949568"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Function+and+genetics+of+dystrophin+and+dystrophin-related+proteins+in+muscle.&amp;auth=&amp;atitle=Function+and+genetics+of+dystrophin+and+dystrophin-related+proteins+in+muscle." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Function+and+genetics+of+dystrophin+and+dystrophin-related+proteins+in+muscle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Function+and+genetics+of+dystrophin+and+dystrophin-related+proteins+in+muscle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000427-Davies1" id="pone.0000427-Davies1"></a>Davies KE, Nowak KJ (2006) Molecular mechanisms of muscular dystrophies: old and new players. Nat Rev Mol Cell Biol  7: 762–773.  <ul class="find" data-citedArticleID="949592" data-doi="10.1038/nrm2024"><li><a href="http://dx.doi.org/10.1038/nrm2024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanisms+of+muscular+dystrophies%3A+old+and+new+players." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanisms+of+muscular+dystrophies%3A+old+and+new+players.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000427-Coffey1" id="pone.0000427-Coffey1"></a>Coffey AJ, Roberts RG, Green ED, Cole CG, Butler R, et al.  (1992) Construction of a 2.6-Mb contig in yeast artificial chromosomes spanning the human dystrophin gene using an STS-based approach. Genomics  12: 474–484.  <ul class="find" data-citedArticleID="949588" data-doi="10.1016/0888-7543(92)90437-w"><li><a href="http://dx.doi.org/10.1016/0888-7543(92)90437-w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+a+2.6-Mb+contig+in+yeast+artificial+chromosomes+spanning+the+human+dystrophin+gene+using+an+STS-based+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+a+2.6-Mb+contig+in+yeast+artificial+chromosomes+spanning+the+human+dystrophin+gene+using+an+STS-based+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000427-Monaco1" id="pone.0000427-Monaco1"></a>Monaco AP, Walker AP, Millwood I, Larin Z, Lehrach H (1992) A yeast artificial chromosome contig containing the complete Duchenne muscular dystrophy gene. Genomics  12: 465–473.  <ul class="find" data-citedArticleID="949652" data-doi="10.1016/0888-7543(92)90436-v"><li><a href="http://dx.doi.org/10.1016/0888-7543(92)90436-v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+yeast+artificial+chromosome+contig+containing+the+complete+Duchenne+muscular+dystrophy+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+yeast+artificial+chromosome+contig+containing+the+complete+Duchenne+muscular+dystrophy+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000427-Koenig1" id="pone.0000427-Koenig1"></a>Koenig M, Monaco AP, Kunkel LM (1988) The complete sequence of dystrophin predicts a rod-shaped cytoskeletal protein. Cell  53: 219–226.  <ul class="find" data-citedArticleID="949624" data-doi="10.1016/0092-8674(88)90383-2"><li><a href="http://dx.doi.org/10.1016/0092-8674(88)90383-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+sequence+of+dystrophin+predicts+a+rod-shaped+cytoskeletal+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+sequence+of+dystrophin+predicts+a+rod-shaped+cytoskeletal+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000427-Jurica1" id="pone.0000427-Jurica1"></a>Jurica MS, Moore MJ (2003) Pre-mRNA splicing: awash in a sea of proteins. Mol Cell  12: 5–14.  <ul class="find" data-citedArticleID="949618" data-doi="10.1016/s1097-2765(03)00270-3"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00270-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pre-mRNA+splicing%3A+awash+in+a+sea+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pre-mRNA+splicing%3A+awash+in+a+sea+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000427-Konarska1" id="pone.0000427-Konarska1"></a>Konarska MM, Query CC (2005) Insights into the mechanisms of splicing: more lessons from the ribosome. Genes Dev  19: 2255–2260.  <ul class="find" data-citedArticleID="949626" data-doi="10.1101/gad.1363105"><li><a href="http://dx.doi.org/10.1101/gad.1363105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+the+mechanisms+of+splicing%3A+more+lessons+from+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+the+mechanisms+of+splicing%3A+more+lessons+from+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000427-Berget1" id="pone.0000427-Berget1"></a>Berget SM (1995) Exon recognition in vertebrate splicing. J Biol Chem  270: 2411–2414.  <ul class="find" data-citedArticleID="949562"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Exon+recognition+in+vertebrate+splicing.&amp;auth=&amp;atitle=Exon+recognition+in+vertebrate+splicing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exon+recognition+in+vertebrate+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exon+recognition+in+vertebrate+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000427-Sun1" id="pone.0000427-Sun1"></a>Sun H, Chasin LA (2000) Multiple splicing defects in an intronic false exon. Mol Cell Biol  20: 6414–6425.  <ul class="find" data-citedArticleID="949686" data-doi="10.1128/mcb.20.17.6414-6425.2000"><li><a href="http://dx.doi.org/10.1128/mcb.20.17.6414-6425.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+splicing+defects+in+an+intronic+false+exon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+splicing+defects+in+an+intronic+false+exon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000427-Blencowe1" id="pone.0000427-Blencowe1"></a>Blencowe BJ (2000) Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases. Trends Biochem Sci  25: 106–110.  <ul class="find" data-citedArticleID="949572" data-doi="10.1016/s0968-0004(00)01549-8"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)01549-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000427-Lam1" id="pone.0000427-Lam1"></a>Lam BJ, Hertel KJ (2002) A general role for splicing enhancers in exon definition. RNA  8: 1233–1241.  <ul class="find" data-citedArticleID="949628" data-doi="10.1017/s1355838202028030"><li><a href="http://dx.doi.org/10.1017/s1355838202028030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+role+for+splicing+enhancers+in+exon+definition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+role+for+splicing+enhancers+in+exon+definition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000427-Watakabe1" id="pone.0000427-Watakabe1"></a>Watakabe A, Tanaka K, Shimura Y (1993) The role of exon sequences in splice site selection. Genes Dev  7: 407–418.  <ul class="find" data-citedArticleID="949710" data-doi="10.1101/gad.7.3.407"><li><a href="http://dx.doi.org/10.1101/gad.7.3.407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+exon+sequences+in+splice+site+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+exon+sequences+in+splice+site+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000427-Lavigueur1" id="pone.0000427-Lavigueur1"></a>Lavigueur A, La Branche H, Kornblihtt AR, Chabot B (1993) A splicing enhancer in the human fibronectin alternate ED1 exon interacts with SR proteins and stimulates U2 snRNP binding. Genes Dev  7: 2405–2417.  <ul class="find" data-citedArticleID="949630" data-doi="10.1101/gad.7.12a.2405"><li><a href="http://dx.doi.org/10.1101/gad.7.12a.2405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+splicing+enhancer+in+the+human+fibronectin+alternate+ED1+exon+interacts+with+SR+proteins+and+stimulates+U2+snRNP+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+splicing+enhancer+in+the+human+fibronectin+alternate+ED1+exon+interacts+with+SR+proteins+and+stimulates+U2+snRNP+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000427-Xu1" id="pone.0000427-Xu1"></a>Xu R, Teng J, Cooper TA (1993) The cardiac troponin T alternative exon contains a novel purine-rich positive splicing element. Mol Cell Biol  13: 3660–3674.  <ul class="find" data-citedArticleID="949718"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+cardiac+troponin+T+alternative+exon+contains+a+novel+purine-rich+positive+splicing+element.&amp;auth=&amp;atitle=The+cardiac+troponin+T+alternative+exon+contains+a+novel+purine-rich+positive+splicing+element." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cardiac+troponin+T+alternative+exon+contains+a+novel+purine-rich+positive+splicing+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cardiac+troponin+T+alternative+exon+contains+a+novel+purine-rich+positive+splicing+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000427-Sun2" id="pone.0000427-Sun2"></a>Sun Q, Mayeda A, Hampson RK, Krainer AR, Rottman FM (1993) General splicing factor SF2/ASF promotes alternative splicing by binding to an exonic splicing enhancer. Genes Dev  7: 2598–2608.  <ul class="find" data-citedArticleID="949688" data-doi="10.1101/gad.7.12b.2598"><li><a href="http://dx.doi.org/10.1101/gad.7.12b.2598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=General+splicing+factor+SF2%2FASF+promotes+alternative+splicing+by+binding+to+an+exonic+splicing+enhancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22General+splicing+factor+SF2%2FASF+promotes+alternative+splicing+by+binding+to+an+exonic+splicing+enhancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000427-Tian1" id="pone.0000427-Tian1"></a>Tian H, Kole R (1995) Selection of novel exon recognition elements from a pool of random sequences. Mol Cell Biol  15: 6291–6298.  <ul class="find" data-citedArticleID="949694"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selection+of+novel+exon+recognition+elements+from+a+pool+of+random+sequences.&amp;auth=&amp;atitle=Selection+of+novel+exon+recognition+elements+from+a+pool+of+random+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+of+novel+exon+recognition+elements+from+a+pool+of+random+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+of+novel+exon+recognition+elements+from+a+pool+of+random+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000427-Coulter1" id="pone.0000427-Coulter1"></a>Coulter LR, Landree MA, Cooper TA (1997) Identification of a new class of exonic splicing enhancers by in vivo selection. Mol Cell Biol  17: 2143–2150.  <ul class="find" data-citedArticleID="949590"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+a+new+class+of+exonic+splicing+enhancers+by+in+vivo+selection.&amp;auth=&amp;atitle=Identification+of+a+new+class+of+exonic+splicing+enhancers+by+in+vivo+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+new+class+of+exonic+splicing+enhancers+by+in+vivo+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+new+class+of+exonic+splicing+enhancers+by+in+vivo+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000427-Schaal1" id="pone.0000427-Schaal1"></a>Schaal TD, Maniatis T (1999) Selection and characterization of pre-mRNA splicing enhancers: identification of novel SR protein-specific enhancer sequences. Mol Cell Biol  19: 1705–1719.  <ul class="find" data-citedArticleID="949662"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selection+and+characterization+of+pre-mRNA+splicing+enhancers%3A+identification+of+novel+SR+protein-specific+enhancer+sequences.&amp;auth=&amp;atitle=Selection+and+characterization+of+pre-mRNA+splicing+enhancers%3A+identification+of+novel+SR+protein-specific+enhancer+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+and+characterization+of+pre-mRNA+splicing+enhancers%3A+identification+of+novel+SR+protein-specific+enhancer+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+and+characterization+of+pre-mRNA+splicing+enhancers%3A+identification+of+novel+SR+protein-specific+enhancer+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000427-Tian2" id="pone.0000427-Tian2"></a>Tian H, Kole R (2001) Strong RNA splicing enhancers identified by a modified method of cycled selection interact with SR protein. J Biol Chem  276: 33833–33839.  <ul class="find" data-citedArticleID="949696" data-doi="10.1074/jbc.m102957200"><li><a href="http://dx.doi.org/10.1074/jbc.m102957200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+RNA+splicing+enhancers+identified+by+a+modified+method+of+cycled+selection+interact+with+SR+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+RNA+splicing+enhancers+identified+by+a+modified+method+of+cycled+selection+interact+with+SR+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000427-Graveley1" id="pone.0000427-Graveley1"></a>Graveley BR (2000) Sorting out the complexity of SR protein functions. RNA  6: 1197–1211.  <ul class="find" data-citedArticleID="949602" data-doi="10.1017/s1355838200000960"><li><a href="http://dx.doi.org/10.1017/s1355838200000960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sorting+out+the+complexity+of+SR+protein+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sorting+out+the+complexity+of+SR+protein+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000427-Graveley2" id="pone.0000427-Graveley2"></a>Graveley BR, Hertel KJ, Maniatis T (1999) SR proteins are ‘locators’ of the RNA splicing machinery. Curr Biol  9: R6–7.  <ul class="find" data-citedArticleID="949604" data-doi="10.1016/s0960-9822(99)80032-3"><li><a href="http://dx.doi.org/10.1016/s0960-9822(99)80032-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SR+proteins+are+%26%23x2018%3Blocators%26%23x2019%3B+of+the+RNA+splicing+machinery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SR+proteins+are+%26%23x2018%3Blocators%26%23x2019%3B+of+the+RNA+splicing+machinery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000427-Maniatis1" id="pone.0000427-Maniatis1"></a>Maniatis T, Tasic B (2002) Alternative pre-mRNA splicing and proteome expansion in metazoans. Nature  418: 236–243.  <ul class="find" data-citedArticleID="949642" data-doi="10.1038/418236a"><li><a href="http://dx.doi.org/10.1038/418236a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+pre-mRNA+splicing+and+proteome+expansion+in+metazoans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+pre-mRNA+splicing+and+proteome+expansion+in+metazoans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000427-Singh1" id="pone.0000427-Singh1"></a>Singh R, Valcarcel J (2005) Building specificity with nonspecific RNA-binding proteins. Nat Struct Mol Biol  12: 645–653.  <ul class="find" data-citedArticleID="949674" data-doi="10.1038/nsmb961"><li><a href="http://dx.doi.org/10.1038/nsmb961" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Building+specificity+with+nonspecific+RNA-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Building+specificity+with+nonspecific+RNA-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000427-Hertel1" id="pone.0000427-Hertel1"></a>Hertel KJ, Graveley BR (2005) RS domains contact the pre-mRNA throughout spliceosome assembly. Trends Biochem Sci  30: 115–118.  <ul class="find" data-citedArticleID="949610" data-doi="10.1016/j.tibs.2005.01.002"><li><a href="http://dx.doi.org/10.1016/j.tibs.2005.01.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RS+domains+contact+the+pre-mRNA+throughout+spliceosome+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RS+domains+contact+the+pre-mRNA+throughout+spliceosome+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000427-Shen1" id="pone.0000427-Shen1"></a>Shen H, Green MR (2006) RS domains contact splicing signals and promote splicing by a common mechanism in yeast through humans. Genes Dev  20: 1755–1765.  <ul class="find" data-citedArticleID="949668" data-doi="10.1101/gad.1422106"><li><a href="http://dx.doi.org/10.1101/gad.1422106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RS+domains+contact+splicing+signals+and+promote+splicing+by+a+common+mechanism+in+yeast+through+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RS+domains+contact+splicing+signals+and+promote+splicing+by+a+common+mechanism+in+yeast+through+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000427-Stark1" id="pone.0000427-Stark1"></a>Stark JM, Bazett-Jones DP, Herfort M, Roth MB (1998) SR proteins are sufficient for exon bridging across an intron. Proc Natl Acad Sci U S A  95: 2163–2168.  <ul class="find" data-citedArticleID="949682" data-doi="10.1073/pnas.95.5.2163"><li><a href="http://dx.doi.org/10.1073/pnas.95.5.2163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SR+proteins+are+sufficient+for+exon+bridging+across+an+intron." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SR+proteins+are+sufficient+for+exon+bridging+across+an+intron.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000427-Stark2" id="pone.0000427-Stark2"></a>Stark JM, Cooper TA, Roth MB (1999) The relative strengths of SR protein-mediated associations of alternative and constitutive exons can influence alternative splicing. J Biol Chem  274: 29838–29842.  <ul class="find" data-citedArticleID="949684" data-doi="10.1074/jbc.274.42.29838"><li><a href="http://dx.doi.org/10.1074/jbc.274.42.29838" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relative+strengths+of+SR+protein-mediated+associations+of+alternative+and+constitutive+exons+can+influence+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relative+strengths+of+SR+protein-mediated+associations+of+alternative+and+constitutive+exons+can+influence+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000427-Shiga1" id="pone.0000427-Shiga1"></a>Shiga N, Takeshima Y, Sakamoto H, Inoue K, Yokota Y, et al.  (1997) Disruption of the splicing enhancer sequence within exon 27 of the dystrophin gene by a nonsense mutation induces partial skipping of the exon and is responsible for Becker muscular dystrophy. J Clin Invest  100: 2204–2210.  <ul class="find" data-citedArticleID="949670" data-doi="10.1172/jci119757"><li><a href="http://dx.doi.org/10.1172/jci119757" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disruption+of+the+splicing+enhancer+sequence+within+exon+27+of+the+dystrophin+gene+by+a+nonsense+mutation+induces+partial+skipping+of+the+exon+and+is+responsible+for+Becker+muscular+dystrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disruption+of+the+splicing+enhancer+sequence+within+exon+27+of+the+dystrophin+gene+by+a+nonsense+mutation+induces+partial+skipping+of+the+exon+and+is+responsible+for+Becker+muscular+dystrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000427-Schaal2" id="pone.0000427-Schaal2"></a>Schaal TD, Maniatis T (1999) Multiple distinct splicing enhancers in the protein-coding sequences of a constitutively spliced pre-mRNA. Mol Cell Biol  19: 261–273.  <ul class="find" data-citedArticleID="949664"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiple+distinct+splicing+enhancers+in+the+protein-coding+sequences+of+a+constitutively+spliced+pre-mRNA.&amp;auth=&amp;atitle=Multiple+distinct+splicing+enhancers+in+the+protein-coding+sequences+of+a+constitutively+spliced+pre-mRNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+distinct+splicing+enhancers+in+the+protein-coding+sequences+of+a+constitutively+spliced+pre-mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+distinct+splicing+enhancers+in+the+protein-coding+sequences+of+a+constitutively+spliced+pre-mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000427-Mayeda1" id="pone.0000427-Mayeda1"></a>Mayeda A, Screaton GR, Chandler SD, Fu XD, Krainer AR (1999) Substrate specificities of SR proteins in constitutive splicing are determined by their RNA recognition motifs and composite pre-mRNA exonic elements. Mol Cell Biol  19: 1853–1863.  <ul class="find" data-citedArticleID="949650"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Substrate+specificities+of+SR+proteins+in+constitutive+splicing+are+determined+by+their+RNA+recognition+motifs+and+composite+pre-mRNA+exonic+elements.&amp;auth=&amp;atitle=Substrate+specificities+of+SR+proteins+in+constitutive+splicing+are+determined+by+their+RNA+recognition+motifs+and+composite+pre-mRNA+exonic+elements." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Substrate+specificities+of+SR+proteins+in+constitutive+splicing+are+determined+by+their+RNA+recognition+motifs+and+composite+pre-mRNA+exonic+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Substrate+specificities+of+SR+proteins+in+constitutive+splicing+are+determined+by+their+RNA+recognition+motifs+and+composite+pre-mRNA+exonic+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000427-Ito1" id="pone.0000427-Ito1"></a>Ito T, Takeshima Y, Sakamoto H, Nakamura H, Matsuo M (2001) Purine-rich exon sequences are not necessarily splicing enhancer sequence in the dystrophin gene. Kobe J Med Sci  47: 193–202.  <ul class="find" data-citedArticleID="949616"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Purine-rich+exon+sequences+are+not+necessarily+splicing+enhancer+sequence+in+the+dystrophin+gene.&amp;auth=&amp;atitle=Purine-rich+exon+sequences+are+not+necessarily+splicing+enhancer+sequence+in+the+dystrophin+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purine-rich+exon+sequences+are+not+necessarily+splicing+enhancer+sequence+in+the+dystrophin+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purine-rich+exon+sequences+are+not+necessarily+splicing+enhancer+sequence+in+the+dystrophin+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000427-Liu1" id="pone.0000427-Liu1"></a>Liu HX, Cartegni L, Zhang MQ, Krainer AR (2001) A mechanism for exon skipping caused by nonsense or missense mutations in BRCA1 and other genes. Nat Genet  27: 55–58.  <ul class="find" data-citedArticleID="949634"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+mechanism+for+exon+skipping+caused+by+nonsense+or+missense+mutations+in+BRCA1+and+other+genes.&amp;auth=&amp;atitle=A+mechanism+for+exon+skipping+caused+by+nonsense+or+missense+mutations+in+BRCA1+and+other+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mechanism+for+exon+skipping+caused+by+nonsense+or+missense+mutations+in+BRCA1+and+other+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mechanism+for+exon+skipping+caused+by+nonsense+or+missense+mutations+in+BRCA1+and+other+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000427-Cartegni1" id="pone.0000427-Cartegni1"></a>Cartegni L, Krainer AR (2002) Disruption of an SF2/ASF-dependent exonic splicing enhancer in SMN2 causes spinal muscular atrophy in the absence of SMN1. Nat Genet  30: 377–384.  <ul class="find" data-citedArticleID="949578" data-doi="10.1038/ng854"><li><a href="http://dx.doi.org/10.1038/ng854" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disruption+of+an+SF2%2FASF-dependent+exonic+splicing+enhancer+in+SMN2+causes+spinal+muscular+atrophy+in+the+absence+of+SMN1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disruption+of+an+SF2%2FASF-dependent+exonic+splicing+enhancer+in+SMN2+causes+spinal+muscular+atrophy+in+the+absence+of+SMN1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000427-Liu2" id="pone.0000427-Liu2"></a>Liu HX, Zhang M, Krainer AR (1998) Identification of functional exonic splicing enhancer motifs recognized by individual SR proteins. Genes Dev  12: 1998–2012.  <ul class="find" data-citedArticleID="949636" data-doi="10.1101/gad.12.13.1998"><li><a href="http://dx.doi.org/10.1101/gad.12.13.1998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+functional+exonic+splicing+enhancer+motifs+recognized+by+individual+SR+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+functional+exonic+splicing+enhancer+motifs+recognized+by+individual+SR+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000427-Fairbrother1" id="pone.0000427-Fairbrother1"></a>Fairbrother WG, Yeh RF, Sharp PA, Burge CB (2002) Predictive identification of exonic splicing enhancers in human genes. Science  297: 1007–1013.  <ul class="find" data-citedArticleID="949598" data-doi="10.1126/science.1073774"><li><a href="http://dx.doi.org/10.1126/science.1073774" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predictive+identification+of+exonic+splicing+enhancers+in+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predictive+identification+of+exonic+splicing+enhancers+in+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000427-Wang1" id="pone.0000427-Wang1"></a>Wang J, Smith PJ, Krainer AR, Zhang MQ (2005) Distribution of SR protein exonic splicing enhancer motifs in human protein-coding genes. Nucleic Acids Res  33: 5053–5062.  <ul class="find" data-citedArticleID="949704" data-doi="10.1093/nar/gki810"><li><a href="http://dx.doi.org/10.1093/nar/gki810" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+SR+protein+exonic+splicing+enhancer+motifs+in+human+protein-coding+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+SR+protein+exonic+splicing+enhancer+motifs+in+human+protein-coding+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000427-Wu1" id="pone.0000427-Wu1"></a>Wu Y, Zhang Y, Zhang J (2005) Distribution of exonic splicing enhancer elements in human genes. Genomics  86: 329–336.  <ul class="find" data-citedArticleID="949716" data-doi="10.1016/j.ygeno.2005.05.011"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2005.05.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+exonic+splicing+enhancer+elements+in+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+exonic+splicing+enhancer+elements+in+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000427-Hentze1" id="pone.0000427-Hentze1"></a>Hentze MW, Kulozik AE (1999) A perfect message: RNA surveillance and nonsense-mediated decay. Cell  96: 307–310.  <ul class="find" data-citedArticleID="949608" data-doi="10.1016/s0092-8674(00)80542-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80542-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+perfect+message%3A+RNA+surveillance+and+nonsense-mediated+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+perfect+message%3A+RNA+surveillance+and+nonsense-mediated+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000427-Gersappe1" id="pone.0000427-Gersappe1"></a>Gersappe A, Burger L, Pintel DJ (1999) A premature termination codon in either exon of minute virus of mice P4 promoter-generated pre-mRNA can inhibit nuclear splicing of the intervening intron in an open reading frame-dependent manner. J Biol Chem  274: 22452–22458.  <ul class="find" data-citedArticleID="949600" data-doi="10.1074/jbc.274.32.22452"><li><a href="http://dx.doi.org/10.1074/jbc.274.32.22452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+premature+termination+codon+in+either+exon+of+minute+virus+of+mice+P4+promoter-generated+pre-mRNA+can+inhibit+nuclear+splicing+of+the+intervening+intron+in+an+open+reading+frame-dependent+manner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+premature+termination+codon+in+either+exon+of+minute+virus+of+mice+P4+promoter-generated+pre-mRNA+can+inhibit+nuclear+splicing+of+the+intervening+intron+in+an+open+reading+frame-dependent+manner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000427-Cartegni2" id="pone.0000427-Cartegni2"></a>Cartegni L, Chew SL, Krainer AR (2002) Listening to silence and understanding nonsense: exonic mutations that affect splicing. Nat Rev Genet  3: 285–298.  <ul class="find" data-citedArticleID="949580" data-doi="10.1038/nrg775"><li><a href="http://dx.doi.org/10.1038/nrg775" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Listening+to+silence+and+understanding+nonsense%3A+exonic+mutations+that+affect+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Listening+to+silence+and+understanding+nonsense%3A+exonic+mutations+that+affect+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000427-Maquat1" id="pone.0000427-Maquat1"></a>Maquat LE (2002) NASty effects on fibrillin pre-mRNA splicing: another case of ESE does it, but proposals for translation-dependent splice site choice live on. Genes Dev  16: 1743–1753.  <ul class="find" data-citedArticleID="949646" data-doi="10.1101/gad.1014502"><li><a href="http://dx.doi.org/10.1101/gad.1014502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NASty+effects+on+fibrillin+pre-mRNA+splicing%3A+another+case+of+ESE+does+it%2C+but+proposals+for+translation-dependent+splice+site+choice+live+on." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NASty+effects+on+fibrillin+pre-mRNA+splicing%3A+another+case+of+ESE+does+it%2C+but+proposals+for+translation-dependent+splice+site+choice+live+on.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000427-Wang2" id="pone.0000427-Wang2"></a>Wang J, Hamilton JI, Carter MS, Li S, Wilkinson MF (2002) Alternatively spliced TCR mRNA induced by disruption of reading frame. Science  297: 108–110.  <ul class="find" data-citedArticleID="949706" data-doi="10.1126/science.1069757"><li><a href="http://dx.doi.org/10.1126/science.1069757" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternatively+spliced+TCR+mRNA+induced+by+disruption+of+reading+frame." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternatively+spliced+TCR+mRNA+induced+by+disruption+of+reading+frame.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000427-Wang3" id="pone.0000427-Wang3"></a>Wang J, Chang YF, Hamilton JI, Wilkinson MF (2002) Nonsense-associated altered splicing: a frame-dependent response distinct from nonsense-mediated decay. Mol Cell  10: 951–957.  <ul class="find" data-citedArticleID="949708" data-doi="10.1016/s1097-2765(02)00635-4"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00635-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonsense-associated+altered+splicing%3A+a+frame-dependent+response+distinct+from+nonsense-mediated+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonsense-associated+altered+splicing%3A+a+frame-dependent+response+distinct+from+nonsense-mediated+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000427-Lejeune1" id="pone.0000427-Lejeune1"></a>Lejeune F, Maquat LE (2005) Mechanistic links between nonsense-mediated mRNA decay and pre-mRNA splicing in mammalian cells. Curr Opin Cell Biol  17: 309–315.  <ul class="find" data-citedArticleID="949632" data-doi="10.1016/j.ceb.2005.03.002"><li><a href="http://dx.doi.org/10.1016/j.ceb.2005.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanistic+links+between+nonsense-mediated+mRNA+decay+and+pre-mRNA+splicing+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanistic+links+between+nonsense-mediated+mRNA+decay+and+pre-mRNA+splicing+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000427-Amrani1" id="pone.0000427-Amrani1"></a>Amrani N, Sachs MS, Jacobson A (2006) Early nonsense: mRNA decay solves a translational problem. Nat Rev Mol Cell Biol  7: 415–425.  <ul class="find" data-citedArticleID="949558" data-doi="10.1038/nrm1942"><li><a href="http://dx.doi.org/10.1038/nrm1942" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+nonsense%3A+mRNA+decay+solves+a+translational+problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+nonsense%3A+mRNA+decay+solves+a+translational+problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000427-BehmAnsmant1" id="pone.0000427-BehmAnsmant1"></a>Behm-Ansmant I, Izaurralde E (2006) Quality control of gene expression: a stepwise assembly pathway for the surveillance complex that triggers nonsense-mediated mRNA decay. Genes Dev  20: 391–398.  <ul class="find" data-citedArticleID="949560" data-doi="10.1101/gad.1407606"><li><a href="http://dx.doi.org/10.1101/gad.1407606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quality+control+of+gene+expression%3A+a+stepwise+assembly+pathway+for+the+surveillance+complex+that+triggers+nonsense-mediated+mRNA+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quality+control+of+gene+expression%3A+a+stepwise+assembly+pathway+for+the+surveillance+complex+that+triggers+nonsense-mediated+mRNA+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000427-Sicinski1" id="pone.0000427-Sicinski1"></a>Sicinski P, Geng Y, Ryder-Cook AS, Barnard EA, Darlison MG, et al.  (1989) The molecular basis of muscular dystrophy in the mdx mouse: a point mutation. Science  244: 1578–1580.  <ul class="find" data-citedArticleID="949672" data-doi="10.1126/science.2662404"><li><a href="http://dx.doi.org/10.1126/science.2662404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+basis+of+muscular+dystrophy+in+the+mdx+mouse%3A+a+point+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+basis+of+muscular+dystrophy+in+the+mdx+mouse%3A+a+point+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000427-Chamberlain1" id="pone.0000427-Chamberlain1"></a>Chamberlain JS, Pearlman JA, Muzny DM, Gibbs RA, Ranier JE, et al.  (1988) Expression of the murine Duchenne muscular dystrophy gene in muscle and brain. Science  239: 1416–1418.  <ul class="find" data-citedArticleID="949586" data-doi="10.1126/science.3347839"><li><a href="http://dx.doi.org/10.1126/science.3347839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+murine+Duchenne+muscular+dystrophy+gene+in+muscle+and+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+murine+Duchenne+muscular+dystrophy+gene+in+muscle+and+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000427-Smith1" id="pone.0000427-Smith1"></a>Smith CW, Porro EB, Patton JG, Nadal-Ginard B (1989) Scanning from an independently specified branch point defines the 3′ splice site of mammalian introns. Nature  342: 243–247.  <ul class="find" data-citedArticleID="949676" data-doi="10.1038/342243a0"><li><a href="http://dx.doi.org/10.1038/342243a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scanning+from+an+independently+specified+branch+point+defines+the+3%26%23x2032%3B+splice+site+of+mammalian+introns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scanning+from+an+independently+specified+branch+point+defines+the+3%26%23x2032%3B+splice+site+of+mammalian+introns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000427-Reed1" id="pone.0000427-Reed1"></a>Reed R (1989) The organization of 3′ splice-site sequences in mammalian introns. Genes Dev  3: 2113–2123.  <ul class="find" data-citedArticleID="949658" data-doi="10.1101/gad.3.12b.2113"><li><a href="http://dx.doi.org/10.1101/gad.3.12b.2113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+organization+of+3%26%23x2032%3B+splice-site+sequences+in+mammalian+introns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+organization+of+3%26%23x2032%3B+splice-site+sequences+in+mammalian+introns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000427-Wong1" id="pone.0000427-Wong1"></a>Wong C, Antonarakis SE, Goff SC, Orkin SH, Forget BG, et al.  (1989) Beta-thalassemia due to two novel nucleotide substitutions in consensus acceptor splice sequences of the beta-globin gene. Blood  73: 914–918.  <ul class="find" data-citedArticleID="949714"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Beta-thalassemia+due+to+two+novel+nucleotide+substitutions+in+consensus+acceptor+splice+sequences+of+the+beta-globin+gene.&amp;auth=&amp;atitle=Beta-thalassemia+due+to+two+novel+nucleotide+substitutions+in+consensus+acceptor+splice+sequences+of+the+beta-globin+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beta-thalassemia+due+to+two+novel+nucleotide+substitutions+in+consensus+acceptor+splice+sequences+of+the+beta-globin+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beta-thalassemia+due+to+two+novel+nucleotide+substitutions+in+consensus+acceptor+splice+sequences+of+the+beta-globin+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000427-Smith2" id="pone.0000427-Smith2"></a>Smith CW, Chu TT, Nadal-Ginard B (1993) Scanning and competition between AGs are involved in 3′ splice site selection in mammalian introns. Mol Cell Biol  13: 4939–4952.  <ul class="find" data-citedArticleID="949678"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Scanning+and+competition+between+AGs+are+involved+in+3%26%23x2032%3B+splice+site+selection+in+mammalian+introns.&amp;auth=&amp;atitle=Scanning+and+competition+between+AGs+are+involved+in+3%26%23x2032%3B+splice+site+selection+in+mammalian+introns." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scanning+and+competition+between+AGs+are+involved+in+3%26%23x2032%3B+splice+site+selection+in+mammalian+introns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scanning+and+competition+between+AGs+are+involved+in+3%26%23x2032%3B+splice+site+selection+in+mammalian+introns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000427-Bienvenu1" id="pone.0000427-Bienvenu1"></a>Bienvenu T, Hubert D, Fonknechten N, Dusser D, Kaplan JC, et al.  (1994) Unexpected inactivation of acceptor consensus splice sequence by a -3 C to T transition in intron 2 of the CFTR gene. Hum Genet  94: 65–68.  <ul class="find" data-citedArticleID="949564" data-doi="10.1007/bf02272843"><li><a href="http://dx.doi.org/10.1007/bf02272843" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unexpected+inactivation+of+acceptor+consensus+splice+sequence+by+a+-3+C+to+T+transition+in+intron+2+of+the+CFTR+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unexpected+inactivation+of+acceptor+consensus+splice+sequence+by+a+-3+C+to+T+transition+in+intron+2+of+the+CFTR+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000427-Bione1" id="pone.0000427-Bione1"></a>Bione S, Maestrini E, Rivella S, Mancini M, Regis S, et al.  (1994) Identification of a novel X-linked gene responsible for Emery-Dreifuss muscular dystrophy. Nat Genet  8: 323–327.  <ul class="find" data-citedArticleID="949566" data-doi="10.1038/ng1294-323"><li><a href="http://dx.doi.org/10.1038/ng1294-323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+novel+X-linked+gene+responsible+for+Emery-Dreifuss+muscular+dystrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+novel+X-linked+gene+responsible+for+Emery-Dreifuss+muscular+dystrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000427-Shapiro1" id="pone.0000427-Shapiro1"></a>Shapiro MB, Senapathy P (1987) RNA splice junctions of different classes of eukaryotes: sequence statistics and functional implications in gene expression. Nucleic Acids Res  15: 7155–7174.  <ul class="find" data-citedArticleID="949666" data-doi="10.1093/nar/15.17.7155"><li><a href="http://dx.doi.org/10.1093/nar/15.17.7155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+splice+junctions+of+different+classes+of+eukaryotes%3A+sequence+statistics+and+functional+implications+in+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+splice+junctions+of+different+classes+of+eukaryotes%3A+sequence+statistics+and+functional+implications+in+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000427-Wilton1" id="pone.0000427-Wilton1"></a>Wilton SD, Fletcher S (2006) Modification of pre-mRNA processing: application to dystrophin expression. Curr Opin Mol Ther  8: 130–135.  <ul class="find" data-citedArticleID="949712"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modification+of+pre-mRNA+processing%3A+application+to+dystrophin+expression.&amp;auth=&amp;atitle=Modification+of+pre-mRNA+processing%3A+application+to+dystrophin+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modification+of+pre-mRNA+processing%3A+application+to+dystrophin+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modification+of+pre-mRNA+processing%3A+application+to+dystrophin+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000427-Takeshima1" id="pone.0000427-Takeshima1"></a>Takeshima Y, Nishio H, Sakamoto H, Nakamura H, Matsuo M (1995) Modulation of in vitro splicing of the upstream intron by modifying an intra-exon sequence which is deleted from the dystrophin gene in dystrophin Kobe. J Clin Invest  95: 515–520.  <ul class="find" data-citedArticleID="949690" data-doi="10.1172/jci117693"><li><a href="http://dx.doi.org/10.1172/jci117693" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+in+vitro+splicing+of+the+upstream+intron+by+modifying+an+intra-exon+sequence+which+is+deleted+from+the+dystrophin+gene+in+dystrophin+Kobe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+in+vitro+splicing+of+the+upstream+intron+by+modifying+an+intra-exon+sequence+which+is+deleted+from+the+dystrophin+gene+in+dystrophin+Kobe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000427-Tran1" id="pone.0000427-Tran1"></a>Tran VK, Takeshima Y, Zhang Z, Yagi M, Nishiyama A, et al.  (2006) Splicing analysis disclosed a determinant single nucleotide for exon skipping caused by a novel intraexonic four-nucleotide deletion in the dystrophin gene. J Med Genet  43: 924–930.  <ul class="find" data-citedArticleID="949698" data-doi="10.1136/jmg.2006.042317"><li><a href="http://dx.doi.org/10.1136/jmg.2006.042317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Splicing+analysis+disclosed+a+determinant+single+nucleotide+for+exon+skipping+caused+by+a+novel+intraexonic+four-nucleotide+deletion+in+the+dystrophin+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Splicing+analysis+disclosed+a+determinant+single+nucleotide+for+exon+skipping+caused+by+a+novel+intraexonic+four-nucleotide+deletion+in+the+dystrophin+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000427-Disset1" id="pone.0000427-Disset1"></a>Disset A, Bourgeois CF, Benmalek N, Claustres M, Stevenin J, et al.  (2006) An exon skipping-associated nonsense mutation in the dystrophin gene uncovers a complex interplay between multiple antagonistic splicing elements. Hum Mol Genet  15: 999–1013.  <ul class="find" data-citedArticleID="949594" data-doi="10.1093/hmg/ddl015"><li><a href="http://dx.doi.org/10.1093/hmg/ddl015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+exon+skipping-associated+nonsense+mutation+in+the+dystrophin+gene+uncovers+a+complex+interplay+between+multiple+antagonistic+splicing+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+exon+skipping-associated+nonsense+mutation+in+the+dystrophin+gene+uncovers+a+complex+interplay+between+multiple+antagonistic+splicing+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000427-Dye1" id="pone.0000427-Dye1"></a>Dye BT, Buvoli M, Mayer SA, Lin CH, Patton JG (1998) Enhancer elements activate the weak 3′ splice site of alpha-tropomyosin exon 2. RNA  4: 1523–1536.  <ul class="find" data-citedArticleID="949596" data-doi="10.1017/s1355838298980360"><li><a href="http://dx.doi.org/10.1017/s1355838298980360" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancer+elements+activate+the+weak+3%26%23x2032%3B+splice+site+of+alpha-tropomyosin+exon+2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancer+elements+activate+the+weak+3%26%23x2032%3B+splice+site+of+alpha-tropomyosin+exon+2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000427-Cartegni3" id="pone.0000427-Cartegni3"></a>Cartegni L, Wang J, Zhu Z, Zhang MQ, Krainer AR (2003) ESEfinder: A web resource to identify exonic splicing enhancers. Nucleic Acids Res  31: 3568–3571.  <ul class="find" data-citedArticleID="949582" data-doi="10.1093/nar/gkg616"><li><a href="http://dx.doi.org/10.1093/nar/gkg616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ESEfinder%3A+A+web+resource+to+identify+exonic+splicing+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ESEfinder%3A+A+web+resource+to+identify+exonic+splicing+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000427-Smith3" id="pone.0000427-Smith3"></a>Smith PJ, Zhang C, Wang J, Chew SL, Zhang MQ, et al.  (2006) An increased specificity score matrix for the prediction of SF2/ASF-specific exonic splicing enhancers. Hum Mol Genet  15: 2490–2508.  <ul class="find" data-citedArticleID="949680" data-doi="10.1093/hmg/ddl171"><li><a href="http://dx.doi.org/10.1093/hmg/ddl171" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+increased+specificity+score+matrix+for+the+prediction+of+SF2%2FASF-specific+exonic+splicing+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+increased+specificity+score+matrix+for+the+prediction+of+SF2%2FASF-specific+exonic+splicing+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000427-Zahler1" id="pone.0000427-Zahler1"></a>Zahler AM, Lane WS, Stolk JA, Roth MB (1992) SR proteins: a conserved family of pre-mRNA splicing factors. Genes Dev  6: 837–847.  <ul class="find" data-citedArticleID="949722" data-doi="10.1101/gad.6.5.837"><li><a href="http://dx.doi.org/10.1101/gad.6.5.837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SR+proteins%3A+a+conserved+family+of+pre-mRNA+splicing+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SR+proteins%3A+a+conserved+family+of+pre-mRNA+splicing+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000427-Hertel2" id="pone.0000427-Hertel2"></a>Hertel KJ, Maniatis T (1998) The function of multisite splicing enhancers. Mol Cell  1: 449–455.  <ul class="find" data-citedArticleID="949612" data-doi="10.1016/s1097-2765(00)80045-3"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80045-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+function+of+multisite+splicing+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+function+of+multisite+splicing+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000427-Graveley3" id="pone.0000427-Graveley3"></a>Graveley BR, Hertel KJ, Maniatis T (1998) A systematic analysis of the factors that determine the strength of pre-mRNA splicing enhancers. Embo J  17: 6747–6756.  <ul class="find" data-citedArticleID="949606" data-doi="10.1093/emboj/17.22.6747"><li><a href="http://dx.doi.org/10.1093/emboj/17.22.6747" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systematic+analysis+of+the+factors+that+determine+the+strength+of+pre-mRNA+splicing+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systematic+analysis+of+the+factors+that+determine+the+strength+of+pre-mRNA+splicing+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000427-Robberson1" id="pone.0000427-Robberson1"></a>Robberson BL, Cote GJ, Berget SM (1990) Exon definition may facilitate splice site selection in RNAs with multiple exons. Mol Cell Biol  10: 84–94.  <ul class="find" data-citedArticleID="949660"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Exon+definition+may+facilitate+splice+site+selection+in+RNAs+with+multiple+exons.&amp;auth=&amp;atitle=Exon+definition+may+facilitate+splice+site+selection+in+RNAs+with+multiple+exons." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exon+definition+may+facilitate+splice+site+selection+in+RNAs+with+multiple+exons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exon+definition+may+facilitate+splice+site+selection+in+RNAs+with+multiple+exons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000427-Pollard1" id="pone.0000427-Pollard1"></a>Pollard AJ, Krainer AR, Robson SC, Europe-Finner GN (2002) Alternative splicing of the adenylyl cyclase stimulatory G-protein G alpha(s) is regulated by SF2/ASF and heterogeneous nuclear ribonucleoprotein A1 (hnRNPA1) and involves the use of an unusual TG 3′-splice Site. J Biol Chem  277: 15241–15251.  <ul class="find" data-citedArticleID="949656" data-doi="10.1074/jbc.m109046200"><li><a href="http://dx.doi.org/10.1074/jbc.m109046200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+splicing+of+the+adenylyl+cyclase+stimulatory+G-protein+G+alpha%28s%29+is+regulated+by+SF2%2FASF+and+heterogeneous+nuclear+ribonucleoprotein+A1+%28hnRNPA1%29+and+involves+the+use+of+an+unusual+TG+3%26%23x2032%3B-splice+Site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+splicing+of+the+adenylyl+cyclase+stimulatory+G-protein+G+alpha%28s%29+is+regulated+by+SF2%2FASF+and+heterogeneous+nuclear+ribonucleoprotein+A1+%28hnRNPA1%29+and+involves+the+use+of+an+unusual+TG+3%26%23x2032%3B-splice+Site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000427-Yu1" id="pone.0000427-Yu1"></a>Yu J, Russell JE (2001) Structural and functional analysis of an mRNP complex that mediates the high stability of human beta-globin mRNA. Mol Cell Biol  21: 5879–5888.  <ul class="find" data-citedArticleID="949720" data-doi="10.1128/mcb.21.17.5879-5888.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.17.5879-5888.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+functional+analysis+of+an+mRNP+complex+that+mediates+the+high+stability+of+human+beta-globin+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+functional+analysis+of+an+mRNP+complex+that+mediates+the+high+stability+of+human+beta-globin+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000427-Malone1" id="pone.0000427-Malone1"></a>Malone RW, Felgner PL, Verma IM (1989) Cationic liposome-mediated RNA transfection. Proc Natl Acad Sci U S A  86: 6077–6081.  <ul class="find" data-citedArticleID="949640" data-doi="10.1073/pnas.86.16.6077"><li><a href="http://dx.doi.org/10.1073/pnas.86.16.6077" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cationic+liposome-mediated+RNA+transfection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cationic+liposome-mediated+RNA+transfection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000427-Tsukahara1" id="pone.0000427-Tsukahara1"></a>Tsukahara T, Casciato C, Helfman DM (1994) Alternative splicing of beta-tropomyosin pre-mRNA: multiple cis-elements can contribute to the use of the 5′- and 3′-splice sites of the nonmuscle/smooth muscle exon 6. Nucleic Acids Res  22: 2318–2325.  <ul class="find" data-citedArticleID="949702" data-doi="10.1093/nar/22.12.2318"><li><a href="http://dx.doi.org/10.1093/nar/22.12.2318" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+splicing+of+beta-tropomyosin+pre-mRNA%3A+multiple+cis-elements+can+contribute+to+the+use+of+the+5%26%23x2032%3B-+and+3%26%23x2032%3B-splice+sites+of+the+nonmuscle%2Fsmooth+muscle+exon+6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+splicing+of+beta-tropomyosin+pre-mRNA%3A+multiple+cis-elements+can+contribute+to+the+use+of+the+5%26%23x2032%3B-+and+3%26%23x2032%3B-splice+sites+of+the+nonmuscle%2Fsmooth+muscle+exon+6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000427-Zahler2" id="pone.0000427-Zahler2"></a>Zahler AM, Neugebauer KM, Stolk JA, Roth MB (1993) Human SR proteins and isolation of a cDNA encoding SRp75. Mol Cell Biol  13: 4023–4028.  <ul class="find" data-citedArticleID="949724"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+SR+proteins+and+isolation+of+a+cDNA+encoding+SRp75.&amp;auth=&amp;atitle=Human+SR+proteins+and+isolation+of+a+cDNA+encoding+SRp75." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+SR+proteins+and+isolation+of+a+cDNA+encoding+SRp75." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+SR+proteins+and+isolation+of+a+cDNA+encoding+SRp75.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000427-Tennyson1" id="pone.0000427-Tennyson1"></a>Tennyson CN, Klamut HJ, Worton RG (1995) The human dystrophin gene requires 16 hours to be transcribed and is cotranscriptionally spliced. Nat Genet  9: 184–190.  <ul class="find" data-citedArticleID="949692" data-doi="10.1038/ng0295-184"><li><a href="http://dx.doi.org/10.1038/ng0295-184" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+dystrophin+gene+requires+16+hours+to+be+transcribed+and+is+cotranscriptionally+spliced." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+dystrophin+gene+requires+16+hours+to+be+transcribed+and+is+cotranscriptionally+spliced.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000427-Maniatis2" id="pone.0000427-Maniatis2"></a>Maniatis T, Reed R (2002) An extensive network of coupling among gene expression machines. Nature  416: 499–506.  <ul class="find" data-citedArticleID="949644" data-doi="10.1038/416499a"><li><a href="http://dx.doi.org/10.1038/416499a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+extensive+network+of+coupling+among+gene+expression+machines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+extensive+network+of+coupling+among+gene+expression+machines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000427-Tran2" id="pone.0000427-Tran2"></a>Tran Q, Coleman TP, Roesser JR (2003) Human transformer 2beta and SRp55 interact with a calcitonin-specific splice enhancer. Biochim Biophys Acta  1625: 141–152.  <ul class="find" data-citedArticleID="949700" data-doi="10.1016/s0167-4781(02)00600-0"><li><a href="http://dx.doi.org/10.1016/s0167-4781(02)00600-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+transformer+2beta+and+SRp55+interact+with+a+calcitonin-specific+splice+enhancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+transformer+2beta+and+SRp55+interact+with+a+calcitonin-specific+splice+enhancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000427-Kessler1" id="pone.0000427-Kessler1"></a>Kessler O, Jiang Y, Chasin LA (1993) Order of intron removal during splicing of endogenous adenine phosphoribosyltransferase and dihydrofolate reductase pre-mRNA. Mol Cell Biol  13: 6211–6222.  <ul class="find" data-citedArticleID="949620"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Order+of+intron+removal+during+splicing+of+endogenous+adenine+phosphoribosyltransferase+and+dihydrofolate+reductase+pre-mRNA.&amp;auth=&amp;atitle=Order+of+intron+removal+during+splicing+of+endogenous+adenine+phosphoribosyltransferase+and+dihydrofolate+reductase+pre-mRNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Order+of+intron+removal+during+splicing+of+endogenous+adenine+phosphoribosyltransferase+and+dihydrofolate+reductase+pre-mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Order+of+intron+removal+during+splicing+of+endogenous+adenine+phosphoribosyltransferase+and+dihydrofolate+reductase+pre-mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000427-Caudevilla1" id="pone.0000427-Caudevilla1"></a>Caudevilla C, Codony C, Serra D, Plasencia G, Roman R, et al.  (2001) Localization of an exonic splicing enhancer responsible for mammalian natural trans-splicing. Nucleic Acids Res  29: 3108–3115.  <ul class="find" data-citedArticleID="949584" data-doi="10.1093/nar/29.14.3108"><li><a href="http://dx.doi.org/10.1093/nar/29.14.3108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localization+of+an+exonic+splicing+enhancer+responsible+for+mammalian+natural+trans-splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localization+of+an+exonic+splicing+enhancer+responsible+for+mammalian+natural+trans-splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000427-Blanchette1" id="pone.0000427-Blanchette1"></a>Blanchette M, Chabot B (1999) Modulation of exon skipping by high-affinity hnRNP A1-binding sites and by intron elements that repress splice site utilization. Embo J  18: 1939–1952.  <ul class="find" data-citedArticleID="949570" data-doi="10.1093/emboj/18.7.1939"><li><a href="http://dx.doi.org/10.1093/emboj/18.7.1939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+exon+skipping+by+high-affinity+hnRNP+A1-binding+sites+and+by+intron+elements+that+repress+splice+site+utilization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+exon+skipping+by+high-affinity+hnRNP+A1-binding+sites+and+by+intron+elements+that+repress+splice+site+utilization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000427-Nasim1" id="pone.0000427-Nasim1"></a>Nasim FU, Hutchison S, Cordeau M, Chabot B (2002) High-affinity hnRNP A1 binding sites and duplex-forming inverted repeats have similar effects on 5′ splice site selection in support of a common looping out and repression mechanism. RNA  8: 1078–1089.  <ul class="find" data-citedArticleID="949654" data-doi="10.1017/s1355838202024056"><li><a href="http://dx.doi.org/10.1017/s1355838202024056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-affinity+hnRNP+A1+binding+sites+and+duplex-forming+inverted+repeats+have+similar+effects+on+5%26%23x2032%3B+splice+site+selection+in+support+of+a+common+looping+out+and+repression+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-affinity+hnRNP+A1+binding+sites+and+duplex-forming+inverted+repeats+have+similar+effects+on+5%26%23x2032%3B+splice+site+selection+in+support+of+a+common+looping+out+and+repression+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000427-MartinezContreras1" id="pone.0000427-MartinezContreras1"></a>Martinez-Contreras R, Fisette JF, Nasim FU, Madden R, Cordeau M, et al.  (2006) Intronic binding sites for hnRNP A/B and hnRNP F/H proteins stimulate pre-mRNA splicing. PLoS Biol  4: e21.  <ul class="find" data-citedArticleID="949648" data-doi="10.1371/journal.pbio.0040021"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intronic+binding+sites+for+hnRNP+A%2FB+and+hnRNP+F%2FH+proteins+stimulate+pre-mRNA+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intronic+binding+sites+for+hnRNP+A%2FB+and+hnRNP+F%2FH+proteins+stimulate+pre-mRNA+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000427-Imai1" id="pone.0000427-Imai1"></a>Imai Y, Matsushima Y, Sugimura T, Terada M (1991) A simple and rapid method for generating a deletion by PCR. Nucleic Acids Res  19: 2785.  <ul class="find" data-citedArticleID="949614" data-doi="10.1093/nar/19.10.2785"><li><a href="http://dx.doi.org/10.1093/nar/19.10.2785" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+and+rapid+method+for+generating+a+deletion+by+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+and+rapid+method+for+generating+a+deletion+by+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000427-LykkeAndersen1" id="pone.0000427-LykkeAndersen1"></a>Lykke-Andersen J, Shu MD, Steitz JA (2000) Human Upf proteins target an mRNA for nonsense-mediated decay when bound downstream of a termination codon. Cell  103: 1121–1131.  <ul class="find" data-citedArticleID="949638" data-doi="10.1016/s0092-8674(00)00214-2"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00214-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Upf+proteins+target+an+mRNA+for+nonsense-mediated+decay+when+bound+downstream+of+a+termination+codon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Upf+proteins+target+an+mRNA+for+nonsense-mediated+decay+when+bound+downstream+of+a+termination+codon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000427-Buvoli1" id="pone.0000427-Buvoli1"></a>Buvoli M, Buvoli A, Leinwand LA (2000) Suppression of nonsense mutations in cell culture and mice by multimerized suppressor tRNA genes. Mol Cell Biol  20: 3116–3124.  <ul class="find" data-citedArticleID="949574" data-doi="10.1128/mcb.20.9.3116-3124.2000"><li><a href="http://dx.doi.org/10.1128/mcb.20.9.3116-3124.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+nonsense+mutations+in+cell+culture+and+mice+by+multimerized+suppressor+tRNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+nonsense+mutations+in+cell+culture+and+mice+by+multimerized+suppressor+tRNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000427-Kessler2" id="pone.0000427-Kessler2"></a>Kessler O, Chasin LA (1996) Effects of nonsense mutations on nuclear and cytoplasmic adenine phosphoribosyltransferase RNA. Mol Cell Biol  16: 4426–4435.  <ul class="find" data-citedArticleID="949622"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effects+of+nonsense+mutations+on+nuclear+and+cytoplasmic+adenine+phosphoribosyltransferase+RNA.&amp;auth=&amp;atitle=Effects+of+nonsense+mutations+on+nuclear+and+cytoplasmic+adenine+phosphoribosyltransferase+RNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+nonsense+mutations+on+nuclear+and+cytoplasmic+adenine+phosphoribosyltransferase+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+nonsense+mutations+on+nuclear+and+cytoplasmic+adenine+phosphoribosyltransferase+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000427-Buvoli2" id="pone.0000427-Buvoli2"></a>Buvoli M, Mayer SA, Patton JG (1997) Functional crosstalk between exon enhancers, polypyrimidine tracts and branchpoint sequences. Embo J  16: 7174–7183.  <ul class="find" data-citedArticleID="949576" data-doi="10.1093/emboj/16.23.7174"><li><a href="http://dx.doi.org/10.1093/emboj/16.23.7174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+crosstalk+between+exon+enhancers%2C+polypyrimidine+tracts+and+branchpoint+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+crosstalk+between+exon+enhancers%2C+polypyrimidine+tracts+and+branchpoint+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.XML" value="124600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.PDF" value="858730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g001.PNG_L" value="631981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g001.PNG_M" value="56466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g001.PNG_S" value="4096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g001.TIF" value="677266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g001.PNG_I" value="33563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g002.PNG_L" value="203844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g002.PNG_M" value="76186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g002.PNG_S" value="3162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g002.TIF" value="184552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g002.PNG_I" value="25981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g003.PNG_L" value="420454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g003.PNG_M" value="76662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g003.PNG_S" value="3039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g003.TIF" value="441140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g003.PNG_I" value="25559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g004.PNG_L" value="472057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g004.PNG_M" value="54441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g004.PNG_S" value="3970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g004.TIF" value="531498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g004.PNG_I" value="28527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g005.PNG_L" value="736582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g005.PNG_M" value="81517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g005.PNG_S" value="3775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g005.TIF" value="769582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g005.PNG_I" value="28559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g006.PNG_L" value="236569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g006.PNG_M" value="95352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g006.PNG_S" value="3798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g006.TIF" value="236362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000427.g006.PNG_I" value="33888"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000427&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000427" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000427&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000427&volume=&issue=&title=Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic Mdx Mouse&author_name=Massimo%20Buvoli%2C%20Ada%20Buvoli%2C%20Leslie%20A.%20Leinwand&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000427" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000427" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000427" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000427&amp;t=Interplay%20between%20Exonic%20Splicing%20Enhancers%2C%20mRNA%20Processing%2C%20and%20mRNA%20Surveillance%20in%20the%20Dystrophic%20Mdx%20Mouse" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000427&title=Interplay%20between%20Exonic%20Splicing%20Enhancers%2C%20mRNA%20Processing%2C%20and%20mRNA%20Surveillance%20in%20the%20Dystrophic%20Mdx%20Mouse&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000427&amp;title=Interplay%20between%20Exonic%20Splicing%20Enhancers%2C%20mRNA%20Processing%2C%20and%20mRNA%20Surveillance%20in%20the%20Dystrophic%20Mdx%20Mouse" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000427" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000427&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Interplay between Exonic Splicing Enhancers, mRNA Processing, and mRNA Surveillance in the Dystrophic Mdx Mouse';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000427';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Interplay%20between%20Exonic%20Splicing%20Enhancers%2C%20mRNA%20Processing%2C%20and%20mRNA%20Surveillance%20in%20the%20Dystrophic%20Mdx%20Mouse http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000427" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000427" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Antisense+RNA%22" title="Search for articles in the subject area:'Antisense RNA'"><div class="flagText">Antisense RNA</div></a>
              <div data-categoryid="19975" data-articleid="23692"
                   data-categoryname="Antisense RNA"
                   class="flagImage" title="Flag 'Antisense RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Dystrophin%22" title="Search for articles in the subject area:'Dystrophin'"><div class="flagText">Dystrophin</div></a>
              <div data-categoryid="23541" data-articleid="23692"
                   data-categoryname="Dystrophin"
                   class="flagImage" title="Flag 'Dystrophin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="23692"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23692"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nonsense+mutation%22" title="Search for articles in the subject area:'Nonsense mutation'"><div class="flagText">Nonsense mutation</div></a>
              <div data-categoryid="36165" data-articleid="23692"
                   data-categoryname="Nonsense mutation"
                   class="flagImage" title="Flag 'Nonsense mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="23692"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="23692"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+folding%22" title="Search for articles in the subject area:'RNA folding'"><div class="flagText">RNA folding</div></a>
              <div data-categoryid="20939" data-articleid="23692"
                   data-categoryname="RNA folding"
                   class="flagImage" title="Flag 'RNA folding' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=846'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7193'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9223&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
