

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Monoallelic Expression of Multiple Genes in the CNS</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001293"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001293" />

    <meta name="citation_title" content="Monoallelic Expression of Multiple Genes in the CNS"/>
    <meta itemprop="name" content="Monoallelic Expression of Multiple Genes in the CNS"/>

      <meta name="citation_author" content="Jinhui Wang"/>
            <meta name="citation_author_institution" content="Division of Biology, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America"/>
            <meta name="citation_author_institution" content="Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America"/>
      <meta name="citation_author" content="Zuzana Valo"/>
            <meta name="citation_author_institution" content="Division of Biology, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America"/>
            <meta name="citation_author_institution" content="Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America"/>
      <meta name="citation_author" content="David Smith"/>
            <meta name="citation_author_institution" content="Division of Information Sciences, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America"/>
      <meta name="citation_author" content="Judith Singer-Sam"/>
            <meta name="citation_author_institution" content="Division of Biology, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America"/>
            <meta name="citation_author_institution" content="Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America"/>

    <meta name="citation_date" content="2007/12/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001293.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1293"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_author=EF Torrey; citation_author=AE Bowler; citation_author=EH Taylor; citation_author=II Gottesman; citation_number=1; citation_pages=274; citation_date=1994; citation_publisher=HarperCollins; " />
      <meta name="citation_reference" content="citation_title=Regulation of TCR α and β gene allelic exclusion during T-cell development.; citation_author=M Malissen; citation_author=J Trucy; citation_author=E Jouvin-Marche; citation_author=PA Cazenave; citation_author=R Scollay; citation_journal_title=Immunol Today; citation_volume=13; citation_number=2; citation_pages=315-322; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Allelic exclusion and lymphocyte development. Lessons from transgenic mice.; citation_author=IJ Rimm; citation_author=DB Bloch; citation_author=JG Seidman; citation_journal_title=Mol Biol Med; citation_volume=6; citation_number=3; citation_pages=355-364; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Expression of the Ly49A gene in murine natural killer cell clones is predominantly but not exclusively mono-allelic.; citation_author=W Held; citation_author=DH Raulet; citation_journal_title=Eur J Immunol; citation_volume=27; citation_number=4; citation_pages=2876-2884; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Monoallelic expression of the interleukin-2 locus.; citation_author=GA Holländer; citation_author=S Zuklys; citation_author=C Morel; citation_author=E Mizoguchi; citation_author=K Mobisson; citation_journal_title=Science; citation_volume=279; citation_number=5; citation_pages=2118-2121; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Independent and epigenetic regulation of the interleukin-4 alleles in CD4+ T cells.; citation_author=M Bix; citation_author=RM Locksley; citation_journal_title=Science; citation_volume=281; citation_number=6; citation_pages=1352-1354; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Regulation of IL-4 expression by activation of individual alleles.; citation_author=I Rivière; citation_author=MJ Sunshine; citation_author=DR Littman; citation_journal_title=Immunity; citation_volume=9; citation_number=7; citation_pages=217-228; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Coordinate regulation of the IL-4, IL-13, and IL-5 cytokine cluster in Th2 clones revealed by allelic expression patterns.; citation_author=BL Kelly; citation_author=RM Locksley; citation_journal_title=J Immunol; citation_volume=165; citation_number=8; citation_pages=2982-2986; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Allelic inactivation regulates olfactory receptor gene expression.; citation_author=A Chess; citation_author=I Simon; citation_author=H Cedar; citation_author=R Axel; citation_journal_title=Cell; citation_volume=78; citation_number=9; citation_pages=823-834; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Variable patterns of axonal projections of sensory neurons in the mouse vomeronasal system.; citation_author=I Rodriguez; citation_author=P Feinstein; citation_author=P Mombaerts; citation_journal_title=Cell; citation_volume=97; citation_number=10; citation_pages=199-208; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Independent regulation of the two Pax5 alleles during B-cell development.; citation_author=SL Nutt; citation_author=S Vambrie; citation_author=P Steinlein; citation_author=Z Kozmik; citation_author=A Rolink; citation_journal_title=Nat Genet; citation_volume=21; citation_number=11; citation_pages=390-395; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Random monoallelic expression of three genes clustered within 60 kb of mouse t complex genomic DNA.; citation_author=Y Sano; citation_author=T Shimada; citation_author=H Nakashima; citation_author=RH Nicholson; citation_author=JF Eliason; citation_journal_title=Genome Res; citation_volume=11; citation_number=12; citation_pages=1833-1841; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Monoallelic expression and asynchronous replication of p120 catenin in mouse and human cells.; citation_author=AA Gimelbrant; citation_author=AW Ensminger; citation_author=P Qi; citation_author=J Zucker; citation_author=A Chess; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=13; citation_pages=1354-1359; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Methylated DNA sequences in genomic imprinting.; citation_author=JR Mann; citation_author=PE Szabo; citation_author=MR Reed; citation_author=J Singer-Sam; citation_journal_title=Crit Rev Eukaryot Gene Expression; citation_volume=10; citation_number=14; citation_pages=241-257; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Monoallelic gene expression: a repertoire of recurrent themes.; citation_author=M Goldmit; citation_author=Y Bergman; citation_journal_title=Immunol Rev; citation_volume=200; citation_number=15; citation_pages=197-214; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Use of a reverse transcriptase-polymerase chain reaction assay to analyze allele-specific expression in individual hippocampal neurons.; citation_author=SI Shen; citation_author=C Gao; citation_author=J Singer-Sam; citation_journal_title=Mol Genet Metab; citation_volume=63; citation_number=16; citation_pages=96-102; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Allele-specific expression patterns of interleukin-2 and Pax-5 revealed by a sensitive single-cell RT-PCR analysis.; citation_author=KL Rhoades; citation_author=N Singh; citation_author=I Simon; citation_author=B Glidden; citation_author=H Cedar; citation_journal_title=Curr Biol; citation_volume=10; citation_number=17; citation_pages=789-792; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A genome-wide screen for normally methylated human CpG islands that can identify novel imprinted genes.; citation_author=LZ Strichman-Almashanu; citation_author=RS Lee; citation_author=PO Onyango; citation_author=E Perlman; citation_author=F Flam; citation_journal_title=Genome Res; citation_volume=12; citation_number=18; citation_pages=543-554; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comprehensive analysis of allelic methylation status of CpG islands on human chromosome 21q.; citation_author=Y Yamada; citation_author=H Watanabe; citation_author=F Miura; citation_author=H Soejima; citation_author=M Uchiyama; citation_journal_title=Genome Res; citation_volume=14; citation_number=19; citation_pages=247-266; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative isoschizomer profiling of cytosine methylation: the HELP assay.; citation_author=B Khulan; citation_author=RF Thompson; citation_author=K Ye; citation_author=MJ Fazzari; citation_author=M Suzuki; citation_journal_title=Genome Res; citation_volume=16; citation_number=20; citation_pages=1046-1055; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Array-based profiling of reference-independent methylation status (aPRIMES) identifies frequent promoter methylation and consecutive downregulation of ZIC2 in pediatric medulloblastoma.; citation_author=S Pfister; citation_author=C Schlaeger; citation_author=F Mendrzyk; citation_author=A Wittmann; citation_author=A Benner; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=21; citation_pages=e51; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Microarray-based DNA methylation profiling: technology and applications.; citation_author=A Schumacher; citation_author=P Kapranov; citation_author=Z Kaminsky; citation_author=J Flanagan; citation_author=A Assadzadeh; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=22; citation_pages=528-542; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome organization, function, and imprinting in Prader-Willi and Angelman syndromes.; citation_author=RD Nicholls; citation_author=JL Knepper; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=2; citation_number=23; citation_pages=153-175; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structure and function correlations at the imprinted mouse Snrpn locus.; citation_author=JM Gabriel; citation_author=TA Gray; citation_author=L Stubbs; citation_author=S Saitoh; citation_author=T Ohta; citation_journal_title=Mamm Genome; citation_volume=9; citation_number=24; citation_pages=788-793; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The mouse olfactory receptor gene family.; citation_author=PA Godfrey; citation_author=B Malnic; citation_author=LB Buck; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=25; citation_pages=2156-2161; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The human olfactory receptor gene family.; citation_author=B Malnic; citation_author=PA Godfrey; citation_author=LB Buck; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=26; citation_pages=2584-2589; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cloning and initial characterization of human and mouse Spot 14 genes.; citation_author=JP Grillasca; citation_author=M Gastaldi; citation_author=H Khiri; citation_author=A Dace; citation_author=N Peyrol; citation_journal_title=FEBS Lett; citation_volume=401; citation_number=27; citation_pages=38-42; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A gene for the mouse pink-eyed dilution locus and for human type II oculocutaneous albinism.; citation_author=EM Rinchik; citation_author=SJ Bultman; citation_author=B Horsthemke; citation_author=ST Lee; citation_author=KM Strunk; citation_journal_title=Nature; citation_volume=361; citation_number=28; citation_pages=72-76; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Establishment and maintenance of DNA methylation patterns in mouse Ndn: implications for maintenance of imprinting in target genes of the imprinting center.; citation_author=ML Hanel; citation_author=R Wevrick; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=29; citation_pages=2384-2392; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Retrotransposed genes such as Frat3 in the mouse Chromosome 7C Prader-Willi syndrome region acquire the imprinted status of their insertion site.; citation_author=JH Chai; citation_author=DP Locke; citation_author=T Ohta; citation_author=JM Greally; citation_author=RD Nicholls; citation_journal_title=Mamm Genome; citation_volume=12; citation_number=30; citation_pages=813-821; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An epigenetic state associated with areas of gene duplication.; citation_author=AA Gimelbrant; citation_author=A Chess; citation_journal_title=Genome Res; citation_volume=16; citation_number=31; citation_pages=723-729; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dissociation of IGF2 and H19 imprinting in human brain.; citation_author=NV Pham; citation_author=MT Nguyen; citation_author=JF Hu; citation_author=TH Vu; citation_author=AR Hoffman; citation_journal_title=Brain Res; citation_volume=810; citation_number=32; citation_pages=1-8; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Imprinted expression of the murine Angelman syndrome gene, Ube3a, in hippocampal and Purkinje neurons.; citation_author=U Albrecht; citation_author=JS Sutcliffe; citation_author=BM Cattanach; citation_author=CV Beechey; citation_author=D Armstrong; citation_journal_title=Nat Genet; citation_volume=17; citation_number=33; citation_pages=75-78; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Coordination of the random asynchronous replication of autosomal loci.; citation_author=N Singh; citation_author=FA Ebrahimi; citation_author=AA Gimelbrant; citation_author=AW Ensminger; citation_author=MR Tackett; citation_journal_title=Nat Genet; citation_volume=33; citation_number=34; citation_pages=339-341; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sequence, structure, and evolution of a complete human olfactory receptor gene cluster.; citation_author=G Glusman; citation_author=A Sosinsky; citation_author=E Ben-Asher; citation_author=N Avidan; citation_author=D Sonkin; citation_journal_title=Genomics; citation_volume=63; citation_number=35; citation_pages=227-245; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=High concentrations of long interspersed nuclear element sequence distinguish monoallelically expressed genes.; citation_author=E Allen; citation_author=S Horvath; citation_author=F Tong; citation_author=P Kraft; citation_author=E Spiteri; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=36; citation_pages=9940-9945; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Epigenetic silencing may aid evolution by gene duplication.; citation_author=SN Rodin; citation_author=AD Riggs; citation_journal_title=J Mol Evol; citation_volume=56; citation_number=37; citation_pages=718-729; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes.; citation_author=M Lynch; citation_author=JS Conery; citation_journal_title=Science; citation_volume=290; citation_number=38; citation_pages=1151-1155; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cartilage matrix deficiency (cmd): a new autosomal recessive lethal mutation in the mouse.; citation_author=E Rittenhouse; citation_author=LC Dunn; citation_author=J Cookingham; citation_author=C Calo; citation_author=M Spiegelman; citation_journal_title=Journal of Embryology & Experimental Morphology; citation_volume=43; citation_number=39; citation_pages=71-84; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=The “Spot 14” gene resides on the telomeric end of the 11q13 amplicon and is expressed in lipogenic breast cancers: implications for control of tumor metabolism.; citation_author=JT Moncur; citation_author=JP Park; citation_author=VA Memoli; citation_author=TK Mohandas; citation_author=WB Kinlaw; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=40; citation_pages=6989-6994; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cloning of a novel prolyl 4-hydroxylase subunit expressed in the fibrous cap of human atherosclerotic plaque.; citation_author=C Van Den Diepstraten; citation_author=K Papay; citation_author=Z Bolender; citation_author=A Brown; citation_author=JG Pickering; citation_journal_title=Circulation; citation_volume=108; citation_number=41; citation_pages=508-511; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and comparative analysis of the mouse genome.; citation_author=MGS Consortium; citation_journal_title=Nature; citation_volume=420; citation_number=42; citation_pages=520-562; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The UCSC Table Browser data retrieval tool.; citation_author=D Karolchik; citation_author=AS Hinrichs; citation_author=TS Furey; citation_author=KM Roskin; citation_author=CW Sugnet; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=43; citation_pages=D493-496; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001293" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Monoallelic Expression of Multiple Genes in the CNS"/>
    <meta name="twitter:description" content="The inheritance pattern of a number of major genetic disorders suggests the possible involvement of genes that are expressed from one allele and silent on the other, but such genes are difficult to detect. Since DNA methylation in regulatory regions is often a mark of gene silencing, we modified existing microarray-based assays to detect both methylated and unmethylated DNA sequences in the same sample, a variation we term the MAUD assay. We probed a 65 Mb region of mouse Chr 7 for gene-associated sequences that show two distinct DNA methylation patterns in the mouse CNS. Selected genes were then tested for allele-specific expression in clonal neural stem cell lines derived from reciprocal F1 (C57BL/6&times;JF1) hybrid mice. In addition, using a separate approach, we directly analyzed allele-specific expression of a group of genes interspersed within clusters of OlfR genes, since the latter are subject to allelic exclusion. Altogether, of the 500 known genes in the chromosomal region surveyed, five show monoallelic expression, four identified by the MAUD assay (Agc1, p (pink-eyed dilution), P4ha3 and Thrsp), and one by its proximity to OlfR genes (Trim12). Thrsp (thyroid hormone responsive SPOT14 homolog) is expressed in hippocampus, but the human protein homolog, S14, has also been implicated in aggressive breast cancer. Monoallelic expression of the five genes is not coordinated at a chromosome-wide level, but rather regulated at individual loci. Taken together, our results suggest that at least 1% of previously untested genes are subject to allelic exclusion, and demonstrate a dual approach to expedite their identification."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001293.g006"/>

  <meta property="og:title" content="Monoallelic Expression of Multiple Genes in the CNS" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001293" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3253'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9620'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8788&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001293">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001293" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001293&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Monoallelic Expression of Multiple Genes in the CNS
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jinhui Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Division of Biology, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America, 
                  Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zuzana Valo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Division of Biology, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America, 
                  Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Smith, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Information Sciences, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Judith Singer-Sam
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jsam@coh.org">jsam@coh.org</a></p>

                <p>Affiliations:
                  Division of Biology, Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America, 
                  Beckman Research Institute, City of Hope National Medical Center, Duarte, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001293</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001293" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001293" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001293" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001293" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001293-g001" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001293-g002" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001293-g003" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001293-g004" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001293-g005" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001293-g006" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001293">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001293" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The inheritance pattern of a number of major genetic disorders suggests the possible involvement of genes that are expressed from one allele and silent on the other, but such genes are difficult to detect. Since DNA methylation in regulatory regions is often a mark of gene silencing, we modified existing microarray-based assays to detect both <em>m</em>ethylated <em>a</em>nd <em>u</em>nmethylated <em>D</em>NA sequences in the same sample, a variation we term the MAUD assay. We probed a 65 Mb region of mouse Chr 7 for gene-associated sequences that show two distinct DNA methylation patterns in the mouse CNS. Selected genes were then tested for allele-specific expression in clonal neural stem cell lines derived from reciprocal F<sub>1</sub> (C57BL/6×JF1) hybrid mice. In addition, using a separate approach, we directly analyzed allele-specific expression of a group of genes interspersed within clusters of <em>OlfR</em> genes, since the latter are subject to allelic exclusion. Altogether, of the 500 known genes in the chromosomal region surveyed, five show monoallelic expression, four identified by the MAUD assay (<em>Agc1, p</em> (pink-eyed dilution), <em>P4ha3</em> and <em>Thrsp</em>), and one by its proximity to <em>OlfR</em> genes (<em>Trim12</em>). <em>Thrsp</em> (thyroid hormone responsive SPOT14 homolog) is expressed in hippocampus, but the human protein homolog, S14, has also been implicated in aggressive breast cancer. Monoallelic expression of the five genes is not coordinated at a chromosome-wide level, but rather regulated at individual loci. Taken together, our results suggest that at least 1% of previously untested genes are subject to allelic exclusion, and demonstrate a dual approach to expedite their identification.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang J, Valo Z, Smith D, Singer-Sam J (2007) Monoallelic Expression of Multiple Genes in the CNS. PLoS ONE 2(12):
          e1293.
            doi:10.1371/journal.pone.0001293</p><p><strong>Academic Editor: </strong>Peter Heutink, Vrije Universiteit Medical Centre, Netherlands</p><p><strong>Received:</strong> October 3, 2007; <strong>Accepted:</strong> November 11, 2007; <strong>Published:</strong> December 12, 2007</p><p><strong>Copyright:</strong> © 2007 Wang et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NIH NIMH grant R21 MH073754 to JS.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A number of genetic disorders are characterized by a ~50% discordance rate between identical twins <a href="#pone.0001293-Torrey1">[1]</a>. These include schizophrenia, multiple sclerosis, bipolar disorder and type 1 diabetes. While in most cases, multiple genes have been implicated in the etiology of these diseases, allelic exclusion (i.e., random monoallelic expression) of any one of the genes could account for the observed pattern of inheritance in each case. An increasing number of genes undergoing allelic exclusion are now known. The first autosomal cases were reported for T-cell receptors <a href="#pone.0001293-Malissen1">[2]</a> and immunoglobulins <a href="#pone.0001293-Rimm1">[3]</a>. More recently, allelic exclusion has been demonstrated for additional immune response genes <a href="#pone.0001293-Held1">[4]</a>–<a href="#pone.0001293-Kelly1">[8]</a>, as well as chemosensory receptors, including olfactory receptors (<em>OlfR</em> genes) <a href="#pone.0001293-Chess1">[9]</a> and pheromone receptors <a href="#pone.0001293-Rodriguez1">[10]</a>. A few allelically excluded genes with other functions have been reported as well <a href="#pone.0001293-Nutt1">[11]</a>–<a href="#pone.0001293-Gimelbrant1">[13]</a>. Epigenetic factors contributing to monoallelic expression include DNA methylation differences at regulatory regions as well as DNA replication asynchrony; these epigenetic changes may occur early in development, providing markers for subsequent allele-specific expression (reviewed in <a href="#pone.0001293-Mann1">[14]</a> and <a href="#pone.0001293-Goldmit1">[15]</a>).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Because random allelic exclusion results in two simultaneous patterns of expression in a mixture of cells, it has been difficult to detect in tissues. Several groups have used allele-specific single-cell analysis <a href="#pone.0001293-Shen1">[16]</a>, <a href="#pone.0001293-Rhoades1">[17]</a>, but the method is not sensitive enough for the detection of genes expressed at low or moderate levels. An alternative approach, increasingly in use, involves detection of methylated CpG islands, which are frequently correlated with imprinting and monoallelic expression; the assays usually depend upon restriction enzyme analysis to identify differences in DNA methylation between different samples <a href="#pone.0001293-StrichmanAlmashanu1">[18]</a>, <a href="#pone.0001293-Yamada1">[19]</a>. The advent of high-density microarrays, including tiling arrays that contain probes for all non-repetitive sequences along a given chromosome, has led to the development of assays for detection of methylated DNA by microarray analysis <a href="#pone.0001293-Khulan1">[20]</a>–<a href="#pone.0001293-Schumacher1">[22]</a>. We have modified that approach to detect genes that have a dual DNA methylation pattern within the same tissue. To provide proof-of-principle we chose the mouse CNS as a model system, probing the portion of mouse Chr 7 that includes the imprinted Prader-Willi/Angelman Syndrome (PWS/AS) locus <a href="#pone.0001293-Nicholls1">[23]</a>: The differentially methylated <em>Snrpn</em> gene lies within this locus <a href="#pone.0001293-Gabriel1">[24]</a>, providing a positive control. Our analysis confirmed that the method can be used to identify monoallelic expression and, in addition, identifies four genes expressed monoallelically in the CNS.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To test for monoallelic expression we identified SNPs in candidate genes, and then analyzed allele-specific expression in clonal neural stem cell (NSC) lines we derived from F<sub>1</sub> hybrid mice. In addition, in a separate approach, we directly analyzed allele-specific expression of genes that were selected on the basis of proximity to <em>OlfR</em> gene clusters. The rationale for selection of <em>OlfR</em> genes was that they are the major gene family thus far known to undergo random allelic exclusion in CNS tissue <a href="#pone.0001293-Chess1">[9]</a>, <a href="#pone.0001293-Godfrey1">[25]</a>, <a href="#pone.0001293-Malnic1">[26]</a>. In addition, their occurrence in clusters was similar to that of most other known monoallelically expressed genes, particularly imprinted genes, that have been the most studied (<a href="http://www.mgu.har.mrc.ac.uk/research/imprinting">http://www.mgu.har.mrc.ac.uk/research/im​printing</a>). Using this approach, we identified an additional gene showing monoallelic expression.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>The MAUD Assay</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><a href="#pone-0001293-g001">Figure 1</a> shows a schematic diagram of the microarray-based assay that we used, modified specifically for the simultaneous detection of both methylated and unmethylated DNA sequences. Genomic DNA was cleaved with the enzyme Csp6I (G*TAC) to generate fragments ~0.5 to 1 kb in length. Universal linkers were then added, and each sample was divided into three aliquots. The first (“A”) aliquot was digested with McrBC, which cleaves methylated DNA sequences, leaving unmethylated DNA intact. The second (“B”) aliquot was treated with a mix of methyl-sensitive restriction enzymes (“RE mix”, composed of HpaII, AciI, and HpyCH4IV) that cleave DNA only at unmethylated sites. A third (“C”) aliquot was treated with both McrBC and the RE mix, providing a negative control. Following amplification by LM-PCR, and fluorescent (Cy3 or Cy5) labeling, two microarray-based hybridizations were performed: the first compared McrBC-treated sample A with the control sample C, and the second compared RE-mix-treated sample B with the same control. Hits are defined as sequences that give a strong positive signal in both hybridizations.</p>
<div class="figure" id="pone-0001293-g001"><div class="img"><a name="pone-0001293-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Outline of the MAUD assay.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Open circles, unmethylated DNA, closed circles, methylated DNA. RE mix; AciI, HpaII and HpyCH4IV. The boxed sequences remain intact following the indicated enzymatic treatment, and are subsequently amplified by LMPCR.</p>
<span>doi:10.1371/journal.pone.0001293.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We analyzed the 65 Mb portion of mouse Chr 7 shown in <a href="#pone-0001293-g002">Figure 2</a>, utilizing tiling arrays containing probes for all non-repetitive DNA at 100 bp intervals. While we examined only sequences in the vicinity of genes, we selected tiling arrays rather than promoter or GC-island arrays to include sequences that might be missed by a narrow definition of GC-rich regions or because of incomplete annotation of transcription start sites. For each gene, we scanned up to 16 kb of DNA surrounding the transcription start site(s) to include 8 kb of the 5′ region as well as the first exon and first intron.</p>
<div class="figure" id="pone-0001293-g002"><div class="img"><a name="pone-0001293-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Region of mouse Chr 7 assayed and summary of results.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p><em>Upper line.</em> The location in Mb of the PWS/AS locus is shown by the turquoise bar, and that of the <em>OlfR</em> clusters by the green bars. <em>Lower line</em>. The location of genes assayed for allele-specific expression is shown. Monoallelically expressed genes are indicated by red triangles; bi-allelically expressed genes by vertical lines (see text).</p>
<span>doi:10.1371/journal.pone.0001293.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The region we selected includes the imprinted PWS/AS locus as well as several clusters of olfactory receptor (<em>OlfR</em>) genes, as shown in <a href="#pone-0001293-g002">Figure 2</a>. While subsequent tests of allele-specificity necessarily relied on cells derived from F<sub>1</sub> hybrids, for microarray analysis we used an inbred strain (C57BL/6), avoiding potential artifacts due to use of hybrid strains. We assayed DNA from mouse brain (cortex region, i.e., forebrain). While the forebrain is a complex mixture of various cell types, we reasoned that a high proportion of brain-derived total RNA is transcribed from neurons and astrocytes, so that a dual DNA methylation pattern in either or both of these cell types might be detectable above background.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p><a href="#pone-0001293-g003">Figures 3A and 3B</a> show the hybridization profiles for the promoters of <em>Snrpn</em> and <em>H47</em>. While the <em>Snprn</em> gene is known to be imprinted and differentially methylated <a href="#pone.0001293-Gabriel1">[24]</a>, <em>H47</em> is bi-allelically expressed (our unpublished data). For both genes there is a clear peak in the lower track, representing unmethylated DNA; however, only the <em>Snrpn</em> promoter shows a peak in the upper track, representing methylated DNA. Control experiments confirmed that the active and silent alleles of the <em>Snrpn</em> gene were in the McrBC- and RE mix- treated samples, respectively, and also that the <em>H47</em> promoter is unmethylated (<a href="#pone.0001293.s001">Figure S1</a>).</p>
<div class="figure" id="pone-0001293-g003"><div class="img"><a name="pone-0001293-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>MAUD assay controls and reproducibility.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Hybridization profiles are shown for A. <em>Snrpn</em> and B. <em>H47</em>. For each gene, the x-axis shows our data aligned with the nucleotide position of genes and restriction sites along mouse Chr 7 (UCSC genome browser). The y-axis represents the log<sub>2</sub> ratio for RE treated-DNA vs. control (upper track, blue bars) and McrBC- treated DNA vs. control (lower track, purple bars). Each bar represents the value for an individual probe. The numbers above the peaks indicate the peak height for mouse 1; similar log<sub>2</sub> ratios were found for all three mice. Below each set of tracks, the vertical lines above show the location of Csp6I sites, and the lines below show the location of DNA methylation-sensitive HpaII AciI and HpyCH4IV sites. Horizontal arrows indicate the start site and orientation of transcription for each gene. The small turquoise boxes highlight the sequences analyzed for differential DNA methylation in <a href="#pone.0001293.s004">Figure S4</a>. C. Evidence for reproducibility. The Venn diagram shows positive hits (genes with clear peaks in both upper and lower tracks) in the three mice assayed. Criteria for peak selection are described in <a href="#pone.0001293.s007">Text S1</a>.</p>
<span>doi:10.1371/journal.pone.0001293.g003</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>The patterns of signal intensity of the peaks in <a href="#pone-0001293-g003">Figures 3A and 3B</a> are compatible with the method we used to prepare the DNA samples, i.e., signal intensity was generally highest in the middle of Csp6I restriction fragments, with double peaks seen if a particular GC-rich region spanned more than one fragment. Often the central portion of a GC-rich region, containing a dense concentration of methyl-sensitive restriction sites, showed poor signal intensity, but was surrounded by sizable peak shoulders (see, for example, <a href="#pone.0001293.s002">Figure S2</a>, <em>Agc1</em> promoter region). Because these peaks may escape conventional peak-finding algorithms, we used visual inspection to select genes for further analysis by allele-specific RT-PCR. Of 50 genes we selected, 10 genes were not assayable for technical reasons (e.g., no SNPs or no RT-PCR product in brain). Of the remaining 40 genes, 36 showed bi-allelic expression, and four showed monoallelic expression (see below and <a href="#pone.0001293.s006">Table S2</a>).</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>As a first step in development of a computational tool for peak analysis, we used the newly found monoallelically expressed genes and <em>Snrpn</em> as a “learning tool” to optimize criteria for peak selection (<a href="#pone.0001293.s007">Text S1</a>). A Venn diagram (<a href="#pone-0001293-g003">Figure 3B</a>) summarizes the result we obtained by using the automated criteria to analyze 3 separate mouse samples. The diagram shows that the method is reproducible, and that it yields a manageable number of candidate genes that show a dual DNA methylation pattern in all three mice.</p>


<h4>Verification of Monoallelic Expression</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To test which of the identified genes showed monoallelic expression, we analyzed the RNA of six hybrid clonal lines of NSCs. Three of the clonal lines were derived from pooled CNS tissue of F<sub>1</sub> (B6♀×JF1♂) female mice, and the other three lines were similarly derived from F<sub>1</sub> progeny of the reciprocal cross. <a href="#pone-0001293-g004">Figure 4</a> shows the different patterns of expression for the four genes found to be monoallelically expressed in at least three of the clonal lines: <em>Agc1</em>, coding for aggrecan (chondroitin sulphate proteoglycan), a necessary component of the matrix of cartilage-containing tissues; <em>Thrsp,</em> coding for a thyroid hormone- responsive protein that is expressed in the hippocampus (Allen Brain Atlas, <a href="http://www.brain-map.org">www.brain-map.org</a>), and that also catalyzes the synthesis of long chain fatty acids in breast tissue in human <a href="#pone.0001293-Grillasca1">[27]</a>; <em>p</em>, pink-eyed dilution protein (human gene <em>OCA2</em>), which is linked to albinism <a href="#pone.0001293-Rinchik1">[28]</a>; and <em>P4ha3</em>, coding for the alpha subunit of a protein that catalyzes the hydroxylation of proline in collagen. For two of the genes, <em>p</em> and <em>P4ha3</em>, all of the cell lines tested show monoallelic expression. The other two genes, <em>Agc1</em> and <em>Thrsp</em>, are expressed from the B6 allele in some clonal lines and bi-allelically in others. Complete results for each gene are shown in <a href="#pone.0001293.s003">Figure S3</a>, together with standard curves demonstrating the quantitative nature and resolution of the sequencing assay for each SNP used. For each of the genes, differential DNA methylation of a region included within the observed peaks was confirmed by bisulfite sequencing (<a href="#pone.0001293.s004">Figure S4</a>).</p>
<div class="figure" id="pone-0001293-g004"><div class="img"><a name="pone-0001293-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Monoallelic expression of <em>Agc1, p, P4ha3</em> and <em>Thrsp</em> in NSC clones.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The panels show sequencing results following RT-PCR of RNA from representative F<sub>1</sub> hybrid NSC clonal lines. Results for all six lines as well as standard curves showing the resolution of the sequencing assay are shown in <a href="#pone.0001293.s003">Figure S3</a>.</p>
<span>doi:10.1371/journal.pone.0001293.g004</span></div>

<h4>Direct Assay of Genes near <em>OlfR</em> Clusters</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We analyzed allele-specific expression of 10 genes that are in the vicinity of <em>OlfR</em> gene clusters on Chr 7 (see <a href="#pone-0001293-g002">Figure 2</a>), that contain usable SNPs, and that are expressed at adequate levels in NSCs. Nine of these showed bi-allelic expression (<a href="#pone.0001293.s006">Table S2</a>), and one, <em>Trim12</em>, tripartite motif protein (no human homolog) showed monoallelic expression (<a href="#pone-0001293-g005">Figure 5</a>). In addition to its proximity to <em>OlfR</em> clusters, a BLAT search (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>) revealed that the 1500-bp <em>Trim12</em> gene is surrounded by five genomic sequences that are nearly identical to it (80% to 96% identity over at least 30% of the gene within a 270 kb region surrounding it).</p>
<div class="figure" id="pone-0001293-g005"><div class="img"><a name="pone-0001293-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Monoallelic expression of <em>Trim12</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A restriction fragment length polymorphism permitted the use of agarose gel electrophoresis rather than sequencing to analyze allele-specific expression. A. RT-PCR products from parental strains B6 and JF1 and the six NSC clonal lines are shown. B. PCR of genomic DNA from two NSC clonal lines 2A1 and 3A1, the NSC parental strains NSC2 and NSC3, and brain DNA from strains F<sub>1</sub> (B6♀×JF1♂) (F<sub>1</sub>a), and the reciprocal cross (F<sub>1</sub>b) are shown.</p>
<span>doi:10.1371/journal.pone.0001293.g005</span></div>

<h4>Summary of Results</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p><a href="#pone-0001293-g002">Figure 2</a> summarizes our results, showing the location of the five monoallelically expresssed genes (red triangles) scattered among the 45 genes that are bi-allelically expressed (vertical lines). We see no evidence for enrichment of monoallelic expression in a particular chromosomal region. <a href="#pone-0001293-g006">Figure 6</a> shows the allele-specific expression pattern of the five genes on each of the two copies of Chr 7 in each clonal NSC line. Red lines indicate the expressed allele(s) in each case. It is apparent that there is no chromosome-wide coordination of allele-specific expression. For <em>P4ha3</em>, <em>p</em> and <em>Agc1</em>, the clonal lines derived from the same cell line (i.e., NSC2 or NSC3) share the same allele-specific expression pattern. For two of the genes, <em>Agc1</em> and <em>p,</em> the parent NSC2 or NSC3 cell line is already skewed (<a href="#pone.0001293.s003">Figure S3</a>), suggesting that the original cell lines we obtained were oligoclonal to begin with.</p>
<div class="figure" id="pone-0001293-g006"><div class="img"><a name="pone-0001293-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001293" data-uri="info:doi/10.1371/journal.pone.0001293.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001293.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001293.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001293.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001293.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Monoallelic expression patterns are locus-specific.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The region of Chr 7 from 48 Mb to 115 Mb is shown schematically (not drawn to scale). The clonal lines were derived from either NSC2 cells (JF1♀×B6♂) or NSC3 cells (B6♀×JF1♂) as shown. *, not determined.</p>
<span>doi:10.1371/journal.pone.0001293.g006</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>For four of the genes, the results summarized in <a href="#pone-0001293-g006">Figure 6</a> do not suggest any parent-of-origin effects. The results shown for <em>p</em> might suggest the possibility of imprinting, since the maternal allele is expressed in each of the 5 clonal lines assayed; however, the bi-allelic pattern seen in F<sub>1</sub> (B6×JF1) brain RNA for both reciprocal crosses (<a href="#pone.0001293.s003">Figure S3</a>) suggests that this is not the case for most CNS cells. The possibility of imprinting in neural stem cells cannot be ruled out, however.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have detected five genes that are monoallelically expressed in the CNS by analysis of DNA methylation patterns or proximity to <em>OlfR</em> genes, followed by the direct assay of F<sub>1</sub> hybrid clonal cell lines. The approach should prove useful for identification of monoallelic expression in other tissues and elsewhere in the genome, the major limitation being the difficulty of obtaining clonal cell lines for some cell types.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The assay we describe for detection of methylated and unmethylated DNA has a number of advantages in addition to its potential for high-throughput analysis: Inbred strains may be assayed, avoiding artifacts due to strain differences, and the use of tiling arrays allows latitude in defining sequences of interest. However the assay also has limitations: 1) Not all allelic exclusion is associated with differential DNA methylation. An example of this is the imprinted <em>Ndn</em> gene within the PWS/AS locus, which shows heterogeneity of methylation patterns on both alleles in the brain (<a href="#pone.0001293-Hanel1">[29]</a> and our unpublished data). 2) The assay depends upon having completely methylated or unmethylated DNA at the relevant restriction sites within a given Csp6I fragment: If a fragment shows partial methylation on either the silent or active allele, the signal for that allele will probably escape detection. For example, partial methylation at some methyl-sensitive restriction sites is the probable cause of our failure to observe a reproducible signal for two additional imprinted genes within the PWS/AS locus, <em>Peg12</em> and <em>Mkrn3</em> (<a href="#pone.0001293-Chai1">[30]</a> and our unpublished data). 3) Very highly GC-rich regions may show attenuated or no hybridization signal, possibly due to the difficulty of amplification. The latter two limitations are somewhat ameliorated by use of a broad window (transcription start point±8 kb); a gene is included among “hits” if any one of the Csp6I fragments within the window exceeds the signal threshold.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Although rigorous statistical analysis will not be useful until more data sets are obtained by use of our “supervised learning” approach, it seems very likely that we have enriched for genes undergoing allelic exclusion by use of our DNA methylation-based assay. Empirically we found that the assay reduced the list of potentially “interesting” genes to a manageable number (50 to 60 genes showing dual DNA methylation patterns), and that of these genes 10% were monoallelic (4 of the 40 genes for which we found usable SNPs).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We estimate that a minimum of ~1% of previously untested genes (5 of 500 genes surveyed) undergo allelic exclusion in addition to those already known. This estimate is consistent with the findings of Gimelbrant and Chess <a href="#pone.0001293-Gimelbrant2">[31]</a>, who used an approach based on asynchronous replication. For two of the genes, we found monoallelic expression in all six clonal cell lines, while for the three others, we found a mixed pattern, with some lines showing bi-allelic expression. This is not likely to be a tissue culture artifact. When brain tissue was assayed for DNA methylation, the genes showing the most clear-cut dual methylation profiles in the MAUD assay were those showing monoallelic expression in all the clonal lines (<a href="#pone.0001293.s002">Figure S2</a>). In addition our control experiments (unpublished) showed that the cell lines all retained silencing of X-linked genes, and a mixed pattern for genes that are mono- or bi-allelically expressed in the CNS depending upon cell type or stage of development <a href="#pone.0001293-Pham1">[32]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>For the five genes we report here, the pattern of allele-specific expression appears to show no chromosome-wide coordination. This was expected, based upon previously published work showing gene-to-gene variability in expression patterns, even within monoallelically expressed gene clusters. For example, cytokines and the NK cell receptors are regulated in a probabilistic manner, with cells in an activated clonal population capable of expressing either or both alleles; <a href="#pone.0001293-Held1">[4]</a>, <a href="#pone.0001293-Bix1">[6]</a>, <a href="#pone.0001293-Rivire1">[7]</a>. It has been suggested that this stochastic process is part of development of the immune response repertoire. IL-4, Il-5 and IL-13 lie within 140 kb of each other, and are coordinately regulated <a href="#pone.0001293-Kelly1">[8]</a>; however, IL-3, which lies within 450 kb of these genes, shows allele-specific expression that is regulated independently of the neighboring cytokines.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Even when there is co-ordinated control within an imprinted domain, the regulation mechanism may involve oppositely imprinted genes within the domain. For example at the PWS/AS locus the tightly regulated paternally expressed <em>Snrpn</em> gene is within 1 Mb of the <em>Ube3a</em> gene, which is oppositely imprinted in some cells of the CNS and bi-allelically expressed in others <a href="#pone.0001293-Albrecht1">[33]</a>. Singh et al. <a href="#pone.0001293-Singh1">[34]</a> reported evidence for chromosome-wide co-ordination of asynchronous replication of <em>OlfR</em> and immune response genes, but their results are not necessarily at variance with locus-specific control. It may be that allele-specific expression of some subclasses of genes is regulated at a chromosomal level, or that asynchronous DNA replication of large domains does not preclude a random pattern of allele-specific expression.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p><em>Trim12</em>, the monoallelically expressed gene that we selected by proximity to <em>OlfR</em> genes, is also surrounded by recently duplicated sequences and LINE elements, which are generally abundant at <em>OlfR</em> clusters <a href="#pone.0001293-Glusman1">[35]</a>. Previous work has suggested a link between some monoallelic expression (and asynchronous replication) and gene duplication/repetitive elements <a href="#pone.0001293-Gimelbrant2">[31]</a>, <a href="#pone.0001293-Allen1">[36]</a>. There has been recent interest in the origins of this association, including speculation that tissue-specific silencing of duplicated genes might be evolutionarily advantageous directly or indirectly <a href="#pone.0001293-Gimelbrant2">[31]</a>, <a href="#pone.0001293-Rodin1">[37]</a>, <a href="#pone.0001293-Lynch1">[38]</a>. While more work is needed in this area, it will be worthwhile to continue to use the potential link between allelic exclusion and sequence duplication in the search for new candidate genes.</p>

<h4>Relevance of Monoallelic Expression to Human Disorders</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The search for genes undergoing allelic exclusion, in conjunction with mapping of disease susceptibility loci, may help identify candidate genes for genetic disorders that display discordance among identical twins. Such genes are also of interest in that they are more vulnerable to genetic and spontaneous mutation because allelic exclusion results in functional hemizygosity. Finally, an understanding of the silencing mechanism for genes undergoing allelic exclusion should prove valuable in designing eventual therapies in cases where such genes are implicated in specific disorders.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The potential importance of understanding allele-specific gene silencing is underscored for three of the genes we have identified, <em>Agc1</em>, <em>Thrsp</em> and <em>P4ha3</em>. Proof of monoallelic expression for these genes in human tissue would constitute a separate study, but it is worth noting that in mice, mutation of <em>Agc1</em> leads to a lethal phenotype at birth, and that there is a possible link to the human genetic disorder achondroplasia <a href="#pone.0001293-Rittenhouse1">[39]</a>. In the case of <em>Thrsp</em> and <em>P4ha3</em>, studies on humans have shown that over-expression of the genes may be deleterious, associated with a poor prognosis for breast cancer and with atherosclerosis, respectively <a href="#pone.0001293-Grillasca1">[27]</a>, <a href="#pone.0001293-Moncur1">[40]</a>, <a href="#pone.0001293-VanDenDiepstraten1">[41]</a>. For genes such as these, knowledge of the endogenous silencing mechanism might be a first step in reversing over-expression of a mutant allele or re-activating a silent one.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Mouse Strains and Neural Stem Cell (NSC) lines</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Four-to-eight week-old C57BL6 (B6), Japanese Fancy Mouse 1 (JF1) and F<sub>1</sub> hybrid female offspring of B6×JF1 mice were used. Isolation of NSC2 and NSC3 lines and the clonal cell lines derived from them is described in <a href="#pone.0001293.s008">Text S2</a>. Clonal cell lines 2A1, 2A4 and 2A5 were isolated from cell line NSC2, derived from F<sub>1</sub> (JF1♀×B6♂) mice; clonal lines 3A1, 4A5 and 4B3 were from cell line NSC3, derived from F<sub>1</sub> (B6♀×JF1♂) mice.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>All clonal lines tested retained X-chromosome inactivation as measured by allele-specific expression of <em>Rps4x</em> and <em>Pgk1</em>. The chromosome number of several clonal lines tested was near-euploid, but, because of some variation in karyotype with passage number, for each gene showing monoallelic expression we verified the presence of both alleles by PCR (<a href="#pone.0001293.s003">Figure S3</a>).</p>


<h4>DNA and RNA Isolation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Genomic DNA was extracted from minced brain by treatment with proteinase K, followed by phenol-chloroform extraction and ethanol precipitation. DNA from cultured NSC lines was extracted with the Wizard® Genomic DNA Purification Kit&nbsp;(Promega, Madison, WI). Total RNA from brain or NSC lines was isolated by use of TRIZOL (Invitrogen, Carlsbad, CA).</p>


<h4>The MAUD Assay</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>DNA (4 µg) was digested with 20 U of Csp6I for 4 hr at 37°C (Fermentas, Hanover, MD). The resulting restriction fragments were ligated to adaptor set TA-1b as previously described <a href="#pone.0001293-Schumacher1">[22]</a>, except that incubations were carried out at 16°C for 16 hr. DNA was then divided into aliquots, and digested in one of three ways (restriction enzymes and buffers were all from New England Biolabs (Ipswich, MA)): <em>A</em>. DNA was treated with McrBC; <em>B</em>, DNA was digested first with HpaII and HpyCH4IV in buffer 1, then with AciI in buffer 3; the two steps each consisted of two rounds of incubation, 1.5 hr each, with 3–5 U enzyme per µg DNA added at each round: <em>C</em>, control DNA was treated with McrBC and the three additional restriction enzymes. All samples were purified by use of a Qiaquick PCR purification kit (Qiagen, Valencia CA) following each enzymatic step.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Amplification by LM-PCR was carried out as previously described <a href="#pone.0001293-Schumacher1">[22]</a> with modification. Each ligated and digested DNA sample (600 ng) was amplified in 6 Eppendorf tubes, each containing 100 ng of DNA, 1× ThermoPol Reaction Buffer, 2.0 mM MgSO<sub>4</sub>, 200 µM dNTPs, 400 pm of primer TA-1b, and 3 U Vent<sup>exo−</sup> (New England Biolabs), in a final volume of 100 µl. The amplified DNA was purified by use of the QIAquick PCR purification kit. Real time PCR was performed for each of the amplified DNA samples to confirm at least an 8-fold enrichment for known differentially methylated sequences in samples <em>A</em> and <em>B</em> compared with sample <em>C</em> (data not shown).</p>


<h4>Microarray Analysis and Data Processing</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Amplified DNA samples were shipped to Nimblegen Systems Inc. (Madison, WI) for hybridization to MM8 set16 tiling arrays (mouse Chr 7 48 Mb to 115 Mb). For each mouse, results of the two hybridizations (<em>A</em> vs. <em>C</em>, and <em>B</em> vs. <em>C</em>, see <a href="#pone-0001293-g001">Figure 1</a>) were examined visually by use of SignalMap (Nimblegen) or the UCSC Genome Bioinformatics browser (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>) <a href="#pone.0001293-Consortium1">[42]</a>, <a href="#pone.0001293-Karolchik1">[43]</a>. The distance from each peak to the nearest gene on mouse Chr 7 was determined after downloading the pertinent files from the UCSC website. Peaks of interest were defined as either less than 8 kb upstream of a transcription start site or within the first exon or first intron of a given gene. “Hits” were selected for further analysis after exclusion of <em>OlfR</em> genes and genes not expressed in the brain as determined by use of the Unigene gene viewer (<a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>), and the Allen Brain Atlas (<a href="http://www.brain-map.org">http://www.brain-map.org</a>). Details of the automated peak selection protocol we used subsequently are in <a href="#pone.0001293.s007">Text S1</a>.</p>


<h4>Assay for Monoallelic Expression</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Primers, SNPs and specific PCR conditions are listed in <a href="#pone.0001293.s005">Table S1</a>. Primers were designed by use of primer3 (<a href="http://frodo.wi.mit.edu">http://frodo.wi.mit.edu</a>). SNPs were identified by RT-PCR of B6 and JF1 brain RNA followed by automated DNA sequencing (City of Hope DNA sequencing Core). RT was carried out on total RNA (1.0 µg) by use of ThermoScript<sup>TM</sup> kits (Invitrogen) with oligo(dT) primers. One µl of each 20-µl reaction mix was then used for amplification by PCR. Each reaction mix contained, in a final volume of 50 µl, PCR buffer II, 200 µM dNTPs, MgCl<sub>2</sub>, 100 pm of each primer, and 0.8 U Amplitaq DNA polymerase (Applied Biosystems, Foster City CA). After an initial denaturation (5 min 95°C), amplification was carried out for up to 35 cycles of 94°C for 45 s, annealing temperature for 45 s, 72°C for 1 min, followed by a final elongation step at 72°C for 5 min. RT-PCR products were gel-purified from a 1.5% agarose gel by use of a QIAquick Gel Extraction Kit (Qiagen) prior to sequencing.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001293"></div><a name="pone.0001293.s001" id="pone.0001293.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">DNA methylation of control genes. A. Sequence analysis of the amplified Snrpn promoter shows allele-specific DNA methylation. Two left panels: Identification of a SNP between strains B6 and JF1. Middle panel: Both alleles are present in F1 progeny of the cross (JF1MAT×B6PAT). Two right panels: The unmethylated paternal or methylated maternal allele remains intact following treatment with McrBC or the RE mix (HpaII-AciI-HpyCH4IV), respectively. The primers used were previously described to identify the same polymorphism 104 bp upstream of the major transcription start site for Snrpn in Mus musculus castaneus-Ei <a href="#pone.0001293-Torrey1">[1]</a>. B. Bisulfite analysis of CpG sites within the H47 promoter. DNA from mouse forebrains was treated with McrBC or the RE mix, but only McrBC-treated DNA yielded a PCR product; amplified DNA samples were sequenced directly following bisulfite treatment without sub-cloning. Each line shows results for one mouse (n = 3). The CpG sites within the amplicon are numbered. Restriction sites for McrBC, AciI, HpaII and HpyCH4IV are indicated by M, A, H and Y, respectively. The legend to <a href="#pone.0001293.s004">Figure S4</a> contains details of bisulfite sequencing. Primers and chromosomal co-ordinates for the region analyzed are listed in <a href="#pone.0001293.s005">Table S1</a>. Reference 1. Xin Z, Tachibana M, Guggiari M, Heard E, Shinkai Y, et al. (2003) Role of histone methyltransferase G9a in CpG methylation of the Prader-Willi syndrome imprinting center. J Biol Chem 278: 14996–15000.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.37 MB PDF)</p>
<a name="pone.0001293.s002" id="pone.0001293.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Detection of dual DNA methylation patterns by the MAUD assay. Results are shown for genes subsequently found to be monoallelically expressed. A. Agc1. B. p (pink-eye dilution). C. P4ha3. D. Thrsp. The x-axis shows the nucleotide position along mouse Chr 7 (numbered at the top). The y-axis indicates the log2 ratio for RE mix-treated DNA vs. control (top track) and McrBC-treated DNA vs. control (bottom track). Maxium peak height (log2) ratios are indicated for peaks that are coincident in both tracks. Results are shown for mouse 1; similar ratios were found for all three mice. Below the two tracks, the blue vertical lines (top) show the location of Csp6I sites, and the black lines (bottom) show the location of DNA methylation-sensitive HpaII AciI and HpyCH4IV sites. Short horizontal arrows indicate the start site and orientation of transcription for each gene; just above each arrow, the positions of exons (bars) and introns (small arrows) are shown. The turquoise boxes highlight the regions analyzed by bisulfite sequencing (<a href="#pone.0001293.s004">Figure S4</a>). The figures were obtained by alignment of our custom tracks with annotation showing the location of the genes and restriction enzyme sites indicated (UCSC Genome Browser).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0001293.s003" id="pone.0001293.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Monoallelic expression of Agc1, p, P4ha3 and Thrsp. Allele-specific expression for each gene was measured by automated sequencing of RT-PCR products containing SNPs. For A-D, Top row, brain tissue from B6 and JF1 mice and F1 hybrid progeny. Middle and bottom rows, NSC2- and NSC3- derived clonal lines, respectively. The parental cell lines NSC2 and NSC3 are shown at the left. Results of PCR of genomic DNA from representative clonal lines are shown as indicated, verifying the presence of both alleles. E. Quantitation and resolution of the assay. For each gene RT-PCR products of strains B6 and JF1 were mixed in the proportions shown prior to automated sequencing (% input). The relative intensity (peak height) of the signal for each base at SNP sites was measured to determine the % signal. For each RNA sample analyzed, if more than one base was detected at a SNP, expression was considered to be monoallelic if the predominant base comprised at least 95% of the signal. At least three technical replicates were performed for each cell line, giving the same result.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.31 MB PDF)</p>
<a name="pone.0001293.s004" id="pone.0001293.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Bisulfite analysis of CpG sites for Agc1, p, P4ha3 and Thrsp. Upper panels, McrBC-treated DNA; lower panels, RE mix-treated DNA. CpG sites included within each amplicon are numbered. Bisulfite sequencing of DNA of mouse 1 was carried out by use of the Methylation-Gold Kit (ZYMO, Orange, CA). PCR reaction mixes contained in a 20 µl volume, 50 ng of bisulfite-modified DNA, 1× PCR buffer, 2.0 mM MgCl2, 200 µM dNTPs, 40 pm of each primer, and HotStart Taq DNA polymerase, 0.5 U (Qiagen). PCR conditions were: 95°C for 15 min, followed by 35 cycles of 94°C for 45 s, annealing temperature for 45 s, and 72°C for 40 s, followed by a final incubation at 72°C for 5 min. PCR products were purified by use of the QIAquick Gel Extraction Kit (Qiagen), and then subcloned into the vector pCR2.1 (Invitrogen) prior to sequencing. Restriction sites for McrBC, AciI, HpaII and HpyCH4IV are indicated by M, A, H and Y, respectively. Primer sequences and annealing temperatures are listed in <a href="#pone.0001293.s005">Table S1</a>. The chromosomal co-ordinates of the regions analyzed are listed in <a href="#pone.0001293.s005">Table S1</a>, and visually shown in <a href="#pone.0001293.s002">Figure S2</a>. Note that for Thrsp CpG site 2 appears to be methylated although it is resistant to McrBC cleavage, consistent with occasional bias that we and others observe following subcloning of bisulfite-treated DNA.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0001293.s005" id="pone.0001293.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s005">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">List of primers. The list includes primers for PCR, RT-PCR and bisulfite sequencing as well as PCR conditions.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0001293.s006" id="pone.0001293.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s006">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Bi-allelic expression of genes on mouse Chr 7. The list includes 36 genes analyzed by RT-PCR of clonal hybrid neural stem cell lines</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.09 MB PDF)</p>
<a name="pone.0001293.s007" id="pone.0001293.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s007">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Automated peak selection. The computational method used to analyze the peaks obtained by microarray hybridization is described.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001293.s008" id="pone.0001293.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001293.s008">Text S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Isolation of NSCs and clonal cell lines</p>
<p class="siDoi">doi:10.1371/journal.pone.0001293.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. Yanhong Shi for help and advice in establishing neural stem cell lines, and Steve Bates for performing the karyotyping of neural stem cells. We also thank Dr. Susan Kane and Ms. Anja Scholze for thoughtful comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JS JW. Performed the experiments: JW ZV. Analyzed the data: JS JW ZV. Contributed reagents/materials/analysis tools: DS. Wrote the paper: JS JW ZV DS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001293-Torrey1" id="pone.0001293-Torrey1"></a>Torrey EF, Bowler AE, Taylor EH, Gottesman II (1994) Schizophrenia and Manic-Depressive Disorder. New York: HarperCollins.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001293-Malissen1" id="pone.0001293-Malissen1"></a>Malissen M, Trucy J, Jouvin-Marche E, Cazenave PA, Scollay R, et al.  (1992) Regulation of TCR α and β gene allelic exclusion during T-cell development. Immunol Today  13: 315–322.  <ul class="find" data-citedArticleID="1028394" data-doi="10.1016/0167-5699(92)90044-8"><li><a href="http://dx.doi.org/10.1016/0167-5699(92)90044-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+TCR+%CE%B1+and+%CE%B2+gene+allelic+exclusion+during+T-cell+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+TCR+%CE%B1+and+%CE%B2+gene+allelic+exclusion+during+T-cell+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001293-Rimm1" id="pone.0001293-Rimm1"></a>Rimm IJ, Bloch DB, Seidman JG (1989) Allelic exclusion and lymphocyte development. Lessons from transgenic mice. Mol Biol Med  6: 355–364.  <ul class="find" data-citedArticleID="1028412"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Allelic+exclusion+and+lymphocyte+development.+Lessons+from+transgenic+mice.&amp;auth=&amp;atitle=Allelic+exclusion+and+lymphocyte+development.+Lessons+from+transgenic+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allelic+exclusion+and+lymphocyte+development.+Lessons+from+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allelic+exclusion+and+lymphocyte+development.+Lessons+from+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001293-Held1" id="pone.0001293-Held1"></a>Held W, Raulet DH (1997) Expression of the <em>Ly49A</em> gene in murine natural killer cell clones is predominantly but not exclusively mono-allelic. Eur J Immunol  27: 2876–2884.  <ul class="find" data-citedArticleID="1028382" data-doi="10.1002/eji.1830271120"><li><a href="http://dx.doi.org/10.1002/eji.1830271120" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+Ly49A+gene+in+murine+natural+killer+cell+clones+is+predominantly+but+not+exclusively+mono-allelic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+Ly49A+gene+in+murine+natural+killer+cell+clones+is+predominantly+but+not+exclusively+mono-allelic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001293-Hollnder1" id="pone.0001293-Hollnder1"></a>Holländer GA, Zuklys S, Morel C, Mizoguchi E, Mobisson K, et al.  (1998) Monoallelic expression of the interleukin-2 locus. Science  279: 2118–2121.  <ul class="find" data-citedArticleID="1028384" data-doi="10.1126/science.279.5359.2118"><li><a href="http://dx.doi.org/10.1126/science.279.5359.2118" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monoallelic+expression+of+the+interleukin-2+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monoallelic+expression+of+the+interleukin-2+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001293-Bix1" id="pone.0001293-Bix1"></a>Bix M, Locksley RM (1998) Independent and epigenetic regulation of the interleukin-4 alleles in CD4+ T cells. Science  281: 1352–1354.  <ul class="find" data-citedArticleID="1028358" data-doi="10.1126/science.281.5381.1352"><li><a href="http://dx.doi.org/10.1126/science.281.5381.1352" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independent+and+epigenetic+regulation+of+the+interleukin-4+alleles+in+CD4%2B+T+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independent+and+epigenetic+regulation+of+the+interleukin-4+alleles+in+CD4%2B+T+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001293-Rivire1" id="pone.0001293-Rivire1"></a>Rivière I, Sunshine MJ, Littman DR (1998) Regulation of IL-4 expression by activation of individual alleles. Immunity  9: 217–228.  <ul class="find" data-citedArticleID="1028418" data-doi="10.1016/s1074-7613(00)80604-4"><li><a href="http://dx.doi.org/10.1016/s1074-7613(00)80604-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+IL-4+expression+by+activation+of+individual+alleles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+IL-4+expression+by+activation+of+individual+alleles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001293-Kelly1" id="pone.0001293-Kelly1"></a>Kelly BL, Locksley RM (2000) Coordinate regulation of the IL-4, IL-13, and IL-5 cytokine cluster in Th2 clones revealed by allelic expression patterns. J Immunol  165: 2982–2986.  <ul class="find" data-citedArticleID="1028388"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Coordinate+regulation+of+the+IL-4%2C+IL-13%2C+and+IL-5+cytokine+cluster+in+Th2+clones+revealed+by+allelic+expression+patterns.&amp;auth=&amp;atitle=Coordinate+regulation+of+the+IL-4%2C+IL-13%2C+and+IL-5+cytokine+cluster+in+Th2+clones+revealed+by+allelic+expression+patterns." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coordinate+regulation+of+the+IL-4%2C+IL-13%2C+and+IL-5+cytokine+cluster+in+Th2+clones+revealed+by+allelic+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coordinate+regulation+of+the+IL-4%2C+IL-13%2C+and+IL-5+cytokine+cluster+in+Th2+clones+revealed+by+allelic+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001293-Chess1" id="pone.0001293-Chess1"></a>Chess A, Simon I, Cedar H, Axel R (1994) Allelic inactivation regulates olfactory receptor gene expression. Cell  78: 823–834.  <ul class="find" data-citedArticleID="1028362" data-doi="10.1016/s0092-8674(94)90562-2"><li><a href="http://dx.doi.org/10.1016/s0092-8674(94)90562-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allelic+inactivation+regulates+olfactory+receptor+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allelic+inactivation+regulates+olfactory+receptor+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001293-Rodriguez1" id="pone.0001293-Rodriguez1"></a>Rodriguez I, Feinstein P, Mombaerts P (1999) Variable patterns of axonal projections of sensory neurons in the mouse vomeronasal system. Cell  97: 199–208.  <ul class="find" data-citedArticleID="1028422" data-doi="10.1016/s0092-8674(00)80730-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80730-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variable+patterns+of+axonal+projections+of+sensory+neurons+in+the+mouse+vomeronasal+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variable+patterns+of+axonal+projections+of+sensory+neurons+in+the+mouse+vomeronasal+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001293-Nutt1" id="pone.0001293-Nutt1"></a>Nutt SL, Vambrie S, Steinlein P, Kozmik Z, Rolink A, et al.  (1999) Independent regulation of the two Pax5 alleles during B-cell development. Nat Genet  21: 390–395.  <ul class="find" data-citedArticleID="1028404" data-doi="10.1038/7720"><li><a href="http://dx.doi.org/10.1038/7720" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independent+regulation+of+the+two+Pax5+alleles+during+B-cell+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independent+regulation+of+the+two+Pax5+alleles+during+B-cell+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001293-Sano1" id="pone.0001293-Sano1"></a>Sano Y, Shimada T, Nakashima H, Nicholson RH, Eliason JF, et al.  (2001) Random monoallelic expression of three genes clustered within 60 kb of mouse t complex genomic DNA. Genome Res  11: 1833–1841.  <ul class="find" data-citedArticleID="1028424"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Random+monoallelic+expression+of+three+genes+clustered+within+60+kb+of+mouse+t+complex+genomic+DNA.&amp;auth=&amp;atitle=Random+monoallelic+expression+of+three+genes+clustered+within+60+kb+of+mouse+t+complex+genomic+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Random+monoallelic+expression+of+three+genes+clustered+within+60+kb+of+mouse+t+complex+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Random+monoallelic+expression+of+three+genes+clustered+within+60+kb+of+mouse+t+complex+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001293-Gimelbrant1" id="pone.0001293-Gimelbrant1"></a>Gimelbrant AA, Ensminger AW, Qi P, Zucker J, Chess A (2005) Monoallelic expression and asynchronous replication of p120 catenin in mouse and human cells. J Biol Chem  280: 1354–1359.  <ul class="find" data-citedArticleID="1028368" data-doi="10.1074/jbc.m411283200"><li><a href="http://dx.doi.org/10.1074/jbc.m411283200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monoallelic+expression+and+asynchronous+replication+of+p120+catenin+in+mouse+and+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monoallelic+expression+and+asynchronous+replication+of+p120+catenin+in+mouse+and+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001293-Mann1" id="pone.0001293-Mann1"></a>Mann JR, Szabo PE, Reed MR, Singer-Sam J (2000) Methylated DNA sequences in genomic imprinting. Crit Rev Eukaryot Gene Expression  10: 241–257.  <ul class="find" data-citedArticleID="1028398" data-doi="10.1615/critreveukargeneexpr.v10.i3-4.30"><li><a href="http://dx.doi.org/10.1615/critreveukargeneexpr.v10.i3-4.30" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylated+DNA+sequences+in+genomic+imprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylated+DNA+sequences+in+genomic+imprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001293-Goldmit1" id="pone.0001293-Goldmit1"></a>Goldmit M, Bergman Y (2004) Monoallelic gene expression: a repertoire of recurrent themes. Immunol Rev  200: 197–214.  <ul class="find" data-citedArticleID="1028376" data-doi="10.1111/j.0105-2896.2004.00158.x"><li><a href="http://dx.doi.org/10.1111/j.0105-2896.2004.00158.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monoallelic+gene+expression%3A+a+repertoire+of+recurrent+themes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monoallelic+gene+expression%3A+a+repertoire+of+recurrent+themes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001293-Shen1" id="pone.0001293-Shen1"></a>Shen SI, Gao C, Singer-Sam J (1998) Use of a reverse transcriptase-polymerase chain reaction assay to analyze allele-specific expression in individual hippocampal neurons. Mol Genet Metab  63: 96–102.  <ul class="find" data-citedArticleID="1028428" data-doi="10.1006/mgme.1997.2668"><li><a href="http://dx.doi.org/10.1006/mgme.1997.2668" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+a+reverse+transcriptase-polymerase+chain+reaction+assay+to+analyze+allele-specific+expression+in+individual+hippocampal+neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+a+reverse+transcriptase-polymerase+chain+reaction+assay+to+analyze+allele-specific+expression+in+individual+hippocampal+neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001293-Rhoades1" id="pone.0001293-Rhoades1"></a>Rhoades KL, Singh N, Simon I, Glidden B, Cedar H, et al.  (2000) Allele-specific expression patterns of interleukin-2 and Pax-5 revealed by a sensitive single-cell RT-PCR analysis. Curr Biol  10: 789–792.  <ul class="find" data-citedArticleID="1028410" data-doi="10.1016/s0960-9822(00)00565-0"><li><a href="http://dx.doi.org/10.1016/s0960-9822(00)00565-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allele-specific+expression+patterns+of+interleukin-2+and+Pax-5+revealed+by+a+sensitive+single-cell+RT-PCR+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allele-specific+expression+patterns+of+interleukin-2+and+Pax-5+revealed+by+a+sensitive+single-cell+RT-PCR+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001293-StrichmanAlmashanu1" id="pone.0001293-StrichmanAlmashanu1"></a>Strichman-Almashanu LZ, Lee RS, Onyango PO, Perlman E, Flam F, et al.  (2002) A genome-wide screen for normally methylated human CpG islands that can identify novel imprinted genes. Genome Res  12: 543–554.  <ul class="find" data-citedArticleID="1028432" data-doi="10.1101/gr.224102"><li><a href="http://dx.doi.org/10.1101/gr.224102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+screen+for+normally+methylated+human+CpG+islands+that+can+identify+novel+imprinted+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+screen+for+normally+methylated+human+CpG+islands+that+can+identify+novel+imprinted+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001293-Yamada1" id="pone.0001293-Yamada1"></a>Yamada Y, Watanabe H, Miura F, Soejima H, Uchiyama M, et al.  (2004) A comprehensive analysis of allelic methylation status of CpG islands on human chromosome 21q. Genome Res  14: 247–266.  <ul class="find" data-citedArticleID="1028438" data-doi="10.1101/gr.1351604"><li><a href="http://dx.doi.org/10.1101/gr.1351604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+analysis+of+allelic+methylation+status+of+CpG+islands+on+human+chromosome+21q." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+analysis+of+allelic+methylation+status+of+CpG+islands+on+human+chromosome+21q.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001293-Khulan1" id="pone.0001293-Khulan1"></a>Khulan B, Thompson RF, Ye K, Fazzari MJ, Suzuki M, et al.  (2006) Comparative isoschizomer profiling of cytosine methylation: the HELP assay. Genome Res  16: 1046–1055.  <ul class="find" data-citedArticleID="1028390" data-doi="10.1101/gr.5273806"><li><a href="http://dx.doi.org/10.1101/gr.5273806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+isoschizomer+profiling+of+cytosine+methylation%3A+the+HELP+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+isoschizomer+profiling+of+cytosine+methylation%3A+the+HELP+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001293-Pfister1" id="pone.0001293-Pfister1"></a>Pfister S, Schlaeger C, Mendrzyk F, Wittmann A, Benner A, et al.  (2007) Array-based profiling of reference-independent methylation status (aPRIMES) identifies frequent promoter methylation and consecutive downregulation of ZIC2 in pediatric medulloblastoma. Nucleic Acids Res  35: e51.  <ul class="find" data-citedArticleID="1028406" data-doi="10.1093/nar/gkm094"><li><a href="http://dx.doi.org/10.1093/nar/gkm094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Array-based+profiling+of+reference-independent+methylation+status+%28aPRIMES%29+identifies+frequent+promoter+methylation+and+consecutive+downregulation+of+ZIC2+in+pediatric+medulloblastoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Array-based+profiling+of+reference-independent+methylation+status+%28aPRIMES%29+identifies+frequent+promoter+methylation+and+consecutive+downregulation+of+ZIC2+in+pediatric+medulloblastoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001293-Schumacher1" id="pone.0001293-Schumacher1"></a>Schumacher A, Kapranov P, Kaminsky Z, Flanagan J, Assadzadeh A, et al.  (2006) Microarray-based DNA methylation profiling: technology and applications. Nucleic Acids Res  34: 528–542.  <ul class="find" data-citedArticleID="1028426" data-doi="10.1093/nar/gkj461"><li><a href="http://dx.doi.org/10.1093/nar/gkj461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray-based+DNA+methylation+profiling%3A+technology+and+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray-based+DNA+methylation+profiling%3A+technology+and+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001293-Nicholls1" id="pone.0001293-Nicholls1"></a>Nicholls RD, Knepper JL (2001) Genome organization, function, and imprinting in Prader-Willi and Angelman syndromes. Annu Rev Genomics Hum Genet  2: 153–175.  <ul class="find" data-citedArticleID="1028402"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+organization%2C+function%2C+and+imprinting+in+Prader-Willi+and+Angelman+syndromes.&amp;auth=&amp;atitle=Genome+organization%2C+function%2C+and+imprinting+in+Prader-Willi+and+Angelman+syndromes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+organization%2C+function%2C+and+imprinting+in+Prader-Willi+and+Angelman+syndromes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+organization%2C+function%2C+and+imprinting+in+Prader-Willi+and+Angelman+syndromes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001293-Gabriel1" id="pone.0001293-Gabriel1"></a>Gabriel JM, Gray TA, Stubbs L, Saitoh S, Ohta T, et al.  (1998) Structure and function correlations at the imprinted mouse Snrpn locus. Mamm Genome  9: 788–793.  <ul class="find" data-citedArticleID="1028366" data-doi="10.1007/s003359900868"><li><a href="http://dx.doi.org/10.1007/s003359900868" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+correlations+at+the+imprinted+mouse+Snrpn+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+correlations+at+the+imprinted+mouse+Snrpn+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001293-Godfrey1" id="pone.0001293-Godfrey1"></a>Godfrey PA, Malnic B, Buck LB (2004) The mouse olfactory receptor gene family. Proc Natl Acad Sci U S A  101: 2156–2161.  <ul class="find" data-citedArticleID="1028374" data-doi="10.1073/pnas.0308051100"><li><a href="http://dx.doi.org/10.1073/pnas.0308051100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mouse+olfactory+receptor+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mouse+olfactory+receptor+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001293-Malnic1" id="pone.0001293-Malnic1"></a>Malnic B, Godfrey PA, Buck LB (2004) The human olfactory receptor gene family. Proc Natl Acad Sci U S A  101: 2584–2589.  <ul class="find" data-citedArticleID="1028396" data-doi="10.1073/pnas.0307882100"><li><a href="http://dx.doi.org/10.1073/pnas.0307882100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+olfactory+receptor+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+olfactory+receptor+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001293-Grillasca1" id="pone.0001293-Grillasca1"></a>Grillasca JP, Gastaldi M, Khiri H, Dace A, Peyrol N, et al.  (1997) Cloning and initial characterization of human and mouse Spot 14 genes. FEBS Lett  401: 38–42.  <ul class="find" data-citedArticleID="1028378" data-doi="10.1016/s0014-5793(96)01433-0"><li><a href="http://dx.doi.org/10.1016/s0014-5793(96)01433-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+initial+characterization+of+human+and+mouse+Spot+14+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+initial+characterization+of+human+and+mouse+Spot+14+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001293-Rinchik1" id="pone.0001293-Rinchik1"></a>Rinchik EM, Bultman SJ, Horsthemke B, Lee ST, Strunk KM, et al.  (1993) A gene for the mouse pink-eyed dilution locus and for human type II oculocutaneous albinism. Nature  361: 72–76.  <ul class="find" data-citedArticleID="1028414" data-doi="10.1038/361072a0"><li><a href="http://dx.doi.org/10.1038/361072a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+for+the+mouse+pink-eyed+dilution+locus+and+for+human+type+II+oculocutaneous+albinism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+for+the+mouse+pink-eyed+dilution+locus+and+for+human+type+II+oculocutaneous+albinism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001293-Hanel1" id="pone.0001293-Hanel1"></a>Hanel ML, Wevrick R (2001) Establishment and maintenance of DNA methylation patterns in mouse Ndn: implications for maintenance of imprinting in target genes of the imprinting center. Mol Cell Biol  21: 2384–2392.  <ul class="find" data-citedArticleID="1028380" data-doi="10.1128/mcb.21.7.2384-2392.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.7.2384-2392.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Establishment+and+maintenance+of+DNA+methylation+patterns+in+mouse+Ndn%3A+implications+for+maintenance+of+imprinting+in+target+genes+of+the+imprinting+center." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Establishment+and+maintenance+of+DNA+methylation+patterns+in+mouse+Ndn%3A+implications+for+maintenance+of+imprinting+in+target+genes+of+the+imprinting+center.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001293-Chai1" id="pone.0001293-Chai1"></a>Chai JH, Locke DP, Ohta T, Greally JM, Nicholls RD (2001) Retrotransposed genes such as Frat3 in the mouse Chromosome 7C Prader-Willi syndrome region acquire the imprinted status of their insertion site. Mamm Genome  12: 813–821.  <ul class="find" data-citedArticleID="1028360" data-doi="10.1007/s00335-001-2083-1"><li><a href="http://dx.doi.org/10.1007/s00335-001-2083-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retrotransposed+genes+such+as+Frat3+in+the+mouse+Chromosome+7C+Prader-Willi+syndrome+region+acquire+the+imprinted+status+of+their+insertion+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retrotransposed+genes+such+as+Frat3+in+the+mouse+Chromosome+7C+Prader-Willi+syndrome+region+acquire+the+imprinted+status+of+their+insertion+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001293-Gimelbrant2" id="pone.0001293-Gimelbrant2"></a>Gimelbrant AA, Chess A (2006) An epigenetic state associated with areas of gene duplication. Genome Res  16: 723–729.  <ul class="find" data-citedArticleID="1028370" data-doi="10.1101/gr.5023706"><li><a href="http://dx.doi.org/10.1101/gr.5023706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+epigenetic+state+associated+with+areas+of+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+epigenetic+state+associated+with+areas+of+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001293-Pham1" id="pone.0001293-Pham1"></a>Pham NV, Nguyen MT, Hu JF, Vu TH, Hoffman AR (1998) Dissociation of IGF2 and H19 imprinting in human brain. Brain Res  810: 1–8.  <ul class="find" data-citedArticleID="1028408" data-doi="10.1016/s0006-8993(98)00783-5"><li><a href="http://dx.doi.org/10.1016/s0006-8993(98)00783-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissociation+of+IGF2+and+H19+imprinting+in+human+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissociation+of+IGF2+and+H19+imprinting+in+human+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001293-Albrecht1" id="pone.0001293-Albrecht1"></a>Albrecht U, Sutcliffe JS, Cattanach BM, Beechey CV, Armstrong D, et al.  (1997) Imprinted expression of the murine Angelman syndrome gene, Ube3a, in hippocampal and Purkinje neurons. Nat Genet  17: 75–78.  <ul class="find" data-citedArticleID="1028354" data-doi="10.1038/ng0997-75"><li><a href="http://dx.doi.org/10.1038/ng0997-75" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Imprinted+expression+of+the+murine+Angelman+syndrome+gene%2C+Ube3a%2C+in+hippocampal+and+Purkinje+neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Imprinted+expression+of+the+murine+Angelman+syndrome+gene%2C+Ube3a%2C+in+hippocampal+and+Purkinje+neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001293-Singh1" id="pone.0001293-Singh1"></a>Singh N, Ebrahimi FA, Gimelbrant AA, Ensminger AW, Tackett MR, et al.  (2003) Coordination of the random asynchronous replication of autosomal loci. Nat Genet  33: 339–341.  <ul class="find" data-citedArticleID="1028430" data-doi="10.1038/ng1102"><li><a href="http://dx.doi.org/10.1038/ng1102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coordination+of+the+random+asynchronous+replication+of+autosomal+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coordination+of+the+random+asynchronous+replication+of+autosomal+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001293-Glusman1" id="pone.0001293-Glusman1"></a>Glusman G, Sosinsky A, Ben-Asher E, Avidan N, Sonkin D, et al.  (2000) Sequence, structure, and evolution of a complete human olfactory receptor gene cluster. Genomics  63: 227–245.  <ul class="find" data-citedArticleID="1028372" data-doi="10.1006/geno.1999.6030"><li><a href="http://dx.doi.org/10.1006/geno.1999.6030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence%2C+structure%2C+and+evolution+of+a+complete+human+olfactory+receptor+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence%2C+structure%2C+and+evolution+of+a+complete+human+olfactory+receptor+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001293-Allen1" id="pone.0001293-Allen1"></a>Allen E, Horvath S, Tong F, Kraft P, Spiteri E, et al.  (2003) High concentrations of long interspersed nuclear element sequence distinguish monoallelically expressed genes. Proc Natl Acad Sci U S A  100: 9940–9945.  <ul class="find" data-citedArticleID="1028356" data-doi="10.1073/pnas.1737401100"><li><a href="http://dx.doi.org/10.1073/pnas.1737401100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+concentrations+of+long+interspersed+nuclear+element+sequence+distinguish+monoallelically+expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+concentrations+of+long+interspersed+nuclear+element+sequence+distinguish+monoallelically+expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001293-Rodin1" id="pone.0001293-Rodin1"></a>Rodin SN, Riggs AD (2003) Epigenetic silencing may aid evolution by gene duplication. J Mol Evol  56: 718–729.  <ul class="find" data-citedArticleID="1028420" data-doi="10.1007/s00239-002-2446-6"><li><a href="http://dx.doi.org/10.1007/s00239-002-2446-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+silencing+may+aid+evolution+by+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+silencing+may+aid+evolution+by+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001293-Lynch1" id="pone.0001293-Lynch1"></a>Lynch M, Conery JS (2000) The evolutionary fate and consequences of duplicate genes. Science  290: 1151–1155.  <ul class="find" data-citedArticleID="1028392" data-doi="10.1126/science.290.5494.1151"><li><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+fate+and+consequences+of+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+fate+and+consequences+of+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001293-Rittenhouse1" id="pone.0001293-Rittenhouse1"></a>Rittenhouse E, Dunn LC, Cookingham J, Calo C, Spiegelman M, et al.  (1978) Cartilage matrix deficiency (cmd): a new autosomal recessive lethal mutation in the mouse. Journal of Embryology &amp; Experimental Morphology  43: 71–84.  <ul class="find" data-citedArticleID="1028416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cartilage+matrix+deficiency+%28cmd%29%3A+a+new+autosomal+recessive+lethal+mutation+in+the+mouse.&amp;auth=&amp;atitle=Cartilage+matrix+deficiency+%28cmd%29%3A+a+new+autosomal+recessive+lethal+mutation+in+the+mouse." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cartilage+matrix+deficiency+%28cmd%29%3A+a+new+autosomal+recessive+lethal+mutation+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cartilage+matrix+deficiency+%28cmd%29%3A+a+new+autosomal+recessive+lethal+mutation+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001293-Moncur1" id="pone.0001293-Moncur1"></a>Moncur JT, Park JP, Memoli VA, Mohandas TK, Kinlaw WB (1998) The “Spot 14” gene resides on the telomeric end of the 11q13 amplicon and is expressed in lipogenic breast cancers: implications for control of tumor metabolism. Proc Natl Acad Sci U S A  95: 6989–6994.  <ul class="find" data-citedArticleID="1028400" data-doi="10.1073/pnas.95.12.6989"><li><a href="http://dx.doi.org/10.1073/pnas.95.12.6989" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+%E2%80%9CSpot+14%E2%80%9D+gene+resides+on+the+telomeric+end+of+the+11q13+amplicon+and+is+expressed+in+lipogenic+breast+cancers%3A+implications+for+control+of+tumor+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+%E2%80%9CSpot+14%E2%80%9D+gene+resides+on+the+telomeric+end+of+the+11q13+amplicon+and+is+expressed+in+lipogenic+breast+cancers%3A+implications+for+control+of+tumor+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001293-VanDenDiepstraten1" id="pone.0001293-VanDenDiepstraten1"></a>Van Den Diepstraten C, Papay K, Bolender Z, Brown A, Pickering JG (2003) Cloning of a novel prolyl 4-hydroxylase subunit expressed in the fibrous cap of human atherosclerotic plaque. Circulation  108: 508–511.  <ul class="find" data-citedArticleID="1028436" data-doi="10.1161/01.cir.0000080883.53863.5c"><li><a href="http://dx.doi.org/10.1161/01.cir.0000080883.53863.5c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+a+novel+prolyl+4-hydroxylase+subunit+expressed+in+the+fibrous+cap+of+human+atherosclerotic+plaque." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+a+novel+prolyl+4-hydroxylase+subunit+expressed+in+the+fibrous+cap+of+human+atherosclerotic+plaque.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001293-Consortium1" id="pone.0001293-Consortium1"></a>Consortium MGS (2002) Initial sequencing and comparative analysis of the mouse genome. Nature  420: 520–562.  <ul class="find" data-citedArticleID="1028364"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+comparative+analysis+of+the+mouse+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+comparative+analysis+of+the+mouse+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+comparative+analysis+of+the+mouse+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+comparative+analysis+of+the+mouse+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001293-Karolchik1" id="pone.0001293-Karolchik1"></a>Karolchik D, Hinrichs AS, Furey TS, Roskin KM, Sugnet CW, et al.  (2004) The UCSC Table Browser data retrieval tool. Nucleic Acids Res  32: D493–496.  <ul class="find" data-citedArticleID="1028386" data-doi="10.1093/nar/gkh103"><li><a href="http://dx.doi.org/10.1093/nar/gkh103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Table+Browser+data+retrieval+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Table+Browser+data+retrieval+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.XML" value="89314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.PDF" value="281370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g001.PNG_L" value="313194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g001.PNG_M" value="61878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g001.PNG_S" value="10192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g001.TIF" value="620814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g001.PNG_I" value="21148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g002.PNG_L" value="59998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g002.PNG_M" value="17425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g002.PNG_S" value="6822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g002.TIF" value="133518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g002.PNG_I" value="6029"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g003.PNG_L" value="636892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g003.PNG_M" value="83051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g003.PNG_S" value="11105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g003.TIF" value="1092952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g003.PNG_I" value="26990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g004.PNG_L" value="2109711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g004.PNG_M" value="241594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g004.PNG_S" value="15900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g004.TIF" value="3729944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g004.PNG_I" value="93501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g005.PNG_L" value="417225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g005.PNG_M" value="59674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g005.PNG_S" value="6030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g005.TIF" value="339402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g005.PNG_I" value="20453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g006.PNG_L" value="154721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g006.PNG_M" value="53531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g006.PNG_S" value="11793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g006.TIF" value="510706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.g006.PNG_I" value="23762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s001.PDF" value="374122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s002.PDF" value="40194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s003.PDF" value="1309070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s004.PDF" value="45956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s005.PDF" value="32530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s006.PDF" value="90988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s007.DOC" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001293.s008.DOC" value="37888"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001293&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001293" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001293&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001293&volume=&issue=&title=Monoallelic Expression of Multiple Genes in the CNS&author_name=Jinhui%20Wang%2C%20Zuzana%20Valo%2C%20David%20Smith%2C%20Judith%20Singer-Sam&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001293" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001293" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001293" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001293&amp;t=Monoallelic%20Expression%20of%20Multiple%20Genes%20in%20the%20CNS" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001293&title=Monoallelic%20Expression%20of%20Multiple%20Genes%20in%20the%20CNS&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001293&amp;title=Monoallelic%20Expression%20of%20Multiple%20Genes%20in%20the%20CNS" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001293" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001293&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Monoallelic Expression of Multiple Genes in the CNS';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001293';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Monoallelic%20Expression%20of%20Multiple%20Genes%20in%20the%20CNS http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001293" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001293" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Central+nervous+system%22" title="Search for articles in the subject area:'Central nervous system'"><div class="flagText">Central nervous system</div></a>
              <div data-categoryid="47337" data-articleid="25418"
                   data-categoryname="Central nervous system"
                   class="flagImage" title="Flag 'Central nervous system' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="25418"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="25418"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25418"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25418"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25418"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+sequencing%22" title="Search for articles in the subject area:'RNA sequencing'"><div class="flagText">RNA sequencing</div></a>
              <div data-categoryid="19765" data-articleid="25418"
                   data-categoryname="RNA sequencing"
                   class="flagImage" title="Flag 'RNA sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="25418"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=166'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3771'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6654&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=17721">Publisher's Note: Correction to Cell Line Derivation</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
