

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001508"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001508" />

    <meta name="citation_title" content="Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology"/>
    <meta itemprop="name" content="Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology"/>

      <meta name="citation_author" content="Paolo Monaci"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Alessandra Luzzago"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Claudia Santini"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Alessandra De Pra"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Mirko Arcuri"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Francesca Magistri"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Alessandro Bellini"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Helenia Ansuini"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Maria Ambrosio"/>
            <meta name="citation_author_institution" content="Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy"/>
      <meta name="citation_author" content="Virginia Ammendola"/>
            <meta name="citation_author_institution" content="Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy"/>
      <meta name="citation_author" content="Maria Giulia Bigotti"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
            <meta name="citation_author_institution" content="Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy"/>
      <meta name="citation_author" content="Agostino Cirillo"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Maurizio Nuzzo"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Annamaria Assunta Nasti"/>
            <meta name="citation_author_institution" content="Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy"/>
      <meta name="citation_author" content="Philippe Neuner"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Laura Orsatti"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Monica Pezzanera"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Andrea Sbardellati"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Giuseppe Silvestre"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
            <meta name="citation_author_institution" content="Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy"/>
      <meta name="citation_author" content="Paolo Uva"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Valentina Viti"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Gaetano Barbato"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Stefano Colloca"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Anna Demartis"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Emanuele De Rinaldis"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Saverio Giampaoli"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Armin Lahm"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Fabio Palombo"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Fabio Talamo"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Alessandra Vitelli"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Alfredo Nicosia"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>
      <meta name="citation_author" content="Riccardo Cortese"/>
            <meta name="citation_author_institution" content="Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy"/>

    <meta name="citation_date" content="2008/1/30"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001508.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1508"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Synthetic antibodies as therapeutics.; citation_author=G Fuh; citation_journal_title=Expert Opin Biol Ther; citation_volume=7; citation_number=1; citation_pages=73-87; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Therapeutic antibodies for human diseases at the dawn of the twenty-first century.; citation_author=OH Brekke; citation_author=I Sandlie; citation_journal_title=Nat Rev Drug Discov; citation_volume=2; citation_number=2; citation_pages=52-62; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparing antibody and small-molecule therapies for cancer.; citation_author=K Imai; citation_author=A Takaoka; citation_journal_title=Nat Rev Cancer; citation_volume=6; citation_number=3; citation_pages=714-727; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Monoclonal antibody therapy of cancer.; citation_author=GP Adams; citation_author=LM Weiner; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=4; citation_pages=1147-1157; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human antibodies with sub-nanomolar affinities isolated from a large non-immunized phage display library.; citation_author=TJ Vaughan; citation_author=AJ Williams; citation_author=K Pritchard; citation_author=JK Osbourn; citation_author=AR Pope; citation_journal_title=Nat Biotechnol; citation_volume=14; citation_number=5; citation_pages=309-314; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Structural repertoire of the human VH segments.; citation_author=C Chothia; citation_author=AM Lesk; citation_author=E Gherardi; citation_author=IM Tomlinson; citation_author=G Walter; citation_journal_title=J Mol Biol; citation_volume=227; citation_number=6; citation_pages=799-817; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Conformations of immunoglobulin hypervariable regions.; citation_author=C Chothia; citation_author=AM Lesk; citation_author=A Tramontano; citation_author=M Levitt; citation_author=SJ Smith-Gill; citation_journal_title=Nature; citation_volume=342; citation_number=7; citation_pages=877-883; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Diversity in the CDR3 region of V(H) is sufficient for most antibody specificities.; citation_author=JL Xu; citation_author=MM Davis; citation_journal_title=Immunity; citation_volume=13; citation_number=8; citation_pages=37-45; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human neoplasms elicit multiple specific immune responses in the autologous host.; citation_author=U Sahin; citation_author=O Tureci; citation_author=H Schmitt; citation_author=B Cochlovius; citation_author=T Johannes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=9; citation_pages=11810-11813; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Transferrin: structure, function and potential therapeutic actions.; citation_author=PT Gomme; citation_author=KB McCann; citation_author=J Bertolini; citation_journal_title=Drug Discov Today; citation_volume=10; citation_number=10; citation_pages=267-273; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phase Ia trial of murine immunoglobulin A antitransferrin receptor antibody 42/6.; citation_author=D Brooks; citation_author=C Taylor; citation_author=B Dos Santos; citation_author=H Linden; citation_author=A Houghton; citation_journal_title=Clin Cancer Res; citation_volume=1; citation_number=11; citation_pages=1259-1265; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Technology evaluation: TransMID, KS Biomedix/Nycomed/Sosei/PharmaEngine.; citation_author=NG Rainov; citation_author=A Soling; citation_journal_title=Curr Opin Mol Ther; citation_volume=7; citation_number=12; citation_pages=483-492; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A neutralizing monoclonal antibody (mAb A24) directed against the transferrin receptor induces apoptosis of tumor T lymphocytes from ATL patients.; citation_author=IC Moura; citation_author=Y Lepelletier; citation_author=B Arnulf; citation_author=P England; citation_author=C Baude; citation_journal_title=Blood; citation_volume=103; citation_number=13; citation_pages=1838-1845; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A role for transferrin receptor in triggering apoptosis when targeted with gambogic acid.; citation_author=S Kasibhatla; citation_author=KA Jessen; citation_author=S Maliartchouk; citation_author=JY Wang; citation_author=NM English; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=14; citation_pages=12095-12100; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation.; citation_author=P Lu; citation_author=C Vogel; citation_author=R Wang; citation_author=X Yao; citation_author=EM Marcotte; citation_journal_title=Nat Biotechnol; citation_volume=25; citation_number=15; citation_pages=117-124; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A combined transmembrane topology and signal peptide prediction method.; citation_author=L Kall; citation_author=A Krogh; citation_author=EL Sonnhammer; citation_journal_title=J Mol Biol; citation_volume=338; citation_number=16; citation_pages=1027-1036; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A comprehensive transcript index of the human genome generated using microarrays and computational approaches.; citation_author=EE Schadt; citation_author=SW Edwards; citation_author=D GuhaThakurta; citation_author=D Holder; citation_author=L Ying; citation_journal_title=Genome Biol; citation_volume=5; citation_number=17; citation_pages=R73; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A model system for optimising the selection of membrane antigen-specific human antibodies on intact cells using phage antibody display technology.; citation_author=JP Hegmans; citation_author=K Radosevic; citation_author=JS Voerman; citation_author=JA Burgers; citation_author=HC Hoogsteden; citation_journal_title=J Immunol Methods; citation_volume=262; citation_number=18; citation_pages=191-204; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Secreted and cell surface genes expressed in benign and malignant colorectal tumors.; citation_author=P Buckhaults; citation_author=C Rago; citation_author=B St Croix; citation_author=KE Romans; citation_author=S Saha; citation_journal_title=Cancer Res; citation_volume=61; citation_number=19; citation_pages=6996-7001; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels.; citation_author=A Shevchenko; citation_author=M Wilm; citation_author=O Vorm; citation_author=M Mann; citation_journal_title=Anal Chem; citation_volume=68; citation_number=20; citation_pages=850-858; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001508" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology"/>
    <meta name="twitter:description" content="A novel and efficient tagArray technology was developed that allows rapid identification of antibodies which bind to receptors with a specific expression profile, in the absence of biological information. This method is based on the cloning of a specific, short nucleotide sequence (tag) in the phagemid coding for each phage-displayed antibody fragment (phage-Ab) present in a library. In order to set up and validate the method we identified about 10,000 different phage-Abs binding to receptors expressed in their native form on the cell surface (10 k Membranome collection) and tagged each individual phage-Ab. The frequency of each phage-Ab in a given population can at this point be inferred by measuring the frequency of its associated tag sequence through standard DNA hybridization methods. Using tiny amounts of biological samples we identified phage-Abs binding to receptors preferentially expressed on primary tumor cells rather than on cells obtained from matched normal tissues. These antibodies inhibited cell proliferation in vitro and tumor development in vivo, thus representing therapeutic lead candidates."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001508.g006"/>

  <meta property="og:title" content="Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001508" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7597'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5148'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4836&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001508">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001508" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001508&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paolo Monaci
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Paolo Monaci, Alessandra Luzzago</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:paolo_monaci@merck.com">paolo_monaci@merck.com</a></p>

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alessandra Luzzago
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Paolo Monaci, Alessandra Luzzago</p>

              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claudia Santini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alessandra De Pra, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mirko Arcuri, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Francesca Magistri, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alessandro Bellini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Helenia Ansuini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria Ambrosio, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Virginia Ammendola, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria Giulia Bigotti, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy, 
                  Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Agostino Cirillo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maurizio Nuzzo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Annamaria Assunta Nasti, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philippe Neuner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura Orsatti, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Monica Pezzanera, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrea Sbardellati, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Giuseppe Silvestre, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy, 
                  Centro di Ricerca per l'Ingegneria Genetica (CEINGE), Napoli, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paolo Uva, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Valentina Viti, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gaetano Barbato, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stefano Colloca, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>

                <p>
                    Current address: Okairòs, Pomezia, Rome, Italy
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anna Demartis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Emanuele De Rinaldis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Saverio Giampaoli, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Armin Lahm, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fabio Palombo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fabio Talamo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alessandra Vitelli, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alfredo Nicosia, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>

                <p>
                    Current address: Okairòs, Pomezia, Rome, Italy
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Riccardo Cortese
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biotechnology Department, Istituto di Ricerca di Biologia Molecolare (IRBM) P. Angeletti, Pomezia, Rome, Italy
                </p>

                <p>
                    Current address: Okairòs, Pomezia, Rome, Italy
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 30, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001508</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001508" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001508" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001508" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001508" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001508-g001" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001508-g002" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001508-g003" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001508-g004" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001508-g005" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001508-g006" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001508">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001508" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>A novel and efficient <em>tagArray</em> technology was developed that allows rapid identification of antibodies which bind to receptors with a specific expression profile, in the absence of biological information. This method is based on the cloning of a specific, short nucleotide sequence (<em>tag</em>) in the phagemid coding for each phage-displayed antibody fragment (phage-Ab) present in a library. In order to set up and validate the method we identified about 10,000 different phage-Abs binding to receptors expressed in their native form on the cell surface (10 k <em>Membranome</em> collection) and tagged each individual phage-Ab. The frequency of each phage-Ab in a given population can at this point be inferred by measuring the frequency of its associated tag sequence through standard DNA hybridization methods. Using tiny amounts of biological samples we identified phage-Abs binding to receptors preferentially expressed on primary tumor cells rather than on cells obtained from matched normal tissues. These antibodies inhibited cell proliferation <em>in vitro</em> and tumor development <em>in vivo</em>, thus representing therapeutic lead candidates.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Monaci P, Luzzago A, Santini C, De Pra A, Arcuri M, et al.  (2008) Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology. PLoS ONE 3(1):
          e1508.
            doi:10.1371/journal.pone.0001508</p><p><strong>Academic Editor: </strong>Simon Williams, Texas Tech University Health Sciences Center, United States of America</p><p><strong>Received:</strong> November 14, 2007; <strong>Accepted:</strong> December 22, 2007; <strong>Published:</strong> January 30, 2008</p><p><strong>Copyright:</strong> © 2008 Monaci et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In recent years monoclonal antibodies (mAbs) have proven to be excellent therapeutic agents <a href="#pone.0001508-Fuh1">[1]</a>, <a href="#pone.0001508-Brekke1">[2]</a>. They have long half-life, favorable pharmaco-kinetics in humans, none or very few adverse reactions and a well established industry-scale production process <a href="#pone.0001508-Fuh1">[1]</a>, <a href="#pone.0001508-Imai1">[3]</a>, <a href="#pone.0001508-Adams1">[4]</a>. But their most significant feature is the ability to specifically bind very diverse molecules with high affinity. Among these, the receptors expressed on the surface of the eukaryotic cell are particularly important as therapeutic targets. These receptors mediate the response of the cell to environmental stimuli, and thus it is not surprising that they play a key role in a large number of diseases, including cancers, infections and auto-immune diseases (e.g. Crohn's disease, rheumatoid arthritis, asthma).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Therapeutic mAbs are generally selected following the identification of a suitable target. Often, key information is provided by large scale transcriptional studies which identify genes differentially expressed in cells under normal and pathological conditions. However this approach omits several potentially interesting targets, such as those whose protein expression level does not vary in tandem with the corresponding mRNA level, or those whose structure/epitope is dependant on a post-translational protein modification, or even those cases in which protein-protein interactions generate novel, pathologically related epitopes. We therefore devised a highly sensitive method to survey the differential binding of a large number of clones which can be adapted to the very small scale of tissues biopsies. This strategy allows the identification of epitopes with a specific expression profile (e.g., tumor-specific), independently of any biological information. The method is based on i) the availability of a defined collection of phage-Abs binding to the epitopes of membrane proteins (i.e. the <em>Membranome</em> collection) and ii) the possibility of tagging every phage-Ab with a specific DNA tag sequence.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p><em>In silico</em> analysis of the human genome predicts the total size of the membrane protein (MP) coding genes-the <em>Membranome</em>–to be less than 5,000. Thus, by using high throughput screening methods and phage-displayed libraries, it is feasible to put together a collection containing antibodies that bind to all or most MPs. Here we report the creation of just such a large and diverse collection of Ab fragments binding MPs, generated by panning phage-displayed libraries of single-chain antibodies (scFv) on whole cells. Furthermore, we inserted a unique oligonucleotide tag sequence in each phagemid of the <em>Membranome</em> collection, and generated two populations by panning the tagged collection on tumor and on normal tissue. Tumor-specific phage-Abs were identified by comparing the frequency of each tag between the two populations. This was quickly achieved by hybridizing the tag populations to a customized tag-array microchip. The tumor-specific phage-Abs can be used to identify their target and, after conversion into human IgGs, for the phenotypic analysis of their biological properties.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Strategy for generating a collection of phage-Ab binding <em>Membranome</em> proteins</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We define as <em>Membranome</em> the ensemble of human genes coding for proteins associated to the cell membrane. <em>In silico</em> analysis identified about 4,600 genes populating the human <em>Membranome</em> (approx 20% of the human genome; see <a href="#s4">Materials and methods</a>). We built up a collection of phage-Ab that bind to the <em>Membranome</em> proteins by high-throughput <em>in vitro</em> screening of naïve phage-Ab libraries, using various cell lines as “selectors”. MPs expressed on the cell surface maintain their native features: folding, post-translation modification, expression of splicing variants and formation of multimeric complexes. In addition, cultured cell lines are reagents readily available with reproducible properties. We analyzed <em>in silico</em> the variation in expression levels of MP-encoding genes in a set of 25 cell lines (see <a href="#pone.0001508.s001">Table S1</a>). For each cell line we ranked the MP-encoding genes according to their level of gene expression. We considered genes expressed in a given cell line when included in the top ranking expression level of 200, 300 or 400. This analysis reveals that about 30% of the MP-encoding genes are expressed in only one of the 25 cell lines, whereas only a small fraction (around 2–3%) are expressed in all the cell lines. These data provide an estimate of the diversity of expression of the MP- encoding genes across different cell lines. They also indicate that increasing the number of cell lines generates a set expressing as a whole an increasing number and eventually all the MP-encoding genes.</p>


<h4>Building up the <em>Membranome</em> phage-Ab repository</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The workflow adopted to build up the <em>Membranome</em> phage-Ab collection is schematically described in <a href="#pone-0001508-g001">Figure 1</a>. We used three naïve phage-Ab libraries derived from B cells of human healthy donors which include over 10<sup>10</sup> different clones each. <a href="#pone.0001508-Vaughan1">[5]</a>. These libraries were panned separately on 64 different human cell lines derived from 25 different tissues (see <a href="#pone.0001508.s002">Table S2</a>). We performed only two rounds of panning to preserve the complexity of the selected population of phage and limit the preferential amplification of the most biologically viable clones. Phage-Abs from each selected pool were individually screened by cell-ELISA for their ability to bind the selector cell line. Positive clones with a novel VH-CDR3 sequence were progressively added to the <em>Membranome</em> phage-Ab collection. Automation of this process enabled high throughput screening, and more than 112,000 clones were analyzed by cell ELISA. Among these, about 40,000 clones (41%) were confirmed positive and found to include 9,925 different VH-CDR3 sequences (8.8%). This set of 9,925 clones is referred to as the 10k <em>Membranome</em> phage-Ab collection. On average, 290 unique phage clones were selected from each cell line and over 53% of these had a new VH-CD3 sequence. The frequency (and also the number) of the new phage-Abs clones derived from each cell line is steady, and not affected by the order in which each cell line was used for selection. This strongly suggests that this process, in addition to efficiently selecting a core of antibodies that recognize common surface proteins, also selects antibodies against target antigens over-expressed in a given cell line, thus enhancing the repertoire of binding specificities included in the collection.</p>
<div class="figure" id="pone-0001508-g001"><div class="img"><a name="pone-0001508-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Strategy adopted to generate the 10k <em>Membranome</em> phage-Ab collection</span></strong></p><span>doi:10.1371/journal.pone.0001508.g001</span></div>

<h4>Use of the <em>Membranome</em> phage-Ab collection</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The 10 k <em>Membranome</em> phage-Ab collection is a “specialized”, low-complexity library of clones which bind receptors expressed in their native form on the cell surface. This collection can be surveyed to rapidly identify antibodies binding epitopes of known cell-surface receptors. The most attractive use of the <em>Membranome</em> collection, however, is to select antibodies targeting epitopes of unknown receptors with specific expression features, which is the case of the tumor-associated receptors specifically over-expressed in tumor. Identifying such antibodies, however, is a goal barred by the lack of an efficient protocol for differential screening. Let us consider the case of a target and a reference biological sample. Panning the <em>Membranome</em> phage-Ab collection on each of these samples generated two phage populations. The goal is to identify clones preferentially or exclusively present in one of the two populations. Testing the binding specificity of individual clones from these populations would require large quantities of biological samples, which generally are available only in limited quantities, and would survey only a minor part of the population.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We addressed the problem from a different perspective. Sequencing a number of clones from the test and the reference population of phage-Ab could reveal clones which are differentially represented in the two pools, but clearly, a statistically significant comparison of the two populations would require a very large number of sequences. To efficiently acquire this information we developed a novel DNA-based screening technology we called <em>tagArray</em>. First we designed a set of short oligonucleotides that could be used as tags and cloned each single tag into each phagemid coding for a phage-Ab (see <a href="#s4">Materials and methods</a> for details). In this way we generated an association between a specific tag and a specific scFv sequence, which makes it possible to measure the frequency of each scFv clone in a population by measuring frequency of the associated tag. Moreover, standard phage-binding screening protocols can be replaced by well-established DNA amplification and hybridization methods and high-throughput technologies, such as DNA microarray. The tagArray technology enables the entire populations to be compared in an efficient and sensitive way, using minimal amounts of selector samples. The screening protocol is organized in four steps: i) the tagged phage population is panned in parallel on a biological and a reference sample generating two distinct phage populations; ii) tag sequences are amplified from the target and reference phage populations, labeled with different fluorochromes and hybridized on a double channel DNA microarray containing the complete repertoire of tag sequences; iii) analysis of the hybridization data identifies the tags differentially represented in the two populations; iv) rescue of the scFv sequence associated to the relative tags are rescued from the phage population (see <a href="#s4">Materials and methods</a>).</p>


<h4>Identification of phage-Abs that bind tumor cells by <em>tagArray</em> screening</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>During the process of collecting the 10,000 phage-Ab of the 10 k-Membranome collection, we generated a preliminary version composed of the first 4,000 clones that were identified (4k-Membranome phage-Ab collection). A set of tag sequences was inserted in the phagemids of the 4k-<em>Membranome</em> collection, as detailed in <a href="#s4">Materials and methods</a>. Biopsy specimens from tumor and adjacent normal tissues were collected from two colorectal cancer (CRC) patients (pt#78 and pt#80), samples were disaggregated by enzymatic treatment and the epithelial cells affinity purified. The tagged 4 k <em>Membranome</em> collection was panned on these primary epithelial cells to generate two pairs of tumor (T) and normal (N) phage populations (T<sub>78</sub> and N<sub>78</sub>; T<sub>80</sub> and N<sub>80</sub>; <a href="#pone-0001508-g002">Figure 2</a>). Tag sequences from each T and N matched population were amplified, labeled with different fluorochromes, mixed and hybridized to a DNA microarray containing the complete repertoire of tag sequences. The intensity of the two fluorochromes was measured for each spot of the microarray and analyzed as average intensity and ratio from two independent fluor-reversal experiments. The former reflects the average abundance whereas the latter reveals the differential frequency of each tag in the matched populations. Candidate tags were selected on the basis of statistical criteria (see <a href="#s4">Materials and methods</a>) limiting our analysis to tags whose average intensity was higher than log<sub>2</sub>10. Since we aimed at identifying clones specifically enriched in the T as compared to the matched N phage population, we focused our attention on tags exhibiting the highest T/N ratio in both patients (average T/N ratio higher than 70% of the maximal ratio detected at the same intensity). This process identified a set of 174 tags among which we randomly chose a subset of 61 elements. By utilizing a nested PCR amplification protocol employing primers encompassing the tag sequence, we recovered 45 different phage-Abs associated with the selected tags from the T populations. This partial redundancy in the phage-Abs identified was expected since, under the experimental conditions adopted, a phage-Ab can be associated to more than one tag.</p>
<div class="figure" id="pone-0001508-g002"><div class="img"><a name="pone-0001508-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Workflow of the <em>tagArray</em> screening procedure employed to identify tumor-specific phage-Abs.</span></strong></p><span>doi:10.1371/journal.pone.0001508.g002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Flow cytometry was used to measure the binding of three phage-Abs to the primary cells from tumor and normal tissue obtained from pt#78 and pt#80. As reported in <a href="#pone-0001508-g003">Figure 3a</a>, all three phage-Abs differentially bound tumor cells, whereas the matched normal cells and clone HL60-D3-232 retained its binding specificity when formatted as IgG (Mbr-4; <a href="#pone-0001508-g003">Figure 3b</a>). It is worth noting that the binding hierarchy measured by flow cytometry matches that obtained by <em>tagArray</em> analysis.</p>
<div class="figure" id="pone-0001508-g003"><div class="img"><a name="pone-0001508-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Binding specificity of selected clones.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Binding of phage-Ab HL60-D3-232, HeLa-D2-20, HL60-C3-16 and a control phage-Ab (a) or of Mbr-4 and a control isotypic IgG (Mbr-ctrl; b) to human epithelial primary cells was measured by flow cytometry. Primary cells were obtained from matched tumor and adjacent normal tissue from CRC patients #78 (white and dark grey bars, respectively) and #80 (pale grey and black bars, respectively). Data are reported as percentage of gated cells. In panel a, numbers on top of tumor sample bars refer to the difference of percentage between that tumor and the matched normal sample (top line) and to the T/N ratio detected by <em>tagArray</em> analysis for the corresponding tags (bottom line).</p>
<span>doi:10.1371/journal.pone.0001508.g003</span></div>

<h4>Identification of target receptors</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The phage-borne scFv identified as tumor-specific by tagArray analysis were converted into human IgG1 format. Vectors expressing the heavy and the light chains were co-transfected in 293-EBNA cells and the secreted IgGs were affinity purified from the medium and immobilized on beads. Identification of antigens targeted by these monoclonal antibodies (from now on referred to as Mbr-#) followed the strategy illustrated in <a href="#pone-0001508-g004">Figure 4</a>. Membrane proteins on living cells were selectively labeled with fluorescent dye and solubilized in non-denaturing buffer. The proteins immune-purified from this mixture by mAb-coated beads were then analyzed by 1D-PAGE. Bands generated by specific binding of MPs were identified by dual-wavelength scan, excised from the gel and digested for Mass Spectrometry protein identification (see <a href="#s4">Materials and methods</a> for details). This biochemical approach identified the transferrin receptor (TfRC) as the target of five different mAbs (Mbr-2, 4, 43, 112 and 114). These results were confirmed by detecting a specific binding of the same mAbs to CHO cell transfected with a TfRC expression plasmid (data not shown).</p>
<div class="figure" id="pone-0001508-g004"><div class="img"><a name="pone-0001508-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Schematic outline of the biochemical strategy adopted for the identification of the target antigens.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Membrane proteins on living cells are selectively labeled with fluorescent dyes (i), then isolated in native conformation and in soluble form by extraction with appropriate buffers (ii). The target antigen is pulled down using the beads-immobilized IgG of interest (iii). Immuno-purified material is analyzed by 1D-PAGE and membrane proteins discriminated from non-specific binders by dual-wavelength scan of the gel (iv). Positive band is excised and processed for MS-based protein identification.</p>
<span>doi:10.1371/journal.pone.0001508.g004</span></div>

<h4>Biological activity of selected IgGs</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We characterized the binding of the mAbs to TfRC by surface plasmon resonance. This analysis revealed K<sub>D</sub>s in the 5–140 nM range with k-off values ranging from 2 to 40 msec<sup>−1</sup> for the binding of IgGs to the soluble TfRC extra-cellular domain (see <a href="#pone.0001508.s003">Table S3</a>). Apparent K<sub>D</sub>s between 3 and 12 nM on HCT-116 cells were instead obtained using cell-ELISA. A molar excess of the natural ligand transferrin inhibited binding of the five mAbs to TfRC (data not shown).</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Mbr-2, Mbr-4 and Mbr-114 inhibited cell proliferation <em>in vitro</em> in 6 cell lines at the concentration of 100 µg/mL (<a href="#pone-0001508-g005">Figure 5a</a>). Mbr-4, the clone exhibiting the highest binding affinity for its target receptor, was further characterized and shown to effectively promote apoptosis of HCT-116 cells at concentrations ranging from 0.8 to 20 µg/L (<a href="#pone-0001508-g005">Figure 5b</a>). Additional experiments indicated that anti-proliferative and pro-apoptotic activities are exerted by competition with transferrin for binding to TfRC (data not shown).</p>
<div class="figure" id="pone-0001508-g005"><div class="img"><a name="pone-0001508-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>In vitro characterization of anti-TfRC Mbr-4.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(a) The indicated cell lines were incubated for 72 hrs at 37°C with 100 µg/mL of Mbr-2, Mbr-4, Mbr-114 or an isotypic control. Cell viability is reported as percentage of the isotopic control treated cells. (b) HCT-116 cells were incubated for 48, 72 and 96 hrs with 0.8 (•), 4 (▪) and 20 (▴) µg/mL of Mbr-4 or with an isotypic control (○). Cell apoptosis is reported as percentage of Annexin V-, 7AAD-positive cells.</p>
<span>doi:10.1371/journal.pone.0001508.g005</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Mbr-4 <em>in vivo</em> anti-tumor activity was assessed by xenograft experiment (<a href="#pone-0001508-g006">Figure 6a and 6b</a>). HCT-116 cells were injected subcutaneously in BALB/c nude mice, immediately followed by an intra peritoneal injection of Mbr-4 or an isotypic control. Both mAbs were administered 3 times per week for 21 days and the percentage of tumor-free mice and tumor growth monitored: The former dramatically differed between the HCT-116 and BALB/c groups. At day 11 measurable tumors were detected in all control mice (n = 12) whereas only 1 tumor was observed in the group treated with Mbr-4 (n = 12; p&lt;3×10<sup>−5</sup>). On treatment end at day 21, only 2 mice had a tumor in the Mbr-4 treated group. A significant difference between the two groups was detected until day 64 when the experiment was stopped. In line with these results, tumor volume was also lower in the Mbr-4 treated group of mice than in controls.</p>
<div class="figure" id="pone-0001508-g006"><div class="img"><a name="pone-0001508-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001508" data-uri="info:doi/10.1371/journal.pone.0001508.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001508.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001508.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001508.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001508.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>In vivo anti-tumoral activity of anti-TfRC Mbr-4.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>BALB/c nude mice were challenged with a subcutaneous injection of HCT-116 cells at day = 0. A first group of mice (filled triangles; n = 12) was administered with 10 i.p. injections of 0.5 mg IgG-4 three times a week starting from day = 0. Mice from this group were followed up to day = 64. A second group of mice (empty circles, n = 12) was administered injections of control isotypic IgG according to the same protocol and followed up to day = 35, when mice were sacrificed for ethical reasons. Both groups were monitored for tumor growth expressed as percentage of free mice (panel a) and tumor volume (b). Error bars in panel b indicate standard error of the mean (S.E.M).</p>
<span>doi:10.1371/journal.pone.0001508.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In the majority of cases the identification of a therapeutic target necessarily precedes the selection of a therapeutic mAb, whereas we demonstrate that this “therapeutic target to mAb” approach can be reversed. Here we describe a strategy which enables the identification of antibodies that bind unknown receptors with a specific expression profile. The first key stage of this strategy is the selection of a relatively low-complexity, “specialized” library containing phage-Abs that bind to MPs (the <em>Membranome</em> collection). The second critical element is the development of a technology that enables an efficient differential screening of phage clones, using minimal amounts of biological samples.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We generated the <em>Membranome</em> collection by panning a phage-Ab library on cultured cells This process simultaneously selects ligands for several cell surface receptors. The complexity of the library, the display level of the target receptor on the cell surface and the affinity of the interaction between phage-Ab and the target receptor strongly affect this process. All these factors interact in a competitive environment where biological viability of the phage-Ab also plays a major role. Empirically, however, we observed that on average the number of new clones identified by panning on each different cell line remains relatively constant, indicating that we are still far from saturation of the library sequence diversity.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The amino acid sequence of VH3-CDR3, the hypervariable region which in most cases acts as a major binding determinant <a href="#pone.0001508-Chothia1">[6]</a>–<a href="#pone.0001508-Xu1">[8]</a>, is a unique and distinctive feature of each member of the repository. The <em>Membranome</em> phage-Ab collection hosts VH-CDR3 sequences ranging from 3 to 25 residues, with the frequency peaking around 10 residues, and analysis of the complete scFv region reveals a broad set of germline families. Together these two findings indicate that a wide variety of binding motifs are present in the collection. We estimate that the current version of the repository (10,000 clones) should target between 500 and 1,000 different antigens, corresponding to 10–20% of the complete <em>Membranome</em>. We also attempted to evaluate the diversity of antigens targeted by the phage-Ab repository through two independent approaches (see <a href="#pone.0001508.s004">Text S1</a>), and believe that a conservative figure combining both approaches could be anything between 10 and 20 clones per target. It is worth noting that the 18 target receptors identified so far are membrane proteins with one or two trans-membrane domains and large (&gt;700aa) extra-cellular domains, along with several GPI-anchored membrane proteins.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The tagArray technology efficiently surveys the <em>Membranome</em> collection and singles out clones which recognize receptor molecules differentially expressed between two complex biological systems of interest, achieved by coupling phage-display to high throughput oligonucleotide microarray screening. To illustrate a practical application of the <em>Membranome</em> phage-Ab collection, we describe the identification of tumor-specific mAbs. An early version of the collection which included 4,000 members was surveyed for clones specifically binding epithelial cells from primary colorectal tumors. This screening revealed that for each patient, several phage-Abs specifically recognized primary cells from tumor versus matched normal tissue although different patients shared only a small fraction of these phage-Abs. A similar finding has been reported for the screening of cDNA expression clones from tumors using autologous or heterologous sera <a href="#pone.0001508-Sahin1">[9]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We focused our attention on the phage-Ab that specifically reacted with tumor cells from two patients. Phage-Abs are good binders and their interaction with the target molecule is efficiently detected through the phage moiety. However, to perform functional assays and to identify the target Ag, we converted the selected phage-Ab into human IgG1. Using biochemical methods combined with mass spectroscopy, TfRC, the main iron transporter which binds and internalizes the iron carrier transferrin, was identified as the target for six of these tumor specific mAbs. Fast-paced tumor replication requires high levels of iron uptake, thus TfRC is often over-expressed in breast, gastric, colon cancer and glioblastoma <a href="#pone.0001508-Gomme1">[10]</a> and has therefore prompted the use of TRC as a target for anti-cancer therapy <a href="#pone.0001508-Brooks1">[11]</a>. Some examples are in a phase II clinical trial, transferrin fused to tetanus toxoid protein showed 30% of responders in glioblastoma patients, leading to the approval of a phase III trial <a href="#pone.0001508-Rainov1">[12]</a>. In another study, a mouse monoclonal was shown to induce apoptosis only in T lymphocytes of acute T-cell leukemia patients, but not in PBMC from healthy donors <a href="#pone.0001508-Moura1">[13]</a> and in line with this finding, an anti-TfRC IgA showed clinical responses in lymphoma patients. More recently, induction of apoptosis trough TfRC was reported using Gambogic acid on a panel of cell lines <a href="#pone.0001508-Kasibhatla1">[14]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>tagArray technology is a versatile tool for profiling protein expression on any cell surface. An increasing number of reports show that, although closely related, transcriptional and proteomic data do not always coincide <a href="#pone.0001508-Lu1">[15]</a>. Transcriptional profiling has a further drawback of not detecting post-translational modifications or variations in protein stability which also drive important biological processes. Capitalizing on DNA-based technology, the <em>Membranome</em> phage-Ab collection offers a complementary approach to standard proteomic analysis by applying unmatched throughput and high sensitivity of microarray technology to protein expression analysis. This method provides a comprehensive multiplexed readout (&gt;20,000 data points) of the expression of receptors displayed on the cell surface, and allows highly sensitive monitoring of their variations. We anticipate that this technology will have a fundamental impact on the identification of disease-associated biomarkers. This same technology has the potential to provide novel information about the biological mechanisms underlying several pathological conditions and, at the same time, deliver useful reagents for validating therapeutic hypotheses. We believe that the strategy and methods described in this paper can be applied to other biological systems such as the secretome, phosphoproteome, acetylome.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4><em>In Silico</em> identification of MP-coding genes</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All protein coding genes within the NCBI Human Gene database (26,651 genes) were analyzed using the NCBI Gene Ontology (GO) annotation and the <em>Phobius</em> transmembrane segment and signal peptide prediction algorithm <a href="#pone.0001508-Kall1">[16]</a>. A gene was classified as a Membrane Protein (MP) coding gene if either it was annotated as localized in the “Plasma Membrane” according to the Gene Ontology or one of the encoded peptides was predicted to be a membrane protein by the <em>Phobius</em> algorithm. Approximately 26% of the human genes were thus classified as MP genes and then subjected to a semi-manual inspection to exclude proteins that are part of intracellular compartments or the nucleus. Genes annotated to be attached to the membrane by a GPI–anchor were manually extracted from databases and added to the list. This left 4,609 MP genes, approximately 19% of all genes (the human <em>Membranome</em>) as our target gene population coding for cell surface antigens. In this analysis additional complexity generated by alternative splicing events or post-translational modifications were not taken into account.</p>


<h4>MP genes expression analysis in cell lines</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>mRNA gene expression data from 25 cell lines (see <a href="#pone.0001508.s001">Table S1</a>) were analyzed for variation of expression levels across the cell lines. Expression levels were inferred using custom designed exon-junction microarrays carrying multiple probes for each transcript as described <a href="#pone.0001508-Schadt1">[17]</a>. To estimate the reliability of absolute mRNA quantification obtained by exon-junction microarrays, 40 genes were independently assayed by TaqMan on the A549 cell line and a good correlation was observed (data not shown). Transcript abundance for about 3,700 MP genes represented on the exon-junction chip was expressed as an interval of 100 to 60,000 arbitrary expression units. “Cell line surface diversity”, here defined as difference in composition of surface membrane proteins, was estimated by the comparative analysis of the highest expressed MP genes. For each cell line, MP genes were ranked according to their expression values and those above an arbitrary ranking threshold (rank &lt; = 200, 300 or 400) were selected as the most abundant membrane proteins. This analysis revealed that a considerable portion of the <em>Membranome</em> (about 30%) was expressed at the highest level in only one out of the 25 cell lines, (see <a href="#pone.0001508.s001">Table S1</a>). On the contrary, only a small fraction (around 2–3%) of MP genes showed ubiquitous high expression levels across all cell lines. Comparable results were obtained when, instead of the ranking threshold, an absolute cutoff threshold was applied to expression levels. Individual cell lines contributed to various degrees to the fraction of cell-specific MP genes, with HUVEC and G361 contributing most. In order to obtain a global picture of the cell line surface diversity, a clustering analysis was carried out. All genes expressed above the ranking threshold (rank &lt; = 200, 300 or 400) in at least one cell line were selected and clustered hierarchically (average link) using the euclidean distance measure.</p>


<h4>Phage selection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Human, non-immunized scFv phage-displayed libraries were used as source of binders. These are high complexity libraries (over 10<sup>10</sup> different clones each) derived from B cells of healthy donors <a href="#pone.0001508-Vaughan1">[5]</a>. Each phage-Ab library (about 10<sup>11</sup> TU) was independently panned on 64 different cell lines (listed in <a href="#pone.0001508.s002">Table S2</a>). In each selection phage libraries were incubated with MPBS (3% powdered non-fat dry milk in PBS solution) for 30 min at room temperature (RT). Human cells used in the selection were detached from the plate using 2.5 mM EDTA in PBS and re-suspended to a final concentration of 1×10<sup>7</sup> cells/mL. Following centrifugation, cells were re-suspended with pre-adsorbed phage-Abs solution and incubated for 1 hour at RT. After extensive washing with PBS, cell-bound phage were centrifuged 5 min at 2 k rpm and re-suspended in 800 µL of trypsin solution (1 µg/ml in sodium phosphate). <em>E. Coli</em> TG1 cells (New England Biolabs, Beverly, MA) were infected with eluted phage and plated on 2XTY agar containing 2% glucose and 100 µg/mL ampicillin (2XTYAG). Phage rescue and amplification was carried out as described <a href="#pone.0001508-Hegmans1">[18]</a>. The selected phage were panned again on the same cell line. A variable number of clones (in the range 100–1,000) were randomly chosen form the pool of selected phage and tested by phage ELISA for their ability to bind the same cell line.</p>


<h4>Whole-cell phage-ELISA</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Cells were seeded overnight in 96 well plates at the concentration of 4×10<sup>4</sup> cells in 100 µL of propagation medium per well. Following blocking with MPBS, phage supernatants were added to each well and incubated for 1 hour at RT. After washing with PBS, binding of phage antibodies was revealed with HRP-conjugated anti-M13 antibody (Amersham Biosciences, Pittsburg, PA) in MPBS, using tetramethylbenzidine (Sigma Aldrich, Chicago, IL) as substrate. Phage-Abs whose binding to the target cell line measured by A<sub>450nm</sub>-A<sub>620nm</sub> was at least three fold higher than that observed with an unrelated phage and higher than 0.3 units were defined as positive.</p>


<h4>tagArray repertoire</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We identified 12 sequences sharing the following properties: i) 7 nucleotides long; ii) G2W5 base composition; iii) do not contain G-C palindromes; iv) do not contain self complementary sequences; v) have the same melting temperature, which is 10°C higher as compared to each of the other 11 sequences. By combining 4 of these building blocks we generated a repertoire of 20,736 (12<sup>4</sup>) sequences of 28 nucleotides. Each member of this repertoire has a 10°C difference or higher in Tm with any other member of the repertoire.</p>


<h4>Cloning the tag repertoire</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Phagemid pCANTAB5 DNA was linearized by SapI restriction and a synthetic dsDNA fragment with compatible cohesive ends was inserted (<span class="gene">AGCC<span class="underline">TTAATTAA</span>TACGACTCACTATAGAGGCCTGGTA<span class="underline">CCCGGG</span>TCGACTGTGTGTCT</span>). The resulting vector pCB5-PacI/XmaI has unique PacI and XmaI restriction sites (underlined in the sequence above).</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>An oligonucleotide was synthesized by “mix and split” procedure which contained the complete (G2W5)<sub>4</sub> sequence repertoire flanked by constant regions (tagrep; <span class="gene">CC<span class="underline">TTAATTAA</span>TACGACTCACTATAG</span>(G2W5)<sub>4</sub><span class="gene"><span class="underline">CCCGGG</span>GG</span>). An oligonucleotide with sequence <span class="gene">GGCCCGGG</span> was annealed to tagrep template and extended with DNA polymerase Klenov fragment. The double-stranded DNA product thus generated was digested with PacI and XmaI restriction enzymes and cloned into the corresponding sites of pCB5-PacI/XmaI (pCB5-(G2W5)<sub>4</sub> library). By transformation in <em>E.coli</em> TG1 cells a number of colonies much higher than the library complexity was obtained.</p>


<h4>Tagging the 4k Mbr collection</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The phagemid DNA derived from the 4k <em>Membranome</em> phage-Ab collection was digested with SfiI and NotI restriction enzymes. The 800bp fragment including the scFv sequences was purified and cloned into the corresponding sites of the pCB5-(G2W5)<sub>4</sub> library. Upon transformation in <em>E.coli,</em> TG1 cells of about 10,000 colonies were collected. Indeed, a computer-aided simulation indicated 2.5 times the scFv complexity as the best compromise between library complexity and the association of multiple tags to the same scFv sequence. A similar protocol was adopted to tag the 10k <em>Membranome</em> phage-Ab collection.</p>


<h4>Epithelial cell immunoaffinity purification</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Tumor epithelial cells were purified using a protocol derived from Buckhaults et al <a href="#pone.0001508-Buckhaults1">[19]</a>. In brief, fresh dissected tissue was digested with collagenase and DNase at 37°C in DMEM+ (DMEM, 1% FCS, 20 mM HEPES, Penicillin-Streptomycin). The resulting material was filtered through a nylon mesh to obtain single cell suspensions. After red blood cells lysis, cells were immediately used for panning or re-suspended in 10% DMSO/FBS and stored in liquid N<sub>2</sub>. In either case, before use, Ep-CAM positive epithelial cells were selected using autoMACS separator (Miltenyi Biotec, Bergisch Gladbach, Germany) according to manufacturer's instruction, and tested for their via1bility using Guava Personal Cytometer (Guava Technologies Inc., Hayward, CA).</p>


<h4>tagArray screening</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>The tagged 4 k Mbr phage-Ab library was panned on primary human epithelial cells derived from tumor or matched normal samples as described above. Usually 10<sup>7</sup> cells were incubated with 10<sup>11</sup> phage. Following extensive washing, cell-bound phage were eluted and used to infect TG1 cells. The Ampicillin-resistant bacterial colonies were collected and phagemid DNA purified by DNA purification kit (Qiagen GmbH, Hilden, Germany). Tag sequences were amplified by PCR. Forward primer was 5′-labelled with Cy5 or Cy3 dyes. Reaction mix included 30 ng phagemid DNA, 5 µM labeled forward and reverse primers, 200 uM dNTPs and 2.5 units Taq polymerase (Promega, Madison, WI) in a final 50 µL volume. Nine explicitly tagged-phagemids with different relative ratios and absolute amounts were added to the reaction. These “spike in” probes monitored whether the amplification process maintained the original relative frequencies of the clones in the amplified population. The amplification product was first purified from salts, free primers and dNTPs by a commercially available removal kit (Qiagen GmbH, Hilden, Germany). An aliquot was then run onto an agarose gel and the Cy5- or Cy3-labeled fragment was quantified by a fluorescence scanner (Typhoon; Perkin Elmer, Waltham, MA).</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>The Hybridization mix was assembled in a 490 µL final volume containing 3.5 mM Cy5- and Cy-3 labeled probes, 0.5 µM blocker and reverse primer. The latter perfectly hybridizes to the phagemid sequences flanking the tag, thus allowing the tag to interact with complementary tag sequences immobilized on a custom designed microArray chip (Agilent Technologies, Palo Alto, CA). The reaction mix was incubated at 99.9°C for 2 minutes and then snap-cooled by transferring back in ice for 1 min. The mix was then applied onto <em>tagArray</em>_44 k chip and incubated at 64°C for 15 hours in a rotating chamber. The slide was then washed at RT in 6xSSPEL (6× SSPE, 0.005% N-Lauroylsarcosine), 0.06xSSPEL, rinsed in stabilization and drying solution (Agilent Technologies, Palo Alto, CA) and analyzed with Agilent scanner. Images were acquired at 10μm resolution using an Agilent scanner, with the XDR option enabled (eXtended Dynamic Range: for each slide two images were generated with photomultiplier tube voltages of 100 and 10, respectively). Images were then processed using the Feature Extraction software (v 9.1, Agilent Technologies, Palo Alto, CA) generating the net signal for each channel (Cy3 and Cy5) and p-values after background and dye-bias correction. Results reported were obtained by combining data from two independent fluor reversal experiments.</p>


<h4>Rescue scFv associated to selected tags</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>For each tag sequence we generated a primer including the two 5′ blocks (X<sub>1</sub>X<sub>2</sub>) preceded by the adjacent 5′-CGACTCACTATAG-3′ constant region (primer dynoX<sub>1</sub>X<sub>2</sub>) and a primer encompassing the 4 blocks (primer X<sub>1</sub>X<sub>2</sub>X<sub>3</sub>X<sub>4</sub>). A reverse primer complementary to the phagemid region downstream of the scFv was used in conjunction with the dynoX<sub>1</sub>X<sub>2</sub>primer in a PCR reaction using the T phagemid DNA population as template. The product of this first amplification was diluted and re-amplified using the primer X<sub>1</sub>X<sub>2</sub>X<sub>3</sub>X<sub>4</sub> and the reverse primer. The PCR products obtained were sequenced and the corresponding 10k <em>Membranome</em> phage-Ab clones identified.</p>


<h4>scFv to IgG1 conversion</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>The scFv VH and VL sequences were introduced into a mammalian expression plasmid to produce the IgG1 heavy or light chain by using the Gateway™ technology (Invitrogen, Carlsbad, CA). In our hands, the frequency of recombinant clones was higher than 90% for both VH and VL insertion, close to the efficiency reported for different vector systems based on Gateway technology. The two plasmids secreting the IgG1 light and heavy chains were co-transfected into 293-EBNA cells. Cell culture medium was harvested every 3 days, samples collected together and purified on protein A affinity column.</p>


<h4>Target antigen identification</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>A-549, HCT-116 and MCF7 cells were grown at 70–80% confluence and cell surface proteins labeled with Cy5 fluorescent mono-reactive dye (Amersham, Uppsala, Sweden). About 20% of the cells were washed with PBS and incubated for 30 minutes at 4C° in the presence of Cy5 dye (usually 1 vial of Cy5 dye/10<sup>8</sup> cells). The Cy5 solution was removed and the cells washed with PBS. Cy5-labeled and unlabeled cells were then incubated for 30 min at 4%C in non-denaturing lysis buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 1% Triton-X100), complemented with Protease Inhibitors cocktail (Amersham, Uppsala, Sweden) at a concentration of 10<sup>6</sup> cells/mL. The supernatant of Cy5-labeled and unlabeled cells was then recovered, pooled and cleared by centrifugation. The native membrane protein preparations (NMPs) thus generated were immune-precipitated by incubation with a mAb covalently bound to CNBr-activated sepharose beads (Amersham, Uppsala, Sweden) at +4°C for 2hrs on a rotating wheel. The beads were then washed with lysis buffer and the immune-precipitated proteins were eluted in reducing loading buffer and analyzed by dual-wavelength 1D-PAGE using a Typhoon 9410 (Amersham, Uppsala, Sweden). Protein identification was performed essentially according to Shevchenko <em>et al.</em> <a href="#pone.0001508-Shevchenko1">[20]</a>. Briefly, the protein bands specific for membrane proteins were excised and digested <em>in-gel</em> with modified porcine trypsin (Promega, Madison,WI). Tryptic peptides were extracted from the gel pieces, dried down, resuspended in 0.1% Trifluoracetic Acid and desalted by µC<sub>18</sub> ZipTip (Millipore, Bedford, MA). Peptides were then analyzed either by Peptide Mass Fingerprint experiments (MALDI-TOF-MS) using a Voyager DE sSTR (Applied Biosystems, Foster City, CA) and the MASCOT software package or by capillary LC-ESI-IT-MS/MS using a LCQ Deca XP Plus (Thermo, Waltham, MA) equipped with a micro-electrospray source connected to a in-house packed C18 column (100 mm × 0.10 mm) and the TurboSequest software (Xcorr values &gt;2.5).</p>


<h4>Surface Plasmon Resonance</h4>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1"></a><p>Sensorgrams were recorded on a Biacore 3000 instrument operating at 25°C with CM5 chips (Biacore AB, Uppsala, Sweden). The antibodies were immobilized using the unmodified amine-coupling chemistry procedure suggested by Biacore. Regeneration conditions were established for each antibody according to the return of the baseline to the initial values. Double referencing was used by subtracting the sensogram resulting from an injection of running buffer. Mass transport influence was minimized combining flow rate and low amount of mAbs immobilized (ranging within 500–2000 RU). The titrations to determine the kinetic constants of complex formation were designed with 10 different analyte concentrations, plus one duplicate concentration. All the titrations were performed in duplicates with the kinject mode, using a flow rate of 50 µL/min and a constant time (3 min). The data were analyzed with the Bia-Evaluation 4.1 software using the Langmuir 1:1 binding model, and the results are given as averages of the duplicates. When feasible, the titrations were also performed at equilibrium. In this case the flow rate used was 10 µL/min, with a contact time of around 6 min. The equilibrium data were analyzed using the equilibrium model within the Bia Evaluation software.</p>


<h4>Flow cytometry</h4>
<a id="article1.body1.sec4.sec14.p1" name="article1.body1.sec4.sec14.p1"></a><p>PEG-precipitated phage-Abs were re-suspended in PBBS (2% BSA in PBS) and incubated for 30 min at RT. Primary human cells obtained from tumor or normal tissue were incubated with phage-Ab solution 60min at 4°C (2×10<sup>5</sup> cells with pre-blocked phage in 100 µL). After washing with PBBS, cells were incubated with biotin-conjugated anti-M13 phage monoclonal antibody (Progen Biotechnik, Heidelber, Germany) for 30 min at +4°C. then washed and incubated with streptavidin-APC (Molecular Probes/Invitrogen, Carlsbad, CA) or with anti- Ep-CAM FITC Mab (Miltenyi Biotec, Bergisch Gladbach, Germany) 30 min at 4°C. Finally, cells were washed and fixed with 1% PAF in PBS. mAbs were labeled by incubation with Zenon Human IgG APC labeling kit (Molecular Probes/Invitrogen, Carlsbad, CA) according to manufacturer's guidelines. Target cells were then incubated with pre-labeled mAb solution for 30 min at RT (2×10<sup>5</sup> cells in 100 µL of PBBS for 30 min). The same test was performed by using the anti Ep-CAM FITC MAb (Miltenyi Biotec, Bergisch Gladbach, Germany). In both cases cells were then washed and fixed with 1% paraformaldehyde in PBS.</p>


<h4>Mice</h4>
<a id="article1.body1.sec4.sec15.p1" name="article1.body1.sec4.sec15.p1"></a><p>Mice were bred under specific pathogen-free conditions by Charles River Breeding Laboratories (Calco, Como, Italy). In all manoeuvres, mice were treated in accordance with European guidelines. In particular, at the time of injection mice were fully anesthetized with ketamine (Merial Italia, Milano, Italy) at 100 mg/kg of body weight and xylazine (BIO 98; Bologna, Italy) at 5.2 mg/kg.</p>


<h4>Cell viability and apoptosis assays</h4>
<a id="article1.body1.sec4.sec16.p1" name="article1.body1.sec4.sec16.p1"></a><p>HCT-116 cells were cultured in McCoy's supplemented with 10% fetal calf serum (FCS) in a humidified atmosphere with 5% CO<sub>2</sub> at 37°C. For cell viability assays, cells were seeded in 96-well plates (10<sup>3</sup> cells/well) and incubated for 72 hrs with 100 µg/mL of Mbr-4 or an isotypic IgG used as negative control. Cell viability was then measured using a fluorometric assay according to the manufacturer's instructions (Cell Titer Blue assay; Promega, Madison, WI) and expressed as percentage of the value measured by cells treated with the negative control. For cell apoptosis assays, cells were seeded in 24-well plated (5×10<sup>3</sup> cells/well) and incubated for 48, 72 and 96 hrs with 0.8, 4 and 20 µg/mL, respectively, of mAb-4 or an isotypic IgG as negative control. As positive control 500nM apoptosis inducer Staurosporine was also tested. Cell apoptosis was monitored by measuring Annexin V and 7-aminoactinomycin (7-AAD) expression using a Guava Personal Cytometer (Guava Technologies Inc., Hayward, CA) and expressed as percentage of the Annexin V-, AAD-positive cells. No variation in the percentage of necrotic cells was detected. Data were analyzed with one-way ANOVA (p&lt;0.05).</p>


<h4>In vivo experiments</h4>
<a id="article1.body1.sec4.sec17.p1" name="article1.body1.sec4.sec17.p1"></a><p>BALB/c nude mice were challenged with a subcutaneous injection of 4×10<sup>6</sup> HCT-116 cells (day 0). The same animals were injected with 0.5 mg of Mbr-4 or a control isotypic IgG at day 0, 2, 4, 7, 9, 11, 14, 16, 18, and 21 (12 animals per group). Mice were inspected three times a week and tumor masses measured with calipers in two perpendicular diameters. Growth was monitored until a tumor exceeded an average diameter of 10mm, at which time mice were euthanized for humane reasons. Data reported are representative of two independent experiments.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001508"></div><a name="pone.0001508.s001" id="pone.0001508.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001508.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001508.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001508.s002" id="pone.0001508.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001508.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001508.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001508.s003" id="pone.0001508.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001508.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001508.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.01 MB XLS)</p>
<a name="pone.0001508.s004" id="pone.0001508.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001508.s004">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001508.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.02 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We wish to thank Manuel Pomeranz and Elisabetta Tora (IRBM ROBOLAB) for providing the cell cultures used in this work. We also thank Manuela Emili for help with graphics and Janet Clench for editing the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PM RC. Performed the experiments: VV FP FT AV CS ADP MArcuri FM AB GB SC AD SG HA MAmbrosio VA MB AC MN AAN LO MP GS. Contributed reagents/materials/analysis tools: PU ED ALahm PN AS. Wrote the paper: PM. Other: Planned and supervised the selection of the Membranome collection: AN ALuzzago. Performed selection and characterization Membranome collection: CS ADP MArcuri FM AB. Conceived and implemented the Membranome collection: RC. Characterized binding properties of the selected IgGs: GB. Converted, expressed and purified IgGs: SC. Contributed software tools for tagArray analysis: ED PU. Set up and performed tagArray experiments: MN GS SG. Contributed software tools for tagArray analysis: ALahm. Peformed in vitro and in vivo experiments using tagArray: HA. Converted, expressed and purified IgGs: AAN AC MB VA MAmbrosio. Synthesized oligonucleotides: PN. Set up and performed Ag identification experiments: AD FT LO. Identified tag sequences: AS. Performed in vitro and in vivo experiments using tagArray: AV MP FP VV. Conceived, developed and implemented the tagArray technology: PM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001508-Fuh1" id="pone.0001508-Fuh1"></a>Fuh G (2007) Synthetic antibodies as therapeutics. Expert Opin Biol Ther  7: 73–87.  <ul class="find" data-citedArticleID="1047934" data-doi="10.1517/14712598.7.1.73"><li><a href="http://dx.doi.org/10.1517/14712598.7.1.73" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthetic+antibodies+as+therapeutics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthetic+antibodies+as+therapeutics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001508-Brekke1" id="pone.0001508-Brekke1"></a>Brekke OH, Sandlie I (2003) Therapeutic antibodies for human diseases at the dawn of the twenty-first century. Nat Rev Drug Discov  2: 52–62.  <ul class="find" data-citedArticleID="1047924" data-doi="10.1038/nrd984"><li><a href="http://dx.doi.org/10.1038/nrd984" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Therapeutic+antibodies+for+human+diseases+at+the+dawn+of+the+twenty-first+century." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Therapeutic+antibodies+for+human+diseases+at+the+dawn+of+the+twenty-first+century.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001508-Imai1" id="pone.0001508-Imai1"></a>Imai K, Takaoka A (2006) Comparing antibody and small-molecule therapies for cancer. Nat Rev Cancer  6: 714–727.  <ul class="find" data-citedArticleID="1047940" data-doi="10.1038/nrc1913"><li><a href="http://dx.doi.org/10.1038/nrc1913" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparing+antibody+and+small-molecule+therapies+for+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparing+antibody+and+small-molecule+therapies+for+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001508-Adams1" id="pone.0001508-Adams1"></a>Adams GP, Weiner LM (2005) Monoclonal antibody therapy of cancer. Nat Biotechnol  23: 1147–1157.  <ul class="find" data-citedArticleID="1047922" data-doi="10.1038/nbt1137"><li><a href="http://dx.doi.org/10.1038/nbt1137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monoclonal+antibody+therapy+of+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monoclonal+antibody+therapy+of+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001508-Vaughan1" id="pone.0001508-Vaughan1"></a>Vaughan TJ, Williams AJ, Pritchard K, Osbourn JK, Pope AR, et al.  (1996) Human antibodies with sub-nanomolar affinities isolated from a large non-immunized phage display library. Nat Biotechnol  14: 309–314.  <ul class="find" data-citedArticleID="1047958" data-doi="10.1038/nbt0396-309"><li><a href="http://dx.doi.org/10.1038/nbt0396-309" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+antibodies+with+sub-nanomolar+affinities+isolated+from+a+large+non-immunized+phage+display+library." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+antibodies+with+sub-nanomolar+affinities+isolated+from+a+large+non-immunized+phage+display+library.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001508-Chothia1" id="pone.0001508-Chothia1"></a>Chothia C, Lesk AM, Gherardi E, Tomlinson IM, Walter G, et al.  (1992) Structural repertoire of the human VH segments. J Mol Biol  227: 799–817.  <ul class="find" data-citedArticleID="1047930" data-doi="10.1016/0022-2836(92)90224-8"><li><a href="http://dx.doi.org/10.1016/0022-2836(92)90224-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+repertoire+of+the+human+VH+segments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+repertoire+of+the+human+VH+segments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001508-Chothia2" id="pone.0001508-Chothia2"></a>Chothia C, Lesk AM, Tramontano A, Levitt M, Smith-Gill SJ, et al.  (1989) Conformations of immunoglobulin hypervariable regions. Nature  342: 877–883.  <ul class="find" data-citedArticleID="1047932" data-doi="10.1038/342877a0"><li><a href="http://dx.doi.org/10.1038/342877a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformations+of+immunoglobulin+hypervariable+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformations+of+immunoglobulin+hypervariable+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001508-Xu1" id="pone.0001508-Xu1"></a>Xu JL, Davis MM (2000) Diversity in the CDR3 region of V(H) is sufficient for most antibody specificities. Immunity  13: 37–45.  <ul class="find" data-citedArticleID="1047960" data-doi="10.1016/s1074-7613(00)00006-6"><li><a href="http://dx.doi.org/10.1016/s1074-7613(00)00006-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+in+the+CDR3+region+of+V%28H%29+is+sufficient+for+most+antibody+specificities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+in+the+CDR3+region+of+V%28H%29+is+sufficient+for+most+antibody+specificities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001508-Sahin1" id="pone.0001508-Sahin1"></a>Sahin U, Tureci O, Schmitt H, Cochlovius B, Johannes T, et al.  (1995) Human neoplasms elicit multiple specific immune responses in the autologous host. Proc Natl Acad Sci U S A  92: 11810–11813.  <ul class="find" data-citedArticleID="1047952" data-doi="10.1073/pnas.92.25.11810"><li><a href="http://dx.doi.org/10.1073/pnas.92.25.11810" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+neoplasms+elicit+multiple+specific+immune+responses+in+the+autologous+host." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+neoplasms+elicit+multiple+specific+immune+responses+in+the+autologous+host.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001508-Gomme1" id="pone.0001508-Gomme1"></a>Gomme PT, McCann KB, Bertolini J (2005) Transferrin: structure, function and potential therapeutic actions. Drug Discov Today  10: 267–273.  <ul class="find" data-citedArticleID="1047936" data-doi="10.1016/s1359-6446(04)03333-1"><li><a href="http://dx.doi.org/10.1016/s1359-6446(04)03333-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transferrin%3A+structure%2C+function+and+potential+therapeutic+actions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transferrin%3A+structure%2C+function+and+potential+therapeutic+actions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001508-Brooks1" id="pone.0001508-Brooks1"></a>Brooks D, Taylor C, Dos Santos B, Linden H, Houghton A, et al.  (1995) Phase Ia trial of murine immunoglobulin A antitransferrin receptor antibody 42/6. Clin Cancer Res  1: 1259–1265.  <ul class="find" data-citedArticleID="1047926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phase+Ia+trial+of+murine+immunoglobulin+A+antitransferrin+receptor+antibody+42%2F6.&amp;auth=&amp;atitle=Phase+Ia+trial+of+murine+immunoglobulin+A+antitransferrin+receptor+antibody+42%2F6." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+Ia+trial+of+murine+immunoglobulin+A+antitransferrin+receptor+antibody+42%2F6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+Ia+trial+of+murine+immunoglobulin+A+antitransferrin+receptor+antibody+42%2F6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001508-Rainov1" id="pone.0001508-Rainov1"></a>Rainov NG, Soling A (2005) Technology evaluation: TransMID, KS Biomedix/Nycomed/Sosei/PharmaEngine. Curr Opin Mol Ther  7: 483–492.  <ul class="find" data-citedArticleID="1047950"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Technology+evaluation%3A+TransMID%2C+KS+Biomedix%2FNycomed%2FSosei%2FPharmaEngine.&amp;auth=&amp;atitle=Technology+evaluation%3A+TransMID%2C+KS+Biomedix%2FNycomed%2FSosei%2FPharmaEngine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Technology+evaluation%3A+TransMID%2C+KS+Biomedix%2FNycomed%2FSosei%2FPharmaEngine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Technology+evaluation%3A+TransMID%2C+KS+Biomedix%2FNycomed%2FSosei%2FPharmaEngine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001508-Moura1" id="pone.0001508-Moura1"></a>Moura IC, Lepelletier Y, Arnulf B, England P, Baude C, et al.  (2004) A neutralizing monoclonal antibody (mAb A24) directed against the transferrin receptor induces apoptosis of tumor T lymphocytes from ATL patients. Blood  103: 1838–1845.  <ul class="find" data-citedArticleID="1047948" data-doi="10.1182/blood-2003-07-2440"><li><a href="http://dx.doi.org/10.1182/blood-2003-07-2440" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+neutralizing+monoclonal+antibody+%28mAb+A24%29+directed+against+the+transferrin+receptor+induces+apoptosis+of+tumor+T+lymphocytes+from+ATL+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+neutralizing+monoclonal+antibody+%28mAb+A24%29+directed+against+the+transferrin+receptor+induces+apoptosis+of+tumor+T+lymphocytes+from+ATL+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001508-Kasibhatla1" id="pone.0001508-Kasibhatla1"></a>Kasibhatla S, Jessen KA, Maliartchouk S, Wang JY, English NM, et al.  (2005) A role for transferrin receptor in triggering apoptosis when targeted with gambogic acid. Proc Natl Acad Sci U S A  102: 12095–12100.  <ul class="find" data-citedArticleID="1047944" data-doi="10.1073/pnas.0406731102"><li><a href="http://dx.doi.org/10.1073/pnas.0406731102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+transferrin+receptor+in+triggering+apoptosis+when+targeted+with+gambogic+acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+transferrin+receptor+in+triggering+apoptosis+when+targeted+with+gambogic+acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001508-Lu1" id="pone.0001508-Lu1"></a>Lu P, Vogel C, Wang R, Yao X, Marcotte EM (2007) Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation. Nat Biotechnol  25: 117–124.  <ul class="find" data-citedArticleID="1047946" data-doi="10.1038/nbt1270"><li><a href="http://dx.doi.org/10.1038/nbt1270" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Absolute+protein+expression+profiling+estimates+the+relative+contributions+of+transcriptional+and+translational+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Absolute+protein+expression+profiling+estimates+the+relative+contributions+of+transcriptional+and+translational+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001508-Kall1" id="pone.0001508-Kall1"></a>Kall L, Krogh A, Sonnhammer EL (2004) A combined transmembrane topology and signal peptide prediction method. J Mol Biol  338: 1027–1036.  <ul class="find" data-citedArticleID="1047942" data-doi="10.1016/j.jmb.2004.03.016"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.03.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+combined+transmembrane+topology+and+signal+peptide+prediction+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+combined+transmembrane+topology+and+signal+peptide+prediction+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001508-Schadt1" id="pone.0001508-Schadt1"></a>Schadt EE, Edwards SW, GuhaThakurta D, Holder D, Ying L, et al.  (2004) A comprehensive transcript index of the human genome generated using microarrays and computational approaches. Genome Biol  5: R73.  <ul class="find" data-citedArticleID="1047954"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+comprehensive+transcript+index+of+the+human+genome+generated+using+microarrays+and+computational+approaches.&amp;auth=&amp;atitle=A+comprehensive+transcript+index+of+the+human+genome+generated+using+microarrays+and+computational+approaches." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+transcript+index+of+the+human+genome+generated+using+microarrays+and+computational+approaches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+transcript+index+of+the+human+genome+generated+using+microarrays+and+computational+approaches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001508-Hegmans1" id="pone.0001508-Hegmans1"></a>Hegmans JP, Radosevic K, Voerman JS, Burgers JA, Hoogsteden HC, et al.  (2002) A model system for optimising the selection of membrane antigen-specific human antibodies on intact cells using phage antibody display technology. J Immunol Methods  262: 191–204.  <ul class="find" data-citedArticleID="1047938" data-doi="10.1016/s0022-1759(02)00005-4"><li><a href="http://dx.doi.org/10.1016/s0022-1759(02)00005-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+model+system+for+optimising+the+selection+of+membrane+antigen-specific+human+antibodies+on+intact+cells+using+phage+antibody+display+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+model+system+for+optimising+the+selection+of+membrane+antigen-specific+human+antibodies+on+intact+cells+using+phage+antibody+display+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001508-Buckhaults1" id="pone.0001508-Buckhaults1"></a>Buckhaults P, Rago C, St Croix B, Romans KE, Saha S, et al.  (2001) Secreted and cell surface genes expressed in benign and malignant colorectal tumors. Cancer Res  61: 6996–7001.  <ul class="find" data-citedArticleID="1047928"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Secreted+and+cell+surface+genes+expressed+in+benign+and+malignant+colorectal+tumors.&amp;auth=&amp;atitle=Secreted+and+cell+surface+genes+expressed+in+benign+and+malignant+colorectal+tumors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secreted+and+cell+surface+genes+expressed+in+benign+and+malignant+colorectal+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secreted+and+cell+surface+genes+expressed+in+benign+and+malignant+colorectal+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001508-Shevchenko1" id="pone.0001508-Shevchenko1"></a>Shevchenko A, Wilm M, Vorm O, Mann M (1996) Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels. Anal Chem  68: 850–858.  <ul class="find" data-citedArticleID="1047956" data-doi="10.1021/ac950914h"><li><a href="http://dx.doi.org/10.1021/ac950914h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+spectrometric+sequencing+of+proteins+silver-stained+polyacrylamide+gels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+spectrometric+sequencing+of+proteins+silver-stained+polyacrylamide+gels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.XML" value="81519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.PDF" value="412676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g001.PNG_L" value="191799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g001.PNG_M" value="30516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g001.PNG_S" value="3636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g001.TIF" value="327444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g001.PNG_I" value="21755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g002.PNG_L" value="910373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g002.PNG_M" value="59382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g002.PNG_S" value="3998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g002.TIF" value="1515510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g002.PNG_I" value="24313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g003.PNG_L" value="209577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g003.PNG_M" value="16917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g003.PNG_S" value="4152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g003.TIF" value="417474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g003.PNG_I" value="18971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g004.PNG_L" value="682361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g004.PNG_M" value="64080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g004.PNG_S" value="4208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g004.TIF" value="1523236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g004.PNG_I" value="23994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g005.PNG_L" value="348724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g005.PNG_M" value="43060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g005.PNG_S" value="5212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g005.TIF" value="509406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g005.PNG_I" value="35501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g006.PNG_L" value="192466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g006.PNG_M" value="32134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g006.PNG_S" value="4332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g006.TIF" value="327686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.g006.PNG_I" value="23763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.s001.XLS" value="16384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.s002.XLS" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.s003.XLS" value="13824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001508.s004.DOC" value="20992"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001508&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001508" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001508&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001508&volume=&issue=&title=Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology&author_name=Paolo%20Monaci%2C%20Alessandra%20Luzzago%2C%20Claudia%20Santini%2C%20Alessandra%20De%20Pra%2C%20Mirko%20Arcuri%2C%20Francesca%20Magistri%2C%20Alessandro%20Bellini%2C%20Helenia%20Ansuini%2C%20Maria%20Ambrosio%2C%20Virginia%20Ammendola%2C%20Maria%20Giulia%20Bigotti%2C%20Agostino%20Cirillo%2C%20Maurizio%20Nuzzo%2C%20Annamaria%20Assunta%20Nasti%2C%20Philippe%20Neuner%2C%20Laura%20Orsatti%2C%20Monica%20Pezzanera%2C%20Andrea%20Sbardellati%2C%20Giuseppe%20Silvestre%2C%20Paolo%20Uva%2C%20Valentina%20Viti%2C%20Gaetano%20Barbato%2C%20Stefano%20Colloca%2C%20Anna%20Demartis%2C%20Emanuele%20De%20Rinaldis%2C%20Saverio%20Giampaoli%2C%20Armin%20Lahm%2C%20Fabio%20Palombo%2C%20Fabio%20Talamo%2C%20Alessandra%20Vitelli%2C%20Alfredo%20Nicosia%2C%20Riccardo%20Cortese&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001508" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001508" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001508" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001508&amp;t=Differential%20Screening%20of%20Phage-Ab%20Libraries%20by%20Oligonucleotide%20Microarray%20Technology" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001508&title=Differential%20Screening%20of%20Phage-Ab%20Libraries%20by%20Oligonucleotide%20Microarray%20Technology&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001508&amp;title=Differential%20Screening%20of%20Phage-Ab%20Libraries%20by%20Oligonucleotide%20Microarray%20Technology" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001508" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001508&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Differential Screening of Phage-Ab Libraries by Oligonucleotide Microarray Technology';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001508';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Differential%20Screening%20of%20Phage-Ab%20Libraries%20by%20Oligonucleotide%20Microarray%20Technology http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001508" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001508" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacteriophages%22" title="Search for articles in the subject area:'Bacteriophages'"><div class="flagText">Bacteriophages</div></a>
              <div data-categoryid="43083" data-articleid="25844"
                   data-categoryname="Bacteriophages"
                   class="flagImage" title="Flag 'Bacteriophages' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+binding%22" title="Search for articles in the subject area:'Cell binding'"><div class="flagText">Cell binding</div></a>
              <div data-categoryid="19609" data-articleid="25844"
                   data-categoryname="Cell binding"
                   class="flagImage" title="Flag 'Cell binding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+membranes%22" title="Search for articles in the subject area:'Cell membranes'"><div class="flagText">Cell membranes</div></a>
              <div data-categoryid="17599" data-articleid="25844"
                   data-categoryname="Cell membranes"
                   class="flagImage" title="Flag 'Cell membranes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25844"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Library+screening%22" title="Search for articles in the subject area:'Library screening'"><div class="flagText">Library screening</div></a>
              <div data-categoryid="17295" data-articleid="25844"
                   data-categoryname="Library screening"
                   class="flagImage" title="Flag 'Library screening' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="25844"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25844"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+tagged+site+analysis%22" title="Search for articles in the subject area:'Sequence tagged site analysis'"><div class="flagText">Sequence tagged site analysis</div></a>
              <div data-categoryid="48851" data-articleid="25844"
                   data-categoryname="Sequence tagged site analysis"
                   class="flagImage" title="Flag 'Sequence tagged site analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2382'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3381'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2857&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
