

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene PCSK9 Based on Phylogenetic Analysis in 14 Primate Species</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001098"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001098" />

    <meta name="citation_title" content="Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene PCSK9 Based on Phylogenetic Analysis in 14 Primate Species"/>
    <meta itemprop="name" content="Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene PCSK9 Based on Phylogenetic Analysis in 14 Primate Species"/>

      <meta name="citation_author" content="Keyue Ding"/>
            <meta name="citation_author_institution" content="Division of Cardiovascular Diseases, Mayo Clinic and Foundation, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="Samantha J. McDonough"/>
            <meta name="citation_author_institution" content="Division of Cardiovascular Diseases, Mayo Clinic and Foundation, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="Iftikhar J. Kullo"/>
            <meta name="citation_author_institution" content="Division of Cardiovascular Diseases, Mayo Clinic and Foundation, Rochester, Minnesota, United States of America"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001098.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1098"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Autocatalytic processing of site-1 protease removes propeptide and permits cleavage of sterol regulatory element-binding proteins.; citation_author=PJ Espenshade; citation_author=D Cheng; citation_author=JL Goldstein; citation_author=MS Brown; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=1; citation_pages=22795-22804; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A proteolytic pathway that controls the cholesterol content of membranes, cells, and blood.; citation_author=MS Brown; citation_author=JL Goldstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=2; citation_pages=11041-11048; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Biosynthesis and cellular trafficking of the convertase SKI-1/S1P: ectodomain shedding requires SKI-1 activity.; citation_author=A Elagoz; citation_author=S Benjannet; citation_author=A Mammarbassi; citation_author=L Wickham; citation_author=NG Seidah; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=3; citation_pages=11265-11275; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=NARC-1/PCSK9 and its natural mutants: zymogen cleavage and effects on the low density lipoprotein (LDL) receptor and LDL cholesterol.; citation_author=S Benjannet; citation_author=D Rhainds; citation_author=R Essalmani; citation_author=J Mayne; citation_author=L Wickham; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=4; citation_pages=48865-48875; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional characterization of Narc 1, a novel proteinase related to proteinase K.; citation_author=S Naureckiene; citation_author=L Ma; citation_author=K Sreekumar; citation_author=U Purandare; citation_author=CF Lo; citation_journal_title=Arch Biochem Biophys; citation_volume=420; citation_number=5; citation_pages=55-67; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The secretory proprotein convertase neural apoptosis-regulated convertase 1 (NARC-1): liver regeneration and neuronal differentiation.; citation_author=NG Seidah; citation_author=S Benjannet; citation_author=L Wickham; citation_author=J Marcinkiewicz; citation_author=SB Jasmin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=6; citation_pages=928-933; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Wild-type PCSK9 inhibits LDL clearance but does not affect apoB-containing lipoprotein production in mouse and cultured cells.; citation_author=F Lalanne; citation_author=G Lambert; citation_author=MJ Amar; citation_author=M Chetiveaux; citation_author=Y Zair; citation_journal_title=J Lipid Res; citation_volume=46; citation_number=7; citation_pages=1312-1319; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adenoviral-mediated expression of Pcsk9 in mice results in a low-density lipoprotein receptor knockout phenotype.; citation_author=KN Maxwell; citation_author=JL Breslow; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=8; citation_pages=7100-7105; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Post-transcriptional regulation of low density lipoprotein receptor protein by proprotein convertase subtilisin/kexin type 9a in mouse liver.; citation_author=SW Park; citation_author=YA Moon; citation_author=JD Horton; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=9; citation_pages=50630-50638; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mutations in PCSK9 cause autosomal dominant hypercholesterolemia.; citation_author=M Abifadel; citation_author=M Varret; citation_author=JP Rabes; citation_author=D Allard; citation_author=K Ouguerram; citation_journal_title=Nat Genet; citation_volume=34; citation_number=10; citation_pages=154-156; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mutations in the PCSK9 gene in Norwegian subjects with autosomal dominant hypercholesterolemia.; citation_author=TP Leren; citation_journal_title=Clin Genet; citation_volume=65; citation_number=11; citation_pages=419-422; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A mutation in PCSK9 causing autosomal-dominant hypercholesterolemia in a Utah pedigree.; citation_author=KM Timms; citation_author=S Wagner; citation_author=ME Samuels; citation_author=K Forbey; citation_author=H Goldfine; citation_journal_title=Hum Genet; citation_volume=114; citation_number=12; citation_pages=349-353; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evidence for effect of mutant PCSK9 on apolipoprotein B secretion as the cause of unusually severe dominant hypercholesterolaemia.; citation_author=XM Sun; citation_author=ER Eden; citation_author=I Tosi; citation_author=CK Neuwirth; citation_author=D Wile; citation_journal_title=Hum Mol Genet; citation_volume=14; citation_number=13; citation_pages=1161-1169; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Severe hypercholesterolemia in four British families with the D374Y mutation in the PCSK9 gene: long-term follow-up and treatment response.; citation_author=RP Naoumova; citation_author=I Tosi; citation_author=D Patel; citation_author=C Neuwirth; citation_author=SD Horswell; citation_journal_title=Arterioscler Thromb Vasc Biol; citation_volume=25; citation_number=14; citation_pages=2654-2660; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic variants in PCSK9 affect the cholesterol level in Japanese.; citation_author=K Shioji; citation_author=T Mannami; citation_author=Y Kokubo; citation_author=N Inamoto; citation_author=S Takagi; citation_journal_title=J Hum Genet; citation_volume=49; citation_number=15; citation_pages=109-114; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Low LDL cholesterol in individuals of African descent resulting from frequent nonsense mutations in PCSK9.; citation_author=J Cohen; citation_author=A Pertsemlidis; citation_author=IK Kotowski; citation_author=R Graham; citation_author=CK Garcia; citation_journal_title=Nat Genet; citation_volume=37; citation_number=16; citation_pages=161-165; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A spectrum of PCSK9 alleles contributes to plasma levels of low-density lipoprotein cholesterol.; citation_author=IK Kotowski; citation_author=A Pertsemlidis; citation_author=A Luke; citation_author=RS Cooper; citation_author=GL Vega; citation_journal_title=Am J Hum Genet; citation_volume=78; citation_number=17; citation_pages=410-422; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Missense mutations in the PCSK9 gene are associated with hypocholesterolemia and possibly increased response to statin therapy.; citation_author=KE Berge; citation_author=L Ose; citation_author=TP Leren; citation_journal_title=Arterioscler Thromb Vasc Biol; citation_volume=26; citation_number=18; citation_pages=1094-1100; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The c.43_44insCTG variation in PCSK9 is associated with low plasma LDL-cholesterol in a Caucasian population.; citation_author=P Yue; citation_author=M Averna; citation_author=X Lin; citation_author=G Schonfeld; citation_journal_title=Hum Mutat; citation_volume=27; citation_number=19; citation_pages=460-466; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence variations in PCSK9, low LDL, and protection against coronary heart disease.; citation_author=JC Cohen; citation_author=E Boerwinkle; citation_author=TH Mosley; citation_author=HH Hobbs; citation_journal_title=N Engl J Med; citation_volume=354; citation_number=20; citation_pages=1264-1272; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Skeletal and dental morphology supports diphyletic origin of baboons and mandrills.; citation_author=JG Fleagle; citation_author=WS McGraw; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=21; citation_pages=1157-1161; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Serum concentrations of lipids, vitamins A and E, vitamin D metabolites, and carotenoids in nine primate species at four zoos.; citation_author=S Crissey; citation_author=J Barr; citation_author=K Slifka; citation_author=P Bowen; citation_author=M Stacewicz-Sapuntzakis; citation_journal_title=Zoo biology; citation_volume=18; citation_number=22; citation_pages=551-564; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Elevated gene expression levels distinguish human from non-human primate brains.; citation_author=M Caceres; citation_author=J Lachuer; citation_author=MA Zapala; citation_author=JC Redmond; citation_author=L Kudo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=23; citation_pages=13030-13035; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=SREBPs: activators of the complete program of cholesterol and fatty acid synthesis in the liver.; citation_author=JD Horton; citation_author=JL Goldstein; citation_author=MS Brown; citation_journal_title=J Clin Invest; citation_volume=109; citation_number=24; citation_pages=1125-1131; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Protease variants, LDL, and coronary heart disease.; citation_author=AR Tall; citation_journal_title=N Engl J Med; citation_volume=354; citation_number=25; citation_pages=1310-1312; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dual regulation of the LDL receptor–some clarity and new questions.; citation_author=AD Attie; citation_author=NG Seidah; citation_journal_title=Cell Metab; citation_volume=1; citation_number=26; citation_pages=290-292; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The proprotein convertases and their implication in sterol and/or lipid metabolism.; citation_author=NG Seidah; citation_author=AM Khatib; citation_author=A Prat; citation_journal_title=Biol Chem; citation_volume=387; citation_number=27; citation_pages=871-877; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages.; citation_author=Z Yang; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=28; citation_pages=908-917; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=M Nei; citation_author=S Kumar; citation_number=29; citation_date=2000; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Episodic adaptive evolution of primate lysozymes.; citation_author=W Messier; citation_author=CB Stewart; citation_journal_title=Nature; citation_volume=385; citation_number=30; citation_pages=151-154; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution.; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=31; citation_pages=568-573; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of the tumour suppressor BRCA1 in humans and chimpanzees. Australian Breast Cancer Family Study.; citation_author=GA Huttley; citation_author=S Easteal; citation_author=MC Southey; citation_author=A Tesoriero; citation_author=GG Giles; citation_journal_title=Nat Genet; citation_volume=25; citation_number=32; citation_pages=410-413; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Positive natural selection in the human lineage.; citation_author=PC Sabeti; citation_author=SF Schaffner; citation_author=B Fry; citation_author=J Lohmueller; citation_author=P Varilly; citation_journal_title=Science; citation_volume=312; citation_number=33; citation_pages=1614-1620; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HyPhy: hypothesis testing using phylogenies.; citation_author=SL Pond; citation_author=SD Frost; citation_author=SV Muse; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=34; citation_pages=676-679; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genomics approach to the detection of positive selection in cattle: adaptive evolution of the T-cell and natural killer cell-surface protein CD2.; citation_author=DJ Lynn; citation_author=AR Freeman; citation_author=C Murray; citation_author=DG Bradley; citation_journal_title=Genetics; citation_volume=170; citation_number=35; citation_pages=1189-1196; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Ka/Ks ratio: diagnosing the form of sequence evolution.; citation_author=LD Hurst; citation_journal_title=Trends Genet; citation_volume=18; citation_number=36; citation_pages=486; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Toward a phylogenetic classification of Primates based on DNA evidence complemented by fossil evidence.; citation_author=M Goodman; citation_author=CA Porter; citation_author=J Czelusniak; citation_author=SL Page; citation_author=H Schneider; citation_journal_title=Mol Phylogenet Evol; citation_volume=9; citation_number=37; citation_pages=585-598; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Likelihood models for detecting positively selected amino acid sites and applications to the HIV-1 envelope gene.; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Genetics; citation_volume=148; citation_number=38; citation_pages=929-936; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for heterogeneous selection pressure at amino acid sites.; citation_author=Z Yang; citation_author=R Nielsen; citation_author=N Goldman; citation_author=AM Pedersen; citation_journal_title=Genetics; citation_volume=155; citation_number=39; citation_pages=431-449; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical bayes inference of amino acid sites under positive selection.; citation_author=Z Yang; citation_author=WS Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=40; citation_pages=1107-1118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Accuracy and power of statistical methods for detecting adaptive evolution in protein coding sequences and for identifying positively selected sites.; citation_author=WS Wong; citation_author=Z Yang; citation_author=N Goldman; citation_author=R Nielsen; citation_journal_title=Genetics; citation_volume=168; citation_number=41; citation_pages=1041-1051; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Asymptotic properties of maximum likelihood estimators and likelihood ratio tests under non-standard conditions.; citation_author=S Self; citation_author=K Liang; citation_journal_title=J Am Stat Assoc; citation_volume=82; citation_number=42; citation_pages=605-610; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Molecular biology of PCSK9: its role in LDL metabolism.; citation_author=JD Horton; citation_author=JC Cohen; citation_author=HH Hobbs; citation_journal_title=Trends Biochem Sci; citation_volume=32; citation_number=43; citation_pages=71-77; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Of PCSK9, cholesterol homeostasis and parasitic infections: Possible survival benefits of loss-of-function PCSK9 genetic polymorphisms.; citation_author=M Mbikay; citation_author=J Mayne; citation_author=NG Seidah; citation_author=M Chretien; citation_journal_title=Med Hypotheses; citation_number=44; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=When less is more: gene loss as an engine of evolutionary change.; citation_author=MV Olson; citation_journal_title=Am J Hum Genet; citation_volume=64; citation_number=45; citation_pages=18-23; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The evolutionary history of human and chimpanzee Y-chromosome gene loss.; citation_author=GH Perry; citation_author=RY Tito; citation_author=BC Verrelli; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=46; citation_pages=853-859; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=J Zhang; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=47; citation_pages=2472-2479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Accuracy and power of the likelihood ratio test in detecting adaptive molecular evolution.; citation_author=M Anisimova; citation_author=JP Bielawski; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=48; citation_pages=1585-1592; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A composite estimate of primate phylogeny.; citation_author=A Purvis; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=348; citation_number=49; citation_pages=405-421; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Catalytic activity is not required for secreted PCSK9 to reduce LDL receptors in HepG2 cells.; citation_author=MC McNutt; citation_author=TA Lagace; citation_author=JD Horton; citation_journal_title=J Biol Chem; citation_volume=282; citation_number=50; citation_pages=20799-20803; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The proprotein convertases are potential targets in the treatment of dyslipidemia.; citation_author=NG Seidah; citation_author=A Prat; citation_journal_title=J Mol Med; citation_volume=85; citation_number=51; citation_pages=685-696; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The cellular trafficking of the secretory proprotein convertase PCSK9 and its dependence on the LDLR.; citation_author=N Nassoury; citation_author=DA Blasiole; citation_author=A Tebon Oler; citation_author=S Benjannet; citation_author=J Hamelin; citation_journal_title=Traffic; citation_volume=8; citation_number=52; citation_pages=718-732; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Intra- and interspecific variation in primate gene expression patterns.; citation_author=W Enard; citation_author=P Khaitovich; citation_author=J Klose; citation_author=S Zollner; citation_author=F Heissig; citation_journal_title=Science; citation_volume=296; citation_number=53; citation_pages=340-343; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Implication of the proprotein convertase NARC-1/PCSK9 in the development of the nervous system.; citation_author=S Poirier; citation_author=A Prat; citation_author=E Marcinkiewicz; citation_author=J Paquin; citation_author=BP Chitramuthu; citation_journal_title=J Neurochem; citation_volume=98; citation_number=54; citation_pages=838-850; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Decreased plasma cholesterol and hypersensitivity to statins in mice lacking Pcsk9.; citation_author=S Rashid; citation_author=DE Curtis; citation_author=R Garuti; citation_author=NN Anderson; citation_author=Y Bashmakov; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=55; citation_pages=5374-5379; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Proapoptotic effects of NARC 1 ( = PCSK9), the gene encoding a novel serine proteinase.; citation_author=B Bingham; citation_author=R Shen; citation_author=S Kotnis; citation_author=CF Lo; citation_author=BA Ozenberger; citation_journal_title=Cytometry A; citation_volume=69; citation_number=56; citation_pages=1123-1131; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Population genetic and phylogenetic evidence for positive selection on regulatory mutations at the factor VII locus in humans.; citation_author=MW Hahn; citation_author=MV Rockman; citation_author=N Soranzo; citation_author=DB Goldstein; citation_author=GA Wray; citation_journal_title=Genetics; citation_volume=167; citation_number=57; citation_pages=867-877; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Natural selection and population history in the human angiotensinogen gene (AGT): 736 complete AGT sequences in chromosomes from around the world.; citation_author=T Nakajima; citation_author=S Wooding; citation_author=T Sakagami; citation_author=M Emi; citation_author=K Tokunaga; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=58; citation_pages=898-916; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CYP3A variation and the evolution of salt-sensitivity variants.; citation_author=EE Thompson; citation_author=H Kuttab-Boulos; citation_author=D Witonsky; citation_author=L Yang; citation_author=BA Roe; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=59; citation_pages=1059-1069; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Positive selection on MMP3 regulation has shaped heart disease risk.; citation_author=MV Rockman; citation_author=MW Hahn; citation_author=N Soranzo; citation_author=DA Loisel; citation_author=DB Goldstein; citation_journal_title=Curr Biol; citation_volume=14; citation_number=60; citation_pages=1531-1539; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Differential susceptibility to hypertension is due to selection during the out-of-Africa expansion.; citation_author=JH Young; citation_author=YP Chang; citation_author=JD Kim; citation_author=JP Chretien; citation_author=MJ Klag; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=61; citation_pages=e82; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution of 5′ flanking regions of 87 candidate genes for atherosclerotic cardiovascular disease.; citation_author=K Ding; citation_author=IJ Kullo; citation_journal_title=Genet Epidemiol; citation_volume=30; citation_number=62; citation_pages=557-569; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Patterns of population differentiation of candidate genes for cardiovascular disease.; citation_author=IJ Kullo; citation_author=K Ding; citation_journal_title=BMC genetics; citation_volume=8; citation_number=63; citation_pages=48; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparative analyses of multi-species sequences from targeted genomic regions.; citation_author=JW Thomas; citation_author=JW Touchman; citation_author=RW Blakesley; citation_author=GG Bouffard; citation_author=SM Beckstrom-Sternberg; citation_journal_title=Nature; citation_volume=424; citation_number=64; citation_pages=788-793; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Episodic evolution of pyrin in primates: human mutations recapitulate ancestral amino acid states.; citation_author=P Schaner; citation_author=N Richards; citation_author=A Wadhwa; citation_author=I Aksentijevich; citation_author=D Kastner; citation_journal_title=Nat Genet; citation_volume=27; citation_number=65; citation_pages=318-321; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=SIFT: Predicting amino acid changes that affect protein function.; citation_author=PC Ng; citation_author=S Henikoff; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=66; citation_pages=3812-3814; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Human non-synonymous SNPs: server and survey.; citation_author=V Ramensky; citation_author=P Bork; citation_author=S Sunyaev; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=67; citation_pages=3894-3900; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A computer program for aligning a cDNA sequence with a genomic DNA sequence.; citation_author=L Florea; citation_author=G Hartzell; citation_author=Z Zhang; citation_author=GM Rubin; citation_author=W Miller; citation_journal_title=Genome Res; citation_volume=8; citation_number=68; citation_pages=967-974; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=69; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Comparison of models for nucleotide substitution used in maximum-likelihood phylogenetic estimation.; citation_author=Z Yang; citation_author=N Goldman; citation_author=A Friday; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=70; citation_pages=316-324; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation.; citation_author=Z Yang; citation_author=JP Bielawski; citation_journal_title=Trends in Ecology and Evolution; citation_volume=15; citation_number=71; citation_pages=496-503; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=72; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of MRG, a neuron-specific gene family implicated in nociception.; citation_author=SS Choi; citation_author=BT Lahn; citation_journal_title=Genome Res; citation_volume=13; citation_number=73; citation_pages=2252-2259; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001098" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene PCSK9 Based on Phylogenetic Analysis in 14 Primate Species"/>
    <meta name="twitter:description" content="BackgroundCholesterol homeostasis is maintained through finely tuned mechanisms regulating intestinal absorption, hepatic biosynthesis and secretion as well as plasma clearance. Proprotein convertase subtilisin/kexin type 9 (PCSK9) is a secreted enzyme of the serine protease family that reduces cellular uptake of plasma low-density lipoprotein (LDL) cholesterol by promoting LDL receptor (LDL-R) degradation. Species-specific positive selection has been noted in the LDLR promoter, leading to differential expression of LDLR among primates. Whether PCSK9 experienced significant selective pressure to maintain a functional relationship with its target protein, LDL-R, is unknown.Methodology/Principal FindingsWe compiled the sequences of the coding regions of PCSK9 from 14 primate species in the clade of Hominoids, Old World monkeys and New World monkeys. To detect selective pressure at the protein level, the ratios of nonsynonymous/synonymous substitution rate (dN/dS) under different evolutionary models were calculated across the phylogeny of PCSK9. Maximum likelihood analyses of dN/dS ratios for the aligned coding region sequences among 14 primate species indicated that PCSK9 was subject to a strong functional constraint (i.e., purifying selection). However, positive selection was noted in the functional carboxyl-terminal (C-terminal) domain in many branches across the phylogeny, especially in the lineage leading to the orangutan. Furthermore, at least five positively selected amino acids were detected in this lineage using the branch-site model A. In a sliding-window analysis, several dN/dS peaks in the C-terminal domain in both the human and the orangutan branches were noted.ConclusionsThese results suggest that among primates, differential selective pressure has shaped evolutionary patterns in the functional domains of PCSK9, an important regulator of cholesterol homeostasis."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001098.g005"/>

  <meta property="og:title" content="Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene PCSK9 Based on Phylogenetic Analysis in 14 Primate Species" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001098" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1394'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9073'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5813&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001098">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001098" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001098&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene <i>PCSK9</i> Based on Phylogenetic Analysis in 14 Primate Species
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Keyue Ding, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Cardiovascular Diseases, Mayo Clinic and Foundation, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Samantha J. McDonough, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Cardiovascular Diseases, Mayo Clinic and Foundation, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Iftikhar J. Kullo
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kullo.iftikhar@mayo.edu">kullo.iftikhar@mayo.edu</a></p>

                <p>Affiliation:
                  Division of Cardiovascular Diseases, Mayo Clinic and Foundation, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001098</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001098" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001098" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001098" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001098" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001098-g001" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001098-g002" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001098-t001" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001098-g003" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001098-t002" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001098-g004" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001098-t003" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001098-g005" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001098">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001098" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Cholesterol homeostasis is maintained through finely tuned mechanisms regulating intestinal absorption, hepatic biosynthesis and secretion as well as plasma clearance. Proprotein convertase subtilisin/kexin type 9 (PCSK9) is a secreted enzyme of the serine protease family that reduces cellular uptake of plasma low-density lipoprotein (LDL) cholesterol by promoting LDL receptor (LDL-R) degradation. Species-specific positive selection has been noted in the <em>LDLR</em> promoter, leading to differential expression of <em>LDLR</em> among primates. Whether <em>PCSK9</em> experienced significant selective pressure to maintain a functional relationship with its target protein, LDL-R, is unknown.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We compiled the sequences of the coding regions of <em>PCSK9</em> from 14 primate species in the clade of Hominoids, Old World monkeys and New World monkeys. To detect selective pressure at the protein level, the ratios of nonsynonymous/synonymous substitution rate (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>) under different evolutionary models were calculated across the phylogeny of <em>PCSK9</em>. Maximum likelihood analyses of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratios for the aligned coding region sequences among 14 primate species indicated that <em>PCSK9</em> was subject to a strong functional constraint (i.e., purifying selection). However, positive selection was noted in the functional carboxyl-terminal (C-terminal) domain in many branches across the phylogeny, especially in the lineage leading to the orangutan. Furthermore, at least five positively selected amino acids were detected in this lineage using the branch-site model A. In a sliding-window analysis, several <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> peaks in the C-terminal domain in both the human and the orangutan branches were noted.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These results suggest that among primates, differential selective pressure has shaped evolutionary patterns in the functional domains of <em>PCSK9</em>, an important regulator of cholesterol homeostasis.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ding K, McDonough SJ, Kullo IJ (2007) Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene <em>PCSK9</em> Based on Phylogenetic Analysis in 14 Primate Species. PLoS ONE 2(10):
          e1098.
            doi:10.1371/journal.pone.0001098</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> July 27, 2007; <strong>Accepted:</strong> October 8, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Ding et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by funds from the Mayo Foundation and NIH grant HL75794.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The low-density lipoprotein (LDL) receptor gene (<em>LDLR</em>) plays a key role in cholesterol homeostasis by receptor-mediated endocytosis of LDL cholesterol. Proprotein convertase subtilisin/kexin type 9 (<em>PCSK9</em>, MIM 607786) – a secreted enzyme of the serine protease family – is a newly discovered regulator of <em>LDLR</em> <a href="#pone.0001098-Espenshade1">[1]</a>–<a href="#pone.0001098-Elagoz1">[3]</a>. The human <em>PCSK9</em> locus spanning 25 kb and containing 12 exons, resides on chromosome 1p32. The <em>PCSK9</em> protein contains five functional domains, a signal-peptide and prodomain at its N-terminus, followed by a catalytic domain, a putative domain and a cysteine-rich carboxy-terminal domain <a href="#pone.0001098-Benjannet1">[4]</a>–<a href="#pone.0001098-Seidah1">[6]</a> (<a href="#pone-0001098-g001">Figure 1</a>). <em>PCSK9</em> induces LDL-R breakdown <a href="#pone.0001098-Lalanne1">[7]</a>, internalization and recycling <a href="#pone.0001098-Maxwell1">[8]</a>, <a href="#pone.0001098-Park1">[9]</a> and thereby reduces LDL clearance, and increases plasma levels of LDL cholesterol. Overexpression of wild-type <em>Pcsk9</em> gene in mice results in reduced number of LDL-R and hypercholesterolemia <a href="#pone.0001098-Benjannet1">[4]</a>, <a href="#pone.0001098-Maxwell1">[8]</a>, <a href="#pone.0001098-Park1">[9]</a>.</p>
<div class="figure" id="pone-0001098-g001"><div class="img"><a name="pone-0001098-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distribution of non-synonymous variations along <em>PCSK9</em>.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>There are five functional domains in the <em>PCSK9</em> protein <a href="#pone.0001098-Benjannet1">[4]</a>–<a href="#pone.0001098-Seidah1">[6]</a>: 1) a signal peptide (SP) (1~30 aa), 2) a prodomain (31~147 aa), 3) a catalytic domain (148~425 aa), 4) a putative P domain (426~525 aa), and 5) a C-terminal domain (526~691 aa). Gain-of-function mutations are only identified in families with hypercholesterolemia <a href="#pone.0001098-Abifadel1">[10]</a> or subjects with high LDL cholesterol levels <a href="#pone.0001098-Kotowski1">[17]</a>, and loss-of-function mutations in subjects with low levels of LDL cholesterol <a href="#pone.0001098-Shioji1">[15]</a>–<a href="#pone.0001098-Yue1">[19]</a>. Some non-synonymous mutations have been identified in subjects with either high or low plasma LDL cholesterol <a href="#pone.0001098-Benjannet1">[4]</a>, <a href="#pone.0001098-Kotowski1">[17]</a>, and are labeled ‘both’ in the figure. Rare mutations found in families with autosomal dominant hypercholesterolemia are labeled with an asterisk. The gain-of-function mutations are: S127R, F216L, R237W, D374Y, H417Q, R469W, E482G, F515L and H553R. The loss-of-function mutations are: 14insL, E57K, Y142X, L253F, H391N, Q554E, and C679X. Mutations associated with either high- or low- plasma levels of LDL cholesterol subjects are: R46L, A53V, N425S, A443T, I474V, Q619P, and E670G.</p>
<span>doi:10.1371/journal.pone.0001098.g001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Mutations in <em>PCSK9</em> can cause severe autosomal dominant hypercholesterolemia <a href="#pone.0001098-Benjannet1">[4]</a>, <a href="#pone.0001098-Maxwell1">[8]</a>–<a href="#pone.0001098-Naoumova1">[14]</a> (i.e., ‘gain-of-function’ mutations), and also low circulating levels of LDL cholesterol <a href="#pone.0001098-Shioji1">[15]</a>–<a href="#pone.0001098-Yue1">[19]</a> (i.e., ‘loss-of-function’ mutations). Kotowski et al. <a href="#pone.0001098-Kotowski1">[17]</a> described a spectrum of nonsense/missense mutations in <em>PCSK9</em> that were associated with low or elevated LDL cholesterol levels, in both black and white subjects. Relatively common sequence variants in <em>PCSK9</em> also contribute significantly to inter-individual variation in plasma levels of LDL cholesterol in the general population <a href="#pone.0001098-Kotowski1">[17]</a>. Cohen et al. <a href="#pone.0001098-Cohen2">[20]</a> showed that two nonsense mutations (Y142X and C679X) in blacks, and one missense mutation (R46L) in whites, were associated with reduced plasma levels of LDL cholesterol and lower incidence of coronary heart disease events. We have summarized these non-synonymous variations in <em>PCSK9</em> in <a href="#pone-0001098-g001">Figure 1</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Extant primates show a wide range of phenotypic adaptations to diverse environmental conditions, including substrate and diet <a href="#pone.0001098-Fleagle1">[21]</a>. Significant differences in lipid profiles occur among primates; for example, New World and Old World monkeys have significantly lower serum total cholesterol, triglycerides, and LDL cholesterol levels than Hominoids <a href="#pone.0001098-Crissey1">[22]</a>. Among Hominoids, gorillas have the highest circulating total cholesterol, triglycerides, and high-density lipoprotein (HDL) cholesterol levels <a href="#pone.0001098-Crissey1">[22]</a>. Caceres et al. <a href="#pone.0001098-Caceres1">[23]</a> found that several genes related to lipid metabolism were differentially expressed in humans and non-human primates. <em>LDLR</em> has also been shown to be differentially expressed among mammals <a href="#pone.0001098-Horton1">[24]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Activation of the sterol regulatory element binding protein-2 (SREBP-2), a key transcription factor of <em>LDLR</em>, not only leads to increased expression of <em>LDLR</em>, but also of <em>PCSK9</em> <a href="#pone.0001098-Tall1">[25]</a>, and the mRNA expression of <em>LDLR</em> and <em>PCSK9</em> are coordinately up-regulated in absence of sterols <a href="#pone.0001098-Attie1">[26]</a>. The dual regulation of <em>LDLR</em> <a href="#pone.0001098-Attie1">[26]</a> suggests that <em>PCSK9</em> might be involved in a co-evolutionary network of LDL cholesterol metabolism, i.e., variation at one gene evolving with variation at the other. <em>PCSK9</em> is only found in vertebrates, suggesting it is the product of recent evolution <a href="#pone.0001098-Seidah2">[27]</a>. However, it is unclear whether natural selection has driven the evolution of <em>PCSK9</em> in vertebrates, especially in primates.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The ratio of nonsynonymous/synonymous substitution rate (i.e., ω = <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>) provides a sensitive measure to detect selective pressure at the protein level <a href="#pone.0001098-Yang1">[28]</a>. A significantly higher non-synonymous substitution rate than synonymous substitution rate (i.e., <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&gt;1) is evidence for adaptive evolution at the molecular level, whereas <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&lt;1 suggests purifying selection (i.e., selective constraint) <a href="#pone.0001098-Nei1">[29]</a>. This criterion has been used to identify several cases of positive selection, such as the primate stomach lysozyme (<em>LYZ</em>) <a href="#pone.0001098-Messier1">[30]</a>, <a href="#pone.0001098-Yang2">[31]</a>, and <em>BRCA1</em> in humans and chimpanzees <a href="#pone.0001098-Huttley1">[32]</a> (for a list of genes under positive selection in the human lineage, see review by Sabeti et al. <a href="#pone.0001098-Sabeti1">[33]</a>).</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In the present study, we used a phylogeny-based maximum likelihood method to analyze nonsynonymous and synonymous substitution rate of <em>PCSK9</em> sequences across a range of primates, including Hominoids, Old World monkeys, and New World monkeys. We aimed to address three questions: 1) whether the ratio of nonsynonymous and synonymous evolutionary rate of <em>PCSK9</em> varied significantly among various primate lineages, 2) whether there is an episodic adaptive evolution of <em>PCSK9</em> in primates, and 3) which, of any, amino acids of <em>PCSK9</em> were under positive selection.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Comparative analysis of coding regions of PCSK9</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We characterized the coding regions of <em>PCSK9</em> in human and non-human primates (<a href="#pone-0001098-t001">Table 1</a>). There was considerable evidence for interspecies genomic alterations within <em>PCSK9</em> (<a href="#pone-0001098-g002">Figure 2</a>). First, in the signal peptide domain, a variable number of CTG codons [encoding Leucine (Leu, L)] were identified (<a href="#pone-0001098-g002">Figure 2</a>). A species-specific nine Leu repeat (L9) was noted only in humans and chimpanzee; whereas in other species, the number of Leu repeats varied from L6 to L8. In the CTG repeat region, interspecies nucleotide differences were synonymous except nonsynonymous changes in the Old World monkeys (CTG→CCA, 46→48) and in spider monkey (CTG→CGG, 67→69). Second, in the C-terminal domain, species-specific loss-of-function mutation was seen in the clade of New World monkeys (via a premature stop codon in tamarin and dusky titi). We speculate that this nonsense mutation leads to a loss-of-function since the adjacent C679X mutation in humans is a loss-of-function mutation.</p>
<div class="figure" id="pone-0001098-g002"><div class="img"><a name="pone-0001098-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Protein sequence alignment of <em>PCSK9</em> in 14 primates.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>“.” Indicates identity to the first sequence (i.e., human) in each alignment. “-” indicates an alignment gap, and “X” indicates a stop codon. The coordinates after 84 should be minus one to be consistent with that in Human reference sequence (NP_777596), since an insertion at position 84 was present in the dusky titi. The signal peptide (SP) domain (1–90) shows the evolution of Leucine (Leu) repeats (15–23) in <em>PCSK9</em>, and the C-terminal domain shows the premature stop codon (X) in the tamarin (686), and dusky titi (689).</p>
<span>doi:10.1371/journal.pone.0001098.g002</span></div><div class="figure" id="pone-0001098-t001"><div class="img"><a name="pone-0001098-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>A listing of the primates in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001098.t001</span></div>

<h4>Variable d<sub>N</sub>/d<sub>S</sub> ratios for the C-terminal domain of PCSK9 among primate lineages</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A neighbor-joining (nj) phylogenetic tree of <em>PCSK9</em> from 14 primate species was reconstructed from the coding sequence alignment, and the maximum likelihood estimate of the tree topology was acquired using the ‘hyphy’ package <a href="#pone.0001098-Pond1">[34]</a>. We used this gene tree in subsequent analyses to detect whether non-neutral evolution might have operated on <em>PCSK9</em>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Under a one-ratio model, which assumes the same <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> for the entire tree, the cumulative <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> for the coding regions of <em>PCSK9</em> was 0.186 (the log likelihood value <em>l</em><sub>0</sub> = −5109.09). We tested variable <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratios for <em>PCSK9</em> among lineages, using the free-ratio model, which assumes a different <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio for each branch in the phylogeny (<a href="#pone.0001098.s001">Figure S1</a>). The free-ratio model led to a log likelihood <em>l</em><sub>1</sub> = −5097.59. The free-ratio model was not found to be significantly better than the one-ratio model (the statistic 2Δ<em>l</em> = 2(<em>l</em><sub>1</sub>−<em>l</em><sub>0</sub>), <em>P</em> = 0.480, degrees of freedom (df) = 24).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In addition to testing the entire coding regions, it is important to test the structural and functional domains of proteins separately <a href="#pone.0001098-Lynn1">[35]</a>. We evaluated selective pressures in different structural and functional domains by repeating the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> analyses within these domains (the domain structure of <em>PCSK9</em> is shown in <a href="#pone-0001098-g001">Figure 1</a>). For the C-terminal domain of <em>PCSK9</em>, the free-ratio model (<em>l</em><sub>1</sub> = −1351.35) was found to be significantly better than the one-ratio model (<em>l</em><sub>0</sub> = −1369.71, the cumulative <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 0.386) (2Δ<em>l</em> = 36.72, <em>P</em> = 0.047, df = 24), suggesting variable selective constraint across the phylogeny for the C-terminal domain (<a href="#pone-0001098-g003">Figure 3</a>). We did not observe a significant difference between these two models for each of the other four domains and when the four domains were combined (data not shown).</p>
<div class="figure" id="pone-0001098-g003"><div class="img"><a name="pone-0001098-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Ratios of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> estimated for the C-terminal domain of <em>PCSK9</em> in indicated branches of the primate phylogeny.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Values of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> along each branch were calculated by using the free-ratio model using the CODEML program in ‘PAML’ <a href="#pone.0001098-Yang7">[72]</a>. Branch lengths were estimated by maximum likelihood under this model. A <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> value of &gt;1 suggests that positive selection has acted along that lineage. ‘Inf’ indicates cases where <em>d</em><sub>S</sub> = 0. The phylogenetic tree was deduced from the entire coding sequence of <em>PCSK9</em>.</p>
<span>doi:10.1371/journal.pone.0001098.g003</span></div>

<h4>Non-neutral evolution of the PCSK9 C-terminal domain</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Comparison of the rates of nonsynonymous and synonymous DNA changes (i.e., the ratios of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>) between species can be used to assess the types of selective pressures that may have acted on a gene <a href="#pone.0001098-Hurst1">[36]</a>. In the entire <em>PCSK9</em> sequence, <em>d</em><sub>S</sub> exceeded <em>d</em><sub>N</sub> in most of the branches in the primate phylogeny (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&lt;1.0) (<a href="#pone.0001098.s001">Figure S1</a>), indicating that functional constraint (i.e., purifying selection) might have acted on <em>PCSK9</em> throughout primate evolution. In the branch of bonobo and gorilla, the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio was = 1 (i.e., neutral evolution).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We then compared the values of <em>d</em><sub>N</sub> and <em>d</em><sub>S</sub> between species in the C-terminal domain of <em>PCSK9</em> since non-homogeneity in <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio was noted among the primate lineages. We found that many branches of the primate phylogeny, including internal branches, showed evidence of evolution under relaxed selective constraint or positive selection (i.e., <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&gt;1.0) (<a href="#pone-0001098-g003">Figure 3</a>). In the Hominoid clade, <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> values was infinity (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.e001&amp;representation=PNG" class="inline-graphic"></span>: 7.1/0.0) in the lineage leading to orangutan, and 1.218 (5.1/1.1) to the common ancestors of bonobo and gorrila. The <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> in the chimpanzee and human lineages approximated one [0.9890 (4.0/1.1) and 0.5657 (2.0/1.0), respectively], indicating relaxed selective constraint in these two lineages. In addition, we noted <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&gt;1 in lineages leading to colobus and rhesus macaque in the clade of Old World monkeys, as well as spider monkey, squirrel monkey, and the common ancestors of marmoset and tamarin (<a href="#pone-0001098-g003">Figure 3</a>). Thus, the C-terminal domain has been subject to positive selection for at least 33 million years (i.e., the primate divergence time) <a href="#pone.0001098-Goodman1">[37]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Then, by two-ratio likelihood tests using PAML, we tested for the presence of positive selection in the C-terminal domain of <em>PCSK9</em> in the branch of orangutan and the lineage leading to the common ancestors of bonobo and gorilla in the Hominoid clade. Log likelihood values and <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> estimates from each maximum likelihood model were considered, and the likelihood ratio test results are presented in <a href="#pone-0001098-t002">Table 2</a>. The null hypotheses 1–3 were rejected, indicating that the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio in the branch of orangutan is significantly higher than the background ratio of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> (i.e., the null hypothesis of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio homogeneity among lineages was rejected). Although the alternative hypotheses of positive selection (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&gt;1) in the branch of orangutan (null hypotheses 7 and 8 in <a href="#pone-0001098-t002">Table 2</a>) were not accepted at the level of 0.05, the statistical significance was marginal (<em>P</em> = 0.087 and 0.090, respectively). However, we did not observe the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio in the lineage leading to bonobo and gorilla to be significantly different from the background <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio.</p>
<div class="figure" id="pone-0001098-t002"><div class="img"><a name="pone-0001098-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Log likelihood values, parameter estimates under different models, and likelihood ratio statistics (2Δℓ<em></em>) for <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> hypotheses testing.</span></strong></p><span>doi:10.1371/journal.pone.0001098.t002</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To find out whether the C-terminal domain was under relaxed selective constraint or positive selection, we also plotted <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratios estimated by Nei-Gojobori method for pairwise comparisons within the primates (<a href="#pone-0001098-g004">Figure 4</a>). In the whole gene sequence and non C-terminal domains region, <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> was &lt;1 in all pairwise comparisons, indicating a history of functional selective constraint. However, a higher <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> value was noted in the C-terminal domain, including <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&gt;1 in five out of 10 pairwise comparisons within Hominoids. For example, a <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio of 1.122 was noted in the human vs. orangutan comparison.</p>
<div class="figure" id="pone-0001098-g004"><div class="img"><a name="pone-0001098-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Positive selection or relaxed selective constraint of the C-terminal domain of <em>PCSK9</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p><em>d</em><sub>N</sub> is plotted versus <em>d</em><sub>S</sub> for all pairwise combinations of primate sequences. The pairwise ratios of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> were calculated using the Nei-Gojobori method implemented in the package ‘PAML’ <a href="#pone.0001098-Yang7">[72]</a>. Pairwise combinations of Hominoids (HOM), Old World monkeys (OWM), and New World monkeys (NWM) are plotted; for example, ‘Human’ represents the points that are making comparisons between human and another primate. We plotted the entire sequence, non C-terminal domain, and C-terminal domain separately. The higher pairwise <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio in the C-terminal domain suggests that this domain is evolving in a non-neutral model, which maybe due to positive selection or relaxed selective constraint in some lineages. The entire sequence and non C-terminal domain of <em>PCSK9</em> showed a net signature of purifying selection.</p>
<span>doi:10.1371/journal.pone.0001098.g004</span></div>

<h4>Amino acids sites under positive selection</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Finally, we identified the particular codon sites that have been subject to positive selection using the site-models <a href="#pone.0001098-Nielsen1">[38]</a>, <a href="#pone.0001098-Yang3">[39]</a> and the branch-site models <a href="#pone.0001098-Yang4">[40]</a>. <a href="#pone-0001098-t003">Table 3</a> lists the log likelihood values and parameter estimates for the C-terminal domain of <em>PCSK9</em> under several site models and branch-site models. We used two likelihood ratio tests (LRTs) to test for positive selection. In the site models, the first test compared M1a (neutral) against M2a (selection) <a href="#pone.0001098-Nielsen1">[38]</a>, <a href="#pone.0001098-Yang4">[40]</a>, <a href="#pone.0001098-Wong1">[41]</a>, in which 2Δ<em>l</em> is 0.68 (df = 2, <em>P</em>&gt;0.05), and no amino acid sites were under positive selection. The second test compared M7 against M8, in which no sites were shown under positive selection (2Δ<em>l</em> = 1.66, <em>P</em>&gt;0.05). We also used the branch-site model A to detect the codon sites under positive selection by the Bayes Emprical Bayes (BEB) approaches in the lineage of orangutan (i.e., the foreground lineage) <a href="#pone.0001098-Yang4">[40]</a>. The 2Δ<em>l</em> between the null model (neutral, <em>l</em> = −1366.56) and the alternative model (selection, <em>l</em> = −1363.55) is 6.02. The critical values at the 5% and 1% levels for the LRT are 2.71 and 5.41, respectively <a href="#pone.0001098-Self1">[42]</a>. Thus, the test for the branch-site model A is significant (df = 2, <em>P</em>&lt;0.01), indicating presence of codon sites under positive selection (544A, 551H, 556G, 661V, and 681S, <em>P<sub>b</sub></em>&gt;95%) in the C-terminal domain of <em>PCSK9</em>.</p>
<div class="figure" id="pone-0001098-t003"><div class="img"><a name="pone-0001098-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Log likelihood values and parameter estimates under the site models and branch-site models.</span></strong></p><span>doi:10.1371/journal.pone.0001098.t003</span></div>

<h4>Sliding window analysis</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> profiles in the sliding window analysis across <em>PCSK9</em> sequence are shown in <a href="#pone-0001098-g005">Figure 5</a>. As expected, the cumulated <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio in primates in the sliding window analysis appear quite stochastic and bears weak correlation to the domain structure, although the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> is slightly higher in the C-terminal domain. However, in the lineages leading to humans and orangutan, we observed that nonsynonymous substitutions were significantly more concentrated within C-terminal domain of <em>PCSK9</em> (i.e., three <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> peaks). The <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> peaks were consistent between the lineage leading to humans and to orangutan.</p>
<div class="figure" id="pone-0001098-g005"><div class="img"><a name="pone-0001098-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001098" data-uri="info:doi/10.1371/journal.pone.0001098.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001098.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001098.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001098.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Sliding-window analysis of the cumulative <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> across primates (black), the lineage leading to human (green), and orangutan (blue).</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The gene average <em>d</em><sub>S</sub> across primates is 0.5680, in the lineage leading to human is 0.0030, and orangutan is 0.0204.</p>
<span>doi:10.1371/journal.pone.0001098.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The main finding of our study is that there is evidence for functional constraint (i.e., purifying selection) in the coding sequences of <em>PCSK9</em> through primate evolution. We noted that a functional domain of <em>PCSK9</em> (i.e., C-terminal domain) was less conserved at the amino acid level than other gene regions, and likelihood ratio tests (LRTs) revealed evidence of positive selection in the lineage leading to orangutan of the Hominoid clade on this domain. Furthermore, we identified the particular codon sites that have been subject to positive selection in this lineage. We discuss the implications of these comparative sequence data for understanding the evolutionary history of primate <em>PCSK9</em>, hypotheses concerning their role in primate phenotypic evolution, and insights into <em>PCSK9</em>-associated human diseases.</p>

<h4>Evolutionary history of Leucine (L) repeats in the signal peptide (SP) domain and premature stop codon in the C-terminal domain</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Comparative sequence analysis revealed a dynamic evolutionary history of leucine (Leu, L) repeats in the signal peptide domain. The number of Leu repeats varied from L9 to L6 among different clades (<a href="#pone-0001098-g002">Figure 2</a>). An additional in-frame insertion (CTG) leading to a L9→L10 polymorphism in African-Americans and Caucasians <a href="#pone.0001098-Yue1">[19]</a> is associated with hypocholesterolemia. We speculate that the number of Leu repeat may influence levels of the <em>PCSK9</em> protein and thereby levels of LDL cholesterol, although this needs confirmation.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In the C-terminal domain of <em>PCSK9</em>, a premature stop codon was seen in the New World monkeys – tamarin and dusky titi (<a href="#pone-0001098-g002">Figure 2</a>) – but not in the Hominoids. As mentioned before, monkeys have significantly lower LDL cholesterol levels than Hominoids <a href="#pone.0001098-Crissey1">[22]</a>. It is unknown whether the loss-of-function mutation in the C-terminal domain is a random phenomenon or a common feature that influences cholesterol metabolism. A premature stop codon mutation in human <em>PCSK9</em> (C679X) is considered to be under positive selection, and it is speculated that loss of <em>PCSK9</em> function interferes with the life cycle of the malaria parasite through cholesterol restriction <a href="#pone.0001098-Horton2">[43]</a>, <a href="#pone.0001098-Mbikay1">[44]</a>. The “less-is-more” hypothesis of Olson <a href="#pone.0001098-Olson1">[45]</a> posits that loss of gene function during hominoid evolution may in some cases have conferred a fitness benefit and led to adaptive evolution that may help explain differences among primates <a href="#pone.0001098-Perry1">[46]</a>.</p>


<h4>Amino acid substitution patterns</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio for the coding regions of <em>PCSK9</em> across the primate species was &lt;1 (cumulative <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 0.186). There were no lineages with <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&gt;1, and <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratios did not vary among branches (<em>P</em> = 0.480, <a href="#pone.0001098.s001">Figure S1</a>). This is not unexpected, as averaging <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> across all sites is not a powerful test of adaptive evolution <a href="#pone.0001098-Yang3">[39]</a>. However, the nonsynonymous substitution rate in the C-terminal domain was significantly higher than in other domains. The cumulative <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 0.386 is higher than that in the entire coding region (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 0.186), suggesting that different selection pressures have acted on amino acid changes across different functional regions of this gene.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The hypothesis of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> homogeneity among branches was rejected for the C-terminal domain (<em>P</em> = 0.047, <a href="#pone-0001098-g003">Figure 3</a>), which could reflect either relaxed selective constraint or positive selection for amino acid substitution along one or more lineages. We were particularly interested in two lineages in the Hominoid clade leading to orangutan (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = infinity), as well as the common ancestor of bonobo and gorilla (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 1.213), and tested whether the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio was significantly &gt;1 on these two branches. Likelihood ratio tests (LRTs) from a two-ratio model revealed that positive selection (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>&gt;1) had acted on the orangutan lineage although the statistical evidence was marginal (<em>P</em> = 0.087 and <em>P</em> = 0.090) (<a href="#pone-0001098-t002">Table 2</a>). However, the pattern of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> heterogeneity across lineages is consistent with a relaxed selective constraint. The recently developed branch-site model A is powered to detect the particular amino acid sites that have been subject to positive selection in a given lineage (i.e., a foreground branch) <a href="#pone.0001098-Yang4">[40]</a>, <a href="#pone.0001098-Zhang1">[47]</a>, and at least five positively selected amino acids of the C-terminal domain existed in the lineage leading to orangutan (<a href="#pone-0001098-t003">Table 3</a>). No amino acids under positive selection were detected using site models (<a href="#pone-0001098-t003">Table 3</a>). It should be noted that the power of the LRTs is dependent on the number of coding sequences <a href="#pone.0001098-Anisimova1">[48]</a>. We sampled 14 primate species in Hominoid, Old World monkey, and New World monkey clades in our phylogenetic analyses (<a href="#pone-0001098-t001">Table 1</a>). A greater number of the species might have permitted more robust inferences of positive selection on the C-terminal domain of <em>PCSK9</em>.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The sliding-window analysis ratio further characterized the non-random nonsynonymous substitution along <em>PCSK9</em> and <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> peaks were obvious in the C-terminal domain (<a href="#pone-0001098-g005">Figure 5</a>). Although the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio in the lineage of humans is &lt;1 (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 0.566), three striking peaks of <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> (&gt; 4) in human lineage were noted in the C-terminal domain. However, these peaks could be partly explained on the basis that human <em>PCSK9</em> shows very little synonymous divergence (<em>d</em><sub>S</sub> = 0.003).</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>In the present study, we calculated the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratios across the phylogenetic tree using the ‘gene’ tree instead of the ‘species’ tree. We also performed analyses using the ‘species’ tree <a href="#pone.0001098-Purvis1">[49]</a>, in which bonobo is most closely related to chimpanzee and gorilla is sister to the human/chimpanzee clade. Although the log likelihood values (<em>l</em><sub>0</sub> for one-ratio model and <em>l</em><sub>1</sub> for free-ratio model) under the ‘species’ tree were different from that under the ‘gene’ tree, we did not find a significant difference in the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio between the ‘species’ tree and ‘gene’ tree for the coding regions and C-terminal domain. Although we did not observe <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio &gt;1 in the lineage leading to bonobo or gorilla in the C-terminal domain, we did note <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = infinity in the lineage leading to orangutan. In addition, we used the branch-site A model to test for positive selection in the lineage leading to orangutan based on the species tree. The 2Δ<em>l</em> between the null model (neutral, <em>l</em> = −1408.2) and the alternative model (selection, <em>l</em> = −1406.7) is 3.0 (df = 2, <em>P</em>&lt;0.05) (critical value is 2.71 at 5% significance level <a href="#pone.0001098-Self1">[42]</a>). We detected three codon sites under positive selection (544A, 551H, and 681S, <em>P<sub>b</sub></em>&gt;95%) in the C-terminal domain of <em>PCSK9</em> (using BEB analysis). Zhang et al. <a href="#pone.0001098-Zhang1">[47]</a> suggested a critical value of 3.84 (for <em>P</em>&lt;0.05), however, such a threshold may be too conservative for a sequence length of 200 codons <a href="#pone.0001098-Zhang1">[47]</a>.</p>


<h4>Structural and functional implication of PCSK9</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The correct folding of the C-terminal domain is crucial for <em>PCSK9</em> function but catalytic activity is not required for <em>PCSK9</em> to bind and degrade LDL-R in cultured human hepatoma cells <a href="#pone.0001098-McNutt1">[50]</a>. The C-terminal domain of a proprotein convertase contains unique sequences regulating their cellular localization and trafficking <a href="#pone.0001098-Seidah3">[51]</a>. For example, <em>PCSK9</em> exhibits a Cys-His-rich domain that is required for cell surface binding in an LDL-R-dependent fashion <a href="#pone.0001098-Seidah1">[6]</a> and plays a role in the regulation of auto-processing of <em>PCSK9</em>. The structural characteristic of C-terminal domain may determine the colocalization of <em>PCSK9</em> with LDLR at the cell surface <a href="#pone.0001098-Nassoury1">[52]</a> or lead to other novel functional properties. Hence, positive selection operating on the C-terminal domain was most likely directed at creating novel biochemical properties.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Species-specific differences in <em>PCSK9</em> expression patterns have been noted in brain and liver among humans, chimpanzee, and orangutan <a href="#pone.0001098-Enard1">[53]</a>. <em>PCSK9</em> is transiently expressed during embryonic development in telencephalon and cerebellum where <em>LDLR</em> expression is not prominent <a href="#pone.0001098-Seidah1">[6]</a>, <a href="#pone.0001098-Poirier1">[54]</a>. Specific knockdown of <em>Pcsk9</em> mRNA led to embryonic death at 4 days after fertilization in zebrafish <a href="#pone.0001098-Poirier1">[54]</a>, and complete knockout of <em>Pcsk9</em> in mouse led to a ~50% reduction in circulating levels of LDL cholesterol, but did not result in a lethal phenotype <a href="#pone.0001098-Rashid1">[55]</a>. Over-expression of <em>PCSK9</em> induces apoptosis in neural development <a href="#pone.0001098-Seidah1">[6]</a>, <a href="#pone.0001098-Bingham1">[56]</a>, which results in a higher percentage of differentiated neurons and promotes cortical neurogenesis. These results indicate a novel function of <em>PCSK9</em> in central nervous system development, distinct from that in cholesterogenic organs such as liver <a href="#pone.0001098-Poirier1">[54]</a>. One could hypothesize that relaxed selective constraint or positive selection has operated on the C-terminal domain of <em>PCSK9</em> due to the key role of <em>PCSK9</em> in early brain development.</p>


<h4>Implications for human diseases</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>There is increasing interest in identifying gene loci affected by natural selection since they are medically important <a href="#pone.0001098-Hahn1">[57]</a>–<a href="#pone.0001098-Kullo1">[63]</a>. Loss-of-function or gain-of-function mutations in <em>PCSK9</em> have been reported to be associated with significant alterations in plasma levels of LDL cholesterol (<a href="#pone-0001098-g001">Figure 1</a>). Both evolutionary conservation indicating negative purifying selection and accelerated evolution driven by positive selection signify functionally significant regions of the genome <a href="#pone.0001098-Thomas1">[64]</a>. To assess the potential severity of human <em>PCSK9</em> mutations, we assessed the levels of conservation or divergence of non-synonymous mutations listed in <a href="#pone-0001098-g001">Figure 1</a> by aligning the amino acids among 14 species (<a href="#pone-0001098-g002">Figure 2</a>). We expected that the amino acids known to be important for <em>PCSK9</em> function (i.e., residues at which disease-causing mutations occur) would be highly conserved. All gain-of-function mutations in <em>PCSK9</em> leading to hypercholesterolemia in humans are 100% conserved at the amino acid level across all the primates we sampled. In case of loss-of-function mutations leading to hypocholesterolemia, all but two (E57K and Q554E) are 100% conserved across the primates. Mutations leading to both hypercholesterolemia and hypocholesterolemia appear to be less conserved, since four out of seven such mutations are not all conserved, including A53V, I474V, Q554E, and E670G. We noted a striking pattern of I474V variation (SNP rs562556) across the primates. The ancestral state of the 474<sup>th</sup> amino acid (M or V) in New World monkeys is not clear given the lack of an outgroup. The ‘V’ allele diverged to ‘I’ or ‘V’ in the Hominoid clade, suggesting a dynamic evolutionary history of the 474<sup>th</sup> amino acid. The human mutation I to V replicates the ancestral state, and the recurrence of this ancestral state has functional consequences <a href="#pone.0001098-Schaner1">[65]</a>.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>To survey polymorphisms within human populations, we analyzed <em>PCSK9</em> SNPs using resequenced data from 24 African-Americans and 23 European-Americans (i.e., 47 individuals) in SeattleSNPs database (pga.gs.washington.edu/). A total of 229 polymorphic sites in African-Americans and 125 polymorphic sites in European-Americans were found in the human panel, eight of which resulted in amino acid changes. In addition, an in-frame insertion/deletion (CTG) in the signal-peptide domain was noted in both populations. Six of the eight non-synonymous sites are located in the putative domain and C-terminal domain (some investigators combine these two domains as ‘C-terminal’ domain), corresponding to the regions that have been predicted to be under positive selection. None of the non-synonymous sites was found in the catalytic domain. We used SIFT <a href="#pone.0001098-Ng1">[66]</a> and PolyPhen <a href="#pone.0001098-Ramensky1">[67]</a> to predict the effect of the amino acid changes (<a href="#pone.0001098.s003">Table S2</a>). In case of amino acid 474, the nonsynonymous substitution was predicted to be damaging (i.e., cause functional alteration), but the derived allele frequency of 0.79 in African-Americans and 0.87 in European-Americans suggests that positive selection acted to increase the frequency of this polymorphism. In humans, a signature of recent positive selection was noted on this common variation using long-range haplotype (LRH) test; that is, positive selection had acted on the derived allele ‘I’ in African-Americans and the ancestral allele ‘V’ in European-Americans (Ding and Kullo, manuscript in revision). In addition, a signature of positive selection on the derived allele of E670G (rs505151), which resides in the C-terminal domain, was also noted in African-Americans. We speculate that non-conserved mutations across the primates might be the substrate for non-neutral evolution and responsible for the phenotypic variation in the general population.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>In conclusion, phylogenetic analysis of the cholesterol metabolism gene <em>PCSK9</em> across a range of primates reveals lineage-specific patterns of variation. Although the gain-of-function mutations at <em>PCSK9</em> reflect strong functional constraint and a history of purifying selection, a signature of relaxed selective constraint or positive selection was noted in the C-terminal domain of <em>PCSK9</em>. It is possible that different modes of selection have operated on different functional domains of <em>PCSK9</em>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Primate Genomic DNA Sources</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The comparative sequences of the <em>PCSK9</em> coding regions were obtained in 14 species from three sources. First, the human (accession no.: NM_174936) and chimpanzee (XM_427085) mRNA sequences of <em>PCSK9</em> were downloaded from NCBI (<a href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</a>). Next, we acquired the BAC (bacterial artificial chromosome) clone sequence including <em>PCSK9</em> from Programs for Genomic Application (PGA) at Berkeley (pga.lbl.gov/seq), including colobus (AC188217), dusky titi (AC188268), squirrel monkey (AC188233), and marmoset (AC188221). Coding regions of <em>PCSK9</em> for these species were extracted by aligning the human mRNA sequence to the BAC sequence using the ‘sim4’ program <a href="#pone.0001098-Florea1">[68]</a>. Finally, DNA samples for a primate panel, including rhesus macaque, pigtailed macaque, bonobo, gorilla, chimpanzee, orangutan, tamarin, spider monkey, woolly monkey, and lemur, were obtained from Coriell Cell Repositories (Camden, NJ). The species name and scientific name for each species are listed in <a href="#pone-0001098-t001">Table 1</a>.</p>


<h4>Sequencing of PCSK9 exons from Primate Genomic DNA</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>In the primate panel, <em>PCSK9</em> was amplified and sequenced exon by exon from genomic DNA with high fidelity polymerase chain reaction (PCR). Primers and PCR conditions are listed in <a href="#pone.0001098.s002">Table S1</a>. PCR products were sequenced directly in both forward and reverse directions. Exon reads were assembled together to create virtual transcript for each primate using the Sequencher® program (version 4.5, <a href="http://www.genecodes.com">www.genecodes.com</a>) and visually checked for accuracy. The lemur <em>PCSK9</em> sequence could not be obtained due to difficulty in PCR amplification. Sequences of coding regions for eight species in this primate panel were obtained. A total of 2072 bp of <em>PCSK9</em> coding sequence (the length is based on the human sequence and excludes the stop codon) in 14 species was compiled. All sequences have been submitted to the GenBank database under the accession nos. EF692496–EF692509 (<a href="#pone-0001098-t001">Table 1</a>).</p>


<h4>Detecting lineage-specific episodes of positive selection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Sequences were aligned using ClustalW <a href="#pone.0001098-Thompson2">[69]</a>, followed by manual inspection and analysis. We used the ‘HYPHY’ package to estimate the topology of phylogenetic tree using the maximum likelihood method <a href="#pone.0001098-Pond1">[34]</a>. Since the gene tree was different from the species tree, analyses were done based on gene tree as well as the species tree.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>We used the maximum likelihood method based on codon-substitution model by Yang <a href="#pone.0001098-Yang1">[28]</a>, <a href="#pone.0001098-Yang2">[31]</a>, <a href="#pone.0001098-Yang5">[70]</a> to test whether there was a significant difference in <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio (i.e., ω) among lineages and whether <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> was significantly &gt;1 (i.e., positive selection) in a given lineage. The ‘one-ratio’ model assumes the same ratio for all branches in the phylogeny. The most general model – ‘free-ratio’ model – assumes an independent <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio for each branch in the phylogeny. If there is a phylogenetic tree of many species, this model involves as many <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> parameters as the number of branches in the tree. The models used in the phylogenetic analysis can be compared using the likelihood-ratio test to examine interesting hypotheses <a href="#pone.0001098-Yang2">[31]</a>. The null hypothesis is the ‘one-ratio’ model, and can be used to test whether there is a differential <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio among lineages. Positive selection or relaxed selective constraint in some lineages could contribute to the heterogeneity in the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio.</p>


<h4>Detecting amino acid sites under positive selection</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The above methods for lineage-specific selection assume that all amino acid sites have the same <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio, i.e., averages the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio across all sites. Since many amino sites might be under strong purifying selection due to functional constraint (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub>≈zero) and positive selection often operates episodically on a few amino acid sites <a href="#pone.0001098-Zhang1">[47]</a>, it seems likely that this is a more conservative test and amino acid sites under positive selection cannot be detected.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Several methods have been developed to address this problem, such as the site models which allow <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> to vary among codons <a href="#pone.0001098-Nielsen1">[38]</a>, <a href="#pone.0001098-Yang6">[71]</a>. In the present study, we also used an improved branch-site likelihood method to detect positive selection at the amino acid sites <a href="#pone.0001098-Yang1">[28]</a>, <a href="#pone.0001098-Yang4">[40]</a>, <a href="#pone.0001098-Zhang1">[47]</a>. This branch-site model <a href="#pone.0001098-Yang1">[28]</a>, <a href="#pone.0001098-Yang4">[40]</a>, <a href="#pone.0001098-Zhang1">[47]</a> assumed that the branches on the phylogeny are divided <em>a priori</em> into foreground (i.e, may have experienced positive selection) and background lineages. We used the likelihood-ratio test 2 (i.e., the branch-site test of positive selection) constructed from this branch-site model <a href="#pone.0001098-Zhang1">[47]</a>. The null hypothesis of this LRT is the branch-site model A list above but with ω<sub>2</sub> fixed = 1, which can be used to directly test for positive selection on the foreground lineages <a href="#pone.0001098-Zhang1">[47]</a>. The Bayes empirical Bayes (BEB) approach was used to calculate the posterior probabilities that a codon belongs to the site class of positive selection on the foreground lineages <a href="#pone.0001098-Yang4">[40]</a>. The test should be compared with the 50:50 mixture of point mass 0 and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.e002&amp;representation=PNG" class="inline-graphic"></span> (with critical values to be 2.71 and 5.41, at the 5% and 1% significance levels, respectively) <a href="#pone.0001098-Self1">[42]</a>. Zhang et al <a href="#pone.0001098-Zhang1">[47]</a> also suggested the use of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001098.e003&amp;representation=PNG" class="inline-graphic"></span> distribution for assessing the significance of the test (3.84 and 5.99 at the 5% and 1% significance levels, respectively). This LRT test seemed conservative overall, but exhibited better power in detecting positive selection than the branch-based test <a href="#pone.0001098-Zhang1">[47]</a>.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>We used the ‘CODEML’ program in PAML version 3.15 <a href="#pone.0001098-Yang7">[72]</a> to calculate the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio and perform the maximum likelihood phylogenetic analysis. To calculate the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio at lineages (defined as all branches in the phylogeny, both terminal species nodes and internodes), sequences associated with species-specific premature stop codons were removed.</p>


<h4>Sliding-window analysis of d<sub>N</sub>/d<sub>S</sub></h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Sliding-window analysis of <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> was performed with a window size of 90 bp (30 codons) and a sliding increment of 15 bp (5 codons). We used the approach by Choi and Lahn <a href="#pone.0001098-Choi1">[73]</a> to calculate the <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> of each window as the ratio between window-specific <em>d</em><sub>N</sub> and gene-average <em>d</em><sub>S</sub>, since noise in window-specific <em>d</em><sub>S</sub> can sometimes hamper the analysis. In addition, the use of gene-average instead of window-specific <em>d</em><sub>S</sub> should not introduce any systematic bias <a href="#pone.0001098-Choi1">[73]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001098"></div><a name="pone.0001098.s001" id="pone.0001098.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001098.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogeny of coding regions of <em>PCSK9</em>. <em>PCSK9</em> was resequenced from a panel of primates including Hominoids, Old World monkeys, and New World monkeys. Branch lengths were estimated by maximum likelihood under the free-ratio model, which assumes an independent <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio for each branch.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001098.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB EPS)</p>
<a name="pone.0001098.s002" id="pone.0001098.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001098.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Polymerase chain reaction (PCR) primers and conditions for <em>PCSK9</em> exon analyses</p>
<p class="siDoi">doi:10.1371/journal.pone.0001098.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0001098.s003" id="pone.0001098.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001098.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">SIFT and Polyphen prediction of amino acid polymorphisms</p>
<p class="siDoi">doi:10.1371/journal.pone.0001098.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We acknowledge the technical support of Mayo Research Computing Facility and the Supercomputing Institute of University of Minnesota, Minneapolis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KD IK. Performed the experiments: SM. Analyzed the data: KD IK. Wrote the paper: KD IK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001098-Espenshade1" id="pone.0001098-Espenshade1"></a>Espenshade PJ, Cheng D, Goldstein JL, Brown MS (1999) Autocatalytic processing of site-1 protease removes propeptide and permits cleavage of sterol regulatory element-binding proteins. J Biol Chem  274: 22795–22804.  <ul class="find" data-citedArticleID="1010978" data-doi="10.1074/jbc.274.32.22795"><li><a href="http://dx.doi.org/10.1074/jbc.274.32.22795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autocatalytic+processing+of+site-1+protease+removes+propeptide+and+permits+cleavage+of+sterol+regulatory+element-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autocatalytic+processing+of+site-1+protease+removes+propeptide+and+permits+cleavage+of+sterol+regulatory+element-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001098-Brown1" id="pone.0001098-Brown1"></a>Brown MS, Goldstein JL (1999) A proteolytic pathway that controls the cholesterol content of membranes, cells, and blood. Proc Natl Acad Sci U S A  96: 11041–11048.  <ul class="find" data-citedArticleID="1010960" data-doi="10.1073/pnas.96.20.11041"><li><a href="http://dx.doi.org/10.1073/pnas.96.20.11041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+proteolytic+pathway+that+controls+the+cholesterol+content+of+membranes%2C+cells%2C+and+blood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+proteolytic+pathway+that+controls+the+cholesterol+content+of+membranes%2C+cells%2C+and+blood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001098-Elagoz1" id="pone.0001098-Elagoz1"></a>Elagoz A, Benjannet S, Mammarbassi A, Wickham L, Seidah NG (2002) Biosynthesis and cellular trafficking of the convertase SKI-1/S1P: ectodomain shedding requires SKI-1 activity. J Biol Chem  277: 11265–11275.  <ul class="find" data-citedArticleID="1010974" data-doi="10.1074/jbc.m109011200"><li><a href="http://dx.doi.org/10.1074/jbc.m109011200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthesis+and+cellular+trafficking+of+the+convertase+SKI-1%2FS1P%3A+ectodomain+shedding+requires+SKI-1+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthesis+and+cellular+trafficking+of+the+convertase+SKI-1%2FS1P%3A+ectodomain+shedding+requires+SKI-1+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001098-Benjannet1" id="pone.0001098-Benjannet1"></a>Benjannet S, Rhainds D, Essalmani R, Mayne J, Wickham L, et al.  (2004) NARC-1/PCSK9 and its natural mutants: zymogen cleavage and effects on the low density lipoprotein (LDL) receptor and LDL cholesterol. J Biol Chem  279: 48865–48875.  <ul class="find" data-citedArticleID="1010954" data-doi="10.1074/jbc.m409699200"><li><a href="http://dx.doi.org/10.1074/jbc.m409699200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NARC-1%2FPCSK9+and+its+natural+mutants%3A+zymogen+cleavage+and+effects+on+the+low+density+lipoprotein+%28LDL%29+receptor+and+LDL+cholesterol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NARC-1%2FPCSK9+and+its+natural+mutants%3A+zymogen+cleavage+and+effects+on+the+low+density+lipoprotein+%28LDL%29+receptor+and+LDL+cholesterol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001098-Naureckiene1" id="pone.0001098-Naureckiene1"></a>Naureckiene S, Ma L, Sreekumar K, Purandare U, Lo CF, et al.  (2003) Functional characterization of Narc 1, a novel proteinase related to proteinase K. Arch Biochem Biophys  420: 55–67.  <ul class="find" data-citedArticleID="1011020" data-doi="10.1016/j.abb.2003.09.011"><li><a href="http://dx.doi.org/10.1016/j.abb.2003.09.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+characterization+of+Narc+1%2C+a+novel+proteinase+related+to+proteinase+K." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+characterization+of+Narc+1%2C+a+novel+proteinase+related+to+proteinase+K.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001098-Seidah1" id="pone.0001098-Seidah1"></a>Seidah NG, Benjannet S, Wickham L, Marcinkiewicz J, Jasmin SB, et al.  (2003) The secretory proprotein convertase neural apoptosis-regulated convertase 1 (NARC-1): liver regeneration and neuronal differentiation. Proc Natl Acad Sci U S A  100: 928–933.  <ul class="find" data-citedArticleID="1011050" data-doi="10.1073/pnas.0335507100"><li><a href="http://dx.doi.org/10.1073/pnas.0335507100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+secretory+proprotein+convertase+neural+apoptosis-regulated+convertase+1+%28NARC-1%29%3A+liver+regeneration+and+neuronal+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+secretory+proprotein+convertase+neural+apoptosis-regulated+convertase+1+%28NARC-1%29%3A+liver+regeneration+and+neuronal+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001098-Lalanne1" id="pone.0001098-Lalanne1"></a>Lalanne F, Lambert G, Amar MJ, Chetiveaux M, Zair Y, et al.  (2005) Wild-type PCSK9 inhibits LDL clearance but does not affect apoB-containing lipoprotein production in mouse and cultured cells. J Lipid Res  46: 1312–1319.  <ul class="find" data-citedArticleID="1011000" data-doi="10.1194/jlr.m400396-jlr200"><li><a href="http://dx.doi.org/10.1194/jlr.m400396-jlr200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wild-type+PCSK9+inhibits+LDL+clearance+but+does+not+affect+apoB-containing+lipoprotein+production+in+mouse+and+cultured+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wild-type+PCSK9+inhibits+LDL+clearance+but+does+not+affect+apoB-containing+lipoprotein+production+in+mouse+and+cultured+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001098-Maxwell1" id="pone.0001098-Maxwell1"></a>Maxwell KN, Breslow JL (2004) Adenoviral-mediated expression of Pcsk9 in mice results in a low-density lipoprotein receptor knockout phenotype. Proc Natl Acad Sci U S A  101: 7100–7105.  <ul class="find" data-citedArticleID="1011006" data-doi="10.1073/pnas.0402133101"><li><a href="http://dx.doi.org/10.1073/pnas.0402133101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adenoviral-mediated+expression+of+Pcsk9+in+mice+results+in+a+low-density+lipoprotein+receptor+knockout+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adenoviral-mediated+expression+of+Pcsk9+in+mice+results+in+a+low-density+lipoprotein+receptor+knockout+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001098-Park1" id="pone.0001098-Park1"></a>Park SW, Moon YA, Horton JD (2004) Post-transcriptional regulation of low density lipoprotein receptor protein by proprotein convertase subtilisin/kexin type 9a in mouse liver. J Biol Chem  279: 50630–50638.  <ul class="find" data-citedArticleID="1011030" data-doi="10.1074/jbc.m410077200"><li><a href="http://dx.doi.org/10.1074/jbc.m410077200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Post-transcriptional+regulation+of+low+density+lipoprotein+receptor+protein+by+proprotein+convertase+subtilisin%2Fkexin+type+9a+in+mouse+liver." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Post-transcriptional+regulation+of+low+density+lipoprotein+receptor+protein+by+proprotein+convertase+subtilisin%2Fkexin+type+9a+in+mouse+liver.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001098-Abifadel1" id="pone.0001098-Abifadel1"></a>Abifadel M, Varret M, Rabes JP, Allard D, Ouguerram K, et al.  (2003) Mutations in PCSK9 cause autosomal dominant hypercholesterolemia. Nat Genet  34: 154–156.  <ul class="find" data-citedArticleID="1010948" data-doi="10.1038/ng1161"><li><a href="http://dx.doi.org/10.1038/ng1161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+PCSK9+cause+autosomal+dominant+hypercholesterolemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+PCSK9+cause+autosomal+dominant+hypercholesterolemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001098-Leren1" id="pone.0001098-Leren1"></a>Leren TP (2004) Mutations in the PCSK9 gene in Norwegian subjects with autosomal dominant hypercholesterolemia. Clin Genet  65: 419–422.  <ul class="find" data-citedArticleID="1011002" data-doi="10.1111/j.0009-9163.2004.0238.x"><li><a href="http://dx.doi.org/10.1111/j.0009-9163.2004.0238.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+the+PCSK9+gene+in+Norwegian+subjects+with+autosomal+dominant+hypercholesterolemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+the+PCSK9+gene+in+Norwegian+subjects+with+autosomal+dominant+hypercholesterolemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001098-Timms1" id="pone.0001098-Timms1"></a>Timms KM, Wagner S, Samuels ME, Forbey K, Goldfine H, et al.  (2004) A mutation in PCSK9 causing autosomal-dominant hypercholesterolemia in a Utah pedigree. Hum Genet  114: 349–353.  <ul class="find" data-citedArticleID="1011070" data-doi="10.1007/s00439-003-1071-9"><li><a href="http://dx.doi.org/10.1007/s00439-003-1071-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mutation+in+PCSK9+causing+autosomal-dominant+hypercholesterolemia+in+a+Utah+pedigree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mutation+in+PCSK9+causing+autosomal-dominant+hypercholesterolemia+in+a+Utah+pedigree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001098-Sun1" id="pone.0001098-Sun1"></a>Sun XM, Eden ER, Tosi I, Neuwirth CK, Wile D, et al.  (2005) Evidence for effect of mutant PCSK9 on apolipoprotein B secretion as the cause of unusually severe dominant hypercholesterolaemia. Hum Mol Genet  14: 1161–1169.  <ul class="find" data-citedArticleID="1011060" data-doi="10.1093/hmg/ddi128"><li><a href="http://dx.doi.org/10.1093/hmg/ddi128" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+effect+of+mutant+PCSK9+on+apolipoprotein+B+secretion+as+the+cause+of+unusually+severe+dominant+hypercholesterolaemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+effect+of+mutant+PCSK9+on+apolipoprotein+B+secretion+as+the+cause+of+unusually+severe+dominant+hypercholesterolaemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001098-Naoumova1" id="pone.0001098-Naoumova1"></a>Naoumova RP, Tosi I, Patel D, Neuwirth C, Horswell SD, et al.  (2005) Severe hypercholesterolemia in four British families with the D374Y mutation in the PCSK9 gene: long-term follow-up and treatment response. Arterioscler Thromb Vasc Biol  25: 2654–2660.  <ul class="find" data-citedArticleID="1011016" data-doi="10.1161/01.atv.0000190668.94752.ab"><li><a href="http://dx.doi.org/10.1161/01.atv.0000190668.94752.ab" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Severe+hypercholesterolemia+in+four+British+families+with+the+D374Y+mutation+in+the+PCSK9+gene%3A+long-term+follow-up+and+treatment+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Severe+hypercholesterolemia+in+four+British+families+with+the+D374Y+mutation+in+the+PCSK9+gene%3A+long-term+follow-up+and+treatment+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001098-Shioji1" id="pone.0001098-Shioji1"></a>Shioji K, Mannami T, Kokubo Y, Inamoto N, Takagi S, et al.  (2004) Genetic variants in PCSK9 affect the cholesterol level in Japanese. J Hum Genet  49: 109–114.  <ul class="find" data-citedArticleID="1011058" data-doi="10.1007/s10038-003-0114-3"><li><a href="http://dx.doi.org/10.1007/s10038-003-0114-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variants+in+PCSK9+affect+the+cholesterol+level+in+Japanese." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variants+in+PCSK9+affect+the+cholesterol+level+in+Japanese.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001098-Cohen1" id="pone.0001098-Cohen1"></a>Cohen J, Pertsemlidis A, Kotowski IK, Graham R, Garcia CK, et al.  (2005) Low LDL cholesterol in individuals of African descent resulting from frequent nonsense mutations in PCSK9. Nat Genet  37: 161–165.  <ul class="find" data-citedArticleID="1010966" data-doi="10.1038/ng1509"><li><a href="http://dx.doi.org/10.1038/ng1509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Low+LDL+cholesterol+in+individuals+of+African+descent+resulting+from+frequent+nonsense+mutations+in+PCSK9." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Low+LDL+cholesterol+in+individuals+of+African+descent+resulting+from+frequent+nonsense+mutations+in+PCSK9.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001098-Kotowski1" id="pone.0001098-Kotowski1"></a>Kotowski IK, Pertsemlidis A, Luke A, Cooper RS, Vega GL, et al.  (2006) A spectrum of PCSK9 alleles contributes to plasma levels of low-density lipoprotein cholesterol. Am J Hum Genet  78: 410–422.  <ul class="find" data-citedArticleID="1010996" data-doi="10.1086/500615"><li><a href="http://dx.doi.org/10.1086/500615" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+spectrum+of+PCSK9+alleles+contributes+to+plasma+levels+of+low-density+lipoprotein+cholesterol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+spectrum+of+PCSK9+alleles+contributes+to+plasma+levels+of+low-density+lipoprotein+cholesterol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001098-Berge1" id="pone.0001098-Berge1"></a>Berge KE, Ose L, Leren TP (2006) Missense mutations in the PCSK9 gene are associated with hypocholesterolemia and possibly increased response to statin therapy. Arterioscler Thromb Vasc Biol  26: 1094–1100.  <ul class="find" data-citedArticleID="1010956" data-doi="10.1161/01.atv.0000204337.81286.1c"><li><a href="http://dx.doi.org/10.1161/01.atv.0000204337.81286.1c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Missense+mutations+in+the+PCSK9+gene+are+associated+with+hypocholesterolemia+and+possibly+increased+response+to+statin+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Missense+mutations+in+the+PCSK9+gene+are+associated+with+hypocholesterolemia+and+possibly+increased+response+to+statin+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001098-Yue1" id="pone.0001098-Yue1"></a>Yue P, Averna M, Lin X, Schonfeld G (2006) The c.43_44insCTG variation in PCSK9 is associated with low plasma LDL-cholesterol in a Caucasian population. Hum Mutat  27: 460–466.  <ul class="find" data-citedArticleID="1011090" data-doi="10.1002/humu.20316"><li><a href="http://dx.doi.org/10.1002/humu.20316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+c.43_44insCTG+variation+in+PCSK9+is+associated+with+low+plasma+LDL-cholesterol+in+a+Caucasian+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+c.43_44insCTG+variation+in+PCSK9+is+associated+with+low+plasma+LDL-cholesterol+in+a+Caucasian+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001098-Cohen2" id="pone.0001098-Cohen2"></a>Cohen JC, Boerwinkle E, Mosley TH Jr, Hobbs HH (2006) Sequence variations in PCSK9, low LDL, and protection against coronary heart disease. N Engl J Med  354: 1264–1272.  <ul class="find" data-citedArticleID="1010968" data-doi="10.1056/nejmoa054013"><li><a href="http://dx.doi.org/10.1056/nejmoa054013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+variations+in+PCSK9%2C+low+LDL%2C+and+protection+against+coronary+heart+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+variations+in+PCSK9%2C+low+LDL%2C+and+protection+against+coronary+heart+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001098-Fleagle1" id="pone.0001098-Fleagle1"></a>Fleagle JG, McGraw WS (1999) Skeletal and dental morphology supports diphyletic origin of baboons and mandrills. Proc Natl Acad Sci U S A  96: 1157–1161.  <ul class="find" data-citedArticleID="1010980" data-doi="10.1073/pnas.96.3.1157"><li><a href="http://dx.doi.org/10.1073/pnas.96.3.1157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Skeletal+and+dental+morphology+supports+diphyletic+origin+of+baboons+and+mandrills." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Skeletal+and+dental+morphology+supports+diphyletic+origin+of+baboons+and+mandrills.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001098-Crissey1" id="pone.0001098-Crissey1"></a>Crissey S, Barr J, Slifka K, Bowen P, Stacewicz-Sapuntzakis M, et al.  (1999) Serum concentrations of lipids, vitamins A and E, vitamin D metabolites, and carotenoids in nine primate species at four zoos. Zoo biology  18: 551–564.  <ul class="find" data-citedArticleID="1010970" data-doi="10.1002/(sici)1098-2361(1999)18:6<551::aid-zoo9&gt;3.0.co;2-s"><li><a href="http://dx.doi.org/10.1002/(sici)1098-2361(1999)18:6<551::aid-zoo9&gt;3.0.co;2-s" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serum+concentrations+of+lipids%2C+vitamins+A+and+E%2C+vitamin+D+metabolites%2C+and+carotenoids+in+nine+primate+species+at+four+zoos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serum+concentrations+of+lipids%2C+vitamins+A+and+E%2C+vitamin+D+metabolites%2C+and+carotenoids+in+nine+primate+species+at+four+zoos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001098-Caceres1" id="pone.0001098-Caceres1"></a>Caceres M, Lachuer J, Zapala MA, Redmond JC, Kudo L, et al.  (2003) Elevated gene expression levels distinguish human from non-human primate brains. Proc Natl Acad Sci U S A  100: 13030–13035.  <ul class="find" data-citedArticleID="1010962" data-doi="10.1073/pnas.2135499100"><li><a href="http://dx.doi.org/10.1073/pnas.2135499100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elevated+gene+expression+levels+distinguish+human+from+non-human+primate+brains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elevated+gene+expression+levels+distinguish+human+from+non-human+primate+brains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001098-Horton1" id="pone.0001098-Horton1"></a>Horton JD, Goldstein JL, Brown MS (2002) SREBPs: activators of the complete program of cholesterol and fatty acid synthesis in the liver. J Clin Invest  109: 1125–1131.  <ul class="find" data-citedArticleID="1010988" data-doi="10.1172/jci0215593"><li><a href="http://dx.doi.org/10.1172/jci0215593" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SREBPs%3A+activators+of+the+complete+program+of+cholesterol+and+fatty+acid+synthesis+in+the+liver." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SREBPs%3A+activators+of+the+complete+program+of+cholesterol+and+fatty+acid+synthesis+in+the+liver.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001098-Tall1" id="pone.0001098-Tall1"></a>Tall AR (2006) Protease variants, LDL, and coronary heart disease. N Engl J Med  354: 1310–1312.  <ul class="find" data-citedArticleID="1011062" data-doi="10.1056/nejme068026"><li><a href="http://dx.doi.org/10.1056/nejme068026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protease+variants%2C+LDL%2C+and+coronary+heart+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protease+variants%2C+LDL%2C+and+coronary+heart+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001098-Attie1" id="pone.0001098-Attie1"></a>Attie AD, Seidah NG (2005) Dual regulation of the LDL receptor–some clarity and new questions. Cell Metab  1: 290–292.  <ul class="find" data-citedArticleID="1010952" data-doi="10.1016/j.cmet.2005.04.006"><li><a href="http://dx.doi.org/10.1016/j.cmet.2005.04.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dual+regulation+of+the+LDL+receptor%E2%80%93some+clarity+and+new+questions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dual+regulation+of+the+LDL+receptor%E2%80%93some+clarity+and+new+questions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001098-Seidah2" id="pone.0001098-Seidah2"></a>Seidah NG, Khatib AM, Prat A (2006) The proprotein convertases and their implication in sterol and/or lipid metabolism. Biol Chem  387: 871–877.  <ul class="find" data-citedArticleID="1011052" data-doi="10.1515/bc.2006.110"><li><a href="http://dx.doi.org/10.1515/bc.2006.110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+proprotein+convertases+and+their+implication+in+sterol+and%2For+lipid+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+proprotein+convertases+and+their+implication+in+sterol+and%2For+lipid+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001098-Yang1" id="pone.0001098-Yang1"></a>Yang Z, Nielsen R (2002) Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages. Mol Biol Evol  19: 908–917.  <ul class="find" data-citedArticleID="1011074" data-doi="10.1093/oxfordjournals.molbev.a004148"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001098-Nei1" id="pone.0001098-Nei1"></a>Nei M, Kumar S (2000) Molecular evolution and phylogenetics. Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0001098-Messier1" id="pone.0001098-Messier1"></a>Messier W, Stewart CB (1997) Episodic adaptive evolution of primate lysozymes. Nature  385: 151–154.  <ul class="find" data-citedArticleID="1011012" data-doi="10.1038/385151a0"><li><a href="http://dx.doi.org/10.1038/385151a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Episodic+adaptive+evolution+of+primate+lysozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Episodic+adaptive+evolution+of+primate+lysozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001098-Yang2" id="pone.0001098-Yang2"></a>Yang Z (1998) Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution. Mol Biol Evol  15: 568–573.  <ul class="find" data-citedArticleID="1011076" data-doi="10.1093/oxfordjournals.molbev.a025957"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025957" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Likelihood+ratio+tests+for+detecting+positive+selection+and+application+to+primate+lysozyme+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Likelihood+ratio+tests+for+detecting+positive+selection+and+application+to+primate+lysozyme+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001098-Huttley1" id="pone.0001098-Huttley1"></a>Huttley GA, Easteal S, Southey MC, Tesoriero A, Giles GG, et al.  (2000) Adaptive evolution of the tumour suppressor BRCA1 in humans and chimpanzees. Australian Breast Cancer Family Study. Nat Genet  25: 410–413.  <ul class="find" data-citedArticleID="1010994"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Adaptive+evolution+of+the+tumour+suppressor+BRCA1+in+humans+and+chimpanzees.+Australian+Breast+Cancer+Family+Study.&amp;auth=&amp;atitle=Adaptive+evolution+of+the+tumour+suppressor+BRCA1+in+humans+and+chimpanzees.+Australian+Breast+Cancer+Family+Study." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+the+tumour+suppressor+BRCA1+in+humans+and+chimpanzees.+Australian+Breast+Cancer+Family+Study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+the+tumour+suppressor+BRCA1+in+humans+and+chimpanzees.+Australian+Breast+Cancer+Family+Study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001098-Sabeti1" id="pone.0001098-Sabeti1"></a>Sabeti PC, Schaffner SF, Fry B, Lohmueller J, Varilly P, et al.  (2006) Positive natural selection in the human lineage. Science  312: 1614–1620.  <ul class="find" data-citedArticleID="1011046" data-doi="10.1126/science.1124309"><li><a href="http://dx.doi.org/10.1126/science.1124309" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+natural+selection+in+the+human+lineage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+natural+selection+in+the+human+lineage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001098-Pond1" id="pone.0001098-Pond1"></a>Pond SL, Frost SD, Muse SV (2005) HyPhy: hypothesis testing using phylogenies. Bioinformatics  21: 676–679.  <ul class="find" data-citedArticleID="1011036" data-doi="10.1093/bioinformatics/bti079"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HyPhy%3A+hypothesis+testing+using+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HyPhy%3A+hypothesis+testing+using+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001098-Lynn1" id="pone.0001098-Lynn1"></a>Lynn DJ, Freeman AR, Murray C, Bradley DG (2005) A genomics approach to the detection of positive selection in cattle: adaptive evolution of the T-cell and natural killer cell-surface protein CD2. Genetics  170: 1189–1196.  <ul class="find" data-citedArticleID="1011004" data-doi="10.1534/genetics.104.039040"><li><a href="http://dx.doi.org/10.1534/genetics.104.039040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomics+approach+to+the+detection+of+positive+selection+in+cattle%3A+adaptive+evolution+of+the+T-cell+and+natural+killer+cell-surface+protein+CD2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomics+approach+to+the+detection+of+positive+selection+in+cattle%3A+adaptive+evolution+of+the+T-cell+and+natural+killer+cell-surface+protein+CD2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001098-Hurst1" id="pone.0001098-Hurst1"></a>Hurst LD (2002) The Ka/Ks ratio: diagnosing the form of sequence evolution. Trends Genet  18: 486.  <ul class="find" data-citedArticleID="1010992" data-doi="10.1016/s0168-9525(02)02722-1"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02722-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Ka%2FKs+ratio%3A+diagnosing+the+form+of+sequence+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Ka%2FKs+ratio%3A+diagnosing+the+form+of+sequence+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001098-Goodman1" id="pone.0001098-Goodman1"></a>Goodman M, Porter CA, Czelusniak J, Page SL, Schneider H, et al.  (1998) Toward a phylogenetic classification of Primates based on DNA evidence complemented by fossil evidence. Mol Phylogenet Evol  9: 585–598.  <ul class="find" data-citedArticleID="1010984" data-doi="10.1006/mpev.1998.0495"><li><a href="http://dx.doi.org/10.1006/mpev.1998.0495" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+phylogenetic+classification+of+Primates+based+on+DNA+evidence+complemented+by+fossil+evidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+phylogenetic+classification+of+Primates+based+on+DNA+evidence+complemented+by+fossil+evidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001098-Nielsen1" id="pone.0001098-Nielsen1"></a>Nielsen R, Yang Z (1998) Likelihood models for detecting positively selected amino acid sites and applications to the HIV-1 envelope gene. Genetics  148: 929–936.  <ul class="find" data-citedArticleID="1011026"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene.&amp;auth=&amp;atitle=Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001098-Yang3" id="pone.0001098-Yang3"></a>Yang Z, Nielsen R, Goldman N, Pedersen AM (2000) Codon-substitution models for heterogeneous selection pressure at amino acid sites. Genetics  155: 431–449.  <ul class="find" data-citedArticleID="1011078"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.&amp;auth=&amp;atitle=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001098-Yang4" id="pone.0001098-Yang4"></a>Yang Z, Wong WS, Nielsen R (2005) Bayes empirical bayes inference of amino acid sites under positive selection. Mol Biol Evol  22: 1107–1118.  <ul class="find" data-citedArticleID="1011080" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001098-Wong1" id="pone.0001098-Wong1"></a>Wong WS, Yang Z, Goldman N, Nielsen R (2004) Accuracy and power of statistical methods for detecting adaptive evolution in protein coding sequences and for identifying positively selected sites. Genetics  168: 1041–1051.  <ul class="find" data-citedArticleID="1011072" data-doi="10.1534/genetics.104.031153"><li><a href="http://dx.doi.org/10.1534/genetics.104.031153" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+and+power+of+statistical+methods+for+detecting+adaptive+evolution+in+protein+coding+sequences+and+for+identifying+positively+selected+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+and+power+of+statistical+methods+for+detecting+adaptive+evolution+in+protein+coding+sequences+and+for+identifying+positively+selected+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001098-Self1" id="pone.0001098-Self1"></a>Self S, Liang K (1987) Asymptotic properties of maximum likelihood estimators and likelihood ratio tests under non-standard conditions. J Am Stat Assoc  82: 605–610.  <ul class="find" data-citedArticleID="1011056" data-doi="10.1080/01621459.1987.10478472"><li><a href="http://dx.doi.org/10.1080/01621459.1987.10478472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asymptotic+properties+of+maximum+likelihood+estimators+and+likelihood+ratio+tests+under+non-standard+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asymptotic+properties+of+maximum+likelihood+estimators+and+likelihood+ratio+tests+under+non-standard+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001098-Horton2" id="pone.0001098-Horton2"></a>Horton JD, Cohen JC, Hobbs HH (2007) Molecular biology of PCSK9: its role in LDL metabolism. Trends Biochem Sci  32: 71–77.  <ul class="find" data-citedArticleID="1010990" data-doi="10.1016/j.tibs.2006.12.008"><li><a href="http://dx.doi.org/10.1016/j.tibs.2006.12.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+biology+of+PCSK9%3A+its+role+in+LDL+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+biology+of+PCSK9%3A+its+role+in+LDL+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001098-Mbikay1" id="pone.0001098-Mbikay1"></a>Mbikay M, Mayne J, Seidah NG, Chretien M (2007) Of PCSK9, cholesterol homeostasis and parasitic infections: Possible survival benefits of loss-of-function PCSK9 genetic polymorphisms. Med Hypotheses.   <ul class="find" data-citedArticleID="1011008" data-doi="10.1016/j.mehy.2007.03.018"><li><a href="http://dx.doi.org/10.1016/j.mehy.2007.03.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Of+PCSK9%2C+cholesterol+homeostasis+and+parasitic+infections%3A+Possible+survival+benefits+of+loss-of-function+PCSK9+genetic+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Of+PCSK9%2C+cholesterol+homeostasis+and+parasitic+infections%3A+Possible+survival+benefits+of+loss-of-function+PCSK9+genetic+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001098-Olson1" id="pone.0001098-Olson1"></a>Olson MV (1999) When less is more: gene loss as an engine of evolutionary change. Am J Hum Genet  64: 18–23.  <ul class="find" data-citedArticleID="1011028" data-doi="10.1086/302219"><li><a href="http://dx.doi.org/10.1086/302219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=When+less+is+more%3A+gene+loss+as+an+engine+of+evolutionary+change." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22When+less+is+more%3A+gene+loss+as+an+engine+of+evolutionary+change.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001098-Perry1" id="pone.0001098-Perry1"></a>Perry GH, Tito RY, Verrelli BC (2007) The evolutionary history of human and chimpanzee Y-chromosome gene loss. Mol Biol Evol  24: 853–859.  <ul class="find" data-citedArticleID="1011032" data-doi="10.1093/molbev/msm002"><li><a href="http://dx.doi.org/10.1093/molbev/msm002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+history+of+human+and+chimpanzee+Y-chromosome+gene+loss." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+history+of+human+and+chimpanzee+Y-chromosome+gene+loss.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001098-Zhang1" id="pone.0001098-Zhang1"></a>Zhang J, Nielsen R, Yang Z (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol  22: 2472–2479.  <ul class="find" data-citedArticleID="1011092" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001098-Anisimova1" id="pone.0001098-Anisimova1"></a>Anisimova M, Bielawski JP, Yang Z (2001) Accuracy and power of the likelihood ratio test in detecting adaptive molecular evolution. Mol Biol Evol  18: 1585–1592.  <ul class="find" data-citedArticleID="1010950" data-doi="10.1093/oxfordjournals.molbev.a003945"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003945" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+and+power+of+the+likelihood+ratio+test+in+detecting+adaptive+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+and+power+of+the+likelihood+ratio+test+in+detecting+adaptive+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001098-Purvis1" id="pone.0001098-Purvis1"></a>Purvis A (1995) A composite estimate of primate phylogeny. Philos Trans R Soc Lond B Biol Sci  348: 405–421.  <ul class="find" data-citedArticleID="1011038"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+composite+estimate+of+primate+phylogeny.&amp;auth=&amp;atitle=A+composite+estimate+of+primate+phylogeny." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+composite+estimate+of+primate+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+composite+estimate+of+primate+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001098-McNutt1" id="pone.0001098-McNutt1"></a>McNutt MC, Lagace TA, Horton JD (2007) Catalytic activity is not required for secreted PCSK9 to reduce LDL receptors in HepG2 cells. J Biol Chem  282: 20799–20803.  <ul class="find" data-citedArticleID="1011010" data-doi="10.1074/jbc.c700095200"><li><a href="http://dx.doi.org/10.1074/jbc.c700095200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catalytic+activity+is+not+required+for+secreted+PCSK9+to+reduce+LDL+receptors+in+HepG2+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catalytic+activity+is+not+required+for+secreted+PCSK9+to+reduce+LDL+receptors+in+HepG2+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001098-Seidah3" id="pone.0001098-Seidah3"></a>Seidah NG, Prat A (2007) The proprotein convertases are potential targets in the treatment of dyslipidemia. J Mol Med  85: 685–696.  <ul class="find" data-citedArticleID="1011054" data-doi="10.1007/s00109-007-0172-7"><li><a href="http://dx.doi.org/10.1007/s00109-007-0172-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+proprotein+convertases+are+potential+targets+in+the+treatment+of+dyslipidemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+proprotein+convertases+are+potential+targets+in+the+treatment+of+dyslipidemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001098-Nassoury1" id="pone.0001098-Nassoury1"></a>Nassoury N, Blasiole DA, Tebon Oler A, Benjannet S, Hamelin J, et al.  (2007) The cellular trafficking of the secretory proprotein convertase PCSK9 and its dependence on the LDLR. Traffic  8: 718–732.  <ul class="find" data-citedArticleID="1011018" data-doi="10.1111/j.1600-0854.2007.00562.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0854.2007.00562.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cellular+trafficking+of+the+secretory+proprotein+convertase+PCSK9+and+its+dependence+on+the+LDLR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cellular+trafficking+of+the+secretory+proprotein+convertase+PCSK9+and+its+dependence+on+the+LDLR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001098-Enard1" id="pone.0001098-Enard1"></a>Enard W, Khaitovich P, Klose J, Zollner S, Heissig F, et al.  (2002) Intra- and interspecific variation in primate gene expression patterns. Science  296: 340–343.  <ul class="find" data-citedArticleID="1010976" data-doi="10.1126/science.1068996"><li><a href="http://dx.doi.org/10.1126/science.1068996" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intra-+and+interspecific+variation+in+primate+gene+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intra-+and+interspecific+variation+in+primate+gene+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001098-Poirier1" id="pone.0001098-Poirier1"></a>Poirier S, Prat A, Marcinkiewicz E, Paquin J, Chitramuthu BP, et al.  (2006) Implication of the proprotein convertase NARC-1/PCSK9 in the development of the nervous system. J Neurochem  98: 838–850.  <ul class="find" data-citedArticleID="1011034" data-doi="10.1111/j.1471-4159.2006.03928.x"><li><a href="http://dx.doi.org/10.1111/j.1471-4159.2006.03928.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Implication+of+the+proprotein+convertase+NARC-1%2FPCSK9+in+the+development+of+the+nervous+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Implication+of+the+proprotein+convertase+NARC-1%2FPCSK9+in+the+development+of+the+nervous+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001098-Rashid1" id="pone.0001098-Rashid1"></a>Rashid S, Curtis DE, Garuti R, Anderson NN, Bashmakov Y, et al.  (2005) Decreased plasma cholesterol and hypersensitivity to statins in mice lacking Pcsk9. Proc Natl Acad Sci U S A  102: 5374–5379.  <ul class="find" data-citedArticleID="1011042" data-doi="10.1073/pnas.0501652102"><li><a href="http://dx.doi.org/10.1073/pnas.0501652102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Decreased+plasma+cholesterol+and+hypersensitivity+to+statins+in+mice+lacking+Pcsk9." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Decreased+plasma+cholesterol+and+hypersensitivity+to+statins+in+mice+lacking+Pcsk9.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001098-Bingham1" id="pone.0001098-Bingham1"></a>Bingham B, Shen R, Kotnis S, Lo CF, Ozenberger BA, et al.  (2006) Proapoptotic effects of NARC 1 ( = PCSK9), the gene encoding a novel serine proteinase. Cytometry A  69: 1123–1131.  <ul class="find" data-citedArticleID="1010958" data-doi="10.1002/cyto.a.20346"><li><a href="http://dx.doi.org/10.1002/cyto.a.20346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proapoptotic+effects+of+NARC+1+%28%E2%80%8A%3D%E2%80%8APCSK9%29%2C+the+gene+encoding+a+novel+serine+proteinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proapoptotic+effects+of+NARC+1+%28%E2%80%8A%3D%E2%80%8APCSK9%29%2C+the+gene+encoding+a+novel+serine+proteinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001098-Hahn1" id="pone.0001098-Hahn1"></a>Hahn MW, Rockman MV, Soranzo N, Goldstein DB, Wray GA (2004) Population genetic and phylogenetic evidence for positive selection on regulatory mutations at the factor VII locus in humans. Genetics  167: 867–877.  <ul class="find" data-citedArticleID="1010986" data-doi="10.1534/genetics.103.025726"><li><a href="http://dx.doi.org/10.1534/genetics.103.025726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetic+and+phylogenetic+evidence+for+positive+selection+on+regulatory+mutations+at+the+factor+VII+locus+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetic+and+phylogenetic+evidence+for+positive+selection+on+regulatory+mutations+at+the+factor+VII+locus+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001098-Nakajima1" id="pone.0001098-Nakajima1"></a>Nakajima T, Wooding S, Sakagami T, Emi M, Tokunaga K, et al.  (2004) Natural selection and population history in the human angiotensinogen gene (AGT): 736 complete AGT sequences in chromosomes from around the world. Am J Hum Genet  74: 898–916.  <ul class="find" data-citedArticleID="1011014" data-doi="10.1086/420793"><li><a href="http://dx.doi.org/10.1086/420793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001098-Thompson1" id="pone.0001098-Thompson1"></a>Thompson EE, Kuttab-Boulos H, Witonsky D, Yang L, Roe BA, et al.  (2004) CYP3A variation and the evolution of salt-sensitivity variants. Am J Hum Genet  75: 1059–1069.  <ul class="find" data-citedArticleID="1011066" data-doi="10.1086/426406"><li><a href="http://dx.doi.org/10.1086/426406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CYP3A+variation+and+the+evolution+of+salt-sensitivity+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CYP3A+variation+and+the+evolution+of+salt-sensitivity+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001098-Rockman1" id="pone.0001098-Rockman1"></a>Rockman MV, Hahn MW, Soranzo N, Loisel DA, Goldstein DB, et al.  (2004) Positive selection on MMP3 regulation has shaped heart disease risk. Curr Biol  14: 1531–1539.  <ul class="find" data-citedArticleID="1011044" data-doi="10.1016/j.cub.2004.08.051"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.08.051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+selection+on+MMP3+regulation+has+shaped+heart+disease+risk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+selection+on+MMP3+regulation+has+shaped+heart+disease+risk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001098-Young1" id="pone.0001098-Young1"></a>Young JH, Chang YP, Kim JD, Chretien JP, Klag MJ, et al.  (2005) Differential susceptibility to hypertension is due to selection during the out-of-Africa expansion. PLoS Genet  1: e82.  <ul class="find" data-citedArticleID="1011088" data-doi="10.1371/journal.pgen.0010082.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010082.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+susceptibility+to+hypertension+is+due+to+selection+during+the+out-of-Africa+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+susceptibility+to+hypertension+is+due+to+selection+during+the+out-of-Africa+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001098-Ding1" id="pone.0001098-Ding1"></a>Ding K, Kullo IJ (2006) Molecular evolution of 5′ flanking regions of 87 candidate genes for atherosclerotic cardiovascular disease. Genet Epidemiol  30: 557–569.  <ul class="find" data-citedArticleID="1010972" data-doi="10.1002/gepi.20169"><li><a href="http://dx.doi.org/10.1002/gepi.20169" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+of+5%E2%80%B2+flanking+regions+of+87+candidate+genes+for+atherosclerotic+cardiovascular+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+of+5%E2%80%B2+flanking+regions+of+87+candidate+genes+for+atherosclerotic+cardiovascular+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001098-Kullo1" id="pone.0001098-Kullo1"></a>Kullo IJ, Ding K (2007) Patterns of population differentiation of candidate genes for cardiovascular disease. BMC genetics  8: 48.  <ul class="find" data-citedArticleID="1010998" data-doi="10.1186/1471-2156-8-48"><li><a href="http://dx.doi.org/10.1186/1471-2156-8-48" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+population+differentiation+of+candidate+genes+for+cardiovascular+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+population+differentiation+of+candidate+genes+for+cardiovascular+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001098-Thomas1" id="pone.0001098-Thomas1"></a>Thomas JW, Touchman JW, Blakesley RW, Bouffard GG, Beckstrom-Sternberg SM, et al.  (2003) Comparative analyses of multi-species sequences from targeted genomic regions. Nature  424: 788–793.  <ul class="find" data-citedArticleID="1011064" data-doi="10.1038/nature01858"><li><a href="http://dx.doi.org/10.1038/nature01858" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analyses+of+multi-species+sequences+from+targeted+genomic+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analyses+of+multi-species+sequences+from+targeted+genomic+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001098-Schaner1" id="pone.0001098-Schaner1"></a>Schaner P, Richards N, Wadhwa A, Aksentijevich I, Kastner D, et al.  (2001) Episodic evolution of pyrin in primates: human mutations recapitulate ancestral amino acid states. Nat Genet  27: 318–321.  <ul class="find" data-citedArticleID="1011048" data-doi="10.1038/85893"><li><a href="http://dx.doi.org/10.1038/85893" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Episodic+evolution+of+pyrin+in+primates%3A+human+mutations+recapitulate+ancestral+amino+acid+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Episodic+evolution+of+pyrin+in+primates%3A+human+mutations+recapitulate+ancestral+amino+acid+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001098-Ng1" id="pone.0001098-Ng1"></a>Ng PC, Henikoff S (2003) SIFT: Predicting amino acid changes that affect protein function. Nucleic Acids Res  31: 3812–3814.  <ul class="find" data-citedArticleID="1011024" data-doi="10.1093/nar/gkg509"><li><a href="http://dx.doi.org/10.1093/nar/gkg509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SIFT%3A+Predicting+amino+acid+changes+that+affect+protein+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SIFT%3A+Predicting+amino+acid+changes+that+affect+protein+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001098-Ramensky1" id="pone.0001098-Ramensky1"></a>Ramensky V, Bork P, Sunyaev S (2002) Human non-synonymous SNPs: server and survey. Nucleic Acids Res  30: 3894–3900.  <ul class="find" data-citedArticleID="1011040" data-doi="10.1093/nar/gkf493"><li><a href="http://dx.doi.org/10.1093/nar/gkf493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+non-synonymous+SNPs%3A+server+and+survey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+non-synonymous+SNPs%3A+server+and+survey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001098-Florea1" id="pone.0001098-Florea1"></a>Florea L, Hartzell G, Zhang Z, Rubin GM, Miller W (1998) A computer program for aligning a cDNA sequence with a genomic DNA sequence. Genome Res  8: 967–974.  <ul class="find" data-citedArticleID="1010982"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence.&amp;auth=&amp;atitle=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001098-Thompson2" id="pone.0001098-Thompson2"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1011068" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001098-Yang5" id="pone.0001098-Yang5"></a>Yang Z, Goldman N, Friday A (1994) Comparison of models for nucleotide substitution used in maximum-likelihood phylogenetic estimation. Mol Biol Evol  11: 316–324.  <ul class="find" data-citedArticleID="1011082"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation.&amp;auth=&amp;atitle=Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001098-Yang6" id="pone.0001098-Yang6"></a>Yang Z, Bielawski JP (2000) Statistical methods for detecting molecular adaptation. Trends in Ecology and Evolution  15: 496–503.  <ul class="find" data-citedArticleID="1011084" data-doi="10.1016/s0169-5347(00)01994-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)01994-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+methods+for+detecting+molecular+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+methods+for+detecting+molecular+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001098-Yang7" id="pone.0001098-Yang7"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="1011086" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001098-Choi1" id="pone.0001098-Choi1"></a>Choi SS, Lahn BT (2003) Adaptive evolution of MRG, a neuron-specific gene family implicated in nociception. Genome Res  13: 2252–2259.  <ul class="find" data-citedArticleID="1010964" data-doi="10.1101/gr.1431603"><li><a href="http://dx.doi.org/10.1101/gr.1431603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+MRG%2C+a+neuron-specific+gene+family+implicated+in+nociception." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+MRG%2C+a+neuron-specific+gene+family+implicated+in+nociception.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.XML" value="141727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.PDF" value="467265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g001.PNG_L" value="49850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g001.PNG_M" value="13493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g001.PNG_S" value="4225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g001.TIF" value="118546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g001.PNG_I" value="5586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g002.PNG_L" value="298939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g002.PNG_M" value="104074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g002.PNG_S" value="8756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g002.TIF" value="605224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g002.PNG_I" value="59212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t001.PNG_L" value="148622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t001.PNG_M" value="206771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t001.PNG_S" value="21843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t001.TIF" value="583418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t001.PNG_I" value="68400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g003.PNG_L" value="96592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g003.PNG_M" value="19307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g003.PNG_S" value="5429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g003.TIF" value="525042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g003.PNG_I" value="7045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.e001.PNG" value="7274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.e001.TIF" value="23192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t002.PNG_L" value="197046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t002.PNG_M" value="113761"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t002.PNG_S" value="12667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t002.TIF" value="806794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t002.PNG_I" value="35144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g004.PNG_L" value="104299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g004.PNG_M" value="30190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g004.PNG_S" value="6084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g004.TIF" value="328258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g004.PNG_I" value="11220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t003.PNG_L" value="162248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t003.PNG_M" value="89668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t003.PNG_S" value="11172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t003.TIF" value="700206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.t003.PNG_I" value="27498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g005.PNG_L" value="25916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g005.PNG_M" value="63459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g005.PNG_S" value="8626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g005.TIF" value="116172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.g005.PNG_I" value="23972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.e002.PNG" value="5138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.e002.TIF" value="21080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.e003.PNG" value="5138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.e003.TIF" value="21080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.s001.EPS" value="5313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.s002.DOC" value="102400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001098.s003.DOC" value="36864"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001098&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001098" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001098&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001098&volume=&issue=&title=Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene PCSK9 Based on Phylogenetic Analysis in 14 Primate Species&author_name=Keyue%20Ding%2C%20Samantha%20J.%20McDonough%2C%20Iftikhar%20J.%20Kullo&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001098" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001098" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001098" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001098&amp;t=Evidence%20for%20Positive%20Selection%20in%20the%20C-terminal%20Domain%20of%20the%20Cholesterol%20Metabolism%20Gene%20PCSK9%20Based%20on%20Phylogenetic%20Analysis%20in%2014%20Primate%20Species" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001098&title=Evidence%20for%20Positive%20Selection%20in%20the%20C-terminal%20Domain%20of%20the%20Cholesterol%20Metabolism%20Gene%20PCSK9%20Based%20on%20Phylogenetic%20Analysis%20in%2014%20Primate%20Species&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001098&amp;title=Evidence%20for%20Positive%20Selection%20in%20the%20C-terminal%20Domain%20of%20the%20Cholesterol%20Metabolism%20Gene%20PCSK9%20Based%20on%20Phylogenetic%20Analysis%20in%2014%20Primate%20Species" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001098" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001098&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evidence for Positive Selection in the C-terminal Domain of the Cholesterol Metabolism Gene PCSK9 Based on Phylogenetic Analysis in 14 Primate Species';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001098';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evidence%20for%20Positive%20Selection%20in%20the%20C-terminal%20Domain%20of%20the%20Cholesterol%20Metabolism%20Gene%20PCSK9%20Based%20on%20Phylogenetic%20Analysis%20in%2014%20Primate%20Species http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001098" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001098" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cholesterol%22" title="Search for articles in the subject area:'Cholesterol'"><div class="flagText">Cholesterol</div></a>
              <div data-categoryid="20521" data-articleid="25030"
                   data-categoryname="Cholesterol"
                   class="flagImage" title="Flag 'Cholesterol' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="31811" data-articleid="25030"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="25030"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="25030"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22New+World+monkeys%22" title="Search for articles in the subject area:'New World monkeys'"><div class="flagText">New World monkeys</div></a>
              <div data-categoryid="44139" data-articleid="25030"
                   data-categoryname="New World monkeys"
                   class="flagImage" title="Flag 'New World monkeys' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Orangutans%22" title="Search for articles in the subject area:'Orangutans'"><div class="flagText">Orangutans</div></a>
              <div data-categoryid="43615" data-articleid="25030"
                   data-categoryname="Orangutans"
                   class="flagImage" title="Flag 'Orangutans' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25030"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Primates%22" title="Search for articles in the subject area:'Primates'"><div class="flagText">Primates</div></a>
              <div data-categoryid="42883" data-articleid="25030"
                   data-categoryname="Primates"
                   class="flagImage" title="Flag 'Primates' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4314'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1871'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6675&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9951">Positive Selection or Relaxed Selective Constraint?</a><br>Posted by timpanister</p>
      <p><a href="/annotation/listThread.action?root=18661">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
