

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Virtual Northern Analysis of the Human Genome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000460"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000460" />

    <meta name="citation_title" content="Virtual Northern Analysis of the Human Genome"/>
    <meta itemprop="name" content="Virtual Northern Analysis of the Human Genome"/>

      <meta name="citation_author" content="Evan H. Hurowitz"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Stanford University School of Medicine, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Iddo Drori"/>
            <meta name="citation_author_institution" content="Department of Statistics, Stanford University, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Victoria C. Stodden"/>
            <meta name="citation_author_institution" content="Department of Statistics, Stanford University, Stanford, California, United States of America"/>
      <meta name="citation_author" content="David L. Donoho"/>
            <meta name="citation_author_institution" content="Department of Statistics, Stanford University, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Patrick O. Brown"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Stanford University School of Medicine, Stanford, California, United States of America"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Stanford University School of Medicine, Stanford, California, United States of America"/>

    <meta name="citation_date" content="2007/5/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000460.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e460"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome.; citation_author=ES Lander; citation_author=LM Linton; citation_author=B Birren; citation_author=C Nusbaum; citation_author=MC Zody; citation_journal_title=Nature; citation_volume=409; citation_number=1; citation_pages=860-921; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The sequence of the human genome.; citation_author=JC Venter; citation_author=MD Adams; citation_author=EW Myers; citation_author=PW Li; citation_author=RJ Mural; citation_journal_title=Science; citation_volume=291; citation_number=2; citation_pages=1304-1351; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Finishing the euchromatic sequence of the human genome.; citation_journal_title=Nature; citation_volume=431; citation_number=3; citation_pages=931-45; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integrative annotation of 21,037 human genes validated by full-length cDNA clones.; citation_author=T Imanishi; citation_author=T Itoh; citation_author=Y Suzuki; citation_author=C O'Donovan; citation_author=S Fukuchi; citation_journal_title=PLOS Biology; citation_volume=2; citation_number=4; citation_pages=1-20; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Construction and characterization of a full length-enriched and a 5′-end-enriched cDNA library.; citation_author=Y Suzuki; citation_author=K Yoshitomo-Nakagawa; citation_author=K Maruyama; citation_author=A Suyama; citation_author=S Sugano; citation_journal_title=Gene; citation_volume=200; citation_number=5; citation_pages=149-56; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.; citation_author=RL Strausberg; citation_author=EA Feingold; citation_author=LH Grouse; citation_author=JG Derge; citation_author=RD Klausner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=6; citation_pages=16899-16903; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A genomic view of alternative splicing.; citation_author=B Modrek; citation_author=C Lee; citation_journal_title=Nat Genet; citation_volume=30; citation_number=7; citation_pages=13-19; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information: update.; citation_author=DL Wheeler; citation_author=DM Church; citation_author=R Edgar; citation_author=S Federhen; citation_author=W Helmberg; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=8; citation_pages=D35-40; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of mRNA lengths in Saccharomyces cerevisiae.; citation_author=EH Hurowitz; citation_author=PO Brown; citation_journal_title=Genome Biol; citation_volume=5; citation_number=9; citation_pages=R2; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The UCSC Genome Browser Database.; citation_author=D Karolchik; citation_author=R Baertsch; citation_author=M Diekhans; citation_author=TS Furey; citation_author=A Hinrichs; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=10; citation_pages=51-54; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays.; citation_author=JM Johnson; citation_author=J Castle; citation_author=P Garrett-Engele; citation_author=Z Kan; citation_author=PM Loerch; citation_journal_title=Science; citation_volume=302; citation_number=11; citation_pages=2141-2144; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An alternative-exon database and its statistical analysis.; citation_author=S Stamm; citation_author=J Zhu; citation_author=K Nakai; citation_author=P Stoilov; citation_author=O Stoss; citation_journal_title=DNA Cell Biol; citation_volume=19; citation_number=12; citation_pages=739-56; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genome-wide detection of tissue-specific alternative splicing in the human transcriptome.; citation_author=Q Xu; citation_author=B Modrek; citation_author=C Lee; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=13; citation_pages=3754-3766; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Untranslated regions of mRNAs.; citation_author=F Mignone; citation_author=C Gissi; citation_author=S Liuni; citation_author=G Pesole; citation_journal_title=Genome Biol; citation_volume=3; citation_number=14; citation_pages=reviews0004.1-10; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Animal microRNAs confer robustness to gene expression and have a significant impact on 3′UTR evolution.; citation_author=A Stark; citation_author=J Brennecke; citation_author=N Bushati; citation_author=RB Russell; citation_author=SM Cohen; citation_journal_title=Cell; citation_volume=123; citation_number=15; citation_pages=1133-1146; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A reagent for the single-step simultaneous isolation of RNA, DNA and proteins from cell and tissue samples.; citation_author=P Chomczynski; citation_journal_title=Biotechniques; citation_volume=15; citation_number=16; citation_pages=532; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Molecular portraits of human breast tumours.; citation_author=CM Perou; citation_author=T Sorlie; citation_author=MB Eisen; citation_author=M van de Rijn; citation_author=SS Jeffrey; citation_journal_title=Nature; citation_volume=406; citation_number=17; citation_pages=747-752; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Precision and functional specificity in mRNA decay.; citation_author=Y Wang; citation_author=CL Liu; citation_author=JD Storey; citation_author=RJ Tibshirani; citation_author=D Herschlag; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=18; citation_pages=5860-5865; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Atomic decomposition by basis pursuit.; citation_author=SS Chen; citation_author=DL Donoho; citation_author=MA Saunders; citation_journal_title=SIAM J Sci Comp; citation_volume=20; citation_number=19; citation_pages=33-61; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Pdco: Primal-dual interior method for convex objectives.; citation_author=MA Saunders; citation_author=B Kim; citation_number=20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=For most underdetermined systems of linear equations, the minimal l1-norm near-solution approximates the sparsest near-solution.; citation_author=DL Donoho; citation_journal_title=Comm Pure and Appl Math; citation_number=21; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sparse nonnegative solutions of underdetermined linear equations by linear programming.; citation_author=DL Donoho; citation_author=J Tanner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=22; citation_pages=9446-9451; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Stable recovery of sparse overcomplete representations in the presence of noise.; citation_author=DL Donoho; citation_author=M Elad; citation_author=VN Temlyakov; citation_journal_title=IEEE Trans on Information Theory; citation_volume=52; citation_number=23; citation_pages=6-18; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000460" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Virtual Northern Analysis of the Human Genome"/>
    <meta name="twitter:description" content="BackgroundWe applied the Virtual Northern technique to human brain mRNA to systematically measure human mRNA transcript lengths on a genome-wide scale.Methodology/Principal FindingsWe used separation by gel electrophoresis followed by hybridization to cDNA microarrays to measure 8,774 mRNA transcript lengths representing at least 6,238 genes at high (&gt;90%) confidence. By comparing these transcript lengths to the Refseq and H-Invitational full-length cDNA databases, we found that nearly half of our measurements appeared to represent novel transcript variants. Comparison of length measurements determined by hybridization to different cDNAs derived from the same gene identified clones that potentially correspond to alternative transcript variants. We observed a close linear relationship between ORF and mRNA lengths in human mRNAs, identical in form to the relationship we had previously identified in yeast. Some functional classes of protein are encoded by mRNAs whose untranslated regions (UTRs) tend to be longer or shorter than average; these functional classes were similar in both human and yeast.Conclusions/SignificanceHuman transcript diversity is extensive and largely unannotated. Our length dataset can be used as a new criterion for judging the completeness of cDNAs and annotating mRNA sequences. Similar relationships between the lengths of the UTRs in human and yeast mRNAs and the functions of the proteins they encode suggest that UTR sequences serve an important regulatory role among eukaryotes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000460.g007"/>

  <meta property="og:title" content="Virtual Northern Analysis of the Human Genome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000460" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3460'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3331'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4347&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000460">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000460" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000460&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Virtual Northern Analysis of the Human Genome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Evan H. Hurowitz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Iddo Drori, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, Stanford University, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Victoria C. Stodden, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, Stanford University, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David L. Donoho, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, Stanford University, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrick O. Brown
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pbrown@cmgm.stanford.edu">pbrown@cmgm.stanford.edu</a></p>

                <p>Affiliations:
                  Department of Biochemistry, Stanford University School of Medicine, Stanford, California, United States of America, 
                  Howard Hughes Medical Institute, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 23, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000460</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000460" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000460" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000460" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000460" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000460-g001" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-g002" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-t001" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-t002" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-g003" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-t003" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-g004" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-g005" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-t004" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-t005" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-t006" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-g006" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000460-g007" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000460">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000460" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>We applied the Virtual Northern technique to human brain mRNA to systematically measure human mRNA transcript lengths on a genome-wide scale.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We used separation by gel electrophoresis followed by hybridization to cDNA microarrays to measure 8,774 mRNA transcript lengths representing at least 6,238 genes at high (&gt;90%) confidence. By comparing these transcript lengths to the Refseq and H-Invitational full-length cDNA databases, we found that nearly half of our measurements appeared to represent novel transcript variants. Comparison of length measurements determined by hybridization to different cDNAs derived from the same gene identified clones that potentially correspond to alternative transcript variants. We observed a close linear relationship between ORF and mRNA lengths in human mRNAs, identical in form to the relationship we had previously identified in yeast. Some functional classes of protein are encoded by mRNAs whose untranslated regions (UTRs) tend to be longer or shorter than average; these functional classes were similar in both human and yeast.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Human transcript diversity is extensive and largely unannotated. Our length dataset can be used as a new criterion for judging the completeness of cDNAs and annotating mRNA sequences. Similar relationships between the lengths of the UTRs in human and yeast mRNAs and the functions of the proteins they encode suggest that UTR sequences serve an important regulatory role among eukaryotes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hurowitz EH, Drori I, Stodden VC, Donoho DL, Brown PO (2007) Virtual Northern Analysis of the Human Genome. PLoS ONE 2(5):
          e460.
            doi:10.1371/journal.pone.0000460</p><p><strong>Academic Editor: </strong>Juan Valcarcel, Centre de Regulació Genòmica, Spain</p><p><strong>Received:</strong> March 12, 2007; <strong>Accepted:</strong> April 9, 2007; <strong>Published:</strong> May 23, 2007</p><p><strong>Copyright:</strong> © 2007 Hurowitz et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NIH grant CA77097 and by the Howard Hughes Medical Institute. P.O.B. is an investigator of the Howard Hughes Medical Institute. Neither the NIH nor HHMI participated in the design and conduct of the study or the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Now that the human genome sequence is nearly complete <a href="#pone.0000460-Lander1">[1]</a>–<a href="#pone.0000460-International1">[3]</a>, the next step is to characterize the organization, function, and diversity of the human genome. Reliable computational detection and analysis of genes in mammalian genomes remains a challenge due to the low percentage of coding sequence, the existence of many short exons and long introns, and the high diversity of alternate transcript forms <a href="#pone.0000460-Lander1">[1]</a>. Therefore, most efforts to annotate the human genome have relied heavily on the analysis of expressed sequences generated from human RNA. Recently however, the focus has shifted from the generation of ESTs, which are generally short clones representing a fraction of their parent transcript, to the generation of full-length cDNAs. Due to a number of large-scale full-length cDNA sequencing projects, over 20,000 human genes have been validated by at least one putative full-length cDNA <a href="#pone.0000460-Imanishi1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Although full-length cDNA sequencing projects provide the basis for virtually all human gene identification and analysis, they suffer from several limitations. First, they are expensive and labor-intensive. Second, there are no fool-proof methods for cloning only full-length cDNAs, or identifying cDNA clones that are not full-length. There are a number of well-accepted cloning methods that enrich for full-length cDNAs <a href="#pone.0000460-Imanishi1">[4]</a>, but these methods may yield the true 5′-end only 80% of the time <a href="#pone.0000460-Suzuki1">[5]</a>. Methods for identifying cDNA clones that are not full-length typically involve either comparison of the clones to other clones, computational analysis of the cDNA's sequence to identify a translational initiation site, or computational analysis of the genome sequence upstream from the cDNA to identify a putative promoter <a href="#pone.0000460-Strausberg1">[6]</a>. Although all of these approaches are valid and important analyses, none of them actually ensure that the clone is full-length, especially in cases where the full transcript may be difficult to clone, for example, due to secondary structure in the transcript. Third, long transcripts are under-represented in cDNA clone libraries. Finally and most importantly, full-length cDNA projects suffer from a strong sampling bias due to very large differences in expression levels between different transcripts. For that reason, only the most abundantly expressed transcripts are well-sampled in cDNA libraries. Most genes are represented in these libraries by fewer than two full-length transcripts <a href="#pone.0000460-Imanishi1">[4]</a>, allowing many inabundant transcripts and transcript variants to go undetected <a href="#pone.0000460-Modrek1">[7]</a>. Furthermore, the small numbers of cDNA clones representing most genes makes estimates of the relative abundance of transcripts from tissue to tissue, and variant to variant, unreliable. Due to these limitations, it is unlikely that the goal of completely characterizing the human transcriptome, including all transcript variants across all tissues, disease states, and developmental stages, will be accomplished by full-length cDNA sequencing alone.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Characterization of RNA transcripts by length does not have the resolution to identify precise sites of transcript initiation or termination, precise splice sites, or even exon-intron structure, but it does provide an unbiased measurement of transcript length, a quantity that is relatively difficult to obtain through full-length cDNA sequencing alone. This independent characterization of mRNA length is useful in determining if clones are in fact full-length, and provides an additional parameter for computationally identifying genes from the genome sequence. By comparing our length measurements to the Unigene, Refseq <a href="#pone.0000460-Wheeler1">[8]</a>, and H-Inv <a href="#pone.0000460-Imanishi1">[4]</a> databases, our measurements allowed us to evaluate current progress in cataloging the transcriptome.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Evaluation of the human Virtual Northern</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We applied the Virtual Northern technique <a href="#pone.0000460-Hurowitz1">[9]</a> to the human genome in order to further characterize the human transcriptome. Virtual Northern analysis uses gel electrophoretic separation of mRNAs by length, and DNA microarray analysis to read out the results for a large set of genes in parallel. Briefly, we separated human brain mRNA by length on an agarose gel, sliced the gel into 50 narrow sections each containing RNA from a small range of lengths, and hybridized the RNA from each slice to a separate cDNA microarray (<a href="#pone-0000460-g001">Figure 1</a>). The data for each cDNA from all 50 microarrays were combined into a profile that peaks in the slice, or slices, that contain mRNAs complementary to a given cDNA sequence represented on the microarray (<a href="#pone-0000460-g002">Figure 2</a>). We searched each length profile for peaks by <em>l</em><sub>1</sub> norm baseline deconvolution, and estimated a confidence value for each peak with a bootstrapping approach. The experiment was performed in triplicate and the results from all three replicates were combined to create a dataset of peaks along with their associated confidence values for each cDNA. Concordance between the three replicates was strongly correlated with microarray signal strength, ranging from virtually 100% for the most strongly expressed genes to almost 0% for the most weakly expressed genes. Replication of the experiment greatly reduced the noise, and increased confidence of many weak but legitimate peaks. Using a set of well-studied (“gold standard”) genes with mRNAs of known length, we converted the locations of the peaks within each profile into transcript lengths in nucleotides. The conversion explicitly accounted for the length of each transcript's poly(A) tail by adding a fixed length to each known length. A fixed length of 225 nucleotides was calculated to provide the best fit between the peak locations and the known transcript lengths. We also explicitly calculated our accuracy, allowing us to calibrate our bootstrap values to the probability that an observed peak represents a real transcript (the true positive rate).</p>
<div class="figure" id="pone-0000460-g001"><div class="img"><a name="pone-0000460-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Virtual Northern scheme.</span></strong></p><span>doi:10.1371/journal.pone.0000460.g001</span></div><div class="figure" id="pone-0000460-g002"><div class="img"><a name="pone-0000460-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Example length profile with deconvolution results.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>An example length profile is shown in blue. The normalized ratio from each length fraction is plotted against the length fractions in order, where the first length fraction is the one with the highest gel mobility. The rolling baseline is shown in green, and the deconvolution result is shown in red.</p>
<span>doi:10.1371/journal.pone.0000460.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The resulting dataset contained 21,933 combined length profiles with peaks ranging in confidence from 6% to 100%. At a high stringency 90% confidence cutoff, we identified 8,774 clones with at least one length, and a mean confidence for all peaks of 97% (<a href="#pone-0000460-t001">Table 1</a>). Of those 8,774 clones, 118 (1.3%) had a two peaks. The mean and median of all lengths at 90% were 2,165 and 1,996 nucleotides respectively. These values are similar to a poly(A) tail-corrected mean of 2,635 nucleotides and median of 1,965 nucleotides previously estimated for the human genome <a href="#pone.0000460-Lander1">[1]</a>.</p>
<div class="figure" id="pone-0000460-t001"><div class="img"><a name="pone-0000460-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Results of data filtering and peak finding</span></strong></p><span>doi:10.1371/journal.pone.0000460.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Using information downloaded from the National Center for Biotechnology Information (NCBI) website (<a href="http://www.ncbi.nlm.nih.gov/Ftp/">http://www.ncbi.nlm.nih.gov/Ftp/</a>) on August 8, 2005, each cDNA clone spotted on the microarrays could be associated with the human gene or genomic locus from which it derives <a href="#pone.0000460-Wheeler1">[8]</a>. Out of 41,418 cDNA clones on the microarrays, 31,708 (76.6%) of them could be assigned to at least one Entrez gene (<a href="#pone-0000460-t002">Table 2</a>). The remaining 9,710 clones with no assignment to an Entrez gene represent putative expressed sequences whose cognate genes remain uncharacterized. Since multiple clones can map to the same locus, the 31,708 clones represent 15,552 distinct genes. We identified a transcript at ≥90% confidence for 6,238 (40%) of the 15,552 currently annotated human genes represented on the microarrays. As shown in <a href="#pone-0000460-t002">Table 2</a>, the percentage of clones for which we identified a peak was much lower among the clones with no assignment to an Entrez gene than those with a gene assignment. This probably reflects the enrichment in this set of clones for cloning artifacts, or rarely expressed genes that would be difficult to detect by either hybridization or cDNA cloning based methods.</p>
<div class="figure" id="pone-0000460-t002"><div class="img"><a name="pone-0000460-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Association between cDNA clones and human genes</span></strong></p><span>doi:10.1371/journal.pone.0000460.t002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The precision with which we measured human transcript lengths was estimated based on the deviations between transcript lengths inferred from the Virtual Northern method and the lengths of the mRNAs in our “gold standard” set (see <a href="#s4">Materials and Methods</a>), and on the variation among multiple measurements for the same gene. As we previously found in our analysis of yeast transcripts, one major limitation of precision was the precision with which the gel was sliced <a href="#pone.0000460-Hurowitz1">[9]</a>. In this case, due to the approximately exponential relationship between gel mobility and RNA length, the range of lengths represented by each 2 mm slice increased from 24 to 331 nucleotides with decreasing distance from the origin over the range analyzed. However, as a fraction of RNA length the variation represented in each slice remained a constant 5.6%. Nevertheless, the average absolute deviation between 184 Virtual Northern measurements and the length of their matching gold standard mRNA was 12.2%. This estimation of precision is twice the width of each slice, but it is likely an overestimate of the true deviation. As shown in <a href="#pone-0000460-g003">Figure 3</a>, sixteen outliers were excluded from the calculation of the best fit line. With those outliers also excluded from the precision calculation, the average absolute deviation becomes 5.5%. We therefore consider 5.5% (approximately the gel slice width) to be a more accurate estimate of our measurement precision since the larger estimate is probably inflated due to the inclusion of false positive measurements. Additionally, the deviations between multiple measurements for the same gene were calculated. That average standard deviation was calculated to be only 2.7% over 1,846 combined profiles representing 492 unique sequences represented on the microarrays. Since the variations among replicate measurements are substantially less than the imprecision in the transformation of gel mobility to mRNA length, we conclude that the measurement precision is 5.5%, approximately the gel slice width.</p>
<div class="figure" id="pone-0000460-g003"><div class="img"><a name="pone-0000460-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Calibrating the relationship between gel mobility and transcript length.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>The precise gel mobilities of all peaks from gold standard genes are plotted against the natural log of the sum of their matching Refseq length and an estimated poly(A) tail length of 225 nucleotides. The least squares fit to a line is shown by a black line with the parameters y = 0.054731 x+5.997276 (R<sup>2</sup> = 0.99). Closed circles represent points used to determine the calibration line. Points shown by open circles were excluded from the least squares calculation.</p>
<span>doi:10.1371/journal.pone.0000460.g003</span></div>

<h4>Unigene Cluster Analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Many of the distinct clones in our dataset for which we measured a transcript length are associated with the same Unigene cluster. We examined all such clusters to determine if the length measurements from all clones associated with the same cluster were the same. Distinct clones belonging to the same Unigene cluster can sometimes detect alternative transcripts with distinct expression patterns (Jeremy Gollub, personal communication). This discordant behavior can be caused by a number of technical factors including chimeric or misidentified clones, Unigene cluster or human genome assembly errors, or cross-hybridization artifacts. A more interesting cause of discordant expression patterns is the inclusion of clones in the same Unigene cluster that inadvertently represent different transcript variants of the same gene.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We began by analyzing the transcipt length measurements derived from microarray hybridization data for the 1,236 cDNA clones that map to more than one Unigene cluster. Clones can map to as many as three clusters due to any of the factors discussed above. We compared the length measurements for each of the 1,236 clones to the length measurements of all of the other clones in our dataset that map to any of the same clusters. For many of the clusters in this set, there were no other clones in our dataset that mapped to the same cluster, so in many cases we could only compare a clone to a subset of the clusters to which it belongs. We performed this analysis on lengths of 34%, 50%, and 90% confidence, but obtained similar results in each case. The following represents the analysis of the 90% confidence lengths. For 217 clones, we could compare the clone to only one cluster. In 108 (50%) of those cases, the clone matched (<em>i.e.</em> shared a length in common with) that cluster. For 115 clones, we compared the clone to two clusters. In 61 (53%) of those cases the clone matched only one cluster, in another 29 (25%) of those cases the clone matched both clusters, and in 25 cases (22%) it matched neither. For the two clones that we could compare to three clusters, one matched one of the three clusters and the other matched two. Although we were not able to compare each clone to each of the clusters to which it belongs, there is clearly a high prevalence of both clones that match only a single cluster and clones that match multiple clusters. Therefore, the set of clones that map to multiple Unigene clusters are not simply incorrectly assigned to multiple clusters. Many appear to hybridize to mRNAs transcribed from multiple genes.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We then analyzed 4,883 Unigene clusters with an average of 2.6 clones per cluster (<a href="#pone-0000460-t003">Table 3</a>). Excluding clones that mapped to multiple Unigene clusters, we grouped the clones from each cluster into subclusters based on whether or not the microarray hybridization results derived from those clones yielded the same transcript length measurements. Clones yielding concordant transcript lengths were placed in the same subcluster while clones yielding discordant transcript lengths were placed in separate subclusters. Using lengths of at least 90% confidence, 28% of the Unigene clusters were broken up into 2 to 3 subclusters.</p>
<div class="figure" id="pone-0000460-t003"><div class="img"><a name="pone-0000460-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Unigene cluster analysis</span></strong></p><span>doi:10.1371/journal.pone.0000460.t003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>We also identified groups of clones from the same Unigene cluster containing overlapping sequence. Using information available from the UCSC genome browser <a href="#pone.0000460-Karolchik1">[10]</a>, we determined the genome coordinates of the cDNA clones in our dataset. Excluding clones that mapped to multiple locations in the genome, we subclustered clones belonging to the same Unigene cluster on the basis of whether or not their genome coordinates were overlapping. With the 90% confidence length set, 67% of the Unigene clusters were broken up into 2 to 5 subclusters based on their genome coordinates.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>We then compared the length-based subclusters to the coordinate-based subclusters in order to explore the possibility that the length-based subclusters represent alternate transcript variants that are being detected by clones that map to distinctly different sections of the underlying gene. As a result, we found a high degree of similarity between the patterns of length- and coordinated-based subclustering. With the 90% confidence length set, 64% of the pairs of clones in different length subclusters were also in different coordinate subclusters. This was a significantly higher percentage than would be expected based on chance (chi-squared test, p = 7.2×10<sup>−9</sup>). One interesting result is shown in <a href="#pone-0000460-g004">Figure 4</a> for four clones representing the glucose transporter <em>SLC2A1</em> gene. IMAGE clone 151248 was split from the other two clones in its Unigene cluster on the basis of both length and genome alignment. Although IMAGE clones 207358, 25389, and 2547341 are perfect matches for the 3′-end of the <em>SLC2A1</em> gene and identify lengths that agree with the annotated <em>SLC2A1</em> transcript, clone 151248 aligns within an intron and identifies a distinctly shorter transcript. Clone 151248 may therefore represent a novel exon of <em>SLC2A1</em>, a novel gene residing in <em>SLC2A1</em>'s intron, an unspliced <em>SLC2A1</em> pre-mRNA species, or even accidentally cloned genomic DNA. Based on additional EST evidence and conservation with multiple other vertebrate genome sequences <a href="#pone.0000460-Karolchik1">[10]</a>, it appears to represent a novel <em>SLC2A1</em> transcript as shown in <a href="#pone-0000460-g004">Figure 4</a>. Likewise, the majority of discrepancies between lengths measured by different clones from the same Unigene cluster that we have identified are due to clones that deviate from the established annotation. Many of them likely represent novel transcripts.</p>
<div class="figure" id="pone-0000460-g004"><div class="img"><a name="pone-0000460-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Solute carrier family 2 (facilitated glucose transporter), member 1 (<em>SLC2A1</em>) gene.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The <em>SLC2A1</em> gene is pictured schematically. The transcribed portion of the gene is shown with the filled boxes representing exons. The ORF is represented by the taller boxes. The genomic positions of four cDNA clones that map only to the <em>SLC2A1</em> gene, and a proposed novel <em>SLC2A1</em> transcript, are shown relative to the <em>SLC2A1</em> gene. The transcript length measured for each clone is shown in parentheses.</p>
<span>doi:10.1371/journal.pone.0000460.g004</span></div>

<h4>Comparison of measured transcript length to the Refseq and H-Inv databases</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We compared our transcript length dataset to the lengths of the sequences in the human Refseq database. Using information downloaded from the NCBI website (<a href="http://www.ncbi.nlm.nih.gov/Ftp/">http://www.ncbi.nlm.nih.gov/Ftp/</a>), we were able to associate 17,364 unique clones for which we had a total of 23,324 length measurements with 11,530 Entrez genes, and 15,104 Refseqs. There are more Refseqs than genes, because the Refseq database stores alternate transcripts associated with the same loci, so some genes in the dataset have as many as 21 associated Refseqs.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We compared each of the length measurements in our dataset with each of the Refseqs for the clone's parent gene. We declared a match if our length measurement deviated by no more than three fraction lengths (16.5%) from a poly(A) tail-corrected Refseq length for the same gene. Considering only length measurements of ≥90% confidence, 61% of 7,182 lengths matched a Refseq. In order to determine the significance of the match between Refseq and our measurements, we used a permutation analysis to calculate the match that would occur between our data and Refseq based on chance. The association between genes and Refseq lengths was randomized 10 times, and each time the match between our data and the Refseq lengths of unrelated genes was calculated. The frequency of random matches was consistently about 22% across all permutations at all confidence levels. For the length measurements that did not match their corresponding Refseq lengths, we calculated which was longer. The Refseq was typically longer. At 90% confidence, the Refseq length exceeded our length measurement in 4,378 cases (77%).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>It is not entirely surprising that our transcript length dataset matches Refseq only half of the time. If we assume that the human genome encodes 25,000 genes <a href="#pone.0000460-International1">[3]</a>, that 74% of all genes are alternately spliced with an average of 2.7 alternate splice forms <a href="#pone.0000460-Johnson1">[11]</a>, <a href="#pone.0000460-Lander1">[1]</a>, <a href="#pone.0000460-Imanishi1">[4]</a>, and that 24% of all genes have alternate polyadenylation sites <a href="#pone.0000460-Lander1">[1]</a>, we would conservatively estimate that the human genome encodes a total of 70,000 different transcripts. As of this writing the Refseq database (Release 12) contains 29,476 mRNA sequences, approximately 42% of the estimated 70,000. Furthermore, the brain is known to be the tissue with the greatest amount of alternative splicing <a href="#pone.0000460-Stamm1">[12]</a>, <a href="#pone.0000460-Xu1">[13]</a>. It is therefore likely that our Virtual Northern experiment has detected many mRNA transcript variants that remain to be annotated in Refseq.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>We also compared our transcript length dataset to the lengths of the cDNA clones in the H-Invitational (H-Inv) full-length cDNA database <a href="#pone.0000460-Imanishi1">[4]</a>. Release 2 of the H-Inv database contains 56,419 cDNA clones representing a total of 25,585 different loci. Entrez genes were associated with 40,623 of those. We were therefore able to associate 15,403 clones for which we had a total of 20,583 length measurements with 45,005 H-Inv cDNAs. We compared our length measurements to the H-Inv clone lengths just as we compared them to Refseq. The correspondence between our 90% confidence length measurements and the H-Inv cDNAs was 69%. Although we calculated a higher correspondence between our dataset and the H-Inv clones than we did to Refseq, an analogous permutation analysis showed that the probability of a random match was also higher, approximately 35%. Thus, the correspondences between our dataset and Refseq, and our dataset and H-Inv are similar. We also examined the lengths in our dataset that did not match an H-Inv clone to see which was longer. Here, we found an almost 50-50 split across all confidence levels, indicating no systematic difference in lengths between our measurements and the H-Inv clones.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>In order to better understand the relationship between our lengths and the Refseq and H-Inv databases, we directly compared the Refseq and H-Inv databases using the same procedure with which we had compared them to our data. The correspondence in length between the two was 58% with a 21% probability of a random match. Thus the correspondence between a well-annotated cDNA clone set and an mRNA transcript database based on cDNA sequence was no greater than the correspondence between either resource and our length dataset. This was somewhat unexpected since it is possible for Refseqs to be based on cDNAs included in the H-Inv clone set. We also found a strong bias for the Refseq to be longer than the H-Inv clone. Nearly 79% of the Refseqs that were not approximately equal in length to their corresponding H-Inv clones were longer. Comparison of Refseqs and available cDNA sequences from the same gene to the genome sequence quickly explained the strong bias toward longer Refseqs. In every case we've examined, the Refseq extends from the 5′-most cDNA to the 3′-most cDNA even when those cDNAs are anomalies that sometimes extend far past the endpoints of virtually all other available cDNA sequences. Thus, there is a tendency for Refseqs to contain additional sequence on their 5′- and 3′-ends beyond what is commonly observed for transcripts from the same gene. It remains to be determined whether this additional sequence represents biologically relevant sequence whose inclusion in the mRNA is rarely needed, or if it represents rare transcriptional accidents.</p>


<h4>Relationship between ORF Length and mRNA length</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We had previously identified a close linear relationship between open reading frame (ORF) length and mRNA length in the yeast genome <a href="#pone.0000460-Hurowitz1">[9]</a>, so we were interested in determining whether a similar relationship exists for human mRNAs. Since Refseq entries include coding sequence information, we extracted the position of the ORF from every Refseq entry and used that information to calculate the length of the 5′-UTR, ORF, and 3′-UTR for every Refseq. We excluded Refseqs for noncoding RNAs and Refseqs with calculated 5′- or 3′-UTRs less than 20 nucleotides from further analysis (20% of the total). <a href="#pone-0000460-g005">Figure 5</a> shows a plot of the total length of each human Refseq mRNA versus the length of its ORF. Remarkably, mRNA length shows an excellent fit to a linear relationship with ORF length (R = 0.86). Furthermore, the parameters of the best fit line also show that, even over the entire range from 300 to 104,000 nucleotides, transcript length closely approximates the ORF length plus a fixed length of approximately 1,263 nucleotides. This is remarkably similar to the relationship between ORF and mRNA lengths in the yeast genome. The difference is the average total UTR length, which is approximately four times longer in humans than in yeast. Furthermore, the average total UTR length estimated by the linear fit agrees well with previous estimates of UTR lengths in human mRNAs of 1,238 nucleotides <a href="#pone.0000460-Mignone1">[14]</a> and 1,070 nucleotides <a href="#pone.0000460-Lander1">[1]</a>. In order to further characterize the relationship between coding regions and untranslated regions in human mRNAs, we calculated the correlations between ORF and UTR length, 5′-UTR and 3′-UTR length, 5′-UTR and ORF length, and 3′-UTR and ORF length (<a href="#pone-0000460-t004">Table 4</a>). These correlations are all very small and not all positive. This means that the lengths of the three components of an mRNA, the 5′-UTR, ORF, and 3′-UTR, are essentially uncorrelated.</p>
<div class="figure" id="pone-0000460-g005"><div class="img"><a name="pone-0000460-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Relationship between ORF length and transcript length.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Refseq length in nucleotides is plotted against ORF length in nucleotides. The black line is the linear least squares fit. It has the parameters mRNA = 1.03 (ORF)+1263 (R<sup>2</sup> = 0.74).</p>
<span>doi:10.1371/journal.pone.0000460.g005</span></div><div class="figure" id="pone-0000460-t004"><div class="img"><a name="pone-0000460-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Correlations between ORF, UTR, and mRNA lengths</span></strong></p><span>doi:10.1371/journal.pone.0000460.t004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We also performed a similar analysis with the H-Inv database. H-Inv clone records also include coding sequence information, so we were able to calculate the 5′-UTR, ORF, and 3′-UTR lengths for every H-Inv cDNA. Removal of H-Inv clones with 5′- or 3′-UTRs less than 20 nucleotides filtered out nearly half of the clones (42%). We calculated the same correlations as before. As shown in <a href="#pone-0000460-t004">Table 4</a>, the correlations are all quite different between H-Inv and Refseq. Collectively, these results suggest that many of the H-Inv cDNA clones are not full-length. ORF length is negatively correlated with both 5′- and 3′-UTR length. This means that the 5′-UTRs for longer ORFs are generally shorter. Since full-length cDNAs are produced by reverse transcribing mRNA by priming from the poly(A) tail, it follows that putatively full-length cDNAs are more likely to be complete on their 3′-end. Since the primary difficulty in reverse transcribing long mRNAs is the relatively high probability that the reverse transcriptase will fall off the RNA before it finishes transcribing the entire mRNA, it also follows that cDNAs with longer ORFs are less likely to contain complete 5′-UTRs (or even complete ORFs). This is exactly what we observed. The most negative correlation we calculated was between ORF and 5′-UTR length. It should be noted that the presence of significant numbers of incomplete clones in the H-Inv set did not affect the overall correspondence between the lengths from our dataset and the lengths of the H-Inv clones. The genes used in the comparison were represented by an average of three H-Inv clones. As long as any one of those clones was full-length, our length measurement could register a match. We repeated the comparison of our length dataset to the H-Inv clone lengths but with the clones with short UTRs removed. The percentage of length measurements with a match fell 5–6% at all confidence levels, but the probability of a random match fell by a slightly greater amount at all confidence levels, indicating that removal of those clones essentially removed only accidental matches.</p>


<h4>Identification of functionally distinct groups of genes whose UTR lengths deviate significantly from the norm</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We investigated whether the length of an mRNA or any of its component sequences has any relationship to the biological role of the protein it encodes. In order to accomplish this, we downloaded the Gene Ontology (GO) annotation (biological process, molecular function, and cellular component) for every clone in our dataset that was associated with a single Entrez gene. We calculated the total UTR length for every length measurement in our dataset that could be associated with a Refseq by subtracting the length of the Refseq's ORF from the length measurement. We then compared the distribution of lengths for every distinct GO classification annotated to at least 10 lengths in our dataset with the distribution of all lengths in the dataset using a Student's t-test (two-tailed, unequal variance). We also performed this analysis using the Refseq lengths themselves. These analyses identified groups of genes with the same GO annotation whose UTR lengths were significantly longer or shorter than average. The results using our length measurements and the Refseq lengths were largely the same. The results were also largely the same across all confidence levels. <a href="#pone-0000460-t005">Tables 5</a> and <a href="#pone-0000460-t006">6</a> show the five most significant GO annotations for UTRs both longer and shorter than average along with their associated ontology and t-statistics (using the 90% confidence lengths).</p>
<div class="figure" id="pone-0000460-t005"><div class="img"><a name="pone-0000460-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Top five GO annotations whose UTRs are significantly shorter than average</span></strong></p><span>doi:10.1371/journal.pone.0000460.t005</span></div><div class="figure" id="pone-0000460-t006"><div class="img"><a name="pone-0000460-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Top five GO annotations whose UTRs are significantly longer than average</span></strong></p><span>doi:10.1371/journal.pone.0000460.t006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>One remarkable thing about the lists of GO annotations associated with deviant UTR lengths is their similarity to the analogous lists generated for the yeast genome <a href="#pone.0000460-Hurowitz1">[9]</a>. In both organisms, the UTRs of mRNAs encoding ribosomal, nucleolar, and proteasomal proteins are shorter than average, while the UTRs of mRNAs encoding proteins involved in signal transduction, transcriptional regulation, regulation of metabolism, and cell cycle regulation are longer than average. We previously hypothesized that longer than average UTRs contain additional sequences important for the regulation of the translation, cellular localization, or decay of their mRNA. We believe that the observation that the UTRs of mRNAs encoding proteins of regulatory function are longer than average in man as well as yeast strengthens this hypothesis and points to the importance of post-transcriptional regulation in the fate of mRNAs of eukaryotes in general.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>We also searched for GO annotations enriched among genes for which we had identified more than one putative transcript length. We performed a t-test analysis as before. However, no GO annotations showed statistical significance. We then repeated this analysis to search for GO annotations enriched in genes with multiple Refseqs. In this analysis, we identified a small number of GO annotations that had more Refseqs than average. Although these GO annotations were only marginally significant statistically, many of them were involved in signal transduction, specifically protein phosphorylation and dephosphorylation, and transcriptional regulation. A previous analysis of alternative splicing in human <a href="#pone.0000460-Modrek1">[7]</a> also identified genes involved in signal transduction and transcriptional regulation as having higher than average numbers of alternative splice forms.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We applied the Virtual Northern technique to the human genome. Using mRNA purified from human brain as our sample, we obtained provisional length measurements from 21,257 cDNA clones representing a total of 11,536 human genes. Thus, we were able to derive at least one measurement of transcript length for nearly half of the 25,000 genes the human genome is predicted to encode, and from 6,238 of those genes at high (90%) confidence. This is a reasonably high fraction considering that we analyzed mRNA from only a single organ, albeit the organ with the highest transcriptional diversity <a href="#pone.0000460-Stamm1">[12]</a>, <a href="#pone.0000460-Xu1">[13]</a>. Our transcript length dataset has a mean and median of 2,165 nucleotides and 1,996 nucleotides respectively. These numbers agree well with previous estimates for the human genome <a href="#pone.0000460-Lander1">[1]</a>. At high (≥90%) confidence, only about 1.3% of the clones in our dataset detected two transcript lengths. Current estimates for alternative splicing are that 74% of multi-exon genes have alternate splice forms, and that alternatively spliced genes have an average of 2.7 different splice forms <a href="#pone.0000460-Johnson1">[11]</a>, <a href="#pone.0000460-Imanishi1">[4]</a>. Our detection rate for alternative transcript variants is expected to fall short of those estimates for two reasons. First, we only examined a single tissue, so our analysis was only able to detect transcript variants expressed in the brain. Second, our length fractionation procedure had a theoretical maximum resolution of about 5–6% of total transcript length. Any transcript variants whose lengths differ by less than that would not be reliably resolved. That range is sufficient to exclude detection of alternative splices resulting from the use of alternate exons of similar length, the inclusion/exclusion of a single short exon, or the use of alternate nearby 5′ or 3′-splice sites.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We compared our transcript length dataset to the lengths of the sequences in the human Refseq database. A total of 23,324 length measurements were compared to 15,104 Refseqs. For length measurements of greater than 90% confidence, 61% matched the length of a transcript recorded in the Refseq database. A rough calculation estimates that the Refseq database currently contains only about 40% of the transcripts encoded by the human genome. With about half of the length measurements in our dataset unrepresented by a Refseq sequence, our data supports that estimate.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Analysis of the transcript length measurements without a matching Refseq showed that the Refseqs from the corresponding gene were longer than our length measurement nearly three quarters of the time. We also observed this bias toward longer Refseqs in our comparison between the Refseq and H-Inv full-length cDNA clone databases. This appears to be due to the inclusion in many Refseqs of all of the sequence from the 5′-most cDNA to the 3′-most cDNA for a given gene. Although this is presumably done to counter the difficulty of identifying full-length cDNAs and to ensure for many mRNAs that the full transcript is represented, it appears that, for many genes, the most extreme boundaries for a transcript are not necessarily the most common. It remains to be determined whether the additional sequence occasionally observed on the ends of transcripts is, for some genes, biologically relevant sequence regulated to occur in only a fraction of transcripts, or if it is merely unregulated, irrelevant transcriptional noise.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We found a striking relationship between human mRNA lengths and the lengths of the ORFs they encode. Comparison of the ORF and mRNA lengths of the sequences in the Refseq database identified a close linear relationship between ORF and mRNA length. Remarkably, this linear relationship has the same form as the relationship we identified between yeast mRNAs and their ORFs, namely that the mRNA length equals the ORF length plus a fixed length of approximately 1,263 nucleotides. Thus in man, as in yeast, the distribution of lengths of untranslated regions in mRNAs does not depend on the length of the ORF.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Several analyses suggested that the H-Inv full-length cDNA clone database contains many clones that are unlikely to be full-length. First, nearly half of the clones in the database have 5′- or 3′-UTRs shorter than 20 nucleotides. In many of these cases the 5′-UTR is completely missing. In contrast to the strong correlation between ORF and mRNA length, and the small correlation between ORF and 5′-UTR length that we identified in Refseq, the H-Inv clones showed a weaker correlation between ORF and clone length and a moderately negative correlation between ORF and 5′-UTR length. This is exactly the relationship we would expect for a set of cDNAs that are not truly full-length, since the primary difficulty in producing full-length cDNAs is obtaining the full 5′-end. Comparison of the H-Inv clones to our mRNA length dataset can now be used as an additional criterion to determine which individual cDNAs are not full-length.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Having a discovered a relationship in yeast between the length of an mRNA's untranslated region and the function of the protein that it encodes, we performed a similar analysis with our human dataset. We were able to identify various functional classes whose UTRs were either longer or shorter than average. Remarkably, there was substantial similarity between yeast and man in this relationship. In both organisms, the UTRs of mRNAs encoding ribosomal, nucleolar, and proteasomal proteins were shorter than average while the UTRs of mRNAs encoding proteins involved in signal transduction, transcriptional regulation, and the regulation of metabolism were longer than average. A recent study of microRNAs extended this observation to <em>Drosophila</em>, and suggested that the observed relationship was due to an evolutionary pressure to enrich or deplete microRNA target sites from the mRNAs of certain genes <a href="#pone.0000460-Stark1">[15]</a>. Ubiquitously expressed genes involved in basic cellular processes appear to have evolved mRNAs with short UTRs, perhaps reflecting a relative paucity of regulation by RNA binding proteins and microRNAs. In contrast, genes involved in more complex regulatory processes such as embryonic development have evolved longer UTRs with many microRNA target sites and perhaps also binding sites for RNA binding proteins, providing an additional layer of regulation. A few functional groups of genes were more likely to have mRNAs of more than one length. Our analysis corroborated a previous analysis of alternative splicing, which identified genes involved in signal transduction and transcriptional regulation as having higher than average numbers of alternative splice forms <a href="#pone.0000460-Modrek1">[7]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>RNA preparation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>One whole human brain from a 36 year old male was flash frozen in liquid nitrogen approximately 18 hours postmortem. Brain tissue was provided by Hannes Vogel, Professor of Pathology at Stanford University School of Medicine. Consent was obtained from the donor to use his tissues for research purposes, and the Stanford Medical Board approved their use in this study. The whole brain was crudely pulverized while frozen, and 360 g of tissue was homogenized in a Waring blender containing 38% phenol, 5% glycerol, 10% 1 M sodium acetate pH 4.8 by volume, guanidinium thiocyanate at a final concentration of 0.8 M, ammonium thiocyanate at a final concentration of 0.4 M, and approximately 1 mg/L of the red dye Sudan III. Total RNA was prepared as previously described <a href="#pone.0000460-Chomczynski1">[16]</a>, and poly(A) purified using the Oligotex™ mRNA kit from Qiagen to create the final human brain mRNA sample.</p>


<h4>Length fractionation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A 50 µg aliquot of the poly(A) RNA and a RNA ladder (a mix of the Millennium™ and Century™ markers from Ambion) was heat denatured for 10 minutes at 65°C in formamide, placed on ice, and immediately loaded on a 1.1% low melting point agarose gel in 1X TAE running buffer. The poly(A) RNA was loaded in a wide (5 cm) well to prevent overloading of the gel. The RNA was then separated by electrophoresis at high voltage (~9 V/cm) for 2–2.5 hours. During electrophoresis, the running buffer was passively recirculated, and the apparatus was cooled to prevent the gel and running buffer from overheating. The ladder lane was excised from the gel, and visualized by ethidium bromide staining. Based on the mobility of the ladder bands, a portion of the poly(A) RNA lane, corresponding to RNAs of approximately 400–7000 nucleotides in length, was cut into 50 slices of 2 mm each. RNA was then recovered from each length fraction by β-agarase digestion. Each agarose slice was melted at 70°C for 10 minutes, and then digested with 2 Units of AgarACE enzyme (Promega) at 42°C for 2 hours. The products from each agarase reaction were recovered using a Microcon-30 column (Amicon) and washed twice with 10 mM Tris pH 7.0 buffer. In total, length fractionation was performed in triplicate using three gels that were run and sliced independently.</p>


<h4>Microarray analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>One third of the recovered material from each length fraction was reverse transcribed in the presence of amino allyl-dUTP. A mix of dT<sub>20</sub> and random nonamer was used to prime the reaction in order to maximize the reverse transcription of both polyadenylated and non-polyadenylated RNAs, and to facilitate the labeling of the entire RNA sequence. The resulting cDNA was fluorescently labeled by coupling reactive Cy5 to the amino allyl groups on the incorporated dUTP. Each labeled length fraction was then hybridized to a separate human cDNA microarray. Construction of the human microarrays and hybridization of labeled cDNA to them was accomplished as previously described <a href="#pone.0000460-Perou1">[17]</a>. Printing of the microarrays was preformed by the Stanford Functional Genomics Facility. To provide an internal hybridization reference, a 1.5 µg aliquot of the human poly(A) RNA was reverse transcribed using the same protocol as the length fractions, fluorescently labeled by coupling the cDNA to Cy3, and included in each hybridization. Microarrays were scanned with an Axon Instruments (Foster City, CA) scanner, and the data collected with GENEPIX PRO 5.1 software (Axon Instruments).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The Cy5/Cy3 fluorescence ratio data was then filtered for quality. Spots with defects apparent from visual inspection, or a coefficient of variation of the signal intensity in either channel greater than 120 were excluded from further analysis. Spots were also filtered on the basis of the background-corrected intensity in the reference channel. The cutoff was determined independently for each microarray by calculating the standard deviations of ratios from duplicated spots. For each microarray, the mean of those standard deviations was calculated with the data from spots with a background-corrected reference channel intensity less than 150 filtered out. If the mean was greater than 0.2, the filter cutoff was increased in increments of 25 until the mean was less than 0.2. For 128 of the 150 microarrays, a cutoff of 150 was used. For the remaining 22 microarrays, the cutoff ranged from 175 to 675. Finally, for each electrophoretic profile based on the 50 measurements for each analyzed DNA sequence, the median background-corrected reference channel intensity (MRI) for all spots in the profile was calculated. Profiles with more than 10 missing data points, or whose MRI was less than 250, were excluded from further analysis. Sets of three electrophoretic profiles based on the same analyzed DNA sequence from each independent length fractionation were excluded from further analysis if any of the individual profiles were excluded.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>For normalization between the length fractions, an internal standard was prepared with a pool of <em>in vitro</em>-transcribed <em>Bacillus subtilis</em> RNAs <a href="#pone.0000460-Wang1">[18]</a>. PCR products representing five different <em>B. subtilis</em> DNAs were printed onto the microarrays and <em>in vitro</em> transcribed into RNA. A mix of 400 pg of each <em>B. subtilis</em> RNA was doped into each gel slice during the agarase digestion, and into each 1.5 µg reference aliquot before labeling. The internal standard gives us a way to account for differences in the efficiency of RNA recovery between the length fractions. The data from each length fraction was normalized by the value that set the median of the <em>B. subtilis</em> ratios to 1.5. Due to slight systematic errors in the normalization of a handful of length fractions, each length fraction was additionally normalized by making a least squares fit of the mean of all ratios from each of the fifty length fractions to a fourth order polynomial, and normalizing the data from each length fraction by the value that set each mean to the polynomial's corresponding value. This normalization “smoothing” had only a minor, fine-tuning effect, modifying each length fraction by only 1.2-fold on average.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>In order to facilitate comparison of data between the three separate length fractionations, an internal standard was prepared with a pool of <em>in vitro</em>-transcribed <em>Methanococcus jannaschii</em> RNAs. PCR products representing approximately 40 regions from the <em>M. jannaschii</em> genome ranging from 250–10,000 bps were printed onto the microarrays, and <em>in vitro</em> transcribed into RNA. A mix of 400 pg of each <em>M. jannaschii</em> RNA was doped into each 50 µg aliquot of the brain mRNA preceding gel electrophoresis, and into each 1.5 µg reference aliquot before labeling. This set of synthetic mRNAs fractionated along with the human brain mRNA gives us a series of profiles with known and consistent characteristics to assist in comparing the data between the three separate length fractionations. Both the <em>B. subtilis</em> and <em>M. jannaschii</em> RNAs were <em>in vitro</em>-transcribed using respectively the T3 and T7 MEGAscript® kits from Ambion.</p>


<h4>Aligning the data from the three gels</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Since the RNA from each length fractionation was electrophoresed for a slightly different period of time, the slices from each gel occur at different distances from the well and represent different ranges of RNA length. In order to compare the length profiles for a given gene generated from each length fractionation, it was necessary to scale the data for two of the length fractionations along the gel mobility axis so that they correspond to the data from the third length fractionation. Peaks were manually identified in the profiles of the <em>M. jannaschii</em> controls in all three fractionations. Corresponding peaks were compared, and nearly linear relationships between gel mobilities for each of the gels were identified. The least squares fit to a quadratic equation was calculated for each relationship. We applied piecewise cubic spline interpolation to each length profile, and used the above relationships to convert the gel mobilities for the length profiles from gels 2 and 3 into their equivalent mobilities in gel 1.</p>


<h4>Peak finding by <em>l</em><sub>1</sub> norm baseline deconvolution</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>In order to identify a small set of underlying peaks within a large set of potentially noisy microarray-derived length measurements, we applied a convolutional model to our data. In an abstract fashion, finding peaks in our length profiles is equivalent to finding the sparsest solution to an underdetermined system of equations: min ||<em>x</em>||<sub>0</sub> subject to <em>y</em> = <em>Ax</em>, where <em>y</em> is an observed measurement, <em>x</em> is the underlying signal, <em>A</em> is the convolution operator, and ||<em>x</em>||<sub>0</sub> represents the number of non-zeros in the signal. This is a non-convex combinatorial optimization problem, and in general finding the sparsest solution is NP hard. Therefore we solve the problem for the <em>l</em><sub>1</sub> norm <a href="#pone.0000460-Chen1">[19]</a>, which can be cast as a standard linear program which is convex and tractable, and solved efficiently using general purpose solvers such as simplex and interior point methods <a href="#pone.0000460-Saunders1">[20]</a>. In addition when the solution is sufficiently sparse there is equivalence between the <em>ell</em><sub>1</sub> and sparsest solutions <a href="#pone.0000460-Donoho1">[21]</a>, <a href="#pone.0000460-Donoho2">[22]</a>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>In order to account for noise in our data <a href="#pone.0000460-Donoho3">[23]</a>, we cast our problem as: min ||<em>x</em>||<sub>1</sub> subject to ||<em>y</em>−<em>Ax</em>||≤ε. While this model accounts for spurious peaks within noisy data, it is not sufficiently suitable for our purposes. In our case, we would also like to accommodate signals with low frequency content which the deconvolution model represents as multiple peaks. We therefore incorporate a rolling baseline into the model such that the observation <em>y</em> is represented as a smooth baseline <em>u</em> with peaks on top <em>v</em> and noise, and solve the optimization problem: min ||<em>x</em>||<sub>1</sub>+<em>u</em>||<em>β</em>||<sub>1</sub>+½||<em>r</em>||<sub>2</sub><sup>2</sup> subject to <em>u</em>+<em>v</em>+<em>r</em> = <em>y</em>, <em>Ax</em> = <em>v</em>, and Δ<sup>2</sup><em>u</em> = <em>β</em>. In other words, we solve for the underlying peaks which when convolved with the kernel and added to the smooth baseline result in the observed data. <a href="#pone-0000460-g002">Figure 2</a> shows an original length profile along with the rolling baseline and the result of deconvolution.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>The kernel was derived directly from the data by identifying a large number of likely peaks, lining up those peaks, and taking the median. Peaks were identified in all length profiles by counting each position greater than its two neighboring positions with a normalized ratio greater than four (a high threshold for this dataset) as a peak. The median of the normalized ratio in all of those peak positions was calculated as well as the median of the values of each of the five length fractions to each side of the peak position. This created an 11 element kernel (<a href="#pone-0000460-g006">Figure 6</a>).</p>
<div class="figure" id="pone-0000460-g006"><div class="img"><a name="pone-0000460-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Deconvolution kernel.</span></strong></p><a id="article1.body1.sec4.sec5.fig1.caption1.p1" name="article1.body1.sec4.sec5.fig1.caption1.p1"></a><p>The stereotypical peak used as the deconvolution kernel to identify potential peaks is shown.</p>
<span>doi:10.1371/journal.pone.0000460.g006</span></div>

<h4>Estimating a confidence value for each peak</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>In order to ascertain a confidence for each peak, we assigned a probability to each length fraction for every profile. This value describes the likelihood at every position that we have uncovered a true peak at that position. We calculated those probabilities by sampling each value of the deconvolved signal with Poisson noise to create 100 parametric bootstrap replicates. More specifically, given the baseline deconvolution result according to our model, we simulated the experimental data separately at each position by convolution and a Poisson model. Then, we reconstructed each simulation by deconvolution to obtain 100 bootstrap replicates. We counted the fraction of these reconstructions in which there was a peak and computed a probability.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>The final dataset of peaks was determined from the bootstrap probability profiles derived from each length profile. First, the results from each of the three gels were combined by averaging the probabilities for each position from each of the three experiments. Every non-zero position in the combined profiles with a probability value greater than its two adjacent positions was called a peak. A precise position for each peak was calculated as the weighted average of each peak position and its two adjacent positions. The probability for each peak was taken to be the sum of the probabilities of the peak position and its two adjacent positions.</p>


<h4>Conversion of gel mobility to transcript length</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>A “gold standard” set of genes was developed to help evaluate the human Virtual Northern data set. The set consists of 100 abundantly expressed genes that are represented on the microarrays by at least one clone, have at least one identified peak in the dataset, and have at least one curated Refseq <a href="#pone.0000460-Wheeler1">[8]</a>. The gold standard genes are derived from the Entrez genes whose associated Unigene clusters have the largest number of clones in them. They are therefore extremely well-sampled by currently existing cDNA and EST databases giving us a far greater than average understanding of the sequence and structure of their mRNAs. Although these genes are all abundantly expressed in at least one tissue, they are not necessarily abundantly expressed in brain. The distribution of microarray signals for the gold standard genes is not significantly different from the dataset as a whole, so they provide a fair representation of the entire dataset, and do not just represent the most well-measured genes in the dataset.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>The estimated mobilities of all peaks identified from clones corresponding to gold standard genes were compared to the known lengths of all of the gold standard genes. As expected, the estimated mobilities showed an excellent linear fit to the natural log of the known lengths. Gross outliers (spots deviating from the linear fit by greater than 0.2) were removed, and the least squares fit to a linear equation was calculated (<a href="#pone-0000460-g003">Figure 3</a>). A model of the poly(A) tail was incorporated into the linear equation by adding a fixed length to each known length. A fixed length of 225 nucleotides was calculated along with the slope and y-intercept to provide the best fit between estimated mobilities and transcript length. Using the resulting equation, the estimated mobility of each peak could be converted to an inferred transcript length in nucleotides.</p>


<h4>Calibration of bootstrap confidence value to true peak probability</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The bootstrap probabilities provide a confidence value for every peak. They are related to, but not equivalent to the probability that a peak is <em>bona fide</em>. In order to determine the correspondence between the bootstrap probabilities and the probability that a peak is real, we compared our length dataset to the known lengths from our gold standard gene list. Measured lengths within three fractions of a known length were considered true positives, and all other measured lengths were considered false positives. For each bootstrap value, we calculated the probability that a peak is real as the fraction of peaks at that value that were true positives. The resulting curve (<a href="#pone-0000460-g007">Figure 7</a>) was fit by least squares to the sigmoidal equation y = A+B/(1+e <sup>(m−x)/l</sup>), and that equation was used to convert bootstrap probabilities into true positive peak probabilities.</p>
<div class="figure" id="pone-0000460-g007"><div class="img"><a name="pone-0000460-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000460" data-uri="info:doi/10.1371/journal.pone.0000460.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000460.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000460.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000460.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000460.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Calibration between bootstrap value and true positive rate.</span></strong></p><a id="article1.body1.sec4.sec8.fig1.caption1.p1" name="article1.body1.sec4.sec8.fig1.caption1.p1"></a><p>The bootstrap value is plotted against the true positive rate for bootstrap values from 0 to 0.6. The least squares fit to a sigmoidal equation is shown by a solid line with the parameters y = −0.07563+1.09892/(1+e <sup>(0.15928−x)/0.08257</sup>).</p>
<span>doi:10.1371/journal.pone.0000460.g007</span></div>

<h4>Data Availability</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>All data is available for download at <a href="http://microarray-pubs.stanford.edu/humanVN/">http://microarray-pubs.stanford.edu/huma​nVN/</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Special thanks to Professor Hannes Vogel of Stanford's Department of Pathology who provided the human brain tissue used in this study, and to Dr. Jeremy Gollub, formerly a postdoctoral scholar in Stanford's Department of Biochemistry, who provided useful discussion about our Unigene analyses.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PB EH. Performed the experiments: EH. Analyzed the data: EH ID VS. Wrote the paper: EH. Other: Edited paper: PB. Advised on the data analysis DD.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000460-Lander1" id="pone.0000460-Lander1"></a>Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, et al.  (2001) Initial sequencing and analysis of the human genome. Nature  409: 860–921.  <ul class="find" data-citedArticleID="953062"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+analysis+of+the+human+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+analysis+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000460-Venter1" id="pone.0000460-Venter1"></a>Venter JC, Adams MD, Myers EW, Li PW, Mural RJ, et al.  (2001) The sequence of the human genome. Science  291: 1304–1351.  <ul class="find" data-citedArticleID="953080" data-doi="10.1126/science.1058040"><li><a href="http://dx.doi.org/10.1126/science.1058040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sequence+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sequence+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000460-International1" id="pone.0000460-International1"></a>International Human Genome Sequencing Consortium (2004) Finishing the euchromatic sequence of the human genome. Nature  431: 931–45.  <ul class="find" data-citedArticleID="953056" data-doi="10.1038/nature03001"><li><a href="http://dx.doi.org/10.1038/nature03001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finishing+the+euchromatic+sequence+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finishing+the+euchromatic+sequence+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000460-Imanishi1" id="pone.0000460-Imanishi1"></a>Imanishi T, Itoh T, Suzuki Y, O'Donovan C, Fukuchi S, et al.  (2004) Integrative annotation of 21,037 human genes validated by full-length cDNA clones. PLOS Biology  2: 1–20.  <ul class="find" data-citedArticleID="953054"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Integrative+annotation+of+21%2C037+human+genes+validated+by+full-length+cDNA+clones.&amp;auth=&amp;atitle=Integrative+annotation+of+21%2C037+human+genes+validated+by+full-length+cDNA+clones." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrative+annotation+of+21%2C037+human+genes+validated+by+full-length+cDNA+clones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrative+annotation+of+21%2C037+human+genes+validated+by+full-length+cDNA+clones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000460-Suzuki1" id="pone.0000460-Suzuki1"></a>Suzuki Y, Yoshitomo-Nakagawa K, Maruyama K, Suyama A, Sugano S (1997) Construction and characterization of a full length-enriched and a 5′-end-enriched cDNA library. Gene  200: 149–56.  <ul class="find" data-citedArticleID="953078" data-doi="10.1016/s0378-1119(97)00411-3"><li><a href="http://dx.doi.org/10.1016/s0378-1119(97)00411-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+and+characterization+of+a+full+length-enriched+and+a+5%26%23x2032%3B-end-enriched+cDNA+library." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+and+characterization+of+a+full+length-enriched+and+a+5%26%23x2032%3B-end-enriched+cDNA+library.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000460-Strausberg1" id="pone.0000460-Strausberg1"></a>Strausberg RL, Feingold EA, Grouse LH, Derge JG, Klausner RD, et al.  (2002) Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences. Proc Natl Acad Sci U S A  99: 16899–16903.  <ul class="find" data-citedArticleID="953076" data-doi="10.1073/pnas.242603899"><li><a href="http://dx.doi.org/10.1073/pnas.242603899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+and+initial+analysis+of+more+than+15%2C000+full-length+human+and+mouse+cDNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+and+initial+analysis+of+more+than+15%2C000+full-length+human+and+mouse+cDNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000460-Modrek1" id="pone.0000460-Modrek1"></a>Modrek B, Lee C (2002) A genomic view of alternative splicing. Nat Genet  30: 13–19.  <ul class="find" data-citedArticleID="953066" data-doi="10.1038/ng0102-13"><li><a href="http://dx.doi.org/10.1038/ng0102-13" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+view+of+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+view+of+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000460-Wheeler1" id="pone.0000460-Wheeler1"></a>Wheeler DL, Church DM, Edgar R, Federhen S, Helmberg W, et al.  (2004) Database resources of the National Center for Biotechnology Information: update. Nucleic Acids Res  32: D35–40.  <ul class="find" data-citedArticleID="953084" data-doi="10.1093/nar/gkh073"><li><a href="http://dx.doi.org/10.1093/nar/gkh073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information%3A+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information%3A+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000460-Hurowitz1" id="pone.0000460-Hurowitz1"></a>Hurowitz EH, Brown PO (2003) Genome-wide analysis of mRNA lengths in <em>Saccharomyces cerevisiae</em>. Genome Biol  5: R2.  <ul class="find" data-citedArticleID="953052" data-doi="10.1186/gb-2003-5-1-r2"><li><a href="http://dx.doi.org/10.1186/gb-2003-5-1-r2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+mRNA+lengths+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+mRNA+lengths+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000460-Karolchik1" id="pone.0000460-Karolchik1"></a>Karolchik D, Baertsch R, Diekhans M, Furey TS, Hinrichs A, et al.  (2003) The UCSC Genome Browser Database. Nucleic Acids Res  31: 51–54.  <ul class="find" data-citedArticleID="953060" data-doi="10.1093/nar/gkg129"><li><a href="http://dx.doi.org/10.1093/nar/gkg129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Genome+Browser+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Genome+Browser+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000460-Johnson1" id="pone.0000460-Johnson1"></a>Johnson JM, Castle J, Garrett-Engele P, Kan Z, Loerch PM, et al.  (2003) Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays. Science  302: 2141–2144.  <ul class="find" data-citedArticleID="953058" data-doi="10.1126/science.1090100"><li><a href="http://dx.doi.org/10.1126/science.1090100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+survey+of+human+alternative+pre-mRNA+splicing+with+exon+junction+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+survey+of+human+alternative+pre-mRNA+splicing+with+exon+junction+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000460-Stamm1" id="pone.0000460-Stamm1"></a>Stamm S, Zhu J, Nakai K, Stoilov P, Stoss O, et al.  (2000) An alternative-exon database and its statistical analysis. DNA Cell Biol  19: 739–56.  <ul class="find" data-citedArticleID="953072" data-doi="10.1089/104454900750058107"><li><a href="http://dx.doi.org/10.1089/104454900750058107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+alternative-exon+database+and+its+statistical+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+alternative-exon+database+and+its+statistical+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000460-Xu1" id="pone.0000460-Xu1"></a>Xu Q, Modrek B, Lee C (2002) Genome-wide detection of tissue-specific alternative splicing in the human transcriptome. Nucleic Acids Res  30: 3754–3766.  <ul class="find" data-citedArticleID="953086" data-doi="10.1093/nar/gkf492"><li><a href="http://dx.doi.org/10.1093/nar/gkf492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+detection+of+tissue-specific+alternative+splicing+in+the+human+transcriptome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+detection+of+tissue-specific+alternative+splicing+in+the+human+transcriptome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000460-Mignone1" id="pone.0000460-Mignone1"></a>Mignone F, Gissi C, Liuni S, Pesole G (2002) Untranslated regions of mRNAs. Genome Biol  3: reviews0004.1–10.  <ul class="find" data-citedArticleID="953064" data-doi="10.1186/gb-2002-3-3-reviews0004"><li><a href="http://dx.doi.org/10.1186/gb-2002-3-3-reviews0004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Untranslated+regions+of+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Untranslated+regions+of+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000460-Stark1" id="pone.0000460-Stark1"></a>Stark A, Brennecke J, Bushati N, Russell RB, Cohen SM (2005) Animal microRNAs confer robustness to gene expression and have a significant impact on 3′UTR evolution. Cell  123: 1133–1146.  <ul class="find" data-citedArticleID="953074" data-doi="10.1016/j.cell.2005.11.023"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.11.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Animal+microRNAs+confer+robustness+to+gene+expression+and+have+a+significant+impact+on+3%26%23x2032%3BUTR+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Animal+microRNAs+confer+robustness+to+gene+expression+and+have+a+significant+impact+on+3%26%23x2032%3BUTR+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000460-Chomczynski1" id="pone.0000460-Chomczynski1"></a>Chomczynski P (1993) A reagent for the single-step simultaneous isolation of RNA, DNA and proteins from cell and tissue samples. Biotechniques  15: 532.  <ul class="find" data-citedArticleID="953044"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+reagent+for+the+single-step+simultaneous+isolation+of+RNA%2C+DNA+and+proteins+from+cell+and+tissue+samples.&amp;auth=&amp;atitle=A+reagent+for+the+single-step+simultaneous+isolation+of+RNA%2C+DNA+and+proteins+from+cell+and+tissue+samples." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+reagent+for+the+single-step+simultaneous+isolation+of+RNA%2C+DNA+and+proteins+from+cell+and+tissue+samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+reagent+for+the+single-step+simultaneous+isolation+of+RNA%2C+DNA+and+proteins+from+cell+and+tissue+samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000460-Perou1" id="pone.0000460-Perou1"></a>Perou CM, Sorlie T, Eisen MB, van de Rijn M, Jeffrey SS, et al.  (2000) Molecular portraits of human breast tumours. Nature  406: 747–752.  <ul class="find" data-citedArticleID="953068" data-doi="10.1038/35021093"><li><a href="http://dx.doi.org/10.1038/35021093" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+portraits+of+human+breast+tumours." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+portraits+of+human+breast+tumours.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000460-Wang1" id="pone.0000460-Wang1"></a>Wang Y, Liu CL, Storey JD, Tibshirani RJ, Herschlag D, et al.  (2002) Precision and functional specificity in mRNA decay. Proc Natl Acad Sci U S A  99: 5860–5865.  <ul class="find" data-citedArticleID="953082" data-doi="10.1073/pnas.092538799"><li><a href="http://dx.doi.org/10.1073/pnas.092538799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Precision+and+functional+specificity+in+mRNA+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Precision+and+functional+specificity+in+mRNA+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000460-Chen1" id="pone.0000460-Chen1"></a>Chen SS, Donoho DL, Saunders MA (1999) Atomic decomposition by basis pursuit. SIAM J Sci Comp  20: 33–61.  <ul class="find" data-citedArticleID="953042" data-doi="10.1137/s1064827596304010"><li><a href="http://dx.doi.org/10.1137/s1064827596304010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atomic+decomposition+by+basis+pursuit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atomic+decomposition+by+basis+pursuit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000460-Saunders1" id="pone.0000460-Saunders1"></a>Saunders MA, Kim B (2005) Pdco: Primal-dual interior method for convex objectives.   Available: <a href="http://www.stanford.edu/group/SOL/software/pdco.html">http://www.stanford.edu/group/SOL/softwa​re/pdco.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0000460-Donoho1" id="pone.0000460-Donoho1"></a>Donoho DL (2004) For most underdetermined systems of linear equations, the minimal <em>l</em><sup>1</sup>-norm near-solution approximates the sparsest near-solution. Comm Pure and Appl Math.   <ul class="find" data-citedArticleID="953046" data-doi="10.1002/cpa.20131"><li><a href="http://dx.doi.org/10.1002/cpa.20131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=For+most+underdetermined+systems+of+linear+equations%2C+the+minimal+l1-norm+near-solution+approximates+the+sparsest+near-solution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22For+most+underdetermined+systems+of+linear+equations%2C+the+minimal+l1-norm+near-solution+approximates+the+sparsest+near-solution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000460-Donoho2" id="pone.0000460-Donoho2"></a>Donoho DL, Tanner J (2005) Sparse nonnegative solutions of underdetermined linear equations by linear programming. Proc Natl Acad Sci U S A  102: 9446–9451.  <ul class="find" data-citedArticleID="953048" data-doi="10.1073/pnas.0502269102"><li><a href="http://dx.doi.org/10.1073/pnas.0502269102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sparse+nonnegative+solutions+of+underdetermined+linear+equations+by+linear+programming." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sparse+nonnegative+solutions+of+underdetermined+linear+equations+by+linear+programming.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000460-Donoho3" id="pone.0000460-Donoho3"></a>Donoho DL, Elad M, Temlyakov VN (2006) Stable recovery of sparse overcomplete representations in the presence of noise. IEEE Trans on Information Theory  52: 6–18.  <ul class="find" data-citedArticleID="953050" data-doi="10.1109/tit.2005.860430"><li><a href="http://dx.doi.org/10.1109/tit.2005.860430" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stable+recovery+of+sparse+overcomplete+representations+in+the+presence+of+noise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stable+recovery+of+sparse+overcomplete+representations+in+the+presence+of+noise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.XML" value="98308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.PDF" value="1438722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g001.PNG_L" value="80146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g001.PNG_M" value="89777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g001.PNG_S" value="3779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g001.TIF" value="436256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g001.PNG_I" value="55182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g002.PNG_L" value="24124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g002.PNG_M" value="37109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g002.PNG_S" value="2172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g002.TIF" value="139720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g002.PNG_I" value="16360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t001.PNG_L" value="56155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t001.PNG_M" value="57319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t001.PNG_S" value="2737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t001.TIF" value="268168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t001.PNG_I" value="27024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t002.PNG_L" value="60678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t002.PNG_M" value="61515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t002.PNG_S" value="2925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t002.TIF" value="277632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t002.PNG_I" value="28587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g003.PNG_L" value="17582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g003.PNG_M" value="19029"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g003.PNG_S" value="2062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g003.TIF" value="42666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g003.PNG_I" value="8431"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t003.PNG_L" value="29939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t003.PNG_M" value="26738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t003.PNG_S" value="2166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t003.TIF" value="275906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t003.PNG_I" value="12422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g004.PNG_L" value="19383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g004.PNG_M" value="14282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g004.PNG_S" value="1821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g004.TIF" value="42594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g004.PNG_I" value="6659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g005.PNG_L" value="22595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g005.PNG_M" value="27574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g005.PNG_S" value="1945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g005.TIF" value="50586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g005.PNG_I" value="11149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t004.PNG_L" value="30235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t004.PNG_M" value="29996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t004.PNG_S" value="2281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t004.TIF" value="186670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t004.PNG_I" value="14358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t005.PNG_L" value="62463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t005.PNG_M" value="64715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t005.PNG_S" value="3773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t005.TIF" value="299360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t005.PNG_I" value="30355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t006.PNG_L" value="63874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t006.PNG_M" value="65628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t006.PNG_S" value="3827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t006.TIF" value="303796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.t006.PNG_I" value="30823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g006.PNG_L" value="11992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g006.PNG_M" value="14205"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g006.PNG_S" value="1596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g006.TIF" value="31836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g006.PNG_I" value="6431"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g007.PNG_L" value="17959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g007.PNG_M" value="20122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g007.PNG_S" value="1960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g007.TIF" value="42240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000460.g007.PNG_I" value="8413"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000460&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000460" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000460&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000460&volume=&issue=&title=Virtual Northern Analysis of the Human Genome&author_name=Evan%20H.%20Hurowitz%2C%20Iddo%20Drori%2C%20Victoria%20C.%20Stodden%2C%20David%20L.%20Donoho%2C%20Patrick%20O.%20Brown&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000460" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000460" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000460" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000460&amp;t=Virtual%20Northern%20Analysis%20of%20the%20Human%20Genome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000460&title=Virtual%20Northern%20Analysis%20of%20the%20Human%20Genome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000460&amp;title=Virtual%20Northern%20Analysis%20of%20the%20Human%20Genome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000460" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000460&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Virtual Northern Analysis of the Human Genome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000460';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Virtual%20Northern%20Analysis%20of%20the%20Human%20Genome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000460" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000460" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complementary+DNA%22" title="Search for articles in the subject area:'Complementary DNA'"><div class="flagText">Complementary DNA</div></a>
              <div data-categoryid="32053" data-articleid="23758"
                   data-categoryname="Complementary DNA"
                   class="flagImage" title="Flag 'Complementary DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="23758"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23758"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="23758"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="23758"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="23758"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23758"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Untranslated+regions%22" title="Search for articles in the subject area:'Untranslated regions'"><div class="flagText">Untranslated regions</div></a>
              <div data-categoryid="40559" data-articleid="23758"
                   data-categoryname="Untranslated regions"
                   class="flagImage" title="Flag 'Untranslated regions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1809'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2131'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4166&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
