

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Ultrastructural Characterization of the Giant Volcano-like Virus Factory of Acanthamoeba polyphaga Mimivirus</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000328"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000328" />

    <meta name="citation_title" content="Ultrastructural Characterization of the Giant Volcano-like Virus Factory of Acanthamoeba polyphaga Mimivirus"/>
    <meta itemprop="name" content="Ultrastructural Characterization of the Giant Volcano-like Virus Factory of Acanthamoeba polyphaga Mimivirus"/>

      <meta name="citation_author" content="Marie Suzan-Monti"/>
            <meta name="citation_author_institution" content="Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Bernard La Scola"/>
            <meta name="citation_author_institution" content="Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Lina Barrassi"/>
            <meta name="citation_author_institution" content="Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Leon Espinosa"/>
            <meta name="citation_author_institution" content="Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Didier Raoult"/>
            <meta name="citation_author_institution" content="Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France"/>

    <meta name="citation_date" content="2007/3/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000328.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e328"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A giant virus in amoebae.; citation_author=B La Scola; citation_author=S Audic; citation_author=C Robert; citation_author=L Jungang; citation_author=X de Lamballerie; citation_journal_title=Science; citation_volume=299; citation_number=1; citation_pages=2033; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomic and evolutionary aspects of Mimivirus.; citation_author=M Suzan-Monti; citation_author=B La Scola; citation_author=D Raoult; citation_journal_title=Virus Res; citation_volume=117; citation_number=2; citation_pages=145-155; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cryo-electron microscopy of the giant Mimivirus.; citation_author=C Xiao; citation_author=PR Chipman; citation_author=AJ Battisti; citation_author=VD Bowman; citation_author=P Renesto; citation_journal_title=J Mol Biol; citation_volume=353; citation_number=3; citation_pages=493-496; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The 1.2-megabase genome sequence of Mimivirus.; citation_author=D Raoult; citation_author=S Audic; citation_author=C Robert; citation_author=C Abergel; citation_author=P Renesto; citation_journal_title=Science; citation_volume=306; citation_number=4; citation_pages=1344-1350; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Common origin of four diverse families of large eukaryotic DNA viruses.; citation_author=LM Iyer; citation_author=L Aravind; citation_author=EV Koonin; citation_journal_title=J Virol; citation_volume=75; citation_number=5; citation_pages=11720-11734; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Virus factories: associations of cell organelles for viral replication and morphogenesis.; citation_author=RR Novoa; citation_author=G Calderita; citation_author=R Arranz; citation_author=J Fontana; citation_author=H Granzow; citation_journal_title=Biol Cell; citation_volume=97; citation_number=6; citation_pages=147-172; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of interaction between DNA and 4′,6-diamidino-2-phenylindole by optical spectroscopy.; citation_author=M Kubista; citation_author=B Akerman; citation_author=B Norden; citation_journal_title=Biochemistry; citation_volume=26; citation_number=7; citation_pages=4545-4553; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=On the extended depth of focus algorithms for bright field microscopy.; citation_author=AG Valdecasas; citation_author=D Marshall; citation_author=JM Becerra; citation_author=JJ Terrero; citation_journal_title=Micron; citation_volume=32; citation_number=8; citation_pages=559-569; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mimivirus in pneumonia patients.; citation_author=B La Scola; citation_author=TJ Marrie; citation_author=JP Auffray; citation_author=D Raoult; citation_journal_title=Emerg Inf Dis; citation_volume=11; citation_number=9; citation_pages=449-452; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ameba-associated microorganisms and diagnosis of nosocomial pneumonia.; citation_author=P Berger; citation_author=L Papazian; citation_author=M Drancourt; citation_author=B La Scola; citation_author=JP Auffray; citation_journal_title=Emerg Inf Dis; citation_volume=12; citation_number=10; citation_pages=248-255; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Laboratory infection of a technician by mimivirus.; citation_author=D Raoult; citation_author=P Renesto; citation_author=P Brouqui; citation_journal_title=Ann Int Med; citation_volume=144; citation_number=11; citation_pages=702-703; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microorganisms resistant to free-living amoebae.; citation_author=G Greub; citation_author=D Raoult; citation_journal_title=Clin Microbiol Rev; citation_volume=17; citation_number=12; citation_pages=413-433; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phycodnaviridae–large DNA algal viruses.; citation_author=JL Van Etten; citation_author=MV Graves; citation_author=DG Muller; citation_author=W Boland; citation_author=N Delaroque; citation_journal_title=Arch Virol; citation_volume=147; citation_number=13; citation_pages=1479-1516; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Unusual life style of giant chlorella viruses.; citation_author=JL Van Etten; citation_journal_title=Ann Rev Genet; citation_volume=37; citation_number=14; citation_pages=153-195; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mimivirus giant particles incorporate a large fraction of anonymous and unique gene products.; citation_author=P Renesto; citation_author=C Abergel; citation_author=P Decloquement; citation_author=D Moinier; citation_author=S Azza; citation_journal_title=J Virol; citation_volume=80; citation_number=15; citation_pages=11678-11685; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Poxviridae: the viruses and their replication.; citation_author=B Moss; citation_number=16; citation_pages=2637-2672; citation_date=2001; citation_publisher=Lippincott Williams and Wilkins; " />
      <meta name="citation_reference" content="citation_title=Assembly of African Swine Fever Virus: quantitative ultrastructural analysis in vitro and in vivo.; citation_author=SM Brookes; citation_author=LK Dixon; citation_author=RME Parkhouse; citation_journal_title=Virology; citation_volume=224; citation_number=17; citation_pages=84-92; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Role of the host cell nucleus in the replication of African swine fever virus DNA.; citation_author=R Garcia-Beato; citation_author=ML Salas; citation_author=E Vinuela; citation_author=J Salas; citation_journal_title=Virology; citation_volume=188; citation_number=18; citation_pages=637-649; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The Iridoviruses.; citation_author=T Williams; citation_journal_title=Adv Virus Res; citation_volume=46; citation_number=19; citation_pages=345-412; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A decade of advances in iridovirus research.; citation_author=T Williams; citation_author=V Barbosa-Solomieu; citation_author=VG Chinchar; citation_journal_title=Adv Virus Res; citation_volume=65; citation_number=20; citation_pages=173-248; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Intracellular virus DNA distribution and the acquisition of the nucleoprotein core during African Swine Fever Virus particle assembly: ultrastructural in situ hybridisation and DNase-gold labelling.; citation_author=SM Brookes; citation_author=AD Hyatt; citation_author=T Wise; citation_author=RME Parkhouse; citation_journal_title=Virology; citation_volume=249; citation_number=21; citation_pages=175-188; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Assembly of African Swine Fever Virus: role of polyprotein pp220.; citation_author=G Andrés; citation_author=C Simón-Mateo; citation_author=E Vińuela; citation_journal_title=J Virol; citation_volume=71; citation_number=22; citation_pages=2331-2341; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Aggresomes resemble sites specialized for virus assembly.; citation_author=CM Heath; citation_author=M Windsor; citation_author=T Wileman; citation_journal_title=J Cell Biol; citation_volume=153; citation_number=23; citation_pages=449-455; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Preliminary report on the pathogenicity of Legionella pneumophila for freshwater and soil amoebae.; citation_author=TJ Rowbotham; citation_journal_title=J Clin Pathol; citation_volume=33; citation_number=24; citation_pages=1179-1183; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=OsiriX: an open-source software for navigating in multidimensional DICOM images.; citation_author=A Rosset; citation_author=L Spadola; citation_author=O Ratib; citation_journal_title=J Digit Imaging; citation_volume=17; citation_number=25; citation_pages=205-216; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Giant viruses infecting algae.; citation_author=JL Van Etten; citation_author=RH Meints; citation_journal_title=Ann Rev Microbiol; citation_volume=53; citation_number=26; citation_pages=447-494; citation_date=1999; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000328" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Ultrastructural Characterization of the Giant Volcano-like Virus Factory of Acanthamoeba polyphaga Mimivirus"/>
    <meta name="twitter:description" content="Acanthamoeba polyphaga Mimivirus is a giant double-stranded DNA virus defining a new genus, the Mimiviridae, among the Nucleo-Cytoplasmic Large DNA Viruses (NCLDV). We used utrastructural studies to shed light on the different steps of the Mimivirus replication cycle: entry via phagocytosis, release of viral DNA into the cell cytoplasm through fusion of viral and vacuolar membranes, and finally viral morphogenesis in an extraordinary giant cytoplasmic virus factory (VF). Fluorescent staining of the AT-rich Mimivirus DNA showed that it enters the host nucleus prior to the generation of a cytoplasmic independent replication centre that forms the core of the VF. Assembly and filling of viral capsids were observed within the replication centre, before release into the cell cytoplasm where progeny virions accumulated. 3D reconstruction from fluorescent and differential contrast interference images revealed the VF emerging from the cell surface as a volcano-like structure. Its size dramatically grew during the 24 h infectious lytic cycle. Our results showed that Mimivirus replication is an extremely efficient process that results from a rapid takeover of cellular machinery, and takes place in a unique and autonomous giant assembly centre, leading to the release of a large number of complex virions through amoebal lysis."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000328.g008"/>

  <meta property="og:title" content="Ultrastructural Characterization of the Giant Volcano-like Virus Factory of Acanthamoeba polyphaga Mimivirus" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000328" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6683'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1055'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8030&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000328">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000328" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000328&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Ultrastructural Characterization of the Giant Volcano-like Virus Factory of <i>Acanthamoeba polyphaga Mimivirus</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marie Suzan-Monti
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:marie.suzan@medecine.univ-mrs.fr">marie.suzan@medecine.univ-mrs.fr</a> (MS-M); <a href="mailto:Didier.Raoult@medecine.univ-mrs.fr">Didier.Raoult@medecine.univ-mrs.fr</a> (DR)</p>

                <p>Affiliation:
                  Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernard La Scola, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lina Barrassi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leon Espinosa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Didier Raoult
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:marie.suzan@medecine.univ-mrs.fr">marie.suzan@medecine.univ-mrs.fr</a> (MS-M); <a href="mailto:Didier.Raoult@medecine.univ-mrs.fr">Didier.Raoult@medecine.univ-mrs.fr</a> (DR)</p>

                <p>Affiliation:
                  Unité des Rickettsies, Centre National de la Recherche Scientifique (CNRS) UMR 6020, IFR 48, Faculté de Médecine, Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000328</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000328" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000328" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000328" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000328" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000328-g001" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000328-g002" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000328-g003" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000328-g004" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000328-g005" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000328-g006" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000328-g007" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000328-g008" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000328">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000328" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Acanthamoeba polyphaga Mimivirus</em> is a giant double-stranded DNA virus defining a new genus, the <em>Mimiviridae</em>, among the Nucleo-Cytoplasmic Large DNA Viruses (NCLDV). We used utrastructural studies to shed light on the different steps of the <em>Mimivirus</em> replication cycle: entry via phagocytosis, release of viral DNA into the cell cytoplasm through fusion of viral and vacuolar membranes, and finally viral morphogenesis in an extraordinary giant cytoplasmic virus factory (VF). Fluorescent staining of the AT-rich <em>Mimivirus</em> DNA showed that it enters the host nucleus prior to the generation of a cytoplasmic independent replication centre that forms the core of the VF. Assembly and filling of viral capsids were observed within the replication centre, before release into the cell cytoplasm where progeny virions accumulated. 3D reconstruction from fluorescent and differential contrast interference images revealed the VF emerging from the cell surface as a volcano-like structure. Its size dramatically grew during the 24 h infectious lytic cycle. Our results showed that <em>Mimivirus</em> replication is an extremely efficient process that results from a rapid takeover of cellular machinery, and takes place in a unique and autonomous giant assembly centre, leading to the release of a large number of complex virions through amoebal lysis.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Suzan-Monti M, Scola BL, Barrassi L, Espinosa L, Raoult D (2007) Ultrastructural Characterization of the Giant Volcano-like Virus Factory of <em>Acanthamoeba polyphaga Mimivirus</em>. PLoS ONE 2(3):
          e328.
            doi:10.1371/journal.pone.0000328</p><p><strong>Academic Editor: </strong>Paul Digard, University of Cambridge, United Kingdom</p><p><strong>Received:</strong> November 23, 2006; <strong>Accepted:</strong> March 6, 2007; <strong>Published:</strong> March 28, 2007</p><p><strong>Copyright:</strong> © 2007 Suzan-Monti et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Centre National de la Recherche Scientifique (CNRS)</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>During the environmental study of an outbreak of pneumonia, a giant icosahedral DNA virus was discovered growing in amoebae. This virus was named <em>Mimivirus</em> (for <em>mi</em>micking <em>mi</em>crobe <em>virus</em>) <a href="#pone.0000328-LaScola1">[1]</a>. With a diameter of about 650 nm, <em>Mimivirus</em> is the largest virus known to date. Morphologically, <em>Mimivirus</em> resembles Nucleo-Cytoplasmic Large DNA Viruses (NCLDV), such as the <em>Iridoviruses, Asfarviruses</em> and <em>Phycodnaviruses</em> <a href="#pone.0000328-SuzanMonti1">[2]</a>. <em>Mimivirus</em> comprises a central dense core that is surrounded by two lipid membrane layers inside a capsid protein shell covered by fibrils <a href="#pone.0000328-Xiao1">[3]</a>. The sequence of its 1.2 Mb genome revealed 1262 putative open reading frames (Genbank accession number NC_006450; <a href="#pone.0000328-Raoult1">[4]</a>). A phylogenetic study based on concatenated sequences of the eight class I genes <a href="#pone.0000328-Iyer1">[5]</a> common to <em>Mimivirus</em> and to all NCLDVs revealed that <em>Mimivirus</em> belonged to this lineage, but stood apart from <em>Phycodnavirida</em>e, <em>Iridoviridae, Asfarviridae</em> and <em>Poxviridae</em> on the phylogenetic tree <a href="#pone.0000328-Raoult1">[4]</a>. In <em>A. polyphaga, Mimivirus</em> replicative cycle was described as starting with a 4 h eclipse phase, followed by cytoplasmic accumulation of newly synthesized viruses, and ending with cell lysis and virus release 24 h post-infection (p.i.) <a href="#pone.0000328-LaScola1">[1]</a>, <a href="#pone.0000328-SuzanMonti1">[2]</a>. Transmission electron microscopy (TEM) analysis of infected <em>A. polyphaga</em> suggested that viral replication, including DNA synthesis and particle assembly, might occur in and near the cell nucleus <a href="#pone.0000328-LaScola1">[1]</a> and the existence of a virus factory was proposed <a href="#pone.0000328-SuzanMonti1">[2]</a>. As already described for a large variety of unrelated viruses, virus factories are perinuclear or cytoplasmic structures where virus replication and assembly take place. Their formation is the result of complex interactions between viral and cellular components and they induce profound alteration of the infected cell structure like recruitment of organelles and organisation of cellular compartments <a href="#pone.0000328-Novoa1">[6]</a>. This paper describes for the first time the morphological characteristics of <em>Mimivirus</em> volcano-like giant virus factory, as determined by an extensive ultrastructural study and by tracking fluorescently-labelled viral DNA and viral proteins during the 24 h time course of infection. Our results reinforce the emerging picture of <em>Mimivirus</em> as a very complex and unique amoebal pathogen.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Ultrastructural aspects of the <em>Mimivirus</em> replication cycle</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><em>A. polyphaga</em> were infected with a cell-free <em>Mimivirus</em> supernatant at a multiplicity of infection of 10, and processed for TEM at different times p.i.. At 30 min after infection, defined as the 0 h p.i. time point, <em>Mimivirus</em> appears to enter the amoebae by phagocytosis (<a href="#pone-0000328-g001">Figure 1A</a>) and was next observed within the phagocytic vacuoles of the amoebae (<a href="#pone-0000328-g001">Figure 1B</a>). Empty particles could be seen with an open vertex (<a href="#pone-0000328-g001">Figure 1C</a>). At 4 h p.i., several viruses could be found within the same vacuole either as fully closed or as empty open particles (<a href="#pone-0000328-g001">Figure 1D and G</a>). The most interesting phenomenon observed was the internal <em>Mimivirus</em> membrane extruding from the particle to fuse with the vacuole membrane (<a href="#pone-0000328-g001">Figure 1E and F</a>), and the apparent pouring out of electron dense material, most likely the viral DNA, into the cell cytoplasm (<a href="#pone-0000328-g001">Figure 1G and H</a>). Moreover, these events might occur through the vertex described by Xiao <em>et al.</em> <a href="#pone.0000328-Xiao1">[3]</a>, since this structure appeared to be open on empty particles (<a href="#pone-0000328-g001">Figure 1C and G</a>). It should be noted that the external structures, the outer layers and fibrils, remained intact on the empty particles at this stage. Next, condensed genetic <em>Mimivirus</em> material appears to enter the cell nucleus (<a href="#pone-0000328-g001">Figure 1I and J</a>). These structures were never seen in uninfected amoebae (data not shown). At later p.i. times, we previously described viral particles at the periphery of what we originally thought to be the cell nucleus <a href="#pone.0000328-LaScola1">[1]</a>, <a href="#pone.0000328-SuzanMonti1">[2]</a>. Further detailed examination of other series of ultrathin sections of <em>A. polyphaga</em> at 4 h p.i. revealed the appearance of an electron-dense structure, clearly distinct from the nucleus that might represent a cytoplasmic viral replication centre surrounded by mitochondria (<a href="#pone-0000328-g001">Figure 1K and L</a>). Contrary to the cell nucleus, this structure did not appear to be surrounded by a membrane. The size of this structure increased rapidly between 5 h and 8 h p.i. At 8 h p.i., newly synthesized viral particles were observed at the periphery of the putative replication centre (<a href="#pone-0000328-g002">Figure 2A</a>), surrounded by an electron-lucent zone, forming a virus factory (VF). At 12 h p.i., almost all the cytoplasmic space was occupied by the VF, and the cell nucleus could still be observed at the periphery (<a href="#pone-0000328-g002">Figure 2B</a>). These observations indicated that <em>Mimivirus</em> replication and assembly took place in a very specific cytoplasmic structure composed of a dense central core from which newly formed particles appeared.</p>
<div class="figure" id="pone-0000328-g001"><div class="img"><a name="pone-0000328-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Ultrastructural aspects of the early steps of <em>Mimivirus</em> replication cycle.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Transmission electron microscopy pictures were taken at 0 h p.i. (A–C) or at 4 h p.i. (D–L). (A) <em>Mimivirus</em> particle being phagocytosed by an amoeba; bar = 2 µm. (B) Several single viral particles within intra-cytoplasmic vacuoles; bar = 2 µm. (C) Higher magnification of the boxed area in B showed the open vertex of an empty particle (arrow); bar = 1 µm. (D) Close contact of the membranes of two vacuoles (arrow), one with several <em>Mimivirus</em> particles and the other with a single viral particle; bars = 1 µm. (E) Extrusion of the internal <em>Mimivirus</em> membrane toward the vacuole membrane; bar = 200 nm. (F) Higher magnification of the contact zone between viral and vacuole membranes (arrow); bar = 100 nm. (G) Full closed, empty with open vertex (arrow) and opening <em>Mimivirus</em> particles; bar = 500 nm. (H) Higher magnification of the opening <em>Mimivirus</em> particle in G. The fused viral and vacuole membranes were clearly visible (arrow); bar = 100 nm. (I) Condensed electron dense material inside the cell nucleus (arrow); bar = 2 µm. (J) Higher magnification of the condensed electron dense material between the nuclear membrane and the nucleolus (arrow); bar = 500 nm. (K) An electron dense structure (arrow), distinct from the cell nucleus was observed; bar = 5 µm. (L) Higher magnification of this heterogeneous structure, surrounded with mitochondria; bar = 200 nm.</p>
<span>doi:10.1371/journal.pone.0000328.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The <em>Mimivirus</em> factory could be divided into three zones: the inner replication centre, the intermediate assembly zone and the peripheral zone where the newly formed particles acquired their fibrils. This later zone appeared electron-lucent, probably due to exclusion of cellular material and organelles by the expanding VF (<a href="#pone-0000328-g002">Figure 2C</a>, <a href="#pone-0000328-g003">3A</a>). Closer examination of the VF replication centre suggested a possible sequence of events from assembly of the capsid shell, to the release of complete viral particles with a condensed core surrounded with fibrils. The replication centre of the VF showed a heterogeneous structure with dense inclusions (<a href="#pone-0000328-g002">Figure 2C</a>, <a href="#pone-0000328-g003">3A</a>). The hexagonal shape of the capsid appeared as assembling progressed (<a href="#pone-0000328-g002">Figure 2C–G</a>), and empty capsids were then filled with electron-dense material before being released (<a href="#pone-0000328-g002">Figure 2C</a>). The vertex was clearly visible on the virus particle, opposite to the side linked to the replication centre (<a href="#pone-0000328-g002">Figure 2C–G</a>). Membranes underlining the capsid layer were observed in growing (<a href="#pone-0000328-g002">Figure 2D–G</a>) and released (<a href="#pone-0000328-g002">Figure 2G</a>) viral particles. These membranes did not encircle the replication centre and were always observed at its periphery. Their origin is still unknown.</p>
<div class="figure" id="pone-0000328-g002"><div class="img"><a name="pone-0000328-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Ultrastructural aspects of the late steps of <em>Mimivirus</em> replication cycle - capsid assembly.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>(A) At 8 h p.i. the virus factory (VF) appeared composed of a dense replication centre surrounded by new viral particles. Nu : cell nucleus; bar = 5 µm. (B) At 12 h p.i. the cell cytoplasm was filled with newly synthesised viruses. The cell nucleus (Nu) was expelled to the periphery; bar = 3 µm. (C–G) Pictures were taken at 8, 12 or 16 h p.i. (C) Different stages of viral particles morphogenesis from the replication centre : beginning of hexagonal capsid assembly (white and black arrowheads); complete empty capsid (thick closed black arrow); filling of empty capsids with condensed electron dense material (thick open black and white arrows); release of full closed viral particles surrounded by fibrils at the periphery of the virus factory; bar = 500 nm. (D–G) Different aspects of viral capsid assembly : beginning of capsid assembly (D bar = 500 nm; G bar = 100 nm); almost complete capsids detaching from the replication center (E bar = 100 nm; F bar = 200 nm); complete capsid being filled with electron dense material (F) or complete viral particle without fibril (G). Membranes were observed beneath the capsid layer (F, G long black arrow). The vertex (small black arrows) was on the external side, opposite to the attachment and filling side (C–G).</p>
<span>doi:10.1371/journal.pone.0000328.g002</span></div><div class="figure" id="pone-0000328-g003"><div class="img"><a name="pone-0000328-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Ultrastructural aspects of the late steps of <em>Mimivirus</em> replication cycle – encapsidation of viral DNA.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Pictures were taken at 8, 12 or 16 h p.i. (A) Different stages of capsid assembly and DNA encapsidation : complete empty capsid (arrowhead); progressive stages of viral DNA insertion (black arrows) through a portal opposite to the vertex (white arrows); bar = 500 nm. (B) Viral DNA insertion into a capsid (arrowhead) and two different detaching steps of full complete viruses from the replication centre (arrows); bar = 500 nm. (C) Higher magnification of the complete viruses seen in B; bar = 200 nm; (D) Insertion of condensed viral DNA into a viral capsid; bar = 200 nm.</p>
<span>doi:10.1371/journal.pone.0000328.g003</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p><a href="#pone-0000328-g003">Figure 3</a> illustrated how viral DNA might be encapsidated into nascent <em>Mimivirus</em> particles. The encapsidation process occurred at the replication centre periphery once capsid assembly is almost complete (<a href="#pone-0000328-g003">Figure 3A</a>). Viral DNA condensation seemed to begin within the replication centre (<a href="#pone-0000328-g003">Figure 3D</a>) before being inserted into viral capsids through an open portal located on the opposite side to the vertex (<a href="#pone-0000328-g003">Figure 3B–D</a>). <a href="#pone-0000328-g003">Figure 3</a> C and D showed how viral DNA condensation progressed within the viral capsid to form the core centre of the viral particle.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Altogether these observations compelled us to modify our original interpretation and to further characterize the different stages of the <em>Mimivirus</em> assembly pathway.</p>


<h4>Morphological description of the <em>Mimivirus</em> factory</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The formation kinetics of the VF, and its viral DNA content in particular, were studied by direct fluorescent staining with the blue fluorescent stain DAPI. The choice of this molecule was based on the fact that DAPI preferentially stains dsDNA by association with AT clusters in the minor groove <a href="#pone.0000328-Kubista1">[7]</a>. The <em>Mimivirus</em> genome has a high AT proportion (72%, <a href="#pone.0000328-Raoult1">[4]</a>) compared to <em>A. polyphaga</em> (genomic AT content estimated to be 49%, determined using 96 shotgun sequences of amoebae genomic DNA; data not shown). <em>Mimivirus</em>-infected <em>A. polyphaga</em> were consequently stained at different time points p.i. Fluorescence and differential interference contrast (DIC) images of the same field are presented in <a href="#pone-0000328-g004">Figure 4</a>. Representative images are also shown in the Supporting Information (<a href="#pone.0000328.s001">Text S1</a>, <a href="#pone.0000328.s003">Figure S1</a>). In uninfected amoebae, cell nuclei showed a characteristic ring-like staining pattern with unlabeled nucleoli surrounded with labelled chromatin, similar to the nucleus morphology observed with TEM (<a href="#pone-0000328-g004">Figure 4A and B</a>). At 0 h p.i. <em>Mimivirus</em> nucleic acid staining appeared as bright single or clustered dots within the cell cytoplasm, contrasting with the cytoplasmic background DAPI labelling (<a href="#pone-0000328-g004">Figure 4C and D</a>). At 1 h p.i., the <em>Mimivirus</em> DAPI-stained DNA dots reached the cell nuclei, where size and staining intensity increased until 3 h p.i. At 4 h p.i. strongly stained clusters showing a heterogeneous structure appeared outside of the cell nuclei (<a href="#pone-0000328-g004">Figure 4E and F</a>). In most of the cells showing these structures, only one cluster could be seen per infected amoeba. Similar observations were made using standard DNA-staining histological dyes such as carbolic toluidine blue (data not shown). Such structures were not observed in uninfected amoebae. The size of these clusters peaked between 8–12 h p.i. and sustained their maximal size and staining intensity until the end of the replication cycle. However at 8 h p.i. the clusters exhibited a homogeneous structure (<a href="#pone-0000328-g004">Figure 4G and H</a>), whereas at 18 h p.i., they showed a heterogeneous less organised morphology. The time course of the development of these structures and their morphology clearly showed that they corresponded to the replication centre of the VFs observed by electron microscopy.</p>
<div class="figure" id="pone-0000328-g004"><div class="img"><a name="pone-0000328-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Mimivirus infectious cycle.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p><em>Mimivirus</em> infected <em>A. polyphaga</em> were stained with DAPI at different times points p.i. and representative pictures are shown. A, B : non infected amoebae; C, D: 0 h p.i. <em>Mimivirus</em> particles inside the cytoplasm and near the cell nucleus could be seen; E, F: 4 h p.i. The heterogeneous structure of the VF appeared near the cell nucleus. No viral particles were detectable in the cytoplasm; G, H : 8 h p.i. The intensively stained VF appeared as an homogeneous structure and neosynthesized viral particles accumulated around the VF; I, J : 18 h p.i. The VF was still intensely stained with quite a different structure, whereas the cell cytoplasm was completely filled with new viral particles. Fluorescence (left column) and DIC (right column) images of the same slide field were taken with a 63×/1.4 oil lens. Fluorescence pictures were taken with an exposure time of 1 sec (A) and 64 msec with gain 2 (C, E, G, I). Inset pictures corresponded to the same as E, G and I taken at a different exposure time 64 msec (E, G) and 16 msec respectively (I).</p>
<span>doi:10.1371/journal.pone.0000328.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Because our experimental conditions used non-synchronized infected cells, we quantified the proportion of each of these morphologically distinct types of replication centre at different time points p.i. in order to determine whether there might be a progression from one type to the other across the time course of infection. Results are shown in <a href="#pone-0000328-g005">Figure 5</a>. Four different core centre morphologies were characterised and quantified (<a href="#pone-0000328-g005">Figure 5A</a>) : i) type I with a clustered morphology (see also <a href="#pone-0000328-g004">Figure 4E</a>), predominant in the first 4 hours of infection; ii) type II in which the core centre appeared as a completely homogeneous structure with a blurry aspect in microscopy images (see <a href="#pone-0000328-g004">Figure 4G</a>), most likely resulting from the fusion of the clusters seen earlier. This form was predominant from 6 to 12 h p.i.; iii) type III in which the whole <em>Mimivirus</em> DNA cluster was surrounded by more and more small bright dots, similar to virus particles, quickly filling the cytoplasmic volume. This form was detected from 7 h p.i. to the end of the infection; and iv) type IV in which the clusters had a heterogeneous morphology with a disorganized appearance with holes and fiber-like patterns (see also <a href="#pone-0000328-g004">Figure 4I</a>). This form was detected in the latest times of infection. These results allowed us to propose a progression of the different characteristic <em>Mimivirus</em> production stages from an early heterogeneous stage (I) corresponding to the appearance and formation of the VF core centre, followed by a homogeneous “mature” stage (II) corresponding to the growing core centre, then by a “productive” stage of the VF (III), and finally by a heterogeneous “degenerative” stage (IV) which most probably signed the exhaustion of the <em>Mimivirus</em> factory. Confocal data were used to build 3D volumic reconstruction of the three main types of <em>Mimivirus</em> factory replication centre during the time course of infection (see also Supporting Information <a href="#pone.0000328.s001">Text S1</a>, <a href="#pone.0000328.s004">Figure S2A</a>). Variation of the DAPI intensity staining is indicative of the variation of the DNA content in the core centre. A quantitative analysis is presented below.</p>
<div class="figure" id="pone-0000328-g005"><div class="img"><a name="pone-0000328-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Evolution and 3D reconstruction of <em>Mimivirus</em> factory.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A: Characterisation and quantification of the different types of replication centre. Fluorescence pictures were taken at 4 h (I, exposure time 32 msec), 8 h (II and III, exposure time 64 and 128 msec respectively) and 16 h (IV, exposure time 32 msec) p.i. Bar = 10 µm. Histogram : a total of 717 DAPI-stained cells were analysed to quantify the proportion of each replication centre time at the indicated time points. B: DIC pictures of an <em>APM</em> infected <em>A. polyphaga</em> at 8 h p.i. taken with a 63×/1.4 oil lens. Different sections according to the depth of focus are shown downward (upper part, from a to d). Bar = 10 µm. The lower part represented a 3D reconstruction combining DIC and DAPI staining of the VF present in different infected cells of a microscope field. The dotted line box framed the cell analysed in the upper part. Blue : DAPI-staining.</p>
<span>doi:10.1371/journal.pone.0000328.g005</span></div>

<h4>Specific aspects of the volcano-like giant <em>Mimivirus</em> factory</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The height of the cells during the late phase of infection, and the limited depth of focus of the 63× objective, allowed us to explore their 3D organisation based on DIC images. Details from one cell are shown in the upper part of <a href="#pone-0000328-g005">Figure 5B</a>. The maximum height of the cell was estimated to be about 10 µm using the difference between the uppermost and the lowest focused images that could be obtained (<a href="#pone-0000328-g005">Figure 5B</a>, sections a and d, respectively); this measurement was confirmed using confocal optical slides. The uppermost focused image always corresponded to the top of the VF as identified by DAPI staining (section a). Intermediate images (sections b and c) showed the newly synthesized virus particles spreading all around the VF and finally organizing into a single-layer crystal-like structure at the bottom of the cell (section d, see also <a href="#pone-0000328-g004">Figure 4J</a>). The Extended Depth of Focus (EDF) technique <a href="#pone.0000328-Valdecasas1">[8]</a> allowed us to generate a topological view of all regions on the same focal plane. EDF allowed us to build a 3D image of the volcano-like structure found in <em>Mimivirus</em>-infected cells as shown in <a href="#pone-0000328-g005">Figure 5B</a>, lower part. The position of the VF replication centre in the EDF image was obtained by the overlay of the DAPI fluorescence image. The area (µm<sup>2</sup>) of the VF increased from 110 µm<sup>2</sup> at 4 h p.i. to 250 µm<sup>2</sup> at 12 h p.i. which represented about 42% of the cell surface (data not shown). These characteristics classified the <em>Mimivirus</em> factory among the largest described until now.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>All the results obtained by transmission electron microscopy, widefield fluorescence, confocal and volume reconstruction analyses allowed us to propose a 3D model&nbsp;of the morphology of <em>Mimivirus</em> factory (Supporting Information, <a href="#pone.0000328.s004">Figure S2 B</a>).</p>


<h4>Fluorescence intensity quantification</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>During the infection cycle, nuclear DAPI staining peaked from 0 to 3 h p.i. and then decreased. After 8 h p.i., nuclei showed weaker labelling and a modified appearance: newly synthesized viral particles could be observed as single dots in the cytoplasm. The number of dots increased dramatically by the end of the replication cycle, at which time they completely filled the intracellular space. One interpretation of these results is that during the 0–3 h p.i. period, the increased nuclear fluorescence intensity was the consequence of the transient nuclear localization of <em>Mimivirus</em> DNA, which then moved into the cytoplasm to form the highly fluorescent VF. The brightness of the VF fluorescence indicated the accumulation of AT-rich DNA (<a href="#pone-0000328-g004">Figures 4</a> and <a href="#pone.0000328.s003">S1</a>). Fluorescence intensity was analyzed and quantified as described in the <a href="#s4">Materials and Methods</a> section, and results are shown in <a href="#pone-0000328-g006">Figure 6</a>. The fluorescence attributes (mean intensity and area) of nuclei and <em>Mimivirus</em> factory showed a concomitant and inverse evolution compared to each other, with the most remarkable point around 5 h p.i.: at this time, the cell area occupied by nuclei showed a 50% drop, whereas the cell area occupied by the <em>Mimivirus</em> factory showed a 50% increase (<a href="#pone-0000328-g006">Figure 6 A and B</a>). Statistical analysis revealed a significant increase of the mean nuclear fluorescence intensity between 0 h and 3 h p.i. (p&lt;0.01) and a significant decrease between 0 h and 8 h p.i. (p&lt;0.01; <a href="#pone-0000328-g006">Figure 6 C</a>). Conversely, there was a significant increase in the mean VF fluorescence intensity between 4 h and 8 h p.i. (p&lt;0.01; <a href="#pone-0000328-g006">Figure 6 D</a>). Taken together, these observations favour a model in which the major site of <em>Mimivirus</em> DNA replication is the cytoplasmic VF, and further suggest that there is a relationship between the two different structures during the replication cycle. Quantification of total fluorescence intensity at different time points p.i. showed an 7-fold increase in total DNA in the cell between 0 h and 8 h p.i., which is exponential growth (e<sup>0.2568x</sup>, <a href="#pone-0000328-g007">Figure 7</a>), equivalent to a doubling time of 2.7 h. In comparison, the total fluorescence intensity in uninfected amoebae varied from 1 at 0 h to 1.2 at 8 h (<a href="#pone-0000328-g007">Figure 7</a>, hatched bars). These results complemented the microscopy results, and allowed the first insights into <em>Mimivirus</em> replication cycle.</p>
<div class="figure" id="pone-0000328-g006"><div class="img"><a name="pone-0000328-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Quantification of the kinetics of <em>Mimivirus</em> factory formation.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Fluorescence picture of a representative field from <em>A. polyphaga</em> at 5 h p.i., stained with DAPI (40× magnification/0.7 lens). The nuclei are marked around their edge with a blue line to allow quantification of their areas compared to those of the <em>Mimivirus</em> factories. (B) Quantification curve showing the normalized parameter (y-axis) as a function of time p.i. (x-axis). Red line: nuclei intensity; green line: nuclei surface fraction; blue line: <em>Mimivirus</em> factory surface fraction. Evolution of the mean fluorescence intensity over time p.i. in cell nuclei (C) or in virus factories (D), at 100 and 4 msec exposure time respectively, using R software; *** = p&lt;0.01 (Wilcoxon test).</p>
<span>doi:10.1371/journal.pone.0000328.g006</span></div><div class="figure" id="pone-0000328-g007"><div class="img"><a name="pone-0000328-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Increase in total cellular DNA content in <em>A. polyphaga</em> during the <em>Mimivirus</em> infection cycle.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>A. Estimation of the total amount of cellular DNA by fluorescence intensity quantification of DAPI staining in <em>Mimivirus</em> infected (bars) or uninfected amoebae (hatched bars). Total intensity = staining area (pixels) × mean intensity (intensity/pixel). The total intensity at different time points was divided by the intensity at time = 0 h p.i. for normalization. The bar height represents the variation of DNA content compared to the t0 timepoint for different cellular compartments. Red bar: nucleus; green bar: cytoplasm; blue bar: virus factory. B. The increase of total cellular DNA was extrapolated from 0 to 8 h p.i. and fitted with the exponential equation: y = A0.e<sup>(Rx)</sup>. Data showed a high&nbsp;predictability with the exponential growth model (R2 = 0.94).</p>
<span>doi:10.1371/journal.pone.0000328.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p><em>Mimivirus</em> is a pathogen resistant to phagocytic destruction in amoebae, and as such should be considered as a possible new causative agent of human pneumonia. Indeed, links with human pneumonia were recently reported <a href="#pone.0000328-LaScola2">[9]</a>–<a href="#pone.0000328-Raoult2">[11]</a>. Among the different amoebae-resistant pathogens studied to date, <em>Mimivirus</em> appears to be the only one with such a rapid lytic effect on amoebae <a href="#pone.0000328-Greub1">[12]</a>. Until now, little was known about the different steps of the <em>Mimivirus</em> replication cycle. Our initial electron microscopy observations of <em>in vitro</em> <em>Mimivirus</em>-infected <em>A. polyphaga</em> showed the intra-cytoplasmic production and accumulation of newly synthesised viruses within a 24 h lytic cycle <a href="#pone.0000328-LaScola1">[1]</a>, <a href="#pone.0000328-SuzanMonti1">[2]</a>. In these papers, we initially speculated that <em>Mimivirus</em> multiplied in the nuclei of infected cells. Indeed, we mistakenly identified the host nucleus as the VF because of its size and aspect. In the present study, in addition to ultrastructural characterization, the unusual size of <em>Mimivirus</em> allowed us to follow the different stages of its replication cycle using fluorescence and DIC microscopy. This enabled us to characterize the formation and growth of the giant <em>Mimivirus</em> VF, and to describe how progeny virions are synthesised, assembled and released from the replication centre to invade the cytoplasmic space. We propose the following replication cycle (<a href="#pone-0000328-g008">Figure 8</a>), composed of an early phase between 0–3 h p.i. (steps 1–4) and of a late phase thereafter (steps 5–8).</p>
<div class="figure" id="pone-0000328-g008"><div class="img"><a name="pone-0000328-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000328" data-uri="info:doi/10.1371/journal.pone.0000328.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000328.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000328.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000328.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000328.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Schematic representation of <em>APM</em> replication cycle.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p><em>Mimivirus</em> entry through a phagocytic vacuole (1). Fusion of phagocytic vacuoles (2) and delivery of <em>Mimivirus</em> genetic material into the cell cytoplasm (3). <em>Mimivirus</em> DNA entry into the host nucleus (3), where the first round of DNA replication might begin (4). At 3 h p.i. <em>Mimivirus</em> DNA came out the host nucleus to form the VF replication centre (5). At 5 h p.i. the VF size showed a 50% increase and viral proteins began to be detected. Proviral capsid assembly and viral capsids budding from the VF central core could be observed (6). Empty or DNA filled capsids accumulated nearby the central core, resulting in a growing VF with viral particles free in the cytoplasm (7). Complete viral capsids surrounded by fibrils might be released through cell lysis (8).</p>
<span>doi:10.1371/journal.pone.0000328.g008</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Electron microscopy images indicated that <em>Mimivirus</em> entry into the amoebae was most likely due to a phagocytic process, followed by fusion of phagocytic vacuoles. <em>Mimivirus</em> genetic material was delivered into the cytoplasm at this stage after fusion of the viral and vacuole membranes, most probably through the virus vertex. Whether the <em>Mimivirus</em> vertex plays also a role in the attachment to the cell surface, as described for Phycodnaviruses <a href="#pone.0000328-VanEtten1">[13]</a>, <a href="#pone.0000328-VanEtten2">[14]</a>, remains to be established. Quantification from fluorescent nucleic acid labelling studies between 0 h and 3 h p.i. showed an increase in intensity in the amoebae nucleus, reflecting an increase in AT-content that might be the result of <em>Mimivirus</em> DNA acquisition. We hypothesise that <em>Mimivirus</em> DNA first enters the amoebae nucleus, shortly after infection, probably for a first round of replication. At 3 h p.i., AT-rich <em>Mimivirus</em> DNA becomes localised to the cell cytoplasm in a structure distinct from the cell nucleus. This might be interpreted as the exit of the <em>Mimivirus</em> genetic material from the nucleus to form the replication centre of the VF in the cell cytoplasm. This structure has also been observed in ultrastructural studies, and it may be the major site of <em>Mimivirus</em> DNA production, independent of the cell nucleus machinery. At 8 h p.i., transmission electron microscopy, direct and indirect fluorescence labelling and quantification of <em>Mimivirus</em> DNA or protein allowed us to clearly distinguish the <em>Mimivirus</em> factory from the cell nucleus. This stage was characterized by an increase in viral DNA production within the cytoplasmic VF, while DNA staining in the host nucleus decreased. The <em>Mimivirus</em> factory size increased with time and with the production of progeny virions. This datasuit may be illustrated by a 3D model of the <em>Mimivirus</em> factory, composed of a replication centre made of unpackaged DNA, all around which viruses are formed in an assembly zone before being released in the cell cytoplasm (<a href="#pone.0000328.s004">Figure S2B</a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The present results indicate that pre-formed capsids are filled with viral DNA, since all of the successive steps of capsid formation could be observed. As viral capsids were shown to assemble at the periphery of the replication centre, it might be envisaged that viral proteins are partially or fully concentrated or synthesised in the replications centre; this idea is supported by results obtained with a <em>Mimivirus</em>-specific mAb recognizing the late virion-associated R710 protein (Supporting Information <a href="#pone.0000328.s001">Text S1</a>, <a href="#pone.0000328.s005">Figure S3</a>). Furthermore, proteomic data analysis showed that no cellular host proteins seem to be incorporated within the virus particles <a href="#pone.0000328-Renesto1">[15]</a> which might be indicative of an active mechanism of cell protein exclusion. It is not known whether gene extinction and cell machinery hijacking occurs in the <em>Mimivirus</em> factory to allow its replication and production, as has been described for other VFs <a href="#pone.0000328-Novoa1">[6]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>A large variety of virus factories have been described for unrelated viruses <a href="#pone.0000328-Novoa1">[6]</a>. It has previously been demonstrated that the replication site is predominantly cytoplasmic for <em>Poxviridae</em> <a href="#pone.0000328-Moss1">[16]</a>, nuclear and cytoplasmic for <em>Asfarviridae</em> <a href="#pone.0000328-Brookes1">[17]</a>, <a href="#pone.0000328-GarciaBeato1">[18]</a> and nuclear for <em>Iridoviridae</em> and <em>Phycodnaviridae</em> <a href="#pone.0000328-VanEtten1">[13]</a>, <a href="#pone.0000328-VanEtten2">[14]</a>, <a href="#pone.0000328-Williams1">[19]</a>, <a href="#pone.0000328-Williams2">[20]</a>, whereas the assembly sites are all cytoplasmic. The main characteristics of these viruses, as well as their replication and assembly sites, are summarized in Supporting Information, <a href="#pone.0000328.s001">Table S1</a>. Here, we described a new VF, which might be specific to the <em>Mimiviridae</em>, with a still undescribed replication centre which may insure a high degree of replication autonomy for this virus family regarding the host cell machinery. Similarities to the <em>Asfarviridae</em> could be observed, particularly in terms of early nuclear viral DNA replication, DNA insertion/encapsidation into pre-formed capsids, and number of VF per infected cell <a href="#pone.0000328-Brookes1">[17]</a>, <a href="#pone.0000328-Brookes2">[21]</a>. However major differences are noticeable such as the weak detection of membranes within the VF <a href="#pone.0000328-Andrs1">[22]</a>, or the larger <em>Mimivirus</em> factory area compared to ASFV <a href="#pone.0000328-Brookes1">[17]</a>. This is also true when comparison is made with other large DNA viruses factories which may occupy a large region of the infected cell <a href="#pone.0000328-Novoa1">[6]</a>. Another difference is the absence of membrane surrounding the Mimivirus factory. Several questions are raised by our results. First, what is the source of nucleotides for building such a large DNA structure? Second, is there an exploitation of the aggresome pathway by cytoplasmic <em>Mimivirus</em> DNA to concentrate viral proteins at the assembly site, as previously reported for African swine fever virus <a href="#pone.0000328-Heath1">[23]</a>? Third, how are host proteins and organelles excluded from the VF region, and how is the cellular cytoskeleton reorganized?</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In conclusion, the <em>Mimivirus</em> particle, composed of RNA transcripts combined with more than 100 viral proteins, appears to be particularly complex. Very specific mechanisms and complex interactions between viral and cellular factors must be involved to build this remarkably large and efficient VF, which can rapidly generate such a sophisticated microorganism.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Viral infection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>A. polyphaga</em> were seeded at 4×10<sup>5</sup> cells/ml in Page's amoebal saline (PAS) <a href="#pone.0000328-Rowbotham1">[24]</a>, infected with titrated <em>Mimivirus</em> at an amoeba cell:virus ratio of 1:10 and centrifuged at 1,000× g for 30 min. Amoebae viability was estimated by counting the cells immediately after centrifugation and every two hours after that for the next 32 h.</p>


<h4>Electron microscopy and immunofluorescence</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p><em>Mimivirus</em>-infected <em>A. polyphaga</em> were prepared for TEM as follows. Cells were washed three times in PBS, resuspended in 5% glutaraldehyde (Sigma) in PBS for 1 h at 4°C and then washed again three times in PBS. The cell pellet was fixed in 1% osmic acid, washed twice in PBS, dehydrated in 50, 70, 95 and 100% alcohol and embedded in Epon.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For fluorescence labelling, 100 µl of cell suspension at 4×10<sup>5 </sup>cells/ml were put into a Cytospin chamber, centrifuged for 10 min at 800 rpm in a Shandon Cytospin 4 (Thermo Electron Corporation) and then fixed for 10 min in methanol. For direct fluorescence with DAPI (4′,6′-diamidino-2-phenylindole) staining, cells were covered with 5 µM DAPI from a ready-to-use solution, “ProLong Gold Antifade Reagent” (Molecular Probes) and stained for 10 min in the dark prior to observation. For indirect immunofluorescence, 100 µl of mAb P4C8G2, raised against purified <em>Mimivirus</em> (data not shown), was diluted 1:100 in PBS with 3% (w/v) non-fat dry milk and added to the slides. Slides were incubated in a moist chamber at 37°C for 30 min. After three washes in PBS, the slides were incubated for 30 min at 37°C with 100 µl of a FITC-conjugated goat anti-mouse Ig (Jackson ImmunoResearch) diluted 1:100 in PBS containing 0.2% Evans blue. After three washes with PBS, the slides were mounted using a phosphate-buffered glycerol medium, pH 8, prior to observation.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Cells were observed using upright microscopes (Olympus BX 51and Zeiss Axio Imager) equipped with 40×, 63× or 100× lenses. DIC images were acquired using the Axio Imager microscope. All images were acquired with a cooled (−30°C) DS1-QM (Nikon) black and white camera driven by “Lucia G” software (Nikon &amp; LIM Ltd. Prague, Czech Republic). DAPI-fluorescence images were taken using a DAPI filter (360/55 nm; 460/50 nm). FITC-mAb images were taken using an FITC filter (480/20; 535/40). Confocal images were acquired with an LSM 510 Zeiss microscope, with DAPI staining observed using a UV diode (405 nm), z step = 0.3 µm. 3D volumic reconstruction was achieved using OsiriX Medical Imaging Software <a href="#pone.0000328-Rosset1">[25]</a>. The topology of infected cells was obtained using limited depth focus DIC images. The 3D reconstruction was obtained with Lucia software's EDF algorithm. The position of the VF replication centre in the EDF image was obtained by overlay of the DAPI fluorescence image.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Image analysis was performed using “Lucia G” and ImageJ software (Rasband, W.S., ImageJ, National Institutes of Health, Bethesda, Maryland, USA, 1997–2006, <a href="http://rsb.info.nih.gov/ij/">http://rsb.info.nih.gov/ij/</a>). Images were acquired in 12 bit depth with the same exposure parameters. Three images per field were recorded, and a total of 1030 cells were analyzed using the following protocol. The absolute value of intensity was measured in regions of interest (ROIs): nucleus, cytoplasm, DNA clusters, monoclonal Ab staining, background. The following parameters were measured: area = sum of ROIs area; area fraction = area/area of cells in the field; intensity = [mean of (ROIs intensity/ROI area)]–background. In order to compare in the same graph the variations of&nbsp;the parameters having different units (Intensity, Area Fraction) we calculated for each parameter the “centered and normalized” value using the formula: X<sub>t</sub> = (X<sub>t</sub>−mean<sub>X</sub>)/(X<sub>max</sub>−X<sub>min</sub>).</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>To quantify the evolution of the mean fluorescence intensity, measurements were compared: 0 h p.i. versus 3 h p.i. and 0 h p.i. versus 8 h p.i. For this purpose, four parameters were measured in DAPI-stained <em>Mimivirus</em> infected cells: nuclear area, mean nuclear fluorescence intensity, VF area and mean VF fluorescence intensity. For fluorescence quantification, acquisition time was 4 msec for the VFs and 100 msec for the nuclei. Statistical analyses were performed with the Wilcoxon test using R software (R Development Core Team (2006). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. ISBN 3-900051-07-0, URL <a href="http://www.R-project.org">http://www.R-project.org</a>.)</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000328"></div><a name="pone.0000328.s001" id="pone.0000328.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000328.s001">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000328.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000328.s002" id="pone.0000328.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000328.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">General characteristics of virus factories of large dsDNA viruses</p>
<p class="siDoi">doi:10.1371/journal.pone.0000328.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000328.s003" id="pone.0000328.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000328.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Kinetics of Mimivirus factory formation. The cellular location of Mimivirus AT-rich DNA was monitored by DAPI staining during the time course of A. polyphaga infection. Fluorescence images were taken with a 40× lens with an exposure time of 64 msec (main images) or with a 63×/1.4 oil lens with an exposure time of 128 msec (inset images).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000328.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.35 MB TIF)</p>
<a name="pone.0000328.s004" id="pone.0000328.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000328.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">3D reconstruction and model of Mimivirus factory. (A) Volumic reconstruction of Mimivirus factory. DAPI stained Mimivirus infected A. polyphaga were observed with a confocal microscope at 16 h p.i. Fluorescence intensity was represented by a rainbow logarithmic look up table. The respective 2D maximum intensity projections of the II, III and IV regions are shown in the lower part : II, mature stage of the growing VF; III, productive stage; IV, degenerative stage. Bar = 10 µm. (B) 3D model of Mimivirus factory.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000328.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.31 MB TIF)</p>
<a name="pone.0000328.s005" id="pone.0000328.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000328.s005">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Molecular characterization of the Mimivirus factory. Combined labelling of Mimivirus AT-rich DNA with DAPI staining (direct fluorescence; blue) and Mimivirus R710 protein with a specific mAb by indirect immunofluorescence (green) was performed during the time course of A. polyphaga infection. The protein showed a punctuated staining pattern starting at 6 h post-infection around the DAPI-stained Mimivirus factory. Thereafter, the number and intensity of anti-R710 mAb-stained Mimivirus factory increased until the end of the viral cycle.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000328.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.76 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors gratefully acknowledge Nicolas Aldrovandi and Bernard Campagna for their technical assistance with electron microscopy, Claude Nappez for anti-<em>Mimivirus</em> monoclonal antibody production and Catherine Robert for unpublished <em>A. polyphaga</em> sequencing results. We would like also to thank Pascal Weber at the Institut de Biologie du Développement de Marseille (IBDM) for his help with the confocal microscope, and Dr Valéry Combes for critically reading the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BL DR MS. Performed the experiments: LB LE. Analyzed the data: DR MS LB LE. Wrote the paper: BL DR MS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000328-LaScola1" id="pone.0000328-LaScola1"></a>La Scola B, Audic S, Robert C, Jungang L, de Lamballerie X, et al.  (2003) A giant virus in amoebae. Science  299: 2033.  <ul class="find" data-citedArticleID="940534" data-doi="10.1126/science.1081867"><li><a href="http://dx.doi.org/10.1126/science.1081867" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+giant+virus+in+amoebae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+giant+virus+in+amoebae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000328-SuzanMonti1" id="pone.0000328-SuzanMonti1"></a>Suzan-Monti M, La Scola B, Raoult D (2006) Genomic and evolutionary aspects of Mimivirus. Virus Res  117: 145–155.  <ul class="find" data-citedArticleID="940552" data-doi="10.1016/j.virusres.2005.07.011"><li><a href="http://dx.doi.org/10.1016/j.virusres.2005.07.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+and+evolutionary+aspects+of+Mimivirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+and+evolutionary+aspects+of+Mimivirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000328-Xiao1" id="pone.0000328-Xiao1"></a>Xiao C, Chipman PR, Battisti AJ, Bowman VD, Renesto P, et al.  (2005) Cryo-electron microscopy of the giant Mimivirus. J Mol Biol  353: 493–496.  <ul class="find" data-citedArticleID="940566" data-doi="10.1016/j.jmb.2005.08.060"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.08.060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryo-electron+microscopy+of+the+giant+Mimivirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryo-electron+microscopy+of+the+giant+Mimivirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000328-Raoult1" id="pone.0000328-Raoult1"></a>Raoult D, Audic S, Robert C, Abergel C, Renesto P, et al.  (2004) The 1.2-megabase genome sequence of Mimivirus. Science  306: 1344–1350.  <ul class="find" data-citedArticleID="940542" data-doi="10.1126/science.1101485"><li><a href="http://dx.doi.org/10.1126/science.1101485" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+1.2-megabase+genome+sequence+of+Mimivirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+1.2-megabase+genome+sequence+of+Mimivirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000328-Iyer1" id="pone.0000328-Iyer1"></a>Iyer LM, Aravind L, Koonin EV (2001) Common origin of four diverse families of large eukaryotic DNA viruses. J Virol  75: 11720–11734.  <ul class="find" data-citedArticleID="940530" data-doi="10.1128/jvi.75.23.11720-11734.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.23.11720-11734.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+origin+of+four+diverse+families+of+large+eukaryotic+DNA+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+origin+of+four+diverse+families+of+large+eukaryotic+DNA+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000328-Novoa1" id="pone.0000328-Novoa1"></a>Novoa RR, Calderita G, Arranz R, Fontana J, Granzow H, et al.  (2005) Virus factories: associations of cell organelles for viral replication and morphogenesis. Biol Cell  97: 147–172.  <ul class="find" data-citedArticleID="940540" data-doi="10.1042/bc20040058"><li><a href="http://dx.doi.org/10.1042/bc20040058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virus+factories%3A+associations+of+cell+organelles+for+viral+replication+and+morphogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virus+factories%3A+associations+of+cell+organelles+for+viral+replication+and+morphogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000328-Kubista1" id="pone.0000328-Kubista1"></a>Kubista M, Akerman B, Norden B (1987) Characterization of interaction between DNA and 4′,6-diamidino-2-phenylindole by optical spectroscopy. Biochemistry  26: 4545–4553.  <ul class="find" data-citedArticleID="940532" data-doi="10.1021/bi00388a057"><li><a href="http://dx.doi.org/10.1021/bi00388a057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+interaction+between+DNA+and+4%26%23x2032%3B%2C6-diamidino-2-phenylindole+by+optical+spectroscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+interaction+between+DNA+and+4%26%23x2032%3B%2C6-diamidino-2-phenylindole+by+optical+spectroscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000328-Valdecasas1" id="pone.0000328-Valdecasas1"></a>Valdecasas AG, Marshall D, Becerra JM, Terrero JJ (2001) On the extended depth of focus algorithms for bright field microscopy. Micron  32: 559–569.  <ul class="find" data-citedArticleID="940554" data-doi="10.1016/s0968-4328(00)00061-5"><li><a href="http://dx.doi.org/10.1016/s0968-4328(00)00061-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+extended+depth+of+focus+algorithms+for+bright+field+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+extended+depth+of+focus+algorithms+for+bright+field+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000328-LaScola2" id="pone.0000328-LaScola2"></a>La Scola B, Marrie TJ, Auffray JP, Raoult D (2005) Mimivirus in pneumonia patients. Emerg Inf Dis  11: 449–452.  <ul class="find" data-citedArticleID="940536" data-doi="10.3201/eid1103.040538"><li><a href="http://dx.doi.org/10.3201/eid1103.040538" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mimivirus+in+pneumonia+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mimivirus+in+pneumonia+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000328-Berger1" id="pone.0000328-Berger1"></a>Berger P, Papazian L, Drancourt M, La Scola B, Auffray JP, et al.  (2006) Ameba-associated microorganisms and diagnosis of nosocomial pneumonia. Emerg Inf Dis  12: 248–255.  <ul class="find" data-citedArticleID="940518" data-doi="10.3201/eid1202.050434"><li><a href="http://dx.doi.org/10.3201/eid1202.050434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ameba-associated+microorganisms+and+diagnosis+of+nosocomial+pneumonia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ameba-associated+microorganisms+and+diagnosis+of+nosocomial+pneumonia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000328-Raoult2" id="pone.0000328-Raoult2"></a>Raoult D, Renesto P, Brouqui P (2006) Laboratory infection of a technician by mimivirus. Ann Int Med  144: 702–703.  <ul class="find" data-citedArticleID="940544" data-doi="10.7326/0003-4819-144-9-200605020-00025"><li><a href="http://dx.doi.org/10.7326/0003-4819-144-9-200605020-00025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laboratory+infection+of+a+technician+by+mimivirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laboratory+infection+of+a+technician+by+mimivirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000328-Greub1" id="pone.0000328-Greub1"></a>Greub G, Raoult D (2004) Microorganisms resistant to free-living amoebae. Clin Microbiol Rev  17: 413–433.  <ul class="find" data-citedArticleID="940526" data-doi="10.1128/cmr.17.2.413-433.2004"><li><a href="http://dx.doi.org/10.1128/cmr.17.2.413-433.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microorganisms+resistant+to+free-living+amoebae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microorganisms+resistant+to+free-living+amoebae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000328-VanEtten1" id="pone.0000328-VanEtten1"></a>Van Etten JL, Graves MV, Muller DG, Boland W, Delaroque N (2002) Phycodnaviridae–large DNA algal viruses. Arch Virol  147: 1479–1516.  <ul class="find" data-citedArticleID="940556" data-doi="10.1007/s00705-002-0822-6"><li><a href="http://dx.doi.org/10.1007/s00705-002-0822-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phycodnaviridae%26%23x2013%3Blarge+DNA+algal+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phycodnaviridae%26%23x2013%3Blarge+DNA+algal+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000328-VanEtten2" id="pone.0000328-VanEtten2"></a>Van Etten JL (2003) Unusual life style of giant chlorella viruses. Ann Rev Genet  37: 153–195.  <ul class="find" data-citedArticleID="940558"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Unusual+life+style+of+giant+chlorella+viruses.&amp;auth=&amp;atitle=Unusual+life+style+of+giant+chlorella+viruses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unusual+life+style+of+giant+chlorella+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unusual+life+style+of+giant+chlorella+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000328-Renesto1" id="pone.0000328-Renesto1"></a>Renesto P, Abergel C, Decloquement P, Moinier D, Azza S, et al.  (2006) Mimivirus giant particles incorporate a large fraction of anonymous and unique gene products. J Virol  80: 11678–11685.  <ul class="find" data-citedArticleID="940546" data-doi="10.1128/jvi.00940-06"><li><a href="http://dx.doi.org/10.1128/jvi.00940-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mimivirus+giant+particles+incorporate+a+large+fraction+of+anonymous+and+unique+gene+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mimivirus+giant+particles+incorporate+a+large+fraction+of+anonymous+and+unique+gene+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000328-Moss1" id="pone.0000328-Moss1"></a>Moss B (2001)  Poxviridae: the viruses and their replication. In: Knipe DM, Holey PM, editors. <em>Fields Virology</em>. Philadelphia: Lippincott Williams and Wilkins. pp. 2637–2672.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000328-Brookes1" id="pone.0000328-Brookes1"></a>Brookes SM, Dixon LK, Parkhouse RME (1996) Assembly of African Swine Fever Virus: quantitative ultrastructural analysis in vitro and in vivo. Virology  224: 84–92.  <ul class="find" data-citedArticleID="940520" data-doi="10.1006/viro.1996.0509"><li><a href="http://dx.doi.org/10.1006/viro.1996.0509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assembly+of+African+Swine+Fever+Virus%3A+quantitative+ultrastructural+analysis+in+vitro+and+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assembly+of+African+Swine+Fever+Virus%3A+quantitative+ultrastructural+analysis+in+vitro+and+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000328-GarciaBeato1" id="pone.0000328-GarciaBeato1"></a>Garcia-Beato R, Salas ML, Vinuela E, Salas J (1992) Role of the host cell nucleus in the replication of African swine fever virus DNA. Virology  188: 637–649.  <ul class="find" data-citedArticleID="940524" data-doi="10.1016/0042-6822(92)90518-t"><li><a href="http://dx.doi.org/10.1016/0042-6822(92)90518-t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+host+cell+nucleus+in+the+replication+of+African+swine+fever+virus+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+host+cell+nucleus+in+the+replication+of+African+swine+fever+virus+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000328-Williams1" id="pone.0000328-Williams1"></a>Williams T (1996) The Iridoviruses. Adv Virus Res  46: 345–412.  <ul class="find" data-citedArticleID="940562" data-doi="10.1016/s0065-3527(08)60076-7"><li><a href="http://dx.doi.org/10.1016/s0065-3527(08)60076-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Iridoviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Iridoviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000328-Williams2" id="pone.0000328-Williams2"></a>Williams T, Barbosa-Solomieu V, Chinchar VG (2005) A decade of advances in iridovirus research. Adv Virus Res  65: 173–248.  <ul class="find" data-citedArticleID="940564" data-doi="10.1016/s0065-3527(05)65006-3"><li><a href="http://dx.doi.org/10.1016/s0065-3527(05)65006-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+decade+of+advances+in+iridovirus+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+decade+of+advances+in+iridovirus+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000328-Brookes2" id="pone.0000328-Brookes2"></a>Brookes SM, Hyatt AD, Wise T, Parkhouse RME (1998) Intracellular virus DNA distribution and the acquisition of the nucleoprotein core during African Swine Fever Virus particle assembly: ultrastructural <em>in situ</em> hybridisation and DNase-gold labelling. Virology  249: 175–188.  <ul class="find" data-citedArticleID="940522" data-doi="10.1006/viro.1998.9308"><li><a href="http://dx.doi.org/10.1006/viro.1998.9308" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intracellular+virus+DNA+distribution+and+the+acquisition+of+the+nucleoprotein+core+during+African+Swine+Fever+Virus+particle+assembly%3A+ultrastructural+in+situ+hybridisation+and+DNase-gold+labelling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intracellular+virus+DNA+distribution+and+the+acquisition+of+the+nucleoprotein+core+during+African+Swine+Fever+Virus+particle+assembly%3A+ultrastructural+in+situ+hybridisation+and+DNase-gold+labelling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000328-Andrs1" id="pone.0000328-Andrs1"></a>Andrés G, Simón-Mateo C, Vińuela E (1997) Assembly of African Swine Fever Virus: role of polyprotein pp220. J Virol  71: 2331–2341.  <ul class="find" data-citedArticleID="940516"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Assembly+of+African+Swine+Fever+Virus%3A+role+of+polyprotein+pp220.&amp;auth=&amp;atitle=Assembly+of+African+Swine+Fever+Virus%3A+role+of+polyprotein+pp220." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assembly+of+African+Swine+Fever+Virus%3A+role+of+polyprotein+pp220." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assembly+of+African+Swine+Fever+Virus%3A+role+of+polyprotein+pp220.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000328-Heath1" id="pone.0000328-Heath1"></a>Heath CM, Windsor M, Wileman T (2001) Aggresomes resemble sites specialized for virus assembly. J Cell Biol  153: 449–455.  <ul class="find" data-citedArticleID="940528" data-doi="10.1083/jcb.153.3.449"><li><a href="http://dx.doi.org/10.1083/jcb.153.3.449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aggresomes+resemble+sites+specialized+for+virus+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aggresomes+resemble+sites+specialized+for+virus+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000328-Rowbotham1" id="pone.0000328-Rowbotham1"></a>Rowbotham TJ (1980) Preliminary report on the pathogenicity of Legionella pneumophila for freshwater and soil amoebae. J Clin Pathol  33: 1179–1183.  <ul class="find" data-citedArticleID="940550" data-doi="10.1136/jcp.33.12.1179"><li><a href="http://dx.doi.org/10.1136/jcp.33.12.1179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preliminary+report+on+the+pathogenicity+of+Legionella+pneumophila+for+freshwater+and+soil+amoebae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preliminary+report+on+the+pathogenicity+of+Legionella+pneumophila+for+freshwater+and+soil+amoebae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000328-Rosset1" id="pone.0000328-Rosset1"></a>Rosset A, Spadola L, Ratib O (2004) OsiriX: an open-source software for navigating in multidimensional DICOM images. J Digit Imaging  17: 205–216.  <ul class="find" data-citedArticleID="940548" data-doi="10.1007/s10278-004-1014-6"><li><a href="http://dx.doi.org/10.1007/s10278-004-1014-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OsiriX%3A+an+open-source+software+for+navigating+in+multidimensional+DICOM+images." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OsiriX%3A+an+open-source+software+for+navigating+in+multidimensional+DICOM+images.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000328-VanEtten3" id="pone.0000328-VanEtten3"></a>Van Etten JL, Meints RH (1999) Giant viruses infecting algae. Ann Rev Microbiol  53: 447–494.  <ul class="find" data-citedArticleID="940560" data-doi="10.1146/annurev.micro.53.1.447"><li><a href="http://dx.doi.org/10.1146/annurev.micro.53.1.447" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Giant+viruses+infecting+algae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Giant+viruses+infecting+algae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.XML" value="75544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.PDF" value="1258343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g001.PNG_L" value="2548673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g001.PNG_M" value="246133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g001.PNG_S" value="8459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g001.TIF" value="2319034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g001.PNG_I" value="131768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g002.PNG_L" value="469271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g002.PNG_M" value="253393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g002.PNG_S" value="10774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g002.TIF" value="583964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g002.PNG_I" value="100198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g003.PNG_L" value="280789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g003.PNG_M" value="280789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g003.PNG_S" value="6562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g003.TIF" value="348920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g003.PNG_I" value="127283"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g004.PNG_L" value="1595797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g004.PNG_M" value="131760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g004.PNG_S" value="11963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g004.TIF" value="1435720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g004.PNG_I" value="183959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g005.PNG_L" value="1419874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g005.PNG_M" value="196223"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g005.PNG_S" value="10959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g005.TIF" value="2284518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g005.PNG_I" value="117375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g006.PNG_L" value="3573103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g006.PNG_M" value="185660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g006.PNG_S" value="5521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g006.TIF" value="5199300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g006.PNG_I" value="65415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g007.PNG_L" value="54542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g007.PNG_M" value="27434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g007.PNG_S" value="2715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g007.TIF" value="664294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g007.PNG_I" value="28233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g008.PNG_L" value="22225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g008.PNG_M" value="88256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g008.PNG_S" value="3035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g008.TIF" value="88578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.g008.PNG_I" value="38533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.s001.DOC" value="35328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.s002.DOC" value="36864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.s003.TIF" value="349672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.s004.TIF" value="306760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000328.s005.TIF" value="757914"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000328&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000328" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000328&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000328&volume=&issue=&title=Ultrastructural Characterization of the Giant Volcano-like Virus Factory of Acanthamoeba polyphaga Mimivirus&author_name=Marie%20Suzan-Monti%2C%20Bernard%20La%20Scola%2C%20Lina%20Barrassi%2C%20Leon%20Espinosa%2C%20Didier%20Raoult&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000328" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000328" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000328" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000328&amp;t=Ultrastructural%20Characterization%20of%20the%20Giant%20Volcano-like%20Virus%20Factory%20of%20Acanthamoeba%20polyphaga%20Mimivirus" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000328&title=Ultrastructural%20Characterization%20of%20the%20Giant%20Volcano-like%20Virus%20Factory%20of%20Acanthamoeba%20polyphaga%20Mimivirus&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000328&amp;title=Ultrastructural%20Characterization%20of%20the%20Giant%20Volcano-like%20Virus%20Factory%20of%20Acanthamoeba%20polyphaga%20Mimivirus" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000328" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000328&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Ultrastructural Characterization of the Giant Volcano-like Virus Factory of Acanthamoeba polyphaga Mimivirus';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000328';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Ultrastructural%20Characterization%20of%20the%20Giant%20Volcano-like%20Virus%20Factory%20of%20Acanthamoeba%20polyphaga%20Mimivirus http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000328" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000328" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amoebas%22" title="Search for articles in the subject area:'Amoebas'"><div class="flagText">Amoebas</div></a>
              <div data-categoryid="43161" data-articleid="23494"
                   data-categoryname="Amoebas"
                   class="flagImage" title="Flag 'Amoebas' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Capsids%22" title="Search for articles in the subject area:'Capsids'"><div class="flagText">Capsids</div></a>
              <div data-categoryid="22271" data-articleid="23494"
                   data-categoryname="Capsids"
                   class="flagImage" title="Flag 'Capsids' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cytoplasm%22" title="Search for articles in the subject area:'Cytoplasm'"><div class="flagText">Cytoplasm</div></a>
              <div data-categoryid="17277" data-articleid="23494"
                   data-categoryname="Cytoplasm"
                   class="flagImage" title="Flag 'Cytoplasm' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cytoplasmic+staining%22" title="Search for articles in the subject area:'Cytoplasmic staining'"><div class="flagText">Cytoplasmic staining</div></a>
              <div data-categoryid="17577" data-articleid="23494"
                   data-categoryname="Cytoplasmic staining"
                   class="flagImage" title="Flag 'Cytoplasmic staining' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DAPI+staining%22" title="Search for articles in the subject area:'DAPI staining'"><div class="flagText">DAPI staining</div></a>
              <div data-categoryid="17417" data-articleid="23494"
                   data-categoryname="DAPI staining"
                   class="flagImage" title="Flag 'DAPI staining' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+replication%22" title="Search for articles in the subject area:'DNA replication'"><div class="flagText">DNA replication</div></a>
              <div data-categoryid="19773" data-articleid="23494"
                   data-categoryname="DNA replication"
                   class="flagImage" title="Flag 'DNA replication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mimivirus%22" title="Search for articles in the subject area:'Mimivirus'"><div class="flagText">Mimivirus</div></a>
              <div data-categoryid="43395" data-articleid="23494"
                   data-categoryname="Mimivirus"
                   class="flagImage" title="Flag 'Mimivirus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+replication%22" title="Search for articles in the subject area:'Viral replication'"><div class="flagText">Viral replication</div></a>
              <div data-categoryid="20015" data-articleid="23494"
                   data-categoryname="Viral replication"
                   class="flagImage" title="Flag 'Viral replication' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8710'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4904'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3850&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
