

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in S. pombe</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000722"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000722" />

    <meta name="citation_title" content="Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in S. pombe"/>
    <meta itemprop="name" content="Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in S. pombe"/>

      <meta name="citation_author" content="Majid Eshaghi"/>
            <meta name="citation_author_institution" content="Systems Biology, Genome Institute of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="R. Krishna M. Karuturi"/>
            <meta name="citation_author_institution" content="Computational and Mathematical Biology, Genome Institute of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Juntao Li"/>
            <meta name="citation_author_institution" content="Computational and Mathematical Biology, Genome Institute of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Zhaoqing Chu"/>
            <meta name="citation_author_institution" content="Systems Biology, Genome Institute of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Edison T. Liu"/>
            <meta name="citation_author_institution" content="Cancer Biology, Genome Institute of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Jianhua Liu"/>
            <meta name="citation_author_institution" content="Systems Biology, Genome Institute of Singapore, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore"/>

    <meta name="citation_date" content="2007/8/8"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000722.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e722"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Nuclear position leaves its mark on replication timing.; citation_author=DM Gilbert; citation_journal_title=J Cell Biol; citation_volume=152; citation_number=1; citation_pages=F11-15; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A genomic view of eukaryotic DNA replication.; citation_author=DM MacAlpine; citation_author=SP Bell; citation_journal_title=Chromosome Res; citation_volume=13; citation_number=2; citation_pages=309-326; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA replication in eukaryotic cells.; citation_author=SP Bell; citation_author=A Dutta; citation_journal_title=Annu Rev Biochem; citation_volume=71; citation_number=3; citation_pages=333-374; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Replication dynamics of the yeast genome.; citation_author=MK Raghuraman; citation_author=EA Winzeler; citation_author=D Collingwood; citation_author=S Hunt; citation_author=L Wodicka; citation_journal_title=Science; citation_volume=294; citation_number=4; citation_pages=115-121; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome-wide DNA replication profile for Drosophila melanogaster: a link between transcription and replication timing.; citation_author=D Schubeler; citation_author=D Scalzo; citation_author=C Kooperberg; citation_author=B van Steensel; citation_author=J Delrow; citation_journal_title=Nat Genet; citation_volume=32; citation_number=5; citation_pages=438-442; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mapping of early firing origins on a replication profile of budding yeast.; citation_author=N Yabuki; citation_author=H Terashima; citation_author=K Kitada; citation_journal_title=Genes Cells; citation_volume=7; citation_number=6; citation_pages=781-789; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Temporal profile of replication of human chromosomes.; citation_author=Y Jeon; citation_author=S Bekiranov; citation_author=N Karnani; citation_author=P Kapranov; citation_author=S Ghosh; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=7; citation_pages=6419-6424; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of replication timing in fission yeast.; citation_author=SM Kim; citation_author=JA Huberman; citation_journal_title=Embo J; citation_volume=20; citation_number=8; citation_pages=6115-6126; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA replication origins fire stochastically in fission yeast.; citation_author=PK Patel; citation_author=B Arcangioli; citation_author=SP Baker; citation_author=A Bensimon; citation_author=N Rhind; citation_journal_title=Mol Biol Cell; citation_volume=17; citation_number=9; citation_pages=308-316; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fission yeast genes involved in coupling mitosis to completion of DNA replication.; citation_author=T Enoch; citation_author=AM Carr; citation_author=P Nurse; citation_journal_title=Genes Dev; citation_volume=6; citation_number=10; citation_pages=2035-2046; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of the Schizosaccharomyces pombe rad3 gene, involved in the DNA damage and DNA synthesis checkpoints.; citation_author=BL Seaton; citation_author=J Yucel; citation_author=P Sunnerhagen; citation_author=S Subramani; citation_journal_title=Gene; citation_volume=119; citation_number=11; citation_pages=83-89; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of new elements involved in checkpoint and feedback controls in fission yeast.; citation_author=F al-Khodairy; citation_author=E Fotou; citation_author=KS Sheldrick; citation_author=DJ Griffiths; citation_author=AR Lehmann; citation_journal_title=Mol Biol Cell; citation_volume=5; citation_number=12; citation_pages=147-160; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A kinase from fission yeast responsible for blocking mitosis in S phase.; citation_author=H Murakami; citation_author=H Okayama; citation_journal_title=Nature; citation_volume=374; citation_number=13; citation_pages=817-819; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=S-phase-specific activation of Cds1 kinase defines a subpathway of the checkpoint response in Schizosaccharomyces pombe.; citation_author=HD Lindsay; citation_author=DJ Griffiths; citation_author=RJ Edwards; citation_author=PU Christensen; citation_author=JM Murray; citation_journal_title=Genes Dev; citation_volume=12; citation_number=14; citation_pages=382-395; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Fork reversal and ssDNA accumulation at stalled replication forks owing to checkpoint defects.; citation_author=JM Sogo; citation_author=M Lopes; citation_author=M Foiani; citation_journal_title=Science; citation_volume=297; citation_number=15; citation_pages=599-602; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Replication checkpoint enforced by kinases Cds1 and Chk1.; citation_author=MN Boddy; citation_author=B Furnari; citation_author=O Mondesert; citation_author=P Russell; citation_journal_title=Science; citation_volume=280; citation_number=16; citation_pages=909-912; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Structure and function of the radical enzyme ribonucleotide reductase.; citation_author=H Eklund; citation_author=U Uhlin; citation_author=M Farnegardh; citation_author=DT Logan; citation_author=P Nordlund; citation_journal_title=Prog Biophys Mol Biol; citation_volume=77; citation_number=17; citation_pages=177-268; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Regulation of DNA-replication origins during cell-cycle progression.; citation_author=K Shirahige; citation_author=Y Hori; citation_author=K Shiraishi; citation_author=M Yamashita; citation_author=K Takahashi; citation_journal_title=Nature; citation_volume=395; citation_number=18; citation_pages=618-621; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genomic mapping of single-stranded DNA in hydroxyurea-challenged yeasts identifies origins of replication.; citation_author=W Feng; citation_author=D Collingwood; citation_author=ME Boeck; citation_author=LA Fox; citation_author=GM Alvino; citation_journal_title=Nat Cell Biol; citation_volume=8; citation_number=19; citation_pages=148-155; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A Mec1- and Rad53-dependent checkpoint controls late-firing origins of DNA replication.; citation_author=C Santocanale; citation_author=JF Diffley; citation_journal_title=Nature; citation_volume=395; citation_number=20; citation_pages=615-618; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genome-wide characterization of fission yeast DNA replication origins.; citation_author=C Heichinger; citation_author=CJ Penkett; citation_author=J Bahler; citation_author=P Nurse; citation_journal_title=Embo J; citation_volume=25; citation_number=21; citation_pages=5171-5179; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic and physiological analysis of DNA replication in fission yeast.; citation_author=SA MacNeill; citation_author=PA Fantes; citation_journal_title=Methods Enzymol; citation_volume=283; citation_number=22; citation_pages=440-459; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Identification of cell cycle-regulated genes in fission yeast.; citation_author=X Peng; citation_author=RK Karuturi; citation_author=LD Miller; citation_author=K Lin; citation_author=Y Jia; citation_journal_title=Mol Biol Cell; citation_volume=16; citation_number=23; citation_pages=1026-1042; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA replication timing: random thoughts about origin firing.; citation_author=N Rhind; citation_journal_title=Nat Cell Biol; citation_volume=8; citation_number=24; citation_pages=1313-1316; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide mapping of the cohesin complex in the yeast Saccharomyces cerevisiae.; citation_author=EF Glynn; citation_author=PC Megee; citation_author=HG Yu; citation_author=C Mistrot; citation_author=E Unal; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=25; citation_pages=E259; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Normalization for cDNA microarray data: a robust composite method addressing single and multiple slide systematic variation.; citation_author=YH Yang; citation_author=S Dudoit; citation_author=P Luu; citation_author=DM Lin; citation_author=V Peng; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=26; citation_pages=e15; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome-wide distribution of DNA replication origins at A+T-rich islands in Schizosaccharomyces pombe.; citation_author=M Segurado; citation_author=A de Luis; citation_author=F Antequera; citation_journal_title=EMBO Rep; citation_volume=4; citation_number=27; citation_pages=1048-1053; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide localization of pre-RC sites and identification of replication origins in fission yeast.; citation_author=M Hayashi; citation_author=Y Katou; citation_author=T Itoh; citation_author=M Tazumi; citation_author=Y Yamada; citation_journal_title=Embo J; citation_volume=26; citation_number=28; citation_pages=1327-1339; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=DNA replication origins in the Schizosaccharomyces pombe genome.; citation_author=J Dai; citation_author=RY Chuang; citation_author=TJ Kelly; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=29; citation_pages=337-342; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA replication in the fission yeast: robustness in the face of uncertainty.; citation_author=I Legouras; citation_author=G Xouri; citation_author=S Dimopoulos; citation_author=J Lygeros; citation_author=Z Lygerou; citation_journal_title=Yeast; citation_volume=23; citation_number=30; citation_pages=951-962; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular genetic analysis of fission yeast Schizosaccharomyces pombe.; citation_author=S Moreno; citation_author=A Klar; citation_author=P Nurse; citation_journal_title=Methods Enzymol; citation_volume=194; citation_number=31; citation_pages=795-823; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Schizosaccharomyces pombe.; citation_author=V Wood; citation_author=R Gwilliam; citation_author=MA Rajandream; citation_author=M Lyne; citation_author=R Lyne; citation_journal_title=Nature; citation_volume=415; citation_number=32; citation_pages=871-880; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000722" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in S. pombe"/>
    <meta name="twitter:description" content="BackgroundDuring S. pombe S-phase, initiation of DNA replication occurs at multiple sites (origins) that are enriched with AT-rich sequences, at various times. Current studies of genome-wide DNA replication profiles have focused on the DNA replication timing and origin location. However, the replication and/or firing efficiency of the individual origins on the genomic scale remain unclear.Methodology/Principal FindingsUsing the genome-wide ORF-specific DNA microarray analysis, we show that in S. pombe, individual origins fire with varying efficiencies and at different times during S-phase. The increase in DNA copy number plotted as a function of time is approximated to the near-sigmoidal model, when considering the replication start and end timings at individual loci in cells released from HU-arrest. Replication efficiencies differ from origin to origin, depending on the origin's firing efficiency. We have found that DNA replication is inefficient early in S-phase, due to inefficient firing at origins. Efficient replication occurs later, attributed to efficient but late-firing origins. Furthermore, profiles of replication timing in cds1&Delta; cells are abnormal, due to the failure in resuming replication at the collapsed forks. The majority of the inefficient origins, but not the efficient ones, are found to fire in cds1&Delta; cells after HU removal, owing to the firing at the remaining unused (inefficient) origins during HU treatment.Conclusions/SignificanceTaken together, our results indicate that efficient DNA replication/firing occurs late in S-phase progression in cells after HU removal, due to efficient late-firing origins. Additionally, checkpoint kinase Cds1p is required for maintaining the efficient replication/firing late in S-phase. We further propose that efficient late-firing origins are essential for ensuring completion of DNA duplication by the end of S-phase."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000722.g008"/>

  <meta property="og:title" content="Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in S. pombe" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000722" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1239'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3778'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7161&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000722">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000722" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000722&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in <i>S. pombe</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Majid Eshaghi
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Majid Eshaghi, R. Krishna M. Karuturi</p>

              
              

                <p>Affiliation:
                  Systems Biology, Genome Institute of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            R. Krishna M. Karuturi
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Majid Eshaghi, R. Krishna M. Karuturi</p>

              
              

                <p>Affiliation:
                  Computational and Mathematical Biology, Genome Institute of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Juntao Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Mathematical Biology, Genome Institute of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhaoqing Chu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Systems Biology, Genome Institute of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edison T. Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cancer Biology, Genome Institute of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jianhua Liu
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:liujh@gis.a-star.edu.sg">liujh@gis.a-star.edu.sg</a></p>

                <p>Affiliations:
                  Systems Biology, Genome Institute of Singapore, Singapore, Singapore, 
                  Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 08, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000722</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000722" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000722" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000722" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000722" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000722-g001" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-g002" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-g003" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-g004" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-t001" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-t002" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-g005" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-g006" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-g007" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000722-g008" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000722">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000722" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>During <em>S. pombe</em> S-phase, initiation of DNA replication occurs at multiple sites (origins) that are enriched with AT-rich sequences, at various times. Current studies of genome-wide DNA replication profiles have focused on the DNA replication timing and origin location. However, the replication and/or firing efficiency of the individual origins on the genomic scale remain unclear.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using the genome-wide ORF-specific DNA microarray analysis, we show that in <em>S. pombe</em>, individual origins fire with varying efficiencies and at different times during S-phase. The increase in DNA copy number plotted as a function of time is approximated to the near-sigmoidal model, when considering the replication start and end timings at individual loci in cells released from HU-arrest. Replication efficiencies differ from origin to origin, depending on the origin's firing efficiency. We have found that DNA replication is inefficient early in S-phase, due to inefficient firing at origins. Efficient replication occurs later, attributed to efficient but late-firing origins. Furthermore, profiles of replication timing in <em>cds1Δ</em> cells are abnormal, due to the failure in resuming replication at the collapsed forks. The majority of the inefficient origins, but not the efficient ones, are found to fire in <em>cds1Δ</em> cells after HU removal, owing to the firing at the remaining unused (inefficient) origins during HU treatment.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Taken together, our results indicate that efficient DNA replication/firing occurs late in S-phase progression in cells after HU removal, due to efficient late-firing origins. Additionally, checkpoint kinase Cds1p is required for maintaining the efficient replication/firing late in S-phase. We further propose that efficient late-firing origins are essential for ensuring completion of DNA duplication by the end of S-phase.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Eshaghi M, Karuturi RKM, Li J, Chu Z, Liu ET, et al.  (2007) Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in <em>S. pombe</em>. PLoS ONE 2(8):
          e722.
            doi:10.1371/journal.pone.0000722</p><p><strong>Academic Editor: </strong>Nick Rhind, University of Massachusetts Medical School, United States of America</p><p><strong>Received:</strong> February 13, 2007; <strong>Accepted:</strong> July 9, 2007; <strong>Published:</strong> August 8, 2007</p><p><strong>Copyright:</strong> © 2007 Eshaghi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Agency for Science, Technology and Research (A-STAR), Singapore.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>DNA replication is a key event in the cell cycle, occurring within a confined period termed S-phase. Replication initiates at various times at multiple sites (origins) in eukaryotic genomes <a href="#pone.0000722-Gilbert1">[1]</a>–<a href="#pone.0000722-Bell1">[3]</a>. Microarray analysis of enriched heavy:light nascent DNA in <em>Saccharomyces cerevisiae</em> has revealed a genomic view of DNA replication timing profiles: some regions of the genome are always replicated early in S phase, some in the middle, and others at the end, due to a strict timing of (efficient) firing at origins <a href="#pone.0000722-Raghuraman1">[4]</a>. Similar profiles of DNA replication timing have been generated using microarrays that monitor DNA copy number increase without enrichment of nascent DNA in <em>S. cerevisiae</em> and <em>Drosophila melanogaster</em> <a href="#pone.0000722-Schubeler1">[5]</a>, <a href="#pone.0000722-Yabuki1">[6]</a>. Studies in human HeLa cells have shown that more than 60% of the genome is slowly replicated throughout the S-phase, owing to a flexible timing of firing or inefficient firing at origins <a href="#pone.0000722-Jeon1">[7]</a>. It has been shown that DNA replication/firing in <em>S. pombe</em> is less efficient than that in <em>S. cerevisiae</em> <a href="#pone.0000722-Kim1">[8]</a>, <a href="#pone.0000722-Patel1">[9]</a>. Although inefficient origins are not readily detected by classic 2D-gel electrophoresis, a molecular combing technique that analyzes DNA replication/firing at single origins has been able to demonstrate the existence of infrequent firing events at origins in <em>S. pombe</em> <a href="#pone.0000722-Patel1">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In <em>S. pombe</em>, replication checkpoint controls mediated via the ATM-related kinase Rad3p (Mec1p in <em>S. cerevisiae</em>) <a href="#pone.0000722-Enoch1">[10]</a>–<a href="#pone.0000722-alKhodairy1">[12]</a> and checking DNA synthesis kinase Cds1p (Rad53p in <em>S. cerevisiae</em>) <a href="#pone.0000722-Murakami1">[13]</a>, <a href="#pone.0000722-Lindsay1">[14]</a>) play a pivotal role in restoring stalled replication forks and in the regulation of replication timing <a href="#pone.0000722-Lindsay1">[14]</a>–<a href="#pone.0000722-Boddy1">[16]</a>. Hydroxyurea (HU) is a commonly used compound that inhibits ribonucleotide reductase (RNR) leading to a depletion of the intracellular pool of dNTPs that induces DNA replication blocks <a href="#pone.0000722-Eklund1">[17]</a>. Patches (~5 Kb in size) of single-stranded DNA (ssDNA) are found to accumulate at sites of origins upon replication blocks <a href="#pone.0000722-Shirahige1">[18]</a>, <a href="#pone.0000722-Feng1">[19]</a>. The replication checkpoint appears to prevent the accumulation of ssDNA patches at some origins, likely the late-firing ones <a href="#pone.0000722-Santocanale1">[20]</a>, consistent with its role in regulation of replication timing. Genome-wide profiling of enriched ssDNA induced by HU treatment has revealed several hundreds of potential origins of replication in both <em>S. cerevisiae</em> and <em>S. pombe</em> <a href="#pone.0000722-Feng1">[19]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The genome-wide profiling of replication has been successfully applied to identify early-firing origins in the genome, although it has limitations in predicting the late-firing origins that are closely located to early-firing origins and especially the inefficient late-firing origins <a href="#pone.0000722-MacAlpine1">[2]</a>. The correlation between the fold-enrichment of HU-induced ssDNA and replication efficiency has been noted for some of the origins <a href="#pone.0000722-Feng1">[19]</a>,<a href="#pone.0000722-Heichinger1">[21]</a>. Estimation of firing efficiency by the fold-enrichment of ssDNA has been further extended to the whole genome <a href="#pone.0000722-Heichinger1">[21]</a>, although a small number of late-firing origins have been found to exhibit a high magnitude of ssDNA formation <a href="#pone.0000722-Yabuki1">[6]</a>, <a href="#pone.0000722-Feng1">[19]</a>. Therefore there remains a need to develop a method to directly determine the firing efficiencies of origins of DNA replication on a genomic scale.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study, we have used <em>S. pombe</em> genome-wide ORF-specific DNA microarrays to monitor DNA copy number increase in cells released from HU-block. A near-sigmoid model is applied for the determination of the rate of DNA copy number-increase as a function of time at individual loci across the genome. This is the first-of-its-kind study on assessing both the replication timing and efficiency at the genomic scale. We show that the rate of DNA copy number-increase in cells released after HU-arrest, is generally slow in early S-phase due to the inefficient early-firing at origins. Efficient replication appears to occur late in S-phase. Furthermore, we show that inefficient origins, but not efficient ones, are more likely to fire in <em>cds1Δ</em> cells, attributing to the remaining unused (inefficient) origins after HU removal.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Nomenclatures</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To simplify, the DNA replication process is broken down into three steps: start (<em>i.e.</em>, initiation of replication by firing and/or passive replication at individual loci), progression, and end (<em>i.e.</em>, completion of replication). We define replication start timing (<em>i.e.</em>, firing timing at origin loci or initial passive replication timing at non-origin loci), average replication timing (or half-completion timing), and (full-) completion timing as <em>T<sub>0</sub></em>, <em>T<sub>50</sub></em>, and <em>T<sub>100</sub></em>, respectively. The time period from the start to the end is defined as the duplication time <em>ΔT</em> ( = <em>T<sub>100</sub></em>-<em>T<sub>0</sub></em>). As one and only one copy of DNA at all loci would be synthesized during the S-phase, <em>η<sub>r</sub></em> · <em>ΔT</em> = 1, where <em>η<sub>r</sub></em> is the (average) replication efficiency. The replication efficiency <em>η<sub>r</sub></em> of the loci at or near the origin sites is the maximal estimate of the firing efficiency <em>η<sub>f</sub></em> of the origins.</p>


<h4>Duplication Time Is Extended in <em>rad3Δ</em> and <em>cds1Δ</em> Cells Released after HU-block</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Cells bearing a <em>rad3Δ</em> or <em>cds1Δ</em> allele are known to exhibit hypersensitivity to temporal treatment with HU, consistent with their replication checkpoint function. To determine whether DNA duplication could resume in <em>rad3Δ</em> or <em>cds1Δ</em> cells after temporal HU treatment, we treated log-phase growing cells for 3 hr with HU at a final concentration of 8 mM. A typical 2C-DNA content profile was observed for all strains before HU treatment, owing to a short G1-phase (<a href="#pone-0000722-g001">Figure 1</a>). After 3 hr of HU treatment, almost all <em>rad3Δ, cds1Δ</em>, and wild type cells showed a 1C-DNA content profile, indicating an early S-phase arrest.</p>
<div class="figure" id="pone-0000722-g001"><div class="img"><a name="pone-0000722-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Characterization of the DNA-content increase in HU-treated cells.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>DNA content profiles by FACS analysis in HU-challenged wild type (A), <em>rad3Δ</em> (B), and <em>cds1Δ</em> (C) cells. 1C and 2C indicate cells containing 1 and 2 copies of genome, respectively. Minus and plus times indicate cells before and after HU-release, respectively. Regression analysis of DNA content increase in wild type (E) and <em>cds1Δ</em> (F) cells. <em>X</em>- and <em>Y</em>-axis indicate the time after HU-removal and the percent of DNA content (<em>i.e.</em>, median) increase, respectively.</p>
<span>doi:10.1371/journal.pone.0000722.g001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>DNA synthesis gradually resumed in wild type cells released after HU block, as judged by the increase in DNA content determined by FACS analysis (<a href="#pone-0000722-g001">Figure 1A</a>). Approximately 20% of the genome was synthesized in the first 30-min of the S-phase and the remaining 80% was completed in the second 30-min (<a href="#pone-0000722-g001">Figure 1A and 1D</a>), close to the time required for genome duplication in cells with undisturbed S-phase <a href="#pone.0000722-Heichinger1">[21]</a>, <a href="#pone.0000722-MacNeill1">[22]</a>. The time taken for the genome duplication was ~60 min in wild type cells released after HU-arrest (<a href="#pone-0000722-g001">Figure 1A</a>). In <em>rad3Δ</em> or <em>cds1Δ</em> cells, on the other hand, less than 20% of the genome was duplicated in the first 60-min after HU removal (<a href="#pone-0000722-g001">Figure 1B, 1C, and 1E</a>). Significantly, the time taken for the entire genome duplication was approximately 4 h in both <em>rad3Δ</em> and <em>cds1Δ</em> cells released after HU block. Given that HU-induced stalled forks collapse and fail to resume DNA replication in <em>rad3Δ</em> and <em>cds1Δ</em> cells after HU-release, this result suggests that post-HU initiation of DNA replication was likely to occur at unused origins.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>It is worth noting that ~20% of cells in the <em>rad3Δ</em> strain, but not <em>cds1Δ</em>, exhibited a cut phenotype (cells with premature assembly of division septa) at ~2 hr after release from HU treatment (Data not shown). Consistent with this, an additional population of cells containing incomplete genomes (less than 1C-DNA contents) appeared after HU removal in the <em>rad3Δ</em> strain (<a href="#pone-0000722-g001">Figure 1B</a>, see asterisk). This heterogeneity of DNA content impeded determination of DNA replication profiling in <em>rad3Δ</em> cells. Therefore, genome-wide replication profiling of wild type and <em>cds1Δ</em> cells released after HU-block were performed in this study, but profiling of <em>rad3Δ</em> cells was omitted.</p>


<h4>Genome-wide Profiling of DNA Replication Timing and Efficiency</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To investigate genome-wide DNA replication timing and efficiency, we applied the <em>S. pombe</em> genome-wide ORF-specific microarray <a href="#pone.0000722-Peng1">[23]</a> to determine the increase in DNA copy numbers at individual loci in cells released after HU block. The microarray has an average resolution of one locus/~2.4 Kb. Each locus (or ORF) was represented by two different 50-mer oligonucleotides whose average ratio was used for profiling. Cell samples were taken at 5-min intervals after HU removal for a period of 60 min. Genomic DNA extracted from cell samples was labeled with cyanine-dye Cy5 and subjected to DNA microarray hybridization. To reduce the dye-bias in the dual-color microarray analysis, we applied a common reference of genomic DNA (Cy3-labeled) extracted from cell samples at the 0-min time point. After the 0-min normalization, profiles would not be affected by the type of the common references used (<em>e.g.</em>, G1, G2 or asynchronous cells). To ensure reproducibility, two independent time-course experiments were carried out.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Upon HU treatment for up to 3 hr, very little DNA synthesis was detected in check point-proficient cells <a href="#pone.0000722-Feng1">[19]</a>. Assuming that a replication origin could fire at ~30% efficiency and accumulate DNA patches of ~5 Kb in length <a href="#pone.0000722-Patel1">[9]</a>, a cell containing ~500 origins would have synthesized ~750 Kb ( = 30%×5×500) or ~5% of the genome during the HU treatment. The vast majority (~95%) of the remaining genome would be synthesized in cells released after HU block. Therefore, the tiny amount of HU-induced DNA deposits at origins is unlikely to distort the global profiles of replication timing and efficiency after the smoothing process (see <a href="#s4">Materials and Methods</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>We applied the near-sigmoid model to fit DNA copy number increase (after 0-min normalization) as a function of time at individual loci for estimation of replication start timing <em>T<sub>0</sub></em> and end timings <em>T<sub>100</sub></em> (exemplified in <a href="#pone-0000722-g002">Figure 2</a>). To this end, the <em>T<sub>0</sub></em> and <em>T<sub>100</sub></em> at the vast majority of loci (&gt;96%) were obtained based on the threshold of FDR (false discovery rate) less than 0.01%. Approximately 190 loci (~4%) of the genome could not be fit to the near-sigmoid model (see <a href="#s4">Materials and Methods</a>). After smoothing with a moving window of 3 loci, genome-wide profiles of the average replication timing <em>T<sub>50</sub></em> and the duplication time <em>ΔT</em> (or replication efficiency <em>η<sub>r</sub></em>) were revealed (<a href="#pone-0000722-g002">Figure 2</a> and <a href="#pone-0000722-g003">3</a>; <a href="#pone.0000722.s001">Table S1</a>).</p>
<div class="figure" id="pone-0000722-g002"><div class="img"><a name="pone-0000722-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Schematic modeling of the near-sigmoid regression for profiling of DNA replication timing and efficiency.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Modeling of the near-sigmoid regression for the increase in DNA copy number at indicated individual loci (a–e). (B) Profile assembly of the copy number changes at individual loci of the subchromosomal region. (C) Simplified form of replication profiles displaying average replication timing <em>T<sub>50</sub></em> and duplication time <em>ΔT</em>. (D) Conversion between duplication time <em>ΔT</em> and replication efficiency <em>η</em>. (E) Potential distribution of the <em>T<sub>50</sub></em> versus <em>η</em> at individual loci of the genome.</p>
<span>doi:10.1371/journal.pone.0000722.g002</span></div><div class="figure" id="pone-0000722-g003"><div class="img"><a name="pone-0000722-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Global profiles of DNA replication timing and efficiency in wild type cells released from HU block.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>X- and Y-axis are times after HU removal and physical locations of individual loci of chromosome <em>I</em> (A), <em>II</em> (B), and <em>III</em> (C), respectively. Average replication timing <em>T<sub>50</sub></em> profiles are in think black lines. Efficiency is indicated by <em>T<sub>0</sub></em> and <em>T<sub>100</sub></em> profiles at individual loci in grey lines. Approximately 39 origins that have been validated by 2D-gel electrophoresis are marked with vertical lines. Predicted origins and AT-rich islands are indicated in blue and purple dots on the top of the profiles, respectively. Horizontal orange and green lines at the bottom of the profiles indicate efficient and inefficient subchromosomal regions, respectively. Solid red circles and triangles indicate centromeres and telomeres. No adjacent telomeres indicated in chromosome <em>III</em> due to the presence of multiple arrays of large rDNA sequences at the ends.</p>
<span>doi:10.1371/journal.pone.0000722.g003</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>We defined loci whose replication efficiency <em>η<sub>r</sub></em> was greater than the median <em>η<sub>r</sub></em><sub>-<em>median</em></sub> (or duplication time <em>ΔT</em> was less than the median <em>ΔT<sub>median</sub></em>) as efficient-replication loci or efficient loci (<a href="#pone-0000722-g002">Figure 2C</a>, loci <em>b, c,</em> and <em>e</em>). On the other hand, loci whose <em>η<sub>r</sub></em> was less than <em>η<sub>r-median</sub></em> were known as inefficient loci (<a href="#pone-0000722-g002">Figure 2C</a>, loci <em>a</em> and <em>d</em>). It is worth noting that inefficient loci would need to start replication early (through firing at origins and/or passive replication) in order to complete DNA duplication prior to the end of S-phase. This, the <em>T<sub>50</sub></em> at inefficient loci should have a limited range around half the length of the S-phase (<a href="#pone-0000722-g002">Figure 2D and 2E</a>). On the other hand, the <em>T<sub>50</sub></em> at efficient loci would have wide range from early to late in S-phase (<a href="#pone-0000722-g002">Figure 2E</a>). A volcano-like shape illustrates the potential distribution of average-replication timings for all loci with various replication efficiencies (<a href="#pone-0000722-g002">Figure 2E</a>).</p>


<h4>Average Replication Timing at Efficient Loci Appears to Be Late in S-phase Progression in Cells Released after HU-block</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The profiles of average replication timing <em>T<sub>50</sub></em> and efficiency <em>η</em> (<em>ΔT</em> was also included in <a href="#pone-0000722-g003">Figure 3</a>) revealed that the <em>T<sub>50</sub></em> at all loci of the genome ranged from ~25.8 to ~52.5 min with a median of ~39.2 min at various efficiencies (<a href="#pone-0000722-g004">Figure 4A</a>). The average replication timing <em>T<sub>50</sub></em> varied slightly in different chromosomes. The median of average replication timing <em>T<sub>50-median</sub></em> was ~39.2 and ~40 min in chromosome <em>I</em> and <em>II</em>, respectively. On the other hand, the <em>T<sub>50-median</sub></em> in chromosome <em>III</em> was ~36.7 min, &gt;2.5 min earlier than the other two chromosomes (<em>p</em>-value ~ 2.2×10<sup>−16</sup>) (<a href="#pone-0000722-g004">Figure 4A</a>). This result is consistent with the observation by Heichinger et al. <a href="#pone.0000722-Heichinger1">[21]</a>.</p>
<div class="figure" id="pone-0000722-g004"><div class="img"><a name="pone-0000722-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Characteristics of DNA replication timing <em>T<sub>50</sub></em> and efficiency <em>η</em> at various loci in wild type cells released after HU block.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Box plot indicates the non-outliers minimum (left end of the line), the lower quartile (left end of the rectangle), median (centre of the rectangle), the upper quartile (right end of the rectangle), and the non-outlier maximum (right end of the line). (A) The box-plots of <em>T<sub>50</sub></em> distribution of individual loci of the genome and individual chromosomes as indicated. The <em>T<sub>50-median</sub></em> of the loci of the genome or individual chromosomes is presented in parentheses. (B) The box-plots of <em>T<sub>50</sub></em> distribution of efficient and inefficient loci. Efficient and inefficient loci are those whose <em>η</em> is greater and less than the <em>η<sub>-median</sub></em>, respectively. The <em>T<sub>50-median</sub></em> of efficient and inefficient loci is indicated in parentheses. (C) Numbers of loci in the binned <em>ΔT</em> series or <em>η</em> series. (D) Plot of efficiency <em>η</em> versus average replication timing <em>T<sub>50</sub></em> of individual loci. Only ~0.3% (16 out of 4733) loci of the genome exhibited their efficiency <em>η</em> of greater than 1 (or duplicating the DNA in less than 10 min). The arrow indicates the asymmetric distribution of the <em>T<sub>50</sub></em>. (E–G) The <em>T<sub>50</sub></em> distribution of loci and the number of loci in the binned <em>η</em> series in chromosome <em>I</em> (E), <em>II</em> (F), and <em>III</em> (G). It is displayed as described in (B) and (C). (H) The plot of <em>T<sub>50</sub></em> versus <em>η</em> or <em>ΔT</em> of efficient and inefficient regions. <em>X</em>- and <em>Y</em>-axis indicate <em>T<sub>50</sub></em> and <em>η</em> or <em>ΔT</em>, respectively. The efficient and inefficient regions are indicated by closed and open circles, respectively.</p>
<span>doi:10.1371/journal.pone.0000722.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We next investigated replication efficiency <em>η<sub>r</sub></em> at various loci of the genome. We set the unit of the replication efficiency <em>η<sub>r</sub></em> as 0.1 min<sup>−1</sup>. Loci whose <em>η<sub>r</sub></em> was greater that the median <em>η<sub>r-median</sub></em> of ~0.316 (0.1-min<sup>−1</sup>) were designated as efficient-replication loci or efficient loci. Conversely, inefficient loci were those whose <em>η<sub>r</sub></em> was less than <em>η<sub>r-median</sub></em>. Surprisingly, the median <em>T<sub>50-median</sub></em> of the efficient loci (<em>i.e.</em>, ~41.7 min) was substantially later (or greater) than the <em>T<sub>50-median</sub></em> of the inefficient loci (<em>i.e.</em>, ~36.7 min), indicating that efficient replication tends to occur late in S-phase (<a href="#pone-0000722-g004">Figure 4B–D</a>; <a href="#pone-0000722-t001">Table 1</a>). The late efficient-replication was a common feature in all three chromosomes (<a href="#pone-0000722-g004">Figure 4E–G</a>). It is probably not surprising that cells would need to synthesize DNA more efficiently in unreplicated loci or gaps late in S-phase. Otherwise a delay in S-phase would be inevitable.</p>
<div class="figure" id="pone-0000722-t001"><div class="img"><a name="pone-0000722-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>The medians of replication/firing timing and efficiency at loci/origins</span></strong></p><span>doi:10.1371/journal.pone.0000722.t001</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>To investigate regulation of replication in subchromosomal regions, we searched for regions (<em>e.g.</em>, ~20 consecutive loci or more) that were enriched either efficient or inefficient loci (<em>p</em>-value&lt;0.05) in the genome. To this end, 23 and 14 subchromosomal regions were found to be efficiently and inefficiently replicated, respectively (<a href="#pone-0000722-g003">Figure 3</a>, see orange and green bars). Inefficient replication regions (5 out of 14) were found to be over-represented in chromosome <em>III</em> (<em>p</em>-value&lt;0.02) (<a href="#pone-0000722-g003">Figure 3C</a>), consistent with the observation that DNA replication is less efficient on chromosome <em>III</em> compared to the other two chromosomes. On the other hand, subtelomeric regions (3 out of 4) in chromosome <em>I</em> and <em>II</em> (chromosome <em>III</em> telomeres are separated from the arms by the multiple arrays of rDNA) were identified as efficient regions that replicated late (<a href="#pone-0000722-g003">Figure 3</a>), consistent with the report on late replication in subtelomeric regions <a href="#pone.0000722-Kim1">[8]</a>. The average replication timings <em>T<sub>50</sub></em> of the efficient-replication regions appeared to be remarkably later than the <em>T<sub>50</sub></em> of the inefficient regions (<em>p</em>-value&lt;10<sup>−15</sup>) (<a href="#pone-0000722-g004">Figure 4H</a>).</p>


<h4>Chromosomal Locations, Firing Timings and Efficiencies of Potential Replication Origins</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Peaks identified from the profiles of average replication timing <em>T<sub>50</sub></em> were commonly used to predict the location of replication origins (or origin clusters). Timing of firing at origins is approximated by the average replication timing of the peak loci. It has been noted that this approach has limitations in identifying (efficient) late-firing origins closely located to the other (efficient) early-firing origins <a href="#pone.0000722-MacAlpine1">[2]</a>, <a href="#pone.0000722-Raghuraman1">[4]</a>. As firing at origins is relatively inefficient in <em>S. pombe</em> <a href="#pone.0000722-Kim1">[8]</a>, <a href="#pone.0000722-Patel1">[9]</a>, <a href="#pone.0000722-Rhind1">[24]</a>, it would not only underestimate the number of efficient late-firing origins, but also would fail to identify most, if not all, inefficient late-firing origins. This is because inefficient late-firing origins are unlikely to be self-sufficient in replication of the origin DNA (see <a href="#s4">Material and Methods</a>). Nevertheless, this is still the most effective way to predict origins of replication at the genomic scale <a href="#pone.0000722-Heichinger1">[21]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The PeakFinder software <a href="#pone.0000722-Glynn1">[25]</a> was applied to identify the position of peaks or potential origins based on the profiles of average replication timing. To this end, ~516 peaks were identified that represent potential origins (or origin clusters) of replication in the genome (<a href="#pone-0000722-g003">Figure 3</a>, see blue dots on the top of the replication profiles). Of the 48 origins that have been tested by 2D-gel electrophoresis (see ref in Segurado), 39 (~81%) were found to overlap with the peaks identified in this study with a window of 12-Kb distance (<a href="#pone-0000722-t002">Table 2</a>), and the remaining 15 origins were found to match with peaks a bit below the threshold, suggesting that the peaks of the <em>T<sub>50</sub></em> profiles derived from the <em>T<sub>0</sub></em> and <em>T<sub>100</sub></em> are good approximations of potential origins in the genome.</p>
<div class="figure" id="pone-0000722-t002"><div class="img"><a name="pone-0000722-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>The comparisons of the peaks identified in <em>T<sub>50</sub></em> profiles with the ORI sites reported previously by others</span></strong></p><span>doi:10.1371/journal.pone.0000722.t002</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>We estimated the average firing timings <em>T<sub>50</sub></em> and the (maximum) firing efficiencies <em>η<sub>f</sub></em> at the predicted origins using the average replication timing <em>T<sub>50</sub></em> and replication efficiency <em>η<sub>r</sub></em> of the peak loci (<a href="#pone-0000722-g005">Figure 5A</a>). Origins whose firing efficiency <em>η<sub>f</sub></em> was greater than the median <em>η<sub>f-median</sub></em> of 0.316 (0.1 min<sup>−1</sup>) were designated as efficient origins (the <em>η<sub>f-median</sub></em> is different from the <em>η<sub>r-median</sub></em>) (<a href="#pone-0000722-t001">Table 1</a>). On the other hand, origins whose <em>η<sub>f</sub></em> was less than the <em>η<sub>f-median</sub></em> were classified as inefficient origins. Notably, most inefficient origins fired near the middle of the S-phase (<a href="#pone-0000722-g005">Figure 5B and 5C</a>). It is intriguing that most efficient origins appeared to fire late in S-phase in cells released after HU block (<a href="#pone-0000722-g005">Figure 5B and 5C</a>), given that the method applied was known to favor early-firing origins <a href="#pone.0000722-MacAlpine1">[2]</a>,</p>
<div class="figure" id="pone-0000722-g005"><div class="img"><a name="pone-0000722-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Characteristics of average firing timing <em>T<sub>50</sub></em> and (maximum) firing efficiency <em>η</em> at various predicted origins in wild type cells released after HU block.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) The box-plots of <em>T<sub>50</sub></em> distribution of predicted origins of the genome and individual chromosomes as indicated. The <em>T<sub>50-median</sub></em> of the origins of the genome or individual chromosomes is presented in parentheses. (B) The box-plots of <em>T<sub>50</sub></em> distribution of efficient and inefficient origins. Efficient and inefficient origins are those whose <em>η</em> is greater and less than the <em>η<sub>-median</sub></em>, respectively. The <em>T<sub>50-median</sub></em> of efficient and inefficient origins is indicated in parentheses. (C) Plot of firing efficiency <em>η</em> versus average firing timing <em>T<sub>50</sub></em> of individual origins. The arrow indicates the asymmetric distribution of the <em>T<sub>50</sub></em>. (D) The Venn diagram of predicted peaks and loci containing HU-induced ssDNA deposit. (E) The over-representation of ssDNA in predicated origins of chromosome <em>III</em>. Numbers of the predicated origins/peaks and those overlapped with the loci containing ssDNA deposit are indicated. The asterisk indicates the significant enrichment of ssDNA in chromosome <em>III</em>.</p>
<span>doi:10.1371/journal.pone.0000722.g005</span></div><a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Rad3p-Cds1p mediated checkpoint is known to restore stalled forks during HU-induced DNA replication block <a href="#pone.0000722-Patel1">[9]</a>, <a href="#pone.0000722-Sogo1">[15]</a>, <a href="#pone.0000722-Feng1">[19]</a>. To validate that the majority of the stalled forks resumed DNA replication in cells released from a HU-block, we first determined the sites of DNA synthesis in cells treated with 8-mM HU for 3 h (equivalent to the cells at 0-min time point). To achieve this, microarray analysis was performed using Cy5-labeled sample DNA prepared from HU-treated cells (<em>i.e.</em>, 0-min samples) against Cy3-labeled reference DNA from G1- or G2-phase cells. Self-hybridizations of genomic DNA from G1 or G2-phase cells were used as control. In either case, four independent microarray hybridizations were carried out for reproducibility. Loci containing extensive stretches of newly synthesized DNA, resulting from firing at origins in HU-treated cells, would exhibit an obvious increase in DNA copy number. SAM procedure <a href="#pone.0000722-Yang1">[26]</a> was applied to identify loci with significant amounts of nascent DNA by comparing the ratios of copy number at individual loci in HU-treated experiments with the self-hybridization control experiments. As a result, ~394 loci were found to have large amounts of newly synthesized DNA at the threshold of 10% FDR (<a href="#pone-0000722-g005">Figure 5D</a>). More than 80% of the loci containing nascent DNA after HU-release overlapped with the predicted origins derived from the peaks of the <em>T<sub>50</sub></em> profiles with a window of 12-Kb distance, indicating that the majority of the stalled forks are rendered competent for DNA replication in checkpoint-proficient cells released from a HU block. The HU-induced nascent ssDNA stretches were found to be over-represented on chromosome <em>III</em>, consistent with the observation of early replication/firing in chromosome <em>III</em> (<a href="#pone-0000722-g005">Figure 5E</a>, see asterisk).</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>To investigate whether the peaks derived from the <em>T<sub>50</sub></em> (<em>i.e.</em>, the average of <em>T<sub>0</sub></em> and <em>T<sub>100</sub></em>) profiles were similar to the origins predicted by other studies, we compared the 516 peaks with the reported origin sets based on an <em>in-silico</em> study <a href="#pone.0000722-Segurado1">[27]</a>, a chIP-chip analysis <a href="#pone.0000722-Hayashi1">[28]</a>, an HU-induced ssDNA enrichment <a href="#pone.0000722-Feng1">[19]</a>, and from replication profiling in unperturbed S-phase <a href="#pone.0000722-Heichinger1">[21]</a> (<a href="#pone.0000722.s002">Table S2</a>). The majority (~80%) of the origins in the previous reports overlapped with the 516 peaks, indicating an adequate prediction of origins using the near-sigmoid approach (<a href="#pone-0000722-t002">Table 2</a>).</p>


<h4>Late Efficient Replication Patterns Are Disrupted in HU-challenged <em>cds1Δ</em> Cells</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To investigate the role of Cds1p-mediated checkpoint function in regulation of global DNA replication, we performed DNA replication profiling in <em>cds1Δ</em> cells released from a HU block. To this end, HU-challenged <em>cds1Δ</em> cells were sampled at 15-min intervals for a period of 4 h. Microarray analysis of DNA copy number-increases in <em>cds1Δ</em> cells was performed using an identical method for the wild type (see above). Genome-wide profiles of DNA replication timing and efficiency in <em>cds1Δ</em> cells are shown in <a href="#pone-0000722-g006">Figure 6</a> (<a href="#pone.0000722.s003">Table S3</a>). It took ~240 min to complete DNA replication in <em>cds1Δ</em> cells released from a HU block, 4 times longer than HU-challenged wild type cells. The profiles in <em>cds1Δ</em> cells were disrupted from those in wild type cells, indicating that Cds1p function is involved in maintaining global regulation of DNA replication.</p>
<div class="figure" id="pone-0000722-g006"><div class="img"><a name="pone-0000722-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Global DNA replication profiles of HU-challenged <em>cds1Δ</em> cells.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The profiles are displayed as described in <a href="#pone-0000722-g003">Figure 3</a>.</p>
<span>doi:10.1371/journal.pone.0000722.g006</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>It was clear that the average replication timing <em>T<sub>50</sub></em> at various loci ranged from ~70 min to ~135 min with the median <em>T<sub>50-median</sub></em> of 100 min according to the replication profiles in HU-challenged <em>cds1Δ</em> cells (<a href="#pone-0000722-g007">Figure 7A</a>). The global profiles were totally disrupted in <em>cds1Δ</em> cells. However, the early replication of chromosome <em>III</em> (<em>i.e.</em>, <em>T<sub>50-median</sub></em> = ~97.5 min) compared to that of the other two chromosomes was not altered in <em>cds1Δ</em> cells (<a href="#pone-0000722-g007">Figure 7A</a>), suggesting that the early replication of chromosome <em>III</em> is independent on the Cds1p function. As DNA duplication in HU-challenged <em>cds1Δ</em> cells took ~4 times longer than that of wild type cells, the unit of replication efficiency <em>η<sub>r</sub></em> was thus defined as 1/40 (or 0.025) min<sup>−1</sup>, four-fold less efficient than wild type cells. The relative replication efficiency at various loci in <em>cds1Δ</em> cells was categorized based on the median <em>η<sub>r-median</sub></em> ~0.267 (0.025 min<sup>−1</sup>). The <em>T<sub>50-median</sub></em> of efficient loci in <em>cds1Δ</em> cells was very similar to that of inefficient loci (<a href="#pone-0000722-g007">Figure 7B–D</a>), indicating the late efficient-replication origins were most affected by the lack of Cds1p after release from a HU block (<a href="#pone-0000722-g004">Figure 4B–D</a>). This result suggests that the failure in resuming DNA replication at the forks that previously initiated DNA replication, would extend the duration of S-phase. Alternatively, it may imply that the checkpoint function is not only required for restoring stalled forks, but also involved in ensuring efficient replication late in S-phase.</p>
<div class="figure" id="pone-0000722-g007"><div class="img"><a name="pone-0000722-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Characteristics of DNA replication timing <em>T<sub>50</sub></em> and efficiency <em>η</em> at various loci in <em>cds1Δ</em> cells released after HU block.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>(A) The box-plots of <em>T<sub>50</sub></em> distribution of individual loci of the genome and individual chromosomes in <em>cds1Δ</em> cells as indicated. The <em>T<sub>50-median</sub></em> of the loci of the genome or individual chromosomes is presented in parentheses. (B) The box-plots of <em>T<sub>50</sub></em> distribution of efficient and inefficient loci in <em>cds1Δ</em> cells. (C) Numbers of loci in the binned <em>ΔT</em> series or <em>η</em> series. (D) Plot of efficiency <em>η</em> versus average replication timing <em>T<sub>50</sub></em> of individual loci in <em>cds1Δ</em> cells. Less than ~0.1% (3 out of 4787) loci of the genome exhibited their efficiency <em>η</em> of greater than 1 (or duplicating the DNA in less than 40 min). The arrow indicates the asymmetric distribution of the <em>T<sub>50</sub></em>. (E) The plot of <em>T<sub>50</sub></em> versus <em>η</em> or <em>ΔT</em> of efficient and inefficient regions in <em>cds1Δ</em> cells. <em>X</em>- and <em>Y</em>-axis indicate <em>T<sub>50</sub></em> and <em>η</em> or <em>ΔT</em>, respectively. The efficient and inefficient regions are indicated by closed and open circles, respectively.</p>
<span>doi:10.1371/journal.pone.0000722.g007</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>We subsequently searched for subchromosomal regions that were clustered for efficient and inefficient replication loci in <em>cds1Δ</em> cells using the identical approach in wild type. We identified 11 efficient and 10 inefficient regions. Inefficient replication regions were over-represented on chromosome <em>III</em>, similar to wild type cells. The average replication timing, <em>T<sub>50,</sub></em> of efficient regions was comparable to that of inefficient regions in <em>cds1Δ</em> cells (<a href="#pone-0000722-g007">Figure 7E</a>). The late-efficient replication pattern in wild type was clearly disrupted in <em>cds1Δ</em> cells, indicating that the involvement of Cds1p function in maintaining the efficient replication late in S-phase.</p>


<h4>Checkpoint Function Is More Apparent on Efficient Origins than Inefficient Ones in <em>cds1Δ</em> cells Released From a HU-block</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Chromosomal locations of origins were approximated by the peaks identified using the Peakfinder software <a href="#pone.0000722-Glynn1">[25]</a> based on the profiles of the average replication timing <em>T<sub>50</sub></em> in <em>cds1Δ</em> cells. A total of 598 peaks were identified from the profiles in HU-challenged <em>cds1Δ</em> cells: 269, 226, and 103 were found in chromosome <em>I, II</em>, and <em>III</em>, respectively (<a href="#pone-0000722-g008">Figure 8A–8C</a>). The number of origins identified in <em>cds1Δ</em> profiles was ~14% more than that found in wild type profiles.</p>
<div class="figure" id="pone-0000722-g008"><div class="img"><a name="pone-0000722-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000722" data-uri="info:doi/10.1371/journal.pone.0000722.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000722.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000722.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000722.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Characteristics of average firing timing <em>T<sub>50</sub></em> and (maximum) firing efficiency <em>η</em> at various predicted origins in <em>cds1Δ</em> cells released after HU block.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>(A) The box-plots of <em>T<sub>50</sub></em> distribution of predicted origins of the genome and individual chromosomes <em>cds1Δ</em> cells as indicated. The <em>T<sub>50-median</sub></em> of the origins of the genome or individual chromosomes is presented in parentheses. (B) The box-plots of <em>T<sub>50</sub></em> distribution of efficient and inefficient origins <em>cds1Δ</em> cells. The <em>T<sub>50-median</sub></em> of efficient and inefficient origins is indicated in parentheses. (C) Plot of firing efficiency <em>η</em> versus average firing timing <em>T<sub>50</sub></em> of individual origins <em>cds1Δ</em> cells. The arrow indicates the asymmetric distribution of the <em>T<sub>50</sub></em>. (D) Venn diagram of peaks predicted in wild type and <em>cds1Δ</em> cells. Overlapping peaks are those whose distance is less than 12 Kb. (E) The <em>T<sub>50</sub></em> distribution of the wild type-unique and overlapping origins in wild type cells. The asterisk indicates the significant increase in firing efficiency of the wild type-unique origins. (F) The <em>T<sub>50</sub></em> distribution of the cds1-unique and overlapping origins in <em>cds1Δ</em> cells.</p>
<span>doi:10.1371/journal.pone.0000722.g008</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>It has been shown that Cds1p-mediated checkpoint is required for restoring stalled forks in cells treated with HU <a href="#pone.0000722-Sogo1">[15]</a>. Stalled forks would collapse in <em>cds1Δ</em> cells and fail to resume replication. Assuming that the same subset of origins has fired and gave rise to nascent ssDNA stretches of ~5 Kb in size in either wild type or <em>cds1Δ</em> cells during HU treatment, we would expect that the stalled forks close to the origins would be able to resume replication in wild type cells but not in <em>cds1Δ</em> cells, representing a subset of the wild type-unique origins.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>To identify the wild type-unique origins, chromosomal location of origins in wild type cells was compared to that in <em>cds1Δ</em> cells using the threshold of 12 Kb-distance (see <a href="#s4">Materials and Methods</a>). Surprisingly, less than 10% origins (48 out of 516) were found to be wild type-unique (<a href="#pone-0000722-g008">Figure 8D</a>). The majority of the predicted origins were found to overlap between the wild type and <em>cds1Δ</em> cells with the window of 12-Kb distance (<a href="#pone-0000722-g008">Figure 8D</a>). It is very unlikely that the majority of origins would have been reserved (not fired) during HU treatment (for 3 h) and could fire in either strain after HU removal. Given that firing efficiency at origins in general is low in <em>S. pombe</em> <a href="#pone.0000722-Patel1">[9]</a>, it is possible that those overlapping origins represent a subset of inefficient origins.</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>To test whether overlapping origins were inefficient, we performed comparison analysis of the firing efficiencies between the two origin-subsets: the wild type-unique and the overlapping origins. As expected, the firing efficiency of the wild type-unique origins appeared to be higher than that of overlapping ones (p-value ~7.3×10<sup>−3</sup>), consistent with the notion that Cds1p checkpoint function is essential for restoring stalled forks induced by HU treatment (<a href="#pone-0000722-g008">Figure 8E</a>). The result indicates that checkpoint function has a greater effect on efficient origins compared to inefficient ones. A small subset of <em>cds1</em>-unique origins was found in <em>cds1Δ</em> cells (<a href="#pone-0000722-g008">Figure 8F</a>). These <em>cds1</em>-unique origins might represent the late inefficient origins which failed to be identified in wild type cells.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Genome-wide microarray analyses have been widely used to determine profiles of average DNA replication timing at the genomic scale <a href="#pone.0000722-Raghuraman1">[4]</a>–<a href="#pone.0000722-Jeon1">[7]</a>, <a href="#pone.0000722-Heichinger1">[21]</a>. However, direct estimation of replication efficiency at various loci of the genome based on the genome-wide replication profiles has not been performed previously. In this study, we demonstrate that replication efficiency, together with average replication timing, can be estimated using a novel approach-the near-sigmoid fitting for the increase in DNA copy number as a function of time at individual loci (see <a href="#s4">Materials and Methods</a>). The near-sigmoid model approach permits estimation of replication start timing <em>T<sub>0</sub></em> and replication end timing <em>T<sub>100</sub></em> at various loci of the genome. Based on the <em>T<sub>0</sub></em> and <em>T<sub>100</sub></em> at each locus, we attain the genome-wide profiles of average replication time <em>T<sub>50</sub></em> (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e001&amp;representation=PNG" class="inline-graphic"></span>) and replication efficiency <em>η</em> (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e002&amp;representation=PNG" class="inline-graphic"></span>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Genome-wide profiles of average replication timing <em>T<sub>50</sub></em> and replication efficiency <em>η</em> were determined in wild type and <em>cds1Δ</em> cells released from a HU block (<a href="#pone-0000722-g003">Figure 3</a> and <a href="#pone-0000722-g006">Figure 6</a>). DNA duplication takes about ~60 min in wild type cell and ~240 min <em>cds1Δ</em> cells after HU removal (<a href="#pone-0000722-g001">Figure 1</a>). The time taken to complete duplication at the majority of loci ranges from ~15.8 min to ~53.8 min with the median of 39.2 min in wild type cells and from ~37.5 min to ~157.5 min with the median of 150 min in <em>cds1Δ</em> cells. The relative rate of DNA copy number-increase is used to define replication efficiencies in wild type (<em>i.e.</em>, 1/10-min<sup>−1</sup>) and <em>cds1Δ</em> (<em>i.e.</em>, 1/40-min<sup>−1</sup>) cells. The medians of replication efficiency <em>η<sub>r-median</sub></em> of 0.255/10-min and <em>η<sub>r-median</sub></em> of 0.267/40-min are used to categorize efficient or inefficient loci in wild type and <em>cds1Δ</em> cells, respectively (<a href="#pone-0000722-t001">Table 1</a>). Significantly, efficient loci (whose <em>η</em>&gt;0.255/10-min) in wild type cells tend to be replicated late while inefficient ones (whose <em>η</em>&lt;0.255/10-min) early in S-phase (<a href="#pone-0000722-g004">Figure 4</a>). However, efficient loci (<em>η</em>&gt;0.267/40-min) in <em>cds1Δ</em> cells do not show late replication when compared to inefficient loci (<em>η</em>&lt;0.267/40-min) (<a href="#pone-0000722-g007">Figure 7</a>). It is not clear whether the disruption of the late-efficient replication in <em>cds1Δ</em> cells is solely due to the failure in resuming replication from the collapsed forks. Alternatively, Cds1p checkpoint kinase may play a role in regulation of late-efficient replication in cells after HU removal.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>It is intriguing to find that efficient replication occurs late in S-phase. To ensure completion of DNA duplication by the end of S-phase in an inefficient replication system such as <em>S. pombe</em> <a href="#pone.0000722-Patel1">[9]</a>, <a href="#pone.0000722-Dai1">[29]</a>, it is necessary to increase the efficiency of origins that fire late in S-phase. Disruption of the late efficient replication would lead to a delay in completion of duplication or S-phase. It is most likely that the extended S-phase in <em>cds1Δ</em> cells after HU removal is partly attributed to the disruption of the late efficient replication.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>To assess whether the HU-induced newly-replicated DNA would distort the profiles of replication timing and efficiency in cells released from a HU block, we performed microarray analysis to compare DNA extracted from HU-treated cells and G1 or G2 cells. Assuming that ~500 origins fire at an average efficiency of ~30% to produce newly-replicated DNA stretches of ~5 Kb in size <a href="#pone.0000722-Patel1">[9]</a>, ~5% of the genome would have been synthesized after HU treatment for 3 h. Approximately 394 loci are found to have newly-replicated DNA, out of which, 318 (80%) co-localize to the peaks of the <em>T<sub>50</sub></em> profiles with a window of 12-Kb distance. This indicates that the replication/resumption profiles generated based on the common reference of 0-min samples are a fair representation of the actual replication profiles.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Peaks identified from the global profiles of DNA replication timing <em>T<sub>50</sub></em> have been commonly used to estimate origins of replication <a href="#pone.0000722-MacAlpine1">[2]</a>, <a href="#pone.0000722-Raghuraman1">[4]</a>. We have adopted this approach to identify peaks from the <em>T<sub>50</sub></em> profiles in wild type and <em>cds1Δ</em> cells. A total of 516 and 598 peaks were identified in wild type and <em>cds1Δ</em> cells, respectively. Origins that initiated DNA replication prior to HU treatment could resume replication in wild type cells after HU removal. These origins would presumably be readily identifiable. However, forks resulting form origins that initiated DNA replication prior to HU treatment would fail to resume replication in <em>cds1Δ</em> cells due to collapse and therefore are unlikely to be identified. To investigate whether we identified different sets of origins in wild type and <em>cds1Δ</em> cells, we compared the actual locations of peaks in wild type and <em>cds1Δ</em> cells. Significantly, the majority of peaks overlapped with a window of 12-Kb distance between the two sets of origins (<a href="#pone-0000722-g008">Figure 8</a>). This may not be surprising in <em>S. pombe</em> in which firing is inefficient; and initiation of DNA replication from inefficient origins would not be effective during HU treatment. Therefore, after HU removal, the remaining unused origins could be competent to initiate DNA replication in both wild type and <em>cds1Δ</em> cells. This is further supported by the observation that origins found in wild type but not in <em>cds1Δ</em> cells are efficient origins (<a href="#pone-0000722-g008">Figure 8E</a>). These results are consistent with the notion that replication in <em>S. pombe</em> is generally considered inefficient <a href="#pone.0000722-Patel1">[9]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Heichinger <em>et al.</em>, has recently reported the profiling of average DNA replication timings using <em>cdc25-22</em> cells after temperature block-and-release <a href="#pone.0000722-Heichinger1">[21]</a>. The firing efficiency at various origins is estimated based on the fold-enrichment of HU-induced ssDNA. It was thus concluded that the early-firing origins are more efficient that the late-firing ones. The discrepancy may be explained by the fact that the late-firing origins are inefficient in cells undergoing unperturbed S-phase. However, in checkpoint activated cells, DNA replication from efficient origins are delayed.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>We present a first-of-its-kind study that permit direct assessment of genome-wide replication/firing efficiency at individual loci/origins. Opposed to well-defined, site-specific, and efficient (~90%) origins in budding yeast, origins of replication in <em>S. pombe</em> appear to be located preferentially at A+T-rich regions in the genome with an average firing efficiency of ~30% <a href="#pone.0000722-Patel1">[9]</a>, <a href="#pone.0000722-Segurado1">[27]</a>, <a href="#pone.0000722-Dai1">[29]</a>. Approximately 50% of the loci have shown to possess a low level (below the threshold) of HU-induced newly replicated DNA (Eshaghi and Liu, unpublished data), supporting the idea that about half of the genome (<em>i.e.</em>, all intergenic sequences) have regions that could potentially act as sites for initiation of DNA replication <a href="#pone.0000722-Dai1">[29]</a>. Our results support the stochastic model <a href="#pone.0000722-Rhind1">[24]</a>, <a href="#pone.0000722-Legouras1">[30]</a> in which random gaps are likely to be filled by the efficient late-firing origins (<a href="#pone-0000722-g005">Figure 5</a>).</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains and Culture Manipulations</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Strains YJL188 (<em>leu1-32 ura4-D18 h<sup>−</sup></em>), YJL1687 (<em>leu1-32 ura4-D18 cds1Δ::ura4<sup>+</sup> h<sup>−</sup></em>), YJL1715 (<em>leu1-32 ura4-D18 rad3Δ::ura4<sup>+</sup> h<sup>−</sup></em>), and medium YES <a href="#pone.0000722-Moreno1">[31]</a> were used in this study.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>To monitor DNA replication in hydroxyurea (HU) (Sigma-Aldrich Corp. St. Louis, MO)-challenged cells, ~800 ml of <em>S. pombe</em> culture in YES supplemented with 8mM HU were grown at 30°C in an orbital rotating shaker (New Brunswick Scientific Co., Inc., Edison, NJ) for 3 hrs. The HU-challenged cells were harvested by centrifugation (Thermo Scientific, Inc., Waltham, MA) at 5,000 rpm, 4°C for 1 min and subsequently washed twice with 4°C pre-chilled YES. The cells were resuspended to the original volume of YES medium. Samples were taken at 5-min intervals for a period of 60 min. For DNA content analysis, cell samples (~5 ml each) were spun down, resuspended in ice-cold 70% ethanol and stored at 4°C for subsequent analysis. For genomic DNA extraction and analysis of copy number, cell samples (~40 ml each) were spun down, quickly chilled in liquid nitrogen and stored at −80°C for further analysis.</p>


<h4>Fluorescence-associated Cell Sorting (FACS) Analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>To analyze DNA content, cells were fixed in ice-cold 70% ethanol spun down and resuspended in 50 mM sodium citrate buffer pH 7.0. To remove RNA, the cell suspension was treated with RNase A (Sigma) at a final concentration of 100 µg/ml and left at room temperature overnight. Cells were subsequently washed with 50 mM sodium citrate buffer and stained with propidium iodine (Sigma) at a final concentration of 8 µg/ml. Fluorescence intensities of individual cells were measured by BD FACScan flow cytometry (BD Biosciences, Franklin Lakes, NJ).</p>


<h4>Genomic DNA preparation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The frozen cell samples were thawed and washed with SE buffer (1.2 M sorbitol/0.1 M EDTA pH 8). The cells were subsequently resuspended in 1 ml SE buffer supplemented with 2 µg/ml Zymolyase 100T (ICN Biomedicals, Inc., Costa Mesa, CA) and incubated at 37°C for ~15 min. The spheroplasts were centrifuged at 2,000 rpm for 5 min and the supernatant was decanted. The spheroplasts were resuspended in 1 ml TNE buffer (50 mM Tris-HCl/100 mM NaCl/50 mM EDTA pH8) supplemented with 0.5% SDS and 1 µg/ml protease K (Roche, Basel, Switzerland) and incubated at 65°C for 30 min. To precipitate proteins, the cell lysates were first chilled on ice, mixed with pre-cooled 100 µl of 5 M Potassium Acetate, and incubated on ice for 30 min. Proteins were removed by centrifuging at 14,000 rpm for 15 min. The supernatant containing DNA was transferred into fresh tubes. The DNA was precipitated by the addition of equal volume of isopropanol and incubated at room temperature for 30 min. DNA was centrifuged and washed with 70% cold ethanol. The DNA pellet was subsequently resuspended in TE buffer supplemented with 0.1 µg/ml RNase A (Roche) and incubated at room temperature for 1 hr. DNA was extracted by phenol/chloroform/isoamyl alcohol (25:14:1) (Fluka Chemical Corp, Milwaukee, WI) extraction and precipitated by ethanol or isopropanol. The precipitated DNA was resuspended in TE buffer and fragmented by sonication (Branson, Danbury, CT) at 20% strength using three 20 sec bursts.</p>


<h4>Microarray Manufacture and Hybridization</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To manufacture ORF-specific DNA microarrays, approximately 10,000 oligonucleotides (50-mer in length) that represented 4,929 ORFs (one ORF is represented by two oligomers) in the <em>S. pombe</em> genome <a href="#pone.0000722-Wood1">[32]</a> were synthesized (Proligo, Hamburg, Germany). The oligonucleotides were resuspended in 0.3× SSC buffer and spotted onto poly-lysine-coated glass slides by using an arrayer machine (GeneMachines, San Carlos, CA). Spotted glass slides were processed according to DeRisi's protocol (<a href="http://derisilab.ucsf.edu">http://derisilab.ucsf.edu</a>).</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Sheared DNA was labeled using a random-priming protocol with amino-allyl-dUTP (aa-dUTP) using the BioPrime DNA Labeling kit (Invitrogen Corporation, Carlsbad, CA) according the manufacture's instruction. Labeled DNA was extracted with phenol/chloroform/isoamyl alcohol and precipitated with ethanol or isopropanol. The labeled DNA was then coupled with cyanine dyes Cy5 (Amersham, Buckinghamshire, UK) (e.g. DNA samples at various time points after HU removal) or Cy3 (e.g. DNA samples at the 0-min time point as common reference) according to a standard protocol. Cy5 and Cy3 labeled DNA were co-hybridized to microarrays.</p>


<h4>Microarray Data Acquisition and LOWESS Normalization</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Microarray slides were scanned using a GenePix scanner (Axon Instruments, Union City, CA) that was controlled by the software GenePix Pro4 (Axon Instruments). GenePix Results files were generated and globally normalized based on a median of ratios. To ensure accuracy of the measured data, the ratio of a feature was collected if its intensity in either channel was two-fold or greater than that of the background. To eliminate intensity-dependent dye bias, individual microarrays were further normalized using locally weighted linear regression and smooth scatter plot (LOWESS) <a href="#pone.0000722-Yang1">[26]</a>.</p>


<h4>Genome-wide Profiling of DNA Replication Timing and Efficiency</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>All steps in the process (including DNA-content-based microarray data normalization, regression of near-sigmoidal models to estimate replication initiation timing, completion timing and prediction of peaks using PeakFinder) were performed as follows.</p>
<h5>DNA-content-based data normalization.</h5><a id="article1.body1.sec4.sec6.sec1.p1" name="article1.body1.sec4.sec6.sec1.p1"></a><p>To determine DNA copy number increase at various loci, individual microarrays were normalized based on the DNA content at respective time points as described elsewhere <a href="#pone.0000722-Yabuki1">[6]</a>. In brief, the DNA content profiles were obtained through FACS analysis in two independent time series experiments. Each series was first normalized to 0-mean unit-variance to eliminate series dependent variation of mean and variance. Subsequently the logistic regression was carried out using SigmaPlot 8.0 software (Systat Software Inc., San Jose, CA) in which we fitted a 4-parameter near-sigmoid curve shown (<a href="#pone-0000722-g001">Figure 1D and 1E</a>) by equation (1) for the two normalized DNA content series together by estimating the parameters using least squares method.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>Where <em>f(t)</em> is the estimated DNA-content at time “<em>t</em>”. <em>a</em> is the scale parameter; <em>b</em> is the rate of DNA content increase; <em>t</em><sub>0</sub> is the replication half-completion timing; and <em>y</em><sub>0</sub> is the initial DNA content. Considering that the initial DNA content and the amount of increase of DNA after replication completion is from 1 copy to 2 copies of genome, we therefore simplified the equation (1) as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Where <em>y</em><sub>0</sub> = 1 and <em>a</em> = 1 in (1). The median of ratios of individual microarrays was adjusted to be the logarithm of the estimated DNA content at respective time points from equation (2). That is,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>Where <em>D<sub>it</sub></em> and <em>C<sub>it</sub></em> are ratios of locus <em>i</em> at time <em>t</em> before and after the normalization by equation (3), respectively. Normalized datasets were applied for further analysis of replication timing and efficiency.</p>

<h5>Regression analysis using the near-sigmoidal model.</h5><a id="article1.body1.sec4.sec6.sec2.p1" name="article1.body1.sec4.sec6.sec2.p1"></a><p>To estimate replication initiation and completion timings, the increase of copy number at individual loci, as a function of time, was subjected to a regression fitting for a 3-piece linear or near-sigmoidal model as an approximation of the sigmoidal. Before regression analysis, the datasets was first 0-mean and unit-variance normalized. Subsequently, missing values were filled by averaging neighbor loci log-ratios for regression analysis. In brief, each locus' log-ratio was fitted to a near-sigmoidal model as in equation (4) using the least square method. <em>T<sub>0</sub></em> and <em>T<sub>100</sub></em> indicate replication initiation timing and completion timing, respectively. Thus, <em>T<sub>50</sub></em> is the average of <em>T<sub>0</sub></em> and <em>T<sub>100</sub></em>, <em>i.e.</em>, (<em>T<sub>0</sub></em>+<em>T<sub>100</sub></em>)/2. The duplication time <em>ΔT</em> = <em>T<sub>100</sub></em>-<em>T<sub>0</sub></em> is a measure of efficiency for DNA replication. That is, the greater the duplication time <em>ΔT</em> is, the less efficient the DNA replication process is at the locus.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>Where <em>M<sub>it</sub></em> is the model of log-ratio of locus <em>i</em> at time <em>t</em>. All other estimated parameters are for the locus <em>i</em> in the notations. <em>T<sup>i</sup></em><sub>0</sub> and <em>T<sup>i</sup></em><sub>100</sub> are replication initiation timing and completion timing, respectively. <em>E<sub>iL</sub></em> and <em>E<sub>iU</sub></em> are the lower and upper limits of replication, respectively. <em>K<sub>L</sub></em> and <em>K<sub>U</sub></em> are the number of observations for locus <em>i</em> up to <em>T</em><sub>0</sub> and after <em>T</em><sub>100</sub>, respectively. The values are estimated by exhaustive search to minimize the <em>MSE</em> (<em>m</em>ean-<em>s</em>quare <em>e</em>rror) between <em>M<sub>it</sub></em> and log(<em>C<sub>it</sub></em>), <em>i.e.</em>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e007&amp;representation=PNG" class="inline-graphic"></span> where <em>K</em> is the total number of observations for locus <em>i</em> at all time points together. The p-value of the fit was calculated using an ANOVA table. To eliminate errors, the estimates of <em>T<sub>50</sub></em> and <em>ΔT</em> were smoothed using a moving average of 3 loci.</p>

<h5>Prediction of peaks.</h5><a id="article1.body1.sec4.sec6.sec3.p1" name="article1.body1.sec4.sec6.sec3.p1"></a><p>To predict peaks or potential origins, we applied PeakFinder <a href="#pone.0000722-Glynn1">[25]</a> to identify peaks based on the profiles of replication half-completion timing <em>T<sub>50</sub></em>. We set the parameters of the software as Gaussian Smoothing, Raw data, N = 1, round = 1, Left Delta = Right Delta = 0. The efficiency of origins was approximated using the efficiency of loci at the origins or closest to the origins.</p>

<h5>Identification of efficient and inefficient replication regions.</h5><a id="article1.body1.sec4.sec6.sec4.p1" name="article1.body1.sec4.sec6.sec4.p1"></a><p>To identify efficient and inefficient regions, we applied a sliding window to score all the loci by the following equation: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000722.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where <em>m</em> is the median of <em>ΔT</em> at all loci; and <em>w</em> is the half window size, so we defined the efficient region in which all loci had a score that was more than the certain threshold <em>α</em>; and the inefficient region in which all loci had a score that was less than −<em>α</em>. In this study, we set w = 5, <em>α</em> = 4, and the minimal number of loci in a region is 19. The median of <em>ΔT</em> at all loci is ~31.6 min in wild type cells and ~150 min in <em>cds1Δ</em> cells.</p>



<h4>Statistical Analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>A standard binomial test was employed to examine the significant enrichment, and the Wilcoxon rank test was employed to examine the differences between two groups with unknown distribution at various significance levels.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>The completed microarray datasets have been submitted to the GEO Databases (the accession number is GSE6977).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000722"></div><a name="pone.0000722.s001" id="pone.0000722.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000722.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Profiles of replication timing and efficiency in wild type cells</p>
<p class="siDoi">doi:10.1371/journal.pone.0000722.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.33 MB DOC)</p>
<a name="pone.0000722.s002" id="pone.0000722.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000722.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Comparisons of peaks/ORI between this study and the studies previously published by others</p>
<p class="siDoi">doi:10.1371/journal.pone.0000722.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000722.s003" id="pone.0000722.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000722.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Profiles of replication timing and efficiency in cds1 cells</p>
<p class="siDoi">doi:10.1371/journal.pone.0000722.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.29 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank L. D. Miller and X. Peng for manufacturing <em>S. pombe</em> DNA microarrays. The authors appreciate C.-Y. Wong, P. Robson, V.M. D'Souza, and T. Lufkin for critical reading and discussion of the manuscript. The author would also like to thank the anonymous reviewers for critical comments that greatly improved the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: J Liu. Performed the experiments: ME. Analyzed the data: RK J Li J Liu ME. Contributed reagents/materials/analysis tools: ZC. Wrote the paper: J Liu. Other: Involved in the design of the S. pombe DNA microarray: EL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000722-Gilbert1" id="pone.0000722-Gilbert1"></a>Gilbert DM (2001) Nuclear position leaves its mark on replication timing. J Cell Biol  152: F11–15.  <ul class="find" data-citedArticleID="975974" data-doi="10.1083/jcb.152.2.f11"><li><a href="http://dx.doi.org/10.1083/jcb.152.2.f11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+position+leaves+its+mark+on+replication+timing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+position+leaves+its+mark+on+replication+timing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000722-MacAlpine1" id="pone.0000722-MacAlpine1"></a>MacAlpine DM, Bell SP (2005) A genomic view of eukaryotic DNA replication. Chromosome Res  13: 309–326.  <ul class="find" data-citedArticleID="975990" data-doi="10.1007/s10577-005-1508-1"><li><a href="http://dx.doi.org/10.1007/s10577-005-1508-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+view+of+eukaryotic+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+view+of+eukaryotic+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000722-Bell1" id="pone.0000722-Bell1"></a>Bell SP, Dutta A (2002) DNA replication in eukaryotic cells. Annu Rev Biochem  71: 333–374.  <ul class="find" data-citedArticleID="975962" data-doi="10.1146/annurev.biochem.71.110601.135425"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.71.110601.135425" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+in+eukaryotic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+in+eukaryotic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000722-Raghuraman1" id="pone.0000722-Raghuraman1"></a>Raghuraman MK, Winzeler EA, Collingwood D, Hunt S, Wodicka L, et al.  (2001) Replication dynamics of the yeast genome. Science  294: 115–121.  <ul class="find" data-citedArticleID="976002" data-doi="10.1126/science.294.5540.115"><li><a href="http://dx.doi.org/10.1126/science.294.5540.115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+dynamics+of+the+yeast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+dynamics+of+the+yeast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000722-Schubeler1" id="pone.0000722-Schubeler1"></a>Schubeler D, Scalzo D, Kooperberg C, van Steensel B, Delrow J, et al.  (2002) Genome-wide DNA replication profile for Drosophila melanogaster: a link between transcription and replication timing. Nat Genet  32: 438–442.  <ul class="find" data-citedArticleID="976008" data-doi="10.1038/ng1005"><li><a href="http://dx.doi.org/10.1038/ng1005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+DNA+replication+profile+for+Drosophila+melanogaster%3A+a+link+between+transcription+and+replication+timing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+DNA+replication+profile+for+Drosophila+melanogaster%3A+a+link+between+transcription+and+replication+timing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000722-Yabuki1" id="pone.0000722-Yabuki1"></a>Yabuki N, Terashima H, Kitada K (2002) Mapping of early firing origins on a replication profile of budding yeast. Genes Cells  7: 781–789.  <ul class="find" data-citedArticleID="976020" data-doi="10.1046/j.1365-2443.2002.00559.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2443.2002.00559.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+early+firing+origins+on+a+replication+profile+of+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+early+firing+origins+on+a+replication+profile+of+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000722-Jeon1" id="pone.0000722-Jeon1"></a>Jeon Y, Bekiranov S, Karnani N, Kapranov P, Ghosh S, et al.  (2005) Temporal profile of replication of human chromosomes. Proc Natl Acad Sci U S A  102: 6419–6424.  <ul class="find" data-citedArticleID="975982" data-doi="10.1073/pnas.0405088102"><li><a href="http://dx.doi.org/10.1073/pnas.0405088102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal+profile+of+replication+of+human+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal+profile+of+replication+of+human+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000722-Kim1" id="pone.0000722-Kim1"></a>Kim SM, Huberman JA (2001) Regulation of replication timing in fission yeast. Embo J  20: 6115–6126.  <ul class="find" data-citedArticleID="975984" data-doi="10.1093/emboj/20.21.6115"><li><a href="http://dx.doi.org/10.1093/emboj/20.21.6115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+replication+timing+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+replication+timing+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000722-Patel1" id="pone.0000722-Patel1"></a>Patel PK, Arcangioli B, Baker SP, Bensimon A, Rhind N (2006) DNA replication origins fire stochastically in fission yeast. Mol Biol Cell  17: 308–316.  <ul class="find" data-citedArticleID="975998" data-doi="10.1091/mbc.e05-07-0657"><li><a href="http://dx.doi.org/10.1091/mbc.e05-07-0657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+origins+fire+stochastically+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+origins+fire+stochastically+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000722-Enoch1" id="pone.0000722-Enoch1"></a>Enoch T, Carr AM, Nurse P (1992) Fission yeast genes involved in coupling mitosis to completion of DNA replication. Genes Dev  6: 2035–2046.  <ul class="find" data-citedArticleID="975970" data-doi="10.1101/gad.6.11.2035"><li><a href="http://dx.doi.org/10.1101/gad.6.11.2035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fission+yeast+genes+involved+in+coupling+mitosis+to+completion+of+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fission+yeast+genes+involved+in+coupling+mitosis+to+completion+of+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000722-Seaton1" id="pone.0000722-Seaton1"></a>Seaton BL, Yucel J, Sunnerhagen P, Subramani S (1992) Isolation and characterization of the Schizosaccharomyces pombe rad3 gene, involved in the DNA damage and DNA synthesis checkpoints. Gene  119: 83–89.  <ul class="find" data-citedArticleID="976010" data-doi="10.1016/0378-1119(92)90069-2"><li><a href="http://dx.doi.org/10.1016/0378-1119(92)90069-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+the+Schizosaccharomyces+pombe+rad3+gene%2C+involved+in+the+DNA+damage+and+DNA+synthesis+checkpoints." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+the+Schizosaccharomyces+pombe+rad3+gene%2C+involved+in+the+DNA+damage+and+DNA+synthesis+checkpoints.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000722-alKhodairy1" id="pone.0000722-alKhodairy1"></a>al-Khodairy F, Fotou E, Sheldrick KS, Griffiths DJ, Lehmann AR, et al.  (1994) Identification and characterization of new elements involved in checkpoint and feedback controls in fission yeast. Mol Biol Cell  5: 147–160.  <ul class="find" data-citedArticleID="976024" data-doi="10.1091/mbc.5.2.147"><li><a href="http://dx.doi.org/10.1091/mbc.5.2.147" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+new+elements+involved+in+checkpoint+and+feedback+controls+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+new+elements+involved+in+checkpoint+and+feedback+controls+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000722-Murakami1" id="pone.0000722-Murakami1"></a>Murakami H, Okayama H (1995) A kinase from fission yeast responsible for blocking mitosis in S phase. Nature  374: 817–819.  <ul class="find" data-citedArticleID="975996" data-doi="10.1038/374817a0"><li><a href="http://dx.doi.org/10.1038/374817a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+kinase+from+fission+yeast+responsible+for+blocking+mitosis+in+S+phase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+kinase+from+fission+yeast+responsible+for+blocking+mitosis+in+S+phase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000722-Lindsay1" id="pone.0000722-Lindsay1"></a>Lindsay HD, Griffiths DJ, Edwards RJ, Christensen PU, Murray JM, et al.  (1998) S-phase-specific activation of Cds1 kinase defines a subpathway of the checkpoint response in Schizosaccharomyces pombe. Genes Dev  12: 382–395.  <ul class="find" data-citedArticleID="975988" data-doi="10.1101/gad.12.3.382"><li><a href="http://dx.doi.org/10.1101/gad.12.3.382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=S-phase-specific+activation+of+Cds1+kinase+defines+a+subpathway+of+the+checkpoint+response+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22S-phase-specific+activation+of+Cds1+kinase+defines+a+subpathway+of+the+checkpoint+response+in+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000722-Sogo1" id="pone.0000722-Sogo1"></a>Sogo JM, Lopes M, Foiani M (2002) Fork reversal and ssDNA accumulation at stalled replication forks owing to checkpoint defects. Science  297: 599–602.  <ul class="find" data-citedArticleID="976016" data-doi="10.1126/science.1074023"><li><a href="http://dx.doi.org/10.1126/science.1074023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fork+reversal+and+ssDNA+accumulation+at+stalled+replication+forks+owing+to+checkpoint+defects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fork+reversal+and+ssDNA+accumulation+at+stalled+replication+forks+owing+to+checkpoint+defects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000722-Boddy1" id="pone.0000722-Boddy1"></a>Boddy MN, Furnari B, Mondesert O, Russell P (1998) Replication checkpoint enforced by kinases Cds1 and Chk1. Science  280: 909–912.  <ul class="find" data-citedArticleID="975964" data-doi="10.1126/science.280.5365.909"><li><a href="http://dx.doi.org/10.1126/science.280.5365.909" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+checkpoint+enforced+by+kinases+Cds1+and+Chk1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+checkpoint+enforced+by+kinases+Cds1+and+Chk1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000722-Eklund1" id="pone.0000722-Eklund1"></a>Eklund H, Uhlin U, Farnegardh M, Logan DT, Nordlund P (2001) Structure and function of the radical enzyme ribonucleotide reductase. Prog Biophys Mol Biol  77: 177–268.  <ul class="find" data-citedArticleID="975968" data-doi="10.1016/s0079-6107(01)00014-1"><li><a href="http://dx.doi.org/10.1016/s0079-6107(01)00014-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+the+radical+enzyme+ribonucleotide+reductase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+the+radical+enzyme+ribonucleotide+reductase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000722-Shirahige1" id="pone.0000722-Shirahige1"></a>Shirahige K, Hori Y, Shiraishi K, Yamashita M, Takahashi K, et al.  (1998) Regulation of DNA-replication origins during cell-cycle progression. Nature  395: 618–621.  <ul class="find" data-citedArticleID="976014" data-doi="10.1038/27007"><li><a href="http://dx.doi.org/10.1038/27007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+DNA-replication+origins+during+cell-cycle+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+DNA-replication+origins+during+cell-cycle+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000722-Feng1" id="pone.0000722-Feng1"></a>Feng W, Collingwood D, Boeck ME, Fox LA, Alvino GM, et al.  (2006) Genomic mapping of single-stranded DNA in hydroxyurea-challenged yeasts identifies origins of replication. Nat Cell Biol  8: 148–155.  <ul class="find" data-citedArticleID="975972" data-doi="10.1038/ncb1358"><li><a href="http://dx.doi.org/10.1038/ncb1358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+mapping+of+single-stranded+DNA+in+hydroxyurea-challenged+yeasts+identifies+origins+of+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+mapping+of+single-stranded+DNA+in+hydroxyurea-challenged+yeasts+identifies+origins+of+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000722-Santocanale1" id="pone.0000722-Santocanale1"></a>Santocanale C, Diffley JF (1998) A Mec1- and Rad53-dependent checkpoint controls late-firing origins of DNA replication. Nature  395: 615–618.  <ul class="find" data-citedArticleID="976006"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+Mec1-+and+Rad53-dependent+checkpoint+controls+late-firing+origins+of+DNA+replication.&amp;auth=&amp;atitle=A+Mec1-+and+Rad53-dependent+checkpoint+controls+late-firing+origins+of+DNA+replication." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Mec1-+and+Rad53-dependent+checkpoint+controls+late-firing+origins+of+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Mec1-+and+Rad53-dependent+checkpoint+controls+late-firing+origins+of+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000722-Heichinger1" id="pone.0000722-Heichinger1"></a>Heichinger C, Penkett CJ, Bahler J, Nurse P (2006) Genome-wide characterization of fission yeast DNA replication origins. Embo J  25: 5171–5179.  <ul class="find" data-citedArticleID="975980" data-doi="10.1038/sj.emboj.7601390"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601390" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+characterization+of+fission+yeast+DNA+replication+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+characterization+of+fission+yeast+DNA+replication+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000722-MacNeill1" id="pone.0000722-MacNeill1"></a>MacNeill SA, Fantes PA (1997) Genetic and physiological analysis of DNA replication in fission yeast. Methods Enzymol  283: 440–459.  <ul class="find" data-citedArticleID="975992" data-doi="10.1016/s0076-6879(97)83036-4"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)83036-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+and+physiological+analysis+of+DNA+replication+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+and+physiological+analysis+of+DNA+replication+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000722-Peng1" id="pone.0000722-Peng1"></a>Peng X, Karuturi RK, Miller LD, Lin K, Jia Y, et al.  (2005) Identification of cell cycle-regulated genes in fission yeast. Mol Biol Cell  16: 1026–1042.  <ul class="find" data-citedArticleID="976000" data-doi="10.1091/mbc.e04-04-0299"><li><a href="http://dx.doi.org/10.1091/mbc.e04-04-0299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+cell+cycle-regulated+genes+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+cell+cycle-regulated+genes+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000722-Rhind1" id="pone.0000722-Rhind1"></a>Rhind N (2006) DNA replication timing: random thoughts about origin firing. Nat Cell Biol  8: 1313–1316.  <ul class="find" data-citedArticleID="976004" data-doi="10.1038/ncb1206-1313"><li><a href="http://dx.doi.org/10.1038/ncb1206-1313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+timing%3A+random+thoughts+about+origin+firing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+timing%3A+random+thoughts+about+origin+firing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000722-Glynn1" id="pone.0000722-Glynn1"></a>Glynn EF, Megee PC, Yu HG, Mistrot C, Unal E, et al.  (2004) Genome-wide mapping of the cohesin complex in the yeast Saccharomyces cerevisiae. PLoS Biol  2: E259.  <ul class="find" data-citedArticleID="975976" data-doi="10.1371/journal.pbio.0020259"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+mapping+of+the+cohesin+complex+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+mapping+of+the+cohesin+complex+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000722-Yang1" id="pone.0000722-Yang1"></a>Yang YH, Dudoit S, Luu P, Lin DM, Peng V, et al.  (2002) Normalization for cDNA microarray data: a robust composite method addressing single and multiple slide systematic variation. Nucleic Acids Res  30: e15.  <ul class="find" data-citedArticleID="976022" data-doi="10.1093/nar/30.4.e15"><li><a href="http://dx.doi.org/10.1093/nar/30.4.e15" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Normalization+for+cDNA+microarray+data%3A+a+robust+composite+method+addressing+single+and+multiple+slide+systematic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Normalization+for+cDNA+microarray+data%3A+a+robust+composite+method+addressing+single+and+multiple+slide+systematic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000722-Segurado1" id="pone.0000722-Segurado1"></a>Segurado M, de Luis A, Antequera F (2003) Genome-wide distribution of DNA replication origins at A+T-rich islands in Schizosaccharomyces pombe. EMBO Rep  4: 1048–1053.  <ul class="find" data-citedArticleID="976012" data-doi="10.1038/sj.embor.7400008"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+distribution+of+DNA+replication+origins+at+A%2BT-rich+islands+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+distribution+of+DNA+replication+origins+at+A%2BT-rich+islands+in+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000722-Hayashi1" id="pone.0000722-Hayashi1"></a>Hayashi M, Katou Y, Itoh T, Tazumi M, Yamada Y, et al.  (2007) Genome-wide localization of pre-RC sites and identification of replication origins in fission yeast. Embo J  26: 1327–1339.  <ul class="find" data-citedArticleID="975978" data-doi="10.1038/sj.emboj.7601585"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601585" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+localization+of+pre-RC+sites+and+identification+of+replication+origins+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+localization+of+pre-RC+sites+and+identification+of+replication+origins+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000722-Dai1" id="pone.0000722-Dai1"></a>Dai J, Chuang RY, Kelly TJ (2005) DNA replication origins in the Schizosaccharomyces pombe genome. Proc Natl Acad Sci U S A  102: 337–342.  <ul class="find" data-citedArticleID="975966" data-doi="10.1073/pnas.0408811102"><li><a href="http://dx.doi.org/10.1073/pnas.0408811102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+origins+in+the+Schizosaccharomyces+pombe+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+origins+in+the+Schizosaccharomyces+pombe+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000722-Legouras1" id="pone.0000722-Legouras1"></a>Legouras I, Xouri G, Dimopoulos S, Lygeros J, Lygerou Z (2006) DNA replication in the fission yeast: robustness in the face of uncertainty. Yeast  23: 951–962.  <ul class="find" data-citedArticleID="975986"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+replication+in+the+fission+yeast%3A+robustness+in+the+face+of+uncertainty.&amp;auth=&amp;atitle=DNA+replication+in+the+fission+yeast%3A+robustness+in+the+face+of+uncertainty." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+in+the+fission+yeast%3A+robustness+in+the+face+of+uncertainty." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+in+the+fission+yeast%3A+robustness+in+the+face+of+uncertainty.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000722-Moreno1" id="pone.0000722-Moreno1"></a>Moreno S, Klar A, Nurse P (1991) Molecular genetic analysis of fission yeast Schizosaccharomyces pombe. Methods Enzymol  194: 795–823.  <ul class="find" data-citedArticleID="975994" data-doi="10.1016/0076-6879(91)94059-l"><li><a href="http://dx.doi.org/10.1016/0076-6879(91)94059-l" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetic+analysis+of+fission+yeast+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetic+analysis+of+fission+yeast+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000722-Wood1" id="pone.0000722-Wood1"></a>Wood V, Gwilliam R, Rajandream MA, Lyne M, Lyne R, et al.  (2002) The genome sequence of Schizosaccharomyces pombe. Nature  415: 871–880.  <ul class="find" data-citedArticleID="976018" data-doi="10.1038/nature724"><li><a href="http://dx.doi.org/10.1038/nature724" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.XML" value="110385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.PDF" value="811541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g001.PNG_L" value="823899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g001.PNG_M" value="190649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g001.PNG_S" value="17169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g001.TIF" value="1449488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g001.PNG_I" value="102278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g002.PNG_L" value="679905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g002.PNG_M" value="111993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g002.PNG_S" value="18501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g002.TIF" value="1441204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g002.PNG_I" value="122738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g003.PNG_L" value="2428104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g003.PNG_M" value="207503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g003.PNG_S" value="18904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g003.TIF" value="5539498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g003.PNG_I" value="158065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g004.PNG_L" value="293415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g004.PNG_M" value="56808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g004.PNG_S" value="11282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g004.TIF" value="511248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g004.PNG_I" value="45419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t001.PNG_L" value="87771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t001.PNG_M" value="117260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t001.PNG_S" value="15331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t001.TIF" value="334608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t001.PNG_I" value="41306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t002.PNG_L" value="98612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t002.PNG_M" value="54592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t002.PNG_S" value="8684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t002.TIF" value="395014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.t002.PNG_I" value="16291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g005.PNG_L" value="183809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g005.PNG_M" value="59613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g005.PNG_S" value="10267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g005.TIF" value="310992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g005.PNG_I" value="21172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g006.PNG_L" value="2479967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g006.PNG_M" value="209124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g006.PNG_S" value="18463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g006.TIF" value="5594574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g006.PNG_I" value="159673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g007.PNG_L" value="185515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g007.PNG_M" value="56788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g007.PNG_S" value="9987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g007.TIF" value="331542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g007.PNG_I" value="23347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g008.PNG_L" value="200142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g008.PNG_M" value="54696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g008.PNG_S" value="10127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g008.TIF" value="358956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.g008.PNG_I" value="28424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e001.PNG" value="6765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e001.TIF" value="22776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e002.PNG" value="6547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e002.TIF" value="22576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e003.PNG" value="8151"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e003.TIF" value="24076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e004.PNG" value="7758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e004.TIF" value="23788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e005.PNG" value="9746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e005.TIF" value="25820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e006.PNG" value="27077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e006.TIF" value="42708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e007.PNG" value="9460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e007.TIF" value="25332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e008.PNG" value="12131"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.e008.TIF" value="27424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.s001.DOC" value="332049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.s002.DOC" value="34660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000722.s003.DOC" value="290876"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000722&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000722" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000722&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000722&volume=&issue=&title=Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in S. pombe&author_name=Majid%20Eshaghi%2C%20R.%20Krishna%20M.%20Karuturi%2C%20Juntao%20Li%2C%20Zhaoqing%20Chu%2C%20Edison%20T.%20Liu%2C%20Jianhua%20Liu&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000722" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000722" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000722" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000722&amp;t=Global%20Profiling%20of%20DNA%20Replication%20Timing%20and%20Efficiency%20Reveals%20that%20Efficient%20Replication%2FFiring%20Occurs%20Late%20during%20S-Phase%20in%20S.%20pombe" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000722&title=Global%20Profiling%20of%20DNA%20Replication%20Timing%20and%20Efficiency%20Reveals%20that%20Efficient%20Replication%2FFiring%20Occurs%20Late%20during%20S-Phase%20in%20S.%20pombe&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000722&amp;title=Global%20Profiling%20of%20DNA%20Replication%20Timing%20and%20Efficiency%20Reveals%20that%20Efficient%20Replication%2FFiring%20Occurs%20Late%20during%20S-Phase%20in%20S.%20pombe" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000722" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000722&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Global Profiling of DNA Replication Timing and Efficiency Reveals that Efficient Replication/Firing Occurs Late during S-Phase in S. pombe';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000722';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Global%20Profiling%20of%20DNA%20Replication%20Timing%20and%20Efficiency%20Reveals%20that%20Efficient%20Replication%2FFiring%20Occurs%20Late%20during%20S-Phase%20in%20S.%20pombe http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000722" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000722" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="24282"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+extraction%22" title="Search for articles in the subject area:'DNA extraction'"><div class="flagText">DNA extraction</div></a>
              <div data-categoryid="19965" data-articleid="24282"
                   data-categoryname="DNA extraction"
                   class="flagImage" title="Flag 'DNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+replication%22" title="Search for articles in the subject area:'DNA replication'"><div class="flagText">DNA replication</div></a>
              <div data-categoryid="19773" data-articleid="24282"
                   data-categoryname="DNA replication"
                   class="flagImage" title="Flag 'DNA replication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="24282"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24282"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Saccharomyces+cerevisiae%22" title="Search for articles in the subject area:'Saccharomyces cerevisiae'"><div class="flagText">Saccharomyces cerevisiae</div></a>
              <div data-categoryid="43143" data-articleid="24282"
                   data-categoryname="Saccharomyces cerevisiae"
                   class="flagImage" title="Flag 'Saccharomyces cerevisiae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Schizosaccharomyces+pombe%22" title="Search for articles in the subject area:'Schizosaccharomyces pombe'"><div class="flagText">Schizosaccharomyces pombe</div></a>
              <div data-categoryid="33499" data-articleid="24282"
                   data-categoryname="Schizosaccharomyces pombe"
                   class="flagImage" title="Flag 'Schizosaccharomyces pombe' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Synthesis+phase%22" title="Search for articles in the subject area:'Synthesis phase'"><div class="flagText">Synthesis phase</div></a>
              <div data-categoryid="19095" data-articleid="24282"
                   data-categoryname="Synthesis phase"
                   class="flagImage" title="Flag 'Synthesis phase' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3924'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1280'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6576&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=16145">Comment on the paper by Eshaghi et al.</a><br>Posted by Joel_Huberman</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
