

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Mapping of QTL for Resistance against the Crucifer Specialist Herbivore Pieris brassicae in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000578"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000578" />

    <meta name="citation_title" content="Mapping of QTL for Resistance against the Crucifer Specialist Herbivore Pieris brassicae in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2"/>
    <meta itemprop="name" content="Mapping of QTL for Resistance against the Crucifer Specialist Herbivore Pieris brassicae in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2"/>

      <meta name="citation_author" content="Marina Pfalz"/>
            <meta name="citation_author_institution" content="Department of Genetics & Evolution, Max Planck Institute for Chemical Ecology, Jena, Germany"/>
      <meta name="citation_author" content="Heiko Vogel"/>
            <meta name="citation_author_institution" content="Department of Entomology, Max Planck Institute for Chemical Ecology, Jena, Germany"/>
      <meta name="citation_author" content="Thomas Mitchell-Olds"/>
            <meta name="citation_author_institution" content="Department of Genetics & Evolution, Max Planck Institute for Chemical Ecology, Jena, Germany"/>
            <meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina, United States of America"/>
      <meta name="citation_author" content="Juergen Kroymann"/>
            <meta name="citation_author_institution" content="Department of Genetics & Evolution, Max Planck Institute for Chemical Ecology, Jena, Germany"/>

    <meta name="citation_date" content="2007/6/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000578.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e578"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Naturally occurring genetic variation in Arabidopsis thaliana.; citation_author=M Koornneef; citation_author=C Alonso-Blanco; citation_author=D Vreugdenhill; citation_journal_title=Annu Rev Plant Biol; citation_volume=55; citation_number=1; citation_pages=141-172; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recombinant inbred lines for mapping RFLP and phenotypic markers.; citation_author=C Lister; citation_author=C Dean; citation_journal_title=Plant J; citation_volume=4; citation_number=2; citation_pages=745-750; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Development of an AFLP based linkage map of Ler, Col and Cvi Arabidopsis thaliana ecotypes and construction of a Ler/Cvi recombinant inbred line population.; citation_author=C Alonso-Blanco; citation_author=AJM Peeters; citation_author=M Koornneef; citation_author=C Lister; citation_author=C Dean; citation_journal_title=Plant J; citation_volume=14; citation_number=3; citation_pages=259-271; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The TASTY locus on chromosome 1 of Arabidopsis affects feeding of the insect herbivore Trichoplusia ni.; citation_author=G Jander; citation_author=J Cui; citation_author=B Nhan; citation_author=NE Pierce; citation_author=FM Ausubel; citation_journal_title=Plant Physiol; citation_volume=126; citation_number=4; citation_pages=890-898; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis epithiospecifier protein promotes the hydrolysis of glucosinolates to nitriles and influences Trichoplusia ni herbivory.; citation_author=V Lambrix; citation_author=M Reichelt; citation_author=T Mitchell-Olds; citation_author=DJ Kliebenstein; citation_author=J Gershenzon; citation_journal_title=Plant Cell; citation_volume=13; citation_number=5; citation_pages=2793-2807; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of quantitative trait loci controlling glucosinolates, myrosinase and insect resistance in Arabidopsis thaliana.; citation_author=DJ Kliebenstein; citation_author=D Pedersen; citation_author=B Barker; citation_author=T Mitchell-Olds; citation_journal_title=Genetics; citation_volume=161; citation_number=6; citation_pages=325-332; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of an Arabidopsis insect resistance quantitative trait locus.; citation_author=J Kroymann; citation_author=S Donnerhacke; citation_author=D Schnabelrauch; citation_author=T Mitchell-Olds; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=7; citation_pages=14587-14592; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The gene controlling the quantitative trait locus EPITHIOSPECIFIER MODIFIER1 alters glucosinolate hydrolysis and insect resistance in Arabidopsis.; citation_author=Z Zhang; citation_author=JA Ober; citation_author=DJ Kliebenstein; citation_journal_title=Plant Cell; citation_volume=18; citation_number=8; citation_pages=1524-1536; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The glucosinolate-myrosinase system in an ecological and evolutionary context.; citation_author=DJ Kliebenstein; citation_author=J Kroymann; citation_author=T Mitchell-Olds; citation_journal_title=Curr Opin Plant Biol; citation_volume=8; citation_number=9; citation_pages=264-271; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Glucosinolate metabolism and its control.; citation_author=CD Grubb; citation_author=S Abel; citation_journal_title=Trends Plant Sci; citation_volume=11; citation_number=10; citation_pages=89-100; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biology and biochemistry of glucosinolates.; citation_author=BA Halkier; citation_author=J Gershenzon; citation_journal_title=Annu Rev Plant Biol; citation_volume=57; citation_number=11; citation_pages=303-333; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative quantitative trait loci mapping of aliphatic, indolic and benzylic glucosinolate production in Arabidopsis thaliana leaves and seeds.; citation_author=DJ Kliebenstein; citation_author=J Gershenzon; citation_author=T Mitchell-Olds; citation_journal_title=Genetics; citation_volume=159; citation_number=12; citation_pages=359-370; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetics of aliphatic glucosinolates: III. Side chain structure of aliphatic glucosinolates in Arabidopsis thaliana.; citation_author=R Mithen; citation_author=J Clarke; citation_author=C Lister; citation_author=C Dean; citation_journal_title=Heredity; citation_volume=74; citation_number=13; citation_pages=210-215; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Gene duplication and the diversification of secondary metabolism: side chain modification of glucosinolates in Arabidopsis thaliana.; citation_author=DJ Kliebenstein; citation_author=VM Lambrix; citation_author=M Reichelt; citation_author=J Gershenzon; citation_author=T Mitchell-Olds; citation_journal_title=Plant Cell; citation_volume=13; citation_number=14; citation_pages=681-693; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetics of aliphatic glucosinolates: I. Side chain elongation in Brassica napus and Arabidopsis thaliana.; citation_author=R Magrath; citation_author=F Bano; citation_author=M Morgner; citation_author=I Parkin; citation_author=A Sharpe; citation_journal_title=Heredity; citation_volume=72; citation_number=15; citation_pages=290-299; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=α-Keto acid elongation and glucosinolate biosynthesis in Arabidopsis thaliana.; citation_author=H Campos de Quiros; citation_author=R Magrath; citation_author=D McCallum; citation_author=J Kroymann; citation_author=D Schnabelrauch; citation_journal_title=Theor Appl Genet; citation_volume=101; citation_number=16; citation_pages=429-437; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A gene controlling variation in Arabidopsis thaliana glucosinolate composition is part of the methionine chain elongation pathway.; citation_author=J Kroymann; citation_author=S Textor; citation_author=JG Tokuhisa; citation_author=KL Falk; citation_author=S Bartram; citation_journal_title=Plant Physiol; citation_volume=127; citation_number=17; citation_pages=1077-1088; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic control of natural variation in Arabidopsis glucosinolate accumulation.; citation_author=DJ Kliebenstein; citation_author=J Kroymann; citation_author=P Brown; citation_author=A Figuth; citation_author=D Pedersen; citation_journal_title=Plant Physiol; citation_volume=126; citation_number=18; citation_pages=811-825; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of a new glucosinolate-rich cell type in Arabidopsis flower stalk.; citation_author=OA Koroleva; citation_author=A Davies; citation_author=R Deeken; citation_author=MR Thorpe; citation_author=AD Tomos; citation_journal_title=Plant Physiol; citation_volume=124; citation_number=19; citation_pages=599-608; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Guard cell- and phloem idioblast-specific expression of thioglucoside glucohydrolase 1 (myrosinase) in Arabidopsis.; citation_author=H Husebye; citation_author=S Chadchawan; citation_author=P Winge; citation_author=OP Thangstad; citation_author=AM Bones; citation_journal_title=Plant Physiol; citation_volume=128; citation_number=20; citation_pages=1180-1188; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cell specific cross-species expression of myrosinases in Brassica napus, Arabidopsis thaliana and Nicotiana tabacum.; citation_author=OP Thangstad; citation_author=B Gilde; citation_author=S Chadchawan; citation_author=M Seem; citation_author=H Husebye; citation_journal_title=Plant Mol Biol; citation_volume=54; citation_number=21; citation_pages=597-611; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis myrosinases TGG1 and TGG2 have redundant function in glucosinolate breakdown and insect defense.; citation_author=C Barth; citation_author=G Jander; citation_journal_title=Plant J; citation_volume=46; citation_number=22; citation_pages=549-562; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Glucosinolate research in the Arabidopsis era.; citation_author=U Wittstock; citation_author=BA Halkier; citation_journal_title=Trends Plant Sci; citation_volume=7; citation_number=23; citation_pages=263-270; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The ecological genetics of aliphatic glucosinolates.; citation_author=AF Raybold; citation_author=CL Moyes; citation_journal_title=Heredity; citation_volume=87; citation_number=24; citation_pages=383-391; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Disarming the mustard oil bomb.; citation_author=A Ratzka; citation_author=H Vogel; citation_author=DJ Kliebenstein; citation_author=T Mitchell-Olds; citation_author=J Kroymann; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=25; citation_pages=11223-11228; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Successful hervivore attack due to metabolic diversion of a plant chemical defense.; citation_author=U Wittstock; citation_author=N Agerbirk; citation_author=EJ Stauber; citation_author=CE Olsen; citation_author=M Hippler; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=26; citation_pages=4859-4864; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Positive selection driving diversification in plant secondary metabolism.; citation_author=M Benderoth; citation_author=S Textor; citation_author=AJ Windsor; citation_author=T Mitchell-Olds; citation_author=J Gershenzon; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=27; citation_pages=9118-9123; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci.; citation_author=MR Tuinstra; citation_author=G Ejeta; citation_author=PB Goldsbrough; citation_journal_title=Theor Appl Genet; citation_volume=95; citation_number=28; citation_pages=1005-1011; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Experimental manipulation of putative selective agents provides evidence for the role of natural enemies in the evolution of plant defense.; citation_author=R Mauricio; citation_author=MD Rausher; citation_journal_title=Evolution; citation_volume=51; citation_number=29; citation_pages=1435-1444; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Costs of resistance to natural enemies in field populations of the annual plant, Arabidopsis thaliana.; citation_author=R Mauricio; citation_journal_title=Am Nat; citation_volume=151; citation_number=30; citation_pages=20-28; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Variation in trichome density and resistance against a specialist insect herbivore in natural populations of Arabidopsis thaliana.; citation_author=R Handley; citation_author=B Ekbom; citation_author=J Ågren; citation_journal_title=Ecol Entomol; citation_volume=30; citation_number=31; citation_pages=284-292; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mapping of quantitative trait loci in multiple populations of Arabidopsis thaliana identifies natural allelic variation for trichome density.; citation_author=VV Symonds; citation_author=AV Godoy; citation_author=T Alconada; citation_author=JF Botto; citation_author=TE Juenger; citation_journal_title=Genetics; citation_volume=169; citation_number=32; citation_pages=1649-1658; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ontogenetics of QTL: the genetic architecture of trichome density over time in Arabidopsis thaliana.; citation_author=R Mauricio; citation_journal_title=Genetica; citation_volume=123; citation_number=33; citation_pages=75-85; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The molecular basis of quantitative genetic variation in central and secondary metabolism in Arabidopsis.; citation_author=T Mitchell-Olds; citation_author=D Pedersen; citation_journal_title=Genetics; citation_volume=149; citation_number=34; citation_pages=739-747; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis cDNA sequence encoding myrosinase.; citation_author=S Chadchawan; citation_author=J Bishop; citation_author=OP Thangstad; citation_author=AM Bones; citation_author=T Mitchell-Olds; citation_journal_title=Plant Physiol; citation_volume=103; citation_number=35; citation_pages=671-672; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The myrosinase gene family in Arabidopsis thaliana: gene organization, expression and evolution.; citation_author=J Xue; citation_author=M Jorgensen; citation_author=U Pihlgren; citation_author=L Rask; citation_journal_title=Plant Mol Biol; citation_volume=27; citation_number=36; citation_pages=911-922; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The third myrosinase gene TGG3 in Arabidopsis thaliana is a pseudogene specifically expressed in stamen and petal.; citation_author=J Zhang; citation_author=B Pontoppidan; citation_author=J Xue; citation_author=L Rask; citation_author=J Meijer; citation_journal_title=Physiol Plant; citation_volume=115; citation_number=37; citation_pages=25-34; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Glucosinolate and amino acid biosynthesis in Arabidopsis.; citation_author=BG Field; citation_author=G Cardon; citation_author=M Traka; citation_author=J Botterman; citation_author=G Vancanneyt; citation_journal_title=Plant Physiol; citation_volume=235; citation_number=38; citation_pages=828-839; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Insect pests of crucifers and their control.; citation_author=L Bonnemail; citation_journal_title=Annu Rev Entomol; citation_volume=10; citation_number=39; citation_pages=233-258; citation_date=1965; " />
      <meta name="citation_reference" content="citation_title=Induction of crop damage by the cabbage white butterfly Pieris rapae (Lepidoptera, Pieridae) on cabbage.; citation_author=PR Samson; citation_author=PW Geier; citation_journal_title=Protect Ecol; citation_volume=5; citation_number=40; citation_pages=199-233; citation_date=1983; " />
      <meta name="citation_reference" content="citation_author=DS Hill; citation_number=41; citation_pages=411-413; citation_date=1987; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Leaf surface compound from Brassica oleracea (Cruciferae) induces oviposition by Pieris brassicae (Lepidoptera: Pieridae).; citation_author=JJA van Loon; citation_author=A Blaakmeer; citation_author=FC Griepink; citation_author=TA van Beek; citation_author=LM Schoonhoven; citation_journal_title=Chemoecology; citation_volume=3; citation_number=42; citation_pages=39-44; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Relative activities of glucosinolates as oviposition stimulants for Pieris rapae and Pieris napi oleracea.; citation_author=XP Huang; citation_author=JAA Renwick; citation_journal_title=J Chem Ecol; citation_volume=20; citation_number=43; citation_pages=1025-1037; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Tarsal contact chemoreceptor response to glucosinolates and cardenolides mediating oviposition in Pieris rapae.; citation_author=E Städler; citation_author=JAA Renwick; citation_author=CD Radke; citation_author=K Sachdevgupta; citation_journal_title=Physiol Entomol; citation_volume=20; citation_number=44; citation_pages=175-187; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Behavioral and chemosensory responses to a host recognition cue by larvae of Pieris rapae.; citation_author=CI Miles; citation_author=ML del Campo; citation_author=JAA Renwick; citation_journal_title=J Comp Physiol A – Neuroethol Sens Neur Behav Physiol; citation_volume=191; citation_number=45; citation_pages=147-155; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gustation and foodplant selection in some Lepidopterous larvae.; citation_author=LM Schoonhoven; citation_journal_title=Entomol Exp Appl; citation_volume=12; citation_number=46; citation_pages=555-561; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Experience-based food consumption by larvae of Pieris rapae: addiction to glucosinolates?; citation_author=JAA Renwick; citation_author=K Lopez; citation_journal_title=Entomol Exp Appl; citation_volume=91; citation_number=47; citation_pages=51-58; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The chemical world of crucivores: lures, treats and traps.; citation_author=JAA Renwick; citation_journal_title=Entomol Exp Appl; citation_volume=104; citation_number=48; citation_pages=35-42; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of two rearing procedures for diamond-back moth Lepidoptera plutellidae.; citation_author=AM Shelton; citation_author=RJ Cooley; citation_author=MK Kroening; citation_author=WT Wilsey; citation_author=SD Eigenbrode; citation_journal_title=J Entomol Sci; citation_volume=26; citation_number=49; citation_pages=17-26; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Bay-0×Shahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis.; citation_author=OS Loudet; citation_author=S Chaillou; citation_author=C Camilleri; citation_author=D Bouchez; citation_author=F Daniel-Vedele; citation_journal_title=Theor Appl Genet; citation_volume=104; citation_number=50; citation_pages=1173-1184; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.; citation_journal_title=Nature; citation_volume=408; citation_number=51; citation_pages=796-815; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.; citation_author=ES Lander; citation_author=P Green; citation_author=J Abrahamson; citation_author=A Barlow; citation_author=MJ Daly; citation_journal_title=Genomics; citation_volume=1; citation_number=52; citation_pages=174-181; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Approach to determination of HPLC response factors for glucosinolates.; citation_author=R Buchner; citation_number=53; citation_pages=50-58; citation_date=1987; citation_publisher=Martinus Nijhoff Publishers; " />
      <meta name="citation_reference" content="citation_title=Variation of glucosinolate accumulation among different organs and developmental stages of Arabidopsis thaliana.; citation_author=PD Brown; citation_author=JG Tokuhisa; citation_author=M Reichelt; citation_author=J Gershenzon; citation_journal_title=Phytochemistry; citation_volume=62; citation_number=54; citation_pages=471-481; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=S Wang; citation_author=CJ Basten; citation_author=ZB Zeng; citation_number=55; citation_date=2001; citation_publisher=Department of Statistics, North Carolina State University; " />
      <meta name="citation_reference" content="citation_title=LOD significance thresholds for QTL analysis in experimental populations of diploid species.; citation_author=JW van Ooijen; citation_journal_title=Heredity; citation_volume=83; citation_number=56; citation_pages=613-624; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Markov chain Monte Carlo approach to detect polygene loci for complex traits.; citation_author=JM Satagopan; citation_author=BS Yandell; citation_author=MA Newton; citation_author=TC Osborn; citation_journal_title=Genetics; citation_volume=144; citation_number=57; citation_pages=805-816; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Epistasis and balanced polymorphism influencing complex trait variation.; citation_author=J Kroymann; citation_author=T Mitchell-Olds; citation_journal_title=Nature; citation_volume=435; citation_number=58; citation_pages=95-98; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000578" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Mapping of QTL for Resistance against the Crucifer Specialist Herbivore Pieris brassicae in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2"/>
    <meta name="twitter:description" content="BackgroundIn Arabidopsis thaliana and other crucifers, the glucosinolate-myrosinase system contributes to resistance against herbivory by generalist insects. As yet, it is unclear how crucifers defend themselves against crucifer-specialist insect herbivores.Methodology/Principal FindingsWe analyzed natural variation for resistance against two crucifer specialist lepidopteran herbivores, Pieris brassicae and Plutella xylostella, among Arabidopsis thaliana accessions and in a new Arabidopsis recombinant inbred line (RIL) population generated from the parental accessions Da(1)-12 and Ei-2. This RIL population consists of 201 individual F8 lines genotyped with 84 PCR-based markers. We identified six QTL for resistance against Pieris herbivory, but found only one weak QTL for Plutella resistance. To elucidate potential factors causing these resistance QTL, we investigated leaf hair (trichome) density, glucosinolates and myrosinase activity, traits known to influence herbivory by generalist insects. We identified several previously unknown QTL for these traits, some of which display a complex pattern of epistatic interactions.Conclusions/SignificanceAlthough some trichome, glucosinolate or myrosinase QTL co-localize with Pieris QTL, none of these traits explained the resistance QTL convincingly, indicating that resistance against specialist insect herbivores is influenced by other traits than resistance against generalists."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000578.g004"/>

  <meta property="og:title" content="Mapping of QTL for Resistance against the Crucifer Specialist Herbivore Pieris brassicae in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000578" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8092'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6763'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3361&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000578">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000578" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000578&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Mapping of QTL for Resistance against the Crucifer Specialist Herbivore <i>Pieris brassicae</i> in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marina Pfalz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics & Evolution, Max Planck Institute for Chemical Ecology, Jena, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Heiko Vogel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Entomology, Max Planck Institute for Chemical Ecology, Jena, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Mitchell-Olds, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Genetics & Evolution, Max Planck Institute for Chemical Ecology, Jena, Germany, 
                  Department of Biology, Duke University, Durham, North Carolina, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Juergen Kroymann
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kroymann@ice.mpg.de">kroymann@ice.mpg.de</a></p>

                <p>Affiliation:
                  Department of Genetics & Evolution, Max Planck Institute for Chemical Ecology, Jena, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 27, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000578</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000578" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000578" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000578" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000578" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000578-g001" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000578-g002" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000578-g003" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000578-g004" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000578-t001" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000578-t002" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000578">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000578" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>In <em>Arabidopsis thaliana</em> and other crucifers, the glucosinolate-myrosinase system contributes to resistance against herbivory by generalist insects. As yet, it is unclear how crucifers defend themselves against crucifer-specialist insect herbivores.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We analyzed natural variation for resistance against two crucifer specialist lepidopteran herbivores, <em>Pieris brassicae</em> and <em>Plutella xylostella</em>, among <em>Arabidopsis thaliana</em> accessions and in a new Arabidopsis recombinant inbred line (RIL) population generated from the parental accessions Da(1)-12 and Ei-2. This RIL population consists of 201 individual F<sub>8</sub> lines genotyped with 84 PCR-based markers. We identified six QTL for resistance against <em>Pieris</em> herbivory, but found only one weak QTL for <em>Plutella</em> resistance. To elucidate potential factors causing these resistance QTL, we investigated leaf hair (trichome) density, glucosinolates and myrosinase activity, traits known to influence herbivory by generalist insects. We identified several previously unknown QTL for these traits, some of which display a complex pattern of epistatic interactions.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Although some trichome, glucosinolate or myrosinase QTL co-localize with <em>Pieris</em> QTL, none of these traits explained the resistance QTL convincingly, indicating that resistance against specialist insect herbivores is influenced by other traits than resistance against generalists.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Pfalz M, Vogel H, Mitchell-Olds T, Kroymann J (2007) Mapping of QTL for Resistance against the Crucifer Specialist Herbivore <em>Pieris brassicae</em> in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2. PLoS ONE 2(6):
          e578.
            doi:10.1371/journal.pone.0000578</p><p><strong>Academic Editor: </strong>Hany El-Shemy, Cairo University, Egypt</p><p><strong>Received:</strong> April 25, 2007; <strong>Accepted:</strong> June 1, 2007; <strong>Published:</strong> June 27, 2007</p><p><strong>Copyright:</strong> © 2007 Pfalz et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Max Planck Society, the Bundesministerium für Bildung und Forschung (PTJ-BIO/0313063), the Deutsche Forschungsgemeinschaft (KR 2237/2-1 and KR 2237/2-2), and the European Commission (Contract QLRT-2000-01097).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Arabidopsis thaliana</em> recombinant inbred lines (RILs) have been widely used for mapping of quantitative trait loci (QTL) (reviewed in <a href="#pone.0000578-Koornneef1">[1]</a>). Taking advantage of RILs derived from crosses between the accessions Columbia (Col) and Landsberg <em>erecta</em> (L<em>er</em>) <a href="#pone.0000578-Lister1">[2]</a>, and between L<em>er</em> and Cape Verdi Islands (Cvi) <a href="#pone.0000578-AlonsoBlanco1">[3]</a>, several insect resistance QTL have been mapped and, subsequently, several were cloned and characterized <a href="#pone.0000578-Jander1">[4]</a>–<a href="#pone.0000578-Zhang1">[8]</a>. In most cases, these studies involved lepidopteran species with a broad host range (generalists) such as <em>Spodoptera exigua</em> or <em>Trichoplusia ni</em>, and found that generalist insects were sensitive towards glucosinolate-based defenses. Glucosinolates (β-thioglucoside–N-hydroxysulfates) are amino acid-derived secondary plant metabolites that can be hydrolyzed by myrosinases, enzymes with β-thioglucoside glucohydrolase activity <a href="#pone.0000578-Kliebenstein2">[9]</a>–<a href="#pone.0000578-Halkier1">[11]</a>. In <em>Arabidopsis thaliana</em>, three major classes of glucosinolates are known: aliphatic glucosinolates derived from chain-extended methionine homologues, indole glucosinolates derived from tryptophan, and benzyl glucosinolates originating from a phenylalanine precursor <a href="#pone.0000578-Kliebenstein3">[12]</a>. Two major loci, <em>AOP</em> <a href="#pone.0000578-Mithen1">[13]</a>, <a href="#pone.0000578-Kliebenstein4">[14]</a> and <em>MAM</em> <a href="#pone.0000578-Kroymann1">[7]</a>, <a href="#pone.0000578-Magrath1">[15]</a>–<a href="#pone.0000578-Kroymann2">[17]</a>, and several minor loci <a href="#pone.0000578-Kliebenstein5">[18]</a> control composition and quantity of aliphatic glucosinolates. Methylthioalkylmalate synthases encoded at the <em>MAM</em> locus determine the side chain length of the methionine-derived precursors, while 2-oxoglutarate-dependent dioxygenases encoded at <em>AOP</em> modify the side chain structure. In intact plant tissue, glucosinolates and myrosinases are localized in separate cell types <a href="#pone.0000578-Koroleva1">[19]</a>–<a href="#pone.0000578-Barth1">[22]</a>. Upon tissue disruption, myrosinase-catalyzed glucosinolate hydrolysis results in the formation of bioactive products, including isothiocyanates, nitriles, thiocyanates and others <a href="#pone.0000578-Wittstock1">[23]</a>. The types of breakdown products formed depend on the glucosinolate structure, as well as on myrosinase-associated or –binding proteins that can direct the formation of breakdown products towards nitriles or isothiocyanates <a href="#pone.0000578-Lambrix1">[5]</a>, <a href="#pone.0000578-Zhang1">[8]</a>. Typically, plant damage caused by generalist insect herbivores is negatively correlated with increasing glucosinolate concentration or myrosinase activity, and resistance QTL co-localize with glucosinolate biosynthesis or hydrolysis QTL, providing evidence for a major role of the glucosinolate-myrosinase system in the defense of cruciferous plants against generalist insect herbivores <a href="#pone.0000578-Lambrix1">[5]</a>–<a href="#pone.0000578-Kliebenstein2">[9]</a>, <a href="#pone.0000578-Raybold1">[24]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Entirely unclear, however, is how cruciferous plants defend themselves against specialist insect herbivores. Several counteradaptations have been identified in crucifer specialist lepidopterans that render the glucosinolate-myrosinase system ineffective. <em>Plutella xylostella</em> (diamondback moth) larvae express a glucosinolate sulfatase in their gut that removes the sulfate moiety from glucosinolates, thereby preventing myrosinase-catalyzed hydrolysis and formation of toxic breakdown products <a href="#pone.0000578-Ratzka1">[25]</a>. <em>Pieris rapae</em> (cabbage white butterfly) possesses a nitrile-specifier protein (NSP) that redirects glucosinolate hydrolysis towards the formation of nitriles instead of highly toxic isothiocyanates when plant tissue is ingested by <em>Pieris</em> larvae <a href="#pone.0000578-Wittstock2">[26]</a>. Nonetheless, Arabidopsis accessions vary for resistance against specialist insect herbivores. In this paper, we analyze quantitative genetic variation for resistance against two crucifer specialist lepidopteran herbivores, <em>Pieris brassicae</em> and <em>Plutella xylostella</em>, among Arabidopsis accessions and in a new Arabidopsis recombinant inbred line, Da(1)-12×Ei-2, and we investigate whether variation in glucosinolate profiles, myrosinase activity or trichomes contributes to variation in resistance.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Natural Variation for Resistance against Crucifer Specialist Insect Herbivores among Arabidopsis Accessions</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We analyzed 16 Arabidopsis accessions for natural variation in resistance against two crucifer specialist insects, <em>Pieris brassicae</em> and <em>Plutella xylostella</em>. We found substantial variation for resistance against <em>P. brassicae</em> (<em>F</em> = 20.31, <em>df</em> = 15, <em>N</em> = 973, <em>P</em>&lt;0.00001), with Col-0 being the most resistant and Tsu-0 the most susceptible accession (<a href="#pone-0000578-g001"><strong>Figure 1</strong></a>). Variation for resistance against <em>P. xylostella</em> was less pronounced, but nonetheless statistically significant (<em>F</em> = 2.38, <em>df</em> = 15, <em>N</em> = 912, <em>P</em>&lt;0.005). Resistance to <em>P. brassicae</em> and <em>P. xylostella</em> was positively correlated (<em>r</em> = 0.55, <em>df</em> = 14, <em>P</em>&lt;0.05), suggesting that some determinants of plant resistance affect both specialists similarly.</p>
<div class="figure" id="pone-0000578-g001"><div class="img"><a name="pone-0000578-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Natural variation among 16 Arabidopsis accessions for insect resistance, growth and defense-related traits.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Accessions are ranked according to increasing susceptibility to herbivory by <em>Pieris brassicae</em> larvae. Shown are least squares means and standard errors (vertical bars) for <em>Pieris brassicae</em> herbivory, <em>Plutella xylostella</em> herbivory, plant diameter, leaf upper-side trichomes, myrosinase activity, aliphatic and indole glucosinolates. Values for Da(1)-12 are set as 100.</p>
<span>doi:10.1371/journal.pone.0000578.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We also analyzed natural genetic variation for trichome density, glucosinolate content and myrosinase activity, traits known to influence resistance against generalist insect herbivores. As expected, these traits varied among Arabidopsis accessions (<a href="#pone-0000578-g001"><strong>Figure 1</strong></a>). However, no single trait alone could explain the observed variation in resistance to <em>P. xylostella</em> or <em>P. brassicae</em> among Arabidopsis accessions. We therefore chose to analyze quantitative variation for resistance and defense-related traits in a new Arabidopsis recombinant inbred line population, derived from a cross between the parental accessions Da(1)-12 and Ei-2 <a href="#pone.0000578-Lambrix1">[5]</a>. Although these lines did not represent the extreme phenotypes in the distribution of resistance against <em>P. brassicae</em>, they provided a variety of advantages regarding the composition of alleles at glucosinolate biosynthesis and hydrolysis loci compared to ‘standard’ mapping populations such as Col×L<em>er</em> (2) or L<em>er</em>×Cvi <a href="#pone.0000578-AlonsoBlanco1">[3]</a>. Leaves of Da(1)-12 and Ei-2 synthesize aliphatic and indole glucosinolates. Both lines accumulate similar quantities of glucosinolates in their leaf tissue (<a href="#pone-0000578-g001"><strong>Figure 1</strong></a>). Furthermore, in both lines the predominant aliphatic glucosinolates are derived from a homo-methionine precursor, indicating the presence of an intact <em>MAM2</em> gene and the absence of a functional <em>MAM1</em> gene in the <em>MAM</em> gene cluster <a href="#pone.0000578-Kroymann1">[7]</a>, <a href="#pone.0000578-Benderoth1">[27]</a>. However, Da(1)-12 and Ei-2 differ in their alleles at the <em>AOP</em> locus <a href="#pone.0000578-Kliebenstein4">[14]</a>. Da(1)-12 possesses an <em>OHP</em> allele at <em>AOP</em>, while Ei-2 carries an <em>ALK</em> allele. Therefore, Da(1)-12 produces mainly 3-hydroxypropyl and 3-methylsulfinyl glucosinolates, and Ei-2 accumulates allyl glucosinolate. Finally, during glucosinolate hydrolysis, Da(1)-12 produces isothiocyanates whereas Ei-2 generates predominantly nitriles <a href="#pone.0000578-Lambrix1">[5]</a>. This combination of alleles at glucosinolate biosynthesis and hydrolysis loci helps reduce complexity in the investigation of potential impact of the glucosinolate-myrosinase system <a href="#pone.0000578-Kliebenstein2">[9]</a> on herbivory. It avoids epistatic interactions between known major biosynthesis loci, <em>AOP</em> and <em>MAM</em>, <a href="#pone.0000578-Kliebenstein3">[12]</a> while allowing analysis of potential effects of interactions between glucosinolate biosynthesis and hydrolysis loci on crucifer specialists. In addition, growth rates of both Da(1)-12 and Ei-2 were nearly identical (<a href="#pone-0000578-g001"><strong>Figure 1</strong></a>), improving the accuracy of estimating the quantity of tissue removal during herbivory.</p>


<h4>RIL Genotyping</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>F<sub>9</sub> progeny from 201 Da(1)-12×Ei-2 RILs was genotyped with 84 markers. Out of a total of 16,884 PCRs, only 126 failed or yielded ambiguous results. Residual heterozygosity was low, although the observed value of ca. 1.2% was larger than 0.4% which is expected for F<sub>9</sub> progeny obtained by repeated selfing, possibly indicating some heterozygote advantage. Also, a significant deviation from expected 1:1 genotype frequencies was observed for a large segment of chromosome 1, comprising markers <em>C1P12</em> to <em>B12</em> (<a href="#pone-0000578-g002"><strong>Figure 2</strong></a>), with the most significant excess of the Da(1)-12 genotype at marker <em>F1K23_2</em> (χ2 = 37.05, df = 1, P&lt;0.001). Such distortion has also been observed for other RIL populations <a href="#pone.0000578-Lister1">[2]</a>, <a href="#pone.0000578-AlonsoBlanco1">[3]</a>, and may have been caused by unintentional selection during RIL generation. Nonetheless, the order of all genetic markers in the Da(1)-12×Ei-2 RILs was compatible with their physical position in the Col-0 sequence.</p>
<div class="figure" id="pone-0000578-g002"><div class="img"><a name="pone-0000578-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Genetic map of Da(1)-12×Ei-2 recombinant inbred lines.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Shown are markers and genetic distances between adjacent markers along the chromosomes. Triangles indicate the approximate location of centromeres. The white bar for chromosome 1 indicates a region with marker distortion. Grey bars indicate 2-LOD support intervals for <em>Pieris</em> resistance QTL. Known genes involved in glucosinolate biosynthesis, hydrolysis, and gene regulation are shown below chromosomes; explanations, AGI numbers and references are given in <a href="#pone.0000578.s002">Table S2</a>.</p>
<span>doi:10.1371/journal.pone.0000578.g002</span></div>

<h4>QTL for Resistance against Crucifer Specialist Insect Herbivores</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We identified six QTL for resistance against <em>P. brassicae</em> herbivory, each two on chromosomes 1 and 5, and each one on chromosomes 2 and 4 (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). At all QTL, the Da(1)-12 genotype confers higher resistance to <em>P. brassicae</em>, and each Da(1)-12 allele increases resistance by 10–20%, consistent with higher resistance in the Da(1)-12 parental line (<a href="#pone-0000578-g001"><strong>Figure 1</strong></a>). Together, these QTL explain nearly half of the phenotypic variance in our experiments (<em>R</em><sup>2</sup> = 48.4%). In contrast, we did not find any QTL for resistance against <em>P. xylostella</em> herbivory with composite interval mapping (CIM), while Bayesian interval mapping (BIM) indicated the presence of one weak QTL, located at the same position as the <em>Pieris</em> herbivory QTL on chromosome 2 (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>).</p>
<div class="figure" id="pone-0000578-g003"><div class="img"><a name="pone-0000578-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>QTL for insect resistance and defense-related traits in Da(1)-12×Ei-2 recombinant inbred lines.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A. QTL for resistance against <em>Pieris brassicae</em>, obtained with composite interval mapping (CIM). The horizontal red line indicates the significance threshold for this trait. B. Bayesian interval mapping (BIM) detects the same QTL as CIM. Horizontal red lines indicate high density probability regions in BIM, vertical red lines high density probability peaks. C. Additive effects in BIM, shown as a scatter plot with a smoothing spline fit (solid blue line) plus or minus two standard errors (dashed blue lines). For all six QTL, <em>Pieris</em> larvae cause greater damage when plants carry the Ei-2 alleles at the QTL. Hence, the Da(1)-12 alleles confer higher resistance. D. QTL for resistance against specialist lepidopterans, plant diameter, myrosinase activity, trichome density on the leaf upper and under-sides and perimeter, and for glucosinolates. For glucosinolates, QTL for individual compounds and for sum variables are given. Abbreviations are as follows: 3OHP = 3-hydroxypropyl; 3MSOP = 3-methylsulfinylpropyl; 4MSOB = 4-methylsulfinylbutyl; 7MSOH = 7-methylsulfinylheptyl; 8MSOO = 8-methylsulfinyloctyl; 8MTO = 8-methylthiooctyl; I3M = indol-3-yl-methyl; 1MO-I3M = 1-methoxy-indol-3-yl-methyl; 4OH-I3M = 4-hydroxy-indol-3-yl-methyl; 4MO-I3M = 4-methoxy-indol-3-yl-methyl. Total 3, 4, 7 and 8 are the sums of homomethionine-, di-homomethionine-, penta-homomethionine-, and hexa-homomethionine-derived glucosinolates, respectively. Total MSO, MT, and alkenyls are the sums of aliphatic glucosinolates with methylsulfinyl-, methylthio-, and alkenyl-groups, respectively. Total aliphatics and indoles are the sums of methionine- and tryptophan-derived glucosinolates, respectively, and total GS is the sum of all glucosinolates. Colored arrows correspond to 2-LOD support intervals for QTL identified with CIM, black arrows for high density probability regions in BIM, with vertical black bars showing the position of the high density probability peaks. Arrow directions correspond to effect directions; arrows pointing left indicate that the Ei-2 allele has a stronger effect on a particular trait. Arrow fill colors code for the relative effect strength of a QTL, arrow background colors for <em>R</em><sup>2</sup>, the percentage of variance explained by a QTL. Horizontal red lines connecting colored arrows indicate epistatic interactions between QTL.</p>
<span>doi:10.1371/journal.pone.0000578.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We tested for epistasis between major herbivory QTL but detected a significant interaction only between markers <em>C1P12</em> and <em>MSAT1.1</em> (<em>F</em> = 6.01, <em>df</em> = 1, <em>N</em> = 180, <em>P</em>&lt;0.05) which correspond to the two QTL on chromosome 1. Here, a Da(1)-12 allele at <em>MSAT1.1</em> reduces plant damage by ca. 18% when plants carry the Da(1)-12 allele at <em>C1P12</em>, but only by ca. 4% when the allele at <em>C1P12</em> is Ei-2. <em>Vice versa</em>, a Da(1)-12 allele at <em>C1P12</em> reduces damage by ca. 20% in plants with a Da(1)-12 allele at <em>MSAT1.1</em> but only by ca. 5% in plants with an Ei-2 allele at <em>MSAT1.1</em>.</p>


<h4>Confirmation of a <em>Pieris</em> Resistance QTL with a Heterogeneous Inbred Family Strategy</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We used a heterogeneous inbred family (HIF) strategy <a href="#pone.0000578-Tuinstra1">[28]</a> to confirm the <em>Pieris</em> resistance QTL near marker <em>C1P12</em>. This strategy utilizes residual heterozygosity in a RIL population, and compares the phenotypes of genotyped progeny from a line heterozygous at a QTL candidate marker. HIF allows the rapid generation of a plant family homozygous for the majority of the genome but segregating at the candidate QTL.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We chose RIL DE196 which was heterozygous at both <em>Pieris</em> QTL on chromosome 1 (<a href="#pone.0000578.s001"><strong>Table S1</strong></a>). From progeny of this line, we selected plants that carried either only Da(1)-12 or only Ei-2 alleles at marker <em>MSAT1.1</em>, but segregated at <em>C1P12</em>. Our statistical model accounted for plant size, flat and position effects. As expected, plants with a Da(1)-12 allele at <em>C1P12</em> experienced significantly less damage in <em>P. brassicae</em> herbivory screens than plants with an Ei-2 allele at this locus when <em>MSAT1.1</em> was homozygous Da(1)-12 (<em>F</em> = 18.70; <em>df</em> = 1; <em>N</em> = 130, <em>P</em>&lt;0.05) but not when <em>MSAT1.1</em> was homozygous Ei-2 (<em>F</em> = 0.37; <em>df</em> = 2, <em>N</em> = 176, n.s.).</p>


<h4>QTL for Trichome Density</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Leaf hairs, or trichomes, can contribute to plant defense against herbivorous insects in Arabidopsis <a href="#pone.0000578-Mauricio1">[29]</a>, <a href="#pone.0000578-Mauricio2">[30]</a> and related plant species <a href="#pone.0000578-Handley1">[31]</a>. Therefore, we mapped QTL for trichome numbers on the leaf upper and under-side surfaces and the leaf perimeter (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). We identified three QTL controlling trichome density on the leaf upper sides, one on chromosome 3 and two with opposing effect on chromosome 4. These QTL may correspond to trichome QTL that have been mapped in several other Arabidopsis recombinant inbred lines <a href="#pone.0000578-Symonds1">[32]</a>. For trichomes on the leaf perimeter, we identified only one QTL on chromosome 2. This QTL maps to approximately the same position as a major trichome QTL previously identified in the Col×L<em>er</em> RIL population <a href="#pone.0000578-Mauricio3">[33]</a>. Two QTL with opposing effects control trichome density on the leaf under-side, located on chromosomes 3 and 5, with the one in the center of chromosome 3 sharing its position with a QTL for trichome density on the leaf upper-side. Finally, we found epistatic interactions for both upper- and under-side trichomes. For trichome numbers on the leaf upper-side, we found a significant interaction (<em>F</em> = 4.13, <em>df</em> = 1, <em>N</em> = 94, <em>P</em>&lt;0.05) between markers <em>3FM12</em>, close to the QTL LOD peak on chromosome 3, and <em>MSAT4.18</em>, which corresponds to the QTL near the center of chromosome 4. Trichome numbers were highest when both markers carried Ei-2 alleles and lowest when both markers had the Da(1)-12 genotype. A Da(1)-12 allele at <em>MSAT4.18</em> reduced trichome numbers on the leaf upper-side by ca. 17% when the allele at <em>3FM12</em> was Da(1)-12 but only by ca. 3.5% when <em>3FM12</em> had the Ei-2 allele. Likewise, the two QTL for leaf under-side trichomes interacted epistatically (<em>F</em> = 5.49, <em>df</em> = 1, <em>N</em> = 92, <em>P</em>&lt;0.05). Here, trichome numbers were highest when <em>3FM12</em> carried the Ei-2 allele and <em>MIO24</em>, on chromosome 5, had the Da(1)-12 allele. Substitution of the Da(1)-12 allele at <em>MIO24</em> with an Ei-2 allele resulted in a reduction of trichome numbers by ca. 64%, substitution of the Ei-2 allele at <em>3FM12</em> in a reduction by ca. 87%. Finally, substitution of both alleles with the reciprocal genotypes led to a reduction by ca. 96% such that trichomes were rarely detected on the leaf under-side of RILs with a Da(1)-12 allele at <em>3FM12</em> and an Ei-2 allele at <em>MIO24</em>.</p>


<h4>QTL for Myrosinase Activity</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We have identified three QTL for myrosinase activity that exceeded the significance threshold, one on chromosome 3, and two on chromosome 5 (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). At the QTL near the top of chromosome 3, the Ei-2 genotype confers higher myrosinase activity, while at the other QTL the Da(1)-12 alleles are more active. In a previous study, Mitchell-Olds and Pedersen <a href="#pone.0000578-MitchellOlds1">[34]</a> had identified two different myrosinase QTL in the Col×L<em>er</em> RIL population, one on chromosome 1 and the other near the center of chromosome 3. These QTL map to different locations than the ones identified in the present work. However, two of the known myrosinase genes in <em>Arabidopsis thaliana, TGG1</em> and <em>TGG2</em> <a href="#pone.0000578-Chadchawan1">[35]</a>, <a href="#pone.0000578-Xue1">[36]</a>, map close to the LOD peak of the first myrosinase QTL on chromosome 5 (<a href="#pone-0000578-g002"><strong>Figures 2</strong></a><strong>, </strong><a href="#pone-0000578-g003"><strong>3</strong></a>), and likely cause this QTL. The third known myrosinase gene, <em>TGG3</em>, is a pseudogene in all accessions investigated so far <a href="#pone.0000578-Zhang2">[37]</a>, and does not map within the 2-LOD support interval of the second QTL on chromosome 5, although it is located in its vicinity. Hence, two of the QTL identified in Da(1)-12×Ei-2 represent novel myrosinase QTL.</p>


<h4>QTL for Aliphatic Glucosinolates</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Ei-2 leaves produce ca. 50 – 60% more total glucosinolates than Da(1)-12 leaves, but this difference is small compared to the variation present among <em>A. thaliana</em> accessions (<a href="#pone-0000578-g001"><strong>Figure 1</strong></a>; <a href="#pone.0000578-Kliebenstein5">[18]</a>). Only few QTL control total glucosinolate quantity, one on chromosome 1 and one near the top of chromosome 4 (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>), which very likely corresponds to <em>AOP</em> (<a href="#pone-0000578-g001"><strong>Figure 1</strong></a>; <a href="#pone.0000578-Kliebenstein4">[14]</a>). Total aliphatic glucosinolate accumulation is influenced by three QTL; <em>AOP</em>, a QTL near the bottom of chromosome 1, and a QTL near the top of chromosome 5. While a QTL for aliphatic glucosinolates near the top of chromosome 5 has also been identified in the L<em>er</em>×Cvi <a href="#pone.0000578-Kliebenstein3">[12]</a> and Col×L<em>er</em> RILs <a href="#pone.0000578-Kliebenstein1">[6]</a>, the QTL near the bottom of chromosome 1 was previously unknown. The <em>AOP</em> locus (or a closely linked gene) also constitutes a QTL for nearly all individual aliphatic glucosinolates except for the hexa-homomethionine-derived glucosinolates, 8-methylsulfinyloctyl and 8-methylthiooctyl glucosinolate.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Because both parental lines lack a functional <em>MAM1</em> gene, most aliphatic glucosinolates are homomethionine-derived, with only small quantities of di-homomethionine-derived glucosinolates being detectable. However, the genetic architecture underlying the biosynthesis of short chain aliphatic glucosinolates (<em>i.e.</em>, homo- and di-homomethionine derivatives) is nonetheless complex. The side chain structure of homomethionine-derived glucosinolates is modified by alleles at the <em>AOP</em> locus. RILs with the Da(1)-12 <em>OHP</em> allele accumulate 3-hydroxypropyl and 3-methylsulfinylpropyl glucosinolates, while lines with the Ei-2 <em>ALK</em> allele accumulate allyl glucosinolates. Therefore, QTL effects for 3-hydroxypropyl and 3-methylsulfinyl glucosinolates have the same direction, but are opposite to the QTL effect for allyl glucosinolate (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). The quantity of 3-methylsulfinylpropyl glucosinolate is also influenced by a QTL on chromosome 5, and by an epistatic interaction between this QTL and <em>AOP</em>. The QTL on chromosome 5 is located near marker <em>nga139</em>, in the vicinity of the <em>MAM</em> genes (<a href="#pone-0000578-g002"><strong>Figure 2</strong></a>). Among the RILs that harbor the Da(1)-12 allele at <em>AOP</em> and are, thus, capable of producing 3-methylsulfinylpropyl glucosinolate, those lines with a Da(1)-12 allele at <em>nga139</em> accumulate two- to threefold more 3-methylsulfinylpropyl glucosinolate than lines with an Ei-2 allele at this marker (<a href="#pone-0000578-g004"><strong>Figure 4</strong></a>).</p>
<div class="figure" id="pone-0000578-g004"><div class="img"><a name="pone-0000578-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000578.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Epistatic interactions in the biosynthesis of homomethionine-derived glucosinolates.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Left: 3-methylsulfinylpropyl glucosinolate (3MSOP) is produced when the genotype at the <em>AOP</em> locus (marker <em>F4C21</em>) is Da(1)-12. An Ei-2 allele at marker <em>nga139</em> reduces the accumulation of this glucosinolate by 60%. Right: allyl glucosinolate is produced when the genotype at <em>AOP</em> is Ei-2. A Da(1)-12 allele at <em>3FM12</em> increases and a Da(1)-12 allele at <em>nga249</em> decreases leaf allyl glucosinolate accumulation. <em>N</em> indicates the number of RILs with a particular combination of genotypes at the different markers.</p>
<span>doi:10.1371/journal.pone.0000578.g004</span></div><a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Likewise, allyl glucosinolate accumulation is not only determined by <em>AOP</em> but also by two further QTL, on chromosome 3 (marker <em>3FM12</em>) and near the top of chromosome 5 (marker <em>nga249</em>), and by pairwise epistatic interactions between <em>AOP</em> and these QTL (<a href="#pone-0000578-g004"><strong>Figure 4</strong></a>). Allyl glucosinolates are only synthesized when RILs harbor the Ei-2 allele at <em>AOP</em>, with a Da(1)-12 allele at marker <em>3FM12</em> increasing and a Da(1)-12 allele at marker <em>nga249</em> decreasing allyl glucosinolate accumulation. Since allyl glucosinolates account for the majority of alkenyl glucosinolates in Da(1)-12×Ei-2 leaves, total alkenyl glucosinolates follow the same pattern of QTL and epistatic interactions.</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>Within the <em>MAM</em> gene cluster, <em>MAM1</em> and/or <em>MAM2</em> control variation in short-chain aliphatic glucosinolate accumulation <a href="#pone.0000578-Kroymann1">[7]</a>, <a href="#pone.0000578-Benderoth1">[27]</a>. The third gene in this cluster, <em>MAM-L</em>, is essential for the production of long chain glucosinolates, and a <em>MAM-L</em> knock-out abolished the formation of long chain aliphatic glucosinolates <a href="#pone.0000578-Field1">[38]</a>. With the possible exception of 3-methylsulfinylpropyl glucosinolate, the <em>MAM</em> locus has no detectable influence on glucosinolate profile variation in Da(1)-12×Ei-2 (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). Nonetheless, in other regions of the genome we found several QTL for long chain aliphatic glucosinolates, <em>i.e.</em> for penta- and hexa-homomethionine-derived glucosinolates. Two QTL are located on chromosome 1, two on chromosome 4, and one is positioned near the bottom of chromosome 5. Hence, these additional QTL control variation in the accumulation of long-chain aliphatic glucosinolates in Da(1)-12×Ei-2, even though the <em>MAM</em> locus does not contribute to this variability.</p>


<h4>QTL for Indole Glucosinolates</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>We identified five QTL for total indole glucosinolate accumulation, each one on chromosomes 1, 2 and 3, and two on chromosome 4. The QTL near the top of chromosome 4 had the strongest effect on indole glucosinolate accumulation. This QTL co-localizes with the <em>AOP</em> locus, suggesting that <em>AOP</em> influences not only aliphatic glucosinolates but also indole glucosinolate accumulation, either directly by catalyzing a biosynthetic reaction step or indirectly via utilization of a pool of metabolites that is shared in aliphatic and indole glucosinolate biosynthesis. Alternately, a gene tightly linked to <em>AOP</em> could also explain this indole glucosinolate QTL.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>We also detected complex epistatic interactions between the QTL for total indole glucosinolate accumulation. We found pairwise epistatic interactions between the QTL on chromosome 3 (marker <em>MSAT3.19</em>) and the QTL near the bottom of chromosome 2 (marker <em>C22</em>), between <em>MSAT3.19</em> and the QTL near the top on chromosome 4 (marker <em>F4C21</em>), and between <em>MSAT3.19</em> and the QTL at the center of chromosome 4 (marker <em>T6G15</em>), as well as a triple interaction between <em>C22</em>, <em>MSAT3.19</em> and <em>F4C21</em> (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>, <a href="#pone-0000578-t001"><strong>Table 1</strong></a>).</p>
<div class="figure" id="pone-0000578-t001"><div class="img"><a name="pone-0000578-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000578.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000578.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Epistatic interactions in glucosinolate biosynthesis</span></strong></p><span>doi:10.1371/journal.pone.0000578.t001</span></div><a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>The QTL pattern for indol-3-yl-methyl glucosinolate, the most abundant indole glucosinolate in <em>A. thaliana</em> leaves <a href="#pone.0000578-Kliebenstein5">[18]</a>, reflects largely the QTL pattern for total indole glucosinolate accumulation, except that the QTL for total indole glucosinolates on chromosome 3 did not exceed the significance threshold for indol-3-yl-methyl glucosinolate. Nonetheless, marker <em>MSAT3.19</em> showed the same pattern of epistatic interactions for indol-3-yl-methyl glucosinolates as for total indole glucosinolates, when we included it in our statistics models.</p>
<a id="article1.body1.sec2.sec8.p4" name="article1.body1.sec2.sec8.p4"></a><p>Da(1)-12×Ei-2 leaves synthesize three further indole glucosinolates, 1-methoxy-indol-3-yl-methyl, 4-hydroxy-indol-3-yl-methyl and 4-methoxy-indol-3-yl-methyl glucosinolate. 1-methoxy-indol-3-methyl glucosinolate shares two of its three QTL with indol-3-yl-methyl glucosinolate but has an additional QTL near the top of chromosome 5. This additional QTL might correspond to the position of a gene responsible for the generation of the methoxy-group at position 1 of the tryptophan moiety.</p>
<a id="article1.body1.sec2.sec8.p5" name="article1.body1.sec2.sec8.p5"></a><p>4-hydroxy-indol-3-yl-methyl glucosinolate accumulation is controlled by two QTL, one near the top of chromosome 1 and the second near the bottom of chromosome 5 (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). The position of this QTL on chromosome 5 and its effect direction are shared by one of the QTL controlling 4-methoxy-indol-3-yl-methyl glucosinolate accumulation, suggesting a biosynthetic connection between 4-hydroxy-indol-3-yl-methyl and 4-methoxy-indol-3-yl-methyl glucosinolate. However, the second QTL for 4-methoxy-indol-3-yl-methyl glucosinolate is located at a different position, near the bottom of chromosome 1, while the second 4-hydroxy-indol-3-yl-methyl glucosinolate QTL maps near the top of chromosome 1.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Is Herbivory by <em>Pieris brassicae</em> Influenced by Variation in Glucosinolates, Myrosinase Activity or Trichomes?</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>QTL for different traits may be considered to co-localize when their 2-LOD support intervals overlap. Co-localization of QTL does, of course, not prove that these QTL are caused by the same gene. Likewise, linked QTL for different traits may have the same cause, even when their 2-LOD support intervals do not overlap, due to the complexity of the statistics that guide QTL mapping. Nonetheless, co-localization of QTL for different traits may provide an approximation for comparing the genetic architecture underlying different but potentially related traits.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>We have identified six QTL for resistance against <em>P. brassicae</em>, each two on chromosomes 1 and 5, and each one on chromosomes 2 and 4. For many of the traits that we investigated because they are known to influence resistance against generalist insect herbivores, we found one or more QTL that co-localize with <em>Pieris</em> resistance QTL. However, one of the three myrosinase activity QTL does not co-localize with a <em>Pieris</em> resistance QTL. Only one of the three QTL for leaf upper-side trichomes, on chromosome 4, co-localizes with a resistance QTL but this QTL shows the wrong sign for its effect. One would expect that trichomes provide physical resistance against insect herbivory and, thereby, reduce plant damage. However, plant damage increases with increasing numbers of leaf trichomes controlled by this locus. None of the leaf perimeter trichome QTL co-localizes with a resistance QTL, and for QTL controlling leaf under-side trichomes, only one of two QTL co-localizes.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>For total glucosinolate accumulation, we identified two QTL, but neither co-localizes with a <em>Pieris</em> resistance QTL. Likewise, none of the QTL for total aliphatic glucosinolate accumulation co-localizes with a resistance QTL, and only two of the five QTL for total indole glucosinolate accumulation co-localize with herbivory QTL. A similar picture emerges for QTL controlling sums of homomethionine, di-homomethionine, penta-homomethionine and hexa-homomethionine-derived glucosinolates or for QTL controlling total methylthio, methylsulfinyl or alkenyl glucosinolates (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). Hence, sum variables for glucosinolate classes do not account for resistance against <em>P. brassicae</em>. But also QTL for individual glucosinolates do not explain the QTL for <em>Pieris</em> resistance. The QTL with the largest impact on glucosinolate profiles, <em>AOP</em>, does not co-localize with any of the resistance QTL. And for all individual glucosinolates, either one or more QTL do not co-localize with herbivory QTL, or at least one QTL has a different sign for its effect than the others, while all <em>Pieris</em> QTL have the same sign for their effect, with the Da(1)-12 allele improving resistance against herbivory by <em>Pieris</em> larvae. Finally, none of the <em>Pieris</em> QTL maps near the two major loci that specify glucosinolate hydrolysis product identity, <em>ESP</em> <a href="#pone.0000578-Lambrix1">[5]</a> and <em>ESM1</em> <a href="#pone.0000578-Zhang1">[8]</a> (<a href="#pone-0000578-g002"><strong>Figure 2</strong></a>), although both parental lines, Da(1)-12 and Ei-2, display sequence polymorphisms in the <em>ESP</em> genomic region that correlate with <em>ESP</em> expression <a href="#pone.0000578-Lambrix1">[5]</a>. Hence, we conclude that none of the investigated traits, trichomes, myrosinase activity or glucosinolate accumulation, appears to cause the QTL for <em>Pieris</em> resistance in Da(1)-12×Ei-2 RILs. Further fine-mapping and, ultimately, cloning of the underlying genes causing the <em>Pieris</em> resistance QTL is necessary to help understand how Arabidopsis and other crucifers defend themselves against specialist insect herbivores such as <em>P. brassicae</em>.</p>


<h4>Prospects of Improving Insect Resistance in Cruciferous Crops</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p><em>Pieris brassicae</em> and other Pieridae are some of the most serious pests on cruciferous crop plants such as rapeseed, cauliflower, or broccoli <a href="#pone.0000578-Bonnemail1">[39]</a>–<a href="#pone.0000578-Hill1">[41]</a>. We have analyzed herbivory by <em>P. brassicae</em> larvae with a new RIL population, obtained from a cross between the parental lines Da(1)-12 and Ei-2. We found no detectable effect of glucosinolates or myrosinase activity on larval herbivory, indicating that the variation in the glucosinolate-myrosinase system that is present in our RIL population does not contribute to variation in plant damage caused by <em>Pieris</em> larvae. Nonetheless, the glucosinolate-myrosinase system does play a role in the interaction between <em>P. brassicae</em> and <em>A. thaliana</em> or other Brassicaceae: Adult <em>Pieris</em> females use glucosinolates and their hydrolysis products to locate host plants for oviposition, and hydrolysis products have a stimulating effect on oviposition for <em>P. brassicae</em> and other Pieridae <a href="#pone.0000578-Raybold1">[24]</a>, <a href="#pone.0000578-vanLoon1">[42]</a>–<a href="#pone.0000578-Miles1">[45]</a>. Likewise, glucosinolate breakdown products serve as a stimulant for larval feeding initiation <a href="#pone.0000578-Schoonhoven1">[46]</a>–<a href="#pone.0000578-Renwick2">[48]</a>. This may explain why herbivory by <em>Pieris rapae</em>, a close relative of <em>P. brassicae</em>, is significantly reduced in <em>tgg1 tgg2</em> double mutants which have very low levels of Arabidopsis wild type myrosinase activity <a href="#pone.0000578-Barth1">[22]</a>. Hence, a reduction of glucosinolate levels or myrosinase activity in cruciferous crops could potentially reduce plant damage caused by <em>P. brassicae</em>. However, a decrease in the effectiveness of the glucosinolate-myrosinase system would very likely render crucifer crops more susceptible to generalist insect herbivores which are sensitive towards glucosinolate-based defenses <a href="#pone.0000578-Lambrix1">[5]</a>–<a href="#pone.0000578-Kliebenstein2">[9]</a>, <a href="#pone.0000578-Barth1">[22]</a>. Furthermore, such a manipulation of the glucosinolate-myrosinase system bears the risk that plants could become more attractive to herbivores which usually do not consume crucifers because these insect species have no effective means to withstand toxic products originating from glucosinolate hydrolysis. Thus, manipulating the glucosinolate-myrosinase system to increase resistance against insect herbivores may be problematic. The detection of QTL that appear to be independent of the glucosinolate-myrosinase system may provide a way to solve this dilemma. Manipulating the genes that underlie the detected resistance QTL could help increase crop protection against <em>P. brassicae</em>, without interfering with a complex defense system that protects crucifers effectively against most herbivorous insects.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plant and Insect Growth Conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>Arabidopsis thaliana</em> plants for RIL development were grown under continuous light, supplied by Osram Fluora L36/W77 neon bulbs with an intensity of 150 µmol s<sup>−1</sup> m<sup>−2</sup> at 20°C and 60% relative humidity in an environment-controlled growth room. Plants for insect herbivory trials, leaf glucosinolate extraction, and myrosinase assays were grown in 11.5 h day/12.5 h night cycles at 22°C and 60% relative humidity (day conditions), and 16°C and 80% relative humidity (night conditions) in an environment-controlled growth room. Here, light was supplied by NH 360 FLX Sunlux ACE bulbs with an intensity of 200 µmol s<sup>−1</sup> m<sup>−2</sup>. Plants were grown in an autoclaved 1:3 vermiculate/potting soil mix with 20 ml time-release fertilizer (Osmocote) per flat. After sowing into damp potting medium, flats were covered with clear plastic grow domes, and seeds were stratified for 3–4 days at 6°C in the dark. In general, seeds germinated within 2–3 days, and grow domes were removed 5 days after transfer to the light. Then seedlings were transferred to fresh soil in 96-celled 32.5×51 cm2 flats at a density of 1 seedling per cell. All assays were carried out with 3-week old plants.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p><em>Pieris brassicae</em> eggs were obtained from Seritech (Warwick, UK). After hatching, insects were reared on <em>Brassica napus var. oleifera</em> for 2–3 days before the experiments. <em>Plutella xylostella</em> eggs were obtained from New York State Agricultural Experiment Station Geneva, NY, USA, and a colony was maintained at the Max Planck Institute for Chemical Ecology, Jena, Germany. Insects were raised on artificial diet according to published procedures <a href="#pone.0000578-Shelton1">[49]</a>.</p>


<h4>Generation of Da(1)-12×Ei-2 Recombinant Inbred Lines</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Da(1)-12 (accession no. N917) and Ei-2 (N1124) accessions were obtained from the Arabidopsis stock center (Nottingham, U.K.). Except for the initial cross between both accessions, all following generations were propagated by selfing. 215 F<sub>2</sub> plants were randomly selected from the progeny of a single heterozygous F<sub>1</sub> plant. For every advanced generation, 4–8 seeds per line were planted, and a single plant was randomly chosen from each line for seed production. Finally, seeds from a single F<sub>8</sub> plant per line were bulk-collected resulting in a final set of 201 Da(1)-12×Ei-2 RILs. This new RIL population will be made available through the Arabidopsis stock centers.</p>


<h4>DNA Extraction, Genotyping, Genetic Map</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>DNA was isolated as described in <a href="#pone.0000578-Kroymann2">[17]</a>. Molecular markers were obtained from publicly available sources (<a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>; <a href="http://www.inra.fr/qtlat">http://www.inra.fr/qtlat</a>, <a href="#pone.0000578-Shelton1">[49]</a>), or were generated from microsatellite loci identified in the Col-0 genome sequence <a href="#pone.0000578-Arabidopsis1">[51]</a>. More than 150 potential markers were tested for polymorphism between parental lines, using DNA from Da(1)-12, Ei-2, and a 1:1 mixture of DNAs from both lines. 84 PCR products were found suitable for genotyping (<a href="#pone-0000578-t002"><strong>Table 2</strong></a>), and allowed to distinguish between Da(1)-12 and Ei-2 genotypes, and between homozygous and heterozygous loci on 4% Metaphor agarose (Biozym diagnostics, Germany). PCR reactions contained, in general, ca. 30 ng DNA, 2.3 µl 10x PCR buffer (Qiagen, Germany), 4 nmol of each dNTP, 1.25 pmol of each of both primers, 70 nmol MgCl<sub>2</sub>, and 0.15 U <em>Taq</em> DNA polymerase (Qiagen, Germany) in a 23 µl volume. Cycling conditions were 94°C for 2 min, followed by 38 cycles of 94°C for 15 s, 50°C or 55°C for 15 s, and 72°C for 30 s, with a final extension of 72°C for 2 min on an Applied Biosystems 9700 Thermocycler. Genotyping was carried out with DNA extracted from individual F<sub>9</sub> progeny. A genetic map was constructed with MAPMAKER/EXP Version 3.0 <a href="#pone.0000578-Lander1">[52]</a>. Genotype data for the Da(1)-12×Ei-2 RILs are available in <a href="#pone.0000578.s001"><strong>Table S1</strong></a>.</p>
<div class="figure" id="pone-0000578-t002"><div class="img"><a name="pone-0000578-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000578" data-uri="info:doi/10.1371/journal.pone.0000578.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000578.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000578.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000578.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000578.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000578.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Markers used for genotyping of Da(1)-12×Ei-2 RILs</span></strong></p><span>doi:10.1371/journal.pone.0000578.t002</span></div>

<h4>Insect Herbivory Screens</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p><em>P. brassicae</em> herbivory screens were performed at 7 different times, and each experiment was carried out with at least 3 replicates per RIL. RIL replicates were completely randomized over 96-celled flats. At the beginning of each experiment, plant diameter was measured. One larva was placed on each plant rosette without prior starvation, and larvae were allowed to move freely. After 24 hours of herbivory, the leaf area removed by the insects was assessed visually, and an artificial scale was established to determine the percentage of removed rosette tissue. In total, more than 9000 data points were collected for <em>P. brassicae</em> herbivory. <em>P. xylostella</em> herbivory screens were performed similarly, except that larvae were starved for 6 hours prior transfer to plants, and larvae were allowed to feed for 2–3 days. Total sample size for <em>P. xylostella</em> herbivory exceeded 2400 plants.</p>


<h4>Trichomes</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Trichome analysis was carried out with 96 RILs chosen to include lines with a maximum number of recombination breakpoints. Per RIL, 4 replicates were analyzed, and trichomes from the 3<sup>rd</sup> to 6<sup>th</sup> true leaves of 3-week old plants were counted with a Stemi SV6 binocular (Carl Zeiss, Jena, Germany). Every leaf was placed underneath the binocular such that the leaf tip touched the border of the visual field. This way, only the upper half of the leaf was visible and used for trichome analysis. Trichome numbers were counted for the leaf upper and under-sides and for the leaf perimeter.</p>


<h4>Glucosinolate Extraction and HPLC Analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Glucosinolates were extracted in a 96-well format as described in <a href="#pone.0000578-Kliebenstein5">[18]</a>. HPLC separation and identification of extracted desulfo-glucosinolates was carried out as described in <a href="#pone.0000578-Kroymann2">[17]</a>. In brief, HPLCs were run on a Hewlett Packard HP 1100 system (Agilent), equipped with a HP Lichrocart 250-4 RP18e 5 µm column. The elution was accomplished with a water (solvent A) – acetonitrile (solvent B) gradient using the following program: 1.5 – 5% (v/v) B (6 min), 5 – 7% B (2 min), 7 – 21% B (10 min), 21 – 29% B (5 min), 29 – 43% B (7 min), 43 – 92% B (0.5 min), 92% B (2.5 min), 92 – 1.5% B (0.5 min), 1.5% B (4.5 min). Desulfo-glucosinolates were identified according to retention time and UV spectra, and quantified from HPLC peak areas at A229 nm, using published response factors <a href="#pone.0000578-Buchner1">[53]</a>, <a href="#pone.0000578-Brown1">[54]</a> to correct for different UV absorption capacities of individual glucosinolates.</p>


<h4>Myrosinase Extraction and Analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Myrosinase extraction from 100 mg leaf tissue and UV-spectrophotometric activity assays were carried out as described in <a href="#pone.0000578-Barth1">[22]</a>. Two independent experiments were conducted, once with the complete RIL population, once with the 96 most informative RILs. Relative myrosinase activity was measured as a spectrophotometrical change at 227 nm through breakdown of sinigrin (allyl glucosinolate) within 15 min using a Multiskan Spectrum (Thermo Fisher Scientific, Germany) spectrophotometric plate reader.</p>


<h4>Quantitative Analyses</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Systat Version 10 (SPSS Inc.) was used to analyze natural variation for insect resistance and related traits in the following 16 Arabidopsis accessions: Akita (Akita, Japan, Versailles identification no. 252 AV), Col-0, Ct-1 (Catania, Italy, N1094), Cvi-0 (Cape Verdi Islands, N902), Da(1)-12 (Czech Republic or Slowakia, N917), Edi-0 (Edinburgh, UK, N1122), Ei-2 (Eifel, Germany, N1124), Kn-0 (Kaunas, Lithuania, N1286), L<em>er</em>, Shahdara (Shahdara river (Pamir), Tadjikistan, N929), Mh-1 (Muehlen, Poland, N1368), Mt-0 (Martuba/Cyrenaika, Lybia, N1380), N13 (Konchezero, Russia, CS22491), Oy-0 (Oystese, Norway, N1436), Stw-0 (Stobowa/Orel, Russia, N1538), and Tsu-0 (Tsu, Japan, N1564). ANOVA was used to obtain least squares means for each accession for <em>P. brassicae</em> and <em>P. xylostella</em> herbivory, for plant diameter in <em>P. brassicae</em> herbivory experiments, for myrosinase activity, and for trichomes on the leaf upper and under-side surfaces and the leaf perimeter. For <em>P. brassicae</em> (N = 973) and <em>P. xylostella</em> herbivory (<em>N</em> = 912), the ANOVA model was TRAIT = CONSTANT+ACCESSION+EXPERIMENT+FLAT(EXPER​IMENT)+COLUMN+ROW+PLANTDIAMETER. COLUMN and ROW are variables to control for position effects. These variables are particularly important in <em>P. brassicae</em> herbivory screens to compensate for larval movement during experiments (<em>F</em> = 19.18, <em>df</em> = 11, <em>N</em> = 973, <em>P</em>&lt;0.000001 for COLUMN, and <em>F</em> = 41.67, <em>df</em> = 7, <em>P</em>&lt;0.000001 for ROW). EXPERIMENT accounts for variation between experiment replicates, FLAT(EXPERIMENT) for variation between flats within an experiment. Similarly, least squares means were obtained for plant diameter in <em>P. brassicae</em> (<em>N</em> = 973) and <em>P. xylostella</em> (<em>N</em> = 912) herbivory screens with the model TRAIT = CONSTANT+ACCESSION+EXPERIMENT+FLAT(EXPER​IMENT)+COLUMN+ROW.Myrosinase activity (<em>N</em> = 68), glucosinolates (<em>N</em> = 83) and leaf trichome density (<em>N</em> = 153) were evaluated in only one experiment each and all accessions were grown completely randomized within one flat. Therefore, the model was TRAIT = CONSTANT+ACCESSION+COLUMN+ROW to obtain least squares means for myrosinase activity. In this model, COLUMN and ROW are variables used to control for the time delay that occurs during the processing of individual microtiter plate positions in our plate reader. The model for the analysis of all individual glucosinolates was TRAIT = CONSTANT+ACCESSION. Glucosinolate sum variables were generated by summing up least squares means for individual glucosinolates after correcting for different UV absorption capacities with published response factors <a href="#pone.0000578-Lander1">[52]</a>, <a href="#pone.0000578-Buchner1">[53]</a>. Finally, least squares means for trichome density on the leaf upper and under-sides and the leaf perimeter were obtained with the model TRAIT = CONSTANT+ACCESSION+LEAF, with LEAF being a variable to control for a potential effect of leaf developmental stage on trichome density. For the Da(1)-12×Ei-2 RIL population, similar ANOVA models were used as described above, with a few exceptions: A variable FLAT was included in the glucosinolate and trichome models to account for variation between flats, and the variables EXPERIMENT and FLAT(EXPERIMENT) were included to control for variation between experimental replicates and between flats within an experiment replicate in the myrosinase assays. Sample sizes were <em>N</em> = 9132 for <em>P. brassicae</em> and <em>N</em> = 2441 for <em>P. xylostella</em> herbivory screens, <em>N</em> = 930 for myrosinase activity assays, <em>N</em> = 1484 for trichome density, and <em>N</em> = 972 for glucosinolates. Again, data from <em>P. brassicae</em> herbivory screens were also used to analyze variation in plant diameter.</p>


<h4>QTL Mapping and Analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Windows QTL cartographer V2.5 <a href="#pone.0000578-Wang1">[55]</a> was used for composite interval mapping (CIM) of QTL. The standard model (Model 6) was used with forward regression, a window size of 10 cM, and 5 background control markers. QTL were scanned at a walk speed of 0.5 cM. Statistical significance of QTL for each trait was assessed by permuting each data set 1000 times, with a significance level of 0.05. Furthermore, 2-LOD support intervals <a href="#pone.0000578-vanOoijen1">[56]</a> were obtained from the QTL cartographer output. For each QTL, the effect strength was estimated as the proportional difference, (LSM<sub>Da(1)-12</sub>−LSM<sub>Ei-2</sub>)/((LSM<sub>Da(1)-12</sub>+LSM<sub>Ei-2</sub>)/2), where LSM is the ANOVA least squares mean at the marker closest to the QTL peak. Positive values indicate that the Da(1)-12 genotype has a stronger effect, negative values that Ei-2 has a stronger effect. Finally, the proportion of explained variance, <em>R</em><sup>2</sup>, was obtained from the QTL cartographer output.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>In addition, Bayesian interval mapping (BIM) <a href="#pone.0000578-Satagopan1">[57]</a> as implemented in R/bim (<a href="http://www.stat.wisc.edu/%E2%88%BCyandell/qtl/software/bmqtl">http://www.stat.wisc.edu/~yandell/qtl/so​ftware/bmqtl</a>) was used. For each trait, 400,000 Markov-Chain-Monte-Carlo steps were simulated and iterations were recorded at every 400<sup>th</sup> step, with 1000 pre-burn-in and 20000 burn-in steps. Prior for the number of QTL was Poisson, with zero initial QTL.</p>
<a id="article1.body1.sec4.sec9.p3" name="article1.body1.sec4.sec9.p3"></a><p>CIM and BIM yielded very similar results, and the high density probability peak from BIM for a given QTL was usually found within the 2-LOD support interval for a QTL in CIM (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>). The most notable difference between both methods was a weak QTL for <em>Plutella</em> herbivory that was identified with BIM but not with CIM. Since this QTL was located at the same position as one of the QTL for <em>Pieris</em> herbivory, we consider this QTL real (<a href="#pone-0000578-g003"><strong>Figure 3</strong></a>).</p>
<a id="article1.body1.sec4.sec9.p4" name="article1.body1.sec4.sec9.p4"></a><p>Because epistatic interactions appear to be an important factor in the genetic architecture of complex traits <a href="#pone.0000578-Kroymann3">[58]</a> and have been documented for glucosinolate biosynthesis <a href="#pone.0000578-Kliebenstein3">[12]</a>, the markers most closely linked to QTL peaks were tested for potential epistatic interactions. Based on this <em>a priori</em> expectation that main QTL might interact with one another, a significance threshold of 0.05 was chosen. First, for every trait all single markers and all pairwise interactions between these markers were included in the ANOVA models. If more than one significant interaction with a particular marker was detected, also higher-order interaction terms between markers were included.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000578"></div><a name="pone.0000578.s001" id="pone.0000578.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000578.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Genotype data for Da(1)-12×Ei-2 RILs</p>
<p class="siDoi">doi:10.1371/journal.pone.0000578.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.26 MB XLS)</p>
<a name="pone.0000578.s002" id="pone.0000578.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000578.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Genes of the glucosinolate-myrosinase system, AGI numbers, and references</p>
<p class="siDoi">doi:10.1371/journal.pone.0000578.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The cross between Da(1)-12 and Ei-2 parental lines was carried out by Daniel J. Kliebenstein. We thank Andreas Weber and colleagues for help with RIL development, and Petra Eißmann, Andrea Hirsch, Tabea Birk, and Kathrin Eberhardt for technical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JK HV MP. Performed the experiments: MP. Analyzed the data: JK MP. Contributed reagents/materials/analysis tools: TM. Wrote the paper: JK MP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000578-Koornneef1" id="pone.0000578-Koornneef1"></a>Koornneef M, Alonso-Blanco C, Vreugdenhill D (2004) Naturally occurring genetic variation in <em>Arabidopsis thaliana</em>. Annu Rev Plant Biol  55: 141–172.  <ul class="find" data-citedArticleID="963086" data-doi="10.1146/annurev.arplant.55.031903.141605"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.55.031903.141605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Naturally+occurring+genetic+variation+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Naturally+occurring+genetic+variation+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000578-Lister1" id="pone.0000578-Lister1"></a>Lister C, Dean C (1993) Recombinant inbred lines for mapping RFLP and phenotypic markers. Plant J  4: 745–750.  <ul class="find" data-citedArticleID="963100" data-doi="10.1046/j.1365-313x.1993.04040745.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1993.04040745.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombinant+inbred+lines+for+mapping+RFLP+and+phenotypic+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombinant+inbred+lines+for+mapping+RFLP+and+phenotypic+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000578-AlonsoBlanco1" id="pone.0000578-AlonsoBlanco1"></a>Alonso-Blanco C, Peeters AJM, Koornneef M, Lister C, Dean C, et al.  (1998) Development of an AFLP based linkage map of L<em>er</em>, Col and Cvi <em>Arabidopsis thaliana</em> ecotypes and construction of a L<em>er</em>/Cvi recombinant inbred line population. Plant J  14: 259–271.  <ul class="find" data-citedArticleID="963042" data-doi="10.1046/j.1365-313x.1998.00115.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1998.00115.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+an+AFLP+based+linkage+map+of+Ler%2C+Col+and+Cvi+Arabidopsis+thaliana+ecotypes+and+construction+of+a+Ler%2FCvi+recombinant+inbred+line+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+an+AFLP+based+linkage+map+of+Ler%2C+Col+and+Cvi+Arabidopsis+thaliana+ecotypes+and+construction+of+a+Ler%2FCvi+recombinant+inbred+line+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000578-Jander1" id="pone.0000578-Jander1"></a>Jander G, Cui J, Nhan B, Pierce NE, Ausubel FM (2001) The TASTY locus on chromosome 1 of Arabidopsis affects feeding of the insect herbivore <em>Trichoplusia ni</em>. Plant Physiol  126: 890–898.  <ul class="find" data-citedArticleID="963074" data-doi="10.1104/pp.126.2.890"><li><a href="http://dx.doi.org/10.1104/pp.126.2.890" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+TASTY+locus+on+chromosome+1+of+Arabidopsis+affects+feeding+of+the+insect+herbivore+Trichoplusia+ni." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+TASTY+locus+on+chromosome+1+of+Arabidopsis+affects+feeding+of+the+insect+herbivore+Trichoplusia+ni.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000578-Lambrix1" id="pone.0000578-Lambrix1"></a>Lambrix V, Reichelt M, Mitchell-Olds T, Kliebenstein DJ, Gershenzon J (2001) The Arabidopsis epithiospecifier protein promotes the hydrolysis of glucosinolates to nitriles and influences <em>Trichoplusia ni</em> herbivory. Plant Cell  13: 2793–2807.  <ul class="find" data-citedArticleID="963096" data-doi="10.2307/3871535"><li><a href="http://dx.doi.org/10.2307/3871535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+epithiospecifier+protein+promotes+the+hydrolysis+of+glucosinolates+to+nitriles+and+influences+Trichoplusia+ni+herbivory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+epithiospecifier+protein+promotes+the+hydrolysis+of+glucosinolates+to+nitriles+and+influences+Trichoplusia+ni+herbivory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000578-Kliebenstein1" id="pone.0000578-Kliebenstein1"></a>Kliebenstein DJ, Pedersen D, Barker B, Mitchell-Olds T (2002) Comparative analysis of quantitative trait loci controlling glucosinolates, myrosinase and insect resistance in <em>Arabidopsis thaliana</em>. Genetics  161: 325–332.  <ul class="find" data-citedArticleID="963076"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana.&amp;auth=&amp;atitle=Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000578-Kroymann1" id="pone.0000578-Kroymann1"></a>Kroymann J, Donnerhacke S, Schnabelrauch D, Mitchell-Olds T (2003) Evolutionary dynamics of an Arabidopsis insect resistance quantitative trait locus. Proc Natl Acad Sci USA  100: 14587–14592.  <ul class="find" data-citedArticleID="963090" data-doi="10.1073/pnas.1734046100"><li><a href="http://dx.doi.org/10.1073/pnas.1734046100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+an+Arabidopsis+insect+resistance+quantitative+trait+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+an+Arabidopsis+insect+resistance+quantitative+trait+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000578-Zhang1" id="pone.0000578-Zhang1"></a>Zhang Z, Ober JA, Kliebenstein DJ (2006) The gene controlling the quantitative trait locus <em>EPITHIOSPECIFIER MODIFIER1</em> alters glucosinolate hydrolysis and insect resistance in <em>Arabidopsis</em>. Plant Cell  18: 1524–1536.  <ul class="find" data-citedArticleID="963150" data-doi="10.1105/tpc.105.039602"><li><a href="http://dx.doi.org/10.1105/tpc.105.039602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+gene+controlling+the+quantitative+trait+locus+EPITHIOSPECIFIER+MODIFIER1+alters+glucosinolate+hydrolysis+and+insect+resistance+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+gene+controlling+the+quantitative+trait+locus+EPITHIOSPECIFIER+MODIFIER1+alters+glucosinolate+hydrolysis+and+insect+resistance+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000578-Kliebenstein2" id="pone.0000578-Kliebenstein2"></a>Kliebenstein DJ, Kroymann J, Mitchell-Olds T (2005) The glucosinolate-myrosinase system in an ecological and evolutionary context. Curr Opin Plant Biol  8: 264–271.  <ul class="find" data-citedArticleID="963078" data-doi="10.1016/j.pbi.2005.03.002"><li><a href="http://dx.doi.org/10.1016/j.pbi.2005.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+glucosinolate-myrosinase+system+in+an+ecological+and+evolutionary+context." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+glucosinolate-myrosinase+system+in+an+ecological+and+evolutionary+context.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000578-Grubb1" id="pone.0000578-Grubb1"></a>Grubb CD, Abel S (2006) Glucosinolate metabolism and its control. Trends Plant Sci  11: 89–100.  <ul class="find" data-citedArticleID="963062" data-doi="10.1016/j.tplants.2005.12.006"><li><a href="http://dx.doi.org/10.1016/j.tplants.2005.12.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glucosinolate+metabolism+and+its+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glucosinolate+metabolism+and+its+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000578-Halkier1" id="pone.0000578-Halkier1"></a>Halkier BA, Gershenzon J (2006) Biology and biochemistry of glucosinolates. Annu Rev Plant Biol  57: 303–333.  <ul class="find" data-citedArticleID="963064" data-doi="10.1146/annurev.arplant.57.032905.105228"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biology+and+biochemistry+of+glucosinolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biology+and+biochemistry+of+glucosinolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000578-Kliebenstein3" id="pone.0000578-Kliebenstein3"></a>Kliebenstein DJ, Gershenzon J, Mitchell-Olds T (2001) Comparative quantitative trait loci mapping of aliphatic, indolic and benzylic glucosinolate production in <em>Arabidopsis thaliana</em> leaves and seeds. Genetics  159: 359–370.  <ul class="find" data-citedArticleID="963080"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds.&amp;auth=&amp;atitle=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000578-Mithen1" id="pone.0000578-Mithen1"></a>Mithen R, Clarke J, Lister C, Dean C (1995) Genetics of aliphatic glucosinolates: III. Side chain structure of aliphatic glucosinolates in <em>Arabidopsis thaliana</em>. Heredity  74: 210–215.  <ul class="find" data-citedArticleID="963116" data-doi="10.1038/hdy.1995.29"><li><a href="http://dx.doi.org/10.1038/hdy.1995.29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+aliphatic+glucosinolates%3A+III.+Side+chain+structure+of+aliphatic+glucosinolates+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+aliphatic+glucosinolates%3A+III.+Side+chain+structure+of+aliphatic+glucosinolates+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000578-Kliebenstein4" id="pone.0000578-Kliebenstein4"></a>Kliebenstein DJ, Lambrix VM, Reichelt M, Gershenzon J, Mitchell-Olds T (2001) Gene duplication and the diversification of secondary metabolism: side chain modification of glucosinolates in <em>Arabidopsis thaliana</em>. Plant Cell  13: 681–693.  <ul class="find" data-citedArticleID="963082" data-doi="10.2307/3871415"><li><a href="http://dx.doi.org/10.2307/3871415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+duplication+and+the+diversification+of+secondary+metabolism%3A+side+chain+modification+of+glucosinolates+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+duplication+and+the+diversification+of+secondary+metabolism%3A+side+chain+modification+of+glucosinolates+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000578-Magrath1" id="pone.0000578-Magrath1"></a>Magrath R, Bano F, Morgner M, Parkin I, Sharpe A, et al.  (1994) Genetics of aliphatic glucosinolates: I. Side chain elongation in <em>Brassica napus</em> and <em>Arabidopsis thaliana</em>. Heredity  72: 290–299.  <ul class="find" data-citedArticleID="963104" data-doi="10.1038/hdy.1994.39"><li><a href="http://dx.doi.org/10.1038/hdy.1994.39" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+aliphatic+glucosinolates%3A+I.+Side+chain+elongation+in+Brassica+napus+and+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+aliphatic+glucosinolates%3A+I.+Side+chain+elongation+in+Brassica+napus+and+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000578-CamposdeQuiros1" id="pone.0000578-CamposdeQuiros1"></a>Campos de Quiros H, Magrath R, McCallum D, Kroymann J, Schnabelrauch D, et al.  (2000) α-Keto acid elongation and glucosinolate biosynthesis in <em>Arabidopsis thaliana</em>. Theor Appl Genet  101: 429–437.  <ul class="find" data-citedArticleID="963056" data-doi="10.1007/s001220051500"><li><a href="http://dx.doi.org/10.1007/s001220051500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%26%23x3b1%3B-Keto+acid+elongation+and+glucosinolate+biosynthesis+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%26%23x3b1%3B-Keto+acid+elongation+and+glucosinolate+biosynthesis+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000578-Kroymann2" id="pone.0000578-Kroymann2"></a>Kroymann J, Textor S, Tokuhisa JG, Falk KL, Bartram S, et al.  (2001) A gene controlling variation in <em>Arabidopsis thaliana</em> glucosinolate composition is part of the methionine chain elongation pathway. Plant Physiol  127: 1077–1088.  <ul class="find" data-citedArticleID="963092" data-doi="10.1104/pp.010416"><li><a href="http://dx.doi.org/10.1104/pp.010416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+controlling+variation+in+Arabidopsis+thaliana+glucosinolate+composition+is+part+of+the+methionine+chain+elongation+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+controlling+variation+in+Arabidopsis+thaliana+glucosinolate+composition+is+part+of+the+methionine+chain+elongation+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000578-Kliebenstein5" id="pone.0000578-Kliebenstein5"></a>Kliebenstein DJ, Kroymann J, Brown P, Figuth A, Pedersen D, et al.  (2001) Genetic control of natural variation in Arabidopsis glucosinolate accumulation. Plant Physiol  126: 811–825.  <ul class="find" data-citedArticleID="963084" data-doi="10.1104/pp.126.2.811"><li><a href="http://dx.doi.org/10.1104/pp.126.2.811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+control+of+natural+variation+in+Arabidopsis+glucosinolate+accumulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+control+of+natural+variation+in+Arabidopsis+glucosinolate+accumulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000578-Koroleva1" id="pone.0000578-Koroleva1"></a>Koroleva OA, Davies A, Deeken R, Thorpe MR, Tomos AD, et al.  (2000) Identification of a new glucosinolate-rich cell type in Arabidopsis flower stalk. Plant Physiol  124: 599–608.  <ul class="find" data-citedArticleID="963088" data-doi="10.1104/pp.124.2.599"><li><a href="http://dx.doi.org/10.1104/pp.124.2.599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+new+glucosinolate-rich+cell+type+in+Arabidopsis+flower+stalk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+new+glucosinolate-rich+cell+type+in+Arabidopsis+flower+stalk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000578-Husebye1" id="pone.0000578-Husebye1"></a>Husebye H, Chadchawan S, Winge P, Thangstad OP, Bones AM (2002) Guard cell- and phloem idioblast-specific expression of thioglucoside glucohydrolase 1 (myrosinase) in Arabidopsis. Plant Physiol  128: 1180–1188.  <ul class="find" data-citedArticleID="963072" data-doi="10.1104/pp.010925"><li><a href="http://dx.doi.org/10.1104/pp.010925" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guard+cell-+and+phloem+idioblast-specific+expression+of+thioglucoside+glucohydrolase+1+%28myrosinase%29+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Guard+cell-+and+phloem+idioblast-specific+expression+of+thioglucoside+glucohydrolase+1+%28myrosinase%29+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000578-Thangstad1" id="pone.0000578-Thangstad1"></a>Thangstad OP, Gilde B, Chadchawan S, Seem M, Husebye H, et al.  (2004) Cell specific cross-species expression of myrosinases in <em>Brassica napus, Arabidopsis thaliana</em> and <em>Nicotiana tabacum</em>. Plant Mol Biol  54: 597–611.  <ul class="find" data-citedArticleID="963138" data-doi="10.1023/b:plan.0000038272.99590.10"><li><a href="http://dx.doi.org/10.1023/b:plan.0000038272.99590.10" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+specific+cross-species+expression+of+myrosinases+in+Brassica+napus%2C+Arabidopsis+thaliana+and+Nicotiana+tabacum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+specific+cross-species+expression+of+myrosinases+in+Brassica+napus%2C+Arabidopsis+thaliana+and+Nicotiana+tabacum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000578-Barth1" id="pone.0000578-Barth1"></a>Barth C, Jander G (2006) Arabidopsis myrosinases TGG1 and TGG2 have redundant function in glucosinolate breakdown and insect defense. Plant J  46: 549–562.  <ul class="find" data-citedArticleID="963046" data-doi="10.1111/j.1365-313x.2006.02716.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02716.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+myrosinases+TGG1+and+TGG2+have+redundant+function+in+glucosinolate+breakdown+and+insect+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+myrosinases+TGG1+and+TGG2+have+redundant+function+in+glucosinolate+breakdown+and+insect+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000578-Wittstock1" id="pone.0000578-Wittstock1"></a>Wittstock U, Halkier BA (2002) Glucosinolate research in the <em>Arabidopsis</em> era. Trends Plant Sci  7: 263–270.  <ul class="find" data-citedArticleID="963144" data-doi="10.1016/s1360-1385(02)02273-2"><li><a href="http://dx.doi.org/10.1016/s1360-1385(02)02273-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glucosinolate+research+in+the+Arabidopsis+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glucosinolate+research+in+the+Arabidopsis+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000578-Raybold1" id="pone.0000578-Raybold1"></a>Raybold AF, Moyes CL (2001) The ecological genetics of aliphatic glucosinolates. Heredity  87: 383–391.  <ul class="find" data-citedArticleID="963120" data-doi="10.1046/j.1365-2540.2001.00954.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2540.2001.00954.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ecological+genetics+of+aliphatic+glucosinolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ecological+genetics+of+aliphatic+glucosinolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000578-Ratzka1" id="pone.0000578-Ratzka1"></a>Ratzka A, Vogel H, Kliebenstein DJ, Mitchell-Olds T, Kroymann J (2002) Disarming the mustard oil bomb. Proc Natl Acad Sci USA  99: 11223–11228.  <ul class="find" data-citedArticleID="963118" data-doi="10.1073/pnas.172112899"><li><a href="http://dx.doi.org/10.1073/pnas.172112899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disarming+the+mustard+oil+bomb." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disarming+the+mustard+oil+bomb.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000578-Wittstock2" id="pone.0000578-Wittstock2"></a>Wittstock U, Agerbirk N, Stauber EJ, Olsen CE, Hippler M, et al.  (2004) Successful hervivore attack due to metabolic diversion of a plant chemical defense. Proc Natl Acad Sci USA  101: 4859–4864.  <ul class="find" data-citedArticleID="963146" data-doi="10.1073/pnas.0308007101"><li><a href="http://dx.doi.org/10.1073/pnas.0308007101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Successful+hervivore+attack+due+to+metabolic+diversion+of+a+plant+chemical+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Successful+hervivore+attack+due+to+metabolic+diversion+of+a+plant+chemical+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000578-Benderoth1" id="pone.0000578-Benderoth1"></a>Benderoth M, Textor S, Windsor AJ, Mitchell-Olds T, Gershenzon J, et al.  (2006) Positive selection driving diversification in plant secondary metabolism. Proc Natl Acad Sci USA  103: 9118–9123.  <ul class="find" data-citedArticleID="963048" data-doi="10.1073/pnas.0601738103"><li><a href="http://dx.doi.org/10.1073/pnas.0601738103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+selection+driving+diversification+in+plant+secondary+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+selection+driving+diversification+in+plant+secondary+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000578-Tuinstra1" id="pone.0000578-Tuinstra1"></a>Tuinstra MR, Ejeta G, Goldsbrough PB (1997) Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci. Theor Appl Genet  95: 1005–1011.  <ul class="find" data-citedArticleID="963140" data-doi="10.1007/s001220050654"><li><a href="http://dx.doi.org/10.1007/s001220050654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterogeneous+inbred+family+%28HIF%29+analysis%3A+a+method+for+developing+near-isogenic+lines+that+differ+at+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterogeneous+inbred+family+%28HIF%29+analysis%3A+a+method+for+developing+near-isogenic+lines+that+differ+at+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000578-Mauricio1" id="pone.0000578-Mauricio1"></a>Mauricio R, Rausher MD (1997) Experimental manipulation of putative selective agents provides evidence for the role of natural enemies in the evolution of plant defense. Evolution  51: 1435–1444.  <ul class="find" data-citedArticleID="963106" data-doi="10.2307/2411196"><li><a href="http://dx.doi.org/10.2307/2411196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+manipulation+of+putative+selective+agents+provides+evidence+for+the+role+of+natural+enemies+in+the+evolution+of+plant+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+manipulation+of+putative+selective+agents+provides+evidence+for+the+role+of+natural+enemies+in+the+evolution+of+plant+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000578-Mauricio2" id="pone.0000578-Mauricio2"></a>Mauricio R (1998) Costs of resistance to natural enemies in field populations of the annual plant, <em>Arabidopsis thaliana</em>. Am Nat  151: 20–28.  <ul class="find" data-citedArticleID="963108" data-doi="10.1086/286099"><li><a href="http://dx.doi.org/10.1086/286099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Costs+of+resistance+to+natural+enemies+in+field+populations+of+the+annual+plant%2C+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Costs+of+resistance+to+natural+enemies+in+field+populations+of+the+annual+plant%2C+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000578-Handley1" id="pone.0000578-Handley1"></a>Handley R, Ekbom B, Ågren J (2005) Variation in trichome density and resistance against a specialist insect herbivore in natural populations of <em>Arabidopsis thaliana</em>. Ecol Entomol  30: 284–292.  <ul class="find" data-citedArticleID="963066" data-doi="10.1111/j.0307-6946.2005.00699.x"><li><a href="http://dx.doi.org/10.1111/j.0307-6946.2005.00699.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+trichome+density+and+resistance+against+a+specialist+insect+herbivore+in+natural+populations+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+trichome+density+and+resistance+against+a+specialist+insect+herbivore+in+natural+populations+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000578-Symonds1" id="pone.0000578-Symonds1"></a>Symonds VV, Godoy AV, Alconada T, Botto JF, Juenger TE, et al.  (2005) Mapping of quantitative trait loci in multiple populations of <em>Arabidopsis thaliana</em> identifies natural allelic variation for trichome density. Genetics  169: 1649–1658.  <ul class="find" data-citedArticleID="963136" data-doi="10.1534/genetics.104.031948"><li><a href="http://dx.doi.org/10.1534/genetics.104.031948" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+quantitative+trait+loci+in+multiple+populations+of+Arabidopsis+thaliana+identifies+natural+allelic+variation+for+trichome+density." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+quantitative+trait+loci+in+multiple+populations+of+Arabidopsis+thaliana+identifies+natural+allelic+variation+for+trichome+density.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000578-Mauricio3" id="pone.0000578-Mauricio3"></a>Mauricio R (2005) Ontogenetics of QTL: the genetic architecture of trichome density over time in <em>Arabidopsis thaliana</em>. Genetica  123: 75–85.  <ul class="find" data-citedArticleID="963110" data-doi="10.1007/s10709-002-2714-9"><li><a href="http://dx.doi.org/10.1007/s10709-002-2714-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ontogenetics+of+QTL%3A+the+genetic+architecture+of+trichome+density+over+time+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ontogenetics+of+QTL%3A+the+genetic+architecture+of+trichome+density+over+time+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000578-MitchellOlds1" id="pone.0000578-MitchellOlds1"></a>Mitchell-Olds T, Pedersen D (1998) The molecular basis of quantitative genetic variation in central and secondary metabolism in Arabidopsis. Genetics  149: 739–747.  <ul class="find" data-citedArticleID="963114"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+molecular+basis+of+quantitative+genetic+variation+in+central+and+secondary+metabolism+in+Arabidopsis.&amp;auth=&amp;atitle=The+molecular+basis+of+quantitative+genetic+variation+in+central+and+secondary+metabolism+in+Arabidopsis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+basis+of+quantitative+genetic+variation+in+central+and+secondary+metabolism+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+basis+of+quantitative+genetic+variation+in+central+and+secondary+metabolism+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000578-Chadchawan1" id="pone.0000578-Chadchawan1"></a>Chadchawan S, Bishop J, Thangstad OP, Bones AM, Mitchell-Olds T, et al.  (1993) Arabidopsis cDNA sequence encoding myrosinase. Plant Physiol  103: 671–672.  <ul class="find" data-citedArticleID="963058" data-doi="10.1104/pp.103.2.671"><li><a href="http://dx.doi.org/10.1104/pp.103.2.671" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+cDNA+sequence+encoding+myrosinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+cDNA+sequence+encoding+myrosinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000578-Xue1" id="pone.0000578-Xue1"></a>Xue J, Jorgensen M, Pihlgren U, Rask L (1995) The myrosinase gene family in <em>Arabidopsis thaliana</em>: gene organization, expression and evolution. Plant Mol Biol  27: 911–922.  <ul class="find" data-citedArticleID="963148" data-doi="10.1007/bf00037019"><li><a href="http://dx.doi.org/10.1007/bf00037019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+myrosinase+gene+family+in+Arabidopsis+thaliana%3A+gene+organization%2C+expression+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+myrosinase+gene+family+in+Arabidopsis+thaliana%3A+gene+organization%2C+expression+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000578-Zhang2" id="pone.0000578-Zhang2"></a>Zhang J, Pontoppidan B, Xue J, Rask L, Meijer J (2002) The third myrosinase gene <em>TGG3</em> in <em>Arabidopsis thaliana</em> is a pseudogene specifically expressed in stamen and petal. Physiol Plant  115: 25–34.  <ul class="find" data-citedArticleID="963152" data-doi="10.1034/j.1399-3054.2002.1150103.x"><li><a href="http://dx.doi.org/10.1034/j.1399-3054.2002.1150103.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+third+myrosinase+gene+TGG3+in+Arabidopsis+thaliana+is+a+pseudogene+specifically+expressed+in+stamen+and+petal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+third+myrosinase+gene+TGG3+in+Arabidopsis+thaliana+is+a+pseudogene+specifically+expressed+in+stamen+and+petal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000578-Field1" id="pone.0000578-Field1"></a>Field BG, Cardon G, Traka M, Botterman J, Vancanneyt G, et al.  (2004) Glucosinolate and amino acid biosynthesis in Arabidopsis. Plant Physiol  235: 828–839.  <ul class="find" data-citedArticleID="963060" data-doi="10.1104/pp.104.039347"><li><a href="http://dx.doi.org/10.1104/pp.104.039347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glucosinolate+and+amino+acid+biosynthesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glucosinolate+and+amino+acid+biosynthesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000578-Bonnemail1" id="pone.0000578-Bonnemail1"></a>Bonnemail L (1965) Insect pests of crucifers and their control. Annu Rev Entomol  10: 233–258.  <ul class="find" data-citedArticleID="963050" data-doi="10.1146/annurev.en.10.010165.001313"><li><a href="http://dx.doi.org/10.1146/annurev.en.10.010165.001313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insect+pests+of+crucifers+and+their+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insect+pests+of+crucifers+and+their+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000578-Samson1" id="pone.0000578-Samson1"></a>Samson PR, Geier PW (1983) Induction of crop damage by the cabbage white butterfly <em>Pieris rapae</em> (Lepidoptera, Pieridae) on cabbage. Protect Ecol  5: 199–233.  <ul class="find" data-citedArticleID="963126"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Induction+of+crop+damage+by+the+cabbage+white+butterfly+Pieris+rapae+%28Lepidoptera%2C+Pieridae%29+on+cabbage.&amp;auth=&amp;atitle=Induction+of+crop+damage+by+the+cabbage+white+butterfly+Pieris+rapae+%28Lepidoptera%2C+Pieridae%29+on+cabbage." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+crop+damage+by+the+cabbage+white+butterfly+Pieris+rapae+%28Lepidoptera%2C+Pieridae%29+on+cabbage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+crop+damage+by+the+cabbage+white+butterfly+Pieris+rapae+%28Lepidoptera%2C+Pieridae%29+on+cabbage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000578-Hill1" id="pone.0000578-Hill1"></a>Hill DS (1987) Agricultural insect pests of temperate regions and their control. Cambridge: Cambridge University Press. pp. 411–413.  <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0000578-vanLoon1" id="pone.0000578-vanLoon1"></a>van Loon JJA, Blaakmeer A, Griepink FC, van Beek TA, Schoonhoven LM, et al.  (1992) Leaf surface compound from <em>Brassica oleracea</em> (Cruciferae) induces oviposition by <em>Pieris brassicae</em> (Lepidoptera: Pieridae). Chemoecology  3: 39–44.  <ul class="find" data-citedArticleID="963154" data-doi="10.1007/bf01261455"><li><a href="http://dx.doi.org/10.1007/bf01261455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Leaf+surface+compound+from+Brassica+oleracea+%28Cruciferae%29+induces+oviposition+by+Pieris+brassicae+%28Lepidoptera%3A+Pieridae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Leaf+surface+compound+from+Brassica+oleracea+%28Cruciferae%29+induces+oviposition+by+Pieris+brassicae+%28Lepidoptera%3A+Pieridae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000578-Huang1" id="pone.0000578-Huang1"></a>Huang XP, Renwick JAA (1994) Relative activities of glucosinolates as oviposition stimulants for <em>Pieris rapae</em> and <em>Pieris napi oleracea</em>. J Chem Ecol  20: 1025–1037.  <ul class="find" data-citedArticleID="963070" data-doi="10.1007/bf02059739"><li><a href="http://dx.doi.org/10.1007/bf02059739" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relative+activities+of+glucosinolates+as+oviposition+stimulants+for+Pieris+rapae+and+Pieris+napi+oleracea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relative+activities+of+glucosinolates+as+oviposition+stimulants+for+Pieris+rapae+and+Pieris+napi+oleracea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000578-Stdler1" id="pone.0000578-Stdler1"></a>Städler E, Renwick JAA, Radke CD, Sachdevgupta K (1995) Tarsal contact chemoreceptor response to glucosinolates and cardenolides mediating oviposition in <em>Pieris rapae</em>. Physiol Entomol  20: 175–187.  <ul class="find" data-citedArticleID="963134" data-doi="10.1111/j.1365-3032.1995.tb00814.x"><li><a href="http://dx.doi.org/10.1111/j.1365-3032.1995.tb00814.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tarsal+contact+chemoreceptor+response+to+glucosinolates+and+cardenolides+mediating+oviposition+in+Pieris+rapae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tarsal+contact+chemoreceptor+response+to+glucosinolates+and+cardenolides+mediating+oviposition+in+Pieris+rapae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000578-Miles1" id="pone.0000578-Miles1"></a>Miles CI, del Campo ML, Renwick JAA (2005) Behavioral and chemosensory responses to a host recognition cue by larvae of <em>Pieris rapae</em>. J Comp Physiol A – Neuroethol Sens Neur Behav Physiol  191: 147–155.  <ul class="find" data-citedArticleID="963112" data-doi="10.1007/s00359-004-0580-x"><li><a href="http://dx.doi.org/10.1007/s00359-004-0580-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Behavioral+and+chemosensory+responses+to+a+host+recognition+cue+by+larvae+of+Pieris+rapae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Behavioral+and+chemosensory+responses+to+a+host+recognition+cue+by+larvae+of+Pieris+rapae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000578-Schoonhoven1" id="pone.0000578-Schoonhoven1"></a>Schoonhoven LM (1969) Gustation and foodplant selection in some Lepidopterous larvae. Entomol Exp Appl  12: 555–561.  <ul class="find" data-citedArticleID="963130" data-doi="10.1111/j.1570-7458.1969.tb02553.x"><li><a href="http://dx.doi.org/10.1111/j.1570-7458.1969.tb02553.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gustation+and+foodplant+selection+in+some+Lepidopterous+larvae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gustation+and+foodplant+selection+in+some+Lepidopterous+larvae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000578-Renwick1" id="pone.0000578-Renwick1"></a>Renwick JAA, Lopez K (1999) Experience-based food consumption by larvae of <em>Pieris rapae</em>: addiction to glucosinolates? Entomol Exp Appl  91: 51–58.  <ul class="find" data-citedArticleID="963122" data-doi="10.1046/j.1570-7458.1999.00465.x"><li><a href="http://dx.doi.org/10.1046/j.1570-7458.1999.00465.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experience-based+food+consumption+by+larvae+of+Pieris+rapae%3A+addiction+to+glucosinolates%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experience-based+food+consumption+by+larvae+of+Pieris+rapae%3A+addiction+to+glucosinolates%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000578-Renwick2" id="pone.0000578-Renwick2"></a>Renwick JAA (2002) The chemical world of crucivores: lures, treats and traps. Entomol Exp Appl  104: 35–42.  <ul class="find" data-citedArticleID="963124" data-doi="10.1007/978-94-017-2776-1_4"><li><a href="http://dx.doi.org/10.1007/978-94-017-2776-1_4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chemical+world+of+crucivores%3A+lures%2C+treats+and+traps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chemical+world+of+crucivores%3A+lures%2C+treats+and+traps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000578-Shelton1" id="pone.0000578-Shelton1"></a>Shelton AM, Cooley RJ, Kroening MK, Wilsey WT, Eigenbrode SD (1991) Comparative analysis of two rearing procedures for diamond-back moth <em>Lepidoptera plutellidae</em>. J Entomol Sci  26: 17–26.  <ul class="find" data-citedArticleID="963132"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+analysis+of+two+rearing+procedures+for+diamond-back+moth+Lepidoptera+plutellidae.&amp;auth=&amp;atitle=Comparative+analysis+of+two+rearing+procedures+for+diamond-back+moth+Lepidoptera+plutellidae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+two+rearing+procedures+for+diamond-back+moth+Lepidoptera+plutellidae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+two+rearing+procedures+for+diamond-back+moth+Lepidoptera+plutellidae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000578-Loudet1" id="pone.0000578-Loudet1"></a>Loudet OS, Chaillou S, Camilleri C, Bouchez D, Daniel-Vedele F (2002) Bay-0×Shahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis. Theor Appl Genet  104: 1173–1184.  <ul class="find" data-citedArticleID="963102" data-doi="10.1007/s00122-001-0825-9"><li><a href="http://dx.doi.org/10.1007/s00122-001-0825-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bay-0%26%23xd7%3BShahdara+recombinant+inbred+line+population%3A+a+powerful+tool+for+the+genetic+dissection+of+complex+traits+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bay-0%26%23xd7%3BShahdara+recombinant+inbred+line+population%3A+a+powerful+tool+for+the+genetic+dissection+of+complex+traits+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000578-Arabidopsis1" id="pone.0000578-Arabidopsis1"></a>Arabidopsis Genome Initiative (2000) Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em>. Nature  408: 796–815.  <ul class="find" data-citedArticleID="963044" data-doi="10.1038/35048692"><li><a href="http://dx.doi.org/10.1038/35048692" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+genome+sequence+of+the+flowering+plant+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+genome+sequence+of+the+flowering+plant+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000578-Lander1" id="pone.0000578-Lander1"></a>Lander ES, Green P, Abrahamson J, Barlow A, Daly MJ, et al.  (1987) MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations. Genomics  1: 174–181.  <ul class="find" data-citedArticleID="963098" data-doi="10.1016/j.ygeno.2008.12.003"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2008.12.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAPMAKER%3A+an+interactive+computer+package+for+constructing+primary+genetic+linkage+maps+of+experimental+and+natural+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAPMAKER%3A+an+interactive+computer+package+for+constructing+primary+genetic+linkage+maps+of+experimental+and+natural+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000578-Buchner1" id="pone.0000578-Buchner1"></a>Buchner R (1987)  Approach to determination of HPLC response factors for glucosinolates. In: Wathelet JP, editor. Glucosinolates in Rapeseeds: Analytical Aspects. Boston: Martinus Nijhoff Publishers. pp. 50–58.  <ul class="find-nolinks"></ul></li><li><span class="label">54.
              </span><a name="pone.0000578-Brown1" id="pone.0000578-Brown1"></a>Brown PD, Tokuhisa JG, Reichelt M, Gershenzon J (2003) Variation of glucosinolate accumulation among different organs and developmental stages of <em>Arabidopsis thaliana</em>. Phytochemistry  62: 471–481.  <ul class="find" data-citedArticleID="963052" data-doi="10.1016/s0031-9422(02)00549-6"><li><a href="http://dx.doi.org/10.1016/s0031-9422(02)00549-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+of+glucosinolate+accumulation+among+different+organs+and+developmental+stages+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+of+glucosinolate+accumulation+among+different+organs+and+developmental+stages+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000578-Wang1" id="pone.0000578-Wang1"></a>Wang S, Basten CJ, Zeng ZB (2001) Windows QTL cartographer 2.5. Raleigh, NC: Department of Statistics, North Carolina State University.   (<a href="http://statgen.ncsu.edu.qtlcart/WQTLCart.htm">http://statgen.ncsu.edu.qtlcart/WQTLCart​.htm</a>).  <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0000578-vanOoijen1" id="pone.0000578-vanOoijen1"></a>van Ooijen JW (1999) LOD significance thresholds for QTL analysis in experimental populations of diploid species. Heredity  83: 613–624.  <ul class="find" data-citedArticleID="963156" data-doi="10.1038/sj.hdy.6886230"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6886230" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LOD+significance+thresholds+for+QTL+analysis+in+experimental+populations+of+diploid+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LOD+significance+thresholds+for+QTL+analysis+in+experimental+populations+of+diploid+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000578-Satagopan1" id="pone.0000578-Satagopan1"></a>Satagopan JM, Yandell BS, Newton MA, Osborn TC (1996) Markov chain Monte Carlo approach to detect polygene loci for complex traits. Genetics  144: 805–816.  <ul class="find" data-citedArticleID="963128"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Markov+chain+Monte+Carlo+approach+to+detect+polygene+loci+for+complex+traits.&amp;auth=&amp;atitle=Markov+chain+Monte+Carlo+approach+to+detect+polygene+loci+for+complex+traits." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Markov+chain+Monte+Carlo+approach+to+detect+polygene+loci+for+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Markov+chain+Monte+Carlo+approach+to+detect+polygene+loci+for+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000578-Kroymann3" id="pone.0000578-Kroymann3"></a>Kroymann J, Mitchell-Olds T (2005) Epistasis and balanced polymorphism influencing complex trait variation. Nature  435: 95–98.  <ul class="find" data-citedArticleID="963094" data-doi="10.1038/nature03480"><li><a href="http://dx.doi.org/10.1038/nature03480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epistasis+and+balanced+polymorphism+influencing+complex+trait+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epistasis+and+balanced+polymorphism+influencing+complex+trait+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.XML" value="173154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.PDF" value="403013"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g001.PNG_L" value="316884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g001.PNG_M" value="15684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g001.PNG_S" value="1814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g001.TIF" value="786898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g001.PNG_I" value="8125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g002.PNG_L" value="378500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g002.PNG_M" value="19786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g002.PNG_S" value="1907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g002.TIF" value="757966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g002.PNG_I" value="10544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g003.PNG_L" value="1180686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g003.PNG_M" value="69602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g003.PNG_S" value="6332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g003.TIF" value="2739558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g003.PNG_I" value="66607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g004.PNG_L" value="303119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g004.PNG_M" value="24747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g004.PNG_S" value="2216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g004.TIF" value="545736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.g004.PNG_I" value="11771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t001.PNG_L" value="228610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t001.PNG_M" value="57599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t001.PNG_S" value="8761"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t001.TIF" value="1218144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t001.PNG_I" value="123020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t002.PNG_L" value="812705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t002.PNG_M" value="101169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t002.PNG_S" value="13173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t002.TIF" value="3784826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.t002.PNG_I" value="175719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.s001.XLS" value="262656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000578.s002.DOC" value="58368"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000578&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000578" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000578&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000578&volume=&issue=&title=Mapping of QTL for Resistance against the Crucifer Specialist Herbivore Pieris brassicae in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2&author_name=Marina%20Pfalz%2C%20Heiko%20Vogel%2C%20Thomas%20Mitchell-Olds%2C%20Juergen%20Kroymann&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000578" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000578" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000578" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000578&amp;t=Mapping%20of%20QTL%20for%20Resistance%20against%20the%20Crucifer%20Specialist%20Herbivore%20Pieris%20brassicae%20in%20a%20New%20Arabidopsis%20Inbred%20Line%20Population%2C%20Da(1)-12%C3%97Ei-2" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000578&title=Mapping%20of%20QTL%20for%20Resistance%20against%20the%20Crucifer%20Specialist%20Herbivore%20Pieris%20brassicae%20in%20a%20New%20Arabidopsis%20Inbred%20Line%20Population%2C%20Da(1)-12%C3%97Ei-2&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000578&amp;title=Mapping%20of%20QTL%20for%20Resistance%20against%20the%20Crucifer%20Specialist%20Herbivore%20Pieris%20brassicae%20in%20a%20New%20Arabidopsis%20Inbred%20Line%20Population%2C%20Da(1)-12%C3%97Ei-2" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000578" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000578&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Mapping of QTL for Resistance against the Crucifer Specialist Herbivore Pieris brassicae in a New Arabidopsis Inbred Line Population, Da(1)-12×Ei-2';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000578';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Mapping%20of%20QTL%20for%20Resistance%20against%20the%20Crucifer%20Specialist%20Herbivore%20Pieris%20brassicae%20in%20a%20New%20Arabidopsis%20Inbred%20Line%20Population%2C%20Da(1)-12%C3%97Ei-2 http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000578" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000578" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="23994"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+1%22" title="Search for articles in the subject area:'Chromosome 1'"><div class="flagText">Chromosome 1</div></a>
              <div data-categoryid="20449" data-articleid="23994"
                   data-categoryname="Chromosome 1"
                   class="flagImage" title="Flag 'Chromosome 1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Herbivory%22" title="Search for articles in the subject area:'Herbivory'"><div class="flagText">Herbivory</div></a>
              <div data-categoryid="49491" data-articleid="23994"
                   data-categoryname="Herbivory"
                   class="flagImage" title="Flag 'Herbivory' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Indoles%22" title="Search for articles in the subject area:'Indoles'"><div class="flagText">Indoles</div></a>
              <div data-categoryid="20495" data-articleid="23994"
                   data-categoryname="Indoles"
                   class="flagImage" title="Flag 'Indoles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Insects%22" title="Search for articles in the subject area:'Insects'"><div class="flagText">Insects</div></a>
              <div data-categoryid="42993" data-articleid="23994"
                   data-categoryname="Insects"
                   class="flagImage" title="Flag 'Insects' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leaves%22" title="Search for articles in the subject area:'Leaves'"><div class="flagText">Leaves</div></a>
              <div data-categoryid="17987" data-articleid="23994"
                   data-categoryname="Leaves"
                   class="flagImage" title="Flag 'Leaves' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant-herbivore+interactions%22" title="Search for articles in the subject area:'Plant-herbivore interactions'"><div class="flagText">Plant-herbivore interactions</div></a>
              <div data-categoryid="33687" data-articleid="23994"
                   data-categoryname="Plant-herbivore interactions"
                   class="flagImage" title="Flag 'Plant-herbivore interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Trichomes%22" title="Search for articles in the subject area:'Trichomes'"><div class="flagText">Trichomes</div></a>
              <div data-categoryid="22041" data-articleid="23994"
                   data-categoryname="Trichomes"
                   class="flagImage" title="Flag 'Trichomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4831'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4995'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4104&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=44835">Well done</a><br>Posted by Damien</p>
      <p><a href="/annotation/listThread.action?root=6943">Interesting...</a><br>Posted by Damien</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
