

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001901"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001901" />

    <meta name="citation_title" content="Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions"/>
    <meta itemprop="name" content="Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions"/>

      <meta name="citation_author" content="Eli Zamir"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Benjamin Geiger"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Zvi Kam"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001901.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1901"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Single Cell Profiling of Potentiated Phospho-Protein Networks in Cancer Cells.; citation_author=JM Irish; citation_author=R Hovland; citation_author=PO Krutzik; citation_author=OD Perez; citation_author=O Bruserud; citation_journal_title=Cell; citation_volume=118; citation_number=1; citation_pages=217-228; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=11-color, 13-parameter flow cytometry: identification of human naive T cells by phenotype, function, and T-cell receptor diversity.; citation_author=SC De Rosa; citation_author=LA Herzenberg; citation_author=M Roederer; citation_journal_title=Nat Med; citation_volume=7; citation_number=2; citation_pages=245-248; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Computational prediction of proteotypic peptides for quantitative proteomics.; citation_author=P Mallick; citation_author=M Schirle; citation_author=SS Chen; citation_author=MR Flory; citation_author=H Lee; citation_journal_title=Nat Biotechnol; citation_volume=25; citation_number=3; citation_pages=125-131; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative analysis of complex protein mixtures using isotope-coded affinity tags.; citation_author=SP Gygi; citation_author=B Rist; citation_author=SA Gerber; citation_author=F Turecek; citation_author=MH Gelb; citation_journal_title=Nat Biotechnol; citation_volume=17; citation_number=4; citation_pages=994-999; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Integrative analysis of the cancer transcriptome.; citation_author=DR Rhodes; citation_author=AM Chinnaiyan; citation_journal_title=Nat Genet 37 Suppl; citation_number=5; citation_pages=S31-37; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiplexed protein measurement: technologies and applications of protein and antibody arrays.; citation_author=SF Kingsmore; citation_journal_title=Nat Rev Drug Discov; citation_volume=5; citation_number=6; citation_pages=310-320; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular complexity and dynamics of cell-matrix adhesions.; citation_author=E Zamir; citation_author=B Geiger; citation_journal_title=J Cell Sci; citation_volume=114; citation_number=7; citation_pages=3583-3590; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Functional atlas of the integrin adhesome.; citation_author=R Zaidel-Bar; citation_author=S Itzkovitz; citation_author=A Ma'ayan; citation_author=R Iyengar; citation_author=B Geiger; citation_journal_title=Nat Cell Biol; citation_volume=9; citation_number=8; citation_pages=858-867; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Hierarchical assembly of cell-matrix adhesion complexes.; citation_author=R Zaidel-Bar; citation_author=M Cohen; citation_author=L Addadi; citation_author=B Geiger; citation_journal_title=Biochem Soc Trans; citation_volume=32; citation_number=9; citation_pages=416-420; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Dynamics and segregation of cell-matrix adhesions in cultured fibroblasts.; citation_author=E Zamir; citation_author=M Katz; citation_author=Y Posen; citation_author=N Erez; citation_author=KM Yamada; citation_journal_title=Nat Cell Biol; citation_volume=2; citation_number=10; citation_pages=191-196; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular diversity of cell-matrix adhesions.; citation_author=E Zamir; citation_author=BZ Katz; citation_author=S Aota; citation_author=KM Yamada; citation_author=B Geiger; citation_journal_title=J Cell Sci; citation_volume=112 (Pt 11); citation_number=11; citation_pages=1655-1669; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Early molecular events in the assembly of matrix adhesions at the leading edge of migrating cells.; citation_author=R Zaidel-Bar; citation_author=C Ballestrem; citation_author=Z Kam; citation_author=B Geiger; citation_journal_title=J Cell Sci; citation_volume=116; citation_number=12; citation_pages=4605-4613; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular diversity of plaques of epithelial-adhering junctions.; citation_author=CM Borrmann; citation_author=C Mertens; citation_author=A Schmidt; citation_author=L Langbein; citation_author=C Kuhn; citation_journal_title=Ann N Y Acad Sci; citation_volume=915; citation_number=13; citation_pages=144-150; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evolving endosomes: how many varieties and why?; citation_author=E Perret; citation_author=A Lakkaraju; citation_author=S Deborde; citation_author=R Schreiner; citation_author=E Rodriguez-Boulan; citation_journal_title=Curr Opin Cell Biol; citation_volume=17; citation_number=14; citation_pages=423-434; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quantum dots for live cells, in vivo imaging, and diagnostics.; citation_author=X Michalet; citation_author=FF Pinaud; citation_author=LA Bentolila; citation_author=JM Tsay; citation_author=S Doose; citation_journal_title=Science; citation_volume=307; citation_number=15; citation_pages=538-544; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In vivo molecular and cellular imaging with quantum dots.; citation_author=X Gao; citation_author=L Yang; citation_author=JA Petros; citation_author=FF Marshall; citation_author=JW Simons; citation_journal_title=Curr Opin Biotechnol; citation_volume=16; citation_number=16; citation_pages=63-72; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A guide to choosing fluorescent proteins.; citation_author=NC Shaner; citation_author=PA Steinbach; citation_author=RY Tsien; citation_journal_title=Nat Methods; citation_volume=2; citation_number=17; citation_pages=905-909; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Five-colour in vivo imaging of neurons in Caenorhabditis elegans.; citation_author=H Hutter; citation_journal_title=J Microsc; citation_volume=215; citation_number=18; citation_pages=213-218; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Spectral imaging and its applications in live cell microscopy.; citation_author=T Zimmermann; citation_author=J Rietdorf; citation_author=R Pepperkok; citation_journal_title=FEBS Lett; citation_volume=546; citation_number=19; citation_pages=87-92; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Spectral karyotyping analysis of human and mouse chromosomes.; citation_author=HM Padilla-Nash; citation_author=L Barenboim-Stapleton; citation_author=MJ Difilippantonio; citation_author=T Ried; citation_journal_title=Nat Protoc; citation_volume=1; citation_number=20; citation_pages=3129-3142; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Analyzing proteome topology and function by automated multidimensional fluorescence microscopy.; citation_author=W Schubert; citation_author=B Bonnekoh; citation_author=AJ Pommer; citation_author=L Philipsen; citation_author=R Bockelmann; citation_journal_title=Nat Biotechnol; citation_volume=24; citation_number=21; citation_pages=1270-1278; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Probing molecular processes in live cells by quantitative multidimensional microscopy.; citation_author=Z Kam; citation_author=E Zamir; citation_author=B Geiger; citation_journal_title=Trends Cell Biol; citation_volume=11; citation_number=22; citation_pages=329-334; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Six-color segmentation of multicolor images in the infection studies of Listeria monocytogenes.; citation_author=B Ma; citation_author=F He; citation_author=J Jablonska; citation_author=S Winkelbach; citation_author=W Lindenmaier; citation_journal_title=Microsc Res Tech; citation_volume=70; citation_number=23; citation_pages=171-178; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays.; citation_author=U Alon; citation_author=N Barkai; citation_author=DA Notterman; citation_author=K Gish; citation_author=S Ybarra; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=24; citation_pages=6745-6750; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A paxillin tyrosine phosphorylation switch regulates the assembly and form of cell-matrix adhesions.; citation_author=R Zaidel-Bar; citation_author=R Milo; citation_author=Z Kam; citation_author=B Geiger; citation_journal_title=J Cell Sci; citation_volume=120; citation_number=25; citation_pages=137-148; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Marching at the front and dragging behind: differential alphaVbeta3-integrin turnover regulates focal adhesion behavior.; citation_author=C Ballestrem; citation_author=B Hinz; citation_author=BA Imhof; citation_author=B Wehrle-Haller; citation_journal_title=J Cell Biol; citation_volume=155; citation_number=26; citation_pages=1319-1332; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Focal contacts as mechanosensors: externally applied local mechanical force induces growth of focal contacts by an mDia1-dependent and ROCK-independent mechanism.; citation_author=D Riveline; citation_author=E Zamir; citation_author=NQ Balaban; citation_author=US Schwarz; citation_author=T Ishizaki; citation_journal_title=J Cell Biol; citation_volume=153; citation_number=27; citation_pages=1175-1186; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Imaging mass spectrometry at cellular length scales.; citation_author=AF Altelaar; citation_author=SL Luxembourg; citation_author=LA McDonnell; citation_author=SR Piersma; citation_author=RM Heeren; citation_journal_title=Nat Protoc; citation_volume=2; citation_number=28; citation_pages=1185-1196; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cell behaviour on micropatterned substrata: limits of extracellular matrix geometry for spreading and adhesion.; citation_author=D Lehnert; citation_author=B Wehrle-Haller; citation_author=C David; citation_author=U Weiland; citation_author=C Ballestrem; citation_author=BA Imhof; citation_author=M Bastmeyer; citation_journal_title=J Cell Sci; citation_volume=117; citation_number=29; citation_pages=41-52; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The mechanisms and dynamics of (alpha)v(beta)3 integrin clustering in living cells.; citation_author=C Cluzel; citation_author=F Saltel; citation_author=J Lussi; citation_author=F Paulhe; citation_author=BA Imhof; citation_author=B Wehrle-Haller; citation_journal_title=J Cell Biol; citation_volume=171; citation_number=30; citation_pages=383-92; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Focal adhesion size controls tension-dependent recruitment of alpha-smooth muscle actin to stress fibers.; citation_author=JM Goffin; citation_author=P Pittet; citation_author=G Csucs; citation_author=JW Lussi; citation_author=JJ Meister; citation_author=B Hinz; citation_journal_title=J Cell Biol; citation_volume=172; citation_number=31; citation_pages=259-68; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cell spreading and focal adhesion dynamics are regulated by spacing of integrin ligands.; citation_author=EA Cavalcanti-Adam; citation_author=T Volberg; citation_author=A Micoulet; citation_author=H Kessler; citation_author=B Geiger; citation_author=JP Spatz; citation_journal_title=Biophys J; citation_volume=92; citation_number=32; citation_pages=2964-74; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative analysis of cytoskeletal organization by digital fluorescent microscopy.; citation_author=N Lichtenstein; citation_author=B Geiger; citation_author=Z Kam; citation_journal_title=Cytometry A; citation_volume=54; citation_number=33; citation_pages=8-18; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001901" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions"/>
    <meta name="twitter:description" content="BackgroundCellular processes occur within dynamic and multi-molecular compartments whose characterization requires analysis at high spatio-temporal resolution. Notable examples for such complexes are cell-matrix adhesion sites, consisting of numerous cytoskeletal and signaling proteins. These adhesions are highly variable in their morphology, dynamics, and apparent function, yet their molecular diversity is poorly defined.Methodology/Principal FindingsWe present here a compositional imaging approach for the analysis and display of multi-component compositions. This methodology is based on microscopy-acquired multicolor data, multi-dimensional clustering of pixels according to their composition similarity and display of the cellular distribution of these composition clusters. We apply this approach for resolving the molecular complexes associated with focal-adhesions, and the time-dependent effects of Rho-kinase inhibition. We show here compositional variations between adhesion sites, as well as ordered variations along the axis of individual focal-adhesions. The multicolor clustering approach also reveals distinct sensitivities of different focal-adhesion-associated complexes to Rho-kinase inhibition.Conclusions/SignificanceMulticolor compositional imaging resolves &ldquo;molecular signatures&rdquo; characteristic to focal-adhesions and related structures, as well as sub-domains within these adhesion sites. This analysis enhances the spatial information with additional &ldquo;contents-resolved&rdquo; dimensions. We propose that compositional imaging can serve as a powerful tool for studying complex multi-molecular assemblies in cells and for mapping their distribution at sub-micron resolution."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001901.g005"/>

  <meta property="og:title" content="Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001901" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3335'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9900'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1274&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001901">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001901" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001901&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eli Zamir, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel
                </p>

                <p>
                    Current address: Cell Biology and Biophysics Unit, European Molecular Biology Laboratory, Heidelberg, Germany; 
                    Department of Systemic Cell Biology, Max-Planck Institute of Molecular Physiology, Dortmund, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Benjamin Geiger
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:benny.geiger@weizmann.ac.il">benny.geiger@weizmann.ac.il</a></p>

                <p>Affiliation:
                  Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zvi Kam
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001901</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001901" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001901" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001901" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001901" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001901-g001" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001901-t001" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001901-g002" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001901-g003" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001901-t002" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001901-g004" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001901-g005" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001901">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001901" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Cellular processes occur within dynamic and multi-molecular compartments whose characterization requires analysis at high spatio-temporal resolution. Notable examples for such complexes are cell-matrix adhesion sites, consisting of numerous cytoskeletal and signaling proteins. These adhesions are highly variable in their morphology, dynamics, and apparent function, yet their molecular diversity is poorly defined.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We present here a compositional imaging approach for the analysis and display of multi-component compositions. This methodology is based on microscopy-acquired multicolor data, multi-dimensional clustering of pixels according to their composition similarity and display of the cellular distribution of these composition clusters. We apply this approach for resolving the molecular complexes associated with focal-adhesions, and the time-dependent effects of Rho-kinase inhibition. We show here compositional variations between adhesion sites, as well as ordered variations along the axis of individual focal-adhesions. The multicolor clustering approach also reveals distinct sensitivities of different focal-adhesion-associated complexes to Rho-kinase inhibition.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Multicolor compositional imaging resolves “molecular signatures” characteristic to focal-adhesions and related structures, as well as sub-domains within these adhesion sites. This analysis enhances the spatial information with additional “contents-resolved” dimensions. We propose that compositional imaging can serve as a powerful tool for studying complex multi-molecular assemblies in cells and for mapping their distribution at sub-micron resolution.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zamir E, Geiger B, Kam Z (2008) Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions. PLoS ONE 3(4):
          e1901.
            doi:10.1371/journal.pone.0001901</p><p><strong>Editor: </strong>Neil Hotchin, University of Birmingham, United Kingdom</p><p><strong>Received:</strong> October 27, 2007; <strong>Accepted:</strong> February 20, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Zamir et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by grants from the Israel Science Foundation, the US-Israel Bi-National Foundation, the Kahn fund for Systems Biology at the Weizmann Institute of Science and The Cell Migration Consortium (NIH Grant U54 GM64346).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Molecular processes in cells involve multiple components that are dynamically interacting with each other. The characterization of such events requires, therefore, a capability to simultaneously localize and quantify the relevant molecules at the maximal resolution. A wide range of methods were developed for measuring the levels of a large number of molecules, biochemically or optically<a href="#pone.0001901-Irish1">[1]</a>–<a href="#pone.0001901-Kingsmore1">[6]</a>, yet these approaches lack the sub-cellular spatial information. In this study we developed a light-microscopy-based imaging approach which defines multi-component compositions at single pixel resolutions, and applied novel analysis for defining specific “molecular signatures” and visualizing their sub-cellular distributions.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Compositional imaging was applied here to study the molecular reorganization of cell-matrix adhesions in rat embryo fibroblasts (REF52). The assembly and modulation of cell adhesions are highly regulated dynamic processes, characterized by the selective recruitment of specific subsets of molecules, derived from a repertoire of over 150 proteins, including transmembrane receptors (primarily different integrins), cytoskeletal and adapter proteins (such as actin, vinculin, paxillin, zyxin and α-actinin) and enzymes (such as focal-adhesion kinase, FAK)<a href="#pone.0001901-Zamir1">[7]</a>, <a href="#pone.0001901-ZaidelBar1">[8]</a>. A fundamental feature of cell-matrix adhesions is the high diversity in their molecular composition and dynamics, which was studied by simultaneous two-component labeling of fixed cells, time-resolved experiments with GFP-tagged adhesion components and time-lapse movies of cells fixed and labeled at the end-point<a href="#pone.0001901-Zamir1">[7]</a>–<a href="#pone.0001901-ZaidelBar3">[12]</a>. Based on morphological and molecular criteria, several types of cell-matrix adhesions were distinguished in cultured cells. These include focal-complexes, which are small and short-lived contacts formed at the edge of the lamellipodium, focal-adhesions, which are associated with the ends of contractile actin stress-fibers, elongated fibrillar-adhesions, which play a role in extracellular matrix fibrillogenesis, and “3D matrix adhesions”, which are formed with pre-assembled matrix fibrils. Each of these forms is characterized by distinct dynamic properties, stability, mechanosensitivity and molecular composition<a href="#pone.0001901-Zamir1">[7]</a>, <a href="#pone.0001901-ZaidelBar2">[9]</a>–<a href="#pone.0001901-Zamir3">[11]</a>. However, the relationships between these functional and molecular features are still poorly understood. A major reason is the presence of composition heterogeneity between adhesions of apparently the same type, and within a single adhesion structure, and the lack of tools for exploring and visualizing these multi-component compositions at high spatial resolutions in relation to cellular behavior. Here we show that compositional imaging provides such a powerful tool for exploring the molecular diversity of focal-adhesions, both in control cells and following modulation of Rho-kinase activity.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Compositional imaging of focal-adhesions was obtained by simultaneous labeling of cells for different combinations of 5 focal-adhesion components, and acquiring the corresponding 5-color images. Multi-dimensional cluster analysis of the pixels of these images identifies typical “compositional signatures” found in the adhesion sites of the labeled cells. Coloring pixels according to these signatures revealed their unique organization between and within spatial sub-domains of focal-adhesions and stress-fibers. We further show that cellular perturbations, such as modulation of actomyosin contractility by Rho-kinase inhibition, differentially affect the abundance and distribution of the various signatures along focal-adhesions and stress-fibers. We discuss here the implications of this compositional mapping for characterizing focal-adhesion assembly, and its general applicability for molecular characterization of complex sub-cellular structures.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Resolving compositional signatures</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>REF52 cells, stably expressing β<sub>3</sub>-integrin tagged with EGFP (β<sub>3</sub>-integrin-GFP), were fluorescently labeled for 4 additional focal-adhesion-associated molecules. Altogether, 8 molecules (actin, β<sub>3</sub>-integrin, paxillin, α-actinin, vinculin, FAK, zyxin and phosphotyrosine) were visualized using 4 labeling sets (A–D, as defined in <a href="#pone-0001901-g001">Fig. 1</a> and <a href="#pone-0001901-t001">Table 1</a>). This labeling was applied to well-spread, untreated fibroblasts, to cells treated with the Rho-kinase inhibitor Y-27632 and to cells at 3, 15 or 60 minutes after washout of the drug and recovery from this treatment. The effect of this inhibitor is shown in <a href="#pone-0001901-g001">Figure 1</a>, for one set of labeled components.</p>
<div class="figure" id="pone-0001901-g001"><div class="img"><a name="pone-0001901-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Images of REF52 cells labeled for four sets of five focal-adhesion-associated components.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>REF52 cells, stably expressing β<sub>3</sub>-integrin-GFP, were fixed 24 hours after plating and labeled for actin and paxillin, as well as for: (A) vinculin and α-actinin, (B) zyxin and α-actinin, (C) FAK and α-actinin, (D) vinculin and phosphotyrosine (PY). (A') REF52 cells treated for 3 hours with 100 µM of the Rho-kinase inhibitor Y-27632 and labeled as in (A). The fluorophores used here include: Cy5 (for the IR1 channel), Alexa-750 (IR2), Cy3 or Alexa-555 (red), GFP (green) and CPITC (blue). Images were acquired using selective excitation and emission filter sets for five fluorescent channels. Scale bar, 10 µm.</p>
<span>doi:10.1371/journal.pone.0001901.g001</span></div><div class="figure" id="pone-0001901-t001"><div class="img"><a name="pone-0001901-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001901.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001901.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Multicolor labeling scheme.</span></strong></p><span>doi:10.1371/journal.pone.0001901.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Multicolor data indicates the levels of the labeled components (5 for this data set) at a single pixel resolution. It is, however, practically impossible to comprehend the complex relationships between all 5 components by direct visual comparison of the images of the separate components (as apparent from <a href="#pone-0001901-g001">Fig. 1</a>), by superimposing all possible two- or three-color combinations (<a href="#pone-0001901-g002">Fig. 2A</a>), or by superimposing all 5 colors (Supplementary <a href="#pone.0001901.s008">Fig. S7</a>). Quantitative multi-parametric approach is therefore essential to extract and visualize composition information from the multicolor data. Towards that end, we clustered the pixels of the multicolor data based on multicolor composition similarity (<a href="#pone-0001901-g002">Fig. 2</a>). This was performed on a pool containing all pixels, positively labeled for at least one component, from images of 4 cells for each of the 5 treatments (untreated cells, Y-27632-treated cells, and Y-27632-treated cells followed by 3 recovery times, altogether 20 cells for each labeling combination). Each of the resulting clusters (<a href="#pone-0001901-g002">Fig. 2</a> and Supplementary <a href="#pone.0001901.s006">Fig. S5</a>) defines a distinct compositional signature (<a href="#pone-0001901-g003">Fig. 3</a> right and Supplementary <a href="#pone.0001901.s001">Table S1</a>). In order to compare fluorescent levels in the different signatures we calculated the averaged color-scaled vectors, V<sub>s</sub> (see <a href="#s4">Materials and Methods</a>). For convenience, the clusters and the corresponding compositional signatures were named with a letter identifying the labeling set from which they were resolved (A–D), and a number such that signatures of different labeling sets that are similar (in respect to the shared components of these sets) carry the same number (<a href="#pone-0001901-g003">Fig. 3</a> and <a href="#pone-0001901-t002">Table 2</a>). For example, signatures A1, B1, C1 and D1 contain actin, with very low levels of the other tested components (<a href="#pone-0001901-g003">Fig. 3</a>, right). It should nevertheless be noted that similar signatures in the four sets should not be considered identical, since they may differ in the level of components that are not shared. Since the standard deviations of the non-normalized vectors reflect the variations in intensity, (not in compositions), error bars in <a href="#pone-0001901-g003">Fig. 3</a> were calculated from the standard deviation of the normalized vectors and multiplied by the cluster-averaged ratio between the non-normalized and the normalized values. This is an accurate estimate, based on the central limit theorem.</p>
<div class="figure" id="pone-0001901-g002"><div class="img"><a name="pone-0001901-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Identifying compositional signatures of adhesion sites by cluster analysis.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>REF52 cells were subjected to: no treatment, Y-27632, and Y-27632 with a subsequent recovery period of 3, 15 or 60 minutes. The cells were then fixed, labeled for vinculin, paxillin, α-actinin and actin, and their 5-color images (including β<sub>3</sub>-integrin-GFP) were analyzed. For each of the 5 treatments 4 cells were sampled, creating a pool of 20 cells. (A) Images showing a single non-treated, 5-color-labeled, cell in all 10 possible combinations of 2 components superposition images. Scale bar, 10 µm. (B) A matrix presenting the composition of all pixels above background in the 20 multicolor images (original). Color indicates the fractional intensity of a given component in each given pixel. The rows were then reordered according to the top-down clustering algorithm<a href="#pone.0001901-Alon1">[24]</a> based on compositional similarity (clustered). The process was deliberately designed to over-divide the data into 32 clusters. (C) Bottom-up merging of the over-divided clusters. The dendrogram presents the hierarchical distance between the merged clusters during the merging process. The significance of each cluster along the merging process (i.e. each node in the dendrogram) was further evaluated visually, based on the spatial coherence of the sub-cellular distribution of its pixels. Thus, the initial 32 clusters were merged to 10 final ones, which define compositional signatures, and were assigned distinguishable colors for visualizing their sub-cellular distribution.</p>
<span>doi:10.1371/journal.pone.0001901.g002</span></div><div class="figure" id="pone-0001901-g003"><div class="img"><a name="pone-0001901-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Five-component compositional signatures of cell-matrix adhesions and stress-fibers.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>The process for defining compositional signatures, as shown in <a href="#pone-0001901-g002">Figure 2</a>, was performed for four labeling sets (A–D) of five components each (see <a href="#pone-0001901-g001">Fig. 1</a> legend). Right, bar-plots presenting the average intensity for each component in each cluster, defining the compositional signatures. These intensities are scaled, but not normalized to one-unit composition vector (V<sub>s</sub>, see <a href="#s4">Materials and Methods</a> section), to allow comparison of actual labeling intensities between signatures. Clusters from different labeling sets, which have similar composition for the shared components, were given the same number and color. The asterisk symbol indicates clusters that were later defined as noise based on their spatial distribution in the cells. The standard deviations were calculated as described in the text. Left, scatter-plots presenting, for each labeling set, the fractional intensity of 2 components (out of 5-components composition) in each pixel, for all possible 2-from-5 combinations. Each dot in the scatter-plots corresponds to a pixel, colored according to the cluster it is assigned to. Due to the normalization to one-unit composition vector (see <a href="#s4">Materials and Methods</a> section), at each two-components projection the pixels cannot span more than a quarter-circle with a radius of one, but can be at smaller radiuses due to the other 3 components excluded from the projection.</p>
<span>doi:10.1371/journal.pone.0001901.g003</span></div><div class="figure" id="pone-0001901-t002"><div class="img"><a name="pone-0001901-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001901.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001901.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Localization and abundance of composition signatures in non-treated REF52 cells.</span></strong></p><span>doi:10.1371/journal.pone.0001901.t002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The importance of resolving the clusters based on all 5 components simultaneously can be appreciated by examining all possible two-component projections of the clusters composition (<a href="#pone-0001901-g003">Fig. 3</a>, left). Clearly, the composition clusters resolved using 5 components become highly superimposed, and therefore irresolvable, when projected on two-component combinations. Noteworthy, although five components could have potentially give rise to a huge number of compositions (<em>N<sup>5</sup></em> where <em>N</em> is the number of intensity variations resolved for each component), the number of different compositions that actually exist in cells is much smaller, in the range of 10. Thus, this analysis identifies which compositions are present, out of a much larger repertoire of possible compositions.</p>


<h4>Mapping compositional signatures</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Given the signature assignment for each pixel, it is possible to visualize the sub-cellular distribution of the different compositions in REF52 cells by coloring each pixel according to its cluster identity (<a href="#pone-0001901-g004">Fig. 4</a> and Supplementary <a href="#pone.0001901.s002">Figs. S1</a>, <a href="#pone.0001901.s003">S2</a>, <a href="#pone.0001901.s004">S3</a>, and <a href="#pone.0001901.s005">S4</a>). Big part of these signatures is mapped in non-treated cells exclusively to either adhesion sites or stress-fibers (<a href="#pone-0001901-t002">Table 2</a>). In addition, the compositional analysis reveals sub-domains with distinct compositions located within these structures. Specifically, signatures that are enriched with paxillin and contain low levels of actin (A2, B2, C2 and D2) are located at the edge of focal-adhesions (<a href="#pone-0001901-g004">Fig. 4</a> and Supplementary <a href="#pone.0001901.s002">Figs. S1</a>, <a href="#pone.0001901.s003">S2</a>, <a href="#pone.0001901.s004">S3</a> and <a href="#pone.0001901.s005">S4</a>). In labeling set B, the signature containing high level of zyxin together with paxillin, α-actinin, β3 and actin (B9) is found in focal-adhesions, but apparently absent from focal-complexes, which are dominated by signature B2 (low zyxin, see <a href="#pone-0001901-g004">Fig. 4</a> and Supplementary <a href="#pone.0001901.s003">Fig. S2</a>). This is consistent with previous findings about the localization of zyxin<a href="#pone.0001901-ZaidelBar3">[12]</a>. In labeling sets that contain vinculin (sets A and D) a high heterogeneity exists between cells regarding the abundance of signatures A2, A3, A8 and A9 in focal-adhesions (Supplementary <a href="#pone.0001901.s007">Fig. S6</a>). Yet, the relative spatial relations between signatures localized along the focal-adhesion is preserved, such that A2 is located in the distal edge of the focal-adhesions, followed by A3, then A9 and then A8, which is located at the edge proximal to the stress-fibers. It is noteworthy that, in contrast to the diversity between focal-adhesions of different cells, a remarkable similarity was noted between focal-adhesions located in the same cell, and particularly in the same sub-cellular region (Supplementary <a href="#pone.0001901.s007">Fig. S6</a>). This suggests that both global cellular states and local factors can modulate various types of focal-adhesion assembly, without altering the spatial organization of distinct sub-domains within focal-adhesions.</p>
<div class="figure" id="pone-0001901-g004"><div class="img"><a name="pone-0001901-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Sub-cellular localization of the compositional clusters in non-treated and treated REF52 cells.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Images of the 5-components-labeled cells, in which each pixel is colored according to its cluster assignment (the color-code is indicated on the right, and is consistent with <a href="#pone.0001901.s004">Fig. 3</a>). The rows correspond to the four labeling sets (A–D), as shown in <a href="#pone-0001901-g003">Figure 3</a>. The columns correspond to treatments (non-treated, Y-27632 treatment, and Y-27632 treatment with a subsequent recovery period of 3, 15 and 60 minutes). Scale bar, 10 µm.</p>
<span>doi:10.1371/journal.pone.0001901.g004</span></div>

<h4>Effects of Rho-kinase inhibition on compositional signatures</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Addition of the Rho-kinase inhibitor Y-27632 to well-spread REF52 cells has a profound effect on the actin cytoskeleton and the associated matrix adhesions. The changes observed include the destruction of stress-fibers, loss of focal-adhesions, and development of large lamellipodia enriched with focal-complexes and α-actinin/actin-rich meshwork (<a href="#pone-0001901-g004">Fig. 4</a>, and Supplementary <a href="#pone.0001901.s002">Figs. S1</a>, <a href="#pone.0001901.s003">S2</a>, <a href="#pone.0001901.s004">S3</a> and <a href="#pone.0001901.s005">S4</a>). Examination of the distribution of the various compositional signatures enabled to dissect and monitor the differential sensitivities of the various molecular complexes to perturbation altering mechanical forces exerted by actin (<a href="#pone-0001901-g004">Figs. 4</a>, <a href="#pone-0001901-g005">5</a> and Supplementary <a href="#pone.0001901.s002">Figs. S1</a>, <a href="#pone.0001901.s003">S2</a>, <a href="#pone.0001901.s004">S3</a> and <a href="#pone.0001901.s005">S4</a>). Indeed, some signatures were very sensitive to Rho-kinase inhibition, while others were resistant or even enhanced by it (<a href="#pone-0001901-g005">Fig. 5</a>) and reorganized into elongated structures (<a href="#pone-0001901-g004">Fig. 4</a> and Supplementary <a href="#pone.0001901.s002">Figs. S1</a>, <a href="#pone.0001901.s003">S2</a>, <a href="#pone.0001901.s004">S3</a> and <a href="#pone.0001901.s005">S4</a>).</p>
<div class="figure" id="pone-0001901-g005"><div class="img"><a name="pone-0001901-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001901" data-uri="info:doi/10.1371/journal.pone.0001901.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001901.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001901.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001901.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001901.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The effect of Rho-kinase inhibition and recovery on the abundance of compositional signatures.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) The number of pixels assigned to each signature was counted in non-treated cells and in cells treated with Y-27632 (based on Fig. 4 and Supplementary <a href="#pone.0001901.s002">Figs. S1</a>, <a href="#pone.0001901.s003">S2</a>, <a href="#pone.0001901.s004">S3</a> and <a href="#pone.0001901.s005">S4</a>). The response of each signature to Y-27632 was then defined as: Log(number of pixels in Y-27632 treated cells/number of pixels in non-treated cells). Thus, positive values indicate increase in abundance, negative values indicate decrease and zero indicates no change, in response to Y-27632. Crosses exclude letter-number combinations that do not correspond to a defined signature (according to <a href="#pone-0001901-g003">Fig. 3</a>), or that correspond to signatures defined as noise (see <a href="#pone-0001901-g003">Fig. 3</a>) or signatures absent in non-treated cells (see <a href="#pone-0001901-t002">Table 2</a>). (B) Changes in the abundance of compositional signatures in response to Rho-kinase inhibition and following its recovery. Each position of each signature in the scatter plot is determined by its response to the Y-27632 treatment (horizontal axis) and to the recovery treatment (vertical axis). The response to Y-27632 was calculated as described for (A). The response to recovery was calculated as Log(number of pixels after recovery of 60 minutes/number of pixels in Y-27632 treated cells). The signatures are marked with a letter A–D, indicating the particular labeling set, and a color indicating its number, consistently with <a href="#pone-0001901-g003">Figure 3</a>. The diagonal dashed line marks the expected trend if the response to Y-27632 and the recovery were exactly opposite processes.</p>
<span>doi:10.1371/journal.pone.0001901.g005</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Compositions dominated by actin, either alone (A1, B1, C1, D1) or with additional protein (e.g. paxillin (A4, B4, C4, D4), FAK (C11), α-actinin (A6, B6, C6) zyxin (B10) or α-actinin and zyxin (B11) or with β<sub>3</sub>-integrin and other components (A8, B8, C8)) were highly sensitive to Rho-kinase inhibition (<a href="#pone-0001901-g005">Fig. 5</a>). Compositions with high levels of zyxin (B9, B10, B11) or vinculin (A9, D6) were also reduced upon Y-27632 treatment. On the other hand, compositions dominated by α-actinin (A5, B5, C5), or α-actinin with some actin (A7, B7, C7), were resistant or even enhanced in response to Rho-kinase inhibition. Interestingly, for the compositions that are dominated by actin and α-actinin, the response to Y-27632 was correlated with the actin/α-actinin ratio between these components, such that compositions with higher ratio (A6, B6, C6) were more sensitive and compositions with lower actin-to-α-actinin ratio (A7, B7, C7) were more resistant (<a href="#pone-0001901-g005">Fig. 5A</a>). Upon removal of Y-27632 from the medium, restoration of focal-adhesions and stress-fibers, as well as a reorganization of the lamellipodium, were apparent (<a href="#pone-0001901-g004">Fig. 4</a> and Supplementary <a href="#pone.0001901.s002">Figs. S1</a>, <a href="#pone.0001901.s003">S2</a>, <a href="#pone.0001901.s004">S3</a> and <a href="#pone.0001901.s005">S4</a>). Thus, almost all the signatures that were diminished upon Y-27632 treatment were restored (<a href="#pone-0001901-g005">Fig. 5B</a>, signatures in the upper-left quarter are along the diagonal line). Some of the signatures that were resistant to Y-27632 further increased following Y-27632 removal (<a href="#pone-0001901-g005">Fig. 5B</a>, signatures in the upper-right quarter).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this article we apply multi-component analysis for spatial mapping of compositional signatures along cell adhesion sites and the associated cytoskeleton at a light-microscopy resolution. Compositional diversity is a common feature displayed by many types of sub-cellular structures (e.g. adhesion sites<a href="#pone.0001901-Zamir1">[7]</a>, <a href="#pone.0001901-Borrmann1">[13]</a>, endosomes<a href="#pone.0001901-Perret1">[14]</a>). Yet, an effective approach for calculating and visualizing these variations has not been developed. Here we demonstrate that 5-color labeling, combined with multi-dimensional clustering of the data, can reveal molecular sub-domains in focal-adhesions and stress-fibers, and highlights the overall molecular reorganization induced in these structures by modulation of Rho-kinase activity.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Simultaneous image analysis of multiple cellular components requires multicolor labeling, quantitative imaging of each individual label, pixel-by-pixel composition analysis, and visualization. Each of these stages can be approached by different methods. Imaging of multiple colors within a single cell can include the use of quantum-dots, which have narrow and tunable emission peaks <a href="#pone.0001901-Michalet1">[15]</a>, <a href="#pone.0001901-Gao1">[16]</a>, a broad range of fluorescent dyes, and tagging with a variety of inherently fluorescent proteins in live cells <a href="#pone.0001901-Shaner1">[17]</a>. Microscopes can resolve colors using selective filters, spectral imaging detectors and color un-mixing methods<a href="#pone.0001901-Hutter1">[18]</a>–<a href="#pone.0001901-PadillaNash1">[20]</a>. A notable alternative for spectral separation is the multi-epitope-ligand cartography, in which the sample is being subjected to repeated cycles of labeling, imaging and bleaching, which enables to image tens of proteins in the same specimen<a href="#pone.0001901-Schubert1">[21]</a>. All these methodologies can, in principle, be combined with the imaging and processing strategy reported here. It should be emphasized that the traditional practice of presenting the entire data in a single image, for example by superposition or ratio imaging <a href="#pone.0001901-Zamir3">[11]</a>, <a href="#pone.0001901-Kam1">[22]</a>, is inadequate for imaging of more than 2 colors (Supplementary <a href="#pone.0001901.s008">Fig. S7</a>). Other approaches, based on reducing the dimensionality of the data (e.g. by superimposing all the color channels to a single RGB image <a href="#pone.0001901-Ma1">[23]</a>) or reducing the intensity resolution (e.g. by binary presentation of above or below an intensity threshold <a href="#pone.0001901-Schubert1">[21]</a>), lead to a major loss of information, and are therefore also incompatible with mapping of quantitative variations in multiple molecular constituents. The labeling approach taken in the present study was based on standard procedures (e.g. combination of primary antibodies from different species, pre-complexing of the primary antibody with tagged secondary Fab fragments and tagging with fluorescent proteins). Careful calibration of the combinations, concentrations and incubation times of the labeling reagents achieved saturated labeling with intensities proportional to local concentrations and a minimal cross-interference between reagents. The fact that the sites of adhesion to an ECM-coated slide are organized along one focal plan facilitates the quantitative imaging of these structures via a single, wide-field, image. The thickness of these structures is below the z-axis resolution of conventional light microscopy, implying similar information for wide-field and confocal microscopy.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The cluster analysis of a very large number of pixels (in the order of one million for a set of 20 images) poses a computational challenge. Hierarchical clustering algorithms require long computation time (<em>n<sup>2</sup></em>(log <em>n</em>), where <em>n</em> is the number of objects to cluster) and large memory (<em>n<sup>2</sup>)</em>. Partitional clustering algorithms (e.g. k-means or fuzzy clustering) are fast but require to pre-determine the total number of clusters and find local, rather than global, clustering optimum. To deal with the difficulty of identifying typical species in large data sets, we applied fast deterministic annealing clustering algorithm, with a computation time of <em>n</em>(log <em>n</em>)<a href="#pone.0001901-Alon1">[24]</a>. Following excessive splitting of the pixels population, we used hierarchical clustering, with visual monitoring of spatial coherence, to merge clusters displaying similar compositional signatures.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Cell-matrix adhesions offer an example for the need for, and the power of, multicolor compositional imaging. These sub-cellular structures consist of a large number of components, most of which can interact with multiple partners and, thus, form molecularly heterogeneous networks <a href="#pone.0001901-Zamir1">[7]</a>, <a href="#pone.0001901-ZaidelBar1">[8]</a>. Since their molecular heterogeneity is believed to be correlated with their mechanical and signaling functions, it is important to develop a quantitative approach for their compositional mapping. Previous studies revealed variations in the spatial distributions of individual components, based on co-labeling of a low number of components (up to three) (e.g. tensin <a href="#pone.0001901-Zamir2">[10]</a>, <a href="#pone.0001901-Zamir3">[11]</a>, phospho-paxillin <a href="#pone.0001901-ZaidelBar4">[25]</a>, zyxin <a href="#pone.0001901-ZaidelBar3">[12]</a> and integrins <a href="#pone.0001901-Zamir2">[10]</a>). Yet this information is limited, and its functional significance has not yet been elucidated. The molecular mapping described here sheds new light on the molecular diversity of integrin adhesions. It suggests that these structures are regulated at multiple levels, including the whole cell level, the “regional” level, possibly controlled by short-range diffusible factors (e.g. Rho or Rac), and the local level, most likely involving direct signaling from the matrix or the cytoskeleton. This study also shows that individual focal-adhesions often contain internal sub-domains, which are regularly ordered along the axis of the adhesion site. Given the fact that integrin adhesions are dynamic structures, growing in a centripetal direction <a href="#pone.0001901-Ballestrem1">[26]</a>, <a href="#pone.0001901-Riveline1">[27]</a>, and that their assembly is induced by mechanical force <a href="#pone.0001901-ZaidelBar2">[9]</a>, <a href="#pone.0001901-Zamir2">[10]</a>, <a href="#pone.0001901-ZaidelBar3">[12]</a>, one may now extend the molecular characterization of integrin adhesions and address the diverse physiological roles of different compositional signatures, and their involvement in the assembly and signaling activities of these adhesion sites.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In conclusion, we demonstrated here that multicolor compositional imaging is a powerful tool for probing the state of molecular organization of sub-cellular structures. It should be emphasized that this approach is rather general, and applicable to any imaging method that can end up with multi-component-per-pixel information, such as mass-spectrometric imaging <a href="#pone.0001901-Altelaar1">[28]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cells and Y-27632 treatment</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Rat embryo fibroblasts (REF52), stably and uniformly expressing β<sub>3</sub>-integrin-GFP, were kindly provided by Dr. C. Ballestrem and Prof. B. Imhof (Centre Medical University, Geneva) and cultured as previously described<a href="#pone.0001901-Ballestrem1">[26]</a>. β<sub>3</sub>-integrin-GFP had been used in previous studies as a marker for β<sub>3</sub>-integrin localization in fixed and live cells <a href="#pone.0001901-Ballestrem1">[26]</a>, <a href="#pone.0001901-Lehnert1">[29]</a>–<a href="#pone.0001901-CavalcantiAdam1">[32]</a>, and was shown to form heterodimers with α<sub>v</sub>-integrin, and to localize at cell-matrix adhesions in an ECM-substrate dependent manner, as the endogenous, wild-type, β<sub>3</sub>-integrin <a href="#pone.0001901-Ballestrem1">[26]</a>. Cells were maintained in culture in Dulbecco's modified Eagle's medium (DMEM), supplemented with 10% fetal calf serum, glutamine and penicillin. For inhibition of Rho-kinase, cells were plated on glass coverslips, incubated for 24 hours, and then treated with 100 µM Y-27632 (Calbiochem, San Diego, CA, USA) for 3 hours. These conditions resulted in a complete and uniform destruction of focal-adhesions and stress-fibers, which is essential for following their reassembly in different fixed cells shortly after recovery. For recovery, Y-27632-treated cells were rinsed and incubated with drug-free medium for 3, 15 or 60 minutes. At the end of these treatments cells were fixed as described<a href="#pone.0001901-Zamir3">[11]</a>.</p>


<h4>Multicolor labeling</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For each labeling set, fixed cells were labeled for 4 adhesion components (in addition to the stable expression of β<sub>3</sub>-integrin-GFP) as described in <a href="#pone-0001901-t001">Table 1</a>. The relevant primary antibodies, and the pre-complexed IgG-Fabs, of each set were mixed together and applied to the cells for 1 hour. The cells were then washed with PBS and incubated for 1 hour with a mixture containing the secondary antibodies and CPITC-conjugated phalloidin (Sigma-Aldrich Co.). Finally, the cells were washed with PBS, fixed with 3% paraformaldehyde in PBS for 30 minutes and mounted in Elvanol (Mowiol 4-88, Hoechst, Frankfurt, Germany). The labeling conditions, including the antibody concentrations and incubation times, were calibrated to achieve saturated staining and a minimal cross-interference between the different antibodies.</p>


<h4>Multicolor fluorescent microscopy</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Cells were examined using 100x/1.3 objective on IX71 inverted microscope (Olympus Ltd., Tokyo) equipped with Prior Scientific (Cambridge, UK) ProScan filter wheels, shutters and stage controller. Images were recorded using a Quantix back-illuminated CCD camera (Photometrix, Roper Scientific Inc., Tucson AZ). Pixel size is 0.13 µm. A QUINT “zero pixel shift” filters set (Chroma Technology Corp. VT) provided 5 excitation (ex) and emission (em) channels with the following wavelength peaks (±bandwidths): blue (ex: 400±20 nm, em: 450±30 nm), green (ex: 484±30 nm, em: 510±15 nm), red (ex: 548±20 nm, em: 560±14 nm), IR1 (ex: 630±40 nm, em: 670±50 nm) and IR2 (ex: 745±50 nm, em: 770±60 nm). The microscope instrumentation was controlled by software written within the Image Visualization Environment (University of California San Francisco, <a href="http://www.msg.ucsf.edu/IVE/">http://www.msg.ucsf.edu/IVE/</a>). Bleed-through between fluorescent channels were measured using singly labeled samples, and were 0–1% for all pairs, beside 2% and 3.4% from Alexa 750 to the IR1 and red channels respectively. The usage of labeling colors was optimized by having the better reagents (e.g. phalloidin) conjugated with the dimmer fluorophores (e.g. CPITC), and vise versa (e.g. Alexa-555 with Fab).</p>


<h4>Image processing</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>As was shown before for cell-matrix adhesions<a href="#pone.0001901-Zamir3">[11]</a>, <a href="#pone.0001901-Kam1">[22]</a> and stress-fibers<a href="#pone.0001901-Lichtenstein1">[33]</a> images, it is important to subtract local background from each pixel in order to obtain the correct intensities of these structures. For all images, beside those of actin, local background subtraction was done by high-pass filtration as previously described <a href="#pone.0001901-Zamir3">[11, see Fig. 1 there]</a>. By comparing with other local background estimates, this procedure was found to work well when the integrated intensity of small labeled-structures is much smaller than the integrated intensity of the background, as is the case for cell adhesions. Since this is not the case for actin filaments, these structures were first found by high-pass filtration and intensity threshold, defining actin filaments mask. Then, a high-pass filtration was again applied on the original actin images, but this time excluding from the calculation of the local background pixels falling inside the actin mask<a href="#pone.0001901-Lichtenstein1">[33]</a>. Slightly negative pixels, resulted from the filtration process, were set to zero. A representative set of original images and the corresponding filtered set are included in the Supplementary Materials (Supplementary <a href="#pone.0001901.s009">Fig. S8</a>). The subtraction of local background enables to define for each image a threshold level that segments adhesion structures in the whole image<a href="#pone.0001901-Zamir3">[11]</a>. Thresholds were interactively tuned for each color of each image. The number of pixels corresponding to cell adhesions is too small for robust automated histogram-based thresholding methods, while interactive threshold-setting was found to be robust to small changes and was extensively applied for quantitative studies of these structures (e.g.<a href="#pone.0001901-Zamir3">[11]</a>, <a href="#pone.0001901-Kam1">[22]</a>, <a href="#pone.0001901-Lichtenstein1">[33]</a>).</p>


<h4>Data preparation</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>For each labeling set, five values, corresponding to fluorescence intensities of the five labeled components in the filtered images, are associated with each pixel (“pixel color vectors”: V(Set,Img,Pix(x,y),Col) where Set = A–D labeling sets; Img = 1–20 images; Col = 1–5 labeled components). These vectors were pooled from images of 20 cells for each labeling set (4 cells for each of the 5 treatments), and organized as a matrix with rows corresponding to pixels and five columns corresponding to the labeled components. Rows corresponding to pixels with intensity below threshold levels for all colors (as explained above in “Image processing”) were not included. Each pixel can be considered as a point in a five-dimensional composition space, in which each dimension corresponds to one of the labeled components. Pixels with similar intensity pattern for the five components will be close to each other in such a space, and hence cluster analysis can identify groups of pixels with similar molecular content.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Before clustering the pixels on the basis of their molecular composition, two steps of normalization should be performed. The goal of the first normalization is to give all labeled components equal importance (i.e. “weights”) in the calculation of distances between compositions. Since different components are labeled using different reagents and fluorophores they can have different labeling efficiencies (i.e. emitted light intensity per epitope). Higher labeling efficiency leads to proportionally higher average intensity and differences of intensities between two concentrations of antigens. To make the intensity differences independent of the labeling efficiencies, the intensity of each color (i.e. each column in the matrix) was scaled by the average color intensity: V<sub>s</sub> = V(Set,Img,Pix(x,y),Col)/ V<sub>avg</sub>(Set, Col) where V<sub>avg</sub> is the average color intensity for all color values above threshold in all pixels of all images in the labeled set.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>The purpose of the second normalization step is to define the composition space by the stoichometric ratio between the components, irrespective of their total amount. Therefore, the length of each pixel vector (i.e. of each row in the matrix) was normalized to one, so that each component will be defined by the fractional intensity (V<sub>sn</sub> = V<sub>s</sub>(Set,Img,Pix(x,y),Col)/Norm(Set,Img,Pix(​x,y)),where Norm(Set,Img,Pix(x,y)) is the square root of the sum of squares for the 5 Color components of each vector V<sub>s</sub>). Without this normalization step the clustering is dominated by total-intensity similarities, and therefore becomes insensitive to stoichometric ratio similarities (data not shown). Associated with each pixel vector, the corresponding image coordinates were saved for a later reconstruction of the compositional images.</p>


<h4>Cluster analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Cluster analysis was applied to the rows of the normalized matrix (corresponding to pixels from all 20 multicolor images of one labeling set) in order to find groups of pixels with similar compositions. Pooling pixels from images of all the cells under the different treatments into one data set (of the order of 10<sup>6</sup> pixels) facilitates quantitative comparison of the results. The clustering task was achieved in two steps. In the first step the data was clustered by a top-down clustering algorithm that can handle large data sets at relatively short computation time. The algorithm splits recursively each population of pixels into two clusters that minimize the sum of Euclidean distances to their centers-of-mass, creating <em>2<sup>k</sup></em> clusters following <em>k</em> splits<a href="#pone.0001901-Alon1">[24]</a>. To ensure sufficient separation between distinct compositions, an excessive splitting was first performed, ending up with 32 clusters. In the second step, these 32 clusters were processed by bottom-up hierarchical clustering. This algorithm finds the two clusters with the shortest Euclidean distance between their centers-of-mass, merges them to a new cluster, and repeats itself recursively till all clusters are merged. The significance of each merging step along this process (i.e. each node in the clustering dendrogram, as in <a href="#pone-0001901-g002">Fig. 2C</a> and Supplementary <a href="#pone.0001901.s006">Fig. S5</a>) was evaluated based on the multi-dimensional distance between the merged clusters (as apparent in the dendrograms in <a href="#pone-0001901-g002">Fig. 2C</a> and Supplementary <a href="#pone.0001901.s006">Fig. S5</a>). The merging was further monitored visually on the resulting compositional images, and approved only if the two clusters involved were also spatially intermixed. This process yielded 8–12 final clusters for each labeling set. Compositional signatures were defined as the average cluster composition (i.e. its center-of-mass). Images presenting the spatial distribution of signatures within cells were created by assigning to each pixel a color indicating its cluster identity. Cluster analyses and construction of composition images were performed by programs written within the Image Visualization Environment (University of California San Francisco, <a href="http://www.msg.ucsf.edu/IVE/">http://www.msg.ucsf.edu/IVE/</a>) and Matlab (The MathWorks, Inc. MA).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001901"></div><a name="pone.0001901.s001" id="pone.0001901.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s001">Supplementary Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Compositional signatures.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.14 MB DOC)</p>
<a name="pone.0001901.s002" id="pone.0001901.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Sub-cellular localization of compositional clusters (labeling set A). Sub-cellular localization of the compositional clusters in REF52 cells before (non-treated) or after treatment with the Rho-kinase inhibitor Y-27632 without recovery (Y-27632) or with recovery of different durations. The cells were labeled for vinculin, paxillin, α-actinin, β<sub>3</sub>-integrin and actin (labeling set A) as described in <a href="#s4">Materials and Methods</a>. Each column shows the 4 cells sampled for the indicated treatment. Each pixel is colored according to its cluster assignment, as indicated by the color-code on the right. The numbers and the colors of the signatures are consistent with <a href="#pone-0001901-g003">Figure 3</a>. Scale bar, 10 µm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(9.64 MB TIF)</p>
<a name="pone.0001901.s003" id="pone.0001901.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Sub-cellular localization of compositional clusters (labeling set B). As Supplementary <a href="#pone.0001901.s002">Figure S1</a>, with REF52 cells labeled for zyxin, paxillin, α-actinin, β<sub>3</sub>-integrin and actin (labeling set B).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(10.16 MB TIF)</p>
<a name="pone.0001901.s004" id="pone.0001901.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Sub-cellular localization of compositional clusters (labeling set C). As Supplementary <a href="#pone.0001901.s002">Figure S1</a>, with REF52 cells labeled for FAK, paxillin, α-actinin, β<sub>3</sub>-integrin and actin (labeling set C).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(10.06 MB TIF)</p>
<a name="pone.0001901.s005" id="pone.0001901.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Sub-cellular localization of compositional clusters (labeling set D). As Supplementary <a href="#pone.0001901.s002">Figure S1</a>, with REF52 cells labeled for vinculin, paxillin, PY, β<sub>3</sub>-integrin and actin (labeling set D).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(8.49 MB TIF)</p>
<a name="pone.0001901.s006" id="pone.0001901.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s006">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Hierarchical merging of over-divided clusters. Bottom-up merging of the 32 over-divided clusters (obtained from the top-down clustering step) for each labeling set, as performed by the hierarchical clustering algorithm. The dendrograms indicate the order of the merging and the distance between the merged clusters. The colored circles mark the nodes that correspond to the final clusters.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.46 MB TIF)</p>
<a name="pone.0001901.s007" id="pone.0001901.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s007">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Organization of subdomains within single focal-adhesions. Focal-adhesions of non-treated cells, labeled for components of set A (magnified inserts from Supplementary <a href="#pone.0001901.s002">Fig. S1</a>). Note: (i) the diversity between cells, (ii) the high similarity between focal-adhesions in the same cell and (iii) the conserved order between signatures (A2-A3-A9-A8-stress-fibers) along the long focal-adhesions axes. Scale bar, 5 µm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(1.33 MB TIF)</p>
<a name="pone.0001901.s008" id="pone.0001901.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s008">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Visualization of multicolor data by superposition versus compositional imaging. Five-color images of a single cell labeled for vinculin, paxillin, α-actinin, β<sub>3</sub>-integrin, and actin (as shown in <a href="#pone-0001901-g001">Fig. 1</a>) were superimposed with different combinations of artificial colors, as indicated at the bottom (images 1–15). Alternatively (image “CI”), the same 5-color data was subjected to compositional imaging (as shown in <a href="#pone-0001901-g002">Fig. 2</a>). Scale bar, 10 µm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(8.80 MB TIF)</p>
<a name="pone.0001901.s009" id="pone.0001901.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001901.s009">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Example of original and filtered images in 16-bits TIFF format. An example of original images (upper row) and their corresponding filtered images (lower row) of a cell labeled for labeling set A (i.e. from left to right, vinculin, paxillin, α-actinin, β<sub>3</sub>-integrin and actin), in 16-bits TIFF format. This non-compressed presentation pixels retain their true values (as acquired and used for the analysis), and can be viewed using programs that support 16-bits TIFF format, such as ImageJ.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001901.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(4.92 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>This work is dedicated to the memory of Ahuva (Berta) Zamir. We would like to thank Paul Milman from CHROMA for the generous gift of the QUINT filters set, and Tova Volberg for her assistance. BG is the Erwin Neter Professor for Cell and Tumor Biology. ZK is the Israel Pollak Professor of Biophysics.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BG ZK EZ. Performed the experiments: EZ. Analyzed the data: ZK EZ. Contributed reagents/materials/analysis tools: ZK EZ. Wrote the paper: BG ZK EZ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001901-Irish1" id="pone.0001901-Irish1"></a>Irish JM, Hovland R, Krutzik PO, Perez OD, Bruserud O, et al.  (2004) Single Cell Profiling of Potentiated Phospho-Protein Networks in Cancer Cells. Cell  118: 217–228.  <ul class="find" data-citedArticleID="1083818" data-doi="10.1016/j.cell.2004.06.028"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.06.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+Cell+Profiling+of+Potentiated+Phospho-Protein+Networks+in+Cancer+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+Cell+Profiling+of+Potentiated+Phospho-Protein+Networks+in+Cancer+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001901-DeRosa1" id="pone.0001901-DeRosa1"></a>De Rosa SC, Herzenberg LA, Roederer M (2001) 11-color, 13-parameter flow cytometry: identification of human naive T cells by phenotype, function, and T-cell receptor diversity. Nat Med  7: 245–248.  <ul class="find" data-citedArticleID="1083808"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=11-color%2C+13-parameter+flow+cytometry%3A+identification+of+human+naive+T+cells+by+phenotype%2C+function%2C+and+T-cell+receptor+diversity.&amp;auth=&amp;atitle=11-color%2C+13-parameter+flow+cytometry%3A+identification+of+human+naive+T+cells+by+phenotype%2C+function%2C+and+T-cell+receptor+diversity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=11-color%2C+13-parameter+flow+cytometry%3A+identification+of+human+naive+T+cells+by+phenotype%2C+function%2C+and+T-cell+receptor+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%2211-color%2C+13-parameter+flow+cytometry%3A+identification+of+human+naive+T+cells+by+phenotype%2C+function%2C+and+T-cell+receptor+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001901-Mallick1" id="pone.0001901-Mallick1"></a>Mallick P, Schirle M, Chen SS, Flory MR, Lee H, et al.  (2007) Computational prediction of proteotypic peptides for quantitative proteomics. Nat Biotechnol  25: 125–131.  <ul class="find" data-citedArticleID="1083830" data-doi="10.1038/nbt1275"><li><a href="http://dx.doi.org/10.1038/nbt1275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+prediction+of+proteotypic+peptides+for+quantitative+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+prediction+of+proteotypic+peptides+for+quantitative+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001901-Gygi1" id="pone.0001901-Gygi1"></a>Gygi SP, Rist B, Gerber SA, Turecek F, Gelb MH, et al.  (1999) Quantitative analysis of complex protein mixtures using isotope-coded affinity tags. Nat Biotechnol  17: 994–999.  <ul class="find" data-citedArticleID="1083814" data-doi="10.1038/13690"><li><a href="http://dx.doi.org/10.1038/13690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+analysis+of+complex+protein+mixtures+using+isotope-coded+affinity+tags." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+analysis+of+complex+protein+mixtures+using+isotope-coded+affinity+tags.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001901-Rhodes1" id="pone.0001901-Rhodes1"></a>Rhodes DR, Chinnaiyan AM (2005) Integrative analysis of the cancer transcriptome. Nat Genet 37 Suppl S31–37.  <ul class="find" data-citedArticleID="1083838" data-doi="10.1038/ng1570"><li><a href="http://dx.doi.org/10.1038/ng1570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrative+analysis+of+the+cancer+transcriptome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrative+analysis+of+the+cancer+transcriptome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001901-Kingsmore1" id="pone.0001901-Kingsmore1"></a>Kingsmore SF (2006) Multiplexed protein measurement: technologies and applications of protein and antibody arrays. Nat Rev Drug Discov  5: 310–320.  <ul class="find" data-citedArticleID="1083822" data-doi="10.1038/nrd2006"><li><a href="http://dx.doi.org/10.1038/nrd2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplexed+protein+measurement%3A+technologies+and+applications+of+protein+and+antibody+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplexed+protein+measurement%3A+technologies+and+applications+of+protein+and+antibody+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001901-Zamir1" id="pone.0001901-Zamir1"></a>Zamir E, Geiger B (2001) Molecular complexity and dynamics of cell-matrix adhesions. J Cell Sci  114: 3583–3590.  <ul class="find" data-citedArticleID="1083854"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+complexity+and+dynamics+of+cell-matrix+adhesions.&amp;auth=&amp;atitle=Molecular+complexity+and+dynamics+of+cell-matrix+adhesions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+complexity+and+dynamics+of+cell-matrix+adhesions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+complexity+and+dynamics+of+cell-matrix+adhesions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001901-ZaidelBar1" id="pone.0001901-ZaidelBar1"></a>Zaidel-Bar R, Itzkovitz S, Ma'ayan A, Iyengar R, Geiger B (2007) Functional atlas of the integrin adhesome. Nat Cell Biol  9: 858–867.  <ul class="find" data-citedArticleID="1083846" data-doi="10.1038/ncb0807-858"><li><a href="http://dx.doi.org/10.1038/ncb0807-858" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+atlas+of+the+integrin+adhesome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+atlas+of+the+integrin+adhesome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001901-ZaidelBar2" id="pone.0001901-ZaidelBar2"></a>Zaidel-Bar R, Cohen M, Addadi L, Geiger B (2004) Hierarchical assembly of cell-matrix adhesion complexes. Biochem Soc Trans  32: 416–420.  <ul class="find" data-citedArticleID="1083848" data-doi="10.1042/bst0320416"><li><a href="http://dx.doi.org/10.1042/bst0320416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchical+assembly+of+cell-matrix+adhesion+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchical+assembly+of+cell-matrix+adhesion+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001901-Zamir2" id="pone.0001901-Zamir2"></a>Zamir E, Katz M, Posen Y, Erez N, Yamada KM, et al.  (2000) Dynamics and segregation of cell-matrix adhesions in cultured fibroblasts. Nat Cell Biol  2: 191–196.  <ul class="find" data-citedArticleID="1083856" data-doi="10.1038/35008607"><li><a href="http://dx.doi.org/10.1038/35008607" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamics+and+segregation+of+cell-matrix+adhesions+in+cultured+fibroblasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamics+and+segregation+of+cell-matrix+adhesions+in+cultured+fibroblasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001901-Zamir3" id="pone.0001901-Zamir3"></a>Zamir E, Katz BZ, Aota S, Yamada KM, Geiger B, et al.  (1999) Molecular diversity of cell-matrix adhesions. J Cell Sci  112 (Pt 11): 1655–1669.  <ul class="find" data-citedArticleID="1083858"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+diversity+of+cell-matrix+adhesions.&amp;auth=&amp;atitle=Molecular+diversity+of+cell-matrix+adhesions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+diversity+of+cell-matrix+adhesions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+diversity+of+cell-matrix+adhesions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001901-ZaidelBar3" id="pone.0001901-ZaidelBar3"></a>Zaidel-Bar R, Ballestrem C, Kam Z, Geiger B (2003) Early molecular events in the assembly of matrix adhesions at the leading edge of migrating cells. J Cell Sci  116: 4605–4613.  <ul class="find" data-citedArticleID="1083850" data-doi="10.1242/jcs.00792"><li><a href="http://dx.doi.org/10.1242/jcs.00792" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+molecular+events+in+the+assembly+of+matrix+adhesions+at+the+leading+edge+of+migrating+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+molecular+events+in+the+assembly+of+matrix+adhesions+at+the+leading+edge+of+migrating+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001901-Borrmann1" id="pone.0001901-Borrmann1"></a>Borrmann CM, Mertens C, Schmidt A, Langbein L, Kuhn C, et al.  (2000) Molecular diversity of plaques of epithelial-adhering junctions. Ann N Y Acad Sci  915: 144–150.  <ul class="find" data-citedArticleID="1083802" data-doi="10.1111/j.1749-6632.2000.tb05237.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.2000.tb05237.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+diversity+of+plaques+of+epithelial-adhering+junctions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+diversity+of+plaques+of+epithelial-adhering+junctions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001901-Perret1" id="pone.0001901-Perret1"></a>Perret E, Lakkaraju A, Deborde S, Schreiner R, Rodriguez-Boulan E (2005) Evolving endosomes: how many varieties and why? Curr Opin Cell Biol  17: 423–434.  <ul class="find" data-citedArticleID="1083836" data-doi="10.1016/j.ceb.2005.06.008"><li><a href="http://dx.doi.org/10.1016/j.ceb.2005.06.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolving+endosomes%3A+how+many+varieties+and+why%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolving+endosomes%3A+how+many+varieties+and+why%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001901-Michalet1" id="pone.0001901-Michalet1"></a>Michalet X, Pinaud FF, Bentolila LA, Tsay JM, Doose S, et al.  (2005) Quantum dots for live cells, in vivo imaging, and diagnostics. Science  307: 538–544.  <ul class="find" data-citedArticleID="1083832" data-doi="10.1126/science.1104274"><li><a href="http://dx.doi.org/10.1126/science.1104274" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantum+dots+for+live+cells%2C+in+vivo+imaging%2C+and+diagnostics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantum+dots+for+live+cells%2C+in+vivo+imaging%2C+and+diagnostics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001901-Gao1" id="pone.0001901-Gao1"></a>Gao X, Yang L, Petros JA, Marshall FF, Simons JW, et al.  (2005) In vivo molecular and cellular imaging with quantum dots. Curr Opin Biotechnol  16: 63–72.  <ul class="find" data-citedArticleID="1083810" data-doi="10.1016/j.copbio.2004.11.003"><li><a href="http://dx.doi.org/10.1016/j.copbio.2004.11.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+molecular+and+cellular+imaging+with+quantum+dots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+molecular+and+cellular+imaging+with+quantum+dots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001901-Shaner1" id="pone.0001901-Shaner1"></a>Shaner NC, Steinbach PA, Tsien RY (2005) A guide to choosing fluorescent proteins. Nat Methods  2: 905–909.  <ul class="find" data-citedArticleID="1083844" data-doi="10.1038/nmeth819"><li><a href="http://dx.doi.org/10.1038/nmeth819" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+guide+to+choosing+fluorescent+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+guide+to+choosing+fluorescent+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001901-Hutter1" id="pone.0001901-Hutter1"></a>Hutter H (2004) Five-colour in vivo imaging of neurons in Caenorhabditis elegans. J Microsc  215: 213–218.  <ul class="find" data-citedArticleID="1083816" data-doi="10.1111/j.0022-2720.2004.01367.x"><li><a href="http://dx.doi.org/10.1111/j.0022-2720.2004.01367.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Five-colour+in+vivo+imaging+of+neurons+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Five-colour+in+vivo+imaging+of+neurons+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001901-Zimmermann1" id="pone.0001901-Zimmermann1"></a>Zimmermann T, Rietdorf J, Pepperkok R (2003) Spectral imaging and its applications in live cell microscopy. FEBS Lett  546: 87–92.  <ul class="find" data-citedArticleID="1083860" data-doi="10.1016/s0014-5793(03)00521-0"><li><a href="http://dx.doi.org/10.1016/s0014-5793(03)00521-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spectral+imaging+and+its+applications+in+live+cell+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spectral+imaging+and+its+applications+in+live+cell+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001901-PadillaNash1" id="pone.0001901-PadillaNash1"></a>Padilla-Nash HM, Barenboim-Stapleton L, Difilippantonio MJ, Ried T (2006) Spectral karyotyping analysis of human and mouse chromosomes. Nat Protoc  1: 3129–3142.  <ul class="find" data-citedArticleID="1083834" data-doi="10.1038/nprot.2006.358"><li><a href="http://dx.doi.org/10.1038/nprot.2006.358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spectral+karyotyping+analysis+of+human+and+mouse+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spectral+karyotyping+analysis+of+human+and+mouse+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001901-Schubert1" id="pone.0001901-Schubert1"></a>Schubert W, Bonnekoh B, Pommer AJ, Philipsen L, Bockelmann R, et al.  (2006) Analyzing proteome topology and function by automated multidimensional fluorescence microscopy. Nat Biotechnol  24: 1270–1278.  <ul class="find" data-citedArticleID="1083842" data-doi="10.1038/nbt1250"><li><a href="http://dx.doi.org/10.1038/nbt1250" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analyzing+proteome+topology+and+function+by+automated+multidimensional+fluorescence+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analyzing+proteome+topology+and+function+by+automated+multidimensional+fluorescence+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001901-Kam1" id="pone.0001901-Kam1"></a>Kam Z, Zamir E, Geiger B (2001) Probing molecular processes in live cells by quantitative multidimensional microscopy. Trends Cell Biol  11: 329–334.  <ul class="find" data-citedArticleID="1083820" data-doi="10.1016/s0962-8924(01)02067-0"><li><a href="http://dx.doi.org/10.1016/s0962-8924(01)02067-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+molecular+processes+in+live+cells+by+quantitative+multidimensional+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+molecular+processes+in+live+cells+by+quantitative+multidimensional+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001901-Ma1" id="pone.0001901-Ma1"></a>Ma B, He F, Jablonska J, Winkelbach S, Lindenmaier W, et al.  (2007) Six-color segmentation of multicolor images in the infection studies of Listeria monocytogenes. Microsc Res Tech  70: 171–178.  <ul class="find" data-citedArticleID="1083828" data-doi="10.1002/jemt.20401"><li><a href="http://dx.doi.org/10.1002/jemt.20401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Six-color+segmentation+of+multicolor+images+in+the+infection+studies+of+Listeria+monocytogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Six-color+segmentation+of+multicolor+images+in+the+infection+studies+of+Listeria+monocytogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001901-Alon1" id="pone.0001901-Alon1"></a>Alon U, Barkai N, Notterman DA, Gish K, Ybarra S, et al.  (1999) Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays. Proc Natl Acad Sci U S A  96: 6745–6750.  <ul class="find" data-citedArticleID="1083796"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays.&amp;auth=&amp;atitle=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001901-ZaidelBar4" id="pone.0001901-ZaidelBar4"></a>Zaidel-Bar R, Milo R, Kam Z, Geiger B (2007) A paxillin tyrosine phosphorylation switch regulates the assembly and form of cell-matrix adhesions. J Cell Sci  120: 137–148.  <ul class="find" data-citedArticleID="1083852" data-doi="10.1242/jcs.03314"><li><a href="http://dx.doi.org/10.1242/jcs.03314" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+paxillin+tyrosine+phosphorylation+switch+regulates+the+assembly+and+form+of+cell-matrix+adhesions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+paxillin+tyrosine+phosphorylation+switch+regulates+the+assembly+and+form+of+cell-matrix+adhesions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001901-Ballestrem1" id="pone.0001901-Ballestrem1"></a>Ballestrem C, Hinz B, Imhof BA, Wehrle-Haller B (2001) Marching at the front and dragging behind: differential alphaVbeta3-integrin turnover regulates focal adhesion behavior. J Cell Biol  155: 1319–1332.  <ul class="find" data-citedArticleID="1083800" data-doi="10.1083/jcb.200107107"><li><a href="http://dx.doi.org/10.1083/jcb.200107107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Marching+at+the+front+and+dragging+behind%3A+differential+alphaVbeta3-integrin+turnover+regulates+focal+adhesion+behavior." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Marching+at+the+front+and+dragging+behind%3A+differential+alphaVbeta3-integrin+turnover+regulates+focal+adhesion+behavior.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001901-Riveline1" id="pone.0001901-Riveline1"></a>Riveline D, Zamir E, Balaban NQ, Schwarz US, Ishizaki T, et al.  (2001) Focal contacts as mechanosensors: externally applied local mechanical force induces growth of focal contacts by an mDia1-dependent and ROCK-independent mechanism. J Cell Biol  153: 1175–1186.  <ul class="find" data-citedArticleID="1083840" data-doi="10.1083/jcb.153.6.1175"><li><a href="http://dx.doi.org/10.1083/jcb.153.6.1175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Focal+contacts+as+mechanosensors%3A+externally+applied+local+mechanical+force+induces+growth+of+focal+contacts+by+an+mDia1-dependent+and+ROCK-independent+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Focal+contacts+as+mechanosensors%3A+externally+applied+local+mechanical+force+induces+growth+of+focal+contacts+by+an+mDia1-dependent+and+ROCK-independent+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001901-Altelaar1" id="pone.0001901-Altelaar1"></a>Altelaar AF, Luxembourg SL, McDonnell LA, Piersma SR, Heeren RM (2007) Imaging mass spectrometry at cellular length scales. Nat Protoc  2: 1185–1196.  <ul class="find" data-citedArticleID="1083798" data-doi="10.1038/nprot.2007.117"><li><a href="http://dx.doi.org/10.1038/nprot.2007.117" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Imaging+mass+spectrometry+at+cellular+length+scales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Imaging+mass+spectrometry+at+cellular+length+scales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001901-Lehnert1" id="pone.0001901-Lehnert1"></a>Lehnert D, Wehrle-Haller B, David C, Weiland U, Ballestrem C, Imhof BA, Bastmeyer M (2004) Cell behaviour on micropatterned substrata: limits of extracellular matrix geometry for spreading and adhesion. J Cell Sci  117: 41–52.  <ul class="find" data-citedArticleID="1083824" data-doi="10.1242/jcs.00836"><li><a href="http://dx.doi.org/10.1242/jcs.00836" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+behaviour+on+micropatterned+substrata%3A+limits+of+extracellular+matrix+geometry+for+spreading+and+adhesion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+behaviour+on+micropatterned+substrata%3A+limits+of+extracellular+matrix+geometry+for+spreading+and+adhesion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001901-Cluzel1" id="pone.0001901-Cluzel1"></a>Cluzel C, Saltel F, Lussi J, Paulhe F, Imhof BA, Wehrle-Haller B (2005) The mechanisms and dynamics of (alpha)v(beta)3 integrin clustering in living cells. J Cell Biol  171: 383–92.  <ul class="find" data-citedArticleID="1083806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+mechanisms+and+dynamics+of+%28alpha%29v%28beta%293+integrin+clustering+in+living+cells.&amp;auth=&amp;atitle=The+mechanisms+and+dynamics+of+%28alpha%29v%28beta%293+integrin+clustering+in+living+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mechanisms+and+dynamics+of+%28alpha%29v%28beta%293+integrin+clustering+in+living+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mechanisms+and+dynamics+of+%28alpha%29v%28beta%293+integrin+clustering+in+living+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001901-Goffin1" id="pone.0001901-Goffin1"></a>Goffin JM, Pittet P, Csucs G, Lussi JW, Meister JJ, Hinz B (2006) Focal adhesion size controls tension-dependent recruitment of alpha-smooth muscle actin to stress fibers. J Cell Biol  172: 259–68.  <ul class="find" data-citedArticleID="1083812" data-doi="10.1083/jcb.200506179"><li><a href="http://dx.doi.org/10.1083/jcb.200506179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Focal+adhesion+size+controls+tension-dependent+recruitment+of+alpha-smooth+muscle+actin+to+stress+fibers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Focal+adhesion+size+controls+tension-dependent+recruitment+of+alpha-smooth+muscle+actin+to+stress+fibers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001901-CavalcantiAdam1" id="pone.0001901-CavalcantiAdam1"></a>Cavalcanti-Adam EA, Volberg T, Micoulet A, Kessler H, Geiger B, Spatz JP (2007) Cell spreading and focal adhesion dynamics are regulated by spacing of integrin ligands. Biophys J  92: 2964–74.  <ul class="find" data-citedArticleID="1083804" data-doi="10.1529/biophysj.106.089730"><li><a href="http://dx.doi.org/10.1529/biophysj.106.089730" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+spreading+and+focal+adhesion+dynamics+are+regulated+by+spacing+of+integrin+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+spreading+and+focal+adhesion+dynamics+are+regulated+by+spacing+of+integrin+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001901-Lichtenstein1" id="pone.0001901-Lichtenstein1"></a>Lichtenstein N, Geiger B, Kam Z (2003) Quantitative analysis of cytoskeletal organization by digital fluorescent microscopy. Cytometry A  54: 8–18.  <ul class="find" data-citedArticleID="1083826" data-doi="10.1002/cyto.a.10053"><li><a href="http://dx.doi.org/10.1002/cyto.a.10053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+analysis+of+cytoskeletal+organization+by+digital+fluorescent+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+analysis+of+cytoskeletal+organization+by+digital+fluorescent+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.XML" value="104081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.PDF" value="997279"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g001.PNG_L" value="763856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g001.PNG_M" value="168010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g001.PNG_S" value="14530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g001.TIF" value="1023034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g001.PNG_I" value="55739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t001.PNG_L" value="225743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t001.PNG_M" value="121789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t001.PNG_S" value="13049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t001.TIF" value="1162120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t001.PNG_I" value="37960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g002.PNG_L" value="1468281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g002.PNG_M" value="253425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g002.PNG_S" value="18786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g002.TIF" value="2225158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g002.PNG_I" value="85316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g003.PNG_L" value="5473088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g003.PNG_M" value="264475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g003.PNG_S" value="18935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g003.TIF" value="9143462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g003.PNG_I" value="105241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t002.PNG_L" value="191805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t002.PNG_M" value="106331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t002.PNG_S" value="14586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t002.TIF" value="1045444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.t002.PNG_I" value="33308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g004.PNG_L" value="7576492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g004.PNG_M" value="585834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g004.PNG_S" value="21246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g004.TIF" value="10218242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g004.PNG_I" value="179235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g005.PNG_L" value="169612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g005.PNG_M" value="97722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g005.PNG_S" value="14771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g005.TIF" value="291742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.g005.PNG_I" value="50510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s001.DOC" value="135168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s002.TIF" value="9703532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s003.TIF" value="10220718"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s004.TIF" value="10099502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s005.TIF" value="8578452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s006.TIF" value="623530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s007.TIF" value="1341230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s008.TIF" value="9209556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001901.s009.TIF" value="4927269"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001901&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001901" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001901&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001901&volume=&issue=&title=Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions&author_name=Eli%20Zamir%2C%20Benjamin%20Geiger%2C%20Zvi%20Kam&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001901" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001901" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001901" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001901&amp;t=Quantitative%20Multicolor%20Compositional%20Imaging%20Resolves%20Molecular%20Domains%20in%20Cell-Matrix%20Adhesions" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001901&title=Quantitative%20Multicolor%20Compositional%20Imaging%20Resolves%20Molecular%20Domains%20in%20Cell-Matrix%20Adhesions&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001901&amp;title=Quantitative%20Multicolor%20Compositional%20Imaging%20Resolves%20Molecular%20Domains%20in%20Cell-Matrix%20Adhesions" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001901" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001901&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Quantitative Multicolor Compositional Imaging Resolves Molecular Domains in Cell-Matrix Adhesions';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001901';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Quantitative%20Multicolor%20Compositional%20Imaging%20Resolves%20Molecular%20Domains%20in%20Cell-Matrix%20Adhesions http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001901" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001901" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Actins%22" title="Search for articles in the subject area:'Actins'"><div class="flagText">Actins</div></a>
              <div data-categoryid="17565" data-articleid="26626"
                   data-categoryname="Actins"
                   class="flagImage" title="Flag 'Actins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+adhesion%22" title="Search for articles in the subject area:'Cell adhesion'"><div class="flagText">Cell adhesion</div></a>
              <div data-categoryid="20007" data-articleid="26626"
                   data-categoryname="Cell adhesion"
                   class="flagImage" title="Flag 'Cell adhesion' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Composite+images%22" title="Search for articles in the subject area:'Composite images'"><div class="flagText">Composite images</div></a>
              <div data-categoryid="17367" data-articleid="26626"
                   data-categoryname="Composite images"
                   class="flagImage" title="Flag 'Composite images' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cytoskeleton%22" title="Search for articles in the subject area:'Cytoskeleton'"><div class="flagText">Cytoskeleton</div></a>
              <div data-categoryid="18893" data-articleid="26626"
                   data-categoryname="Cytoskeleton"
                   class="flagImage" title="Flag 'Cytoskeleton' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence+imaging%22" title="Search for articles in the subject area:'Fluorescence imaging'"><div class="flagText">Fluorescence imaging</div></a>
              <div data-categoryid="17303" data-articleid="26626"
                   data-categoryname="Fluorescence imaging"
                   class="flagImage" title="Flag 'Fluorescence imaging' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Focal+adhesions%22" title="Search for articles in the subject area:'Focal adhesions'"><div class="flagText">Focal adhesions</div></a>
              <div data-categoryid="23563" data-articleid="26626"
                   data-categoryname="Focal adhesions"
                   class="flagImage" title="Flag 'Focal adhesions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Imaging+techniques%22" title="Search for articles in the subject area:'Imaging techniques'"><div class="flagText">Imaging techniques</div></a>
              <div data-categoryid="2295" data-articleid="26626"
                   data-categoryname="Imaging techniques"
                   class="flagImage" title="Flag 'Imaging techniques' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Integrins%22" title="Search for articles in the subject area:'Integrins'"><div class="flagText">Integrins</div></a>
              <div data-categoryid="19417" data-articleid="26626"
                   data-categoryname="Integrins"
                   class="flagImage" title="Flag 'Integrins' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7250'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2395'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8969&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=573">Referee Comments: Referee 1 (Maddy Parsons)</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=7053">Referee Comments: Referee 2 (Joshua Rappoport)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
