

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Biomedically Enriched Collection of 7000 Human ORF Clones</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001528"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001528" />

    <meta name="citation_title" content="A Biomedically Enriched Collection of 7000 Human ORF Clones"/>
    <meta itemprop="name" content="A Biomedically Enriched Collection of 7000 Human ORF Clones"/>

      <meta name="citation_author" content="Andreas Rolfs"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Yanhui Hu"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Lars Ebert"/>
            <meta name="citation_author_institution" content="Deutsches Ressourcenzentrum fuer Genomforschung (RZPD), Heidelberg, Germany"/>
      <meta name="citation_author" content="Dietmar Hoffmann"/>
            <meta name="citation_author_institution" content="Sanofi-Aventis, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Dongmei Zuo"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Niro Ramachandran"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Jacob Raphael"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Fontina Kelley"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Seamus McCarron"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Daniel A. Jepson"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Binghua Shen"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Munira M. A. Baqui"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Joseph Pearlberg"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Elena Taycher"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Craig DeLoughery"/>
            <meta name="citation_author_institution" content="Sanofi-Aventis, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Andreas Hoerlein"/>
            <meta name="citation_author_institution" content="Deutsches Ressourcenzentrum fuer Genomforschung (RZPD), Heidelberg, Germany"/>
      <meta name="citation_author" content="Bernhard Korn"/>
            <meta name="citation_author_institution" content="Deutsches Ressourcenzentrum fuer Genomforschung (RZPD), Heidelberg, Germany"/>
      <meta name="citation_author" content="Joshua LaBaer"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2008/1/30"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001528.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1528"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Systematic recovery and analysis of full-ORF human cDNA clones.; citation_author=A Baross; citation_author=YS Butterfield; citation_author=SM Coughlin; citation_author=T Zeng; citation_author=M Griffith; citation_journal_title=Genome Res; citation_volume=14; citation_number=1; citation_pages=2083-2092; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC).; citation_author=DS Gerhard; citation_author=L Wagner; citation_author=EA Feingold; citation_author=CM Shenmen; citation_author=LH Grouse; citation_journal_title=Genome Res; citation_volume=14; citation_number=2; citation_pages=2121-2127; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetics. Working the (gene count) numbers: finally, a firm answer?; citation_author=E Pennisi; citation_journal_title=Science; citation_volume=316; citation_number=3; citation_pages=1113; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A genome annotation-driven approach to cloning the human ORFeome.; citation_author=JE Collins; citation_author=CL Wright; citation_author=CA Edwards; citation_author=MP Davis; citation_author=JA Grinham; citation_journal_title=Genome Biol; citation_volume=5; citation_number=4; citation_pages=R84; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Building a human kinase gene repository: bioinformatics, molecular cloning, and functional validation.; citation_author=J Park; citation_author=Y Hu; citation_author=TV Murthy; citation_author=F Vannberg; citation_author=B Shen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=5; citation_pages=8114-8119; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences.; citation_author=RL Strausberg; citation_author=EA Feingold; citation_author=LH Grouse; citation_author=JG Derge; citation_author=RD Klausner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=6; citation_pages=16899-16903; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Human ORFeome version 1.1: a platform for reverse proteomics.; citation_author=JF Rual; citation_author=T Hirozane-Kishikawa; citation_author=T Hao; citation_author=N Bertin; citation_author=S Li; citation_journal_title=Genome Res; citation_volume=14; citation_number=7; citation_pages=2128-2135; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=hORFeome v3.1: a resource of human open reading frames representing over 10,000 human genes.; citation_author=P Lamesch; citation_author=N Li; citation_author=S Milstein; citation_author=C Fan; citation_author=T Hao; citation_journal_title=Genomics; citation_volume=89; citation_number=8; citation_pages=307-315; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=From genome to proteome: developing expression clone resources for the human genome.; citation_author=G Temple; citation_author=P Lamesch; citation_author=S Milstein; citation_author=DE Hill; citation_author=L Wagner; citation_journal_title=Hum Mol Genet; citation_volume=15 Spec No 1; citation_number=9; citation_pages=R31-43; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional proteomics approach to investigate the biological activities of cDNAs implicated in breast cancer.; citation_author=AE Witt; citation_author=LM Hines; citation_author=NL Collins; citation_author=Y Hu; citation_author=RN Gunawardane; citation_journal_title=J Proteome Res; citation_volume=5; citation_number=10; citation_pages=599-610; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Analysis of genomic and proteomic data using advanced literature mining.; citation_author=Y Hu; citation_author=LM Hines; citation_author=H Weng; citation_author=D Zuo; citation_author=M Rivera; citation_journal_title=J Proteome Res; citation_volume=2; citation_number=11; citation_pages=405-412; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=NCBI Reference Sequence project: update and current status.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=12; citation_pages=34-37; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=13; citation_pages=D501-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=14; citation_pages=D61-65; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=RefSeq and LocusLink: NCBI gene-centered resources.; citation_author=KD Pruitt; citation_author=DR Maglott; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=15; citation_pages=137-140; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Approaching a complete repository of sequence-verified protein-encoding clones for Saccharomyces cerevisiae.; citation_author=Y Hu; citation_author=A Rolfs; citation_author=B Bhullar; citation_author=TV Murthy; citation_author=C Zhu; citation_journal_title=Genome Res; citation_volume=17; citation_number=16; citation_pages=536-543; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A full-genomic sequence-verified protein-coding gene collection for Francisella tularensis.; citation_author=T Murthy; citation_author=A Rolfs; citation_author=Y Hu; citation_author=Z Shi; citation_author=J Raphael; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=17; citation_pages=e577; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Functional dynamics of PDZ binding domains: a normal-mode analysis.; citation_author=P De Los Rios; citation_author=F Cecconi; citation_author=A Pretre; citation_author=G Dietler; citation_author=O Michielin; citation_journal_title=Biophys J; citation_volume=89; citation_number=18; citation_pages=14-21; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cadherin adhesion depends on a salt bridge at the N-terminus.; citation_author=OJ Harrison; citation_author=EM Corps; citation_author=PJ Kilshaw; citation_journal_title=J Cell Sci; citation_volume=118; citation_number=19; citation_pages=4123-4130; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The surprising complexity of signal sequences.; citation_author=RS Hegde; citation_author=HD Bernstein; citation_journal_title=Trends Biochem Sci; citation_volume=31; citation_number=20; citation_pages=563-571; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Self-assembling protein microarrays.; citation_author=N Ramachandran; citation_author=E Hainsworth; citation_author=B Bhullar; citation_author=S Eisenstein; citation_author=B Rosen; citation_journal_title=Science; citation_volume=305; citation_number=21; citation_pages=86-90; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sequence assembly and finishing methods.; citation_author=R Staden; citation_author=DP Judge; citation_author=JK Bonfield; citation_journal_title=Methods Biochem Anal; citation_volume=43; citation_number=22; citation_pages=303-322; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Primer design for large scale sequencing.; citation_author=S Haas; citation_author=M Vingron; citation_author=A Poustka; citation_author=S Wiemann; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=23; citation_pages=3006-3012; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A novel approach to sequence validating protein expression clones with automated decision making.; citation_author=E Taycher; citation_author=A Rolfs; citation_author=Y Hu; citation_author=D Zuo; citation_author=SE Mohr; citation_journal_title=BMC Bioinformatics; citation_volume=8; citation_number=24; citation_pages=198; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001528" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Biomedically Enriched Collection of 7000 Human ORF Clones"/>
    <meta name="twitter:description" content="We report the production and availability of over 7000 fully sequence verified plasmid ORF clones representing over 3400 unique human genes. These ORF clones were derived using the human MGC collection as template and were produced in two formats: with and without stop codons. Thus, this collection supports the production of either native protein or proteins with fusion tags added to either or both ends. The template clones used to generate this collection were enriched in three ways. First, gene redundancy was removed. Second, clones were selected to represent the best available GenBank reference sequence. Finally, a literature-based software tool was used to evaluate the list of target genes to ensure that it broadly reflected biomedical research interests. The target gene list was compared with 4000 human diseases and over 8500 biological and chemical MeSH classes in &sim;15 Million publications recorded in PubMed at the time of analysis. The outcome of this analysis revealed that relative to the genome and the MGC collection, this collection is enriched for the presence of genes with published associations with a wide range of diseases and biomedical terms without displaying a particular bias towards any single disease or concept. Thus, this collection is likely to be a powerful resource for researchers who wish to study protein function in a set of genes with documented biomedical significance."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001528.g003"/>

  <meta property="og:title" content="A Biomedically Enriched Collection of 7000 Human ORF Clones" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001528" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6071'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6246'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=459&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001528">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001528" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001528&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Biomedically Enriched Collection of 7000 Human ORF Clones
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andreas Rolfs, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yanhui Hu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lars Ebert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Deutsches Ressourcenzentrum fuer Genomforschung (RZPD), Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dietmar Hoffmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Sanofi-Aventis, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dongmei Zuo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Niro Ramachandran, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jacob Raphael, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fontina Kelley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Seamus McCarron, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel A. Jepson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Binghua Shen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Munira M. A. Baqui, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph Pearlberg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elena Taycher, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Craig DeLoughery, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Sanofi-Aventis, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andreas Hoerlein, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Deutsches Ressourcenzentrum fuer Genomforschung (RZPD), Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernhard Korn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Deutsches Ressourcenzentrum fuer Genomforschung (RZPD), Heidelberg, Germany
                </p>

                <p>
                    Current address: Genomics and Proteomics Core Facilities, German Cancer Research Center (DKFZ), Heidelberg, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joshua LaBaer
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Joshua_labaer@hms.harvard.edu">Joshua_labaer@hms.harvard.edu</a></p>

                <p>Affiliation:
                  Harvard Institute of Proteomics, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 30, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001528</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001528" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001528" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001528" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001528" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001528-g001" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001528-t001" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001528-g002" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001528-t002" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001528-g003" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001528-t003" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001528">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001528" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>We report the production and availability of over 7000 fully sequence verified plasmid ORF clones representing over 3400 unique human genes. These ORF clones were derived using the human MGC collection as template and were produced in two formats: with and without stop codons. Thus, this collection supports the production of either native protein or proteins with fusion tags added to either or both ends. The template clones used to generate this collection were enriched in three ways. First, gene redundancy was removed. Second, clones were selected to represent the best available GenBank reference sequence. Finally, a literature-based software tool was used to evaluate the list of target genes to ensure that it broadly reflected biomedical research interests. The target gene list was compared with 4000 human diseases and over 8500 biological and chemical MeSH classes in ~15 Million publications recorded in PubMed at the time of analysis. The outcome of this analysis revealed that relative to the genome and the MGC collection, this collection is enriched for the presence of genes with published associations with a wide range of diseases and biomedical terms without displaying a particular bias towards any single disease or concept. Thus, this collection is likely to be a powerful resource for researchers who wish to study protein function in a set of genes with documented biomedical significance.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rolfs A, Hu Y, Ebert L, Hoffmann D, Zuo D, et al.  (2008) A Biomedically Enriched Collection of 7000 Human ORF Clones. PLoS ONE 3(1):
          e1528.
            doi:10.1371/journal.pone.0001528</p><p><strong>Academic Editor: </strong>Suzannah Rutherford, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received:</strong> October 19, 2007; <strong>Accepted:</strong> November 28, 2007; <strong>Published:</strong> January 30, 2008</p><p><strong>Copyright:</strong> © 2008 Rolfs et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>We would like to thank Sanofi-Aventis and the Breast Cancer Research Foundation for generous support of this project. The project has partly been funded by the German Ministry of Science, NGFN2 (grants 01GR0413 and 01GR0420).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The study of protein function often demands high quality plasmid clones that contain the relevant open reading frames (ORFs) in a format compatible with protein expression. Increasingly, high throughput methods have created the demand for clones that encode a class of proteins of interest or the entire proteome of a species. Functional studies rely on <em>in vivo</em> expression for phenotypic studies or expression and purification by various means for biochemical analysis. Utilizing recombinational cloning vectors and including only the coding sequences, with all untranslated sequences removed, ensures maximum flexibility, including protein expression in a broad experimental range with various tagging options for either end of the protein. In addition, to avoid erroneous or ambiguous results regarding the expressed proteins, it is important that the plasmids are clonal isolates that are fully sequence verified.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>For many eukaryotic species, including humans, the number of protein coding sequences exceeds 15,000 genes, making the production of comprehensive sequence-verified ORF clone collections daunting and expensive. In fact, a complete set of source material for expressed genes in humans does not yet exist <a href="#pone.0001528-Baross1">[1]</a>–<a href="#pone.0001528-Pennisi1">[3]</a>. One strategy is for researchers to focus on (a) meaningful subset(s) of genes for functional studies relevant to the biological questions they wish to address. For a human ORF collection the criteria for selecting genes are mostly driven by researchers' interest and clone availability, resulting often in either collections of special interest <a href="#pone.0001528-Collins1">[4]</a> <a href="#pone.0001528-Park1">[5]</a>, or more ‘random’ lists of genes in collections (RZPD, Invitrogen).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In recent years, a publicly funded project, the Mammalian Gene Collection (MGC), aimed to create for multiple species, but especially for man and mouse, collections of well annotated, fully sequence validated cDNA clones <a href="#pone.0001528-Strausberg1">[6]</a>. However, the MGC clones cannot easily be employed directly in functional proteomics experiments because they are in many different vector backbones and contain 5′ and 3′ untranslated sequences. On the other hand, because they are fully sequenced and well annotated, these clones provide an excellent starting point for creating ORF clones. At least one such ORF set has been made so far, although that set comprises pools of clones that are not sequence verified <a href="#pone.0001528-Rual1">[7]</a> <a href="#pone.0001528-Lamesch1">[8]</a> and thus has potential ambiguity. Currently, there are also four human ORF collections available from commercial distributors that were clonally isolated and at least partially sequence validated. The recently created ORFeome Collaboration (<a href="http://www.orfeomecollaboration.org/">http://www.orfeomecollaboration.org/</a>) <a href="#pone.0001528-Temple1">[9]</a> is a project planned to bring to all researchers an ORF clone collection that provides at least one representative ORF clone for all human genes, similar in quality and scope to the MGC clones, with all clones being fully sequence validated.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>A limitation of the recombinational cloning vectors used for these ORF clones is that each clone must be committed to one of two non-interchangeable formats: <em>closed</em> (with stop codon; can express native protein) or <em>fusion</em> (no stop codon; enables the addition of carboxyl-terminal fusion peptides). As each format has unique advantages not available for the other, the ideal collection would include both.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Previously we reported the production of two smaller human clone sets in the Creator™ system. One set focused on kinase genes, both well-studied as well as novel or hypothetical ones <a href="#pone.0001528-Park1">[5]</a>; the other clone set covered over 1000 genes associated with breast cancer <a href="#pone.0001528-Witt1">[10]</a>, identified in publications using software developed in our group <a href="#pone.0001528-Hu1">[11]</a>. Here we report the production and complete sequence validation of 7000 clones, HLFEX7000 (&gt;3400 unique human genes in two formats), and the distribution of these genes with respect to their relationship to disease and biological terms in publications in PubMed.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Gene Selection</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To make the most useful ORF clone set of the MGC clones, we wished to select an enriched set of genes that is of particular interest to both medicine and biology. In addition, we wished to exclude clones that corresponded to partial gene products and to eliminate redundancy. We first excluded the subset of all MGC clones where the CDS length was less than 90% of the length of the longest corresponding NCBI RefSeq sequence <a href="#pone.0001528-Pruitt1">[12]</a>–<a href="#pone.0001528-Pruitt3">[14]</a>. We then removed redundancy within the MGC clone set, and picked the clone closest to the longest reference sequence by CDS length as a best MGC representative for each gene. This reduced the number of candidate template clones from 13493 to 7992, representing 7992 genes.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Our discussions with researchers indicated that a focused set of genes in both formats (closed and fusion) would be of more value that a large set in only one format. To ensure that our final gene set (Supplementary <a href="#pone.0001528.s001">Table S1</a>) was enriched for genes related to human diseases without any specific bias, the candidate list was used to query MedGene <a href="#pone.0001528-Hu1">[11]</a> for genes associated with about 4000 human diseases. As described, MedGene is an automated literature-mining tool, which comprehensively summarizes and estimates the relative strengths of all human gene-disease relationships reported in Medline/PubMed. The result of this query was compared with queries using either all unique genes represented in MGC or all ~33,000 human genes listed at the time in LocusLink (2004, now: EntrezGene <a href="#pone.0001528-Pruitt4">[15]</a>). As shown for a subset of diseases in <a href="#pone-0001528-g001">Figure 1</a>; <a href="#pone-0001528-t001">Table 1</a> (complete list: Supplementary <a href="#pone.0001528.s002">Table S2</a>), the resulting target list: (a) was highly enriched for the presence of genes with published associations with a wide range of human diseases; (b) had a similar relative ratio among the various diseases to that of both the genome and the MGC; and (c) displayed a broad overlap among different diseases allowing multiple diseases to be addressed with this set of ORF clones.</p>
<div class="figure" id="pone-0001528-g001"><div class="img"><a name="pone-0001528-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001528.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001528.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Genes associated with Disease Classes (MeSH) in Publications.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The clone target list (HFLEX7000) was compared with all human genes (EntrezGene, 2004) and all genes represented by MGC (2004) with respect to published relationships of the genes to human diseases. The targeted genes reveal similar proportionality to the other gene lists but a general enrichment of genes related to diseases (<a href="#pone-0001528-t001">Table 1</a>; Supplementary <a href="#pone.0001528.s002">Table S2</a>).</p>
<span>doi:10.1371/journal.pone.0001528.g001</span></div><div class="figure" id="pone-0001528-t001"><div class="img"><a name="pone-0001528-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001528.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001528.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>MeSH Term Analysis for Gene and Diseases Association in Publications; Examples</span></strong></p><span>doi:10.1371/journal.pone.0001528.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>In addition to disease relationships, we expanded our evaluation to include other search terms relevant for biological research by employing a new database, BioGene, which is based on a similar concept to MedGene. Instead of disease terms, BioGene has a co-citation index for all human genes with all biological and chemical Medical Subject Heading (MeSH) classes (<a href="http://www.nlm.nih.gov/mesh">http://www.nlm.nih.gov/mesh</a>), such as “lipids”, “pain” and “tetrahydrofolates”, and is available at <a href="http://biogene.med.harvard.edu/BIOGENE/">http://biogene.med.harvard.edu/BIOGENE/</a>. As shown in <a href="#pone-0001528-g002">Figure 2</a>; <a href="#pone-0001528-t002">Table 2</a> for 34 biological MeSH classes (complete listing for all analyzed MeSH terms in Supplementary <a href="#pone.0001528.s003">Table S3</a>), the candidate list is enriched for genes linked to all biological MeSH terms in the literature, but proportional to that of the entire MGC clones and to the entire genome.</p>
<div class="figure" id="pone-0001528-g002"><div class="img"><a name="pone-0001528-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001528.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001528.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Genes associated with Biological MeSH Terms in Publications.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The clone target list (HFLEX7000) was compared with all human genes (EntrezGene, 2004) and all genes represented by MGC (2004) with respect to published relationships of the genes to all biological MeSH terms and MeSH nodes (34). The targeted genes reveal similar proportionality to the other gene lists but a general enrichment of genes related to MeSH terms (<a href="#pone-0001528-t002">Table 2</a>; Supplementary <a href="#pone.0001528.s003">Table S3</a>).</p>
<span>doi:10.1371/journal.pone.0001528.g002</span></div><div class="figure" id="pone-0001528-t002"><div class="img"><a name="pone-0001528-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001528.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001528.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Biological and Chemical MeSH Class Analysis for Genes Associated in Publications with MeSH Class; Examples</span></strong></p><span>doi:10.1371/journal.pone.0001528.t002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Thus, the target set of 3557 genes had a similar overall distribution of genes as the MGC and the human genome, but in general has a higher representation of genes that have been linked to both diseases and biological terms in the literature.</p>


<h4>Clone Production and Sequence Validation</h4>
<h5>Production of Clone Collection.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>We generated the ORF clones via a processing pipeline that relies heavily on the use of robotics and is supported by the FLEXGene LIMS to produce clones in a highly automated, efficient, and accurate manner as published previously <a href="#pone.0001528-Park1">[5]</a>, <a href="#pone.0001528-Witt1">[10]</a>, <a href="#pone.0001528-Hu2">[16]</a>, <a href="#pone.0001528-Murthy1">[17]</a>.</p>
<a id="article1.body1.sec2.sec2.sec1.p2" name="article1.body1.sec2.sec2.sec1.p2"></a><p>The process of converting MGC cDNA clones into ORF clones (<a href="#pone-0001528-g003">Figure 3</a>) was initiated by populating our production tracking database (FLEXGene) with the relevant MGC information, e.g. IMAGE ID, GI number, clone sequence, start/stop of CDS, CDS length, gene information, plate and position in IMAGE/MGC collection. All ORFs were normalized to start with ATG, and natural stop codons either to TAG, or, for the format without C-terminal stop, to TTG (Leu). PCR amplicons were gel purified and captured using the In-Fusion™ enzyme into a modified recombinational cloning vector, pGWNcoXho, which increased the efficiency of capturing DNA fragments larger than 1.5 kb <a href="#pone.0001528-Hu2">[16]</a>. After transformation into <em>E. coli</em>, constructs were clonally selected and isolated. In total, we successfully produced clonal glycerol stocks for 3,528 of 3,557 targeted genes, an overall success rate of ~98% (<a href="#pone-0001528-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001528-g003"><div class="img"><a name="pone-0001528-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001528.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001528.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Workflow diagram of clone production.</span></strong></p><a id="article1.body1.sec2.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec1.fig1.caption1.p1"></a><p>The entire production process from the design of primers to production of glycerol stocks is shown. The process started by identifying MGC clones in the available plates and then creating array files along with matching PCR primer order files that included two primers anchored at the 3′ end, one for each format. The primers were used to amplify the ORFs from the matching MGC clones. PCR products were monitored in agarose gels, and products were purified prior to capture via In-Fusion reaction. Competent bacterial strains were transformed with the reaction followed by the robotic isolation of 4 resulting colonies per format, which were used to prepare 15% glycerol stocks. Prior to sequencing a single isolate plate of 96 targets were created. As indicated, step specific results were stored in our LIMS.</p>
<span>doi:10.1371/journal.pone.0001528.g003</span></div><div class="figure" id="pone-0001528-t003"><div class="img"><a name="pone-0001528-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001528" data-uri="info:doi/10.1371/journal.pone.0001528.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001528.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001528.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001528.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001528.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001528.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Clone Production and Sequencing Summary</span></strong></p><span>doi:10.1371/journal.pone.0001528.t003</span></div>
<h5>DNA sequence analysis and clone acceptance.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>Based on a pilot study of 96 genes in which we sequenced all available isolates (4 closed and 4 fusion), we expected that 90% of the clones would yield a valid clone. Thus, it was most efficient to sequence a single isolate for each attempted clone format and return to evaluate additional isolates for any that failed. Clones were accepted if they had no truncation mutations, no frameshift mutations and no more than one single amino acid difference with the reference sequence. Clones with any nucleotide changes in the att- sequences were rejected, as changes in these regions could make it impossible to transfer the ORF into expression vectors (<a href="#pone-0001528-t003">Table 3</a>).</p>
<a id="article1.body1.sec2.sec2.sec2.p2" name="article1.body1.sec2.sec2.sec2.p2"></a><p>All rejected clones were manually inspected including a BLAST-based comparison to GenBank/EMBL to assess whether the clone matched any other entry for this gene. This step helped to rescue some rejected clones which were found to be ultimately acceptable due to updated MGC sequence entries.</p>
<a id="article1.body1.sec2.sec2.sec2.p3" name="article1.body1.sec2.sec2.sec2.p3"></a><p>Consistent with the pilot study, 6963 (90%) of the sequenced clones were acceptable based on the above criteria, with the vast majority (6669, or 95.8%) matching exactly to the reference sequence (<a href="#pone-0001528-t003">Table 3</a>, for complete listing of clones see Supplementary <a href="#pone.0001528.s004">Table S4</a>). There were 25 clones that had identical discrepancies in both formats (with and without stop codon). As the two formats were independently produced from the same source clone, this suggests that there may be mistakes in as many as 0.3% of MGC reference sequences.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Starting from the MGC resource, we created protein expression ORF clones in two different formats for over 3400 human genes, HFLEX7000, making them the largest contribution of fully sequence verified ORF clones to the ORFeome Collaboration (<a href="http://www.Orfeomecollaboration.org">www.Orfeomecollaboration.org</a> ). The selection criteria for this subset were based on a combination of publication records for the individual gene and their association with biological as well as human disease MeSH terms, as defined by two programs, MedGene and BioGene. We aimed to reflect within this subset a similar distribution as it was present in MGC or the genome, and not to create a functionally or disease specific subset.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To assure the quality of this cDNA clone collection, we fully sequence verified all clones. By employing the appropriate formatted clone, users can add peptide tags to either end of the expressed protein or express protein without any additional amino acids at all. This is important for application reasons, e.g., for some proteins, the C-terminal amino acids may be important functionally (PDZ domain <a href="#pone.0001528-DeLosRios1">[18]</a>) requiring a translation stop at the natural position, whereas for other proteins the natural N-terminus is relevant (e.g., signal peptides for membrane protein trafficking <a href="#pone.0001528-Harrison1">[19]</a>, <a href="#pone.0001528-Hegde1">[20]</a>). Some applications exploit the use of fusion tags at the C-terminus as an experimental readout (e.g., yeast two hybrid <a href="#pone.0001528-Rual1">[7]</a>), or for capturing expressed proteins and confirming full length expression <a href="#pone.0001528-Ramachandran1">[21]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We targeted over 3500 unique genes and obtained a fully sequence validated ORF clone for 97% (&gt;3400) of the genes. The strategy of selecting only one clonal isolate per gene for sequencing successfully yielded 90% acceptable clones. This success rate dropped to 80% when second isolates of the failed clones were sequenced, raising questions about the likelihood of success of sequencing additional isolates for clones that failed after two attempts. Also, capture efficiency, as measured by the number of colonies after transformation, was not a predictor of eventual clone success; clones with either high or low colony count numbers were equally likely to be rejected at subsequent steps.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>One set of troublesome ORFs identified during PCR and confirmed during sequencing revealed duplication of either the 5′ (near the ATG) or 3′ (near the stop codon) sequences used to design the PCR primer elsewhere in the clone. This led to inappropriate PCR priming and ultimately an inability to clone the gene. Any project using a similar strategy to convert MGC clones into ORF clones might find the same problems, and alternatives, e.g. restriction enzyme/ligation based or fragmented PCR cloning, should be considered for any such ORFs.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In summary, the clones from MGC provide an excellent resource for ORF clone production. The 97% success rate to produce fully sequence validated clones, of which 96% match perfectly to the template clone, underlines that this strategy is feasible in a cost effective manner. Together with our other human ORF sets, notably several hundred DNA binding proteins, over 500 kinases, 1000 breast cancer associated genes, this much broader collection of 3500 genes will be of great benefit to the research community.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>As with all our sequence validated clone collections, human, yeast, or microorganism, the HFLEX7000 clones are available at <a href="http://plasmid.hms.harvard.edu">http://plasmid.hms.harvard.edu</a>. Furthermore, this collection as part of the global ORFeome Collaboration will be available from the ORFeome distributors (<a href="http://www.orfeomecollaboration.org/">http://www.orfeomecollaboration.org/</a>).</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>ORF-specific primer design and strategic 96-well plate organization</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>ORF sequences were parsed out in the FLEXGene LIMS from the information provided with each MGC clone, and primer sequences were designed using a nearest neighbor algorithm as described earlier <a href="#pone.0001528-Park1">[5]</a>, <a href="#pone.0001528-Witt1">[10]</a>, <a href="#pone.0001528-Hu2">[16]</a>, <a href="#pone.0001528-Murthy1">[17]</a>. Natural stop codons were either normalized to TAG or replaced with TTG (Leu) in the final primer designs. In addition to ORF-specific, start and stop regions, the 5′ and 3′ primers included fixed sequences that correspond to partial att sequence-specific recombination recognition sites that flank the ORF in the resultant plasmid clones.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Details regarding amplification, purification and capture into a linear version of pDONR221 by In-fusion™ reaction were previously published <a href="#pone.0001528-Hu2">[16]</a>; PCR success as measured as signal in agarose gels, and capture success as determined as colonies after transformation were stored in FLEXGene LIMS as reported elsewhere <a href="#pone.0001528-Witt1">[10]</a>, <a href="#pone.0001528-Hu2">[16]</a>, <a href="#pone.0001528-Murthy1">[17]</a>.</p>


<h4>Clone isolation and production of glycerol stocks</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Transformations into <em>E. coli</em> (DH5alpha, T1 resistant) were handled in 96-well plates, and robotically plated to 48-sector LB/agar dishes with the appropriate antibiotic selection and grown overnight at 37°C. Colonies were robotically visualized and counted, and single isolates from each sector were picked for inoculation into 1mL growth media (LB/antibiotic) using a customized Megapix robot (Genetix), and 96-well culture blocks were grown overnight at 37°C in the presence of appropriate antibiotic. Inoculated cultures were assayed for growth via OD<sub>600</sub> measurement as a measure of transformation efficiency, and aliquots were stored as 15% glycerol stocks in 96-well plate format.</p>


<h4>Sequence reactions</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>High-throughput sequencing was carried out on an Applied Biosystems (ABI) capillary sequencer using dye-terminator and fluorescent cycle sequencing with don3 (<span class="gene">TCTTGTGCAATGTAACATCAG</span>) and don5 (<span class="gene">CGTTAACGCTAGCATGGA</span>) primers. Raw sequence data were automatically analyzed for quality, vector and repeat content using the pregap4 tool of the Staden Software Package <a href="#pone.0001528-Staden1">[22]</a>. Reads passing this initial quality control were automatically assembled (gap4 tool of the Staden Software Package). The primer walking method was used to finish insert sequencing, with primers automatically designed by PRIDE <a href="#pone.0001528-Haas1">[23]</a>. Sequencing was finished when an overall sequence quality of phred40 for the insert sequence and the vector-insert transition was achieved.</p>


<h4>Sequence Analysis Software</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>After the clone sequence was assembled in the Staten package, the assembled sequences were verified using in house developed software <a href="#pone.0001528-Taycher1">[24]</a>. Clones with acceptable linker as well as CDS sequences were collected for distribution. Clones were not accepted if they had discrepancies leading to protein truncations, frame shifts, discrepancies in the linker regions, or more than two amino acid differences with the reference polypeptide. Sequences of all clones started and ended at the BsrGI restriction site (TGTACA) of the vector, allowing QC of in-frame analysis as well as intactness of att recombination sites. Only clones that failed the CDS region evaluation underwent BLAST search against all available GenBank records, and were re-evaluated using matching BLAST hits in pairwise alignments, allowing us to rescue ~5% of the clones.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001528"></div><a name="pone.0001528.s001" id="pone.0001528.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001528.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Target Gene List for HFLEX7000. MGC clones targeted for conversion into ORF clones are listed with Gene Symbol, Entrez GeneID, gene name, GenBank Acc. No., and CDS Length</p>
<p class="siDoi">doi:10.1371/journal.pone.0001528.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.62 MB XLS)</p>
<a name="pone.0001528.s002" id="pone.0001528.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001528.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Disease MeSH Term Analysis. Complete Analysis for Gene and Diseases Association in Publications</p>
<p class="siDoi">doi:10.1371/journal.pone.0001528.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.57 MB XLS)</p>
<a name="pone.0001528.s003" id="pone.0001528.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001528.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Biological and Chemical MeSH Class Analysis. Complete BioGene analysis of biological and chemical MeSH class associations with genes in PubMed, using either all human genes (2004), unique genes in MGC (2004), or HFLEX7000 (targets). Numerical values and percentiles of each class associated with genes are shown (#, %). Relative MeSH Class associations in either MGC or HFLEX7000 to the genome, and in HFLEX7000 to MGC examine a potential bias in MGC or HFLEX7000 towards specific MeSH terms.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001528.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.87 MB XLS)</p>
<a name="pone.0001528.s004" id="pone.0001528.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001528.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">HFLEX7000 Clone List. Complete listing of all accepted, fully sequence validated ORF clones with Gene Symbol, Entrez GeneID, PlasmID CloneID, Clone GenBank Acc. No., Clone IMAGE ID, and Reference GenBank Acc. No.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001528.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.20 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank S. Wiemann and Ute Ernst for sequencing and discussions and S. Henze for excellent technical support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BK JL AR CD. Performed the experiments: AR JR FK SM DJ MB. Analyzed the data: BK YH AR DZ LE AH. Contributed reagents/materials/analysis tools: YH ET DZ LE DH NR BS JP AH. Wrote the paper: YH AR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001528-Baross1" id="pone.0001528-Baross1"></a>Baross A, Butterfield YS, Coughlin SM, Zeng T, Griffith M, et al.  (2004) Systematic recovery and analysis of full-ORF human cDNA clones. Genome Res  14: 2083–2092.  <ul class="find" data-citedArticleID="1049276" data-doi="10.1101/gr.2473704"><li><a href="http://dx.doi.org/10.1101/gr.2473704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+recovery+and+analysis+of+full-ORF+human+cDNA+clones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+recovery+and+analysis+of+full-ORF+human+cDNA+clones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001528-Gerhard1" id="pone.0001528-Gerhard1"></a>Gerhard DS, Wagner L, Feingold EA, Shenmen CM, Grouse LH, et al.  (2004) The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC). Genome Res  14: 2121–2127.  <ul class="find" data-citedArticleID="1049282" data-doi="10.1101/gr.2596504"><li><a href="http://dx.doi.org/10.1101/gr.2596504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+status%2C+quality%2C+and+expansion+of+the+NIH+full-length+cDNA+project%3A+the+Mammalian+Gene+Collection+%28MGC%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+status%2C+quality%2C+and+expansion+of+the+NIH+full-length+cDNA+project%3A+the+Mammalian+Gene+Collection+%28MGC%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001528-Pennisi1" id="pone.0001528-Pennisi1"></a>Pennisi E (2007) Genetics. Working the (gene count) numbers: finally, a firm answer? Science  316: 1113.  <ul class="find" data-citedArticleID="1049300" data-doi="10.1126/science.316.5828.1113a"><li><a href="http://dx.doi.org/10.1126/science.316.5828.1113a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics.+Working+the+%28gene+count%29+numbers%3A+finally%2C+a+firm+answer%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics.+Working+the+%28gene+count%29+numbers%3A+finally%2C+a+firm+answer%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001528-Collins1" id="pone.0001528-Collins1"></a>Collins JE, Wright CL, Edwards CA, Davis MP, Grinham JA, et al.  (2004) A genome annotation-driven approach to cloning the human ORFeome. Genome Biol  5: R84.  <ul class="find" data-citedArticleID="1049278"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+genome+annotation-driven+approach+to+cloning+the+human+ORFeome.&amp;auth=&amp;atitle=A+genome+annotation-driven+approach+to+cloning+the+human+ORFeome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome+annotation-driven+approach+to+cloning+the+human+ORFeome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome+annotation-driven+approach+to+cloning+the+human+ORFeome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001528-Park1" id="pone.0001528-Park1"></a>Park J, Hu Y, Murthy TV, Vannberg F, Shen B, et al.  (2005) Building a human kinase gene repository: bioinformatics, molecular cloning, and functional validation. Proc Natl Acad Sci U S A  102: 8114–8119.  <ul class="find" data-citedArticleID="1049298" data-doi="10.1073/pnas.0503141102"><li><a href="http://dx.doi.org/10.1073/pnas.0503141102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Building+a+human+kinase+gene+repository%3A+bioinformatics%2C+molecular+cloning%2C+and+functional+validation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Building+a+human+kinase+gene+repository%3A+bioinformatics%2C+molecular+cloning%2C+and+functional+validation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001528-Strausberg1" id="pone.0001528-Strausberg1"></a>Strausberg RL, Feingold EA, Grouse LH, Derge JG, Klausner RD, et al.  (2002) Generation and initial analysis of more than 15,000 full-length human and mouse cDNA sequences. Proc Natl Acad Sci U S A  99: 16899–16903.  <ul class="find" data-citedArticleID="1049316" data-doi="10.1073/pnas.242603899"><li><a href="http://dx.doi.org/10.1073/pnas.242603899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+and+initial+analysis+of+more+than+15%2C000+full-length+human+and+mouse+cDNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+and+initial+analysis+of+more+than+15%2C000+full-length+human+and+mouse+cDNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001528-Rual1" id="pone.0001528-Rual1"></a>Rual JF, Hirozane-Kishikawa T, Hao T, Bertin N, Li S, et al.  (2004) Human ORFeome version 1.1: a platform for reverse proteomics. Genome Res  14: 2128–2135.  <ul class="find" data-citedArticleID="1049312" data-doi="10.1101/gr.2973604"><li><a href="http://dx.doi.org/10.1101/gr.2973604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+ORFeome+version+1.1%3A+a+platform+for+reverse+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+ORFeome+version+1.1%3A+a+platform+for+reverse+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001528-Lamesch1" id="pone.0001528-Lamesch1"></a>Lamesch P, Li N, Milstein S, Fan C, Hao T, et al.  (2007) hORFeome v3.1: a resource of human open reading frames representing over 10,000 human genes. Genomics  89: 307–315.  <ul class="find" data-citedArticleID="1049294" data-doi="10.1016/j.ygeno.2006.11.012"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2006.11.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=hORFeome+v3.1%3A+a+resource+of+human+open+reading+frames+representing+over+10%2C000+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22hORFeome+v3.1%3A+a+resource+of+human+open+reading+frames+representing+over+10%2C000+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001528-Temple1" id="pone.0001528-Temple1"></a>Temple G, Lamesch P, Milstein S, Hill DE, Wagner L, et al.  (2006) From genome to proteome: developing expression clone resources for the human genome. Hum Mol Genet  15 Spec No 1: R31–43.  <ul class="find" data-citedArticleID="1049320" data-doi="10.1093/hmg/ddl048"><li><a href="http://dx.doi.org/10.1093/hmg/ddl048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genome+to+proteome%3A+developing+expression+clone+resources+for+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genome+to+proteome%3A+developing+expression+clone+resources+for+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001528-Witt1" id="pone.0001528-Witt1"></a>Witt AE, Hines LM, Collins NL, Hu Y, Gunawardane RN, et al.  (2006) Functional proteomics approach to investigate the biological activities of cDNAs implicated in breast cancer. J Proteome Res  5: 599–610.  <ul class="find" data-citedArticleID="1049322" data-doi="10.1021/pr050395r"><li><a href="http://dx.doi.org/10.1021/pr050395r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+proteomics+approach+to+investigate+the+biological+activities+of+cDNAs+implicated+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+proteomics+approach+to+investigate+the+biological+activities+of+cDNAs+implicated+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001528-Hu1" id="pone.0001528-Hu1"></a>Hu Y, Hines LM, Weng H, Zuo D, Rivera M, et al.  (2003) Analysis of genomic and proteomic data using advanced literature mining. J Proteome Res  2: 405–412.  <ul class="find" data-citedArticleID="1049290" data-doi="10.1021/pr0340227"><li><a href="http://dx.doi.org/10.1021/pr0340227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+genomic+and+proteomic+data+using+advanced+literature+mining." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+genomic+and+proteomic+data+using+advanced+literature+mining.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001528-Pruitt1" id="pone.0001528-Pruitt1"></a>Pruitt KD, Tatusova T, Maglott DR (2003) NCBI Reference Sequence project: update and current status. Nucleic Acids Res  31: 34–37.  <ul class="find" data-citedArticleID="1049302" data-doi="10.1093/nar/gkg111"><li><a href="http://dx.doi.org/10.1093/nar/gkg111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+Reference+Sequence+project%3A+update+and+current+status." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+Reference+Sequence+project%3A+update+and+current+status.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001528-Pruitt2" id="pone.0001528-Pruitt2"></a>Pruitt KD, Tatusova T, Maglott DR (2005) NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res  33: D501–504.  <ul class="find" data-citedArticleID="1049304" data-doi="10.1093/nar/gkl842"><li><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001528-Pruitt3" id="pone.0001528-Pruitt3"></a>Pruitt KD, Tatusova T, Maglott DR (2007) NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res  35: D61–65.  <ul class="find" data-citedArticleID="1049306" data-doi="10.1093/nar/gkl842"><li><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+reference+sequences+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+reference+sequences+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001528-Pruitt4" id="pone.0001528-Pruitt4"></a>Pruitt KD, Maglott DR (2001) RefSeq and LocusLink: NCBI gene-centered resources. Nucleic Acids Res  29: 137–140.  <ul class="find" data-citedArticleID="1049308" data-doi="10.1093/nar/29.1.137"><li><a href="http://dx.doi.org/10.1093/nar/29.1.137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RefSeq+and+LocusLink%3A+NCBI+gene-centered+resources." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RefSeq+and+LocusLink%3A+NCBI+gene-centered+resources.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001528-Hu2" id="pone.0001528-Hu2"></a>Hu Y, Rolfs A, Bhullar B, Murthy TV, Zhu C, et al.  (2007) Approaching a complete repository of sequence-verified protein-encoding clones for Saccharomyces cerevisiae. Genome Res  17: 536–543.  <ul class="find" data-citedArticleID="1049292" data-doi="10.1101/gr.6037607"><li><a href="http://dx.doi.org/10.1101/gr.6037607" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approaching+a+complete+repository+of+sequence-verified+protein-encoding+clones+for+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approaching+a+complete+repository+of+sequence-verified+protein-encoding+clones+for+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001528-Murthy1" id="pone.0001528-Murthy1"></a>Murthy T, Rolfs A, Hu Y, Shi Z, Raphael J, et al.  (2007) A full-genomic sequence-verified protein-coding gene collection for Francisella tularensis. PLoS ONE  2: e577.  <ul class="find" data-citedArticleID="1049296" data-doi="10.1371/journal.pone.0000577"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+full-genomic+sequence-verified+protein-coding+gene+collection+for+Francisella+tularensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+full-genomic+sequence-verified+protein-coding+gene+collection+for+Francisella+tularensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001528-DeLosRios1" id="pone.0001528-DeLosRios1"></a>De Los Rios P, Cecconi F, Pretre A, Dietler G, Michielin O, et al.  (2005) Functional dynamics of PDZ binding domains: a normal-mode analysis. Biophys J  89: 14–21.  <ul class="find" data-citedArticleID="1049280" data-doi="10.1529/biophysj.104.055004"><li><a href="http://dx.doi.org/10.1529/biophysj.104.055004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+dynamics+of+PDZ+binding+domains%3A+a+normal-mode+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+dynamics+of+PDZ+binding+domains%3A+a+normal-mode+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001528-Harrison1" id="pone.0001528-Harrison1"></a>Harrison OJ, Corps EM, Kilshaw PJ (2005) Cadherin adhesion depends on a salt bridge at the N-terminus. J Cell Sci  118: 4123–4130.  <ul class="find" data-citedArticleID="1049286" data-doi="10.1242/jcs.02539"><li><a href="http://dx.doi.org/10.1242/jcs.02539" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cadherin+adhesion+depends+on+a+salt+bridge+at+the+N-terminus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cadherin+adhesion+depends+on+a+salt+bridge+at+the+N-terminus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001528-Hegde1" id="pone.0001528-Hegde1"></a>Hegde RS, Bernstein HD (2006) The surprising complexity of signal sequences. Trends Biochem Sci  31: 563–571.  <ul class="find" data-citedArticleID="1049288" data-doi="10.1016/j.tibs.2006.08.004"><li><a href="http://dx.doi.org/10.1016/j.tibs.2006.08.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+surprising+complexity+of+signal+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+surprising+complexity+of+signal+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001528-Ramachandran1" id="pone.0001528-Ramachandran1"></a>Ramachandran N, Hainsworth E, Bhullar B, Eisenstein S, Rosen B, et al.  (2004) Self-assembling protein microarrays. Science  305: 86–90.  <ul class="find" data-citedArticleID="1049310" data-doi="10.1126/science.1097639"><li><a href="http://dx.doi.org/10.1126/science.1097639" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-assembling+protein+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-assembling+protein+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001528-Staden1" id="pone.0001528-Staden1"></a>Staden R, Judge DP, Bonfield JK (2001) Sequence assembly and finishing methods. Methods Biochem Anal  43: 303–322.  <ul class="find" data-citedArticleID="1049314" data-doi="10.1002/0471223921.ch13"><li><a href="http://dx.doi.org/10.1002/0471223921.ch13" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+assembly+and+finishing+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+assembly+and+finishing+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001528-Haas1" id="pone.0001528-Haas1"></a>Haas S, Vingron M, Poustka A, Wiemann S (1998) Primer design for large scale sequencing. Nucleic Acids Res  26: 3006–3012.  <ul class="find" data-citedArticleID="1049284" data-doi="10.1093/nar/26.12.3006"><li><a href="http://dx.doi.org/10.1093/nar/26.12.3006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primer+design+for+large+scale+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primer+design+for+large+scale+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001528-Taycher1" id="pone.0001528-Taycher1"></a>Taycher E, Rolfs A, Hu Y, Zuo D, Mohr SE, et al.  (2007) A novel approach to sequence validating protein expression clones with automated decision making. BMC Bioinformatics  8: 198.  <ul class="find" data-citedArticleID="1049318" data-doi="10.1186/1471-2105-8-198"><li><a href="http://dx.doi.org/10.1186/1471-2105-8-198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+approach+to+sequence+validating+protein+expression+clones+with+automated+decision+making." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+approach+to+sequence+validating+protein+expression+clones+with+automated+decision+making.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.XML" value="95175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.PDF" value="308671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g001.PNG_L" value="238333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g001.PNG_M" value="44334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g001.PNG_S" value="6728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g001.TIF" value="426280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g001.PNG_I" value="15210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t001.PNG_L" value="323699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t001.PNG_M" value="186610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t001.PNG_S" value="17166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t001.TIF" value="1321918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t001.PNG_I" value="57543"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g002.PNG_L" value="152096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g002.PNG_M" value="31270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g002.PNG_S" value="5563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g002.TIF" value="310836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g002.PNG_I" value="11001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t002.PNG_L" value="340162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t002.PNG_M" value="194532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t002.PNG_S" value="19349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t002.TIF" value="1471408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t002.PNG_I" value="59692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g003.PNG_L" value="121156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g003.PNG_M" value="32102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g003.PNG_S" value="7633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g003.TIF" value="296492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.g003.PNG_I" value="17704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t003.PNG_L" value="185342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t003.PNG_M" value="101010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t003.PNG_S" value="12265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t003.TIF" value="858430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.t003.PNG_I" value="30405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.s001.XLS" value="623104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.s002.XLS" value="569856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.s003.XLS" value="1874432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001528.s004.XLS" value="1202688"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001528&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001528" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001528&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001528&volume=&issue=&title=A Biomedically Enriched Collection of 7000 Human ORF Clones&author_name=Andreas%20Rolfs%2C%20Yanhui%20Hu%2C%20Lars%20Ebert%2C%20Dietmar%20Hoffmann%2C%20Dongmei%20Zuo%2C%20Niro%20Ramachandran%2C%20Jacob%20Raphael%2C%20Fontina%20Kelley%2C%20Seamus%20McCarron%2C%20Daniel%20A.%20Jepson%2C%20Binghua%20Shen%2C%20Munira%20M.%20A.%20Baqui%2C%20Joseph%20Pearlberg%2C%20Elena%20Taycher%2C%20Craig%20DeLoughery%2C%20Andreas%20Hoerlein%2C%20Bernhard%20Korn%2C%20Joshua%20LaBaer&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001528" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001528" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001528" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001528&amp;t=A%20Biomedically%20Enriched%20Collection%20of%207000%20Human%20ORF%20Clones" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001528&title=A%20Biomedically%20Enriched%20Collection%20of%207000%20Human%20ORF%20Clones&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001528&amp;title=A%20Biomedically%20Enriched%20Collection%20of%207000%20Human%20ORF%20Clones" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001528" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001528&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Biomedically Enriched Collection of 7000 Human ORF Clones';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001528';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Biomedically%20Enriched%20Collection%20of%207000%20Human%20ORF%20Clones http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001528" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001528" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Expressed+sequence+tags%22" title="Search for articles in the subject area:'Expressed sequence tags'"><div class="flagText">Expressed sequence tags</div></a>
              <div data-categoryid="34427" data-articleid="25882"
                   data-categoryname="Expressed sequence tags"
                   class="flagImage" title="Flag 'Expressed sequence tags' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="25882"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="47601" data-articleid="25882"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25882"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25882"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+expression%22" title="Search for articles in the subject area:'Protein expression'"><div class="flagText">Protein expression</div></a>
              <div data-categoryid="17513" data-articleid="25882"
                   data-categoryname="Protein expression"
                   class="flagImage" title="Flag 'Protein expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="25882"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="25882"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7777'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3042'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6312&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
