

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000950"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000950" />

    <meta name="citation_title" content="Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies"/>
    <meta itemprop="name" content="Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies"/>

      <meta name="citation_author" content="Marco Salemi"/>
            <meta name="citation_author_institution" content="Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Brant R. Burkhardt"/>
            <meta name="citation_author_institution" content="Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Rebecca R. Gray"/>
            <meta name="citation_author_institution" content="Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Guity Ghaffari"/>
            <meta name="citation_author_institution" content="Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America"/>
            <meta name="citation_author_institution" content="Department of Pediatrics, Division of Immunology, Rheumatology, and Infectious Diseases, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="John W. Sleasman"/>
            <meta name="citation_author_institution" content="Department of Pediatrics, Division of Allergy, Immunology, and Rheumatology, University of South Florida and All Children's Hospital, St. Petersburg, Florida, United States of America"/>
      <meta name="citation_author" content="Maureen M. Goodenow"/>
            <meta name="citation_author_institution" content="Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America"/>
            <meta name="citation_author_institution" content="Department of Pediatrics, Division of Immunology, Rheumatology, and Infectious Diseases, University of Florida, Gainesville, Florida, United States of America"/>

    <meta name="citation_date" content="2007/9/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000950.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e950"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Chemokine receptors as HIV-1 coreceptors: roles in viral entry, tropism, and disease.; citation_author=EA Berger; citation_author=PM Murphy; citation_author=JM Farber; citation_journal_title=Ann Rev Immunol; citation_volume=17; citation_number=1; citation_pages=675-700; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Macrophage-tropic variants initiate human immunodeficiency virus type 1 infection after sexual, parenteral, and vertical transmission.; citation_author=AB van'T Wout; citation_author=NA Koostra; citation_author=GA Mulder-Kampinga; citation_author=N Albrecht-van Lent; citation_author=HJ Scherpbier; citation_journal_title=J Clin Invest; citation_volume=94; citation_number=2; citation_pages=2060-2067; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Consistent viral evolutionary changes associated with the progression of human immunodeficiency virus type 1 infection.; citation_author=R Shankarappa; citation_author=JB Margolick; citation_author=SJ Gange; citation_author=AG Rodrigo; citation_author=D Upchurch; citation_journal_title=J Virol; citation_volume=73; citation_number=3; citation_pages=10489-10502; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Prognostic value of HIV-1 syncytium-inducing phenotype for rate of CD4+ cell depletion and progression to AIDS.; citation_author=M Koot; citation_author=AB van'T Wout; citation_author=NA Koostra; citation_author=REY Degoede; citation_author=M Tersmette; citation_journal_title=Ann Intern Med; citation_volume=118; citation_number=4; citation_pages=681-688; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Change in coreceptor use correlates with disease progression in HIV-1-infected individuals.; citation_author=RI Connor; citation_author=KE Sheridan; citation_author=D Ceradini; citation_author=S Choe; citation_author=NR Landau; citation_journal_title=J Exp Med; citation_volume=185; citation_number=5; citation_pages=621-628; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The HIV coreceptor switch: a population dynamical perspective.; citation_author=RR Regoes; citation_author=S Bonhoeffer; citation_journal_title=Trends Microbiol; citation_volume=6; citation_number=6; citation_pages=269-277; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Emergence of CXCR4-using human immunodeficiency virus type 1 (HIV-1) variants in a minority of HIV-1-infected patients following treatment with CCR5 antagonist Maraviroc is from a pretreatment CXCR4-using virus reservoirs.; citation_author=M Westby; citation_author=M Lewis; citation_author=J Whitcomb; citation_author=M Youle; citation_author=AL Pozniak; citation_journal_title=J Virol; citation_volume=80; citation_number=7; citation_pages=4909-4920; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The X4 phenotype of HIV-1 evolves from R5 in two children of mothers, carrying X4, and is not linked to transmission.; citation_author=P Clevestig; citation_author=I Maljkovic; citation_author=C Casper; citation_author=E Carlenor; citation_author=S Lindgen; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=5; citation_number=8; citation_pages=371-378; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complex determinants in human immunodeficiency virus type 1 envelope gp120 mediate CXCR4-dependent infection in macrophages.; citation_author=G Ghaffari; citation_author=DL Tuttle; citation_author=D Briggs; citation_author=BR Burkhardt; citation_author=D Bhatt; citation_journal_title=J Virol; citation_volume=79; citation_number=9; citation_pages=13250-13261; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selective transmission of CCR5-utilizing HIV-1: the ‘gatekeeper’ problem resolved?; citation_author=L Margolis; citation_author=R Shattock; citation_journal_title=Nat Rev Microbiol; citation_volume=4; citation_number=10; citation_pages=312-317; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The region of the envelope gene of human immunodeficiency virus type 1 responsible for determination of cell tropism.; citation_author=AJ Cann; citation_author=MJ Curcher; citation_author=M Boyd; citation_author=W O'Brien; citation_author=JQ Zhao; citation_journal_title=J Virol; citation_volume=66; citation_number=11; citation_pages=305-309; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Evidence that the structural conformation of envelope gp120 affects human immunodeficiency virus type 1 infectivity, host range, and syncytium-forming ability.; citation_author=L Stamatos; citation_author=C Cheng-Mayer; citation_journal_title=J Virol; citation_volume=67; citation_number=12; citation_pages=5635-5639; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Patterns of amino acid variability in NSI-like and SI-like V3 sequences and a linked change in the CD4-binding domain of the HIV-1 Env protein.; citation_author=LB Milich; citation_author=B Margolin; citation_author=R Swanstrom; citation_journal_title=Virology; citation_volume=239; citation_number=13; citation_pages=108-118; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus type 1 coreceptor switching: V1/V2 gain-of-fitness mutations compensate for V3 loss-of-fitness mutations.; citation_author=C Pastore; citation_author=R Nedellec; citation_author=A Ramos; citation_author=S Pontow; citation_author=L Ratner; citation_journal_title=J Virol; citation_volume=80; citation_number=14; citation_pages=750-758; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of determinants on a dualtropic human immunodeficiency virus type 1 envelope glycoprotein that confer usage of CXCR4.; citation_author=MW Cho; citation_author=MK Lee; citation_author=MC Carney; citation_author=JF Berson; citation_author=RW Doms; citation_author=MA Martin; citation_journal_title=J Virol.; citation_volume=72; citation_number=15; citation_pages=2509-2515; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evidence for common structural determinants of human immunodeficiency virus type 1 coreceptor activity provided through functional analysis of CCR5/CXCR4 chimeric coreceptors.; citation_author=S Pontow; citation_author=L Ratner; citation_journal_title=J Virol.; citation_volume=75; citation_number=16; citation_pages=11503-11514; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The challenge of viral reservoirs in HIV-1 infection.; citation_author=JN Blankson; citation_author=D Persaud; citation_author=RF Siliciano; citation_journal_title=Annual Reviews in Medicine; citation_volume=53; citation_number=17; citation_pages=557-593; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=HIV-infected individuals receiving effective antiviral therapy for extended periods of time continually replenish their viral reservoir.; citation_author=TW Chun; citation_author=DC Nickle; citation_author=JS Justement; citation_author=D Large; citation_author=A Semerjian; citation_journal_title=J Clin Invest; citation_volume=115; citation_number=18; citation_pages=3250-3255; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of envelope sequences from the genital tract and peripheral blood of women infected with clade A human immunodeficiency virus type 1.; citation_author=M Poss; citation_author=AG Rodrigo; citation_author=JJ Gosink; citation_author=GH Learn; citation_author=D de Vange Panteleeff; citation_journal_title=J Virol; citation_volume=72; citation_number=19; citation_pages=8240-51; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Persistent HIV type 1 infection in semen and blood compartments in patients after long-term potent antiretroviral therapy.; citation_author=JK Craigo; citation_author=BK Patterson; citation_author=S Paranjpe; citation_author=K Kulka; citation_author=M Ding; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=20; citation_number=20; citation_pages=1196-11209; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic differences between blood-and brain-derived viral sequences from human immunodeficiency virus type 1-infected patients: Evidence of conserved elements in the V3 region of the envelope protein of brain-derived sequences.; citation_author=BT Korber; citation_author=KJ Kunstman; citation_author=BK Patterson; citation_author=M Furtado; citation_author=MM McEvilly; citation_journal_title=J Virol.; citation_volume=68; citation_number=21; citation_pages=7467-7481; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Unique HIV type 1 V3 region sequences derived from six different regions of brain: region-specific evolution within host-determined quasispecies.; citation_author=J Chang; citation_author=R Jozwiak; citation_author=B Wang; citation_author=T Ng; citation_author=YC Ge; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=14; citation_number=22; citation_pages=25-30; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Independent evolution of HIV type 1 in different brain regions.; citation_author=P Shapshak; citation_author=DM Segal; citation_author=KA Crandall; citation_author=RK Fujimura; citation_author=BT Zhang; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=15; citation_number=23; citation_pages=811-820; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evolutionary Indicators of human immunodeficiency virus type 1 reservoirs and compartments.; citation_author=DC Nickle; citation_author=MA Jensen; citation_author=D Shriner; citation_author=SJ Brodie; citation_author=LM Frenkel; citation_journal_title=J Virol; citation_volume=77; citation_number=24; citation_pages=5540-5546; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Independent evolution of human immunodeficiency virus (HIV) drug resistance mutations in diverse areas of the brain in HIV-infected patients, with and without dementia, on antiretroviral treatment.; citation_author=TK Smit; citation_author=BJ Brew; citation_author=W Tourtelotte; citation_author=S Morgello; citation_author=BB Gelman; citation_number=25; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HIV-1 phylodynamic analysis in distinct brain compartments provides a model for the neuropathogenesis of AIDS.; citation_author=M Salemi; citation_author=SL Lamers; citation_author=S Yu; citation_author=T de Oliveira; citation_author=WM Fitch; citation_journal_title=J Virol; citation_volume=79; citation_number=26; citation_pages=11343-11352; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The HIV coreceptors CXCR4 and CCR5 are differentially expressed and regulated on human T lymphocytes.; citation_author=CC Bleul; citation_author=L Wu; citation_author=JA Hoxie; citation_author=TA Springer; citation_author=CR Mackay; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=94; citation_number=27; citation_pages=1925-1930; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=In vivo distribution of the human immunodeficiency virus/simian immunodeficiency virus coreceptors: CXCR4, CCR3, CCR5.; citation_author=L Zhang; citation_author=T He; citation_author=A Talal; citation_author=G Wang; citation_author=SH Frankel; citation_journal_title=J Virol; citation_volume=72; citation_number=28; citation_pages=5035-5045; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Selective thymocyte depletion in neonatal HIV-1 thymic infection.; citation_author=M Rosenzweig; citation_author=DP Clark; citation_author=GN Gaulton; citation_journal_title=AIDS; citation_volume=7; citation_number=29; citation_pages=1601-1605; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=CXCR4 expression during lymphopoiesis: implications for human immunodeficiency virus type 1 infection of the thymus.; citation_author=SG Kitchen; citation_author=JA Zack; citation_journal_title=J Virol.; citation_volume=171; citation_number=30; citation_pages=6928-6934; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=CXCR4 and CCR5 on human thymocytes: biological function and role in HIV-1 infection.; citation_author=MB Zaitseva; citation_author=S Lee; citation_author=RL Rabin; citation_author=HL Tiffany; citation_author=JM Farber; citation_journal_title=J Immunol; citation_volume=161; citation_number=31; citation_pages=3103-13; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Effect of cytokines on HIV-induced depletion of thymocytes in vivo.; citation_author=CH Uittenbogaart; citation_author=WJ Boscardin; citation_author=DJ Anisman-Posner; citation_author=PS Koka; citation_author=G Bristol; citation_journal_title=AIDS; citation_volume=14; citation_number=32; citation_pages=1317-1325; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Generation of HIV latency during thymopoiesis.; citation_author=DG Brooks; citation_author=SG Kitchen; citation_author=CM Kitchen; citation_author=DD Scripture-Adams; citation_author=JA Zack; citation_journal_title=Nat Med; citation_volume=7; citation_number=33; citation_pages=459-464; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evidence for increase T-cell turnover and decreased thymic output in HIV infection.; citation_author=DC Douek; citation_author=MR Betts; citation_author=BJ Hill; citation_author=SJ Little; citation_author=R Lempicki; citation_journal_title=J Immunol; citation_volume=167; citation_number=34; citation_pages=6663-6668; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HIV infection rapidly induces and maintains a substantial suppression of thymocytes proliferation.; citation_author=ML Dion; citation_author=JF Poulin; citation_author=R Bordi; citation_author=M Sylvestre; citation_author=R Corsini; citation_journal_title=Immunity; citation_volume=21; citation_number=35; citation_pages=757-768; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Early progression of disease in HIV-infected infants with thymus dysfunction.; citation_author=AP Kourtis; citation_author=C Ibegbu; citation_author=AJ Nahmias; citation_author=FK Lee; citation_author=WS Clark; citation_journal_title=N Engl J Med; citation_volume=335; citation_number=36; citation_pages=1431-1436; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=HIV type 1 envelope quasispecies in the thymus and lymph nodes of AIDS patients.; citation_author=K Alves; citation_author=M Canzian; citation_author=EL Delwart; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=2; citation_number=37; citation_pages=161-165; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Unifying the epidemiological and evolutionary dynamics of pathogens.; citation_author=BT Grenfell; citation_author=OG Pybus; citation_author=JR Gog; citation_author=JL Wood; citation_author=JM Daly; citation_journal_title=Science; citation_volume=303; citation_number=38; citation_pages=327-332; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Envelope V3 amino acid sequence predicts HIV-1 phenotype (coreceptor usage and tropism for macrophages).; citation_author=DR Briggs; citation_author=DL Tuttle; citation_author=JW Sleasman; citation_author=MM Goodenow; citation_journal_title=AIDS; citation_volume=14; citation_number=39; citation_pages=2937-2939; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Improved coreceptor usage prediction and genotypic monitoring of R5-to-X4 transition by motif analysis of human immunodeficiency virus type 1 Env V3 loop sequences.; citation_author=MA Jensen; citation_author=FS Li; citation_author=AB van't Wout; citation_author=DC Nickle; citation_author=D Shriner; citation_journal_title=J. Virol.; citation_volume=77; citation_number=40; citation_pages=13376-13388; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A Reliable Phenotype Predictor for Human Immunodeficiency Virus Type 1 Subtype C Based on Envelope V3 Sequences.; citation_author=MA Jensen; citation_author=M Coetzer; citation_author=AB van't Wout; citation_author=L, JI Morris; citation_journal_title=J. Virol.; citation_volume=80; citation_number=41; citation_pages=4698-4704; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mosaic structure of the human immunodeficiency virus type 1 genome infecting lymphoid cells and the brain: evidence for frequent in vivo recombination events in the evolution of regional populations.; citation_author=A Morris; citation_author=M Marsden; citation_author=K Halcrow; citation_author=ES Hughes; citation_author=RP Brettle; citation_journal_title=J. Virol.; citation_volume=73; citation_number=42; citation_pages=8720-8731; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Pervasive genomic recombination of HIV-1 in vivo.; citation_author=D Shriner; citation_author=AG Rodrigo; citation_author=DC Nickle; citation_author=JI Mullins; citation_journal_title=Genetics; citation_volume=167; citation_number=43; citation_pages=1573-1583; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Frequent intrapatient recombination between human immunodeficiency virus type 1 R5 and X4 envelopes: implications for coreceptor switch.; citation_author=M Mild; citation_author=J Esbjornnson; citation_author=EM Fenyo; citation_author=P Medstrand; citation_journal_title=J. Virol.; citation_volume=81; citation_number=44; citation_pages=3369-3376; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Recombination during PCR.; citation_author=A Meyerhans; citation_author=JP Vartanian; citation_author=S Wain-Hobson; citation_journal_title=Nucleic Acids Res.; citation_volume=18; citation_number=45; citation_pages=1687-1691; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=In vivo sequence of HIV-1 enve gene: evidence for recombination among variants found in a single individual. AIDS Res. Hum.; citation_author=RM Howell; citation_author=JE Fitzgibbon; citation_author=M Noe; citation_author=ZJ Ren; citation_author=DJ Gocke; citation_journal_title=Retroviruses; citation_volume=7; citation_number=46; citation_pages=869-876; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Simple determination of human immunodeficiency virus type 1 syncytium-inducing V3 genotype by PCR.; citation_author=RAM Fouchier; citation_author=M Brouwer; citation_author=SM Broersen; citation_author=H Schuitemaker; citation_journal_title=J Clin Microbiol; citation_volume=33; citation_number=47; citation_pages=906-911; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Episodic adaptive evolution of primate lysozymes.; citation_author=W Messier; citation_author=CB Stewart; citation_journal_title=Nature; citation_volume=385; citation_number=48; citation_pages=151-154; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Influence of random genetic drift on human immunodeficiency virus type 1 env evolution during chronic infection.; citation_author=D Shriner; citation_author=R Shankarappa; citation_author=MA Jensen; citation_author=DC Nickle; citation_author=JE Mittler; citation_journal_title=Genetics; citation_volume=166; citation_number=49; citation_pages=1155-1164; citation_date=2004b; " />
      <meta name="citation_reference" content="citation_title=Evolution of the human immunodeficiency virus envelope gene is dominated by purifying selection.; citation_author=CT Edwards; citation_author=EC Holmes; citation_author=OG Pybus; citation_author=DJ Wilson; citation_author=RP Viscidi; citation_journal_title=Genetics.; citation_volume=174; citation_number=50; citation_pages=1441-1453; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution of CXCR4-using HIV-1 SF162 is associated with two unique envelope mutations.; citation_author=Y Kiselyeva; citation_author=R Nedellec; citation_author=A Ramos; citation_author=C Pastore; citation_author=LB Margolis; citation_journal_title=J Virol; citation_volume=0; citation_number=51; citation_pages=JVI.02310-06v1; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Env gp120 sequence analysis of HIV type 1 strains from diverse areas of the brain shows preponderance of CCR5 usage.; citation_author=M Sha; citation_author=TK Smit; citation_author=S Morgello; citation_author=W Tourtelotte; citation_author=B Gelman; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=22; citation_number=52; citation_pages=177-181; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The V3 domain of the HIV-1 gp120 envelope glycoprotein is critical for chemokine-mediated blockade of infection.; citation_author=F Cocchi; citation_author=AL DeVico; citation_author=A Garzino-Demo; citation_author=SK Ayra; citation_author=RC Gallo; citation_journal_title=Science; citation_volume=270; citation_number=53; citation_pages=1811-1815; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Coreceptor switch in R5-tropic simian/human immunodeficiency virus-infected macaques.; citation_author=SH Ho; citation_author=S Tasca; citation_author=L Shek; citation_author=A Li; citation_author=A Gettie; citation_journal_title=J Virol; citation_volume=81; citation_number=54; citation_pages=8621-8633; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Thymocyte-thymic epithelial cell interaction leads to high-level replication of human immunodeficiency virus exclusively in mature CD4(+) CD8(−) CD3(+) thymocytes: a critical role for tumor necrosis factor and interleukin-7.; citation_author=L Chene; citation_author=MT Nugeyre; citation_author=E Guillemard; citation_author=N Moulian; citation_author=F Barre-Sinoussi; citation_journal_title=J Virol; citation_volume=73; citation_number=55; citation_pages=7533-7542; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Physiological and molecular mechanisms of lymphocyte homing.; citation_author=LJ Picker; citation_author=EC Butcher; citation_journal_title=Annu Rev Immunol; citation_volume=10; citation_number=56; citation_pages=561-591; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Naïve and memory cell turnover as drivers of CCR5-to-CXCR4 tropism switch in human immunodeficiency virus type 1: implications for therapy.; citation_author=RM Ribeiro; citation_author=MD Hazenberg; citation_author=AS Perelson; citation_author=MP Davenport; citation_journal_title=J Virol; citation_volume=80; citation_number=57; citation_pages=802-809; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Age involution in the normal human adult thymus.; citation_author=JG Simpson; citation_author=ES Gray; citation_author=JS Beck; citation_journal_title=Clin Exp Immunol; citation_volume=19; citation_number=58; citation_pages=261-265; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=The reappearance of 10 differentiation antigens on peripheral blood lymphocytes after allogenic bone marrow transplantation.; citation_author=L Leino; citation_author=E-M Lilius; citation_author=J Nikoskelainen; citation_author=T-T Pelliniemi; citation_author=A Rajamaki; citation_journal_title=Bone Mar Transpl; citation_volume=8; citation_number=59; citation_pages=339-344; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Age, thymopoiesis, and CD4+ T-lymphocyte regeneration afdter intensive chemotherapy.; citation_author=C Mackall; citation_author=T Fleisher; citation_author=M Brown; citation_author=M Andrich; citation_author=C Chen; citation_journal_title=N Engl J Med; citation_volume=332; citation_number=60; citation_pages=143-149; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evidence for thymic function in the elderly.; citation_author=DC Douek; citation_author=RA Koup; citation_journal_title=Vaccine; citation_volume=18; citation_number=61; citation_pages=1638-1641; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Changes in thymic function with age and during the treatment of HIV infection.; citation_author=DC Douek; citation_author=RD McFarland; citation_author=PH Keiser; citation_author=EA Gage; citation_author=JM Massey; citation_journal_title=Nature; citation_volume=396; citation_number=62; citation_pages=690-695; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Independent variation and positive selection in env V1 and V2 domains within maternal-infant strains of human immunodeficiency virus type 1 in vivo.; citation_author=SL Lamers; citation_author=JW Sleasman; citation_author=JX She; citation_author=KA Barrie; citation_author=SM Pomeroy; citation_journal_title=J Virol; citation_volume=67; citation_number=63; citation_pages=3951-3960; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Expression of CCR5 increases during monocyte differentiation and directly mediates macrophage susceptibility to infection by human immunodeficiency virus type 1.; citation_author=DL Tuttle; citation_author=JK Harrison; citation_author=C Anders; citation_author=JW Sleasman; citation_author=MM Goodenow; citation_journal_title=J Virol; citation_volume=72; citation_number=64; citation_pages=4962-4969; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Postmortem isolation and analysis of human genomic DNA and HIV-infected tissues: a molecular diagnostics application.; citation_author=B Burkhardt; citation_author=D Kephart; citation_author=C Coberley; citation_author=J Sleasman; citation_author=M Goodenow; citation_journal_title=Promega Notes; citation_volume=71; citation_number=65; citation_pages=30-31; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Increased replication of non-syncytium-inducing HIV type 1 isolates in monocyte-derived macrophages is linked to advanced disease in infected children.; citation_author=DL Tuttle; citation_author=CB Anders; citation_author=MJ Aquino-De Jesus; citation_author=PP Poole; citation_author=SL Lamers; citation_journal_title=AIDS Res Hum Retroviruses; citation_volume=20; citation_number=66; citation_pages=353-362; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=HIVbase: a PC/Windows-based software offering storage and querying power for locally held HIV-1 genetic, experimental and clinical data.; citation_author=S Lamers; citation_author=S Beason; citation_author=L Dunlap; citation_author=R Compton; citation_author=M Salemi; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=67; citation_pages=436-438; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A model for alignment of env V1 and V2 hypervariable domains from human and simian immunodeficiency viruses.; citation_author=SL Lamers; citation_author=JW Sleasman; citation_author=MM Goodenow; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=12; citation_number=68; citation_pages=1169-1178; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=HIV-1 coreceptor preference is distinct from target cell tropism: related but independent viral phenotypes.; citation_author=MM Goodenow; citation_author=RG Collman; citation_journal_title=J Leuc Biol; citation_volume=80; citation_number=69; citation_pages=965-972; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A simple and robust statistical test for detecting the presence of recombination.; citation_author=TC Bruen; citation_author=P Herve; citation_author=D Bryant; citation_journal_title=Genetics; citation_volume=172; citation_number=70; citation_pages=2665-2681; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Application of phylogenetic networks in evolutionary studies.; citation_author=DH Huson; citation_author=D Bryant; citation_journal_title=Mol. Biol. Evol.; citation_volume=23; citation_number=71; citation_pages=254-267; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Full-length human immunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination.; citation_author=KS Lole; citation_author=RC Bollinger; citation_author=RS Paranjape; citation_author=D Gadkari; citation_author=SS Kulkarni; citation_journal_title=J Virol.; citation_volume=73; citation_number=72; citation_pages=152-160; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Detecting recombination in viral sequences.; citation_author=M Salminen; citation_number=73; citation_pages=348-377; citation_date=2003; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Phylogeny Inference based on parsimony and other methods with PAUP*.; citation_author=D Swofford; citation_author=J Sullivan; citation_number=74; citation_pages=160-206; citation_date=2003; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Relaxed Phylogenetics and Dating with Confidence.; citation_author=AJ Drummond; citation_author=SYW Ho; citation_author=MJ Phillips; citation_author=A Rambaut; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=75; citation_pages=1-12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Cabios; citation_volume=13; citation_number=76; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Codon substitution models for heterogeneous selection pressure at amino acid sites.; citation_author=Z Yang; citation_author=R Nielsen; citation_author=N Goldman; citation_author=AM Pedersen; citation_journal_title=Genetics; citation_volume=155; citation_number=77; citation_pages=431-449; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A cladistic measure of gene flow inferred from the phylogenies of alleles.; citation_author=M Slatikin; citation_author=WP Maddison; citation_journal_title=Genetics; citation_volume=123; citation_number=78; citation_pages=401-416; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Toward defining the course of evolution: minimum change for a specific tree topology.; citation_author=WM Fitch; citation_journal_title=Syst Zool; citation_volume=20; citation_number=79; citation_pages=401-416; citation_date=1971; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000950" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies"/>
    <meta name="twitter:description" content="BackgroundDuring HIV-1 infection coreceptor switch from CCR5- (R5)- to CXCR4 (X4)-using viruses is associated with disease progression. X4 strains of HIV-1 are highly cytopathic to immature thymocytes. Virtually no studies have evaluated the HIV-1 quasispecies present in vivo within thymic and lymphoid tissues or the evolutionary relationship between R5 and X4 viruses in tissues and peripheral blood.Methodology/Principal FindingsHigh-resolution phylodynamic analysis was applied to virus envelope quasispecies in longitudinal peripheral blood mononuclear cells (PBMCs) and lymphoid and non-lymphoid tissues collected post mortem from therapy na&iuml;ve children with AIDS. There were three major findings. First, continued evolution of R5 viruses in PBMCs, spleen and lymph nodes involved multiple bottlenecks, independent of coreceptor switch, resulting in fitter quasispecies driven by positive selection. Second, evolution of X4 strains appeared to be a sequential process requiring the initial fixation of positively selected mutations in V1-V2 and C2 domains of R5 variants before the emergence of high charge V3 X4 variants. Third, R5 viruses persisted after the emergence of CXCR4-using strains, which were found predominantly but not exclusively in the thymus.Conclusions/SignificanceOur data indicate that the evolution of X4 strains is a multi-step, temporally structured process and that the thymus may play an important role in the evolution/amplification of coreceptor variants. Development of new therapeutic protocols targeting virus in the thymus could be important to control HIV-1 infection prior to advanced disease."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000950.g005"/>

  <meta property="og:title" content="Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000950" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7668'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7905'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2810&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000950">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000950" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000950&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marco Salemi
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Marco Salemi, Brant R. Burkhardt</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:salemi@pathology.ufl.edu">salemi@pathology.ufl.edu</a> (MS); <a href="mailto:goodenow@ufl.edu">goodenow@ufl.edu</a> (MG)</p>

                <p>Affiliation:
                  Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Brant R. Burkhardt
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Marco Salemi, Brant R. Burkhardt</p>

              
              

                <p>Affiliation:
                  Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America
                </p>

                <p>
                    Current address: Department of Pathology and Laboratory Medicine, Children's Hospital of Philadelphia, Philadelphia, Pennsylvania, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rebecca R. Gray, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America, 
                  Department of Anthropology, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guity Ghaffari, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America, 
                  Department of Pediatrics, Division of Immunology, Rheumatology, and Infectious Diseases, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John W. Sleasman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pediatrics, Division of Allergy, Immunology, and Rheumatology, University of South Florida and All Children's Hospital, St. Petersburg, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maureen M. Goodenow
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:salemi@pathology.ufl.edu">salemi@pathology.ufl.edu</a> (MS); <a href="mailto:goodenow@ufl.edu">goodenow@ufl.edu</a> (MG)</p>

                <p>Affiliations:
                  Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America, 
                  Department of Pediatrics, Division of Immunology, Rheumatology, and Infectious Diseases, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000950</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000950" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000950" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000950" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000950" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000950-g001" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000950-g002" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000950-g003" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000950-g004" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000950-g005" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000950">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000950" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>During HIV-1 infection coreceptor switch from CCR5- (R5)- to CXCR4 (X4)-using viruses is associated with disease progression. X4 strains of HIV-1 are highly cytopathic to immature thymocytes. Virtually no studies have evaluated the HIV-1 quasispecies present <em>in vivo</em> within thymic and lymphoid tissues or the evolutionary relationship between R5 and X4 viruses in tissues and peripheral blood.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>High-resolution phylodynamic analysis was applied to virus envelope quasispecies in longitudinal peripheral blood mononuclear cells (PBMCs) and lymphoid and non-lymphoid tissues collected <em>post mortem</em> from therapy naïve children with AIDS. There were three major findings. First, continued evolution of R5 viruses in PBMCs, spleen and lymph nodes involved multiple bottlenecks, independent of coreceptor switch, resulting in fitter quasispecies driven by positive selection. Second, evolution of X4 strains appeared to be a sequential process requiring the initial fixation of positively selected mutations in V1-V2 and C2 domains of R5 variants before the emergence of high charge V3 X4 variants. Third, R5 viruses persisted after the emergence of CXCR4-using strains, which were found predominantly but not exclusively in the thymus.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our data indicate that the evolution of X4 strains is a multi-step, temporally structured process and that the thymus may play an important role in the evolution/amplification of coreceptor variants. Development of new therapeutic protocols targeting virus in the thymus could be important to control HIV-1 infection prior to advanced disease.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Salemi M, Burkhardt BR, Gray RR, Ghaffari G, Sleasman JW, et al.  (2007) Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies. PLoS ONE 2(9):
          e950.
            doi:10.1371/journal.pone.0000950</p><p><strong>Academic Editor: </strong>Sheila Bowyer, National Institute for Communicable Diseases, South Africa</p><p><strong>Received:</strong> April 6, 2007; <strong>Accepted:</strong> September 6, 2007; <strong>Published:</strong> September 26, 2007</p><p><strong>Copyright:</strong> © 2007 Salemi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Research was supported by PHS R01 awards HD032259, AI047723, and AI065265; General Clinical Research Center of the University of Florida MO1-RR00082; University of Florida Children's Miracle Network; Center for Research in Pediatric Immune Deficiency; Laura McClamma Research Fellowship; Pediatric Clinical Research Center of All Children's Hospital, University of South Florida, and the Maternal and Child Health Bureau, R60 MC 00003-01, Department of Health and Human Services, Health Resources and Services Administration; Robert A. Good Chair for Immunology (JWS) and Stephany W. Holloway University Chair for AIDS Research (MMG).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Infection of target cells by human immunodeficiency virus type 1 (HIV-1) requires binding of the viral surface protein gp120 to the cellular surface protein CD4 and chemokine receptors CCR5 or CXCR4 <a href="#pone.0000950-Berger1">[1]</a>. R5 viruses using the CCR5 coreceptor represent the predominant viral quasispecies during the early and chronic phases of the infection <a href="#pone.0000950-vanTWout1">[2]</a>, <a href="#pone.0000950-Shankarappa1">[3]</a>. X4 viruses using the CXCR4 coreceptor appear at a later stage in about 50% of individuals infected by HIV-1 subtype B and are associated with accelerated disease progression <a href="#pone.0000950-Koot1">[4]</a>, <a href="#pone.0000950-Connor1">[5]</a>. The reasons for coreceptor evolution during the course of infection and the origin and evolution of X4 strains are not fully understood, although several hypotheses have been proposed <a href="#pone.0000950-Regoes1">[6]</a>. Appearance of X4 viruses might reflect emergence of quasispecies sequestered in tissues at the time of infection <a href="#pone.0000950-Westby1">[7]</a> or evolution <em>de novo</em> from R5 viruses <a href="#pone.0000950-Clevestig1">[8]</a>–<a href="#pone.0000950-Margolis1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The primary genetic determinants of HIV-1 coreceptor use are concentrated within the 35-amino acid hypervariable V3 loop of the envelope protein gp120 <a href="#pone.0000950-Cann1">[11]</a>–<a href="#pone.0000950-Milich1">[13]</a>. Although a small number of basic amino acid substitutions in V3 may be sufficient for changes in coreceptor preference, combinations of V3 mutations can lead to major loss of entry fitness in culture, unless compensated by mutations in or near V1-V2 in gp120 <a href="#pone.0000950-Pastore1">[14]</a>, indicating that complex, discontinuous determinants contribute to X4 coreceptor use, at least on certain cell types <a href="#pone.0000950-Ghaffari1">[9]</a>, <a href="#pone.0000950-Cho1">[15]</a>, <a href="#pone.0000950-Pontow1">[16]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Continuing HIV-1 replication in anatomic or cellular reservoirs and release of latent virus from infected reservoirs can contribute to viral rebound following interruption of combination anti-retroviral therapy (ART) <a href="#pone.0000950-Blankson1">[17]</a>, <a href="#pone.0000950-Chun1">[18]</a>. Genital tissues and blood appear to serve as distinct reservoirs harboring latent HIV-1 during prolonged drug therapy <a href="#pone.0000950-Poss1">[19]</a>, <a href="#pone.0000950-Craigo1">[20]</a>, while the brain is a viral compartment harboring HIV-1 subpopulations with specific genetic characteristics <a href="#pone.0000950-Korber1">[21]</a>–<a href="#pone.0000950-Salemi1">[26]</a>. CD4 T lymphocytes in infants and children predominantly express CD45RA, whereas in adults about equal ratios of CD45RA or CD45RO are expressed <a href="#pone.0000950-Bleul1">[27]</a>. Only a subset of activated CD4 CD45RO T cells express CCR5, while the preponderance of CD4 T-lymphocytes, independent of CD45 isoform, express CXCR4 coreceptors <a href="#pone.0000950-Bleul1">[27]</a>, <a href="#pone.0000950-Zhang1">[28]</a>. The thymus harbors a large number of immature and mature CD4 thymocytes expressing CXCR4, but relatively limited CCR5-expressing cells, implicating the thymus as a critical compartment for HIV-1 pathogenesis <a href="#pone.0000950-Rosenzweig1">[29]</a>–<a href="#pone.0000950-Uittenbogaart1">[32]</a>. X4 viral strains are highly cytopathic to immature thymocytes <em>ex vivo</em> <a href="#pone.0000950-Brooks1">[33]</a>. Within HIV-1 infected individuals, significant reduction in thymocyte proliferation, output and function occurs in the absence of ART <a href="#pone.0000950-Douek1">[34]</a>, <a href="#pone.0000950-Dion1">[35]</a>, while HIV-induced destruction of the thymus decreases the capacity for T-cell immune reconstitution resulting in rapid disease progression in infected children <a href="#pone.0000950-Kourtis1">[36]</a>. Despite the importance of X4 strains for pathogenesis, virtually no studies have evaluated coreceptor use or the evolutionary patterns across hypervariable regions of HIV-1 <em>env</em> quasispecies infecting the thymus <em>in vivo</em> <a href="#pone.0000950-Alves1">[37]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Recently, a “<em>phylodynamic</em>” framework using phylogeny and coalescence theory was developed and applied to study evolutionary dynamics of pathogens within infected hosts <a href="#pone.0000950-Salemi1">[26]</a>, <a href="#pone.0000950-Grenfell1">[38]</a>. In the present work, we applied high-resolution <em>phylodynamics</em> to analyze HIV-1 subpopulations (<em>virodemes</em>) infecting the thymus, lymphoid and non-lymphoid tissues that may act as viral compartments and/or reservoirs <a href="#pone.0000950-Nickle1">[24]</a>, and longitudinal peripheral blood mononuclear cells (PBMCs) from HIV-1 infected children. The goal was to track the tempo and mode of appearance of X4 strains <em>in vivo</em>, to investigate the role of the thymus, and to uncover the direction of viral gene flow among tissues.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Characterization of HIV-1 viral quasispecies in tissues and peripheral blood</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In each subject V3 amino acid residues revealed a mixture of sequences with low or high net charge predicting, as confirmed by two independent algorithms, CCR5 or CXCR4 coreceptor use respectively <a href="#pone.0000950-Briggs1">[39]</a>–<a href="#pone.0000950-Jensen2">[41]</a>. Three envelope sequences from the thymus, for which the two algorithms gave discordant results, were characterized by functional analysis with single-cycle, Env-pseudotyped viruses <a href="#pone.0000950-Ghaffari1">[9]</a>. Two (from subject S1 and S2, respectively) used the CXCR4 coreceptor exclusively, while one (from subject S4) used both CCR5 and CXCR4 coreceptors.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Maximum likelihood (ML) phylogenetic trees estimated from the V1-V3 alignments of sequences sampled at the time of death from four subjects displayed significant branches among the quasispecies independent of length of infection (<a href="#pone-0000950-g001">Figure 1</a>). In subjects S1, S2 and S3 a well-supported subclade of R5-using viral variants was localized in PBMCs and distinct from quasispecies in contemporaneous tissues where X4 and R5 variants commingled (<a href="#pone-0000950-g001">Fig. 1a, 1b, and 1c</a>). X4-using strains were identified exclusively in the thymus from subjects S1 and S3, in thymus and PBMCs from subject S2, and in thymus, lymphoid tissues and peripheral lymphocytes from subject S4. Sporadic X4 strains were intermixed with R5 ones in patients S1 (<a href="#pone-0000950-g001">Figure 1a</a>) and S3 (<a href="#pone-0000950-g001">Figure 1c</a>). In contrast, a well-supported monophyletic clade of X4 strains emerged from an R5 population in patients S2 (<a href="#pone-0000950-g001">Figure 1b</a>) and S4 (<a href="#pone-0000950-g001">Figure 1d</a>). In all cases, X4 variants always clustered on branches that appeared to emerge from R5 ancestors. The tree inferred for sequences from each subject included at least one significantly supported branch within the R5 lineage, suggesting that emergence of successful viral subpopulations may require selective pressures not exclusively linked to evolution of V3 variants with altered coreceptor use.</p>
<div class="figure" id="pone-0000950-g001"><div class="img"><a name="pone-0000950-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Maximum likelihood trees of HIV-1 V1-V3 sequences from <em>post mortem</em> tissues and PBMCs of different subjects.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Branch lengths were estimated with the HKY+Γ model and were drawn in scale with the bar at the bottom indicating 0.01 nucleotide substitutions per site. The position of the root was inferred by maximum likelihood enforcing the molecular clock constraint. One * along a branch represent significant statistical support for the clade subtending that branch (zero-branch-length test <em>p</em>&lt;0.001 and a bootstrap value &gt;70%). Colored boxes and circles represent different tissues according to the color table in the figure. Monophyletic clades of PBMCs with R5 sequences are highlighted within solid circles. Broken circles highlight monophyletic clades of X4 sequences. a. Subject S1. b. Subject S2. c. Subject S3. d Subject S4.</p>
<span>doi:10.1371/journal.pone.0000950.g001</span></div>

<h4>Analysis of recombinant sequences</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>HIV-1 frequently recombines <em>in vivo</em> <a href="#pone.0000950-Morris1">[42]</a>–<a href="#pone.0000950-Mild1">[44]</a>. Since intra-patient recombination would lead to the creation of mosaic genomes violating the tree-like assumption of evolution, we carefully checked for recombinant sequences within our data sets before performing high-resolution phylogenetic analysis. ML trees inferred from V1-V2 or C2-V3 domains were identical for S1 and S3, indicating no obvious intra-patient recombination. The finding was confirmed by the PHI test for recombination (<em>p</em>&gt;0.05). In contrast, six sequences (10.9%) from S2 and 47 (31.3%) from S4 clustered in different clades depending on the domain used to infer the trees. Significant evidence of recombination was detected by the PHI test in both S2 and S4 alignments (<em>p</em>&lt;10<sup>−6</sup>). When putative recombinant sequences were removed from S2 and S4 alignments, the PHI test was no longer significant (<em>p</em>&gt;0.05). Recombinant sequences of R5 or X4 phenotype were predominantly detected in tissue samples rather than peripheral lymphocytes (<a href="#pone.0000950.s001">Table S1</a>). In subject S4 about two-thirds of the recombinant sequences were found in the brain. It is important to notice that the low rate of PCR-mediated recombination (&lt;2% per 1000 nucleotides) <a href="#pone.0000950-Meyerhans1">[45]</a>, <a href="#pone.0000950-Howell1">[46]</a>, and the significantly different distribution of recombinant sequences in different tissues makes highly unlikely that PCR-recombinants, if any, may have biased the results of the analysis.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Recombination breakpoints were mapped by bootscanning. In all 53 recombinant sequences, putative breakpoints were localized within the C2 domain, while no recombination was found within V1-V2 (data not shown). Representative bootscannings of two R5 and two X4 sequences are shown in <a href="#pone-0000950-g002">Figure 2</a>. R5 recombinants originated from ancestral sequences in peripheral blood, while the X4 recombinants combined R5 V1-V2 sequences from PBMCs and X4 V3 sequences from the thymus clade (subject S2), or the thymus/spleen/lymph clade (subject S4). Bootscanning plots for all other X4 recombinant sequences showed the same pattern (data not shown).</p>
<div class="figure" id="pone-0000950-g002"><div class="img"><a name="pone-0000950-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Recombination breakpoints in HIV-1 V1-V3 recombinant sequences from different subjects.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The cladograms (top panels) represent the main non-recombinant lineages (putative parental sequences) within the maximum likelihood trees of subject S2 and S4. The bootscanning plots (middle and bottom panel) were obtained for representative HIV-1 V1-V3 recombinant sequences (query sequences). Horizontal axis indicates the nucleotide position along a query sequence; vertical axis gives the bootstrap support for the clustering of the query sequence and the parental sequence (or clade) with a matching color in the cladogram. Vertical dotted lines localize potential recombination breakpoints. a Subject S2. Middle panel: bootscanning plot of an R5 recombinant sequence from early PBMC (PBMC_1t). Bottom panel: bootscanning plot of an X4 recombinant sequence from early PBMC (PBMC_1t). b. Subject S4. Middle panel: bootscanning plot of an R5 recombinant sequence from late PBMC (PBMC_3t). Bottom panel: bootscanning plot of an X4 recombinant sequence from thymus.</p>
<span>doi:10.1371/journal.pone.0000950.g002</span></div>

<h4>Tempo and mode of R5 and X4 variants evolution during infection</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Subjects S2 and S4 harbored X4 variants both in PBMCs and the thymus, and were selected for an in depth study of <em>in vivo</em> evolution of R5 and X4 quasispecies. Non-recombinant sequences in PBMCs over the course of infection and from terminal tissues were combined for a high-resolution phylodynamic analysis <a href="#pone.0000950-Salemi1">[26]</a>, <a href="#pone.0000950-Grenfell1">[38]</a>. The genealogy of HIV-1 V1-V3 sequences, sampled over two years of infection from subject S2, showed three main lineages, A, B, and C (<a href="#pone-0000950-g003">Figure 3a</a>). Each lineage was well supported by &gt;75% Bayesian posterior probability, <em>p</em> values≤0.001 in the zero branch length test, and &gt;70% bootstrap. Moreover, both ML and Bayesian-based methods inferred the same root for the tree. Lineage A including the R5 viral sequences from early and late PBMCs displayed clear temporal structure. Strains from early PBMC samples passed through an initial population bottleneck, followed by a second bottleneck leading to the emergence of a new monophyletic subcluster that included sequences only from late PBMC samples. HIV-1 sequences from <em>post mortem</em> lung, spleen, and lymph nodes were exclusively R5, clustered as a separate phylogenetic lineage within clade B, and included at least two bottlenecks. Temporal structure was also evident in clade C where initial populations of R5 and X4 strains isolated from early PBMCs were replaced through a bottleneck by a subclade containing only X4 variants from early and late PBMCs and from the thymus. HIV-1 X4 strains isolated from late PBMCs emerged after a second bottleneck, while the last bottleneck gave rise to a subclade consisting exclusively of X4 viral strains from the thymus.</p>
<div class="figure" id="pone-0000950-g003"><div class="img"><a name="pone-0000950-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Maximum likelihood trees of HIV-1 V1-V3 sequences from longitudinal PBMCs and <em>post mortem</em> tissues of different subjects.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Branch lengths were estimated with the HKY+Γ model and drawn in scale with the bar at the bottom indicating 0.1 nucleotide substitutions per site. The trees were rooted by maximum likelihood enforcing a molecular clock and taking into account different sampling dates. One * along a branch represent significant statistical support for the clade subtending that branch (zero-branch-length test <em>p</em>&lt;0.001; Bayesian posterior &gt;0.75, and/or bootstrap value &gt;70%). Colored boxes and circles represent different tissues according to the color table to the left of each tree. Internal thick edges indicate population bottlenecks a. Patient S2. PBMC 1t and 2t were sampled 15 and 22 months after birth, respectively. Other tissues were sampled <em>post mortem</em> 26 months after birth. The main statistically supported clades, designated A, B, and C are highlighted within broken boxes. The solid box within clade C highlights X4 sequences. b. Patient S4. PBMC 1t, 2t, and 3t were sampled 3, 22, and 77 months after birth, respectively. The other tissues were sampled <em>post mortem</em> 77 months after birth. The main statistically supported clades, designated A, B, and C are highlighted by broken boxes. The solid box within clade C highlights X4 sequences.</p>
<span>doi:10.1371/journal.pone.0000950.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The inferred phylogeny for subject S4 was based on evaluation of sequences over a period of about 6.5 years and indicated at least three statistically supported clusters: A, B, and C (<a href="#pone-0000950-g003">Figure 3b</a>). All viral sequences from PBMCs at sampling time T1 and T2, as well as from the brain, displayed V3 loops predicted to use the R5 coreceptor. The only exception was the presence of an early X4 PBMC variant at time T2. As in subject S2, sequences of HIV-1 strains from S4 PBMCs at different time points were temporally structured: PBMCs sampled at time T1 clustered at the base of the tree, near to the root, and were replaced through a bottleneck by a new population from samples collected at later time points. Sequences from brain were exclusively R5 and belonged to a separate monophyletic clade (clade B in <a href="#pone-0000950-g003">Figure 3b</a>). Clade C included HIV-1 strains from PBMCs at T3 with contemporaneous variants in the lung, lymph nodes, spleen and thymus. In contrast, the monophyletic cluster at the top of the tree included only X4 HIV-1 variants that were found in thymus, lymph nodes, and spleen. Overall, the structure of the trees from both individuals suggested a gradual emergence from R5 to X4 sequences through continuous selection of new variants evolving over time followed by an expansion after the last bottleneck of the X4 population.</p>


<h4><em>In vivo</em> evolutionary rates of R5 and X4 populations</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To test the hypothesis that the expansion of the X4 population of viruses might be due to an increased replication rate that would accelerate evolution, molecular clock analysis was used to estimate the rate of evolution for R5 and for X4 variants. Mean evolutionary rates of R5 or X4 strains within subject S2 were not significantly different. The mean evolutionary rate of the R5 strains was 1.17×10<sup>−2</sup> nucleotide substitutions per site per year (0.1–2.2×10<sup>−2</sup> lower and higher 95% highest posterior density, HPD), while the rate for X4 strains was 1.6×10<sup>−2</sup> nucleotide substitutions per site per year (0.7–2.6×10<sup>−2</sup> lower and higher 95% HPD). Although absence of longitudinal X4 sequences within subject S4 precluded estimation of evolutionary rates for X4 subpopulations, an evolutionary rate of 2.4×10<sup>−2</sup> nucleotide substitutions per site per year (1.2–3.7×10<sup>−2</sup> lower and higher 95% HPD) was estimated for the S4 R5 strains in PBMCs. Evolutionary rate for R5 strains in subject S4 were not significantly different (<em>p</em>&lt;0.001) from the evolutionary rate of either the R5 or the X4 strains within subject S2.</p>


<h4>Selection analysis during population bottlenecks</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Multiple viral population bottlenecks within R5 strains preceded the bottleneck leading to appearance of X4 variants in both individuals (<a href="#pone-0000950-g004">Figure 4</a>). To investigate the evolutionary driving forces at work during such bottlenecks, we performed a ML-based selection pressure analysis of the internal branches in the S2 and S4 genealogy (<a href="#pone-0000950-g004">Figure 4a and 4b</a>). In each case, the best fitting model was the one that allowed for both positive and negative selection along the internal branches involved in the bottlenecks (p&lt;0.05). Estimated dN/dS ratios were greater than 2 (positive selection) or less than 0.5 (purifying selection) along the internal branches. In contrast, dN/dS values were not significantly different from 1 along the internal branches of the clade that included the R5 HIV-1 quasispecies from the brain of patient S4. HIV-1 evolutionary dynamics appeared remarkably similar within both subjects. In general, bottlenecks driven by positive selection were usually followed by a bottleneck driven by purifying selection.</p>
<div class="figure" id="pone-0000950-g004"><div class="img"><a name="pone-0000950-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Selection pressure in HIV-1 V1-V3 during intra-patient viral population bottlenecks.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Rooted Bayesian phylogenetic trees were obtained using the SRD06 relaxed clock model. Branch lengths were drawn in scale with the bar at the bottom indicating 0.01 nucleotide substitutions per site. Triangles represent clades for which statistically unsupported internal branches were collapsed. Ancestral (Anc) sequences were reconstructed by maximum likelihood. Internal branches under significant positive (dN/dS&gt;2) or negative (dN/dS&lt;0.05) selection are drawn in thick red and black lines, respectively. Open circles emphasize the parent (ancestral sequence before the bottleneck) and child (ancestral sequence following the bottleneck) nodes. Specific amino acid positions (according to the HXB2 HIV-1 reference sequence) under positive selection within ancestral sequences are given in the tables at the bottom of each tree. Sequences with predicted X4 coreceptor use are indicated. a. Subject S2. Longitudinal PBMC and <em>post mortem</em> tissue samples were the same used for the tree in <a href="#pone-0000950-g002">Figure 2a</a>. b. Subject S4. Longitudinal PBMC and <em>post mortem</em> tissue samples were the same, with the exclusion of sequences from brain, used for the tree in <a href="#pone-0000950-g002">Figure 2b</a>.</p>
<span>doi:10.1371/journal.pone.0000950.g004</span></div>

<h4>Genotypic changes associated with selection</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To identify amino acid replacements most likely involved in the adaptive response of the viral quasispecies to selection pressure, V1-V3 ancestral sequences involved in the major bottlenecks from patient S2 and S4 were inferred (<a href="#pone-0000950-g004">Figure 4</a>). A similar evolutionary pattern appeared to underlie the gradual development of X4 variants from an initial R5 population in both subjects (the full V1-V3 alignment of the ancestral sequences from both subjects is given in supplemental <a href="#pone.0000950.s002">Figure S1</a>). Sites under positive selection generally occurred within the N-terminal portion of V1, within amino acid positions 132-151, and the C-terminal portion of V2, within positions 188-190, while the few sites under positive selection in C2 were interspersed along the domain. V3 mutations under positive selection were distributed across the domain and often involved replacements with high-charged basic amino acids along the branches leading to X4 variants. Generally, positively selected substitutions in V1, V2, and C2 appeared along the earlier branches of the genealogies (between Anc1/Anc2 of subject S2 or between Anc1/Anc2, Anc2/Anc3, Anc3/Anc4 of subject S4), and were fixed in all subsequent viral populations. Selected substitutions in V3 appeared only after V1-V2 changes along the late bottlenecks (between Anc3/Anc4 and Anc5/Anc6 of S2 or between Anc5/Anc6 and Anc7/Anc8 of S4). The mutation from serine to arginine at position 306, which is associated with coreceptor use <a href="#pone.0000950-Fouchier1">[47]</a>, appeared in the ancestral sequences at the origin of X4 lineages (between Anc5/Anc6 in S2 and Anc7/Anc8 in S4). Position 268 in C2 was also under positive selection in both subjects, although in one case G268E appeared during an early bottleneck within the R5 population (subject S4), while in a second case, E268K appeared during a late bottleneck within the X4 population (subject S2).</p>


<h4>Migration analysis</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>While viral sequences from the thymus of subject S2 were X4, mixtures of R5 and X4 quasispecies were found in the thymus and other lymphoid organs from subject S4. To assess the HIV-1 population dynamic within patient S4, the direction of gene flow among <em>virodemes</em> in late PBMC samples and <em>post-mortem</em> tissues was tracked (<a href="#pone-0000950-g005">Figure 5</a>). Sequences from the brain represented a separate compartment of R5 strains <a href="#pone.0000950-Korber1">[21]</a>–<a href="#pone.0000950-Salemi1">[26]</a>, and were not included in the gene-flow analysis. Migration events among different tissues were significantly less than those expected from a random model in which each <em>virodeme</em> is freely diffusible and equally likely to exchange virus with any other one (<em>p</em>&lt;0.0001). Results supported a model of restricted gene-flow within viral subpopulations in the thymus and other tissues (<a href="#pone-0000950-g005">Figure 5</a>). PBMCs and thymus accounted for about 86% of total HIV-1 gene outflow, with 53% from PBMCs (R5 sequences) and 33% from thymus (both R5 and X4 sequences).</p>
<div class="figure" id="pone-0000950-g005"><div class="img"><a name="pone-0000950-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000950" data-uri="info:doi/10.1371/journal.pone.0000950.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000950.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000950.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000950.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000950.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>HIV-1 migration analysis among different tissues for patient S4.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Each circle is proportional to the percentage of observed migrations (given within the circle) inferred from the S4 maximum likelihood cladogram in Supplemental <a href="#pone.0000950.s003">Figure s2</a>. Migration counts less than 1% were not indicated.</p>
<span>doi:10.1371/journal.pone.0000950.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Studies of HIV-1 evolution <em>in vivo</em> have focused primarily on a “whole body” approach, where viral evolution is mainly inferred from cell-free or cell-associated HIV-1 genomes in blood or, occasionally, within one or two tissues <a href="#pone.0000950-Shankarappa1">[3]</a>, <a href="#pone.0000950-Poss1">[19]</a>–<a href="#pone.0000950-Korber1">[21]</a>, <a href="#pone.0000950-Shapshak1">[23]</a>, <a href="#pone.0000950-Nickle1">[24]</a>, <a href="#pone.0000950-Salemi1">[26]</a>. In contrast, our study included detailed mapping of the evolutionary patterns of HIV-1 <em>virodemes</em> in blood, as well as lymphoid and non-lymphoid tissues, and applied phylogenetic and population genetic tools to examine the dynamics of virus interaction within the host. Most studies have focused almost exclusively on the V3 loop as the genetic marker, while none tested for positive selection in the internal branches of reconstructed genealogies, which is a hallmark of ancestral episodic selection leading to adaptive response <a href="#pone.0000950-Messier1">[48]</a>. Inclusion of <em>env</em> V1-V2 domains coupled with internal branch tests for positive selection in our analysis was critical to uncover episodic selection within HIV-1 quasispecies.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The role of selection and random genetic drift in the <em>in vivo</em> evolution of HIV-1 envelope and in the emergence of CXCR4 variants associated with rapid disease progression has been debated for some time with some evidence supporting each model <a href="#pone.0000950-Shankarappa1">[3]</a>, <a href="#pone.0000950-Shriner2">[49]</a>, <a href="#pone.0000950-Edwards1">[50]</a>. In our study, positive selection involving amino acid residues in the V1, V2 and C2 domains was detected before the emergence of X4 strains, i.e. independent of coreceptor switch. In contrast, the ancestral sequences at the origin of the X4 strains in the thymus and other lymphoid organs contained amino acid replacements in V3 leading to increase net charge of the V3 loops.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The natural history of HIV-1 among the infected subjects seemed to be defined by sequential population bottlenecks characterized by temporally ordered patterns of amino acid substitutions. Within each individual X4 variants evolved <em>de novo</em> from R5 ancestors ruling out the hypothesis of long term sequestration of transmitted X4 variants. Although, in general, different spectra of amino acid replacements in the V1-C2 gp120 domains developed in each individual, sites under positive selection in early bottlenecks were generally restricted within the N-terminal portion of V1 and the C-terminal portion of V2. Three positions in V2, C2, and V3, were found to be under positive selection across subjects. Substitutions that accumulated in specific amino acid residues <em>in vivo</em> were identical to amino acid changes that developed during <em>in vitro</em> evolution <a href="#pone.0000950-Pastore1">[14]</a>. <em>In vitro</em>, combinations of V3 substitutions can lead to major loss of entry fitness or even lethality unless compensated by mutations in or near V1-V2 <a href="#pone.0000950-Pastore1">[14]</a>, <a href="#pone.0000950-Kiselyeva1">[51]</a>. Our study provides evidence that changes outside the V3 domain may be essential for setting the background for the emergence HIV-1 X4 strains <em>in vivo</em>, in agreement with reports indicating that other Env regions outside V3 contribute to CXCR4 coreceptor use and cell tropism <a href="#pone.0000950-Ghaffari1">[9]</a>, <a href="#pone.0000950-Pastore1">[14]</a>–<a href="#pone.0000950-Pontow1">[16]</a>. Overall, our data suggest that the evolution of HIV-1 envelope involved a complex, but nonetheless ordered and potentially restricted, developmental program that was recapitulated in different individuals.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In addition to amino acid substitutions in V1-C2, recombination was detected between V1-V2 and V3 in two individuals. Rather than random distribution of putative breakpoints in the recombinants, essentially all sequences had crossover localized to the C2 region. Evolution of HIV-1 in the brain of one subject was highly compartmentalized and limited to CCR5-using variants, in agreement with previous findings <a href="#pone.0000950-Salemi1">[26]</a>, <a href="#pone.0000950-Sha1">[52]</a>. An elevated recombination rate among the R5 sequences in the brain, which our data identified, would be consistent with a long independent evolution of a segregated viral subpopulation in a separate compartment. Since only <em>post mortem</em> samples were available from brain tissue, phylogenetic analysis could not exclude earlier input of viral sequences and turnover similar to PBMCs. Analysis of X4 strains of recombinant origin showed that recombination always occurred within C2 between ancestral V1V2 sequences of R5 phenotype from PBMCs and ancestral V3 sequences of X4 phenotype from thymus, suggesting that the thymus may play an important role in the amplification of CXCR4-using strains.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We also found evidence of bottlenecks characterized by strong purifying selection, after a positive selection episode, suggesting the presence of temporary adaptive peaks in the fitness landscape. In fact, it is expected that if a population reaches a local high-fitness peak, most of the variants in the next generation would be removed by purifying selection or genetic drift. Emergence of fitter HIV-1 strains through the bottlenecks could be due to multiple selective factors: antiretroviral therapy, cellular immune control, change in target cell populations or change in host milieu favoring X4 variants <a href="#pone.0000950-Cocchi1">[53]</a>, and/or emergence of viral variants with enhanced entry efficiency. In the subjects in our study, positive selection occurred in the absence of combination antiretroviral treatment, ruling out drug selective pressure. All patients had severely suppressed CD4 T cells and none of the known B-cell epitopes (listed at the Los Alamos HIV databases) were localized in the V1-V3 region of sequences from any of our subjects, arguing against immune control as a major selection factor, although lack of immunity, as found in the SIV model, may provide a selective advantage <a href="#pone.0000950-Ho1">[54]</a>. Our data are consistent with a model of enhanced viral entry efficiency combined with host milieu as major selective pressures driving episodic selection.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The results obtained in the present work also point to the potential importance of the thymus for the evolution/amplification of the X4 coreceptor use. Thymocytes express high levels of CD4 and CXCR4 and a transcriptionally active environment that promotes viral replication <a href="#pone.0000950-Chene1">[55]</a>. Trafficking of lymphocytes between thymus and secondary lymphoid tissues is highly regulated and typically unidirectional <a href="#pone.0000950-Picker1">[56]</a>. The finding in one of the subjects that HIV-1 X4 gene flow was from thymus to lymphoid tissues, but not <em>vice versa</em>, is consistent with trafficking patterns of thymocytes and implicates X4-infected thymocytes as a potential mechanism for systemic dissemination of X4 variants. Studies based on experimental observations and mathematical models suggest that the increased turnover rate of naïve T-cells over time in HIV-1 infected patients could be a consequence of progressive depletion of memory T-cells in the periphery, and may explain the increased fitness of X4 viruses and their emergence in about 50% of the individuals during the late stage of the disease <a href="#pone.0000950-Ribeiro1">[57]</a>. Our finding that the evolutionary rates of R5 and X4 HIV-1 sub-populations are not significantly different support the hypothesis that amplification of X4 variants is due to factors other than an increased rate of evolution and may be linked to the availability of target cells. In contrast to early reports that thymic function declines during adolescence and almost disappears in early adulthood <a href="#pone.0000950-Simpson1">[58]</a>, a substantial body of evidence now shows that the adult thymus retains some thymopoietic function and continues to produce naïve T-cells for export to the periphery <a href="#pone.0000950-Leino1">[59]</a>–<a href="#pone.0000950-Douek3">[62]</a>, raising the possibility that our findings from HIV-1 infected children may be applicable to older individuals, as well.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Some limitations of the present study should be recognized. We examined HIV-1 quasispecies in thymus tissues from only four subjects. Since no <em>ante mortem</em> thymic or lymphoid tissues were available, the exact evolutionary history of the X4 population in the thymus at earlier time points is uncertain. For example, we cannot exclude that similar X4 variants may have emerged in other tissues (such as lymph nodes or spleen) prior to or concomitant with their appearance in the thymus. Also, the result of the migration analysis is not informative about the viral flow during early infection and would require further investigation using longitudinal samples from different subjects to be confirmed. However, while it is feasible to collect serial PBMC samples from earlier time points, serial biopsies of tissues such as thymus and spleen are hardly an option in human patients. Such a difficulty points to the importance of animal models, such as SIV-infected rhesus macaque, to discern the role for the thymus and secondary lymphoid tissues in evolution of X4 variants.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Further clarification of the key-role of the thymus and other lymphoid tissues in the evolution/amplification of X4 strains might have important consequences for the development of effective therapeutic strategies. R5 coreceptor-blocking agents may be extremely efficient, during the early stage of the disease, in avoiding the emergence of the X4 quasispecies associated with certain types of disease progression. On the other hand, the development of X4 entry inhibitors and/or drugs able to target the export of X4 infected T-cells from the thymus could be critical to control HIV-1 infection in advanced disease stages. If breakdown of immunity contributes to selection for X4 variants, then immune-based strategies may delay or prevent amplification of X4-using viruses independent of the tissue of origin. A detailed mapping of V1-V3 sites under positive selection associated with increased viral entry efficiency will lay the foundations for the development and evaluation of such novel drugs, and our study has shown the potential power of phylodynamics and high-resolution phylogeny for accomplishing this important task.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Subjects and samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Four pediatric subjects infected by maternal HIV-1 transmission (S1, S2, and S4) or by neonatal blood transfusion (S3) were enrolled under a protocol approved by the Institutional Review Board of the University of Florida, College of Medicine. The mothers of the children enrolled in the study has already given consent for the collection and storage of blood and tissue samples and for collection of clinical data as part of the protocol implemented in Dr. Goodenow's lab entitled: Biological implications of HIV-1 genetic variability. All patients developed AIDS before one year of age, and died of AIDS-related illnesses by 8 months (S1), 26 months (S2), 6.5 years (S4), or 7.5 years (S3) of age. Subjects received antiretroviral therapy with nucleoside reverse transcriptase inhibitors (NRTI), but no combination therapy with non-NRTI or protease inhibitors. Tissues including lung, mesenteric lymph nodes, spleen, thymus, and brain were obtained <em>post mortem</em>, while peripheral blood mononuclear cells (PBMCs) were obtained at or near the time of death for all subjects and over the course of infection for S2, and S4. In patient S4 brain tissues were mostly sampled from the frontal lobe. DNA was extracted from cryopreserved PBMC samples as previously described <a href="#pone.0000950-Lamers1">[63]</a>, <a href="#pone.0000950-Tuttle1">[64]</a>. Tissues were quick frozen in liquid nitrogen in 50 ml conical tubes and stored at −80°C until processed for DNA extraction. DNA was extracted from multiple biopsies from each tissue using the Dneasy tissue extraction kit (QIAGEN, Valencia, CA) <a href="#pone.0000950-Burkhardt1">[65]</a>. Several DNA extractions from each tissue were pooled together, and multiple PCR amplifications were performed on the combined DNA extraction to ensure representation of viral sequences within a tissue.</p>


<h4>Amplification, cloning and sequencing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The V1-V3 hypervariable region of envelope was amplified using primers and conditions previously described <a href="#pone.0000950-Tuttle2">[66]</a>, followed by ligation into PCR 2.1 vector (Invitrogen, Carlsbad, CA) and transformation of competent Top10F/(Invitrogen, Carlsbad, CA) cells. Sequences were prepared with DYEnamic ET dye terminator cycle sequencing kit for MegaBACE DNA Analysis Systems (GE Healthcare, Chalfont St. Giles, United Kingdom), and run on a MegaBACE 1000 (GE Healthcare) in the Genome Sequencing Service Laboratory at the University of Florida.</p>


<h4>Analysis of V1-V3 sequences and coreceptor usage prediction</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Sequences were edited, verified, and entered into HIV<sub>base</sub> for retrieval and analysis <a href="#pone.0000950-Lamers2">[67]</a>. For each domain an amino acid alignment was obtained manually using our motif-base alignment method <a href="#pone.0000950-Lamers3">[68]</a> and translated back to nucleotides for further analysis. HIV-1 subtype was assessed with the Rega HIV subtypying tool version 2.0 (<a href="http://www.bioafrica.net/virus-genotype/html/subtypinghiv.html">http://www.bioafrica.net/virus-genotype/​html/subtypinghiv.html</a>). V1-V3 sequences from all subjects clustered with subtype B reference sequences. Coreceptor usage was predicted with two different algorithms: 1. By calculating the net charge of the V3 loop based on number and position of amino acid residues (K+R)-(D+E) <a href="#pone.0000950-Briggs1">[39]</a>; 2. By using a position-specific scoring matrix (PSSM) developed for subtype B sequences <a href="#pone.0000950-Jensen1">[40]</a>, <a href="#pone.0000950-Jensen2">[41]</a>. Both methods gave the same results except for three sequences for which coreceptor usage had to be determined experimentally (see below).</p>


<h4>Determination of coreceptor usage and actual phenotype</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The V1-V5 hypervariable region of envelope of each strain was amplified as described previously <a href="#pone.0000950-Tuttle2">[66]</a>. V1-V5 sequences were cloned into pcDNA expression vectors and used to generate single-cycle viruses tagged with luciferase (luc), as previously described <a href="#pone.0000950-Ghaffari1">[9]</a>, <a href="#pone.0000950-Tuttle2">[66]</a>. To determine coreceptor use and cell tropism, PBMC, monocyte-derived macrophages, and MT-2 cells were infected with V1-V5 single-cycle <em>env</em> pseudo-typed viruses in the absence or presence of monoclonal antibody specific for either CCR5 (2D7) or CXCR4 (12G5) (AIDS Research and Reference Reagent program, Division of AIDS, NIAID, NIH). Virus phenotype was defined <a href="#pone.0000950-Goodenow1">[69]</a>.</p>


<h4>Recombination analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>HIV-1 gp120 <em>env</em> sequences (V1 to V3 domains) from all individuals were evaluated in a single phylogenetic tree that verified the integrity of the data. Separate phylogenetic trees for V1-V2 and C2-V3 domains were also obtained to detect putative recombinant sequences that may cluster differently in different trees. The presence of recombination was confirmed with the PHI test, which is based on the notion of refined incompatibility score <a href="#pone.0000950-Bruen1">[70]</a>, and it is implemented in SplitsTree package version 4.8 <a href="#pone.0000950-Huson1">[71]</a>. Extensive simulation studies and comparison with other available methods have shown that not only the PHI test is extremely powerful in detecting recombination, but it is also the method producing the lowest number of false positives <a href="#pone.0000950-Bruen1">[70]</a>. Significance of the PHI statistic for the presence of recombination is assessed with the normal approximation of a permutation test where, under the null hypothesis of no recombination, sites along the alignment are randomly permuted to obtain the null distribution of PHI: <em>p</em>&lt;0.05 indicate significant presence of recombination <a href="#pone.0000950-Goodenow1">[69]</a>.</p>


<h4>Mapping recombination breakpoints</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Recombinant sequences were analyzed with the bootscanning method implemented in the Simplot package <a href="#pone.0000950-Lole1">[72]</a> to locate putative recombination breakpoints. Bootscanning infers phylogenetic trees using a sliding window along an alignment including a query sequence (the putative recombinant sequence) and putative parental (non-recombinant) sequences. For each tree along the alignment 1000 bootstrap replicates are generated and the bootstrap support for the clustering of the query sequence with each of the pre-defined parental groups is recorded. Bootscanning plots, like the ones showed in <a href="#pone-0000950-g003">Figure 3</a>, display how the bootstrap support (y-axis) for the clustering of the query sequence with the parental sequences changes along the alignment (x-axis). Since recombination leads to the creation of mosaic genomes originated from different ancestors, a recombination event can be detected by the “jumping” of the query sequence between highly supported phylogenetic clades in trees obtained from different genomic regions <a href="#pone.0000950-Salminen1">[73]</a>. Bootscanning plots were obtained using a window of 200 nucleotides sliding forward in steps of 20 nucleotides. Trees were inferred by NJ using HKY estimated distances with a transition transversion ratio empirically estimated by maximum likelihood for each window.</p>


<h4>Phylogenetic Analysis of non-recombinant data sets</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>A total of 33 sequences for S1 (median per tissue 8, range 5 to 13); 49 sequences for S2 (median/tissue 9, range 6 to 12); 17 sequences for S3 (median/tissue 8, range 1 to 17); and 103 sequences for S4 (median/tissue 15, range 3 to 35) were analyzed. The best fitting nucleotide substitution model was tested with a hierarchical likelihood ratio test, using a neighbor-joining (NJ) tree with LogDet corrected distances <a href="#pone.0000950-Swofford1">[74]</a>. Maximum likelihood (ML) trees were then inferred with the selected model (HKY+Γ, 8 categories, for all data sets) and ML-estimated substitution parameters. The heuristic search for the best tree was performed using an NJ tree as starting tree and the TBR branch-swapping algorithm. Neighbor-Joining (NJ) trees were also estimated using pair-wise distances inferred by ML with the best fitting nucleotide substitution model. Calculations were performed with PAUP* 4.0b10 written by David L. Swofford. Statistical support for internal branches in the NJ trees was obtained by bootstrapping (1000 replicates) for the NJ trees and the ML-based zero branch length test for the ML trees <a href="#pone.0000950-Swofford1">[74]</a>. Trees were rooted by ML rooting by selecting the rooted tree with the best likelihood under the molecular clock constraint, or by outgroup rooting using the earliest PBMC samples as outgroup. The location of the root was confirmed by inferring rooted Bayesian trees with a relaxed clock model and exponentially distributed evolutionary rates as prior. The Bayesian calculation consisted of 50,000,000 generations Markov Chains Monte Carlo (MCMC) with sampling every 5000<sup>th</sup> generation using the BEAST software package version 1.4 (<a href="http://evolve.zoo.ox.ac.uk/beast/">http://evolve.zoo.ox.ac.uk/beast/</a>). Convergence of the MCMC was assessed by calculating the effective sampling size (ESS) of the combined runs <a href="#pone.0000950-Drummond1">[75]</a>. All parameter estimates showed significant ESS (&gt;250). Bayesian trees were also obtained with the program MrBayes v3.1.2, using the HKY+Γ model, running in parallel two MCMC for 10,000,000 generations with sampling every 100<sup>th</sup> generation. Convergence was assessed by comparing the average standard deviation of split frequencies (<em>p</em>&lt;0.0001). Statistical support for each clade in the Bayesian trees was obtained by calculating clade-specific Bayesian posterior probabilities with MrBayes. In each case, ML and Bayesian methods inferred the same topology.</p>


<h4>Reconstruction of ancestral sequences and positive selection analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Ancestral amino acid sequences in the genealogy obtained for each patient were inferred by the maximum likelihood method, using the codon substitution model M0 <a href="#pone.0000950-Yang1">[76]</a>. Positive selection analysis was performed by comparing the different <em>Branch</em> maximum likelihood codon substitution models using the improved test 2, which tests for different dN/dS (nonsynonymous/synonymous substitutions) ratios along given branches of the tree <a href="#pone.0000950-Yang1">[76]</a>, <a href="#pone.0000950-Yang2">[77]</a>. Three different models were compared: model 0, assuming a single dN/dS for the entire tree; model 1, assuming a different dN/dS for each branch in the tree, and model 2 assuming a baseline dN/dS = 1 for the entire tree and several different dN/dS along the branches of the major bottleneck events in the genealogy. Model 0, model 1, and model 2 are nested and a hierarchical likelihood ratio test was used to check which model fitted the data significantly better <a href="#pone.0000950-Yang1">[76]</a>, <a href="#pone.0000950-Yang2">[77]</a>. Average dN/dS ratios among maximum likelihood inferred ancestral sequences at the internal nodes of the significantly supported monophyletic clades of the trees were also compared. Specific amino acid changes along the internal branches of the tree were inferred by maximum likelihood. Calculations were performed with the PAML package <a href="#pone.0000950-Yang1">[76]</a>.</p>


<h4>Molecular clock analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Ultrametric trees were obtained by enforcing a molecular clock on the inferred genealogy, and re-estimating the branch lengths and substitution parameters with maximum likelihood with the previously selected evolutionary model. The clock hypothesis was tested with the likelihood ratio test. Evolutionary rates with a strict and a relaxed clock model using a lognormal distribution as prior <a href="#pone.0000950-Drummond1">[75]</a> were obtained with the Bayesian framework implemented in the BEAST program version 1.4, by running 50,000,000 Markov Chain Monte Carlo MCMC with sampling every 5000<sup>th</sup> generation. Calculating the ESS assessed convergence of the MCMC procedure. ESS&gt;250 were considered significant.</p>


<h4>Gene flow tests and migration counts</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>The hypothesis of compartmentalization, i.e. the existence of distinct HIV-1 sub-populations in different tissues, was tested by the Slatkin and Maddison test for gene flow <a href="#pone.0000950-Slatikin1">[78]</a> using the MacClade version 4 program (Sinauer Associates, Sunderland, MA). A one-character data matrix is obtained from the original data set by assigning to each <em>taxon</em> in the tree a one-letter code indicating its tissue of origin. Then, the phylogenetic tree obtained from the nucleotide aligniment is imported in MacClade and the putative origin of each ancestral sequence (i.e. internal node) in the tree is inferred with the Fitch algorithm <a href="#pone.0000950-Fitch1">[79]</a> by finding the most parsimonious reconstruction (MPR) of the ancestral character (this can be accomplished by selecting the <em>Trace character</em> option in the <em>Trace</em> menu of MacClade). The result is a tree colored as in supplemental <a href="#pone.0000950.s003">Figure S2</a>, were each colored branch represent the tissue of origin of the internal node (ancestral sequence) or tip node (actual sequence) subtending that branch. A change in color (i.e. in tissue assignment) between two branches connected trough a node represent a migration event from one tissue to another that must have occurred during the genealogical evolution of the sequences under investigation. The final tree-length, i.e. the number of observed migrations in the genealogy, computed by MacClade can be compared to the tree-length distribution of 10,000 trees obtained by random joining-splitting (by selecting the <em>Character steps/etc</em> option from the <em>Char</em> menu). Observed genealogies significantly shorter than random trees indicate the presence of subdivided populations <a href="#pone.0000950-Slatikin1">[78]</a>. Specific migrations among different compartments (states) were traced with the <em>State changes and stasis</em> tool (MacClade), which counts the number of changes in a tree for each pair-wise state. When multiple MPRs were present (as in our data sets), the algorithm calculated the average migration count over all possible MPRs for each pair. The resulting pair-wise migration matrix is then normalized to obtain the percentage of observed migration to/from different tissues in the tree.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000950"></div><a name="pone.0000950.s001" id="pone.0000950.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000950.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Distribution of HIV-1 V1V3 recombinant sequences</p>
<p class="siDoi">doi:10.1371/journal.pone.0000950.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000950.s002" id="pone.0000950.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000950.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">V1-V3 multiple alignment of reconstructed ancestral sequences from patients S2 and S4. Ancestral (Anc) sequences, corresponding to the internal nodes in the trees in <a href="#pone-0000950-g004">Figure 4</a>, are aligned with the HIV-1 reference strain HXB2 on top. Blank spaces indicate gaps. A dash indicates that the amino acid matches the corresponding position of HXB2. In the V3 loop, red and blue shadings indicate positively and negatively charged amino acids, respectively, which are important for the determination of coreceptor use. Two arrows indicate residues 11 and 25 considered important for coreceptor use (Fouchier et al. 1994). Specific sites under positive selection are outlined by solid boxes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000950.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.24 MB TIF)</p>
<a name="pone.0000950.s003" id="pone.0000950.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000950.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Maximum parsimony migration analysis for patient S4. Cladogram of HIV-1 sequences from different tissues showing clade C of the maximum likelihood tree given in <a href="#pone-0000950-g003">Figure 3</a>. The color of a branch indicates the tissue of origin of the top node (ancestral sequence) or tip (actual sequence) of that branch according to the color table in the figure. Tissues of origin for the internal nodes (ancestral sequences) of the cladogram were inferred with the Fitch (1971) algorithm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000950.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.28 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The following reagents were obtained through the AIDS Research and Reference Reagent Program, Division of AIDS, NIAID, NIH: 3T3.CD4 CXCR4, 3T3.CD4 CCR5, from Dr. Dan R. Littman; CCR5 monoclonal antibody (2D7) from Millennium Pharmaceuticals, Inc. and PharMingen; CXCR4 monoclonal antibody (12G5) from Dr. James Hoxie.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MS MG. Performed the experiments: BB. Analyzed the data: MS BB RG MG. Contributed reagents/materials/analysis tools: MS JS MG GG. Wrote the paper: MS. Other: Performed some sequencing: GG. Performed the experiments for determination of co-receptor usage: GG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000950-Berger1" id="pone.0000950-Berger1"></a>Berger EA, Murphy PM, Farber JM (1999) Chemokine receptors as HIV-1 coreceptors: roles in viral entry, tropism, and disease. Ann Rev Immunol  17: 675–700.  <ul class="find" data-citedArticleID="997164" data-doi="10.1146/annurev.immunol.17.1.657"><li><a href="http://dx.doi.org/10.1146/annurev.immunol.17.1.657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemokine+receptors+as+HIV-1+coreceptors%3A+roles+in+viral+entry%2C+tropism%2C+and+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemokine+receptors+as+HIV-1+coreceptors%3A+roles+in+viral+entry%2C+tropism%2C+and+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000950-vanTWout1" id="pone.0000950-vanTWout1"></a>van'T Wout AB, Koostra NA, Mulder-Kampinga GA, Albrecht-van Lent N, Scherpbier HJ, et al.  (1994) Macrophage-tropic variants initiate human immunodeficiency virus type 1 infection after sexual, parenteral, and vertical transmission. J Clin Invest  94: 2060–2067.  <ul class="find" data-citedArticleID="997318" data-doi="10.1172/jci117560"><li><a href="http://dx.doi.org/10.1172/jci117560" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Macrophage-tropic+variants+initiate+human+immunodeficiency+virus+type+1+infection+after+sexual%2C+parenteral%2C+and+vertical+transmission." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Macrophage-tropic+variants+initiate+human+immunodeficiency+virus+type+1+infection+after+sexual%2C+parenteral%2C+and+vertical+transmission.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000950-Shankarappa1" id="pone.0000950-Shankarappa1"></a>Shankarappa R, Margolick JB, Gange SJ, Rodrigo AG, Upchurch D, et al.  (1999) Consistent viral evolutionary changes associated with the progression of human immunodeficiency virus type 1 infection. J Virol  73: 10489–10502.  <ul class="find" data-citedArticleID="997284"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection.&amp;auth=&amp;atitle=Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000950-Koot1" id="pone.0000950-Koot1"></a>Koot M, van'T Wout AB, Koostra NA, Degoede REY, Tersmette M, et al.  (1993) Prognostic value of HIV-1 syncytium-inducing phenotype for rate of CD4<sup>+</sup> cell depletion and progression to AIDS. Ann Intern Med  118: 681–688.  <ul class="find" data-citedArticleID="997232" data-doi="10.7326/0003-4819-118-9-199305010-00004"><li><a href="http://dx.doi.org/10.7326/0003-4819-118-9-199305010-00004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prognostic+value+of+HIV-1+syncytium-inducing+phenotype+for+rate+of+CD4%2B+cell+depletion+and+progression+to+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prognostic+value+of+HIV-1+syncytium-inducing+phenotype+for+rate+of+CD4%2B+cell+depletion+and+progression+to+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000950-Connor1" id="pone.0000950-Connor1"></a>Connor RI, Sheridan KE, Ceradini D, Choe S, Landau NR (1997) Change in coreceptor use correlates with disease progression in HIV-1-infected individuals. J Exp Med  185: 621–628.  <ul class="find" data-citedArticleID="997192" data-doi="10.1084/jem.185.4.621"><li><a href="http://dx.doi.org/10.1084/jem.185.4.621" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Change+in+coreceptor+use+correlates+with+disease+progression+in+HIV-1-infected+individuals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Change+in+coreceptor+use+correlates+with+disease+progression+in+HIV-1-infected+individuals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000950-Regoes1" id="pone.0000950-Regoes1"></a>Regoes RR, Bonhoeffer S (2005) The HIV coreceptor switch: a population dynamical perspective. Trends Microbiol  6: 269–277.  <ul class="find" data-citedArticleID="997272" data-doi="10.1016/j.tim.2005.04.005"><li><a href="http://dx.doi.org/10.1016/j.tim.2005.04.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+HIV+coreceptor+switch%3A+a+population+dynamical+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+HIV+coreceptor+switch%3A+a+population+dynamical+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000950-Westby1" id="pone.0000950-Westby1"></a>Westby M, Lewis M, Whitcomb J, Youle M, Pozniak AL, et al.  (2006) Emergence of CXCR4-using human immunodeficiency virus type 1 (HIV-1) variants in a minority of HIV-1-infected patients following treatment with CCR5 antagonist Maraviroc is from a pretreatment CXCR4-using virus reservoirs. J Virol  80: 4909–4920.  <ul class="find" data-citedArticleID="997308" data-doi="10.1128/jvi.80.10.4909-4920.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.10.4909-4920.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+of+CXCR4-using+human+immunodeficiency+virus+type+1+%28HIV-1%29+variants+in+a+minority+of+HIV-1-infected+patients+following+treatment+with+CCR5+antagonist+Maraviroc+is+from+a+pretreatment+CXCR4-using+virus+reservoirs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+of+CXCR4-using+human+immunodeficiency+virus+type+1+%28HIV-1%29+variants+in+a+minority+of+HIV-1-infected+patients+following+treatment+with+CCR5+antagonist+Maraviroc+is+from+a+pretreatment+CXCR4-using+virus+reservoirs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000950-Clevestig1" id="pone.0000950-Clevestig1"></a>Clevestig P, Maljkovic I, Casper C, Carlenor E, Lindgen S, et al.  (2005) The X4 phenotype of HIV-1 evolves from R5 in two children of mothers, carrying X4, and is not linked to transmission. AIDS Res Hum Retrovir  5: 371–378.  <ul class="find" data-citedArticleID="997188" data-doi="10.1089/aid.2005.21.371"><li><a href="http://dx.doi.org/10.1089/aid.2005.21.371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+X4+phenotype+of+HIV-1+evolves+from+R5+in+two+children+of+mothers%2C+carrying+X4%2C+and+is+not+linked+to+transmission." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+X4+phenotype+of+HIV-1+evolves+from+R5+in+two+children+of+mothers%2C+carrying+X4%2C+and+is+not+linked+to+transmission.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000950-Ghaffari1" id="pone.0000950-Ghaffari1"></a>Ghaffari G, Tuttle DL, Briggs D, Burkhardt BR, Bhatt D, et al.  (2005) Complex determinants in human immunodeficiency virus type 1 envelope gp120 mediate CXCR4-dependent infection in macrophages. J Virol  79: 13250–13261.  <ul class="find" data-citedArticleID="997212" data-doi="10.1128/jvi.79.21.13250-13261.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.21.13250-13261.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+determinants+in+human+immunodeficiency+virus+type+1+envelope+gp120+mediate+CXCR4-dependent+infection+in+macrophages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+determinants+in+human+immunodeficiency+virus+type+1+envelope+gp120+mediate+CXCR4-dependent+infection+in+macrophages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000950-Margolis1" id="pone.0000950-Margolis1"></a>Margolis L, Shattock R (2006) Selective transmission of CCR5-utilizing HIV-1: the ‘gatekeeper’ problem resolved? Nat Rev Microbiol  4: 312–317.  <ul class="find" data-citedArticleID="997250" data-doi="10.1038/nrmicro1387"><li><a href="http://dx.doi.org/10.1038/nrmicro1387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+transmission+of+CCR5-utilizing+HIV-1%3A+the+%E2%80%98gatekeeper%E2%80%99+problem+resolved%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+transmission+of+CCR5-utilizing+HIV-1%3A+the+%E2%80%98gatekeeper%E2%80%99+problem+resolved%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000950-Cann1" id="pone.0000950-Cann1"></a>Cann AJ, Curcher MJ, Boyd M, O'Brien W, Zhao JQ, et al.  (1992) The region of the envelope gene of human immunodeficiency virus type 1 responsible for determination of cell tropism. J Virol  66: 305–309.  <ul class="find" data-citedArticleID="997178"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+region+of+the+envelope+gene+of+human+immunodeficiency+virus+type+1+responsible+for+determination+of+cell+tropism.&amp;auth=&amp;atitle=The+region+of+the+envelope+gene+of+human+immunodeficiency+virus+type+1+responsible+for+determination+of+cell+tropism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+region+of+the+envelope+gene+of+human+immunodeficiency+virus+type+1+responsible+for+determination+of+cell+tropism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+region+of+the+envelope+gene+of+human+immunodeficiency+virus+type+1+responsible+for+determination+of+cell+tropism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000950-Stamatos1" id="pone.0000950-Stamatos1"></a>Stamatos L, Cheng-Mayer C (1993) Evidence that the structural conformation of envelope gp120 affects human immunodeficiency virus type 1 infectivity, host range, and syncytium-forming ability. J Virol  67: 5635–5639.  <ul class="find" data-citedArticleID="997298"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+that+the+structural+conformation+of+envelope+gp120+affects+human+immunodeficiency+virus+type+1+infectivity%2C+host+range%2C+and+syncytium-forming+ability.&amp;auth=&amp;atitle=Evidence+that+the+structural+conformation+of+envelope+gp120+affects+human+immunodeficiency+virus+type+1+infectivity%2C+host+range%2C+and+syncytium-forming+ability." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+the+structural+conformation+of+envelope+gp120+affects+human+immunodeficiency+virus+type+1+infectivity%2C+host+range%2C+and+syncytium-forming+ability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+the+structural+conformation+of+envelope+gp120+affects+human+immunodeficiency+virus+type+1+infectivity%2C+host+range%2C+and+syncytium-forming+ability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000950-Milich1" id="pone.0000950-Milich1"></a>Milich LB, Margolin B, Swanstrom R (1997) Patterns of amino acid variability in NSI-like and SI-like V3 sequences and a linked change in the CD4-binding domain of the HIV-1 Env protein. Virology  239: 108–118.  <ul class="find" data-citedArticleID="997258" data-doi="10.1006/viro.1997.8821"><li><a href="http://dx.doi.org/10.1006/viro.1997.8821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+amino+acid+variability+in+NSI-like+and+SI-like+V3+sequences+and+a+linked+change+in+the+CD4-binding+domain+of+the+HIV-1+Env+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+amino+acid+variability+in+NSI-like+and+SI-like+V3+sequences+and+a+linked+change+in+the+CD4-binding+domain+of+the+HIV-1+Env+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000950-Pastore1" id="pone.0000950-Pastore1"></a>Pastore C, Nedellec R, Ramos A, Pontow S, Ratner L, et al.  (2006) Human immunodeficiency virus type 1 coreceptor switching: V1/V2 gain-of-fitness mutations compensate for V3 loss-of-fitness mutations. J Virol  80: 750–758.  <ul class="find" data-citedArticleID="997264" data-doi="10.1128/jvi.80.2.750-758.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.2.750-758.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+type+1+coreceptor+switching%3A+V1%2FV2+gain-of-fitness+mutations+compensate+for+V3+loss-of-fitness+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+type+1+coreceptor+switching%3A+V1%2FV2+gain-of-fitness+mutations+compensate+for+V3+loss-of-fitness+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000950-Cho1" id="pone.0000950-Cho1"></a>Cho MW, Lee MK, Carney MC, Berson JF, Doms RW, Martin MA (1998) Identification of determinants on a dualtropic human immunodeficiency virus type 1 envelope glycoprotein that confer usage of CXCR4. J Virol.  72: 2509–2515.  <ul class="find" data-citedArticleID="997184"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+determinants+on+a+dualtropic+human+immunodeficiency+virus+type+1+envelope+glycoprotein+that+confer+usage+of+CXCR4.&amp;auth=&amp;atitle=Identification+of+determinants+on+a+dualtropic+human+immunodeficiency+virus+type+1+envelope+glycoprotein+that+confer+usage+of+CXCR4." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+determinants+on+a+dualtropic+human+immunodeficiency+virus+type+1+envelope+glycoprotein+that+confer+usage+of+CXCR4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+determinants+on+a+dualtropic+human+immunodeficiency+virus+type+1+envelope+glycoprotein+that+confer+usage+of+CXCR4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000950-Pontow1" id="pone.0000950-Pontow1"></a>Pontow S, Ratner L (2001) Evidence for common structural determinants of human immunodeficiency virus type 1 coreceptor activity provided through functional analysis of CCR5/CXCR4 chimeric coreceptors. J Virol.  75: 11503–11514.  <ul class="find" data-citedArticleID="997268" data-doi="10.1128/jvi.75.23.11503-11514.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.23.11503-11514.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+common+structural+determinants+of+human+immunodeficiency+virus+type+1+coreceptor+activity+provided+through+functional+analysis+of+CCR5%2FCXCR4+chimeric+coreceptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+common+structural+determinants+of+human+immunodeficiency+virus+type+1+coreceptor+activity+provided+through+functional+analysis+of+CCR5%2FCXCR4+chimeric+coreceptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000950-Blankson1" id="pone.0000950-Blankson1"></a>Blankson JN, Persaud D, Siliciano RF (2002) The challenge of viral reservoirs in HIV-1 infection. Annual Reviews in Medicine  53: 557–593.  <ul class="find" data-citedArticleID="997166" data-doi="10.1146/annurev.med.53.082901.104024"><li><a href="http://dx.doi.org/10.1146/annurev.med.53.082901.104024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+challenge+of+viral+reservoirs+in+HIV-1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+challenge+of+viral+reservoirs+in+HIV-1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000950-Chun1" id="pone.0000950-Chun1"></a>Chun TW, Nickle DC, Justement JS, Large D, Semerjian A, et al.  (2005) HIV-infected individuals receiving effective antiviral therapy for extended periods of time continually replenish their viral reservoir. J Clin Invest  115: 3250–3255.  <ul class="find" data-citedArticleID="997186" data-doi="10.1172/jci26197"><li><a href="http://dx.doi.org/10.1172/jci26197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-infected+individuals+receiving+effective+antiviral+therapy+for+extended+periods+of+time+continually+replenish+their+viral+reservoir." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-infected+individuals+receiving+effective+antiviral+therapy+for+extended+periods+of+time+continually+replenish+their+viral+reservoir.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000950-Poss1" id="pone.0000950-Poss1"></a>Poss M, Rodrigo AG, Gosink JJ, Learn GH, de Vange Panteleeff D, et al.  (1998) Evolution of envelope sequences from the genital tract and peripheral blood of women infected with clade A human immunodeficiency virus type 1. J Virol  72: 8240–51.  <ul class="find" data-citedArticleID="997270"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+envelope+sequences+from+the+genital+tract+and+peripheral+blood+of+women+infected+with+clade+A+human+immunodeficiency+virus+type+1.&amp;auth=&amp;atitle=Evolution+of+envelope+sequences+from+the+genital+tract+and+peripheral+blood+of+women+infected+with+clade+A+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+envelope+sequences+from+the+genital+tract+and+peripheral+blood+of+women+infected+with+clade+A+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+envelope+sequences+from+the+genital+tract+and+peripheral+blood+of+women+infected+with+clade+A+human+immunodeficiency+virus+type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000950-Craigo1" id="pone.0000950-Craigo1"></a>Craigo JK, Patterson BK, Paranjpe S, Kulka K, Ding M, et al.  (2004) Persistent HIV type 1 infection in semen and blood compartments in patients after long-term potent antiretroviral therapy. AIDS Res Hum Retrovir  20: 1196–11209.  <ul class="find" data-citedArticleID="997194" data-doi="10.1089/0889222042544947"><li><a href="http://dx.doi.org/10.1089/0889222042544947" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Persistent+HIV+type+1+infection+in+semen+and+blood+compartments+in+patients+after+long-term+potent+antiretroviral+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Persistent+HIV+type+1+infection+in+semen+and+blood+compartments+in+patients+after+long-term+potent+antiretroviral+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000950-Korber1" id="pone.0000950-Korber1"></a>Korber BT, Kunstman KJ, Patterson BK, Furtado M, McEvilly MM, et al.  (1994) Genetic differences between blood-and brain-derived viral sequences from human immunodeficiency virus type 1-infected patients: Evidence of conserved elements in the V3 region of the envelope protein of brain-derived sequences. J Virol.  68: 7467–7481.  <ul class="find" data-citedArticleID="997234"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+differences+between+blood-and+brain-derived+viral+sequences+from+human+immunodeficiency+virus+type+1-infected+patients%3A+Evidence+of+conserved+elements+in+the+V3+region+of+the+envelope+protein+of+brain-derived+sequences.&amp;auth=&amp;atitle=Genetic+differences+between+blood-and+brain-derived+viral+sequences+from+human+immunodeficiency+virus+type+1-infected+patients%3A+Evidence+of+conserved+elements+in+the+V3+region+of+the+envelope+protein+of+brain-derived+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+differences+between+blood-and+brain-derived+viral+sequences+from+human+immunodeficiency+virus+type+1-infected+patients%3A+Evidence+of+conserved+elements+in+the+V3+region+of+the+envelope+protein+of+brain-derived+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+differences+between+blood-and+brain-derived+viral+sequences+from+human+immunodeficiency+virus+type+1-infected+patients%3A+Evidence+of+conserved+elements+in+the+V3+region+of+the+envelope+protein+of+brain-derived+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000950-Chang1" id="pone.0000950-Chang1"></a>Chang J, Jozwiak R, Wang B, Ng T, Ge YC, et al.  (1998) Unique HIV type 1 V3 region sequences derived from six different regions of brain: region-specific evolution within host-determined quasispecies. AIDS Res Hum Retrovir  14: 25–30.  <ul class="find" data-citedArticleID="997180" data-doi="10.1089/aid.1998.14.25"><li><a href="http://dx.doi.org/10.1089/aid.1998.14.25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unique+HIV+type+1+V3+region+sequences+derived+from+six+different+regions+of+brain%3A+region-specific+evolution+within+host-determined+quasispecies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unique+HIV+type+1+V3+region+sequences+derived+from+six+different+regions+of+brain%3A+region-specific+evolution+within+host-determined+quasispecies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000950-Shapshak1" id="pone.0000950-Shapshak1"></a>Shapshak P, Segal DM, Crandall KA, Fujimura RK, Zhang BT, et al.  (1999) Independent evolution of HIV type 1 in different brain regions. AIDS Res Hum Retrovir  15: 811–820.  <ul class="find" data-citedArticleID="997286" data-doi="10.1089/088922299310719"><li><a href="http://dx.doi.org/10.1089/088922299310719" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independent+evolution+of+HIV+type+1+in+different+brain+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independent+evolution+of+HIV+type+1+in+different+brain+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000950-Nickle1" id="pone.0000950-Nickle1"></a>Nickle DC, Jensen MA, Shriner D, Brodie SJ, Frenkel LM, et al.  (2003) Evolutionary Indicators of human immunodeficiency virus type 1 reservoirs and compartments. J Virol  77: 5540–5546.  <ul class="find" data-citedArticleID="997262" data-doi="10.1128/jvi.77.9.5540-5546.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.9.5540-5546.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+Indicators+of+human+immunodeficiency+virus+type+1+reservoirs+and+compartments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+Indicators+of+human+immunodeficiency+virus+type+1+reservoirs+and+compartments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000950-Smit1" id="pone.0000950-Smit1"></a>Smit TK, Brew BJ, Tourtelotte W, Morgello S, Gelman BB, et al.  (2004) Independent evolution of human immunodeficiency virus (HIV) drug resistance mutations in diverse areas of the brain in HIV-infected patients, with and without dementia, on antiretroviral treatment.   <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000950-Salemi1" id="pone.0000950-Salemi1"></a>Salemi M, Lamers SL, Yu S, de Oliveira T, Fitch WM, et al.  (2005) HIV-1 phylodynamic analysis in distinct brain compartments provides a model for the neuropathogenesis of AIDS. J Virol  79: 11343–11352.  <ul class="find" data-citedArticleID="997278" data-doi="10.1128/jvi.79.17.11343-11352.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.17.11343-11352.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+phylodynamic+analysis+in+distinct+brain+compartments+provides+a+model+for+the+neuropathogenesis+of+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+phylodynamic+analysis+in+distinct+brain+compartments+provides+a+model+for+the+neuropathogenesis+of+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000950-Bleul1" id="pone.0000950-Bleul1"></a>Bleul CC, Wu L, Hoxie JA, Springer TA, Mackay CR (1997) The HIV coreceptors CXCR4 and CCR5 are differentially expressed and regulated on human T lymphocytes. Proc Natl Acad Sci USA  94: 1925–1930.  <ul class="find" data-citedArticleID="997168" data-doi="10.1073/pnas.94.5.1925"><li><a href="http://dx.doi.org/10.1073/pnas.94.5.1925" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+HIV+coreceptors+CXCR4+and+CCR5+are+differentially+expressed+and+regulated+on+human+T+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+HIV+coreceptors+CXCR4+and+CCR5+are+differentially+expressed+and+regulated+on+human+T+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000950-Zhang1" id="pone.0000950-Zhang1"></a>Zhang L, He T, Talal A, Wang G, Frankel SH, et al.  (1998) In vivo distribution of the human immunodeficiency virus/simian immunodeficiency virus coreceptors: CXCR4, CCR3, CCR5. J Virol  72: 5035–5045.  <ul class="find" data-citedArticleID="997316"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+vivo+distribution+of+the+human+immunodeficiency+virus%2Fsimian+immunodeficiency+virus+coreceptors%3A+CXCR4%2C+CCR3%2C+CCR5.&amp;auth=&amp;atitle=In+vivo+distribution+of+the+human+immunodeficiency+virus%2Fsimian+immunodeficiency+virus+coreceptors%3A+CXCR4%2C+CCR3%2C+CCR5." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+distribution+of+the+human+immunodeficiency+virus%2Fsimian+immunodeficiency+virus+coreceptors%3A+CXCR4%2C+CCR3%2C+CCR5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+distribution+of+the+human+immunodeficiency+virus%2Fsimian+immunodeficiency+virus+coreceptors%3A+CXCR4%2C+CCR3%2C+CCR5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000950-Rosenzweig1" id="pone.0000950-Rosenzweig1"></a>Rosenzweig M, Clark DP, Gaulton GN (1993) Selective thymocyte depletion in neonatal HIV-1 thymic infection. AIDS  7: 1601–1605.  <ul class="find" data-citedArticleID="997276" data-doi="10.1097/00002030-199312000-00009"><li><a href="http://dx.doi.org/10.1097/00002030-199312000-00009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+thymocyte+depletion+in+neonatal+HIV-1+thymic+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+thymocyte+depletion+in+neonatal+HIV-1+thymic+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000950-Kitchen1" id="pone.0000950-Kitchen1"></a>Kitchen SG, Zack JA (1997) CXCR4 expression during lymphopoiesis: implications for human immunodeficiency virus type 1 infection of the thymus. J Virol.  171: 6928–6934.  <ul class="find" data-citedArticleID="997230"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CXCR4+expression+during+lymphopoiesis%3A+implications+for+human+immunodeficiency+virus+type+1+infection+of+the+thymus.&amp;auth=&amp;atitle=CXCR4+expression+during+lymphopoiesis%3A+implications+for+human+immunodeficiency+virus+type+1+infection+of+the+thymus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CXCR4+expression+during+lymphopoiesis%3A+implications+for+human+immunodeficiency+virus+type+1+infection+of+the+thymus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CXCR4+expression+during+lymphopoiesis%3A+implications+for+human+immunodeficiency+virus+type+1+infection+of+the+thymus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000950-Zaitseva1" id="pone.0000950-Zaitseva1"></a>Zaitseva MB, Lee S, Rabin RL, Tiffany HL, Farber JM, et al.  (1998) CXCR4 and CCR5 on human thymocytes: biological function and role in HIV-1 infection. J Immunol  161: 3103–13.  <ul class="find" data-citedArticleID="997314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CXCR4+and+CCR5+on+human+thymocytes%3A+biological+function+and+role+in+HIV-1+infection.&amp;auth=&amp;atitle=CXCR4+and+CCR5+on+human+thymocytes%3A+biological+function+and+role+in+HIV-1+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CXCR4+and+CCR5+on+human+thymocytes%3A+biological+function+and+role+in+HIV-1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CXCR4+and+CCR5+on+human+thymocytes%3A+biological+function+and+role+in+HIV-1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000950-Uittenbogaart1" id="pone.0000950-Uittenbogaart1"></a>Uittenbogaart CH, Boscardin WJ, Anisman-Posner DJ, Koka PS, Bristol G, et al.  (2000) Effect of cytokines on HIV-induced depletion of thymocytes in vivo. AIDS  14: 1317–1325.  <ul class="find" data-citedArticleID="997306" data-doi="10.1097/00002030-200007070-00003"><li><a href="http://dx.doi.org/10.1097/00002030-200007070-00003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+cytokines+on+HIV-induced+depletion+of+thymocytes+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+cytokines+on+HIV-induced+depletion+of+thymocytes+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000950-Brooks1" id="pone.0000950-Brooks1"></a>Brooks DG, Kitchen SG, Kitchen CM, Scripture-Adams DD, Zack JA (2001) Generation of HIV latency during thymopoiesis. Nat Med  7: 459–464.  <ul class="find" data-citedArticleID="997172" data-doi="10.1038/86531"><li><a href="http://dx.doi.org/10.1038/86531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+of+HIV+latency+during+thymopoiesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+of+HIV+latency+during+thymopoiesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000950-Douek1" id="pone.0000950-Douek1"></a>Douek DC, Betts MR, Hill BJ, Little SJ, Lempicki R, et al.  (2001) Evidence for increase T-cell turnover and decreased thymic output in HIV infection. J Immunol  167: 6663–6668.  <ul class="find" data-citedArticleID="997198"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+for+increase+T-cell+turnover+and+decreased+thymic+output+in+HIV+infection.&amp;auth=&amp;atitle=Evidence+for+increase+T-cell+turnover+and+decreased+thymic+output+in+HIV+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+increase+T-cell+turnover+and+decreased+thymic+output+in+HIV+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+increase+T-cell+turnover+and+decreased+thymic+output+in+HIV+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000950-Dion1" id="pone.0000950-Dion1"></a>Dion ML, Poulin JF, Bordi R, Sylvestre M, Corsini R, et al.  (2004) HIV infection rapidly induces and maintains a substantial suppression of thymocytes proliferation. Immunity  21: 757–768.  <ul class="find" data-citedArticleID="997196" data-doi="10.1016/j.immuni.2004.10.013"><li><a href="http://dx.doi.org/10.1016/j.immuni.2004.10.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+infection+rapidly+induces+and+maintains+a+substantial+suppression+of+thymocytes+proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+infection+rapidly+induces+and+maintains+a+substantial+suppression+of+thymocytes+proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000950-Kourtis1" id="pone.0000950-Kourtis1"></a>Kourtis AP, Ibegbu C, Nahmias AJ, Lee FK, Clark WS, et al.  (1997) Early progression of disease in HIV-infected infants with thymus dysfunction. N Engl J Med  335: 1431–1436.  Erratum in: New Engl J Med 336: 595.  <ul class="find" data-citedArticleID="997236" data-doi="10.1056/nejm199611073351904"><li><a href="http://dx.doi.org/10.1056/nejm199611073351904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+progression+of+disease+in+HIV-infected+infants+with+thymus+dysfunction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+progression+of+disease+in+HIV-infected+infants+with+thymus+dysfunction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000950-Alves1" id="pone.0000950-Alves1"></a>Alves K, Canzian M, Delwart EL (2002) HIV type 1 envelope quasispecies in the thymus and lymph nodes of AIDS patients. AIDS Res Hum Retrovir  2: 161–165.  <ul class="find" data-citedArticleID="997162" data-doi="10.1089/08892220252779700"><li><a href="http://dx.doi.org/10.1089/08892220252779700" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+type+1+envelope+quasispecies+in+the+thymus+and+lymph+nodes+of+AIDS+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+type+1+envelope+quasispecies+in+the+thymus+and+lymph+nodes+of+AIDS+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000950-Grenfell1" id="pone.0000950-Grenfell1"></a>Grenfell BT, Pybus OG, Gog JR, Wood JL, Daly JM, et al.  (2004) Unifying the epidemiological and evolutionary dynamics of pathogens. Science  303: 327–332.  <ul class="find" data-citedArticleID="997216" data-doi="10.1126/science.1090727"><li><a href="http://dx.doi.org/10.1126/science.1090727" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unifying+the+epidemiological+and+evolutionary+dynamics+of+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unifying+the+epidemiological+and+evolutionary+dynamics+of+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000950-Briggs1" id="pone.0000950-Briggs1"></a>Briggs DR, Tuttle DL, Sleasman JW, Goodenow MM (2000) Envelope V3 amino acid sequence predicts HIV-1 phenotype (coreceptor usage and tropism for macrophages). AIDS  14: 2937–2939.  <ul class="find" data-citedArticleID="997170" data-doi="10.1097/00002030-200012220-00016"><li><a href="http://dx.doi.org/10.1097/00002030-200012220-00016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Envelope+V3+amino+acid+sequence+predicts+HIV-1+phenotype+%28coreceptor+usage+and+tropism+for+macrophages%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Envelope+V3+amino+acid+sequence+predicts+HIV-1+phenotype+%28coreceptor+usage+and+tropism+for+macrophages%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000950-Jensen1" id="pone.0000950-Jensen1"></a>Jensen MA, Li FS, van't Wout AB, Nickle DC, Shriner D, et al.  (2003) Improved coreceptor usage prediction and genotypic monitoring of R5-to-X4 transition by motif analysis of human immunodeficiency virus type 1 Env V3 loop sequences. J. Virol.  77: 13376–13388.  <ul class="find" data-citedArticleID="997224" data-doi="10.1128/jvi.77.24.13376-13388.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.24.13376-13388.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+coreceptor+usage+prediction+and+genotypic+monitoring+of+R5-to-X4+transition+by+motif+analysis+of+human+immunodeficiency+virus+type+1+Env+V3+loop+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+coreceptor+usage+prediction+and+genotypic+monitoring+of+R5-to-X4+transition+by+motif+analysis+of+human+immunodeficiency+virus+type+1+Env+V3+loop+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000950-Jensen2" id="pone.0000950-Jensen2"></a>Jensen MA, Coetzer M, van't Wout AB, Morris L, JI (2006) A Reliable Phenotype Predictor for Human Immunodeficiency Virus Type 1 Subtype C Based on Envelope V3 Sequences. J. Virol.  80: 4698–4704.  <ul class="find" data-citedArticleID="997226" data-doi="10.1128/jvi.80.10.4698-4704.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.10.4698-4704.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Reliable+Phenotype+Predictor+for+Human+Immunodeficiency+Virus+Type+1+Subtype+C+Based+on+Envelope+V3+Sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Reliable+Phenotype+Predictor+for+Human+Immunodeficiency+Virus+Type+1+Subtype+C+Based+on+Envelope+V3+Sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000950-Morris1" id="pone.0000950-Morris1"></a>Morris A, Marsden M, Halcrow K, Hughes ES, Brettle RP, et al.  (1999) Mosaic structure of the human immunodeficiency virus type 1 genome infecting lymphoid cells and the brain: evidence for frequent in vivo recombination events in the evolution of regional populations. J. Virol.  73: 8720–8731.  <ul class="find" data-citedArticleID="997260"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mosaic+structure+of+the+human+immunodeficiency+virus+type+1+genome+infecting+lymphoid+cells+and+the+brain%3A+evidence+for+frequent+in+vivo+recombination+events+in+the+evolution+of+regional+populations.&amp;auth=&amp;atitle=Mosaic+structure+of+the+human+immunodeficiency+virus+type+1+genome+infecting+lymphoid+cells+and+the+brain%3A+evidence+for+frequent+in+vivo+recombination+events+in+the+evolution+of+regional+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mosaic+structure+of+the+human+immunodeficiency+virus+type+1+genome+infecting+lymphoid+cells+and+the+brain%3A+evidence+for+frequent+in+vivo+recombination+events+in+the+evolution+of+regional+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mosaic+structure+of+the+human+immunodeficiency+virus+type+1+genome+infecting+lymphoid+cells+and+the+brain%3A+evidence+for+frequent+in+vivo+recombination+events+in+the+evolution+of+regional+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000950-Shriner1" id="pone.0000950-Shriner1"></a>Shriner D, Rodrigo AG, Nickle DC, Mullins JI (2004) Pervasive genomic recombination of HIV-1 in vivo. Genetics  167: 1573–1583.  <ul class="find" data-citedArticleID="997288" data-doi="10.1534/genetics.103.023382"><li><a href="http://dx.doi.org/10.1534/genetics.103.023382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pervasive+genomic+recombination+of+HIV-1+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pervasive+genomic+recombination+of+HIV-1+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000950-Mild1" id="pone.0000950-Mild1"></a>Mild M, Esbjornnson J, Fenyo EM, Medstrand P (2007) Frequent intrapatient recombination between human immunodeficiency virus type 1 R5 and X4 envelopes: implications for coreceptor switch. J. Virol.  81: 3369–3376.  <ul class="find" data-citedArticleID="997256" data-doi="10.1128/jvi.01295-06"><li><a href="http://dx.doi.org/10.1128/jvi.01295-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+intrapatient+recombination+between+human+immunodeficiency+virus+type+1+R5+and+X4+envelopes%3A+implications+for+coreceptor+switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+intrapatient+recombination+between+human+immunodeficiency+virus+type+1+R5+and+X4+envelopes%3A+implications+for+coreceptor+switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000950-Meyerhans1" id="pone.0000950-Meyerhans1"></a>Meyerhans A, Vartanian JP, Wain-Hobson S (1990) Recombination during PCR. Nucleic Acids Res.  18: 1687–1691.  <ul class="find" data-citedArticleID="997254" data-doi="10.1093/nar/18.7.1687"><li><a href="http://dx.doi.org/10.1093/nar/18.7.1687" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+during+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+during+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000950-Howell1" id="pone.0000950-Howell1"></a>Howell RM, Fitzgibbon JE, Noe M, Ren ZJ, Gocke DJ, et al.  (1991) <em>In vivo</em> sequence of HIV-1 enve gene: evidence for recombination among variants found in a single individual. AIDS Res. Hum. Retroviruses  7: 869–876.  <ul class="find" data-citedArticleID="997220" data-doi="10.1089/aid.1991.7.869"><li><a href="http://dx.doi.org/10.1089/aid.1991.7.869" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+sequence+of+HIV-1+enve+gene%3A+evidence+for+recombination+among+variants+found+in+a+single+individual.+AIDS+Res.+Hum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+sequence+of+HIV-1+enve+gene%3A+evidence+for+recombination+among+variants+found+in+a+single+individual.+AIDS+Res.+Hum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000950-Fouchier1" id="pone.0000950-Fouchier1"></a>Fouchier RAM, Brouwer M, Broersen SM, Schuitemaker H (1995) Simple determination of human immunodeficiency virus type 1 syncytium-inducing V3 genotype by PCR. J Clin Microbiol  33: 906–911.  <ul class="find" data-citedArticleID="997210"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simple+determination+of+human+immunodeficiency+virus+type+1+syncytium-inducing+V3+genotype+by+PCR.&amp;auth=&amp;atitle=Simple+determination+of+human+immunodeficiency+virus+type+1+syncytium-inducing+V3+genotype+by+PCR." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simple+determination+of+human+immunodeficiency+virus+type+1+syncytium-inducing+V3+genotype+by+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simple+determination+of+human+immunodeficiency+virus+type+1+syncytium-inducing+V3+genotype+by+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000950-Messier1" id="pone.0000950-Messier1"></a>Messier W, Stewart CB (1997) Episodic adaptive evolution of primate lysozymes. Nature  385: 151–154.  <ul class="find" data-citedArticleID="997252" data-doi="10.1038/385151a0"><li><a href="http://dx.doi.org/10.1038/385151a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Episodic+adaptive+evolution+of+primate+lysozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Episodic+adaptive+evolution+of+primate+lysozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000950-Shriner2" id="pone.0000950-Shriner2"></a>Shriner D, Shankarappa R, Jensen MA, Nickle DC, Mittler JE, et al.  (2004b) Influence of random genetic drift on human immunodeficiency virus type 1 env evolution during chronic infection. Genetics  166: 1155–1164.  <ul class="find" data-citedArticleID="997290" data-doi="10.1534/genetics.166.3.1155"><li><a href="http://dx.doi.org/10.1534/genetics.166.3.1155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+random+genetic+drift+on+human+immunodeficiency+virus+type+1+env+evolution+during+chronic+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+random+genetic+drift+on+human+immunodeficiency+virus+type+1+env+evolution+during+chronic+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000950-Edwards1" id="pone.0000950-Edwards1"></a>Edwards CT, Holmes EC, Pybus OG, Wilson DJ, Viscidi RP, et al.  (2006) Evolution of the human immunodeficiency virus envelope gene is dominated by purifying selection. Genetics.  174: 1441–1453.  <ul class="find" data-citedArticleID="997206" data-doi="10.1534/genetics.105.052019"><li><a href="http://dx.doi.org/10.1534/genetics.105.052019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+human+immunodeficiency+virus+envelope+gene+is+dominated+by+purifying+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+human+immunodeficiency+virus+envelope+gene+is+dominated+by+purifying+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000950-Kiselyeva1" id="pone.0000950-Kiselyeva1"></a>Kiselyeva Y, Nedellec R, Ramos A, Pastore C, Margolis LB, et al.  (2007) Evolution of CXCR4-using HIV-1 SF162 is associated with two unique envelope mutations. J Virol  0: JVI.02310–06v1.  <ul class="find" data-citedArticleID="997228" data-doi="10.1128/jvi.02310-06"><li><a href="http://dx.doi.org/10.1128/jvi.02310-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+CXCR4-using+HIV-1+SF162+is+associated+with+two+unique+envelope+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+CXCR4-using+HIV-1+SF162+is+associated+with+two+unique+envelope+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000950-Sha1" id="pone.0000950-Sha1"></a>Sha M, Smit TK, Morgello S, Tourtelotte W, Gelman B, et al.  (2006) Env gp120 sequence analysis of HIV type 1 strains from diverse areas of the brain shows preponderance of CCR5 usage. AIDS Res Hum Retrovir  22: 177–181.  <ul class="find" data-citedArticleID="997282" data-doi="10.1089/aid.2006.22.177"><li><a href="http://dx.doi.org/10.1089/aid.2006.22.177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Env+gp120+sequence+analysis+of+HIV+type+1+strains+from+diverse+areas+of+the+brain+shows+preponderance+of+CCR5+usage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Env+gp120+sequence+analysis+of+HIV+type+1+strains+from+diverse+areas+of+the+brain+shows+preponderance+of+CCR5+usage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000950-Cocchi1" id="pone.0000950-Cocchi1"></a>Cocchi F, DeVico AL, Garzino-Demo A, Ayra SK, Gallo RC, et al.  (1985) The V3 domain of the HIV-1 gp120 envelope glycoprotein is critical for chemokine-mediated blockade of infection. Science  270: 1811–1815.  <ul class="find" data-citedArticleID="997190" data-doi="10.1038/nm1196-1244"><li><a href="http://dx.doi.org/10.1038/nm1196-1244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+V3+domain+of+the+HIV-1+gp120+envelope+glycoprotein+is+critical+for+chemokine-mediated+blockade+of+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+V3+domain+of+the+HIV-1+gp120+envelope+glycoprotein+is+critical+for+chemokine-mediated+blockade+of+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000950-Ho1" id="pone.0000950-Ho1"></a>Ho SH, Tasca S, Shek L, Li A, Gettie A, et al.  (2007) Coreceptor switch in R5-tropic simian/human immunodeficiency virus-infected macaques. J Virol  81: 8621–8633.  <ul class="find" data-citedArticleID="997218" data-doi="10.1128/jvi.00759-07"><li><a href="http://dx.doi.org/10.1128/jvi.00759-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coreceptor+switch+in+R5-tropic+simian%2Fhuman+immunodeficiency+virus-infected+macaques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coreceptor+switch+in+R5-tropic+simian%2Fhuman+immunodeficiency+virus-infected+macaques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000950-Chene1" id="pone.0000950-Chene1"></a>Chene L, Nugeyre MT, Guillemard E, Moulian N, Barre-Sinoussi F, et al.  (1999) Thymocyte-thymic epithelial cell interaction leads to high-level replication of human immunodeficiency virus exclusively in mature CD4(+) CD8(−) CD3(+) thymocytes: a critical role for tumor necrosis factor and interleukin-7. J Virol  73: 7533–7542.  <ul class="find" data-citedArticleID="997182"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Thymocyte-thymic+epithelial+cell+interaction+leads+to+high-level+replication+of+human+immunodeficiency+virus+exclusively+in+mature+CD4%28%2B%29+CD8%28%3F%29+CD3%28%2B%29+thymocytes%3A+a+critical+role+for+tumor+necrosis+factor+and+interleukin-7.&amp;auth=&amp;atitle=Thymocyte-thymic+epithelial+cell+interaction+leads+to+high-level+replication+of+human+immunodeficiency+virus+exclusively+in+mature+CD4%28%2B%29+CD8%28%3F%29+CD3%28%2B%29+thymocytes%3A+a+critical+role+for+tumor+necrosis+factor+and+interleukin-7." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thymocyte-thymic+epithelial+cell+interaction+leads+to+high-level+replication+of+human+immunodeficiency+virus+exclusively+in+mature+CD4%28%2B%29+CD8%28%E2%88%92%29+CD3%28%2B%29+thymocytes%3A+a+critical+role+for+tumor+necrosis+factor+and+interleukin-7." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thymocyte-thymic+epithelial+cell+interaction+leads+to+high-level+replication+of+human+immunodeficiency+virus+exclusively+in+mature+CD4%28%2B%29+CD8%28%E2%88%92%29+CD3%28%2B%29+thymocytes%3A+a+critical+role+for+tumor+necrosis+factor+and+interleukin-7.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000950-Picker1" id="pone.0000950-Picker1"></a>Picker LJ, Butcher EC (1992) Physiological and molecular mechanisms of lymphocyte homing. Annu Rev Immunol  10: 561–591.  <ul class="find" data-citedArticleID="997266" data-doi="10.1146/annurev.iy.10.040192.003021"><li><a href="http://dx.doi.org/10.1146/annurev.iy.10.040192.003021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physiological+and+molecular+mechanisms+of+lymphocyte+homing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physiological+and+molecular+mechanisms+of+lymphocyte+homing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000950-Ribeiro1" id="pone.0000950-Ribeiro1"></a>Ribeiro RM, Hazenberg MD, Perelson AS, Davenport MP (2006) Naïve and memory cell turnover as drivers of CCR5-to-CXCR4 tropism switch in human immunodeficiency virus type 1: implications for therapy. J Virol  80: 802–809.  <ul class="find" data-citedArticleID="997274" data-doi="10.1128/jvi.80.2.802-809.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.2.802-809.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Na%C3%AFve+and+memory+cell+turnover+as+drivers+of+CCR5-to-CXCR4+tropism+switch+in+human+immunodeficiency+virus+type+1%3A+implications+for+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Na%C3%AFve+and+memory+cell+turnover+as+drivers+of+CCR5-to-CXCR4+tropism+switch+in+human+immunodeficiency+virus+type+1%3A+implications+for+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000950-Simpson1" id="pone.0000950-Simpson1"></a>Simpson JG, Gray ES, Beck JS (1975) Age involution in the normal human adult thymus. Clin Exp Immunol  19: 261–265.  <ul class="find" data-citedArticleID="997292"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Age+involution+in+the+normal+human+adult+thymus.&amp;auth=&amp;atitle=Age+involution+in+the+normal+human+adult+thymus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Age+involution+in+the+normal+human+adult+thymus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Age+involution+in+the+normal+human+adult+thymus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000950-Leino1" id="pone.0000950-Leino1"></a>Leino L, Lilius E-M, Nikoskelainen J, Pelliniemi T-T, Rajamaki A (1991) The reappearance of 10 differentiation antigens on peripheral blood lymphocytes after allogenic bone marrow transplantation. Bone Mar Transpl  8: 339–344.  <ul class="find" data-citedArticleID="997244"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+reappearance+of+10+differentiation+antigens+on+peripheral+blood+lymphocytes+after+allogenic+bone+marrow+transplantation.&amp;auth=&amp;atitle=The+reappearance+of+10+differentiation+antigens+on+peripheral+blood+lymphocytes+after+allogenic+bone+marrow+transplantation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+reappearance+of+10+differentiation+antigens+on+peripheral+blood+lymphocytes+after+allogenic+bone+marrow+transplantation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+reappearance+of+10+differentiation+antigens+on+peripheral+blood+lymphocytes+after+allogenic+bone+marrow+transplantation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000950-Mackall1" id="pone.0000950-Mackall1"></a>Mackall C, Fleisher T, Brown M, Andrich M, Chen C, et al.  (1995) Age, thymopoiesis, and CD4+ T-lymphocyte regeneration afdter intensive chemotherapy. N Engl J Med  332: 143–149.  <ul class="find" data-citedArticleID="997248" data-doi="10.1056/nejm199501193320303"><li><a href="http://dx.doi.org/10.1056/nejm199501193320303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Age%2C+thymopoiesis%2C+and+CD4%2B+T-lymphocyte+regeneration+afdter+intensive+chemotherapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Age%2C+thymopoiesis%2C+and+CD4%2B+T-lymphocyte+regeneration+afdter+intensive+chemotherapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000950-Douek2" id="pone.0000950-Douek2"></a>Douek DC, Koup RA (2000) Evidence for thymic function in the elderly. Vaccine  18: 1638–1641.  <ul class="find" data-citedArticleID="997200" data-doi="10.1016/s0264-410x(99)00499-5"><li><a href="http://dx.doi.org/10.1016/s0264-410x(99)00499-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+thymic+function+in+the+elderly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+thymic+function+in+the+elderly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000950-Douek3" id="pone.0000950-Douek3"></a>Douek DC, McFarland RD, Keiser PH, Gage EA, Massey JM, et al.  (1998) Changes in thymic function with age and during the treatment of HIV infection. Nature  396: 690–695.  <ul class="find" data-citedArticleID="997202"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Changes+in+thymic+function+with+age+and+during+the+treatment+of+HIV+infection.&amp;auth=&amp;atitle=Changes+in+thymic+function+with+age+and+during+the+treatment+of+HIV+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+thymic+function+with+age+and+during+the+treatment+of+HIV+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+thymic+function+with+age+and+during+the+treatment+of+HIV+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000950-Lamers1" id="pone.0000950-Lamers1"></a>Lamers SL, Sleasman JW, She JX, Barrie KA, Pomeroy SM, et al.  (1993) Independent variation and positive selection in env V1 and V2 domains within maternal-infant strains of human immunodeficiency virus type 1 in vivo. J Virol  67: 3951–3960.  <ul class="find" data-citedArticleID="997238"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Independent+variation+and+positive+selection+in+env+V1+and+V2+domains+within+maternal-infant+strains+of+human+immunodeficiency+virus+type+1+in+vivo.&amp;auth=&amp;atitle=Independent+variation+and+positive+selection+in+env+V1+and+V2+domains+within+maternal-infant+strains+of+human+immunodeficiency+virus+type+1+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independent+variation+and+positive+selection+in+env+V1+and+V2+domains+within+maternal-infant+strains+of+human+immunodeficiency+virus+type+1+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independent+variation+and+positive+selection+in+env+V1+and+V2+domains+within+maternal-infant+strains+of+human+immunodeficiency+virus+type+1+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000950-Tuttle1" id="pone.0000950-Tuttle1"></a>Tuttle DL, Harrison JK, Anders C, Sleasman JW, Goodenow MM (1998) Expression of CCR5 increases during monocyte differentiation and directly mediates macrophage susceptibility to infection by human immunodeficiency virus type 1. J Virol  72: 4962–4969.  <ul class="find" data-citedArticleID="997302"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+of+CCR5+increases+during+monocyte+differentiation+and+directly+mediates+macrophage+susceptibility+to+infection+by+human+immunodeficiency+virus+type+1.&amp;auth=&amp;atitle=Expression+of+CCR5+increases+during+monocyte+differentiation+and+directly+mediates+macrophage+susceptibility+to+infection+by+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+CCR5+increases+during+monocyte+differentiation+and+directly+mediates+macrophage+susceptibility+to+infection+by+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+CCR5+increases+during+monocyte+differentiation+and+directly+mediates+macrophage+susceptibility+to+infection+by+human+immunodeficiency+virus+type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000950-Burkhardt1" id="pone.0000950-Burkhardt1"></a>Burkhardt B, Kephart D, Coberley C, Sleasman J, Goodenow M (1999) Postmortem isolation and analysis of human genomic DNA and HIV-infected tissues: a molecular diagnostics application. Promega Notes  71: 30–31.  <ul class="find" data-citedArticleID="997176"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Postmortem+isolation+and+analysis+of+human+genomic+DNA+and+HIV-infected+tissues%3A+a+molecular+diagnostics+application.&amp;auth=&amp;atitle=Postmortem+isolation+and+analysis+of+human+genomic+DNA+and+HIV-infected+tissues%3A+a+molecular+diagnostics+application." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Postmortem+isolation+and+analysis+of+human+genomic+DNA+and+HIV-infected+tissues%3A+a+molecular+diagnostics+application." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Postmortem+isolation+and+analysis+of+human+genomic+DNA+and+HIV-infected+tissues%3A+a+molecular+diagnostics+application.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000950-Tuttle2" id="pone.0000950-Tuttle2"></a>Tuttle DL, Anders CB, Aquino-De Jesus MJ, Poole PP, Lamers SL, et al.  (2002) Increased replication of non-syncytium-inducing HIV type 1 isolates in monocyte-derived macrophages is linked to advanced disease in infected children. AIDS Res Hum Retroviruses  20: 353–362.  <ul class="find" data-citedArticleID="997304" data-doi="10.1089/088922202753519133"><li><a href="http://dx.doi.org/10.1089/088922202753519133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+replication+of+non-syncytium-inducing+HIV+type+1+isolates+in+monocyte-derived+macrophages+is+linked+to+advanced+disease+in+infected+children." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+replication+of+non-syncytium-inducing+HIV+type+1+isolates+in+monocyte-derived+macrophages+is+linked+to+advanced+disease+in+infected+children.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000950-Lamers2" id="pone.0000950-Lamers2"></a>Lamers S, Beason S, Dunlap L, Compton R, Salemi M (2003) HIVbase: a PC/Windows-based software offering storage and querying power for locally held HIV-1 genetic, experimental and clinical data. Bioinformatics  20: 436–438.  <ul class="find" data-citedArticleID="997240" data-doi="10.1093/bioinformatics/btg445"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg445" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIVbase%3A+a+PC%2FWindows-based+software+offering+storage+and+querying+power+for+locally+held+HIV-1+genetic%2C+experimental+and+clinical+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIVbase%3A+a+PC%2FWindows-based+software+offering+storage+and+querying+power+for+locally+held+HIV-1+genetic%2C+experimental+and+clinical+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000950-Lamers3" id="pone.0000950-Lamers3"></a>Lamers SL, Sleasman JW, Goodenow MM (1996) A model for alignment of env V1 and V2 hypervariable domains from human and simian immunodeficiency viruses. AIDS Res Hum Retrovir  12: 1169–1178.  <ul class="find" data-citedArticleID="997242" data-doi="10.1089/aid.1996.12.1169"><li><a href="http://dx.doi.org/10.1089/aid.1996.12.1169" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+model+for+alignment+of+env+V1+and+V2+hypervariable+domains+from+human+and+simian+immunodeficiency+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+model+for+alignment+of+env+V1+and+V2+hypervariable+domains+from+human+and+simian+immunodeficiency+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000950-Goodenow1" id="pone.0000950-Goodenow1"></a>Goodenow MM, Collman RG (2006) HIV-1 coreceptor preference is distinct from target cell tropism: related but independent viral phenotypes. J Leuc Biol  80: 965–972.  <ul class="find" data-citedArticleID="997214" data-doi="10.1189/jlb.0306148"><li><a href="http://dx.doi.org/10.1189/jlb.0306148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+coreceptor+preference+is+distinct+from+target+cell+tropism%3A+related+but+independent+viral+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+coreceptor+preference+is+distinct+from+target+cell+tropism%3A+related+but+independent+viral+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000950-Bruen1" id="pone.0000950-Bruen1"></a>Bruen TC, Herve P, Bryant D (2006) A simple and robust statistical test for detecting the presence of recombination. Genetics  172: 2665–2681.  <ul class="find" data-citedArticleID="997174" data-doi="10.1534/genetics.105.048975"><li><a href="http://dx.doi.org/10.1534/genetics.105.048975" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+and+robust+statistical+test+for+detecting+the+presence+of+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+and+robust+statistical+test+for+detecting+the+presence+of+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000950-Huson1" id="pone.0000950-Huson1"></a>Huson DH, Bryant D (2005) Application of phylogenetic networks in evolutionary studies. Mol. Biol. Evol.  23: 254–267.  <ul class="find" data-citedArticleID="997222" data-doi="10.1093/molbev/msj030"><li><a href="http://dx.doi.org/10.1093/molbev/msj030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+phylogenetic+networks+in+evolutionary+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+phylogenetic+networks+in+evolutionary+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000950-Lole1" id="pone.0000950-Lole1"></a>Lole KS, Bollinger RC, Paranjape RS, Gadkari D, Kulkarni SS, et al.  (1999) Full-length human immunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination. J Virol.  73: 152–160.  <ul class="find" data-citedArticleID="997246"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Full-length+human+immunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination.&amp;auth=&amp;atitle=Full-length+human+immunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full-length+human+immunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full-length+human+immunodeficiency+virus+type+1+genomes+from+subtype+C-infected+seroconverters+in+India%2C+with+evidence+of+intersubtype+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000950-Salminen1" id="pone.0000950-Salminen1"></a>Salminen M (2003)  Detecting recombination in viral sequences. In: Salemi M, Vandamme AM, editors. The Phylogenetic Handbook-a practical approach to DNA and protein phylogeny. New York: Cambridge University Press. pp. 348–377.  <ul class="find-nolinks"></ul></li><li><span class="label">74.
              </span><a name="pone.0000950-Swofford1" id="pone.0000950-Swofford1"></a>Swofford D, Sullivan J (2003)  Phylogeny Inference based on parsimony and other methods with PAUP*. In: Salemi M, Vandamme AM, editors. The Phylogenetic Handbook-a practical approach to DNA and protein phylogeny. New York: Cambridge University Press. pp. 160–206.  <ul class="find-nolinks"></ul></li><li><span class="label">75.
              </span><a name="pone.0000950-Drummond1" id="pone.0000950-Drummond1"></a>Drummond AJ, Ho SYW, Phillips MJ, Rambaut A (2006) Relaxed Phylogenetics and Dating with Confidence. PLoS Biol  4: 1–12.  <ul class="find" data-citedArticleID="997204" data-doi="10.1371/journal.pbio.0040088"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relaxed+Phylogenetics+and+Dating+with+Confidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relaxed+Phylogenetics+and+Dating+with+Confidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000950-Yang1" id="pone.0000950-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Cabios  13: 555–556.  <ul class="find" data-citedArticleID="997310" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000950-Yang2" id="pone.0000950-Yang2"></a>Yang Z, Nielsen R, Goldman N, Pedersen AM (2000) Codon substitution models for heterogeneous selection pressure at amino acid sites. Genetics  155: 431–449.  <ul class="find" data-citedArticleID="997312"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon+substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.&amp;auth=&amp;atitle=Codon+substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000950-Slatikin1" id="pone.0000950-Slatikin1"></a>Slatikin M, Maddison WP (1989) A cladistic measure of gene flow inferred from the phylogenies of alleles. Genetics  123: 401–416.  <ul class="find" data-citedArticleID="997294"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles.&amp;auth=&amp;atitle=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000950-Fitch1" id="pone.0000950-Fitch1"></a>Fitch WM (1971) Toward defining the course of evolution: minimum change for a specific tree topology. Syst Zool  20: 401–416.  <ul class="find" data-citedArticleID="997208" data-doi="10.2307/2412116"><li><a href="http://dx.doi.org/10.2307/2412116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+defining+the+course+of+evolution%3A+minimum+change+for+a+specific+tree+topology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+defining+the+course+of+evolution%3A+minimum+change+for+a+specific+tree+topology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.XML" value="137832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.PDF" value="414602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g001.PNG_L" value="510555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g001.PNG_M" value="72889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g001.PNG_S" value="8903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g001.TIF" value="1112366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g001.PNG_I" value="27481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g002.PNG_L" value="1683121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g002.PNG_M" value="124103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g002.PNG_S" value="12666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g002.TIF" value="3379808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g002.PNG_I" value="63921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g003.PNG_L" value="780863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g003.PNG_M" value="116015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g003.PNG_S" value="10441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g003.TIF" value="1526240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g003.PNG_I" value="43451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g004.PNG_L" value="1643451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g004.PNG_M" value="108580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g004.PNG_S" value="8279"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g004.TIF" value="3094218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g004.PNG_I" value="37893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g005.PNG_L" value="439465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g005.PNG_M" value="81114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g005.PNG_S" value="9768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g005.TIF" value="1023050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.g005.PNG_I" value="28584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.s001.DOC" value="27136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.s002.TIF" value="241178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000950.s003.TIF" value="278493"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000950&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000950" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000950&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000950&volume=&issue=&title=Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies&author_name=Marco%20Salemi%2C%20Brant%20R.%20Burkhardt%2C%20Rebecca%20R.%20Gray%2C%20Guity%20Ghaffari%2C%20John%20W.%20Sleasman%2C%20Maureen%20M.%20Goodenow&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000950" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000950" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000950" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000950&amp;t=Phylodynamics%20of%20HIV-1%20in%20Lymphoid%20and%20Non-Lymphoid%20Tissues%20Reveals%20a%20Central%20Role%20for%20the%20Thymus%20in%20Emergence%20of%20CXCR4-Using%20Quasispecies" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000950&title=Phylodynamics%20of%20HIV-1%20in%20Lymphoid%20and%20Non-Lymphoid%20Tissues%20Reveals%20a%20Central%20Role%20for%20the%20Thymus%20in%20Emergence%20of%20CXCR4-Using%20Quasispecies&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000950&amp;title=Phylodynamics%20of%20HIV-1%20in%20Lymphoid%20and%20Non-Lymphoid%20Tissues%20Reveals%20a%20Central%20Role%20for%20the%20Thymus%20in%20Emergence%20of%20CXCR4-Using%20Quasispecies" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000950" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000950&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Phylodynamics of HIV-1 in Lymphoid and Non-Lymphoid Tissues Reveals a Central Role for the Thymus in Emergence of CXCR4-Using Quasispecies';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000950';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Phylodynamics%20of%20HIV-1%20in%20Lymphoid%20and%20Non-Lymphoid%20Tissues%20Reveals%20a%20Central%20Role%20for%20the%20Thymus%20in%20Emergence%20of%20CXCR4-Using%20Quasispecies http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000950" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000950" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">




















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Coreceptors%22" title="Search for articles in the subject area:'Coreceptors'"><div class="flagText">Coreceptors</div></a>
              <div data-categoryid="19997" data-articleid="24738"
                   data-categoryname="Coreceptors"
                   class="flagImage" title="Flag 'Coreceptors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="24738"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24738"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24738"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24738"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="24738"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Thymus%22" title="Search for articles in the subject area:'Thymus'"><div class="flagText">Thymus</div></a>
              <div data-categoryid="33757" data-articleid="24738"
                   data-categoryname="Thymus"
                   class="flagImage" title="Flag 'Thymus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+evolution%22" title="Search for articles in the subject area:'Viral evolution'"><div class="flagText">Viral evolution</div></a>
              <div data-categoryid="20461" data-articleid="24738"
                   data-categoryname="Viral evolution"
                   class="flagImage" title="Flag 'Viral evolution' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7048'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4773'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1881&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=8579">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
