

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001693"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001693" />

    <meta name="citation_title" content="Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli"/>
    <meta itemprop="name" content="Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli"/>

      <meta name="citation_author" content="Momiao Xiong"/>
            <meta name="citation_author_institution" content="Human Genetics Center, School of Public Health, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Frank C. Arnett"/>
            <meta name="citation_author_institution" content="Division of Rheumatology and Clinical Immunogenetics, Medical School, The University of Texas Medical School at Houston, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Xinjian Guo"/>
            <meta name="citation_author_institution" content="Division of Rheumatology and Clinical Immunogenetics, Medical School, The University of Texas Medical School at Houston, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Hao Xiong"/>
            <meta name="citation_author_institution" content="Department of Computer Science, Texas A&M University, College Station, Texas, United States of America"/>
      <meta name="citation_author" content="Xiaodong Zhou"/>
            <meta name="citation_author_institution" content="Division of Rheumatology and Clinical Immunogenetics, Medical School, The University of Texas Medical School at Houston, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001693.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1693"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=New challenges in gene expression data analysis and the extended GEPAS.; citation_author=J Herrero; citation_author=JM Vaquerizas; citation_author=F Al-Shahrour; citation_author=L Conde; citation_author=A Mateos; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=1; citation_pages=W485-91; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Dynamics of biological systems: role of systems biology in medical research.; citation_author=HE Assmus; citation_author=R Herwig; citation_author=K-H Kwang-Hyun Cho; citation_author=O Wolkenhauer; citation_journal_title=Expert Review of Molecular Diagnostics; citation_volume=6; citation_number=2; citation_pages=891-902; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Network biology: understanding the cell's functional organization.; citation_author=A-L Barabási; citation_author=ZN Oltvai; citation_journal_title=Nature Reviews Genetics; citation_volume=5; citation_number=3; citation_pages=101-113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The dynamic systems approach to control and regulation of intracellular networks.; citation_author=O Wolkenhauer; citation_author=M Ullah; citation_author=P Wellstead; citation_author=KH Cho; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=4; citation_pages=1846-53; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of genetic networks from a small number of gene expression patterns under the Boolean network model.; citation_author=T Akutsu; citation_author=S Miyano; citation_author=S Kuhara; citation_journal_title=Pacific Symposium on Biocomputing; citation_number=5; citation_pages=17-28; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Random Boolean network models and the yeast transcriptional network.; citation_author=S Kauffman; citation_author=C Peterson; citation_author=B Samuelsson; citation_author=C Troein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=6; citation_pages=14796-14799; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Boolean Dynamics of Genetic Regulatory Networks Inferred from Microarray Time Series Data.; citation_author=S Martin; citation_author=Z Zhang; citation_author=A Martino; citation_author=JL Faulon; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=7; citation_pages=866-874; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Modeling and simulation of genetic regulatory systems: a literature review.; citation_author=H de Jong; citation_journal_title=J Comput Biol; citation_volume=9; citation_number=8; citation_pages=67-103; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A Bayesian approach to reconstructing genetic regulatory networks with hidden factors.; citation_author=MJ Beal; citation_author=F Falciani; citation_author=Z Ghahramani; citation_author=C Rangel; citation_author=DL Wild; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=9; citation_pages=349-356; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Applying dynamic Bayesian networks to perturbed gene expression data.; citation_author=N Dojer; citation_author=A Gambin; citation_author=A Mizera; citation_author=B Wilczynski; citation_author=J Tiuryn; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=10; citation_pages=249; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reconstructing gene regulatory networks with bayesian networks by combining expression data with multiple sources of prior knowledge.; citation_author=A Werhli; citation_author=D Husmeier; citation_journal_title=Stat Appl Genet Mol Biol; citation_volume=6; citation_number=11; citation_pages=15; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Inferring quantitative models of regulatory networks from expression data.; citation_author=I Nachman; citation_author=A Regeve; citation_author=N Friedman; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=12; citation_pages=Suppl. 1248-1256; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=System dynamics. Third Edition.; citation_author=K Ogata; citation_number=13; citation_date=1998; citation_publisher=Prentice Hall; " />
      <meta name="citation_reference" content="citation_title=Modeling biological responses using gene expression profiling and linear dynamical systems. Proceedings of the 2nd International Conference on Systems Biology.; citation_author=C Rangel; citation_author=DL Wild; citation_author=F Falciani; citation_author=Z Ghahramani; citation_author=A Gaiba; citation_journal_title=Omipress; citation_number=14; citation_pages=248-256; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=maximum likelihood from incomplete data via the EM algorithm.; citation_author=A Dempster; citation_author=N Laird; citation_author=D Rubin; citation_journal_title=Journal of the Royal Statistical Society,; citation_volume=Seres B 39; citation_number=15; citation_pages=1-38; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Robust Maximum-Likelihood Estimation of Multivariable Dynamic Systems.; citation_author=S Gibson; citation_author=B Ninness; citation_journal_title=Automatica; citation_volume=41; citation_number=16; citation_pages=1667-1682; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Understanding through modeling – A historical perspective and review of biochemical system theory as a powerful tool for systems biology.; citation_author=EO Voit; citation_author=JH Schwacke; citation_number=17; citation_date=2007; citation_publisher=CRC Press; " />
      <meta name="citation_reference" content="citation_title=Dynamic properties of network motifs contribute to biological network organization.; citation_author=RJ Prill; citation_author=PA Iglesias; citation_author=A Levchenko; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=18; citation_pages=e343; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Robustness of cellular functions.; citation_author=J Stelling; citation_author=U Sauer; citation_author=Z Szallasi; citation_author=FJ Doyle; citation_author=J Doyle; citation_journal_title=Cell; citation_volume=118; citation_number=19; citation_pages=675-85; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Feedback stabilization of uncertain systems in the presence of a direct link.; citation_author=NC Martins; citation_author=MA Dahleh; citation_author=N Elia; citation_journal_title=IEEE Transactions on Automatic Control; citation_volume=51; citation_number=20; citation_pages=438-447; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Systems biology-An engineering perspective.; citation_author=A Kremling; citation_author=J Saez-Rodriguez; citation_journal_title=J Biotechnol; citation_volume=129; citation_number=21; citation_pages=329-351; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Endothelial and fibroblastic activation in scleroderma. The myth of the “uninvolved skin”.; citation_author=HN Claman; citation_author=RC Giorno; citation_author=JR Seibold; citation_journal_title=Arthritis Rheum; citation_volume=34; citation_number=22; citation_pages=1495-1501; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Clinical, histopathological, immunological and fibroblast studies in 30 patients with subcutaneous injections of modelants including silicone and mineral oils.; citation_author=AR Cabral; citation_author=J Alcocer-Varela; citation_author=R Orozco-Topete; citation_author=E Reyes; citation_author=L Fernandez-Dominguez; citation_journal_title=Rev Invest Clin; citation_volume=46; citation_number=23; citation_pages=257-66; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Lung collagen cross-links in rats with experimentally induced pulmonary fibrosis.; citation_author=JE Gerriets; citation_author=KM Reiser; citation_author=JA Last; citation_journal_title=Biochim Biophys Acta; citation_volume=1316; citation_number=24; citation_pages=121-31; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mechanism of silica-induced ROS generation in Rat2 fibroblast cells.; citation_author=KA Kim; citation_author=YH Kim; citation_author=M Seok Seo; citation_author=W Kyu Lee; citation_author=S Won Kim; citation_journal_title=Toxicol Lett; citation_volume=135; citation_number=25; citation_pages=185-91; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Effects of silica on human lung fibroblast in culture.; citation_author=G Arcangeli; citation_author=V Cupelli; citation_author=G Giuliano; citation_journal_title=Sci Total Environ; citation_volume=270; citation_number=26; citation_pages=135-9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Connecting quantitative regulatory-network models to the genome.; citation_author=Y Pan; citation_author=T Durfee; citation_author=J Bockhorst; citation_author=M Craven; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=27; citation_pages=1367-1376; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structural kinetic modeling of metabolic networks. Proc. Natl Acad. Sci.; citation_author=R Steuer; citation_author=T Gross; citation_author=J Selbig; citation_author=B Blasius; citation_journal_title=USA; citation_volume=103; citation_number=28; citation_pages=11868-11873; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=SPARC, an upstream regulator of CTGF in response to TGF-β stimulation.; citation_author=XD Zhou; citation_author=MM Xiong; citation_author=FK Tan; citation_author=XJ Guo; citation_author=FC Arnett; citation_journal_title=Arthritis & Rheum; citation_volume=54; citation_number=29; citation_pages=3885-3889; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A systems biology approach to genetic studies of complex diseases.; citation_author=M Xiong; citation_author=CA Feghali-Bostwick; citation_author=FC Arnett; citation_author=X Zhou; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=30; citation_pages=5325-5332; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Smad-dependent and Smad-independent pathways in TGF-beta family signalling.; citation_author=R Derynck; citation_author=YE Zhang; citation_journal_title=Nature; citation_volume=425; citation_number=31; citation_pages=577-584; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Potential regulatory elements of the constitutive up-regulated α2(I) collagen gene in scleroderma dermal fibroblasts.; citation_author=M Jinnin; citation_author=H Ihn; citation_author=Y Mimura; citation_author=Y Asano; citation_author=K Tamaki; citation_journal_title=Biochem Biophys Res Commun.; citation_volume=343; citation_number=32; citation_pages=904-909; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Co-ordinate increase in the expression of type I and type III collagen genes in progressive systemic sclerosis.; citation_author=SA Jimenez; citation_author=G Feldman; citation_author=RL Bashey; citation_author=R Bienkowski; citation_author=J Rosenbloom; citation_journal_title=Biochem J; citation_volume=237; citation_number=33; citation_pages=837-843; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Increased collagen synthesis by scleroderma skin fibroblasts in vitro: a possible defect in the regulation or activation of the scleroderma fibroblast.; citation_author=EC LeRoy; citation_journal_title=J Clin Invest; citation_volume=54; citation_number=34; citation_pages=880-889; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Stability of genetic regulatory networks with time delay.; citation_author=L Chen; citation_author=K Aihara; citation_journal_title=IEEE Transactions on Circuits and Systems-I: Fundamental Theory and Applications; citation_volume=49; citation_number=35; citation_pages=602-608; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Dynamic Properties of Network Motifs Contribute to Biological Network Organization.; citation_author=RJ Prill; citation_author=PA Iglesias; citation_author=A Levchenko; citation_journal_title=PLoS Biol; citation_volume=3(11); citation_number=36; citation_pages=e343; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dynamic stability of signal transduction networks depending on downstream and upstream specificity of protein kinases.; citation_author=B Binder; citation_author=R Heinrich; citation_journal_title=Mol Biol Rep; citation_volume=29; citation_number=37; citation_pages=51-55; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Systems biology-An engineering perspective.; citation_author=A Kremling; citation_author=J Saez-Rodriguez; citation_journal_title=J Biotechnol; citation_volume=129; citation_number=38; citation_pages=329-351; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A new measure of process output controllability.; citation_author=DR Vinson; citation_author=C Georgaki; citation_journal_title=Journal of Process Control; citation_volume=10; citation_number=39; citation_pages=185-194; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Feedback dynamics and cell function: Why systems biology is called Systems Biology.; citation_author=O Wolkenhauer; citation_author=M Mesarovic; citation_journal_title=Mol. Biosyst; citation_volume=1; citation_number=40; citation_pages=14-16; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Robustness and evolvability in genetic regulatory networks.; citation_author=M Aldana; citation_author=E Balleza; citation_author=S Kauffman; citation_author=O Resendiz; citation_journal_title=J Theor Biol; citation_volume=245; citation_number=41; citation_pages=433-448; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Preliminary criteria for the classification of systemic sclerosis (scleroderma).; citation_journal_title=Arthritis Rheum; citation_volume=23; citation_number=42; citation_pages=581-590; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Scleroderma (systemic sclerosis): classification, subsets and pathogenesis.; citation_author=EC LeRoy; citation_author=C Black; citation_author=R Fleischmajer; citation_author=S Jablonska; citation_author=T Krieg; citation_journal_title=J Rheumatol; citation_volume=15; citation_number=43; citation_pages=202-205; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Linear Estimation.; citation_author=T Kailath; citation_author=AH Sayed; citation_author=B Hassibi; citation_number=44; citation_date=1999; citation_publisher=Prentice Hall; " />
      <meta name="citation_reference" content="citation_title=Condition numbers and perturbation of the weighted Moore-Penrose inverse and weighted linear squares problem.; citation_author=Y Wei; citation_author=D Wang; citation_journal_title=Applied mathematics and Computation; citation_volume=145; citation_number=45; citation_pages=45-58; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001693" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli"/>
    <meta name="twitter:description" content="Diseases are believed to arise from dysregulation of biological systems (pathways) perturbed by environmental triggers. Biological systems as a whole are not just the sum of their components, rather ever-changing, complex and dynamic systems over time in response to internal and external perturbation. In the past, biologists have mainly focused on studying either functions of isolated genes or steady-states of small biological pathways. However, it is systems dynamics that play an essential role in giving rise to cellular function/dysfunction which cause diseases, such as growth, differentiation, division and apoptosis. Biological phenomena of the entire organism are not only determined by steady-state characteristics of the biological systems, but also by intrinsic dynamic properties of biological systems, including stability, transient-response, and controllability, which determine how the systems maintain their functions and performance under a broad range of random internal and external perturbations. As a proof of principle, we examine signal transduction pathways and genetic regulatory pathways as biological systems. We employ widely used state-space equations in systems science to model biological systems, and use expectation-maximization (EM) algorithms and Kalman filter to estimate the parameters in the models. We apply the developed state-space models to human fibroblasts obtained from the autoimmune fibrosing disease, scleroderma, and then perform dynamic analysis of partial TGF-&beta; pathway in both normal and scleroderma fibroblasts stimulated by silica. We find that TGF-&beta; pathway under perturbation of silica shows significant differences in dynamic properties between normal and scleroderma fibroblasts. Our findings may open a new avenue in exploring the functions of cells and mechanism operative in disease development."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001693.g006"/>

  <meta property="og:title" content="Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001693" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1406'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3632'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=607&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001693">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001693" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001693&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Momiao Xiong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genetics Center, School of Public Health, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frank C. Arnett, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Rheumatology and Clinical Immunogenetics, Medical School, The University of Texas Medical School at Houston, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xinjian Guo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Rheumatology and Clinical Immunogenetics, Medical School, The University of Texas Medical School at Houston, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hao Xiong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Computer Science, Texas A&M University, College Station, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaodong Zhou
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:xiaodong.zhou@uth.tmc.edu">xiaodong.zhou@uth.tmc.edu</a></p>

                <p>Affiliation:
                  Division of Rheumatology and Clinical Immunogenetics, Medical School, The University of Texas Medical School at Houston, The University of Texas Health Science Center at Houston (UTHSC-H), Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001693</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001693" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001693" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001693" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001693" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001693-g001" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001693-t001" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001693-g002" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001693-g003" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001693-g004" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001693-g005" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001693-g006" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001693">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001693" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Diseases are believed to arise from dysregulation of biological systems (pathways) perturbed by environmental triggers. Biological systems as a whole are not just the sum of their components, rather ever-changing, complex and dynamic systems over time in response to internal and external perturbation. In the past, biologists have mainly focused on studying either functions of isolated genes or steady-states of small biological pathways. However, it is systems dynamics that play an essential role in giving rise to cellular function/dysfunction which cause diseases, such as growth, differentiation, division and apoptosis. Biological phenomena of the entire organism are not only determined by steady-state characteristics of the biological systems, but also by intrinsic dynamic properties of biological systems, including stability, transient-response, and controllability, which determine how the systems maintain their functions and performance under a broad range of random internal and external perturbations. As a proof of principle, we examine signal transduction pathways and genetic regulatory pathways as biological systems. We employ widely used state-space equations in systems science to model biological systems, and use expectation-maximization (EM) algorithms and Kalman filter to estimate the parameters in the models. We apply the developed state-space models to human fibroblasts obtained from the autoimmune fibrosing disease, scleroderma, and then perform dynamic analysis of partial TGF-β pathway in both normal and scleroderma fibroblasts stimulated by silica. We find that TGF-β pathway under perturbation of silica shows significant differences in dynamic properties between normal and scleroderma fibroblasts. Our findings may open a new avenue in exploring the functions of cells and mechanism operative in disease development.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Xiong M, Arnett FC, Guo X, Xiong H, Zhou X (2008) Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli. PLoS ONE 3(2):
          e1693.
            doi:10.1371/journal.pone.0001693</p><p><strong>Academic Editor: </strong>Raya Khanin, University of Glasgow, Germany</p><p><strong>Received:</strong> May 2, 2007; <strong>Accepted:</strong> January 31, 2008; <strong>Published:</strong> February 27, 2008</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>Supported by UTHSC-H grants from NIH/NIAMS Center of Research Translation (CORT) in Scleroderma (1 P50 AR054144) (F.C.A) and 5RO3AR050517-02 (X.Z.), NIH/NCRR Center for Clinical and Translational Sciences (Houston CTSA Program) (1UL1 RR 024148), NIH PHS NCRR GCRC grant M01RR002558, University of Texas Health Science Center at Houston CreFF (X.Z.), Department of the Army, Medical Research Acquisition Activity: PR064803 (X.Z) and the Scleroderma Foundation (X.Z.).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Identifying differentially expressed genes across distinct conditions and clustering co-expressed genes into different functional groups have been general approaches for unraveling molecular mechanisms involved in disease pathogenesis <a href="#pone.0001693-Herrero1">[1]</a>. Although these approaches are valuable for looking at isolated events and their correlations, they do not explain the behavior of a bio-system.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Another approach to deciphering pathogenesis of complex diseases is system thinking. Human complex diseases are believed to arise from malfunction of a specific biological system, rather than from isolated events. It is increasingly recognized that biological systems as a whole are not just the sum of their components but, rather, ever-changing, complex, interacted and dynamic systems over time in response to internal events and environmental stimuli <a href="#pone.0001693-Assmus1">[2]</a>. Cellular functions, such as growth, differentiation, division and apoptosis, and biological phenomena of the entire organisms are not only determined by steady-state characteristics of the biological systems, but also determined by inherent dynamic properties of biological systems. Dynamic properties include stability, transient-response, observability and controllability, which determine how the systems maintain their functions and performance under a broad range of random internal and external perturbations. Similar to differential expression of genes between normal and abnormal tissues, we can also observe the differential dynamic properties of the biological systems across different types of tissues and conditions. Dynamic properties are correlated with the health status of individuals and are of central importance for comprehensively understanding human biological systems and ultimately complex diseases.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The dynamic behavior of most complex biological systems emerges from the orchestrated activity of many components (e.g. genes and proteins) that interact with each other to form complicated biological networks involving gene regulation and signal transduction <a href="#pone.0001693-Barabsi1">[3]</a>. The nodes and links together are referred to as networks. This report is a study of gene regulatory network that focuses on dynamic properties of a biological system.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Investigation of dynamic properties of gene networks has three major tasks: development of mathematic models, estimation of the parameters in the models, and dynamic analysis. Mathematical modeling is to use mathematical language to describe the dynamic characteristics of a system <a href="#pone.0001693-Wolkenhauer1">[4]</a>. In the past decade, various methods have been developed to model gene networks, including Boolean networks <a href="#pone.0001693-Akutsu1">[5]</a>–<a href="#pone.0001693-Martin1">[7]</a>, differential equations and Bayesian networks <a href="#pone.0001693-deJong1">[8]</a>–<a href="#pone.0001693-Werhli1">[11]</a>, and vector autoregressive model <a href="#pone.0001693-Nachman1">[12]</a>. A very powerful approach in modeling complex systems is the state-space approach <a href="#pone.0001693-Ogata1">[13]</a>, <a href="#pone.0001693-Rangel1">[14]</a>, which is a special subclass of dynamic Bayesian networks. It provides a general framework for the application of dynamic systems theory in the analysis of gene regulation. The state-space approach is the core of modern systems theory. Application of the state-space equations to modeling gene networks allows us to use a large body of methodologies and tools in dynamic systems theory for studying dynamics of gene networks. We use Kalman Filter and Expectation-Maximization (EM) to estimate the parameters in the model <a href="#pone.0001693-Dempster1">[15]</a>, <a href="#pone.0001693-Gibson1">[16]</a>. After state-space model of the gene networks is established, the next task is to perform dynamic analysis for the model in response to perturbation of internal and external stimuli. Dynamic analysis attempts to extract inherent features of the systems that capture and describe the behaviors of the system over time under different operating conditions. The most important operating principle of a dynamic system is its stability (i. e., the ability to return to the original state or equilibrium state after perturbation). The concept of stability can be easily illustrated by the example of a marble sitting at the bowl. When the marble is in the bottom of the bowl it is stable. No matter where the marble is pushed, up the side of the bowl or from the bottom of the bowl, after it is released, the marble will finally settle to the bottom of the bowl at the original, stable equilibrium point. However, when the marble is on the top of an inverted bowl, it is unstable. The marble can remain on the top of the bowl only when the forces acting on the marble on the top of the bowl is completely balanced. Any slight perturbation in the marble's steady state will destroy the balance of the marble and cause it to move away from the top of the bowl. This indicates that when the system is in unstable state small perturbation can cause the system move away from the steady-state <a href="#pone.0001693-Voit1">[17]</a>. The biological systems are in constant change under the influences of genetic and environmental differences. The ability of the systems to maintain the stable states after perturbation and to resist diverse disturbance of the internal and external forces is critical to the viability of living organisms and plays a central role in biology <a href="#pone.0001693-Prill1">[18]</a>, <a href="#pone.0001693-Stelling1">[19]</a>. Consequently, studying stability of biological systems is of great importance for discovering mechanism of complex diseases. Although there has been long history to investigate the stability of biologic systems, to our knowledge, very few studies have been reported on stability of gene networks. Particularly, the relationship between stability of gene networks and status of diseases has not been explored. One of purpose of this paper is to use gene expression data to show that similar to the example of the marble in the bowl, the gene networks will also have stable and unstable states and that unstable gene networks may be associated with the diseases.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Another important property of the dynamic systems is the transient response to disturbance of internal noises and external environmental forces, which measures how fast the systems respond to the perturbation and characterizes damping and oscillation properties of the process in response to the perturbation <a href="#pone.0001693-Ogata1">[13]</a>. Feedback close loops are the basis for maintaining normal function of cells and organisms in the face of internal and external perturbation <a href="#pone.0001693-Stelling1">[19]</a>, <a href="#pone.0001693-Martins1">[20]</a>. The essential feature of the transient response of a feedback closed-loop system largely depends on the location of the closed-loop poles. A simple and popular method for searching the poles of the closed-loop system is the root-locus analysis that plots a curve of the location of the poles of a transfer function of the feedback system over the range of the variable (usually loop gain) to determine whether the system will become unstable or oscillate <a href="#pone.0001693-Ogata1">[13]</a>. The third important property of a dynamic system is controllability. Controllability is defined as the capacity of the system to move from undesired states to certain desired final states in finite time through accessible inputs <a href="#pone.0001693-Kremling1">[21]</a>. Germline or somatic mutations lead to the subsequent transcriptional and translational alterations which will affect the phenotypes of the cells and cause diseases. Therapeutic interventions such as radiation, drug and gene therapy intend to alter gene expressions from an undesired state or abnormal state to a desired or normal state. Theoretic and practical analyses in modern control theory demonstrate that there exist systems which we are not able to change from undesired states to desired states. Now the question arises: are all genetic networks controllable? Can always therapeutic interventions change levels of gene expressions to desired states? Controllability provides answers to these questions. It provides a convenient and sufficient criterion for assessing whether we can change undesired gene expression levels to desired gene expression levels. Controllability describes the ability of biological systems to adapt to the changes of environments and deeply characterizes the internal structure of the system. The controllability of the biological networks may reflect the severity of the disease. Thus, the controllability is a fundamental design principle of biological system.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In summary, stability, transient response, feedback and controllability are basic dynamic properties of the biological systems and are essential to the function of the cells and organisms. As a proof of principle, in this report we investigate the differential dynamic properties of TGF-β pathway in response to perturbation of silica between normal and scleroderma fibroblasts. Scleroderma or systemic sclerosis (SSc) is a typical complex disease in which fibrosis occurs in multiple organs. Although etiopathogenesis is unknown, both genetic and environmental factors are believed to play critical roles. The major source of fibrosis in SSc is over production of collagens from fibroblasts. Fibroblasts obtained from SSc patients appeared to be genetically engineered to produce more collagens and cytokines <a href="#pone.0001693-Claman1">[22]</a>. Silica exposure is an important environmental risk factor in some cases, which has been found in association with the development and perturbation of SSc <a href="#pone.0001693-Cabral1">[23]</a>. Subcutaneous injections of silica have been reported to induce sclerodermatous skin changes and activation of skin fibroblasts <a href="#pone.0001693-Cabral1">[23]</a>. Therefore, interactions between fibroblasts and silica may represent a magnification of biological events occurring in SSc and/or SSc-like disorders. The biological system of fibroblasts reacting to silica exposure must involve complex regulations and coordination of molecules to maintain their desirable status. Although multiple experiments of the <em>in vivo</em> and the <em>in vitro</em> response to silica particles have revealed that fibroblasts are activated to produce more collagens and other extracellular matrix (ECM) components <a href="#pone.0001693-Gerriets1">[24]</a>–<a href="#pone.0001693-Arcangeli1">[26]</a>, there is a lack of a mathematical model to quantify interactions among the molecules, and to predict dynamic behaviors of this bio-system. The purpose of this report is to use gene expression responses of scleroderma and normal fibroblasts exposed to the stimulus of silica as an example to address the issue of differential dynamic properties of the biological systems in response to perturbation by environments across different conditions. To accomplish this, we first formulate a regulatory network involving TGFBRII, CTGF, SPARC, COL1A2, COL3A1 and TIMP3 as a biological system that is associated with TGF-β signaling, and then apply mathematical methods and computational algorithms from engineering and control theory <a href="#pone.0001693-Ogata1">[13]</a> to perform dynamic analysis of this network for both normal and scleroderma fibroblasts in response to perturbation of environmental Stimuli. Based on the results of dynamic network analysis, we examine the differential dynamic properties of this network between normal and scleroderma fibroblasts and reveal the relationship between the dynamic properties of gene networks and the phenotypes of the cells.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>State Space Model of a gene network responding to silica</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Gene regulation involves a large number of biochemical events. Although kinetic models can be developed for gene regulation <a href="#pone.0001693-Nachman1">[12]</a>, <a href="#pone.0001693-Pan1">[27]</a>, <a href="#pone.0001693-Steuer1">[28]</a>, they involve many kinetic parameters that are difficult to be estimated from gene expression data with small number of samples. An alternative model of gene expression is a state-space model. It can effectively deal with time invariant or time varying, linear or nonlinear complex systems with multiple inputs and outputs. A state-space model includes three types of variables: input variables, output variables and state variables. A key idea behind state-space model is the concept of the state. The state of a dynamic regulatory system is the smallest set of variables which are referred to as state variables such that the current knowledge of these variables together with the current and future knowledge of the input variables (environments or controls) will completely determine the behavior of the regulatory system. All state variables are hypothetical variables. State variables represent biological forces to regulate transcription of genes, which describe the behavior of gene transcription. Since the mechanisms of gene regulation in the network have still not been well understood, the state variables that determine the regulation may be unknown and hidden in the regulatory process, the concept of state variables is very suitable for description of the regulatory process. The expression levels of genes are output variables and can be observed. The expression levels of the genes are determined by the state variables, which describe states of regulation of the gene expressions.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Previously, we found that the SPARC (secreted protein, acidic, and rich in cysteine) gene is involved in the regulation of extracellular matrix genes such as <em>COL1A2</em>, <em>COL3A1</em>, <em>CTGF</em> and <em>TIMP3</em>, and this regulation is associated with activation of the TGF-β pathway <a href="#pone.0001693-Zhou1">[29]</a>, <a href="#pone.0001693-Xiong1">[30]</a>. We used this partial TGF-β pathway as an example to illustrate how to perform dynamic analysis of biological networks. This regulatory network was modeled by linear state-space equations defined as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>x<sub>k</sub></em> is the vector of state variables that describes the behavior of gene regulation, but are hidden; <em>y<sub>k</sub></em> is the output vector whose elements denote the measured gene expression levels; <em>u<sub>k</sub></em> is the input vector; <em>w</em> and <em>v</em> are noises assumed to be white Gaussian noise with zero means and variance <em>Q</em> and <em>R</em> respectively, and they are independent of each other. The inputs can be any external stimuli that influence gene regulation, things like environmental forces, drugs, proteins, RNAs, or the effects from the genes outside the model. Matrix <em>A</em> is called state transition matrix whose elements denote the regulatory strength of one gene on another, <em>B</em> is input to state matrix whose elements quantify the regulatory effects of the input variables on the genes of the network, <em>C</em> is state to output matrix whose elements quantify the dependence of the measured gene expression levels on the hidden regulatory states, and <em>D</em> is input to output matrix whose elements measures the strength of dependence of the observed gene expression levels on the inputs. Matrices <em>A</em>, <em>B</em>, <em>C</em>, <em>D</em> and variance matrices <em>Q</em> and <em>R</em> together make up the parameters of the dynamical system for gene regulatory networks.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We performed experiments on cultured human fibroblasts. We have 5 pairs each of normal and SSc patients' samples. For each sample, we have two replications were perturbed by Silicon. The transcript levels of six genes: <em>SPARC</em>, <em>CTGF</em>, <em>COL1A2</em>, <em>COL3A1</em>, <em>TIMP3</em> and <em>TGFBRII were</em> measured daily from 1 to 5 days. Let <em>x<sub>1</sub></em>, <em>x<sub>2</sub></em>, <em>x<sub>3</sub></em>, <em>x<sub>4</sub></em> and <em>x<sub>5</sub></em> be the expression levels of the <em>SPARC</em>, <em>TIMP3</em>, <em>COL3A1</em>, <em>CTGF</em> and <em>COL1A2</em>, respectively. Let <em>u<sub>1</sub></em> and <em>u<sub>2</sub></em> be the expression of the <em>TGFBRII</em> <a href="#pone.0001693-Derynck1">[31]</a> and 10 µg silica. The estimated state-space model for the normal cell line and SSc are respectively, given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e002&amp;representation=PNG" class="inline-graphic"></span><br>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e003&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>A graph will be used to represent a genetic network. The nodes in the graph will represent the variables that correspond to the expressions of the genes. The edge between two nodes denotes that two variables are dependent. The number next to edges is the elements of the parameter matrices A, B, C and D in the state-space model. The estimated state-space model is shown in <a href="#pone-0001693-g001">Figure 1</a> where the numbers next to the edges are the coefficients in the above equation for the normal (black color) and SSc (red color) fibroblasts, respectively. We observe differential regulation of <em>SPRAC</em> on <em>CTGF</em>, <em>COL3A1</em> and <em>COL1A2</em>, and <em>CTGF</em> on <em>COL3A1</em> between the SSc and normal fibroblasts. <a href="#pone-0001693-g001">Figure 1</a> and above equations also demonstrate that the effects of silica (environmental factor) on <em>TIMP3</em>, <em>COL3A1</em> and <em>COL1A2</em> between the SSc and normal fibroblasts are different. Their coefficients in the state-space equations for the normal fibroblasts are negative, but become positive for the SSc fibroblasts. This implies that perturbation of scleroderma fibroblasts by silica will increase the expressions of <em>COL1A2</em> and <em>COL3A1</em>. This statement can be supported by early observation that excessive amounts of various collagens mainly type I and type III collagens were generated in the fibroblasts from affected scleroderma skin <a href="#pone.0001693-Jinnin1">[32]</a>–<a href="#pone.0001693-LeRoy1">[34]</a>.</p>
<div class="figure" id="pone-0001693-g001"><div class="img"><a name="pone-0001693-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>State-space model for the regulatory gene network responding to silica stimulation in cultured human fibroblasts.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The numbers next to the edges are the coefficients in the state-space equations for the normal (black color) and SSc (red color) fibroblasts, respectively. The numbers in the boxes denote the mean expression values of the genes in normal (black color) and SSc (red color) fibroblasts.</p>
<span>doi:10.1371/journal.pone.0001693.g001</span></div>

<h4>Stability</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The most important dynamic property of gene regulatory networks is concerned with stability. Stability is an organizing principle of gene regulatory networks <a href="#pone.0001693-Chen1">[35]</a>–<a href="#pone.0001693-Binder1">[37]</a>. When gene regulatory networks are perturbed, the expressions of the genes in the network will be changed in response to perturbation of environments. There are two possibilities. One possibility is that although the expressions of the genes will be changed after perturbation of environments, they will finally return to their original values or stay at other equilibrium values forever. In this case, regulatory networks will maintain their steady states under perturbation of environments and hence will function normally. Another possibility is that the expressions of the genes after perturbation of the environments will diverge from their original states and never stay at any steady states, which will finally lead to damage and malfunction of the gene regulatory network. Formally, a dynamic system is called stable if their state variables return to, or towards their original states or equilibrium states following internal and external perturbations <a href="#pone.0001693-Kremling2">[38]</a>. The stability of the system is a property of the system itself. One of the methods for assessing the stability of the linear dynamic systems is to analyze eigenvalues of the state transition matrix <em>A</em> of the linear dynamic systems. For a discrete linear system, if the norm of all eigenvalues of the transition matrix <em>A</em> is less than 1 then the system is stable.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The eigenvalues of the transition matrix <em>A</em> of the state-space model for silica responding gene network for normal and SSc fibroblasts are given in <a href="#pone-0001693-t001">Table 1</a>. It indicates that all eigenvalues of the transition matrix <em>A</em> for the normal fibroblasts are less than 1, but for SSc fibroblasts, three eigenvalues whose absolute values are larger than 1. Therefore, the examined network for normal fibroblasts after perturbation of silica stimulation are relatively stable, but for SSc fibroblasts are unstable. Unstable gene regulatory or signal transduction networks will lead to erratic changes and malfunction of the whole biological system, which may be the case in the SSc fibroblasts that are associated with dramatic and irregular changes of <em>COL1A2</em> and <em>COL3A1</em>.</p>
<div class="figure" id="pone-0001693-t001"><div class="img"><a name="pone-0001693-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001693.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001693.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Eigenvalues of the transition matrix A of the state-space model for the genes in a regulatory network responding to silica in cultured human normal and SSc fibroblasts.</span></strong></p><span>doi:10.1371/journal.pone.0001693.t001</span></div>

<h4>Transient-Response Analysis of Genetic Networks</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The dynamic behavior of a system is encoded in the temporal evolution of its states. Cell functions are essentially temporary processes and largely determined by the dynamic properties of the biological systems in the cells. Transient and steady state responses are two steps of the response of a gene network to perturbation of external environments. The transient response of the gene network to perturbation of environments is defined as rapid changes of the expressions of the genes in the network over time which go from their initial states to final states after perturbation of external input <a href="#pone.0001693-Ogata1">[13]</a>. Steady-state response studies the system behavior at infinite time. The transient response of the dynamic systems is also a property of the system itself. The transient response of the gene network to environmental changes characterizes the dynamical process of the gene regulation networks in response to perturbation of environments. It can be used to study damped vibration behavior of the gene network and reveal how fast the gene networks respond to perturbation of environments and how accurately the networks can finally achieve the designed steady-state values. In the previous section we studied stability of the whole gene network, but we did not investigate the stability of the expression of the individual gene in the network. Since the transient response analysis of the gene network will study the dynamic process of the expression of the individual gene, it can be used to assess whether the expression of individual gene after perturbation of environment is stable or unstable. Although many transient response analyses is concerned with delay time, rise time, peak time, maximum overshoot and setting time, in this report, our transient response analysis mainly focuses on investigation of the stability, divergence or oscillation of individual gene expression.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Popular methods for investigation of the transient responses of the dynamic systems are to study the time domain characteristics of the system under perturbation of the external signals. The transient response of the dynamic system depends on the input signal. Different input signal will lead to the different transient response of the system. There are numerous types of signal in practice. For the convenience of analysis and comparison, we consider two types of signals: (i) unit-step signal and (ii) unit-impulse signal as shown in the <a href="#pone-0001693-g002">Figure 2</a>.</p>
<div class="figure" id="pone-0001693-g002"><div class="img"><a name="pone-0001693-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Unit step signal and impulse signal.</span></strong></p><span>doi:10.1371/journal.pone.0001693.g002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>For discrete dynamic systems, the transient response of the system is obtained by using the inverse <em>z</em> transform method <a href="#pone.0001693-Ogata1">[13]</a>. To investigate the transient response of the genes in the network responding to silica, the silica was taken as input signal. <a href="#pone-0001693-g003">Figures 3A and 3B</a> show transient response of genes to a unit step perturbation of silica for normal and SSc fibroblasts, respectively. <a href="#pone-0001693-g003">Figures 3C and 3D</a> show transient response of genes to an impulse perturbation of silica for normal and SSc fibroblasts, respectively. <a href="#pone-0001693-g003">Figures 3A, 3B, 3C and 3D</a> demonstrate that the transient response of <em>SPARC</em>, <em>TIMP3</em>, <em>CTGF</em> to the perturbation of silica between the normal and SSc fibroblasts are similar, but the transient response of <em>COL1A2</em> and <em>COL3A1</em> to the perturbation of the silica between the normal and the SSc fibroblasts were dramatically different. The expressions of <em>COL1A2</em> and <em>COL3A1</em> after perturbation of the silica in normal fibroblasts quickly reach the steady states. However, the expressions of <em>COL1A2</em> and <em>COL3A1</em> in the SSc fibroblasts after perturbation of silica were unstable and will never reach the steady-state values. This phenomenon suggests that dynamic responses of the expressions of <em>COL1A2</em> and <em>COL3A1</em> in the SSc fibroblasts to environmental stimuli are irregular.</p>
<div class="figure" id="pone-0001693-g003"><div class="img"><a name="pone-0001693-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Step response and impulse response of the genes to perturbation of silica in cultured fibroblasts.</span></strong></p><span>doi:10.1371/journal.pone.0001693.g003</span></div>

<h4>Root-Locus Analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The performance of the gene networks under the design for stability, time response and reliability can be studied by analysis of their corresponding closed-loop system. The basic features of the stability and transient response of the closed-loop system are largely determined by the location of the closed-loop poles, which in turn is related to the value of the loop gain <a href="#pone.0001693-Ogata1">[13]</a>. The roots of the characteristic equation of a system which is derived from the denominator of transfer function of the closed-loop system are the system's closed-loop poles. In general, the poles are complex variable and can be represented on the complex plane which is called s-plane. (Negative real) poles on the left hand side of the complex plane cause the response to decrease, while poles on the right hand side cause it to increase. Consequently, if the poles of the closed-loop system lie in the left half s plane, the system is stable. If any of these poles lies in the right-hand side of the s-plane, then the system is unstable. In this case, with increasing time, the transient response of the system will increase monotonically or oscillate with increasing magnitude <a href="#pone.0001693-Ogata1">[13]</a>. As the loop gain changes the location of the closed-loop, poles will also changes. A root locus is defined as the locus of the poles of a transfer function of a closed-loop as a specific parameter (generally, loop gain) is varied from zero to infinity. The locus of the poles will be plotted on the complex plane (s-plane) as the system gain is varied on some interval. Since the location of the poles will change as the gain changes a system that is stable for gain K<sub>1</sub> may become unstable for a different gain K<sub>2</sub>. We often observe that the root-locus will move from the left-hand of the s-plane to the right-hand of the s-plane, which implies that stable system becomes unstable as the system gain changes from one region to another region.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The root-locus plots the locations of the poles of the closed-loop single input and single output system (SISO) as the system gain varies. We use the symbol “x” to denote the poles of the closed-loop SISO and the symbol circle “o” to denote the zeros of the open-loop SISO. If the pole and zero coincide then the symbol : will be used to represent this situation. To study the dynamic behavior of the five genes to respond to the perturbation of silica, we consider the SISO system which takes one of the five genes as the output and silica as the input.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p><a href="#pone-0001693-g004">Figures 4A, 4B, 4C, 4D and 4E</a>, and <a href="#pone-0001693-g005">Figures 5A, 5B, 5C, 5D and 5E</a> show the root-locus plot of <em>SPARC</em>, <em>TIMP3</em>, <em>COL3A1</em>, <em>CTGF</em>, and <em>COL1A2</em> with silica as the input in the normal and SSc fibroblasts, respectively. We noted that three remarkable features emerged from two panels of the Figures. First, all poles of the closed-loop SISO systems for five genes in the normal cell lines lie in the left hand side of the s-plane, but their corresponding poles in the SSc fibroblasts lie in the right hand side of the s-plane. This indicates that the expressions of all five genes to respond to the disturbance of the environmental silica in normal fibroblasts are stable, but become unstable in the SSc fibroblasts, which confirms the previous stability assessment. Second, although all poles of the closed-loop SISO for five genes in the normal fibroblasts are on the left hand side of the s-plane, the <em>SPARC</em>, <em>COL3A1, CTGF and COL1A2</em>, each has at least one branch of the root-locus plot which will enter the right-hand side of the s-plane. This indicates that the system becomes unstable as the increasing system gain reaches the some range. This may imply that the regulations of these four genes are sensitive to the changes of the system. Third, the poles and zeros of the SISO on the right hand sides of the s-plane for the <em>SPARC</em> and <em>TIMP3</em> in the SSc fibroblasts have the same location, i.e., the poles and zeros are cancelled out. This shows that the expressions of <em>CTGF</em>, <em>COL1A2</em> and <em>COL3A1</em> in response to the perturbation of the silica are more unstable than that of <em>SPARC</em> and <em>TIMP3</em>. Differential regulations of <em>CTGF</em>, <em>COL1A2</em> and <em>COL3A1</em> in response to the perturbation of silica between the normal and SSc fibroblasts may imply that the interactions of these three genes with the silica may be involved in the pathogenesis of the SSc. Forth, these Figures also demonstrate that when normal fibroblasts were changed to SSc fibroblasts, the root-locus will be moved toward the right-half s-plane. Classical control theory indicates that moving of the root-locus toward the right-half s-plane will reduce stability and increase response time of the system.</p>
<div class="figure" id="pone-0001693-g004"><div class="img"><a name="pone-0001693-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Root-locus of gene expression in normal fibroblasts.</span></strong></p><span>doi:10.1371/journal.pone.0001693.g004</span></div><div class="figure" id="pone-0001693-g005"><div class="img"><a name="pone-0001693-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Root-locus of gene expression in SSc fibroblasts.</span></strong></p><span>doi:10.1371/journal.pone.0001693.g005</span></div>

<h4>Controllability</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Changes in expression levels of genes and proteins in the regulatory networks will lead to status transition of the cells from normal cells to abnormal cells. One way to correct molecular changes is to transform cells from an undesirable state to a desirable one by altering gene or protein expressions. Now the question is whether we can use potentially therapeutic interventions to change gene or protein expressions from undesired states to desired states. This important issue can be addressed by examining the controllability of gene regulatory networks.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The fundamental controllability in gene regulation is associated with two questions. The first question is whether an input (therapy) can be found such that the system states can be driven from the undesired initial value to the desired values in a given time interval. The second question is how difficult it may be to change the system from undesired states to the desired states if the system is controllable.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The system (regulatory network) is called controllable, if for any state of the system, there exists a finite time and an admissible control function such that the system can achieve the desired state transition. In other words, the state controllability indicates that we can find an input to change the states from any initial value to any final value within some finite time. The controllability provides a binary information about whether the system is controllable or not, but it does not provide a quantitative measure to quantify the amount of control effort needed to accomplish the control task. It has been recognized that to get into insides of controllability of the system it is indispensible to define a quantity to measure how the system is controllable. In other words, we need to develop a measure to evaluate the amount of control efforts required to change the system from the initial state to the desired state <a href="#pone.0001693-Vinson1">[39]</a>. The test for controllability is that the controllability matrix (see <a href="#s4">methods</a>) has full rank, i.e., the rank of the controllability matrix is equal to the number of the state variables of the system. To assess how difficult to achieve control goal, we calculated the conditional number of the controllability matrix which measures the degree of difficulty to change the state of the system (or gene expression in our problem) by the external forces such as treatments. The larger the conditional number of controllability matrix, the more control efforts required to accomplish control task.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>The rank of controllability matrix of the system in the state-space model of this partial TGF-β pathway under perturbation of silica in both normal and SSc fibroblasts is equal to 5 which is the number of the state variables in the model. Thus, TGF-β pathway is controllable in both normal and SSc fibroblasts. However, the conditional numbers of the controllability matrix of the system for normal and SSc fibroblasts were 80 and 398, respectively, which showed that the conditional number of the controllability matrix for the SSc fibroblasts is much larger than that for normal fibroblasts. Therefore, much more control efforts are required to change gene expressions to desired levels for the SSc fibroblasts than that for the normal fibroblasts. This implies that the controllability of this network between the normal and SSc fibroblasts are differentiable.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In the past, large efforts have been devoted to studying the function of individual genes and static properties of biological pathways. However, the molecular concentrations and activities in living organisms are in constant change as a result of their interactions. The pathogenesis of disease involves evolution and temporal process. The functions of the cells, tissues and entire organisms are not only due to the steady states of the biological pathways, but also due to the dynamic interactions of biological pathways with the external environments. Although investigation of the function of individual genes, proteins and steady-states of the biological pathways is still valuable, it is time to devote more efforts and resources to study dynamic behaviors and properties of the biological pathways. It is dynamic properties that play a central role in giving rise to the function of cells and organisms <a href="#pone.0001693-Wolkenhauer2">[40]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To exemplify this principle, we studied the differential dynamic properties of a partial TGF-β signaling network under perturbation of silica between normal and SSc fibroblasts. We took this network as a dynamic system and performed dynamic system analysis. Investigation of differential dynamics of this network between the normal and SSc fibroblasts consisted of three steps. The first step was to use the EM algorithm and Kalman filter to estimate the parameters in the state-space model of this TGF-β signaling network. The second step was to study stability, the transient response and controllability of the system, and to perform root-locus analysis based on the identified state-space model of the gene network. The third step was to assess whether the dynamic properties of this network between the normal and SSc fibroblasts were different.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our study in dynamic analysis of these gene regulations addressed several remarkable issues. First, the stability analysis may be used as a powerful tool for identifying biological pathways that are associated with the diseases. The stability is one of systems-level principles underlying complex biological pathways <a href="#pone.0001693-Aldana1">[41]</a>. The stability of the system is the ability of the system to return to the equilibrium states after perturbation of the internal and external stimuli. The requirements for stable biological pathways are necessary conditions for the normal operations of the cells and organisms. The unstable biological pathway will inevitably lead to the malfunction of cells or even death of the living organism. Our results showed that a gene network in responding to perturbation of silica is relatively stable in the normal fibroblasts, but unstable in the SSc fibroblasts. This assessment of differential stability of biological pathway between normal and abnormal cells represents a novel approach in study associations of biological pathways with human diseases.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Second, root-locus analysis can provide valuable information for finding genes that show strong differential dynamics between normal and abnormal cells. Not all genes in the unstable pathway show unstable dynamics. Expressions of some genes in the unstable pathway may be stable themselves. Our task is to distinguish the genes that show stable expressions from those show unstable expressions in the unstable pathway. The state transition matrix of the state-space model of the studied gene network in the SSc fibroblasts has three poles that were in the right hand sides of the complex s-plane (<a href="#pone-0001693-g003">Figures 3</a> and <a href="#pone-0001693-g004">4</a>), which implies that this network in the SSc fibroblasts is unstable. The zeros of the genes of SPARC and TIMP3 in SISO system coincided with three poles. Therefore, although this gene network was unstable in the SSc fibroblasts, the expressions of the genes of SPARC and TIMP3 were stable. At least one branch of the root locus plots of other three genes (<em>CTGF</em>, <em>COL1A2</em> and <em>COL3A1</em>) were on the right hand sides of the s-plane. This indicates that the responses of the genes of CTGF, COL1A2 and COL3A1 to the perturbation of silica in the SSc fibroblasts were unstable no matter how the system gains were changed. These findings can be confirmed by the transient response analysis of the genes. The poles and zeros of characteristic equations of the SISO systems of the genes in response to the perturbation of internal and external signals are intrinsic properties of the gene regulations and are largely not affected by the expressions of other genes. Unlike the concept of differential expressions of the genes where the differentially expressed genes may be just consequences of differential expressions of other genes lying up in the pathway, the differential stability of the response of the genes to the perturbation of the signal between normal and abnormal tissues may be involved in the pathogenesis of the diseases. Therefore, the genes showing differential stability are supposed to be associated with the diseases. The root-locus analysis and the transient response will provide new tools for identifying the genes that are associated with the diseases. The differential stability and the transient response of the gene in the response to perturbation of the environment between the normal and abnormal cells characterize the interaction between the genes and environments. Therefore, the root-locus analysis and the transient response analysis also provide a powerful tool for detection of the gene-environment interaction.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Third, the controllability of biological pathway is an important property of the system. Germline or somatic mutations lead to the subsequent transcriptional and translational alterations which will finally cause diseases. Therapeutic interventions such as radiation, drug and gene therapy are intended to alter gene expressions from an undesired state to a desired or normal state. Gene regulation is a complex biological system. Theoretic and practical analyses in modern control theory demonstrate that there exist systems which we are not able (or find difficult) to change from undesired states to desired states of gene regulation. Now the question arises: are all biological pathways controllable? Are degrees of controllability of the biological pathways different between normal and abnormal Cells? The controllability measures the ability to move a system around in its entire state space using certain admissible intervention. In this report, we developed a conditional number of controllability matrix, to measure the degree of controllability of the system. Our results show that although a gene expression network responding to silica in both normal and SSc fibroblasts is controllable, the degree of controllability of this regulatory network between the normal and SSc fibroblasts is different. This regulatory network in the SSc fibroblasts has a low degree of controllability. In other words, adjustment of regulation of genes in the network by external intervention in the SSc fibroblasts is more difficult than that in the normal fibroblasts. We suspect that the degree of controllability is correlated with the severity of the diseases. When the diseases are at the initial stages, the biological systems are easy to move from abnormal states to the normal states. The degree of controllability of the system will provide valuable information on the curability of the diseases. Although in the past a number of authors have studied dynamic properties of biological networks, their studies have mainly used kinetic data or artificial data and nonlinear models <a href="#pone.0001693-Steuer1">[28]</a>, <a href="#pone.0001693-Chen1">[35]</a>–<a href="#pone.0001693-Binder1">[37]</a>. Due to limitation of experiments, many kinetic parameters in the genetic regulation have not been available in practice. Large-scale kinetic analysis of biological networks is infeasible. Here we use gene expressions and linear models to study dynamic properties of genetic networks. The results of this report showed that the dynamic properties of genetic network between normal and abnormal cells were differential.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In summary, dynamic properties of the biological systems are intrinsic system properties. The gene expressions are the phenotype of the cells. Their changes are governed by the hidden dynamic properties of the gene regulatory systems. It is dynamic properties that determine the phenotypes of the cells. This report represents a paradigm shift from the studies of individual components and static properties of the system to the studies of dynamic properties of the system consisting of individual components.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Although the preliminary results are appealing, they suffer from several limitations. First, sample sizes were small. Small sample size will limit the accuracy of the state-space models for biological pathways, which in turn affect estimation of dynamic properties of the systems. No much attention in control theory has been paid to investigation of impact of uncertainty inherent in dynamic systems on dynamic properties of the system. We will treat biological networks as stochastic dynamic system and study dynamic properties of stochastic dynamic systems in the future. Second, the quantities to characterize the dynamic properties are essentially random variables. Their distributions are unknown. We have not developed statistical methods to test significant differences in the dynamic properties of the pathways between the normal and abnormal cells. Third, the relations between the dynamic properties of the genes and their genotypes have not been investigated. Fourth, we have not performed large-scale dynamic analysis of the biological pathways. More theoretical development and large-scale real data analysis for the dynamic properties of the biological pathways are urgently needed.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Dermal fibroblast cultures</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Skin biopsies of clinically uninvolved skin (3 mm punch) were obtained from 5 patients with SSc and 5 normal controls after informed consent was granted. All five patients fulfilled American College of Rheumatology criteria for SSc <a href="#pone.0001693-Subcommittee1">[42]</a>. All five had diffuse skin involvement as defined by LeRoy et al <a href="#pone.0001693-LeRoy2">[43]</a>, and disease duration of less than five years. Skin biopsies from five normal controls with no history of autoimmune diseases undergoing dermatologic surgery were matched for age (+/− 5 years) and sex. The study was approved by the Committee for the Protection of Human Subjects at University of Texas Health Science Center at Houston.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The skin sample was transported in Dulbecco's Modified Essential Media (DMEM) with 10% fetal calf serum (FCS) (supplemented with an antibiotic and antimycotic) for processing the same day. The tissue sample was washed in 70% ethanol, PBS, and DMEM with 10% FCS. Cultured fibroblast cell strains were established by mincing tissues and placing them into 60 mm culture dishes secured by glass cover slips. The primary cultures were maintained in DMEM with 10% FCS and supplemented with antibiotic and antimycotic.</p>


<h4>Silica stimulation on fibroblasts</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The 5th passage of fibroblast strains were plated at a density of 2.5×10<sup>5</sup> cells in a 35 mm dish and grown until 80% confluence. Culture media then were replaced with FCS–free DMEM containing different doses (1, 5, 10, 25 and 50 µM) of silica particles obtained from Sigma-Aldrich, St Louis, MO. After 24-hour culture at this condition, the fibroblasts were harvested for extraction of RNA. The RNAs were examined with RT-PCR for gene expression of <em>COL1A2, COL3A1, TGFBRII, CTGF, SPARC</em> and <em>TIMP3</em>. The results from this dose-response assay provided an optimal dose (10 µM) in a time-dependent exposure for fibroblasts, in which 24-, 48-, 72-, 96- and 120-hour exposure of silica were assayed in cultured fibroblasts.</p>


<h4>Quantitative RT-PCR</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Quantitative real time RT-PCR was performed using an ABI 7900 sequence detector (Applied Biosystems, Foster City, CA). The specific primers and probes for each gene were purchased through Assays-on-Demand from Applied Biosystems. As described previously (19), total RNA from each sample was extracted from cultured fibroblasts described above using TRIzol reagent (Invitrogen Life Technology) and treated with Dnase I (Ambion, Austin, TX). cDNA was synthesized using Superscript II reverse transcriptase (Invitrogen Life Technology). Synthesized cDNAs were mixed with primers/probes in the 2× Taqman universal PCR buffer, and then assayed on an ABI 7900. The data obtained from assays were analyzed with SDS 2.1 software (Applied Biosystems). The amount of total RNA in each sample was normalized with 18 S rRNA transcript levels.</p>


<h4>State-Space Model and Parameter Estimation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>A biological pathway is taken as a dynamic system. The biological system is modeled by linear state-space equations defined as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where <em>x<sub>k</sub></em> is a vector of state variables at the time <em>k</em> that determine the dynamics of the regulation and unobserved, <em>u<sub>k</sub></em> is a vector of input variables at the time <em>k</em> such as drugs, environmental forces, and other state variables that lie outside the model, <em>y<sub>k</sub></em> are observed variables at the time , for example, the gene expressions, <em>A, B, C,</em> and <em>D</em> are matrices called state matrix, input matrix, output matrix and direct transmission matrix, respectively, <em>w</em> and <em>v</em> are noises assumed to be white Gaussian noise with zero means and variance <em>Q</em> and <em>R</em> respectively, and they are independent of each other.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>A fundamental and widely applicable parameter estimation method is Maximum Likelihood (ML) method that maximizes the likelihood of the observed data with respect to parameters. However, the state-space models involve unobserved state variables that are unavailable. It makes calculation of the likelihood in the setting of state-space models very difficult. To solve this problem, we use expectation-maximization (EM) Algorithm that is widely used iterative parameter estimation method <a href="#pone.0001693-Dempster1">[15]</a>. Specifically, we first assume that the state variables are available and then calculate the likelihood of both the observed data and hidden state variables which will be maximized with respect to the parameters in the models. After the estimated parameters are in our hands we then specify new state space models using the estimated parameters.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>For the convenience of presentation, equation (2) can be rewritten as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e006&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e007&amp;representation=PNG" class="inline-graphic"></span> Then, the conditional density function of <em>ζ<sub>k</sub></em>, given <em>Z<sub>k</sub></em> is given by <em>N</em>(Γ<em>z<sub>k</sub></em>, Π), where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e008&amp;representation=PNG" class="inline-graphic"></span>. Assume that the distribution of the initial state is given by <em>x</em><sub>1</sub>~<em>N</em>(<em>μ</em><sub>1</sub>, <em>P</em><sub>1</sub>). Let a sequence of input-output data samples and the state be denoted by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e009&amp;representation=PNG" class="inline-graphic"></span><br>The E-M algorithm for estimation of the parameters in the state-space model of discrete dynamic systems consists of two iterated steps: E-step and M-step. They are summarized as follows <a href="#pone.0001693-Gibson1">[16]</a>.</p>


<h4>E-Step</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Calculate the expectation of the augmented log-likelihood function of both the observed data and hidden state variables defined as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e010&amp;representation=PNG" class="inline-graphic"></span><br>To calculate <em>Q</em>(<em>θ, θ′</em>), we <strong>first</strong> need to calculate the conditional likelihood function <em>P<sub>θ</sub></em>(<em>X</em>, <em>Y|U</em>). From the model (3), we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>Combining equations (4) and (5), we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>Let<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e014&amp;representation=PNG" class="inline-graphic"></span><br>Taking expectation <em>E<sub>θ</sub></em>{.|<em>Y<sub>N</sub></em>, <em>U<sub>N</sub></em>} on both sides of equation (6), we obtain<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>To calculate the matrices Φ and Ψ, we use the following quantities<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br>and they can be calculated using Kalman smoother <a href="#pone.0001693-Kailath1">[44]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>The quantities <em>x̂<sub>k|k</sub></em>, <em>P<sub>k|k</sub></em>, <em>P<sub>k|k</sub></em><sub>−1</sub> are calculated by the Kalman filter equations as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br></p>


<h4>M-step</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Maximizing the likelihood function defined in equation (7) with respect to parameters yields<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br>Since the network has structure, which enforces certain parameters in the model to be zeros and leaves others to free to change, we develop constrained EM algorithms.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>First we define a matrix product operation of two matrices called Hadamard product, denoted by ○, as element wise product, i.e.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e020&amp;representation=PNG" class="inline-graphic"></span><br>Then, we define a modification of the vector <em>V,</em> denoted by [<em>V</em>]<sub>mod</sub>, as the vector in which all elements corresponding to the zeros elements in the matrix Γ are deleted. We define a modification of the matrix as the matrix in which if intersection of the row and column corresponds to the zeros elements in the matrix Γ then such row and column are deleted. The equation for estimation of parameters (18) is reduced to<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e021&amp;representation=PNG" class="inline-graphic"></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e022&amp;representation=PNG" class="inline-graphic"></span><br></p>


<h4>Transient-Response Analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Response of a biological pathway to perturbation of internal and external stimuli has two parts: the transient and the steady state response. The time varying process generated in going from the initial state to the final state in the response to the perturbation of the internal and external stimuli is called transient response. Steady-state response studies the system behavior at infinite time. Transient-response analysis of biological pathways can be used to quantify their dynamics. It can reveal how fast the biological pathways respond to perturbation of environments and how accurately the pathways can finally achieve the desired steady-state values. It can also be used to study damped vibration behavior and stability of the biological pathways.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>The transient response of the dynamic systems depends on the input signals. Different signal will cause different response. There are numerous types of signal in practice. For the convenience of comparison, we consider two types of signals: (i) unit-step signal and (ii) unit-impulse signal as shown in <a href="#pone-0001693-g002">Figure 2</a>.</p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>The transient response of dynamic systems can be studied by transfer function that is used to characterize the input-output relationships of a linear, time-invariant, differential equation system. The transfer function is defined as the ratio of the Laplace transform of the output to the Laplace transform of the input under the assumption of zero initial conditions. The transfer function of the response of the biological pathway to unit-step and unit-impulse input signals are given by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e023&amp;representation=PNG" class="inline-graphic"></span> and <em>Y</em>(<em>s</em>) = <em>G</em>(<em>s</em>) respectively, where <em>G</em>(<em>s</em>) is the transfer function of the biological pathway. The transient-response analysis of the biological pathway can be performed by inverse Laplace transformation. We performed the transient-response analysis with MATLAB <a href="#pone.0001693-Ogata1">[13]</a>.</p>


<h4>Stability Analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The most important dynamic property of biological pathways is concerned with stability. Dynamic systems are called stable if their variables such as gene expressions return to, or towards, their original or equilibrium states following internal and external perturbations. For any practical purpose, the biological pathways must be stable. Unstable gene regulations will lead to the malfunction or even the death of the cells. A biological pathway will remain at steady state until occurrence of external perturbation. Depending on dynamic behavior of the system after perturbation of environments, the steady-states of the system are either stable (the system returns to the initial state or changes to other steady-states) or unstable (the system leaves the initial equilibrium state).</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>One of the methods for assessing the stability of the linear dynamic systems is to analyze eigenvalues of the state transition matrix <em>A</em> of the linear dynamic systems. For a continuous linear dynamic system, if real parts of all eigenvalues of the transition matrix <em>A</em> are strictly negative then the system is stable. For a discrete linear system, if the norm of all eigenvalues of the state transition matrix <em>A</em> is less than 1 then the system is stable.</p>


<h4>Root-Locus Analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Open and close loop poles and zeros largely determine the stability and performance of the open and close systems. They provide valuable information on how to improve stability and transient response of the systems. Root-locus analysis, in which the roots of the characteristic equation of the closed-loop system are plotted for all values of a system parameter, is a powerful tool for study and design of dynamic pathway. The loop gain is often chosen to be the parameter. Varying the gain value will change the location of the closed-loop poles.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>Consider a SISO system that consists of a gene regulator and an input to the gene regulator shown in <a href="#pone-0001693-g006">Figure 6</a>. The transfer function of the closed-loop system is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e024&amp;representation=PNG" class="inline-graphic"></span><br>which implies the following characteristic equation of this closed-loop system:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e025&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br></p>
<div class="figure" id="pone-0001693-g006"><div class="img"><a name="pone-0001693-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001693" data-uri="info:doi/10.1371/journal.pone.0001693.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001693.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001693.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001693.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Scheme of a SISO system</span></strong></p><span>doi:10.1371/journal.pone.0001693.g006</span></div><a id="article1.body1.sec4.sec9.p3" name="article1.body1.sec4.sec9.p3"></a><p>In general, <em>G</em>(<em>s</em>) <em>H</em>(<em>s</em>) involves a gain parameter <em>K</em>. A plot of the points in the complex plane satisfying the characteristic equation (12) is the root locus. As the gain parameter changes the root locus will plot a curve in the complex s-plane. A simple method for plotting root-locus has been developed by W. R. Evans <a href="#pone.0001693-Zhou1">[29]</a>. In this report, we use MATLAB to generate root-locus plots <a href="#pone.0001693-Ogata1">[13]</a>.</p>


<h4>Controllability</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>A dynamic system is called controllable, if there is an admissible control function, which can change the system from any given initial state to any finite state or to the origin of the state space in the finite time. Define the controllability matrix of the system as <em>H</em> = [<em>B</em>, <em>AB</em>, …, <em>A<sup>n</sup></em><sup>−1</sup><em>B</em>], where <em>A</em> and <em>B</em> are the state transition matrix and input to the state matrix in the linear dynamic system, respectively. If rank (<em>H</em>) = <em>n</em>, i.e., the rank of the controllability matrix is equal to the number of the state variables of the system, then the genetic network is completely controllable.</p>
<a id="article1.body1.sec4.sec10.p2" name="article1.body1.sec4.sec10.p2"></a><p>We use the condition number of the controllability matrix to measure the degree of the controllability of the system. The condition number of the controllability matrix is defined as <a href="#pone.0001693-Wei1">[45]</a><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e026&amp;representation=PNG" class="inline-graphic"></span><br>where <em>H</em><sup>−</sup> is a generalized inverse of the matrix <em>H</em> and ||.|| denotes a matrix norm. This can be justified by the following arguments. The general solution of the discrete linear system is given by <a href="#pone.0001693-Ogata1">[13]</a><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e027&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br>By definition, if the system is controllable, then at some time <em>t<sub>k</sub></em>, we have <em>x<sub>k</sub></em> = 0, which implies that<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e028&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br></p>
<a id="article1.body1.sec4.sec10.p3" name="article1.body1.sec4.sec10.p3"></a><p>Equation (14) can be reduced to<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e029&amp;representation=PNG" class="inline-graphic"></span><br>Or<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e030&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br>where <em>u</em> is a control vector. Solving the equation (15) yields<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001693.e031&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br>The norm of the control vector represents the amount of control efforts required to change the states from initial value to the desired value and hence measures the degree of the controllability. From equation (16), we note that the norm of the control vector ||<em>u</em>|| is proportional to the condition number: <em>κ</em>(<em>H</em>) = ||<em>H</em><sup>−</sup>||||<em>H</em>||. Therefore, we can use the condition number of the controllability matrix to measure the degree of controllability.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: XZ MX. Performed the experiments: XG. Analyzed the data: MX HX. Contributed reagents/materials/analysis tools: XZ. Wrote the paper: XZ MX FA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001693-Herrero1" id="pone.0001693-Herrero1"></a>Herrero J, Vaquerizas JM, Al-Shahrour F, Conde L, Mateos A, et al.  (2004) New challenges in gene expression data analysis and the extended GEPAS. Nucleic Acids Res  32: W485–91.  <ul class="find" data-citedArticleID="1064410" data-doi="10.1093/nar/gkh421"><li><a href="http://dx.doi.org/10.1093/nar/gkh421" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+challenges+in+gene+expression+data+analysis+and+the+extended+GEPAS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+challenges+in+gene+expression+data+analysis+and+the+extended+GEPAS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001693-Assmus1" id="pone.0001693-Assmus1"></a>Assmus HE, Herwig R, Kwang-Hyun Cho K-H, Wolkenhauer O (2006) Dynamics of biological systems: role of systems biology in medical research. Expert Review of Molecular Diagnostics  6: 891–902.  <ul class="find" data-citedArticleID="1064386" data-doi="10.1586/14737159.6.6.891"><li><a href="http://dx.doi.org/10.1586/14737159.6.6.891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamics+of+biological+systems%3A+role+of+systems+biology+in+medical+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamics+of+biological+systems%3A+role+of+systems+biology+in+medical+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001693-Barabsi1" id="pone.0001693-Barabsi1"></a>Barabási A-L, Oltvai ZN (2004) Network biology: understanding the cell's functional organization. Nature Reviews Genetics  5: 101–113.  <ul class="find" data-citedArticleID="1064388" data-doi="10.1038/nrg1272"><li><a href="http://dx.doi.org/10.1038/nrg1272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+biology%3A+understanding+the+cell%27s+functional+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+biology%3A+understanding+the+cell%27s+functional+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001693-Wolkenhauer1" id="pone.0001693-Wolkenhauer1"></a>Wolkenhauer O, Ullah M, Wellstead P, Cho KH (2005) The dynamic systems approach to control and regulation of intracellular networks. FEBS Lett  579: 1846–53.  <ul class="find" data-citedArticleID="1064460" data-doi="10.1016/j.febslet.2005.02.008"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.02.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamic+systems+approach+to+control+and+regulation+of+intracellular+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamic+systems+approach+to+control+and+regulation+of+intracellular+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001693-Akutsu1" id="pone.0001693-Akutsu1"></a>Akutsu T, Miyano S, Kuhara S (1999) Identification of genetic networks from a small number of gene expression patterns under the Boolean network model. Pacific Symposium on Biocomputing 17–28.  <ul class="find" data-citedArticleID="1064380" data-doi="10.1142/9789814447300_0003"><li><a href="http://dx.doi.org/10.1142/9789814447300_0003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+genetic+networks+from+a+small+number+of+gene+expression+patterns+under+the+Boolean+network+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+genetic+networks+from+a+small+number+of+gene+expression+patterns+under+the+Boolean+network+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001693-Kauffman1" id="pone.0001693-Kauffman1"></a>Kauffman S, Peterson C, Samuelsson B, Troein C (2003) Random Boolean network models and the yeast transcriptional network. Proc Natl Acad Sci U S A  100: 14796–14799.  <ul class="find" data-citedArticleID="1064418" data-doi="10.1073/pnas.2036429100"><li><a href="http://dx.doi.org/10.1073/pnas.2036429100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Random+Boolean+network+models+and+the+yeast+transcriptional+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Random+Boolean+network+models+and+the+yeast+transcriptional+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001693-Martin1" id="pone.0001693-Martin1"></a>Martin S, Zhang Z, Martino A, Faulon JL (2007) Boolean Dynamics of Genetic Regulatory Networks Inferred from Microarray Time Series Data. Bioinformatics  23: 866–874.  <ul class="find" data-citedArticleID="1064430" data-doi="10.1093/bioinformatics/btm021"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Boolean+Dynamics+of+Genetic+Regulatory+Networks+Inferred+from+Microarray+Time+Series+Data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Boolean+Dynamics+of+Genetic+Regulatory+Networks+Inferred+from+Microarray+Time+Series+Data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001693-deJong1" id="pone.0001693-deJong1"></a>de Jong H (2002) Modeling and simulation of genetic regulatory systems: a literature review. J Comput Biol  9: 67–103.  <ul class="find" data-citedArticleID="1064468" data-doi="10.1089/10665270252833208"><li><a href="http://dx.doi.org/10.1089/10665270252833208" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+and+simulation+of+genetic+regulatory+systems%3A+a+literature+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+and+simulation+of+genetic+regulatory+systems%3A+a+literature+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001693-Beal1" id="pone.0001693-Beal1"></a>Beal MJ, Falciani F, Ghahramani Z, Rangel C, Wild DL (2005) A Bayesian approach to reconstructing genetic regulatory networks with hidden factors. Bioinformatics  21: 349–356.  <ul class="find" data-citedArticleID="1064390" data-doi="10.1093/bioinformatics/bti014"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Bayesian+approach+to+reconstructing+genetic+regulatory+networks+with+hidden+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Bayesian+approach+to+reconstructing+genetic+regulatory+networks+with+hidden+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001693-Dojer1" id="pone.0001693-Dojer1"></a>Dojer N, Gambin A, Mizera A, Wilczynski B, Tiuryn J (2006) Applying dynamic Bayesian networks to perturbed gene expression data. BMC Bioinformatics  7: 249.  <ul class="find" data-citedArticleID="1064404" data-doi="10.1186/1471-2105-7-249"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applying+dynamic+Bayesian+networks+to+perturbed+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applying+dynamic+Bayesian+networks+to+perturbed+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001693-Werhli1" id="pone.0001693-Werhli1"></a>Werhli A, Husmeier D (2007) Reconstructing gene regulatory networks with bayesian networks by combining expression data with multiple sources of prior knowledge. Stat Appl Genet Mol Biol  6: 15.  <ul class="find" data-citedArticleID="1064458" data-doi="10.2202/1544-6115.1282"><li><a href="http://dx.doi.org/10.2202/1544-6115.1282" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+gene+regulatory+networks+with+bayesian+networks+by+combining+expression+data+with+multiple+sources+of+prior+knowledge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+gene+regulatory+networks+with+bayesian+networks+by+combining+expression+data+with+multiple+sources+of+prior+knowledge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001693-Nachman1" id="pone.0001693-Nachman1"></a>Nachman I, Regeve A, Friedman N (2004) Inferring quantitative models of regulatory networks from expression data. Bioinformatics  20: Suppl. 1248–1256.  <ul class="find" data-citedArticleID="1064434" data-doi="10.1093/bioinformatics/bth941"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth941" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+quantitative+models+of+regulatory+networks+from+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+quantitative+models+of+regulatory+networks+from+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001693-Ogata1" id="pone.0001693-Ogata1"></a>Ogata K (1998) System dynamics. Third Edition. Upper Saddle Rive, New Jersey: Prentice Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001693-Rangel1" id="pone.0001693-Rangel1"></a>Rangel C, Wild DL, Falciani F, Ghahramani Z, Gaiba A (2001) Modeling biological responses using gene expression profiling and linear dynamical systems. Proceedings of the 2nd International Conference on Systems Biology. Omipress 248–256.  <ul class="find" data-citedArticleID="1064444"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modeling+biological+responses+using+gene+expression+profiling+and+linear+dynamical+systems.+Proceedings+of+the+2nd+International+Conference+on+Systems+Biology.&amp;auth=&amp;atitle=Modeling+biological+responses+using+gene+expression+profiling+and+linear+dynamical+systems.+Proceedings+of+the+2nd+International+Conference+on+Systems+Biology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+biological+responses+using+gene+expression+profiling+and+linear+dynamical+systems.+Proceedings+of+the+2nd+International+Conference+on+Systems+Biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+biological+responses+using+gene+expression+profiling+and+linear+dynamical+systems.+Proceedings+of+the+2nd+International+Conference+on+Systems+Biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001693-Dempster1" id="pone.0001693-Dempster1"></a>Dempster A, Laird N, Rubin D (1977) maximum likelihood from incomplete data via the EM algorithm. Journal of the Royal Statistical Society,  Seres B 39: 1–38.  <ul class="find" data-citedArticleID="1064400"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=maximum+likelihood+from+incomplete+data+via+the+EM+algorithm.&amp;auth=&amp;atitle=maximum+likelihood+from+incomplete+data+via+the+EM+algorithm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=maximum+likelihood+from+incomplete+data+via+the+EM+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22maximum+likelihood+from+incomplete+data+via+the+EM+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001693-Gibson1" id="pone.0001693-Gibson1"></a>Gibson S, Ninness B (2005) Robust Maximum-Likelihood Estimation of Multivariable Dynamic Systems. Automatica  41: 1667–1682.  <ul class="find" data-citedArticleID="1064408" data-doi="10.1016/j.automatica.2005.05.008"><li><a href="http://dx.doi.org/10.1016/j.automatica.2005.05.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robust+Maximum-Likelihood+Estimation+of+Multivariable+Dynamic+Systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robust+Maximum-Likelihood+Estimation+of+Multivariable+Dynamic+Systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001693-Voit1" id="pone.0001693-Voit1"></a>Voit EO, Schwacke JH (2007)  Understanding through modeling – A historical perspective and review of biochemical system theory as a powerful tool for systems biology. In: Konpopka AK, editor. Systems biology: principles, methods and concepts. CRC Press.   <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0001693-Prill1" id="pone.0001693-Prill1"></a>Prill RJ, Iglesias PA, Levchenko A (2005) Dynamic properties of network motifs contribute to biological network organization. PLoS Biol  3: e343.  <ul class="find" data-citedArticleID="1064440" data-doi="10.1371/journal.pbio.0030343"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+properties+of+network+motifs+contribute+to+biological+network+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+properties+of+network+motifs+contribute+to+biological+network+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001693-Stelling1" id="pone.0001693-Stelling1"></a>Stelling J, Sauer U, Szallasi Z, Doyle FJ 3rd, Doyle J (2004) Robustness of cellular functions. Cell  118: 675–85.  <ul class="find" data-citedArticleID="1064446" data-doi="10.1016/j.cell.2004.09.008"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.09.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness+of+cellular+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness+of+cellular+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001693-Martins1" id="pone.0001693-Martins1"></a>Martins NC, Dahleh MA, Elia N (2006) Feedback stabilization of uncertain systems in the presence of a direct link. IEEE Transactions on Automatic Control  51: 438–447.  <ul class="find" data-citedArticleID="1064432" data-doi="10.1109/tac.2006.871940"><li><a href="http://dx.doi.org/10.1109/tac.2006.871940" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feedback+stabilization+of+uncertain+systems+in+the+presence+of+a+direct+link." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Feedback+stabilization+of+uncertain+systems+in+the+presence+of+a+direct+link.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001693-Kremling1" id="pone.0001693-Kremling1"></a>Kremling A, Saez-Rodriguez J (2007) Systems biology-An engineering perspective. J Biotechnol  129: 329–351.  <ul class="find" data-citedArticleID="1064422" data-doi="10.1016/j.jbiotec.2007.02.009"><li><a href="http://dx.doi.org/10.1016/j.jbiotec.2007.02.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systems+biology-An+engineering+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systems+biology-An+engineering+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001693-Claman1" id="pone.0001693-Claman1"></a>Claman HN, Giorno RC, Seibold JR (1991) Endothelial and fibroblastic activation in scleroderma. The myth of the “uninvolved skin”. Arthritis Rheum  34: 1495–1501.  <ul class="find" data-citedArticleID="1064398" data-doi="10.1002/art.1780341204"><li><a href="http://dx.doi.org/10.1002/art.1780341204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endothelial+and+fibroblastic+activation+in+scleroderma.+The+myth+of+the+%E2%80%9Cuninvolved+skin%E2%80%9D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endothelial+and+fibroblastic+activation+in+scleroderma.+The+myth+of+the+%E2%80%9Cuninvolved+skin%E2%80%9D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001693-Cabral1" id="pone.0001693-Cabral1"></a>Cabral AR, Alcocer-Varela J, Orozco-Topete R, Reyes E, Fernandez-Dominguez L, et al.  (1994) Clinical, histopathological, immunological and fibroblast studies in 30 patients with subcutaneous injections of modelants including silicone and mineral oils. Rev Invest Clin  46: 257–66.  <ul class="find" data-citedArticleID="1064394"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Clinical%2C+histopathological%2C+immunological+and+fibroblast+studies+in+30+patients+with+subcutaneous+injections+of+modelants+including+silicone+and+mineral+oils.&amp;auth=&amp;atitle=Clinical%2C+histopathological%2C+immunological+and+fibroblast+studies+in+30+patients+with+subcutaneous+injections+of+modelants+including+silicone+and+mineral+oils." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clinical%2C+histopathological%2C+immunological+and+fibroblast+studies+in+30+patients+with+subcutaneous+injections+of+modelants+including+silicone+and+mineral+oils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clinical%2C+histopathological%2C+immunological+and+fibroblast+studies+in+30+patients+with+subcutaneous+injections+of+modelants+including+silicone+and+mineral+oils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001693-Gerriets1" id="pone.0001693-Gerriets1"></a>Gerriets JE, Reiser KM, Last JA (1996) Lung collagen cross-links in rats with experimentally induced pulmonary fibrosis. Biochim Biophys Acta  1316: 121–31.  <ul class="find" data-citedArticleID="1064406" data-doi="10.1016/0925-4439(96)00019-1"><li><a href="http://dx.doi.org/10.1016/0925-4439(96)00019-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lung+collagen+cross-links+in+rats+with+experimentally+induced+pulmonary+fibrosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lung+collagen+cross-links+in+rats+with+experimentally+induced+pulmonary+fibrosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001693-Kim1" id="pone.0001693-Kim1"></a>Kim KA, Kim YH, Seok Seo M, Kyu Lee W, Won Kim S, et al.  (2002) Mechanism of silica-induced ROS generation in Rat2 fibroblast cells. Toxicol Lett  135: 185–91.  <ul class="find" data-citedArticleID="1064420" data-doi="10.1016/s0378-4274(02)00237-0"><li><a href="http://dx.doi.org/10.1016/s0378-4274(02)00237-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+silica-induced+ROS+generation+in+Rat2+fibroblast+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+silica-induced+ROS+generation+in+Rat2+fibroblast+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001693-Arcangeli1" id="pone.0001693-Arcangeli1"></a>Arcangeli G, Cupelli V, Giuliano G (2001) Effects of silica on human lung fibroblast in culture. Sci Total Environ  270: 135–9.  <ul class="find" data-citedArticleID="1064384" data-doi="10.1016/s0048-9697(00)00781-6"><li><a href="http://dx.doi.org/10.1016/s0048-9697(00)00781-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+silica+on+human+lung+fibroblast+in+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+silica+on+human+lung+fibroblast+in+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001693-Pan1" id="pone.0001693-Pan1"></a>Pan Y, Durfee T, Bockhorst J, Craven M (2007) Connecting quantitative regulatory-network models to the genome. Bioinformatics  23: 1367–1376.  <ul class="find" data-citedArticleID="1064438" data-doi="10.1093/bioinformatics/btm228"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Connecting+quantitative+regulatory-network+models+to+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Connecting+quantitative+regulatory-network+models+to+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001693-Steuer1" id="pone.0001693-Steuer1"></a>Steuer R, Gross T, Selbig J, Blasius B (2006) Structural kinetic modeling of metabolic networks. <em>Proc. Natl Acad. Sci.</em> USA  103: 11868–11873.  <ul class="find" data-citedArticleID="1064448" data-doi="10.1073/pnas.0600013103"><li><a href="http://dx.doi.org/10.1073/pnas.0600013103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+kinetic+modeling+of+metabolic+networks.+Proc.+Natl+Acad.+Sci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+kinetic+modeling+of+metabolic+networks.+Proc.+Natl+Acad.+Sci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001693-Zhou1" id="pone.0001693-Zhou1"></a>Zhou XD, Xiong MM, Tan FK, Guo XJ, Arnett FC (2006) SPARC, an upstream regulator of CTGF in response to TGF-β stimulation. Arthritis &amp; Rheum  54: 3885–3889.  <ul class="find" data-citedArticleID="1064466" data-doi="10.1002/art.22249"><li><a href="http://dx.doi.org/10.1002/art.22249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SPARC%2C+an+upstream+regulator+of+CTGF+in+response+to+TGF-%CE%B2+stimulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SPARC%2C+an+upstream+regulator+of+CTGF+in+response+to+TGF-%CE%B2+stimulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001693-Xiong1" id="pone.0001693-Xiong1"></a>Xiong M, Feghali-Bostwick CA, Arnett FC, Zhou X (2005) A systems biology approach to genetic studies of complex diseases. FEBS Lett  579: 5325–5332.  <ul class="find" data-citedArticleID="1064464" data-doi="10.1016/j.febslet.2005.08.058"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.08.058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systems+biology+approach+to+genetic+studies+of+complex+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systems+biology+approach+to+genetic+studies+of+complex+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001693-Derynck1" id="pone.0001693-Derynck1"></a>Derynck R, Zhang YE (2003) Smad-dependent and Smad-independent pathways in TGF-beta family signalling. Nature  425: 577–584.  <ul class="find" data-citedArticleID="1064402" data-doi="10.1038/nature02006"><li><a href="http://dx.doi.org/10.1038/nature02006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Smad-dependent+and+Smad-independent+pathways+in+TGF-beta+family+signalling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Smad-dependent+and+Smad-independent+pathways+in+TGF-beta+family+signalling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001693-Jinnin1" id="pone.0001693-Jinnin1"></a>Jinnin M, Ihn H, Mimura Y, Asano Y, Tamaki K (2006) Potential regulatory elements of the constitutive up-regulated α2(I) collagen gene in scleroderma dermal fibroblasts. Biochem Biophys Res Commun.  343: 904–909.  <ul class="find" data-citedArticleID="1064414" data-doi="10.1016/j.bbrc.2006.03.037"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.03.037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Potential+regulatory+elements+of+the+constitutive+up-regulated+%CE%B12%28I%29+collagen+gene+in+scleroderma+dermal+fibroblasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Potential+regulatory+elements+of+the+constitutive+up-regulated+%CE%B12%28I%29+collagen+gene+in+scleroderma+dermal+fibroblasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001693-Jimenez1" id="pone.0001693-Jimenez1"></a>Jimenez SA, Feldman G, Bashey RL, Bienkowski R, Rosenbloom J (1986) Co-ordinate increase in the expression of type I and type III collagen genes in progressive systemic sclerosis. Biochem J  237: 837–843.  <ul class="find" data-citedArticleID="1064412"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Co-ordinate+increase+in+the+expression+of+type+I+and+type+III+collagen+genes+in+progressive+systemic+sclerosis.&amp;auth=&amp;atitle=Co-ordinate+increase+in+the+expression+of+type+I+and+type+III+collagen+genes+in+progressive+systemic+sclerosis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Co-ordinate+increase+in+the+expression+of+type+I+and+type+III+collagen+genes+in+progressive+systemic+sclerosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Co-ordinate+increase+in+the+expression+of+type+I+and+type+III+collagen+genes+in+progressive+systemic+sclerosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001693-LeRoy1" id="pone.0001693-LeRoy1"></a>LeRoy EC (1974) Increased collagen synthesis by scleroderma skin fibroblasts in vitro: a possible defect in the regulation or activation of the scleroderma fibroblast. J Clin Invest  54: 880–889.  <ul class="find" data-citedArticleID="1064426" data-doi="10.1172/jci107827"><li><a href="http://dx.doi.org/10.1172/jci107827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+collagen+synthesis+by+scleroderma+skin+fibroblasts+in+vitro%3A+a+possible+defect+in+the+regulation+or+activation+of+the+scleroderma+fibroblast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+collagen+synthesis+by+scleroderma+skin+fibroblasts+in+vitro%3A+a+possible+defect+in+the+regulation+or+activation+of+the+scleroderma+fibroblast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001693-Chen1" id="pone.0001693-Chen1"></a>Chen L, Aihara K (2002) Stability of genetic regulatory networks with time delay. IEEE Transactions on Circuits and Systems-I: Fundamental Theory and Applications  49: 602–608.  <ul class="find" data-citedArticleID="1064396" data-doi="10.1109/tcsi.2002.1001949"><li><a href="http://dx.doi.org/10.1109/tcsi.2002.1001949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stability+of+genetic+regulatory+networks+with+time+delay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stability+of+genetic+regulatory+networks+with+time+delay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001693-Prill2" id="pone.0001693-Prill2"></a>Prill RJ, Iglesias PA, Levchenko A (2005) Dynamic Properties of Network Motifs Contribute to Biological Network Organization. PLoS Biol  3(11): e343.  <ul class="find" data-citedArticleID="1064442" data-doi="10.1371/journal.pbio.0030343"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+Properties+of+Network+Motifs+Contribute+to+Biological+Network+Organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+Properties+of+Network+Motifs+Contribute+to+Biological+Network+Organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001693-Binder1" id="pone.0001693-Binder1"></a>Binder B, Heinrich R (2002) Dynamic stability of signal transduction networks depending on downstream and upstream specificity of protein kinases. Mol Biol Rep  29: 51–55.  <ul class="find" data-citedArticleID="1064392"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dynamic+stability+of+signal+transduction+networks+depending+on+downstream+and+upstream+specificity+of+protein+kinases.&amp;auth=&amp;atitle=Dynamic+stability+of+signal+transduction+networks+depending+on+downstream+and+upstream+specificity+of+protein+kinases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+stability+of+signal+transduction+networks+depending+on+downstream+and+upstream+specificity+of+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+stability+of+signal+transduction+networks+depending+on+downstream+and+upstream+specificity+of+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001693-Kremling2" id="pone.0001693-Kremling2"></a>Kremling A, Saez-Rodriguez J (2007) Systems biology-An engineering perspective. J Biotechnol  129: 329–351.  <ul class="find" data-citedArticleID="1064424" data-doi="10.1016/j.jbiotec.2007.02.009"><li><a href="http://dx.doi.org/10.1016/j.jbiotec.2007.02.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systems+biology-An+engineering+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systems+biology-An+engineering+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001693-Vinson1" id="pone.0001693-Vinson1"></a>Vinson DR, Georgaki C (2000) A new measure of process output controllability. Journal of Process Control  10: 185–194.  <ul class="find" data-citedArticleID="1064452" data-doi="10.1016/s0959-1524(99)00045-1"><li><a href="http://dx.doi.org/10.1016/s0959-1524(99)00045-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+measure+of+process+output+controllability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+measure+of+process+output+controllability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001693-Wolkenhauer2" id="pone.0001693-Wolkenhauer2"></a>Wolkenhauer O, Mesarovic M (2005) Feedback dynamics and cell function: Why systems biology is called Systems Biology. Mol. Biosyst  1: 14–16.  <ul class="find" data-citedArticleID="1064462" data-doi="10.1039/b502088n"><li><a href="http://dx.doi.org/10.1039/b502088n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feedback+dynamics+and+cell+function%3A+Why+systems+biology+is+called+Systems+Biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Feedback+dynamics+and+cell+function%3A+Why+systems+biology+is+called+Systems+Biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001693-Aldana1" id="pone.0001693-Aldana1"></a>Aldana M, Balleza E, Kauffman S, Resendiz O (2007) Robustness and evolvability in genetic regulatory networks. J Theor Biol  245: 433–448.  <ul class="find" data-citedArticleID="1064382" data-doi="10.1016/j.jtbi.2006.10.027"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2006.10.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness+and+evolvability+in+genetic+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness+and+evolvability+in+genetic+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001693-Subcommittee1" id="pone.0001693-Subcommittee1"></a>Subcommittee for Scleroderma Criteria of the American Rheumatism Association Diagnostic and Therapeutic Criteria Committee (1980) Preliminary criteria for the classification of systemic sclerosis (scleroderma). Arthritis Rheum  23: 581–590.  <ul class="find" data-citedArticleID="1064450" data-doi="10.1002/art.1780230510"><li><a href="http://dx.doi.org/10.1002/art.1780230510" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preliminary+criteria+for+the+classification+of+systemic+sclerosis+%28scleroderma%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preliminary+criteria+for+the+classification+of+systemic+sclerosis+%28scleroderma%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001693-LeRoy2" id="pone.0001693-LeRoy2"></a>LeRoy EC, Black C, Fleischmajer R, Jablonska S, Krieg T, et al.  (1988) Scleroderma (systemic sclerosis): classification, subsets and pathogenesis. J Rheumatol  15: 202–205.  <ul class="find" data-citedArticleID="1064428"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Scleroderma+%28systemic+sclerosis%29%3A+classification%2C+subsets+and+pathogenesis.&amp;auth=&amp;atitle=Scleroderma+%28systemic+sclerosis%29%3A+classification%2C+subsets+and+pathogenesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scleroderma+%28systemic+sclerosis%29%3A+classification%2C+subsets+and+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scleroderma+%28systemic+sclerosis%29%3A+classification%2C+subsets+and+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001693-Kailath1" id="pone.0001693-Kailath1"></a>Kailath T, Sayed AH, Hassibi B (1999) Linear Estimation. Upper Saddle River, New Jersey: Prentice Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0001693-Wei1" id="pone.0001693-Wei1"></a>Wei Y, Wang D (2003) Condition numbers and perturbation of the weighted Moore-Penrose inverse and weighted linear squares problem. Applied mathematics and Computation  145: 45–58.  <ul class="find" data-citedArticleID="1064456" data-doi="10.1016/s0096-3003(02)00437-x"><li><a href="http://dx.doi.org/10.1016/s0096-3003(02)00437-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Condition+numbers+and+perturbation+of+the+weighted+Moore-Penrose+inverse+and+weighted+linear+squares+problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Condition+numbers+and+perturbation+of+the+weighted+Moore-Penrose+inverse+and+weighted+linear+squares+problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.XML" value="112617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.PDF" value="540792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e001.PNG" value="10930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e001.TIF" value="26564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e002.PNG" value="28495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e002.TIF" value="32126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e003.PNG" value="38736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e003.TIF" value="57792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g001.PNG_L" value="290919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g001.PNG_M" value="148453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g001.PNG_S" value="11439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g001.TIF" value="455246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g001.PNG_I" value="53264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.t001.PNG_L" value="36729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.t001.PNG_M" value="45852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.t001.PNG_S" value="9779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.t001.TIF" value="208814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.t001.PNG_I" value="15299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g002.PNG_L" value="29360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g002.PNG_M" value="22171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g002.PNG_S" value="4549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g002.TIF" value="81438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g002.PNG_I" value="8418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g003.PNG_L" value="291706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g003.PNG_M" value="116619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g003.PNG_S" value="9596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g003.TIF" value="511478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g003.PNG_I" value="41955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g004.PNG_L" value="191477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g004.PNG_M" value="54152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g004.PNG_S" value="6117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g004.TIF" value="382772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g004.PNG_I" value="19247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g005.PNG_L" value="197527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g005.PNG_M" value="53836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g005.PNG_S" value="5692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g005.TIF" value="380390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g005.PNG_I" value="18814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e004.PNG" value="11004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e004.TIF" value="26612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e005.PNG" value="6517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e005.TIF" value="22472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e006.PNG" value="10235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e006.TIF" value="26332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e007.PNG" value="10538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e007.TIF" value="26544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e008.PNG" value="8377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e008.TIF" value="24428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e009.PNG" value="9634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e009.TIF" value="25736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e010.PNG" value="9633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e010.TIF" value="25900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e011.PNG" value="11068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e011.TIF" value="27220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e012.PNG" value="12141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e012.TIF" value="28668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e013.PNG" value="16032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e013.TIF" value="32288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e014.PNG" value="18713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e014.TIF" value="34692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e015.PNG" value="16028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e015.TIF" value="32672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e016.PNG" value="19494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e016.TIF" value="35760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e017.PNG" value="22161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e017.TIF" value="38476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e018.PNG" value="26135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e018.TIF" value="41912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e019.PNG" value="11979"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e019.TIF" value="28016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e020.PNG" value="7685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e020.TIF" value="23768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e021.PNG" value="10400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e021.TIF" value="26376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e022.PNG" value="8826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e022.TIF" value="24880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e023.PNG" value="6671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e023.TIF" value="22620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e024.PNG" value="10405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e024.TIF" value="26172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e025.PNG" value="7459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e025.TIF" value="23484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g006.PNG_L" value="13743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g006.PNG_M" value="14505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g006.PNG_S" value="4334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g006.TIF" value="21312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.g006.PNG_I" value="4989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e026.PNG" value="7419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e026.TIF" value="23696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e027.PNG" value="12483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e027.TIF" value="28308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e028.PNG" value="9868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e028.TIF" value="25648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e029.PNG" value="10680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e029.TIF" value="27072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e030.PNG" value="6966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e030.TIF" value="22912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e031.PNG" value="6927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001693.e031.TIF" value="22876"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001693&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001693" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001693&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001693&volume=&issue=&title=Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli&author_name=Momiao%20Xiong%2C%20Frank%20C.%20Arnett%2C%20Xinjian%20Guo%2C%20Hao%20Xiong%2C%20Xiaodong%20Zhou&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001693" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001693" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001693" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001693&amp;t=Differential%20Dynamic%20Properties%20of%20Scleroderma%20Fibroblasts%20in%20Response%20to%20Perturbation%20of%20Environmental%20Stimuli" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001693&title=Differential%20Dynamic%20Properties%20of%20Scleroderma%20Fibroblasts%20in%20Response%20to%20Perturbation%20of%20Environmental%20Stimuli&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001693&amp;title=Differential%20Dynamic%20Properties%20of%20Scleroderma%20Fibroblasts%20in%20Response%20to%20Perturbation%20of%20Environmental%20Stimuli" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001693" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001693&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Differential Dynamic Properties of Scleroderma Fibroblasts in Response to Perturbation of Environmental Stimuli';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001693';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Differential%20Dynamic%20Properties%20of%20Scleroderma%20Fibroblasts%20in%20Response%20to%20Perturbation%20of%20Environmental%20Stimuli http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001693" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001693" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Dynamical+systems%22" title="Search for articles in the subject area:'Dynamical systems'"><div class="flagText">Dynamical systems</div></a>
              <div data-categoryid="21479" data-articleid="26212"
                   data-categoryname="Dynamical systems"
                   class="flagImage" title="Flag 'Dynamical systems' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fibroblasts%22" title="Search for articles in the subject area:'Fibroblasts'"><div class="flagText">Fibroblasts</div></a>
              <div data-categoryid="47975" data-articleid="26212"
                   data-categoryname="Fibroblasts"
                   class="flagImage" title="Flag 'Fibroblasts' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26212"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="26212"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulatory+networks%22" title="Search for articles in the subject area:'Gene regulatory networks'"><div class="flagText">Gene regulatory networks</div></a>
              <div data-categoryid="17555" data-articleid="26212"
                   data-categoryname="Gene regulatory networks"
                   class="flagImage" title="Flag 'Gene regulatory networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="31807" data-articleid="26212"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Scleroderma%22" title="Search for articles in the subject area:'Scleroderma'"><div class="flagText">Scleroderma</div></a>
              <div data-categoryid="46569" data-articleid="26212"
                   data-categoryname="Scleroderma"
                   class="flagImage" title="Flag 'Scleroderma' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22System+stability%22" title="Search for articles in the subject area:'System stability'"><div class="flagText">System stability</div></a>
              <div data-categoryid="32597" data-articleid="26212"
                   data-categoryname="System stability"
                   class="flagImage" title="Flag 'System stability' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8679'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6994'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1895&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
