

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000049"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000049" />

    <meta name="citation_title" content="Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection"/>
    <meta itemprop="name" content="Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection"/>

      <meta name="citation_author" content="Akiko Kashiwagi"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics Engineering, Graduate School of Information Science and Technology, Osaka University, Suita, Osaka, Japan"/>
      <meta name="citation_author" content="Itaru Urabe"/>
            <meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Engineering, Osaka University, Suita, Osaka, Japan"/>
      <meta name="citation_author" content="Kunihiko Kaneko"/>
            <meta name="citation_author_institution" content="Graduate School of Frontier Biosciences, Osaka University, Suita, Osaka, Japan"/>
            <meta name="citation_author_institution" content="Department of Pure and Applied Sciences, The University of Tokyo, Tokyo, Japan"/>
            <meta name="citation_author_institution" content="Complex Systems Biology Project, Exploratory Research for Advanced Technology, Japan Science and Technology Agency, The University of Tokyo, Tokyo, Japan"/>
      <meta name="citation_author" content="Tetsuya Yomo"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics Engineering, Graduate School of Information Science and Technology, Osaka University, Suita, Osaka, Japan"/>
            <meta name="citation_author_institution" content="Graduate School of Frontier Biosciences, Osaka University, Suita, Osaka, Japan"/>
            <meta name="citation_author_institution" content="Complex Systems Biology Project, Exploratory Research for Advanced Technology, Japan Science and Technology Agency, Osaka University, Suita, Osaka, Japan"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000049.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e49"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_author=SA Kauffman; citation_number=1; citation_pages=734; citation_date=1993; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Construction of a genetic toggle switch in Escherichia coli.; citation_author=TS Gardner; citation_author=CR Cantor; citation_author=JJ Collins; citation_journal_title=Nature; citation_volume=403; citation_number=2; citation_pages=339-342; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Positive feedback in eukaryotic gene networks: cell differentiation by graded to binary response conversion.; citation_author=A Becskei; citation_author=B Seraphin; citation_author=L Serrano; citation_journal_title=Embo J; citation_volume=20; citation_number=3; citation_pages=2528-2535; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multistability in the lactose utilization network of Escherichia coli.; citation_author=EM Ozbudak; citation_author=M Thattai; citation_author=HN Lim; citation_author=BI Shraiman; citation_author=A Van Oudenaarden; citation_journal_title=Nature; citation_volume=427; citation_number=4; citation_pages=737-740; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Enhancement of cellular memory by reducing stochastic transitions.; citation_author=M Acar; citation_author=A Becskei; citation_author=A van Oudenaarden; citation_journal_title=Nature; citation_volume=435; citation_number=5; citation_pages=228-232; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prediction and measurement of an autoregulatory genetic module.; citation_author=FJ Isaacs; citation_author=J Hasty; citation_author=CR Cantor; citation_author=JJ Collins; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=6; citation_pages=7714-7719; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multistability: a major means of differentiation and evolution in biological systems.; citation_author=M Laurent; citation_author=N Kellershohn; citation_journal_title=Trends Biochem Sci; citation_volume=24; citation_number=7; citation_pages=418-422; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Isologous diversification: a theory of cell differentiation.; citation_author=K Kaneko; citation_author=T Yomo; citation_journal_title=Bull Math Biol; citation_volume=59; citation_number=8; citation_pages=139-196; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Sympatric speciation: compliance with phenotype diversification from a single genotype.; citation_author=K Kaneko; citation_author=T Yomo; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=267; citation_number=9; citation_pages=2367-2373; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cell fates as high-dimensional attractor states of a complex gene regulatory network.; citation_author=S Huang; citation_author=G Eichler; citation_author=Y Bar-Yam; citation_author=DE Ingber; citation_journal_title=Phys Rev Lett; citation_volume=94; citation_number=10; citation_pages=128701; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=K Kaneko; citation_number=11; citation_pages=371; citation_date=2006; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_author=G Forgacs; citation_author=SA Newman; citation_number=12; citation_pages=337; citation_date=2005; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_author=B Albert; citation_author=A Jhonson; citation_author=J Lewis; citation_author=M Raff; citation_author=P Walter; citation_number=13; citation_pages=1616; citation_date=2002; citation_publisher=Garland Publishing; " />
      <meta name="citation_reference" content="citation_title=Shape-dependent control of cell growth, differentiation, and apoptosis: switching between attractors in cell regulatory networks.; citation_author=S Huang; citation_author=DE Ingber; citation_journal_title=Exp Cell Res; citation_volume=261; citation_number=14; citation_pages=91-103; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A synthetic oscillatory network of transcriptional regulators.; citation_author=MB Elowitz; citation_author=S Leibler; citation_journal_title=Nature; citation_volume=403; citation_number=15; citation_pages=335-338; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Engineered gene circuits.; citation_author=J Hasty; citation_author=D McMillen; citation_author=JJ Collins; citation_journal_title=Nature; citation_volume=420; citation_number=16; citation_pages=224-230; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Use of the tetracycline promoter for the tightly regulated production of a murine antibody fragment in Escherichia coli.; citation_author=A Skerra; citation_journal_title=Gene; citation_volume=151; citation_number=17; citation_pages=131-135; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=FACS-optimized mutants of the green fluorescent protein (GFP).; citation_author=BP Cormack; citation_author=RH Valdivia; citation_author=S Falkow; citation_journal_title=Gene; citation_volume=173; citation_number=18; citation_pages=33-38; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Random mutagenesis of glutamine synthetase from Escherichia coli: Correlation between structure, activity, and fitness.; citation_author=W-Z Xu; citation_author=J Fukuhara; citation_author=K Yamamoto; citation_author=T Yomo; citation_author=I Urabe; citation_journal_title=Journal of Fermentation and Bioengineering; citation_volume=77; citation_number=19; citation_pages=252-258; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Tightly regulated tac promoter vectors useful for the expression of unfused and fused proteins in Escherichia coli.; citation_author=E Amann; citation_author=B Ochs; citation_author=KJ Abel; citation_journal_title=Gene; citation_volume=69; citation_number=20; citation_pages=301-315; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Rapidly maturing variants of the Discosoma red fluorescent protein (DsRed).; citation_author=BJ Bevis; citation_author=BS Glick; citation_journal_title=Nat Biotechnol; citation_volume=20; citation_number=21; citation_pages=83-87; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Kinetics of the formation and isomerization of methotrexate complexes of recombinant human dihydrofolate reductase.; citation_author=JR Appleman; citation_author=N Prendergast; citation_author=TJ Delcamp; citation_author=JH Freisheim; citation_author=RL Blakley; citation_journal_title=J Biol Chem; citation_volume=263; citation_number=22; citation_pages=10304-10313; citation_date=1988; " />
      <meta name="citation_reference" content="citation_author=FM Ausubel; citation_author=R Brent; citation_author=RE Kingston; citation_author=DD Moore; citation_author=JG Seidman; citation_number=23; citation_pages=600; citation_date=1988; citation_publisher=John Wiley & Sons, Inc; " />
      <meta name="citation_reference" content="citation_author=P Gerhardt; citation_author=RGE Murray; citation_author=WA Wood; citation_author=NR Krieg; citation_number=24; citation_pages=791; citation_date=1994; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Differentiation plasticity of hematopoietic cells.; citation_author=T Graf; citation_journal_title=Blood; citation_volume=99; citation_number=25; citation_pages=3089-3101; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=In vivo degradation of mutant lac repressor.; citation_author=T Platt; citation_author=JH Miller; citation_author=K Weber; citation_journal_title=Nature; citation_volume=228; citation_number=26; citation_pages=1154-1156; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Degradation of the DNA-binding domain of wild-type and i-d lac repressors in Escherichia coli.; citation_author=M Schlotmann; citation_author=K Beyreuther; citation_journal_title=Eur J Biochem; citation_volume=95; citation_number=27; citation_pages=39-49; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Noise in transcription negative feedback loops: simulation and experimental analysis.; citation_author=Y Dublanche; citation_author=K Michalodimitrakis; citation_author=N Kummerer; citation_author=M Foglierini; citation_author=L Serrano; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=28; citation_pages=41; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ultrasensitivity and noise propagation in a synthetic transcriptional cascade.; citation_author=S Hooshangi; citation_author=S Thiberge; citation_author=R Weiss; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=29; citation_pages=3581-3586; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Stochastic gene expression in a single cell.; citation_author=MB Elowitz; citation_author=AJ Levine; citation_author=ED Siggia; citation_author=PS Swain; citation_journal_title=Science; citation_volume=297; citation_number=30; citation_pages=1183-1186; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Noise-based switches and amplifiers for gene expression.; citation_author=J Hasty; citation_author=J Pradines; citation_author=M Dolnik; citation_author=JJ Collins; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=31; citation_pages=2075-2080; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Noise in protein expression scales with natural protein abundance.; citation_author=A Bar-Even; citation_author=J Paulsson; citation_author=N Maheshri; citation_author=M Carmi; citation_author=E O'Shea; citation_journal_title=Nat Genet; citation_volume=38; citation_number=32; citation_pages=636-643; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dominance of Milnor Attractors and Noise-Induced Selection in a Multiattractor System.; citation_author=K Kaneko; citation_journal_title=Phys Rev Lett; citation_volume=78; citation_number=33; citation_pages=2736-2739; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Externally controlled attractor selection in a high-dimensional system.; citation_author=FH Willeboordse; citation_author=K Kaneko; citation_journal_title=Phys Rev E; citation_volume=72; citation_number=34; citation_pages=026207-026213; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene regulation at the single-cell level.; citation_author=N Rosenfeld; citation_author=JW Young; citation_author=U Alon; citation_author=PS Swain; citation_author=MB Elowitz; citation_journal_title=Science; citation_volume=307; citation_number=35; citation_pages=1962-1965; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ubiquity of log-normal distributions in intra-cellular reaction dynamics.; citation_author=C Furusawa; citation_author=T Suzuki; citation_author=A Kashiwagi; citation_author=T Yomo; citation_author=K Kaneko; citation_journal_title=BIOPHYSICS; citation_volume=1; citation_number=36; citation_pages=25-31; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=37; citation_pages=1659; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Construction and fine mapping of recombinant plasmids containing the rrnB ribosomal RNA operon of E. coli.; citation_author=J Brosius; citation_author=A Ullrich; citation_author=MA Raker; citation_author=A Gray; citation_author=TJ Dull; citation_journal_title=Plasmid; citation_volume=6; citation_number=38; citation_pages=112-118; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Studies on transformation of Escherichia coli with plasmids.; citation_author=D Hanahan; citation_journal_title=J.Mol Biol; citation_volume=166; citation_number=39; citation_pages=557-580; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.; citation_author=KA Datsenko; citation_author=BL Wanner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=40; citation_pages=6640-6645; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Plasticity of fitness and diversification process during an experimental molecular evolution.; citation_author=A Kashiwagi; citation_author=W Noumachi; citation_author=M Katsuno; citation_author=MT Alam; citation_author=I Urabe; citation_journal_title=J Mol Evol; citation_volume=52; citation_number=41; citation_pages=502-509; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Numerical Recipes. Cambridge: Cambridge University Press. 848 p; citation_author=WH Press; citation_author=SA Teukolsky; citation_author=WT Vetterling; citation_author=BP Flannery; citation_number=42; citation_date=1986; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000049" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection"/>
    <meta name="twitter:description" content="Cells switch between various stable genetic programs (attractors) to accommodate environmental conditions. Signal transduction machineries efficiently convey environmental changes to the gene regulation apparatus in order to express the appropriate genetic program. However, since the number of environmental conditions is much larger than that of available genetic programs so that the cell may utilize the same genetic program for a large set of conditions, it may not have evolved a signaling pathway for every environmental condition, notably those that are rarely encountered. Here we show that in the absence of signal transduction, switching to the appropriate attractor state expressing the genes that afford adaptation to the external condition can occur. In a synthetic bistable gene switch in Escherichia coli in which mutually inhibitory operons govern the expression of two genes required in two alternative nutritional environments, cells reliably selected the &ldquo;adaptive attractor&rdquo; driven by gene expression noise. A mathematical model suggests that the &ldquo;non-adaptive attractor&rdquo; is avoided because in unfavorable conditions, cellular activity is lower, which suppresses mRNA metabolism, leading to larger fluctuations in gene expression. This, in turn, renders the non-adaptive state less stable. Although attractor selection is not as efficient as signal transduction via a dedicated cascade, it is simple and robust, and may represent a primordial mechanism for adaptive responses that preceded the evolution of signaling cascades for the frequently encountered environmental changes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000049.g006"/>

  <meta property="og:title" content="Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000049" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4255'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4334'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8818&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000049">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000049" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000049&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Akiko Kashiwagi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Bioinformatics Engineering, Graduate School of Information Science and Technology, Osaka University, Suita, Osaka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Itaru Urabe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biotechnology, Graduate School of Engineering, Osaka University, Suita, Osaka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kunihiko Kaneko, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Graduate School of Frontier Biosciences, Osaka University, Suita, Osaka, Japan, 
                  Department of Pure and Applied Sciences, The University of Tokyo, Tokyo, Japan, 
                  Complex Systems Biology Project, Exploratory Research for Advanced Technology, Japan Science and Technology Agency, The University of Tokyo, Tokyo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tetsuya Yomo
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:yomo@ist.osaka-u.ac.jp">yomo@ist.osaka-u.ac.jp</a>
            </p>

                <p>Affiliations:
                  Department of Bioinformatics Engineering, Graduate School of Information Science and Technology, Osaka University, Suita, Osaka, Japan, 
                  Graduate School of Frontier Biosciences, Osaka University, Suita, Osaka, Japan, 
                  Complex Systems Biology Project, Exploratory Research for Advanced Technology, Japan Science and Technology Agency, Osaka University, Suita, Osaka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000049</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000049" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000049" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000049" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000049" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000049-g001" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000049-g002" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000049-g003" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000049-g004" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000049-g005" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000049-g006" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000049">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000049" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Cells switch between various stable genetic programs (attractors) to accommodate environmental conditions. Signal transduction machineries efficiently convey environmental changes to the gene regulation apparatus in order to express the appropriate genetic program. However, since the number of environmental conditions is much larger than that of available genetic programs so that the cell may utilize the same genetic program for a large set of conditions, it may not have evolved a signaling pathway for every environmental condition, notably those that are rarely encountered. Here we show that in the absence of signal transduction, switching to the appropriate attractor state expressing the genes that afford adaptation to the external condition can occur. In a synthetic bistable gene switch in <em>Escherichia coli</em> in which mutually inhibitory operons govern the expression of two genes required in two alternative nutritional environments, cells reliably selected the “adaptive attractor” driven by gene expression noise. A mathematical model suggests that the “non-adaptive attractor” is avoided because in unfavorable conditions, cellular activity is lower, which suppresses mRNA metabolism, leading to larger fluctuations in gene expression. This, in turn, renders the non-adaptive state less stable. Although attractor selection is not as efficient as signal transduction via a dedicated cascade, it is simple and robust, and may represent a primordial mechanism for adaptive responses that preceded the evolution of signaling cascades for the frequently encountered environmental changes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kashiwagi A, Urabe I, Kaneko K, Yomo T (2006) Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection. PLoS ONE 1(1):
          e49.
            doi:10.1371/journal.pone.0000049</p><p><strong>Academic Editor: </strong>Sui Huang, Children's Hospital Boston, United States of America</p><p><strong>Received:</strong> August 28, 2006; <strong>Accepted:</strong> October 11, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> © 2006 Kashiwagi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by Grants-in-Aid (Nos. 11CE2006, 15207020, and 15013235), “The 21st Century Center of Excellence Program” and “Special Coordination Funds for Promoting Science and Technology: Yuragi Project” of the Ministry of Education, Culture, Sports, Science and Technology, Japan, and by a grant from Takeda Science Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Cells alter their gene expression in response to environmental changes or external signals to switch between coherent genetic programs in order to produce a phenotypic state, among many available, that best copes with the new environment. It is increasingly becoming clear that such genetic programs represent attractor states: discrete stable states of gene expression patterns generated by the dynamics of the regulatory interactions between the genes <a href="#pone.0000049-Kauffman1">[1]</a>. Small gene networks with mutual regulation of genes can generate multiple attractor states (multistability) and have recently been studied in synthetic <a href="#pone.0000049-Gardner1">[2]</a>, <a href="#pone.0000049-Becskei1">[3]</a> or in natural gene networks <a href="#pone.0000049-Ozbudak1">[4]</a>, <a href="#pone.0000049-Acar1">[5]</a>. These studies have elucidated dynamic properties, including multistability, switch-like behavior, memory effect, oscillation, and robustness in the presence of molecular noise <a href="#pone.0000049-Gardner1">[2]</a>, <a href="#pone.0000049-Becskei1">[3]</a>, <a href="#pone.0000049-Isaacs1">[6]</a>. The presence of multiple attractors has fundamental biological significance, notably in cell differentiation and sympatric speciation <a href="#pone.0000049-Laurent1">[7]</a>–<a href="#pone.0000049-Kaneko3">[11]</a>. Specifically, individual attractor states have been suggested to correspond to particular functional cell states or cell types in metazoan <a href="#pone.0000049-Kauffman1">[1]</a>, <a href="#pone.0000049-Huang1">[10]</a>, <a href="#pone.0000049-Forgacs1">[12]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Given the existence of distinct, stable gene expression programs epitomized by the attractor, the question is how cells switch into the appropriate attractor that is commensurate with the environmental condition. For instance, if the nutritional situation requires expression of gene A, how do cells switch into the attrator state in which A is stably expressed? The existing paradigm is that cells have evolved a signal transduction machinery to sense the environmental change and transmit it to the gene regulatory network. In the simplest case, such as in bacteria, the environmental signal may be a metabolite that directly regulates the transcriptional complex that controls the operon involved in its utilization, such as in the case of the lactose operon <a href="#pone.0000049-Albert1">[13]</a>. In more complex systems, membrane receptor proteins sense environmental changes and trigger a cascade of intracellular molecular events involving “second messengers” or protein modifcation cascades that lead to concerted changes in the expression of several genes. Such molecular signal transduction machineries may have evolved to allow cells to respond rapidly and specifically to frequently occurring changes in the extracellular environment.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>However, since the space of environmental conditions is much larger than that of cellular response programs, there is not a program for each condition, and cells need to choose the optimal program for a given condition. Thus, many environmental conditions map onto the same cellular response. Therefore, it is unlikely that cells have evolved a specific signal transduction pathway for every environment it may encounter as in the case of lactose utilization. In fact, infrequently occurring environmental conditions or unspecific, physical perturbations devoid of molecular specificity can evoke specific cellular programs, such as proliferation, quiescence or apoptosis <a href="#pone.0000049-Huang2">[14]</a>. Some attractor states may provide the optimal gene expression program for the cell to adapt to and cope with a particular, rare environment, yet no specific signaling pathway may exist that connects this rare external condition with the appropriate genetic program. This raises the question: how do cells in the case of rarely occurring environmental changes switch to the adaptive attractor state of the network that expresses the appropriate genes?</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here we use an artificial network with mutually inhibitory operons in <em>E. coli</em>, to show that cells possess an inherent ability to adaptively respond to environmental changes by selecting a gene expression state which allows for higher cell activity, without the help of a signal transduction apparatus. Unlike previous work in which genetic switches were linked to fluorescent proteins that report the state of the network <a href="#pone.0000049-Gardner1">[2]</a>, <a href="#pone.0000049-Isaacs1">[6]</a>, <a href="#pone.0000049-Elowitz1">[15]</a>, we link the bistable switch also to enzymes with phenotypic properties that interact with the environment. We show that cells can select the adaptive attractor and propose that this selection process is a general consequence of the stochastic nature of gene network dynamics. When the network state reaches an attractor that is adaptive, cells exhibit high cellular activity, increasing the turnover rates of mRNAs. This in turn supresses the influence of gene expression noise. In contrast, for the non-adaptive attractor, which accordingly has low cellular activity, the metabolic rate is smaller, and hence, noise overwhelms the deterministic component of the dynamics of the newtork. This causes the cell to be kicked out of the non-adaptive attractor. Thus, the synergism between (i) the deterministic bistable gene expression dynamics, (ii) cellular activity that depends on the match beween gene expression and environmental condition and (iii) stochastic fluctuations in the level of low-abundance molecules, allows the adaptive attractor to be selected.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Structure and Properties of the Network with Mutually Inhibitory Operons</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To study adaptive responses to environmental changes without signal transduction machinery, we constructed a synthetic gene network that can exhibit bistability and monitored its ability to switch between two stable attractor states in response to evnironmental change. Gene regulatory circuits that are based on mutually inhibiting operons have previously been reported <a href="#pone.0000049-Gardner1">[2]</a>. However, unlike existing work on synthetic networks (for example, <a href="#pone.0000049-Gardner1">[2]</a>, <a href="#pone.0000049-Hasty1">[16]</a>), for our purposes, the network need to express not only state-reporting fluorescence proteins, but also proteins with selectable phenotypes. To obtain a bistable gene network that is linked to metabolic phenotype switching we used the following two mutually inhibitory operons (construct pALL7, <a href="#pone-0000049-g001">Figure 1A</a>). Operon1 is composed of the <em>tetA</em> promoter <a href="#pone.0000049-Skerra1">[17]</a>, Lac Repressor gene (<em>lacI</em>) <a href="#pone.0000049-Albert1">[13]</a>, green fluorescence protein (GFP) gene (<em>egfp</em>) <a href="#pone.0000049-Cormack1">[18]</a> and a mutant glutamine synthetase (GLS-H) gene (<em>gls-h</em>; the original post-translational regulation was eliminated) <a href="#pone.0000049-Xu1">[19]</a>. Operon2 is composed of the <em>trc</em> promoter <a href="#pone.0000049-Amann1">[20]</a>, Tet Repressor gene (<em>tetR</em>) <a href="#pone.0000049-Skerra1">[17]</a>, red fluorescence protein (RFP) gene (<em>dsred.T4</em>) <a href="#pone.0000049-Bevis1">[21]</a>, and the mouse dihydrofolate reductase (mDHFR) gene (<em>mdhfr</em>) <a href="#pone.0000049-Appleman1">[22]</a>. The two fluorescent proteins are introduced to monitor the transcriptional level of the two operons. GLS-H and mDHFR confer a metabolic phenotype to each of the two attractor, since these enzymes compensate for two distinct conditions of nutrient depletion.</p>
<div class="figure" id="pone-0000049-g001"><div class="img"><a name="pone-0000049-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The Plasmid Structures of the Mutual Inhibition Network</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) The structure of the mutually inhibitory operons in pALL7.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p2" name="article1.body1.sec2.sec1.fig1.caption1.p2"></a><p>t<sub>0</sub> and <em>rrn</em>BT<sub>1</sub>T<sub>2</sub> terminators terminated transcription from Operon 1 and Operon 2, respectively.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p3" name="article1.body1.sec2.sec1.fig1.caption1.p3"></a><p>(B) Summary of the phenotype characteristics of pALL7.</p>
<span>doi:10.1371/journal.pone.0000049.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>As a basic property of the circuit architecture, the cells harboring pALL7 can be in a monostable and a bistable behavioral regimes, depending on culture conditions. (<a href="#pone-0000049-g001">Figure 1B</a> and <a href="#pone-0000049-g002">2</a>). After several serial overnight cultures in Medium N (see Experimental procedures), which does not impose restrictions on essential nutrients, the cells proliferated sufficiently fast so that the gene products of the two operons, including the repressors, were kept low due to dilution. Consequently mutual inhibition was too weak and did not produce bistability. In this monostable regime cells were reproducibly distributed around low levels of expression for both operons (blue dots). Two-color flow cytometry analysis (<a href="#pone-0000049-g002">Figure 2</a>) indicate low levels of expression of Operon 1 (green fluorescence) and Operon 2 (red fluorescence) for each cell. We refer to this state of weak expression from both operons as Attractor W, which is expected to appear at the low total concentration of the two repressors relative to the dissociation constants for their interacton with their promoter binding sites.</p>
<div class="figure" id="pone-0000049-g002"><div class="img"><a name="pone-0000049-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The Three Possible Attractors Generated by the Mutual Inhibition Network under Different Conditions</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Blue dots represent the expression pattern for cells cultured overnight in Medium N, while purple dots represent that for cells that were transferred from Medium N to Medium N plus nalidixic acid and cultured for 3 days until the number of cells increased sufficiently for flow cytometric analysis.</p>
<a id="article1.body1.sec2.sec1.fig2.caption1.p2" name="article1.body1.sec2.sec1.fig2.caption1.p2"></a><p>The diagonal correlation for purple dots was not related to any relationship of GFP and RFP expression, but was attributable to the leakage of fluorescence of GFP and RFP to the 600 nm dichroic filter and the band-pass filter at 525 nm±25 nm, respectively.</p>
<span>doi:10.1371/journal.pone.0000049.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>In contrast, when the total concentration of gene products is high, the network was shifted into the bistable regime, exhibiting the two Attractors 1 and 2, as expected (<a href="#pone-0000049-g002">Figure 2</a>); either strong expression of Operon 1 with strong repression of Operon 2 (Attractor 1), or vice versa (Attractor 2). This was achieved by slowing down cell proliferation using 170 µg/ml nalidixic acid <a href="#pone.0000049-Ausubel1">[23]</a>, <a href="#pone.0000049-Gerhardt1">[24]</a> which reduces the specific growth rate by 30~40%. In this bistable regime, the balanced state of Attractor W is no longer stable. When either of the operons happens to express its encoded repressor at a slightly higher level than the other, the other operon is slightly suppressed, which in turn decreases the concentration of the repressor for the former operon, leading to a further increase in expression of the former. Thus, the cells are tipped into either of the two self-stabilizing Attractors 1 and 2, as originally intended by the circuit design.</p>


<h4>Shift in Gene Expression Specifically toward the Adaptive Attractors</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We next studied the “adaptive” response of the network to external changes by exposing the cells to culture conditions that require the presence of either enzyme (GLS-H or mDHFR) whose mutually exclusive expression is associated with the two attractors. Thus, we ask whether cells can find the “adaptive attractor” that copes with the nutrient condition (Scheme in <a href="#pone-0000049-g001">Figure 1B</a>). For this purpose, we used two environmental conditions to implement the respective nutrient depletion: Medium M lacks glutamine, so that cells are required to syntheszie it to keep up cellular activity. Cells carrying pALL7 can overcome glutamine depletion if glutamine synthetase (GLS-H) in Operon 1 is stably expressed, that is, when they are in Attractor 1. Conversely, Medium T consists of Medium N plus trimethoprim lactate, which causes tetrahydrofolate depletion <a href="#pone.0000049-Appleman1">[22]</a> in the host cells. In this case the host cells carrying pALL7 can overcome tetrahydrofolate depletion if mDHFR in Operon 2 is expressed, which is active when cells are in Attractor 2. In summary, the Attractor 1 (2) is adaptive in mdeium M (T), respectively.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We put the cells carrying pALL7 to around Attractor W in the monostable regime by overnight culture in Medium N and then transferred them to either Medium M or T supplemented with 170 µg/ml nalidixic acid. In the flow-cytometry measurements, we observed that each cell underwent a unidirectional shift in gene expression pattern toward the attractor expressing the adaptive enzyme in response to respective nutrient depletion (<a href="#pone-0000049-g003">Figure 3A</a>). To monitor the time evolution of the shift we plotted the number of cells per unit volume against the green fluorescence/red fluorescence (G/R) ratio which reflects relative transcriptional activities from the two operons. At 0.5 h after transfer to either Medium M or T, the cells initially retained gene expression patterns, staying around Attractor W. Thereafter, in Medium M the distribution of cells shifted toward Attractor 1. This attractor is adaptive in medium M since glutamine synthetase in Operon 1 is expressed and compensates for the glutamine depletion in this medium. Conversely, the population transferred to Medium T was pulled toward Attractor 2. This shift is also adaptive since the high mDHFR expression compensates for tetrahydrofolate depletion in Medium T. Note that the control transfer to Medium N supplemented with the same concentration of nalidixic acid resulted in a stochastic shift of gene expression toward that of either Attractor 1 or 2 (<a href="#pone-0000049-g002">Figure 2</a>). Therefore, the nutrient depletions in Medium M and T caused a selective, unidirectional shift toward the corresponding adaptive attractors.</p>
<div class="figure" id="pone-0000049-g003"><div class="img"><a name="pone-0000049-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The Initial Time Course of Adaptive Response in Gene Expression</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Temporal shifts in the distribution of gene expression.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p2" name="article1.body1.sec2.sec2.fig1.caption1.p2"></a><p>After transfer to Medium M cells were sampled at 0.5 h (green dash-double-dotted line), 2 h (green solid line), 5 h (green broken line), and 7.5 h (green dash-dotted line).</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p3" name="article1.body1.sec2.sec2.fig1.caption1.p3"></a><p>After transfer to Medium T cells were sampled at 0.5 h (red dash-double-dotted line), 3h (red solid line), 5 h (red broken line), and 7.5 h (red dash-dotted line).</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p4" name="article1.body1.sec2.sec2.fig1.caption1.p4"></a><p>Green and red fluorescence of each cell was measured by flow cytometry. By taking the ratio between the two fluorescence intensities, the influence of the diagonal correlation with the slope of 1 shown in <a href="#pone-0000049-g002">Figure 2</a> could be cancelled.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p5" name="article1.body1.sec2.sec2.fig1.caption1.p5"></a><p>(B) Temporal changes in the number of cells showing the noted fluorescence ratios.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p6" name="article1.body1.sec2.sec2.fig1.caption1.p6"></a><p>The temporal change of the cell number at the denoted ratio on the x axis in Medium M from 0.5 h to 2 h (green solid line) was calculated by subtracting the cell distribution of (A) at 2 h from that at 0.5 h.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p7" name="article1.body1.sec2.sec2.fig1.caption1.p7"></a><p>Similarly, the temporal changes in Medium M were calculated from 2 h to 5 h (green broken line), and from 5 h to 7.5 h (green dash-dotted line).</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p8" name="article1.body1.sec2.sec2.fig1.caption1.p8"></a><p>The temporal changes in Medium T were calculated from 0.5 h to 3 h (red solid line), from 3 h to 5 h (red broken line), and from 5 h to 7.5 h (red dash-dotted line).</p>
<span>doi:10.1371/journal.pone.0000049.g003</span></div>

<h4>Adaptive Response to Changing Environments</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We next examined how cells adjust their gene expression adaptively to fluctuating environments. Cells carrying pALL7 were subjected to serial overnight culture with sequential medium changes in two different orders (<a href="#pone-0000049-g004">Figure 4A and 4B</a>). In these experiments, nalidixic acid was omitted from all three media to minimize the risk of unexpected genomic rearrangements during long-term cultivation. When cultured in Medium N, most of the cells exhibited a neutral position of G/R, <em>i.e.</em>, were in Attractor W. Following transfer to Medium T (<a href="#pone-0000049-g004">Figure 4A</a>), the cells displayed a low G/R, indicating strong expression of Operon 2, encoding mDHFR, which compensates for the depletion of tetrahydrofolate in this medium. After return to Medium N, the cells again displayed the neutral position. The cells showed successive changes in G/R: neutral→high→neutral, when the medium was changed in the order N→M→N; consistently, the high G/R ratio here indicated expression from Operon 1, which encodes glutamine synthetase that compensates for the lack of glutamine in Medium M. Further transfer to Medium T caused the G/R ratio to shift back to the low value, indicating that the cells retained the ability to adjust their gene expression adaptively to Medium T. When the medium was changed in a different order (<a href="#pone-0000049-g004">Figure 4B</a>), the same three distributions appeared in response to the three media.</p>
<div class="figure" id="pone-0000049-g004"><div class="img"><a name="pone-0000049-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Attractor Selection in Changing Environments</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>
                     <em>E. coli</em> OSU1 cells with pALL7 were subjected to serial overnight culture with an inoculum size of 6×10<sup>7</sup> cells/l every day in changing environments.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p2" name="article1.body1.sec2.sec3.fig1.caption1.p2"></a><p>(A) Days 1–5 in Medium N, days 6–7 in Medium T, days 8–10 in Medium N, days 11–13 in Medium M, days 14–15 in Medium N, and day 16 in Medium T.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p3" name="article1.body1.sec2.sec3.fig1.caption1.p3"></a><p>On the last day of serial overnight cultures in the same medium, the cells were subjected to flow cytometric analysis.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p4" name="article1.body1.sec2.sec3.fig1.caption1.p4"></a><p>(B) Days 1–5 in Medium N, days 6–7 in Medium M, days 8–9 in Medium T, days 10–11 in Medium N, days 12–13 in Medium M, and day 14 in Medium T.</p>
<span>doi:10.1371/journal.pone.0000049.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Microscopic analysis confirmed that at the single cell level these reproducible distributions with the peaks of high, neutral, and low G/R correspond to Attractor 1, Attractor W, and Attractor 2, respectively (<a href="#pone-0000049-g005">Figure 5</a>). The cells in Medium N expressed GFP and RFP weakly, indicating Attractor W. On the other hand, cells in Medium T showed strong expression of RFP and repression of GFP compared to those in Medium N, indicating Attractor 2. Similarly, cells in Medium M exhibited strong expression of GFP and repression of RFP, consistent with Attractor 1. The alternative appearance of the three attractors in each medium was also supported by mRNA quantification; the average numbers of mRNA copies transcribed from Operon 1 and Operon 2 per cell were 10.0 and “undetectable”, respectively, in Medium M, whereas they were “undetectable” and 6.0, respectively, in Medium T, and “undetectable” and 0.3, respectively, in Medium N. These results clearly show that the cells chose the adaptive attractors reproducibly in fluctuating environment.</p>
<div class="figure" id="pone-0000049-g005"><div class="img"><a name="pone-0000049-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Microscopic Examination of Cells Cultured Overnight in Three Different Media</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>GFP and RFP fluorescence of cells in Medium M, N, and T were examined using an Olympus IX70 microscope and a KEYENCE VB-6010 CCD camera.</p>
<a id="article1.body1.sec2.sec3.fig2.caption1.p2" name="article1.body1.sec2.sec3.fig2.caption1.p2"></a><p>For the GFP channel, a BA470-490 excitation filter, DM505 dichromatic beam splitter, and BA515-550 emission filter were used.</p>
<a id="article1.body1.sec2.sec3.fig2.caption1.p3" name="article1.body1.sec2.sec3.fig2.caption1.p3"></a><p>For the RFP channel, a BA520-550 excitation filter, DM565 dichromatic beam splitter, and BA580IF emission filter were used.</p>
<a id="article1.body1.sec2.sec3.fig2.caption1.p4" name="article1.body1.sec2.sec3.fig2.caption1.p4"></a><p>The exposure time and the sensitivity expressed as ISO values are shown in the pictures.</p>
<a id="article1.body1.sec2.sec3.fig2.caption1.p5" name="article1.body1.sec2.sec3.fig2.caption1.p5"></a><p>Scale bar, 5 µm.</p>
<span>doi:10.1371/journal.pone.0000049.g005</span></div>

<h4>The Response Is Not Due to Proliferation of Fit Cells</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>A fundamental question of cell state switching is whether (i) an external signal that causes a commitment to the expression of a particular phenotype does so by somehow instructing the gene transcription apparatus to express the appropriate (set of) gene(s) in all cells, or (ii) the signal merely promotes survival and expansion of the few cells that “happen” to express that desired phenotype <a href="#pone.0000049-Graf1">[25]</a>. In other words, if the switching is only due to the scenario (ii), the two attractors are selected with equal probability first but only the cells in the adaptative attractor will proliferate, causing a shift in the population distribution. We found that the scenario (ii) alone cannot explain the observed macroscopic shift toward adaptive attractor, but the scenario (i) indeed is necessary, as demonstrated by careful monitoring of the time course in which cells redistribute between each of attractors, as explianed in the following.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>At 0.5–2 h, when the total cell concentration had hardly increased as indicated by the minimal increase in the total areas under the curve (AUC) of distribution (<a href="#pone-0000049-g003">Figure 3A</a>), the cell distribution began to shift toward the adaptive attractor. To better represent the shift in population (change in histogram of cell number for a given G/R ratio) <a href="#pone-0000049-g003">Figure 3B</a> shows the increment in cell number within each bin in the histogram up to each sampling time, obtained by subtratcing the cell number from that at the previous sampling time point in <a href="#pone-0000049-g003">Figure 3A</a>. In Medium M, the subpopulation of cells with high G/R ratio clearly increased within the time period from 0.5 to 2 h (1.5×10<sup>7</sup> cells/l in the G/R ratio between 10<sup>0.3</sup> and 10), while that with low G/R ratio decreased (1.2×10<sup>7</sup> cells/l in the G/R ratio between 10<sup>−0.5</sup>and 10<sup>0.3</sup>). The small difference in the total AUC between the downward peak for cells with low G/R ratio and the upward peak for cells with high G/R ratio indicates little cell growth from 0.5 to 2 h. Since nutrient starvation for less than 5 h caused little cell rupture (data not shown), the two allmost equal AUCs of downward and upward peaks indicate that each cell underwent a unidirectional alteration in its gene expression pattern from Attractor W toward Attractor 1 during this time period. Taken together, this kinetic analysis suggests that the observed unidirectional shift until 2 h can be attributed not to the prolification of cells randomely tipped toward the adaptive attractor (scenario (ii)) but to a deterministic change in gene expresssion of each cell (scenario (i)).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>In contrast, for the period from 2 to 5 h, the change in gene expression toward Attractor 1 and the growth of cells at high G/R occurred simultaneously, as indicated by the smaller total area of the downward peak for cells with low G/R ratio compared with that of the upward peak for those with high G/R ratio. From 5 to 7.5 h, as indicated by the small downward peak for cells with the low G/R ratio, the change in gene expression toward Attractor 1 was almost complete, while cells with the high G/R ratio continued to grow. Thus, the time course analysis indicates that after exposure to Medium M the cells that happened to have been tipped toward the adaptive attractor at the beginning were pulled to these attractors and started growing in the favorable condition (scenario (ii)) while the other cells that happened to have been tipped to the non-adaptive attractor escaped from them and switched to the adaptive attractor in a scenario (i) process. Upon exposure to Medium T, the cells showed the same time course of attractor selection but in the opposite direction. These unidirectional changes in gene expression suggest that cells are capable of response adaptively to environmental changes through attractor selection. Thus, within the limits of measuring the time course of gene expression for each cell, both mechanisms appear to play a role. It remains to be determined to what extent each of two scenarios contribute to the observed adaptive attractor selection.</p>


<h4>Fitness-Induced Gene Expression without Signal Transduction Machinery</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Given that the mechanism of regulation of selecting the attractor cannot be solely explained by selective growth advantage of randomly swicthed cells, one may suspect that adaptive attractor selection in response to nutrient depletion may be due to some direct but ‘hidden’ instructive signal transduction that may somehow be “hard-wired” in the host cell's genome, connecting nutrient situation to the respective cis-region of the two metabolic genes. To exclude this possibility, we carried out a ‘promoter swap’ experiment. The <em>mdhfr</em> and <em>gls-h</em> genes were swapped in the pALL7 plasmids, so that the same regulatory circuit was maintained (plasmid pALL8, <a href="#pone.0000049.s003">Figure S1A</a>). This new plasmid was introduced into the same host cells, <em>E. coli</em> OSU1. The transformed cells were subjected to a series of overnight cultures in essentially the same media with a slight modification (legend of <a href="#pone.0000049.s003">Figure S1</a>). As shown in <a href="#pone.0000049.s003">Figure S1B</a>, attractor selection took place in the same way with the mutually inhibitory network of the pALL8 plasmid as with the pALL7 plasmid but only with the difference that with pALL8, the correlation between the media and the strongly expressed operon was the reverse of that observed with pALL7.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We did not observe a non-adaptive response with pALL8, in that for instance, depletion of tetrahydrofolate would have activated Operon 2, which contains the promoter that drives mDHFR in pALL7 but glutamin synthetase in pALL8. Thus, the promoter swap experiments corroborate the principle that expression of operons was dependent on the fitness conveyed by the encoded enzyme to the environmental condition and was not due to some unexpected fixed functional relation (such as an unknown signaling pathway) between the DNA sequence of these nominally generic promoter regions and the environmental signal.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>In summary, we have shown experimentally that by introducing a gene network with bistable attractors expressing phenotypes that are sensitve to the environment, most of the cells changed the attractor state and associated gene expression in response to the environment, so as to mount an adaptive response by producing the enzymes necessary to compensate for the nutrient depletion.</p>


<h4>Theoretical Model for Adaptive Attractor Selection</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>How can we understand the above adaptive attractor selection without specific signal transduction machinery? Here, we propose that such selection is a rather general process allowing cells to grow by selecting and maintaining stochastic gene expression patterns. The basis for the alternative expression patterns in the present gene network is a deterministic bistable system. Thus, we propose here first a model for bistability for our network that links attractor state with a metabolic phenotype.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We analyzed a standard model of mutually inhibitory operons which can generate bistability <a href="#pone.0000049-Gardner1">[2]</a>, but introduced terms to capture the phenotypic consequence of a stable state, which determines the adaptation to an environment, and a noise term:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>m</em>1 and <em>m</em>2 are the concentrations of the mRNAs or their protein products (which are here lumped together into one term since the phenotype switch of the population occurs at a slower time scale than transcription), transcribed from Operon1 and Operon2, respectively. <em>S</em>(<em>A</em>) and <em>D</em>(<em>A</em>) are the rate coefficients of synthesis and degradation and/or dilution due to the cell volume growth, respectively. Importantly, they depend on <em>A</em>, which represents cellular activity. <em>η</em>
               <sub>1</sub> and <em>η</em>
               <sub>2</sub> represent independent white noise in gene expression.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>By setting the equations (1) without noise to d<em>m</em>/d<em>t</em> = 0, one can obtain the fixed-point solutions for the system. This yields a single attractor of <em>m</em>1* = <em>m</em>2* for <em>S/D&lt;0.5</em> (corresponding to Attractor W in the monostable regime in the experiment in the absence of nalidixic acid). For <em>S</em>/<em>D</em>&gt;0.5, there are two attractors satisfying (<em>m</em>1 = <em>m</em>*, <em>m</em>2 = 1/<em>m</em>*) and (<em>m</em>1 = 1/<em>m</em>*, <em>m</em>2 = <em>m</em>*) (see Supporting Information). Without the noise term, the initial condition with <em>m</em>1&gt;<em>m</em>2 is attracted to the attractor located in the region of <em>m</em>1&gt;<em>m</em>2 and vice versa.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>To capture the phenotypic consequence that allows adaptation we introduce the variable <em>A</em> = “cellular activity”, which is increased when cells approach the adaptive attractor, expressing the genes that allows survival and optimal growth in a given environment. It is not easy to define “cellular activity” on a material basis, as it is a complex function of the concentrations of ATP and other chemicals. However, cellular activity can be lumped together and represented as the variable <em>A</em> that increases monotonously with cell growth rate, particularly for unicellular organisms.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>The first important postulate in the model is that both synthesis <em>S</em>(<em>A</em>) and degradation and/or dilution <em>D</em>(<em>A</em>) are increasing functions of activity <em>A</em> (albeit in distinct ways) which in turn is correlated with the nutrition condition and growth rate. Since the rate of metabolic processes increase with activity <em>A</em>, the amplitudes of the synthesis rate of mRNA or its protein product, <em>S</em>(<em>A</em>), is expected to increase with <em>A</em>. As the degradation rates of LacI and TetR are much smaller than the specific growth rate of cells in this work <a href="#pone.0000049-Platt1">[26]</a>–<a href="#pone.0000049-Hooshangi1">[29]</a>, the amplitude of <em>D</em>(<em>A</em>) is mostly determind by the dilution due to cell growth and thus increases with <em>A</em>.</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>The different attractors show different activities. The adaptive attractor has higher activity, i.e., larger values of <em>S</em>(<em>A</em>) and <em>D</em>(<em>A</em>). In other words, the deterministic part of Eq. (1) (i.e., all terms without the noise terms) take larger (smaller) values for an adaptive (non-adaptive) attractor, respectively. (Recall <em>S</em>(<em>A</em>) and <em>D</em>(<em>A</em>) are increasing functions of the activity <em>A</em>).</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p>We now discuss selection of adaptive attractor. First we note that such selection of attractors is not possible in the above deterministic dynamics alone. However, gene expression is always accompanied with considerable random fluctuations, or noise <a href="#pone.0000049-Elowitz2">[30]</a>, represented by the term <em>η</em> which can account for noise-driven transitions between attractor states <a href="#pone.0000049-Hasty2">[31]</a> .</p>
<a id="article1.body1.sec2.sec6.p8" name="article1.body1.sec2.sec6.p8"></a><p>The second basic postulate in the model is that the noise amplitude is independent of the activity <em>A</em>, or at least it does not vanish with the decrease of the activity. Specifically, in a first approximation we assume a constant amplitude of noise. However, the specific form of noise in the Langevin Eq. (1) is not important, as long as a considerable amount of noise remains when the network is in the non-adaptive attractors, even if noise strength may depend on (<em>m</em>1, <em>m</em>2), or activity to some extent.</p>
<a id="article1.body1.sec2.sec6.p9" name="article1.body1.sec2.sec6.p9"></a><p>In general, for noise arising from chemical reactions associated with the activity, its strength should increase with the activity. However, even if the activity vanishes, fluctuation in gene expression may remain due to basal (housekeeping) biochemical processes in the cell. Indeed, recent measurement of noise on a variety of environmental conditions suggests that considerable amount of noise remains independent of the growth speed of a cell that is correlated with cellular activity <a href="#pone.0000049-BarEven1">[32]</a>. Hence, it is natural to assume that some part of noise is activity independent, even though the actual form of noise strength is difficult to predict at the present stage. In our experiment, we found that considerable noise remains even for a cells in the “non-adaptive condition” where they are in a state with low activity. When cells located around Attractor 1 in Medium M were transferred to Medium T (<a href="#pone-0000049-g004">Figure 4B</a>), most of the cells moved to Attractor 2, while a small fraction of cells remained at Attractor 1. The observation that the latter cells, though they had very low activity around the non-adaptive attractor, maintained a similar scattering in fluorescence intensity (<a href="#pone.0000049.s004">Figure S2B</a>,) suggests that noise is present even in states with low activity. On the other hand, we found that the relative contribution of noise to the deterministic metabolic rate decreased as cells approached the adaptive attractors. In <a href="#pone-0000049-g003">Figure 3A</a>, the distribution of the G/R ratio became narrower as cells shifted their gene expression toward the adaptive attractors. The relative variances in G/R ratio (estimated as full width at half-maximum of the histogram) decreased by 72% (in Medium T) or 57% (in Medium M) by the time of 7.5 h, suggesting that the relative magnitude of the noise term decreased while that of the deterministic metabolic term became large with increasing activity.</p>
<a id="article1.body1.sec2.sec6.p10" name="article1.body1.sec2.sec6.p10"></a><p>We now describe how, given the above two basic postulates, the adaptive attractor is selected. Let us first consider the case in which an environmental change causes a marked decrease in <em>A</em> because the gene expression pattern is inappropriate (i.e., the cell is in the non-adaptive attractor). Then, the deterministic metabolic rate in Eq. (1) will be so small that it will approach the same magnitude as that of the noise term, <em>η</em>. The dynamics of gene expression will therefore be dominated mostly by random fluctuations. This is true as long as the network is in a region with the state of low activity. On the other hand, when the network moves into a region with high acttivity <em>A</em>, <em>i.e.</em>, to the adaptive attractor, then the metabolic rate inceases, and the deterministic part becomes much larger than the noise term. Consequently, the dynamics of the system will be governed by the deterministic part of Eq. (1). Thus, whichever gene expression state the network starts with, gene expression will continue to fluctuate until it arrives at the adaptive attractor which is more stable against the noise because of the relatively lager metabolic rate of the first and second terms in Eq. (1).</p>
<a id="article1.body1.sec2.sec6.p11" name="article1.body1.sec2.sec6.p11"></a><p>With a simple form of <em>D</em>(<em>A</em>) and <em>S</em>(<em>A</em>), the above dynamics for attractor selection was confirmed by numerical simulation of Eq. (1) (<a href="#pone-0000049-g006">Figure 6</a>). (for details on the mathematical model, see Supporting Information).</p>
<div class="figure" id="pone-0000049-g006"><div class="img"><a name="pone-0000049-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000049" data-uri="info:doi/10.1371/journal.pone.0000049.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000049.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000049.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000049.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000049.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Simulation of Dynamics of Networks with Mutually Inhibitory Operons</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Black, green, and red indicate activity <em>A</em>, and the expression levels at the two operons, <em>m</em>1 and <em>m</em>2, respectively.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p2" name="article1.body1.sec2.sec6.fig1.caption1.p2"></a><p>Culture conditions were captured by <em>N</em>1, <em>N</em>2 (see Supporting Information). We employed a changing environment: (<em>N</em>1, <em>N</em>2) = (10,10) from 0 to 2000 step, (0,10) to 8000, (10,10) to 10000, (10, 0) to 16000, (10,10) to 20000.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p3" name="article1.body1.sec2.sec6.fig1.caption1.p3"></a><p>The parameters used in this simulation were: <em>S</em>(<em>A</em>) = 6<em>A</em>/(2+<em>A</em>) and <em>D</em>(<em>A</em>) = <em>A</em>, <em>N</em>_<em>thr</em>
                     <sub>1</sub> = <em>N</em>_<em>thr</em>
                     <sub>2</sub> = 2, <em>P</em> = <em>C</em> = 0.01, and <em>n</em>1 = <em>n</em>2 = 5.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p4" name="article1.body1.sec2.sec6.fig1.caption1.p4"></a><p>Initially, the environment contained both nutrients at sufficient levels from <em>t</em> = 0 to 2000, and activity <em>A</em> was less than but very close to 1.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p5" name="article1.body1.sec2.sec6.fig1.caption1.p5"></a><p>Then, following the change in the environment causing depletion of <em>N</em>1 = 0 but with <em>N</em>2 = 10, the system started to lose the activity, which destabilized the Attractor W, and entered the regime with two attractors, where the attractor with higher levels of <em>m</em>1 is selected.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p6" name="article1.body1.sec2.sec6.fig1.caption1.p6"></a><p>When returned to the nutrient-rich environment at <em>t</em> = 8000, the system came back to the single attractor with <em>m</em>1 = <em>m</em>2 = 1.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p7" name="article1.body1.sec2.sec6.fig1.caption1.p7"></a><p>Then, the environmental conditions were changed to depletion of the other nutrient, <em>N</em>1 = 10, <em>N</em>2 = 0 at <em>t</em> = 10000, and the system first deviated to express more of <em>m</em>1 but less <em>m</em>2.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p8" name="article1.body1.sec2.sec6.fig1.caption1.p8"></a><p>This deviation to the side of the non-adaptive attractor occurred by chance, but the small value of <em>m</em>2 did not compensate for the nutrient depletion and the activity is reduced close to zero.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p9" name="article1.body1.sec2.sec6.fig1.caption1.p9"></a><p>Then, the noise term in equations (1) took over in influence on the system.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p10" name="article1.body1.sec2.sec6.fig1.caption1.p10"></a><p>This fluctuation decreased only when <em>m</em>2 happened to rise sufficiently to allow activity to increase again at around <em>t</em> = 11000.</p>
<span>doi:10.1371/journal.pone.0000049.g006</span></div><a id="article1.body1.sec2.sec6.p12" name="article1.body1.sec2.sec6.p12"></a><p>The present scheme of adaptive attractor selection generally works for a system with dx/d<em>t</em> = f(x, <em>A</em>)+<em>η</em>, with f(x, <em>A</em>) = {f<sub>1</sub>(x, <em>A</em>),f<sub>2</sub>(x, <em>A</em>),…} where f<sub>i</sub>(x, <em>A</em>) is the metabolic rate of mRNA or protein for gene i. In general, if |f(x, <em>A</em>)| increases with <em>A</em>, the attractor with small <em>A</em> is disturbed by noise so that an attratcor with high <em>A</em> is selected.</p>
<a id="article1.body1.sec2.sec6.p13" name="article1.body1.sec2.sec6.p13"></a><p>Switching between attractors has been studied in dynamical systems as noise-induced selection of attractors <a href="#pone.0000049-Kaneko4">[33]</a>, <a href="#pone.0000049-Willeboordse1">[34]</a>. In these previous studies, however, the direction of switching was not adaptive. In contrast, in the case presented here, due to the coupling of the attractor states with the cellular activity which influences the rate of the turnover of the deterministic state variables, an adaptive attractor with its higher cellular activity is inevitably selected in the presence of noise.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Perspective for Genomic Networks</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We propose here the concept of attractor selection based on experimental and theoretical studies on a simple network with mutually inhibitory operons in which the network autonomously chooses the appropriate gene expression pattern associated with the adaptive attractor in response to environmental changes that are not mediated by a cognate signal transduction machinery. This mechanism for environmental adaptation of gene expression crucially depends on the ratio of the metabolic rate (mRNA, protein turnover) controlled by the regulatory circuit and on the presence of noise in gene expression. The latter is relatively high compared to the deterministic regulation when cells are in unfavorable conditions so that the overall metabolic regulation is weak due to low cellular activity. Thus, the ratio of (deterministic) metabolic regulation relative to noise is reduced or increased around non-adaptive or adaptive attractors, respectively. As long as this correlation is satisfied, network structures other than the simple network investigated here will be able to respond to environmental changes through attractor selection if they possess multiple attractors [theoretically confirmed by Furusawa and Kaneko, unpublished]. Thus, noise-driven selection between two alternative attractor states differs from previously studied decision between functional states, such as glucose/lactose utilization in <em>E. coli</em> or lysogeny/lysis in lambda phages, in that there is no explicit, specific molecular link between proteins of the gene network and an external signal (metabolite as allosteric regulator of a key regulatory protein; UV light, etc.).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Our simple regulatory circuit utilizes common promoters and their repressors. However, the noise in gene expression from plasmids does not appear to differ markedly from that of the genomic genes <a href="#pone.0000049-Rosenfeld1">[35]</a>, <a href="#pone.0000049-Furusawa1">[36]</a>. The existence of attractor selection in genome-encoded networks should be investigated in future studies, although the response of the whole genomic network in terms of attractor selection will be not as simple as that described here as it will involve combinations of many molecular mechanisms, including canonical signal transduction pathways.</p>


<h4>Comparison between Signal Transduction and Attractor Selection</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Signal transduction is a basic mechanism for cellular responses to environmental changes, which is based on complex networks that connect the environment to genome. Each signal transduction network must have evolved through natural selection by responding to environmental changes that have occurred frequently in the past. Signaling pathways then have been conserved because they allow the host cells to adapt rapidly to specific environmental changes.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>However, in addition to such frequently occurring environmental changes, unicellular organisms may encounter a large number of different starvation conditions each of which may appear only a few times during their lifespan. In addition, cells may not have evolved an attractor state to every environmental condition, yet some may confer better adaptation than others. It is then hard to imagine that the cells would have evolved and maintained specific signal transduction machineries for all possible rare environmental changes so as to map every environment to an optimal attractor, because of the high cost of maintaining all these signaling machineries. Even for frequently occurring environmental changes, attractor selection can be beneficial for cells since it requires no specific signal transduction apparatus, if speed of state transition is not an issue.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Due to its stochastic nature, attractor selection may not be as efficient as ordinary signal transduction, but it may prevent cells from dying in fluctuating environments. Molecular signal transduction may have evolved for efficient adaptation, but in view of our present results it is plausible that attractor selection may still be utilized as a primitive adaptation for sustainability in broadly changing environments. Attractor selection may facilitate the design of a network that can robustly respond in an adaptive manner to unknown environmental changes without requiring a large number of specific sensors and transducers. It may also be viewed as a sort of Darwinian preadaptation for the evolution of signal-specific transduction pathways when a particular new environmental condition becomes dominant and hence contributes to evolvability.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plasmid construction</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All plasmids were constructed using standard molecular cloning techniques <a href="#pone.0000049-Sambrook1">[37]</a>. Genes and promoters were obtained from the following sources: Ptrc from pTrc99A (Amersham Biosciences); PtetA from pASK-IBA3 (Sigma Genosys); <em>lacI</em> from pTrc99A; <em>tetR</em> from pcDNA6/TR (Invitrogen); <em>egfp</em> from pEGFP (BD Biosciences Clontech); <em>dsRed.T4</em> (a gift from Dr. B. S. Glick, The University of Chicago) <a href="#pone.0000049-Bevis1">[21]</a>; <em>mdhfr</em> from pQE16 (Qiagen); and <em>gls-h</em> from pKGN-H <a href="#pone.0000049-Xu1">[19]</a>. The plasmid contained the ColE1 origin of replication and ampicillin resistance gene. The t0 terminator from pPROTet.E333-lacZ (BD Biosciences Clontech) and <em>rrn</em>BT<sub>1</sub>T<sub>2</sub> terminator <a href="#pone.0000049-Brosius1">[38]</a> from pTrc99A were used to terminate transcription from Operon 1 and Operon 2, respectively.</p>


<h4>Strain, media, and growth conditions</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>
               <em>E. coli</em> strain OSU1, a derivative of DH1 <a href="#pone.0000049-Hanahan1">[39]</a>, was constructed by replacing the <em>glnA</em> gene with the <em>cat</em> gene by homologous recombination <a href="#pone.0000049-Datsenko1">[40]</a>. For cells carrying pALL7, the following three media were employed. 90 mg/l ampicillin was added to Medium C to prepare Medium M <a href="#pone.0000049-Kashiwagi1">[41]</a>. Medium N consisted of Medium M plus 90 µM glutamine, which compensates for the lack of the <em>glnA</em> gene in the genome of OSU1. Medium T was Medium N plus 580 mg/l of trimethoprim lactate, which completely inhibits bacterial DHFR and requires a high level of expression of mDHFR for survival of <em>E. coli</em> cells. To match the effective strength of PtetA to that of Ptrc, anhydrotetracycline, which inhibits Tet repressor, was added to the three media at equal concentrations of 0.5 µg/l. All culture steps were conducted at 37°C with aeration after inoculation of 6×10<sup>7</sup> cells/l, unless otherwise noted. The number of cells was measured using a Particle Analyzer SD-2000 (SYSMEX). The cells from each population were stored at −80°C before flow cytometric analysis.</p>


<h4>Flow cytometric analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The samples stored at −80°C were thawed at 25°C in a water bath and kept on ice before analysis. The single-cell fluorescence measurements were carried out on a COULTER<sup>®</sup> EPICS® ELITE Flow Cytometer (Beckman Coulter) with a 488-nm Argon excitation laser and band-pass filters of 525±25 nm for green fluorescence and a 600 nm dichroic filter for red fluorescence. All flow event data were converted to text format using WinMDI Version 2.8 and analyzed with MATHEMATICA 5.0 (Wolfram Research) and DeltaGraph Version 5.0.1.</p>


<h4>mRNA measurement</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Total bacterial RNA was prepared with RNeasy kits (Qiagen). Northern blotting and hybridization were performed according to standard methods and visualized with the AlkPhos Direct™ Labeling and Detection System (Amersham Biosciences). Double-stranded <em>lacI</em> and <em>dsred.T4</em> DNAs were used as probes for transcripts from Operon 1 and Operon 2, respectively. Hybridization signals for the mRNAs from Operon 1 and Operon 2 were quantified using ImageJ 1.29.</p>


<h4>Model analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The numerical simulation of the Langevin equation (1) was conducted by the Runge-Kutta method using a program developed in Microsoft Visual C++, as well as by FORTRAN program using algorithms in Numerical Recipes <a href="#pone.0000049-Press1">[42]</a>. The noise term was calculated as an independent Gaussian random number. We confirmed the accuracy by reducing the time grid for integration.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000049"></div><a name="pone.0000049.s002" id="pone.0000049.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000049.s002">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supporting text for this paper.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000049.s002</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000049.s003" id="pone.0000049.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000049.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Adaptive Response of the Network in Cells with pALL8. (A) The structure of pALL8 is the same as pALL7 shown in <a href="#pone-0000049-g001">Figure 1</a> except for the exchanged positions of <em>mdhfr</em> and <em>gls-h</em>. Ptrc and PtetA represent the <em>trc</em> promoter and <em>tetA</em> promoter, respectively. (B) The adaptive shifts in gene expression of pALL8 that are opposite to those of pALL7. Cells carrying pALL8 were subjected to a series of overnight cultures in the same manner as in <a href="#pone-0000049-g004">Figure 4</a> with a slightly modified media as follows. As trimethoprim lactate at the concentration used in the original Medium T completely suppressed the growth of cells, it was diluted to 3 mg/l. To further support the cells with trimethoprim lactate, we increased the anhydrotetracycline concentration from 0.5 µg/l to 0.8 µg/l to achieve higher levels of expression of mDHFR (Medium T′). The same concentration of anhydrotetracycline was used in the other two media (Medium M′ and N′) for accurate comparison. Cells grown in Medium N′ were subjected to a series of overnight cultures with an inoculum size of 7×107 cells/l every day in Medium N′ for 4 days (blue), in Medium T′ for 4 days (green), and in Medium M′ for 7 days (red) in the same way as described in <a href="#pone-0000049-g004">Figure 4</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000049.s003</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000049.s004" id="pone.0000049.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000049.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Flow Cytometric Analysis of Attractor Selection by Changing Environments. (A) The original data from the flow cytometry results shown in <a href="#pone-0000049-g004">Figure 4A</a>. (B) The original data for <a href="#pone-0000049-g004">Figure 4B</a>. For each culture, 10,000 events were collected. The weak positive correlation observed in Medium T was due to leakage of the red fluorescence to the green fluorescence gate and was not related to GFP expression. Note that when cells located around Attractor 1 in Medium M were transferred to Medium T (upper right panel in B), a small fraction of cells stayed in this attractor and exhibited the same scattering of fluorescence intensity, indicating gene expression noise in this non-adaptive state.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000049.s004</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0000049.s005" id="pone.0000049.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000049.s005">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Basic Characteristic of the Model in Equation (1). Phase space spanned by <em>m</em>1 and <em>m</em>2 with nullclines, d<em>m</em>1/d<em>t</em> = 0 or d<em>m</em>2/d<em>t</em> = 0 in Eq. (1), calculated for (<em>N</em>1,<em>N</em>2) = (0,10) in Eq. (2), by adopting S(<em>A</em>) = 6<em>A</em>/(2+<em>A</em>) and <em>D</em>(<em>A</em>) = <em>A</em> (solid lines) with the parameter values <em>P</em> = <em>C</em> = 0.01, <em>N</em>_thr<sub>1</sub> = <em>N</em>_thr<sub>2</sub> = 2, <em>n</em>
                  <sub>1</sub> = <em>n</em>
                  <sub>2</sub> = 5. The separatrix (dashed line) is given by <em>m</em>1 = <em>m</em>2. The fixed-point attractors are given by the intersections of the two solid lines (filled circles).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000049.s005</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0000049.s006" id="pone.0000049.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000049.s006">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Bifurcation Diagram of the Model with Equations (1) and (2). Bifurcation diagram for d<em>m</em>1/d<em>t</em> = 0 and d<em>m</em>2/dt = 0 in Eq. (1) for the same condition as in <a href="#pone.0000049.s005">Figure S3</a> (solid line). Since <em>m</em>2 = 1/<em>m</em>1 for <em>A</em>&lt;1 and <em>m</em>2 = <em>m</em>1 for <em>A</em>&gt;1 are adopted as steady-state(s), only <em>m</em>1 (horizontal axis) is indicated as a function of change of the parameter <em>A</em> (vertical axis). The bifurcation point (<em>m</em>1,<em>m</em>2) = (1, 1), as well as the adaptive attractor with large and the non-adaptive attractor with small are represented by the three filled circles. The dependency of <em>A</em> on (steady-state) is computed from the condition d<em>A</em>/d<em>t</em> = 0 in Eq. (2) (broken line) where <em>m</em>2 = 1/<em>m</em>1 is adopted as <em>A</em>&lt;1.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000049.s006</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000049.s001" id="pone.0000049.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000049.s001">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Dependence of the Fraction of Selection of Adaptive Attractor on the Noise Strength for the Model (1)–(2). The frequency the cell is in the adaptive attractor is plotted against the standard deviation of the noise per single time step. For each noise strength, 10,000 runs of simulation were performed by changing the environment at the time step 2,000, while the frequency of adaptive attractor selection is computed from the number of runs that ended with <em>m</em>1&gt;<em>m</em>2 at the time step 20,000. The vertical axis represents the ratio of such runs to the total number of runs, 10,000.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000049.s001</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.25 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are indebted to Dr. Barry L. Wanner (Purdue University) and Dr. Nancy Kleckner (Harvard University) for their help in constructing the host cells for the mutually inhibitory network. We also thank Dr. Chikara Furusawa, Dr. Satoshi Sawai, Dr. Katsuhiko Sato and Dr. Tsuyoshi Chawanya for fruitful discussion.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TY AK. Performed the experiments: AK. Analyzed the data: TY AK. Wrote the paper: TY KK. Other: Overviewed the manuscript: IU.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000049-Kauffman1" id="pone.0000049-Kauffman1"></a>Kauffman SA (1993) The origin of order. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000049-Gardner1" id="pone.0000049-Gardner1"></a>Gardner TS, Cantor CR, Collins JJ (2000) Construction of a genetic toggle switch in <em>Escherichia coli</em>. Nature  403: 339–342.  <ul class="find" data-citedArticleID="916886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Construction+of+a+genetic+toggle+switch+in+Escherichia+coli.&amp;auth=&amp;atitle=Construction+of+a+genetic+toggle+switch+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+a+genetic+toggle+switch+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+a+genetic+toggle+switch+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000049-Becskei1" id="pone.0000049-Becskei1"></a>Becskei A, Seraphin B, Serrano L (2001) Positive feedback in eukaryotic gene networks: cell differentiation by graded to binary response conversion. Embo J  20: 2528–2535.  <ul class="find" data-citedArticleID="916866" data-doi="10.1093/emboj/20.10.2528"><li><a href="http://dx.doi.org/10.1093/emboj/20.10.2528" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+feedback+in+eukaryotic+gene+networks%3A+cell+differentiation+by+graded+to+binary+response+conversion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+feedback+in+eukaryotic+gene+networks%3A+cell+differentiation+by+graded+to+binary+response+conversion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000049-Ozbudak1" id="pone.0000049-Ozbudak1"></a>Ozbudak EM, Thattai M, Lim HN, Shraiman BI, Van Oudenaarden A (2004) Multistability in the lactose utilization network of <em>Escherichia coli</em>. Nature  427: 737–740.  <ul class="find" data-citedArticleID="916920" data-doi="10.1038/nature02298"><li><a href="http://dx.doi.org/10.1038/nature02298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multistability+in+the+lactose+utilization+network+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multistability+in+the+lactose+utilization+network+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000049-Acar1" id="pone.0000049-Acar1"></a>Acar M, Becskei A, van Oudenaarden A (2005) Enhancement of cellular memory by reducing stochastic transitions. Nature  435: 228–232.  <ul class="find" data-citedArticleID="916854" data-doi="10.1038/nature03524"><li><a href="http://dx.doi.org/10.1038/nature03524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancement+of+cellular+memory+by+reducing+stochastic+transitions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancement+of+cellular+memory+by+reducing+stochastic+transitions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000049-Isaacs1" id="pone.0000049-Isaacs1"></a>Isaacs FJ, Hasty J, Cantor CR, Collins JJ (2003) Prediction and measurement of an autoregulatory genetic module. Proc Natl Acad Sci U S A  100: 7714–7719.  <ul class="find" data-citedArticleID="916904" data-doi="10.1073/pnas.1332628100"><li><a href="http://dx.doi.org/10.1073/pnas.1332628100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+and+measurement+of+an+autoregulatory+genetic+module." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+and+measurement+of+an+autoregulatory+genetic+module.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000049-Laurent1" id="pone.0000049-Laurent1"></a>Laurent M, Kellershohn N (1999) Multistability: a major means of differentiation and evolution in biological systems. Trends Biochem Sci  24: 418–422.  <ul class="find" data-citedArticleID="916918" data-doi="10.1016/s0968-0004(99)01473-5"><li><a href="http://dx.doi.org/10.1016/s0968-0004(99)01473-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multistability%3A+a+major+means+of+differentiation+and+evolution+in+biological+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multistability%3A+a+major+means+of+differentiation+and+evolution+in+biological+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000049-Kaneko1" id="pone.0000049-Kaneko1"></a>Kaneko K, Yomo T (1997) Isologous diversification: a theory of cell differentiation. Bull Math Biol  59: 139–196.  <ul class="find" data-citedArticleID="916906" data-doi="10.1007/bf02459474"><li><a href="http://dx.doi.org/10.1007/bf02459474" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isologous+diversification%3A+a+theory+of+cell+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isologous+diversification%3A+a+theory+of+cell+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000049-Kaneko2" id="pone.0000049-Kaneko2"></a>Kaneko K, Yomo T (2000) Sympatric speciation: compliance with phenotype diversification from a single genotype. Proc R Soc Lond B Biol Sci  267: 2367–2373.  <ul class="find" data-citedArticleID="916908" data-doi="10.1098/rspb.2000.1293"><li><a href="http://dx.doi.org/10.1098/rspb.2000.1293" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sympatric+speciation%3A+compliance+with+phenotype+diversification+from+a+single+genotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sympatric+speciation%3A+compliance+with+phenotype+diversification+from+a+single+genotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000049-Huang1" id="pone.0000049-Huang1"></a>Huang S, Eichler G, Bar-Yam Y, Ingber DE (2005) Cell fates as high-dimensional attractor states of a complex gene regulatory network. Phys Rev Lett  94: 128701.  <ul class="find" data-citedArticleID="916900" data-doi="10.1103/physrevlett.94.128701"><li><a href="http://dx.doi.org/10.1103/physrevlett.94.128701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+fates+as+high-dimensional+attractor+states+of+a+complex+gene+regulatory+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+fates+as+high-dimensional+attractor+states+of+a+complex+gene+regulatory+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000049-Kaneko3" id="pone.0000049-Kaneko3"></a>Kaneko K (2006) Life: An Introduction to Complex Systems Biology. Berlin: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0000049-Forgacs1" id="pone.0000049-Forgacs1"></a>Forgacs G, Newman SA (2005) Biological Physics of the Developing Embryo:. Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0000049-Albert1" id="pone.0000049-Albert1"></a>Albert B, Jhonson A, Lewis J, Raff M, Walter P (2002) MOLECULAR BIOLOGY OF THE CELL. New York: Garland Publishing.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000049-Huang2" id="pone.0000049-Huang2"></a>Huang S, Ingber DE (2000) Shape-dependent control of cell growth, differentiation, and apoptosis: switching between attractors in cell regulatory networks. Exp Cell Res  261: 91–103.  <ul class="find" data-citedArticleID="916902" data-doi="10.1006/excr.2000.5044"><li><a href="http://dx.doi.org/10.1006/excr.2000.5044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shape-dependent+control+of+cell+growth%2C+differentiation%2C+and+apoptosis%3A+switching+between+attractors+in+cell+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shape-dependent+control+of+cell+growth%2C+differentiation%2C+and+apoptosis%3A+switching+between+attractors+in+cell+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000049-Elowitz1" id="pone.0000049-Elowitz1"></a>Elowitz MB, Leibler S (2000) A synthetic oscillatory network of transcriptional regulators. Nature  403: 335–338.  <ul class="find" data-citedArticleID="916878" data-doi="10.1038/35002125"><li><a href="http://dx.doi.org/10.1038/35002125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+synthetic+oscillatory+network+of+transcriptional+regulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+synthetic+oscillatory+network+of+transcriptional+regulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000049-Hasty1" id="pone.0000049-Hasty1"></a>Hasty J, McMillen D, Collins JJ (2002) Engineered gene circuits. Nature  420: 224–230.  <ul class="find" data-citedArticleID="916894" data-doi="10.1038/nature01257"><li><a href="http://dx.doi.org/10.1038/nature01257" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineered+gene+circuits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineered+gene+circuits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000049-Skerra1" id="pone.0000049-Skerra1"></a>Skerra A (1994) Use of the tetracycline promoter for the tightly regulated production of a murine antibody fragment in <em>Escherichia coli</em>. Gene  151: 131–135.  <ul class="find" data-citedArticleID="916932" data-doi="10.1016/0378-1119(94)90643-2"><li><a href="http://dx.doi.org/10.1016/0378-1119(94)90643-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+the+tetracycline+promoter+for+the+tightly+regulated+production+of+a+murine+antibody+fragment+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+the+tetracycline+promoter+for+the+tightly+regulated+production+of+a+murine+antibody+fragment+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000049-Cormack1" id="pone.0000049-Cormack1"></a>Cormack BP, Valdivia RH, Falkow S (1996) FACS-optimized mutants of the green fluorescent protein (GFP). Gene  173: 33–38.  <ul class="find" data-citedArticleID="916872" data-doi="10.1016/0378-1119(95)00685-0"><li><a href="http://dx.doi.org/10.1016/0378-1119(95)00685-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FACS-optimized+mutants+of+the+green+fluorescent+protein+%28GFP%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FACS-optimized+mutants+of+the+green+fluorescent+protein+%28GFP%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000049-Xu1" id="pone.0000049-Xu1"></a>Xu W-Z, Fukuhara J, Yamamoto K, Yomo T, Urabe I (1994) Random mutagenesis of glutamine synthetase from <em>Escherichia coli</em>: Correlation between structure, activity, and fitness. Journal of Fermentation and Bioengineering  77: 252–258.  <ul class="find" data-citedArticleID="916936" data-doi="10.1016/0922-338x(94)90229-1"><li><a href="http://dx.doi.org/10.1016/0922-338x(94)90229-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Random+mutagenesis+of+glutamine+synthetase+from+Escherichia+coli%3A+Correlation+between+structure%2C+activity%2C+and+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Random+mutagenesis+of+glutamine+synthetase+from+Escherichia+coli%3A+Correlation+between+structure%2C+activity%2C+and+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000049-Amann1" id="pone.0000049-Amann1"></a>Amann E, Ochs B, Abel KJ (1988) Tightly regulated tac promoter vectors useful for the expression of unfused and fused proteins in <em>Escherichia coli</em>. Gene  69: 301–315.  <ul class="find" data-citedArticleID="916858" data-doi="10.1016/0378-1119(88)90440-4"><li><a href="http://dx.doi.org/10.1016/0378-1119(88)90440-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tightly+regulated+tac+promoter+vectors+useful+for+the+expression+of+unfused+and+fused+proteins+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tightly+regulated+tac+promoter+vectors+useful+for+the+expression+of+unfused+and+fused+proteins+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000049-Bevis1" id="pone.0000049-Bevis1"></a>Bevis BJ, Glick BS (2002) Rapidly maturing variants of the <em>Discosoma</em> red fluorescent protein (DsRed). Nat Biotechnol  20: 83–87.  <ul class="find" data-citedArticleID="916868" data-doi="10.1038/nbt0102-83"><li><a href="http://dx.doi.org/10.1038/nbt0102-83" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapidly+maturing+variants+of+the+Discosoma+red+fluorescent+protein+%28DsRed%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapidly+maturing+variants+of+the+Discosoma+red+fluorescent+protein+%28DsRed%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000049-Appleman1" id="pone.0000049-Appleman1"></a>Appleman JR, Prendergast N, Delcamp TJ, Freisheim JH, Blakley RL (1988) Kinetics of the formation and isomerization of methotrexate complexes of recombinant human dihydrofolate reductase. J Biol Chem  263: 10304–10313.  <ul class="find" data-citedArticleID="916860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Kinetics+of+the+formation+and+isomerization+of+methotrexate+complexes+of+recombinant+human+dihydrofolate+reductase.&amp;auth=&amp;atitle=Kinetics+of+the+formation+and+isomerization+of+methotrexate+complexes+of+recombinant+human+dihydrofolate+reductase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetics+of+the+formation+and+isomerization+of+methotrexate+complexes+of+recombinant+human+dihydrofolate+reductase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetics+of+the+formation+and+isomerization+of+methotrexate+complexes+of+recombinant+human+dihydrofolate+reductase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000049-Ausubel1" id="pone.0000049-Ausubel1"></a>Ausubel FM, Brent R, Kingston RE, Moore DD, Seidman JG, et al.  (1988) Current protocols in molecular biology;.  Chanda VB, editor. New York: John Wiley &amp; Sons, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000049-Gerhardt1" id="pone.0000049-Gerhardt1"></a>Gerhardt P, Murray RGE, Wood WA, Krieg NR (1994) Methods for General and Molecular Bacteriology. Washington, D.C.: ASM Press.   <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0000049-Graf1" id="pone.0000049-Graf1"></a>Graf T (2002) Differentiation plasticity of hematopoietic cells. Blood  99: 3089–3101.  <ul class="find" data-citedArticleID="916890" data-doi="10.1182/blood.v99.9.3089"><li><a href="http://dx.doi.org/10.1182/blood.v99.9.3089" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differentiation+plasticity+of+hematopoietic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differentiation+plasticity+of+hematopoietic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000049-Platt1" id="pone.0000049-Platt1"></a>Platt T, Miller JH, Weber K (1970) In vivo degradation of mutant lac repressor. Nature  228: 1154–1156.  <ul class="find" data-citedArticleID="916922" data-doi="10.1038/2281154a0"><li><a href="http://dx.doi.org/10.1038/2281154a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+degradation+of+mutant+lac+repressor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+degradation+of+mutant+lac+repressor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000049-Schlotmann1" id="pone.0000049-Schlotmann1"></a>Schlotmann M, Beyreuther K (1979) Degradation of the DNA-binding domain of wild-type and i-d lac repressors in Escherichia coli. Eur J Biochem  95: 39–49.  <ul class="find" data-citedArticleID="916930" data-doi="10.1111/j.1432-1033.1979.tb12937.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1979.tb12937.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degradation+of+the+DNA-binding+domain+of+wild-type+and+i-d+lac+repressors+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degradation+of+the+DNA-binding+domain+of+wild-type+and+i-d+lac+repressors+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000049-Dublanche1" id="pone.0000049-Dublanche1"></a>Dublanche Y, Michalodimitrakis K, Kummerer N, Foglierini M, Serrano L (2006) Noise in transcription negative feedback loops: simulation and experimental analysis. Mol Syst Biol  2: 41.  <ul class="find" data-citedArticleID="916876" data-doi="10.1038/msb4100081"><li><a href="http://dx.doi.org/10.1038/msb4100081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Noise+in+transcription+negative+feedback+loops%3A+simulation+and+experimental+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Noise+in+transcription+negative+feedback+loops%3A+simulation+and+experimental+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000049-Hooshangi1" id="pone.0000049-Hooshangi1"></a>Hooshangi S, Thiberge S, Weiss R (2005) Ultrasensitivity and noise propagation in a synthetic transcriptional cascade. Proc Natl Acad Sci U S A  102: 3581–3586.  <ul class="find" data-citedArticleID="916898" data-doi="10.1073/pnas.0408507102"><li><a href="http://dx.doi.org/10.1073/pnas.0408507102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrasensitivity+and+noise+propagation+in+a+synthetic+transcriptional+cascade." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrasensitivity+and+noise+propagation+in+a+synthetic+transcriptional+cascade.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000049-Elowitz2" id="pone.0000049-Elowitz2"></a>Elowitz MB, Levine AJ, Siggia ED, Swain PS (2002) Stochastic gene expression in a single cell. Science  297: 1183–1186.  <ul class="find" data-citedArticleID="916880" data-doi="10.1126/science.1070919"><li><a href="http://dx.doi.org/10.1126/science.1070919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+gene+expression+in+a+single+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+gene+expression+in+a+single+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000049-Hasty2" id="pone.0000049-Hasty2"></a>Hasty J, Pradines J, Dolnik M, Collins JJ (2000) Noise-based switches and amplifiers for gene expression. Proc Natl Acad Sci U S A  97: 2075–2080.  <ul class="find" data-citedArticleID="916896" data-doi="10.1073/pnas.040411297"><li><a href="http://dx.doi.org/10.1073/pnas.040411297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Noise-based+switches+and+amplifiers+for+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Noise-based+switches+and+amplifiers+for+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000049-BarEven1" id="pone.0000049-BarEven1"></a>Bar-Even A, Paulsson J, Maheshri N, Carmi M, O'Shea E, et al.  (2006) Noise in protein expression scales with natural protein abundance. Nat Genet  38: 636–643.  <ul class="find" data-citedArticleID="916864" data-doi="10.1038/ng1807"><li><a href="http://dx.doi.org/10.1038/ng1807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Noise+in+protein+expression+scales+with+natural+protein+abundance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Noise+in+protein+expression+scales+with+natural+protein+abundance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000049-Kaneko4" id="pone.0000049-Kaneko4"></a>Kaneko K (1997) Dominance of Milnor Attractors and Noise-Induced Selection in a Multiattractor System. Phys Rev Lett  78: 2736–2739.  <ul class="find" data-citedArticleID="916912" data-doi="10.1103/physrevlett.78.2736"><li><a href="http://dx.doi.org/10.1103/physrevlett.78.2736" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominance+of+Milnor+Attractors+and+Noise-Induced+Selection+in+a+Multiattractor+System." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dominance+of+Milnor+Attractors+and+Noise-Induced+Selection+in+a+Multiattractor+System.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000049-Willeboordse1" id="pone.0000049-Willeboordse1"></a>Willeboordse FH, Kaneko K (2005) Externally controlled attractor selection in a high-dimensional system. Phys Rev E  72: 026207–026213.  <ul class="find" data-citedArticleID="916934" data-doi="10.1103/physreve.72.026207"><li><a href="http://dx.doi.org/10.1103/physreve.72.026207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Externally+controlled+attractor+selection+in+a+high-dimensional+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Externally+controlled+attractor+selection+in+a+high-dimensional+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000049-Rosenfeld1" id="pone.0000049-Rosenfeld1"></a>Rosenfeld N, Young JW, Alon U, Swain PS, Elowitz MB (2005) Gene regulation at the single-cell level. Science  307: 1962–1965.  <ul class="find" data-citedArticleID="916926" data-doi="10.1126/science.1106914"><li><a href="http://dx.doi.org/10.1126/science.1106914" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+regulation+at+the+single-cell+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+regulation+at+the+single-cell+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000049-Furusawa1" id="pone.0000049-Furusawa1"></a>Furusawa C, Suzuki T, Kashiwagi A, Yomo T, Kaneko K (2005) Ubiquity of log-normal distributions in intra-cellular reaction dynamics. BIOPHYSICS  1: 25–31.  <ul class="find" data-citedArticleID="916884" data-doi="10.2142/biophysics.1.25"><li><a href="http://dx.doi.org/10.2142/biophysics.1.25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ubiquity+of+log-normal+distributions+in+intra-cellular+reaction+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ubiquity+of+log-normal+distributions+in+intra-cellular+reaction+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000049-Sambrook1" id="pone.0000049-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular Cloning A laboratory manual. New York: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0000049-Brosius1" id="pone.0000049-Brosius1"></a>Brosius J, Ullrich A, Raker MA, Gray A, Dull TJ, et al.  (1981) Construction and fine mapping of recombinant plasmids containing the rrnB ribosomal RNA operon of <em>E. coli</em>. Plasmid  6: 112–118.  <ul class="find" data-citedArticleID="916870" data-doi="10.1016/0147-619x(81)90058-5"><li><a href="http://dx.doi.org/10.1016/0147-619x(81)90058-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+and+fine+mapping+of+recombinant+plasmids+containing+the+rrnB+ribosomal+RNA+operon+of+E.+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+and+fine+mapping+of+recombinant+plasmids+containing+the+rrnB+ribosomal+RNA+operon+of+E.+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000049-Hanahan1" id="pone.0000049-Hanahan1"></a>Hanahan D (1983) Studies on transformation of <em>Escherichia coli</em> with plasmids. J.Mol Biol  166: 557–580.  <ul class="find" data-citedArticleID="916892" data-doi="10.1016/s0022-2836(83)80284-8"><li><a href="http://dx.doi.org/10.1016/s0022-2836(83)80284-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+on+transformation+of+Escherichia+coli+with+plasmids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+on+transformation+of+Escherichia+coli+with+plasmids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000049-Datsenko1" id="pone.0000049-Datsenko1"></a>Datsenko KA, Wanner BL (2000) One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci U S A  97: 6640–6645.  <ul class="find" data-citedArticleID="916874" data-doi="10.1073/pnas.120163297"><li><a href="http://dx.doi.org/10.1073/pnas.120163297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000049-Kashiwagi1" id="pone.0000049-Kashiwagi1"></a>Kashiwagi A, Noumachi W, Katsuno M, Alam MT, Urabe I, et al.  (2001) Plasticity of fitness and diversification process during an experimental molecular evolution. J Mol Evol  52: 502–509.  <ul class="find" data-citedArticleID="916914" data-doi="10.1007/s002390010180"><li><a href="http://dx.doi.org/10.1007/s002390010180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasticity+of+fitness+and+diversification+process+during+an+experimental+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasticity+of+fitness+and+diversification+process+during+an+experimental+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000049-Press1" id="pone.0000049-Press1"></a>Press WH, Teukolsky SA, Vetterling WT, Flannery BP (1986) Numerical Recipes. Cambridge: Cambridge University Press. 848 p.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.XML" value="117997"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.PDF" value="413199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g001.PNG_L" value="129854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g001.PNG_M" value="16955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g001.PNG_S" value="1807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g001.TIF" value="343174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g001.PNG_I" value="9105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g002.PNG_L" value="1866669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g002.PNG_M" value="219004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g002.PNG_S" value="5238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g002.TIF" value="2219080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g002.PNG_I" value="68966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g003.PNG_L" value="521932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g003.PNG_M" value="55242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g003.PNG_S" value="2866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g003.TIF" value="773454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g003.PNG_I" value="24875"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g004.PNG_L" value="822678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g004.PNG_M" value="109407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g004.PNG_S" value="3238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g004.TIF" value="1167458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g004.PNG_I" value="44034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g005.PNG_L" value="2045870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g005.PNG_M" value="295939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g005.PNG_S" value="7680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g005.TIF" value="2497784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.e001.PNG" value="4338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.e001.TIF" value="19776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g006.PNG_L" value="1953873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g006.PNG_M" value="234226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g006.PNG_S" value="6140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g006.TIF" value="2435818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.g006.PNG_I" value="70881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.s002.PDF" value="41850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.s003.PDF" value="14296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.s004.PDF" value="52727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.s005.PDF" value="28059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.s006.PDF" value="15360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000049.s001.PDF" value="250895"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000049&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000049" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000049&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000049&volume=&issue=&title=Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection&author_name=Akiko%20Kashiwagi%2C%20Itaru%20Urabe%2C%20Kunihiko%20Kaneko%2C%20Tetsuya%20Yomo&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000049" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000049" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000049" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000049&amp;t=Adaptive%20Response%20of%20a%20Gene%20Network%20to%20Environmental%20Changes%20by%20Fitness-Induced%20Attractor%20Selection" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000049&title=Adaptive%20Response%20of%20a%20Gene%20Network%20to%20Environmental%20Changes%20by%20Fitness-Induced%20Attractor%20Selection&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000049&amp;title=Adaptive%20Response%20of%20a%20Gene%20Network%20to%20Environmental%20Changes%20by%20Fitness-Induced%20Attractor%20Selection" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000049" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000049&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Adaptive Response of a Gene Network to Environmental Changes by Fitness-Induced Attractor Selection';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000049';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Adaptive%20Response%20of%20a%20Gene%20Network%20to%20Environmental%20Changes%20by%20Fitness-Induced%20Attractor%20Selection http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000049" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000049" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->

<div class="block">
    <div class="header">
        <h3>Included in the Following Collection</h3>
    </div>
    <p><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i34" title="Browse the Open-Access Collection">Software & Modeling</a></dd><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i31" title="Browse the Open-Access Collection">Gene Network Engineering</a></dd></p>
</div>

<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="22938"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="22938"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulatory+networks%22" title="Search for articles in the subject area:'Gene regulatory networks'"><div class="flagText">Gene regulatory networks</div></a>
              <div data-categoryid="17555" data-articleid="22938"
                   data-categoryname="Gene regulatory networks"
                   class="flagImage" title="Flag 'Gene regulatory networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glutamine%22" title="Search for articles in the subject area:'Glutamine'"><div class="flagText">Glutamine</div></a>
              <div data-categoryid="40885" data-articleid="22938"
                   data-categoryname="Glutamine"
                   class="flagImage" title="Flag 'Glutamine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Operons%22" title="Search for articles in the subject area:'Operons'"><div class="flagText">Operons</div></a>
              <div data-categoryid="21291" data-articleid="22938"
                   data-categoryname="Operons"
                   class="flagImage" title="Flag 'Operons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="22938"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+metabolism%22" title="Search for articles in the subject area:'Protein metabolism'"><div class="flagText">Protein metabolism</div></a>
              <div data-categoryid="19757" data-articleid="22938"
                   data-categoryname="Protein metabolism"
                   class="flagImage" title="Flag 'Protein metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signal+transduction%22" title="Search for articles in the subject area:'Signal transduction'"><div class="flagText">Signal transduction</div></a>
              <div data-categoryid="17357" data-articleid="22938"
                   data-categoryname="Signal transduction"
                   class="flagImage" title="Flag 'Signal transduction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8604'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1844'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3448&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=42995">Very interesting article</a><br>Posted by suchire</p>
      <p><a href="/annotation/listThread.action?root=45005">Re-thinking cellular adaptation</a><br>Posted by pedrobeltrao</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
