

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001715"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001715" />

    <meta name="citation_title" content="Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes"/>
    <meta itemprop="name" content="Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes"/>

      <meta name="citation_author" content="Daniel E. Rozen"/>
            <meta name="citation_author_institution" content="Department of Genetics, Wageningen University, Wageningen, The Netherlands"/>
            <meta name="citation_author_institution" content="University of Manchester, Manchester, United Kingdom"/>
      <meta name="citation_author" content="Michelle G. J. L. Habets"/>
            <meta name="citation_author_institution" content="Department of Genetics, Wageningen University, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Andreas Handel"/>
            <meta name="citation_author_institution" content="Department of Biology, Emory University, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="J. Arjan G. M. de Visser"/>
            <meta name="citation_author_institution" content="Department of Genetics, Wageningen University, Wageningen, The Netherlands"/>

    <meta name="citation_date" content="2008/3/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001715.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1715"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Limits to the adaptive potential of small populations.; citation_author=Y Willi; citation_author=J Van Buskirk; citation_author=AA Hoffmann; citation_journal_title=Annu Rev Ecol, Evol Syst; citation_volume=37; citation_number=1; citation_pages=433-458; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The fate of competing beneficial mutations in an asexual population.; citation_author=PJ Gerrish; citation_author=RE Lenski; citation_journal_title=Genetica; citation_volume=103; citation_number=2; citation_pages=127-144; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The speed of adaptation in large asexual populations.; citation_author=CO Wilke; citation_journal_title=Genetics; citation_volume=167; citation_number=3; citation_pages=2045-2053; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Diminishing returns from mutation supply rate in asexual populations.; citation_author=J de Visser; citation_author=CW Zeyl; citation_author=PJ Gerrish; citation_author=JL Blanchard; citation_author=RE Lenski; citation_journal_title=Science; citation_volume=283; citation_number=4; citation_pages=404-406; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Relationship between population size and fitness.; citation_author=DH Reed; citation_journal_title=Conserv Biol; citation_volume=19; citation_number=5; citation_pages=563-568; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Relation of Recombination to Mutational Advance.; citation_author=HJ Muller; citation_journal_title=Mutat Res; citation_volume=1; citation_number=6; citation_pages=2-9; citation_date=1964; " />
      <meta name="citation_reference" content="citation_title=Risk of population extinction from fixation of deleterious and reverse mutations.; citation_author=R Lande; citation_journal_title=Genetica; citation_volume=103; citation_number=7; citation_pages=21-27; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The Mutational Meltdown in Asexual Populations.; citation_author=M Lynch; citation_author=R Burger; citation_author=D Butcher; citation_author=W Gabriel; citation_journal_title=J Hered; citation_volume=84; citation_number=8; citation_pages=339-344; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Mutation Accumulation and the Extinction of Small Populations.; citation_author=M Lynch; citation_author=J Conery; citation_author=R Burger; citation_journal_title=Am Nat; citation_volume=146; citation_number=9; citation_pages=489-518; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Fitness of Rna Virus Decreased by Muller Ratchet.; citation_author=L Chao; citation_journal_title=Nature; citation_volume=348; citation_number=10; citation_pages=454-455; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Understanding the Evolutionary Fate of Finite Populations: The Dynamics of Mutational Effects.; citation_author=OK Silander; citation_author=O Tenaillon; citation_author=L Chao; citation_journal_title=PLoS Biol; citation_volume=5; citation_number=11; citation_pages=e94; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Genetic Theory of Natural Selection. Dover, New York USA.; citation_author=RA Fisher; citation_number=12; citation_date=1958; " />
      <meta name="citation_reference" content="citation_title=Some genetic aspects of sex.; citation_author=HJ Muller; citation_journal_title=Am Nat; citation_volume=66; citation_number=13; citation_date=1932; " />
      <meta name="citation_reference" content="citation_title=Evolution by small steps and rugged landscapes in the RNA virus phi 6.; citation_author=CL Burch; citation_author=L Chao; citation_journal_title=Genetics; citation_volume=151; citation_number=14; citation_pages=921-927; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Diminishing returns of population size in the rate of RNA virus adaptation.; citation_author=R Miralles; citation_author=A Moya; citation_author=SF Elena; citation_journal_title=J Virol; citation_volume=74; citation_number=15; citation_pages=3566-3571; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The population genetics of adaptation: The distribution of factors fixed during adaptive evolution.; citation_author=HA Orr; citation_journal_title=Evolution; citation_volume=52; citation_number=16; citation_pages=935-949; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Fitness effects of fixed beneficial mutations in microbial populations.; citation_author=DE Rozen; citation_author=J de Visser; citation_author=PJ Gerrish; citation_journal_title=Curr Biol; citation_volume=12; citation_number=17; citation_pages=1040-1045; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An evolutionary advantage of haploidy in large yeast populations.; citation_author=C Zeyl; citation_author=T Vanderford; citation_author=M Carter; citation_journal_title=Science; citation_volume=299; citation_number=18; citation_pages=555-558; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mutations of intermediate effect are responsible for adaptation in evolving Pseudomonas fluorescens populations.; citation_author=RDH Barrett; citation_author=RC MacLean; citation_author=G Bell; citation_journal_title=Biol Lett; citation_volume=2; citation_number=19; citation_pages=236-238; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Distribution of fitness effects among beneficial mutations before selection in experimental populations of bacteria.; citation_author=R Kassen; citation_author=T Bataillon; citation_journal_title=Nat Genet; citation_volume=38; citation_number=20; citation_pages=484-488; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fitness effects of advantageous mutations in evolving Escherichia coli populations.; citation_author=M Imhof; citation_author=C Schlotterer; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=21; citation_pages=1113-1117; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Limits to adaptation in asexual populations.; citation_author=J De Visser; citation_author=DE Rozen; citation_journal_title=J Evol Biol; citation_volume=18; citation_number=22; citation_pages=779-788; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Deterministic and stochastic regimes of asexual evolution on rugged fitness landscapes.; citation_author=K Jain; citation_author=J Krug; citation_journal_title=Genetics; citation_volume=175; citation_number=23; citation_pages=1275-1288; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The roles of mutation, inbreeding, crossbreeding, and selection in evolution.; citation_author=S Wright; citation_journal_title=Proceedings of the Sixth International Congress on Genetics; citation_number=24; citation_pages=355-366; citation_date=1932; " />
      <meta name="citation_reference" content="citation_title=Population genetics-Peak shifts in large populations.; citation_author=CJ Goodnight; citation_journal_title=Heredity; citation_volume=96; citation_number=25; citation_pages=5-6; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Rapid evolutionary escape by large populations from local fitness peaks is likely in nature.; citation_author=DM Weinreich; citation_author=L Chao; citation_journal_title=Evolution; citation_volume=59; citation_number=26; citation_pages=1175-1182; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microbial experiments on adaptive landscapes.; citation_author=N Colegrave; citation_author=A Buckling; citation_journal_title=Bioessays; citation_volume=27; citation_number=27; citation_pages=1167-1173; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolvability of an RNA virus is determined by its mutational neighbourhood.; citation_author=CL Burch; citation_author=L Chao; citation_journal_title=Nature; citation_volume=406; citation_number=28; citation_pages=625-628; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evidence for Multiple Adaptive Peaks from Populations of Bacteria Evolving in a Structured Habitat.; citation_author=R Korona; citation_author=CH Nakatsu; citation_author=LJ Forney; citation_author=RE Lenski; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=91; citation_number=29; citation_pages=9037-9041; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Darwinian evolution can follow only very few mutational paths to fitter proteins.; citation_author=DM Weinreich; citation_author=NF Delaney; citation_author=MA DePristo; citation_author=DL Hartl; citation_journal_title=Science; citation_volume=312; citation_number=30; citation_pages=111-114; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiple Fitness Peaks and Epistasis.; citation_author=MC Whitlock; citation_author=PC Phillips; citation_author=FBG Moore; citation_author=SJ Tonsor; citation_journal_title=Annu Rev Ecol Syst; citation_volume=26; citation_number=31; citation_pages=601-629; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=An equivalence principle for the incorporation of favorable mutations in asexual populations.; citation_author=M Hegreness; citation_author=N Shoresh; citation_author=D Hartl; citation_author=R Kishony; citation_journal_title=Science; citation_volume=311; citation_number=32; citation_pages=1615-1617; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Parallel changes in qene expression after 20,000 generations of evolution in Escherichia coli.; citation_author=TF Cooper; citation_author=DE Rozen; citation_author=RE Lenski; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=33; citation_pages=1072-1077; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Parallel changes in global protein profiles during long-term experimental evolution in Escherichia coli.; citation_author=L Pelosi; citation_author=L Kuhn; citation_author=D Guetta; citation_author=J Garin; citation_author=J Geiselmann; citation_journal_title=Genetics; citation_volume=173; citation_number=34; citation_pages=1851-1869; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tests of parallel molecular evolution in a long-term experiment with Escherichia coli.; citation_author=R Woods; citation_author=D Schneider; citation_author=CL Winkworth; citation_author=MA Riley; citation_author=RE Lenski; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=35; citation_pages=9107-9112; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The effect of population structure on the adaptive radiation of microbial populations evolving in spatially structured environments.; citation_author=M Habets; citation_author=DE Rozen; citation_author=RF Hoekstra; citation_author=J de Visser; citation_journal_title=Ecol Lett; citation_volume=9; citation_number=36; citation_pages=1041-1048; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Randomization and Monte Carlo Methods in Biology.; citation_author=BFJ Manly; citation_number=37; citation_pages=260; citation_date=1991; citation_publisher=Chapman and Hall; " />
      <meta name="citation_reference" content="citation_title=The population genetics of adaptation on correlated fitness landscapes: The block model.; citation_author=HA Orr; citation_journal_title=Evolution; citation_volume=60; citation_number=38; citation_pages=1113-1124; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein Evolution on Partially Correlated Landscapes.; citation_author=AS Perelson; citation_author=CA Macken; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=92; citation_number=39; citation_pages=9657-9661; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Protein Evolution on Rugged Landscapes.; citation_author=CA Macken; citation_author=AS Perelson; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=86; citation_number=40; citation_pages=6191-6195; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Towards a General-Theory of Adaptive Walks on Rugged Landscapes.; citation_author=S Kauffman; citation_author=S Levin; citation_journal_title=J Theor Biol; citation_volume=128; citation_number=41; citation_pages=11-45; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Perspective: A critique of Sewall Wright's shifting balance theory of evolution.; citation_author=JA Coyne; citation_author=NH Barton; citation_author=M Turelli; citation_journal_title=Evolution; citation_volume=51; citation_number=42; citation_pages=643-671; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Adaptation to structurally different environments.; citation_author=R Korona; citation_journal_title=Proc R Soc Lond, Ser B: Biol Sci; citation_volume=263; citation_number=43; citation_pages=1665-1669; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Adaptive radiation in a heterogeneous environment.; citation_author=PB Rainey; citation_author=M Travisano; citation_journal_title=Nature; citation_volume=394; citation_number=44; citation_pages=69-72; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The effect of a bacteriophage on diversification of the opportunistic bacterial pathogen, Pseudomonas aeruginosa.; citation_author=MA Brockhurst; citation_author=A Buckling; citation_author=PB Rainey; citation_journal_title=Proc R Soc Lond, Ser B: Biol Sci; citation_volume=272; citation_number=45; citation_pages=1385-1391; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The effect of spatial heterogeneity and parasites on the evolution of host diversity.; citation_author=MA Brockhurst; citation_author=PB Rainey; citation_author=A Buckling; citation_journal_title=Proc R Soc Lond, Ser B: Biol Sci; citation_volume=271; citation_number=46; citation_pages=107-111; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The effects of competition and predation on diversification in a model adaptive radiation.; citation_author=JR Meyer; citation_author=R Kassen; citation_journal_title=Nature; citation_volume=446; citation_number=47; citation_pages=432-435; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Bacterial-Colonization and Infection Resulting from Multiplication of a Single Organism.; citation_author=LG Rubin; citation_journal_title=Reviews of Infectious Diseases; citation_volume=9; citation_number=48; citation_pages=488-493; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Transmission bottlenecks as determinants of virulence in rapidly evolving pathogens.; citation_author=CT Bergstrom; citation_author=P McElhany; citation_author=LA Real; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=96; citation_number=49; citation_pages=5095-5100; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Long-Term Experimental Evolution in Escherichia-Coli .1. Adaptation and Divergence During 2,000 Generations.; citation_author=RE Lenski; citation_author=MR Rose; citation_author=SC Simpson; citation_author=SC Tadler; citation_journal_title=Am Nat; citation_volume=138; citation_number=50; citation_pages=1315-1341; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=From bad to good: Fitness reversals and the ascent of deleterious mutations.; citation_author=MC Cowperthwaite; citation_author=JJ Bull; citation_author=LA Meyers; citation_journal_title=PLoS Comp Biol; citation_volume=2; citation_number=51; citation_pages=1292-1300; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The evolutionary origin of complex features.; citation_author=RE Lenski; citation_author=C Ofria; citation_author=RT Pennock; citation_author=C Adami; citation_journal_title=Nature; citation_volume=423; citation_number=52; citation_pages=139-144; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Adaptive mutations in bacteria: High rate and small effects.; citation_author=L Perfeito; citation_author=L Fernandes; citation_author=C Mota; citation_author=I Gordo; citation_journal_title=Science; citation_volume=317; citation_number=53; citation_pages=813-815; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001715" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes"/>
    <meta name="twitter:description" content="BackgroundSmall populations are thought to be adaptively handicapped, not only because they suffer more from deleterious mutations but also because they have limited access to new beneficial mutations, particularly those conferring large benefits.Methodology/Principal FindingsHere, we test this widely held conjecture using both simulations and experiments with small and large bacterial populations evolving in either a simple or a complex nutrient environment. Consistent with expectations, we find that small populations are adaptively constrained in the simple environment; however, in the complex environment small populations not only follow more heterogeneous adaptive trajectories, but can also attain higher fitness than the large populations. Large populations are constrained to near deterministic fixation of rare large-benefit mutations. While such determinism speeds adaptation on the smooth adaptive landscape represented by the simple environment, it can limit the ability of large populations from effectively exploring the underlying topography of rugged adaptive landscapes characterized by complex environments.ConclusionsOur results show that adaptive constraints often faced by small populations can be circumvented during evolution on rugged adaptive landscapes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001715.g003"/>

  <meta property="og:title" content="Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001715" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=971'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4901'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=977&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001715">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001715" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001715&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel E. Rozen
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Daniel E. Rozen, Michelle G. J. L. Habets</p>

              
              <p>*E-mail: <a href="mailto:daniel.rozen@manchester.ac.uk">daniel.rozen@manchester.ac.uk</a></p>

                <p>Affiliations:
                  Department of Genetics, Wageningen University, Wageningen, The Netherlands, 
                  University of Manchester, Manchester, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michelle G. J. L. Habets
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Daniel E. Rozen, Michelle G. J. L. Habets</p>

              
              

                <p>Affiliation:
                  Department of Genetics, Wageningen University, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andreas Handel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Emory University, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J. Arjan G. M. de Visser
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Wageningen University, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 05, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001715</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001715" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001715" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001715" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001715" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001715-g001" data-doi="info:doi/10.1371/journal.pone.0001715" data-uri="info:doi/10.1371/journal.pone.0001715.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001715-g002" data-doi="info:doi/10.1371/journal.pone.0001715" data-uri="info:doi/10.1371/journal.pone.0001715.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001715-g003" data-doi="info:doi/10.1371/journal.pone.0001715" data-uri="info:doi/10.1371/journal.pone.0001715.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001715">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001715" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Small populations are thought to be adaptively handicapped, not only because they suffer more from deleterious mutations but also because they have limited access to new beneficial mutations, particularly those conferring large benefits.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here, we test this widely held conjecture using both simulations and experiments with small and large bacterial populations evolving in either a simple or a complex nutrient environment. Consistent with expectations, we find that small populations are adaptively constrained in the simple environment; however, in the complex environment small populations not only follow more heterogeneous adaptive trajectories, but can also attain higher fitness than the large populations. Large populations are constrained to near deterministic fixation of rare large-benefit mutations. While such determinism speeds adaptation on the smooth adaptive landscape represented by the simple environment, it can limit the ability of large populations from effectively exploring the underlying topography of rugged adaptive landscapes characterized by complex environments.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results show that adaptive constraints often faced by small populations can be circumvented during evolution on rugged adaptive landscapes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rozen DE, Habets MGJL, Handel A, de Visser JAGM (2008) Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes. PLoS ONE 3(3):
          e1715.
            doi:10.1371/journal.pone.0001715</p><p><strong>Academic Editor: </strong>Rees Kassen, University of Ottawa, Canada</p><p><strong>Received:</strong> October 2, 2007; <strong>Accepted:</strong> January 18, 2008; <strong>Published:</strong> March 5, 2008</p><p><strong>Copyright:</strong> © 2008 Rozen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants to J.A.G.M. de V from NWO and a US NSF postdoctoral fellowship in Microbial Biology to D.E.R. There are no competing interests that would influence conclusions of the authors.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>It is widely held that the efficiency of natural selection is positively related to the size of an evolving population <a href="#pone.0001715-Willi1">[1]</a>–<a href="#pone.0001715-Reed1">[5]</a>. This intuition derives from the expectations that small populations are more subject to the chance fixation of deleterious mutations by genetic drift <a href="#pone.0001715-Muller1">[6]</a>–<a href="#pone.0001715-Silander1">[11]</a> and that fewer beneficial mutations arise in small populations compared to large ones <a href="#pone.0001715-Fisher1">[12]</a>–<a href="#pone.0001715-Zeyl1">[18]</a>. An unexplored factor that could mitigate these constraints in small populations arises from the fact that beneficial mutations are unevenly distributed, with few mutations causing large fitness benefits and most causing more modest gains <a href="#pone.0001715-Burch1">[14]</a>, <a href="#pone.0001715-Orr1">[16]</a>, <a href="#pone.0001715-Rozen1">[17]</a>, <a href="#pone.0001715-Barrett1">[19]</a>–<a href="#pone.0001715-Imhof1">[21]</a>. This skewed distribution implies that smaller populations will substitute a more diverse set of beneficial mutations <a href="#pone.0001715-DeVisser1">[22]</a>, with the consequence that they may follow more heterogeneous adaptive trajectories than large populations <a href="#pone.0001715-Jain1">[23]</a>, particularly if mutations interact epistatically. While the effects of genetic drift on the fixation of deleterious mutations are well appreciated and studied, for example in Phase 1 of Wright's Shifting Balance Theory <a href="#pone.0001715-Wright1">[24]</a>, the effects of stochasticity on the fixation of beneficial mutations have not been considered in any experimental context. The aim of the present contribution is to examine conditions where heterogeneity in the emergence and fixation of beneficial mutations enables the adaptive constraints associated with a limited population size to be overcome.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Using the fitness landscape metaphor of Wright <a href="#pone.0001715-Wright1">[24]</a>, we consider the evolution of populations on two distinct fitness landscapes, one that is “smooth” with a single fitness peak, and another that is “rugged” with several peaks. In this scenario, the landscape refers to a topographical map between individual genotypes and their corresponding fitness values. We imagine that populations begin their evolutionary trajectories from fitness valleys; they will have already drifted, or been otherwise displaced, from a local fitness peak in Phase 1 of the Shifting Balance process (a process that is thought to be more efficient in small populations <a href="#pone.0001715-Goodnight1">[25]</a>, but see Weinreich <em>et al.</em> <a href="#pone.0001715-Weinreich1">[26]</a>) and are awaiting the appearance and fixation of new beneficial mutations that will bring them into the domain of attraction of other, perhaps higher, peaks <a href="#pone.0001715-Wright1">[24]</a>. The adaptive route taken by any given population is expected to be a function of the underlying topography of the fitness landscape <a href="#pone.0001715-Colegrave1">[27]</a>–<a href="#pone.0001715-Korona1">[29]</a>. On smooth adaptive landscapes with only a single peak, locating the fittest solution, or global optimum, is a matter of successively substituting the largest available beneficial mutations. It is assumed that this process will be slower in small populations as a result of their diminished access to (large) beneficial mutations <a href="#pone.0001715-Silander1">[11]</a>, <a href="#pone.0001715-Burch1">[14]</a>–<a href="#pone.0001715-Rozen1">[17]</a>. In contrast, reaching the global optimum on rugged landscapes is expected to be a function of the interactions between the specific mutations that become substituted, because fitness on complex landscapes is determined by the epistatic effects among combinations of mutations <a href="#pone.0001715-Colegrave1">[27]</a>, <a href="#pone.0001715-Weinreich2">[30]</a>, <a href="#pone.0001715-Whitlock1">[31]</a>. Therefore, on rugged fitness landscapes small populations, owing to increased variability in the fitness effects of beneficial mutations that become substituted <a href="#pone.0001715-DeVisser1">[22]</a>, may locate a more diverse set of adaptive peaks, and on occasion ascend higher adaptive peaks than large populations. In contrast, by deterministically substituting only the largest beneficial mutations <a href="#pone.0001715-DeVisser1">[22]</a>, <a href="#pone.0001715-Hegreness1">[32]</a>, large populations will be limited to fewer adaptive routes that climb the nearest fitness peak with the steepest slope, but not necessarily the highest peak. We note that this general prediction is consistent with Wright's conjecture that small populations (subdivided demes) offer the best opportunity to allow the Shifting Balance process to proceed <a href="#pone.0001715-Wright1">[24]</a>, <a href="#pone.0001715-Whitlock1">[31]</a>. However, the solution we outline introduces a critical role for a second stochastic factor and suggests that small populations are not only more likely to drift away from local fitness peaks in Phase 1, but also that they are more efficient seekers of distant, and occasionally higher, fitness peaks under the influence of natural selection in Phase 2 of the Shifting Balance process due to their broader sampling from the distribution of beneficial mutations.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Here we first test these predictions experimentally using evolving bacterial populations, and then explore the generality and limitations of our results using simulations. Twenty-four small and six large populations initiated with a single clone of <em>E. coli</em> were allowed to evolve for 500 generations in either a simple or a complex nutrient environment. The simple environment is a glucose minimal medium (DM) that has been shown to lead to considerable adaptive parallelism <a href="#pone.0001715-Cooper1">[33]</a>–<a href="#pone.0001715-Woods1">[35]</a>, consistent with a relatively smooth fitness landscape. The complex environment, Luria-Bertani Broth (LB), contains a variety of carbon sources and other nutrients that offer a broader array of adaptive options, consistent with a more rugged fitness landscape <a href="#pone.0001715-Habets1">[36]</a>. Population size was manipulated by adjusting the culture volume, causing a ~50-fold difference (5×10<sup>5</sup> versus 2.5×10<sup>7</sup>). The extent of adaptation of each population was determined by measuring the relative fitness of population samples against a differently marked ancestor in head-to-head competition in the same environment in which they had evolved.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Fitness estimates of populations taken after 500 generations support the prediction that adaptation is more heterogeneous in small populations (<a href="#pone-0001715-g001">Fig 1</a>); significant among population variation for fitness was found for small populations in both environments (simple: <em>F</em><sub>21,44</sub> = 4.21, <em>P</em>&lt;0.001; complex: <em>F</em><sub>22,45</sub> = 3.58, <em>P</em>&lt;0.001), while large populations show no apparent heterogeneity for fitness in either one (simple: <em>F</em><sub>5,12</sub> = 0.80, <em>P</em> = 0.57; complex: <em>F</em><sub>5,12</sub> = 0.85, <em>P</em> = 0.54). Furthermore, using an <em>F</em>-test to directly compare variance components, we found that the among-population fitness variation was higher for small than large populations in the complex nutrient environment (<em>F</em><sub>22,5</sub> = 5.20, <em>P</em> = 0.038), but did not differ in the simple environment (<em>F</em><sub>21,5</sub> = 2.10, <em>P</em> = 0.21). To account for the asymmetry in sample size between small and large populations, we tested the robustness of this <em>F</em>-test by using a bootstrap procedure <a href="#pone.0001715-Manly1">[37]</a> and found results that were consistent with the original test (complex environment: <em>P</em> = 0.043; simple environment <em>P</em> = 0.45). As expected, given that large populations have increased access to beneficial mutations conferring large benefits, we found that large populations adapted faster than small ones in the simple environment (<em>t</em><sub>11</sub> = −3.40, 2-tailed <em>P</em> = 0.0059). However, the reverse was found in the complex environment (<em>t</em><sub>19</sub> = 3.70, 2-tailed <em>P</em> = 0.0015). As earlier, this result was confirmed using a bootstrap approach (<em>P</em> = 0.0073 and <em>P</em> = 0.0039, respectively). These data reveal that although adaptive heterogeneity is increased in small populations, the evolutionary consequences of this variation are highly dependent upon the topography of the adaptive landscape, because only on the rugged fitness landscape are benefits to this heterogeneity realized.</p>
<div class="figure" id="pone-0001715-g001"><div class="img"><a name="pone-0001715-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001715" data-uri="info:doi/10.1371/journal.pone.0001715.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001715.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001715.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Relative fitness of large and small bacterial populations after evolution on either a simple or complex nutrient environment.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>A value of 1 indicates no change.</p>
<span>doi:10.1371/journal.pone.0001715.g001</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>To explore the interactions between population size, environment, and fitness gain in more detail, the adaptive trajectories of a subset of small and large populations during evolution in the complex resource environment were obtained (<a href="#pone-0001715-g002">Fig. 2</a>). Whereas large populations showed parallel fitness gains (<em>F</em><sub>10,36</sub> = 1.90, <em>P</em> = 0.077), small populations explored the rugged adaptive landscape in different ways (<em>F</em><sub>22,46</sub> = 4.06, <em>P</em>&lt;0.001) indicating that they have followed divergent adaptive trajectories. The effect of this heterogeneity is particularly evident for three small populations with final fitness significantly higher than even the most fit large population, consistent with their having attained a distinct fitness peak (<em>t</em><sub>2</sub> = 3.90, 1-tailed, <em>P</em> = 0.03) (dotted lines <a href="#pone-0001715-g002">Fig 2a</a>). The fitness trajectories of these three populations were significantly different from those of the other nine small populations (<em>F</em><sub>1,33</sub><em> = </em>11.83, <em>P = </em>0.0016). Moreover, while they were more fit after 500 generations (<em>t</em><sub>10</sub> = 5.13, 2-tailed <em>P</em>&lt;0.001), at 100 generations their average fitness improvement was significantly lower than that of the other nine small populations (<em>t</em><sub>10</sub> = 2.79, 2-tailed <em>P</em> = 0.019).</p>
<div class="figure" id="pone-0001715-g002"><div class="img"><a name="pone-0001715-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001715" data-uri="info:doi/10.1371/journal.pone.0001715.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001715.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001715.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Fitness trajectories of 12 small (A) and six large (B) populations evolving in the complex environment.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Dotted lines highlight small populations that have attained higher fitness than other small and even the most fit large populations (see text for details).</p>
<span>doi:10.1371/journal.pone.0001715.g002</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Our experimental data demonstrate that the dynamics of fitness gain in complex environments depend on the topographical details of the fitness landscape. We show that this dependence is a function of the adaptive routes followed by evolving populations, and that this varies significantly between small and large populations. Finally, we show that adaptive walks that ascend “steep hills” do not always climb the highest peaks. Indeed, only those small populations that initially substituted smaller beneficial mutations obtained the largest fitness gains.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>An assumption of our experimental model is that simple and complex resource environments correspond to adaptive landscapes that are “smooth” and additive, and “rugged” and epistatic, respectively. Although our data are consistent with this interpretation and there is precedent for this approach <a href="#pone.0001715-Colegrave1">[27]</a>, it is not feasible to experimentally determine <em>a priori</em> the epistatic contingency of evolution in any experimental environment. In order to overcome this limitation, we performed computer simulations of populations evolving on adaptive landscapes where the levels of mutational epistasis could be explicitly defined. The simulations enabled us to consider the evolutionary response of small and large populations over a vastly extended time scale which allowed us to determine if our simulated populations had reached a local or global fitness peak. Additionally, the simulations allowed us to focus exclusively on the role of epistasis, where in the experiments there remains the possibility that population divergence resulted, at least partially, from differential niche specialization.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>In our computer simulations, digital bacteria undergo iterated cycles of exponential growth and serial dilution. Every clone grows according to a fitness value that is initially scaled to 1. Additionally, every clone has a fitness neighbourhood of fixed size <em>L</em>, which corresponds to the number of possible 1-step mutations the clone can reach. During growth, mutant offspring arise at a rate, <em>μ</em>, and thereby obtain a new fitness value that corresponds to one of the <em>L</em> neighbouring fitness values. Each mutant clone can either retain a fraction of the fitness neighbourhood of its parent, or obtain an entirely new fitness neighbourhood. If the parental fitness neighbourhood is retained, the result is a smooth fitness landscape with few maxima among the <em>L</em> fitness values and no epistasis. At the other extreme, if all fitness neighbours are replaced, the result is a maximally rugged fitness landscape with complete epistasis and many local optima. In both cases, the landscapes we utilize are likely to be exaggerated versions of what might be found in nature. Our use follows earlier pioneering fitness landscape simulations <a href="#pone.0001715-Orr2">[38]</a>–<a href="#pone.0001715-Kauffman1">[41]</a>, and is intended to establish the simplest boundary conditions and to complement but not to faithfully reproduce the experiment.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>Broadly, the simulations provide strong qualitative support for our interpretation of the experimental results. <a href="#pone-0001715-g003">Figures 3a and 3b</a> show the fitness trajectories for fifty individual small or large simulated populations evolving on either a smooth (<a href="#pone-0001715-g003">Fig. 3a</a>) or rugged (<a href="#pone-0001715-g003">Fig. 3b</a>) fitness landscape. In a manner consistent with our experimental results, a number of small populations on the rugged landscape, but not on the smooth landscape, obtain higher long-term fitness than even the most fit large populations. That this result is only found on the rugged landscape supports the idea that the dynamics of fitness gain are highly dependent on the topography of the underlying fitness landscape, with epistatic interactions among mutations providing the critical advantage to small populations. We next calculated the time averaged variation in fitness among small and large populations as a function of landscape topography (<a href="#pone-0001715-g003">Fig. 3c</a>), from which we draw two conclusions. First, this analysis shows that among population heterogeneity is higher for small than large populations irrespective of landscape complexity. Secondly, it reveals that variance in evolutionary response is increased for both small and large populations during adaptation on rugged adaptive landscapes relative to their behaviour on the smooth landscape. This latter effect is likely the result of the fact that rugged landscapes contain more fitness peaks, while the former is a consequence of the fact that small populations follow more heterogeneous adaptive trajectories. Most interestingly, these simulation results show that the benefits that accrue to small populations by following diverse adaptive trajectories are only realized when fitness is determined by epistatic interactions among beneficial mutations. Otherwise, small populations remain adaptively constrained.</p>
<div class="figure" id="pone-0001715-g003"><div class="img"><a name="pone-0001715-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001715" data-uri="info:doi/10.1371/journal.pone.0001715.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001715.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001715.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001715.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001715.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001715.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Simulation results of fitness gain in 50 small (dotted line) and large (unbroken line) populations on either a smooth (a) or complex (b) fitness landscape.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>The number of 1-step neighbours, <em>L,</em> is 500 and the mutation rate, <em>μ</em>, is 5e-6. Variation in fitness across treatments and population size is shown in <a href="#pone-0001715-g003">Fig 3c</a>.</p>
<span>doi:10.1371/journal.pone.0001715.g003</span></div><a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>In summary, our data provide experimental and theoretical evidence that limits to adaptation in small populations can be overcome during evolution on complex fitness landscapes. Furthermore, we show that the topography of the fitness landscape is an important determinant of this outcome, because those small populations with the greatest final fitness improvement were ones that initially ascended relatively shallow slopes. It is important to note that benefits from more effective landscape searching are far from assured in small populations. Indeed, many small populations, both in the experiment and in the simulations, faced handicaps consistent with their diminished access to beneficial mutations of large effect. However, whereas the outcome of adaptation in large populations is nearly deterministic, adaptation in small populations can generate unpredictable results and unexpected benefits.</p>
<a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>Although our experiments were not designed to specifically test Wright's Shifting Balance Theory <a href="#pone.0001715-Wright1">[24]</a> which was developed to understand the evolution of novelty and complexity in sexual species <a href="#pone.0001715-Coyne1">[42]</a>, it has not escaped our notice that our results are of particular relevance to Phases 2 and 3 of the theory. In Phase 2, populations previously displaced from their original adaptive peaks via genetic drift in Phase 1, are envisioned to ascend new peaks via the accumulation of beneficial mutations. Genetic drift in small populations, in Phase 1, and epistatic interactions among mutations, in Phase 2, are thought to facilitate this process. The results here, despite the fact that they were obtained from an asexual species, are consistent with this view in two ways. First, we find that small populations are better able to locate a diverse range of fitness peaks than large populations, and second that advantages to this diversity are only realized on landscapes where epistatic interactions are expected to be common. In Phase 3, migration from the fittest populations causes demes resident on lower fitness peaks to cross fitness valleys in order to shift towards higher fitness peaks. Because our experiments did not include a migration treatment we lack experimental support for this final phase; however, it seems likely that appropriate rates of migration among small populations would have the effect of causing more efficient peak shifts among small than large populations. Tests of this conjecture are currently in progress.</p>
<a id="article1.body1.sec2.p10" name="article1.body1.sec2.p10"></a><p>Two further implications emerge from our data. First, although our experimental results partly depend on the specifics of the environments applied in our study, it is likely that our complex nutrient treatment actually underestimates the complexity of most environments, because it lacks both spatial structure <a href="#pone.0001715-Korona2">[43]</a>, <a href="#pone.0001715-Rainey1">[44]</a> and interactions with other organisms such as predators and parasites <a href="#pone.0001715-Brockhurst1">[45]</a>–<a href="#pone.0001715-Meyer1">[47]</a>. If natural fitness landscapes are actually more rugged than those used here the potential evolutionary advantage to small, marginal, or fragmented populations may be further enhanced, and this may somewhat mitigate risks to threatened populations of animals and plants. Second, we note that our experimental population densities are consistent with the population bottlenecks experienced by many microbial pathogens during initial infection <a href="#pone.0001715-Rubin1">[48]</a>. Such bottlenecks are thought to be costly to microbial populations due to the accumulation of deleterious mutations by Muller's ratchet, which may in turn serve to diminish microbial virulence <a href="#pone.0001715-Bergstrom1">[49]</a>. However, the emergence of heterogeneous populations following infection in small inocula may serve as an important diversifying factor for microbes, perhaps providing microbes with an adaptive edge in the co-evolutionary “tug-of-war” between pathogens and their hosts. These implications await further testing.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Bacteria and media</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The <em>Escherichia coli</em> B strains used in this experiment, REL606 and REL607, have been used extensively in experimental evolution and are described elsewhere <a href="#pone.0001715-Lenski1">[50]</a>. These ancestral strains are genetically identical except for a difference in their ability to catabolize L-Arabinose, which can be used as a marker to distinguish both strains when plated on tetrazolium-aribinose (TA) indicator plates.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Two nutrient environments were used for the serial transfer experiment: a simple medium -with glucose as sole carbon source- and a complex medium. The simple medium is Davis' minimal broth supplemented with 2* 10<sup>−6</sup> thiamine hydrochloride and 0.25 g glucose per liter (DM250) and the complex medium is a 1/10 dilution of Luria-Bertani broth (1/10LB). The contrasting environments were chosen following experiments showing that <em>E. coli</em> maintained in spatially-structured Petri plates containing LB evolved more phenotypic and genetic heterogeneity than populations evolved in DM, consistent with more niches and alternative adaptive peaks in the former environment <a href="#pone.0001715-Habets1">[36]</a>. The population density at stationary phase for both media types is 5*10<sup>8 </sup>cells/mL.</p>


<h4>Evolution experiment</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Populations derived from both ancestral strains were maintained for 500 generations by serial transfer in unshaken tubes (4.8 mL) or in wells of a 96-well plate (100 µL), representing large and small populations, respectively. Our results indicate that possible differences between tubes and wells of a 96-well plate have not introduced experimental artifacts: 1) this potential artifact would not predict increased heterogeneity among small populations, which we observe in both resource environments; 2) it would predict that large populations would show increased fitness in large populations irrespective of environment, when instead we find a clear interaction between population size and resource complexity; and 3) it would not predict a transient decrease in the fitness of small populations that obtain the highest fitness. Every 24 hours the populations were diluted by a 1,000-fold dilution into fresh medium, and then incubated at 37°C; for the small populations 5*10<sup>4 </sup>cells were transferred (<em>N</em><sub>e = </sub>5*10<sup>5</sup>) using a 96-pin replicator (Boekel Scientific), while for the large populations this was 2.5*10<sup>6</sup> cells (<em>N</em><sub>e = </sub>2,5*10<sup>7</sup>). Each culture underwent roughly 10 generations of daily growth. Every 100 generations, the populations were stored in a 15% glycerol solution at −80°C. Large populations were replicated six-fold, small populations were replicated 48-fold, in two separate 96-well plates. For reasons of experimental tractability, following the 500 generations fitness assays were conducted on a randomly sampled subset of 24 from the 48 small populations from each medium type. Fitness trajectories of small populations were estimated for a random set of 12 populations from the original 24, and for all six large populations.</p>


<h4>Fitness assays</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The relative fitness of evolved populations was measured according to previous protocols <a href="#pone.0001715-Lenski1">[50]</a> by competing populations against the reciprocally marked ancestral clone for 10 generations. Conditions in competitions were equivalent to those during serial transfer. Prior to the competition, competitors were separately grown for 24 hours in the appropriate medium, to insure that both were in equal physiological states. At the beginning and the end of the competition, the frequency of both competitors was determined by plating onto TA. From these frequencies, relative fitness was estimated as the ratio of each strain's Malthusian parameter. Competitions for mean fitness of the populations were replicated three-fold; all other competitions were replicated six-fold.</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>To account for the difference in sample size between small and large populations, we tested the robustness of the results of our analyses by using a bootstrap procedure <a href="#pone.0001715-Manly1">[37]</a>. This was achieved by resampling with replacement from the original replicate fitness estimates to generate 10,000 sets of 24 versus six pseudo-populations (or 12 versus 6 pseudo-populations for the analysis of fitness trajectories). For each set, an <em>F</em>-test on the among population variation in fitness of the 24 versus the six pseudo-populations was calculated. This distribution of <em>F</em>-values was then used to calculate the proportion of test values higher than the <em>F</em>-value corresponding to the real data, which reflects the probability that the higher among-population variation in small than large populations arose from random processes (or an asymmetry in sample size) <a href="#pone.0001715-Manly1">[37]</a>. A similar bootstrapping approach was employed to carry out <em>t</em>-tests comparing the rate of adaptation in small versus large populations in both environment types.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Fitness data were analyzed using <em>t</em>-tests with unequal variances; the greater fitness variation of small versus large populations precludes the use of standard ANOVA. Repeated-measures ANOVA were used to examine the adaptive trajectories of small or large populations. In order to avoid the problem of heterogeneous variances, we applied this ANOVA to small and large populations separately. We were particularly interested to see whether small populations showed evidence for significant heterogeneity in their respective adaptive dynamics, which would be apparent as a significant interaction between population and time of the repeated-measures ANOVA on individual adaptive trajectories.</p>


<h4>Simulation design</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The simulations were designed to approximate critical features of our experiments. The digital bacteria grow by dividing at rates determined by their fitness. The population starts at size N0 and growth continues until the population reaches carrying capacity, at which point serial transfer, modelled as multinomial sampling, reduces the population size back to <em>N</em>0 which initiates another round of exponential growth. This procedure is iterated until the desired number of generations is reached. Effective population sizes, <em>N<sub>e</sub></em><sub>,</sub> calculated as <em>N</em>0 * (generations grown between transfers)<a href="#pone.0001715-Gerrish1">[2]</a>, are equivalent to those used during the bacterial experiments (5e<sup>5</sup> or 2.5e<sup>7</sup> for small and large populations, respectively). Initial populations are clonal, but at division, each clone generates mutants at a rate <em>μ</em> that differ in fitness from the parent clone. By convention, the ancestral clone is assigned a fitness value of 1 and offspring a value of 1+s, where values of s are drawn from an exponential distribution <em>f</em>(<em>s</em>) = α<em>e<sup>−</sup></em><sup>α<em>s</em></sup>, with α<em> = </em>42.5 <a href="#pone.0001715-Silander1">[11]</a>. Because recent simulations have suggested that higher fitness peaks can be reached by first going through an intermediate step with reduced fitness <a href="#pone.0001715-Cowperthwaite1">[51]</a>, <a href="#pone.0001715-Lenski2">[52]</a>, we also performed simulations that included only a small fraction of mutations that lead to fitness larger than that of the ancestral strain, while the majority of mutations reduced fitness (i.e.: deleterious mutations ≫ beneficial mutations). However, due to the strong bottlenecks imposed by the repeated serial passages, these less fit mutants never survived long enough to produce consecutive, fitter mutants. Therefore, their inclusion had no effect on the outcome of the simulations. Because including deleterious mutations significantly increased computational demands but did not affect our conclusions, the results presented here are for simulations that only included beneficial mutations. At division, offspring remain unchanged or attain the state of any of <em>L</em> single mutant neighbours. Once a mutation occurs, a new mutant is created with fitness drawn randomly from the <em>L</em> possible neighbourhood values. Additionally, the new mutant obtains its own one-step neighbourhood of <em>L</em> mutants. To generate a smooth landscape, the newly created mutant is assigned a mutant neighbourhood which is identical to that of the ancestral strain, leading to a landscape with only a single global optimum. To approximate a rugged fitness landscape, we consider the other extreme, where 100% of the neighbourhood is replaced, with values for the possible fitness increase resampled from <em>f</em>(<em>s</em>). This leads to a completely rugged landscape, where a single mutation changes the fitness effects of all other possible mutations. The mutation rate <em>μ</em> was set to 5e<sup>−6</sup> <a href="#pone.0001715-Perfeito1">[53]</a> and the total number of 1-mutant neighbours, L, to 500. While these two parameters are chosen rather arbitrarily, we found that the results remain qualitatively unchanged for different parameter values, as long as the effective mutation supply rate for small populations is significantly smaller than the mutant neighbourhood, i.e. Ne* <em>μ</em>≪L, and Ne* <em>μ</em>≈L for large populations. Such a situation allows the small populations to evolve stochastically, while the large populations will evolve in an essentially deterministic manner. The model was written in Matlab and will be provided upon request to A.H.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Rolf Hoekstra and Fons Debets for helpful discussions throughout the completion of this work, and Gabriella Bukovinszkine Kiss, Bertha Koopmanschap, Marc Maas, and Anita Mulder for technical support. We are additionally grateful to Ed Harris for statistical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: Jd AH DR MH. Performed the experiments: Jd AH DR MH. Analyzed the data: Jd AH DR MH. Contributed reagents/materials/analysis tools: Jd AH DR MH. Wrote the paper: Jd AH DR MH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001715-Willi1" id="pone.0001715-Willi1"></a>Willi Y, Van Buskirk J, Hoffmann AA (2006) Limits to the adaptive potential of small populations. Annu Rev Ecol, Evol Syst  37: 433–458.  <ul class="find" data-citedArticleID="1066670" data-doi="10.1146/annurev.ecolsys.37.091305.110145"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.37.091305.110145" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Limits+to+the+adaptive+potential+of+small+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Limits+to+the+adaptive+potential+of+small+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001715-Gerrish1" id="pone.0001715-Gerrish1"></a>Gerrish PJ, Lenski RE (1998) The fate of competing beneficial mutations in an asexual population. Genetica  103: 127–144.  <ul class="find" data-citedArticleID="1066600" data-doi="10.1007/978-94-011-5210-5_12"><li><a href="http://dx.doi.org/10.1007/978-94-011-5210-5_12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fate+of+competing+beneficial+mutations+in+an+asexual+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fate+of+competing+beneficial+mutations+in+an+asexual+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001715-Wilke1" id="pone.0001715-Wilke1"></a>Wilke CO (2004) The speed of adaptation in large asexual populations. Genetics  167: 2045–2053.  <ul class="find" data-citedArticleID="1066668" data-doi="10.1534/genetics.104.027136"><li><a href="http://dx.doi.org/10.1534/genetics.104.027136" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+speed+of+adaptation+in+large+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+speed+of+adaptation+in+large+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001715-deVisser1" id="pone.0001715-deVisser1"></a>de Visser J, Zeyl CW, Gerrish PJ, Blanchard JL, Lenski RE (1999) Diminishing returns from mutation supply rate in asexual populations. Science  283: 404–406.  <ul class="find" data-citedArticleID="1066678" data-doi="10.1126/science.283.5400.404"><li><a href="http://dx.doi.org/10.1126/science.283.5400.404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diminishing+returns+from+mutation+supply+rate+in+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diminishing+returns+from+mutation+supply+rate+in+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001715-Reed1" id="pone.0001715-Reed1"></a>Reed DH (2005) Relationship between population size and fitness. Conserv Biol  19: 563–568.  <ul class="find" data-citedArticleID="1066654" data-doi="10.1111/j.1523-1739.2005.00444.x"><li><a href="http://dx.doi.org/10.1111/j.1523-1739.2005.00444.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+population+size+and+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+population+size+and+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001715-Muller1" id="pone.0001715-Muller1"></a>Muller HJ (1964) The Relation of Recombination to Mutational Advance. Mutat Res  1: 2–9.  <ul class="find" data-citedArticleID="1066638" data-doi="10.1016/0027-5107(64)90047-8"><li><a href="http://dx.doi.org/10.1016/0027-5107(64)90047-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Relation+of+Recombination+to+Mutational+Advance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Relation+of+Recombination+to+Mutational+Advance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001715-Lande1" id="pone.0001715-Lande1"></a>Lande R (1998) Risk of population extinction from fixation of deleterious and reverse mutations. Genetica  103: 21–27.  <ul class="find" data-citedArticleID="1066620" data-doi="10.1007/978-94-011-5210-5_2"><li><a href="http://dx.doi.org/10.1007/978-94-011-5210-5_2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Risk+of+population+extinction+from+fixation+of+deleterious+and+reverse+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Risk+of+population+extinction+from+fixation+of+deleterious+and+reverse+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001715-Lynch1" id="pone.0001715-Lynch1"></a>Lynch M, Burger R, Butcher D, Gabriel W (1993) The Mutational Meltdown in Asexual Populations. J Hered  84: 339–344.  <ul class="find" data-citedArticleID="1066626"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Mutational+Meltdown+in+Asexual+Populations.&amp;auth=&amp;atitle=The+Mutational+Meltdown+in+Asexual+Populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mutational+Meltdown+in+Asexual+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mutational+Meltdown+in+Asexual+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001715-Lynch2" id="pone.0001715-Lynch2"></a>Lynch M, Conery J, Burger R (1995) Mutation Accumulation and the Extinction of Small Populations. Am Nat  146: 489–518.  <ul class="find" data-citedArticleID="1066628" data-doi="10.1086/285812"><li><a href="http://dx.doi.org/10.1086/285812" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+Accumulation+and+the+Extinction+of+Small+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+Accumulation+and+the+Extinction+of+Small+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001715-Chao1" id="pone.0001715-Chao1"></a>Chao L (1990) Fitness of Rna Virus Decreased by Muller Ratchet. Nature  348: 454–455.  <ul class="find" data-citedArticleID="1066586" data-doi="10.1038/348454a0"><li><a href="http://dx.doi.org/10.1038/348454a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+of+Rna+Virus+Decreased+by+Muller+Ratchet." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+of+Rna+Virus+Decreased+by+Muller+Ratchet.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001715-Silander1" id="pone.0001715-Silander1"></a>Silander OK, Tenaillon O, Chao L (2007) Understanding the Evolutionary Fate of Finite Populations: The Dynamics of Mutational Effects. PLoS Biol  5: e94.  <ul class="find" data-citedArticleID="1066660" data-doi="10.1371/journal.pbio.0050094"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0050094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+the+Evolutionary+Fate+of+Finite+Populations%3A+The+Dynamics+of+Mutational+Effects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+the+Evolutionary+Fate+of+Finite+Populations%3A+The+Dynamics+of+Mutational+Effects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001715-Fisher1" id="pone.0001715-Fisher1"></a>Fisher RA (1958) The Genetic Theory of Natural Selection. <strong>Dover, New York USA</strong>.  <ul class="find" data-citedArticleID="1066598"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Genetic+Theory+of+Natural+Selection.+Dover%2C+New+York+USA.&amp;auth=&amp;atitle=The+Genetic+Theory+of+Natural+Selection.+Dover%2C+New+York+USA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Genetic+Theory+of+Natural+Selection.+Dover%2C+New+York+USA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Genetic+Theory+of+Natural+Selection.+Dover%2C+New+York+USA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001715-Muller2" id="pone.0001715-Muller2"></a>Muller HJ (1932) Some genetic aspects of sex. Am Nat  66:   <ul class="find" data-citedArticleID="1066640" data-doi="10.1086/280418"><li><a href="http://dx.doi.org/10.1086/280418" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Some+genetic+aspects+of+sex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Some+genetic+aspects+of+sex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001715-Burch1" id="pone.0001715-Burch1"></a>Burch CL, Chao L (1999) Evolution by small steps and rugged landscapes in the RNA virus phi 6. Genetics  151: 921–927.  <ul class="find" data-citedArticleID="1066582"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+by+small+steps+and+rugged+landscapes+in+the+RNA+virus+phi+6.&amp;auth=&amp;atitle=Evolution+by+small+steps+and+rugged+landscapes+in+the+RNA+virus+phi+6." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+by+small+steps+and+rugged+landscapes+in+the+RNA+virus+phi+6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+by+small+steps+and+rugged+landscapes+in+the+RNA+virus+phi+6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001715-Miralles1" id="pone.0001715-Miralles1"></a>Miralles R, Moya A, Elena SF (2000) Diminishing returns of population size in the rate of RNA virus adaptation. J Virol  74: 3566–3571.  <ul class="find" data-citedArticleID="1066636" data-doi="10.1128/jvi.74.8.3566-3571.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.8.3566-3571.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diminishing+returns+of+population+size+in+the+rate+of+RNA+virus+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diminishing+returns+of+population+size+in+the+rate+of+RNA+virus+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001715-Orr1" id="pone.0001715-Orr1"></a>Orr HA (1998) The population genetics of adaptation: The distribution of factors fixed during adaptive evolution. Evolution  52: 935–949.  <ul class="find" data-citedArticleID="1066642" data-doi="10.2307/2411226"><li><a href="http://dx.doi.org/10.2307/2411226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+population+genetics+of+adaptation%3A+The+distribution+of+factors+fixed+during+adaptive+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+population+genetics+of+adaptation%3A+The+distribution+of+factors+fixed+during+adaptive+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001715-Rozen1" id="pone.0001715-Rozen1"></a>Rozen DE, de Visser J, Gerrish PJ (2002) Fitness effects of fixed beneficial mutations in microbial populations. Curr Biol  12: 1040–1045.  <ul class="find" data-citedArticleID="1066656" data-doi="10.1016/s0960-9822(02)00896-5"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)00896-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+effects+of+fixed+beneficial+mutations+in+microbial+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+effects+of+fixed+beneficial+mutations+in+microbial+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001715-Zeyl1" id="pone.0001715-Zeyl1"></a>Zeyl C, Vanderford T, Carter M (2003) An evolutionary advantage of haploidy in large yeast populations. Science  299: 555–558.  <ul class="find" data-citedArticleID="1066676" data-doi="10.1126/science.1078417"><li><a href="http://dx.doi.org/10.1126/science.1078417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionary+advantage+of+haploidy+in+large+yeast+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionary+advantage+of+haploidy+in+large+yeast+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001715-Barrett1" id="pone.0001715-Barrett1"></a>Barrett RDH, MacLean RC, Bell G (2006) Mutations of intermediate effect are responsible for adaptation in evolving Pseudomonas fluorescens populations. Biol Lett  2: 236–238.  <ul class="find" data-citedArticleID="1066574" data-doi="10.1098/rsbl.2006.0439"><li><a href="http://dx.doi.org/10.1098/rsbl.2006.0439" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+of+intermediate+effect+are+responsible+for+adaptation+in+evolving+Pseudomonas+fluorescens+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+of+intermediate+effect+are+responsible+for+adaptation+in+evolving+Pseudomonas+fluorescens+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001715-Kassen1" id="pone.0001715-Kassen1"></a>Kassen R, Bataillon T (2006) Distribution of fitness effects among beneficial mutations before selection in experimental populations of bacteria. Nat Genet  38: 484–488.  <ul class="find" data-citedArticleID="1066612" data-doi="10.1038/ng1751"><li><a href="http://dx.doi.org/10.1038/ng1751" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+fitness+effects+among+beneficial+mutations+before+selection+in+experimental+populations+of+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+fitness+effects+among+beneficial+mutations+before+selection+in+experimental+populations+of+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001715-Imhof1" id="pone.0001715-Imhof1"></a>Imhof M, Schlotterer C (2001) Fitness effects of advantageous mutations in evolving Escherichia coli populations. Proc Natl Acad Sci USA  98: 1113–1117.  <ul class="find" data-citedArticleID="1066608" data-doi="10.1073/pnas.98.3.1113"><li><a href="http://dx.doi.org/10.1073/pnas.98.3.1113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+effects+of+advantageous+mutations+in+evolving+Escherichia+coli+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+effects+of+advantageous+mutations+in+evolving+Escherichia+coli+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001715-DeVisser1" id="pone.0001715-DeVisser1"></a>De Visser J, Rozen DE (2005) Limits to adaptation in asexual populations. J Evol Biol  18: 779–788.  <ul class="find" data-citedArticleID="1066596" data-doi="10.1111/j.1420-9101.2005.00879.x"><li><a href="http://dx.doi.org/10.1111/j.1420-9101.2005.00879.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Limits+to+adaptation+in+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Limits+to+adaptation+in+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001715-Jain1" id="pone.0001715-Jain1"></a>Jain K, Krug J (2007) Deterministic and stochastic regimes of asexual evolution on rugged fitness landscapes. Genetics  175: 1275–1288.  <ul class="find" data-citedArticleID="1066610" data-doi="10.1534/genetics.106.067165"><li><a href="http://dx.doi.org/10.1534/genetics.106.067165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deterministic+and+stochastic+regimes+of+asexual+evolution+on+rugged+fitness+landscapes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deterministic+and+stochastic+regimes+of+asexual+evolution+on+rugged+fitness+landscapes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001715-Wright1" id="pone.0001715-Wright1"></a>Wright S (1932) The roles of mutation, inbreeding, crossbreeding, and selection in evolution. Proceedings of the Sixth International Congress on Genetics 355–366.  <ul class="find" data-citedArticleID="1066674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+roles+of+mutation%2C+inbreeding%2C+crossbreeding%2C+and+selection+in+evolution.&amp;auth=&amp;atitle=The+roles+of+mutation%2C+inbreeding%2C+crossbreeding%2C+and+selection+in+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+roles+of+mutation%2C+inbreeding%2C+crossbreeding%2C+and+selection+in+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+roles+of+mutation%2C+inbreeding%2C+crossbreeding%2C+and+selection+in+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001715-Goodnight1" id="pone.0001715-Goodnight1"></a>Goodnight CJ (2006) Population genetics-Peak shifts in large populations. Heredity  96: 5–6.  <ul class="find" data-citedArticleID="1066602" data-doi="10.1038/sj.hdy.6800746"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800746" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetics-Peak+shifts+in+large+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetics-Peak+shifts+in+large+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001715-Weinreich1" id="pone.0001715-Weinreich1"></a>Weinreich DM, Chao L (2005) Rapid evolutionary escape by large populations from local fitness peaks is likely in nature. Evolution  59: 1175–1182.  <ul class="find" data-citedArticleID="1066662" data-doi="10.1554/04-392"><li><a href="http://dx.doi.org/10.1554/04-392" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+evolutionary+escape+by+large+populations+from+local+fitness+peaks+is+likely+in+nature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+evolutionary+escape+by+large+populations+from+local+fitness+peaks+is+likely+in+nature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001715-Colegrave1" id="pone.0001715-Colegrave1"></a>Colegrave N, Buckling A (2005) Microbial experiments on adaptive landscapes. Bioessays  27: 1167–1173.  <ul class="find" data-citedArticleID="1066588" data-doi="10.1002/bies.20292"><li><a href="http://dx.doi.org/10.1002/bies.20292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+experiments+on+adaptive+landscapes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+experiments+on+adaptive+landscapes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001715-Burch2" id="pone.0001715-Burch2"></a>Burch CL, Chao L (2000) Evolvability of an RNA virus is determined by its mutational neighbourhood. Nature  406: 625–628.  <ul class="find" data-citedArticleID="1066584"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolvability+of+an+RNA+virus+is+determined+by+its+mutational+neighbourhood.&amp;auth=&amp;atitle=Evolvability+of+an+RNA+virus+is+determined+by+its+mutational+neighbourhood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolvability+of+an+RNA+virus+is+determined+by+its+mutational+neighbourhood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolvability+of+an+RNA+virus+is+determined+by+its+mutational+neighbourhood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001715-Korona1" id="pone.0001715-Korona1"></a>Korona R, Nakatsu CH, Forney LJ, Lenski RE (1994) Evidence for Multiple Adaptive Peaks from Populations of Bacteria Evolving in a Structured Habitat. Proc Natl Acad Sci USA  91: 9037–9041.  <ul class="find" data-citedArticleID="1066616" data-doi="10.1073/pnas.91.19.9037"><li><a href="http://dx.doi.org/10.1073/pnas.91.19.9037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+Multiple+Adaptive+Peaks+from+Populations+of+Bacteria+Evolving+in+a+Structured+Habitat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+Multiple+Adaptive+Peaks+from+Populations+of+Bacteria+Evolving+in+a+Structured+Habitat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001715-Weinreich2" id="pone.0001715-Weinreich2"></a>Weinreich DM, Delaney NF, DePristo MA, Hartl DL (2006) Darwinian evolution can follow only very few mutational paths to fitter proteins. Science  312: 111–114.  <ul class="find" data-citedArticleID="1066664" data-doi="10.1126/science.1123539"><li><a href="http://dx.doi.org/10.1126/science.1123539" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Darwinian+evolution+can+follow+only+very+few+mutational+paths+to+fitter+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Darwinian+evolution+can+follow+only+very+few+mutational+paths+to+fitter+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001715-Whitlock1" id="pone.0001715-Whitlock1"></a>Whitlock MC, Phillips PC, Moore FBG, Tonsor SJ (1995) Multiple Fitness Peaks and Epistasis. Annu Rev Ecol Syst  26: 601–629.  <ul class="find" data-citedArticleID="1066666" data-doi="10.1146/annurev.es.26.110195.003125"><li><a href="http://dx.doi.org/10.1146/annurev.es.26.110195.003125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+Fitness+Peaks+and+Epistasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+Fitness+Peaks+and+Epistasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001715-Hegreness1" id="pone.0001715-Hegreness1"></a>Hegreness M, Shoresh N, Hartl D, Kishony R (2006) An equivalence principle for the incorporation of favorable mutations in asexual populations. Science  311: 1615–1617.  <ul class="find" data-citedArticleID="1066606" data-doi="10.1126/science.1122469"><li><a href="http://dx.doi.org/10.1126/science.1122469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+equivalence+principle+for+the+incorporation+of+favorable+mutations+in+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+equivalence+principle+for+the+incorporation+of+favorable+mutations+in+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001715-Cooper1" id="pone.0001715-Cooper1"></a>Cooper TF, Rozen DE, Lenski RE (2003) Parallel changes in qene expression after 20,000 generations of evolution in Escherichia coli. Proc Natl Acad Sci USA  100: 1072–1077.  <ul class="find" data-citedArticleID="1066590" data-doi="10.1073/pnas.0334340100"><li><a href="http://dx.doi.org/10.1073/pnas.0334340100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+changes+in+qene+expression+after+20%2C000+generations+of+evolution+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+changes+in+qene+expression+after+20%2C000+generations+of+evolution+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001715-Pelosi1" id="pone.0001715-Pelosi1"></a>Pelosi L, Kuhn L, Guetta D, Garin J, Geiselmann J, et al.  (2006) Parallel changes in global protein profiles during long-term experimental evolution in Escherichia coli. Genetics  173: 1851–1869.  <ul class="find" data-citedArticleID="1066646" data-doi="10.1534/genetics.105.049619"><li><a href="http://dx.doi.org/10.1534/genetics.105.049619" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+changes+in+global+protein+profiles+during+long-term+experimental+evolution+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+changes+in+global+protein+profiles+during+long-term+experimental+evolution+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001715-Woods1" id="pone.0001715-Woods1"></a>Woods R, Schneider D, Winkworth CL, Riley MA, Lenski RE (2006) Tests of parallel molecular evolution in a long-term experiment with Escherichia coli. Proc Natl Acad Sci USA  103: 9107–9112.  <ul class="find" data-citedArticleID="1066672" data-doi="10.1073/pnas.0602917103"><li><a href="http://dx.doi.org/10.1073/pnas.0602917103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tests+of+parallel+molecular+evolution+in+a+long-term+experiment+with+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tests+of+parallel+molecular+evolution+in+a+long-term+experiment+with+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001715-Habets1" id="pone.0001715-Habets1"></a>Habets M, Rozen DE, Hoekstra RF, de Visser J (2006) The effect of population structure on the adaptive radiation of microbial populations evolving in spatially structured environments. Ecol Lett  9: 1041–1048.  <ul class="find" data-citedArticleID="1066604" data-doi="10.1111/j.1461-0248.2006.00955.x"><li><a href="http://dx.doi.org/10.1111/j.1461-0248.2006.00955.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+population+structure+on+the+adaptive+radiation+of+microbial+populations+evolving+in+spatially+structured+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+population+structure+on+the+adaptive+radiation+of+microbial+populations+evolving+in+spatially+structured+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001715-Manly1" id="pone.0001715-Manly1"></a>Manly BFJ (1991) Randomization and Monte Carlo Methods in Biology. London: Chapman and Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0001715-Orr2" id="pone.0001715-Orr2"></a>Orr HA (2006) The population genetics of adaptation on correlated fitness landscapes: The block model. Evolution  60: 1113–1124.  <ul class="find" data-citedArticleID="1066644" data-doi="10.1111/j.0014-3820.2006.tb01191.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2006.tb01191.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+population+genetics+of+adaptation+on+correlated+fitness+landscapes%3A+The+block+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+population+genetics+of+adaptation+on+correlated+fitness+landscapes%3A+The+block+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001715-Perelson1" id="pone.0001715-Perelson1"></a>Perelson AS, Macken CA (1995) Protein Evolution on Partially Correlated Landscapes. Proc Natl Acad Sci USA  92: 9657–9661.  <ul class="find" data-citedArticleID="1066648" data-doi="10.1073/pnas.92.21.9657"><li><a href="http://dx.doi.org/10.1073/pnas.92.21.9657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+Evolution+on+Partially+Correlated+Landscapes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+Evolution+on+Partially+Correlated+Landscapes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001715-Macken1" id="pone.0001715-Macken1"></a>Macken CA, Perelson AS (1989) Protein Evolution on Rugged Landscapes. Proc Natl Acad Sci USA  86: 6191–6195.  <ul class="find" data-citedArticleID="1066630" data-doi="10.1073/pnas.86.16.6191"><li><a href="http://dx.doi.org/10.1073/pnas.86.16.6191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+Evolution+on+Rugged+Landscapes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+Evolution+on+Rugged+Landscapes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001715-Kauffman1" id="pone.0001715-Kauffman1"></a>Kauffman S, Levin S (1987) Towards a General-Theory of Adaptive Walks on Rugged Landscapes. J Theor Biol  128: 11–45.  <ul class="find" data-citedArticleID="1066614" data-doi="10.1016/s0022-5193(87)80029-2"><li><a href="http://dx.doi.org/10.1016/s0022-5193(87)80029-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+a+General-Theory+of+Adaptive+Walks+on+Rugged+Landscapes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+a+General-Theory+of+Adaptive+Walks+on+Rugged+Landscapes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001715-Coyne1" id="pone.0001715-Coyne1"></a>Coyne JA, Barton NH, Turelli M (1997) Perspective: A critique of Sewall Wright's shifting balance theory of evolution. Evolution  51: 643–671.  <ul class="find" data-citedArticleID="1066594" data-doi="10.2307/2411143"><li><a href="http://dx.doi.org/10.2307/2411143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perspective%3A+A+critique+of+Sewall+Wright%27s+shifting+balance+theory+of+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perspective%3A+A+critique+of+Sewall+Wright%27s+shifting+balance+theory+of+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001715-Korona2" id="pone.0001715-Korona2"></a>Korona R (1996) Adaptation to structurally different environments. Proc R Soc Lond, Ser B: Biol Sci  263: 1665–1669.  <ul class="find" data-citedArticleID="1066618" data-doi="10.1098/rspb.1996.0243"><li><a href="http://dx.doi.org/10.1098/rspb.1996.0243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptation+to+structurally+different+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptation+to+structurally+different+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001715-Rainey1" id="pone.0001715-Rainey1"></a>Rainey PB, Travisano M (1998) Adaptive radiation in a heterogeneous environment. Nature  394: 69–72.  <ul class="find" data-citedArticleID="1066652" data-doi="10.1038/27900"><li><a href="http://dx.doi.org/10.1038/27900" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+radiation+in+a+heterogeneous+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+radiation+in+a+heterogeneous+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001715-Brockhurst1" id="pone.0001715-Brockhurst1"></a>Brockhurst MA, Buckling A, Rainey PB (2005) The effect of a bacteriophage on diversification of the opportunistic bacterial pathogen, Pseudomonas aeruginosa. Proc R Soc Lond, Ser B: Biol Sci  272: 1385–1391.  <ul class="find" data-citedArticleID="1066578" data-doi="10.1098/rspb.2005.3086"><li><a href="http://dx.doi.org/10.1098/rspb.2005.3086" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+a+bacteriophage+on+diversification+of+the+opportunistic+bacterial+pathogen%2C+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+a+bacteriophage+on+diversification+of+the+opportunistic+bacterial+pathogen%2C+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001715-Brockhurst2" id="pone.0001715-Brockhurst2"></a>Brockhurst MA, Rainey PB, Buckling A (2004) The effect of spatial heterogeneity and parasites on the evolution of host diversity. Proc R Soc Lond, Ser B: Biol Sci  271: 107–111.  <ul class="find" data-citedArticleID="1066580" data-doi="10.1098/rspb.2003.2556"><li><a href="http://dx.doi.org/10.1098/rspb.2003.2556" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+spatial+heterogeneity+and+parasites+on+the+evolution+of+host+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+spatial+heterogeneity+and+parasites+on+the+evolution+of+host+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001715-Meyer1" id="pone.0001715-Meyer1"></a>Meyer JR, Kassen R (2007) The effects of competition and predation on diversification in a model adaptive radiation. Nature  446: 432–435.  <ul class="find" data-citedArticleID="1066634" data-doi="10.1038/nature05599"><li><a href="http://dx.doi.org/10.1038/nature05599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+competition+and+predation+on+diversification+in+a+model+adaptive+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+competition+and+predation+on+diversification+in+a+model+adaptive+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001715-Rubin1" id="pone.0001715-Rubin1"></a>Rubin LG (1987) Bacterial-Colonization and Infection Resulting from Multiplication of a Single Organism. Reviews of Infectious Diseases  9: 488–493.  <ul class="find" data-citedArticleID="1066658" data-doi="10.1093/clinids/9.3.488"><li><a href="http://dx.doi.org/10.1093/clinids/9.3.488" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial-Colonization+and+Infection+Resulting+from+Multiplication+of+a+Single+Organism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial-Colonization+and+Infection+Resulting+from+Multiplication+of+a+Single+Organism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001715-Bergstrom1" id="pone.0001715-Bergstrom1"></a>Bergstrom CT, McElhany P, Real LA (1999) Transmission bottlenecks as determinants of virulence in rapidly evolving pathogens. Proc Natl Acad Sci USA  96: 5095–5100.  <ul class="find" data-citedArticleID="1066576" data-doi="10.1073/pnas.96.9.5095"><li><a href="http://dx.doi.org/10.1073/pnas.96.9.5095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transmission+bottlenecks+as+determinants+of+virulence+in+rapidly+evolving+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transmission+bottlenecks+as+determinants+of+virulence+in+rapidly+evolving+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001715-Lenski1" id="pone.0001715-Lenski1"></a>Lenski RE, Rose MR, Simpson SC, Tadler SC (1991) Long-Term Experimental Evolution in Escherichia-Coli .1. Adaptation and Divergence During 2,000 Generations. Am Nat  138: 1315–1341.  <ul class="find" data-citedArticleID="1066622" data-doi="10.1086/285289"><li><a href="http://dx.doi.org/10.1086/285289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-Term+Experimental+Evolution+in+Escherichia-Coli+.1.+Adaptation+and+Divergence+During+2%2C000+Generations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-Term+Experimental+Evolution+in+Escherichia-Coli+.1.+Adaptation+and+Divergence+During+2%2C000+Generations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001715-Cowperthwaite1" id="pone.0001715-Cowperthwaite1"></a>Cowperthwaite MC, Bull JJ, Meyers LA (2006) From bad to good: Fitness reversals and the ascent of deleterious mutations. PLoS Comp Biol  2: 1292–1300.  <ul class="find" data-citedArticleID="1066592" data-doi="10.1371/journal.pcbi.0020141"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+bad+to+good%3A+Fitness+reversals+and+the+ascent+of+deleterious+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+bad+to+good%3A+Fitness+reversals+and+the+ascent+of+deleterious+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001715-Lenski2" id="pone.0001715-Lenski2"></a>Lenski RE, Ofria C, Pennock RT, Adami C (2003) The evolutionary origin of complex features. Nature  423: 139–144.  <ul class="find" data-citedArticleID="1066624" data-doi="10.1038/nature01568"><li><a href="http://dx.doi.org/10.1038/nature01568" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+origin+of+complex+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+origin+of+complex+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001715-Perfeito1" id="pone.0001715-Perfeito1"></a>Perfeito L, Fernandes L, Mota C, Gordo I (2007) Adaptive mutations in bacteria: High rate and small effects. Science  317: 813–815.  <ul class="find" data-citedArticleID="1066650" data-doi="10.1126/science.1142284"><li><a href="http://dx.doi.org/10.1126/science.1142284" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+mutations+in+bacteria%3A+High+rate+and+small+effects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+mutations+in+bacteria%3A+High+rate+and+small+effects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.XML" value="77956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.PDF" value="226284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g001.PNG_L" value="92725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g001.PNG_M" value="24680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g001.PNG_S" value="5289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g001.TIF" value="217724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g001.PNG_I" value="9023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g002.PNG_L" value="370046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g002.PNG_M" value="31569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g002.PNG_S" value="4935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g002.TIF" value="650472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g002.PNG_I" value="11913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g003.PNG_L" value="2113853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g003.PNG_M" value="134969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g003.PNG_S" value="7649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g003.TIF" value="2580816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001715.g003.PNG_I" value="43001"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001715&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001715" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001715&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001715&volume=&issue=&title=Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes&author_name=Daniel%20E.%20Rozen%2C%20Michelle%20G.%20J.%20L.%20Habets%2C%20Andreas%20Handel%2C%20J.%20Arjan%20G.%20M.%20de%20Visser&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001715" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001715" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001715" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001715&amp;t=Heterogeneous%20Adaptive%20Trajectories%20of%20Small%20Populations%20on%20Complex%20Fitness%20Landscapes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001715&title=Heterogeneous%20Adaptive%20Trajectories%20of%20Small%20Populations%20on%20Complex%20Fitness%20Landscapes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001715&amp;title=Heterogeneous%20Adaptive%20Trajectories%20of%20Small%20Populations%20on%20Complex%20Fitness%20Landscapes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001715" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001715&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Heterogeneous Adaptive Trajectories of Small Populations on Complex Fitness Landscapes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001715';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Heterogeneous%20Adaptive%20Trajectories%20of%20Small%20Populations%20on%20Complex%20Fitness%20Landscapes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001715" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001715" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+adaptation%22" title="Search for articles in the subject area:'Evolutionary adaptation'"><div class="flagText">Evolutionary adaptation</div></a>
              <div data-categoryid="17795" data-articleid="26256"
                   data-categoryname="Evolutionary adaptation"
                   class="flagImage" title="Flag 'Evolutionary adaptation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fitness+epistasis%22" title="Search for articles in the subject area:'Fitness epistasis'"><div class="flagText">Fitness epistasis</div></a>
              <div data-categoryid="22323" data-articleid="26256"
                   data-categoryname="Fitness epistasis"
                   class="flagImage" title="Flag 'Fitness epistasis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+deletion%22" title="Search for articles in the subject area:'Gene deletion'"><div class="flagText">Gene deletion</div></a>
              <div data-categoryid="33017" data-articleid="26256"
                   data-categoryname="Gene deletion"
                   class="flagImage" title="Flag 'Gene deletion' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microbial+mutation%22" title="Search for articles in the subject area:'Microbial mutation'"><div class="flagText">Microbial mutation</div></a>
              <div data-categoryid="20011" data-articleid="26256"
                   data-categoryname="Microbial mutation"
                   class="flagImage" title="Flag 'Microbial mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="26256"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+size%22" title="Search for articles in the subject area:'Population size'"><div class="flagText">Population size</div></a>
              <div data-categoryid="33245" data-articleid="26256"
                   data-categoryname="Population size"
                   class="flagImage" title="Flag 'Population size' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Substitution+mutation%22" title="Search for articles in the subject area:'Substitution mutation'"><div class="flagText">Substitution mutation</div></a>
              <div data-categoryid="35145" data-articleid="26256"
                   data-categoryname="Substitution mutation"
                   class="flagImage" title="Flag 'Substitution mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topography%22" title="Search for articles in the subject area:'Topography'"><div class="flagText">Topography</div></a>
              <div data-categoryid="22509" data-articleid="26256"
                   data-categoryname="Topography"
                   class="flagImage" title="Flag 'Topography' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7891'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2459'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=460&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=15781">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=15177">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
