

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Short RNA Guides Cleavage by Eukaryotic RNase III</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000472"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000472" />

    <meta name="citation_title" content="Short RNA Guides Cleavage by Eukaryotic RNase III"/>
    <meta itemprop="name" content="Short RNA Guides Cleavage by Eukaryotic RNase III"/>

      <meta name="citation_author" content="Bruno Lamontagne"/>
            <meta name="citation_author_institution" content="Groupe ARN (RNA Group), Laboratoire de Génomique Fonctionnelle (Laboratory for Functional Genomics), Département de Microbiologie et d'Infectiologie, Faculté de Médecine et des Sciences de la Santé, Université de Sherbrooke, Sherbrooke, Québec, Canada"/>
      <meta name="citation_author" content="Sherif Abou Elela"/>
            <meta name="citation_author_institution" content="Groupe ARN (RNA Group), Laboratoire de Génomique Fonctionnelle (Laboratory for Functional Genomics), Département de Microbiologie et d'Infectiologie, Faculté de Médecine et des Sciences de la Santé, Université de Sherbrooke, Sherbrooke, Québec, Canada"/>

    <meta name="citation_date" content="2007/5/30"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000472.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e472"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Dicers at RISC; the mechanism of RNAi.; citation_author=M Tijsterman; citation_author=RH Plasterk; citation_journal_title=Cell; citation_volume=117; citation_number=1; citation_pages=1-3; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=siRNA-mediated gene silencing in vitro and in vivo.; citation_author=H Xia; citation_author=Q Mao; citation_author=HL Paulson; citation_author=BL Davidson; citation_journal_title=Nat Biotechnol; citation_volume=20; citation_number=2; citation_pages=1006-1010; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The RNase III family: a conserved structure and expanding functions in eukaryotic dsRNA metabolism.; citation_author=B Lamontagne; citation_author=S Larose; citation_author=J Boulanger; citation_author=SA Elela; citation_journal_title=Curr Issues Mol Biol; citation_volume=3; citation_number=3; citation_pages=71-78; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The global regulator RNase III modulates translation repression by the transcription elongation factor N.; citation_author=HR Wilson; citation_author=D Yu; citation_author=HK Peters; citation_author=JG Zhou; citation_author=DL Court; citation_journal_title=Embo J; citation_volume=21; citation_number=4; citation_pages=4154-4161; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNase III-Mediated Silencing of a Glucose-Dependent Repressor in Yeast.; citation_author=D Ge; citation_author=B Lamontagne; citation_author=SA Elela; citation_journal_title=Curr Biol; citation_volume=15; citation_number=5; citation_pages=140-145; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RNase III-dependent Regulation of Yeast Telomerase.; citation_author=S Larose; citation_author=N Laterreur; citation_author=G Ghazal; citation_author=J Gagnon; citation_author=RJ Wellinger; citation_journal_title=J Biol Chem; citation_volume=282; citation_number=6; citation_pages=4373-4381; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Multiple RNA surveillance pathways limit aberrant expression of iron uptake mRNAs and prevent iron toxicity in S. cerevisiae.; citation_author=A Lee; citation_author=AK Henras; citation_author=G Chanfreau; citation_journal_title=Mol Cell; citation_volume=19; citation_number=7; citation_pages=39-51; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural insight into the mechanism of double-stranded RNA processing by ribonuclease III.; citation_author=J Gan; citation_author=JE Tropea; citation_author=BP Austin; citation_author=DL Court; citation_author=DS Waugh; citation_journal_title=Cell; citation_volume=124; citation_number=8; citation_pages=355-366; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Depletion of yeast RNase III blocks correct U2 3′ end formation and results in polyadenylated but functional U2 snRNA.; citation_author=S Abou Elela; citation_author=M Ares; citation_journal_title=Embo J; citation_volume=17; citation_number=9; citation_pages=3738-3746; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=RNase III cleaves eukaryotic preribosomal RNA at a U3 snoRNP-dependent site.; citation_author=S Abou Elela; citation_author=H Igel; citation_author=M Ares; citation_journal_title=Cell; citation_volume=85; citation_number=10; citation_pages=115-124; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genome-wide prediction and analysis of yeast RNase III-dependent snoRNA processing signals.; citation_author=G Ghazal; citation_author=D Ge; citation_author=J Gervais-Bird; citation_author=J Gagnon; citation_author=S Abou Elela; citation_journal_title=Mol Cell Biol; citation_volume=25; citation_number=11; citation_pages=2981-2994; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular requirements for duplex recognition and cleavage by eukaryotic RNase III: discovery of an RNA-dependent DNA cleavage activity by yeast Rnt1p.; citation_author=B Lamontagne; citation_author=RN Hannoush; citation_author=MJ Damha; citation_author=S Aboul Elela; citation_journal_title=J Mol Biol; citation_volume=338; citation_number=12; citation_pages=401-418; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recognition of a conserved class of RNA tetraloops by Saccharomyces cerevisiae RNase III.; citation_author=G Chanfreau; citation_author=M Buckle; citation_author=A Jacquier; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=13; citation_pages=3142-3147; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the RNA determinants for bacterial and yeast RNase III binding and cleavage.; citation_author=B Lamontagne; citation_author=SA Elela; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=14; citation_pages=2231-2241; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Solution structure of conserved AGNN tetraloops: insights into Rnt1p RNA processing.; citation_author=I Lebars; citation_author=B Lamontagne; citation_author=S Yoshizawa; citation_author=S Aboul-Elela; citation_author=D Fourmy; citation_journal_title=Embo J; citation_volume=20; citation_number=15; citation_pages=7250-7258; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Characterization of the reactivity determinants of a novel hairpin substrate of yeast RNase III.; citation_author=G Ghazal; citation_author=SA Elela; citation_journal_title=J Mol Biol; citation_volume=363; citation_number=16; citation_pages=332-344; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence dependence of substrate recognition and cleavage by yeast RNase III.; citation_author=B Lamontagne; citation_author=G Ghazal; citation_author=I Lebars; citation_author=S Yoshizawa; citation_author=D Fourmy; citation_journal_title=J Mol Biol; citation_volume=327; citation_number=17; citation_pages=985-1000; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Alternative 3′-end processing of U5 snRNA by RNase III.; citation_author=G Chanfreau; citation_author=SA Elela; citation_author=M Ares; citation_author=C Guthrie; citation_journal_title=Genes Dev; citation_volume=11; citation_number=18; citation_pages=2741-2751; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Small nucleolar RNAs: an abundant group of noncoding RNAs with diverse cellular functions.; citation_author=T Kiss; citation_journal_title=Cell; citation_volume=109; citation_number=19; citation_pages=145-148; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Argonaute2, a link between genetic and biochemical analyses of RNAi.; citation_author=SM Hammond; citation_author=S Boettcher; citation_author=AA Caudy; citation_author=R Kobayashi; citation_author=GJ Hannon; citation_journal_title=Science; citation_volume=293; citation_number=20; citation_pages=1146-1150; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=5′-,3′-inverted thymidine-modified antisense oligodeoxynucleotide targeting midkine. Its design and application for cancer therapy.; citation_author=Y Takei; citation_author=K Kadomatsu; citation_author=H Itoh; citation_author=W Sato; citation_author=K Nakazawa; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=21; citation_pages=23800-23806; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=High-efficiency transformation of yeast by electroporation.; citation_author=DM Becker; citation_author=L Guarente; citation_journal_title=Methods Enzymol; citation_volume=194; citation_number=22; citation_pages=182-187; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=U2 RNA from yeast is unexpectedly large and contains homology to vertebrate U4, U5, and U6 small nuclear RNAs.; citation_author=M Ares; citation_journal_title=Cell; citation_volume=47; citation_number=23; citation_pages=49-59; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Efficient reduction of target RNAs by small interfering RNA and RNase H-dependent antisense agents. A comparative analysis.; citation_author=TA Vickers; citation_author=S Koo; citation_author=CF Bennett; citation_author=ST Crooke; citation_author=NM Dean; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=24; citation_pages=7108-7118; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ribonucleases H of the budding yeast, Saccharomyces cerevisiae.; citation_author=U Wintersberger; citation_author=P Frank; citation_journal_title=Methods Enzymol; citation_volume=341; citation_number=25; citation_pages=414-429; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Small nuclear RNAs from Saccharomyces cerevisiae: unexpected diversity in abundance, size, and molecular complexity.; citation_author=N Riedel; citation_author=JA Wise; citation_author=H Swerdlow; citation_author=A Mak; citation_author=C Guthrie; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=83; citation_number=26; citation_pages=8097-8101; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Evolution of secondary metabolite production: potential roles for antibiotics as prebiotic effectors of catalytic RNA reactions.; citation_author=J Davies; citation_author=U von Ahsen; citation_author=H Wank; citation_author=R Schroeder; citation_journal_title=Ciba Found Symp; citation_volume=171; citation_number=27; citation_pages=24-32; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=RNA as an enzyme.; citation_author=TR Cech; citation_journal_title=Biochem Int; citation_volume=18; citation_number=28; citation_pages=7-14; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Relics from the RNA world.; citation_author=DC Jeffares; citation_author=AM Poole; citation_author=D Penny; citation_journal_title=J Mol Evol; citation_volume=46; citation_number=29; citation_pages=18-36; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The path from the RNA world.; citation_author=AM Poole; citation_author=DC Jeffares; citation_author=D Penny; citation_journal_title=J Mol Evol; citation_volume=46; citation_number=30; citation_pages=1-17; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Small regulatory RNAs in mammals.; citation_author=JS Mattick; citation_author=IV Makunin; citation_journal_title=Hum Mol Genet; citation_volume=14 Spec No 1; citation_number=31; citation_pages=R121-132; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Birth of the snoRNPs: the evolution of the modification-guide snoRNAs.; citation_author=DL Lafontaine; citation_author=D Tollervey; citation_journal_title=Trends Biochem Sci; citation_volume=23; citation_number=32; citation_pages=383-388; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Gene silencing in vitro and in vivo using intronic microRNAs.; citation_author=SL Lin; citation_author=SY Ying; citation_journal_title=Methods Mol Biol; citation_volume=342; citation_number=33; citation_pages=295-312; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=miRNAs on the move: miRNA biogenesis and the RNAi machinery.; citation_author=EP Murchison; citation_author=GJ Hannon; citation_journal_title=Curr Opin Cell Biol; citation_volume=16; citation_number=34; citation_pages=223-229; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Yeast as a model system for understanding the control of DNA replication in Eukaryotes.; citation_author=R Bartlett; citation_author=P Nurse; citation_journal_title=Bioessays; citation_volume=12; citation_number=35; citation_pages=457-463; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Apoptosis in yeast: a new model system with applications in cell biology and medicine.; citation_author=F Madeo; citation_author=S Engelhardt; citation_author=E Herker; citation_author=N Lehmann; citation_author=C Maldener; citation_journal_title=Curr Genet; citation_volume=41; citation_number=36; citation_pages=208-216; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cell cycle-dependent nuclear localization of yeast RNase III is required for efficient cell division.; citation_author=M Catala; citation_author=B Lamontagne; citation_author=S Larose; citation_author=G Ghazal; citation_author=SA Elela; citation_journal_title=Mol Biol Cell; citation_volume=15; citation_number=37; citation_pages=3015-3030; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Guide to Yeast Genetics and Molecular Biology.; citation_author=C Guthrie; citation_author=GR Fink; citation_number=38; citation_date=1991; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Methods in Yeast Genetics: A Laboratory Course Manual.; citation_author=MD Rose; citation_author=F Winston; citation_author=P Hieter; citation_number=39; citation_date=1990; citation_publisher=Cold Spring Harbor; " />
      <meta name="citation_reference" content="citation_title=A novel base-pairing interaction between U2 and U6 snRNAs suggests a mechanism for the catalytic activation of the spliceosome.; citation_author=HD Madhani; citation_author=C Guthrie; citation_journal_title=Cell; citation_volume=71; citation_number=40; citation_pages=803-817; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Purification and characterization of Saccharomyces cerevisiae Rnt1p nuclease.; citation_author=B Lamontagne; citation_author=S Abou Elela; citation_journal_title=Methods Enzymol; citation_volume=342; citation_number=41; citation_pages=159-167; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Lethal and temperature-sensitive mutations and their suppressors identify an essential structural element in U2 small nuclear RNA.; citation_author=M Ares; citation_author=AH Igel; citation_journal_title=Genes Dev; citation_volume=4; citation_number=42; citation_pages=2132-2145; citation_date=1990; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000472" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Short RNA Guides Cleavage by Eukaryotic RNase III"/>
    <meta name="twitter:description" content="In eukaryotes, short RNAs guide a variety of enzymatic activities that range from RNA editing to translation repression. It is hypothesized that pre-existing proteins evolved to bind and use guide RNA during evolution. However, the capacity of modern proteins to adopt new RNA guides has never been demonstrated. Here we show that Rnt1p, the yeast orthologue of the bacterial dsRNA-specific RNase III, can bind short RNA transcripts and use them as guides for sequence-specific cleavage. Target cleavage occurred at a constant distance from the Rnt1p binding site, leaving the guide RNA intact for subsequent cleavage. Our results indicate that RNase III may trigger sequence-specific RNA degradation independent of the RNAi machinery, and they open the road for a new generation of precise RNA silencing tools that do not trigger a dsRNA-mediated immune response."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000472.g006"/>

  <meta property="og:title" content="Short RNA Guides Cleavage by Eukaryotic RNase III" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000472" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4740'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5490'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3726&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000472">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000472" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000472&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Short RNA Guides Cleavage by Eukaryotic RNase III
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bruno Lamontagne, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Groupe ARN (RNA Group), Laboratoire de Génomique Fonctionnelle (Laboratory for Functional Genomics), Département de Microbiologie et d'Infectiologie, Faculté de Médecine et des Sciences de la Santé, Université de Sherbrooke, Sherbrooke, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sherif Abou Elela
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Sherif.Abou.Elela@USherbrooke.ca">Sherif.Abou.Elela@USherbrooke.ca</a></p>

                <p>Affiliation:
                  Groupe ARN (RNA Group), Laboratoire de Génomique Fonctionnelle (Laboratory for Functional Genomics), Département de Microbiologie et d'Infectiologie, Faculté de Médecine et des Sciences de la Santé, Université de Sherbrooke, Sherbrooke, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 30, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000472</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000472" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000472" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000472" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000472" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000472-g001" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000472-t001" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000472-g002" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000472-g003" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000472-t002" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000472-g004" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000472-g005" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000472-g006" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000472">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000472" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>In eukaryotes, short RNAs guide a variety of enzymatic activities that range from RNA editing to translation repression. It is hypothesized that pre-existing proteins evolved to bind and use guide RNA during evolution. However, the capacity of modern proteins to adopt new RNA guides has never been demonstrated. Here we show that Rnt1p, the yeast orthologue of the bacterial dsRNA-specific RNase III, can bind short RNA transcripts and use them as guides for sequence-specific cleavage. Target cleavage occurred at a constant distance from the Rnt1p binding site, leaving the guide RNA intact for subsequent cleavage. Our results indicate that RNase III may trigger sequence-specific RNA degradation independent of the RNAi machinery, and they open the road for a new generation of precise RNA silencing tools that do not trigger a dsRNA-mediated immune response.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lamontagne B, Abou Elela S (2007) Short RNA Guides Cleavage by Eukaryotic RNase III. PLoS ONE 2(5):
          e472.
            doi:10.1371/journal.pone.0000472</p><p><strong>Academic Editor: </strong>Thomas Preiss, Victor Chang Cardiac Research Institute, Australia</p><p><strong>Received:</strong> April 17, 2007; <strong>Accepted:</strong> May 2, 2007; <strong>Published:</strong> May 30, 2007</p><p><strong>Copyright:</strong> © 2007 Lamontagne, Abou Elela. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by funds from the Canadian Institutes for Health Research (CIHR) and Genome Canada/Genome Quebec Competition III. S.A. is a Chercheur-Boursier Senior of the Fonds de la Recherche en Santé du Québec.</p><p><strong>Competing interests:</strong> For clarity and transparency we declare a potential duality of interest because we have submitted a patent covering this discovery. However, there is no value for this primary patent submission and we have no reason to believe at this time that the publication will affect neither the potential nor value of the potential patent.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The capacity of short RNA duplexes to direct sequence-specific RNA degradation provides an almost universal tool for design-based gene silencing. This technique termed RNA interference (RNAi), is initiated by either endogenous RNA duplexes generated by members of the RNase III family (e.g. Drosha and Dicer)<a href="#pone.0000472-Tijsterman1">[1]</a> or through the introduction of exogenous duplexes<a href="#pone.0000472-Xia1">[2]</a>. However, the components of the RNAi machinery, with the exception of RNase III<a href="#pone.0000472-Lamontagne1">[3]</a>, are not conserved in bacteria and certain eukaryotes including <em>Saccharomyces cerevisiae</em>. In these organisms, most documented gene specific mRNA degradation events including those performed by RNase III are not sequence but structure dependent<a href="#pone.0000472-Wilson1">[4]</a>–<a href="#pone.0000472-Lee1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Members of the RNase III family share a conserved dsRNA-binding domain (dsRBD) and a catalytic domain<a href="#pone.0000472-Lamontagne1">[3]</a>, <a href="#pone.0000472-Gan1">[8]</a>. In yeast there is only one isoform of RNase III (Rnt1p)<a href="#pone.0000472-Lamontagne1">[3]</a> involved in the processing of several non-coding RNAs<a href="#pone.0000472-AbouElela1">[9]</a>–<a href="#pone.0000472-Ghazal1">[11]</a> and the degradation of a wide variety of mRNAs<a href="#pone.0000472-Ge1">[5]</a>, <a href="#pone.0000472-Larose1">[6]</a>. Unlike most RNase IIIs, Rnt1p has reduced affinity for generic A-form helix and instead recognizes hairpins as short as 5 base pairs (bp) when capped with NGNN or AAGU tetraloops<a href="#pone.0000472-Lamontagne2">[12]</a>. Rnt1p's substrates are cleaved 14 and 16 nucleotides (nts) away from the terminal tetraloop making Rnt1p a helical ruler <a href="#pone.0000472-Chanfreau1">[13]</a>, <a href="#pone.0000472-Lamontagne3">[14]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Since RNAi does not exist in <em>S. cerevisiae</em>, we asked whether there is an independent strategy to target specific RNA sequences for degradation by RNase III. Our hypothesis was that Rnt1p could function as an RNP complex and use a small RNA guide to cleave a specific RNA sequence. To test this hypothesis we generated RNA transcripts containing a 5 bp hairpin that binds Rnt1p fused to sequences complementary to different RNA targets. As predicted, the different RNA guides successfully bound to Rnt1p and directed a specific cleavage at a fixed distance from the RNA hairpin <em>in vitro</em> and reduced the expression of abundant nuclear RNAs <em>in vivo</em>. Together, our data indicate that RNase III may function as a sequence specific RNP complex and reveal a new approach for the regulation of nuclear RNA.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Rnt1p does not require a complete RNA helix for cleavage</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Most RNase IIIs and other dsRBPs identify their substrate by recognizing the distance between the minor grooves generated by one turn (i.e. 11 bp) of an A-form RNA helix<a href="#pone.0000472-Lamontagne3">[14]</a>. In contrast, yeast Rnt1p has low affinity to duplex RNA and instead recognizes the fold of NGNN<a href="#pone.0000472-Lebars1">[15]</a> or AAGU<a href="#pone.0000472-Ghazal2">[16]</a> tetraloops suggesting that this enzyme may not require the conventional 11 bp duplex for cleavage<a href="#pone.0000472-Lamontagne2">[12]</a>, <a href="#pone.0000472-Lamontagne3">[14]</a>. In order to determine the minimum length of the RNA duplex required for Rnt1p cleavage, we generated a series of RNA transcripts with a fixed tetraloop sequence but with different single and double-stranded RNA lengths (<a href="#pone-0000472-g001">Figure 1A</a>). The sequence of the first three substrates was based on the Rnt1p cleavage signals found at the 3′-end of U2 snRNA because it was previously shown that this RNA could be cleaved even when the cleavage site is unpaired<a href="#pone.0000472-AbouElela1">[9]</a>. The cleavage efficiencies of the different substrates were compared under single (trace RNA amount) and multiple (1:8 protein excess) turnover conditions (<a href="#pone-0000472-g001">Figure 1B</a>). As expected, the U2 3′-end model substrate (U2C) was cleaved by Rnt1p in all conditions at the expected fixed distance 14 and 16 nucleotides from the terminal tetraloop (<a href="#pone-0000472-g001">Figure 1B</a>). Deletion of the nucleotides in the stem at the 3′-end of the tetraloop (U2LE), which leaves an eleven base-pair stem linked to an 18 nucleotide-5′ extension, was cleaved by Rnt1p once at 14 nts from the terminal tetraloop (<a href="#pone-0000472-g001">Figure 1B</a>). Similarly, the deletion of the 5′-end (U2RI) did not inhibit the cleavage at the 3′-end extension. The cleavage kinetics of these different substrates indicate that reducing the duplex length reduces Rnt1p's turnover but increases catalytic efficiency (<a href="#pone-0000472-t001">Table 1</a>). These data confirm that Rnt1p can cleave single-stranded RNA and suggests that long RNA duplexes are not required for cleavage.</p>
<div class="figure" id="pone-0000472-g001"><div class="img"><a name="pone-0000472-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Rnt1p does not require a complete A-form helix for cleavage.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Schematic representations of Rnt1p substrates used in B and C. U2C, U2LE, and U2RI were derived from Rnt1p cleavage site at the 3′-end of U2 snRNA<a href="#pone.0000472-AbouElela1">[9]</a>. EL18-18, EL18-15, EL18/5′, and EL18/3′ are derived from the cleavage site at the 3′ end of U5 snRNA<a href="#pone.0000472-Chanfreau2">[18]</a>. The arrowheads indicate major Rnt1p cleavage sites. (B) and (C) The different 5′-end labeled substrates were incubated in the absence (N) or presence of recombinant Rnt1p. Cleavage was carried out either in enzyme excess to measure the single turnover rate (ST) or in RNA excess to measure the multiple turnover rate (MT). The cleavage products were fractionated by 20% denaturing PAGE and visualized by autoradiogram. The cleavage efficiencies are presented as fractional velocities relative to the parental substrate. The values reflect the average of three independent experiments. The RNA marker (M) is indicated on the left. The positions of the cleavage products (P) and the substrates (S) are indicated by arrowheads on the right.</p>
<span>doi:10.1371/journal.pone.0000472.g001</span></div><div class="figure" id="pone-0000472-t001"><div class="img"><a name="pone-0000472-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Kinetic parameters of Rnt1p cleavage of U2 snRNA 3′-end stem-loop derivatives</span></strong></p><span>doi:10.1371/journal.pone.0000472.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>RNA footprinting, chemical interference, and binding assays indicate that a minimum of a 5 bp stem capped with a NGNN tetraloop is required for Rnt1p binding<a href="#pone.0000472-Ghazal2">[16]</a>, <a href="#pone.0000472-Lamontagne4">[17]</a>. However, it is not clear whether the binding of this 5 bp short stem reflects the natural mechanism of substrate selection or arises from a non-specific or unproductive mode of binding. To differentiate between these two possibilities, we synthesized RNA substrates derived from a cleavage site with a known tertiary structure<a href="#pone.0000472-Lebars1">[15]</a>, <a href="#pone.0000472-Chanfreau2">[18]</a>. The engineered substrates consist of U5 snRNA 5 bp stem<a href="#pone.0000472-Chanfreau2">[18]</a> attached to heterologous ssRNA extensions at their 5′- (EL18/5′), 3′- (EL18/3′) or at both ends (EL18-18 and EL18-15) (<a href="#pone-0000472-g001">Figure 1A</a>). Interestingly, Rnt1p only cleaved substrates with two single-stranded RNA extensions (EL18-18 and EL18-15). (<a href="#pone-0000472-g001">Figure 1C</a>). Additional assays using a variety of substrates indicated that a minimal 9 and 11 nucleotide extensions at the 5′- and 3′-ends respectively are required for cleavage (data not shown). This indicates that RNA helices longer than 11 bp are not required for cleavage by Rnt1p.</p>


<h4>Directing Rnt1p cleavage using an RNA guide</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The capacity of Rnt1p to form a stable complex with short RNA hairpins underscores the capacity of this enzyme to form RNP complexes under certain conditions. Rnt1p/RNA complexes are catalytically active since they mediate cleavage when attached to single-stranded RNA extensions (<a href="#pone-0000472-g001">Figure 1</a>). This is reminiscent of known RNP complexes like the snoRNP<a href="#pone.0000472-Kiss1">[19]</a> and the RISC complexes<a href="#pone.0000472-Hammond1">[20]</a> that use RNA as a guide to modify or cleave an independent RNA molecule <em>in trans</em>. Therefore, Rnt1p cleavage may also be guided by short RNA transcripts. To test this possibility, we generated RNA transcripts that contain an Rnt1p binding signal (5 bp stem) fused to RNA extensions complementary to the sequence of an independently transcribed single-stranded RNA fragment. One guide had long extensions (EL18-15) to allow cleavage in both guide and target RNA (<a href="#pone-0000472-g002">Figure 2A</a>) while the other had short extensions (EL9-11) to allow cleavage only in the target RNA (TL) (<a href="#pone-0000472-g002">Figure 2B</a>). As expected, a gel shift assay indicated that both guides bound their 72 nts long RNA substrate (TL) with similar efficiency (<a href="#pone-0000472-g002">Figures 2C and D</a>). The short guide EL9-11 readily formed a complex with its targeted RNA, while the long guide EL18-15 formed complexes only when the RNA mixture was pre-heated. This suggests that 20 nt long complementarities between the guide (EL9-11) and its target RNA are sufficient for complex formation. On the other hand, additional extensions in the guide sequence may increase the chance of intramolecular secondary structure, which may occlude target identification.</p>
<div class="figure" id="pone-0000472-g002"><div class="img"><a name="pone-0000472-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Rnt1p cleaves intermolecular RNA substrates.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Illustrations of an RNA guide that could be cleaved by Rnt1p (EL18-15) (A) or a guide that cannot be cleaved by Rnt1p (EL9-11) (B) when in complex with the target RNA (TL). The arrowheads indicate the positions of the observed cleavage sites within the guide and target sequences. (C) and (D) illustrate the gel shift assay used to monitor the interaction of EL18-15 and EL9-11 with the target sequence (TL). The reaction products were loaded on a 12% non-denaturing PAGE. The complex formation was quantified using the Instant Imager and the average percent shift (%) obtained from two independent experiments is indicated below each gel. The complexes are indicated on the right. RNA cleavage was assayed using EL18-15 (E) or EL9-11 (F). The RNA was incubated with different ratios of 5′-end labeled target (TL) and two different Rnt1p concentrations (20 and 80 nM) for 20 minutes. The cleavage products were analyzed by 20% denaturing PAGE and the bands were quantified using Instant Imager. Average percent (%) cleavage of three independent experiments is indicated below each gel. The RNA marker is displayed on the left. The positions of the substrate (S) and product (P) are showed on the right.</p>
<span>doi:10.1371/journal.pone.0000472.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The ability of both guides to direct Rnt1p cleavage was tested by incubating them with 5′-end labeled target RNA (TL) and recombinant Rnt1p (<a href="#pone-0000472-g002">Figures 2E and F</a>). As shown, Rnt1p cleaved the substrate 14 nt from the guide tetraloop releasing a 33 nt product. However, no cleavage in the substrate was detected at the second predicted cleavage site 16 nt from the guide tetraloop (54 nt product). The inability of Rnt1p to cleave the second predicted cleavage site is not surprising since Rnt1p is known to bind to its substrate asymmetrically and is only tolerant of structural variations in one side of the tetraloop <a href="#pone.0000472-Lamontagne2">[12]</a>, <a href="#pone.0000472-Ghazal2">[16]</a>, <a href="#pone.0000472-Lamontagne4">[17]</a>. It is interesting to note that the uncleavable RNA guide (EL9-11) was more efficient in directing cleavage than the long cleavable RNA guide since uncleavable guide may be recycled to induce more than one round of target cleavage. We conclude that Rnt1p may use an RNA guide to cleave an independent RNA target.</p>


<h4>The guide RNA supports standard cleavage kinetics</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The biological significance of the guide/Rnt1p complex and its potential as an effective tool for gene silencing depends on the cleavage efficiency of this complex. To evaluate the efficiency of the guide driven cleavage, we compared it to that generated using standard Rnt1p substrates<a href="#pone.0000472-Lamontagne3">[14]</a>. Binding and cleavage parameters were monitored using the EL9-11 guide, a shorter version that pairs with the target using only one 3′-end extension (EL3′-11), or a long 3′-Branch based substrate allowing classical Rnt1p cleavage <em>in cis</em> (<a href="#pone-0000472-g003">Figure 3A</a>). The RNA/protein complexes were resolved using a standard gel mobility shift assay, and complex formation was quantified and plotted as a factor of protein concentration (<a href="#pone-0000472-g003">Figure 3B</a>). The classical substrate (3′-Branch) and the guide RNA with two extensions (EL9-11) bound to Rnt1p with a similar apparent dissociation constant (K'<sub>d</sub>) of about 0.80 µM, while the guide RNA with only one single target complementary extension bound less efficiently with a K'<sub>d</sub> of 1.9 µM (<a href="#pone-0000472-t001">Table 1</a>). This data suggest that decreasing the single-stranded RNA extension length decreases the affinity to Rnt1p perhaps by inhibiting interactions fostered by deleted sequences.</p>
<div class="figure" id="pone-0000472-g003"><div class="img"><a name="pone-0000472-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison between inter- and intra-molecular RNA cleavage by different RNase IIIs.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Illustration of the different substrates used in C and D. 3′-Branch indicates a substrate allowing intramolecular cleavage by Rnt1p. EL3′-11 and EL9-11 indicate respectively a guide RNA with a single or two target complementary extensions. The target is indicated by TL. The arrowheads indicate the position of the observed cleavage by Rnt1p. (B) Quantitative analysis of RNA binding to Rnt1p. Increasing concentrations of Rnt1p (0.25 to 6 µM) were incubated with 3 fmol of 3′-Branch (▪), EL9-11:TL (σ) and EL3′-11:TL (τ) and the binding percentage (%) was plotted against the protein concentration. The curve fits were obtained using the Graph Pad Prism 4.0 program. Each data point is an average of four experiments. The target RNA in the <em>trans</em> reactions and the <em>cis</em> RNA were 5′-end labeled and incubated with members of the RNase III family. Rnt1p, bacterial RNase III (RIII), Pac1 and human Dicer were incubated in RNA excess under a 10 mM (C) or 150 mM (D) KCl. The position of the RNA ladder is shown on the left. (E) Sketch of a 36 nt fragment containing sequences complementary to EL3′-11 inserted into a U2 3′-end flanking region to replace a canonical Rnt1p substrate. The position of the oligonucleotide used for primer extension is indicated. (F) Mapping the cleavage of the U2 3′-end region with RNase IIIs. Yeast total RNA (20µg) from YHM111-U2L2 was incubated with EL3′-11 and RNase IIIs in 10 and 150 mM KCl. A primer complementary to the 3′-flanking sequence of U2 snRNA was extended in all cleavage reactions. The reference DNA sequence is shown on the left. The arrowhead indicates a specific cleavage product. The asterisk indicates a secondary structure at the mature U2 3′-end.</p>
<span>doi:10.1371/journal.pone.0000472.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The cleavage efficiencies of the different substrates were tested by incubating each of them with Rnt1p and Mg<sup>2+</sup>. The target RNA was labeled at the 5′-end to track the product generation under low (<a href="#pone-0000472-g003">Figure 3C</a>) and physiological salt concentrations (<a href="#pone-0000472-g003">Figure 3D</a>). Rnt1p cleaved all three substrates at the predicted site 14 nts from the tetraloop with efficiency close to that of previously tested natural cleavage sites (regardless of the salt conditions)<a href="#pone.0000472-Lamontagne1">[3]</a>, <a href="#pone.0000472-Lamontagne3">[14]</a>, <a href="#pone.0000472-Lamontagne4">[17]</a>. The guide's ability to direct Rnt1p cleavage was dependent on the presence of the conserved NGNN tetraloop (data not shown). The cleavage kinetics induced by the EL3′-11 were compared to that generated <em>in cis</em> within the 3′-Branch RNA (<a href="#pone-0000472-t002">Table 2</a>). Surprisingly, the guide-based cleavage exhibited about a 3-times higher K'<sub>M</sub> and a faster k<sub>cat</sub> than <em>cis</em> cleavage, while its specificity constant (k<sub>cat</sub>/K'<sub>M</sub>) was slightly reduced. We conclude that guide-mediated and classical Rnt1p substrates observe similar kinetic parameters.</p>
<div class="figure" id="pone-0000472-t002"><div class="img"><a name="pone-0000472-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Kinetic parameters of Rnt1p cleavage of inter- and intra-molecular substrates</span></strong></p><span>doi:10.1371/journal.pone.0000472.t002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Guide-induced RNA cleavage may be a unique feature of Rnt1p or it may be shared by other members of the RNase III family. We examined the guide's capacity to induce target cleavage by RNase IIIs from bacteria (RNase III), fission yeast (Pac 1) and human (Dicer) (<a href="#pone-0000472-g003">Figures 3C and D</a>). As expected, all enzymes cleaved their natural substrates suggesting that all enzymes are active (data not shown). Interestingly, the different enzymes were capable of cleaving the 3′-Branch at both salt concentrations (<a href="#pone-0000472-g003">Figure 3C and D</a>, lanes 2–5) albeit with different efficiencies. This suggests that most RNase IIIs can tolerate a three-way junction and may cleave substrates with less than 16 consecutive base pairs. However, only Rnt1p was able to cleave the 3′-Branch substrate at a fixed distance from the NGNN tetraloop (<a href="#pone-0000472-g003">Figures 3C and D</a>, lane 2 and data not shown). The guide RNA that forms 19 bp with the target sequence (EL9-11) induced cleavage by Rnt1p, bacterial RNase III, and Pac1 in both low and high salt concentrations (<a href="#pone-0000472-g003">Figures 3C and D</a>, lanes 7–10). RNase III and Pac1 cleavages were not nucleotide or loop specific and the cleavage sites were different in different salt conditions (<a href="#pone.0000472.s001">Figure S1</a>). Surprisingly, at low salt concentration all enzymes cleaved the target RNA in the presence of EL3′-11 that forms only 11 base pairs with the target (<a href="#pone-0000472-g003">Figure 3C</a>, lanes 12–15). At high salt concentration, very little EL3′-11 dependent cleavage was detected except when Rnt1p was present (<a href="#pone-0000472-g003">Figure 3D</a>, lanes 12–15). These data indicate that with the exception of Rnt1p, most RNase IIIs require a duplex longer than one turn of a helix to support intermolecular RNA cleavage.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To evaluate the potential of guide RNA <em>in vivo</em>, we tested its capacity to identify a given sequence in a natural mixture of yeast total RNA. The Rnt1p cleavage signal at the 3′-end of pre-U2 snRNA was replaced by a sequence complementary to the EL3′-11 guide and the new U2/target construct was expressed <em>in vivo</em> (<a href="#pone-0000472-g003">Figure 3E</a>). In this way the processing of the U2 3′-end that is normally carried out by Rnt1p <em>in cis</em><a href="#pone.0000472-AbouElela1">[9]</a> can only take place if the guide induces Rnt1p cleavage <em>in trans</em>. Total RNA was extracted from yeast expressing the U2/target transcript and incubated <em>in vitro</em> with the guide EL3′-11 and Rnt1p, bacterial RNase III, Pac1, or Dicer. The cleavage site in each case was visualized either by northern blot (data not shown) or by primer extension (<a href="#pone-0000472-g003">Figure 3F</a>). The U2/target RNA was cleaved by Rnt1p producing a single nucleotide cleavage 14 nucleotides from the guide tetraloop as predicted. The failure of Pac1, bacterial RNase III, and Dicer to cleave the U2/target could be explained by problems in target accessibility or competition with other imperfect duplexes in the yeast transcriptome. We conclude that guide RNAs may specifically select Rnt1p targets in a complex mix of natural RNA.</p>


<h4>The guide RNA restores cleavage of mutated Rnt1p substrate <em>in vivo</em></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Guide RNA's potential to regulate gene expression was evaluated by examining their capacity to restore cleavage to mutated Rnt1p cleavage site in both cell extract and in total RNA (<a href="#pone-0000472-g004">Figure 4</a>). The guide EL9-11 with 2 single stranded extensions did not cleave the U2/target in cell extract (<a href="#pone-0000472-g004">Figure 4B</a>, lane 3) and only induced very weak cleavage in total RNA (<a href="#pone-0000472-g004">Figure 4B</a>, lane 7). In contrast, EL3′-11 with 1 single-stranded extension induced strong cleavage in total RNA extracted from yeast (lane 8) but not in cell extract (lane 4). We reasoned that the weak activity in cell extracts was due to the guide instability. Indeed, labeled EL3′-11 is readily degraded in cell extract (data not shown). To enhance the guide RNA stability in cell extract and later <em>in vivo</em>, we generated EL3′-11 RNA with an inverted deoxythymidine (dT) at the 3′-end. The addition of an inverted deoxythymidine reduces 3′ to 5′ exonuclease attack<a href="#pone.0000472-Takei1">[21]</a>. The modified guide (EL3′-11dT) directed efficient cleavage in cell extract (lane 5) and was stable in cell extract for up to 2 hours while unprotected EL3′-11 degraded after 10 minutes (data not shown). Moreover, Northern blot analysis indicated that as little as 1 nmol of EL3′-11dT could induce the cleavage of up to 50% of the U2/target in 20 minutes (<a href="#pone-0000472-g004">Figure 4C</a>).</p>
<div class="figure" id="pone-0000472-g004"><div class="img"><a name="pone-0000472-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Guide RNA restored cleavage to a mutated Rnt1p cleavage site <em>in vivo.</em></span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Secondary structure of RNA guides complementary to a mutated Rnt1p cleavage site at the 3′-end of U2 snRNA (L2). The position of the oligonucleotide used for primer extension is indicated below as well as putative poly(A) signals (+96, +117, and +306). (B) RNA guides were incubated in yeast extract or with yeast total RNA and recombinant Rnt1p for 20 min. The cleavage site was mapped using primer complementary to the 3′-flanking sequence of U2 snRNA. The reference DNA sequence produced using the same primer is shown on the left. The product corresponding to the cleaved RNA is indicated. Bacterial tRNA was used as negative control for the primer extension. (C) Yeast strain YHM111-U2L2 was electroporated with EL3′-11dT and the RNA extracted after 10 minutes of incubation. The RNA bands were analyzed by northern blot using a probe complementary to mature U2 snRNA sequence. A probe directed against RPR1 was used as loading control. The arrowhead indicates the position of the cleavage product. (D) Cleavage site mapping of yeast YHM111-U2L2 electroporated with EL3′-11dT. Total RNA was extracted between 10 minutes and 2 hours post-electroporation and annealed to the primer used in B. The reference DNA sequence is shown on the left. The product corresponding to the cleaved RNA is indicated. (E) Analysis of U2 snRNA 3′-end formation. RNA samples described in D were hybridized to an RNA probe (DraI-SmaI fragment) complementary to the 3′- flanking sequences of U2 snRNA, and digested with RNase T1. The mature U2 3′-end and the ends of the extended forms are indicated on the right. The Rnt1p-directed cleavage product is indicated by an arrowhead. The position of the different 3′-ends detected is indicated using wild-type U2 sequence as reference. A probe against actin was used as internal control for loading and quantification.</p>
<span>doi:10.1371/journal.pone.0000472.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In order to assay the guide activity <em>in vivo</em>, we had to establish a method for RNA transfection. To do this, we adapted an electroporation based transformation strategy that is normally used for DNA transformation<a href="#pone.0000472-Becker1">[22]</a>. Different concentrations of EL3′-11dT were transfected into yeast cells and total RNA was extracted after different incubation times. The guide-dependent cleavage product was monitored by primer extension complementary to the sequence downstream of the predicted cleavage site. As shown in <a href="#pone-0000472-g004">figure 4d, a</a> single 5′-end corresponding to predicted cleavage product was detected in RNA extracted from cells transfected with 2 nmol EL3′-11dT after 10 minutes (lane 5) but not in the control cell transfected with water (lane 9). The cleavage product increased after 20 minutes of incubation (lane 6) but disappeared after one hour (lane 7) as expected since Rnt1p cleavage products are highly unstable<a href="#pone.0000472-AbouElela1">[9]</a>.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Rapid RNA processing and difficulties detecting unprocessed RNA precursors suggest that RNA processing takes place co-transcriptionally. However, the importance of co-transcriptional RNA cleavage to RNA maturation remains unclear. We have taken advantage of the newly developed guide technology to assess whether the co-transcriptional Rnt1p cleavage of the pre-U2 3′-end is required for U2 maturation. We monitored the generation of mature U2 snRNA from a transcript that depends on the guide EL3′-11dT for cleavage (<a href="#pone-0000472-g004">Figure 4E</a>). As expected, mainly mature U2 was detected in wild type cells (lane 2). In contrast, cells expressing U2/target accumulated unprocessed U2 and no mature 3′-end was detected (lane 3). Electroporation of different concentrations of guide induced cleavage in the target sequence decreasing the amount of the U2/target precursors (top panel). A product cleaved at the primary cleavage site of Rnt1p<a href="#pone.0000472-AbouElela1">[9]</a> was detected in a guide-dependent manner (e.g. lanes 4–8). However, only a small amount of mature 3′-end was observed after the guide electroporation even after 2 hours (lanes 8–10). Since the reduction in RNA precursors does not lead to corresponding accumulation of mature U2, we conclude that most of the guide-dependent cleavage events do not generate stable RNA. This data clearly demonstrate that the processing efficiency of non-coding RNA depends on the nature and timing of the endonucleolytic cleavage initiating the maturation process.</p>


<h4>Guide-specific cleavage of natural RNAs</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To evaluate the guide cleavage strategy as a tool for gene silencing, we designed and tested a series of guide RNAs targeting the branch site of U2 snRNA<a href="#pone.0000472-Ares1">[23]</a> (<a href="#pone-0000472-g005">Figures 5A and B</a>). Guides with different stem lengths and loop structures were used to demonstrate cleavage specificity (EU2dT, EU2+2bp and EU2+4bp). As expected, a guide RNA with a 5 bp stem and 11 nt complementary to the targeted U2 branch site (EU2dT) induced a substantial cleavage when incubated with total RNA and recombinant Rnt1p (<a href="#pone-0000472-g005">Figure 5C</a>, lane 2). Increasing EU2dT stem length by two base pairs (EU2+2 bp) shifted cleavage by 2 nt (lane 3). Insertion of an additional 2 bp (EU2+4 bp) shifted cleavage further by two nucleotides, while strongly reducing cleavage efficiency (lane 4). The guide EU2+4 bp is long enough to be directly cleaved by Rnt1p <em>in cis</em> independent of the target, which explains the reduced efficiency of target cleavage. These results indicate that RNA guides may act as helical scales marking the distance to the cleavage site. As expected, guide RNA with a mutation in the conserved second base of the Rnt1p tetraloop (EU2dT/ACUC) blocked cleavage (lane 5). Mutations altering the guide homing sequence complementary to the target site (EU2dT/2M and EU2dT/4M) also blocked cleavage (lanes 6 and 7) confirming the cleavage specificity.</p>
<div class="figure" id="pone-0000472-g005"><div class="img"><a name="pone-0000472-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Guide RNA directs sequence specific cleavage in a natural RNA sequence.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Secondary structure of the U2 snRNA branch site region (nucleotides 1 to 86). The gray box, the arrowhead and the brackets represent respectively the targeted region by Rnt1p, the anticipated cleavage site by Rnt1p and the region used for <em>in vitro</em> cleavage assays (U2-Br-35). (B) Sketches representing the secondary structure of guides recognized by Rnt1p and complementarity to the U2 branch site. Sequences in bold represent the nucleotides complementary to the U2 target. The gray boxes indicate mutations relative to the control (EU2dT). (C) <em>In vitro</em> cleavage of 5′-end labeled U2-Br-35 with Rnt1p and the different RNA guides. The cleavage reactions were performed in RNA excess with a guide/target ratio of 1:1. The positions of the cleavage products are indicated on the right and the RNA marker is displayed on the left. (D) Total yeast RNA and recombinant Rnt1p or yeast cell extract prepared from strain YHM111-U2L2 were used to analyze Rnt1p-directed cleavage using EU2dT. Primer complementary to the 3′-flanking sequence of the U2 snRNA branch site was extended on the extracted RNA to map the cleavage site. The reference DNA sequence produced using the same primer is shown on the left. The product corresponding to the cleaved RNA and the U2 5′-end are indicated on the right. Bacterial tRNA was used as negative control for the primer extension. (E) Cleavage comparison between Rnt1p and RNase H in total RNA or cell extract prepared from yeast YHM111-U2L2. The cleavage specificity was determined by primer extension. The reference DNA sequence produced using the same primer is shown on the left. The RNA guide and the DNA oligo used with RNase H targeted the same nucleotides. The positions of Rnt1p and RNase H cleavage products and the U2 5′-end are indicated on the right.</p>
<span>doi:10.1371/journal.pone.0000472.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>EU2dT was chosen for subsequent assays in cell extract and <em>in vivo</em> based on its performance in the <em>in vitro</em> cleavage assay. Incubation of EU2dT with total RNA and recombinant Rnt1p or cell extract introduces a single cleavage site at the predicted distance from the loop within the U2 branch site (<a href="#pone-0000472-g005">Figure 5D</a>) indicating that this guide is both specific and stable even in the presence of total yeast RNA, proteins and ribonucleases. To accurately evaluate the value and efficiency of the guide RNA-mediated cleavage, we wanted to compare it to another enzymatic activity that targets sequence specific endonucleolytic cleavage <em>in trans</em>. RNase H DNA-mediated cleavage of RNA has been exploited as a tool for gene silencing<a href="#pone.0000472-Vickers1">[24]</a> and is routinely used to cleave RNA <em>in vitro</em><a href="#pone.0000472-Wintersberger1">[25]</a>. Therefore, we chose RNase H as a benchmark for evaluating the utility of Rnt1p. We directly compared the performance of Rnt1p/EU2dT to an RNase H/DNA oligonucleotide (EU2-DNA) targeting the same sequence in total RNA and cell extract (<a href="#pone-0000472-g005">Figure 5E</a>). Both oligonucleotides induced cleavage of the targeted U2 snRNA when present in total RNA (lanes 5 and 7). However, unlike with Rnt1p, the scissile bond of RNase H cleavage was difficult to predict. In cell extract, EU2dT induced the cleavage of a single phosphodiester bond (lane 10), while the EU2-DNA induced a heterogeneous cleavage pattern (lane 11). The comparatively higher precision of Rnt1p was confirmed by other experiments using guides and DNA oligonucleotides targeting a variety of RNA transcripts (data not shown). At high salt concentrations the Rnt1p guide may support structure sensitive cleavage like RNase H where only single-stranded RNA is cleaved, and thus it may be used as a probe for RNA structure (data not shown). However, unlike RNase H, Rnt1p guide may also act as an RNA restriction enzyme allowing structure independent cleavage at low salt concentration (data not shown) increasing its utility as gene silencer. We conclude that Rnt1p provides an effective alternative to RNase H as a probe for RNA structure and as an RNA restriction enzyme.</p>


<h4>Inhibition of gene expression using guide RNAs</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To examine the potential of Rnt1p guides as regulators of gene expression <em>in vivo</em>, we electroporated EU2dT guide into yeast cells and monitored both the transfection efficiency and the degradation of U2 snRNA. After electroporation, equal cellular distribution of 5′-fluorescein labeled EU2dT (EU2dT-Fl) was observed in 53±8% of the cells (data not shown). Therefore, the maximum expected inhibition level of the targeted RNA by any electroporated guide is about 50%. Induction of U2 snRNA cleavage by EU2dT <em>in vivo</em> was monitored by primer extension. As shown in <a href="#pone-0000472-g006">figure 6a</a>, electroporation of EU2dT generated cleavage product after a 10 minute incubation. Increasing the incubation period resulted in the degradation of the cleavage product (lanes 3 to 7). Consistent with the estimated half-life of the guide RNA, the amount of intact U2 in treated cells was restored to pre-treatment levels after 2 hours of incubation. Mutations altering the Rnt1p binding site (EU2dT/ACUC) or the sequence complementary to U2 (EU2dT/2M and EU2dT/4M) blocked cleavage (lanes 10–15) confirming the reaction specificity. Northern blot quantification demonstrated that more than 40% of U2 RNA in electroporated cells was degraded after 10 minutes (<a href="#pone-0000472-g006">Figure 6B</a>). Increasing the incubation time beyond 30 minutes gradually restored U2 expression presumably due to the degradation of the guide. Moreover, transfection of mutated guide RNA left the level of U2 snRNA unchanged (data not shown). The capacity to degrade nearly half of the highly expressed U2 snRNA (200–500 molecules per haploid cell<a href="#pone.0000472-Riedel1">[26]</a>) in 10 minutes is a clear indication of the efficiency of the guide as a tool for gene regulation. We conclude that Rnt1p guides are efficient tools for nuclear site-directed RNA degradation <em>in vivo</em>.</p>
<div class="figure" id="pone-0000472-g006"><div class="img"><a name="pone-0000472-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000472" data-uri="info:doi/10.1371/journal.pone.0000472.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000472.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000472.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000472.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000472.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Small RNA guides induce Rnt1p cleavage <em>in vivo.</em></span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Mapping the RNA-directed cleavage of U2 from living yeast cells. Yeast cells were electroporated with 2 nmols of EU2dT or mutant RNA guides and total RNA was extracted between 10 minutes and 2 hours after electroporation. The primer PE-U2-Br was annealed with the RNA samples and extended on U2 snRNA. The corresponding DNA sequence is shown on the left. (B) Relative levels of cleaved U2 snRNA after electroporation were established using RNA extracted in A and loaded on 1.2% agarose-formaldehyde gel, transferred to a nylon membrane and hybridized with probes specific for nucleotides 7 to 29 into mature U2 or against the 25S rRNA to serve as internal control and visualized by Phosphor Imager. The intensity of each band was quantified using Image Quant 5.0. The levels of U2 snRNA were normalized against the 25S rRNA and the electroporation efficiency (53±8%), and were plotted as a function of time after electroporation with EU2dT.</p>
<span>doi:10.1371/journal.pone.0000472.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>RNAi and RNA turnover are often treated as two distinct mechanisms of gene regulation despite the fact that RNAi is in essence a process of targeted RNA degradation. The sequence specificity of RNAi-based gene silencing and its induction by small RNA <em>in trans</em> makes it appear quite different from the largely non-specific ribonuclease-based RNA degradation. In this study, we have shown that a classical endoribonuclease may use small RNA guides for sequence specific cleavage in analogy to the RNAi degradation mechanism. Rnt1p, a member of the RNase III family, was shown to form an RNP complex with a specific RNA hairpin and uses it to guide the cleavage of targeted RNA <em>in trans</em>. This feature allowed us to transform Rnt1p from a structure-based enzyme into a sequence-specific RNP, supporting evolutionary models explaining the origin of RNA guided protein complexes. The capacity of short and largely single-stranded RNA to guide RNase III cleavage demonstrates the flexibility and tremendous potential of these enzymes as regulators of gene expression.</p>

<h4>Evolution of an RNA-protein complex</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The debate about the evolutionary origin of RNA-protein complexes started soon after the discovery of the first catalytic RNA<a href="#pone.0000472-Davies1">[27]</a>. The protein dominance of modern enzymatic activity led to the belief that RNAs are ancient relics of catalysis<a href="#pone.0000472-Cech1">[28]</a>. It was proposed that ancient RNA enzymes were taken over by gradually evolving proteins creating in the process several intermediates incorporating both RNA and proteins moieties<a href="#pone.0000472-Jeffares1">[29]</a>, <a href="#pone.0000472-Poole1">[30]</a>. In this “RNA first” model of RNP, an RNA molecule with an established function would recruit a protein to enhance activity. However, the recent explosion in the discovery of small RNAs that guide protein functions ranging from rRNA modification to translation repression<a href="#pone.0000472-Mattick1">[31]</a> started to paint a different story. It is now argued that many guide RNAs including C/D and H/ACA box snoRNAs evolved from pre-existing RNA that acquired affinity for ancient proteins and used it to target their function<a href="#pone.0000472-Lafontaine1">[32]</a>. The “protein first” model of RNP argues that proteins with established functions scavenged non-functional or duplicated RNA transcripts for better or modified substrate specificity. The protein first model could be easily extended to components of the RNAi machinery<a href="#pone.0000472-Tijsterman1">[1]</a> that includes classical ribonucleases like RNase III<a href="#pone.0000472-Lamontagne1">[3]</a> and small RNAs. Many small RNAs discarded from introns could guide sequence specific cleavage <em>in trans</em><a href="#pone.0000472-Lin1">[33]</a>. The ability of small RNA to guide cleavage by Rnt1p, the yeast orthologue of RNase III (<a href="#pone-0000472-g002">Figure 2</a>), directly supports the basic notion of the protein first model of RNP. Indeed, the work presented in this study shows that modern catalytically independent proteins like Rnt1p could easily be adapted for an RNP-like function. The kinetics of Rnt1p-guided cleavages indicated that the enzyme recycles to cleave several guide/target complexes (<a href="#pone-0000472-t002">Table 2</a>). This suggests that the artificial Rnt1p/guide complex is not as stable as known natural RNP complexes like snoRNPs for example. Therefore, if natural Rnt1p/guide complexes exist in yeast, they are probably stabilized by other protein chaperones or through RNA features that stabilizes the RNP complex<a href="#pone.0000472-Lafontaine1">[32]</a>. However, it is not clear how established proteins would acquire the affinity for these novel RNAs. We propose that the maturation product that is normally wasted (e.g. hairpins generated as processing by-products) may be recycled into protein binding sites leading to the evolution of stable RNP complexes. Indeed, Rnt1p natural cleavage products possess the features necessary to function as guide RNAs. They contain intact Rnt1p binding sites and single stranded extensions that may function as homing devices<a href="#pone.0000472-AbouElela1">[9]</a>, <a href="#pone.0000472-Lamontagne3">[14]</a>, <a href="#pone.0000472-Lebars1">[15]</a>. To function, Rnt1p cleavage products only need protection from exonucleases and a target sequence to cleave. Indeed, vertebrate pri-miRNAs, which are essentially stem-loop structures similar to Rnt1p substrates, are processed by paralogues of RNase III, to mature into effective guides for RNA degradation and translation repression<a href="#pone.0000472-Murchison1">[34]</a>.</p>


<h4>Yeast transfection: New applications for an old model</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Yeast is the most studied eukaryotic model mainly because of the powerful genetic and molecular biology tools available for both gene and genome analysis<a href="#pone.0000472-Bartlett1">[35]</a>, <a href="#pone.0000472-Madeo1">[36]</a>. Here we present a method by which small RNA molecules could be transiently introduced into yeast and the effect on RNA could be monitored independent of any effects or limitations that often come with transformation based methods. In some cases, it is very difficult to express small RNA (100 nt or less) in yeast as RNA expression from a Pol II promoter often leads to transcript polyadenylation, transport to the cytoplasm and rapid degradation. Pol III based strategies are more successful but RNA with single-stranded ends are also rapidly degraded and require the addition of a special structure that may alter the anticipated RNA activity. Direct RNA electroporation circumvents most of these problems and may be used to identify new chemistry for oligonucleotide-based gene silencing or to study the kinetics of RNA degradation <em>in vivo</em>. Chemically modified RNA like the 2′-O-methylribonucleotide form that is very popular for gene silencing in mammalian cells may now be successfully introduced using electroporation and could be tested for the first time in yeast.</p>


<h4>RNase IIIs tools for gene silencing</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Currently, RNAi is the most successful method for design-based gene silencing. In vertebrates and many eukaryotes the introduction of short RNA duplex with sequence specific to any target genes of choice has various success rates. However, the main problem with this approach is target specificity and secondary effects triggered by the introduction of dsRNA or by the induction of the RNAi mechanism. In contrast, targeting RNA for cleavage using guide RNA uses a largely single-stranded RNA and introduces a single cleavage site that leads to RNA degradation using the normal degradation machinery (e.g. exosome). Thus, this method should reduce secondary effects associated with RNAi. In yeast, this method is restricted to nuclear RNA since Rnt1p is localized in the nucleus<a href="#pone.0000472-Catala1">[37]</a>. The nuclear specificity of this approach distinguishes it from other available approaches that appear to be active in more than one cellular compartment. Similar nuclear degradation strategies in mammalian cells may also be envisioned. It is established that Drosha, the mammalian paralogue of Rnt1p, cleaves a stem-loop structure analogous to that of Rnt1p. Thus, it is possible to imagine a similar strategy using the Drosha recognition signal to direct cleavage in independent RNA species.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains and plasmids</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Yeast was grown and manipulated according to standard procedures<a href="#pone.0000472-Guthrie1">[38]</a>, <a href="#pone.0000472-Rose1">[39]</a>. All the experiments were performed using the yeast strain YHM111 (<em>MATa, trp1, ura3-52, ade2-101, his3, lys2, snr20</em>::LYS2)<a href="#pone.0000472-Madhani1">[40]</a>. The plasmids pRS314/U2 and pRS314/U2δStem were generated by subcloning <em>Pvu</em> II fragments from pRS315/U2 and pRS315/U2δStem<a href="#pone.0000472-AbouElela1">[9]</a> into pRS314. The U2/target RNA was expressed from pRS314/U2δStem/L2, which was generated by inserting a synthetic 36 bp dsDNA oligonucleotide (5′-CTAGAAGCTAATGTTTTGGCCTCTTCAAGATTATGG​-3′)into the <em>Nhe</em>I site of pRS314/U2δStem. The strain YHM111 was transformed with pRS314/U2 or pRS314/U2δStem/L2 to yield strains YHM111-U2 and YHM111-U2L2.</p>


<h4>Enzymatic assays</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Recombinant Rnt1p, Pac1, and <em>E.coli</em> RNase III were produced in bacteria and purified as described before<a href="#pone.0000472-Lamontagne5">[41]</a>. Recombinant human Dicer was purchase from Stratagene (La Jolla, CA). The RNA transcripts used for the cleavage and binding assays were generated by T7 RNA polymerase using oligonucleotides as templates. Some transcripts including the EL3′11dT were chemically synthesized and purchased from Integrated DNA Technologies (Coralville, IA). The RNA transcripts derived from T7 RNA polymerase were dephosphorylated using antarctic phosphatase (New England Biolabs, Ipswich, MA) and 5′-end labeled using [γ-<sup>32</sup>P] ATP as described<a href="#pone.0000472-Lamontagne4">[17]</a>. <em>In vitro</em> cleavages were performed by incubating the different RNA substrates with 20–80 nM of enzyme for 20 minutes at 30°C (or 37°C for Dicer). The reactions were carried out in 20 µl of cleavage buffer<a href="#pone.0000472-Lamontagne5">[41]</a> for Rnt1p, Dicer and <em>E. coli</em> RNase III. For Pac1, the substrates were incubated in the presence of 80 nM of enzyme for 20 minutes at 30°C in 20 µl reaction buffer<a href="#pone.0000472-Lamontagne3">[14]</a>. Cleavage comparison between Rnt1p and bacterial RNase H (USB, OH) in total RNA were performed using 80 nM of enzymes. All experiments were repeated three times and the average calculated. All kinetic calculations were performed using the Graph Pad Prism 4.0 program (GraphPad Software, CA).</p>


<h4>RNA Gel mobility Shift Assay</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>RNA binding experiments were performed using guide concentrations that ranged between 0.08 and 0.64 µM and 0.32 µM of unlabelled target RNA spiked with a trace of labeled target in 20 µl reaction buffer (30 mM Tris pH 7.5, 150 mM KCl, 5 mM spermidine, 10 mM MgCl<sub>2</sub>, 0.1 mM dithiothreitol (DTT), and 0.1 mM EDTA pH 7.5) for 5 minutes at 30°C. After incubation, 20% glycerol was added and 4 µl of each reaction was fractionated on 12% non-denaturing polyacrylamide gels. Both bound and unbound RNA fractions were quantified using Instant Imager (Packard, Meriden, CT). Each experiment was repeated twice.</p>


<h4>Protein Gel mobility Shift Assay</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Protein binding experiments were conducted essentially as described before<a href="#pone.0000472-Lamontagne2">[12]</a> with 3 fmol of 5′-end labeled RNA. For the <em>trans</em> reaction, 3 fmol of 5′-end labeled RNA guide was incubated with 2 pmol of cold RNA target prior to the incubation with Rnt1p. Experiments were repeated three times.</p>


<h4>Yeast extracts preparation</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>YHM111-U2 and YHM111-U2L2 strains were grown in Yeast Complete media without tryptophan (YC-trp). The growing cells were collected and the extracts prepared as previously described<a href="#pone.0000472-Lamontagne2">[12]</a>.</p>


<h4>Primer extension</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Primer extension was performed essentially as described before<a href="#pone.0000472-Ares2">[42]</a>. Briefly, 5 µg of total RNA was incubated with 1 ng of 5′-end radiolabeled primer. The extensions in <a href="#pone-0000472-g003">figures 3</a> and <a href="#pone-0000472-g004">4</a> were performed using U2/3′-end oligonucleotide (5′-TTACATATTGGTTGC-3′)<a href="#pone.0000472-AbouElela1">[9]</a>, while those in <a href="#pone-0000472-g005">figures 5</a> and <a href="#pone-0000472-g006">6</a> were performed using the U2-Br-PE oligonucleotide (5′-GGGTGCCAAAAAATGTG-3′).</p>


<h4>Northern Blot</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The northern blots were performed essentially as described before<a href="#pone.0000472-AbouElela1">[9]</a>. The RNA was extracted from yeast cells and 10 µg was loaded on 1.2% agarose-formaldehyde gel and transferred to a nylon membrane (Hybond-XL, Amersham). The RNA was visualized using radiolabeled oligonucleotide probes complementary to U2 snRNA (5′-GGGTGCCAAAAAATGTG-3′), RPR1 (5′-GGGCCAATGCCAAAAGCGACATTAACCCGG-3′) and rRNA 25S (5′-ATCGACTAACCCACGTCCAACTGCTGTTGACGTGG-​3′).</p>


<h4>RNase protection assay</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>A probe complementary to the 3′-end of U2L2 was derived from T7 transcription of the plasmid pRS314/U2δStem/L2 digested with <em>Dra</em>I, 97 nt bases upstream the mature U2 3′-end. The probe covers 97 bases in the mature U2 snRNA and 480 bases downstream of the mature 3′-end. A probe complementary to actin was derived from T7 transcription of the plasmid pKS/Actin digested with <em>Hind</em>III. Total yeast RNA (5 µg) was incubated at 42°C for 12 h with ~10<sup>5</sup> C.P.M. of probe in 80% formamide hybridization buffer<a href="#pone.0000472-AbouElela1">[9]</a>. The hybridization mix was digested with 100 U/ml RNase T1 for 1 hour at 30°C, and the protected fragments were separated on 8% denaturing acrylamide gel.</p>


<h4>RNA electroporation in yeast living cells</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Electrocompetent cells were prepared as described before<a href="#pone.0000472-Becker1">[22]</a> with modifications. Yeast strains YHM111-U2 and YHM111-U2L2 were grown overnight in 500 ml of YC-Trp at 30°C to an OD<sub>600</sub> of 0.8. The culture was chilled on ice for 30 min and centrifuged at 5000 rpm for 5 min at 4°C. The supernatant was discarded and the pellet was washed twice in 50 ml ice-cold sterile water. After the second centrifugation, the pellet was resuspended in 20 ml ice-cold 1M sorbitol and centrifuged at 5000 rpm for 5 min at 4°C. The final pellet was resuspended in 500 µl of 1M sorbitol and used directly for electroporation. For electroporation, 40 µl of yeast suspension per transformation was used with RNA guide (0.2 to 4 nmols). The pulse was performed at 1.5 kV, 25 µF, and 200 Ω with the Bio-Rad MicroPulser (Bio-Rad, Richmond, CA). Immediately after the pulse, 1 ml of ice -cold 1M sorbitol was added and transferred into a tube containing 4 ml of YC-trp media for incubation at 30°C.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000472"></div><a name="pone.0000472.s001" id="pone.0000472.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000472.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Mapping guide-induced RNA cleavage by RNase IIIs. The substrate 3′-Branch and the RNA target (TL) in the RNA/target complexes EL9-11:TL and EL3′-11:TL were 5′-end labeled and incubated with Rnt1p (A), bacterial RNase III (B), S. pombe Pac1 (C), and human Dicer (D) in presence of Mg2+ and the cleavage products were mapped. The black and gray arrowheads indicate cleavage sites when the reactions were performed at 10 and 150 mM monovalent salt concentration respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000472.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.23 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Christine Guthrie for yeast strain YHM111 and Mathieu Lavoie for the preparation of electrocompetent cells and performing the gel shift assays. We are grateful to Mathieu Lavoie, Julian Venables, Maurille Fournier, François Dragon, Lucien Jr Bergeron, Jean-Pierre Perreault, Martin Simard, and Liam Good for critical reading of this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SA. Performed the experiments: BL. Wrote the paper: SA BL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000472-Tijsterman1" id="pone.0000472-Tijsterman1"></a>Tijsterman M, Plasterk RH (2004) Dicers at RISC; the mechanism of RNAi. Cell  117: 1–3.  <ul class="find" data-citedArticleID="954062" data-doi="10.1016/s0092-8674(04)00293-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00293-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dicers+at+RISC%3B+the+mechanism+of+RNAi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dicers+at+RISC%3B+the+mechanism+of+RNAi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000472-Xia1" id="pone.0000472-Xia1"></a>Xia H, Mao Q, Paulson HL, Davidson BL (2002) siRNA-mediated gene silencing in vitro and in vivo. Nat Biotechnol  20: 1006–1010.  <ul class="find" data-citedArticleID="954070" data-doi="10.1038/nbt739"><li><a href="http://dx.doi.org/10.1038/nbt739" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=siRNA-mediated+gene+silencing+in+vitro+and+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22siRNA-mediated+gene+silencing+in+vitro+and+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000472-Lamontagne1" id="pone.0000472-Lamontagne1"></a>Lamontagne B, Larose S, Boulanger J, Elela SA (2001) The RNase III family: a conserved structure and expanding functions in eukaryotic dsRNA metabolism. Curr Issues Mol Biol  3: 71–78.  <ul class="find" data-citedArticleID="954028"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+RNase+III+family%3A+a+conserved+structure+and+expanding+functions+in+eukaryotic+dsRNA+metabolism.&amp;auth=&amp;atitle=The+RNase+III+family%3A+a+conserved+structure+and+expanding+functions+in+eukaryotic+dsRNA+metabolism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RNase+III+family%3A+a+conserved+structure+and+expanding+functions+in+eukaryotic+dsRNA+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RNase+III+family%3A+a+conserved+structure+and+expanding+functions+in+eukaryotic+dsRNA+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000472-Wilson1" id="pone.0000472-Wilson1"></a>Wilson HR, Yu D, Peters HK 3rd, Zhou JG, Court DL (2002) The global regulator RNase III modulates translation repression by the transcription elongation factor N. Embo J  21: 4154–4161.  <ul class="find" data-citedArticleID="954066" data-doi="10.1093/emboj/cdf395"><li><a href="http://dx.doi.org/10.1093/emboj/cdf395" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+global+regulator+RNase+III+modulates+translation+repression+by+the+transcription+elongation+factor+N." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+global+regulator+RNase+III+modulates+translation+repression+by+the+transcription+elongation+factor+N.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000472-Ge1" id="pone.0000472-Ge1"></a>Ge D, Lamontagne B, Elela SA (2005) RNase III-Mediated Silencing of a Glucose-Dependent Repressor in Yeast. Curr Biol  15: 140–145.  <ul class="find" data-citedArticleID="954012" data-doi="10.1016/j.cub.2004.12.001"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.12.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNase+III-Mediated+Silencing+of+a+Glucose-Dependent+Repressor+in+Yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNase+III-Mediated+Silencing+of+a+Glucose-Dependent+Repressor+in+Yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000472-Larose1" id="pone.0000472-Larose1"></a>Larose S, Laterreur N, Ghazal G, Gagnon J, Wellinger RJ, et al.  (2007) RNase III-dependent Regulation of Yeast Telomerase. J Biol Chem  282: 4373–4381.  <ul class="find" data-citedArticleID="954038" data-doi="10.1074/jbc.m607145200"><li><a href="http://dx.doi.org/10.1074/jbc.m607145200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNase+III-dependent+Regulation+of+Yeast+Telomerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNase+III-dependent+Regulation+of+Yeast+Telomerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000472-Lee1" id="pone.0000472-Lee1"></a>Lee A, Henras AK, Chanfreau G (2005) Multiple RNA surveillance pathways limit aberrant expression of iron uptake mRNAs and prevent iron toxicity in S. cerevisiae. Mol Cell  19: 39–51.  <ul class="find" data-citedArticleID="954042" data-doi="10.1016/j.molcel.2005.05.021"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.05.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+RNA+surveillance+pathways+limit+aberrant+expression+of+iron+uptake+mRNAs+and+prevent+iron+toxicity+in+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+RNA+surveillance+pathways+limit+aberrant+expression+of+iron+uptake+mRNAs+and+prevent+iron+toxicity+in+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000472-Gan1" id="pone.0000472-Gan1"></a>Gan J, Tropea JE, Austin BP, Court DL, Waugh DS, et al.  (2006) Structural insight into the mechanism of double-stranded RNA processing by ribonuclease III. Cell  124: 355–366.  <ul class="find" data-citedArticleID="954010" data-doi="10.1016/j.cell.2005.11.034"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.11.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+insight+into+the+mechanism+of+double-stranded+RNA+processing+by+ribonuclease+III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+insight+into+the+mechanism+of+double-stranded+RNA+processing+by+ribonuclease+III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000472-AbouElela1" id="pone.0000472-AbouElela1"></a>Abou Elela S, Ares M Jr (1998) Depletion of yeast RNase III blocks correct U2 3′ end formation and results in polyadenylated but functional U2 snRNA. Embo J  17: 3738–3746.  <ul class="find" data-citedArticleID="953988" data-doi="10.1093/emboj/17.13.3738"><li><a href="http://dx.doi.org/10.1093/emboj/17.13.3738" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Depletion+of+yeast+RNase+III+blocks+correct+U2+3%26%23x2032%3B+end+formation+and+results+in+polyadenylated+but+functional+U2+snRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Depletion+of+yeast+RNase+III+blocks+correct+U2+3%26%23x2032%3B+end+formation+and+results+in+polyadenylated+but+functional+U2+snRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000472-AbouElela2" id="pone.0000472-AbouElela2"></a>Abou Elela S, Igel H, Ares M Jr (1996) RNase III cleaves eukaryotic preribosomal RNA at a U3 snoRNP-dependent site. Cell  85: 115–124.  <ul class="find" data-citedArticleID="953990" data-doi="10.1016/s0092-8674(00)81087-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81087-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNase+III+cleaves+eukaryotic+preribosomal+RNA+at+a+U3+snoRNP-dependent+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNase+III+cleaves+eukaryotic+preribosomal+RNA+at+a+U3+snoRNP-dependent+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000472-Ghazal1" id="pone.0000472-Ghazal1"></a>Ghazal G, Ge D, Gervais-Bird J, Gagnon J, Abou Elela S (2005) Genome-wide prediction and analysis of yeast RNase III-dependent snoRNA processing signals. Mol Cell Biol  25: 2981–2994.  <ul class="find" data-citedArticleID="954014" data-doi="10.1128/mcb.25.8.2981-2994.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.8.2981-2994.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+prediction+and+analysis+of+yeast+RNase+III-dependent+snoRNA+processing+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+prediction+and+analysis+of+yeast+RNase+III-dependent+snoRNA+processing+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000472-Lamontagne2" id="pone.0000472-Lamontagne2"></a>Lamontagne B, Hannoush RN, Damha MJ, Aboul Elela S (2004) Molecular requirements for duplex recognition and cleavage by eukaryotic RNase III: discovery of an RNA-dependent DNA cleavage activity by yeast Rnt1p. J Mol Biol  338: 401–418.  <ul class="find" data-citedArticleID="954030" data-doi="10.1016/j.jmb.2004.02.059"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.02.059" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+requirements+for+duplex+recognition+and+cleavage+by+eukaryotic+RNase+III%3A+discovery+of+an+RNA-dependent+DNA+cleavage+activity+by+yeast+Rnt1p." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+requirements+for+duplex+recognition+and+cleavage+by+eukaryotic+RNase+III%3A+discovery+of+an+RNA-dependent+DNA+cleavage+activity+by+yeast+Rnt1p.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000472-Chanfreau1" id="pone.0000472-Chanfreau1"></a>Chanfreau G, Buckle M, Jacquier A (2000) Recognition of a conserved class of RNA tetraloops by Saccharomyces cerevisiae RNase III. Proc Natl Acad Sci U S A  97: 3142–3147.  <ul class="find" data-citedArticleID="954004" data-doi="10.1073/pnas.97.7.3142"><li><a href="http://dx.doi.org/10.1073/pnas.97.7.3142" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+a+conserved+class+of+RNA+tetraloops+by+Saccharomyces+cerevisiae+RNase+III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+a+conserved+class+of+RNA+tetraloops+by+Saccharomyces+cerevisiae+RNase+III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000472-Lamontagne3" id="pone.0000472-Lamontagne3"></a>Lamontagne B, Elela SA (2004) Evaluation of the RNA determinants for bacterial and yeast RNase III binding and cleavage. J Biol Chem  279: 2231–2241.  <ul class="find" data-citedArticleID="954032" data-doi="10.1074/jbc.m309324200"><li><a href="http://dx.doi.org/10.1074/jbc.m309324200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+RNA+determinants+for+bacterial+and+yeast+RNase+III+binding+and+cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+RNA+determinants+for+bacterial+and+yeast+RNase+III+binding+and+cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000472-Lebars1" id="pone.0000472-Lebars1"></a>Lebars I, Lamontagne B, Yoshizawa S, Aboul-Elela S, Fourmy D (2001) Solution structure of conserved AGNN tetraloops: insights into Rnt1p RNA processing. Embo J  20: 7250–7258.  <ul class="find" data-citedArticleID="954040" data-doi="10.1093/emboj/20.24.7250"><li><a href="http://dx.doi.org/10.1093/emboj/20.24.7250" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solution+structure+of+conserved+AGNN+tetraloops%3A+insights+into+Rnt1p+RNA+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solution+structure+of+conserved+AGNN+tetraloops%3A+insights+into+Rnt1p+RNA+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000472-Ghazal2" id="pone.0000472-Ghazal2"></a>Ghazal G, Elela SA (2006) Characterization of the reactivity determinants of a novel hairpin substrate of yeast RNase III. J Mol Biol  363: 332–344.  <ul class="find" data-citedArticleID="954016" data-doi="10.1016/j.jmb.2006.08.028"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.08.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+reactivity+determinants+of+a+novel+hairpin+substrate+of+yeast+RNase+III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+reactivity+determinants+of+a+novel+hairpin+substrate+of+yeast+RNase+III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000472-Lamontagne4" id="pone.0000472-Lamontagne4"></a>Lamontagne B, Ghazal G, Lebars I, Yoshizawa S, Fourmy D, et al.  (2003) Sequence dependence of substrate recognition and cleavage by yeast RNase III. J Mol Biol  327: 985–1000.  <ul class="find" data-citedArticleID="954034" data-doi="10.1016/s0022-2836(03)00231-6"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00231-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+dependence+of+substrate+recognition+and+cleavage+by+yeast+RNase+III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+dependence+of+substrate+recognition+and+cleavage+by+yeast+RNase+III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000472-Chanfreau2" id="pone.0000472-Chanfreau2"></a>Chanfreau G, Elela SA, Ares M Jr, Guthrie C (1997) Alternative 3′-end processing of U5 snRNA by RNase III. Genes Dev  11: 2741–2751.  <ul class="find" data-citedArticleID="954006" data-doi="10.1101/gad.11.20.2741"><li><a href="http://dx.doi.org/10.1101/gad.11.20.2741" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+3%26%23x2032%3B-end+processing+of+U5+snRNA+by+RNase+III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+3%26%23x2032%3B-end+processing+of+U5+snRNA+by+RNase+III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000472-Kiss1" id="pone.0000472-Kiss1"></a>Kiss T (2002) Small nucleolar RNAs: an abundant group of noncoding RNAs with diverse cellular functions. Cell  109: 145–148.  <ul class="find" data-citedArticleID="954024" data-doi="10.1016/s0092-8674(02)00718-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00718-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small+nucleolar+RNAs%3A+an+abundant+group+of+noncoding+RNAs+with+diverse+cellular+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small+nucleolar+RNAs%3A+an+abundant+group+of+noncoding+RNAs+with+diverse+cellular+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000472-Hammond1" id="pone.0000472-Hammond1"></a>Hammond SM, Boettcher S, Caudy AA, Kobayashi R, Hannon GJ (2001) Argonaute2, a link between genetic and biochemical analyses of RNAi. Science  293: 1146–1150.  <ul class="find" data-citedArticleID="954020" data-doi="10.1126/science.1064023"><li><a href="http://dx.doi.org/10.1126/science.1064023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Argonaute2%2C+a+link+between+genetic+and+biochemical+analyses+of+RNAi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Argonaute2%2C+a+link+between+genetic+and+biochemical+analyses+of+RNAi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000472-Takei1" id="pone.0000472-Takei1"></a>Takei Y, Kadomatsu K, Itoh H, Sato W, Nakazawa K, et al.  (2002) 5′-,3′-inverted thymidine-modified antisense oligodeoxynucleotide targeting midkine. Its design and application for cancer therapy. J Biol Chem  277: 23800–23806.  <ul class="find" data-citedArticleID="954060" data-doi="10.1074/jbc.m112100200"><li><a href="http://dx.doi.org/10.1074/jbc.m112100200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=5%26%23x2032%3B-%2C3%26%23x2032%3B-inverted+thymidine-modified+antisense+oligodeoxynucleotide+targeting+midkine.+Its+design+and+application+for+cancer+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%225%26%23x2032%3B-%2C3%26%23x2032%3B-inverted+thymidine-modified+antisense+oligodeoxynucleotide+targeting+midkine.+Its+design+and+application+for+cancer+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000472-Becker1" id="pone.0000472-Becker1"></a>Becker DM, Guarente L (1991) High-efficiency transformation of yeast by electroporation. Methods Enzymol  194: 182–187.  <ul class="find" data-citedArticleID="953998" data-doi="10.1016/0076-6879(91)94015-5"><li><a href="http://dx.doi.org/10.1016/0076-6879(91)94015-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-efficiency+transformation+of+yeast+by+electroporation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-efficiency+transformation+of+yeast+by+electroporation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000472-Ares1" id="pone.0000472-Ares1"></a>Ares M Jr (1986) U2 RNA from yeast is unexpectedly large and contains homology to vertebrate U4, U5, and U6 small nuclear RNAs. Cell  47: 49–59.  <ul class="find" data-citedArticleID="953992" data-doi="10.1016/0092-8674(86)90365-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(86)90365-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=U2+RNA+from+yeast+is+unexpectedly+large+and+contains+homology+to+vertebrate+U4%2C+U5%2C+and+U6+small+nuclear+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22U2+RNA+from+yeast+is+unexpectedly+large+and+contains+homology+to+vertebrate+U4%2C+U5%2C+and+U6+small+nuclear+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000472-Vickers1" id="pone.0000472-Vickers1"></a>Vickers TA, Koo S, Bennett CF, Crooke ST, Dean NM, et al.  (2003) Efficient reduction of target RNAs by small interfering RNA and RNase H-dependent antisense agents. A comparative analysis. J Biol Chem  278: 7108–7118.  <ul class="find" data-citedArticleID="954064" data-doi="10.1074/jbc.m210326200"><li><a href="http://dx.doi.org/10.1074/jbc.m210326200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+reduction+of+target+RNAs+by+small+interfering+RNA+and+RNase+H-dependent+antisense+agents.+A+comparative+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+reduction+of+target+RNAs+by+small+interfering+RNA+and+RNase+H-dependent+antisense+agents.+A+comparative+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000472-Wintersberger1" id="pone.0000472-Wintersberger1"></a>Wintersberger U, Frank P (2001) Ribonucleases H of the budding yeast, Saccharomyces cerevisiae. Methods Enzymol  341: 414–429.  <ul class="find" data-citedArticleID="954068" data-doi="10.1016/s0076-6879(01)41167-0"><li><a href="http://dx.doi.org/10.1016/s0076-6879(01)41167-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribonucleases+H+of+the+budding+yeast%2C+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribonucleases+H+of+the+budding+yeast%2C+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000472-Riedel1" id="pone.0000472-Riedel1"></a>Riedel N, Wise JA, Swerdlow H, Mak A, Guthrie C (1986) Small nuclear RNAs from Saccharomyces cerevisiae: unexpected diversity in abundance, size, and molecular complexity. Proc Natl Acad Sci U S A  83: 8097–8101.  <ul class="find" data-citedArticleID="954056" data-doi="10.1073/pnas.83.21.8097"><li><a href="http://dx.doi.org/10.1073/pnas.83.21.8097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small+nuclear+RNAs+from+Saccharomyces+cerevisiae%3A+unexpected+diversity+in+abundance%2C+size%2C+and+molecular+complexity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small+nuclear+RNAs+from+Saccharomyces+cerevisiae%3A+unexpected+diversity+in+abundance%2C+size%2C+and+molecular+complexity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000472-Davies1" id="pone.0000472-Davies1"></a>Davies J, von Ahsen U, Wank H, Schroeder R (1992) Evolution of secondary metabolite production: potential roles for antibiotics as prebiotic effectors of catalytic RNA reactions. Ciba Found Symp  171: 24–32.  discussion 32–44. <ul class="find" data-citedArticleID="954008" data-doi="10.1002/9780470514344.ch3"><li><a href="http://dx.doi.org/10.1002/9780470514344.ch3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+secondary+metabolite+production%3A+potential+roles+for+antibiotics+as+prebiotic+effectors+of+catalytic+RNA+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+secondary+metabolite+production%3A+potential+roles+for+antibiotics+as+prebiotic+effectors+of+catalytic+RNA+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000472-Cech1" id="pone.0000472-Cech1"></a>Cech TR (1989) RNA as an enzyme. Biochem Int  18: 7–14.  <ul class="find" data-citedArticleID="954002"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RNA+as+an+enzyme.&amp;auth=&amp;atitle=RNA+as+an+enzyme." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+as+an+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+as+an+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000472-Jeffares1" id="pone.0000472-Jeffares1"></a>Jeffares DC, Poole AM, Penny D (1998) Relics from the RNA world. J Mol Evol  46: 18–36.  <ul class="find" data-citedArticleID="954022" data-doi="10.1007/pl00006280"><li><a href="http://dx.doi.org/10.1007/pl00006280" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relics+from+the+RNA+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relics+from+the+RNA+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000472-Poole1" id="pone.0000472-Poole1"></a>Poole AM, Jeffares DC, Penny D (1998) The path from the RNA world. J Mol Evol  46: 1–17.  <ul class="find" data-citedArticleID="954054" data-doi="10.1007/pl00006275"><li><a href="http://dx.doi.org/10.1007/pl00006275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+path+from+the+RNA+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+path+from+the+RNA+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000472-Mattick1" id="pone.0000472-Mattick1"></a>Mattick JS, Makunin IV (2005) Small regulatory RNAs in mammals. Hum Mol Genet  14 Spec No 1: R121–132.  <ul class="find" data-citedArticleID="954050" data-doi="10.1093/hmg/ddi101"><li><a href="http://dx.doi.org/10.1093/hmg/ddi101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small+regulatory+RNAs+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small+regulatory+RNAs+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000472-Lafontaine1" id="pone.0000472-Lafontaine1"></a>Lafontaine DL, Tollervey D (1998) Birth of the snoRNPs: the evolution of the modification-guide snoRNAs. Trends Biochem Sci  23: 383–388.  <ul class="find" data-citedArticleID="954026" data-doi="10.1016/s0968-0004(98)01260-2"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01260-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Birth+of+the+snoRNPs%3A+the+evolution+of+the+modification-guide+snoRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Birth+of+the+snoRNPs%3A+the+evolution+of+the+modification-guide+snoRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000472-Lin1" id="pone.0000472-Lin1"></a>Lin SL, Ying SY (2006) Gene silencing in vitro and in vivo using intronic microRNAs. Methods Mol Biol  342: 295–312.  <ul class="find" data-citedArticleID="954044" data-doi="10.1385/1-59745-123-1:295"><li><a href="http://dx.doi.org/10.1385/1-59745-123-1:295" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+silencing+in+vitro+and+in+vivo+using+intronic+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+silencing+in+vitro+and+in+vivo+using+intronic+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000472-Murchison1" id="pone.0000472-Murchison1"></a>Murchison EP, Hannon GJ (2004) miRNAs on the move: miRNA biogenesis and the RNAi machinery. Curr Opin Cell Biol  16: 223–229.  <ul class="find" data-citedArticleID="954052" data-doi="10.1016/j.ceb.2004.04.003"><li><a href="http://dx.doi.org/10.1016/j.ceb.2004.04.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRNAs+on+the+move%3A+miRNA+biogenesis+and+the+RNAi+machinery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRNAs+on+the+move%3A+miRNA+biogenesis+and+the+RNAi+machinery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000472-Bartlett1" id="pone.0000472-Bartlett1"></a>Bartlett R, Nurse P (1990) Yeast as a model system for understanding the control of DNA replication in Eukaryotes. Bioessays  12: 457–463.  <ul class="find" data-citedArticleID="953996" data-doi="10.1002/bies.950121002"><li><a href="http://dx.doi.org/10.1002/bies.950121002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+as+a+model+system+for+understanding+the+control+of+DNA+replication+in+Eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+as+a+model+system+for+understanding+the+control+of+DNA+replication+in+Eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000472-Madeo1" id="pone.0000472-Madeo1"></a>Madeo F, Engelhardt S, Herker E, Lehmann N, Maldener C, et al.  (2002) Apoptosis in yeast: a new model system with applications in cell biology and medicine. Curr Genet  41: 208–216.  <ul class="find" data-citedArticleID="954046" data-doi="10.1007/s00294-002-0310-2"><li><a href="http://dx.doi.org/10.1007/s00294-002-0310-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Apoptosis+in+yeast%3A+a+new+model+system+with+applications+in+cell+biology+and+medicine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Apoptosis+in+yeast%3A+a+new+model+system+with+applications+in+cell+biology+and+medicine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000472-Catala1" id="pone.0000472-Catala1"></a>Catala M, Lamontagne B, Larose S, Ghazal G, Elela SA (2004) Cell cycle-dependent nuclear localization of yeast RNase III is required for efficient cell division. Mol Biol Cell  15: 3015–3030.  <ul class="find" data-citedArticleID="954000" data-doi="10.1091/mbc.e04-03-0183"><li><a href="http://dx.doi.org/10.1091/mbc.e04-03-0183" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+cycle-dependent+nuclear+localization+of+yeast+RNase+III+is+required+for+efficient+cell+division." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+cycle-dependent+nuclear+localization+of+yeast+RNase+III+is+required+for+efficient+cell+division.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000472-Guthrie1" id="pone.0000472-Guthrie1"></a>Guthrie C, Fink GR (1991) Guide to Yeast Genetics and Molecular Biology. San Diego, CA: Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0000472-Rose1" id="pone.0000472-Rose1"></a>Rose MD, Winston F, Hieter P (1990) Methods in Yeast Genetics: A Laboratory Course Manual. New York: Cold Spring Harbor.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0000472-Madhani1" id="pone.0000472-Madhani1"></a>Madhani HD, Guthrie C (1992) A novel base-pairing interaction between U2 and U6 snRNAs suggests a mechanism for the catalytic activation of the spliceosome. Cell  71: 803–817.  <ul class="find" data-citedArticleID="954048" data-doi="10.1016/0092-8674(92)90556-r"><li><a href="http://dx.doi.org/10.1016/0092-8674(92)90556-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+base-pairing+interaction+between+U2+and+U6+snRNAs+suggests+a+mechanism+for+the+catalytic+activation+of+the+spliceosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+base-pairing+interaction+between+U2+and+U6+snRNAs+suggests+a+mechanism+for+the+catalytic+activation+of+the+spliceosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000472-Lamontagne5" id="pone.0000472-Lamontagne5"></a>Lamontagne B, Abou Elela S (2001) Purification and characterization of Saccharomyces cerevisiae Rnt1p nuclease. Methods Enzymol  342: 159–167.  <ul class="find" data-citedArticleID="954036" data-doi="10.1016/s0076-6879(01)42543-2"><li><a href="http://dx.doi.org/10.1016/s0076-6879(01)42543-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+characterization+of+Saccharomyces+cerevisiae+Rnt1p+nuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+characterization+of+Saccharomyces+cerevisiae+Rnt1p+nuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000472-Ares2" id="pone.0000472-Ares2"></a>Ares M Jr, Igel AH (1990) Lethal and temperature-sensitive mutations and their suppressors identify an essential structural element in U2 small nuclear RNA. Genes Dev  4: 2132–2145.  <ul class="find" data-citedArticleID="953994" data-doi="10.1101/gad.4.12a.2132"><li><a href="http://dx.doi.org/10.1101/gad.4.12a.2132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethal+and+temperature-sensitive+mutations+and+their+suppressors+identify+an+essential+structural+element+in+U2+small+nuclear+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethal+and+temperature-sensitive+mutations+and+their+suppressors+identify+an+essential+structural+element+in+U2+small+nuclear+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.XML" value="98843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.PDF" value="592986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g001.PNG_L" value="1128797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g001.PNG_M" value="34156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g001.PNG_S" value="3014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g001.TIF" value="1528240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g001.PNG_I" value="27166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t001.PNG_L" value="31641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t001.PNG_M" value="21215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t001.PNG_S" value="3133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t001.TIF" value="240260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t001.PNG_I" value="35465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g002.PNG_L" value="829143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g002.PNG_M" value="43024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g002.PNG_S" value="2937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g002.TIF" value="1097826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g002.PNG_I" value="22569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g003.PNG_L" value="3501975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g003.PNG_M" value="69912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g003.PNG_S" value="4261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g003.TIF" value="4314274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g003.PNG_I" value="43184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t002.PNG_L" value="40167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t002.PNG_M" value="27342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t002.PNG_S" value="3407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t002.TIF" value="270036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.t002.PNG_I" value="46590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g004.PNG_L" value="2129360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g004.PNG_M" value="62786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g004.PNG_S" value="3635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g004.TIF" value="2719858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g004.PNG_I" value="29007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g005.PNG_L" value="3122885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g005.PNG_M" value="63356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g005.PNG_S" value="3950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g005.TIF" value="3798262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g005.PNG_I" value="36442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g006.PNG_L" value="1605270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g006.PNG_M" value="50664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g006.PNG_S" value="2762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g006.TIF" value="1840216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.g006.PNG_I" value="18183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000472.s001.TIF" value="989408"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000472&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000472" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000472&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000472&volume=&issue=&title=Short RNA Guides Cleavage by Eukaryotic RNase III&author_name=Bruno%20Lamontagne%2C%20Sherif%20Abou%20Elela&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000472" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000472" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000472" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000472&amp;t=Short%20RNA%20Guides%20Cleavage%20by%20Eukaryotic%20RNase%20III" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000472&title=Short%20RNA%20Guides%20Cleavage%20by%20Eukaryotic%20RNase%20III&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000472&amp;title=Short%20RNA%20Guides%20Cleavage%20by%20Eukaryotic%20RNase%20III" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000472" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000472&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Short RNA Guides Cleavage by Eukaryotic RNase III';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000472';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Short%20RNA%20Guides%20Cleavage%20by%20Eukaryotic%20RNase%20III http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000472" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000472" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23782"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Electroporation%22" title="Search for articles in the subject area:'Electroporation'"><div class="flagText">Electroporation</div></a>
              <div data-categoryid="35955" data-articleid="23782"
                   data-categoryname="Electroporation"
                   class="flagImage" title="Flag 'Electroporation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="23782"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="23782"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+structure%22" title="Search for articles in the subject area:'RNA structure'"><div class="flagText">RNA structure</div></a>
              <div data-categoryid="35143" data-articleid="23782"
                   data-categoryname="RNA structure"
                   class="flagImage" title="Flag 'RNA structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribonucleases%22" title="Search for articles in the subject area:'Ribonucleases'"><div class="flagText">Ribonucleases</div></a>
              <div data-categoryid="46199" data-articleid="23782"
                   data-categoryname="Ribonucleases"
                   class="flagImage" title="Flag 'Ribonucleases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribozymes%22" title="Search for articles in the subject area:'Ribozymes'"><div class="flagText">Ribozymes</div></a>
              <div data-categoryid="17535" data-articleid="23782"
                   data-categoryname="Ribozymes"
                   class="flagImage" title="Flag 'Ribozymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="23782"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1585'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9237'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=844&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
