

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001977"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001977" />

    <meta name="citation_title" content="Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis"/>
    <meta itemprop="name" content="Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis"/>

      <meta name="citation_author" content="Jeremy L. Peirce"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
      <meta name="citation_author" content="Karl W. Broman"/>
            <meta name="citation_author_institution" content="Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Lu Lu"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
      <meta name="citation_author" content="Elissa J. Chesler"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
      <meta name="citation_author" content="Guomin Zhou"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
      <meta name="citation_author" content="David C. Airey"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
      <meta name="citation_author" content="Amanda E. Birmingham"/>
            <meta name="citation_author_institution" content="Dharmacon, Inc., Lafayette, Colorado, United States of America"/>
      <meta name="citation_author" content="Robert W. Williams"/>
            <meta name="citation_author_institution" content="Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America"/>

    <meta name="citation_date" content="2008/4/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001977.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1977"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=From QTL to gene: the harvest begins.; citation_author=R Korstanje; citation_author=B Paigen; citation_journal_title=Nat Genet; citation_volume=31; citation_number=1; citation_pages=235-236; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The nature and identification of quantitative trait loci: a community's view.; citation_author=O Abiola; citation_author=JM Angel; citation_author=P Avner; citation_author=AA Bachmanov; citation_author=JK Belknap; citation_journal_title=Nat Rev Genet; citation_volume=4; citation_number=2; citation_pages=911-916; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.; citation_author=A Darvasi; citation_author=M Soller; citation_journal_title=Genetics; citation_volume=141; citation_number=3; citation_pages=1199-1207; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Eae19, a new locus on rat chromosome 15 regulating experimental autoimmune encephalomyelitis. Genetics.; citation_author=J Sheng; citation_author=M Jagodic; citation_author=I Dahlman; citation_author=K Becanovic; citation_author=R Nohra; citation_number=4; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An advanced intercross line resolves Eae18 into two narrow quantitative trait loci syntenic to multiple sclerosis candidate loci.; citation_author=M Jagodic; citation_author=K Becanovic; citation_author=JR Sheng; citation_author=X Wu; citation_author=L Backdahl; citation_journal_title=J Immunol; citation_volume=173; citation_number=5; citation_pages=1366-1373; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fine mapping of trypanosomiasis resistance loci in murine advanced intercross lines.; citation_author=F Iraqi; citation_author=SJ Clapcott; citation_author=P Kumari; citation_author=CS Haley; citation_author=SJ Kemp; citation_journal_title=Mamm Genome; citation_volume=11; citation_number=6; citation_pages=645-648; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Using advanced intercross lines for high-resolution mapping of HDL cholesterol quantitative trait loci.; citation_author=X Wang; citation_author=I Le Roy; citation_author=E Nicodeme; citation_author=R Li; citation_author=R Wagner; citation_journal_title=Genome Res; citation_volume=13; citation_number=7; citation_pages=1654-1664; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fine mapping and identification of candidate pulmonary adenoma susceptibility 1 genes using advanced intercross lines.; citation_author=M Wang; citation_author=WJ Lemon; citation_author=G Liu; citation_author=Y Wang; citation_author=FA Iraqi; citation_journal_title=Cancer Res; citation_volume=63; citation_number=8; citation_pages=3317-3324; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fine mapping of quantitative trait loci using advanced intercross lines of mice and positional cloning of the corresponding genes.; citation_author=F Iraqi; citation_journal_title=Exp Lung Res; citation_volume=26; citation_number=9; citation_pages=641-649; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mouse Genetics.; citation_author=LM Silver; citation_number=10; citation_date=1995; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Power of daughter and granddaughter designs for determining linkage between marker loci and quantitative trait loci in dairy cattle.; citation_author=JI Weller; citation_author=Y Kashi; citation_author=M Soller; citation_journal_title=J Dairy Sci; citation_volume=73; citation_number=11; citation_pages=2525-2537; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Genetic architecture of the mouse hippocampus: identification of gene loci with selective regional effects.; citation_author=JL Peirce; citation_author=EJ Chesler; citation_author=RW Williams; citation_author=L Lu; citation_journal_title=Genes Brain Behav; citation_volume=2; citation_number=12; citation_pages=238-252; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Complex trait analysis of the hippocampus: mapping and biometric analysis of two novel gene loci with specific effects on hippocampal structure in mice.; citation_author=L Lu; citation_author=DC Airey; citation_author=RW Williams; citation_journal_title=J Neurosci; citation_volume=21; citation_number=13; citation_pages=3503-3514; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inappropriate choice of the experimental unit leads to a dramatic overestimation of the significance of quantitative trait loci for prepulse inhibition and startle response in recombinant congenic mice.; citation_author=AA Palmer; citation_author=DC Airey; citation_journal_title=Neuropsychopharmacology; citation_volume=28; citation_number=14; citation_pages=818; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A simple method for combining QTL maps from multiple data sources.; citation_author=JL Peirce; citation_author=K Broman; citation_author=L Lu; citation_author=RW Williams; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=15; citation_pages=e1036; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait locus analysis using recombinant inbred intercrosses (RIX): theoretical and empirical considerations.; citation_author=F Zou; citation_author=J Gelfond; citation_author=DC Airey; citation_author=L Lu; citation_author=KF Manly; citation_number=16; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait mapping in a diallel cross of recombinant inbred lines.; citation_author=S-W Tsaih; citation_author=L Lu; citation_author=DC Airey; citation_author=RW Williams; citation_author=GA Churchill; citation_journal_title=Mamm Genome; citation_volume=16; citation_number=17; citation_pages=344-355; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A new set of BXD recombinant inbred lines from advanced intercross populations in mice.; citation_author=JL Peirce; citation_author=L Lu; citation_author=J Gu; citation_author=LM Silver; citation_author=RW Williams; citation_journal_title=BMC Genet; citation_volume=5; citation_number=18; citation_pages=7; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The genomes of recombinant inbred lines.; citation_author=KW Broman; citation_journal_title=Genetics; citation_volume=169; citation_number=19; citation_pages=1133-1146; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.; citation_author=GA Churchill; citation_author=DC Airey; citation_author=H Allayee; citation_author=JM Angel; citation_author=AD Attie; citation_journal_title=Nat Genet; citation_volume=36; citation_number=20; citation_pages=1133-1137; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic structure of the LXS panel of recombinant inbred mouse strains: a powerful resource for complex trait analysis.; citation_author=RW Williams; citation_author=B Bennett; citation_author=L Lu; citation_author=J Gu; citation_author=JC DeFries; citation_journal_title=Mamm Genome; citation_volume=15; citation_number=21; citation_pages=637-647; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function.; citation_author=EJ Chesler; citation_author=L Lu; citation_author=S Shou; citation_author=Y Qu; citation_author=J Gu; citation_journal_title=Nat Genet; citation_volume=37; citation_number=22; citation_pages=233-242; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic correlates of gene expression in recombinant inbred strains: a relational model system to explore neurobehavioral phenotypes.; citation_author=EJ Chesler; citation_author=J Wang; citation_author=L Lu; citation_author=Y Qu; citation_author=KF Manly; citation_journal_title=Neuroinformatics; citation_volume=1; citation_number=23; citation_pages=343-357; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait locus analysis using recombinant inbred intercrosses (RIX): theoretical and empirical considerations. Genetics.; citation_author=F Zou; citation_author=JA Gelfond; citation_author=DC Airey; citation_author=L Lu; citation_author=KF Manly; citation_number=24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-tagged mice (GTM): two sets of genome-wide congenic strains.; citation_author=OA Iakoubova; citation_author=CL Olsson; citation_author=KM Dains; citation_author=DA Ross; citation_author=A Andalibi; citation_journal_title=Genomics; citation_volume=74; citation_number=25; citation_pages=89-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Quantitative-trait homozygosity and association mapping and empirical genomewide significance in large, complex pedigrees: fasting serum-insulin level in the Hutterites.; citation_author=M Abney; citation_author=C Ober; citation_author=MS McPeek; citation_journal_title=Am J Hum Genet; citation_volume=70; citation_number=26; citation_pages=920-934; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Eye1 and Eye2: gene loci that modulate eye size, lens weight, and retinal area in the mouse.; citation_author=G Zhou; citation_author=RW Williams; citation_journal_title=Invest Ophthalmol Vis Sci; citation_volume=40; citation_number=27; citation_pages=817-825; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Recombinant inbred strains.; citation_author=BA Taylor; citation_title=Genetic Variants and Strains of the Laboratory Mouse; citation_number=28; citation_pages=773-796; citation_date=1989; citation_publisher=Oxford UP; " />
      <meta name="citation_reference" content="citation_title=Genotyping new BXD recombinant inbred mouse strains and comparison of BXD and consensus maps.; citation_author=BA Taylor; citation_author=C Wnek; citation_author=BS Kotlus; citation_author=N Roemer; citation_author=T MacTaggart; citation_journal_title=Mamm Genome; citation_volume=10; citation_number=29; citation_pages=335-348; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The genetic structure of recombinant inbred mice: high-resolution consensus maps for complex trait analysis.; citation_author=RW Williams; citation_author=J Gu; citation_author=S Qi; citation_author=L Lu; citation_journal_title=Genome Biol; citation_volume=2; citation_number=30; citation_pages=RESEARCH0046; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multipoint estimation of identity-by-descent probabilities at arbitrary positions among marker loci on general pedigrees.; citation_author=E Sobel; citation_author=H Sengul; citation_author=DE Weeks; citation_journal_title=Hum Hered; citation_volume=52; citation_number=31; citation_pages=121-131; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Descent graphs in pedigree analysis: applications to haplotyping, location scores, and marker-sharing statistics.; citation_author=E Sobel; citation_author=K Lange; citation_journal_title=Am J Hum Genet; citation_volume=58; citation_number=32; citation_pages=1323-1337; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Crossover interference in the mouse.; citation_author=KW Broman; citation_author=LB Rowe; citation_author=GA Churchill; citation_author=K Paigen; citation_journal_title=Genetics; citation_volume=160; citation_number=33; citation_pages=1123-1131; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A statistical framework for quantitative trait mapping.; citation_author=S Sen; citation_author=GA Churchill; citation_journal_title=Genetics; citation_volume=159; citation_number=34; citation_pages=371-387; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=R/qtl: QTL mapping in experimental crosses.; citation_author=KW Broman; citation_author=H Wu; citation_author=S Sen; citation_author=GA Churchill; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=35; citation_pages=889-890; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Empirical threshold values for quantitative trait mapping.; citation_author=GA Churchill; citation_author=RW Doerge; citation_journal_title=Genetics; citation_volume=138; citation_number=36; citation_pages=963-971; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=R: A language for data analysis and graphics.; citation_author=R Ihaka; citation_author=R Gentleman; citation_journal_title=J Computational and Graphical Statistics; citation_volume=5; citation_number=37; citation_pages=299-314; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001977" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis"/>
    <meta name="twitter:description" content="BackgroundAdvanced intercross lines (AIL) are segregating populations created using a multi-generation breeding protocol for fine mapping complex trait loci (QTL) in mice and other organisms. Applying QTL mapping methods for intercross and backcross populations, often followed by na&iuml;ve permutation of individuals and phenotypes, does not account for the effect of AIL family structure in which final generations have been expanded and leads to inappropriately low significance thresholds. The critical problem with na&iuml;ve mapping approaches in AIL populations is that the individual is not an exchangeable unit.Methodology/Principal FindingsThe effect of family structure has immediate implications for the optimal AIL creation (many crosses, few animals per cross, and population expansion before the final generation) and we discuss these and the utility of AIL populations for QTL fine mapping. We also describe Genome Reshuffling for Advanced Intercross Permutation, (GRAIP) a method for analyzing AIL data that accounts for family structure. GRAIP permutes a more interchangeable unit in the final generation crosses &ndash; the parental genome &ndash; and simulating regeneration of a permuted AIL population based on exchanged parental identities. GRAIP determines appropriate genome-wide significance thresholds and locus-specific P-values for AILs and other populations with similar family structures. We contrast GRAIP with na&iuml;ve permutation using a large densely genotyped mouse AIL population (1333 individuals from 32 crosses). A na&iuml;ve permutation using coat color as a model phenotype demonstrates high false-positive locus identification and uncertain significance levels, which are corrected using GRAIP. GRAIP also detects an established hippocampus weight locus and a new locus, Hipp9a.Conclusions and SignificanceGRAIP determines appropriate genome-wide significance thresholds and locus-specific P-values for AILs and other populations with similar family structures. The effect of family structure has immediate implications for the optimal AIL creation and we discuss these and the utility of AIL populations."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001977.g007"/>

  <meta property="og:title" content="Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001977" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7440'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1912'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9269&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001977">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001977" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001977&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeremy L. Peirce
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:jpeirce@utmem.edu">jpeirce@utmem.edu</a></p>

                <p>Affiliations:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America, 
                  Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karl W. Broman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biostatistics, Johns Hopkins University, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lu Lu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America, 
                  Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elissa J. Chesler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America, 
                  Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guomin Zhou, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David C. Airey, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amanda E. Birmingham, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Dharmacon, Inc., Lafayette, Colorado, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert W. Williams
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Neuroscience, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America, 
                  Center for Genomics and Bioinformatics, Department of Anatomy and Neurobiology, University of Tennessee Health Science Center, Memphis, Tennessee, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001977</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001977" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001977" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001977" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001977" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001977-g001" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001977-g002" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001977-g003" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001977-g004" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001977-g005" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001977-g006" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001977-g007" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001977">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001977" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Advanced intercross lines (AIL) are segregating populations created using a multi-generation breeding protocol for fine mapping complex trait loci (QTL) in mice and other organisms. Applying QTL mapping methods for intercross and backcross populations, often followed by naïve permutation of individuals and phenotypes, does not account for the effect of AIL family structure in which final generations have been expanded and leads to inappropriately low significance thresholds. The critical problem with naïve mapping approaches in AIL populations is that the individual is not an exchangeable unit.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The effect of family structure has immediate implications for the optimal AIL creation (many crosses, few animals per cross, and population expansion before the final generation) and we discuss these and the utility of AIL populations for QTL fine mapping. We also describe Genome Reshuffling for Advanced Intercross Permutation, (GRAIP) a method for analyzing AIL data that accounts for family structure. GRAIP permutes a more interchangeable unit in the final generation crosses – the parental genome – and simulating regeneration of a permuted AIL population based on exchanged parental identities. GRAIP determines appropriate genome-wide significance thresholds and locus-specific P-values for AILs and other populations with similar family structures. We contrast GRAIP with naïve permutation using a large densely genotyped mouse AIL population (1333 individuals from 32 crosses). A naïve permutation using coat color as a model phenotype demonstrates high false-positive locus identification and uncertain significance levels, which are corrected using GRAIP. GRAIP also detects an established hippocampus weight locus and a new locus, <em>Hipp9a</em>.</p>


<h3>Conclusions and Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>GRAIP determines appropriate genome-wide significance thresholds and locus-specific P-values for AILs and other populations with similar family structures. The effect of family structure has immediate implications for the optimal AIL creation and we discuss these and the utility of AIL populations.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Peirce JL, Broman KW, Lu L, Chesler EJ, Zhou G, et al.  (2008) Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis. PLoS ONE 3(4):
          e1977.
            doi:10.1371/journal.pone.0001977</p><p><strong>Editor: </strong>Wim E. Crusio, Centre National de la Recherche Scientifique, France</p><p><strong>Received:</strong> February 4, 2007; <strong>Accepted:</strong> March 4, 2008; <strong>Published:</strong> April 23, 2008</p><p><strong>Copyright:</strong> © 2008 Peirce et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Our thanks for financial support from (1) The Informatics Center for Mouse Neurogenetics; (2) P20-MH62009 from NIMH, NIDA, and NSF to RWW. (2) INIA grants U01AA13499 and U24AA135B from NIAAA to RWW, and INIA grant U01AA01442501 from NIAAA to LL. As mentioned in the text, genotypes were provided for 1333 AIL mice by the Mammalian Genotyping Service (MGS; National Heart, Lung, and Blood Institute (NHLBI) Contract Number HV48141), for which we would like to express our gratitude to Dr. James Weber and colleagues. Except for MGS contribution of genotypes, no funding agency had a role in design, conduct, data collection, analysis, or interpretation or in preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The often-striking variation in heritable traits is usually produced by a multitude of polymorphic genes and a variety of environmental factors. Quantitative trait locus (QTL) mapping provides an effective approach to localizing regions of the genome that are likely to contain modifiers of the phenotype. Coarse mapping of a QTL to a 15–30 cM interval has become a relatively routine matter in traits with at least moderate heritability <a href="#pone.0001977-Korstanje1">[1]</a>, <a href="#pone.0001977-Abiola1">[2]</a>, but fine mapping–narrowing the QTL interval to include only a few candidate genes–is still a much more challenging task. Numerous genetic strategies for narrowing QTL intervals have been attempted with varying degrees of success.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Advanced intercross lines (AILs), first introduced by Darvasi and Soller <a href="#pone.0001977-Darvasi1">[3]</a>, are one such strategy that is capable of producing a population able to narrow mapping intervals for all QTLs for a given trait at once, given genotype information across the relevant intervals. Since their theoretical introduction, <a href="#pone.0001977-Darvasi1">[3]</a> AIL populations have been used by several groups to refine the positions of QTLs in mice <a href="#pone.0001977-Sheng1">[4]</a>–<a href="#pone.0001977-Wang2">[8]</a>. The statistical analysis in each case that we are aware of has been a standard mapping method designed for application to intercross (F2), backcross (N2), or recombinant inbred (RI) and similar populations where each individual (or strain in the case of RIs) can be treated as an independent observation.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>An AIL (<a href="#pone-0001977-g001">Fig. 1</a>, right panel) is generated by intercrossing a population of F2 animals to generate an F3 population. Members of the F3 population are themselves bred to create the fourth generation of the cross (G4) and so on, either by randomizing the choice of breeding pairs or by selecting the least related pairs for breeding at each generation in order to minimize fixation. Note that beyond F3, mating is not filial, which is indicted by the use of “G” for subsequent generations.</p>
<div class="figure" id="pone-0001977-g001"><div class="img"><a name="pone-0001977-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>RI and AIL breeding schemes.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The left panel of this figure diagrams breeding of a small recombinant inbred (RI) strain set. Each strain is essentially a set of repeated intercrosses starting with inbred parental strains. The animals are considered inbred at 20 generations. The right panel is an example of breeding an advanced intercross line. (Typically such lines consist of 50–100 animals at each generation rather than the 8 shown.) Letters A–H indicate 8 unique F2 animals, and their offspring “inherit” these identifiers. Breeding pairs are chosen for minimum relatedness at each generation.</p>
<span>doi:10.1371/journal.pone.0001977.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Given the complex breeding history of an AIL populations, however, the assumption of independence is formally incorrect, and this can become a serious problem depending on the details of the construction of the testing population (N) from the breeding population (N<sub>b</sub>), following Darvasi and Soller's nomenclature. Darvasi and Soller consider populations where N = N<sub>b</sub> as well as populations where a large number of animals (N) are derived from a moderate N<sub>b</sub>. It is with variations on the latter population structure that we are concerned here since populations where N = N<sub>b</sub> quickly become impractical given space and funding constraints.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Darvasi and Soller's simulation assumes that when expanding N<sub>b</sub> to generate N animals for testing, one offspring is taken from each of N crosses and tested (A. Darvasi, personal communication). In practice, AIL populations generated in the lab have generally been expanded in the final generation either by generating multiple litters from the same crosses (the UTHSC cross described below for example), or by retaining a larger number of animals in the penultimate generation and using this expanded set of animals to produce one litter per cross <a href="#pone.0001977-Iraqi1">[6]</a>, <a href="#pone.0001977-Iraqi2">[9]</a>. In the AIL population generated at UTHSC, for instance, there are many offspring (41±20; range 4–80 per cross) derived from 32 crosses in the final generation, with wide variation in family size. We will refer to the offspring of each cross generically as a family, but we are particularly concerned with the families resulting in the final generation, in our case the G10 generation resulting from a G9xG9 cross. Because there are few families, each with many members, use of an analysis method that deals appropriately with family relatedness and non-syntenic association in an AIL is crucial. Despite rotation or expansion of parents in the final generation, families will vary dramatically in their relatedness to each other and in the extent of fixation and non-syntenic association within each family and between closely related families based on the relatedness of particular breeding pairs.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>While family structure can probably be ignored if only one animal per cross is tested and there are a large number of crosses, there are several serious problems that result from neglecting family structure in AILs when multiple animals per family make up the N tested animals. First, simply shuffling genotypes and phenotypes as can be done with a population of genetically independent individuals constitutes over-randomization in an AIL and gives low genome-wide significance thresholds. Consider an analogous population—RIs. RI animals are fully inbred extended intercrosses (<a href="#pone-0001977-g001">Fig. 1</a>, left panel) usually formed from two or more inbred strains. Phenotyping a large population of several recombinant inbred strains is similar to the AIL situation in that the effective number of independent observations is smaller than the number of animals, because within strain the RIs are highly related (identical, in fact) to each other. This is similar to the observation that AILs in the same family are more highly related to each other than to AIL offspring in other families. This analogy is not complete in that AIL offspring between families are not independent while different RIs, at least those generated by pure repeated intercrosses from inbred strains, are independent.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>The second issue is non-syntenic association of markers within the offspring of each cross. In each family, assuming a 1:1 ratio of alleles in the population, which is the best-case scenario, 12.5% of loci will be fixed in both parents and thus in all offspring <a href="#pone.0001977-Silver1">[10]</a>. If a region containing a gene that affects the phenotype is fixed, all of the other fixed loci in that family will be in disequilibrium with the phenotypic variation, resulting in bias of particular loci and in false positive QTLs, especially where the number of families is small. While drift exacerbates this problem in AILs, it is the same problem that would be present in a population of F3 animals if more than one F3 from a particular F2×F2 cross were phenotyped and analyzed without respect to pedigree. (In such a population, however, a granddaughter design <a href="#pone.0001977-Weller1">[11]</a> could be used for analysis if the F2 genotype were known.) Of course, chance genotype correlations will exist in independent populations as well, but their strength will be rapidly reduced as the number of independent individuals increases. Likewise, a larger number of families in the final AIL generation will reduce the impact of this problem.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>A third problem is that there are likely to be different numbers of animals per family. There might be five litters of a particularly fecund family and one or two litters of a less fecund one if breeders are not rotated regularly, for instance, as was the case in our G9xG9 expansion to generate our G10 generation for testing. The genotype correlations present in larger families will have a greater effect on the mapping outcome than those present in smaller families.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>Another way to think about this problem is to take it to its logical extreme – a set of RI lines. Typically in an RI analysis the phenotypes of a number of RI animals of the same strain are averaged and the mean is used as the strain's phenotype, so each paired phenotype and genotype are independent. If the individual observations were used directly, instead, as would be the case with an F2 population—one entry for each animal rather than each strain, randomization by shuffling will obliterate the within-strain structure, resulting in overly low significance thresholds, and even small non-syntenic correlations would be exacerbated dramatically. This would be even more extreme than in an AIL because the strains are fully inbred so fixed loci are more prevalent. If different numbers of animals are selected to represent each strain's offspring, using the individual observations directly would also mean that a more fecund RI strain would have a much larger influence on the QTL map than a less fecund strain despite the fact that each represents only one genome.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>When evaluating the utility of AIL populations for fine mapping QTLs when genotyped in a narrow interval and for nominating new QTLs in a genome-wide scan, it is important to have an understanding of the ways in which family structure affects nominal significance at arbitrary points on the genome. Assumptions of independence are violated in AIL populations, and non-syntenic association can cause occurrence of nominally significant results to be more frequent than expected. When using selectively genotyped AIL populations for fine mapping purposes this is particularly problematic since genome-wide changes in the distribution of nominal significance measures will not be readily observable.</p>
<a id="article1.body1.sec1.p11" name="article1.body1.sec1.p11"></a><p>In order to appropriately address these challenges we developed GRAIP, (<a href="#pone-0001977-g002">Fig. 2</a>) an approach that interchanges genomes in the parents of the final cross of an AIL and appropriately simulates the final generation. The GRAIP approach also calculates P-values by locus to compensate for the non-random distribution of alleles at each locus. We describe an application of this method to a large (N = 1333 animals) AIL population developed at the University of Tennessee Health Science Center (UTHSC) and densely genotyped at 329 markers in the expanded population of the final generation. This extensively genotyped resource (most AIL populations are only genotyped near known QTL regions) facilitates generation of genome-wide QTL maps using an AIL population and allows us to evaluate the potential of AILs, when analyzed with approaches that account for family structure, for discovery of new QTLs in addition to fine mapping of known loci. By estimating the proportion of the genome above a given LOD score, we also evaluate the likelihood of mistakenly identifying signal in a QTL region when the strength of association at most markers is unknown, as is the case in most AIL-based studies.</p>
<div class="figure" id="pone-0001977-g002"><div class="img"><a name="pone-0001977-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The GRAIP approach.</span></strong></p><a id="article1.body1.sec1.fig2.caption1.p1" name="article1.body1.sec1.fig2.caption1.p1"></a><p>This cartoon summarizes the GRAIP approach. First (1) parental happlotypes are regenerated if they are not already known and (2) the parents are permuted. Next (3) the population of offspring is regenerated using the permuted genotypes and (4) permuted maps are generated using the non-permuted phenotypes. Finally, (5) the significance of the permuted maps are compared at a pointwise and whole genome basis with the original map.</p>
<span>doi:10.1371/journal.pone.0001977.g002</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>A direct comparison of GRAIP with a naïve permutation (randomization of phenotypes without respect to family structure) is shown below for two sample populations: an AIL and a set of RI strains taken as individual observations using coat color as a phenotype. Coat color is a well-characterized oligogenic trait for which all loci that segregate between B6 and D2 are known. We also present a comparison of GRAIP and naïve permutation for hippocampus weight, which is a far more polygenic phenotype that has been previously but not exhaustively characterized. While not all determinants for hippocampus weight are known, it is a good model of a more polygenic trait.</p>

<h4>Comparison of naïve permutation and GRAIP using coat color as a model phenotype</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Using coat color and hippocampus size <a href="#pone.0001977-Peirce1">[12]</a>, <a href="#pone.0001977-Lu1">[13]</a> as model phenotypes with well established genetic determinants, we compared the results of a naïve QTL mapping and permutation protocol (interval maps generated and permuted for significance as if the AIL were an F2 population) with GRAIP. For coat color, the loci expected to segregate between C57BL/6J (B6) and DBA/2J (D2) has been exhaustively characterized. We expect to detect two segregating loci: the brown locus (<em>Tyrp1</em>) on Chr. 4 at 80Mb and the dilute locus (<em>Myo5a</em>) on Chr. 9 at 155 Mb. (The agouti locus does not segregate between B6 and D2.) We generated both simple (naïve permutation) and GRAIP maps in the UTHSC AIL population (<a href="#pone-0001977-g003">Fig. 3a</a>) as well as in the 34 C57BL/6J×DBA/2J (BXD) RI strains (<a href="#pone-0001977-g004">Fig. 4b</a>) available from The Jackson Laboratory (TJL) as a control.</p>
<div class="figure" id="pone-0001977-g003"><div class="img"><a name="pone-0001977-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Coat color (A) and hippocampus weight (B) in the UTHSC AIL population Red traces are the simple mapping output, and the red bar is genome-wide P = 0.05 by naïve permutation.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Black traces are GRAIP permutation output. Note that for ease of graphing on a -log scale we have adjusted P&lt;1/10000 to P = 0.0001, so the maximum –log P = 4. Simple mapping results are on the left hand scale, while GRAIP results are on the right. On the Chr.4 coat color locus simple mapping value is truncated at LOD = 25, to simplify reading the graph. Shaded gray regions are significant at genome-wide P = 0.05 or better in the GRAIP results.</p>
<span>doi:10.1371/journal.pone.0001977.g003</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>If we examine the raw LOD scores from mapping coat color in the UTHSC AIL (<a href="#pone-0001977-g003">Fig. 3a</a>), using a naïve approach, it is clear that there are strong loci on Chrs 4 and 9, as expected, but there are loci with LOD scores above 4.1 (genome-wide adjusted p = 0.05 for the naïve permutation) on all chromosomes except Chr.10 (scale on left axis, trace and line indicating genome-wide significance in red). Admittedly, the strongest loci are Chrs 4 (maximum LOD score of 132) and 9 (maximum LOD score of 49) while other loci have LOD scores below 25, but since we know that only two loci are segregating, we can be relatively confident that the other loci detected as influences on coat color are false positives.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Enumeration of new QTLs in this population would therefore be impossible using a simple mapping method. Even confirmation of QTLs observed in simpler populations would be quite problematic given that, for coat color, 30% of the map is associated with a LOD score of at least 4.1, which could easily lead to mischaracterization of spurious association as confirmation of a QTL interval in a more sparsely genotyped population, at least for oligogenic traits.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>In contrast, the GRAIP results for coat color mapping (<a href="#pone-0001977-g003">Fig 3a</a>; scale on right axis, trace in black, significant loci shaded) show significance only on Chrs 4 and 9, (genome-wide P&lt;0.013, the minimum possible P-value with 10,000 permutations since 1.3% of permuted genome scans have at least one locus-specific P&lt;0.0001) exactly as expected. No other loci are close to a genome-wide P&lt;0.05.</p>


<h4>Mapping coat color using BXD individual observations using a naïve permutation and GRAIP</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>RI strains are similar to AIL strains in that within each family (i.e. strain), animals are genetically similar, albeit considerably more so than in an AIL. Treatment of RI individuals as unique yields inflated LOD scores <a href="#pone.0001977-Palmer1">[14]</a> in a manner similar to AIL mapping experiments, while application of GRAIP reconstitutes mapping results in a manner similar to mapping using strain means.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We applied a slightly modified version of the GRAIP approach (constrained to produce an inbred final generation) to a set of individual observations of coat color in BXDs using a 680 animal virtual population, (34 strains, with the number of animals per strain varying randomly between 2 and 40, with a mean of 20) assigning each animal the coat color associated with the strain. Determining haplotypes of the BXD parents in the previous generation was trivial since they are the same as the BXD offspring, and recombination does not affect the outcome since the parents and offspring are inbred.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The QTL map for the 34 BXDs from TJL for coat color is quite clear—the locus on Chr. 4 is highly significant. (There are too few strains to detect the locus on Chr. 9 using our coat color encoding scheme.)</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The comparison between the simple BXD coat color map based on strain means (<a href="#pone-0001977-g004">Fig. 4b</a>) and the simple map with naïve permutation based on individual animals (<a href="#pone-0001977-g004">Fig. 4a</a>; scale on left axis, trace and line indicating genome-wide significance in red) is striking. Not only is the significance of the actual QTL on Chr. 4 dramatically inflated (LOD = 140 (figure truncates at LOD = 50), up from LOD = 7 in the strain mean map) as would be expected <a href="#pone.0001977-Palmer1">[14]</a> but many other QTLs, including many with extremely high LOD scores, have sprung up as well. In fact there is at least one QTL with a LOD score over 5 (genome-wide P&lt;0.05 for the naïve permutation) on every chromosome.</p>
<div class="figure" id="pone-0001977-g004"><div class="img"><a name="pone-0001977-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>BXD coat color QTL-maps.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Coat color QTL maps treating BXD observations as independent individuals versus mapping strain means. (A) Comparison of simple mapping and GRAIP for BXDs treated as individuals. Red traces are simple mapping output, and the red bar is P = 0.05 for the naïve permutation. Black traces are GRAIP mapping output (5000 permutations) and shaded gray region is significant at genome-wide 0.05 or better in the GRAIP results. (B) simple mapping output for BXD strain means. Black bar indicates P = 0.05 for the naïve (appropriate, in this case) permutation.</p>
<span>doi:10.1371/journal.pone.0001977.g004</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Applying the GRAIP method eliminates these additional QTLs. (<a href="#pone-0001977-g004">Fig. 4a</a>; scale on right axis, trace in black, significant loci shaded) Only the Chr.4 QTL remains significant (P&lt;1/5000 with 5000 permutations is a genome-wide adjusted P&lt;0.024).</p>


<h4>Hippocampus weight</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>For examination of hippocampus weight in AILs(<a href="#pone-0001977-g003">Fig. 3b</a>) we focused mainly on the <em>Hipp1a</em> locus, which has been consistently identified in other populations. There are clearly other loci involved in a phenotype like hippocampus weight, and it would not be surprising for us to identify additional loci. Indeed, a glance at the simple mapping output for our AIL population indicates loci significant by naïve permutation (scale on left axis, trace and line indicating genome-wide significance in red) on Chrs 1, 8, 9, 12, and 16. The GRAIP results for the same population (scale on right axis, trace in black, significant loci shaded) only attribute genome-wide significance to loci on Chrs 1 and 9, however. Since the locus on Chr.9 has not been previously observed, except as an unpublished suggestive QTL for bi-lateral hippocampus weight (observation made using data from Lu and colleagues <a href="#pone.0001977-Lu1">[13]</a>; BXD published phenotypes record 10376 on <a href="http://www.genenetwork.org">www.genenetwork.org</a>) we have named it <em>Hipp9a</em>. While we cannot rule out the possibility that the additional loci are also real, it is encouraging that <em>Hipp1a</em> is replicated using the GRAIP approach and interesting that there are fewer loci significant by naïve permutation with hippocampus weight (where many loci may exist) than with coat color (where only two known loci are segregating).</p>


<h4>How much does family matter?</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>As can be seen from the box plot of hippocampus weight, (<a href="#pone-0001977-g005">Fig. 5</a>) there is a significant effect of family (p&lt;0.0001) on this phenotype, a phenomenon that we have observed for a variety of heritable characters including coat color (p&lt;0.0001). In order to evaluate the relative importance of a large number of crosses versus sheer number of animals, we sub-sampled our observations of hippocampus weight, progressively removing either entire families or an equal number of randomly selected animals. A typical result of performing this procedure once is shown in <a href="#pone-0001977-g006">Fig. 6</a>, We predicted that the number of families is more important than the total number of animals to the power of the population to detect linkage. The locus specific P value, for the well-established <em>Hipp1a</em> QTL on distal Chr. 1, chosen at the point of the best P-value in the original map, decreases relatively smoothly with number of samples in the randomly diminished population. In the population where individuals are removed by family, however, the change in significance is less monotonic. This is a direct result of the family effect on the phenotype. For instance, if a given family has a high overall score in a phenotype but an allele at a given QTL that would predispose for a low phenotype, each member of the family will reduce the evidence for that QTL. In some cases (the first three sets of two families removed) the level of significance is reduced while in the last four it is increased. The removal of some families seems to have a negative effect on the significance of a given locus while removal of others has a strong positive effect, while the removal of randomly chosen animals has a much more consistently negative effect on overall significance. Choosing different families or order of families to remove consistently shows these non-monotonic effects on significance, while the effect of removing groups random individuals on significance is consistently monotonic.</p>
<div class="figure" id="pone-0001977-g005"><div class="img"><a name="pone-0001977-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Box plot of hippocampus weight by family.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Whiskers represent the distribution of the highest and lowest 25% of observations. The line across the box represents the median value, while the “+” indicates the mean. Family 2 is missing because there were not hippocampus weight observations in that group.</p>
<span>doi:10.1371/journal.pone.0001977.g005</span></div><div class="figure" id="pone-0001977-g006"><div class="img"><a name="pone-0001977-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Effects of progressively removing samples from a population.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Measurement were taken at the position with the best P-value in the <em>Hipp1a</em> locus. Samples removed either by family or by an equivalent number of randomly selected individuals. –log P measured at the most significant position in the original BXD data set for hippocampus weight, near the physical center of the <em>Hipp1a</em> interval.</p>
<span>doi:10.1371/journal.pone.0001977.g006</span></div>

<h4>The value of locus-specific P values</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The common assumption in QTL analysis of large populations with independent observations is that any position in the genome is equally likely to be the best P-value in the genome under the null hypothesis of no linkage. Put another way, the assumption is that the P-values at all positions are identically distributed if there is no influence of genotype at that position. In the case of the UTHSC AIL population, however (<a href="#pone-0001977-g007">Fig. 7</a>) this is very clearly not the case. For coat color, the LOD score equivalent to a locus-specific P = 0.05 (referred to as 95% LOD) by GRAIP on Chr.1 varied from 6.4 to 14.7. For body weight the same range was 1.8 to 3.1—both considerably more varied than is typical for even medium sized F2 populations <a href="#pone.0001977-Peirce2">[15]</a>. The exact origin of this variation is unclear, but we examined several possible relationships in a set of 50 permutations. In this set, variation in the LOD score equivalent to a locus-specific P = 0.05 is not correlated with the fraction of missing data at the locus. (r = −0.07, P = 0.62) for coat color as it is for small RI strain sets <a href="#pone.0001977-Peirce2">[15]</a>.</p>
<div class="figure" id="pone-0001977-g007"><div class="img"><a name="pone-0001977-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001977" data-uri="info:doi/10.1371/journal.pone.0001977.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001977.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001977.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001977.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001977.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Variation in LOD score distribution by position on Chr 1.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Distribution of 95<sup>th</sup> percentile LOD scores by marker for 10,000 GRAIP permutations of coat color and body weight QTL mapping in the AIL population. Note that the maximum and minimum values of the 95<sup>th</sup> percentile on this chromosome alone are separated by a difference in LOD of 8.3 for coat color and 1.3 for body weight, which indicates that the same LOD score is equivalent to a considerably different P value depending on position and original phenotype.</p>
<span>doi:10.1371/journal.pone.0001977.g007</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>In order to examine whether the variation in 95% LOD scores is related to a marker's correlation with unlinked markers known to affect the phenotype (non-syntenic correlations), we calculated the absolute value of the Pearson's correlation coefficient between genotype at the major (<em>Tyrp1</em>) locus for coat color in the original data and genotype at 24 unlinked test markers on Chr.1 in data permuted using GRAIP, averaged over 1000 permutations (marker-<em>Tyrp1</em> correlation). We then calculated the Pearson's correlation coefficient between the marker-<em>Tyrp1</em> correlation and the 95% LOD score, which was significantly positive (r = 0.56, P = 0.004). In other words, GRAIP re-shuffled markers that were more highly correlated with the <em>Tyrp1</em> locus were associated with higher 95% LOD scores.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>This strong positive relationship may also explain the striking difference between variation in locus-specific P-values for body weight and coat color since coat color (using our encoding scheme) has only one major determinant and one minor determinant, while body weight has many influences of small effect. As described above, the LOD score at each position in each GRAIP permutation for coat color therefore varies widely depending on its correlation with the original genotype at <em>Tyrp1</em>, the major segregating coat color locus detected using our encoding of coat color.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have developed and implemented GRAIP, a permutation and simulation-based mapping approach for the analysis of AIL data, based on the idea that the identity of the parents of the final, phenotyped generation is an interchangeable unit, but the individuals in the final generation themselves, at least where the final generation expanded beyond one individual per pair of parents, are not. This concept is similar to permutation by interchanging the identity of parents used to generate a recombinant inbred intercross (RIX) population, <a href="#pone.0001977-Zou1">[16]</a>, <a href="#pone.0001977-Tsaih1">[17]</a> though generation of haplotypes and handling of segregation in generation of the permuted population is obviously unnecessary in an RIX population, and typically phenotypic measurements are made on the RI×RI F1 means rather than being applied to individual animals.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The GRAIP approach is necessary for most AIL populations because a naïve permutation approach does not take into account the effects of family structure in the AIL population, which are important when multiple offspring from the final generation are phenotyped. A naïve permutation is appropriate for a population without substantial family structure because it permutes the relationship of individual and phenotype. When the genetic factors we are attempting to detect are confounded with family identity, however, studying linkage without respect to the relatedness of family members yields significant non-syntenic linkage. The more oligogenic the trait the more different it is between families, and the stronger apparent non-syntenic QTLs will be, though family differences are significant even in a highly polygenic trait like hippocampus weight. GRAIP addresses this problem by accounting for the family structure of the AIL cross and choosing an appropriate unit, parental identity, for permutation. It also utilizes locus-specific P-values to account for the widely varying relationship between LOD score and P-value in the AIL population, which allows us to extend the utility of the method to genome-wide QTL analysis with appropriate genome-wide adjusted significance thresholds.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Mapping of both hippocampus weight and coat color in an AIL population using GRAIP returned expected loci. In addition, analysis of hippocampus weight using GRAIP returned a novel locus on Chr.9. For coat color, where only two loci are segregating between the parental strains, novel loci would have been indicative of a serious flaw of the method to discriminate false positives from true positives, but only the expected loci were returned. This was precisely the case when a naïve permutation was compared with GRAIP for coat color. In contrast, loci significant by naïve permutation for coat color were found on nearly all chromosomes.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The approach also performed well on individual observations of RI phenotypes rather than strain means, again returning the expected coat color loci. Treating multiple RI animals as individual observations is clearly not itself a valid method of analysis <a href="#pone.0001977-Palmer1">[14]</a> and is not a good choice for RI data even when adjusted using GRAIP. It is, however, a useful qualitative test of the approach.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The genome reshuffling and R/qtl mapping steps necessary to generate an original QTL map and GRAIP permutations can be executed in a day on a modern desktop computer for a small (3000-4000 permutations of a densely genotyped data set—a number sufficient to define the criteria for a genome-wide P&lt;0.05) number of permutations and in 4–5 days for a larger (15,000–20,000) set of permutations. Naturally, for smaller sets of genotypes much more typical in AIL experiments these times will also be considerably reduced.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>After applying GRAIP we used a simple mapping model to calculate the original and permuted QTL maps, which are then used to generate locus-specific P-values and genome-wide significance thresholds. Since GRAIP preserves the correlation structure, the distribution of permuted LOD statistics at each locus can be meaningfully compared to the LOD at each locus in the observed data to calculate locus-specific and genome-adjusted P-values that will ensure appropriately controlled type I error rates. The use of a more complex analysis method, such as a mixed model with random effects of parents, might improve the power to detect QTLs, and would not require modification of the GRAIP procedure itself.</p>

<h4>Alternate approaches and limitations of the GRAIP approach</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>One of the difficulties in analysis of this particular AIL pedigree was lack of complete genotype and pedigree information for all generations. Complete pedigree information, if available, would have allowed simulation of inheritance of each allele for the full pedigree. The approach we have taken accounts for relationships that come only from the final generations, and so does not fully solve the problem of uneven relatedness, for instance of members of the parental generation which was chosen as the permutable unit. We expect kinship coefficients, calculated assuming the parents are unrelated, will not be too much smaller than the true kinship coefficients, and so our approach accounts for the majority, but not all, of the relationship problem.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Assuming a lack of complete pedigree information, a intuitive alternative approach would be to permute phenotype within sibship. Unfortunately, this approach does not very fully break the association between genotype and phenotype. Siblings have more similar genotypes and also more similar phenotypes, and so real evidence for linkage would still appear in data permuted in this way. Likewise, permutation of genotype within sibship or regeneration of genotypes within sibship from parental haplotypes without permuting parental identity does not fully break the genotype/phenotype association.</p>


<h4>The usefulness of AILs for mapping of QTLs</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The most broadly applicable question related to AILs is whether they are a good population for fine mapping QTLs. Of particular concern are differences in the extent of non-syntenic correlation with observed markers, since this could affect the relative significance of neighboring and distant markers and bias the interpretation of mapping position. While AILs generated with larger numbers of smaller families, rotated breeding in the final generation, and a larger number of founders will very likely suffer much less severely from the reduction in power due to family structure, in our view the fundamental problem of fixation of loci within families and non-independence of samples remains as long as more than one animal per cross in the final generation is phenotyped, and these issues could substantially interfere with estimation of the QTL confidence interval. Although researchers have narrowed identified regions in AIL crosses that have subsequently shown overlap with congenic analyses <a href="#pone.0001977-Sheng1">[4]</a>, it is unclear to what extent this reflects widespread utility of the method.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>While we have found our AILs quite valuable as a founder population for a set of new, highly recombinant BXD RI strains there are better methods for generating useful and highly recombinant RI strains with two <a href="#pone.0001977-Peirce3">[18]</a> or more <a href="#pone.0001977-Broman1">[19]</a>, <a href="#pone.0001977-Churchill1">[20]</a> progenitor strains that result in full independence of recombination events between strains, so this use in itself probably does not constitute a reason to generate a high-generation AIL.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Alternative strategies such as large F2 crosses, RI mapping experiments with larger mapping panels <a href="#pone.0001977-Peirce3">[18]</a>, <a href="#pone.0001977-Williams1">[21]</a> especially in conjunction with transcriptome-QTL and genetic correlation approaches <a href="#pone.0001977-Chesler1">[22]</a>, <a href="#pone.0001977-Chesler2">[23]</a>, RIX crosses, <a href="#pone.0001977-Zou2">[24]</a> or construction of congenics <a href="#pone.0001977-Iakoubova1">[25]</a>, offer more rapid turnaround and, except perhaps in the case of the RIX, much simpler analysis procedures.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>The RIX population deserves particular mention here. Because it is based on intercrossing RI lines, which have several times the recombination density of F2 crosses, (especially if constructed using RI lines generated by inbreeding of AIL progeny) it offers a rapid means of generating a non-inbred population with a high density of recombination—essentially the goal of creating an AIL population. Since RI lines are in general already densely genotyped, balanced crosses emphasizing recombinations in areas of interest are relatively straightforward to construct and can be analyzed in combination with parental RI lines. In addition, RIX crosses have a distinct advantage over AILs in that they share with RIs the property of being a reference population <a href="#pone.0001977-Chesler2">[23]</a>. Thus multiple animals can be phenotyped to reduce variation or examine the genetic relatedness of multiple phenotypes.</p>


<h4>Recommendations on making an AIL population in mice and other populations</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The GRAIP approach is designed to account for the effects of family structure on the phenotypes of an AIL population. Minimizing these effects reduces the differences we would expect between GRAIP and a naïve permutation approach, which immediately suggests several recommendations for improving AIL populations.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>We believe that the issues described here in mice should apply to AILs developed in other organisms, where the final generation is expanded, but in organisms whose breeding characteristics allow large populations in intermediate generations and do not require expansion of the final generation to create enough individuals for QTL mapping projects, for instance Arabidopsis or maize, the issues we discuss can be alleviated.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Since this approach is not always feasible, subject to the relative costs of breeding and phenotyping, we would recommend expanding the AIL population 1-2 generations before the final generation of an AIL and making sure to rotate parents frequently in the process of generating the final generation. An AIL population based on 150 distinct families with 6 offspring each is likely to be a more powerful mapping population than one based on 30 crosses with 30 offspring each though in both cases 900 animals are being phenotyped.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>The approach taken by Iraqi and colleagues <a href="#pone.0001977-Iraqi1">[6]</a> of dramatically expanding the set of parents of the final generation and using a small number of animals per family should also dramatically improve the power of AILs as a mapping population. The small effect of randomly removing sample size shown in <a href="#pone-0001977-g006">Figure 6</a> suggests that analysis of large numbers of animals from each family is relatively inefficient, though as in the case of RI populations this may be less the case when heritability is low.</p>
<a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5"></a><p>Additionally, we would suggest retaining DNA samples from each member of each generation of the cross and, again subject to the cost of phenotyping, considering generating phenotypes for parents and grandparents of the final generation. It is possible that more complex models may allow retention of more information as in similar populations with more extensively retained histories <a href="#pone.0001977-Abney1">[26]</a> , and at a minimum retention of DNA samples at each generation will dramatically simplify determination of haplotypes in the final generation.</p>
<a id="article1.body1.sec3.sec3.p6" name="article1.body1.sec3.sec3.p6"></a><p>Finally, we would also recommend testing only loci that are well established using other, simpler populations and analyzing data using the GRAIP method rather than applying a simple mapping method. These precautions will improve the likelihood that fine mapping results using an AIL population represent genuine improvements in the location of genes underlying quantitative loci.</p>


<h4>Summary</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>AILs are an interesting approach to the problem of fine mapping, but generation of an AI population with strong mapping potential is more dependent on careful design of the breeding and testing populations than on sheer number of animals produced. Family structure is a serious problem with implications for published and future studies using this type of population and must be taken into account both at the design and analysis stages in order to avoid frequent false positive results and bias in identification of exact QTL positions, which will simply look like unexpectedly good fine mapping results in the absence of more genome-wide genotype data. The AIL approach is a potentially valuable method for fine mapping, provided care is taken in the generation of the mapping population. It is particularly important that family effects in the final generation be minimized by using a few individuals from any particular cross in the final generation—one offspring per cross is ideal!—and to generate the final phenotyped generation using as many crosses as possible.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Intercross and RI strain breeding and care</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>C57BL/6J (B6) and DBA/2J (D2) male and female animals were purchased from TJL (Bar Harbor, ME) and bred at UTHSC in a specific pathogen free (SPF) facility to generate B6D2F1 and D2B6F1 animals, which were intercrossed to generate an F2 population as described previously <a href="#pone.0001977-Zhou1">[27]</a>. All animals, parental, BXD, and all of the AIL families considered in this paper were bred and cared for according to the animal care and husbandry guidelines of UTHSC.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Commercially available BXD strains were purchased from TJL and bred in-house as required. These strains were generated by repeatedly intercrossing offspring of B6 and D2 parental strains in the mid-1970s (BXD1 through BXD32) and 1990s (BXD33 through BXD42) by Benjamin Taylor and colleagues <a href="#pone.0001977-Taylor1">[28]</a>, <a href="#pone.0001977-Taylor2">[29]</a>.</p>


<h4>Advanced intercross generation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>B6 and D2 male and female animals were purchased from TJL and bred at UTHSC in a specific pathogen free (SPF) facility to generate B6D2F1 and D2B6F1 animals, which were intercrossed to generate an F2 population. For all breeding following the F2 generation, two males and two females were placed in a breeding cage. When a female was observed to be pregnant, the other animals were removed from the cage. Approximately 30 breeding cages were maintained at each generation beyond F2. Animals from the F2 pool were randomly chosen and mated to create a F3 population in this manner. Following F3, breeding followed a version of the advanced intercross technique described by Darvasi and Soller <a href="#pone.0001977-Darvasi1">[3]</a>. Instead of random breeding, however, matings were chosen at each generation to minimize the number of common parents by examining common ancestors in the third generation prior to the generation being set up. Breeding partners were chosen manually so that there was no more than one common ancestor in the previous three generations.</p>


<h4>Genotyping</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Genotyping of our F2 <a href="#pone.0001977-Zhou1">[27]</a> and the BXD RI strains <a href="#pone.0001977-Williams2">[30]</a> strains has been described elsewhere. For the advanced intercross a total of 329 microsatellite loci polymorphic between B6 and D2 strains and distributed across all autosomes and the X chromosome were amplified and typed. 329 loci gives a resolution of approximately 4.6 cM on the unexpanded map of the mouse genome. Since Darvasi and Soller <a href="#pone.0001977-Darvasi1">[3]</a> estimated a n/2 expansion and we were genotyping 10<sup>th</sup> generation AIL individuals we expected a post-expansion genotyping density of 22.8 cM. Given the largen and uneven family sizes in the final generation, however, the observed expansion was extremely uneven, though average expansion was similar to expectation. Markers only a few cM apart were apparently unlinked, while other regions showed no expansion or even contraction. This observation is likely to be exclusively an artifact of the final generation expansion.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Genotypes were provided for all 1333 10<sup>th</sup> generation AIL mice courtesy of the Mammalian Genotyping Service (MGS; National Heart, Lung, and Blood Institute (NHLBI) Contract Number HV48141), and the authors would like to express their gratitude to Dr. James Weber and colleagues for this extensive and valuable data set. Genotype files are currently available online at <a href="http://www.nervenet.org/mmfiles/mmlist.html">www.nervenet.org/mmfiles/mmlist.html</a>. Physical positions were taken from the May 2004 University of California, Santa Cruise (UCSC) mouse genome assembly (mm5), which used data obtained from the Build 33 assembly by the National Center for Biotechnology Information (NCBI).</p>


<h4>Phenotyping</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We used hippocampus weight and coat color as example phenotypes. Coat color was assessed in the entire advanced intercross while hippocampus weight was only measured in 679 animals. Measurement of these weights has been described previously <a href="#pone.0001977-Lu1">[13]</a>. We used a numerical scale to encode coat color (0 = black, 1 = gray, 2 = brown, 3 = DBA (dilute, brown, non-agouti)) as a “darkness of coat” observation. This is a convenient method of capturing the dilute (<em>d</em>) and brown (<em>b</em>) loci, which segregate between B6 and D2. The coat color loci segregating between B6 and D2 have known molecular determinants: tyrosinase-related protein 1 (<em>Tyrp1</em>; Chr.4: 79.1 Mb; also known as the brown (<em>b</em>) locus) and myosin 5a (<em>Myo5a</em>; Chr.9: 75.4 Mb; also known as the dilute (<em>d</em>) locus).</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Because of the encoding, we expect the brown locus (phenotypes 0 and 1 have the black allele, while phenotypes 2 and 3 have the brown allele) to be more strongly detected than the dilute locus (phenotypes 0 and 2 have the non-dilute allele, while phenotypes 1 and 3 have the dilute allele), depending on the exact number of animals with each coat color. This combination of what are actually multiple separate effects into a single scale is typical of complex traits, where the multiple contributions of separate genes are almost always collapsed along a single axis.</p>


<h4>Estimation of parental haplotypes</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Parental haplotypes were estimated using SimWalk 2.6 <a href="#pone.0001977-Sobel1">[31]</a>, <a href="#pone.0001977-Sobel2">[32]</a>, selecting at most 13 siblings per family for speed of computation and considering one chromosome at a time. Where the software was unable to predict to which strand a particular allele belonged, we assigned allele to strand so as to minimize number of recombination events. In regions where both parents were heterozygous the phase was often ambiguous. Provided that individual genotypes in the penultimate generation can be accurately inferred from the final generation, however, these haplotypes are sufficient for simple single marker association because analysis occurs only at the marker and does not require flanking marker status. Inclusion of grandparental identity and thereby relations among families helps to reduce uncertainty, but simple single marker-only mapping is still preferable if the density of markers is sufficient. All haplotypes used are available at <a href="http://www.nervenet.org/papers/GRAIP.html">http://www.nervenet.org/papers/GRAIP.htm​l</a>.</p>


<h4>Genome reshuffling for AI permutation (GRAIP) genotypes</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Since the unit to be exchanged is the identity of the parents of the final, phenotyped generation, we first shuffled the identities of individuals within sex in this generation. Next we generated virtual gametes and combined them to create the genotypes of the permuted final AIL generation. Briefly, the number of chiasmata per chromosome in each gamete is determined by drawing from a Poisson distribution with mean equal to the length of the chromosome in Morgans. Positions of the chiasmata are drawn from a uniform distribution of the length of the chromosome, iteratively thinned to ensure a maximum of one recombination in a given distance (default is 10 cM) centered around a given chiasma. This estimate is slightly low because simulating interference in this manner lowers the mean number of recombinations. However, even separate runs with two-fold variation in recombination rates were extremely similar. This is expected since we are estimating association only at actual marker positions. We are using standard MGI (Mouse Genome Informatics) estimates of genetic position, not AIL-based calculated genetic positions which are highly biased by repetition of early recombinations in the population, to estimate missing physical positions (below). Recombination positions could also be simulated using a more sophisticated model <a href="#pone.0001977-Broman2">[33]</a>.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>A starting haplotype is then randomly chosen and the script imposes the recombination pattern on the marker positions and switches between haplotypes as needed to simulate recombination. Since sex is assigned to each generated individual, the X chromosome from the father is either assigned as an intact, randomly chosen haplotype (for females) or treated as null (for males), and the genotype of the X chromosome in the zygote is generated appropriately. The output of this process is a set of genotype files each containing the set of genotypes of a permuted AIL population.</p>


<h4>Mapping</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The original population, naïve permutations, and GRAIP permuted genotypes were treated identically with respect to generation of QTL maps. Since the AIL we are using as a test population is relatively densely genotyped we mapped using regression at marker locations in all cases, which also had the effect of halving the time required and eliminating potential worries about the accuracy of assigning genotype probabilities between markers. We used the multiple imputation method of Sen and Churchill <a href="#pone.0001977-Sen1">[34]</a> as implemented in the R/qtl package for the R environment <a href="#pone.0001977-Broman3">[35]</a> to handle missing data, imputing 16 sets of genotypes per map. (No significant differences were observed between output using 16 and 64 imputations, and mapping results omitting imputation are also quite similar.) The X chromosome was handled as if it were an autosome since the mapping model is designed for an F2 population and requires identity of the cross direction to handle X chromosome data correctly, so loci on this chromosome should be treated with care. Genome-wide adjusted significance thresholds for naïve permutations were generated by the method of Churchill and Doerge. <a href="#pone.0001977-Churchill2">[36]</a></p>


<h4>Generating locus-specific P-values for physical positions</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>We permuted AIL parental identities, regenerated permuted final AIL generations, and performed QTL mapping for all permutations using the mapping protocol described above. Once all GRAIP permuted maps were generated, we calculated locus-specific p-values for each marker for the original data using the observations from the permuted genome maps at each locus as a null distribution. While locus-specific P-values derived from permutations are not crucial for all populations, the AIL family structure causes dramatic variation between LOD score at a given locus and corresponding P-value.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>We describe determination of locus specific P-values and discuss their usefulness for different data sets in detail elsewhere <a href="#pone.0001977-Peirce2">[15]</a>. Briefly, at every locus j we compare L(j), the LOD score at that locus in the original data with L*(i,j), the LOD scores at locus j in each of i permuted data sets. The locus-specific P-value, P(j) is the proportion of L*(i,j) ≤L(j). The distribution of locus specific P-values was uniform for randomly generated phenotypes in which a family bias was chosen and individual phenotypes selected from a normal distribution around the family-biased mean.</p>
<a id="article1.body1.sec4.sec8.p3" name="article1.body1.sec4.sec8.p3"></a><p>P-values at regular intervals were interpolated based on the known physical and genetic positions of markers. We linearly interpolated missing physical positions using flanking markers with known physical and genetic positions and linearly interpolated P values on a regularly spaced set of physical positions.</p>


<h4>Generating genome-wide adjusted P-values</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>We computed locus specific P-values, P*(i,j), for each L*(i,j) in the manner described above where L(j) is L*(j) for a fixed i<sup>th</sup> GRAIP permutation. Genome-wide adjusted P-value at locus j, Pa(j), can then be computed by creating an ordered list, MP*, of the minimum P-value from each GRAIP permuted map. Then Pa(j) is the proportion of MP*≤P(j). This is similar to Churchill and Doerge <a href="#pone.0001977-Churchill2">[36]</a> with the addition of explicit conversion of best permuted observation (typically expressed as a LOD score) in each genome scan to a locus-specific P-value.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>Our implementation varied slightly from the procedure described above in the final step. To generate MP* we sampled with replacement at least 1000 times from our available set P*(i) genome scans in order to maintain consistency with our general approach to combining multiple data sets, described elsewhere <a href="#pone.0001977-Peirce2">[15]</a>. For large samples these methods produce essentially the same MP* distribution.</p>


<h4>Applying GRAIP to BXD RI data</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>As previously introduced, the BXD strains are similar to AIL strains in that a population of BXD strains is usually organized into a set of individual offspring. Since these offspring are inbred, a phenotypic mean is often associated with a single copy of each strain's genotype to reduce non-genetic noise. We created two virtual populations of BXD RI animals using the 34 strains from The Jackson Laboratory and mapped coat color using three approaches, each employing R/qtl as described. We first associated our coat color phenotype with a single genome for each strain as described above.</p>
<a id="article1.body1.sec4.sec10.p2" name="article1.body1.sec4.sec10.p2"></a><p>Next we generated a virtual population of 680 BXD RI animals, with the number of animals per strain as a random, uniformly distributed even number between 2 and 40, with a mean of 20. We generated GRAIP-shuffled genomes for these populations as described above with the slight modification that male and female parent identities were permuted together, since in this case the parents are themselves are constrained to be inbred.</p>


<h4>Testing the effects of family structure on QTL significance</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>In order to provide an example of the effects of eliminating different segments of the population on a more complex phenotype, we examined hippocampal weight, a phenotype we have measured and published on before <a href="#pone.0001977-Peirce1">[12]</a>, <a href="#pone.0001977-Lu1">[13]</a> which has a particularly reliable QTL on distal Chr.1, which is also detected in the AIL population. In order to demonstrate the effect of families on QTL significance, we progressively removed either entire crosses or an equivalent number of randomly chosen individuals from the original population (31 crosses, 679 individuals). We evaluated significance using the locus-specific P value, evaluated at a marker estimated to be near the middle of the well-established <em>Hipp1</em> interval on distal Chr. 1 on a physical scale.</p>


<h4>Software</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>All software described is available at <a href="http://www.nervenet.org/papers/GRAIP.html">http://www.nervenet.org/papers/GRAIP.htm​l</a>. For the analyses above, we wrote a script, <strong>GRAIPGeno.py</strong>, using Python 2.4 to generate permutations of final generation genotypes. Since each GRAIP-based permutation requires generation of a complete QTL map, we wrote a simple R <a href="#pone.0001977-Ihaka1">[37]</a> script, <strong>GRAIP.R</strong>, to automate the process, which was performed using R/qtl <a href="#pone.0001977-Broman3">[35]</a>. Parsing of mapping output files from R/qtl, generation of locus-specific P-values, and generation of genome-wide adjusted P-values were all handled by Python scripts described elsewhere <a href="#pone.0001977-Peirce2">[15]</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>As mentioned in the text, genotypes were provided for 1333 AIL mice by the Mammalian Genotyping Service (MGS; National Heart, Lung, and Blood Institute (NHLBI) Contract Number HV48141), for which we would like to express our gratitude to Dr. James Weber and colleagues. We also thank Arthur Centeno for computer support and Pamela Franklin for administrative assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RW JP KB. Performed the experiments: JP. Analyzed the data: DA EC JP. Contributed reagents/materials/analysis tools: RW LL GZ. Wrote the paper: JP. Other: Generated AILs whose analysis is discussed in the paper and provided editiorial comments and support: RW. Generated AILs whose analysis is discussed in the paper and provided editorial comments and support: LL. Helped with writing and presentation of code: AB. Provided suggestions for approaches and analyzed early AIL data for this cross: DA. Provided useful discussions on analysis of AILs: EC. Wrote the code: JP. Generated AIL population used for analysis in this paper: GZ. Provided initial idea for GRAIP as well as suggestions for symbolic representation of the process: KB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001977-Korstanje1" id="pone.0001977-Korstanje1"></a>Korstanje R, Paigen B (2002) From QTL to gene: the harvest begins. Nat Genet  31: 235–236.  <ul class="find" data-citedArticleID="1090110" data-doi="10.1038/ng0702-235"><li><a href="http://dx.doi.org/10.1038/ng0702-235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+QTL+to+gene%3A+the+harvest+begins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+QTL+to+gene%3A+the+harvest+begins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001977-Abiola1" id="pone.0001977-Abiola1"></a>Abiola O, Angel JM, Avner P, Bachmanov AA, Belknap JK, et al.  (2003) The nature and identification of quantitative trait loci: a community's view. Nat Rev Genet  4: 911–916.  <ul class="find" data-citedArticleID="1090080" data-doi="10.1038/nrg1206"><li><a href="http://dx.doi.org/10.1038/nrg1206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nature+and+identification+of+quantitative+trait+loci%3A+a+community%27s+view." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nature+and+identification+of+quantitative+trait+loci%3A+a+community%27s+view.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001977-Darvasi1" id="pone.0001977-Darvasi1"></a>Darvasi A, Soller M (1995) Advanced intercross lines, an experimental population for fine genetic mapping. Genetics  141: 1199–1207.  <ul class="find" data-citedArticleID="1090098" data-doi="10.1016/b978-0-12-374984-0.00021-8"><li><a href="http://dx.doi.org/10.1016/b978-0-12-374984-0.00021-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advanced+intercross+lines%2C+an+experimental+population+for+fine+genetic+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advanced+intercross+lines%2C+an+experimental+population+for+fine+genetic+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001977-Sheng1" id="pone.0001977-Sheng1"></a>Sheng J, Jagodic M, Dahlman I, Becanovic K, Nohra R, et al.  (2005) Eae19, a new locus on rat chromosome 15 regulating experimental autoimmune encephalomyelitis. Genetics.  <ul class="find" data-citedArticleID="1090124" data-doi="10.1534/genetics.104.035261"><li><a href="http://dx.doi.org/10.1534/genetics.104.035261" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eae19%2C+a+new+locus+on+rat+chromosome+15+regulating+experimental+autoimmune+encephalomyelitis.+Genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eae19%2C+a+new+locus+on+rat+chromosome+15+regulating+experimental+autoimmune+encephalomyelitis.+Genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001977-Jagodic1" id="pone.0001977-Jagodic1"></a>Jagodic M, Becanovic K, Sheng JR, Wu X, Backdahl L, et al.  (2004) An advanced intercross line resolves Eae18 into two narrow quantitative trait loci syntenic to multiple sclerosis candidate loci. J Immunol  173: 1366–1373.  <ul class="find" data-citedArticleID="1090108"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+advanced+intercross+line+resolves+Eae18+into+two+narrow+quantitative+trait+loci+syntenic+to+multiple+sclerosis+candidate+loci.&amp;auth=&amp;atitle=An+advanced+intercross+line+resolves+Eae18+into+two+narrow+quantitative+trait+loci+syntenic+to+multiple+sclerosis+candidate+loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+advanced+intercross+line+resolves+Eae18+into+two+narrow+quantitative+trait+loci+syntenic+to+multiple+sclerosis+candidate+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+advanced+intercross+line+resolves+Eae18+into+two+narrow+quantitative+trait+loci+syntenic+to+multiple+sclerosis+candidate+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001977-Iraqi1" id="pone.0001977-Iraqi1"></a>Iraqi F, Clapcott SJ, Kumari P, Haley CS, Kemp SJ, et al.  (2000) Fine mapping of trypanosomiasis resistance loci in murine advanced intercross lines. Mamm Genome  11: 645–648.  <ul class="find" data-citedArticleID="1090104" data-doi="10.1007/s003350010133"><li><a href="http://dx.doi.org/10.1007/s003350010133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine+mapping+of+trypanosomiasis+resistance+loci+in+murine+advanced+intercross+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine+mapping+of+trypanosomiasis+resistance+loci+in+murine+advanced+intercross+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001977-Wang1" id="pone.0001977-Wang1"></a>Wang X, Le Roy I, Nicodeme E, Li R, Wagner R, et al.  (2003) Using advanced intercross lines for high-resolution mapping of HDL cholesterol quantitative trait loci. Genome Res  13: 1654–1664.  <ul class="find" data-citedArticleID="1090138" data-doi="10.1101/gr.1185803"><li><a href="http://dx.doi.org/10.1101/gr.1185803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+advanced+intercross+lines+for+high-resolution+mapping+of+HDL+cholesterol+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+advanced+intercross+lines+for+high-resolution+mapping+of+HDL+cholesterol+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001977-Wang2" id="pone.0001977-Wang2"></a>Wang M, Lemon WJ, Liu G, Wang Y, Iraqi FA, et al.  (2003) Fine mapping and identification of candidate pulmonary adenoma susceptibility 1 genes using advanced intercross lines. Cancer Res  63: 3317–3324.  <ul class="find" data-citedArticleID="1090140"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fine+mapping+and+identification+of+candidate+pulmonary+adenoma+susceptibility+1+genes+using+advanced+intercross+lines.&amp;auth=&amp;atitle=Fine+mapping+and+identification+of+candidate+pulmonary+adenoma+susceptibility+1+genes+using+advanced+intercross+lines." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine+mapping+and+identification+of+candidate+pulmonary+adenoma+susceptibility+1+genes+using+advanced+intercross+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine+mapping+and+identification+of+candidate+pulmonary+adenoma+susceptibility+1+genes+using+advanced+intercross+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001977-Iraqi2" id="pone.0001977-Iraqi2"></a>Iraqi F (2000) Fine mapping of quantitative trait loci using advanced intercross lines of mice and positional cloning of the corresponding genes. Exp Lung Res  26: 641–649.  <ul class="find" data-citedArticleID="1090106" data-doi="10.1080/01902140150216729"><li><a href="http://dx.doi.org/10.1080/01902140150216729" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine+mapping+of+quantitative+trait+loci+using+advanced+intercross+lines+of+mice+and+positional+cloning+of+the+corresponding+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine+mapping+of+quantitative+trait+loci+using+advanced+intercross+lines+of+mice+and+positional+cloning+of+the+corresponding+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001977-Silver1" id="pone.0001977-Silver1"></a>Silver LM (1995) Mouse Genetics. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001977-Weller1" id="pone.0001977-Weller1"></a>Weller JI, Kashi Y, Soller M (1990) Power of daughter and granddaughter designs for determining linkage between marker loci and quantitative trait loci in dairy cattle. J Dairy Sci  73: 2525–2537.  <ul class="find" data-citedArticleID="1090142"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Power+of+daughter+and+granddaughter+designs+for+determining+linkage+between+marker+loci+and+quantitative+trait+loci+in+dairy+cattle.&amp;auth=&amp;atitle=Power+of+daughter+and+granddaughter+designs+for+determining+linkage+between+marker+loci+and+quantitative+trait+loci+in+dairy+cattle." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Power+of+daughter+and+granddaughter+designs+for+determining+linkage+between+marker+loci+and+quantitative+trait+loci+in+dairy+cattle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Power+of+daughter+and+granddaughter+designs+for+determining+linkage+between+marker+loci+and+quantitative+trait+loci+in+dairy+cattle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001977-Peirce1" id="pone.0001977-Peirce1"></a>Peirce JL, Chesler EJ, Williams RW, Lu L (2003) Genetic architecture of the mouse hippocampus: identification of gene loci with selective regional effects. Genes Brain Behav  2: 238–252.  <ul class="find" data-citedArticleID="1090116" data-doi="10.1034/j.1601-183x.2003.00030.x"><li><a href="http://dx.doi.org/10.1034/j.1601-183x.2003.00030.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+architecture+of+the+mouse+hippocampus%3A+identification+of+gene+loci+with+selective+regional+effects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+architecture+of+the+mouse+hippocampus%3A+identification+of+gene+loci+with+selective+regional+effects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001977-Lu1" id="pone.0001977-Lu1"></a>Lu L, Airey DC, Williams RW (2001) Complex trait analysis of the hippocampus: mapping and biometric analysis of two novel gene loci with specific effects on hippocampal structure in mice. J Neurosci  21: 3503–3514.  <ul class="find" data-citedArticleID="1090112"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice.&amp;auth=&amp;atitle=Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+trait+analysis+of+the+hippocampus%3A+mapping+and+biometric+analysis+of+two+novel+gene+loci+with+specific+effects+on+hippocampal+structure+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001977-Palmer1" id="pone.0001977-Palmer1"></a>Palmer AA, Airey DC (2003) Inappropriate choice of the experimental unit leads to a dramatic overestimation of the significance of quantitative trait loci for prepulse inhibition and startle response in recombinant congenic mice. Neuropsychopharmacology  28: 818.  author reply 819.  <ul class="find" data-citedArticleID="1090114" data-doi="10.1038/sj.npp.1300064"><li><a href="http://dx.doi.org/10.1038/sj.npp.1300064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inappropriate+choice+of+the+experimental+unit+leads+to+a+dramatic+overestimation+of+the+significance+of+quantitative+trait+loci+for+prepulse+inhibition+and+startle+response+in+recombinant+congenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inappropriate+choice+of+the+experimental+unit+leads+to+a+dramatic+overestimation+of+the+significance+of+quantitative+trait+loci+for+prepulse+inhibition+and+startle+response+in+recombinant+congenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001977-Peirce2" id="pone.0001977-Peirce2"></a>Peirce JL, Broman K, Lu L, Williams RWA simple method for combining QTL maps from multiple data sources. PLoS ONE  2: (10)e1036.  <ul class="find" data-citedArticleID="1090118" data-doi="10.1371/journal.pone.0001036"><li><a href="http://dx.doi.org/10.1371/journal.pone.0001036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+method+for+combining+QTL+maps+from+multiple+data+sources." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+method+for+combining+QTL+maps+from+multiple+data+sources.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001977-Zou1" id="pone.0001977-Zou1"></a>Zou F, Gelfond J, Airey DC, Lu L, Manly KF, et al. Quantitative trait locus analysis using recombinant inbred intercrosses (RIX): theoretical and empirical considerations.  <ul class="find" data-citedArticleID="1090150" data-doi="10.1534/genetics.104.035709"><li><a href="http://dx.doi.org/10.1534/genetics.104.035709" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+locus+analysis+using+recombinant+inbred+intercrosses+%28RIX%29%3A+theoretical+and+empirical+considerations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+locus+analysis+using+recombinant+inbred+intercrosses+%28RIX%29%3A+theoretical+and+empirical+considerations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001977-Tsaih1" id="pone.0001977-Tsaih1"></a>Tsaih S-W, Lu L, Airey DC, Williams RW, Churchill GA (2005) Quantitative trait mapping in a diallel cross of recombinant inbred lines. Mamm Genome  16: 344–355.  <ul class="find" data-citedArticleID="1090136" data-doi="10.1007/s00335-004-2466-1"><li><a href="http://dx.doi.org/10.1007/s00335-004-2466-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+mapping+in+a+diallel+cross+of+recombinant+inbred+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+mapping+in+a+diallel+cross+of+recombinant+inbred+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001977-Peirce3" id="pone.0001977-Peirce3"></a>Peirce JL, Lu L, Gu J, Silver LM, Williams RW (2004) A new set of BXD recombinant inbred lines from advanced intercross populations in mice. BMC Genet  5: 7.  <ul class="find" data-citedArticleID="1090120"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice.&amp;auth=&amp;atitle=A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001977-Broman1" id="pone.0001977-Broman1"></a>Broman KW (2005) The genomes of recombinant inbred lines. Genetics  169: 1133–1146.  <ul class="find" data-citedArticleID="1090084" data-doi="10.1534/genetics.104.035212"><li><a href="http://dx.doi.org/10.1534/genetics.104.035212" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genomes+of+recombinant+inbred+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genomes+of+recombinant+inbred+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001977-Churchill1" id="pone.0001977-Churchill1"></a>Churchill GA, Airey DC, Allayee H, Angel JM, Attie AD, et al.  (2004) The Collaborative Cross, a community resource for the genetic analysis of complex traits. Nat Genet  36: 1133–1137.  <ul class="find" data-citedArticleID="1090094" data-doi="10.1038/ng1104-1133"><li><a href="http://dx.doi.org/10.1038/ng1104-1133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Collaborative+Cross%2C+a+community+resource+for+the+genetic+analysis+of+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Collaborative+Cross%2C+a+community+resource+for+the+genetic+analysis+of+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001977-Williams1" id="pone.0001977-Williams1"></a>Williams RW, Bennett B, Lu L, Gu J, DeFries JC, et al.  (2004) Genetic structure of the LXS panel of recombinant inbred mouse strains: a powerful resource for complex trait analysis. Mamm Genome  15: 637–647.  <ul class="find" data-citedArticleID="1090144" data-doi="10.1007/s00335-004-2380-6"><li><a href="http://dx.doi.org/10.1007/s00335-004-2380-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure+of+the+LXS+panel+of+recombinant+inbred+mouse+strains%3A+a+powerful+resource+for+complex+trait+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure+of+the+LXS+panel+of+recombinant+inbred+mouse+strains%3A+a+powerful+resource+for+complex+trait+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001977-Chesler1" id="pone.0001977-Chesler1"></a>Chesler EJ, Lu L, Shou S, Qu Y, Gu J, et al.  (2005) Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function. Nat Genet  37: 233–242.  <ul class="find" data-citedArticleID="1090090" data-doi="10.1038/ng1518"><li><a href="http://dx.doi.org/10.1038/ng1518" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+trait+analysis+of+gene+expression+uncovers+polygenic+and+pleiotropic+networks+that+modulate+nervous+system+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+trait+analysis+of+gene+expression+uncovers+polygenic+and+pleiotropic+networks+that+modulate+nervous+system+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001977-Chesler2" id="pone.0001977-Chesler2"></a>Chesler EJ, Wang J, Lu L, Qu Y, Manly KF, et al.  (2003) Genetic correlates of gene expression in recombinant inbred strains: a relational model system to explore neurobehavioral phenotypes. Neuroinformatics  1: 343–357.  <ul class="find" data-citedArticleID="1090092" data-doi="10.1385/ni:1:4:343"><li><a href="http://dx.doi.org/10.1385/ni:1:4:343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+correlates+of+gene+expression+in+recombinant+inbred+strains%3A+a+relational+model+system+to+explore+neurobehavioral+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+correlates+of+gene+expression+in+recombinant+inbred+strains%3A+a+relational+model+system+to+explore+neurobehavioral+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001977-Zou2" id="pone.0001977-Zou2"></a>Zou F, Gelfond JA, Airey DC, Lu L, Manly KF, et al.  (2005) Quantitative trait locus analysis using recombinant inbred intercrosses (RIX): theoretical and empirical considerations. Genetics.  <ul class="find" data-citedArticleID="1090152" data-doi="10.1534/genetics.104.035709"><li><a href="http://dx.doi.org/10.1534/genetics.104.035709" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+locus+analysis+using+recombinant+inbred+intercrosses+%28RIX%29%3A+theoretical+and+empirical+considerations.+Genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+locus+analysis+using+recombinant+inbred+intercrosses+%28RIX%29%3A+theoretical+and+empirical+considerations.+Genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001977-Iakoubova1" id="pone.0001977-Iakoubova1"></a>Iakoubova OA, Olsson CL, Dains KM, Ross DA, Andalibi A, et al.  (2001) Genome-tagged mice (GTM): two sets of genome-wide congenic strains. Genomics  74: 89–104.  <ul class="find" data-citedArticleID="1090100" data-doi="10.1006/geno.2000.6497"><li><a href="http://dx.doi.org/10.1006/geno.2000.6497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-tagged+mice+%28GTM%29%3A+two+sets+of+genome-wide+congenic+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-tagged+mice+%28GTM%29%3A+two+sets+of+genome-wide+congenic+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001977-Abney1" id="pone.0001977-Abney1"></a>Abney M, Ober C, McPeek MS (2002) Quantitative-trait homozygosity and association mapping and empirical genomewide significance in large, complex pedigrees: fasting serum-insulin level in the Hutterites. Am J Hum Genet  70: 920–934.  <ul class="find" data-citedArticleID="1090082" data-doi="10.1086/339705"><li><a href="http://dx.doi.org/10.1086/339705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative-trait+homozygosity+and+association+mapping+and+empirical+genomewide+significance+in+large%2C+complex+pedigrees%3A+fasting+serum-insulin+level+in+the+Hutterites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative-trait+homozygosity+and+association+mapping+and+empirical+genomewide+significance+in+large%2C+complex+pedigrees%3A+fasting+serum-insulin+level+in+the+Hutterites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001977-Zhou1" id="pone.0001977-Zhou1"></a>Zhou G, Williams RW (1999) Eye1 and Eye2: gene loci that modulate eye size, lens weight, and retinal area in the mouse. Invest Ophthalmol Vis Sci  40: 817–825.  <ul class="find" data-citedArticleID="1090148"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Eye1+and+Eye2%3A+gene+loci+that+modulate+eye+size%2C+lens+weight%2C+and+retinal+area+in+the+mouse.&amp;auth=&amp;atitle=Eye1+and+Eye2%3A+gene+loci+that+modulate+eye+size%2C+lens+weight%2C+and+retinal+area+in+the+mouse." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eye1+and+Eye2%3A+gene+loci+that+modulate+eye+size%2C+lens+weight%2C+and+retinal+area+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eye1+and+Eye2%3A+gene+loci+that+modulate+eye+size%2C+lens+weight%2C+and+retinal+area+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001977-Taylor1" id="pone.0001977-Taylor1"></a>Taylor BA (1989)  Recombinant inbred strains. In: Lyon ML, Searle AG, editors. Genetic Variants and Strains of the Laboratory Mouse. Oxford: Oxford UP. pp. 773–796.  <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0001977-Taylor2" id="pone.0001977-Taylor2"></a>Taylor BA, Wnek C, Kotlus BS, Roemer N, MacTaggart T, et al.  (1999) Genotyping new BXD recombinant inbred mouse strains and comparison of BXD and consensus maps. Mamm Genome  10: 335–348.  <ul class="find" data-citedArticleID="1090134" data-doi="10.1007/s003359900998"><li><a href="http://dx.doi.org/10.1007/s003359900998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+new+BXD+recombinant+inbred+mouse+strains+and+comparison+of+BXD+and+consensus+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+new+BXD+recombinant+inbred+mouse+strains+and+comparison+of+BXD+and+consensus+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001977-Williams2" id="pone.0001977-Williams2"></a>Williams RW, Gu J, Qi S, Lu L (2001) The genetic structure of recombinant inbred mice: high-resolution consensus maps for complex trait analysis. Genome Biol  2: RESEARCH0046.  <ul class="find" data-citedArticleID="1090146"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+genetic+structure+of+recombinant+inbred+mice%3A+high-resolution+consensus+maps+for+complex+trait+analysis.&amp;auth=&amp;atitle=The+genetic+structure+of+recombinant+inbred+mice%3A+high-resolution+consensus+maps+for+complex+trait+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+structure+of+recombinant+inbred+mice%3A+high-resolution+consensus+maps+for+complex+trait+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+structure+of+recombinant+inbred+mice%3A+high-resolution+consensus+maps+for+complex+trait+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001977-Sobel1" id="pone.0001977-Sobel1"></a>Sobel E, Sengul H, Weeks DE (2001) Multipoint estimation of identity-by-descent probabilities at arbitrary positions among marker loci on general pedigrees. Hum Hered  52: 121–131.  <ul class="find" data-citedArticleID="1090128" data-doi="10.1159/000053366"><li><a href="http://dx.doi.org/10.1159/000053366" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multipoint+estimation+of+identity-by-descent+probabilities+at+arbitrary+positions+among+marker+loci+on+general+pedigrees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multipoint+estimation+of+identity-by-descent+probabilities+at+arbitrary+positions+among+marker+loci+on+general+pedigrees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001977-Sobel2" id="pone.0001977-Sobel2"></a>Sobel E, Lange K (1996) Descent graphs in pedigree analysis: applications to haplotyping, location scores, and marker-sharing statistics. Am J Hum Genet  58: 1323–1337.  <ul class="find" data-citedArticleID="1090130"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Descent+graphs+in+pedigree+analysis%3A+applications+to+haplotyping%2C+location+scores%2C+and+marker-sharing+statistics.&amp;auth=&amp;atitle=Descent+graphs+in+pedigree+analysis%3A+applications+to+haplotyping%2C+location+scores%2C+and+marker-sharing+statistics." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Descent+graphs+in+pedigree+analysis%3A+applications+to+haplotyping%2C+location+scores%2C+and+marker-sharing+statistics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Descent+graphs+in+pedigree+analysis%3A+applications+to+haplotyping%2C+location+scores%2C+and+marker-sharing+statistics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001977-Broman2" id="pone.0001977-Broman2"></a>Broman KW, Rowe LB, Churchill GA, Paigen K (2002) Crossover interference in the mouse. Genetics  160: 1123–1131.  <ul class="find" data-citedArticleID="1090086"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Crossover+interference+in+the+mouse.&amp;auth=&amp;atitle=Crossover+interference+in+the+mouse." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crossover+interference+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crossover+interference+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001977-Sen1" id="pone.0001977-Sen1"></a>Sen S, Churchill GA (2001) A statistical framework for quantitative trait mapping. Genetics  159: 371–387.  <ul class="find" data-citedArticleID="1090122"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+statistical+framework+for+quantitative+trait+mapping.&amp;auth=&amp;atitle=A+statistical+framework+for+quantitative+trait+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+statistical+framework+for+quantitative+trait+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+statistical+framework+for+quantitative+trait+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001977-Broman3" id="pone.0001977-Broman3"></a>Broman KW, Wu H, Sen S, Churchill GA (2003) R/qtl: QTL mapping in experimental crosses. Bioinformatics  19: 889–890.  <ul class="find" data-citedArticleID="1090088" data-doi="10.1093/bioinformatics/btg112"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=R%2Fqtl%3A+QTL+mapping+in+experimental+crosses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22R%2Fqtl%3A+QTL+mapping+in+experimental+crosses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001977-Churchill2" id="pone.0001977-Churchill2"></a>Churchill GA, Doerge RW (1994) Empirical threshold values for quantitative trait mapping. Genetics  138: 963–971.  <ul class="find" data-citedArticleID="1090096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Empirical+threshold+values+for+quantitative+trait+mapping.&amp;auth=&amp;atitle=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+threshold+values+for+quantitative+trait+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001977-Ihaka1" id="pone.0001977-Ihaka1"></a>Ihaka R, Gentleman R (1996) R: A language for data analysis and graphics. J Computational and Graphical Statistics  5: 299–314.  <ul class="find" data-citedArticleID="1090102" data-doi="10.2307/1390807"><li><a href="http://dx.doi.org/10.2307/1390807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=R%3A+A+language+for+data+analysis+and+graphics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22R%3A+A+language+for+data+analysis+and+graphics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.XML" value="99920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.PDF" value="479264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g001.PNG_L" value="213033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g001.PNG_M" value="37357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g001.PNG_S" value="6127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g001.TIF" value="311412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g001.PNG_I" value="13574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g002.PNG_L" value="35628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g002.PNG_M" value="30235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g002.PNG_S" value="5168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g002.TIF" value="65364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g002.PNG_I" value="11138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g003.PNG_L" value="332028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g003.PNG_M" value="144647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g003.PNG_S" value="11462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g003.TIF" value="551864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g003.PNG_I" value="48824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g004.PNG_L" value="370239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g004.PNG_M" value="123089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g004.PNG_S" value="9587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g004.TIF" value="636156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g004.PNG_I" value="41353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g005.PNG_L" value="381647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g005.PNG_M" value="28071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g005.PNG_S" value="5435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g005.TIF" value="656768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g005.PNG_I" value="10139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g006.PNG_L" value="358751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g006.PNG_M" value="50589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g006.PNG_S" value="7535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g006.TIF" value="453116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g006.PNG_I" value="16362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g007.PNG_L" value="248178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g007.PNG_M" value="102041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g007.PNG_S" value="11009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g007.TIF" value="397434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001977.g007.PNG_I" value="35239"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001977&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001977" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001977&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001977&volume=&issue=&title=Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis&author_name=Jeremy%20L.%20Peirce%2C%20Karl%20W.%20Broman%2C%20Lu%20Lu%2C%20Elissa%20J.%20Chesler%2C%20Guomin%20Zhou%2C%20David%20C.%20Airey%2C%20Amanda%20E.%20Birmingham%2C%20Robert%20W.%20Williams&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001977" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001977" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001977" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001977&amp;t=Genome%20Reshuffling%20for%20Advanced%20Intercross%20Permutation%20(GRAIP)%3A%20Simulation%20and%20Permutation%20for%20Advanced%20Intercross%20Population%20Analysis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001977&title=Genome%20Reshuffling%20for%20Advanced%20Intercross%20Permutation%20(GRAIP)%3A%20Simulation%20and%20Permutation%20for%20Advanced%20Intercross%20Population%20Analysis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001977&amp;title=Genome%20Reshuffling%20for%20Advanced%20Intercross%20Permutation%20(GRAIP)%3A%20Simulation%20and%20Permutation%20for%20Advanced%20Intercross%20Population%20Analysis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001977" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001977&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome Reshuffling for Advanced Intercross Permutation (GRAIP): Simulation and Permutation for Advanced Intercross Population Analysis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001977';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome%20Reshuffling%20for%20Advanced%20Intercross%20Permutation%20(GRAIP)%3A%20Simulation%20and%20Permutation%20for%20Advanced%20Intercross%20Population%20Analysis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001977" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001977" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="26778"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+sexual+behavior%22" title="Search for articles in the subject area:'Animal sexual behavior'"><div class="flagText">Animal sexual behavior</div></a>
              <div data-categoryid="21247" data-articleid="26778"
                   data-categoryname="Animal sexual behavior"
                   class="flagImage" title="Flag 'Animal sexual behavior' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="26778"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="26778"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hippocampus%22" title="Search for articles in the subject area:'Hippocampus'"><div class="flagText">Hippocampus</div></a>
              <div data-categoryid="47345" data-articleid="26778"
                   data-categoryname="Hippocampus"
                   class="flagImage" title="Flag 'Hippocampus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Inbred+strains%22" title="Search for articles in the subject area:'Inbred strains'"><div class="flagText">Inbred strains</div></a>
              <div data-categoryid="35433" data-articleid="26778"
                   data-categoryname="Inbred strains"
                   class="flagImage" title="Flag 'Inbred strains' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Permutation%22" title="Search for articles in the subject area:'Permutation'"><div class="flagText">Permutation</div></a>
              <div data-categoryid="33641" data-articleid="26778"
                   data-categoryname="Permutation"
                   class="flagImage" title="Flag 'Permutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="26778"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7005'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4714'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8814&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
