

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Kinetic Pathway of Pyrophosphorolysis by a Retrotransposon Reverse Transcriptase</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001389"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001389" />

    <meta name="citation_title" content="Kinetic Pathway of Pyrophosphorolysis by a Retrotransposon Reverse Transcriptase"/>
    <meta itemprop="name" content="Kinetic Pathway of Pyrophosphorolysis by a Retrotransposon Reverse Transcriptase"/>

      <meta name="citation_author" content="Manjula Pandey"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Rutgers University, Piscataway, New Jersey, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Medicine and Dentistry of New Jersey, Piscataway, New Jersey, United States of America"/>
      <meta name="citation_author" content="Smita S. Patel"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Medicine and Dentistry of New Jersey, Piscataway, New Jersey, United States of America"/>
      <meta name="citation_author" content="Abram Gabriel"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Rutgers University, Piscataway, New Jersey, United States of America"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001389.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1389"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=DNA polymerases: structural diversity and common mechanisms.; citation_author=TA Steitz; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=1; citation_pages=17395-17398; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=An attempt to unify the structure of polymerases.; citation_author=M Delarue; citation_author=O Poch; citation_author=N Tordo; citation_author=D Moras; citation_author=P Argos; citation_journal_title=Protein Eng; citation_volume=3; citation_number=2; citation_pages=461-467; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Viral RNA-dependent DNA polymerase.; citation_author=D Baltimore; citation_journal_title=Nature; citation_volume=226; citation_number=3; citation_pages=1209-1211; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=RNA-dependent DNA polymerase in virions of Rous sarcoma virus.; citation_author=HM Temin; citation_author=S Mizutani; citation_journal_title=Nature; citation_volume=226; citation_number=4; citation_pages=1211-1213; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=A detailed model of reverse transcription and a test of crucial aspects.; citation_author=E Gilboa; citation_author=SW Mitra; citation_author=S Goff; citation_author=D Baltimore; citation_journal_title=Cell; citation_volume=18; citation_number=5; citation_pages=93-100; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Reverse Transcriptase.; citation_number=6; citation_pages=492; citation_date=1993; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Reverse transcriptase motifs in the catalytic subunit of telomerase.; citation_author=J Lingner; citation_author=TR Hughes; citation_author=A Shevchenko; citation_author=M Mann; citation_author=V Lundblad; citation_journal_title=Science; citation_volume=276; citation_number=7; citation_pages=561-567; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Site-specific mutagenesis of AIDS virus reverse transcriptase.; citation_author=BA Larder; citation_author=DJ Purifoy; citation_author=KL Powell; citation_author=G Darby; citation_journal_title=Nature; citation_volume=327; citation_number=8; citation_pages=716-717; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Subunit-selective mutagenesis indicates minimal polymerase activity in heterodimer-associated p51 HIV-1 reverse transcriptase.; citation_author=SFJ Le Grice; citation_author=T Naas; citation_author=B Wohlgensinger; citation_author=O Schatz; citation_journal_title=EMBO J; citation_volume=10; citation_number=9; citation_pages=3905-3911; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A Ty1 reverse transcriptase active-site aspartate mutation blocks transposition but not polymerization.; citation_author=O Uzun; citation_author=A Gabriel; citation_journal_title=J Virol; citation_volume=75; citation_number=10; citation_pages=6337-6347; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Functional roles of carboxylate residues comprising the DNA polymerase active site triad of Ty3 reverse transcriptase.; citation_author=A Bibillo; citation_author=D Lener; citation_author=GJ Klarmann; citation_author=SF Le Grice; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=11; citation_pages=171-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Insights into the role of an active site aspartate in Ty1 reverse transcriptase polymerization.; citation_author=M Pandey; citation_author=S Patel; citation_author=A Gabriel; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=12; citation_pages=47840-47848; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Efficient pyrophosphorolysis by a hepatitis B virus polymerase may be a primer-unblocking mechanism.; citation_author=S Urban; citation_author=KP Fischer; citation_author=DL Tyrrell; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=13; citation_pages=4984-4989; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Excision of incorporated nucleotide analogue chain-terminators can diminish their inhibitory effects on viral RNA-dependent RNA polymerases.; citation_author=CM D'Abramo; citation_author=L Cellai; citation_author=M Gotte; citation_journal_title=J Mol Biol; citation_volume=337; citation_number=14; citation_pages=1-14; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Primer unblocking by HIV-1 reverse transcriptase and resistance to nucleoside RT inhibitors (NRTIs).; citation_author=V Goldschmidt; citation_author=R Marquet; citation_journal_title=Int J Biochem Cell Biol; citation_volume=36; citation_number=15; citation_pages=1687-1705; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Unblocking of chain-terminated primer by HIV-1 reverse transcriptase through a nucleotide-dependent mechanism.; citation_author=PR Meyer; citation_author=SE Matsuura; citation_author=AG So; citation_author=WA Scott; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=16; citation_pages=13471-13476; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Relationship between 3'-azido-3'-deoxythymidine resistance and primer unblocking activity in foscarnet-resistant mutants of human immunodeficiency virus type 1 reverse transcriptase.; citation_author=PR Meyer; citation_author=SE Matsuura; citation_author=D Zonarich; citation_author=RR Chopra; citation_author=E Pendarvis; citation_journal_title=J Virol; citation_volume=77; citation_number=17; citation_pages=6127-6137; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Trapping HIV-1 reverse transcriptase before and after translocation on DNA.; citation_author=SG Sarafianos; citation_author=AD Clark; citation_author=S Tuske; citation_author=CJ Squire; citation_author=K Das; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=18; citation_pages=16280-16288; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structures of HIV-1 reverse transcriptase with pre- and post-translocation AZTMP-terminated DNA.; citation_author=SG Sarafianos; citation_author=AD Clark; citation_author=K Das; citation_author=S Tuske; citation_author=JJ Birktoft; citation_journal_title=Embo J; citation_volume=21; citation_number=19; citation_pages=6614-6624; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structure of a covalently trapped catalytic complex of HIV-1 reverse transcriptase: implications for drug resistance.; citation_author=H Huang; citation_author=R Chopra; citation_author=GL Verdine; citation_author=SC Harrison; citation_journal_title=Science; citation_volume=282; citation_number=20; citation_pages=1669-1675; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Biochemical analysis of catalytically crucial aspartate mutants of human immunodeficiency virus type 1 reverse transcriptase.; citation_author=N Kaushik; citation_author=N Rege; citation_author=PN Yadav; citation_author=SG Sarafianos; citation_author=MJ Modak; citation_journal_title=Biochemistry; citation_volume=35; citation_number=21; citation_pages=11536-11546; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Inhibition of reverse transcription in vivo by elevated manganese ion concentration.; citation_author=EC Bolton; citation_author=AS Mildvan; citation_author=JD Boeke; citation_journal_title=Mol Cell; citation_volume=9; citation_number=22; citation_pages=879-889; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Elongation complexes of Thermus thermophilus RNA polymerase that possess distinct translocation conformations.; citation_author=E Kashkina; citation_author=M Anikin; citation_author=TH Tahirov; citation_author=SN Kochetkov; citation_author=DG Vassylyev; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=23; citation_pages=4036-4045; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Expression of an active form of recombinant Ty1 reverse transcriptase in Escherichia coli: a fusion protein containing the C-terminal region of the Ty1 integrase linked to the reverse transcriptase–RNase H domain exhibits polymerase and RNase H activities.; citation_author=M Wilhelm; citation_author=M Boutabout; citation_author=F-X Wilhelm; citation_journal_title=Biochem J; citation_volume=348; citation_number=24; citation_pages=337-342; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reverse transcriptase and integrase of the Saccharomyces cerevisiae Ty1 element.; citation_author=FX Wilhelm; citation_author=M Wilhelm; citation_author=A Gabriel; citation_journal_title=Cytogenet Genome Res; citation_volume=110; citation_number=25; citation_pages=269-287; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structure and mechanism of DNA polymerase Beta.; citation_author=WA Beard; citation_author=SH Wilson; citation_journal_title=Chem Rev; citation_volume=106; citation_number=26; citation_pages=361-382; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structural basis for the transition from initiation to elongation transcription in T7 RNA polymerase.; citation_author=YW Yin; citation_author=TA Steitz; citation_journal_title=Science; citation_volume=298; citation_number=27; citation_pages=1387-1395; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Poliovirus RNA-dependent RNA polymerase (3Dpol): pre-steady-state kinetic analysis of ribonucleotide incorporation in the presence of Mg2+.; citation_author=JJ Arnold; citation_author=CE Cameron; citation_journal_title=Biochemistry; citation_volume=43; citation_number=28; citation_pages=5126-5137; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phenotypic mechanism of HIV-1 resistance to 3'-azido-3'-deoxythymidine (AZT): increased polymerization processivity and enhanced sensitivity to pyrophosphate of the mutant viral reverse transcriptase.; citation_author=D Arion; citation_author=N Kaushik; citation_author=S McCormick; citation_author=G Borkow; citation_author=MA Parniak; citation_journal_title=Biochemistry; citation_volume=37; citation_number=29; citation_pages=15908-15917; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Kinetic mechanism of the DNA-dependent DNA polymerase activity of human immunodeficiency virus reverse transcriptase.; citation_author=JC Hsieh; citation_author=S Zinnen; citation_author=P Modrich; citation_journal_title=J Biol Chem; citation_volume=268; citation_number=30; citation_pages=24607-24613; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus reverse transcriptase. A kinetic analysis of RNA-dependent and DNA-dependent DNA polymerization.; citation_author=JE Reardon; citation_journal_title=J Biol Chem; citation_volume=268; citation_number=31; citation_pages=8743-8751; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Pre-steady-state kinetic analysis of processive DNA replication including complete characterization of an exonuclease-deficient mutant.; citation_author=SS Patel; citation_author=I Wong; citation_author=KA Johnson; citation_journal_title=Biochemistry; citation_volume=30; citation_number=32; citation_pages=511-525; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Transient state kinetics of transcription elongation by T7 RNA polymerase.; citation_author=VS Anand; citation_author=SS Patel; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=33; citation_pages=35677-35685; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Magnesium-induced assembly of a complete DNA polymerase catalytic complex.; citation_author=VK Batra; citation_author=WA Beard; citation_author=DD Shock; citation_author=JM Krahn; citation_author=LC Pedersen; citation_journal_title=Structure; citation_volume=14; citation_number=34; citation_pages=757-766; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Energy analysis of chemistry for correct insertion by DNA polymerase beta.; citation_author=P Lin; citation_author=LC Pedersen; citation_author=VK Batra; citation_author=WA Beard; citation_author=SH Wilson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=35; citation_pages=13294-13299; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mechanics of myosin motor: force and step size.; citation_author=MY Jiang; citation_author=MP Sheetz; citation_journal_title=Bioessays; citation_volume=16; citation_number=36; citation_pages=531-532; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Insights into DNA polymerization mechanisms from structure and function analysis of HIV-1 reverse transcriptase.; citation_author=PH Patel; citation_author=A Jacobo-Molina; citation_author=J Ding; citation_author=C Tantillo; citation_author=AD Clark; citation_journal_title=Biochemistry; citation_volume=34; citation_number=37; citation_pages=5351-5365; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The structural mechanism of translocation and helicase activity in T7 RNA polymerase.; citation_author=YW Yin; citation_author=TA Steitz; citation_journal_title=Cell; citation_volume=116; citation_number=38; citation_pages=393-404; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A model for the mechanism of polymerase translocation.; citation_author=R Guajardo; citation_author=R Sousa; citation_journal_title=J Mol Biol; citation_volume=265; citation_number=39; citation_pages=8-19; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Direct observation of base-pair stepping by RNA polymerase.; citation_author=EA Abbondanzieri; citation_author=WJ Greenleaf; citation_author=JW Shaevitz; citation_author=R Landick; citation_author=SM Block; citation_journal_title=Nature; citation_volume=438; citation_number=40; citation_pages=460-465; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA polymerase active site is highly mutable: evolutionary consequences.; citation_author=PH Patel; citation_author=LA Loeb; citation_journal_title=PNAS; citation_volume=97; citation_number=41; citation_pages=5095-5100; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Transient-state kinetic analysis of enzyme reaction pathways.; citation_author=KA Johnson; citation_number=42; citation_pages=1-61; citation_date=1992; citation_publisher=Academic Press; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001389" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Kinetic Pathway of Pyrophosphorolysis by a Retrotransposon Reverse Transcriptase"/>
    <meta name="twitter:description" content="DNA and RNA polymerases use a common phosphoryl transfer mechanism for base addition that requires two or three acidic amino acid residues at their active sites. We previously showed, for the reverse transcriptase (RT) encoded by the yeast retrotransposon Ty1, that one of the three conserved active site aspartates (D211) can be substituted by asparagine and still retain in vitro polymerase activity, although in vivo transposition is lost. Transposition is partially restored by second site suppressor mutations in the RNAse H domain. The novel properties of this amino acid substitution led us to express the WT and D211N mutant enzymes, and study their pre-steady state kinetic parameters. We found that the kpol was reduced by a factor of 223 in the mutant, although the Kd for nucleotide binding was unaltered. Further, the mutant enzyme had a marked preference for Mn2+ over Mg2+. To better understand the functions of this residue within the Ty1 RT active site, we have now examined the in vitro properties of WT and D211N mutant Ty1 RTs in carrying out pyrophosphorolysis, the reverse reaction to polymerization, where pyrophosphate is the substrate and dNTPs are the product. We find that pyrophosphorolysis is efficient only when the base-paired primer template region is &gt;14 bases, and that activity increases when the primer end is blunt-ended or recessed by only a few bases. Using pre-steady state kinetic analysis, we find that the rate of pyrophosphorolysis (kpyro) in the D211N mutant is nearly 320 fold lower than the WT enzyme, and that the mutant enzyme has an &sim;170 fold lower apparent Kd for pyrophosphate. These findings indicate that subtle substrate differences can strongly affect the enzyme's ability to properly position the primer-end to carry out pyrophosphorolysis. Further the kinetic data suggests that the D211 residue has a role in pyrophosphate binding and release, which could affect polymerase translocation, and help explain the D211N mutant's transposition defect."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001389.g007"/>

  <meta property="og:title" content="Kinetic Pathway of Pyrophosphorolysis by a Retrotransposon Reverse Transcriptase" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001389" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3686'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3499'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=290&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001389">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001389" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001389&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Kinetic Pathway of Pyrophosphorolysis by a Retrotransposon Reverse Transcriptase
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Manjula Pandey, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Molecular Biology and Biochemistry, Rutgers University, Piscataway, New Jersey, United States of America, 
                  Department of Biochemistry, University of Medicine and Dentistry of New Jersey, Piscataway, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Smita S. Patel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Medicine and Dentistry of New Jersey, Piscataway, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Abram Gabriel
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gabriel@cabm.rutgers.edu">gabriel@cabm.rutgers.edu</a></p>

                <p>Affiliation:
                  Department of Molecular Biology and Biochemistry, Rutgers University, Piscataway, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001389</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001389" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001389" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001389" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001389" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001389-g001" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001389-g002" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001389-g003" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001389-g004" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001389-g005" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001389-g006" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001389-g007" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001389">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001389" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>DNA and RNA polymerases use a common phosphoryl transfer mechanism for base addition that requires two or three acidic amino acid residues at their active sites. We previously showed, for the reverse transcriptase (RT) encoded by the yeast retrotransposon Ty1, that one of the three conserved active site aspartates (D<sub>211</sub>) can be substituted by asparagine and still retain <em>in vitro</em> polymerase activity, although <em>in vivo</em> transposition is lost. Transposition is partially restored by second site suppressor mutations in the RNAse H domain. The novel properties of this amino acid substitution led us to express the WT and D<sub>211</sub>N mutant enzymes, and study their pre-steady state kinetic parameters. We found that the <em>k<sub>pol</sub></em> was reduced by a factor of 223 in the mutant, although the <em>K<sub>d</sub></em> for nucleotide binding was unaltered. Further, the mutant enzyme had a marked preference for Mn<sup>2+</sup> over Mg<sup>2+</sup>. To better understand the functions of this residue within the Ty1 RT active site, we have now examined the <em>in vitro</em> properties of WT and D<sub>211</sub>N mutant Ty1 RTs in carrying out pyrophosphorolysis, the reverse reaction to polymerization, where pyrophosphate is the substrate and dNTPs are the product. We find that pyrophosphorolysis is efficient only when the base-paired primer template region is &gt;14 bases, and that activity increases when the primer end is blunt-ended or recessed by only a few bases. Using pre-steady state kinetic analysis, we find that the rate of pyrophosphorolysis (<em>k<sub>pyro</sub></em>) in the D<sub>211</sub>N mutant is nearly 320 fold lower than the WT enzyme, and that the mutant enzyme has an ~170 fold lower apparent <em>K<sub>d</sub></em> for pyrophosphate. These findings indicate that subtle substrate differences can strongly affect the enzyme's ability to properly position the primer-end to carry out pyrophosphorolysis. Further the kinetic data suggests that the D<sub>211</sub> residue has a role in pyrophosphate binding and release, which could affect polymerase translocation, and help explain the D<sub>211</sub>N mutant's transposition defect.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Pandey M, Patel SS, Gabriel A (2008) Kinetic Pathway of Pyrophosphorolysis by a Retrotransposon Reverse Transcriptase. PLoS ONE 3(1):
          e1389.
            doi:10.1371/journal.pone.0001389</p><p><strong>Academic Editor: </strong>Steve Sandler, University of Massachusetts, United States of America</p><p><strong>Received:</strong> August 3, 2007; <strong>Accepted:</strong> December 2, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Pandey et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was supported, in part, from PHS grants GM60534 (to AG) and GM55310 and GM51966 (to SP). The sponsors or funders had no role in the design or conduct of the study, nor in the collection, analysis, or interpretation of the data, nor in the preparation, review or approval on the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Template dependent nucleotide polymerases use a remarkably well conserved two metal ion dependent phosphoryl transferase mechanism for nucleotide addition. Templates, primer-ends and nucleotides are positioned at the active site, the 3′ hydroxyl of the primer-end attacks the α-phosphate of the incoming dNTP, pyrophosphate (PPi) is released and the primer-end translocates to allow for the addition of the next nucleotide <a href="#pone.0001389-Steitz1">[1]</a>. A conserved structural feature of polymerase active sites is the presence of two or three acidic residues (depending on the class of polymerases) which coordinate the positioning of the metal ions <a href="#pone.0001389-Delarue1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Reverse transcriptases (RTs) are a class of RNA or DNA- dependentent DNA polymerases first discovered within retroviral particles <a href="#pone.0001389-Baltimore1">[3]</a>, <a href="#pone.0001389-Temin1">[4]</a>. They are essential for retroviral replication, and copy the genomic RNA into double stranded DNA that can then integrate into a new genomic location<a href="#pone.0001389-Gilboa1">[5]</a>. RTs, or the genes encoding them, have subsequently been discovered in a wide range of endogenous retrotransposons and in some infectious DNA viruses, where they are used for element replication, as well as in telomerases, that use a short RNA template to create terminal DNA repeats at chromosome ends. All RTs contain three conservatively spaced aspartates, two of which form the canonical “YXDD” sequence <a href="#pone.0001389-Skalka1">[6]</a>, <a href="#pone.0001389-Lingner1">[7]</a>. Early studies with retroviral RTs showed that mutation of any of these aspartates severely disrupts polymerase function <em>in vitro</em>, and eliminates the biological activity of the affected enzyme <a href="#pone.0001389-Larder1">[8]</a>, <a href="#pone.0001389-LeGrice1">[9]</a>. In previous studies of the RT encoded by the yeast retrotransposon Ty1, we found that a substitution of the second D in the YXDD box (D<sub>211</sub>N), had surprisingly little effect on the <em>in vitro</em> ability of the enzyme to add nucleotides to homopolymer substrates, although a D<sub>211</sub>E substitution as well as substitution of either of the other two aspartates (D<sub>129</sub> and D<sub>210</sub>) completely blocked polymerization <a href="#pone.0001389-Uzun1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>While the D<sub>211</sub>N mutation completely inhibits transposition, we also identified second site suppressor mutations in the RNAse H domain of Ty1 RT, which restored 5–10% of the D<sub>211</sub>N mutant's transposition capability <a href="#pone.0001389-Uzun1">[10]</a>. Although this was the first reported case of a retrotransposon with apparently loosened requirements for the acidic residues in the active site, there is now a report that a different, unrelated retrotransposon RT from yeast (Ty3) has similar if not even more extreme flexibility at these sites <a href="#pone.0001389-Bibillo1">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Because of the unusual properties of the mutant Ty1 RT, we undertook a pre-steady state analysis of single base incorporation to compare the kinetic parameters of the two enzymes, determine the nature of the defect in the D<sub>211</sub>N mutant, and to better understand the role of this residue in polymerization and transposition. Our results indicated that the rate constant for single nucleotide addition (<em>k<sub>pol</sub></em>) was reduced 223 fold for the mutant enzyme relative to WT, but that nucleotide binding was unaffected. Each enzyme showed distinctive patterns of pausing during polymerization, suggesting differences in translocation or PPi removal for the two enzymes. Further, the two enzymes showed marked differences in utilization of Mg<sup>2+</sup> versus Mn<sup>2+</sup> , suggesting a role for the D<sub>211</sub> side chain in binding the metal involved in nucleophilic attack <a href="#pone.0001389-Pandey1">[12]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Pyrophosphorolysis is the reverse of the polymerization reaction, in which the terminal base at a primer-end is excised in the presence of a properly positioned PPi group, generating a dNTP and a primer one base shorter than its initial length. Pyrophosphorolysis has been useful in studying the events occurring at the polymerase active site. In particular it has become apparent that excision of incorporated nucleotide analogue chain-terminators is an important drug resistance mechanism utilized <em>in vivo</em> by mutant versions of HIV1 RT, and possibly other viral polymerases <a href="#pone.0001389-Urban1">[13]</a>–<a href="#pone.0001389-Goldschmidt1">[15]</a>. This process is mechanistically analogous to pyrophosphorolysis, except that ATP is the PPi donor <a href="#pone.0001389-Meyer1">[16]</a>, <a href="#pone.0001389-Meyer2">[17]</a>. Analysis of this phenomenon, and the effects of various analogs and mutations in HIV1 RT on the relative efficiency of this process, has helped to explicate the events at the active site that result in pyrophosphorolysis and the related phenomenon of translocation and processive synthesis <a href="#pone.0001389-Sarafianos1">[18]</a>, <a href="#pone.0001389-Sarafianos2">[19]</a>. The primer-end in the active site of a polymerase can be present in one of two positions, referred to as the P and N sites. With the primer-end in the P site, an incoming nucleotide can bind to the N site, and chemistry can occur between the 3′ hydroxyl and the alpha phosphate of the dNTP. Post-chemistry, the primer-end resides in the N site, which also houses the PPi group. The chemical step at the active site corresponds to the closing down of a finger domain over the active site, leading to a “closed” conformation <a href="#pone.0001389-Huang1">[20]</a>. Translocation then involves the movement of the primer-end from the N site back to the P site, accompanied by release of the PPi from the N site pocket. This corresponds to a switch back to an “open” conformation. An important insight from the work on HIV-1 nucleotide analog resistance mutants is that the equilibrium between the primer-end at the P site versus the N site is an important determinant in the ability to carry out excision, since this reaction can only occur if the primer-end is positioned at the N site, which may not be a favorable placement.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Given the unusual observations regarding the biochemical properties of the D<sub>211</sub>N mutant in polymerization, and the importance of primer-end placement in the active site, we set out to determine how the substitution would affect the process of pyrophosphorolysis. Previous studies on HIV1 RT and Ty3 RT have shown that certain mutations do not necessarily affect polymerization and pyrophosphorolysis equally, suggesting that binding or positioning of the substrates in one direction or the other can be differentially affected <a href="#pone.0001389-Bibillo1">[11]</a>, <a href="#pone.0001389-Kaushik1">[21]</a>. Since this could give further insights into the functions of the D<sub>211</sub> residue, we carried out a series of experiments to characterize determinants of pyrophosphorolysis for the WT and mutant enzyme, using either Mg<sup>2+</sup> or Mn<sup>2+ </sup>as the divalent cation. We then measured the apparent <em>K<sub>d</sub></em> for PPi and the <em>k<sub>pyro</sub></em> using pre-steady state kinetics. This study of pyrophosphorolysis, along with our previous examination of polymerization, form the frame work for future understanding of the mechanistic basis of the activity of the Ty1 RT second site suppressor mutants.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>D<sub>211 </sub>N mutant RT enzyme is defective in pyrophosphorolysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We initially compared the time course of pyrophosphorolysis (0–3600 sec) for the two enzymes using our previous <sup>32</sup>P-labeled 14-mer/28-mer DNA/DNA substrate <a href="#pone.0001389-Pandey1">[12]</a> (RAG*998/928, <a href="#pone-0001389-g001">Fig. 1</a>) in the presence of 1 mM sodium pyrophosphate and 10 mM Mg<sup>2+</sup>. The behavior of the two enzymes was quite distinct under substrate excess conditions (<a href="#pone-0001389-g002">Fig. 2</a>). The WT enzyme was able to excise a base from ~10% of the labeled primer within 60 seconds. The fraction of excised substrate increased to ~17% over the time course (<a href="#pone-0001389-g002">Fig. 2A</a>). Time points beyond 120 seconds also showed base addition. Under the same conditions, pyrophosphorolysis with the D<sub>211</sub>N RT was insignificant, amounting to &lt;1% of the total substrate at the longest time point (<a href="#pone-0001389-g002">Fig. 2B</a>). We did not observe any base addition with the mutant enzyme either. Since there are no nucleoside triphosphates initially present in the in vitro reaction, the presence of base addition in WT RT reactions (<a href="#pone-0001389-g002">Fig. 2A</a>) suggested that nucleoside triphosphates were being generated by pyrophosphorolysis of the unlabeled strand of the double stranded substrate, and subsequently used for polymerization. This is consistent with the observation that the excised base from the unlabeled strand (dATP) is complementary to the next base beyond the primer, for this primer/template combination.</p>
<div class="figure" id="pone-0001389-g001"><div class="img"><a name="pone-0001389-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>DNA sequences used in this study.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Sequence shows the PPT region of Ty1. All the oligonucleotides (RAGs) used in this study correspond either to the plus or minus strand of this sequence, depending on the direction of the arrow. The relative positions of the oligonucleotides and modifications are shown in this figure.</p>
<span>doi:10.1371/journal.pone.0001389.g001</span></div><div class="figure" id="pone-0001389-g002"><div class="img"><a name="pone-0001389-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Pyrophosphorolysis by WT and D<sub>211</sub>N Ty1 RT of a recessed primer terminus.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Pyrophosphorolysis reactions by WT or mutant D<sub>211</sub>N Ty1 RT in the presence of 10 mM Mg<sup>2+ </sup>and 1 mM sodium pyrophosphate were analyzed by <em>in vitro</em> assay as described under <a href="#s4">materials and methods</a> using 5′ <sup>32</sup>P-end labeled 14-mer/28-mer substrate (RAG*998/RAG 928, <a href="#pone-0001389-g001">Fig. 1</a>. Labeled end is depicted by *). A, represents time course of WT reactions in seconds with or without PPi. B represents time course of D<sub>211</sub>N reactions in seconds with or without PPi. Control reactions in both panels are in the absence of Mg<sup>2+ </sup>or enzyme.</p>
<span>doi:10.1371/journal.pone.0001389.g002</span></div>

<h4>Pyrophosphorolysis on the 5′ blunt end</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To determine whether pyrophosphorolysis is simultaneously occurring at the other end of the 14-mer/28-mer DNA/DNA substrate, we labeled the 5′ end of the 28-mer oligonucleotide with <sup>32</sup>P, instead of the 14-mer, and repeated the time course reactions with this new substrate (RAG 998/*928, <a href="#pone-0001389-g001">Fig. 1</a>) under the same conditions. Unexpectedly, WT reactions were much more extensive on this end of the substrate, with ~4.2% of the total substrate excised by 10 sec. By 900 seconds almost 50% of the substrate had at least one base excised. By the end of the time course, a substantial portion of the substrate had been reduced by 8 bases (<a href="#pone-0001389-g003">Fig. 3A</a>). The large amount of dNTPs generated by pyrophosphorolysis of the blunt end could support the forward reaction observed in <a href="#pone-0001389-g002">Fig. 2A</a>. The D<sub>211</sub>N mutant also shows improved pyrophosphorolysis (~14% of the total substrate by 3600 seconds) but was not nearly as robust as the WT enzyme (<a href="#pone-0001389-g003">Fig. 3B</a>).</p>
<div class="figure" id="pone-0001389-g003"><div class="img"><a name="pone-0001389-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Pyrophosphorolysis by WT and D<sub>211</sub>N Ty1 RT of a blunt-ended primer terminus.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Pyrophosphorolysis reactions of WT or mutant D<sub>211</sub>N Ty1 RT were carried out under same conditions as <a href="#pone-0001389-g002">Fig. 2</a> in the presence of 10 mM Mg<sup>2+</sup>, using 5′ <sup>32</sup>P-end labeled 28-mer/14-mer substrate (RAG*928/998, <a href="#pone-0001389-g001">Fig. 1</a>). A, represents time course of WT reactions in seconds in the presence or absence of PPi. B, represents time course of D<sub>211</sub>N reactions with or without PPi. Control reactions are in the absence of Mg<sup>2+ </sup>or enzyme.</p>
<span>doi:10.1371/journal.pone.0001389.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The apparent preferential pyrophosphorolysis at the blunt end, and the generation of polymerization substrates made examination of the excision reaction at the recessed end too complex. To simplify, we modified the original substrate by introducing a mismatched guanosine at the 3′ end of the 28-mer template (resulting in RAG*998/1109, <a href="#pone-0001389-g001">Fig. 1</a>). In the presence of PPi, the WT enzyme was able to excise a base from the labeled primer without showing any forward reaction products (<a href="#pone.0001389.s001">Fig. S1A</a>). Therefore, a mismatch at the blunt end blocks base addition at the recessed end, presumably by inhibiting pyrophosphorolysis at the mismatched blunt end. The D<sub>211</sub>N mutant reactions were very poor and no pyrophosphorolysis activity was observed even at 3600 seconds (<a href="#pone.0001389.s001">Fig. S1B</a>).</p>


<h4>Primer length effects</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The preceding analysis indicated that the recessed primer end of the 14-mer/28-mer or 28′-mer substrates (RAG*998/928, RAG*998/1109, <a href="#pone-0001389-g001">Fig. 1</a>, where “ <strong>’</strong> ” refers to the terminal mismatch) is a poor substrate for pyrophosphorolysis. To determine the effects of altering substrate lengths and/or the degree of primer end recessing, we used a variety of substrates (<a href="#pone-0001389-g004">Fig. 4</a>) that differed only by the length of the 5′-end <sup>32</sup>P-labeled primer, as a 14-mer, 20-mer, 24-mer, 27-mer or 28-mer (RAGs 998, 1250,1249, 1248 and 952 respectively as in <a href="#pone-0001389-g001">Fig. 1</a>). Primers were annealed to the same terminally mismatched 28′-mer template. Further, we carried out the 60-minute reactions using either Mg<sup>2+</sup> or Mn<sup>2+</sup> as the divalent metal. For both WT and D<sub>211</sub>N mutant enzymes (<a href="#pone-0001389-g004">Fig. 4A &amp; B</a>) pyrophosphorolysis activity significantly improved with the increase in length beyond the 14-mer in the presence of Mg<sup>2+</sup>. In particular, the WT enzyme was able to excise &gt;30% of the 24-mer and 28-mer substrates (<a href="#pone-0001389-g004">Fig. 4C</a>). The D<sub>211</sub>N mutant enzyme pyrophosphorolysis activity also improved proportionately with longer primers compared to the 14-mer primer but was still defective in comparison to WT (<a href="#pone-0001389-g004">Fig. 4C and 4D</a>).</p>
<div class="figure" id="pone-0001389-g004"><div class="img"><a name="pone-0001389-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Effect of primer length on pyrophosphorolysis.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Pyrophosphorolysis by WT or mutant D<sub>211</sub>N Ty1 RT were carried out for 60 minutes in the presence of either 10 mM Mg<sup>2+</sup> or 2 mM Mn<sup>2+</sup>. Substrates are 5′ <sup>32</sup>P-end labeled primers of lengths 14, 20, 24, 27, 28-mer (RAG 998, RAG 1250, RAG 1249, RAG 1248, RAG 952 respectively, <a href="#pone-0001389-g001">Fig. 1</a>), each paired with the 28′-mer template oligo (RAG 1109, <a href="#pone-0001389-g001">Fig. 1</a>). A, shows WT reactions with different primer length substrates in the presence or absence of PPi. B, represents D<sub>211</sub>N reactions with or without PPi. Control reactions on both the panels are in the absence of enzyme or Metal <sup>2+</sup>. C and D represent bar graphs for WT and DN Ty1 RT reactions depicting ratios of intensities of cleaved products to the total products in reactions (I cleaved products/I total bands). Black bars are for reactions in Mg<sup>2+</sup> while grey bars are for reactions in Mn<sup>2+</sup>.</p>
<span>doi:10.1371/journal.pone.0001389.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As previously reported <a href="#pone.0001389-Pandey1">[12]</a>, <a href="#pone.0001389-Bolton1">[22]</a>, for polymerization, the WT enzyme has a marked Mg<sup>2+</sup> preference, and is inhibited by Mn<sup>2+</sup>, whereas the mutant enzyme prefers Mn<sup>2+</sup> to Mg<sup>2+.</sup> In the presence of Mg<sup>2+</sup>, the WT enzyme excised a larger percentage of substrates than in Mn<sup>2+</sup>(<a href="#pone-0001389-g004">Fig. 4C</a>). The D<sub>211</sub>N mutant enzyme showed no clear preference under these conditions (<a href="#pone-0001389-g004">Fig. 4D</a>). For both enzymes, the patterns of excision differ in the presence of the different metals, with a greater number of bases removed from any given excised substrate in the presence of Mg<sup>2+</sup>(<a href="#pone-0001389-g004">Fig. 4A and B</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>These results demonstrate that substrates with double stranded primer/template regions greater than 14 bps are better substrates for the Ty1 RT pyrophosphorolysis reaction. Of note, during processive excision, as seen in <a href="#pone-0001389-g004">Fig. 4A</a>, most excised products remained longer than 14-bases long, and a 14 base primer appears to be a limit to excision. Increased primer length is not the only factor associated with increased pyrophosphorolysis activity, however, since the 24-mer was the most active, a blunt 28-mer was next most active and a 20-mer was more active than the 27-mer. The Mn<sup>2+</sup> results for WT mirrored those with Mg<sup>2+</sup>, suggesting that metal ion was not the determining factor in relative activity on different substrates.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>We further explored the influence of different substrate arrangements on pyrophosphorolysis (<a href="#pone.0001389.s002">Fig. S2</a>). These comparisons demonstrated that the poor reaction of the original 14 mer/28 mer is not due to the length of the recessed end, since a substrate with the same recessed 14 bases, but a double stranded region extended by an additional 11 bases underwent robust pyrophosphorolysis.</p>


<h4>Pyrophosphorolysis processivity</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In our previous study <a href="#pone.0001389-Pandey1">[12]</a> we found that both the WT and D<sub>211</sub>N mutant were highly processive for the polymerase reaction. Since we observed multiple excision products, we asked whether excision of multiple bases was processive or distributive. The reactions were carried out in the presence or absence of a trap in excess to the labeled substrate, and in the presence of Mg<sup>2+</sup> or Mn<sup>2+</sup>. Trap effectiveness controls and control reactions without enzyme or PPi or divalent ions were also conducted. <a href="#pone-0001389-g005">Fig. 5A</a> shows that either in the presence or absence of trap, and in the presence of metal, the WT enzyme can excise multiple bases. This indicates that pyrophosphorolysis is processive in the presence of either divalent metal ion. The D<sub>211</sub>N enzyme was similarly insensitive to the presence of the trap (<a href="#pone-0001389-g005">Fig. 5B</a>) indicating that the D<sub>211</sub>N RT is also processive for pyrophosphorolysis in the presence of either divalent metal ion.</p>
<div class="figure" id="pone-0001389-g005"><div class="img"><a name="pone-0001389-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Processivity of the Ty1 pyrophosphorolysis.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Pyrophosphorolysis by WT or mutant D<sub>211</sub>N Ty1 RT were carried out for 60 minutes in the presence or the absence of trap and in the presence of either 10 mM Mg<sup>2+</sup> or 2 mM Mn<sup>2+</sup>. Substrates are 5′ <sup>32</sup>P-end labeled substrate (RAG*1249/1109, <a href="#pone-0001389-g001">Fig. 1</a>). A, shows WT reactions in the presence or absence of 200 fold excess of cold 24-mer/28′-mer DNA/DNA substrate trap and with or without PPi (depicted as + and – respectively). B, similarly represents D<sub>211</sub>N reactions. Control reactions on both the panels are in the absence of enzymes. Trap effectiveness controls are also shown.</p>
<span>doi:10.1371/journal.pone.0001389.g005</span></div>

<h4>Pre-steady-state kinetics of pyrophosphorolysis for WT and mutant D<sub>211 </sub>N Ty1 RT</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Our steady state studies showed that the D<sub>211</sub>N mutant enzyme was much less proficient at pyrophosphorolysis, but did not provide a quantitative analysis of the kinetic differences. To examine this, we turned to pre-steady state single base excision kinetic analysis. Earlier, we had shown that single base addition onto the 14-mer/28-mer RAG 998/928 substrate was ~223 fold slower for the mutant enzyme than the WT RT, but that dNTP binding was only minimally affected <a href="#pone.0001389-Pandey1">[12]</a>. To measure the reverse reaction of polymerization, we generated the corresponding 15-mer/28′-mer reverse substrate, RAG*1316/1109 (<a href="#pone-0001389-g001">Fig. 1</a>), which was comparable to the original 14-mer/28-mer but had the next base (A) added to the primer-end. The only difference between our previous polymerization substrate and our current pyrophosphorolysis substrate was a single base mispair at the opposite end to eliminate pyrophosphorolysis of the blunt end substrate. As this change is not involved in the measured reaction, and is distant from the substrate at the active site, it should not affect the derived kinetic parameters for either polymerization or pyrophosphorolysis.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We first compared pyrophosphorolysis with the RAG*998/1109 (14-mer/28′-mer) and RAG*1316/1109 (15-mer/28′-mer) substrates using WT Ty1 RT, under the enzyme excess conditions to be used for the pre-steady state studies. Whereas, only 6.8% of the 14-mer primer was excised by at least one base, 50.1% of the 15-mer primer was excised in the presence of 1 mM PPi during a 30 minute reaction (data not shown). Therefore, this 15-mer/28′-mer substrate could be used to compare with our previous single correct base incorporation kinetics. Here, we are interested in determining the parameters of Ty1 RT pyrophosphorolysis, <em>k<sub>pyro</sub></em> and <em>K<sub>d</sub></em> for PPi.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The 5′ end-labeled 15-mer primer was annealed to a 28′-mer template, mixed with RT and varying concentrations of sodium pyrophosphate and reactions were initiated by the addition of Mg<sup>2+</sup>. Reactions were quenched at various times and products were visualized and quantitated. The exponential rate constants were derived by fitting the data to equation 1, as shown by the solid lines in <a href="#pone-0001389-g006">Fig. 6</a> A &amp; B. Note that the concentration of excision products (i.e. the amplitude) in both <a href="#pone-0001389-g006">Fig. 6A and 6B</a> increases as a function of the initial PPi concentration, indicating that the reaction has come to equilibrium at the active site. This phenomenon represents the balance between the thermodynamically unfavorable pyrophosphorolyis reaction and the much more favorable polymerization reaction that uses the dNTP generated at the active site. As the concentration of PPi increases, the equilibrium is shifted towards pyrophosphorolysis.</p>
<div class="figure" id="pone-0001389-g006"><div class="img"><a name="pone-0001389-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Concentration dependence of the rate of pyrophosphorolysis by WT and D<sub>211</sub>N Ty1 RT.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A and B show time courses of a single turnover at various sodium pyrophosphate concentrations: 0.6 mM (closed circle), 1.25 mM (open circle), 2.5 mM (closed inverted triangle), 5 mM (open triangle) and 10 mM (closed square) for WT (A) and 0.02 mM (open square), 0.08 mM (closed square), 0.3 mM (open triangle), 1.25 mM (closed inverted triangle), 2.5 mM (open circle) and 5 mM (closed circle) for the D<sub>211</sub>N enzyme (B) respectively. A 5′ <sup>32</sup>P-end labeled substrate 15-mer/28′-mer (RAG*1316/1109, <a href="#pone-0001389-g001">Fig. 1</a>) was used for the experiment. For WT, samples were taken at 0, 0.1, 0.2, 0.4, 0.6, 0.8, 1, 1.5, 2, 3, 4, 6, 8, 10, 20, 30, 60 second time points. For the mutant, samples were taken at 5 or 10 second intervals for the first minute and then every minute for the next 4 minutes. C and D show pre-steady-state rate dependence of nucleotide cleavage on PPi concentration for WT and mutant respectively, based on the rates derived from Equation 1 (see Experimental Procedures) in A and B. Curves represent the best fits to Equation 2 (see Experimental Procedures).</p>
<span>doi:10.1371/journal.pone.0001389.g006</span></div><a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>The derived exponential rate constants were plotted against their respective PPi concentrations and the curve was fitted to a hyperbolic equation (Equation 2 and <a href="#pone-0001389-g006">Fig. 6C &amp; D</a><em>)</em> from which <em>K<sub>d</sub></em> and <em>k<sub>pyro</sub></em> values were derived. In deriving <em>K<sub>d</sub></em>, we assumed that PPi is in rapid equilibrium with <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.e001&amp;representation=PNG" class="inline-graphic"></span>. In this way, we found apparent <em>K<sub>d</sub></em> values of 6.52±2.43 mM and 0.039±0.007 mM and <em>k<sub>pyro</sub></em> values 0.93±0.15 sec<sup>−1</sup> and 0.0029±0.0001 sec<sup>−1</sup> respectively for the WT and mutant D<sub>211</sub>N enzymes. These correspond to an ~170 fold greater binding affinity of PPi for the mutant enzyme, and an ~320 fold decrease in the rate of pyrophosphorolysis for this enzyme. The derived catalytic efficiency was only ~2 fold lower for the mutant RT enzyme compared to the WT enzyme. In our previous study, using similar approaches for the forward polymerization reaction, we found apparent <em>K<sub>d</sub></em> values for dATP of 2.4±0.4microM and 1.0±0.04 microM and <em>k<sub>pol</sub></em> values of 7.8±0.4 sec<sup>−1</sup> and 0.035±0.002 sec<sup>−1</sup> respectively for the WT and mutant D<sub>211</sub>N enzymes. These values indicate a similar binding affinity of dATP for the two enzymes, but an ~100 fold lower catalytic efficiency for the mutant enzyme. Thus while both the forward and reverse reactions are significantly slower for the mutant enzyme than the WT enzyme, the differences in binding affinity and catalytic efficiency for the two substrates imply that the mutation at D<sub>211</sub> affects different steps in the forward and reverse reactions and can have a strong influence on the internal equilibrium between these two competing reactions.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study we have examined the determinants of pyrophosphorolysis of a retrotransposon reverse transcriptase. By carrying out a series of reactions using different, but related, partially double stranded substrates we have observed several factors that influence the ability of the enzyme to remove terminal primer bases in the presence of PPi. Our initial substrate (a 14 base recessed 3′ end plus a 14 base double stranded region) was only poorly excised and only a single base was generally removed. Either reduction of the recessed end or an increase in the double stranded primer/template region increased pyrophosphorolysis and made it more processive. In particular, the same 14 base recessed end, with a double stranded region extended to 24 bases, had a much greater capacity for pyrophosphorolysis than the original substrate (<a href="#pone-0001389-g004">Fig. 4</a> and <a href="#pone.0001389.s002">Fig. S2</a>). In our earlier study <a href="#pone.0001389-Pandey1">[12]</a> we examined dissociation of the enzyme from the 14/28 primer/template, and found a much higher affinity for Ty1 RT (0.06 min<sup>−1</sup>) than for a retroviral RT that we measured concurrently (5.5 min<sup>−1</sup>). This suggests that the limited pyrophosphorolysis observed for the smaller substrate is not due to poor binding. Instead it is more likely that the 14/28 substrate assumes a conformation at the polymerase active site incompatible with pyrophosphorolysis. Even a single additional primer strand base greatly improved pyrophosphorolysis. Studies of the elongation complex of <em>T. thermophilus</em> RNA polymerase have shown that a single base pair addition or subtraction to the primer/template complex has marked consequences on the complex conformation and its ability to support pyrophosphorolysis <a href="#pone.0001389-Kashkina1">[23]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>As shown in <a href="#pone-0001389-g007">Fig. 7</a> (and discussed below), while the primer 3′ end at the P site can lead to polymerization, the primer-end must be at the N site for pyrophosphorolysis to occur, and this may be disfavored with short substrates. Interestingly, even with processive pyrophosphorolysis of longer primers, the reaction appeared to stop beyond a primer length of 14, further indicating an impediment to continued reaction past this substrate. The RNAse H active site for Ty1 RT has been mapped to between 14 and 15 bases from the 3′ end of the primer terminus in the polymerase active site (as opposed to 18 bases in retroviruses) <a href="#pone.0001389-Wilhelm1">[24]</a>, <a href="#pone.0001389-Wilhelm2">[25]</a>. Given our previous genetic results on interactions between the Ty1 RT polymerase and RNAse H active sites <a href="#pone.0001389-Uzun1">[10]</a>, it is plausible that suboptimal positioning of the substrate at the RNAse H active site could influence the conformation at the polymerase active site in ways that disfavor pyrophosphorolysis. For example, we found that while blunt ends were very good substrates for processive pyrophosphorolysis (<a href="#pone-0001389-g003">Fig. 3</a> and <a href="#pone.0001389.s001">Figs. S1</a> and <a href="#pone.0001389.s002">S2</a>), a terminal mismatch or a 3′ extension eliminated the activity. In each case, the primer-end needs to be base paired to the template strand to be in a proper conformation. With a blunt end, there is no extended single stranded template sequence to help position the primer terminus at the P site, and this could facilitate translocation to the N site and subsequent pyrophosphorolysis (<a href="#pone-0001389-g007">Fig. 7</a>).</p>
<div class="figure" id="pone-0001389-g007"><div class="img"><a name="pone-0001389-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001389" data-uri="info:doi/10.1371/journal.pone.0001389.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001389.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001389.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001389.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Scheme for phosphoryl transfer reaction by polymerases, from the perspective of pyrophosphorolysis.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>The polymerase active site contains both a P site (blue circle) in which the 3′ primer-end resides during polymerization, and an N site (pink circle) in which the templated dNTP resides during polymerization. Translocation is required for processive polymerization or pyrophosphorolysis and refers to movement of the 3′ primer-end between the P and N sites (cartoons 1–4). For pyrophosphorolysis to occur, both the 3′ primer-end and a PPi molecule must be correctly positioned in the N site (cartoon 4). It is not known how binding of PPi affects the equilibrium of the 3′ primer-end between the P and N sites (cartoon 2 vs. 4), or whether PPi-binding affinities are affected by the location of the 3′ primer-end (cartoons 1 to 2 versus 3 to 4). Once a ternary complex forms between enzyme, primer template and PPi (cartoon 4), a conformational change is postulated (cartoon 5) that favors the chemical step of nucleophilic attack by the PPi of the 5′ phosphodiester bond in the terminal templated nucleoside monophosphate, and the breakage of the phosphodiester bond to form a dNTP and a new 3′ OH in the P site (cartoon 6 ). Subsequent conformational change in the enzyme (cartoon 7) allows removal of the dNTP and subsequent translocation of the new 3′ primer-end from the P site to the N site (cartoons 1–4).</p>
<span>doi:10.1371/journal.pone.0001389.g007</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We speculate that pyrophosphorolysis takes place by at least a 3 step mechanism (<a href="#pone-0001389-g007">Fig. 7</a>). First, both PPi and the 3′ primer-end form a complex at the N site (structures 1–4). Second, a conformational change takes place that brings the primer-end in close proximity with PPi (structure 5). This change from an “open” to a “closed” conformation is analogous to the pre-chemical isomerization steps during polymerization observed for HIV-1 RT, DNA polymerase β and other DNA and RNA polymerases <a href="#pone.0001389-Sarafianos2">[19]</a>, <a href="#pone.0001389-Beard1">[26]</a>-<a href="#pone.0001389-Arnold1">[28]</a>. Third, the phosphoryl transfer reaction occurs to generate a dNTP at the N site (structure 6). This would need to be followed by release of the dNTP (structures 7 and 1) for any additional PPi to enter the N site.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>By carrying out pre-steady state kinetic analysis of the two enzymes, we have identified a source of difference between the WT and mutant enzyme that could contribute to their <em>in vitro</em> and <em>in vivo</em> behaviors. The WT Ty1 RT has an apparent <em>K<sub>d</sub></em> for PPi of ~7 mM, similar to the calculated <em>K<sub>d</sub></em> s of several other polymerases examined by related methods <a href="#pone.0001389-Arion1">[29]</a>–<a href="#pone.0001389-Anand1">[33]</a>. This millimolar range <em>K<sub>d</sub></em> means that under physiological conditions it is unlikely that PPi is bound at the active site, relative to dNTPs, which have <em>K<sub>d</sub></em>s in the micromolar range. Therefore it was unexpected that the D<sub>211</sub>N mutant had an apparent <em>K<sub>d</sub></em> for PPi in the micromolar range, lower than any other reported polymerase. Since we are determining <em>K<sub>d</sub></em> for PPi indirectly, by measuring product formation as a function of PPi concentration, one reason for the lower PPi <em>K</em><sub>d</sub> of D211N mutant is its slower chemical step relative to that of the WT enzyme. The <em>k<sub>pyro</sub></em> of the mutant enzyme is ~320 fold slower than that of the WT. Due to the slower chemical step, PPi bound enzyme intermediates accumulate in the mutant enzyme reaction, but not in the WT. Thus, the apparent <em>K<sub>d</sub></em> in the case of the mutant would correspond to the equilibrium among structures 1 through 4, which reflects the true binding affinity. In other words, the PPi <em>K</em><sub>d</sub> of the mutant is more close to the “true” <em>K</em><sub>d</sub> and the PPi <em>K</em><sub>d</sub> of the WT is close to its <em>K</em><sub>m</sub>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The difference in the apparent <em>K<sub>d</sub></em> for PPi for WT versus D<sub>211</sub>N mutant RTs, despite their similar <em>K<sub>d</sub></em>s for dNTP, could relate to the potential distortions of the active site architecture in the mutant enzyme due to the D<sub>211</sub> to N<sub>211</sub> substitution, its effect on metal binding, and its effect on PPi binding. Our previous work, as well as crystallographic studies of HIV-1 RT, suggests that the D<sub>211</sub> side chain is more likely involved in the chemical steps and the coordination of the “A site” metal ion, than in the “B site” metal ion which is involved in binding the triphosphates of the dNTP, and presumably, the PPi after chemistry. Modeling studies of the DNA polymerase β active site based on recent high resolution structures suggest that during formation of the transition state, the proton from the primer 3′ hydroxyl is transferred to form a distorted H bond between two aspartates, equivalent to our D<sub>211</sub> and D<sub>210</sub>, and that this interaction could drive subsequent release of both PPi and the catalytic Mg<sup>2+</sup> <a href="#pone.0001389-Batra1">[34]</a>, <a href="#pone.0001389-Lin1">[35]</a>. This implies that a relay of interactions links D<sub>211</sub> and PPi and that alteration of that relay by N<sub>211</sub> in the mutant enzyme could affect the apparent affinity of PPi for the enzyme.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The fate of PPi at the active site has become a major focus of attention in models for translocation of polymerases along a template. These models have been made for RNA polymerases in particular, but the basic concepts should apply to all polymerases. One model, referred to as the “power stroke” mechanism, posits that the energy of cleavage of the dNTP results in a change in active site conformation that favors translocation <a href="#pone.0001389-Jiang1">[36]</a>, <a href="#pone.0001389-Patel2">[37]</a>. Yin and Steitz propose for T7 RNA polymerase, that the release of PPi produces the conformational change resulting in translocation <a href="#pone.0001389-Yin2">[38]</a>. Sarafianos <em>et al</em>. propose that movement of the negatively charged loop formed by the HIV1 RT equivalents of D<sub>210</sub> and D<sub>211</sub>, observed pre- and post- translocation, provides a “springboard” for translocation <a href="#pone.0001389-Sarafianos2">[19]</a>. A second model, termed “Brownian-ratchet motion” proposes that the primer-end moves back and forth between P and N sites, and that addition of dNTP serve to ratchet the primer-end in the P site where polymerization can occur <a href="#pone.0001389-Guajardo1">[39]</a>, <a href="#pone.0001389-Abbondanzieri1">[40]</a>. In either case, the apparent tighter binding of PPi to the active site of the D<sub>211</sub>N mutant could push the equilibrium more toward the reverse reaction, and impede processive synthesis <em>in vivo</em>. Further, since PPi and dNTP reside at the same binding site, PPi might more effectively compete with dNTP in the mutant enzyme. This could explain our previous observations about the <em>in vitro</em> polymerization processivity for the two enzymes. On short substrates the two enzymes behaved similarly, but on longer substrates the WT enzyme was much more processive in the presence of Mg<sup>2+</sup> <a href="#pone.0001389-Pandey1">[12]</a>. Accumulation of PPi during processive synthesis could have inhibited further polymerization in the mutant enzyme. Future analysis of polymerization and processivity under competitive conditions will help to better understand this relationship.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plasmids and Strains</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The plasmid p6H Ty1 IN-RT-RH (AGE 2186) contained WT Ty1 RT–RH plus a 115 amino acid contiguous C- terminal portion of Ty1 integrase fused to the N-terminus of the RT-RH domain, all preceded by 6-histidine tag and was kindly provided by Dr. F. X. Wilhelm (IBMC, Strasbourg). Construct of the analogous mutant expression plasmid strain AGE 2352, with a D<sub>211</sub>N polymerase active site mutation has been described <a href="#pone.0001389-Pandey1">[12]</a>. The WT and mutant plasmids were transformed into <em>E. coli</em> expression strain M15 containing pREP4 (from Qiagen) to generate AGE2193 and AGE2354 respectively <a href="#pone.0001389-Pandey1">[12]</a>. The two enzymes are expressed at similar levels in <em>E. coli</em>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p><em>Reagents-</em>Denaturing acrylamide gel solutions were from National Diagnostics. Molecular size markers for DNA were from New England Biolabs. The ultra pure dNTPs were from Amersham Pharmacia Biosciences, Inc. Radiolabeled nucleotides were from Perkin Elmer Life Sciences. PAGE purified DNA oligonucleotides were from Integrated DNA Technologies, Inc. Nucleotide removal kit was from Qiagen and P-30 columns were from BioRad. Sodium pyrophosphate, magnesium chloride and manganese chloride were from Fisher Scientific and other chemicals Tris-HCl, NaCl, dithiothreitol and glycerol etc. were from Sigma chemical Co.</p>


<h4>Expression and Purification of Recombinant Ty1 RT</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>WT and mutant Ty1 RTs containing hexahistidine tags were expressed in E. coli strain M15 [pREP4] (Qiagen) purified by Ni<sup>2+</sup>-nitroloacetic acid-agarose (Qiagen) affinity chromatography as described <a href="#pone.0001389-Pandey1">[12]</a>.</p>


<h4>5′-<sup>32</sup>P Labeling of oligonucleotides</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>DNA oligonucleotides were 5′ end-labeled using [γ-<sup>32</sup>P] ATP and T4 polynucleotide kinase. Unincorporated nucleotide was removed with nucleotide removal kits (Qiagen) or by P-30 spun columns (Biorad).</p>


<h4>Denaturing gels</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Reaction samples were stopped by mixing with loading buffer (95% formamide, 35 mM EDTA [pH 8], 0.1% bromophenol blue, 0.1% xylene cyanol). Samples were heated to 95°C for 5 min prior to loading 7 microliters on a 1x TBE (0.089 M Tris-HCl, 0.089 M boric acid and 0.002 M sodium EDTA buffered at pH 8.5), 7 M urea gel containing the appropriate percentage of polyacrylamide. Electrophoresis was performed in 1×TBE buffer at ~1–1.2 watts/cm. Gels were visualized with a Storm 860 PhosphorImager and quantitated using the Image Quant 1.2 software (Molecular Dynamics).</p>


<h4>Excision Assay with DNA Templates</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>In a typical assay, a DNA/DNA template/primer was prepared by annealing a 28-mer plus-strand sequence from the polypurine tract region of Ty1 RT (5′-ATT ACA TT<span class="underline">A TGG GTG GTA </span>TGT TGG AAT A -3′, where the polypurine tract is underlined) or an equivalent 28′-mer plus-stand sequence with a ‘g’ at the 3′ end, (to generate a mispaired end) with the otherwise complementary 14 to 28-mer primer whose 5′-end is <sup>32</sup>P-labeled (<a href="#pone-0001389-g001">Fig. 1</a>, RAG 998 and RAG 1109). Additional substrates used throughout this study are described in <a href="#pone-0001389-g001">Fig. 1</a>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>DNA substrate and enzyme were prepared in the reaction buffer (17 mM Tris-HCl (pH 7.5), 17 mM NaCl, 1 mM dithiothreitol and 20% glycerol) and then pre-mixed with either 10 mM MgCl<sub>2 </sub>or 2 mM MnCl<sub>2</sub> and a PPi donor (sodium pyrophosphate or NTPs) in the same buffer. The separate mixes were pre-warmed at 22°C for 10 minutes and were then combined in equal volumes to start the reactions at 22°C. Unless specified in the results and figure legends section, an assay mixture contained 40 nM template-primer, ~20 nM of active WT or mutant D<sub>211</sub>N Ty1 RT, with PPi donor and MgCl<sub>2</sub> or MnCl<sub>2</sub> at specified concentrations. Reactions were incubated at 22°C for specified times, and then terminated by the addition of loading buffer, denatured and separated by electrophoresis in 7 M urea-17% polyacrylamide gels. The amounts of the products (i.e. bands smaller than the end-labeled primer) were determined relative to total labeled products seen using a phosphorimager, after subtracting the background from the – PPi lane.</p>


<h4>Pyrophosphorolysis Processivity Assay</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The pyrophosphorolysis reactions were carried out by pre-incubating the end labeled 24-mer/28′-mer substrate (RAG*1249/RAG 1109, <a href="#pone-0001389-g001">Fig. 1</a>), (40 nM) with WT or D<sub>211</sub>N Ty1 RT in the reaction buffer for 60 min at 22°C. Reactions were initiated by adding a mixture containing 1 mM sodium pyrophosphate with 10 mM Mg<sup>2+</sup> or 2 mM Mn<sup>2+</sup> along with or without an unlabeled 8 µM of 24-mer/28′-mer DNA/DNA substrate trap (at a final concentration of 200 fold more than the labeled primer/template). Reactions were terminated after 1 hour by adding loading buffer. A trap effectiveness control was also carried out where the reaction was initiated by adding PPi with Mg<sup>2+</sup> or Mn<sup>2+ </sup>to a mix containing enzyme, trap and primer/template (where enzyme was added to the labeled substrate in the presence of trap and then pre-warmed along with the two). The terminated processivity reaction and control reactions were resolved by 17% polyacrylamide-urea denaturing gel electrophoresis and analyzed as above.</p>


<h4><em>K<sub>d</sub></em> and <em>k<sub>pyro</sub></em> determination -</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The pre-steady-state kinetic parameters for pyrophosphorolysis in the presence of sodium pyrophosphate and 20 mM Mg<sup>2+</sup> by the WT and the mutant enzymes were measured as reported by Patel et al. <a href="#pone.0001389-Patel3">[41]</a> with some modifications. Reactions were carried in enzyme excess by rapid quench analysis <a href="#pone.0001389-Johnson1">[42]</a> for WT using a KinTek RQF-3 Rapid Quench Flow apparatus, and manually for the D<sub>211</sub>N mutant. Protein concentrations for both enzymes were determined at OD <sub>280 </sub>in a 1 cm cell in the presence of 8 M urea, (extinction coefficient for the 76. 861 kDa protein = 86,390/cm M). Enzymes were used at final concentrations of 547.92 nM (WT) and 365.34 nM (mutant), assuming that the enzyme is a monomer (F.X. Wilhelm, unpublished results) which were equivalent to active concentrations of 40.46 nM for WT and 40.10 nM for D<sub>211</sub>N as determined by 3 minute extension reactions at 200 micromolar dATP <a href="#pone.0001389-Pandey1">[12]</a>. DNA/DNA template/primer substrate was prepared by annealing a 28′-mer plus-stand sequence from the polypurine tract region of Ty1 RT (RAG 1109) with a complementary 15-mer (RAG 1316) at a template/primer ratio is 0.85/1, to generate the following substrate with a mispair at the 3′ end of the template to avoid forward reaction due to the other end pyrophosphorolysis:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.e002&amp;representation=PNG" class="inline-graphic"></span><br>Two separate mixes were made. One was a 2X mix of DNA substrate, PPi and enzyme prepared in the reaction buffer (17 mM Tris-HCl (pH 7.5), 17 mM NaCl, 1 mM dithiothreitol and 20% glycerol) along with different sodium pyrophosphate concentrations and was loaded in one syringe. The second mix consisted of 2X divalent cation (MgCl<sub>2</sub>) in the same buffer and was loaded in the second syringe. These separate mixes were pre-warmed at 22°C for 10 minutes and were then combined in equal volumes to start the reactions at 22°C. The assay mixture contained 20 nM template-primer, ~40 nM of active WT or mutant D<sub>211</sub>N Ty1 RT, 20 mM MgCl<sub>2</sub> and various concentrations of sodium pyrophosphate. Samples were taken at several time points and quenched with EDTA, at a final concentration of 35 mM. For D<sub>211</sub>N mutant Ty1 RT, reactions were carried out manually under the same conditions. Products were denatured and resolved in 17% denaturing gels, scanned and then analyzed. Amounts of the excised products (&lt;15-mer) were determined relative to total labeled products. The time course of a single turnover for individual [PPi] was fitted to an exponential equation (Equation 1). The calculated rates were plotted against respective [PPi] to fit this data to a hyperbolic equation (Equation 2).</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p><em>Equation 1:</em><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.e003&amp;representation=PNG" class="inline-graphic"></span><br>where k is the observed rate per second, [P<sub>n-1</sub>]<sub>t</sub> is the amount of product at a given time, t, [P<sub>n-1</sub>]<sub>max</sub> is the maximum amount of product formed is in nM and y<sub>o</sub> is the y intercept.</p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p><em>Equation 2:</em><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001389.e004&amp;representation=PNG" class="inline-graphic"></span><br></p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001389"></div><a name="pone.0001389.s001" id="pone.0001389.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001389.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Pyrophosphorolysis by WT and D211N Ty1 RT of a recessed primer terminus in the presence of a mismatched other end eliminates forward reaction. Time course of pyrophosphorolysis for WT or mutant D211N Ty1 RT were carried out under same conditions using 5′ 32P-end labeled 14-mer/28′-mer substrate (RAG*998/1109, <a href="#pone-0001389-g001">Fig. 1</a>) in the presence of 10 mM Mg2+. A, shows time course of WT reactions in seconds in the presence or absence of PPi. B, represents time course of D211N reactions with or without PPi. Control reactions are in the absence of Mg2+ or enzyme.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001389.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.09 MB TIF)</p>
<a name="pone.0001389.s002" id="pone.0001389.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001389.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Effects of different primer-ends and double strand length on pyrophosphorolysis. Pyrophosphorolysis reactions by WT were carried out for 60 minutes in the presence of either 10 mM Mg2+ or 2 mM Mn2+. A, shows relative positions of substrates 1–7 graphically. Substrates 1–7 are 5′ 32P-end labeled (1; RAG*1253/1050, 2; RAG*1249/1109, 3; RAG*998/1109, 4; RAG 998/*928, 5; RAG 955/*928, 6; RAG 831/*928, 7; RAG 952/*928, <a href="#pone-0001389-g001">Fig. 1</a>). B, shows WT reactions with different substrates in the presence of Mg2+ with or without sodium pyrophosphate. C, represents reactions with Mn2+. D, shows bar graph of I cleaved products/I total bands, black bars are for the reactions with Mg2+ and grey bars are for the reactions with Mn2+.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001389.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.25 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We greatly appreciate helpful discussions with Drs. Stefan Sarafianos, Samuel Wilson, and Isaac Wong, and advice and assistance by members of the Gabriel and Patel laboratories.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SP AG MP. Performed the experiments: MP. Analyzed the data: SP AG MP. Contributed reagents/materials/analysis tools: SP AG. Wrote the paper: AG MP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001389-Steitz1" id="pone.0001389-Steitz1"></a>Steitz TA (1999) DNA polymerases: structural diversity and common mechanisms. J Biol Chem  274: 17395–17398.  <ul class="find" data-citedArticleID="1037358" data-doi="10.1074/jbc.274.25.17395"><li><a href="http://dx.doi.org/10.1074/jbc.274.25.17395" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+polymerases%3A+structural+diversity+and+common+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+polymerases%3A+structural+diversity+and+common+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001389-Delarue1" id="pone.0001389-Delarue1"></a>Delarue M, Poch O, Tordo N, Moras D, Argos P (1990) An attempt to unify the structure of polymerases. Protein Eng  3: 461–467.  <ul class="find" data-citedArticleID="1037310" data-doi="10.1093/protein/3.6.461"><li><a href="http://dx.doi.org/10.1093/protein/3.6.461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+attempt+to+unify+the+structure+of+polymerases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+attempt+to+unify+the+structure+of+polymerases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001389-Baltimore1" id="pone.0001389-Baltimore1"></a>Baltimore D (1970) Viral RNA-dependent DNA polymerase. Nature  226: 1209–1211.  <ul class="find" data-citedArticleID="1037298" data-doi="10.1038/2261209a0"><li><a href="http://dx.doi.org/10.1038/2261209a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+RNA-dependent+DNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+RNA-dependent+DNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001389-Temin1" id="pone.0001389-Temin1"></a>Temin HM, Mizutani S (1970) RNA-dependent DNA polymerase in virions of Rous sarcoma virus. Nature  226: 1211–1213.  <ul class="find" data-citedArticleID="1037360" data-doi="10.1038/2261211a0"><li><a href="http://dx.doi.org/10.1038/2261211a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA-dependent+DNA+polymerase+in+virions+of+Rous+sarcoma+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA-dependent+DNA+polymerase+in+virions+of+Rous+sarcoma+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001389-Gilboa1" id="pone.0001389-Gilboa1"></a>Gilboa E, Mitra SW, Goff S, Baltimore D (1979) A detailed model of reverse transcription and a test of crucial aspects. Cell  18: 93–100.  <ul class="find" data-citedArticleID="1037312" data-doi="10.1016/0092-8674(79)90357-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(79)90357-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+detailed+model+of+reverse+transcription+and+a+test+of+crucial+aspects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+detailed+model+of+reverse+transcription+and+a+test+of+crucial+aspects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001389-Skalka1" id="pone.0001389-Skalka1"></a> Skalka AM, Goff SP, editors.  (1993) Reverse Transcriptase. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0001389-Lingner1" id="pone.0001389-Lingner1"></a>Lingner J, Hughes TR, Shevchenko A, Mann M, Lundblad V, et al.  (1997) Reverse transcriptase motifs in the catalytic subunit of telomerase. Science  276: 561–567.  <ul class="find" data-citedArticleID="1037336" data-doi="10.1126/science.276.5312.561"><li><a href="http://dx.doi.org/10.1126/science.276.5312.561" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+transcriptase+motifs+in+the+catalytic+subunit+of+telomerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+transcriptase+motifs+in+the+catalytic+subunit+of+telomerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001389-Larder1" id="pone.0001389-Larder1"></a>Larder BA, Purifoy DJ, Powell KL, Darby G (1987) Site-specific mutagenesis of AIDS virus reverse transcriptase. Nature  327: 716–717.  <ul class="find" data-citedArticleID="1037330" data-doi="10.1038/327716a0"><li><a href="http://dx.doi.org/10.1038/327716a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-specific+mutagenesis+of+AIDS+virus+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-specific+mutagenesis+of+AIDS+virus+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001389-LeGrice1" id="pone.0001389-LeGrice1"></a>Le Grice SFJ, Naas T, Wohlgensinger B, Schatz O (1991) Subunit-selective mutagenesis indicates minimal polymerase activity in heterodimer-associated p51 HIV-1 reverse transcriptase. EMBO J  10: 3905–3911.  <ul class="find" data-citedArticleID="1037332"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Subunit-selective+mutagenesis+indicates+minimal+polymerase+activity+in+heterodimer-associated+p51+HIV-1+reverse+transcriptase.&amp;auth=&amp;atitle=Subunit-selective+mutagenesis+indicates+minimal+polymerase+activity+in+heterodimer-associated+p51+HIV-1+reverse+transcriptase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subunit-selective+mutagenesis+indicates+minimal+polymerase+activity+in+heterodimer-associated+p51+HIV-1+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subunit-selective+mutagenesis+indicates+minimal+polymerase+activity+in+heterodimer-associated+p51+HIV-1+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001389-Uzun1" id="pone.0001389-Uzun1"></a>Uzun O, Gabriel A (2001) A Ty1 reverse transcriptase active-site aspartate mutation blocks transposition but not polymerization. J Virol  75: 6337–6347.  <ul class="find" data-citedArticleID="1037364" data-doi="10.1128/jvi.75.14.6337-6347.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.14.6337-6347.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Ty1+reverse+transcriptase+active-site+aspartate+mutation+blocks+transposition+but+not+polymerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Ty1+reverse+transcriptase+active-site+aspartate+mutation+blocks+transposition+but+not+polymerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001389-Bibillo1" id="pone.0001389-Bibillo1"></a>Bibillo A, Lener D, Klarmann GJ, Le Grice SF (2005) Functional roles of carboxylate residues comprising the DNA polymerase active site triad of Ty3 reverse transcriptase. Nucleic Acids Res  33: 171–181.  <ul class="find" data-citedArticleID="1037304" data-doi="10.1093/nar/gki150"><li><a href="http://dx.doi.org/10.1093/nar/gki150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+roles+of+carboxylate+residues+comprising+the+DNA+polymerase+active+site+triad+of+Ty3+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+roles+of+carboxylate+residues+comprising+the+DNA+polymerase+active+site+triad+of+Ty3+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001389-Pandey1" id="pone.0001389-Pandey1"></a>Pandey M, Patel S, Gabriel A (2004) Insights into the role of an active site aspartate in Ty1 reverse transcriptase polymerization. J Biol Chem  279: 47840–47848.  <ul class="find" data-citedArticleID="1037342" data-doi="10.1074/jbc.m406019200"><li><a href="http://dx.doi.org/10.1074/jbc.m406019200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+the+role+of+an+active+site+aspartate+in+Ty1+reverse+transcriptase+polymerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+the+role+of+an+active+site+aspartate+in+Ty1+reverse+transcriptase+polymerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001389-Urban1" id="pone.0001389-Urban1"></a>Urban S, Fischer KP, Tyrrell DL (2001) Efficient pyrophosphorolysis by a hepatitis B virus polymerase may be a primer-unblocking mechanism. Proc Natl Acad Sci U S A  98: 4984–4989.  <ul class="find" data-citedArticleID="1037362" data-doi="10.1073/pnas.091324398"><li><a href="http://dx.doi.org/10.1073/pnas.091324398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+pyrophosphorolysis+by+a+hepatitis+B+virus+polymerase+may+be+a+primer-unblocking+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+pyrophosphorolysis+by+a+hepatitis+B+virus+polymerase+may+be+a+primer-unblocking+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001389-DAbramo1" id="pone.0001389-DAbramo1"></a>D'Abramo CM, Cellai L, Gotte M (2004) Excision of incorporated nucleotide analogue chain-terminators can diminish their inhibitory effects on viral RNA-dependent RNA polymerases. J Mol Biol  337: 1–14.  <ul class="find" data-citedArticleID="1037308" data-doi="10.1016/j.jmb.2004.01.030"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.01.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Excision+of+incorporated+nucleotide+analogue+chain-terminators+can+diminish+their+inhibitory+effects+on+viral+RNA-dependent+RNA+polymerases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Excision+of+incorporated+nucleotide+analogue+chain-terminators+can+diminish+their+inhibitory+effects+on+viral+RNA-dependent+RNA+polymerases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001389-Goldschmidt1" id="pone.0001389-Goldschmidt1"></a>Goldschmidt V, Marquet R (2004) Primer unblocking by HIV-1 reverse transcriptase and resistance to nucleoside RT inhibitors (NRTIs). Int J Biochem Cell Biol  36: 1687–1705.  <ul class="find" data-citedArticleID="1037314" data-doi="10.1016/j.biocel.2004.02.028"><li><a href="http://dx.doi.org/10.1016/j.biocel.2004.02.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primer+unblocking+by+HIV-1+reverse+transcriptase+and+resistance+to+nucleoside+RT+inhibitors+%28NRTIs%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primer+unblocking+by+HIV-1+reverse+transcriptase+and+resistance+to+nucleoside+RT+inhibitors+%28NRTIs%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001389-Meyer1" id="pone.0001389-Meyer1"></a>Meyer PR, Matsuura SE, So AG, Scott WA (1998) Unblocking of chain-terminated primer by HIV-1 reverse transcriptase through a nucleotide-dependent mechanism. Proc Natl Acad Sci U S A  95: 13471–13476.  <ul class="find" data-citedArticleID="1037338" data-doi="10.1073/pnas.95.23.13471"><li><a href="http://dx.doi.org/10.1073/pnas.95.23.13471" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unblocking+of+chain-terminated+primer+by+HIV-1+reverse+transcriptase+through+a+nucleotide-dependent+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unblocking+of+chain-terminated+primer+by+HIV-1+reverse+transcriptase+through+a+nucleotide-dependent+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001389-Meyer2" id="pone.0001389-Meyer2"></a>Meyer PR, Matsuura SE, Zonarich D, Chopra RR, Pendarvis E, et al.  (2003) Relationship between 3'-azido-3'-deoxythymidine resistance and primer unblocking activity in foscarnet-resistant mutants of human immunodeficiency virus type 1 reverse transcriptase. J Virol  77: 6127–6137.  <ul class="find" data-citedArticleID="1037340" data-doi="10.1128/jvi.77.11.6127-6137.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.11.6127-6137.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+3%27-azido-3%27-deoxythymidine+resistance+and+primer+unblocking+activity+in+foscarnet-resistant+mutants+of+human+immunodeficiency+virus+type+1+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+3%27-azido-3%27-deoxythymidine+resistance+and+primer+unblocking+activity+in+foscarnet-resistant+mutants+of+human+immunodeficiency+virus+type+1+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001389-Sarafianos1" id="pone.0001389-Sarafianos1"></a>Sarafianos SG, Clark AD Jr, Tuske S, Squire CJ, Das K, et al.  (2003) Trapping HIV-1 reverse transcriptase before and after translocation on DNA. J Biol Chem  278: 16280–16288.  <ul class="find" data-citedArticleID="1037352" data-doi="10.1074/jbc.m212911200"><li><a href="http://dx.doi.org/10.1074/jbc.m212911200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trapping+HIV-1+reverse+transcriptase+before+and+after+translocation+on+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trapping+HIV-1+reverse+transcriptase+before+and+after+translocation+on+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001389-Sarafianos2" id="pone.0001389-Sarafianos2"></a>Sarafianos SG, Clark AD Jr, Das K, Tuske S, Birktoft JJ, et al.  (2002) Structures of HIV-1 reverse transcriptase with pre- and post-translocation AZTMP-terminated DNA. Embo J  21: 6614–6624.  <ul class="find" data-citedArticleID="1037354" data-doi="10.1093/emboj/cdf637"><li><a href="http://dx.doi.org/10.1093/emboj/cdf637" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structures+of+HIV-1+reverse+transcriptase+with+pre-+and+post-translocation+AZTMP-terminated+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structures+of+HIV-1+reverse+transcriptase+with+pre-+and+post-translocation+AZTMP-terminated+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001389-Huang1" id="pone.0001389-Huang1"></a>Huang H, Chopra R, Verdine GL, Harrison SC (1998) Structure of a covalently trapped catalytic complex of HIV-1 reverse transcriptase: implications for drug resistance. Science  282: 1669–1675.  <ul class="find" data-citedArticleID="1037320" data-doi="10.1126/science.282.5394.1669"><li><a href="http://dx.doi.org/10.1126/science.282.5394.1669" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+a+covalently+trapped+catalytic+complex+of+HIV-1+reverse+transcriptase%3A+implications+for+drug+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+a+covalently+trapped+catalytic+complex+of+HIV-1+reverse+transcriptase%3A+implications+for+drug+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001389-Kaushik1" id="pone.0001389-Kaushik1"></a>Kaushik N, Rege N, Yadav PN, Sarafianos SG, Modak MJ, et al.  (1996) Biochemical analysis of catalytically crucial aspartate mutants of human immunodeficiency virus type 1 reverse transcriptase. Biochemistry  35: 11536–11546.  <ul class="find" data-citedArticleID="1037328" data-doi="10.1021/bi960364x"><li><a href="http://dx.doi.org/10.1021/bi960364x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+analysis+of+catalytically+crucial+aspartate+mutants+of+human+immunodeficiency+virus+type+1+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+analysis+of+catalytically+crucial+aspartate+mutants+of+human+immunodeficiency+virus+type+1+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001389-Bolton1" id="pone.0001389-Bolton1"></a>Bolton EC, Mildvan AS, Boeke JD (2002) Inhibition of reverse transcription in vivo by elevated manganese ion concentration. Mol Cell  9: 879–889.  <ul class="find" data-citedArticleID="1037306" data-doi="10.1016/s1097-2765(02)00495-1"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00495-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+reverse+transcription+in+vivo+by+elevated+manganese+ion+concentration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+reverse+transcription+in+vivo+by+elevated+manganese+ion+concentration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001389-Kashkina1" id="pone.0001389-Kashkina1"></a>Kashkina E, Anikin M, Tahirov TH, Kochetkov SN, Vassylyev DG, et al.  (2006) Elongation complexes of Thermus thermophilus RNA polymerase that possess distinct translocation conformations. Nucleic Acids Res  34: 4036–4045.  <ul class="find" data-citedArticleID="1037326" data-doi="10.1093/nar/gkl559"><li><a href="http://dx.doi.org/10.1093/nar/gkl559" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elongation+complexes+of+Thermus+thermophilus+RNA+polymerase+that+possess+distinct+translocation+conformations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elongation+complexes+of+Thermus+thermophilus+RNA+polymerase+that+possess+distinct+translocation+conformations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001389-Wilhelm1" id="pone.0001389-Wilhelm1"></a>Wilhelm M, Boutabout M, Wilhelm F-X (2000) Expression of an active form of recombinant Ty1 reverse transcriptase in <em>Escherichia coli</em>: a fusion protein containing the C-terminal region of the Ty1 integrase linked to the reverse transcriptase–RNase H domain exhibits polymerase and RNase H activities. Biochem J  348: 337–342.  <ul class="find" data-citedArticleID="1037366" data-doi="10.1042/0264-6021:3480337"><li><a href="http://dx.doi.org/10.1042/0264-6021:3480337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+an+active+form+of+recombinant+Ty1+reverse+transcriptase+in+Escherichia+coli%3A+a+fusion+protein+containing+the+C-terminal+region+of+the+Ty1+integrase+linked+to+the+reverse+transcriptase%E2%80%93RNase+H+domain+exhibits+polymerase+and+RNase+H+activities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+an+active+form+of+recombinant+Ty1+reverse+transcriptase+in+Escherichia+coli%3A+a+fusion+protein+containing+the+C-terminal+region+of+the+Ty1+integrase+linked+to+the+reverse+transcriptase%E2%80%93RNase+H+domain+exhibits+polymerase+and+RNase+H+activities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001389-Wilhelm2" id="pone.0001389-Wilhelm2"></a>Wilhelm FX, Wilhelm M, Gabriel A (2005) Reverse transcriptase and integrase of the Saccharomyces cerevisiae Ty1 element. Cytogenet Genome Res  110: 269–287.  <ul class="find" data-citedArticleID="1037368" data-doi="10.1159/000084960"><li><a href="http://dx.doi.org/10.1159/000084960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+transcriptase+and+integrase+of+the+Saccharomyces+cerevisiae+Ty1+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+transcriptase+and+integrase+of+the+Saccharomyces+cerevisiae+Ty1+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001389-Beard1" id="pone.0001389-Beard1"></a>Beard WA, Wilson SH (2006) Structure and mechanism of DNA polymerase Beta. Chem Rev  106: 361–382.  <ul class="find" data-citedArticleID="1037302" data-doi="10.1021/cr0404904"><li><a href="http://dx.doi.org/10.1021/cr0404904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+mechanism+of+DNA+polymerase+Beta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+mechanism+of+DNA+polymerase+Beta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001389-Yin1" id="pone.0001389-Yin1"></a>Yin YW, Steitz TA (2002) Structural basis for the transition from initiation to elongation transcription in T7 RNA polymerase. Science  298: 1387–1395.  <ul class="find" data-citedArticleID="1037370" data-doi="10.1126/science.1077464"><li><a href="http://dx.doi.org/10.1126/science.1077464" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+the+transition+from+initiation+to+elongation+transcription+in+T7+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+the+transition+from+initiation+to+elongation+transcription+in+T7+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001389-Arnold1" id="pone.0001389-Arnold1"></a>Arnold JJ, Cameron CE (2004) Poliovirus RNA-dependent RNA polymerase (3Dpol): pre-steady-state kinetic analysis of ribonucleotide incorporation in the presence of Mg2+. Biochemistry  43: 5126–5137.  <ul class="find" data-citedArticleID="1037296" data-doi="10.1021/bi035212y"><li><a href="http://dx.doi.org/10.1021/bi035212y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Poliovirus+RNA-dependent+RNA+polymerase+%283Dpol%29%3A+pre-steady-state+kinetic+analysis+of+ribonucleotide+incorporation+in+the+presence+of+Mg2%2B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Poliovirus+RNA-dependent+RNA+polymerase+%283Dpol%29%3A+pre-steady-state+kinetic+analysis+of+ribonucleotide+incorporation+in+the+presence+of+Mg2%2B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001389-Arion1" id="pone.0001389-Arion1"></a>Arion D, Kaushik N, McCormick S, Borkow G, Parniak MA (1998) Phenotypic mechanism of HIV-1 resistance to 3'-azido-3'-deoxythymidine (AZT): increased polymerization processivity and enhanced sensitivity to pyrophosphate of the mutant viral reverse transcriptase. Biochemistry  37: 15908–15917.  <ul class="find" data-citedArticleID="1037294" data-doi="10.1021/bi981200e"><li><a href="http://dx.doi.org/10.1021/bi981200e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+mechanism+of+HIV-1+resistance+to+3%27-azido-3%27-deoxythymidine+%28AZT%29%3A+increased+polymerization+processivity+and+enhanced+sensitivity+to+pyrophosphate+of+the+mutant+viral+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+mechanism+of+HIV-1+resistance+to+3%27-azido-3%27-deoxythymidine+%28AZT%29%3A+increased+polymerization+processivity+and+enhanced+sensitivity+to+pyrophosphate+of+the+mutant+viral+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001389-Hsieh1" id="pone.0001389-Hsieh1"></a>Hsieh JC, Zinnen S, Modrich P (1993) Kinetic mechanism of the DNA-dependent DNA polymerase activity of human immunodeficiency virus reverse transcriptase. J Biol Chem  268: 24607–24613.  <ul class="find" data-citedArticleID="1037318"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Kinetic+mechanism+of+the+DNA-dependent+DNA+polymerase+activity+of+human+immunodeficiency+virus+reverse+transcriptase.&amp;auth=&amp;atitle=Kinetic+mechanism+of+the+DNA-dependent+DNA+polymerase+activity+of+human+immunodeficiency+virus+reverse+transcriptase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetic+mechanism+of+the+DNA-dependent+DNA+polymerase+activity+of+human+immunodeficiency+virus+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetic+mechanism+of+the+DNA-dependent+DNA+polymerase+activity+of+human+immunodeficiency+virus+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001389-Reardon1" id="pone.0001389-Reardon1"></a>Reardon JE (1993) Human immunodeficiency virus reverse transcriptase. A kinetic analysis of RNA-dependent and DNA-dependent DNA polymerization. J Biol Chem  268: 8743–8751.  <ul class="find" data-citedArticleID="1037350"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+immunodeficiency+virus+reverse+transcriptase.+A+kinetic+analysis+of+RNA-dependent+and+DNA-dependent+DNA+polymerization.&amp;auth=&amp;atitle=Human+immunodeficiency+virus+reverse+transcriptase.+A+kinetic+analysis+of+RNA-dependent+and+DNA-dependent+DNA+polymerization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+reverse+transcriptase.+A+kinetic+analysis+of+RNA-dependent+and+DNA-dependent+DNA+polymerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+reverse+transcriptase.+A+kinetic+analysis+of+RNA-dependent+and+DNA-dependent+DNA+polymerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001389-Patel1" id="pone.0001389-Patel1"></a>Patel SS, Wong I, Johnson KA (1991) Pre-steady-state kinetic analysis of processive DNA replication including complete characterization of an exonuclease-deficient mutant. Biochemistry  30: 511–525.  <ul class="find" data-citedArticleID="1037344" data-doi="10.1021/bi00216a029"><li><a href="http://dx.doi.org/10.1021/bi00216a029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pre-steady-state+kinetic+analysis+of+processive+DNA+replication+including+complete+characterization+of+an+exonuclease-deficient+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pre-steady-state+kinetic+analysis+of+processive+DNA+replication+including+complete+characterization+of+an+exonuclease-deficient+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001389-Anand1" id="pone.0001389-Anand1"></a>Anand VS, Patel SS (2006) Transient state kinetics of transcription elongation by T7 RNA polymerase. J Biol Chem  281: 35677–35685.  <ul class="find" data-citedArticleID="1037292" data-doi="10.1074/jbc.m608180200"><li><a href="http://dx.doi.org/10.1074/jbc.m608180200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transient+state+kinetics+of+transcription+elongation+by+T7+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transient+state+kinetics+of+transcription+elongation+by+T7+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001389-Batra1" id="pone.0001389-Batra1"></a>Batra VK, Beard WA, Shock DD, Krahn JM, Pedersen LC, et al.  (2006) Magnesium-induced assembly of a complete DNA polymerase catalytic complex. Structure  14: 757–766.  <ul class="find" data-citedArticleID="1037300" data-doi="10.1016/j.str.2006.01.011"><li><a href="http://dx.doi.org/10.1016/j.str.2006.01.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Magnesium-induced+assembly+of+a+complete+DNA+polymerase+catalytic+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Magnesium-induced+assembly+of+a+complete+DNA+polymerase+catalytic+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001389-Lin1" id="pone.0001389-Lin1"></a>Lin P, Pedersen LC, Batra VK, Beard WA, Wilson SH, et al.  (2006) Energy analysis of chemistry for correct insertion by DNA polymerase beta. Proc Natl Acad Sci U S A  103: 13294–13299.  <ul class="find" data-citedArticleID="1037334" data-doi="10.1073/pnas.0606006103"><li><a href="http://dx.doi.org/10.1073/pnas.0606006103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Energy+analysis+of+chemistry+for+correct+insertion+by+DNA+polymerase+beta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Energy+analysis+of+chemistry+for+correct+insertion+by+DNA+polymerase+beta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001389-Jiang1" id="pone.0001389-Jiang1"></a>Jiang MY, Sheetz MP (1994) Mechanics of myosin motor: force and step size. Bioessays  16: 531–532.  <ul class="find" data-citedArticleID="1037322" data-doi="10.1002/bies.950160803"><li><a href="http://dx.doi.org/10.1002/bies.950160803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanics+of+myosin+motor%3A+force+and+step+size." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanics+of+myosin+motor%3A+force+and+step+size.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001389-Patel2" id="pone.0001389-Patel2"></a>Patel PH, Jacobo-Molina A, Ding J, Tantillo C, Clark AD, et al.  (1995) Insights into DNA polymerization mechanisms from structure and function analysis of HIV-1 reverse transcriptase. Biochemistry  34: 5351–5365.  <ul class="find" data-citedArticleID="1037346" data-doi="10.1021/bi00016a006"><li><a href="http://dx.doi.org/10.1021/bi00016a006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+DNA+polymerization+mechanisms+from+structure+and+function+analysis+of+HIV-1+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+DNA+polymerization+mechanisms+from+structure+and+function+analysis+of+HIV-1+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001389-Yin2" id="pone.0001389-Yin2"></a>Yin YW, Steitz TA (2004) The structural mechanism of translocation and helicase activity in T7 RNA polymerase. Cell  116: 393–404.  <ul class="find" data-citedArticleID="1037372" data-doi="10.1016/s0092-8674(04)00120-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00120-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structural+mechanism+of+translocation+and+helicase+activity+in+T7+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structural+mechanism+of+translocation+and+helicase+activity+in+T7+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001389-Guajardo1" id="pone.0001389-Guajardo1"></a>Guajardo R, Sousa R (1997) A model for the mechanism of polymerase translocation. J Mol Biol  265: 8–19.  <ul class="find" data-citedArticleID="1037316" data-doi="10.1006/jmbi.1996.0707"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+model+for+the+mechanism+of+polymerase+translocation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+model+for+the+mechanism+of+polymerase+translocation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class=