

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001740"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001740" />

    <meta name="citation_title" content="Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History"/>
    <meta itemprop="name" content="Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History"/>

      <meta name="citation_author" content="Corinne Vacher"/>
            <meta name="citation_author_institution" content="Institut Scientifique de Recherche Agronomique (INRA), UMR1202 Biodiversité Gènes et Communautés, Villenave d'Ornon, France"/>
      <meta name="citation_author" content="Dominique Piou"/>
            <meta name="citation_author_institution" content="Institut Scientifique de Recherche Agronomique (INRA), UMR1202 Biodiversité Gènes et Communautés, Villenave d'Ornon, France"/>
            <meta name="citation_author_institution" content="Département de la Santé des Forêts, Ministère de l'Agriculture et de la Pêche, Cenon, France"/>
      <meta name="citation_author" content="Marie-Laure Desprez-Loustau"/>
            <meta name="citation_author_institution" content="Institut Scientifique de Recherche Agronomique (INRA), UMR1202 Biodiversité Gènes et Communautés, Villenave d'Ornon, France"/>

    <meta name="citation_date" content="2008/3/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001740.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1740"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The structure and function of complex networks.; citation_author=MEJ Newman; citation_journal_title=SIAM Review; citation_volume=45; citation_number=1; citation_pages=167-256; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Parasitism: Ecology and Evolution of Intimate Interactions.; citation_author=C Combes; citation_number=2; citation_date=2001; citation_publisher=University Press Chicago; " />
      <meta name="citation_reference" content="citation_title=Structure in plant-animal interaction assemblages.; citation_author=TM Lewinsohn; citation_author=PI Prado; citation_author=P Jordano; citation_author=J Bascompte; citation_author=JM Olesen; citation_journal_title=Oikos; citation_volume=113; citation_number=3; citation_pages=174-184; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The nested assembly of plant-animal mutualistic networks.; citation_author=J Bascompte; citation_author=P Jordano; citation_author=CJ Melian; citation_author=JM Olesen; citation_journal_title=PNAS; citation_volume=100; citation_number=4; citation_pages=9383-9387; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Asymmetries in specialization in ant-plant mutualistic networks.; citation_author=PR Guimaraes; citation_author=V Rico-Gray; citation_author=SF dos Reis; citation_author=JN Thompson; citation_journal_title=Proc R Soc B-Biol Sci; citation_volume=273; citation_number=5; citation_pages=2041-2047; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The nested structure of marine cleaning symbiosis: is it like flowers and bees?; citation_author=PR Guimaraes; citation_author=C Sazima; citation_author=SF dos Reis; citation_author=I Sazima; citation_journal_title=Biol Lett; citation_volume=3; citation_number=6; citation_pages=51-54; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Finding NEMO: nestedness engendered by mutualistic organization in anemonefish and their hosts.; citation_author=J Ollerton; citation_author=D McCollin; citation_author=DG Fautin; citation_author=GR Allen; citation_journal_title=Proc R B-Biol Sci; citation_volume=274; citation_number=7; citation_pages=591-598; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ecological networks nestedness and sampling effort.; citation_author=A Nielsen; citation_author=J Bascompte; citation_journal_title=Journal of Ecology; citation_volume=95; citation_number=8; citation_pages=1134-1141; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Asymmetric specialization: A pervasive feature of plant-pollinator interactions.; citation_author=DP Vazquez; citation_author=MA Aizen; citation_journal_title=Ecology; citation_volume=85; citation_number=9; citation_pages=1251-1257; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Linkage rules for plant-pollinator networks: trait complementarity or exploitation barriers?; citation_author=L Santamaria; citation_author=MA Rodriguez-Girones; citation_journal_title=PLoS Biol; citation_volume=5; citation_number=10; citation_pages=e31; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Species abundance and asymmetric interaction strength in ecological networks.; citation_author=DP Vazquez; citation_author=CJ Melian; citation_author=NM Williams; citation_author=N Bluthgen; citation_author=BR Krasnov; citation_journal_title=Oikos; citation_volume=116; citation_number=11; citation_pages=1120-1127; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The modularity of pollination networks.; citation_author=JM Olesen; citation_author=J Bascompte; citation_author=YL Dupont; citation_author=P Jordano; citation_journal_title=PNAS; citation_volume=104; citation_number=12; citation_pages=19891-19896; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Compartments in insect-plant associations and their consequences for community structure.; citation_author=PI Prado; citation_author=TM Lewinsohn; citation_journal_title=J Anim Ecol; citation_volume=73; citation_number=13; citation_pages=1168-1178; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Asymmetries compartments and null interactions in an Amazonian ant-plant community.; citation_author=CR Fonseca; citation_author=G Ganade; citation_journal_title=J Anim Ecol; citation_volume=65; citation_number=14; citation_pages=339-347; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Interaction intimacy affects structure and coevolutionary dynamics in mutualistic networks.; citation_author=PR Guimaraes; citation_author=V Rico-Gray; citation_author=PS Oliveira; citation_author=PJ Izzo; citation_author=SF dos Reis; citation_journal_title=Curr Biol; citation_volume=17; citation_number=15; citation_pages=1797-803; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Compartmentalization in plant-insect flower visitor webs.; citation_author=LV Dicks; citation_author=SA Corbet; citation_author=RF Pywell; citation_journal_title=J Anim Ecol; citation_volume=71; citation_number=16; citation_pages=32-43; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Compartments revealed in food-web structure.; citation_author=AE Krause; citation_author=KA Frank; citation_author=DM Mason; citation_author=RE Ulanowicz; citation_author=WW Taylor; citation_journal_title=Nature; citation_volume=426; citation_number=17; citation_pages=282-285; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic constraints and adaptation explain food-web structure.; citation_author=MF Cattin; citation_author=LF Bersier; citation_author=C Banasek-Richter; citation_author=R Baltensperger; citation_author=JP Gabriel; citation_journal_title=Nature; citation_volume=427; citation_number=18; citation_pages=835-839; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Reconstructing the early evolution of Fungi using a six-gene phylogeny.; citation_author=TY James; citation_author=F Kauff; citation_author=CL Schoch; citation_author=PB Matheny; citation_author=V Hofstetter; citation_journal_title=Nature; citation_volume=443; citation_number=19; citation_pages=818-822; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=C Chappey; citation_author=AE Lash; citation_author=DD Leipe; citation_author=TL Madden; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=20; citation_pages=10-14; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Life-history strategies of plant pathogens: Distribution patterns and phylogenetic analysis.; citation_author=G Garcia-Guzman; citation_author=E Morales; citation_journal_title=Ecology; citation_volume=88; citation_number=21; citation_pages=589-596; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Angiosperm Phylogeny Website Version 7 May 2006.; citation_author=PF Stevens; citation_number=22; citation_date=2001 onwards; " />
      <meta name="citation_reference" content="citation_title=Phylomatic: tree assembly for applied phylogenetics.; citation_author=CO Webb; citation_author=MJ Donoghue; citation_journal_title=Mol Ecol Notes; citation_volume=5; citation_number=23; citation_pages=181-183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Darwin's abominable mystery: Insights from a supertree of the angiosperms.; citation_author=TJ Davies; citation_author=TG Barraclough; citation_author=MW Chase; citation_author=PS Soltis; citation_author=DE Soltis; citation_journal_title=PNAS; citation_volume=101; citation_number=24; citation_pages=1904-1909; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ape: Analyses of Phylogenetics and Evolution version 18-5.; citation_author=E Paradis; citation_author=K Strimmer; citation_author=J Claude; citation_author=G Jobb; citation_author=R Opgen-Rhein; citation_number=25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=igraph: Routines for network analysis R package.; citation_author=G Csardi; citation_number=26; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Community structure in social and biological networks.; citation_author=M Girvan; citation_author=MEJ Newman; citation_journal_title=PNAS; citation_volume=99; citation_number=27; citation_pages=7821-7826; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Centrality in Social Networks I: Conceptual Clarification.; citation_author=LC Freeman; citation_journal_title=Social Networks; citation_volume=1; citation_number=28; citation_pages=215-239; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=A faster algorithm for betweenness centrality.; citation_author=U Brandes; citation_journal_title=Journal of Mathematical Sociology; citation_volume=25; citation_number=29; citation_pages=163-177; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=TREEVIEW: An application to display phylogenetic trees on personal computers.; citation_author=RDM Page; citation_journal_title=Computer Applications in the Biosciences; citation_volume=12; citation_number=30; citation_pages=357-358; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Permutation Methods: A Distance Function Approach.; citation_author=PW Mielke; citation_author=KJ Berry; citation_number=31; citation_date=2001; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=vegan: Community Ecology Package version 18-3.; citation_author=J Oksanen; citation_author=R Kindt; citation_author=P Legendre; citation_author=B O'Hara; citation_number=32; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The measure of order and disorder in the distribution of species in fragmented habitat.; citation_author=W Atmar; citation_author=BD Patterson; citation_journal_title=Oecologia; citation_volume=96; citation_number=33; citation_pages=373-382; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Improving the analyses of nestedness for large sets of matrices.; citation_author=PR Guimaraes; citation_author=P Guimaraes; citation_journal_title=Environ Modell Soft; citation_volume=21; citation_number=34; citation_pages=1512-1513; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria, ISBN 3-900051-07-0.; citation_number=35; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Is the composition of phytophagous insects and parasitic fungi among trees predictable?; citation_author=M Brandle; citation_author=R Brandl; citation_journal_title=Oikos; citation_volume=113; citation_number=36; citation_pages=296-304; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Taxus baccata L.; citation_author=PA Thomas; citation_author=A Polwart; citation_journal_title=J Ecol; citation_volume=91(3); citation_number=37; citation_pages=489-524; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Species richness on trees: a comparison of parasitic fungi and insects.; citation_author=M Brandle; citation_author=R Brandl; citation_journal_title=Evol Ecol Res; citation_volume=5; citation_number=38; citation_pages=941-952; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rapid evolutionary changes in a globally invading fungal pathogen (Dutch elm disease).; citation_author=CM Brasier; citation_author=W Buck; citation_journal_title=Biol Invasions; citation_volume=3; citation_number=39; citation_pages=223-233; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Diseases of Forest and Ornamental Trees.; citation_author=DH Phillips; citation_author=DA Burdekin; citation_number=40; citation_date=1982; citation_publisher=The Macmillan Press; " />
      <meta name="citation_reference" content="citation_title=The structure of plant-animal mutualistic networks.; citation_author=J Bascompte; citation_author=P Jordano; citation_number=41; citation_pages=143-149; citation_date=2006; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic signal in plant pathogen-host range.; citation_author=GS Gilbert; citation_author=CO Webb; citation_journal_title=PNAS; citation_volume=104; citation_number=42; citation_pages=4979-4983; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Fungal molecular evolution: gene trees and geologic time.; citation_author=ML Berbee; citation_author=JW Taylor; citation_number=43; citation_pages=229-245; citation_date=2001; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence for the early colonization of land by fungi and plants.; citation_author=DS Heckman; citation_author=DM Geiser; citation_author=BR Eidell; citation_author=RL Stauffer; citation_author=NL Kardos; citation_journal_title=Science; citation_volume=293; citation_number=44; citation_pages=1129-1133; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The oldest fossil ascomycetes.; citation_author=TN Taylor; citation_author=T Hass; citation_author=H Kerp; citation_journal_title=Nature; citation_volume=399; citation_number=45; citation_pages=648-648; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The land flora: a phototroph-fungus partnership?; citation_author=MA Selosse; citation_author=F Le Tacon; citation_journal_title=TREE; citation_volume=13; citation_number=46; citation_pages=15-20; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The phylogeny of plant and animal pathogens in the Ascomycota.; citation_author=ML Berbee; citation_journal_title=Physiol Mol Plant Pathol; citation_volume=59; citation_number=47; citation_pages=165-187; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolution of the angiosperms: calibrating the family tree.; citation_author=N Wikstrom; citation_author=V Savolainen; citation_author=MW Chase; citation_journal_title=Proc R Soc B-Biol Sci; citation_volume=268; citation_number=48; citation_pages=2211-2220; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The age of the angiosperms: A molecular timescale without a clock.; citation_author=CD Bell; citation_author=DE Soltis; citation_author=PS Soltis; citation_journal_title=Evolution; citation_volume=59; citation_number=49; citation_pages=1245-1258; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Patterns of association between crucifers and their flower-mimic pathogens: Host jumps are more common than coevolution or cospeciation.; citation_author=BA Roy; citation_journal_title=Evolution; citation_volume=55; citation_number=50; citation_pages=41-53; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The evolutionary ecology of novel plant-pathogen interactions.; citation_author=IM Parker; citation_author=GS Gilbert; citation_journal_title=Ann Rev Ecol Evol Syst; citation_volume=35; citation_number=51; citation_pages=675-700; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Species abundance and the distribution of specialization in host-parasite interaction networks.; citation_author=DP Vazquez; citation_author=R Poulin; citation_author=BR Krasnov; citation_author=GI Shenbrot; citation_journal_title=J Anim Ecol; citation_volume=74; citation_number=52; citation_pages=946-955; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Glacial refugia: Hotspots but not melting pots of genetic diversity.; citation_author=RJ Petit; citation_author=I Aguinagalde; citation_author=JL de Beaulieu; citation_author=C Bittkau; citation_author=S Brewer; citation_journal_title=Science; citation_volume=300; citation_number=53; citation_pages=1563-1565; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of trophic associations.; citation_author=AR Ives; citation_author=HCJ Godfray; citation_journal_title=Am Nat; citation_volume=168; citation_number=54; citation_pages=E1-E14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Non-random coextinctions in phylogenetically structured mutualistic networks.; citation_author=EL Rezende; citation_author=JE Lavabre; citation_author=PR Guimaraes; citation_author=P Jordano; citation_author=J Bascompte; citation_journal_title=Nature; citation_volume=448; citation_number=55; citation_pages=925-U6; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001740" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History"/>
    <meta name="twitter:description" content="BackgroundCompartmentalization and nestedness are common patterns in ecological networks. The aim of this study was to elucidate some of the processes shaping these patterns in a well resolved network of host/pathogen interactions.Methology/Principal FindingsBased on a long-term (1972&ndash;2005) survey of forest health at the regional scale (all French forests; 15 million ha), we uncovered an almost fully connected network of 51 tree taxa and 157 parasitic fungal species. Our analyses revealed that the compartmentalization of the network maps out the ancient evolutionary history of seed plants, but not the ancient evolutionary history of fungal species. The very early divergence of the major fungal phyla may account for this asymmetric influence of past evolutionary history. Unlike compartmentalization, nestedness did not reflect any consistent phylogenetic signal. Instead, it seemed to reflect the ecological features of the current species, such as the relative abundance of tree species and the life-history strategies of fungal pathogens. We discussed how the evolution of host range in fungal species may account for the observed nested patterns.Conclusion/SignificanceOverall, our analyses emphasized how the current complexity of ecological networks results from the diversification of the species and their interactions over evolutionary times. They confirmed that the current architecture of ecological networks is not only dependant on recent ecological processes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001740.g005"/>

  <meta property="og:title" content="Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001740" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1185'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=69'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4688&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001740">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001740" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001740&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Corinne Vacher
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:cvacher@bordeaux.inra.fr">cvacher@bordeaux.inra.fr</a></p>

                <p>Affiliation:
                  Institut Scientifique de Recherche Agronomique (INRA), UMR1202 Biodiversité Gènes et Communautés, Villenave d'Ornon, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dominique Piou, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institut Scientifique de Recherche Agronomique (INRA), UMR1202 Biodiversité Gènes et Communautés, Villenave d'Ornon, France, 
                  Département de la Santé des Forêts, Ministère de l'Agriculture et de la Pêche, Cenon, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marie-Laure Desprez-Loustau
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut Scientifique de Recherche Agronomique (INRA), UMR1202 Biodiversité Gènes et Communautés, Villenave d'Ornon, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 05, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001740</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001740" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001740" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001740" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001740" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001740-g001" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001740-g002" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001740-t001" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001740-g003" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001740-g004" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001740-g005" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001740">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001740" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">


    <div id="fch_corr" class="fch_corr">
      <a id="corrections" name="corrections" toc="corrections" title= "Corrections (1)"></a>
      <div class="corr_title">
        <span></span>
        <strong>Corrections</strong>
      </div>
      <div class="citation">
          <p>
            <span class="date"> 19 Mar 2008:  </span>
   
  <span class="citation_author">Vacher
    C,
    </span>
   
  <span class="citation_author">Piou
    D,
    </span>
    <span class="citation_author">Desprez-Loustau
    M-L
      </span>

<span class="citation_date">(2008)</span>
<span
    class="citation_article_title">Correction: Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History.</span>
<span class="citation_journal_title">PLoS ONE</span><span
    class="citation_issue"> 3(3):</span>
<span class="citation_start_page">10.1371/annotation/c03a4beb-a3fa-4b78-8724-b3c8ce28c415.</span>
<span class="citation_doi">
 doi: 10.1371/annotation/c03a4beb-a3fa-4b78-8724-b3c8ce28c415</span>
              <span>|</span>
              <a href= "/article/info:doi/10.1371%2Fannotation%2Fc03a4beb-a3fa-4b78-8724-b3c8ce28c415">View correction</a> </span>
          </p>
      </div>
    </div>





<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Compartmentalization and nestedness are common patterns in ecological networks. The aim of this study was to elucidate some of the processes shaping these patterns in a well resolved network of host/pathogen interactions.</p>


<h3>Methology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Based on a long-term (1972–2005) survey of forest health at the regional scale (all French forests; 15 million ha), we uncovered an almost fully connected network of 51 tree taxa and 157 parasitic fungal species. Our analyses revealed that the compartmentalization of the network maps out the ancient evolutionary history of seed plants, but not the ancient evolutionary history of fungal species. The very early divergence of the major fungal phyla may account for this asymmetric influence of past evolutionary history. Unlike compartmentalization, nestedness did not reflect any consistent phylogenetic signal. Instead, it seemed to reflect the ecological features of the current species, such as the relative abundance of tree species and the life-history strategies of fungal pathogens. We discussed how the evolution of host range in fungal species may account for the observed nested patterns.</p>


<h3>Conclusion/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Overall, our analyses emphasized how the current complexity of ecological networks results from the diversification of the species and their interactions over evolutionary times. They confirmed that the current architecture of ecological networks is not only dependant on recent ecological processes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Vacher C, Piou D, Desprez-Loustau M-L (2008) Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History. PLoS ONE 3(3):
          e1740.
            doi:10.1371/journal.pone.0001740</p><p><strong>Academic Editor: </strong>Jerome Chave, Centre National de la Recherche Scientifique, France</p><p><strong>Received:</strong> December 4, 2007; <strong>Accepted:</strong> February 5, 2008; <strong>Published:</strong> March 5, 2008</p><p><strong>Copyright:</strong> © 2008 Vacher et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded by the INRA.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A network is a set of items, called <em>vertices</em> (or <em>nodes</em>), connected by <em>edges</em> <a href="#pone.0001740-Newman1">[1]</a>. Networks have been used to portray the complexity of systems in various fields of research. In ecology, networks are a valuable tool for representing the diversity of species (<em>vertices</em>) and their interactions (<em>edges</em>). They were originally used for studies of predator/prey interactions (food webs), and have more recently been used for studies of long-lasting, ‘intimate’ interactions <a href="#pone.0001740-Combes1">[2]</a> between two sets of species (e.g. plant species and their pollinators or host species and their parasites). Unlike food web networks, plant/pollinator and host/parasite networks have two types of vertices (each type representing one set of species) with edges connecting vertices of unlike type only. These networks are therefore described as <em>bipartite</em> networks <a href="#pone.0001740-Newman1">[1]</a> and are commonly depicted with a two-layer graph or a binary matrix <a href="#pone.0001740-Lewinsohn1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Two patterns have repeatedly been found in bipartite networks of species interactions: nestedness and, to a lesser extent, compartmentalization. A nested network displays both asymmetric specialization—i.e. species with few interactions (‘specialist’ species) preferentially interact with species with many interactions (‘generalist’ species)— and a dense core of interactions created by symmetric interactions between generalist species. Significant nested structures have been found in many bipartite mutualistic networks, including plant/pollinator networks <a href="#pone.0001740-Bascompte1">[4]</a>, plant/seed disperser networks <a href="#pone.0001740-Bascompte1">[4]</a>, ant/plant networks <a href="#pone.0001740-Guimaraes1">[5]</a>, fish cleaning symbiosis <a href="#pone.0001740-Guimaraes2">[6]</a> and anemone fish/anemone networks <a href="#pone.0001740-Ollerton1">[7]</a>. They have also been found in some bipartite antagonistic networks, such as plant/phytophagous insect networks <a href="#pone.0001740-Lewinsohn1">[3]</a>. Recent findings indicate that this pervasive pattern is a relatively robust measure of network structure, less prone to variations of sampling effort in space and time than number of species and links within the network <a href="#pone.0001740-Nielsen1">[8]</a>. The ecological and evolutionary processes having shaped this pattern are currently a matter of debate <a href="#pone.0001740-Vazquez1">[9]</a>–<a href="#pone.0001740-Vazquez2">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Compartmentalization is characterized by recognizable subsets of interacting species, with species more likely to be linked within than across subsets. No consensus on its prevalence in ecological networks has yet been reached <a href="#pone.0001740-Olesen1">[12]</a>. It has been found in a few bipartite networks, including plant/phytophageous insects networks <a href="#pone.0001740-Prado1">[13]</a>, plant/ant networks <a href="#pone.0001740-Fonseca1">[14]</a>, <a href="#pone.0001740-Guimaraes3">[15]</a> and plant/pollinator networks <a href="#pone.0001740-Olesen1">[12]</a>, <a href="#pone.0001740-Dicks1">[16]</a>, and seems more frequent in networks of large size <a href="#pone.0001740-Olesen1">[12]</a>. Several processes have been identified as potentially playing a role in the emergence of compartmentalization. On the ecological timescale, compartmentalization may arise through spatial <a href="#pone.0001740-Krause1">[17]</a> or temporal <a href="#pone.0001740-Dicks1">[16]</a> segregation of the species. Species occurring in the same place, at the same time are more likely to fall into the same compartment, because they have a higher probability of interacting with each other than with species occurring elsewhere or at another time. However, compartmentalization may also reflect more ancient events, such as phylogenetic splits <a href="#pone.0001740-Lewinsohn1">[3]</a>, <a href="#pone.0001740-Fonseca1">[14]</a>, <a href="#pone.0001740-Cattin1">[18]</a>. Typically, a pollinator's diet is constrained by its phylogenetic origin. For example, dipteran pollinators do not have long enough tongues to reach the nectar in tubular flowers, whereas lepidopteran pollinators do. Thus, the insects of the Diptera and Lepidoptera families tend to fall into different compartments, corresponding to open and tubular flowers, respectively <a href="#pone.0001740-Dicks1">[16]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study, we described a well resolved network of host/pathogen interactions in the forest ecosystem. We addressed the following questions: Are there significant compartments and nested structures in this antagonistic web of species? Which ecological and evolutionary processes have shaped these patterns? Based on a long-term (1972–2005) survey of forest health at the regional scale (all French forests; 15 million ha), we uncovered an almost fully connected network of 51 tree taxa and 157 parasitic fungal species. We analysed the influence of several species traits (phylogeny, abundance, distributional range, life-history strategy) on their position in the network architecture. Our analyses showed that there are still signs of the ancient evolutionary history of the species in the current network architecture.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Tree-fungus interaction records</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>For this study, we used a compilation of 11087 records of forest tree diseases caused by parasitic fungi. All the observations originated from the database of the <em>Département Santé des Forêts</em> (DSF) for the 1972–2005 period. DSF is the French governmental organization in charge of forest health monitoring. It consists of a network of skilled foresters evenly covering the state-owned and private forests of the country. The primary aim of the DSF network is to prevent disease spread, pest outbreaks and other types of damage by alerting the authorities as soon as a threat to forest health is identified. The foresters report all types of damage noticed during their daily work in the forest, when they consider that they may reduce the survival or the economical value of trees. Their reports contain the full description of the observed symptoms and, when applicable, the <em>in situ</em> identification of the biotic agent which is responsible for them. When the <em>in situ</em> morphological identification is tricky, samples are sent to the National Plant Protection Laboratory (LNPV) which specializes in the molecular identification of plant pests and pathogens. When the damage of a forest plant has several causes (for instance, an insect attack followed by the spread of a fungal parasite), all of them are recorded and ranked in their order of importance. Hence, pests and pathogens recorded in the database are not only primary pests and pathogens but also secondary or weakness pests and pathogens. Over 67,000 cases of insect attack, fungal disease, abiotic stresses or decline were reported between 1972 and 2005.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>For this study, we only used observations in which fungi had been identified to the species level. We updated fungal species names and corrected species synonyms with the <em>Index Fungorum</em> database (<a href="http://www.indexfungorum.org">www.indexfungorum.org</a>). We obtained a database of 157 fungal species (see <a href="#pone.0001740.s001">Table S1</a> in Supplementary Material) and 51 host taxa (see Table S2), forming 547 interactions. These data will be uploaded to the Interaction Web Database (<a href="http://www.nceas.ucsb.edu/interactionweb/">www.nceas.ucsb.edu/interactionweb/</a>).</p>


<h4>Fungal species traits</h4>
<h5>Phylogeny.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>All the fungal species belonged to Dikarya, which consists of two monophyletic phyla: Ascomycota and Basidiomycota. Ascomycota is the largest phylum and is divided into three monophyletic subphyla: Taphrinomycotina, Saccharomycotina and Pezizomycotina. Basidiomycota is also divided into three subphyla: Pucciniomycotina, Ustilagomycotina and Agaricomycotina <a href="#pone.0001740-James1">[19]</a>. Based on the <em>Index Fungorum</em> database (<a href="http://www.indexfungorum.org">www.indexfungorum.org</a>) and the NCBI taxonomy browser <a href="#pone.0001740-Wheeler1">[20]</a>, we classified the fungal species into phyla and subphyla. Only three species could not be assigned to a subphylum (<a href="#pone.0001740.s001">Table S1</a>).</p>

<h5>Life-history strategy.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>We also classified the fungal species into 10 nutritional types (referred to as ‘life-history strategies’, as suggested by Garcia-Guzman and Morales <a href="#pone.0001740-GarciaGuzman1">[21]</a> based on the parasitic lifestyle (biotroph versus necrotroph) and on the plant organs and tissues attacked: (1) strict foliar necrotroph parasites, (2) canker agents, (3) stem decay fungi, (4) obligate biotroph parasites, (5) root decay fungi, (6) other foliar and twig necrotroph parasites, (7) stem blue stain agents, (8) parasites of fine roots, (9) wilting agents, (10) other root fungi. We included only the five first strategies, which accounted for 87% of the fungal species, in statistical analyses.</p>



<h4>Tree species traits</h4>
<h5>Phylogeny.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>Of the 51 tree taxa, 41 corresponded to true species, 4 corresponded to groups of cultivars belonging to the same genetic continuum and 6 corresponded to groups of several species belonging to the same genus (<a href="#pone.0001740.s002">Table S2</a>). The tree species were equally distributed between two phyla of Division Spermatophyta: the Magnoliophyta and the Coniferophyta. We used the Angiosperm Phylogeny Website <a href="#pone.0001740-Stevens1">[22]</a> to classify the tree taxa further, into seven subphyla (Proteales, Malpighiales, Fabales, Rosales, Fagales, Malvales, Sapindales, Lamiales and Pinales). Moreover, phylogenetic distances between the species belonging to the Magnoliphyta (angiosperms) were estimated by using the Phylomatic software <a href="#pone.0001740-Webb1">[23]</a>. This tool allowed us to create an hypothesis for the phylogenetic relationships among the tree species based on the dated angiosperm supertree of Davies and collaborators <a href="#pone.0001740-Davies1">[24]</a>. The resultant tree was ultrametric, with branch length reflecting estimated time between branching events. Pairwise phylogenetic distances were then extracted by using the <em>cophenetic.phylo</em> function of the R ape package <a href="#pone.0001740-Paradis1">[25]</a>.</p>

<h5>Abundance and sampling intensity.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>An estimate of area covered by each tree taxon (hereafter called abundance) was available (<em>Inventaire Forestier National</em>, 2000 census report). An estimate of the total number of times each tree taxon had been encountered and examined by foresters during their daily work was also available from the DSF database. This number, further referred to as sampling intensity, was expected to be highly correlated with abundance. We assumed that over the long period of the survey (1972–2005) and over the large geographical scale considered here (the entire French territory), the probability for a tree taxa of being damaged was on average equal for all tree taxa. We therefore defined the sampling intensity of a tree taxon as the total number of DSF records for all types of damage (i.e. damage caused by insects, mammals, human activities or abiotic stresses). Damage caused by parasitic fungi were excluded from the calculation in order to obtain an estimate of sampling intensity independent from the data analysed in this study (i.e. tree/fungus interaction records).</p>

<h5>Distributional range.</h5><a id="article1.body1.sec2.sec3.sec3.p1" name="article1.body1.sec2.sec3.sec3.p1"></a><p>The <em>Inventaire Forestier National</em>, the French agency responsible for monitoring forest productivity and composition, has divided the country into 309 geographical units, each of which is homogeneous in terms of its climate, soil and relief. Presence/absence data for all the tree taxa except one (<em>Pinus radiata</em>) were available for these 309 geographical units (<em>Inventaire Forestier National</em>, 2000 census report) and were used to estimate the distributional overlap between tree taxa.</p>



<h4>Detection of compartmentalization and statistical validation</h4>
<h5>Detection.</h5><a id="article1.body1.sec2.sec4.sec1.p1" name="article1.body1.sec2.sec4.sec1.p1"></a><p>We first identified the connected components of the interaction network and characterized them in terms of size, using the <em>clusters</em> function of the R igraph package <a href="#pone.0001740-Csardi1">[26]</a>. The connected components of the network represent, in grossest terms, the pieces of the network: two vertices are in the same component if and only if there is some path between them. We defined the size of a component as the number of its vertices. We then applied the clustering algorithm proposed by Girvan and Newman <a href="#pone.0001740-Girvan1">[27]</a> to the largest component, to highlight its structure. As described by Girvan and Newman <a href="#pone.0001740-Girvan1">[27]</a>, we first calculated betweenness for all edges of the largest component, using the <em>edge.betweenness</em> function of the R igraph package <a href="#pone.0001740-Csardi1">[26]</a>. Betweenness, which was originally defined for graph vertices <a href="#pone.0001740-Freeman1">[28]</a>, <a href="#pone.0001740-Brandes1">[29]</a> and was then extended to graph edges <a href="#pone.0001740-Girvan1">[27]</a>, is approximately equal to the number of shortest paths going through a vertex or an edge. As described by Girvan and Newman <a href="#pone.0001740-Girvan1">[27]</a>, if a network contains clusters that are loosely connected by a small number of edges, then edges connecting clusters have a high edge betweenness, because all shortest paths between different clusters must go along these edges. Edges with the highest betweenness were therefore removed from the graph, and we recalculated betweenness for all the remaining edges <a href="#pone.0001740-Girvan1">[27]</a>. This sequence was repeated until the clusters were separated. Each cluster was then split in turn, starting with the largest. The algorithm was repeated until no edges remained. The nested hierarchy of clusters was converted into a tree format, using the <em>as.phylo.formula</em> function of the R ape package <a href="#pone.0001740-Paradis1">[25]</a>. The hierarchical tree was represented with TreeView <a href="#pone.0001740-Page1">[30]</a>. The order of tree leaves on the hierarchical tree was used to reorder the rows and columns of the interaction matrix.</p>

<h5>Statistical validation.</h5><a id="article1.body1.sec2.sec4.sec2.p1" name="article1.body1.sec2.sec4.sec2.p1"></a><p>The clusters were validated statistically by testing whether element similarity (i.e. similarity between fungal species as a function of host and similarity between host taxa as a function of their pathogens) was significantly higher within than between clusters. We used the multiresponse permutation procedure (MRPP), a non parametric test of differences between predefined groups <a href="#pone.0001740-Mielke1">[31]</a>. The MRPP statistics <em>δ</em> is the weighted mean of within-group means of pair-wise dissimilarity between group elements. As described by Prado and Lewinshon <a href="#pone.0001740-Prado1">[13]</a>, dissimilarity was calculated as a Jaccard distance and group size was taken as the group weight. We used the <em>mrpp</em> function of the R vegan package <a href="#pone.0001740-Oksanen1">[32]</a> to calculate the expected statistics E(<em>δ</em>) if groups were assembled at random. The within-group chance-corrected agreement (A), defined as 1- <em>δ</em>/E(<em>δ</em>), has a maximum of 1 when there is no dissimilarity between the elements of any group. The P-value is the probability of obtaining, by chance, a value of A equal to or larger than the observed value.</p>



<h4>Detection of nestedness and statistical validation</h4>
<h5>Detection.</h5><a id="article1.body1.sec2.sec5.sec1.p1" name="article1.body1.sec2.sec5.sec1.p1"></a><p>Nestedness was defined as N = (100-T)/100, where T is the matrix temperature—a measure of matrix disorder with values ranging from 0° (perfectly nested) to 100° (perfectly non nested) <a href="#pone.0001740-Bascompte1">[4]</a>. Values of N close to 1 therefore indicate a high degree of nestedness. The adjacency matrix was maximally packed to calculate T (see the article by Atmar and Patterson <a href="#pone.0001740-Atmar1">[33]</a> for further details). An isocline of perfect nestedness was then calculated and deviations from this isocline (i.e. unexpected presences and absences of interactions deviating from a perfectly nested pattern) were scored. Matrix temperature T is the average degree of deviation from this isocline. All nestedness analyses were performed with an improved version of Nestedness Calculator software <a href="#pone.0001740-Atmar1">[33]</a> called ANINHADO v.2.03. <a href="#pone.0001740-Guimaraes4">[34]</a>.</p>

<h5>Statistical validation.</h5><a id="article1.body1.sec2.sec5.sec2.p1" name="article1.body1.sec2.sec5.sec2.p1"></a><p>The significance of nestedness was assessed using two null models. Null model I is the null model implemented in Nestedness Calculator software <a href="#pone.0001740-Atmar1">[33]</a>. It assumes that each cell of the interaction matrix has the same probability of being occupied. This probability is estimated as the number of “1s” in the matrix divided by the number of cells. This model generates networks in which differences in the number of interactions between species are small. Hence, deviations from this null model may be due to both differences in the number of interactions between species and an asymmetric distribution of interactions between species. Null model II was developed by Bascompte and collaborators <a href="#pone.0001740-Bascompte1">[4]</a> to cope with this problem. It assumes that the probability of each cell being occupied is the average of the probabilities of occupancy of its row and column. Hence, deviations from this null model result solely from an asymmetric distribution of interactions between species. For each type of null model, a population of 1000 random networks was generated using ANINHADO software <a href="#pone.0001740-Guimaraes4">[34]</a>. The P-value is the probability of a random replicate being at least as nested as the observed matrix.</p>



<h4>Influence of species traits on their position in the compartmentalized network</h4>
<h5>Phylogeny of tree and fungal species.</h5><a id="article1.body1.sec2.sec6.sec1.p1" name="article1.body1.sec2.sec6.sec1.p1"></a><p>We first performed Monte Carlo tests with 10000 replicates, using the <em>chisq.test</em> function of the R stats package <a href="#pone.0001740-R1">[35]</a>, to assess whether the distribution of tree species and fungal species in the different network compartments was random with respect to their phyla and subphyla. In the case of tree species, we also performed the multiresponse permutation procedure (MRPP) to test whether the phylogenetic distance between species was significantly lower within than between compartments, by using the <em>mrpp</em> function of the R vegan package <a href="#pone.0001740-Oksanen1">[32]</a> and by taking group size as the group weight.</p>

<h5>Distributional range of tree species.</h5><a id="article1.body1.sec2.sec6.sec2.p1" name="article1.body1.sec2.sec6.sec2.p1"></a><p>Then we performed the multiresponse permutation procedure (MRPP), by using the <em>mrpp</em> function of the R vegan package <a href="#pone.0001740-Oksanen1">[32]</a>, to test whether the dissimilarity between tree species in their distributional ranges was significantly lower within than between compartments. The dissimilarity between two species in their distributional range was defined as the Jaccard distance between their presence/absence vectors <a href="#pone.0001740-Brandle1">[36]</a>. A distance of zero indicates that the two species have identical ranges, whereas a distance of one indicates that ranges do not overlap at all. As previously, group size was taken as group weight in the permutation procedure.</p>

<h5>Life-history strategies of fungal species.</h5><a id="article1.body1.sec2.sec6.sec3.p1" name="article1.body1.sec2.sec6.sec3.p1"></a><p>Finally we investigated whether tree taxa from different compartments were linked by fungal species having a particular life-history strategy in common, by calculating the number of tree groups linked together by each fungal species and comparing this number for different strategies, using Kruskal-Wallis rank sum tests, implemented with the <em>kruskal.test</em> function of the R stats package <a href="#pone.0001740-R1">[35]</a>.</p>



<h4>Influence of species traits on their position in the nested network</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Species position in the nested network was defined as their rank in the interaction matrix when reorganized for nestedness. We first assessed the relationships between the total number of interactions per species and their rank with Kendall's rank correlation tests, performed with the <em>cor.test</em> function of the R stats package <a href="#pone.0001740-R1">[35]</a>, in order to verify that low-ranked species were those belonging to the dense core of interactions of the nested network. Then we characterized the core tree species by their phylum, abundance and sampling intensity and the core fungal species by their phylum and life-history strategy.</p>
<h5>Phylogeny of tree and fungal species.</h5><a id="article1.body1.sec2.sec7.sec1.p1" name="article1.body1.sec2.sec7.sec1.p1"></a><p>We compared mean species rank as a function of phylum, using Wilcoxon rank sum tests, carried out with the <em>wilcox.test</em> function of the R stats package <a href="#pone.0001740-R1">[35]</a>.</p>

<h5>Abundance and sampling intensity of tree species.</h5><a id="article1.body1.sec2.sec7.sec2.p1" name="article1.body1.sec2.sec7.sec2.p1"></a><p>The relationships between the abundance of a given tree taxon and rank in the nested structures were assessed with Kendall's rank correlation tests, performed with the <em>cor.test</em> function of the R stats package <a href="#pone.0001740-R1">[35]</a>. The same test was performed to assess the relationships between the sampling intensity of a given tree taxon and rank in the nested structures.</p>

<h5>Life-history strategies of fungal species.</h5><a id="article1.body1.sec2.sec7.sec3.p1" name="article1.body1.sec2.sec7.sec3.p1"></a><p>We performed Kruskal-Wallis rank sum tests, implemented with the <em>kruskal.test</em> function of the R stats package <a href="#pone.0001740-R1">[35]</a>, to compare the rank of fungal species in the nested structures for the different life-history strategies.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Most of the parasitic fungal species were highly specialized (57% of the species had only one or two host taxa), but the tree/fungus interaction network was almost fully connected. Only three pair-wise host-parasite interactions were isolated from the largest connected component, which consisted of 48 host taxa, 154 fungal species and 544 interactions (<a href="#pone-0001740-g001">Fig. 1</a>).</p>
<div class="figure" id="pone-0001740-g001"><div class="img"><a name="pone-0001740-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Architecture of the tree-fungus network.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Black circles correspond to tree species whereas white circles correspond to fungal species. The network was drawn with PAJEK software (<a href="http://vlado.fmf.uni-lj.si/pub/networks/pajek/">http://vlado.fmf.uni-lj.si/pub/networks/​pajek/</a>).</p>
<span>doi:10.1371/journal.pone.0001740.g001</span></div>
<h4>Isolated, pair-wise interactions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>One of the three isolated interactions involved yew (<em>Taxus baccata</em>) and the parasite species <em>Phomopsis juniperivora</em>, consistent with previous reports showing that yew is affected by few serious fungal diseases <a href="#pone.0001740-Thomas1">[37]</a>, <a href="#pone.0001740-Brandle2">[38]</a>. Another isolated interaction was that between elm (<em>Ulmus sp.</em>) and <em>Ophiostoma novo-ulmi</em> (grouped with <em>O. ulmi</em>), the causal agent of Dutch elm disease. The dramatic consequences of this disease in Europe <a href="#pone.0001740-Brasier1">[39]</a> may account for foresters involved in forest health monitoring not focusing on the other diseases that occur on elm. The third isolated interaction involved birch (<em>Betula sp.</em>) and the parasite species <em>Taphrina betulina</em>. This isolation resulted from a recording bias. Birch may actually be linked to the main network component through interactions with <em>Armillaria</em> species <a href="#pone.0001740-Phillips1">[40]</a>.</p>


<h4>Compartmentalization</h4>
<h5>Detection and statistical validation.</h5><a id="article1.body1.sec3.sec2.sec1.p1" name="article1.body1.sec3.sec2.sec1.p1"></a><p>Sequential splitting of the largest component of the network with the edge betweenness algorithm <a href="#pone.0001740-Girvan1">[27]</a> led to the identification of six compartments (<a href="#pone-0001740-g002">Fig. 2B</a>), which were validated statistically (<a href="#pone-0001740-t001">Table 1</a>). The first split of the network produced two groups of unequal size. The smallest compartment (C1) contained two host taxa (<em>Populus tremulae</em> and cultivated poplars) and 15 fungal species, whereas the largest group consisted of 46 host taxa and 139 fungal species. The first split of this larger group also yielded two groups of unequal size, the smallest (C2) containing one host taxon (<em>Castanea sativa</em>) and six fungal species. Subsequent splits of the largest group produced compartment C3, consisting of one host taxon (<em>Tilia</em> spp) and two fungal species, and compartment C4, containing two host taxa (<em>Acer</em> spp) and 11 fungal species. The fifth split of the network yielded groups of almost equal size: C5 and C6. C5 was composed of 15 host taxa and 51 fungal species, whereas C6 was composed of 27 host taxa and 69 fungal species. The interaction dissimilarity between fungal species was higher within these two large compartments than within the smaller compartments C1 to C4 (<a href="#pone-0001740-t001">Table 1</a>). We therefore assessed the significance of the fifth network split for fungal species, by excluding the species of compartments C1 to C4. The result remained significant, despite the decrease in the index of within-group agreement (MRPP; A = 0.069; p-value&lt;0.001).</p>
<div class="figure" id="pone-0001740-g002"><div class="img"><a name="pone-0001740-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Structure of the largest connected component of the tree/parasitic fungus interaction network.</span></strong></p><a id="article1.body1.sec3.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec2.sec1.fig1.caption1.p1"></a><p>(A) Matrix ordered for nestedness. (B) Matrix ordered for compartmentalization. Six compartments are aligned on the first diagonal of the matrix. These compartments, C1 to C6, run from the bottom right corner to the upper left corner. C1, C4, C5 and C6 are highlighted in gray. (C) Compartment C5 rearranged for nestedness. (D) Compartment C6 rearranged for nestedness.</p>
<span>doi:10.1371/journal.pone.0001740.g002</span></div><div class="figure" id="pone-0001740-t001"><div class="img"><a name="pone-0001740-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001740.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001740.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Multiresponse permutation procedure (MRPP) analysis of group dissimilarities, showing mean parasite dissimilarity between hosts and mean host dissimilarity between parasites in each compartment of the largest connected component of the tree-parasitic fungi network.</span></strong></p><span>doi:10.1371/journal.pone.0001740.t001</span></div>
<h5>Relationships with the phylogeny of tree species.</h5><a id="article1.body1.sec3.sec2.sec2.p1" name="article1.body1.sec3.sec2.sec2.p1"></a><p>All the host taxa of compartments C1 to C5 belonged to the Magnoliophyta, whereas all but three of the taxa in C6 belonged to the Conipherophyta (<a href="#pone-0001740-g003">Fig. 3</a>). Interestingly, the three host taxa from the Magnoliophyta that fell within the C6 group (<em>Prunus avium</em>, <em>Sorbus domestica</em> and <em>Sorbus torminalis</em>) were all from the Rosaceae family—a well supported branch of the Rosales subphylum <a href="#pone.0001740-Stevens1">[22]</a>. The host taxa of the C1 compartment were the only two host taxa from the Malpighiales subphylum; the host taxon of the C3 compartment was the only host taxon from the Malvales subphylum, and the host taxa of the C4 compartment were the only host taxa of the Sapindales subphylum. Thus, tree taxa were not randomly distributed within the compartments in terms of their phylogenetic origin (Monte Carlo test; phylum: p-value&lt;0.001; subphylum: p-value&lt;0.001). Phylogenetic distance analyses confirmed that the taxa belonging to the Magnoliophyta were not distributed at random within the compartments C1 to C6 (MRPP; A = 0.324; p-value&lt;0.001).</p>
<div class="figure" id="pone-0001740-g003"><div class="img"><a name="pone-0001740-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Hierarchical tree showing the sequential splits of the largest connected component of the tree/parasitic fungus network.</span></strong></p><a id="article1.body1.sec3.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.sec2.fig1.caption1.p1"></a><p>Due to space constraints, only tree taxa are represented. Branches corresponding to the six network compartments (C1 to C6) are indicated. Tree leaf symbols correspond to tree phyla (gray squares: Magnoliophyta; black squares: Conipherophyta). R denotes tree species of the Rosaceae family.</p>
<span>doi:10.1371/journal.pone.0001740.g003</span></div>
<h5>Relationships with the distributional range of tree species.</h5><a id="article1.body1.sec3.sec2.sec3.p1" name="article1.body1.sec3.sec2.sec3.p1"></a><p>The distribution of tree taxa between compartments C1 to C6 in terms of their distributional range was also significantly different from random (MRPP; A = 0.042; p-value = 0.003). The high distributional overlap of the two taxa belonging to the small compartment C4 (<em>Acer</em> species) and, to a lesser extent, that of the two taxa belonging to the small compartment C1 (<em>Populus</em> species) may account for this result (mean Jaccard distance; 0.54, 0.15, 0.75, 0.87 for compartments C1, C4, C5 and C6, respectively). We therefore performed again the analysis, but only for the large compartments C5 and C6. The distribution of tree taxa in terms of their distributional range was then marginally significantly different from random (MRPP; A = 0.007; p-value = 0.087). Finally we excluded the three species belonging to the Rosaceae family from the C6 compartment to investigate whether the emergence of the large compartments of the network may have been driven by the spatial segregation between Conipherophyta and Magnoliophyta. The index of within-group agreement increased and the statistics remained marginally significant (MRPP; A = 0.010; p-value = 0.056).</p>

<h5>Relationships with the phylogeny of fungal species.</h5><a id="article1.body1.sec3.sec2.sec4.p1" name="article1.body1.sec3.sec2.sec4.p1"></a><p>The distribution of fungal species between compartments was random with respect to subphylum (Monte Carlo test; p-value = 0.219), but there was a slight deviation from randomness for phylum (Monte Carlo test; p-value = 0.043), with the small compartments—C1 to C4—having a higher proportion of Ascomycota than would be expected if the distribution were random (90.2% <em>versus</em> 69.5%).</p>

<h5>Relationships with the life-history strategy of fungal species.</h5><a id="article1.body1.sec3.sec2.sec5.p1" name="article1.body1.sec3.sec2.sec5.p1"></a><p>Life-history strategy had a significant effect on the total number of interactions per fungal species (Kruskal-Wallis rank sum tests; χ<sup>2</sup> = 11.5, df = 4, p-value = 0.021) and the number of tree groups linked together by fungal species (Kruskal-Wallis rank sum tests; χ<sup>2</sup> = 21.5, df = 4, p-value&lt;0.001). Root decay fungi had the largest host range: they had the highest mean number of host taxa (<a href="#pone-0001740-g004">Fig. 4A</a>) and linked together the largest number of network compartments (<a href="#pone-0001740-g004">Fig. 4B</a>).</p>
<div class="figure" id="pone-0001740-g004"><div class="img"><a name="pone-0001740-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of the major pathogenic types (C: canker agents, OP: obligate biotroph parasites, RDF: root decay fungi, SDF: stem decay fungi, SF: strict foliar necrotroph parasites).</span></strong></p><a id="article1.body1.sec3.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec2.sec5.fig1.caption1.p1"></a><p>Bars indicate standard errors of the means. (A) For the number of interactions per species. (B) For the number of tree groups linked together. Two tree taxa belong to the same group if they belong to the same network compartment. (C) For the rank in the network's largest connected component after rearrangement for nestedness.</p>
<span>doi:10.1371/journal.pone.0001740.g004</span></div>


<h4>Nestedness</h4>
<h5>Detection and statistical validation.</h5><a id="article1.body1.sec3.sec3.sec1.p1" name="article1.body1.sec3.sec3.sec1.p1"></a><p>The largest component of the network showed significant nestedness (N = 0.900; p-value&lt;0.001 both for null models I and II) (<a href="#pone-0001740-g002">Fig. 2A</a>). Compartment C5 was significantly nested (N = 0.775; p-value&lt;0.001 for null model I; p-value = 0.002 for null model II) (<a href="#pone-0001740-g002">Fig. 2C</a>), as was compartment C6 (N = 0.870; p-value&lt;0.001 both for null models I and II) (<a href="#pone-0001740-g002">Fig. 2D</a>). As expected, the number of interactions per species and their rank in the nested structures were significantly and negatively correlated, both for tree species (Kendall's rank correlation test; τ = −0.90; p-value&lt;0.001 for the network largest component; τ = −0.91; p-value&lt;0.001 for compartment C5; τ = −0.90; p-value&lt;0.001 for compartment C6) and fungal species (Kendall's rank correlation test; τ = −0.86; p-value&lt;0.001 for the network largest component; τ = −0.79; p-value&lt;0.001 for compartment C5; τ = −0.88; p-value&lt;0.001 for compartment C6). Hence, low-ranked species had more interactions than high-ranked species and belonged to the core of the nested structures.</p>

<h5>Relationships with the phylogeny of tree and fungal species.</h5><a id="article1.body1.sec3.sec3.sec2.p1" name="article1.body1.sec3.sec3.sec2.p1"></a><p>The ranks of the tree taxa belonging to Conipherophyta and Magnoliophyta phyla were similar in the largest network component (Wilcoxon rank sum test; W = 291; p-value = 0.959), as were the ranks of the fungal species belonging to the Ascomycota and Basidiomycota phyla (Wilcoxon rank sum test; W = 2914, p-value = 0.117). The ranks of fungal phyla differed slightly between compartments C5 (Wilcoxon rank sum tests; W = 229, p-value = 0.080) and C6 (Wilcoxon rank sum tests; W = 633, p-value = 0.059). In compartment C5, fungal species belonging to the Ascomycota had a tendency to have lower ranks whereas the opposite trend was found in compartment C6.</p>

<h5>Relationships with the abundance and sampling intensity of tree species.</h5><a id="article1.body1.sec3.sec3.sec3.p1" name="article1.body1.sec3.sec3.sec3.p1"></a><p>As expected, the correlation between the abundance and sampling intensity of tree species was significant and positive (Kendall's rank correlation test; τ = 0.59; p-value&lt;0.001). The correlation between the abundance of given tree taxon and rank in the nested matrix (<a href="#pone-0001740-g005">Fig. 5</a>) was significant and negative for the largest component of the network (Kendall's rank correlation test; τ = −0.45; p-value&lt;0.001) and for compartment C6 (Kendall's rank correlation test; τ = −0.68; p-value&lt;0.001). A similar trend was found for compartment C5 (Kendall's rank correlation test; τ = −0.35; p-value = 0.067). Unsurprisingly, the correlation between the sampling intensity of a given tree taxon and rank in the nested matrix was also significant and negative for the largest component of the network (Kendall's rank correlation test; τ = −0.68; p-value&lt;0.001), for compartment C5 (Kendall's rank correlation test; τ = −0.64; p-value&lt;0.001) and for compartment C6 (Kendall's rank correlation test; τ = −0.76; p-value&lt;0.001).</p>
<div class="figure" id="pone-0001740-g005"><div class="img"><a name="pone-0001740-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001740" data-uri="info:doi/10.1371/journal.pone.0001740.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001740.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001740.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001740.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001740.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Relationship between the area covered by tree taxa (log-transformed) and their rank in the nested matrices after rearrangement for nestedness.</span></strong></p><a id="article1.body1.sec3.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.sec3.fig1.caption1.p1"></a><p>Symbols correspond to tree phyla (gray circles: Magnoliophyta; black squares: Conipherophyta). (A) Largest connected component. (B) Compartment C5. (C) Compartment C6.</p>
<span>doi:10.1371/journal.pone.0001740.g005</span></div>
<h5>Relationships with the life-history strategy of fungal species.</h5><a id="article1.body1.sec3.sec3.sec4.p1" name="article1.body1.sec3.sec3.sec4.p1"></a><p>The ranks of the fungal species depended on their life-history strategy, both in the largest network component (Kruskall-Wallis rank sum test; χ<sup>2</sup> = 7.83, df = 4, p-value = 0.097) and in the C6 compartment (Kruskall-Wallis rank sum test; χ<sup>2</sup> = 8.63, df = 4, p-value = 0.071). As expected from their large host range, root decay fungi had lower ranks on average (<a href="#pone-0001740-g004">Fig. 4C</a>).</p>


</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Our analyses revealed that, in French forests, 48 tree taxa (all but 5 identified to species level) and 154 fungal species were fully connected by host/parasite interactions. Consistent with several predictions <a href="#pone.0001740-Guimaraes1">[5]</a>, <a href="#pone.0001740-Bascompte2">[41]</a> concerning the topology of antagonistic webs of species, we found that the tree/parasitic fungus network was significantly compartmentalized. Moreover, as expected from previous studies <a href="#pone.0001740-Lewinsohn1">[3]</a>, <a href="#pone.0001740-Fonseca1">[14]</a>, <a href="#pone.0001740-Cattin1">[18]</a>, we showed that compartmentalization reflected ancient events in species phylogeny. Our analyses showed that this compartmentalization of host/parasite interactions reflected major phylogenetic splits, but only in host phylogeny. All the tree species of the Conipherophyta phylum (gymnosperms) were grouped within a single compartment, whereas the distribution of the remaining species into the other five compartments paralleled the phylogenetic divisions within the Magnoliophyta group (angiosperms). Thus, on a broad evolutionary scale, two tree species with a similar history are likely to share the same set of parasitic fungal species. This finding parallels that of a recent study on plant/pathogen interactions, which showed that the likelihood of a pathogen infecting two plant species decreased continuously with phylogenetic distance between the plants, even to ancient evolutionary distances <a href="#pone.0001740-Gilbert1">[42]</a>. Our results therefore confirm that “the deep evolutionary history of seed plants is mapped by the present day assemblages of exploiters” <a href="#pone.0001740-Brandle1">[36]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>A remarkable result of our study is that major splits in parasite phylogeny (in particular, the split between Ascomycota and Basidiomycota) are hardly reflected in the compartmentalization of the network. Thus, our findings suggest that the deep evolutionary history of the fungal species of Dikarya is not mapped by their present parasitic interactions with tree species. This may be due to the very early divergence of the Ascomycota and Basidiomycota phyla, which are assumed to have separated 400 million years ago <a href="#pone.0001740-Berbee1">[43]</a>, or even earlier <a href="#pone.0001740-Heckman1">[44]</a>. Molecular analyses and fossil records of plant/fungus associations <a href="#pone.0001740-Taylor1">[45]</a> indicate that plants have had to contend with Ascomycota and Basidiomycota ever since their invasion of the land, perhaps 460 million years ago <a href="#pone.0001740-Berbee1">[43]</a>, <a href="#pone.0001740-Selosse1">[46]</a>. Land plants have probably been the main nutrient source for fungi through much of their evolutionary history <a href="#pone.0001740-Berbee1">[43]</a>, <a href="#pone.0001740-Berbee2">[47]</a>, and many different types of nutrition (mutualist, parasite, saprobe) have evolved <a href="#pone.0001740-James1">[19]</a>. Both Ascomycota and Basidiomycota had probably developed parasitic associations with land plants long before the divergence between Magnoliophyta and Conipherotyta, which occurred 140 to 180 million years ago <a href="#pone.0001740-Wikstrom1">[48]</a>, <a href="#pone.0001740-Bell1">[49]</a>. It has even been suggested that the common ancestor of the Basidiomycota was a plant parasite <a href="#pone.0001740-James1">[19]</a>. The observed compartmentalization of the tree/parasitic fungus network may therefore be the result of parasitic fungal species splitting into two groups when the Conipherophyta and Magnoliophyta diverged (both groups containing Ascomycota and Basidiomycota species) and the subsequent coevolution of each set of fungal species with its plant phylum.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>In addition, our results showed that compartmentalization reflected the current distributional range of tree species <a href="#pone.0001740-Brandle1">[36]</a>. Tree species having overlapping distributional ranges had a tendency to belong to the same compartment. In particular, our analyses suggested that the segregation between Conipherophyta and Magnoliophyta in terms of distributional range may have reinforced the pattern of compartmentalization. Other than in western pine plantations, gymnosperm species are dominant only in alpine regions and in some parts of lower mountain ranges. They therefore tend to be associated with climates in which precipitation levels are high and the annual temperature range is very broad. This grouping together of gymnosperm species may have facilitated host jumps between these species <a href="#pone.0001740-Roy1">[50]</a>, and their occurrence in harsh climates may have prevented interactions with cold-sensitive fungal species. This may account for gymnosperm species having their own set of parasitic fungal species, different from that of angiosperm species. Moreover, the wider geographic range of angiosperm species may account for their associations with parasitic fungi being more diverse than those of gymnosperm species (unlike gymnosperm species, angiosperm species fell into different compartments). Analyses of parasitic fungus assemblages in German forests also revealed a similar trend (see <a href="#pone-0001740-g001">Fig. 1</a> and <a href="#pone-0001740-t001">Table 1</a> in the article by Brandle and Brandl <a href="#pone.0001740-Brandle1">[36]</a>). Finally the particular spatial distribution of the Rosaceae tree species—species covering very small total areas but spread across the entire country as isolated trees or small populations—may account for their scattering throughout the network, with three of these species even falling into the compartment containing all the gymnosperm species.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>In addition to significant compartmentalization, we also found significant nestedness, with the largest compartments of the network also being significantly nested. Unlike compartmentalization, nestedness did not reflect any consistent phylogenetic signal. Instead, it seemed to reflect the life-history strategies of fungal species and the current abundance of tree species. We found that root decay fungi formed the core of the nested structures, which suggests that the life-history strategy of these fungal species allowed them to expand their host range. The high saprophytic abilities of these species may account for this result: the ability to survive well without a host could have increased their opportunities for and likelihood of host shifts <a href="#pone.0001740-Parker1">[51]</a>. Moreover, consistent with previous analyses of host-parasite interaction networks <a href="#pone.0001740-Vazquez3">[52]</a>, we found a trend toward the host species in the core of the nested structures being the most abundant species. Hence, specialist fungal species interact preferentially with abundant tree species. Three different explanations have been proposed by Vazquez and Aizen <a href="#pone.0001740-Vazquez1">[9]</a> to account for such an asymmetric pattern of specialization. All of them may apply to the studied tree/fungus network. The first explanation is the selection for specialization on abundant species (because abundant species constitute a more reliable source of reward than rare species with fluctuating populations). Although the relative abundances of tree species changed considerably during the Quaternary Period due to climatic variations <a href="#pone.0001740-Petit1">[53]</a>, the dominant tree taxa (<em>Quercus robur, Quercus petraea, Fagus silvatica, Pinus sylvestris</em>) have remained the same during the last thousand years. It may have been easier for specialist fungal parasites to maintain on these abundant hosts than on rarer hosts. Variation in sampling among tree species is a second explanation for the observed pattern. Indeed, we found that the tree species in the core of the nested structures were the most abundant and consequently the most frequently encountered and sampled. This is because DSF foresters report tree/fungus interactions observed during their daily work in the forest, and encounter abundant tree species more often than rare tree species. Rare fungal species had therefore a higher probability to be observed on abundant tree species than on rare tree species. This could account for fungal species having seemingly few interactions preferentially interacting with the tree species currently most abundant in France. The third explanation is that the same kind of sampling bias occurs in nature: fungal “ parasites ‘sample’ abundant hosts more often than rare ones” <a href="#pone.0001740-Vazquez3">[52]</a>, similarly to DSF foresters. Consequently, abundant tree species interact with a higher number of parasite species than rare tree species. This third explanation is based on the assumption of ecological neutrality at the individual level (i.e. interactions between individuals occur at random). Models based on this assumption generated patterns of specialization which were closed to the observed patterns for several bipartite networks <a href="#pone.0001740-Vazquez1">[9]</a>, <a href="#pone.0001740-Vazquez3">[52]</a>.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Overall, our analyses emphasized how the current complexity of ecological networks results from the diversification of the species and their interactions over evolutionary times. They confirmed that the current architecture of ecological networks is not only dependant on recent ecological processes <a href="#pone.0001740-Ives1">[54]</a>, <a href="#pone.0001740-Rezende1">[55]</a>. Compartmentalization analyses suggested that the current architecture of the tree/parasitic fungus network results mainly from ancient speciation events in seed plants. Ancient speciation events in fungi were hardly reflected in the network architecture. Such asymmetries in the phylogenetic signal have recently been found in several plant-animal mutualistic networks <a href="#pone.0001740-Rezende1">[55]</a> and in one host-parasitoid network <a href="#pone.0001740-Ives1">[54]</a>. Here we proposed that the very early divergence of the major fungal phyla may account for this asymmetrical influence of past evolutionary history. Nestedness analyses suggested that the network architecture has also been shaped by the evolution of host range in fungal species. The influence of these evolutionary processes (i.e. speciation, host range evolution) on the network architecture will be compared to the influence of human-induced changes in a next study. In particular, we will investigate the extent to which the species introduced by human activities during the last centuries (c.a. 30 species equally distributed between trees and fungi) have altered the network architecture.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001740"></div><a name="pone.0001740.s001" id="pone.0001740.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001740.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of parasitic fungal species</p>
<p class="siDoi">doi:10.1371/journal.pone.0001740.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.35 MB PDF)</p>
<a name="pone.0001740.s002" id="pone.0001740.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001740.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">List of forest tree taxa</p>
<p class="siDoi">doi:10.1371/journal.pone.0001740.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.13 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the <em>Département Santé des Forêts</em> (DSF) for allowing us to use their database. We thank Emmanuelle Helion for updating fungal species names. We also thank François Delmotte and two anonymous reviewers for helpful comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Analyzed the data: CV. Contributed reagents/materials/analysis tools: CV MD DP. Wrote the paper: CV MD DP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001740-Newman1" id="pone.0001740-Newman1"></a>Newman MEJ (2003) The structure and function of complex networks. SIAM Review  45: 167–256.  <ul class="find" data-citedArticleID="1068942" data-doi="10.1137/s003614450342480"><li><a href="http://dx.doi.org/10.1137/s003614450342480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+and+function+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+and+function+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001740-Combes1" id="pone.0001740-Combes1"></a>Combes C (2001) Parasitism: Ecology and Evolution of Intimate Interactions. Chicago: University Press Chicago.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001740-Lewinsohn1" id="pone.0001740-Lewinsohn1"></a>Lewinsohn TM, Prado PI, Jordano P, Bascompte J, Olesen JM (2006) Structure in plant-animal interaction assemblages. Oikos  113: 174–184.  <ul class="find" data-citedArticleID="1068938" data-doi="10.1111/j.0030-1299.2006.14583.x"><li><a href="http://dx.doi.org/10.1111/j.0030-1299.2006.14583.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+in+plant-animal+interaction+assemblages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+in+plant-animal+interaction+assemblages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001740-Bascompte1" id="pone.0001740-Bascompte1"></a>Bascompte J, Jordano P, Melian CJ, Olesen JM (2003) The nested assembly of plant-animal mutualistic networks. PNAS  100: 9383–9387.  <ul class="find" data-citedArticleID="1068884" data-doi="10.1073/pnas.1633576100"><li><a href="http://dx.doi.org/10.1073/pnas.1633576100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nested+assembly+of+plant-animal+mutualistic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nested+assembly+of+plant-animal+mutualistic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001740-Guimaraes1" id="pone.0001740-Guimaraes1"></a>Guimaraes PR, Rico-Gray V, dos Reis SF, Thompson JN (2006) Asymmetries in specialization in ant-plant mutualistic networks. Proc R Soc B-Biol Sci  273: 2041–2047.  <ul class="find" data-citedArticleID="1068922" data-doi="10.1098/rspb.2006.3548"><li><a href="http://dx.doi.org/10.1098/rspb.2006.3548" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asymmetries+in+specialization+in+ant-plant+mutualistic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asymmetries+in+specialization+in+ant-plant+mutualistic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001740-Guimaraes2" id="pone.0001740-Guimaraes2"></a>Guimaraes PR, Sazima C, dos Reis SF, Sazima I (2007) The nested structure of marine cleaning symbiosis: is it like flowers and bees? Biol Lett  3: 51–54.  <ul class="find" data-citedArticleID="1068924" data-doi="10.1098/rsbl.2006.0562"><li><a href="http://dx.doi.org/10.1098/rsbl.2006.0562" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nested+structure+of+marine+cleaning+symbiosis%3A+is+it+like+flowers+and+bees%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nested+structure+of+marine+cleaning+symbiosis%3A+is+it+like+flowers+and+bees%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001740-Ollerton1" id="pone.0001740-Ollerton1"></a>Ollerton J, McCollin D, Fautin DG, Allen GR (2007) Finding NEMO: nestedness engendered by mutualistic organization in anemonefish and their hosts. Proc R B-Biol Sci  274: 591–598.  <ul class="find" data-citedArticleID="1068950" data-doi="10.1098/rspb.2006.3758"><li><a href="http://dx.doi.org/10.1098/rspb.2006.3758" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+NEMO%3A+nestedness+engendered+by+mutualistic+organization+in+anemonefish+and+their+hosts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+NEMO%3A+nestedness+engendered+by+mutualistic+organization+in+anemonefish+and+their+hosts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001740-Nielsen1" id="pone.0001740-Nielsen1"></a>Nielsen A, Bascompte J (2007) Ecological networks nestedness and sampling effort. Journal of Ecology  95: 1134–1141.  <ul class="find" data-citedArticleID="1068944" data-doi="10.1111/j.1365-2745.2007.01271.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2745.2007.01271.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+networks+nestedness+and+sampling+effort." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+networks+nestedness+and+sampling+effort.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001740-Vazquez1" id="pone.0001740-Vazquez1"></a>Vazquez DP, Aizen MA (2004) Asymmetric specialization: A pervasive feature of plant-pollinator interactions. Ecology  85: 1251–1257.  <ul class="find" data-citedArticleID="1068980" data-doi="10.1890/03-3112"><li><a href="http://dx.doi.org/10.1890/03-3112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asymmetric+specialization%3A+A+pervasive+feature+of+plant-pollinator+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asymmetric+specialization%3A+A+pervasive+feature+of+plant-pollinator+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001740-Santamaria1" id="pone.0001740-Santamaria1"></a>Santamaria L, Rodriguez-Girones MA (2007) Linkage rules for plant-pollinator networks: trait complementarity or exploitation barriers? PLoS Biol  5: e31.  <ul class="find" data-citedArticleID="1068970" data-doi="10.1371/journal.pbio.0050031"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0050031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+rules+for+plant-pollinator+networks%3A+trait+complementarity+or+exploitation+barriers%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+rules+for+plant-pollinator+networks%3A+trait+complementarity+or+exploitation+barriers%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001740-Vazquez2" id="pone.0001740-Vazquez2"></a>Vazquez DP, Melian CJ, Williams NM, Bluthgen N, Krasnov BR, et al.  (2007) Species abundance and asymmetric interaction strength in ecological networks. Oikos  116: 1120–1127.  <ul class="find" data-citedArticleID="1068982" data-doi="10.1111/j.2007.0030-1299.15828.x"><li><a href="http://dx.doi.org/10.1111/j.2007.0030-1299.15828.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species+abundance+and+asymmetric+interaction+strength+in+ecological+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species+abundance+and+asymmetric+interaction+strength+in+ecological+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001740-Olesen1" id="pone.0001740-Olesen1"></a>Olesen JM, Bascompte J, Dupont YL, Jordano P (2007) The modularity of pollination networks. PNAS  104: 19891–19896.  <ul class="find" data-citedArticleID="1068948" data-doi="10.1073/pnas.0706375104"><li><a href="http://dx.doi.org/10.1073/pnas.0706375104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+modularity+of+pollination+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+modularity+of+pollination+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001740-Prado1" id="pone.0001740-Prado1"></a>Prado PI, Lewinsohn TM (2004) Compartments in insect-plant associations and their consequences for community structure. J Anim Ecol  73: 1168–1178.  <ul class="find" data-citedArticleID="1068962" data-doi="10.1111/j.0021-8790.2004.00891.x"><li><a href="http://dx.doi.org/10.1111/j.0021-8790.2004.00891.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compartments+in+insect-plant+associations+and+their+consequences+for+community+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compartments+in+insect-plant+associations+and+their+consequences+for+community+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001740-Fonseca1" id="pone.0001740-Fonseca1"></a>Fonseca CR, Ganade G (1996) Asymmetries compartments and null interactions in an Amazonian ant-plant community. J Anim Ecol  65: 339–347.  <ul class="find" data-citedArticleID="1068912" data-doi="10.2307/5880"><li><a href="http://dx.doi.org/10.2307/5880" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asymmetries+compartments+and+null+interactions+in+an+Amazonian+ant-plant+community." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asymmetries+compartments+and+null+interactions+in+an+Amazonian+ant-plant+community.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001740-Guimaraes3" id="pone.0001740-Guimaraes3"></a>Guimaraes PR Jr, Rico-Gray V, Oliveira PS, Izzo PJ, dos Reis SF, et al.  (2007) Interaction intimacy affects structure and coevolutionary dynamics in mutualistic networks. Curr Biol  17: 1797–803.  <ul class="find" data-citedArticleID="1068926" data-doi="10.1016/j.cub.2007.09.059"><li><a href="http://dx.doi.org/10.1016/j.cub.2007.09.059" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+intimacy+affects+structure+and+coevolutionary+dynamics+in+mutualistic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+intimacy+affects+structure+and+coevolutionary+dynamics+in+mutualistic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001740-Dicks1" id="pone.0001740-Dicks1"></a>Dicks LV, Corbet SA, Pywell RF (2002) Compartmentalization in plant-insect flower visitor webs. J Anim Ecol  71: 32–43.  <ul class="find" data-citedArticleID="1068910" data-doi="10.1046/j.0021-8790.2001.00572.x"><li><a href="http://dx.doi.org/10.1046/j.0021-8790.2001.00572.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compartmentalization+in+plant-insect+flower+visitor+webs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compartmentalization+in+plant-insect+flower+visitor+webs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001740-Krause1" id="pone.0001740-Krause1"></a>Krause AE, Frank KA, Mason DM, Ulanowicz RE, Taylor WW (2003) Compartments revealed in food-web structure. Nature  426: 282–285.  <ul class="find" data-citedArticleID="1068936" data-doi="10.1038/nature02115"><li><a href="http://dx.doi.org/10.1038/nature02115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compartments+revealed+in+food-web+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compartments+revealed+in+food-web+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001740-Cattin1" id="pone.0001740-Cattin1"></a>Cattin MF, Bersier LF, Banasek-Richter C, Baltensperger R, Gabriel JP (2004) Phylogenetic constraints and adaptation explain food-web structure. Nature  427: 835–839.  <ul class="find" data-citedArticleID="1068902" data-doi="10.1038/nature02327"><li><a href="http://dx.doi.org/10.1038/nature02327" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+constraints+and+adaptation+explain+food-web+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+constraints+and+adaptation+explain+food-web+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001740-James1" id="pone.0001740-James1"></a>James TY, Kauff F, Schoch CL, Matheny PB, Hofstetter V, et al.  (2006) Reconstructing the early evolution of Fungi using a six-gene phylogeny. Nature  443: 818–822.  <ul class="find" data-citedArticleID="1068934" data-doi="10.1038/nature05110"><li><a href="http://dx.doi.org/10.1038/nature05110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+the+early+evolution+of+Fungi+using+a+six-gene+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+the+early+evolution+of+Fungi+using+a+six-gene+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001740-Wheeler1" id="pone.0001740-Wheeler1"></a>Wheeler DL, Chappey C, Lash AE, Leipe DD, Madden TL, et al.  (2000) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  28: 10–14.  <ul class="find" data-citedArticleID="1068988" data-doi="10.1093/nar/28.1.10"><li><a href="http://dx.doi.org/10.1093/nar/28.1.10" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001740-GarciaGuzman1" id="pone.0001740-GarciaGuzman1"></a>Garcia-Guzman G, Morales E (2007) Life-history strategies of plant pathogens: Distribution patterns and phylogenetic analysis. Ecology  88: 589–596.  <ul class="find" data-citedArticleID="1068916" data-doi="10.1890/05-1174"><li><a href="http://dx.doi.org/10.1890/05-1174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life-history+strategies+of+plant+pathogens%3A+Distribution+patterns+and+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life-history+strategies+of+plant+pathogens%3A+Distribution+patterns+and+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001740-Stevens1" id="pone.0001740-Stevens1"></a>Stevens PF (2001 onwards) Angiosperm Phylogeny Website Version 7 May 2006.   URL <a href="http://wwwmobotorg/MOBOT/research/APweb/">http://wwwmobotorg/MOBOT/research/APweb/</a>. <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001740-Webb1" id="pone.0001740-Webb1"></a>Webb CO, Donoghue MJ (2005) Phylomatic: tree assembly for applied phylogenetics. Mol Ecol Notes  5: 181–183.  <ul class="find" data-citedArticleID="1068986" data-doi="10.1111/j.1471-8286.2004.00829.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2004.00829.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylomatic%3A+tree+assembly+for+applied+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylomatic%3A+tree+assembly+for+applied+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001740-Davies1" id="pone.0001740-Davies1"></a>Davies TJ, Barraclough TG, Chase MW, Soltis PS, Soltis DE, et al.  (2004) Darwin's abominable mystery: Insights from a supertree of the angiosperms. PNAS  101: 1904–1909.  <ul class="find" data-citedArticleID="1068908" data-doi="10.1073/pnas.0308127100"><li><a href="http://dx.doi.org/10.1073/pnas.0308127100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Darwin%27s+abominable+mystery%3A+Insights+from+a+supertree+of+the+angiosperms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Darwin%27s+abominable+mystery%3A+Insights+from+a+supertree+of+the+angiosperms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001740-Paradis1" id="pone.0001740-Paradis1"></a>Paradis E, Strimmer K, Claude J, Jobb G, Opgen-Rhein R, et al.  (2006) ape: Analyses of Phylogenetics and Evolution version 18-5.   R package URL <a href="http://wwwR-projectorg">http://wwwR-projectorg</a>. <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001740-Csardi1" id="pone.0001740-Csardi1"></a>Csardi G (2005) igraph: Routines for network analysis R package.   URL <a href="http://wwwR-projectorg">http://wwwR-projectorg</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0001740-Girvan1" id="pone.0001740-Girvan1"></a>Girvan M, Newman MEJ (2002) Community structure in social and biological networks. PNAS  99: 7821–7826.  <ul class="find" data-citedArticleID="1068920" data-doi="10.1073/pnas.122653799"><li><a href="http://dx.doi.org/10.1073/pnas.122653799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Community+structure+in+social+and+biological+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Community+structure+in+social+and+biological+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001740-Freeman1" id="pone.0001740-Freeman1"></a>Freeman LC (1979) Centrality in Social Networks I: Conceptual Clarification. Social Networks  1: 215–239.  <ul class="find" data-citedArticleID="1068914" data-doi="10.1016/0378-8733(78)90021-7"><li><a href="http://dx.doi.org/10.1016/0378-8733(78)90021-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Centrality+in+Social+Networks+I%3A+Conceptual+Clarification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Centrality+in+Social+Networks+I%3A+Conceptual+Clarification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001740-Brandes1" id="pone.0001740-Brandes1"></a>Brandes U (2001) A faster algorithm for betweenness centrality. Journal of Mathematical Sociology  25: 163–177.  <ul class="find" data-citedArticleID="1068894" data-doi="10.1080/0022250x.2001.9990249"><li><a href="http://dx.doi.org/10.1080/0022250x.2001.9990249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+faster+algorithm+for+betweenness+centrality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+faster+algorithm+for+betweenness+centrality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001740-Page1" id="pone.0001740-Page1"></a>Page RDM (1996) TREEVIEW: An application to display phylogenetic trees on personal computers. Computer Applications in the Biosciences  12: 357–358.  <ul class="find" data-citedArticleID="1068952" data-doi="10.1093/bioinformatics/12.4.357"><li><a href="http://dx.doi.org/10.1093/bioinformatics/12.4.357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREEVIEW%3A+An+application+to+display+phylogenetic+trees+on+personal+computers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREEVIEW%3A+An+application+to+display+phylogenetic+trees+on+personal+computers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001740-Mielke1" id="pone.0001740-Mielke1"></a>Mielke PW, Berry KJ (2001) Permutation Methods: A Distance Function Approach. New York: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001740-Oksanen1" id="pone.0001740-Oksanen1"></a>Oksanen J, Kindt R, Legendre P, O'Hara B (2006) vegan: Community Ecology Package version 18-3.   R package URL <a href="http://www.R-project.org">http://www.R-project.org</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0001740-Atmar1" id="pone.0001740-Atmar1"></a>Atmar W, Patterson BD (1993) The measure of order and disorder in the distribution of species in fragmented habitat. Oecologia  96: 373–382.  <ul class="find" data-citedArticleID="1068882" data-doi="10.1007/bf00317508"><li><a href="http://dx.doi.org/10.1007/bf00317508" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+measure+of+order+and+disorder+in+the+distribution+of+species+in+fragmented+habitat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+measure+of+order+and+disorder+in+the+distribution+of+species+in+fragmented+habitat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001740-Guimaraes4" id="pone.0001740-Guimaraes4"></a>Guimaraes PR, Guimaraes P (2006) Improving the analyses of nestedness for large sets of matrices. Environ Modell Soft  21: 1512–1513.  <ul class="find" data-citedArticleID="1068928" data-doi="10.1016/j.envsoft.2006.04.002"><li><a href="http://dx.doi.org/10.1016/j.envsoft.2006.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improving+the+analyses+of+nestedness+for+large+sets+of+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improving+the+analyses+of+nestedness+for+large+sets+of+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001740-R1" id="pone.0001740-R1"></a>R Development Core Team (2006) R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria, ISBN 3-900051-07-0.   URL <a href="http://wwwR-projectorg">http://wwwR-projectorg</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001740-Brandle1" id="pone.0001740-Brandle1"></a>Brandle M, Brandl R (2006) Is the composition of phytophagous insects and parasitic fungi among trees predictable? Oikos  113: 296–304.  <ul class="find" data-citedArticleID="1068896" data-doi="10.1111/j.2006.0030-1299.14418.x"><li><a href="http://dx.doi.org/10.1111/j.2006.0030-1299.14418.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+the+composition+of+phytophagous+insects+and+parasitic+fungi+among+trees+predictable%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+the+composition+of+phytophagous+insects+and+parasitic+fungi+among+trees+predictable%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001740-Thomas1" id="pone.0001740-Thomas1"></a>Thomas PA, Polwart A (2003) Taxus baccata L. J Ecol  91(3): 489–524.  <ul class="find" data-citedArticleID="1068978" data-doi="10.1046/j.1365-2745.2003.00783.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2745.2003.00783.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taxus+baccata+L." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Taxus+baccata+L.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001740-Brandle2" id="pone.0001740-Brandle2"></a>Brandle M, Brandl R (2003) Species richness on trees: a comparison of parasitic fungi and insects. Evol Ecol Res  5: 941–952.  <ul class="find" data-citedArticleID="1068898"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Species+richness+on+trees%3A+a+comparison+of+parasitic+fungi+and+insects.&amp;auth=&amp;atitle=Species+richness+on+trees%3A+a+comparison+of+parasitic+fungi+and+insects." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species+richness+on+trees%3A+a+comparison+of+parasitic+fungi+and+insects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species+richness+on+trees%3A+a+comparison+of+parasitic+fungi+and+insects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001740-Brasier1" id="pone.0001740-Brasier1"></a>Brasier CM, Buck W (2001) Rapid evolutionary changes in a globally invading fungal pathogen (Dutch elm disease). Biol Invasions  3: 223–233.  <ul class="find" data-citedArticleID="1068900"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rapid+evolutionary+changes+in+a+globally+invading+fungal+pathogen+%28Dutch+elm+disease%29.&amp;auth=&amp;atitle=Rapid+evolutionary+changes+in+a+globally+invading+fungal+pathogen+%28Dutch+elm+disease%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+evolutionary+changes+in+a+globally+invading+fungal+pathogen+%28Dutch+elm+disease%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+evolutionary+changes+in+a+globally+invading+fungal+pathogen+%28Dutch+elm+disease%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001740-Phillips1" id="pone.0001740-Phillips1"></a>Phillips DH, Burdekin DA (1982) Diseases of Forest and Ornamental Trees. London: The Macmillan Press.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001740-Bascompte2" id="pone.0001740-Bascompte2"></a>Bascompte J, Jordano P (2006)  The structure of plant-animal mutualistic networks. In: Pascual M, Dunne J, editors. Ecological Networks. Oxford: Oxford University Press. pp. 143–149.  <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001740-Gilbert1" id="pone.0001740-Gilbert1"></a>Gilbert GS, Webb CO (2007) Phylogenetic signal in plant pathogen-host range. PNAS  104: 4979–4983.  <ul class="find" data-citedArticleID="1068918" data-doi="10.1073/pnas.0607968104"><li><a href="http://dx.doi.org/10.1073/pnas.0607968104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+signal+in+plant+pathogen-host+range." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+signal+in+plant+pathogen-host+range.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001740-Berbee1" id="pone.0001740-Berbee1"></a>Berbee ML, Taylor JW (2001)  Fungal molecular evolution: gene trees and geologic time. In: McLaughlin D, McLaughlin E, Lemke P, editors. The Mycota. Berlin: Springer-Verlag. pp. 229–245.  <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0001740-Heckman1" id="pone.0001740-Heckman1"></a>Heckman DS, Geiser DM, Eidell BR, Stauffer RL, Kardos NL, et al.  (2001) Molecular evidence for the early colonization of land by fungi and plants. Science  293: 1129–1133.  <ul class="find" data-citedArticleID="1068930" data-doi="10.1126/science.1061457"><li><a href="http://dx.doi.org/10.1126/science.1061457" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+for+the+early+colonization+of+land+by+fungi+and+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+for+the+early+colonization+of+land+by+fungi+and+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001740-Taylor1" id="pone.0001740-Taylor1"></a>Taylor TN, Hass T, Kerp H (1999) The oldest fossil ascomycetes. Nature  399: 648–648.  <ul class="find" data-citedArticleID="1068976" data-doi="10.1038/21349"><li><a href="http://dx.doi.org/10.1038/21349" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+oldest+fossil+ascomycetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+oldest+fossil+ascomycetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001740-Selosse1" id="pone.0001740-Selosse1"></a>Selosse MA, Le Tacon F (1998) The land flora: a phototroph-fungus partnership? TREE  13: 15–20.  <ul class="find" data-citedArticleID="1068972" data-doi="10.1016/s0169-5347(97)01230-5"><li><a href="http://dx.doi.org/10.1016/s0169-5347(97)01230-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+land+flora%3A+a+phototroph-fungus+partnership%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+land+flora%3A+a+phototroph-fungus+partnership%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001740-Berbee2" id="pone.0001740-Berbee2"></a>Berbee ML (2001) The phylogeny of plant and animal pathogens in the Ascomycota. Physiol Mol Plant Pathol  59: 165–187.  <ul class="find" data-citedArticleID="1068892"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+phylogeny+of+plant+and+animal+pathogens+in+the+Ascomycota.&amp;auth=&amp;atitle=The+phylogeny+of+plant+and+animal+pathogens+in+the+Ascomycota." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phylogeny+of+plant+and+animal+pathogens+in+the+Ascomycota." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phylogeny+of+plant+and+animal+pathogens+in+the+Ascomycota.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001740-Wikstrom1" id="pone.0001740-Wikstrom1"></a>Wikstrom N, Savolainen V, Chase MW (2001) Evolution of the angiosperms: calibrating the family tree. Proc R Soc B-Biol Sci  268: 2211–2220.  <ul class="find" data-citedArticleID="1068990" data-doi="10.1098/rspb.2001.1782"><li><a href="http://dx.doi.org/10.1098/rspb.2001.1782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+angiosperms%3A+calibrating+the+family+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+angiosperms%3A+calibrating+the+family+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001740-Bell1" id="pone.0001740-Bell1"></a>Bell CD, Soltis DE, Soltis PS (2005) The age of the angiosperms: A molecular timescale without a clock. Evolution  59: 1245–1258.  <ul class="find" data-citedArticleID="1068888" data-doi="10.1554/05-005"><li><a href="http://dx.doi.org/10.1554/05-005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+age+of+the+angiosperms%3A+A+molecular+timescale+without+a+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+age+of+the+angiosperms%3A+A+molecular+timescale+without+a+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001740-Roy1" id="pone.0001740-Roy1"></a>Roy BA (2001) Patterns of association between crucifers and their flower-mimic pathogens: Host jumps are more common than coevolution or cospeciation. Evolution  55: 41–53.  <ul class="find" data-citedArticleID="1068968" data-doi="10.1554/0014-3820(2001)055[0041:poabca]2.0.co;2"><li><a href="http://dx.doi.org/10.1554/0014-3820(2001)055[0041:poabca]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+association+between+crucifers+and+their+flower-mimic+pathogens%3A+Host+jumps+are+more+common+than+coevolution+or+cospeciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+association+between+crucifers+and+their+flower-mimic+pathogens%3A+Host+jumps+are+more+common+than+coevolution+or+cospeciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001740-Parker1" id="pone.0001740-Parker1"></a>Parker IM, Gilbert GS (2004) The evolutionary ecology of novel plant-pathogen interactions. Ann Rev Ecol Evol Syst  35: 675–700.  <ul class="find" data-citedArticleID="1068956" data-doi="10.1146/annurev.ecolsys.34.011802.132339"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.34.011802.132339" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+ecology+of+novel+plant-pathogen+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+ecology+of+novel+plant-pathogen+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001740-Vazquez3" id="pone.0001740-Vazquez3"></a>Vazquez DP, Poulin R, Krasnov BR, Shenbrot GI (2005) Species abundance and the distribution of specialization in host-parasite interaction networks. J Anim Ecol  74: 946–955.  <ul class="find" data-citedArticleID="1068984" data-doi="10.1111/j.1365-2656.2005.00992.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2656.2005.00992.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species+abundance+and+the+distribution+of+specialization+in+host-parasite+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species+abundance+and+the+distribution+of+specialization+in+host-parasite+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001740-Petit1" id="pone.0001740-Petit1"></a>Petit RJ, Aguinagalde I, de Beaulieu JL, Bittkau C, Brewer S, et al.  (2003) Glacial refugia: Hotspots but not melting pots of genetic diversity. Science  300: 1563–1565.  <ul class="find" data-citedArticleID="1068958" data-doi="10.1126/science.1083264"><li><a href="http://dx.doi.org/10.1126/science.1083264" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glacial+refugia%3A+Hotspots+but+not+melting+pots+of+genetic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glacial+refugia%3A+Hotspots+but+not+melting+pots+of+genetic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001740-Ives1" id="pone.0001740-Ives1"></a>Ives AR, Godfray HCJ (2006) Phylogenetic analysis of trophic associations. Am Nat  168: E1–E14.  <ul class="find" data-citedArticleID="1068932" data-doi="10.1086/505157"><li><a href="http://dx.doi.org/10.1086/505157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+trophic+associations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+trophic+associations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001740-Rezende1" id="pone.0001740-Rezende1"></a>Rezende EL, Lavabre JE, Guimaraes PR, Jordano P, Bascompte J (2007) Non-random coextinctions in phylogenetically structured mutualistic networks. Nature  448: 925–U6.  <ul class="find" data-citedArticleID="1068966" data-doi="10.1038/nature05956"><li><a href="http://dx.doi.org/10.1038/nature05956" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-random+coextinctions+in+phylogenetically+structured+mutualistic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-random+coextinctions+in+phylogenetically+structured+mutualistic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.XML" value="104486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.PDF" value="400605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g001.PNG_L" value="2326625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g001.PNG_M" value="468254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g001.PNG_S" value="13862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g001.TIF" value="3199442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g001.PNG_I" value="148322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g002.PNG_L" value="805913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g002.PNG_M" value="98441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g002.PNG_S" value="9860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g002.TIF" value="931128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g002.PNG_I" value="30051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.t001.PNG_L" value="101839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.t001.PNG_M" value="61198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.t001.PNG_S" value="10035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.t001.TIF" value="547486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.t001.PNG_I" value="19067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g003.PNG_L" value="749965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g003.PNG_M" value="126683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g003.PNG_S" value="11367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g003.TIF" value="1413140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g003.PNG_I" value="59996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g004.PNG_L" value="92717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g004.PNG_M" value="24260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g004.PNG_S" value="9289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g004.TIF" value="181286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g004.PNG_I" value="27224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g005.PNG_L" value="180536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g005.PNG_M" value="28143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g005.PNG_S" value="10136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g005.TIF" value="276496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.g005.PNG_I" value="55593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.s001.PDF" value="353765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001740.s002.PDF" value="129619"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001740&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001740" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001740&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001740&volume=&issue=&title=Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History&author_name=Corinne%20Vacher%2C%20Dominique%20Piou%2C%20Marie-Laure%20Desprez-Loustau&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001740" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001740" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001740" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001740&amp;t=Architecture%20of%20an%20Antagonistic%20Tree%2FFungus%20Network%3A%20The%20Asymmetric%20Influence%20of%20Past%20Evolutionary%20History" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001740&title=Architecture%20of%20an%20Antagonistic%20Tree%2FFungus%20Network%3A%20The%20Asymmetric%20Influence%20of%20Past%20Evolutionary%20History&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001740&amp;title=Architecture%20of%20an%20Antagonistic%20Tree%2FFungus%20Network%3A%20The%20Asymmetric%20Influence%20of%20Past%20Evolutionary%20History" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001740" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001740&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001740';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Architecture%20of%20an%20Antagonistic%20Tree%2FFungus%20Network%3A%20The%20Asymmetric%20Influence%20of%20Past%20Evolutionary%20History http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001740" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001740" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->
<div class="block">
    <div class="header">
        <h3>Related PLOS Articles</h3>
    </div>
      <p><a href = "/article/info:doi/10.1371/annotation/c03a4beb-a3fa-4b78-8724-b3c8ce28c415" title="Read Open-Access Article">Correction: Architecture of an Antagonistic Tree/Fungus Network: The Asymmetric Influence of Past Evolutionary History</a></p>
</div>


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+ecology%22" title="Search for articles in the subject area:'Evolutionary ecology'"><div class="flagText">Evolutionary ecology</div></a>
              <div data-categoryid="48159" data-articleid="26304"
                   data-categoryname="Evolutionary ecology"
                   class="flagImage" title="Flag 'Evolutionary ecology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+structure%22" title="Search for articles in the subject area:'Fungal structure'"><div class="flagText">Fungal structure</div></a>
              <div data-categoryid="19535" data-articleid="26304"
                   data-categoryname="Fungal structure"
                   class="flagImage" title="Flag 'Fungal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungi%22" title="Search for articles in the subject area:'Fungi'"><div class="flagText">Fungi</div></a>
              <div data-categoryid="43047" data-articleid="26304"
                   data-categoryname="Fungi"
                   class="flagImage" title="Flag 'Fungi' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Host-pathogen+interactions%22" title="Search for articles in the subject area:'Host-pathogen interactions'"><div class="flagText">Host-pathogen interactions</div></a>
              <div data-categoryid="19119" data-articleid="26304"
                   data-categoryname="Host-pathogen interactions"
                   class="flagImage" title="Flag 'Host-pathogen interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Parasite+evolution%22" title="Search for articles in the subject area:'Parasite evolution'"><div class="flagText">Parasite evolution</div></a>
              <div data-categoryid="19291" data-articleid="26304"
                   data-categoryname="Parasite evolution"
                   class="flagImage" title="Flag 'Parasite evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="26304"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Species+interactions%22" title="Search for articles in the subject area:'Species interactions'"><div class="flagText">Species interactions</div></a>
              <div data-categoryid="4813" data-articleid="26304"
                   data-categoryname="Species interactions"
                   class="flagImage" title="Flag 'Species interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Trees%22" title="Search for articles in the subject area:'Trees'"><div class="flagText">Trees</div></a>
              <div data-categoryid="42711" data-articleid="26304"
                   data-categoryname="Trees"
                   class="flagImage" title="Flag 'Trees' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2995'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5250'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2298&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9169">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=22197">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
