

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000638"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000638" />

    <meta name="citation_title" content="Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations"/>
    <meta itemprop="name" content="Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations"/>

      <meta name="citation_author" content="Jeffrey A. Johnson"/>
            <meta name="citation_author_institution" content="Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Jin-Fen Li"/>
            <meta name="citation_author_institution" content="Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Xierong Wei"/>
            <meta name="citation_author_institution" content="Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Jonathan Lipscomb"/>
            <meta name="citation_author_institution" content="Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Diane Bennett"/>
            <meta name="citation_author_institution" content="HIV Incidence and Case Surveillance Branch, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Ashley Brant"/>
            <meta name="citation_author_institution" content="Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Mian-er Cong"/>
            <meta name="citation_author_institution" content="Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Thomas Spira"/>
            <meta name="citation_author_institution" content="Global AIDS Program, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Robert W. Shafer"/>
            <meta name="citation_author_institution" content="Stanford University Medical Center, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Walid Heneine"/>
            <meta name="citation_author_institution" content="Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America"/>

    <meta name="citation_date" content="2007/7/25"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000638.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e638"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Guidelines for the use of antiretroviral agents in HIV-1-infected adults and adolescents.; citation_number=1; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Blinded multicenter comparison of methods to detect a drug-resistant mutant of human immunodeficiency virus type 1 at low frequency.; citation_author=EK Halvas; citation_author=GM Aldrovandi; citation_author=P Balfe; citation_author=IA Beck; citation_author=VF Boltz; citation_journal_title=J Clin Microbiol; citation_volume=44; citation_number=2; citation_pages=2612-2614; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Emergence of drug-resistant HIV-1 after intrapartum administration of single-dose nevirapine is substantially underestimated.; citation_author=JA Johnson; citation_author=J-F Li; citation_author=L Morris; citation_author=N Martinson; citation_author=G Gray; citation_journal_title=J Infect Dis; citation_volume=192; citation_number=3; citation_pages=16-23; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sensitive drug resistance assays reveal long-term persistence of HIV-1 variants with the K103N nevirapine (NVP) resistance mutation in some women and infants after administration of single-dose NVP: HIVNET 012.; citation_author=T Flys; citation_author=DV Nissley; citation_author=CW Claasen; citation_author=D Jones; citation_author=C Shi; citation_journal_title=J Infect Dis; citation_volume=192; citation_number=4; citation_pages=24-29; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Persistence of nevirapine-resistant HIV-1 in women after single-dose nevirapine therapy for prevention of maternal-to-fetal HIV-1 transmission.; citation_author=S Palmer; citation_author=V Boltz; citation_author=N Martinson; citation_author=F Maldarelli; citation_author=G Gray; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=5; citation_pages=7094-7099; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Intrapartum exposure to nevirapine and subsequent maternal responses to nevirapine-based antiretroviral therapy.; citation_author=G Jourdain; citation_author=N Ngo-Giang-Huong; citation_author=S Le Cœur; citation_author=C Bowonwatanuwong; citation_author=P Kantipong; citation_journal_title=N Engl J Med; citation_volume=351; citation_number=6; citation_pages=229-40; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Low frequency non-nucleoside reverse transcriptase inhibitor (NNRTI)-resistant variants contribute to failure of efavirenz-containing regimens in NNRTI-experienced patients with negative standard genotypes for NNRTI mutations.; citation_author=J Mellors; citation_author=S Palmer; citation_author=D Nissley; citation_journal_title=Antiviral Ther; citation_volume=8; citation_number=7; citation_pages=S150; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Competitive fitness of nevirpine-resistant human immunodeficiency type 1 mutants.; citation_author=JA Collins; citation_author=MG Thompson; citation_author=E Paintsil; citation_author=M Ricketts; citation_author=J Gedzior; citation_journal_title=J Virol; citation_volume=78; citation_number=8; citation_pages=603-611; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus fitness in vivo: Calculations based on a single zidovudine resistance mutation at codon 215 of reverse transcriptase.; citation_author=J Goudsmit; citation_author=A deRonde; citation_author=DD Ho; citation_author=AS Perelson; citation_journal_title=J Virol; citation_volume=70; citation_number=9; citation_pages=5662-5664; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=HIV-1resistant strains acquired at the time of primary infection massively fuel the cellular reservoir and persist of lengthy periods of time.; citation_author=J Ghosn; citation_author=I Pellegrin; citation_author=C Goujard; citation_author=C Deveau; citation_author=JP Viard; citation_journal_title=AIDS; citation_volume=20; citation_number=10; citation_pages=159-170; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the LiPA HIV-1 RT assay version 1: comparison of sequence and hybridization based genotyping systems.; citation_author=M Sturmer; citation_author=B Morgenstern; citation_author=S Staszewski; citation_author=HW Doerr; citation_journal_title=J Clin Virol; citation_volume=25; citation_number=11; citation_pages=S65-72; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection and quantification of multiple drug resistance mutations in HIV-1 reverse transcriptase by an oligonucleotide ligation assay.; citation_author=ML Villahermosa; citation_author=I Beck; citation_author=L Perez-Alvarez; citation_author=G Contreras; citation_author=LM Frenkel; citation_journal_title=J Hum Virology; citation_volume=4; citation_number=12; citation_pages=238-248; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=LigAmp for sensitive detection of single-nucleotide differences.; citation_author=C Shi; citation_author=SH Eshleman; citation_author=D Jones; citation_author=N Fukushima; citation_author=L Hua; citation_journal_title=Nat Methods; citation_volume=1; citation_number=13; citation_pages=141-147; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Changes in human immunodeficiency virus type 1 populations after treatment interruption in patients failing antiretroviral therapy.; citation_author=AJ Hance; citation_author=V Lemiale; citation_author=J Izopet; citation_author=D Lecossier; citation_author=V Joly; citation_journal_title=J Virol; citation_volume=75; citation_number=14; citation_pages=6410-6417; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multiple, linked human immunodeficiency virus type 1 drug resistance mutations in treatment-experienced patients are missed by standard genotype analysis.; citation_author=S Palmer; citation_author=M Kearney; citation_author=F Maldarelli; citation_author=EK Halvas; citation_author=CJ Bixby; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=15; citation_pages=406-413; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detection of minor drug-resistant populations by parallel allele-specific sequencing.; citation_author=F Cai; citation_author=H Chen; citation_author=CB Hicks; citation_author=JA Bartlett; citation_author=J Zhu; citation_journal_title=Nat Methods; citation_volume=4; citation_number=16; citation_pages=123-125; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Marqulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiva; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=17; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nevirapine-resistant human immunodeficiency virus: kinetics of replication and estimated prevalence in untreated patients.; citation_author=D Havlir; citation_author=S Eastman; citation_author=A Gamst; citation_author=DD Richman; citation_journal_title=J Virol; citation_volume=70; citation_number=18; citation_pages=7894-7899; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The epidemiology of antiretroviral drug resistance among drug-naive HIV-1 infected persons in 10 U.S. cities.; citation_author=HS Weinstock; citation_author=I Zaidi; citation_author=W Heneine; citation_author=D Bennett; citation_author=JG Garcia-Lerma; citation_journal_title=J Infect Dis; citation_volume=189; citation_number=19; citation_pages=2174-80; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The fitness cost of mutations associated with human immunodeficiency virus type 1 drug resistance is modulated by mutational interactions.; citation_author=M Cong; citation_author=W Heneine; citation_author=JG García-Lerma; citation_journal_title=J Virol; citation_volume=81; citation_number=20; citation_pages=3037-3041; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000638" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations"/>
    <meta name="twitter:description" content="
Background
The success of antiretroviral therapy is known to be compromised by drug-resistant HIV-1 at frequencies detectable by conventional bulk sequencing. Currently, there is a need to assess the clinical consequences of low-frequency drug resistant variants occurring below the detection limit of conventional genotyping. Sensitive detection of drug-resistant subpopulations, however, requires simple and practical methods for routine testing.

Methodology
We developed highly-sensitive and simple real-time PCR assays for nine key drug resistance mutations and show that these tests overcome substantial sequence heterogeneity in HIV-1 clinical specimens. We specifically used early wildtype virus samples from the pre-antiretroviral drug era to measure background reactivity and were able to define highly-specific screening cut-offs that are up to 67-fold more sensitive than conventional genotyping. We also demonstrate that sequencing the mutation-specific PCR products provided a direct and novel strategy to further detect and link associated resistance mutations, allowing easy identification of multi-drug-resistant variants. Resistance mutation associations revealed in mutation-specific amplicon sequences were verified by clonal sequencing.

Significance
Combined, sensitive real-time PCR testing and mutation-specific amplicon sequencing provides a powerful and simple approach that allows for improved detection and evaluation of HIV-1 drug resistance mutations.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000638.g004"/>

  <meta property="og:title" content="Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000638" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8750'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3671'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3472&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000638">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000638" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000638&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeffrey A. Johnson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:JJohnson1@cdc.gov">JJohnson1@cdc.gov</a></p>

                <p>Affiliation:
                  Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jin-Fen Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xierong Wei, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonathan Lipscomb, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Diane Bennett, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  HIV Incidence and Case Surveillance Branch, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ashley Brant, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mian-er Cong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Spira, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Global AIDS Program, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert W. Shafer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford University Medical Center, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Walid Heneine
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory Branch, Division of HIV/AIDS Prevention, National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, Centers for Disease Control and Prevention, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 25, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000638</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000638" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000638" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000638" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000638" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000638-g001" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000638-t001" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000638-g002" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000638-t002" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000638-g003" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000638-g004" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000638">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000638" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The success of antiretroviral therapy is known to be compromised by drug-resistant HIV-1 at frequencies detectable by conventional bulk sequencing. Currently, there is a need to assess the clinical consequences of low-frequency drug resistant variants occurring below the detection limit of conventional genotyping. Sensitive detection of drug-resistant subpopulations, however, requires simple and practical methods for routine testing.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We developed highly-sensitive and simple real-time PCR assays for nine key drug resistance mutations and show that these tests overcome substantial sequence heterogeneity in HIV-1 clinical specimens. We specifically used early wildtype virus samples from the pre-antiretroviral drug era to measure background reactivity and were able to define highly-specific screening cut-offs that are up to 67-fold more sensitive than conventional genotyping. We also demonstrate that sequencing the mutation-specific PCR products provided a direct and novel strategy to further detect and link associated resistance mutations, allowing easy identification of multi-drug-resistant variants. Resistance mutation associations revealed in mutation-specific amplicon sequences were verified by clonal sequencing.</p>


<h3>Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Combined, sensitive real-time PCR testing and mutation-specific amplicon sequencing provides a powerful and simple approach that allows for improved detection and evaluation of HIV-1 drug resistance mutations.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Johnson JA, Li J-F, Wei X, Lipscomb J, Bennett D, et al.  (2007) Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations. PLoS ONE 2(7):
          e638.
            doi:10.1371/journal.pone.0000638</p><p><strong>Academic Editor: </strong>Beatriz Grinsztejn, HIV/AIDS Clinical Research Unit, Brazil</p><p><strong>Received:</strong> May 25, 2007; <strong>Accepted:</strong> June 21, 2007; <strong>Published:</strong> July 25, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>None.</p><p><strong>Competing interests:</strong> J.A.J. and W.H. are named on a CDC patent application on the use of the real-time polymerase chain reaction assays presented in this article.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Highly active antiretroviral therapy (HAART) can provide sustained clinical benefit for HIV-1 infected persons, but treatment success is jeopardized by drug resistance. Drug resistance testing supports the management of persons on HAART and is recommended to help guide treatment choices <a href="#pone.0000638-US1">[1]</a>. Resistance-related mutations, however, are conventionally detected by bulk sequence analysis of viral RNA from plasma which does not reliably detect variants comprising less than 20% of the viruses in a sample <a href="#pone.0000638-Halvas1">[2]</a>. Identifying drug-resistant variants at frequencies below the detection capability of conventional genotyping requires new diagnostic methods. Currently, there is increasing recognition that identification of low-frequency drug-resistant viruses is vital for evaluating the full clinical impact of drug resistance and for understanding the dynamics of drug resistance emergence and persistence <a href="#pone.0000638-Johnson1">[3]</a>–<a href="#pone.0000638-Mellors1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A few seminal studies illustrated the advantages of sensitive drug resistance assays with women who received intrapartum single-dose nevirapine (SD-NVP). These reports on sensitive testing for nevirapine resistance have shown that drug resistance emerges more frequently and persists longer than previously demonstrated by conventional sequencing <a href="#pone.0000638-Johnson1">[3]</a>, <a href="#pone.0000638-Flys1">[4]</a>, <a href="#pone.0000638-Palmer1">[5]</a>. Persisting minority nevirapine-resistant viruses may contribute to poor virologic responses when subsequent regimens contain nevirapine-related drugs <a href="#pone.0000638-Jourdain1">[6]</a>, <a href="#pone.0000638-Mellors1">[7]</a>. Accurate accounting of transmitted drug resistance is also a concern. Because of reduced fitness in the absence of antiretroviral treatment, transmitted drug-resistant variants can decay to levels that are undetectable by conventional sequence analysis <a href="#pone.0000638-Collins1">[8]</a>, <a href="#pone.0000638-Goudsmit1">[9]</a>, <a href="#pone.0000638-Ghosn1">[10]</a>. Therefore, the ability to detect low-frequency variants would allow for more informed decisions on the selection of active drugs for both drug-naïve and drug-experienced persons beginning new treatment regimens.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Early hybridization methods to improve HIV-1 resistance mutation detection, such as the Line Probe Assay (LiPA), offered a modest improvement in sensitivity over bulk sequencing but experienced frequent detection failures due to the considerable nucleotide polymorphisms present in HIV-1 <a href="#pone.0000638-Sturmer1">[11]</a>. More recent point-mutation assays offer substantial improvements in sensitivity <a href="#pone.0000638-Villahermosa1">[12]</a>, <a href="#pone.0000638-Shi1">[13]</a>, <a href="#pone.0000638-Hance1">[14]</a> and, previously, we had shown that real-time PCR assays can be both highly specific and sensitive with subtype C viruses from SD-NVP-experienced women <a href="#pone.0000638-Johnson1">[3]</a>. However, point-mutation assays are susceptible to polymorphisms and their performance with large-scale clinical testing is unclear. Moreover, point-mutation testing is inherently limited in its genotypic information because it does not detect mutations beyond what is interrogated by the assay. Other approaches that evaluate numerous virus sequences per sample are highly-informative research tools but are too complex or costly for routine clinical testing <a href="#pone.0000638-Palmer2">[15]</a>, <a href="#pone.0000638-Cai1">[16]</a>, <a href="#pone.0000638-Marqulies1">[17]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To both simplify and improve the sensitivity of HIV drug resistance testing, we describe a strategy that combines real-time PCR point-mutation assays and direct sequencing of resistance mutation-specific PCR products to identify and link additional mutations. For this purpose, we focused on developing and validating nine assays for key drug resistance mutations in subtype B HIV-1 clinical specimens as a basis for later expansion to other virus mutations and subtypes. Because these assays can detect nearly 2-logs less mutant virus than conventional bulk sequencing they are able to detect resistance-associated mutations that might emerge at low frequencies as part of the normal viral quasispecies <a href="#pone.0000638-Havlir1">[18]</a>. Therefore, to maximize specificity, we used pre-antiretroviral (pre-ARV) drug era wildtype virus samples to define cutoffs that exclude the detection of naturally-occurring minority mutants. We also used primer mixtures and designed mismatches to circumvent the genomic plasticity of HIV-1 and show high sensitivity on a large panel of clinical samples. We demonstrate that these assays are able to detect low-frequency drug resistance and permit easy identification of linked mutations, thus, providing a practical strategy for routine drug resistance testing.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Virus template amplification</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>HIV-1 genomic RNA was extracted (Qiagen UltraSens RNA kit) from 200 µL plasma or serum and reconstituted in 50 µL of buffer provided with the kit. To ensure sufficient template for repeat testing, virus sequences were first amplified from 5 µL HIV-1 RNA by reverse transcriptase-polymerase chain reaction (RT-PCR) using the reverse primer RTP-REV2 [5′-CTT CTG TAT GTC ATT GAC AGT CC], and forward primer RTP-F1 [5′-CCT CAG ATC ACT CTT TGG CAA CG], which span from n.t. 1 in protease to n.t. 777 in RT. PCR amplification conditions were 40 cycles of 95°C for 45 seconds, 50°C for 30 seconds, and 72°C for 2 minutes. To better evaluate the success of amplifying each sample, the reverse transcriptase and PCR amplification steps were performed separately. We later assessed the validated procedures could be combined into a one-step RT-PCR to reduce specimen handling (not shown). When only RT template was desired, a shorter amplicon generated by the primer pair, RTP-REV2 and RTP-F2 [5′-AAA GTT AAA CAA TGG CCA TTG ACA G] (n.t. 58 to 777 in RT), was used and occasionally provided improved amplification sensitivity. Both primer sets were also successful in generating amplified virus template from proviral sequences (not shown).</p>


<h4>Real-time PCR</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Real-time PCR-based mutation-specific assays were developed for the protease L90M and the reverse transcriptase M41L, K65R, K70R, K103N, Y181C, M184V, and both T215Y and F resistance-associated mutations in HIV-1 subtype B. Mutation testing was performed in 96-well plates using 2 µL of 1:20 dilutions of the RT-PCR products, except that samples with viral loads below 5000 copies/mL were not diluted. The principle of the real-time PCR assay is to compare the differential amplifications of a mutation-specific PCR and a PCR that amplifies all viruses in the sample (total virus copy reaction) (<a href="#pone-0000638-g001">Figure 1</a>). The HIV-1 total copy primers, ComFWD and ComREV, span n.t. 258–420 in RT and were used with the common probes, com1P and 2P (<a href="#pone-0000638-g001">Figure 1A</a>, see <a href="#pone-0000638-t001">Table 1</a>). The same common reaction was used for all resistance mutation tests to reduce labor and costs. For multiple mutation screening, several resistance mutation-specific reactions can be performed simultaneously. The cycle number at which the fluorescence emission exceeds the background fluorescence threshold is the threshold cycle (CT) and is the unit of measure for comparing the differences in amplification signals (ΔCT) between the total copy and mutation-specific reactions (<a href="#pone-0000638-g001">Figure 1B</a>). All samples were tested in duplicate with the means of the total copy and mutation-specific CTs used for the determination of the ΔCT.</p>
<div class="figure" id="pone-0000638-g001"><div class="img"><a name="pone-0000638-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Principle of the real-time PCR assay.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A. HIV-1 template generated from RT-PCR of viral RNA is subjected to both total copy and mutation-specific real-time reactions. B. The difference in the total copy and mutation-specific reactions (ΔCT) is used to differentiate mutant and wildtype specimens. In this example, the experimental cutoff is a ΔCT of 10.5 cycles. A mutation-specific CT within 10.5 cycles of the total copy reaction CT would indicate the presence of mutant virus.</p>
<span>doi:10.1371/journal.pone.0000638.g001</span></div><div class="figure" id="pone-0000638-t001"><div class="img"><a name="pone-0000638-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000638.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000638.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Oligonucleotides for real-time PCR drug resistance testing.</span></strong></p><span>doi:10.1371/journal.pone.0000638.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The mutation-specific primers (<a href="#pone-0000638-t001">Table 1</a>) were designed to preferentially anneal with the targeted mutation nucleotide(s), thus having reduced affinity for wildtype sequences. To accommodate the various polymorphisms in large populations, degenerate nucleotides were placed at complementary positions in the primers. Specificity was enhanced by creating designed mismatches at nucleotide(s) -2 to -4 positions from the primer 3′-end. Furthermore, to cover the spectrum of polymorphisms present, mixtures of multiple degenerate primers were often required. Mutation-specific primer mixtures were experimentally evaluated and the ratios that best balanced differences in primer avidities and minimized cross-interference in primer annealing were selected. Each change was re-evaluated against wildtype and mutant samples. For example, the M41L assay combined seven different mutation-specific primers for the detection of both the CTT and CTC mutant codons within the polymorphic sequences of the resistant samples.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The real-time PCR probes anneal to sequences within the total copy and mutation-specific amplicons and merely act as reporters of primer extension. The probes were 5′labeled with FAM (6-carboxyfluorescein) and internally quenched with a black-hole quencher (BHQ) placed at the positions indicated by the quotation marks (“ “) in <a href="#pone-0000638-t001">Table 1</a>. In the probe design, the quencher was placed between 9–18 nucleotides from the 5′ FAM-labeled base at the position providing the best quenching of background fluorescence. Fluorescent signals, reported as relative fluorescent units (RFU), were generated by degradation of the fluorescent probes which resulted in the separation of the fluorophore from the quencher during each round of chain elongation. The threshold for each test was set at an RFU level that corresponded to the beginning of the log phase of the amplification curves. For additional information on the use of probes and interpretation of ΔCT see the supporting information on assay design in <a href="#pone.0000638.s002">Method S1</a>.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Real-time PCRs were initiated with a hot-start incubation at 94°C for 11 minutes before proceeding to 45 cycles of melting at 94°C for 30 seconds, annealing at 50°C for 15 seconds and extension at 60°C for 30 seconds. All reactions were performed in a total volume of 50 µL/well in 96-well PCR plates using iCycler real-time PCR thermocyclers with optical units (Bio-Rad) and AmpliTaq Gold polymerase (2.5 U/reaction; Applied Biosystems). Final reagent concentrations were 320 nM for the forward and reverse primers, 160 nM probe(s), and 400 µM dNTPs. Low viral load samples that generated total copy CTs which appeared after 26 cycles sometimes yielded false-positive results. To avoid this complication, all samples with CTs above 26 cycles were further amplified by nested PCR prior to real-time PCR testing. To adequately subtract background fluorescence, high virus load samples that produced total copy CTs appearing less than 10 cycles were diluted 1:100–1000 in RNase/DNase-free reagent-grade water and retested. We found that 1:20 dilutions of RT-PCR products from all but the samples with virus loads below 5000 copies/ml provided adequate template for real-time PCR testing.</p>


<h4>Assay evaluations on plasmids and clinical specimens</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Each mutation-specific primer mixture was initially evaluated against both cloned lab-generated and patient-derived mutant virus sequences that were serially diluted 10-fold in backgrounds of wildtype sequence plasmids. The supporting information on plasmid evaluations in <a href="#pone.0000638.s003">Method S2</a> describes the use of cloned virus sequences for the preliminary selection of mutation-specific primers and determining assay absolute detection limits.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To both evaluate the frequency of natural polymorphisms at codons associated with drug resistance and establish assay cutoffs for screening subtype B virus infections, we tested 138 serum samples collected from 117 individuals infected with HIV-1 in the US between 1982–1985, prior to the era of antiretroviral drug use. Within these early HIV specimens were longitudinal samples from 23 individuals which were examined for evidence of polymorphic changes over time. Assay sensitivities for clinical screening were determined using samples from a total of 302 individuals with drug resistance mutations detectable by bulk sequence genotyping. The resistance mutation samples were collected in the US and Canada during 1998–2005, and included a portion of US samples from a previously reported surveillance study <a href="#pone.0000638-Weinstock1">[19]</a>. We evaluated archived specimens which the CDC Institutional Review Board determined did not involve research on identifiable subjects. Samples with resistance mutations were obtained from 51 individuals with protease L90M, and those with reverse transcriptase mutations included 78 subjects with M41L, 26 with K65R, 59 with K70R, 81 with K103N, 28 with Y181C, 67 with M184V, 44 with T215Y, and 35 with T215F. To increase the stringency of assay evaluations, specimens with substantial numbers of polymorphisms in primer binding sites were included.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>For the purpose of clinical testing, each assay cutoff was placed at a ΔCT that excluded all the early wildtype virus samples and, when possible, was at least one amplification cycle less than the lowest pre-antiretroviral wildtype ΔCT as an added buffer against non-specific reactivity. It is expected that the sensitivities observed with plasmid dilutions may not be achievable for some clinical samples because of low virus copy numbers or genomic differences which affect primer performance.</p>


<h4>Low-frequency mutation detection</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>After validating the assays on early wildtype and known mutant viruses, we sought to demonstrate the ease with which the sensitive assays could detect low-frequency mutations. Tests for five resistance mutations were applied to a small assortment of convenient mutant virus clinical samples previously used in assay validation that had no evidence of the targeted mutation by standard genotyping. Tests for L90M, K103N, Y181C, M184V and T215Y were applied to 30, 19, 13, 11 and 10 specimens, respectively.</p>


<h4>Assessing mutation associations in mutation-specific amplicons</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To overcome the single mutation detection limitation of point-mutation assays, we evaluated whether additional information on resistance mutations could be gained from the real-time PCR assays. To address this, we performed direct sequencing (BigDye reagent, Prism 3130XL analyzer, Applied Biosystems) of the products from positive mutation-specific reactions and compared these to their respective sample bulk sequence for evidence of nucleotide differences. In order to include other codons of interest with the K103N test, we extended the amplicon by using the 184V REV reverse primer (<a href="#pone-0000638-t001">Table 1</a>), which allowed sequencing to RT codon 221. Any other resistance mutation(s) found in the mutation-specific amplicon would indicate that they were on the same viral strand(s) as the mutation that was specifically targeted in the reaction. The absence of nucleotide mixtures at resistance codons within mutation-specific amplicons is informative because it indicates the targeted mutation is entirely associated with those mutations. Conversely, nucleotide mixtures suggest that the interrogated mutation is linked to more than one resistance genotype.</p>


<h4>Clonal sequencing</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To verify mutation associations depicted in the sequences of mutation-specific amplicons, nested amplifications of the original RT-PCRs were cloned into the TA vector (Invitrogen) for <em>E. coli</em> transformation <a href="#pone.0000638-Johnson1">[3]</a>. Colonies were screened with the same assay used for the low frequency mutation detection and positive clones were sequence analyzed as above.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Assay sensitivities on cloned virus sequences</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Relative limits of detection were compared in a simple laboratory setting using serial dilutions of cloned mutant template. The ΔCT that was equivalent to a 0.5 log greater reactivity than the wildtype mean ΔCT on the dilution curve was used to compare assay sensitivities (<a href="#pone-0000638-g002">Figure 2</a>). This approach yielded detection limits of 0.001% and 0.02% for L90M and K103N, respectively. Absolute detection limits for the remaining assays were likewise determined on cloned control samples and the corresponding frequencies found to be 0.02% for M41L, 0.05% for K70R, 0.08% for K65R, Y181C, M184V, and T215F, and 0.2% for T215Y (<a href="#pone-0000638-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000638-g002"><div class="img"><a name="pone-0000638-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Mutation-specific assay reactivity on plasmids.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Cloned L90M (A.) and K103N (B.) mutant virus sequence was diluted 10-fold, from 100% to 0.001%, in backgrounds of wildtype sequence to determine assay detection limits. Plotted are the mean ΔCT versus log<sub>10</sub> of the mutant dilution series (•), and the mean ΔCT for wildtype sequence alone (▪). The lower detection limit (lower dotted line) was placed at the ΔCT equivalent to 0.5 log<sub>10</sub> below (0.5-log greater reactivity than) the wildtype ΔCT. Dilutions that fall outside the linear range are not considered. For comparison, the mutant virus frequency equivalences for the established clinical cutoffs are also shown (dashed line).</p>
<span>doi:10.1371/journal.pone.0000638.g002</span></div><div class="figure" id="pone-0000638-t002"><div class="img"><a name="pone-0000638-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000638.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000638.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Assay ΔCT measures, cutoffs, and sensitivities on clinical samples.</span></strong></p><span>doi:10.1371/journal.pone.0000638.t002</span></div>

<h4>Assay sensitivity and performance with clinical samples</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The viral RNA extraction from plasma followed by the RTP-F1- RTP-REV2 RT-PCR could amplify as little as 10 input RNA copies when using the total copy primers to detect amplified product (data not shown). In some cases, as little as 5 RNA copies could be amplified with the RTP-F2 and RTP-REV2 RT-only primer pair. However, to obtain sufficient amplification with both the total copy and mutation-specific reactions with majority mutant virus samples, around 20–80 input RNA copies were required depending on the assay.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Assay cutoff values intended for population-wide clinical screening were established using 138 patient-derived wildtype specimens collected before the era of ARV drug treatment. Following the selection of each assay cut-off, assay sensitivity was evaluated using a total of 302 samples with sequence-detectable drug resistance mutations.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>With some longitudinal wildtype samples collected in the pre-ARV drug era, we observed ΔCTs that differed as much as 6.5 cycles between time points. The greatest ΔCT decrease was seen with K70R, which resulted in this assay having the narrowest window of mutation detection (<a href="#pone-0000638-g003">Figure 3</a>). The ΔCT variation with longitudinal samples suggested that mutations at positions associated with drug resistance can naturally arise and fluctuate over time within individuals. Using the L90M assay as an example, the resulting distribution of collated ΔCTs from the pre-ARV era wildtype samples supported a ΔCT cutoff of 10.5 cycles for L90M clinical testing (ΔCTs ranged from 12.0–28.0 cycles) (<a href="#pone-0000638-g003">Figure 3</a>). Extrapolating from the dilution curve for cloned L90M sequences, this placement corresponded to a frequency mean of 0.2% mutant virus (see <a href="#pone-0000638-g002">Figure 2</a>). At this cut-off, all 51 genotyped 90M samples were positive (ΔCTs ranged from -9.1 to 5.2 cycles) (<a href="#pone-0000638-t002">Table 2</a>). The tests for mutations in the HIV-1 reverse transcriptase were likewise evaluated and the resulting assay limits and performances are provided in <a href="#pone-0000638-t002">Table 2</a> and <a href="#pone-0000638-g003">Figure 3</a>. For each assay, the mean ΔCTs of the specimens documented to have mutations were significantly lower than the ΔCTs of the wildtype samples (all p values &lt;0.0001, T-test).</p>
<div class="figure" id="pone-0000638-g003"><div class="img"><a name="pone-0000638-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Assay reactivities with clinical samples having sequence-detectable mutations and with pre-antiretroviral wildtype virus samples.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>The range of reactivity for each assay is shown for wildtype and mutant samples. The upper and lower ΔCT and the mean (hash) for each group are indicated. Assay cutoffs (horizontal line) were established to exclude all wildtype viruses from the pre-antiretroviral era.</p>
<span>doi:10.1371/journal.pone.0000638.g003</span></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Because of unusual polymorphisms, some samples comprised almost entirely of mutant virus produced ΔCTs near or above the cutoff. In these situations, elevated ΔCTs resulting from weak primer binding could be interpreted as mutant viruses present at low frequencies. Hence, this testing format is best-suited to provide highly specific population-level resistance screening and is not necessarily applicable to mutant virus quantitation.</p>


<h4>Identification of low-frequency mutations and confirmation by cloning</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>To demonstrate the ability of real-time PCR assays to detect drug-resistant viruses present as minor variants in specimens, assays for L90M, K103N, Y181C, M184V and T215Y were applied to assortment of clinical samples that had major resistance mutations, but had no evidence of the targeted mutation by bulk sequencing. Each assay identified at least one mutant sample with a hidden low-frequency mutation in the few samples tested. L90M was identified in 2/30 samples (ΔCTs = 1.0, 5.0 cycles), K103N in 3/19 (ΔCTs = 7.6, 7.7, 9.8), Y181C in 1/13 (ΔCT = 7.6), M184V in 2/11 (ΔCTs = 5.9, 7.3), and T215Y in 1 of 10 samples (ΔCT = 5.6). One representative low-frequency variant for each of the five mutations tested was verified by clonal sequencing which found the mutation frequencies to be between 0.7%–11%.</p>


<h4>Linking high and low-frequency resistance mutations</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>To overcome the point-mutation testing limitation of single mutation detection, we directly sequenced positive mutation-specific PCR products to ascertain whether additional genotypic information could be easily obtained. Two samples that demonstrate the usefulness of this approach are described. Bulk sequencing of one sample showed nucleotide mixtures in the RT at the resistance-associated codons G190G/A, L210L/W and an undecipherable mixture at codon 215 in reverse transcriptase (sample ‘A’) (<a href="#pone-0000638-g004">Figure 4A</a>). This sample was tested with both the T215Y and T215F PCR assays, but was found to be positive for only the T215Y mutation (ΔCT = 4.1). Direct sequencing of the T215Y-positive amplicon revealed that both the G190A and L210W mutations were present as unmixed codons (<a href="#pone-0000638-g004">Figure 4A</a>). The mutation-specific sequence resolved the resistance mutation at codon 215 and suggested all three mutations (G190A, L210W, and T215Y) were linked on the same genome. The co-linkage of all three mutations was confirmed in 6 of 6 T215Y-positive clones. A second sample (sample ‘B’, <a href="#pone-0000638-g004">Figure 4B</a>) that had only T215F and K219Q mutations by bulk genotyping was positive for low-frequency K103N using the extended K103N assay (ΔCT = 9.8). The sequence of the K103N-positive amplicon contained the K219Q mutation and a T215F/V mixture. Also in this amplicon was another low-frequency mutation, M184V, which appeared as a 50% mixture with wildtype sequence. Hence, the K103N-specific sequence showed that low-frequency K103N mutants were associated with K219Q, M184V and M, and T215F and V. Clonal sequencing confirmed K103N in 3/81 clones (4%) obtained from the sample and all 3 clones verified K103N was linked with K219Q. Two of the clones also had the M184V and T215V mutations. The two clonal genotypes are provided in <a href="#pone.0000638.s001">Figure S1</a>. The ability to resolve codon 219 in the extended K103N mutation-specific sequence demonstrated that resistance mutations as far apart as 117 amino acids could be evaluated for linkage.</p>
<div class="figure" id="pone-0000638-g004"><div class="img"><a name="pone-0000638-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000638" data-uri="info:doi/10.1371/journal.pone.0000638.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000638.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000638.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000638.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000638.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Detection of other associated resistance mutations in mutation-specific amplicons.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>A. The undecipherable codon 215 in the bulk sequence of this sample was resolved (positive) with the T215Y test. The sequence of the T215Y-positive amplicon showed that the mutations present in the bulk sequence were linked. B. The low-frequency K103N amplicon sequence from this sample uncovered another previously undetected mutation, M184V. 215X, undecipherable codon 215.</p>
<span>doi:10.1371/journal.pone.0000638.g004</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We describe a simple and sensitive approach for mutant virus screening that is able to detect drug-selected resistance mutations at frequencies as low as 0.3% in clinical samples, allowing for the identification of minority HIV-1 variants. The real-time PCR-based point mutation assays were robust with the 474 total subtype B virus specimens evaluated, supporting their use for clinical testing. Improved low-frequency mutation detection was provided by clinical testing cutoffs that were 10–67-fold more sensitive than conventional sequencing. These cutoffs were above the background reactivities observed with drug-naïve wildtype HIV collected in the pre-antiretroviral drug era and, thus, identify mutations occurring at frequencies above those found naturally in virus quasispecies. Although this paper focused on resistance mutation testing in subtype B viruses, we earlier demonstrated that real-time PCR assays can also be successfully developed for subtype C viruses which are globally the most prevalent <a href="#pone.0000638-Johnson1">[3]</a>. When possible, oligonucleotides are designed so that they might also be used with more than one subtype; however, screening for resistance in non-B subtypes requires that the oligonucleotides are properly validated for those subtypes.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Setting stringent assay cutoffs to avoid detecting natural polymorphisms resulted in primer designs that provided sensitivities of 96% to &gt;99% and specificities of &gt;99% with samples that included highly polymorphic sequences (<a href="#pone-0000638-t002">Table 2</a>). Although the real-time PCR assays were able to detect as little as 0.001%–0.2% cloned mutant sequences, increasing ΔCT cutoffs to expand the mutation detection range would make it difficult to differentiate drug-selected mutants from naturally-occurring variants. However, in antiretroviral studies of infected persons in which pre-drug exposure samples are available, a comparative method could be used to evaluate drug resistance rather than an absolute ΔCT cut-off. In these settings, a substantial decrease in ΔCT between the pre- and post-treatment samples for an individual could indicate the emergence of a mutation even if the ΔCT does not drop below the cut-off established for screening. Furthermore, in experimental settings where baseline genotypes are known, individual primers that best match the virus sequence may be used, instead of mixtures, to maximize assay sensitivity.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Evidence of improved resistance mutation detection was found in testing only a few samples which uncovered hidden mutations. However, to overcome the limitation of single mutation detection, we directly sequenced mutation-specific reactions as a simple way to rapidly assess mutation associations and demonstrated that the genotypic findings were similar to that obtained by cloning virus templates. Sequencing mutation-specific amplicons also identified additional low-frequency drug resistance mutations when they were linked to the targeted mutation, as was seen with the discovery of M184V in sample B. Therefore, previously hidden multi-drug resistance could easily be uncovered.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Sensitive testing can be streamlined by using a tailored and concise panel of mutation-specific tests that span the protease and RT regions, followed by sequencing the mutation-specific amplicons from positive tests to evaluate for linked mutations. This would allow for sensitive primary screening of resistance as well as the identification of other mutations present in the individual. The capacity to identify linked mutations could be important for understanding the persistence <a href="#pone.0000638-Cong1">[20]</a> and clinical impact of mutant variants.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>In conclusion, we present a panel of real-time PCR assays that provide a sensitive and user-friendly method for screening HIV-1 drug resistance mutations. The substantial oligonucleotide modifications that allowed for successful detection of mutations within diverse sequence backgrounds, combined with extensive validation and improved sensitivity, make these assays feasible for large-scale resistance testing. Furthermore, coupling mutation-specific sequencing to sensitive screening expands the capability of point-mutation testing and provides a powerful approach for studying the dynamics and clinical consequences of drug-resistant HIV-1. The simplicity of this methodology and the abundance of real-time PCR materials currently make sensitive PCR assays more practical for broader drug resistance testing than the more complex and expensive testing methods.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000638"></div><a name="pone.0000638.s001" id="pone.0000638.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000638.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The HXB2 RT nucleotides, bulk sequence of sample B, and sample B low-frequency K103N clones are shown. The sites of the codon 103, 184, 215 and 219 resistance-associated nucleotides are underlined (_). Dots (.) over the sequences indicate nucleotides that differ from HXB2. A ‘C’ at 103 = K103N, a ‘G’ at 184 = M184V, a ‘TT’ at 215 = T215F, a ‘GT’ at 215 = T215V, and a ‘C’ at 219 = K219Q.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000638.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000638.s002" id="pone.0000638.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000638.s002">Method S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">PCR assay design.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000638.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000638.s003" id="pone.0000638.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000638.s003">Method S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Assay evaluations on plasmids.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000638.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: WH JJ. Performed the experiments: JJ JL XW JL AB MC. Analyzed the data: JJ JL XW JL DB AB. Contributed reagents/materials/analysis tools: RS TS WH JJ DB. Wrote the paper: WH JJ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000638-US1" id="pone.0000638-US1"></a>US Department of Health and Human Services (2006) <em>Guidelines for the use of antiretroviral agents in HIV-1-infected adults and adolescents.</em>   Available: <a href="http://AIDSinfo.nih.gov">http://AIDSinfo.nih.gov</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000638-Halvas1" id="pone.0000638-Halvas1"></a>Halvas EK, Aldrovandi GM, Balfe P, Beck IA, Boltz VF, et al.  (2006) Blinded multicenter comparison of methods to detect a drug-resistant mutant of human immunodeficiency virus type 1 at low frequency. J Clin Microbiol  44: 2612–2614.  <ul class="find" data-citedArticleID="968216" data-doi="10.1128/jcm.00449-06"><li><a href="http://dx.doi.org/10.1128/jcm.00449-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Blinded+multicenter+comparison+of+methods+to+detect+a+drug-resistant+mutant+of+human+immunodeficiency+virus+type+1+at+low+frequency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Blinded+multicenter+comparison+of+methods+to+detect+a+drug-resistant+mutant+of+human+immunodeficiency+virus+type+1+at+low+frequency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000638-Johnson1" id="pone.0000638-Johnson1"></a>Johnson JA, Li J-F, Morris L, Martinson N, Gray G, et al.  (2005) Emergence of drug-resistant HIV-1 after intrapartum administration of single-dose nevirapine is substantially underestimated. J Infect Dis  192: 16–23.  <ul class="find" data-citedArticleID="968232" data-doi="10.1086/430741"><li><a href="http://dx.doi.org/10.1086/430741" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+of+drug-resistant+HIV-1+after+intrapartum+administration+of+single-dose+nevirapine+is+substantially+underestimated." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+of+drug-resistant+HIV-1+after+intrapartum+administration+of+single-dose+nevirapine+is+substantially+underestimated.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000638-Flys1" id="pone.0000638-Flys1"></a>Flys T, Nissley DV, Claasen CW, Jones D, Shi C, et al.  (2005) Sensitive drug resistance assays reveal long-term persistence of HIV-1 variants with the K103N nevirapine (NVP) resistance mutation in some women and infants after administration of single-dose NVP: HIVNET 012. J Infect Dis  192: 24–29.  <ul class="find" data-citedArticleID="968226" data-doi="10.1086/430742"><li><a href="http://dx.doi.org/10.1086/430742" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitive+drug+resistance+assays+reveal+long-term+persistence+of+HIV-1+variants+with+the+K103N+nevirapine+%28NVP%29+resistance+mutation+in+some+women+and+infants+after+administration+of+single-dose+NVP%3A+HIVNET+012." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitive+drug+resistance+assays+reveal+long-term+persistence+of+HIV-1+variants+with+the+K103N+nevirapine+%28NVP%29+resistance+mutation+in+some+women+and+infants+after+administration+of+single-dose+NVP%3A+HIVNET+012.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000638-Palmer1" id="pone.0000638-Palmer1"></a>Palmer S, Boltz V, Martinson N, Maldarelli F, Gray G, et al.  (2006) Persistence of nevirapine-resistant HIV-1 in women after single-dose nevirapine therapy for prevention of maternal-to-fetal HIV-1 transmission. Proc Natl Acad Sci USA  103: 7094–7099.  <ul class="find" data-citedArticleID="968238" data-doi="10.1073/pnas.0602033103"><li><a href="http://dx.doi.org/10.1073/pnas.0602033103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Persistence+of+nevirapine-resistant+HIV-1+in+women+after+single-dose+nevirapine+therapy+for+prevention+of+maternal-to-fetal+HIV-1+transmission." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Persistence+of+nevirapine-resistant+HIV-1+in+women+after+single-dose+nevirapine+therapy+for+prevention+of+maternal-to-fetal+HIV-1+transmission.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000638-Jourdain1" id="pone.0000638-Jourdain1"></a>Jourdain G, Ngo-Giang-Huong N, Le Cœur S, Bowonwatanuwong C, Kantipong P, et al.  (2004) Intrapartum exposure to nevirapine and subsequent maternal responses to nevirapine-based antiretroviral therapy. N Engl J Med  351: 229–40.  <ul class="find" data-citedArticleID="968212" data-doi="10.1056/nejmoa041305"><li><a href="http://dx.doi.org/10.1056/nejmoa041305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intrapartum+exposure+to+nevirapine+and+subsequent+maternal+responses+to+nevirapine-based+antiretroviral+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intrapartum+exposure+to+nevirapine+and+subsequent+maternal+responses+to+nevirapine-based+antiretroviral+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000638-Mellors1" id="pone.0000638-Mellors1"></a>Mellors J, Palmer S, Nissley D, et al.  (2006) Low frequency non-nucleoside reverse transcriptase inhibitor (NNRTI)-resistant variants contribute to failure of efavirenz-containing regimens in NNRTI-experienced patients with negative standard genotypes for NNRTI mutations. Antiviral Ther  8: Suppl 1S150.  <ul class="find" data-citedArticleID="968240"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Low+frequency+non-nucleoside+reverse+transcriptase+inhibitor+%28NNRTI%29-resistant+variants+contribute+to+failure+of+efavirenz-containing+regimens+in+NNRTI-experienced+patients+with+negative+standard+genotypes+for+NNRTI+mutations.&amp;auth=&amp;atitle=Low+frequency+non-nucleoside+reverse+transcriptase+inhibitor+%28NNRTI%29-resistant+variants+contribute+to+failure+of+efavirenz-containing+regimens+in+NNRTI-experienced+patients+with+negative+standard+genotypes+for+NNRTI+mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Low+frequency+non-nucleoside+reverse+transcriptase+inhibitor+%28NNRTI%29-resistant+variants+contribute+to+failure+of+efavirenz-containing+regimens+in+NNRTI-experienced+patients+with+negative+standard+genotypes+for+NNRTI+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Low+frequency+non-nucleoside+reverse+transcriptase+inhibitor+%28NNRTI%29-resistant+variants+contribute+to+failure+of+efavirenz-containing+regimens+in+NNRTI-experienced+patients+with+negative+standard+genotypes+for+NNRTI+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000638-Collins1" id="pone.0000638-Collins1"></a>Collins JA, Thompson MG, Paintsil E, Ricketts M, Gedzior J, et al.  (2004) Competitive fitness of nevirpine-resistant human immunodeficiency type 1 mutants. J Virol  78: 603–611.  <ul class="find" data-citedArticleID="968206" data-doi="10.1128/jvi.78.2.603-611.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.2.603-611.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Competitive+fitness+of+nevirpine-resistant+human+immunodeficiency+type+1+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Competitive+fitness+of+nevirpine-resistant+human+immunodeficiency+type+1+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000638-Goudsmit1" id="pone.0000638-Goudsmit1"></a>Goudsmit J, deRonde A, Ho DD, Perelson AS (1996) Human immunodeficiency virus fitness in vivo: Calculations based on a single zidovudine resistance mutation at codon 215 of reverse transcriptase. J Virol  70: 5662–5664.  <ul class="find" data-citedArticleID="968218"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+immunodeficiency+virus+fitness+in+vivo%3A+Calculations+based+on+a+single+zidovudine+resistance+mutation+at+codon+215+of+reverse+transcriptase.&amp;auth=&amp;atitle=Human+immunodeficiency+virus+fitness+in+vivo%3A+Calculations+based+on+a+single+zidovudine+resistance+mutation+at+codon+215+of+reverse+transcriptase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+fitness+in+vivo%3A+Calculations+based+on+a+single+zidovudine+resistance+mutation+at+codon+215+of+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+fitness+in+vivo%3A+Calculations+based+on+a+single+zidovudine+resistance+mutation+at+codon+215+of+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000638-Ghosn1" id="pone.0000638-Ghosn1"></a>Ghosn J, Pellegrin I, Goujard C, Deveau C, Viard JP, et al.  (2006) HIV-1resistant strains acquired at the time of primary infection massively fuel the cellular reservoir and persist of lengthy periods of time. AIDS  20: 159–170.  <ul class="find" data-citedArticleID="968208" data-doi="10.1097/01.aids.0000199820.47703.a0"><li><a href="http://dx.doi.org/10.1097/01.aids.0000199820.47703.a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1resistant+strains+acquired+at+the+time+of+primary+infection+massively+fuel+the+cellular+reservoir+and+persist+of+lengthy+periods+of+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1resistant+strains+acquired+at+the+time+of+primary+infection+massively+fuel+the+cellular+reservoir+and+persist+of+lengthy+periods+of+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000638-Sturmer1" id="pone.0000638-Sturmer1"></a>Sturmer M, Morgenstern B, Staszewski S, Doerr HW (2002) Evaluation of the LiPA HIV-1 RT assay version 1: comparison of sequence and hybridization based genotyping systems. J Clin Virol  25: S65–72.  <ul class="find" data-citedArticleID="968202" data-doi="10.1016/s1386-6532(02)00190-7"><li><a href="http://dx.doi.org/10.1016/s1386-6532(02)00190-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+LiPA+HIV-1+RT+assay+version+1%3A+comparison+of+sequence+and+hybridization+based+genotyping+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+LiPA+HIV-1+RT+assay+version+1%3A+comparison+of+sequence+and+hybridization+based+genotyping+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000638-Villahermosa1" id="pone.0000638-Villahermosa1"></a>Villahermosa ML, Beck I, Perez-Alvarez L, Contreras G, Frenkel LM, et al.  (2001) Detection and quantification of multiple drug resistance mutations in HIV-1 reverse transcriptase by an oligonucleotide ligation assay. J Hum Virology  4: 238–248.  <ul class="find" data-citedArticleID="968222"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Detection+and+quantification+of+multiple+drug+resistance+mutations+in+HIV-1+reverse+transcriptase+by+an+oligonucleotide+ligation+assay.&amp;auth=&amp;atitle=Detection+and+quantification+of+multiple+drug+resistance+mutations+in+HIV-1+reverse+transcriptase+by+an+oligonucleotide+ligation+assay." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+quantification+of+multiple+drug+resistance+mutations+in+HIV-1+reverse+transcriptase+by+an+oligonucleotide+ligation+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+quantification+of+multiple+drug+resistance+mutations+in+HIV-1+reverse+transcriptase+by+an+oligonucleotide+ligation+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000638-Shi1" id="pone.0000638-Shi1"></a>Shi C, Eshleman SH, Jones D, Fukushima N, Hua L, et al.  (2004) LigAmp for sensitive detection of single-nucleotide differences. Nat Methods  1: 141–147.  <ul class="find" data-citedArticleID="968214" data-doi="10.1038/nmeth713"><li><a href="http://dx.doi.org/10.1038/nmeth713" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LigAmp+for+sensitive+detection+of+single-nucleotide+differences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LigAmp+for+sensitive+detection+of+single-nucleotide+differences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000638-Hance1" id="pone.0000638-Hance1"></a>Hance AJ, Lemiale V, Izopet J, Lecossier D, Joly V, et al.  (2001) Changes in human immunodeficiency virus type 1 populations after treatment interruption in patients failing antiretroviral therapy. J Virol  75: 6410–6417.  <ul class="find" data-citedArticleID="968224" data-doi="10.1128/jvi.75.14.6410-6417.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.14.6410-6417.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+human+immunodeficiency+virus+type+1+populations+after+treatment+interruption+in+patients+failing+antiretroviral+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+human+immunodeficiency+virus+type+1+populations+after+treatment+interruption+in+patients+failing+antiretroviral+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000638-Palmer2" id="pone.0000638-Palmer2"></a>Palmer S, Kearney M, Maldarelli F, Halvas EK, Bixby CJ, et al.  (2005) Multiple, linked human immunodeficiency virus type 1 drug resistance mutations in treatment-experienced patients are missed by standard genotype analysis. J Clin Microbiol  43: 406–413.  <ul class="find" data-citedArticleID="968228" data-doi="10.1128/jcm.43.1.406-413.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.1.406-413.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple%2C+linked+human+immunodeficiency+virus+type+1+drug+resistance+mutations+in+treatment-experienced+patients+are+missed+by+standard+genotype+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple%2C+linked+human+immunodeficiency+virus+type+1+drug+resistance+mutations+in+treatment-experienced+patients+are+missed+by+standard+genotype+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000638-Cai1" id="pone.0000638-Cai1"></a>Cai F, Chen H, Hicks CB, Bartlett JA, Zhu J, et al.  (2007) Detection of minor drug-resistant populations by parallel allele-specific sequencing. Nat Methods  4: 123–125.  <ul class="find" data-citedArticleID="968234" data-doi="10.1038/nmeth995"><li><a href="http://dx.doi.org/10.1038/nmeth995" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+minor+drug-resistant+populations+by+parallel+allele-specific+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+minor+drug-resistant+populations+by+parallel+allele-specific+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000638-Marqulies1" id="pone.0000638-Marqulies1"></a>Marqulies M, Egholm M, Altman WE, Attiva S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="968230"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000638-Havlir1" id="pone.0000638-Havlir1"></a>Havlir D, Eastman S, Gamst A, Richman DD (1996) Nevirapine-resistant human immunodeficiency virus: kinetics of replication and estimated prevalence in untreated patients. J Virol  70: 7894–7899.  <ul class="find" data-citedArticleID="968236"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Nevirapine-resistant+human+immunodeficiency+virus%3A+kinetics+of+replication+and+estimated+prevalence+in+untreated+patients.&amp;auth=&amp;atitle=Nevirapine-resistant+human+immunodeficiency+virus%3A+kinetics+of+replication+and+estimated+prevalence+in+untreated+patients." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nevirapine-resistant+human+immunodeficiency+virus%3A+kinetics+of+replication+and+estimated+prevalence+in+untreated+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nevirapine-resistant+human+immunodeficiency+virus%3A+kinetics+of+replication+and+estimated+prevalence+in+untreated+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000638-Weinstock1" id="pone.0000638-Weinstock1"></a>Weinstock HS, Zaidi I, Heneine W, Bennett D, Garcia-Lerma JG, et al.  (2004) The epidemiology of antiretroviral drug resistance among drug-naive HIV-1 infected persons in 10 U.S. cities. J Infect Dis  189: 2174–80.  <ul class="find" data-citedArticleID="968204" data-doi="10.1086/420789"><li><a href="http://dx.doi.org/10.1086/420789" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+epidemiology+of+antiretroviral+drug+resistance+among+drug-naive+HIV-1+infected+persons+in+10+U.S.+cities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+epidemiology+of+antiretroviral+drug+resistance+among+drug-naive+HIV-1+infected+persons+in+10+U.S.+cities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000638-Cong1" id="pone.0000638-Cong1"></a>Cong M, Heneine W, García-Lerma JG (2007) The fitness cost of mutations associated with human immunodeficiency virus type 1 drug resistance is modulated by mutational interactions. J Virol  81: 3037–3041.  <ul class="find" data-citedArticleID="968220" data-doi="10.1128/jvi.02712-06"><li><a href="http://dx.doi.org/10.1128/jvi.02712-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fitness+cost+of+mutations+associated+with+human+immunodeficiency+virus+type+1+drug+resistance+is+modulated+by+mutational+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fitness+cost+of+mutations+associated+with+human+immunodeficiency+virus+type+1+drug+resistance+is+modulated+by+mutational+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.XML" value="80387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.PDF" value="329232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g001.PNG_L" value="143419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g001.PNG_M" value="242692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g001.PNG_S" value="19564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g001.TIF" value="568066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g001.PNG_I" value="43047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t001.PNG_L" value="496527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t001.PNG_M" value="425722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t001.PNG_S" value="38095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t001.TIF" value="2510576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t001.PNG_I" value="120753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g002.PNG_L" value="52904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g002.PNG_M" value="73418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g002.PNG_S" value="12028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g002.TIF" value="121262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g002.PNG_I" value="22689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t002.PNG_L" value="136261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t002.PNG_M" value="190490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t002.PNG_S" value="15970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t002.TIF" value="585896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.t002.PNG_I" value="25960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g003.PNG_L" value="91206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g003.PNG_M" value="69760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g003.PNG_S" value="14582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g003.TIF" value="209154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g003.PNG_I" value="21252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g004.PNG_L" value="291415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g004.PNG_M" value="369883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g004.PNG_S" value="29252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g004.TIF" value="829090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.g004.PNG_I" value="111578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.s001.PDF" value="13179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.s002.DOC" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000638.s003.DOC" value="28160"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000638&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000638" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000638&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000638&volume=&issue=&title=Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations&author_name=Jeffrey%20A.%20Johnson%2C%20Jin-Fen%20Li%2C%20Xierong%20Wei%2C%20Jonathan%20Lipscomb%2C%20Diane%20Bennett%2C%20Ashley%20Brant%2C%20Mian-er%20Cong%2C%20Thomas%20Spira%2C%20Robert%20W.%20Shafer%2C%20Walid%20Heneine&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000638" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000638" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000638" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000638&amp;t=Simple%20PCR%20Assays%20Improve%20the%20Sensitivity%20of%20HIV-1%20Subtype%20B%20Drug%20Resistance%20Testing%20and%20Allow%20Linking%20of%20Resistance%20Mutations" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000638&title=Simple%20PCR%20Assays%20Improve%20the%20Sensitivity%20of%20HIV-1%20Subtype%20B%20Drug%20Resistance%20Testing%20and%20Allow%20Linking%20of%20Resistance%20Mutations&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000638&amp;title=Simple%20PCR%20Assays%20Improve%20the%20Sensitivity%20of%20HIV-1%20Subtype%20B%20Drug%20Resistance%20Testing%20and%20Allow%20Linking%20of%20Resistance%20Mutations" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000638" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000638&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Simple PCR Assays Improve the Sensitivity of HIV-1 Subtype B Drug Resistance Testing and Allow Linking of Resistance Mutations';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000638';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Simple%20PCR%20Assays%20Improve%20the%20Sensitivity%20of%20HIV-1%20Subtype%20B%20Drug%20Resistance%20Testing%20and%20Allow%20Linking%20of%20Resistance%20Mutations http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000638" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000638" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="24114"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drug+screening%22" title="Search for articles in the subject area:'Drug screening'"><div class="flagText">Drug screening</div></a>
              <div data-categoryid="18917" data-articleid="24114"
                   data-categoryname="Drug screening"
                   class="flagImage" title="Flag 'Drug screening' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="24114"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microbial+mutation%22" title="Search for articles in the subject area:'Microbial mutation'"><div class="flagText">Microbial mutation</div></a>
              <div data-categoryid="20011" data-articleid="24114"
                   data-categoryname="Microbial mutation"
                   class="flagImage" title="Flag 'Microbial mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="24114"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="24114"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="24114"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="24114"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=678'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7934'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7275&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
