

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Influenza A H5N1 Immigration Is Filtered Out at Some International Borders</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001697"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001697" />

    <meta name="citation_title" content="Influenza A H5N1 Immigration Is Filtered Out at Some International Borders"/>
    <meta itemprop="name" content="Influenza A H5N1 Immigration Is Filtered Out at Some International Borders"/>

      <meta name="citation_author" content="Robert G. Wallace"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Irvine, Irvine, California, United States of America"/>
      <meta name="citation_author" content="Walter M. Fitch"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Irvine, Irvine, California, United States of America"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001697.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1697"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Human infection by avian influenza A H5N1.; citation_author=KY Yuen; citation_author=SS Wong; citation_journal_title=Hong Kong Med J; citation_volume=11; citation_number=1; citation_pages=189-199 (2005); citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Three Indonesian clusters of H5N1 virus infection in 2005.; citation_author=IN Kandun; citation_author=H Wibisono; citation_author=ER Sedyaningsih; citation_author= Yusharmen; citation_author=W Hadisoedarsuno; citation_journal_title=N Engl J Med; citation_volume=355; citation_number=2; citation_pages=2186-2194; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Detecting human-to-human transmission of avian influenza A (H5N1).; citation_author=Y Yang; citation_author=ME Halloran; citation_author=J Sugimoto; citation_author=IM Longini; citation_journal_title=Emerg Infect Dis.; citation_volume=13; citation_number=3; citation_pages=1348-1353; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome analysis linking recent European and African influenza (H5N1) viruses.; citation_author=SL Salzberg; citation_author=C Kingsford; citation_author=G Cattoli; citation_author=DJ Spiro; citation_author=DA Janies; citation_journal_title=Emerg Infect Dis; citation_volume=13; citation_number=4; citation_pages=713-718; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Emerging diseases of Africa and the Middle East.; citation_author=M Rweyemamu; citation_author=R Paskin; citation_author=A Benkirane; citation_author=V Martin; citation_author=P Roeder; citation_journal_title=Annals of New York Academy of Sciences; citation_volume=916; citation_number=5; citation_pages=61-70; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Avian influenza H5N1 in Africa: an epidemiological twist.; citation_author=FO Fasina; citation_author=SP Bisschop; citation_author=RG Webster; citation_journal_title=Lancet Infect Dis.; citation_volume=7; citation_number=6; citation_pages=696-697; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Establishment of multiple sublineages of H5N1 influenza virus in Asia: implications for pandemic control.; citation_author=H Chen; citation_author=GJD Smith; citation_author=KS Li; citation_author=J Wang; citation_author=XH Fan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=7; citation_pages=2845-2850; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Emergence and predominance of an H5N1 influenza variant in China.; citation_author=GJD Smith; citation_author=XH Fan; citation_author=J Wang; citation_author=KS Li; citation_author=K Qin; citation_author=JX Zhang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=8; citation_pages=16936-16941; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A statistical phylogeography of influenza A H5N1.; citation_author=RG Wallace; citation_author=H HoDac; citation_author=RH Lathrop; citation_author=WM Fitch; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=9; citation_pages=4473-4478; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MigraPhyla: Statistical Analysis of Migration Events through a Phylogeny. Version 1.0b manual.; citation_author=H HoDac; citation_author=RH Lathrop; citation_author=RG Wallace; citation_author=WM Fitch; citation_number=10; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Spatial aspects of influenza epidemics.; citation_author=AD Cliff; citation_author=P Haggett; citation_author=JK Ord; citation_number=11; citation_date=1986; citation_publisher=Pion; " />
      <meta name="citation_reference" content="citation_title=Genomic analysis and geographic visualization of the spread of avian influenza (H5N1).; citation_author=D Janies; citation_author=AW Hill; citation_author=R Guralnick; citation_author=F Habib; citation_author=E Waltari; citation_journal_title=Syst Biol; citation_volume=56; citation_number=12; citation_pages=321-329; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evolution and adaptation of H5N1 influenza virus in avian and human hosts in Indonesia and Vietnam.; citation_author=GJ Smith; citation_author=TS Naipospos; citation_author=TD Nguyen; citation_author=MD de Jong; citation_author=D Vijaykrishna; citation_journal_title=Virology.; citation_volume=350; citation_number=13; citation_pages=258-268; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The concept of scale and the human dimensions of global change: a survey.; citation_author=GC Gibson; citation_author=E Ostrom; citation_author=TK Ahn; citation_journal_title=Ecological Economics; citation_volume=32; citation_number=14; citation_pages=217-239; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=New strain of influenza A virus (H5N1), Thailand.; citation_author=S Chutinimitkul; citation_author=T Songserm; citation_author=A Amonsin; citation_author=S Payungporn; citation_author=K Suwannakarn; citation_journal_title=Emerg Infect Dis; citation_volume=13; citation_number=15; citation_pages=506-507; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nested clade analyses of phylogeographic data: testing hypotheses about gene flow and population history.; citation_author=AR Templeton; citation_journal_title=Molecular Ecology; citation_volume=7; citation_number=16; citation_pages=381-397; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Statistical phylogeography: methods of evaluating and minimizing inference errors.; citation_author=AR Templeton; citation_journal_title=Molecular Ecology; citation_volume=13; citation_number=17; citation_pages=789-809; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=GeoDis: a program for the cladistic nested analysis of the geographical distribution of genetic haplotypes.; citation_author=D Posada; citation_author=KA Crandall; citation_author=AR Templeton; citation_journal_title=Mol Ecol; citation_volume=9; citation_number=18; citation_pages=487-488; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=UniFrac: A new phylogenetic method for comparing microbial communities.; citation_author=C Lozupone; citation_author=R Knight; citation_journal_title=Applied and Environmental Microbiology; citation_volume=71; citation_number=19; citation_pages=8228-8235; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Market integration and economic development: a long-run comparison.; citation_author=W Keller; citation_author=CH Shiue; citation_journal_title=Review of Development Economics; citation_volume=11; citation_number=20; citation_pages=107-123; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phylogenies and community ecology.; citation_author=CO Webb; citation_author=DD Ackerly; citation_author=MA McPeek; citation_author=MJ Donoghue; citation_journal_title=Annu Rev Ecol Syst; citation_volume=33; citation_number=21; citation_pages=475-505; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Predicting the global spread of H5N1 avian influenza.; citation_author=AM Kilpatrick; citation_author=AA Chmura; citation_author=DW Gibbons; citation_author=RC Fleischer; citation_author=PP Marra; citation_author=P Daszak; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=22; citation_pages=19368-19373; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation of a genotypically unique H5N1 influenza virus from duck meat imported into Japan from China.; citation_author=M Mase; citation_author=M Eto; citation_author=N Tanimura; citation_author=K Imai; citation_author=K Tsukamoto; citation_journal_title=Virology; citation_volume=339; citation_number=23; citation_pages=101-9; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Avian flu: multiple introductions of H5N1 in Nigeria.; citation_author=MF Ducatez; citation_author=CM Olinger; citation_author=AA Owoade; citation_author=S De Landtsheer; citation_author=W Ammerlaan; citation_journal_title=Nature; citation_volume=442; citation_number=24; citation_pages=37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular and antigenic evolution and geographical spread of H5N1 highly pathogenic avian influenza viruses in western Africa.; citation_author=MF Ducatez; citation_author=CM Olinger; citation_author=AA Owoade; citation_author=Z Tarnagda; citation_author=MC Tahita; citation_journal_title=Journal of General Virology; citation_volume=88; citation_number=25; citation_pages=2297-2306; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic variation increases during biological invasion by a Cuban lizard.; citation_author=JJ Kolbe; citation_author=RE Glor; citation_author=LR Schettino; citation_author=AC Lara; citation_author=A Larson; citation_journal_title=Nature; citation_volume=431; citation_number=26; citation_pages=177-181; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Assessing the congruence among distance matrices: single malt scotch whiskies revisited.; citation_author=P Legendre; citation_author=F-J Lapointe; citation_journal_title=Aust N Z J Stat; citation_volume=46; citation_number=27; citation_pages=615-629; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Spatiotemporal distributions of reported cases of the avian influenza H5N1 (bird flu) in Southern China in early 2004.; citation_author=TJ Oyana; citation_author=D Dai; citation_author=KE Scott; citation_journal_title=Avian Diseases; citation_volume=50; citation_number=28; citation_pages=508-515; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Poultry production dynamics, bird migration cycles, and the emergence of highly pathogenic avian influenza in East and Southeast Asia.; citation_author=M Gilbert; citation_author=X Xiao; citation_author=W Wint; citation_author=J Slingenbergh; citation_number=29; citation_date=2007; citation_publisher=Springer Verlag; " />
      <meta name="citation_reference" content="citation_title=Domestic ducks and H5N1 influenza epidemic, Thailand.; citation_author=T Songserm; citation_author=R Jam-on; citation_author=N Sae-Heng; citation_author=N Meemak; citation_author=DJ Hulse-Post; citation_journal_title=Emerging Infectious Diseases,; citation_volume=12; citation_number=30; citation_pages=575-581; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Predicting the global spread of H5N1 avian influenza.; citation_author=AM Kilpatrick; citation_author=AA Chmura; citation_author=DW Gibbons; citation_author=RC Fleischer; citation_author=PP Marra; citation_author=P Daszak; citation_journal_title=Proc Natl Acad Sci U S A.; citation_volume=103; citation_number=31; citation_pages=19368-19373; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Applying the scientific method when assessing the influence of migratory birds on the dispersal of H5N1.; citation_author=PL Flint; citation_journal_title=Virology Journal; citation_volume=4; citation_number=32; citation_pages=132; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The phylogenetic structure of a neotropical forest tree community.; citation_author=SW Kembel; citation_author=SP Hubbell; citation_journal_title=Ecology; citation_volume=87; citation_number=33; citation_pages=S86-S99; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Trait evolution, community assembly, and the phylogenetic structure of ecological communities.; citation_author=NJB Kraft; citation_author=WK Cornwell; citation_author=CO Webb; citation_author=DD Ackerly; citation_journal_title=The American Naturalist; citation_volume=170; citation_number=34; citation_pages=271-283; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Insights into the historical construction of species-rich biomes from dated plant phylogenies, phylogenetic community structure and neutral ecological theory.; citation_author=RT Pennington; citation_author=JA Richardson; citation_author=M Lavin; citation_journal_title=New Phytologist; citation_volume=172; citation_number=35; citation_pages=605-616; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genesis of a highly pathogenic and potentially pandemic H5N1 influenza virus in eastern Asia.; citation_author=KS Li; citation_author=Y Guan; citation_author=J Wang; citation_author=GJD Smith; citation_author=KM Xu; citation_journal_title=Nature; citation_volume=430; citation_number=36; citation_pages=209-213; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The spatiotemporal dynamics of AIDS and TB in the New York Metropolitan region from a sociogeographic viewpoint: understanding the linkages of central city and suburbs.; citation_author=R Wallace; citation_author=D Wallace; citation_author=H Andrews; citation_author=M Fullilove; citation_author=R Fullilove; citation_journal_title=Environment and Planning A.; citation_volume=27; citation_number=37; citation_pages=1085-1108; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Strategies for containing an emerging influenza pandemic in Southeast Asia.; citation_author=NM Ferguson; citation_author=DAT Cummings; citation_author=S Cauchemez; citation_author=C Fraser; citation_author=S Riley; citation_journal_title=Nature; citation_volume=437; citation_number=38; citation_pages=209-214; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Predicting the evolution of human influenza A.; citation_author=RM Bush; citation_author=CA Bender; citation_author=K Subbarao; citation_author=NJ Cox; citation_author=WM Fitch; citation_journal_title=Science; citation_volume=286; citation_number=39; citation_pages=1921-1925; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Long intervals of stasis punctuated by bursts of positive selection in the seasonal evolution of influenza A virus.; citation_author=YI Wolf; citation_author=C Viboud; citation_author=EC Holmes; citation_author=EV Koonin; citation_author=DJ Lipman; citation_journal_title=Biology Direct; citation_volume=1; citation_number=40; citation_pages=34; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stochastic processes are key determinants of short-term evolution in influenza A virus.; citation_author=MI Nelson; citation_author=L Simonsen; citation_author=C Viboud; citation_author=MA Miller; citation_author=J Taylor; citation_journal_title=PLoS Pathogens; citation_volume=2; citation_number=41; citation_pages=1144-1151; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Simultaneous amino acid substitutions at antigenic sites drive influenza A hemagglutinin evolution.; citation_author=A Chun-Chieh Shih; citation_author=T-C Hsiao; citation_author=M-S Ho; citation_author=W-H Li; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=42; citation_pages=6283-6288; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Epochal evolution shapes the phylodynamics of interpandemic influenza A (H3N2) in humans.; citation_author=J Koelle; citation_author=S Cobey; citation_author=B Genfell; citation_author=M Pascual; citation_journal_title=Science; citation_volume=314; citation_number=43; citation_pages=1898-1903; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The generation of influenza outbreaks by a network of host immune responses against a limited set of antigenic types.; citation_author=M Recker; citation_author=OG Pybus; citation_author=S Nee; citation_author=S Gupta; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=44; citation_pages=7711-7716; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Positive Darwinian evolution in human influenza A viruses.; citation_author=WM Fitch; citation_author=X Leiter; citation_author=Q Li; citation_author=P Palese; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=45; citation_pages=4270-4274; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The structure of evolutionary theory.; citation_author=S Gould; citation_number=46; citation_date=2002; citation_publisher=Belknap Press; " />
      <meta name="citation_reference" content="citation_title=Geographical genetics.; citation_author=BK Epperson; citation_number=47; citation_date=2003; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_title=GenBank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=48; citation_pages=D16-D20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The value of a database in surveillance and vaccine selection.; citation_author=C Macken; citation_author=H Lu; citation_author=J Goodman; citation_author=L Boykin; citation_number=49; citation_pages=103-106; citation_date=2001; citation_publisher=Elsevier; " />
      <meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs.; citation_author=R Chenna; citation_author=H Sugawara; citation_author=T Koike; citation_author=R Lopez; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=50; citation_pages=3497-3500; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PAUP*: Phylogenetic Analysis Using Parsimony (And Other Methods) 4.0 beta for Windows/DOS. Sinauer, Sunderland. Sinauer.com.; citation_author=D Swofford; citation_number=51; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ph.D. dissertation, The University of Texas at Austin.; citation_author=DJ Zwickl; citation_number=52; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A cladistic measure of gene flow inferred from the phylogenies of alleles.; citation_author=M Slatkin; citation_author=W Maddison; citation_journal_title=Genetics; citation_volume=123; citation_number=53; citation_pages=603-613; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Mesquite: a modular system for evolutionary analysis.; citation_author=WP Maddison; citation_author=DR Maddison; citation_number=54; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Testing species phylogenies and phylogenetic methods with congruence.; citation_author=M Miyamoto; citation_author=WM Fitch; citation_journal_title=Syst Biol; citation_volume=44; citation_number=55; citation_pages=64-76; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Separating population structure from population history: a cladistic analysis of the geographical distribution of mitochondrial DNA haplotypes in the tiger salamander, Ambystoma tigrinum.; citation_author=AR Templeton; citation_author=E Routman; citation_author=CA Phillips; citation_journal_title=Genetics; citation_volume=140; citation_number=56; citation_pages=767-782; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Nested clade analysis statistics.; citation_author=D Posada; citation_author=KA Crandall; citation_author=AR Templeton; citation_journal_title=Mol Ecol Notes; citation_volume=6; citation_number=57; citation_pages=590-593; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=TCS: a computer program to estimate gene genealogies.; citation_author=M Clement; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Mol Ecol; citation_volume=9; citation_number=58; citation_pages=1657-1659; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation.; citation_author=AR Templeton; citation_author=KA Crandall; citation_author=CF Sing; citation_journal_title=Genetics; citation_volume=132; citation_number=59; citation_pages=619-633; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Empirical tests of some predictions from coalescence theory.; citation_author=KA Crandall; citation_author=AR Templeton; citation_journal_title=Genetics; citation_volume=134; citation_number=60; citation_pages=959-969; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Intraspecific gene genealogies: trees grafting into networks.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Trends Ecol Evol; citation_volume=16; citation_number=61; citation_pages=37-45; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Root probabilities for intraspecific gene trees under neutral coalescent theory.; citation_author=J Castelloe; citation_author=AR Templeton; citation_journal_title=Mol Phylogenet Evol; citation_volume=3; citation_number=62; citation_pages=102-113; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Use of statistical parsimony in HIV analyses.; citation_author=KA Crandall; citation_number=63; citation_pages=186-206; citation_date=1999; citation_publisher=American Mathematical Society; " />
      <meta name="citation_reference" content="citation_title=UniFrac—an online tool for comparing microbial community diversity in a phylogenetic context.; citation_author=C Lozupone; citation_author=M Hamady; citation_author=R Knight; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=64; citation_pages=371; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Exploring the phylogenetic structure of ecological communities: an example for rain forest trees.; citation_author=CO Webb; citation_journal_title=The American Naturalist; citation_volume=156; citation_number=65; citation_pages=145-155; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylocom: software for the analysis of community phylogenetic structure and trait evolution. Version 3.41.; citation_author=CO Webb; citation_author=DD Ackerly; citation_author=SW Kembel; citation_number=66; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Null model analysis of species co-occurrence patterns.; citation_author=N Gotelli; citation_journal_title=Ecology; citation_volume=81; citation_number=67; citation_pages=2606-2621; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Swap algorithms in null model analysis.; citation_author=N Gotelli; citation_author=GL Entsminger; citation_journal_title=Ecology; citation_volume=84; citation_number=68; citation_pages=532-535; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Congruence among distance matrices: Program CADM user's guide.; citation_author=P Legendre; citation_number=69; citation_date=2001; citation_publisher=Département de sciences biologiques, Université de Montréal; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001697" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Influenza A H5N1 Immigration Is Filtered Out at Some International Borders"/>
    <meta name="twitter:description" content="BackgroundGeographic spread of highly pathogenic influenza A H5N1, the bird flu strain, appears a necessary condition for accelerating the evolution of a related human-to-human infection. As H5N1 spreads the virus diversifies in response to the variety of socioecological environments encountered, increasing the chance a human infection emerges. Genetic phylogenies have for the most part provided only qualitative evidence that localities differ in H5N1 diversity. For the first time H5N1 variation is quantified across geographic space.Methodology and Principal FindingsWe constructed a statistical phylogeography of 481 H5N1 hemagglutinin genetic sequences from samples collected across 28 Eurasian and African localities through 2006. The MigraPhyla protocol showed southern China was a source of multiple H5N1 strains. Nested clade analysis indicated H5N1 was widely dispersed across southern China by both limited dispersal and long distance colonization. The UniFrac metric, a measure of shared phylogenetic history, grouped H5N1 from Indonesia, Japan, Thailand and Vietnam with those from southeastern Chinese provinces engaged in intensive international trade. Finally, H5N1's accumulative phylogenetic diversity was greatest in southern China and declined beyond. The gradient was interrupted by areas of greater and lesser phylogenetic dispersion, indicating H5N1 migration was restricted at some geopolitical borders. Thailand and Vietnam, just south of China, showed significant phylogenetic clustering, suggesting newly invasive H5N1 strains have been repeatedly filtered out at their northern borders even as both countries suffered recurring outbreaks of endemic strains. In contrast, Japan, while successful in controlling outbreaks, has been subjected to multiple introductions of the virus.ConclusionsThe analysis demonstrates phylogenies can provide local health officials with more than hypotheses about relatedness. Pathogen dispersal, the functional relationships among disease ecologies across localities, and the efficacy of control efforts can also be inferred, all from viral genetic sequences alone."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001697.g005"/>

  <meta property="og:title" content="Influenza A H5N1 Immigration Is Filtered Out at Some International Borders" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001697" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1486'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9792'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5638&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001697">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001697" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001697&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Influenza A H5N1 Immigration Is Filtered Out at Some International Borders
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert G. Wallace
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rwallace@uci.edu">rwallace@uci.edu</a></p>

                <p>Affiliation:
                  Department of Ecology and Evolutionary Biology, University of California Irvine, Irvine, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Walter M. Fitch
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology and Evolutionary Biology, University of California Irvine, Irvine, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001697</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001697" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001697" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001697" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001697" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001697-g001" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001697-g002" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001697-g003" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001697-g004" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001697-g005" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001697">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001697" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Geographic spread of highly pathogenic influenza A H5N1, the bird flu strain, appears a necessary condition for accelerating the evolution of a related human-to-human infection. As H5N1 spreads the virus diversifies in response to the variety of socioecological environments encountered, increasing the chance a human infection emerges. Genetic phylogenies have for the most part provided only qualitative evidence that localities differ in H5N1 diversity. For the first time H5N1 variation is quantified across geographic space.</p>


<h3>Methodology and Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We constructed a statistical phylogeography of 481 H5N1 hemagglutinin genetic sequences from samples collected across 28 Eurasian and African localities through 2006. The MigraPhyla protocol showed southern China was a source of multiple H5N1 strains. Nested clade analysis indicated H5N1 was widely dispersed across southern China by both limited dispersal and long distance colonization. The UniFrac metric, a measure of shared phylogenetic history, grouped H5N1 from Indonesia, Japan, Thailand and Vietnam with those from southeastern Chinese provinces engaged in intensive international trade. Finally, H5N1's accumulative phylogenetic diversity was greatest in southern China and declined beyond. The gradient was interrupted by areas of greater and lesser phylogenetic dispersion, indicating H5N1 migration was restricted at some geopolitical borders. Thailand and Vietnam, just south of China, showed significant phylogenetic clustering, suggesting newly invasive H5N1 strains have been repeatedly filtered out at their northern borders even as both countries suffered recurring outbreaks of endemic strains. In contrast, Japan, while successful in controlling outbreaks, has been subjected to multiple introductions of the virus.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The analysis demonstrates phylogenies can provide local health officials with more than hypotheses about relatedness. Pathogen dispersal, the functional relationships among disease ecologies across localities, and the efficacy of control efforts can also be inferred, all from viral genetic sequences alone.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wallace RG, Fitch WM (2008) Influenza A H5N1 Immigration Is Filtered Out at Some International Borders. PLoS ONE 3(2):
          e1697.
            doi:10.1371/journal.pone.0001697</p><p><strong>Academic Editor: </strong>Philip Awadalla, University of Montreal, Canada</p><p><strong>Received:</strong> October 16, 2007; <strong>Accepted:</strong> January 17, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> © 2008 Wallace, Fitch. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was supported by National Institutes of Health NIAID grant 1R21AI063275-01.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Highly pathogenic influenza A H5N1 has killed millions of birds from a variety of taxonomic orders. Since 2003, an additional 353 human infections have been confirmed with a mortality rate over 60% (WHO, 24 January 2008). The infection's malignant pathogenesis <a href="#pone.0001697-Yuen1">[1]</a> and what appear to be lengthening, though still limited, chains of human transmission <a href="#pone.0001697-Kandun1">[2]</a>, <a href="#pone.0001697-Yang1">[3]</a> have placed the world on high alert. H5N1 appears to be the leading candidate among several influenza subtypes to evolve a human-to-human infection that sets off a worldwide pandemic.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Since early 2005 avian H5N1 has migrated out of East Asia across Eurasia and as far west as England and West Africa <a href="#pone.0001697-Salzberg1">[4]</a>. There are apparent dangers in the extent of such geographic spread above and beyond starting new outbreaks. First, H5N1 is finding the regions of the world where animal health monitoring remains underdeveloped or degraded by national structural adjustment programs associated with international loans <a href="#pone.0001697-Rweyemamu1">[5]</a>, <a href="#pone.0001697-Fasina1">[6]</a>. Unchecked transmission in these areas increases the genetic variation with which H5N1 can evolve human-specific phenotypes. In spreading over three continents fast-evolving H5N1 also contacts an increasing variety of socioecological environments, including locale-specific combinations of prevalent host types, modes of poultry farming, and animal health measures.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Geographic variation in H5N1 spread, then, appears of essential importance to public health. Whether some localities have been subjected to greater H5N1 immigration is one of several pertinent questions that remain little-addressed. Are some localities more resistant to introgression? Do some countries host greater H5N1 diversity? Circumstantial evidence, derived largely by visually inspecting H5N1 phylogenies and by the relative frequencies of reassortant genotypes, has suggested southern China is the source of, and plays host to, multiple H5N1 strains <a href="#pone.0001697-Chen1">[7]</a>, <a href="#pone.0001697-Smith1">[8]</a>. We present here the first attempt to quantify such diversity across geographic space using a statistical phylogeography of 481 hemagglutinin nucleotide sequences from H5N1 samples isolated 1996–2006 in 28 Eurasian and African localities (<a href="#pone.0001697.s001">Table S1</a>). Specifically, we test the hypothesis H5N1 diversity declines from southern China and that some localities host greater diversity than expected along the spatial cline.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Influenza A H5N1 Migration</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>As a first step we tracked the migration of highly pathogenic H5N1 during its evolutionary history. A phylogenetic tree of the 481 hemagglutinin sequences was constructed via gamma-corrected, general time-reversible maximum likelihood (<a href="#pone.0001697.s003">Figure S1</a>). The migration events through the tree were traced with MigraPhyla <a href="#pone.0001697-Wallace1">[9]</a>, <a href="#pone.0001697-HoDac1">[10]</a>. The method infers the localities of the tree nodes by maximum parsimony, and by Monte Carlo simulation tests the significance of migration events that accrue back and forth between all pairs of localities over time.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>MigraPhyla shows that the Chinese province of Guangdong was a primary source for multiple regional and international H5N1 migration events, including for all four regional clades: 1, 2.1, 2.2, and 2.3 (<a href="#pone-0001697-g001">Figure 1</a>) <a href="#pone.0001697-Chen1">[7]</a>, <a href="#pone.0001697-Wallace1">[9]</a>. With only about half the number of viruses sequenced as Guangxi, Thailand, and Indonesia, in this analysis Guangdong accounted for 41 of the 115 (36%) total migration events inferred (<a href="#pone.0001697.s004">Figure S2</a>). The results may be an artifact of the method of inference, which searches for the most parsimonious distribution of ancestral localities through the original genetic tree, often favoring dispersal from a single or limited number of epicenters <a href="#pone.0001697-Wallace1">[9]</a>. However, Guangdong's widespread representation across multiple tree clades for both hemagglutinin and neuraminidase is unmistakable (<a href="#pone.0001697.s005">Figures S3</a> and <a href="#pone.0001697.s006">S4</a>), imparting the province an integral role in H5N1's epidemiology. Even if H5N1 strains emerged elsewhere in the region instead, Guangdong's socioeconomic centrality may have acted as an epidemiological attractant, drawing in novel poultry trade-borne strains from around southern China before dispersing them again back out across China and beyond <a href="#pone.0001697-Cliff1">[11]</a>.</p>
<div class="figure" id="pone-0001697-g001"><div class="img"><a name="pone-0001697-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>H5N1 Migration as Inferred by Parsimony.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Map of H5N1 migration events inferred by parsimony through maximum likelihood phylogeny for 481 hemagglutinin nucleotide sequences sampled across 28 Eurasian and African localities 1996–2006 (2≤<em>n</em>≤65 isolates per locality). Orange vectors are statistically significant (α = .05) under an upper-tail Monte Carlo test of 10,000 trials and a sparse false discovery rate (sFDR) correction. Non-significant vectors are color-coded by Monte Carlo <em>P</em> value: the brighter the yellow, the greater the support. Quintiles are defined by breaks in ranked <em>P</em> values of more than .01, except within the final quintile. The map is based on satellite photos made available in World Wind 1.4 (<a href="http://worldwind.arc.nasa.gov/">http://worldwind.arc.nasa.gov/</a>).</p>
<span>doi:10.1371/journal.pone.0001697.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The phylogeography of H5N1 involves much more than spread from a primary epicenter. Regional dynamics are apparent, including among Chinese provinces and in Africa (<a href="#pone-0001697-g001">Figure 1</a>). Secondary epicenters, such as Qinghai and Nigeria, seeded tertiary outbreaks, including, in the case of Novosibirsk, back east into China <a href="#pone.0001697-Janies1">[12]</a>. The origins of other outbreaks, on the other hand, appear more straight-forward. In spite of a ten-fold increase in the number of samples from our previous analysis, Indonesia's epidemic still appears founded by a single migration event from southern China <a href="#pone.0001697-Wallace1">[9]</a>, <a href="#pone.0001697-Smith2">[13]</a>. A few viruses from a single batch of infected poultry could very well have acted as the progenitors of outbreaks now stretching across 31 of Indonesia's 33 provinces.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>As patterns of migration and trade within countries can qualitatively differ from those among countries <a href="#pone.0001697-Gibson1">[14]</a>, we next explored H5N1 spread within China. A section of the original hemagglutinin maximum likelihood tree was used to map the migration events for 67 samples of a single clade 2.3 H5N1 strain circulating in southern China 2005–2006 (<a href="#pone.0001697.s001">Table S1</a>, <a href="#pone-0001697-g002">Figure 2a,b</a>). Over the course of a year, the strain traversed much of southern China and moved into Thailand <a href="#pone.0001697-Chutinimitkul1">[15]</a>. The result was reproduced by nested clade analysis <a href="#pone.0001697-Templeton1">[16]</a>, <a href="#pone.0001697-Templeton2">[17]</a>, another phylogeographic method (<a href="#pone-0001697-g002">Figure 2c</a>). For 12 of 16 nested clades with geographic variation across H5N1 genotypes, nested clade analysis's permutation tests failed to reject the null hypothesis that H5N1 was randomly distributed across southern China. Four of the five subclades whose nested clade distances were significantly different from randomness proved more widely dispersed from their sister subclades than expected by chance. Templeton's <a href="#pone.0001697-Templeton2">[17]</a>, <a href="#pone.0001697-Posada1">[18]</a> inference key showed the significant clades to be defined alternatively by restricted dispersal with genetic isolation by distance and long-distance colonization and expansion. A closely related co-circulating strain, as represented by 49 clade 2.3 samples isolated 2004–2006, engaged in similarly extensive dispersal (<a href="#pone-0001697-g002">Figure 2d,e</a>).</p>
<div class="figure" id="pone-0001697-g002"><div class="img"><a name="pone-0001697-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>H5N1 Migration in Southern China.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>a) H5N1 migration events as assigned by parsimony for 67 samples of a single clade 2.3 H5N1 strain circulating in southern China 2005–2006 and b) mapped onto a World Wind satellite photo. c) Statistical parsimony haplotype network for same 67 samples. Rectangle represents the hypothesized ancestral genotype. All genotypes, except two, are represented by a single isolate. The exceptions are represented by two isolates (‘2’). Squares represent significant nested clade distances and triangles significant within clade distances as determined by nested clade analysis. Black shapes represent nodes at which subclades express significantly greater distances than expected by chance begin and extend to the tips of the branch. Gray shapes represent the starting node of a subclade of lesser distance than expected by chance. d) Migration events through section of H5N1 maximum likelihood tree showing 49 samples of a single clade 2.3 H5N1 strain circulating in southern China 2004–2006 and sister group to the 67 isolates shown in (a). e) The migration events for the 49 isolates mapped onto a World Wind satellite photo.</p>
<span>doi:10.1371/journal.pone.0001697.g002</span></div>

<h4>Evolutionary History Shared Across Localities</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In tracing migration events through a genetic tree we can learn much about H5N1's dispersal patterns, including their extent and magnitude. However, there remains little understanding of the functional relationships among the localities' disease ecologies. To clarify some of the mechanisms involved in H5N1 spread we conducted two additional analyses. In the first, we determined which Eurasian and African localities hosted similar H5N1 evolutionary histories by conducting a principal components analysis on the UniFrac metric for the full hemagglutinin phylogenetic tree. UniFrac measures the fraction of the phylogenetic distance leading to samples of either one of two localities, but not to both <a href="#pone.0001697-Lozupone1">[19]</a>. It measures the branch length unique to each locality. Conversely, the pair of localities with the smallest UniFrac distance shares the greatest amount of evolutionary history.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The first three principal components accounted for 68% of the total variation in shared phylogenetic history across localities (<a href="#pone-0001697-g003">Figure 3</a>). PC1 separated eastern Chinese localities from several nearby Asian countries and, farther west, areas infected by the Qinghai-like H5N1 strain. In effect, PC1 reproduces the major patterns depicted in the MigraPhyla map (<a href="#pone-0001697-g001">Figure 1</a>), with nearby localities sharing H5N1 evolutionary history. We learn in addition that China's H5N1 strains group together north and south to the exclusion of those outside the country, with one exception. Hunan, a southern province, grouped with the other Asian countries: Indonesia, Japan, Thailand, and Vietnam. As measured here Hunan appears peripheral to a phylogeographic structure arrayed over northern and southern China.</p>
<div class="figure" id="pone-0001697-g003"><div class="img"><a name="pone-0001697-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>UniFrac PCA of H5N1 Shared History by Locality.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>First three principal components for PCA of UniFrac metric for 481 H5N1 hemagglutinin sequences across 28 localities. Localities in red are situated in southern China, in dark red southwestern China (Hunan, Guizhou, Yunnan, and Guangxi), in pink northern China, in green in nearby Asian countries (Indonesia, Japan, Thailand, and Vietnam), and in olive green regions reached by the Qinghai-like strain (e.g., Mongolia, Crimea, Iraq, Africa). Localities referred to in the text include FJ = Fujian, GD = Guangdong, HK = Hong Kong, HU = Hunan, IN = Indonesia, IT = Italy, JA = Japan, TH = Thailand, VN = Vietnam.</p>
<span>doi:10.1371/journal.pone.0001697.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>On the other hand, PC2 divided northern from southern China, implying that however integrated the two H5N1 socioecologies may be, each region also hosts its own localized diffusion. The other Asian countries nearby clustered together along this second axis with southern China, in particular with Guangdong, Fujian, and Hong Kong, the southern face of China's international trade, indicating a shared phylogeography. Chinese provinces farther west (Hunan, Guizhou, Yunnan, and Guangxi), and historically less integrated with trade <a href="#pone.0001697-Keller1">[20]</a>, clustered on their own. Finally, localities infected by the Qinghai-like strain tightly clustered in the PCA space. In a matter of months, across three continents, the Qinghai-like strain successfully invaded a large section of what was, until 2005, H5N1-free territory <a href="#pone.0001697-Salzberg1">[4]</a>.</p>


<h4>Phylogenetic Structure Across Localities</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We next tested the extent to which the H5N1 isolates were distributed across the phylogenetic tree. Some localities may host samples from widely disparate phylogenetic clades, while other localities may host samples from a single clade alone. Two measures of phylogenetic dispersion, the net relatedness index (NRI) and the nearest taxon index (NTI), were rank-ordered by great circle distance from Guangdong, a putative primary epicenter for H5N1 (<a href="#pone-0001697-g004">Figures 4</a> and <a href="#pone.0001697.s007">S5</a>). The indices represent standardized effect sizes of community phylogenetic structure <a href="#pone.0001697-Webb1">[21]</a>. NRI is defined for the mean phylogenetic distance (MPD) among a locality's samples, measuring dispersion over the whole tree. NTI is defined for the mean phylogenetic distance between nearest tree neighbors (MNND), measuring the extent to which locality samples are clustered at the tree's tips.</p>
<div class="figure" id="pone-0001697-g004"><div class="img"><a name="pone-0001697-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Phylogenetic Dispersion Across H5N1 Localities.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Net relatedness index (NRI) across 28 localities listed by great circle distance (<em>km</em>) from Guangdong. The upper-tail <em>P</em> values for the Monte Carlo test for phylogenetic clustering are colorized along a grayscale and shown in boxes at the tips of their respective index bars.</p>
<span>doi:10.1371/journal.pone.0001697.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The great majority of localities in the study were defined by statistically significant phylogenetic clustering associated with local evolutionary radiation. However, some localities were less clustered than others. The NRI results show localities within China listed down through Hebei in <a href="#pone-0001697-g003">Figure 3</a>, except Hunan, tended to be less phylogenetically clustered than localities below Hebei. NRI for two of these localities, Guangdong (NRI = 0.9141) and Guizhou (0.6794), were statistically indistinguishable from random dispersion. The lesser clustering for areas nearer Guangdong agrees with previous work indicating southern Chinese provinces have hosted multiple H5N1 strains <a href="#pone.0001697-Fasina1">[6]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>On the other hand, Vietnam (8.5366) and Thailand (11.5646), although relatively close to Guangdong, showed extreme phylogenetic clustering (<em>P</em> = 0). The result suggests an as yet unknown mechanism has repeatedly filtered out H5N1 as the virus moved from southern China towards or into Indochina. Whether the filter is embodied by customs regulation, limited transportation infrastructure, or the vagaries of the poultry market remains an open question. The filtering occurred even as repeated outbreaks accrued within Indochina, among regional localities <a href="#pone.0001697-Wallace1">[9]</a>, and a limited number of migration events were recorded between the two areas.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Other borders appear less resistant to invasion when exposed. Japan (NRI = 4.0085) hosted widely differing H5N1 strains that originated in northern and southern China (<a href="#pone-0001697-g001">Figure 1</a>), by way of South Korea and China direct <a href="#pone.0001697-Kilpatrick1">[22]</a>, <a href="#pone.0001697-Mase1">[23]</a>, and, in late 2006, the Qinghai-like strain. Nigeria (4.9317) has hosted multiple introductions of the strain <a href="#pone.0001697-Ducatez1">[24]</a>. Nigeria's greater NRI score may have resulted in part from how quickly introgressions of related haplotypes followed each other into the country. Ducatez <em>et al</em>. <a href="#pone.0001697-Ducatez2">[25]</a> report near-simultaneous outbreaks in Nigeria were caused by different Qinghai-like sublineages.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>At the other extreme, Italy's (NRI = −4.2206) H5N1 samples were significantly evenly distributed across the phylogeny. Italy hosted two Qinghai-like strains and what appears to be an older haplotype (A/mallard/Italy/3401/2005(H5N1)) from Guangxi direct (<a href="#pone-0001697-g001">Figure 1</a>). The Guangxi connection, however, is likely an artifact of long-branch attraction in the original gene tree (<a href="#pone.0001697.s003">Figure S1</a>). Chinese province Hebei (−0.4348) is nearly, though not significantly, evenly distributed. The three lone Hebei isolates included in the study were derived from different strains and localities (<a href="#pone-0001697-g001">Figure 1</a>) <a href="#pone.0001697-Wallace1">[9]</a>. Multiple introductions from several sources can provide a population sink greater genetic variation than expressed by any single native source <a href="#pone.0001697-Kolbe1">[26]</a>.</p>


<h4>Spatial Correlation and Sampling Effort</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Our analyses of UniFrac and the indices of phylogenetic dispersion did not directly test for spatial correlation. To determine how the phylogenetic measures reviewed here were related to each other, to spatial distance, and to sample size, we conducted two convergence tests and a pairwise Mantel test for correspondence <a href="#pone.0001697-Legendre1">[27]</a> for six distance matrices defined for the H5N1 hemagglutinin phylogeny. The six matrices were 1) MigraPhyla migration events through the phylogeny symmetricized across the migration matrix diagonal, showing average two-way traffic between localities; 2) differences in isolate sample size; 3) geographic distances; 4) UniFrac distances; 5) differences in MPD; and 6) differences in MNND.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The test for global convergence across distance matrices (CADM) rejected overall incongruence (<em>P</em> = .0001)—at least two of the matrices converged (<a href="#pone.0001697.s002">Table S2</a>). The <em>a posteriori</em> convergence tests we next conducted failed to reject the null hypothesis that the matrix of symmetricized migration events (<em>P</em> = .30) was incongruent with the other five matrices. The generalized Mantel test, based on Spearman correlation of ranks and a false discovery rate correction, identified seven pairwise comparisons as significantly convergent (<a href="#pone-0001697-g005">Figure 5</a>). In rank order of the strength of correlation, the significant pairs were MPD-UniFrac (<em>r</em> = .93), MPD-MNND (.78), MNND-UniFrac (.74), geographic distance-MNND (.29), geographic distance-MPD (.27), geographic distance-UniFrac (.23), and sample size difference-MNND (.22).</p>
<div class="figure" id="pone-0001697-g005"><div class="img"><a name="pone-0001697-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001697" data-uri="info:doi/10.1371/journal.pone.0001697.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001697.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001697.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001697.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001697.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Generalized Mantel Test Across H5N1 Distance Matrices.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Pairwise Mantel test for correspondence based on Spearman correlation ranks for six distance matrices defined for H5N1 phylogeny of 481 hemagglutinin nucleotide sequences across 28 Eurasian and African localities: 1) migration events through the phylogeny symmetricized across the migration matrix diagonal (SME), 2) differences in isolates sample size (SS), 3) geographic distances (Geo), 4) UniFrac distances, 5) mean phylogenetic distances (MPD), and 6) mean nearest neighbor distances (MNND). Correlations are blocked out in color and upper-tail <em>P</em> values in gray scale. <em>P</em> values for pairwise comparisons between localities significant under false discovery rate correction are boxed in gold borders.</p>
<span>doi:10.1371/journal.pone.0001697.g005</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The Mantel tests showed that the shared evolutionary history and phylogenetic dispersion for these H5N1 localities were, as of 2006, tightly associated. Indeed, the third principal component in the UniFrac PCA (<a href="#pone-0001697-g002">Figure 2</a>), which we left unexplained above, strongly correlated with NRI (<em>r</em> = .66). A second result supported what the NRI results alone suggested: phylogenetic dispersion appears spatially autocorrelated, with important exceptions (<em>e.g.</em>, Italy, Iraq). Finally, MNND was dependent on sample size and significantly varied with sampling effort. MPD, however, did not, implying that the conclusion southern China hosted greater overall H5N1 diversity is statistically unrelated to differential sampling across space.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our MigraPhyla analysis through 2006 agreed with previous work showing several regional and international strains of highly pathogenic influenza A H5N1 originated in southern China (<a href="#pone-0001697-g001">Figure 1</a>). Spatially finer analysis indicated novel outbreaks widely dispersed across southern China over the course of a year, by both limited dispersal and longer strikes (<a href="#pone-0001697-g002">Figure 2</a>). Oyana <em>et al.</em> <a href="#pone.0001697-Oyana1">[28]</a> showed the time of dispersal through China following an initial outbreak may be compressed several orders of magnitude to a matter of weeks. The exact mechanism for H5N1 spread through China remains only loosely documented and requires additional attention. Migratory birds, the poultry trade, and the transport of free-range ducks have all been implicated in outbreaks across Southeast Asia <a href="#pone.0001697-Chen1">[7]</a>, <a href="#pone.0001697-Gilbert1">[29]</a>, <a href="#pone.0001697-Songserm1">[30]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>A principal components analysis of the UniFrac metric identified combinations of localities that shared the greatest amount of H5N1 evolutionary history (<a href="#pone-0001697-g003">Figure 3</a>). UniFrac has so far been largely applied to determining the effects of environment type on microbial evolution, particularly for bacteria. This appears the first time the metric has been used in a strictly phylogeographic context. The method offers a means by which to identify groups of localities, at a variety of spatial scales, whose isolates evolve together out from a local common ancestor.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The PCA showed variation in H5N1's shared history segregated along orthogonal axes defined by separate phylogeographic processes. First, localities in China grouped to the exclusion of nearby Asian countries, the rest of Eurasia, and Africa. Along the second axis northern and southern China separated. In spite of clear epidemiological interaction between the two Chinese regions, each also hosted its own localized diffusion. Indonesia, Japan, Thailand and Vietnam also grouped with southern China, particularly with southeastern provinces engaged in intensive international trade. The latter result suggests H5N1 migration among these areas is largely structured by poultry shipments, even as migratory birds may also seed local outbreaks (<em>e.g.,</em> Lake Qinghai) <a href="#pone.0001697-Chen1">[7]</a>, <a href="#pone.0001697-Kilpatrick2">[31]</a>, <a href="#pone.0001697-Flint1">[32]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The indices of dispersion reported here, another phylogenetic measure, show H5N1's accumulative diversity drops off from southern China, the putative region of origin. The step gradient is punctuated by areas of greater and lesser phylogenetic spread. These interruptions might arise from a variation in epidemiological permeability across provincial and international borders. Some localities, Thailand and Vietnam for example, appear more resistant to H5N1 introduction than others. Identifying these localities—and their relevant borders—may aid efforts at learning what socioecological factors block (or enable) H5N1 migration at a variety of spatial scales. The methods presented here may also provide some of the metrics needed to define criteria for bird flu mitigation. For instance, localities with diminishing increases in accumulative phylogenetic dispersion may increasingly filter out H5N1 immigration.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>H5N1 UniFrac and phylogenetic dispersion appeared strongly correlated. We propose that localities in regions newly invaded by a single strain share both viral phylogenetic history and, by definition, limited dispersion (no other strains have yet invaded). The most obvious example is that of the Qinghai-like strain that dominates Europe and Africa. Similarly, local areas subjected to repeated invasions from a nearby source share many related strains before subsequent differentiation by isolation by distance accrues with additional geographic spread. Localities abutting sources, then, share both phylogenetic history and a comparatively evener dispersion. There are important exceptions for H5H1, as Thailand and Vietnam demonstrate.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>This appears the first time measures of phylogenetic community structure have been derived from the genetic sequences of a single spreading pathogen. The indices of phylogenetic dispersion have typically been applied at the community level (<em>e.g.</em>, among plant species in a neotropical forest) <a href="#pone.0001697-Kembel1">[33]</a>. The H5N1 results did agree with what are emerging as general principles in studies of community assembly. We discuss two here. By simulation analysis Kraft <em>et al.</em> <a href="#pone.0001697-Kraft1">[34]</a> show the process of habitat filtering that produces phylogenetic clustering is best detected by the net relatedness index than by the nearest taxon index, as was the case here for H5N1. The assembly rules for H5N1 diversity, however, may arise as much from biogeographic processes limiting dispersal as from socioecological factors filtering out emerging strains <a href="#pone.0001697-Pennington1">[35]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The balance between the two processes may shift back and forth. A H5N1 strain that spreads out to a broader regional scale temporally diminishes the effects of isolation by distance, including on measures of phylogenetic dispersion. Since its emergence highly pathogenic H5N1 has undergone a series of demic selection events wherein a strain emerges to outcompete indigent strains across a wide spatial extent. The Z reassortant outcompeted at least five other strains to emerge as the dominant genotype in East Asia <a href="#pone.0001697-Li1">[36]</a>. The Qinghai-like strain has dispersed across Eurasia and into Africa <a href="#pone.0001697-Salzberg1">[4]</a>. The Fujian-like strain is the most recent to be detected <a href="#pone.0001697-Smith1">[8]</a>. For areas already hosting endemic bird flu, a newly dispersed strain will increase phylogenetic evenness. For areas never before invaded, an expanding strain by definition causes significant phylogenetic clustering, in this case, accompanied by a shared phylogenetic history (see localities infected by the Qinghai-like strain in <a href="#pone-0001697-g003">Figure 3</a>). Areas that evade invasion by a widely dispersed strain altogether may do so by chance, but may instead do so by virtue of a socioecological filter blocking entry.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In spreading, H5N1 changes the relationship between phylogenetic dispersion and shared history across localities. A spreading pathogen also changes the combination of interventions necessary for its control <a href="#pone.0001697-Wallace2">[37]</a>. Intervention strategies that may have worked when H5N1 outbreaks were confined to Southeast Asia <a href="#pone.0001697-Ferguson1">[38]</a> are for the most part rendered obsolete. A stockpile of oseltamivir or vaccines amassed for blockading any potential outbreak of human-specific strains in East Asia becomes of dubious utility with bird flu spread across—and evolving in—Eurasia and Africa. Intervention strategies must undergo qualitative changes when the epidemic's increasing spatial extent changes the mix of locally endemic strains.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Kraft <em>et al.</em> also show habitat filtering generates stronger phylogenetic structure when novel derived traits are favored, as appears the case in influenza. Strains of seasonal influenza A H3N2 with the greatest change at 18 HA1 codons serve as progenitors for the next year's dominant strain <a href="#pone.0001697-Bush1">[39]</a>. Debate continues over whether the positive selection to which antigenic sites are subjected is continual or punctuated <a href="#pone.0001697-Wolf1">[40]</a>–<a href="#pone.0001697-ChunChiehShih1">[42]</a>. Other work suggests influenza evolution results from punctuated changes in antigenic properties rather than genetic change alone <a href="#pone.0001697-Koelle1">[43]</a>. Yet another model proposes these antigenic changes arise from shifts in host population immunity <a href="#pone.0001697-Recker1">[44]</a>. By whatever mechanism antigenic drift emerges, an accumulation of derived traits driven by host immune response appears favored.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Influenza's accelerated evolution, however, is complicated by multiple reversals and parallelisms, with specific amino acid residues evolving time and again <a href="#pone.0001697-Wolf1">[40]</a>, <a href="#pone.0001697-Fitch1">[45]</a>. Parallelisms and correlated evolution among residue sites may allow H5N1 to infect cycles of host species repeatedly encountered across geographic space <a href="#pone.0001697-Wallace1">[9]</a>. The homoplasies may complicate phylogenetic reconstruction, but the capacity for parallel changes can be considered a character itself, a notion that harkens back to E. Ray Lankester's original intent in defining ‘homoplasy’ <a href="#pone.0001697-Gould1">[46]</a>. Homoplasies are a real feature of influenza's biology and can be thought of as a type of homology, not its antonym. Indeed, for certain influenzas, parallel changes may serve as a predictor of the emergence of a new dominant strain <a href="#pone.0001697-Wolf1">[40]</a>. The relationships among influenza's molecular evolution, phylogenetic structure, and geographic spread remain an active field of research.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>The methodologies described here may benefit studies of other animal and human pathogens. The phylogeographic dynamics of a variety of newly emergent threats such as HIV, SARS, and hantavirus, as well as those of deeper historical origin, such as HTLV and KSHV, may be better illuminated by the approach. For many of these pathogens the key problem for control efforts may not be in determining whether these phylogenetic variables are spatially autocorrelated, but in explaining their residuals, what spatial distance does not explain <a href="#pone.0001697-Epperson1">[47]</a>. Assuming pathogen spread is in part deterministic, arising from specific causes rather than by chance alone, localities with greater and lesser dispersion than expected may be more exposed to, or protected from, new strains of the disease under investigation. In using genetic sequences to learn which of such outcomes prevail where, health officials may be able to better track the finer patterns in pathogen spread.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>From GenBank <a href="#pone.0001697-Benson1">[48]</a> and the Influenza Sequence Database <a href="#pone.0001697-Macken1">[49]</a> we downloaded 481 influenza A H5N1 hemagglutinin and 429 neuraminidase nucleotide sequences isolated from a variety of host species 1996–2006 across 28 Eurasian and African localities (<a href="#pone.0001697.s001">Table S1</a>). The hemagglutinin sequences used in this study ranged over 1716 positions, indels included, covering all of the precursor peptide, HA1 and HA2, except HA2's final six nucleotide sites. The neuraminidase sequences sampled for this study covered the entire monomer, across 1419 sites, indels included. Neuraminidase sequences were downloaded only for hosts that also contributed hemagglutinin sequences to the study.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Our analyses used a convenience sample of sequences made publicly available by a variety of researchers from around the world, sampling at a variety of geographic scales. The localities sampled were represented by a variable number of isolates (<em>n</em> = 2 to 65) and included the Chinese city of Shanghai; the Hong Kong Special Administrative Region of China; the Chinese provinces of Hebei, Henan, Hubei, Hunan, Guangdong, Guangxi, Guizhou, Fujian, Jilin, Qinghai, and Yunnan; the countries of Burkina Faso, Cote d'Ivoire, Egypt, Indonesia, Iraq, Italy, Japan, Mongolia, Niger, Nigeria, Sudan, Thailand, and Vietnam; the Russian city of Novosibirsk and its environs; and the Crimean region. Duplicate sequences sampled from the same locality add no phylogeographic information to the analysis and were not included among the 481 hemagglutinin samples used. On the other hand, duplicate sequences from different localities were included, as they are by definition differentiated by a migration event.</p>


<h4>Phylogeny</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The nucleotide sequences were aligned in ClustalW <a href="#pone.0001697-Chenna1">[50]</a>. Leading and trailing gaps from truncated sequencing were treated as missing data. Gaps within the sequences (<em>e.g.</em>, of a length of 19–20 residues in the neuraminidase stalk) were treated as real indels and defined as a fifth character state. Tree-building was a two-step process: A preliminary input tree was first constructed by maximum parsimony in PAUP* 4.0b10 <a href="#pone.0001697-Swofford1">[51]</a>. Character state changes were ordered by outgroup comparison with isolate A/chicken/Scotland/59(H5N1). Most-parsimonious trees were heuristically generated by stepwise addition and branch swapping via tree-bisection-recombination. One tree was chosen by visual inspection as representative of the set of 1000 and used as the preliminary input tree for the construction of a maximum likelihood tree. The maximum likelihood tree was created via a gamma-corrected, general time-reversible model in GARLI <a href="#pone.0001697-Zwickl1">[52]</a>. Bootstrap values were derived in PAUP by a fast-heuristic search of 1000 replicates.</p>


<h4>Statistical Phylogeography</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>As in Slatkin and Maddison <a href="#pone.0001697-Slatkin1">[53]</a>, the localities of the isolates in the resulting gene tree were assigned to the tips as a single character with 28 states. Moving recursively up the tree, PAUP* 4.0 assigned the localities of ancestral nodes by maximum parsimony, such that the tree supported the fewest possible migration events between localities consistent with the gene tree. PAUP's assignments may represent only one of several possible most-parsimonious traces. For the hemagglutinin tree several nodes support multiple localities (<a href="#pone.0001697.s005">Figure S3</a>) <a href="#pone.0001697-Maddison1">[54]</a>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>For each gene, the total number of migration events occurring through the tree as assigned by PAUP was tallied for every possible pair of localities using the MigraPhyla protocol <a href="#pone.0001697-Wallace1">[9]</a>, <a href="#pone.0001697-HoDac1">[10]</a>. A Monte Carlo test of 10,000 trials was conducted to determine the probability that the frequencies of migration events between each pair of localities in the original migration tree were more than expected when the localities were randomly distributed across the tree's tips. The localities of the real data sets were randomized without changing the sample sizes for each locality. The migration events among localities through the gene tree were inferred by DELTRAN optimization and the migration events among localities summed for each randomized trial. Under DELTRAN, character state changes in the phylogeny—here, migration events—are delayed and concentrated toward the tips of the tree. To test the significance of the resulting <em>P</em> values across all localities, α, the nominal Type 1 error rate, was controlled for multiple tests across locality pairs with a sparse false discovery rate correction <a href="#pone.0001697-Wallace1">[9]</a>. The series of programs for running the MigraPhyla protocol are currently available at <a href="http://pd.bio.uci.edu/ee/WallaceR/MigraPhyla.html">http://pd.bio.uci.edu/ee/WallaceR/MigraP​hyla.html</a>.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>To determine the degree to which the hemagglutinin and neuraminidase differed in migration structure we reran the DELTRAN migration analysis on the 429 H5N1 isolates for which hemagglutinin and neuraminidase sequences (HA429 and NA429) were both available (<a href="#pone.0001697.s001">Table S1</a> and <a href="#pone.0001697.s006">Figure S4</a>). All localities from the HA481 analysis were included, except Burkina Faso, for which no neuraminidase sequences were available at the time of analysis. As a first effort at exploring the effects of sampling schema on our migration estimates, we also included the HA481 migration matrix without migration events to and from Burkina Faso. To compare the three matrices' structure we conducted a singular value decomposition of each matrix and calculated the angles at which each resulting eigenvector differed between the genes <a href="#pone.0001697-Wallace1">[9]</a>. Specifically, we calculated the arccos of the scalar product of the resulting eigenvectors, divided by the product of their norms, contrasting the three migration matrices in round robin fashion. Many of the eigenvectors differed orthogonally (<a href="#pone.0001697.s008">Figure S6</a>). HA429 and HA481 showed the least convergence, indicating changing sampling effort across localities can change the resulting migration structure. A fuller sensitivity analysis exploring the effects of sampling on migration estimates is planned.</p>


<h4>Methodological Equivalence</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>In an effort at testing for methodological equivalence <a href="#pone.0001697-Miyamoto1">[55]</a>, we compared a map of migration events derived by MigraPhyla for 67 Chinese clade 2.3 hemagglutinin sequences isolated 2005–2006 to the results of a nested clade analysis (NCA) <a href="#pone.0001697-Templeton1">[16]</a>, <a href="#pone.0001697-Templeton2">[17]</a>, <a href="#pone.0001697-Templeton3">[56]</a>, <a href="#pone.0001697-Posada2">[57]</a> of the same isolates.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>For the MigraPhyla analysis, the node localities—and their associated migration events—were retained from the original tree of 481 sequences. For the NCA analysis a gene network of the 67 isolates was first constructed via statistical parsimony in TCS 1.21 <a href="#pone.0001697-Clement1">[58]</a>. Traditional phylogenies can fail to account for many of the inherent properties of intraspecific genealogies, such as extant ancestral taxa, multifurcated tree branching, and true homoplasies brought about by recombination. These properties can be assimilated by a parsimonious network of isolates with statistically supported connections <a href="#pone.0001697-Templeton4">[59]</a>–<a href="#pone.0001697-Posada3">[61]</a>. The best network expresses a high probability (≥95%) its linkages—regardless of the number of nucleotide steps between isolates—are true. In short, the network assimilates statistically supported deviations from parsimony. TCS also calculates the root probabilities for each isolate using Castelloe and Templeton's <a href="#pone.0001697-Castelloe1">[62]</a> heuristic.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>Loops in the network represent homoplasies, including possible recombination events. The method hypothesized isolate A/chicken/Guangxi/463/2006(H5N1), with the greatest root probability, to be the ancestral haplotype, almost certainly incorrect with many isolates sampled in 2005 also included here. As expected for an RNA pathogen with a high mutation rate and a low sample-to-population ratio, the resulting network also contained many more missing haplotypes than such networks typically represent. Viral haplotype networks have for the most part been restricted to characterizing within-patient evolution, the diversity from which is comparatively restricted <a href="#pone.0001697-Crandall2">[63]</a>.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>In spite of these complications, the geographic associations among the haplotypes were tested via NCA. To statistically test for associations between the parsimonious network and geography the branches are first organized in a nested fashion by single steps in the network. The nesting begins at the tip isolates and includes those internal taxa one step away—the 0-step—from the terminal. These are subsequently nested into a nest of 1-step clades, then 2-step clades, and so on, until the next nest would include all the isolates inside. Two distances are now constructed: the clade distance, measuring the geographic spread of a clade (say, within a 1-step nest); and the nested clade distance, measuring the geographic distances among clades within the same nested category (among the 1-step nest clades within their 2-step nest).</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>With GeoDis 2.5 <a href="#pone.0001697-Posada1">[18]</a> the statistical significance of the two distances across the clade nests was determined by contrasting them with nonparametric distributions of distances generated by Monte Carlo randomizations of the data, under the constraint haplotype frequencies and sample sizes are preserved <a href="#pone.0001697-Posada2">[57]</a>. If patterns are significant they require biological interpretation. Templeton <a href="#pone.0001697-Templeton2">[17]</a> provides a key (updated on the GeoDis website) with which the distribution of significant results across the nests can be used to discern which evolutionary process likely caused the results at each nest level. The factors potentially influencing haplotype distribution include ongoing restricted gene flow or dispersal, historical range expansion, ancestral allopatric fragmentation, hybridization, and inadequate sampling.</p>


<h4>Shared Phylogenetic History and Dispersion</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We determined which localities shared similar H5N1 evolutionary histories by conducting a principal components analysis on the UniFrac metric for the phylogenetic tree. UniFrac measures the phylogenetic distance between the samples of two localities in terms of the branch length in the tree that is unique to each locality <a href="#pone.0001697-Lozupone1">[19]</a>, <a href="#pone.0001697-Lozupone2">[64]</a>. The pair of localities with the smallest UniFrac metric shares the greatest amount of evolutionary history. UniFrac was weighted here by the relative abundances across localities and standardized by the different rates of evolution across branches.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>We next tested for geographic structure in the extent to which H5N1 isolates were distributed across the phylogenetic tree. Two measures of phylogenetic dispersion, the net relatedness index (NRI) and the nearest taxon index (NTI), represent standardized effect sizes of community phylogenetic structure <a href="#pone.0001697-Webb1">[21]</a>, <a href="#pone.0001697-Webb2">[65]</a>. They define for each locality the differences between the average phylogenetic distance among samples for the observed data and their analogues for randomized null data, scaled by the standard deviation of the null data. The phylogenetic distances were derived from the original maximum likelihood tree. NRI is defined for the mean phylogenetic distance (MPD) among a locality's samples, measuring dispersion over the whole tree. NTI is defined for the mean phylogenetic distance between nearest tree neighbors (MNND), measuring the extent to which locality samples are clustered at the tree's tips.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>The significance of the dispersion indices was determined in Phylocom 3.41 <a href="#pone.0001697-Webb3">[66]</a> via one-tail tests against a null model of 999 randomized trials and 50 independent swaps. For each independent swap, the cells of submatrices in the locality matrix defined by presence/absence “checkerboards” are swapped <a href="#pone.0001697-Gotelli1">[67]</a>, <a href="#pone.0001697-Gotelli2">[68]</a>. The marginals of the locality matrix for each swap are constrained to those of the original matrix, preserving the localities' sample frequencies.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>To determine whether the phylogenetic measures reviewed here were related to each other or to spatial distance or sample size, we conducted global and <em>a posteriori</em> convergence tests and a generalized pairwise Mantel test for correspondence in CADM <a href="#pone.0001697-Legendre1">[27]</a>, <a href="#pone.0001697-Legendre2">[69]</a> for six distance matrices defined for the H5N1 hemagglutinin phylogeny. The six matrices were 1) MigraPhyla migration events through the phylogeny symmetricized across the migration matrix diagonal (SME), showing average two-way traffic between localities; 2) differences in isolate sample size (SS); 3) geographic distances (Geo); 4) UniFrac distances; 5) MPD distances; and 6) MNND distances.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>The CADM method, based on Friedman's chi-square and Kendall's coefficient of concordance <em>W</em>, tests the null of overall incongruence across all inputted distance matrices <a href="#pone.0001697-Legendre1">[27]</a>. The significance of the chi-square is determined by way of a one-tail Mantel-like permutation test of, in this case, 9999 trials, permuting the labels of the matrices' rows and columns (and not the distances themselves or their ranks). If the null is rejected, the <em>a posteriori</em> tests can be used to identify the incongruent matrices. Unlike the global test, in which all the matrices in the analysis are permutated, only a single distance matrix is permutated at a time for the <em>a posteriori</em> tests. Finally, Mantel tests can be conducted to identify pairs of congruent matrices. The tests are based on Spearman's correlation coefficient and conducted in round robin fashion across the matrices.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001697"></div><a name="pone.0001697.s001" id="pone.0001697.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Study H5N1 hemagglutinin (HA) and neuraminidase (NA) sequences by sample locality.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.08 MB PDF)</p>
<a name="pone.0001697.s002" id="pone.0001697.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Global and a posteriori convergence across distance matrices (CADM) tests for six distance matrices defined by the H5N1 hemagglutinin phylogeny.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001697.s003" id="pone.0001697.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">a) Maximum likelihood radial phylogeny for 481 sequences of H5N1 hemagglutinin (HA) sampled across 28 localities in Eurasia and Africa. Tree is rooted by isolate A/chicken/Scotland/59. b) 50% majority rule consensus tree with bootstrap values for 1000 replicates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.16 MB PDF)</p>
<a name="pone.0001697.s004" id="pone.0001697.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Migration events defined via DELTRAN parsimony through H5N1 maximum likelihood phylogeny of 481 hemagglutinin sequences sampled across 28 localities in Eurasia and Africa.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.25 MB PDF)</p>
<a name="pone.0001697.s005" id="pone.0001697.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s005">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">H5N1 migration events assigned on H5N1 hemagglutinin tree by DELTRAN parsimony. Change in color represents migration event. PAUP assignments may represent only one of several possible most-parsimonious traces. Mesquite trace here shows several nodes support multiple localities. Branches that support multiple localities are striped with associated colors. For the best view of isolate names and branch colors, zoom in to at least 800% in Adobe Acrobat.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.08 MB PDF)</p>
<a name="pone.0001697.s006" id="pone.0001697.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s006">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Map of DELTRAN H5N1 migration events inferred through maximum-likelihood phylogeny for 429 hemagglutinin (a) and neuraminidase (b) nucleotide sequences sampled across 28 Eurasian and African localities 1996–2006 (1≤n≤64 isolates per locality). Orange vectors are statistically significant (α = .05) under an upper-tail Monte Carlo test of 10,000 trials and a sparse false discovery rate (sFDR) correction. Non-significant vectors are color-coded by Monte Carlo <em>P</em> value: the brighter the yellow, the greater the support. Quintiles are defined by breaks in ranked <em>P</em> values of more than .01, except within the final quintile. The map is based on satellite photos made available in World Wind 1.4 (<a href="http://worldwind.arc.nasa.gov/">http://worldwind.arc.nasa.gov/</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.54 MB PDF)</p>
<a name="pone.0001697.s007" id="pone.0001697.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s007">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Net relatedness index (NRI), nearest taxon index (NTI), and associated upper-tail <em>P</em> values across 28 localities listed by great circle distance from Guangdong. The original phylogenetic tree of 481 hemagglutinin sequences from which the indices were calculated was constructed by maximum likelihood (<a href="#pone.0001697.s003">Figure S1</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.11 MB PDF)</p>
<a name="pone.0001697.s008" id="pone.0001697.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001697.s008">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">A scalar product test for phylogeographic concordance. Degrees difference between eigenvectors for a round robin comparison of migration matrices for the H5N1 hemagglutinin phylogeny of 429 sequences (HA429), the neuraminidase phylogeny of 429 sequences (NA429), and the hemagglutinin phylogeny of 481 sequences (HA481). For HA481 migration events to Burkina Faso were left out as no Burkina Faso neuraminidase sequences were available. For clarity's sake, only the real components of the differences in the eigenvectors are shown. The imaginary components, between eigenvectors 3 and 11 across the three comparisons, suggest pulsing transient diffusion for H5N1 phylogeography. Many of the eigenvectors differ orthogonally (by around 90 degrees) with the smallest average differences for HA429/NA429 and NA429/HA481.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001697.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.16 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Cadhla Ramsden for comments on MigraPhyla, Hal Stern and Rodrick Wallace for general discussions about statistics, Cam Webb for correspondence about methods for detecting phylogenetic structure, and UCI's ecological genetics study group for invaluable advice about presentation. We also thank our two reviewers and the hundreds of scientists and technicians involved in sequencing the nucleotide samples used in this study. As requested, we acknowledge the Indonesian Ministry of Health in providing access to the Indonesian H5N1 sequences used here.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: WF RW. Performed the experiments: RW. Analyzed the data: RW. Contributed reagents/materials/analysis tools: WF RW. Wrote the paper: WF RW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001697-Yuen1" id="pone.0001697-Yuen1"></a>Yuen KY, Wong SS (2005) Human infection by avian influenza A H5N1. Hong Kong Med J  11: 189–199 (2005).  <ul class="find" data-citedArticleID="1064872"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+infection+by+avian+influenza+A+H5N1.&amp;auth=&amp;atitle=Human+infection+by+avian+influenza+A+H5N1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+infection+by+avian+influenza+A+H5N1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+infection+by+avian+influenza+A+H5N1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001697-Kandun1" id="pone.0001697-Kandun1"></a>Kandun IN, Wibisono H, Sedyaningsih ER, Yusharmen , Hadisoedarsuno W, et al.  (2006) Three Indonesian clusters of H5N1 virus infection in 2005. N Engl J Med  355: 2186–2194.  <ul class="find" data-citedArticleID="1064788" data-doi="10.1056/nejmoa060930"><li><a href="http://dx.doi.org/10.1056/nejmoa060930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+Indonesian+clusters+of+H5N1+virus+infection+in+2005." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+Indonesian+clusters+of+H5N1+virus+infection+in+2005.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001697-Yang1" id="pone.0001697-Yang1"></a>Yang Y, Halloran ME, Sugimoto J, Longini IM (2007) Detecting human-to-human transmission of avian influenza A (H5N1). Emerg Infect Dis.  13: 1348–1353.  <ul class="find" data-citedArticleID="1064870" data-doi="10.3201/eid1309.070111"><li><a href="http://dx.doi.org/10.3201/eid1309.070111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+human-to-human+transmission+of+avian+influenza+A+%28H5N1%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+human-to-human+transmission+of+avian+influenza+A+%28H5N1%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001697-Salzberg1" id="pone.0001697-Salzberg1"></a>Salzberg SL, Kingsford C, Cattoli G, Spiro DJ, Janies DA, et al.  (2007) Genome analysis linking recent European and African influenza (H5N1) viruses. Emerg Infect Dis  13: 713–718.  <ul class="find" data-citedArticleID="1064838" data-doi="10.3201/eid1305.070013"><li><a href="http://dx.doi.org/10.3201/eid1305.070013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+analysis+linking+recent+European+and+African+influenza+%28H5N1%29+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+analysis+linking+recent+European+and+African+influenza+%28H5N1%29+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001697-Rweyemamu1" id="pone.0001697-Rweyemamu1"></a>Rweyemamu M, Paskin R, Benkirane A, Martin V, Roeder P, et al.  (2000) Emerging diseases of Africa and the Middle East. Annals of New York Academy of Sciences  916: 61–70.  <ul class="find" data-citedArticleID="1064836" data-doi="10.1111/j.1749-6632.2000.tb05275.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.2000.tb05275.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emerging+diseases+of+Africa+and+the+Middle+East." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emerging+diseases+of+Africa+and+the+Middle+East.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001697-Fasina1" id="pone.0001697-Fasina1"></a>Fasina FO, Bisschop SP, Webster RG (2007) Avian influenza H5N1 in Africa: an epidemiological twist. Lancet Infect Dis.  7: 696–697.  <ul class="find" data-citedArticleID="1064766" data-doi="10.1016/s1473-3099(07)70244-x"><li><a href="http://dx.doi.org/10.1016/s1473-3099(07)70244-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Avian+influenza+H5N1+in+Africa%3A+an+epidemiological+twist." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Avian+influenza+H5N1+in+Africa%3A+an+epidemiological+twist.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001697-Chen1" id="pone.0001697-Chen1"></a>Chen H, Smith GJD, Li KS, Wang J, Fan XH, et al.  (2006) Establishment of multiple sublineages of H5N1 influenza virus in Asia: implications for pandemic control. Proc Natl Acad Sci U S A  103: 2845–2850.  <ul class="find" data-citedArticleID="1064744" data-doi="10.1073/pnas.0511120103"><li><a href="http://dx.doi.org/10.1073/pnas.0511120103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Establishment+of+multiple+sublineages+of+H5N1+influenza+virus+in+Asia%3A+implications+for+pandemic+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Establishment+of+multiple+sublineages+of+H5N1+influenza+virus+in+Asia%3A+implications+for+pandemic+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001697-Smith1" id="pone.0001697-Smith1"></a>Smith GJD, Fan XH, Wang J, Li KS, Qin K, Zhang JX, et al.  (2006) Emergence and predominance of an H5N1 influenza variant in China. Proc Natl Acad Sci U S A  103: 16936–16941.  <ul class="find" data-citedArticleID="1064842" data-doi="10.1073/pnas.0608157103"><li><a href="http://dx.doi.org/10.1073/pnas.0608157103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+and+predominance+of+an+H5N1+influenza+variant+in+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+and+predominance+of+an+H5N1+influenza+variant+in+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001697-Wallace1" id="pone.0001697-Wallace1"></a>Wallace RG, HoDac H, Lathrop RH, Fitch WM (2007) A statistical phylogeography of influenza A H5N1. Proc Natl Acad Sci U S A  104: 4473–4478.  <ul class="find" data-citedArticleID="1064858" data-doi="10.1073/pnas.0700435104"><li><a href="http://dx.doi.org/10.1073/pnas.0700435104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+statistical+phylogeography+of+influenza+A+H5N1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+statistical+phylogeography+of+influenza+A+H5N1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001697-HoDac1" id="pone.0001697-HoDac1"></a>HoDac H, Lathrop RH, Wallace RG, Fitch WM (2007) MigraPhyla: Statistical Analysis of Migration Events through a Phylogeny. Version 1.0b manual.   Currently available at <a href="http://pd.bio.uci.edu/ee/WallaceR/MigraPhyla.html">http://pd.bio.uci.edu/ee/WallaceR/MigraP​hyla.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001697-Cliff1" id="pone.0001697-Cliff1"></a>Cliff AD, Haggett P, Ord JK (1986) Spatial aspects of influenza epidemics. London: Pion.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001697-Janies1" id="pone.0001697-Janies1"></a>Janies D, Hill AW, Guralnick R, Habib F, Waltari E, et al.  (2007) Genomic analysis and geographic visualization of the spread of avian influenza (H5N1). Syst Biol  56: 321–329.  <ul class="find" data-citedArticleID="1064786"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genomic+analysis+and+geographic+visualization+of+the+spread+of+avian+influenza+%28H5N1%29.&amp;auth=&amp;atitle=Genomic+analysis+and+geographic+visualization+of+the+spread+of+avian+influenza+%28H5N1%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analysis+and+geographic+visualization+of+the+spread+of+avian+influenza+%28H5N1%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analysis+and+geographic+visualization+of+the+spread+of+avian+influenza+%28H5N1%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001697-Smith2" id="pone.0001697-Smith2"></a>Smith GJ, Naipospos TS, Nguyen TD, de Jong MD, Vijaykrishna D, et al.  (2006) Evolution and adaptation of H5N1 influenza virus in avian and human hosts in Indonesia and Vietnam. Virology.  350: 258–268.  <ul class="find" data-citedArticleID="1064844" data-doi="10.1016/j.virol.2006.03.048"><li><a href="http://dx.doi.org/10.1016/j.virol.2006.03.048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+and+adaptation+of+H5N1+influenza+virus+in+avian+and+human+hosts+in+Indonesia+and+Vietnam." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+and+adaptation+of+H5N1+influenza+virus+in+avian+and+human+hosts+in+Indonesia+and+Vietnam.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001697-Gibson1" id="pone.0001697-Gibson1"></a>Gibson GC, Ostrom E, Ahn TK (2000) The concept of scale and the human dimensions of global change: a survey. Ecological Economics  32: 217–239.  <ul class="find" data-citedArticleID="1064774" data-doi="10.1016/s0921-8009(99)00092-0"><li><a href="http://dx.doi.org/10.1016/s0921-8009(99)00092-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+concept+of+scale+and+the+human+dimensions+of+global+change%3A+a+survey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+concept+of+scale+and+the+human+dimensions+of+global+change%3A+a+survey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001697-Chutinimitkul1" id="pone.0001697-Chutinimitkul1"></a>Chutinimitkul S, Songserm T, Amonsin A, Payungporn S, Suwannakarn K, et al.  (2007) New strain of influenza A virus (H5N1), Thailand. Emerg Infect Dis  13: 506–507.  <ul class="find" data-citedArticleID="1064750" data-doi="10.3201/eid1303.061103"><li><a href="http://dx.doi.org/10.3201/eid1303.061103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+strain+of+influenza+A+virus+%28H5N1%29%2C+Thailand." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+strain+of+influenza+A+virus+%28H5N1%29%2C+Thailand.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001697-Templeton1" id="pone.0001697-Templeton1"></a>Templeton AR (1998) Nested clade analyses of phylogeographic data: testing hypotheses about gene flow and population history. Molecular Ecology  7: 381–397.  <ul class="find" data-citedArticleID="1064850" data-doi="10.1046/j.1365-294x.1998.00308.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.1998.00308.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nested+clade+analyses+of+phylogeographic+data%3A+testing+hypotheses+about+gene+flow+and+population+history." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nested+clade+analyses+of+phylogeographic+data%3A+testing+hypotheses+about+gene+flow+and+population+history.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001697-Templeton2" id="pone.0001697-Templeton2"></a>Templeton AR (2004) Statistical phylogeography: methods of evaluating and minimizing inference errors. Molecular Ecology  13: 789–809.  <ul class="find" data-citedArticleID="1064852" data-doi="10.1046/j.1365-294x.2003.02041.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.02041.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+phylogeography%3A+methods+of+evaluating+and+minimizing+inference+errors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+phylogeography%3A+methods+of+evaluating+and+minimizing+inference+errors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001697-Posada1" id="pone.0001697-Posada1"></a>Posada D, Crandall KA, Templeton AR (2000) GeoDis: a program for the cladistic nested analysis of the geographical distribution of genetic haplotypes. Mol Ecol  9: 487–488.  <ul class="find" data-citedArticleID="1064828" data-doi="10.1046/j.1365-294x.2000.00887.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.00887.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeoDis%3A+a+program+for+the+cladistic+nested+analysis+of+the+geographical+distribution+of+genetic+haplotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeoDis%3A+a+program+for+the+cladistic+nested+analysis+of+the+geographical+distribution+of+genetic+haplotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001697-Lozupone1" id="pone.0001697-Lozupone1"></a>Lozupone C, Knight R (2005) UniFrac: A new phylogenetic method for comparing microbial communities. Applied and Environmental Microbiology  71: 8228–8235.  <ul class="find" data-citedArticleID="1064810" data-doi="10.1128/aem.71.12.8228-8235.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.12.8228-8235.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UniFrac%3A+A+new+phylogenetic+method+for+comparing+microbial+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UniFrac%3A+A+new+phylogenetic+method+for+comparing+microbial+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001697-Keller1" id="pone.0001697-Keller1"></a>Keller W, Shiue CH (2007) Market integration and economic development: a long-run comparison. Review of Development Economics  11: 107–123.  <ul class="find" data-citedArticleID="1064790" data-doi="10.1111/j.1467-9361.2007.00359.x"><li><a href="http://dx.doi.org/10.1111/j.1467-9361.2007.00359.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Market+integration+and+economic+development%3A+a+long-run+comparison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Market+integration+and+economic+development%3A+a+long-run+comparison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001697-Webb1" id="pone.0001697-Webb1"></a>Webb CO, Ackerly DD, McPeek MA, Donoghue MJ (2002) Phylogenies and community ecology. Annu Rev Ecol Syst  33: 475–505.  <ul class="find" data-citedArticleID="1064862" data-doi="10.1146/annurev.ecolsys.33.010802.150448"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.33.010802.150448" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenies+and+community+ecology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenies+and+community+ecology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001697-Kilpatrick1" id="pone.0001697-Kilpatrick1"></a>Kilpatrick AM, Chmura AA, Gibbons DW, Fleischer RC, Marra PP, Daszak P (2006) Predicting the global spread of H5N1 avian influenza. Proc Natl Acad Sci U S A  103: 19368–19373.  <ul class="find" data-citedArticleID="1064794" data-doi="10.1073/pnas.0609227103"><li><a href="http://dx.doi.org/10.1073/pnas.0609227103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+the+global+spread+of+H5N1+avian+influenza." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+the+global+spread+of+H5N1+avian+influenza.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001697-Mase1" id="pone.0001697-Mase1"></a>Mase M, Eto M, Tanimura N, Imai K, Tsukamoto K, et al.  (2005) Isolation of a genotypically unique H5N1 influenza virus from duck meat imported into Japan from China. Virology  339: 101–9.  <ul class="find" data-citedArticleID="1064818" data-doi="10.1016/j.virol.2005.05.010"><li><a href="http://dx.doi.org/10.1016/j.virol.2005.05.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+a+genotypically+unique+H5N1+influenza+virus+from+duck+meat+imported+into+Japan+from+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+a+genotypically+unique+H5N1+influenza+virus+from+duck+meat+imported+into+Japan+from+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001697-Ducatez1" id="pone.0001697-Ducatez1"></a>Ducatez MF, Olinger CM, Owoade AA, De Landtsheer S, Ammerlaan W, et al.  (2006) Avian flu: multiple introductions of H5N1 in Nigeria. Nature  442: 37.  <ul class="find" data-citedArticleID="1064760" data-doi="10.1038/442037a"><li><a href="http://dx.doi.org/10.1038/442037a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Avian+flu%3A+multiple+introductions+of+H5N1+in+Nigeria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Avian+flu%3A+multiple+introductions+of+H5N1+in+Nigeria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001697-Ducatez2" id="pone.0001697-Ducatez2"></a>Ducatez MF, Olinger CM, Owoade AA, Tarnagda Z, Tahita MC, et al.  (2007) Molecular and antigenic evolution and geographical spread of H5N1 highly pathogenic avian influenza viruses in western Africa. Journal of General Virology  88: 2297–2306.  <ul class="find" data-citedArticleID="1064762" data-doi="10.1099/vir.0.82939-0"><li><a href="http://dx.doi.org/10.1099/vir.0.82939-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+antigenic+evolution+and+geographical+spread+of+H5N1+highly+pathogenic+avian+influenza+viruses+in+western+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+antigenic+evolution+and+geographical+spread+of+H5N1+highly+pathogenic+avian+influenza+viruses+in+western+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001697-Kolbe1" id="pone.0001697-Kolbe1"></a>Kolbe JJ, Glor RE, Schettino LR, Lara AC, Larson A, et al.  (2004) Genetic variation increases during biological invasion by a Cuban lizard. Nature  431: 177–181.  <ul class="find" data-citedArticleID="1064800" data-doi="10.1038/nature02807"><li><a href="http://dx.doi.org/10.1038/nature02807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+increases+during+biological+invasion+by+a+Cuban+lizard." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+increases+during+biological+invasion+by+a+Cuban+lizard.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001697-Legendre1" id="pone.0001697-Legendre1"></a>Legendre P, Lapointe F-J (2004) Assessing the congruence among distance matrices: single malt scotch whiskies revisited. Aust N Z J Stat  46: 615–629.  <ul class="find" data-citedArticleID="1064804" data-doi="10.1111/j.1467-842x.2004.00357.x"><li><a href="http://dx.doi.org/10.1111/j.1467-842x.2004.00357.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+the+congruence+among+distance+matrices%3A+single+malt+scotch+whiskies+revisited." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+the+congruence+among+distance+matrices%3A+single+malt+scotch+whiskies+revisited.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001697-Oyana1" id="pone.0001697-Oyana1"></a>Oyana TJ, Dai D, Scott KE (2006) Spatiotemporal distributions of reported cases of the avian influenza H5N1 (bird flu) in Southern China in early 2004. Avian Diseases  50: 508–515.  <ul class="find" data-citedArticleID="1064824" data-doi="10.1637/7597-040506.1"><li><a href="http://dx.doi.org/10.1637/7597-040506.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatiotemporal+distributions+of+reported+cases+of+the+avian+influenza+H5N1+%28bird+flu%29+in+Southern+China+in+early+2004." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatiotemporal+distributions+of+reported+cases+of+the+avian+influenza+H5N1+%28bird+flu%29+in+Southern+China+in+early+2004.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001697-Gilbert1" id="pone.0001697-Gilbert1"></a>Gilbert M, Xiao X, Wint W, Slingenbergh J (2007)  Poultry production dynamics, bird migration cycles, and the emergence of highly pathogenic avian influenza in East and Southeast Asia. In: Sauerborn R, Valérie LR, editors. Global environmental change and infectious diseases: Impacts and adaption strategies. Berlin: Springer Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0001697-Songserm1" id="pone.0001697-Songserm1"></a>Songserm T, Jam-on R, Sae-Heng N, Meemak N, Hulse-Post DJ, et al.  (2006) Domestic ducks and H5N1 influenza epidemic, Thailand. Emerging Infectious Diseases,  12: 575–581.  <ul class="find" data-citedArticleID="1064846" data-doi="10.3201/eid1204.051614"><li><a href="http://dx.doi.org/10.3201/eid1204.051614" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domestic+ducks+and+H5N1+influenza+epidemic%2C+Thailand." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domestic+ducks+and+H5N1+influenza+epidemic%2C+Thailand.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001697-Kilpatrick2" id="pone.0001697-Kilpatrick2"></a>Kilpatrick AM, Chmura AA, Gibbons DW, Fleischer RC, Marra PP, Daszak P (2006) Predicting the global spread of H5N1 avian influenza. Proc Natl Acad Sci U S A.  103: 19368–19373.  <ul class="find" data-citedArticleID="1064796" data-doi="10.1073/pnas.0609227103"><li><a href="http://dx.doi.org/10.1073/pnas.0609227103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+the+global+spread+of+H5N1+avian+influenza." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+the+global+spread+of+H5N1+avian+influenza.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001697-Flint1" id="pone.0001697-Flint1"></a>Flint PL (2007) Applying the scientific method when assessing the influence of migratory birds on the dispersal of H5N1. Virology Journal  4: 132.  doi:10.1186/1743-422X-4-132.  <ul class="find" data-citedArticleID="1064772" data-doi="10.1186/1743-422x-4-132"><li><a href="http://dx.doi.org/10.1186/1743-422x-4-132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applying+the+scientific+method+when+assessing+the+influence+of+migratory+birds+on+the+dispersal+of+H5N1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applying+the+scientific+method+when+assessing+the+influence+of+migratory+birds+on+the+dispersal+of+H5N1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001697-Kembel1" id="pone.0001697-Kembel1"></a>Kembel SW, Hubbell SP (2006) The phylogenetic structure of a neotropical forest tree community. Ecology  87: S86–S99.  <ul class="find" data-citedArticleID="1064792" data-doi="10.1890/0012-9658(2006)87[86:tpsoan]2.0.co;2"><li><a href="http://dx.doi.org/10.1890/0012-9658(2006)87[86:tpsoan]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phylogenetic+structure+of+a+neotropical+forest+tree+community." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phylogenetic+structure+of+a+neotropical+forest+tree+community.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001697-Kraft1" id="pone.0001697-Kraft1"></a>Kraft NJB, Cornwell WK, Webb CO, Ackerly DD (2007) Trait evolution, community assembly, and the phylogenetic structure of ecological communities. The American Naturalist  170: 271–283.  <ul class="find" data-citedArticleID="1064802" data-doi="10.1086/519400"><li><a href="http://dx.doi.org/10.1086/519400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trait+evolution%2C+community+assembly%2C+and+the+phylogenetic+structure+of+ecological+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trait+evolution%2C+community+assembly%2C+and+the+phylogenetic+structure+of+ecological+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001697-Pennington1" id="pone.0001697-Pennington1"></a>Pennington RT, Richardson JA, Lavin M (2006) Insights into the historical construction of species-rich biomes from dated plant phylogenies, phylogenetic community structure and neutral ecological theory. New Phytologist  172: 605–616.  <ul class="find" data-citedArticleID="1064826" data-doi="10.1111/j.1469-8137.2006.01902.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.2006.01902.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+the+historical+construction+of+species-rich+biomes+from+dated+plant+phylogenies%2C+phylogenetic+community+structure+and+neutral+ecological+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+the+historical+construction+of+species-rich+biomes+from+dated+plant+phylogenies%2C+phylogenetic+community+structure+and+neutral+ecological+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001697-Li1" id="pone.0001697-Li1"></a>Li KS, Guan Y, Wang J, Smith GJD, Xu KM, et al.  (2004) Genesis of a highly pathogenic and potentially pandemic H5N1 influenza virus in eastern Asia. Nature  430: 209–213.  <ul class="find" data-citedArticleID="1064808" data-doi="10.1038/nature02746"><li><a href="http://dx.doi.org/10.1038/nature02746" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genesis+of+a+highly+pathogenic+and+potentially+pandemic+H5N1+influenza+virus+in+eastern+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genesis+of+a+highly+pathogenic+and+potentially+pandemic+H5N1+influenza+virus+in+eastern+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001697-Wallace2" id="pone.0001697-Wallace2"></a>Wallace R, Wallace D, Andrews H, Fullilove M, Fullilove R (1995) The spatiotemporal dynamics of AIDS and TB in the New York Metropolitan region from a sociogeographic viewpoint: understanding the linkages of central city and suburbs. Environment and Planning A.  27: 1085–1108.  <ul class="find" data-citedArticleID="1064860" data-doi="10.1068/a271085"><li><a href="http://dx.doi.org/10.1068/a271085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+spatiotemporal+dynamics+of+AIDS+and+TB+in+the+New+York+Metropolitan+region+from+a+sociogeographic+viewpoint%3A+understanding+the+linkages+of+central+city+and+suburbs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+spatiotemporal+dynamics+of+AIDS+and+TB+in+the+New+York+Metropolitan+region+from+a+sociogeographic+viewpoint%3A+understanding+the+linkages+of+central+city+and+suburbs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001697-Ferguson1" id="pone.0001697-Ferguson1"></a>Ferguson NM, Cummings DAT, Cauchemez S, Fraser C, Riley S, et al.  (2005) Strategies for containing an emerging influenza pandemic in Southeast Asia. Nature  437: 209–214.  <ul class="find" data-citedArticleID="1064768" data-doi="10.1038/nature04017"><li><a href="http://dx.doi.org/10.1038/nature04017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strategies+for+containing+an+emerging+influenza+pandemic+in+Southeast+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strategies+for+containing+an+emerging+influenza+pandemic+in+Southeast+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001697-Bush1" id="pone.0001697-Bush1"></a>Bush RM, Bender CA, Subbarao K, Cox NJ, Fitch WM (1999) Predicting the evolution of human influenza A. Science  286: 1921–1925.  <ul class="find" data-citedArticleID="1064740" data-doi="10.1126/science.286.5446.1921"><li><a href="http://dx.doi.org/10.1126/science.286.5446.1921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+the+evolution+of+human+influenza+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+the+evolution+of+human+influenza+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001697-Wolf1" id="pone.0001697-Wolf1"></a>Wolf YI, Viboud C, Holmes EC, Koonin EV, Lipman DJ (2006) Long intervals of stasis punctuated by bursts of positive selection in the seasonal evolution of influenza A virus. Biology Direct  1: 34.  <ul class="find" data-citedArticleID="1064868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Long+intervals+of+stasis+punctuated+by+bursts+of+positive+selection+in+the+seasonal+evolution+of+influenza+A+virus.&amp;auth=&amp;atitle=Long+intervals+of+stasis+punctuated+by+bursts+of+positive+selection+in+the+seasonal+evolution+of+influenza+A+virus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long+intervals+of+stasis+punctuated+by+bursts+of+positive+selection+in+the+seasonal+evolution+of+influenza+A+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long+intervals+of+stasis+punctuated+by+bursts+of+positive+selection+in+the+seasonal+evolution+of+influenza+A+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001697-Nelson1" id="pone.0001697-Nelson1"></a>Nelson MI, Simonsen L, Viboud C, Miller MA, Taylor J, et al.  (2006) Stochastic processes are key determinants of short-term evolution in influenza A virus. PLoS Pathogens  2: 1144–1151.  <ul class="find" data-citedArticleID="1064822" data-doi="10.1371/journal.ppat.0020125"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0020125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+processes+are+key+determinants+of+short-term+evolution+in+influenza+A+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+processes+are+key+determinants+of+short-term+evolution+in+influenza+A+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001697-ChunChiehShih1" id="pone.0001697-ChunChiehShih1"></a>Chun-Chieh Shih A, Hsiao T-C, Ho M-S, Li W-H (2007) Simultaneous amino acid substitutions at antigenic sites drive influenza A hemagglutinin evolution. Proc Natl Acad Sci U S A  104: 6283–6288.  <ul class="find" data-citedArticleID="1064748" data-doi="10.1073/pnas.0701396104"><li><a href="http://dx.doi.org/10.1073/pnas.0701396104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simultaneous+amino+acid+substitutions+at+antigenic+sites+drive+influenza+A+hemagglutinin+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simultaneous+amino+acid+substitutions+at+antigenic+sites+drive+influenza+A+hemagglutinin+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001697-Koelle1" id="pone.0001697-Koelle1"></a>Koelle J, Cobey S, Genfell B, Pascual M (2006) Epochal evolution shapes the phylodynamics of interpandemic influenza A (H3N2) in humans. Science  314: 1898–1903.  <ul class="find" data-citedArticleID="1064798" data-doi="10.1126/science.1132745"><li><a href="http://dx.doi.org/10.1126/science.1132745" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epochal+evolution+shapes+the+phylodynamics+of+interpandemic+influenza+A+%28H3N2%29+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epochal+evolution+shapes+the+phylodynamics+of+interpandemic+influenza+A+%28H3N2%29+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001697-Recker1" id="pone.0001697-Recker1"></a>Recker M, Pybus OG, Nee S, Gupta S (2007) The generation of influenza outbreaks by a network of host immune responses against a limited set of antigenic types. Proc Natl Acad Sci U S A  104: 7711–7716.  <ul class="find" data-citedArticleID="1064834" data-doi="10.1073/pnas.0702154104"><li><a href="http://dx.doi.org/10.1073/pnas.0702154104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+generation+of+influenza+outbreaks+by+a+network+of+host+immune+responses+against+a+limited+set+of+antigenic+types." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+generation+of+influenza+outbreaks+by+a+network+of+host+immune+responses+against+a+limited+set+of+antigenic+types.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001697-Fitch1" id="pone.0001697-Fitch1"></a>Fitch WM, Leiter X, Li Q, Palese P (1991) Positive Darwinian evolution in human influenza A viruses. Proc Natl Acad Sci U S A  88: 4270–4274.  <ul class="find" data-citedArticleID="1064770" data-doi="10.1073/pnas.88.10.4270"><li><a href="http://dx.doi.org/10.1073/pnas.88.10.4270" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+Darwinian+evolution+in+human+influenza+A+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+Darwinian+evolution+in+human+influenza+A+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001697-Gould1" id="pone.0001697-Gould1"></a>Gould S (2002) The structure of evolutionary theory. Boston: Belknap Press.   <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0001697-Epperson1" id="pone.0001697-Epperson1"></a>Epperson BK (2003) Geographical genetics. Princeton: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0001697-Benson1" id="pone.0001697-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2006) GenBank. Nucleic Acids Res  34: D16–D20.  <ul class="find" data-citedArticleID="1064738" data-doi="10.1093/nar/gkm929"><li><a href="http://dx.doi.org/10.1093/nar/gkm929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenBank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenBank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001697-Macken1" id="pone.0001697-Macken1"></a>Macken C, Lu H, Goodman J, Boykin L (2001)  The value of a database in surveillance and vaccine selection. In: Osterhaus ADME, Cox N, Hampson AW, editors. Options for the Control of Influenza IV. Amsterdam: Elsevier. pp. 103–106.  <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0001697-Chenna1" id="pone.0001697-Chenna1"></a>Chenna R, Sugawara H, Koike T, Lopez R, Gibson TJ, et al.  (2003) Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res  31: 3497–3500.  <ul class="find" data-citedArticleID="1064746" data-doi="10.1093/nar/gkg500"><li><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sequence+alignment+with+the+Clustal+series+of+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sequence+alignment+with+the+Clustal+series+of+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001697-Swofford1" id="pone.0001697-Swofford1"></a>Swofford D (2002) PAUP*: Phylogenetic Analysis Using Parsimony (And Other Methods) 4.0 beta for Windows/DOS. Sinauer, Sunderland. Sinauer.com.   <ul class="find-nolinks"></ul></li><li><span class="label">52.
              </span><a name="pone.0001697-Zwickl1" id="pone.0001697-Zwickl1"></a>Zwickl DJ (2006) Ph.D. dissertation, The University of Texas at Austin.   <a href="http://www.zo.utexas.edu/faculty/antisense/garli/Garli.html">http://www.zo.utexas.edu/faculty/antisen​se/garli/Garli.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">53.
              </span><a name="pone.0001697-Slatkin1" id="pone.0001697-Slatkin1"></a>Slatkin M, Maddison W (1989) A cladistic measure of gene flow inferred from the phylogenies of alleles. Genetics  123: 603–613.  <ul class="find" data-citedArticleID="1064840"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles.&amp;auth=&amp;atitle=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001697-Maddison1" id="pone.0001697-Maddison1"></a>Maddison WP, Maddison DR (2006) Mesquite: a modular system for evolutionary analysis.   Version 1.12 <a href="http://mesquiteproject.org">http://mesquiteproject.org</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">55.
              </span><a name="pone.0001697-Miyamoto1" id="pone.0001697-Miyamoto1"></a>Miyamoto M, Fitch WM (1995) Testing species phylogenies and phylogenetic methods with congruence. Syst Biol  44: 64–76.  <ul class="find" data-citedArticleID="1064820"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Testing+species+phylogenies+and+phylogenetic+methods+with+congruence.&amp;auth=&amp;atitle=Testing+species+phylogenies+and+phylogenetic+methods+with+congruence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Testing+species+phylogenies+and+phylogenetic+methods+with+congruence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Testing+species+phylogenies+and+phylogenetic+methods+with+congruence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001697-Templeton3" id="pone.0001697-Templeton3"></a>Templeton AR, Routman E, Phillips CA (1995) Separating population structure from population history: a cladistic analysis of the geographical distribution of mitochondrial DNA haplotypes in the tiger salamander, <em>Ambystoma tigrinum</em>. Genetics  140: 767–782.  <ul class="find" data-citedArticleID="1064854"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Separating+population+structure+from+population+history%3A+a+cladistic+analysis+of+the+geographical+distribution+of+mitochondrial+DNA+haplotypes+in+the+tiger+salamander%2C+Ambystoma+tigrinum.&amp;auth=&amp;atitle=Separating+population+structure+from+population+history%3A+a+cladistic+analysis+of+the+geographical+distribution+of+mitochondrial+DNA+haplotypes+in+the+tiger+salamander%2C+Ambystoma+tigrinum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Separating+population+structure+from+population+history%3A+a+cladistic+analysis+of+the+geographical+distribution+of+mitochondrial+DNA+haplotypes+in+the+tiger+salamander%2C+Ambystoma+tigrinum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Separating+population+structure+from+population+history%3A+a+cladistic+analysis+of+the+geographical+distribution+of+mitochondrial+DNA+haplotypes+in+the+tiger+salamander%2C+Ambystoma+tigrinum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001697-Posada2" id="pone.0001697-Posada2"></a>Posada D, Crandall KA, Templeton AR (2006) Nested clade analysis statistics. Mol Ecol Notes  6: 590–593.  <ul class="find" data-citedArticleID="1064830" data-doi="10.1111/j.1471-8286.2006.01368.x"><li><a href="http://dx.doi.org/10.1111/j.1471-8286.2006.01368.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nested+clade+analysis+statistics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nested+clade+analysis+statistics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001697-Clement1" id="pone.0001697-Clement1"></a>Clement M, Posada D, Crandall KA (2000) TCS: a computer program to estimate gene genealogies. Mol Ecol  9: 1657–1659.  <ul class="find" data-citedArticleID="1064752" data-doi="10.1046/j.1365-294x.2000.01020.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.01020.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TCS%3A+a+computer+program+to+estimate+gene+genealogies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TCS%3A+a+computer+program+to+estimate+gene+genealogies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001697-Templeton4" id="pone.0001697-Templeton4"></a>Templeton AR, Crandall KA, Sing CF (1992) A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation. Genetics  132: 619–633.  <ul class="find" data-citedArticleID="1064856"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation.&amp;auth=&amp;atitle=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001697-Crandall1" id="pone.0001697-Crandall1"></a>Crandall KA, Templeton AR (1993) Empirical tests of some predictions from coalescence theory. Genetics  134: 959–969.  <ul class="find" data-citedArticleID="1064756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Empirical+tests+of+some+predictions+from+coalescence+theory.&amp;auth=&amp;atitle=Empirical+tests+of+some+predictions+from+coalescence+theory." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+tests+of+some+predictions+from+coalescence+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+tests+of+some+predictions+from+coalescence+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001697-Posada3" id="pone.0001697-Posada3"></a>Posada D, Crandall KA (2001) Intraspecific gene genealogies: trees grafting into networks. Trends Ecol Evol  16: 37–45.  <ul class="find" data-citedArticleID="1064832" data-doi="10.1016/s0169-5347(00)02026-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)02026-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intraspecific+gene+genealogies%3A+trees+grafting+into+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intraspecific+gene+genealogies%3A+trees+grafting+into+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001697-Castelloe1" id="pone.0001697-Castelloe1"></a>Castelloe J, Templeton AR (1994) Root probabilities for intraspecific gene trees under neutral coalescent theory. Mol Phylogenet Evol  3: 102–113.  <ul class="find" data-citedArticleID="1064742" data-doi="10.1006/mpev.1994.1013"><li><a href="http://dx.doi.org/10.1006/mpev.1994.1013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Root+probabilities+for+intraspecific+gene+trees+under+neutral+coalescent+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Root+probabilities+for+intraspecific+gene+trees+under+neutral+coalescent+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001697-Crandall2" id="pone.0001697-Crandall2"></a>Crandall KA (1999)  Use of statistical parsimony in HIV analyses. In: Seillier-Moiseiwitsch F, editor. Statistics in Molecular Biology and Genetics. American Mathematical Society. pp. 186–206.  <ul class="find-nolinks"></ul></li><li><span class="label">64.
              </span><a name="pone.0001697-Lozupone2" id="pone.0001697-Lozupone2"></a>Lozupone C, Hamady M, Knight R (2006) UniFrac—an online tool for comparing microbial community diversity in a phylogenetic context. BMC Bioinformatics  7: 371.  <ul class="find" data-citedArticleID="1064812"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=UniFrac%3Fan+online+tool+for+comparing+microbial+community+diversity+in+a+phylogenetic+context.&amp;auth=&amp;atitle=UniFrac%3Fan+online+tool+for+comparing+microbial+community+diversity+in+a+phylogenetic+context." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UniFrac%E2%80%94an+online+tool+for+comparing+microbial+community+diversity+in+a+phylogenetic+context." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UniFrac%E2%80%94an+online+tool+for+comparing+microbial+community+diversity+in+a+phylogenetic+context.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001697-Webb2" id="pone.0001697-Webb2"></a>Webb CO (2000) Exploring the phylogenetic structure of ecological communities: an example for rain forest trees. The American Naturalist  156: 145–155.  <ul class="find" data-citedArticleID="1064864" data-doi="10.1086/303378"><li><a href="http://dx.doi.org/10.1086/303378" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+phylogenetic+structure+of+ecological+communities%3A+an+example+for+rain+forest+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+phylogenetic+structure+of+ecological+communities%3A+an+example+for+rain+forest+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001697-Webb3" id="pone.0001697-Webb3"></a>Webb CO, Ackerly DD, Kembel SW (2007) Phylocom: software for the analysis of community phylogenetic structure and trait evolution. Version 3.41.   <a href="http://www.phylodiversity.net/phylocom/">http://www.phylodiversity.net/phylocom/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">67.
              </span><a name="pone.0001697-Gotelli1" id="pone.0001697-Gotelli1"></a>Gotelli N (2000) Null model analysis of species co-occurrence patterns. Ecology  81: 2606–2621.  <ul class="find" data-citedArticleID="1064778" data-doi="10.1890/0012-9658(2000)081[2606:nmaosc]2.0.co;2"><li><a href="http://dx.doi.org/10.1890/0012-9658(2000)081[2606:nmaosc]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Null+model+analysis+of+species+co-occurrence+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Null+model+analysis+of+species+co-occurrence+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001697-Gotelli2" id="pone.0001697-Gotelli2"></a>Gotelli N, Entsminger GL (2003) Swap algorithms in null model analysis. Ecology  84: 532–535.  <ul class="find" data-citedArticleID="1064780" data-doi="10.1890/0012-9658(2003)084[0532:sainma]2.0.co;2"><li><a href="http://dx.doi.org/10.1890/0012-9658(2003)084[0532:sainma]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Swap+algorithms+in+null+model+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Swap+algorithms+in+null+model+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001697-Legendre2" id="pone.0001697-Legendre2"></a>Legendre P (2001) Congruence among distance matrices: Program CADM user's guide. Département de sciences biologiques, Université de Montréal.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.XML" value="120060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.PDF" value="521740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g001.PNG_L" value="9912046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g001.PNG_M" value="510498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g001.PNG_S" value="17959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g001.TIF" value="13304264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g001.PNG_I" value="154745"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g002.PNG_L" value="5995716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g002.PNG_M" value="247421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g002.PNG_S" value="19480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g002.TIF" value="8857940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g002.PNG_I" value="206405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g003.PNG_L" value="688045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g003.PNG_M" value="158857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g003.PNG_S" value="10177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g003.TIF" value="1288494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g003.PNG_I" value="59636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g004.PNG_L" value="843868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g004.PNG_M" value="74780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g004.PNG_S" value="7836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g004.TIF" value="2305428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g004.PNG_I" value="27829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g005.PNG_L" value="1864939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g005.PNG_M" value="177352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g005.PNG_S" value="9532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g005.TIF" value="3047016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.g005.PNG_I" value="59729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s001.PDF" value="82642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s002.DOC" value="20992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s003.PDF" value="163028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s004.PDF" value="246989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s005.PDF" value="79584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s006.PDF" value="536804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s007.PDF" value="113247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001697.s008.PDF" value="159159"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001697&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001697" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001697&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001697&volume=&issue=&title=Influenza A H5N1 Immigration Is Filtered Out at Some International Borders&author_name=Robert%20G.%20Wallace%2C%20Walter%20M.%20Fitch&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001697" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001697" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001697" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001697&amp;t=Influenza%20A%20H5N1%20Immigration%20Is%20Filtered%20Out%20at%20Some%20International%20Borders" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001697&title=Influenza%20A%20H5N1%20Immigration%20Is%20Filtered%20Out%20at%20Some%20International%20Borders&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001697&amp;title=Influenza%20A%20H5N1%20Immigration%20Is%20Filtered%20Out%20at%20Some%20International%20Borders" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001697" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001697&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Influenza A H5N1 Immigration Is Filtered Out at Some International Borders';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001697';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Influenza%20A%20H5N1%20Immigration%20Is%20Filtered%20Out%20at%20Some%20International%20Borders http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001697" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001697" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">




















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="26220"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22China%22" title="Search for articles in the subject area:'China'"><div class="flagText">China</div></a>
              <div data-categoryid="40325" data-articleid="26220"
                   data-categoryname="China"
                   class="flagImage" title="Flag 'China' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26220"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22H5N1%22" title="Search for articles in the subject area:'H5N1'"><div class="flagText">H5N1</div></a>
              <div data-categoryid="47163" data-articleid="26220"
                   data-categoryname="H5N1"
                   class="flagImage" title="Flag 'H5N1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Influenza%22" title="Search for articles in the subject area:'Influenza'"><div class="flagText">Influenza</div></a>
              <div data-categoryid="48257" data-articleid="26220"
                   data-categoryname="Influenza"
                   class="flagImage" title="Flag 'Influenza' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="26220"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="26220"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogeography%22" title="Search for articles in the subject area:'Phylogeography'"><div class="flagText">Phylogeography</div></a>
              <div data-categoryid="32107" data-articleid="26220"
                   data-categoryname="Phylogeography"
                   class="flagImage" title="Flag 'Phylogeography' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6085'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3932'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3047&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=3357">Referee comments: Referee 1 (Daniel Perez)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
